

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":1,"applicationTime":1905,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/18/2/117/365636","@type":"ScholarlyArticle","name":"Species-specific Typing of DNA Based on Palindrome Frequency Patterns","datePublished":"2011-03-23","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/18/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2011-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr004","keywords":["comparative genomics","DNA palindrome","hierarchical clustering"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/18/2/117/365636","author":[{"name":"Lamprea-Burgunder, Estelle","affiliation":"1  Institute of Cell Biology ,  University of Bern ,  3012 Bern ,  Switzerland","@type":"Person"},{"name":"Ludin, Philipp","affiliation":"1  Institute of Cell Biology ,  University of Bern ,  3012 Bern ,  Switzerland    2  Swiss Tropical and Public Health Institute ,  Socinstrasse 57, 4002 Basel ,  Switzerland    3  University of Basel ,  4000 Basel ,  Switzerland","@type":"Person"},{"name":"Mäser, Pascal","affiliation":"1  Institute of Cell Biology ,  University of Bern ,  3012 Bern ,  Switzerland    2  Swiss Tropical and Public Health Institute ,  Socinstrasse 57, 4002 Basel ,  Switzerland    3  University of Basel ,  4000 Basel ,  Switzerland","@type":"Person"}],"description":"Abstract.  DNA in its natural, double-stranded form may contain palindromes, sequences which read the same from either side because they are identical to their ","pageStart":"117","pageEnd":"124","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/2/1/m_cover.gif?Expires=2147483647&Signature=zPrik-vQsfP8tHq7QCbk4-ZAM~7ALug60QU9E2a0HTM3jWpw4mhYvM5ZjIfk8gvh0~D~8ZHzMVsbLbVV9I~imi38eK1-WbEZj8UJvCZsVN9ocsj3MzknfjxMwhJRMN7OhfXTufW~YjTAGZRrjum43uomBF~AyUhUCSuUuosVsuEI7xRAk7wYC1Ni9rgonZdcOzQdfiuitUvT5RG~wLQZYhdUWA83IS9zyFUehXeSL1v2Ik301pcwodAx4daAPn-F6iCk6UwJJGptSyDThKiltCKPrcoMzV3V1WqixkUPlkjoJdqdq8RIeTplSpeuudNseKNAwRJXHEN70rQsx2BmAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Species-specific Typing of DNA Based on Palindrome Frequency Patterns","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/2/1/m_cover.gif?Expires=2147483647&Signature=zPrik-vQsfP8tHq7QCbk4-ZAM~7ALug60QU9E2a0HTM3jWpw4mhYvM5ZjIfk8gvh0~D~8ZHzMVsbLbVV9I~imi38eK1-WbEZj8UJvCZsVN9ocsj3MzknfjxMwhJRMN7OhfXTufW~YjTAGZRrjum43uomBF~AyUhUCSuUuosVsuEI7xRAk7wYC1Ni9rgonZdcOzQdfiuitUvT5RG~wLQZYhdUWA83IS9zyFUehXeSL1v2Ik301pcwodAx4daAPn-F6iCk6UwJJGptSyDThKiltCKPrcoMzV3V1WqixkUPlkjoJdqdq8RIeTplSpeuudNseKNAwRJXHEN70rQsx2BmAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Species-specific Typing of DNA Based on Palindrome Frequency Patterns" />
    <meta property="og:description" content="Abstract.  DNA in its natural, double-stranded form may contain palindromes, sequences which read the same from either side because they are identical to their " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/18/2/117/365636" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2011-03-23" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Lamprea-Burgunder, Estelle" /><meta name="citation_author_institution" content="Institute of Cell Biology, University of Bern, 3012 Bern, Switzerland" /><meta name="citation_author" content="Ludin, Philipp" /><meta name="citation_author_institution" content="Institute of Cell Biology, University of Bern, 3012 Bern, Switzerland" /><meta name="citation_author_institution" content="Swiss Tropical and Public Health Institute, Socinstrasse 57, 4002 Basel, Switzerland" /><meta name="citation_author_institution" content="University of Basel, 4000 Basel, Switzerland" /><meta name="citation_author" content="Mäser, Pascal" /><meta name="citation_author_institution" content="Institute of Cell Biology, University of Bern, 3012 Bern, Switzerland" /><meta name="citation_author_institution" content="Swiss Tropical and Public Health Institute, Socinstrasse 57, 4002 Basel, Switzerland" /><meta name="citation_author_institution" content="University of Basel, 4000 Basel, Switzerland" /><meta name="citation_title" content="Species-specific Typing of DNA Based on Palindrome Frequency Patterns" /><meta name="citation_firstpage" content="117" /><meta name="citation_lastpage" content="124" /><meta name="citation_doi" content="10.1093/dnares/dsr004" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="18" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2011/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Genetical implications of the structure of deoxyribonucleic acid; Nature;  citation_year=1953;  citation_volume=171;  citation_pages=964-7; " /><meta name="citation_reference" content="citation_title=Organization of DNA replication; Cold Spring Harb. Perspect. Biol.;  citation_year=2010;  citation_volume=2;  citation_pages=a000737" /><meta name="citation_reference" content="citation_title=Structure and function of type II restriction endonucleases; Nucleic Acids Res;  citation_year=2001;  citation_volume=29;  citation_pages=3705-27; " /><meta name="citation_reference" content="citation_title=Symmetry elements in DNA structure important for recognition/methylation by DNA [amino]-methyltransferases; Nucleic Acids Res;  citation_year=2004;  citation_volume=32;  citation_pages=3930-4; " /><meta name="citation_reference" content="citation_title=Functional selection and characterization of DNA binding sites for trp repressor of Escherichia coli; J. Biol. Chem;  citation_year=1994;  citation_volume=269;  citation_pages=27869-75; " /><meta name="citation_reference" content="citation_title=Identifying estrogen receptor target genes; Mol. Oncol;  citation_year=2007;  citation_volume=1;  citation_pages=138-43; " /><meta name="citation_reference" content="citation_title=CRISPR/Cas, the immune system of bacteria and archaea; Science;  citation_year=2010;  citation_volume=327;  citation_pages=167-70; " /><meta name="citation_reference" content="citation_title=Nonrandom clusters of palindromes in herpesvirus genomes; J. Comput. Biol;  citation_year=2005;  citation_volume=12;  citation_pages=331-54; " /><meta name="citation_reference" content="citation_title=Regulation of the initiation of chromosomal replication in bacteria; FEMS Microbiol. Rev;  citation_year=2007;  citation_volume=31;  citation_pages=378-87; " /><meta name="citation_reference" content="citation_title=The nucleotide sequence of IS5 from Escherichia coli; Gene;  citation_year=1981;  citation_volume=14;  citation_pages=165-74; " /><meta name="citation_reference" content="citation_title=Repeat expansion by homologous recombination in the mouse germ line at palindromic sequences; Proc. Natl Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=8326-33; " /><meta name="citation_reference" content="citation_title=Long palindromic sequences induce double-strand breaks during meiosis in yeast; Mol. Cell. Biol;  citation_year=2000;  citation_volume=20;  citation_pages=3449-58; " /><meta name="citation_reference" content="citation_title=A paucity of palindromes in phi X174; J. Theor. Biol;  citation_year=1981;  citation_volume=93;  citation_pages=143-55; " /><meta name="citation_reference" content="citation_title=Determination of bias in the relative abundance of oligonucleotides in DNA sequences; J. Comput. Biol;  citation_year=2001;  citation_volume=8;  citation_pages=151-75; " /><meta name="citation_reference" content="citation_title=Compositional biases of bacterial genomes and evolutionary implications; J. Bacteriol;  citation_year=1997;  citation_volume=179;  citation_pages=3899-913; " /><meta name="citation_reference" content="citation_title=Distribution of potential type II restriction sites (palindromes) in prokaryotes; Biochem. Biophys. Res. Commun;  citation_year=2003;  citation_volume=310;  citation_pages=280-5; " /><meta name="citation_reference" content="citation_title=Over- and under-representation of short oligonucleotides in DNA sequences; Proc. Natl Acad. Sci. USA;  citation_year=1992;  citation_volume=89;  citation_pages=1358-62; " /><meta name="citation_reference" content="citation_title=Palindrome content of the yeast Saccharomyces cerevisiae genome; Curr. Genet;  citation_year=2005;  citation_volume=47;  citation_pages=289-97; " /><meta name="citation_reference" content="citation_title=The human genome-wide distribution of DNA palindromes; Funct. Integr. Genomics;  citation_year=2007;  citation_volume=7;  citation_pages=221-7; " /><meta name="citation_reference" content="citation_title=Avoidance of palindromic words in bacterial and archaeal genomes: a close connection with restriction enzymes; Nucleic Acids Res;  citation_year=1997;  citation_volume=25;  citation_pages=2430-9; " /><meta name="citation_reference" content="citation_title=CpG-rich islands and the function of DNA methylation; Nature;  citation_year=1986;  citation_volume=321;  citation_pages=209-13; " /><meta name="citation_reference" content="citation_title=DNA mismatch correction by Very Short Patch repair may have altered the abundance of oligonucleotides in the E. coli genome; Nucleic Acids Res;  citation_year=1992;  citation_volume=20;  citation_pages=1663-8; " /><meta name="citation_reference" content="citation_title=Cluster analysis and display of genome-wide expression patterns; Proc. Natl Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=14863-8; " /><meta name="citation_reference" content="citation_title=EMBOSS: the European Molecular Biology Open Software Suite; Trends Genet;  citation_year=2000;  citation_volume=16;  citation_pages=276-7; " /><meta name="citation_reference" content="citation_title=A new synthesis in epigenetics: towards a unified function of DNA methylation from invertebrates to vertebrates; Cell. Mol. Life Sci;  citation_year=2007;  citation_volume=64;  citation_pages=2522-4; " /><meta name="citation_reference" content="citation_title=Finding functional features in Saccharomyces genomes by phylogenetic footprinting; Science;  citation_year=2003;  citation_volume=301;  citation_pages=71-6; " /><meta name="citation_reference" content="citation_title=Metagenomics: DNA sequencing of environmental samples; Nat. Rev. Genet;  citation_year=2005;  citation_volume=6;  citation_pages=805-14; " /><meta name="citation_reference" content="citation_title=Environmental shotgun sequencing: its potential and challenges for studying the hidden world of microbes; PLoS Biol;  citation_year=2007;  citation_volume=5;  citation_pages=e82" /><meta name="citation_reference" content="citation_title=Comparisons of eukaryotic genomic sequences; Proc. Natl Acad. Sci. USA;  citation_year=1994;  citation_volume=91;  citation_pages=12832-6; " /><meta name="citation_reference" content="citation_title=Heterogeneity of genomes: measures and values; Proc. Natl Acad. Sci. USA;  citation_year=1994;  citation_volume=91;  citation_pages=12837-41; " /><meta name="citation_reference" content="citation_title=Application of tetranucleotide frequencies for the assignment of genomic fragments; Environ. Microbiol.;  citation_year=2004;  citation_volume=6;  citation_pages=938-47; " /><meta name="citation_reference" content="citation_title=The Sorcerer II Global Ocean Sampling expedition: northwest Atlantic through eastern tropical Pacific; PLoS Biol;  citation_year=2007;  citation_volume=5;  citation_pages=e77" /><meta name="citation_reference" content="citation_title=Basic local alignment search tool; J. Mol. Biol.;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=Community-wide analysis of microbial genome sequence signatures; Genome Biol;  citation_year=2009;  citation_volume=10;  citation_pages=R85" /><meta name="citation_reference" content="citation_title=Capturing whole-genome characteristics in short sequences using a naive Bayesian classifier; Genome Res;  citation_year=2001;  citation_volume=11;  citation_pages=1404-9; " /><meta name="citation_reference" content="citation_title=Accurate phylogenetic classification of variable-length DNA fragments; Nat. Methods;  citation_year=2007;  citation_volume=4;  citation_pages=63-72; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/18/2/117/1043757/dsr004.pdf" /><meta name="description" content="Abstract.  DNA in its natural, double-stranded form may contain palindromes, sequences which read the same from either side because they are identical to their " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/18/2/117/365636" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/18/2/117/365636" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/18/2/117/365636");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsr004';
    </script>
<title>
	Species-specific Typing of DNA Based on Palindrome Frequency Patterns | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./365636" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BddMPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMYW1wcmVhLUJ1cmd1bmRlciwgRXN0ZWxsZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBDZWxsIEJpb2xvZ3ksIFVuaXZlcnNpdHkgb2YgQmVybiwgMzAxMiBCZXJuLCBTd2l0emVybGFuZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikx1ZGluLCBQaGlsaXBwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIENlbGwgQmlvbG9neSwgVW5pdmVyc2l0eSBvZiBCZXJuLCAzMDEyIEJlcm4sIFN3aXR6ZXJsYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU3dpc3MgVHJvcGljYWwgYW5kIFB1YmxpYyBIZWFsdGggSW5zdGl0dXRlLCBTb2NpbnN0cmFzc2UgNTcsIDQwMDIgQmFzZWwsIFN3aXR6ZXJsYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVW5pdmVyc2l0eSBvZiBCYXNlbCwgNDAwMCBCYXNlbCwgU3dpdHplcmxhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNw6RzZXIsIFBhc2NhbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBDZWxsIEJpb2xvZ3ksIFVuaXZlcnNpdHkgb2YgQmVybiwgMzAxMiBCZXJuLCBTd2l0emVybGFuZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlN3aXNzIFRyb3BpY2FsIGFuZCBQdWJsaWMgSGVhbHRoIEluc3RpdHV0ZSwgU29jaW5zdHJhc3NlIDU3LCA0MDAyIEJhc2VsLCBTd2l0emVybGFuZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVuaXZlcnNpdHkgb2YgQmFzZWwsIDQwMDAgQmFzZWwsIFN3aXR6ZXJsYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJTcGVjaWVzLXNwZWNpZmljIFR5cGluZyBvZiBETkEgQmFzZWQgb24gUGFsaW5kcm9tZSBGcmVxdWVuY3kgUGF0dGVybnMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxMTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjEyNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzcjAwNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDExLzA0LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpY2FsIGltcGxpY2F0aW9ucyBvZiB0aGUgc3RydWN0dXJlIG9mIGRlb3h5cmlib251Y2xlaWMgYWNpZDsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTUzOyAgY2l0YXRpb25fdm9sdW1lPTE3MTsgIGNpdGF0aW9uX3BhZ2VzPTk2NC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Pcmdhbml6YXRpb24gb2YgRE5BIHJlcGxpY2F0aW9uOyBDb2xkIFNwcmluZyBIYXJiLiBQZXJzcGVjdC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPWEwMDA3MzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmUgYW5kIGZ1bmN0aW9uIG9mIHR5cGUgSUkgcmVzdHJpY3Rpb24gZW5kb251Y2xlYXNlczsgTnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz0zNzA1LTI3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeW1tZXRyeSBlbGVtZW50cyBpbiBETkEgc3RydWN0dXJlIGltcG9ydGFudCBmb3IgcmVjb2duaXRpb24vbWV0aHlsYXRpb24gYnkgRE5BIFthbWlub10tbWV0aHlsdHJhbnNmZXJhc2VzOyBOdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTM5MzAtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuY3Rpb25hbCBzZWxlY3Rpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgRE5BIGJpbmRpbmcgc2l0ZXMgZm9yIHRycCByZXByZXNzb3Igb2YgRXNjaGVyaWNoaWEgY29saTsgSi4gQmlvbC4gQ2hlbTsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjk7ICBjaXRhdGlvbl9wYWdlcz0yNzg2OS03NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZnlpbmcgZXN0cm9nZW4gcmVjZXB0b3IgdGFyZ2V0IGdlbmVzOyBNb2wuIE9uY29sOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0xMzgtNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNSSVNQUi9DYXMsIHRoZSBpbW11bmUgc3lzdGVtIG9mIGJhY3RlcmlhIGFuZCBhcmNoYWVhOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTMyNzsgIGNpdGF0aW9uX3BhZ2VzPTE2Ny03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm9ucmFuZG9tIGNsdXN0ZXJzIG9mIHBhbGluZHJvbWVzIGluIGhlcnBlc3ZpcnVzIGdlbm9tZXM7IEouIENvbXB1dC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTMzMS01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVndWxhdGlvbiBvZiB0aGUgaW5pdGlhdGlvbiBvZiBjaHJvbW9zb21hbCByZXBsaWNhdGlvbiBpbiBiYWN0ZXJpYTsgRkVNUyBNaWNyb2Jpb2wuIFJldjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTM3OC04NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG51Y2xlb3RpZGUgc2VxdWVuY2Ugb2YgSVM1IGZyb20gRXNjaGVyaWNoaWEgY29saTsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MTk4MTsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTE2NS03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVwZWF0IGV4cGFuc2lvbiBieSBob21vbG9nb3VzIHJlY29tYmluYXRpb24gaW4gdGhlIG1vdXNlIGdlcm0gbGluZSBhdCBwYWxpbmRyb21pYyBzZXF1ZW5jZXM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9OTg7ICBjaXRhdGlvbl9wYWdlcz04MzI2LTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Mb25nIHBhbGluZHJvbWljIHNlcXVlbmNlcyBpbmR1Y2UgZG91YmxlLXN0cmFuZCBicmVha3MgZHVyaW5nIG1laW9zaXMgaW4geWVhc3Q7IE1vbC4gQ2VsbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTM0NDktNTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcGF1Y2l0eSBvZiBwYWxpbmRyb21lcyBpbiBwaGkgWDE3NDsgSi4gVGhlb3IuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTE5ODE7ICBjaXRhdGlvbl92b2x1bWU9OTM7ICBjaXRhdGlvbl9wYWdlcz0xNDMtNTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldGVybWluYXRpb24gb2YgYmlhcyBpbiB0aGUgcmVsYXRpdmUgYWJ1bmRhbmNlIG9mIG9saWdvbnVjbGVvdGlkZXMgaW4gRE5BIHNlcXVlbmNlczsgSi4gQ29tcHV0LiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0xNTEtNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBvc2l0aW9uYWwgYmlhc2VzIG9mIGJhY3RlcmlhbCBnZW5vbWVzIGFuZCBldm9sdXRpb25hcnkgaW1wbGljYXRpb25zOyBKLiBCYWN0ZXJpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MTc5OyAgY2l0YXRpb25fcGFnZXM9Mzg5OS05MTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc3RyaWJ1dGlvbiBvZiBwb3RlbnRpYWwgdHlwZSBJSSByZXN0cmljdGlvbiBzaXRlcyAocGFsaW5kcm9tZXMpIGluIHByb2thcnlvdGVzOyBCaW9jaGVtLiBCaW9waHlzLiBSZXMuIENvbW11bjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTA7ICBjaXRhdGlvbl9wYWdlcz0yODAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3Zlci0gYW5kIHVuZGVyLXJlcHJlc2VudGF0aW9uIG9mIHNob3J0IG9saWdvbnVjbGVvdGlkZXMgaW4gRE5BIHNlcXVlbmNlczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT04OTsgIGNpdGF0aW9uX3BhZ2VzPTEzNTgtNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBhbGluZHJvbWUgY29udGVudCBvZiB0aGUgeWVhc3QgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlIGdlbm9tZTsgQ3Vyci4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NDc7ICBjaXRhdGlvbl9wYWdlcz0yODktOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBodW1hbiBnZW5vbWUtd2lkZSBkaXN0cmlidXRpb24gb2YgRE5BIHBhbGluZHJvbWVzOyBGdW5jdC4gSW50ZWdyLiBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MjIxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUF2b2lkYW5jZSBvZiBwYWxpbmRyb21pYyB3b3JkcyBpbiBiYWN0ZXJpYWwgYW5kIGFyY2hhZWFsIGdlbm9tZXM6IGEgY2xvc2UgY29ubmVjdGlvbiB3aXRoIHJlc3RyaWN0aW9uIGVuenltZXM7IE51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MjQzMC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DcEctcmljaCBpc2xhbmRzIGFuZCB0aGUgZnVuY3Rpb24gb2YgRE5BIG1ldGh5bGF0aW9uOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTE5ODY7ICBjaXRhdGlvbl92b2x1bWU9MzIxOyAgY2l0YXRpb25fcGFnZXM9MjA5LTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgbWlzbWF0Y2ggY29ycmVjdGlvbiBieSBWZXJ5IFNob3J0IFBhdGNoIHJlcGFpciBtYXkgaGF2ZSBhbHRlcmVkIHRoZSBhYnVuZGFuY2Ugb2Ygb2xpZ29udWNsZW90aWRlcyBpbiB0aGUgRS4gY29saSBnZW5vbWU7IE51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTY2My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbHVzdGVyIGFuYWx5c2lzIGFuZCBkaXNwbGF5IG9mIGdlbm9tZS13aWRlIGV4cHJlc3Npb24gcGF0dGVybnM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9OTU7ICBjaXRhdGlvbl9wYWdlcz0xNDg2My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FTUJPU1M6IHRoZSBFdXJvcGVhbiBNb2xlY3VsYXIgQmlvbG9neSBPcGVuIFNvZnR3YXJlIFN1aXRlOyBUcmVuZHMgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0yNzYtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBuZXcgc3ludGhlc2lzIGluIGVwaWdlbmV0aWNzOiB0b3dhcmRzIGEgdW5pZmllZCBmdW5jdGlvbiBvZiBETkEgbWV0aHlsYXRpb24gZnJvbSBpbnZlcnRlYnJhdGVzIHRvIHZlcnRlYnJhdGVzOyBDZWxsLiBNb2wuIExpZmUgU2NpOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTY0OyAgY2l0YXRpb25fcGFnZXM9MjUyMi00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GaW5kaW5nIGZ1bmN0aW9uYWwgZmVhdHVyZXMgaW4gU2FjY2hhcm9teWNlcyBnZW5vbWVzIGJ5IHBoeWxvZ2VuZXRpYyBmb290cHJpbnRpbmc7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzAxOyAgY2l0YXRpb25fcGFnZXM9NzEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWV0YWdlbm9taWNzOiBETkEgc2VxdWVuY2luZyBvZiBlbnZpcm9ubWVudGFsIHNhbXBsZXM7IE5hdC4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ODA1LTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbnZpcm9ubWVudGFsIHNob3RndW4gc2VxdWVuY2luZzogaXRzIHBvdGVudGlhbCBhbmQgY2hhbGxlbmdlcyBmb3Igc3R1ZHlpbmcgdGhlIGhpZGRlbiB3b3JsZCBvZiBtaWNyb2JlczsgUExvUyBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lODIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJpc29ucyBvZiBldWthcnlvdGljIGdlbm9taWMgc2VxdWVuY2VzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTkxOyAgY2l0YXRpb25fcGFnZXM9MTI4MzItNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGV0ZXJvZ2VuZWl0eSBvZiBnZW5vbWVzOiBtZWFzdXJlcyBhbmQgdmFsdWVzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTkxOyAgY2l0YXRpb25fcGFnZXM9MTI4MzctNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFwcGxpY2F0aW9uIG9mIHRldHJhbnVjbGVvdGlkZSBmcmVxdWVuY2llcyBmb3IgdGhlIGFzc2lnbm1lbnQgb2YgZ2Vub21pYyBmcmFnbWVudHM7IEVudmlyb24uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTkzOC00NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFNvcmNlcmVyIElJIEdsb2JhbCBPY2VhbiBTYW1wbGluZyBleHBlZGl0aW9uOiBub3J0aHdlc3QgQXRsYW50aWMgdGhyb3VnaCBlYXN0ZXJuIHRyb3BpY2FsIFBhY2lmaWM7IFBMb1MgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTc3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFzaWMgbG9jYWwgYWxpZ25tZW50IHNlYXJjaCB0b29sOyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTIxNTsgIGNpdGF0aW9uX3BhZ2VzPTQwMy0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tbXVuaXR5LXdpZGUgYW5hbHlzaXMgb2YgbWljcm9iaWFsIGdlbm9tZSBzZXF1ZW5jZSBzaWduYXR1cmVzOyBHZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPVI4NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNhcHR1cmluZyB3aG9sZS1nZW5vbWUgY2hhcmFjdGVyaXN0aWNzIGluIHNob3J0IHNlcXVlbmNlcyB1c2luZyBhIG5haXZlIEJheWVzaWFuIGNsYXNzaWZpZXI7IEdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xNDA0LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFjY3VyYXRlIHBoeWxvZ2VuZXRpYyBjbGFzc2lmaWNhdGlvbiBvZiB2YXJpYWJsZS1sZW5ndGggRE5BIGZyYWdtZW50czsgTmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz02My03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE4LzIvMTE3LzEwNDM3NTcvZHNyMDA0LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBETkEgaW4gaXRzIG5hdHVyYWwsIGRvdWJsZS1zdHJhbmRlZCBmb3JtIG1heSBjb250YWluIHBhbGluZHJvbWVzLCBzZXF1ZW5jZXMgd2hpY2ggcmVhZCB0aGUgc2FtZSBmcm9tIGVpdGhlciBzaWRlIGJlY2F1c2UgdGhleSBhcmUgaWRlbnRpY2FsIHRvIHRoZWlyICIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzY1NjM2JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZL8yfhaL80CaCQ66PUNkT+kXlEwe" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f2%2f117%2f365636" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f2%2f117%2f365636" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=14229" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/18/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/2/1/m_cover.gif?Expires=2147483647&amp;Signature=zPrik-vQsfP8tHq7QCbk4-ZAM~7ALug60QU9E2a0HTM3jWpw4mhYvM5ZjIfk8gvh0~D~8ZHzMVsbLbVV9I~imi38eK1-WbEZj8UJvCZsVN9ocsj3MzknfjxMwhJRMN7OhfXTufW~YjTAGZRrjum43uomBF~AyUhUCSuUuosVsuEI7xRAk7wYC1Ni9rgonZdcOzQdfiuitUvT5RG~wLQZYhdUWA83IS9zyFUehXeSL1v2Ik301pcwodAx4daAPn-F6iCk6UwJJGptSyDThKiltCKPrcoMzV3V1WqixkUPlkjoJdqdq8RIeTplSpeuudNseKNAwRJXHEN70rQsx2BmAw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/18/2">
                        <div class="volume trailing-comma">Volume 18</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2011        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5304508">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5304508">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5304513">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5304513">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5304517">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5304517">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5304527">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5304527">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5304541">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5304541">4. Conclusion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5304543">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5304543">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5304545">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5304545">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5304547">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5304547">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/18/2/107/365532">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/18/2/125/365741">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Species-specific Typing of DNA Based on Palindrome Frequency Patterns
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Estelle Lamprea-Burgunder</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Institute of Cell Biology</div>, <div class="insititution">University of Bern</div>, <div class="addr-line">3012 Bern</div>, <div class="country">Switzerland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Estelle+Lamprea-Burgunder">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lamprea-Burgunder E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lamprea-Burgunder E&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Estelle Lamprea-Burgunder</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Philipp Ludin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Institute of Cell Biology</div>, <div class="insititution">University of Bern</div>, <div class="addr-line">3012 Bern</div>, <div class="country">Switzerland</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Swiss Tropical and Public Health Institute</div>, <div class="addr-line">Socinstrasse 57, 4002 Basel</div>, <div class="country">Switzerland</div></div><div class="aff"><div class="label -label">3</div><div class="insititution">University of Basel</div>, <div class="addr-line">4000 Basel</div>, <div class="country">Switzerland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Philipp+Ludin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ludin P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ludin P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Philipp Ludin</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Pascal M&#228;ser</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Institute of Cell Biology</div>, <div class="insititution">University of Bern</div>, <div class="addr-line">3012 Bern</div>, <div class="country">Switzerland</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Swiss Tropical and Public Health Institute</div>, <div class="addr-line">Socinstrasse 57, 4002 Basel</div>, <div class="country">Switzerland</div></div><div class="aff"><div class="label -label">3</div><div class="insititution">University of Basel</div>, <div class="addr-line">4000 Basel</div>, <div class="country">Switzerland</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +41-61-284-8338. Fax. +41-61-284-8101. E-mail: <a href="mailto:pascal.maeser@unibas.ch" target="_blank">pascal.maeser@unibas.ch</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Pascal+M%c3%a4ser">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=M&#228;ser P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;M&#228;ser P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Pascal Mäser</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Hiroyuki Toh</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="label -label">‡</div><div class="footnote-content">Present address: Roche Diagnostics, 6343 Rotkreuz, Switzerland.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 18, Issue 2, 1 April 2011, Pages 117–124, <a href='https://doi.org/10.1093/dnares/dsr004'>https://doi.org/10.1093/dnares/dsr004</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">23 March 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">20 October 2010</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">24 February 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="365636" href="/dnaresearch/article-pdf/18/2/117/1043757/dsr004.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Estelle Lamprea-Burgunder, Philipp Ludin, Pascal Mäser;  Species-specific Typing of DNA Based on Palindrome Frequency Patterns, <em>DNA Research</em>, Volume 18, Issue 2, 1 April 2011, Pages 117–124, <a href='https://doi.org/10.1093/dnares/dsr004'>https://doi.org/10.1093/dnares/dsr004</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/365636?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/365636?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/365636?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/365636?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/365636?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/365636?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/365636?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Species-specific%20Typing%20of%20DNA%20Based%20on%20Palindrome%20Frequency%20Patterns&amp;publicationDate=2011-03-23&amp;volumeNum=18&amp;issueNum=2&amp;author=Lamprea-Burgunder%2C%20Estelle%3B%20Ludin%2C%20Philipp&amp;startPage=117&amp;endPage=124&amp;contentId=10.1093%2Fdnares%2Fdsr004&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f2%2f117%2f365636" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5304508 id="5304508" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>DNA in its natural, double-stranded form may contain palindromes, sequences which read the same from either side because they are identical to their reverse complement on the sister strand. Short palindromes are underrepresented in all kinds of genomes. The frequency distribution of short palindromes exhibits more than twice the inter-species variance of non-palindromic sequences, which renders palindromes optimally suited for the typing of DNA. Here, we show that based on palindrome frequency, DNA sequences can be discriminated to the level of species of origin. By plotting the ratios of actual occurrence to expectancy, we generate palindrome frequency patterns that allow to cluster different sequences of the same genome and to assign plasmids, and in some cases even viruses to their respective host genomes. This finding will be of use in the growing field of metagenomics.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;comparative genomics&quot;">comparative genomics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;DNA palindrome&quot;">DNA palindrome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;hierarchical clustering&quot;">hierarchical clustering</a></div>                            <h2 scrollto-destination=5304513 id="5304513" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The double helix forms the structural basis of semi-conservative DNA replication.<sup><a reveal-id="DSR004C1" data-open="DSR004C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSR004C2" data-open="DSR004C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Less intuitively, it also has implications on the information content of DNA for double-stranded DNA as such only has about half the storage capacity of single-stranded DNA. This is because a given sequence and its reverse complement, while the same in the double-stranded form, are different entities in single-stranded DNA—except for those sequences which are identical to their reverse complement. Centrally symmetric when double-stranded, such sequences read the same from either 5′ end and are called DNA palindromes (e.g. 5′-GATC-3′). Consider the 4<sup>4</sup> = 256 different single-stranded sequences of length 4; only the 4<sup>2</sup> = 16 possible palindromes are unique in the double-stranded form. The remaining 240 form 120 pairs of identical sequences when complemented to the double strands (e.g. 5′-GACT-3′ and 5′-AGTC-3′). Thus, the total number of different double-stranded sequences of length 4bp is 120 + 16 = 136. It follows that the generally accepted maximal information content of 2 bit per base pair only holds true if the two sister strands can be distinguished (which requires extra information).</p><p>Given that palindromes are the only sequences which are unique in double-stranded DNA, it is not surprising that they are of particular importance in genome biology. Dimeric restriction endonucleases and DNA methyltransferases bind palindromic recognition sites.<sup><a reveal-id="DSR004C3" data-open="DSR004C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSR004C4" data-open="DSR004C4" class="link link-ref link-reveal xref-bibr">4</a></sup> The same applies to transcription factors such as the bacterial trp repressor<sup><a reveal-id="DSR004C5" data-open="DSR004C5" class="link link-ref link-reveal xref-bibr">5</a></sup> or the mammalian oestrogen receptor.<sup><a reveal-id="DSR004C6" data-open="DSR004C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Palindromes also fulfil an important role as spacers in the prokaryotic CRISPR/Cas system (clustered regularly interspaced short palindromic repeats), which forms the basis of immune memory against bacteriophages and plasmids.<sup><a reveal-id="DSR004C7" data-open="DSR004C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Viral and bacterial genomes possess palindromic replication origins.<sup><a reveal-id="DSR004C8" data-open="DSR004C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSR004C9" data-open="DSR004C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Palindromes also contribute to genome instability: as target sites for insertion sequence elements<sup><a reveal-id="DSR004C10" data-open="DSR004C10" class="link link-ref link-reveal xref-bibr">10</a></sup> and for homologous strand invasion during recombination,<sup><a reveal-id="DSR004C11" data-open="DSR004C11" class="link link-ref link-reveal xref-bibr">11</a></sup> and by inducing double-strand breaks due to hairpin-specific nucleases.<sup><a reveal-id="DSR004C12" data-open="DSR004C12" class="link link-ref link-reveal xref-bibr">12</a></sup></p><p>While statistically, palindromes are expected to occur half as often as non-palindromic sequences in double-stranded DNA, they are even rarer than this in natural DNA sequences. Short palindromes were found to be underrepresented in various genomes including bacteriophages,<sup><a reveal-id="DSR004C13" data-open="DSR004C13" class="link link-ref link-reveal xref-bibr">13</a></sup> bacteria,<sup><a reveal-id="DSR004C14  DSR004C16" data-open="DSR004C14  DSR004C16" class="link link-ref link-reveal xref-bibr">14–16</a></sup> and fungi.<sup><a reveal-id="DSR004C17" data-open="DSR004C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSR004C18" data-open="DSR004C18" class="link link-ref link-reveal xref-bibr">18</a></sup> In the human genome, palindromes were found to be underrepresented in exons but overrepresented in introns and in upstream regions of genes.<sup><a reveal-id="DSR004C19" data-open="DSR004C19" class="link link-ref link-reveal xref-bibr">19</a></sup> In bacteria, restriction endonucleases which cleave palindromic DNA recognition sites were proposed as a selective force against palindromes.<sup><a reveal-id="DSR004C20" data-open="DSR004C20" class="link link-ref link-reveal xref-bibr">20</a></sup> In vertebrate genomes, the underrepresentation of palindromes is partly attributable to the drift of CG dinucleotides to TG by deamination of methylated cytosine.<sup><a reveal-id="DSR004C21" data-open="DSR004C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Other factors accounting for the scarcity of palindromes in genomic DNA sequences are the potential adverse effects of palindromes on chromatin structure,<sup><a reveal-id="DSR004C17" data-open="DSR004C17" class="link link-ref link-reveal xref-bibr">17</a></sup> bias of the mismatch repair system,<sup><a reveal-id="DSR004C22" data-open="DSR004C22" class="link link-ref link-reveal xref-bibr">22</a></sup> and selection against palindromes to avoid inappropriate binding of transcription factors.<sup><a reveal-id="DSR004C17" data-open="DSR004C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Here, we make use of the large number of available sequenced genomes, scanning them for the occurrence of short palindromes and demonstrating that (i) the underrepresentation of short palindromes is ubiquitous and (ii) the frequency distribution of short palindromes lends itself for species-specific typing of DNA sequences.</p>                            <h2 scrollto-destination=5304517 id="5304517" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5304518 id="5304518" class="section-title" data-legacy-id=s2a>2.1. Genome sequences</h3>
<p>Genomic DNA sequences were analysed of 200 species from 10 different phylogenetic groups, 20 species per group: vertebrates, invertebrates, fungi, plants, protozoa, bacteria, archaea, mitochondria, dsDNA viruses, and retroviruses. Complete genomes or chromosomes were analysed if available, otherwise large contigs of at least 100 kb. Twenty was the maximal number of available genome sequences for groups like invertebrates or plants; to be able to compare the variances between the different groups, the number of species per group was therefore fixed to 20 (randomly selected). See <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a> for a complete list of species including accession numbers.</p>                            <h3 scrollto-destination=5304520 id="5304520" class="section-title" data-legacy-id=s2b>2.2. Calculation of palindrome expectancy</h3>
<div class="block-child-p">The number <em>N</em> of different DNA palindromes of length <em>l</em> is given by: <a id="DSR004M1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSR004M1"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsr004eq1.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr004/2/m_dsr004eq1.gif?Expires=2147483647&amp;Signature=DZYD9rGTa3j-UOWqSo9Fvzlp64mSpTb25u5hYXY8TLWWC-k6fdVIb9JeTUpHA1oZUZDjkmBnfblPIaEOQJkTsWXJW1LHxEBuddPXL7VuEfq7pZpz3azZH~~yTfgOUu0JQQjjTZronA38sVuHgo9bBACucFrUwt3r7QbLmV7QxyYozyau9btW1VLEjLJZR8-3sjzxR0SHOQ~zyoxEW9OOLuEwLKYvw3MN9kSqL6WXnzExA6Lo2wKncADk7vQLeW3UIgFw6STxjk4lQrwQX3UYshKdkNXR7TZhLzhF6kbpA1An0c9pIHy4FsFXRhv9SDQ7xjFmodI1KoTqSc55DZXNLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr004eq1.gif" /></a></div></div><div class="label">(1)</div></div> since palindromes are centrally symmetric. The expectancy (<em>E</em>) of a palindrome (pal) of length <em>l</em><sub>pal</sub> and GC ratio gc<sub>pal</sub> in a DNA sequence (seq) of length <em>l</em><sub>seq</sub> and GC ratio gc<sub>seq</sub> is: <a id="DSR004M2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSR004M2"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsr004eq2.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr004/2/m_dsr004eq2.gif?Expires=2147483647&amp;Signature=bzvACN~PMXNQ8XgFssDEjtSDb~dI8qC68PZJ-ywlapX-RQDUmqQgXxbVrRR0ZY0khWbjbDGLDW6J58r~6svS5QPPmRUIDxqBNNlzNcFwLtrTTbhHvDEfFZsid8UzaFQZXCcWrbWT8j0Bs-CIovSSiIXp3JdTcNb2BpFf1VXPNRB4fx10W-OpJKQ5Y7OXQvJMmrszofvl7LWMtIFuA38Ia-Wk87Dyj5SMOyIoBBPBR4KAaXBTE7AFWUC9GYjD6IfBJ7TCn4wuYNr~Swciqo4l4xLQgZYGqzSt2HGaQ5XFYeLBJ1~W75r0hsqpNXxT62dD2g1YB8jZ~4w7Xx7i1OeYHQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr004eq2.gif" /></a></div></div><div class="label">(2)</div></div> The ratio <em>R</em> of palindrome frequency was defined as: <a id="DSR004M3" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSR004M3"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsr004eq3.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr004/2/m_dsr004eq3.gif?Expires=2147483647&amp;Signature=OP1RUnZ16tes2Py2ULz4t3RmkVUV7~8GSZqWt0Py9jEMJH~fhuAtuesZwOg8XL7ia5grAHO45EvW6ytwhoSRIxUGl8eXkQCrwK~VqslfGXaje~v1kaYLo2QDziu4OM2RMT3HsF8Lv1NCB4TqMHX769hNJUHIjUZhEV11Vef9Mk~oBpOcN2i77ljY58JqhwAlajmfbeOWPQN8oh~0IKw-QUklw01y4SnYPeII3XBOncKaquUm3RcTPf5a8FRzqjT6AvKorlcCy15fUiNBS2Mm3bEpchGMT1-260nedKy1I5nG0i7Dq5vZsztPFAAUV1Wztmo3QVaQLkFySqLZUBFsjg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr004eq3.gif" /></a></div></div><div class="label">(3)</div></div> where <em>n</em> is the actual occurrence of the palindrome (pal) in the sequence (seq).</div>                            <h3 scrollto-destination=5304522 id="5304522" class="section-title" data-legacy-id=s2c>2.3. Counting of palindromes</h3>
<p>The counting of palindromes in DNA sequences (Fasta format) was performed with a Perl script, available on request under the GNU public licence. Input DNA sequences were first rid of all perfect repeats (word size four or longer, repeated in tandem for at least five times) to avoid a possible bias from telomeric or centromeric repeats. Then, the occurrence was counted of each of the 16 different palindromes of length 4 (Table <a reveal-id="DSR004TB1" data-open="DSR004TB1" class="link link-reveal link-table xref-fig">1</a>). To allow for comparison of variance, the same number of control sequences were included that did not contain any palindromic duplets nor compatible ends (Table <a reveal-id="DSR004TB1" data-open="DSR004TB1" class="link link-reveal link-table xref-fig">1</a>). Each of these controls was counted alongside its reverse complement to render the result independent of the DNA strand searched. The log<sub>2</sub> of the ratios <em>R</em> of occurrence to expectancy for each 4-mer palindrome was plotted as vectors of 16 components, which were clustered by average linkage based on the city-block distance (i.e. the sum of absolute differences in the components of a given pair of vectors). Clustering was performed with the programs Cluster and TreeView<sup><a reveal-id="DSR004C23" data-open="DSR004C23" class="link link-ref link-reveal xref-bibr">23</a></sup> from the Eisen lab (<a class="link link-uri" href="http://rana.lbl.gov/eisen/">http://rana.lbl.gov/eisen/</a>). </p>                        <a id="5304524" scrollto-destination="5304524"></a>
<div content-id="DSR004TB1" class="table-modal"><div class="table-wrap"><div id="DSR004TB1" class="table-wrap-title" data-id="DSR004TB1"><div class="label -label">Table 1.</div><div class="caption"><p>The 16 palindromes of length 4 and an equal number of non-palindromes, their mean ratio <em>R</em> of occurrence to expectancy, and variance of <em>R</em>, across 200 genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Palindrome </th><th><em>R</em> </th><th>Var(<em>R</em>) </th><th>Non-palindrome </th><th><em>R</em> </th><th>Var(<em>R</em>) </th></tr></thead><tbody><tr><td>AATT </td><td>0.97 </td><td>0.06 </td><td>AAGG/CCTT </td><td><strong>1.23</strong> </td><td>0.09 </td></tr><tr><td>ATAT </td><td><strong>0.85</strong> </td><td>0.05 </td><td>ACAG/CTGT </td><td>1.04 </td><td>0.08 </td></tr><tr><td>TATA </td><td><strong>0.68</strong> </td><td>0.07 </td><td>ACTG/CAGT </td><td>0.94 </td><td>0.06 </td></tr><tr><td>TTAA </td><td><strong>0.84</strong> </td><td>0.10 </td><td>AGAC/GTCT </td><td><strong>0.87</strong> </td><td>0.05 </td></tr><tr><td>ACGT </td><td><strong>0.63</strong> </td><td>0.12 </td><td>CAAC/GTTG </td><td><strong>1.10</strong> </td><td>0.06 </td></tr><tr><td>AGCT </td><td>1.13 </td><td>0.13 </td><td>CAGA/TCTG </td><td><strong>1.16</strong> </td><td>0.12 </td></tr><tr><td>CATG </td><td>0.99 </td><td>0.09 </td><td>CCAA/TTGG </td><td><strong>1.24</strong> </td><td>0.12 </td></tr><tr><td>CTAG </td><td><strong>0.67</strong> </td><td>0.14 </td><td>CTGA/TCAG </td><td>1.06 </td><td>0.08 </td></tr><tr><td>GATC </td><td><strong>0.86</strong> </td><td>0.16 </td><td>GAGA/TCTC </td><td><strong>1.14</strong> </td><td>0.09 </td></tr><tr><td>GTAC </td><td><strong>0.71</strong> </td><td>0.05 </td><td>GGAA/TTCC </td><td><strong>1.30</strong> </td><td>0.11 </td></tr><tr><td>TCGA </td><td><strong>0.84</strong> </td><td>0.39 </td><td>GTCA/TGAC </td><td><strong>0.87</strong> </td><td>0.04 </td></tr><tr><td>TGCA </td><td><strong>1.15</strong> </td><td>0.15 </td><td>GTGA/TCAC </td><td>0.94 </td><td>0.04 </td></tr><tr><td>CCGG </td><td>0.89 </td><td>0.24 </td><td>TCCT/AGGA </td><td><strong>1.29</strong> </td><td>0.12 </td></tr><tr><td>CGCG </td><td><strong>0.62</strong> </td><td>0.27 </td><td>TGAG/CTCA </td><td>1.07 </td><td>0.07 </td></tr><tr><td>GCGC </td><td><strong>0.80</strong> </td><td>0.28 </td><td>TGGT/ACCA </td><td><strong>1.13</strong> </td><td>0.08 </td></tr><tr><td>GGCC </td><td><strong>1.15</strong> </td><td>0.32 </td><td>TGTC/GACA </td><td>0.92 </td><td>0.04 </td></tr><tr><td>Overall </td><td><strong>0.86</strong> </td><td>0.19 </td><td>Overall </td><td><strong>1.08</strong> </td><td>0.10 </td></tr></tbody></table></div><div content-id="DSR004TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Palindrome </th><th><em>R</em> </th><th>Var(<em>R</em>) </th><th>Non-palindrome </th><th><em>R</em> </th><th>Var(<em>R</em>) </th></tr></thead><tbody><tr><td>AATT </td><td>0.97 </td><td>0.06 </td><td>AAGG/CCTT </td><td><strong>1.23</strong> </td><td>0.09 </td></tr><tr><td>ATAT </td><td><strong>0.85</strong> </td><td>0.05 </td><td>ACAG/CTGT </td><td>1.04 </td><td>0.08 </td></tr><tr><td>TATA </td><td><strong>0.68</strong> </td><td>0.07 </td><td>ACTG/CAGT </td><td>0.94 </td><td>0.06 </td></tr><tr><td>TTAA </td><td><strong>0.84</strong> </td><td>0.10 </td><td>AGAC/GTCT </td><td><strong>0.87</strong> </td><td>0.05 </td></tr><tr><td>ACGT </td><td><strong>0.63</strong> </td><td>0.12 </td><td>CAAC/GTTG </td><td><strong>1.10</strong> </td><td>0.06 </td></tr><tr><td>AGCT </td><td>1.13 </td><td>0.13 </td><td>CAGA/TCTG </td><td><strong>1.16</strong> </td><td>0.12 </td></tr><tr><td>CATG </td><td>0.99 </td><td>0.09 </td><td>CCAA/TTGG </td><td><strong>1.24</strong> </td><td>0.12 </td></tr><tr><td>CTAG </td><td><strong>0.67</strong> </td><td>0.14 </td><td>CTGA/TCAG </td><td>1.06 </td><td>0.08 </td></tr><tr><td>GATC </td><td><strong>0.86</strong> </td><td>0.16 </td><td>GAGA/TCTC </td><td><strong>1.14</strong> </td><td>0.09 </td></tr><tr><td>GTAC </td><td><strong>0.71</strong> </td><td>0.05 </td><td>GGAA/TTCC </td><td><strong>1.30</strong> </td><td>0.11 </td></tr><tr><td>TCGA </td><td><strong>0.84</strong> </td><td>0.39 </td><td>GTCA/TGAC </td><td><strong>0.87</strong> </td><td>0.04 </td></tr><tr><td>TGCA </td><td><strong>1.15</strong> </td><td>0.15 </td><td>GTGA/TCAC </td><td>0.94 </td><td>0.04 </td></tr><tr><td>CCGG </td><td>0.89 </td><td>0.24 </td><td>TCCT/AGGA </td><td><strong>1.29</strong> </td><td>0.12 </td></tr><tr><td>CGCG </td><td><strong>0.62</strong> </td><td>0.27 </td><td>TGAG/CTCA </td><td>1.07 </td><td>0.07 </td></tr><tr><td>GCGC </td><td><strong>0.80</strong> </td><td>0.28 </td><td>TGGT/ACCA </td><td><strong>1.13</strong> </td><td>0.08 </td></tr><tr><td>GGCC </td><td><strong>1.15</strong> </td><td>0.32 </td><td>TGTC/GACA </td><td>0.92 </td><td>0.04 </td></tr><tr><td>Overall </td><td><strong>0.86</strong> </td><td>0.19 </td><td>Overall </td><td><strong>1.08</strong> </td><td>0.10 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Values significantly deviating from 1 are given in bold (<em>P</em> &lt; 0.01, one-way ANOVA followed by Dunnett's multiple comparison test).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5304524" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR004TB1" class="table-wrap-title" data-id="DSR004TB1"><div class="label -label">Table 1.</div><div class="caption"><p>The 16 palindromes of length 4 and an equal number of non-palindromes, their mean ratio <em>R</em> of occurrence to expectancy, and variance of <em>R</em>, across 200 genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Palindrome </th><th><em>R</em> </th><th>Var(<em>R</em>) </th><th>Non-palindrome </th><th><em>R</em> </th><th>Var(<em>R</em>) </th></tr></thead><tbody><tr><td>AATT </td><td>0.97 </td><td>0.06 </td><td>AAGG/CCTT </td><td><strong>1.23</strong> </td><td>0.09 </td></tr><tr><td>ATAT </td><td><strong>0.85</strong> </td><td>0.05 </td><td>ACAG/CTGT </td><td>1.04 </td><td>0.08 </td></tr><tr><td>TATA </td><td><strong>0.68</strong> </td><td>0.07 </td><td>ACTG/CAGT </td><td>0.94 </td><td>0.06 </td></tr><tr><td>TTAA </td><td><strong>0.84</strong> </td><td>0.10 </td><td>AGAC/GTCT </td><td><strong>0.87</strong> </td><td>0.05 </td></tr><tr><td>ACGT </td><td><strong>0.63</strong> </td><td>0.12 </td><td>CAAC/GTTG </td><td><strong>1.10</strong> </td><td>0.06 </td></tr><tr><td>AGCT </td><td>1.13 </td><td>0.13 </td><td>CAGA/TCTG </td><td><strong>1.16</strong> </td><td>0.12 </td></tr><tr><td>CATG </td><td>0.99 </td><td>0.09 </td><td>CCAA/TTGG </td><td><strong>1.24</strong> </td><td>0.12 </td></tr><tr><td>CTAG </td><td><strong>0.67</strong> </td><td>0.14 </td><td>CTGA/TCAG </td><td>1.06 </td><td>0.08 </td></tr><tr><td>GATC </td><td><strong>0.86</strong> </td><td>0.16 </td><td>GAGA/TCTC </td><td><strong>1.14</strong> </td><td>0.09 </td></tr><tr><td>GTAC </td><td><strong>0.71</strong> </td><td>0.05 </td><td>GGAA/TTCC </td><td><strong>1.30</strong> </td><td>0.11 </td></tr><tr><td>TCGA </td><td><strong>0.84</strong> </td><td>0.39 </td><td>GTCA/TGAC </td><td><strong>0.87</strong> </td><td>0.04 </td></tr><tr><td>TGCA </td><td><strong>1.15</strong> </td><td>0.15 </td><td>GTGA/TCAC </td><td>0.94 </td><td>0.04 </td></tr><tr><td>CCGG </td><td>0.89 </td><td>0.24 </td><td>TCCT/AGGA </td><td><strong>1.29</strong> </td><td>0.12 </td></tr><tr><td>CGCG </td><td><strong>0.62</strong> </td><td>0.27 </td><td>TGAG/CTCA </td><td>1.07 </td><td>0.07 </td></tr><tr><td>GCGC </td><td><strong>0.80</strong> </td><td>0.28 </td><td>TGGT/ACCA </td><td><strong>1.13</strong> </td><td>0.08 </td></tr><tr><td>GGCC </td><td><strong>1.15</strong> </td><td>0.32 </td><td>TGTC/GACA </td><td>0.92 </td><td>0.04 </td></tr><tr><td>Overall </td><td><strong>0.86</strong> </td><td>0.19 </td><td>Overall </td><td><strong>1.08</strong> </td><td>0.10 </td></tr></tbody></table></div><div content-id="DSR004TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Palindrome </th><th><em>R</em> </th><th>Var(<em>R</em>) </th><th>Non-palindrome </th><th><em>R</em> </th><th>Var(<em>R</em>) </th></tr></thead><tbody><tr><td>AATT </td><td>0.97 </td><td>0.06 </td><td>AAGG/CCTT </td><td><strong>1.23</strong> </td><td>0.09 </td></tr><tr><td>ATAT </td><td><strong>0.85</strong> </td><td>0.05 </td><td>ACAG/CTGT </td><td>1.04 </td><td>0.08 </td></tr><tr><td>TATA </td><td><strong>0.68</strong> </td><td>0.07 </td><td>ACTG/CAGT </td><td>0.94 </td><td>0.06 </td></tr><tr><td>TTAA </td><td><strong>0.84</strong> </td><td>0.10 </td><td>AGAC/GTCT </td><td><strong>0.87</strong> </td><td>0.05 </td></tr><tr><td>ACGT </td><td><strong>0.63</strong> </td><td>0.12 </td><td>CAAC/GTTG </td><td><strong>1.10</strong> </td><td>0.06 </td></tr><tr><td>AGCT </td><td>1.13 </td><td>0.13 </td><td>CAGA/TCTG </td><td><strong>1.16</strong> </td><td>0.12 </td></tr><tr><td>CATG </td><td>0.99 </td><td>0.09 </td><td>CCAA/TTGG </td><td><strong>1.24</strong> </td><td>0.12 </td></tr><tr><td>CTAG </td><td><strong>0.67</strong> </td><td>0.14 </td><td>CTGA/TCAG </td><td>1.06 </td><td>0.08 </td></tr><tr><td>GATC </td><td><strong>0.86</strong> </td><td>0.16 </td><td>GAGA/TCTC </td><td><strong>1.14</strong> </td><td>0.09 </td></tr><tr><td>GTAC </td><td><strong>0.71</strong> </td><td>0.05 </td><td>GGAA/TTCC </td><td><strong>1.30</strong> </td><td>0.11 </td></tr><tr><td>TCGA </td><td><strong>0.84</strong> </td><td>0.39 </td><td>GTCA/TGAC </td><td><strong>0.87</strong> </td><td>0.04 </td></tr><tr><td>TGCA </td><td><strong>1.15</strong> </td><td>0.15 </td><td>GTGA/TCAC </td><td>0.94 </td><td>0.04 </td></tr><tr><td>CCGG </td><td>0.89 </td><td>0.24 </td><td>TCCT/AGGA </td><td><strong>1.29</strong> </td><td>0.12 </td></tr><tr><td>CGCG </td><td><strong>0.62</strong> </td><td>0.27 </td><td>TGAG/CTCA </td><td>1.07 </td><td>0.07 </td></tr><tr><td>GCGC </td><td><strong>0.80</strong> </td><td>0.28 </td><td>TGGT/ACCA </td><td><strong>1.13</strong> </td><td>0.08 </td></tr><tr><td>GGCC </td><td><strong>1.15</strong> </td><td>0.32 </td><td>TGTC/GACA </td><td>0.92 </td><td>0.04 </td></tr><tr><td>Overall </td><td><strong>0.86</strong> </td><td>0.19 </td><td>Overall </td><td><strong>1.08</strong> </td><td>0.10 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Values significantly deviating from 1 are given in bold (<em>P</em> &lt; 0.01, one-way ANOVA followed by Dunnett's multiple comparison test).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5304524" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5304525 id="5304525" class="section-title" data-legacy-id=s2d>2.4. Random controls</h3>
<p>Random sequences of variable length were generated with the program <em>makenucseq</em> of the EMBOSS package.<sup><a reveal-id="DSR004C24" data-open="DSR004C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Randomly selected, non-overlapping 10 kb fragments of bacterial genomes were generated with a self-made Perl script using <em>srand(</em>time<em>)</em> as the random number seed.</p>                            <h2 scrollto-destination=5304527 id="5304527" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5304528 id="5304528" class="section-title" data-legacy-id=s3a>3.1. Palindrome occurrence across the tree of life</h3>
<p>We counted the occurrence of the 16 palindromic words of length 4 (Table <a reveal-id="DSR004TB1" data-open="DSR004TB1" class="link link-reveal link-table xref-fig">1</a>), along with an equal number of non-palindromic words of length 4 (Table <a reveal-id="DSR004TB1" data-open="DSR004TB1" class="link link-reveal link-table xref-fig">1</a>), in DNA sequences of selected genomes. Twenty different species were analysed for each of 10 different phylogenetic groups, i.e. the vertebrates, invertebrates, fungi, plants, protozoa, mitochondria, bacteria, archaea, double-stranded DNA viruses, and retroviruses. Perfect repeats were removed from the input sequences to avoid introduction of a trivial bias from regions of extremely low complexity such as telomeric or centromeric repeats. For each input DNA sequence and each 4-mer word, we then calculated the ratio <em>R</em> of actual occurrence of the word divided by the expected number of occurrences, given its GC content and that of the input DNA sequence. Most of the palindromes were underrepresented (<em>R</em> &lt; 1) across all genomes analysed. Overall, the palindromes exhibited a mean <em>R</em> of 0.86, in contrast to a mean <em>R</em> of 1.08 for the non-palindromic controls (Table <a reveal-id="DSR004TB1" data-open="DSR004TB1" class="link link-reveal link-table xref-fig">1</a>). The underrepresentation of palindromes was most pronounced in vertebrate genomes, plants, double-stranded DNA viruses, and retroviruses (Fig. <a reveal-id="DSR004F1" data-open="DSR004F1" class="link link-reveal link-table xref-fig">1</a>). Contrary to previous reports,<sup><a reveal-id="DSR004C20" data-open="DSR004C20" class="link link-ref link-reveal xref-bibr">20</a></sup> palindromes were underrepresented even in mitochondrial genomes, demonstrating that the infrequence of palindromes in prokaryote genomes cannot solely be explained by the selective pressure exerted by restriction enzymes. Additional selective forces against palindromes might comprise their impact on DNA structure or their role as transcription factor-binding sites.<sup><a reveal-id="DSR004C17" data-open="DSR004C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Whatever the underlying forces, short palindromes are underrepresented in all kinds of genomes (Fig. <a reveal-id="DSR004F1" data-open="DSR004F1" class="link link-reveal link-table xref-fig">1</a>). Exactly which palindromes and how strongly depend on the source of the DNA. Interestingly, the inter-genome frequencies of short palindromes exhibit more than twice the variance of the non-palindromic control sequences (22 versus 9%; Table <a reveal-id="DSR004TB1" data-open="DSR004TB1" class="link link-reveal link-table xref-fig">1</a>), whereas the intra-genome frequencies, e.g. between different chromosomes of the same organism, are uniform (Figs <a reveal-id="DSR004F2  DSR004F4" data-open="DSR004F2  DSR004F4" class="link link-reveal link-table xref-fig">2–4</a>). This renders short palindromes optimally suited for the typing of DNA. </p>                        <a id="5304530" scrollto-destination="5304530"></a>
<div id="DSR004F1" data-id="DSR004F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr004/2/m_dsr00401.jpeg?Expires=2147483647&Signature=OHSBKAV2T4926x3IUYKcJIyn3OBocEX2L~7BjPHNOrpPfSPvy7r-LU7yVoyITpBNBo3tzzx4dsLIURlWYVRdmOC4X0Ch~5oxA1Lz4DTz0OgmzeV3aTTJurDhjft3xUARt-QcBbc-GdMdgIs-HKkK0tI35H~6fKpHKLF6ZC71RcDEjP7ZEpM08QEiGRFVgIxPYkF~hlyWeRN0SjTN3afGUy6OHwu8fJPVCmfzBDtawYkSbUdBDLQpDgcjr2HNxHEdxoDpdwmrpGnwD0K7mrMZGYUT9~En1ANomhu8Eur9hG7HItm6m9J-7An5T7l0dmNZvJEW7zVPWWEJT4-w9JFJgA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Frequency of palindromes throughout a diverse selection of genomes. Palindrome frequency is expressed as the ratio (R) of occurrence to expectancy. Palindromes are underrepresented (R &lt; 1, dotted line) in all kinds of genomes, most strongly in vertebrates, plants, and viruses, and they exhibit about twice the inter-species variance in frequency (error bars) than non-palindromes. Twenty different genomes were analysed per group (see Section 2)." path-from-xml="dsr00401.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5304530/dsr00401.jpeg" path-from-xml="dsr00401.jpeg" target="_blank">View large</a><a section="5304530" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr004/2/dsr00401.jpeg?Expires=2147483647&Signature=mQMuD7YDMcPVRE0w0cYbpWz6xscOFpicF9E~P57Q80EJnCUOjH6lIdmPZZ0dEY2xDYRHFOgUhuuUiE9PJ9s3bTPvaTGjJCiRjjgkRoopsKryYxTo7M-53ITFwMe4RWZ2TD3Hk0grD4me1AAuaQ45jAOtceuSDaMj5cnoj4uqmt79unTno-IYuqj2bnD4ea5vHI61jjWX9KDDoDjDnm8CT4XeJ3N9wwzvLf5xImfwRb29od~uwjYkL20CMngXAURiCSmb7OT7IpZE38tOSYLWW5uqrPtydWe~O12b~LLw0z~EBusuop6hwQlIH-JRvYu-LNe5Erq4E6GeKC9LFB0spg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5304530&ar=365636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00401.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Frequency of palindromes throughout a diverse selection of genomes. Palindrome frequency is expressed as the ratio (<em>R</em>) of occurrence to expectancy. Palindromes are underrepresented (<em>R</em> &lt; 1, dotted line) in all kinds of genomes, most strongly in vertebrates, plants, and viruses, and they exhibit about twice the inter-species variance in frequency (error bars) than non-palindromes. Twenty different genomes were analysed per group (see Section 2).</p></div></div><div content-id="DSR004F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr004/2/m_dsr00401.jpeg?Expires=2147483647&Signature=OHSBKAV2T4926x3IUYKcJIyn3OBocEX2L~7BjPHNOrpPfSPvy7r-LU7yVoyITpBNBo3tzzx4dsLIURlWYVRdmOC4X0Ch~5oxA1Lz4DTz0OgmzeV3aTTJurDhjft3xUARt-QcBbc-GdMdgIs-HKkK0tI35H~6fKpHKLF6ZC71RcDEjP7ZEpM08QEiGRFVgIxPYkF~hlyWeRN0SjTN3afGUy6OHwu8fJPVCmfzBDtawYkSbUdBDLQpDgcjr2HNxHEdxoDpdwmrpGnwD0K7mrMZGYUT9~En1ANomhu8Eur9hG7HItm6m9J-7An5T7l0dmNZvJEW7zVPWWEJT4-w9JFJgA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Frequency of palindromes throughout a diverse selection of genomes. Palindrome frequency is expressed as the ratio (R) of occurrence to expectancy. Palindromes are underrepresented (R &lt; 1, dotted line) in all kinds of genomes, most strongly in vertebrates, plants, and viruses, and they exhibit about twice the inter-species variance in frequency (error bars) than non-palindromes. Twenty different genomes were analysed per group (see Section 2)." path-from-xml="dsr00401.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5304530/dsr00401.jpeg" path-from-xml="dsr00401.jpeg" target="_blank">View large</a><a section="5304530" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr004/2/dsr00401.jpeg?Expires=2147483647&Signature=mQMuD7YDMcPVRE0w0cYbpWz6xscOFpicF9E~P57Q80EJnCUOjH6lIdmPZZ0dEY2xDYRHFOgUhuuUiE9PJ9s3bTPvaTGjJCiRjjgkRoopsKryYxTo7M-53ITFwMe4RWZ2TD3Hk0grD4me1AAuaQ45jAOtceuSDaMj5cnoj4uqmt79unTno-IYuqj2bnD4ea5vHI61jjWX9KDDoDjDnm8CT4XeJ3N9wwzvLf5xImfwRb29od~uwjYkL20CMngXAURiCSmb7OT7IpZE38tOSYLWW5uqrPtydWe~O12b~LLw0z~EBusuop6hwQlIH-JRvYu-LNe5Erq4E6GeKC9LFB0spg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5304530&ar=365636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00401.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Frequency of palindromes throughout a diverse selection of genomes. Palindrome frequency is expressed as the ratio (<em>R</em>) of occurrence to expectancy. Palindromes are underrepresented (<em>R</em> &lt; 1, dotted line) in all kinds of genomes, most strongly in vertebrates, plants, and viruses, and they exhibit about twice the inter-species variance in frequency (error bars) than non-palindromes. Twenty different genomes were analysed per group (see Section 2).</p></div></div>                        <a id="5304531" scrollto-destination="5304531"></a>
<div id="DSR004F2" data-id="DSR004F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr004/2/m_dsr00402.jpeg?Expires=2147483647&Signature=4WPyDP3-ddOuEVCETF-c4mEA~~vULZq1Psva4xU7korqkL3TXhx7H8eKJYvkIeigrN5cTtUPTAJrQ5RxYdLAGxkAiBVrGRSCUwHCryH84hilSQJk7zQoU0HKb-Px1N1HqLJUlPObbca-sL61T2hQcbpMof0LBCgfvpvpU-Bt9izeA8NvmGQOx8dZUW3I3xAuMfdXGOEC6U-oRasGZgudHGCo63gpchuw220f9OecB5W2X7kivuFPstWx3C6GvLS~trnEYy5eW3xL6xJTtHDhe0Z~AoywVImy6H9fl30sxfwHoaUs-xg0cIH2D5HEqqWyo2IsevKRpyG3hC5tHHBkXQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Examples of palindrome frequency patterns. Frequency of the 16 palindromes of length 4 in selected genomes, expressed as log2 of ratio (R) of actual to expected occurrence. Hierarchical clustering was performed based on the city-block distance.23 (Top) Mean and variance by palindrome. (Bottom) The signals from three random sequences are shown for comparison." path-from-xml="dsr00402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5304531/dsr00402.jpeg" path-from-xml="dsr00402.jpeg" target="_blank">View large</a><a section="5304531" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr004/2/dsr00402.jpeg?Expires=2147483647&Signature=s-SvP0tCeppFF9UCG4pu-yQgWLqB1IfJ1ezRFWcv-IH9xXGoZiGFizLlvknuqq12KKHyUVw0NlUvGBQ3m3Rj-MyH8VZB0ZIAx0s7RTtYllJAOjuajlAN34ynxzEABgbwhmhYscUBue3f7eYmeCWOtnAkpMd7AobU57qs582phuuiZzLqBuDmbJIuCOBdXSal56OJ4mro474ViJ9pFW~vOpzpPnhRyDM8x6DNFN-DLQpTMNWI2-56UOT7N6G80JXQspQj0jP2ZCkoDKUixgCftfWeR9ctgfwkr2rfO-rBwZgdZj4Bd3kIeW4OKTQKzQyKUfslijYxWaJqdshTj1NeuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5304531&ar=365636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Examples of palindrome frequency patterns. Frequency of the 16 palindromes of length 4 in selected genomes, expressed as log<sub>2</sub> of ratio (<em>R</em>) of actual to expected occurrence. Hierarchical clustering was performed based on the city-block distance.<sup><a reveal-id="DSR004C23" data-open="DSR004C23" class="link link-ref link-reveal xref-bibr">23</a></sup> (Top) Mean and variance by palindrome. (Bottom) The signals from three random sequences are shown for comparison.</p></div></div><div content-id="DSR004F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr004/2/m_dsr00402.jpeg?Expires=2147483647&Signature=4WPyDP3-ddOuEVCETF-c4mEA~~vULZq1Psva4xU7korqkL3TXhx7H8eKJYvkIeigrN5cTtUPTAJrQ5RxYdLAGxkAiBVrGRSCUwHCryH84hilSQJk7zQoU0HKb-Px1N1HqLJUlPObbca-sL61T2hQcbpMof0LBCgfvpvpU-Bt9izeA8NvmGQOx8dZUW3I3xAuMfdXGOEC6U-oRasGZgudHGCo63gpchuw220f9OecB5W2X7kivuFPstWx3C6GvLS~trnEYy5eW3xL6xJTtHDhe0Z~AoywVImy6H9fl30sxfwHoaUs-xg0cIH2D5HEqqWyo2IsevKRpyG3hC5tHHBkXQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Examples of palindrome frequency patterns. Frequency of the 16 palindromes of length 4 in selected genomes, expressed as log2 of ratio (R) of actual to expected occurrence. Hierarchical clustering was performed based on the city-block distance.23 (Top) Mean and variance by palindrome. (Bottom) The signals from three random sequences are shown for comparison." path-from-xml="dsr00402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5304531/dsr00402.jpeg" path-from-xml="dsr00402.jpeg" target="_blank">View large</a><a section="5304531" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr004/2/dsr00402.jpeg?Expires=2147483647&Signature=s-SvP0tCeppFF9UCG4pu-yQgWLqB1IfJ1ezRFWcv-IH9xXGoZiGFizLlvknuqq12KKHyUVw0NlUvGBQ3m3Rj-MyH8VZB0ZIAx0s7RTtYllJAOjuajlAN34ynxzEABgbwhmhYscUBue3f7eYmeCWOtnAkpMd7AobU57qs582phuuiZzLqBuDmbJIuCOBdXSal56OJ4mro474ViJ9pFW~vOpzpPnhRyDM8x6DNFN-DLQpTMNWI2-56UOT7N6G80JXQspQj0jP2ZCkoDKUixgCftfWeR9ctgfwkr2rfO-rBwZgdZj4Bd3kIeW4OKTQKzQyKUfslijYxWaJqdshTj1NeuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5304531&ar=365636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Examples of palindrome frequency patterns. Frequency of the 16 palindromes of length 4 in selected genomes, expressed as log<sub>2</sub> of ratio (<em>R</em>) of actual to expected occurrence. Hierarchical clustering was performed based on the city-block distance.<sup><a reveal-id="DSR004C23" data-open="DSR004C23" class="link link-ref link-reveal xref-bibr">23</a></sup> (Top) Mean and variance by palindrome. (Bottom) The signals from three random sequences are shown for comparison.</p></div></div>                        <a id="5304532" scrollto-destination="5304532"></a>
<div id="DSR004F3" data-id="DSR004F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr004/2/m_dsr00403.jpeg?Expires=2147483647&Signature=TTJEzed05ecf-RKLz5EuRN3iOSwBFo2I-KDt2YzQ7h5c6rxEPYRyXn0q4m7itt~rfvkauyP5YANIGFYTsOPCiotJzgPzFdJiBFlyW2-BgIP~HbtO1gBOQz4clGao~WMHRDe2z0iTLTck6b95v0UOt8B~LIycT6ieTW51pgD72hoO2alXGFV9AbYSglzElokqnQcm2lMbDm118fVTtRmCL~UwVLaMxujfXmZtU6Lb9Wvc9CZ7IlsVKnYU2LKSlv4INB9pcDra0EbtH4Z~PBzQtpQ8SV1SONveCN9TLjasmej5Oppky9z7ueFwq~ZMGxTFV~OhcbYL4APMQ7E9DMQ~oQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Variance of palindrome frequencies in random DNA sequence of different lengths (n = 20 for each). The mean variance for each palindrome of length 4 across the 20 different sequences is compared with those across the first 20 human chromosomes (dotted grey line) and across the 20 different vertebrate chromosomes analysed in Fig. 1 (see Supplementary Table S1)." path-from-xml="dsr00403.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5304532/dsr00403.jpeg" path-from-xml="dsr00403.jpeg" target="_blank">View large</a><a section="5304532" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr004/2/dsr00403.jpeg?Expires=2147483647&Signature=PejebFrZ4~jcGY0XVEao3hW~ByZ3Wucql09-9MS04B~KEuhvlxcVoYmKYSAZm8wOyAhlqplSbr7X16rLDinWPe0txijbOHeU1vF6sRAzaWuwy22DcxSXibReW63QbQlfhJ-bJdhnf9aN7zs3X9gBH~Ate2ZzxHXkkhSyUupyczSgVW4TIoFIvNzZEYrQLi1LyvnNIy1iY6sTciytpmKkN1HNB3ig0jfjiX-WjqNr8mXOf9yDL6xX6W1FDzaebsCaG9dYFiBodGD8xZ2rlw7G~bE900CjejVdKUSdGtKFf5vnZaOql3fkjAXE36F1bdRVJafoK4GZCfV-TgBNEPUK4g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5304532&ar=365636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00403.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Variance of palindrome frequencies in random DNA sequence of different lengths (<em>n</em> = 20 for each). The mean variance for each palindrome of length 4 across the 20 different sequences is compared with those across the first 20 human chromosomes (dotted grey line) and across the 20 different vertebrate chromosomes analysed in Fig. <a reveal-id="DSR004F1" data-open="DSR004F1" class="link link-reveal link-table xref-fig">1</a> (see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>).</p></div></div><div content-id="DSR004F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr004/2/m_dsr00403.jpeg?Expires=2147483647&Signature=TTJEzed05ecf-RKLz5EuRN3iOSwBFo2I-KDt2YzQ7h5c6rxEPYRyXn0q4m7itt~rfvkauyP5YANIGFYTsOPCiotJzgPzFdJiBFlyW2-BgIP~HbtO1gBOQz4clGao~WMHRDe2z0iTLTck6b95v0UOt8B~LIycT6ieTW51pgD72hoO2alXGFV9AbYSglzElokqnQcm2lMbDm118fVTtRmCL~UwVLaMxujfXmZtU6Lb9Wvc9CZ7IlsVKnYU2LKSlv4INB9pcDra0EbtH4Z~PBzQtpQ8SV1SONveCN9TLjasmej5Oppky9z7ueFwq~ZMGxTFV~OhcbYL4APMQ7E9DMQ~oQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Variance of palindrome frequencies in random DNA sequence of different lengths (n = 20 for each). The mean variance for each palindrome of length 4 across the 20 different sequences is compared with those across the first 20 human chromosomes (dotted grey line) and across the 20 different vertebrate chromosomes analysed in Fig. 1 (see Supplementary Table S1)." path-from-xml="dsr00403.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5304532/dsr00403.jpeg" path-from-xml="dsr00403.jpeg" target="_blank">View large</a><a section="5304532" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr004/2/dsr00403.jpeg?Expires=2147483647&Signature=PejebFrZ4~jcGY0XVEao3hW~ByZ3Wucql09-9MS04B~KEuhvlxcVoYmKYSAZm8wOyAhlqplSbr7X16rLDinWPe0txijbOHeU1vF6sRAzaWuwy22DcxSXibReW63QbQlfhJ-bJdhnf9aN7zs3X9gBH~Ate2ZzxHXkkhSyUupyczSgVW4TIoFIvNzZEYrQLi1LyvnNIy1iY6sTciytpmKkN1HNB3ig0jfjiX-WjqNr8mXOf9yDL6xX6W1FDzaebsCaG9dYFiBodGD8xZ2rlw7G~bE900CjejVdKUSdGtKFf5vnZaOql3fkjAXE36F1bdRVJafoK4GZCfV-TgBNEPUK4g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5304532&ar=365636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00403.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Variance of palindrome frequencies in random DNA sequence of different lengths (<em>n</em> = 20 for each). The mean variance for each palindrome of length 4 across the 20 different sequences is compared with those across the first 20 human chromosomes (dotted grey line) and across the 20 different vertebrate chromosomes analysed in Fig. <a reveal-id="DSR004F1" data-open="DSR004F1" class="link link-reveal link-table xref-fig">1</a> (see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>).</p></div></div>                        <a id="5304533" scrollto-destination="5304533"></a>
<div id="DSR004F4" data-id="DSR004F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr004/2/m_dsr00404.jpeg?Expires=2147483647&Signature=aDLqdgQFlcMq6cEPwnz6iXoDmEZv4PgDpKhQcetSyog7QFhTOuf94sFkWQlV7wqR49gTHzwkC2167fPinmnpID1DSXs-RxS0RWeY9rVucVEt~2jpSy-MUKE3vv97KfBIZuqtjfA~0XFqv-jvIJNx~m~uBaputqS9610Klx~osA2pDs9ZOalyPmFICGD3XC-CSznm2kS-erarmZLpf0MFUlOrnGd~LHdZ8e0updVqVF~~QUVUCVidleSTWnAfERKnDoNKqQDd8iCzrfk-upobl--FPDr5eIc3CyU8VO03tiUj9uA91G6pxq-di85ER7EAoyOrUPPF6aAaQkaNZCi8Iw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Case studies on Caenorhabditis spp. (A), mammalian chromosomes (B), and sensu stricto yeasts (C). Most of the chromosomes are correctly resolved by clustering based on palindrome frequency. Perfect tandem repeats were removed prior to analysis to avoid trivial differences from repetitive regions. Note the striking difference between vertebrate and invertebrate DNA." path-from-xml="dsr00404.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5304533/dsr00404.jpeg" path-from-xml="dsr00404.jpeg" target="_blank">View large</a><a section="5304533" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr004/2/dsr00404.jpeg?Expires=2147483647&Signature=ruAaX1VYZJYqtjOUZt9mITX9DZuFsd9gbPiLY0yMCliJOCKfJa0jJ3--jeAoDNC8as1fak1PnbAnnLBqcVClGLoNQs3zXLWnPraWeeQHg32eEi4uKZbpcHvmDg2tx8QKHMp7S5kqqrcMUGM-qsWe1d2kU6ctm8YmKj3nvjQdAHZ6JlYK6zmRiM2rfCFvy8otUKFuZuIdiF7m3bdX6F74-rdynKDQKBVjWquZ5HgmW0T-cFOQBSzAkiCpcJEwQ10vPlYOaoJaMyvLx94p2UXFImCpQjA3Bs~Y0Su9Bx2JaBGteexULd9cBbL4ojPBrEMiXFeFrjLphKoYUuHcROzm9Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5304533&ar=365636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00404.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Case studies on <em>Caenorhabditis</em> spp. (A), mammalian chromosomes (B), and <em>sensu stricto</em> yeasts (C). Most of the chromosomes are correctly resolved by clustering based on palindrome frequency. Perfect tandem repeats were removed prior to analysis to avoid trivial differences from repetitive regions. Note the striking difference between vertebrate and invertebrate DNA.</p></div></div><div content-id="DSR004F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr004/2/m_dsr00404.jpeg?Expires=2147483647&Signature=aDLqdgQFlcMq6cEPwnz6iXoDmEZv4PgDpKhQcetSyog7QFhTOuf94sFkWQlV7wqR49gTHzwkC2167fPinmnpID1DSXs-RxS0RWeY9rVucVEt~2jpSy-MUKE3vv97KfBIZuqtjfA~0XFqv-jvIJNx~m~uBaputqS9610Klx~osA2pDs9ZOalyPmFICGD3XC-CSznm2kS-erarmZLpf0MFUlOrnGd~LHdZ8e0updVqVF~~QUVUCVidleSTWnAfERKnDoNKqQDd8iCzrfk-upobl--FPDr5eIc3CyU8VO03tiUj9uA91G6pxq-di85ER7EAoyOrUPPF6aAaQkaNZCi8Iw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Case studies on Caenorhabditis spp. (A), mammalian chromosomes (B), and sensu stricto yeasts (C). Most of the chromosomes are correctly resolved by clustering based on palindrome frequency. Perfect tandem repeats were removed prior to analysis to avoid trivial differences from repetitive regions. Note the striking difference between vertebrate and invertebrate DNA." path-from-xml="dsr00404.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5304533/dsr00404.jpeg" path-from-xml="dsr00404.jpeg" target="_blank">View large</a><a section="5304533" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr004/2/dsr00404.jpeg?Expires=2147483647&Signature=ruAaX1VYZJYqtjOUZt9mITX9DZuFsd9gbPiLY0yMCliJOCKfJa0jJ3--jeAoDNC8as1fak1PnbAnnLBqcVClGLoNQs3zXLWnPraWeeQHg32eEi4uKZbpcHvmDg2tx8QKHMp7S5kqqrcMUGM-qsWe1d2kU6ctm8YmKj3nvjQdAHZ6JlYK6zmRiM2rfCFvy8otUKFuZuIdiF7m3bdX6F74-rdynKDQKBVjWquZ5HgmW0T-cFOQBSzAkiCpcJEwQ10vPlYOaoJaMyvLx94p2UXFImCpQjA3Bs~Y0Su9Bx2JaBGteexULd9cBbL4ojPBrEMiXFeFrjLphKoYUuHcROzm9Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5304533&ar=365636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00404.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Case studies on <em>Caenorhabditis</em> spp. (A), mammalian chromosomes (B), and <em>sensu stricto</em> yeasts (C). Most of the chromosomes are correctly resolved by clustering based on palindrome frequency. Perfect tandem repeats were removed prior to analysis to avoid trivial differences from repetitive regions. Note the striking difference between vertebrate and invertebrate DNA.</p></div></div>                            <h3 scrollto-destination=5304534 id="5304534" class="section-title" data-legacy-id=s3b>3.2. Clustering of DNA based on palindrome frequency</h3>
<p>Here, we represent a given DNA sequence by a vector of 16 numbers: for each of the 16 palindromes of length 4, the log<sub>2</sub> of the ratio <em>R</em> of actual to expected frequency (given the GC content of the analysed DNA and that of the palindrome). When such vectors, generated from a diverse selection of DNA sequences, were aligned and hierarchically clustered based on the city-block distance, different DNA sequences of the same species readily grouped together (see Fig. <a reveal-id="DSR004F2" data-open="DSR004F2" class="link link-reveal link-table xref-fig">2</a> for a representative set of diverse genomes). The clustering worked for all kinds of genome sequences tested—eukaryote, prokaryote, plastid, or virus—but the topology of the resulting tree was not phylogenetically meaningful (Fig. <a reveal-id="DSR004F2" data-open="DSR004F2" class="link link-reveal link-table xref-fig">2</a>). The lack of a large-scale phylogenetic signal was equally apparent from the analysis of the complete set of 200 genomes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). The resolution of palindrome frequency clustering would increase further by using the 64 different palindromes of length 6. However, this would also require the input sequences to be longer. On the basis of the random sequences included in Fig. <a reveal-id="DSR004F2" data-open="DSR004F2" class="link link-reveal link-table xref-fig">2</a>, the present approach appeared to work for sequences longer than about 10 kb. To obtain a better estimate on the minimally required size of input DNA, we analysed randomly generated sequences of increasing length (Fig. <a reveal-id="DSR004F3" data-open="DSR004F3" class="link link-reveal link-table xref-fig">3</a>). Above 9 kb, the average variance of <em>R</em> per palindrome dropped below the value obtained for different vertebrate chromosomes (0.025, dashed grey line in Fig. <a reveal-id="DSR004F3" data-open="DSR004F3" class="link link-reveal link-table xref-fig">3</a>). For comparison, the average variance of <em>R</em> per palindrome across human chromosomes was 0.0008 (dotted grey line in Fig. <a reveal-id="DSR004F3" data-open="DSR004F3" class="link link-reveal link-table xref-fig">3</a>), demonstrating again that the variance of palindrome frequency is much lower intra- than inter-genome.</p><p>Invertebrates exhibiting the smallest inter-genome variance of palindrome frequency (Fig. <a reveal-id="DSR004F1" data-open="DSR004F1" class="link link-reveal link-table xref-fig">1</a>), we chose <em>Caenorhabditis</em> species to challenge its power of discrimination. The complete nuclear genomes of <em>C. briggsae</em> and <em>C. elegans</em> were compared as described above and all the chromosomes were correctly resolved in spite of the weak patterns (Fig. <a reveal-id="DSR004F4" data-open="DSR004F4" class="link link-reveal link-table xref-fig">4</a>A). Clustering based on palindrome frequency also segregated different mammalian chromosomes which, in contrast to invertebrate DNA, showed the characteristic pattern caused by strong underrepresentation of palindromes containing a CG dinucleotide (ACGT, TCGA, CCGG, GCGC, and CGCG; Fig. <a reveal-id="DSR004F4" data-open="DSR004F4" class="link link-reveal link-table xref-fig">4</a>B). This is in agreement with the model that in vertebrates, DNA methylation is restricted to cytosines followed by guanine (CpG), whereas in invertebrates, cytosines are methylated in a wider context.<sup><a reveal-id="DSR004C25" data-open="DSR004C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Spontaneous mutation of the palindromic CG to the non-palindromic TG by deamination of methylated cytosine thus eliminates short palindromes from vertebrate DNA. The limit of resolution of palindrome frequency clustering was reached with a data set of highly similar <em>sensu stricto</em> yeasts.<sup><a reveal-id="DSR004C26" data-open="DSR004C26" class="link link-ref link-reveal xref-bibr">26</a></sup> The different chromosomes of the closely related species <em>Saccharomyces cerevisiae</em>, <em>S. bayanus</em>, <em>S. mikatae</em>, and <em>S. kudriavzevii</em> did not segregate perfectly; those of the more distantly related <em>S. castellii</em> did (Fig. <a reveal-id="DSR004F4" data-open="DSR004F4" class="link link-reveal link-table xref-fig">4</a>C).</p><p>Clustering based on palindrome frequency also worked for prokaryotes, generating species-specific patterns for archaea as well as bacteria. Prokaryote genomes exhibited highly diverse patterns (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). Natural plasmids of <em>Escherichia coli</em> clearly clustered with the host DNA (Fig. <a reveal-id="DSR004F5" data-open="DSR004F5" class="link link-reveal link-table xref-fig">5</a>A). The same applied to certain dsDNA bacteriophages such as Lambda or P2. However, other dsDNA phages such as T3, as well as all analysed ssDNA phages, did not exhibit the same palindrome frequency patterns as <em>E. coli</em> (Fig. <a reveal-id="DSR004F5" data-open="DSR004F5" class="link link-reveal link-table xref-fig">5</a>A). An interesting picture emerged when comparing human viruses: while all ssRNA minus-strand viruses and the retro-transcribing HIV clustered with human DNA, dsDNA viruses and ssRNA plus-strand viruses did not (Fig. <a reveal-id="DSR004F5" data-open="DSR004F5" class="link link-reveal link-table xref-fig">5</a>B). </p>                        <a id="5304538" scrollto-destination="5304538"></a>
<div id="DSR004F5" data-id="DSR004F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr004/2/m_dsr00405.jpeg?Expires=2147483647&Signature=q9onhGm7Mh7wAjNW0T7rBFKTg24cRxfQb2U5dJRwZRwDjk5HIhrzyoj6bdc4J9TzGk1ip-F9GaiX8h0Zkw7cpMou4aNzh1FnCD5T3WMkf0HLESNLSbkIzxu8wrXfi1gd4WTBxrDw1vA37EHEHPZDCXaJkVL5d4sP4CPTU78XQvizJKu-DzPyV1orXQvNDxZxBN6z1ec0L~AHvaaJK87WdMAldH3SwHKzEa-R2VUI7m0luO61z6nP7GpV8kCNfEoRlEJWfRO9vYtMAsvJoz39DikBzpqE9C3JjGfjCRzw7qOfKhMHa~1Cw9lcCHgYTxCp~x6GPZ14ICOAhrGCV1sL7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Palindrome frequency patterns of host genomic DNA (A, E. coli; B, Homo sapiens; labelled in black) and associated viruses (colour-coded according to nucleic acid type of the genome) or plasmids (grey)." path-from-xml="dsr00405.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5304538/dsr00405.jpeg" path-from-xml="dsr00405.jpeg" target="_blank">View large</a><a section="5304538" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr004/2/dsr00405.jpeg?Expires=2147483647&Signature=Sop1KAOJuJWKL9JN1kYeob0ZTFTnYpVkzDMHiresZMrTm0Aj5AbavYNHmzKvMpq0uB6UKzxbTvE3obNO9lTNKPtAtD~exEVSWo66t-0P03-qm0vklsZsOb6HV3l~ct4Jx6jca~u~QWcd~JwfErF0mO-y3O5SXcEuvSUBTtWIcDTOl7pN6kIlkY5nSgtQBeo~XQXKc~hwATiD3ZxXKWi5epTplVhhk5StNxWHmJPk0xwGptFuWuTvZuZX2CcbPLHLFk36oXQLXkiOzWQqqSzwLvY2K7acK7uSQgiqmuwYjD4QMXt3Tof7o1MBUURsWtIATSzGni0kZ4FUvrjgpLxCmQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5304538&ar=365636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00405.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Palindrome frequency patterns of host genomic DNA (A, <em>E. coli</em>; B, <em>Homo sapiens</em>; labelled in black) and associated viruses (colour-coded according to nucleic acid type of the genome) or plasmids (grey).</p></div></div><div content-id="DSR004F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr004/2/m_dsr00405.jpeg?Expires=2147483647&Signature=q9onhGm7Mh7wAjNW0T7rBFKTg24cRxfQb2U5dJRwZRwDjk5HIhrzyoj6bdc4J9TzGk1ip-F9GaiX8h0Zkw7cpMou4aNzh1FnCD5T3WMkf0HLESNLSbkIzxu8wrXfi1gd4WTBxrDw1vA37EHEHPZDCXaJkVL5d4sP4CPTU78XQvizJKu-DzPyV1orXQvNDxZxBN6z1ec0L~AHvaaJK87WdMAldH3SwHKzEa-R2VUI7m0luO61z6nP7GpV8kCNfEoRlEJWfRO9vYtMAsvJoz39DikBzpqE9C3JjGfjCRzw7qOfKhMHa~1Cw9lcCHgYTxCp~x6GPZ14ICOAhrGCV1sL7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Palindrome frequency patterns of host genomic DNA (A, E. coli; B, Homo sapiens; labelled in black) and associated viruses (colour-coded according to nucleic acid type of the genome) or plasmids (grey)." path-from-xml="dsr00405.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5304538/dsr00405.jpeg" path-from-xml="dsr00405.jpeg" target="_blank">View large</a><a section="5304538" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr004/2/dsr00405.jpeg?Expires=2147483647&Signature=Sop1KAOJuJWKL9JN1kYeob0ZTFTnYpVkzDMHiresZMrTm0Aj5AbavYNHmzKvMpq0uB6UKzxbTvE3obNO9lTNKPtAtD~exEVSWo66t-0P03-qm0vklsZsOb6HV3l~ct4Jx6jca~u~QWcd~JwfErF0mO-y3O5SXcEuvSUBTtWIcDTOl7pN6kIlkY5nSgtQBeo~XQXKc~hwATiD3ZxXKWi5epTplVhhk5StNxWHmJPk0xwGptFuWuTvZuZX2CcbPLHLFk36oXQLXkiOzWQqqSzwLvY2K7acK7uSQgiqmuwYjD4QMXt3Tof7o1MBUURsWtIATSzGni0kZ4FUvrjgpLxCmQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5304538&ar=365636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr00405.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Palindrome frequency patterns of host genomic DNA (A, <em>E. coli</em>; B, <em>Homo sapiens</em>; labelled in black) and associated viruses (colour-coded according to nucleic acid type of the genome) or plasmids (grey).</p></div></div>                            <h3 scrollto-destination=5304539 id="5304539" class="section-title" data-legacy-id=s3c>3.3. Potential application to metagenomics</h3>
<p>The quickly developing field of environmental shotgun sequencing allows metagenomic analyses of communities of microorganisms, the majority of which cannot be cultured in the lab and have therefore remained undetected until recently.<sup><a reveal-id="DSR004C27" data-open="DSR004C27" class="link link-ref link-reveal xref-bibr">27</a></sup> A key challenge in interpreting environmental shotgun sequencing data is the binning of non-overlapping DNA scaffolds into groups which, ideally, correspond to the different species of microorganisms present.<sup><a reveal-id="DSR004C28" data-open="DSR004C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Standard methods such as similarity searches to known genomes or phylogenetic analysis of marker genes are of limited use when dealing with DNA fragments sampled from previously undescribed species.<sup><a reveal-id="DSR004C28" data-open="DSR004C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Di-, tri- and tetra-nucleotide frequencies have been proposed to provide DNA signatures.<sup><a reveal-id="DSR004C29" data-open="DSR004C29" class="link link-ref link-reveal xref-bibr">29</a>–<a reveal-id="DSR004C31" data-open="DSR004C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Palindrome frequencies carrying a species-specific signal (Figs <a reveal-id="DSR004F2" data-open="DSR004F2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSR004F4" data-open="DSR004F4" class="link link-reveal link-table xref-fig">4</a>), the ratios of occurrence to expectancy as applied here may also be useful to bin environmental shotgun sequencing data, provided that the contigs to be analysed are longer than 9 kb (Fig. <a reveal-id="DSR004F3" data-open="DSR004F3" class="link link-reveal link-table xref-fig">3</a>). From the 2007 <em>Sorcerer II</em> Global Ocean Sampling Expedition, which at that time predominantly produced novel sequences,<sup><a reveal-id="DSR004C32" data-open="DSR004C32" class="link link-ref link-reveal xref-bibr">32</a></sup> the hundred largest contigs, sized between 11 and 59 kb, were analysed as described above. This revealed a diverse picture of palindrome frequency patterns with several major clusters (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). However, the analysed sequences still did not return high-quality hits when searched with <em>blastn</em><sup><a reveal-id="DSR004C33" data-open="DSR004C33" class="link link-ref link-reveal xref-bibr">33</a></sup> against the NCBI non-redundant nucleotide collection, with only one exception of 99% identity to <em>Prochlorococcus</em> phage P-SSM4 (GenBank accession no. AY940168). Thus, it was not possible to assess the benefit of palindrome frequency clustering with this data set. To nevertheless test the potential of the method, we randomly selected 10 non-overlapping fragments of length 10 kb from each of the 20 different bacterial genomes analysed in Fig. <a reveal-id="DSR004F1" data-open="DSR004F1" class="link link-reveal link-table xref-fig">1</a> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). When these 200 sequences were clustered according to palindrome frequency patterns, over 90% of them correctly assembled according to species of origin.</p>                            <h2 scrollto-destination=5304541 id="5304541" class="section-title" data-legacy-id=s4>4. Conclusion</h2>
<p>Accustomed to reading DNA as linear sequences, we tend to forget that it is double-stranded in nature. In double-stranded DNA, the only sequences which are unique are palindromes. Here, we confirm the notion that short palindromes are underrepresented across all different kinds of genomes. The frequency distribution of short palindromes exhibits highest inter-species but low intra-species variance. We take advantage of this to type DNA based on palindrome frequency, generating highly specific patterns which discriminate the DNA from different species, clustering together sequences from the same species. The method allows for the assignment of plasmids and certain viruses to their respective host genomes. Although the underlying selective forces are not fully understood, these patterns are highly useful for analysis of DNA sequences of unknown origin, such as those generated by the gigabase in metagenomic high-throughput sequencing surveys. Palindrome frequency ratios as presented here could be incorporated into more sophisticated classifiers such as self-organizing maps,<sup><a reveal-id="DSR004C34" data-open="DSR004C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Bayesian classifiers,<sup><a reveal-id="DSR004C35" data-open="DSR004C35" class="link link-ref link-reveal xref-bibr">35</a></sup> or support vector machines.<sup><a reveal-id="DSR004C36" data-open="DSR004C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Concentrating on palindromes may help to estimate the diversity of microbial communities and to bin different, non-overlapping sequences originating from the same genome, to classify sequences by comparison to reference patterns, and to assign plasmids and bacteriophages to their respective host genomes.</p>                            <h2 scrollto-destination=5304543 id="5304543" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5304545 id="5304545" class="section-title" data-legacy-id=s6>Funding</h2>
<p>We wish to thank the Swiss National Science Foundation for financial support.</p>                            <h2 scrollto-destination=5304547 id="5304547" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSR004C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Watson</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Crick</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Genetical implications of the structure of deoxyribonucleic acid</div>, <div class="source">Nature</div> , <div class="year">1953</div>, vol. <div class="volume">171</div> (pg. <div class="fpage">964</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/171964b0" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/171964b0</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetical%20implications%20of%20the%20structure%20of%20deoxyribonucleic%20acid&amp;author=J.%20Watson&amp;author=F.%20Crick&amp;publication_year=1953&amp;journal=Nature&amp;volume=171&amp;pages=964-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/171964b0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F171964b0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F171964b0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/13063483" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chagin</div> <div class="given-names">V.O.</div></div>,  <div class="name"><div class="surname">Stear</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">Cardoso</div> <div class="given-names">M.C.</div></div>. </div><div class="article-title">Organization of DNA replication</div>, <div class="source">Cold Spring Harb. Perspect. Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">2</div> pg. <div class="fpage">a000737</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Organization%20of%20DNA%20replication&amp;author=V.O.%20Chagin&amp;author=J.H.%20Stear&amp;author=M.C.%20Cardoso&amp;publication_year=2010&amp;journal=Cold%20Spring%20Harb.%20Perspect.%20Biol.&amp;volume=2&amp;pages=a000737" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/cshperspect.a000737" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fcshperspect.a000737" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fcshperspect.a000737"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20452942" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pingoud</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Jeltsch</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Structure and function of type II restriction endonucleases</div>, <div class="source">Nucleic Acids Res</div> , <div class="year">2001</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">3705</div>-<div class="lpage">27</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/29.18.3705" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/29.18.3705</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20and%20function%20of%20type%20II%20restriction%20endonucleases&amp;author=A.%20Pingoud&amp;author=A.%20Jeltsch&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res&amp;volume=29&amp;pages=3705-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/29.18.3705" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.18.3705" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F29.18.3705"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11557805" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zinoviev</div> <div class="given-names">V.V.</div></div>,  <div class="name"><div class="surname">Yakishchik</div> <div class="given-names">S.I.</div></div>,  <div class="name"><div class="surname">Evdokimov</div> <div class="given-names">A.A.</div></div>,  <div class="name"><div class="surname">Malygin</div> <div class="given-names">E.G.</div></div>,  <div class="name"><div class="surname">Hattman</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Symmetry elements in DNA structure important for recognition/methylation by DNA [amino]-methyltransferases</div>, <div class="source">Nucleic Acids Res</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">3930</div>-<div class="lpage">4</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkh712" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkh712</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Symmetry%20elements%20in%20DNA%20structure%20important%20for%20recognition%2Fmethylation%20by%20DNA%20%5Bamino%5D-methyltransferases&amp;author=V.V.%20Zinoviev&amp;author=S.I.%20Yakishchik&amp;author=A.A.%20Evdokimov&amp;author=E.G.%20Malygin&amp;author=S.%20Hattman&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;pages=3930-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh712" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh712" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh712"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15280508" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Czernik</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Shin</div> <div class="given-names">D.S.</div></div>,  <div class="name"><div class="surname">Hurlburt</div> <div class="given-names">B.K.</div></div>. </div><div class="article-title">Functional selection and characterization of DNA binding sites for trp repressor of <em>Escherichia coli</em></div>, <div class="source">J. Biol. Chem</div> , <div class="year">1994</div>, vol. <div class="volume">269</div> (pg. <div class="fpage">27869</div>-<div class="lpage">75</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20selection%20and%20characterization%20of%20DNA%20binding%20sites%20for%20trp%20repressor%20of%20Escherichia%20coli&amp;author=P.J.%20Czernik&amp;author=D.S.%20Shin&amp;author=B.K.%20Hurlburt&amp;publication_year=1994&amp;journal=J.%20Biol.%20Chem&amp;volume=269&amp;pages=27869-75" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7961717" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Welboren</div> <div class="given-names">W.J.</div></div>,  <div class="name"><div class="surname">Stunnenberg</div> <div class="given-names">H.G.</div></div>,  <div class="name"><div class="surname">Sweep</div> <div class="given-names">F.C.</div></div>,  <div class="name"><div class="surname">Span</div> <div class="given-names">P.N.</div></div>. </div><div class="article-title">Identifying estrogen receptor target genes</div>, <div class="source">Mol. Oncol</div> , <div class="year">2007</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">138</div>-<div class="lpage">43</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.molonc.2007.04.001" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.molonc.2007.04.001</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identifying%20estrogen%20receptor%20target%20genes&amp;author=W.J.%20Welboren&amp;author=H.G.%20Stunnenberg&amp;author=F.C.%20Sweep&amp;author=P.N.%20Span&amp;publication_year=2007&amp;journal=Mol.%20Oncol&amp;volume=1&amp;pages=138-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molonc.2007.04.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molonc.2007.04.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molonc.2007.04.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19383291" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Horvath</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Barrangou</div> <div class="given-names">R.</div></div>. </div><div class="article-title">CRISPR/Cas, the immune system of bacteria and archaea</div>, <div class="source">Science</div> , <div class="year">2010</div>, vol. <div class="volume">327</div> (pg. <div class="fpage">167</div>-<div class="lpage">70</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1179555" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1179555</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CRISPR%2FCas%2C%20the%20immune%20system%20of%20bacteria%20and%20archaea&amp;author=P.%20Horvath&amp;author=R.%20Barrangou&amp;publication_year=2010&amp;journal=Science&amp;volume=327&amp;pages=167-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1179555" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1179555" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1179555"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20056882" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Leung</div> <div class="given-names">M.Y.</div></div>,  <div class="name"><div class="surname">Choi</div> <div class="given-names">K.P.</div></div>,  <div class="name"><div class="surname">Xia</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">L.H.</div></div>. </div><div class="article-title">Nonrandom clusters of palindromes in herpesvirus genomes</div>, <div class="source">J. Comput. Biol</div> , <div class="year">2005</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">331</div>-<div class="lpage">54</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1089/cmb.2005.12.331" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1089/cmb.2005.12.331</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nonrandom%20clusters%20of%20palindromes%20in%20herpesvirus%20genomes&amp;author=M.Y.%20Leung&amp;author=K.P.%20Choi&amp;author=A.%20Xia&amp;author=L.H.%20Chen&amp;publication_year=2005&amp;journal=J.%20Comput.%20Biol&amp;volume=12&amp;pages=331-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/cmb.2005.12.331" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fcmb.2005.12.331" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2Fcmb.2005.12.331"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15857246" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zakrzewska-Czerwinska</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Jakimowicz</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Zawilak-Pawlik</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Messer</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Regulation of the initiation of chromosomal replication in bacteria</div>, <div class="source">FEMS Microbiol. Rev</div> , <div class="year">2007</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">378</div>-<div class="lpage">87</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1574-6976.2007.00070.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1574-6976.2007.00070.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20the%20initiation%20of%20chromosomal%20replication%20in%20bacteria&amp;author=J.%20Zakrzewska-Czerwinska&amp;author=D.%20Jakimowicz&amp;author=A.%20Zawilak-Pawlik&amp;author=W.%20Messer&amp;publication_year=2007&amp;journal=FEMS%20Microbiol.%20Rev&amp;volume=31&amp;pages=378-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1574-6976.2007.00070.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1574-6976.2007.00070.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1574-6976.2007.00070.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17459114" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schoner</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Kahn</div> <div class="given-names">M.</div></div>. </div><div class="article-title">The nucleotide sequence of IS5 from <em>Escherichia coli</em></div>, <div class="source">Gene</div> , <div class="year">1981</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">165</div>-<div class="lpage">74</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/0378-1119(81)90112-8" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/0378-1119(81)90112-8</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20nucleotide%20sequence%20of%20IS5%20from%20Escherichia%20coli&amp;author=B.%20Schoner&amp;author=M.%20Kahn&amp;publication_year=1981&amp;journal=Gene&amp;volume=14&amp;pages=165-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(81)90112-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(81)90112-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(81)90112-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6269959" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhou</div> <div class="given-names">Z.H.</div></div>,  <div class="name"><div class="surname">Akgun</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Jasin</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Repeat expansion by homologous recombination in the mouse germ line at palindromic sequences</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2001</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">8326</div>-<div class="lpage">33</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.151008498" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.151008498</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repeat%20expansion%20by%20homologous%20recombination%20in%20the%20mouse%20germ%20line%20at%20palindromic%20sequences&amp;author=Z.H.%20Zhou&amp;author=E.%20Akgun&amp;author=M.%20Jasin&amp;publication_year=2001&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=8326-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.151008498" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.151008498" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.151008498"> </span></div> </div></div></div></div></div><div content-id="DSR004C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nasar</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Jankowski</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Nag</div> <div class="given-names">D.K.</div></div>. </div><div class="article-title">Long palindromic sequences induce double-strand breaks during meiosis in yeast</div>, <div class="source">Mol. Cell. Biol</div> , <div class="year">2000</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">3449</div>-<div class="lpage">58</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1128/MCB.20.10.3449-3458.2000" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1128/MCB.20.10.3449-3458.2000</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Long%20palindromic%20sequences%20induce%20double-strand%20breaks%20during%20meiosis%20in%20yeast&amp;author=F.%20Nasar&amp;author=C.%20Jankowski&amp;author=D.K.%20Nag&amp;publication_year=2000&amp;journal=Mol.%20Cell.%20Biol&amp;volume=20&amp;pages=3449-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.20.10.3449-3458.2000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.20.10.3449-3458.2000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.20.10.3449-3458.2000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10779335" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Duggleby</div> <div class="given-names">R.G.</div></div>. </div><div class="article-title">A paucity of palindromes in phi X174</div>, <div class="source">J. Theor. Biol</div> , <div class="year">1981</div>, vol. <div class="volume">93</div> (pg. <div class="fpage">143</div>-<div class="lpage">55</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/0022-5193(81)90061-8" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/0022-5193(81)90061-8</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20paucity%20of%20palindromes%20in%20phi%20X174&amp;author=R.G.%20Duggleby&amp;publication_year=1981&amp;journal=J.%20Theor.%20Biol&amp;volume=93&amp;pages=143-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-5193(81)90061-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-5193(81)90061-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-5193(81)90061-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6278229" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Elhai</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Determination of bias in the relative abundance of oligonucleotides in DNA sequences</div>, <div class="source">J. Comput. Biol</div> , <div class="year">2001</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">151</div>-<div class="lpage">75</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1089/106652701300312922" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1089/106652701300312922</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Determination%20of%20bias%20in%20the%20relative%20abundance%20of%20oligonucleotides%20in%20DNA%20sequences&amp;author=J.%20Elhai&amp;publication_year=2001&amp;journal=J.%20Comput.%20Biol&amp;volume=8&amp;pages=151-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/106652701300312922" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2F106652701300312922" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2F106652701300312922"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11454303" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Karlin</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Mrazek</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Campbell</div> <div class="given-names">A.M.</div></div>. </div><div class="article-title">Compositional biases of bacterial genomes and evolutionary implications</div>, <div class="source">J. Bacteriol</div> , <div class="year">1997</div>, vol. <div class="volume">179</div> (pg. <div class="fpage">3899</div>-<div class="lpage">913</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Compositional%20biases%20of%20bacterial%20genomes%20and%20evolutionary%20implications&amp;author=S.%20Karlin&amp;author=J.%20Mrazek&amp;author=A.M.%20Campbell&amp;publication_year=1997&amp;journal=J.%20Bacteriol&amp;volume=179&amp;pages=3899-913" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9190805" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fuglsang</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Distribution of potential type II restriction sites (palindromes) in prokaryotes</div>, <div class="source">Biochem. Biophys. Res. Commun</div> , <div class="year">2003</div>, vol. <div class="volume">310</div> (pg. <div class="fpage">280</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.bbrc.2003.09.014" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.bbrc.2003.09.014</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distribution%20of%20potential%20type%20II%20restriction%20sites%20(palindromes)%20in%20prokaryotes&amp;author=A.%20Fuglsang&amp;publication_year=2003&amp;journal=Biochem.%20Biophys.%20Res.%20Commun&amp;volume=310&amp;pages=280-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2003.09.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2003.09.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2003.09.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14521907" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Burge</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Campbell</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Karlin</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Over- and under-representation of short oligonucleotides in DNA sequences</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1992</div>, vol. <div class="volume">89</div> (pg. <div class="fpage">1358</div>-<div class="lpage">62</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.89.4.1358" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.89.4.1358</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Over-%20and%20under-representation%20of%20short%20oligonucleotides%20in%20DNA%20sequences&amp;author=C.%20Burge&amp;author=A.M.%20Campbell&amp;author=S.%20Karlin&amp;publication_year=1992&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=89&amp;pages=1358-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.89.4.1358" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.89.4.1358" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.89.4.1358"> </span></div> </div></div></div></div></div><div content-id="DSR004C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lisnic</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Svetec</div> <div class="given-names">I.K.</div></div>,  <div class="name"><div class="surname">Saric</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Nikolic</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Zgaga</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">Palindrome content of the yeast <em>Saccharomyces cerevisiae</em> genome</div>, <div class="source">Curr. Genet</div> , <div class="year">2005</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">289</div>-<div class="lpage">97</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00294-005-0573-5" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00294-005-0573-5</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Palindrome%20content%20of%20the%20yeast%20Saccharomyces%20cerevisiae%20genome&amp;author=B.%20Lisnic&amp;author=I.K.%20Svetec&amp;author=H.%20Saric&amp;author=I.%20Nikolic&amp;author=Z.%20Zgaga&amp;publication_year=2005&amp;journal=Curr.%20Genet&amp;volume=47&amp;pages=289-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00294-005-0573-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00294-005-0573-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00294-005-0573-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15776233" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lu</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Jia</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Droge</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">J.</div></div>. </div><div class="article-title">The human genome-wide distribution of DNA palindromes</div>, <div class="source">Funct. Integr. Genomics</div> , <div class="year">2007</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">221</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s10142-007-0047-6" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s10142-007-0047-6</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20human%20genome-wide%20distribution%20of%20DNA%20palindromes&amp;author=L.%20Lu&amp;author=H.%20Jia&amp;author=P.%20Droge&amp;author=J.%20Li&amp;publication_year=2007&amp;journal=Funct.%20Integr.%20Genomics&amp;volume=7&amp;pages=221-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10142-007-0047-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10142-007-0047-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10142-007-0047-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17340149" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gelfand</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Koonin</div> <div class="given-names">E.V.</div></div>. </div><div class="article-title">Avoidance of palindromic words in bacterial and archaeal genomes: a close connection with restriction enzymes</div>, <div class="source">Nucleic Acids Res</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">2430</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/25.12.2430" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/25.12.2430</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Avoidance%20of%20palindromic%20words%20in%20bacterial%20and%20archaeal%20genomes%3A%20a%20close%20connection%20with%20restriction%20enzymes&amp;author=M.%20Gelfand&amp;author=E.V.%20Koonin&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res&amp;volume=25&amp;pages=2430-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.12.2430" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.12.2430" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.12.2430"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9171096" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bird</div> <div class="given-names">A.</div></div>. </div><div class="article-title">CpG-rich islands and the function of DNA methylation</div>, <div class="source">Nature</div> , <div class="year">1986</div>, vol. <div class="volume">321</div> (pg. <div class="fpage">209</div>-<div class="lpage">13</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/321209a0" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/321209a0</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CpG-rich%20islands%20and%20the%20function%20of%20DNA%20methylation&amp;author=A.%20Bird&amp;publication_year=1986&amp;journal=Nature&amp;volume=321&amp;pages=209-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/321209a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F321209a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F321209a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2423876" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bhagwat</div> <div class="given-names">A.S.</div></div>,  <div class="name"><div class="surname">McClelland</div> <div class="given-names">M.</div></div>. </div><div class="article-title">DNA mismatch correction by Very Short Patch repair may have altered the abundance of oligonucleotides in the <em>E. coli</em> genome</div>, <div class="source">Nucleic Acids Res</div> , <div class="year">1992</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1663</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/20.7.1663" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/20.7.1663</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20mismatch%20correction%20by%20Very%20Short%20Patch%20repair%20may%20have%20altered%20the%20abundance%20of%20oligonucleotides%20in%20the%20E.%20coli%20genome&amp;author=A.S.%20Bhagwat&amp;author=M.%20McClelland&amp;publication_year=1992&amp;journal=Nucleic%20Acids%20Res&amp;volume=20&amp;pages=1663-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/20.7.1663" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F20.7.1663" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F20.7.1663"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1579457" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eisen</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Spellman</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Botstein</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Cluster analysis and display of genome-wide expression patterns</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1998</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">14863</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cluster%20analysis%20and%20display%20of%20genome-wide%20expression%20patterns&amp;author=M.%20Eisen&amp;author=P.%20Spellman&amp;author=P.%20Brown&amp;author=D.%20Botstein&amp;publication_year=1998&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=14863-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.25.14863" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.25.14863" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.25.14863"> </span></div> </div></div></div></div></div><div content-id="DSR004C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rice</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Longden</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Bleasby</div> <div class="given-names">A.</div></div>. </div><div class="article-title">EMBOSS: the European Molecular Biology Open Software Suite</div>, <div class="source">Trends Genet</div> , <div class="year">2000</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">276</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S0168-9525(00)02024-2" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S0168-9525(00)02024-2</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EMBOSS%3A%20the%20European%20Molecular%20Biology%20Open%20Software%20Suite&amp;author=P.%20Rice&amp;author=I.%20Longden&amp;author=A.%20Bleasby&amp;publication_year=2000&amp;journal=Trends%20Genet&amp;volume=16&amp;pages=276-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(00)02024-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(00)02024-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(00)02024-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10827456" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mandrioli</div> <div class="given-names">M.</div></div>. </div><div class="article-title">A new synthesis in epigenetics: towards a unified function of DNA methylation from invertebrates to vertebrates</div>, <div class="source">Cell. Mol. Life Sci</div> , <div class="year">2007</div>, vol. <div class="volume">64</div> (pg. <div class="fpage">2522</div>-<div class="lpage">4</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00018-007-7231-7" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00018-007-7231-7</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20synthesis%20in%20epigenetics%3A%20towards%20a%20unified%20function%20of%20DNA%20methylation%20from%20invertebrates%20to%20vertebrates&amp;author=M.%20Mandrioli&amp;publication_year=2007&amp;journal=Cell.%20Mol.%20Life%20Sci&amp;volume=64&amp;pages=2522-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00018-007-7231-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00018-007-7231-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00018-007-7231-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17712527" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cliften</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Sudarsanam</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Desikan</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Finding functional features in <em>Saccharomyces</em> genomes by phylogenetic footprinting</div>, <div class="source">Science</div> , <div class="year">2003</div>, vol. <div class="volume">301</div> (pg. <div class="fpage">71</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1084337" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1084337</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Finding%20functional%20features%20in%20Saccharomyces%20genomes%20by%20phylogenetic%20footprinting&amp;author=P.%20Cliften&amp;author=P.%20Sudarsanam&amp;author=A.%20Desikan&amp;publication_year=2003&amp;journal=Science&amp;volume=301&amp;pages=71-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1084337" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1084337" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1084337"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12775844" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tringe</div> <div class="given-names">S.G.</div></div>,  <div class="name"><div class="surname">Rubin</div> <div class="given-names">E.M.</div></div>. </div><div class="article-title">Metagenomics: DNA sequencing of environmental samples</div>, <div class="source">Nat. Rev. Genet</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">805</div>-<div class="lpage">14</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nrg1709" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nrg1709</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metagenomics%3A%20DNA%20sequencing%20of%20environmental%20samples&amp;author=S.G.%20Tringe&amp;author=E.M.%20Rubin&amp;publication_year=2005&amp;journal=Nat.%20Rev.%20Genet&amp;volume=6&amp;pages=805-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1709" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1709" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1709"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16304596" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eisen</div> <div class="given-names">J.A.</div></div>. </div><div class="article-title">Environmental shotgun sequencing: its potential and challenges for studying the hidden world of microbes</div>, <div class="source">PLoS Biol</div> , <div class="year">2007</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e82</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Environmental%20shotgun%20sequencing%3A%20its%20potential%20and%20challenges%20for%20studying%20the%20hidden%20world%20of%20microbes&amp;author=J.A.%20Eisen&amp;publication_year=2007&amp;journal=PLoS%20Biol&amp;volume=5&amp;pages=e82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0050082" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0050082" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0050082"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17355177" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Karlin</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ladunga</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Comparisons of eukaryotic genomic sequences</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1994</div>, vol. <div class="volume">91</div> (pg. <div class="fpage">12832</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.91.26.12832" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.91.26.12832</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparisons%20of%20eukaryotic%20genomic%20sequences&amp;author=S.%20Karlin&amp;author=I.%20Ladunga&amp;publication_year=1994&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=91&amp;pages=12832-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.91.26.12832" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.91.26.12832" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.91.26.12832"> </span></div> </div></div></div></div></div><div content-id="DSR004C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Karlin</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ladunga</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Blaisdell</div> <div class="given-names">B.E.</div></div>. </div><div class="article-title">Heterogeneity of genomes: measures and values</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1994</div>, vol. <div class="volume">91</div> (pg. <div class="fpage">12837</div>-<div class="lpage">41</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.91.26.12837" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.91.26.12837</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Heterogeneity%20of%20genomes%3A%20measures%20and%20values&amp;author=S.%20Karlin&amp;author=I.%20Ladunga&amp;author=B.E.%20Blaisdell&amp;publication_year=1994&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=91&amp;pages=12837-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.91.26.12837" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.91.26.12837" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.91.26.12837"> </span></div> </div></div></div></div></div><div content-id="DSR004C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Teeling</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Meyerdierks</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bauer</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Amann</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Glockner</div> <div class="given-names">F.O.</div></div>. </div><div class="article-title">Application of tetranucleotide frequencies for the assignment of genomic fragments</div>, <div class="source">Environ. Microbiol.</div> , <div class="year">2004</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">938</div>-<div class="lpage">47</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1462-2920.2004.00624.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1462-2920.2004.00624.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Application%20of%20tetranucleotide%20frequencies%20for%20the%20assignment%20of%20genomic%20fragments&amp;author=H.%20Teeling&amp;author=A.%20Meyerdierks&amp;author=M.%20Bauer&amp;author=R.%20Amann&amp;author=F.O.%20Glockner&amp;publication_year=2004&amp;journal=Environ.%20Microbiol.&amp;volume=6&amp;pages=938-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/emi.2004.6.issue-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Femi.2004.6.issue-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Femi.2004.6.issue-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15305919" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rusch</div> <div class="given-names">D.B.</div></div>,  <div class="name"><div class="surname">Halpern</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Sutton</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">The Sorcerer II Global Ocean Sampling expedition: northwest Atlantic through eastern tropical Pacific</div>, <div class="source">PLoS Biol</div> , <div class="year">2007</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e77</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Sorcerer%20II%20Global%20Ocean%20Sampling%20expedition%3A%20northwest%20Atlantic%20through%20eastern%20tropical%20Pacific&amp;author=D.B.%20Rusch&amp;author=A.L.%20Halpern&amp;author=G.%20Sutton&amp;publication_year=2007&amp;journal=PLoS%20Biol&amp;volume=5&amp;pages=e77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0050077" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0050077" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0050077"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17355176" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">E.W.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">Basic local alignment search tool</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1990</div>, vol. <div class="volume">215</div> (pg. <div class="fpage">403</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.W.%20Myers&amp;author=D.J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol.&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dick</div> <div class="given-names">G.J.</div></div>,  <div class="name"><div class="surname">Andersson</div> <div class="given-names">A.F.</div></div>,  <div class="name"><div class="surname">Baker</div> <div class="given-names">B.J.</div></div>, et al. </div><div class="article-title">Community-wide analysis of microbial genome sequence signatures</div>, <div class="source">Genome Biol</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">R85</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Community-wide%20analysis%20of%20microbial%20genome%20sequence%20signatures&amp;author=G.J.%20Dick&amp;author=A.F.%20Andersson&amp;author=B.J.%20Baker&amp;publication_year=2009&amp;journal=Genome%20Biol&amp;volume=10&amp;pages=R85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-8-r85" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-8-r85" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-8-r85"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19698104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sandberg</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Winberg</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Branden</div> <div class="given-names">C.I.</div></div>,  <div class="name"><div class="surname">Kaske</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ernberg</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Coster</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Capturing whole-genome characteristics in short sequences using a naive Bayesian classifier</div>, <div class="source">Genome Res</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1404</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.186401" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.186401</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Capturing%20whole-genome%20characteristics%20in%20short%20sequences%20using%20a%20naive%20Bayesian%20classifier&amp;author=R.%20Sandberg&amp;author=G.%20Winberg&amp;author=C.I.%20Branden&amp;author=A.%20Kaske&amp;author=I.%20Ernberg&amp;author=J.%20Coster&amp;publication_year=2001&amp;journal=Genome%20Res&amp;volume=11&amp;pages=1404-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.186401" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.186401" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.186401"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11483581" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR004C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR004C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McHardy</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">Martin</div> <div class="given-names">H.G.</div></div>,  <div class="name"><div class="surname">Tsirigos</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hugenholtz</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Rigoutsos</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Accurate phylogenetic classification of variable-length DNA fragments</div>, <div class="source">Nat. Methods</div> , <div class="year">2007</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">63</div>-<div class="lpage">72</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nmeth976" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nmeth976</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accurate%20phylogenetic%20classification%20of%20variable-length%20DNA%20fragments&amp;author=A.C.%20McHardy&amp;author=H.G.%20Martin&amp;author=A.%20Tsirigos&amp;author=P.%20Hugenholtz&amp;author=I.%20Rigoutsos&amp;publication_year=2007&amp;journal=Nat.%20Methods&amp;volume=4&amp;pages=63-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth976" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth976" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth976"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17179938" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2011. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=365636&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr004/2/dsr004-dsr004supp_table.xls?Expires=2147483647&amp;Signature=U3rlyyjw5uzy5iI2hi7QVoHt24FBVV~mPEzFJzpqfQQ6Qw9XMZzBUux-sICLv4cvcVjpoZpLRoPYryosygMzdqbzeln-bbRXEfR1AqKS1ju6CgYqd72cGkvPX5~Sd~sTYFKtncZFULIF3Qc9y~E75itkyOLCslyELI0lsrJt62-pw4A1UW1SE~TqCGZrKw97eaGwW4xiK8phe-nEQF~3n-HEPb78PjvK6KqzidS1SrC~TC7ioV6ziduR56m8Bg33ou3ZJNdudgXzkS~MzAtADhPaPji9HClkfGJ4N0Kq-OW9piuXhwBqtNqP~jf7xNoUcH7m2HTRSmjci6f1-QvADQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/2/10.1093/dnares/dsr004/2/dsr004-dsr004supp_figs.pdf?Expires=2147483647&amp;Signature=XY88fpeJvM8ADdK03V-d8o62OA6mbaXDs6ZTWmzCe2iIQrzNXw8q04kwmXIimXZ4M4p1wmVBGP1dZbm44--7WTEsAXburgAdhE2sLsxfD-u1-bXMmU9Y0LuGrS5IgSAKRfPLg10e1qncSD8tCmsSXxClxCgh7-FvJMAUAHWQfJmIcEp0hKA-b4yTT3e4bjMD11oUrkPvGhbWYotDIL-zJ2nJru6vlUS~1EZ0pj~mWzD~fCTKjxsPUqDxLGniSE69LSkncuG511p5g6pexU-U4xjtJzQlSeIOIAMtUlKhXEe5akFUqGXFwh2jbhTgPs3mmItyG-sr3rOnstizgF~RKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">640</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000290061400004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">7</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsr004"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="365636"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000290061400004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f18%2f2%2f117%2f365636">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000290061400004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (7)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F18%2F2%2F117%2F365636" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/365636'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Species-specific Typing of DNA Based on Palindrome Frequency Patterns" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsr004");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f18%2f2%2f117%2f365636">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

