

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/19/6/449/480862","@type":"ScholarlyArticle","name":"Large-Scale Development of Cost-Effective Single-Nucleotide Polymorphism Marker Assays for Genetic Mapping in Pigeonpea and Comparative Mapping in Legumes","datePublished":"2012-10-26","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/19/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2012-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss025","keywords":["pigeonpea","SNP","linkage map","comparative genomics","molecular breeding"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/19/6/449/480862","author":[{"name":"Saxena, Rachit K.","affiliation":"1  Center of Excellence in Genomics (CEG), International Crops Research Institute for the Semi-Arid Tropics (ICRISAT) ,  Patancheru 502324 ,  India","@type":"Person"},{"name":"Varma Penmetsa, R.","affiliation":"2  Department of Plant Pathology, University of California ,  354 Hutchison Hall, One Shields Avenue, Davis, CA 95616-8680 ,  USA","@type":"Person"},{"name":"Upadhyaya, Hari D.","affiliation":"3  Grain Legumes Program, International Crops Research Institute for the Semi-Arid Tropics (ICRISAT) ,  Patancheru 502324 ,  India","@type":"Person"},{"name":"Kumar, Ashish","affiliation":"1  Center of Excellence in Genomics (CEG), International Crops Research Institute for the Semi-Arid Tropics (ICRISAT) ,  Patancheru 502324 ,  India    4  Department of Plant Pathology, College of Agriculture, Rewa, Jawahar lal Nehru Krishi Vishwa Vidyalaya (JNKVV) ,  Jabalpur 482004 ,  India","@type":"Person"},{"name":"Carrasquilla-Garcia,  Noelia","affiliation":"2  Department of Plant Pathology, University of California ,  354 Hutchison Hall, One Shields Avenue, Davis, CA 95616-8680 ,  USA","@type":"Person"},{"name":"Schlueter, Jessica A.","affiliation":"5  Department of Bioinformatics and Genomics, University of North Carolina ,  Charlotte, NC 28223 ,  USA","@type":"Person"},{"name":"Farmer, Andrew","affiliation":"6  National Center for Genome Resources (NCGR) ,  Santa Fe, NM 87505 ,  USA","@type":"Person"},{"name":"Whaley, Adam M.","affiliation":"5  Department of Bioinformatics and Genomics, University of North Carolina ,  Charlotte, NC 28223 ,  USA","@type":"Person"},{"name":"Sarma, Birinchi K.","affiliation":"2  Department of Plant Pathology, University of California ,  354 Hutchison Hall, One Shields Avenue, Davis, CA 95616-8680 ,  USA    7  Department of Mycology and Plant Pathology ,  Banaras Hindu University ,  Varanasi 221005 ,  India","@type":"Person"},{"name":"May, Gregory D.","affiliation":"6  National Center for Genome Resources (NCGR) ,  Santa Fe, NM 87505 ,  USA","@type":"Person"},{"name":"Cook, Douglas R.","affiliation":"2  Department of Plant Pathology, University of California ,  354 Hutchison Hall, One Shields Avenue, Davis, CA 95616-8680 ,  USA","@type":"Person"},{"name":"Varshney, Rajeev K.","affiliation":"1  Center of Excellence in Genomics (CEG), International Crops Research Institute for the Semi-Arid Tropics (ICRISAT) ,  Patancheru 502324 ,  India    8  CGIAR Generation Challenge Programme (GCP) ,  c/o CIMMYT, 06600 Mexico DF ,  Mexico","@type":"Person"}],"description":"Abstract.  Single-nucleotide polymorphisms (SNPs, &gt;2000) were discovered by using RNA-seq and allele-specific sequencing approaches in pigeonpea (Cajanus caj","pageStart":"449","pageEnd":"461","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/6/1/m_cover.gif?Expires=2147483647&Signature=Ymkrtwq9Mdq2ZQmOzrJQ9eKJPKFscP9Dz2fTIBloB-EFD65VEbXzJ4q~-snakwczzkKy9c50sLOIuFB24I0pJdxFnD8zEvpoQeJN~fsnXK1B-LAdZWf~TapU9jar-9Ld8SBPcVyEg-rfuS2fuiV~p46oyz8WKEmF6jt77FBZlqwLfrKM-gNHPhShm2YFMhabay-3DyoXYu6Q7X3TnYZMYg~174agIvVPSgQxLl9pv~mIvWSGZ03BEh643g2rDaJxMKBM3buLpU~BXQmbxGfuwt5uacMTBRmycBDC6hAgFbX~JMal3DalHVMVxXeDECp8wnY2n4dGA5ngut3kqgWBXQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Large-Scale Development of Cost-Effective Single-Nucleotide Polymorphism Marker Assays for Genetic Mapping in Pigeonpea and Comparative Mapping in Legumes","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/6/1/m_cover.gif?Expires=2147483647&Signature=Ymkrtwq9Mdq2ZQmOzrJQ9eKJPKFscP9Dz2fTIBloB-EFD65VEbXzJ4q~-snakwczzkKy9c50sLOIuFB24I0pJdxFnD8zEvpoQeJN~fsnXK1B-LAdZWf~TapU9jar-9Ld8SBPcVyEg-rfuS2fuiV~p46oyz8WKEmF6jt77FBZlqwLfrKM-gNHPhShm2YFMhabay-3DyoXYu6Q7X3TnYZMYg~174agIvVPSgQxLl9pv~mIvWSGZ03BEh643g2rDaJxMKBM3buLpU~BXQmbxGfuwt5uacMTBRmycBDC6hAgFbX~JMal3DalHVMVxXeDECp8wnY2n4dGA5ngut3kqgWBXQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Large-Scale Development of Cost-Effective Single-Nucleotide Polymorphism Marker Assays for Genetic Mapping in Pigeonpea and Comparative Mapping in Legumes" />
    <meta property="og:description" content="Abstract.  Single-nucleotide polymorphisms (SNPs, &amp;gt;2000) were discovered by using RNA-seq and allele-specific sequencing approaches in pigeonpea (Cajanus caj" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/19/6/449/480862" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2012-10-26" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Saxena, Rachit K." /><meta name="citation_author_institution" content="Center of Excellence in Genomics (CEG), International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru 502324, India" /><meta name="citation_author" content="Varma Penmetsa, R." /><meta name="citation_author_institution" content="Department of Plant Pathology, University of California, 354 Hutchison Hall, One Shields Avenue, Davis, CA 95616-8680, USA" /><meta name="citation_author" content="Upadhyaya, Hari D." /><meta name="citation_author_institution" content="Grain Legumes Program, International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru 502324, India" /><meta name="citation_author" content="Kumar, Ashish" /><meta name="citation_author_institution" content="Center of Excellence in Genomics (CEG), International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru 502324, India" /><meta name="citation_author_institution" content="Department of Plant Pathology, College of Agriculture, Rewa, Jawahar lal Nehru Krishi Vishwa Vidyalaya (JNKVV), Jabalpur 482004, India" /><meta name="citation_author" content="Carrasquilla-Garcia,  Noelia" /><meta name="citation_author_institution" content="Department of Plant Pathology, University of California, 354 Hutchison Hall, One Shields Avenue, Davis, CA 95616-8680, USA" /><meta name="citation_author" content="Schlueter, Jessica A." /><meta name="citation_author_institution" content="Department of Bioinformatics and Genomics, University of North Carolina, Charlotte, NC 28223, USA" /><meta name="citation_author" content="Farmer, Andrew" /><meta name="citation_author_institution" content="National Center for Genome Resources (NCGR), Santa Fe, NM 87505, USA" /><meta name="citation_author" content="Whaley, Adam M." /><meta name="citation_author_institution" content="Department of Bioinformatics and Genomics, University of North Carolina, Charlotte, NC 28223, USA" /><meta name="citation_author" content="Sarma, Birinchi K." /><meta name="citation_author_institution" content="Department of Plant Pathology, University of California, 354 Hutchison Hall, One Shields Avenue, Davis, CA 95616-8680, USA" /><meta name="citation_author_institution" content="Department of Mycology and Plant Pathology, Banaras Hindu University, Varanasi 221005, India" /><meta name="citation_author" content="May, Gregory D." /><meta name="citation_author_institution" content="National Center for Genome Resources (NCGR), Santa Fe, NM 87505, USA" /><meta name="citation_author" content="Cook, Douglas R." /><meta name="citation_author_institution" content="Department of Plant Pathology, University of California, 354 Hutchison Hall, One Shields Avenue, Davis, CA 95616-8680, USA" /><meta name="citation_author" content="Varshney, Rajeev K." /><meta name="citation_author_institution" content="Center of Excellence in Genomics (CEG), International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru 502324, India" /><meta name="citation_author_institution" content="CGIAR Generation Challenge Programme (GCP), c/o CIMMYT, 06600 Mexico DF, Mexico" /><meta name="citation_title" content="Large-Scale Development of Cost-Effective Single-Nucleotide Polymorphism Marker Assays for Genetic Mapping in Pigeonpea and Comparative Mapping in Legumes" /><meta name="citation_firstpage" content="449" /><meta name="citation_lastpage" content="461" /><meta name="citation_doi" content="10.1093/dnares/dss025" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="19" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2012/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Genomics-assisted breeding for crop improvement; Trends Plant Sci.;  citation_year=2005;  citation_volume=10;  citation_pages=621-30; " /><meta name="citation_reference" content="citation_title=Advances in cereal genomics and applications in crop breeding; Trends Biotech.;  citation_year=2006;  citation_volume=24;  citation_pages=490-9; " /><meta name="citation_reference" content="citation_title=Legume genomics and breeding; Plant Breed. Rev.;  citation_year=2010;  citation_volume=33;  citation_pages=257-304; " /><meta name="citation_reference" content="citation_title=The first set of EST resource for gene discovery and marker development in pigeonpea (Cajanus cajan L.); BMC Plant Biol.;  citation_year=2010;  citation_volume=10;  citation_pages=45" /><meta name="citation_reference" content="citation_title=Development of genic-SSR markers by deep transcriptome sequencing in pigeonpea [Cajanus cajan (L.) Millspaugh]; BMC Plant Biol.;  citation_year=2011;  citation_volume=11;  citation_pages=17" /><meta name="citation_reference" content="citation_title=Defining the transcriptome assembly and its use for genome dynamics and transcriptome profiling studies in pigeonpea (Cajanus cajan L.); DNA Res.;  citation_year=2011;  citation_volume=18;  citation_pages=153-64; " /><meta name="citation_reference" content="citation_title=A comprehensive transcriptome assembly of pigeonpea (Cajanus cajan L.) using Sanger and second-generation sequencing platforms; Mol. Plant;  citation_year=2012;  citation_volume=5;  citation_pages=1020-1028; " /><meta name="citation_reference" content="citation_title=Novel SSR markers for polymorphism detection in pigeonpea (Cajanus spp.); Plant Breed.;  citation_year=2010;  citation_volume=129;  citation_pages=142-8; " /><meta name="citation_reference" content="citation_title=Analysis of BAC-end sequences (BESs) and development of BES-SSR markers for genetic mapping and hybrid purity assessment in pigeonpea (Cajanus spp.); BMC Plant Biol.;  citation_year=2011;  citation_volume=11;  citation_pages=56" /><meta name="citation_reference" content="citation_title=Single feature polymorphisms (SFPs) for drought tolerance in pigeonpea [Cajanus cajan (L.) Millspaugh]; Funct. Integr. Genomics;  citation_year=2011;  citation_volume=11;  citation_pages=651-7; " /><meta name="citation_reference" content="citation_title=Low level of genetic diversity in cultivated pigeonpea compared to its wild relatives is revealed by diversity arrays technology; Theor. Appl. Genet.;  citation_year=2006;  citation_volume=113;  citation_pages=585-95; " /><meta name="citation_reference" content="citation_title=First genetic map of pigeonpea based on Diversity Array Technology (DArT) markers; J. Genet.;  citation_year=2011;  citation_volume=90;  citation_pages=103-9; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of pigeonpea (Cajanus cajan), an orphan legume crop of resource-poor farmers; Nat. Biotechnol.;  citation_year=2012;  citation_volume=30;  citation_pages=83-9; " /><meta name="citation_reference" content="citation_title=An intra-specific consensus genetic map of pigeonpea [Cajanus cajan (L.) Millspaugh] derived from six mapping populations; Theor. Appl. Genet.;  citation_year=2012;  citation_volume=125;  citation_pages=1325-1338; " /><meta name="citation_reference" content="citation_title=Genetic mapping and quantitative trait locus analysis of resistance to sterility mosaic disease in pigeonpea [Cajanus cajan (L.) Millsp.]; Field Crops Res.;  citation_year=2011;  citation_volume=123;  citation_pages=53-61; " /><meta name="citation_reference" content="citation_title=Applications of single nucleotide polymorphisms in crop genetics; Curr. Opin. Plant Biol.;  citation_year=2002;  citation_volume=5;  citation_pages=94-100; " /><meta name="citation_reference" content="citation_title=Development of versatile gene-based SNP assays in maize (Zea mays L.); Mol. Breed.;  citation_year=2011;  citation_volume=3;  citation_pages=779-790; " /><meta name="citation_reference" content="citation_title=Development and implementation of high-throughput SNP genotyping in barley; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=582" /><meta name="citation_reference" content="citation_title=Single nucleotide polymorphism genotyping in polyploidy wheat with the Illumina GoldenGate assay; Theor. App. Genet.;  citation_year=2009;  citation_volume=119;  citation_pages=507-17; " /><meta name="citation_reference" content="citation_title=High-throughput SNP genotyping with the GoldenGate assay in maize; Mol. Breed.;  citation_year=2009;  citation_volume=25;  citation_pages=441-51; " /><meta name="citation_reference" content="citation_title=SNP discovery by amplicon sequencing and multiplex SNP genotyping in the allopolyploid species Brassica napus; Genome;  citation_year=2010;  citation_volume=53;  citation_pages=948-56; " /><meta name="citation_reference" content="citation_title=High-throughput genotyping with the GoldenGate assay in the complex genome of soybean; Theor. Appl. Genet.;  citation_year=2008;  citation_volume=116;  citation_pages=945-52; " /><meta name="citation_reference" content="citation_title=A consensus genetic map of cowpea [Vigna unguiculata (L) Walp.] and synteny based on EST-derived SNPs; Proc. Natl Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=18159-64; " /><meta name="citation_reference" content="citation_title=Highly-multiplexed SNP genotyping for genetic mapping and germplasm diversity studies in pea; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=468" /><meta name="citation_reference" content="citation_title=A large maize (Zea mays L.) SNP genotyping array: development and germplasm genotyping, and genetic mapping to compare with the B73 reference genome; PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e28334" /><meta name="citation_reference" content="citation_title=Transcript-specific, single-nucleotide polymorphism discovery and linkage analysis in hexaploid bread wheat (Triticum aestivum L.); Plant Biotechnol. J.;  citation_year=2011;  citation_volume=9;  citation_pages=1086-99; " /><meta name="citation_reference" content="citation_title=SNP marker diversity in common bean (Phaseolus vulgaris L.); Theor. Appl. Genet.;  citation_year=2011;  citation_volume=123;  citation_pages=827-45; " /><meta name="citation_reference" content="citation_title=Large-scale development of cost-effective SNP marker assays for diversity assessment and genetic mapping in chickpea and comparative mapping in legumes; Plant Biotechnol. J.;  citation_year=2012;  citation_volume=10;  citation_pages=716-732; " /><meta name="citation_reference" content="citation_title=Isolation and characterization of novel microsatellite markers and their application for diversity assessment in cultivated groundnut (Arachis hypogaea); BMC Plant Biol.;  citation_year=2008;  citation_volume=8;  citation_pages=55" /><meta name="citation_reference" content="citation_title=Management of high-throughput DNA sequencing projects: Alpheus; J. Comput. Sci. Syst. Biol.;  citation_year=2008;  citation_volume=1;  citation_pages=132-48; " /><meta name="citation_reference" content="citation_title=Characterization of a SNPs panel for meat traceability in six cattle breeds; Food Control;  citation_year=2009;  citation_volume=20;  citation_pages=856-60; " /><meta name="citation_reference" content="citation_title=JoinMap4. Software for the calculation of genetic linkage maps in experimental populations; Plant Res. Int., Wageningen: The Netherlands.;  citation_year=2006; " /><meta name="citation_reference" content="citation_title=The estimation of map distances from recombination values; Ann. Eugen.;  citation_year=1943;  citation_volume=12;  citation_pages=172-5; " /><meta name="citation_reference" content="citation_title=MapChart: software for the graphical presentation of linkage maps and QTLs; J. Hered.;  citation_year=2002;  citation_volume=93;  citation_pages=77-8; " /><meta name="citation_reference" content="citation_title=Fast index based algorithms and software for matching position specific scoring matrices; BMC Bioinformatics;  citation_year=2006;  citation_volume=7;  citation_pages=389" /><meta name="citation_reference" content="citation_title=The automatic detection of homologous regions (ADHoRe) and its application to microlinearity between Arabidopsis and rice; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=1792-801; " /><meta name="citation_reference" content="citation_title=Circos: an information aesthetic for comparative genomics; Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=1639-45; " /><meta name="citation_reference" content="citation_title=Genetic patterns of domestication in pigeonpea (Cajanus cajan (L.) Millsp.) and wild Cajanus relatives; PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e39563" /><meta name="citation_reference" content="citation_title=Genome sequence of the palaeopolyploid soybean; Nature;  citation_year=2010;  citation_volume=463;  citation_pages=178-83; " /><meta name="citation_reference" content="citation_title=SSR-based diversity in elite pigeonpea genotypes for developing mapping populations to map resistance to Fusarium wilt and sterility mosaic disease; Plant Breed.;  citation_year=2010;  citation_volume=129;  citation_pages=135-41; " /><meta name="citation_reference" content="citation_title=Three sequenced legume genomes and many crop species: Rich opportunities for translational genomics; Plant Physiol.;  citation_year=2009;  citation_volume=151;  citation_pages=970-7; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/19/6/449/1300280/dss025.pdf" /><meta name="description" content="Abstract.  Single-nucleotide polymorphisms (SNPs, &gt;2000) were discovered by using RNA-seq and allele-specific sequencing approaches in pigeonpea (Cajanus caj" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/19/6/449/480862" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/19/6/449/480862" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/19/6/449/480862");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dss025';
    </script>
<title>
	Large-Scale Development of Cost-Effective Single-Nucleotide Polymorphism Marker Assays for Genetic Mapping in Pigeonpea and Comparative Mapping in Legumes | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./480862" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYhuPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXhlbmEsIFJhY2hpdCBLLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNlbnRlciBvZiBFeGNlbGxlbmNlIGluIEdlbm9taWNzIChDRUcpLCBJbnRlcm5hdGlvbmFsIENyb3BzIFJlc2VhcmNoIEluc3RpdHV0ZSBmb3IgdGhlIFNlbWktQXJpZCBUcm9waWNzIChJQ1JJU0FUKSwgUGF0YW5jaGVydSA1MDIzMjQsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVmFybWEgUGVubWV0c2EsIFIuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBQbGFudCBQYXRob2xvZ3ksIFVuaXZlcnNpdHkgb2YgQ2FsaWZvcm5pYSwgMzU0IEh1dGNoaXNvbiBIYWxsLCBPbmUgU2hpZWxkcyBBdmVudWUsIERhdmlzLCBDQSA5NTYxNi04NjgwLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJVcGFkaHlheWEsIEhhcmkgRC4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFpbiBMZWd1bWVzIFByb2dyYW0sIEludGVybmF0aW9uYWwgQ3JvcHMgUmVzZWFyY2ggSW5zdGl0dXRlIGZvciB0aGUgU2VtaS1BcmlkIFRyb3BpY3MgKElDUklTQVQpLCBQYXRhbmNoZXJ1IDUwMjMyNCwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLdW1hciwgQXNoaXNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudGVyIG9mIEV4Y2VsbGVuY2UgaW4gR2Vub21pY3MgKENFRyksIEludGVybmF0aW9uYWwgQ3JvcHMgUmVzZWFyY2ggSW5zdGl0dXRlIGZvciB0aGUgU2VtaS1BcmlkIFRyb3BpY3MgKElDUklTQVQpLCBQYXRhbmNoZXJ1IDUwMjMyNCwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFBsYW50IFBhdGhvbG9neSwgQ29sbGVnZSBvZiBBZ3JpY3VsdHVyZSwgUmV3YSwgSmF3YWhhciBsYWwgTmVocnUgS3Jpc2hpIFZpc2h3YSBWaWR5YWxheWEgKEpOS1ZWKSwgSmFiYWxwdXIgNDgyMDA0LCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNhcnJhc3F1aWxsYS1HYXJjaWEsICBOb2VsaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFBsYW50IFBhdGhvbG9neSwgVW5pdmVyc2l0eSBvZiBDYWxpZm9ybmlhLCAzNTQgSHV0Y2hpc29uIEhhbGwsIE9uZSBTaGllbGRzIEF2ZW51ZSwgRGF2aXMsIENBIDk1NjE2LTg2ODAsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNjaGx1ZXRlciwgSmVzc2ljYSBBLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvaW5mb3JtYXRpY3MgYW5kIEdlbm9taWNzLCBVbml2ZXJzaXR5IG9mIE5vcnRoIENhcm9saW5hLCBDaGFybG90dGUsIE5DIDI4MjIzLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGYXJtZXIsIEFuZHJldyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIENlbnRlciBmb3IgR2Vub21lIFJlc291cmNlcyAoTkNHUiksIFNhbnRhIEZlLCBOTSA4NzUwNSwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iV2hhbGV5LCBBZGFtIE0uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9pbmZvcm1hdGljcyBhbmQgR2Vub21pY3MsIFVuaXZlcnNpdHkgb2YgTm9ydGggQ2Fyb2xpbmEsIENoYXJsb3R0ZSwgTkMgMjgyMjMsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNhcm1hLCBCaXJpbmNoaSBLLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgUGxhbnQgUGF0aG9sb2d5LCBVbml2ZXJzaXR5IG9mIENhbGlmb3JuaWEsIDM1NCBIdXRjaGlzb24gSGFsbCwgT25lIFNoaWVsZHMgQXZlbnVlLCBEYXZpcywgQ0EgOTU2MTYtODY4MCwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNeWNvbG9neSBhbmQgUGxhbnQgUGF0aG9sb2d5LCBCYW5hcmFzIEhpbmR1IFVuaXZlcnNpdHksIFZhcmFuYXNpIDIyMTAwNSwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNYXksIEdyZWdvcnkgRC4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBDZW50ZXIgZm9yIEdlbm9tZSBSZXNvdXJjZXMgKE5DR1IpLCBTYW50YSBGZSwgTk0gODc1MDUsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNvb2ssIERvdWdsYXMgUi4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFBsYW50IFBhdGhvbG9neSwgVW5pdmVyc2l0eSBvZiBDYWxpZm9ybmlhLCAzNTQgSHV0Y2hpc29uIEhhbGwsIE9uZSBTaGllbGRzIEF2ZW51ZSwgRGF2aXMsIENBIDk1NjE2LTg2ODAsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlZhcnNobmV5LCBSYWplZXYgSy4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZW50ZXIgb2YgRXhjZWxsZW5jZSBpbiBHZW5vbWljcyAoQ0VHKSwgSW50ZXJuYXRpb25hbCBDcm9wcyBSZXNlYXJjaCBJbnN0aXR1dGUgZm9yIHRoZSBTZW1pLUFyaWQgVHJvcGljcyAoSUNSSVNBVCksIFBhdGFuY2hlcnUgNTAyMzI0LCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNHSUFSIEdlbmVyYXRpb24gQ2hhbGxlbmdlIFByb2dyYW1tZSAoR0NQKSwgYy9vIENJTU1ZVCwgMDY2MDAgTWV4aWNvIERGLCBNZXhpY28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkxhcmdlLVNjYWxlIERldmVsb3BtZW50IG9mIENvc3QtRWZmZWN0aXZlIFNpbmdsZS1OdWNsZW90aWRlIFBvbHltb3JwaGlzbSBNYXJrZXIgQXNzYXlzIGZvciBHZW5ldGljIE1hcHBpbmcgaW4gUGlnZW9ucGVhIGFuZCBDb21wYXJhdGl2ZSBNYXBwaW5nIGluIExlZ3VtZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI0NDkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjQ2MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzczAyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEyLzEyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pY3MtYXNzaXN0ZWQgYnJlZWRpbmcgZm9yIGNyb3AgaW1wcm92ZW1lbnQ7IFRyZW5kcyBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NjIxLTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZHZhbmNlcyBpbiBjZXJlYWwgZ2Vub21pY3MgYW5kIGFwcGxpY2F0aW9ucyBpbiBjcm9wIGJyZWVkaW5nOyBUcmVuZHMgQmlvdGVjaC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz00OTAtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGVndW1lIGdlbm9taWNzIGFuZCBicmVlZGluZzsgUGxhbnQgQnJlZWQuIFJldi47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz0yNTctMzA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZmlyc3Qgc2V0IG9mIEVTVCByZXNvdXJjZSBmb3IgZ2VuZSBkaXNjb3ZlcnkgYW5kIG1hcmtlciBkZXZlbG9wbWVudCBpbiBwaWdlb25wZWEgKENhamFudXMgY2FqYW4gTC4pOyBCTUMgUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz00NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIGdlbmljLVNTUiBtYXJrZXJzIGJ5IGRlZXAgdHJhbnNjcmlwdG9tZSBzZXF1ZW5jaW5nIGluIHBpZ2VvbnBlYSBbQ2FqYW51cyBjYWphbiAoTC4pIE1pbGxzcGF1Z2hdOyBCTUMgUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlZmluaW5nIHRoZSB0cmFuc2NyaXB0b21lIGFzc2VtYmx5IGFuZCBpdHMgdXNlIGZvciBnZW5vbWUgZHluYW1pY3MgYW5kIHRyYW5zY3JpcHRvbWUgcHJvZmlsaW5nIHN0dWRpZXMgaW4gcGlnZW9ucGVhIChDYWphbnVzIGNhamFuIEwuKTsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0xNTMtNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29tcHJlaGVuc2l2ZSB0cmFuc2NyaXB0b21lIGFzc2VtYmx5IG9mIHBpZ2VvbnBlYSAoQ2FqYW51cyBjYWphbiBMLikgdXNpbmcgU2FuZ2VyIGFuZCBzZWNvbmQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIHBsYXRmb3JtczsgTW9sLiBQbGFudDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTAyMC0xMDI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob3ZlbCBTU1IgbWFya2VycyBmb3IgcG9seW1vcnBoaXNtIGRldGVjdGlvbiBpbiBwaWdlb25wZWEgKENhamFudXMgc3BwLik7IFBsYW50IEJyZWVkLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjk7ICBjaXRhdGlvbl9wYWdlcz0xNDItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgQkFDLWVuZCBzZXF1ZW5jZXMgKEJFU3MpIGFuZCBkZXZlbG9wbWVudCBvZiBCRVMtU1NSIG1hcmtlcnMgZm9yIGdlbmV0aWMgbWFwcGluZyBhbmQgaHlicmlkIHB1cml0eSBhc3Nlc3NtZW50IGluIHBpZ2VvbnBlYSAoQ2FqYW51cyBzcHAuKTsgQk1DIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW5nbGUgZmVhdHVyZSBwb2x5bW9ycGhpc21zIChTRlBzKSBmb3IgZHJvdWdodCB0b2xlcmFuY2UgaW4gcGlnZW9ucGVhIFtDYWphbnVzIGNhamFuIChMLikgTWlsbHNwYXVnaF07IEZ1bmN0LiBJbnRlZ3IuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NjUxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvdyBsZXZlbCBvZiBnZW5ldGljIGRpdmVyc2l0eSBpbiBjdWx0aXZhdGVkIHBpZ2VvbnBlYSBjb21wYXJlZCB0byBpdHMgd2lsZCByZWxhdGl2ZXMgaXMgcmV2ZWFsZWQgYnkgZGl2ZXJzaXR5IGFycmF5cyB0ZWNobm9sb2d5OyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTExMzsgIGNpdGF0aW9uX3BhZ2VzPTU4NS05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Rmlyc3QgZ2VuZXRpYyBtYXAgb2YgcGlnZW9ucGVhIGJhc2VkIG9uIERpdmVyc2l0eSBBcnJheSBUZWNobm9sb2d5IChEQXJUKSBtYXJrZXJzOyBKLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9OTA7ICBjaXRhdGlvbl9wYWdlcz0xMDMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RHJhZnQgZ2Vub21lIHNlcXVlbmNlIG9mIHBpZ2VvbnBlYSAoQ2FqYW51cyBjYWphbiksIGFuIG9ycGhhbiBsZWd1bWUgY3JvcCBvZiByZXNvdXJjZS1wb29yIGZhcm1lcnM7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz04My05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbnRyYS1zcGVjaWZpYyBjb25zZW5zdXMgZ2VuZXRpYyBtYXAgb2YgcGlnZW9ucGVhIFtDYWphbnVzIGNhamFuIChMLikgTWlsbHNwYXVnaF0gZGVyaXZlZCBmcm9tIHNpeCBtYXBwaW5nIHBvcHVsYXRpb25zOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEyNTsgIGNpdGF0aW9uX3BhZ2VzPTEzMjUtMTMzODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBtYXBwaW5nIGFuZCBxdWFudGl0YXRpdmUgdHJhaXQgbG9jdXMgYW5hbHlzaXMgb2YgcmVzaXN0YW5jZSB0byBzdGVyaWxpdHkgbW9zYWljIGRpc2Vhc2UgaW4gcGlnZW9ucGVhIFtDYWphbnVzIGNhamFuIChMLikgTWlsbHNwLl07IEZpZWxkIENyb3BzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTIzOyAgY2l0YXRpb25fcGFnZXM9NTMtNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFwcGxpY2F0aW9ucyBvZiBzaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc21zIGluIGNyb3AgZ2VuZXRpY3M7IEN1cnIuIE9waW4uIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz05NC0xMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIHZlcnNhdGlsZSBnZW5lLWJhc2VkIFNOUCBhc3NheXMgaW4gbWFpemUgKFplYSBtYXlzIEwuKTsgTW9sLiBCcmVlZC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTc3OS03OTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IGFuZCBpbXBsZW1lbnRhdGlvbiBvZiBoaWdoLXRocm91Z2hwdXQgU05QIGdlbm90eXBpbmcgaW4gYmFybGV5OyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz01ODIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc20gZ2Vub3R5cGluZyBpbiBwb2x5cGxvaWR5IHdoZWF0IHdpdGggdGhlIElsbHVtaW5hIEdvbGRlbkdhdGUgYXNzYXk7IFRoZW9yLiBBcHAuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTk7ICBjaXRhdGlvbl9wYWdlcz01MDctMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2gtdGhyb3VnaHB1dCBTTlAgZ2Vub3R5cGluZyB3aXRoIHRoZSBHb2xkZW5HYXRlIGFzc2F5IGluIG1haXplOyBNb2wuIEJyZWVkLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTQ0MS01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U05QIGRpc2NvdmVyeSBieSBhbXBsaWNvbiBzZXF1ZW5jaW5nIGFuZCBtdWx0aXBsZXggU05QIGdlbm90eXBpbmcgaW4gdGhlIGFsbG9wb2x5cGxvaWQgc3BlY2llcyBCcmFzc2ljYSBuYXB1czsgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTUzOyAgY2l0YXRpb25fcGFnZXM9OTQ4LTU2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLXRocm91Z2hwdXQgZ2Vub3R5cGluZyB3aXRoIHRoZSBHb2xkZW5HYXRlIGFzc2F5IGluIHRoZSBjb21wbGV4IGdlbm9tZSBvZiBzb3liZWFuOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTExNjsgIGNpdGF0aW9uX3BhZ2VzPTk0NS01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb25zZW5zdXMgZ2VuZXRpYyBtYXAgb2YgY293cGVhIFtWaWduYSB1bmd1aWN1bGF0YSAoTCkgV2FscC5dIGFuZCBzeW50ZW55IGJhc2VkIG9uIEVTVC1kZXJpdmVkIFNOUHM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA2OyAgY2l0YXRpb25fcGFnZXM9MTgxNTktNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2hseS1tdWx0aXBsZXhlZCBTTlAgZ2Vub3R5cGluZyBmb3IgZ2VuZXRpYyBtYXBwaW5nIGFuZCBnZXJtcGxhc20gZGl2ZXJzaXR5IHN0dWRpZXMgaW4gcGVhOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz00NjgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGxhcmdlIG1haXplIChaZWEgbWF5cyBMLikgU05QIGdlbm90eXBpbmcgYXJyYXk6IGRldmVsb3BtZW50IGFuZCBnZXJtcGxhc20gZ2Vub3R5cGluZywgYW5kIGdlbmV0aWMgbWFwcGluZyB0byBjb21wYXJlIHdpdGggdGhlIEI3MyByZWZlcmVuY2UgZ2Vub21lOyBQTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTI4MzM0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdC1zcGVjaWZpYywgc2luZ2xlLW51Y2xlb3RpZGUgcG9seW1vcnBoaXNtIGRpc2NvdmVyeSBhbmQgbGlua2FnZSBhbmFseXNpcyBpbiBoZXhhcGxvaWQgYnJlYWQgd2hlYXQgKFRyaXRpY3VtIGFlc3RpdnVtIEwuKTsgUGxhbnQgQmlvdGVjaG5vbC4gSi47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTEwODYtOTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNOUCBtYXJrZXIgZGl2ZXJzaXR5IGluIGNvbW1vbiBiZWFuIChQaGFzZW9sdXMgdnVsZ2FyaXMgTC4pOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyMzsgIGNpdGF0aW9uX3BhZ2VzPTgyNy00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGFyZ2Utc2NhbGUgZGV2ZWxvcG1lbnQgb2YgY29zdC1lZmZlY3RpdmUgU05QIG1hcmtlciBhc3NheXMgZm9yIGRpdmVyc2l0eSBhc3Nlc3NtZW50IGFuZCBnZW5ldGljIG1hcHBpbmcgaW4gY2hpY2twZWEgYW5kIGNvbXBhcmF0aXZlIG1hcHBpbmcgaW4gbGVndW1lczsgUGxhbnQgQmlvdGVjaG5vbC4gSi47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz03MTYtNzMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jc29sYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2Ygbm92ZWwgbWljcm9zYXRlbGxpdGUgbWFya2VycyBhbmQgdGhlaXIgYXBwbGljYXRpb24gZm9yIGRpdmVyc2l0eSBhc3Nlc3NtZW50IGluIGN1bHRpdmF0ZWQgZ3JvdW5kbnV0IChBcmFjaGlzIGh5cG9nYWVhKTsgQk1DIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz01NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hbmFnZW1lbnQgb2YgaGlnaC10aHJvdWdocHV0IEROQSBzZXF1ZW5jaW5nIHByb2plY3RzOiBBbHBoZXVzOyBKLiBDb21wdXQuIFNjaS4gU3lzdC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTEzMi00ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbiBvZiBhIFNOUHMgcGFuZWwgZm9yIG1lYXQgdHJhY2VhYmlsaXR5IGluIHNpeCBjYXR0bGUgYnJlZWRzOyBGb29kIENvbnRyb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz04NTYtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUpvaW5NYXA0LiBTb2Z0d2FyZSBmb3IgdGhlIGNhbGN1bGF0aW9uIG9mIGdlbmV0aWMgbGlua2FnZSBtYXBzIGluIGV4cGVyaW1lbnRhbCBwb3B1bGF0aW9uczsgUGxhbnQgUmVzLiBJbnQuLCBXYWdlbmluZ2VuOiBUaGUgTmV0aGVybGFuZHMuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXN0aW1hdGlvbiBvZiBtYXAgZGlzdGFuY2VzIGZyb20gcmVjb21iaW5hdGlvbiB2YWx1ZXM7IEFubi4gRXVnZW4uOyAgY2l0YXRpb25feWVhcj0xOTQzOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTcyLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcENoYXJ0OiBzb2Z0d2FyZSBmb3IgdGhlIGdyYXBoaWNhbCBwcmVzZW50YXRpb24gb2YgbGlua2FnZSBtYXBzIGFuZCBRVExzOyBKLiBIZXJlZC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9OTM7ICBjaXRhdGlvbl9wYWdlcz03Ny04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GYXN0IGluZGV4IGJhc2VkIGFsZ29yaXRobXMgYW5kIHNvZnR3YXJlIGZvciBtYXRjaGluZyBwb3NpdGlvbiBzcGVjaWZpYyBzY29yaW5nIG1hdHJpY2VzOyBCTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTM4OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBhdXRvbWF0aWMgZGV0ZWN0aW9uIG9mIGhvbW9sb2dvdXMgcmVnaW9ucyAoQURIb1JlKSBhbmQgaXRzIGFwcGxpY2F0aW9uIHRvIG1pY3JvbGluZWFyaXR5IGJldHdlZW4gQXJhYmlkb3BzaXMgYW5kIHJpY2U7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTc5Mi04MDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNpcmNvczogYW4gaW5mb3JtYXRpb24gYWVzdGhldGljIGZvciBjb21wYXJhdGl2ZSBnZW5vbWljczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0xNjM5LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIHBhdHRlcm5zIG9mIGRvbWVzdGljYXRpb24gaW4gcGlnZW9ucGVhIChDYWphbnVzIGNhamFuIChMLikgTWlsbHNwLikgYW5kIHdpbGQgQ2FqYW51cyByZWxhdGl2ZXM7IFBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lMzk1NjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIHBhbGFlb3BvbHlwbG9pZCBzb3liZWFuOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDYzOyAgY2l0YXRpb25fcGFnZXM9MTc4LTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TU1ItYmFzZWQgZGl2ZXJzaXR5IGluIGVsaXRlIHBpZ2VvbnBlYSBnZW5vdHlwZXMgZm9yIGRldmVsb3BpbmcgbWFwcGluZyBwb3B1bGF0aW9ucyB0byBtYXAgcmVzaXN0YW5jZSB0byBGdXNhcml1bSB3aWx0IGFuZCBzdGVyaWxpdHkgbW9zYWljIGRpc2Vhc2U7IFBsYW50IEJyZWVkLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjk7ICBjaXRhdGlvbl9wYWdlcz0xMzUtNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRocmVlIHNlcXVlbmNlZCBsZWd1bWUgZ2Vub21lcyBhbmQgbWFueSBjcm9wIHNwZWNpZXM6IFJpY2ggb3Bwb3J0dW5pdGllcyBmb3IgdHJhbnNsYXRpb25hbCBnZW5vbWljczsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTUxOyAgY2l0YXRpb25fcGFnZXM9OTcwLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xOS82LzQ0OS8xMzAwMjgwL2RzczAyNS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgU2luZ2xlLW51Y2xlb3RpZGUgcG9seW1vcnBoaXNtcyAoU05QcywgJmd0OzIwMDApIHdlcmUgZGlzY292ZXJlZCBieSB1c2luZyBSTkEtc2VxIGFuZCBhbGxlbGUtc3BlY2lmaWMgc2VxdWVuY2luZyBhcHByb2FjaGVzIGluIHBpZ2VvbnBlYSAoQ2FqYW51cyBjYWoiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQ4MDg2MiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGRfZyKrhlszS6B49cd8s0HuGWWfaw==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f6%2f449%2f480862" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f6%2f449%2f480862" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=18878" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/19/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/6/1/m_cover.gif?Expires=2147483647&amp;Signature=Ymkrtwq9Mdq2ZQmOzrJQ9eKJPKFscP9Dz2fTIBloB-EFD65VEbXzJ4q~-snakwczzkKy9c50sLOIuFB24I0pJdxFnD8zEvpoQeJN~fsnXK1B-LAdZWf~TapU9jar-9Ld8SBPcVyEg-rfuS2fuiV~p46oyz8WKEmF6jt77FBZlqwLfrKM-gNHPhShm2YFMhabay-3DyoXYu6Q7X3TnYZMYg~174agIvVPSgQxLl9pv~mIvWSGZ03BEh643g2rDaJxMKBM3buLpU~BXQmbxGfuwt5uacMTBRmycBDC6hAgFbX~JMal3DalHVMVxXeDECp8wnY2n4dGA5ngut3kqgWBXQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/19/6">
                        <div class="volume trailing-comma">Volume 19</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7290810">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7290810">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7290812">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7290812">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7290817">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7290817">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7290829">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7290829">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7290859">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7290859">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7290866">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7290866">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7290868">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7290868">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7290870">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7290870">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7290872">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7290872">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/19/6/435/480512">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/19/6/463/481047">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Large-Scale Development of Cost-Effective Single-Nucleotide Polymorphism Marker Assays for Genetic Mapping in Pigeonpea and Comparative Mapping in Legumes
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rachit K. Saxena</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Center of Excellence in Genomics (CEG), International Crops Research Institute for the Semi-Arid Tropics (ICRISAT)</div>, <div class="addr-line">Patancheru 502324</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rachit+K.+Saxena">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Saxena R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Saxena R&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Rachit K. Saxena</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">R. Varma Penmetsa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Department of Plant Pathology, University of California</div>, <div class="addr-line">354 Hutchison Hall, One Shields Avenue, Davis, CA 95616-8680</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=R.+Varma+Penmetsa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Varma Penmetsa R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Varma Penmetsa R&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">R. Varma Penmetsa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hari D. Upadhyaya</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Grain Legumes Program, International Crops Research Institute for the Semi-Arid Tropics (ICRISAT)</div>, <div class="addr-line">Patancheru 502324</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hari+D.+Upadhyaya">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Upadhyaya H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Upadhyaya H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hari D. Upadhyaya</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ashish Kumar</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Center of Excellence in Genomics (CEG), International Crops Research Institute for the Semi-Arid Tropics (ICRISAT)</div>, <div class="addr-line">Patancheru 502324</div>, <div class="country">India</div></div><div class="aff"><div class="label -label">4</div><div class="insititution">Department of Plant Pathology, College of Agriculture, Rewa, Jawahar lal Nehru Krishi Vishwa Vidyalaya (JNKVV)</div>, <div class="addr-line">Jabalpur 482004</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ashish+Kumar">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kumar A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kumar A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ashish Kumar</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Noelia Carrasquilla-Garcia</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Department of Plant Pathology, University of California</div>, <div class="addr-line">354 Hutchison Hall, One Shields Avenue, Davis, CA 95616-8680</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Noelia+Carrasquilla-Garcia">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Carrasquilla-Garcia  ">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Carrasquilla-Garcia  &quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Noelia Carrasquilla-Garcia</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jessica A. Schlueter</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="insititution">Department of Bioinformatics and Genomics, University of North Carolina</div>, <div class="addr-line">Charlotte, NC 28223</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jessica+A.+Schlueter">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Schlueter J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Schlueter J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jessica A. Schlueter</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Andrew Farmer</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="insititution">National Center for Genome Resources (NCGR)</div>, <div class="addr-line">Santa Fe, NM 87505</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Andrew+Farmer">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Farmer A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Farmer A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Andrew Farmer</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Adam M. Whaley</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="insititution">Department of Bioinformatics and Genomics, University of North Carolina</div>, <div class="addr-line">Charlotte, NC 28223</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Adam+M.+Whaley">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Whaley A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Whaley A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Adam M. Whaley</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Birinchi K. Sarma</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Department of Plant Pathology, University of California</div>, <div class="addr-line">354 Hutchison Hall, One Shields Avenue, Davis, CA 95616-8680</div>, <div class="country">USA</div></div><div class="aff"><div class="label -label">7</div><div class="addr-line">Department of Mycology and Plant Pathology</div>, <div class="insititution">Banaras Hindu University</div>, <div class="addr-line">Varanasi 221005</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Birinchi+K.+Sarma">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sarma B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sarma B&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Birinchi K. Sarma</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Gregory D. May</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="insititution">National Center for Genome Resources (NCGR)</div>, <div class="addr-line">Santa Fe, NM 87505</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Gregory+D.+May">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=May G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;May G&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Gregory D. May</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Douglas R. Cook</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Department of Plant Pathology, University of California</div>, <div class="addr-line">354 Hutchison Hall, One Shields Avenue, Davis, CA 95616-8680</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Douglas+R.+Cook">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cook D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cook D&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Douglas R. Cook</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rajeev K. Varshney</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Center of Excellence in Genomics (CEG), International Crops Research Institute for the Semi-Arid Tropics (ICRISAT)</div>, <div class="addr-line">Patancheru 502324</div>, <div class="country">India</div></div><div class="aff"><div class="label -label">8</div><div class="insititution">CGIAR Generation Challenge Programme (GCP)</div>, <div class="addr-line">c/o CIMMYT, 06600 Mexico DF</div>, <div class="country">Mexico</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +914030713305. Fax. +914030713071. E-mail: <a href="mailto:r.k.varshney@cgiar.org" target="_blank">r.k.varshney@cgiar.org</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rajeev+K.+Varshney">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Varshney R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Varshney R&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Rajeev K. Varshney</a>
                                        </span>
                                </div>
                            </div>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 19, Issue 6, 1 December 2012, Pages 449–461, <a href='https://doi.org/10.1093/dnares/dss025'>https://doi.org/10.1093/dnares/dss025</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">26 October 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">23 July 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">19 September 2012</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="480862" href="/dnaresearch/article-pdf/19/6/449/1300280/dss025.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Rachit K. Saxena, R. Varma Penmetsa, Hari D. Upadhyaya, Ashish Kumar, Noelia Carrasquilla-Garcia, Jessica A. Schlueter, Andrew Farmer, Adam M. Whaley, Birinchi K. Sarma, Gregory D. May, Douglas R. Cook, Rajeev K. Varshney;  Large-Scale Development of Cost-Effective Single-Nucleotide Polymorphism Marker Assays for Genetic Mapping in Pigeonpea and Comparative Mapping in Legumes, <em>DNA Research</em>, Volume 19, Issue 6, 1 December 2012, Pages 449–461, <a href='https://doi.org/10.1093/dnares/dss025'>https://doi.org/10.1093/dnares/dss025</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/480862?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/480862?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/480862?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/480862?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/480862?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/480862?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/480862?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Large-Scale%20Development%20of%20Cost-Effective%20Single-Nucleotide%20Polymorphism%20Marker%20Assays%20for%20Genetic%20Mapping%20in%20Pigeonpea%20and%20Comparative%20Mapping%20in%20Legumes&amp;publicationDate=2012-10-26&amp;volumeNum=19&amp;issueNum=6&amp;author=Saxena%2C%20Rachit%20K.%3B%20Varma%20Penmetsa%2C%20R.&amp;startPage=449&amp;endPage=461&amp;contentId=10.1093%2Fdnares%2Fdss025&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f6%2f449%2f480862" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7290810 id="7290810" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Single-nucleotide polymorphisms (SNPs, &gt;2000) were discovered by using RNA-seq and allele-specific sequencing approaches in pigeonpea (<em>Cajanus cajan</em>). For making the SNP genotyping cost-effective, successful competitive allele-specific polymerase chain reaction (KASPar) assays were developed for 1616 SNPs and referred to as PKAMs (pigeonpea KASPar assay markers). Screening of PKAMs on 24 genotypes [23 from cultivated species and 1 wild species (<em>Cajanus scarabaeoides</em>)] defined a set of 1154 polymorphic markers (77.4%) with a polymorphism information content (PIC) value from 0.04 to 0.38. One thousand and ninety-four PKAMs showed polymorphisms between parental lines of the reference mapping population (<em>C. cajan</em> ICP 28 × <em>C. scarabaeoides</em> ICPW 94). By using high-quality marker genotyping data on 167 F<sub>2</sub> lines from the population, a comprehensive genetic map comprising 875 PKAMs with an average inter-marker distance of 1.11 cM was developed. Previously mapped 35 simple sequence repeat markers were integrated into the PKAM map and an integrated genetic map of 996.21 cM was constructed. Mapped PKAMs showed a higher degree of synteny with the genome of <em>Glycine max</em> followed by <em>Medicago truncatula</em> and <em>Lotus japonicus</em> and least with <em>Vigna unguiculata</em>. These PKAMs will be useful for genetics research and breeding applications in pigeonpea and for utilizing genome information from other legume species.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="pigeonpea">pigeonpea</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SNP">SNP</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;linkage map&quot;">linkage map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;comparative genomics&quot;">comparative genomics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;molecular breeding&quot;">molecular breeding</a></div>                            <h2 scrollto-destination=7290812 id="7290812" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Recent advances in genomics have provided various opportunities to a number of crop species of significant agronomical importance for enhancing crop productivity. One of the foremost applications of genomics in breeding is the prediction of a phenotype from the genotype and the process is called genomics-assisted breeding (GAB).<sup><a reveal-id="DSS025C1" data-open="DSS025C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Several success stories of GAB are available in many temperate cereal crops<sup><a reveal-id="DSS025C2" data-open="DSS025C2" class="link link-ref link-reveal xref-bibr">2</a></sup> and some legume species<sup><a reveal-id="DSS025C3" data-open="DSS025C3" class="link link-ref link-reveal xref-bibr">3</a></sup> also. However, legume crops like pigeonpea (<em>Cajanus cajan</em> L. Millspaugh), which is grown in ∼5 million hectares in the developing countries of Asia and Africa have remained untouched by GAB. This may be attributed mainly to two reasons: (i) paucity of genomic tools and (ii) narrow genetic diversity in the gene pool of the pigeonpea. Following 5 years of intensive research efforts and investment in genomics, however, a significant amount of genomic resources such as a large collection of expressed sequence tags (ESTs) or transcript reads,<sup><a reveal-id="DSS025C4  DSS025C7" data-open="DSS025C4  DSS025C7" class="link link-ref link-reveal xref-bibr">4–7</a></sup> large-scale molecular markers including simple sequence repeats (SSRs),<sup><a reveal-id="DSS025C8" data-open="DSS025C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSS025C9" data-open="DSS025C9" class="link link-ref link-reveal xref-bibr">9</a></sup> single-feature polymorphisms,<sup><a reveal-id="DSS025C10" data-open="DSS025C10" class="link link-ref link-reveal xref-bibr">10</a></sup> single-nucleotide polymorphisms (SNPs)<sup><a reveal-id="DSS025C5  DSS025C7" data-open="DSS025C5  DSS025C7" class="link link-ref link-reveal xref-bibr">5–7</a></sup> and diversity array technology (DArT)<sup><a reveal-id="DSS025C11" data-open="DSS025C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSS025C12" data-open="DSS025C12" class="link link-ref link-reveal xref-bibr">12</a></sup> markers have been developed. Very recently, the draft genome sequence has also become available.<sup><a reveal-id="DSS025C13" data-open="DSS025C13" class="link link-ref link-reveal xref-bibr">13</a></sup></p><p>Identification of molecular markers for applying in a breeding programme requires the development of a genetic map and quantitative trait locus (QTL) analysis. In the case of pigeonpea, although a few genetic maps have been developed,<sup><a reveal-id="DSS025C9" data-open="DSS025C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSS025C12" data-open="DSS025C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSS025C14" data-open="DSS025C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSS025C15" data-open="DSS025C15" class="link link-ref link-reveal xref-bibr">15</a></sup> the marker density in those maps is very low. A major challenge, therefore, before the pigeonpea community is the development of a saturated genetic map. In this context, SNP markers have attracted significant attention as these markers represent the most abundant class of polymorphisms in genomes and are amenable for high-throughput genotyping.<sup><a reveal-id="DSS025C3" data-open="DSS025C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSS025C16" data-open="DSS025C16" class="link link-ref link-reveal xref-bibr">16</a></sup> In general, the implementation of SNPs for genetic studies involves a three-step process: (i) SNP discovery after aligning the ESTs, sequence reads generated by Sanger or next generation sequencing (NGS) technologies for different genotypes in a given species; (ii) validation of SNPs to distinguish DNA polymorphisms of actual allelic variants from those of other biological phenomenon, such as gene duplication events; i.e. paralogous or homeologous genes, as well as those of technical errors, primarily sequencing errors, in case SNPs have been identified using <em>in silico</em> approaches; (iii) SNP genotyping of germplasm collection or genetic/breeding populations. A wide range of molecular techniques suitable for pursuing the mentioned three steps have been available,<sup><a reveal-id="DSS025C17" data-open="DSS025C17" class="link link-ref link-reveal xref-bibr">17</a></sup> each characterized by a distinct cost scale and throughput capacity, and utilizing different technology platforms. NGS technologies are bringing us the capacity to identify, at affordable cost, large numbers of SNPs for even non-model species. Similarly, the availability of a number of SNP genotyping platforms, in a high-throughput manner, is making SNP genotyping cost-effective.</p><p>Depending on the sample size and number of SNPs to be analysed, medium- to high-throughput assay platforms such as BeadXpress and GoldenGate assays from Illumina Inc. with a varying set of multiplexes (96, 384, 768 or 1536 SNPs per assay) are available. Such platforms have been developed and used in several crop species such as barley,<sup><a reveal-id="DSS025C18" data-open="DSS025C18" class="link link-ref link-reveal xref-bibr">18</a></sup> wheat,<sup><a reveal-id="DSS025C19" data-open="DSS025C19" class="link link-ref link-reveal xref-bibr">19</a></sup> maize<sup><a reveal-id="DSS025C20" data-open="DSS025C20" class="link link-ref link-reveal xref-bibr">20</a></sup> and oil seed rape,<sup><a reveal-id="DSS025C21" data-open="DSS025C21" class="link link-ref link-reveal xref-bibr">21</a></sup> and legumes such as soybean,<sup><a reveal-id="DSS025C22" data-open="DSS025C22" class="link link-ref link-reveal xref-bibr">22</a></sup> cowpea<sup><a reveal-id="DSS025C23" data-open="DSS025C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and pea.<sup><a reveal-id="DSS025C24" data-open="DSS025C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Furthermore, in some crops like maize, Infinium assays with the capacity of genotyping ∼50 000 SNPs have become available.<sup><a reveal-id="DSS025C25" data-open="DSS025C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Such platforms, however, are cost-effective only when a minimum of 96, 384, 762, 1536 or thousands of SNPs are used with a large number of samples. In cases like marker-assisted selection where only a few markers are required for genotyping large-scale segregating populations, Illumina-based genotyping assays do not seem to be cost-effective. In such cases, the competitive allele-specific polymerase chain reaction (PCR) (KASPar) assay from KBiosciences (<a class="link link-uri" href="http://www.kbioscience.co.uk">www.kbioscience.co.uk</a>) seems to be an effective marker assay. The KASPar genotyping assay is a competitive allele-specific PCR-based fluorescent SNP genotyping system. It is a user-friendly system that provides flexibility in the numbers of SNPs and genotypes to be used for assays. Details about this technology are available at <a class="link link-uri" href="http://www.kbioscience.co.uk/reagents/KASP.html">http://www.kbioscience.co.uk/reagents/KASP.html</a>. Because of the importance of KASPar assays in genotyping more samples with a few SNPs, they have been developed in wheat,<sup><a reveal-id="DSS025C26" data-open="DSS025C26" class="link link-ref link-reveal xref-bibr">26</a></sup> common bean<sup><a reveal-id="DSS025C27" data-open="DSS025C27" class="link link-ref link-reveal xref-bibr">27</a></sup> and chickpea.<sup><a reveal-id="DSS025C28" data-open="DSS025C28" class="link link-ref link-reveal xref-bibr">28</a></sup></p><p>With an objective to develop a flexible and cost-effective SNP genotyping platform in pigeonpea, this study reports the compilation of informative SNP data sets, development and characterization of KASPar assays, and development of an SNP-based genetic linkage map of pigeonpea and its use for comparative genomics with closely related legume species like soybean (<em>Glycine max</em>), cowpea (<em>Vigna ungiculata</em>), <em>Medicago truncatula</em> and <em>Lotus japonicus</em>.</p>                            <h2 scrollto-destination=7290817 id="7290817" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=7290818 id="7290818" class="section-title" data-legacy-id=s2a>2.1. Mapping population and DNA isolation</h3>
<p>Two <em>Cajanus</em> spp., one from cultivated pigeonpea (<em>C. cajan</em>) ICP 28 and the another accession from the wild relative of pigeonpea (<em>C. scarabaeoides</em>) ICPW 94, were used as crossing parents for the development of an F<sub>2</sub> population of 167 individuals. Accordingly, a single plant of ICP 28 accession was used as a female parent and crossed with the pollen parent ICPW 94 plant and F<sub>1</sub>s were produced. All F<sub>1</sub>s were selfed under nylon bags and grown at Patancheru in southern India (17°N). A single F<sub>1</sub> plant having the highest number of F<sub>2</sub> seeds was selected to develop a mapping population of 167 F<sub>2</sub> individuals. To characterize developed SNP markers, a set of 24 genotypes was utilized for screening the polymorphism (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). These genotypes represent parents of 14 mapping populations which are segregating for various agronomically important traits. Total DNA from the parents of different mapping populations and F<sub>2</sub> lines derived from ICP 28 × ICPW 94 were isolated from two to three young leaves following the standard DNA isolation protocol as mentioned in the study of Cuc <em>et al</em>.<sup><a reveal-id="DSS025C29" data-open="DSS025C29" class="link link-ref link-reveal xref-bibr">29</a></sup> The DNA quantity for each sample was assessed on 0.8% agarose gel.</p>                            <h3 scrollto-destination=7290820 id="7290820" class="section-title" data-legacy-id=s2b>2.2. Identification of SNPs</h3>
<p>The complete Illumina GA IIx data set was comprised 128.9 million, 36 bp short single-end reads from 12 genotypes (ICPL 87119, ICPL 87091, BSMR 736, TAT 10, ICP 7035, TTB 7, ICPL 332, ICPL 20096, ICPB 2049, ICPL 99050, ICP 28 and ICPW 94; Table <a reveal-id="DSS025TB1" data-open="DSS025TB1" class="link link-reveal link-table xref-fig">1</a>). Identification of SNPs from assembled data was carried out using the Alpheus software system.<sup><a reveal-id="DSS025C30" data-open="DSS025C30" class="link link-ref link-reveal xref-bibr">30</a></sup> SNPs were identified on the basis of alignment of sequence reads generated from each of the counter genotypes against the reference assembly, i.e. pigeonpea transcriptome assembly which was developed by assembling four different sequence data sets<sup><a reveal-id="DSS025C7" data-open="DSS025C7" class="link link-ref link-reveal xref-bibr">7</a></sup> and allowing not more than two mismatches. Based on the alignment results, variants at a particular nucleotide position were identified. Significant variants were selected based on two criteria: (i) the allele frequency between two genotypes &gt;0.8 (the number of a specific allele divided by the number of all alleles for the specific SNP between two genotypes should be higher than 80%) and (ii) the number of tags aligned to the reference &gt;5. </p>                        <a id="7290822" scrollto-destination="7290822"></a>
<div content-id="DSS025TB1" class="table-modal"><div class="table-wrap"><div id="DSS025TB1" class="table-wrap-title" data-id="DSS025TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary on the identification of SNPs in <em>Cajanus</em> spp. accessions</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Genotypes </th><th>Illumina GA IIx reads </th><th>Total number of SNPs </th><th>Number of unique genes with identified SNPs </th></tr></thead><tbody><tr><td>ICPL 87119 </td><td>7 182 619 </td><td>7488 </td><td>3116 </td></tr><tr><td>ICPL 87091 </td><td>8 977 567 </td><td> </td><td> </td></tr><tr><td>BSMR 736 </td><td>11 065 219 </td><td>2115 </td><td>1454 </td></tr><tr><td>TAT 10 </td><td>7 932 691 </td><td> </td><td> </td></tr><tr><td>ICP 7035 </td><td>1 3 223 516 </td><td>1256 </td><td>983 </td></tr><tr><td>TTB 7 </td><td>4 122 216 </td><td> </td><td> </td></tr><tr><td>ICPL 332 </td><td>16 361 115 </td><td>2452 </td><td>1819 </td></tr><tr><td>ICPL 20096 </td><td>9 507 797 </td><td> </td><td> </td></tr><tr><td>ICPB 2049 </td><td>11 494 670 </td><td>1892 </td><td>1435 </td></tr><tr><td>ICPL 99050 </td><td>13 498 156 </td><td> </td><td> </td></tr><tr><td>ICP 28 </td><td>9 721 562 </td><td>1910 </td><td>1352 </td></tr><tr><td>ICPW 94 </td><td>15 828 791 </td><td> </td><td> </td></tr><tr><td>Total </td><td>128 915 919 </td><td>17 113 </td><td>10 159 </td></tr><tr><td colspan="4">TOGs </td></tr><tr><td> ICP 28 </td><td> </td><td rowspan="2">752 </td><td rowspan="2">670 </td></tr><tr><td> ICPW 94<sup>a</sup> </td><td> </td></tr></tbody></table></div><div content-id="DSS025TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Genotypes </th><th>Illumina GA IIx reads </th><th>Total number of SNPs </th><th>Number of unique genes with identified SNPs </th></tr></thead><tbody><tr><td>ICPL 87119 </td><td>7 182 619 </td><td>7488 </td><td>3116 </td></tr><tr><td>ICPL 87091 </td><td>8 977 567 </td><td> </td><td> </td></tr><tr><td>BSMR 736 </td><td>11 065 219 </td><td>2115 </td><td>1454 </td></tr><tr><td>TAT 10 </td><td>7 932 691 </td><td> </td><td> </td></tr><tr><td>ICP 7035 </td><td>1 3 223 516 </td><td>1256 </td><td>983 </td></tr><tr><td>TTB 7 </td><td>4 122 216 </td><td> </td><td> </td></tr><tr><td>ICPL 332 </td><td>16 361 115 </td><td>2452 </td><td>1819 </td></tr><tr><td>ICPL 20096 </td><td>9 507 797 </td><td> </td><td> </td></tr><tr><td>ICPB 2049 </td><td>11 494 670 </td><td>1892 </td><td>1435 </td></tr><tr><td>ICPL 99050 </td><td>13 498 156 </td><td> </td><td> </td></tr><tr><td>ICP 28 </td><td>9 721 562 </td><td>1910 </td><td>1352 </td></tr><tr><td>ICPW 94 </td><td>15 828 791 </td><td> </td><td> </td></tr><tr><td>Total </td><td>128 915 919 </td><td>17 113 </td><td>10 159 </td></tr><tr><td colspan="4">TOGs </td></tr><tr><td> ICP 28 </td><td> </td><td rowspan="2">752 </td><td rowspan="2">670 </td></tr><tr><td> ICPW 94<sup>a</sup> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>ICPW 94 is an accession of <em>C. scarabaeoides</em>, a wild relative of pigeonpea (<em>C. cajan</em>).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7290822" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS025TB1" class="table-wrap-title" data-id="DSS025TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary on the identification of SNPs in <em>Cajanus</em> spp. accessions</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Genotypes </th><th>Illumina GA IIx reads </th><th>Total number of SNPs </th><th>Number of unique genes with identified SNPs </th></tr></thead><tbody><tr><td>ICPL 87119 </td><td>7 182 619 </td><td>7488 </td><td>3116 </td></tr><tr><td>ICPL 87091 </td><td>8 977 567 </td><td> </td><td> </td></tr><tr><td>BSMR 736 </td><td>11 065 219 </td><td>2115 </td><td>1454 </td></tr><tr><td>TAT 10 </td><td>7 932 691 </td><td> </td><td> </td></tr><tr><td>ICP 7035 </td><td>1 3 223 516 </td><td>1256 </td><td>983 </td></tr><tr><td>TTB 7 </td><td>4 122 216 </td><td> </td><td> </td></tr><tr><td>ICPL 332 </td><td>16 361 115 </td><td>2452 </td><td>1819 </td></tr><tr><td>ICPL 20096 </td><td>9 507 797 </td><td> </td><td> </td></tr><tr><td>ICPB 2049 </td><td>11 494 670 </td><td>1892 </td><td>1435 </td></tr><tr><td>ICPL 99050 </td><td>13 498 156 </td><td> </td><td> </td></tr><tr><td>ICP 28 </td><td>9 721 562 </td><td>1910 </td><td>1352 </td></tr><tr><td>ICPW 94 </td><td>15 828 791 </td><td> </td><td> </td></tr><tr><td>Total </td><td>128 915 919 </td><td>17 113 </td><td>10 159 </td></tr><tr><td colspan="4">TOGs </td></tr><tr><td> ICP 28 </td><td> </td><td rowspan="2">752 </td><td rowspan="2">670 </td></tr><tr><td> ICPW 94<sup>a</sup> </td><td> </td></tr></tbody></table></div><div content-id="DSS025TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Genotypes </th><th>Illumina GA IIx reads </th><th>Total number of SNPs </th><th>Number of unique genes with identified SNPs </th></tr></thead><tbody><tr><td>ICPL 87119 </td><td>7 182 619 </td><td>7488 </td><td>3116 </td></tr><tr><td>ICPL 87091 </td><td>8 977 567 </td><td> </td><td> </td></tr><tr><td>BSMR 736 </td><td>11 065 219 </td><td>2115 </td><td>1454 </td></tr><tr><td>TAT 10 </td><td>7 932 691 </td><td> </td><td> </td></tr><tr><td>ICP 7035 </td><td>1 3 223 516 </td><td>1256 </td><td>983 </td></tr><tr><td>TTB 7 </td><td>4 122 216 </td><td> </td><td> </td></tr><tr><td>ICPL 332 </td><td>16 361 115 </td><td>2452 </td><td>1819 </td></tr><tr><td>ICPL 20096 </td><td>9 507 797 </td><td> </td><td> </td></tr><tr><td>ICPB 2049 </td><td>11 494 670 </td><td>1892 </td><td>1435 </td></tr><tr><td>ICPL 99050 </td><td>13 498 156 </td><td> </td><td> </td></tr><tr><td>ICP 28 </td><td>9 721 562 </td><td>1910 </td><td>1352 </td></tr><tr><td>ICPW 94 </td><td>15 828 791 </td><td> </td><td> </td></tr><tr><td>Total </td><td>128 915 919 </td><td>17 113 </td><td>10 159 </td></tr><tr><td colspan="4">TOGs </td></tr><tr><td> ICP 28 </td><td> </td><td rowspan="2">752 </td><td rowspan="2">670 </td></tr><tr><td> ICPW 94<sup>a</sup> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>ICPW 94 is an accession of <em>C. scarabaeoides</em>, a wild relative of pigeonpea (<em>C. cajan</em>).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7290822" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=7290823 id="7290823" class="section-title" data-legacy-id=s2c>2.3. KASPar genotyping assay</h3>
<p>For each putative SNP, the criteria used for the selection of high-quality SNPs for KBioscience competitive allele-specific PCR (KASPar) assay<sup><a reveal-id="DSS025C31" data-open="DSS025C31" class="link link-ref link-reveal xref-bibr">31</a></sup> included: (i) an SNP flanked by at least 50 bases on either side; (ii) frequency difference between the two genotypes ≥5; (iii) read depth ≥5. For each SNP, two allele-specific forward primers and one common reverse primer were designed. By using these primers, KASPar assays were performed in a final reaction volume of 5 μL containing 1× KASP reaction mix (KBioscience, Hoddesdon, UK), 0.07 μL of assay mix (12 μM each allele-specific forward primer and 30 μM reverse primer) and 10–20 ng of genomic DNA. The Gene Pro Thermal cycler (Bioer Technology, Hangzhou, China) was used for the following cycling conditions: 15 min at 94°C; 10 touchdown cycles of 20 s at 94°C and 60 s at 65–57°C (the annealing temperature for each cycle being reduced by 0.8°C per cycle); and 26–35 cycles of 20 s at 94°C and 60 s at 57°C. Fluorescence detection of the reactions was performed using an Omega Fluorostar scanner (BMG LABTECH GmbH, Offenburg, Germany) and the data were analysed using the KlusterCaller 1.1 software (KBioscience). Details on the KASPar principle, amplification of targeted region, fluorescence detection and allele calling are available at <a class="link link-uri" href="http://www.kbioscience.co.uk/reagents/KASP_manual.pdf">http://www.kbioscience.co.uk/reagents/KASP_manual.pdf</a>. The polymorphism information content (PIC) values for developed markers across 24 genotypes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>) were calculated by using the PowerMarker software (<a class="link link-uri" href="http://statgen.ncsu.edu/powermarker/">http://statgen.ncsu.edu/powermarker/</a>).</p>                            <h3 scrollto-destination=7290825 id="7290825" class="section-title" data-legacy-id=s2d>2.4. Linkage mapping</h3>
<p>Genotyping data generated using KASPar assays on 167 F<sub>2</sub> individuals of an ICP 28 × ICPW 94 population were analysed for linkage analysis using JoinMap version 4.0 with ‘regression mapping algorithm’.<sup><a reveal-id="DSS025C32" data-open="DSS025C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Prior to linkage analysis, marker segregation data were subjected to the goodness-of-fit test (χ<sup>2</sup>) to assess deviations from the expected Mendelian segregation ratio of 1:2:1 at a 5% level of significance. Map calculations were performed at a logarithm (base 10) of odds (LOD) value of 5.0, recombination frequency ≤0.40 and <em>χ</em><sup>2</sup> jump threshold for removal of loci = 5. A Kosambi map function was used to convert the recombination frequency into cM distances<sup><a reveal-id="DSS025C33" data-open="DSS025C33" class="link link-ref link-reveal xref-bibr">33</a></sup> and the third round was set to allow the mapping of an optimum number of loci in the genetic map. Mean <em>χ</em><sup>2</sup> contributions or average contributions to the goodness of fit of each locus were also checked to determine the best fitting position for markers in genetic maps. The markers showing negative map distances or a large jump in mean <em>χ</em><sup>2</sup> values were subsequently discarded. The graphical maps of the linkage groups (LGs) were constructed by using MapChart version 2.2.<sup><a reveal-id="DSS025C34" data-open="DSS025C34" class="link link-ref link-reveal xref-bibr">34</a></sup></p>                            <h3 scrollto-destination=7290827 id="7290827" class="section-title" data-legacy-id=s2e>2.5. Comparative genome analysis</h3>
<p>DNA sequences for corresponding mapped SNP markers were used for comparative analysis with the genetic map of cowpea<sup><a reveal-id="DSS025C23" data-open="DSS025C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and the genome assemblies of soybean (<a class="link link-uri" href="http://ftp://ftp.jgi-psf.org/pub/JGI_data/phytozome/v7.0/Gmax/assembly">ftp://ftp.jgi-psf.org/pub/JGI_data/phytozome/v7.0/Gmax/assembly</a>), <em>M. truncatula</em> (<a class="link link-uri" href="http://www.medicagohapmap.org/downloads.php">http://www.medicagohapmap.org/downloads.php</a>) and <em>L. japonicus</em> (<a class="link link-uri" href="http://ftp://ftp.kazusa.or.jp/pub/lotus/lotus_r2.5/pseudomolecule">ftp://ftp.kazusa.or.jp/pub/lotus/lotus_r2.5/pseudomolecule</a>). Vmatch<sup><a reveal-id="DSS025C35" data-open="DSS025C35" class="link link-ref link-reveal xref-bibr">35</a></sup> was used to identify reciprocal best matches between the pigeonpea sequences and other legume sequences. Hits matching a minimum of 70% sequence identity were retained for the comparative study. Identification of homologous blocks was performed using i-ADHoRe v2.1.<sup><a reveal-id="DSS025C36" data-open="DSS025C36" class="link link-ref link-reveal xref-bibr">36</a></sup> For the purpose of developing Circos images, cM distances on the pigeonpea LGs were scaled up by a factor of 250 000 to match similar bp lengths of the chromosomes of other legumes' genomes. Synteny blocks were visualized by using Circos26.<sup><a reveal-id="DSS025C37" data-open="DSS025C37" class="link link-ref link-reveal xref-bibr">37</a></sup> Scales along the outer edges of the pigeonpea and cowpea LGs show actual cM distances, whereas those along the outer edges of the soybean, <em>Medicago</em> and <em>Lotus</em> chromosomes are in Mb.</p>                            <h2 scrollto-destination=7290829 id="7290829" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=7290830 id="7290830" class="section-title" data-legacy-id=s3a>3.1. Development of a mapping population</h3>
<p>Although a set of 72 F<sub>2</sub> plants were available from an earlier cross (<em>C. cajan</em> ICP 28 × <em>C. scarabaeoides</em> ICPW 94) that were used to develop an SSR-based genetic map,<sup><a reveal-id="DSS025C9" data-open="DSS025C9" class="link link-ref link-reveal xref-bibr">9</a></sup> a new cross with the same accessions was made to develop a bigger population (167 F<sub>2</sub> lines) for developing a high-resolution genetic map.</p>                            <h3 scrollto-destination=7290833 id="7290833" class="section-title" data-legacy-id=s3b>3.2. Assembly of informative SNPs</h3>
<p>With a goal of increasing the cost-effective and high-throughput genetic marker repertoire in pigeonpea, the following two different sequence resources were surveyed for the presence of SNPs: (i) Illumina GA IIx transcript sequence data and (ii) tentative orthologous genes (TOGs) of closely related legumes.</p>                            <h4 scrollto-destination=7290837 id="7290837" class="section-title" data-legacy-id=s3b1>3.2.1. SNPs from Illumina GA IIx transcript sequence data</h4>
<p>For the identification of SNPs, 128.9 million Illumina reads of 12 different genotypes (ICPL 87119, ICPL 87091, BSMR 736, TAT 10, ICP 7035, TTB 7, ICPL 332, ICPL 20096, ICPB 2049, ICPL 99050, ICP 28 and ICPW 94) were aligned against the transcriptome assembly (CcTA v2).<sup><a reveal-id="DSS025C7" data-open="DSS025C7" class="link link-ref link-reveal xref-bibr">7</a></sup> The CcTA v2 comprised 21 434 transcriptome assembly contigs (TACs) developed from transcriptome data sets from 21 pigeonpea genotypes (128.9 million Illumina GA IIx reads from 12 genotypes, 2.19 million FLX/454 reads from 3 genotypes and 18 353 Sanger ESTs from 6 genotypes).<sup><a reveal-id="DSS025C7" data-open="DSS025C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Variants were identified using the ‘Alpheus’ program<sup><a reveal-id="DSS025C30" data-open="DSS025C30" class="link link-ref link-reveal xref-bibr">30</a></sup> by comparing the sequence tags from two genotypes of a given mapping population combination. In total, a set of 17 113 SNPs in 10 159 unique sequences were identified between six crosses (Table <a reveal-id="DSS025TB1" data-open="DSS025TB1" class="link link-reveal link-table xref-fig">1</a>). The number of SNPs in an individual cross ranged from 1256 (TTB 7 × ICPL 7035) to 7488 (ICPL 87119 × ICPL 87091) (Table <a reveal-id="DSS025TB1" data-open="DSS025TB1" class="link link-reveal link-table xref-fig">1</a>). However, only six SNPs were found common across three populations (ICPL 20096 × ICPL 332, TTB 7 × ICPL 7035 and BSMR 736 × TAT 10). For the ICP 28 and ICPW 94 combination, a total of 1910 SNPs were identified in 1352 TACs. By considering only one SNP per TAC (gene) and primer designing criteria of KASPar assays, 1167 SNPs were further selected.</p>                            <h4 scrollto-destination=7290839 id="7290839" class="section-title" data-legacy-id=s3b2>3.2.2. SNPs from TOGs</h4>
<p>After sequencing ICP 28 and ICPW 94 accessions with 670 TOGs, a set of 752 SNPs were identified and used in designing the GoldenGate assay in a separate study.<sup><a reveal-id="DSS025C38" data-open="DSS025C38" class="link link-ref link-reveal xref-bibr">38</a></sup> From these 752 SNPs, a total of 660 SNPs satisfied the required primer designing criteria for KASPar assays.</p><p>After combining the above-mentioned data sets, a non-redundant set of 1827 SNPs (1167 from Illumina GA IIx transcript data, referred as GAIIx-SNPs and 660 TOG-SNPs) was complied.</p>                            <h3 scrollto-destination=7290842 id="7290842" class="section-title" data-legacy-id=s3c>3.3. Development and validation of KASPar assays</h3>
<p>A total of 1827 non-redundant SNPs were used for the development of KASPar assays (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). However, successful assays could be developed for 1616 SNPs (88.4%) with scorable allele calls. These marker assays have been referred to as PKAMs (pigeonpea KASPar assay markers). All 1616 PKAMs were screened on 24 pigeonpea genotypes, including 23 cultivated and one wild-type ICPW 94, representing parents of 14 mapping populations (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>), further defined a subset of 1154 polymorphic markers (77.4%). Among these polymorphic PKAMs, 1043 were polymorphic exclusively in wild species. Data obtained from 24 genotypes were used to calculate the PIC value of each PKAM marker, and PIC values ranged from 0.04 to 0.38 with an average of 0.09 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). In terms of the parental polymorphism of different mapping populations, polymorphism rates varied considerably, depending on the crossing parental lines under comparison, from a low of 14 polymorphic PKAMs (ICP 8863 × ICPL 20097) to a high of 1094 polymorphic PKAMs (ICP 28 × ICPW 94) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>).</p>                            <h3 scrollto-destination=7290844 id="7290844" class="section-title" data-legacy-id=s3d>3.4. SNP-based genetic map</h3>
<p>With a goal of developing an SNP-based genetic map in pigeonpea, genotyping data were obtained on 167 progenies for all the 1094 polymorphic PKAMs. However, high-quality data obtained for 1008 PKAMs were considered for further analysis. Genotyping data obtained for all 1008 PKAMs were tested for the Mendelian/non-Mendelian segregation pattern. As a result, the 33 PKAMs showing non-Mendelian inheritance were removed from further analysis. Subsequently, genotyping data for 975 PKAMs (470 PKAMs based on GAIIx-SNPs and 505 PKAMs from TOG-SNPs) were used for linkage analysis, with JoinMap4.0 used to construct the genetic linkage map.<sup><a reveal-id="DSS025C32" data-open="DSS025C32" class="link link-ref link-reveal xref-bibr">32</a></sup></p><p>In summary, 11 LGs were generated using an LOD threshold value of 5.0, which is in agreement with the haploid chromosome number (11) in pigeonpea. A total of 55 PKAMs that failed to be assigned to these 11 LGs were not incorporated in further analyses. While assigning the order of PKAMs in different individual LGs, map positions could not be assigned for 45 PKAMs. As a result, the developed genetic map contains 875 (444 GAIIx-SNP and 431 TOG-SNP) loci on 11 LGs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>), ranging from 25.7 cM (CcLG05) to 124.25 cM (CcLG11) in length, and a total map length of 967.03 cM (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). The number of markers per LG ranges from 25 (CcLG05) to 134 (CcLG02), with an average of 79.54 (Table <a reveal-id="DSS025TB2" data-open="DSS025TB2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). The highest marker density with an inter-marker distance of 0.84 cM was observed on CcLG02, while CcLG09 had the lowest marker density with average marker spacing 1.79 cM. Most of the spaces between two markers were smaller than 20 cM on the genetic linkage map. However, there were only two spaces where the distances between markers were large, i.e. 23.56 cM between PKAM0211 and PKAM0417 on CcLG02 and 28.85 cM between PKAM0671 and PKAM0543 on CcLG06. </p>                        <a id="7290847" scrollto-destination="7290847"></a>
<div content-id="DSS025TB2" class="table-modal"><div class="table-wrap"><div id="DSS025TB2" class="table-wrap-title" data-id="DSS025TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Features of SNP-based and integrated pigeonpea genetic maps</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Pigeonpea LG </th><th colspan="3">SNP-based map<hr /> </th><th colspan="3">Integrated map (SNPs + SSRs)<hr /> </th></tr><tr><th>Size (cM) </th><th>Number of loci </th><th>Average inter- loci distance </th><th>Size (cM) </th><th>Number of loci </th><th>Average inter-loci distance </th></tr></thead><tbody><tr><td>CcLG01 </td><td>95.83 </td><td>79 </td><td>1.21 </td><td>107.29 </td><td>82 </td><td>1.31 </td></tr><tr><td>CcLG02 </td><td>112.55 </td><td>134 </td><td>0.84 </td><td>112.55 </td><td>136 </td><td>0.83 </td></tr><tr><td>CcLG03 </td><td>107.85 </td><td>118 </td><td>0.91 </td><td>120.88 </td><td>123 </td><td>0.98 </td></tr><tr><td>CcLG04 </td><td>70.29 </td><td>65 </td><td>1.08 </td><td>70.77 </td><td>70 </td><td>1.01 </td></tr><tr><td>CcLG05 </td><td>25.7 </td><td>25 </td><td>1.03 </td><td>28.41 </td><td>28 </td><td>1.01 </td></tr><tr><td>CcLG06 </td><td>81.28 </td><td>82 </td><td>0.99 </td><td>80.93 </td><td>85 </td><td>0.95 </td></tr><tr><td>CcLG07 </td><td>93.22 </td><td>84 </td><td>1.11 </td><td>94.14 </td><td>88 </td><td>1.07 </td></tr><tr><td>CcLG08 </td><td>96.48 </td><td>71 </td><td>1.36 </td><td>97.43 </td><td>73 </td><td>1.33 </td></tr><tr><td>CcLG09 </td><td>96.75 </td><td>54 </td><td>1.79 </td><td>96.81 </td><td>58 </td><td>1.67 </td></tr><tr><td>CcLG10 </td><td>62.83 </td><td>69 </td><td>0.91 </td><td>62.83 </td><td>71 </td><td>0.88 </td></tr><tr><td>CcLG11 </td><td>124.25 </td><td>94 </td><td>1.32 </td><td>124.18 </td><td>96 </td><td>1.29 </td></tr><tr><td>Total </td><td>967.03 </td><td>875 </td><td>1.11 </td><td>996.21 </td><td>910 </td><td>1.09 </td></tr></tbody></table></div><div content-id="DSS025TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Pigeonpea LG </th><th colspan="3">SNP-based map<hr /> </th><th colspan="3">Integrated map (SNPs + SSRs)<hr /> </th></tr><tr><th>Size (cM) </th><th>Number of loci </th><th>Average inter- loci distance </th><th>Size (cM) </th><th>Number of loci </th><th>Average inter-loci distance </th></tr></thead><tbody><tr><td>CcLG01 </td><td>95.83 </td><td>79 </td><td>1.21 </td><td>107.29 </td><td>82 </td><td>1.31 </td></tr><tr><td>CcLG02 </td><td>112.55 </td><td>134 </td><td>0.84 </td><td>112.55 </td><td>136 </td><td>0.83 </td></tr><tr><td>CcLG03 </td><td>107.85 </td><td>118 </td><td>0.91 </td><td>120.88 </td><td>123 </td><td>0.98 </td></tr><tr><td>CcLG04 </td><td>70.29 </td><td>65 </td><td>1.08 </td><td>70.77 </td><td>70 </td><td>1.01 </td></tr><tr><td>CcLG05 </td><td>25.7 </td><td>25 </td><td>1.03 </td><td>28.41 </td><td>28 </td><td>1.01 </td></tr><tr><td>CcLG06 </td><td>81.28 </td><td>82 </td><td>0.99 </td><td>80.93 </td><td>85 </td><td>0.95 </td></tr><tr><td>CcLG07 </td><td>93.22 </td><td>84 </td><td>1.11 </td><td>94.14 </td><td>88 </td><td>1.07 </td></tr><tr><td>CcLG08 </td><td>96.48 </td><td>71 </td><td>1.36 </td><td>97.43 </td><td>73 </td><td>1.33 </td></tr><tr><td>CcLG09 </td><td>96.75 </td><td>54 </td><td>1.79 </td><td>96.81 </td><td>58 </td><td>1.67 </td></tr><tr><td>CcLG10 </td><td>62.83 </td><td>69 </td><td>0.91 </td><td>62.83 </td><td>71 </td><td>0.88 </td></tr><tr><td>CcLG11 </td><td>124.25 </td><td>94 </td><td>1.32 </td><td>124.18 </td><td>96 </td><td>1.29 </td></tr><tr><td>Total </td><td>967.03 </td><td>875 </td><td>1.11 </td><td>996.21 </td><td>910 </td><td>1.09 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7290847" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS025TB2" class="table-wrap-title" data-id="DSS025TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Features of SNP-based and integrated pigeonpea genetic maps</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Pigeonpea LG </th><th colspan="3">SNP-based map<hr /> </th><th colspan="3">Integrated map (SNPs + SSRs)<hr /> </th></tr><tr><th>Size (cM) </th><th>Number of loci </th><th>Average inter- loci distance </th><th>Size (cM) </th><th>Number of loci </th><th>Average inter-loci distance </th></tr></thead><tbody><tr><td>CcLG01 </td><td>95.83 </td><td>79 </td><td>1.21 </td><td>107.29 </td><td>82 </td><td>1.31 </td></tr><tr><td>CcLG02 </td><td>112.55 </td><td>134 </td><td>0.84 </td><td>112.55 </td><td>136 </td><td>0.83 </td></tr><tr><td>CcLG03 </td><td>107.85 </td><td>118 </td><td>0.91 </td><td>120.88 </td><td>123 </td><td>0.98 </td></tr><tr><td>CcLG04 </td><td>70.29 </td><td>65 </td><td>1.08 </td><td>70.77 </td><td>70 </td><td>1.01 </td></tr><tr><td>CcLG05 </td><td>25.7 </td><td>25 </td><td>1.03 </td><td>28.41 </td><td>28 </td><td>1.01 </td></tr><tr><td>CcLG06 </td><td>81.28 </td><td>82 </td><td>0.99 </td><td>80.93 </td><td>85 </td><td>0.95 </td></tr><tr><td>CcLG07 </td><td>93.22 </td><td>84 </td><td>1.11 </td><td>94.14 </td><td>88 </td><td>1.07 </td></tr><tr><td>CcLG08 </td><td>96.48 </td><td>71 </td><td>1.36 </td><td>97.43 </td><td>73 </td><td>1.33 </td></tr><tr><td>CcLG09 </td><td>96.75 </td><td>54 </td><td>1.79 </td><td>96.81 </td><td>58 </td><td>1.67 </td></tr><tr><td>CcLG10 </td><td>62.83 </td><td>69 </td><td>0.91 </td><td>62.83 </td><td>71 </td><td>0.88 </td></tr><tr><td>CcLG11 </td><td>124.25 </td><td>94 </td><td>1.32 </td><td>124.18 </td><td>96 </td><td>1.29 </td></tr><tr><td>Total </td><td>967.03 </td><td>875 </td><td>1.11 </td><td>996.21 </td><td>910 </td><td>1.09 </td></tr></tbody></table></div><div content-id="DSS025TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Pigeonpea LG </th><th colspan="3">SNP-based map<hr /> </th><th colspan="3">Integrated map (SNPs + SSRs)<hr /> </th></tr><tr><th>Size (cM) </th><th>Number of loci </th><th>Average inter- loci distance </th><th>Size (cM) </th><th>Number of loci </th><th>Average inter-loci distance </th></tr></thead><tbody><tr><td>CcLG01 </td><td>95.83 </td><td>79 </td><td>1.21 </td><td>107.29 </td><td>82 </td><td>1.31 </td></tr><tr><td>CcLG02 </td><td>112.55 </td><td>134 </td><td>0.84 </td><td>112.55 </td><td>136 </td><td>0.83 </td></tr><tr><td>CcLG03 </td><td>107.85 </td><td>118 </td><td>0.91 </td><td>120.88 </td><td>123 </td><td>0.98 </td></tr><tr><td>CcLG04 </td><td>70.29 </td><td>65 </td><td>1.08 </td><td>70.77 </td><td>70 </td><td>1.01 </td></tr><tr><td>CcLG05 </td><td>25.7 </td><td>25 </td><td>1.03 </td><td>28.41 </td><td>28 </td><td>1.01 </td></tr><tr><td>CcLG06 </td><td>81.28 </td><td>82 </td><td>0.99 </td><td>80.93 </td><td>85 </td><td>0.95 </td></tr><tr><td>CcLG07 </td><td>93.22 </td><td>84 </td><td>1.11 </td><td>94.14 </td><td>88 </td><td>1.07 </td></tr><tr><td>CcLG08 </td><td>96.48 </td><td>71 </td><td>1.36 </td><td>97.43 </td><td>73 </td><td>1.33 </td></tr><tr><td>CcLG09 </td><td>96.75 </td><td>54 </td><td>1.79 </td><td>96.81 </td><td>58 </td><td>1.67 </td></tr><tr><td>CcLG10 </td><td>62.83 </td><td>69 </td><td>0.91 </td><td>62.83 </td><td>71 </td><td>0.88 </td></tr><tr><td>CcLG11 </td><td>124.25 </td><td>94 </td><td>1.32 </td><td>124.18 </td><td>96 </td><td>1.29 </td></tr><tr><td>Total </td><td>967.03 </td><td>875 </td><td>1.11 </td><td>996.21 </td><td>910 </td><td>1.09 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7290847" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=7290848 id="7290848" class="section-title" data-legacy-id=s3e>3.5. Integrated genetic map</h3>
<p>With an objective to provide anchor points to integrate an SSR-based genetic linkage map with the SNP-based genetic linkage map, the newly developed mapping population (ICP 28 × ICPW 94) was also genotyped with targeted SSRs (2–5) from each LG of the SSR-based genetic linkage map previously developed by Bohra <em>et al</em>.<sup><a reveal-id="DSS025C9" data-open="DSS025C9" class="link link-ref link-reveal xref-bibr">9</a></sup> In this context, genotyping data generated for a total of 35 SSRs. All the 35 SSRs were mapped onto 11 LGs (CcLG01–CcLG11). The number of SSR markers per LG varied from two (CcLG02, CcLG08, CcLG10 and CcLG11) to five (CcLG03 and CcLG04) (Table <a reveal-id="DSS025TB2" data-open="DSS025TB2" class="link link-reveal link-table xref-fig">2</a>). After integration of the SSR markers, the total distance of the integrated genetic linkage map increased by 29.18 cM (Fig. <a reveal-id="DSS025F1" data-open="DSS025F1" class="link link-reveal link-table xref-fig">1</a>). Subsequently, the integrated genetic map was compared with the SSR-based genetic linkage map and the marker order was consistent between similar LGs. However, in the case of CcLG03, out of five common markers, two markers, namely CcM1593 and CcM2045, had different positions. All pairwise comparisons between the integrated LGs and SSR-based LGs have been shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>. As expected, the marker order and distances were well conserved between the integrated genetic map and the SNP-based genetic map. </p>                        <a id="7290850" scrollto-destination="7290850"></a>
<div id="DSS025F1" data-id="DSS025F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss025/2/m_dss02501.jpeg?Expires=2147483647&Signature=IAVL6r7sobC5uKbODsKOI5G9t5F8DMc~t0ID6sI8HkiIXh1QL7KoK~XQr2otpdjdtz7KAMJ5a5vNEeZ9M-5ZNgp1i5ESwgr-f7WbDcorgFEMmij~xA~zz1LgZazYwXYiiuS7Cbb0tbM18OQjhbKAmvYUGpHO3OMYYkclwgLXZ27LWvVlQCgtqI4v4T~MaajP7GWVbYsqJfeCEiotX4dkxvr43URK~5SaKmlP1rqWiOaenqRCmlhooQRjiyqRvsvwMVpAnJE7VSkx1oMid4bekU3nJhOIxVzQL8mpbbXLhUZOFlrrzmegLW6aztoWiubC~Gm7QGP1C5RgqZGXAung1Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="An integrated genetic map of pigeonpea. This genetic map is developed on the F2 mapping population derived from ICP 28 × ICPW 94. The map comprises 910 loci (875 PKAMs and 35 SSRs) in which 444 PKAMs were developed based on GAIIx-SNPs, shown in red; 431 PKAMs were developed based on TOGs-SNPs, shown in green, and 35 SSRs previously mapped by Bohra et al.9 shown in black." path-from-xml="dss02501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7290850/dss02501.jpeg" path-from-xml="dss02501.jpeg" target="_blank">View large</a><a section="7290850" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss025/2/dss02501.jpeg?Expires=2147483647&Signature=Zfk2rsY3hrRJPBzxO-lrR4L42ElwUOdXNnEld-j98x~x406ur0fUSKUt8B8BczM1Z2MyhYXFw78eTJRoz9NtPVlmwUsnKw91sCmX9k8dIE5aG-mhxmXXU0Qg-OUNCR4lvBy-mPmiSGkIauSOMRpmdcq~SKFXhI3MKWzbrkWXUO6~KfB3WCcK4YnrQdf3uHE80Lp0RgjfjrmB5RtUqzfkUw0jd66ndKLzkAcSGmVt3mqTHrDQQXVbO47WfwB4s2Y8l1afFdAGux2DNWfDto5jp3dcDxqnQCSN7eASafEjhRGDM1WZzQgT4cF5x4rguYlI4-aqwZxUF0myS8oBSZcDZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7290850&ar=480862&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>An integrated genetic map of pigeonpea. This genetic map is developed on the F<sub>2</sub> mapping population derived from ICP 28 × ICPW 94. The map comprises 910 loci (875 PKAMs and 35 SSRs) in which 444 PKAMs were developed based on GAIIx-SNPs, shown in red; 431 PKAMs were developed based on TOGs-SNPs, shown in green, and 35 SSRs previously mapped by Bohra <em>et al</em>.<sup><a reveal-id="DSS025C9" data-open="DSS025C9" class="link link-ref link-reveal xref-bibr">9</a></sup> shown in black.</p></div></div><div content-id="DSS025F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss025/2/m_dss02501.jpeg?Expires=2147483647&Signature=IAVL6r7sobC5uKbODsKOI5G9t5F8DMc~t0ID6sI8HkiIXh1QL7KoK~XQr2otpdjdtz7KAMJ5a5vNEeZ9M-5ZNgp1i5ESwgr-f7WbDcorgFEMmij~xA~zz1LgZazYwXYiiuS7Cbb0tbM18OQjhbKAmvYUGpHO3OMYYkclwgLXZ27LWvVlQCgtqI4v4T~MaajP7GWVbYsqJfeCEiotX4dkxvr43URK~5SaKmlP1rqWiOaenqRCmlhooQRjiyqRvsvwMVpAnJE7VSkx1oMid4bekU3nJhOIxVzQL8mpbbXLhUZOFlrrzmegLW6aztoWiubC~Gm7QGP1C5RgqZGXAung1Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="An integrated genetic map of pigeonpea. This genetic map is developed on the F2 mapping population derived from ICP 28 × ICPW 94. The map comprises 910 loci (875 PKAMs and 35 SSRs) in which 444 PKAMs were developed based on GAIIx-SNPs, shown in red; 431 PKAMs were developed based on TOGs-SNPs, shown in green, and 35 SSRs previously mapped by Bohra et al.9 shown in black." path-from-xml="dss02501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7290850/dss02501.jpeg" path-from-xml="dss02501.jpeg" target="_blank">View large</a><a section="7290850" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss025/2/dss02501.jpeg?Expires=2147483647&Signature=Zfk2rsY3hrRJPBzxO-lrR4L42ElwUOdXNnEld-j98x~x406ur0fUSKUt8B8BczM1Z2MyhYXFw78eTJRoz9NtPVlmwUsnKw91sCmX9k8dIE5aG-mhxmXXU0Qg-OUNCR4lvBy-mPmiSGkIauSOMRpmdcq~SKFXhI3MKWzbrkWXUO6~KfB3WCcK4YnrQdf3uHE80Lp0RgjfjrmB5RtUqzfkUw0jd66ndKLzkAcSGmVt3mqTHrDQQXVbO47WfwB4s2Y8l1afFdAGux2DNWfDto5jp3dcDxqnQCSN7eASafEjhRGDM1WZzQgT4cF5x4rguYlI4-aqwZxUF0myS8oBSZcDZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7290850&ar=480862&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>An integrated genetic map of pigeonpea. This genetic map is developed on the F<sub>2</sub> mapping population derived from ICP 28 × ICPW 94. The map comprises 910 loci (875 PKAMs and 35 SSRs) in which 444 PKAMs were developed based on GAIIx-SNPs, shown in red; 431 PKAMs were developed based on TOGs-SNPs, shown in green, and 35 SSRs previously mapped by Bohra <em>et al</em>.<sup><a reveal-id="DSS025C9" data-open="DSS025C9" class="link link-ref link-reveal xref-bibr">9</a></sup> shown in black.</p></div></div>                            <h3 scrollto-destination=7290851 id="7290851" class="section-title" data-legacy-id=s3f>3.6. Genome relationships of pigeonpea with closely related legume species</h3>
<p>For comparative genome analysis and to identify conserved synteny between genomes of pigeonpea and other related legume species, we combined both the genetic map position information for pigeonpea loci and genome sequence information of closely related species of different clades. A set of 875 mapped loci for which both genetic map positions and sequence information were available (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>) was compared with the genome assemblies of soybean (Glyma1), <em>Medicago</em> (Mt 3.5) and <em>Lotus</em> (Lj 2.5 pseudomolecules) and with the transcript-specific genetic map of cowpea.<sup><a reveal-id="DSS025C23" data-open="DSS025C23" class="link link-ref link-reveal xref-bibr">23</a></sup> In the comparison of pigeonpea with soybean, the highest percentage of the sequence similarity was identified. As expected, each pigeonpea LG shows extensive synteny with two or more chromosomes in soybean (Fig. <a reveal-id="DSS025F2" data-open="DSS025F2" class="link link-reveal link-table xref-fig">2</a>a), probably due to the independent duplication event in the soybean genome.<sup><a reveal-id="DSS025C39" data-open="DSS025C39" class="link link-ref link-reveal xref-bibr">39</a></sup> A total of 687 pigeonpea unique loci matched with 2480 soybean sequence stretches distributed on different chromosomes of soybean (Glyma1 assembly; Table <a reveal-id="DSS025TB3" data-open="DSS025TB3" class="link link-reveal link-table xref-fig">3</a>). Maximum similarity was identified between CcLG04 with Gm13 followed by CcLG02 with Gm10, CcLG03 with Gm06 and Gm04, CcLG03 with Gm06, CcLG06 with Gm01, CcLG10 with Gm15, CcLG07 with Gm18, CcLG08 with Gm15, CcLG11 with Gm17, CcLG01 with Gm18, CcLG09 with Gm08 and CcLG05 with Gm08. </p>                        <a id="7290856" scrollto-destination="7290856"></a>
<div content-id="DSS025TB3" class="table-modal"><div class="table-wrap"><div id="DSS025TB3" class="table-wrap-title" data-id="DSS025TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Detailed results on the comparison of mapped marker loci of pigeonpea with the soybean (<em>G. max</em>) genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">LGs </th><th rowspan="2">Pigeonpea unique loci (no.) </th><th colspan="30"><em>Glycine max</em> chromosomes/Scaffolds<hr /> </th></tr><tr><th>Gm01 </th><th>Gm02 </th><th>Gm03 </th><th>Gm04 </th><th>Gm05 </th><th>Gm06 </th><th>Gm07 </th><th>Gm08 </th><th>Gm09 </th><th>Gm10 </th><th>Gm11 </th><th>Gm12 </th><th>Gm13 </th><th>Gm14 </th><th>Gm15 </th><th>Gm16 </th><th>Gm17 </th><th>Gm18 </th><th>Gm19 </th><th>Gm20 </th><th>Scaffold_1129 </th><th>Scaffold_41 </th><th>Scaffold_42 </th><th>Scaffold_317 </th><th>Scaffold_96 </th><th>Scaffold_23 </th><th>Scaffold_1337 </th><th>Scaffold_1655 </th><th>Scaffold_90 </th><th>Total </th></tr></thead><tbody><tr><td>CcLG01 </td><td>52 </td><td>7 </td><td>10 </td><td>15 </td><td>3 </td><td>2 </td><td>1 </td><td>16 </td><td>18 </td><td>2 </td><td>8 </td><td>32 </td><td>2 </td><td>2 </td><td>4 </td><td>2 </td><td>7 </td><td>2 </td><td>45 </td><td>— </td><td>5 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>183 </td></tr><tr><td>CcLG02 </td><td>101 </td><td>4 </td><td>42 </td><td>6 </td><td>3 </td><td>— </td><td>10 </td><td>3 </td><td>3 </td><td>4 </td><td><strong>83</strong> </td><td>1 </td><td>6 </td><td>14 </td><td>41 </td><td>5 </td><td>2 </td><td>11 </td><td>2 </td><td>7 </td><td>59 </td><td>2 </td><td>2 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>310 </td></tr><tr><td>CcLG03 </td><td>99 </td><td>5 </td><td>6 </td><td>48 </td><td><strong>78</strong> </td><td>6 </td><td><strong>78</strong> </td><td>7 </td><td>5 </td><td>2 </td><td>7 </td><td>3 </td><td>3 </td><td>7 </td><td>7 </td><td>4 </td><td>6 </td><td>3 </td><td>2 </td><td>44 </td><td>6 </td><td>— </td><td>1 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>328 </td></tr><tr><td>CcLG04 </td><td>52 </td><td>2 </td><td>2 </td><td>— </td><td>2 </td><td>2 </td><td>12 </td><td>— </td><td>3 </td><td>1 </td><td>— </td><td>53 </td><td>54 </td><td><strong>106</strong> </td><td>— </td><td>5 </td><td>1 </td><td>— </td><td>5 </td><td>0 </td><td>1 </td><td>— </td><td>— </td><td>1 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>250 </td></tr><tr><td>CcLG05 </td><td>23 </td><td>2 </td><td>— </td><td>— </td><td>7 </td><td><strong>27</strong> </td><td>4 </td><td>10 </td><td><strong>27</strong> </td><td>2 </td><td>1 </td><td>— </td><td>— </td><td>— </td><td>1 </td><td>3 </td><td>— </td><td>— </td><td>1 </td><td>2 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>87 </td></tr><tr><td>CcLG06 </td><td>66 </td><td><strong>69</strong> </td><td>34 </td><td>3 </td><td>5 </td><td>2 </td><td>5 </td><td>8 </td><td>4 </td><td>20 </td><td>6 </td><td>35 </td><td>3 </td><td>3 </td><td>4 </td><td>3 </td><td>6 </td><td>8 </td><td>4 </td><td>6 </td><td>5 </td><td>— </td><td>— </td><td>— </td><td>2 </td><td>2 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>237 </td></tr><tr><td>CcLG07 </td><td>66 </td><td>5 </td><td>8 </td><td>6 </td><td>19 </td><td>4 </td><td>12 </td><td>14 </td><td>16 </td><td>23 </td><td>8 </td><td>4 </td><td>2 </td><td>3 </td><td>— </td><td>1 </td><td>14 </td><td>2 </td><td><strong>57</strong> </td><td>20 </td><td>4 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>2 </td><td>— </td><td>— </td><td>— </td><td>224 </td></tr><tr><td>CcLG08 </td><td>54 </td><td>5 </td><td>5 </td><td>2 </td><td>1 </td><td>4 </td><td>1 </td><td>12 </td><td>10 </td><td>49 </td><td>6 </td><td>3 </td><td>5 </td><td>18 </td><td>3 </td><td><strong>51</strong> </td><td>16 </td><td>5 </td><td>1 </td><td>2 </td><td>5 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>1 </td><td>1 </td><td> </td><td>206 </td></tr><tr><td>CcLG09 </td><td>41 </td><td>— </td><td>— </td><td>2 </td><td>1 </td><td>26 </td><td>— </td><td>5 </td><td><strong>31</strong> </td><td>10 </td><td>— </td><td>2 </td><td>2 </td><td>11 </td><td>— </td><td>2 </td><td>4 </td><td>1 </td><td>3 </td><td>12 </td><td>2 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>114 </td></tr><tr><td>CcLG10 </td><td>56 </td><td>9 </td><td>7 </td><td>8 </td><td>24 </td><td>7 </td><td>18 </td><td>8 </td><td>33 </td><td>5 </td><td>3 </td><td>11 </td><td>16 </td><td>40 </td><td>6 </td><td><strong>64</strong> </td><td>3 </td><td>5 </td><td>17 </td><td>4 </td><td>7 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>1 </td><td>296 </td></tr><tr><td>CcLG11 </td><td>77 </td><td>6 </td><td>5 </td><td>8 </td><td>5 </td><td>30 </td><td>1 </td><td>26 </td><td>21 </td><td>10 </td><td>2 </td><td>6 </td><td>1 </td><td>20 </td><td>8 </td><td>6 </td><td>21 </td><td><strong>47</strong> </td><td>13 </td><td>5 </td><td>4 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>245 </td></tr><tr><td>Total </td><td><strong>687</strong> </td><td><strong>114</strong> </td><td><strong>119</strong> </td><td><strong>98</strong> </td><td><strong>148</strong> </td><td><strong>110</strong> </td><td><strong>142</strong> </td><td><strong>109</strong> </td><td><strong>171</strong> </td><td><strong>128</strong> </td><td><strong>124</strong> </td><td><strong>150</strong> </td><td><strong>94</strong> </td><td><strong>224</strong> </td><td><strong>74</strong> </td><td><strong>146</strong> </td><td><strong>80</strong> </td><td><strong>84</strong> </td><td><strong>150</strong> </td><td><strong>102</strong> </td><td><strong>98</strong> </td><td><strong>2</strong> </td><td><strong>3</strong> </td><td><strong>1</strong> </td><td><strong>2</strong> </td><td><strong>2</strong> </td><td><strong>2</strong> </td><td><strong>1</strong> </td><td><strong>1</strong> </td><td><strong>1</strong> </td><td><strong>2480</strong> </td></tr></tbody></table></div><div content-id="DSS025TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">LGs </th><th rowspan="2">Pigeonpea unique loci (no.) </th><th colspan="30"><em>Glycine max</em> chromosomes/Scaffolds<hr /> </th></tr><tr><th>Gm01 </th><th>Gm02 </th><th>Gm03 </th><th>Gm04 </th><th>Gm05 </th><th>Gm06 </th><th>Gm07 </th><th>Gm08 </th><th>Gm09 </th><th>Gm10 </th><th>Gm11 </th><th>Gm12 </th><th>Gm13 </th><th>Gm14 </th><th>Gm15 </th><th>Gm16 </th><th>Gm17 </th><th>Gm18 </th><th>Gm19 </th><th>Gm20 </th><th>Scaffold_1129 </th><th>Scaffold_41 </th><th>Scaffold_42 </th><th>Scaffold_317 </th><th>Scaffold_96 </th><th>Scaffold_23 </th><th>Scaffold_1337 </th><th>Scaffold_1655 </th><th>Scaffold_90 </th><th>Total </th></tr></thead><tbody><tr><td>CcLG01 </td><td>52 </td><td>7 </td><td>10 </td><td>15 </td><td>3 </td><td>2 </td><td>1 </td><td>16 </td><td>18 </td><td>2 </td><td>8 </td><td>32 </td><td>2 </td><td>2 </td><td>4 </td><td>2 </td><td>7 </td><td>2 </td><td>45 </td><td>— </td><td>5 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>183 </td></tr><tr><td>CcLG02 </td><td>101 </td><td>4 </td><td>42 </td><td>6 </td><td>3 </td><td>— </td><td>10 </td><td>3 </td><td>3 </td><td>4 </td><td><strong>83</strong> </td><td>1 </td><td>6 </td><td>14 </td><td>41 </td><td>5 </td><td>2 </td><td>11 </td><td>2 </td><td>7 </td><td>59 </td><td>2 </td><td>2 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>310 </td></tr><tr><td>CcLG03 </td><td>99 </td><td>5 </td><td>6 </td><td>48 </td><td><strong>78</strong> </td><td>6 </td><td><strong>78</strong> </td><td>7 </td><td>5 </td><td>2 </td><td>7 </td><td>3 </td><td>3 </td><td>7 </td><td>7 </td><td>4 </td><td>6 </td><td>3 </td><td>2 </td><td>44 </td><td>6 </td><td>— </td><td>1 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>328 </td></tr><tr><td>CcLG04 </td><td>52 </td><td>2 </td><td>2 </td><td>— </td><td>2 </td><td>2 </td><td>12 </td><td>— </td><td>3 </td><td>1 </td><td>— </td><td>53 </td><td>54 </td><td><strong>106</strong> </td><td>— </td><td>5 </td><td>1 </td><td>— </td><td>5 </td><td>0 </td><td>1 </td><td>— </td><td>— </td><td>1 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>250 </td></tr><tr><td>CcLG05 </td><td>23 </td><td>2 </td><td>— </td><td>— </td><td>7 </td><td><strong>27</strong> </td><td>4 </td><td>10 </td><td><strong>27</strong> </td><td>2 </td><td>1 </td><td>— </td><td>— </td><td>— </td><td>1 </td><td>3 </td><td>— </td><td>— </td><td>1 </td><td>2 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>87 </td></tr><tr><td>CcLG06 </td><td>66 </td><td><strong>69</strong> </td><td>34 </td><td>3 </td><td>5 </td><td>2 </td><td>5 </td><td>8 </td><td>4 </td><td>20 </td><td>6 </td><td>35 </td><td>3 </td><td>3 </td><td>4 </td><td>3 </td><td>6 </td><td>8 </td><td>4 </td><td>6 </td><td>5 </td><td>— </td><td>— </td><td>— </td><td>2 </td><td>2 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>237 </td></tr><tr><td>CcLG07 </td><td>66 </td><td>5 </td><td>8 </td><td>6 </td><td>19 </td><td>4 </td><td>12 </td><td>14 </td><td>16 </td><td>23 </td><td>8 </td><td>4 </td><td>2 </td><td>3 </td><td>— </td><td>1 </td><td>14 </td><td>2 </td><td><strong>57</strong> </td><td>20 </td><td>4 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>2 </td><td>— </td><td>— </td><td>— </td><td>224 </td></tr><tr><td>CcLG08 </td><td>54 </td><td>5 </td><td>5 </td><td>2 </td><td>1 </td><td>4 </td><td>1 </td><td>12 </td><td>10 </td><td>49 </td><td>6 </td><td>3 </td><td>5 </td><td>18 </td><td>3 </td><td><strong>51</strong> </td><td>16 </td><td>5 </td><td>1 </td><td>2 </td><td>5 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>1 </td><td>1 </td><td> </td><td>206 </td></tr><tr><td>CcLG09 </td><td>41 </td><td>— </td><td>— </td><td>2 </td><td>1 </td><td>26 </td><td>— </td><td>5 </td><td><strong>31</strong> </td><td>10 </td><td>— </td><td>2 </td><td>2 </td><td>11 </td><td>— </td><td>2 </td><td>4 </td><td>1 </td><td>3 </td><td>12 </td><td>2 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>114 </td></tr><tr><td>CcLG10 </td><td>56 </td><td>9 </td><td>7 </td><td>8 </td><td>24 </td><td>7 </td><td>18 </td><td>8 </td><td>33 </td><td>5 </td><td>3 </td><td>11 </td><td>16 </td><td>40 </td><td>6 </td><td><strong>64</strong> </td><td>3 </td><td>5 </td><td>17 </td><td>4 </td><td>7 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>1 </td><td>296 </td></tr><tr><td>CcLG11 </td><td>77 </td><td>6 </td><td>5 </td><td>8 </td><td>5 </td><td>30 </td><td>1 </td><td>26 </td><td>21 </td><td>10 </td><td>2 </td><td>6 </td><td>1 </td><td>20 </td><td>8 </td><td>6 </td><td>21 </td><td><strong>47</strong> </td><td>13 </td><td>5 </td><td>4 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>245 </td></tr><tr><td>Total </td><td><strong>687</strong> </td><td><strong>114</strong> </td><td><strong>119</strong> </td><td><strong>98</strong> </td><td><strong>148</strong> </td><td><strong>110</strong> </td><td><strong>142</strong> </td><td><strong>109</strong> </td><td><strong>171</strong> </td><td><strong>128</strong> </td><td><strong>124</strong> </td><td><strong>150</strong> </td><td><strong>94</strong> </td><td><strong>224</strong> </td><td><strong>74</strong> </td><td><strong>146</strong> </td><td><strong>80</strong> </td><td><strong>84</strong> </td><td><strong>150</strong> </td><td><strong>102</strong> </td><td><strong>98</strong> </td><td><strong>2</strong> </td><td><strong>3</strong> </td><td><strong>1</strong> </td><td><strong>2</strong> </td><td><strong>2</strong> </td><td><strong>2</strong> </td><td><strong>1</strong> </td><td><strong>1</strong> </td><td><strong>1</strong> </td><td><strong>2480</strong> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The numbers shown in bold represent the highest matches between pigeonpea and soybean.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7290856" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS025TB3" class="table-wrap-title" data-id="DSS025TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Detailed results on the comparison of mapped marker loci of pigeonpea with the soybean (<em>G. max</em>) genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">LGs </th><th rowspan="2">Pigeonpea unique loci (no.) </th><th colspan="30"><em>Glycine max</em> chromosomes/Scaffolds<hr /> </th></tr><tr><th>Gm01 </th><th>Gm02 </th><th>Gm03 </th><th>Gm04 </th><th>Gm05 </th><th>Gm06 </th><th>Gm07 </th><th>Gm08 </th><th>Gm09 </th><th>Gm10 </th><th>Gm11 </th><th>Gm12 </th><th>Gm13 </th><th>Gm14 </th><th>Gm15 </th><th>Gm16 </th><th>Gm17 </th><th>Gm18 </th><th>Gm19 </th><th>Gm20 </th><th>Scaffold_1129 </th><th>Scaffold_41 </th><th>Scaffold_42 </th><th>Scaffold_317 </th><th>Scaffold_96 </th><th>Scaffold_23 </th><th>Scaffold_1337 </th><th>Scaffold_1655 </th><th>Scaffold_90 </th><th>Total </th></tr></thead><tbody><tr><td>CcLG01 </td><td>52 </td><td>7 </td><td>10 </td><td>15 </td><td>3 </td><td>2 </td><td>1 </td><td>16 </td><td>18 </td><td>2 </td><td>8 </td><td>32 </td><td>2 </td><td>2 </td><td>4 </td><td>2 </td><td>7 </td><td>2 </td><td>45 </td><td>— </td><td>5 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>183 </td></tr><tr><td>CcLG02 </td><td>101 </td><td>4 </td><td>42 </td><td>6 </td><td>3 </td><td>— </td><td>10 </td><td>3 </td><td>3 </td><td>4 </td><td><strong>83</strong> </td><td>1 </td><td>6 </td><td>14 </td><td>41 </td><td>5 </td><td>2 </td><td>11 </td><td>2 </td><td>7 </td><td>59 </td><td>2 </td><td>2 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>310 </td></tr><tr><td>CcLG03 </td><td>99 </td><td>5 </td><td>6 </td><td>48 </td><td><strong>78</strong> </td><td>6 </td><td><strong>78</strong> </td><td>7 </td><td>5 </td><td>2 </td><td>7 </td><td>3 </td><td>3 </td><td>7 </td><td>7 </td><td>4 </td><td>6 </td><td>3 </td><td>2 </td><td>44 </td><td>6 </td><td>— </td><td>1 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>328 </td></tr><tr><td>CcLG04 </td><td>52 </td><td>2 </td><td>2 </td><td>— </td><td>2 </td><td>2 </td><td>12 </td><td>— </td><td>3 </td><td>1 </td><td>— </td><td>53 </td><td>54 </td><td><strong>106</strong> </td><td>— </td><td>5 </td><td>1 </td><td>— </td><td>5 </td><td>0 </td><td>1 </td><td>— </td><td>— </td><td>1 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>250 </td></tr><tr><td>CcLG05 </td><td>23 </td><td>2 </td><td>— </td><td>— </td><td>7 </td><td><strong>27</strong> </td><td>4 </td><td>10 </td><td><strong>27</strong> </td><td>2 </td><td>1 </td><td>— </td><td>— </td><td>— </td><td>1 </td><td>3 </td><td>— </td><td>— </td><td>1 </td><td>2 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>87 </td></tr><tr><td>CcLG06 </td><td>66 </td><td><strong>69</strong> </td><td>34 </td><td>3 </td><td>5 </td><td>2 </td><td>5 </td><td>8 </td><td>4 </td><td>20 </td><td>6 </td><td>35 </td><td>3 </td><td>3 </td><td>4 </td><td>3 </td><td>6 </td><td>8 </td><td>4 </td><td>6 </td><td>5 </td><td>— </td><td>— </td><td>— </td><td>2 </td><td>2 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>237 </td></tr><tr><td>CcLG07 </td><td>66 </td><td>5 </td><td>8 </td><td>6 </td><td>19 </td><td>4 </td><td>12 </td><td>14 </td><td>16 </td><td>23 </td><td>8 </td><td>4 </td><td>2 </td><td>3 </td><td>— </td><td>1 </td><td>14 </td><td>2 </td><td><strong>57</strong> </td><td>20 </td><td>4 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>2 </td><td>— </td><td>— </td><td>— </td><td>224 </td></tr><tr><td>CcLG08 </td><td>54 </td><td>5 </td><td>5 </td><td>2 </td><td>1 </td><td>4 </td><td>1 </td><td>12 </td><td>10 </td><td>49 </td><td>6 </td><td>3 </td><td>5 </td><td>18 </td><td>3 </td><td><strong>51</strong> </td><td>16 </td><td>5 </td><td>1 </td><td>2 </td><td>5 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>1 </td><td>1 </td><td> </td><td>206 </td></tr><tr><td>CcLG09 </td><td>41 </td><td>— </td><td>— </td><td>2 </td><td>1 </td><td>26 </td><td>— </td><td>5 </td><td><strong>31</strong> </td><td>10 </td><td>— </td><td>2 </td><td>2 </td><td>11 </td><td>— </td><td>2 </td><td>4 </td><td>1 </td><td>3 </td><td>12 </td><td>2 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>114 </td></tr><tr><td>CcLG10 </td><td>56 </td><td>9 </td><td>7 </td><td>8 </td><td>24 </td><td>7 </td><td>18 </td><td>8 </td><td>33 </td><td>5 </td><td>3 </td><td>11 </td><td>16 </td><td>40 </td><td>6 </td><td><strong>64</strong> </td><td>3 </td><td>5 </td><td>17 </td><td>4 </td><td>7 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>1 </td><td>296 </td></tr><tr><td>CcLG11 </td><td>77 </td><td>6 </td><td>5 </td><td>8 </td><td>5 </td><td>30 </td><td>1 </td><td>26 </td><td>21 </td><td>10 </td><td>2 </td><td>6 </td><td>1 </td><td>20 </td><td>8 </td><td>6 </td><td>21 </td><td><strong>47</strong> </td><td>13 </td><td>5 </td><td>4 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>245 </td></tr><tr><td>Total </td><td><strong>687</strong> </td><td><strong>114</strong> </td><td><strong>119</strong> </td><td><strong>98</strong> </td><td><strong>148</strong> </td><td><strong>110</strong> </td><td><strong>142</strong> </td><td><strong>109</strong> </td><td><strong>171</strong> </td><td><strong>128</strong> </td><td><strong>124</strong> </td><td><strong>150</strong> </td><td><strong>94</strong> </td><td><strong>224</strong> </td><td><strong>74</strong> </td><td><strong>146</strong> </td><td><strong>80</strong> </td><td><strong>84</strong> </td><td><strong>150</strong> </td><td><strong>102</strong> </td><td><strong>98</strong> </td><td><strong>2</strong> </td><td><strong>3</strong> </td><td><strong>1</strong> </td><td><strong>2</strong> </td><td><strong>2</strong> </td><td><strong>2</strong> </td><td><strong>1</strong> </td><td><strong>1</strong> </td><td><strong>1</strong> </td><td><strong>2480</strong> </td></tr></tbody></table></div><div content-id="DSS025TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">LGs </th><th rowspan="2">Pigeonpea unique loci (no.) </th><th colspan="30"><em>Glycine max</em> chromosomes/Scaffolds<hr /> </th></tr><tr><th>Gm01 </th><th>Gm02 </th><th>Gm03 </th><th>Gm04 </th><th>Gm05 </th><th>Gm06 </th><th>Gm07 </th><th>Gm08 </th><th>Gm09 </th><th>Gm10 </th><th>Gm11 </th><th>Gm12 </th><th>Gm13 </th><th>Gm14 </th><th>Gm15 </th><th>Gm16 </th><th>Gm17 </th><th>Gm18 </th><th>Gm19 </th><th>Gm20 </th><th>Scaffold_1129 </th><th>Scaffold_41 </th><th>Scaffold_42 </th><th>Scaffold_317 </th><th>Scaffold_96 </th><th>Scaffold_23 </th><th>Scaffold_1337 </th><th>Scaffold_1655 </th><th>Scaffold_90 </th><th>Total </th></tr></thead><tbody><tr><td>CcLG01 </td><td>52 </td><td>7 </td><td>10 </td><td>15 </td><td>3 </td><td>2 </td><td>1 </td><td>16 </td><td>18 </td><td>2 </td><td>8 </td><td>32 </td><td>2 </td><td>2 </td><td>4 </td><td>2 </td><td>7 </td><td>2 </td><td>45 </td><td>— </td><td>5 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>183 </td></tr><tr><td>CcLG02 </td><td>101 </td><td>4 </td><td>42 </td><td>6 </td><td>3 </td><td>— </td><td>10 </td><td>3 </td><td>3 </td><td>4 </td><td><strong>83</strong> </td><td>1 </td><td>6 </td><td>14 </td><td>41 </td><td>5 </td><td>2 </td><td>11 </td><td>2 </td><td>7 </td><td>59 </td><td>2 </td><td>2 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>310 </td></tr><tr><td>CcLG03 </td><td>99 </td><td>5 </td><td>6 </td><td>48 </td><td><strong>78</strong> </td><td>6 </td><td><strong>78</strong> </td><td>7 </td><td>5 </td><td>2 </td><td>7 </td><td>3 </td><td>3 </td><td>7 </td><td>7 </td><td>4 </td><td>6 </td><td>3 </td><td>2 </td><td>44 </td><td>6 </td><td>— </td><td>1 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>328 </td></tr><tr><td>CcLG04 </td><td>52 </td><td>2 </td><td>2 </td><td>— </td><td>2 </td><td>2 </td><td>12 </td><td>— </td><td>3 </td><td>1 </td><td>— </td><td>53 </td><td>54 </td><td><strong>106</strong> </td><td>— </td><td>5 </td><td>1 </td><td>— </td><td>5 </td><td>0 </td><td>1 </td><td>— </td><td>— </td><td>1 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>250 </td></tr><tr><td>CcLG05 </td><td>23 </td><td>2 </td><td>— </td><td>— </td><td>7 </td><td><strong>27</strong> </td><td>4 </td><td>10 </td><td><strong>27</strong> </td><td>2 </td><td>1 </td><td>— </td><td>— </td><td>— </td><td>1 </td><td>3 </td><td>— </td><td>— </td><td>1 </td><td>2 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>87 </td></tr><tr><td>CcLG06 </td><td>66 </td><td><strong>69</strong> </td><td>34 </td><td>3 </td><td>5 </td><td>2 </td><td>5 </td><td>8 </td><td>4 </td><td>20 </td><td>6 </td><td>35 </td><td>3 </td><td>3 </td><td>4 </td><td>3 </td><td>6 </td><td>8 </td><td>4 </td><td>6 </td><td>5 </td><td>— </td><td>— </td><td>— </td><td>2 </td><td>2 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>237 </td></tr><tr><td>CcLG07 </td><td>66 </td><td>5 </td><td>8 </td><td>6 </td><td>19 </td><td>4 </td><td>12 </td><td>14 </td><td>16 </td><td>23 </td><td>8 </td><td>4 </td><td>2 </td><td>3 </td><td>— </td><td>1 </td><td>14 </td><td>2 </td><td><strong>57</strong> </td><td>20 </td><td>4 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>2 </td><td>— </td><td>— </td><td>— </td><td>224 </td></tr><tr><td>CcLG08 </td><td>54 </td><td>5 </td><td>5 </td><td>2 </td><td>1 </td><td>4 </td><td>1 </td><td>12 </td><td>10 </td><td>49 </td><td>6 </td><td>3 </td><td>5 </td><td>18 </td><td>3 </td><td><strong>51</strong> </td><td>16 </td><td>5 </td><td>1 </td><td>2 </td><td>5 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>1 </td><td>1 </td><td> </td><td>206 </td></tr><tr><td>CcLG09 </td><td>41 </td><td>— </td><td>— </td><td>2 </td><td>1 </td><td>26 </td><td>— </td><td>5 </td><td><strong>31</strong> </td><td>10 </td><td>— </td><td>2 </td><td>2 </td><td>11 </td><td>— </td><td>2 </td><td>4 </td><td>1 </td><td>3 </td><td>12 </td><td>2 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>114 </td></tr><tr><td>CcLG10 </td><td>56 </td><td>9 </td><td>7 </td><td>8 </td><td>24 </td><td>7 </td><td>18 </td><td>8 </td><td>33 </td><td>5 </td><td>3 </td><td>11 </td><td>16 </td><td>40 </td><td>6 </td><td><strong>64</strong> </td><td>3 </td><td>5 </td><td>17 </td><td>4 </td><td>7 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>1 </td><td>296 </td></tr><tr><td>CcLG11 </td><td>77 </td><td>6 </td><td>5 </td><td>8 </td><td>5 </td><td>30 </td><td>1 </td><td>26 </td><td>21 </td><td>10 </td><td>2 </td><td>6 </td><td>1 </td><td>20 </td><td>8 </td><td>6 </td><td>21 </td><td><strong>47</strong> </td><td>13 </td><td>5 </td><td>4 </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>245 </td></tr><tr><td>Total </td><td><strong>687</strong> </td><td><strong>114</strong> </td><td><strong>119</strong> </td><td><strong>98</strong> </td><td><strong>148</strong> </td><td><strong>110</strong> </td><td><strong>142</strong> </td><td><strong>109</strong> </td><td><strong>171</strong> </td><td><strong>128</strong> </td><td><strong>124</strong> </td><td><strong>150</strong> </td><td><strong>94</strong> </td><td><strong>224</strong> </td><td><strong>74</strong> </td><td><strong>146</strong> </td><td><strong>80</strong> </td><td><strong>84</strong> </td><td><strong>150</strong> </td><td><strong>102</strong> </td><td><strong>98</strong> </td><td><strong>2</strong> </td><td><strong>3</strong> </td><td><strong>1</strong> </td><td><strong>2</strong> </td><td><strong>2</strong> </td><td><strong>2</strong> </td><td><strong>1</strong> </td><td><strong>1</strong> </td><td><strong>1</strong> </td><td><strong>2480</strong> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The numbers shown in bold represent the highest matches between pigeonpea and soybean.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7290856" target="_blank">View Large</a></div></div>                        <a id="7290857" scrollto-destination="7290857"></a>
<div id="DSS025F2" data-id="DSS025F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss025/2/m_dss02502.jpeg?Expires=2147483647&Signature=paTcYU5LzfECDqAeI3F7TjMu4CpyFMV9E4ZiEeB9Xg-7Bkf1RjqhBWplsstPapyjagDS4ke0JFMhIF7CLAwnYslbfN~rYySsbYJw9Ql7ixMjG746ktdYAZEXOtgSRKYcEJHiXl7Lc4hURpdMQR9UNF7HdC4ZmXc6Mm6AsEFF8ZPrkcUBxsfCbWoi~bnW1OyIxmMrWcjU6oCjLtWWQnrHCjL6AFs3H~ko5e5vHwUvFrXmdhd9IaMbu3KsqUzmKRNFW3KjSjpBY2VNn4ai1rVtVqz6r0Ul~byGHePOf62JBYJL7HKWltNT7s04qwa8IriiOeAYmN-~MhJdNcGb3vzygA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Syntenic relationships of individual LGs of pigeonpea with other legumes. Each line radiating from a pigeonpea LG represents a similarity match found in a block between pigeonpea and other legumes. (a) Pigeonpea LGs showing synteny with the genome assembly of soybean, (b) pigeonpea LGs showing synteny with the cowpea transcript map, (c) pigeonpea LGs showing synteny with the genome assembly of Medicago and (d) pigeonpea LGs showing synteny with the genome assembly of Lotus." path-from-xml="dss02502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7290857/dss02502.jpeg" path-from-xml="dss02502.jpeg" target="_blank">View large</a><a section="7290857" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss025/2/dss02502.jpeg?Expires=2147483647&Signature=uX3Lqru9Dm5u3Onf4iaM6e2Wtd1jYmW4X94RNKqhik9WSoZAI~q2xLeTtnv2rmKjk3k4PTbG3uheCNmkCgO7j97cZtAV-mu4iM9y4gwWC9sG7uMcQF-fn7hvOuv0rwrlvNgzIqW69djH3Spc9UKbJem9ZPhXSqGh8WVsps4nrGJc0rxmyARtl8J4xYp8RQs16TQ2JuIOfC4Xp5b7lTWfQaX1FWd3BcynP3ErKhDh0kRxNiqXkQ0ToccYS0clB4lv-n8AXEwCDFWjml91u5QxVSv2wRQ0r4tMq0bkXV-D0H5bOp4ELbpClaCzn~UC2UznOygdxU1NiSVoZpd9DBNjxw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7290857&ar=480862&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Syntenic relationships of individual LGs of pigeonpea with other legumes. Each line radiating from a pigeonpea LG represents a similarity match found in a block between pigeonpea and other legumes. (a) Pigeonpea LGs showing synteny with the genome assembly of soybean, (b) pigeonpea LGs showing synteny with the cowpea transcript map, (c) pigeonpea LGs showing synteny with the genome assembly of <em>Medicago</em> and (d) pigeonpea LGs showing synteny with the genome assembly of <em>Lotus</em>.</p></div></div><div content-id="DSS025F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss025/2/m_dss02502.jpeg?Expires=2147483647&Signature=paTcYU5LzfECDqAeI3F7TjMu4CpyFMV9E4ZiEeB9Xg-7Bkf1RjqhBWplsstPapyjagDS4ke0JFMhIF7CLAwnYslbfN~rYySsbYJw9Ql7ixMjG746ktdYAZEXOtgSRKYcEJHiXl7Lc4hURpdMQR9UNF7HdC4ZmXc6Mm6AsEFF8ZPrkcUBxsfCbWoi~bnW1OyIxmMrWcjU6oCjLtWWQnrHCjL6AFs3H~ko5e5vHwUvFrXmdhd9IaMbu3KsqUzmKRNFW3KjSjpBY2VNn4ai1rVtVqz6r0Ul~byGHePOf62JBYJL7HKWltNT7s04qwa8IriiOeAYmN-~MhJdNcGb3vzygA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Syntenic relationships of individual LGs of pigeonpea with other legumes. Each line radiating from a pigeonpea LG represents a similarity match found in a block between pigeonpea and other legumes. (a) Pigeonpea LGs showing synteny with the genome assembly of soybean, (b) pigeonpea LGs showing synteny with the cowpea transcript map, (c) pigeonpea LGs showing synteny with the genome assembly of Medicago and (d) pigeonpea LGs showing synteny with the genome assembly of Lotus." path-from-xml="dss02502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7290857/dss02502.jpeg" path-from-xml="dss02502.jpeg" target="_blank">View large</a><a section="7290857" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss025/2/dss02502.jpeg?Expires=2147483647&Signature=uX3Lqru9Dm5u3Onf4iaM6e2Wtd1jYmW4X94RNKqhik9WSoZAI~q2xLeTtnv2rmKjk3k4PTbG3uheCNmkCgO7j97cZtAV-mu4iM9y4gwWC9sG7uMcQF-fn7hvOuv0rwrlvNgzIqW69djH3Spc9UKbJem9ZPhXSqGh8WVsps4nrGJc0rxmyARtl8J4xYp8RQs16TQ2JuIOfC4Xp5b7lTWfQaX1FWd3BcynP3ErKhDh0kRxNiqXkQ0ToccYS0clB4lv-n8AXEwCDFWjml91u5QxVSv2wRQ0r4tMq0bkXV-D0H5bOp4ELbpClaCzn~UC2UznOygdxU1NiSVoZpd9DBNjxw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7290857&ar=480862&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Syntenic relationships of individual LGs of pigeonpea with other legumes. Each line radiating from a pigeonpea LG represents a similarity match found in a block between pigeonpea and other legumes. (a) Pigeonpea LGs showing synteny with the genome assembly of soybean, (b) pigeonpea LGs showing synteny with the cowpea transcript map, (c) pigeonpea LGs showing synteny with the genome assembly of <em>Medicago</em> and (d) pigeonpea LGs showing synteny with the genome assembly of <em>Lotus</em>.</p></div></div><p>In the case of cowpea in which the genetic map was used for the comparison, least matches were observed between pigeonpea and cowpea genomes. Only 57 unique pigeonpea loci showed synteny with 62 loci on the  cowpea map (Fig. <a reveal-id="DSS025F2" data-open="DSS025F2" class="link link-reveal link-table xref-fig">2</a>b, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). In the case of pigeonpea and <em>Medicago</em>, 228 unique pigeonpea loci showed significant matches with 349 genomic regions on the <em>Medicago</em> chromosome (Fig. <a reveal-id="DSS025F2" data-open="DSS025F2" class="link link-reveal link-table xref-fig">2</a>c, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). A total of 20 pigeonpea loci from CaLG02 showed similarity to MtChr01 genomic regions. A similar number of loci from CcLG03 showed similarity to MtChr03 genomic regions. CcLG04 showed almost equal similarity to MtChr04 (21) and MtChr05 (18). Similarly, loci from CcLG07 showed maximum matches to MtChr07 followed by CcLG08 with MtChr02, CcLG05 with MtChr08, CcLG06 with MtChr05 and CcLG11 with MtChr04. In the comparison of pigeonpea with <em>Lotus</em>, 216 pigeonpea unique loci matched with 303 different genomic regions on the <em>Lotus</em> chromosomes (Fig. <a reveal-id="DSS025F2" data-open="DSS025F2" class="link link-reveal link-table xref-fig">2</a>d, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). In brief, each LG of pigeonpea showed considerable synteny with one or more chromosomes of <em>Medicago</em> and <em>Lotus</em>. The distribution of similarity hits across eight pigeonpea LGs was varied from 2 (CcLG05) to 9 (CcLG09) while comparing with cowpea, from 14 (CcLG09) to 55 (CcLG04) with <em>Medicago</em> and from 18 (CcLG01) to 74 (CcL03) with <em>Lotus</em>.</p>                            <h2 scrollto-destination=7290859 id="7290859" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>The current availability of more than 3000 PCR-based markers in pigeonpea<sup><a reveal-id="DSS025C8" data-open="DSS025C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSS025C9" data-open="DSS025C9" class="link link-ref link-reveal xref-bibr">9</a></sup> could not provide high or significant marker density in any of the populations to be adequate to allow a thorough scan of the genome for QTL discovery, association analysis, map-based cloning and anchoring of the genome sequence with the genetic map. This can be attributed to the low level of polymorphism in <em>Cajanus</em> spp. as well as the small number of lines in the mapping populations used for developing the genetic maps. To overcome the above-mentioned problems to some extent, a new mapping population with 167 F<sub>2</sub> lines, compared with 72 lines used in the map of Bohra <em>et al</em>.<sup><a reveal-id="DSS025C9" data-open="DSS025C9" class="link link-ref link-reveal xref-bibr">9</a></sup> , was developed and used for developing the genetic map. Furthermore, SNP markers were targeted for developing the cost-effective genotyping platform and developing the genetic map. As the SNP markers were derived from genes, the comparison of the SNP-based genetic map of pigeonpea with the genome sequence assemblies of soybean, <em>Medicago</em> and <em>Lotus</em> and the transcript map of cowpea provided the genome relationship of pigeonpea with the genomes of these legumes.</p><p>When compared with the other marker systems, SNP markers are unique with regard to their amenability to high-throughput and low-cost (per data point) genotyping platforms.<sup><a reveal-id="DSS025C17" data-open="DSS025C17" class="link link-ref link-reveal xref-bibr">17</a></sup> In the case of pigeonpea, a total of 17 113 SNPs were discovered after comparing the transcript sequence reads from 12 parental lines of six different mapping populations with the transcriptome and/or genome sequence of pigeonpea.<sup><a reveal-id="DSS025C6" data-open="DSS025C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSS025C13" data-open="DSS025C13" class="link link-ref link-reveal xref-bibr">13</a></sup> To prove the usefulness of these predicted polymorphisms for practical plant breeding applications, validation of these SNPs is required. For this purpose, a number of SNP genotyping platforms such as GoldenGate, Infinium and KASPar assays are available. However, in this study, due to its cost- effective and flexible nature, the KASPar assay was developed for 1616 SNPs. KASPar assays can be flexibly used to validate any number of SNP markers on a desired range of accessions, unlike many other SNP genotyping platforms such as GoldenGate or Infinium assay, to produce a sufficient number of polymorphic markers in a given population to obtain a better coverage. In the published literature, three reports are available on the development of KASPar assays in crop plants. For instance, in the case of wheat, KASPar assays were developed for 1114 SNPs and validated on 23 wheat varieties and also used for integrating SNP markers into the genetic map of wheat.<sup><a reveal-id="DSS025C26" data-open="DSS025C26" class="link link-ref link-reveal xref-bibr">26</a></sup> In the case of common bean, KASpar assays have been developed for 94 SNPs and used for analysing genetic diversity in 70 accessions.<sup><a reveal-id="DSS025C27" data-open="DSS025C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Very recently, Hiremath <em>et al</em>.<sup><a reveal-id="DSS025C28" data-open="DSS025C28" class="link link-ref link-reveal xref-bibr">28</a></sup> developed KASpar assays for 2005 SNPs in chickpea and used these for genetic diversity analysis and genetic mapping in chickpea and comparative mapping in legumes. A comparison of genotyping ∼100 chickpea lines with ∼500 SNPs on GoldenGate and KASPar assays shows the superiority of KASPar assays over GoldenGate assays in terms of cost as well as time used. In summary, all these three studies underline the importance of KASPar assays for SNP genotyping on a large scale for genetics and breeding applications. In the present study, though 1827 SNPs were attempted for conversion into KASPar assays, only 1616 (88.4%) markers could be successfully converted. The failure of the remaining SNP markers (11.6%) to be validated is likely due to the presence of paralogous sequences, incorrect primer designing and/or the need to optimize PCR conditions. This conversion rate is higher than that of the other KASPar studies on wheat (67%)<sup><a reveal-id="DSS025C26" data-open="DSS025C26" class="link link-ref link-reveal xref-bibr">26</a></sup> and chickpea (80.6%).<sup><a reveal-id="DSS025C28" data-open="DSS025C28" class="link link-ref link-reveal xref-bibr">28</a></sup> This rate of conversion from selected SNPs to functional KASPar assays could probably be increased with optimization of primer designing and amplification conditions. However, we have made no attempt to optimize the failed assays. For ease of use, developed KASPar assay-based markers were designated as PKAMs.</p><p>Screening of a new set of SNP markers (KASPar assays) on a set of 24 diverse pigeonpea genotypes representing the parents of 14 mapping populations segregating for various economically important traits provides readily available polymorphic markers for developing genetic maps and undertaking trait mapping in the respective mapping populations. In fact, these crossing combinations were selected based on diversity revealed through trait phenotyping and SSR profiling data.<sup><a reveal-id="DSS025C40" data-open="DSS025C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Although KASPar assays were developed for SNPs identified to be polymorphic between ICP 28 and ICPW 94, only 71.4% (1154) markers showed polymorphism in the tested genotypes. The remaining 28.6% (462) markers did not show polymorphism at all, including between the ICP 28 and ICPW 94 genotypes, indicating incorrect prediction. Out of 462 monomorphic markers, 379 (82.1%) markers were identified based on the Illumina GA IIx transcript sequence data, and the remaining 83 (17.9%) SNPs were identified based on allele re-sequencing of TOGs. This emphasizes the need of stringent selection criteria and the validation of <em>in silico</em> identified SNPs via allele re-sequencing.<sup><a reveal-id="DSS025C28" data-open="DSS025C28" class="link link-ref link-reveal xref-bibr">28</a></sup> In brief, this study provides a list of polymorphic markers for different mapping populations that segregate for a number of important traits like <em>Fusarium</em> wilt, sterility mosaic disease and fertility restoration etc. that are important for pigeonpea improvement. The number of polymorphic markers identified in this study in intra-specific mapping populations was less (up to 55 markers in a given cross; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>); however, these polymorphic markers would be helpful in enriching the recently developed SSR-based genetic linkage maps of intra-specific mapping populations.<sup><a reveal-id="DSS025C14" data-open="DSS025C14" class="link link-ref link-reveal xref-bibr">14</a></sup></p><p>The present study reports a comprehensive genetic map comprising 875 loci by using 167 F<sub>2</sub> individuals derived from ICP 28 × ICPW 94. Initial construction of a skeletal map with un-skewed markers and followed by the integration of distorted markers helped in minimizing the possibility for spurious assignments of markers.<sup><a reveal-id="DSS025C9" data-open="DSS025C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Eight hundred and seventy-five marker loci were mapped on 11 LGs corresponding to the 11 chromosome pairs of the pigeonpea genome. The total length of the map was 967.03 cM and the average marker spacing was 1.11 cM. The current pigeonpea linkage map is a considerable improvement over the previous pigeonpea genetic linkage maps using SSR and DArT markers.<sup><a reveal-id="DSS025C9" data-open="DSS025C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSS025C12" data-open="DSS025C12" class="link link-ref link-reveal xref-bibr">12</a></sup> The marker density in the current map is almost three times higher than that in the previous maps. This higher marker density would be useful in determining double recombinants affecting a single marker. SNP genotyping using KASPar assays resulted in a much lower genotyping error rate than that obtained with markers like SSRs. In addition, SNP markers showing null alleles or an excess number of double recombinants were removed from the analyses. Owing to this careful error checking, the current map shows an increase in the total marker density compared with the previous maps developed by using SSR and DArT markers.<sup><a reveal-id="DSS025C9" data-open="DSS025C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSS025C12" data-open="DSS025C12" class="link link-ref link-reveal xref-bibr">12</a></sup> However, we have noticed that in two LGs (i.e. CcLG02 and CcLG06), a few marker spaces were larger than 20 cM. Therefore, it is required to develop more markers and fill the gaps in the large marker space to increase the density of the linkage map. Earlier to this map, Bohra <em>et al</em>.<sup><a reveal-id="DSS025C9" data-open="DSS025C9" class="link link-ref link-reveal xref-bibr">9</a></sup> developed an SSR marker-based map comprising 239 loci by using 72 F<sub>2</sub> lines derived from ICP 28 × ICPW 94. To develop a consensus map, the newly developed mapping population (ICP 28 × ICPW 94) was also genotyped with targeted SSRs and an integrated genetic linkage map covering 996.21 cM was constructed. With the help of a recently available pigeonpea draft genome sequence,<sup><a reveal-id="DSS025C13" data-open="DSS025C13" class="link link-ref link-reveal xref-bibr">13</a></sup> efforts are underway to develop a large number of SSR and SNP markers. Therefore, this map should serve as a ‘reference map’ for other future genetic maps of pigeonpea. Moreover, as the SNP markers are derived from the transcriptome sequences, these markers and the map would be very useful for linking the future genetic maps and the genome sequence of pigeonpea. In addition to the polymorphic markers in parental combinations of intra-specific mapping populations, most of the mapped PKAM markers in the inter-specific mapping population were monomorphic in cultivated parental lines. However, these mapped loci have provided a resource that can be used for conducting association analysis and linkage disequilibrium estimation in pigeonpea germplasm.</p><p>The recent development of a large data set of crop genomic sequences has aided in global gene predictions as well as in the identification of sequences important in gene regulation. In addition, comparative sequence analysis of crops is poised to contribute to the exploration of the genetic bases for differences and similarities among species. We are likely at last to understand the genetic explanation of how species have adapted to perform their shared or unique biological functions. Analysis of the sequences containing mapped SNPs onto pigeonpea genetic maps against two model legume species (<em>Lotus</em> and <em>Medicago</em>) and two crop legume species (soybean and cowpea) showed maximum similarity to soybean sequences (2480 soybean sequences). In general, markers mapped on one LG hit sequences from several chromosomes of soybean, suggesting the occurrence of chromosomal rearrangement in both the genomes. As soybean has under gone recent complete genome duplication,<sup><a reveal-id="DSS025C39" data-open="DSS025C39" class="link link-ref link-reveal xref-bibr">39</a></sup> almost all the pigeonpea sequences have shown two to three different hits with soybean genome sequences. It also proves the taxonomical classification of pigeonpea and soybean, which lie in close proximity under subfamily <em>Papilionoideae.</em><sup><a reveal-id="DSS025C41" data-open="DSS025C41" class="link link-ref link-reveal xref-bibr">41</a></sup> With the model legume species 39.8% (<em>Medicago</em>) and 34.6% (<em>Lotus</em>) sequence similarities were identified whereas in the case of cowpea, 7.1% pigeonpea sequences could provide similarity results. This may be attributed to the fact that for sequence similarity analysis with pigeonpea sequences, genome assemblies of soybean, <em>Medicago</em> and <em>Lotus</em> were used, and during the time of analysis, genome sequence information was not available for cowpea; hence, the analysis was done by comparing with the high-density linkage map developed by Muchero <em>et al</em>.<sup><a reveal-id="DSS025C23" data-open="DSS025C23" class="link link-ref link-reveal xref-bibr">23</a></sup> We discovered that synteny blocks in each of the 11 pigeonpea LGs were syntenic to their counterparts of the chromosomes of four legumes, implying certain colinearity for the syntenic chromosome/linkage pairs. The conserved sequences identified among five legumes (pigeonpea, soybean, cowpea, <em>Medicago</em> and <em>Lotus</em>) and the data from the comparative genome analysis should facilitate studies on genome evolution and analysis of the structural genome, but more importantly should facilitate the functional inference of genes in pigeonpea. The determination of gene functions is difficult in non-model species including pigeonpea; thus, functional genome analysis will have to rely heavily on the establishment of orthologies from model species.</p><p>In summary, this study provides an extensive resource of SNPs, their conversion in cost-effective KASPar assays and their application in constructing a dense genetic map in pigeonpea and in similarity analysis across five legumes. The developed genetic map is the most comprehensive genetic map for pigeonpea based on a single mapping population. Through mapped SNPs, we have identified complex syntenic relationships between soybean and pigeonpea by comparative genomics analysis. We consider that it will be possible for pigeonpea breeders to attain one of their most important goals, to rapidly and economically genotype thousands of accessions with a large and flexible number of markers. Although this is the first reported sizeable scale SNP mapping effort in pigeonpea, a larger number of informative SNPs mapped at minimum intervals will be necessary for broader applications. The extensive genomics resources developed by the whole-genome sequencing of pigeonpea, coupled to future re-sequencing multiple breeding lines, promise to considerably increase the number of informative SNPs, permitting exceptional levels of precision genetic analysis in pigeonpea breeding.</p>                            <h2 scrollto-destination=7290866 id="7290866" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data</a> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=7290868 id="7290868" class="section-title" data-legacy-id=s6>Funding</h2>
<p>The authors are thankful to CGIAR Generation Challenge Programme for SP2 commissioned project (G4008.47) and Theme Leader Discretionary Funds for support in carrying out this study.</p>                            <h2 scrollto-destination=7290870 id="7290870" class="backacknowledgements-title" >Acknowledgements</h2>
<p>Thanks are also due to Anuja Dubey, Pavana Hiremath, Sarwar Azam, A Bhanuprakash, Trushar Shah and Poornima Reddy for their help in some analysis and useful discussions while preparing the manuscript.</p>                            <h2 scrollto-destination=7290872 id="7290872" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSS025C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Graner</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sorrells</div> <div class="given-names">M.E.</div></div>. </div><div class="article-title">Genomics-assisted breeding for crop improvement</div>, <div class="source">Trends Plant Sci.</div> , <div class="year">2005</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">621</div>-<div class="lpage">30</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.tplants.2005.10.004" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.tplants.2005.10.004</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomics-assisted%20breeding%20for%20crop%20improvement&amp;author=R.K.%20Varshney&amp;author=A.%20Graner&amp;author=M.E.%20Sorrells&amp;publication_year=2005&amp;journal=Trends%20Plant%20Sci.&amp;volume=10&amp;pages=621-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2005.10.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2005.10.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2005.10.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16290213" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Hoisington</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Tyagi</div> <div class="given-names">A.K.</div></div>. </div><div class="article-title">Advances in cereal genomics and applications in crop breeding</div>, <div class="source">Trends Biotech.</div> , <div class="year">2006</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">490</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.tibtech.2006.08.006" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.tibtech.2006.08.006</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Advances%20in%20cereal%20genomics%20and%20applications%20in%20crop%20breeding&amp;author=R.K.%20Varshney&amp;author=D.A.%20Hoisington&amp;author=A.K.%20Tyagi&amp;publication_year=2006&amp;journal=Trends%20Biotech.&amp;volume=24&amp;pages=490-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibtech.2006.08.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibtech.2006.08.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibtech.2006.08.006"> </span></div> </div></div></div></div></div><div content-id="DSS025C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Thudi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">May</div> <div class="given-names">G.D.</div></div>,  <div class="name"><div class="surname">Jackson</div> <div class="given-names">S.A.</div></div>. </div><div class="article-title">Legume genomics and breeding</div>, <div class="source">Plant Breed. Rev.</div> , <div class="year">2010</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">257</div>-<div class="lpage">304</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/9780470535486.ch6" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/9780470535486.ch6</a></div></div></div></div></div><div content-id="DSS025C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Raju</div> <div class="given-names">N.L.</div></div>,  <div class="name"><div class="surname">Gnanesh</div> <div class="given-names">B.N.</div></div>,  <div class="name"><div class="surname">Lekha</div> <div class="given-names">P.T.</div></div>, et al. </div><div class="article-title">The first set of EST resource for gene discovery and marker development in pigeonpea (<em>Cajanus cajan</em> L.)</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">10</div> pg. <div class="fpage">45</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2229-10-45" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2229-10-45</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20first%20set%20of%20EST%20resource%20for%20gene%20discovery%20and%20marker%20development%20in%20pigeonpea%20(Cajanus%20cajan%20L.)&amp;author=N.L.%20Raju&amp;author=B.N.%20Gnanesh&amp;author=P.T.%20Lekha&amp;publication_year=2010&amp;journal=BMC%20Plant%20Biol.&amp;volume=10&amp;pages=45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-10-45" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-10-45" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-10-45"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20222972" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dutta</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kumawat</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Singh</div> <div class="given-names">B.P.</div></div>, et al. </div><div class="article-title">Development of genic-SSR markers by deep transcriptome sequencing in pigeonpea [<em>Cajanus cajan</em> (L.) Millspaugh]</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2011</div>, vol. <div class="volume">11</div> pg. <div class="fpage">17</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2229-11-17" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2229-11-17</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20genic-SSR%20markers%20by%20deep%20transcriptome%20sequencing%20in%20pigeonpea%20%5BCajanus%20cajan%20(L.)%20Millspaugh%5D&amp;author=S.%20Dutta&amp;author=G.%20Kumawat&amp;author=B.P.%20Singh&amp;publication_year=2011&amp;journal=BMC%20Plant%20Biol.&amp;volume=11&amp;pages=17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-11-17" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-11-17" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-11-17"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21251263" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dubey</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Farmer</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Schlueter</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Defining the transcriptome assembly and its use for genome dynamics and transcriptome profiling studies in pigeonpea (<em>Cajanus cajan</em> L.)</div>, <div class="source">DNA Res.</div> , <div class="year">2011</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">153</div>-<div class="lpage">64</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsr007" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dsr007</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Defining%20the%20transcriptome%20assembly%20and%20its%20use%20for%20genome%20dynamics%20and%20transcriptome%20profiling%20studies%20in%20pigeonpea%20(Cajanus%20cajan%20L.)&amp;author=A.%20Dubey&amp;author=A.%20Farmer&amp;author=J.%20Schlueter&amp;publication_year=2011&amp;journal=DNA%20Res.&amp;volume=18&amp;pages=153-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsr007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsr007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsr007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21565938" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kudapa</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Bharti</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Cannon</div> <div class="given-names">S.B.</div></div>, et al. </div><div class="article-title">A comprehensive transcriptome assembly of pigeonpea (<em>Cajanus cajan</em> L.) using Sanger and second-generation sequencing platforms</div>, <div class="source">Mol. Plant</div> , <div class="year">2012</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">1020</div>-<div class="lpage">1028</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comprehensive%20transcriptome%20assembly%20of%20pigeonpea%20(Cajanus%20cajan%20L.)%20using%20Sanger%20and%20second-generation%20sequencing%20platforms&amp;author=H.%20Kudapa&amp;author=A.K.%20Bharti&amp;author=S.B.%20Cannon&amp;publication_year=2012&amp;journal=Mol.%20Plant&amp;volume=5&amp;pages=1020-1028" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/mp/ssr111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmp%2Fssr111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmp%2Fssr111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22241453" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saxena</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Prathima</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Saxena</div> <div class="given-names">K.B.</div></div>,  <div class="name"><div class="surname">Hoisington</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Singh</div> <div class="given-names">N.K.</div></div>,  <div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>. </div><div class="article-title">Novel SSR markers for polymorphism detection in pigeonpea (<em>Cajanus</em> spp.)</div>, <div class="source">Plant Breed.</div> , <div class="year">2010</div>, vol. <div class="volume">129</div> (pg. <div class="fpage">142</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1439-0523.2009.01680.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1439-0523.2009.01680.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20SSR%20markers%20for%20polymorphism%20detection%20in%20pigeonpea%20(Cajanus%20spp.)&amp;author=R.K.%20Saxena&amp;author=C.%20Prathima&amp;author=K.B.%20Saxena&amp;author=D.A.%20Hoisington&amp;author=N.K.%20Singh&amp;author=R.K.%20Varshney&amp;publication_year=2010&amp;journal=Plant%20Breed.&amp;volume=129&amp;pages=142-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbr.2010.129.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbr.2010.129.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbr.2010.129.issue-2"> </span></div> </div></div></div></div></div><div content-id="DSS025C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bohra</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Dubey</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Saxena</div> <div class="given-names">R.K.</div></div>, et al. </div><div class="article-title">Analysis of BAC-end sequences (BESs) and development of BES-SSR markers for genetic mapping and hybrid purity assessment in pigeonpea (<em>Cajanus</em> spp.)</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2011</div>, vol. <div class="volume">11</div> pg. <div class="fpage">56</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2229-11-56" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2229-11-56</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20BAC-end%20sequences%20(BESs)%20and%20development%20of%20BES-SSR%20markers%20for%20genetic%20mapping%20and%20hybrid%20purity%20assessment%20in%20pigeonpea%20(Cajanus%20spp.)&amp;author=A.%20Bohra&amp;author=A.%20Dubey&amp;author=R.K.%20Saxena&amp;publication_year=2011&amp;journal=BMC%20Plant%20Biol.&amp;volume=11&amp;pages=56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-11-56" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-11-56" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-11-56"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21447154" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saxena</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Cui</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Thakur</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Walter</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Close</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>. </div><div class="article-title">Single feature polymorphisms (SFPs) for drought tolerance in pigeonpea [<em>Cajanus cajan</em> (L.) Millspaugh]</div>, <div class="source">Funct. Integr. Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">651</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s10142-011-0227-2" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s10142-011-0227-2</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single%20feature%20polymorphisms%20(SFPs)%20for%20drought%20tolerance%20in%20pigeonpea%20%5BCajanus%20cajan%20(L.)%20Millspaugh%5D&amp;author=R.K.%20Saxena&amp;author=X.%20Cui&amp;author=V.%20Thakur&amp;author=B.%20Walter&amp;author=T.J.%20Close&amp;author=R.K.%20Varshney&amp;publication_year=2011&amp;journal=Funct.%20Integr.%20Genomics&amp;volume=11&amp;pages=651-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10142-011-0227-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10142-011-0227-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10142-011-0227-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21547435" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Pang</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Ash</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">Low level of genetic diversity in cultivated pigeonpea compared to its wild relatives is revealed by diversity arrays technology</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">113</div> (pg. <div class="fpage">585</div>-<div class="lpage">95</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-006-0317-z" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-006-0317-z</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Low%20level%20of%20genetic%20diversity%20in%20cultivated%20pigeonpea%20compared%20to%20its%20wild%20relatives%20is%20revealed%20by%20diversity%20arrays%20technology&amp;author=S.%20Yang&amp;author=W.%20Pang&amp;author=G.%20Ash&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=113&amp;pages=585-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0317-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0317-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0317-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16845522" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Saxena</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Kulwal</div> <div class="given-names">P.L.</div></div>, et al. </div><div class="article-title">First genetic map of pigeonpea based on Diversity Array Technology (DArT) markers</div>, <div class="source">J. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">90</div> (pg. <div class="fpage">103</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s12041-011-0050-5" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s12041-011-0050-5</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=First%20genetic%20map%20of%20pigeonpea%20based%20on%20Diversity%20Array%20Technology%20(DArT)%20markers&amp;author=S.%20Yang&amp;author=R.K.%20Saxena&amp;author=P.L.%20Kulwal&amp;publication_year=2011&amp;journal=J.%20Genet.&amp;volume=90&amp;pages=103-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s12041-011-0050-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12041-011-0050-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs12041-011-0050-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21677394" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Draft genome sequence of pigeonpea (<em>Cajanus cajan</em>), an orphan legume crop of resource-poor farmers</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2012</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">83</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nbt.2022" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nbt.2022</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20pigeonpea%20(Cajanus%20cajan)%2C%20an%20orphan%20legume%20crop%20of%20resource-poor%20farmers&amp;author=R.K.%20Varshney&amp;author=W.%20Chen&amp;author=Y.%20Li&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol.&amp;volume=30&amp;pages=83-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2022"> </span></div> </div></div></div></div></div><div content-id="DSS025C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bohra</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Saxena</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Gnanesh</div> <div class="given-names">B.N.</div></div>, et al. </div><div class="article-title">An intra-specific consensus genetic map of pigeonpea [<em>Cajanus cajan</em> (L.) Millspaugh] derived from six mapping populations</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">125</div> (pg. <div class="fpage">1325</div>-<div class="lpage">1338</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20intra-specific%20consensus%20genetic%20map%20of%20pigeonpea%20%5BCajanus%20cajan%20(L.)%20Millspaugh%5D%20derived%20from%20six%20mapping%20populations&amp;author=A.%20Bohra&amp;author=R.K.%20Saxena&amp;author=B.N.%20Gnanesh&amp;publication_year=2012&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=125&amp;pages=1325-1338" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-012-1916-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-012-1916-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-012-1916-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22772726" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gnanesh</div> <div class="given-names">B.N.</div></div>,  <div class="name"><div class="surname">Bohra</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sharma</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Genetic mapping and quantitative trait locus analysis of resistance to sterility mosaic disease in pigeonpea [<em>Cajanus cajan</em> (L.) Millsp.]</div>, <div class="source">Field Crops Res.</div> , <div class="year">2011</div>, vol. <div class="volume">123</div> (pg. <div class="fpage">53</div>-<div class="lpage">61</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.fcr.2011.04.011" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.fcr.2011.04.011</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20mapping%20and%20quantitative%20trait%20locus%20analysis%20of%20resistance%20to%20sterility%20mosaic%20disease%20in%20pigeonpea%20%5BCajanus%20cajan%20(L.)%20Millsp.%5D&amp;author=B.N.%20Gnanesh&amp;author=A.%20Bohra&amp;author=M.%20Sharma&amp;publication_year=2011&amp;journal=Field%20Crops%20Res.&amp;volume=123&amp;pages=53-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fcr.2011.04.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fcr.2011.04.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fcr.2011.04.011"> </span></div> </div></div></div></div></div><div content-id="DSS025C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rafalski</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Applications of single nucleotide polymorphisms in crop genetics</div>, <div class="source">Curr. Opin. Plant Biol.</div> , <div class="year">2002</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">94</div>-<div class="lpage">100</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S1369-5266(02)00240-6" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S1369-5266(02)00240-6</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Applications%20of%20single%20nucleotide%20polymorphisms%20in%20crop%20genetics&amp;author=A.%20Rafalski&amp;publication_year=2002&amp;journal=Curr.%20Opin.%20Plant%20Biol.&amp;volume=5&amp;pages=94-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1369-5266(02)00240-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1369-5266(02)00240-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1369-5266(02)00240-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11856602" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mammadov</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Mingus</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Thompson</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kumpatla</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Development of versatile gene-based SNP assays in maize (<em>Zea mays</em> L.)</div>, <div class="source">Mol. Breed.</div> , <div class="year">2011</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">779</div>-<div class="lpage">790</div>)</div></div></div></div><div content-id="DSS025C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Close</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Bhat</div> <div class="given-names">P.R.</div></div>,  <div class="name"><div class="surname">Lonardi</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Development and implementation of high-throughput SNP genotyping in barley</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">582</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-10-582" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-10-582</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20implementation%20of%20high-throughput%20SNP%20genotyping%20in%20barley&amp;author=T.J.%20Close&amp;author=P.R.%20Bhat&amp;author=S.%20Lonardi&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=582" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-582" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-582" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-582"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19961604" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Akhunov</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Nicolet</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Dvorak</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Single nucleotide polymorphism genotyping in polyploidy wheat with the Illumina GoldenGate assay</div>, <div class="source">Theor. App. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">119</div> (pg. <div class="fpage">507</div>-<div class="lpage">17</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-009-1059-5" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-009-1059-5</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single%20nucleotide%20polymorphism%20genotyping%20in%20polyploidy%20wheat%20with%20the%20Illumina%20GoldenGate%20assay&amp;author=E.%20Akhunov&amp;author=C.%20Nicolet&amp;author=J.%20Dvorak&amp;publication_year=2009&amp;journal=Theor.%20App.%20Genet.&amp;volume=119&amp;pages=507-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-009-1059-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-009-1059-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-009-1059-5"> </span></div> </div></div></div></div></div><div content-id="DSS025C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yan</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Shah</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">High-throughput SNP genotyping with the GoldenGate assay in maize</div>, <div class="source">Mol. Breed.</div> , <div class="year">2009</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">441</div>-<div class="lpage">51</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11032-009-9343-2" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s11032-009-9343-2</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20SNP%20genotyping%20with%20the%20GoldenGate%20assay%20in%20maize&amp;author=J.%20Yan&amp;author=X.%20Yang&amp;author=T.%20Shah&amp;publication_year=2009&amp;journal=Mol.%20Breed.&amp;volume=25&amp;pages=441-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-009-9343-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-009-9343-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-009-9343-2"> </span></div> </div></div></div></div></div><div content-id="DSS025C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Durstewitz</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Polley</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Plieske</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Luerssen</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Graner</div> <div class="given-names">E.M.</div></div>,  <div class="name"><div class="surname">Wieseke</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Ganal</div> <div class="given-names">M.W.</div></div>. </div><div class="article-title">SNP discovery by amplicon sequencing and multiplex SNP genotyping in the allopolyploid species <em>Brassica napus</em></div>, <div class="source">Genome</div> , <div class="year">2010</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">948</div>-<div class="lpage">56</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1139/G10-079" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1139/G10-079</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20discovery%20by%20amplicon%20sequencing%20and%20multiplex%20SNP%20genotyping%20in%20the%20allopolyploid%20species%20Brassica%20napus&amp;author=G.%20Durstewitz&amp;author=A.%20Polley&amp;author=J.%20Plieske&amp;author=H.%20Luerssen&amp;author=E.M.%20Graner&amp;author=R.%20Wieseke&amp;author=M.W.%20Ganal&amp;publication_year=2010&amp;journal=Genome&amp;volume=53&amp;pages=948-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G10-079" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG10-079" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG10-079"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21076510" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hyten</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Choi</div> <div class="given-names">I.Y.</div></div>, et al. </div><div class="article-title">High-throughput genotyping with the GoldenGate assay in the complex genome of soybean</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">116</div> (pg. <div class="fpage">945</div>-<div class="lpage">52</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-008-0726-2" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-008-0726-2</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20genotyping%20with%20the%20GoldenGate%20assay%20in%20the%20complex%20genome%20of%20soybean&amp;author=D.L.%20Hyten&amp;author=Q.%20Song&amp;author=I.Y.%20Choi&amp;publication_year=2008&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=116&amp;pages=945-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-008-0726-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-008-0726-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-008-0726-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18278477" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Muchero</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Diop</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Bhat</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">A consensus genetic map of cowpea [<em>Vigna unguiculata</em> (L) Walp.] and synteny based on EST-derived SNPs</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2009</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">18159</div>-<div class="lpage">64</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.0905886106" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.0905886106</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20consensus%20genetic%20map%20of%20cowpea%20%5BVigna%20unguiculata%20(L)%20Walp.%5D%20and%20synteny%20based%20on%20EST-derived%20SNPs&amp;author=W.%20Muchero&amp;author=N.%20Diop&amp;author=P.%20Bhat&amp;publication_year=2009&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=18159-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0905886106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0905886106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0905886106"> </span></div> </div></div></div></div></div><div content-id="DSS025C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Deulvot</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Charrel</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Marty</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Highly-multiplexed SNP genotyping for genetic mapping and germplasm diversity studies in pea</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">468</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-11-468" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-11-468</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Highly-multiplexed%20SNP%20genotyping%20for%20genetic%20mapping%20and%20germplasm%20diversity%20studies%20in%20pea&amp;author=C.%20Deulvot&amp;author=H.%20Charrel&amp;author=A.%20Marty&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=468" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-468" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-468" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-468"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20701750" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ganal</div> <div class="given-names">M.W.</div></div>,  <div class="name"><div class="surname">Durstewitz</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Polley</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">A large maize (<em>Zea mays</em> L.) SNP genotyping array: development and germplasm genotyping, and genetic mapping to compare with the B73 reference genome</div>, <div class="source">PLoS One</div> , <div class="year">2011</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e28334</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0028334" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pone.0028334</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20large%20maize%20(Zea%20mays%20L.)%20SNP%20genotyping%20array%3A%20development%20and%20germplasm%20genotyping%2C%20and%20genetic%20mapping%20to%20compare%20with%20the%20B73%20reference%20genome&amp;author=M.W.%20Ganal&amp;author=G.%20Durstewitz&amp;author=A.%20Polley&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e28334" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0028334" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0028334" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0028334"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22174790" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Allen</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Barker</div> <div class="given-names">G.L.A.</div></div>,  <div class="name"><div class="surname">Berry</div> <div class="given-names">S.T.</div></div>, et al. </div><div class="article-title">Transcript-specific, single-nucleotide polymorphism discovery and linkage analysis in hexaploid bread wheat (<em>Triticum aestivum L</em>.)</div>, <div class="source">Plant Biotechnol. J.</div> , <div class="year">2011</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">1086</div>-<div class="lpage">99</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1467-7652.2011.00628.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1467-7652.2011.00628.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcript-specific%2C%20single-nucleotide%20polymorphism%20discovery%20and%20linkage%20analysis%20in%20hexaploid%20bread%20wheat%20(Triticum%20aestivum%20L.)&amp;author=A.M.%20Allen&amp;author=G.L.A.%20Barker&amp;author=S.T.%20Berry&amp;publication_year=2011&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=9&amp;pages=1086-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.2011.9.issue-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2011.9.issue-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.2011.9.issue-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21627760" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cortés</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Chavarro</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Blair</div> <div class="given-names">M.W.</div></div>. </div><div class="article-title">SNP marker diversity in common bean (<em>Phaseolus vulgaris</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">123</div> (pg. <div class="fpage">827</div>-<div class="lpage">45</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-011-1630-8" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-011-1630-8</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20marker%20diversity%20in%20common%20bean%20(Phaseolus%20vulgaris%20L.)&amp;author=A.J.%20Cort%C3%A9s&amp;author=M.C.%20Chavarro&amp;author=M.W.%20Blair&amp;publication_year=2011&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=123&amp;pages=827-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-011-1630-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-011-1630-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-011-1630-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21785951" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hiremath</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Kumar</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Penmetsa</div> <div class="given-names">R.V.</div></div>, et al. </div><div class="article-title">Large-scale development of cost-effective SNP marker assays for diversity assessment and genetic mapping in chickpea and comparative mapping in legumes</div>, <div class="source">Plant Biotechnol. J.</div> , <div class="year">2012</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">716</div>-<div class="lpage">732</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20development%20of%20cost-effective%20SNP%20marker%20assays%20for%20diversity%20assessment%20and%20genetic%20mapping%20in%20chickpea%20and%20comparative%20mapping%20in%20legumes&amp;author=P.J.%20Hiremath&amp;author=A.%20Kumar&amp;author=R.V.%20Penmetsa&amp;publication_year=2012&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=10&amp;pages=716-732" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1467-7652.2012.00710.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1467-7652.2012.00710.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1467-7652.2012.00710.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22703242" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cuc</div> <div class="given-names">L.M.</div></div>,  <div class="name"><div class="surname">Mace</div> <div class="given-names">E.S.</div></div>,  <div class="name"><div class="surname">Crouch</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">Quang</div> <div class="given-names">V.D.</div></div>,  <div class="name"><div class="surname">Long</div> <div class="given-names">T.D.</div></div>,  <div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>. </div><div class="article-title">Isolation and characterization of novel microsatellite markers and their application for diversity assessment in cultivated groundnut (<em>Arachis hypogaea</em>)</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">8</div> pg. <div class="fpage">55</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2229-8-55" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2229-8-55</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20novel%20microsatellite%20markers%20and%20their%20application%20for%20diversity%20assessment%20in%20cultivated%20groundnut%20(Arachis%20hypogaea)&amp;author=L.M.%20Cuc&amp;author=E.S.%20Mace&amp;author=J.H.%20Crouch&amp;author=V.D.%20Quang&amp;author=T.D.%20Long&amp;author=R.K.%20Varshney&amp;publication_year=2008&amp;journal=BMC%20Plant%20Biol.&amp;volume=8&amp;pages=55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-8-55" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-8-55" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-8-55"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18482440" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miller</div> <div class="given-names">N.A.</div></div>,  <div class="name"><div class="surname">Kingsmore</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Farmer</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Management of high-throughput DNA sequencing projects: Alpheus</div>, <div class="source">J. Comput. Sci. Syst. Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">132</div>-<div class="lpage">48</div>)<div class="pub-id-doi"><a href="https://doi.org/10.4172/jcsb.1000013" class="link link-pub-id-doi" target="_blank">https://doi.org/10.4172/jcsb.1000013</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Management%20of%20high-throughput%20DNA%20sequencing%20projects%3A%20Alpheus&amp;author=N.A.%20Miller&amp;author=S.F.%20Kingsmore&amp;author=A.%20Farmer&amp;publication_year=2008&amp;journal=J.%20Comput.%20Sci.%20Syst.%20Biol.&amp;volume=1&amp;pages=132-48" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20151039" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Orru</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Catillo</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Napolitano</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">De Matteis</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Scata</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Signorelli</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Moioli</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Characterization of a SNPs panel for meat traceability in six cattle breeds</div>, <div class="source">Food Control</div> , <div class="year">2009</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">856</div>-<div class="lpage">60</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.foodcont.2008.10.015" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.foodcont.2008.10.015</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20a%20SNPs%20panel%20for%20meat%20traceability%20in%20six%20cattle%20breeds&amp;author=L.%20Orru&amp;author=G.%20Catillo&amp;author=F.%20Napolitano&amp;author=G.%20De%20Matteis&amp;author=M.C.%20Scata&amp;author=F.%20Signorelli&amp;author=B.%20Moioli&amp;publication_year=2009&amp;journal=Food%20Control&amp;volume=20&amp;pages=856-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.foodcont.2008.10.015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.foodcont.2008.10.015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.foodcont.2008.10.015"> </span></div> </div></div></div></div></div><div content-id="DSS025C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Van Ooijen</div> <div class="given-names">J.W.</div></div>. </div><div class="article-title">JoinMap4. Software for the calculation of genetic linkage maps in experimental populations</div>, <div class="source">Plant Res. Int., Wageningen: The Netherlands.</div> , <div class="year">2006</div></div></div></div></div><div content-id="DSS025C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kosambi</div> <div class="given-names">D.D.</div></div>. </div><div class="article-title">The estimation of map distances from recombination values</div>, <div class="source">Ann. Eugen.</div> , <div class="year">1943</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">172</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20estimation%20of%20map%20distances%20from%20recombination%20values&amp;author=D.D.%20Kosambi&amp;publication_year=1943&amp;journal=Ann.%20Eugen.&amp;volume=12&amp;pages=172-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-1809.1943.tb02321.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-1809.1943.tb02321.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-1809.1943.tb02321.x"> </span></div> </div></div></div></div></div><div content-id="DSS025C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Voorrips</div> <div class="given-names">R.E.</div></div>. </div><div class="article-title">MapChart: software for the graphical presentation of linkage maps and QTLs</div>, <div class="source">J. Hered.</div> , <div class="year">2002</div>, vol. <div class="volume">93</div> (pg. <div class="fpage">77</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/jhered/93.1.77" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/jhered/93.1.77</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MapChart%3A%20software%20for%20the%20graphical%20presentation%20of%20linkage%20maps%20and%20QTLs&amp;author=R.E.%20Voorrips&amp;publication_year=2002&amp;journal=J.%20Hered.&amp;volume=93&amp;pages=77-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/93.1.77" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2F93.1.77" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2F93.1.77"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12011185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Beckstette</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Homann</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Giegerich</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Kurtz</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Fast index based algorithms and software for matching position specific scoring matrices</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">389</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2105-7-389" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2105-7-389</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20index%20based%20algorithms%20and%20software%20for%20matching%20position%20specific%20scoring%20matrices&amp;author=M.%20Beckstette&amp;author=R.%20Homann&amp;author=R.%20Giegerich&amp;author=S.%20Kurtz&amp;publication_year=2006&amp;journal=BMC%20Bioinformatics&amp;volume=7&amp;pages=389" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-7-389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-7-389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-7-389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16930469" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vandepoele</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Saeys</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Simillion</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Raes</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Van de Peer</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">The automatic detection of homologous regions (ADHoRe) and its application to microlinearity between Arabidopsis and rice</div>, <div class="source">Genome Res.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">1792</div>-<div class="lpage">801</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.400202" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.400202</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20automatic%20detection%20of%20homologous%20regions%20(ADHoRe)%20and%20its%20application%20to%20microlinearity%20between%20Arabidopsis%20and%20rice&amp;author=K.%20Vandepoele&amp;author=Y.%20Saeys&amp;author=C.%20Simillion&amp;author=J.%20Raes&amp;author=Y.%20Van%20de%20Peer&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=1792-801" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.400202" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.400202" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.400202"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12421767" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Krzywinski</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Schein</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Birol</div> <div class="given-names">I.</div></div>, et al. </div><div class="article-title">Circos: an information aesthetic for comparative genomics</div>, <div class="source">Genome Res.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">1639</div>-<div class="lpage">45</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.092759.109" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.092759.109</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circos%3A%20an%20information%20aesthetic%20for%20comparative%20genomics&amp;author=M.%20Krzywinski&amp;author=J.%20Schein&amp;author=I.%20Birol&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=1639-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.092759.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.092759.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.092759.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19541911" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kassa</div> <div class="given-names">M.T.</div></div>,  <div class="name"><div class="surname">Penmetsa</div> <div class="given-names">R.V.</div></div>,  <div class="name"><div class="surname">Carrasquilla-Garcia</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Genetic patterns of domestication in pigeonpea (<em>Cajanus cajan</em> (L.) Millsp.) and wild <em>Cajanus</em> relatives</div>, <div class="source">PLoS One</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e39563</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20patterns%20of%20domestication%20in%20pigeonpea%20(Cajanus%20cajan%20(L.)%20Millsp.)%20and%20wild%20Cajanus%20relatives&amp;author=M.T.%20Kassa&amp;author=R.V.%20Penmetsa&amp;author=N.%20Carrasquilla-Garcia&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e39563" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0039563" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0039563" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0039563"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22745789" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schmutz</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Cannon</div> <div class="given-names">S.B.</div></div>,  <div class="name"><div class="surname">Schlueter</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Genome sequence of the palaeopolyploid soybean</div>, <div class="source">Nature</div> , <div class="year">2010</div>, vol. <div class="volume">463</div> (pg. <div class="fpage">178</div>-<div class="lpage">83</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature08670" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature08670</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20palaeopolyploid%20soybean&amp;author=J.%20Schmutz&amp;author=S.B.%20Cannon&amp;author=J.%20Schlueter&amp;publication_year=2010&amp;journal=Nature&amp;volume=463&amp;pages=178-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08670" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08670" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08670"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20075913" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS025C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saxena</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Saxena</div> <div class="given-names">K.B.</div></div>,  <div class="name"><div class="surname">Kumar</div> <div class="given-names">R.V.</div></div>,  <div class="name"><div class="surname">Hoisington</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>. </div><div class="article-title">SSR-based diversity in elite pigeonpea genotypes for developing mapping populations to map resistance to Fusarium wilt and sterility mosaic disease</div>, <div class="source">Plant Breed.</div> , <div class="year">2010</div>, vol. <div class="volume">129</div> (pg. <div class="fpage">135</div>-<div class="lpage">41</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1439-0523.2009.01698.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1439-0523.2009.01698.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SSR-based%20diversity%20in%20elite%20pigeonpea%20genotypes%20for%20developing%20mapping%20populations%20to%20map%20resistance%20to%20Fusarium%20wilt%20and%20sterility%20mosaic%20disease&amp;author=R.K.%20Saxena&amp;author=K.B.%20Saxena&amp;author=R.V.%20Kumar&amp;author=D.A.%20Hoisington&amp;author=R.K.%20Varshney&amp;publication_year=2010&amp;journal=Plant%20Breed.&amp;volume=129&amp;pages=135-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbr.2010.129.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbr.2010.129.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbr.2010.129.issue-2"> </span></div> </div></div></div></div></div><div content-id="DSS025C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS025C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cannon</div> <div class="given-names">S.B.</div></div>,  <div class="name"><div class="surname">May</div> <div class="given-names">G.D.</div></div>,  <div class="name"><div class="surname">Jackson</div> <div class="given-names">S.A.</div></div>. </div><div class="article-title">Three sequenced legume genomes and many crop species: Rich opportunities for translational genomics</div>, <div class="source">Plant Physiol.</div> , <div class="year">2009</div>, vol. <div class="volume">151</div> (pg. <div class="fpage">970</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.109.144659" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.109.144659</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Three%20sequenced%20legume%20genomes%20and%20many%20crop%20species%3A%20Rich%20opportunities%20for%20translational%20genomics&amp;author=S.B.%20Cannon&amp;author=G.D.%20May&amp;author=S.A.%20Jackson&amp;publication_year=2009&amp;journal=Plant%20Physiol.&amp;volume=151&amp;pages=970-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.109.144659" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.144659" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.109.144659"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19759344" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Satoshi Tabata</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=480862&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss025/2/dss025-dss025supp_table6.xls?Expires=2147483647&amp;Signature=TJPPyKjVChWQ4xpJQG4EZSjMTvbPIBUmALqc~WjC6voEc6VmUjC-j6k6SAFYXRnKU6xnaoewHYsWX3Fug6eXcjCOz9JiRywXcC77bKYJMICo285uJ~iwzF713w4V9lsfPoVGcafPg7gcqYSgzakANPpPFbahCH6Vw-5-DJrGbeJ3xBEYoBJ1fZZqcgtVmiHOib090GNUHS5Ndi1v1WuI1I-z7RxjrsTnRE2gMdIbc~CgLqrXPE0p1guV08ESmepXE9gHbgt5z0aXekjxLl4wyHGY1mR-xvXnnRGQ4wVZYGBw7KwMPH5OAT42kNz2WhvKfKI7wz0m1vBpbCYAP64~Vw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-9</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss025/2/dss025-dss025supp_table5.xls?Expires=2147483647&amp;Signature=MdXT~h3lJLREqka988FmGiqbkt9V1CQhOvTUZtb~O1sorWHiLi7hBKmmH3RNglWuGZ0xSOFcqV-LEztmupV2J9O2AGRxEoantz5VL0Nlowh5VNbvUW-Mepa3n-ETGM0XmUzeYJfZG5q~WLc31LJQ0v~CBfm0XlBjHhy-azt3xRoTijti8uiwJsSzpGZPLs9GGAgK8RYar0~9AB5p~aeExc6~QHoPyQZs1TB1AjBtXHdm0P2MS6VwMqKY9tGEziYVwOSwFWQhQtVWdbOsGUiYyR1t6NffxF9HFfIJyzKcsRVJJFqQoKaF9XqvcUGyQV5ROdLnY-ZzbnprQWE5SBzd1w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss025/2/dss025-dss025supp_table4.xls?Expires=2147483647&amp;Signature=z5kmV9zkvBWKWL9HlyvdO2Ygb5Tm-kAXRZwOYiY1PTsGrIKUhGxhomE2svAd4HPuVN~IAJa~hHtqpz1Hbq5qTAvqVxwDxqLswshNRK5ghDTTEDpXkejBU0REcBC~LAUFVMky-LUF-PP1lqiv4G1UNEkdCJzvstSJPexzvxuuKLzvCA7e59JObK9P2NZs80sHoHo3MjkuSykTSWgjcsdRoFcNJFuqH5dfQikQRiqtKLhwDVoIdh~xppcrEAZT7auQjH5qWgDJXqt0IzWT-7ZXjltkmdz25Sx7w~RJvMbT1rbz638KhSPZEzsFnTj6ypPaPfLE0O7tsKSjX19rKdhERw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss025/2/dss025-dss025supp_table3.xls?Expires=2147483647&amp;Signature=cmlc-bag6CF2jw~EXeDMpXAXorudwNeoKZfPgWaCA2bNx~HyDL03qUD2Nc08Rw6xxP3ID6o1InKAlbpvpsnI3YGL3HOeMRAhWMx9KgaiDeZxeYzKRNC3zYh~ZHB2ZS0HbedtONtNeeD7To1W5D8hnFwHlYR-8zURzuNXZRSv0wMfr1qtlretkIHsOcoQA8bAbsfancw8~VDFYluSv0JSY57S5jl8xr2MF0lNhj5T3tDpRwAH7xDrF5cGks2vZdE6ohwBDSWRi6WCTfUJTdYiapgKvJ4VeVB-evOzyONIVl0cmHQHRFagJzlNnLBTLlUiiJ0Mfzf9LYIaSp3V1Nr7qA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss025/2/dss025-dss025supp_table2.xls?Expires=2147483647&amp;Signature=D9rPzL4f78eXhlkBR2FsRDcErQ40pJqF2a0sXcANXTMwk4z37D-hfE7ECIldExqbBVe7BDXVHf7XTuJc7mgaUBd7D3LupowualbeNwxssu7yA~L3wbUF8gMa49Pnt2k1sQmCGVzvUrucnAOEuWUI9f7lrcDV8ENPr-Fn0tgvcRvVdXtsVENSfG0-HSVx~vznIsBsfc0kTMxQE7wnL1~Kbm59QgZUqo9qQkPLocu8B4pQn2tgj2frIY3LUg-MsczB8K3V6vOmYB7tmrlCfrNFF4kw72aPISPB~W5UVKG4ZprVKyt5rf2xyTjZhqjsBsZ9M1Cg1QWekfdzTaMbYF1ddA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss025/2/dss025-dss025supp_table1.doc?Expires=2147483647&amp;Signature=mf94xvzt24fMAr7wJQeD5jXAR20Aj~IaKOep86~X1HYpF199FjeuR8I3ALej~uik5y-jwhQ-fr2-lcglrs1jaM3rgOXsELVJWJ~nzLNyy72c~8WljCVyCzXL98ouaK-fhuw2YXsTrw6iSss8JNP1Je3ezSMTK7hmOuHcUCFaOEL~DlASjXyZiktSLX~tKvWG7lKxqiUq1uSvEnYwnPmVa7-trDIcv60pCeiiy9JIAGGhj0u2GuCJD~rcsIJbal~tGkFU8UwwM5G2~OrdrULu9JjtSKgnRjozjlgVLU5t7rSEXKSgAlTJSrWs4-IJHh2PT7XDB-NKnmENIQEecCM7xw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss025/2/dss025-dss025supp_fig2.doc?Expires=2147483647&amp;Signature=y6TaXClz2Wgm9~hdsCLzf6SKWWX8k6LXIaK0tVTM2MiHnfvHnPPsm1CsZ1AdrnsX~upQmKAwlstPcCRhZ4xZiogenQEtG4ZRN0jdG2kL9pHaFoRLEGx24o-rKVp8byCNSRJN14cviCvpJwjh4l6LGNfxj8QZL2iUBAVJ5bwYp1FW9IuzEFXubv06yI7~x8H0aXQIOqrtmyrb5mlKXqdxQT7~UGLnZOpT~RI-O963Wx2K3UAMqQKkvHOUmgpcznFIO-U3FaIE359eJ8nQxaESHz8GJDTa3PZHt2ijnjk7QrigW7TpIjNOgvm8QzC7gkd-qwllIArwaXdZdXoajJF5-A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss025/2/dss025-dss025supp_fig1.png?Expires=2147483647&amp;Signature=bh9EJ6-F62NO-ZWqTYOsJ178xuCh3PxgIxBbN8F8Vw5E6NvUuvg1-KLICK7ajZut1z-eAjsRFn7oNCD1b0RzgY8qa1nuqwqEp4RaZ-Fy7jwtH6ihp~qXVMs~AG3iqYiXlDiQkq203G0dlz~5QUIJPy4jmdBwXTyPOBvvAvQfAWDSaoOfTqIFNnLVn6mb6NuzrWFEWyE6J7TXYzBID05LNn7UWdFUlS5LivQVqlvbSzOhgkfSPfDcTTS3Z-XTfDzVrL6cHYnTtTC6TdCGEOfZuJiWlSMEVtWHjksttGTso9b8StzV3PwEejHsjYGYwK6-VQy03a~rh7l23MWfvvIF-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss025/2/dss025-dss025supp.doc?Expires=2147483647&amp;Signature=4z4wvwbiAJbkmAXmchvOA5UJ4Pw6VrxgKkdMSdNr3POceaFUiwtOrStfR2PnDQclMpUzZ0gRdU-t7v6cUpbZx63St9ukyCd1JJ3X~tKlJZ7dnKO96D8g1p63FQo4-5AmfT2V4lEh7TEnbVAAcBVY-FipHA~r8yUVU-gKyusF0rx8ZXLIw8GqpmrICMY1m7Shd5P1t0EHe7KgFUkmPNYszqBjqfyg9HJskZJiCm-AJsTh1gZhiFc9tmvjbfpGLdDohFCUaSWAO9dumAONViKBUDspkTbLRa6Ra0cv8exgPg6eqhlym3hF794Fd7AEQcsAqpYKoyxvx7LgbClLfqy1uQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">356</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000313135200002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">42</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dss025"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="480862"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000313135200002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f19%2f6%2f449%2f480862">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000313135200002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (42)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F19%2F6%2F449%2F480862" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/480862'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Large-Scale Development of Cost-Effective Single-Nucleotide Polymorphism Marker Assays for Genetic Mapping in Pigeonpea and Comparative Mapping in Legumes" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dss025");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f19%2f6%2f449%2f480862">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

