

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/1/29/440058","@type":"ScholarlyArticle","name":"Motif mismatches in microsatellites: insights from genome-wide investigation among 20 insect species","datePublished":"2014-11-06","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2015-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu036","keywords":["microsatellite","insect genomes","single-nucleotide polymorphism","imperfect motif","simple sequence repeat"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/1/29/440058","author":[{"name":"Behura, Susanta K.","affiliation":"Eck Institute for Global Health and Department of Biological Sciences ,  University of Notre Dame ,  Notre Dame, IN 46556 ,  USA","@type":"Person"},{"name":"Severson, David W.","affiliation":"Eck Institute for Global Health and Department of Biological Sciences ,  University of Notre Dame ,  Notre Dame, IN 46556 ,  USA","@type":"Person"}],"description":"Abstract.  We present a detailed genome-wide comparative study of motif mismatches of microsatellites among 20 insect species representing five taxonomic orders","pageStart":"29","pageEnd":"38","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=2147483647&Signature=Vhf6NToQrkVldazSweShdrlrfCnUbBpTokrfu8JXGVRMaoyIjioben46GoxM3BSEoRr~te73o-dfY7QEM91wVumB~VX8hAxN9ywAZ1zRAbXXmtU23LiP8gcw~y1-D8ACKe3CZXmetX85Oj4q~Op8xej8CbOs0MGBcAElTMf~GIynbEl8oPRbKlaZUEBETFvDqCw1PlymDq~5degwzATpcv5kHJhRgAnT8PjHfQEqeZxMFeCt-B-phqwpjq5AMA0hAnSz5a5U5SMfCC3Naxv-~~yWr1GxeMNVTp3iBMlBqhdGNwFRsnBBquWLbTiF47z2gcE0wubs8CNqU4CX2qcpAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Motif mismatches in microsatellites: insights from genome-wide investigation among 20 insect species","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=2147483647&Signature=Vhf6NToQrkVldazSweShdrlrfCnUbBpTokrfu8JXGVRMaoyIjioben46GoxM3BSEoRr~te73o-dfY7QEM91wVumB~VX8hAxN9ywAZ1zRAbXXmtU23LiP8gcw~y1-D8ACKe3CZXmetX85Oj4q~Op8xej8CbOs0MGBcAElTMf~GIynbEl8oPRbKlaZUEBETFvDqCw1PlymDq~5degwzATpcv5kHJhRgAnT8PjHfQEqeZxMFeCt-B-phqwpjq5AMA0hAnSz5a5U5SMfCC3Naxv-~~yWr1GxeMNVTp3iBMlBqhdGNwFRsnBBquWLbTiF47z2gcE0wubs8CNqU4CX2qcpAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Motif mismatches in microsatellites: insights from genome-wide investigation among 20 insect species" />
    <meta property="og:description" content="Abstract.  We present a detailed genome-wide comparative study of motif mismatches of microsatellites among 20 insect species representing five taxonomic orders" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/1/29/440058" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2014-11-06" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Behura, Susanta K." /><meta name="citation_author_institution" content="Eck Institute for Global Health and Department of Biological Sciences, University of Notre Dame, Notre Dame, IN 46556, USA" /><meta name="citation_author" content="Severson, David W." /><meta name="citation_author_institution" content="Eck Institute for Global Health and Department of Biological Sciences, University of Notre Dame, Notre Dame, IN 46556, USA" /><meta name="citation_title" content="Motif mismatches in microsatellites: insights from genome-wide investigation among 20 insect species" /><meta name="citation_firstpage" content="29" /><meta name="citation_lastpage" content="38" /><meta name="citation_doi" content="10.1093/dnares/dsu036" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2015/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Hypervariability of simple sequences as a general source for polymorphic DNA markers; Nucleic Acids Res.;  citation_year=1989;  citation_volume=17;  citation_pages=6463-71; " /><meta name="citation_reference" content="citation_title=Mutation of human short tandem repeats; Hum. Mol. Genet.;  citation_year=1993;  citation_volume=2;  citation_pages=1123-8; " /><meta name="citation_reference" content="citation_title=Equilibrium distributions of microsatellite repeat length resulting from a balance between slippage events and point mutations; Proc. Natl. Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=10774-78; " /><meta name="citation_reference" content=" citation_edition=1st edition; citation_publisher=Oxford University Press, Oxford, UK; Microsatellites: evolution and applications;  citation_year=1999; " /><meta name="citation_reference" content="citation_title=The relationship between microsatellite slippage mutation rate and the number of repeat units; Mol. Biol. Evol.;  citation_year=2003;  citation_volume=20;  citation_pages=2123-31; " /><meta name="citation_reference" content="citation_title=Slipped-strand mispairing: a major mechanism for DNA sequence evolution; Mol. Biol. Evol.;  citation_year=1987;  citation_volume=4;  citation_pages=203-21; " /><meta name="citation_reference" content="citation_title=Slippage synthesis of simple sequence DNA; Nucleic Acids Res.;  citation_year=1992;  citation_volume=20;  citation_pages=211-5; " /><meta name="citation_reference" content="citation_title=Evolutionary dynamics of microsatellite DNA; Chromosoma;  citation_year=2000;  citation_volume=109;  citation_pages=365-71; " /><meta name="citation_reference" content="citation_title=Homoplasy and mutation model at microsatellite loci and their consequences for population genetics analysis; Mol. Ecol.;  citation_year=2002;  citation_volume=11;  citation_pages=1591-604; " /><meta name="citation_reference" content="citation_title=Microsatellites: simple sequences with complex evolution; Nat. Rev. Genet.;  citation_year=2004;  citation_volume=5;  citation_pages=435-45; " /><meta name="citation_reference" content="citation_title=Joint inference of microsatellite mutation models, population history and genealogies using transdimensional Markov Chain Monte Carlo; Genetics;  citation_year=2011;  citation_volume=188;  citation_pages=151-64; " /><meta name="citation_reference" content="citation_title=Stepwise mutation model and distribution of allelic frequencies in a finite population; Proc. Natl. Acad. Sci. USA;  citation_year=1978;  citation_volume=75;  citation_pages=2868-72; " /><meta name="citation_reference" content="citation_title=Mutational processes of simple-sequence repeat loci in human populations; Proc. Natl. Acad. Sci. USA;  citation_year=1994;  citation_volume=91;  citation_pages=3166-70; " /><meta name="citation_reference" content="citation_title=Microsatellites: genomic distribution, putative functions and mutational mechanisms: a review; Mol. Ecol.;  citation_year=2002;  citation_volume=11;  citation_pages=2453-65; " /><meta name="citation_reference" content="citation_title=Simple sequence repeats as advantageous mutators in evolution; Trends Genet.;  citation_year=2006;  citation_volume=22;  citation_pages=253-9; " /><meta name="citation_reference" content="citation_title=Complex evolution of a salmonid microsatellite locus and its consequences in inferring allelic divergence from size information; Mol. Biol. Evol.;  citation_year=1997;  citation_volume=14;  citation_pages=230-8; " /><meta name="citation_reference" content="citation_title=Estimating the age of alleles by use of intraallelic variability; Am. J. Hum. Genet.;  citation_year=1997;  citation_volume=60;  citation_pages=447-58; " /><meta name="citation_reference" content="citation_title=Sequence interruptions confer differential stability at microsatellite alleles in mismatch repair-deficient cells; Hum. Mol. Genet.;  citation_year=2000;  citation_volume=9;  citation_pages=2707-13; " /><meta name="citation_reference" content="citation_title=Functional analysis of intra-allelic variation at NACP-Rep1 in the alpha-synuclein gene; Hum. Genet.;  citation_year=2003;  citation_volume=113;  citation_pages=426-31; " /><meta name="citation_reference" content="citation_title=Microsatellite variation and evolution of human lactase persistence; Hum. Genet.;  citation_year=2005;  citation_volume=117;  citation_pages=329-39; " /><meta name="citation_reference" content="citation_title=Investigating the allelic evolution of an imperfect microsatellite locus in the Hawaiian mushroom Rhodocollybia laulaha; J. Hered.;  citation_year=2011;  citation_volume=102;  citation_pages=727-34; " /><meta name="citation_reference" content="citation_title=Intra-allelic mutation at human telomeres; Biochem. Soc. Trans.;  citation_year=2006;  citation_volume=34;  citation_pages=581-2; " /><meta name="citation_reference" content="citation_title=SciRoKo: a new tool for whole genome microsatellite search and investigation; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=1683-5; " /><meta name="citation_reference" content="citation_title=Generalised discriminant analysis based on distances; Aust. N. Z. J. Stat.;  citation_year=2003;  citation_volume=45;  citation_pages=301-18; " /><meta name="citation_reference" content="citation_title=A new method for non-parametric multivariate analysis of variance; Aust. Ecol.;  citation_year=2001;  citation_volume=26;  citation_pages=32-46; " /><meta name="citation_reference" content="citation_title=The Drosophila melanogaster genetic reference panel; Nature;  citation_year=2012;  citation_volume=482;  citation_pages=173-8; " /><meta name="citation_reference" content="citation_title=Analysis of microsatellite variation in Drosophila melanogaster with population-scale genome sequencing; PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e33036" /><meta name="citation_reference" content="citation_title=The WEKA data mining software: an update; SIGKDD Exploration;  citation_year=2009;  citation_volume=11;  citation_pages=1" /><meta name="citation_reference" content="citation_title=MEGA4: molecular evolutionary genetics analysis (MEGA) software version 4.0; Mol. Biol. Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=1596-9; " /><meta name="citation_reference" content="citation_title=DnaSP v5: a software for comprehensive analysis of DNA polymorphism data; Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1451-2; " /><meta name="citation_reference" content="citation_title=Molecular marker systems in insects: current trends and future avenues; Mol. Ecol.;  citation_year=2006;  citation_volume=15;  citation_pages=3087-113; " /><meta name="citation_reference" content="citation_title=Microsatellite discovery by deep sequencing of enriched genomic libraries; Biotechniques;  citation_year=2009;  citation_volume=46;  citation_pages=217-23; " /><meta name="citation_reference" content="citation_title=Microsatellite evolution inferred from human-chimpanzee genomic sequence alignments; Proc. Natl. Acad. Sci. USA;  citation_year=2002;  citation_volume=99;  citation_pages=8748-53; " /><meta name="citation_reference" content="citation_title=Evolutionary analysis of amino acid repeats across the genomes of 12 Drosophila species; Mol. Biol. Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=2598-609; " /><meta name="citation_reference" content="citation_title=Comparisons of mutation rate variation at genome-wide microsatellites: evolutionary insights from two cultivated rice and their wild relatives; BMC Evol. Biol.;  citation_year=2008;  citation_volume=8;  citation_pages=11" /><meta name="citation_reference" content="citation_title=Genome-wide comparative analysis of simple sequence coding repeats among 25 insect species; Gene;  citation_year=2012;  citation_volume=504;  citation_pages=226-32; " /><meta name="citation_reference" content="citation_title=Microsatellite markers from Ceanothus roderickii (Rhamnaceae) using next-generation sequencing technology; Am. J. Bot.;  citation_year=2012;  citation_volume=99;  citation_pages=e127-130; " /><meta name="citation_reference" content="citation_title=Genetic linkage maps for Asian and American lotus constructed using novel SSR markers derived from the genome of sequenced cultivar; BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=653" /><meta name="citation_reference" content="citation_title=Using next-generation sequencing approaches to isolate simple sequence repeat (SSR) loci in the plant sciences; Am. J. Bot.;  citation_year=2012;  citation_volume=99;  citation_pages=193-208; " /><meta name="citation_reference" content="citation_title=IMEx: Imperfect Microsatellite Extractor; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=1181-7; " /><meta name="citation_reference" content="citation_title=Rapid divergence of microsatellite abundance among species of Drosophila; Mol. Biol. Evol.;  citation_year=2003;  citation_volume=20;  citation_pages=1143-57; " /><meta name="citation_reference" content="citation_title=The relative rates of evolution of sex chromosomes and autosomes; Am. Nat.;  citation_year=1987;  citation_volume=130;  citation_pages=113-46; " /><meta name="citation_reference" content="citation_title=A test for faster X evolution in Drosophila; Mol. Biol. Evol.;  citation_year=2002;  citation_volume=19;  citation_pages=1816-9; " /><meta name="citation_reference" content="citation_title=Long microsatellite alleles in Drosophila melanogaster have a downward mutation bias and short persistence times, which cause their genome-wide underrepresentation; Genetics;  citation_year=2000;  citation_volume=155;  citation_pages=1213-20; " /><meta name="citation_reference" content="citation_title=Size-dependent mutability and microsatellite constraints; Mol. Biol. Evol.;  citation_year=1999;  citation_volume=16;  citation_pages=960-6; " /><meta name="citation_reference" content="citation_title=Drosophila virilis has long and highly polymorphic microsatellites; Mol. Biol. Evol.;  citation_year=2000;  citation_volume=17;  citation_pages=1641-11646; " /><meta name="citation_reference" content="citation_title=The direction of microsatellite mutations is dependent upon allele length; Nat. Genet.;  citation_year=2000;  citation_volume=24;  citation_pages=396-9; " /><meta name="citation_reference" content="citation_title=Simple sequence repeats: genetic modulators of brain function and behavior; Trends Neurosci.;  citation_year=2008;  citation_volume=31;  citation_pages=328-34; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/1/29/1207394/dsu036.pdf" /><meta name="description" content="Abstract.  We present a detailed genome-wide comparative study of motif mismatches of microsatellites among 20 insect species representing five taxonomic orders" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/1/29/440058" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/1/29/440058" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/1/29/440058");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsu036';
    </script>
<title>
	Motif mismatches in microsatellites: insights from genome-wide investigation among 20 insect species | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./440058" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZFgPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCZWh1cmEsIFN1c2FudGEgSy4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJFY2sgSW5zdGl0dXRlIGZvciBHbG9iYWwgSGVhbHRoIGFuZCBEZXBhcnRtZW50IG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMsIFVuaXZlcnNpdHkgb2YgTm90cmUgRGFtZSwgTm90cmUgRGFtZSwgSU4gNDY1NTYsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNldmVyc29uLCBEYXZpZCBXLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkVjayBJbnN0aXR1dGUgZm9yIEdsb2JhbCBIZWFsdGggYW5kIERlcGFydG1lbnQgb2YgQmlvbG9naWNhbCBTY2llbmNlcywgVW5pdmVyc2l0eSBvZiBOb3RyZSBEYW1lLCBOb3RyZSBEYW1lLCBJTiA0NjU1NiwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJNb3RpZiBtaXNtYXRjaGVzIGluIG1pY3Jvc2F0ZWxsaXRlczogaW5zaWdodHMgZnJvbSBnZW5vbWUtd2lkZSBpbnZlc3RpZ2F0aW9uIGFtb25nIDIwIGluc2VjdCBzcGVjaWVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN1MDM2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTUvMDIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IeXBlcnZhcmlhYmlsaXR5IG9mIHNpbXBsZSBzZXF1ZW5jZXMgYXMgYSBnZW5lcmFsIHNvdXJjZSBmb3IgcG9seW1vcnBoaWMgRE5BIG1hcmtlcnM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk4OTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTY0NjMtNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11dGF0aW9uIG9mIGh1bWFuIHNob3J0IHRhbmRlbSByZXBlYXRzOyBIdW0uIE1vbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0xMTIzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVxdWlsaWJyaXVtIGRpc3RyaWJ1dGlvbnMgb2YgbWljcm9zYXRlbGxpdGUgcmVwZWF0IGxlbmd0aCByZXN1bHRpbmcgZnJvbSBhIGJhbGFuY2UgYmV0d2VlbiBzbGlwcGFnZSBldmVudHMgYW5kIHBvaW50IG11dGF0aW9uczsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9OTU7ICBjaXRhdGlvbl9wYWdlcz0xMDc3NC03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iIGNpdGF0aW9uX2VkaXRpb249MXN0IGVkaXRpb247IGNpdGF0aW9uX3B1Ymxpc2hlcj1PeGZvcmQgVW5pdmVyc2l0eSBQcmVzcywgT3hmb3JkLCBVSzsgTWljcm9zYXRlbGxpdGVzOiBldm9sdXRpb24gYW5kIGFwcGxpY2F0aW9uczsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJlbGF0aW9uc2hpcCBiZXR3ZWVuIG1pY3Jvc2F0ZWxsaXRlIHNsaXBwYWdlIG11dGF0aW9uIHJhdGUgYW5kIHRoZSBudW1iZXIgb2YgcmVwZWF0IHVuaXRzOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MjEyMy0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2xpcHBlZC1zdHJhbmQgbWlzcGFpcmluZzogYSBtYWpvciBtZWNoYW5pc20gZm9yIEROQSBzZXF1ZW5jZSBldm9sdXRpb247IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTIwMy0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2xpcHBhZ2Ugc3ludGhlc2lzIG9mIHNpbXBsZSBzZXF1ZW5jZSBETkE7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTIxMS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgZHluYW1pY3Mgb2YgbWljcm9zYXRlbGxpdGUgRE5BOyBDaHJvbW9zb21hOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTEwOTsgIGNpdGF0aW9uX3BhZ2VzPTM2NS03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SG9tb3BsYXN5IGFuZCBtdXRhdGlvbiBtb2RlbCBhdCBtaWNyb3NhdGVsbGl0ZSBsb2NpIGFuZCB0aGVpciBjb25zZXF1ZW5jZXMgZm9yIHBvcHVsYXRpb24gZ2VuZXRpY3MgYW5hbHlzaXM7IE1vbC4gRWNvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xNTkxLTYwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9zYXRlbGxpdGVzOiBzaW1wbGUgc2VxdWVuY2VzIHdpdGggY29tcGxleCBldm9sdXRpb247IE5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTQzNS00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Sm9pbnQgaW5mZXJlbmNlIG9mIG1pY3Jvc2F0ZWxsaXRlIG11dGF0aW9uIG1vZGVscywgcG9wdWxhdGlvbiBoaXN0b3J5IGFuZCBnZW5lYWxvZ2llcyB1c2luZyB0cmFuc2RpbWVuc2lvbmFsIE1hcmtvdiBDaGFpbiBNb250ZSBDYXJsbzsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTg4OyAgY2l0YXRpb25fcGFnZXM9MTUxLTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdGVwd2lzZSBtdXRhdGlvbiBtb2RlbCBhbmQgZGlzdHJpYnV0aW9uIG9mIGFsbGVsaWMgZnJlcXVlbmNpZXMgaW4gYSBmaW5pdGUgcG9wdWxhdGlvbjsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5Nzg7ICBjaXRhdGlvbl92b2x1bWU9NzU7ICBjaXRhdGlvbl9wYWdlcz0yODY4LTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdXRhdGlvbmFsIHByb2Nlc3NlcyBvZiBzaW1wbGUtc2VxdWVuY2UgcmVwZWF0IGxvY2kgaW4gaHVtYW4gcG9wdWxhdGlvbnM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTkxOyAgY2l0YXRpb25fcGFnZXM9MzE2Ni03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9zYXRlbGxpdGVzOiBnZW5vbWljIGRpc3RyaWJ1dGlvbiwgcHV0YXRpdmUgZnVuY3Rpb25zIGFuZCBtdXRhdGlvbmFsIG1lY2hhbmlzbXM6IGEgcmV2aWV3OyBNb2wuIEVjb2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MjQ1My02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2ltcGxlIHNlcXVlbmNlIHJlcGVhdHMgYXMgYWR2YW50YWdlb3VzIG11dGF0b3JzIGluIGV2b2x1dGlvbjsgVHJlbmRzIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTI1My05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV4IGV2b2x1dGlvbiBvZiBhIHNhbG1vbmlkIG1pY3Jvc2F0ZWxsaXRlIGxvY3VzIGFuZCBpdHMgY29uc2VxdWVuY2VzIGluIGluZmVycmluZyBhbGxlbGljIGRpdmVyZ2VuY2UgZnJvbSBzaXplIGluZm9ybWF0aW9uOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MjMwLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzdGltYXRpbmcgdGhlIGFnZSBvZiBhbGxlbGVzIGJ5IHVzZSBvZiBpbnRyYWFsbGVsaWMgdmFyaWFiaWxpdHk7IEFtLiBKLiBIdW0uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT02MDsgIGNpdGF0aW9uX3BhZ2VzPTQ0Ny01ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2UgaW50ZXJydXB0aW9ucyBjb25mZXIgZGlmZmVyZW50aWFsIHN0YWJpbGl0eSBhdCBtaWNyb3NhdGVsbGl0ZSBhbGxlbGVzIGluIG1pc21hdGNoIHJlcGFpci1kZWZpY2llbnQgY2VsbHM7IEh1bS4gTW9sLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTI3MDctMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bmN0aW9uYWwgYW5hbHlzaXMgb2YgaW50cmEtYWxsZWxpYyB2YXJpYXRpb24gYXQgTkFDUC1SZXAxIGluIHRoZSBhbHBoYS1zeW51Y2xlaW4gZ2VuZTsgSHVtLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTEzOyAgY2l0YXRpb25fcGFnZXM9NDI2LTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb3NhdGVsbGl0ZSB2YXJpYXRpb24gYW5kIGV2b2x1dGlvbiBvZiBodW1hbiBsYWN0YXNlIHBlcnNpc3RlbmNlOyBIdW0uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTc7ICBjaXRhdGlvbl9wYWdlcz0zMjktMzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludmVzdGlnYXRpbmcgdGhlIGFsbGVsaWMgZXZvbHV0aW9uIG9mIGFuIGltcGVyZmVjdCBtaWNyb3NhdGVsbGl0ZSBsb2N1cyBpbiB0aGUgSGF3YWlpYW4gbXVzaHJvb20gUmhvZG9jb2xseWJpYSBsYXVsYWhhOyBKLiBIZXJlZC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTAyOyAgY2l0YXRpb25fcGFnZXM9NzI3LTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRyYS1hbGxlbGljIG11dGF0aW9uIGF0IGh1bWFuIHRlbG9tZXJlczsgQmlvY2hlbS4gU29jLiBUcmFucy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz01ODEtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2NpUm9LbzogYSBuZXcgdG9vbCBmb3Igd2hvbGUgZ2Vub21lIG1pY3Jvc2F0ZWxsaXRlIHNlYXJjaCBhbmQgaW52ZXN0aWdhdGlvbjsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xNjgzLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmVyYWxpc2VkIGRpc2NyaW1pbmFudCBhbmFseXNpcyBiYXNlZCBvbiBkaXN0YW5jZXM7IEF1c3QuIE4uIFouIEouIFN0YXQuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQ1OyAgY2l0YXRpb25fcGFnZXM9MzAxLTE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5ldyBtZXRob2QgZm9yIG5vbi1wYXJhbWV0cmljIG11bHRpdmFyaWF0ZSBhbmFseXNpcyBvZiB2YXJpYW5jZTsgQXVzdC4gRWNvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz0zMi00NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIERyb3NvcGhpbGEgbWVsYW5vZ2FzdGVyIGdlbmV0aWMgcmVmZXJlbmNlIHBhbmVsOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDgyOyAgY2l0YXRpb25fcGFnZXM9MTczLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIG1pY3Jvc2F0ZWxsaXRlIHZhcmlhdGlvbiBpbiBEcm9zb3BoaWxhIG1lbGFub2dhc3RlciB3aXRoIHBvcHVsYXRpb24tc2NhbGUgZ2Vub21lIHNlcXVlbmNpbmc7IFBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lMzMwMzYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgV0VLQSBkYXRhIG1pbmluZyBzb2Z0d2FyZTogYW4gdXBkYXRlOyBTSUdLREQgRXhwbG9yYXRpb247ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUVHQTQ6IG1vbGVjdWxhciBldm9sdXRpb25hcnkgZ2VuZXRpY3MgYW5hbHlzaXMgKE1FR0EpIHNvZnR3YXJlIHZlcnNpb24gNC4wOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MTU5Ni05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EbmFTUCB2NTogYSBzb2Z0d2FyZSBmb3IgY29tcHJlaGVuc2l2ZSBhbmFseXNpcyBvZiBETkEgcG9seW1vcnBoaXNtIGRhdGE7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTQ1MS0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgbWFya2VyIHN5c3RlbXMgaW4gaW5zZWN0czogY3VycmVudCB0cmVuZHMgYW5kIGZ1dHVyZSBhdmVudWVzOyBNb2wuIEVjb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MzA4Ny0xMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3Jvc2F0ZWxsaXRlIGRpc2NvdmVyeSBieSBkZWVwIHNlcXVlbmNpbmcgb2YgZW5yaWNoZWQgZ2Vub21pYyBsaWJyYXJpZXM7IEJpb3RlY2huaXF1ZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDY7ICBjaXRhdGlvbl9wYWdlcz0yMTctMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3Jvc2F0ZWxsaXRlIGV2b2x1dGlvbiBpbmZlcnJlZCBmcm9tIGh1bWFuLWNoaW1wYW56ZWUgZ2Vub21pYyBzZXF1ZW5jZSBhbGlnbm1lbnRzOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT05OTsgIGNpdGF0aW9uX3BhZ2VzPTg3NDgtNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbmFyeSBhbmFseXNpcyBvZiBhbWlubyBhY2lkIHJlcGVhdHMgYWNyb3NzIHRoZSBnZW5vbWVzIG9mIDEyIERyb3NvcGhpbGEgc3BlY2llczsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTI1OTgtNjA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJpc29ucyBvZiBtdXRhdGlvbiByYXRlIHZhcmlhdGlvbiBhdCBnZW5vbWUtd2lkZSBtaWNyb3NhdGVsbGl0ZXM6IGV2b2x1dGlvbmFyeSBpbnNpZ2h0cyBmcm9tIHR3byBjdWx0aXZhdGVkIHJpY2UgYW5kIHRoZWlyIHdpbGQgcmVsYXRpdmVzOyBCTUMgRXZvbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTExIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgY29tcGFyYXRpdmUgYW5hbHlzaXMgb2Ygc2ltcGxlIHNlcXVlbmNlIGNvZGluZyByZXBlYXRzIGFtb25nIDI1IGluc2VjdCBzcGVjaWVzOyBHZW5lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTUwNDsgIGNpdGF0aW9uX3BhZ2VzPTIyNi0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9zYXRlbGxpdGUgbWFya2VycyBmcm9tIENlYW5vdGh1cyByb2Rlcmlja2lpIChSaGFtbmFjZWFlKSB1c2luZyBuZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZyB0ZWNobm9sb2d5OyBBbS4gSi4gQm90LjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT05OTsgIGNpdGF0aW9uX3BhZ2VzPWUxMjctMTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGxpbmthZ2UgbWFwcyBmb3IgQXNpYW4gYW5kIEFtZXJpY2FuIGxvdHVzIGNvbnN0cnVjdGVkIHVzaW5nIG5vdmVsIFNTUiBtYXJrZXJzIGRlcml2ZWQgZnJvbSB0aGUgZ2Vub21lIG9mIHNlcXVlbmNlZCBjdWx0aXZhcjsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9NjUzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXNpbmcgbmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmcgYXBwcm9hY2hlcyB0byBpc29sYXRlIHNpbXBsZSBzZXF1ZW5jZSByZXBlYXQgKFNTUikgbG9jaSBpbiB0aGUgcGxhbnQgc2NpZW5jZXM7IEFtLiBKLiBCb3QuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTk5OyAgY2l0YXRpb25fcGFnZXM9MTkzLTIwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SU1FeDogSW1wZXJmZWN0IE1pY3Jvc2F0ZWxsaXRlIEV4dHJhY3RvcjsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xMTgxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhcGlkIGRpdmVyZ2VuY2Ugb2YgbWljcm9zYXRlbGxpdGUgYWJ1bmRhbmNlIGFtb25nIHNwZWNpZXMgb2YgRHJvc29waGlsYTsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTExNDMtNTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByZWxhdGl2ZSByYXRlcyBvZiBldm9sdXRpb24gb2Ygc2V4IGNocm9tb3NvbWVzIGFuZCBhdXRvc29tZXM7IEFtLiBOYXQuOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTEzMDsgIGNpdGF0aW9uX3BhZ2VzPTExMy00NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSB0ZXN0IGZvciBmYXN0ZXIgWCBldm9sdXRpb24gaW4gRHJvc29waGlsYTsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTE4MTYtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG9uZyBtaWNyb3NhdGVsbGl0ZSBhbGxlbGVzIGluIERyb3NvcGhpbGEgbWVsYW5vZ2FzdGVyIGhhdmUgYSBkb3dud2FyZCBtdXRhdGlvbiBiaWFzIGFuZCBzaG9ydCBwZXJzaXN0ZW5jZSB0aW1lcywgd2hpY2ggY2F1c2UgdGhlaXIgZ2Vub21lLXdpZGUgdW5kZXJyZXByZXNlbnRhdGlvbjsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTU1OyAgY2l0YXRpb25fcGFnZXM9MTIxMy0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2l6ZS1kZXBlbmRlbnQgbXV0YWJpbGl0eSBhbmQgbWljcm9zYXRlbGxpdGUgY29uc3RyYWludHM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz05NjAtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RHJvc29waGlsYSB2aXJpbGlzIGhhcyBsb25nIGFuZCBoaWdobHkgcG9seW1vcnBoaWMgbWljcm9zYXRlbGxpdGVzOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MTY0MS0xMTY0NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGRpcmVjdGlvbiBvZiBtaWNyb3NhdGVsbGl0ZSBtdXRhdGlvbnMgaXMgZGVwZW5kZW50IHVwb24gYWxsZWxlIGxlbmd0aDsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0zOTYtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2ltcGxlIHNlcXVlbmNlIHJlcGVhdHM6IGdlbmV0aWMgbW9kdWxhdG9ycyBvZiBicmFpbiBmdW5jdGlvbiBhbmQgYmVoYXZpb3I7IFRyZW5kcyBOZXVyb3NjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0zMjgtMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMi8xLzI5LzEyMDczOTQvZHN1MDM2LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBXZSBwcmVzZW50IGEgZGV0YWlsZWQgZ2Vub21lLXdpZGUgY29tcGFyYXRpdmUgc3R1ZHkgb2YgbW90aWYgbWlzbWF0Y2hlcyBvZiBtaWNyb3NhdGVsbGl0ZXMgYW1vbmcgMjAgaW5zZWN0IHNwZWNpZXMgcmVwcmVzZW50aW5nIGZpdmUgdGF4b25vbWljIG9yZGVycyIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDQwMDU4JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZJdMwtODqSYgBJAqj7L1UKhPK2W0" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f29%2f440058" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f29%2f440058" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=17206" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=2147483647&amp;Signature=Vhf6NToQrkVldazSweShdrlrfCnUbBpTokrfu8JXGVRMaoyIjioben46GoxM3BSEoRr~te73o-dfY7QEM91wVumB~VX8hAxN9ywAZ1zRAbXXmtU23LiP8gcw~y1-D8ACKe3CZXmetX85Oj4q~Op8xej8CbOs0MGBcAElTMf~GIynbEl8oPRbKlaZUEBETFvDqCw1PlymDq~5degwzATpcv5kHJhRgAnT8PjHfQEqeZxMFeCt-B-phqwpjq5AMA0hAnSz5a5U5SMfCC3Naxv-~~yWr1GxeMNVTp3iBMlBqhdGNwFRsnBBquWLbTiF47z2gcE0wubs8CNqU4CX2qcpAA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/1">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6572770">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6572770">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6572772">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6572772">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6572775">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6572775">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6572804">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6572804">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6572874">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6572874">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6572881">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6572881">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6572883">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6572883"><span class="link link-uri" href="http://dnaresearch.oxfordjournals.org/lookup/suppl/doi:10.1093/dnares/dsu036/-/DC1" target="_blank">Funding</span></a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6572885">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6572885">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6572887">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6572887">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/1/19/439863">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/22/1/39/440428">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Motif mismatches in microsatellites: insights from genome-wide investigation among 20 insect species
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Susanta K. Behura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Eck Institute for Global Health and Department of Biological Sciences</div>, <div class="insititution">University of Notre Dame</div>, <div class="addr-line">Notre Dame, IN 46556</div>, <div class="country">USA</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +1 574-904-2794. Fax. +1 574-631-7413. E-mail: <a href="mailto:sbehura@nd.edu" target="_blank">sbehura@nd.edu</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Susanta+K.+Behura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Behura S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Behura S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Susanta K. Behura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">David W. Severson</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Eck Institute for Global Health and Department of Biological Sciences</div>, <div class="insititution">University of Notre Dame</div>, <div class="addr-line">Notre Dame, IN 46556</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=David+W.+Severson">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Severson D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Severson D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">David W. Severson</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Dr Toshihiko Shiroishi</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 1, 1 February 2015, Pages 29–38, <a href='https://doi.org/10.1093/dnares/dsu036'>https://doi.org/10.1093/dnares/dsu036</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">06 November 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">16 June 2014</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">10 October 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="440058" href="/dnaresearch/article-pdf/22/1/29/1207394/dsu036.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Susanta K. Behura, David W. Severson;  Motif mismatches in microsatellites: insights from genome-wide investigation among 20 insect species, <em>DNA Research</em>, Volume 22, Issue 1, 1 February 2015, Pages 29–38, <a href='https://doi.org/10.1093/dnares/dsu036'>https://doi.org/10.1093/dnares/dsu036</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/440058?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/440058?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/440058?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/440058?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/440058?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/440058?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/440058?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Motif%20mismatches%20in%20microsatellites%3A%20insights%20from%20genome-wide%20investigation%20among%2020%20insect%20species&amp;publicationDate=2014-11-06&amp;volumeNum=22&amp;issueNum=1&amp;author=Behura%2C%20Susanta%20K.%3B%20Severson%2C%20David%20W.&amp;startPage=29&amp;endPage=38&amp;contentId=10.1093%2Fdnares%2Fdsu036&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f29%2f440058" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6572770 id="6572770" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>We present a detailed genome-wide comparative study of motif mismatches of microsatellites among 20 insect species representing five taxonomic orders. The results show that varying proportions (∼15–46%) of microsatellites identified in these species are imperfect in motif structure, and that they also vary in chromosomal distribution within genomes. It was observed that the genomic abundance of imperfect repeats is significantly associated with the length and number of motif mismatches of microsatellites. Furthermore, microsatellites with a higher number of mismatches tend to have lower abundance in the genome, suggesting that sequence heterogeneity of repeat motifs is a key determinant of genomic abundance of microsatellites. This relationship seems to be a general feature of microsatellites even in unrelated species such as yeast, roundworm, mouse and human. We provide a mechanistic explanation of the evolutionary link between motif heterogeneity and genomic abundance of microsatellites by examining the patterns of motif mismatches and allele sequences of single-nucleotide polymorphisms identified within microsatellite loci. Using Drosophila Reference Genetic Panel data, we further show that pattern of allelic variation modulates motif heterogeneity of microsatellites, and provide estimates of allele age of specific imperfect microsatellites found within protein-coding genes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="microsatellite">microsatellite</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;insect genomes&quot;">insect genomes</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;single-nucleotide polymorphism&quot;">single-nucleotide polymorphism</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;imperfect motif&quot;">imperfect motif</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;simple sequence repeat&quot;">simple sequence repeat</a></div>                            <h2 scrollto-destination=6572772 id="6572772" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Microsatellites, also known as simple sequence repeats, are tandem repeats of 1–6 bp motifs, which are ubiquitously found in eukaryotic genomes. One of the characteristic features of microsatellites is that they undergo rapid expansion or contraction in repeat number, leading to extensive length polymorphisms.<sup><a reveal-id="DSU036C1  DSU036C5" data-open="DSU036C1  DSU036C5" class="link link-ref link-reveal xref-bibr">1–5</a></sup> Replication slippage is considered as the major cause of variation of microsatellite repeats.<sup><a reveal-id="DSU036C6  DSU036C8" data-open="DSU036C6  DSU036C8" class="link link-ref link-reveal xref-bibr">6–8</a></sup> Slippage error creates a loop in one of the strands that gives rise to an insertion or a deletion in the subsequent replications, depending on the strand specificity (replicating or template strand) of loop formation. This leads to an increase or decrease of repeat length of the microsatellite. Several models have been proposed that explain the mutation processes of microsatellites.<sup><a reveal-id="DSU036C8  DSU036C11" data-open="DSU036C8  DSU036C11" class="link link-ref link-reveal xref-bibr">8–11</a></sup> While the stepwise mutation model<sup><a reveal-id="DSU036C12" data-open="DSU036C12" class="link link-ref link-reveal xref-bibr">12</a></sup> suggests that change in microsatellite loci occurs by one repeat unit at a time, study<sup><a reveal-id="DSU036C13" data-open="DSU036C13" class="link link-ref link-reveal xref-bibr">13</a></sup> also shows that differential changes in the repeat number of microsatellites are associated with distinct probability under different mutational processes and demographic histories.</p><p>It is well known that microsatellite loci are associated with an elevated rate of mutation that leads to abundant sequence polymorphisms of the repeats.<sup><a reveal-id="DSU036C14" data-open="DSU036C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSU036C15" data-open="DSU036C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Mutation in specific microsatellite loci often creates imperfect motifs,<sup><a reveal-id="DSU036C16" data-open="DSU036C16" class="link link-ref link-reveal xref-bibr">16</a></sup> and they are generally referred to as imperfect microsatellites. Studies have shown that intra-allelic variability of microsatellites is significantly associated with selective force that maintains microsatellite loci in genome.<sup><a reveal-id="DSU036C17  DSU036C21" data-open="DSU036C17  DSU036C21" class="link link-ref link-reveal xref-bibr">17–21</a></sup> In spite of the known facts that repeat variation of microsatellite loci is critical for their maintenance in the genome and in some cases may have functional consequences,<sup><a reveal-id="DSU036C18" data-open="DSU036C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSU036C22" data-open="DSU036C22" class="link link-ref link-reveal xref-bibr">22</a></sup> our understanding of motif imperfection in microsatellites is very limited.<sup><a reveal-id="DSU036C11" data-open="DSU036C11" class="link link-ref link-reveal xref-bibr">11</a></sup> In this study, we aim at understanding genome-wide patterns of motif mismatches among different insect species (<em>n</em> = 20) with a primary objective of investigating the relationship between microsatellite length and motif mismatches, and its role in genomic abundance of imperfect microsatellites across species.</p>                            <h2 scrollto-destination=6572775 id="6572775" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6572776 id="6572776" class="section-title" data-legacy-id=s2a>2.1. Genome sequences</h3>
<p>The genome sequences of insects investigated in this study include 15 species of order Diptera [12 Drosophilidae species (<em>D. melanogaster</em>, <em>D. simulans</em>, <em>D. sechellia</em>, <em>D. yakuba</em>, <em>D. erecta</em>, <em>D. ananassae</em>, <em>D. pseudoobscura</em>, <em>D. persimilis</em>, <em>D. willistoni</em>, <em>D. grimshawi</em>, <em>D. virilis</em>, and <em>D. mojavensis</em>) and 3 Culicidae species: <em>Aedes aegypti</em>, <em>Anopheles gambiae</em> and <em>Culex quinquefasciatus</em>], 2 species of Hymenoptera (<em>Nasonia vitripennis</em> and <em>Apis mellifera</em>) and 1 species each of Lepidoptera (<em>Bombyx mori</em>), Coleoptera (<em>Tribolium castaneum</em>) and Hemiptera (<em>Acyrthosiphon pisum</em>). Moreover, we also selected a few non-insect species, such as the budding yeast (<em>Saccharomyces cerevisiae</em>), the roundworm (<em>Caenorhabditis elegans</em>) and the mammalian genomes of mouse (<em>Mus musculus</em>) and human (<em>Homo sapiens</em>) for specific comparative analyses. In addition, we analysed chromosome X sequences of 29 inbred lines of <em>D. melanogaster</em> from the DGRP (Drosophila Genetic Reference Panel; <a class="link link-uri" href="http://dgrp.gnets.ncsu.edu/">http://dgrp.gnets.ncsu.edu/</a>) to investigate population-level sequence variation of imperfect microsatellites.</p><p>The genome sequences of <em>A. aegypti</em> (AaegL1), <em>A. gambiae</em> (AgamP3) and <em>C. quinquefasciatus</em> (CpipJ1) were downloaded from <a class="link link-uri" href="https://www.vectorbase.org/">https://www.vectorbase.org/</a>. The genome sequences of the 12 <em>Drosophila</em> species were obtained from FlyBase (<a class="link link-uri" href="http://www.flybase.org">www.flybase.org</a>). The assembly version of most of the 12 <em>Drosophila</em> genomes was r1.3 except <em>D. melanogaster</em> (r5.27), <em>D. pseudoobscura</em> (r2.10) and <em>D. virilis</em> (r1.2). The wasp (Nvit_1.0) and honey bee genome (Amel_2.0) sequences were downloaded from <a class="link link-uri" href="http://hymenopteragenome.org/">http://hymenopteragenome.org/</a>. The aphid (v2.0), silkworm (v2.0) and beetle genome (tacs_superscaffolds v4.0) were obtained from <a class="link link-uri" href="http://www.aphidbase.com/">http://www.aphidbase.com/</a>, <a class="link link-uri" href="http://ftp://silkdb.org/pub/current/Genome/">ftp://silkdb.org/pub/current/Genome/</a> and <a class="link link-uri" href="http://ftp://ftp.bioinformatics.ksu.edu/">ftp://ftp.bioinformatics.ksu.edu/</a>, respectively. The four non-insect genome sequences were downloaded from the UCSC genome server (<a class="link link-uri" href="http://ftp://hgdownload.cse.ucsc.edu/goldenPath/currentGenomes/">ftp://hgdownload.cse.ucsc.edu/goldenPath/currentGenomes/</a>). The assembly versions were yeast (SacCer_Apr2011), roundworm (WS220/ce10), mouse (GRCm38/mm10) and human (hg19, GRCh37). For the insect species, the names have been abbreviated as the first letter of the genus followed by three letters of the species name throughout the text and illustrations.</p>                            <h3 scrollto-destination=6572779 id="6572779" class="section-title" data-legacy-id=s2b>2.2. Identification of imperfect microsatellites</h3>
<p>The imperfect microsatellites were identified from the genome assemblies using the SciRoKo 3.4 program, employing the approach that implemented fixed penalty motif mismatch criteria.<sup><a reveal-id="DSU036C23" data-open="DSU036C23" class="link link-ref link-reveal xref-bibr">23</a></sup> The minimum score 15 and mismatch penalty 5 were used as search criteria to identify imperfect microsatellites of mono-, di-, tri-, tetra-, penta- and hexanucleotide repeats in each species. We also used slightly different parameters such as a minimum score of 15 and mismatch penalty of 3, and a minimum score of 10 and mismatch penalty of 5 to search microsatellites to investigate whether changes in parameter values affected the results of this study. In none of the searches, more than three consecutive mismatches per locus were allowed to identify the imperfect microsatellites.</p>                            <h3 scrollto-destination=6572783 id="6572783" class="section-title" data-legacy-id=s2c>2.3. Motif mismatches and microsatellite length</h3>
<p>The number of imperfect microsatellites along with their length and number of mismatches of individual locus, as predicted by SciRoKo, were used for different comparative analyses among the species. The genomic abundance of imperfect microsatellites with different length and motif mismatches was chosen for different comparative analyses. First, we targeted microsatellites of length &lt;30 bp for comparison between perfect (no mismatch) and imperfect repeats (each containing exactly one mismatch) in each genome. Here, microsatellites of length 20, 21, 22, 23 and 24 bp were identified in genome-wide manner in each species. Note that microsatellites of length &lt;20 bp are not appropriate for this analysis as they are always perfect under our search conditions. Similarly, microsatellites above 25 bp in length may have two mismatches as per our search conditions. Second, we targeted longer microsatellites (35, 40, 45, 50, 55, 60, 65, 70, 75, 80 and 85 bp) to compare repeats that have different number (1, 2, 4 or 6) of mismatches. The third comparison was made between microsatellites that were at least 30 bp long and had &lt;3 or ≥3 mismatches (note that microsatellites &lt;30 bp canno't have more than three mismatches as per our search conditions). Selected non-insect species, such as the budding yeast (<em>S. cerevisiae</em>), the roundworm (<em>C. elegans</em>) and the mammalian genomes of mouse (<em>M. musculus</em>) and human (<em>H. sapiens</em>), were also analysed in genome-wide manner in a similar manner.</p>                            <h3 scrollto-destination=6572787 id="6572787" class="section-title" data-legacy-id=s2d>2.4. Generalized discriminant analysis between perfect and imperfect loci across species</h3>
<p>The generalized discriminant analysis was performed using the principal coordinates method.<sup><a reveal-id="DSU036C24" data-open="DSU036C24" class="link link-ref link-reveal xref-bibr">24</a></sup> The variation of frequency of perfect microsatellites and microsatellites that have same number of mismatches across the 20 species were decomposed into the principal coordinates. Then they were tested by permutation to determine if the variation was significant between perfect and imperfect repeats. First, distance matrices were generated separately from the frequency data of both groups of loci across species. They were then decomposed into component eigenvalues by the principal coordinate analysis. The canonical axes scores (position of multivariate points on the canonical axes) were then used to determine correlations between variables, and the statistical significance of the correlation was inferred from permutation procedure implemented in the program named ‘canonical analysis of principal coordinates’ (available at <a class="link link-uri" href="https://www.stat.auckland.ac.nz/~mja/Programs.htm">https://www.stat.auckland.ac.nz/~mja/Programs.htm</a>).</p>                            <h3 scrollto-destination=6572791 id="6572791" class="section-title" data-legacy-id=s2e>2.5. Permutational analysis of variance and pair-wise a posteriori comparison</h3>
<p>The permutational analysis of variance (PERMANOVA)<sup><a reveal-id="DSU036C25" data-open="DSU036C25" class="link link-ref link-reveal xref-bibr">25</a></sup> was used to determine association between the number of mismatches and the length of microsatellites among the 20 species. The Euclidean distance was used as the measure of data variation. The pair-wise <em>a posteriori</em> comparisons were performed between different lengths of microsatellites (35, 40, 45, 50, 55, 60, 65, 70, 75, 80 and 85 bp containing 1, 2, 4 or 6 mismatches) among the 20 insect species as well as among the 4 non-insect species using PERMANOVA <em>t</em>-statistic. No data transformation or standardization was done prior to analysis. Unrestricted permutation of the data was allowed to perform the permutation tests (9,999 times) to determine the statistical significance of the analysis. The <em>P</em>-value of &lt;0.05 was considered as the threshold of significance, unless stated otherwise.</p>                            <h3 scrollto-destination=6572795 id="6572795" class="section-title" data-legacy-id=s2f>2.6. Identification and analysis of single-nucleotide polymorphisms localized within imperfect microsatellites</h3>
<p>We identified single-nucleotide polymorphisms (SNPs) that are localized within microsatellite loci in <em>D. melanogaster</em> and <em>A. gambiae</em> from the NCBI SNP database (<a class="link link-uri" href="http://ftp://ftp.ncbi.nlm.nih.gov/snp/">ftp://ftp.ncbi.nlm.nih.gov/snp/</a>). The microsatellites were partitioned to either as M<sub>1</sub> types if the SNP alleles matched to the imperfect motif sequences or as M<sub>0</sub> types if SNP alleles did not match to the motif sequences. To determine the significance of association of M<sub>0</sub>/M<sub>1</sub> allelic patterns with variation of motif mismatches of imperfect microsatellites, microsatellites containing either &lt;3 mismatches or ≥3 mismatches were determined in each species with repeat length being controlled (≥30 bp) for each locus. The count data of these four groups of imperfect microsatellites were used in 2 × 2 contingency tests (Yates' chi-square) to determine statistical significance of association between allelic variation and motif imperfection of microsatellites.</p>                            <h3 scrollto-destination=6572799 id="6572799" class="section-title" data-legacy-id=s2g>2.7. Analysis of imperfect microsatellites from DGRP data</h3>
<p>To further investigate the role of mutations in motif mismatches and to estimate the allele age of imperfect microsatellite loci, we analysed genome sequences of <em>D. melanogaster</em> inbred lines from the DGRP data.<sup><a reveal-id="DSU036C26" data-open="DSU036C26" class="link link-ref link-reveal xref-bibr">26</a></sup> We restricted this analysis only to microsatellite sequences of chromosome X among 29 randomly selected inbred lines. A comprehensive analysis of the microsatellites in the entire DGRP was not an aim of the present study because such a study has been performed earlier.<sup><a reveal-id="DSU036C27" data-open="DSU036C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Mutations in the microsatellites among the inbred lines were identified by sequence alignment with the reference genome. The imperfect microsatellite data (from DGRP lines) were used as a training dataset to classify the data of imperfect microsatellites generated from the dbSNP by linear regression using <em>Weka Explorer</em>.<sup><a reveal-id="DSU036C28" data-open="DSU036C28" class="link link-ref link-reveal xref-bibr">28</a></sup></p><p>The phylogenetic analyses were conducted using <em>D. melanogaster</em> imperfect microsatellite loci from different inbred lines. The sequence of each locus was extended up to 1 kb of both ends of microsatellite and the extended sequences were used to calculate evolutionary distances among the inbred lines using the maximum composite likelihood implemented in MEGA4.<sup><a reveal-id="DSU036C29" data-open="DSU036C29" class="link link-ref link-reveal xref-bibr">29</a></sup> The neighbor-joining method implemented in MEGA4 was used for constructing phylogenetic trees. The total number of mutations, number of polymorphic sites, average number of nucleotide differences and the number of fixed mutations between inbred lines were calculated by using DnaSP v 5.10.<sup><a reveal-id="DSU036C30" data-open="DSU036C30" class="link link-ref link-reveal xref-bibr">30</a></sup> The allele age of microsatellite SNPs was calculated based on allele frequency and polymorphisms among the inbred lines in the extended sequences of microsatellites by the coalescence method.<sup><a reveal-id="DSU036C17" data-open="DSU036C17" class="link link-ref link-reveal xref-bibr">17</a></sup> The coalescence times were predicted by using a maximum likelihood approach implemented in the algorithm ‘bdmc21’ (available at <a class="link link-uri" href="http://www.rannala.org/">http://www.rannala.org/</a>).</p>                            <h2 scrollto-destination=6572804 id="6572804" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=6572806 id="6572806" class="section-title" data-legacy-id=s3a>3.1. Frequency of imperfect microsatellites in insect genomes</h3>
<p>The number of imperfect microsatellites varies among the insect species as given in Table <a reveal-id="DSU036TB1" data-open="DSU036TB1" class="link link-reveal link-table xref-fig">1</a>. The imperfect repeats account to ∼15–46% of all simple sequence repeats identified from the genome sequences of the 20 insect species. The imperfect microsatellites represented &lt;1% of the genome size of most of these insects except few <em>Drosophila</em> species (Fig. <a reveal-id="DSU036F1" data-open="DSU036F1" class="link link-reveal link-table xref-fig">1</a>). The data in Table <a reveal-id="DSU036TB1" data-open="DSU036TB1" class="link link-reveal link-table xref-fig">1</a> and Fig. <a reveal-id="DSU036F1" data-open="DSU036F1" class="link link-reveal link-table xref-fig">1</a> show that species even within the Drosophila genus have extensive variation in the frequency of imperfect microsatellites. It was observed that specific <em>Drosophila</em> species (such as <em>D. mojavensis</em>, <em>D. virilis</em> and <em>D. grimshawi</em>) have a higher amount of imperfect microsatellites than other insects. Moreover, our data show that the imperfect microsatellites are differentially distributed among different chromosomes within the genome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). This was evident in insects where chromosomes have been annotated from genome assemblies. It was observed that chromosome X harbours the highest proportion of imperfect microsatellites in <em>D. melanogaster</em>, a pattern that was also found in <em>D. simulans</em> and <em>D. yakuba</em>. However, this pattern is missing in non-<em>Drosophila</em> insects such as <em>A. gambiae</em> and <em>T. castaneum</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Furthermore, the proportion of imperfect microsatellites varies differentially among species depending on the motif length of microsatellites (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). The di- and trinucleotide repeats are associated with an elevated rate of motif mismatches compared with all other repeat sizes. Moreover, this pattern seems to be a conserved feature across the insect genomes. </p>                        <a id="6572810" scrollto-destination="6572810"></a>
<div content-id="DSU036TB1" class="table-modal"><div class="table-wrap"><div id="DSU036TB1" class="table-wrap-title" data-id="DSU036TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Number of imperfect microsatellites in different insect genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>Imperfect microsatellites </th><th>% of all microsatellites </th></tr></thead><tbody><tr><td>Aaeg </td><td>22,620 </td><td>15.5 </td></tr><tr><td>Agam </td><td>30,207 </td><td>28.1 </td></tr><tr><td>Apis </td><td>44,563 </td><td>24.1 </td></tr><tr><td>Cqui </td><td>20,454 </td><td>19.5 </td></tr><tr><td>Dana </td><td>18,453 </td><td>31.4 </td></tr><tr><td>Dere </td><td>14,315 </td><td>36.6 </td></tr><tr><td>Dgri </td><td>74,637 </td><td>46.1 </td></tr><tr><td>Dmel </td><td>17,392 </td><td>31.6 </td></tr><tr><td>Dmoj </td><td>85,525 </td><td>42.3 </td></tr><tr><td>Dper </td><td>40,641 </td><td>34.6 </td></tr><tr><td>Dpse </td><td>36,289 </td><td>33.6 </td></tr><tr><td>Dsec </td><td>11,520 </td><td>28.2 </td></tr><tr><td>Dsim </td><td>10,995 </td><td>30.0 </td></tr><tr><td>Dvir </td><td>62,282 </td><td>42.8 </td></tr><tr><td>Dwil </td><td>55,121 </td><td>35.6 </td></tr><tr><td>Dyak </td><td>17,449 </td><td>35.4 </td></tr><tr><td>Amel </td><td>51,702 </td><td>30.7 </td></tr><tr><td>Nvit </td><td>28,074 </td><td>23.4 </td></tr><tr><td>Bmor </td><td>19,732 </td><td>18.7 </td></tr><tr><td>Tcas </td><td>5,155 </td><td>28.8 </td></tr></tbody></table></div><div content-id="DSU036TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>Imperfect microsatellites </th><th>% of all microsatellites </th></tr></thead><tbody><tr><td>Aaeg </td><td>22,620 </td><td>15.5 </td></tr><tr><td>Agam </td><td>30,207 </td><td>28.1 </td></tr><tr><td>Apis </td><td>44,563 </td><td>24.1 </td></tr><tr><td>Cqui </td><td>20,454 </td><td>19.5 </td></tr><tr><td>Dana </td><td>18,453 </td><td>31.4 </td></tr><tr><td>Dere </td><td>14,315 </td><td>36.6 </td></tr><tr><td>Dgri </td><td>74,637 </td><td>46.1 </td></tr><tr><td>Dmel </td><td>17,392 </td><td>31.6 </td></tr><tr><td>Dmoj </td><td>85,525 </td><td>42.3 </td></tr><tr><td>Dper </td><td>40,641 </td><td>34.6 </td></tr><tr><td>Dpse </td><td>36,289 </td><td>33.6 </td></tr><tr><td>Dsec </td><td>11,520 </td><td>28.2 </td></tr><tr><td>Dsim </td><td>10,995 </td><td>30.0 </td></tr><tr><td>Dvir </td><td>62,282 </td><td>42.8 </td></tr><tr><td>Dwil </td><td>55,121 </td><td>35.6 </td></tr><tr><td>Dyak </td><td>17,449 </td><td>35.4 </td></tr><tr><td>Amel </td><td>51,702 </td><td>30.7 </td></tr><tr><td>Nvit </td><td>28,074 </td><td>23.4 </td></tr><tr><td>Bmor </td><td>19,732 </td><td>18.7 </td></tr><tr><td>Tcas </td><td>5,155 </td><td>28.8 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p>The insect names are abbreviated with four letters; first letter represents the genus name and last three letters represent the species name. The percentage of microsatellites that are imperfect in the corresponding genome is also shown.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6572810" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU036TB1" class="table-wrap-title" data-id="DSU036TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Number of imperfect microsatellites in different insect genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>Imperfect microsatellites </th><th>% of all microsatellites </th></tr></thead><tbody><tr><td>Aaeg </td><td>22,620 </td><td>15.5 </td></tr><tr><td>Agam </td><td>30,207 </td><td>28.1 </td></tr><tr><td>Apis </td><td>44,563 </td><td>24.1 </td></tr><tr><td>Cqui </td><td>20,454 </td><td>19.5 </td></tr><tr><td>Dana </td><td>18,453 </td><td>31.4 </td></tr><tr><td>Dere </td><td>14,315 </td><td>36.6 </td></tr><tr><td>Dgri </td><td>74,637 </td><td>46.1 </td></tr><tr><td>Dmel </td><td>17,392 </td><td>31.6 </td></tr><tr><td>Dmoj </td><td>85,525 </td><td>42.3 </td></tr><tr><td>Dper </td><td>40,641 </td><td>34.6 </td></tr><tr><td>Dpse </td><td>36,289 </td><td>33.6 </td></tr><tr><td>Dsec </td><td>11,520 </td><td>28.2 </td></tr><tr><td>Dsim </td><td>10,995 </td><td>30.0 </td></tr><tr><td>Dvir </td><td>62,282 </td><td>42.8 </td></tr><tr><td>Dwil </td><td>55,121 </td><td>35.6 </td></tr><tr><td>Dyak </td><td>17,449 </td><td>35.4 </td></tr><tr><td>Amel </td><td>51,702 </td><td>30.7 </td></tr><tr><td>Nvit </td><td>28,074 </td><td>23.4 </td></tr><tr><td>Bmor </td><td>19,732 </td><td>18.7 </td></tr><tr><td>Tcas </td><td>5,155 </td><td>28.8 </td></tr></tbody></table></div><div content-id="DSU036TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>Imperfect microsatellites </th><th>% of all microsatellites </th></tr></thead><tbody><tr><td>Aaeg </td><td>22,620 </td><td>15.5 </td></tr><tr><td>Agam </td><td>30,207 </td><td>28.1 </td></tr><tr><td>Apis </td><td>44,563 </td><td>24.1 </td></tr><tr><td>Cqui </td><td>20,454 </td><td>19.5 </td></tr><tr><td>Dana </td><td>18,453 </td><td>31.4 </td></tr><tr><td>Dere </td><td>14,315 </td><td>36.6 </td></tr><tr><td>Dgri </td><td>74,637 </td><td>46.1 </td></tr><tr><td>Dmel </td><td>17,392 </td><td>31.6 </td></tr><tr><td>Dmoj </td><td>85,525 </td><td>42.3 </td></tr><tr><td>Dper </td><td>40,641 </td><td>34.6 </td></tr><tr><td>Dpse </td><td>36,289 </td><td>33.6 </td></tr><tr><td>Dsec </td><td>11,520 </td><td>28.2 </td></tr><tr><td>Dsim </td><td>10,995 </td><td>30.0 </td></tr><tr><td>Dvir </td><td>62,282 </td><td>42.8 </td></tr><tr><td>Dwil </td><td>55,121 </td><td>35.6 </td></tr><tr><td>Dyak </td><td>17,449 </td><td>35.4 </td></tr><tr><td>Amel </td><td>51,702 </td><td>30.7 </td></tr><tr><td>Nvit </td><td>28,074 </td><td>23.4 </td></tr><tr><td>Bmor </td><td>19,732 </td><td>18.7 </td></tr><tr><td>Tcas </td><td>5,155 </td><td>28.8 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p>The insect names are abbreviated with four letters; first letter represents the genus name and last three letters represent the species name. The percentage of microsatellites that are imperfect in the corresponding genome is also shown.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6572810" target="_blank">View Large</a></div></div>                        <a id="6572812" scrollto-destination="6572812"></a>
<div id="DSU036F1" data-id="DSU036F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/m_dsu03601.png?Expires=2147483647&Signature=SXXg2Ne0HXBywkaY0gpKhi1tIbXz3sUnrWNUtG5dopa4R0aaAJK9izUlJFLhUVoXmhqW9E52~anPHky8-Vbml6NlT2Ifnql0M1tspwsiMFjGBfH0xHsjxhjB7bg5IrcbNbbHiDNjggrnsZIKzxPsJsaUhbBWtJDTYAgxwIN5vuOsjwRtCF4L1rzyotRDYUUAFSTP69Y~soQFsi1uMldxlreBgcsH0pdGFWeY4EuULIr8R~NMt6lZSLf9ePMpFhkNYgfRKv0hd6-NvdAVgVuTWd2Yd9WvRIFAyI6f97RwpSHIPYdn8cgmdlxh9RKz44~b9baE5ylWCqFNRGWtjKmvOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Imperfect microsatellites as the percentage of genome size of insects. The insect names (abbreviated) are shown in the x-axis and the percentages are shown in the y-axis." path-from-xml="dsu03601.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6572812/dsu03601.tif" path-from-xml="dsu03601.tif" target="_blank">View large</a><a section="6572812" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/dsu03601.png?Expires=2147483647&Signature=yrYTFw2wThR~MfLz55~l20-9gf0996Wrd4jKN1kIcJzW3uz3r6hqk~PUbB5PPwyXTX5QN8tV-6jsJSYX4vGoVZdpu696aGqNGK80UWE-kg5YCD2KvzUG67JHLGOnsY38TczuwxHzPgpbuG9Uc6QJPY5IN04qb3Tivf2CSaCui2TQqOlxiq5FHl27RlS6fiZ86SMLWtnu9-bPFk11mo3AS4EY~4dDIvDp5sI3l-LRkpm~xt9oPfYgoCW4YFcMPFvRdK3LY6hcuk2jKBAQ8T4jfjBza4h3QH4g-8-Kxv6Ri3cuq2UTaJFbCgeijfFeN7e6GGjUD13FPQugAsTAraBKYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6572812&ar=440058&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03601.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Imperfect microsatellites as the percentage of genome size of insects. The insect names (abbreviated) are shown in the <em>x</em>-axis and the percentages are shown in the <em>y</em>-axis.</p></div></div><div content-id="DSU036F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/m_dsu03601.png?Expires=2147483647&Signature=SXXg2Ne0HXBywkaY0gpKhi1tIbXz3sUnrWNUtG5dopa4R0aaAJK9izUlJFLhUVoXmhqW9E52~anPHky8-Vbml6NlT2Ifnql0M1tspwsiMFjGBfH0xHsjxhjB7bg5IrcbNbbHiDNjggrnsZIKzxPsJsaUhbBWtJDTYAgxwIN5vuOsjwRtCF4L1rzyotRDYUUAFSTP69Y~soQFsi1uMldxlreBgcsH0pdGFWeY4EuULIr8R~NMt6lZSLf9ePMpFhkNYgfRKv0hd6-NvdAVgVuTWd2Yd9WvRIFAyI6f97RwpSHIPYdn8cgmdlxh9RKz44~b9baE5ylWCqFNRGWtjKmvOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Imperfect microsatellites as the percentage of genome size of insects. The insect names (abbreviated) are shown in the x-axis and the percentages are shown in the y-axis." path-from-xml="dsu03601.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6572812/dsu03601.tif" path-from-xml="dsu03601.tif" target="_blank">View large</a><a section="6572812" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/dsu03601.png?Expires=2147483647&Signature=yrYTFw2wThR~MfLz55~l20-9gf0996Wrd4jKN1kIcJzW3uz3r6hqk~PUbB5PPwyXTX5QN8tV-6jsJSYX4vGoVZdpu696aGqNGK80UWE-kg5YCD2KvzUG67JHLGOnsY38TczuwxHzPgpbuG9Uc6QJPY5IN04qb3Tivf2CSaCui2TQqOlxiq5FHl27RlS6fiZ86SMLWtnu9-bPFk11mo3AS4EY~4dDIvDp5sI3l-LRkpm~xt9oPfYgoCW4YFcMPFvRdK3LY6hcuk2jKBAQ8T4jfjBza4h3QH4g-8-Kxv6Ri3cuq2UTaJFbCgeijfFeN7e6GGjUD13FPQugAsTAraBKYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6572812&ar=440058&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03601.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Imperfect microsatellites as the percentage of genome size of insects. The insect names (abbreviated) are shown in the <em>x</em>-axis and the percentages are shown in the <em>y</em>-axis.</p></div></div>                            <h3 scrollto-destination=6572818 id="6572818" class="section-title" data-legacy-id=s3b>3.2. Motif mismatches and microsatellite length</h3>
<p>The mean length of imperfect microsatellites is higher than that of perfect microsatellites in each species (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). To understand the relationship between microsatellite length and motif imperfection, we identified microsatellites of different lengths (20, 21, 22, 23 and 24 bp) that either lack mismatch (perfect motifs) or have exactly one mismatch in each locus. The generalized discriminant analysis<sup><a reveal-id="DSU036C24" data-open="DSU036C24" class="link link-ref link-reveal xref-bibr">24</a></sup> was conducted to determine the significance of variation between perfect and imperfect microsatellites across the genomes. Results of this analysis showed a significant relationship (first squared canonical correlation = 0.9024, <em>P</em> = 0.0001) between perfect and imperfect microsatellites among the species, wherein the imperfect microsatellites of different lengths revealed lower canonical variation than that of the perfect microsatellites in each species (Fig. <a reveal-id="DSU036F2" data-open="DSU036F2" class="link link-reveal link-table xref-fig">2</a>). </p>                        <a id="6572821" scrollto-destination="6572821"></a>
<div id="DSU036F2" data-id="DSU036F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/m_dsu03602.png?Expires=2147483647&Signature=uvDQRS8JXu-H0RII3i4HcJOTIxGOBTohtDjhaKZqKYGFT-Ecj8tHp8APixb5NeFk3-3rDFtAvKtJjwxuowSQ8~IjcDcIXSBMRhvBJRHHCis6KxvKTEx5Rynky5JUB5lzzydKcdBrMefZz3zgCShL9jT5y36-CsNGVnsf2hyyQVQ58M6E~ek-T8Lx~y-UTBQDSLs3wEqbdHOn3LOsmNnldiHt4sL0EDpHA0SXBPnVFpgTuh6~h09UqQtg~6HB61F0lI7yBOGOQFW6wrwluuCMnh5me2qpD7hKnbmH5C1hrqKxwqewaXSP18m1InZPp7HwoHemtySvtYfDdVhWBp-C~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Stacked bar graphs showing variation in the number of perfect and imperfect repeats among the 20 insect species. The stacks in the graph represent different lengths of microsatellites as shown below the graphs. The scatter graph on the right shows the canonical variation of frequency of microsatellites of different length for each species. The species are indicated as 1 through 20 on x-axis and the canonical variation is shown in y-axis." path-from-xml="dsu03602.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6572821/dsu03602.tif" path-from-xml="dsu03602.tif" target="_blank">View large</a><a section="6572821" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/dsu03602.png?Expires=2147483647&Signature=pura6Q3G5xRHiUuMeFuYWjoOfGxzjCqFkq5ul18NzjtZDIvnPbhS8UiD8I3-ppZPW3ImzYpZCMxNbZ1806hW-rG6KQlm-V2iaoMleVKa28cJGT8lDEi8gcSkQXVXxvRTMLO6456Ey6bfy4iC~Nvl73v1HuWBQgQMCA0WH7abjBSIJtjDZZbDVFlrQ6o25ADgKs6QPASzpASug18~ctlQY2HhrXRYakxIXP-X3LGsQUZGr2WHXTnY9wJ-RZpiQjbiiAkJBGxMZbORtZ3ghIVj6z-hFZrTnLaLcmIDgFF0jVj0Vw35vqdxzTOZG-exm5jkC60vdJfMeR4F5TGvJ20LVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6572821&ar=440058&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03602.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Stacked bar graphs showing variation in the number of perfect and imperfect repeats among the 20 insect species. The stacks in the graph represent different lengths of microsatellites as shown below the graphs. The scatter graph on the right shows the canonical variation of frequency of microsatellites of different length for each species. The species are indicated as 1 through 20 on <em>x</em>-axis and the canonical variation is shown in <em>y</em>-axis.</p></div></div><div content-id="DSU036F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/m_dsu03602.png?Expires=2147483647&Signature=uvDQRS8JXu-H0RII3i4HcJOTIxGOBTohtDjhaKZqKYGFT-Ecj8tHp8APixb5NeFk3-3rDFtAvKtJjwxuowSQ8~IjcDcIXSBMRhvBJRHHCis6KxvKTEx5Rynky5JUB5lzzydKcdBrMefZz3zgCShL9jT5y36-CsNGVnsf2hyyQVQ58M6E~ek-T8Lx~y-UTBQDSLs3wEqbdHOn3LOsmNnldiHt4sL0EDpHA0SXBPnVFpgTuh6~h09UqQtg~6HB61F0lI7yBOGOQFW6wrwluuCMnh5me2qpD7hKnbmH5C1hrqKxwqewaXSP18m1InZPp7HwoHemtySvtYfDdVhWBp-C~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Stacked bar graphs showing variation in the number of perfect and imperfect repeats among the 20 insect species. The stacks in the graph represent different lengths of microsatellites as shown below the graphs. The scatter graph on the right shows the canonical variation of frequency of microsatellites of different length for each species. The species are indicated as 1 through 20 on x-axis and the canonical variation is shown in y-axis." path-from-xml="dsu03602.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6572821/dsu03602.tif" path-from-xml="dsu03602.tif" target="_blank">View large</a><a section="6572821" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/dsu03602.png?Expires=2147483647&Signature=pura6Q3G5xRHiUuMeFuYWjoOfGxzjCqFkq5ul18NzjtZDIvnPbhS8UiD8I3-ppZPW3ImzYpZCMxNbZ1806hW-rG6KQlm-V2iaoMleVKa28cJGT8lDEi8gcSkQXVXxvRTMLO6456Ey6bfy4iC~Nvl73v1HuWBQgQMCA0WH7abjBSIJtjDZZbDVFlrQ6o25ADgKs6QPASzpASug18~ctlQY2HhrXRYakxIXP-X3LGsQUZGr2WHXTnY9wJ-RZpiQjbiiAkJBGxMZbORtZ3ghIVj6z-hFZrTnLaLcmIDgFF0jVj0Vw35vqdxzTOZG-exm5jkC60vdJfMeR4F5TGvJ20LVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6572821&ar=440058&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03602.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Stacked bar graphs showing variation in the number of perfect and imperfect repeats among the 20 insect species. The stacks in the graph represent different lengths of microsatellites as shown below the graphs. The scatter graph on the right shows the canonical variation of frequency of microsatellites of different length for each species. The species are indicated as 1 through 20 on <em>x</em>-axis and the canonical variation is shown in <em>y</em>-axis.</p></div></div><p>Then to know if the extent of motif mismatches is related to microsatellite length, we determined the frequency of microsatellites of different lengths (35, 40, 45, 50, 55, 60, 65, 70, 75, 80 and 85 bp) harbouring different numbers of mismatches (1, 2, 4 and 6) in each of 20 species (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). The length and mismatch criteria were selected based on the search parameters as described above. The dataset was then used to perform permutational multifactorial analysis of variance of mismatch numbers and length of microsatellites. Results of this analysis showed that both the factors (length and mismatches) have significant association with the variation of genomic frequency of microsatellites among the species (<em>F</em> = 2.6, <em>P</em> = 0.02 for length and <em>F</em> = 45.02, <em>P</em> = 0.001 for mismatch). This suggested that genomic abundance of imperfect microsatellites may have an evolutionary link with the extent of sequence heterogeneity of repeat motifs across species. Moreover, it was also observed that genomic frequency of the microsatellites having only one mismatch correlates better with that of loci harbouring two mismatches than loci having four or six mismatches (Table <a reveal-id="DSU036TB2" data-open="DSU036TB2" class="link link-reveal link-table xref-fig">2</a>). This again supported for a possible link between genomic frequency and extent of motif imperfection of microsatellites. </p>                        <a id="6572825" scrollto-destination="6572825"></a>
<div content-id="DSU036TB2" class="table-modal"><div class="table-wrap"><div id="DSU036TB2" class="table-wrap-title" data-id="DSU036TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Pearson correlation of frequency of imperfect microsatellites. The microsatellite lengths are shown in bp in the first column. The frequencies of microsatellites having 1 mismatch are compared with that of microsatellites having 2, 4 or 6 mismatches among the 20 insect species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Repeat<br />length (bp) </th><th>Correlation<br />(1 mismatch vs. 2 mismatches) </th><th>Correlation<br />(1 mismatch vs. 4 mismatches) </th><th>Correlation<br />(1 mismatch vs. 6 mismatches) </th></tr></thead><tbody><tr><td>35 </td><td>0.959 </td><td>0.142 </td><td>0.096 </td></tr><tr><td>40 </td><td>0.887 </td><td>0.791 </td><td>0.190 </td></tr><tr><td>45 </td><td>0.814 </td><td>0.605 </td><td>0.010 </td></tr><tr><td>50 </td><td>0.721 </td><td>0.386 </td><td>0.300 </td></tr><tr><td>55 </td><td>0.749 </td><td>0.094 </td><td>0.143 </td></tr><tr><td>60 </td><td>0.920 </td><td>0.118 </td><td>0.022 </td></tr><tr><td>65 </td><td>0.839 </td><td>0.169 </td><td>0.009 </td></tr><tr><td>70 </td><td>0.865 </td><td>0.226 </td><td>−0.136 </td></tr><tr><td>75 </td><td>0.967 </td><td>0.361 </td><td>0.054 </td></tr><tr><td>80 </td><td>0.939 </td><td>0.692 </td><td>0.271 </td></tr><tr><td>85 </td><td>0.756 </td><td>0.474 </td><td>0.430 </td></tr></tbody></table></div><div content-id="DSU036TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Repeat<br />length (bp) </th><th>Correlation<br />(1 mismatch vs. 2 mismatches) </th><th>Correlation<br />(1 mismatch vs. 4 mismatches) </th><th>Correlation<br />(1 mismatch vs. 6 mismatches) </th></tr></thead><tbody><tr><td>35 </td><td>0.959 </td><td>0.142 </td><td>0.096 </td></tr><tr><td>40 </td><td>0.887 </td><td>0.791 </td><td>0.190 </td></tr><tr><td>45 </td><td>0.814 </td><td>0.605 </td><td>0.010 </td></tr><tr><td>50 </td><td>0.721 </td><td>0.386 </td><td>0.300 </td></tr><tr><td>55 </td><td>0.749 </td><td>0.094 </td><td>0.143 </td></tr><tr><td>60 </td><td>0.920 </td><td>0.118 </td><td>0.022 </td></tr><tr><td>65 </td><td>0.839 </td><td>0.169 </td><td>0.009 </td></tr><tr><td>70 </td><td>0.865 </td><td>0.226 </td><td>−0.136 </td></tr><tr><td>75 </td><td>0.967 </td><td>0.361 </td><td>0.054 </td></tr><tr><td>80 </td><td>0.939 </td><td>0.692 </td><td>0.271 </td></tr><tr><td>85 </td><td>0.756 </td><td>0.474 </td><td>0.430 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6572825" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU036TB2" class="table-wrap-title" data-id="DSU036TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Pearson correlation of frequency of imperfect microsatellites. The microsatellite lengths are shown in bp in the first column. The frequencies of microsatellites having 1 mismatch are compared with that of microsatellites having 2, 4 or 6 mismatches among the 20 insect species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Repeat<br />length (bp) </th><th>Correlation<br />(1 mismatch vs. 2 mismatches) </th><th>Correlation<br />(1 mismatch vs. 4 mismatches) </th><th>Correlation<br />(1 mismatch vs. 6 mismatches) </th></tr></thead><tbody><tr><td>35 </td><td>0.959 </td><td>0.142 </td><td>0.096 </td></tr><tr><td>40 </td><td>0.887 </td><td>0.791 </td><td>0.190 </td></tr><tr><td>45 </td><td>0.814 </td><td>0.605 </td><td>0.010 </td></tr><tr><td>50 </td><td>0.721 </td><td>0.386 </td><td>0.300 </td></tr><tr><td>55 </td><td>0.749 </td><td>0.094 </td><td>0.143 </td></tr><tr><td>60 </td><td>0.920 </td><td>0.118 </td><td>0.022 </td></tr><tr><td>65 </td><td>0.839 </td><td>0.169 </td><td>0.009 </td></tr><tr><td>70 </td><td>0.865 </td><td>0.226 </td><td>−0.136 </td></tr><tr><td>75 </td><td>0.967 </td><td>0.361 </td><td>0.054 </td></tr><tr><td>80 </td><td>0.939 </td><td>0.692 </td><td>0.271 </td></tr><tr><td>85 </td><td>0.756 </td><td>0.474 </td><td>0.430 </td></tr></tbody></table></div><div content-id="DSU036TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Repeat<br />length (bp) </th><th>Correlation<br />(1 mismatch vs. 2 mismatches) </th><th>Correlation<br />(1 mismatch vs. 4 mismatches) </th><th>Correlation<br />(1 mismatch vs. 6 mismatches) </th></tr></thead><tbody><tr><td>35 </td><td>0.959 </td><td>0.142 </td><td>0.096 </td></tr><tr><td>40 </td><td>0.887 </td><td>0.791 </td><td>0.190 </td></tr><tr><td>45 </td><td>0.814 </td><td>0.605 </td><td>0.010 </td></tr><tr><td>50 </td><td>0.721 </td><td>0.386 </td><td>0.300 </td></tr><tr><td>55 </td><td>0.749 </td><td>0.094 </td><td>0.143 </td></tr><tr><td>60 </td><td>0.920 </td><td>0.118 </td><td>0.022 </td></tr><tr><td>65 </td><td>0.839 </td><td>0.169 </td><td>0.009 </td></tr><tr><td>70 </td><td>0.865 </td><td>0.226 </td><td>−0.136 </td></tr><tr><td>75 </td><td>0.967 </td><td>0.361 </td><td>0.054 </td></tr><tr><td>80 </td><td>0.939 </td><td>0.692 </td><td>0.271 </td></tr><tr><td>85 </td><td>0.756 </td><td>0.474 </td><td>0.430 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6572825" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6572826 id="6572826" class="section-title" data-legacy-id=s3c>3.3. Motif heterogeneity and genomic abundance of microsatellites</h3>
<p>To further understand the relationship between motif heterogeneity and genomic abundance of microsatellites, we performed the following analyses. First, we calculated the expected number of mismatches in each imperfect microsatellite from the average rate of per-site mismatch (sum of mismatches divided by the sum of loci lengths) in individual genomes. Then for each imperfect microsatellite, the expected number of mismatches (repeat length multiplied with per-site mismatches) was calculated and compared with the observed number of mismatches in the loci. The results of this comparison showed that microsatellites that carry a higher amount of mismatches than expected are significantly related (Spearman <em>r</em> = 0.938, df = 19, <em>P</em> = 0.000001) with microsatellites that carry a lower amount of mismatches than expected across the species (Table <a reveal-id="DSU036TB3" data-open="DSU036TB3" class="link link-reveal link-table xref-fig">3</a>). The former group of microsatellites consistently showed lower frequency in the genome than the latter group, indicating that genomic frequency of microsatellites is negatively influenced by motif heterogeneity of the repeats. </p>                        <a id="6572829" scrollto-destination="6572829"></a>
<div content-id="DSU036TB3" class="table-modal"><div class="table-wrap"><div id="DSU036TB3" class="table-wrap-title" data-id="DSU036TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Number of microsatellites where motif mismatches are either higher or lower than the expected values of mismatches in different insect genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>Higher than expected </th><th>Lower than expected </th></tr></thead><tbody><tr><td>Aaeg </td><td>5,596 </td><td>17,024 </td></tr><tr><td>Agam </td><td>14,846 </td><td>15,361 </td></tr><tr><td>Apis </td><td>13,755 </td><td>30,812 </td></tr><tr><td>Cqui </td><td>6,525 </td><td>13,929 </td></tr><tr><td>Dana </td><td>5,766 </td><td>12,687 </td></tr><tr><td>Dere </td><td>5,322 </td><td>8,993 </td></tr><tr><td>Dgri </td><td>28,832 </td><td>45,805 </td></tr><tr><td>Dmel </td><td>7,945 </td><td>9,447 </td></tr><tr><td>Dmoj </td><td>34,735 </td><td>50,790 </td></tr><tr><td>Dper </td><td>15,174 </td><td>25,467 </td></tr><tr><td>Dpse </td><td>11,947 </td><td>24,342 </td></tr><tr><td>Dsec </td><td>4,116 </td><td>7,404 </td></tr><tr><td>Dsim </td><td>3,762 </td><td>7,233 </td></tr><tr><td>Dvir </td><td>23,855 </td><td>38,427 </td></tr><tr><td>Dwil </td><td>17,832 </td><td>37,289 </td></tr><tr><td>Dyak </td><td>6,386 </td><td>11,063 </td></tr><tr><td>Amel </td><td>17,893 </td><td>33,809 </td></tr><tr><td>Nvit </td><td>12,709 </td><td>15,365 </td></tr><tr><td>Bmor </td><td>5,443 </td><td>14,289 </td></tr><tr><td>Tcas </td><td>2,359 </td><td>2,796 </td></tr></tbody></table></div><div content-id="DSU036TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>Higher than expected </th><th>Lower than expected </th></tr></thead><tbody><tr><td>Aaeg </td><td>5,596 </td><td>17,024 </td></tr><tr><td>Agam </td><td>14,846 </td><td>15,361 </td></tr><tr><td>Apis </td><td>13,755 </td><td>30,812 </td></tr><tr><td>Cqui </td><td>6,525 </td><td>13,929 </td></tr><tr><td>Dana </td><td>5,766 </td><td>12,687 </td></tr><tr><td>Dere </td><td>5,322 </td><td>8,993 </td></tr><tr><td>Dgri </td><td>28,832 </td><td>45,805 </td></tr><tr><td>Dmel </td><td>7,945 </td><td>9,447 </td></tr><tr><td>Dmoj </td><td>34,735 </td><td>50,790 </td></tr><tr><td>Dper </td><td>15,174 </td><td>25,467 </td></tr><tr><td>Dpse </td><td>11,947 </td><td>24,342 </td></tr><tr><td>Dsec </td><td>4,116 </td><td>7,404 </td></tr><tr><td>Dsim </td><td>3,762 </td><td>7,233 </td></tr><tr><td>Dvir </td><td>23,855 </td><td>38,427 </td></tr><tr><td>Dwil </td><td>17,832 </td><td>37,289 </td></tr><tr><td>Dyak </td><td>6,386 </td><td>11,063 </td></tr><tr><td>Amel </td><td>17,893 </td><td>33,809 </td></tr><tr><td>Nvit </td><td>12,709 </td><td>15,365 </td></tr><tr><td>Bmor </td><td>5,443 </td><td>14,289 </td></tr><tr><td>Tcas </td><td>2,359 </td><td>2,796 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6572829" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU036TB3" class="table-wrap-title" data-id="DSU036TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Number of microsatellites where motif mismatches are either higher or lower than the expected values of mismatches in different insect genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>Higher than expected </th><th>Lower than expected </th></tr></thead><tbody><tr><td>Aaeg </td><td>5,596 </td><td>17,024 </td></tr><tr><td>Agam </td><td>14,846 </td><td>15,361 </td></tr><tr><td>Apis </td><td>13,755 </td><td>30,812 </td></tr><tr><td>Cqui </td><td>6,525 </td><td>13,929 </td></tr><tr><td>Dana </td><td>5,766 </td><td>12,687 </td></tr><tr><td>Dere </td><td>5,322 </td><td>8,993 </td></tr><tr><td>Dgri </td><td>28,832 </td><td>45,805 </td></tr><tr><td>Dmel </td><td>7,945 </td><td>9,447 </td></tr><tr><td>Dmoj </td><td>34,735 </td><td>50,790 </td></tr><tr><td>Dper </td><td>15,174 </td><td>25,467 </td></tr><tr><td>Dpse </td><td>11,947 </td><td>24,342 </td></tr><tr><td>Dsec </td><td>4,116 </td><td>7,404 </td></tr><tr><td>Dsim </td><td>3,762 </td><td>7,233 </td></tr><tr><td>Dvir </td><td>23,855 </td><td>38,427 </td></tr><tr><td>Dwil </td><td>17,832 </td><td>37,289 </td></tr><tr><td>Dyak </td><td>6,386 </td><td>11,063 </td></tr><tr><td>Amel </td><td>17,893 </td><td>33,809 </td></tr><tr><td>Nvit </td><td>12,709 </td><td>15,365 </td></tr><tr><td>Bmor </td><td>5,443 </td><td>14,289 </td></tr><tr><td>Tcas </td><td>2,359 </td><td>2,796 </td></tr></tbody></table></div><div content-id="DSU036TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>Higher than expected </th><th>Lower than expected </th></tr></thead><tbody><tr><td>Aaeg </td><td>5,596 </td><td>17,024 </td></tr><tr><td>Agam </td><td>14,846 </td><td>15,361 </td></tr><tr><td>Apis </td><td>13,755 </td><td>30,812 </td></tr><tr><td>Cqui </td><td>6,525 </td><td>13,929 </td></tr><tr><td>Dana </td><td>5,766 </td><td>12,687 </td></tr><tr><td>Dere </td><td>5,322 </td><td>8,993 </td></tr><tr><td>Dgri </td><td>28,832 </td><td>45,805 </td></tr><tr><td>Dmel </td><td>7,945 </td><td>9,447 </td></tr><tr><td>Dmoj </td><td>34,735 </td><td>50,790 </td></tr><tr><td>Dper </td><td>15,174 </td><td>25,467 </td></tr><tr><td>Dpse </td><td>11,947 </td><td>24,342 </td></tr><tr><td>Dsec </td><td>4,116 </td><td>7,404 </td></tr><tr><td>Dsim </td><td>3,762 </td><td>7,233 </td></tr><tr><td>Dvir </td><td>23,855 </td><td>38,427 </td></tr><tr><td>Dwil </td><td>17,832 </td><td>37,289 </td></tr><tr><td>Dyak </td><td>6,386 </td><td>11,063 </td></tr><tr><td>Amel </td><td>17,893 </td><td>33,809 </td></tr><tr><td>Nvit </td><td>12,709 </td><td>15,365 </td></tr><tr><td>Bmor </td><td>5,443 </td><td>14,289 </td></tr><tr><td>Tcas </td><td>2,359 </td><td>2,796 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6572829" target="_blank">View Large</a></div></div><p>To further confirm that genomic abundance of microsatellites is associated with motif mismatches of microsatellites, we performed PERMANOVA<sup><a reveal-id="DSU036C25" data-open="DSU036C25" class="link link-ref link-reveal xref-bibr">25</a></sup> test among different groups of motif mismatches (1, 2, 4 and 6 mismatches) of microsatellites among species (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). The results of this analysis revealed that the relationship between higher mismatches and lower genomic abundance of microsatellites is statistically significant (<em>P</em> = 0.0001). The pair-wise <em>a posteriori</em> comparison between microsatellites of different lengths showed that the significance level of association of genomic abundance varies differentially across the insect species based on the length of microsatellites (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). These results clearly suggested that microsatellite abundance in genome is intricately related to sequence imperfection of motifs and the length of the microsatellites across species.</p><p>We also compared microsatellites by controlling the length and number of motif mismatches of microsatellites across the species. Here, we restricted the comparison among loci that are at least 30 bp in length and have either &lt;3 or ≥3 mismatches. These length and mismatch criteria were selected based on our search parameters as described in the Materials and methods section. The result of this analysis also revealed that loci that have less than three mismatches are more prevalent in the genome than those with three or more number of mismatches (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). This pattern was consistent not only in the insect species, but also in other unrelated species such as yeast, roundworm, mouse and human (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>).</p><p>We also wanted to confirm that our observations were not influenced by the search parameters we used to identify the microsatellites. To test that, we used different parameters such as minimum score 15 and mismatch penalty 3 or minimum score of 10 and mismatch penalty 5 to search microsatellites in specific genomes. When data of these searches were used to repeat the analyses described above, we still found that microsatellites with higher mismatches tend to have lower prevalence in the genome (data not shown). This confirmed that the link between sequence imperfection and microsatellite abundance is a robust evolutionary process and is not influenced by the search parameters of microsatellites.</p>                            <h3 scrollto-destination=6572846 id="6572846" class="section-title" data-legacy-id=s3d>3.4. Mutation and imperfection of motif sequences</h3>
<p>To investigate role of mutation in motif imperfection, SNPs localized within microsatellite loci were identified from the dbSNP database (see Materials and methods). Then, we identified loci where motif mismatches corresponded to the alternate alleles of the cognate SNP (Fig. <a reveal-id="DSU036F3" data-open="DSU036F3" class="link link-reveal link-table xref-fig">3</a>). These were referred to as M<sub>1</sub> microsatellites. We also found microsatellites where motif mismatches did not correspond to the SNP alleles (Fig. <a reveal-id="DSU036F3" data-open="DSU036F3" class="link link-reveal link-table xref-fig">3</a>). These were referred to as M<sub>0</sub> microsatellites. The microsatellites with different numbers of mismatches (≥3 or &lt;3 mismatches) and having either M<sub>0</sub> or M<sub>1</sub> allele patterns were identified in <em>D. melanogaster</em> and <em>A. gambiae</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). The number of mismatches (≥3 or &lt;3) was particularly chosen as 1) microsatellites &lt;30 bp in length are predominant in both genomes, and 2) loci that are &lt;30 bp have less than three mismatches as per our search conditions. The count data of microsatellites, based on the number of motif mismatches and pattern of allelic variation (M<sub>0</sub> or M<sub>1</sub>), were analysed by Yates' chi-squared test. The result of this analysis revealed significant (<em>χ</em><sup>2</sup> = 315.5, <em>P</em> &lt; 0.0001) association between allele sequences of SNPs and mismatch patterns of microsatellites in <em>D. melanogaster</em>. A similar result was observed in <em>A. gambiae</em> where the association was also statistically significant (<em>χ</em><sup>2</sup> = 13.6, <em>P</em> = 0.0002). Taken together, these findings show that patterns of allelic changes of microsatellites have a significant role in modulating the number of motif mismatches of imperfect microsatellites. </p>                        <a id="6572851" scrollto-destination="6572851"></a>
<div id="DSU036F3" data-id="DSU036F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/m_dsu03603.png?Expires=2147483647&Signature=P3mRh9n328aFrj~nEYQ20dozIq6hJ2mfBdgMqqAtbFt4Rio1M6kHCuMWJwZbhL21D~gY0UTmzmyl3WwJY-enaB81iy8pZmsygbSx7gzne-1zmAq-IqG0y0OBlcgeWx9SGcuwlw-Uzg8t3F9S0mRbkGKNrutKKxYoCQFkFlgNfjtCTc3YZLwu4RuhPWGrOexHQ~-Jaaof6PqIPLvSsYQsj1L6c2PsQgX5JGqyfccui4kclS8aFBI8Q3aIXP8iaAr~55wOUI~oIL~vSrKml-HVqPZGuxiKWUlHPGJA-BBLRV3IKWEWJ7fBArcnHGJmsmqX2eAJag5sFBTJbM56omO1EQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Imperfect microsatellites with M1 and M0 SNP alleles. (A) The M1 type: these are microsatellites where the SNP alleles match with the variant nucleotide of the motif sequences. The SNP ID is shown by rs # and the alleles are shown within parenthesis. The microsatellite and the mismatch motifs are shown, and both A and G are represented in the motifs. (B) The M0 type: these are microsatellites where the alternative SNP alleles are not found in the motif sequences of the microsatellite. Note that the C allele of the SNP is not represented in the mismatch motif." path-from-xml="dsu03603.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6572851/dsu03603.tif" path-from-xml="dsu03603.tif" target="_blank">View large</a><a section="6572851" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/dsu03603.png?Expires=2147483647&Signature=AhmCcih80np5Vs9fScoRIS~d8Y8N5VpO3GSwLOcwbdZb-nGkeG9W0MpPozru5fZaBw7byMwRyfLDnM0cZyTCebPDlKzq3hCpd0yOrTn5cA0PAX1XaNmV5HnhR-vzfvFAWGljLhj836XrCY5SZjkNQgSVb6gYCokBoZxIsNt-3mlBKwuMIi6qgHjXJaLj19iKIQ8I-NSKv51ANU9wW6up-T2tJSYAMqIctfWgNdhaO2RYKX7ctwn5MXQ06J9c1BFpI7goceqQqyFLhfPAZVFsoSr2PmmR9bYv9furvSKe6OZ18E9hTHabLnZCIsDw4k2GwFNpamVmwBUi3RI-3pwjCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6572851&ar=440058&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03603.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Imperfect microsatellites with M<sub>1</sub> and M<sub>0</sub> SNP alleles. (A) The M<sub>1</sub> type: these are microsatellites where the SNP alleles match with the variant nucleotide of the motif sequences. The SNP ID is shown by rs # and the alleles are shown within parenthesis. The microsatellite and the mismatch motifs are shown, and both A and G are represented in the motifs. (B) The M<sub>0</sub> type: these are microsatellites where the alternative SNP alleles are not found in the motif sequences of the microsatellite. Note that the C allele of the SNP is not represented in the mismatch motif.</p></div></div><div content-id="DSU036F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/m_dsu03603.png?Expires=2147483647&Signature=P3mRh9n328aFrj~nEYQ20dozIq6hJ2mfBdgMqqAtbFt4Rio1M6kHCuMWJwZbhL21D~gY0UTmzmyl3WwJY-enaB81iy8pZmsygbSx7gzne-1zmAq-IqG0y0OBlcgeWx9SGcuwlw-Uzg8t3F9S0mRbkGKNrutKKxYoCQFkFlgNfjtCTc3YZLwu4RuhPWGrOexHQ~-Jaaof6PqIPLvSsYQsj1L6c2PsQgX5JGqyfccui4kclS8aFBI8Q3aIXP8iaAr~55wOUI~oIL~vSrKml-HVqPZGuxiKWUlHPGJA-BBLRV3IKWEWJ7fBArcnHGJmsmqX2eAJag5sFBTJbM56omO1EQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Imperfect microsatellites with M1 and M0 SNP alleles. (A) The M1 type: these are microsatellites where the SNP alleles match with the variant nucleotide of the motif sequences. The SNP ID is shown by rs # and the alleles are shown within parenthesis. The microsatellite and the mismatch motifs are shown, and both A and G are represented in the motifs. (B) The M0 type: these are microsatellites where the alternative SNP alleles are not found in the motif sequences of the microsatellite. Note that the C allele of the SNP is not represented in the mismatch motif." path-from-xml="dsu03603.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6572851/dsu03603.tif" path-from-xml="dsu03603.tif" target="_blank">View large</a><a section="6572851" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/dsu03603.png?Expires=2147483647&Signature=AhmCcih80np5Vs9fScoRIS~d8Y8N5VpO3GSwLOcwbdZb-nGkeG9W0MpPozru5fZaBw7byMwRyfLDnM0cZyTCebPDlKzq3hCpd0yOrTn5cA0PAX1XaNmV5HnhR-vzfvFAWGljLhj836XrCY5SZjkNQgSVb6gYCokBoZxIsNt-3mlBKwuMIi6qgHjXJaLj19iKIQ8I-NSKv51ANU9wW6up-T2tJSYAMqIctfWgNdhaO2RYKX7ctwn5MXQ06J9c1BFpI7goceqQqyFLhfPAZVFsoSr2PmmR9bYv9furvSKe6OZ18E9hTHabLnZCIsDw4k2GwFNpamVmwBUi3RI-3pwjCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6572851&ar=440058&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03603.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Imperfect microsatellites with M<sub>1</sub> and M<sub>0</sub> SNP alleles. (A) The M<sub>1</sub> type: these are microsatellites where the SNP alleles match with the variant nucleotide of the motif sequences. The SNP ID is shown by rs # and the alleles are shown within parenthesis. The microsatellite and the mismatch motifs are shown, and both A and G are represented in the motifs. (B) The M<sub>0</sub> type: these are microsatellites where the alternative SNP alleles are not found in the motif sequences of the microsatellite. Note that the C allele of the SNP is not represented in the mismatch motif.</p></div></div>                            <h3 scrollto-destination=6572855 id="6572855" class="section-title" data-legacy-id=s3e>3.5. Insights from population data</h3>
<p>We also analysed sequences of imperfect microsatellites of multiple inbred lines (<em>n</em> = 29) of <em>D. melanogaster</em> (DGRP data) to understand within-species variation of motif sequences of imperfect microsatellite loci. We identified a total of 118,446 imperfect microsatellites from the chromosome X sequences of the 29 inbred lines of <em>D. melanogaster</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>). Of these, a total of 1,025 loci showed polymorphic microsatellites (Fig. <a reveal-id="DSU036F4" data-open="DSU036F4" class="link link-reveal link-table xref-fig">4</a>). We used the allele sequences and motif mismatch patterns of these 1,025 microsatellites to train the imperfect microsatellites identified from dbSNP data. Linear regression-based data classification of SNP allelic variation (M<sub>0</sub> versus M<sub>1</sub>) and the corresponding motif mismatch patterns of microsatellites were performed using Weka3.6.<sup><a reveal-id="DSU036C28" data-open="DSU036C28" class="link link-ref link-reveal xref-bibr">28</a></sup> The results of these analyses revealed a significant relationship (<em>P</em> &lt; 0.05) between allele variation and motif mismatches of imperfect microsatellites, and also showed that for accumulation of a single M<sub>1</sub> type of mutation, the number of repeat mismatches in the microsatellite decreases by 56%. This further confirmed that pattern of allelic variation plays a major role in motif heterogeneity of microsatellites. </p>                        <a id="6572859" scrollto-destination="6572859"></a>
<div id="DSU036F4" data-id="DSU036F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/m_dsu03604.png?Expires=2147483647&Signature=Ze5EDqimxdwSsDujw7x3XU4pIbKGu-EKZX2Sp~11FRULMa263WRRUfnnG8DdpcANYIBta5OfjHspTy6JIdlONHLldAJHuj~7VZ1ffzFGOvp3FmqSVco9tWZC~LmdqcDnOK3anJDXMIU1FRATOloRk11~DHJOWpkPX6hTdlYAw6X~KcroC6Yf6HppoQISxI9Uz2G4mNCAlaBJLBlY7Ud0-VrVaG6ijpsHA8cxtDbThEGBxzoO-C-rEYdTcUC2zYDjYdDriRBuwp71p59w7tWnlA3822XMvqL9vD~rO~rlmiokn2~g-tbBkfI-JvK96mbmYoyg1ErRIykUB0W2Fdsa1Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Multiple sequence alignment of microsatellite loci among different inbred lines of D. melanogaster. The chromosomal position of the microsatellite loci is shown below the alignment. The inbred line ID corresponding to each sequence is shown. The reference SNP identified in D. melanogaster (dbSNP data) is shown by a downward arrow along with SNP ID and alleles." path-from-xml="dsu03604.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6572859/dsu03604.tif" path-from-xml="dsu03604.tif" target="_blank">View large</a><a section="6572859" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/dsu03604.png?Expires=2147483647&Signature=eOsb-nns1ujUB6LgmUGmGT307TF97Ihk3oufxuAiLKfaFjh7idW7bSjOBPOUI9mFNLHsdc1N8AGhpuY8IXp4Iwji3cuJkFykeFtFOK-C6ct91~7kkgI2frKByMSrwS6qA4G8Ji4wm~HnAqXrOaIv1iobd~ls2OLd~hNLTyVZxiZW1NcgNuz31g3lnf5Z0MVW1hHbfbCYgalWrTXexID5P4uvb-Eo8sIEzs47UFEvv-wVLszrCRkyvzuDinH5MWiOvRlk-NFsXpkSl98kxi8ReWWYKWe8hUsrMVdy23nCOS~C9B9FQkNDnCdHLgzo8~Rj0jyUyZLt2-8cA2Ds6A~l9g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6572859&ar=440058&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03604.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Multiple sequence alignment of microsatellite loci among different inbred lines of <em>D. melanogaster</em>. The chromosomal position of the microsatellite loci is shown below the alignment. The inbred line ID corresponding to each sequence is shown. The reference SNP identified in <em>D. melanogaster</em> (dbSNP data) is shown by a downward arrow along with SNP ID and alleles.</p></div></div><div content-id="DSU036F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/m_dsu03604.png?Expires=2147483647&Signature=Ze5EDqimxdwSsDujw7x3XU4pIbKGu-EKZX2Sp~11FRULMa263WRRUfnnG8DdpcANYIBta5OfjHspTy6JIdlONHLldAJHuj~7VZ1ffzFGOvp3FmqSVco9tWZC~LmdqcDnOK3anJDXMIU1FRATOloRk11~DHJOWpkPX6hTdlYAw6X~KcroC6Yf6HppoQISxI9Uz2G4mNCAlaBJLBlY7Ud0-VrVaG6ijpsHA8cxtDbThEGBxzoO-C-rEYdTcUC2zYDjYdDriRBuwp71p59w7tWnlA3822XMvqL9vD~rO~rlmiokn2~g-tbBkfI-JvK96mbmYoyg1ErRIykUB0W2Fdsa1Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Multiple sequence alignment of microsatellite loci among different inbred lines of D. melanogaster. The chromosomal position of the microsatellite loci is shown below the alignment. The inbred line ID corresponding to each sequence is shown. The reference SNP identified in D. melanogaster (dbSNP data) is shown by a downward arrow along with SNP ID and alleles." path-from-xml="dsu03604.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6572859/dsu03604.tif" path-from-xml="dsu03604.tif" target="_blank">View large</a><a section="6572859" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/dsu03604.png?Expires=2147483647&Signature=eOsb-nns1ujUB6LgmUGmGT307TF97Ihk3oufxuAiLKfaFjh7idW7bSjOBPOUI9mFNLHsdc1N8AGhpuY8IXp4Iwji3cuJkFykeFtFOK-C6ct91~7kkgI2frKByMSrwS6qA4G8Ji4wm~HnAqXrOaIv1iobd~ls2OLd~hNLTyVZxiZW1NcgNuz31g3lnf5Z0MVW1hHbfbCYgalWrTXexID5P4uvb-Eo8sIEzs47UFEvv-wVLszrCRkyvzuDinH5MWiOvRlk-NFsXpkSl98kxi8ReWWYKWe8hUsrMVdy23nCOS~C9B9FQkNDnCdHLgzo8~Rj0jyUyZLt2-8cA2Ds6A~l9g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6572859&ar=440058&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03604.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Multiple sequence alignment of microsatellite loci among different inbred lines of <em>D. melanogaster</em>. The chromosomal position of the microsatellite loci is shown below the alignment. The inbred line ID corresponding to each sequence is shown. The reference SNP identified in <em>D. melanogaster</em> (dbSNP data) is shown by a downward arrow along with SNP ID and alleles.</p></div></div><p>We also estimated the age of specific alleles of imperfect microsatellites based on the population data (DGRP). First, we identified flanking mutations (within 1 kb on both ends) of microsatellites that were in complete linkage with segregating mutation within the microsatellite (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). Based on the results of phylogenetic analysis, the extended sequences showed differential genealogical relations among the inbred lines (Fig. <a reveal-id="DSU036F5" data-open="DSU036F5" class="link link-reveal link-table xref-fig">5</a>). We determined the number of fixed mutations, total number of variable sites and the average number of nucleotide differences in the extended sequences of imperfect microsatellites (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). Based on the allele frequency of mutations within and the flanking sequences of the imperfect microsatellite, we predicted generation times that might have lapsed since the origin of the mutation in the microsatellite. The maximum likelihood method<sup><a reveal-id="DSU036C17" data-open="DSU036C17" class="link link-ref link-reveal xref-bibr">17</a></sup> was used for making this prediction from the joint distribution of the number of copies in the population and the coalescence times of the intra-allelic variation. The results revealed that the maximum likelihoods varied between 1,380 and 1,530 generations (Fig. <a reveal-id="DSU036F6" data-open="DSU036F6" class="link link-reveal link-table xref-fig">6</a>). Assuming 36 days as the average generation time of <em>D. melanogaster</em>, it was thus estimated that these alleles might have arisen 136–150 yrs ago. </p>                        <a id="6572866" scrollto-destination="6572866"></a>
<div id="DSU036F5" data-id="DSU036F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/m_dsu03605.png?Expires=2147483647&Signature=jx-SDfHvZ32xKXJ5Pp3r~oHE43XwsJY4GDod9UQf5TMyGBnAmsQhVpXfnC2EmlFQpG1MmVKj9AP9lUld-~ah75CyO~piJjvICONblMm-YqPuVcSHnHXVoYttREo3ihV2lS4UhyDD5J3fo8F-MvHYWhoAaWDnG0ywmu6kKVhoQGrE~cTEhm9jn6rWXG77Uekj~4MavefPXsqG6vAnFUGKOYt5LTRO4VTp0QVQN~begTSXQhtFeNEPcE3ErYRqiPPLzHxd-AynLrP1IY8WqX7BoWd8sjrP03tt8ErSOC0Kki7tJ~tO2fiEjPBEI7-4HsxJw5sZWS7IuI6ff4Mv7velFg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Neighbour-joining phylogenetic trees of microsatellites loci (with 1 kb flanking sequences of both ends) of D. melanogaster inbred lines. The chromosomal positions, the microsatellite repeat and the SNP position within the microsatellites (upward arrow) are shown on the top of each tree. The sequence origins (inbred line # or the reference genome) are also shown for each branch along with bootstrap support values. The phylogenetic grouping of D. melanogaster lines that carry the alternative allele of the SNP is also shown. The scale of branch length is shown below each tree." path-from-xml="dsu03605.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6572866/dsu03605.tif" path-from-xml="dsu03605.tif" target="_blank">View large</a><a section="6572866" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/dsu03605.png?Expires=2147483647&Signature=op0EACE2nk~LixCHL4DNfXWbTrvlkKkoWns4tO6o8-puz9Cc-ggUnjONIcqrLBImBNUCZ1mVOK9PJmkiSTyxhbWlGswusgjLDOaCD2yQKuJ1Ig605-y6P3LiozSFxlyElSGuPCFPRot-JZ87q9AYrSzgrSiVUiNROMcs0iEEAp3YzP65DDL0rc3Tn~yigpKO24NzubXdNsyuaGJlg3K4v~~0TIhdDKLqZ7nwngXTzte19HfixYJnI8FVSC68ZPmbXzKut47beqZafTjd1PX1SG84pnVfiA2aMvo7NG4nGRTNRzHIxFbYDbwqxKM6VoYVQk4uCF0MgtFiOmUwBzQdig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6572866&ar=440058&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03605.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Neighbour-joining phylogenetic trees of microsatellites loci (with 1 kb flanking sequences of both ends) of <em>D. melanogaster</em> inbred lines. The chromosomal positions, the microsatellite repeat and the SNP position within the microsatellites (upward arrow) are shown on the top of each tree. The sequence origins (inbred line # or the reference genome) are also shown for each branch along with bootstrap support values. The phylogenetic grouping of <em>D. melanogaster</em> lines that carry the alternative allele of the SNP is also shown. The scale of branch length is shown below each tree.</p></div></div><div content-id="DSU036F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/m_dsu03605.png?Expires=2147483647&Signature=jx-SDfHvZ32xKXJ5Pp3r~oHE43XwsJY4GDod9UQf5TMyGBnAmsQhVpXfnC2EmlFQpG1MmVKj9AP9lUld-~ah75CyO~piJjvICONblMm-YqPuVcSHnHXVoYttREo3ihV2lS4UhyDD5J3fo8F-MvHYWhoAaWDnG0ywmu6kKVhoQGrE~cTEhm9jn6rWXG77Uekj~4MavefPXsqG6vAnFUGKOYt5LTRO4VTp0QVQN~begTSXQhtFeNEPcE3ErYRqiPPLzHxd-AynLrP1IY8WqX7BoWd8sjrP03tt8ErSOC0Kki7tJ~tO2fiEjPBEI7-4HsxJw5sZWS7IuI6ff4Mv7velFg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Neighbour-joining phylogenetic trees of microsatellites loci (with 1 kb flanking sequences of both ends) of D. melanogaster inbred lines. The chromosomal positions, the microsatellite repeat and the SNP position within the microsatellites (upward arrow) are shown on the top of each tree. The sequence origins (inbred line # or the reference genome) are also shown for each branch along with bootstrap support values. The phylogenetic grouping of D. melanogaster lines that carry the alternative allele of the SNP is also shown. The scale of branch length is shown below each tree." path-from-xml="dsu03605.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6572866/dsu03605.tif" path-from-xml="dsu03605.tif" target="_blank">View large</a><a section="6572866" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/dsu03605.png?Expires=2147483647&Signature=op0EACE2nk~LixCHL4DNfXWbTrvlkKkoWns4tO6o8-puz9Cc-ggUnjONIcqrLBImBNUCZ1mVOK9PJmkiSTyxhbWlGswusgjLDOaCD2yQKuJ1Ig605-y6P3LiozSFxlyElSGuPCFPRot-JZ87q9AYrSzgrSiVUiNROMcs0iEEAp3YzP65DDL0rc3Tn~yigpKO24NzubXdNsyuaGJlg3K4v~~0TIhdDKLqZ7nwngXTzte19HfixYJnI8FVSC68ZPmbXzKut47beqZafTjd1PX1SG84pnVfiA2aMvo7NG4nGRTNRzHIxFbYDbwqxKM6VoYVQk4uCF0MgtFiOmUwBzQdig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6572866&ar=440058&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03605.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Neighbour-joining phylogenetic trees of microsatellites loci (with 1 kb flanking sequences of both ends) of <em>D. melanogaster</em> inbred lines. The chromosomal positions, the microsatellite repeat and the SNP position within the microsatellites (upward arrow) are shown on the top of each tree. The sequence origins (inbred line # or the reference genome) are also shown for each branch along with bootstrap support values. The phylogenetic grouping of <em>D. melanogaster</em> lines that carry the alternative allele of the SNP is also shown. The scale of branch length is shown below each tree.</p></div></div>                        <a id="6572872" scrollto-destination="6572872"></a>
<div id="DSU036F6" data-id="DSU036F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/m_dsu03606.png?Expires=2147483647&Signature=K3tTaJcb1uecghXgULyFcSQrwR9x~c4qA9rFhEgG2hwN3wD9PEc92MhGshxcwtlqqdTCic-oMxI5YEpcvJDm0DgCIzmgPFm63oepTRryyRu4VdhZfZi44SZGK9FfvIjkdkuck97nlcGvm1izcfEJ7Jldasmu4uUgyLHBiAg08JSN524u~C8oADtqRcE4JZQfjPzQCoLj~LIWhzb7KoiGjvXH07RSGoaJvR16THWoYO7S-1S06KKMthqYeY1HArFyl6ZHz6HmvhpYCUPlMBrvXXk8op-MYFYSeJ9OsAbEs4UwKQuickH2RNlhEYit-MfSpAY-cNgipLdBTsMxmwkH6A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of likelihoods of allele age (generations ago) of the mutation (SNP) in the microsatellite sequences. The SNP IDs along with their alleles are shown above the graphs. The x-axis represents time (in generations) when the mutation likely occurred, and the corresponding estimates of likelihoods are shown in the y-axis. The specific generation time when the likelihood value is highest is indicated by an upward arrow." path-from-xml="dsu03606.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6572872/dsu03606.tif" path-from-xml="dsu03606.tif" target="_blank">View large</a><a section="6572872" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/dsu03606.png?Expires=2147483647&Signature=Lm00lK~8aXzUQmhT5zx7yuQXV1s0me5BYg18KP0H5LlrEJ2SKf3crtKoYugsOjLZCT24NqKkAR0nDg0nBsd7Bjm~V~wotIPTunzftA9FI5XKMbVv-beywC5f9duYeRvJOc7~2M5J8HJuzpn3KyiAKl-2Q2zddfLjWVE~Fg2HKf~QnWaiybA7ksv4mCS4ZDheR1Z9IiLSS2lnMF7~fO8TnuikzPsupc0GaSq8dO8IkuS8xjLCJSj78RqCC7WnwrWffMmnTLlJWCwhT3yVQNjO7uvO3CFrq9WuGhri0DYLpI3bzgwXJr1wWelsxrzgDW5t61yYZSEzCgPQRdzsv1vWNQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6572872&ar=440058&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03606.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of likelihoods of allele age (generations ago) of the mutation (SNP) in the microsatellite sequences. The SNP IDs along with their alleles are shown above the graphs. The <em>x</em>-axis represents time (in generations) when the mutation likely occurred, and the corresponding estimates of likelihoods are shown in the <em>y</em>-axis. The specific generation time when the likelihood value is highest is indicated by an upward arrow.</p></div></div><div content-id="DSU036F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/m_dsu03606.png?Expires=2147483647&Signature=K3tTaJcb1uecghXgULyFcSQrwR9x~c4qA9rFhEgG2hwN3wD9PEc92MhGshxcwtlqqdTCic-oMxI5YEpcvJDm0DgCIzmgPFm63oepTRryyRu4VdhZfZi44SZGK9FfvIjkdkuck97nlcGvm1izcfEJ7Jldasmu4uUgyLHBiAg08JSN524u~C8oADtqRcE4JZQfjPzQCoLj~LIWhzb7KoiGjvXH07RSGoaJvR16THWoYO7S-1S06KKMthqYeY1HArFyl6ZHz6HmvhpYCUPlMBrvXXk8op-MYFYSeJ9OsAbEs4UwKQuickH2RNlhEYit-MfSpAY-cNgipLdBTsMxmwkH6A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of likelihoods of allele age (generations ago) of the mutation (SNP) in the microsatellite sequences. The SNP IDs along with their alleles are shown above the graphs. The x-axis represents time (in generations) when the mutation likely occurred, and the corresponding estimates of likelihoods are shown in the y-axis. The specific generation time when the likelihood value is highest is indicated by an upward arrow." path-from-xml="dsu03606.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6572872/dsu03606.tif" path-from-xml="dsu03606.tif" target="_blank">View large</a><a section="6572872" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/dsu03606.png?Expires=2147483647&Signature=Lm00lK~8aXzUQmhT5zx7yuQXV1s0me5BYg18KP0H5LlrEJ2SKf3crtKoYugsOjLZCT24NqKkAR0nDg0nBsd7Bjm~V~wotIPTunzftA9FI5XKMbVv-beywC5f9duYeRvJOc7~2M5J8HJuzpn3KyiAKl-2Q2zddfLjWVE~Fg2HKf~QnWaiybA7ksv4mCS4ZDheR1Z9IiLSS2lnMF7~fO8TnuikzPsupc0GaSq8dO8IkuS8xjLCJSj78RqCC7WnwrWffMmnTLlJWCwhT3yVQNjO7uvO3CFrq9WuGhri0DYLpI3bzgwXJr1wWelsxrzgDW5t61yYZSEzCgPQRdzsv1vWNQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6572872&ar=440058&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03606.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of likelihoods of allele age (generations ago) of the mutation (SNP) in the microsatellite sequences. The SNP IDs along with their alleles are shown above the graphs. The <em>x</em>-axis represents time (in generations) when the mutation likely occurred, and the corresponding estimates of likelihoods are shown in the <em>y</em>-axis. The specific generation time when the likelihood value is highest is indicated by an upward arrow.</p></div></div>                            <h2 scrollto-destination=6572874 id="6572874" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>The results obtained from this study provide insights into sequence imperfection of microsatellite motifs in a genome-wide manner. We chose insect genomes for this investigation as microsatellites have been extensively exploited as molecular markers in ecology and population studies of numerous insect species,<sup><a reveal-id="DSU036C31" data-open="DSU036C31" class="link link-ref link-reveal xref-bibr">31</a></sup> yet the imperfect microsatellites of insects remain poorly understood. As studies on evolution of microsatellites are gaining renewed attention with the advent of high-throughput DNA sequencing and whole-genome sequences,<sup><a reveal-id="DSU036C32" data-open="DSU036C32" class="link link-ref link-reveal xref-bibr">32</a>–<a reveal-id="DSU036C39" data-open="DSU036C39" class="link link-ref link-reveal xref-bibr">39</a></sup> the present study was initiated as an effort to gain insights into the motif mismatch patterns of microsatellites of insects as well as specific non-insect eukaryotes. In this investigation, imperfect microsatellites were computationally identified from the draft genome sequences of the chosen species by SciRoko.<sup><a reveal-id="DSU036C23" data-open="DSU036C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Apart from SciRoKo, other computational algorithms such as IMEx, NWISL and Phobos are also available to identify imperfect microsatellites<sup><a reveal-id="DSU036C40" data-open="DSU036C40" class="link link-ref link-reveal xref-bibr">40</a></sup> (<a class="link link-uri" href="http://ssr.nwisrl.ars.usda.gov/">http://ssr.nwisrl.ars.usda.gov/</a> and <a class="link link-uri" href="http://www.ruhr-uni-bochum.de/ecoevo/cm/cm_phobos.htm">www.ruhr-uni-bochum.de/ecoevo/cm/cm_phobos.htm</a>). However, comparing the efficacy of different algorithms in detecting imperfect microsatellites was not the aim of this study. We analysed each genome using the same tool (SciRoKo) and the same parameters appropriate for data comparison across genomes. The genome-wide search of microsatellites in insect species by SciRoKo revealed that the proportion of microsatellites with imperfect motifs varies from species to species even within the same genus. The 12 Drosophila species clearly represent that pattern (Table <a reveal-id="DSU036TB1" data-open="DSU036TB1" class="link link-reveal link-table xref-fig">1</a> and Fig. <a reveal-id="DSU036F1" data-open="DSU036F1" class="link link-reveal link-table xref-fig">1</a>). Earlier study has shown that microsatellite abundance diverges rapidly between Drosophila species, and that considerable variation in the relative abundance of motif size classes exists between species.<sup><a reveal-id="DSU036C41" data-open="DSU036C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Moreover, differential association of microsatellites with coding sequences of Drosophila genome may impose differential constraint on microsatellite abundance between species. In fact, the number of microsatellites associated with coding sequences varies extensively among the 12 Drosophila genomes as shown from the previous study.<sup><a reveal-id="DSU036C36" data-open="DSU036C36" class="link link-ref link-reveal xref-bibr">36</a></sup> At the same time, it is possible that the quality of genome assembly may affect identification of microsatellites comprehensively. However, the 20 insect genomes we analysed represented different qualities of genome assemblies of the insect species. Yet, our results showed highly consistent patterns in microsatellite imperfections across species (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>), indicating that the results of this study are not affected by differences in assembly quality of the genomes.</p><p>We found that sex chromosomes tend to have elevated frequency of imperfect microsatellites than autosomes in <em>Drosophila</em> species. It is likely that differential association of microsatellites with coding sequences of different chromosomes may impose differential constraint on non-random prevalence of imperfect microsatellites within the genome. These coding repeats may be under differential selection pressure in the genome. It is known that repeats representing amino acid runs in <em>Drosophila</em> are under positive selection.<sup><a reveal-id="DSU036C34" data-open="DSU036C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Furthermore, in <em>D. melanogaster</em>, the adaptive evolution proceeds more quickly in X than autosomes.<sup><a reveal-id="DSU036C42" data-open="DSU036C42" class="link link-ref link-reveal xref-bibr">42</a></sup> This is commonly referred to as the faster-X hypothesis. Such X bias adaptive evolution is associated with a number of phenotypic, morphological and behavioural changes in the fly.<sup><a reveal-id="DSU036C43" data-open="DSU036C43" class="link link-ref link-reveal xref-bibr">43</a></sup> As sex chromosomes in <em>Drosophila</em> have elevated frequency of imperfect microsatellites than autosomes, the possibility of coding microsatellites playing a role in faster X evolution of Drosophila genome canno't be ruled out. Furthermore, our earlier study has shown that the trinucleotide repeats represent the highest fraction of codon repeats in insects.<sup><a reveal-id="DSU036C36" data-open="DSU036C36" class="link link-ref link-reveal xref-bibr">36</a></sup> From the current study, we also observe that trinucleotide microsatellites show higher variation in motif imperfect than other motif sizes in several insects including <em>D. melanogaster</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). It is likely that rapid variation of codon repeats in chromosome X may cause faster adaptive evolution of genes localized in X chromosome than autosomes. However, it is unlikely that such a mechanism is operational in all insect species. This is because no evidence of elevated frequency of imperfect microsatellites in chromosome X compared with autosomes was observed in the non-<em>Drosophila</em> species we investigated.</p><p>We further observed that microsatellite length and the extent of motif imperfection are significantly associated with the genomic abundance of imperfect microsatellites. Previously, it has been shown that longer microsatellites (&gt;15 repeats) are associated with mutational bias that reduces the length of microsatellites.<sup><a reveal-id="DSU036C44" data-open="DSU036C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Hence, the older mutations are more likely to be deleted from the longer microsatellites, causing a paucity of long microsatellites in the genome. Although the length of microsatellites is known to have a role in the mutational activities of microsatellites,<sup><a reveal-id="DSU036C45" data-open="DSU036C45" class="link link-ref link-reveal xref-bibr">45</a>,<a reveal-id="DSU036C46" data-open="DSU036C46" class="link link-ref link-reveal xref-bibr">46</a></sup> results from the current investigation show that sequence imperfection of motifs is a key determinant of abundance of imperfect microsatellites in the genome. Such an evolutionary link is not specific to insects, but is also observed in unrelated eukaryotes including yeast, worm and mammals. Consistent with this theory, the perfect microsatellites are found more abundantly than the imperfect microsatellites in each genome (Table <a reveal-id="DSU036TB1" data-open="DSU036TB1" class="link link-reveal link-table xref-fig">1</a>). Moreover, it is known that mutations in microsatellite alleles shorter than a critical length favour repeat expansion, whereas mutations acting on longer alleles favour contraction of microsatellite repeats.<sup><a reveal-id="DSU036C47" data-open="DSU036C47" class="link link-ref link-reveal xref-bibr">47</a></sup> The intricacies between length and sequence mismatches may also be critical in maintaining imperfect repeats in the genome.<sup><a reveal-id="DSU036C21" data-open="DSU036C21" class="link link-ref link-reveal xref-bibr">21</a></sup> This is also indicated from comparison of genomic abundance of imperfect microsatellites among insect species based on different lengths of microsatellites (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). The posterior <em>t</em>-test <em>P</em>-values were significant for all the pair-wise comparisons among the 20 insect species. We, however, failed to observe such relationships among the non-insect species most probably due to the sample size (only four) used for analysis. Also, the large differences in the genome sequences of these species (worm, yeast, human and mouse) may account for the lack of significant relationships in imperfect microsatellites in the genomes.</p><p>Recently, many inbred lines of <em>D. melanogaster</em> were sequenced that generated a very useful resource of genetic variation data of fruit fly, and also provided new insights on the faster X hypothesis.<sup><a reveal-id="DSU036C26" data-open="DSU036C26" class="link link-ref link-reveal xref-bibr">26</a></sup> We used these data to identify imperfect microsatellites in multiple inbred lines. The data were used to classify the reference SNP alleles (dbSNP data) of imperfect microsatellites to understand how mutation affects sequence imperfection in microsatellites. The results of that analysis suggested that there is a significant relationship between motif imperfection and mutational patterns in the microsatellite loci. Our data showed that the extent of motif heterogeneity of imperfect microsatellites is significantly linked to the nature of allelic changes in microsatellites. When a new allele is introduced that differs in sequence from that of existing mismatches in the microsatellite, the extent of sequence heterogeneity increases. On the other hand, when the introduced alleles match to the pre-existing motif mismatches, it does no't affect the motif heterogeneity for which the locus is more likely to be maintained in the genome than a locus where motif heterogeneity builds up by mutation. If mutations continue in a microsatellite wherein the introduced alleles build up motif heterogeneity, over time the locus eventually becomes a non-microsatellite sequence. Our result shows that, depending on the introduced allele sequences, a single mutation can reduce motif mismatches by 56%. It is plausible that this relationship may be instrumental in regulating the overall slippage events of simple sequence repeats in the genome.</p><p>In <em>D. melanogaster</em>, we further observed that imperfect microsatellites are also localized within specific protein-coding genes (Fig. <a reveal-id="DSU036F5" data-open="DSU036F5" class="link link-reveal link-table xref-fig">5</a>). The microsatellite-associated SNP rs204392359 is localized in the intron of gene <em>stardust</em> (<em>sdt</em>). Similarly, the SNP rs205546548 is localized within a microsatellite in downstream region of gene <em>trf2</em> (TATA box-binding protein-related factor 2), and the SNP rs204306547 is localized within an intronic microsatellite of the gene <em>CG3655</em>. Furthermore, we estimated that alleles of these microsatellites originated within the last 150 yrs, suggesting that these imperfect microsatellites may be undergoing recent mutations in <em>D. melanogaster.</em> It is possible that these imperfect microsatellite and the associated mutations have a functional role in <em>D. melanogaster</em>, as increasing evidence now suggests that microsatellites are associated with functional and evolutionary roles in eukaryotic genomes.<sup><a reveal-id="DSU036C14" data-open="DSU036C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSU036C48" data-open="DSU036C48" class="link link-ref link-reveal xref-bibr">48</a></sup> Moreover, in <em>A. gambiae</em> genome, numerous protein-coding genes harbour mutation in microsatellite sequences within the respective genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>), indicating that motif imperfection of microsatellites may have extensive functional consequences in <em>A. gambiae</em>.</p><p>In conclusion, the current study is the first detailed investigation of imperfect microsatellites in insect species representing five taxonomic orders (Diptera, Hymenoptera, Lepidoptera, Coleoptera and Hemiptera). The results obtained from this investigation provide new insights into the evolution of sequence imperfection in microsatellite loci across genomes. The outcome of this study may have utility in assessing the role of motif imperfection of microsatellites in genome structure and function of insects.</p>                            <h2 scrollto-destination=6572881 id="6572881" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at</a> www.dnaresearch.oxfordjournals.org</p>                            <h2 scrollto-destination=6572883 id="6572883" class="section-title" data-legacy-id=s6><a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/lookup/suppl/doi:10.1093/dnares/dsu036/-/DC1" target="_blank">Funding</a></h2>
<p>This work was supported from grants RO1-AI079125-A1, RO1-AI081795 and R21-AI101345 from the National Institute of Allergy and Infectious Diseases, National Institutes of Health, USA. Funding to pay the Open Access publication charges for this article was provided by the University of Notre Dame.</p>                            <h2 scrollto-destination=6572885 id="6572885" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors are thankful to Paul Hickner and Joanne Cunningham for critically reading the manuscript.</p>                            <h2 scrollto-destination=6572887 id="6572887" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSU036C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tautz</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Hypervariability of simple sequences as a general source for polymorphic DNA markers</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1989</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">6463</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hypervariability%20of%20simple%20sequences%20as%20a%20general%20source%20for%20polymorphic%20DNA%20markers&amp;author=D.%20Tautz&amp;publication_year=1989&amp;journal=Nucleic%20Acids%20Res.&amp;volume=17&amp;pages=6463-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/17.16.6463" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F17.16.6463" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F17.16.6463"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2780284" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Weber</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Wong</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Mutation of human short tandem repeats</div>, <div class="source">Hum. Mol. Genet.</div> , <div class="year">1993</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">1123</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutation%20of%20human%20short%20tandem%20repeats&amp;author=J.L.%20Weber&amp;author=C.%20Wong&amp;publication_year=1993&amp;journal=Hum.%20Mol.%20Genet.&amp;volume=2&amp;pages=1123-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/hmg/2.8.1123" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fhmg%2F2.8.1123" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fhmg%2F2.8.1123"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8401493" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kruglyak</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Durrett</div> <div class="given-names">R.T.</div></div>,  <div class="name"><div class="surname">Schug</div> <div class="given-names">M.D.</div></div>,  <div class="name"><div class="surname">Aquadro</div> <div class="given-names">C.F.</div></div>. </div><div class="article-title">Equilibrium distributions of microsatellite repeat length resulting from a balance between slippage events and point mutations</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1998</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">10774</div>-<div class="lpage">78</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Equilibrium%20distributions%20of%20microsatellite%20repeat%20length%20resulting%20from%20a%20balance%20between%20slippage%20events%20and%20point%20mutations&amp;author=S.%20Kruglyak&amp;author=R.T.%20Durrett&amp;author=M.D.%20Schug&amp;author=C.F.%20Aquadro&amp;publication_year=1998&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=10774-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.18.10774" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.18.10774" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.18.10774"> </span></div> </div></div></div></div></div><div content-id="DSU036C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Goldstein</div> <div class="given-names">D.B.</div></div>,  <div class="name"><div class="surname">Schlötterer</div> <div class="given-names">C.</div></div>. </div>, <div class="source">Microsatellites: evolution and applications</div> , <div class="year">1999</div><div class="edition">1st edition</div><div class="publisher-loc">Oxford, UK</div><div class="publisher-name">Oxford University Press</div></div></div></div></div><div content-id="DSU036C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lai</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Sun</div> <div class="given-names">F.</div></div>. </div><div class="article-title">The relationship between microsatellite slippage mutation rate and the number of repeat units</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2003</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">2123</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20relationship%20between%20microsatellite%20slippage%20mutation%20rate%20and%20the%20number%20of%20repeat%20units&amp;author=Y.%20Lai&amp;author=F.%20Sun&amp;publication_year=2003&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=20&amp;pages=2123-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msg228" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsg228" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsg228"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12949124" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Levinson</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Gutman</div> <div class="given-names">G.A.</div></div>. </div><div class="article-title">Slipped-strand mispairing: a major mechanism for DNA sequence evolution</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1987</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">203</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Slipped-strand%20mispairing%3A%20a%20major%20mechanism%20for%20DNA%20sequence%20evolution&amp;author=G.%20Levinson&amp;author=G.A.%20Gutman&amp;publication_year=1987&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=4&amp;pages=203-21" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3328815" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schlötterer</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Tautz</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Slippage synthesis of simple sequence DNA</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1992</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">211</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Slippage%20synthesis%20of%20simple%20sequence%20DNA&amp;author=C.%20Schl%C3%B6tterer&amp;author=D.%20Tautz&amp;publication_year=1992&amp;journal=Nucleic%20Acids%20Res.&amp;volume=20&amp;pages=211-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/20.2.211" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F20.2.211" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F20.2.211"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1741246" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schlötterer</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Evolutionary dynamics of microsatellite DNA</div>, <div class="source">Chromosoma</div> , <div class="year">2000</div>, vol. <div class="volume">109</div> (pg. <div class="fpage">365</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20dynamics%20of%20microsatellite%20DNA&amp;author=C.%20Schl%C3%B6tterer&amp;publication_year=2000&amp;journal=Chromosoma&amp;volume=109&amp;pages=365-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s004120000089" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004120000089" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs004120000089"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11072791" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Estoup</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Jarne</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Cornuet</div> <div class="given-names">J.M.</div></div>. </div><div class="article-title">Homoplasy and mutation model at microsatellite loci and their consequences for population genetics analysis</div>, <div class="source">Mol. Ecol.</div> , <div class="year">2002</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1591</div>-<div class="lpage">604</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Homoplasy%20and%20mutation%20model%20at%20microsatellite%20loci%20and%20their%20consequences%20for%20population%20genetics%20analysis&amp;author=A.%20Estoup&amp;author=P.%20Jarne&amp;author=J.M.%20Cornuet&amp;publication_year=2002&amp;journal=Mol.%20Ecol.&amp;volume=11&amp;pages=1591-604" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-294X.2002.01576.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-294X.2002.01576.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-294X.2002.01576.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12207711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ellegren</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Microsatellites: simple sequences with complex evolution</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">435</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellites%3A%20simple%20sequences%20with%20complex%20evolution&amp;author=H.%20Ellegren&amp;publication_year=2004&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=5&amp;pages=435-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1348" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1348" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1348"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15153996" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wu</div> <div class="given-names">C.H.</div></div>,  <div class="name"><div class="surname">Drummond</div> <div class="given-names">A.J.</div></div>. </div><div class="article-title">Joint inference of microsatellite mutation models, population history and genealogies using transdimensional Markov Chain Monte Carlo</div>, <div class="source">Genetics</div> , <div class="year">2011</div>, vol. <div class="volume">188</div> (pg. <div class="fpage">151</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Joint%20inference%20of%20microsatellite%20mutation%20models%2C%20population%20history%20and%20genealogies%20using%20transdimensional%20Markov%20Chain%20Monte%20Carlo&amp;author=C.H.%20Wu&amp;author=A.J.%20Drummond&amp;publication_year=2011&amp;journal=Genetics&amp;volume=188&amp;pages=151-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.110.125260" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.110.125260" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.110.125260"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21385725" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kimura</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ohta</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Stepwise mutation model and distribution of allelic frequencies in a finite population</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1978</div>, vol. <div class="volume">75</div> (pg. <div class="fpage">2868</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stepwise%20mutation%20model%20and%20distribution%20of%20allelic%20frequencies%20in%20a%20finite%20population&amp;author=M.%20Kimura&amp;author=T.%20Ohta&amp;publication_year=1978&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=75&amp;pages=2868-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.75.6.2868" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.75.6.2868" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.75.6.2868"> </span></div> </div></div></div></div></div><div content-id="DSU036C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Di Rienzo</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Peterson</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">Garza</div> <div class="given-names">J.C.</div></div>, et al. </div><div class="article-title">Mutational processes of simple-sequence repeat loci in human populations</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1994</div>, vol. <div class="volume">91</div> (pg. <div class="fpage">3166</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutational%20processes%20of%20simple-sequence%20repeat%20loci%20in%20human%20populations&amp;author=A.%20Di%20Rienzo&amp;author=A.C.%20Peterson&amp;author=J.C.%20Garza&amp;publication_year=1994&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=91&amp;pages=3166-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.91.8.3166" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.91.8.3166" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.91.8.3166"> </span></div> </div></div></div></div></div><div content-id="DSU036C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">Y.C.</div></div>,  <div class="name"><div class="surname">Korol</div> <div class="given-names">A.B.</div></div>,  <div class="name"><div class="surname">Fahima</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Beiles</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Nevo</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Microsatellites: genomic distribution, putative functions and mutational mechanisms: a review</div>, <div class="source">Mol. Ecol.</div> , <div class="year">2002</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">2453</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellites%3A%20genomic%20distribution%2C%20putative%20functions%20and%20mutational%20mechanisms%3A%20a%20review&amp;author=Y.C.%20Li&amp;author=A.B.%20Korol&amp;author=T.%20Fahima&amp;author=A.%20Beiles&amp;author=E.%20Nevo&amp;publication_year=2002&amp;journal=Mol.%20Ecol.&amp;volume=11&amp;pages=2453-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-294X.2002.01643.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-294X.2002.01643.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-294X.2002.01643.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12453231" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kashi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">King</div> <div class="given-names">D.G.</div></div>. </div><div class="article-title">Simple sequence repeats as advantageous mutators in evolution</div>, <div class="source">Trends Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">253</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20sequence%20repeats%20as%20advantageous%20mutators%20in%20evolution&amp;author=Y.%20Kashi&amp;author=D.G.%20King&amp;publication_year=2006&amp;journal=Trends%20Genet.&amp;volume=22&amp;pages=253-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2006.03.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2006.03.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2006.03.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16567018" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Angers</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Bernatchez</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Complex evolution of a salmonid microsatellite locus and its consequences in inferring allelic divergence from size information</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1997</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">230</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complex%20evolution%20of%20a%20salmonid%20microsatellite%20locus%20and%20its%20consequences%20in%20inferring%20allelic%20divergence%20from%20size%20information&amp;author=B.%20Angers&amp;author=L.%20Bernatchez&amp;publication_year=1997&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=14&amp;pages=230-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025759" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a025759" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a025759"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9066791" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Slatkin</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Rannala</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Estimating the age of alleles by use of intraallelic variability</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="year">1997</div>, vol. <div class="volume">60</div> (pg. <div class="fpage">447</div>-<div class="lpage">58</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimating%20the%20age%20of%20alleles%20by%20use%20of%20intraallelic%20variability&amp;author=M.%20Slatkin&amp;author=B.%20Rannala&amp;publication_year=1997&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=60&amp;pages=447-58" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9012419" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bacon</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Farrington</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Dunlop</div> <div class="given-names">M.G.</div></div>. </div><div class="article-title">Sequence interruptions confer differential stability at microsatellite alleles in mismatch repair-deficient cells</div>, <div class="source">Hum. Mol. Genet.</div> , <div class="year">2000</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">2707</div>-<div class="lpage">13</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20interruptions%20confer%20differential%20stability%20at%20microsatellite%20alleles%20in%20mismatch%20repair-deficient%20cells&amp;author=A.L.%20Bacon&amp;author=S.M.%20Farrington&amp;author=M.G.%20Dunlop&amp;publication_year=2000&amp;journal=Hum.%20Mol.%20Genet.&amp;volume=9&amp;pages=2707-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/hmg/9.18.2707" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fhmg%2F9.18.2707" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fhmg%2F9.18.2707"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11063729" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chiba-Falek</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Touchman</div> <div class="given-names">J.W.</div></div>,  <div class="name"><div class="surname">Nussbaum</div> <div class="given-names">R.L.</div></div>. </div><div class="article-title">Functional analysis of intra-allelic variation at NACP-Rep1 in the alpha-synuclein gene</div>, <div class="source">Hum. Genet.</div> , <div class="year">2003</div>, vol. <div class="volume">113</div> (pg. <div class="fpage">426</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20analysis%20of%20intra-allelic%20variation%20at%20NACP-Rep1%20in%20the%20alpha-synuclein%20gene&amp;author=O.%20Chiba-Falek&amp;author=J.W.%20Touchman&amp;author=R.L.%20Nussbaum&amp;publication_year=2003&amp;journal=Hum.%20Genet.&amp;volume=113&amp;pages=426-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00439-003-1002-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00439-003-1002-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00439-003-1002-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12923682" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Coelho</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Luiselli</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Bertorelle</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">Microsatellite variation and evolution of human lactase persistence</div>, <div class="source">Hum. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">117</div> (pg. <div class="fpage">329</div>-<div class="lpage">39</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellite%20variation%20and%20evolution%20of%20human%20lactase%20persistence&amp;author=M.%20Coelho&amp;author=D.%20Luiselli&amp;author=G.%20Bertorelle&amp;publication_year=2005&amp;journal=Hum.%20Genet.&amp;volume=117&amp;pages=329-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00439-005-1322-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00439-005-1322-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00439-005-1322-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15928901" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Keirle</div> <div class="given-names">M.R.</div></div>,  <div class="name"><div class="surname">Avis</div> <div class="given-names">P.G.</div></div>,  <div class="name"><div class="surname">Feldheim</div> <div class="given-names">K.A.</div></div>,  <div class="name"><div class="surname">Hemmes</div> <div class="given-names">D.E.</div></div>,  <div class="name"><div class="surname">Mueller</div> <div class="given-names">G.M.</div></div>. </div><div class="article-title">Investigating the allelic evolution of an imperfect microsatellite locus in the Hawaiian mushroom <em>Rhodocollybia laulaha</em></div>, <div class="source">J. Hered.</div> , <div class="year">2011</div>, vol. <div class="volume">102</div> (pg. <div class="fpage">727</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Investigating%20the%20allelic%20evolution%20of%20an%20imperfect%20microsatellite%20locus%20in%20the%20Hawaiian%20mushroom%20Rhodocollybia%20laulaha&amp;author=M.R.%20Keirle&amp;author=P.G.%20Avis&amp;author=K.A.%20Feldheim&amp;author=D.E.%20Hemmes&amp;author=G.M.%20Mueller&amp;publication_year=2011&amp;journal=J.%20Hered.&amp;volume=102&amp;pages=727-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/esr099" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2Fesr099" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2Fesr099"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21914665" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Britt-Compton</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Baird</div> <div class="given-names">D.M.</div></div>. </div><div class="article-title">Intra-allelic mutation at human telomeres</div>, <div class="source">Biochem. Soc. Trans.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">581</div>-<div class="lpage">2</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intra-allelic%20mutation%20at%20human%20telomeres&amp;author=B.%20Britt-Compton&amp;author=D.M.%20Baird&amp;publication_year=2006&amp;journal=Biochem.%20Soc.%20Trans.&amp;volume=34&amp;pages=581-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/BST0340581" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2FBST0340581" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2FBST0340581"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16856866" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kofler</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Schlötterer</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Lelley</div> <div class="given-names">T.</div></div>. </div><div class="article-title">SciRoKo: a new tool for whole genome microsatellite search and investigation</div>, <div class="source">Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">1683</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SciRoKo%3A%20a%20new%20tool%20for%20whole%20genome%20microsatellite%20search%20and%20investigation&amp;author=R.%20Kofler&amp;author=C.%20Schl%C3%B6tterer&amp;author=T.%20Lelley&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=1683-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm157" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm157" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm157"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17463017" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Anderson</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Robinson</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Generalised discriminant analysis based on distances</div>, <div class="source">Aust. N. Z. J. Stat.</div> , <div class="year">2003</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">301</div>-<div class="lpage">18</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Generalised%20discriminant%20analysis%20based%20on%20distances&amp;author=M.J.%20Anderson&amp;author=J.%20Robinson&amp;publication_year=2003&amp;journal=Aust.%20N.%20Z.%20J.%20Stat.&amp;volume=45&amp;pages=301-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/anzs.2003.45.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fanzs.2003.45.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fanzs.2003.45.issue-3"> </span></div> </div></div></div></div></div><div content-id="DSU036C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Anderson</div> <div class="given-names">M.J.</div></div>. </div><div class="article-title">A new method for non-parametric multivariate analysis of variance</div>, <div class="source">Aust. Ecol.</div> , <div class="year">2001</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">32</div>-<div class="lpage">46</div>)</div></div></div></div><div content-id="DSU036C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mackay</div> <div class="given-names">T.F.</div></div>,  <div class="name"><div class="surname">Richards</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Stone</div> <div class="given-names">E.A.</div></div>, et al. </div><div class="article-title">The <em>Drosophila melanogaster</em> genetic reference panel</div>, <div class="source">Nature</div> , <div class="year">2012</div>, vol. <div class="volume">482</div> (pg. <div class="fpage">173</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Drosophila%20melanogaster%20genetic%20reference%20panel&amp;author=T.F.%20Mackay&amp;author=S.%20Richards&amp;author=E.A.%20Stone&amp;publication_year=2012&amp;journal=Nature&amp;volume=482&amp;pages=173-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10811" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10811" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10811"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22318601" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fondon</div> <div class="given-names">J.W.</div>III</div>,  <div class="name"><div class="surname">Martin</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Richards</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gibbs</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">Mittelman</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Analysis of microsatellite variation in <em>Drosophila melanogaster</em> with population-scale genome sequencing</div>, <div class="source">PLoS ONE</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e33036</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20microsatellite%20variation%20in%20Drosophila%20melanogaster%20with%20population-scale%20genome%20sequencing&amp;author=J.W.%20Fondon&amp;author=A.%20Martin&amp;author=S.%20Richards&amp;author=R.A.%20Gibbs&amp;author=D.%20Mittelman&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e33036" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0033036" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0033036" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0033036"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22427938" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hall</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Frank</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Holmes</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Pfahringer</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Reutemann</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Witten</div> <div class="given-names">I.H.</div></div>. </div><div class="article-title">The WEKA data mining software: an update</div>, <div class="source">SIGKDD Exploration</div> , <div class="year">2009</div>, vol. <div class="volume">11</div> pg. <div class="fpage">1</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20WEKA%20data%20mining%20software%3A%20an%20update&amp;author=M.%20Hall&amp;author=E.%20Frank&amp;author=G.%20Holmes&amp;author=B.%20Pfahringer&amp;author=P.%20Reutemann&amp;author=I.H.%20Witten&amp;publication_year=2009&amp;journal=SIGKDD%20Exploration&amp;volume=11&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1145/1656274" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1145%2F1656274" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1145%2F1656274"> </span></div> </div></div></div></div></div><div content-id="DSU036C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Dudley</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>. </div><div class="article-title">MEGA4: molecular evolutionary genetics analysis (MEGA) software version 4.0</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">1596</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA4%3A%20molecular%20evolutionary%20genetics%20analysis%20(MEGA)%20software%20version%204.0&amp;author=K.%20Tamura&amp;author=J.%20Dudley&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=24&amp;pages=1596-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm092" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm092" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm092"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17488738" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Librado</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Rozas</div> <div class="given-names">J.</div></div>. </div><div class="article-title">DnaSP v5: a software for comprehensive analysis of DNA polymorphism data</div>, <div class="source">Bioinformatics</div> , <div class="year">2009</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">1451</div>-<div class="lpage">2</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DnaSP%20v5%3A%20a%20software%20for%20comprehensive%20analysis%20of%20DNA%20polymorphism%20data&amp;author=P.%20Librado&amp;author=J.%20Rozas&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1451-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp187" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp187" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp187"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19346325" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Behura</div> <div class="given-names">S.K.</div></div>. </div><div class="article-title">Molecular marker systems in insects: current trends and future avenues</div>, <div class="source">Mol. Ecol.</div> , <div class="year">2006</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">3087</div>-<div class="lpage">113</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20marker%20systems%20in%20insects%3A%20current%20trends%20and%20future%20avenues&amp;author=S.K.%20Behura&amp;publication_year=2006&amp;journal=Mol.%20Ecol.&amp;volume=15&amp;pages=3087-113" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.2006.15.issue-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.2006.15.issue-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.2006.15.issue-11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16968257" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Santana</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Coetzee</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Steenkamp</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Microsatellite discovery by deep sequencing of enriched genomic libraries</div>, <div class="source">Biotechniques</div> , <div class="year">2009</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">217</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellite%20discovery%20by%20deep%20sequencing%20of%20enriched%20genomic%20libraries&amp;author=Q.%20Santana&amp;author=M.%20Coetzee&amp;author=E.%20Steenkamp&amp;publication_year=2009&amp;journal=Biotechniques&amp;volume=46&amp;pages=217-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2144/000113085" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2144%2F000113085" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2144%2F000113085"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19317665" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Webster</div> <div class="given-names">M.T.</div></div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">N.G.</div></div>,  <div class="name"><div class="surname">Ellegren</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Microsatellite evolution inferred from human-chimpanzee genomic sequence alignments</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2002</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">8748</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellite%20evolution%20inferred%20from%20human-chimpanzee%20genomic%20sequence%20alignments&amp;author=M.T.%20Webster&amp;author=N.G.%20Smith&amp;author=H.%20Ellegren&amp;publication_year=2002&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=99&amp;pages=8748-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.122067599" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.122067599" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.122067599"> </span></div> </div></div></div></div></div><div content-id="DSU036C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huntley</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Clark</div> <div class="given-names">A.G.</div></div>. </div><div class="article-title">Evolutionary analysis of amino acid repeats across the genomes of 12 Drosophila species</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">2598</div>-<div class="lpage">609</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20analysis%20of%20amino%20acid%20repeats%20across%20the%20genomes%20of%2012%20Drosophila%20species&amp;author=M.A.%20Huntley&amp;author=A.G.%20Clark&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=24&amp;pages=2598-609" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm129" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm129" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm129"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17602168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gao</div> <div class="given-names">L.Z.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Comparisons of mutation rate variation at genome-wide microsatellites: evolutionary insights from two cultivated rice and their wild relatives</div>, <div class="source">BMC Evol. Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">8</div> pg. <div class="fpage">11</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparisons%20of%20mutation%20rate%20variation%20at%20genome-wide%20microsatellites%3A%20evolutionary%20insights%20from%20two%20cultivated%20rice%20and%20their%20wild%20relatives&amp;author=L.Z.%20Gao&amp;author=H.%20Xu&amp;publication_year=2008&amp;journal=BMC%20Evol.%20Biol.&amp;volume=8&amp;pages=11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-8-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-8-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-8-11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18199337" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Behura</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Severson</div> <div class="given-names">D.W.</div></div>. </div><div class="article-title">Genome-wide comparative analysis of simple sequence coding repeats among 25 insect species</div>, <div class="source">Gene</div> , <div class="year">2012</div>, vol. <div class="volume">504</div> (pg. <div class="fpage">226</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20comparative%20analysis%20of%20simple%20sequence%20coding%20repeats%20among%2025%20insect%20species&amp;author=S.K.%20Behura&amp;author=D.W.%20Severson&amp;publication_year=2012&amp;journal=Gene&amp;volume=504&amp;pages=226-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2012.05.020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2012.05.020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2012.05.020"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22633877" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Burge</div> <div class="given-names">D.O.</div></div>,  <div class="name"><div class="surname">Jui</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Kembel</div> <div class="given-names">S.S.</div></div>,  <div class="name"><div class="surname">Zhukovsky</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Microsatellite markers from <em>Ceanothus roderickii</em> (Rhamnaceae) using next-generation sequencing technology</div>, <div class="source">Am. J. Bot.</div> , <div class="year">2012</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">e127</div>-<div class="lpage">130</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellite%20markers%20from%20Ceanothus%20roderickii%20(Rhamnaceae)%20using%20next-generation%20sequencing%20technology&amp;author=D.O.%20Burge&amp;author=G.%20Jui&amp;author=S.S.%20Kembel&amp;author=K.%20Zhukovsky&amp;publication_year=2012&amp;journal=Am.%20J.%20Bot.&amp;volume=99&amp;pages=e127-130" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3732/ajb.1100431" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3732%2Fajb.1100431" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3732%2Fajb.1100431"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22378832" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Han</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">VanBuren</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Genetic linkage maps for Asian and American lotus constructed using novel SSR markers derived from the genome of sequenced cultivar</div>, <div class="source">BMC Genomics</div> , <div class="year">2012</div>, vol. <div class="volume">13</div> pg. <div class="fpage">653</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20linkage%20maps%20for%20Asian%20and%20American%20lotus%20constructed%20using%20novel%20SSR%20markers%20derived%20from%20the%20genome%20of%20sequenced%20cultivar&amp;author=M.%20Yang&amp;author=Y.%20Han&amp;author=R.%20VanBuren&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=653" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-653" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-653" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-653"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23170872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zalapa</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Cuevas</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Using next-generation sequencing approaches to isolate simple sequence repeat (SSR) loci in the plant sciences</div>, <div class="source">Am. J. Bot.</div> , <div class="year">2012</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">193</div>-<div class="lpage">208</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20next-generation%20sequencing%20approaches%20to%20isolate%20simple%20sequence%20repeat%20(SSR)%20loci%20in%20the%20plant%20sciences&amp;author=J.E.%20Zalapa&amp;author=H.%20Cuevas&amp;author=H.%20Zhu&amp;publication_year=2012&amp;journal=Am.%20J.%20Bot.&amp;volume=99&amp;pages=193-208" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3732/ajb.1100394" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3732%2Fajb.1100394" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3732%2Fajb.1100394"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22186186" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mudunuri</div> <div class="given-names">S.B.</div></div>,  <div class="name"><div class="surname">Nagarajaram</div> <div class="given-names">H.A.</div></div>. </div><div class="article-title">IMEx: Imperfect Microsatellite Extractor</div>, <div class="source">Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">1181</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=IMEx%3A%20Imperfect%20Microsatellite%20Extractor&amp;author=S.B.%20Mudunuri&amp;author=H.A.%20Nagarajaram&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=1181-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm097" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm097" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm097"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17379689" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ross</div> <div class="given-names">C.L.</div></div>,  <div class="name"><div class="surname">Dyer</div> <div class="given-names">K.A.</div></div>,  <div class="name"><div class="surname">Erez</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Jaenike</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Markow</div> <div class="given-names">T.A.</div></div>. </div><div class="article-title">Rapid divergence of microsatellite abundance among species of Drosophila</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2003</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1143</div>-<div class="lpage">57</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20divergence%20of%20microsatellite%20abundance%20among%20species%20of%20Drosophila&amp;author=C.L.%20Ross&amp;author=K.A.%20Dyer&amp;author=T.%20Erez&amp;author=S.J.%20Miller&amp;author=J.%20Jaenike&amp;author=T.A.%20Markow&amp;publication_year=2003&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=20&amp;pages=1143-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msg137" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsg137" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsg137"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12777536" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Charlesworth</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Coyne</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Barton</div> <div class="given-names">N.</div></div>. </div><div class="article-title">The relative rates of evolution of sex chromosomes and autosomes</div>, <div class="source">Am. Nat.</div> , <div class="year">1987</div>, vol. <div class="volume">130</div> (pg. <div class="fpage">113</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20relative%20rates%20of%20evolution%20of%20sex%20chromosomes%20and%20autosomes&amp;author=B.%20Charlesworth&amp;author=J.A.%20Coyne&amp;author=N.%20Barton&amp;publication_year=1987&amp;journal=Am.%20Nat.&amp;volume=130&amp;pages=113-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/284701" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F284701" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F284701"> </span></div> </div></div></div></div></div><div content-id="DSU036C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Betancourt</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Presgraves</div> <div class="given-names">D.C.</div></div>,  <div class="name"><div class="surname">Swanson</div> <div class="given-names">W.J.</div></div>. </div><div class="article-title">A test for faster X evolution in Drosophila</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2002</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">1816</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20test%20for%20faster%20X%20evolution%20in%20Drosophila&amp;author=A.J.%20Betancourt&amp;author=D.C.%20Presgraves&amp;author=W.J.%20Swanson&amp;publication_year=2002&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=19&amp;pages=1816-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a004006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a004006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a004006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12270910" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Harr</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Schlötterer</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Long microsatellite alleles in <em>Drosophila melanogaster</em> have a downward mutation bias and short persistence times, which cause their genome-wide underrepresentation</div>, <div class="source">Genetics</div> , <div class="year">2000</div>, vol. <div class="volume">155</div> (pg. <div class="fpage">1213</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Long%20microsatellite%20alleles%20in%20Drosophila%20melanogaster%20have%20a%20downward%20mutation%20bias%20and%20short%20persistence%20times%2C%20which%20cause%20their%20genome-wide%20underrepresentation&amp;author=B.%20Harr&amp;author=C.%20Schl%C3%B6tterer&amp;publication_year=2000&amp;journal=Genetics&amp;volume=155&amp;pages=1213-20" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10880482" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Falush</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Iwasa</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Size-dependent mutability and microsatellite constraints</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1999</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">960</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Size-dependent%20mutability%20and%20microsatellite%20constraints&amp;author=D.%20Falush&amp;author=Y.%20Iwasa&amp;publication_year=1999&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=16&amp;pages=960-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026185" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a026185" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a026185"> </span></div> </div></div></div></div></div><div content-id="DSU036C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schlötterer</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Harr</div> <div class="given-names">B.</div></div>. </div><div class="article-title"><em>Drosophila virilis</em> has long and highly polymorphic microsatellites</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2000</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">1641</div>-<div class="lpage">11646</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Drosophila%20virilis%20has%20long%20and%20highly%20polymorphic%20microsatellites&amp;author=C.%20Schl%C3%B6tterer&amp;author=B.%20Harr&amp;publication_year=2000&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=17&amp;pages=1641-11646" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026263" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a026263" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a026263"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11070052" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Peng</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Fang</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">The direction of microsatellite mutations is dependent upon allele length</div>, <div class="source">Nat. Genet.</div> , <div class="year">2000</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">396</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20direction%20of%20microsatellite%20mutations%20is%20dependent%20upon%20allele%20length&amp;author=X.%20Xu&amp;author=M.%20Peng&amp;author=Z.%20Fang&amp;publication_year=2000&amp;journal=Nat.%20Genet.&amp;volume=24&amp;pages=396-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/74238" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F74238" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F74238"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10742105" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU036C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU036C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fondon</div> <div class="given-names">J.W.</div>III</div>,  <div class="name"><div class="surname">Hammock</div> <div class="given-names">E.A.</div></div>,  <div class="name"><div class="surname">Hannan</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">King</div> <div class="given-names">D.G.</div></div>. </div><div class="article-title">Simple sequence repeats: genetic modulators of brain function and behavior</div>, <div class="source">Trends Neurosci.</div> , <div class="year">2008</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">328</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20sequence%20repeats%3A%20genetic%20modulators%20of%20brain%20function%20and%20behavior&amp;author=J.W.%20Fondon&amp;author=E.A.%20Hammock&amp;author=A.J.%20Hannan&amp;author=D.G.%20King&amp;publication_year=2008&amp;journal=Trends%20Neurosci.&amp;volume=31&amp;pages=328-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tins.2008.03.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tins.2008.03.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tins.2008.03.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18550185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=440058&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/dsu036-dsu036supp_table9.xlsx?Expires=2147483647&amp;Signature=cYuVtxYZ5ORdsMh7YyGI8tbKC9BxOnXgEStY~6m5gYQI4UKP0c0Q3v2eGtI96d5kgosVV9P9Mbgbt0C0IJ6hMpmNRqojCtRFCBgOGHMyBcCd5l2gzKghMizd9JBseBhwEdUjLZbMl~c~nX9uQ786ilRA7YJ1KgU3gsghSLlq8yijF2eA9GHReYZ3l57RLzkgcvYvoWyopNkKCaEEdMqna~rGTO4rDfwdodWKiekLQssvKGvQN8O7MmmQ9ZjVfx46avkPARD34gjeyxnOPzRk~nIp2NECORMFSBgHYly6ivnU84ArWGiG5ipGvfFpcDI76nxPfKGk5D3fWAhF24sZXw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-10</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/dsu036-dsu036supp_table8.docx?Expires=2147483647&amp;Signature=tWhXSGijuyOlhoNgKjx9Whm3jnGQ-VssYFIXR6boZSs6ZbsO16FHU6~qFiqq2i2jogKsyqWt-gs2-806ie2xiCDeP7Jud7G95vlsdgeTuOLu8GbD5nGTGe93majjvQQXeQ6bVeBmSx9hzybof8nylfCBoz9keol4vkUL2h1NC3Siir50gxufBfoeeSb-xZsjjp4bByzdsWzVbjPFs4i1SZUJ2vw8MD71SPzu7OFRSW3CFhHEDE4V5oZoU-~ZOpvcvGLg11cgoCGWMSMFPiELobKZdTgP1r9dUaoaz0ItxYupwEUuK8y7VUZ5pIia1FVn4hYPSnAo9E~UNes0G46rMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-9</a> - docx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/dsu036-dsu036supp_table7.docx?Expires=2147483647&amp;Signature=qrxeDDc0c1TZMiY3zjhrJqpCciWD7zcoFRU-lrT95sVSecj2XxMeySHANLDkVSY6ckf7Za4pVSVCe7WvPB4DnBvL1Z~rLneMM6odz6SAUZQY0DQ65FtAbXLXogOOK~LV05ZFQgnUbttiZAoqgi2kFmdtrlnfT-WOGF-ZSSCboORJ6arew48EpL5eeoTQExmgoHHupb-nJyBUyf7b5I7YBZzbXVhPUPDt~qjSWgyKVEFv7bzVpOinAtMz9FaU5TJqVPOagqF~j-v98MGEK4SJ4S6GjjsbcGeboYfTGbwQopdYM6-IN~X-i3nCp2cvDmmxBWrhzU0hhxgIwoUwSPaPdQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - docx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/dsu036-dsu036supp_table6.xlsx?Expires=2147483647&amp;Signature=yZt7SZabNXoj6TaDR3HbDdIMqIutKItRVwqQJollXos5lSBnAfNYnKE1H~fPeS-WG6YA9-09IfW-8fE3kKYd1VEugGJYJV2aLyy1nIwQHPNllbqGQIxUUzAgY1QYoaLLj5-dTPZ5ToDPx68XlPQzqzqeEccBpGz7VCxN8w9hQXMGS6DH53XdDpVQ09GCZy0Yt6XI9nfv~GTwJarypafQJyeWa3u81rquuhDHoZ5bZPTZtsixXAvnf9iFirYLXa2JMS4zsdwAikngui-yk7HIq0YHxs2p3zB~apipMgI4BgDXpG1-0c4OwT6QGv96VVDKWo2k63BoIke~lUUEDZxONQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/dsu036-dsu036supp_table5.docx?Expires=2147483647&amp;Signature=Inm2cfP7jQJwCuj0~QVbAGXnc2iHQguRhGFzmobME3pe9AuS~9xG8e-CKFarzLSdPdEfoEoJtexU-zM4JWX3HVi-zBaXkYuy5A9zWIyLb-bcCWJe5cyY649L2GoSMAxaqF~pBiqNNF3eKPGJEy4U9K-sBMuMCSln8JobF9I8MfUJ7B0UA9Jb02ZjkvbRBouRaxGOZSTf2d87FsT9HbGN6u48jnX~owdo1DaPBIPyueoQJ86aOujO~~3YF8d28F2dtvLDROjSPXCwTG-1Qdlb9qHx~VlzYAeIr~DKrdAEUbND2SBQTKDuvtowK9KJJzx3x0evxaEodfHDULaOO-1eIA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - docx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/dsu036-dsu036supp_table4.docx?Expires=2147483647&amp;Signature=x29eahuIlwDkMUkgMkczGUQ4wVOZvH1XgRwb0~9Ij4PV~JWcU4wE38-rfszYqNh1QmhFdOWuz~7wY4EV99pRaM3pMMMSnMDnmjoRzCSnU62vj8Y0QFlKYCb1HOrdoKeYkMn5d10IUoESTKBUg~OaF5D6~NREoCj7CXg7M5cP9j3mB0MRxzmaztu0C-IzxgyQ-UPkxhR~LfQ~tJY5-zy-cBUFmClaXcGIUUvOkgE0qtJzmz4l8twzzST13H8OCc9JugWnLYSu4ZZeu59pqGKxPoF4zgFv3uOYe3nNFp6CbA6pfDTBoexS5RZjyIbOxOli4hHJdc~lhqWb0pjwRGntMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - docx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/dsu036-dsu036supp_table3.docx?Expires=2147483647&amp;Signature=LTPmn96Si07zQno0pCOxrTIQnysALtLjiMca7IyDTq1gViFeao2HutLe3w-sP7DP09w4jFvfMx4PjdK180RLhoh5-mSE~a0l5zxb3gPgNmdMW9FkVUfeIM1isAJhqwLnDNdRsh0Vc52n80VsngV1BWhl8d6xsdfbfT6w5IxatM9qXA~Tvg22DVFXCVeCRH8Q9KDBFwaaeK8W-qc790lUN9gFo9GFj5OKHSj2nQeFZifhuenaejMclty8EIaHQNO~nvuvU2NI7Pz9epe5U~waoOs5a66GJyalkJRzsWQ-cSu66H8qZeeEOSLXqam-m4XT0zw2ry8CbyhKlLyw~1hrIw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - docx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/dsu036-dsu036supp_table2.docx?Expires=2147483647&amp;Signature=AOqN~3umPBUGhp99UUDDqNONnxeUMAMPEWWkpXUaUAxN6apggFeUwo7DAW4tLx1ta27jHO0R4WEKqLl2sq8FTI7kZ4torm6yjb91y4ImSJoVqJ3Jp797v1E5zhydsbPofr7sPrjsPmSKx-pC2AxnfviPqNAIiEGnpcKFygPZPgKv2BLNfS6-cdzBGhTGQ0PzhLdZiJHo9-HKJF3NvxUaQX1CHRKv7KDGof~aYEJkM1k4XTjMawUf1Q8SQZPc8TLB8pToj4WCPargatuo~bjp9bf2BvYUtppw0f3MOkN4O4EotVX0ZvLr-pSiwMer8Rmlbgk8eoRLuOvsuCunwe1vGw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - docx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/dsu036-dsu036supp_table1.docx?Expires=2147483647&amp;Signature=Y2pKOOnxgt7J3dkNvoddIuNrRWtCebAdZH9ZgWVH-4zLHmSmbfjPyIXCmdOpRsbGC5mmRHJJPy~~Up9veEhnIQE46yptLtcPSO~WXjfzgG9psa0eip9qTrKrx7Yk1DMdFQiFw85eXj~sF3VU1YjiRbR6lxzZ~4-WxLJ~vleDODSji-qK4z7vdD5cOOu1zKNJEBJF-NCNG9Y54LcaSsv~Pmi4Rykeh-qDFXNWZOjYAOa4KlA7YG6F1s1ay4~W5uIhGfI2NN8l~eRDKeep19TE5k~2uwsaREoutbEoFojFi4z0Jv~87pb9VpwheBzVvvmaQ-zgJQned5QR0ZfnF0sIfA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - docx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu036/2/dsu036-dsu036supp_figure1.png?Expires=2147483647&amp;Signature=t7rKavlu--93fp-R9MZYXpTVy03RZ~OmeN4ZJ-tkrz6YwHY6v7L-g~cMBFR0yBM7AwL-XcmO9ifinpGyS7AKktoMnwVoYLsaWgORzQk1ktJNsGy6ZXNwuOAKrfbpd1ag35Mst~kOUrmkRsuLvyFu~YwrLCRoMsUl0o2CS549HN3m7EOFYM2Xdi0G-xCO9xdhorK0MWQX9un4e9DEeShPW1mL-8euavEx7tGQgnsw9ZrnWq68cd2YHb6fm9lVq1jiD7GT9WqnVQbuui10DpUj1bAXI70UxUj63SqGX~9WqXMfjUuEF3fHD57wJ5iqz9GuoX3TUJbwfVT9STaOtxoQAw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - png file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">293</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000350442100004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">7</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsu036"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="440058"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000350442100004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f1%2f29%2f440058">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000350442100004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (7)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F1%2F29%2F440058" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/440058'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Motif mismatches in microsatellites: insights from genome-wide investigation among 20 insect species" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsu036");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f29%2f440058">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

