

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/1/25/4161430","@type":"ScholarlyArticle","name":"Genome sequencing and comparative genomics reveal the potential pathogenic mechanism of  Cercospora sojina  Hara on soybean","datePublished":"2017-09-18","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2018-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx035","keywords":["<italic>Cercospora sojina</italic>","soybean","genome","pathogenicity"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/1/25/4161430","author":[{"name":"Luo, Xuming","affiliation":"State Key Laboratory of Plant Genomics, Institute of Microbiology, Chinese Academy of Sciences, Beijing 100101, China  University of Chinese Academy of Sciences, Beijing 100049, China","@type":"Person"},{"name":"Cao, Jidong","affiliation":"State Key Laboratory of Plant Genomics, Institute of Microbiology, Chinese Academy of Sciences, Beijing 100101, China","@type":"Person"},{"name":"Huang, Junkai","affiliation":"State Key Laboratory of Plant Genomics, Institute of Microbiology, Chinese Academy of Sciences, Beijing 100101, China  University of Chinese Academy of Sciences, Beijing 100049, China","@type":"Person"},{"name":"Wang, Zongyi","affiliation":"Beijing Key Laboratory of Agricultural Product Detection and Control for Spoilage Organisms and Pesticides, Beijing University of Agriculture, Beijing 102206, China","@type":"Person"},{"name":"Guo, Zhengyan","affiliation":"State Key Laboratory of Microbial Resources, Institute of Microbiology, Chinese Academy of Sciences, Beijing 100101, China","@type":"Person"},{"name":"Chen, Yihua","affiliation":"State Key Laboratory of Microbial Resources, Institute of Microbiology, Chinese Academy of Sciences, Beijing 100101, China","@type":"Person"},{"name":"Ma, Shumei","affiliation":"Department of Plant Protection, College of Agriculture Resources and Environment, Heilongjiang University, Harbin 150080, China","@type":"Person"},{"name":"Liu, Jun","affiliation":"State Key Laboratory of Plant Genomics, Institute of Microbiology, Chinese Academy of Sciences, Beijing 100101, China  University of Chinese Academy of Sciences, Beijing 100049, China","@type":"Person"}],"description":"Abstract.  Frogeye leaf spot, caused by Cercospora sojina Hara, is a common disease of soybean in most soybean-growing countries of the world. In this study, we","pageStart":"25","pageEnd":"37","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome sequencing and comparative genomics reveal the potential pathogenic mechanism of  Cercospora sojina  Hara on soybean","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome sequencing and comparative genomics reveal the potential pathogenic mechanism of  Cercospora sojina  Hara on soybean" />
    <meta property="og:description" content="Abstract.  Frogeye leaf spot, caused by Cercospora sojina Hara, is a common disease of soybean in most soybean-growing countries of the world. In this study, we" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/1/25/4161430" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-09-18" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Luo, Xuming" /><meta name="citation_author_institution" content="State Key Laboratory of Plant Genomics, Institute of Microbiology, Chinese Academy of Sciences, Beijing 100101, China" /><meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing 100049, China" /><meta name="citation_author" content="Cao, Jidong" /><meta name="citation_author_institution" content="State Key Laboratory of Plant Genomics, Institute of Microbiology, Chinese Academy of Sciences, Beijing 100101, China" /><meta name="citation_author" content="Huang, Junkai" /><meta name="citation_author_institution" content="State Key Laboratory of Plant Genomics, Institute of Microbiology, Chinese Academy of Sciences, Beijing 100101, China" /><meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing 100049, China" /><meta name="citation_author" content="Wang, Zongyi" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Agricultural Product Detection and Control for Spoilage Organisms and Pesticides, Beijing University of Agriculture, Beijing 102206, China" /><meta name="citation_author" content="Guo, Zhengyan" /><meta name="citation_author_institution" content="State Key Laboratory of Microbial Resources, Institute of Microbiology, Chinese Academy of Sciences, Beijing 100101, China" /><meta name="citation_author" content="Chen, Yihua" /><meta name="citation_author_institution" content="State Key Laboratory of Microbial Resources, Institute of Microbiology, Chinese Academy of Sciences, Beijing 100101, China" /><meta name="citation_author" content="Ma, Shumei" /><meta name="citation_author_institution" content="Department of Plant Protection, College of Agriculture Resources and Environment, Heilongjiang University, Harbin 150080, China " /><meta name="citation_author" content="Liu, Jun" /><meta name="citation_author_institution" content="State Key Laboratory of Plant Genomics, Institute of Microbiology, Chinese Academy of Sciences, Beijing 100101, China" /><meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing 100049, China" /><meta name="citation_title" content="Genome sequencing and comparative genomics reveal the potential pathogenic mechanism of Cercospora sojina Hara on soybean" /><meta name="citation_firstpage" content="25" /><meta name="citation_lastpage" content="37" /><meta name="citation_doi" content="10.1093/dnares/dsx035" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2018/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Frogeye leaf spot of soybean: a review and proposed race designations for isolates of Cercospora sojina Hara,;  citation_journal_title=Crop Sci;  citation_year=2008;  citation_volume=48;  citation_pages=14-24; " /><meta name="citation_reference" content="citation_title=Sources of resistance in soybean (Glycine-max) to frog-eye leaf-spot caused by Cercosporidium-sojinum;  citation_journal_title=Indian J. Agric. Sci;  citation_year=1994;  citation_volume=64;  citation_pages=886-7; " /><meta name="citation_reference" content="citation_title=More Cercospora species infect soybeans across the Americas than meets the eye,;  citation_journal_title=PloS One;  citation_year=2015;  citation_volume=10;  citation_pages=e0133495" /><meta name="citation_reference" content="citation_title=Identification of soybean genotypes resistant to Cercospora sojina by field screening and molecular markers,;  citation_journal_title=Plant Dis;  citation_year=2009;  citation_volume=93;  citation_pages=408-11; " /><meta name="citation_reference" content="citation_title=Effective management of soybean rust and frogeye leaf spot using a mixture of flusilazole and carbendazim,;  citation_journal_title=Crop Prot;  citation_year=2008;  citation_volume=27;  citation_pages=566-71; " /><meta name="citation_reference" content="citation_title=Sensitivity of Cercospora sojina isolates to quinone outside inhibitor fungicides,;  citation_journal_title=Crop Prot;  citation_year=2012;  citation_volume=40;  citation_pages=63-8; " /><meta name="citation_reference" content="citation_title=Primary report on the identification for physiological races of Cercospora sojina Hara in northeast China;  citation_journal_title=Acta Phytopathol. Sin;  citation_year=1997;  citation_volume=27;  citation_pages=180" /><meta name="citation_reference" content="citation_title=The photoactivated Cercospora toxin cercosporin: contributions to plant disease and fundamental biology,;  citation_journal_title=Annu. Rev. Phytopathol;  citation_year=2000;  citation_volume=38;  citation_pages=461-90; " /><meta name="citation_reference" content="citation_title=Phylogenetic analysis of cercospora and mycosphaerella based on the internal transcribed spacer region of ribosomal DNA,;  citation_journal_title=Phytopathology;  citation_year=2001;  citation_volume=91;  citation_pages=648-58; " /><meta name="citation_reference" content="citation_title=The genome sequence of the rice blast fungus Magnaporthe grisea,;  citation_journal_title=Nature;  citation_year=2005;  citation_volume=434;  citation_pages=980-6; " /><meta name="citation_reference" content="citation_title=Immunity to rice blast disease by suppression of effector-triggered necrosis;  citation_journal_title=Curr. Biol;  citation_year=2016;  citation_volume=26;  citation_pages=2399-411; " /><meta name="citation_reference" content="citation_title=The Magnaporthe oryzae effector AvrPiz-t targets the RING E3 Ubiquitin Ligase APIP6 to suppress pathogen-associated molecular pattern–triggered immunity in rice;  citation_journal_title=Plant Cell;  citation_year=2012;  citation_volume=24;  citation_pages=4748-62; " /><meta name="citation_reference" content="citation_title=MoEnd3 regulates appressorium formation and virulence through mediating endocytosis in rice blast fungus Magnaporthe oryzae;  citation_journal_title=PLoS Pathogens;  citation_year=2017;  citation_volume=13;  citation_pages=e1006449" /><meta name="citation_reference" content="citation_title=MoSwi6, an APSES family transcription factor, interacts with MoMps1 and is required for hyphal and conidial morphogenesis, appressorial function and pathogenicity of Magnaporthe oryzae;  citation_journal_title=Mol. Plant Pathol;  citation_year=2012;  citation_volume=13;  citation_pages=677-89; " /><meta name="citation_reference" content="citation_title=HYR1-mediated detoxification of reactive oxygen species is required for full virulence in the rice blast fungus;  citation_journal_title=PloS Pathogens;  citation_year=2011;  citation_volume=7;  citation_pages=e1001335" /><meta name="citation_reference" content="citation_title=Secretory proteins are delivered to the septin-organized penetration interface during root infection by Verticillium dahliae;  citation_journal_title=PloS Pathol;  citation_year=2017;  citation_volume=13;  citation_pages=e1006275" /><meta name="citation_reference" content="citation_title=Hyphopodium-specific VdNoxB/VdPls1-dependent ROS-Ca2+ signaling is required for plant infection by Verticillium dahliae;  citation_journal_title=PloS Pathol;  citation_year=2016;  citation_volume=12;  citation_pages=e1005793" /><meta name="citation_reference" content="citation_title=Mating-type distribution and genetic diversity of Cercospora sojina populations on soybean from Arkansas: evidence for potential sexual reproduction,;  citation_journal_title=Phytopathology;  citation_year=2013;  citation_volume=103;  citation_pages=1045-51; " /><meta name="citation_reference" content="citation_title=Simple and reliable DNA extraction method for the dark pigmented fungus, Cercospora sojina,;  citation_journal_title=Plant Pathol. J;  citation_year=2010;  citation_volume=26;  citation_pages=289-92; " /><meta name="citation_reference" content="citation_title=Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data,;  citation_journal_title=Nat. Methods;  citation_year=2013;  citation_volume=10;  citation_pages=563-9; " /><meta name="citation_reference" content="citation_title=Assembling large genomes with single-molecule sequencing and locality-sensitive hashing,;  citation_journal_title=Nat. Biotechnol;  citation_year=2015;  citation_volume=33;  citation_pages=623-30; " /><meta name="citation_reference" content="citation_title=SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler,;  citation_journal_title=Gigascience;  citation_year=2012;  citation_volume=1;  citation_pages=18" /><meta name="citation_reference" content="citation_title=Gene prediction in novel fungal genomes using an ab initio algorithm with unsupervised training,;  citation_journal_title=Genome Res;  citation_year=2008;  citation_volume=18;  citation_pages=1979-90; " /><meta name="citation_reference" content="citation_title=Ab initio gene finding in Drosophila genomic DNA,;  citation_journal_title=Genome Res;  citation_year=2000;  citation_volume=10;  citation_pages=516-22; " /><meta name="citation_reference" content="citation_title=Improved microbial gene identification with GLIMMER;  citation_journal_title=Nucleic Acids Res;  citation_year=1999;  citation_volume=27;  citation_pages=4636-41; " /><meta name="citation_reference" content="citation_title=Approaches to fungal genome annotation,;  citation_journal_title=Mycology;  citation_year=2011;  citation_volume=2;  citation_pages=118-41; " /><meta name="citation_reference" content="citation_title=De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis,;  citation_journal_title=Nat. Protoc;  citation_year=2013;  citation_volume=8;  citation_pages=1494-512; " /><meta name="citation_reference" content="citation_title=The KEGG resource for deciphering the genome,;  citation_journal_title=Nucleic Acids Res;  citation_year=2004;  citation_volume=32;  citation_pages=D277-80; " /><meta name="citation_reference" content="citation_title=The COG database: a tool for genome-scale analysis of protein functions and evolution,;  citation_journal_title=Nucleic Acids Res;  citation_year=2000;  citation_volume=28;  citation_pages=33-6; " /><meta name="citation_reference" content="citation_title=Pfam: the protein families database,;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=D222-30; " /><meta name="citation_reference" content="citation_title=Profile hidden Markov models,;  citation_journal_title=Bioinformatics;  citation_year=1998;  citation_volume=14;  citation_pages=755-63; " /><meta name="citation_reference" content="citation_title=PHI-base: a new interface and further additions for the multi-species pathogen–host interactions database;  citation_journal_title=Nucleic Acids Res;  citation_year=2017;  citation_volume=45;  citation_pages=D604-10; " /><meta name="citation_reference" content="citation_title=B2G-FAR, a species-centered GO annotation repository,;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=919-24; " /><meta name="citation_reference" content="citation_title=Tandem repeats finder: a program to analyze DNA sequences,;  citation_journal_title=Nucleic Acids Res;  citation_year=1999;  citation_volume=27;  citation_pages=573-80; " /><meta name="citation_reference" content="citation_title=The epigenomic landscape of prokaryotes,;  citation_journal_title=PloS Genet;  citation_year=2016;  citation_volume=12;  citation_pages=e1005854" /><meta name="citation_reference" content="citation_title=The genome portal of the Department of Energy Joint Genome Institute: 2014 updates,;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=D26-31; " /><meta name="citation_reference" content="citation_title=A consistent phylogenetic backbone for the fungi,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2012;  citation_volume=29;  citation_pages=1319-34; " /><meta name="citation_reference" content="citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput,;  citation_journal_title=Nucleic Acids Res;  citation_year=2004;  citation_volume=32;  citation_pages=1792-7; " /><meta name="citation_reference" content="citation_title=MEGA7: molecular evolutionary genetics analysis version 7.0 for bigger datasets,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2016;  citation_volume=33;  citation_pages=1870-4; " /><meta name="citation_reference" content="citation_title=GATA: a graphic alignment tool for comparative sequence analysis,;  citation_journal_title=BMC Bioinformatics;  citation_year=2005;  citation_volume=6;  citation_pages=9" /><meta name="citation_reference" content="citation_title=Challenges in homology search: HMMER3 and convergent evolution of coiled-coil regions,;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=e121" /><meta name="citation_reference" content="citation_title=dbCAN: a web resource for automated carbohydrate-active enzyme annotation,;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=W445-51; " /><meta name="citation_reference" content="citation_title=SMURF: genomic mapping of fungal secondary metabolite clusters,;  citation_journal_title=Fungal Genet. Biol;  citation_year=2010;  citation_volume=47;  citation_pages=736-41; " /><meta name="citation_reference" content="citation_title=antiSMASH: rapid identification, annotation and analysis of secondary metabolite biosynthesis gene clusters in bacterial and fungal genome sequences,;  citation_journal_title=Nucleic Acids Res;  citation_year=2011;  citation_volume=39;  citation_pages=W339-46; " /><meta name="citation_reference" content="citation_title=Identification of genes expressed during cercosporin biosynthesis in Cercospora zeae-maydis,;  citation_journal_title=Physiol. Mol. Plant P;  citation_year=2002;  citation_volume=61;  citation_pages=237-48; " /><meta name="citation_reference" content="citation_title=Activation of ethylene signaling pathways enhances disease resistance by regulating ROS and phytoalexin production in rice,;  citation_journal_title=Plant J;  citation_year=2017;  citation_volume=89;  citation_pages=338-53; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of Cercospora sojina isolate S9, a fungus causing frogeye leaf spot (FLS) disease of soybean;  citation_journal_title=Genomics Data;  citation_year=2017;  citation_volume=12;  citation_pages=79-80; " /><meta name="citation_reference" content="citation_title=Repetitive elements dynamics in cell identity programming, maintenance and disease,;  citation_journal_title=Curr. Opin. Cell Biol;  citation_year=2014;  citation_volume=31;  citation_pages=67-73; " /><meta name="citation_reference" content="citation_title=Epigenetic regulation of gene expression: how the genome integrates intrinsic and environmental signals,;  citation_journal_title=Nat. Genet;  citation_year=2003;  citation_volume=33;  citation_pages=245-54; " /><meta name="citation_reference" content="citation_title=Diverse lifestyles and strategies of plant pathogenesis encoded in the genomes of eighteen Dothideomycetes fungi,;  citation_journal_title=PloS Pathog;  citation_year=2012;  citation_volume=8;  citation_pages=e1003037" /><meta name="citation_reference" content="citation_title=Direct detection of DNA methylation during single-molecule, real-time sequencing;  citation_journal_title=Nature Methods;  citation_year=2010;  citation_volume=7;  citation_pages=461-5; " /><meta name="citation_reference" content="citation_title=Resistance to gray leaf spot of maize: genetic architecture and mechanisms elucidated through nested association mapping and near-isogenic line analysis,;  citation_journal_title=PloS Genet;  citation_year=2015;  citation_volume=11;  citation_pages=e1005045" /><meta name="citation_reference" content="citation_title=Finished genome of the fungal wheat pathogen Mycosphaerella graminicola reveals dispensome structure, chromosome plasticity, and stealth pathogenesis,;  citation_journal_title=PloS Genet;  citation_year=2011;  citation_volume=7;  citation_pages=e1002070" /><meta name="citation_reference" content="citation_title=Predicting transmembrane protein topology with a hidden Markov model: application to complete genomes,;  citation_journal_title=J. Mol. Biol;  citation_year=2001;  citation_volume=305;  citation_pages=567-80; " /><meta name="citation_reference" content="citation_title=SignalP 4.0: discriminating signal peptides from transmembrane regions,;  citation_journal_title=Nat. Methods;  citation_year=2011;  citation_volume=8;  citation_pages=785-6; " /><meta name="citation_reference" content="citation_title=Comparative genome analysis of filamentous fungi reveals gene family expansions associated with fungal pathogenesis,;  citation_journal_title=PloS One;  citation_year=2008;  citation_volume=3;  citation_pages=e2300" /><meta name="citation_reference" content="citation_title=The role of effectors of biotrophic and hemibiotrophic fungi in infection,;  citation_journal_title=Cell Microbiol;  citation_year=2011;  citation_volume=13;  citation_pages=1849-57; " /><meta name="citation_reference" content="citation_title=Comparative secretome investigation of magnaporthe oryzae proteins responsive to nitrogen starvation,;  citation_journal_title=J. Proteome Res;  citation_year=2011;  citation_volume=10;  citation_pages=3136-48; " /><meta name="citation_reference" content="citation_title=Color me bad: microbial pigments as virulence factors,;  citation_journal_title=Trends Microbiol;  citation_year=2009;  citation_volume=17;  citation_pages=406-13; " /><meta name="citation_reference" content="citation_title=Microbial carbohydrate esterases deacetylating plant polysaccharides;  citation_journal_title=Biotechnol. Adv;  citation_year=2012;  citation_volume=30;  citation_pages=1575-88; " /><meta name="citation_reference" content="citation_title=Microbial carbohydrate esterases in cold adapted environments,;  citation_journal_title=Gene;  citation_year=2008;  citation_volume=410;  citation_pages=234-40; " /><meta name="citation_reference" content="citation_title=Bacterial glycosidases for the production of universal red blood cells,;  citation_journal_title=Nat. Biotechnol;  citation_year=2007;  citation_volume=25;  citation_pages=454-64; " /><meta name="citation_reference" content="citation_title=Ultrastructural-localization of glycoconjugates in the fungus ascocalyx-abietina, the scleroderris canker agent of conifers, using lectin gold complexes,;  citation_journal_title=J. Histochem. Cytochem;  citation_year=1986;  citation_volume=34;  citation_pages=855-67; " /><meta name="citation_reference" content="citation_title=Comparative analysis of fungal genomes reveals different plant cell wall degrading capacity in fungi,;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=274" /><meta name="citation_reference" content="citation_title=Comparative genomics yields insights into niche adaptation of plant vascular wilt pathogens,;  citation_journal_title=PloS Pathog;  citation_year=2011;  citation_volume=7;  citation_pages=e1002137" /><meta name="citation_reference" content="citation_title=Insights from the genome of the biotrophic fungal plant pathogen Ustilago maydis,;  citation_journal_title=Nature;  citation_year=2006;  citation_volume=444;  citation_pages=97-101; " /><meta name="citation_reference" content="citation_title=Transcriptional programming and functional interactions within the Phytophthora sojae RXLR effector repertoire,;  citation_journal_title=Plant Cell;  citation_year=2011;  citation_volume=23;  citation_pages=2064-86; " /><meta name="citation_reference" content="citation_title=Epigenetic regulation of development and pathogenesis in fungal plant pathogens;  citation_journal_title=Mol. Plant Pathol;  citation_year=2016; " /><meta name="citation_reference" content="citation_title=MethSMRT: an integrative database for DNA N6-methyladenine and N4-methylcytosine generated by single-molecular real-time sequencing,;  citation_journal_title=Nucleic Acids Res;  citation_year=2017;  citation_volume=45;  citation_pages=D85-9; " /><meta name="citation_reference" content="citation_title=N6-methyl-adenine: an epigenetic signal for DNA-protein interactions,;  citation_journal_title=Nat. Rev. Microbiol;  citation_year=2006;  citation_volume=4;  citation_pages=183-92; " /><meta name="citation_reference" content="citation_title=Studies on growth inhibition by lectins of Penicillia and Aspergilli;  citation_journal_title=Arch. Microbiol;  citation_year=1978;  citation_volume=116;  citation_pages=119-24; " /><meta name="citation_reference" content="citation_title=Expression of soybean lectin in transgenic tobacco results in enhanced resistance to pathogens and pests;  citation_journal_title=Plant Science;  citation_year=2013;  citation_volume=211;  citation_pages=17-22; " /><meta name="citation_reference" content="citation_title=Carbohydrate-binding modules: fine-tuning polysaccharide recognition,;  citation_journal_title=Biochem. J;  citation_year=2004;  citation_volume=382;  citation_pages=769-81; " /><meta name="citation_reference" content="citation_title=Dimension, shape, and conformational flexibility of a two domain fungal cellulase in solution probed by small angle X-ray scattering,;  citation_journal_title=J. Biol. Chem;  citation_year=2002;  citation_volume=277;  citation_pages=40887-92; " /><meta name="citation_reference" content="citation_title=Regulation of secondary metabolism in filamentous fungi,;  citation_journal_title=Annu. Rev. Phytopathol;  citation_year=2005;  citation_volume=43;  citation_pages=437-58; " /><meta name="citation_reference" content="citation_title=Fungal melanin: what do we know about structure?;  citation_journal_title=Front Microbiol;  citation_year=2015;  citation_volume=6;  citation_pages=1463" /><meta name="citation_reference" content="citation_title=The bacterial effector AvrB-induced RIN4 hyperphosphorylation is mediated by a receptor-like cytoplasmic kinase complex in Arabidopsis;  citation_journal_title=Mol. Plant-Microbe In;  citation_year=2017;  citation_volume=30;  citation_pages=502-12; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/1/25/24968426/dsx035.pdf" /><meta name="description" content="Abstract.  Frogeye leaf spot, caused by Cercospora sojina Hara, is a common disease of soybean in most soybean-growing countries of the world. In this study, we" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/1/25/4161430" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/1/25/4161430" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/1/25/4161430");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx035';
    </script>
<title>
	Genome sequencing and comparative genomics reveal the potential pathogenic mechanism of Cercospora sojina Hara on soybean | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4161430" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZ61ATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTHVvLCBYdW1pbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTdGF0ZSBLZXkgTGFib3JhdG9yeSBvZiBQbGFudCBHZW5vbWljcywgSW5zdGl0dXRlIG9mIE1pY3JvYmlvbG9neSwgQ2hpbmVzZSBBY2FkZW15IG9mIFNjaWVuY2VzLCBCZWlqaW5nIDEwMDEwMSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJVbml2ZXJzaXR5IG9mIENoaW5lc2UgQWNhZGVteSBvZiBTY2llbmNlcywgQmVpamluZyAxMDAwNDksIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2FvLCBKaWRvbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTdGF0ZSBLZXkgTGFib3JhdG9yeSBvZiBQbGFudCBHZW5vbWljcywgSW5zdGl0dXRlIG9mIE1pY3JvYmlvbG9neSwgQ2hpbmVzZSBBY2FkZW15IG9mIFNjaWVuY2VzLCBCZWlqaW5nIDEwMDEwMSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIdWFuZywgSnVua2FpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU3RhdGUgS2V5IExhYm9yYXRvcnkgb2YgUGxhbnQgR2Vub21pY3MsIEluc3RpdHV0ZSBvZiBNaWNyb2Jpb2xvZ3ksIENoaW5lc2UgQWNhZGVteSBvZiBTY2llbmNlcywgQmVpamluZyAxMDAxMDEsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVW5pdmVyc2l0eSBvZiBDaGluZXNlIEFjYWRlbXkgb2YgU2NpZW5jZXMsIEJlaWppbmcgMTAwMDQ5LCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IldhbmcsIFpvbmd5aSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJlaWppbmcgS2V5IExhYm9yYXRvcnkgb2YgQWdyaWN1bHR1cmFsIFByb2R1Y3QgRGV0ZWN0aW9uIGFuZCBDb250cm9sIGZvciBTcG9pbGFnZSBPcmdhbmlzbXMgYW5kIFBlc3RpY2lkZXMsIEJlaWppbmcgVW5pdmVyc2l0eSBvZiBBZ3JpY3VsdHVyZSwgQmVpamluZyAxMDIyMDYsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR3VvLCBaaGVuZ3lhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlN0YXRlIEtleSBMYWJvcmF0b3J5IG9mIE1pY3JvYmlhbCBSZXNvdXJjZXMsIEluc3RpdHV0ZSBvZiBNaWNyb2Jpb2xvZ3ksIENoaW5lc2UgQWNhZGVteSBvZiBTY2llbmNlcywgQmVpamluZyAxMDAxMDEsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2hlbiwgWWlodWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTdGF0ZSBLZXkgTGFib3JhdG9yeSBvZiBNaWNyb2JpYWwgUmVzb3VyY2VzLCBJbnN0aXR1dGUgb2YgTWljcm9iaW9sb2d5LCBDaGluZXNlIEFjYWRlbXkgb2YgU2NpZW5jZXMsIEJlaWppbmcgMTAwMTAxLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1hLCBTaHVtZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFBsYW50IFByb3RlY3Rpb24sIENvbGxlZ2Ugb2YgQWdyaWN1bHR1cmUgUmVzb3VyY2VzIGFuZCBFbnZpcm9ubWVudCwgSGVpbG9uZ2ppYW5nIFVuaXZlcnNpdHksIEhhcmJpbiAxNTAwODAsIENoaW5hICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxpdSwgSnVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU3RhdGUgS2V5IExhYm9yYXRvcnkgb2YgUGxhbnQgR2Vub21pY3MsIEluc3RpdHV0ZSBvZiBNaWNyb2Jpb2xvZ3ksIENoaW5lc2UgQWNhZGVteSBvZiBTY2llbmNlcywgQmVpamluZyAxMDAxMDEsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVW5pdmVyc2l0eSBvZiBDaGluZXNlIEFjYWRlbXkgb2YgU2NpZW5jZXMsIEJlaWppbmcgMTAwMDQ5LCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iR2Vub21lIHNlcXVlbmNpbmcgYW5kIGNvbXBhcmF0aXZlIGdlbm9taWNzIHJldmVhbCB0aGUgcG90ZW50aWFsIHBhdGhvZ2VuaWMgbWVjaGFuaXNtIG9mIENlcmNvc3BvcmEgc29qaW5hIEhhcmEgb24gc295YmVhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIzNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzeDAzNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE4LzAyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnJvZ2V5ZSBsZWFmIHNwb3Qgb2Ygc295YmVhbjogYSByZXZpZXcgYW5kIHByb3Bvc2VkIHJhY2UgZGVzaWduYXRpb25zIGZvciBpc29sYXRlcyBvZiBDZXJjb3Nwb3JhIHNvamluYSBIYXJhLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3JvcCBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NDg7ICBjaXRhdGlvbl9wYWdlcz0xNC0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U291cmNlcyBvZiByZXNpc3RhbmNlIGluIHNveWJlYW4gKEdseWNpbmUtbWF4KSB0byBmcm9nLWV5ZSBsZWFmLXNwb3QgY2F1c2VkIGJ5IENlcmNvc3BvcmlkaXVtLXNvamludW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUluZGlhbiBKLiBBZ3JpYy4gU2NpOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTY0OyAgY2l0YXRpb25fcGFnZXM9ODg2LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vcmUgQ2VyY29zcG9yYSBzcGVjaWVzIGluZmVjdCBzb3liZWFucyBhY3Jvc3MgdGhlIEFtZXJpY2FzIHRoYW4gbWVldHMgdGhlIGV5ZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9ZTAxMzM0OTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBzb3liZWFuIGdlbm90eXBlcyByZXNpc3RhbnQgdG8gQ2VyY29zcG9yYSBzb2ppbmEgYnkgZmllbGQgc2NyZWVuaW5nIGFuZCBtb2xlY3VsYXIgbWFya2Vycyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IERpczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT05MzsgIGNpdGF0aW9uX3BhZ2VzPTQwOC0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmZWN0aXZlIG1hbmFnZW1lbnQgb2Ygc295YmVhbiBydXN0IGFuZCBmcm9nZXllIGxlYWYgc3BvdCB1c2luZyBhIG1peHR1cmUgb2YgZmx1c2lsYXpvbGUgYW5kIGNhcmJlbmRhemltLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3JvcCBQcm90OyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9NTY2LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZW5zaXRpdml0eSBvZiBDZXJjb3Nwb3JhIHNvamluYSBpc29sYXRlcyB0byBxdWlub25lIG91dHNpZGUgaW5oaWJpdG9yIGZ1bmdpY2lkZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Dcm9wIFByb3Q7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz02My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmltYXJ5IHJlcG9ydCBvbiB0aGUgaWRlbnRpZmljYXRpb24gZm9yIHBoeXNpb2xvZ2ljYWwgcmFjZXMgb2YgQ2VyY29zcG9yYSBzb2ppbmEgSGFyYSBpbiBub3J0aGVhc3QgQ2hpbmE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFjdGEgUGh5dG9wYXRob2wuIFNpbjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTE4MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBwaG90b2FjdGl2YXRlZCBDZXJjb3Nwb3JhIHRveGluIGNlcmNvc3BvcmluOiBjb250cmlidXRpb25zIHRvIHBsYW50IGRpc2Vhc2UgYW5kIGZ1bmRhbWVudGFsIGJpb2xvZ3ksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIFBoeXRvcGF0aG9sOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9NDYxLTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlsb2dlbmV0aWMgYW5hbHlzaXMgb2YgY2VyY29zcG9yYSBhbmQgbXljb3NwaGFlcmVsbGEgYmFzZWQgb24gdGhlIGludGVybmFsIHRyYW5zY3JpYmVkIHNwYWNlciByZWdpb24gb2Ygcmlib3NvbWFsIEROQSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBoeXRvcGF0aG9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTkxOyAgY2l0YXRpb25fcGFnZXM9NjQ4LTU4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIHNlcXVlbmNlIG9mIHRoZSByaWNlIGJsYXN0IGZ1bmd1cyBNYWduYXBvcnRoZSBncmlzZWEsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NDM0OyAgY2l0YXRpb25fcGFnZXM9OTgwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltbXVuaXR5IHRvIHJpY2UgYmxhc3QgZGlzZWFzZSBieSBzdXBwcmVzc2lvbiBvZiBlZmZlY3Rvci10cmlnZ2VyZWQgbmVjcm9zaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz0yMzk5LTQxMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIE1hZ25hcG9ydGhlIG9yeXphZSBlZmZlY3RvciBBdnJQaXotdCB0YXJnZXRzIHRoZSBSSU5HIEUzIFViaXF1aXRpbiBMaWdhc2UgQVBJUDYgdG8gc3VwcHJlc3MgcGF0aG9nZW4tYXNzb2NpYXRlZCBtb2xlY3VsYXIgcGF0dGVybuKAk3RyaWdnZXJlZCBpbW11bml0eSBpbiByaWNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9NDc0OC02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9FbmQzIHJlZ3VsYXRlcyBhcHByZXNzb3JpdW0gZm9ybWF0aW9uIGFuZCB2aXJ1bGVuY2UgdGhyb3VnaCBtZWRpYXRpbmcgZW5kb2N5dG9zaXMgaW4gcmljZSBibGFzdCBmdW5ndXMgTWFnbmFwb3J0aGUgb3J5emFlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIFBhdGhvZ2VuczsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPWUxMDA2NDQ5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9Td2k2LCBhbiBBUFNFUyBmYW1pbHkgdHJhbnNjcmlwdGlvbiBmYWN0b3IsIGludGVyYWN0cyB3aXRoIE1vTXBzMSBhbmQgaXMgcmVxdWlyZWQgZm9yIGh5cGhhbCBhbmQgY29uaWRpYWwgbW9ycGhvZ2VuZXNpcywgYXBwcmVzc29yaWFsIGZ1bmN0aW9uIGFuZCBwYXRob2dlbmljaXR5IG9mIE1hZ25hcG9ydGhlIG9yeXphZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBQbGFudCBQYXRob2w7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz02NzctODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhZUjEtbWVkaWF0ZWQgZGV0b3hpZmljYXRpb24gb2YgcmVhY3RpdmUgb3h5Z2VuIHNwZWNpZXMgaXMgcmVxdWlyZWQgZm9yIGZ1bGwgdmlydWxlbmNlIGluIHRoZSByaWNlIGJsYXN0IGZ1bmd1czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxvUyBQYXRob2dlbnM7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAxMzM1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VjcmV0b3J5IHByb3RlaW5zIGFyZSBkZWxpdmVyZWQgdG8gdGhlIHNlcHRpbi1vcmdhbml6ZWQgcGVuZXRyYXRpb24gaW50ZXJmYWNlIGR1cmluZyByb290IGluZmVjdGlvbiBieSBWZXJ0aWNpbGxpdW0gZGFobGlhZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxvUyBQYXRob2w7ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz1lMTAwNjI3NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUh5cGhvcG9kaXVtLXNwZWNpZmljIFZkTm94Qi9WZFBsczEtZGVwZW5kZW50IFJPUy1DYTIrIHNpZ25hbGluZyBpcyByZXF1aXJlZCBmb3IgcGxhbnQgaW5mZWN0aW9uIGJ5IFZlcnRpY2lsbGl1bSBkYWhsaWFlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbG9TIFBhdGhvbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPWUxMDA1NzkzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWF0aW5nLXR5cGUgZGlzdHJpYnV0aW9uIGFuZCBnZW5ldGljIGRpdmVyc2l0eSBvZiBDZXJjb3Nwb3JhIHNvamluYSBwb3B1bGF0aW9ucyBvbiBzb3liZWFuIGZyb20gQXJrYW5zYXM6IGV2aWRlbmNlIGZvciBwb3RlbnRpYWwgc2V4dWFsIHJlcHJvZHVjdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBoeXRvcGF0aG9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTEwMzsgIGNpdGF0aW9uX3BhZ2VzPTEwNDUtNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbXBsZSBhbmQgcmVsaWFibGUgRE5BIGV4dHJhY3Rpb24gbWV0aG9kIGZvciB0aGUgZGFyayBwaWdtZW50ZWQgZnVuZ3VzLCBDZXJjb3Nwb3JhIHNvamluYSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBhdGhvbC4gSjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTI4OS05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm9uaHlicmlkLCBmaW5pc2hlZCBtaWNyb2JpYWwgZ2Vub21lIGFzc2VtYmxpZXMgZnJvbSBsb25nLXJlYWQgU01SVCBzZXF1ZW5jaW5nIGRhdGEsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz01NjMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzZW1ibGluZyBsYXJnZSBnZW5vbWVzIHdpdGggc2luZ2xlLW1vbGVjdWxlIHNlcXVlbmNpbmcgYW5kIGxvY2FsaXR5LXNlbnNpdGl2ZSBoYXNoaW5nLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9NjIzLTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TT0FQZGVub3ZvMjogYW4gZW1waXJpY2FsbHkgaW1wcm92ZWQgbWVtb3J5LWVmZmljaWVudCBzaG9ydC1yZWFkIGRlIG5vdm8gYXNzZW1ibGVyLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2lnYXNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTE4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBwcmVkaWN0aW9uIGluIG5vdmVsIGZ1bmdhbCBnZW5vbWVzIHVzaW5nIGFuIGFiIGluaXRpbyBhbGdvcml0aG0gd2l0aCB1bnN1cGVydmlzZWQgdHJhaW5pbmcsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MTk3OS05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWIgaW5pdGlvIGdlbmUgZmluZGluZyBpbiBEcm9zb3BoaWxhIGdlbm9taWMgRE5BLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTUxNi0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1wcm92ZWQgbWljcm9iaWFsIGdlbmUgaWRlbnRpZmljYXRpb24gd2l0aCBHTElNTUVSOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTQ2MzYtNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFwcHJvYWNoZXMgdG8gZnVuZ2FsIGdlbm9tZSBhbm5vdGF0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TXljb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTExOC00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGUgbm92byB0cmFuc2NyaXB0IHNlcXVlbmNlIHJlY29uc3RydWN0aW9uIGZyb20gUk5BLXNlcSB1c2luZyB0aGUgVHJpbml0eSBwbGF0Zm9ybSBmb3IgcmVmZXJlbmNlIGdlbmVyYXRpb24gYW5kIGFuYWx5c2lzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBQcm90b2M7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTE0OTQtNTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgS0VHRyByZXNvdXJjZSBmb3IgZGVjaXBoZXJpbmcgdGhlIGdlbm9tZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9RDI3Ny04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIENPRyBkYXRhYmFzZTogYSB0b29sIGZvciBnZW5vbWUtc2NhbGUgYW5hbHlzaXMgb2YgcHJvdGVpbiBmdW5jdGlvbnMgYW5kIGV2b2x1dGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MzMtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGZhbTogdGhlIHByb3RlaW4gZmFtaWxpZXMgZGF0YWJhc2UsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPUQyMjItMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb2ZpbGUgaGlkZGVuIE1hcmtvdiBtb2RlbHMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTc1NS02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UEhJLWJhc2U6IGEgbmV3IGludGVyZmFjZSBhbmQgZnVydGhlciBhZGRpdGlvbnMgZm9yIHRoZSBtdWx0aS1zcGVjaWVzIHBhdGhvZ2Vu4oCTaG9zdCBpbnRlcmFjdGlvbnMgZGF0YWJhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE3OyAgY2l0YXRpb25fdm9sdW1lPTQ1OyAgY2l0YXRpb25fcGFnZXM9RDYwNC0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QjJHLUZBUiwgYSBzcGVjaWVzLWNlbnRlcmVkIEdPIGFubm90YXRpb24gcmVwb3NpdG9yeSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9OTE5LTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYW5kZW0gcmVwZWF0cyBmaW5kZXI6IGEgcHJvZ3JhbSB0byBhbmFseXplIEROQSBzZXF1ZW5jZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTU3My04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGVwaWdlbm9taWMgbGFuZHNjYXBlIG9mIHByb2thcnlvdGVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxvUyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPWUxMDA1ODU0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBwb3J0YWwgb2YgdGhlIERlcGFydG1lbnQgb2YgRW5lcmd5IEpvaW50IEdlbm9tZSBJbnN0aXR1dGU6IDIwMTQgdXBkYXRlcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9RDI2LTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbnNpc3RlbnQgcGh5bG9nZW5ldGljIGJhY2tib25lIGZvciB0aGUgZnVuZ2ksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz0xMzE5LTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NVVNDTEU6IG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCB3aXRoIGhpZ2ggYWNjdXJhY3kgYW5kIGhpZ2ggdGhyb3VnaHB1dCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9MTc5Mi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NRUdBNzogbW9sZWN1bGFyIGV2b2x1dGlvbmFyeSBnZW5ldGljcyBhbmFseXNpcyB2ZXJzaW9uIDcuMCBmb3IgYmlnZ2VyIGRhdGFzZXRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9MTg3MC00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HQVRBOiBhIGdyYXBoaWMgYWxpZ25tZW50IHRvb2wgZm9yIGNvbXBhcmF0aXZlIHNlcXVlbmNlIGFuYWx5c2lzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz05IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhbGxlbmdlcyBpbiBob21vbG9neSBzZWFyY2g6IEhNTUVSMyBhbmQgY29udmVyZ2VudCBldm9sdXRpb24gb2YgY29pbGVkLWNvaWwgcmVnaW9ucyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9ZTEyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPWRiQ0FOOiBhIHdlYiByZXNvdXJjZSBmb3IgYXV0b21hdGVkIGNhcmJvaHlkcmF0ZS1hY3RpdmUgZW56eW1lIGFubm90YXRpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPVc0NDUtNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNNVVJGOiBnZW5vbWljIG1hcHBpbmcgb2YgZnVuZ2FsIHNlY29uZGFyeSBtZXRhYm9saXRlIGNsdXN0ZXJzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnVuZ2FsIEdlbmV0LiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ3OyAgY2l0YXRpb25fcGFnZXM9NzM2LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1hbnRpU01BU0g6IHJhcGlkIGlkZW50aWZpY2F0aW9uLCBhbm5vdGF0aW9uIGFuZCBhbmFseXNpcyBvZiBzZWNvbmRhcnkgbWV0YWJvbGl0ZSBiaW9zeW50aGVzaXMgZ2VuZSBjbHVzdGVycyBpbiBiYWN0ZXJpYWwgYW5kIGZ1bmdhbCBnZW5vbWUgc2VxdWVuY2VzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz1XMzM5LTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBnZW5lcyBleHByZXNzZWQgZHVyaW5nIGNlcmNvc3BvcmluIGJpb3N5bnRoZXNpcyBpbiBDZXJjb3Nwb3JhIHplYWUtbWF5ZGlzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGh5c2lvbC4gTW9sLiBQbGFudCBQOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTYxOyAgY2l0YXRpb25fcGFnZXM9MjM3LTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY3RpdmF0aW9uIG9mIGV0aHlsZW5lIHNpZ25hbGluZyBwYXRod2F5cyBlbmhhbmNlcyBkaXNlYXNlIHJlc2lzdGFuY2UgYnkgcmVndWxhdGluZyBST1MgYW5kIHBoeXRvYWxleGluIHByb2R1Y3Rpb24gaW4gcmljZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEo7ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9ODk7ICBjaXRhdGlvbl9wYWdlcz0zMzgtNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyYWZ0IGdlbm9tZSBzZXF1ZW5jZSBvZiBDZXJjb3Nwb3JhIHNvamluYSBpc29sYXRlIFM5LCBhIGZ1bmd1cyBjYXVzaW5nIGZyb2dleWUgbGVhZiBzcG90IChGTFMpIGRpc2Vhc2Ugb2Ygc295YmVhbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21pY3MgRGF0YTsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTc5LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXBldGl0aXZlIGVsZW1lbnRzIGR5bmFtaWNzIGluIGNlbGwgaWRlbnRpdHkgcHJvZ3JhbW1pbmcsIG1haW50ZW5hbmNlIGFuZCBkaXNlYXNlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gQ2VsbCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9NjctNzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVwaWdlbmV0aWMgcmVndWxhdGlvbiBvZiBnZW5lIGV4cHJlc3Npb246IGhvdyB0aGUgZ2Vub21lIGludGVncmF0ZXMgaW50cmluc2ljIGFuZCBlbnZpcm9ubWVudGFsIHNpZ25hbHMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9MjQ1LTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXZlcnNlIGxpZmVzdHlsZXMgYW5kIHN0cmF0ZWdpZXMgb2YgcGxhbnQgcGF0aG9nZW5lc2lzIGVuY29kZWQgaW4gdGhlIGdlbm9tZXMgb2YgZWlnaHRlZW4gRG90aGlkZW9teWNldGVzIGZ1bmdpLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxvUyBQYXRob2c7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAzMDM3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlyZWN0IGRldGVjdGlvbiBvZiBETkEgbWV0aHlsYXRpb24gZHVyaW5nIHNpbmdsZS1tb2xlY3VsZSwgcmVhbC10aW1lIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZSBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz00NjEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVzaXN0YW5jZSB0byBncmF5IGxlYWYgc3BvdCBvZiBtYWl6ZTogZ2VuZXRpYyBhcmNoaXRlY3R1cmUgYW5kIG1lY2hhbmlzbXMgZWx1Y2lkYXRlZCB0aHJvdWdoIG5lc3RlZCBhc3NvY2lhdGlvbiBtYXBwaW5nIGFuZCBuZWFyLWlzb2dlbmljIGxpbmUgYW5hbHlzaXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9ZTEwMDUwNDUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GaW5pc2hlZCBnZW5vbWUgb2YgdGhlIGZ1bmdhbCB3aGVhdCBwYXRob2dlbiBNeWNvc3BoYWVyZWxsYSBncmFtaW5pY29sYSByZXZlYWxzIGRpc3BlbnNvbWUgc3RydWN0dXJlLCBjaHJvbW9zb21lIHBsYXN0aWNpdHksIGFuZCBzdGVhbHRoIHBhdGhvZ2VuZXNpcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsb1MgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAyMDcwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlZGljdGluZyB0cmFuc21lbWJyYW5lIHByb3RlaW4gdG9wb2xvZ3kgd2l0aCBhIGhpZGRlbiBNYXJrb3YgbW9kZWw6IGFwcGxpY2F0aW9uIHRvIGNvbXBsZXRlIGdlbm9tZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MzA1OyAgY2l0YXRpb25fcGFnZXM9NTY3LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaWduYWxQIDQuMDogZGlzY3JpbWluYXRpbmcgc2lnbmFsIHBlcHRpZGVzIGZyb20gdHJhbnNtZW1icmFuZSByZWdpb25zLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz03ODUtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgZ2Vub21lIGFuYWx5c2lzIG9mIGZpbGFtZW50b3VzIGZ1bmdpIHJldmVhbHMgZ2VuZSBmYW1pbHkgZXhwYW5zaW9ucyBhc3NvY2lhdGVkIHdpdGggZnVuZ2FsIHBhdGhvZ2VuZXNpcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz1lMjMwMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByb2xlIG9mIGVmZmVjdG9ycyBvZiBiaW90cm9waGljIGFuZCBoZW1pYmlvdHJvcGhpYyBmdW5naSBpbiBpbmZlY3Rpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTE4NDktNTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIHNlY3JldG9tZSBpbnZlc3RpZ2F0aW9uIG9mIG1hZ25hcG9ydGhlIG9yeXphZSBwcm90ZWlucyByZXNwb25zaXZlIHRvIG5pdHJvZ2VuIHN0YXJ2YXRpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBQcm90ZW9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0zMTM2LTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2xvciBtZSBiYWQ6IG1pY3JvYmlhbCBwaWdtZW50cyBhcyB2aXJ1bGVuY2UgZmFjdG9ycyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz00MDYtMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3JvYmlhbCBjYXJib2h5ZHJhdGUgZXN0ZXJhc2VzIGRlYWNldHlsYXRpbmcgcGxhbnQgcG9seXNhY2NoYXJpZGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW90ZWNobm9sLiBBZHY7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0xNTc1LTg4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb2JpYWwgY2FyYm9oeWRyYXRlIGVzdGVyYXNlcyBpbiBjb2xkIGFkYXB0ZWQgZW52aXJvbm1lbnRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00MTA7ICBjaXRhdGlvbl9wYWdlcz0yMzQtNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhY3RlcmlhbCBnbHljb3NpZGFzZXMgZm9yIHRoZSBwcm9kdWN0aW9uIG9mIHVuaXZlcnNhbCByZWQgYmxvb2QgY2VsbHMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz00NTQtNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVsdHJhc3RydWN0dXJhbC1sb2NhbGl6YXRpb24gb2YgZ2x5Y29jb25qdWdhdGVzIGluIHRoZSBmdW5ndXMgYXNjb2NhbHl4LWFiaWV0aW5hLCB0aGUgc2NsZXJvZGVycmlzIGNhbmtlciBhZ2VudCBvZiBjb25pZmVycywgdXNpbmcgbGVjdGluIGdvbGQgY29tcGxleGVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gSGlzdG9jaGVtLiBDeXRvY2hlbTsgIGNpdGF0aW9uX3llYXI9MTk4NjsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPTg1NS02NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgYW5hbHlzaXMgb2YgZnVuZ2FsIGdlbm9tZXMgcmV2ZWFscyBkaWZmZXJlbnQgcGxhbnQgY2VsbCB3YWxsIGRlZ3JhZGluZyBjYXBhY2l0eSBpbiBmdW5naSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTI3NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIGdlbm9taWNzIHlpZWxkcyBpbnNpZ2h0cyBpbnRvIG5pY2hlIGFkYXB0YXRpb24gb2YgcGxhbnQgdmFzY3VsYXIgd2lsdCBwYXRob2dlbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbG9TIFBhdGhvZzsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDIxMzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnNpZ2h0cyBmcm9tIHRoZSBnZW5vbWUgb2YgdGhlIGJpb3Ryb3BoaWMgZnVuZ2FsIHBsYW50IHBhdGhvZ2VuIFVzdGlsYWdvIG1heWRpcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT00NDQ7ICBjaXRhdGlvbl9wYWdlcz05Ny0xMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb25hbCBwcm9ncmFtbWluZyBhbmQgZnVuY3Rpb25hbCBpbnRlcmFjdGlvbnMgd2l0aGluIHRoZSBQaHl0b3BodGhvcmEgc29qYWUgUlhMUiBlZmZlY3RvciByZXBlcnRvaXJlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTIwNjQtODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVwaWdlbmV0aWMgcmVndWxhdGlvbiBvZiBkZXZlbG9wbWVudCBhbmQgcGF0aG9nZW5lc2lzIGluIGZ1bmdhbCBwbGFudCBwYXRob2dlbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gUGxhbnQgUGF0aG9sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZXRoU01SVDogYW4gaW50ZWdyYXRpdmUgZGF0YWJhc2UgZm9yIEROQSBONi1tZXRoeWxhZGVuaW5lIGFuZCBONC1tZXRoeWxjeXRvc2luZSBnZW5lcmF0ZWQgYnkgc2luZ2xlLW1vbGVjdWxhciByZWFsLXRpbWUgc2VxdWVuY2luZyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE3OyAgY2l0YXRpb25fdm9sdW1lPTQ1OyAgY2l0YXRpb25fcGFnZXM9RDg1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU42LW1ldGh5bC1hZGVuaW5lOiBhbiBlcGlnZW5ldGljIHNpZ25hbCBmb3IgRE5BLXByb3RlaW4gaW50ZXJhY3Rpb25zLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MTgzLTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHVkaWVzIG9uIGdyb3d0aCBpbmhpYml0aW9uIGJ5IGxlY3RpbnMgb2YgUGVuaWNpbGxpYSBhbmQgQXNwZXJnaWxsaTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXJjaC4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0xOTc4OyAgY2l0YXRpb25fdm9sdW1lPTExNjsgIGNpdGF0aW9uX3BhZ2VzPTExOS0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwcmVzc2lvbiBvZiBzb3liZWFuIGxlY3RpbiBpbiB0cmFuc2dlbmljIHRvYmFjY28gcmVzdWx0cyBpbiBlbmhhbmNlZCByZXNpc3RhbmNlIHRvIHBhdGhvZ2VucyBhbmQgcGVzdHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjExOyAgY2l0YXRpb25fcGFnZXM9MTctMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNhcmJvaHlkcmF0ZS1iaW5kaW5nIG1vZHVsZXM6IGZpbmUtdHVuaW5nIHBvbHlzYWNjaGFyaWRlIHJlY29nbml0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvY2hlbS4gSjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zODI7ICBjaXRhdGlvbl9wYWdlcz03NjktODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpbWVuc2lvbiwgc2hhcGUsIGFuZCBjb25mb3JtYXRpb25hbCBmbGV4aWJpbGl0eSBvZiBhIHR3byBkb21haW4gZnVuZ2FsIGNlbGx1bGFzZSBpbiBzb2x1dGlvbiBwcm9iZWQgYnkgc21hbGwgYW5nbGUgWC1yYXkgc2NhdHRlcmluZyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJpb2wuIENoZW07ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9Mjc3OyAgY2l0YXRpb25fcGFnZXM9NDA4ODctOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZ3VsYXRpb24gb2Ygc2Vjb25kYXJ5IG1ldGFib2xpc20gaW4gZmlsYW1lbnRvdXMgZnVuZ2ksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIFBoeXRvcGF0aG9sOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9NDM3LTU4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5nYWwgbWVsYW5pbjogd2hhdCBkbyB3ZSBrbm93IGFib3V0IHN0cnVjdHVyZT87ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZyb250IE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MTQ2MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBiYWN0ZXJpYWwgZWZmZWN0b3IgQXZyQi1pbmR1Y2VkIFJJTjQgaHlwZXJwaG9zcGhvcnlsYXRpb24gaXMgbWVkaWF0ZWQgYnkgYSByZWNlcHRvci1saWtlIGN5dG9wbGFzbWljIGtpbmFzZSBjb21wbGV4IGluIEFyYWJpZG9wc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFBsYW50LU1pY3JvYmUgSW47ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz01MDItMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNS8xLzI1LzI0OTY4NDI2L2RzeDAzNS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgRnJvZ2V5ZSBsZWFmIHNwb3QsIGNhdXNlZCBieSBDZXJjb3Nwb3JhIHNvamluYSBIYXJhLCBpcyBhIGNvbW1vbiBkaXNlYXNlIG9mIHNveWJlYW4gaW4gbW9zdCBzb3liZWFuLWdyb3dpbmcgY291bnRyaWVzIG9mIHRoZSB3b3JsZC4gSW4gdGhpcyBzdHVkeSwgd2UiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQxNjE0MzAmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkYsbncIZlb6g9y9iyzj74Tu6tXpc=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f25%2f4161430" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f25%2f4161430" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=107546" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&amp;Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/1">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="117294481">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#117294481">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="117294483">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#117294483">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="117294489">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#117294489">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="117294512">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#117294512">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="117294548">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#117294548">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="117294557">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#117294557">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="117294559">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#117294559">Authors’ contributions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="117294561">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#117294561">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="117294563">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#117294563">Conflict of interest</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="117294565">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#117294565">Accession number</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="117294567">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#117294567">Data availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="117294569">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#117294569">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/1/11/4106313">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/1/39/4103403">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome sequencing and comparative genomics reveal the potential pathogenic mechanism of <em>Cercospora sojina</em> Hara on soybean
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xuming Luo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Plant Genomics, Institute of Microbiology, Chinese Academy of Sciences, Beijing 100101, China</div><div class="aff">University of Chinese Academy of Sciences, Beijing 100049, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xuming+Luo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Luo X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Luo X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xuming Luo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jidong Cao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Plant Genomics, Institute of Microbiology, Chinese Academy of Sciences, Beijing 100101, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jidong+Cao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cao J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cao J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jidong Cao</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Junkai Huang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Plant Genomics, Institute of Microbiology, Chinese Academy of Sciences, Beijing 100101, China</div><div class="aff">University of Chinese Academy of Sciences, Beijing 100049, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Junkai+Huang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Huang J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Huang J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Junkai Huang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zongyi Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Beijing Key Laboratory of Agricultural Product Detection and Control for Spoilage Organisms and Pesticides, Beijing University of Agriculture, Beijing 102206, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zongyi+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang Z&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Zongyi Wang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zhengyan Guo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Microbial Resources, Institute of Microbiology, Chinese Academy of Sciences, Beijing 100101, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zhengyan+Guo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Guo Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Guo Z&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Zhengyan Guo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yihua Chen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Microbial Resources, Institute of Microbiology, Chinese Academy of Sciences, Beijing 100101, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yihua+Chen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chen Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chen Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yihua Chen</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shumei Ma</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Plant Protection, College of Agriculture Resources and Environment, Heilongjiang University, Harbin 150080, China </div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx035-COR1" data-open="dsx035-COR1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +010 6480 6131. Email: <a href="mailto:msm2006@126.com" target="_blank">msm2006@126.com</a> (S.M.); <a href="mailto:junliu@im.ac.cn" target="_blank">junliu@im.ac.cn</a> (J.L.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shumei+Ma">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ma S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ma S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shumei Ma</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jun Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Plant Genomics, Institute of Microbiology, Chinese Academy of Sciences, Beijing 100101, China</div><div class="aff">University of Chinese Academy of Sciences, Beijing 100049, China</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx035-COR1" data-open="dsx035-COR1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +010 6480 6131. Email: <a href="mailto:msm2006@126.com" target="_blank">msm2006@126.com</a> (S.M.); <a href="mailto:junliu@im.ac.cn" target="_blank">junliu@im.ac.cn</a> (J.L.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jun+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liu J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jun Liu</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx035-FM2"><div class="fn"><div class="footnote-content">Xuming Luo, Jidong Cao and Junkai Huang contributed equally to this work. </div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx035-FM3"><div class="fn"><div class="footnote-content">Edited by Dr. Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 1, 1 February 2018, Pages 25–37, <a href='https://doi.org/10.1093/dnares/dsx035'>https://doi.org/10.1093/dnares/dsx035</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">18 September 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">21 May 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">16 August 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4161430" href="/dnaresearch/article-pdf/25/1/25/24968426/dsx035.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Xuming Luo, Jidong Cao, Junkai Huang, Zongyi Wang, Zhengyan Guo, Yihua Chen, Shumei Ma, Jun Liu;  Genome sequencing and comparative genomics reveal the potential pathogenic mechanism of <em>Cercospora sojina</em> Hara on soybean, <em>DNA Research</em>, Volume 25, Issue 1, 1 February 2018, Pages 25–37, <a href='https://doi.org/10.1093/dnares/dsx035'>https://doi.org/10.1093/dnares/dsx035</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4161430?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4161430?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4161430?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4161430?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4161430?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4161430?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4161430?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome%20sequencing%20and%20comparative%20genomics%20reveal%20the%20potential%20pathogenic%20mechanism%20of%20%26lt%3Bem%26gt%3BCercospora%20sojina%26lt%3B%2Fem%26gt%3B%20Hara%20on%20soybean&amp;publicationDate=2017-09-18&amp;volumeNum=25&amp;issueNum=1&amp;author=Luo%2C%20Xuming%3B%20Cao%2C%20Jidong&amp;startPage=25&amp;endPage=37&amp;contentId=10.1093%2Fdnares%2Fdsx035&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f25%2f4161430" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=117294481 id="117294481" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Frogeye leaf spot, caused by <em>Cercospora sojina</em> Hara, is a common disease of soybean in most soybean-growing countries of the world. In this study, we report a high-quality genome sequence of <em>C. sojina</em> by Single Molecule Real-Time sequencing method. The 40.8-Mb genome encodes 11,655 predicated genes, and 8,474 genes are revealed by RNA sequencing. <em>Cercospora sojina</em> genome contains large numbers of gene clusters that are involved in synthesis of secondary metabolites, including mycotoxins and pigments. However, much less carbohydrate-binding module protein encoding genes are identified in <em>C. sojina</em> genome, when compared with other phytopathogenic fungi. Bioinformatics analysis reveals that <em>C. sojina</em> harbours about 752 secreted proteins, and 233 of them are effectors. During early infection, the genes for metabolite biosynthesis and effectors are significantly enriched, suggesting that they may play essential roles in pathogenicity. We further identify 13 effectors that can inhibit BAX-induced cell death. Taken together, our results provide insights into the infection mechanisms of <em>C. sojina</em> on soybean.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Cercospora sojina&quot;"><em>Cercospora sojina</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="soybean">soybean</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="genome">genome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="pathogenicity">pathogenicity</a></div>                            <h2 scrollto-destination=117294483 id="117294483" class="section-title" >1. Introduction</h2>
<p>The causal agent of frogeye leaf spot (FLS), <em>Cercospora sojina</em> Hara, is a worldwide destructive pathogen on soybean.<a reveal-id="dsx035-B1" data-open="dsx035-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> It was first reported in Japan in 1915.<a reveal-id="dsx035-B2" data-open="dsx035-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> After that, many soybean growing countries were reported the occurrence of this disease, such as the USA, China, and Argentina.<a reveal-id="dsx035-B3" data-open="dsx035-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> The main measurement to control this disease is to grow resistant soybean varieties or to apply chemical fungicides, which usually lose the effects rapidly in fields due to race differentiation and gene mutations of the pathogen.<a reveal-id="dsx035-B4 dsx035-B5 dsx035-B6" data-open="dsx035-B4 dsx035-B5 dsx035-B6" class="link link-ref link-reveal xref-bibr"><sup>4–6</sup></a> FLS causes about 10–60% yield loss in soybean growing regions, such as Argentina and Nigeria, and it has been reported to be the most expensive disease in the history of soybean production in Argentina.<a reveal-id="dsx035-B3" data-open="dsx035-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a></p><p>Taxonomically, <em>C. sojina</em> belongs to the order <em>Capnodiales</em> in the class of <em>Dothideomycetes</em>. Currently, it is known that there are 22 races of <em>C. sojina</em> in Brazil and 12 races in the USA.<a reveal-id="dsx035-B1" data-open="dsx035-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> Later, using 93 isolates of <em>C. sojina</em> and 38 putative soybean differentials, Mian et al. proposed a core set of 11 races that represent the major diversity of the 93 isolates in the USA.<a reveal-id="dsx035-B1" data-open="dsx035-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> We previously reported the 14 races in north China<a reveal-id="dsx035-B7" data-open="dsx035-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a>; of the identified races of <em>C. sojina</em>, the occurrence frequency of race 1 is 43.5%, emerging as the dominant race among others, which causes yield loss up to 38% in the field.<a reveal-id="dsx035-B7" data-open="dsx035-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a></p><p>Despite the importance of <em>C. sojina</em>, the infection mechanism and the genetic information are not known for this pathogen. For example, most of the species can produce a non-specific coloured mycotoxin cercosporin in the <em>Cercospora</em> genus, which is indispensable for their pathogenicity,<a reveal-id="dsx035-B8" data-open="dsx035-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> but <em>C. sojina</em> may not produce cercosporin.<a reveal-id="dsx035-B9" data-open="dsx035-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Does the genome harbour the cercosporin biosynthesis genes? Nowadays, the main strategy to unravel the mystery of the pathogen infection mechanisms is to obtain their genome information. <em>Magnaporthe grisea</em>, one of the best studied fungi, was sequenced at genome level in 2005.<sup>10</sup> The genome annotation shows that the pathogen may carry over 700 secreted proteins, and most of them are believed to be virulence effectors. Later, several effectors have been implicated in suppressing immune responses in rice.<a reveal-id="dsx035-B11" data-open="dsx035-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsx035-B12" data-open="dsx035-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> The other genes, such as MoEnd3, MoSwi6, and MoHYR1 have been demonstrated to be essential for appressorium formation, melanin accumulation, and reactive oxygen species (ROS) scavenging.<a reveal-id="dsx035-B13 dsx035-B14 dsx035-B15" data-open="dsx035-B13 dsx035-B14 dsx035-B15" class="link link-ref link-reveal xref-bibr"><sup>13–15</sup></a> Similarly, the genome sequence availability and gene annotation have helped to uncover the infection mechanism of <em>Verticillium dahliae</em> substantially, where this pathogen shows distinct infection structure compared with <em>M. oryzae</em> and other fungi.<a reveal-id="dsx035-B16" data-open="dsx035-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsx035-B17" data-open="dsx035-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a></p><p>Although the breeding programme and fungicide application make great success in controlling FLS in last decades, their efficiencies are facing challenges recently.<a reveal-id="dsx035-B4" data-open="dsx035-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx035-B7" data-open="dsx035-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> In fact, <em>Cercospora</em> species may undergo positive selections and rapid evolution. Soares et al. reported that more <em>Cercospora</em> species were able to infect soybean and caused similar disease symptom as <em>C. kikuchii</em>, the closest species of <em>C. sojina</em>.<a reveal-id="dsx035-B3" data-open="dsx035-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> Importantly, they detected interlineage recombination among <em>Cercospora</em> species, along with a high frequency of mutations linked to fungicide resistance.<a reveal-id="dsx035-B3" data-open="dsx035-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> Moreover, it has been observed that <em>C. sojina</em> populations are genetically diverse and likely undergoing sexual reproduction.<a reveal-id="dsx035-B18" data-open="dsx035-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> The above-mentioned reports imply that <em>C. sojina</em> could adapt to the changing environment flexibly.</p><p>In this study, we report the 40.8 Mb complete genome sequence of <em>C. sojina</em> race 1. The genome annotation and whole genome transcriptome assays reveal that <em>C. sojina</em> encodes a different set of proteins that distinguishes it from other fungi species in terms of infection strategy and disease development. We demonstrate that the secondary metabolites and effectors play essential roles in the soybean–<em>C. sojina</em> pathosystem. Moreover, the genome information can be further used for comparative genomic studies with other <em>Cercospora</em> species to unravel their evolution and infection mechanism on soybean.</p>                            <h2 scrollto-destination=117294489 id="117294489" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=117294490 id="117294490" class="section-title" >2.1. Fungal growth conditions and DNA preparation</h3>
<p><em>Cercospora sojina</em> (<em>C. sojina</em>) race 1 was isolated from soybean fields in Heilong Jiang province of China. Briefly, the fungi single spore was isolated from the soybean seedling infested by <em>C. sojina</em> race 1. The isolated spores were grown on potato dextrose agar (PDA) medium at 28°C. Then, the mycelia were removed from the media and ground in liquid nitrogen. Genomic DNA was extracted using a modified cetyltrimethylammonium bromide (CTAB) method.<a reveal-id="dsx035-B19" data-open="dsx035-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a></p>                            <h3 scrollto-destination=117294492 id="117294492" class="section-title" >2.2. Genome sequencing and assembly</h3>
<p><em>Cercospora</em><em>sojina</em> race1 genome was sequenced by Single Molecule Real-Time (SMRT) method in Biomarker Technologies (Beijing, China).<a reveal-id="dsx035-B20" data-open="dsx035-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> DNA libraries with 270 bp and 10 kb inserts were constructed. The 270-bp library was constructed following Illumina’s standard protocol, including fragmentation of genomic DNA, end repair, adaptor ligation and PCR amplification. The 270-bp library was quantified using 2100 Bioanalyzer (Agilent, USA) and subjected to paired-ended 150 bp sequencing by Illumina HiSeq4000. The sequencing data (filtered reads: 2.99G, sequencing depth: 90×) were used to estimate the genome size, repeat content, and heterozygosity. Then, the 10-kb library was constructed following PacBio’s standard methods, including fragmentation of genomic DNA, end repair, adaptor ligation, and templates purification. The 10-kb library was quantified by 2100 Bioanalyzer (Agilent, USA) and sequenced by SMRT, and the sequencing data (filtered reads: 4.92G, sequencing depth: 123×) was assembled by CANU (Version-1.2) with default parameters.<a reveal-id="dsx035-B21" data-open="dsx035-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> Finally, Illumina reads were used for error correction and gap filling with SOAPdenovo GAPCLOSER v1.12.<sup>22</sup></p>                            <h3 scrollto-destination=117294494 id="117294494" class="section-title" >2.3. Genome annotations</h3>
<p>Protein-encoding genes were annotated by a combination of three independent ab initio predicators GeneMark (Version 4.30),<a reveal-id="dsx035-B23" data-open="dsx035-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> SNAP<a reveal-id="dsx035-B24" data-open="dsx035-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a>, and Glimmer.<a reveal-id="dsx035-B25" data-open="dsx035-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> Transcriptome data were incorporated into PASA<a reveal-id="dsx035-B26" data-open="dsx035-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> to improve quality of <em>C. sojina</em> annotation. Briefly, the transcriptome assemblies were mapped to the genome using Trinity.<a reveal-id="dsx035-B27" data-open="dsx035-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> Then PASA alignment assemblies based on overlapping transcript alignments from Trinity and use EVidenceModeler (EVM) to compute weighted consensus gene structure annotations. Finally, PASA was used to update the EVM consensus predictions. </p><p>Functional annotations for all predicted gene models were made using multiple databases, including Swiss-Prot, nr, KEGG<a reveal-id="dsx035-B28" data-open="dsx035-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a>, and COG<a reveal-id="dsx035-B29" data-open="dsx035-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> by blastP with E-values of ≤1e−5. Domain-calling analyses of protein-encoding genes were performed using the Pfam database<a reveal-id="dsx035-B30" data-open="dsx035-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> and HMMER.<a reveal-id="dsx035-B31" data-open="dsx035-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> Potential virulence-related proteins were identified by searching against the pathogen–host interaction database (PHI-base)<a reveal-id="dsx035-B32" data-open="dsx035-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> by blastP with E-values of ≤1e−5. Blast2GO<a reveal-id="dsx035-B33" data-open="dsx035-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> was used for Go enrichment analysis of genes that only belong to <em>C. sojina</em>, and do not have homologue in the non-plant pathogen <em>Aspergillus nidulans</em> and <em>Neurospora crassa</em>. </p>                            <h3 scrollto-destination=117294497 id="117294497" class="section-title" >2.4. Repetitive sequences and whole genome DNA methylation analysis</h3>
<p>Tandem repeat sequences were identified using Tandem Repeats Finder (TrF).<a reveal-id="dsx035-B34" data-open="dsx035-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> Transposable elements (TEs) were excavated strictly using three softwares, including a <em>de novo</em> software Repeat Modeler (<a class="link link-uri" href="http://repeatmasker.org/RepeatModeler" target="_blank">http://repeatmasker.org/RepeatModeler/</a>) and two database-based softwares Repeat Protein Masker (<a class="link link-uri" href="http://www.repeatmasker.org/cgi-bin/RepeatProteinMaskRequest" target="_blank">www.repeatmasker.org/cgi-bin/RepeatProteinMaskRequest</a>) and Repeat Masker (<a class="link link-uri" href="http://www.repeatmasker.org" target="_blank">www.repeatmasker.org/</a>). All the parameters are set as default. Whole-genome DNA modification detection and motif analysis were performed according to Blow et al.<a reveal-id="dsx035-B35" data-open="dsx035-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> using the PacBio SMRT software (version = 2.2.3, www.pacb.com). </p>                            <h3 scrollto-destination=117294499 id="117294499" class="section-title" >2.5. Phylogenetic analysis and synteny analysis</h3>
<p>The sequences of fungi were downloaded from DOE Joint Genome Institute (JGI).<a reveal-id="dsx035-B36" data-open="dsx035-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> A group of consistent phylogenetic ‘backbone’ genes (phylogeneticly conserved) in the fungi genome were used to construct phylogenetic tree.<a reveal-id="dsx035-B37" data-open="dsx035-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> Putative ‘backbone’ genes of the other 11 fungi were identified using stand-alone blast with E-values of ≤1e−20. And the ‘backbone’ genes were concatenated into one sequence. Sequence alignment was done using MUSCLE<a reveal-id="dsx035-B38" data-open="dsx035-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> and the phylogenetic tree was generated by MEGA 7.0<a reveal-id="dsx035-B39" data-open="dsx035-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> using a UPGMA method. <em>Ustilago maydis</em> was used as outgroup. Synteny of <em>Cercospora zeae-maydis</em>, <em>C. sojina</em>, and <em>Pseudocercospora fijiensis</em> was analysed using GATA.<a reveal-id="dsx035-B40" data-open="dsx035-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a></p>                            <h3 scrollto-destination=117294501 id="117294501" class="section-title" >2.6. Comparison analysis of carbohydrate-active enzymes and secondary metabolism genes</h3>
<p>The proteome of <em>C. sojina</em> and 14 other above-mentioned fungal species were downloaded from DOE Joint Genome Institute (JGI).<a reveal-id="dsx035-B36" data-open="dsx035-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> HMMER 3.0 packages were used for homology search.<a reveal-id="dsx035-B41" data-open="dsx035-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> Family-specific HMM profiles were downloaded from dbCAN database.<a reveal-id="dsx035-B42" data-open="dsx035-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> The executable file hmmscan and the hmmscan-parser script provided by dbCAN were used to generate and extract the searching results, respectively.</p><p>Putative polyketide synthases (PKS) and non-ribosomal peptide synthases (NRPS) genes were identified using the web-based software SMURF with default settings.<a reveal-id="dsx035-B43" data-open="dsx035-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> The modules of different domains in individual NRPS and PKS proteins were identified via searching the antiSMASH database (antibiotics and Secondary Metabolite Analysis Shell).<a reveal-id="dsx035-B44" data-open="dsx035-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> The core genes were annotated using stand-alone BLAST (E-values ≤1e−10) against Swiss-Prot database.</p>                            <h3 scrollto-destination=117294504 id="117294504" class="section-title" >2.7. Secondary metabolites extraction and quantification</h3>
<p>Cercosporin extraction and quantification were performed according to the method described by Shim and Dunkle.<a reveal-id="dsx035-B45" data-open="dsx035-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> The pigments of <em>C. sojina</em> were induced in the complete medium (CM) with 20 mmol/l cyclic adenosine monophosphate (cAMP) or starvation treatments for 4 days at 28°C. The supernatants were collected and purified by the C18-SPE cartridge. The elutes in 40% methanol fraction were further purified by a HPLC on C18 preparation column. The fractions with grey, light yellow, and dark grey were further identified by a reverse phase HPLC with a PDA detector (Shimadzu Corporation, Japan).</p>                            <h3 scrollto-destination=117294506 id="117294506" class="section-title" >2.8. Transcriptome analysis and quantitative RT-PCR</h3>
<p>For transcriptome analysis, the fungus was grown in minimal nutrient medium (3 g NaNO<sub>3</sub>, 1 g K<sub>2</sub>HPO<sub>4</sub>, 0.5 g MgSO<sub>4</sub>⋅7H<sub>2</sub>O, 0.5 g KCl, 0.01 g FeSO<sub>4</sub>, and 30 g Sucrose per litre) at 28°C for 6 days. For starvation treatment, the fungal mycelia was transferred to minimal nutrient medium lacking NaNO<sub>3</sub>, then was harvested at 24 and 48 h, respectively. Each treatment had three biological replicates.</p><p>Library preparation and bioinformatics analysis were performed according to the method of Yang et al.<a reveal-id="dsx035-B46" data-open="dsx035-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> One microgram RNA per sample was subjected to RNA-seq library construction. The RNA-seq libraries were quantified using 2100 Bioanalyzer (Agilent, USA), and sequenced (paired-end, 100 bp each) by the Illumina genome analyzer (Hiseq 2000; Illumina, USA). Quantification of gene expression levels were estimated by fragments per kilobase of transcript per million fragments mapped. Differential expression analysis was performed using the DESeq R package.</p><p>For quantitative RT-PCR, infected leaves were collected at indicated time points. Total RNA was extracted by Trizol method (Invitrogen). The cDNAs were synthesized using HiScript II Q RT SuperMix kit with genomic DNA wiper (Vazyme Biotech). Reactions were performed on CFX96<sup>TM</sup> Real-time System (Bio-RAD) with the SYBR qPCR Mix (Vazyme Biotech). The 2<sup>−</sup><sup>ΔΔCT</sup> method was used for calculating the relative gene expression levels. <em>Actin</em> gene of <em>C. sojina</em> was used as the internal control.</p>                            <h3 scrollto-destination=117294510 id="117294510" class="section-title" >2.9. Functional study of putative effectors</h3>
<p>The putative effector genes were cloned into binary vector pMD-1 (T7 tag) driven by 35S promoter using ClonExpress II One Step Cloning Kit (Vazyme Biotech). Then, these constructs were transformed into <em>Agrobacterium tumefaciens</em> strain EHA105 by electroporation. Leaves of 4-week-old <em>Nicotiana benthamiana</em> were infiltrated with EHA105 strains harbouring indicated effectors using needleless syringes. GFP and <em>Phytophothora sojae</em> effector Avr1b served as the negative control and positive control, respectively. Twenty-four hours later, plants were infiltrated with EHA105 harbouring pVX-BAX (OD<sub>600</sub> = 0.4). Cell death symptoms were evaluated and photographed at 72 h after pVX-BAX infiltration. Results are representatives of six biological replicates.</p>                            <h2 scrollto-destination=117294512 id="117294512" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=117294513 id="117294513" class="section-title" >3.1. Assembly of <em>C. sojina</em> genome</h3>
<p>Like most of the fungi, <em>C. sojina</em> showed similar infection cycle, but it also demonstrated some distinctions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s1_dsx035.png?Expires=2147483647&Signature=qJjhcd6gnL0yEg8KHMZgFR1Vn7mua1fBvzyU93lE~Fe0wgTkQx1yrrwFsPNFlJXDoFYM8StFeY6w-NI7gbqDUiTW23h-vRGcp1-quFJe3MaEatHREWBHiW14cq22pdwdV2GrD9EN4~BfjcLObKfu86-QPcLmfKZk-oUjAi1Wd7ZXy2vsNozqLmqiX9Mu1YrMkCPb~71-ka39RB58-d0bofOyQJLmyPeQ7Dz2qyt9-qSLSovX-94abqWxLCXG7pAflXAwI7lT5M9~xneIDTQF-IHbKEXXQMnW8VUxpAuITuSVpM7-Z6jsMcxfiGOruTl5roNOiVRE40DbgNFHILhdHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). It does not form appressorium, but infects the plants by branched hyphae through open stomata (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s1_dsx035.png?Expires=2147483647&Signature=qJjhcd6gnL0yEg8KHMZgFR1Vn7mua1fBvzyU93lE~Fe0wgTkQx1yrrwFsPNFlJXDoFYM8StFeY6w-NI7gbqDUiTW23h-vRGcp1-quFJe3MaEatHREWBHiW14cq22pdwdV2GrD9EN4~BfjcLObKfu86-QPcLmfKZk-oUjAi1Wd7ZXy2vsNozqLmqiX9Mu1YrMkCPb~71-ka39RB58-d0bofOyQJLmyPeQ7Dz2qyt9-qSLSovX-94abqWxLCXG7pAflXAwI7lT5M9~xneIDTQF-IHbKEXXQMnW8VUxpAuITuSVpM7-Z6jsMcxfiGOruTl5roNOiVRE40DbgNFHILhdHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1C and D</a></span>). Compared with other hemibiotrophic fungi, FLS disease development is relatively slower (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s1_dsx035.png?Expires=2147483647&Signature=qJjhcd6gnL0yEg8KHMZgFR1Vn7mua1fBvzyU93lE~Fe0wgTkQx1yrrwFsPNFlJXDoFYM8StFeY6w-NI7gbqDUiTW23h-vRGcp1-quFJe3MaEatHREWBHiW14cq22pdwdV2GrD9EN4~BfjcLObKfu86-QPcLmfKZk-oUjAi1Wd7ZXy2vsNozqLmqiX9Mu1YrMkCPb~71-ka39RB58-d0bofOyQJLmyPeQ7Dz2qyt9-qSLSovX-94abqWxLCXG7pAflXAwI7lT5M9~xneIDTQF-IHbKEXXQMnW8VUxpAuITuSVpM7-Z6jsMcxfiGOruTl5roNOiVRE40DbgNFHILhdHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1C</a></span>). In order to investigate the infection mechanism, we therefore extracted the genomic DNA from the mycelia and sequenced the pathogen at genome level.</p><p>The genome of <em>C. sojina</em> race 1 was assembled from the data generated by a recently developed <span class="underline">S</span>ingle <span class="underline">M</span>olecule <span class="underline">R</span>eal-<span class="underline">T</span>ime (SMRT) sequencing technique, an effective method to decode the difficulty to detect but important regions, such as non-coding regions and repetitive elements, which can assist in obtaining gapless eukaryotic genome sequence.<a reveal-id="dsx035-B20" data-open="dsx035-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a></p><p>Subreads distribution analyses confirm the high quality of the 10-kb library (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s2_dsx035.png?Expires=2147483647&amp;Signature=GAmhNNkPPorptimeeGfE7Qa9FV0FGH06W11B3pArRgxVSQnAOcbYmGKt5FbqPKpINmHNqyr8zP0G2LataT7PDkaSFVAoPqISefjL2k5pWwdUMRNQe6IR9sdL0yILcgR9zqgd9TtJ7MPLTZohTydZzcJ~mhNvmlRiucL6X6EdTGN3AR1mj-7TXER6WrA2L481eJRW4cRVXQRIqiR2jYMLW8hmd87Al6I7ytZeC1aC~uXZ~P7gALM-TXz3lbbPmCFe2yN8946PUPavE07LdsIVtjmU2I~Nw2B-EZC9W94F7t7VtLWsJmbhx82zvisoagefOLiribmzBE9LdQFOH~XiIw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035_supplemental_table.pdf?Expires=2147483647&amp;Signature=XlkQOdSYFRueozu21xQI6RlMVIoRZIHZIEX1uhZQa5Xp0fHWe9WI1vIRTUN03DUwOOW-mIdJPPE3v2MpkX98iXRQpyVFImKFX3IdNILEQq5rd6R6nJqBBfGUrASpTDEXsxPIUUYapxiIbm25MuP17HKmMAHgUeF2cLA711JW6F28a2kPH-3cwuKwsP-m7Fsyl2ZQK5bJMjp9P1uuI0VzmKjeqfGJV3SWyP7QrHehB3Gz1RI0ajnnj-W9J9N32N31pH4FKIqgjOHaho5bNVQsbQ3uhjj3FRzn65eAZ3304cFTsnZLJXF4LqyHXoEJLaM9~S1N-RuN~raOEiPsEfqIqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Table S1</a></span>). The sequencing data (4,920,479,113 bp clean reads) were <em>de novo</em> assembled using CANU,<a reveal-id="dsx035-B21" data-open="dsx035-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> leading to the generation of 62 contigs, with an N50 length of 1.59 Mb and a total assembly size around ∼40.84 Mb (<a reveal-id="dsx035-T1" data-open="dsx035-T1" class="link link-reveal link-table xref-fig">Table 1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035_supplemental_table.pdf?Expires=2147483647&amp;Signature=XlkQOdSYFRueozu21xQI6RlMVIoRZIHZIEX1uhZQa5Xp0fHWe9WI1vIRTUN03DUwOOW-mIdJPPE3v2MpkX98iXRQpyVFImKFX3IdNILEQq5rd6R6nJqBBfGUrASpTDEXsxPIUUYapxiIbm25MuP17HKmMAHgUeF2cLA711JW6F28a2kPH-3cwuKwsP-m7Fsyl2ZQK5bJMjp9P1uuI0VzmKjeqfGJV3SWyP7QrHehB3Gz1RI0ajnnj-W9J9N32N31pH4FKIqgjOHaho5bNVQsbQ3uhjj3FRzn65eAZ3304cFTsnZLJXF4LqyHXoEJLaM9~S1N-RuN~raOEiPsEfqIqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Twenty-four largest scaffolds were displayed by circos-plot (<a reveal-id="dsx035-F1" data-open="dsx035-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). A total of 11,655 protein-coding genes are predicted, in which the gene density is ∼285 genes per 1 Mb. However, 277 tRNA and 281 pseudogenes are predicted in the genome. Notably, 8,474 putative protein-coding genes were supported by the RNA-seq data. It is also worth noting that the genome size of <em>C. sojina</em> race 1 is much larger than <em>C. sojina</em> isolate S9, where the genome was assembled by 124 bp library and genome size was estimated around ∼30.8 M.<a reveal-id="dsx035-B47" data-open="dsx035-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a><div content-id="dsx035-T1" class="table-modal"><div class="table-wrap"><div id="dsx035-T1" data-id="dsx035-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Genome features of <em>Cercospora sojina</em></p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Features </th><th><em>C. sojina</em> </th></tr></thead><tbody><tr><td>Size (bp) </td><td>40,835,411 </td></tr><tr><td>Coverage </td><td>120× </td></tr><tr><td>(G+C) percentage (%) </td><td>53.12 </td></tr><tr><td>N50 (bp) </td><td>1,594,385 </td></tr><tr><td>Protein-coding genes </td><td>11,655 </td></tr><tr><td>Average gene length (bp) </td><td>1,441 </td></tr><tr><td>Gene density (no. gene per Mb) </td><td>285 </td></tr><tr><td>tRNA genes </td><td>277 </td></tr><tr><td>Pseudogene </td><td>281 </td></tr></tbody></table></div><div content-id="dsx035-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Features </th><th><em>C. sojina</em> </th></tr></thead><tbody><tr><td>Size (bp) </td><td>40,835,411 </td></tr><tr><td>Coverage </td><td>120× </td></tr><tr><td>(G+C) percentage (%) </td><td>53.12 </td></tr><tr><td>N50 (bp) </td><td>1,594,385 </td></tr><tr><td>Protein-coding genes </td><td>11,655 </td></tr><tr><td>Average gene length (bp) </td><td>1,441 </td></tr><tr><td>Gene density (no. gene per Mb) </td><td>285 </td></tr><tr><td>tRNA genes </td><td>277 </td></tr><tr><td>Pseudogene </td><td>281 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsx035-T1" data-id="dsx035-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Genome features of <em>Cercospora sojina</em></p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Features </th><th><em>C. sojina</em> </th></tr></thead><tbody><tr><td>Size (bp) </td><td>40,835,411 </td></tr><tr><td>Coverage </td><td>120× </td></tr><tr><td>(G+C) percentage (%) </td><td>53.12 </td></tr><tr><td>N50 (bp) </td><td>1,594,385 </td></tr><tr><td>Protein-coding genes </td><td>11,655 </td></tr><tr><td>Average gene length (bp) </td><td>1,441 </td></tr><tr><td>Gene density (no. gene per Mb) </td><td>285 </td></tr><tr><td>tRNA genes </td><td>277 </td></tr><tr><td>Pseudogene </td><td>281 </td></tr></tbody></table></div><div content-id="dsx035-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Features </th><th><em>C. sojina</em> </th></tr></thead><tbody><tr><td>Size (bp) </td><td>40,835,411 </td></tr><tr><td>Coverage </td><td>120× </td></tr><tr><td>(G+C) percentage (%) </td><td>53.12 </td></tr><tr><td>N50 (bp) </td><td>1,594,385 </td></tr><tr><td>Protein-coding genes </td><td>11,655 </td></tr><tr><td>Average gene length (bp) </td><td>1,441 </td></tr><tr><td>Gene density (no. gene per Mb) </td><td>285 </td></tr><tr><td>tRNA genes </td><td>277 </td></tr><tr><td>Pseudogene </td><td>281 </td></tr></tbody></table></div></div></p>                        <a id="117294517" scrollto-destination="117294517"></a>
<div id="dsx035-F1" data-id="dsx035-F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/m_dsx035f1.png?Expires=2147483647&Signature=gmVYdw4eL7gizl~iAVaZNKnBKy16M2g2ICKkIKxjxqWvSay6SPfIVnNa5AC1WqmA-PqN~AYEkZmIcupC0tYwDJIK9UXfX0sY72Rn4PR3dCG~2SMtk9FPPNoiqzwPPuil0R37eflYV2atFeLr0chPlHgS~AJ29T~PetssdAkz6d~asF3IQkkHxweSTgKOgPhtTiSvUWqUP8LOYjD9DW2FLwPYyCbfbZ3aeVrwtldNhYAzj~KD04YpB4P-jsBZ-1~TYSt1nIkEAF7ONazCriyuEcV1AUTJyJ0B5MVDafPzVMnsx6Kx2WDgkxUh2ZYTnOs9rG0w9xic1G1q6qFo0HU0bw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Circos-plot of C. sojina. The largest 24 scaffolds of C. sojina are displayed by circos-plot (Mb scale). The circos from outside to inside are: (a) 24 largest scaffolds; (b) DNA methylations; (c) GC content; (d) carbohydrate enzymes; (e). putative effectors; (f) PHI-base genes; (g) duplicated genes; The DNA methylations and GC contents are statistical results of 20 kb non-overlapping windows. The inner lines link duplicated genes." path-from-xml="dsx035f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/117294517/dsx035f1.tif" path-from-xml="dsx035f1.tif" target="_blank">View large</a><a section="117294517" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035f1.png?Expires=2147483647&Signature=zjUL1joyMuNt9~AiKhsjSvu3LmjJlsUQCywckLyKhCmfdskzapEy5BsjvsHWMSZuwLj6ZUdDAk-Z1oICZwkQry5x3jXM7Lb5nwDcAXfQQBDfEpXkyNlPS~5i4wTvMkhDXIYQh6hRDgBq0AFEEEBW7mJGtttC4Ka-N7acirRmeLfdqvVnQMdLN8fDkzibczp8DJm0lGvGv-0dLIQJuA5I3zecW3MUNMo9-zarn8kcBuitXaolxP1QCVglN0FTCYq8MNq6TsWlzjitRZswSDM8k6jXMWAsSTS0pJR4RuecGJvyGDvdvwnplHx0oarYxbVRqpMogA9mZShGgMuaofaKWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=117294517&ar=4161430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx035f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Circos-plot of <em>C. sojina</em>. The largest 24 scaffolds of <em>C. sojina</em> are displayed by circos-plot (Mb scale). The circos from outside to inside are: (a) 24 largest scaffolds; (b) DNA methylations; (c) GC content; (d) carbohydrate enzymes; (e). putative effectors; (f) PHI-base genes; (g) duplicated genes; The DNA methylations and GC contents are statistical results of 20 kb non-overlapping windows. The inner lines link duplicated genes.</p></div></div><div content-id="dsx035-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/m_dsx035f1.png?Expires=2147483647&Signature=gmVYdw4eL7gizl~iAVaZNKnBKy16M2g2ICKkIKxjxqWvSay6SPfIVnNa5AC1WqmA-PqN~AYEkZmIcupC0tYwDJIK9UXfX0sY72Rn4PR3dCG~2SMtk9FPPNoiqzwPPuil0R37eflYV2atFeLr0chPlHgS~AJ29T~PetssdAkz6d~asF3IQkkHxweSTgKOgPhtTiSvUWqUP8LOYjD9DW2FLwPYyCbfbZ3aeVrwtldNhYAzj~KD04YpB4P-jsBZ-1~TYSt1nIkEAF7ONazCriyuEcV1AUTJyJ0B5MVDafPzVMnsx6Kx2WDgkxUh2ZYTnOs9rG0w9xic1G1q6qFo0HU0bw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Circos-plot of C. sojina. The largest 24 scaffolds of C. sojina are displayed by circos-plot (Mb scale). The circos from outside to inside are: (a) 24 largest scaffolds; (b) DNA methylations; (c) GC content; (d) carbohydrate enzymes; (e). putative effectors; (f) PHI-base genes; (g) duplicated genes; The DNA methylations and GC contents are statistical results of 20 kb non-overlapping windows. The inner lines link duplicated genes." path-from-xml="dsx035f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/117294517/dsx035f1.tif" path-from-xml="dsx035f1.tif" target="_blank">View large</a><a section="117294517" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035f1.png?Expires=2147483647&Signature=zjUL1joyMuNt9~AiKhsjSvu3LmjJlsUQCywckLyKhCmfdskzapEy5BsjvsHWMSZuwLj6ZUdDAk-Z1oICZwkQry5x3jXM7Lb5nwDcAXfQQBDfEpXkyNlPS~5i4wTvMkhDXIYQh6hRDgBq0AFEEEBW7mJGtttC4Ka-N7acirRmeLfdqvVnQMdLN8fDkzibczp8DJm0lGvGv-0dLIQJuA5I3zecW3MUNMo9-zarn8kcBuitXaolxP1QCVglN0FTCYq8MNq6TsWlzjitRZswSDM8k6jXMWAsSTS0pJR4RuecGJvyGDvdvwnplHx0oarYxbVRqpMogA9mZShGgMuaofaKWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=117294517&ar=4161430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx035f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Circos-plot of <em>C. sojina</em>. The largest 24 scaffolds of <em>C. sojina</em> are displayed by circos-plot (Mb scale). The circos from outside to inside are: (a) 24 largest scaffolds; (b) DNA methylations; (c) GC content; (d) carbohydrate enzymes; (e). putative effectors; (f) PHI-base genes; (g) duplicated genes; The DNA methylations and GC contents are statistical results of 20 kb non-overlapping windows. The inner lines link duplicated genes.</p></div></div>                            <h3 scrollto-destination=117294518 id="117294518" class="section-title" >3.2. Repetitive elements and potential methylation sites</h3>
<p>Repetitive DNA sequence and TEs play important roles in the evolution, the genome structure, and gene functions of fungi.<a reveal-id="dsx035-B48" data-open="dsx035-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> A total of 11,138,239 bp (∼11 M) repeat sequences were identified in <em>C. sojina</em> genome, including DNA transposon, LTR retrotransposon, tandem repeat sequence and other unclassified transposons (<a reveal-id="dsx035-F2" data-open="dsx035-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). The repeat sequence accounts for 25.56% of the genome. Interestingly, the majority of repetitive sequences (96.36%) are TEs, whereas the tandem repeat sequences just account for 0.93%. Notably, DNA transposon and LTR retrotransposons account for 28 and 25% of all TEs, respectively. </p>                        <a id="117294520" scrollto-destination="117294520"></a>
<div id="dsx035-F2" data-id="dsx035-F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/m_dsx035f2.png?Expires=2147483647&Signature=Smb1ZI7J6mAJWXovxjOPqOQ-xVHICoHZ4td67mcT-mLTty820DNEwK3yyXl03muY-8ji~uLmesMXCyVD1uxwOj4k583Sp-nS8-Fbm8k05cVZCz7WnVp18OiDgwxHwOjQKoWmWlMwrC8sClKfw7~mrYByQ655gmDo64Ei-el9FOpFd896R8X3Len1T-yeGm3TA6duTCrI0er3oWz7LpjGOi35iZStcEDsp80wvnVJzF4r3FZekhkkolp0pwOlbe6oLqaaxgmkOmkPKp0~qJAP5XhQc2iFDcmk5ZuGiVVdh-~XK7LTURq9VNe5O7Lf3bIGtvzBLLt842~tGGqkT3HFZg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Repeat elements and DNA methylation sites of C. sojina. (A) The percentage of different types of repetitive sequences in the C. sojina genome. (B) Statistic analysis of candidate DNA methylation sites from primary sequence of C. sojina genome. m4C, m6A, and the unidentified represent 4-methyl-cytosine, 6-methyl-adenosine, and the unidentified methylation sites, respectively. (C) Distribution of repetitive elements and different types of DNA methylations in scaffold 1 of C. sojina. Black histogram indicates the distribution of repetitive elements. All data are statistical results of 20 kb windows. Asterisks indicate regions with high and low frequency of DNA methylations, respectively. (D, E) The number of different type of methylations per 20 kb was calculated in total genome and repetitive elements of C. sojina." path-from-xml="dsx035f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/117294520/dsx035f2.tif" path-from-xml="dsx035f2.tif" target="_blank">View large</a><a section="117294520" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035f2.png?Expires=2147483647&Signature=UcfxhmlgIj6~jf~KKFch39Ych2xLXFpIy2ZNUXMT41al7DpiyZPFeFtCXjTJRy66OV2rLEPOA8~WCLC8MvjvQQYTKsTOW-fRAPUt8y6z37il93RWGTqe6~zHViIW1A6k9iEJsJ9E2f0h5TsONMEPr9ethf67yBcuyHbvwfu-eh2CIqW3WdEaqpzG-8h2TMolqg5U~AcPU0zZ1AIylaWzrj6WGLlK25uskl0u9-5VSr2WzCle0QZbObpaPtABLAGqX8mEiJ7i7Abyrs7U0yXo5xzmO8zOOnvRlFE-Pzmr1DU7cJzPQEQB~P9XP1WkIgdu9ZBo1bDJEq8-H5TyM8ffAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=117294520&ar=4161430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx035f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Repeat elements and DNA methylation sites of <em>C. sojina.</em> (A) The percentage of different types of repetitive sequences in the <em>C. sojina</em> genome. (B) Statistic analysis of candidate DNA methylation sites from primary sequence of <em>C. sojina</em> genome<em>.</em> m4C, m6A, and the unidentified represent 4-methyl-cytosine, 6-methyl-adenosine, and the unidentified methylation sites, respectively. (C) Distribution of repetitive elements and different types of DNA methylations in scaffold 1 of <em>C. sojina</em>. Black histogram indicates the distribution of repetitive elements. All data are statistical results of 20 kb windows. Asterisks indicate regions with high and low frequency of DNA methylations, respectively. (D, E) The number of different type of methylations per 20 kb was calculated in total genome and repetitive elements of <em>C. sojina</em>.</p></div></div><div content-id="dsx035-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/m_dsx035f2.png?Expires=2147483647&Signature=Smb1ZI7J6mAJWXovxjOPqOQ-xVHICoHZ4td67mcT-mLTty820DNEwK3yyXl03muY-8ji~uLmesMXCyVD1uxwOj4k583Sp-nS8-Fbm8k05cVZCz7WnVp18OiDgwxHwOjQKoWmWlMwrC8sClKfw7~mrYByQ655gmDo64Ei-el9FOpFd896R8X3Len1T-yeGm3TA6duTCrI0er3oWz7LpjGOi35iZStcEDsp80wvnVJzF4r3FZekhkkolp0pwOlbe6oLqaaxgmkOmkPKp0~qJAP5XhQc2iFDcmk5ZuGiVVdh-~XK7LTURq9VNe5O7Lf3bIGtvzBLLt842~tGGqkT3HFZg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Repeat elements and DNA methylation sites of C. sojina. (A) The percentage of different types of repetitive sequences in the C. sojina genome. (B) Statistic analysis of candidate DNA methylation sites from primary sequence of C. sojina genome. m4C, m6A, and the unidentified represent 4-methyl-cytosine, 6-methyl-adenosine, and the unidentified methylation sites, respectively. (C) Distribution of repetitive elements and different types of DNA methylations in scaffold 1 of C. sojina. Black histogram indicates the distribution of repetitive elements. All data are statistical results of 20 kb windows. Asterisks indicate regions with high and low frequency of DNA methylations, respectively. (D, E) The number of different type of methylations per 20 kb was calculated in total genome and repetitive elements of C. sojina." path-from-xml="dsx035f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/117294520/dsx035f2.tif" path-from-xml="dsx035f2.tif" target="_blank">View large</a><a section="117294520" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035f2.png?Expires=2147483647&Signature=UcfxhmlgIj6~jf~KKFch39Ych2xLXFpIy2ZNUXMT41al7DpiyZPFeFtCXjTJRy66OV2rLEPOA8~WCLC8MvjvQQYTKsTOW-fRAPUt8y6z37il93RWGTqe6~zHViIW1A6k9iEJsJ9E2f0h5TsONMEPr9ethf67yBcuyHbvwfu-eh2CIqW3WdEaqpzG-8h2TMolqg5U~AcPU0zZ1AIylaWzrj6WGLlK25uskl0u9-5VSr2WzCle0QZbObpaPtABLAGqX8mEiJ7i7Abyrs7U0yXo5xzmO8zOOnvRlFE-Pzmr1DU7cJzPQEQB~P9XP1WkIgdu9ZBo1bDJEq8-H5TyM8ffAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=117294520&ar=4161430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx035f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Repeat elements and DNA methylation sites of <em>C. sojina.</em> (A) The percentage of different types of repetitive sequences in the <em>C. sojina</em> genome. (B) Statistic analysis of candidate DNA methylation sites from primary sequence of <em>C. sojina</em> genome<em>.</em> m4C, m6A, and the unidentified represent 4-methyl-cytosine, 6-methyl-adenosine, and the unidentified methylation sites, respectively. (C) Distribution of repetitive elements and different types of DNA methylations in scaffold 1 of <em>C. sojina</em>. Black histogram indicates the distribution of repetitive elements. All data are statistical results of 20 kb windows. Asterisks indicate regions with high and low frequency of DNA methylations, respectively. (D, E) The number of different type of methylations per 20 kb was calculated in total genome and repetitive elements of <em>C. sojina</em>.</p></div></div><p>DNA methylation is involved in many important cell processes, such as genomic imprinting and gene transcription regulation.<a reveal-id="dsx035-B49" data-open="dsx035-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> Although DNA methylation has been found in higher plants and animals for years, it is just reported in some fungi recently.<a reveal-id="dsx035-B50" data-open="dsx035-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> Using SMRT, we were able to detect m6A and m4C methylation in particular.<a reveal-id="dsx035-B51" data-open="dsx035-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> In total, 1,015,733 m4C (4-methyl-cytosine) and 17,409 m6A (6-methyl-adenosine) were identified in <em>C. sojina</em> genome (<a reveal-id="dsx035-F2" data-open="dsx035-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). However, majority of methylation sites (8,453,041) were uncategorized. Interestingly, most of the categorized DNA methylations are m4C, accounting for 98.3%, whereas m6A only accounts for 1.7%. In consistent with the identified methylation sites, we detect multiple motifs that may be recognized by transmethylase specifically (<span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035_supplemental_table.pdf?Expires=2147483647&amp;Signature=XlkQOdSYFRueozu21xQI6RlMVIoRZIHZIEX1uhZQa5Xp0fHWe9WI1vIRTUN03DUwOOW-mIdJPPE3v2MpkX98iXRQpyVFImKFX3IdNILEQq5rd6R6nJqBBfGUrASpTDEXsxPIUUYapxiIbm25MuP17HKmMAHgUeF2cLA711JW6F28a2kPH-3cwuKwsP-m7Fsyl2ZQK5bJMjp9P1uuI0VzmKjeqfGJV3SWyP7QrHehB3Gz1RI0ajnnj-W9J9N32N31pH4FKIqgjOHaho5bNVQsbQ3uhjj3FRzn65eAZ3304cFTsnZLJXF4LqyHXoEJLaM9~S1N-RuN~raOEiPsEfqIqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). However, compared with m6A, m4C DNA methylations occur with low frequency in the regions of repetitive elements (<a reveal-id="dsx035-F2" data-open="dsx035-F2" class="link link-reveal link-table xref-fig">Fig. 2C–E</a>).</p>                            <h3 scrollto-destination=117294522 id="117294522" class="section-title" >3.3. Comparative genomic analysis</h3>
<p>The evolutionary relationship of <em>C. sojina</em> and other fungi species was analysed using a group of phylogenetic backbone genes of the fungi.<a reveal-id="dsx035-B37" data-open="dsx035-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> Phylogenetic analysis reveals that <em>C. sojina</em> is evolutionally close to <em>Cercospora zeae-maydis</em>, a plant pathogen that can cause leaf spot disease on maize<a reveal-id="dsx035-B52" data-open="dsx035-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> (<a reveal-id="dsx035-F3" data-open="dsx035-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). In addition, <em>C. sojina</em> is also close to the other three Dothideomycetes pathogen <em>Pseudocercospora fijiensis</em>, <em>Sphaerulina musiva</em>, and <em>Dothistroma septosporum</em> (<a reveal-id="dsx035-F3" data-open="dsx035-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). The <em>C. sojina</em> homologous proteins show an average identity of 79.9, 65.9, 67.5, and 65.8% with that of <em>C. zeae-maydis</em>, <em>P. fijiensis</em>, <em>S. musiva</em>, and <em>D. septosporum</em>, respectively. </p>                        <a id="117294524" scrollto-destination="117294524"></a>
<div id="dsx035-F3" data-id="dsx035-F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/m_dsx035f3.png?Expires=2147483647&Signature=Wg41qQBEZWQ2gtHi6WOtrRqH-P9NGzkWmfuvk62Iu5ktGq4wdtP-kArGP46gu5DZlP27ESCwG7rjVwP5KkOqN~AnUOS7MwaJu4ssAz3GTWQvZ5lIulniG5BUeQVNF5TnZkr2COGX-S7dHjIr4EDPQHVEmzrFNPMlBzNT2DVAkmB1SkqPAi5MzWggqauRHQMjilaBfgZxaH--EQpTUM8E8EWnKk23gRHVj-GvAZlAh6P0neEz2I5WhzfRb9bz1osxgjt8~lfUEEd5o71TcuRaUqYFj~csYzif1cM1lMFMP~xIC0A69eGN4ifZSqnuVG6HMdJy9lwIV847-Gnkn7pseQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic and synteny analysis of C. sojina with other fungal species. (A) Phylogenetic tree of different fungal species. The UPGMA phylogenetic tree was constructed based on the consistent phylogenetic backbone genes of the fungi. The number represents branch lengths. (B) Synteny of Cercospora zeae-maydis, C. sojina, and Pseudocercospora fijiensis. Rectangle boxes represent order of gene models. Non-coding regions are not depicted." path-from-xml="dsx035f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/117294524/dsx035f3.tif" path-from-xml="dsx035f3.tif" target="_blank">View large</a><a section="117294524" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035f3.png?Expires=2147483647&Signature=zLthjueuSRvFEiAsAMcAhy-3V5lHFip91UEL7R5Be8QSi8jtwrwZOfEuXzpBkw3YYBDYp-Npe9eqjGrW3Cijsoe9znYzN3LPqkqFByJXdmN634YB4fvgI8j8NBuWxjNv9yrk86qlU6uxw1UupOv92UZcS~oyW22lC9aZ~sex4VNTdty-Bdo47VGqt32dxrYaDETg24OrQrLhRjWrKN-kzlwlu628z8qxNmmaowaeYQS9K33G9IDwcWh8n47x~gfvbjxtJ0uQsPfuRV~~QtLTzegH7-WH6J~aCfV9ooqKm5nLufu-wsRXpdha-nKvy50iLA9idB~dNPEBBIoPPt7uDg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=117294524&ar=4161430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx035f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic and synteny analysis of <em>C. sojina</em> with other fungal species. (A) Phylogenetic tree of different fungal species. The UPGMA phylogenetic tree was constructed based on the consistent phylogenetic backbone genes of the fungi. The number represents branch lengths. (B) Synteny of <em>Cercospora zeae-maydis</em>, <em>C. sojina</em>, and <em>Pseudocercospora fijiensis.</em> Rectangle boxes represent order of gene models. Non-coding regions are not depicted.</p></div></div><div content-id="dsx035-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/m_dsx035f3.png?Expires=2147483647&Signature=Wg41qQBEZWQ2gtHi6WOtrRqH-P9NGzkWmfuvk62Iu5ktGq4wdtP-kArGP46gu5DZlP27ESCwG7rjVwP5KkOqN~AnUOS7MwaJu4ssAz3GTWQvZ5lIulniG5BUeQVNF5TnZkr2COGX-S7dHjIr4EDPQHVEmzrFNPMlBzNT2DVAkmB1SkqPAi5MzWggqauRHQMjilaBfgZxaH--EQpTUM8E8EWnKk23gRHVj-GvAZlAh6P0neEz2I5WhzfRb9bz1osxgjt8~lfUEEd5o71TcuRaUqYFj~csYzif1cM1lMFMP~xIC0A69eGN4ifZSqnuVG6HMdJy9lwIV847-Gnkn7pseQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic and synteny analysis of C. sojina with other fungal species. (A) Phylogenetic tree of different fungal species. The UPGMA phylogenetic tree was constructed based on the consistent phylogenetic backbone genes of the fungi. The number represents branch lengths. (B) Synteny of Cercospora zeae-maydis, C. sojina, and Pseudocercospora fijiensis. Rectangle boxes represent order of gene models. Non-coding regions are not depicted." path-from-xml="dsx035f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/117294524/dsx035f3.tif" path-from-xml="dsx035f3.tif" target="_blank">View large</a><a section="117294524" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035f3.png?Expires=2147483647&Signature=zLthjueuSRvFEiAsAMcAhy-3V5lHFip91UEL7R5Be8QSi8jtwrwZOfEuXzpBkw3YYBDYp-Npe9eqjGrW3Cijsoe9znYzN3LPqkqFByJXdmN634YB4fvgI8j8NBuWxjNv9yrk86qlU6uxw1UupOv92UZcS~oyW22lC9aZ~sex4VNTdty-Bdo47VGqt32dxrYaDETg24OrQrLhRjWrKN-kzlwlu628z8qxNmmaowaeYQS9K33G9IDwcWh8n47x~gfvbjxtJ0uQsPfuRV~~QtLTzegH7-WH6J~aCfV9ooqKm5nLufu-wsRXpdha-nKvy50iLA9idB~dNPEBBIoPPt7uDg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=117294524&ar=4161430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx035f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic and synteny analysis of <em>C. sojina</em> with other fungal species. (A) Phylogenetic tree of different fungal species. The UPGMA phylogenetic tree was constructed based on the consistent phylogenetic backbone genes of the fungi. The number represents branch lengths. (B) Synteny of <em>Cercospora zeae-maydis</em>, <em>C. sojina</em>, and <em>Pseudocercospora fijiensis.</em> Rectangle boxes represent order of gene models. Non-coding regions are not depicted.</p></div></div><p>Although <em>C. sojina</em> is evolutionarily distant from the non-plant pathogen <em>Aspergillus nidulans</em> and <em>Neurospora crassa</em>, 47.18 and 43.07% proteins of <em>C. sojina</em> have homologues in <em>A. nidulans</em> and <em>N. crassa</em>, respectively. Therefore, we examined the potential gene family expansions in <em>C. sojina</em>. The results show that 5,652 genes exclusively exist in <em>C. sojina</em> genome but not in <em>A. nidulans</em> and <em>N. crassa</em>. However, near 70% of these genes cannot be annotated from current databases (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s3_dsx035.png?Expires=2147483647&Signature=g9JKyTqOaHoBFBmuB7luI2qZP4hydrljwUd6oqSZvhLyJR8XhnGmqWq~UExdSHgpabtp3ttuVqboIgi~94yQWaazxgevTtmPytIinVDPDelP1x4NOYIaUBBtTAAX54IDxmtUb3Zfpk~wMv6Jz8IaCWqhZFURL~08FhsRHNrA5TLI6hsoKIft0scOvDrnYAaA4gCr~WaAankWaiKy8nXYC4voJe1J~q7BP6v3Kwfbp1wYaxJYkZbvSpDCfKnITmyODIMd04Ei65QWgXA6iCBKCljy6WPAVGZBWITRHAg6HREb9Mxtd1CokNPpjhs1X0ZTS3letrrQH0DWrNGdccEZjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3A</a></span>). Go enrichment analysis of the 1,675 annotated genes reveals that most of the genes are involved in metabolic process, biosynthetic process, and response to stresses or stimuli (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s3_dsx035.png?Expires=2147483647&Signature=g9JKyTqOaHoBFBmuB7luI2qZP4hydrljwUd6oqSZvhLyJR8XhnGmqWq~UExdSHgpabtp3ttuVqboIgi~94yQWaazxgevTtmPytIinVDPDelP1x4NOYIaUBBtTAAX54IDxmtUb3Zfpk~wMv6Jz8IaCWqhZFURL~08FhsRHNrA5TLI6hsoKIft0scOvDrnYAaA4gCr~WaAankWaiKy8nXYC4voJe1J~q7BP6v3Kwfbp1wYaxJYkZbvSpDCfKnITmyODIMd04Ei65QWgXA6iCBKCljy6WPAVGZBWITRHAg6HREb9Mxtd1CokNPpjhs1X0ZTS3letrrQH0DWrNGdccEZjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3B</a></span>). Notably, these genes are predicted to have binding function (ion binding or protein binding), hydrolase activity, transferase activity or oxidoreductase activity (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s3_dsx035.png?Expires=2147483647&Signature=g9JKyTqOaHoBFBmuB7luI2qZP4hydrljwUd6oqSZvhLyJR8XhnGmqWq~UExdSHgpabtp3ttuVqboIgi~94yQWaazxgevTtmPytIinVDPDelP1x4NOYIaUBBtTAAX54IDxmtUb3Zfpk~wMv6Jz8IaCWqhZFURL~08FhsRHNrA5TLI6hsoKIft0scOvDrnYAaA4gCr~WaAankWaiKy8nXYC4voJe1J~q7BP6v3Kwfbp1wYaxJYkZbvSpDCfKnITmyODIMd04Ei65QWgXA6iCBKCljy6WPAVGZBWITRHAg6HREb9Mxtd1CokNPpjhs1X0ZTS3letrrQH0DWrNGdccEZjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3C</a></span>). As <em>C. sojina</em> is evolutionarily distant from the non-plant pathogens, we speculate that the gene family expansion, a common event in the evolution of phytopathogenic fungi,<a reveal-id="dsx035-B52" data-open="dsx035-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a><sup>,</sup><a reveal-id="dsx035-B53" data-open="dsx035-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> might have occurred in <em>C. sojina</em> genome, which eventually makes <em>C. sojina</em> be a plant pathogen.</p><p>In addition, synteny analysis of <em>C. sojina</em> genome with the other three genomes of <em>Dothideomycetes</em> spp., the <em>C. zeae-maydis</em>, <em>P. fijiensis</em>, and <em>Mycosphaerella graminicola</em>, reveals that <em>C. sojina</em> genome displays different synteny with those fungi (<a reveal-id="dsx035-F3" data-open="dsx035-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a> and <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s4_dsx035.png?Expires=2147483647&Signature=KT0T7B-LuGPET1zIQdqiYRExTAgP4jrgM0Xn4nQidKfvueB9wW3Dc0PrXHIbVMTy5~tp~3LnLzjBD0KOdoHPDeXpZYUjru9TnDNmIrNzWM-NdtUKVance5mJ~xf6h1P9YialMXyMa7zsytFE-P~BW-yyw7Tl4TTUfMGWLVh4parGhRxTR2CapSIUznurF5a0mopbHCW9gj7pH5lZeou9fQZSjIkQxERGYKegDioc6xrMs3CuAfahxte7Oohj~0k3l6yLlFjASKN~9TVSezFFvjVczoxcEM7zE6JU3Z2aQqqEjdRZbKmb~N6OBkBhmutzSukjLe5b~fZgBIX4dZDAwQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). Of all the sequenced genomes, <em>C. zeae-maydis</em> shows highest synteny with <em>C. sojina</em>. For example, scaffolds 2, 4, and 5 of <em>C. zeae-maydis</em> correspond well with the scaffold 1 of <em>C. sojina</em>, and Scaffold 3 and 10 show well syntney to the scaffold 3 of <em>C. sojina</em> (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s4_dsx035.png?Expires=2147483647&Signature=KT0T7B-LuGPET1zIQdqiYRExTAgP4jrgM0Xn4nQidKfvueB9wW3Dc0PrXHIbVMTy5~tp~3LnLzjBD0KOdoHPDeXpZYUjru9TnDNmIrNzWM-NdtUKVance5mJ~xf6h1P9YialMXyMa7zsytFE-P~BW-yyw7Tl4TTUfMGWLVh4parGhRxTR2CapSIUznurF5a0mopbHCW9gj7pH5lZeou9fQZSjIkQxERGYKegDioc6xrMs3CuAfahxte7Oohj~0k3l6yLlFjASKN~9TVSezFFvjVczoxcEM7zE6JU3Z2aQqqEjdRZbKmb~N6OBkBhmutzSukjLe5b~fZgBIX4dZDAwQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4A</a></span>). Importantly, we observed that the 24 largest scaffolds, which accounts for 91.2% of <em>C. sojina</em> genome, show very high synteny with the 13 core chromosomes of <em>M. graminicola</em>,<a reveal-id="dsx035-B53" data-open="dsx035-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> but not with the rest 8 dispersed chromosomes (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s4_dsx035.png?Expires=2147483647&Signature=KT0T7B-LuGPET1zIQdqiYRExTAgP4jrgM0Xn4nQidKfvueB9wW3Dc0PrXHIbVMTy5~tp~3LnLzjBD0KOdoHPDeXpZYUjru9TnDNmIrNzWM-NdtUKVance5mJ~xf6h1P9YialMXyMa7zsytFE-P~BW-yyw7Tl4TTUfMGWLVh4parGhRxTR2CapSIUznurF5a0mopbHCW9gj7pH5lZeou9fQZSjIkQxERGYKegDioc6xrMs3CuAfahxte7Oohj~0k3l6yLlFjASKN~9TVSezFFvjVczoxcEM7zE6JU3Z2aQqqEjdRZbKmb~N6OBkBhmutzSukjLe5b~fZgBIX4dZDAwQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4C</a></span>), indicating that <em>C. sojina</em> shares the conserved and core genes of <em>Dothideomycetes</em>.</p>                            <h3 scrollto-destination=117294527 id="117294527" class="section-title" >3.4. The secretome and potential effectors</h3>
<p>The genome of <em>C. sojina</em> contains 11,655 protein-coding genes, covering approximately 41% sequence of the genome (<a reveal-id="dsx035-T1" data-open="dsx035-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Among them, a total of 9,506 genes were annotated using multiple databases (<span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s5_dsx035.png?Expires=2147483647&amp;Signature=QB2xqD5EpuxgUgGPesAsk2Ok4ytAQfhqXhcF-vbvDnvL-BYrU6lvQkIrwxUZAaqQiTgnoUP~ZK~YTcIia4nsZMXeidTY5AnvKBxSVIWMnLfHz8DltOOXeMNC8AhJEtGCRanj5mcjNbp6jpJgBYRGNpwZq8hi-4xblvbwEFbScvY0DzE2xG-34iWvaJlZx0xUbPuMDSR0W26jCofEssqAa2-htPGdB8UFvvJVvhzoWKeL2zdpj~f2~Z3SHONeApRKhs07GlE6NcSzYAyBffSkbOwxtjnk~TQpjSvjAu88dAAH-zulhTKVRqtZloMYT6eXpJBFCz5-jgaKPLbLTrukQw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035_supplemental_table.pdf?Expires=2147483647&amp;Signature=XlkQOdSYFRueozu21xQI6RlMVIoRZIHZIEX1uhZQa5Xp0fHWe9WI1vIRTUN03DUwOOW-mIdJPPE3v2MpkX98iXRQpyVFImKFX3IdNILEQq5rd6R6nJqBBfGUrASpTDEXsxPIUUYapxiIbm25MuP17HKmMAHgUeF2cLA711JW6F28a2kPH-3cwuKwsP-m7Fsyl2ZQK5bJMjp9P1uuI0VzmKjeqfGJV3SWyP7QrHehB3Gz1RI0ajnnj-W9J9N32N31pH4FKIqgjOHaho5bNVQsbQ3uhjj3FRzn65eAZ3304cFTsnZLJXF4LqyHXoEJLaM9~S1N-RuN~raOEiPsEfqIqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Table S4</a></span>). Domain calling analysis reveals that more than 7,500 types of domains exist in <em>C. sojina</em> proteome. The arsenal of potentially secreted proteins were predicted, and proteins containing a signal peptide, but lacking transmembrane domain and glycosylphosphatidylinositol (GPI) modification site were considered as secreted proteins. A combination of software tools for the prediction of transmembrane domain,<a reveal-id="dsx035-B54" data-open="dsx035-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> signal peptide motifs,<a reveal-id="dsx035-B55" data-open="dsx035-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> and GPI modification site indicate that <em>C. sojina</em> has similar number of potential secreted proteins (∼750) when compared with other fungal species.<a reveal-id="dsx035-B56" data-open="dsx035-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a></p><p>Pathogen-secreted effectors play critical roles in facilitating the proliferation of pathogens, often by suppressing plant immune system.<a reveal-id="dsx035-B57" data-open="dsx035-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> In total, 233 proteins are predicted as the putative small (≤400 amino acids) cysteine-rich (≥4 cysteine residues) proteins. Through domain calling analysis, 205 functional motifs and domains were found in 141 putative effectors, including 60 effectors with multiple domains. Notably, the most abundant domain is PF14295.4 (<em>n</em> = 6), which mediates protein−protein interactions. Other common domains include abhydrolase domain (PF12697.5, <em>n</em> = 5), Hydrolase domain (PF12146.6, <em>n</em> = 5), and PAN domain (PF00024.24, <em>n</em> = 5).</p>                            <h3 scrollto-destination=117294530 id="117294530" class="section-title" >3.5. Up-regulation of pathogenicity-related genes by whole genome transcription assays</h3>
<p>Because the infection progress of <em>C. sojina</em> on soybean is very slow, it is difficult to collect enough samples to examine the gene expression of the in planta hyphae. However, starvation treatments could mimic the physiology of pathogen during infection.<a reveal-id="dsx035-B58" data-open="dsx035-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> Therefore, we used the mycelia that were grown in nutrient-limited culture for 24 and 48 h, and performed transcriptome analysis by RNA sequencing; 3,227 and 3,223 differentially expressed genes (DEGs) were identified at 24 and 48 h post-starvation treatment (hpt), respectively (<span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s6_dsx035.png?Expires=2147483647&Signature=XL8ea0QNTf4JST3i0Fs1zpAceVRCNnf8A8jvyVZLj9iT1EXboJH~ch1UbdaSh3y0i3NvywYnx-XsOplZxexCvyNeCwlE5lNLPZ6NDgEL8xBclg8GRbCSq4zN~2gafZSIcx2vnO8J4vWFVWRm2MMC1f0hhYVqpJTwOa~w0g9unId4JFhCWV4vqcef93QgjNTzJP5B3j9mvpiCUFL6x2qZQzAdbZYIba8vmWlV0HDnsMwZGOp5RvaI~VLCLvI2~YoJeTHhGkh68PUNwOwN9-BGUR6ppSYcDS7YW7vZzF5X2kp2QSgDsbMBYZgr407Rqy~ceH0XeSZ5VpzLy0JkD7fh3Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). A total of 4,051 DEGs were identified during starvation treatment, and 2,399 genes were differentially expressed at both 24 and 48 hpt. Of all the DEGs, 1,530 and 1,508 genes were upregulated, while 1,697 and 1,715 genes were downregulated at 24 and 48 hpt, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s6_dsx035.png?Expires=2147483647&Signature=XL8ea0QNTf4JST3i0Fs1zpAceVRCNnf8A8jvyVZLj9iT1EXboJH~ch1UbdaSh3y0i3NvywYnx-XsOplZxexCvyNeCwlE5lNLPZ6NDgEL8xBclg8GRbCSq4zN~2gafZSIcx2vnO8J4vWFVWRm2MMC1f0hhYVqpJTwOa~w0g9unId4JFhCWV4vqcef93QgjNTzJP5B3j9mvpiCUFL6x2qZQzAdbZYIba8vmWlV0HDnsMwZGOp5RvaI~VLCLvI2~YoJeTHhGkh68PUNwOwN9-BGUR6ppSYcDS7YW7vZzF5X2kp2QSgDsbMBYZgr407Rqy~ceH0XeSZ5VpzLy0JkD7fh3Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>).</p><p>Notably, four classes of DEGs caused our attention. These genes are annotated to be involved in PHI, secretome, putative carbohydrate-active enzymes (CAZymes), and secondary metabolic processes. First, 1,036 PHI genes are differentially expressed after starvation treatment (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s7_dsx035.png?Expires=2147483647&amp;Signature=hv4jFu1qBL5AAhCZB-E0qY1Npzi4ZKbmxR5RXJkFvbid2723XnzFzZqRlY07FWNRQi~VjisGGfCVsS3a08eQqq3vY8xP0njSK8Q6detsOuh7-9yiYyMZwyokAgBs7IXd9OyFI3VgsAGHb8Dp1UQZ2Ztcr1nysOfkEjeP4tynWENxj3TnZNsXZ24~RnJBVaztSvNCs0PcstO4zyVUnX5Zz~JnXZ47DspXPlV4NVhVEj88QJj0BMm4QW2LCuqSE7~8CyznMXPKgJ7HG6mj1Vv~UxCt8kKj2hTrxT5bXhEil1SHMEP1SWg-58bJJlh~BZV3CjLZBcJLyAQHHSQX18HT~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7A</a></span>). A total of 591 PHI genes are significantly upregulated, demonstrating the important roles of these genes in responding to stimulus. Second, 260 secreted protein-coding genes, including 81 effectors, are differentially expressed (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s7_dsx035.png?Expires=2147483647&amp;Signature=hv4jFu1qBL5AAhCZB-E0qY1Npzi4ZKbmxR5RXJkFvbid2723XnzFzZqRlY07FWNRQi~VjisGGfCVsS3a08eQqq3vY8xP0njSK8Q6detsOuh7-9yiYyMZwyokAgBs7IXd9OyFI3VgsAGHb8Dp1UQZ2Ztcr1nysOfkEjeP4tynWENxj3TnZNsXZ24~RnJBVaztSvNCs0PcstO4zyVUnX5Zz~JnXZ47DspXPlV4NVhVEj88QJj0BMm4QW2LCuqSE7~8CyznMXPKgJ7HG6mj1Vv~UxCt8kKj2hTrxT5bXhEil1SHMEP1SWg-58bJJlh~BZV3CjLZBcJLyAQHHSQX18HT~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7B</a></span>). There is 62.5% (50/80) effector-coding gene expression being significantly upregulated, and some effectors with conserved domains, such as Wall Stress-responsive Component domain, glycoside hydrolase, fungal hydrophobin, cutinase, leucine rich repeat or peptidase domains, may play critical roles in fungal pathogenicity (<span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035_supplemental_table.pdf?Expires=2147483647&amp;Signature=XlkQOdSYFRueozu21xQI6RlMVIoRZIHZIEX1uhZQa5Xp0fHWe9WI1vIRTUN03DUwOOW-mIdJPPE3v2MpkX98iXRQpyVFImKFX3IdNILEQq5rd6R6nJqBBfGUrASpTDEXsxPIUUYapxiIbm25MuP17HKmMAHgUeF2cLA711JW6F28a2kPH-3cwuKwsP-m7Fsyl2ZQK5bJMjp9P1uuI0VzmKjeqfGJV3SWyP7QrHehB3Gz1RI0ajnnj-W9J9N32N31pH4FKIqgjOHaho5bNVQsbQ3uhjj3FRzn65eAZ3304cFTsnZLJXF4LqyHXoEJLaM9~S1N-RuN~raOEiPsEfqIqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Third, 198 CAZymes were differentially expressed. Interestingly, almost half of them are glycoside hydrolases (87/198), implying their essential roles in early infection (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s7_dsx035.png?Expires=2147483647&amp;Signature=hv4jFu1qBL5AAhCZB-E0qY1Npzi4ZKbmxR5RXJkFvbid2723XnzFzZqRlY07FWNRQi~VjisGGfCVsS3a08eQqq3vY8xP0njSK8Q6detsOuh7-9yiYyMZwyokAgBs7IXd9OyFI3VgsAGHb8Dp1UQZ2Ztcr1nysOfkEjeP4tynWENxj3TnZNsXZ24~RnJBVaztSvNCs0PcstO4zyVUnX5Zz~JnXZ47DspXPlV4NVhVEj88QJj0BMm4QW2LCuqSE7~8CyznMXPKgJ7HG6mj1Vv~UxCt8kKj2hTrxT5bXhEil1SHMEP1SWg-58bJJlh~BZV3CjLZBcJLyAQHHSQX18HT~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7C</a></span>). Further, the secondary metabolism-related genes, including 5 PKS and 16 NRPS/NRPS-like genes are significantly upregulated (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s7_dsx035.png?Expires=2147483647&amp;Signature=hv4jFu1qBL5AAhCZB-E0qY1Npzi4ZKbmxR5RXJkFvbid2723XnzFzZqRlY07FWNRQi~VjisGGfCVsS3a08eQqq3vY8xP0njSK8Q6detsOuh7-9yiYyMZwyokAgBs7IXd9OyFI3VgsAGHb8Dp1UQZ2Ztcr1nysOfkEjeP4tynWENxj3TnZNsXZ24~RnJBVaztSvNCs0PcstO4zyVUnX5Zz~JnXZ47DspXPlV4NVhVEj88QJj0BMm4QW2LCuqSE7~8CyznMXPKgJ7HG6mj1Vv~UxCt8kKj2hTrxT5bXhEil1SHMEP1SWg-58bJJlh~BZV3CjLZBcJLyAQHHSQX18HT~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7D</a></span>). These genes are likely involved in mycotoxin biosynthesis in fungi. Therefore, the increased expression of PKS and NRPS/NRPS-like genes suggests that these genes may play essential roles in mycotoxins biosynthesis (<span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035_supplemental_table.pdf?Expires=2147483647&amp;Signature=XlkQOdSYFRueozu21xQI6RlMVIoRZIHZIEX1uhZQa5Xp0fHWe9WI1vIRTUN03DUwOOW-mIdJPPE3v2MpkX98iXRQpyVFImKFX3IdNILEQq5rd6R6nJqBBfGUrASpTDEXsxPIUUYapxiIbm25MuP17HKmMAHgUeF2cLA711JW6F28a2kPH-3cwuKwsP-m7Fsyl2ZQK5bJMjp9P1uuI0VzmKjeqfGJV3SWyP7QrHehB3Gz1RI0ajnnj-W9J9N32N31pH4FKIqgjOHaho5bNVQsbQ3uhjj3FRzn65eAZ3304cFTsnZLJXF4LqyHXoEJLaM9~S1N-RuN~raOEiPsEfqIqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>).</p>                            <h3 scrollto-destination=117294533 id="117294533" class="section-title" >3.6. Gene clusters for secondary metabolites</h3>
<p><em>Cercospora</em><em>sojina</em> genome encodes 16 non-ribosomal peptide synthetases (NRPS), 20 PKS, 18 fatty acid synthases, 3 terpene synthases, 2 geranylgeranyl diphosphate synthases, and 1 terpenoid cyclases (<span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035_supplemental_table.pdf?Expires=2147483647&amp;Signature=XlkQOdSYFRueozu21xQI6RlMVIoRZIHZIEX1uhZQa5Xp0fHWe9WI1vIRTUN03DUwOOW-mIdJPPE3v2MpkX98iXRQpyVFImKFX3IdNILEQq5rd6R6nJqBBfGUrASpTDEXsxPIUUYapxiIbm25MuP17HKmMAHgUeF2cLA711JW6F28a2kPH-3cwuKwsP-m7Fsyl2ZQK5bJMjp9P1uuI0VzmKjeqfGJV3SWyP7QrHehB3Gz1RI0ajnnj-W9J9N32N31pH4FKIqgjOHaho5bNVQsbQ3uhjj3FRzn65eAZ3304cFTsnZLJXF4LqyHXoEJLaM9~S1N-RuN~raOEiPsEfqIqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S7</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035_supplemental_table.pdf?Expires=2147483647&amp;Signature=XlkQOdSYFRueozu21xQI6RlMVIoRZIHZIEX1uhZQa5Xp0fHWe9WI1vIRTUN03DUwOOW-mIdJPPE3v2MpkX98iXRQpyVFImKFX3IdNILEQq5rd6R6nJqBBfGUrASpTDEXsxPIUUYapxiIbm25MuP17HKmMAHgUeF2cLA711JW6F28a2kPH-3cwuKwsP-m7Fsyl2ZQK5bJMjp9P1uuI0VzmKjeqfGJV3SWyP7QrHehB3Gz1RI0ajnnj-W9J9N32N31pH4FKIqgjOHaho5bNVQsbQ3uhjj3FRzn65eAZ3304cFTsnZLJXF4LqyHXoEJLaM9~S1N-RuN~raOEiPsEfqIqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S8</a></span>). These enzymes are involved in synthesis of secondary metabolites, including mycotoxins, pigments, and alkaloids. Annotation of secondary metabolite biosynthesis genes shows that <em>C. sojina</em> lacks PKS-NRPS hybrids, PKS-like proteins, and dimethylallyl tryptophan synthases (<span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035_supplemental_table.pdf?Expires=2147483647&amp;Signature=XlkQOdSYFRueozu21xQI6RlMVIoRZIHZIEX1uhZQa5Xp0fHWe9WI1vIRTUN03DUwOOW-mIdJPPE3v2MpkX98iXRQpyVFImKFX3IdNILEQq5rd6R6nJqBBfGUrASpTDEXsxPIUUYapxiIbm25MuP17HKmMAHgUeF2cLA711JW6F28a2kPH-3cwuKwsP-m7Fsyl2ZQK5bJMjp9P1uuI0VzmKjeqfGJV3SWyP7QrHehB3Gz1RI0ajnnj-W9J9N32N31pH4FKIqgjOHaho5bNVQsbQ3uhjj3FRzn65eAZ3304cFTsnZLJXF4LqyHXoEJLaM9~S1N-RuN~raOEiPsEfqIqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S7</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035_supplemental_table.pdf?Expires=2147483647&amp;Signature=XlkQOdSYFRueozu21xQI6RlMVIoRZIHZIEX1uhZQa5Xp0fHWe9WI1vIRTUN03DUwOOW-mIdJPPE3v2MpkX98iXRQpyVFImKFX3IdNILEQq5rd6R6nJqBBfGUrASpTDEXsxPIUUYapxiIbm25MuP17HKmMAHgUeF2cLA711JW6F28a2kPH-3cwuKwsP-m7Fsyl2ZQK5bJMjp9P1uuI0VzmKjeqfGJV3SWyP7QrHehB3Gz1RI0ajnnj-W9J9N32N31pH4FKIqgjOHaho5bNVQsbQ3uhjj3FRzn65eAZ3304cFTsnZLJXF4LqyHXoEJLaM9~S1N-RuN~raOEiPsEfqIqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S8</a></span>).</p><p>In the <em>Cercospora</em> genus, most of the species can produce a non-specific mycotoxin cercosporin. However, it has been disputed that if <em>C. sojina</em> produces cercosporin.<a reveal-id="dsx035-B9" data-open="dsx035-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Nevertheless, we identified a similar gene cluster with eight cercosporin biosynthesis genes in <em>C. sojina</em> genome (<a reveal-id="dsx035-F4" data-open="dsx035-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>). These eight genes display high amino acid sequence similarity to <em>C. nicotianae</em> cercosporin biosynthesis genes in the same tandem order (<a reveal-id="dsx035-F4" data-open="dsx035-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>). Furthermore, we observed the increased transcription of the eight genes during infection (<a reveal-id="dsx035-F4" data-open="dsx035-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>). These data imply that <em>C. sojina</em> may produce cercosporin during infection. However, we were unable to detect the cercosporin in either cultured mycelium or infected plant tissue according to the method that was used in other <em>Cercospora</em> species (<span class="content-section supplementary-material"><a path-from-xml="sup8" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s8_dsx035.png?Expires=2147483647&Signature=xYLzLoFDodQVSlRs4eZVkzTLlaQnC24DmAbcPBwx-uLz7LIbkhgkutbzV0GQQPNk2Zsm4JIOwIgpZhHm0pxOCra2hjNgSOq-hJDBDNHuR-bz7dw7ycivNRPmmYY7LeC0k-gZ8iPe~26TQ1YrswuFDTj~2WKNHtAkXgs3hSAQ1hMcvdo9sEMEAcWDLk18ni8eKhTnySTy3PTj39QggibYD05obykXOs1M0PDuUD0TPVBazH5fvRSB-0PG~9XxxooCiKbzKznMY-ddG7M19uTsOR72vT1uWu29QCexFP5Tca6ZNSybL6MZz6qe5kgsnTQa9eybNzT25CwwnW5som8juw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>). </p>                        <a id="117294536" scrollto-destination="117294536"></a>
<div id="dsx035-F4" data-id="dsx035-F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/m_dsx035f4.png?Expires=2147483647&Signature=1aUVBJp~aRsQUkBGYdSuYWwPRsC8-OsLQODviBcKE7uXpAxE0fv-l3RQV~RLB0MSYSsHh8GtKvJBkbrKIdAAEdddehYsuWU1BbxyfKJPFKcUkmdaz98RVdtrODO9aSqAG5jBi2iOmjN~SABJXSqaaLvdvSAvQ12FbV5fkB6gTm1W76RFfwACROHFkRqaY9i7ArayBT0jWpQJa2AtBlo6O97Z0It~uAx1FC9fwfcRQwsugMlZTzUgxNeIT8axLhbvsDel3sjtSsv0AKYwMNw5nIN2fdstIwa0HsgYr~lRkII2ashpdWYoQDMFfq5j2wfF~u8qsGaglIzWCr02Rcm~pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Putative gene clusters for cercosporin biosynthesis in C. sojina. (A) The eight cercosporin toxin biosynthetic genes in C. sojina genome. The genes involved in cercosporin biosynthesis of Cercospora nicotianae (C. nicotianae) were blasted against C. sojina using BlastP. The black arrow represents the direction of sense strand. (B) Expression of candidate genes involved in cercosporin biosynthesis at 48 h after C. sojina infection. The mRNA levels were detected using qRT-PCR. Values are means ± SD (n = 3 biological replicates)." path-from-xml="dsx035f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/117294536/dsx035f4.tif" path-from-xml="dsx035f4.tif" target="_blank">View large</a><a section="117294536" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035f4.png?Expires=2147483647&Signature=Jms8e0ZEwtVf6lH1JiC6NJ4ch-Y5dJNG6VKpMISbx5-EfOOya-TD9c4nG-KKrqAx7ujCrYUfBzBYwEHnsNW7-D8Rayy6q4ZRyRl3EtwH7zLs9Csjjo5cZuIvhvt3L0Gr9UMiw3Yzyv9XkT1c3ug7MVIweWzYeqmEgKXyxn6YKhqU5HE1ZSq4zq02r-lN9s0auhaPv8ODN9F3lNtxc2puVTvmPgXcYVipUte-a9p2Xkq6tEUczfNDkzpGnxPYu5rNH225hF3ovFsXFnPjfm1Eiz4VGLFoO-Jn~oL0DZUM~NLHrB5m63ic3EuMMVL67UzWVPUfNRVOCh~xMQMpn-FCKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=117294536&ar=4161430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx035f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Putative gene clusters for cercosporin biosynthesis in <em>C. sojina.</em> (A) The eight cercosporin toxin biosynthetic genes in <em>C. sojina</em> genome. The genes involved in cercosporin biosynthesis of <em>Cercospora nicotianae</em> (<em>C. nicotianae</em>) were blasted against <em>C. sojina</em> using BlastP. The black arrow represents the direction of sense strand. (B) Expression of candidate genes involved in cercosporin biosynthesis at 48 h after <em>C. sojina</em> infection. The mRNA levels were detected using qRT-PCR. Values are means<strong><span class="small-caps"> </span></strong>±<strong><span class="small-caps"> </span></strong>SD (<em>n</em><strong><span class="small-caps"> </span></strong>=<strong><span class="small-caps"> </span></strong>3 biological replicates).</p></div></div><div content-id="dsx035-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/m_dsx035f4.png?Expires=2147483647&Signature=1aUVBJp~aRsQUkBGYdSuYWwPRsC8-OsLQODviBcKE7uXpAxE0fv-l3RQV~RLB0MSYSsHh8GtKvJBkbrKIdAAEdddehYsuWU1BbxyfKJPFKcUkmdaz98RVdtrODO9aSqAG5jBi2iOmjN~SABJXSqaaLvdvSAvQ12FbV5fkB6gTm1W76RFfwACROHFkRqaY9i7ArayBT0jWpQJa2AtBlo6O97Z0It~uAx1FC9fwfcRQwsugMlZTzUgxNeIT8axLhbvsDel3sjtSsv0AKYwMNw5nIN2fdstIwa0HsgYr~lRkII2ashpdWYoQDMFfq5j2wfF~u8qsGaglIzWCr02Rcm~pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Putative gene clusters for cercosporin biosynthesis in C. sojina. (A) The eight cercosporin toxin biosynthetic genes in C. sojina genome. The genes involved in cercosporin biosynthesis of Cercospora nicotianae (C. nicotianae) were blasted against C. sojina using BlastP. The black arrow represents the direction of sense strand. (B) Expression of candidate genes involved in cercosporin biosynthesis at 48 h after C. sojina infection. The mRNA levels were detected using qRT-PCR. Values are means ± SD (n = 3 biological replicates)." path-from-xml="dsx035f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/117294536/dsx035f4.tif" path-from-xml="dsx035f4.tif" target="_blank">View large</a><a section="117294536" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035f4.png?Expires=2147483647&Signature=Jms8e0ZEwtVf6lH1JiC6NJ4ch-Y5dJNG6VKpMISbx5-EfOOya-TD9c4nG-KKrqAx7ujCrYUfBzBYwEHnsNW7-D8Rayy6q4ZRyRl3EtwH7zLs9Csjjo5cZuIvhvt3L0Gr9UMiw3Yzyv9XkT1c3ug7MVIweWzYeqmEgKXyxn6YKhqU5HE1ZSq4zq02r-lN9s0auhaPv8ODN9F3lNtxc2puVTvmPgXcYVipUte-a9p2Xkq6tEUczfNDkzpGnxPYu5rNH225hF3ovFsXFnPjfm1Eiz4VGLFoO-Jn~oL0DZUM~NLHrB5m63ic3EuMMVL67UzWVPUfNRVOCh~xMQMpn-FCKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=117294536&ar=4161430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx035f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Putative gene clusters for cercosporin biosynthesis in <em>C. sojina.</em> (A) The eight cercosporin toxin biosynthetic genes in <em>C. sojina</em> genome. The genes involved in cercosporin biosynthesis of <em>Cercospora nicotianae</em> (<em>C. nicotianae</em>) were blasted against <em>C. sojina</em> using BlastP. The black arrow represents the direction of sense strand. (B) Expression of candidate genes involved in cercosporin biosynthesis at 48 h after <em>C. sojina</em> infection. The mRNA levels were detected using qRT-PCR. Values are means<strong><span class="small-caps"> </span></strong>±<strong><span class="small-caps"> </span></strong>SD (<em>n</em><strong><span class="small-caps"> </span></strong>=<strong><span class="small-caps"> </span></strong>3 biological replicates).</p></div></div><p>Pigments are the other important group of secondary metabolites for successful invasion of pathogens. Generally, pathogen-produced pigment is able to protect pathogen from host oxidative stress during infection.<a reveal-id="dsx035-B59" data-open="dsx035-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> We found that <em>C. sojina</em> genome encodes multiple putative PKS that are responsible for pigment production (<a reveal-id="dsx035-F5" data-open="dsx035-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a> and <span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035_supplemental_table.pdf?Expires=2147483647&amp;Signature=XlkQOdSYFRueozu21xQI6RlMVIoRZIHZIEX1uhZQa5Xp0fHWe9WI1vIRTUN03DUwOOW-mIdJPPE3v2MpkX98iXRQpyVFImKFX3IdNILEQq5rd6R6nJqBBfGUrASpTDEXsxPIUUYapxiIbm25MuP17HKmMAHgUeF2cLA711JW6F28a2kPH-3cwuKwsP-m7Fsyl2ZQK5bJMjp9P1uuI0VzmKjeqfGJV3SWyP7QrHehB3Gz1RI0ajnnj-W9J9N32N31pH4FKIqgjOHaho5bNVQsbQ3uhjj3FRzn65eAZ3304cFTsnZLJXF4LqyHXoEJLaM9~S1N-RuN~raOEiPsEfqIqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). We also found that <em>C. sojina</em> can produce some grey pigments, and the pigment was significantly induced by both starvation and cAMP treatments (<a reveal-id="dsx035-F5" data-open="dsx035-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>), suggesting that the pigments may be related to pathogen virulence. Therefore, we further isolated and partially purified the pigments. Three major components, the grey, the light yellow, and the dark grey pigments were obtained (<span class="content-section supplementary-material"><a path-from-xml="sup9" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s9_dsx035.png?Expires=2147483647&amp;Signature=bO24WWrNJHJkh~KHOdX5jAAxzQRNjL83gYmx0KKWgYdvSc33GtALEFwLsolGZG61cGSVNfmScbEn6JY0w2xBV87vXc5Sh-bWMbnbmCD4JprPDcoHjmaSOAYo3IczuNFIHVOara9w7kwnNvw2V6iMkvCgIZYaL3t9P8o7b66r7C6SD20O6qktBYAK0cHs1oYuZiyyPHzUco-Vmz8vwZThQ3xNPuLi6A-A7vg05klEM~Q296ZI246loy0IBfP7TnX53WYIf2geQ2KID54DkFUhpBEpgT0QuCTm2rUUub9T~7GTRsySuxBchXbHkpbW764z~05b~Pmvt9XmFQTxRoHlzQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>), and the dark grey pigment is the most abundant one. </p>                        <a id="117294538" scrollto-destination="117294538"></a>
<div id="dsx035-F5" data-id="dsx035-F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/m_dsx035f5.png?Expires=2147483647&Signature=f8C-n7nWODWjNC9eSgKhSBjXKqOBDzZyVZv89vB6BqvDrifXZIWaMySY40Cqk3Ds6P-9IAK7ePwqkp3uhZXxT2xWL4LehFP2BAHAInHZa4XTTn0IEhjRd8IceMGwaUzE8qb6rKAXRY6XSGYg1oVLPnsCNUlPDW71zEvy5ppPkCFxMxw4t2y99geFoezqMzYzlfj4Sby-dfzbwx1l9GV~~TJkVtOQkaTEp3V6OH5V9OeKMWDG2bPM1YZm0m9EtP2WyuNfnAwEy8uR40-acD2PJ9j4fbFSTDii6gUVVvaNEfQXhKcEiCt6Sec3DnNVo5j~lQdFCphG5hPYSQEdgafG1Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Putative PKS gene clusters for pigment production in C. sojina. (A) Genes encoding methyltransferases, cytochrome P450s, oxidoreductases, dehydrogenases, acyltransferases, MFS transporters, and transcriptional factors were clustered with PKS genes. These clusters are responsible for the pigment synthesis. The black arrows represent the direction of sense chain. The orange arrows highlight PKS genes. The blue and grey arrows represent putative pigment biosynthesis genes and other genes that are not involved in pigment biosynthesis. (B) Pigments produced by C. sojina in different media. The supernatants were collected at 4 days after treatments on mycelia. Czapek medium served as a negative control. Results are representatives of three biological replicates. PDB and CM represent potato-dextrose broth medium and complete medium, respectively." path-from-xml="dsx035f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/117294538/dsx035f5.tif" path-from-xml="dsx035f5.tif" target="_blank">View large</a><a section="117294538" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035f5.png?Expires=2147483647&Signature=LbstpSFCzldtSnicK24F5Hi4fRtrO6Z9oikMBVkapekX1hHed7dRSvCpVUTaDlqXz0MHdDi9Umg6IeWcTMPelM7L5v0CjATyClapugVnlJl2MHSguaWyxJ5QutnvkWdUVVr0XuVMrGf8jtLaOM1IJBhT64BFA-BuAVUT-w2j5waZVKGtk7aEe3lEvu~dOUmYvhnxi3AB2-R19u9T3RR5SYezXTd7UyblokpFBbWKmuZAfHATZ4q0zToN0YUaV9Ci3bOAgr8u68a6MFir4KJ9~AP7~2JBx~BFI1OCngX6m5MXuTUkGGsHfbqvCbDg4SdW1N2RXSbC17TbAgRY3iEo2A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=117294538&ar=4161430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx035f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Putative PKS gene clusters for pigment production in <em>C. sojina</em>. (A) Genes encoding methyltransferases, cytochrome P450s, oxidoreductases, dehydrogenases, acyltransferases, MFS transporters, and transcriptional factors were clustered with PKS genes. These clusters are responsible for the pigment synthesis. The black arrows represent the direction of sense chain. The orange arrows highlight PKS genes. The blue and grey arrows represent putative pigment biosynthesis genes and other genes that are not involved in pigment biosynthesis. (B) Pigments produced by <em>C. sojina</em> in different media. The supernatants were collected at 4 days after treatments on mycelia. Czapek medium served as a negative control. Results are representatives of three biological replicates. PDB and CM represent potato-dextrose broth medium and complete medium, respectively.</p></div></div><div content-id="dsx035-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/m_dsx035f5.png?Expires=2147483647&Signature=f8C-n7nWODWjNC9eSgKhSBjXKqOBDzZyVZv89vB6BqvDrifXZIWaMySY40Cqk3Ds6P-9IAK7ePwqkp3uhZXxT2xWL4LehFP2BAHAInHZa4XTTn0IEhjRd8IceMGwaUzE8qb6rKAXRY6XSGYg1oVLPnsCNUlPDW71zEvy5ppPkCFxMxw4t2y99geFoezqMzYzlfj4Sby-dfzbwx1l9GV~~TJkVtOQkaTEp3V6OH5V9OeKMWDG2bPM1YZm0m9EtP2WyuNfnAwEy8uR40-acD2PJ9j4fbFSTDii6gUVVvaNEfQXhKcEiCt6Sec3DnNVo5j~lQdFCphG5hPYSQEdgafG1Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Putative PKS gene clusters for pigment production in C. sojina. (A) Genes encoding methyltransferases, cytochrome P450s, oxidoreductases, dehydrogenases, acyltransferases, MFS transporters, and transcriptional factors were clustered with PKS genes. These clusters are responsible for the pigment synthesis. The black arrows represent the direction of sense chain. The orange arrows highlight PKS genes. The blue and grey arrows represent putative pigment biosynthesis genes and other genes that are not involved in pigment biosynthesis. (B) Pigments produced by C. sojina in different media. The supernatants were collected at 4 days after treatments on mycelia. Czapek medium served as a negative control. Results are representatives of three biological replicates. PDB and CM represent potato-dextrose broth medium and complete medium, respectively." path-from-xml="dsx035f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/117294538/dsx035f5.tif" path-from-xml="dsx035f5.tif" target="_blank">View large</a><a section="117294538" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035f5.png?Expires=2147483647&Signature=LbstpSFCzldtSnicK24F5Hi4fRtrO6Z9oikMBVkapekX1hHed7dRSvCpVUTaDlqXz0MHdDi9Umg6IeWcTMPelM7L5v0CjATyClapugVnlJl2MHSguaWyxJ5QutnvkWdUVVr0XuVMrGf8jtLaOM1IJBhT64BFA-BuAVUT-w2j5waZVKGtk7aEe3lEvu~dOUmYvhnxi3AB2-R19u9T3RR5SYezXTd7UyblokpFBbWKmuZAfHATZ4q0zToN0YUaV9Ci3bOAgr8u68a6MFir4KJ9~AP7~2JBx~BFI1OCngX6m5MXuTUkGGsHfbqvCbDg4SdW1N2RXSbC17TbAgRY3iEo2A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=117294538&ar=4161430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx035f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Putative PKS gene clusters for pigment production in <em>C. sojina</em>. (A) Genes encoding methyltransferases, cytochrome P450s, oxidoreductases, dehydrogenases, acyltransferases, MFS transporters, and transcriptional factors were clustered with PKS genes. These clusters are responsible for the pigment synthesis. The black arrows represent the direction of sense chain. The orange arrows highlight PKS genes. The blue and grey arrows represent putative pigment biosynthesis genes and other genes that are not involved in pigment biosynthesis. (B) Pigments produced by <em>C. sojina</em> in different media. The supernatants were collected at 4 days after treatments on mycelia. Czapek medium served as a negative control. Results are representatives of three biological replicates. PDB and CM represent potato-dextrose broth medium and complete medium, respectively.</p></div></div>                            <h3 scrollto-destination=117294539 id="117294539" class="section-title" >3.7. Carbohydrate-active enzymes</h3>
<p>Successful phytopathogenic fungi can break down and utilize the plant cell wall polysaccharides by CAZymes. <em>Cercospora</em><em>sojina</em> harbours 596 predicted CAZymes (<span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035_supplemental_table.pdf?Expires=2147483647&amp;Signature=XlkQOdSYFRueozu21xQI6RlMVIoRZIHZIEX1uhZQa5Xp0fHWe9WI1vIRTUN03DUwOOW-mIdJPPE3v2MpkX98iXRQpyVFImKFX3IdNILEQq5rd6R6nJqBBfGUrASpTDEXsxPIUUYapxiIbm25MuP17HKmMAHgUeF2cLA711JW6F28a2kPH-3cwuKwsP-m7Fsyl2ZQK5bJMjp9P1uuI0VzmKjeqfGJV3SWyP7QrHehB3Gz1RI0ajnnj-W9J9N32N31pH4FKIqgjOHaho5bNVQsbQ3uhjj3FRzn65eAZ3304cFTsnZLJXF4LqyHXoEJLaM9~S1N-RuN~raOEiPsEfqIqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). Compared with other fungi in <em>Dothideomycetes</em>, <em>C. sojina</em> has a larger group of potential carbohydrate esterases, which can catalyze the <em>O</em>-de- or <em>N</em>-deacylation of substituted saccharides<a reveal-id="dsx035-B60" data-open="dsx035-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a><sup>,</sup><a reveal-id="dsx035-B61" data-open="dsx035-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035_supplemental_table.pdf?Expires=2147483647&amp;Signature=XlkQOdSYFRueozu21xQI6RlMVIoRZIHZIEX1uhZQa5Xp0fHWe9WI1vIRTUN03DUwOOW-mIdJPPE3v2MpkX98iXRQpyVFImKFX3IdNILEQq5rd6R6nJqBBfGUrASpTDEXsxPIUUYapxiIbm25MuP17HKmMAHgUeF2cLA711JW6F28a2kPH-3cwuKwsP-m7Fsyl2ZQK5bJMjp9P1uuI0VzmKjeqfGJV3SWyP7QrHehB3Gz1RI0ajnnj-W9J9N32N31pH4FKIqgjOHaho5bNVQsbQ3uhjj3FRzn65eAZ3304cFTsnZLJXF4LqyHXoEJLaM9~S1N-RuN~raOEiPsEfqIqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). In <em>C. sojina</em> genome, there are around 23.5% potential secreted proteins (177/752) that were predicted as CAZyme, demonstrating that <em>C. sojina</em> may employ a large group of CAZymes to digest host cell walls during invasion.</p><p>Interestingly, one of the families of CAZymes, the glycoside hydrolase GH109 family, is highly enriched (<a reveal-id="dsx035-F6" data-open="dsx035-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). The biochemical function of GH109 family is proved to be α-<em>N</em>-acetylgalactosaminidase (αNAGAL), which can cleave the terminal alpha-linked <em>N</em>-acetylgalactosamine epitope of blood group A.<a reveal-id="dsx035-B62" data-open="dsx035-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> However, it has been shown that soybean lectin can specifically bind <em>N</em>-acetyl galactosamine, a component of fungal cell wall.<a reveal-id="dsx035-B63" data-open="dsx035-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> These data suggest that αNAGAL may be able to compete with lectin to bind <em>N</em>-acetylgalactosamine. We find that <em>C. sojina</em> encodes 14 putative GH109 genes, which is more than most of fungi, such as <em>M. oryzae</em>, <em>B. cinerea</em>, and <em>N. crassa</em> (<span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035_supplemental_table.pdf?Expires=2147483647&amp;Signature=XlkQOdSYFRueozu21xQI6RlMVIoRZIHZIEX1uhZQa5Xp0fHWe9WI1vIRTUN03DUwOOW-mIdJPPE3v2MpkX98iXRQpyVFImKFX3IdNILEQq5rd6R6nJqBBfGUrASpTDEXsxPIUUYapxiIbm25MuP17HKmMAHgUeF2cLA711JW6F28a2kPH-3cwuKwsP-m7Fsyl2ZQK5bJMjp9P1uuI0VzmKjeqfGJV3SWyP7QrHehB3Gz1RI0ajnnj-W9J9N32N31pH4FKIqgjOHaho5bNVQsbQ3uhjj3FRzn65eAZ3304cFTsnZLJXF4LqyHXoEJLaM9~S1N-RuN~raOEiPsEfqIqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>). These data suggest that the expansion of GH109 family in <em>C. sojina</em> may contribute to overcome lectin-mediated resistance in soybean. </p>                        <a id="117294542" scrollto-destination="117294542"></a>
<div id="dsx035-F6" data-id="dsx035-F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/m_dsx035f6.png?Expires=2147483647&Signature=ihL7-RqosCemSziH3gb7~OrI4Pdq2HhjXGLDGmWPZBTkORxNhuPXZlZaZtjCN7R-~LL8KCWh0o0RY5siJ3ltzsBsUTCkMhj9CQSMReAWNw2Dd8y~Ix~WU5-LTgIfqtOb-JkPrtnDQk85cUZJwao0aQKj3dNEBiMPmUcIz-Ehf2OjnKOifs6GlO83fkkb9ss8YRw5W6gzTP0YlzQDOldtaQhwSbA05i7CK6~8cOhVlNTEtPwHA~B~P58ACV8WFUDPUOJ7Ygf~Z2UwSaFP~bw5l3U~YVjqb4BVqUbv94HnalsbVkDktPl9UniGOQ8K5fjH6zyChu75dQu4EGC1TitjRA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of carbohydrate enzymes between C. sojina and nine other fungal species. Nine species were selected to compare with C. sojina. Pi, Phytophothora infestans; Ps, Phytophothora sojae; Nc, Neurospora crassa; Bc, Botrytis cinerea; Cm, Cercospora zeae-maydis; Cs, Cercospora sojina; An, Aspergillus nidulans; Vd, Verticillium dahliae; Mo, Magnaporthe oryzae; Fg, Fusarium graminearum. GH, glycoside hydrolase; GT, glycosyltransferase; PL, polysaccharide lyase; CE, carbohydrate esterase; AA, auxiliary activity family; CBM, carbohydrate-binding module family. The numbers of gene families were normalized by Z score." path-from-xml="dsx035f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/117294542/dsx035f6.tif" path-from-xml="dsx035f6.tif" target="_blank">View large</a><a section="117294542" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035f6.png?Expires=2147483647&Signature=JfRDiHELXstSRPK0-R0CnPTqY2tMkQQasnHcysjbT1NdXDw-QVnyD4mTtfqmB8fkmz6LN0E1ftrSZ0z3MsfqDeu8vqBJjv4BwoHat9PA7eKtvbM6JQZqTwk3MnT-zh2ypnJeSRTB8ZJwzcx6D37oa8fgpVxwVpkNQumUEHF88~QjuHoYZQ8o-xgsJUZPv7Vw2tA8xPz4mq1OdwQGmzBR~-B4FcPplnXAaY5muxYXr4H02xYyzPU1lKajtCK93bncSIOzAEZplUunyJSx1vsoiowUnGJvaQRnuPv7JUJyM3nErwq~bY1oC9SGWZHuWjcv8EvtAz54kRDZhQufbeWIGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=117294542&ar=4161430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx035f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of carbohydrate enzymes between <em>C. sojina</em> and nine other fungal species. Nine species were selected to compare with <em>C. sojina</em>. Pi, <em>Phytophothora infestans</em>; Ps, <em>Phytophothora sojae</em>; Nc, <em>Neurospora crassa</em>; Bc, <em>Botrytis cinerea</em>; Cm, <em>Cercospora zeae-maydis</em>; Cs, <em>Cercospora sojina</em>; An, <em>Aspergillus nidulans</em>; Vd, <em>Verticillium dahliae</em>; Mo, <em>Magnaporthe oryzae</em>; Fg, <em>Fusarium graminearum</em>. GH, glycoside hydrolase; GT, glycosyltransferase; PL, polysaccharide lyase; CE, carbohydrate esterase; AA, auxiliary activity family; CBM, carbohydrate-binding module family. The numbers of gene families were normalized by <em>Z</em> score.</p></div></div><div content-id="dsx035-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/m_dsx035f6.png?Expires=2147483647&Signature=ihL7-RqosCemSziH3gb7~OrI4Pdq2HhjXGLDGmWPZBTkORxNhuPXZlZaZtjCN7R-~LL8KCWh0o0RY5siJ3ltzsBsUTCkMhj9CQSMReAWNw2Dd8y~Ix~WU5-LTgIfqtOb-JkPrtnDQk85cUZJwao0aQKj3dNEBiMPmUcIz-Ehf2OjnKOifs6GlO83fkkb9ss8YRw5W6gzTP0YlzQDOldtaQhwSbA05i7CK6~8cOhVlNTEtPwHA~B~P58ACV8WFUDPUOJ7Ygf~Z2UwSaFP~bw5l3U~YVjqb4BVqUbv94HnalsbVkDktPl9UniGOQ8K5fjH6zyChu75dQu4EGC1TitjRA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of carbohydrate enzymes between C. sojina and nine other fungal species. Nine species were selected to compare with C. sojina. Pi, Phytophothora infestans; Ps, Phytophothora sojae; Nc, Neurospora crassa; Bc, Botrytis cinerea; Cm, Cercospora zeae-maydis; Cs, Cercospora sojina; An, Aspergillus nidulans; Vd, Verticillium dahliae; Mo, Magnaporthe oryzae; Fg, Fusarium graminearum. GH, glycoside hydrolase; GT, glycosyltransferase; PL, polysaccharide lyase; CE, carbohydrate esterase; AA, auxiliary activity family; CBM, carbohydrate-binding module family. The numbers of gene families were normalized by Z score." path-from-xml="dsx035f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/117294542/dsx035f6.tif" path-from-xml="dsx035f6.tif" target="_blank">View large</a><a section="117294542" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035f6.png?Expires=2147483647&Signature=JfRDiHELXstSRPK0-R0CnPTqY2tMkQQasnHcysjbT1NdXDw-QVnyD4mTtfqmB8fkmz6LN0E1ftrSZ0z3MsfqDeu8vqBJjv4BwoHat9PA7eKtvbM6JQZqTwk3MnT-zh2ypnJeSRTB8ZJwzcx6D37oa8fgpVxwVpkNQumUEHF88~QjuHoYZQ8o-xgsJUZPv7Vw2tA8xPz4mq1OdwQGmzBR~-B4FcPplnXAaY5muxYXr4H02xYyzPU1lKajtCK93bncSIOzAEZplUunyJSx1vsoiowUnGJvaQRnuPv7JUJyM3nErwq~bY1oC9SGWZHuWjcv8EvtAz54kRDZhQufbeWIGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=117294542&ar=4161430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx035f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of carbohydrate enzymes between <em>C. sojina</em> and nine other fungal species. Nine species were selected to compare with <em>C. sojina</em>. Pi, <em>Phytophothora infestans</em>; Ps, <em>Phytophothora sojae</em>; Nc, <em>Neurospora crassa</em>; Bc, <em>Botrytis cinerea</em>; Cm, <em>Cercospora zeae-maydis</em>; Cs, <em>Cercospora sojina</em>; An, <em>Aspergillus nidulans</em>; Vd, <em>Verticillium dahliae</em>; Mo, <em>Magnaporthe oryzae</em>; Fg, <em>Fusarium graminearum</em>. GH, glycoside hydrolase; GT, glycosyltransferase; PL, polysaccharide lyase; CE, carbohydrate esterase; AA, auxiliary activity family; CBM, carbohydrate-binding module family. The numbers of gene families were normalized by <em>Z</em> score.</p></div></div><p>Of the annotated carbohydrate esterase genes, family CE1 and CE10 are two major subfamilies (<a reveal-id="dsx035-F6" data-open="dsx035-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). Both families encode proteins with the common activities of carboxylesterase and endo-1,4-β-xylanase.<a reveal-id="dsx035-B64" data-open="dsx035-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> Besides, 34 potential carbohydrate-binding module (CBM) proteins were identified in <em>C. sojina</em> genome (<span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035_supplemental_table.pdf?Expires=2147483647&amp;Signature=XlkQOdSYFRueozu21xQI6RlMVIoRZIHZIEX1uhZQa5Xp0fHWe9WI1vIRTUN03DUwOOW-mIdJPPE3v2MpkX98iXRQpyVFImKFX3IdNILEQq5rd6R6nJqBBfGUrASpTDEXsxPIUUYapxiIbm25MuP17HKmMAHgUeF2cLA711JW6F28a2kPH-3cwuKwsP-m7Fsyl2ZQK5bJMjp9P1uuI0VzmKjeqfGJV3SWyP7QrHehB3Gz1RI0ajnnj-W9J9N32N31pH4FKIqgjOHaho5bNVQsbQ3uhjj3FRzn65eAZ3304cFTsnZLJXF4LqyHXoEJLaM9~S1N-RuN~raOEiPsEfqIqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>), which can digest carbohydrate complex extracellularly.<a reveal-id="dsx035-B64" data-open="dsx035-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> Unexpectedly, only one CBM protein CBM1 was found in <em>C. sojina</em> genome (<span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035_supplemental_table.pdf?Expires=2147483647&amp;Signature=XlkQOdSYFRueozu21xQI6RlMVIoRZIHZIEX1uhZQa5Xp0fHWe9WI1vIRTUN03DUwOOW-mIdJPPE3v2MpkX98iXRQpyVFImKFX3IdNILEQq5rd6R6nJqBBfGUrASpTDEXsxPIUUYapxiIbm25MuP17HKmMAHgUeF2cLA711JW6F28a2kPH-3cwuKwsP-m7Fsyl2ZQK5bJMjp9P1uuI0VzmKjeqfGJV3SWyP7QrHehB3Gz1RI0ajnnj-W9J9N32N31pH4FKIqgjOHaho5bNVQsbQ3uhjj3FRzn65eAZ3304cFTsnZLJXF4LqyHXoEJLaM9~S1N-RuN~raOEiPsEfqIqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). However, many phytopathogenic fungi carry plenty of CBM1. For example, <em>Verticillium dahlia</em> has an expansion of CBM1 containing protein family (∼30 genes).<a reveal-id="dsx035-B65" data-open="dsx035-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a></p>                            <h3 scrollto-destination=117294544 id="117294544" class="section-title" >3.8. Functional analysis of putative effectors</h3>
<p>Effectors are low molecular weight proteins that are secreted by bacteria, oomycetes or fungi to impair the host immune defence and to adapt to specific environment. In the maize pathogen <em>U. maydis</em>, it was observed that most of the genes in secreted protein-coding gene clusters were induced simultaneously in infected tissue.<a reveal-id="dsx035-B66" data-open="dsx035-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> Phylogenetic analysis for 233 putative effectors reveals that 21 predicted effectors are grouped into clusters that contain 2 or 3 high sequence similarity genes in <em>C. sojina</em> (<a reveal-id="dsx035-F7" data-open="dsx035-F7" class="link link-reveal link-table xref-fig">Fig. 7A</a>). Clusters of putative effectors also suggest that local duplications might be involved in expansion of effectors in <em>C. sojina</em>. In addition, 40 putative effectors can be annotated by PHI database (<span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035_supplemental_table.pdf?Expires=2147483647&amp;Signature=XlkQOdSYFRueozu21xQI6RlMVIoRZIHZIEX1uhZQa5Xp0fHWe9WI1vIRTUN03DUwOOW-mIdJPPE3v2MpkX98iXRQpyVFImKFX3IdNILEQq5rd6R6nJqBBfGUrASpTDEXsxPIUUYapxiIbm25MuP17HKmMAHgUeF2cLA711JW6F28a2kPH-3cwuKwsP-m7Fsyl2ZQK5bJMjp9P1uuI0VzmKjeqfGJV3SWyP7QrHehB3Gz1RI0ajnnj-W9J9N32N31pH4FKIqgjOHaho5bNVQsbQ3uhjj3FRzn65eAZ3304cFTsnZLJXF4LqyHXoEJLaM9~S1N-RuN~raOEiPsEfqIqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>), and most of the annotated effectors have been implicated in fungal pathogenesis. </p>                        <a id="117294546" scrollto-destination="117294546"></a>
<div id="dsx035-F7" data-id="dsx035-F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/m_dsx035f7.png?Expires=2147483647&Signature=APNuJjkxy9yBZ4AdFegLDNGgCuF5qHKQLse5cm00-KS6ICQ4FPZt11DXZEyPuqfC9DAzngEFRrciAU2sn8tWD4pSRVOd~B66WAdMsSNSJnhCz~1dVX6AmNX4eZS2KtskX0D~PTvE2LzXBu0qgHistZR9AXi0STxQKEIID9VidpM5F1bCJL-AcH16197W3y-EgWh1tNr4ZAdwRfbl~f0Ctfv0vaxxp5KujT5c-97U2H5weUWOthfkEHb5jkL7jlUhoP2kjdy2FBkJpUT22ayq5Pu7iwF7pJzD9nBXHp3AjR1JL4dTbUawOIM3PmyjIKJyo9tLNGpx0Xaz82fMxULueA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional analysis of putative C. sojina effectors. (A) Phylogenetic analysis of putative effectors. The phylogenetic tree of 233 putative effectors is categorized into six super clades. (B) Selected C. sojina effectors suppress BAX-induced programmed cell death in N. benthamiana. N. benthamiana leaves were infiltrated with agrobacterium carrying C. sojina effector genes (OD600 = 0.4). GFP and Phytophothora sojae effector Avr1b were used as the negative and the positive controls, respectively. Twenty-four hours later, plants were infiltrated with pVX-BAX (OD600 = 0.4), then the photos were taken 3 days later. Results are representatives of six biological replicates. (C) The effectors that can suppress BAX-induced PCD are significantly up-regulated at 48 hpi. Soybean leaves were inoculated with C. sojina. Samples were collected at 0 and 48 hpi, respectively. Quantitative RT-PCR (qRT–PCR) were used to determine gene expression levels. Values are mean ± SD (n = 3 biological replicates)." path-from-xml="dsx035f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/117294546/dsx035f7.tif" path-from-xml="dsx035f7.tif" target="_blank">View large</a><a section="117294546" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035f7.png?Expires=2147483647&Signature=rMDB0-JerNG2xqIPWLZRMFXvI3WKU7SARcXtnpWEttEhCgGHZbgzMafygZIrOn2SKIIIs7hV6LTp6WHNl7w1ZW3mnzCfezRc4N-HME6YffEMhFRKpy3w03XacUhV~7bQG~UyJoM18e5UtYBUS304gFgTRFd61snFIgk3NixlX2Pw57X9BezPiheZQCoL-jYbmmqDZsBDRK0ospFNKlzWN1Wz96hr1XTj-nMkVmMerx1J8OakXbqSFgIh~5~3Y5vAAZDao5723Ry~skL13t5Sqxzy4JUrZ9ouuCVvbDaxeRgPu7byOkhS5xS7AlZz8uTX4Gyu1J2gLQffA6u1L4c6~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=117294546&ar=4161430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx035f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional analysis of putative <em>C. sojina</em> effectors. (A) Phylogenetic analysis of putative effectors. The phylogenetic tree of 233 putative effectors is categorized into six super clades. (B) Selected <em>C. sojina</em> effectors suppress BAX-induced programmed cell death in <em>N. benthamiana. N. benthamiana</em> leaves were infiltrated with agrobacterium carrying <em>C. sojina</em> effector genes (OD600<strong><span class="small-caps"> </span></strong>=<strong><span class="small-caps"> </span></strong>0.4). GFP and <em>Phytophothora sojae</em> effector Avr1b were used as the negative and the positive controls, respectively. Twenty-four hours later, plants were infiltrated with pVX-BAX (OD600<strong><span class="small-caps"> </span></strong>=<strong><span class="small-caps"> </span></strong>0.4), then the photos were taken 3 days later. Results are representatives of six biological replicates. (C) The effectors that can suppress BAX-induced PCD are significantly up-regulated at 48 hpi. Soybean leaves were inoculated with <em>C. sojina</em>. Samples were collected at 0 and 48 hpi, respectively. Quantitative RT-PCR (qRT–PCR) were used to determine gene expression levels. Values are mean ± SD (<em>n</em> = 3 biological replicates).</p></div></div><div content-id="dsx035-F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/m_dsx035f7.png?Expires=2147483647&Signature=APNuJjkxy9yBZ4AdFegLDNGgCuF5qHKQLse5cm00-KS6ICQ4FPZt11DXZEyPuqfC9DAzngEFRrciAU2sn8tWD4pSRVOd~B66WAdMsSNSJnhCz~1dVX6AmNX4eZS2KtskX0D~PTvE2LzXBu0qgHistZR9AXi0STxQKEIID9VidpM5F1bCJL-AcH16197W3y-EgWh1tNr4ZAdwRfbl~f0Ctfv0vaxxp5KujT5c-97U2H5weUWOthfkEHb5jkL7jlUhoP2kjdy2FBkJpUT22ayq5Pu7iwF7pJzD9nBXHp3AjR1JL4dTbUawOIM3PmyjIKJyo9tLNGpx0Xaz82fMxULueA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional analysis of putative C. sojina effectors. (A) Phylogenetic analysis of putative effectors. The phylogenetic tree of 233 putative effectors is categorized into six super clades. (B) Selected C. sojina effectors suppress BAX-induced programmed cell death in N. benthamiana. N. benthamiana leaves were infiltrated with agrobacterium carrying C. sojina effector genes (OD600 = 0.4). GFP and Phytophothora sojae effector Avr1b were used as the negative and the positive controls, respectively. Twenty-four hours later, plants were infiltrated with pVX-BAX (OD600 = 0.4), then the photos were taken 3 days later. Results are representatives of six biological replicates. (C) The effectors that can suppress BAX-induced PCD are significantly up-regulated at 48 hpi. Soybean leaves were inoculated with C. sojina. Samples were collected at 0 and 48 hpi, respectively. Quantitative RT-PCR (qRT–PCR) were used to determine gene expression levels. Values are mean ± SD (n = 3 biological replicates)." path-from-xml="dsx035f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/117294546/dsx035f7.tif" path-from-xml="dsx035f7.tif" target="_blank">View large</a><a section="117294546" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035f7.png?Expires=2147483647&Signature=rMDB0-JerNG2xqIPWLZRMFXvI3WKU7SARcXtnpWEttEhCgGHZbgzMafygZIrOn2SKIIIs7hV6LTp6WHNl7w1ZW3mnzCfezRc4N-HME6YffEMhFRKpy3w03XacUhV~7bQG~UyJoM18e5UtYBUS304gFgTRFd61snFIgk3NixlX2Pw57X9BezPiheZQCoL-jYbmmqDZsBDRK0ospFNKlzWN1Wz96hr1XTj-nMkVmMerx1J8OakXbqSFgIh~5~3Y5vAAZDao5723Ry~skL13t5Sqxzy4JUrZ9ouuCVvbDaxeRgPu7byOkhS5xS7AlZz8uTX4Gyu1J2gLQffA6u1L4c6~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=117294546&ar=4161430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx035f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional analysis of putative <em>C. sojina</em> effectors. (A) Phylogenetic analysis of putative effectors. The phylogenetic tree of 233 putative effectors is categorized into six super clades. (B) Selected <em>C. sojina</em> effectors suppress BAX-induced programmed cell death in <em>N. benthamiana. N. benthamiana</em> leaves were infiltrated with agrobacterium carrying <em>C. sojina</em> effector genes (OD600<strong><span class="small-caps"> </span></strong>=<strong><span class="small-caps"> </span></strong>0.4). GFP and <em>Phytophothora sojae</em> effector Avr1b were used as the negative and the positive controls, respectively. Twenty-four hours later, plants were infiltrated with pVX-BAX (OD600<strong><span class="small-caps"> </span></strong>=<strong><span class="small-caps"> </span></strong>0.4), then the photos were taken 3 days later. Results are representatives of six biological replicates. (C) The effectors that can suppress BAX-induced PCD are significantly up-regulated at 48 hpi. Soybean leaves were inoculated with <em>C. sojina</em>. Samples were collected at 0 and 48 hpi, respectively. Quantitative RT-PCR (qRT–PCR) were used to determine gene expression levels. Values are mean ± SD (<em>n</em> = 3 biological replicates).</p></div></div><p>Next, we attempt to investigate the effector function. Pro-apoptotic mouse protein BAX-induced programmed cell death (PCD) on <em>N. benthamiana</em> could physiologically resemble defence-associated hypersensitive response caused by pathogens, providing a valuable screening approach for effectors that can suppress defence-related PCD.<a reveal-id="dsx035-B67" data-open="dsx035-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> We randomly selected 50 effectors and transiently expressed them in <em>N. benthamiana</em> to screen the potential effectors that can suppress BAX-triggered PCD (BT-PCD). Our results show that about 1/4 (13/50) selected effectors strongly suppress BT-PCD (<a reveal-id="dsx035-F7" data-open="dsx035-F7" class="link link-reveal link-table xref-fig">Fig. 7B</a>). Moreover, qRT-PCR results show that most of these putative effectors are transcriptionally induced at 48 h after <em>C. sojina</em> infection (<a reveal-id="dsx035-F7" data-open="dsx035-F7" class="link link-reveal link-table xref-fig">Fig. 7C</a>), implying they probably contribute to early infection on soybean.</p>                            <h2 scrollto-destination=117294548 id="117294548" class="section-title" >4. Discussion</h2>
<p><em>Cercospora</em> species cause severe leaf spot and blight diseases on many crops worldwide.<a reveal-id="dsx035-B1" data-open="dsx035-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> In this study, we sequenced the genome of the economically important fungi, <em>C. sojina</em> race 1. The genome size is ∼40.84 Mb, and 25.56% of the genome is composed of repeat sequences. However, compared with the genome size of sequenced but not annotated <em>C. sojina</em> isolate using NGS method (IMG genome id: 2506520004, <a class="link link-uri" href="https://img.jgi.doe.gov/cgi-bin/m/main.cgi" target="_blank">https://img.jgi.doe.gov/cgi-bin/m/main.cgi</a>), <em>C. sojina</em> race 1 has a larger genome size (<a reveal-id="dsx035-T1" data-open="dsx035-T1" class="link link-reveal link-table xref-fig">Table 1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s1_dsx035.png?Expires=2147483647&Signature=qJjhcd6gnL0yEg8KHMZgFR1Vn7mua1fBvzyU93lE~Fe0wgTkQx1yrrwFsPNFlJXDoFYM8StFeY6w-NI7gbqDUiTW23h-vRGcp1-quFJe3MaEatHREWBHiW14cq22pdwdV2GrD9EN4~BfjcLObKfu86-QPcLmfKZk-oUjAi1Wd7ZXy2vsNozqLmqiX9Mu1YrMkCPb~71-ka39RB58-d0bofOyQJLmyPeQ7Dz2qyt9-qSLSovX-94abqWxLCXG7pAflXAwI7lT5M9~xneIDTQF-IHbKEXXQMnW8VUxpAuITuSVpM7-Z6jsMcxfiGOruTl5roNOiVRE40DbgNFHILhdHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S12</a></span>). In particular, the assembled <em>C. sojina</em> genome by SMRT generates ∼10 Mb repetitive sequences that are not detected by other sequencing techniques.</p><p>DNA methylation is an important research area of epigenetics in both eukaryotic and prokaryotic. The most studied type of DNA methylation in fungi is m5C, while in prokaryotic it is m4C and m6A.<a reveal-id="dsx035-B68" data-open="dsx035-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> Until recently, the methylase and demethylase for m6A in eukaryotic were identified. However, in fungi, this type of DNA modification is poorly studied. Using SMRT, we identified m6A and m4C in <em>C. sojina</em> for the first time, in which we found 17,407 m6A and 1,015,733 m4C in <em>C. sojina</em> genome (<a reveal-id="dsx035-F2" data-open="dsx035-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). The m6A frequency in this fungus is 435.2/Mb, which is slightly more than 341.3/Mb in yeast.<a reveal-id="dsx035-B69" data-open="dsx035-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a> In bacteria, m6A is regarded as an epigenetic signal for DNA-protein interactions.<a reveal-id="dsx035-B70" data-open="dsx035-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a> Surprisingly, we also observe that <em>C. sojina</em> contains large numbers of m4C, which is only reported in bacteria to our knowledge.<a reveal-id="dsx035-B68" data-open="dsx035-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> Interestingly, the m4C modification occurs with lower frequency in the region of repetitive elements in <em>C. sojina</em> genome (<a reveal-id="dsx035-F2" data-open="dsx035-F2" class="link link-reveal link-table xref-fig">Fig. 2C and D</a>), indicating that m4C may be involved in the transposition of the transposons. Our results also demonstrate that SMRT is a powerful tool in studying fungal genome epigenetic modification.</p><p>It is worth noting that one of the enriched family is glycoside hydroxylate GH109 family (<a reveal-id="dsx035-F6" data-open="dsx035-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). This gene family encodes α-<em>N</em>-acetylgalactosaminidase (αNAGAL), an enzyme that can cleave <em>N</em>-acetyl galactosamine from the conjugated proteins.<a reveal-id="dsx035-B62" data-open="dsx035-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> Interestingly, it has been found that lectin can specifically bind <em>N</em>-acetyl galactosamine.<a reveal-id="dsx035-B62" data-open="dsx035-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> It is also found that soybean lectin can bind the <em>N</em>-acetyl galactosamine, a component of fungal cell wall.<a reveal-id="dsx035-B63" data-open="dsx035-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> Therefore, the αNAGAL may compete with soybean lectin to bind N-acetyl galactosamine. It is known that soybean lectin and other plant lectins can inhibit hyphae growth and spore germination by binding their cell wall components in several fungi, such as <em>Penicillia</em> and <em>Aspergilli</em> species.<a reveal-id="dsx035-B71" data-open="dsx035-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a><sup>,</sup><a reveal-id="dsx035-B72" data-open="dsx035-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> We hypothesize that the expansion of GH109 family in <em>C. sojina</em> genome may contribute to overcome the lectin-mediated disease resistance in soybean.</p><p>CBMs are the most common non-catalytic modules associated with enzymes active in plant cell-wall hydrolysis.<a reveal-id="dsx035-B73" data-open="dsx035-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a> They can increase enzyme efficiency by anchoring the enzyme’s catalytic region to insoluble cellulose.<a reveal-id="dsx035-B74" data-open="dsx035-B74" class="link link-ref link-reveal xref-bibr"><sup>74</sup></a> However, only one CBM1 protein was found in <em>C. sojina</em> genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s1_dsx035.png?Expires=2147483647&Signature=qJjhcd6gnL0yEg8KHMZgFR1Vn7mua1fBvzyU93lE~Fe0wgTkQx1yrrwFsPNFlJXDoFYM8StFeY6w-NI7gbqDUiTW23h-vRGcp1-quFJe3MaEatHREWBHiW14cq22pdwdV2GrD9EN4~BfjcLObKfu86-QPcLmfKZk-oUjAi1Wd7ZXy2vsNozqLmqiX9Mu1YrMkCPb~71-ka39RB58-d0bofOyQJLmyPeQ7Dz2qyt9-qSLSovX-94abqWxLCXG7pAflXAwI7lT5M9~xneIDTQF-IHbKEXXQMnW8VUxpAuITuSVpM7-Z6jsMcxfiGOruTl5roNOiVRE40DbgNFHILhdHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). In light of their relative slower infection process, the deficiency of CBMs in the genome may undermine <em>C. sojina</em> infection in terms of digesting plant cell walls.</p><p>It is believed that mycotoxin plays a critical role during pathogen infection.<a reveal-id="dsx035-B75" data-open="dsx035-B75" class="link link-ref link-reveal xref-bibr"><sup>75</sup></a> Mycotoxin cercosporin produced by <em>Cercospora</em> spp. is considered to be one of the key factors that can enhance their virulence, as their pathogenicity was remarkably impaired in the cercosporin-deficient mutants.<a reveal-id="dsx035-B75" data-open="dsx035-B75" class="link link-ref link-reveal xref-bibr"><sup>75</sup></a> However, <em>C. sojina</em> is one of the few <em>Cercospora</em> spp. that was reported not able to produce cercosporin, although there is a dispute.<a reveal-id="dsx035-B9" data-open="dsx035-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> We made an effort to examine the cercosporin in either cultured mycelium or infected plant tissue. However, we are unable to detect cercosporin in any of the samples although the complete gene cluster for cercosporin biosynthesis exists in <em>C. sojina</em> genome (<a reveal-id="dsx035-F4" data-open="dsx035-F4" class="link link-reveal link-table xref-fig">Fig. 4</a> and <span class="content-section supplementary-material"><a path-from-xml="sup8" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s8_dsx035.png?Expires=2147483647&Signature=xYLzLoFDodQVSlRs4eZVkzTLlaQnC24DmAbcPBwx-uLz7LIbkhgkutbzV0GQQPNk2Zsm4JIOwIgpZhHm0pxOCra2hjNgSOq-hJDBDNHuR-bz7dw7ycivNRPmmYY7LeC0k-gZ8iPe~26TQ1YrswuFDTj~2WKNHtAkXgs3hSAQ1hMcvdo9sEMEAcWDLk18ni8eKhTnySTy3PTj39QggibYD05obykXOs1M0PDuUD0TPVBazH5fvRSB-0PG~9XxxooCiKbzKznMY-ddG7M19uTsOR72vT1uWu29QCexFP5Tca6ZNSybL6MZz6qe5kgsnTQa9eybNzT25CwwnW5som8juw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>). Therefore, our data imply that <em>C. sojina</em> may not employ cercosporin but other mycotoxin to enhance virulence.</p><p>Fungi-derived pigments can act as virulence factors to facilitate infection in plants, and are required for pathogen fitness by serving as UV protectants and ROS scavengers.<a reveal-id="dsx035-B59" data-open="dsx035-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> The well-studied fungal pigment is melanin, which essentially contributes to fungal pathogenesis by altering cytokine responses, decreasing phagocytosis and scavenging ROS, as well as playing an important role in reinforcing fungal cell wall.<a reveal-id="dsx035-B76" data-open="dsx035-B76" class="link link-ref link-reveal xref-bibr"><sup>76</sup></a> Similar as melanin, we observed some pigment production was induced by cAMP or starvation treatments (<a reveal-id="dsx035-F5" data-open="dsx035-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>). We also identified the key gene clusters that encode PKS in <em>C. sojina</em> genome (<a reveal-id="dsx035-F5" data-open="dsx035-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s1_dsx035.png?Expires=2147483647&Signature=qJjhcd6gnL0yEg8KHMZgFR1Vn7mua1fBvzyU93lE~Fe0wgTkQx1yrrwFsPNFlJXDoFYM8StFeY6w-NI7gbqDUiTW23h-vRGcp1-quFJe3MaEatHREWBHiW14cq22pdwdV2GrD9EN4~BfjcLObKfu86-QPcLmfKZk-oUjAi1Wd7ZXy2vsNozqLmqiX9Mu1YrMkCPb~71-ka39RB58-d0bofOyQJLmyPeQ7Dz2qyt9-qSLSovX-94abqWxLCXG7pAflXAwI7lT5M9~xneIDTQF-IHbKEXXQMnW8VUxpAuITuSVpM7-Z6jsMcxfiGOruTl5roNOiVRE40DbgNFHILhdHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Table S7</a></span>). Moreover, the whole genome transcriptome assays also demonstrate that the key genes that are involved in pigment biosynthesis are significantly up-regulated. These data further support the assumption that the pigments produced by <em>C. sojina</em> are involved in its virulence.</p><p>In addition to secondary metabolites, pathogens usually harbour various virulence effectors. These effectors interfere with host immune responses to enhance virulence. For example, bacterial pathogen <em>Pseudomonas syringae</em> delivers over 30 effectors by type III secretion system during infection.<a reveal-id="dsx035-B77" data-open="dsx035-B77" class="link link-ref link-reveal xref-bibr"><sup>77</sup></a> Our work showed that more than one third of the effectors were upregulated during starvation (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s7_dsx035.png?Expires=2147483647&Signature=hv4jFu1qBL5AAhCZB-E0qY1Npzi4ZKbmxR5RXJkFvbid2723XnzFzZqRlY07FWNRQi~VjisGGfCVsS3a08eQqq3vY8xP0njSK8Q6detsOuh7-9yiYyMZwyokAgBs7IXd9OyFI3VgsAGHb8Dp1UQZ2Ztcr1nysOfkEjeP4tynWENxj3TnZNsXZ24~RnJBVaztSvNCs0PcstO4zyVUnX5Zz~JnXZ47DspXPlV4NVhVEj88QJj0BMm4QW2LCuqSE7~8CyznMXPKgJ7HG6mj1Vv~UxCt8kKj2hTrxT5bXhEil1SHMEP1SWg-58bJJlh~BZV3CjLZBcJLyAQHHSQX18HT~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7B</a></span>), and many of them can suppress BAX-induced cell death (<a reveal-id="dsx035-F7" data-open="dsx035-F7" class="link link-reveal link-table xref-fig">Fig. 7B</a>), similar to the finding in <em>Phytophothora sojae</em>.<a reveal-id="dsx035-B67" data-open="dsx035-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> These data demonstrate that <em>C. sojina</em> can probably deploy effectors to promote infection.</p><p>In summary, we report a complete genome sequence of <em>C. sojina</em> by SMRT sequencing method. This sequencing method not only assists us to find the repetitive elements, but also to discover the DNA methylations in fungus. By the genome assembly and annotation, we hypothesize that the specific CAZymes, secondary metabolites, and effectors can help <em>C. sojina</em> to adapt to soybean successfully. Our work also lays the groundwork for future discoveries on this important soybean disease.</p>                            <h2 scrollto-destination=117294557 id="117294557" class="section-title" >Supplementary data</h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s1_dsx035.png?Expires=2147483647&Signature=qJjhcd6gnL0yEg8KHMZgFR1Vn7mua1fBvzyU93lE~Fe0wgTkQx1yrrwFsPNFlJXDoFYM8StFeY6w-NI7gbqDUiTW23h-vRGcp1-quFJe3MaEatHREWBHiW14cq22pdwdV2GrD9EN4~BfjcLObKfu86-QPcLmfKZk-oUjAi1Wd7ZXy2vsNozqLmqiX9Mu1YrMkCPb~71-ka39RB58-d0bofOyQJLmyPeQ7Dz2qyt9-qSLSovX-94abqWxLCXG7pAflXAwI7lT5M9~xneIDTQF-IHbKEXXQMnW8VUxpAuITuSVpM7-Z6jsMcxfiGOruTl5roNOiVRE40DbgNFHILhdHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h2 scrollto-destination=117294559 id="117294559" class="section-title" >Authors’ contributions</h2>
<p>J.L. and S.M. conceived the research plans; X.L., J.C., and J.H. performed the experiments; Z.W., Z.G., and Y.C. provided technical assistance; X.L., J.C., and J.L. wrote the article.</p>                            <h2 scrollto-destination=117294561 id="117294561" class="section-title" >Funding</h2>
<p>The study was supported by Chinese Academy of Sciences (Strategic Priority Research Program Grant NO. XDB11020300), the National Natural Science Foundation of China (Grant 31570252 and Grant 31500220), and by the grant from the State Key Laboratory of Plant Genomics (Grant No. O8KF021011). </p>                            <h2 scrollto-destination=117294563 id="117294563" class="section-title" >Conflict of interest</h2>
<p>None declared. </p>                            <h2 scrollto-destination=117294565 id="117294565" class="section-title" >Accession number</h2>
<p>NFUF00000000</p>                            <h2 scrollto-destination=117294567 id="117294567" class="section-title" >Data availability</h2>
<p>This Whole Genome Shotgun project has been deposited at DDBJ/ENA/GenBank under the accession NFUF00000000. The version described in this paper is version NFUF01000000.</p>                            <h2 scrollto-destination=117294569 id="117294569" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx035-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mian</div> <div class="given-names">M.A.R.</div></div>, <div class="name string-name"><div class="surname">Missaoui</div> <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">Walker</div> <div class="given-names">D.R.</div></div>, <div class="name string-name"><div class="surname">Phillips</div> <div class="given-names">D.V.</div></div>, <div class="name string-name"><div class="surname">Boerma</div> <div class="given-names">H.R.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Frogeye leaf spot of soybean: a review and proposed race designations for isolates of <em>Cercospora sojina</em> Hara,</div> <div class="source">Crop Sci</div> ., <div class="volume">48</div>, <div class="fpage">14</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Frogeye%20leaf%20spot%20of%20soybean%3A%20a%20review%20and%20proposed%20race%20designations%20for%20isolates%20of%20Cercospora%20sojina%20Hara%2C&amp;author=M.A.R.%20Mian&amp;author=A.M.%20Missaoui&amp;author=D.R.%20Walker&amp;author=D.V.%20Phillips&amp;author=H.R.%20Boerma&amp;publication_year=2008&amp;journal=Crop%20Sci&amp;volume=48&amp;pages=14-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2007.08.0432" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2007.08.0432" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2007.08.0432"> </span></div> </div></div></div></div></div><div content-id="dsx035-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gupta</div> <div class="given-names">D.K.</div></div>, <div class="name string-name"><div class="surname">Singh</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Singh</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Srivastava</div> <div class="given-names">L.S.</div></div></div> <div class="year">1994</div>, 
<div class="article-title">Sources of resistance in soybean (Glycine-max) to frog-eye leaf-spot caused by Cercosporidium-sojinum</div>, <div class="source">Indian J. Agric. Sci</div> ., <div class="volume">64</div>, <div class="fpage">886</div>–<div class="lpage">7</div>.</div></div></div></div><div content-id="dsx035-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Soares</div> <div class="given-names">A.P.G.</div></div>, <div class="name string-name"><div class="surname">Guillin</div> <div class="given-names">E.A.</div></div>, <div class="name string-name"><div class="surname">Borges</div> <div class="given-names">L.L.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">More <em>Cercospora</em> species infect soybeans across the Americas than meets the eye,</div> <div class="source">PloS One</div> , <div class="volume">10</div>, <div class="fpage">e0133495</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=More%20Cercospora%20species%20infect%20soybeans%20across%20the%20Americas%20than%20meets%20the%20eye%2C&amp;author=A.P.G.%20Soares&amp;author=E.A.%20Guillin&amp;author=L.L.%20Borges&amp;publication_year=2015&amp;journal=PloS%20One&amp;volume=10&amp;pages=e0133495" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0133495" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0133495" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0133495"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26252018" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mian</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Bond</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Joobeur</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">Identification of soybean genotypes resistant to <em>Cercospora sojina</em> by field screening and molecular markers,</div> <div class="source">Plant Dis</div> ., <div class="volume">93</div>, <div class="fpage">408</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20soybean%20genotypes%20resistant%20to%20Cercospora%20sojina%20by%20field%20screening%20and%20molecular%20markers%2C&amp;author=R.%20Mian&amp;author=J.%20Bond&amp;author=T.%20Joobeur&amp;publication_year=2009&amp;journal=Plant%20Dis&amp;volume=93&amp;pages=408-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/PDIS-93-4-0408" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FPDIS-93-4-0408" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FPDIS-93-4-0408"> </span></div> </div></div></div></div></div><div content-id="dsx035-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Galloway</div> <div class="given-names">J.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Effective management of soybean rust and frogeye leaf spot using a mixture of flusilazole and carbendazim,</div> <div class="source">Crop Prot</div> ., <div class="volume">27</div>, <div class="fpage">566</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effective%20management%20of%20soybean%20rust%20and%20frogeye%20leaf%20spot%20using%20a%20mixture%20of%20flusilazole%20and%20carbendazim%2C&amp;author=J.%20Galloway&amp;publication_year=2008&amp;journal=Crop%20Prot&amp;volume=27&amp;pages=566-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cropro.2007.08.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cropro.2007.08.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cropro.2007.08.014"> </span></div> </div></div></div></div></div><div content-id="dsx035-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">G.R.</div></div>, <div class="name string-name"><div class="surname">Pedersen</div> <div class="given-names">D.K.</div></div>, <div class="name string-name"><div class="surname">Phillips</div> <div class="given-names">D.V.</div></div>, <div class="name string-name"><div class="surname">Bradley</div> <div class="given-names">C.A.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Sensitivity of <em>Cercospora sojina</em> isolates to quinone outside inhibitor fungicides,</div> <div class="source">Crop Prot</div> ., <div class="volume">40</div>, <div class="fpage">63</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sensitivity%20of%20Cercospora%20sojina%20isolates%20to%20quinone%20outside%20inhibitor%20fungicides%2C&amp;author=G.R.%20Zhang&amp;author=D.K.%20Pedersen&amp;author=D.V.%20Phillips&amp;author=C.A.%20Bradley&amp;publication_year=2012&amp;journal=Crop%20Prot&amp;volume=40&amp;pages=63-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cropro.2012.04.025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cropro.2012.04.025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cropro.2012.04.025"> </span></div> </div></div></div></div></div><div content-id="dsx035-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ma</div> <div class="given-names">S.M.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">B.Y.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">Primary report on the identification for physiological races of Cercospora sojina Hara in northeast China</div>, <div class="source">Acta Phytopathol. Sin</div> ., <div class="volume">27</div>, <div class="fpage">180</div>.</div></div></div></div><div content-id="dsx035-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Daub</div> <div class="given-names">M.E.</div></div>, <div class="name string-name"><div class="surname">Ehrenshaft</div> <div class="given-names">M.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">The photoactivated Cercospora toxin cercosporin: contributions to plant disease and fundamental biology,</div> <div class="source">Annu. Rev. Phytopathol</div> ., <div class="volume">38</div>, <div class="fpage">461</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20photoactivated%20Cercospora%20toxin%20cercosporin%3A%20contributions%20to%20plant%20disease%20and%20fundamental%20biology%2C&amp;author=M.E.%20Daub&amp;author=M.%20Ehrenshaft&amp;publication_year=2000&amp;journal=Annu.%20Rev.%20Phytopathol&amp;volume=38&amp;pages=461-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.phyto.38.1.461" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.phyto.38.1.461" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.phyto.38.1.461"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11701851" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Goodwin</div> <div class="given-names">S.B.</div></div>, <div class="name string-name"><div class="surname">Dunkle</div> <div class="given-names">L.D.</div></div>, <div class="name string-name"><div class="surname">Zismann</div> <div class="given-names">V.L.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Phylogenetic analysis of cercospora and mycosphaerella based on the internal transcribed spacer region of ribosomal DNA,</div> <div class="source">Phytopathology</div> , <div class="volume">91</div>, <div class="fpage">648</div>–<div class="lpage">58</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20analysis%20of%20cercospora%20and%20mycosphaerella%20based%20on%20the%20internal%20transcribed%20spacer%20region%20of%20ribosomal%20DNA%2C&amp;author=S.B.%20Goodwin&amp;author=L.D.%20Dunkle&amp;author=V.L.%20Zismann&amp;publication_year=2001&amp;journal=Phytopathology&amp;volume=91&amp;pages=648-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/PHYTO.2001.91.7.648" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FPHYTO.2001.91.7.648" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FPHYTO.2001.91.7.648"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18942994" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dean</div> <div class="given-names">R.A.</div></div>, <div class="name string-name"><div class="surname">Talbot</div> <div class="given-names">N.J.</div></div>, <div class="name string-name"><div class="surname">Ebbole</div> <div class="given-names">D.J.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">The genome sequence of the rice blast fungus <em>Magnaporthe grisea</em>,</div> <div class="source">Nature</div> , <div class="volume">434</div>, <div class="fpage">980</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20the%20rice%20blast%20fungus%20Magnaporthe%20grisea%2C&amp;author=R.A.%20Dean&amp;author=N.J.%20Talbot&amp;author=D.J.%20Ebbole&amp;publication_year=2005&amp;journal=Nature&amp;volume=434&amp;pages=980-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03449" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03449" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03449"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15846337" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Ning</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Shi</div> <div class="given-names">X.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Immunity to rice blast disease by suppression of effector-triggered necrosis</div>. <div class="source">Curr. Biol</div> ., <div class="volume">26</div>, <div class="fpage">2399</div>–<div class="lpage">411</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Immunity%20to%20rice%20blast%20disease%20by%20suppression%20of%20effector-triggered%20necrosis&amp;author=R.%20Wang&amp;author=Y.%20Ning&amp;author=X.%20Shi&amp;publication_year=2016&amp;journal=Curr.%20Biol&amp;volume=26&amp;pages=2399-411" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cub.2016.06.072" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cub.2016.06.072" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cub.2016.06.072"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27641772" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Park</div> <div class="given-names">C.H.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Shirsekar</div> <div class="given-names">G.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">The Magnaporthe oryzae effector AvrPiz-t targets the RING E3 Ubiquitin Ligase APIP6 to suppress pathogen-associated molecular pattern–triggered immunity in rice</div>. <div class="source">Plant Cell</div> , <div class="volume">24</div>, <div class="fpage">4748</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Magnaporthe%20oryzae%20effector%20AvrPiz-t%20targets%20the%20RING%20E3%20Ubiquitin%20Ligase%20APIP6%20to%20suppress%20pathogen-associated%20molecular%20pattern%E2%80%93triggered%20immunity%20in%20rice&amp;author=C.H.%20Park&amp;author=S.%20Chen&amp;author=G.%20Shirsekar&amp;publication_year=2012&amp;journal=Plant%20Cell&amp;volume=24&amp;pages=4748-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.112.105429" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.112.105429" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.112.105429"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23204406" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Gao</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">MoEnd3 regulates appressorium formation and virulence through mediating endocytosis in rice blast fungus <em>Magnaporthe oryzae</em></div>. <div class="source">PLoS Pathogens</div> , <div class="volume">13</div>, <div class="fpage">e1006449</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MoEnd3%20regulates%20appressorium%20formation%20and%20virulence%20through%20mediating%20endocytosis%20in%20rice%20blast%20fungus%20Magnaporthe%20oryzae&amp;author=X.%20Li&amp;author=C.%20Gao&amp;author=L.%20Li&amp;publication_year=2017&amp;journal=PLoS%20Pathogens&amp;volume=13&amp;pages=e1006449" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.ppat.1006449" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.ppat.1006449" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.ppat.1006449"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28628655" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Qi</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Dou</div> <div class="given-names">X.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">MoSwi6, an APSES family transcription factor, interacts with MoMps1 and is required for hyphal and conidial morphogenesis, appressorial function and pathogenicity of <em>Magnaporthe oryzae</em></div>. <div class="source">Mol. Plant Pathol</div> ., <div class="volume">13</div>, <div class="fpage">677</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MoSwi6%2C%20an%20APSES%20family%20transcription%20factor%2C%20interacts%20with%20MoMps1%20and%20is%20required%20for%20hyphal%20and%20conidial%20morphogenesis%2C%20appressorial%20function%20and%20pathogenicity%20of%20Magnaporthe%20oryzae&amp;author=Z.%20Qi&amp;author=Q.%20Wang&amp;author=X.%20Dou&amp;publication_year=2012&amp;journal=Mol.%20Plant%20Pathol&amp;volume=13&amp;pages=677-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mpp.2012.13.issue-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmpp.2012.13.issue-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmpp.2012.13.issue-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22321443" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huang</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Czymmek</div> <div class="given-names">K.J.</div></div>, <div class="name string-name"><div class="surname">Caplan</div> <div class="given-names">J.L.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">HYR1-mediated detoxification of reactive oxygen species is required for full virulence in the rice blast fungus</div>. <div class="source">PloS Pathogens</div> , <div class="volume">7</div>, <div class="fpage">e1001335</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HYR1-mediated%20detoxification%20of%20reactive%20oxygen%20species%20is%20required%20for%20full%20virulence%20in%20the%20rice%20blast%20fungus&amp;author=K.%20Huang&amp;author=K.J.%20Czymmek&amp;author=J.L.%20Caplan&amp;publication_year=2011&amp;journal=PloS%20Pathogens&amp;volume=7&amp;pages=e1001335" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.ppat.1001335" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.ppat.1001335" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.ppat.1001335"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21533213" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">T.T.</div></div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">Y.L.</div></div>, <div class="name string-name"><div class="surname">Guo</div> <div class="given-names">H.S.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Secretory proteins are delivered to the septin-organized penetration interface during root infection by <em>Verticillium dahliae</em></div>. <div class="source">PloS Pathol</div> ., <div class="volume">13</div>, <div class="fpage">e1006275</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Secretory%20proteins%20are%20delivered%20to%20the%20septin-organized%20penetration%20interface%20during%20root%20infection%20by%20Verticillium%20dahliae&amp;author=T.T.%20Zhou&amp;author=Y.L.%20Zhao&amp;author=H.S.%20Guo&amp;publication_year=2017&amp;journal=PloS%20Pathol&amp;volume=13&amp;pages=e1006275" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.ppat.1006275" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.ppat.1006275" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.ppat.1006275"> </span></div> </div></div></div></div></div><div content-id="dsx035-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">Y.L.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">T.T.</div></div>, <div class="name string-name"><div class="surname">Guo</div> <div class="given-names">H.S.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Hyphopodium-specific VdNoxB/VdPls1-dependent ROS-Ca<sup>2+</sup> signaling is required for plant infection by <em>Verticillium dahliae</em></div>. <div class="source">PloS Pathol</div> ., <div class="volume">12</div>, <div class="fpage">e1005793</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hyphopodium-specific%20VdNoxB%2FVdPls1-dependent%20ROS-Ca2%2B%20signaling%20is%20required%20for%20plant%20infection%20by%20Verticillium%20dahliae&amp;author=Y.L.%20Zhao&amp;author=T.T.%20Zhou&amp;author=H.S.%20Guo&amp;publication_year=2016&amp;journal=PloS%20Pathol&amp;volume=12&amp;pages=e1005793" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.ppat.1005793" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.ppat.1005793" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.ppat.1005793"> </span></div> </div></div></div></div></div><div content-id="dsx035-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kim</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Newell</div> <div class="given-names">A.D.</div></div>, <div class="name string-name"><div class="surname">Cota-Sieckmeyer</div> <div class="given-names">R.G.</div></div>, <div class="name string-name"><div class="surname">Rupe</div> <div class="given-names">J.C.</div></div>, <div class="name string-name"><div class="surname">Fakhoury</div> <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">Bluhm</div> <div class="given-names">B.H.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Mating-type distribution and genetic diversity of <em>Cercospora sojina</em> populations on soybean from Arkansas: evidence for potential sexual reproduction,</div> <div class="source">Phytopathology</div> , <div class="volume">103</div>, <div class="fpage">1045</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mating-type%20distribution%20and%20genetic%20diversity%20of%20Cercospora%20sojina%20populations%20on%20soybean%20from%20Arkansas%3A%20evidence%20for%20potential%20sexual%20reproduction%2C&amp;author=H.%20Kim&amp;author=A.D.%20Newell&amp;author=R.G.%20Cota-Sieckmeyer&amp;author=J.C.%20Rupe&amp;author=A.M.%20Fakhoury&amp;author=B.H.%20Bluhm&amp;publication_year=2013&amp;journal=Phytopathology&amp;volume=103&amp;pages=1045-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/PHYTO-09-12-0229-R" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FPHYTO-09-12-0229-R" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FPHYTO-09-12-0229-R"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23721180" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kim</div> <div class="given-names">J.S.</div></div>, <div class="name string-name"><div class="surname">Seo</div> <div class="given-names">S.G.</div></div>, <div class="name string-name"><div class="surname">Jun</div> <div class="given-names">B.K.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">J.W.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">S.H.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Simple and reliable DNA extraction method for the dark pigmented fungus, Cercospora sojina,</div> <div class="source">Plant Pathol. J</div> ., <div class="volume">26</div>, <div class="fpage">289</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20and%20reliable%20DNA%20extraction%20method%20for%20the%20dark%20pigmented%20fungus%2C%20Cercospora%20sojina%2C&amp;author=J.S.%20Kim&amp;author=S.G.%20Seo&amp;author=B.K.%20Jun&amp;author=J.W.%20Kim&amp;author=S.H.%20Kim&amp;publication_year=2010&amp;journal=Plant%20Pathol.%20J&amp;volume=26&amp;pages=289-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.5423/PPJ.2010.26.3.289" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.5423%2FPPJ.2010.26.3.289" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.5423%2FPPJ.2010.26.3.289"> </span></div> </div></div></div></div></div><div content-id="dsx035-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chin</div> <div class="given-names">C.S.</div></div>, <div class="name string-name"><div class="surname">Alexander</div> <div class="given-names">D.H.</div></div>, <div class="name string-name"><div class="surname">Marks</div> <div class="given-names">P.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data,</div> <div class="source">Nat. Methods</div> , <div class="volume">10</div>, <div class="fpage">563</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nonhybrid%2C%20finished%20microbial%20genome%20assemblies%20from%20long-read%20SMRT%20sequencing%20data%2C&amp;author=C.S.%20Chin&amp;author=D.H.%20Alexander&amp;author=P.%20Marks&amp;publication_year=2013&amp;journal=Nat.%20Methods&amp;volume=10&amp;pages=563-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.2474" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2474" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.2474"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23644548" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Berlin</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Koren</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Chin</div> <div class="given-names">C.S.</div></div>, <div class="name string-name"><div class="surname">Drake</div> <div class="given-names">J.P.</div></div>, <div class="name string-name"><div class="surname">Landolin</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Phillippy</div> <div class="given-names">A.M.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Assembling large genomes with single-molecule sequencing and locality-sensitive hashing,</div> <div class="source">Nat. Biotechnol</div> ., <div class="volume">33</div>, <div class="fpage">623</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assembling%20large%20genomes%20with%20single-molecule%20sequencing%20and%20locality-sensitive%20hashing%2C&amp;author=K.%20Berlin&amp;author=S.%20Koren&amp;author=C.S.%20Chin&amp;author=J.P.%20Drake&amp;author=J.M.%20Landolin&amp;author=A.M.%20Phillippy&amp;publication_year=2015&amp;journal=Nat.%20Biotechnol&amp;volume=33&amp;pages=623-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.3238" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.3238" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.3238"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26006009" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Luo</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Xie</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler,</div> <div class="source">Gigascience</div> , <div class="volume">1</div>, <div class="fpage">18</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SOAPdenovo2%3A%20an%20empirically%20improved%20memory-efficient%20short-read%20de%20novo%20assembler%2C&amp;author=R.%20Luo&amp;author=B.%20Liu&amp;author=Y.%20Xie&amp;publication_year=2012&amp;journal=Gigascience&amp;volume=1&amp;pages=18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/2047-217X-1-18" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F2047-217X-1-18" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F2047-217X-1-18"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23587118" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ter-Hovhannisyan</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Lomsadze</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Chernoff</div> <div class="given-names">Y.O.</div></div>, <div class="name string-name"><div class="surname">Borodovsky</div> <div class="given-names">M.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Gene prediction in novel fungal genomes using an ab initio algorithm with unsupervised training,</div> <div class="source">Genome Res</div> ., <div class="volume">18</div>, <div class="fpage">1979</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20prediction%20in%20novel%20fungal%20genomes%20using%20an%20ab%20initio%20algorithm%20with%20unsupervised%20training%2C&amp;author=V.%20Ter-Hovhannisyan&amp;author=A.%20Lomsadze&amp;author=Y.O.%20Chernoff&amp;author=M.%20Borodovsky&amp;publication_year=2008&amp;journal=Genome%20Res&amp;volume=18&amp;pages=1979-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.081612.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.081612.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.081612.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18757608" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Salamov</div> <div class="given-names">A.A.</div></div>, <div class="name string-name"><div class="surname">Solovyev</div> <div class="given-names">V.V.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Ab initio gene finding in Drosophila genomic DNA,</div> <div class="source">Genome Res</div> ., <div class="volume">10</div>, <div class="fpage">516</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ab%20initio%20gene%20finding%20in%20Drosophila%20genomic%20DNA%2C&amp;author=A.A.%20Salamov&amp;author=V.V.%20Solovyev&amp;publication_year=2000&amp;journal=Genome%20Res&amp;volume=10&amp;pages=516-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.10.4.516" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.10.4.516" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.10.4.516"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10779491" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Delcher</div> <div class="given-names">A.L.</div></div>, <div class="name string-name"><div class="surname">Harmon</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Kasif</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">White</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Improved microbial gene identification with GLIMMER</div>. <div class="source">Nucleic Acids Res</div> , <div class="volume">27</div>, <div class="fpage">4636</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20microbial%20gene%20identification%20with%20GLIMMER&amp;author=A.L.%20Delcher&amp;author=D.%20Harmon&amp;author=S.%20Kasif&amp;author=O.%20White&amp;author=S.L.%20Salzberg&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res&amp;volume=27&amp;pages=4636-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.23.4636" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.23.4636" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.23.4636"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10556321" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Haas</div> <div class="given-names">B.J.</div></div>, <div class="name string-name"><div class="surname">Zeng</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Pearson</div> <div class="given-names">M.D.</div></div>, <div class="name string-name"><div class="surname">Cuomo</div> <div class="given-names">C.A.</div></div>, <div class="name string-name"><div class="surname">Wortman</div> <div class="given-names">J.R.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Approaches to fungal genome annotation,</div> <div class="source">Mycology</div> , <div class="volume">2</div>, <div class="fpage">118</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Approaches%20to%20fungal%20genome%20annotation%2C&amp;author=B.J.%20Haas&amp;author=Q.%20Zeng&amp;author=M.D.%20Pearson&amp;author=C.A.%20Cuomo&amp;author=J.R.%20Wortman&amp;publication_year=2011&amp;journal=Mycology&amp;volume=2&amp;pages=118-41" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22059117" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Haas</div> <div class="given-names">B.J.</div></div>, <div class="name string-name"><div class="surname">Papanicolaou</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Yassour</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis,</div> <div class="source">Nat. Protoc</div> ., <div class="volume">8</div>, <div class="fpage">1494</div>–<div class="lpage">512</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20transcript%20sequence%20reconstruction%20from%20RNA-seq%20using%20the%20Trinity%20platform%20for%20reference%20generation%20and%20analysis%2C&amp;author=B.J.%20Haas&amp;author=A.%20Papanicolaou&amp;author=M.%20Yassour&amp;publication_year=2013&amp;journal=Nat.%20Protoc&amp;volume=8&amp;pages=1494-512" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2013.084" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2013.084" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2013.084"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23845962" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kanehisa</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Goto</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kawashima</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Okuno</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Hattori</div> <div class="given-names">M.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">The KEGG resource for deciphering the genome,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">32</div>, <div class="fpage">D277</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20KEGG%20resource%20for%20deciphering%20the%20genome%2C&amp;author=M.%20Kanehisa&amp;author=S.%20Goto&amp;author=S.%20Kawashima&amp;author=Y.%20Okuno&amp;author=M.%20Hattori&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;pages=D277-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh063" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh063" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh063"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14681412" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tatusov</div> <div class="given-names">R.L.</div></div>, <div class="name string-name"><div class="surname">Galperin</div> <div class="given-names">M.Y.</div></div>, <div class="name string-name"><div class="surname">Natale</div> <div class="given-names">D.A.</div></div>, <div class="name string-name"><div class="surname">Koonin</div> <div class="given-names">E.V.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">The COG database: a tool for genome-scale analysis of protein functions and evolution,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">28</div>, <div class="fpage">33</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20COG%20database%3A%20a%20tool%20for%20genome-scale%20analysis%20of%20protein%20functions%20and%20evolution%2C&amp;author=R.L.%20Tatusov&amp;author=M.Y.%20Galperin&amp;author=D.A.%20Natale&amp;author=E.V.%20Koonin&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res&amp;volume=28&amp;pages=33-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.1.33" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.1.33" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.1.33"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10592175" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Finn</div> <div class="given-names">R.D.</div></div>, <div class="name string-name"><div class="surname">Bateman</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Clements</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Pfam: the protein families database,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">D222</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pfam%3A%20the%20protein%20families%20database%2C&amp;author=R.D.%20Finn&amp;author=A.%20Bateman&amp;author=J.%20Clements&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=D222-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1223" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1223" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1223"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24288371" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Profile hidden Markov models,</div> <div class="source">Bioinformatics</div> , <div class="volume">14</div>, <div class="fpage">755</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Profile%20hidden%20Markov%20models%2C&amp;author=S.R.%20Eddy&amp;publication_year=1998&amp;journal=Bioinformatics&amp;volume=14&amp;pages=755-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/14.9.755" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F14.9.755" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F14.9.755"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9918945" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Urban</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Cuzick</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Rutherford</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">PHI-base: a new interface and further additions for the multi-species pathogen–host interactions database</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">45</div>, <div class="fpage">D604</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PHI-base%3A%20a%20new%20interface%20and%20further%20additions%20for%20the%20multi-species%20pathogen%E2%80%93host%20interactions%20database&amp;author=M.%20Urban&amp;author=A.%20Cuzick&amp;author=K.%20Rutherford&amp;publication_year=2017&amp;journal=Nucleic%20Acids%20Res&amp;volume=45&amp;pages=D604-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkw1089" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkw1089" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkw1089"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27915230" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gotz</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Arnold</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Sebastian-Leon</div> <div class="given-names">P.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">B2G-FAR, a species-centered GO annotation repository,</div> <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">919</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=B2G-FAR%2C%20a%20species-centered%20GO%20annotation%20repository%2C&amp;author=S.%20Gotz&amp;author=R.%20Arnold&amp;author=P.%20Sebastian-Leon&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=919-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr059" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr059" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr059"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21335611" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Benson</div> <div class="given-names">G.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Tandem repeats finder: a program to analyze DNA sequences,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">27</div>, <div class="fpage">573</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tandem%20repeats%20finder%3A%20a%20program%20to%20analyze%20DNA%20sequences%2C&amp;author=G.%20Benson&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res&amp;volume=27&amp;pages=573-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.2.573" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.2.573" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.2.573"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9862982" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Blow</div> <div class="given-names">M.J.</div></div>, <div class="name string-name"><div class="surname">Clark</div> <div class="given-names">T.A.</div></div>, <div class="name string-name"><div class="surname">Daum</div> <div class="given-names">C.G.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">The epigenomic landscape of prokaryotes,</div> <div class="source">PloS Genet</div> ., <div class="volume">12</div>, <div class="fpage">e1005854</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20epigenomic%20landscape%20of%20prokaryotes%2C&amp;author=M.J.%20Blow&amp;author=T.A.%20Clark&amp;author=C.G.%20Daum&amp;publication_year=2016&amp;journal=PloS%20Genet&amp;volume=12&amp;pages=e1005854" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1005854" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1005854" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1005854"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26870957" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nordberg</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Cantor</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Dusheyko</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">The genome portal of the Department of Energy Joint Genome Institute: 2014 updates,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">D26</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20portal%20of%20the%20Department%20of%20Energy%20Joint%20Genome%20Institute%3A%202014%20updates%2C&amp;author=H.%20Nordberg&amp;author=M.%20Cantor&amp;author=S.%20Dusheyko&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=D26-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1069" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1069" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1069"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24225321" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ebersberger</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Simoes</div> <div class="given-names">R.D.</div></div>, <div class="name string-name"><div class="surname">Kupczok</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">A consistent phylogenetic backbone for the fungi,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">29</div>, <div class="fpage">1319</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20consistent%20phylogenetic%20backbone%20for%20the%20fungi%2C&amp;author=I.%20Ebersberger&amp;author=R.D.%20Simoes&amp;author=A.%20Kupczok&amp;publication_year=2012&amp;journal=Mol.%20Biol.%20Evol&amp;volume=29&amp;pages=1319-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr285" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr285" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsr285"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22114356" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Edgar</div> <div class="given-names">R.C.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">32</div>, <div class="fpage">1792</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MUSCLE%3A%20multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput%2C&amp;author=R.C.%20Edgar&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;pages=1792-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh340" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh340" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh340"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15034147" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Stecher</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">MEGA7: molecular evolutionary genetics analysis version 7.0 for bigger datasets,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">33</div>, <div class="fpage">1870</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA7%3A%20molecular%20evolutionary%20genetics%20analysis%20version%207.0%20for%20bigger%20datasets%2C&amp;author=S.%20Kumar&amp;author=G.%20Stecher&amp;author=K.%20Tamura&amp;publication_year=2016&amp;journal=Mol.%20Biol.%20Evol&amp;volume=33&amp;pages=1870-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msw054" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsw054" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsw054"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27004904" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nix</div> <div class="given-names">D.A.</div></div>, <div class="name string-name"><div class="surname">Eisen</div> <div class="given-names">M.B.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">GATA: a graphic alignment tool for comparative sequence analysis,</div> <div class="source">BMC Bioinformatics</div> , <div class="volume">6</div>, <div class="fpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GATA%3A%20a%20graphic%20alignment%20tool%20for%20comparative%20sequence%20analysis%2C&amp;author=D.A.%20Nix&amp;author=M.B.%20Eisen&amp;publication_year=2005&amp;journal=BMC%20Bioinformatics&amp;volume=6&amp;pages=9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-6-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-6-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-6-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15655071" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mistry</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Finn</div> <div class="given-names">R.D.</div></div>, <div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div>, <div class="name string-name"><div class="surname">Bateman</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Punta</div> <div class="given-names">M.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Challenges in homology search: HMMER3 and convergent evolution of coiled-coil regions,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">e121</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Challenges%20in%20homology%20search%3A%20HMMER3%20and%20convergent%20evolution%20of%20coiled-coil%20regions%2C&amp;author=J.%20Mistry&amp;author=R.D.%20Finn&amp;author=S.R.%20Eddy&amp;author=A.%20Bateman&amp;author=M.%20Punta&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=e121" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt263" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt263" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt263"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23598997" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yin</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Mao</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Mao</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">Y.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">dbCAN: a web resource for automated carbohydrate-active enzyme annotation,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">W445</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=dbCAN%3A%20a%20web%20resource%20for%20automated%20carbohydrate-active%20enzyme%20annotation%2C&amp;author=Y.%20Yin&amp;author=X.%20Mao&amp;author=J.%20Yang&amp;author=X.%20Chen&amp;author=F.%20Mao&amp;author=Y.%20Xu&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=W445-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks479" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks479" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks479"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22645317" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Khaldi</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Seifuddin</div> <div class="given-names">F.T.</div></div>, <div class="name string-name"><div class="surname">Turner</div> <div class="given-names">G.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">SMURF: genomic mapping of fungal secondary metabolite clusters,</div> <div class="source">Fungal Genet. Biol</div> ., <div class="volume">47</div>, <div class="fpage">736</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SMURF%3A%20genomic%20mapping%20of%20fungal%20secondary%20metabolite%20clusters%2C&amp;author=N.%20Khaldi&amp;author=F.T.%20Seifuddin&amp;author=G.%20Turner&amp;publication_year=2010&amp;journal=Fungal%20Genet.%20Biol&amp;volume=47&amp;pages=736-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fgb.2010.06.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fgb.2010.06.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fgb.2010.06.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20554054" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Medema</div> <div class="given-names">M.H.</div></div>, <div class="name string-name"><div class="surname">Blin</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Cimermancic</div> <div class="given-names">P.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">antiSMASH: rapid identification, annotation and analysis of secondary metabolite biosynthesis gene clusters in bacterial and fungal genome sequences,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">39</div>, <div class="fpage">W339</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=antiSMASH%3A%20rapid%20identification%2C%20annotation%20and%20analysis%20of%20secondary%20metabolite%20biosynthesis%20gene%20clusters%20in%20bacterial%20and%20fungal%20genome%20sequences%2C&amp;author=M.H.%20Medema&amp;author=K.%20Blin&amp;author=P.%20Cimermancic&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res&amp;volume=39&amp;pages=W339-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr466" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr466" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr466"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21672958" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shim</div> <div class="given-names">W.B.</div></div>, <div class="name string-name"><div class="surname">Dunkle</div> <div class="given-names">L.D.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Identification of genes expressed during cercosporin biosynthesis in Cercospora zeae-maydis,</div> <div class="source">Physiol. Mol. Plant P</div> ., <div class="volume">61</div>, <div class="fpage">237</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20genes%20expressed%20during%20cercosporin%20biosynthesis%20in%20Cercospora%20zeae-maydis%2C&amp;author=W.B.%20Shim&amp;author=L.D.%20Dunkle&amp;publication_year=2002&amp;journal=Physiol.%20Mol.%20Plant%20P&amp;volume=61&amp;pages=237-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/pmpp.2002.0437" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fpmpp.2002.0437" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fpmpp.2002.0437"> </span></div> </div></div></div></div></div><div content-id="dsx035-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Cao</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">Activation of ethylene signaling pathways enhances disease resistance by regulating ROS and phytoalexin production in rice,</div> <div class="source">Plant J</div> ., <div class="volume">89</div>, <div class="fpage">338</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Activation%20of%20ethylene%20signaling%20pathways%20enhances%20disease%20resistance%20by%20regulating%20ROS%20and%20phytoalexin%20production%20in%20rice%2C&amp;author=C.%20Yang&amp;author=W.%20Li&amp;author=J.%20Cao&amp;publication_year=2017&amp;journal=Plant%20J&amp;volume=89&amp;pages=338-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2017.89.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2017.89.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2017.89.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27701783" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zeng</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Wei</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Bradley</div> <div class="given-names">C.A.</div></div>, <div class="name string-name"><div class="surname">Ming</div> <div class="given-names">R.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Draft genome sequence of <em>Cercospora sojina</em> isolate S9, a fungus causing frogeye leaf spot (FLS) disease of soybean</div>. <div class="source">Genomics Data</div> , <div class="volume">12</div>, <div class="fpage">79</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20Cercospora%20sojina%20isolate%20S9%2C%20a%20fungus%20causing%20frogeye%20leaf%20spot%20(FLS)%20disease%20of%20soybean&amp;author=F.%20Zeng&amp;author=C.%20Wang&amp;author=G.%20Zhang&amp;author=J.%20Wei&amp;author=C.A.%20Bradley&amp;author=R.%20Ming&amp;publication_year=2017&amp;journal=Genomics%20Data&amp;volume=12&amp;pages=79-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gdata.2017.02.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gdata.2017.02.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gdata.2017.02.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28386531" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bodega</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Orlando</div> <div class="given-names">V.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Repetitive elements dynamics in cell identity programming, maintenance and disease,</div> <div class="source">Curr. Opin. Cell Biol</div> ., <div class="volume">31</div>, <div class="fpage">67</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repetitive%20elements%20dynamics%20in%20cell%20identity%20programming%2C%20maintenance%20and%20disease%2C&amp;author=B.%20Bodega&amp;author=V.%20Orlando&amp;publication_year=2014&amp;journal=Curr.%20Opin.%20Cell%20Biol&amp;volume=31&amp;pages=67-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ceb.2014.09.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ceb.2014.09.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ceb.2014.09.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25240822" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jaenisch</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Bird</div> <div class="given-names">A.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Epigenetic regulation of gene expression: how the genome integrates intrinsic and environmental signals,</div> <div class="source">Nat. Genet</div> ., <div class="volume">33</div>, <div class="fpage">245</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenetic%20regulation%20of%20gene%20expression%3A%20how%20the%20genome%20integrates%20intrinsic%20and%20environmental%20signals%2C&amp;author=R.%20Jaenisch&amp;author=A.%20Bird&amp;publication_year=2003&amp;journal=Nat.%20Genet&amp;volume=33&amp;pages=245-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1089" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1089" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1089"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12610534" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ohm</div> <div class="given-names">R.A.</div></div>, <div class="name string-name"><div class="surname">Feau</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Henrissat</div> <div class="given-names">B.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Diverse lifestyles and strategies of plant pathogenesis encoded in the genomes of eighteen Dothideomycetes fungi,</div> <div class="source">PloS Pathog</div> ., <div class="volume">8</div>, <div class="fpage">e1003037</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diverse%20lifestyles%20and%20strategies%20of%20plant%20pathogenesis%20encoded%20in%20the%20genomes%20of%20eighteen%20Dothideomycetes%20fungi%2C&amp;author=R.A.%20Ohm&amp;author=N.%20Feau&amp;author=B.%20Henrissat&amp;publication_year=2012&amp;journal=PloS%20Pathog&amp;volume=8&amp;pages=e1003037" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.ppat.1003037" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.ppat.1003037" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.ppat.1003037"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23236275" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Flusberg</div> <div class="given-names">B.A.</div></div>, <div class="name string-name"><div class="surname">Webster</div> <div class="given-names">D.R.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">J.H.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Direct detection of DNA methylation during single-molecule, real-time sequencing</div>. <div class="source">Nature Methods</div> , <div class="volume">7</div>, <div class="fpage">461</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Direct%20detection%20of%20DNA%20methylation%20during%20single-molecule%2C%20real-time%20sequencing&amp;author=B.A.%20Flusberg&amp;author=D.R.%20Webster&amp;author=J.H.%20Lee&amp;publication_year=2010&amp;journal=Nature%20Methods&amp;volume=7&amp;pages=461-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1459" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1459" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1459"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20453866" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Benson</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Poland</div> <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Benson</div> <div class="given-names">B.M.</div></div>, <div class="name string-name"><div class="surname">Stromberg</div> <div class="given-names">E.L.</div></div>, <div class="name string-name"><div class="surname">Nelson</div> <div class="given-names">R.J.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Resistance to gray leaf spot of maize: genetic architecture and mechanisms elucidated through nested association mapping and near-isogenic line analysis,</div> <div class="source">PloS Genet</div> ., <div class="volume">11</div>, <div class="fpage">e1005045</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Resistance%20to%20gray%20leaf%20spot%20of%20maize%3A%20genetic%20architecture%20and%20mechanisms%20elucidated%20through%20nested%20association%20mapping%20and%20near-isogenic%20line%20analysis%2C&amp;author=J.M.%20Benson&amp;author=J.A.%20Poland&amp;author=B.M.%20Benson&amp;author=E.L.%20Stromberg&amp;author=R.J.%20Nelson&amp;publication_year=2015&amp;journal=PloS%20Genet&amp;volume=11&amp;pages=e1005045" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1005045" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1005045" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1005045"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25764179" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Goodwin</div> <div class="given-names">S.B.</div></div>, <div class="name string-name"><div class="surname">Ben M'Barek</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Dhillon</div> <div class="given-names">B.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Finished genome of the fungal wheat pathogen <em>Mycosphaerella graminicola</em> reveals dispensome structure, chromosome plasticity, and stealth pathogenesis,</div> <div class="source">PloS Genet</div> ., <div class="volume">7</div>, <div class="fpage">e1002070</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Finished%20genome%20of%20the%20fungal%20wheat%20pathogen%20Mycosphaerella%20graminicola%20reveals%20dispensome%20structure%2C%20chromosome%20plasticity%2C%20and%20stealth%20pathogenesis%2C&amp;author=S.B.%20Goodwin&amp;author=S.%20Ben%20M'Barek&amp;author=B.%20Dhillon&amp;publication_year=2011&amp;journal=PloS%20Genet&amp;volume=7&amp;pages=e1002070" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1002070" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002070" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1002070"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21695235" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Krogh</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Larsson</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">von Heijne</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Sonnhammer</div> <div class="given-names">E.L.L.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Predicting transmembrane protein topology with a hidden Markov model: application to complete genomes,</div> <div class="source">J. Mol. Biol</div> ., <div class="volume">305</div>, <div class="fpage">567</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20transmembrane%20protein%20topology%20with%20a%20hidden%20Markov%20model%3A%20application%20to%20complete%20genomes%2C&amp;author=A.%20Krogh&amp;author=B.%20Larsson&amp;author=G.%20von%20Heijne&amp;author=E.L.L.%20Sonnhammer&amp;publication_year=2001&amp;journal=J.%20Mol.%20Biol&amp;volume=305&amp;pages=567-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.2000.4315" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.2000.4315" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.2000.4315"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11152613" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Petersen</div> <div class="given-names">T.N.</div></div>, <div class="name string-name"><div class="surname">Brunak</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">von Heijne</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Nielsen</div> <div class="given-names">H.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">SignalP 4.0: discriminating signal peptides from transmembrane regions,</div> <div class="source">Nat. Methods</div> , <div class="volume">8</div>, <div class="fpage">785</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SignalP%204.0%3A%20discriminating%20signal%20peptides%20from%20transmembrane%20regions%2C&amp;author=T.N.%20Petersen&amp;author=S.%20Brunak&amp;author=G.%20von%20Heijne&amp;author=H.%20Nielsen&amp;publication_year=2011&amp;journal=Nat.%20Methods&amp;volume=8&amp;pages=785-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1701" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1701" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1701"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21959131" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Soanes</div> <div class="given-names">D.M.</div></div>, <div class="name string-name"><div class="surname">Alam</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Cornell</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">Comparative genome analysis of filamentous fungi reveals gene family expansions associated with fungal pathogenesis,</div> <div class="source">PloS One</div> , <div class="volume">3</div>, <div class="fpage">e2300</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genome%20analysis%20of%20filamentous%20fungi%20reveals%20gene%20family%20expansions%20associated%20with%20fungal%20pathogenesis%2C&amp;author=D.M.%20Soanes&amp;author=I.%20Alam&amp;author=M.%20Cornell&amp;publication_year=2008&amp;journal=PloS%20One&amp;volume=3&amp;pages=e2300" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0002300" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0002300" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0002300"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18523684" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Koeck</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hardham</div> <div class="given-names">A.R.</div></div>, <div class="name string-name"><div class="surname">Dodds</div> <div class="given-names">P.N.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">The role of effectors of biotrophic and hemibiotrophic fungi in infection,</div> <div class="source">Cell Microbiol</div> ., <div class="volume">13</div>, <div class="fpage">1849</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20effectors%20of%20biotrophic%20and%20hemibiotrophic%20fungi%20in%20infection%2C&amp;author=M.%20Koeck&amp;author=A.R.%20Hardham&amp;author=P.N.%20Dodds&amp;publication_year=2011&amp;journal=Cell%20Microbiol&amp;volume=13&amp;pages=1849-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1462-5822.2011.01665.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1462-5822.2011.01665.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1462-5822.2011.01665.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21848815" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Park</div> <div class="given-names">Z.Y.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Comparative secretome investigation of magnaporthe oryzae proteins responsive to nitrogen starvation,</div> <div class="source">J. Proteome Res</div> ., <div class="volume">10</div>, <div class="fpage">3136</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20secretome%20investigation%20of%20magnaporthe%20oryzae%20proteins%20responsive%20to%20nitrogen%20starvation%2C&amp;author=Y.%20Wang&amp;author=J.%20Wu&amp;author=Z.Y.%20Park&amp;publication_year=2011&amp;journal=J.%20Proteome%20Res&amp;volume=10&amp;pages=3136-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/pr200202m" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fpr200202m" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fpr200202m"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21563842" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">G.Y.</div></div>, <div class="name string-name"><div class="surname">Nizet</div> <div class="given-names">V.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Color me bad: microbial pigments as virulence factors,</div> <div class="source">Trends Microbiol</div> ., <div class="volume">17</div>, <div class="fpage">406</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Color%20me%20bad%3A%20microbial%20pigments%20as%20virulence%20factors%2C&amp;author=G.Y.%20Liu&amp;author=V.%20Nizet&amp;publication_year=2009&amp;journal=Trends%20Microbiol&amp;volume=17&amp;pages=406-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tim.2009.06.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tim.2009.06.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tim.2009.06.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19726196" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Biely</div> <div class="given-names">P.</div></div></div> <div class="year">2012</div> 
<div class="article-title">Microbial carbohydrate esterases deacetylating plant polysaccharides</div>. <div class="source">Biotechnol. Adv</div> ., <div class="volume">30</div>, <div class="fpage">1575</div>–<div class="lpage">88</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microbial%20carbohydrate%20esterases%20deacetylating%20plant%20polysaccharides&amp;author=P.%20Biely&amp;publication_year=2012&amp;journal=Biotechnol.%20Adv&amp;volume=30&amp;pages=1575-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.biotechadv.2012.04.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.biotechadv.2012.04.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.biotechadv.2012.04.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22580218" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Aurilia</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Parracino</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">D'Auria</div> <div class="given-names">S.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Microbial carbohydrate esterases in cold adapted environments,</div> <div class="source">Gene</div> , <div class="volume">410</div>, <div class="fpage">234</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microbial%20carbohydrate%20esterases%20in%20cold%20adapted%20environments%2C&amp;author=V.%20Aurilia&amp;author=A.%20Parracino&amp;author=S.%20D'Auria&amp;publication_year=2008&amp;journal=Gene&amp;volume=410&amp;pages=234-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2007.12.019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2007.12.019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2007.12.019"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18242884" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Q.P.</div></div>, <div class="name string-name"><div class="surname">Sulzenbacher</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Yuan</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2007</div>, 
<div class="article-title">Bacterial glycosidases for the production of universal red blood cells,</div> <div class="source">Nat. Biotechnol</div> ., <div class="volume">25</div>, <div class="fpage">454</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacterial%20glycosidases%20for%20the%20production%20of%20universal%20red%20blood%20cells%2C&amp;author=Q.P.%20Liu&amp;author=G.%20Sulzenbacher&amp;author=H.%20Yuan&amp;publication_year=2007&amp;journal=Nat.%20Biotechnol&amp;volume=25&amp;pages=454-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1298" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1298" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1298"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17401360" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Benhamou</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Ouellette</div> <div class="given-names">G.B.</div></div></div> <div class="year">1986</div>, 
<div class="article-title">Ultrastructural-localization of glycoconjugates in the fungus ascocalyx-abietina, the scleroderris canker agent of conifers, using lectin gold complexes,</div> <div class="source">J. Histochem. Cytochem</div> ., <div class="volume">34</div>, <div class="fpage">855</div>–<div class="lpage">67</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ultrastructural-localization%20of%20glycoconjugates%20in%20the%20fungus%20ascocalyx-abietina%2C%20the%20scleroderris%20canker%20agent%20of%20conifers%2C%20using%20lectin%20gold%20complexes%2C&amp;author=N.%20Benhamou&amp;author=G.B.%20Ouellette&amp;publication_year=1986&amp;journal=J.%20Histochem.%20Cytochem&amp;volume=34&amp;pages=855-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1177/34.7.3519750" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1177%2F34.7.3519750" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1177%2F34.7.3519750"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3519750" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">Z.T.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">H.Q.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">C.F.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">J.R.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Comparative analysis of fungal genomes reveals different plant cell wall degrading capacity in fungi,</div> <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">274</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20fungal%20genomes%20reveals%20different%20plant%20cell%20wall%20degrading%20capacity%20in%20fungi%2C&amp;author=Z.T.%20Zhao&amp;author=H.Q.%20Liu&amp;author=C.F.%20Wang&amp;author=J.R.%20Xu&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=274" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-274" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-274" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-274"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23617724" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Klosterman</div> <div class="given-names">S.J.</div></div>, <div class="name string-name"><div class="surname">Subbarao</div> <div class="given-names">K.V.</div></div>, <div class="name string-name"><div class="surname">Kang</div> <div class="given-names">S.C.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Comparative genomics yields insights into niche adaptation of plant vascular wilt pathogens,</div> <div class="source">PloS Pathog</div> ., <div class="volume">7</div>, <div class="fpage">e1002137</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomics%20yields%20insights%20into%20niche%20adaptation%20of%20plant%20vascular%20wilt%20pathogens%2C&amp;author=S.J.%20Klosterman&amp;author=K.V.%20Subbarao&amp;author=S.C.%20Kang&amp;publication_year=2011&amp;journal=PloS%20Pathog&amp;volume=7&amp;pages=e1002137" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.ppat.1002137" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.ppat.1002137" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.ppat.1002137"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21829347" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kamper</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Kahmann</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Bolker</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">Insights from the genome of the biotrophic fungal plant pathogen <em>Ustilago maydis</em>,</div> <div class="source">Nature</div> , <div class="volume">444</div>, <div class="fpage">97</div>–<div class="lpage">101</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insights%20from%20the%20genome%20of%20the%20biotrophic%20fungal%20plant%20pathogen%20Ustilago%20maydis%2C&amp;author=J.%20Kamper&amp;author=R.%20Kahmann&amp;author=M.%20Bolker&amp;publication_year=2006&amp;journal=Nature&amp;volume=444&amp;pages=97-101" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature05248" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature05248" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature05248"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17080091" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Q.Q.</div></div>, <div class="name string-name"><div class="surname">Han</div> <div class="given-names">C.Z.</div></div>, <div class="name string-name"><div class="surname">Ferreira</div> <div class="given-names">A.O.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Transcriptional programming and functional interactions within the <em>Phytophthora sojae</em> RXLR effector repertoire,</div> <div class="source">Plant Cell</div> , <div class="volume">23</div>, <div class="fpage">2064</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20programming%20and%20functional%20interactions%20within%20the%20Phytophthora%20sojae%20RXLR%20effector%20repertoire%2C&amp;author=Q.Q.%20Wang&amp;author=C.Z.%20Han&amp;author=A.O.%20Ferreira&amp;publication_year=2011&amp;journal=Plant%20Cell&amp;volume=23&amp;pages=2064-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.111.086082" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.111.086082" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.111.086082"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21653195" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dubey</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Jeon</div> <div class="given-names">J.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Epigenetic regulation of development and pathogenesis in fungal plant pathogens</div>, <div class="source">Mol. Plant Pathol</div> . doi:10.1111/mpp.12499.</div></div></div></div><div content-id="dsx035-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ye</div> <div class="given-names">P.H.</div></div>, <div class="name string-name"><div class="surname">Luan</div> <div class="given-names">Y.Z.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">K.N.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Y.Z.</div></div>, <div class="name string-name"><div class="surname">Xiao</div> <div class="given-names">C.L.</div></div>, <div class="name string-name"><div class="surname">Xie</div> <div class="given-names">Z.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">MethSMRT: an integrative database for DNA N6-methyladenine and N4-methylcytosine generated by single-molecular real-time sequencing,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">45</div>, <div class="fpage">D85</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MethSMRT%3A%20an%20integrative%20database%20for%20DNA%20N6-methyladenine%20and%20N4-methylcytosine%20generated%20by%20single-molecular%20real-time%20sequencing%2C&amp;author=P.H.%20Ye&amp;author=Y.Z.%20Luan&amp;author=K.N.%20Chen&amp;author=Y.Z.%20Liu&amp;author=C.L.%20Xiao&amp;author=Z.%20Xie&amp;publication_year=2017&amp;journal=Nucleic%20Acids%20Res&amp;volume=45&amp;pages=D85-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkw950" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkw950" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkw950"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27924023" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wion</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Casadesus</div> <div class="given-names">J.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">N6-methyl-adenine: an epigenetic signal for DNA-protein interactions,</div> <div class="source">Nat. Rev. Microbiol</div> ., <div class="volume">4</div>, <div class="fpage">183</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=N6-methyl-adenine%3A%20an%20epigenetic%20signal%20for%20DNA-protein%20interactions%2C&amp;author=D.%20Wion&amp;author=J.%20Casadesus&amp;publication_year=2006&amp;journal=Nat.%20Rev.%20Microbiol&amp;volume=4&amp;pages=183-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro1350" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro1350" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrmicro1350"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16489347" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Barkai-golan</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Mirelman</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Sharon</div> <div class="given-names">N.</div></div></div> <div class="year">1978</div>, 
<div class="article-title">Studies on growth inhibition by lectins of Penicillia and Aspergilli</div>. <div class="source">Arch. Microbiol</div> ., <div class="volume">116</div>, <div class="fpage">119</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Studies%20on%20growth%20inhibition%20by%20lectins%20of%20Penicillia%20and%20Aspergilli&amp;author=R.%20Barkai-golan&amp;author=D.%20Mirelman&amp;author=N.%20Sharon&amp;publication_year=1978&amp;journal=Arch.%20Microbiol&amp;volume=116&amp;pages=119-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00406026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00406026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00406026"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/637658" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guo</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">X.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Expression of soybean lectin in transgenic tobacco results in enhanced resistance to pathogens and pests</div>. <div class="source">Plant Science</div> , <div class="volume">211</div>, <div class="fpage">17</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20of%20soybean%20lectin%20in%20transgenic%20tobacco%20results%20in%20enhanced%20resistance%20to%20pathogens%20and%20pests&amp;author=P.%20Guo&amp;author=Y.%20Wang&amp;author=X.%20Zhou&amp;publication_year=2013&amp;journal=Plant%20Science&amp;volume=211&amp;pages=17-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.plantsci.2013.06.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plantsci.2013.06.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.plantsci.2013.06.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23987807" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Boraston</div> <div class="given-names">A.B.</div></div>, <div class="name string-name"><div class="surname">Bolam</div> <div class="given-names">D.N.</div></div>, <div class="name string-name"><div class="surname">Gilbert</div> <div class="given-names">H.J.</div></div>, <div class="name string-name"><div class="surname">Davies</div> <div class="given-names">G.J.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Carbohydrate-binding modules: fine-tuning polysaccharide recognition,</div> <div class="source">Biochem. J</div> ., <div class="volume">382</div>, <div class="fpage">769</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Carbohydrate-binding%20modules%3A%20fine-tuning%20polysaccharide%20recognition%2C&amp;author=A.B.%20Boraston&amp;author=D.N.%20Bolam&amp;author=H.J.%20Gilbert&amp;author=G.J.%20Davies&amp;publication_year=2004&amp;journal=Biochem.%20J&amp;volume=382&amp;pages=769-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/BJ20040892" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2FBJ20040892" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2FBJ20040892"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15214846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Receveur</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Czjzek</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Schulein</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Panine</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Henrissat</div> <div class="given-names">B.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Dimension, shape, and conformational flexibility of a two domain fungal cellulase in solution probed by small angle X-ray scattering,</div> <div class="source">J. Biol. Chem</div> ., <div class="volume">277</div>, <div class="fpage">40887</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dimension%2C%20shape%2C%20and%20conformational%20flexibility%20of%20a%20two%20domain%20fungal%20cellulase%20in%20solution%20probed%20by%20small%20angle%20X-ray%20scattering%2C&amp;author=V.%20Receveur&amp;author=M.%20Czjzek&amp;author=M.%20Schulein&amp;author=P.%20Panine&amp;author=B.%20Henrissat&amp;publication_year=2002&amp;journal=J.%20Biol.%20Chem&amp;volume=277&amp;pages=40887-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M205404200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M205404200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M205404200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12186865" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yu</div> <div class="given-names">J.H.</div></div>, <div class="name string-name"><div class="surname">Keller</div> <div class="given-names">N.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Regulation of secondary metabolism in filamentous fungi,</div> <div class="source">Annu. Rev. Phytopathol</div> ., <div class="volume">43</div>, <div class="fpage">437</div>–<div class="lpage">58</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20secondary%20metabolism%20in%20filamentous%20fungi%2C&amp;author=J.H.%20Yu&amp;author=N.%20Keller&amp;publication_year=2005&amp;journal=Annu.%20Rev.%20Phytopathol&amp;volume=43&amp;pages=437-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.phyto.43.040204.140214" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.phyto.43.040204.140214" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.phyto.43.040204.140214"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16078891" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B76" class="ref-content "><div class="label -label">76</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nosanchuk</div> <div class="given-names">J.D.</div></div>, <div class="name string-name"><div class="surname">Stark</div> <div class="given-names">R.E.</div></div>, <div class="name string-name"><div class="surname">Casadevall</div> <div class="given-names">A.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Fungal melanin: what do we know about structure?</div> <div class="source">Front Microbiol</div> ., <div class="volume">6</div>, <div class="fpage">1463</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fungal%20melanin%3A%20what%20do%20we%20know%20about%20structure%3F&amp;author=J.D.%20Nosanchuk&amp;author=R.E.%20Stark&amp;author=A.%20Casadevall&amp;publication_year=2015&amp;journal=Front%20Microbiol&amp;volume=6&amp;pages=1463" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26733993" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx035-B77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx035-B77" class="ref-content "><div class="label -label">77</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xu</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Luo</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">J.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">The bacterial effector AvrB-induced RIN4 hyperphosphorylation is mediated by a receptor-like cytoplasmic kinase complex in Arabidopsis</div>. <div class="source">Mol. Plant-Microbe In</div> ., <div class="volume">30</div>, <div class="fpage">502</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20bacterial%20effector%20AvrB-induced%20RIN4%20hyperphosphorylation%20is%20mediated%20by%20a%20receptor-like%20cytoplasmic%20kinase%20complex%20in%20Arabidopsis&amp;author=N.%20Xu&amp;author=X.%20Luo&amp;author=W.%20Li&amp;author=Z.%20Wang&amp;author=J.%20Liu&amp;publication_year=2017&amp;journal=Mol.%20Plant-Microbe%20In&amp;volume=30&amp;pages=502-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-01-17-0017-R" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-01-17-0017-R" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-01-17-0017-R"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4161430&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s1_dsx035.png?Expires=2147483647&amp;Signature=qJjhcd6gnL0yEg8KHMZgFR1Vn7mua1fBvzyU93lE~Fe0wgTkQx1yrrwFsPNFlJXDoFYM8StFeY6w-NI7gbqDUiTW23h-vRGcp1-quFJe3MaEatHREWBHiW14cq22pdwdV2GrD9EN4~BfjcLObKfu86-QPcLmfKZk-oUjAi1Wd7ZXy2vsNozqLmqiX9Mu1YrMkCPb~71-ka39RB58-d0bofOyQJLmyPeQ7Dz2qyt9-qSLSovX-94abqWxLCXG7pAflXAwI7lT5M9~xneIDTQF-IHbKEXXQMnW8VUxpAuITuSVpM7-Z6jsMcxfiGOruTl5roNOiVRE40DbgNFHILhdHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S1</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s2_dsx035.png?Expires=2147483647&amp;Signature=GAmhNNkPPorptimeeGfE7Qa9FV0FGH06W11B3pArRgxVSQnAOcbYmGKt5FbqPKpINmHNqyr8zP0G2LataT7PDkaSFVAoPqISefjL2k5pWwdUMRNQe6IR9sdL0yILcgR9zqgd9TtJ7MPLTZohTydZzcJ~mhNvmlRiucL6X6EdTGN3AR1mj-7TXER6WrA2L481eJRW4cRVXQRIqiR2jYMLW8hmd87Al6I7ytZeC1aC~uXZ~P7gALM-TXz3lbbPmCFe2yN8946PUPavE07LdsIVtjmU2I~Nw2B-EZC9W94F7t7VtLWsJmbhx82zvisoagefOLiribmzBE9LdQFOH~XiIw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S2</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s3_dsx035.png?Expires=2147483647&amp;Signature=g9JKyTqOaHoBFBmuB7luI2qZP4hydrljwUd6oqSZvhLyJR8XhnGmqWq~UExdSHgpabtp3ttuVqboIgi~94yQWaazxgevTtmPytIinVDPDelP1x4NOYIaUBBtTAAX54IDxmtUb3Zfpk~wMv6Jz8IaCWqhZFURL~08FhsRHNrA5TLI6hsoKIft0scOvDrnYAaA4gCr~WaAankWaiKy8nXYC4voJe1J~q7BP6v3Kwfbp1wYaxJYkZbvSpDCfKnITmyODIMd04Ei65QWgXA6iCBKCljy6WPAVGZBWITRHAg6HREb9Mxtd1CokNPpjhs1X0ZTS3letrrQH0DWrNGdccEZjA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S3</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s4_dsx035.png?Expires=2147483647&amp;Signature=KT0T7B-LuGPET1zIQdqiYRExTAgP4jrgM0Xn4nQidKfvueB9wW3Dc0PrXHIbVMTy5~tp~3LnLzjBD0KOdoHPDeXpZYUjru9TnDNmIrNzWM-NdtUKVance5mJ~xf6h1P9YialMXyMa7zsytFE-P~BW-yyw7Tl4TTUfMGWLVh4parGhRxTR2CapSIUznurF5a0mopbHCW9gj7pH5lZeou9fQZSjIkQxERGYKegDioc6xrMs3CuAfahxte7Oohj~0k3l6yLlFjASKN~9TVSezFFvjVczoxcEM7zE6JU3Z2aQqqEjdRZbKmb~N6OBkBhmutzSukjLe5b~fZgBIX4dZDAwQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S4</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s5_dsx035.png?Expires=2147483647&amp;Signature=QB2xqD5EpuxgUgGPesAsk2Ok4ytAQfhqXhcF-vbvDnvL-BYrU6lvQkIrwxUZAaqQiTgnoUP~ZK~YTcIia4nsZMXeidTY5AnvKBxSVIWMnLfHz8DltOOXeMNC8AhJEtGCRanj5mcjNbp6jpJgBYRGNpwZq8hi-4xblvbwEFbScvY0DzE2xG-34iWvaJlZx0xUbPuMDSR0W26jCofEssqAa2-htPGdB8UFvvJVvhzoWKeL2zdpj~f2~Z3SHONeApRKhs07GlE6NcSzYAyBffSkbOwxtjnk~TQpjSvjAu88dAAH-zulhTKVRqtZloMYT6eXpJBFCz5-jgaKPLbLTrukQw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S5</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s6_dsx035.png?Expires=2147483647&amp;Signature=XL8ea0QNTf4JST3i0Fs1zpAceVRCNnf8A8jvyVZLj9iT1EXboJH~ch1UbdaSh3y0i3NvywYnx-XsOplZxexCvyNeCwlE5lNLPZ6NDgEL8xBclg8GRbCSq4zN~2gafZSIcx2vnO8J4vWFVWRm2MMC1f0hhYVqpJTwOa~w0g9unId4JFhCWV4vqcef93QgjNTzJP5B3j9mvpiCUFL6x2qZQzAdbZYIba8vmWlV0HDnsMwZGOp5RvaI~VLCLvI2~YoJeTHhGkh68PUNwOwN9-BGUR6ppSYcDS7YW7vZzF5X2kp2QSgDsbMBYZgr407Rqy~ceH0XeSZ5VpzLy0JkD7fh3Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S6</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s7_dsx035.png?Expires=2147483647&amp;Signature=hv4jFu1qBL5AAhCZB-E0qY1Npzi4ZKbmxR5RXJkFvbid2723XnzFzZqRlY07FWNRQi~VjisGGfCVsS3a08eQqq3vY8xP0njSK8Q6detsOuh7-9yiYyMZwyokAgBs7IXd9OyFI3VgsAGHb8Dp1UQZ2Ztcr1nysOfkEjeP4tynWENxj3TnZNsXZ24~RnJBVaztSvNCs0PcstO4zyVUnX5Zz~JnXZ47DspXPlV4NVhVEj88QJj0BMm4QW2LCuqSE7~8CyznMXPKgJ7HG6mj1Vv~UxCt8kKj2hTrxT5bXhEil1SHMEP1SWg-58bJJlh~BZV3CjLZBcJLyAQHHSQX18HT~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S7</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s8_dsx035.png?Expires=2147483647&amp;Signature=xYLzLoFDodQVSlRs4eZVkzTLlaQnC24DmAbcPBwx-uLz7LIbkhgkutbzV0GQQPNk2Zsm4JIOwIgpZhHm0pxOCra2hjNgSOq-hJDBDNHuR-bz7dw7ycivNRPmmYY7LeC0k-gZ8iPe~26TQ1YrswuFDTj~2WKNHtAkXgs3hSAQ1hMcvdo9sEMEAcWDLk18ni8eKhTnySTy3PTj39QggibYD05obykXOs1M0PDuUD0TPVBazH5fvRSB-0PG~9XxxooCiKbzKznMY-ddG7M19uTsOR72vT1uWu29QCexFP5Tca6ZNSybL6MZz6qe5kgsnTQa9eybNzT25CwwnW5som8juw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S8</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/figure_s9_dsx035.png?Expires=2147483647&amp;Signature=bO24WWrNJHJkh~KHOdX5jAAxzQRNjL83gYmx0KKWgYdvSc33GtALEFwLsolGZG61cGSVNfmScbEn6JY0w2xBV87vXc5Sh-bWMbnbmCD4JprPDcoHjmaSOAYo3IczuNFIHVOara9w7kwnNvw2V6iMkvCgIZYaL3t9P8o7b66r7C6SD20O6qktBYAK0cHs1oYuZiyyPHzUco-Vmz8vwZThQ3xNPuLi6A-A7vg05klEM~Q296ZI246loy0IBfP7TnX53WYIf2geQ2KID54DkFUhpBEpgT0QuCTm2rUUub9T~7GTRsySuxBchXbHkpbW764z~05b~Pmvt9XmFQTxRoHlzQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S9</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx035/5/dsx035_supplemental_table.pdf?Expires=2147483647&amp;Signature=XlkQOdSYFRueozu21xQI6RlMVIoRZIHZIEX1uhZQa5Xp0fHWe9WI1vIRTUN03DUwOOW-mIdJPPE3v2MpkX98iXRQpyVFImKFX3IdNILEQq5rd6R6nJqBBfGUrASpTDEXsxPIUUYapxiIbm25MuP17HKmMAHgUeF2cLA711JW6F28a2kPH-3cwuKwsP-m7Fsyl2ZQK5bJMjp9P1uuI0VzmKjeqfGJV3SWyP7QrHehB3Gz1RI0ajnnj-W9J9N32N31pH4FKIqgjOHaho5bNVQsbQ3uhjj3FRzn65eAZ3304cFTsnZLJXF4LqyHXoEJLaM9~S1N-RuN~raOEiPsEfqIqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Tables</div></a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,686</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx035"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4161430"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f1%2f25%2f4161430">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F1%2F25%2F4161430" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4161430'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome sequencing and comparative genomics reveal the potential pathogenic mechanism of Cercospora sojina Hara on soybean" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx035");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f25%2f4161430">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

