

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/5/511/2754544","@type":"ScholarlyArticle","name":"Modelling the Efficiency of Codon–tRNA Interactions Based on Codon Usage Bias","datePublished":"2014-06-06","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2014-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu017","keywords":["codon usage bias","tRNA adaptation index","protein levels","wobble interactions","ribosome"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/5/511/2754544","author":[{"name":"Sabi, Renana","affiliation":"1  Department of Biomedical Engineering ,  Tel Aviv University ,  Tel Aviv ,  Israel","@type":"Person"},{"name":"Tuller, Tamir","affiliation":"1  Department of Biomedical Engineering ,  Tel Aviv University ,  Tel Aviv ,  Israel    2  The Sagol School of Neuroscience ,  Tel-Aviv University ,  Tel-Aviv ,  Israel","@type":"Person"}],"description":"The tRNA adaptation index (tAI) is a widely used measure of the efficiency by which a coding sequence is recognized by the intra-cellular tRNA pool. This index ","pageStart":"511","pageEnd":"526","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/5/1/m_cover.gif?Expires=2147483647&Signature=gPiJSb93IIOcIMTSkMAi7kcKhziIjvuRKEzK8XVp8CTVu4eB811a4N7c5iRqAkFTvycBeFOL1qu9DtcvWUQE-HFcU90uhCx7UXftpGVaIU0r-58rykRAQjr7OyCGXhRRXufrlhLC-AFU8QeGWWE7goVK9s00kVi5KaqA8BdZOl4ZUh6UGigqEAXS9fKMJcyv7L69eSvyhk8fr0u2lpE6Z~kKXJ-zph2mYgOTE6TZsnpM7S63PaSYvqdIQyt~fHxa7pkm4binASfSKcJvbv9WOoMwYew88jtLZv495c~AkAPvlFj827Es8r960R62hvuI-oun7Y-AfhyLYD5usT~QKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Modelling the Efficiency of Codon–tRNA Interactions Based on Codon Usage Bias","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/5/1/m_cover.gif?Expires=2147483647&Signature=gPiJSb93IIOcIMTSkMAi7kcKhziIjvuRKEzK8XVp8CTVu4eB811a4N7c5iRqAkFTvycBeFOL1qu9DtcvWUQE-HFcU90uhCx7UXftpGVaIU0r-58rykRAQjr7OyCGXhRRXufrlhLC-AFU8QeGWWE7goVK9s00kVi5KaqA8BdZOl4ZUh6UGigqEAXS9fKMJcyv7L69eSvyhk8fr0u2lpE6Z~kKXJ-zph2mYgOTE6TZsnpM7S63PaSYvqdIQyt~fHxa7pkm4binASfSKcJvbv9WOoMwYew88jtLZv495c~AkAPvlFj827Es8r960R62hvuI-oun7Y-AfhyLYD5usT~QKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Modelling the Efficiency of Codon–tRNA Interactions Based on Codon Usage Bias" />
    <meta property="og:description" content="The tRNA adaptation index (tAI) is a widely used measure of the efficiency by which a coding sequence is recognized by the intra-cellular tRNA pool. This index " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/5/511/2754544" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2014-06-06" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Sabi, Renana" /><meta name="citation_author_institution" content="Department of Biomedical Engineering, Tel Aviv University, Tel Aviv, Israel" /><meta name="citation_author" content="Tuller, Tamir" /><meta name="citation_author_institution" content="Department of Biomedical Engineering, Tel Aviv University, Tel Aviv, Israel" /><meta name="citation_author_institution" content="The Sagol School of Neuroscience, Tel-Aviv University, Tel-Aviv, Israel" /><meta name="citation_title" content="Modelling the Efficiency of Codon–tRNA Interactions Based on Codon Usage Bias" /><meta name="citation_firstpage" content="511" /><meta name="citation_lastpage" content="526" /><meta name="citation_doi" content="10.1093/dnares/dsu017" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2014/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Genetic variability maintained in a finite population due to mutational production of neutral and nearly neutral isoalleles;  citation_journal_title=Genet. Res.;  citation_year=1968;  citation_volume=11;  citation_pages=247-69; " /><meta name="citation_reference" content="citation_title=Codon catalog usage and the genome hypothesis;  citation_journal_title=Nucleic Acids Res.;  citation_year=1980;  citation_volume=8;  citation_pages=197" /><meta name="citation_reference" content="citation_title=Synonymous but not the same: the causes and consequences of codon bias;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2010;  citation_volume=12;  citation_pages=32-42; " /><meta name="citation_reference" content="citation_title=Codon usage bias: causative factors, quantification methods and genome-wide patterns: with emphasis on insect genomes;  citation_journal_title=Biol. Rev. Camb. Philos. Soc.;  citation_year=2013;  citation_volume=88;  citation_pages=49-61; " /><meta name="citation_reference" content="citation_title=Coadaptation of isoacceptor tRNA genes and codon usage bias for translation efficiency in Aedes aegypti and Anopheles gambiae;  citation_journal_title=Insect Mol. Biol.;  citation_year=2011;  citation_volume=20;  citation_pages=177-87; " /><meta name="citation_reference" content="citation_title=Comparative analysis of nuclear tRNA genes of Nasonia vitripennis and other arthropods, and relationships to codon usage bias;  citation_journal_title=Insect Mol. Biol.;  citation_year=2010;  citation_volume=19;  citation_pages=49-58; " /><meta name="citation_reference" content="citation_title=Exonic transcription factor binding directs codon choice and affects protein evolution;  citation_journal_title=Science;  citation_year=2013;  citation_volume=342;  citation_pages=1367-72; " /><meta name="citation_reference" content="citation_title=Solving the riddle of codon usage preferences: a test for translational selection;  citation_journal_title=Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=5036-44; " /><meta name="citation_reference" content="citation_title=Translation efficiency is determined by both codon bias and folding energy;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2010;  citation_volume=107;  citation_pages=3645-50; " /><meta name="citation_reference" content="citation_title=Coalescent simulation of intracodon recombination;  citation_journal_title=Genetics;  citation_year=2010;  citation_volume=184;  citation_pages=429-37; " /><meta name="citation_reference" content="citation_title=Codon selection in yeast;  citation_journal_title=J. Biol. Chem.;  citation_year=1982;  citation_volume=257;  citation_pages=3026-31; " /><meta name="citation_reference" content="citation_title=Preferential codon usage in prokaryotic genes: the optimal codon-anticodon interaction energy and the selective codon usage in efficiently expressed genes;  citation_journal_title=Gene;  citation_year=1982;  citation_volume=18;  citation_pages=199-209; " /><meta name="citation_reference" content="citation_title=The ‘effective number of codons’ used in a gene;  citation_journal_title=Gene;  citation_year=1990;  citation_volume=87;  citation_pages=23-9; " /><meta name="citation_reference" content="citation_title=Codon bias and gene expression;  citation_journal_title=FEBS Lett.;  citation_year=1991;  citation_volume=285;  citation_pages=165-9; " /><meta name="citation_reference" content="citation_title=Codon usage in Caenorhabditis elegans: delineation of translational selection and mutational biases;  citation_journal_title=Nucleic Acids Res.;  citation_year=1994;  citation_volume=22;  citation_pages=2437-46; " /><meta name="citation_reference" content="citation_title=The codon adaptation index-a measure of directional synonymous codon usage bias, and its potential applications;  citation_journal_title=Nucleic Acids Res.;  citation_year=1987;  citation_volume=15;  citation_pages=1281-95; " /><meta name="citation_reference" content="citation_title=Protein abundance profiling of the Escherichia coli cytosol;  citation_journal_title=BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=102" /><meta name="citation_reference" content="citation_title=Global analysis of protein expression in yeast;  citation_journal_title=Nature;  citation_year=2003;  citation_volume=425;  citation_pages=737-41; " /><meta name="citation_reference" content="citation_title=Large-scale analysis of the yeast proteome by multidimensional protein identification technology;  citation_journal_title=Nat. Biotechnol.;  citation_year=2001;  citation_volume=19;  citation_pages=242-7; " /><meta name="citation_reference" content="citation_title=Why highly expressed proteins evolve slowly;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2005;  citation_volume=102;  citation_pages=14338-43; " /><meta name="citation_reference" content="citation_title=The complete genome sequence of Escherichia coli K-12;  citation_journal_title=Science;  citation_year=1997;  citation_volume=277;  citation_pages=1453-62; " /><meta name="citation_reference" content="citation_title=A ‘silent’ polymorphism in the MDR1 gene changes substrate specificity;  citation_journal_title=Science;  citation_year=2007;  citation_volume=315;  citation_pages=525-8; " /><meta name="citation_reference" content="citation_title=Relative codon adaptation: a generic codon bias index for prediction of gene expression;  citation_journal_title=DNA Res.;  citation_year=2010;  citation_volume=17;  citation_pages=185-96; " /><meta name="citation_reference" content="citation_title=Correlation between the abundance of Escherichia coli transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the E. coli translational system;  citation_journal_title=J. Mol. Biol.;  citation_year=1981;  citation_volume=151;  citation_pages=389-409; " /><meta name="citation_reference" content="citation_title=Codon usage and tRNA content in unicellular and multicellular organisms;  citation_journal_title=Mol. Biol. Evol.;  citation_year=1985;  citation_volume=2;  citation_pages=13-34; " /><meta name="citation_reference" content="citation_title=mRNAs have greater negative folding free energies than shuffled or codon choice randomized sequences;  citation_journal_title=Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=1578-84; " /><meta name="citation_reference" content="citation_title=Role of GC-biased mutation pressure on synonymous codon choice in Micrococcus luteus a bacterium with a high genomic GC-content;  citation_journal_title=Nucleic Acids Res.;  citation_year=1990;  citation_volume=18;  citation_pages=1565-9; " /><meta name="citation_reference" content="citation_title=Correlation between the abundance of yeast transfer RNAs and the occurrence of the respective codons in protein genes: differences in synonymous codon choice patterns of yeast and Escherichia coli with reference to the abundance of isoaccepting transfer RNAs;  citation_journal_title=J. Mol. Biol.;  citation_year=1982;  citation_volume=158;  citation_pages=573-97; " /><meta name="citation_reference" content="citation_title=Transfer RNA gene redundancy and translational selection in Saccharomyces cerevisiae;  citation_journal_title=J. Mol. Biol.;  citation_year=1997;  citation_volume=268;  citation_pages=322-30; " /><meta name="citation_reference" content="citation_title=Differential translation efficiency of orthologous genes is involved in phenotypic divergence of yeast species;  citation_journal_title=Nat. Genet.;  citation_year=2007;  citation_volume=39;  citation_pages=415-21; " /><meta name="citation_reference" content="citation_title=Variation in the strength of selected codon usage bias among bacteria;  citation_journal_title=Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=1141-53; " /><meta name="citation_reference" content="citation_title=Hearing silence: non-neutral evolution at synonymous sites in mammals;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2006;  citation_volume=7;  citation_pages=98-108; " /><meta name="citation_reference" content="citation_title=A periodic pattern of mRNA secondary structure created by the genetic code;  citation_journal_title=Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=2428-37; " /><meta name="citation_reference" content="citation_title=An evolutionarily conserved mechanism for controlling the efficiency of protein translation;  citation_journal_title=Cell;  citation_year=2010;  citation_volume=141;  citation_pages=344-54; " /><meta name="citation_reference" content="citation_title=Ribosome profiling of mouse embryonic stem cells reveals the complexity and dynamics of mammalian proteomes;  citation_journal_title=Cell;  citation_year=2011;  citation_volume=147;  citation_pages=789-802; " /><meta name="citation_reference" content="citation_title=Studies of codon usage and tRNA genes of 18 unicellular organisms and quantification of Bacillus subtilis tRNAs: gene expression level and species-specific diversity of codon usage based on multivariate analysis;  citation_journal_title=Gene;  citation_year=1999;  citation_volume=238;  citation_pages=143-55; " /><meta name="citation_reference" content="citation_title=Codon-anticodon pairing: the wobble hypothesis;  citation_journal_title=J. Mol. Biol.;  citation_year=1966;  citation_volume=19;  citation_pages=548-55; " /><meta name="citation_reference" content="citation_title=Genetic code. Modified bases and aminoacylation;  citation_journal_title=Nature;  citation_year=1988;  citation_volume=336;  citation_pages=112" /><meta name="citation_reference" content="citation_title=Wobble position modified nucleosides evolved to select transfer RNA codon recognition: a modified-wobble hypothesis;  citation_journal_title=Biochimie;  citation_year=1991;  citation_volume=73;  citation_pages=1345-9; " /><meta name="citation_reference" content="citation_title=Wobble inosine tRNA modification is essential to cell cycle progression in G1/S and G2/M transitions in fission yeast;  citation_journal_title=J. Biol. Chem.;  citation_year=2007;  citation_volume=282;  citation_pages=33459-65; " /><meta name="citation_reference" content="citation_title=tadA, an essential tRNA-specific adenosine deaminase from Escherichia coli;  citation_journal_title=EMBO J.;  citation_year=2002;  citation_volume=21;  citation_pages=3841-51; " /><meta name="citation_reference" content="citation_title=Arabidopsis tRNA adenosine deaminase arginine edits the wobble nucleotide of chloroplast tRNAArg (ACG) and is essential for efficient chloroplast translation;  citation_journal_title=Plant Cell;  citation_year=2009;  citation_volume=21;  citation_pages=2058-71; " /><meta name="citation_reference" content="citation_title=Identification and characterization of a human tRNA-specific adenosine deaminase related to the ADAR family of pre-mRNA editing enzymes;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=1999;  citation_volume=96;  citation_pages=8895-900; " /><meta name="citation_reference" content="citation_title=Molecular mechanism of lysidine synthesis that determines tRNA identity and codon recognition;  citation_journal_title=Mol. Cell;  citation_year=2005;  citation_volume=19;  citation_pages=235-46; " /><meta name="citation_reference" content="citation_title=Analysis of codon: anticodon interactions within the ribosome provides new insights into codon reading and the genetic code structure;  citation_journal_title=RNA;  citation_year=2001;  citation_volume=7;  citation_pages=942-57; " /><meta name="citation_reference" content="citation_title=Unexpected correlations between gene expression and codon usage bias from microarray data for the whole Escherichia coli K-12 genome;  citation_journal_title=Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=6976-85; " /><meta name="citation_reference" content="citation_title=Predicting gene expression level from relative codon usage bias: an application to Escherichia coli genome;  citation_journal_title=DNA Res.;  citation_year=2009;  citation_volume=16;  citation_pages=13-30; " /><meta name="citation_reference" content="citation_title=A simplex method for function minimization;  citation_journal_title=Comput. J.;  citation_year=1965;  citation_volume=7;  citation_pages=308-13; " /><meta name="citation_reference" content="citation_title=Reconstructing ancestral gene content by coevolution;  citation_journal_title=Genome Res.;  citation_year=2010;  citation_volume=20;  citation_pages=122-32; " /><meta name="citation_reference" content="citation_title=Comparative proteomic and transcriptomic profiling of the fission yeast Schizosaccharomyces pombe;  citation_journal_title=Mol. Syst. Biol.;  citation_year=2007;  citation_volume=3;  citation_pages=79" /><meta name="citation_reference" content="citation_title=Conservation of the relative tRNA composition in healthy and cancerous tissues;  citation_journal_title=RNA;  citation_year=2012;  citation_volume=18;  citation_pages=640-52; " /><meta name="citation_reference" content="citation_title=Determinants of protein abundance and translation efficiency in S. cerevisiae;  citation_journal_title=PLoS Comput. Biol.;  citation_year=2007;  citation_volume=3;  citation_pages=e248" /><meta name="citation_reference" content="citation_title=Absolute protein expression profiling estimates the relative contributions of transcriptional and translational regulation;  citation_journal_title=Nat. Biotechnol.;  citation_year=2006;  citation_volume=25;  citation_pages=117-24; " /><meta name="citation_reference" content="citation_title=Transcript features alone enable accurate prediction and understanding of gene expression evolution in S. cerevisiae,;  citation_journal_title=BMC Bioinfomatics;  citation_year=2013;  citation_volume=14;  citation_issue=Suppl 15;  citation_pages=S1" /><meta name="citation_reference" content="citation_title=A sampling of the yeast proteome;  citation_journal_title=Mol. Cell. Biol.;  citation_year=1999;  citation_volume=19;  citation_pages=7357-68; " /><meta name="citation_reference" content="citation_title=Expression pattern and, surprisingly, gene length shape codon usage in Caenorhabditis, Drosophila, and Arabidopsis;  citation_journal_title=Proc. Natl. Acad. Sci.;  citation_year=1999;  citation_volume=96;  citation_pages=4482-7; " /><meta name="citation_reference" content="citation_title=Relationship of codon bias to mRNA concentration and protein length in Saccharomyces cerevisiae;  citation_journal_title=Yeast;  citation_year=2000;  citation_volume=16;  citation_pages=1131-45; " /><meta name="citation_reference" content="citation_title=Comparing protein abundance and mRNA expression levels on a genomic scale;  citation_journal_title=Genome Biol.;  citation_year=2003;  citation_volume=4;  citation_pages=117" /><meta name="citation_reference" content="citation_title=Global signatures of protein and mRNA expression levels;  citation_journal_title=Mol. BioSyst.;  citation_year=2009;  citation_volume=5;  citation_pages=1512-26; " /><meta name="citation_reference" content="  MacQueen J.   Some methods for classification and analysis of multivariate observations  Proceedings of the fifth Berkeley symposium on mathematical statistics and probability  1967  1  281  97  " /><meta name="citation_reference" content="citation_title=Codon usage bias from tRNA's point of view: redundancy, specialization, and efficient decoding for translation optimization;  citation_journal_title=Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=2279-86; " /><meta name="citation_reference" content="citation_title=Balanced codon usage optimizes eukaryotic translational efficiency;  citation_journal_title=PLoS Genet.;  citation_year=2012;  citation_volume=8;  citation_pages=e1002603" /><meta name="citation_reference" content="citation_title=Structure of a purine-purine wobble base pair in the decoding center of the ribosome;  citation_journal_title=Nat. Struct. Mol. Biol.;  citation_year=2004;  citation_volume=11;  citation_pages=1251-2; " /><meta name="citation_reference" content="citation_title=Agmatidine, a modified cytidine in the anticodon of archaeal tRNAIle, base pairs with adenosine but not with guanosine;  citation_journal_title=Proc. Natl. Acad. Sci.;  citation_year=2010;  citation_volume=107;  citation_pages=2872-7; " /><meta name="citation_reference" content="citation_title=Genome sequences of rare, uncultured bacteria obtained by differential coverage binning of multiple metagenomes;  citation_journal_title=Nat. Biotechnol.;  citation_year=2013;  citation_volume=31;  citation_pages=533-8; " /><meta name="citation_reference" content="citation_publisher=Springer, NY;  citation_title=Structure, function, and genetics of ribosomes;  citation_year=1986; " /><meta name="citation_reference" content="citation_title=One core, two shells: bacterial and eukaryotic ribosomes;  citation_journal_title=Nat. Struct. Mol. Biol.;  citation_year=2012;  citation_volume=19;  citation_pages=560-7; " /><meta name="citation_reference" content="citation_title=A role for tRNA modifications in genome structure and codon usage;  citation_journal_title=Cell;  citation_year=2012;  citation_volume=149;  citation_pages=202-13; " /><meta name="citation_reference" content="citation_title=Genes adopt non-optimal codon usage to generate cell cycle-dependent oscillations in protein levels;  citation_journal_title=Mol. Syst. Biol.;  citation_year=2012;  citation_volume=8;  citation_pages=572" /><meta name="citation_reference" content="citation_title=Coding-sequence determinants of gene expression in Escherichia coli;  citation_journal_title=Science;  citation_year=2009;  citation_volume=324;  citation_pages=255-8; " /><meta name="citation_reference" content="citation_title=On relevance of codon usage to expression of synthetic and natural genes in Escherichia coli;  citation_journal_title=Genetics;  citation_year=2010;  citation_volume=185;  citation_pages=1129-34; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/5/511/17462932/dsu017.pdf" /><meta name="description" content="The tRNA adaptation index (tAI) is a widely used measure of the efficiency by which a coding sequence is recognized by the intra-cellular tRNA pool. This index " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/5/511/2754544" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/5/511/2754544" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/5/511/2754544");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsu017';
    </script>
<title>
	Modelling the Efficiency of Codon–tRNA Interactions Based on Codon Usage Bias | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2754544" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BaubATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2FiaSwgUmVuYW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9tZWRpY2FsIEVuZ2luZWVyaW5nLCBUZWwgQXZpdiBVbml2ZXJzaXR5LCBUZWwgQXZpdiwgSXNyYWVsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVHVsbGVyLCBUYW1pciIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbWVkaWNhbCBFbmdpbmVlcmluZywgVGVsIEF2aXYgVW5pdmVyc2l0eSwgVGVsIEF2aXYsIElzcmFlbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlRoZSBTYWdvbCBTY2hvb2wgb2YgTmV1cm9zY2llbmNlLCBUZWwtQXZpdiBVbml2ZXJzaXR5LCBUZWwtQXZpdiwgSXNyYWVsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJNb2RlbGxpbmcgdGhlIEVmZmljaWVuY3kgb2YgQ29kb27igJN0Uk5BIEludGVyYWN0aW9ucyBCYXNlZCBvbiBDb2RvbiBVc2FnZSBCaWFzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNTExIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI1MjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3UwMTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNC8xMC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgdmFyaWFiaWxpdHkgbWFpbnRhaW5lZCBpbiBhIGZpbml0ZSBwb3B1bGF0aW9uIGR1ZSB0byBtdXRhdGlvbmFsIHByb2R1Y3Rpb24gb2YgbmV1dHJhbCBhbmQgbmVhcmx5IG5ldXRyYWwgaXNvYWxsZWxlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXQuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5Njg7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0yNDctNjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIGNhdGFsb2cgdXNhZ2UgYW5kIHRoZSBnZW5vbWUgaHlwb3RoZXNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTgwOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0xOTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeW5vbnltb3VzIGJ1dCBub3QgdGhlIHNhbWU6IHRoZSBjYXVzZXMgYW5kIGNvbnNlcXVlbmNlcyBvZiBjb2RvbiBiaWFzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MzItNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIHVzYWdlIGJpYXM6IGNhdXNhdGl2ZSBmYWN0b3JzLCBxdWFudGlmaWNhdGlvbiBtZXRob2RzIGFuZCBnZW5vbWUtd2lkZSBwYXR0ZXJuczogd2l0aCBlbXBoYXNpcyBvbiBpbnNlY3QgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvbC4gUmV2LiBDYW1iLiBQaGlsb3MuIFNvYy47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODg7ICBjaXRhdGlvbl9wYWdlcz00OS02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29hZGFwdGF0aW9uIG9mIGlzb2FjY2VwdG9yIHRSTkEgZ2VuZXMgYW5kIGNvZG9uIHVzYWdlIGJpYXMgZm9yIHRyYW5zbGF0aW9uIGVmZmljaWVuY3kgaW4gQWVkZXMgYWVneXB0aSBhbmQgQW5vcGhlbGVzIGdhbWJpYWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUluc2VjdCBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTc3LTg3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBhbmFseXNpcyBvZiBudWNsZWFyIHRSTkEgZ2VuZXMgb2YgTmFzb25pYSB2aXRyaXBlbm5pcyBhbmQgb3RoZXIgYXJ0aHJvcG9kcywgYW5kIHJlbGF0aW9uc2hpcHMgdG8gY29kb24gdXNhZ2UgYmlhczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW5zZWN0IE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz00OS01ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhvbmljIHRyYW5zY3JpcHRpb24gZmFjdG9yIGJpbmRpbmcgZGlyZWN0cyBjb2RvbiBjaG9pY2UgYW5kIGFmZmVjdHMgcHJvdGVpbiBldm9sdXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzQyOyAgY2l0YXRpb25fcGFnZXM9MTM2Ny03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U29sdmluZyB0aGUgcmlkZGxlIG9mIGNvZG9uIHVzYWdlIHByZWZlcmVuY2VzOiBhIHRlc3QgZm9yIHRyYW5zbGF0aW9uYWwgc2VsZWN0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz01MDM2LTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2xhdGlvbiBlZmZpY2llbmN5IGlzIGRldGVybWluZWQgYnkgYm90aCBjb2RvbiBiaWFzIGFuZCBmb2xkaW5nIGVuZXJneTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA3OyAgY2l0YXRpb25fcGFnZXM9MzY0NS01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29hbGVzY2VudCBzaW11bGF0aW9uIG9mIGludHJhY29kb24gcmVjb21iaW5hdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTg0OyAgY2l0YXRpb25fcGFnZXM9NDI5LTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiBzZWxlY3Rpb24gaW4geWVhc3Q7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0xOTgyOyAgY2l0YXRpb25fdm9sdW1lPTI1NzsgIGNpdGF0aW9uX3BhZ2VzPTMwMjYtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWZlcmVudGlhbCBjb2RvbiB1c2FnZSBpbiBwcm9rYXJ5b3RpYyBnZW5lczogdGhlIG9wdGltYWwgY29kb24tYW50aWNvZG9uIGludGVyYWN0aW9uIGVuZXJneSBhbmQgdGhlIHNlbGVjdGl2ZSBjb2RvbiB1c2FnZSBpbiBlZmZpY2llbnRseSBleHByZXNzZWQgZ2VuZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmU7ICBjaXRhdGlvbl95ZWFyPTE5ODI7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0xOTktMjA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUg4oCYZWZmZWN0aXZlIG51bWJlciBvZiBjb2RvbnPigJkgdXNlZCBpbiBhIGdlbmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmU7ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9ODc7ICBjaXRhdGlvbl9wYWdlcz0yMy05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiBiaWFzIGFuZCBnZW5lIGV4cHJlc3Npb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZFQlMgTGV0dC47ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9Mjg1OyAgY2l0YXRpb25fcGFnZXM9MTY1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIHVzYWdlIGluIENhZW5vcmhhYmRpdGlzIGVsZWdhbnM6IGRlbGluZWF0aW9uIG9mIHRyYW5zbGF0aW9uYWwgc2VsZWN0aW9uIGFuZCBtdXRhdGlvbmFsIGJpYXNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MjQzNy00NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNvZG9uIGFkYXB0YXRpb24gaW5kZXgtYSBtZWFzdXJlIG9mIGRpcmVjdGlvbmFsIHN5bm9ueW1vdXMgY29kb24gdXNhZ2UgYmlhcywgYW5kIGl0cyBwb3RlbnRpYWwgYXBwbGljYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xMjgxLTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm90ZWluIGFidW5kYW5jZSBwcm9maWxpbmcgb2YgdGhlIEVzY2hlcmljaGlhIGNvbGkgY3l0b3NvbDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0xMDIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HbG9iYWwgYW5hbHlzaXMgb2YgcHJvdGVpbiBleHByZXNzaW9uIGluIHllYXN0OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDI1OyAgY2l0YXRpb25fcGFnZXM9NzM3LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYXJnZS1zY2FsZSBhbmFseXNpcyBvZiB0aGUgeWVhc3QgcHJvdGVvbWUgYnkgbXVsdGlkaW1lbnNpb25hbCBwcm90ZWluIGlkZW50aWZpY2F0aW9uIHRlY2hub2xvZ3k7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0yNDItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2h5IGhpZ2hseSBleHByZXNzZWQgcHJvdGVpbnMgZXZvbHZlIHNsb3dseTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTAyOyAgY2l0YXRpb25fcGFnZXM9MTQzMzgtNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb21wbGV0ZSBnZW5vbWUgc2VxdWVuY2Ugb2YgRXNjaGVyaWNoaWEgY29saSBLLTEyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI3NzsgIGNpdGF0aW9uX3BhZ2VzPTE0NTMtNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEg4oCYc2lsZW504oCZIHBvbHltb3JwaGlzbSBpbiB0aGUgTURSMSBnZW5lIGNoYW5nZXMgc3Vic3RyYXRlIHNwZWNpZmljaXR5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTMxNTsgIGNpdGF0aW9uX3BhZ2VzPTUyNS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWxhdGl2ZSBjb2RvbiBhZGFwdGF0aW9uOiBhIGdlbmVyaWMgY29kb24gYmlhcyBpbmRleCBmb3IgcHJlZGljdGlvbiBvZiBnZW5lIGV4cHJlc3Npb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MTg1LTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db3JyZWxhdGlvbiBiZXR3ZWVuIHRoZSBhYnVuZGFuY2Ugb2YgRXNjaGVyaWNoaWEgY29saSB0cmFuc2ZlciBSTkFzIGFuZCB0aGUgb2NjdXJyZW5jZSBvZiB0aGUgcmVzcGVjdGl2ZSBjb2RvbnMgaW4gaXRzIHByb3RlaW4gZ2VuZXM6IGEgcHJvcG9zYWwgZm9yIGEgc3lub255bW91cyBjb2RvbiBjaG9pY2UgdGhhdCBpcyBvcHRpbWFsIGZvciB0aGUgRS4gY29saSB0cmFuc2xhdGlvbmFsIHN5c3RlbTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk4MTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTE7ICBjaXRhdGlvbl9wYWdlcz0zODktNDA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiB1c2FnZSBhbmQgdFJOQSBjb250ZW50IGluIHVuaWNlbGx1bGFyIGFuZCBtdWx0aWNlbGx1bGFyIG9yZ2FuaXNtczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk4NTsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9MTMtMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPW1STkFzIGhhdmUgZ3JlYXRlciBuZWdhdGl2ZSBmb2xkaW5nIGZyZWUgZW5lcmdpZXMgdGhhbiBzaHVmZmxlZCBvciBjb2RvbiBjaG9pY2UgcmFuZG9taXplZCBzZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTE1NzgtODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJvbGUgb2YgR0MtYmlhc2VkIG11dGF0aW9uIHByZXNzdXJlIG9uIHN5bm9ueW1vdXMgY29kb24gY2hvaWNlIGluIE1pY3JvY29jY3VzIGx1dGV1cyBhIGJhY3Rlcml1bSB3aXRoIGEgaGlnaCBnZW5vbWljIEdDLWNvbnRlbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTE1NjUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29ycmVsYXRpb24gYmV0d2VlbiB0aGUgYWJ1bmRhbmNlIG9mIHllYXN0IHRyYW5zZmVyIFJOQXMgYW5kIHRoZSBvY2N1cnJlbmNlIG9mIHRoZSByZXNwZWN0aXZlIGNvZG9ucyBpbiBwcm90ZWluIGdlbmVzOiBkaWZmZXJlbmNlcyBpbiBzeW5vbnltb3VzIGNvZG9uIGNob2ljZSBwYXR0ZXJucyBvZiB5ZWFzdCBhbmQgRXNjaGVyaWNoaWEgY29saSB3aXRoIHJlZmVyZW5jZSB0byB0aGUgYWJ1bmRhbmNlIG9mIGlzb2FjY2VwdGluZyB0cmFuc2ZlciBSTkFzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTgyOyAgY2l0YXRpb25fdm9sdW1lPTE1ODsgIGNpdGF0aW9uX3BhZ2VzPTU3My05NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNmZXIgUk5BIGdlbmUgcmVkdW5kYW5jeSBhbmQgdHJhbnNsYXRpb25hbCBzZWxlY3Rpb24gaW4gU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI2ODsgIGNpdGF0aW9uX3BhZ2VzPTMyMi0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlmZmVyZW50aWFsIHRyYW5zbGF0aW9uIGVmZmljaWVuY3kgb2Ygb3J0aG9sb2dvdXMgZ2VuZXMgaXMgaW52b2x2ZWQgaW4gcGhlbm90eXBpYyBkaXZlcmdlbmNlIG9mIHllYXN0IHNwZWNpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9NDE1LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WYXJpYXRpb24gaW4gdGhlIHN0cmVuZ3RoIG9mIHNlbGVjdGVkIGNvZG9uIHVzYWdlIGJpYXMgYW1vbmcgYmFjdGVyaWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTExNDEtNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhlYXJpbmcgc2lsZW5jZTogbm9uLW5ldXRyYWwgZXZvbHV0aW9uIGF0IHN5bm9ueW1vdXMgc2l0ZXMgaW4gbWFtbWFsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9OTgtMTA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHBlcmlvZGljIHBhdHRlcm4gb2YgbVJOQSBzZWNvbmRhcnkgc3RydWN0dXJlIGNyZWF0ZWQgYnkgdGhlIGdlbmV0aWMgY29kZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9MjQyOC0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gZXZvbHV0aW9uYXJpbHkgY29uc2VydmVkIG1lY2hhbmlzbSBmb3IgY29udHJvbGxpbmcgdGhlIGVmZmljaWVuY3kgb2YgcHJvdGVpbiB0cmFuc2xhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDE7ICBjaXRhdGlvbl9wYWdlcz0zNDQtNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJpYm9zb21lIHByb2ZpbGluZyBvZiBtb3VzZSBlbWJyeW9uaWMgc3RlbSBjZWxscyByZXZlYWxzIHRoZSBjb21wbGV4aXR5IGFuZCBkeW5hbWljcyBvZiBtYW1tYWxpYW4gcHJvdGVvbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE0NzsgIGNpdGF0aW9uX3BhZ2VzPTc4OS04MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0dWRpZXMgb2YgY29kb24gdXNhZ2UgYW5kIHRSTkEgZ2VuZXMgb2YgMTggdW5pY2VsbHVsYXIgb3JnYW5pc21zIGFuZCBxdWFudGlmaWNhdGlvbiBvZiBCYWNpbGx1cyBzdWJ0aWxpcyB0Uk5BczogZ2VuZSBleHByZXNzaW9uIGxldmVsIGFuZCBzcGVjaWVzLXNwZWNpZmljIGRpdmVyc2l0eSBvZiBjb2RvbiB1c2FnZSBiYXNlZCBvbiBtdWx0aXZhcmlhdGUgYW5hbHlzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmU7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MjM4OyAgY2l0YXRpb25fcGFnZXM9MTQzLTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2Rvbi1hbnRpY29kb24gcGFpcmluZzogdGhlIHdvYmJsZSBoeXBvdGhlc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTY2OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9NTQ4LTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGNvZGUuIE1vZGlmaWVkIGJhc2VzIGFuZCBhbWlub2FjeWxhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTMzNjsgIGNpdGF0aW9uX3BhZ2VzPTExMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdvYmJsZSBwb3NpdGlvbiBtb2RpZmllZCBudWNsZW9zaWRlcyBldm9sdmVkIHRvIHNlbGVjdCB0cmFuc2ZlciBSTkEgY29kb24gcmVjb2duaXRpb246IGEgbW9kaWZpZWQtd29iYmxlIGh5cG90aGVzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2NoaW1pZTsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT03MzsgIGNpdGF0aW9uX3BhZ2VzPTEzNDUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V29iYmxlIGlub3NpbmUgdFJOQSBtb2RpZmljYXRpb24gaXMgZXNzZW50aWFsIHRvIGNlbGwgY3ljbGUgcHJvZ3Jlc3Npb24gaW4gRzEvUyBhbmQgRzIvTSB0cmFuc2l0aW9ucyBpbiBmaXNzaW9uIHllYXN0OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yODI7ICBjaXRhdGlvbl9wYWdlcz0zMzQ1OS02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9dGFkQSwgYW4gZXNzZW50aWFsIHRSTkEtc3BlY2lmaWMgYWRlbm9zaW5lIGRlYW1pbmFzZSBmcm9tIEVzY2hlcmljaGlhIGNvbGk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVNQk8gSi47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0zODQxLTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcmFiaWRvcHNpcyB0Uk5BIGFkZW5vc2luZSBkZWFtaW5hc2UgYXJnaW5pbmUgZWRpdHMgdGhlIHdvYmJsZSBudWNsZW90aWRlIG9mIGNobG9yb3BsYXN0IHRSTkFBcmcgKEFDRykgYW5kIGlzIGVzc2VudGlhbCBmb3IgZWZmaWNpZW50IGNobG9yb3BsYXN0IHRyYW5zbGF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MjA1OC03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgYSBodW1hbiB0Uk5BLXNwZWNpZmljIGFkZW5vc2luZSBkZWFtaW5hc2UgcmVsYXRlZCB0byB0aGUgQURBUiBmYW1pbHkgb2YgcHJlLW1STkEgZWRpdGluZyBlbnp5bWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT05NjsgIGNpdGF0aW9uX3BhZ2VzPTg4OTUtOTAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgbWVjaGFuaXNtIG9mIGx5c2lkaW5lIHN5bnRoZXNpcyB0aGF0IGRldGVybWluZXMgdFJOQSBpZGVudGl0eSBhbmQgY29kb24gcmVjb2duaXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTIzNS00NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgY29kb246IGFudGljb2RvbiBpbnRlcmFjdGlvbnMgd2l0aGluIHRoZSByaWJvc29tZSBwcm92aWRlcyBuZXcgaW5zaWdodHMgaW50byBjb2RvbiByZWFkaW5nIGFuZCB0aGUgZ2VuZXRpYyBjb2RlIHN0cnVjdHVyZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Uk5BOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz05NDItNTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVuZXhwZWN0ZWQgY29ycmVsYXRpb25zIGJldHdlZW4gZ2VuZSBleHByZXNzaW9uIGFuZCBjb2RvbiB1c2FnZSBiaWFzIGZyb20gbWljcm9hcnJheSBkYXRhIGZvciB0aGUgd2hvbGUgRXNjaGVyaWNoaWEgY29saSBLLTEyIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9Njk3Ni04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlZGljdGluZyBnZW5lIGV4cHJlc3Npb24gbGV2ZWwgZnJvbSByZWxhdGl2ZSBjb2RvbiB1c2FnZSBiaWFzOiBhbiBhcHBsaWNhdGlvbiB0byBFc2NoZXJpY2hpYSBjb2xpIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xMy0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzaW1wbGV4IG1ldGhvZCBmb3IgZnVuY3Rpb24gbWluaW1pemF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Db21wdXQuIEouOyAgY2l0YXRpb25feWVhcj0xOTY1OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0zMDgtMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlY29uc3RydWN0aW5nIGFuY2VzdHJhbCBnZW5lIGNvbnRlbnQgYnkgY29ldm9sdXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTIyLTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBwcm90ZW9taWMgYW5kIHRyYW5zY3JpcHRvbWljIHByb2ZpbGluZyBvZiB0aGUgZmlzc2lvbiB5ZWFzdCBTY2hpem9zYWNjaGFyb215Y2VzIHBvbWJlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFN5c3QuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz03OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnNlcnZhdGlvbiBvZiB0aGUgcmVsYXRpdmUgdFJOQSBjb21wb3NpdGlvbiBpbiBoZWFsdGh5IGFuZCBjYW5jZXJvdXMgdGlzc3VlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Uk5BOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9NjQwLTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlcm1pbmFudHMgb2YgcHJvdGVpbiBhYnVuZGFuY2UgYW5kIHRyYW5zbGF0aW9uIGVmZmljaWVuY3kgaW4gUy4gY2VyZXZpc2lhZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBDb21wdXQuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz1lMjQ4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWJzb2x1dGUgcHJvdGVpbiBleHByZXNzaW9uIHByb2ZpbGluZyBlc3RpbWF0ZXMgdGhlIHJlbGF0aXZlIGNvbnRyaWJ1dGlvbnMgb2YgdHJhbnNjcmlwdGlvbmFsIGFuZCB0cmFuc2xhdGlvbmFsIHJlZ3VsYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xMTctMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHQgZmVhdHVyZXMgYWxvbmUgZW5hYmxlIGFjY3VyYXRlIHByZWRpY3Rpb24gYW5kIHVuZGVyc3RhbmRpbmcgb2YgZ2VuZSBleHByZXNzaW9uIGV2b2x1dGlvbiBpbiBTLiBjZXJldmlzaWFlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9tYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9pc3N1ZT1TdXBwbCAxNTsgIGNpdGF0aW9uX3BhZ2VzPVMxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzYW1wbGluZyBvZiB0aGUgeWVhc3QgcHJvdGVvbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQ2VsbC7CoEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9NzM1Ny02ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwcmVzc2lvbiBwYXR0ZXJuIGFuZCwgc3VycHJpc2luZ2x5LCBnZW5lIGxlbmd0aCBzaGFwZSBjb2RvbiB1c2FnZSBpbiBDYWVub3JoYWJkaXRpcywgRHJvc29waGlsYSwgYW5kIEFyYWJpZG9wc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTk2OyAgY2l0YXRpb25fcGFnZXM9NDQ4Mi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWxhdGlvbnNoaXAgb2YgY29kb24gYmlhcyB0byBtUk5BIGNvbmNlbnRyYXRpb24gYW5kIHByb3RlaW4gbGVuZ3RoIGluIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9WWVhc3Q7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xMTMxLTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJpbmcgcHJvdGVpbiBhYnVuZGFuY2UgYW5kIG1STkEgZXhwcmVzc2lvbiBsZXZlbHMgb24gYSBnZW5vbWljIHNjYWxlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTExNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdsb2JhbCBzaWduYXR1cmVzIG9mIHByb3RlaW4gYW5kIG1STkEgZXhwcmVzc2lvbiBsZXZlbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvU3lzdC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTE1MTItMjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IiAgTWFjUXVlZW4gSi4gICBTb21lIG1ldGhvZHMgZm9yIGNsYXNzaWZpY2F0aW9uIGFuZCBhbmFseXNpcyBvZiBtdWx0aXZhcmlhdGUgb2JzZXJ2YXRpb25zICBQcm9jZWVkaW5ncyBvZiB0aGUgZmlmdGggQmVya2VsZXkgc3ltcG9zaXVtIG9uIG1hdGhlbWF0aWNhbCBzdGF0aXN0aWNzIGFuZCBwcm9iYWJpbGl0eSAgMTk2NyAgMSAgMjgxICA5NyAgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgYmlhcyBmcm9tIHRSTkEncyBwb2ludCBvZiB2aWV3OiByZWR1bmRhbmN5LCBzcGVjaWFsaXphdGlvbiwgYW5kIGVmZmljaWVudCBkZWNvZGluZyBmb3IgdHJhbnNsYXRpb24gb3B0aW1pemF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTIyNzktODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhbGFuY2VkIGNvZG9uIHVzYWdlIG9wdGltaXplcyBldWthcnlvdGljIHRyYW5zbGF0aW9uYWwgZWZmaWNpZW5jeTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAyNjAzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJlIG9mIGEgcHVyaW5lLXB1cmluZSB3b2JibGUgYmFzZSBwYWlyIGluIHRoZSBkZWNvZGluZyBjZW50ZXIgb2YgdGhlIHJpYm9zb21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFN0cnVjdC4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTEyNTEtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWdtYXRpZGluZSwgYSBtb2RpZmllZCBjeXRpZGluZSBpbiB0aGUgYW50aWNvZG9uIG9mIGFyY2hhZWFsIHRSTkFJbGUsIGJhc2UgcGFpcnMgd2l0aCBhZGVub3NpbmUgYnV0IG5vdCB3aXRoIGd1YW5vc2luZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDc7ICBjaXRhdGlvbl9wYWdlcz0yODcyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZXMgb2YgcmFyZSwgdW5jdWx0dXJlZCBiYWN0ZXJpYSBvYnRhaW5lZCBieSBkaWZmZXJlbnRpYWwgY292ZXJhZ2UgYmlubmluZyBvZiBtdWx0aXBsZSBtZXRhZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTUzMy04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXIsIE5ZOyAgY2l0YXRpb25fdGl0bGU9U3RydWN0dXJlLCBmdW5jdGlvbiwgYW5kIGdlbmV0aWNzIG9mIHJpYm9zb21lczsgIGNpdGF0aW9uX3llYXI9MTk4NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T25lIGNvcmUsIHR3byBzaGVsbHM6IGJhY3RlcmlhbCBhbmQgZXVrYXJ5b3RpYyByaWJvc29tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gU3RydWN0LiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9NTYwLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcm9sZSBmb3IgdFJOQSBtb2RpZmljYXRpb25zIGluIGdlbm9tZSBzdHJ1Y3R1cmUgYW5kIGNvZG9uIHVzYWdlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE0OTsgIGNpdGF0aW9uX3BhZ2VzPTIwMi0xMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXMgYWRvcHQgbm9uLW9wdGltYWwgY29kb24gdXNhZ2UgdG8gZ2VuZXJhdGUgY2VsbCBjeWNsZS1kZXBlbmRlbnQgb3NjaWxsYXRpb25zIGluIHByb3RlaW4gbGV2ZWxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFN5c3QuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz01NzIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2Rpbmctc2VxdWVuY2UgZGV0ZXJtaW5hbnRzIG9mIGdlbmUgZXhwcmVzc2lvbiBpbiBFc2NoZXJpY2hpYSBjb2xpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTMyNDsgIGNpdGF0aW9uX3BhZ2VzPTI1NS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbiByZWxldmFuY2Ugb2YgY29kb24gdXNhZ2UgdG8gZXhwcmVzc2lvbiBvZiBzeW50aGV0aWMgYW5kIG5hdHVyYWwgZ2VuZXMgaW4gRXNjaGVyaWNoaWEgY29saTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTg1OyAgY2l0YXRpb25fcGFnZXM9MTEyOS0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIxLzUvNTExLzE3NDYyOTMyL2RzdTAxNy5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IlRoZSB0Uk5BIGFkYXB0YXRpb24gaW5kZXggKHRBSSkgaXMgYSB3aWRlbHkgdXNlZCBtZWFzdXJlIG9mIHRoZSBlZmZpY2llbmN5IGJ5IHdoaWNoIGEgY29kaW5nIHNlcXVlbmNlIGlzIHJlY29nbml6ZWQgYnkgdGhlIGludHJhLWNlbGx1bGFyIHRSTkEgcG9vbC4gVGhpcyBpbmRleCAiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTI3NTQ1NDQmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkpmrSmASdnZE/t1RuKp7LnnsFlUU=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f5%2f511%2f2754544" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f5%2f511%2f2754544" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=15969" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/5/1/m_cover.gif?Expires=2147483647&amp;Signature=gPiJSb93IIOcIMTSkMAi7kcKhziIjvuRKEzK8XVp8CTVu4eB811a4N7c5iRqAkFTvycBeFOL1qu9DtcvWUQE-HFcU90uhCx7UXftpGVaIU0r-58rykRAQjr7OyCGXhRRXufrlhLC-AFU8QeGWWE7goVK9s00kVi5KaqA8BdZOl4ZUh6UGigqEAXS9fKMJcyv7L69eSvyhk8fr0u2lpE6Z~kKXJ-zph2mYgOTE6TZsnpM7S63PaSYvqdIQyt~fHxa7pkm4binASfSKcJvbv9WOoMwYew88jtLZv495c~AkAPvlFj827Es8r960R62hvuI-oun7Y-AfhyLYD5usT~QKA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/5">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87197581">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197581">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197618">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197618">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197682">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197682">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197746">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197746">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197756">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197756">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197758">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197758">Acknowledgement</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197760">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197760">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/5/499/2754542">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/21/5/527/2754547">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Modelling the Efficiency of Codon–tRNA Interactions Based on Codon Usage Bias
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Renana Sabi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biomedical Engineering</div>, <div class="insititution">Tel Aviv University</div>, <div class="addr-line">Tel Aviv</div>, <div class="country">Israel</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Renana+Sabi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sabi R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sabi R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Renana Sabi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tamir Tuller</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biomedical Engineering</div>, <div class="insititution">Tel Aviv University</div>, <div class="addr-line">Tel Aviv</div>, <div class="country">Israel</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">The Sagol School of Neuroscience</div>, <div class="insititution">Tel-Aviv University</div>, <div class="addr-line">Tel-Aviv</div>, <div class="country">Israel</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +972-64058363. Fax. +972-64058363. Email: <a href="mailto:tamirtul@post.tau.ac.il" target="_blank">tamirtul@post.tau.ac.il</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tamir+Tuller">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tuller T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tuller T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tamir Tuller</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="an1"><div class="fn"><div class="footnote-content">Edited by Prof. Hiroyuki Toh</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 5, 1 October 2014, Pages 511–526, <a href='https://doi.org/10.1093/dnares/dsu017'>https://doi.org/10.1093/dnares/dsu017</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">06 June 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">10 January 2014</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">26 April 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2754544" href="/dnaresearch/article-pdf/21/5/511/17462932/dsu017.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Renana Sabi, Tamir Tuller;  Modelling the Efficiency of Codon–tRNA Interactions Based on Codon Usage Bias, <em>DNA Research</em>, Volume 21, Issue 5, 1 October 2014, Pages 511–526, <a href='https://doi.org/10.1093/dnares/dsu017'>https://doi.org/10.1093/dnares/dsu017</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2754544?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754544?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754544?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754544?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754544?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754544?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754544?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Modelling%20the%20Efficiency%20of%20Codon%E2%80%93tRNA%20Interactions%20Based%20on%20Codon%20Usage%20Bias&amp;publicationDate=2014-06-06&amp;volumeNum=21&amp;issueNum=5&amp;author=Sabi%2C%20Renana%3B%20Tuller%2C%20Tamir&amp;startPage=511&amp;endPage=526&amp;contentId=10.1093%2Fdnares%2Fdsu017&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f5%2f511%2f2754544" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





<section class="abstract"><p>The tRNA adaptation index (tAI) is a widely used measure of the efficiency by which a coding sequence is recognized by the intra-cellular tRNA pool. This index includes among others weights that represent wobble interactions between codons and tRNA molecules. Currently, these weights are based only on the gene expression in <em>Saccharomyces cerevisiae</em>. However, the efficiencies of the different codon–tRNA interactions are expected to vary among different organisms. In this study, we suggest a new approach for adjusting the tAI weights to any target model organism without the need for gene expression measurements. Our method is based on optimizing the correlation between the tAI and a measure of codon usage bias. Here, we show that in non-fungal the new tAI weights predict protein abundance significantly better than the traditional tAI weights. The unique tRNA–codon adaptation weights computed for 100 different organisms exhibit a significant correlation with evolutionary distance. The reported results demonstrate the usefulness of the new measure in future genomic studies.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;codon usage bias&quot;">codon usage bias</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;tRNA adaptation index&quot;">tRNA adaptation index</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;protein levels&quot;">protein levels</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;wobble interactions&quot;">wobble interactions</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="ribosome">ribosome</a></div>                            <h2 scrollto-destination=87197581 id="87197581" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Allowed by the redundancy of the genetic code, coding regions exhibit non-uniform usage of synonymous codons. This deviation from uniform codon usage is termed codon usage bias (CUB) and is related among others to various aspects of gene translation (and more generally gene expression) and its efficiency;<sup><a reveal-id="DSU017C1" data-open="DSU017C1" class="link link-ref link-reveal xref-bibr">1</a>–<a reveal-id="DSU017C10" data-open="DSU017C10" class="link link-ref link-reveal xref-bibr">10</a></sup> specifically, it was suggested that it regulates transcription and translation, but may also be related to recombination rate. Indeed, it is known for over 30 years that in most organisms the degree of CUB of genes correlates with their expression levels.<sup><a reveal-id="DSU017C11" data-open="DSU017C11" class="link link-ref link-reveal xref-bibr">11</a>–<a reveal-id="DSU017C14" data-open="DSU017C14" class="link link-ref link-reveal xref-bibr">14</a></sup></p><p>Various approaches for quantifying the CUB of a gene have been suggested in the last decades: the effective number of codons, for instance, measures deviations from equal usage of synonymous codons,<sup><a reveal-id="DSU017C13" data-open="DSU017C13" class="link link-ref link-reveal xref-bibr">13</a></sup> while other indices such as the frequency of optimal codons,<sup><a reveal-id="DSU017C15" data-open="DSU017C15" class="link link-ref link-reveal xref-bibr">15</a></sup> the codon bias index,<sup><a reveal-id="DSU017C11" data-open="DSU017C11" class="link link-ref link-reveal xref-bibr">11</a></sup> and the codon adaptation index (CAI)<sup><a reveal-id="DSU017C16" data-open="DSU017C16" class="link link-ref link-reveal xref-bibr">16</a></sup> define a subset of ‘optimal’ codons and measure the frequency of these codons in the coding region of the gene.</p><p>The CUB indices have been employed in hundreds of previous studies. For example, it is known that in many organisms (e.g. <em>Escherichia coli</em>) the CAI exhibits a very high correlation with protein levels (similar to the one obtained between mRNA levels and protein levels<sup><a reveal-id="DSU017C17" data-open="DSU017C17" class="link link-ref link-reveal xref-bibr">17</a></sup>); thus, CAI has been frequently used as a proxy of expression levels (see, for example,<sup><a reveal-id="DSU017C18" data-open="DSU017C18" class="link link-ref link-reveal xref-bibr">18</a>–<a reveal-id="DSU017C20" data-open="DSU017C20" class="link link-ref link-reveal xref-bibr">20</a></sup>). In addition, it has been employed in a vast number of key studies in the recent years.<sup><a reveal-id="DSU017C18" data-open="DSU017C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSU017C19" data-open="DSU017C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSU017C21" data-open="DSU017C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSU017C22" data-open="DSU017C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p><p>One disadvantage of measures that are based on a set of reference genes<sup><a reveal-id="DSU017C11" data-open="DSU017C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSU017C16" data-open="DSU017C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSU017C23" data-open="DSU017C23" class="link link-ref link-reveal xref-bibr">23</a></sup> is the fact that in the case of organisms with poor genomic data and without large scale cellular measurements, creating a meaningful reference set is challenging. Another disadvantage of these measures is the fact that they cannot separate between the various possible causes of CUB in highly expressed genes: some of them may be related directly to the translation process (e.g. co-evolution and/or adaptation to the tRNA pool<sup><a reveal-id="DSU017C8" data-open="DSU017C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSU017C24" data-open="DSU017C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSU017C25" data-open="DSU017C25" class="link link-ref link-reveal xref-bibr">25</a></sup>) and others may not be related to translation (e.g. GC content and folding<sup><a reveal-id="DSU017C9" data-open="DSU017C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSU017C26" data-open="DSU017C26" class="link link-ref link-reveal xref-bibr">26</a>–<a reveal-id="DSU017C28" data-open="DSU017C28" class="link link-ref link-reveal xref-bibr">28</a></sup>).</p><p>In 2004, dos Reis <em>et al.</em><sup><a reveal-id="DSU017C8" data-open="DSU017C8" class="link link-ref link-reveal xref-bibr">8</a></sup> proposed the tRNA adaptation index (tAI), which aims to estimate only the adaptation of codons/genes to some aspects directly related to the elongation step occurs in the ribosome via the adaptation to the tRNA pool, wobble interactions, and properties of the ribosome. Specifically, the tAI considers the fact that different tRNA species can recognize a codon with different affinities.<sup><a reveal-id="DSU017C2" data-open="DSU017C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSU017C8" data-open="DSU017C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSU017C29" data-open="DSU017C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Thus, the tAI is different than CUB-based measurements mentioned above and provides important information related to translation that is not necessarily covered by CUB measures.</p><p>Indeed, measures of the adaptation of genes to the tRNA pool (such as the tAI) have extensively been used in the recent years for studying questions in diverse biomedical disciplines such as evolutionary biology, functional genomics, and systems biology (see, for example,<sup><a reveal-id="DSU017C3" data-open="DSU017C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSU017C30" data-open="DSU017C30" class="link link-ref link-reveal xref-bibr">30</a>–<a reveal-id="DSU017C35" data-open="DSU017C35" class="link link-ref link-reveal xref-bibr">35</a></sup>).</p><div class="block-child-p">Let <em>n<sub>i</sub></em> be the number of tRNA isoacceptors that recognize the <em>i</em>th codon, the absolute adaptiveness value of the <em>i</em>th codon is defined in the following equation: <a id="DSU017M1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU017M1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>W</mi><mi>i</mi></msub></mrow><mspace width="thinmathspace" xmlns=""></mspace><mo xmlns="">=</mo><mspace width="thinmathspace" xmlns=""></mspace><munderover xmlns=""><mo movablelimits="false">∑</mo><mrow><mspace width="thinmathspace"></mspace><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mrow><mrow><msub><mi>n</mi><mi>i</mi></msub></mrow></mrow></munderover><mrow xmlns=""><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><mrow><msub><mi>S</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow><mo stretchy="false">)</mo><mo>⋅</mo><mtext>tGC</mtext><mrow><msub><mtext>N</mtext><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mrow></math></div></div><div class="label">(1)</div></div></div><p>where tGCN<em><sub>ij</sub></em> is the gene copy number of the <em>j</em>th tRNA that recognizes the <em>i</em>th codon (a proxy of the tRNA levels<sup><a reveal-id="DSU017C24" data-open="DSU017C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSU017C29" data-open="DSU017C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSU017C36" data-open="DSU017C36" class="link link-ref link-reveal xref-bibr">36</a></sup>), and <em>S<sub>ij</sub></em> is a selective constraint on the efficiency of the interaction between the <em>i</em>th codon and the <em>j</em>th tRNA, which is scored between 0 (perfect interaction) and 1 (no interaction);<sup><a reveal-id="DSU017C8" data-open="DSU017C8" class="link link-ref link-reveal xref-bibr">8</a></sup> specifically, the <em>S<sub>ij</sub></em> weights can be related to aspects of translation elongation (tRNA, wobble interactions, and properties of the ribosome), as these aspects are expected to affect the efficiency of the codon–anticodon interaction. <em>W<sub>i</sub></em> values are calculated according to Crick's wobble rules for codon–anticodon pairing (Table <a reveal-id="DSU017TB1" data-open="DSU017TB1" class="link link-reveal link-table xref-fig">1</a>). The codon relative adaptiveness value <em>w<sub>i</sub></em> is obtained by dividing each <em>W<sub>i</sub></em> with the maximum <em>W<sub>i</sub></em> value over all codons.<sup><a reveal-id="DSU017C8" data-open="DSU017C8" class="link link-ref link-reveal xref-bibr">8</a></sup> The tAI of a gene is defined as the geometric mean of the <em>w<sub>i</sub></em> values of its codons. <div content-id="DSU017TB1" class="table-modal"><div class="table-wrap"><div id="DSU017TB1" data-id="DSU017TB1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Crick's wobble rules for calculating <em>W<sub>i</sub></em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Codon third position </th><th> </th><th>Anticodon first position </th><th><em>W<sub>i</sub></em> </th></tr></thead><tbody><tr><td><em>i</em> </td><td>U </td><td><em>j</em> </td><td>I </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>U</mi><mo>:</mo><mi>I</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub></mrow><mo xmlns="">+</mo><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>U</mi><mo>:</mo><mi>G</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>,</mo><mi>j</mi><mo>+</mo><mn>1</mn></mrow></msub></mrow></math></span> </td></tr><tr><td><em>i</em> + 1 </td><td>C </td><td><em>j</em> + 1 </td><td>G </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>C</mi><mo>:</mo><mi>G</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>+</mo><mn>1</mn><mo>,</mo><mi>j</mi><mo>+</mo><mn>1</mn></mrow></msub></mrow><mo xmlns="">+</mo><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>C</mi><mo>:</mo><mi>I</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>+</mo><mn>1</mn><mo>,</mo><mi>j</mi></mrow></msub></mrow></math></span> </td></tr><tr><td><em>i</em> + 2 </td><td>A </td><td><em>j</em> + 2 </td><td>U </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>A</mi><mo>:</mo><mi>U</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>+</mo><mn>2</mn><mo>,</mo><mi>j</mi><mo>+</mo><mn>2</mn></mrow></msub></mrow><mo xmlns="">+</mo><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>A</mi><mo>:</mo><mi>I</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>+</mo><mn>2</mn><mo>,</mo><mi>j</mi></mrow></msub></mrow></math></span> </td></tr><tr><td><em>i</em> + 3 </td><td>G </td><td><em>j</em> + 3 </td><td>C </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>G</mi><mo>:</mo><mi>C</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>+</mo><mn>3</mn><mo>,</mo><mi>j</mi></mrow></msub></mrow><mo xmlns="">+</mo><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>G</mi><mo>:</mo><mi>U</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>+</mo><mn>2</mn><mo>,</mo><mi>j</mi><mo>+</mo><mn>2</mn></mrow></msub></mrow></math></span> </td></tr></tbody></table></div><div content-id="DSU017TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Codon third position </th><th> </th><th>Anticodon first position </th><th><em>W<sub>i</sub></em> </th></tr></thead><tbody><tr><td><em>i</em> </td><td>U </td><td><em>j</em> </td><td>I </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>U</mi><mo>:</mo><mi>I</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub></mrow><mo xmlns="">+</mo><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>U</mi><mo>:</mo><mi>G</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>,</mo><mi>j</mi><mo>+</mo><mn>1</mn></mrow></msub></mrow></math></span> </td></tr><tr><td><em>i</em> + 1 </td><td>C </td><td><em>j</em> + 1 </td><td>G </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>C</mi><mo>:</mo><mi>G</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>+</mo><mn>1</mn><mo>,</mo><mi>j</mi><mo>+</mo><mn>1</mn></mrow></msub></mrow><mo xmlns="">+</mo><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>C</mi><mo>:</mo><mi>I</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>+</mo><mn>1</mn><mo>,</mo><mi>j</mi></mrow></msub></mrow></math></span> </td></tr><tr><td><em>i</em> + 2 </td><td>A </td><td><em>j</em> + 2 </td><td>U </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>A</mi><mo>:</mo><mi>U</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>+</mo><mn>2</mn><mo>,</mo><mi>j</mi><mo>+</mo><mn>2</mn></mrow></msub></mrow><mo xmlns="">+</mo><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>A</mi><mo>:</mo><mi>I</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>+</mo><mn>2</mn><mo>,</mo><mi>j</mi></mrow></msub></mrow></math></span> </td></tr><tr><td><em>i</em> + 3 </td><td>G </td><td><em>j</em> + 3 </td><td>C </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>G</mi><mo>:</mo><mi>C</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>+</mo><mn>3</mn><mo>,</mo><mi>j</mi></mrow></msub></mrow><mo xmlns="">+</mo><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>G</mi><mo>:</mo><mi>U</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>+</mo><mn>2</mn><mo>,</mo><mi>j</mi><mo>+</mo><mn>2</mn></mrow></msub></mrow></math></span> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p>The <em>W<sub>i</sub></em> values are calculated based on Equation (1). The 64 codons are clustered in the genetic code into 16 groups, each one consists of four codons. The four codons in each group differ only in their third position (the wobble position). The formulas for calculating the <em>W<sub>i</sub></em> values for each of the four codons in the group are given in the table. <em>i</em> denotes the index of the codon in the quartet which ends with U, <em>i</em> + <em>1</em>, <em>i</em> + <em>2</em>, and <em>i</em> + <em>3</em> denote the three other codons which end with bases C, A, and G, respectively. <em>j</em> denotes the index of the tRNA whose anticodon starts with I; all base pairing between the <em>i</em>th codon and the <em>j</em>th anticodon are WC. <em>j</em> + 1, <em>j</em> + 2, and <em>j</em> + 3 denote the three other tRNAs whose anticodons start with bases G, U, and C, respectively. tGCN<em><sub>ij</sub></em> represents the tRNA gene copy number corresponding to the interaction between the <em>i</em>th codon and the <em>j</em>th tRNA. For each codon, <em>W<sub>i</sub></em> sums over all tRNAs that can pair with the codon. For example, the GCU codon which ends with U can either pair with anticodons that start with I (IGC) and generate a standard WC base pairing, or pair with anticodons that start with G (GGC) and generate a wobble base pairing.</p></div></div></div></div></div><div class="table-wrap"><div id="DSU017TB1" data-id="DSU017TB1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Crick's wobble rules for calculating <em>W<sub>i</sub></em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Codon third position </th><th> </th><th>Anticodon first position </th><th><em>W<sub>i</sub></em> </th></tr></thead><tbody><tr><td><em>i</em> </td><td>U </td><td><em>j</em> </td><td>I </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>U</mi><mo>:</mo><mi>I</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub></mrow><mo xmlns="">+</mo><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>U</mi><mo>:</mo><mi>G</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>,</mo><mi>j</mi><mo>+</mo><mn>1</mn></mrow></msub></mrow></math></span> </td></tr><tr><td><em>i</em> + 1 </td><td>C </td><td><em>j</em> + 1 </td><td>G </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>C</mi><mo>:</mo><mi>G</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>+</mo><mn>1</mn><mo>,</mo><mi>j</mi><mo>+</mo><mn>1</mn></mrow></msub></mrow><mo xmlns="">+</mo><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>C</mi><mo>:</mo><mi>I</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>+</mo><mn>1</mn><mo>,</mo><mi>j</mi></mrow></msub></mrow></math></span> </td></tr><tr><td><em>i</em> + 2 </td><td>A </td><td><em>j</em> + 2 </td><td>U </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>A</mi><mo>:</mo><mi>U</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>+</mo><mn>2</mn><mo>,</mo><mi>j</mi><mo>+</mo><mn>2</mn></mrow></msub></mrow><mo xmlns="">+</mo><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>A</mi><mo>:</mo><mi>I</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>+</mo><mn>2</mn><mo>,</mo><mi>j</mi></mrow></msub></mrow></math></span> </td></tr><tr><td><em>i</em> + 3 </td><td>G </td><td><em>j</em> + 3 </td><td>C </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>G</mi><mo>:</mo><mi>C</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>+</mo><mn>3</mn><mo>,</mo><mi>j</mi></mrow></msub></mrow><mo xmlns="">+</mo><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>G</mi><mo>:</mo><mi>U</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>+</mo><mn>2</mn><mo>,</mo><mi>j</mi><mo>+</mo><mn>2</mn></mrow></msub></mrow></math></span> </td></tr></tbody></table></div><div content-id="DSU017TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Codon third position </th><th> </th><th>Anticodon first position </th><th><em>W<sub>i</sub></em> </th></tr></thead><tbody><tr><td><em>i</em> </td><td>U </td><td><em>j</em> </td><td>I </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>U</mi><mo>:</mo><mi>I</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub></mrow><mo xmlns="">+</mo><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>U</mi><mo>:</mo><mi>G</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>,</mo><mi>j</mi><mo>+</mo><mn>1</mn></mrow></msub></mrow></math></span> </td></tr><tr><td><em>i</em> + 1 </td><td>C </td><td><em>j</em> + 1 </td><td>G </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>C</mi><mo>:</mo><mi>G</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>+</mo><mn>1</mn><mo>,</mo><mi>j</mi><mo>+</mo><mn>1</mn></mrow></msub></mrow><mo xmlns="">+</mo><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>C</mi><mo>:</mo><mi>I</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>+</mo><mn>1</mn><mo>,</mo><mi>j</mi></mrow></msub></mrow></math></span> </td></tr><tr><td><em>i</em> + 2 </td><td>A </td><td><em>j</em> + 2 </td><td>U </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>A</mi><mo>:</mo><mi>U</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>+</mo><mn>2</mn><mo>,</mo><mi>j</mi><mo>+</mo><mn>2</mn></mrow></msub></mrow><mo xmlns="">+</mo><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>A</mi><mo>:</mo><mi>I</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>+</mo><mn>2</mn><mo>,</mo><mi>j</mi></mrow></msub></mrow></math></span> </td></tr><tr><td><em>i</em> + 3 </td><td>G </td><td><em>j</em> + 3 </td><td>C </td><td><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>G</mi><mo>:</mo><mi>C</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>+</mo><mn>3</mn><mo>,</mo><mi>j</mi></mrow></msub></mrow><mo xmlns="">+</mo><mo stretchy="false" xmlns="">(</mo><mn xmlns="">1</mn><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>s</mi><mrow><mi>G</mi><mo>:</mo><mi>U</mi></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mtext xmlns="">tGC</mtext><mrow xmlns=""><msub><mtext>N</mtext><mrow><mi>i</mi><mo>+</mo><mn>2</mn><mo>,</mo><mi>j</mi><mo>+</mo><mn>2</mn></mrow></msub></mrow></math></span> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p>The <em>W<sub>i</sub></em> values are calculated based on Equation (1). The 64 codons are clustered in the genetic code into 16 groups, each one consists of four codons. The four codons in each group differ only in their third position (the wobble position). The formulas for calculating the <em>W<sub>i</sub></em> values for each of the four codons in the group are given in the table. <em>i</em> denotes the index of the codon in the quartet which ends with U, <em>i</em> + <em>1</em>, <em>i</em> + <em>2</em>, and <em>i</em> + <em>3</em> denote the three other codons which end with bases C, A, and G, respectively. <em>j</em> denotes the index of the tRNA whose anticodon starts with I; all base pairing between the <em>i</em>th codon and the <em>j</em>th anticodon are WC. <em>j</em> + 1, <em>j</em> + 2, and <em>j</em> + 3 denote the three other tRNAs whose anticodons start with bases G, U, and C, respectively. tGCN<em><sub>ij</sub></em> represents the tRNA gene copy number corresponding to the interaction between the <em>i</em>th codon and the <em>j</em>th tRNA. For each codon, <em>W<sub>i</sub></em> sums over all tRNAs that can pair with the codon. For example, the GCU codon which ends with U can either pair with anticodons that start with I (IGC) and generate a standard WC base pairing, or pair with anticodons that start with G (GGC) and generate a wobble base pairing.</p></div></div></div></div></p><p>In 1966, Crick<sup><a reveal-id="DSU017C37" data-open="DSU017C37" class="link link-ref link-reveal xref-bibr">37</a></sup> suggested that in some cases wobble pairing may occur in the third base of the codon. According to Crick, the pairing at the third codon position has to obey chemical constrains; thus, some of the optional parings will not occur. For example, the unlikely pairing of guanine–adenine is due to a side group of guanine, which cannot make one of its bonds. In addition to the four standard nucleotides, modified nucleosides often occupy the wobble position of the anticodon (usually position 34 of the tRNA). In fact, the wobble position is the most frequently modified nucleoside in tRNA.<sup><a reveal-id="DSU017C38" data-open="DSU017C38" class="link link-ref link-reveal xref-bibr">38</a>,<a reveal-id="DSU017C39" data-open="DSU017C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Inosine, for example, is a common modification of adenine that occurs in the wobble position of many tRNA species.<sup><a reveal-id="DSU017C39" data-open="DSU017C39" class="link link-ref link-reveal xref-bibr">39</a>–<a reveal-id="DSU017C43" data-open="DSU017C43" class="link link-ref link-reveal xref-bibr">43</a></sup></p><p>Out of the eight <em>S<sub>ij</sub></em> weights, four are related to Watson–Crick (WC) interactions and the others are related to wobble interactions. In prokaryotic genomes, tRNA<sup>IIe</sup> has a unique wobble position nucleoside [lysidine (L) in bacteria and agmatidine (agm) in archaea], which recognizes the AUA codon;<sup><a reveal-id="DSU017C44" data-open="DSU017C44" class="link link-ref link-reveal xref-bibr">44</a></sup> thus, the prokaryotic <em>S<sub>ij</sub></em> set contains one additional wobble <em>S<sub>ij</sub></em> weight. The different possible pairs at the wobble position and the current tAI weights of the corresponding interactions are summarized in Table <a reveal-id="DSU017TB2" data-open="DSU017TB2" class="link link-reveal link-table xref-fig">2</a>. <div content-id="DSU017TB2" class="table-modal"><div class="table-wrap"><div id="DSU017TB2" data-id="DSU017TB2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>The different base-pairings</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"><em>j</em> </th><th colspan="6"><em>i</em><hr /> </th></tr><tr><th>I </th><th>G </th><th>U </th><th>C </th><th>A </th><th>L </th></tr></thead><tbody><tr><td>I </td><td>— </td><td>— </td><td><em>0</em> </td><td><strong>0.28</strong> </td><td><strong>0.9999</strong> </td><td>— </td></tr><tr><td>G </td><td>— </td><td>— </td><td><strong>0.41</strong> </td><td><em>0</em> </td><td>— </td><td>— </td></tr><tr><td>U </td><td>— </td><td><strong>0.68</strong> </td><td>— </td><td>— </td><td><em>0</em> </td><td>— </td></tr><tr><td>C </td><td>— </td><td><em>0</em> </td><td>— </td><td>— </td><td>— </td><td>— </td></tr><tr><td>A </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td></tr><tr><td>L </td><td>— </td><td>— </td><td>— </td><td>— </td><td><strong>0.89</strong> </td><td>— </td></tr></tbody></table></div><div content-id="DSU017TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"><em>j</em> </th><th colspan="6"><em>i</em><hr /> </th></tr><tr><th>I </th><th>G </th><th>U </th><th>C </th><th>A </th><th>L </th></tr></thead><tbody><tr><td>I </td><td>— </td><td>— </td><td><em>0</em> </td><td><strong>0.28</strong> </td><td><strong>0.9999</strong> </td><td>— </td></tr><tr><td>G </td><td>— </td><td>— </td><td><strong>0.41</strong> </td><td><em>0</em> </td><td>— </td><td>— </td></tr><tr><td>U </td><td>— </td><td><strong>0.68</strong> </td><td>— </td><td>— </td><td><em>0</em> </td><td>— </td></tr><tr><td>C </td><td>— </td><td><em>0</em> </td><td>— </td><td>— </td><td>— </td><td>— </td></tr><tr><td>A </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td></tr><tr><td>L </td><td>— </td><td>— </td><td>— </td><td>— </td><td><strong>0.89</strong> </td><td>— </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN2"><div class="fn"><p><em>S<sub>ij</sub></em>-<em>values</em> are given to the pairing between the first position of the <em>j</em>th anticodon (tRNA) and the third position of the <em>i</em>th codon. <em>S<sub>ij</sub></em>-values of WC base pairs are shown in italics, wobble values are shown in bold. Interactions which are not included in the calculation of the tAI are marked with hyphens. Lysidine (L) is a bacterial RNA modification of the DNA nucleotide cytidine (c).<sup><a reveal-id="DSU017C44" data-open="DSU017C44" class="link link-ref link-reveal xref-bibr">44</a>,<a reveal-id="DSU017C45" data-open="DSU017C45" class="link link-ref link-reveal xref-bibr">45</a></sup></p></div></div></div></div></div><div class="table-wrap"><div id="DSU017TB2" data-id="DSU017TB2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>The different base-pairings</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"><em>j</em> </th><th colspan="6"><em>i</em><hr /> </th></tr><tr><th>I </th><th>G </th><th>U </th><th>C </th><th>A </th><th>L </th></tr></thead><tbody><tr><td>I </td><td>— </td><td>— </td><td><em>0</em> </td><td><strong>0.28</strong> </td><td><strong>0.9999</strong> </td><td>— </td></tr><tr><td>G </td><td>— </td><td>— </td><td><strong>0.41</strong> </td><td><em>0</em> </td><td>— </td><td>— </td></tr><tr><td>U </td><td>— </td><td><strong>0.68</strong> </td><td>— </td><td>— </td><td><em>0</em> </td><td>— </td></tr><tr><td>C </td><td>— </td><td><em>0</em> </td><td>— </td><td>— </td><td>— </td><td>— </td></tr><tr><td>A </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td></tr><tr><td>L </td><td>— </td><td>— </td><td>— </td><td>— </td><td><strong>0.89</strong> </td><td>— </td></tr></tbody></table></div><div content-id="DSU017TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"><em>j</em> </th><th colspan="6"><em>i</em><hr /> </th></tr><tr><th>I </th><th>G </th><th>U </th><th>C </th><th>A </th><th>L </th></tr></thead><tbody><tr><td>I </td><td>— </td><td>— </td><td><em>0</em> </td><td><strong>0.28</strong> </td><td><strong>0.9999</strong> </td><td>— </td></tr><tr><td>G </td><td>— </td><td>— </td><td><strong>0.41</strong> </td><td><em>0</em> </td><td>— </td><td>— </td></tr><tr><td>U </td><td>— </td><td><strong>0.68</strong> </td><td>— </td><td>— </td><td><em>0</em> </td><td>— </td></tr><tr><td>C </td><td>— </td><td><em>0</em> </td><td>— </td><td>— </td><td>— </td><td>— </td></tr><tr><td>A </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td><td>— </td></tr><tr><td>L </td><td>— </td><td>— </td><td>— </td><td>— </td><td><strong>0.89</strong> </td><td>— </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN2"><div class="fn"><p><em>S<sub>ij</sub></em>-<em>values</em> are given to the pairing between the first position of the <em>j</em>th anticodon (tRNA) and the third position of the <em>i</em>th codon. <em>S<sub>ij</sub></em>-values of WC base pairs are shown in italics, wobble values are shown in bold. Interactions which are not included in the calculation of the tAI are marked with hyphens. Lysidine (L) is a bacterial RNA modification of the DNA nucleotide cytidine (c).<sup><a reveal-id="DSU017C44" data-open="DSU017C44" class="link link-ref link-reveal xref-bibr">44</a>,<a reveal-id="DSU017C45" data-open="DSU017C45" class="link link-ref link-reveal xref-bibr">45</a></sup></p></div></div></div></div></p><p>In the tAI, WC <em>S<sub>ij</sub></em> weights are fixed to zero (perfect interactions) under the assumption of no constraint on these interactions. The wobble interaction weights are inferred by optimizing the correlation between gene expression levels (mRNA levels) and their corresponding tAI in <em>Saccharomyces cerevisiae</em>;<sup><a reveal-id="DSU017C8" data-open="DSU017C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSU017C46" data-open="DSU017C46" class="link link-ref link-reveal xref-bibr">46</a></sup> the rationale behind this optimization is based on the following relations (which hold in many organisms): (i) there is correlation between mRNA levels and protein levels; (ii) there is correlation between translation rate and protein levels; and (iii) highly translated genes are under selection to include codons with higher adaptation to the tRNA pool.</p><p>The possibility of having different wobble interaction weights across different genomes has not yet been comprehensively studied. Here, we develop a novel generic approach for species-specific estimation of the tAI <em>S<sub>ij</sub></em> weights without the need of gene expression measurements; for convenience, we name the new measure: species-specific tAI (stAI). This measure includes different <em>S<sub>ij</sub></em> weights for each organism. We show that the correlation between protein levels and stAI is higher than that between protein levels and tAI.</p><p>Based on our approach, we infer the wobble <em>S<sub>ij</sub></em> weights for a wide variety of organisms from the three domains of life, in order to examine the conjecture that organisms from different domains have significantly different <em>S<sub>ij</sub></em> weights and to understand these differences.</p>                            <h2 scrollto-destination=87197618 id="87197618" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87197621 id="87197621" class="section-title" data-legacy-id=s2a>2.1. Computing the S<sub>ij</sub> weights of the stAI without the need of gene expression measurements</h3>
<p>The tAI weights are based on optimizing the correlation between tAI (Equation 1) and expression levels in <em>S. cerevisiae</em> and <em>E. coli.</em><sup><a reveal-id="DSU017C8" data-open="DSU017C8" class="link link-ref link-reveal xref-bibr">8</a></sup> However, large scale measurement of mRNA levels and specifically protein abundance (PA) are not available for most of the organisms with sequenced genomes.</p><p>To solve this problem we develop an approach that is based on the assumption that highly expressed genes should have both higher adaptation to the tRNA pool (i.e. higher tAI) and higher CUB (i.e. less uniform distribution of codons).<sup><a reveal-id="DSU017C8" data-open="DSU017C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Thus, there should be a significant correlation between CUB and tAI. Based on this assumption, we find the <em>S<sub>ij</sub></em>-values that optimize the correlation between CUB and stAI. Note that the optimized correlation is at the level of genes while for each gene both measures are based on its codons content. Below we provide additional details about our approach including the CUB measure that we use.</p>                            <h3 scrollto-destination=87197627 id="87197627" class="section-title" data-legacy-id=s2b>2.2. Relative codon bias</h3>
<div class="block-child-p">In order to infer the <em>S<sub>ij</sub></em> weights without the need of expression levels, we used a measure of CUB, which is based solely on the coding sequence. The strength of relative codon bias (RCBS) proposed by Roymondal <em>et al.</em><sup><a reveal-id="DSU017C47" data-open="DSU017C47" class="link link-ref link-reveal xref-bibr">47</a></sup> is an example of an index that is based only on the sequence. The RCBS of codon <em>xyz</em> is expressed as: <a id="DSU017M2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU017M2"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>d</mi><mrow><mi>x</mi><mi>y</mi><mi>z</mi><mspace width="thinmathspace"></mspace></mrow></msub></mrow><mo xmlns="">=</mo><mspace width="thinmathspace" xmlns=""></mspace><mspace width="thinmathspace" xmlns=""></mspace><mstyle xmlns=""><mrow><mfrac><mrow><mspace width="thinmathspace"></mspace><mi>f</mi><mo stretchy="false">(</mo><mi>x</mi><mo>,</mo><mi>y</mi><mo>,</mo><mi>z</mi><mo stretchy="false">)</mo><mspace width="thinmathspace"></mspace><mo>−</mo><mspace width="thinmathspace"></mspace><mrow><mspace width="thinmathspace"></mspace><msub><mi>f</mi><mn>1</mn></msub></mrow><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo><mo>⋅</mo><mrow><mspace width="thinmathspace"></mspace><msub><mi>f</mi><mn>2</mn></msub></mrow><mo stretchy="false">(</mo><mi>y</mi><mo stretchy="false">)</mo><mo>⋅</mo><mrow><mspace width="thinmathspace"></mspace><msub><mi>f</mi><mn>3</mn></msub></mrow><mo stretchy="false">(</mo><mi>z</mi><mo stretchy="false">)</mo></mrow><mrow><mrow><mspace width="thinmathspace"></mspace><msub><mi>f</mi><mn>1</mn></msub></mrow><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo><mo>⋅</mo><mrow><mspace width="thinmathspace"></mspace><msub><mi>f</mi><mn>2</mn></msub></mrow><mo stretchy="false">(</mo><mi>y</mi><mo stretchy="false">)</mo><mo>⋅</mo><mrow><mspace width="thinmathspace"></mspace><msub><mi>f</mi><mn>3</mn></msub></mrow><mo stretchy="false">(</mo><mi>z</mi><mo stretchy="false">)</mo></mrow></mfrac></mrow></mstyle></math></div></div><div class="label">(2)</div></div></div><div class="block-child-p">where <em>f</em>(<em>x</em>, <em>y</em>, <em>z</em>) is the observed frequency of codon <em>xyz</em> (where <em>x</em>, <em>y</em>, <em>z</em> denote the first/second/third nucleotides, respectively, of the codon) and <em>f</em><sub>1</sub>(<em>x</em>), <em>f</em><sub>2</sub>(<em>y</em>), and <em>f</em><sub>3</sub>(<em>z</em>) are the observed frequencies of bases <em>x</em>, <em>y</em>, and <em>z</em> at, respectively, positions 1, 2, and 3 of the codon. These frequencies are computed for each gene separately. The RCBS of a gene of length <em>L</em>, in codons, is calculated as: <a id="DSU017M3" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU017M3"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtext xmlns="">RCBS</mtext><mspace width="thinmathspace" xmlns=""></mspace><mo xmlns="">=</mo><mspace width="thinmathspace" xmlns=""></mspace><mrow xmlns=""><msup><mrow><mo>(</mo><mrow><munderover><mo movablelimits="false">∏</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>L</mi></munderover><mrow><mo stretchy="false">(</mo><mn>1</mn><mo>+</mo><msub><mrow><msup><mi>d</mi><mi>i</mi></msup></mrow><mrow><mi>x</mi><mi>y</mi><mi>z</mi></mrow></msub><mo stretchy="false">)</mo></mrow></mrow><mo>)</mo></mrow><mrow><mn>1</mn><mrow><mo>/</mo></mrow><mi>L</mi></mrow></msup></mrow><mo xmlns="">−</mo><mn xmlns="">1</mn></math></div></div><div class="label">(3)</div></div></div><p>RCBS takes into account base compositional bias, to get a more reliable measure of highly favoured codon frequency while controlling for other features of the coding sequence such as GC content bias.</p><p>According to Equation (2), rare codons will be given lower <em>d<sub>xyz</sub></em> (i.e. a value close to −1) while a very frequent codon will be given a higher <em>d<sub>xyz</sub></em> value (e.g. it can be 1). Thus, very rare codons decrease the final RCBS score of the gene and very frequent ones increase its final RCBS score (see Equation 3). However, we believe that (almost by definition) genes with very high CUB should include both very frequent codons and very rare codons. For example, if a hypothetical amino acid A has two codons, one is ‘optimal’, and the second is ‘not optimal’, we expect a very highly expressed codon usage biased gene to have a very high <em>d<sub>xyz</sub></em> score for the first one and a very low <em>d<sub>xyz</sub></em> score for the second one. But, we wish that both cases/codons will contribute to the same direction and increase the RCBS score.</p><div class="block-child-p">Thus, we employ a modified version of the RCBS, which we term here directional codon bias score (DCBS), as in this measure, both positive and negative codon usage biases contribute (in the same direction) to the total CUB of the gene. We define the directional codon bias (DCB) of a codon triplet <em>xyz</em> as: <a id="DSU017M4" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU017M4"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>d</mi><mrow><mi>x</mi><mi>y</mi><mi>z</mi><mspace width="thinmathspace"></mspace></mrow></msub></mrow><mo xmlns="">=</mo><mspace width="thinmathspace" xmlns=""></mspace><mspace width="thinmathspace" xmlns=""></mspace><mo movablelimits="true" xmlns="">max</mo><mrow xmlns=""><mo>(</mo><mrow><mstyle><mrow><mfrac><mrow><mspace width="thinmathspace"></mspace><mi>f</mi><mo stretchy="false">(</mo><mi>x</mi><mo>,</mo><mi>y</mi><mo>,</mo><mi>z</mi><mo stretchy="false">)</mo><mspace width="thinmathspace"></mspace></mrow><mrow><mrow><mspace width="thinmathspace"></mspace><msub><mi>f</mi><mn>1</mn></msub></mrow><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo><mo>⋅</mo><mrow><mspace width="thinmathspace"></mspace><msub><mi>f</mi><mn>2</mn></msub></mrow><mo stretchy="false">(</mo><mi>y</mi><mo stretchy="false">)</mo><mo>⋅</mo><mrow><mspace width="thinmathspace"></mspace><msub><mi>f</mi><mn>3</mn></msub></mrow><mo stretchy="false">(</mo><mi>z</mi><mo stretchy="false">)</mo></mrow></mfrac></mrow><mo>,</mo><mstyle><mrow><mfrac><mrow><mrow><mspace width="thinmathspace"></mspace><msub><mi>f</mi><mn>1</mn></msub></mrow><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo><mo>⋅</mo><mrow><mspace width="thinmathspace"></mspace><msub><mi>f</mi><mn>2</mn></msub></mrow><mo stretchy="false">(</mo><mi>y</mi><mo stretchy="false">)</mo><mo>⋅</mo><mrow><mspace width="thinmathspace"></mspace><msub><mi>f</mi><mn>3</mn></msub></mrow><mo stretchy="false">(</mo><mi>z</mi><mo stretchy="false">)</mo></mrow><mrow><mspace width="thinmathspace"></mspace><mi>f</mi><mo stretchy="false">(</mo><mi>x</mi><mo>,</mo><mi>y</mi><mo>,</mo><mi>z</mi><mo stretchy="false">)</mo></mrow></mfrac></mrow></mstyle></mstyle></mrow><mo>)</mo></mrow></math></div></div><div class="label">(4)</div></div></div><div class="block-child-p">The DCBS of a gene of length <em>L</em>, in codons, is the following mean (see example in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/dsu017_Supplementary_Data.zip?Expires=2147483647&Signature=oVKv7qorjDac3xeIxZvVwbsJ-qcQdF~HYxNmw~i8D146Xv1Qcmgq-QO5Xyp4rkP~pBak4jTqiqgW5eB1jlrOFDUnSx8Yy3e4tWBtbqsO9MW3bdi~0IwtKzN4kt61Kv3Ss6pyKGlDqjN10WzQPryart8qWqyxiN~OomqZF8tD-d7cS1~XpgDbefils0tEkiTFvHG1Yq3NtEV~g1ymvfW7G4M7X3j5Nvtt3bCc9KRPQK9nVopAY35nPCgW2Q6aHHWhWwRK0eDp76GTBZ9nO5~JEidfjT5qEA7PE-zAWUSy95GP0GpiNeUdA8t7VZksBhnr7DXFXHvHeZlj5N3VJWIFWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>): <a id="DSU017M5" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU017M5"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtext xmlns="">DCBS</mtext><mspace width="thinmathspace" xmlns=""></mspace><mo xmlns="">=</mo><mspace width="thinmathspace" xmlns=""></mspace><mstyle xmlns=""><mrow><mfrac><mrow><msubsup><mo movablelimits="false">∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>L</mi></msubsup><mrow><mrow><msub><mi>d</mi><mrow><mi>x</mi><mi>y</mi><mrow><msub><mi>z</mi><mrow></mrow></msub></mrow></mrow></msub></mrow></mrow></mrow><mi>L</mi></mfrac></mrow></mstyle></math></div></div><div class="label">(5)</div></div></div><p>As we later demonstrate, in our framework the DCBS gives better results than the RCBS.</p><p>Finally, it is important to emphasize the fact that both RCBS and DCBS control for mutation bias. Specifically, when we compute the DCBS (see above), the frequency of each codon [<span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">f</mi><mo stretchy="false" xmlns="">(</mo><mi xmlns="">x</mi><mo xmlns="">,</mo><mi xmlns="">y</mi><mo xmlns="">,</mo><mi xmlns="">z</mi><mo stretchy="false" xmlns="">)</mo></math></span>] is normalized by the expected frequency under the assumption that the different nucleotides are independent [<span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mspace width="thinmathspace"></mspace><msub><mi>f</mi><mn>1</mn></msub></mrow><mo stretchy="false" xmlns="">(</mo><mi xmlns="">x</mi><mo stretchy="false" xmlns="">)</mo><mo xmlns="">⋅</mo><mrow xmlns=""><mspace width="thinmathspace"></mspace><msub><mi>f</mi><mn>2</mn></msub></mrow><mo stretchy="false" xmlns="">(</mo><mi xmlns="">y</mi><mo stretchy="false" xmlns="">)</mo><mo xmlns="">⋅</mo><mrow xmlns=""><mspace width="thinmathspace"></mspace><msub><mi>f</mi><mn>3</mn></msub></mrow><mo stretchy="false" xmlns="">(</mo><mi xmlns="">z</mi><mo stretchy="false" xmlns="">)</mo></math></span>]; the later represents among others the mutation bias. The measure that we use is based on the frequency of the codon normalized by the expected frequency according to the mutation bias, and thus control for mutational bias (see also <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/dsu017_Supplementary_Data.zip?Expires=2147483647&Signature=oVKv7qorjDac3xeIxZvVwbsJ-qcQdF~HYxNmw~i8D146Xv1Qcmgq-QO5Xyp4rkP~pBak4jTqiqgW5eB1jlrOFDUnSx8Yy3e4tWBtbqsO9MW3bdi~0IwtKzN4kt61Kv3Ss6pyKGlDqjN10WzQPryart8qWqyxiN~OomqZF8tD-d7cS1~XpgDbefils0tEkiTFvHG1Yq3NtEV~g1ymvfW7G4M7X3j5Nvtt3bCc9KRPQK9nVopAY35nPCgW2Q6aHHWhWwRK0eDp76GTBZ9nO5~JEidfjT5qEA7PE-zAWUSy95GP0GpiNeUdA8t7VZksBhnr7DXFXHvHeZlj5N3VJWIFWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> regarding the way our approach controls for possible factors affecting CUB).</p>                            <h3 scrollto-destination=87197646 id="87197646" class="section-title" data-legacy-id=s2c>2.3. Inferring the parameters of the stAI</h3>
<p>The stAI inferred here is based on the same equations of the tAI with an organism-specific <em>S<sub>ij</sub>-</em>values' set (Equation 1), which is based on a measure of CUB. For every genome used in this study, the unique <em>S<sub>ij</sub></em> set was inferred by optimizing the non-parametric (Spearman) correlation between DCBS (Equations 4 and 5) and stAI (Equation 1). To avoid convergence to local maxima point, we used various starting points. Specifically, we included in the set of starting points the original weights of the tAI<sup><a reveal-id="DSU017C8" data-open="DSU017C8" class="link link-ref link-reveal xref-bibr">8</a></sup> and also the two extreme values of these weights (all zeros and all ones)<em>.</em> In order to choose a set of starting points, we halved the allowed region of the <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>S</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></math></span> values (i.e. the region: <em>S<sub>ij</sub></em> between 0 and 0.5, and the region: <em>S<sub>ij</sub></em> between 0.5 and 1) and considered all combinations for sampling values from these two regions (2<sup><a reveal-id="DSU017C4" data-open="DSU017C4" class="link link-ref link-reveal xref-bibr">4</a></sup> possibilities for the four eukaryotic wobble <em>S<sub>ij</sub></em> and 2<sup><a reveal-id="DSU017C5" data-open="DSU017C5" class="link link-ref link-reveal xref-bibr">5</a></sup> for the five prokaryotic wobble <em>S<sub>ij</sub></em>); thus, organisms from the same domain shared the same set of starting points. For each specific starting point, we used a hill climbing search method to iteratively optimize the <em>S<sub>ij</sub></em> weights using a variable step size (starting with an initial step size of 0.3 and finishing with step size of 0.001). At each step size, when a new optimum was not found, the step size was decreased by a factor of 1.35. Iteration of the hill climbing included a random choice of <em>S<sub>ij</sub></em> elements to change and a direction (i.e. increasing and decreasing) that increases the correlation between stAI and DCBS. The final chosen set of <em>S<sub>ij</sub></em> was the one exhibited the maximum correlation between the stAI and DCBS. In order to determine whether the chosen set of starting points constituted a sufficient sample of the search space for the algorithm convergence, we added 100 more random starting points. The additional points provided no significant change in the final correlation between stAI and DCBS.</p>                            <h3 scrollto-destination=87197651 id="87197651" class="section-title" data-legacy-id=s2d>2.4. Comparison of the hill climbing method to Nedler–Mead search method</h3>
<p>The Nedler–Mead (NM) optimization<sup><a reveal-id="DSU017C48" data-open="DSU017C48" class="link link-ref link-reveal xref-bibr">48</a></sup> is the search method used to infer the <em>S<sub>ij</sub></em>-values of the original tAI<em>.</em><sup><a reveal-id="DSU017C8" data-open="DSU017C8" class="link link-ref link-reveal xref-bibr">8</a></sup> When considering similar set of initiation points, our heuristic search outperformed the NM in finding the maxima of the objective function (i.e. the correlation between stAI and DCBS) in six of the eight model organisms (and was quite similar in the other two). We do not claim that hill climbing is better than NM; however, in the case of the specific problem analysed here (where the hill climbing explores the search space very well), and when considering the Matlab implementation of NM, the hill climbing was a bit better.</p>                            <h3 scrollto-destination=87197654 id="87197654" class="section-title" data-legacy-id=s2e>2.5. The analysed organisms</h3>
<p>Our analysis included 100 different organisms (archaea, bacteria, and eukarya), in which CUB was correlated with the amount of adaptation to the tRNA pool. The correlation between stAI and DCBS/RCBS determined whether or not a tested genome would participate in the analysis. We excluded organisms in which an insignificant positive correlation or a significant negative correlation was observed; in such organisms, the assumptions that connect stAI to CUB do not hold and thus our method cannot be implemented. A detailed list of the excluded organisms is provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/dsu017_Supplementary_Data.zip?Expires=2147483647&Signature=oVKv7qorjDac3xeIxZvVwbsJ-qcQdF~HYxNmw~i8D146Xv1Qcmgq-QO5Xyp4rkP~pBak4jTqiqgW5eB1jlrOFDUnSx8Yy3e4tWBtbqsO9MW3bdi~0IwtKzN4kt61Kv3Ss6pyKGlDqjN10WzQPryart8qWqyxiN~OomqZF8tD-d7cS1~XpgDbefils0tEkiTFvHG1Yq3NtEV~g1ymvfW7G4M7X3j5Nvtt3bCc9KRPQK9nVopAY35nPCgW2Q6aHHWhWwRK0eDp76GTBZ9nO5~JEidfjT5qEA7PE-zAWUSy95GP0GpiNeUdA8t7VZksBhnr7DXFXHvHeZlj5N3VJWIFWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>.</p>                            <h3 scrollto-destination=87197657 id="87197657" class="section-title" data-legacy-id=s2f>2.6. Generating randomized genes sequences</h3>
<p>Random sequences were generated according to the real genomic codon distribution. For each of the 100 genomes studied in this work, 20 randomizations were performed by randomly drawing codons from the calculated genomic distribution and maintaining the protein content of the original genome.</p>                            <h3 scrollto-destination=87197663 id="87197663" class="section-title" data-legacy-id=s2g>2.7. Genomic sequences</h3>
<p>In addition to the model organisms, which were chosen due to their available proteins measurements, we selected the genomes according to the list from ref.<sup><a reveal-id="DSU017C49" data-open="DSU017C49" class="link link-ref link-reveal xref-bibr">49</a></sup>, while trying to build relatively balanced group size wise (thus since bacteria was significantly larger than other groups in the list, we only included organisms of the three major phylums provided there: <em>Cyanobacteria</em>, <em>Alphprobacteria</em>, <em>Gamma-probacteria</em>).</p><p>Coding sequences of all 100 species were retrieved from the NCBI (<a class="link link-uri" href="http://ftp://ftp.ncbi.nih.gov/genomes/">ftp://ftp.ncbi.nih.gov/genomes/</a>). Genomic tRNA copy numbers of all species except <em>Aspergillus nidulans</em>, <em>Debaryomyces hanasenii</em>, and <em>Candida albicans</em> were obtained from the Genomic tRNA Database (<a class="link link-uri" href="http://gtrnadb.ucsc.edu/">http://gtrnadb.ucsc.edu/</a>). For <em>A. nidulans</em>, <em>D. hanasenii</em>, and <em>C. albicans</em>, we used the tRNA copy number as reported in ref.<sup><a reveal-id="DSU017C30" data-open="DSU017C30" class="link link-ref link-reveal xref-bibr">30</a></sup>. A detailed list of all organisms analysed here is provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/dsu017_Supplementary_Data.zip?Expires=2147483647&Signature=oVKv7qorjDac3xeIxZvVwbsJ-qcQdF~HYxNmw~i8D146Xv1Qcmgq-QO5Xyp4rkP~pBak4jTqiqgW5eB1jlrOFDUnSx8Yy3e4tWBtbqsO9MW3bdi~0IwtKzN4kt61Kv3Ss6pyKGlDqjN10WzQPryart8qWqyxiN~OomqZF8tD-d7cS1~XpgDbefils0tEkiTFvHG1Yq3NtEV~g1ymvfW7G4M7X3j5Nvtt3bCc9KRPQK9nVopAY35nPCgW2Q6aHHWhWwRK0eDp76GTBZ9nO5~JEidfjT5qEA7PE-zAWUSy95GP0GpiNeUdA8t7VZksBhnr7DXFXHvHeZlj5N3VJWIFWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>.</p>                            <h3 scrollto-destination=87197669 id="87197669" class="section-title" data-legacy-id=s2h>2.8. Protein abundance</h3>
<p>Large scale protein abundance (PA) measurements of <em>S. cerevisiae</em>, <em>E. coli</em>, <em>Arabidopsis thaliana</em>, <em>Shigella dysentariae</em>, <em>Caenorhabditis elegans</em>, <em>Drosophila melanogaster</em>, and <em>Leptospira interrogans</em> were retrieved from paxdB (<a class="link link-uri" href="http://pax-db.org/#!home">http://pax-db.org/#!home</a>). For <em>S. cerevisiae</em>, <em>E. coli</em>, <em>S. dysentariae</em>, and <em>L. interrogans</em>, a few datasets were provided. In this case, a weighted average between the different PA values was taken (i.e. we averaged the datasets after normalizing each of them such that they have identical average)<em>. Schizosaccharomyces pombe</em> expression levels were obtained from ref.<sup><a reveal-id="DSU017C50" data-open="DSU017C50" class="link link-ref link-reveal xref-bibr">50</a></sup>. The protein levels of some of the multiple cellular organisms were based on analysis of multiple tissues (<em>A. thaliana</em>, <em>D. melanogaster</em>, and <em>C. elegans</em>) (see details in <a class="link link-uri" href="http://pax-db.org/#!home">http://pax-db.org/#!home</a>). Specifically, we analysed all protein levels data that were available in paxdB (<a class="link link-uri" href="http://pax-db.org/#!home">http://pax-db.org/#!home</a>) on 2012. Note that in mammals it has been shown that the tRNA levels in various tissues tend to be correlative (the ranking of the tRNA genes abundance remains similar while the average value might change<sup><a reveal-id="DSU017C51" data-open="DSU017C51" class="link link-ref link-reveal xref-bibr">51</a></sup>); this is probably the case in many other organisms.</p>                            <h3 scrollto-destination=87197671 id="87197671" class="section-title" data-legacy-id=s2i>2.9. Permutation test for comparing two S<sub>ij</sub> means</h3>
<p>An empirical <em>P</em>-value was computed to test the null hypothesis that the means of two <em>S<sub>ij</sub></em> distributions do not significantly differ between two groups of organisms; let <em>n</em> and <em>m</em> denote the number of organisms in the two groups, respectively. For each <em>S<sub>ij</sub></em> component of the weights vector, we performed the following steps: first, we defined the normalized distance between the <em>S<sub>ij</sub></em> means in the two groups of organisms as the absolute difference between the means divided by the sum of the two corresponding standard deviations (SDs). Secondly, we permute the <em>S<sub>ij</sub></em> elements of the two groups by randomly drawing <em>n</em> values as the first group and <em>m</em> (non-overlapping) values as the second group. The random permutations were performed 100 times, each time the distance between the two random groups was computed. Finally, the <em>P</em>-value was defined as the number of times the random distance was higher or equal to the original distance divided by 100.</p>                            <h3 scrollto-destination=87197676 id="87197676" class="section-title" data-legacy-id=s2j>2.10. Spearman correlation as a measure to guide the optimization</h3>
<p>The main advantage of this measure is the fact that it is a non-parametric measure that captures any monotonic relationship between CUB and stAI. Since this measure has been successfully employed in many papers in the field in this context,<sup><a reveal-id="DSU017C18" data-open="DSU017C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSU017C52" data-open="DSU017C52" class="link link-ref link-reveal xref-bibr">52</a>,<a reveal-id="DSU017C53" data-open="DSU017C53" class="link link-ref link-reveal xref-bibr">53</a></sup> we decided to use it also here.</p>                            <h3 scrollto-destination=87197679 id="87197679" class="section-title" data-legacy-id=s2k>2.11. The general rational related to evaluating the stAI and demonstrating that stAI outperforms tAI</h3>
<p>In this section, we would like to explain and emphasize the rational related to the analyses reported in this study. First, as mentioned in the section Introduction, CUB measurements such as the CAI quantify different gene expression aspects than the tAI. Here, we aim at improving the tAI (and not the CUB indices such as the CAI) and thus, our major baseline for stAI evaluations is the tAI (and not the CUB indices such as the CAI). Secondly, we use the correlations with PA as an indirect way to evaluate the stAI: we expect that genes with higher translation efficiency will have higher PA; we also expect that a better measure related to the adaptation to the tRNA pool will have higher correlation with translation efficiency; thus, we expect that a better measure related to the adaptation to the tRNA pool will have higher correlation with PA. It is clear that there can be CUB-based measurements with higher correlation with PA than stAI (see, for example,<sup><a reveal-id="DSU017C54" data-open="DSU017C54" class="link link-ref link-reveal xref-bibr">54</a></sup>)—however, as mentioned, the aim of this study is not to infer PA predictor but to improve the inference of the tAI parameters.</p>                            <h2 scrollto-destination=87197682 id="87197682" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87197685 id="87197685" class="section-title" data-legacy-id=s3a>3.1. The correlation between the CUB and tRNA pool varies among different organisms</h3>
<p>A correlation between CUB and stAI is expected; however, the strength of this correlation among different organisms can teach us about the evolutionary forces shaping their genomes.</p><p>The correlations between stAI and DCBS obtained in the algorithm vary from a lowest value of 0.1136 (for the archaea <em>Halomicrobium mukohataei</em>) to a highest correlation of 0.7626 (for the fungi <em>Yarrowia lipolitica</em>). The bottom 10% correlations were obtained in prokaryotic genomes (the four archaea: <em>H. mukohataei</em>, <em>Archaeoglobus fulgidus</em>, <em>Pyrobaculum aerophilum</em>, and <em>Metallosphaera sedula</em>; and the six bacteria: <em>Anabaena variabilis</em>, <em>Brucella suis</em>, <em>Gloeobacter violaceus</em>, <em>Prochlorococcus marinus</em> MIT9313, <em>Synechococcus elongates</em>, and <em>Trichodesmium erythraeum</em>); thus, in this organisms, selection for CUB is presumably either weak or/and not strongly related to translation elongation and the tRNA pool.</p><p>The top 10% of the correlations were obtained mainly in eukaryotic genomes (the eight fungi: <em>C. albicans</em>, <em>C. glabrata</em>, <em>Eremothecium gossypii</em>, <em>Saccharomyces bayanus</em>, <em>S. mikatae</em>, <em>S. paradoxus</em>, <em>Cryptococcus neoformans</em>, and <em>Y. lipolitica</em>; and the two bacteria: <em>E. coli</em> and <em>Pasteurella multocida</em>); in these organisms, the selection for CUB is probably strongly related to the tRNA pool and translation elongations. All correlations are reported in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/dsu017_Supplementary_Data.zip?Expires=2147483647&Signature=oVKv7qorjDac3xeIxZvVwbsJ-qcQdF~HYxNmw~i8D146Xv1Qcmgq-QO5Xyp4rkP~pBak4jTqiqgW5eB1jlrOFDUnSx8Yy3e4tWBtbqsO9MW3bdi~0IwtKzN4kt61Kv3Ss6pyKGlDqjN10WzQPryart8qWqyxiN~OomqZF8tD-d7cS1~XpgDbefils0tEkiTFvHG1Yq3NtEV~g1ymvfW7G4M7X3j5Nvtt3bCc9KRPQK9nVopAY35nPCgW2Q6aHHWhWwRK0eDp76GTBZ9nO5~JEidfjT5qEA7PE-zAWUSy95GP0GpiNeUdA8t7VZksBhnr7DXFXHvHeZlj5N3VJWIFWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>.</p>                            <h3 scrollto-destination=87197691 id="87197691" class="section-title" data-legacy-id=s3b>3.2. The stAI exhibits better PA predictions than the tAI in non-fungal organisms</h3>
<div class="block-child-p">The correlations between stAI and PA are presented in Fig. <a reveal-id="DSU017F1" data-open="DSU017F1" class="link link-reveal link-table xref-fig">1</a>. All eight models showed significant correlations. In six of the eight organisms, the correlation between stAI and PA was higher than that between tAI and PA. This result (Table <a reveal-id="DSU017TB3" data-open="DSU017TB3" class="link link-reveal link-table xref-fig">3</a>) indicates that stAI outperforms the current tAI as a predictor of PA in all non-fungal organisms. For the two fungi used here (<em>S. cerevisiae</em> and <em>S. pombe</em>), the original tAI predicted PA better than the stAI<em>.</em> This result is not surprising since the <em>S<sub>ij</sub></em>-values in the tAI were inferred based on the optimization of the correlation between tAI and <em>S. cerevisiae</em> mRNA expression levels<sup><a reveal-id="DSU017C8" data-open="DSU017C8" class="link link-ref link-reveal xref-bibr">8</a></sup> (which strongly correlates with PA in <em>S. cerevisiae</em>; Spearman correlation of 0.74, <em>P</em>&lt; 0.0001<sup><a reveal-id="DSU017C55" data-open="DSU017C55" class="link link-ref link-reveal xref-bibr">55</a></sup>); on the other hand, stAI is based on CUB, which is a less accurate measure of protein levels. However, for most of the sequenced genomes exist to date, expression levels are not available; thus, the stAI is valuable. <div content-id="DSU017TB3" class="table-modal"><div class="table-wrap"><div id="DSU017TB3" data-id="DSU017TB3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>Spearman rank correlation of the original tAI and the stAI with PA</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Number of genes </th><th>Number of proteins </th><th><em>r</em> (tAI, PA) </th><th><em>r</em> (stAI, PA) </th><th>Change (%) </th></tr></thead><tbody><tr><td colspan="6">Non-fungal </td></tr><tr><td><em> E. coli</em> </td><td>4,145 </td><td>688 </td><td>0.5032 </td><td>0.5493 </td><td>+8.39 </td></tr><tr><td><em> S. dysentariae</em> </td><td>4,501 </td><td>1,266 </td><td>0.3574 </td><td>0.36757 </td><td>+2.76 </td></tr><tr><td><em> L. interrogans</em> </td><td>3,667 </td><td>2,114 </td><td>0.0959 </td><td>0.19408 </td><td>+50.58 </td></tr><tr><td><em> A. thaliana</em> </td><td>28,163 </td><td>8,478 </td><td>0.3328 </td><td>0.3762 </td><td>+11.53 </td></tr><tr><td><em> C. elegans</em> </td><td>22,830 </td><td>6,959 </td><td>0.0919 </td><td>0.0956 </td><td>+3.87 </td></tr><tr><td><em> D. melanogaster</em> </td><td>10,926 </td><td>6,510 </td><td>0.4878 </td><td>0.5001 </td><td>+2.46 </td></tr><tr><td colspan="6">Fungi </td></tr><tr><td><em> S. cerevisiae</em> </td><td>5,869 </td><td>2,666 </td><td>0.6915 </td><td>0.5802 </td><td>−19.18 </td></tr><tr><td><em> S. pombe</em> </td><td>5,017 </td><td>1,464 </td><td>0.6554 </td><td>0.56715 </td><td>−15.58 </td></tr></tbody></table></div><div content-id="DSU017TB3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Number of genes </th><th>Number of proteins </th><th><em>r</em> (tAI, PA) </th><th><em>r</em> (stAI, PA) </th><th>Change (%) </th></tr></thead><tbody><tr><td colspan="6">Non-fungal </td></tr><tr><td><em> E. coli</em> </td><td>4,145 </td><td>688 </td><td>0.5032 </td><td>0.5493 </td><td>+8.39 </td></tr><tr><td><em> S. dysentariae</em> </td><td>4,501 </td><td>1,266 </td><td>0.3574 </td><td>0.36757 </td><td>+2.76 </td></tr><tr><td><em> L. interrogans</em> </td><td>3,667 </td><td>2,114 </td><td>0.0959 </td><td>0.19408 </td><td>+50.58 </td></tr><tr><td><em> A. thaliana</em> </td><td>28,163 </td><td>8,478 </td><td>0.3328 </td><td>0.3762 </td><td>+11.53 </td></tr><tr><td><em> C. elegans</em> </td><td>22,830 </td><td>6,959 </td><td>0.0919 </td><td>0.0956 </td><td>+3.87 </td></tr><tr><td><em> D. melanogaster</em> </td><td>10,926 </td><td>6,510 </td><td>0.4878 </td><td>0.5001 </td><td>+2.46 </td></tr><tr><td colspan="6">Fungi </td></tr><tr><td><em> S. cerevisiae</em> </td><td>5,869 </td><td>2,666 </td><td>0.6915 </td><td>0.5802 </td><td>−19.18 </td></tr><tr><td><em> S. pombe</em> </td><td>5,017 </td><td>1,464 </td><td>0.6554 </td><td>0.56715 </td><td>−15.58 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN3"><div class="fn"><p>The correlations between tAI and PA vs. the correlations between stAI and PA in eight model organisms with available PA data. The third column refers to the number of genes with available PA measurements in each organism.</p></div></div></div></div></div><div class="table-wrap"><div id="DSU017TB3" data-id="DSU017TB3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>Spearman rank correlation of the original tAI and the stAI with PA</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Number of genes </th><th>Number of proteins </th><th><em>r</em> (tAI, PA) </th><th><em>r</em> (stAI, PA) </th><th>Change (%) </th></tr></thead><tbody><tr><td colspan="6">Non-fungal </td></tr><tr><td><em> E. coli</em> </td><td>4,145 </td><td>688 </td><td>0.5032 </td><td>0.5493 </td><td>+8.39 </td></tr><tr><td><em> S. dysentariae</em> </td><td>4,501 </td><td>1,266 </td><td>0.3574 </td><td>0.36757 </td><td>+2.76 </td></tr><tr><td><em> L. interrogans</em> </td><td>3,667 </td><td>2,114 </td><td>0.0959 </td><td>0.19408 </td><td>+50.58 </td></tr><tr><td><em> A. thaliana</em> </td><td>28,163 </td><td>8,478 </td><td>0.3328 </td><td>0.3762 </td><td>+11.53 </td></tr><tr><td><em> C. elegans</em> </td><td>22,830 </td><td>6,959 </td><td>0.0919 </td><td>0.0956 </td><td>+3.87 </td></tr><tr><td><em> D. melanogaster</em> </td><td>10,926 </td><td>6,510 </td><td>0.4878 </td><td>0.5001 </td><td>+2.46 </td></tr><tr><td colspan="6">Fungi </td></tr><tr><td><em> S. cerevisiae</em> </td><td>5,869 </td><td>2,666 </td><td>0.6915 </td><td>0.5802 </td><td>−19.18 </td></tr><tr><td><em> S. pombe</em> </td><td>5,017 </td><td>1,464 </td><td>0.6554 </td><td>0.56715 </td><td>−15.58 </td></tr></tbody></table></div><div content-id="DSU017TB3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Number of genes </th><th>Number of proteins </th><th><em>r</em> (tAI, PA) </th><th><em>r</em> (stAI, PA) </th><th>Change (%) </th></tr></thead><tbody><tr><td colspan="6">Non-fungal </td></tr><tr><td><em> E. coli</em> </td><td>4,145 </td><td>688 </td><td>0.5032 </td><td>0.5493 </td><td>+8.39 </td></tr><tr><td><em> S. dysentariae</em> </td><td>4,501 </td><td>1,266 </td><td>0.3574 </td><td>0.36757 </td><td>+2.76 </td></tr><tr><td><em> L. interrogans</em> </td><td>3,667 </td><td>2,114 </td><td>0.0959 </td><td>0.19408 </td><td>+50.58 </td></tr><tr><td><em> A. thaliana</em> </td><td>28,163 </td><td>8,478 </td><td>0.3328 </td><td>0.3762 </td><td>+11.53 </td></tr><tr><td><em> C. elegans</em> </td><td>22,830 </td><td>6,959 </td><td>0.0919 </td><td>0.0956 </td><td>+3.87 </td></tr><tr><td><em> D. melanogaster</em> </td><td>10,926 </td><td>6,510 </td><td>0.4878 </td><td>0.5001 </td><td>+2.46 </td></tr><tr><td colspan="6">Fungi </td></tr><tr><td><em> S. cerevisiae</em> </td><td>5,869 </td><td>2,666 </td><td>0.6915 </td><td>0.5802 </td><td>−19.18 </td></tr><tr><td><em> S. pombe</em> </td><td>5,017 </td><td>1,464 </td><td>0.6554 </td><td>0.56715 </td><td>−15.58 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN3"><div class="fn"><p>The correlations between tAI and PA vs. the correlations between stAI and PA in eight model organisms with available PA data. The third column refers to the number of genes with available PA measurements in each organism.</p></div></div></div></div><div data-id="DSU017F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/m_dsu01701.jpeg?Expires=2147483647&Signature=HsfXlpkhKqYPoq-T6w0H56joVsT-4tZvuKdDTyu1WCurthPowAE~SJjcAM5AYisAifXf4zQI5jHJvNrrTcuDdg5~V5Qiy8tbcyZLzZuZfnPofQjZpXklqWtKKsUpGYMhW-EmdzQQUFYJiMPXN5VnGDBtKlc9ipeGNDaROBR0w2Dd3mka2zRF7zgAp1X7qNNbpZejlMoksZJy1~AcxmR02zrWcLiZnMT55chFId1VmHwmS3qo0HCmkdwHjBWg8CJKXP6PEwsc0I-waZlLrG-CDVA-Jz~5q4y1rTaWhopeIjlNlIoPfparzcsY12H7UEnh2hDLasAjyr-xQMk9TSnosg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Dot plots of log(PA) vs. stAI and the corresponding Spearman rank correlations between stAI and PA. The correlations (and P-values) are calculated for the eight model organisms with PA measurements which include three bacteria (A–C), three non-fungal eukaryotes (D–F), and two fungi (G–H)." path-from-xml="dsu01701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197692/dsu01701.jpeg" path-from-xml="dsu01701.jpeg" target="_blank">View large</a><a section="87197692" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/dsu01701.jpeg?Expires=2147483647&Signature=IME89TI4HZPqVyV5tXEkyRI3yQAmJlEWG~~juiZ5gKUhN59hW69UJVkwZqRGoAIr4RkTbr6NQuvf~WvU~kkbbq4fHZduzD9fGAApcnIJHIq4xxqtWOm6L34w7S0Q02A6i9zg2-GbTpvSTi3yR5f3QCMOvAVdLJ6yQOGpwPgwLSeW7hk4aeIgzYW16Z4GXo5aIh7cZ-u9aRDHbqHvBe0b97Mqo8NICjIJ2pmEGlNH57AS9wcQHTmztYxhrrhAoYnrkUj9WGUdKr~6yItA2TQa8C11T~SnLrk4IjmuV3cjSs7dkYHyrPmwfbDb68WA~95xtIGpN~PdGQOLy2KOftxI~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197692&ar=2754544&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu01701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Dot plots of log(PA) vs. stAI and the corresponding Spearman rank correlations between stAI and PA. The correlations (and <em>P</em>-values) are calculated for the eight model organisms with PA measurements which include three bacteria (A–C), three non-fungal eukaryotes (D–F), and two fungi (G–H).</p></div></div><div content-id="DSU017F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/m_dsu01701.jpeg?Expires=2147483647&Signature=HsfXlpkhKqYPoq-T6w0H56joVsT-4tZvuKdDTyu1WCurthPowAE~SJjcAM5AYisAifXf4zQI5jHJvNrrTcuDdg5~V5Qiy8tbcyZLzZuZfnPofQjZpXklqWtKKsUpGYMhW-EmdzQQUFYJiMPXN5VnGDBtKlc9ipeGNDaROBR0w2Dd3mka2zRF7zgAp1X7qNNbpZejlMoksZJy1~AcxmR02zrWcLiZnMT55chFId1VmHwmS3qo0HCmkdwHjBWg8CJKXP6PEwsc0I-waZlLrG-CDVA-Jz~5q4y1rTaWhopeIjlNlIoPfparzcsY12H7UEnh2hDLasAjyr-xQMk9TSnosg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Dot plots of log(PA) vs. stAI and the corresponding Spearman rank correlations between stAI and PA. The correlations (and P-values) are calculated for the eight model organisms with PA measurements which include three bacteria (A–C), three non-fungal eukaryotes (D–F), and two fungi (G–H)." path-from-xml="dsu01701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197692/dsu01701.jpeg" path-from-xml="dsu01701.jpeg" target="_blank">View large</a><a section="87197692" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/dsu01701.jpeg?Expires=2147483647&Signature=IME89TI4HZPqVyV5tXEkyRI3yQAmJlEWG~~juiZ5gKUhN59hW69UJVkwZqRGoAIr4RkTbr6NQuvf~WvU~kkbbq4fHZduzD9fGAApcnIJHIq4xxqtWOm6L34w7S0Q02A6i9zg2-GbTpvSTi3yR5f3QCMOvAVdLJ6yQOGpwPgwLSeW7hk4aeIgzYW16Z4GXo5aIh7cZ-u9aRDHbqHvBe0b97Mqo8NICjIJ2pmEGlNH57AS9wcQHTmztYxhrrhAoYnrkUj9WGUdKr~6yItA2TQa8C11T~SnLrk4IjmuV3cjSs7dkYHyrPmwfbDb68WA~95xtIGpN~PdGQOLy2KOftxI~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197692&ar=2754544&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu01701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Dot plots of log(PA) vs. stAI and the corresponding Spearman rank correlations between stAI and PA. The correlations (and <em>P</em>-values) are calculated for the eight model organisms with PA measurements which include three bacteria (A–C), three non-fungal eukaryotes (D–F), and two fungi (G–H).</p></div></div></div><p>We emphasize that although previous studies reported a significant positive correlation between CUB and expression levels in the model organisms studied here,<sup><a reveal-id="DSU017C12" data-open="DSU017C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSU017C23" data-open="DSU017C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSU017C56" data-open="DSU017C56" class="link link-ref link-reveal xref-bibr">56</a>,<a reveal-id="DSU017C57" data-open="DSU017C57" class="link link-ref link-reveal xref-bibr">57</a></sup> it is not trivial that <em>Sij</em> optimization based on CUB improves the correlation with protein levels. Specifically, CUB is correlated with protein levels, but mRNA levels and protein levels in different organisms are also usually correlated;<sup><a reveal-id="DSU017C52" data-open="DSU017C52" class="link link-ref link-reveal xref-bibr">52</a>,<a reveal-id="DSU017C58" data-open="DSU017C58" class="link link-ref link-reveal xref-bibr">58</a>,<a reveal-id="DSU017C59" data-open="DSU017C59" class="link link-ref link-reveal xref-bibr">59</a></sup> thus it is not clear that <em>S<sub>ij</sub></em> optimized based on the CUB of the organism necessarily have higher correlation with protein levels than the <em>S<sub>ij</sub></em> optimized based on mRNA levels of <em>S. cerevisiae</em>.</p>                            <h3 scrollto-destination=87197695 id="87197695" class="section-title" data-legacy-id=s3c>3.3. Robustness analysis demonstrates that in non-fungal organisms the stAI outperforms the tAI in terms of the correlation with PA</h3>
<div class="block-child-p">In order to empirically estimate the organism-specific probability that stAI (which is based on DCBS) improves the correlation with PA, a jack-knifing approach was implemented. One round of it involved the implementation of the algorithm for calculating the stAI on a sample of random subset of 50% of the proteins. Finally, the correlation between stAI and PA was computed for the sample and was compared with the correlation of PA with two related indices: the original tAI and stAI that is based on RCBS (i.e. its <em>S<sub>ij</sub></em> were inferred from RCBS and not from DCBS). This procedure was repeated 100 times where each time the index exhibited the highest correlation with PA was counted (Fig. <a reveal-id="DSU017F2" data-open="DSU017F2" class="link link-reveal link-table xref-fig">2</a>). <div data-id="DSU017F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/m_dsu01702.jpeg?Expires=2147483647&Signature=z7BOl0o-XyH4vi43KUKUi2doyLqSDn~VOyfJdwpYyg7VTtCNcW9q-sW0J2Pcnn4OnxvhilBr4PyL7R9ejD3lEypW9BNGQXFj~cEnDCSxR4-8en28~kYUp4U~sWGnRaUX8MNuiOC5cb1QjNOBo7Sz0nTHI39Jg3c-FrzcggiWeFpJSkdY0qKGV6EgWhOpyrQ-~pO2CGAuAmdJ9Ev1gGuqoCBiZ723m7m2BYi1jaAE3LKRUuh1tMoikD~CYogyX46Ipf81cfSBf4RlOyLA7T6o7Zm2ThCL23UFRD0P8k-JnlbP8CYy2iYBh-cAS6tKk5ogKof87g3j~L2s17zDDAOyAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison between stAI and the tAI. The middle bars representing the number of times (based on the jack-knifing analysis) the stAI outperformed the other versions of the tAI; as can be seen, stAI outperforms tAI in all non-fungal organisms." path-from-xml="dsu01702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197696/dsu01702.jpeg" path-from-xml="dsu01702.jpeg" target="_blank">View large</a><a section="87197696" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/dsu01702.jpeg?Expires=2147483647&Signature=0X9cAQLJV7ryFHq7GXBjPQ0bdQBNof-iEkJXVJ1TivJ9rS96AkrwgUSCu5l6qVHGgdltf2aX~6-PAnazZtn3kNvUC4GdvO6M6zs7cu7MncXemkfkx58mJPICMXg33OUnwi0zJwdSdFg4ufdpdT8ZB~V2qTCv5soRFDxKzWVL-CSJpb26uPrQyFn675ZcOfoiY28v86U8uzSCE8tcwQKdnxPp95kzcHBTdHbyVjvhYnNEw0EoygJmvfBDMfbHzCbi7o8A1-RwVb9CqHOH6TKWMb5XkorpLghVnzb7Xo5GDds-kF63IHOWucJ7ztcSyzz6ZAQ1vbvo~5drL0EjOkcMZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197696&ar=2754544&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu01702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison between stAI and the tAI. The middle bars representing the number of times (based on the jack-knifing analysis) the stAI outperformed the other versions of the tAI; as can be seen, stAI outperforms tAI in all non-fungal organisms.</p></div></div><div content-id="DSU017F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/m_dsu01702.jpeg?Expires=2147483647&Signature=z7BOl0o-XyH4vi43KUKUi2doyLqSDn~VOyfJdwpYyg7VTtCNcW9q-sW0J2Pcnn4OnxvhilBr4PyL7R9ejD3lEypW9BNGQXFj~cEnDCSxR4-8en28~kYUp4U~sWGnRaUX8MNuiOC5cb1QjNOBo7Sz0nTHI39Jg3c-FrzcggiWeFpJSkdY0qKGV6EgWhOpyrQ-~pO2CGAuAmdJ9Ev1gGuqoCBiZ723m7m2BYi1jaAE3LKRUuh1tMoikD~CYogyX46Ipf81cfSBf4RlOyLA7T6o7Zm2ThCL23UFRD0P8k-JnlbP8CYy2iYBh-cAS6tKk5ogKof87g3j~L2s17zDDAOyAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison between stAI and the tAI. The middle bars representing the number of times (based on the jack-knifing analysis) the stAI outperformed the other versions of the tAI; as can be seen, stAI outperforms tAI in all non-fungal organisms." path-from-xml="dsu01702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197696/dsu01702.jpeg" path-from-xml="dsu01702.jpeg" target="_blank">View large</a><a section="87197696" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/dsu01702.jpeg?Expires=2147483647&Signature=0X9cAQLJV7ryFHq7GXBjPQ0bdQBNof-iEkJXVJ1TivJ9rS96AkrwgUSCu5l6qVHGgdltf2aX~6-PAnazZtn3kNvUC4GdvO6M6zs7cu7MncXemkfkx58mJPICMXg33OUnwi0zJwdSdFg4ufdpdT8ZB~V2qTCv5soRFDxKzWVL-CSJpb26uPrQyFn675ZcOfoiY28v86U8uzSCE8tcwQKdnxPp95kzcHBTdHbyVjvhYnNEw0EoygJmvfBDMfbHzCbi7o8A1-RwVb9CqHOH6TKWMb5XkorpLghVnzb7Xo5GDds-kF63IHOWucJ7ztcSyzz6ZAQ1vbvo~5drL0EjOkcMZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197696&ar=2754544&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu01702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison between stAI and the tAI. The middle bars representing the number of times (based on the jack-knifing analysis) the stAI outperformed the other versions of the tAI; as can be seen, stAI outperforms tAI in all non-fungal organisms.</p></div></div></div><p>As can be seen, the results demonstrate again that for non-fungal organisms, the species-specific inference of the <em>S<sub>ij</sub></em> tends to predict PA better than the traditional tAI. The 100 <em>S<sub>ij</sub></em> sets, their corresponding correlations between stAI and DCBS and the full taxonomy for each organism, are provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/dsu017_Supplementary_Data.zip?Expires=2147483647&Signature=oVKv7qorjDac3xeIxZvVwbsJ-qcQdF~HYxNmw~i8D146Xv1Qcmgq-QO5Xyp4rkP~pBak4jTqiqgW5eB1jlrOFDUnSx8Yy3e4tWBtbqsO9MW3bdi~0IwtKzN4kt61Kv3Ss6pyKGlDqjN10WzQPryart8qWqyxiN~OomqZF8tD-d7cS1~XpgDbefils0tEkiTFvHG1Yq3NtEV~g1ymvfW7G4M7X3j5Nvtt3bCc9KRPQK9nVopAY35nPCgW2Q6aHHWhWwRK0eDp76GTBZ9nO5~JEidfjT5qEA7PE-zAWUSy95GP0GpiNeUdA8t7VZksBhnr7DXFXHvHeZlj5N3VJWIFWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>.</p>                            <h3 scrollto-destination=87197700 id="87197700" class="section-title" data-legacy-id=s3d>3.4. S<sub>ij</sub> inferred based on CUB are similar to the S<sub>ij</sub> inferred based on PA</h3>
<p>In order to check whether the <em>S<sub>ij</sub></em> that are inferred based on CUB (i.e. based on the DCBS) converge to similar values as those which are based on expression levels, we computed <em>S<sub>ij</sub></em> sets by optimizing the correlation between stAI and PA for the model organisms with available PA measurements. This approach of using expression levels to optimize the tAI was employed in the study of ref.<sup><a reveal-id="DSU017C8" data-open="DSU017C8" class="link link-ref link-reveal xref-bibr">8</a></sup>. The Spearman rank correlation between the concatenated vectors of <em>S<sub>ij</sub></em>-values (35 points) inferred based on the DCBS and the one inferred based on PA is 0.6902 (<em>P</em>-value &lt;10<sup>−5</sup>; permutation <em>P</em>-value &lt;0.001; 35 points). The Euclidean distance between the two vectors is also significantly lower than the one obtained by random permutation of the two vectors; specifically, when we performed 1,000 permutations of these values, all of them had higher Euclidean distance (<em>P</em>-value &lt;0.001). The <em>S<sub>ij</sub></em>-values that were obtained via correlation with DCBS and the ones obtained via correlation with PA are provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/dsu017_Supplementary_Data.zip?Expires=2147483647&Signature=oVKv7qorjDac3xeIxZvVwbsJ-qcQdF~HYxNmw~i8D146Xv1Qcmgq-QO5Xyp4rkP~pBak4jTqiqgW5eB1jlrOFDUnSx8Yy3e4tWBtbqsO9MW3bdi~0IwtKzN4kt61Kv3Ss6pyKGlDqjN10WzQPryart8qWqyxiN~OomqZF8tD-d7cS1~XpgDbefils0tEkiTFvHG1Yq3NtEV~g1ymvfW7G4M7X3j5Nvtt3bCc9KRPQK9nVopAY35nPCgW2Q6aHHWhWwRK0eDp76GTBZ9nO5~JEidfjT5qEA7PE-zAWUSy95GP0GpiNeUdA8t7VZksBhnr7DXFXHvHeZlj5N3VJWIFWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>.</p>                            <h3 scrollto-destination=87197702 id="87197702" class="section-title" data-legacy-id=s3e>3.5. Considering all tRNA–codon pairing possibilities do not improve the performances of the stAI</h3>
<p>There are possible cases of non-standard base pairing that currently are not included in the tAI wobble rules (U–U binding for instance). It is interesting to check whether introducing such additional rules to the model can improve its performances. Using Equation (1), we included in the set of <em>S<sub>ij</sub></em> all missing pairing options (U:U, C:U, U:C, C:C, C:A, G:A, I:G, and G:G). An initial weight of 0.5 was given to all non-WC <em>S<sub>ij</sub></em> (WC <em>S<sub>ij</sub></em> are fixed to zero). Nevertheless, considering all possible pairings in the stAI weights calculation did not improve the correlation of the stAI with DCBS or with PA. The original approach (i.e. WC and wobble only) reached higher maxima values for seven of the eight models. In addition, for five of the eight models, better correlations with PA, were obtained for the original stAI.</p>                            <h3 scrollto-destination=87197706 id="87197706" class="section-title" data-legacy-id=s3f>3.6. Adding constraint on WC interactions do not improve the performances of the stAI</h3>
<p>The first four <em>S<sub>ij</sub></em> weights, which represent the constraint on WC interactions, are fixed to zero. Assuming that these interactions might not be perfect and thus allowing them to change during the optimization did not provide further improvement. The starting point for each model was the one exhibited the maximum correlation between stAI and DCBS in the original search. The original search, in which WC <em>S<sub>ij</sub></em> are fixed to zero, reached higher maxima values for five of the eight models; in addition, in five of the eight models, the original search exhibited a better correlations with PA than the new search.</p>                            <h3 scrollto-destination=87197711 id="87197711" class="section-title" data-legacy-id=s3g>3.7. Distances between the inferred S<sub>ij</sub>-values correlate with evolutionary proximity</h3>
<div class="block-child-p">In the next step, we aimed at understanding if the organism-specific <em>S<sub>ij</sub>-</em>values reflect evolutionary proximity and if they are biologically meaningful. To this end, the inferred <em>S<sub>ij</sub>-</em>value sets of the 100 organisms were clustered into three groups using the <em>k-</em>means algorithm.<sup><a reveal-id="DSU017C60" data-open="DSU017C60" class="link link-ref link-reveal xref-bibr">60</a></sup> We compared the clustering result to the clusters obtained by partitioning the organisms to the three domains of life. The clustering correctly classified 77% of the 26 eukarya, 45% of the 38 bacteria, and 67% of the 36 archaea (Fig. <a reveal-id="DSU017F3" data-open="DSU017F3" class="link link-reveal link-table xref-fig">3</a>). In general, 61% of the total 100 organisms were classified into the correct domain. <div data-id="DSU017F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/m_dsu01703.jpeg?Expires=2147483647&Signature=tv2hxI-4FS-~Nt-1GNjobB3Ai0kr6aPrW~EeIQmF~JpR8vKCDtJy3ZwOlT2m8DtluZVQay5MA9n20CQbAwey89P9S00OscZUOMlHYvE6XjSsHEEF2cTPgYTG2zBLc6WrCRYuYwWcylBPnYGVuGRIGlhzsshflahse8AEeZjTGoYyN3NaFTAKyQgnd3S2SpLFpbpbivcSEuP~cfTqClk-mIhVx7y-LW4JtSQfCoZ0UZ7sy-vyeXgDragWWVkJ9cxtXTyzUZXNZOeb7phn1ZhalCB7ctPQX-f-5RXK5KTeWV6~f5Hc9M-xtFEVrAPdndgRc1aeB6OgdepHhEgcWrk-kA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Principal component analysis (PCA) on the 100 different Sij sets demonstrates clustering of Sij according to evolutionary domains. The first three components of the PCA are presented. Each point in the figure represents one of the 100 analysed organisms; the shape of the point corresponds to the domain of the organism at the tree of life and the colour corresponds to the cluster the point was classified based on the k-means algorithm." path-from-xml="dsu01703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197714/dsu01703.jpeg" path-from-xml="dsu01703.jpeg" target="_blank">View large</a><a section="87197714" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/dsu01703.jpeg?Expires=2147483647&Signature=JtX2OHbPx-gGx2L-KybysGzTUMya4pdWeIEoCb~7zw48tvTl~wff1gJcGjhz4iURhk9TVoXDpxlSUNfsxdHkRoJUtdWZxol29eIzvhpPX9jrRUfep8ukUoJDLX-99bxd5iC0Pt998O6VdiNtI4vSjmZ6JwMGIRn6Lyne6BSzrOMRaggBc41HX4dxm~OfQ3ZqV~Au8OaLEo-U8I0~4mLuE~ybXFsRsmW2PORnVUeH8oLtuKbcBKdfxwxXfqJ5lKgZvrjTJYPJo8mkNZVCqKxp-FkxbAcpqsNLoQpX9KkAefPpp235Apmg3igzyP7NopBxu15TY42OImEIQqRWwDcvZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197714&ar=2754544&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu01703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Principal component analysis (PCA) on the 100 different <em>S<sub>ij</sub></em> sets demonstrates clustering of <em>S<sub>ij</sub></em> according to evolutionary domains. The first three components of the <em>PCA</em> are presented. Each point in the figure represents one of the 100 analysed organisms; the shape of the point corresponds to the domain of the organism at the tree of life and the colour corresponds to the cluster the point was classified based on the <em>k-</em>means algorithm.</p></div></div><div content-id="DSU017F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/m_dsu01703.jpeg?Expires=2147483647&Signature=tv2hxI-4FS-~Nt-1GNjobB3Ai0kr6aPrW~EeIQmF~JpR8vKCDtJy3ZwOlT2m8DtluZVQay5MA9n20CQbAwey89P9S00OscZUOMlHYvE6XjSsHEEF2cTPgYTG2zBLc6WrCRYuYwWcylBPnYGVuGRIGlhzsshflahse8AEeZjTGoYyN3NaFTAKyQgnd3S2SpLFpbpbivcSEuP~cfTqClk-mIhVx7y-LW4JtSQfCoZ0UZ7sy-vyeXgDragWWVkJ9cxtXTyzUZXNZOeb7phn1ZhalCB7ctPQX-f-5RXK5KTeWV6~f5Hc9M-xtFEVrAPdndgRc1aeB6OgdepHhEgcWrk-kA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Principal component analysis (PCA) on the 100 different Sij sets demonstrates clustering of Sij according to evolutionary domains. The first three components of the PCA are presented. Each point in the figure represents one of the 100 analysed organisms; the shape of the point corresponds to the domain of the organism at the tree of life and the colour corresponds to the cluster the point was classified based on the k-means algorithm." path-from-xml="dsu01703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197714/dsu01703.jpeg" path-from-xml="dsu01703.jpeg" target="_blank">View large</a><a section="87197714" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/dsu01703.jpeg?Expires=2147483647&Signature=JtX2OHbPx-gGx2L-KybysGzTUMya4pdWeIEoCb~7zw48tvTl~wff1gJcGjhz4iURhk9TVoXDpxlSUNfsxdHkRoJUtdWZxol29eIzvhpPX9jrRUfep8ukUoJDLX-99bxd5iC0Pt998O6VdiNtI4vSjmZ6JwMGIRn6Lyne6BSzrOMRaggBc41HX4dxm~OfQ3ZqV~Au8OaLEo-U8I0~4mLuE~ybXFsRsmW2PORnVUeH8oLtuKbcBKdfxwxXfqJ5lKgZvrjTJYPJo8mkNZVCqKxp-FkxbAcpqsNLoQpX9KkAefPpp235Apmg3igzyP7NopBxu15TY42OImEIQqRWwDcvZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197714&ar=2754544&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu01703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Principal component analysis (PCA) on the 100 different <em>S<sub>ij</sub></em> sets demonstrates clustering of <em>S<sub>ij</sub></em> according to evolutionary domains. The first three components of the <em>PCA</em> are presented. Each point in the figure represents one of the 100 analysed organisms; the shape of the point corresponds to the domain of the organism at the tree of life and the colour corresponds to the cluster the point was classified based on the <em>k-</em>means algorithm.</p></div></div></div><p>Properly randomized genomes that were generated by maintaining the CUB of the genome and its protein content were used to empirically test the significance of this clustering (see section 2). None of the 20 randomizations outperformed the original clustering (with respect to total correct classifications, empirical <em>P</em>-value &lt;0.05). This result demonstrates that with high probability the reported clustering cannot be obtained randomly even when considering randomized genomes with similar features to the original ones (global CUB and the same proteome), supporting the conjecture that the obtained <em>S<sub>ij</sub>-</em>value similarities correlate with the evolutionary distances and thus have biological meaning.</p><p>Finally, it is important to mention that there is co-evolution between CUB and tRNA levels (see, for example,<sup><a reveal-id="DSU017C24" data-open="DSU017C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSU017C34" data-open="DSU017C34" class="link link-ref link-reveal xref-bibr">34</a>,<a reveal-id="DSU017C61" data-open="DSU017C61" class="link link-ref link-reveal xref-bibr">61</a>,<a reveal-id="DSU017C62" data-open="DSU017C62" class="link link-ref link-reveal xref-bibr">62</a></sup>). Specifically, based on various theories, the CUB should co-evolve with the tRNA pool and the tRNA–codon interaction efficiencies to optimize the organism fitness; thus, CUB, tRNA levels, and tRNA–codon interactions cannot be separated.</p>                            <h3 scrollto-destination=87197722 id="87197722" class="section-title" data-legacy-id=s3h>3.8. Similarities among the inferred S<sub>ij</sub>-values of the analysed organisms</h3>
<p>The mean efficiency of the different inferred codon–anticodon interactions over all the analysed organisms are summarized in Table <a reveal-id="DSU017TB4" data-open="DSU017TB4" class="link link-reveal link-table xref-fig">4</a>. The results emphasize the similarities among the different organisms and domains. <div content-id="DSU017TB4" class="table-modal"><div class="table-wrap"><div id="DSU017TB4" data-id="DSU017TB4" class="table-wrap-title"><div class="label -label">Table 4.</div><div class="caption"><p>The mean inferred wobble <em>S<sub>ij</sub></em>-values</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>SG:U </th><th>SI:C </th><th>SI:A </th><th>SU:G </th><th>SL/agm:A </th></tr></thead><tbody><tr><td>Eukarya </td><td>0.7861 </td><td>0.4659 </td><td>0.9075 </td><td>0.6295 </td><td>— </td></tr><tr><td>Bacteria </td><td>0.6294 </td><td>0.4211 </td><td>0.8773 </td><td>0.698 </td><td>0.7309 </td></tr><tr><td>Archaea </td><td>0.3898 </td><td>0.3774 </td><td>0.5015 </td><td>0.4363 </td><td>0.6453 </td></tr><tr><td>Mean </td><td>0.6 </td><td>0.42 </td><td>0.76 </td><td>0.588 </td><td>0.6881 </td></tr></tbody></table></div><div content-id="DSU017TB4" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>SG:U </th><th>SI:C </th><th>SI:A </th><th>SU:G </th><th>SL/agm:A </th></tr></thead><tbody><tr><td>Eukarya </td><td>0.7861 </td><td>0.4659 </td><td>0.9075 </td><td>0.6295 </td><td>— </td></tr><tr><td>Bacteria </td><td>0.6294 </td><td>0.4211 </td><td>0.8773 </td><td>0.698 </td><td>0.7309 </td></tr><tr><td>Archaea </td><td>0.3898 </td><td>0.3774 </td><td>0.5015 </td><td>0.4363 </td><td>0.6453 </td></tr><tr><td>Mean </td><td>0.6 </td><td>0.42 </td><td>0.76 </td><td>0.588 </td><td>0.6881 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN4"><div class="fn"><p>The mean inferred wobble <em>S<sub>ij</sub></em>-values strength for each domain of life and for the entire analysed dataset (last row).</p></div></div></div></div></div><div class="table-wrap"><div id="DSU017TB4" data-id="DSU017TB4" class="table-wrap-title"><div class="label -label">Table 4.</div><div class="caption"><p>The mean inferred wobble <em>S<sub>ij</sub></em>-values</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>SG:U </th><th>SI:C </th><th>SI:A </th><th>SU:G </th><th>SL/agm:A </th></tr></thead><tbody><tr><td>Eukarya </td><td>0.7861 </td><td>0.4659 </td><td>0.9075 </td><td>0.6295 </td><td>— </td></tr><tr><td>Bacteria </td><td>0.6294 </td><td>0.4211 </td><td>0.8773 </td><td>0.698 </td><td>0.7309 </td></tr><tr><td>Archaea </td><td>0.3898 </td><td>0.3774 </td><td>0.5015 </td><td>0.4363 </td><td>0.6453 </td></tr><tr><td>Mean </td><td>0.6 </td><td>0.42 </td><td>0.76 </td><td>0.588 </td><td>0.6881 </td></tr></tbody></table></div><div content-id="DSU017TB4" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>SG:U </th><th>SI:C </th><th>SI:A </th><th>SU:G </th><th>SL/agm:A </th></tr></thead><tbody><tr><td>Eukarya </td><td>0.7861 </td><td>0.4659 </td><td>0.9075 </td><td>0.6295 </td><td>— </td></tr><tr><td>Bacteria </td><td>0.6294 </td><td>0.4211 </td><td>0.8773 </td><td>0.698 </td><td>0.7309 </td></tr><tr><td>Archaea </td><td>0.3898 </td><td>0.3774 </td><td>0.5015 </td><td>0.4363 </td><td>0.6453 </td></tr><tr><td>Mean </td><td>0.6 </td><td>0.42 </td><td>0.76 </td><td>0.588 </td><td>0.6881 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN4"><div class="fn"><p>The mean inferred wobble <em>S<sub>ij</sub></em>-values strength for each domain of life and for the entire analysed dataset (last row).</p></div></div></div></div></p><p>As mentioned, <em>S<sub>ij</sub></em>-values are between 0 and 1. Since these values represent a constraint on the codon–anticodon interactions, interactions with lower values are considered more efficient. For example, it can be seen from Table <a reveal-id="DSU017TB4" data-open="DSU017TB4" class="link link-reveal link-table xref-fig">4</a> that the inosine–cytosine interaction has the lowest mean value (<em>sI:C</em> = 0.42), while the wobble inosine–adenosine has the largest mean value (<em>sI:A</em> = 0.76). This suggests a good I:C interaction and an inefficient I:A interaction. These findings are supported by Murphy and Ramakrishnan.<sup><a reveal-id="DSU017C63" data-open="DSU017C63" class="link link-ref link-reveal xref-bibr">63</a></sup> where it is stated that the decoding of adenosine-ending codons by inosine is inefficient. It is also mentioned that the inosine–cytosine interaction is very similar to the canonical G:C pair.</p><div class="block-child-p"><em>SL:A</em> and <em>Sagm:A</em> have a similar distribution, and the corresponding <em>P</em>-value proves that the mean values are not significantly different (see Fig. <a reveal-id="DSU017F4" data-open="DSU017F4" class="link link-reveal link-table xref-fig">4</a>E). Since agmatidine is in many ways similar to lysidine (see ref.<sup><a reveal-id="DSU017C64" data-open="DSU017C64" class="link link-ref link-reveal xref-bibr">64</a></sup>), it makes sense that their <em>S<sub>ij</sub></em>-values are similar. <div data-id="DSU017F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/m_dsu01704A.jpeg?Expires=2147483647&amp;Signature=E5K7A5wHii9futTuJE2yvzRSj8~6EzQcgh6z4H4rR0EUoxS58LBWuJGB9fK8k1z9L9KAMfAMD~KVN9zlHqst17Q-W~lEG1O8t2t-2OmAU4vvun9ZScMBHu1oQII6rDkCOd0AQgSAROLphEBb0GlEO7ZDNOuKVgTM7add8hUci1QK2ijcN9V7BCrcd~FtUsC8aQaRuibzNVtsw-gZTq9Tfi2Ssqa6O0BwN1QixzUpd2XoMB6hUQpKJ9XmC7bDktEsAqpp9c8QSPDpYkr0kUd8WwSewISlfpSydbrwE14IIHApQGZywKVkMDCI-8vEuLp8wvs526THcTI0NafM6CVXBw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sij distributions among different domains of life. Each figure contains three histograms representing the Sij in the different domains of life; the mean and SD of the Sij-values in each domain are also reported. The P-values corresponding to the comparison between every two Sij means appear in the bottom of the figure (see section 2 sub-section ‘Permutation test for comparing two Sij means’)." path-from-xml="dsu01704A.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197729/dsu01704A.jpeg" path-from-xml="dsu01704A.jpeg" target="_blank">View large</a><a section="87197729" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/dsu01704A.jpeg?Expires=2147483647&amp;Signature=GDtVWjyc3jn1fGOointfSiwaO~ZdvUNSHPjb3X0y4p2UgiWI8PVYBCvF9jLxoGoB4INRIFndf~2NxY6tkKYdlvuv2euN4T-QmN~wVdIUogPvWyFx92P8AQyEbGV0E7zwKt~tK0nZ9SjXMBFl552zLpNT1jX45wrY8cgqnefFDdQOwFAa1PohNETOewL8DoirQdmBzvTGulqQ~5xMVFDtBb2U6oSpI~66tSmbO9zTdjtHWVeKMV0fFJ8qg5vSqo~NW-nFBFaKW72ZO1EdxtyiezTBYwQDNx-7OZ6qioRAtLZ08fS0XqSveZpSpti2w~yzHB5WpuaGz1rKD3Vm~8Hwgw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87197729&amp;ar=2754544&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu01704A.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>S<sub>ij</sub></em> distributions among different domains of life. Each figure contains three histograms representing the <em>S<sub>ij</sub></em> in the different domains of life; the mean and SD of the <em>S<sub>ij</sub></em>-values in each domain are also reported. The <em>P</em>-values corresponding to the comparison between every two <em>S<sub>ij</sub></em> means appear in the bottom of the figure (see section 2 sub-section ‘Permutation test for comparing two <em>S<sub>ij</sub></em> means’).</p></div></div><div content-id="DSU017F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/m_dsu01704A.jpeg?Expires=2147483647&amp;Signature=E5K7A5wHii9futTuJE2yvzRSj8~6EzQcgh6z4H4rR0EUoxS58LBWuJGB9fK8k1z9L9KAMfAMD~KVN9zlHqst17Q-W~lEG1O8t2t-2OmAU4vvun9ZScMBHu1oQII6rDkCOd0AQgSAROLphEBb0GlEO7ZDNOuKVgTM7add8hUci1QK2ijcN9V7BCrcd~FtUsC8aQaRuibzNVtsw-gZTq9Tfi2Ssqa6O0BwN1QixzUpd2XoMB6hUQpKJ9XmC7bDktEsAqpp9c8QSPDpYkr0kUd8WwSewISlfpSydbrwE14IIHApQGZywKVkMDCI-8vEuLp8wvs526THcTI0NafM6CVXBw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sij distributions among different domains of life. Each figure contains three histograms representing the Sij in the different domains of life; the mean and SD of the Sij-values in each domain are also reported. The P-values corresponding to the comparison between every two Sij means appear in the bottom of the figure (see section 2 sub-section ‘Permutation test for comparing two Sij means’)." path-from-xml="dsu01704A.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197729/dsu01704A.jpeg" path-from-xml="dsu01704A.jpeg" target="_blank">View large</a><a section="87197729" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/dsu01704A.jpeg?Expires=2147483647&amp;Signature=GDtVWjyc3jn1fGOointfSiwaO~ZdvUNSHPjb3X0y4p2UgiWI8PVYBCvF9jLxoGoB4INRIFndf~2NxY6tkKYdlvuv2euN4T-QmN~wVdIUogPvWyFx92P8AQyEbGV0E7zwKt~tK0nZ9SjXMBFl552zLpNT1jX45wrY8cgqnefFDdQOwFAa1PohNETOewL8DoirQdmBzvTGulqQ~5xMVFDtBb2U6oSpI~66tSmbO9zTdjtHWVeKMV0fFJ8qg5vSqo~NW-nFBFaKW72ZO1EdxtyiezTBYwQDNx-7OZ6qioRAtLZ08fS0XqSveZpSpti2w~yzHB5WpuaGz1rKD3Vm~8Hwgw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87197729&amp;ar=2754544&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu01704A.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>S<sub>ij</sub></em> distributions among different domains of life. Each figure contains three histograms representing the <em>S<sub>ij</sub></em> in the different domains of life; the mean and SD of the <em>S<sub>ij</sub></em>-values in each domain are also reported. The <em>P</em>-values corresponding to the comparison between every two <em>S<sub>ij</sub></em> means appear in the bottom of the figure (see section 2 sub-section ‘Permutation test for comparing two <em>S<sub>ij</sub></em> means’).</p></div></div></div>                            <h3 scrollto-destination=87197737 id="87197737" class="section-title" data-legacy-id=s3i>3.9. Differences among the inferred S<sub>ij</sub>-values of different groups of organisms</h3>
<p>To test the hypothesis that the <em>S<sub>ij</sub></em>-values of different organisms groups (i.e. different domains or different phylums within the same domain) have significantly different means, we computed an empirical permutation <em>P</em>-value (see section 2). The <em>S<sub>ij</sub></em> distributions and their corresponding <em>P</em>-values are presented in Fig. <a reveal-id="DSU017F4" data-open="DSU017F4" class="link link-reveal link-table xref-fig">4</a>.</p><div class="block-child-p">As can be seen, the <em>sI:C</em> distribution is similar between the three domains (Fig. <a reveal-id="DSU017F4" data-open="DSU017F4" class="link link-reveal link-table xref-fig">4</a>B); however, <em>sU:G</em>, <em>sI:A</em>, and <em>sG:U</em> tend to be significantly different among the three domains. An empirical <em>P</em>-value was used also for the comparison between the two major phylums within each domain. The only significant difference was obtained for the <em>sI:A</em> distribution of eukarya subgroups <em>Opisthokonta</em> vs. <em>Viridiplantae</em> and bacteria subgroups <em>Proteobacteria</em> vs. <em>Cyanobacteria</em> (see Fig. <a reveal-id="DSU017F5" data-open="DSU017F5" class="link link-reveal link-table xref-fig">5</a>). All other insignificant <em>S<sub>ij</sub></em> distributions among different phylums appear in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/dsu017_Supplementary_Data.zip?Expires=2147483647&amp;Signature=oVKv7qorjDac3xeIxZvVwbsJ-qcQdF~HYxNmw~i8D146Xv1Qcmgq-QO5Xyp4rkP~pBak4jTqiqgW5eB1jlrOFDUnSx8Yy3e4tWBtbqsO9MW3bdi~0IwtKzN4kt61Kv3Ss6pyKGlDqjN10WzQPryart8qWqyxiN~OomqZF8tD-d7cS1~XpgDbefils0tEkiTFvHG1Yq3NtEV~g1ymvfW7G4M7X3j5Nvtt3bCc9KRPQK9nVopAY35nPCgW2Q6aHHWhWwRK0eDp76GTBZ9nO5~JEidfjT5qEA7PE-zAWUSy95GP0GpiNeUdA8t7VZksBhnr7DXFXHvHeZlj5N3VJWIFWA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>. <div data-id="DSU017F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/m_dsu01705.jpeg?Expires=2147483647&amp;Signature=pJrAEOac6h9Li2Q1HbdKyah~Za-~TrqF1jY21-rrW7xkXRtIMCIjGFAsaaWu4KWvjOl4WbE0Z~QSS0hPzuC0Ik5sVhrUHGkocDR3Su4Ta8a3EM5VnLeCHNiUU6eVhrTflPkKIyABpcf~HxGt-kXU84VKhrZx5GBo60jWt2U2yb9VOCSAbtc-Jv8OeVWWdWnHiGqdt3r0zotSGqIO9ms4TRGiQt0ZCQ0CozJCHmE3cFTDZbTGqD3BEqdVcSuwQJ5AMxMjtZ1S~S4Kq8WALQqV0fwT6FbJVReJo2yNeIU0aoRgCZmEyQRn4KVrxQl1Byf9Wugr1386BTHu84uQSjFx~w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="sI:A distribution within the major phylums of the eukaryotic and bacterial domains with a significant empirical P-value (see details in section 2)." path-from-xml="dsu01705.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197742/dsu01705.jpeg" path-from-xml="dsu01705.jpeg" target="_blank">View large</a><a section="87197742" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/dsu01705.jpeg?Expires=2147483647&amp;Signature=Z4E-WhMx~rO7jArFpc~DvjFmZYV~VY5KoT9EX-h-lsilMD7dduemJo3wrT2A56yu6FtmFmo0Gh5TfFk11Em1ADg5XmrBqmM8~~BePzm66G-TE5Eic5eymJx8XCKzwSnWOrJTYOOPINGO~1jhl7ygejswn-LJ8Yqyk20ANt8Jjj6LCm6QZjTjxKeI9sc7tkPiPNcOsJQcobXccM0FvhY5RmbCW~upH9yzw7svXkf3dJQDK7B3~6eC7z40Mso6SEaBaQGHgDPHxJ2MByO2G1GBPZzSm5Miyp8ekzbnSMjW4ieK04omh0LYvR84aC7tezWrHJTV6~VrjGtNsgi3Neolvw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87197742&amp;ar=2754544&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu01705.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>sI:A</em> distribution within the major phylums of the eukaryotic and bacterial domains with a significant empirical <em>P</em>-value (see details in section 2).</p></div></div><div content-id="DSU017F5" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/m_dsu01705.jpeg?Expires=2147483647&amp;Signature=pJrAEOac6h9Li2Q1HbdKyah~Za-~TrqF1jY21-rrW7xkXRtIMCIjGFAsaaWu4KWvjOl4WbE0Z~QSS0hPzuC0Ik5sVhrUHGkocDR3Su4Ta8a3EM5VnLeCHNiUU6eVhrTflPkKIyABpcf~HxGt-kXU84VKhrZx5GBo60jWt2U2yb9VOCSAbtc-Jv8OeVWWdWnHiGqdt3r0zotSGqIO9ms4TRGiQt0ZCQ0CozJCHmE3cFTDZbTGqD3BEqdVcSuwQJ5AMxMjtZ1S~S4Kq8WALQqV0fwT6FbJVReJo2yNeIU0aoRgCZmEyQRn4KVrxQl1Byf9Wugr1386BTHu84uQSjFx~w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="sI:A distribution within the major phylums of the eukaryotic and bacterial domains with a significant empirical P-value (see details in section 2)." path-from-xml="dsu01705.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197742/dsu01705.jpeg" path-from-xml="dsu01705.jpeg" target="_blank">View large</a><a section="87197742" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/dsu01705.jpeg?Expires=2147483647&amp;Signature=Z4E-WhMx~rO7jArFpc~DvjFmZYV~VY5KoT9EX-h-lsilMD7dduemJo3wrT2A56yu6FtmFmo0Gh5TfFk11Em1ADg5XmrBqmM8~~BePzm66G-TE5Eic5eymJx8XCKzwSnWOrJTYOOPINGO~1jhl7ygejswn-LJ8Yqyk20ANt8Jjj6LCm6QZjTjxKeI9sc7tkPiPNcOsJQcobXccM0FvhY5RmbCW~upH9yzw7svXkf3dJQDK7B3~6eC7z40Mso6SEaBaQGHgDPHxJ2MByO2G1GBPZzSm5Miyp8ekzbnSMjW4ieK04omh0LYvR84aC7tezWrHJTV6~VrjGtNsgi3Neolvw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87197742&amp;ar=2754544&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu01705.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>sI:A</em> distribution within the major phylums of the eukaryotic and bacterial domains with a significant empirical <em>P</em>-value (see details in section 2).</p></div></div></div>                            <h2 scrollto-destination=87197746 id="87197746" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>In this study, we describe a new approach for inferring the efficiency of wobble interactions in the tAI without prior knowledge about the expression levels of the analysed organism. The approach is based on the fact that in most organisms highly expressed genes have higher CUB which is, at least partially, due to selection for improved adaptation of the codons to the tRNA pool of the organism. With our approach we infer the efficiency of wobble interactions via optimizing the component of the CUB that is due to adaptation to the tRNA pool (i.e. the correlation between these two measures: CUB and adaptation to the tRNA pool).</p><p>Thus, one limitation of our approach (and other CUB-based approaches) is the fact that it will not work in the case of organisms with no strong enough selection for both CUB and the adaptation to the tRNA pool in highly expressed genes; specifically, we assume that the evolutionary selection for this two phenomena tend to be stronger when the gene expression is higher.</p><p>In addition, we show that with our approach we are able to infer the efficiency of wobble interactions in non-fungal organisms better than the conventional approach (the tAI that does not optimize these values for each organism separately). In addition, we provide the estimations of these values for 100 organisms and show that they vary among different organism and correlate with evolutionary proximity. We report the similarities and differences among the inferred efficiencies of the analysed organisms.</p><p>PA measurements rather than mRNA level measurements are more appropriate for estimating the extent to which a coding sequence feature is related to translation efficiency. Thus, the improved correlation between stAI and PA exhibited for the non-fungal model organisms relatively to the correlation between tAI and PA demonstrates the advantages of our novel approach. Specifically, the improved correlation between stAI and PA indicates a strong association between translation efficiency (and thus PA), and the combined information the stAI provides which includes the co-adaptation of CUB to the tRNA pool, and the efficiency of the different wobble interactions.</p><p>Currently, there are less than a few dozen large scale measurements of protein levels, while there are &gt;25,000 sequenced genomes. In addition, in the case of most of the organisms on earth, it is much easier to sequence their genomes, while it is usually impossible to culture them in order to measure their protein levels (see, for example,<sup><a reveal-id="DSU017C65" data-open="DSU017C65" class="link link-ref link-reveal xref-bibr">65</a></sup>). Our approach can improve the study of translation and evolution in such organisms, even if there are no available gene expression measurements.</p><p>The idea of different domains having different wobble <em>S<sub>ij</sub>-</em>values is supportive with the successful significant clustering reported in this study. The differences between the bacterial and eukaryotic ribosomes<sup><a reveal-id="DSU017C66" data-open="DSU017C66" class="link link-ref link-reveal xref-bibr">66</a>,<a reveal-id="DSU017C67" data-open="DSU017C67" class="link link-ref link-reveal xref-bibr">67</a></sup> might provide a plausible explanation to this result as specific physical, chemical, and geometrical constraints are imposed on each tRNA–codon interaction. In the budding yeast, for example, the wobble inosine tRNA modification is essential for viability.<sup><a reveal-id="DSU017C40" data-open="DSU017C40" class="link link-ref link-reveal xref-bibr">40</a></sup> This result is in line with a recent study<sup><a reveal-id="DSU017C68" data-open="DSU017C68" class="link link-ref link-reveal xref-bibr">68</a></sup> that two kingdom-specific tRNA modifications are major contributors that separate archaeal, bacterial, and eukaryal genomes in terms of their tRNA gene composition. Specifically, with our approach, we were able to provide information about the interaction efficiencies that tend to vary among the different domains (<em>sU:G</em>, <em>sI:A</em>, and <em>sG:U</em>) and within some of the domains (<em>sI:A</em>); in addition, we show that the efficiencies of some of the interactions are conserved in all the domains (<em>sI:C</em>). Combining this information with additional information such as phylogenetic analysis, three dimensional conformations of the ribosome and tRNA molecules and knowledge related to tRNA modifications can provide a better understanding of the exact structure of the ribosome and tRNA molecules, their biochemical interactions, and their evolution.</p><p>We further verified that modelling non-conventional interactions between nucleotides does not significantly improve our model. Thus, our analysis supports the conjecture that, in the analysed organisms, wobble/WC interactions/parameters that appear in the original tAI measure should not be updated.</p><p>Finally, there has been a debate about the causality of the tRNA adaptation/protein level relations. Some previous studies suggested that increasing the adaptation to the tRNA pool has direct effect on translation rate and thus on protein levels.<sup><a reveal-id="DSU017C34" data-open="DSU017C34" class="link link-ref link-reveal xref-bibr">34</a>,<a reveal-id="DSU017C69" data-open="DSU017C69" class="link link-ref link-reveal xref-bibr">69</a></sup> However, other studies have suggested that this relation is not causal: endogenous highly expressed genes have higher adaptation to the tRNA pool via reasons that are not directly related to the translation rate.<sup><a reveal-id="DSU017C62" data-open="DSU017C62" class="link link-ref link-reveal xref-bibr">62</a>,<a reveal-id="DSU017C70" data-open="DSU017C70" class="link link-ref link-reveal xref-bibr">70</a></sup> For example, it has been suggested that the adaption of highly expressed genes to the tRNA pool improves the global ribosomal allocation among genes based on the fact that genes with higher adaptation to the tRNA pool consume less ribosomes;<sup><a reveal-id="DSU017C9" data-open="DSU017C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSU017C70" data-open="DSU017C70" class="link link-ref link-reveal xref-bibr">70</a>,<a reveal-id="DSU017C71" data-open="DSU017C71" class="link link-ref link-reveal xref-bibr">71</a></sup> it was also suggested that evolution maintains a balance between codon frequency and the cellular levels of the tRNA genes such that the actual translation elongation speed is constant;<sup><a reveal-id="DSU017C62" data-open="DSU017C62" class="link link-ref link-reveal xref-bibr">62</a></sup> highly expressed genes have higher adaptation to the tRNA pool since the effect of these genes on maintaining this balance is higher than in the case of lowly expressed genes.<sup><a reveal-id="DSU017C62" data-open="DSU017C62" class="link link-ref link-reveal xref-bibr">62</a></sup> It is important to mention that the success of our approach is robust to the outcome of this debate. The fact that highly expressed genes have higher adaptation to the tRNA pool as reflected by the wobble interactions and the cellular tRNA levels is enough for the success of our approach, the exact biophysical/evolutionary mechanism does not matter.</p>                            <h2 scrollto-destination=87197756 id="87197756" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This study was supported in part by a fellowship from the Edmond J. Safra Center for Bioinformatics at Tel-Aviv University.</p>                            <h2 scrollto-destination=87197758 id="87197758" class="backacknowledgements-title" >Acknowledgement</h2>
<p>We thank Ms Hadas Zur for helpful discussions and comments.</p>                            <h2 scrollto-destination=87197760 id="87197760" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSU017C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kimura</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="article-title">Genetic variability maintained in a finite population due to mutational production of neutral and nearly neutral isoalleles</div>
          <div class="source">Genet. Res.</div> 
          <div class="year">1968</div>
          <div class="volume">11</div>
          <div class="fpage">247</div>
          <div class="lpage">69</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20variability%20maintained%20in%20a%20finite%20population%20due%20to%20mutational%20production%20of%20neutral%20and%20nearly%20neutral%20isoalleles&amp;author=M.%20Kimura&amp;publication_year=1968&amp;journal=Genet.%20Res.&amp;volume=11&amp;pages=247-69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S0016672300011459" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS0016672300011459" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS0016672300011459"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5713805" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Grantham</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Gautier</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Gouy</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Mercier</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Pave</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="article-title">Codon catalog usage and the genome hypothesis</div>
          <div class="source">Nucleic Acids Res.</div> 
          <div class="year">1980</div>
          <div class="volume">8</div>
          <div class="fpage">197</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20catalog%20usage%20and%20the%20genome%20hypothesis&amp;author=R.%20Grantham&amp;author=C.%20Gautier&amp;author=M.%20Gouy&amp;author=R.%20Mercier&amp;author=A.%20Pave&amp;publication_year=1980&amp;journal=Nucleic%20Acids%20Res.&amp;volume=8&amp;pages=197" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/8.1.197-c" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F8.1.197-c" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F8.1.197-c"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6766552" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Plotkin</div>
               <div class="given-names">J.B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kudla</div>
               <div class="given-names">G.</div>
            </div>
          </div>
          <div class="article-title">Synonymous but not the same: the causes and consequences of codon bias</div>
          <div class="source">Nat. Rev. Genet.</div> 
          <div class="year">2010</div>
          <div class="volume">12</div>
          <div class="fpage">32</div>
          <div class="lpage">42</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20but%20not%20the%20same%3A%20the%20causes%20and%20consequences%20of%20codon%20bias&amp;author=J.B.%20Plotkin&amp;author=G.%20Kudla&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=12&amp;pages=32-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2899" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2899" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2899"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21102527" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Behura</div>
               <div class="given-names">S.K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Severson</div>
               <div class="given-names">D.W.</div>
            </div>
          </div>
          <div class="article-title">Codon usage bias: causative factors, quantification methods and genome-wide patterns: with emphasis on insect genomes</div>
          <div class="source">Biol. Rev. Camb. Philos. Soc.</div> 
          <div class="year">2013</div>
          <div class="volume">88</div>
          <div class="fpage">49</div>
          <div class="lpage">61</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20bias%3A%20causative%20factors%2C%20quantification%20methods%20and%20genome-wide%20patterns%3A%20with%20emphasis%20on%20insect%20genomes&amp;author=S.K.%20Behura&amp;author=D.W.%20Severson&amp;publication_year=2013&amp;journal=Biol.%20Rev.%20Camb.%20Philos.%20Soc.&amp;volume=88&amp;pages=49-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/brv.2013.88.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fbrv.2013.88.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fbrv.2013.88.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22889422" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Behura</div>
               <div class="given-names">S.K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Severson</div>
               <div class="given-names">D.W.</div>
            </div>
          </div>
          <div class="article-title">Coadaptation of isoacceptor tRNA genes and codon usage bias for translation efficiency in <em>Aedes aegypti</em> and <em>Anopheles gambiae</em></div>
          <div class="source">Insect Mol. Biol.</div> 
          <div class="year">2011</div>
          <div class="volume">20</div>
          <div class="fpage">177</div>
          <div class="lpage">87</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coadaptation%20of%20isoacceptor%20tRNA%20genes%20and%20codon%20usage%20bias%20for%20translation%20efficiency%20in%20Aedes%20aegypti%20and%20Anopheles%20gambiae&amp;author=S.K.%20Behura&amp;author=D.W.%20Severson&amp;publication_year=2011&amp;journal=Insect%20Mol.%20Biol.&amp;volume=20&amp;pages=177-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/imb.2011.20.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fimb.2011.20.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fimb.2011.20.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21040044" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Behura</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Stanke</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Desjardins</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Werren</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Severson</div>
               <div class="given-names">D.</div>
            </div>
          </div>
          <div class="article-title">Comparative analysis of nuclear tRNA genes of <em>Nasonia vitripennis</em> and other arthropods, and relationships to codon usage bias</div>
          <div class="source">Insect Mol. Biol.</div> 
          <div class="year">2010</div>
          <div class="volume">19</div>
          <div class="fpage">49</div>
          <div class="lpage">58</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20nuclear%20tRNA%20genes%20of%20Nasonia%20vitripennis%20and%20other%20arthropods%2C%20and%20relationships%20to%20codon%20usage%20bias&amp;author=S.%20Behura&amp;author=M.%20Stanke&amp;author=C.%20Desjardins&amp;author=J.%20Werren&amp;author=D.%20Severson&amp;publication_year=2010&amp;journal=Insect%20Mol.%20Biol.&amp;volume=19&amp;pages=49-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/imb.2010.19.issue-s1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fimb.2010.19.issue-s1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fimb.2010.19.issue-s1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20167017" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Stergachis</div>
               <div class="given-names">A.B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Haugen</div>
               <div class="given-names">E.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Shafer</div>
               <div class="given-names">A.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Exonic transcription factor binding directs codon choice and affects protein evolution</div>
          <div class="source">Science</div> 
          <div class="year">2013</div>
          <div class="volume">342</div>
          <div class="fpage">1367</div>
          <div class="lpage">72</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exonic%20transcription%20factor%20binding%20directs%20codon%20choice%20and%20affects%20protein%20evolution&amp;author=A.B.%20Stergachis&amp;author=E.%20Haugen&amp;author=A.%20Shafer&amp;publication_year=2013&amp;journal=Science&amp;volume=342&amp;pages=1367-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1243490" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1243490" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1243490"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24337295" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">dos Reis</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Savva</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wernisch</div>
               <div class="given-names">L.</div>
            </div>
          </div>
          <div class="article-title">Solving the riddle of codon usage preferences: a test for translational selection</div>
          <div class="source">Nucleic Acids Res.</div> 
          <div class="year">2004</div>
          <div class="volume">32</div>
          <div class="fpage">5036</div>
          <div class="lpage">44</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Solving%20the%20riddle%20of%20codon%20usage%20preferences%3A%20a%20test%20for%20translational%20selection&amp;author=M.%20dos%20Reis&amp;author=R.%20Savva&amp;author=L.%20Wernisch&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=5036-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh834" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh834" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh834"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15448185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tuller</div>
               <div class="given-names">T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Waldman</div>
               <div class="given-names">Y.Y.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kupiec</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ruppin</div>
               <div class="given-names">E.</div>
            </div>
          </div>
          <div class="article-title">Translation efficiency is determined by both codon bias and folding energy</div>
          <div class="source">Proc. Natl. Acad. Sci. USA</div> 
          <div class="year">2010</div>
          <div class="volume">107</div>
          <div class="fpage">3645</div>
          <div class="lpage">50</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translation%20efficiency%20is%20determined%20by%20both%20codon%20bias%20and%20folding%20energy&amp;author=T.%20Tuller&amp;author=Y.Y.%20Waldman&amp;author=M.%20Kupiec&amp;author=E.%20Ruppin&amp;publication_year=2010&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=3645-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0909910107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0909910107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0909910107"> </span></div> </div></div></div></div></div><div content-id="DSU017C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Arenas</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Posada</div>
               <div class="given-names">D.</div>
            </div>
          </div>
          <div class="article-title">Coalescent simulation of intracodon recombination</div>
          <div class="source">Genetics</div> 
          <div class="year">2010</div>
          <div class="volume">184</div>
          <div class="fpage">429</div>
          <div class="lpage">37</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coalescent%20simulation%20of%20intracodon%20recombination&amp;author=M.%20Arenas&amp;author=D.%20Posada&amp;publication_year=2010&amp;journal=Genetics&amp;volume=184&amp;pages=429-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.109.109736" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.109.109736" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.109.109736"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19933876" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bennetzen</div>
               <div class="given-names">J.L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Hall</div>
               <div class="given-names">B.</div>
            </div>
          </div>
          <div class="article-title">Codon selection in yeast</div>
          <div class="source">J. Biol. Chem.</div> 
          <div class="year">1982</div>
          <div class="volume">257</div>
          <div class="fpage">3026</div>
          <div class="lpage">31</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20selection%20in%20yeast&amp;author=J.L.%20Bennetzen&amp;author=B.%20Hall&amp;publication_year=1982&amp;journal=J.%20Biol.%20Chem.&amp;volume=257&amp;pages=3026-31" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7037777" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Grosjean</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Fiers</div>
               <div class="given-names">W.</div>
            </div>
          </div>
          <div class="article-title">Preferential codon usage in prokaryotic genes: the optimal codon-anticodon interaction energy and the selective codon usage in efficiently expressed genes</div>
          <div class="source">Gene</div> 
          <div class="year">1982</div>
          <div class="volume">18</div>
          <div class="fpage">199</div>
          <div class="lpage">209</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Preferential%20codon%20usage%20in%20prokaryotic%20genes%3A%20the%20optimal%20codon-anticodon%20interaction%20energy%20and%20the%20selective%20codon%20usage%20in%20efficiently%20expressed%20genes&amp;author=H.%20Grosjean&amp;author=W.%20Fiers&amp;publication_year=1982&amp;journal=Gene&amp;volume=18&amp;pages=199-209" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(82)90157-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(82)90157-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(82)90157-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6751939" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wright</div>
               <div class="given-names">F.</div>
            </div>
          </div>
          <div class="article-title">The ‘effective number of codons’ used in a gene</div>
          <div class="source">Gene</div> 
          <div class="year">1990</div>
          <div class="volume">87</div>
          <div class="fpage">23</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20%E2%80%98effective%20number%20of%20codons%E2%80%99%20used%20in%20a%20gene&amp;author=F.%20Wright&amp;publication_year=1990&amp;journal=Gene&amp;volume=87&amp;pages=23-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(90)90491-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(90)90491-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(90)90491-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2110097" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kurland</div>
               <div class="given-names">C.</div>
            </div>
          </div>
          <div class="article-title">Codon bias and gene expression</div>
          <div class="source">FEBS Lett.</div> 
          <div class="year">1991</div>
          <div class="volume">285</div>
          <div class="fpage">165</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20bias%20and%20gene%20expression&amp;author=C.%20Kurland&amp;publication_year=1991&amp;journal=FEBS%20Lett.&amp;volume=285&amp;pages=165-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0014-5793(91)80797-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0014-5793(91)80797-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0014-5793(91)80797-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1855585" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Stenico</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Lloyd</div>
               <div class="given-names">A.T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Sharp</div>
               <div class="given-names">P.M.</div>
            </div>
          </div>
          <div class="article-title">Codon usage in <em>Caenorhabditis elegans</em>: delineation of translational selection and mutational biases</div>
          <div class="source">Nucleic Acids Res.</div> 
          <div class="year">1994</div>
          <div class="volume">22</div>
          <div class="fpage">2437</div>
          <div class="lpage">46</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20in%20Caenorhabditis%20elegans%3A%20delineation%20of%20translational%20selection%20and%20mutational%20biases&amp;author=M.%20Stenico&amp;author=A.T.%20Lloyd&amp;author=P.M.%20Sharp&amp;publication_year=1994&amp;journal=Nucleic%20Acids%20Res.&amp;volume=22&amp;pages=2437-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/22.13.2437" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F22.13.2437" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F22.13.2437"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8041603" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sharp</div>
               <div class="given-names">P.M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">W.H.</div>
            </div>
          </div>
          <div class="article-title">The codon adaptation index-a measure of directional synonymous codon usage bias, and its potential applications</div>
          <div class="source">Nucleic Acids Res.</div> 
          <div class="year">1987</div>
          <div class="volume">15</div>
          <div class="fpage">1281</div>
          <div class="lpage">95</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20codon%20adaptation%20index-a%20measure%20of%20directional%20synonymous%20codon%20usage%20bias%2C%20and%20its%20potential%20applications&amp;author=P.M.%20Sharp&amp;author=W.H.%20Li&amp;publication_year=1987&amp;journal=Nucleic%20Acids%20Res.&amp;volume=15&amp;pages=1281-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/15.3.1281" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F15.3.1281" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F15.3.1281"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3547335" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ishihama</div>
               <div class="given-names">Y.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Schmidt</div>
               <div class="given-names">T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Rappsilber</div>
               <div class="given-names">J.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Protein abundance profiling of the <em>Escherichia coli</em> cytosol</div>
          <div class="source">BMC Genomics</div> 
          <div class="year">2008</div>
          <div class="volume">9</div>
          <div class="fpage">102</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20abundance%20profiling%20of%20the%20Escherichia%20coli%20cytosol&amp;author=Y.%20Ishihama&amp;author=T.%20Schmidt&amp;author=J.%20Rappsilber&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18304323" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ghaemmaghami</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Huh</div>
               <div class="given-names">W.K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Bower</div>
               <div class="given-names">K.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Global analysis of protein expression in yeast</div>
          <div class="source">Nature</div> 
          <div class="year">2003</div>
          <div class="volume">425</div>
          <div class="fpage">737</div>
          <div class="lpage">41</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20analysis%20of%20protein%20expression%20in%20yeast&amp;author=S.%20Ghaemmaghami&amp;author=W.K.%20Huh&amp;author=K.%20Bower&amp;publication_year=2003&amp;journal=Nature&amp;volume=425&amp;pages=737-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature02046" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature02046" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature02046"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14562106" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Washburn</div>
               <div class="given-names">M.P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wolters</div>
               <div class="given-names">D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Yates</div>
               <div class="given-names">J.R.</div>
            </div>
          </div>
          <div class="article-title">Large-scale analysis of the yeast proteome by multidimensional protein identification technology</div>
          <div class="source">Nat. Biotechnol.</div> 
          <div class="year">2001</div>
          <div class="volume">19</div>
          <div class="fpage">242</div>
          <div class="lpage">7</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20analysis%20of%20the%20yeast%20proteome%20by%20multidimensional%20protein%20identification%20technology&amp;author=M.P.%20Washburn&amp;author=D.%20Wolters&amp;author=J.R.%20Yates&amp;publication_year=2001&amp;journal=Nat.%20Biotechnol.&amp;volume=19&amp;pages=242-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/85686" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F85686" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F85686"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11231557" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Drummond</div>
               <div class="given-names">D.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Bloom</div>
               <div class="given-names">J.D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Adami</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wilke</div>
               <div class="given-names">C.O.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Arnold</div>
               <div class="given-names">F.H.</div>
            </div>
          </div>
          <div class="article-title">Why highly expressed proteins evolve slowly</div>
          <div class="source">Proc. Natl. Acad. Sci. USA</div> 
          <div class="year">2005</div>
          <div class="volume">102</div>
          <div class="fpage">14338</div>
          <div class="lpage">43</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Why%20highly%20expressed%20proteins%20evolve%20slowly&amp;author=D.A.%20Drummond&amp;author=J.D.%20Bloom&amp;author=C.%20Adami&amp;author=C.O.%20Wilke&amp;author=F.H.%20Arnold&amp;publication_year=2005&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=14338-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0504070102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0504070102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0504070102"> </span></div> </div></div></div></div></div><div content-id="DSU017C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Blattner</div>
               <div class="given-names">F.R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Plunkett</div>
               <div class="given-names">G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Bloch</div>
               <div class="given-names">C.A.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">The complete genome sequence of <em>Escherichia coli</em> K-12</div>
          <div class="source">Science</div> 
          <div class="year">1997</div>
          <div class="volume">277</div>
          <div class="fpage">1453</div>
          <div class="lpage">62</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20genome%20sequence%20of%20Escherichia%20coli%20K-12&amp;author=F.R.%20Blattner&amp;author=G.%20Plunkett&amp;author=C.A.%20Bloch&amp;publication_year=1997&amp;journal=Science&amp;volume=277&amp;pages=1453-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.277.5331.1453" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.277.5331.1453" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.277.5331.1453"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9278503" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kimchi-Sarfaty</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Oh</div>
               <div class="given-names">J.M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kim</div>
               <div class="given-names">I.W.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">A ‘silent’ polymorphism in the MDR1 gene changes substrate specificity</div>
          <div class="source">Science</div> 
          <div class="year">2007</div>
          <div class="volume">315</div>
          <div class="fpage">525</div>
          <div class="lpage">8</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20%E2%80%98silent%E2%80%99%20polymorphism%20in%20the%20MDR1%20gene%20changes%20substrate%20specificity&amp;author=C.%20Kimchi-Sarfaty&amp;author=J.M.%20Oh&amp;author=I.W.%20Kim&amp;publication_year=2007&amp;journal=Science&amp;volume=315&amp;pages=525-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1135308" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1135308" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1135308"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17185560" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fox</div>
               <div class="given-names">J.M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Erill</div>
               <div class="given-names">I.</div>
            </div>
          </div>
          <div class="article-title">Relative codon adaptation: a generic codon bias index for prediction of gene expression</div>
          <div class="source">DNA Res.</div> 
          <div class="year">2010</div>
          <div class="volume">17</div>
          <div class="fpage">185</div>
          <div class="lpage">96</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Relative%20codon%20adaptation%3A%20a%20generic%20codon%20bias%20index%20for%20prediction%20of%20gene%20expression&amp;author=J.M.%20Fox&amp;author=I.%20Erill&amp;publication_year=2010&amp;journal=DNA%20Res.&amp;volume=17&amp;pages=185-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsq012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsq012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsq012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20453079" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ikemura</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="article-title">Correlation between the abundance of <em>Escherichia coli</em> transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the <em>E. coli</em> translational system</div>
          <div class="source">J. Mol. Biol.</div> 
          <div class="year">1981</div>
          <div class="volume">151</div>
          <div class="fpage">389</div>
          <div class="lpage">409</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Correlation%20between%20the%20abundance%20of%20Escherichia%20coli%20transfer%20RNAs%20and%20the%20occurrence%20of%20the%20respective%20codons%20in%20its%20protein%20genes%3A%20a%20proposal%20for%20a%20synonymous%20codon%20choice%20that%20is%20optimal%20for%20the%20E.%20coli%20translational%20system&amp;author=T.%20Ikemura&amp;publication_year=1981&amp;journal=J.%20Mol.%20Biol.&amp;volume=151&amp;pages=389-409" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(81)90003-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(81)90003-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(81)90003-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6175758" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ikemura</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="article-title">Codon usage and tRNA content in unicellular and multicellular organisms</div>
          <div class="source">Mol. Biol. Evol.</div> 
          <div class="year">1985</div>
          <div class="volume">2</div>
          <div class="fpage">13</div>
          <div class="lpage">34</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20and%20tRNA%20content%20in%20unicellular%20and%20multicellular%20organisms&amp;author=T.%20Ikemura&amp;publication_year=1985&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=2&amp;pages=13-34" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3916708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Seffens</div>
               <div class="given-names">W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Digby</div>
               <div class="given-names">D.</div>
            </div>
          </div>
          <div class="article-title">mRNAs have greater negative folding free energies than shuffled or codon choice randomized sequences</div>
          <div class="source">Nucleic Acids Res.</div> 
          <div class="year">1999</div>
          <div class="volume">27</div>
          <div class="fpage">1578</div>
          <div class="lpage">84</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=mRNAs%20have%20greater%20negative%20folding%20free%20energies%20than%20shuffled%20or%20codon%20choice%20randomized%20sequences&amp;author=W.%20Seffens&amp;author=D.%20Digby&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=1578-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.7.1578" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.7.1578" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.7.1578"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10075987" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ohama</div>
               <div class="given-names">T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Muto</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Osawa</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="article-title">Role of GC-biased mutation pressure on synonymous codon choice in <em>Micrococcus luteus</em> a bacterium with a high genomic GC-content</div>
          <div class="source">Nucleic Acids Res.</div> 
          <div class="year">1990</div>
          <div class="volume">18</div>
          <div class="fpage">1565</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20GC-biased%20mutation%20pressure%20on%20synonymous%20codon%20choice%20in%20Micrococcus%20luteus%20a%20bacterium%20with%20a%20high%20genomic%20GC-content&amp;author=T.%20Ohama&amp;author=A.%20Muto&amp;author=S.%20Osawa&amp;publication_year=1990&amp;journal=Nucleic%20Acids%20Res.&amp;volume=18&amp;pages=1565-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/18.6.1565" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F18.6.1565" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F18.6.1565"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2326195" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ikemura</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="article-title">Correlation between the abundance of yeast transfer RNAs and the occurrence of the respective codons in protein genes: differences in synonymous codon choice patterns of yeast and <em>Escherichia coli</em> with reference to the abundance of isoaccepting transfer RNAs</div>
          <div class="source">J. Mol. Biol.</div> 
          <div class="year">1982</div>
          <div class="volume">158</div>
          <div class="fpage">573</div>
          <div class="lpage">97</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Correlation%20between%20the%20abundance%20of%20yeast%20transfer%20RNAs%20and%20the%20occurrence%20of%20the%20respective%20codons%20in%20protein%20genes%3A%20differences%20in%20synonymous%20codon%20choice%20patterns%20of%20yeast%20and%20Escherichia%20coli%20with%20reference%20to%20the%20abundance%20of%20isoaccepting%20transfer%20RNAs&amp;author=T.%20Ikemura&amp;publication_year=1982&amp;journal=J.%20Mol.%20Biol.&amp;volume=158&amp;pages=573-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(82)90250-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(82)90250-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(82)90250-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6750137" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Percudani</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Pavesi</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ottonello</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="article-title">Transfer RNA gene redundancy and translational selection in <em>Saccharomyces cerevisiae</em></div>
          <div class="source">J. Mol. Biol.</div> 
          <div class="year">1997</div>
          <div class="volume">268</div>
          <div class="fpage">322</div>
          <div class="lpage">30</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transfer%20RNA%20gene%20redundancy%20and%20translational%20selection%20in%20Saccharomyces%20cerevisiae&amp;author=R.%20Percudani&amp;author=A.%20Pavesi&amp;author=S.%20Ottonello&amp;publication_year=1997&amp;journal=J.%20Mol.%20Biol.&amp;volume=268&amp;pages=322-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1997.0942" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1997.0942" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1997.0942"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9159473" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Man</div>
               <div class="given-names">O.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Pilpel</div>
               <div class="given-names">Y.</div>
            </div>
          </div>
          <div class="article-title">Differential translation efficiency of orthologous genes is involved in phenotypic divergence of yeast species</div>
          <div class="source">Nat. Genet.</div> 
          <div class="year">2007</div>
          <div class="volume">39</div>
          <div class="fpage">415</div>
          <div class="lpage">21</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20translation%20efficiency%20of%20orthologous%20genes%20is%20involved%20in%20phenotypic%20divergence%20of%20yeast%20species&amp;author=O.%20Man&amp;author=Y.%20Pilpel&amp;publication_year=2007&amp;journal=Nat.%20Genet.&amp;volume=39&amp;pages=415-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1967" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1967" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1967"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17277776" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sharp</div>
               <div class="given-names">P.M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Bailes</div>
               <div class="given-names">E.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Grocock</div>
               <div class="given-names">R.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Peden</div>
               <div class="given-names">J.F.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Sockett</div>
               <div class="given-names">R.E.</div>
            </div>
          </div>
          <div class="article-title">Variation in the strength of selected codon usage bias among bacteria</div>
          <div class="source">Nucleic Acids Res.</div> 
          <div class="year">2005</div>
          <div class="volume">33</div>
          <div class="fpage">1141</div>
          <div class="lpage">53</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variation%20in%20the%20strength%20of%20selected%20codon%20usage%20bias%20among%20bacteria&amp;author=P.M.%20Sharp&amp;author=E.%20Bailes&amp;author=R.J.%20Grocock&amp;author=J.F.%20Peden&amp;author=R.E.%20Sockett&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=1141-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki242" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki242" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki242"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15728743" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chamary</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Parmley</div>
               <div class="given-names">J.L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Hurst</div>
               <div class="given-names">L.D.</div>
            </div>
          </div>
          <div class="article-title">Hearing silence: non-neutral evolution at synonymous sites in mammals</div>
          <div class="source">Nat. Rev. Genet.</div> 
          <div class="year">2006</div>
          <div class="volume">7</div>
          <div class="fpage">98</div>
          <div class="lpage">108</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hearing%20silence%3A%20non-neutral%20evolution%20at%20synonymous%20sites%20in%20mammals&amp;author=J.%20Chamary&amp;author=J.L.%20Parmley&amp;author=L.D.%20Hurst&amp;publication_year=2006&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=7&amp;pages=98-108" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1770" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1770" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1770"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16418745" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shabalina</div>
               <div class="given-names">S.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ogurtsov</div>
               <div class="given-names">A.Y.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Spiridonov</div>
               <div class="given-names">N.A.</div>
            </div>
          </div>
          <div class="article-title">A periodic pattern of mRNA secondary structure created by the genetic code</div>
          <div class="source">Nucleic Acids Res.</div> 
          <div class="year">2006</div>
          <div class="volume">34</div>
          <div class="fpage">2428</div>
          <div class="lpage">37</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20periodic%20pattern%20of%20mRNA%20secondary%20structure%20created%20by%20the%20genetic%20code&amp;author=S.A.%20Shabalina&amp;author=A.Y.%20Ogurtsov&amp;author=N.A.%20Spiridonov&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=2428-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl287" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl287" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl287"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16682450" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tuller</div>
               <div class="given-names">T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Carmi</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Vestsigian</div>
               <div class="given-names">K.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">An evolutionarily conserved mechanism for controlling the efficiency of protein translation</div>
          <div class="source">Cell</div> 
          <div class="year">2010</div>
          <div class="volume">141</div>
          <div class="fpage">344</div>
          <div class="lpage">54</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20evolutionarily%20conserved%20mechanism%20for%20controlling%20the%20efficiency%20of%20protein%20translation&amp;author=T.%20Tuller&amp;author=A.%20Carmi&amp;author=K.%20Vestsigian&amp;publication_year=2010&amp;journal=Cell&amp;volume=141&amp;pages=344-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2010.03.031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2010.03.031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2010.03.031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20403328" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ingolia</div>
               <div class="given-names">N.T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Lareau</div>
               <div class="given-names">L.F.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Weissman</div>
               <div class="given-names">J.S.</div>
            </div>
          </div>
          <div class="article-title">Ribosome profiling of mouse embryonic stem cells reveals the complexity and dynamics of mammalian proteomes</div>
          <div class="source">Cell</div> 
          <div class="year">2011</div>
          <div class="volume">147</div>
          <div class="fpage">789</div>
          <div class="lpage">802</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ribosome%20profiling%20of%20mouse%20embryonic%20stem%20cells%20reveals%20the%20complexity%20and%20dynamics%20of%20mammalian%20proteomes&amp;author=N.T.%20Ingolia&amp;author=L.F.%20Lareau&amp;author=J.S.%20Weissman&amp;publication_year=2011&amp;journal=Cell&amp;volume=147&amp;pages=789-802" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2011.10.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2011.10.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2011.10.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22056041" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kanaya</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Yamada</div>
               <div class="given-names">Y.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kudo</div>
               <div class="given-names">Y.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ikemura</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="article-title">Studies of codon usage and tRNA genes of 18 unicellular organisms and quantification of <em>Bacillus subtilis</em> tRNAs: gene expression level and species-specific diversity of codon usage based on multivariate analysis</div>
          <div class="source">Gene</div> 
          <div class="year">1999</div>
          <div class="volume">238</div>
          <div class="fpage">143</div>
          <div class="lpage">55</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Studies%20of%20codon%20usage%20and%20tRNA%20genes%20of%2018%20unicellular%20organisms%20and%20quantification%20of%20Bacillus%20subtilis%20tRNAs%3A%20gene%20expression%20level%20and%20species-specific%20diversity%20of%20codon%20usage%20based%20on%20multivariate%20analysis&amp;author=S.%20Kanaya&amp;author=Y.%20Yamada&amp;author=Y.%20Kudo&amp;author=T.%20Ikemura&amp;publication_year=1999&amp;journal=Gene&amp;volume=238&amp;pages=143-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(99)00225-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(99)00225-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(99)00225-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10570992" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Crick</div>
               <div class="given-names">F.</div>
            </div>
          </div>
          <div class="article-title">Codon-anticodon pairing: the wobble hypothesis</div>
          <div class="source">J. Mol. Biol.</div> 
          <div class="year">1966</div>
          <div class="volume">19</div>
          <div class="fpage">548</div>
          <div class="lpage">55</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon-anticodon%20pairing%3A%20the%20wobble%20hypothesis&amp;author=F.%20Crick&amp;publication_year=1966&amp;journal=J.%20Mol.%20Biol.&amp;volume=19&amp;pages=548-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(66)80022-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(66)80022-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(66)80022-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5969078" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">RajBhandary</div>
               <div class="given-names">U.L.</div>
            </div>
          </div>
          <div class="article-title">Genetic code. Modified bases and aminoacylation</div>
          <div class="source">Nature</div> 
          <div class="year">1988</div>
          <div class="volume">336</div>
          <div class="fpage">112</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20code.%20Modified%20bases%20and%20aminoacylation&amp;author=U.L.%20RajBhandary&amp;publication_year=1988&amp;journal=Nature&amp;volume=336&amp;pages=112" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/336112a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F336112a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F336112a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3185729" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Agris</div>
               <div class="given-names">P.</div>
            </div>
          </div>
          <div class="article-title">Wobble position modified nucleosides evolved to select transfer RNA codon recognition: a modified-wobble hypothesis</div>
          <div class="source">Biochimie</div> 
          <div class="year">1991</div>
          <div class="volume">73</div>
          <div class="fpage">1345</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Wobble%20position%20modified%20nucleosides%20evolved%20to%20select%20transfer%20RNA%20codon%20recognition%3A%20a%20modified-wobble%20hypothesis&amp;author=P.%20Agris&amp;publication_year=1991&amp;journal=Biochimie&amp;volume=73&amp;pages=1345-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0300-9084(91)90163-U" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0300-9084(91)90163-U" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0300-9084(91)90163-U"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1799628" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tsutsumi</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Sugiura</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ma</div>
               <div class="given-names">Y.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Wobble inosine tRNA modification is essential to cell cycle progression in G1/S and G2/M transitions in fission yeast</div>
          <div class="source">J. Biol. Chem.</div> 
          <div class="year">2007</div>
          <div class="volume">282</div>
          <div class="fpage">33459</div>
          <div class="lpage">65</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Wobble%20inosine%20tRNA%20modification%20is%20essential%20to%20cell%20cycle%20progression%20in%20G1%2FS%20and%20G2%2FM%20transitions%20in%20fission%20yeast&amp;author=S.%20Tsutsumi&amp;author=R.%20Sugiura&amp;author=Y.%20Ma&amp;publication_year=2007&amp;journal=J.%20Biol.%20Chem.&amp;volume=282&amp;pages=33459-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M706869200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M706869200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M706869200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17875641" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wolf</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Gerber</div>
               <div class="given-names">A.P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Keller</div>
               <div class="given-names">W.</div>
            </div>
          </div>
          <div class="article-title">tadA, an essential tRNA-specific adenosine deaminase from <em>Escherichia coli</em></div>
          <div class="source">EMBO J.</div> 
          <div class="year">2002</div>
          <div class="volume">21</div>
          <div class="fpage">3841</div>
          <div class="lpage">51</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tadA%2C%20an%20essential%20tRNA-specific%20adenosine%20deaminase%20from%20Escherichia%20coli&amp;author=J.%20Wolf&amp;author=A.P.%20Gerber&amp;author=W.%20Keller&amp;publication_year=2002&amp;journal=EMBO%20J.&amp;volume=21&amp;pages=3841-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/emboj/cdf362" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Femboj%2Fcdf362" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Femboj%2Fcdf362"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12110595" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Delannoy</div>
               <div class="given-names">E.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Le Ret</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Faivre-Nitschke</div>
               <div class="given-names">E.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Arabidopsis tRNA adenosine deaminase arginine edits the wobble nucleotide of chloroplast tRNAArg (ACG) and is essential for efficient chloroplast translation</div>
          <div class="source">Plant Cell</div> 
          <div class="year">2009</div>
          <div class="volume">21</div>
          <div class="fpage">2058</div>
          <div class="lpage">71</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20tRNA%20adenosine%20deaminase%20arginine%20edits%20the%20wobble%20nucleotide%20of%20chloroplast%20tRNAArg%20(ACG)%20and%20is%20essential%20for%20efficient%20chloroplast%20translation&amp;author=E.%20Delannoy&amp;author=M.%20Le%20Ret&amp;author=E.%20Faivre-Nitschke&amp;publication_year=2009&amp;journal=Plant%20Cell&amp;volume=21&amp;pages=2058-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.109.066654" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.109.066654" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.109.066654"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19602623" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Maas</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Gerber</div>
               <div class="given-names">A.P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Rich</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="article-title">Identification and characterization of a human tRNA-specific adenosine deaminase related to the ADAR family of pre-mRNA editing enzymes</div>
          <div class="source">Proc. Natl. Acad. Sci. USA</div> 
          <div class="year">1999</div>
          <div class="volume">96</div>
          <div class="fpage">8895</div>
          <div class="lpage">900</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20a%20human%20tRNA-specific%20adenosine%20deaminase%20related%20to%20the%20ADAR%20family%20of%20pre-mRNA%20editing%20enzymes&amp;author=S.%20Maas&amp;author=A.P.%20Gerber&amp;author=A.%20Rich&amp;publication_year=1999&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=96&amp;pages=8895-900" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.96.16.8895" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.96.16.8895" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.96.16.8895"> </span></div> </div></div></div></div></div><div content-id="DSU017C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ikeuchi</div>
               <div class="given-names">Y.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Soma</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ote</div>
               <div class="given-names">T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kato</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Sekine</div>
               <div class="given-names">Y.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Suzuki</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="article-title">Molecular mechanism of lysidine synthesis that determines tRNA identity and codon recognition</div>
          <div class="source">Mol. Cell</div> 
          <div class="year">2005</div>
          <div class="volume">19</div>
          <div class="fpage">235</div>
          <div class="lpage">46</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20mechanism%20of%20lysidine%20synthesis%20that%20determines%20tRNA%20identity%20and%20codon%20recognition&amp;author=Y.%20Ikeuchi&amp;author=A.%20Soma&amp;author=T.%20Ote&amp;author=J.%20Kato&amp;author=Y.%20Sekine&amp;author=T.%20Suzuki&amp;publication_year=2005&amp;journal=Mol.%20Cell&amp;volume=19&amp;pages=235-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2005.06.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2005.06.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2005.06.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16039592" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lim</div>
               <div class="given-names">V.I.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Curran</div>
               <div class="given-names">J.F.</div>
            </div>
          </div>
          <div class="article-title">Analysis of codon: anticodon interactions within the ribosome provides new insights into codon reading and the genetic code structure</div>
          <div class="source">RNA</div> 
          <div class="year">2001</div>
          <div class="volume">7</div>
          <div class="fpage">942</div>
          <div class="lpage">57</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20codon%3A%20anticodon%20interactions%20within%20the%20ribosome%20provides%20new%20insights%20into%20codon%20reading%20and%20the%20genetic%20code%20structure&amp;author=V.I.%20Lim&amp;author=J.F.%20Curran&amp;publication_year=2001&amp;journal=RNA&amp;volume=7&amp;pages=942-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S135583820100214X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS135583820100214X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS135583820100214X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11453067" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">dos Reis</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wernisch</div>
               <div class="given-names">L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Savva</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="article-title">Unexpected correlations between gene expression and codon usage bias from microarray data for the whole <em>Escherichia coli</em> K-12 genome</div>
          <div class="source">Nucleic Acids Res.</div> 
          <div class="year">2003</div>
          <div class="volume">31</div>
          <div class="fpage">6976</div>
          <div class="lpage">85</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unexpected%20correlations%20between%20gene%20expression%20and%20codon%20usage%20bias%20from%20microarray%20data%20for%20the%20whole%20Escherichia%20coli%20K-12%20genome&amp;author=M.%20dos%20Reis&amp;author=L.%20Wernisch&amp;author=R.%20Savva&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=6976-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg897" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg897" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg897"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14627830" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Roymondal</div>
               <div class="given-names">U.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Das</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Sahoo</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="article-title">Predicting gene expression level from relative codon usage bias: an application to <em>Escherichia coli</em> genome</div>
          <div class="source">DNA Res.</div> 
          <div class="year">2009</div>
          <div class="volume">16</div>
          <div class="fpage">13</div>
          <div class="lpage">30</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20gene%20expression%20level%20from%20relative%20codon%20usage%20bias%3A%20an%20application%20to%20Escherichia%20coli%20genome&amp;author=U.%20Roymondal&amp;author=S.%20Das&amp;author=S.%20Sahoo&amp;publication_year=2009&amp;journal=DNA%20Res.&amp;volume=16&amp;pages=13-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19131380" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nelder</div>
               <div class="given-names">J.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Mead</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="article-title">A simplex method for function minimization</div>
          <div class="source">Comput. J.</div> 
          <div class="year">1965</div>
          <div class="volume">7</div>
          <div class="fpage">308</div>
          <div class="lpage">13</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simplex%20method%20for%20function%20minimization&amp;author=J.A.%20Nelder&amp;author=R.%20Mead&amp;publication_year=1965&amp;journal=Comput.%20J.&amp;volume=7&amp;pages=308-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/comjnl/7.4.308" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fcomjnl%2F7.4.308" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fcomjnl%2F7.4.308"> </span></div> </div></div></div></div></div><div content-id="DSU017C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tuller</div>
               <div class="given-names">T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Birin</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Gophna</div>
               <div class="given-names">U.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kupiec</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ruppin</div>
               <div class="given-names">E.</div>
            </div>
          </div>
          <div class="article-title">Reconstructing ancestral gene content by coevolution</div>
          <div class="source">Genome Res.</div> 
          <div class="year">2010</div>
          <div class="volume">20</div>
          <div class="fpage">122</div>
          <div class="lpage">32</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reconstructing%20ancestral%20gene%20content%20by%20coevolution&amp;author=T.%20Tuller&amp;author=H.%20Birin&amp;author=U.%20Gophna&amp;author=M.%20Kupiec&amp;author=E.%20Ruppin&amp;publication_year=2010&amp;journal=Genome%20Res.&amp;volume=20&amp;pages=122-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.096115.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.096115.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.096115.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19948819" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schmidt</div>
               <div class="given-names">M.W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Houseman</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ivanov</div>
               <div class="given-names">A.R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wolf</div>
               <div class="given-names">D.A.</div>
            </div>
          </div>
          <div class="article-title">Comparative proteomic and transcriptomic profiling of the fission yeast <em>Schizosaccharomyces pombe</em></div>
          <div class="source">Mol. Syst. Biol.</div> 
          <div class="year">2007</div>
          <div class="volume">3</div>
          <div class="fpage">79</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20proteomic%20and%20transcriptomic%20profiling%20of%20the%20fission%20yeast%20Schizosaccharomyces%20pombe&amp;author=M.W.%20Schmidt&amp;author=A.%20Houseman&amp;author=A.R.%20Ivanov&amp;author=D.A.%20Wolf&amp;publication_year=2007&amp;journal=Mol.%20Syst.%20Biol.&amp;volume=3&amp;pages=79" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb4100117" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb4100117" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmsb4100117"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17299416" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mahlab</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Tuller</div>
               <div class="given-names">T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Linial</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="article-title">Conservation of the relative tRNA composition in healthy and cancerous tissues</div>
          <div class="source">RNA</div> 
          <div class="year">2012</div>
          <div class="volume">18</div>
          <div class="fpage">640</div>
          <div class="lpage">52</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conservation%20of%20the%20relative%20tRNA%20composition%20in%20healthy%20and%20cancerous%20tissues&amp;author=S.%20Mahlab&amp;author=T.%20Tuller&amp;author=M.%20Linial&amp;publication_year=2012&amp;journal=RNA&amp;volume=18&amp;pages=640-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.030775.111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.030775.111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.030775.111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22357911" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tuller</div>
               <div class="given-names">T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kupiec</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ruppin</div>
               <div class="given-names">E.</div>
            </div>
          </div>
          <div class="article-title">Determinants of protein abundance and translation efficiency in <em>S. cerevisiae</em></div>
          <div class="source">PLoS Comput. Biol.</div> 
          <div class="year">2007</div>
          <div class="volume">3</div>
          <div class="fpage">e248</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Determinants%20of%20protein%20abundance%20and%20translation%20efficiency%20in%20S.%20cerevisiae&amp;author=T.%20Tuller&amp;author=M.%20Kupiec&amp;author=E.%20Ruppin&amp;publication_year=2007&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=3&amp;pages=e248" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.0030248" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.0030248" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.0030248"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18159940" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lu</div>
               <div class="given-names">P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Vogel</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Yao</div>
               <div class="given-names">X.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Marcotte</div>
               <div class="given-names">E.M.</div>
            </div>
          </div>
          <div class="article-title">Absolute protein expression profiling estimates the relative contributions of transcriptional and translational regulation</div>
          <div class="source">Nat. Biotechnol.</div> 
          <div class="year">2006</div>
          <div class="volume">25</div>
          <div class="fpage">117</div>
          <div class="lpage">24</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Absolute%20protein%20expression%20profiling%20estimates%20the%20relative%20contributions%20of%20transcriptional%20and%20translational%20regulation&amp;author=P.%20Lu&amp;author=C.%20Vogel&amp;author=R.%20Wang&amp;author=X.%20Yao&amp;author=E.M.%20Marcotte&amp;publication_year=2006&amp;journal=Nat.%20Biotechnol.&amp;volume=25&amp;pages=117-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1270" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1270" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1270"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17187058" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zur</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Tuller</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="article-title">Transcript features alone enable accurate prediction and understanding of gene expression evolution in <em>S. cerevisiae</em>,</div>
          <div class="source">BMC Bioinfomatics</div> 
          <div class="year">2013</div>
          <div class="volume">14</div>
          <div class="issue">Suppl 15</div>
          <div class="fpage">S1</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcript%20features%20alone%20enable%20accurate%20prediction%20and%20understanding%20of%20gene%20expression%20evolution%20in%20S.%20cerevisiae%2C&amp;author=H.%20Zur&amp;author=T.%20Tuller&amp;publication_year=2013&amp;journal=BMC%20Bioinfomatics&amp;volume=14&amp;pages=S1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-14-S15-S1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-14-S15-S1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-14-S15-S1"> </span></div> </div></div></div></div></div><div content-id="DSU017C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Futcher</div>
               <div class="given-names">B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Latter</div>
               <div class="given-names">G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Monardo</div>
               <div class="given-names">P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">McLaughlin</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Garrels</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="article-title">A sampling of the yeast proteome</div>
          <div class="source">Mol. Cell. Biol.</div> 
          <div class="year">1999</div>
          <div class="volume">19</div>
          <div class="fpage">7357</div>
          <div class="lpage">68</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20sampling%20of%20the%20yeast%20proteome&amp;author=B.%20Futcher&amp;author=G.%20Latter&amp;author=P.%20Monardo&amp;author=C.%20McLaughlin&amp;author=J.%20Garrels&amp;publication_year=1999&amp;journal=Mol.%20Cell.%C2%A0Biol.&amp;volume=19&amp;pages=7357-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.19.11.7357" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.19.11.7357" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.19.11.7357"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10523624" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Duret</div>
               <div class="given-names">L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Mouchiroud</div>
               <div class="given-names">D.</div>
            </div>
          </div>
          <div class="article-title">Expression pattern and, surprisingly, gene length shape codon usage in <em>Caenorhabditis</em>, <em>Drosophila</em>, and <em>Arabidopsis</em></div>
          <div class="source">Proc. Natl. Acad. Sci.</div> 
          <div class="year">1999</div>
          <div class="volume">96</div>
          <div class="fpage">4482</div>
          <div class="lpage">7</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20pattern%20and%2C%20surprisingly%2C%20gene%20length%20shape%20codon%20usage%20in%20Caenorhabditis%2C%20Drosophila%2C%20and%20Arabidopsis&amp;author=L.%20Duret&amp;author=D.%20Mouchiroud&amp;publication_year=1999&amp;journal=Proc.%20Natl.%20Acad.%20Sci.&amp;volume=96&amp;pages=4482-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.96.8.4482" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.96.8.4482" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.96.8.4482"> </span></div> </div></div></div></div></div><div content-id="DSU017C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Coghlan</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wolfe</div>
               <div class="given-names">K.H.</div>
            </div>
          </div>
          <div class="article-title">Relationship of codon bias to mRNA concentration and protein length in <em>Saccharomyces cerevisiae</em></div>
          <div class="source">Yeast</div> 
          <div class="year">2000</div>
          <div class="volume">16</div>
          <div class="fpage">1131</div>
          <div class="lpage">45</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Relationship%20of%20codon%20bias%20to%20mRNA%20concentration%20and%20protein%20length%20in%20Saccharomyces%20cerevisiae&amp;author=A.%20Coghlan&amp;author=K.H.%20Wolfe&amp;publication_year=2000&amp;journal=Yeast&amp;volume=16&amp;pages=1131-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0061" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0061"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10953085" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Greenbaum</div>
               <div class="given-names">D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Colangelo</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Williams</div>
               <div class="given-names">K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Gerstein</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="article-title">Comparing protein abundance and mRNA expression levels on a genomic scale</div>
          <div class="source">Genome Biol.</div> 
          <div class="year">2003</div>
          <div class="volume">4</div>
          <div class="fpage">117</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparing%20protein%20abundance%20and%20mRNA%20expression%20levels%20on%20a%20genomic%20scale&amp;author=D.%20Greenbaum&amp;author=C.%20Colangelo&amp;author=K.%20Williams&amp;author=M.%20Gerstein&amp;publication_year=2003&amp;journal=Genome%20Biol.&amp;volume=4&amp;pages=117" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2003-4-9-117" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2003-4-9-117" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2003-4-9-117"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12952525" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">de Sousa Abreu</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Penalva</div>
               <div class="given-names">L.O.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Marcotte</div>
               <div class="given-names">E.M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Vogel</div>
               <div class="given-names">C.</div>
            </div>
          </div>
          <div class="article-title">Global signatures of protein and mRNA expression levels</div>
          <div class="source">Mol. BioSyst.</div> 
          <div class="year">2009</div>
          <div class="volume">5</div>
          <div class="fpage">1512</div>
          <div class="lpage">26</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20signatures%20of%20protein%20and%20mRNA%20expression%20levels&amp;author=R.%20de%20Sousa%20Abreu&amp;author=L.O.%20Penalva&amp;author=E.M.%20Marcotte&amp;author=C.%20Vogel&amp;publication_year=2009&amp;journal=Mol.%20BioSyst.&amp;volume=5&amp;pages=1512-26" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20023718" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">MacQueen</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="article-title">Some methods for classification and analysis of multivariate observations</div>
          <div class="source">Proceedings of the fifth Berkeley symposium on mathematical statistics and probability</div> 
          <div class="year">1967</div>
          <div class="volume">1</div>
          <div class="fpage">281</div>
          <div class="lpage">97</div>
        </div></div></div></div><div content-id="DSU017C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rocha</div>
               <div class="given-names">E.P.</div>
            </div>
          </div>
          <div class="article-title">Codon usage bias from tRNA's point of view: redundancy, specialization, and efficient decoding for translation optimization</div>
          <div class="source">Genome Res.</div> 
          <div class="year">2004</div>
          <div class="volume">14</div>
          <div class="fpage">2279</div>
          <div class="lpage">86</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20bias%20from%20tRNA's%20point%20of%20view%3A%20redundancy%2C%20specialization%2C%20and%20efficient%20decoding%20for%20translation%20optimization&amp;author=E.P.%20Rocha&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=2279-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2896904" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2896904" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2896904"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15479947" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Qian</div>
               <div class="given-names">W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Yang</div>
               <div class="given-names">J.-R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Pearson</div>
               <div class="given-names">N.M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Maclean</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="article-title">Balanced codon usage optimizes eukaryotic translational efficiency</div>
          <div class="source">PLoS Genet.</div> 
          <div class="year">2012</div>
          <div class="volume">8</div>
          <div class="fpage">e1002603</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Balanced%20codon%20usage%20optimizes%20eukaryotic%20translational%20efficiency&amp;author=W.%20Qian&amp;author=J.-R.%20Yang&amp;author=N.M.%20Pearson&amp;author=C.%20Maclean&amp;author=J.%20Zhang&amp;publication_year=2012&amp;journal=PLoS%20Genet.&amp;volume=8&amp;pages=e1002603" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1002603" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002603" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1002603"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22479199" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Murphy</div>
               <div class="given-names">F.V.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ramakrishnan</div>
               <div class="given-names">V.</div>
            </div>
          </div>
          <div class="article-title">Structure of a purine-purine wobble base pair in the decoding center of the ribosome</div>
          <div class="source">Nat. Struct. Mol. Biol.</div> 
          <div class="year">2004</div>
          <div class="volume">11</div>
          <div class="fpage">1251</div>
          <div class="lpage">2</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20of%20a%20purine-purine%20wobble%20base%20pair%20in%20the%20decoding%20center%20of%20the%20ribosome&amp;author=F.V.%20Murphy&amp;author=V.%20Ramakrishnan&amp;publication_year=2004&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;volume=11&amp;pages=1251-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb866" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb866" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb866"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15558050" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mandal</div>
               <div class="given-names">D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Köhrer</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Su</div>
               <div class="given-names">D.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Agmatidine, a modified cytidine in the anticodon of archaeal tRNAIle, base pairs with adenosine but not with guanosine</div>
          <div class="source">Proc. Natl. Acad. Sci.</div> 
          <div class="year">2010</div>
          <div class="volume">107</div>
          <div class="fpage">2872</div>
          <div class="lpage">7</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Agmatidine%2C%20a%20modified%20cytidine%20in%20the%20anticodon%20of%20archaeal%20tRNAIle%2C%20base%20pairs%20with%20adenosine%20but%20not%20with%20guanosine&amp;author=D.%20Mandal&amp;author=C.%20K%C3%B6hrer&amp;author=D.%20Su&amp;publication_year=2010&amp;journal=Proc.%20Natl.%20Acad.%20Sci.&amp;volume=107&amp;pages=2872-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0914869107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0914869107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0914869107"> </span></div> </div></div></div></div></div><div content-id="DSU017C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Albertsen</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Hugenholtz</div>
               <div class="given-names">P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Skarshewski</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Nielsen</div>
               <div class="given-names">K.L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Tyson</div>
               <div class="given-names">G.W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Nielsen</div>
               <div class="given-names">P.H.</div>
            </div>
          </div>
          <div class="article-title">Genome sequences of rare, uncultured bacteria obtained by differential coverage binning of multiple metagenomes</div>
          <div class="source">Nat. Biotechnol.</div> 
          <div class="year">2013</div>
          <div class="volume">31</div>
          <div class="fpage">533</div>
          <div class="lpage">8</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequences%20of%20rare%2C%20uncultured%20bacteria%20obtained%20by%20differential%20coverage%20binning%20of%20multiple%20metagenomes&amp;author=M.%20Albertsen&amp;author=P.%20Hugenholtz&amp;author=A.%20Skarshewski&amp;author=K.L.%20Nielsen&amp;author=G.W.%20Tyson&amp;author=P.H.%20Nielsen&amp;publication_year=2013&amp;journal=Nat.%20Biotechnol.&amp;volume=31&amp;pages=533-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2579" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2579" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2579"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23707974" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hardesty</div>
               <div class="given-names">B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kramer</div>
               <div class="given-names">G</div>
            </div>
          </div>
          <div class="source">Structure, function, and genetics of ribosomes</div> 
          <div class="year">1986</div>
          <div class="publisher-loc">NY</div>
          <div class="publisher-name">Springer</div>
        </div></div></div></div><div content-id="DSU017C67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Melnikov</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ben-Shem</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">de Loubresse</div>
               <div class="given-names">N.G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Jenner</div>
               <div class="given-names">L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Yusupova</div>
               <div class="given-names">G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Yusupov</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="article-title">One core, two shells: bacterial and eukaryotic ribosomes</div>
          <div class="source">Nat. Struct. Mol. Biol.</div> 
          <div class="year">2012</div>
          <div class="volume">19</div>
          <div class="fpage">560</div>
          <div class="lpage">7</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=One%20core%2C%20two%20shells%3A%20bacterial%20and%20eukaryotic%20ribosomes&amp;author=S.%20Melnikov&amp;author=A.%20Ben-Shem&amp;author=N.G.%20de%20Loubresse&amp;author=L.%20Jenner&amp;author=G.%20Yusupova&amp;author=M.%20Yusupov&amp;publication_year=2012&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;volume=19&amp;pages=560-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb.2313" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb.2313" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb.2313"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22664983" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Novoa</div>
               <div class="given-names">E.M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Pavon-Eternod</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Pan</div>
               <div class="given-names">T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ribas de Pouplana</div>
               <div class="given-names">L.</div>
            </div>
          </div>
          <div class="article-title">A role for tRNA modifications in genome structure and codon usage</div>
          <div class="source">Cell</div> 
          <div class="year">2012</div>
          <div class="volume">149</div>
          <div class="fpage">202</div>
          <div class="lpage">13</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20role%20for%20tRNA%20modifications%20in%20genome%20structure%20and%20codon%20usage&amp;author=E.M.%20Novoa&amp;author=M.%20Pavon-Eternod&amp;author=T.%20Pan&amp;author=L.%20Ribas%20de%20Pouplana&amp;publication_year=2012&amp;journal=Cell&amp;volume=149&amp;pages=202-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2012.01.050" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2012.01.050" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2012.01.050"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22464330" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Frenkel-Morgenstern</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Danon</div>
               <div class="given-names">T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Christian</div>
               <div class="given-names">T.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Genes adopt non-optimal codon usage to generate cell cycle-dependent oscillations in protein levels</div>
          <div class="source">Mol. Syst. Biol.</div> 
          <div class="year">2012</div>
          <div class="volume">8</div>
          <div class="fpage">572</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genes%20adopt%20non-optimal%20codon%20usage%20to%20generate%20cell%20cycle-dependent%20oscillations%20in%20protein%20levels&amp;author=M.%20Frenkel-Morgenstern&amp;author=T.%20Danon&amp;author=T.%20Christian&amp;publication_year=2012&amp;journal=Mol.%20Syst.%20Biol.&amp;volume=8&amp;pages=572" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb.2012.3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb.2012.3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmsb.2012.3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22373820" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kudla</div>
               <div class="given-names">G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Murray</div>
               <div class="given-names">A.W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Tollervey</div>
               <div class="given-names">D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Plotkin</div>
               <div class="given-names">J.B.</div>
            </div>
          </div>
          <div class="article-title">Coding-sequence determinants of gene expression in <em>Escherichia coli</em></div>
          <div class="source">Science</div> 
          <div class="year">2009</div>
          <div class="volume">324</div>
          <div class="fpage">255</div>
          <div class="lpage">8</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coding-sequence%20determinants%20of%20gene%20expression%20in%20Escherichia%20coli&amp;author=G.%20Kudla&amp;author=A.W.%20Murray&amp;author=D.%20Tollervey&amp;author=J.B.%20Plotkin&amp;publication_year=2009&amp;journal=Science&amp;volume=324&amp;pages=255-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1170160" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1170160" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1170160"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19359587" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU017C71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU017C71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Supek</div>
               <div class="given-names">F.</div>
            </div>
            <div class="name string-name">
              <div class="surname">S˘muc</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="article-title">On relevance of codon usage to expression of synthetic and natural genes in <em>Escherichia coli</em></div>
          <div class="source">Genetics</div> 
          <div class="year">2010</div>
          <div class="volume">185</div>
          <div class="fpage">1129</div>
          <div class="lpage">34</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=On%20relevance%20of%20codon%20usage%20to%20expression%20of%20synthetic%20and%20natural%20genes%20in%20Escherichia%20coli&amp;author=F.%20Supek&amp;author=T.%20S%CB%98muc&amp;publication_year=2010&amp;journal=Genetics&amp;volume=185&amp;pages=1129-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.110.115477" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.110.115477" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.110.115477"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20421604" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2754544&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu017/2/dsu017_Supplementary_Data.zip?Expires=2147483647&amp;Signature=oVKv7qorjDac3xeIxZvVwbsJ-qcQdF~HYxNmw~i8D146Xv1Qcmgq-QO5Xyp4rkP~pBak4jTqiqgW5eB1jlrOFDUnSx8Yy3e4tWBtbqsO9MW3bdi~0IwtKzN4kt61Kv3Ss6pyKGlDqjN10WzQPryart8qWqyxiN~OomqZF8tD-d7cS1~XpgDbefils0tEkiTFvHG1Yq3NtEV~g1ymvfW7G4M7X3j5Nvtt3bCc9KRPQK9nVopAY35nPCgW2Q6aHHWhWwRK0eDp76GTBZ9nO5~JEidfjT5qEA7PE-zAWUSy95GP0GpiNeUdA8t7VZksBhnr7DXFXHvHeZlj5N3VJWIFWA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">734</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000344244900006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">20</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsu017"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2754544"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000344244900006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f5%2f511%2f2754544">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000344244900006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (20)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F5%2F511%2F2754544" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2754544'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Modelling the Efficiency of Codon–tRNA Interactions Based on Codon Usage Bias" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsu017");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f5%2f511%2f2754544">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

