

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/16/2/105/510716","@type":"ScholarlyArticle","name":"Using Triplet Periodicity of Nucleotide Sequences for Finding Potential Reading Frame Shifts in Genes","datePublished":"2009-03-03","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/16/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2009-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsp002","keywords":["triplet periodicity","reading frame","shift"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/16/2/105/510716","author":[{"name":"Frenkel, F.E.","affiliation":"Bioengineering Centre of RAS ,  60-letiya Oktyabrya prosp., 7/1, Moscow 117312 ,  Russia","@type":"Person"},{"name":"Korotkov, E.V.","affiliation":"Bioengineering Centre of RAS ,  60-letiya Oktyabrya prosp., 7/1, Moscow 117312 ,  Russia","@type":"Person"}],"description":"Abstract.  We introduce a novel approach for the detection of possible mutations leading to a reading frame (RF) shift in a gene. Deletions and insertions of DN","pageStart":"105","pageEnd":"114","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/2/1/m_cover.gif?Expires=2147483647&Signature=spYD3Y1ECWwq5jfgvLwgtJGBnOHAN-nZXtWf5jX0Idx1cbrJRV4FKB28EY3pe17T0gsGMxo5dA8Tel6c2xNGAqa1byNOzqnWfnM8ohlcCfLHlrfGOEn2zyCW-15VbU7SOoLTo3OmdKcdJ19GrDbXCyAEcU~VJnaBCNGoOwBtbTnh6-S6KzBaPxjqj1HfbfKuNRe7H3r-0uFeA~Y3Ph79tw8QP6q4D0FiiXdjGGWYTNp3-Ga~m-PPAcOogXTotI1YFUthxQJgvIKyhC8JLbL3Znx728-CpY2KbMeFQlltwFxSw7d-mO5IOuvD7Wc91P7OLYCdnpnsyiP0k-TqunqvaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Using Triplet Periodicity of Nucleotide Sequences for Finding Potential Reading Frame Shifts in Genes","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/2/1/m_cover.gif?Expires=2147483647&Signature=spYD3Y1ECWwq5jfgvLwgtJGBnOHAN-nZXtWf5jX0Idx1cbrJRV4FKB28EY3pe17T0gsGMxo5dA8Tel6c2xNGAqa1byNOzqnWfnM8ohlcCfLHlrfGOEn2zyCW-15VbU7SOoLTo3OmdKcdJ19GrDbXCyAEcU~VJnaBCNGoOwBtbTnh6-S6KzBaPxjqj1HfbfKuNRe7H3r-0uFeA~Y3Ph79tw8QP6q4D0FiiXdjGGWYTNp3-Ga~m-PPAcOogXTotI1YFUthxQJgvIKyhC8JLbL3Znx728-CpY2KbMeFQlltwFxSw7d-mO5IOuvD7Wc91P7OLYCdnpnsyiP0k-TqunqvaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Using Triplet Periodicity of Nucleotide Sequences for Finding Potential Reading Frame Shifts in Genes" />
    <meta property="og:description" content="Abstract.  We introduce a novel approach for the detection of possible mutations leading to a reading frame (RF) shift in a gene. Deletions and insertions of DN" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/16/2/105/510716" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2009-03-03" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Frenkel, F.E." /><meta name="citation_author_institution" content="Bioengineering Centre of RAS, 60-letiya Oktyabrya prosp., 7/1, Moscow 117312, Russia" /><meta name="citation_author" content="Korotkov, E.V." /><meta name="citation_author_institution" content="Bioengineering Centre of RAS, 60-letiya Oktyabrya prosp., 7/1, Moscow 117312, Russia" /><meta name="citation_title" content="Using Triplet Periodicity of Nucleotide Sequences for Finding Potential Reading Frame Shifts in Genes" /><meta name="citation_firstpage" content="105" /><meta name="citation_lastpage" content="114" /><meta name="citation_doi" content="10.1093/dnares/dsp002" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="16" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2009/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_publisher=World Scientific Publishing Co. Pte, Ltd., Singapore; DNA Repair, Genetic Instability, and Cancer;  citation_year=2007; " /><meta name="citation_reference" content="citation_title=Frequent appearance of novel protein-coding sequences by frameshift translation; Genomics;  citation_year=2006;  citation_volume=88;  citation_pages=690-697; " /><meta name="citation_reference" content="citation_title=Functional divergence of proteins through frameshift mutations; Trends Genet.;  citation_year=2005;  citation_volume=21;  citation_pages=428-431; " /><meta name="citation_reference" content="citation_title=Predictive methods using nucleotide sequences; Methods Biochem. Anal.;  citation_year=1998;  citation_volume=39;  citation_pages=231-245; " /><meta name="citation_reference" content="citation_title=Staden: statistical and structural analysis of nucleotide sequences; Methods Mol. Biol.;  citation_year=1994;  citation_volume=25;  citation_pages=69-77; " /><meta name="citation_reference" content="citation_title=Predictive methods using DNA sequences; Methods Biochem. Anal.;  citation_year=2001;  citation_volume=43;  citation_pages=233-252; " /><meta name="citation_reference" content="citation_title=On the origin of the periodicity of three in protein coding DNA sequences; J. Theor. Biol.;  citation_year=1994;  citation_volume=167;  citation_issue=4;  citation_pages=413-414; " /><meta name="citation_reference" content="citation_title=Protein coding sequence identification by simultaneously characterizing the periodic and random features of DNA sequences; J. Biomed. Biotechnol.;  citation_year=2005;  citation_volume=2;  citation_pages=139-146; " /><meta name="citation_reference" content="citation_title=Prediction of protein coding regions by the 3-base periodicity analysis of a DNA sequence; J. Theor. Biol.;  citation_year=2007;  citation_volume=247;  citation_pages=687-694; " /><meta name="citation_reference" content="citation_title=Periodicity of DNA in exons; BMC Mol. Biol.;  citation_year=2004;  citation_volume=5;  citation_pages=12" /><meta name="citation_reference" content="citation_title=The relationship between base composition and codon usage in bacterial genes and its use for the simple and reliable identification of protein-coding sequences; Gene;  citation_year=1984;  citation_volume=30;  citation_issue=1–3;  citation_pages=157-166; " /><meta name="citation_reference" content="citation_title=Sequences and codes: fundamentals of biomolecular cryptography; citation_publisher=Academic Press, San Diego; Biocomputing: Informatics and Genome Projects;  citation_year=1994;  citation_pages=119-174; " /><meta name="citation_reference" content="citation_title=Elucidating sequence codes: three codes for evolution; Ann. NY Acad. Sci.;  citation_year=1999;  citation_volume=870;  citation_pages=330-338; " /><meta name="citation_reference" content="citation_title=Transfer-RNA: the early adaptor; Naturwissenschaften;  citation_year=1981;  citation_volume=68;  citation_pages=217-228; " /><meta name="citation_reference" content="citation_title=Is DNA code periodicity only due to CUF—codons usage frequency?; Conf. Proc. IEEE Eng. Med. Biol. Soc.;  citation_year=2007;  citation_volume=1;  citation_pages=1383-1386; " /><meta name="citation_reference" content="citation_title=The nonrandom location of synonymous codons suggests that reading frame-independent forces have patterned codon preferences; J. Mol. Evol.;  citation_year=1999;  citation_volume=49;  citation_issue=1;  citation_pages=36-43; " /><meta name="citation_reference" content="citation_title=The informational concept of searching for periodicity in symbol sequences; Mol. Biol. (Mosk);  citation_year=2003;  citation_volume=37;  citation_pages=436-451; " /><meta name="citation_reference" content="citation_title=Locating probable genes using Fourier transform approach; Bioinformatics;  citation_year=2002;  citation_volume=18;  citation_issue=1;  citation_pages=196-197; " /><meta name="citation_reference" content="citation_title=Prediction of probable genes by Fourier analysis of genomic sequences; Comput. Appl. Biosci.;  citation_year=1997;  citation_volume=13;  citation_issue=3;  citation_pages=263-270; " /><meta name="citation_reference" content="citation_title=Probabilistic methods of identifying genes in prokaryotic genomes: connections to the HMM theory; Brief. Bioinform.;  citation_year=2004;  citation_volume=5;  citation_issue=2;  citation_pages=118-130; " /><meta name="citation_reference" content="citation_title=Finding genes in DNA with a hidden Markov model; J. Comput. Biol.;  citation_year=1997;  citation_volume=4;  citation_pages=127-141; " /><meta name="citation_reference" content="citation_title=Identification of coding regions in genomic DNA sequences: an application of dynamic programming and neural networks; Nucleic Acids Res.;  citation_year=1993;  citation_volume=21;  citation_pages=607-613; " /><meta name="citation_reference" content="citation_title=A probabilistic model for detecting coding regions in DNA sequences; IMA J. Math. Appl. Med. Biol.;  citation_year=1994;  citation_volume=11;  citation_issue=3;  citation_pages=149-160; " /><meta name="citation_reference" content="citation_title=Information decomposition method for analysis of symbolical sequences; Phys. Lett. A;  citation_year=2003;  citation_volume=312;  citation_pages=198-210; " /><meta name="citation_reference" content="citation_title=KEGG: Kyoto Encyclopedia of Genes and Genomes; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=29-34; " /><meta name="citation_reference" content="citation_publisher=Wiley, New York; Information Theory and Statistics;  citation_year=1959; " /><meta name="citation_reference" content="citation_publisher=CERN, Geneva; Statistics: Lectures on Elementary Statistics and Probability;  citation_year=1964; " /><meta name="citation_reference" content="citation_title=Evolution of the tRNA-like sequences and genome variability; Gene;  citation_year=2004;  citation_volume=335;  citation_pages=57-71; " /><meta name="citation_reference" content="citation_title=Latent periodicity of 21 bases typical for MCP II gene is widely present in various bacterial genes; DNA Seq.;  citation_year=2003;  citation_volume=14;  citation_pages=37-52; " /><meta name="citation_reference" content="citation_title=A general method applicable to the search for similarities in the amino acid sequence of two proteins; J. Mol. Biol.;  citation_year=1970;  citation_volume=48;  citation_issue=3;  citation_pages=443-453; " /><meta name="citation_reference" content="citation_publisher=Cambridge University Press, Cambridge, UK; Biological Sequence Analysis: Probabilistic Models of Proteins and Nucleic Acids;  citation_year=1999; " /><meta name="citation_reference" content="citation_title=Classification analysis of triplet periodicity in protein-coding regions of genes; Gene;  citation_year=2008;  citation_volume=421;  citation_pages=52-60; " /><meta name="citation_reference" content="citation_title=The universal protein resource (UniProt); citation_author=UniProt Consortium; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=193-197; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool; J. Mol. Biol.;  citation_year=1990;  citation_volume=215;  citation_issue=3;  citation_pages=403-410; " /><meta name="citation_reference" content="citation_title=A simplified explanation for the frameshift mutation that created a novel C-terminal motif in the APETALA3 gene lineage; BMC Evol. Biol.;  citation_year=2006;  citation_volume=6;  citation_pages=30" /><meta name="citation_reference" content="citation_title=Evolution and multilevel optimization of the genetic code; Genome Res.;  citation_year=2007;  citation_volume=17;  citation_issue=4;  citation_pages=405-412; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/16/2/105/1371957/dsp002.pdf" /><meta name="description" content="Abstract.  We introduce a novel approach for the detection of possible mutations leading to a reading frame (RF) shift in a gene. Deletions and insertions of DN" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/16/2/105/510716" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/16/2/105/510716" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/16/2/105/510716");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsp002';
    </script>
<title>
	Using Triplet Periodicity of Nucleotide Sequences for Finding Potential Reading Frame Shifts in Genes | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./510716" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BchJPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGcmVua2VsLCBGLkUuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvZW5naW5lZXJpbmcgQ2VudHJlIG9mIFJBUywgNjAtbGV0aXlhIE9rdHlhYnJ5YSBwcm9zcC4sIDcvMSwgTW9zY293IDExNzMxMiwgUnVzc2lhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS29yb3Rrb3YsIEUuVi4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW9lbmdpbmVlcmluZyBDZW50cmUgb2YgUkFTLCA2MC1sZXRpeWEgT2t0eWFicnlhIHByb3NwLiwgNy8xLCBNb3Njb3cgMTE3MzEyLCBSdXNzaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IlVzaW5nIFRyaXBsZXQgUGVyaW9kaWNpdHkgb2YgTnVjbGVvdGlkZSBTZXF1ZW5jZXMgZm9yIEZpbmRpbmcgUG90ZW50aWFsIFJlYWRpbmcgRnJhbWUgU2hpZnRzIGluIEdlbmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMTA1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIxMTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3AwMDIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAwOS8wNC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1Xb3JsZCBTY2llbnRpZmljIFB1Ymxpc2hpbmcgQ28uIFB0ZSwgTHRkLiwgU2luZ2Fwb3JlOyBETkEgUmVwYWlyLCBHZW5ldGljIEluc3RhYmlsaXR5LCBhbmQgQ2FuY2VyOyAgY2l0YXRpb25feWVhcj0yMDA3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GcmVxdWVudCBhcHBlYXJhbmNlIG9mIG5vdmVsIHByb3RlaW4tY29kaW5nIHNlcXVlbmNlcyBieSBmcmFtZXNoaWZ0IHRyYW5zbGF0aW9uOyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT04ODsgIGNpdGF0aW9uX3BhZ2VzPTY5MC02OTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bmN0aW9uYWwgZGl2ZXJnZW5jZSBvZiBwcm90ZWlucyB0aHJvdWdoIGZyYW1lc2hpZnQgbXV0YXRpb25zOyBUcmVuZHMgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9NDI4LTQzMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlZGljdGl2ZSBtZXRob2RzIHVzaW5nIG51Y2xlb3RpZGUgc2VxdWVuY2VzOyBNZXRob2RzIEJpb2NoZW0uIEFuYWwuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9MjMxLTI0NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RhZGVuOiBzdGF0aXN0aWNhbCBhbmQgc3RydWN0dXJhbCBhbmFseXNpcyBvZiBudWNsZW90aWRlIHNlcXVlbmNlczsgTWV0aG9kcyBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9NjktNzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWRpY3RpdmUgbWV0aG9kcyB1c2luZyBETkEgc2VxdWVuY2VzOyBNZXRob2RzIEJpb2NoZW0uIEFuYWwuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9MjMzLTI1MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T24gdGhlIG9yaWdpbiBvZiB0aGUgcGVyaW9kaWNpdHkgb2YgdGhyZWUgaW4gcHJvdGVpbiBjb2RpbmcgRE5BIHNlcXVlbmNlczsgSi4gVGhlb3IuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTE2NzsgIGNpdGF0aW9uX2lzc3VlPTQ7ICBjaXRhdGlvbl9wYWdlcz00MTMtNDE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm90ZWluIGNvZGluZyBzZXF1ZW5jZSBpZGVudGlmaWNhdGlvbiBieSBzaW11bHRhbmVvdXNseSBjaGFyYWN0ZXJpemluZyB0aGUgcGVyaW9kaWMgYW5kIHJhbmRvbSBmZWF0dXJlcyBvZiBETkEgc2VxdWVuY2VzOyBKLiBCaW9tZWQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0xMzktMTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW9uIG9mIHByb3RlaW4gY29kaW5nIHJlZ2lvbnMgYnkgdGhlIDMtYmFzZSBwZXJpb2RpY2l0eSBhbmFseXNpcyBvZiBhIEROQSBzZXF1ZW5jZTsgSi4gVGhlb3IuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI0NzsgIGNpdGF0aW9uX3BhZ2VzPTY4Ny02OTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBlcmlvZGljaXR5IG9mIEROQSBpbiBleG9uczsgQk1DIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTEyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJlbGF0aW9uc2hpcCBiZXR3ZWVuIGJhc2UgY29tcG9zaXRpb24gYW5kIGNvZG9uIHVzYWdlIGluIGJhY3RlcmlhbCBnZW5lcyBhbmQgaXRzIHVzZSBmb3IgdGhlIHNpbXBsZSBhbmQgcmVsaWFibGUgaWRlbnRpZmljYXRpb24gb2YgcHJvdGVpbi1jb2Rpbmcgc2VxdWVuY2VzOyBHZW5lOyAgY2l0YXRpb25feWVhcj0xOTg0OyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25faXNzdWU9MeKAkzM7ICBjaXRhdGlvbl9wYWdlcz0xNTctMTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jZXMgYW5kIGNvZGVzOiBmdW5kYW1lbnRhbHMgb2YgYmlvbW9sZWN1bGFyIGNyeXB0b2dyYXBoeTsgY2l0YXRpb25fcHVibGlzaGVyPUFjYWRlbWljIFByZXNzLCBTYW4gRGllZ287IEJpb2NvbXB1dGluZzogSW5mb3JtYXRpY3MgYW5kIEdlbm9tZSBQcm9qZWN0czsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3BhZ2VzPTExOS0xNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVsdWNpZGF0aW5nIHNlcXVlbmNlIGNvZGVzOiB0aHJlZSBjb2RlcyBmb3IgZXZvbHV0aW9uOyBBbm4uIE5ZIEFjYWQuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9ODcwOyAgY2l0YXRpb25fcGFnZXM9MzMwLTMzODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNmZXItUk5BOiB0aGUgZWFybHkgYWRhcHRvcjsgTmF0dXJ3aXNzZW5zY2hhZnRlbjsgIGNpdGF0aW9uX3llYXI9MTk4MTsgIGNpdGF0aW9uX3ZvbHVtZT02ODsgIGNpdGF0aW9uX3BhZ2VzPTIxNy0yMjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzIEROQSBjb2RlIHBlcmlvZGljaXR5IG9ubHkgZHVlIHRvIENVRuKAlGNvZG9ucyB1c2FnZSBmcmVxdWVuY3k/OyBDb25mLiBQcm9jLiBJRUVFIEVuZy4gTWVkLiBCaW9sLiBTb2MuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0xMzgzLTEzODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBub25yYW5kb20gbG9jYXRpb24gb2Ygc3lub255bW91cyBjb2RvbnMgc3VnZ2VzdHMgdGhhdCByZWFkaW5nIGZyYW1lLWluZGVwZW5kZW50IGZvcmNlcyBoYXZlIHBhdHRlcm5lZCBjb2RvbiBwcmVmZXJlbmNlczsgSi4gTW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT00OTsgIGNpdGF0aW9uX2lzc3VlPTE7ICBjaXRhdGlvbl9wYWdlcz0zNi00MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGluZm9ybWF0aW9uYWwgY29uY2VwdCBvZiBzZWFyY2hpbmcgZm9yIHBlcmlvZGljaXR5IGluIHN5bWJvbCBzZXF1ZW5jZXM7IE1vbC4gQmlvbC4gKE1vc2spOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9NDM2LTQ1MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG9jYXRpbmcgcHJvYmFibGUgZ2VuZXMgdXNpbmcgRm91cmllciB0cmFuc2Zvcm0gYXBwcm9hY2g7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25faXNzdWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTE5Ni0xOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWRpY3Rpb24gb2YgcHJvYmFibGUgZ2VuZXMgYnkgRm91cmllciBhbmFseXNpcyBvZiBnZW5vbWljIHNlcXVlbmNlczsgQ29tcHV0LiBBcHBsLiBCaW9zY2kuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25faXNzdWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTI2My0yNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb2JhYmlsaXN0aWMgbWV0aG9kcyBvZiBpZGVudGlmeWluZyBnZW5lcyBpbiBwcm9rYXJ5b3RpYyBnZW5vbWVzOiBjb25uZWN0aW9ucyB0byB0aGUgSE1NIHRoZW9yeTsgQnJpZWYuIEJpb2luZm9ybS47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX2lzc3VlPTI7ICBjaXRhdGlvbl9wYWdlcz0xMTgtMTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GaW5kaW5nIGdlbmVzIGluIEROQSB3aXRoIGEgaGlkZGVuIE1hcmtvdiBtb2RlbDsgSi4gQ29tcHV0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MTI3LTE0MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgY29kaW5nIHJlZ2lvbnMgaW4gZ2Vub21pYyBETkEgc2VxdWVuY2VzOiBhbiBhcHBsaWNhdGlvbiBvZiBkeW5hbWljIHByb2dyYW1taW5nIGFuZCBuZXVyYWwgbmV0d29ya3M7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTYwNy02MTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcHJvYmFiaWxpc3RpYyBtb2RlbCBmb3IgZGV0ZWN0aW5nIGNvZGluZyByZWdpb25zIGluIEROQSBzZXF1ZW5jZXM7IElNQSBKLiBNYXRoLiBBcHBsLiBNZWQuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25faXNzdWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTE0OS0xNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluZm9ybWF0aW9uIGRlY29tcG9zaXRpb24gbWV0aG9kIGZvciBhbmFseXNpcyBvZiBzeW1ib2xpY2FsIHNlcXVlbmNlczsgUGh5cy4gTGV0dC4gQTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTI7ICBjaXRhdGlvbl9wYWdlcz0xOTgtMjEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1LRUdHOiBLeW90byBFbmN5Y2xvcGVkaWEgb2YgR2VuZXMgYW5kIEdlbm9tZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTI5LTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9V2lsZXksIE5ldyBZb3JrOyBJbmZvcm1hdGlvbiBUaGVvcnkgYW5kIFN0YXRpc3RpY3M7ICBjaXRhdGlvbl95ZWFyPTE5NTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1DRVJOLCBHZW5ldmE7IFN0YXRpc3RpY3M6IExlY3R1cmVzIG9uIEVsZW1lbnRhcnkgU3RhdGlzdGljcyBhbmQgUHJvYmFiaWxpdHk7ICBjaXRhdGlvbl95ZWFyPTE5NjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBvZiB0aGUgdFJOQS1saWtlIHNlcXVlbmNlcyBhbmQgZ2Vub21lIHZhcmlhYmlsaXR5OyBHZW5lOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMzNTsgIGNpdGF0aW9uX3BhZ2VzPTU3LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYXRlbnQgcGVyaW9kaWNpdHkgb2YgMjEgYmFzZXMgdHlwaWNhbCBmb3IgTUNQIElJIGdlbmUgaXMgd2lkZWx5IHByZXNlbnQgaW4gdmFyaW91cyBiYWN0ZXJpYWwgZ2VuZXM7IEROQSBTZXEuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MzctNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2VuZXJhbCBtZXRob2QgYXBwbGljYWJsZSB0byB0aGUgc2VhcmNoIGZvciBzaW1pbGFyaXRpZXMgaW4gdGhlIGFtaW5vIGFjaWQgc2VxdWVuY2Ugb2YgdHdvIHByb3RlaW5zOyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTcwOyAgY2l0YXRpb25fdm9sdW1lPTQ4OyAgY2l0YXRpb25faXNzdWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTQ0My00NTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1DYW1icmlkZ2UgVW5pdmVyc2l0eSBQcmVzcywgQ2FtYnJpZGdlLCBVSzsgQmlvbG9naWNhbCBTZXF1ZW5jZSBBbmFseXNpczogUHJvYmFiaWxpc3RpYyBNb2RlbHMgb2YgUHJvdGVpbnMgYW5kIE51Y2xlaWMgQWNpZHM7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsYXNzaWZpY2F0aW9uIGFuYWx5c2lzIG9mIHRyaXBsZXQgcGVyaW9kaWNpdHkgaW4gcHJvdGVpbi1jb2RpbmcgcmVnaW9ucyBvZiBnZW5lczsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00MjE7ICBjaXRhdGlvbl9wYWdlcz01Mi02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHVuaXZlcnNhbCBwcm90ZWluIHJlc291cmNlIChVbmlQcm90KTsgY2l0YXRpb25fYXV0aG9yPVVuaVByb3QgQ29uc29ydGl1bTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9MTkzLTE5NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFzaWMgbG9jYWwgYWxpZ25tZW50IHNlYXJjaCB0b29sOyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTIxNTsgIGNpdGF0aW9uX2lzc3VlPTM7ICBjaXRhdGlvbl9wYWdlcz00MDMtNDEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNpbXBsaWZpZWQgZXhwbGFuYXRpb24gZm9yIHRoZSBmcmFtZXNoaWZ0IG11dGF0aW9uIHRoYXQgY3JlYXRlZCBhIG5vdmVsIEMtdGVybWluYWwgbW90aWYgaW4gdGhlIEFQRVRBTEEzIGdlbmUgbGluZWFnZTsgQk1DIEV2b2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0zMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBhbmQgbXVsdGlsZXZlbCBvcHRpbWl6YXRpb24gb2YgdGhlIGdlbmV0aWMgY29kZTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9pc3N1ZT00OyAgY2l0YXRpb25fcGFnZXM9NDA1LTQxMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE2LzIvMTA1LzEzNzE5NTcvZHNwMDAyLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBXZSBpbnRyb2R1Y2UgYSBub3ZlbCBhcHByb2FjaCBmb3IgdGhlIGRldGVjdGlvbiBvZiBwb3NzaWJsZSBtdXRhdGlvbnMgbGVhZGluZyB0byBhIHJlYWRpbmcgZnJhbWUgKFJGKSBzaGlmdCBpbiBhIGdlbmUuIERlbGV0aW9ucyBhbmQgaW5zZXJ0aW9ucyBvZiBETiIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NTEwNzE2JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZEDLP4NIsoGoDeOa+CFQTieb2OBg" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f2%2f105%2f510716" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f2%2f105%2f510716" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=20204" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/16/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/2/1/m_cover.gif?Expires=2147483647&amp;Signature=spYD3Y1ECWwq5jfgvLwgtJGBnOHAN-nZXtWf5jX0Idx1cbrJRV4FKB28EY3pe17T0gsGMxo5dA8Tel6c2xNGAqa1byNOzqnWfnM8ohlcCfLHlrfGOEn2zyCW-15VbU7SOoLTo3OmdKcdJ19GrDbXCyAEcU~VJnaBCNGoOwBtbTnh6-S6KzBaPxjqj1HfbfKuNRe7H3r-0uFeA~Y3Ph79tw8QP6q4D0FiiXdjGGWYTNp3-Ga~m-PPAcOogXTotI1YFUthxQJgvIKyhC8JLbL3Znx728-CpY2KbMeFQlltwFxSw7d-mO5IOuvD7Wc91P7OLYCdnpnsyiP0k-TqunqvaA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/16/2">
                        <div class="volume trailing-comma">Volume 16</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2009        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7931385">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7931385">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7931387">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7931387">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7931396">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7931396">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7931420">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7931420">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7931438">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7931438">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7931440">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7931440">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7931442">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7931442">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/16/2/91/510591">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/16/2/115/510928">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Using Triplet Periodicity of Nucleotide Sequences for Finding Potential Reading Frame Shifts in Genes
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">F.E. Frenkel</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Bioengineering Centre of RAS</div>, <div class="addr-line">60-letiya Oktyabrya prosp., 7/1, Moscow 117312</div>, <div class="country">Russia</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +7 499-135-2161. Fax. <div class="fax">+7 499-135-0571</div>. E-mail: <a href="mailto:felix.frenkel@gmail.com" target="_blank">felix.frenkel@gmail.com</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=F.E.+Frenkel">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Frenkel F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Frenkel F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">F.E. Frenkel</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">E.V. Korotkov</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Bioengineering Centre of RAS</div>, <div class="addr-line">60-letiya Oktyabrya prosp., 7/1, Moscow 117312</div>, <div class="country">Russia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=E.V.+Korotkov">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Korotkov E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Korotkov E&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">E.V. Korotkov</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 16, Issue 2, 1 April 2009, Pages 105–114, <a href='https://doi.org/10.1093/dnares/dsp002'>https://doi.org/10.1093/dnares/dsp002</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">03 March 2009</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">26 August 2008</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">05 February 2009</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="510716" href="/dnaresearch/article-pdf/16/2/105/1371957/dsp002.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>F.E. Frenkel, E.V. Korotkov;  Using Triplet Periodicity of Nucleotide Sequences for Finding Potential Reading Frame Shifts in Genes, <em>DNA Research</em>, Volume 16, Issue 2, 1 April 2009, Pages 105–114, <a href='https://doi.org/10.1093/dnares/dsp002'>https://doi.org/10.1093/dnares/dsp002</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/510716?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/510716?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/510716?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/510716?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/510716?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/510716?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/510716?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Using%20Triplet%20Periodicity%20of%20Nucleotide%20Sequences%20for%20Finding%20Potential%20Reading%20Frame%20Shifts%20in%20Genes&amp;publicationDate=2009-03-03&amp;volumeNum=16&amp;issueNum=2&amp;author=Frenkel%2C%20F.E.%3B%20Korotkov%2C%20E.V.&amp;startPage=105&amp;endPage=114&amp;contentId=10.1093%2Fdnares%2Fdsp002&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f2%2f105%2f510716" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7931385 id="7931385" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>We introduce a novel approach for the detection of possible mutations leading to a reading frame (RF) shift in a gene. Deletions and insertions of DNA coding regions are considerable events for genes because an RF shift results in modifications of the extensive region of amino acid sequence coded by a gene. The suggested method is based on the phenomenon of triplet periodicity (TP) in coding regions of genes and its relative resistance to substitutions in DNA sequence. We attempted to extend 326 933 regions of continuous TP found in genes from the KEGG databank by considering possible insertions and deletions. We revealed totally 824 genes where such extension was possible and statistically significant. Then we generated amino acid sequences according to active (KEGG's) and hypothetically ancient RFs in order to find confirmation of a shift at a protein level. Consequently, 64 sequences have protein similarities only for ancient RF, 176 only for active RF, 3 for both and 581 have no protein similarity at all. We aimed to have revealed lower bound for the number of genes in which a shift between RF and TP is possible. Further ways to increase the number of revealed RF shifts are discussed.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;triplet periodicity&quot;">triplet periodicity</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;reading frame&quot;">reading frame</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="shift">shift</a></div>                            <h2 scrollto-destination=7931387 id="7931387" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Mutations in DNA sequences appear as a result of base substitution or deletions, insertions and DNA sequence inversion.<sup><a reveal-id="DSP002C1" data-open="DSP002C1" class="link link-ref link-reveal xref-bibr">1</a></sup> At the level of amino acid sequences, substitutions of DNA bases may lead to substitutions of amino acids, i.e. single nucleotide substitution changes one amino acid or none. In this sense, deletions, insertions and inversions of DNA coding regions are more dramatic events for genes because such mutational alteration may lead to a reading frame (RF) shift and to modification of the extensive region of amino acid sequence. Substitution frequencies are well investigated, whereas RF shifts in genes are less studied, which is due to the difficulty in their identification. However, the investigation of the influence of RF shifts in genes on protein structure is of great interest. A single DNA base substitution may result in only one change of amino acid in a protein, whereas an RF shift causes the change of all amino acids coded downstream of the shift point in a gene. If after this the protein does not lose its function, then it is supposed that a frame shift has occurred in a functionally insignificant site or that the RF shift generates functionally similar amino acid sequences. But if after RF shift the protein has changed its function, then it is of great interest to understand how changes in amino acid sequence determine the protein's new function.</p><p>Answers for these questions can probably be found after a detailed investigation of RF shift statistics. To that end, we need a method for finding RF shifts in existing genes. Currently, general methods for finding RF shifts and inversions comprise the search for similarities between amino acid sequences through the BLAST program or its analogues.<sup><a reveal-id="DSP002C2" data-open="DSP002C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSP002C3" data-open="DSP002C3" class="link link-ref link-reveal xref-bibr">3</a></sup> When using such procedures for finding RF shifts, we should in some way mark out the gene region where the RF shift is expected to be found. Then we recode this region of nucleotide sequence into that of the amino acid according to the new RF, thus obtaining the hypothetic amino acid sequence as a result. Thereafter, we run a similarity search for a hypothetic amino acid sequence in UniProtKB/Swiss-Prot databank. If statistically significant similarities are found, then we can be sure that the gene contains an RF shift, i.e. we see that sequences related to the hypothetic a amino acid sequence do exist. This method has allowed us to find, so far, several hundred genes in which the RF shift has really taken place.<sup><a reveal-id="DSP002C2" data-open="DSP002C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSP002C3" data-open="DSP002C3" class="link link-ref link-reveal xref-bibr">3</a></sup></p><p>However, this scheme of finding RF shifts and inversions has some limitations. First, we have to choose a gene, using some criterion, where an RF shift is supposed to exist, and after that, find the probable site of RF shift and inversion in it. A full-scale search for RF shifts in all genes would require very powerful computational facilities that are not always available, although modern computer systems have made such search possible.<sup><a reveal-id="DSP002C2" data-open="DSP002C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Second, even if we solve the first task, then the UniProtKB/Swiss-Prot databank will be necessary to contain the amino acid sequence having a statistically significant similarity to the hypothetical amino acid sequence. But it is possible that such a sequence will not be found due to the limitation of the UniProtKB/Swiss-Prot databank and because of too great an evolutional dissimilarity accumulated between amino acid sequences. Therefore, the concerned approach can reveal only some RF shifts accumulated to date in existing genes.</p><p>In order to reveal RF shifts and inversions in genes in a more reliable way, some new approach to seeking RF shifts should be developed instead of searching for similarities between hypothetic and real amino acid sequences. As shown in this work, a search for uniform triplet periodicity (TP) within the nucleotide sequence of gene's coding regions may be used as such an approach. Triplet organization of protein coding DNA sequences is a general feature of all presently known live systems.<sup><a reveal-id="DSP002C4  DSP002C12" data-open="DSP002C4  DSP002C12" class="link link-ref link-reveal xref-bibr">4–12</a></sup> The reason for this lies not only in the structure of genetic codes, which is virtually the same either for prokaryotes or for eukaryotes, but also in the saturation of proteins by certain amino acids.<sup><a reveal-id="DSP002C13  DSP002C16" data-open="DSP002C13  DSP002C16" class="link link-ref link-reveal xref-bibr">13–16</a></sup> If an RF shift occurs in genes in the presence of TP, then it will be revealed because a shift between TP and RF will also occur (Fig. <a reveal-id="DSP002F1" data-open="DSP002F1" class="link link-reveal link-table xref-fig">1</a>). Since TP in a DNA sequence is unlikely to be changed by a small number of base substitutions,<sup><a reveal-id="DSP002C17" data-open="DSP002C17" class="link link-ref link-reveal xref-bibr">17</a></sup> then such shift will exist for a long period of time. The presence of such a shift between the TP of a nucleotide sequence and RF may serve as an indication of an RF shift in the concerned gene.</p>                        <a id="7931392" scrollto-destination="7931392"></a>
<div id="DSP002F1" data-id="DSP002F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00201.jpeg?Expires=2147483647&Signature=HccxA6uMU9ADCH~NQHcrNRjfl-svb6DlzMtrbD60qU~9jN5ZteJIIv-E6Ymsv955W0PV-ojwGyj8tChDrgoOVOZ7QEUrVpaBngzxWa3pOBP0~v7Fc-Pu0gLtilo7YB3CnTl5GGNST0jI3HCTRwqrdbeg5~y-L7Q10kvfmeo1LhtixIiBp6aj~SfDlj8jyEcng6yZkZTUF37wsS32kzs7ja8yT59F5rQz2pPzRmr5--xp1VbyQPzmaPt2Pzv1ARcMgeiugf5ncjFmdgnj64mj6Cd4OGrseM7YCj82yNMYVeMdtVXU4zDFNAr7CNNE0jimxJ2jtux156MoWbMnpQHGXQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Influence of one nucleotide deletion on TP of nucleotide sequence. Numbers above sequence S1 show positions of nucleotides in RF. Twenty-fifth base has been deleted from sequence S1. Consequently, sequence S1 can be presented as two sequences—S2 and S3, i.e. S1 = S2aS3. In S2 and S3 sequences, TP (matrices M2 and M3) will be the same, but in sequence S3, there will be cyclical shifts by one base relative to sequence S2 and to RF in sequence S1. This means that first column of matrix M2 corresponds to third column of matrix M3, second column of matrix M2 corresponds to first column of matrix M3, third column of matrix M2 corresponds to second column of matrix M3. After summing sequences S2 and S3 and formation of sequence S4, TPM (M4 matrix) comes out from addition of first column of matrix M2 to first column of matrix M3 and so on, that leads to merging of non-identical columns and considerably decreases statistical significance of TP in sequence S4. By accounting deletions, we get sequence S4 that has TPM (M4 matrix) in which matrices M2 and M3 are merged in consideration of cyclic permutation. Finding and accounting deletions considerably increases statistical significance of TP in sequence S4." path-from-xml="dsp00201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7931392/dsp00201.jpeg" path-from-xml="dsp00201.jpeg" target="_blank">View large</a><a section="7931392" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/dsp00201.jpeg?Expires=2147483647&Signature=XxtUyaP23OaBLHY5ncCMzy1-f2IFebkeLQ5frd8YAYvQYNLgdyF1t~UL4xilp1K~ERgYUosZ059xFzY3EEXj3v-ufSD8fs8TlDD3YteSTdGDtVZehZYzHK1cW0WpiaXXWsUYYK-2Ek37EH8LdHI8uHbDacsKRc~mVgYoi4tnwVhnFKDD78mBrgIPggNaxYM-3T5QBdOEdFYI3Wy2IAl2~r8dUJomWgDrOkrjPKkOnreP2-36iZkbSb1MvJFHCG3YqEZ8OuZ1axTQ8WEP6YE785luLEFMDXJCJlJmZspOy50Wx4Un6gNGVnIXf0-4oNaY9CLwr3sxs7vTeOa8OmaYTA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7931392&ar=510716&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Influence of one nucleotide deletion on TP of nucleotide sequence. Numbers above sequence <em>S</em>1 show positions of nucleotides in RF. Twenty-fifth base has been deleted from sequence <em>S</em>1. Consequently, sequence <em>S</em>1 can be presented as two sequences—<em>S</em>2 and <em>S</em>3, i.e. <em>S</em>1 = <em>S</em>2<em>aS</em>3. In <em>S</em>2 and <em>S</em>3 sequences, TP (matrices <em>M</em>2 and <em>M</em>3) will be the same, but in sequence <em>S</em>3, there will be cyclical shifts by one base relative to sequence <em>S</em>2 and to RF in sequence <em>S</em>1. This means that first column of matrix <em>M</em>2 corresponds to third column of matrix <em>M</em>3, second column of matrix <em>M</em>2 corresponds to first column of matrix <em>M</em>3, third column of matrix <em>M</em>2 corresponds to second column of matrix <em>M</em>3. After summing sequences <em>S</em>2 and <em>S</em>3 and formation of sequence <em>S</em>4, TPM (<em>M</em>4 matrix) comes out from addition of first column of matrix <em>M</em>2 to first column of matrix <em>M</em>3 and so on, that leads to merging of non-identical columns and considerably decreases statistical significance of TP in sequence <em>S</em>4. By accounting deletions, we get sequence <em>S</em>4 that has TPM (<em>M</em>4 matrix) in which matrices <em>M</em>2 and <em>M</em>3 are merged in consideration of cyclic permutation. Finding and accounting deletions considerably increases statistical significance of TP in sequence <em>S</em>4.</p></div></div><div content-id="DSP002F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00201.jpeg?Expires=2147483647&Signature=HccxA6uMU9ADCH~NQHcrNRjfl-svb6DlzMtrbD60qU~9jN5ZteJIIv-E6Ymsv955W0PV-ojwGyj8tChDrgoOVOZ7QEUrVpaBngzxWa3pOBP0~v7Fc-Pu0gLtilo7YB3CnTl5GGNST0jI3HCTRwqrdbeg5~y-L7Q10kvfmeo1LhtixIiBp6aj~SfDlj8jyEcng6yZkZTUF37wsS32kzs7ja8yT59F5rQz2pPzRmr5--xp1VbyQPzmaPt2Pzv1ARcMgeiugf5ncjFmdgnj64mj6Cd4OGrseM7YCj82yNMYVeMdtVXU4zDFNAr7CNNE0jimxJ2jtux156MoWbMnpQHGXQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Influence of one nucleotide deletion on TP of nucleotide sequence. Numbers above sequence S1 show positions of nucleotides in RF. Twenty-fifth base has been deleted from sequence S1. Consequently, sequence S1 can be presented as two sequences—S2 and S3, i.e. S1 = S2aS3. In S2 and S3 sequences, TP (matrices M2 and M3) will be the same, but in sequence S3, there will be cyclical shifts by one base relative to sequence S2 and to RF in sequence S1. This means that first column of matrix M2 corresponds to third column of matrix M3, second column of matrix M2 corresponds to first column of matrix M3, third column of matrix M2 corresponds to second column of matrix M3. After summing sequences S2 and S3 and formation of sequence S4, TPM (M4 matrix) comes out from addition of first column of matrix M2 to first column of matrix M3 and so on, that leads to merging of non-identical columns and considerably decreases statistical significance of TP in sequence S4. By accounting deletions, we get sequence S4 that has TPM (M4 matrix) in which matrices M2 and M3 are merged in consideration of cyclic permutation. Finding and accounting deletions considerably increases statistical significance of TP in sequence S4." path-from-xml="dsp00201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7931392/dsp00201.jpeg" path-from-xml="dsp00201.jpeg" target="_blank">View large</a><a section="7931392" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/dsp00201.jpeg?Expires=2147483647&Signature=XxtUyaP23OaBLHY5ncCMzy1-f2IFebkeLQ5frd8YAYvQYNLgdyF1t~UL4xilp1K~ERgYUosZ059xFzY3EEXj3v-ufSD8fs8TlDD3YteSTdGDtVZehZYzHK1cW0WpiaXXWsUYYK-2Ek37EH8LdHI8uHbDacsKRc~mVgYoi4tnwVhnFKDD78mBrgIPggNaxYM-3T5QBdOEdFYI3Wy2IAl2~r8dUJomWgDrOkrjPKkOnreP2-36iZkbSb1MvJFHCG3YqEZ8OuZ1axTQ8WEP6YE785luLEFMDXJCJlJmZspOy50Wx4Un6gNGVnIXf0-4oNaY9CLwr3sxs7vTeOa8OmaYTA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7931392&ar=510716&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Influence of one nucleotide deletion on TP of nucleotide sequence. Numbers above sequence <em>S</em>1 show positions of nucleotides in RF. Twenty-fifth base has been deleted from sequence <em>S</em>1. Consequently, sequence <em>S</em>1 can be presented as two sequences—<em>S</em>2 and <em>S</em>3, i.e. <em>S</em>1 = <em>S</em>2<em>aS</em>3. In <em>S</em>2 and <em>S</em>3 sequences, TP (matrices <em>M</em>2 and <em>M</em>3) will be the same, but in sequence <em>S</em>3, there will be cyclical shifts by one base relative to sequence <em>S</em>2 and to RF in sequence <em>S</em>1. This means that first column of matrix <em>M</em>2 corresponds to third column of matrix <em>M</em>3, second column of matrix <em>M</em>2 corresponds to first column of matrix <em>M</em>3, third column of matrix <em>M</em>2 corresponds to second column of matrix <em>M</em>3. After summing sequences <em>S</em>2 and <em>S</em>3 and formation of sequence <em>S</em>4, TPM (<em>M</em>4 matrix) comes out from addition of first column of matrix <em>M</em>2 to first column of matrix <em>M</em>3 and so on, that leads to merging of non-identical columns and considerably decreases statistical significance of TP in sequence <em>S</em>4. By accounting deletions, we get sequence <em>S</em>4 that has TPM (<em>M</em>4 matrix) in which matrices <em>M</em>2 and <em>M</em>3 are merged in consideration of cyclic permutation. Finding and accounting deletions considerably increases statistical significance of TP in sequence <em>S</em>4.</p></div></div><p>Currently, some methods have been developed that reveal TP by using regularity in symbol preferences over different triplet positions in the DNA sequence. They use Fourier transformation, hidden Markov chains and other statistical methods based on position-dependent preferences for nucleotides in coding sequences as a mathematical apparatus.<sup><a reveal-id="DSP002C18  DSP002C23" data-open="DSP002C18  DSP002C23" class="link link-ref link-reveal xref-bibr">18–23</a></sup> The methods are aimed at revealing DNA coding sequences and their separation from non-coding regions. A later, method of information decomposition to find TP <sup><a reveal-id="DSP002C17" data-open="DSP002C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSP002C24" data-open="DSP002C24" class="link link-ref link-reveal xref-bibr">24</a></sup> allows the introduction of the term of a TP class (TPC) as a 3 × 4 matrix. In this matrix, columns represent period positions and rows represent nucleotides.</p><p>In the current work, two problems were set. First, we wanted to find all genes where RF shifts can be identified by using TP. For each gene from the KEGG-29 databank, analysed<sup><a reveal-id="DSP002C25" data-open="DSP002C25" class="link link-ref link-reveal xref-bibr">25</a></sup> we extracted a region with TP having maximal statistical significance calculated by information decomposition without allowing any deletions or insertions of nucleotides.<sup><a reveal-id="DSP002C17" data-open="DSP002C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSP002C24" data-open="DSP002C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Then we built a corresponding matrix of TP which was linked to the existing RF of the given DNA region. This meant that the first column of the TP matrix (TPM) corresponded to the first base of ORF presented in the DNA region having TP. Then we searched for a statistically significant extension of the TP region in the same gene in the presence of insertions and deletions of nucleotides by using modified profile analysis (Fig. <a reveal-id="DSP002F1" data-open="DSP002F1" class="link link-reveal link-table xref-fig">1</a>). More than 800 genes contained a statistically significant shift between TP and ORF, which points to the presence of mutations in genes originating from the RF shift.</p><p>Second, we wished to check whether hypothetical amino acid sequences translated by using the RF of TP have homology with sequences from UniProtKB/Swiss-Prot databank (<a class="link link-uri" href="http://www.uniprot.org/">http://www.uniprot.org/</a>). We made such a check for the genes that had mismatches between the gene's RF and TP. We confirmed the existence of such shifts for a part of the genes, since we found similarities between hypothetical amino acid sequences and amino acid sequences from the UniProtKB/Swiss-Prot databank.</p>                            <h2 scrollto-destination=7931396 id="7931396" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=7931397 id="7931397" class="section-title" data-legacy-id=s2a>2.1. Searching for TP in genes by method of information decomposition</h3>
<div class="block-child-p">For each nucleotide sequence <em>S</em> = {<em>s</em>(<em>i</em>), <em>i</em> = 1, 2, … , <em>L</em>}, we carried out a search for a region having maximally expressed TP by the method of information decomposition.<sup><a reveal-id="DSP002C17" data-open="DSP002C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSP002C24" data-open="DSP002C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Let <em>s</em>(<em>i</em>) be letters of the alphabet <em>A</em> = {<em>a</em>,<em>t</em>,<em>c</em>,<em>g</em>}, <em>A</em>(1) = <em>a</em>, <em>A</em>(2) = <em>t</em>, <em>A</em>(3) = <em>c</em> and <em>A</em>(4) = <em>g</em>. Then we also make an artificial periodic sequence <em>U</em> = {<em>u</em>(<em>i</em>), <em>i</em> = 1, 2, … , <em>L</em>} of the same size as nucleotide sequence <em>S</em>. In artificial sequence, <em>u</em>(<em>i</em>) = 1 for <em>i</em> = 1 + 3<em>n</em>, <em>u</em>(<em>i</em>) = 2 for <em>i</em> = 2 + 3<em>n</em>, <em>u</em>(<em>i</em>) = 3 for <em>i</em> = 3 + 3<em>n</em>, where <em>n</em> = 0, 1, 2, … . Then we choose coordinates <em>L</em><sub>1</sub> and <em>L</em><sub>2</sub> starting from the beginning of nucleotide sequence and fill the matrix <em>M</em><sup><a reveal-id="DSP002C4" data-open="DSP002C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSP002C3" data-open="DSP002C3" class="link link-ref link-reveal xref-bibr">3</a></sup> for selected subsequence. Element of matrix <em>m</em>(<em>k</em>,<em>j</em>) shows how many times symbol <em>A</em>(<em>k</em>) in nucleotide subsequence from <em>L</em><sub>1</sub> to <em>L</em><sub>2</sub> matches the number <em>j</em> in artificial periodical sequence <em>U</em>. Each column <em>j</em> of the matrix <em>M</em>(<em>k</em>,<em>j</em>) shows number of bases <em>a</em>, <em>t</em>, <em>c</em> and <em>g</em> that occur in positions <em>i</em> = <em>j</em> + 3<em>n</em> of a sequence <em>S</em>, where <em>n</em> = 0, 1, 2, … . We calculate mutual information as<sup><a reveal-id="DSP002C26" data-open="DSP002C26" class="link link-ref link-reveal xref-bibr">26</a></sup><a id="DSP002M1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSP002M1"><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/dsp00206.jpeg?Expires=2147483647&Signature=jWEelRMuE-L6Ronuf7PQy1eEDBSBMYdemCe3sLWEJExrPJKjnbRQjzVbUa2uScOk1Ce0gESIdJZ5ZdiQAgcZE2ZHDQY5sYa97kBUoFKQOF6EocRBZVagFtEub6jnw4XUhVBxTUe5Z-Gd9UFQuOj7iXUhFQW~-djWj2UqAYzrTvaMRBzUe6wd0SrgmLO~JMzhPLcZdhUftkIb8asWvandvmcqGxonPI~DrlUyHsFli5FK7BogWMKo~qcDDcbSPGm0z0e2xFs-gBaoIy510~qgJ3cgu~jc46~PeZTq4SW3ZGfjjYZ6C8qzseV~9V9ZyCgePnTddpmbWhoAe~Fg7nAnuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsp00206.jpeg"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00206.jpeg?Expires=2147483647&Signature=UdkeZcNQm-7TaSj2HyYbOkBWcG28GisaSJYNHgA3PfAzvif~rq4Ukwfraid0ZuwpTZ9WVyXE73LyHTRZa6O6HpsINCtzKxpAlq5RPKUc1E~wdmSG9apIMHWNyLYEZpvAv35KaN7VMjolaNp6t1EpccSWJ6yCrViWZoW-jJsKDSlFHrIP~DmnZ53N7~z1PfllUx~gR4XmMorCnaI~qI6wAgGZRcY7AwVQx2CzE4BF7fx~lJGswycZGyYDJGF6CO2Q8a3J77kT4U35bknlgK73Xupu62Ra6DHC8WJBTEI5UvAtVJQ6uI3FSPeXgucu1jPQc4O8rFUvGuSgD0pfloIp0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsp00206.jpeg" /></a></div></div><div class="label">1</div></div> where <em>x</em>(<em>i</em>) and <em>y</em>(<em>j</em>) are the frequencies of nucleotide occurrences in sequence <em>S</em> and of numbers 1, 2 and 3 occurrences in artificial sequence, correspondingly.</div><p>We used gene sequences from KEGG-29 databank as source sequences <em>S</em>. All analysed sequences represent coding region (CDS) of genes without introns. That is why, when <em>L</em><sub>1</sub>−1 and <em>L</em><sub>2</sub> are set to a multiple of 3 while defining sequences <em>S</em> and <em>U</em>, then first, second and third columns of matrix <em>M</em> for any values of <em>L</em><sub>1</sub> and <em>L</em><sub>2</sub> will always contain nucleotides corresponding to first, second and third bases of the gene's codon, respectively. In other words, matrix <em>M</em> is linked to RF, which exists in the analysed gene.</p><div class="block-child-p">Doubled mutual information 2<em>I</em> has <em>χ</em><sup>2</sup> distribution with six degrees of freedom. This allows us to estimate statistical significance of the periodicity found. We can reduce <em>I</em> to standard normal distribution:<sup><a reveal-id="DSP002C27" data-open="DSP002C27" class="link link-ref link-reveal xref-bibr">27</a></sup><a id="DSP002M2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSP002M2"><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/dsp00207.gif?Expires=2147483647&Signature=dHkOTGEs0mSYqKy~JkDmHtsTDj7w0LbTYzXnE7GsS3m5F6-eHe7nqIA3Wq-r~0ozFFRSzY~FHj4rBg8lPF~ZeM71Y3sNGVC82kOeZuIvUVcSAbcLFs7x6kpCiIOM67y9twFL~sdmdH5nkul2e4XbrzCpNCTdrjvIHpc7rEqRqV6oTrxs8EVR1qFLcswen3E7qfCWy8RfVyPJZk8nDAB20xwPDR6MTmz8gOwBJuGgI0pnHTAfzvBCLtytwOUIyKhM0BARXWQ2-bOzTizSfbxi8JTATa~QE2JkI6BKQkNoMOeyomAWHRrKgP4Qr8t4RwRCfYDsgDpj5IPEXjWT7aE0UQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsp00207.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00207.gif?Expires=2147483647&Signature=2Aq5goQDrpB4hQ682FuID1iQ4RYM0KdPNN2dHTKdJDteYqsZQuMYmCVtBEkLiILS18BNSq-tsiZgo~nwjBfiIeRZrOk2B-3zRuc~jKYVJ4PXJ05gU97DZWC~sgnnTm4LKW-j327WuUUKirDjynJI~voc5h-YEIkmMTj2b706dWv9EGjlIcnbrOQbpIEWbUsKYMkrQ90Z1EX68CE7juRmBbRqu86URYOfITjbYC6hYL6j2KaXUEp5JByEFrs0p9drMd33~dDBin5rZvfYXVuzOkhe88W-ofIaLFjQNIGWPgCveNTZDzeekpzTR-iStsBml7rP2JPpswrbhTwYjnCUhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsp00207.gif" /></a></div></div><div class="label">2</div></div></div><p>Conformity of 2<em>I</em> to <em>χ</em><sup>2</sup> distribution with six degrees of freedom and of value <em>Z</em> to standard normal distribution are reached in the case of sufficiently large size of statistical data sample, i.e. sufficiently large length of a sequence <em>S</em>. In order to determine the minimal length of a sequence <em>S</em> that makes possible the usage of function <em>χ</em><sup>2</sup> as approximation for 2<em>I</em> value distribution, we tested conformity of 2<em>I</em> to <em>χ</em><sup>2</sup> distribution for various lengths of sequences <em>S</em>. We produced a set of nucleotide sequences for each length in the range from 30 to 1000 nucleotides by using a random number generator. Each of these sets contained 10 000 sequences. Thereafter, mutual information was calculated for each sequence from each set. For each set the histogram showing distribution of 2<em>I</em> value was also built. We compared this histogram with the theoretical distribution by <em>χ</em><sup>2</sup> criteria. It was found that for sequences of length over 60 bp, 2<em>I</em> distribution conformed to <em>χ</em><sup>2</sup>(6) with a probability more than 99%. All sequences with TP, which were found in the current work, were longer than 60 bp. This allows using <em>χ</em><sup>2</sup> distribution for statistical estimations of hitting 2<em>I</em> into interval from some threshold value 2<em>I</em><sub>0</sub> to infinity.</p><p>For sequence <em>S</em>, we calculated the values of 2<em>I</em> for all possible values of <em>L</em><sub>1</sub> and <em>L</em><sub>2</sub> (<em>L</em><sub>1</sub> &lt; <em>L</em><sub>2</sub> ≤ <em>L</em>) and chose the pair (<em>L</em><sub>1</sub>,<em>L</em><sub>2</sub>) for which the value of mutual information was maximal. Let us refer to the nucleotide sequence found in such a way as <em>T</em>.</p><p>If the value of <em>Z</em> was &gt;5.0 for sequence <em>T</em>, then we considered that the region with TP has been found. The value of <em>Z</em> &gt;5.0 ensures the probability of incidental TP revealing in DNA sequence to be &lt;10<sup>−6</sup>. Thereafter, we saved the found maximal sequence for the given gene, its coordinates in the given gene and periodicity matrix <em>M</em>, which shows the type of TP found. We chose a threshold level <em>Z</em> &gt; 5.0 for finding TP in order to keep the number of incidentally found TPs near 1% of all detected regions with TP found in genes from the 29th release of the KEGG databank. To choose a threshold value for <em>Z</em>, we generated a set of random DNA sequences with the same size and sequence length distribution as for genes from the 29th release of KEGG databank. For <em>Z</em> &gt; 5.0, the number of found random sequences was 7200, which is ∼1.5% of found regions with TP (see below). For <em>Z</em> &gt; 6.0, we found 172 such sequences, and for <em>Z</em> &gt; 7.0, we found no such sequences. We intentionally chose the level of <em>Z</em> &gt; 5.0 in order to find the most complete extension of TP regions (see Section 2.2) that exist in various genes. The point is that gene's TP can be split up by insertions and deletions into several sections that may have rather low level of <em>Z</em>, but which is greater than 5.0. However, matrices <em>M</em> for each such section in gene will be identical or very similar, but cyclically shifted against each other (Fig. <a reveal-id="DSP002F1" data-open="DSP002F1" class="link link-reveal link-table xref-fig">1</a>). In this case, consequent joining of these sections into a single one can considerably increase the statistical significance of a joined region that can be found by making an alignment against matrix <em>M</em> (see Section 2.3). Therefore, using a relatively low threshold value of <em>Z</em> will allow to not miss TP regions in genes separated into several sections by insertions and deletions.</p>                            <h3 scrollto-destination=7931404 id="7931404" class="section-title" data-legacy-id=s2b>2.2. Algorithm of TP region extension in genes from KEGG databank</h3>
<p>We applied this algorithm for those DNA sequences in which we have revealed TP without insertions and deletions by the method of information decomposition. Let <em>S</em> = {<em>s</em>(<em>i</em>), <em>i</em> = 1, 2, … , <em>L</em>} be the analysed nucleotide sequence from KEGG databank and <em>t</em><sub>1</sub> and <em>t</em><sub>2</sub> be coordinates of the left and right borders of a region <em>T</em> with continuous TP in sequence <em>S</em>. For region <em>T</em>, we determined TPM and used this matrix to extend the TP region by considering possible nucleotides’ insertions and deletions. Then we carried out local alignment of examined DNA sequence against weight matrix <em>w</em> introduced on the basis of TPM. Let the coordinates of start and end of found local alignment <em>R</em> be <em>r</em><sub>1</sub> and <em>r</em><sub>2</sub>. Under extension of TP region <em>T</em>, we mean those <em>R</em> such that <em>r</em><sub>1</sub> &lt; <em>t</em><sub>1</sub> or <em>r</em><sub>2</sub> &gt; <em>t</em><sub>2</sub>. Our goal is to choose from the KEGG databank only the genes that contain statistically significant extension of the TP region <em>T</em>. To find statistically significant extensions, we carried out global alignment of sequence <em>S</em> against weight matrix <em>w</em> and determined the values Δ<em>F</em><sub><em>T</em></sub> = <em>F</em>(<em>t</em><sub>2</sub>) − <em>F</em>(<em>t</em><sub>1</sub>) and Δ<em>F</em><sub><em>R</em></sub> = <em>F</em>(<em>r</em><sub>2</sub>) − <em>F</em>(<em>r</em><sub>1</sub>), where <em>F</em> is the value of similarity function on path of global alignment (Fig. <a reveal-id="DSP002F2" data-open="DSP002F2" class="link link-reveal link-table xref-fig">2</a>). We selected only those genes that had Δ<em>F</em><sub>1</sub> = Δ<em>F</em><sub><em>R</em></sub> − Δ<em>F</em><sub><em>T</em></sub> &gt; 0. Thereafter, we have to determine whether the value of Δ<em>F</em><sub>1</sub> is statistically significant. To do this, we used the Monte Carlo method. We generated a set of random nucleotide sequences <em>Q</em> on which the region <em>T</em> was left unchanged and the regions of sequence <em>S</em> within the range from 1 to <em>t</em><sub>1</sub> and from <em>t</em><sub>2</sub> to <em>L</em> were shuffled in a random way. The set <em>Q</em> contained 10<sup>6</sup> sequences. For each sequence from the set <em>Q</em>, we built global alignment and determined the value Δ<em>F</em><sub><em>R</em></sub> − Δ<em>F</em><sub><em>T</em></sub>. Then we calculated such value Δ<em>F</em><sub>0</sub> that the probability of Δ<em>F</em><sub><em>R</em></sub> − Δ<em>F</em><sub><em>T</em></sub> ≥ Δ<em>F</em><sub>0</sub> for the set of sequences <em>Q</em> was ∼10<sup>−5</sup>. We chose the value Δ<em>F</em><sub>0</sub> as a threshold and considered that if Δ<em>F</em><sub>1</sub> &gt; Δ<em>F</em><sub>0</sub>, then we found statistically significant extension of the region with continuous TP up to the bounds of region <em>R</em> via considering possible insertions and deletions of nucleotides. We analysed ∼3 × 10<sup>5</sup> genes in such a way, and detected from 0 to 8 cases of region <em>T</em> extension due to accidental factors with a discovering probability of the given interval equal to 95%. As we revealed more than 750 genes with extension of the TP region <em>T</em>, then the fraction of genes with extension of region <em>T</em> due to merely accidental factors is ∼1.1% which is a relatively small value.</p>                        <a id="7931406" scrollto-destination="7931406"></a>
<div id="DSP002F2" data-id="DSP002F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00202.jpeg?Expires=2147483647&Signature=YxuPDO-bje9LReACILiVocz88qr1fNaYoU4ZdJFC2W20RU70kHIcz-pWaUdikPYyVucXlaF2WilxZEU9395rz4fDRx8PXdU2bSRiMdYoOVYhC0kUlIEFXVt3tX1zi0zwHKVqQPXGZbMvZaN14l1lL63e4MhogcnM64TPeRvYHvdG-RzanOoGz0VSJ7W6uP2YFFYBU1B49Jia9y-Um785e5WmxhNVGZjdH5~FF5zY3y6nyQLNasrJtPrn7RW-9HyMGtKvm70jBu6oJiGmZkS16-5iWVPxp0N1DqJybiKHuMOYcYo779qfIYlZKfIXl-Tc5StcYf97fb77MqhphJ5Aag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Example of alignment against weight matrix obtained from TPM. Here, t1 and t2 are coordinates of region with continuous TP found by information decomposition (T region) and r1 and r2 are coordinates of extended region with TP found by dynamic programming (R region). Points r1 and r2 on optimal alignment path have the coordinates (i0,j0) and (im,jm) in matrix, respectively (see Section 2.3)." path-from-xml="dsp00202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7931406/dsp00202.jpeg" path-from-xml="dsp00202.jpeg" target="_blank">View large</a><a section="7931406" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/dsp00202.jpeg?Expires=2147483647&Signature=Y3LFUEEQjSYYMWKmoAqqeKXJq7oDOoq9KISSBL2We0LAFl~r--BI1nDxRYuVH6Sv6~kxAxfU2yHY6nU6aIHeSrpOiLhc06DQgNRZ5SP587ZRyhz11Zao-4uXV0WNvPG7G99RnJZMLR~-pjUZzaPXNeteSF4tGPO-2UQpbC9AKHjVsendstdbxhUclT6R4kDVHk~5ENJ3d8oHP~JWXSSl36yregBY9jV4jDWF13dCbS1PQRlIJDbxg0Xi3Hqvx23z297FjAzEpNZqbm0jCTgGuvOKtx2GaURppTV9cnXrKi8RrOJbrjcmhWLgHeAH-zAzspjpOR2G327Od4~FLEd4oA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7931406&ar=510716&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Example of alignment against weight matrix obtained from TPM. Here, <em>t</em><sub>1</sub> and <em>t</em><sub>2</sub> are coordinates of region with continuous TP found by information decomposition (<em>T</em> region) and <em>r</em><sub>1</sub> and <em>r</em><sub>2</sub> are coordinates of extended region with TP found by dynamic programming (<em>R</em> region). Points <em>r</em><sub>1</sub> and <em>r</em><sub>2</sub> on optimal alignment path have the coordinates (<em>i</em><sub>0</sub>,<em>j</em><sub>0</sub>) and (<em>i</em><sub>m</sub>,<em>j</em><sub>m</sub>) in matrix, respectively (see Section 2.3).</p></div></div><div content-id="DSP002F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00202.jpeg?Expires=2147483647&Signature=YxuPDO-bje9LReACILiVocz88qr1fNaYoU4ZdJFC2W20RU70kHIcz-pWaUdikPYyVucXlaF2WilxZEU9395rz4fDRx8PXdU2bSRiMdYoOVYhC0kUlIEFXVt3tX1zi0zwHKVqQPXGZbMvZaN14l1lL63e4MhogcnM64TPeRvYHvdG-RzanOoGz0VSJ7W6uP2YFFYBU1B49Jia9y-Um785e5WmxhNVGZjdH5~FF5zY3y6nyQLNasrJtPrn7RW-9HyMGtKvm70jBu6oJiGmZkS16-5iWVPxp0N1DqJybiKHuMOYcYo779qfIYlZKfIXl-Tc5StcYf97fb77MqhphJ5Aag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Example of alignment against weight matrix obtained from TPM. Here, t1 and t2 are coordinates of region with continuous TP found by information decomposition (T region) and r1 and r2 are coordinates of extended region with TP found by dynamic programming (R region). Points r1 and r2 on optimal alignment path have the coordinates (i0,j0) and (im,jm) in matrix, respectively (see Section 2.3)." path-from-xml="dsp00202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7931406/dsp00202.jpeg" path-from-xml="dsp00202.jpeg" target="_blank">View large</a><a section="7931406" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/dsp00202.jpeg?Expires=2147483647&Signature=Y3LFUEEQjSYYMWKmoAqqeKXJq7oDOoq9KISSBL2We0LAFl~r--BI1nDxRYuVH6Sv6~kxAxfU2yHY6nU6aIHeSrpOiLhc06DQgNRZ5SP587ZRyhz11Zao-4uXV0WNvPG7G99RnJZMLR~-pjUZzaPXNeteSF4tGPO-2UQpbC9AKHjVsendstdbxhUclT6R4kDVHk~5ENJ3d8oHP~JWXSSl36yregBY9jV4jDWF13dCbS1PQRlIJDbxg0Xi3Hqvx23z297FjAzEpNZqbm0jCTgGuvOKtx2GaURppTV9cnXrKi8RrOJbrjcmhWLgHeAH-zAzspjpOR2G327Od4~FLEd4oA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7931406&ar=510716&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Example of alignment against weight matrix obtained from TPM. Here, <em>t</em><sub>1</sub> and <em>t</em><sub>2</sub> are coordinates of region with continuous TP found by information decomposition (<em>T</em> region) and <em>r</em><sub>1</sub> and <em>r</em><sub>2</sub> are coordinates of extended region with TP found by dynamic programming (<em>R</em> region). Points <em>r</em><sub>1</sub> and <em>r</em><sub>2</sub> on optimal alignment path have the coordinates (<em>i</em><sub>0</sub>,<em>j</em><sub>0</sub>) and (<em>i</em><sub>m</sub>,<em>j</em><sub>m</sub>) in matrix, respectively (see Section 2.3).</p></div></div><div class="block-child-p">Also, we estimated statistical significance <em>Z</em><sub><em>R</em></sub> of alignment <em>R</em> by the Monte Carlo method in a way we described earlier.<sup><a reveal-id="DSP002C28" data-open="DSP002C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSP002C29" data-open="DSP002C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Thereto, we generated a set of sequences <em>QR</em> in which the region <em>R</em> was randomly shuffled. Then for each of these random sequences from <em>QR</em> set, we built global alignment and for each alignment we determined the values Δ<em>G</em> = <em>G</em>(<em>r</em><sub>2</sub>) − <em>G</em>(<em>r</em><sub>1</sub>). <em>G</em> is the similarity function for global alignment <em>R</em>, and it is calculated as we do for function <em>F</em>. For a set of Δ<em>G</em> values, we determined mean value <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00222.gif?Expires=2147483647&amp;Signature=KhOMGy5l3RVYDvle0nFPWiJ2jsmvzFTFnFLLK~ozs7vI8uaA5HX3f95OcvAHBFZrKm7Vp7xNG2TNaCxbIu6c4nYtPLQwrYsjFZHzbOutFy6fBZ4FB1OtEPGcqngZZgx0e2QJ2J-~kh6n0MbrLEnMwxs5Qgj-P4ZUwFH5rILtciYJZs8tmENDD6~E5Br2iHREnaxEmgKx3GfYvoh--PPNY788WPJOIBFVRjW0X7AoeXFg7vqpbP961jF-VJtrg13EQFkLk0iZw~3DDAJjN3w1ozW1mPKxxkLenm3RWFSmVROHS-Ogw3y0tTrd0rppxqnMC3UIFGNAvsDLweS291XEIQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp00222.gif" /></span> and <em>D</em>(Δ<em>G</em>), where <em>D</em>(Δ<em>G</em>) is the dispersion of Δ<em>G</em>. Then we determined the value of <em>Z</em><sub><em>R</em></sub> as: <a id="DSP002M3" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSP002M3"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsp00208.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00208.gif?Expires=2147483647&amp;Signature=HE4qQq3mW43w0UpHopJ1ZLUFNEelrS7bGYdaJg5Hr1ihOADLtIjFszQ5DDhkrcCVxE91p7DUqCuGwK6264uP6uHQA3zBGnVniPjAkoxYK7hz9myQj24f0uC7yIz2Aqs7CFZ7R~o--J~bcubPae7eVaVTdAoCMax7LmPMQU9hfoILdPD0iSv7Gj1e5qAC5w5FT0lx65UJzU94WiAczoTmTipY-etOFScl~pc4WfUMaHyelilgqBLh90fZ6bVLOePCAwW375GixfSGrSQ8rKJH2GMazXnscbWySRV0U8DB6afE433ifnhq7vlneg9do1Yd~f72gOLFjTOYN9Sd02NBSw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsp00208.gif" /></a></div></div><div class="label">3</div></div> where Δ<em>G</em><sub>0</sub> = <em>G</em>(<em>r</em><sub>2</sub>) − <em>G</em>(<em>r</em><sub>1</sub>) for original sequence <em>R</em>.</div>                            <h3 scrollto-destination=7931408 id="7931408" class="section-title" data-legacy-id=s2c>2.3. Implementation of local and global alignment of nucleotide sequence against TPM</h3>
<div class="block-child-p">During carrying out local and global alignment, we used matrices of TP <em>M</em> obtained for regions <em>T</em> of each gene from 29th release of KEGG databank. Using the matrix <em>M</em>, we built corrected position-specific matrix of the base weights as we suggested earlier:<sup><a reveal-id="DSP002C28" data-open="DSP002C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSP002C29" data-open="DSP002C29" class="link link-ref link-reveal xref-bibr">29</a></sup><a id="DSP002M4" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSP002M4"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsp00209.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00209.gif?Expires=2147483647&amp;Signature=gWirFTffRdjjJ7bpMpi8H1bypkqq7X~KDdTtxU3YGfR9JOSRRjEEaZX2RWaKOfaQBuIwe-I9qZy2fXBfuAT~FQCXUpFHtFyuxTtEA4I7AjgMUo~41gKP-yRWmeFfpltarA9um8PS9~5blnCVhJgVB8dKjFcJh-u~21RlF82JNWC18S6bp9qaXIx5Zatv-vD10LuU-NibHfGZUFkaRqhHF0cpo-OTWzzW79HL2kRjRmcC7LxNpGJp8w9ssAdvE0G4mtjtx6QuIn6IiR9WGOw7qSSnM0KqV13kI0QoYZy-kvxzNrejjWaAzCoulIfYVwnBAOt~LsUGYDdptBiNaD1PLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsp00209.gif" /></a></div></div><div class="label">4</div></div> where <em>w</em>′(<em>i</em>,<em>l</em>) is the uncorrected weight of the base <em>a</em><sub><em>i</em></sub> at position <em>l</em> of profile. Corrected weight matrix <em>W</em> was calculated as: <a id="DSP002M5" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSP002M5"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsp00210.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00210.gif?Expires=2147483647&amp;Signature=Q4OaFKneXUKu5lmBZQV7iIL87wJziVrqBRgl4x4HPNXz3NaFxUU-S6y~a-SWydPZOKp-2uO1hmZIjNHWBEc2HPcw9DjraPLVS5psa4gZ2AG-3CnztMjDSVoRNo1NDpWquJGM~C6YCXONx~H5js9O8V62dup-7bduUUoJJ6jLbYaru-aWEAxXcOFYqyTDnCzJKnBAD0j58yZPTT4bGkgLAWjWBVrMjAPqEmROPPYFw0wg6OuljulNKbCD4kCCeMjYFhJNARjQdesqSTMq31dnBNkbetZNnKZKyY4rgs~mbqGuhSnsrAwMbXOfXNgQEyOj2CA~L46RpNo-lzUaRlm83A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsp00210.gif" /></a></div></div><div class="label">5</div></div> where <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00223.gif?Expires=2147483647&amp;Signature=aZ~yRsNyWXW8Qe6o0TEy7cNuUoPjkMdJbO9S0cr2CDJQd0eKDB1FA9hHY~6jwCnHzWpqwjOfHQpZNHYvAr434vQl1UONK46NRlehlqWRjEacMe~Ms84CcRiARngLfRJ~DYT80wbfJzfyIMCIeO8KHQyQyZuAyULxm3wPnGjKdPLAuTQJ6Y7Hhz8q-fLP8-zLdBf1VKcerdfAdsUGrT-pVYE6EMtL1EK8~qqqb9f50glJwal6Hs0BjGFXL-xH6Rzmfvdtj5pKqu96hJb8~aFpXh4dVUoZP1UTbd4mLMsjQO6aWK5B9AyGk7NZT6QZd5AMm-a6CJDMgoN9q35sEEwAPw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp00223.gif" /></span> is the mean value of the uncorrected weight matrix. We added unity to each position of matrix <em>m</em>(<em>i</em>,<em>l</em>) to eliminate the influence of zero elements in matrix <em>M</em> on the weights <em>w</em>′(<em>i</em>,<em>l</em>). The value of <em>C</em> was selected for each matrix <em>M</em> using the <em>QV</em> sequences (see Section 2.4). Thereto, we varied <em>C</em> from interval [−6,+6] with a step equal to 0.2. For each value of <em>C</em> from the interval, we determined alignments for <em>N</em> sequences from <em>QV</em> set, <em>N</em> = 100. For each alignment, we estimated <em>Z</em><sub><em>T</em></sub> (the statistical significance of <em>T</em> region alignment) by the Monte Carlo method (similar to Section 2.2). Thereto, we generated a set of sequences <em>QT</em> (for each sequences from <em>QV</em> set) in which the region <em>T</em> was randomly shuffled. Then we built a global alignment for each random sequence from <em>QT</em> set, and for each alignment, we determined the values Δ<em>G</em>(<em>T</em>) = <em>G</em>(<em>t</em><sub>2</sub>) − <em>G</em>(<em>t</em><sub>1</sub>). <em>G</em>(<em>T</em>) is the similarity function for global alignment of region <em>T</em> for DNA sequences from <em>QT</em> set. For a set of Δ<em>G</em>(<em>T</em>) values, we determined the mean value <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00224.gif?Expires=2147483647&amp;Signature=eeBf9k1ldc9fBn7hzSWvqROCA7qeGr7IL0Bc-gaRGhND6Yy~PZUwo9TbJG3iDSXuwBzs2K04Xr4yg484oHp024YXz78yJNzIKUaiAGlR6GWG~usKXV8BTFFv5oGxzbfQva1HmKvLZJ5Edt-aHE~RKDXkM8FCWiEu6CTfZjn7U7dxIVA~aYQZzD~Nxckznot3NZQrB399NbOLw5opJHYuywpwHhHLY3DCSClwwjwCWG1cUbBJ46Io1QuHLJ4qbH54GqUbHjW9NJjj~rgtg2wsrXmkCLe2Yh64rv8PZ2yQXLFSI0u6X1Wz8HkU3v4RdDY3Au259EQrToBLxAsu8gKmKA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp00224.gif" /></span> and <em>D</em>(Δ<em>G</em>(<em>T</em>)), where <em>D</em>(Δ<em>G</em>(<em>T</em>)) is the dispersion of Δ<em>G</em>(<em>T</em>). Then we determined the value of <em>Z</em><sub><em>T</em></sub> as: <a id="DSP002M6" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSP002M6"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsp00211.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00211.gif?Expires=2147483647&amp;Signature=TBFPmNCt7LM2DdQCgdfuRQTsrGuUHE-gfphaqoGJWKPe-enU4FspFbIatSHeL1xeGIIw9pGSpDDp~37YadjsXY1VwNUJIQQhysbrGn4SbTVD36eQ~tdJ4y5OYQ7i0cfaTa6XmokxKd5e5ZK~zbGyDj9tfyhJk0ShETgUxlSYinVaPqavl2Tt6AtEDWl9i5CDmKqtAJUqJKOYdnhYRCOo5Mf5s8yDeqPe-G1Rc0FEC~hAjbsTuTk9oR82iAqR9-vQilRnfA5r1UWF-Se2A99UEHPY7JMO1ADSiXNOp2y1TN2YxJJuJDKjRYaJNcfbbHMVFm-P5Qn2EaHFtOlomrVouw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsp00211.gif" /></a></div></div><div class="label">6</div></div> where Δ<em>G</em>(<em>T</em>)<sub>0</sub> = <em>G</em>(<em>t</em><sub>2</sub>) − <em>G</em>(<em>t</em><sub>1</sub>) for original sequence <em>T</em> from <em>QV</em> set. Then we calculated: <a id="DSP002M7" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSP002M7"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsp00212.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00212.gif?Expires=2147483647&amp;Signature=gUnnrJBrH9zLnfhZv-iIP0jVTjPXYwdaSfqWP~Xk06tKoYC1ZYzbAQg1~Md9~n1820QWs4XtEBTHVfE3x0-Zd9dccOsIOzV207leWHe32gU52dXcdy8Hmn5S7sKeSBTKIFYUGal7DIM9KUJxelVzkZJqnQT1ZaX1G8xITrXuM6YjOflnaUhmbBVfcRx4GlBdQrahlFWuJSFkSdzZA1hRZHRTMUo8Htt7PXubyyQaxEW3yT33IX-HCYEmwoe0TLsXginXQflu7LqUec9utBC~O1qakpghlmN-PZQ6DI5AmePD3hveVVeaGGbJCjQWTWXcwuArOBp3AdOjIgjv820Mww__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsp00212.gif" /></a></div></div><div class="label">7</div></div></div><p>We take the sum in Equation (<a href="#DSP002M7" class="sectionLink xref-sec">7</a>) for <em>N</em> sequences from <em>QV</em> set. As a result, we have a set of <em>X</em><sub><em>C</em></sub> where each <em>C</em> from interval [−6,+6] has one <em>X</em><sub><em>C</em></sub>. We used for further calculation a value of <em>C</em> that has a maximum of <em>X</em><sub><em>C</em></sub>. We did the selection of <em>C</em> value for each matrix <em>M</em>.</p><p>Transition to weight matrix ensures assignment of higher weight to infrequent bases when they have high frequency in the given position of profile and, vice versa, assignment of lower weight for such bases having low frequency in the given position. To build optimal alignment of sought sequence against profile, we also introduced the weight for opening insertion or deletion <em>ν</em><sub><em>do</em></sub> and weight <em>ν</em><sub><em>dc</em></sub> for their continuation. Thereby the correlation in the formation of adjacent insertions or deletions is taken into account.</p><div class="block-child-p">On the basis of introduced weights, we can find the optimal alignment, between analysed sequence and profile, i.e. we find such their subsequences for alignment, that maximise the similarity function. Let <em>S</em> = {<em>s</em>(<em>j</em>), <em>j</em> = 1, 2, … , <em>L</em>} be the analysed sequence. Let us create a profile matrix <em>q</em>(<em>i</em>,<em>j</em>) of size <em>L</em> as: <a id="DSP002M8" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSP002M8"><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/dsp00213.gif?Expires=2147483647&Signature=zVtegN2gi4h~8WrQlkHnj6ab97Lml88NvIJn5N-CqEaRes9aaoW3yFqNduLYk2WEWfaCH8QADkcBmhyumTfHNl8K9fk-2InA6pVvU608bd1mJUQh~SlC4wRi3wVnx0kDWrofftWkqFmkxYiEGmOekS6r8UGtDtqHHSngdZzyl27UFN19beDC1YUttvIRQFxuN~BCLlKsUS7-pE3TZ4wAdNbj3RkvpjnFCxVaUQOum9P45pekuu9b3jsDesGNDk4QBYgLEmfkstOpwfAWmRUZTtUfxjeujklFUlzqrdckUSlGShgPjjFoqrNsvd7ablR7wbSOd~Trl-tzkVHy9gaZLw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsp00213.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00213.gif?Expires=2147483647&Signature=k5jCLNx7cue8rCGeGKpPRnwmxlyO-R063fer9VT3gePvfIcLmB1zGCbrMtJgAJE8xTE~rBkZxmMXqEpC6-pXyDX~J7xRNhRLBKDMlA3b7XnfOE022ysLTRbrJhu2rjTcIT~9LVMuQcminf8LDAv-Q5eNbdyIND7haP8Z836SxQ8B-s0NF1qJshUv-VNpY7ZWahJ1ndY88YEoHL7DwQdIWBRNZoNbY~J6BW0-esuJVrKbh8ZmC0zJXC7Idtph9RE2-Wp5~rMhLrFdIgVR5MAQ7F~VetoMfmhjAZNS635-hOU~FoFgUuXUtx5yE5GYzqQDx2wlkpZ4F5WddISR8xcqAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsp00213.gif" /></a></div></div><div class="label">8</div></div> where <em>i</em> shows the row number, <em>i</em> = 1, 2, 3, 4, and <em>j</em> the column index of a matrix <em>q</em>, <em>j</em> = 1, 2, … , <em>L</em>.</div><div class="block-child-p">To find local optimal alignment of sequence <em>s</em>(<em>j</em>) against profiles <em>q</em>(<em>i</em>,<em>j</em>), we applied the method of dynamic programming.<sup><a reveal-id="DSP002C30" data-open="DSP002C30" class="link link-ref link-reveal xref-bibr">30</a></sup> We iteratively calculated similarity function <em>F</em> according to the following equation:<sup><a reveal-id="DSP002C31" data-open="DSP002C31" class="link link-ref link-reveal xref-bibr">31</a></sup><a id="DSP002M9" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSP002M9"><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/dsp00214.gif?Expires=2147483647&Signature=BZFlQp3Ndxuft6mN4Rlbf9OpmklnTRZ7CCr8pSp-EUa3hjg9Gyf6Kx1F-Rf2DAtCUIAfFF36btNiuXsUsrsqwqww82o0mtgk0ft8brBuLkM6mxnh6Xbi8fFTvDjNSRZjj3f6fohXU7nZPjJ4o5KkIAVWp6NFcn7s6Gpkv-RsAPHmFhCGK9oSsSTPa8AhOGXBcDKHfJFp70Na~oLAUQ2a634JGtfmd1XJRErw8DeomtoFJfIADmXpoXXBCJdapYyUTR9vrCQm~3zv09LwEZXoJTcw-1X9tHkQkEu6Xan4BGXabzd9tLoWQniHpKL7hbHPSJn~thhAgboabwTe-Wtlbw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsp00214.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00214.gif?Expires=2147483647&Signature=t5kpvJslqkmYTpcXTACSdteBWX3X4EKhPWiipDJPhw~XR7Zb4yrgRdNJACkqfum-l4uyK9ExsUlgtuYBZcrdzhLjyaTogzbwyKN2B~LE-NQEiKkP155Y2SDGkRJwCiRsPsEuMlaF8v~-YZnk1ObaEst-KTIuQLE~M2r8oTFOdskHNy~53RPdNMPd0Smm0iaHZtsANmEaRaMEmiZyvYjQ7xFbqMBjyQwCTQ77BOPMpFG12HNgfkn~Yi34wmRvoCELfsmNTpEImv9qh1TqsnpP6NbujzYuBoqF5Yk80A0N0xdZ77Qy2nkFWbX2iSbSzBICTr37gBxPYnvZHSRJFN-3jQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsp00214.gif" /></a></div></div><div class="label">9</div></div></div><div class="block-child-p">Here, the index <em>i</em> stands for nucleotide in sequence <em>s</em>(<em>i</em>) and index <em>j</em> for the column number in profile matrix <em>q</em>. Initial values for similarity function <em>F</em> are specified as: <a id="DSP002M10" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSP002M10"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsp00215.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00215.gif?Expires=2147483647&amp;Signature=oLVfm5X9TJkuM6BWeV~rJK~QrVmz-aaIKjWzBoRxSNJ7Z64bySaSfrM4sTfMh3rATBy2Tnn5ckG3UXdjsE3VTNPdvIBc0TtCREtlQqg-o3YuKcGx20Da3pp5sBvsLhI6Ji~x-S9c8cHMNuuEcPWIpUgFw2iZo54Q8w~yjOAFlpGoyLsIj5E322-rsoxTnDGPUOF4-XFr6o6I3lrgOUDIwTIaZoLXeUYF02~RSjXBJqsSC-Eare0qsSW1q4Gb2RrV5dZY5Gg~e7~QK9tUhxZt4Hi5YbFkDietMaL-Mqpmrwy7qp0kOirAr-FYrYggdbT-UrYkqTfkdc01NaYysV-1xg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsp00215.gif" /></a></div></div><div class="label">10</div></div><a id="DSP002M11" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSP002M11"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsp00216.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00216.gif?Expires=2147483647&amp;Signature=Y6Ml2~xhq9oNwcHddbb9e5dkVQM4mVnrWsdR3nouGJpaIWO3ZZ7n9E3UdhBu3Fj7Q-G5-6E~UQSHlh6v1EGKvoOMAg3lVPptl0-~r1L4TGmIkL-0vLOR7lhVNRVUKlLS1pUkpF4QoJ9wsssM3qUVGBppG4WoWbcww0gqhH4RKJgFi-PPqMAQ3dML7nUtpP7idPgNFlE9DmD6XRZokUYZDOVALU4gRT0hfIF0rNwV~3pOn8YuybkfDxTaem04MrY5RqazOOFFDnCyNB~Ot-L1eWD4o8347DZvRiz65UUjVPkQL1YjoPZlu6Ms-mgKyewt7WurutHcAvRaKYyUhKR7KA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsp00216.gif" /></a></div></div><div class="label">11</div></div><a id="DSP002M12" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSP002M12"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsp00217.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00217.gif?Expires=2147483647&amp;Signature=dSP17aEbHPv0DXJuP4NNzJJTMyb1kgHFrvHPNVqERKZLWH7rGp93JE8fhRzpis4HD9Y3HJVesAX-USGu9qWAtduRnbqLiVmfV1c1PNap-wdvAF~d5K-Z7Yrzkx6l3ASLMzjnGEKoJUT8f3X71PjRE6iwktDvK5kYALl6IkQ~GESYaxAVPSNKonx~FkyM6uG4tafWqOZu-30j8QfhxIwNRoohxma444v5eLCq~B0QvMc9o7lqO5TGKKxCDqUE22Js9Otfi3PFInbuLReI5opHdRqXkT23H40g1ePQIB0D9OHnMuiSQAnU5obljpX4t-r593h1FME7ViWQwvyEXAiymg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsp00217.gif" /></a></div></div><div class="label">12</div></div> where <em>d</em> is the maximal number of insertions or deletions allowed. For calculations on triplet matrix, <em>d</em> equals 2. During building the local alignment, we determine the maximal value of the similarity function <em>F</em> coordinates (<em>i</em><sub><em>m</em></sub>,<em>j</em><sub><em>m</em></sub>) corresponding to this maximal value. Then we determine the path from points (<em>i</em><sub><em>m</em></sub>,<em>j</em><sub><em>m</em></sub>) to (<em>i</em><sub>0</sub>,<em>j</em><sub>0</sub>), where the value of the similarity function becomes zero for the first time. According to the path made, we built alignment between sequence <em>S</em> and profile matrix <em>q</em>. For building alignment, nucleotides of sequence <em>S</em> and numbers <em>l</em> = mod3(<em>j</em>) indicating columns of matrix <em>w</em> [Equation (<a href="#DSP002M5" class="sectionLink xref-sec">5</a>) were used].</div><div class="block-child-p">We used Equation (<a href="#DSP002M9" class="sectionLink xref-sec">9</a>) for carrying out the global alignment but without using zero in the right part of the equation. All other parameters were the same as in the case of the local alignment. Initial values for similarity function <em>F</em> for global alignment are specified as:<sup><a reveal-id="DSP002C31" data-open="DSP002C31" class="link link-ref link-reveal xref-bibr">31</a></sup><a id="DSP002M13" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSP002M13"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsp00218.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00218.gif?Expires=2147483647&amp;Signature=QrJxEQylBQoCcLtZPntvFLPk9RNdQM81eL9i4NgtU~C3LJHH5wQj1Bm7u~ytorsrWsM~cBQWVrUEaCwyMuKDYN8MXP~VtQmZSo8j~STwOl0Qj9RPFp-3ey8BvcfTgDJK9cjRDoSfqLZg4GPNDscEJNQ6tTJusiVsf-wDAsIJNiHdwuOv9RsSZjoaP2dAHatuN1VGDDQa0wXT76DU7tH46VutNxTO-PZ5QQqhsZjk25mPgd5yA~LYKM1ZZMoBdm5X5-yuZ6nB217zh0hop2amzhchxVIqUi~C5pe3lLqThCyTt7bv85Cgp7yEfT693qfeNKsRGvY7GAUzzxUszwJbwQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsp00218.gif" /></a></div></div><div class="label">13</div></div><a id="DSP002M14" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSP002M14"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsp00219.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00219.gif?Expires=2147483647&amp;Signature=WUJU1-VUn6wMcO17bgE72g8FIycuOfDZC9P5DUAoN7fegm0OAMRaMfrDPJkbeQt0LN7xylZWooyEb46oDzmtdqCFVFigVzBDk~XqLSijQiEE7jsI7OjRueH5m3aqHgBgYLtEmYojdcxdcVx4sULIFZ~56LEItHAk9j94UrM12LH92vReuGURXdZYGUKlyxTzcm9o8IImtnd762ilPzSPpBU15b5b5AlEMl90eQcnWv1FbyfGxgJQ1jOEEECO5pGubCRwc2x2-GkQXz1Y55I8yF~vb3jC6iml5dy0lJRuy9ARC8towt2Cf5ylPXS-dJfZj-OOGO7voVXiI0sgPsn25w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsp00219.gif" /></a></div></div><div class="label">14</div></div><a id="DSP002M15" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSP002M15"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsp00220.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00220.gif?Expires=2147483647&amp;Signature=D9T00rsFrtYIrseU5btF9avJiCMpynIAivhMLpj3BSwJkMsrMCjOfQrsFvr5COskRwJi5WwYWxTxzlIBsT-eCBDxWhClfFi5JWLpvxiu61B-zo62BfdLEMHG~q3NB2l4YeNTj8IyoPM89uMRzn7Eu6cgFvbWOXppjw08nwSp-AiJGKTOY7zxwUF2P~TsMspdX7Hf9rZJXtBFi2WyFhlN9XfWdjYGIBZwVdkyFCJ9xaGi3aH32mFZOzDesDCcNMjRCyfN8n85pmG-V~AFFTEjgvXgU3TumbBC6bcgDZ4eBh~PxzLIsgMqXEWdvmovsBALdtafyYTsG0M2n2DmYuiYnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsp00220.gif" /></a></div></div><div class="label">15</div></div></div>                            <h3 scrollto-destination=7931416 id="7931416" class="section-title" data-legacy-id=s2d>2.4. Choosing values for v<sub>do</sub> and v<sub>dc</sub></h3>
<div class="block-child-p">We chose values <em>ν</em><sub><em>do</em></sub> to prohibit the optimal path in local and global alignments from moving round the nucleotide having minimal weight in matrix <em>w</em>(<em>i</em>,<em>j</em>) because of two sequential insertions. On this basis, we chose <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00225.gif?Expires=2147483647&amp;Signature=2TXsvC-GAHEthkS~K8tqxTzEjegystTaD-nY-jNUMx72dSKdXUGqnDTF0GQR~meZgC--c2cvPh3Nk5Z5WSzXRk8E7e5pAWz0v9~4lyi7KAVUzVX7RRUo42gh3UPG2kj1JeG5URBmGtWM-H4xlR1zbXm5sAVRFV9JXZi6J1KRFycLSDwhQqCJBNeDfuK5gn-xdoQ1Im~1C4aSX9ctM-p6FNb7bFXLtPB9K9ZlpdIJE4uJX~NUYt2i34Ikij8SGG9K9PwjKKigAqDVVcj8dR~RzZ1CPqR9-~stJhnvEDeDdzQd5~AH6l-CbJiYpWkubgRXolWD-GFNNnHDEBgcnrWEGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp00225.gif" /></span> At the same time, it was important to make the alignment able to find region <em>T</em> revealed by the method of information decomposition (Section 2.1) without any insertions and deletions. In order to choose the corresponding coefficient <em>ν</em><sub><em>do</em></sub> for each gene from KEGG databank, we generated a set of random sequences <em>QV</em>, where regions of sequence <em>S</em> in the range from 1 to <em>t</em><sub>1</sub> and from <em>t</em><sub>2</sub> to <em>L</em> were shuffled in a random manner. The region of sequence <em>S</em> in the range from <em>t</em><sub>1</sub> to <em>t</em><sub>2</sub> was constructed by TPM <em>M</em>. Then matrix <em>M</em> was transformed to <em>M</em>′ as (see also Section 2.1): <a id="DSP002M16" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSP002M16"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsp00221.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00221.gif?Expires=2147483647&amp;Signature=NebJqGmWs9252t2Y9rXvuPHY1EXych3lQ1lsvMB~Qlzamwt23yjdo5tc373Y1t2OkoCwI4bvPyytp3xrh6RfzVKx8wi4uaqoIvNr1bqrTouX1KhZsMc-wKbpjb8dJ9t80xi5UdG5xJyu6wfvWdB2yOqgQXxGyhkF2H9C00ERem65NmzMwI1GSsHxakJCXXoYjXXaW-YEeB1G1JfKAvMfT11ZdYVH2O0YN7TwgrxsHGjMRTUw7P7-gck4HkzMMYknC6q~1jtU7ysd4y5HsTp0XnneaUARPDXd1FWRGkJG7JGmlIB3-w2DtyHNsk6~Wm~7954QQItuZtSd3sO0ZOVzyA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsp00221.gif" /></a></div></div><div class="label">16</div></div></div><p>Then we generated in a random manner three nucleotide sequences <em>S</em>1(<em>k</em>), <em>k</em> = 1, 2, … , <em>y</em>(1), <em>S</em>2(<em>k</em>), <em>k</em> = 1, 2, … , <em>y</em>(2) and <em>S</em>3(<em>k</em>), <em>k</em> = 1, 2, … , <em>y</em>(3). Nucleotide probabilities in sequence <em>S</em>1 are equal to probabilities <em>m</em>′(<em>i</em>,1), nucleotide probabilities in sequence <em>S</em>2 are equal to probabilities <em>m</em>′(<em>i</em>,2) and nucleotide probabilities in sequence <em>S</em>3 are equal to probabilities <em>m</em>′(<em>i</em>,3). Index <em>i</em> varies from 1 to 4. Thereafter, nucleotides of sequence <em>S</em>1 took positions <em>k</em> = <em>t</em><sub>1</sub>, <em>t</em><sub>1</sub> + 3, … , <em>t</em><sub>2</sub> − 2; nucleotides of sequence <em>S</em>2 took positions <em>k</em> = <em>t</em><sub>1</sub> + 1, <em>t</em><sub>1</sub> + 4, … , <em>t</em><sub>2</sub> − 1 and nucleotides of sequence <em>S</em>3 took positions <em>k</em> = <em>t</em><sub>1</sub> + 2, <em>t</em><sub>1</sub> + 5, … , <em>t</em><sub>2</sub>. This algorithm was used for the generation of 1000 random sequences contained in <em>QV</em>.</p><p>Selection procedure of the <em>ν</em><sub><em>do</em></sub> value for each gene from the KEGG databank from which region <em>T</em> was extracted is described below. First, we chose the value for <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00226.gif?Expires=2147483647&amp;Signature=ae1ZEw4frhbYg5Thkhoj9WgkcAHCs8ySuky9ylo~kPqqG-hIGOOpGlKRB-gLF9hnfMsRA-m-sQ~fzB9UWfhFgqiAi9XCU~t2ewpvKN6lhUPBwWNMdTED0eBqP35w36XDOSluFpYgH-w9T0MwOTsQeMUjJJmjhTXI4ZcjY9awzPVRl84LZhGsbjEzv2oTDehCDJP7E7ZhTUjHYWZr8n99ghfN-Em8bVrhL0ZJnlz~F1yXcYDRh41BOugH0ai7InBe~2h~QxhK4ycUkrxio5P4oHfkGy0Eop5r5ASNNe8KNF~pjiv2~JcBuN0GGF4UnwcRHAUBXSS1tiqvcsGm2Ox8DQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp00226.gif" /></span> and <em>ν</em><sub><em>dc</em></sub> = 0.25 <em>ν</em><sub><em>do</em></sub>. Then we made alignments for the set of random sequences <em>QV</em> and determined the number of sequences having insertions or deletions within the region from <em>t</em><sub>1</sub> to <em>t</em><sub>2</sub>. If the fraction of sequences in set <em>QV</em> having at least one insertion or deletion within region from <em>t</em><sub>1</sub> to <em>t</em><sub>2</sub> exceeded 1%, then we increased the value of <em>ν</em><sub><em>do</em></sub> by 0.1<span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00227.gif?Expires=2147483647&amp;Signature=uLM3hHpI5cYVi4UwfE0vIivmZaRlcuRyNA5dhTTAtEKyN1ILllXUexTUYpzo6VvxmzUbVg-wBWg6rjx1Mnrt8CeQCrjtmKhzjN7bsTA4KrUwUUmR3qqu0dfevIt3SQu0ajFep1s40jGZVYpBHv7sTSztHWckaxltqtQWKSFRsuyF--5JiZljVv43eqodajCDNhIRlC1MK54ItVcI8sTLW~0c-XpF~LSNz1XpzqqHwB-Hem3WxCe7bUb8KugVoK~GoXBo4AwC7J8IWMXeNrrlxI9lfpUFUttSKVghaiiu9YcPrYMsK6Iksr6e5IhxgkBZ8wVZib1aKxhMqfcHEOtwRA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp00227.gif" /></span> and calculated <em>ν</em><sub><em>dc</em></sub> = 0.25 <em>ν</em><sub><em>do</em></sub> again. Alignments of sequences from set <em>QV</em> were also built again using these new parameters. If the fraction of sequences with insertions or deletions was under 1%, then the process was stopped and the obtained value <em>ν</em><sub><em>do</em></sub> was used for finding the extended region <em>R</em> by local alignment (Sections 2.2 and 2.3). If the fraction of sequences with insertions or deletions was over 1%, then we increased <em>ν</em><sub><em>do</em></sub> and <em>ν</em><sub><em>dc</em></sub> again as shown earlier and alignments of sequences from set <em>QV</em> were built again.</p>                            <h2 scrollto-destination=7931420 id="7931420" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=7931421 id="7931421" class="section-title" data-legacy-id=s3a>3.1. Finding extensions of the regions with TP for genes accumulated in KEGG databank</h3>
<p>We analysed 578 868 genes accumulated in 29th release of the KEGG databank (<a class="link link-uri" href="http://www.genome.ad.jp/kegg/">http://www.genome.ad.jp/kegg/</a>). We found 472 288 regions having continuous TP in 457 333 genes. These data indicate that 79% of genes have regions with TP. These results conform to earlier works on TP detection by either using informational methods or other techniques.<sup><a reveal-id="DSP002C4  DSP002C13" data-open="DSP002C4  DSP002C13" class="link link-ref link-reveal xref-bibr">4–13</a>,<a reveal-id="DSP002C32" data-open="DSP002C32" class="link link-ref link-reveal xref-bibr">32</a></sup> From these 472 288 regions containing continuous TP, we selected only those the lengths of which were significantly less than that of a gene. This means that the distance from the left and right edges of the TP region to the start and end of the gene was more than 30 bp. This criterion was satisfied for 326 933 TP regions. Then we aligned nucleotide sequences of corresponding genes against TPM (see Sections 2.2 and 2.3), which were revealed in the gene by the method of information decomposition (Section 2.1).</p><p>We revealed totally 824 genes in which TP regions were extended by using alignment against TPM. General information describing all these sequences can be found in the Section <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data</a>. Details including periodicity alignment and found protein similarities can be found in online databank installed at <a class="link link-uri" href="http://victoria.biengi.ac.ru/pertails/">http://victoria.biengi.ac.ru/pertails/</a>.</p>                            <h3 scrollto-destination=7931424 id="7931424" class="section-title" data-legacy-id=s3b>3.2. Finding protein similarities for the products obtained using active and ancient gene RFs</h3>
<p>Let us consider those genes in which the region of continuous TP was extended by taking into account nucleotide insertions and deletions (see Section 2.2). Further, we will discuss nucleotide sequences with coordinates from <em>r</em><sub>1</sub> to <em>t</em><sub>1</sub> and from <em>t</em><sub>2</sub> to <em>r</em><sub>2</sub> (Fig. <a reveal-id="DSP002F2" data-open="DSP002F2" class="link link-reveal link-table xref-fig">2</a>). Let us call these sequences <em>T</em>1 and <em>T</em>2 (the region of continuous TP was earlier referred to as <em>T</em>). In sequences <em>T</em>1 and <em>T</em>2, we found TP with insertions and deletions that were very similar to continuous TP <em>T</em>. This is the reason why we found statistically significant alignment from <em>r</em><sub>1</sub> to <em>r</em><sub>2</sub> in sequence <em>S</em> against the weight matrix constructed on the basis of TPM <em>M</em>. We suppose that TP found in sequences <em>T</em>1 and <em>T</em>2 is a trace of some ancient RF that existed in these nucleotide sequences earlier. First column of the matrix <em>M</em> corresponds to the first codon base in sequence <em>S</em>, whereas due to insertions and deletions of nucleotides, in subsequences T1 and T2 matrix M corresponds to alternative ancient RF which may not match the actual RF there. Let us refer to RF specified by matrix <em>M</em> as ‘ancient RF’. We suppose that it is possible to reveal similarity between amino acid sequences obtained by ancient RF from nucleotide sequences <em>T</em>1 and <em>T</em>2 and amino acid sequences accumulated in modern databanks like UniProtKB/Swiss-Prot. Such an assumption is based on the idea that if a gene responsible for the same genetic function existed in several genomes, then insertion or deletion of nucleotides in this gene within one genome does not ultimately lead to analogous changes in another genome. It is important that these sequences should be now known and should not have accumulated many evolutional alterations. This will allow us to see their similarity. We conducted such an investigation within the scope of the present work. Sequences of regions <em>T</em>1 and <em>T</em>2 were translated to amino acid sequences in accordance with the RF existing in the gene and in accordance with ancient RF's revealed by TPM <em>M</em>. For all amino acid sequences obtained in such a way, we searched for their similarities with sequences accumulated in UniProtKB/Swiss-Prot databank<sup><a reveal-id="DSP002C33" data-open="DSP002C33" class="link link-ref link-reveal xref-bibr">33</a></sup> by using BLAST program.<sup><a reveal-id="DSP002C34" data-open="DSP002C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Totally we found 824 <em>T</em> regions in genes that were extended via joining regions <em>T</em>1 and/or <em>T</em>2. Sixty-four of TP regions <em>T</em> had protein similarities with regions <em>T</em>1 and/or <em>T</em>2 only by ancient the RF constructed on the basis of matrix <em>M</em> (see above). At the same time, for 176 <em>T</em> regions, protein similarities in regions <em>T</em>1 and/or <em>T</em>2 were found only for the active RF of a gene, and in three cases, for both RFs. Five hundred and eighty-one <em>T</em> regions have no protein similarities in regions <em>T</em>1 and/or <em>T</em>2 for amino acid sequences constructed either by the existing gene's RF or by the ancient RF constructed on the basis of matrix <em>M</em>.</p><p>Let us consider an example of continuous TP extension in a putative dehydratase gene (locus ‘SMa0056’ in KEGG databank). This gene has a length of 1134 bp and contains the region of continuous TP from 16th to 840th nucleotide. TPM for region <em>T</em> and weight matrix <em>w</em>(<em>i</em>,<em>j</em>) is of the form that is shown in Fig. <a reveal-id="DSP002F3" data-open="DSP002F3" class="link link-reveal link-table xref-fig">3</a>. We extended the found TP region <em>T</em> by adding region <em>T</em>2 that leads to the appearance of TP in this gene from the 840th to 1134th nucleotides. Statistical significance <em>Z</em><sub><em>R</em></sub> of the found alignment <em>R</em> was equal to 17.7. The value Δ<em>F</em><sub>0</sub> equalled 10.25, whereas the value Δ<em>F</em><sub>1</sub> equalled 13.85. This provides the probability of incidental addition of region <em>T</em>2 to region <em>T</em> at a level &lt;10<sup>−5</sup>.</p>                        <a id="7931427" scrollto-destination="7931427"></a>
<div id="DSP002F3" data-id="DSP002F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00203.jpeg?Expires=2147483647&amp;Signature=AX61JCePPN8mk3QHzMXRbnQgRTEGxtaUBDJuUMXBnF5EthEcwaoPiPwYU2W-9oHD628ytTr5CrSC7RnJJvvI4nQj37Do8hrJCysgW6bQH28-GysH3iQX0H~uGpoIIwsSe6oENzG8F9kfr5S0Z-1VAjYL55~yQS38Wyf8de9foRDwdEBCB9aObiuJThdT-EsqG1hhdpXvqtoZLzUNTvswud2rzuLZHRuppRUT47rJ-rStI3tXvAGWAv1Bair4GCCXmmyjRDZevlnkVxb5ZfIjefCGI8wa94Us6mwrHYuURii6~j2h79uATOwLfu3EeZ3Rz6Je8Qz9hlUuRtmDCiH0xg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="TPM M and position-specific weight matrix w(i,j) built for T region from 16th to 840th nucleotides of the locus ‘SMa0056’ (KEGG databank). For the calculation of weight matrix, we used C = − 0.3, " path-from-xml="dsp00203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7931427/dsp00203.jpeg" path-from-xml="dsp00203.jpeg" target="_blank">View large</a><a section="7931427" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/dsp00203.jpeg?Expires=2147483647&amp;Signature=UxyEU0AGmKuCyv6wx-liWQs3IJQ434gFZDgC1rj2vpiFhOEDN~Y6GWBxqTsrM9EpYNqNsMCpRK5W8XaZkkL6YsdHCocCGYHCKj1tyRySrAQ8x1sHoHrvbayMlOtwvNcZrk8IaxL3w9vsAmm8ITB5TqZXBJ5WhGF8W4k21xzQ0AycNCkGwsWNcL~i~KT81b7qnBzj~AcGwZewWDXrfxRDjlTDliPDxoPsSfzWq9oV1jsMdY4FBDfrdzkJqe~~KOnVYQSEUaKV2jVqc2hzWjoa-JNDZJ7LLmLMGOAD8rXQvoiLvNf90iD0beSzcKwJCQcqTZX1vRZVGUSypiNZZ7s81g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=7931427&amp;ar=510716&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsp00203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>TPM <em>M</em> and position-specific weight matrix <em>w</em>(<em>i</em>,<em>j</em>) built for <em>T</em> region from 16th to 840th nucleotides of the locus ‘<span class="underline">SMa0056</span>’ (KEGG databank). For the calculation of weight matrix, we used <em>C</em> = − 0.3, <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00228.gif?Expires=2147483647&amp;Signature=VeiF9opv1OYMP8LXvfWULYujcXyxSBp9Wj1HOCMb2rpjxN2oxPuZBhx9IM-iqa3rqQFXG0woEIPpOXl7WPI9SminbAvKUxpZy4Jwpa8Avl2JiY8SftMhb-RrKJkQns2CXLjcboDMy895~uHN4Qy-tKJEePBobHMmF8qQPPdJPitqXeG8CokZ53VR9Tgru-BEmbMYJ1qLFqRc-KYrA4X49tgRHdUCdeGCL9wjoKrUf5klrmmrwnCWzzUIT4OtcBr73KMtya4KoDTBGz-sgfA0PLCPUMCXmIDqYIQa1Xd5fLcgoiodhF3eXzZj9ah9FVNeZAU7dZ0trSfE-t8hm-JBgQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp00228.gif" /></span></p></div></div><div content-id="DSP002F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00203.jpeg?Expires=2147483647&amp;Signature=AX61JCePPN8mk3QHzMXRbnQgRTEGxtaUBDJuUMXBnF5EthEcwaoPiPwYU2W-9oHD628ytTr5CrSC7RnJJvvI4nQj37Do8hrJCysgW6bQH28-GysH3iQX0H~uGpoIIwsSe6oENzG8F9kfr5S0Z-1VAjYL55~yQS38Wyf8de9foRDwdEBCB9aObiuJThdT-EsqG1hhdpXvqtoZLzUNTvswud2rzuLZHRuppRUT47rJ-rStI3tXvAGWAv1Bair4GCCXmmyjRDZevlnkVxb5ZfIjefCGI8wa94Us6mwrHYuURii6~j2h79uATOwLfu3EeZ3Rz6Je8Qz9hlUuRtmDCiH0xg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="TPM M and position-specific weight matrix w(i,j) built for T region from 16th to 840th nucleotides of the locus ‘SMa0056’ (KEGG databank). For the calculation of weight matrix, we used C = − 0.3, " path-from-xml="dsp00203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7931427/dsp00203.jpeg" path-from-xml="dsp00203.jpeg" target="_blank">View large</a><a section="7931427" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/dsp00203.jpeg?Expires=2147483647&amp;Signature=UxyEU0AGmKuCyv6wx-liWQs3IJQ434gFZDgC1rj2vpiFhOEDN~Y6GWBxqTsrM9EpYNqNsMCpRK5W8XaZkkL6YsdHCocCGYHCKj1tyRySrAQ8x1sHoHrvbayMlOtwvNcZrk8IaxL3w9vsAmm8ITB5TqZXBJ5WhGF8W4k21xzQ0AycNCkGwsWNcL~i~KT81b7qnBzj~AcGwZewWDXrfxRDjlTDliPDxoPsSfzWq9oV1jsMdY4FBDfrdzkJqe~~KOnVYQSEUaKV2jVqc2hzWjoa-JNDZJ7LLmLMGOAD8rXQvoiLvNf90iD0beSzcKwJCQcqTZX1vRZVGUSypiNZZ7s81g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=7931427&amp;ar=510716&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsp00203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>TPM <em>M</em> and position-specific weight matrix <em>w</em>(<em>i</em>,<em>j</em>) built for <em>T</em> region from 16th to 840th nucleotides of the locus ‘<span class="underline">SMa0056</span>’ (KEGG databank). For the calculation of weight matrix, we used <em>C</em> = − 0.3, <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00228.gif?Expires=2147483647&amp;Signature=VeiF9opv1OYMP8LXvfWULYujcXyxSBp9Wj1HOCMb2rpjxN2oxPuZBhx9IM-iqa3rqQFXG0woEIPpOXl7WPI9SminbAvKUxpZy4Jwpa8Avl2JiY8SftMhb-RrKJkQns2CXLjcboDMy895~uHN4Qy-tKJEePBobHMmF8qQPPdJPitqXeG8CokZ53VR9Tgru-BEmbMYJ1qLFqRc-KYrA4X49tgRHdUCdeGCL9wjoKrUf5klrmmrwnCWzzUIT4OtcBr73KMtya4KoDTBGz-sgfA0PLCPUMCXmIDqYIQa1Xd5fLcgoiodhF3eXzZj9ah9FVNeZAU7dZ0trSfE-t8hm-JBgQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsp00228.gif" /></span></p></div></div><p>During alignment of the sequence relative to found continuous TP, we revealed the insertion of the nucleotide at position 841 (nucleotide <em>t</em>) of the locus (Fig. <a reveal-id="DSP002F4" data-open="DSP002F4" class="link link-reveal link-table xref-fig">4</a>). This means that after the 840th nucleotide (towards first nucleotide), a shift occurs between TP and RF. Thus, the first codon base in sequence <em>T</em>2 corresponds to the third column of matrix <em>M</em>, whereas in sequence <em>T</em>, the first codon base <em>T</em>2 corresponds to the first column of matrix <em>M</em> (Fig. <a reveal-id="DSP002F4" data-open="DSP002F4" class="link link-reveal link-table xref-fig">4</a>). We suppose that in the past this gene existed without insertion of the nucleotide <em>t</em> in position 841 (towards first nucleotide of a gene) and mutations could not blur the TP existing in this region (Fig. <a reveal-id="DSP002F5" data-open="DSP002F5" class="link link-reveal link-table xref-fig">5</a>). To check this hypothesis, we recoded <em>T</em>2 region to amino acid sequence by the actual gene's RF and by the ancient RF (Fig. <a reveal-id="DSP002F4" data-open="DSP002F4" class="link link-reveal link-table xref-fig">4</a>). Then we searched for similarities with these two amino acid sequences in UniProtKB/Swiss-Prot databank. Consequently, we found five cases of similarities for hypothetic sequence only, i.e. for the amino acid sequence obtained by RF specified by TPM <em>M</em> (Fig. <a reveal-id="DSP002F4" data-open="DSP002F4" class="link link-reveal link-table xref-fig">4</a>). At the same time, amino acid sequence constructed by RF specified in KEGG had no similarities at all (Fig. <a reveal-id="DSP002F4" data-open="DSP002F4" class="link link-reveal link-table xref-fig">4</a>). Similarities for hypothetical amino acid sequence were found to dehydratases. Match ratio was &gt;30% (<em>e</em>-value in range from 10<sup>−12</sup> to 10<sup>−7</sup>), i.e. found similarity is evolutionally distant and statistically significant. All found similarities can be accessed at <a class="link link-uri" href="http://victoria.biengi.ac.ru/pertails/perinfo.php?perid=270869">http://victoria.biengi.ac.ru/pertails/perinfo.php?perid=270869</a>.</p>                        <a id="7931429" scrollto-destination="7931429"></a>
<div id="DSP002F4" data-id="DSP002F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00204.jpeg?Expires=2147483647&Signature=hhkHe2xjHDO47dnbNNIH-WmDd0LN5A3p-ZztdNcYa91hs5D9Emh3i3BYT-g4nf8-GyO8P~fV8n~prxlKr9aYPImpW3cdlcQ2Nr~0wFKj3e4epHZHADFO3t0Z9saU1N2z6YVMTrZdj8ADTb3O~SOzr5d5zSjv8d16xFI~iGoSkaxHr7XtsvW5mhKDjSx9JSL0lLY8zBDxsM3NxvJns2RdsadFKYvi4bPtwJTS1xYMNBUfOuCmBApVR2BTYiyjQ8MCJte01qP~-Yg09diLXfW4l2Q1IpGy5evXfxjsmJ3qaKn4Y8uiR4iG3en74rWamiVhp8mqbGKGFGACq3CXvoNmXA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequence of ‘SMa0056’ gene from 818th to 877th nucleotide (1) aligned against weight matrix w(i,j). Insertion of 841st nucleotide (nucleotide t) is shown by asterisk. Further under gene, the consensus sequence for TP (2) is shown, which is constructed by matrix М and also by RF existing in gene (3) and RF specified by TPM М named as ancient RF (4). Below translated fragments of the nucleotide sequences are shown. Translation is made according to RF existing in gene (5) and to ancient RF (6). Regions having RF shift are marked in bold." path-from-xml="dsp00204.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7931429/dsp00204.jpeg" path-from-xml="dsp00204.jpeg" target="_blank">View large</a><a section="7931429" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/dsp00204.jpeg?Expires=2147483647&Signature=wBR5-mVZKydLbnOwBLIKXsxF6jk1J70USsLun0Eab-Sysl0UQCnKDtWL7Ex5iyuJ9rOA-rx-CLDVObRK0C~KxCLzYzwGiH226H~Rg-1YT~6IWwL-G7HB2gW1z4gJDkTJo2ZRd3CWNNwkfJJMfG8b-BixfUJvG6bWzNTFJHYupsM0skxu0xM05iwyl4Dg~uU0tb-Xgp8O1UWW~m8XnAdPOYXjEwSWLII3wEXvXM2UtIElXOVt8YK3ljYffWfuHH~k9FWGwKCkYog5Zokat8jvdgO6vLTz~LjYz5~pcnYuFrsMjbjKrnuEgxfJPTd8uc8EBJ5karqWS9GbnteiTsDwqA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7931429&ar=510716&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00204.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sequence of ‘SMa0056’ gene from 818th to 877th nucleotide (1) aligned against weight matrix <em>w</em>(<em>i</em>,<em>j</em>). Insertion of 841st nucleotide (nucleotide <em>t</em>) is shown by asterisk. Further under gene, the consensus sequence for TP (2) is shown, which is constructed by matrix <em>М</em> and also by RF existing in gene (3) and RF specified by TPM <em>М</em> named as ancient RF (4). Below translated fragments of the nucleotide sequences are shown. Translation is made according to RF existing in gene (5) and to ancient RF (6). Regions having RF shift are marked in bold.</p></div></div><div content-id="DSP002F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00204.jpeg?Expires=2147483647&Signature=hhkHe2xjHDO47dnbNNIH-WmDd0LN5A3p-ZztdNcYa91hs5D9Emh3i3BYT-g4nf8-GyO8P~fV8n~prxlKr9aYPImpW3cdlcQ2Nr~0wFKj3e4epHZHADFO3t0Z9saU1N2z6YVMTrZdj8ADTb3O~SOzr5d5zSjv8d16xFI~iGoSkaxHr7XtsvW5mhKDjSx9JSL0lLY8zBDxsM3NxvJns2RdsadFKYvi4bPtwJTS1xYMNBUfOuCmBApVR2BTYiyjQ8MCJte01qP~-Yg09diLXfW4l2Q1IpGy5evXfxjsmJ3qaKn4Y8uiR4iG3en74rWamiVhp8mqbGKGFGACq3CXvoNmXA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequence of ‘SMa0056’ gene from 818th to 877th nucleotide (1) aligned against weight matrix w(i,j). Insertion of 841st nucleotide (nucleotide t) is shown by asterisk. Further under gene, the consensus sequence for TP (2) is shown, which is constructed by matrix М and also by RF existing in gene (3) and RF specified by TPM М named as ancient RF (4). Below translated fragments of the nucleotide sequences are shown. Translation is made according to RF existing in gene (5) and to ancient RF (6). Regions having RF shift are marked in bold." path-from-xml="dsp00204.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7931429/dsp00204.jpeg" path-from-xml="dsp00204.jpeg" target="_blank">View large</a><a section="7931429" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/dsp00204.jpeg?Expires=2147483647&Signature=wBR5-mVZKydLbnOwBLIKXsxF6jk1J70USsLun0Eab-Sysl0UQCnKDtWL7Ex5iyuJ9rOA-rx-CLDVObRK0C~KxCLzYzwGiH226H~Rg-1YT~6IWwL-G7HB2gW1z4gJDkTJo2ZRd3CWNNwkfJJMfG8b-BixfUJvG6bWzNTFJHYupsM0skxu0xM05iwyl4Dg~uU0tb-Xgp8O1UWW~m8XnAdPOYXjEwSWLII3wEXvXM2UtIElXOVt8YK3ljYffWfuHH~k9FWGwKCkYog5Zokat8jvdgO6vLTz~LjYz5~pcnYuFrsMjbjKrnuEgxfJPTd8uc8EBJ5karqWS9GbnteiTsDwqA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7931429&ar=510716&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00204.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sequence of ‘SMa0056’ gene from 818th to 877th nucleotide (1) aligned against weight matrix <em>w</em>(<em>i</em>,<em>j</em>). Insertion of 841st nucleotide (nucleotide <em>t</em>) is shown by asterisk. Further under gene, the consensus sequence for TP (2) is shown, which is constructed by matrix <em>М</em> and also by RF existing in gene (3) and RF specified by TPM <em>М</em> named as ancient RF (4). Below translated fragments of the nucleotide sequences are shown. Translation is made according to RF existing in gene (5) and to ancient RF (6). Regions having RF shift are marked in bold.</p></div></div>                        <a id="7931430" scrollto-destination="7931430"></a>
<div id="DSP002F5" data-id="DSP002F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00205.jpeg?Expires=2147483647&Signature=SQL2aipzdmUl~a8d28wF1lRTizyJM2dZ499nZIyVq5wAmmeNbs5O~K7oJUSzAGgpXpiuZW43-5CbZLXiavv5NTBNQoqNUEykk3n8EEabQqcMOIKjIPalmf4rJavFGzrVz78r8CplRlB0jAYHx9Z8hNuaI7Wn8FewehhJ9i3UARUJKnheBdgrBz51IDV0ZvmnrqKkBY4zHFEZrpG-nDZREkMWfBmGaBPnAQvIiyb9EuXkGLn0iVlDwoF4PIYBwon8JaMfZZ3zZqxms3lXX28RN2XrhL4tKtq3r0MdP0Q9574Brgfye1ONlgrWs8XsRAsv6f0cUCY86ZtZ0akm1fkJsw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Resistance of sequence T with continuous TP of gene ‘mlr4742’ to nucleotides' substitutions. We selected a base in sequence T in a random manner and then substituted it, also in a random manner, by one of the nucleotides a, t, c or g (with probability of each being equal to 0.25). Thereafter, we estimated statistical significance of obtained sequence according to Equation (2). As follows from figure, statistical significance of TP decreases relatively slow during mutations' accumulation. For example, 10% difference from the original sequence still allows revealing TP at statistically significant level." path-from-xml="dsp00205.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7931430/dsp00205.jpeg" path-from-xml="dsp00205.jpeg" target="_blank">View large</a><a section="7931430" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/dsp00205.jpeg?Expires=2147483647&Signature=gjHEZfd6xMQcRUl7SNWj66ge9XjHX4cFzcbYpYnoWjssNJgoiBURHEnB-lRYaKcxEsuOATSVbLCAvabl04cWhrkiGjX7peaXcZ5rUnopl-El-r7xYH3NDJ3kRiuZfAO8L~0NZhyrZIFVZX5DRejxiJVfLKIjrBO7EKZjzOXhQyAIQRZ1pijnXqAB0-LGMY2NGs~tkoI5Xs~U57WFvu5HS07BK8emkwSDtd78kTARXQLMVnD90WaWFYCs2i8OdmbBcv62J~3nNClak67f-yaAsZYEcsF-XGr5~wLgE1kjTehRkugrp9ULpeZ4jf9DpGYmTmsePyU9LshuPmz5yBPlsg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7931430&ar=510716&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00205.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Resistance of sequence T with continuous TP of gene ‘mlr4742’ to nucleotides' substitutions. We selected a base in sequence <em>T</em> in a random manner and then substituted it, also in a random manner, by one of the nucleotides a, t, c or g (with probability of each being equal to 0.25). Thereafter, we estimated statistical significance of obtained sequence according to Equation (<a href="#DSP002M2" class="sectionLink xref-sec">2</a>). As follows from figure, statistical significance of TP decreases relatively slow during mutations' accumulation. For example, 10% difference from the original sequence still allows revealing TP at statistically significant level.</p></div></div><div content-id="DSP002F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/m_dsp00205.jpeg?Expires=2147483647&Signature=SQL2aipzdmUl~a8d28wF1lRTizyJM2dZ499nZIyVq5wAmmeNbs5O~K7oJUSzAGgpXpiuZW43-5CbZLXiavv5NTBNQoqNUEykk3n8EEabQqcMOIKjIPalmf4rJavFGzrVz78r8CplRlB0jAYHx9Z8hNuaI7Wn8FewehhJ9i3UARUJKnheBdgrBz51IDV0ZvmnrqKkBY4zHFEZrpG-nDZREkMWfBmGaBPnAQvIiyb9EuXkGLn0iVlDwoF4PIYBwon8JaMfZZ3zZqxms3lXX28RN2XrhL4tKtq3r0MdP0Q9574Brgfye1ONlgrWs8XsRAsv6f0cUCY86ZtZ0akm1fkJsw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Resistance of sequence T with continuous TP of gene ‘mlr4742’ to nucleotides' substitutions. We selected a base in sequence T in a random manner and then substituted it, also in a random manner, by one of the nucleotides a, t, c or g (with probability of each being equal to 0.25). Thereafter, we estimated statistical significance of obtained sequence according to Equation (2). As follows from figure, statistical significance of TP decreases relatively slow during mutations' accumulation. For example, 10% difference from the original sequence still allows revealing TP at statistically significant level." path-from-xml="dsp00205.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7931430/dsp00205.jpeg" path-from-xml="dsp00205.jpeg" target="_blank">View large</a><a section="7931430" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/dsp00205.jpeg?Expires=2147483647&Signature=gjHEZfd6xMQcRUl7SNWj66ge9XjHX4cFzcbYpYnoWjssNJgoiBURHEnB-lRYaKcxEsuOATSVbLCAvabl04cWhrkiGjX7peaXcZ5rUnopl-El-r7xYH3NDJ3kRiuZfAO8L~0NZhyrZIFVZX5DRejxiJVfLKIjrBO7EKZjzOXhQyAIQRZ1pijnXqAB0-LGMY2NGs~tkoI5Xs~U57WFvu5HS07BK8emkwSDtd78kTARXQLMVnD90WaWFYCs2i8OdmbBcv62J~3nNClak67f-yaAsZYEcsF-XGr5~wLgE1kjTehRkugrp9ULpeZ4jf9DpGYmTmsePyU9LshuPmz5yBPlsg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7931430&ar=510716&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00205.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Resistance of sequence T with continuous TP of gene ‘mlr4742’ to nucleotides' substitutions. We selected a base in sequence <em>T</em> in a random manner and then substituted it, also in a random manner, by one of the nucleotides a, t, c or g (with probability of each being equal to 0.25). Thereafter, we estimated statistical significance of obtained sequence according to Equation (<a href="#DSP002M2" class="sectionLink xref-sec">2</a>). As follows from figure, statistical significance of TP decreases relatively slow during mutations' accumulation. For example, 10% difference from the original sequence still allows revealing TP at statistically significant level.</p></div></div><p>It is also of great interest to point out that the amino acid sequence corresponding to the gene ‘SMa0056’ from the KEGG databank has similar amino acid sequences in the UniProtKB/Swiss-Prot databank. This amino acid sequence is designated as ‘Q931B9_RHIME’ in the UniProtKB/Swiss-Prot databank. Similarity is observed from the start of the protein to the 280th amino acid. This is true for a set of different mandelate racemase/muconate lactonizing enzymes and glucarate dehydratases (e.g. ‘Q024C0_SOLUE’, ‘A9FRR8_SORC5’). This set also contains five amino acid sequences (‘GUDH_STRCO’, ‘GUDX_ECOLI’, ‘GUDH_ECO57’, ‘GUDH_ECOLI’ and ‘GUDH_PSEPU’), where we found similarity to amino acid sequence produced from 842nd to the end of the gene by the ancient RF. Therefore, it can be assumed that these five sequences existed before the insertion of the 841st or independently of it. These sequences have similarity to the sequence ‘Q931B9_RHIME’ from 1st to 280th ±10 amino acids for the gene's (KEGG) RF and from 281st to 378th amino acids for ancient RF.</p>                            <h3 scrollto-destination=7931432 id="7931432" class="section-title" data-legacy-id=s3c>3.3. Discussion</h3>
<p>In this work, we managed to show that investigation of shifts between TP and RF can reveal possible mutations via RF shift in a gene. We found 824 such genes in which there existed some single-type TP regions separated by deletions or insertions of nucleotides. They account for ∼0.2% of the total number of analysed genes. We suppose that in the past RF and TP in these regions were explicitly linked to each other and shifts between them appeared only after deletions or insertions of nucleotides. Such relatively small fractions of genes in which an RF shift can potentially occur can be explained by the following reasons. First, we searched only for relatively short deletions or insertions having lengths up to 2 nucleotides (<em>d</em> = 2). We confined ourselves to small deletions and insertions because while the length of deletion and insertion increases, the value of function <em>F</em> decreases [see Equation (<a href="#DSP002M6" class="sectionLink xref-sec">6</a>)]. Costs of deletion and insertion can be too high; in this case, the value Δ<em>F</em><sub>1</sub> will be statistically insignificant. Thus, the current method will miss the major part of genes containing long deletion and insertion of nucleotides. Second, the technique used works well for the small number of regions where deletions or insertions of nucleotides occurred. If density of deletions and insertions is more than one deletion and insertion for several tens of nucleotides (∼50), then the accurate placement of deletions and insertions using this algorithm will be difficult. This will lead to the statistically insignificant value of Δ<em>F</em><sub>1</sub> for such a gene.</p><p>In general, in this work, we aimed to reveal the lower bound for the number of genes in which a shift between RF and TP is possible. Actually, there can be much more such genes. These values are also confirmed by data in work<sup><a reveal-id="DSP002C2" data-open="DSP002C2" class="link link-ref link-reveal xref-bibr">2</a></sup> where the number of genes with RF shift obtained by BLAST program is &gt;1%.</p><p>The technique used to find shifts between TP and RF and to reveal mutations via RF shift in genes seems to be more preferable than the usage of BLAST program for finding possible similarities. The current method of revealing mutations via an RF shift in the gene does not require finding similarities in the databank of amino acid sequences. Owing to the limited size of the databank, there will always exist a chance that similarities will not be found, although actually mutation via RF shift will exist. We suppose that complete revealing of mutations via an RF shift in genes is possible by the integration of these two techniques. That is, we also have to investigate genes having 0 ≤ Δ<em>F</em><sub>1</sub> ≤ Δ<em>F</em><sub>0</sub> and to consider that we found mutations via an RF shift if regions <em>T</em>1 and <em>T</em>2 have statistically significant similarities. In this case, the TP just indicates the possibility of an RF shift and the fact of such mutations can be considered to be proved only after revealing similarities with regions <em>T</em>1 and <em>T</em>2. On the other hand, enhancement of the algorithm used in the current work can facilitate the use of more perfect algorithms for finding TP, such as Markov models. This probably will allow us to reveal RF shifts induced by a set of nucleotides insertions and deletions in various gene regions.</p><p>From the functional point of view, mutations via RF shift seem to be events that are able to cardinally change the gene function. This fact can explain the relatively small number of such events found during investigations in the past and in the current works.<sup><a reveal-id="DSP002C2" data-open="DSP002C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSP002C3" data-open="DSP002C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSP002C35" data-open="DSP002C35" class="link link-ref link-reveal xref-bibr">35</a></sup> They can make a great contribution to the formation of new genes by copying known genes and generating mutations via an RF shift<sup><a reveal-id="DSP002C2" data-open="DSP002C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSP002C3" data-open="DSP002C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSP002C35" data-open="DSP002C35" class="link link-ref link-reveal xref-bibr">35</a></sup> in them. However, the genetic code has to be adapted for this event in some way,<sup><a reveal-id="DSP002C36" data-open="DSP002C36" class="link link-ref link-reveal xref-bibr">36</a></sup> and the new amino acid sequence has to possess some biological function. Otherwise, overrun of mutational events for the creation of the new gene's function can be too great, and impossible within reasonable evolutional time.</p><p>In the light of these assumptions, TP can be some kind of test to check the gene's integrity. If the gene was duplicated in the genome, then its new copy may fail the check, which opens up possibilities for evolutional changes of the gene's copy via the RF shift and for the creation of a new gene with a new biological function as a result.</p>                            <h2 scrollto-destination=7931438 id="7931438" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available online at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=7931440 id="7931440" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by Russian Academy of Sciences.</p>                            <h2 scrollto-destination=7931442 id="7931442" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSP002C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wei</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">D. J.</div></div>. </div>, <div class="source">DNA Repair, Genetic Instability, and Cancer</div> , <div class="year">2007</div><div class="publisher-loc">Singapore</div><div class="publisher-name">World Scientific Publishing Co. Pte, Ltd.</div></div></div></div></div><div content-id="DSP002C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Feuk</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Marquès-Bonet</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Navarro</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Scherer</div> <div class="given-names">S. W.</div></div>. </div><div class="article-title">Frequent appearance of novel protein-coding sequences by frameshift translation</div>, <div class="source">Genomics</div> , <div class="year">2006</div>, vol. <div class="volume">88</div> (pg. <div class="fpage">690</div>-<div class="lpage">697</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Frequent%20appearance%20of%20novel%20protein-coding%20sequences%20by%20frameshift%20translation&amp;author=K.%20Okamura&amp;author=L.%20Feuk&amp;author=T.%20Marqu%C3%A8s-Bonet&amp;author=A.%20Navarro&amp;author=S.%20W.%20Scherer&amp;publication_year=2006&amp;journal=Genomics&amp;volume=88&amp;pages=690-697" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ygeno.2006.06.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygeno.2006.06.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ygeno.2006.06.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16890400" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP002C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Raes</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Van de Peer</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Functional divergence of proteins through frameshift mutations</div>, <div class="source">Trends Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">428</div>-<div class="lpage">431</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20divergence%20of%20proteins%20through%20frameshift%20mutations&amp;author=J.%20Raes&amp;author=Y.%20Van%20de%20Peer&amp;publication_year=2005&amp;journal=Trends%20Genet.&amp;volume=21&amp;pages=428-431" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2005.05.013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2005.05.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2005.05.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15951050" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP002C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fickett</div> <div class="given-names">J. W.</div></div>. </div><div class="article-title">Predictive methods using nucleotide sequences</div>, <div class="source">Methods Biochem. Anal.</div> , <div class="year">1998</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">231</div>-<div class="lpage">245</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predictive%20methods%20using%20nucleotide%20sequences&amp;author=J.%20W.%20Fickett&amp;publication_year=1998&amp;journal=Methods%20Biochem.%20Anal.&amp;volume=39&amp;pages=231-245" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9707933" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP002C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Staden</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Staden: statistical and structural analysis of nucleotide sequences</div>, <div class="source">Methods Mol. Biol.</div> , <div class="year">1994</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">69</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Staden%3A%20statistical%20and%20structural%20analysis%20of%20nucleotide%20sequences&amp;author=R.%20Staden&amp;publication_year=1994&amp;journal=Methods%20Mol.%20Biol.&amp;volume=25&amp;pages=69-77" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8004181" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP002C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baxevanis</div> <div class="given-names">A. D.</div></div>. </div><div class="article-title">Predictive methods using DNA sequences</div>, <div class="source">Methods Biochem. Anal.</div> , <div class="year">2001</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">233</div>-<div class="lpage">252</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predictive%20methods%20using%20DNA%20sequences&amp;author=A.%20D.%20Baxevanis&amp;publication_year=2001&amp;journal=Methods%20Biochem.%20Anal.&amp;volume=43&amp;pages=233-252" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11449727" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP002C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gutiérrez</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Oliver</div> <div class="given-names">J. L.</div></div>,  <div class="name"><div class="surname">Marín</div> <div class="given-names">A.</div></div>. </div><div class="article-title">On the origin of the periodicity of three in protein coding DNA sequences</div>, <div class="source">J. Theor. Biol.</div> , <div class="year">1994</div>, vol. <div class="volume">167</div> <div class="issue">4</div>(pg. <div class="fpage">413</div>-<div class="lpage">414</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=On%20the%20origin%20of%20the%20periodicity%20of%20three%20in%20protein%20coding%20DNA%20sequences&amp;author=G.%20Guti%C3%A9rrez&amp;author=J.%20L.%20Oliver&amp;author=A.%20Mar%C3%ADn&amp;publication_year=1994&amp;journal=J.%20Theor.%20Biol.&amp;volume=167&amp;pages=413-414" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jtbi.1994.1080" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjtbi.1994.1080" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjtbi.1994.1080"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8207954" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP002C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gao</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Qi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Cao</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tung</div> <div class="given-names">W. W.</div></div>. </div><div class="article-title">Protein coding sequence identification by simultaneously characterizing the periodic and random features of DNA sequences</div>, <div class="source">J. Biomed. Biotechnol.</div> , <div class="year">2005</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">139</div>-<div class="lpage">146</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20coding%20sequence%20identification%20by%20simultaneously%20characterizing%20the%20periodic%20and%20random%20features%20of%20DNA%20sequences&amp;author=J.%20Gao&amp;author=Y.%20Qi&amp;author=Y.%20Cao&amp;author=W.%20W.%20Tung&amp;publication_year=2005&amp;journal=J.%20Biomed.%20Biotechnol.&amp;volume=2&amp;pages=139-146" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1155/JBB.2005.139" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1155%2FJBB.2005.139" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1155%2FJBB.2005.139"> </span></div> </div></div></div></div></div><div content-id="DSP002C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yin</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Yau</div> <div class="given-names">S. S.</div></div>. </div><div class="article-title">Prediction of protein coding regions by the 3-base periodicity analysis of a DNA sequence</div>, <div class="source">J. Theor. Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">247</div> (pg. <div class="fpage">687</div>-<div class="lpage">694</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20protein%20coding%20regions%20by%20the%203-base%20periodicity%20analysis%20of%20a%20DNA%20sequence&amp;author=C.%20Yin&amp;author=S.%20S.%20Yau&amp;publication_year=2007&amp;journal=J.%20Theor.%20Biol.&amp;volume=247&amp;pages=687-694" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jtbi.2007.03.038" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jtbi.2007.03.038" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jtbi.2007.03.038"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17509616" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP002C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eskesen</div> <div class="given-names">S. T.</div></div>,  <div class="name"><div class="surname">Eskesen</div> <div class="given-names">F. N.</div></div>,  <div class="name"><div class="surname">Kinghorn</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Ruvinsky</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Periodicity of DNA in exons</div>, <div class="source">BMC Mol. Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> pg. <div class="fpage">12</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Periodicity%20of%20DNA%20in%20exons&amp;author=S.%20T.%20Eskesen&amp;author=F.%20N.%20Eskesen&amp;author=B.%20Kinghorn&amp;author=A.%20Ruvinsky&amp;publication_year=2004&amp;journal=BMC%20Mol.%20Biol.&amp;volume=5&amp;pages=12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2199-5-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2199-5-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2199-5-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15315715" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP002C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bibb</div> <div class="given-names">M. J.</div></div>,  <div class="name"><div class="surname">Findlay</div> <div class="given-names">P. R.</div></div>,  <div class="name"><div class="surname">Johnson</div> <div class="given-names">M. W.</div></div>. </div><div class="article-title">The relationship between base composition and codon usage in bacterial genes and its use for the simple and reliable identification of protein-coding sequences</div>, <div class="source">Gene</div> , <div class="year">1984</div>, vol. <div class="volume">30</div> <div class="issue">1–3</div>(pg. <div class="fpage">157</div>-<div class="lpage">166</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20relationship%20between%20base%20composition%20and%20codon%20usage%20in%20bacterial%20genes%20and%20its%20use%20for%20the%20simple%20and%20reliable%20identification%20of%20protein-coding%20sequences&amp;author=M.%20J.%20Bibb&amp;author=P.%20R.%20Findlay&amp;author=M.%20W.%20Johnson&amp;publication_year=1984&amp;journal=Gene&amp;volume=30&amp;pages=157-166" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(84)90116-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(84)90116-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(84)90116-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6096212" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP002C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Konopka</div> <div class="given-names">A. K.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Smith</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Sequences and codes: fundamentals of biomolecular cryptography</div>, <div class="source">Biocomputing: Informatics and Genome Projects</div> , <div class="year">1994</div><div class="publisher-loc">San Diego</div><div class="publisher-name">Academic Press</div>(pg. <div class="fpage">119</div>-<div class="lpage">174</div>)</div></div></div></div><div content-id="DSP002C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Trifonov</div> <div class="given-names">E. N.</div></div>. </div><div class="article-title">Elucidating sequence codes: three codes for evolution</div>, <div class="source">Ann. NY Acad. Sci.</div> , <div class="year">1999</div>, vol. <div class="volume">870</div> (pg. <div class="fpage">330</div>-<div class="lpage">338</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Elucidating%20sequence%20codes%3A%20three%20codes%20for%20evolution&amp;author=E.%20N.%20Trifonov&amp;publication_year=1999&amp;journal=Ann.%20NY%20Acad.%20Sci.&amp;volume=870&amp;pages=330-338" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1749-6632.1999.tb08894.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1749-6632.1999.tb08894.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1749-6632.1999.tb08894.x"> </span></div> </div></div></div></div></div><div content-id="DSP002C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eigen</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Winkler-Oswatitsch</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Transfer-RNA: the early adaptor</div>, <div class="source">Naturwissenschaften</div> , <div class="year">1981</div>, vol. <div class="volume">68</div> (pg. <div class="fpage">217</div>-<div class="lpage">228</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transfer-RNA%3A%20the%20early%20adaptor&amp;author=M.%20Eigen&amp;author=R.%20Winkler-Oswatitsch&amp;publication_year=1981&amp;journal=Naturwissenschaften&amp;volume=68&amp;pages=217-228" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF01047323" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01047323" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF01047323"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6909552" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP002C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zoltowski</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Is DNA code periodicity only due to CUF—codons usage frequency?</div>, <div class="source">Conf. Proc. IEEE Eng. Med. Biol. Soc.</div> , <div class="year">2007</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">1383</div>-<div class="lpage">1386</div>)</div></div></div></div><div content-id="DSP002C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Antezana</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Kreitman</div> <div class="given-names">M.</div></div>. </div><div class="article-title">The nonrandom location of synonymous codons suggests that reading frame-independent forces have patterned codon preferences</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1999</div>, vol. <div class="volume">49</div> <div class="issue">1</div>(pg. <div class="fpage">36</div>-<div class="lpage">43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20nonrandom%20location%20of%20synonymous%20codons%20suggests%20that%20reading%20frame-independent%20forces%20have%20patterned%20codon%20preferences&amp;author=M.%20A.%20Antezana&amp;author=M.%20Kreitman&amp;publication_year=1999&amp;journal=J.%20Mol.%20Evol.&amp;volume=49&amp;pages=36-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00006532" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00006532" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00006532"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10368432" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP002C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Korotkov</div> <div class="given-names">E. V.</div></div>,  <div class="name"><div class="surname">Korotkova</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Frenkel’</div> <div class="given-names">F. E.</div></div>,  <div class="name"><div class="surname">Kudriashov</div> <div class="given-names">N. A.</div></div>. </div><div class="article-title">The informational concept of searching for periodicity in symbol sequences</div>, <div class="source">Mol. Biol. (Mosk)</div> , <div class="year">2003</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">436</div>-<div class="lpage">451</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20informational%20concept%20of%20searching%20for%20periodicity%20in%20symbol%20sequences&amp;author=E.%20V.%20Korotkov&amp;author=M.%20A.%20Korotkova&amp;author=F.%20E.%20Frenkel%E2%80%99&amp;author=N.%20A.%20Kudriashov&amp;publication_year=2003&amp;journal=Mol.%20Biol.%20(Mosk)&amp;volume=37&amp;pages=436-451" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1024231109360" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1024231109360" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1024231109360"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12815951" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP002C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Issac</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Singh</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kaur</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Raghava</div> <div class="given-names">G. P. S.</div></div>. </div><div class="article-title">Locating probable genes using Fourier transform approach</div>, <div class="source">Bioinformatics</div> , <div class="year">2002</div>, vol. <div class="volume">18</div> <div class="issue">1</div>(pg. <div class="fpage">196</div>-<div class="lpage">197</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Locating%20probable%20genes%20using%20Fourier%20transform%20approach&amp;author=B.%20Issac&amp;author=H.%20Singh&amp;author=H.%20Kaur&amp;author=G.%20P.%20S.%20Raghava&amp;publication_year=2002&amp;journal=Bioinformatics&amp;volume=18&amp;pages=196-197" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/18.1.196" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F18.1.196" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F18.1.196"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11836230" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP002C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tiwari</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ramachandran</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bhattacharya</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bhattacharya</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ramaswamy</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Prediction of probable genes by Fourier analysis of genomic sequences</div>, <div class="source">Comput. Appl. Biosci.</div> , <div class="year">1997</div>, vol. <div class="volume">13</div> <div class="issue">3</div>(pg. <div class="fpage">263</div>-<div class="lpage">270</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20probable%20genes%20by%20Fourier%20analysis%20of%20genomic%20sequences&amp;author=S.%20Tiwari&amp;author=S.%20Ramachandran&amp;author=A.%20Bhattacharya&amp;author=S.%20Bhattacharya&amp;author=R.%20Ramaswamy&amp;publication_year=1997&amp;journal=Comput.%20Appl.%20Biosci.&amp;volume=13&amp;pages=263-270" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9183531" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP002C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Azad</div> <div class="given-names">R. K.</div></div>,  <div class="name"><div class="surname">Borodovsky</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Probabilistic methods of identifying genes in prokaryotic genomes: connections to the HMM theory</div>, <div class="source">Brief. Bioinform.</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> <div class="issue">2</div>(pg. <div class="fpage">118</div>-<div class="lpage">130</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Probabilistic%20methods%20of%20identifying%20genes%20in%20prokaryotic%20genomes%3A%20connections%20to%20the%20HMM%20theory&amp;author=R.%20K.%20Azad&amp;author=M.%20Borodovsky&amp;publication_year=2004&amp;journal=Brief.%20Bioinform.&amp;volume=5&amp;pages=118-130" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/5.2.118" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2F5.2.118" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbib%2F5.2.118"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15260893" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP002C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Henderson</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Salzberg</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Fasman</div> <div class="given-names">K. H.</div></div>. </div><div class="article-title">Finding genes in DNA with a hidden Markov model</div>, <div class="source">J. Comput. Biol.</div> , <div class="year">1997</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">127</div>-<div class="lpage">141</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Finding%20genes%20in%20DNA%20with%20a%20hidden%20Markov%20model&amp;author=J.%20Henderson&amp;author=S.%20Salzberg&amp;author=K.%20H.%20Fasman&amp;publication_year=1997&amp;journal=J.%20Comput.%20Biol.&amp;volume=4&amp;pages=127-141" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/cmb.1997.4.127" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fcmb.1997.4.127" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2Fcmb.1997.4.127"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9228612" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP002C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Snyder</div> <div class="given-names">E. E.</div></div>,  <div class="name"><div class="surname">Stormo</div> <div class="given-names">G. D.</div></div>. </div><div class="article-title">Identification of coding regions in genomic DNA sequences: an application of dynamic programming and neural networks</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1993</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">607</div>-<div class="lpage">613</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20coding%20regions%20in%20genomic%20DNA%20sequences%3A%20an%20application%20of%20dynamic%20programming%20and%20neural%20networks&amp;author=E.%20E.%20Snyder&amp;author=G.%20D.%20Stormo&amp;publication_year=1993&amp;journal=Nucleic%20Acids%20Res.&amp;volume=21&amp;pages=607-613" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/21.3.607" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F21.3.607" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F21.3.607"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8441672" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP002C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thomas</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Skolnick</div> <div class="given-names">M. H.</div></div>. </div><div class="article-title">A probabilistic model for detecting coding regions in DNA sequences</div>, <div class="source">IMA J. Math. Appl. Med. Biol.</div> , <div class="year">1994</div>, vol. <div class="volume">11</div> <div class="issue">3</div>(pg. <div class="fpage">149</div>-<div class="lpage">160</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20probabilistic%20model%20for%20detecting%20coding%20regions%20in%20DNA%20sequences&amp;author=A.%20Thomas&amp;author=M.%20H.%20Skolnick&amp;publication_year=1994&amp;journal=IMA%20J.%20Math.%20Appl.%20Med.%20Biol.&amp;volume=11&amp;pages=149-160" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/imammb/11.3.149" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fimammb%2F11.3.149" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fimammb%2F11.3.149"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7822887" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP002C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Korotkov</div> <div class="given-names">E. V.</div></div>,  <div class="name"><div class="surname">Korotkova</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Kudryshov</div> <div class="given-names">N. A.</div></div>. </div><div class="article-title">Information decomposition method for analysis of symbolical sequences</div>, <div class="source">Phys. Lett. A</div> , <div class="year">2003</div>, vol. <div class="volume">312</div> (pg. <div class="fpage">198</div>-<div class="lpage">210</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Information%20decomposition%20method%20for%20analysis%20of%20symbolical%20sequences&amp;author=E.%20V.%20Korotkov&amp;author=M.%20A.%20Korotkova&amp;author=N.%20A.%20Kudryshov&amp;publication_year=2003&amp;journal=Phys.%20Lett.%20A&amp;volume=312&amp;pages=198-210" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0375-9601(03)00641-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0375-9601(03)00641-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0375-9601(03)00641-8"> </span></div> </div></div></div></div></div><div content-id="DSP002C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ogata</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Goto</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Fujibuchi</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Bono</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kanehisa</div> <div class="given-names">M.</div></div>. </div><div class="article-title">KEGG: Kyoto Encyclopedia of Genes and Genomes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1999</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">29</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KEGG%3A%20Kyoto%20Encyclopedia%20of%20Genes%20and%20Genomes&amp;author=H.%20Ogata&amp;author=S.%20Goto&amp;author=K.%20Sato&amp;author=W.%20Fujibuchi&amp;author=H.%20Bono&amp;author=M.%20Kanehisa&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=29-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.1.29" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.1.29" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.1.29"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9847135" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP002C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kullback</div> <div class="given-names">S.</div></div>. </div>, <div class="source">Information Theory and Statistics</div> , <div class="year">1959</div><div class="publisher-loc">New York</div><div class="publisher-name">Wiley</div></div></div></div></div><div content-id="DSP002C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hudson</div> <div class="given-names">D. J.</div></div>. </div>, <div class="source">Statistics: Lectures on Elementary Statistics and Probability</div> , <div class="year">1964</div><div class="publisher-loc">Geneva</div><div class="publisher-name">CERN</div></div></div></div></div><div content-id="DSP002C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Frenkel</div> <div class="given-names">F. E.</div></div>,  <div class="name"><div class="surname">Chaley</div> <div class="given-names">M. B.</div></div>,  <div class="name"><div class="surname">Korotkov</div> <div class="given-names">E. V.</div></div>,  <div class="name"><div class="surname">Skryabin</div> <div class="given-names">K. G.</div></div>. </div><div class="article-title">Evolution of the tRNA-like sequences and genome variability</div>, <div class="source">Gene</div> , <div class="year">2004</div>, vol. <div class="volume">335</div> (pg. <div class="fpage">57</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20the%20tRNA-like%20sequences%20and%20genome%20variability&amp;author=F.%20E.%20Frenkel&amp;author=M.%20B.%20Chaley&amp;author=E.%20V.%20Korotkov&amp;author=K.%20G.%20Skryabin&amp;publication_year=2004&amp;journal=Gene&amp;volume=335&amp;pages=57-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2004.03.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2004.03.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2004.03.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15194190" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP002C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chaley</div> <div class="given-names">M. B.</div></div>,  <div class="name"><div class="surname">Korotkov</div> <div class="given-names">E. V.</div></div>,  <div class="name"><div class="surname">Kudryashov</div> <div class="given-names">N. A.</div></div>. </div><div class="article-title">Latent periodicity of 21 bases typical for MCP II gene is widely present in various bacterial genes</div>, <div class="source">DNA Seq.</div> , <div class="year">2003</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">37</div>-<div class="lpage">52</div>)</div></div></div></div><div content-id="DSP002C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Needleman</div> <div class="given-names">S. B.</div></div>,  <div class="name"><div class="surname">Wunsch</div> <div class="given-names">C. D.</div></div>. </div><div class="article-title">A general method applicable to the search for similarities in the amino acid sequence of two proteins</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1970</div>, vol. <div class="volume">48</div> <div class="issue">3</div>(pg. <div class="fpage">443</div>-<div class="lpage">453</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20general%20method%20applicable%20to%20the%20search%20for%20similarities%20in%20the%20amino%20acid%20sequence%20of%20two%20proteins&amp;author=S.%20B.%20Needleman&amp;author=C.%20D.%20Wunsch&amp;publication_year=1970&amp;journal=J.%20Mol.%20Biol.&amp;volume=48&amp;pages=443-453" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(70)90057-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(70)90057-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(70)90057-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5420325" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP002C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Eddy</div> <div class="given-names">S. R.</div></div>,  <div class="name"><div class="surname">Krogh</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Graeme Mitchison</div> <div class="given-names">G.</div></div>. </div>, <div class="source">Biological Sequence Analysis: Probabilistic Models of Proteins and Nucleic Acids</div> , <div class="year">1999</div><div class="publisher-loc">Cambridge, UK</div><div class="publisher-name">Cambridge University Press</div></div></div></div></div><div content-id="DSP002C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Frenkel</div> <div class="given-names">F. E.</div></div>,  <div class="name"><div class="surname">Korotkov</div> <div class="given-names">E. V.</div></div>. </div><div class="article-title">Classification analysis of triplet periodicity in protein-coding regions of genes</div>, <div class="source">Gene</div> , <div class="year">2008</div>, vol. <div class="volume">421</div> (pg. <div class="fpage">52</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Classification%20analysis%20of%20triplet%20periodicity%20in%20protein-coding%20regions%20of%20genes&amp;author=F.%20E.%20Frenkel&amp;author=E.%20V.%20Korotkov&amp;publication_year=2008&amp;journal=Gene&amp;volume=421&amp;pages=52-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2008.06.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2008.06.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2008.06.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18593596" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP002C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="collab">UniProt Consortium</div><div class="article-title">The universal protein resource (UniProt)</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">193</div>-<div class="lpage">197</div>)<div class="citation-links"><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17158163" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP002C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>,  <div class="name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">E. W.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D. J.</div></div>. </div><div class="article-title">Basic local alignment search tool</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1990</div>, vol. <div class="volume">215</div> <div class="issue">3</div>(pg. <div class="fpage">403</div>-<div class="lpage">410</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.%20F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.%20W.%20Myers&amp;author=D.%20J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol.&amp;volume=215&amp;pages=403-410" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP002C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kramer</div> <div class="given-names">E. M.</div></div>,  <div class="name"><div class="surname">Su</div> <div class="given-names">H. J.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">C. C.</div></div>,  <div class="name"><div class="surname">Hu</div> <div class="given-names">J. M.</div></div>. </div><div class="article-title">A simplified explanation for the frameshift mutation that created a novel C-terminal motif in the APETALA3 gene lineage</div>, <div class="source">BMC Evol. Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">6</div> pg. <div class="fpage">30</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simplified%20explanation%20for%20the%20frameshift%20mutation%20that%20created%20a%20novel%20C-terminal%20motif%20in%20the%20APETALA3%20gene%20lineage&amp;author=E.%20M.%20Kramer&amp;author=H.%20J.%20Su&amp;author=C.%20C.%20Wu&amp;author=J.%20M.%20Hu&amp;publication_year=2006&amp;journal=BMC%20Evol.%20Biol.&amp;volume=6&amp;pages=30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-6-30" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-6-30" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-6-30"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16563166" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP002C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP002C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bollenbach</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Vetsigian</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kishony</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Evolution and multilevel optimization of the genetic code</div>, <div class="source">Genome Res.</div> , <div class="year">2007</div>, vol. <div class="volume">17</div> <div class="issue">4</div>(pg. <div class="fpage">405</div>-<div class="lpage">412</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20and%20multilevel%20optimization%20of%20the%20genetic%20code&amp;author=T.%20Bollenbach&amp;author=K.%20Vetsigian&amp;author=R.%20Kishony&amp;publication_year=2007&amp;journal=Genome%20Res.&amp;volume=17&amp;pages=405-412" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.6144007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.6144007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.6144007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17293451" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Hiroyuki Toh</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2009. Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=510716&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp002/2/dsp002-dsp002_supp.xls?Expires=2147483647&amp;Signature=rV0OYSpVOi4MW~lvR~v5Ly8JxOCT2xy4DlTpdOhQbW85eBhE0ac0VD-WclWrrthu1GKq6DZD~URS0lyonWi6kAZ3ooKh~PqZkZzuN1QnVWpopgwvEvkxu57ZovMtBbjuyGLQfomRxvNfU2MEoRy~PD5Tw8GXtQhW-mI4dCxe27AZy2hrM-5p9qJnJXS3uKIrxOq3KStCyrnCTwRRkPwz44jOh2Y7Pg32KU2qr3nH~ZFB33UA~Mry~YDitlvgUPnQ8nLrd1myLQXHpid4~SIOhJmZkEdPEPRfV-mzYuulcYM5kC9feyGPpsonFZaBoh2mHnHbdc4AFVSu8vway8VAkw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">167</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000265770800003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">15</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsp002"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="510716"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000265770800003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f16%2f2%2f105%2f510716">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000265770800003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (15)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F16%2F2%2F105%2F510716" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/510716'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Using Triplet Periodicity of Nucleotide Sequences for Finding Potential Reading Frame Shifts in Genes" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsp002");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f16%2f2%2f105%2f510716">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

