

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/6/559/3868561","@type":"ScholarlyArticle","name":"Expression properties exhibit correlated patterns with the fate of duplicated genes, their divergence, and transcriptional plasticity in  Saccharomycotina ","datePublished":"2017-06-15","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2017-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx025","keywords":["gene expression","gene duplication","transcriptional plasticity","duplicability","<italic>Saccharomyces cerevisiae</italic>"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/6/559/3868561","author":[{"name":"Mattenberger, Florian","affiliation":"Department of Abiotic Stress, Instituto de Biología Molecular y Celular de Plantas, Consejo Superior de Investigaciones Científicas-Universidad Politécnica de Valencia, Valencia 46022, Spain  Systems Biology of Molecular Interactions and Regulation Department, Institute for Integrative Systems Biology (I2SysBio), Consejo Superior de Investigaciones, Científicas-Universitat de Valencia (CSIC-UV), Valencia 46980, Spain","@type":"Person"},{"name":"Sabater-Muñoz, Beatriz","affiliation":"Department of Abiotic Stress, Instituto de Biología Molecular y Celular de Plantas, Consejo Superior de Investigaciones Científicas-Universidad Politécnica de Valencia, Valencia 46022, Spain  Systems Biology of Molecular Interactions and Regulation Department, Institute for Integrative Systems Biology (I2SysBio), Consejo Superior de Investigaciones, Científicas-Universitat de Valencia (CSIC-UV), Valencia 46980, Spain  Department of Genetics, Smurfit Institute of Genetics, University of Dublin, Trinity College, Dublin, Ireland","@type":"Person"},{"name":"Toft, Christina","affiliation":"Department of Genetics, University of Valencia, Burjasot, Valencia 46100, Spain  Department of Biotechnology, Instituto de Agroquímica y Tecnología de los Alimentos, Consejo Superior de Investigaciones Científicas (CSIC), Burjasot, Valencia, Spain","@type":"Person"},{"name":"Sablok, Gaurav","affiliation":"Plant Functional Biology and Climate Change Cluster (C3), University of Technology Sydney, Sydney, NSW 2007, Australia","@type":"Person"},{"name":"Fares, Mario A","affiliation":"Department of Abiotic Stress, Instituto de Biología Molecular y Celular de Plantas, Consejo Superior de Investigaciones Científicas-Universidad Politécnica de Valencia, Valencia 46022, Spain  Systems Biology of Molecular Interactions and Regulation Department, Institute for Integrative Systems Biology (I2SysBio), Consejo Superior de Investigaciones, Científicas-Universitat de Valencia (CSIC-UV), Valencia 46980, Spain  Department of Genetics, Smurfit Institute of Genetics, University of Dublin, Trinity College, Dublin, Ireland","@type":"Person"}],"description":"Abstract.  Gene duplication is an important source of novelties and genome complexity. What genes are preserved as duplicated through long evolutionary times ca","pageStart":"559","pageEnd":"570","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/6/2/m_dnares_24_6cover.png?Expires=2147483647&Signature=z4yCzT-6enr7Fof9QqkUJvtPQAXDRV5XYB7R7L2HBbQZ2iaySlHxBG~hDePAa-rY6Gn-PH9W8mIlJwAyG9R2RE6Vd3RTDct~I7biyNGgXk9CP54tGn91x8VYIMO24kwuQBqahsZQMv3NxRiQKhbGI0yEDanozvBgG~-9FpaC12JFLE6A4BT6rBzjsJLh9UJqvGv0F2hWW2jLvdITi~5PoATpRIsbIK-S1wcvqp49OSN~oRepZtr6iz~107ho8gZAxoaG8nb4DvXKrvfnCuhhmbRD41NP3K2zFyztHicWtTYb1xunX52RqodNq-M3zVvUit2rSw5-mRSvm-jXb-p6nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Expression properties exhibit correlated patterns with the fate of duplicated genes, their divergence, and transcriptional plasticity in  Saccharomycotina ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/6/2/m_dnares_24_6cover.png?Expires=2147483647&Signature=z4yCzT-6enr7Fof9QqkUJvtPQAXDRV5XYB7R7L2HBbQZ2iaySlHxBG~hDePAa-rY6Gn-PH9W8mIlJwAyG9R2RE6Vd3RTDct~I7biyNGgXk9CP54tGn91x8VYIMO24kwuQBqahsZQMv3NxRiQKhbGI0yEDanozvBgG~-9FpaC12JFLE6A4BT6rBzjsJLh9UJqvGv0F2hWW2jLvdITi~5PoATpRIsbIK-S1wcvqp49OSN~oRepZtr6iz~107ho8gZAxoaG8nb4DvXKrvfnCuhhmbRD41NP3K2zFyztHicWtTYb1xunX52RqodNq-M3zVvUit2rSw5-mRSvm-jXb-p6nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Expression properties exhibit correlated patterns with the fate of duplicated genes, their divergence, and transcriptional plasticity in  Saccharomycotina " />
    <meta property="og:description" content="Abstract.  Gene duplication is an important source of novelties and genome complexity. What genes are preserved as duplicated through long evolutionary times ca" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/6/559/3868561" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-06-15" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Mattenberger, Florian" /><meta name="citation_author_institution" content="Department of Abiotic Stress, Instituto de Biología Molecular y Celular de Plantas, Consejo Superior de Investigaciones Científicas-Universidad Politécnica de Valencia, Valencia 46022, Spain" /><meta name="citation_author_institution" content="Systems Biology of Molecular Interactions and Regulation Department, Institute for Integrative Systems Biology (I2SysBio), Consejo Superior de Investigaciones, Científicas-Universitat de Valencia (CSIC-UV), Valencia 46980, Spain" /><meta name="citation_author" content="Sabater-Muñoz, Beatriz" /><meta name="citation_author_institution" content="Department of Abiotic Stress, Instituto de Biología Molecular y Celular de Plantas, Consejo Superior de Investigaciones Científicas-Universidad Politécnica de Valencia, Valencia 46022, Spain" /><meta name="citation_author_institution" content="Systems Biology of Molecular Interactions and Regulation Department, Institute for Integrative Systems Biology (I2SysBio), Consejo Superior de Investigaciones, Científicas-Universitat de Valencia (CSIC-UV), Valencia 46980, Spain" /><meta name="citation_author_institution" content="Department of Genetics, Smurfit Institute of Genetics, University of Dublin, Trinity College, Dublin, Ireland" /><meta name="citation_author" content="Toft, Christina" /><meta name="citation_author_institution" content="Department of Genetics, University of Valencia, Burjasot, Valencia 46100, Spain" /><meta name="citation_author_institution" content="Department of Biotechnology, Instituto de Agroquímica y Tecnología de los Alimentos, Consejo Superior de Investigaciones Científicas (CSIC), Burjasot, Valencia, Spain" /><meta name="citation_author" content="Sablok, Gaurav" /><meta name="citation_author_institution" content="Plant Functional Biology and Climate Change Cluster (C3), University of Technology Sydney, Sydney, NSW 2007, Australia" /><meta name="citation_author" content="Fares, Mario A" /><meta name="citation_author_institution" content="Department of Abiotic Stress, Instituto de Biología Molecular y Celular de Plantas, Consejo Superior de Investigaciones Científicas-Universidad Politécnica de Valencia, Valencia 46022, Spain" /><meta name="citation_author_institution" content="Systems Biology of Molecular Interactions and Regulation Department, Institute for Integrative Systems Biology (I2SysBio), Consejo Superior de Investigaciones, Científicas-Universitat de Valencia (CSIC-UV), Valencia 46980, Spain" /><meta name="citation_author_institution" content="Department of Genetics, Smurfit Institute of Genetics, University of Dublin, Trinity College, Dublin, Ireland" /><meta name="citation_title" content="Expression properties exhibit correlated patterns with the fate of duplicated genes, their divergence, and transcriptional plasticity in Saccharomycotina" /><meta name="citation_firstpage" content="559" /><meta name="citation_lastpage" content="570" /><meta name="citation_doi" content="10.1093/dnares/dsx025" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2017/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_publisher=Springer Verlag, New York;  citation_title=Evolution by Gene Duplication;  citation_year=1970; " /><meta name="citation_reference" content="citation_title=Gene duplication and the uniqueness of vertebrate genomes circa 1970–1999,;  citation_journal_title=Semin. Cell Dev. Biol;  citation_year=1999;  citation_volume=10;  citation_pages=517-22; " /><meta name="citation_reference" content="citation_title=The evolutionary fate and consequences of duplicate genes,;  citation_journal_title=Science;  citation_year=2000;  citation_volume=290;  citation_pages=1151-5; " /><meta name="citation_reference" content="citation_title=Polyploid incidence and evolution,;  citation_journal_title=Annu. Rev. Genet;  citation_year=2000;  citation_volume=34;  citation_pages=401-37; " /><meta name="citation_reference" content="citation_title=Evolutionary dynamics and functional specialization of plant paralogs formed by whole and small-scale genome duplications,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2012;  citation_volume=29;  citation_pages=3541-51; " /><meta name="citation_reference" content="citation_title=Widespread genome duplications throughout the history of flowering plants,;  citation_journal_title=Genome Res;  citation_year=2006;  citation_volume=16;  citation_pages=738-49; " /><meta name="citation_reference" content="citation_title=The arms race is ancient history in Arabidopsis, the wildflower,;  citation_journal_title=Nat. Rev. Genet;  citation_year=2001;  citation_volume=2;  citation_pages=516-27; " /><meta name="citation_reference" content="citation_title=The role of lineage-specific gene family expansion in the evolution of eukaryotes,;  citation_journal_title=Genome Res;  citation_year=2002;  citation_volume=12;  citation_pages=1048-59; " /><meta name="citation_reference" content="citation_title=Genome evolution in polyploids,;  citation_journal_title=Plant Mol. Biol;  citation_year=2000;  citation_volume=42;  citation_pages=225-49; " /><meta name="citation_reference" content="citation_title=Polyploidy and angiosperm diversification,;  citation_journal_title=Am. J. Bot;  citation_year=2009;  citation_volume=96;  citation_pages=336-48; " /><meta name="citation_reference" content="citation_title=Computational approaches to unveiling ancient genome duplications,;  citation_journal_title=Nat. Rev. Genet;  citation_year=2004;  citation_volume=5;  citation_pages=752-63; " /><meta name="citation_reference" content="citation_title=Phylogenetic timing of the fish-specific genome duplication correlates with the diversification of teleost fish,;  citation_journal_title=J. Mol. Evol;  citation_year=2004;  citation_volume=59;  citation_pages=190-203; " /><meta name="citation_reference" content="citation_title=The molecular evolution of the vertebrate behavioural repertoire,;  citation_journal_title=Philos. Trans. R. Soc. Lond. B: Biol. Sci;  citation_year=2016;  citation_volume=371;  citation_pages=20150051" /><meta name="citation_reference" content="citation_title=Gene duplications and the early evolution of neural crest development,;  citation_journal_title=Semin. Cell Dev. Biol;  citation_year=2013;  citation_volume=24;  citation_pages=95-100; " /><meta name="citation_reference" content="citation_title=Polyploidy and the evolution of complex traits,;  citation_journal_title=Int. J. Evol. Biol;  citation_year=2012;  citation_volume=2012;  citation_pages=292068" /><meta name="citation_reference" content="citation_title=Increased glycolytic flux as an outcome of whole-genome duplication in yeast,;  citation_journal_title=Mol. Syst. Biol;  citation_year=2007;  citation_volume=3;  citation_pages=129" /><meta name="citation_reference" content="citation_title=Molecular evidence for an ancient duplication of the entire yeast genome,;  citation_journal_title=Nature;  citation_year=1997;  citation_volume=387;  citation_pages=708-13; " /><meta name="citation_reference" content="citation_title=The roles of whole-genome and small-scale duplications in the functional specialization of Saccharomyces cerevisiae genes,;  citation_journal_title=PLoS Genet;  citation_year=2013;  citation_volume=9;  citation_pages=e1003176" /><meta name="citation_reference" content="citation_title=Widespread paleopolyploidy in model plant species inferred from age distributions of duplicate genes,;  citation_journal_title=Plant Cell;  citation_year=2004;  citation_volume=16;  citation_pages=1667-78; " /><meta name="citation_reference" content="citation_title=Functional divergence of duplicated genes formed by polyploidy during Arabidopsis evolution,;  citation_journal_title=Plant Cell;  citation_year=2004;  citation_volume=16;  citation_pages=1679-91; " /><meta name="citation_reference" content="citation_title=Turning a hobby into a job: how duplicated genes find new functions,;  citation_journal_title=Nat. Rev. Genet;  citation_year=2008;  citation_volume=9;  citation_pages=938-50; " /><meta name="citation_reference" content="citation_title=Rate asymmetry after genome duplication causes substantial long-branch attraction artifacts in the phylogeny of Saccharomyces species;  citation_journal_title=Mol. Biol. Evol;  citation_year=2006;  citation_volume=23;  citation_pages=245-53; " /><meta name="citation_reference" content="citation_title=A burst of protein sequence evolution and a prolonged period of asymmetric evolution follow gene duplication in yeast,;  citation_journal_title=Genome Res;  citation_year=2008;  citation_volume=18;  citation_pages=137-47; " /><meta name="citation_reference" content="citation_title=Bias in plant gene content following different sorts of duplication: tandem, whole-genome, segmental, or by transposition,;  citation_journal_title=Annu. Rev. Plant Biol;  citation_year=2009;  citation_volume=60;  citation_pages=433-53; " /><meta name="citation_reference" content="citation_title=Dosage, duplication, and diploidization: clarifying the interplay of multiple models for duplicate gene evolution over time,;  citation_journal_title=Curr. Opin. Plant Biol;  citation_year=2014;  citation_volume=19;  citation_pages=91-8; " /><meta name="citation_reference" content="citation_title=Preservation of genetic and regulatory robustness in ancient gene duplicates of Saccharomyces cerevisiae,;  citation_journal_title=Genome Res;  citation_year=2014;  citation_volume=24;  citation_pages=1830-41; " /><meta name="citation_reference" content="citation_title=The origins of mutational robustness,;  citation_journal_title=Trends Genet;  citation_year=2015;  citation_volume=31;  citation_pages=373-81; " /><meta name="citation_reference" content="citation_title=Maintenance and loss of duplicated genes by dosage subfunctionalization,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2015;  citation_volume=32;  citation_pages=2141-8; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs,;  citation_journal_title=Nucleic Acids Res;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=The Yeast Gene Order Browser: combining curated homology and syntenic context reveals gene fate in polyploid species,;  citation_journal_title=Genome Res;  citation_year=2005;  citation_volume=15;  citation_pages=1456-61; " /><meta name="citation_reference" content="citation_title=RobiNA: a user-friendly, integrated software solution for RNA-Seq-based transcriptomics,;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=W622-7; " /><meta name="citation_reference" content="citation_title=edgeR: a Bioconductor package for differential expression analysis of digital gene expression data,;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=139-40; " /><meta name="citation_reference" content="citation_title=Differential expression analysis for sequence count data,;  citation_journal_title=Genome Biol;  citation_year=2010;  citation_volume=11;  citation_pages=R106" /><meta name="citation_reference" content="citation_title=The control of the false discovery rate in multiple testing under dependency,;  citation_journal_title=Ann. Stat;  citation_year=2001;  citation_volume=29;  citation_pages=24" /><meta name="citation_reference" content="citation_title=Differences in environmental stress response among yeasts is consistent with species-specific lifestyles,;  citation_journal_title=Mol. Biol. Cell;  citation_year=2016;  citation_volume=27;  citation_pages=1694-705; " /><meta name="citation_reference" content="citation_title=Defining the transcriptomic landscape of Candida glabrata by RNA-Seq,;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=1392-406; " /><meta name="citation_reference" content="citation_title=Yeast genome evolution in the post-genome era,;  citation_journal_title=Curr. Opin. Microbiol;  citation_year=1999;  citation_volume=2;  citation_pages=548-54; " /><meta name="citation_reference" content="citation_title=Global trends of whole-genome duplications revealed by the ciliate Paramecium tetraurelia,;  citation_journal_title=Nature;  citation_year=2006;  citation_volume=444;  citation_pages=171-8; " /><meta name="citation_reference" content="citation_title=The relationship among gene expression, the evolution of gene dosage, and the rate of protein evolution,;  citation_journal_title=PLoS Genet;  citation_year=2010;  citation_volume=6;  citation_pages=e1000944" /><meta name="citation_reference" content="citation_title=Insights into three whole-genome duplications gleaned from the Paramecium caudatum genome sequence,;  citation_journal_title=Genetics;  citation_year=2014;  citation_volume=197;  citation_pages=1417-28; " /><meta name="citation_reference" content="citation_title=Differential retention and divergent resolution of duplicate genes following whole-genome duplication,;  citation_journal_title=Genome Res;  citation_year=2014;  citation_volume=24;  citation_pages=1665-75; " /><meta name="citation_reference" content="citation_title=Genetic influences on translation in yeast,;  citation_journal_title=PLoS Genet;  citation_year=2014;  citation_volume=10;  citation_pages=e1004692" /><meta name="citation_reference" content="citation_title=Differential retention of metabolic genes following whole-genome duplication,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2009;  citation_volume=26;  citation_pages=1067-72; " /><meta name="citation_reference" content="citation_title=Dosage sensitivity and the evolution of gene families in yeast,;  citation_journal_title=Nature;  citation_year=2003;  citation_volume=424;  citation_pages=194-7; " /><meta name="citation_reference" content="citation_title=Maintenance of duplicate genes and their functional redundancy by reduced expression,;  citation_journal_title=Trends Genet;  citation_year=2010;  citation_volume=26;  citation_pages=425-30; " /><meta name="citation_reference" content="citation_title=Gene balance hypothesis: connecting issues of dosage sensitivity across biological disciplines,;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2012;  citation_volume=109;  citation_pages=14746-53; " /><meta name="citation_reference" content="citation_title=Up-to-date catalogues of yeast protein complexes,;  citation_journal_title=Nucleic Acids Res;  citation_year=2009;  citation_volume=37;  citation_pages=825-31; " /><meta name="citation_reference" content="citation_title=Multiple rounds of speciation associated with reciprocal gene loss in polyploid yeasts,;  citation_journal_title=Nature;  citation_year=2006;  citation_volume=440;  citation_pages=341-5; " /><meta name="citation_reference" content="citation_title=Genetic properties influencing the evolvability of gene expression,;  citation_journal_title=Science;  citation_year=2007;  citation_volume=317;  citation_pages=118-21; " /><meta name="citation_reference" content="citation_title=Conflict between noise and plasticity in yeast,;  citation_journal_title=PLoS Genet;  citation_year=2010;  citation_volume=6;  citation_pages=e1001185" /><meta name="citation_reference" content="citation_title=Glycerol stress in Saccharomyces cerevisiae: cellular responses and evolved adaptations;  citation_journal_title=Environ. Microbiol;  citation_year=2017;  citation_volume=19;  citation_issue=3;  citation_pages=990-1007; " /><meta name="citation_reference" content="citation_title=The phenotypic plasticity of duplicated genes in Saccharomyces cerevisiae and the origin of adaptations,;  citation_journal_title=G3 (Bethesda);  citation_year=2017;  citation_volume=7;  citation_pages=63-75; " /><meta name="citation_reference" content="citation_title=Glycerol stress in Saccharomyces cerevisiae: cellular responses and evolved adaptations,;  citation_journal_title=Environ. Microbiol;  citation_year=2017;  citation_volume=19;  citation_pages=990-1007; " /><meta name="citation_reference" content="citation_title=Phenotypic consequences of promoter-mediated transcriptional noise,;  citation_journal_title=Mol. Cell;  citation_year=2006;  citation_volume=24;  citation_pages=853-65; " /><meta name="citation_reference" content="citation_title=Noise in gene expression: origins, consequences, and control,;  citation_journal_title=Science;  citation_year=2005;  citation_volume=309;  citation_pages=2010-3; " /><meta name="citation_reference" content="citation_title=Single-cell proteomic analysis of S. cerevisiae reveals the architecture of biological noise,;  citation_journal_title=Nature;  citation_year=2006;  citation_volume=441;  citation_pages=840-6; " /><meta name="citation_reference" content="citation_title=A genetic signature of interspecies variations in gene expression,;  citation_journal_title=Nat. Genet;  citation_year=2006;  citation_volume=38;  citation_pages=830-4; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/6/559/22295679/dsx025.pdf" /><meta name="description" content="Abstract.  Gene duplication is an important source of novelties and genome complexity. What genes are preserved as duplicated through long evolutionary times ca" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/6/559/3868561" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/6/559/3868561" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/6/559/3868561");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx025';
    </script>
<title>
	Expression properties exhibit correlated patterns with the fate of duplicated genes, their divergence, and transcriptional plasticity in Saccharomycotina | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./3868561" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BfGLATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTWF0dGVuYmVyZ2VyLCBGbG9yaWFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBBYmlvdGljIFN0cmVzcywgSW5zdGl0dXRvIGRlIEJpb2xvZ8OtYSBNb2xlY3VsYXIgeSBDZWx1bGFyIGRlIFBsYW50YXMsIENvbnNlam8gU3VwZXJpb3IgZGUgSW52ZXN0aWdhY2lvbmVzIENpZW50w61maWNhcy1Vbml2ZXJzaWRhZCBQb2xpdMOpY25pY2EgZGUgVmFsZW5jaWEsIFZhbGVuY2lhIDQ2MDIyLCBTcGFpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlN5c3RlbXMgQmlvbG9neSBvZiBNb2xlY3VsYXIgSW50ZXJhY3Rpb25zIGFuZCBSZWd1bGF0aW9uIERlcGFydG1lbnQsIEluc3RpdHV0ZSBmb3IgSW50ZWdyYXRpdmUgU3lzdGVtcyBCaW9sb2d5IChJMlN5c0JpbyksIENvbnNlam8gU3VwZXJpb3IgZGUgSW52ZXN0aWdhY2lvbmVzLCBDaWVudMOtZmljYXMtVW5pdmVyc2l0YXQgZGUgVmFsZW5jaWEgKENTSUMtVVYpLCBWYWxlbmNpYSA0Njk4MCwgU3BhaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYWJhdGVyLU11w7FveiwgQmVhdHJpeiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQWJpb3RpYyBTdHJlc3MsIEluc3RpdHV0byBkZSBCaW9sb2fDrWEgTW9sZWN1bGFyIHkgQ2VsdWxhciBkZSBQbGFudGFzLCBDb25zZWpvIFN1cGVyaW9yIGRlIEludmVzdGlnYWNpb25lcyBDaWVudMOtZmljYXMtVW5pdmVyc2lkYWQgUG9saXTDqWNuaWNhIGRlIFZhbGVuY2lhLCBWYWxlbmNpYSA0NjAyMiwgU3BhaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTeXN0ZW1zIEJpb2xvZ3kgb2YgTW9sZWN1bGFyIEludGVyYWN0aW9ucyBhbmQgUmVndWxhdGlvbiBEZXBhcnRtZW50LCBJbnN0aXR1dGUgZm9yIEludGVncmF0aXZlIFN5c3RlbXMgQmlvbG9neSAoSTJTeXNCaW8pLCBDb25zZWpvIFN1cGVyaW9yIGRlIEludmVzdGlnYWNpb25lcywgQ2llbnTDrWZpY2FzLVVuaXZlcnNpdGF0IGRlIFZhbGVuY2lhIChDU0lDLVVWKSwgVmFsZW5jaWEgNDY5ODAsIFNwYWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBHZW5ldGljcywgU211cmZpdCBJbnN0aXR1dGUgb2YgR2VuZXRpY3MsIFVuaXZlcnNpdHkgb2YgRHVibGluLCBUcmluaXR5IENvbGxlZ2UsIER1YmxpbiwgSXJlbGFuZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRvZnQsIENocmlzdGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgR2VuZXRpY3MsIFVuaXZlcnNpdHkgb2YgVmFsZW5jaWEsIEJ1cmphc290LCBWYWxlbmNpYSA0NjEwMCwgU3BhaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb3RlY2hub2xvZ3ksIEluc3RpdHV0byBkZSBBZ3JvcXXDrW1pY2EgeSBUZWNub2xvZ8OtYSBkZSBsb3MgQWxpbWVudG9zLCBDb25zZWpvIFN1cGVyaW9yIGRlIEludmVzdGlnYWNpb25lcyBDaWVudMOtZmljYXMgKENTSUMpLCBCdXJqYXNvdCwgVmFsZW5jaWEsIFNwYWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2FibG9rLCBHYXVyYXYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJQbGFudCBGdW5jdGlvbmFsIEJpb2xvZ3kgYW5kIENsaW1hdGUgQ2hhbmdlIENsdXN0ZXIgKEMzKSwgVW5pdmVyc2l0eSBvZiBUZWNobm9sb2d5IFN5ZG5leSwgU3lkbmV5LCBOU1cgMjAwNywgQXVzdHJhbGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRmFyZXMsIE1hcmlvIEEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFiaW90aWMgU3RyZXNzLCBJbnN0aXR1dG8gZGUgQmlvbG9nw61hIE1vbGVjdWxhciB5IENlbHVsYXIgZGUgUGxhbnRhcywgQ29uc2VqbyBTdXBlcmlvciBkZSBJbnZlc3RpZ2FjaW9uZXMgQ2llbnTDrWZpY2FzLVVuaXZlcnNpZGFkIFBvbGl0w6ljbmljYSBkZSBWYWxlbmNpYSwgVmFsZW5jaWEgNDYwMjIsIFNwYWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU3lzdGVtcyBCaW9sb2d5IG9mIE1vbGVjdWxhciBJbnRlcmFjdGlvbnMgYW5kIFJlZ3VsYXRpb24gRGVwYXJ0bWVudCwgSW5zdGl0dXRlIGZvciBJbnRlZ3JhdGl2ZSBTeXN0ZW1zIEJpb2xvZ3kgKEkyU3lzQmlvKSwgQ29uc2VqbyBTdXBlcmlvciBkZSBJbnZlc3RpZ2FjaW9uZXMsIENpZW50w61maWNhcy1Vbml2ZXJzaXRhdCBkZSBWYWxlbmNpYSAoQ1NJQy1VViksIFZhbGVuY2lhIDQ2OTgwLCBTcGFpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgR2VuZXRpY3MsIFNtdXJmaXQgSW5zdGl0dXRlIG9mIEdlbmV0aWNzLCBVbml2ZXJzaXR5IG9mIER1YmxpbiwgVHJpbml0eSBDb2xsZWdlLCBEdWJsaW4sIElyZWxhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkV4cHJlc3Npb24gcHJvcGVydGllcyBleGhpYml0IGNvcnJlbGF0ZWQgcGF0dGVybnMgd2l0aCB0aGUgZmF0ZSBvZiBkdXBsaWNhdGVkIGdlbmVzLCB0aGVpciBkaXZlcmdlbmNlLCBhbmQgdHJhbnNjcmlwdGlvbmFsIHBsYXN0aWNpdHkgaW4gU2FjY2hhcm9teWNvdGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjU1OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNTcwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN4MDI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTcvMTIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXIgVmVybGFnLCBOZXcgWW9yazsgIGNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBieSBHZW5lIER1cGxpY2F0aW9uOyAgY2l0YXRpb25feWVhcj0xOTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIGR1cGxpY2F0aW9uIGFuZCB0aGUgdW5pcXVlbmVzcyBvZiB2ZXJ0ZWJyYXRlIGdlbm9tZXMgY2lyY2EgMTk3MOKAkzE5OTksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TZW1pbi4gQ2VsbCBEZXYuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz01MTctMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBldm9sdXRpb25hcnkgZmF0ZSBhbmQgY29uc2VxdWVuY2VzIG9mIGR1cGxpY2F0ZSBnZW5lcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MjkwOyAgY2l0YXRpb25fcGFnZXM9MTE1MS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb2x5cGxvaWQgaW5jaWRlbmNlIGFuZCBldm9sdXRpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9NDAxLTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgZHluYW1pY3MgYW5kIGZ1bmN0aW9uYWwgc3BlY2lhbGl6YXRpb24gb2YgcGxhbnQgcGFyYWxvZ3MgZm9ybWVkIGJ5IHdob2xlIGFuZCBzbWFsbC1zY2FsZSBnZW5vbWUgZHVwbGljYXRpb25zLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9MzU0MS01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2lkZXNwcmVhZCBnZW5vbWUgZHVwbGljYXRpb25zIHRocm91Z2hvdXQgdGhlIGhpc3Rvcnkgb2YgZmxvd2VyaW5nIHBsYW50cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz03MzgtNDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBhcm1zIHJhY2UgaXMgYW5jaWVudCBoaXN0b3J5IGluIEFyYWJpZG9wc2lzLCB0aGUgd2lsZGZsb3dlciw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9NTE2LTI3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcm9sZSBvZiBsaW5lYWdlLXNwZWNpZmljIGdlbmUgZmFtaWx5IGV4cGFuc2lvbiBpbiB0aGUgZXZvbHV0aW9uIG9mIGV1a2FyeW90ZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTA0OC01OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIGV2b2x1dGlvbiBpbiBwb2x5cGxvaWRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9MjI1LTQ5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb2x5cGxvaWR5IGFuZCBhbmdpb3NwZXJtIGRpdmVyc2lmaWNhdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFtLiBKLiBCb3Q7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9OTY7ICBjaXRhdGlvbl9wYWdlcz0zMzYtNDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXB1dGF0aW9uYWwgYXBwcm9hY2hlcyB0byB1bnZlaWxpbmcgYW5jaWVudCBnZW5vbWUgZHVwbGljYXRpb25zLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz03NTItNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeWxvZ2VuZXRpYyB0aW1pbmcgb2YgdGhlIGZpc2gtc3BlY2lmaWMgZ2Vub21lIGR1cGxpY2F0aW9uIGNvcnJlbGF0ZXMgd2l0aCB0aGUgZGl2ZXJzaWZpY2F0aW9uIG9mIHRlbGVvc3QgZmlzaCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIE1vbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01OTsgIGNpdGF0aW9uX3BhZ2VzPTE5MC0yMDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtb2xlY3VsYXIgZXZvbHV0aW9uIG9mIHRoZSB2ZXJ0ZWJyYXRlIGJlaGF2aW91cmFsIHJlcGVydG9pcmUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QaGlsb3MuIFRyYW5zLiBSLiBTb2MuIExvbmQuIEI6IEJpb2wuIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNzE7ICBjaXRhdGlvbl9wYWdlcz0yMDE1MDA1MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgZHVwbGljYXRpb25zIGFuZCB0aGUgZWFybHkgZXZvbHV0aW9uIG9mIG5ldXJhbCBjcmVzdCBkZXZlbG9wbWVudCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNlbWluLiBDZWxsIERldi4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTk1LTEwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9seXBsb2lkeSBhbmQgdGhlIGV2b2x1dGlvbiBvZiBjb21wbGV4IHRyYWl0cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUludC4gSi4gRXZvbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yMDEyOyAgY2l0YXRpb25fcGFnZXM9MjkyMDY4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5jcmVhc2VkIGdseWNvbHl0aWMgZmx1eCBhcyBhbiBvdXRjb21lIG9mIHdob2xlLWdlbm9tZSBkdXBsaWNhdGlvbiBpbiB5ZWFzdCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gU3lzdC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MTI5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGV2aWRlbmNlIGZvciBhbiBhbmNpZW50IGR1cGxpY2F0aW9uIG9mIHRoZSBlbnRpcmUgeWVhc3QgZ2Vub21lLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTM4NzsgIGNpdGF0aW9uX3BhZ2VzPTcwOC0xMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJvbGVzIG9mIHdob2xlLWdlbm9tZSBhbmQgc21hbGwtc2NhbGUgZHVwbGljYXRpb25zIGluIHRoZSBmdW5jdGlvbmFsIHNwZWNpYWxpemF0aW9uIG9mIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZSBnZW5lcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAzMTc2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2lkZXNwcmVhZCBwYWxlb3BvbHlwbG9pZHkgaW4gbW9kZWwgcGxhbnQgc3BlY2llcyBpbmZlcnJlZCBmcm9tIGFnZSBkaXN0cmlidXRpb25zIG9mIGR1cGxpY2F0ZSBnZW5lcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xNjY3LTc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5jdGlvbmFsIGRpdmVyZ2VuY2Ugb2YgZHVwbGljYXRlZCBnZW5lcyBmb3JtZWQgYnkgcG9seXBsb2lkeSBkdXJpbmcgQXJhYmlkb3BzaXMgZXZvbHV0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTE2NzktOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVR1cm5pbmcgYSBob2JieSBpbnRvIGEgam9iOiBob3cgZHVwbGljYXRlZCBnZW5lcyBmaW5kIG5ldyBmdW5jdGlvbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTkzOC01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmF0ZSBhc3ltbWV0cnkgYWZ0ZXIgZ2Vub21lIGR1cGxpY2F0aW9uIGNhdXNlcyBzdWJzdGFudGlhbCBsb25nLWJyYW5jaCBhdHRyYWN0aW9uIGFydGlmYWN0cyBpbiB0aGUgcGh5bG9nZW55IG9mIFNhY2NoYXJvbXljZXMgc3BlY2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MjQ1LTUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGJ1cnN0IG9mIHByb3RlaW4gc2VxdWVuY2UgZXZvbHV0aW9uIGFuZCBhIHByb2xvbmdlZCBwZXJpb2Qgb2YgYXN5bW1ldHJpYyBldm9sdXRpb24gZm9sbG93IGdlbmUgZHVwbGljYXRpb24gaW4geWVhc3QsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MTM3LTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaWFzIGluIHBsYW50IGdlbmUgY29udGVudCBmb2xsb3dpbmcgZGlmZmVyZW50IHNvcnRzIG9mIGR1cGxpY2F0aW9uOiB0YW5kZW0sIHdob2xlLWdlbm9tZSwgc2VnbWVudGFsLCBvciBieSB0cmFuc3Bvc2l0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5udS4gUmV2LiBQbGFudCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTYwOyAgY2l0YXRpb25fcGFnZXM9NDMzLTUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Eb3NhZ2UsIGR1cGxpY2F0aW9uLCBhbmQgZGlwbG9pZGl6YXRpb246IGNsYXJpZnlpbmcgdGhlIGludGVycGxheSBvZiBtdWx0aXBsZSBtb2RlbHMgZm9yIGR1cGxpY2F0ZSBnZW5lIGV2b2x1dGlvbiBvdmVyIHRpbWUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBQbGFudCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9OTEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlc2VydmF0aW9uIG9mIGdlbmV0aWMgYW5kIHJlZ3VsYXRvcnkgcm9idXN0bmVzcyBpbiBhbmNpZW50IGdlbmUgZHVwbGljYXRlcyBvZiBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MTgzMC00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG9yaWdpbnMgb2YgbXV0YXRpb25hbCByb2J1c3RuZXNzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MzczLTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYWludGVuYW5jZSBhbmQgbG9zcyBvZiBkdXBsaWNhdGVkIGdlbmVzIGJ5IGRvc2FnZSBzdWJmdW5jdGlvbmFsaXphdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTIxNDEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2FwcGVkIEJMQVNUIGFuZCBQU0ktQkxBU1Q6IGEgbmV3IGdlbmVyYXRpb24gb2YgcHJvdGVpbiBkYXRhYmFzZSBzZWFyY2ggcHJvZ3JhbXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTMzODktNDAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgWWVhc3QgR2VuZSBPcmRlciBCcm93c2VyOiBjb21iaW5pbmcgY3VyYXRlZCBob21vbG9neSBhbmQgc3ludGVuaWMgY29udGV4dCByZXZlYWxzIGdlbmUgZmF0ZSBpbiBwb2x5cGxvaWQgc3BlY2llcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xNDU2LTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Sb2JpTkE6IGEgdXNlci1mcmllbmRseSwgaW50ZWdyYXRlZCBzb2Z0d2FyZSBzb2x1dGlvbiBmb3IgUk5BLVNlcS1iYXNlZCB0cmFuc2NyaXB0b21pY3MsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPVc2MjItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9ZWRnZVI6IGEgQmlvY29uZHVjdG9yIHBhY2thZ2UgZm9yIGRpZmZlcmVudGlhbCBleHByZXNzaW9uIGFuYWx5c2lzIG9mIGRpZ2l0YWwgZ2VuZSBleHByZXNzaW9uIGRhdGEsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTEzOS00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlmZmVyZW50aWFsIGV4cHJlc3Npb24gYW5hbHlzaXMgZm9yIHNlcXVlbmNlIGNvdW50IGRhdGEsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPVIxMDYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY29udHJvbCBvZiB0aGUgZmFsc2UgZGlzY292ZXJ5IHJhdGUgaW4gbXVsdGlwbGUgdGVzdGluZyB1bmRlciBkZXBlbmRlbmN5LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5uLiBTdGF0OyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9MjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWZmZXJlbmNlcyBpbiBlbnZpcm9ubWVudGFsIHN0cmVzcyByZXNwb25zZSBhbW9uZyB5ZWFzdHMgaXMgY29uc2lzdGVudCB3aXRoIHNwZWNpZXMtc3BlY2lmaWMgbGlmZXN0eWxlcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTE2OTQtNzA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZWZpbmluZyB0aGUgdHJhbnNjcmlwdG9taWMgbGFuZHNjYXBlIG9mIENhbmRpZGEgZ2xhYnJhdGEgYnkgUk5BLVNlcSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9MTM5Mi00MDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVllYXN0IGdlbm9tZSBldm9sdXRpb24gaW4gdGhlIHBvc3QtZ2Vub21lIGVyYSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIE9waW4uIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9NTQ4LTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HbG9iYWwgdHJlbmRzIG9mIHdob2xlLWdlbm9tZSBkdXBsaWNhdGlvbnMgcmV2ZWFsZWQgYnkgdGhlIGNpbGlhdGUgUGFyYW1lY2l1bSB0ZXRyYXVyZWxpYSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT00NDQ7ICBjaXRhdGlvbl9wYWdlcz0xNzEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJlbGF0aW9uc2hpcCBhbW9uZyBnZW5lIGV4cHJlc3Npb24sIHRoZSBldm9sdXRpb24gb2YgZ2VuZSBkb3NhZ2UsIGFuZCB0aGUgcmF0ZSBvZiBwcm90ZWluIGV2b2x1dGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAwOTQ0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5zaWdodHMgaW50byB0aHJlZSB3aG9sZS1nZW5vbWUgZHVwbGljYXRpb25zIGdsZWFuZWQgZnJvbSB0aGUgUGFyYW1lY2l1bSBjYXVkYXR1bSBnZW5vbWUgc2VxdWVuY2UsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xOTc7ICBjaXRhdGlvbl9wYWdlcz0xNDE3LTI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWZmZXJlbnRpYWwgcmV0ZW50aW9uIGFuZCBkaXZlcmdlbnQgcmVzb2x1dGlvbiBvZiBkdXBsaWNhdGUgZ2VuZXMgZm9sbG93aW5nIHdob2xlLWdlbm9tZSBkdXBsaWNhdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xNjY1LTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGluZmx1ZW5jZXMgb24gdHJhbnNsYXRpb24gaW4geWVhc3QsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9ZTEwMDQ2OTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWZmZXJlbnRpYWwgcmV0ZW50aW9uIG9mIG1ldGFib2xpYyBnZW5lcyBmb2xsb3dpbmcgd2hvbGUtZ2Vub21lIGR1cGxpY2F0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9MTA2Ny03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RG9zYWdlIHNlbnNpdGl2aXR5IGFuZCB0aGUgZXZvbHV0aW9uIG9mIGdlbmUgZmFtaWxpZXMgaW4geWVhc3QsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDI0OyAgY2l0YXRpb25fcGFnZXM9MTk0LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1haW50ZW5hbmNlIG9mIGR1cGxpY2F0ZSBnZW5lcyBhbmQgdGhlaXIgZnVuY3Rpb25hbCByZWR1bmRhbmN5IGJ5IHJlZHVjZWQgZXhwcmVzc2lvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTQyNS0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBiYWxhbmNlIGh5cG90aGVzaXM6IGNvbm5lY3RpbmcgaXNzdWVzIG9mIGRvc2FnZSBzZW5zaXRpdml0eSBhY3Jvc3MgYmlvbG9naWNhbCBkaXNjaXBsaW5lcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEwOTsgIGNpdGF0aW9uX3BhZ2VzPTE0NzQ2LTUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VcC10by1kYXRlIGNhdGFsb2d1ZXMgb2YgeWVhc3QgcHJvdGVpbiBjb21wbGV4ZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zNzsgIGNpdGF0aW9uX3BhZ2VzPTgyNS0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXVsdGlwbGUgcm91bmRzIG9mIHNwZWNpYXRpb24gYXNzb2NpYXRlZCB3aXRoIHJlY2lwcm9jYWwgZ2VuZSBsb3NzIGluIHBvbHlwbG9pZCB5ZWFzdHMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NDQwOyAgY2l0YXRpb25fcGFnZXM9MzQxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgcHJvcGVydGllcyBpbmZsdWVuY2luZyB0aGUgZXZvbHZhYmlsaXR5IG9mIGdlbmUgZXhwcmVzc2lvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzE3OyAgY2l0YXRpb25fcGFnZXM9MTE4LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25mbGljdCBiZXR3ZWVuIG5vaXNlIGFuZCBwbGFzdGljaXR5IGluIHllYXN0LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDExODUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HbHljZXJvbCBzdHJlc3MgaW4gU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlOiBjZWxsdWxhciByZXNwb25zZXMgYW5kIGV2b2x2ZWQgYWRhcHRhdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVudmlyb24uIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX2lzc3VlPTM7ICBjaXRhdGlvbl9wYWdlcz05OTAtMTAwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHBoZW5vdHlwaWMgcGxhc3RpY2l0eSBvZiBkdXBsaWNhdGVkIGdlbmVzIGluIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZSBhbmQgdGhlIG9yaWdpbiBvZiBhZGFwdGF0aW9ucyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUczIChCZXRoZXNkYSk7ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTYzLTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HbHljZXJvbCBzdHJlc3MgaW4gU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlOiBjZWxsdWxhciByZXNwb25zZXMgYW5kIGV2b2x2ZWQgYWRhcHRhdGlvbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FbnZpcm9uLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz05OTAtMTAwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGhlbm90eXBpYyBjb25zZXF1ZW5jZXMgb2YgcHJvbW90ZXItbWVkaWF0ZWQgdHJhbnNjcmlwdGlvbmFsIG5vaXNlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9ODUzLTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob2lzZSBpbiBnZW5lIGV4cHJlc3Npb246IG9yaWdpbnMsIGNvbnNlcXVlbmNlcywgYW5kIGNvbnRyb2wsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMwOTsgIGNpdGF0aW9uX3BhZ2VzPTIwMTAtMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2luZ2xlLWNlbGwgcHJvdGVvbWljIGFuYWx5c2lzIG9mIFMuIGNlcmV2aXNpYWUgcmV2ZWFscyB0aGUgYXJjaGl0ZWN0dXJlIG9mIGJpb2xvZ2ljYWwgbm9pc2UsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NDQxOyAgY2l0YXRpb25fcGFnZXM9ODQwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2VuZXRpYyBzaWduYXR1cmUgb2YgaW50ZXJzcGVjaWVzIHZhcmlhdGlvbnMgaW4gZ2VuZSBleHByZXNzaW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zODsgIGNpdGF0aW9uX3BhZ2VzPTgzMC00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjQvNi81NTkvMjIyOTU2NzkvZHN4MDI1LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBHZW5lIGR1cGxpY2F0aW9uIGlzIGFuIGltcG9ydGFudCBzb3VyY2Ugb2Ygbm92ZWx0aWVzIGFuZCBnZW5vbWUgY29tcGxleGl0eS4gV2hhdCBnZW5lcyBhcmUgcHJlc2VydmVkIGFzIGR1cGxpY2F0ZWQgdGhyb3VnaCBsb25nIGV2b2x1dGlvbmFyeSB0aW1lcyBjYSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9Mzg2ODU2MSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQZjpcGwZdx26MsG2EFqxbAJ0j8NA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f559%2f3868561" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f559%2f3868561" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=103821" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/6/2/m_dnares_24_6cover.png?Expires=2147483647&amp;Signature=z4yCzT-6enr7Fof9QqkUJvtPQAXDRV5XYB7R7L2HBbQZ2iaySlHxBG~hDePAa-rY6Gn-PH9W8mIlJwAyG9R2RE6Vd3RTDct~I7biyNGgXk9CP54tGn91x8VYIMO24kwuQBqahsZQMv3NxRiQKhbGI0yEDanozvBgG~-9FpaC12JFLE6A4BT6rBzjsJLh9UJqvGv0F2hWW2jLvdITi~5PoATpRIsbIK-S1wcvqp49OSN~oRepZtr6iz~107ho8gZAxoaG8nb4DvXKrvfnCuhhmbRD41NP3K2zFyztHicWtTYb1xunX52RqodNq-M3zVvUit2rSw5-mRSvm-jXb-p6nw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/6">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="104086069">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086069">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086071">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086071">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086077">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086077">2. Material and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086090">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086090">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086119">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086119">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086124">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086124">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086132">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086132">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/6/549/3865558">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/6/571/3868562">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Expression properties exhibit correlated patterns with the fate of duplicated genes, their divergence, and transcriptional plasticity in <em>Saccharomycotina</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Florian Mattenberger</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Abiotic Stress, Instituto de Biología Molecular y Celular de Plantas, Consejo Superior de Investigaciones Científicas-Universidad Politécnica de Valencia, Valencia 46022, Spain</div><div class="aff">Systems Biology of Molecular Interactions and Regulation Department, Institute for Integrative Systems Biology (I2SysBio), Consejo Superior de Investigaciones, Científicas-Universitat de Valencia (CSIC-UV), Valencia 46980, Spain</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Florian+Mattenberger">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mattenberger F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mattenberger F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Florian Mattenberger</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Beatriz Sabater-Mu&#241;oz</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Abiotic Stress, Instituto de Biología Molecular y Celular de Plantas, Consejo Superior de Investigaciones Científicas-Universidad Politécnica de Valencia, Valencia 46022, Spain</div><div class="aff">Systems Biology of Molecular Interactions and Regulation Department, Institute for Integrative Systems Biology (I2SysBio), Consejo Superior de Investigaciones, Científicas-Universitat de Valencia (CSIC-UV), Valencia 46980, Spain</div><div class="aff">Department of Genetics, Smurfit Institute of Genetics, University of Dublin, Trinity College, Dublin, Ireland</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx025-cor1" data-open="dsx025-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +34 96 3879934. Email: <a href="mailto:mfares@ibmcp.upv.es" target="_blank">mfares@ibmcp.upv.es</a>, <a href="mailto:faresm@tcd.ie" target="_blank">faresm@tcd.ie</a>; <a href="mailto:sabaterb.tcd@gmail.com" target="_blank">sabaterb.tcd@gmail.com</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Beatriz+Sabater-Mu%c3%b1oz">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sabater-Mu&#241;oz B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sabater-Mu&#241;oz B&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Beatriz Sabater-Muñoz</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Christina Toft</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Genetics, University of Valencia, Burjasot, Valencia 46100, Spain</div><div class="aff">Department of Biotechnology, Instituto de Agroquímica y Tecnología de los Alimentos, Consejo Superior de Investigaciones Científicas (CSIC), Burjasot, Valencia, Spain</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Christina+Toft">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Toft C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Toft C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Christina Toft</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Gaurav Sablok</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Plant Functional Biology and Climate Change Cluster (C3), University of Technology Sydney, Sydney, NSW 2007, Australia</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Gaurav+Sablok">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sablok G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sablok G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Gaurav Sablok</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mario A Fares</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Abiotic Stress, Instituto de Biología Molecular y Celular de Plantas, Consejo Superior de Investigaciones Científicas-Universidad Politécnica de Valencia, Valencia 46022, Spain</div><div class="aff">Systems Biology of Molecular Interactions and Regulation Department, Institute for Integrative Systems Biology (I2SysBio), Consejo Superior de Investigaciones, Científicas-Universitat de Valencia (CSIC-UV), Valencia 46980, Spain</div><div class="aff">Department of Genetics, Smurfit Institute of Genetics, University of Dublin, Trinity College, Dublin, Ireland</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx025-cor1" data-open="dsx025-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +34 96 3879934. Email: <a href="mailto:mfares@ibmcp.upv.es" target="_blank">mfares@ibmcp.upv.es</a>, <a href="mailto:faresm@tcd.ie" target="_blank">faresm@tcd.ie</a>; <a href="mailto:sabaterb.tcd@gmail.com" target="_blank">sabaterb.tcd@gmail.com</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mario+A+Fares">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fares M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fares M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mario A Fares</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx025-FM1"><div class="fn"><div class="footnote-content">Florian Mattenberger and Beatriz Sabater-Muñoz contributed equally to this work and should be considered co-first author.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx025-FM2"><div class="fn"><div class="footnote-content">Edited by Dr. Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 6, 1 December 2017, Pages 559–570, <a href='https://doi.org/10.1093/dnares/dsx025'>https://doi.org/10.1093/dnares/dsx025</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">15 June 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">10 February 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">11 May 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="3868561" href="/dnaresearch/article-pdf/24/6/559/22295679/dsx025.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Florian Mattenberger, Beatriz Sabater-Muñoz, Christina Toft, Gaurav Sablok, Mario A Fares;  Expression properties exhibit correlated patterns with the fate of duplicated genes, their divergence, and transcriptional plasticity in <em>Saccharomycotina</em>, <em>DNA Research</em>, Volume 24, Issue 6, 1 December 2017, Pages 559–570, <a href='https://doi.org/10.1093/dnares/dsx025'>https://doi.org/10.1093/dnares/dsx025</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/3868561?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/3868561?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/3868561?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/3868561?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/3868561?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/3868561?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/3868561?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Expression%20properties%20exhibit%20correlated%20patterns%20with%20the%20fate%20of%20duplicated%20genes%2C%20their%20divergence%2C%20and%20transcriptional%20plasticity%20in%20%26lt%3Bem%26gt%3BSaccharomycotina%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2017-06-15&amp;volumeNum=24&amp;issueNum=6&amp;author=Mattenberger%2C%20Florian%3B%20Sabater-Mu%C3%B1oz%2C%20Beatriz&amp;startPage=559&amp;endPage=570&amp;contentId=10.1093%2Fdnares%2Fdsx025&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f559%2f3868561" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=104086069 id="104086069" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Gene duplication is an important source of novelties and genome complexity. What genes are preserved as duplicated through long evolutionary times can shape the evolution of innovations. Identifying factors that influence gene duplicability is therefore an important aim in evolutionary biology. Here, we show that in the yeast <em>Saccharomyces cerevisiae</em> the levels of gene expression correlate with gene duplicability, its divergence, and transcriptional plasticity. Genes that were highly expressed before duplication are more likely to be preserved as duplicates for longer evolutionary times and wider phylogenetic ranges than genes that were lowly expressed. Duplicates with higher expression levels exhibit greater divergence between their gene copies. Duplicates that exhibit higher expression divergence are those enriched for TATA-containing promoters. These duplicates also show transcriptional plasticity, which seems to be involved in the origin of adaptations to environmental stresses in yeast. While the expression properties of genes strongly affect their duplicability, divergence and transcriptional plasticity are enhanced after gene duplication. We conclude that highly expressed genes are more likely to be preserved as duplicates due to their promoter architectures, their greater tolerance to expression noise, and their ability to reduce the noise-plasticity conflict.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene expression&quot;">gene expression</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene duplication&quot;">gene duplication</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcriptional plasticity&quot;">transcriptional plasticity</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="duplicability">duplicability</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Saccharomyces cerevisiae&quot;"><em>Saccharomyces cerevisiae</em></a></div>                            <h2 scrollto-destination=104086071 id="104086071" class="section-title" >1. Introduction</h2>
<p>Gene duplication is believed to be a rich source of novel functions and adaptations.<a reveal-id="dsx025-B1 dsx025-B2 dsx025-B3" data-open="dsx025-B1 dsx025-B2 dsx025-B3" class="link link-ref link-reveal xref-bibr"><sup>1–3</sup></a> This belief is supported by evidence coming from innovations following gene duplications in yeast, plants and animals. Indeed, protein families expanded after whole genome and small-scale duplications yielding an unprecedented morphological diversity in plants.<a reveal-id="dsx025-B4 dsx025-B5 dsx025-B6 dsx025-B7 dsx025-B8 dsx025-B9 dsx025-B10 dsx025-B11" data-open="dsx025-B4 dsx025-B5 dsx025-B6 dsx025-B7 dsx025-B8 dsx025-B9 dsx025-B10 dsx025-B11" class="link link-ref link-reveal xref-bibr"><sup>4–11</sup></a> Other major innovations in animals have also been achieved through gene duplication,<a reveal-id="dsx025-B12" data-open="dsx025-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> including increased synapse and behaviour complexity<a reveal-id="dsx025-B13" data-open="dsx025-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> and the neural crest formation and plasticity in vertebrates.<a reveal-id="dsx025-B14" data-open="dsx025-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> In yeast, gene duplication has contributed to metabolic innovation through the alteration of regulatory and transcriptional networks<a reveal-id="dsx025-B15" data-open="dsx025-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> or the increased glycolytic fluxes.<a reveal-id="dsx025-B16" data-open="dsx025-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> However, it remains unclear why certain duplicates have been preferred over others to persist in the genomes and be the source of innovations.</p><p>Since duplication is immediately followed by relaxed selection constraints on one or the two gene copies, the survival time of each gene copy is a limiting factor in the determination of its functional fate. In the majority of cases, duplication is resolved by the non-functionalization of one of the gene copies and its subsequent erosion from the genome.<a reveal-id="dsx025-B1" data-open="dsx025-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsx025-B3" data-open="dsx025-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> Accordingly, 92% of all genes that were duplicated through whole-genome duplication (WGD) &gt; 100 MYA in <em>Saccharomyces</em> returned to single copy genes ‘shortly’ after duplication.<a reveal-id="dsx025-B17" data-open="dsx025-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> Nonetheless, in many species, including yeast, the number of duplicated genes is larger than predicted by theory ranging between 30% of the genes in yeast<a reveal-id="dsx025-B18" data-open="dsx025-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> and more than 50% in plants.<a reveal-id="dsx025-B6" data-open="dsx025-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsx025-B19" data-open="dsx025-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> Determining what genes remain in the genome as duplicates, and consequently lead to evolutionary leaps, is an important aim in evolutionary biology. However, this objective remains to be achieved.</p><p>A number of hypotheses have been proposed to explain the persistence of certain genes in duplicate. Rapid sequence divergence between gene copies can lead to their functional divergence followed by strong selective constraints on each copy, which could contribute to the preservation of duplicates in the genomes.<a reveal-id="dsx025-B20 dsx025-B21 dsx025-B22 dsx025-B23" data-open="dsx025-B20 dsx025-B21 dsx025-B22 dsx025-B23" class="link link-ref link-reveal xref-bibr"><sup>20–23</sup></a> Functional divergence requires, nevertheless, long evolutionary times and given that selection relaxes after gene duplication, selective pressures are unlikely to retain both gene copies during the first million years following duplication. Preservation of duplicates can also be selectively favoured by the need to maintain gene–dosage balance,<a reveal-id="dsx025-B5" data-open="dsx025-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx025-B21" data-open="dsx025-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a><sup>,</sup><a reveal-id="dsx025-B24" data-open="dsx025-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a><sup>,</sup><a reveal-id="dsx025-B25" data-open="dsx025-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> or provide genetic robustness against deleterious mutations.<a reveal-id="dsx025-B26" data-open="dsx025-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a><sup>,</sup><a reveal-id="dsx025-B27" data-open="dsx025-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> However, all these scenarios do not provide a general mechanistic explanation for what makes duplicates persist or alternatively perish.</p><p>Recently, it has been proposed dosage sub-functionalization as a plausible hypothesis to explain the fate of whole-genome duplicates.<a reveal-id="dsx025-B28" data-open="dsx025-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> According to this hypothesis, highly expressed genes are more likely to be preserved as duplicates than lowly expressed genes. This is because stochastic variations in the levels of expression of the gene copies of highly expressed duplicates would not lead to copies with a lower expression level than that required for purifying selection to act upon them. Therefore, highly expressed duplicates are less likely to return to single copy genes by drift. Whether this hypothesis could be applied to all duplicates regardless of the mechanism that originated them and whether such dosage sub-functionalization could also determine the patterns of divergence between gene copies has not been explored before.</p><p>Here, we present evidence that the levels of gene expression are correlated with the fates of whole-genome and small-scale duplicates, with highly expressed genes being more likely to be retained in double copy after duplication for longer periods of time than lowly expressed genes. Such duplicates are also more phylogenetically stable. We also show that the ancestral levels of gene expression are correlated with the evolution of duplicates expression. Retained duplicated genes evolve strong patterns of transcriptional (also known as phenotypic) plasticity, which are also correlated with the levels of gene expression. Finally, while the levels of gene expression are correlated with the duplicability of genes, duplicates phenotypic plasticity is manifested only after gene duplication; and this plasticity is proportional to the expression divergence between the copies of duplicated genes.</p>                            <h2 scrollto-destination=104086077 id="104086077" class="section-title" >2. Material and methods</h2>
                            <h3 scrollto-destination=104086078 id="104086078" class="section-title" >2.1. Identification of duplicated genes</h3>
<p>Paralogs pairs of duplicated genes were identified as the resulting best reciprocal hits from all-against-all BLAST searches using BLASTP with an <em>E</em>-value cutoff of 1E−5 and a 50 bit score.<a reveal-id="dsx025-B29" data-open="dsx025-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> Paralogs were then divided into two groups according to the mechanism of their origin: WGDs and SSDs. WGDs are those extracted from the reconciled list provided by the Yeast Gene Order Browser (YGOB, <a class="link link-uri" href="http://wolfe.gen.tcd.ie" target="_blank">http://wolfe.gen.tcd.ie</a>//ygob<a reveal-id="dsx025-B30" data-open="dsx025-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a>) (555 pairs of genes), and these were not subjected to subsequent SSD. All other paralogs were considered to belong to the category of SSDs (560 pairs of genes).</p>                            <h3 scrollto-destination=104086080 id="104086080" class="section-title" >2.2. Growth of <em>S. cerevisiae</em> and gene expression analyses</h3>
<p>The transcriptomic profiling was performed in the <em>S. cerevisiae</em> Y06240 haploid <em>msh2</em> deletion strain (BY4741; <em>Mata his3D1 leu2DO met15DO ura3DO msh2::kanMX4</em>), with three technical replicates for each biological stress condition (3% lactic acid (YPL), 3% ethanol (YPE), 3% glycerol (YPG), 0.25mM H<sub>2</sub>O<sub>2</sub> + 1.5% dextrose (YPOxD)) in comparison with the normal growth condition (YPD media) (<a reveal-id="dsx025-F1" data-open="dsx025-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). Total RNA extractions were performed with RNeasy kit (Qiagen) following manufacturer instructions. Ribosomal RNA was removed by using Ribo-Zero Gold rRNA removal yeast (Illumina) depletion kit. Stranded RNA libraries were constructed using TruSeq stranded mRNA (Illumina) from oligo-dT captured mRNAs from depleted samples. Libraries were run in NextSeq 500 (Illumina) at 75nt single read by using High Output 75 cycles kit v2.0 (Illumina). </p>                        <a id="104086082" scrollto-destination="104086082"></a>
<div id="dsx025-F1" data-id="dsx025-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/m_dsx025f1.png?Expires=2147483647&Signature=G39Zde8fdLNtIh78gwsY8pb9Vf9rEFmNebiA2UfBdeLm2G4Qdo9Z4YFoBlB6naSnlC2eO0Vm~-WIaqslBD-XeT72ZmtUAx7qD70R-qd6fomFgygP60AzRXUcpQTvVrK1MV5kLju4IC8X~LqcM1hiMvmYbnu2-WpWT~3-UzugNwEd-1SvS7cLivLvMCh3pbPoGuSm4kuu-DIhxzAkGPvMLaRYYf-ZE2O-GPU61U5DK4xUYmELzG~6-rC7B2Vhw9U7CuyA48Z-gxUiimqLgmMJaWVSASTpRHFleamF55UPprrGoPD0Q4WEcJBmhdtglzn38lcQhkSgpwjsT91jGZJUzA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Growth experiments of S. cerevisiae under stress conditions. An initial isogenic population of S. cerevisiae (strain Y06240) was grown for 24 h in Yeast extract, peptone, dextrose medium (YPD) at 28 °C. This grown population was then subjected to four different stress conditions: ethanol, glycerol, lactate and oxidative stress in a medium supplemented with dextrose (each stress in indicated in a different colour). Growth experiments were performed in triplicate. Samples of each of the populations, in addition to the control population grown in YPD, were processed for RNA sequencing. The technical steps for the extraction and analysis of RNA sequences are also shown." path-from-xml="dsx025f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086082/dsx025f1.tif" path-from-xml="dsx025f1.tif" target="_blank">View large</a><a section="104086082" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/dsx025f1.png?Expires=2147483647&Signature=LeT8Wgxd264x53pn2~m1e-eZuZ2k0Aq1ueVYreZjtJcU8qqlSy-QJzxYJOh8~2Em76-IlNRNsCw-U3~vo3hLoYI-5OpEikMyd8OeLDBw7iNz5XFSC-kxdB3BT6WipfWqlLTL4AbgCLjnGj3Zf9o3Xdo1ujUa1~T3kjN1Dszy7JaxXSOTlA-3UMq40laM4I5L5Kqe1aqwsFS2feht~Dxu8XZwpPTr0TNMa850DpaGKcxzx-qNGjn9-eyXL~yN4of8ahDU95jFlzSKqPQvdMQRoQME6nexwt1QCqpfc-7JJjEGIBpBKtGB~3paoK9-2W542cERGSAbJ0Y-ewJC3FA9ug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086082&ar=3868561&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx025f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Growth experiments of <em>S. cerevisiae</em> under stress conditions. An initial isogenic population of <em>S. cerevisiae</em> (strain Y06240) was grown for 24 h in Yeast extract, peptone, dextrose medium (YPD) at 28 °C. This grown population was then subjected to four different stress conditions: ethanol, glycerol, lactate and oxidative stress in a medium supplemented with dextrose (each stress in indicated in a different colour). Growth experiments were performed in triplicate. Samples of each of the populations, in addition to the control population grown in YPD, were processed for RNA sequencing. The technical steps for the extraction and analysis of RNA sequences are also shown.</p></div></div><div content-id="dsx025-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/m_dsx025f1.png?Expires=2147483647&Signature=G39Zde8fdLNtIh78gwsY8pb9Vf9rEFmNebiA2UfBdeLm2G4Qdo9Z4YFoBlB6naSnlC2eO0Vm~-WIaqslBD-XeT72ZmtUAx7qD70R-qd6fomFgygP60AzRXUcpQTvVrK1MV5kLju4IC8X~LqcM1hiMvmYbnu2-WpWT~3-UzugNwEd-1SvS7cLivLvMCh3pbPoGuSm4kuu-DIhxzAkGPvMLaRYYf-ZE2O-GPU61U5DK4xUYmELzG~6-rC7B2Vhw9U7CuyA48Z-gxUiimqLgmMJaWVSASTpRHFleamF55UPprrGoPD0Q4WEcJBmhdtglzn38lcQhkSgpwjsT91jGZJUzA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Growth experiments of S. cerevisiae under stress conditions. An initial isogenic population of S. cerevisiae (strain Y06240) was grown for 24 h in Yeast extract, peptone, dextrose medium (YPD) at 28 °C. This grown population was then subjected to four different stress conditions: ethanol, glycerol, lactate and oxidative stress in a medium supplemented with dextrose (each stress in indicated in a different colour). Growth experiments were performed in triplicate. Samples of each of the populations, in addition to the control population grown in YPD, were processed for RNA sequencing. The technical steps for the extraction and analysis of RNA sequences are also shown." path-from-xml="dsx025f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086082/dsx025f1.tif" path-from-xml="dsx025f1.tif" target="_blank">View large</a><a section="104086082" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/dsx025f1.png?Expires=2147483647&Signature=LeT8Wgxd264x53pn2~m1e-eZuZ2k0Aq1ueVYreZjtJcU8qqlSy-QJzxYJOh8~2Em76-IlNRNsCw-U3~vo3hLoYI-5OpEikMyd8OeLDBw7iNz5XFSC-kxdB3BT6WipfWqlLTL4AbgCLjnGj3Zf9o3Xdo1ujUa1~T3kjN1Dszy7JaxXSOTlA-3UMq40laM4I5L5Kqe1aqwsFS2feht~Dxu8XZwpPTr0TNMa850DpaGKcxzx-qNGjn9-eyXL~yN4of8ahDU95jFlzSKqPQvdMQRoQME6nexwt1QCqpfc-7JJjEGIBpBKtGB~3paoK9-2W542cERGSAbJ0Y-ewJC3FA9ug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086082&ar=3868561&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx025f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Growth experiments of <em>S. cerevisiae</em> under stress conditions. An initial isogenic population of <em>S. cerevisiae</em> (strain Y06240) was grown for 24 h in Yeast extract, peptone, dextrose medium (YPD) at 28 °C. This grown population was then subjected to four different stress conditions: ethanol, glycerol, lactate and oxidative stress in a medium supplemented with dextrose (each stress in indicated in a different colour). Growth experiments were performed in triplicate. Samples of each of the populations, in addition to the control population grown in YPD, were processed for RNA sequencing. The technical steps for the extraction and analysis of RNA sequences are also shown.</p></div></div><p>RNA libraries were sequenced at Genomic core facility at Servicio Central de Soporte a la Investigación Experimental (SCSIE) from University of Valencia, Spain. Raw reads were analyzed using FastQC report and cleaned with CutAdapt as implemented in RobiNA software package v 1.2.4.<a reveal-id="dsx025-B31" data-open="dsx025-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> Low-quality reads were filtered and trimmed (Phred score inferior to 20 and size less than 40 nt were discarded). The reads were then aligned with Bowtie (up to two mismatches accepted) to the reference transcriptome (PRJNA290217) from the reference S288c strain. The normalization and statistical evaluation of differential gene expression has been performed using edgeR<a reveal-id="dsx025-B32" data-open="dsx025-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> or DESeq<a reveal-id="dsx025-B33" data-open="dsx025-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> with a <em>P</em>-value cut-off of 0.05, using the Benjamini–Yekutieli<a reveal-id="dsx025-B34" data-open="dsx025-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> method for multiple testing correction of <em>P</em>-value, and setting the Log-fold change at min = 1 to determine differential expression. The raw data (reads counts) was normalized according to the default procedure of the differential expression analysis package used (edgeR or DESeq), being the dispersion estimated using the pooled setting, and RPKM (Reads Per Billion) expression values estimated as implemented in RobiNA software.<a reveal-id="dsx025-B31" data-open="dsx025-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> All newly sequenced RNA sequences are available from the Sequence Read Archive with the following accession number (SRP074821). Expression data for each of the <em>S. cerevisiae</em> genes under YPD and each of the four stress conditions as well as the adjusted probabilities to identify significant fold changes are available in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables1_dsx025.xls?Expires=2147483647&amp;Signature=nPaJXHYV6gQ53wdehHE7rsx-iQX0ETTZVzTjn5AmU2zebm8Vy102UjU827wCKDXjTAy9Rw5tHgpdB24X3VQfxnDGbDWNWgQqU-1RIWH87wtlk8HEB~nZXuEpFyi9FlceBEO7TR8mceWMN2rApEkpPf9k3IcugRkKGNWhyrPWUbrj6izwPoG0p7vTYZt13G7yv9G8dktdeUpRxvuF3OFB9UF9p4VQCrgDHJZFyxLMZ9lEkCUn9wD-295ni3IcoZ~85QXn7LHT~ELwTDUdqmWzq8lV~K9rIh7KsVPdATkyvhw2EVan6TBFpv5NPJQ5nSe1xtd976ljXnpi8-ywkvJfxw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S1</a></span>–<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables4_dsx025.xls?Expires=2147483647&amp;Signature=du~ll4wTPKL5K5g7AHRPqwblsCMJEv5cjKBC8E9f5yBP2dbcp5q9V3tCnuGTKGHGNBYckv2c3MLFqNn3q50VNEd1jFW6VRapKUNr2cg42SN9NOuw2PTmX2cM0KMzK4Ym1hCXkzU9xobbhT22BqHzME0KwY4qycOpHXCMCKiZjdNK3wpC9PeywRu1tYW-6Ph0GfNsM2DR7~IWEFWSYmANoITLYCdkixyf9e4fuPfO5mOcJqG5o1k71a0LUuUpFNhRfiA8WZC-hZbQPzV591rP0vNpsrFvg34TEzL~je4gw~icswrgXqXCcqjWUsjs0JUmwA5sBb22VH52q1rbJjtqiA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S4</a></span>.</p>                            <h3 scrollto-destination=104086084 id="104086084" class="section-title" >2.3. Expression data for <em>Lachancea kluyveri</em></h3>
<p>Growth conditions, RNA extraction and sequencing are specified in a previous study.<a reveal-id="dsx025-B35" data-open="dsx025-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> Briefly, authors performed the analyses on <em>L. kluyveri</em> reference strain CBS 3082_a (<em>MAT</em>a). Transcriptomic data were obtained from growth cultures at mid exponential phase and for 20 different media, including YPD and 19 other stress conditions (listed in Ref. [<a reveal-id="dsx025-B35" data-open="dsx025-B35" class="link link-ref link-reveal xref-bibr">35</a>]). RNA sequencing was performed using Illumina HiSeq2000 platform with 50-base pair non-oriented single reads (<span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables5_dsx025.xlsx?Expires=2147483647&Signature=Lr9-K9bUaET87gTU2y6WeX8NfUeVV-VQNgqmGdQOMGKvTtoxNrDVu7~pwMVcGu6266LxmNS1PnEPXRk~YcKCmy0Hmqr6m9jmYH-19eQNKBgomnll9I7Xh-4ZTHz5lFwkf6GkrGAReeav9cDscICWxQD1K3lv35k7zdVB7XaKVhCPlpxNvkB4nvJgUU6oek~Q6CAS8p42OHNF3x4JXUiYiUDH~x~yVq8oJuEp2an0y7Rb4Yl6hRd2Dkx8rdoa6wKuEA~nfV7x6pWD8MLznu8Qi0~uI7lVkNZW-efuzu54Z7Ql7ei3dq7nSSnIrUC0G9Kvmi51y3wH7Mpo4ruisecmuQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>).</p>                            <h3 scrollto-destination=104086086 id="104086086" class="section-title" >2.4. Expression data for <em>Candida glabrata</em></h3>
<p><em>Candida glabrata</em> ATC2001 strain expression data (in the form of RPKMs) were obtained from a previous study.<a reveal-id="dsx025-B36" data-open="dsx025-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> Briefly, authors grown <em>C. glabrata</em> in normal YPD media and in the M199 medium at different pH values for pH shift. Total RNA was isolated by hot phenol–chlorophorm method. Libraries were subjected to ribosomal RNA depletion, and sequenced using Illumina HiSeq2000 platform with 100-bp paired-end strand-specific. Reads were mapped with TOPHAT2, and counted using htseq (-m union, -t exon conditions) and normalized by the number of reads per kilobase of exon per million mapped reads (<span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables6_dsx025.xls?Expires=2147483647&Signature=CLIUJaLyigYLq7rEkzJembnre1zrj5t~H19D4ZNWa6fM2o792BFtRXKqvcZJLL4BkoNkoHcBlqdG99wBzuKVC7K3sWWdZfjkmZEH2ODM-TXpafvfOkcr7BF7c9TB9VV3hWyQSyFoD412F6Wtcet8PgeJFbXM~EqMNVRzenq48d5EGAjrcAKk~vBauk0AMavGBVYK0-DrWcqsDGZfs9pZ35JUhYNU8~hx2S-svLd-i1BR8hm3O3pmYLGJ0Gd~iVtROIiCya~50esHtRpj8ADCKJxqZdNh0Awy1BfvlcVVQJ8Y3ts0sdrn3Qpo-YBcwJ9wrBoWmke0YLx7HLT0dyTviQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). Differentially expressed genes were identified with raw counts by DeSeq and EdgeR, using the same cut-off parameters than the ones used in this study.</p>                            <h3 scrollto-destination=104086088 id="104086088" class="section-title" >2.5. Software</h3>
<p>Calculations and statistics were performed using MS Excel and R 3.2.1. Data management was possible using in-house built PERL scripts.</p>                            <h2 scrollto-destination=104086090 id="104086090" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=104086091 id="104086091" class="section-title" >3.1. Duplicates preservation and phylogenetic stability are correlated with the levels of gene expression</h3>
<p>Highly expressed genes are more likely to be preserved as duplicates after whole-genome duplication (WGD) than lowly expressed genes in <em>S. cerevisiae</em>.<a reveal-id="dsx025-B37" data-open="dsx025-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a><sup>,</sup><a reveal-id="dsx025-B28" data-open="dsx025-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a><sup>,</sup><a reveal-id="dsx025-B38 dsx025-B39 dsx025-B40 dsx025-B41" data-open="dsx025-B38 dsx025-B39 dsx025-B40 dsx025-B41" class="link link-ref link-reveal xref-bibr"><sup>38–41</sup></a> We searched for the orthologs of <em>S. cerevisiae</em> genes in <em>L. kluyveri</em> (strain CBS 3082; synonymous of <em>Saccharomyces kluyveri</em>) and their expression in Yeast Extract Peptone containing dextrose medium (YPD)<a reveal-id="dsx025-B35" data-open="dsx025-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables5_dsx025.xlsx?Expires=2147483647&amp;Signature=Lr9-K9bUaET87gTU2y6WeX8NfUeVV-VQNgqmGdQOMGKvTtoxNrDVu7~pwMVcGu6266LxmNS1PnEPXRk~YcKCmy0Hmqr6m9jmYH-19eQNKBgomnll9I7Xh-4ZTHz5lFwkf6GkrGAReeav9cDscICWxQD1K3lv35k7zdVB7XaKVhCPlpxNvkB4nvJgUU6oek~Q6CAS8p42OHNF3x4JXUiYiUDH~x~yVq8oJuEp2an0y7Rb4Yl6hRd2Dkx8rdoa6wKuEA~nfV7x6pWD8MLznu8Qi0~uI7lVkNZW-efuzu54Z7Ql7ei3dq7nSSnIrUC0G9Kvmi51y3wH7Mpo4ruisecmuQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). <em>Lachancea kluyveri is</em> a respiratory yeast species pre-dating the WGD that took place in <em>S. cerevisiae</em> &gt; 100 million years ago.<a reveal-id="dsx025-B17" data-open="dsx025-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> We found <em>L. kluyveri</em> orthologs for 5643 <em>S. cerevisiae</em> genes. Of the 5643 genes, 1469 genes were orthologs of <em>S. cerevisiae</em> duplicates, including WGDs and small-scale duplicates (SSDs), and 4174 were orthologs of <em>S. cerevisiae</em> singletons (<span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables5_dsx025.xlsx?Expires=2147483647&amp;Signature=Lr9-K9bUaET87gTU2y6WeX8NfUeVV-VQNgqmGdQOMGKvTtoxNrDVu7~pwMVcGu6266LxmNS1PnEPXRk~YcKCmy0Hmqr6m9jmYH-19eQNKBgomnll9I7Xh-4ZTHz5lFwkf6GkrGAReeav9cDscICWxQD1K3lv35k7zdVB7XaKVhCPlpxNvkB4nvJgUU6oek~Q6CAS8p42OHNF3x4JXUiYiUDH~x~yVq8oJuEp2an0y7Rb4Yl6hRd2Dkx8rdoa6wKuEA~nfV7x6pWD8MLznu8Qi0~uI7lVkNZW-efuzu54Z7Ql7ei3dq7nSSnIrUC0G9Kvmi51y3wH7Mpo4ruisecmuQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). The expression of <em>L. kluyveri</em> orthologs of <em>S. cerevisiae</em> duplicates (Median: 10.43; measured as the log<sub>2</sub>-transformed Reads Per billion, RPKM) was significantly greater than that of <em>L. kluyveri</em> orthologs of <em>S. cerevisiae</em> singletons (Median: 9.52) (Wilcoxon rank test: <em>P</em> &lt; 2.2 × 10<sup>−16</sup>). We compared the transcription levels of genes in <em>S. cerevisiae</em> obtained in our study with those from another study<a reveal-id="dsx025-B42" data-open="dsx025-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> that used ribosomal profiling, a technique that measures ribosome occupancy and translation genome wide and provides an accurate measure of the translatable mRNA. RPKMs correlated strongly and significantly with the data of ribosome profiling (Spearman’s correlation: <em>ρ</em> = 0.77, <em>P</em> &lt; 2.2 × 10<sup>−16</sup>, <span class="content-section supplementary-material"><a path-from-xml="sup29" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementary_dataset1_dsx025.pdf?Expires=2147483647&amp;Signature=Yt5wmLlppgvB1A1B~TTHFlVJAyllbCJ3Rzi8NHV8THtyvLU5-NV-E22fGWHITi9lmzdwI0vrSk3DONaH9GbomwAduS7tRkh27WCTh1fX4Wcp8ZZ7nMg~XXC5zkX01sExxEITY58WCkkeeD6qnY49vPSCDwJnUmmb3cui2gjlY7GP4D59kZjs6YuFgGCA2UJg3-XGUbSSE8lHTbiLEO-yb6sgAGiAk9qc1YtiWIF6wf~-cNc0wPZJZY-FPimJ6yF~Moduqi-yIX1d4BIj64YRYt-mu0DRlznhbfw2lUUyAG1gio4sGXtPNqvyQevoPF8E5jEMkcJQSO163HQ8uXdcgw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> S1), indicating that RPKMs are indicative of the levels of gene expression and also the translatable mRNAs.</p><p>Previous studies concluded that highly expressed genes were more likely to be preserved as duplicates after WGDs because of absolute dosage constraints and constraints on dosage balance.<a reveal-id="dsx025-B28" data-open="dsx025-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a><sup>,</sup><a reveal-id="dsx025-B37" data-open="dsx025-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a><sup>,</sup><a reveal-id="dsx025-B39" data-open="dsx025-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a><sup>,</sup><a reveal-id="dsx025-B43 dsx025-B44 dsx025-B45 dsx025-B46" data-open="dsx025-B43 dsx025-B44 dsx025-B45 dsx025-B46" class="link link-ref link-reveal xref-bibr"><sup>43–46</sup></a> Indeed, we found that this trend is true for <em>L. kluyveri</em> orthologs of <em>S. cerevisiae</em> WGDs (<em>N</em> = 561, Median expression: 10.64, Wilcoxon rank test: <em>P</em> &lt; 2.2 × 10<sup>−16</sup>) and also for orthologs of <em>S. cerevisiae</em> SSDs (<em>N</em> = 908, Median expression: 10.28, Wilcoxon rank test: <em>P</em> &lt; 2.2 × 10<sup>−16</sup>). The level of expression of orthologs of WGDs was, nevertheless, higher than that for SSDs (Wilcoxon rank test: <em>P</em> = 7.79 × 10<sup>−5</sup>).</p><p>The higher expression of duplicates compared to singletons can be due to a greater presence of genes encoding protein-complex proteins among duplicates than singletons. We extracted the list of protein complexes from a previous study, using the table of annotated yeast high-throughput complexes available at (<a class="link link-uri" href="http://wodaklab.org/" target="_blank">http://wodaklab.org/</a>cyc2008<a class="link link-uri" href="http:///downloads" target="_blank">/downloads</a>).<a reveal-id="dsx025-B47" data-open="dsx025-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> Genes encoding proteins that are part of protein complexes (<em>N</em> =  1913) (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables7_dsx025.xls?Expires=2147483647&Signature=uM1oaLXJJc-NU6zJaRE6-13I926wV0n-SBUHGQvAvPBXHhaySF-005YTQDKtMX~tZWB8-nCqZHi5IQNyR0e606QB1QQwSxq7yxVMN4badYRgE3srzksixUB3KNrXngDVq8ksfHxIhc7ViKona6uFhTkvmqLTqOQXydIODELSSUDg5dv2gFC7dZXxCsaXFxM6~UM2F~9xaPSXfN~AZ9hdfZI8dleB5529MoDi3o-KOq689yQk53m7jj1HcOlsZal8y~wzBDcXbPCz3yR-p9iMQpaEDRzvxWoOMh7twxkpg8REN5l-FsclrwqY-zY1GlYe2-uyn2tJLuuCcb09QYLH7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>) do exhibit greater expression (Median expression: 11.56) than genes encoding complex-free proteins (<em>N</em> = 3960) (Median expression: 10.61, Wilcoxon rank test: <em>P</em> &lt; 2.2 × 10<sup>−16</sup>). However, neither WGDs were more enriched for complex-encoding genes than singletons in <em>S. cerevisiae</em> (Fisheŕs exact test: <em>F </em>= 1.02, <em>P</em> = 0.76) nor SSDs showed significant difference in terms of enrichment for complex-encoding genes when compared to singletons (Fisheŕs exact test: <em>F</em> = 1.11, <em>P</em> = 0.15).</p><p>One caveat in this analysis is that gene expression in <em>L. kluyveri</em> may not reflect gene expression immediately after WGD. Against this prediction, gene expression in <em>L. kluyveri</em>, a species predating the WGDs and SSDs used in this study, was strongly and significantly correlated with gene expression in <em>S. cerevisiae</em> (Spearman correlation: <em>ρ</em> = 0.59, <em>P</em> &lt; 2.2 × 10<sup>−16</sup>, <a reveal-id="dsx025-F2" data-open="dsx025-F2" class="link link-reveal link-table xref-fig">Fig. 2a</a>). </p>                        <a id="104086096" scrollto-destination="104086096"></a>
<div id="dsx025-F2" data-id="dsx025-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/m_dsx025f2.png?Expires=2147483647&Signature=GKWMiw4~J4Vnpr-CMN2E54jA8tyV8aFMjPhqLxjsPvFPyQub5gqHpTOiB7N3jgJ2ccgwh0XhhbvYODctCIYfVGFVN4IAznFEZCu-3klhO7OUIpZkYWdabKE5zkF7ENHmB-1AN8-SVHHvtmXqsj2z0J161rmHuS5x2FcfqAegEcaY3Jc5grPYvy5USmkBBb0V25rvw0IdeLYxSDx51qxxvxuiKiovLZouSxnfJsJRYxtYt-6cFSGADjStun6IdAlfsQaYTanA6cAHmN3-OdaRO-ghDXX~Yi~1L8fSG5zNNHkWEeqxHZLRTrH9GsEbthQH1lzZUGnAorpdXIvI2Hccfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The levels of gene expression determine gene duplicability. (a) Expression of genes in the pre-WGD species Lachancea kluyveri correlate positively with genes expressions in the post-WGD species Saccharomyces cerevisiae. Axes represent the transcription levels of genes as the logarithmic transformation of the Fraction of Reads Per billion. (b) Estimation of the phylogenetic stability (i.e. number of species) of duplicated genes after the whole genome duplication in yeast. Black and gray circles refer to gene copies. A missing circle in one of the post-WGD species represent a gene copy loss, hence return of that duplicate to single-copy gene in that species. The phylogenetic stability (PS) of that duplicate is calculated as the mean number of species in which gene copies are present. (c) Phylogenetic stability correlates with the levels of gene expression of duplicates. (d) Correlation of PS of WGDs and their expression levels. (e) Correlation of PS of SSDs and their expression levels." path-from-xml="dsx025f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086096/dsx025f2.tif" path-from-xml="dsx025f2.tif" target="_blank">View large</a><a section="104086096" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/dsx025f2.png?Expires=2147483647&Signature=4HKX8u77UiDliL8tT5Uq723vYBT7TKIBAmJtwEec2J3ezsnT4LzhouEyV7sQKq8pANxVkv-e0d67zo6b7dk~gjXL9jBTmC0312YMtdYsUkK3AYsFn8-9x8f-9ErbkN~BNdMTmE-r6wI-qzsQ~f~dOOK7gvZEmZCwCP7M6wQiqeAgbFWZqYejaQ8R~0cuVwtR88Q6PBcnHQJUE4WAlFT7HtCgZDVU8dFEkW3ynKzrmn9UQvAekkalF8FFGQeX2Hvff9l0GFFl9y~HEJSR~0zz2HBFzazUWSZrFUQCypg~Qpue6uNjld6NvM~nYkwkDIpwpvD8bACl-C9fAao1LMO99Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086096&ar=3868561&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx025f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The levels of gene expression determine gene duplicability. (a) Expression of genes in the pre-WGD species <em>Lachancea kluyveri</em> correlate positively with genes expressions in the post-WGD species <em>Saccharomyces cerevisiae</em>. Axes represent the transcription levels of genes as the logarithmic transformation of the Fraction of Reads Per billion. (b) Estimation of the phylogenetic stability (i.e. number of species) of duplicated genes after the whole genome duplication in yeast. Black and gray circles refer to gene copies. A missing circle in one of the post-WGD species represent a gene copy loss, hence return of that duplicate to single-copy gene in that species. The phylogenetic stability (PS) of that duplicate is calculated as the mean number of species in which gene copies are present. (c) Phylogenetic stability correlates with the levels of gene expression of duplicates. (d) Correlation of PS of WGDs and their expression levels. (e) Correlation of PS of SSDs and their expression levels.</p></div></div><div content-id="dsx025-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/m_dsx025f2.png?Expires=2147483647&Signature=GKWMiw4~J4Vnpr-CMN2E54jA8tyV8aFMjPhqLxjsPvFPyQub5gqHpTOiB7N3jgJ2ccgwh0XhhbvYODctCIYfVGFVN4IAznFEZCu-3klhO7OUIpZkYWdabKE5zkF7ENHmB-1AN8-SVHHvtmXqsj2z0J161rmHuS5x2FcfqAegEcaY3Jc5grPYvy5USmkBBb0V25rvw0IdeLYxSDx51qxxvxuiKiovLZouSxnfJsJRYxtYt-6cFSGADjStun6IdAlfsQaYTanA6cAHmN3-OdaRO-ghDXX~Yi~1L8fSG5zNNHkWEeqxHZLRTrH9GsEbthQH1lzZUGnAorpdXIvI2Hccfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The levels of gene expression determine gene duplicability. (a) Expression of genes in the pre-WGD species Lachancea kluyveri correlate positively with genes expressions in the post-WGD species Saccharomyces cerevisiae. Axes represent the transcription levels of genes as the logarithmic transformation of the Fraction of Reads Per billion. (b) Estimation of the phylogenetic stability (i.e. number of species) of duplicated genes after the whole genome duplication in yeast. Black and gray circles refer to gene copies. A missing circle in one of the post-WGD species represent a gene copy loss, hence return of that duplicate to single-copy gene in that species. The phylogenetic stability (PS) of that duplicate is calculated as the mean number of species in which gene copies are present. (c) Phylogenetic stability correlates with the levels of gene expression of duplicates. (d) Correlation of PS of WGDs and their expression levels. (e) Correlation of PS of SSDs and their expression levels." path-from-xml="dsx025f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086096/dsx025f2.tif" path-from-xml="dsx025f2.tif" target="_blank">View large</a><a section="104086096" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/dsx025f2.png?Expires=2147483647&Signature=4HKX8u77UiDliL8tT5Uq723vYBT7TKIBAmJtwEec2J3ezsnT4LzhouEyV7sQKq8pANxVkv-e0d67zo6b7dk~gjXL9jBTmC0312YMtdYsUkK3AYsFn8-9x8f-9ErbkN~BNdMTmE-r6wI-qzsQ~f~dOOK7gvZEmZCwCP7M6wQiqeAgbFWZqYejaQ8R~0cuVwtR88Q6PBcnHQJUE4WAlFT7HtCgZDVU8dFEkW3ynKzrmn9UQvAekkalF8FFGQeX2Hvff9l0GFFl9y~HEJSR~0zz2HBFzazUWSZrFUQCypg~Qpue6uNjld6NvM~nYkwkDIpwpvD8bACl-C9fAao1LMO99Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086096&ar=3868561&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx025f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The levels of gene expression determine gene duplicability. (a) Expression of genes in the pre-WGD species <em>Lachancea kluyveri</em> correlate positively with genes expressions in the post-WGD species <em>Saccharomyces cerevisiae</em>. Axes represent the transcription levels of genes as the logarithmic transformation of the Fraction of Reads Per billion. (b) Estimation of the phylogenetic stability (i.e. number of species) of duplicated genes after the whole genome duplication in yeast. Black and gray circles refer to gene copies. A missing circle in one of the post-WGD species represent a gene copy loss, hence return of that duplicate to single-copy gene in that species. The phylogenetic stability (PS) of that duplicate is calculated as the mean number of species in which gene copies are present. (c) Phylogenetic stability correlates with the levels of gene expression of duplicates. (d) Correlation of PS of WGDs and their expression levels. (e) Correlation of PS of SSDs and their expression levels.</p></div></div><p>Duplicated genes exhibit different patterns of gene retention and phylogenetic stability in the different post-WGD <em>Saccharomyces</em> species.<a reveal-id="dsx025-B48" data-open="dsx025-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> We classified <em>S. cerevisiae</em> duplicated genes according to the presence of the two copies in each of the twelve available species post-dating the WGD (<a reveal-id="dsx025-F2" data-open="dsx025-F2" class="link link-reveal link-table xref-fig">Fig. 2b</a>). We first asked whether the expression of duplicates generated before <em>Saccharomycetales</em> speciation (including WGDs and SSDs) correlates with their phylogenetic stability, measured as the mean number of post-WGD species in which each copy was present (<a reveal-id="dsx025-F2" data-open="dsx025-F2" class="link link-reveal link-table xref-fig">Fig. 2b</a>). To determine the number of species postdating WGD in which each gene copy is present we used the Pillars information available from the Yeast Gene Order Information,<a reveal-id="dsx025-B30" data-open="dsx025-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> which provides gene order and annotation for 12 post-WGD yeast species (<span class="content-section supplementary-material"><a path-from-xml="sup8" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables8_dsx025.xlsx?Expires=2147483647&Signature=dvik8lEucGyEXr1zUF~bCcQUMbwKDkuCB5vdcUQWdjvHBDnz-u2e2m8tQgRe56XUOGfaTQfIPM3qPjRbsjb8lFKgNh6xHzdwfoVg3TFp5qHh-29uIzLg3dfATl~DQoZM~cqCKwFsuXvlLIuJTAv-G-0SwcsMUyAzOzVxx9BOtK8J777hgo40DwVzV6sdYSnkhhnhbQfKeM1XzpFKdmgz55uBYTT9LCSoDXrnqN0tHwDl0jpcngb6qUkDLoVavoTMmc1Qo2lCnHXxPxMHz3vUPNbHS5bv7paJeeE~RHJzwdISd~3aab-61XFvScdDxhepZ2BTtEn~AYgqi9uBhxTmIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). For each gene copy, we counted the number of species in which it is found and averaged this number for the two sister gene copies in a duplicated pair (<a reveal-id="dsx025-F2" data-open="dsx025-F2" class="link link-reveal link-table xref-fig">Fig. 2b</a>). There was a positive and significant correlation between the mean gene copies expression in YPD and their phylogenetic stability (Spearman correlation: <em>ρ</em> = 0.27, <em>P</em> &lt; 2.2 × 10<sup>−16</sup>, <a reveal-id="dsx025-F2" data-open="dsx025-F2" class="link link-reveal link-table xref-fig">Fig. 2c</a>). We then repeated the analysis for WGDs and SSDs separately. WGDs exhibited positive weak but significant correlation between gene expression and phylogenetic stability (Spearman correlation: <em>ρ</em> = 0.13, <em>P</em> = 1.15 × 10<sup>−5</sup>, <a reveal-id="dsx025-F2" data-open="dsx025-F2" class="link link-reveal link-table xref-fig">Fig. 2d</a>). In contrast, SSDs showed strong and significant correlation between gene expression and phylogenetic stability (Spearman correlation: <em>ρ</em> = 0.40, <em>P</em> &lt; 2.2 × 10<sup>−16</sup>, <a reveal-id="dsx025-F2" data-open="dsx025-F2" class="link link-reveal link-table xref-fig">Fig. 2e</a>).</p>                            <h3 scrollto-destination=104086098 id="104086098" class="section-title" >3.2. The magnitude of divergence of duplicates expression correlates with the level of gene expression</h3>
<div class="block-child-p">A pivotal hypothesis to the dosage sub-functionalization proposed by Gout and Lynch<a reveal-id="dsx025-B28" data-open="dsx025-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> is that highly expressed duplicates should exhibit more expression variation despite the action of purifying selection than lowly expressed genes. This is because noise in the expression of highly expressed genes is unlikely to compromise the selective constraints on these genes. That is, genes with higher expression levels should be more ‘noisy’ in their expression when duplicated, and thus they should generate more expression polymorphism in the population than lowly expressed genes. Accordingly, highly expressed duplicates are more likely to yield gene copies with diverged expressions than lowly expressed duplicates. To test this hypothesis, we first measured the fold expression difference (<em>D</em>) between the gene copies <em>i</em> and <em>j</em> when <em>S. cerevisiae</em> was grown under YPD as <a id="E1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>D</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo>=</mo><mn>1</mn><mo>–</mo><mrow><mo stretchy="false">[</mo><mrow><mi>M</mi><mi>i</mi><mi>n</mi><mrow><mo stretchy="false">(</mo><mrow><msub><mi>E</mi><mi>i</mi></msub><mo>,</mo><msub><mi>E</mi><mi>j</mi></msub></mrow><mo stretchy="false">)</mo></mrow></mrow><mo stretchy="false">]</mo></mrow><mo>/</mo><mrow><mo stretchy="false">[</mo><mrow><mi>M</mi><mi>a</mi><mi>x</mi><mrow><mo stretchy="false">(</mo><mrow><msub><mi>E</mi><mi>i</mi></msub><mo>,</mo><msub><mi>E</mi><mi>j</mi></msub></mrow><mo stretchy="false">)</mo></mrow></mrow><mo stretchy="false">]</mo></mrow></mrow></math></div></div></div></div><p>with <em>E</em> referring to the expression of the gene under normal conditions (<span class="content-section supplementary-material"><a path-from-xml="sup9" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables9_dsx025.xls?Expires=2147483647&Signature=JQZteoMM2HYroN2GkZ423jAtKrsmLYLi~NH1TLcUPlzudK4HtOHZd0QqNkPmiH3jNe85eDw~UFgqKAS~zJSJxo6txuAi65Em5aOw6xDcRJDrhWGRe1tvbC~A-Wlr-vmeZhHZwBaYoKmKamkbHcij9gNv2EHe9mlt0FvURfsnzqoJecEYTRk1q8eaFd2w4zYe8DLMTrhkO0xyCvPScqZ~8GcZcwOwkFyDtW1SqGsKS6LCp-9t4a-~TGd13JYw~jf3JngEhD5Czs5FaYnmK4lchm218WV0~VxjlbqYiZI~938tfnRxKqgLI~eA0YQI4zfdx8tagDStmXGpNRNmTSALiA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). <em>D<sub>i,j</sub></em> is normalized by the level of gene expression (i.e. the value is 0 ≤ <em>D<sub>i,j</sub> </em>≤ 1), and thus it is an unbiased measure of the expression divergence between the gene copies. In support of our hypothesis, there was a weak but very significant correlation between the average expression of the gene copies of duplicates and <em>D<sub>i,j</sub></em> (Spearman correlation: <em>ρ</em> = 0.18, <em>P</em> = 4.23 × 10<sup>−8</sup>). This correlation was also maintained when we analyzed separately WGDs (Spearman correlation: <em>ρ</em> = 0.18, <em>P</em> = 6.98 × 10<sup>−5</sup>) and SSDs (Spearman correlation: <em>ρ</em> = 0.17, <em>P</em> = 2.89 × 10<sup>−4</sup>).</p>                            <h3 scrollto-destination=104086101 id="104086101" class="section-title" >3.3. The expression levels and promoter architecture correlate with patterns of expression divergence of duplicates and their transcriptional plasticity</h3>
<p>Because higher expression can increase the chance for expression divergence after gene duplication, we sought to investigate if genes with higher expression can also evolve greater transcriptional plasticity under stress. Transcriptional plasticity is defined here as the ability of the gene to change its expression, while keeping its genotype, when the environment changes. For all four stresses with which <em>S. cerevisiae</em> was challenged (see section Material and methods), transcriptionally altered duplicates belonged to a set of genes with significantly higher expression in YPD growth media than transcriptionally unaltered duplicates (<a reveal-id="dsx025-F3" data-open="dsx025-F3" class="link link-reveal link-table xref-fig">Fig. 3a–d</a>). This was also true, with the exception of ethanol-induced stress, for singletons, albeit the effect was more pronounced in duplicates than it was in singletons. Noticeably, for all four-stress conditions in <em>S. cerevisiae</em>, the levels of expression of duplicates with no altered transcription under stress were significantly higher than that for transcriptionally altered singletons (<a reveal-id="dsx025-F3" data-open="dsx025-F3" class="link link-reveal link-table xref-fig">Fig. 3a–d</a>). </p>                        <a id="104086103" scrollto-destination="104086103"></a>
<div id="dsx025-F3" data-id="dsx025-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/m_dsx025f3.png?Expires=2147483647&Signature=tBslNyiZYF4XTWUK1P3feuDlzDkLToWHplmulfnz-lyiu128QNDClMBvrsRgYI4eD9Z0z4WlOz9VZoE3KGedWVkXQsNf0NEMlNHQTsyAFvWlJB6bKDdcXozyzSQGtfBA4Jm8a5ARWUnGTgcrrQJlNuarU0Em8zo-fn5hc~nkWGvFKAllXZjO4bcDA3IG-ahE04TIOTYkalORjC~R2nGC5ZYY5huci8StlEr2xssQnD5gGCc8e6CboquFT8jk~olevxTm9TrHu3-qWrQ~eHVkS0mjqPWKyBf5QJ9YxLB1NmSTjJBQLga1h-NR7X105cckS~~LKrtIMHmq3D9gnp9ijQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The levels of gene expression determine the transcriptional plasticity of genes. We present the levels of expression as the logarithmic transformation of the Reads Per billion (RPKM) for duplicates (red boxes) and singletons (blue boxes) with transcriptional alterations and duplicates (orange boxes) and singletons (white boxes) without altered expressions when S. cerevisiae is faced with four different stress conditions (a) Ethanol, (b) glycerol, (c) lactate and (d) oxidative stress in a medium supplemented with dextrose. Significant differences in the expression levels of sets of genes are indicated as *, **, ***, when the probabilities are P &lt; 0.05, P &lt; 0.01 and P &lt; 10−6, respectively, using a Wilcoxon rank test." path-from-xml="dsx025f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086103/dsx025f3.tif" path-from-xml="dsx025f3.tif" target="_blank">View large</a><a section="104086103" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/dsx025f3.png?Expires=2147483647&Signature=IfBDDQdrvpUUV~m8FRyqsclwWaDDmkkS3cFnR7~RGVd2VTNxV3CSZ1uluE5zvzJHiDA-fwSWRhHhOwIlosw1EfUwGfenq37uB2ChuFrtTF~-88r2mKNkeqdbtfKkcry1bMXhNXoHzmKc9geji4EbE45pLREroIRwxt~8B6lYjWNMxXsEscitrQd6mOlyL4Fz-LpHnRXKvN79uIb-Cpk~QMQql9Rc8wStfmTd6guoTEp4ySm9BQaph0wwAEGW7N7yjMFwPCF17JZklnDyLfpKmdP4-R99mpNlA8C1170hEz9XVjKBRZz6QI~gZYMm~eNTPDKQluzyLf85gWl47OJkvQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086103&ar=3868561&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx025f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The levels of gene expression determine the transcriptional plasticity of genes. We present the levels of expression as the logarithmic transformation of the Reads Per billion (RPKM) for duplicates (red boxes) and singletons (blue boxes) with transcriptional alterations and duplicates (orange boxes) and singletons (white boxes) without altered expressions when <em>S. cerevisiae</em> is faced with four different stress conditions (a) Ethanol, (b) glycerol, (c) lactate and (d) oxidative stress in a medium supplemented with dextrose. Significant differences in the expression levels of sets of genes are indicated as *, **, ***, when the probabilities are <em>P</em> &lt; 0.05, <em>P</em> &lt; 0.01 and <em>P</em> &lt; 10<sup>−6</sup>, respectively, using a Wilcoxon rank test.</p></div></div><div content-id="dsx025-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/m_dsx025f3.png?Expires=2147483647&Signature=tBslNyiZYF4XTWUK1P3feuDlzDkLToWHplmulfnz-lyiu128QNDClMBvrsRgYI4eD9Z0z4WlOz9VZoE3KGedWVkXQsNf0NEMlNHQTsyAFvWlJB6bKDdcXozyzSQGtfBA4Jm8a5ARWUnGTgcrrQJlNuarU0Em8zo-fn5hc~nkWGvFKAllXZjO4bcDA3IG-ahE04TIOTYkalORjC~R2nGC5ZYY5huci8StlEr2xssQnD5gGCc8e6CboquFT8jk~olevxTm9TrHu3-qWrQ~eHVkS0mjqPWKyBf5QJ9YxLB1NmSTjJBQLga1h-NR7X105cckS~~LKrtIMHmq3D9gnp9ijQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The levels of gene expression determine the transcriptional plasticity of genes. We present the levels of expression as the logarithmic transformation of the Reads Per billion (RPKM) for duplicates (red boxes) and singletons (blue boxes) with transcriptional alterations and duplicates (orange boxes) and singletons (white boxes) without altered expressions when S. cerevisiae is faced with four different stress conditions (a) Ethanol, (b) glycerol, (c) lactate and (d) oxidative stress in a medium supplemented with dextrose. Significant differences in the expression levels of sets of genes are indicated as *, **, ***, when the probabilities are P &lt; 0.05, P &lt; 0.01 and P &lt; 10−6, respectively, using a Wilcoxon rank test." path-from-xml="dsx025f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086103/dsx025f3.tif" path-from-xml="dsx025f3.tif" target="_blank">View large</a><a section="104086103" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/dsx025f3.png?Expires=2147483647&Signature=IfBDDQdrvpUUV~m8FRyqsclwWaDDmkkS3cFnR7~RGVd2VTNxV3CSZ1uluE5zvzJHiDA-fwSWRhHhOwIlosw1EfUwGfenq37uB2ChuFrtTF~-88r2mKNkeqdbtfKkcry1bMXhNXoHzmKc9geji4EbE45pLREroIRwxt~8B6lYjWNMxXsEscitrQd6mOlyL4Fz-LpHnRXKvN79uIb-Cpk~QMQql9Rc8wStfmTd6guoTEp4ySm9BQaph0wwAEGW7N7yjMFwPCF17JZklnDyLfpKmdP4-R99mpNlA8C1170hEz9XVjKBRZz6QI~gZYMm~eNTPDKQluzyLf85gWl47OJkvQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086103&ar=3868561&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx025f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The levels of gene expression determine the transcriptional plasticity of genes. We present the levels of expression as the logarithmic transformation of the Reads Per billion (RPKM) for duplicates (red boxes) and singletons (blue boxes) with transcriptional alterations and duplicates (orange boxes) and singletons (white boxes) without altered expressions when <em>S. cerevisiae</em> is faced with four different stress conditions (a) Ethanol, (b) glycerol, (c) lactate and (d) oxidative stress in a medium supplemented with dextrose. Significant differences in the expression levels of sets of genes are indicated as *, **, ***, when the probabilities are <em>P</em> &lt; 0.05, <em>P</em> &lt; 0.01 and <em>P</em> &lt; 10<sup>−6</sup>, respectively, using a Wilcoxon rank test.</p></div></div><p>We explored other mechanistic explanations for this expression difference between unaltered duplicates and altered singletons. One important factor that contributes to transcriptional plasticity is the existence of the TATA-box motif in the gene promoter, with TATA-containing genes being more sensitive to regulatory changes than TATA-less genes.<a reveal-id="dsx025-B49" data-open="dsx025-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> Importantly, the level of expression of TATA-containing genes was higher than that of TATA-less genes, and this was true for duplicates and singletons in <em>S. cerevisiae</em> (<a reveal-id="dsx025-F4" data-open="dsx025-F4" class="link link-reveal link-table xref-fig">Fig. 4a</a>). The set of transcriptionally altered genes under stress conditions was enriched for TATA-containing genes when compared to the set of genes with no transcriptional plasticity, being this true for duplicates (<a reveal-id="dsx025-F4" data-open="dsx025-F4" class="link link-reveal link-table xref-fig">Fig. 4b</a>) and singletons (<a reveal-id="dsx025-F4" data-open="dsx025-F4" class="link link-reveal link-table xref-fig">Fig. 4c</a>). Generally, TATA-containing genes also exhibit expression noise, which can be coupled with transcriptional plasticity provided that noise and plasticity are not in conflict.<a reveal-id="dsx025-B50" data-open="dsx025-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> Since gene duplication relaxes noise-plasticity conflict,<a reveal-id="dsx025-B50" data-open="dsx025-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> we expected duplicated genes to be enriched for TATA motifs when compared to singletons. Of the 1090 genes containing TATA-motifs, 558 belonged to duplicates (281 were WGDs and 277 were SSDs) (25% of all duplicates) and 532 to singletons (12% of all singletons) (<span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables10_dsx025.xls?Expires=2147483647&amp;Signature=debGbxRXYpaemCUHS4TWDVnJxtf4o8z9C4QjNxPvxzdAo0IOpIyzFUEL8pDQDq1bnnBY6ilAEYkIc0GhfB-DACWRRHoaMzTQ0EgHBL6OF-Y7oEKRPl9JRf0TrKobTpJXfhnOZkcpTD-jCql~hwxDrrVsK4GXg-g8725vQQPbeP783rs6uwubEtf4zsRrxaksiuNY4fGfTa10kbnjHyXc-kVVPQusJxXoR9vBRg6FfFB7em~m0aS349x-WFNmbY5k4zNJ7m29f1CvctqFnfhcS4HGjM-FonJZMijX7Na6A2dD0QaLCsrPL9Ckv35xaZa5~RTWExUOTis3WtOzGlj73w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>). Indeed, duplicated genes were more enriched for TATA-containing genes than singletons (Fisheŕs exact test: <em>F</em> = 2.52, <em>P</em> &lt; 2.2 × 10<sup>−16</sup>), and this was the case for both transcriptionally plastic genes and genes with no transcriptional plasticity (<span class="content-section supplementary-material"><a path-from-xml="sup28" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/figures1_dsx025.pdf?Expires=2147483647&amp;Signature=facQUC2BhmdpBrFQ5azyDi8U5YW-tOBKgyZ2xEuTjcQpc3kX0WubN-mflT6oGcChJmJbIpVPAgd6Wxya3DT8nEASg3yGKpUN~ZlNb0zkZRyhdLLNdzXf0GPnQadacKL-OJhqp3~qJBGliRKfnEHSaxiaKBH~tZUX7tycYycMehkjSsTodTCTgcmnEE7GTI4yiGfIb-TobrBhOHoVZBVQjyjx2asaPITZpSScyUBb74SnhUH30V2BjY-94nt2AE946Tm720WMhzbscD3c7bCmt-seeA1qzNxdTCTUKu3vKp7QY4mpp5lKE0UUrinrKal0-f8drIuhLW5UbgE80buifA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1a</a></span>). Remarkably, duplicates with no transcriptional plasticity were slightly more enriched for TATA-containing genes than singletons with transcriptional plasticity, with the difference being significant in the case of <em>S. cerevisiae</em> grown under oxidative stress (<span class="content-section supplementary-material"><a path-from-xml="sup28" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/figures1_dsx025.pdf?Expires=2147483647&amp;Signature=facQUC2BhmdpBrFQ5azyDi8U5YW-tOBKgyZ2xEuTjcQpc3kX0WubN-mflT6oGcChJmJbIpVPAgd6Wxya3DT8nEASg3yGKpUN~ZlNb0zkZRyhdLLNdzXf0GPnQadacKL-OJhqp3~qJBGliRKfnEHSaxiaKBH~tZUX7tycYycMehkjSsTodTCTgcmnEE7GTI4yiGfIb-TobrBhOHoVZBVQjyjx2asaPITZpSScyUBb74SnhUH30V2BjY-94nt2AE946Tm720WMhzbscD3c7bCmt-seeA1qzNxdTCTUKu3vKp7QY4mpp5lKE0UUrinrKal0-f8drIuhLW5UbgE80buifA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1b</a></span>). </p>                        <a id="104086105" scrollto-destination="104086105"></a>
<div id="dsx025-F4" data-id="dsx025-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/m_dsx025f4.png?Expires=2147483647&Signature=POxHw19RJ2XsHEGQpUbjATFhwJnHqgMs4gmTD7x2CkvxXa7w~5l4EhKdtA2iTb9bKDnT-pLWcctpkVr1RwuKmXFAUSJ8RlPnA6OFEnuSjL9~U0RBD2tIuu3fS1aWG32i-aoOu703ypvRUb8XBRTzKdbVa4GtU1~Or8BEHHm42sypBeWdXwn4cw0Jr5uuJ3SoUKWo7CItwxLm9~-3Se9ZgnIZhJXOLa883nQ3lvQWPMIQH5IPAfr51b7pe~vwqmBU2snfxHExkcQPpYnHq3J9d4SyBMCB-M4NvgMrxTJSCmrafIgk1smytw3UOHRz18vNcyrbeVZA-9jC3vNdVs-WlQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="TATA-containing genes exhibit greater expression levels and transcriptional plasticity than TATA-less genes. (a) Comparison of the expression levels (measured as the logarithmic transformation of the Fragment Reads Per billion) between TATA-containing and TATA-less genes for duplicates (red and orange boxes, respectively) and singletons (blue and white boxes, respectively). Differences were tested using the Wilcoxon rank test and significant differences are identified as * and **, for probabilities of P &lt; 0.05 and P &lt; 0.01, respectively. The percentage of genes with TATA-containing promoters for transcriptionally plastic and transcriptionally unaltered genes when S. cerevisiae is growing under four different stress conditions (ethanol, glycerol, lactate and oxidative stress in a medium supplemented with dextrose) was compared for duplicates (b) and for singletons (c). Significant differences between plastic and unaltered genes were identified using Fisheŕs exact test and are identified as ***, to indicate a probability of P &lt; 2.2 × 10−16." path-from-xml="dsx025f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086105/dsx025f4.tif" path-from-xml="dsx025f4.tif" target="_blank">View large</a><a section="104086105" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/dsx025f4.png?Expires=2147483647&Signature=vskd8XfTQgJr6300qF3SC4uRorZ6JoHqYqlWQIe-L8C5H~App-uu4NmE5m0wf8zWXgj4XIAd-ZnlkEXW9ke0DWPDPDq5xk1-4Vm0PD3rs77WR09xi9IJNKsM78VBPdZ0-rlTj-bauqlKkBJAQBNLxb1bN-OWR9WCnXk4a3LUxQsI3Qg63suV2vKvyNRv9451biIeX57iVwXSKQ-hzrkRttgQFlKdZAUmLA5g1kXvhzmZATf-Eyvy6VEZrSIKLaFKi3Xo~-pi0xKdlzR56YHWcIwPIl1iaAAWiF53-J~ut0In-UjuSdc95KGTMu-rOVO-WGPSVPIZ-SRA8IzgA2m2xA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086105&ar=3868561&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx025f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>TATA-containing genes exhibit greater expression levels and transcriptional plasticity than TATA-less genes. (a) Comparison of the expression levels (measured as the logarithmic transformation of the Fragment Reads Per billion) between TATA-containing and TATA-less genes for duplicates (red and orange boxes, respectively) and singletons (blue and white boxes, respectively). Differences were tested using the Wilcoxon rank test and significant differences are identified as * and **, for probabilities of <em>P</em> &lt; 0.05 and <em>P</em> &lt; 0.01, respectively. The percentage of genes with TATA-containing promoters for transcriptionally plastic and transcriptionally unaltered genes when <em>S. cerevisiae</em> is growing under four different stress conditions (ethanol, glycerol, lactate and oxidative stress in a medium supplemented with dextrose) was compared for duplicates (b) and for singletons (c). Significant differences between plastic and unaltered genes were identified using Fisheŕs exact test and are identified as ***, to indicate a probability of <em>P</em> &lt; 2.2 × 10<sup>−16</sup>.</p></div></div><div content-id="dsx025-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/m_dsx025f4.png?Expires=2147483647&Signature=POxHw19RJ2XsHEGQpUbjATFhwJnHqgMs4gmTD7x2CkvxXa7w~5l4EhKdtA2iTb9bKDnT-pLWcctpkVr1RwuKmXFAUSJ8RlPnA6OFEnuSjL9~U0RBD2tIuu3fS1aWG32i-aoOu703ypvRUb8XBRTzKdbVa4GtU1~Or8BEHHm42sypBeWdXwn4cw0Jr5uuJ3SoUKWo7CItwxLm9~-3Se9ZgnIZhJXOLa883nQ3lvQWPMIQH5IPAfr51b7pe~vwqmBU2snfxHExkcQPpYnHq3J9d4SyBMCB-M4NvgMrxTJSCmrafIgk1smytw3UOHRz18vNcyrbeVZA-9jC3vNdVs-WlQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="TATA-containing genes exhibit greater expression levels and transcriptional plasticity than TATA-less genes. (a) Comparison of the expression levels (measured as the logarithmic transformation of the Fragment Reads Per billion) between TATA-containing and TATA-less genes for duplicates (red and orange boxes, respectively) and singletons (blue and white boxes, respectively). Differences were tested using the Wilcoxon rank test and significant differences are identified as * and **, for probabilities of P &lt; 0.05 and P &lt; 0.01, respectively. The percentage of genes with TATA-containing promoters for transcriptionally plastic and transcriptionally unaltered genes when S. cerevisiae is growing under four different stress conditions (ethanol, glycerol, lactate and oxidative stress in a medium supplemented with dextrose) was compared for duplicates (b) and for singletons (c). Significant differences between plastic and unaltered genes were identified using Fisheŕs exact test and are identified as ***, to indicate a probability of P &lt; 2.2 × 10−16." path-from-xml="dsx025f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086105/dsx025f4.tif" path-from-xml="dsx025f4.tif" target="_blank">View large</a><a section="104086105" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/dsx025f4.png?Expires=2147483647&Signature=vskd8XfTQgJr6300qF3SC4uRorZ6JoHqYqlWQIe-L8C5H~App-uu4NmE5m0wf8zWXgj4XIAd-ZnlkEXW9ke0DWPDPDq5xk1-4Vm0PD3rs77WR09xi9IJNKsM78VBPdZ0-rlTj-bauqlKkBJAQBNLxb1bN-OWR9WCnXk4a3LUxQsI3Qg63suV2vKvyNRv9451biIeX57iVwXSKQ-hzrkRttgQFlKdZAUmLA5g1kXvhzmZATf-Eyvy6VEZrSIKLaFKi3Xo~-pi0xKdlzR56YHWcIwPIl1iaAAWiF53-J~ut0In-UjuSdc95KGTMu-rOVO-WGPSVPIZ-SRA8IzgA2m2xA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086105&ar=3868561&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx025f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>TATA-containing genes exhibit greater expression levels and transcriptional plasticity than TATA-less genes. (a) Comparison of the expression levels (measured as the logarithmic transformation of the Fragment Reads Per billion) between TATA-containing and TATA-less genes for duplicates (red and orange boxes, respectively) and singletons (blue and white boxes, respectively). Differences were tested using the Wilcoxon rank test and significant differences are identified as * and **, for probabilities of <em>P</em> &lt; 0.05 and <em>P</em> &lt; 0.01, respectively. The percentage of genes with TATA-containing promoters for transcriptionally plastic and transcriptionally unaltered genes when <em>S. cerevisiae</em> is growing under four different stress conditions (ethanol, glycerol, lactate and oxidative stress in a medium supplemented with dextrose) was compared for duplicates (b) and for singletons (c). Significant differences between plastic and unaltered genes were identified using Fisheŕs exact test and are identified as ***, to indicate a probability of <em>P</em> &lt; 2.2 × 10<sup>−16</sup>.</p></div></div><p>Finally, the magnitude of expression divergence between duplicates gene copies was correlated with the magnitude of transcriptional plasticity (measured as the fold change in expression of the most altered gene copy between YPD and stress) (<a reveal-id="dsx025-T1" data-open="dsx025-T1" class="link link-reveal link-table xref-fig">Table 1</a>), both of which are in turn correlated with the levels of gene expression. <div content-id="dsx025-T1" class="table-modal"><div class="table-wrap"><div id="dsx025-T1" data-id="dsx025-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Expression divergence between gene copies correlates with transcriptional plasticity of duplicates</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Stress source </th><th>Correlation (Spearman) </th><th>Probability </th></tr></thead><tbody><tr><td>Ethanol </td><td>0.17 </td><td>4.22 × 10<sup>−8</sup> </td></tr><tr><td>Glycerol </td><td>0.20 </td><td>1.78 × 10<sup>−10</sup> </td></tr><tr><td>Lactate </td><td>0.16 </td><td>4.13 × 10<sup>−7</sup> </td></tr><tr><td>Oxidative + dextrose </td><td>0.15 </td><td>7.11 × 10<sup>−7</sup> </td></tr></tbody></table></div><div content-id="dsx025-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Stress source </th><th>Correlation (Spearman) </th><th>Probability </th></tr></thead><tbody><tr><td>Ethanol </td><td>0.17 </td><td>4.22 × 10<sup>−8</sup> </td></tr><tr><td>Glycerol </td><td>0.20 </td><td>1.78 × 10<sup>−10</sup> </td></tr><tr><td>Lactate </td><td>0.16 </td><td>4.13 × 10<sup>−7</sup> </td></tr><tr><td>Oxidative + dextrose </td><td>0.15 </td><td>7.11 × 10<sup>−7</sup> </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsx025-T1" data-id="dsx025-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Expression divergence between gene copies correlates with transcriptional plasticity of duplicates</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Stress source </th><th>Correlation (Spearman) </th><th>Probability </th></tr></thead><tbody><tr><td>Ethanol </td><td>0.17 </td><td>4.22 × 10<sup>−8</sup> </td></tr><tr><td>Glycerol </td><td>0.20 </td><td>1.78 × 10<sup>−10</sup> </td></tr><tr><td>Lactate </td><td>0.16 </td><td>4.13 × 10<sup>−7</sup> </td></tr><tr><td>Oxidative + dextrose </td><td>0.15 </td><td>7.11 × 10<sup>−7</sup> </td></tr></tbody></table></div><div content-id="dsx025-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Stress source </th><th>Correlation (Spearman) </th><th>Probability </th></tr></thead><tbody><tr><td>Ethanol </td><td>0.17 </td><td>4.22 × 10<sup>−8</sup> </td></tr><tr><td>Glycerol </td><td>0.20 </td><td>1.78 × 10<sup>−10</sup> </td></tr><tr><td>Lactate </td><td>0.16 </td><td>4.13 × 10<sup>−7</sup> </td></tr><tr><td>Oxidative + dextrose </td><td>0.15 </td><td>7.11 × 10<sup>−7</sup> </td></tr></tbody></table></div></div></p>                            <h3 scrollto-destination=104086107 id="104086107" class="section-title" >3.4. The levels of gene expression correlate with the patterns of duplicates transcriptional plasticity</h3>
<p>A prediction of the dosage sub-functionalization hypothesis is that the patterns of duplicates transcriptional plasticity should be dependent on the levels of gene expression. For instance, transcriptionally plastic genes that are lowly expressed under normal conditions should only be able to over-express under stress because a decline in their expression could drive one of the gene copies to non-functionalization due to relaxed selective constraints. Because plasticity is often correlated with expression noise<a reveal-id="dsx025-B50" data-open="dsx025-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> and, since surviving duplicates are those whose expression noise falls within the range of expression detectable by selection, expression noise should depend on the levels of duplicates expression. We divided duplicated genes according to the patterns of transcriptional plasticity they show when <em>S. cerevisiae</em> is grown under stress: (i) up-regulated: when the two gene copies were up-regulated under stress; (ii) down-regulated: when the two gene copies were down-regulated under stress; (iii) discordant: when one copy was up-regulated and the other was down-regulated under stress; and (iv) one-altered: when one copy was not altered but its paralogous copy was either up-regulated or down-regulated under stress (<span class="content-section supplementary-material"><a path-from-xml="sup11" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables11_dsx025.xls?Expires=2147483647&amp;Signature=SGZwYk-31ltXxPadHG1Ia4NSpSn-XqbXexl8eXzkIJ7FWbKDa3XDbB5I-BCSAqyRKh16DcQaZrQQq0sV0HGe6dn-LqZqsauLLjkMoSC-~Vv7ZhgFmOGluvPlse9miT6-RLe-MfMt~AfsRvPe~eN~6uLX~2XMVQal~Rs~~Qh-Bs7alLX6Uyc8EGpUd1iR54~xmZ1lFyfLAmskB-1ar6aR-KT4u77SNeZW85G64srY0aXTQhJhuTxiCDK4WApBP6yn1P~sTSGs1JZLwrvtpK3495a6ekafQbucUBNNYZh3qu8UYWED3dHcrVoySXzs8oILlDE5WIsszGT1bSRJ6Exc5Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S11</a></span>–<span class="content-section supplementary-material"><a path-from-xml="sup26" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables26_dsx025.xls?Expires=2147483647&amp;Signature=v~YbrHGzefKER~B-yzFxe0~XsVrWGLvcx0iBC7GC6lws7l4xPgDdVRwrF1TNP2c8YnR~JRO4QtyWbu4oq7fJi-RtRpwY5g9Rx9s9rMXzyLIZGRMyiWnddxxyh0ncDZtal4nypYpVrC35fl32rI5mr4JTYdiBN5YOCwslqbVWjVCPRciHZerf42l9OY~HjEjRuJcayrmcv6HPtOC~Jmc19hLYzGr7-vdxA5reWv-6jHAihzTIU16SgkT-Cz8Y997GdoQttNxTIMqIJx5n-Eiq0G8XRYFSdFQa8emqRoeOJxUPvq5s9gi9ppw8desdYoz0OgaZ0b320U-tkS9u~QfKTw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S26</a></span>). In all four stress conditions, duplicates that were down-regulated were also those that exhibited the highest expression levels under normal conditions, being these followed by duplicates in which only one copy exhibits transcriptional plasticity, then discordant duplicates and finally up-regulated duplicates (<a reveal-id="dsx025-F5" data-open="dsx025-F5" class="link link-reveal link-table xref-fig">Fig. 5a–d</a>). </p>                        <a id="104086109" scrollto-destination="104086109"></a>
<div id="dsx025-F5" data-id="dsx025-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/m_dsx025f5.png?Expires=2147483647&Signature=S6DNTQ6fYHsNqNfcpRjEe0RkFBsbENBiSln7QBmL4ZDOUObu~fqcXyHsLOQem1nAUANwPlcnGJRZz5Ko9w-3~q2TN9voclLLkakptYZ4QwiZfKNUXxbj3mfq-tBqj6Z8GivFT8bhzglodamKEyNba7qeSN6Q~ldCaB9aTs45tCqUE4bzgn0XVaIaABZ04hd~FaPxvStE5Oa5Gxu1deFKXvgqXJOZ19CqUmEGM8ZKw5Sa0ud5mzCYEmQvHcnWBJ4zLxT-z0fwAOucy73wMZPtudEzZ3ndAO8SNsVE0nksWdasnu43Cw5TJFrQn8aaKzzACQgduzRjI45NshgIm46nVQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The expression level of duplicates correlated with their patterns of expression divergence. Expression was measured as the logarithmic transformation of the Reads Per billion (RPKM). Significant differences in the expression levels of gene sets are indicated as *, ** and ***, referring to probabilities of P &lt; 0.05, P &lt; 0.01 and P &lt; 10−6, using a Wilcoxon rank test. (a) Expression under ethanol stress, (b) expression under glycerol stress, (c) expression under lactate stress and (d) expression under oxidative stress." path-from-xml="dsx025f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086109/dsx025f5.tif" path-from-xml="dsx025f5.tif" target="_blank">View large</a><a section="104086109" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/dsx025f5.png?Expires=2147483647&Signature=sGv-uZVqNmDHqnq4SlTXbHZPADco61tP-xdCpcE7DI-q-YXRa6p-wia0VVDCg7r6uJuUMMorOWPFlrZ2ALoNOVw78Ro~q9EvJ1v6NPeJ02y6BW8RHadCzWGXTC3olzeHPORsiSyXhfuepfzppfM-XDGMa-wFPC-EwusKrWY2q2Y1JE78s31zHKd9hxjFsOAUXvvptc~cL2NRjSo7Pk5qrZfAGLlJdAhEHOPIsWPoAuW2IOyk38-1Y1~F5snbrkRI-bUAo8KeWZA80MGSs5LFVPI2yJKIC2TZvUsC5EuH2xMsVkPnNN8fqQbY6jiWHAAvqsJslcgPAHDWtOvGVRwd4g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086109&ar=3868561&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx025f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The expression level of duplicates correlated with their patterns of expression divergence. Expression was measured as the logarithmic transformation of the Reads Per billion (RPKM). Significant differences in the expression levels of gene sets are indicated as *, ** and ***, referring to probabilities of <em>P</em> &lt; 0.05, <em>P</em> &lt; 0.01 and <em>P</em> &lt; 10<sup>−6</sup>, using a Wilcoxon rank test. (a) Expression under ethanol stress, (b) expression under glycerol stress, (c) expression under lactate stress and (d) expression under oxidative stress.</p></div></div><div content-id="dsx025-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/m_dsx025f5.png?Expires=2147483647&Signature=S6DNTQ6fYHsNqNfcpRjEe0RkFBsbENBiSln7QBmL4ZDOUObu~fqcXyHsLOQem1nAUANwPlcnGJRZz5Ko9w-3~q2TN9voclLLkakptYZ4QwiZfKNUXxbj3mfq-tBqj6Z8GivFT8bhzglodamKEyNba7qeSN6Q~ldCaB9aTs45tCqUE4bzgn0XVaIaABZ04hd~FaPxvStE5Oa5Gxu1deFKXvgqXJOZ19CqUmEGM8ZKw5Sa0ud5mzCYEmQvHcnWBJ4zLxT-z0fwAOucy73wMZPtudEzZ3ndAO8SNsVE0nksWdasnu43Cw5TJFrQn8aaKzzACQgduzRjI45NshgIm46nVQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The expression level of duplicates correlated with their patterns of expression divergence. Expression was measured as the logarithmic transformation of the Reads Per billion (RPKM). Significant differences in the expression levels of gene sets are indicated as *, ** and ***, referring to probabilities of P &lt; 0.05, P &lt; 0.01 and P &lt; 10−6, using a Wilcoxon rank test. (a) Expression under ethanol stress, (b) expression under glycerol stress, (c) expression under lactate stress and (d) expression under oxidative stress." path-from-xml="dsx025f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086109/dsx025f5.tif" path-from-xml="dsx025f5.tif" target="_blank">View large</a><a section="104086109" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/dsx025f5.png?Expires=2147483647&Signature=sGv-uZVqNmDHqnq4SlTXbHZPADco61tP-xdCpcE7DI-q-YXRa6p-wia0VVDCg7r6uJuUMMorOWPFlrZ2ALoNOVw78Ro~q9EvJ1v6NPeJ02y6BW8RHadCzWGXTC3olzeHPORsiSyXhfuepfzppfM-XDGMa-wFPC-EwusKrWY2q2Y1JE78s31zHKd9hxjFsOAUXvvptc~cL2NRjSo7Pk5qrZfAGLlJdAhEHOPIsWPoAuW2IOyk38-1Y1~F5snbrkRI-bUAo8KeWZA80MGSs5LFVPI2yJKIC2TZvUsC5EuH2xMsVkPnNN8fqQbY6jiWHAAvqsJslcgPAHDWtOvGVRwd4g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086109&ar=3868561&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx025f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The expression level of duplicates correlated with their patterns of expression divergence. Expression was measured as the logarithmic transformation of the Reads Per billion (RPKM). Significant differences in the expression levels of gene sets are indicated as *, ** and ***, referring to probabilities of <em>P</em> &lt; 0.05, <em>P</em> &lt; 0.01 and <em>P</em> &lt; 10<sup>−6</sup>, using a Wilcoxon rank test. (a) Expression under ethanol stress, (b) expression under glycerol stress, (c) expression under lactate stress and (d) expression under oxidative stress.</p></div></div><p>Interestingly, the mean level of expression under normal conditions of duplicates gene copies was correlated with the level of expression divergence of the gene copies under normal conditions in those duplicates that belong to the category discordant and one-altered, those categories with the highest expression divergence between gene copies, but not in those pairs in which both copies were either up-regulated or down-regulated (<a reveal-id="dsx025-F6" data-open="dsx025-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). </p>                        <a id="104086111" scrollto-destination="104086111"></a>
<div id="dsx025-F6" data-id="dsx025-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/m_dsx025f6.png?Expires=2147483647&Signature=jFmLv05pIf~hqFpsKYhZS7J36CUiXIBD9DnVhLz6eS23np99BtFdbSxpm~qr1Jc4C1O7wArlB1b7EWGj6jfjCtce~18loT2fKg2wdKNRo0mbi8Sk1Ls4f0X7ahD~bDNF-UuarJCA5Jl4gRXrCnEuhd6tDPt~v3PZVwnuFmt8h-8-bPuFxut7tqm~B0MI-kt6tmuD5qzL2kyfa~1ouqzXJVpb9eloIF5UGDlA3trcwGsp2fHktRVdOjRaM8nr7KYcTiSnxnVVZaydhatDSgdcQ1rnyOyOxAnXnSEL0KReaOjPEOINatVwyWsNvCld2cyGWPUmUAmCpt2OAQARXS9VwA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The levels of gene expression correlate with the level of expression divergence in duplicates in which gene copies are either discordant (one copy up-regulated and the other copy down-regulated) or one copy only altered under stress. Transcriptional alteration patterns of duplicates were measured in four stress conditions (ethanol, glycerol, lactate and oxidative stress in a medium supplemented with dextrose). Expression divergence of duplicates was measured as 1 – (the expression of the lowest expressed copy divided by the expression of the highest expressed copy). Spearman correlation (ρ) between the levels of gene expression (measured as the logarithmic transformation of the Reads Per billion (RPKM)) and the levels of expression divergence was tested for significance. Significant plots are indicated in bold. We found significant associations between the levels of gene expression and the expression divergence between gene copies of discordant duplicates and those with one copy altered but not for up-regulated or down-regulated duplicates." path-from-xml="dsx025f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086111/dsx025f6.tif" path-from-xml="dsx025f6.tif" target="_blank">View large</a><a section="104086111" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/dsx025f6.png?Expires=2147483647&Signature=G-UVwiNRKUzp8FUzbxBGupVKE-zP-goK9a6stTqXNAG7RRXlGUAL2Ux2KJ~T1HOl3h~yXdPF1i1XJYTIZP42n7yqLHrIQfJD32aOG172nvPBmmhDbXkF3o3i0QujZivASQNZXE8UZ~2vFhkCws0JLNDJjx7txfWUFILPc5BeDRtUhvoclrqG6coG2FXQqnXJDusJaDeNzy8H0OMs5htDMlcV7tBGLTwLD4fVOvuUBVAehyB05oPazkdS-~fI-JLfBp~uQd7TOWHcWMkPDTA4Pm8jypWXAGw5gDSXfT22PL6-AM5s-Z~oG9Gye5kqOfQUMyD99nreAONnphxforG57Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086111&ar=3868561&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx025f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The levels of gene expression correlate with the level of expression divergence in duplicates in which gene copies are either discordant (one copy up-regulated and the other copy down-regulated) or one copy only altered under stress. Transcriptional alteration patterns of duplicates were measured in four stress conditions (ethanol, glycerol, lactate and oxidative stress in a medium supplemented with dextrose). Expression divergence of duplicates was measured as 1 – (the expression of the lowest expressed copy divided by the expression of the highest expressed copy). Spearman correlation (<em>ρ</em>) between the levels of gene expression (measured as the logarithmic transformation of the Reads Per billion (RPKM)) and the levels of expression divergence was tested for significance. Significant plots are indicated in bold. We found significant associations between the levels of gene expression and the expression divergence between gene copies of discordant duplicates and those with one copy altered but not for up-regulated or down-regulated duplicates.</p></div></div><div content-id="dsx025-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/m_dsx025f6.png?Expires=2147483647&Signature=jFmLv05pIf~hqFpsKYhZS7J36CUiXIBD9DnVhLz6eS23np99BtFdbSxpm~qr1Jc4C1O7wArlB1b7EWGj6jfjCtce~18loT2fKg2wdKNRo0mbi8Sk1Ls4f0X7ahD~bDNF-UuarJCA5Jl4gRXrCnEuhd6tDPt~v3PZVwnuFmt8h-8-bPuFxut7tqm~B0MI-kt6tmuD5qzL2kyfa~1ouqzXJVpb9eloIF5UGDlA3trcwGsp2fHktRVdOjRaM8nr7KYcTiSnxnVVZaydhatDSgdcQ1rnyOyOxAnXnSEL0KReaOjPEOINatVwyWsNvCld2cyGWPUmUAmCpt2OAQARXS9VwA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The levels of gene expression correlate with the level of expression divergence in duplicates in which gene copies are either discordant (one copy up-regulated and the other copy down-regulated) or one copy only altered under stress. Transcriptional alteration patterns of duplicates were measured in four stress conditions (ethanol, glycerol, lactate and oxidative stress in a medium supplemented with dextrose). Expression divergence of duplicates was measured as 1 – (the expression of the lowest expressed copy divided by the expression of the highest expressed copy). Spearman correlation (ρ) between the levels of gene expression (measured as the logarithmic transformation of the Reads Per billion (RPKM)) and the levels of expression divergence was tested for significance. Significant plots are indicated in bold. We found significant associations between the levels of gene expression and the expression divergence between gene copies of discordant duplicates and those with one copy altered but not for up-regulated or down-regulated duplicates." path-from-xml="dsx025f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086111/dsx025f6.tif" path-from-xml="dsx025f6.tif" target="_blank">View large</a><a section="104086111" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/dsx025f6.png?Expires=2147483647&Signature=G-UVwiNRKUzp8FUzbxBGupVKE-zP-goK9a6stTqXNAG7RRXlGUAL2Ux2KJ~T1HOl3h~yXdPF1i1XJYTIZP42n7yqLHrIQfJD32aOG172nvPBmmhDbXkF3o3i0QujZivASQNZXE8UZ~2vFhkCws0JLNDJjx7txfWUFILPc5BeDRtUhvoclrqG6coG2FXQqnXJDusJaDeNzy8H0OMs5htDMlcV7tBGLTwLD4fVOvuUBVAehyB05oPazkdS-~fI-JLfBp~uQd7TOWHcWMkPDTA4Pm8jypWXAGw5gDSXfT22PL6-AM5s-Z~oG9Gye5kqOfQUMyD99nreAONnphxforG57Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086111&ar=3868561&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx025f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The levels of gene expression correlate with the level of expression divergence in duplicates in which gene copies are either discordant (one copy up-regulated and the other copy down-regulated) or one copy only altered under stress. Transcriptional alteration patterns of duplicates were measured in four stress conditions (ethanol, glycerol, lactate and oxidative stress in a medium supplemented with dextrose). Expression divergence of duplicates was measured as 1 – (the expression of the lowest expressed copy divided by the expression of the highest expressed copy). Spearman correlation (<em>ρ</em>) between the levels of gene expression (measured as the logarithmic transformation of the Reads Per billion (RPKM)) and the levels of expression divergence was tested for significance. Significant plots are indicated in bold. We found significant associations between the levels of gene expression and the expression divergence between gene copies of discordant duplicates and those with one copy altered but not for up-regulated or down-regulated duplicates.</p></div></div>                            <h3 scrollto-destination=104086112 id="104086112" class="section-title" >3.5. Gene duplication has contributed to increased transcriptional plasticity in yeast</h3>
<p>We examined the link between gene duplication and phenotypic plasticity by comparing the transcriptomes of <em>S. cerevisiae</em> grown under a number of key stress conditions that this species faces in nature to those transcriptomes of <em>S. cerevisiae</em> grown under normal YPD media (section Material and methods). Transcriptionally altered genes were more enriched for duplicates than for singleton genes over all stress conditions (<a reveal-id="dsx025-F7" data-open="dsx025-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>). This trend was also true when we compared WGDs to singletons and SSDs to singletons (<a reveal-id="dsx025-F7" data-open="dsx025-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>). To determine whether transcriptional plasticity is directly linked to gene duplication, we examined the transcriptional plasticity of <em>S. cerevisiae</em> duplicates and singletons orthologs in the pre-WGD species <em>L. kluyveri</em>. The transcription of <em>L. kluyveri</em> genes was previously assessed in 19 different stress conditions.<a reveal-id="dsx025-B35" data-open="dsx025-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> For each condition, we sought the percentage of genes that were orthologs to <em>S. cerevisiae</em> duplicates (<em>N </em>=<em> </em>1469) and singletons (<em>N </em>=<em> </em>4174) that exhibited transcriptional alteration. In 18 of the 19 conditions, there was no significant difference in the percentage of transcriptionally altered genes under stress between the orthologs of <em>S. cerevisiae</em> duplicates and those of singleton (<a reveal-id="dsx025-T2" data-open="dsx025-T2" class="link link-reveal link-table xref-fig">Table 2</a>). The only exception was SDS stress, in which the percentage of transcriptionally altered orthologs for <em>S. cerevisiae</em> duplicates was higher than that for transcriptionally altered singleton orthologs (Fisheŕs exact test: odds ratio <em>F</em> = 1.22, <em>P </em>= 5 × 10<sup>−3</sup>, <a reveal-id="dsx025-T2" data-open="dsx025-T2" class="link link-reveal link-table xref-fig">Table 2</a>). In all other stresses that were equivalent to the ones used in our experiments (e.g. glycerol, ethanol), there was no significant difference in the number of transcriptionally altered genes between orthologs of <em>S. cerevisiae</em> duplicates and singletons (<a reveal-id="dsx025-T2" data-open="dsx025-T2" class="link link-reveal link-table xref-fig">Table 2</a>). These data indicate that the high transcriptional plasticity of duplicates in <em>S. cerevisiae</em> was acquired after gene duplication. <div content-id="dsx025-T2" class="table-modal"><div class="table-wrap"><div id="dsx025-T2" data-id="dsx025-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Transcriptional alterations of <em>L. kluyveri</em> genes orthologs of <em>S. cerevisiae</em> under nineteen different stress conditions</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Stress </th><th># duplicates orthologs (%) </th><th># singletons orthologs (%) </th><th>Odds’ ratio <em>F</em> </th><th><em>P</em> </th></tr></thead><tbody><tr><td>Galactose </td><td>588 (40.1) </td><td>1586 (37.6) </td><td>0.93 </td><td>0.24 </td></tr><tr><td>Glycerol </td><td>865 (58.8) </td><td>2378 (56.9) </td><td>1.08 </td><td>0.21 </td></tr><tr><td>23 °C </td><td>178 (12.1) </td><td>540 (12.9) </td><td>0.92 </td><td>0.44 </td></tr><tr><td>37 °C </td><td>458 (31.2) </td><td>1282 (30.7) </td><td>1.02 </td><td>0.74 </td></tr><tr><td>YNB </td><td>487 (33.2) </td><td>1366 (32.7) </td><td>1.02 </td><td>0.77 </td></tr><tr><td>Ethanol </td><td>609 (41.5) </td><td>1645 (39.4) </td><td>1.09 </td><td>0.17 </td></tr><tr><td>Methanol </td><td>291 (19.8) </td><td>865 (20.7) </td><td>0.95 </td><td>0.49 </td></tr><tr><td>SDS </td><td>410 (27.9) </td><td>1008 (24.1) </td><td>1.22 </td><td>0.005 </td></tr><tr><td>DMSO </td><td>654 (44.5) </td><td>1758 (42.1) </td><td>1.10 </td><td>0.11 </td></tr><tr><td>NaCl </td><td>234 (15.9) </td><td>757 (18.1) </td><td>0.86 </td><td>0.06 </td></tr><tr><td>CaCl<sub>2</sub> </td><td>874 (59.5) </td><td>2499 (59.9) </td><td>0.98 </td><td>0.80 </td></tr><tr><td>NiSO<sub>4</sub> </td><td>129 (8.8) </td><td>332 (7.9) </td><td>1.11 </td><td>0.32 </td></tr><tr><td>LiCl </td><td>253 (17.2) </td><td>737 (17.7) </td><td>0.97 </td><td>0.72 </td></tr><tr><td>CoSO<sub>4</sub> </td><td>825 (56.2) </td><td>2362 (56.6) </td><td>0.99 </td><td>0.85 </td></tr><tr><td>BME </td><td>375 (25.6) </td><td>1102 (26.4) </td><td>0.96 </td><td>0.53 </td></tr><tr><td>5FU </td><td>298 (20.3) </td><td>922 (22.1) </td><td>0.89 </td><td>0.15 </td></tr><tr><td>Arsenic </td><td>127 (8.6) </td><td>343 (8.2) </td><td>1.06 </td><td>0.62 </td></tr><tr><td>6AU </td><td>230 (15.7) </td><td>674 (16.1) </td><td>0.96 </td><td>0.68 </td></tr><tr><td>Fluconazole </td><td>437 (29.7) </td><td>1195 (28.6) </td><td>1.06 </td><td>0.42 </td></tr></tbody></table></div><div content-id="dsx025-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Stress </th><th># duplicates orthologs (%) </th><th># singletons orthologs (%) </th><th>Odds’ ratio <em>F</em> </th><th><em>P</em> </th></tr></thead><tbody><tr><td>Galactose </td><td>588 (40.1) </td><td>1586 (37.6) </td><td>0.93 </td><td>0.24 </td></tr><tr><td>Glycerol </td><td>865 (58.8) </td><td>2378 (56.9) </td><td>1.08 </td><td>0.21 </td></tr><tr><td>23 °C </td><td>178 (12.1) </td><td>540 (12.9) </td><td>0.92 </td><td>0.44 </td></tr><tr><td>37 °C </td><td>458 (31.2) </td><td>1282 (30.7) </td><td>1.02 </td><td>0.74 </td></tr><tr><td>YNB </td><td>487 (33.2) </td><td>1366 (32.7) </td><td>1.02 </td><td>0.77 </td></tr><tr><td>Ethanol </td><td>609 (41.5) </td><td>1645 (39.4) </td><td>1.09 </td><td>0.17 </td></tr><tr><td>Methanol </td><td>291 (19.8) </td><td>865 (20.7) </td><td>0.95 </td><td>0.49 </td></tr><tr><td>SDS </td><td>410 (27.9) </td><td>1008 (24.1) </td><td>1.22 </td><td>0.005 </td></tr><tr><td>DMSO </td><td>654 (44.5) </td><td>1758 (42.1) </td><td>1.10 </td><td>0.11 </td></tr><tr><td>NaCl </td><td>234 (15.9) </td><td>757 (18.1) </td><td>0.86 </td><td>0.06 </td></tr><tr><td>CaCl<sub>2</sub> </td><td>874 (59.5) </td><td>2499 (59.9) </td><td>0.98 </td><td>0.80 </td></tr><tr><td>NiSO<sub>4</sub> </td><td>129 (8.8) </td><td>332 (7.9) </td><td>1.11 </td><td>0.32 </td></tr><tr><td>LiCl </td><td>253 (17.2) </td><td>737 (17.7) </td><td>0.97 </td><td>0.72 </td></tr><tr><td>CoSO<sub>4</sub> </td><td>825 (56.2) </td><td>2362 (56.6) </td><td>0.99 </td><td>0.85 </td></tr><tr><td>BME </td><td>375 (25.6) </td><td>1102 (26.4) </td><td>0.96 </td><td>0.53 </td></tr><tr><td>5FU </td><td>298 (20.3) </td><td>922 (22.1) </td><td>0.89 </td><td>0.15 </td></tr><tr><td>Arsenic </td><td>127 (8.6) </td><td>343 (8.2) </td><td>1.06 </td><td>0.62 </td></tr><tr><td>6AU </td><td>230 (15.7) </td><td>674 (16.1) </td><td>0.96 </td><td>0.68 </td></tr><tr><td>Fluconazole </td><td>437 (29.7) </td><td>1195 (28.6) </td><td>1.06 </td><td>0.42 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsx025-T2" data-id="dsx025-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Transcriptional alterations of <em>L. kluyveri</em> genes orthologs of <em>S. cerevisiae</em> under nineteen different stress conditions</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Stress </th><th># duplicates orthologs (%) </th><th># singletons orthologs (%) </th><th>Odds’ ratio <em>F</em> </th><th><em>P</em> </th></tr></thead><tbody><tr><td>Galactose </td><td>588 (40.1) </td><td>1586 (37.6) </td><td>0.93 </td><td>0.24 </td></tr><tr><td>Glycerol </td><td>865 (58.8) </td><td>2378 (56.9) </td><td>1.08 </td><td>0.21 </td></tr><tr><td>23 °C </td><td>178 (12.1) </td><td>540 (12.9) </td><td>0.92 </td><td>0.44 </td></tr><tr><td>37 °C </td><td>458 (31.2) </td><td>1282 (30.7) </td><td>1.02 </td><td>0.74 </td></tr><tr><td>YNB </td><td>487 (33.2) </td><td>1366 (32.7) </td><td>1.02 </td><td>0.77 </td></tr><tr><td>Ethanol </td><td>609 (41.5) </td><td>1645 (39.4) </td><td>1.09 </td><td>0.17 </td></tr><tr><td>Methanol </td><td>291 (19.8) </td><td>865 (20.7) </td><td>0.95 </td><td>0.49 </td></tr><tr><td>SDS </td><td>410 (27.9) </td><td>1008 (24.1) </td><td>1.22 </td><td>0.005 </td></tr><tr><td>DMSO </td><td>654 (44.5) </td><td>1758 (42.1) </td><td>1.10 </td><td>0.11 </td></tr><tr><td>NaCl </td><td>234 (15.9) </td><td>757 (18.1) </td><td>0.86 </td><td>0.06 </td></tr><tr><td>CaCl<sub>2</sub> </td><td>874 (59.5) </td><td>2499 (59.9) </td><td>0.98 </td><td>0.80 </td></tr><tr><td>NiSO<sub>4</sub> </td><td>129 (8.8) </td><td>332 (7.9) </td><td>1.11 </td><td>0.32 </td></tr><tr><td>LiCl </td><td>253 (17.2) </td><td>737 (17.7) </td><td>0.97 </td><td>0.72 </td></tr><tr><td>CoSO<sub>4</sub> </td><td>825 (56.2) </td><td>2362 (56.6) </td><td>0.99 </td><td>0.85 </td></tr><tr><td>BME </td><td>375 (25.6) </td><td>1102 (26.4) </td><td>0.96 </td><td>0.53 </td></tr><tr><td>5FU </td><td>298 (20.3) </td><td>922 (22.1) </td><td>0.89 </td><td>0.15 </td></tr><tr><td>Arsenic </td><td>127 (8.6) </td><td>343 (8.2) </td><td>1.06 </td><td>0.62 </td></tr><tr><td>6AU </td><td>230 (15.7) </td><td>674 (16.1) </td><td>0.96 </td><td>0.68 </td></tr><tr><td>Fluconazole </td><td>437 (29.7) </td><td>1195 (28.6) </td><td>1.06 </td><td>0.42 </td></tr></tbody></table></div><div content-id="dsx025-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Stress </th><th># duplicates orthologs (%) </th><th># singletons orthologs (%) </th><th>Odds’ ratio <em>F</em> </th><th><em>P</em> </th></tr></thead><tbody><tr><td>Galactose </td><td>588 (40.1) </td><td>1586 (37.6) </td><td>0.93 </td><td>0.24 </td></tr><tr><td>Glycerol </td><td>865 (58.8) </td><td>2378 (56.9) </td><td>1.08 </td><td>0.21 </td></tr><tr><td>23 °C </td><td>178 (12.1) </td><td>540 (12.9) </td><td>0.92 </td><td>0.44 </td></tr><tr><td>37 °C </td><td>458 (31.2) </td><td>1282 (30.7) </td><td>1.02 </td><td>0.74 </td></tr><tr><td>YNB </td><td>487 (33.2) </td><td>1366 (32.7) </td><td>1.02 </td><td>0.77 </td></tr><tr><td>Ethanol </td><td>609 (41.5) </td><td>1645 (39.4) </td><td>1.09 </td><td>0.17 </td></tr><tr><td>Methanol </td><td>291 (19.8) </td><td>865 (20.7) </td><td>0.95 </td><td>0.49 </td></tr><tr><td>SDS </td><td>410 (27.9) </td><td>1008 (24.1) </td><td>1.22 </td><td>0.005 </td></tr><tr><td>DMSO </td><td>654 (44.5) </td><td>1758 (42.1) </td><td>1.10 </td><td>0.11 </td></tr><tr><td>NaCl </td><td>234 (15.9) </td><td>757 (18.1) </td><td>0.86 </td><td>0.06 </td></tr><tr><td>CaCl<sub>2</sub> </td><td>874 (59.5) </td><td>2499 (59.9) </td><td>0.98 </td><td>0.80 </td></tr><tr><td>NiSO<sub>4</sub> </td><td>129 (8.8) </td><td>332 (7.9) </td><td>1.11 </td><td>0.32 </td></tr><tr><td>LiCl </td><td>253 (17.2) </td><td>737 (17.7) </td><td>0.97 </td><td>0.72 </td></tr><tr><td>CoSO<sub>4</sub> </td><td>825 (56.2) </td><td>2362 (56.6) </td><td>0.99 </td><td>0.85 </td></tr><tr><td>BME </td><td>375 (25.6) </td><td>1102 (26.4) </td><td>0.96 </td><td>0.53 </td></tr><tr><td>5FU </td><td>298 (20.3) </td><td>922 (22.1) </td><td>0.89 </td><td>0.15 </td></tr><tr><td>Arsenic </td><td>127 (8.6) </td><td>343 (8.2) </td><td>1.06 </td><td>0.62 </td></tr><tr><td>6AU </td><td>230 (15.7) </td><td>674 (16.1) </td><td>0.96 </td><td>0.68 </td></tr><tr><td>Fluconazole </td><td>437 (29.7) </td><td>1195 (28.6) </td><td>1.06 </td><td>0.42 </td></tr></tbody></table></div></div></p>                        <a id="104086114" scrollto-destination="104086114"></a>
<div id="dsx025-F7" data-id="dsx025-F7" class="fig fig-section"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/m_dsx025f7.png?Expires=2147483647&Signature=zPjcLucDffbG0ZU-EtFC27aki~G2t9GZTk-nsYMQwxRf2CapvM8Uu2j14ZHQEybXGP6Y8T2wKgO01wdbOgMQRaBXNnMSPICbX5-~Hry8fAMPm6wjv-qZ~t4BjgrBCZvbzE~Y1kZ7DB-WB0qnk~jOSiXSiW3wAYfJ4IGzJIujRduGn~3zppV1VN0pRBAxzHpnopM3y8Gz~nTo-DwdNhy-6KrIhzjdN4ff0oCPonNBtlkIrVw8z-02m4AFPRJg4~Oe-i6CmKuRU24r6L~v1E9mL3v0Os~ipoXlz6~mv7rWZdTl9iSbjrlWfG1d5GmjgU4m7CBkQmBdo4VfDMZYPd25OQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Duplicates are more enriched for transcriptional plasticity than singletons, regardless of the mechanism of duplication. (a) Transcriptional plasticity of duplicates and singletons in S. cerevisiae. Y-axis represents the mean percentage of genes with altered transcription across four stress conditions (ethanol, glycerol, lactate and oxidative stress in a medium supplemented with dextrose). We also compared WGDs and SSDs to singletons. Significant differences were found using Fisher’s exact test, with *** indicating a probability of P &lt; 2.2 × 10−16. (b) Venn diagram representing the overlap in the number of duplicates and singletons between C. glabrata and S. cerevisiae." path-from-xml="dsx025f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086114/dsx025f7.tif" path-from-xml="dsx025f7.tif" target="_blank">View large</a><a section="104086114" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/dsx025f7.png?Expires=2147483647&Signature=x906ez~7cQUE~tWWb1DmCyLMFJ2OVsjKEcjZ~dt64K9tyX9hs1KrmslbxEHJeSktrFgUWW5T9CtfSXqG6Eu83NIWzCsdNlPfDvMV3eZmH3BQCpA9he7Z2pa9SSN3s3lPZdm10pKnNgKyjuTkzWaCFyb2Zg1V2U6QHRGBjAdEjmRKY4tzoNeO2U1~-5EQa36iv8Ol8iR2rNv1cwurS2yNjZ6vcsdB2qwGLmyDdp4qhoVvK0dipsqXqIJ67pjlZbybj184UO1441nuyQkS8bK8-lfNKtKJ~AiXiaEGconrsM8fky5VBgkgVKJpL5tJhbRRgqBwbwny994x82StmyjPjw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086114&ar=3868561&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx025f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Duplicates are more enriched for transcriptional plasticity than singletons, regardless of the mechanism of duplication. (a) Transcriptional plasticity of duplicates and singletons in <em>S. cerevisiae</em>. <em>Y</em>-axis represents the mean percentage of genes with altered transcription across four stress conditions (ethanol, glycerol, lactate and oxidative stress in a medium supplemented with dextrose). We also compared WGDs and SSDs to singletons. Significant differences were found using Fisher’s exact test, with *** indicating a probability of P &lt; 2.2 × 10<sup>−16</sup>. (b) Venn diagram representing the overlap in the number of duplicates and singletons between <em>C. glabrata</em> and <em>S. cerevisiae</em>.</p></div></div><div content-id="dsx025-F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/m_dsx025f7.png?Expires=2147483647&Signature=zPjcLucDffbG0ZU-EtFC27aki~G2t9GZTk-nsYMQwxRf2CapvM8Uu2j14ZHQEybXGP6Y8T2wKgO01wdbOgMQRaBXNnMSPICbX5-~Hry8fAMPm6wjv-qZ~t4BjgrBCZvbzE~Y1kZ7DB-WB0qnk~jOSiXSiW3wAYfJ4IGzJIujRduGn~3zppV1VN0pRBAxzHpnopM3y8Gz~nTo-DwdNhy-6KrIhzjdN4ff0oCPonNBtlkIrVw8z-02m4AFPRJg4~Oe-i6CmKuRU24r6L~v1E9mL3v0Os~ipoXlz6~mv7rWZdTl9iSbjrlWfG1d5GmjgU4m7CBkQmBdo4VfDMZYPd25OQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Duplicates are more enriched for transcriptional plasticity than singletons, regardless of the mechanism of duplication. (a) Transcriptional plasticity of duplicates and singletons in S. cerevisiae. Y-axis represents the mean percentage of genes with altered transcription across four stress conditions (ethanol, glycerol, lactate and oxidative stress in a medium supplemented with dextrose). We also compared WGDs and SSDs to singletons. Significant differences were found using Fisher’s exact test, with *** indicating a probability of P &lt; 2.2 × 10−16. (b) Venn diagram representing the overlap in the number of duplicates and singletons between C. glabrata and S. cerevisiae." path-from-xml="dsx025f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086114/dsx025f7.tif" path-from-xml="dsx025f7.tif" target="_blank">View large</a><a section="104086114" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/dsx025f7.png?Expires=2147483647&Signature=x906ez~7cQUE~tWWb1DmCyLMFJ2OVsjKEcjZ~dt64K9tyX9hs1KrmslbxEHJeSktrFgUWW5T9CtfSXqG6Eu83NIWzCsdNlPfDvMV3eZmH3BQCpA9he7Z2pa9SSN3s3lPZdm10pKnNgKyjuTkzWaCFyb2Zg1V2U6QHRGBjAdEjmRKY4tzoNeO2U1~-5EQa36iv8Ol8iR2rNv1cwurS2yNjZ6vcsdB2qwGLmyDdp4qhoVvK0dipsqXqIJ67pjlZbybj184UO1441nuyQkS8bK8-lfNKtKJ~AiXiaEGconrsM8fky5VBgkgVKJpL5tJhbRRgqBwbwny994x82StmyjPjw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086114&ar=3868561&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx025f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Duplicates are more enriched for transcriptional plasticity than singletons, regardless of the mechanism of duplication. (a) Transcriptional plasticity of duplicates and singletons in <em>S. cerevisiae</em>. <em>Y</em>-axis represents the mean percentage of genes with altered transcription across four stress conditions (ethanol, glycerol, lactate and oxidative stress in a medium supplemented with dextrose). We also compared WGDs and SSDs to singletons. Significant differences were found using Fisher’s exact test, with *** indicating a probability of P &lt; 2.2 × 10<sup>−16</sup>. (b) Venn diagram representing the overlap in the number of duplicates and singletons between <em>C. glabrata</em> and <em>S. cerevisiae</em>.</p></div></div><p>To shed more light on the role of gene duplication in the acquisition of transcriptional plasticity, we examined the transcriptional patterns of a post-WGD species, <em>Candida glabrata</em>, in which some orthologs of <em>S. cerevisiae</em> duplicates are in single gene copy in <em>C. glabrata</em>, while others are preserved as duplicates and for which we had transcriptional information under acidic stress similar to our lactate stress dataset.<a reveal-id="dsx025-B36" data-open="dsx025-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a><sup>,</sup><em>Saccharomyces cerevisiae</em> orthologs in <em>C. glabrata</em> were identified using synteny information available in the Pillars of the Yeast Gene Order Browser (YGOB).<a reveal-id="dsx025-B30" data-open="dsx025-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> In total, we identified 4844 reliable <em>S. cerevisiae</em>: <em>C. glabrata</em> orthologs. Of these 4844 genes, 788 genes were duplicated in <em>C. glabrata</em>, of which 123 were orthologs of <em>S. cerevisiae</em> singletons. These 4844 genes included 1659 out of the 2240 duplicated <em>S. cerevisiae</em> genes and 3185 singletons. Of the 2240 <em>S. cerevisiae</em> duplicates, orthologs for 1019 of them were in single gene copy in <em>C. glabrata</em> (<a reveal-id="dsx025-F7" data-open="dsx025-F7" class="link link-reveal link-table xref-fig">Fig. 7b</a>). Importantly, these 1019 genes exhibited as much transcriptional plasticity under acidic stress in <em>C. glabrata</em> (<em>N </em>= 599, 58.7% of the <em>C. glabrata</em> singletons) as <em>C. glabrata</em> singletons that had no duplicates orthologs in <em>S. cerevisiae</em> (1725 out of 3062 singletons, 56.3%) (Fisher’s exact test: odd’s ratio <em>F</em> = 1.10, <em>P</em> = 0.17). In conclusion, transcriptional plasticity seem to have been acquired after gene duplication because orthologs of duplicates that are in single copy genes in other species exhibit no evidence for transcriptional plasticity in these species.</p>                            <h3 scrollto-destination=104086116 id="104086116" class="section-title" >3.6. Transcriptionally plastic duplicates contribute to the response of <em>S. cerevisiae</em> to stress</h3>
<p>Among the transcriptionally plastic duplicates, many had a significant biological role in the response to stress. For instance, analyses of duplicated genes up-regulated when <em>S. cerevisiae</em> is grown in ethanol identified a number of genes as largely up-regulated that are involved in ethanol metabolism (<span class="content-section supplementary-material"><a path-from-xml="sup27" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables27_dsx025.xls?Expires=2147483647&amp;Signature=YYBwrPZkt9Rh2q-qhR4YNI4Yp~Hmct-K0gz71WZ6KyvFZPUrFxlG8PL12xTVFZvdpSO0hD2xT2SczEgWpMIiD3XMUYoHbcBosNMWSEVk4zIH7jsJSETZxDE46WhIsI1qlJ1Ks6Zhn6qTpeBRLyB4jwGebm5jezle9RY6fYhbAkD0ZF~e5hIUY-zsIfGkoQzdkmvHIUDd8AamLs0qXdLJ9jyuXRdcmnWDN0DlJFjas9b0V7b6~77NoVHYmK-c39HAJGQFX2VwlgZeekoUPiM9i6-S7eQwoM70fOAFLf0R2wArjyc~Du6Hc1mujZcgAR~HUxCx~6Y3kPUrdYhEl8aLsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S27</a></span>). Heading the list of up-regulated duplicates is the one encoding the alcohol dehydrogenase ADH2 and ADH1, directly involved in the quick metabolism of ethanol inside the cell into acetaldehyde (<a reveal-id="dsx025-F8" data-open="dsx025-F8" class="link link-reveal link-table xref-fig">Fig. 8</a>). A number of other duplicated genes that are essential in the metabolism of Ethanol, including transmembrane transporters such as YAT1 to start the tricarboxylic cycle or the enzyme MSL1 that is essential for malate production, among others (<a reveal-id="dsx025-F8" data-open="dsx025-F8" class="link link-reveal link-table xref-fig">Fig. 8</a>), they are all listed as duplicates with the highest up-regulation. This also applies to other stress conditions used in this study (<a reveal-id="dsx025-F8" data-open="dsx025-F8" class="link link-reveal link-table xref-fig">Fig. 8</a>). </p>                        <a id="104086118" scrollto-destination="104086118"></a>
<div id="dsx025-F8" data-id="dsx025-F8" class="fig fig-section"><div class="label fig-label">Figure 8</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/m_dsx025f8.png?Expires=2147483647&Signature=qZFpBUofj0vUYA1ouQahHAEQYnobb1VfL~PFQPfMEX6A5K6SjLcx-M5V8sIs9Ik0li9V6zBqT4o7gRSDQ5TG7Hb87RudciiF0flEq8cKOMWdv23ccxpRgLQCZpbaVzHJeY3uXI0~LxtefIfrZIg0Q1yMxj1uQ3QRvE3kJYRvOWsB78SGib-mZfBl2CZNtRpNIHM26ncV9pfkURVRsWDehr9ochj0lQTjYTgLJSvi~~JpDYKj~2zd5dnV3oaH06zIxMZKIKEy1sM44XsYJPmpxm3c8d7DiSPNJgK8BlE7GXT3b9~x8UGXeCSp8XU7H2HNcKEPFQ1dbtF-QqPczMjwLQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of S. cerevisiae central metabolic pathways for the utilization of non-fermentable carbon sources (glycerol, lactate and ethanol). Solid arrows correspond to enzymatic reactions while dashed arrows correspond to regulatory steps. Proteins encoded by duplicated genes that are transcriptionally altered and are involved in each process are indicated (transporters are shown with a can form, and enzymes exhibit heptagonal forms). " path-from-xml="dsx025f8.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086118/dsx025f8.tif" path-from-xml="dsx025f8.tif" target="_blank">View large</a><a section="104086118" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/dsx025f8.png?Expires=2147483647&Signature=03-GIuxmQE55awUQa-dl6CIlXvwB9fQha5YGBN4b2U3gMDECoWArWk~25CTYPBnUXMIZmIbplcW~IacE~ZAGlZGz6Gk~V0uicY3lZ6UpfeblZpjD~eEY1GkEOUeCJO~aLPQkcin~U-JnXz7HyNzt9olKUTUzOylRZNRsL~~CRHfYlqxM7Cezk5cu8BGOflzcKIi3DzaW2jSDOPUlHa078jKW9NvrYnuLGUI~IH7VFiwVaGqC59ON6LF-g7iNqnsXfpmAjLPPx9-cvTIeSUXflGNQcYIQZUSZsGmqZVS7e1lG2zMjSL0porZjIWU6WXoUl-sopONTO0wEvUoGaUC2nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086118&ar=3868561&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx025f8.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of <em>S. cerevisiae</em> central metabolic pathways for the utilization of non-fermentable carbon sources (glycerol, lactate and ethanol). Solid arrows correspond to enzymatic reactions while dashed arrows correspond to regulatory steps. Proteins encoded by duplicated genes that are transcriptionally altered and are involved in each process are indicated (transporters are shown with a can form, and enzymes exhibit heptagonal forms). </p></div></div><div content-id="dsx025-F8" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 8</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/m_dsx025f8.png?Expires=2147483647&Signature=qZFpBUofj0vUYA1ouQahHAEQYnobb1VfL~PFQPfMEX6A5K6SjLcx-M5V8sIs9Ik0li9V6zBqT4o7gRSDQ5TG7Hb87RudciiF0flEq8cKOMWdv23ccxpRgLQCZpbaVzHJeY3uXI0~LxtefIfrZIg0Q1yMxj1uQ3QRvE3kJYRvOWsB78SGib-mZfBl2CZNtRpNIHM26ncV9pfkURVRsWDehr9ochj0lQTjYTgLJSvi~~JpDYKj~2zd5dnV3oaH06zIxMZKIKEy1sM44XsYJPmpxm3c8d7DiSPNJgK8BlE7GXT3b9~x8UGXeCSp8XU7H2HNcKEPFQ1dbtF-QqPczMjwLQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of S. cerevisiae central metabolic pathways for the utilization of non-fermentable carbon sources (glycerol, lactate and ethanol). Solid arrows correspond to enzymatic reactions while dashed arrows correspond to regulatory steps. Proteins encoded by duplicated genes that are transcriptionally altered and are involved in each process are indicated (transporters are shown with a can form, and enzymes exhibit heptagonal forms). " path-from-xml="dsx025f8.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086118/dsx025f8.tif" path-from-xml="dsx025f8.tif" target="_blank">View large</a><a section="104086118" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/dsx025f8.png?Expires=2147483647&Signature=03-GIuxmQE55awUQa-dl6CIlXvwB9fQha5YGBN4b2U3gMDECoWArWk~25CTYPBnUXMIZmIbplcW~IacE~ZAGlZGz6Gk~V0uicY3lZ6UpfeblZpjD~eEY1GkEOUeCJO~aLPQkcin~U-JnXz7HyNzt9olKUTUzOylRZNRsL~~CRHfYlqxM7Cezk5cu8BGOflzcKIi3DzaW2jSDOPUlHa078jKW9NvrYnuLGUI~IH7VFiwVaGqC59ON6LF-g7iNqnsXfpmAjLPPx9-cvTIeSUXflGNQcYIQZUSZsGmqZVS7e1lG2zMjSL0porZjIWU6WXoUl-sopONTO0wEvUoGaUC2nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086118&ar=3868561&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx025f8.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of <em>S. cerevisiae</em> central metabolic pathways for the utilization of non-fermentable carbon sources (glycerol, lactate and ethanol). Solid arrows correspond to enzymatic reactions while dashed arrows correspond to regulatory steps. Proteins encoded by duplicated genes that are transcriptionally altered and are involved in each process are indicated (transporters are shown with a can form, and enzymes exhibit heptagonal forms). </p></div></div>                            <h2 scrollto-destination=104086119 id="104086119" class="section-title" >4. Discussion</h2>
<p>A number of factors have been considered key players in determining the fates of duplicated genes. However, an explanation that provides a general and inherent mechanism that strongly determines the duplicability of genes has been poorly investigated. Here, we present strong evidence that the levels of gene expression determine the likelihood of a gene to persist duplicated in the genome. We find that genes that are highly expressed in species pre-dating the whole-genome duplication event in <em>Saccharomyces</em> are more likely to be preserved in duplicate in species originated after duplication. This observation is in agreement with the dosage sub-functionalization hypothesis (DSH), as high levels of gene expression would ensure purifying selection on the gene copies despite the stochastic variation in their expression levels.<a reveal-id="dsx025-B28" data-open="dsx025-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> We, nevertheless, show that this is not a link exclusively seen in WGDs because the preservation of SSDs is also dependent on the expression levels of genes, however to a lesser extent than WGDs.</p><p>We also show that highly expressed genes exhibit greater phylogenetic stability (i.e. they are preserved in greater number of post-duplication species) than lowly expressed genes, perhaps due to the higher likelihood of functional divergence between gene copies of highly expressed duplicates, and thus emergence of purifying selection to maintain the functions of gene copies. Therefore, we conclude that the levels of gene expression determine the survival of duplicates across long evolutionary times. Our observations are not confounded by other factors such as dosage balance, mainly an issue in the case of genes encoding proteins that are part of protein complexes,<a reveal-id="dsx025-B28" data-open="dsx025-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a><sup>,</sup><a reveal-id="dsx025-B39" data-open="dsx025-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a><sup>,</sup><a reveal-id="dsx025-B43" data-open="dsx025-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a><sup>,</sup><a reveal-id="dsx025-B45" data-open="dsx025-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a><sup>,</sup><a reveal-id="dsx025-B46" data-open="dsx025-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> as neither WGDs nor SSDs are enriched for complex-encoding genes when compared to singletons in <em>S. cerevisiae</em>.</p><p>A pivotal conclusion derived from the link between gene expression and duplicability is that gene copies with higher expression levels can be noisier in terms of expression without undergoing an imbalance in their selective pressures. This noise could eventually lead to higher divergence between the gene copies of these duplicates when such divergence becomes adaptive. Our results are in agreement with this prediction and show that the magnitude of expression divergence, and perhaps functional divergence, is dependent on the levels of gene expression. Therefore, duplicates with higher expression are more likely to lead to higher divergence between the gene copies. This higher divergence between gene copies has been likely important for the origin of novel adaptations to stressful environments in the yeast <em>Saccharomyces cerevisiae</em>.<a reveal-id="dsx025-B26" data-open="dsx025-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a><sup>,</sup><a reveal-id="dsx025-B27" data-open="dsx025-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a><sup>,</sup><a reveal-id="dsx025-B51 dsx025-B52 dsx025-B53" data-open="dsx025-B51 dsx025-B52 dsx025-B53" class="link link-ref link-reveal xref-bibr"><sup>51–53</sup></a> In agreement with this hypothesis, when <em>S. cerevisiae</em> was faced with a number of stress conditions, the levels of expression divergence between gene copies was positively correlated with their expression plasticity. Importantly, we show that the levels of gene expression also influence the patterns of transcriptional divergence between the gene copies of duplicates. Those duplicates with copies exhibiting discordant expression patterns or with only one copy altered under stress are likely those in which each copy encodes a function under different conditions compared to the other copy.</p><p>Expression noise is generally coupled with plasticity if noise and plasticity do not present a cost-benefit conflict (i.e. a conflict emerges when plasticity is important for adaptation but noisy expression can be detrimental).<a reveal-id="dsx025-B50" data-open="dsx025-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a><sup>,</sup><a reveal-id="dsx025-B54" data-open="dsx025-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a><sup>,</sup><a reveal-id="dsx025-B55" data-open="dsx025-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> We show that duplicates are both more enriched for transcriptionally plastic genes and for genes with TATA motifs in their promoters when compared with singletons. TATA-motifs have been shown to be associated with higher noise and plasticity in TATA-containing genes.<a reveal-id="dsx025-B49" data-open="dsx025-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a><sup>,</sup><a reveal-id="dsx025-B56" data-open="dsx025-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a><sup>,</sup><a reveal-id="dsx025-B57" data-open="dsx025-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> The expression properties of genes that have been preserved as duplicates are, therefore, remarkably different from those that returned to single copy genes. Noticeably, non-plastic duplicates that showed higher expression levels than transcriptionally plastic singletons also exhibited greater enrichment for TATA-containing genes than plastic singletons, linking TATA-containing genes to higher levels of gene expression. The question that remains is then whether it is the expression properties or duplication itself that have determined the fate of duplicates and the origin of adaptations. Our results lead to the conclusion that duplicated genes exhibit significantly different expression properties than singleton genes but also that gene duplication is mainly responsible for the origin of plasticity, as the transcriptional plasticity observed in <em>S. cerevisiae</em> originated after the duplication of genes which were not transcriptionally more plastic before duplication than other non-duplicable genes. Therefore, gene duplication provides the appropriate genetic and selective opportunity for the evolution of transcriptional plasticity. A remarkable result is that the percentage of singletons transcriptionally altered in <em>S. cerevisiae</em> was significantly lower than that of their orthologs altered in <em>L. kluyveri</em>. However, the absolute percentages of altered genes cannot be compared between <em>L. kluyveri</em> and <em>S. cerevisiae</em> as both species represent completely different metabolisms and the conditions of stress under which they were subjected were different. It is, therefore, likely that duplication itself may relax the cost–benefit conflict between noise and plasticity, as previously suggested,<a reveal-id="dsx025-B50" data-open="dsx025-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> allowing for the emergence of plasticity and adaptation to environmental perturbations. In conclusion, our result point to a strong correlation between the expression properties of genes, their duplicability, transcriptional plasticity, and ability to give rise to novel adaptations.</p>                            <h2 scrollto-destination=104086124 id="104086124" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We would like to thank members of Fares’ Lab for a careful reading and discussion of the results in the manuscript. We are also grateful to colleagues at Trinity College for helpful discussions. This work was supported by a grant from the Spanish Ministerio de Economía y Competitividad (MINECO-FEDER; BFU2015-66073-P) to M.A.F. F.M. is supported by a PhD grant from the Spanish Ministerio de Economía y Competitividad (reference: BES-2016-076677). C.T. was supported by a grant Juan de la Cierva from the Spanish Ministerio de Economía y Competitividad (reference: JCA-2012-14056).</p>                            <h3 scrollto-destination=104086126 id="104086126" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=104086128 id="104086128" class="section-title" >Accession numbers</h3>
<p>All newly sequenced RNA sequences are available from the Sequence Read Archive with the following accession number (SRP074821).</p>                            <h3 scrollto-destination=104086130 id="104086130" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables1_dsx025.xls?Expires=2147483647&Signature=nPaJXHYV6gQ53wdehHE7rsx-iQX0ETTZVzTjn5AmU2zebm8Vy102UjU827wCKDXjTAy9Rw5tHgpdB24X3VQfxnDGbDWNWgQqU-1RIWH87wtlk8HEB~nZXuEpFyi9FlceBEO7TR8mceWMN2rApEkpPf9k3IcugRkKGNWhyrPWUbrj6izwPoG0p7vTYZt13G7yv9G8dktdeUpRxvuF3OFB9UF9p4VQCrgDHJZFyxLMZ9lEkCUn9wD-295ni3IcoZ~85QXn7LHT~ELwTDUdqmWzq8lV~K9rIh7KsVPdATkyvhw2EVan6TBFpv5NPJQ5nSe1xtd976ljXnpi8-ywkvJfxw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h2 scrollto-destination=104086132 id="104086132" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx025-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ohno</div> <div class="given-names">S.</div></div></div> <div class="year">1970</div>, <div class="source">Evolution by Gene Duplication</div> . 
<div class="publisher-name">Springer Verlag</div>, 
<div class="publisher-loc">New York</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20by%20Gene%20Duplication&amp;author=S.%20Ohno&amp;publication_year=1970&amp;book=Evolution%20by%20Gene%20Duplication" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/978-3-642-86659-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2F978-3-642-86659-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2F978-3-642-86659-3"> </span></div> </div></div></div></div></div><div content-id="dsx025-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ohno</div> <div class="given-names">S.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Gene duplication and the uniqueness of vertebrate genomes circa 1970–1999,</div> <div class="source">Semin. Cell Dev. Biol</div> ., <div class="volume">10</div>, <div class="fpage">517</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20duplication%20and%20the%20uniqueness%20of%20vertebrate%20genomes%20circa%201970%E2%80%931999%2C&amp;author=S.%20Ohno&amp;publication_year=1999&amp;journal=Semin.%20Cell%20Dev.%20Biol&amp;volume=10&amp;pages=517-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/scdb.1999.0332" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fscdb.1999.0332" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fscdb.1999.0332"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10597635" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lynch</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Conery</div> <div class="given-names">J.S.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">The evolutionary fate and consequences of duplicate genes,</div> <div class="source">Science</div> , <div class="volume">290</div>, <div class="fpage">1151</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolutionary%20fate%20and%20consequences%20of%20duplicate%20genes%2C&amp;author=M.%20Lynch&amp;author=J.S.%20Conery&amp;publication_year=2000&amp;journal=Science&amp;volume=290&amp;pages=1151-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.290.5494.1151" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.290.5494.1151" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.290.5494.1151"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11073452" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Otto</div> <div class="given-names">S.P.</div></div>, <div class="name string-name"><div class="surname">Whitton</div> <div class="given-names">J.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Polyploid incidence and evolution,</div> <div class="source">Annu. Rev. Genet</div> ., <div class="volume">34</div>, <div class="fpage">401</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Polyploid%20incidence%20and%20evolution%2C&amp;author=S.P.%20Otto&amp;author=J.%20Whitton&amp;publication_year=2000&amp;journal=Annu.%20Rev.%20Genet&amp;volume=34&amp;pages=401-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.34.1.401" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.34.1.401" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.34.1.401"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11092833" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Carretero-Paulet</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Fares</div> <div class="given-names">M.A.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Evolutionary dynamics and functional specialization of plant paralogs formed by whole and small-scale genome duplications,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">29</div>, <div class="fpage">3541</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20dynamics%20and%20functional%20specialization%20of%20plant%20paralogs%20formed%20by%20whole%20and%20small-scale%20genome%20duplications%2C&amp;author=L.%20Carretero-Paulet&amp;author=M.A.%20Fares&amp;publication_year=2012&amp;journal=Mol.%20Biol.%20Evol&amp;volume=29&amp;pages=3541-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mss162" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmss162" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmss162"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22734049" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cui</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Wall</div> <div class="given-names">P.K.</div></div>, <div class="name string-name"><div class="surname">Leebens-Mack</div> <div class="given-names">J.H.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">Widespread genome duplications throughout the history of flowering plants,</div> <div class="source">Genome Res</div> ., <div class="volume">16</div>, <div class="fpage">738</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Widespread%20genome%20duplications%20throughout%20the%20history%20of%20flowering%20plants%2C&amp;author=L.%20Cui&amp;author=P.K.%20Wall&amp;author=J.H.%20Leebens-Mack&amp;publication_year=2006&amp;journal=Genome%20Res&amp;volume=16&amp;pages=738-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.4825606" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.4825606" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.4825606"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16702410" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Holub</div> <div class="given-names">E.B.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">The arms race is ancient history in Arabidopsis, the wildflower,</div> <div class="source">Nat. Rev. Genet</div> ., <div class="volume">2</div>, <div class="fpage">516</div>–<div class="lpage">27</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20arms%20race%20is%20ancient%20history%20in%20Arabidopsis%2C%20the%20wildflower%2C&amp;author=E.B.%20Holub&amp;publication_year=2001&amp;journal=Nat.%20Rev.%20Genet&amp;volume=2&amp;pages=516-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35080508" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35080508" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35080508"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11433358" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lespinet</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Wolf</div> <div class="given-names">Y.I.</div></div>, <div class="name string-name"><div class="surname">Koonin</div> <div class="given-names">E.V.</div></div>, <div class="name string-name"><div class="surname">Aravind</div> <div class="given-names">L.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">The role of lineage-specific gene family expansion in the evolution of eukaryotes,</div> <div class="source">Genome Res</div> ., <div class="volume">12</div>, <div class="fpage">1048</div>–<div class="lpage">59</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20lineage-specific%20gene%20family%20expansion%20in%20the%20evolution%20of%20eukaryotes%2C&amp;author=O.%20Lespinet&amp;author=Y.I.%20Wolf&amp;author=E.V.%20Koonin&amp;author=L.%20Aravind&amp;publication_year=2002&amp;journal=Genome%20Res&amp;volume=12&amp;pages=1048-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.174302" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.174302" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.174302"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12097341" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wendel</div> <div class="given-names">J.F.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Genome evolution in polyploids,</div> <div class="source">Plant Mol. Biol</div> ., <div class="volume">42</div>, <div class="fpage">225</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20evolution%20in%20polyploids%2C&amp;author=J.F.%20Wendel&amp;publication_year=2000&amp;journal=Plant%20Mol.%20Biol&amp;volume=42&amp;pages=225-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1006392424384" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1006392424384" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1006392424384"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10688139" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Soltis</div> <div class="given-names">D.E.</div></div>, <div class="name string-name"><div class="surname">Albert</div> <div class="given-names">V.A.</div></div>, <div class="name string-name"><div class="surname">Leebens-Mack</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">Polyploidy and angiosperm diversification,</div> <div class="source">Am. J. Bot</div> ., <div class="volume">96</div>, <div class="fpage">336</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Polyploidy%20and%20angiosperm%20diversification%2C&amp;author=D.E.%20Soltis&amp;author=V.A.%20Albert&amp;author=J.%20Leebens-Mack&amp;publication_year=2009&amp;journal=Am.%20J.%20Bot&amp;volume=96&amp;pages=336-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3732/ajb.0800079" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3732%2Fajb.0800079" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3732%2Fajb.0800079"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21628192" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Van de Peer</div> <div class="given-names">Y.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Computational approaches to unveiling ancient genome duplications,</div> <div class="source">Nat. Rev. Genet</div> ., <div class="volume">5</div>, <div class="fpage">752</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20approaches%20to%20unveiling%20ancient%20genome%20duplications%2C&amp;author=Y.%20Van%20de%20Peer&amp;publication_year=2004&amp;journal=Nat.%20Rev.%20Genet&amp;volume=5&amp;pages=752-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1449" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1449" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1449"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15510166" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hoegg</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Brinkmann</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Taylor</div> <div class="given-names">J.S.</div></div>, <div class="name string-name"><div class="surname">Meyer</div> <div class="given-names">A.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Phylogenetic timing of the fish-specific genome duplication correlates with the diversification of teleost fish,</div> <div class="source">J. Mol. Evol</div> ., <div class="volume">59</div>, <div class="fpage">190</div>–<div class="lpage">203</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20timing%20of%20the%20fish-specific%20genome%20duplication%20correlates%20with%20the%20diversification%20of%20teleost%20fish%2C&amp;author=S.%20Hoegg&amp;author=H.%20Brinkmann&amp;author=J.S.%20Taylor&amp;author=A.%20Meyer&amp;publication_year=2004&amp;journal=J.%20Mol.%20Evol&amp;volume=59&amp;pages=190-203" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-004-2613-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-004-2613-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-004-2613-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15486693" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Grant</div> <div class="given-names">S.G.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">The molecular evolution of the vertebrate behavioural repertoire,</div> <div class="source">Philos. Trans. R. Soc. Lond. B: Biol. Sci</div> ., <div class="volume">371</div>, <div class="fpage">20150051</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20molecular%20evolution%20of%20the%20vertebrate%20behavioural%20repertoire%2C&amp;author=S.G.%20Grant&amp;publication_year=2016&amp;journal=Philos.%20Trans.%20R.%20Soc.%20Lond.%20B%3A%20Biol.%20Sci&amp;volume=371&amp;pages=20150051" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rstb.2015.0051" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frstb.2015.0051" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frstb.2015.0051"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26598730" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Green</div> <div class="given-names">S.A.</div></div>, <div class="name string-name"><div class="surname">Bronner</div> <div class="given-names">M.E.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Gene duplications and the early evolution of neural crest development,</div> <div class="source">Semin. Cell Dev. Biol</div> ., <div class="volume">24</div>, <div class="fpage">95</div>–<div class="lpage">100</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20duplications%20and%20the%20early%20evolution%20of%20neural%20crest%20development%2C&amp;author=S.A.%20Green&amp;author=M.E.%20Bronner&amp;publication_year=2013&amp;journal=Semin.%20Cell%20Dev.%20Biol&amp;volume=24&amp;pages=95-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.semcdb.2012.12.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.semcdb.2012.12.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.semcdb.2012.12.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23287633" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huminiecki</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Conant</div> <div class="given-names">G.C.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Polyploidy and the evolution of complex traits,</div> <div class="source">Int. J. Evol. Biol</div> ., <div class="volume">2012</div>, <div class="fpage">292068</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Polyploidy%20and%20the%20evolution%20of%20complex%20traits%2C&amp;author=L.%20Huminiecki&amp;author=G.C.%20Conant&amp;publication_year=2012&amp;journal=Int.%20J.%20Evol.%20Biol&amp;volume=2012&amp;pages=292068" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1155/2012/292068" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1155%2F2012%2F292068" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1155%2F2012%2F292068"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22900230" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Conant</div> <div class="given-names">G.C.</div></div>, <div class="name string-name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Increased glycolytic flux as an outcome of whole-genome duplication in yeast,</div> <div class="source">Mol. Syst. Biol</div> ., <div class="volume">3</div>, <div class="fpage">129</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Increased%20glycolytic%20flux%20as%20an%20outcome%20of%20whole-genome%20duplication%20in%20yeast%2C&amp;author=G.C.%20Conant&amp;author=K.H.%20Wolfe&amp;publication_year=2007&amp;journal=Mol.%20Syst.%20Biol&amp;volume=3&amp;pages=129" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb4100170" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb4100170" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmsb4100170"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17667951" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div>, <div class="name string-name"><div class="surname">Shields</div> <div class="given-names">D.C.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">Molecular evidence for an ancient duplication of the entire yeast genome,</div> <div class="source">Nature</div> , <div class="volume">387</div>, <div class="fpage">708</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20evidence%20for%20an%20ancient%20duplication%20of%20the%20entire%20yeast%20genome%2C&amp;author=K.H.%20Wolfe&amp;author=D.C.%20Shields&amp;publication_year=1997&amp;journal=Nature&amp;volume=387&amp;pages=708-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/42711" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F42711" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F42711"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9192896" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fares</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Keane</div> <div class="given-names">O.M.</div></div>, <div class="name string-name"><div class="surname">Toft</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Carretero-Paulet</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Jones</div> <div class="given-names">G.W.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">The roles of whole-genome and small-scale duplications in the functional specialization of <em>Saccharomyces cerevisiae</em> genes,</div> <div class="source">PLoS Genet</div> ., <div class="volume">9</div>, <div class="fpage">e1003176</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20roles%20of%20whole-genome%20and%20small-scale%20duplications%20in%20the%20functional%20specialization%20of%20Saccharomyces%20cerevisiae%20genes%2C&amp;author=M.A.%20Fares&amp;author=O.M.%20Keane&amp;author=C.%20Toft&amp;author=L.%20Carretero-Paulet&amp;author=G.W.%20Jones&amp;publication_year=2013&amp;journal=PLoS%20Genet&amp;volume=9&amp;pages=e1003176" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1003176" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1003176" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1003176"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23300483" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Blanc</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Widespread paleopolyploidy in model plant species inferred from age distributions of duplicate genes,</div> <div class="source">Plant Cell</div> , <div class="volume">16</div>, <div class="fpage">1667</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Widespread%20paleopolyploidy%20in%20model%20plant%20species%20inferred%20from%20age%20distributions%20of%20duplicate%20genes%2C&amp;author=G.%20Blanc&amp;author=K.H.%20Wolfe&amp;publication_year=2004&amp;journal=Plant%20Cell&amp;volume=16&amp;pages=1667-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.021345" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.021345" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.021345"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15208399" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Blanc</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Functional divergence of duplicated genes formed by polyploidy during Arabidopsis evolution,</div> <div class="source">Plant Cell</div> , <div class="volume">16</div>, <div class="fpage">1679</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20divergence%20of%20duplicated%20genes%20formed%20by%20polyploidy%20during%20Arabidopsis%20evolution%2C&amp;author=G.%20Blanc&amp;author=K.H.%20Wolfe&amp;publication_year=2004&amp;journal=Plant%20Cell&amp;volume=16&amp;pages=1679-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.021410" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.021410" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.021410"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15208398" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Conant</div> <div class="given-names">G.C.</div></div>, <div class="name string-name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Turning a hobby into a job: how duplicated genes find new functions,</div> <div class="source">Nat. Rev. Genet</div> ., <div class="volume">9</div>, <div class="fpage">938</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Turning%20a%20hobby%20into%20a%20job%3A%20how%20duplicated%20genes%20find%20new%20functions%2C&amp;author=G.C.%20Conant&amp;author=K.H.%20Wolfe&amp;publication_year=2008&amp;journal=Nat.%20Rev.%20Genet&amp;volume=9&amp;pages=938-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2482" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2482" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2482"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19015656" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fares</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Byrne</div> <div class="given-names">K.P.</div></div>, <div class="name string-name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Rate asymmetry after genome duplication causes substantial long-branch attraction artifacts in the phylogeny of Saccharomyces species</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">23</div>, <div class="fpage">245</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rate%20asymmetry%20after%20genome%20duplication%20causes%20substantial%20long-branch%20attraction%20artifacts%20in%20the%20phylogeny%20of%20Saccharomyces%20species&amp;author=M.A.%20Fares&amp;author=K.P.%20Byrne&amp;author=K.H.%20Wolfe&amp;publication_year=2006&amp;journal=Mol.%20Biol.%20Evol&amp;volume=23&amp;pages=245-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msj027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsj027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsj027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16207937" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Scannell</div> <div class="given-names">D.R.</div></div>, <div class="name string-name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">A burst of protein sequence evolution and a prolonged period of asymmetric evolution follow gene duplication in yeast,</div> <div class="source">Genome Res</div> ., <div class="volume">18</div>, <div class="fpage">137</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20burst%20of%20protein%20sequence%20evolution%20and%20a%20prolonged%20period%20of%20asymmetric%20evolution%20follow%20gene%20duplication%20in%20yeast%2C&amp;author=D.R.%20Scannell&amp;author=K.H.%20Wolfe&amp;publication_year=2008&amp;journal=Genome%20Res&amp;volume=18&amp;pages=137-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.6341207" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.6341207" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.6341207"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18025270" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Freeling</div> <div class="given-names">M.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Bias in plant gene content following different sorts of duplication: tandem, whole-genome, segmental, or by transposition,</div> <div class="source">Annu. Rev. Plant Biol</div> ., <div class="volume">60</div>, <div class="fpage">433</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bias%20in%20plant%20gene%20content%20following%20different%20sorts%20of%20duplication%3A%20tandem%2C%20whole-genome%2C%20segmental%2C%20or%20by%20transposition%2C&amp;author=M.%20Freeling&amp;publication_year=2009&amp;journal=Annu.%20Rev.%20Plant%20Biol&amp;volume=60&amp;pages=433-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.arplant.043008.092122" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.arplant.043008.092122" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.arplant.043008.092122"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19575588" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Conant</div> <div class="given-names">G.C.</div></div>, <div class="name string-name"><div class="surname">Birchler</div> <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Pires</div> <div class="given-names">J.C.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Dosage, duplication, and diploidization: clarifying the interplay of multiple models for duplicate gene evolution over time,</div> <div class="source">Curr. Opin. Plant Biol</div> ., <div class="volume">19</div>, <div class="fpage">91</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dosage%2C%20duplication%2C%20and%20diploidization%3A%20clarifying%20the%20interplay%20of%20multiple%20models%20for%20duplicate%20gene%20evolution%20over%20time%2C&amp;author=G.C.%20Conant&amp;author=J.A.%20Birchler&amp;author=J.C.%20Pires&amp;publication_year=2014&amp;journal=Curr.%20Opin.%20Plant%20Biol&amp;volume=19&amp;pages=91-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2014.05.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2014.05.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2014.05.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24907529" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Keane</div> <div class="given-names">O.M.</div></div>, <div class="name string-name"><div class="surname">Toft</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Carretero-Paulet</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Jones</div> <div class="given-names">G.W.</div></div>, <div class="name string-name"><div class="surname">Fares</div> <div class="given-names">M.A.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Preservation of genetic and regulatory robustness in ancient gene duplicates of <em>Saccharomyces cerevisiae</em>,</div> <div class="source">Genome Res</div> ., <div class="volume">24</div>, <div class="fpage">1830</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Preservation%20of%20genetic%20and%20regulatory%20robustness%20in%20ancient%20gene%20duplicates%20of%20Saccharomyces%20cerevisiae%2C&amp;author=O.M.%20Keane&amp;author=C.%20Toft&amp;author=L.%20Carretero-Paulet&amp;author=G.W.%20Jones&amp;author=M.A.%20Fares&amp;publication_year=2014&amp;journal=Genome%20Res&amp;volume=24&amp;pages=1830-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.176792.114" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.176792.114" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.176792.114"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25149527" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fares</div> <div class="given-names">M.A.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">The origins of mutational robustness,</div> <div class="source">Trends Genet</div> ., <div class="volume">31</div>, <div class="fpage">373</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20origins%20of%20mutational%20robustness%2C&amp;author=M.A.%20Fares&amp;publication_year=2015&amp;journal=Trends%20Genet&amp;volume=31&amp;pages=373-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2015.04.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2015.04.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2015.04.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26013677" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gout</div> <div class="given-names">J.F.</div></div>, <div class="name string-name"><div class="surname">Lynch</div> <div class="given-names">M.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Maintenance and loss of duplicated genes by dosage subfunctionalization,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">32</div>, <div class="fpage">2141</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Maintenance%20and%20loss%20of%20duplicated%20genes%20by%20dosage%20subfunctionalization%2C&amp;author=J.F.%20Gout&amp;author=M.%20Lynch&amp;publication_year=2015&amp;journal=Mol.%20Biol.%20Evol&amp;volume=32&amp;pages=2141-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msv095" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsv095" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsv095"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25908670" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>, <div class="name string-name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>, <div class="name string-name"><div class="surname">Schaffer</div> <div class="given-names">A.A.</div></div></div>, et al.   <div class="year">1997</div>, 
<div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">25</div>, <div class="fpage">3389</div>–<div class="lpage">402</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs%2C&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Byrne</div> <div class="given-names">K.P.</div></div>, <div class="name string-name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">The Yeast Gene Order Browser: combining curated homology and syntenic context reveals gene fate in polyploid species,</div> <div class="source">Genome Res</div> ., <div class="volume">15</div>, <div class="fpage">1456</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Yeast%20Gene%20Order%20Browser%3A%20combining%20curated%20homology%20and%20syntenic%20context%20reveals%20gene%20fate%20in%20polyploid%20species%2C&amp;author=K.P.%20Byrne&amp;author=K.H.%20Wolfe&amp;publication_year=2005&amp;journal=Genome%20Res&amp;volume=15&amp;pages=1456-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.3672305" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.3672305" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.3672305"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16169922" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lohse</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Bolger</div> <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">Nagel</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">RobiNA: a user-friendly, integrated software solution for RNA-Seq-based transcriptomics,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">W622</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RobiNA%3A%20a%20user-friendly%2C%20integrated%20software%20solution%20for%20RNA-Seq-based%20transcriptomics%2C&amp;author=M.%20Lohse&amp;author=A.M.%20Bolger&amp;author=A.%20Nagel&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=W622-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks540" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks540" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks540"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22684630" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Robinson</div> <div class="given-names">M.D.</div></div>, <div class="name string-name"><div class="surname">McCarthy</div> <div class="given-names">D.J.</div></div>, <div class="name string-name"><div class="surname">Smyth</div> <div class="given-names">G.K.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">edgeR: a Bioconductor package for differential expression analysis of digital gene expression data,</div> <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">139</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=edgeR%3A%20a%20Bioconductor%20package%20for%20differential%20expression%20analysis%20of%20digital%20gene%20expression%20data%2C&amp;author=M.D.%20Robinson&amp;author=D.J.%20McCarthy&amp;author=G.K.%20Smyth&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=139-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp616" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp616" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp616"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19910308" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Anders</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Huber</div> <div class="given-names">W.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Differential expression analysis for sequence count data,</div> <div class="source">Genome Biol</div> ., <div class="volume">11</div>, <div class="fpage">R106</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20expression%20analysis%20for%20sequence%20count%20data%2C&amp;author=S.%20Anders&amp;author=W.%20Huber&amp;publication_year=2010&amp;journal=Genome%20Biol&amp;volume=11&amp;pages=R106" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2010-11-10-r106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2010-11-10-r106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2010-11-10-r106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20979621" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Benjamini</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Yekutieli</div> <div class="given-names">D.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">The control of the false discovery rate in multiple testing under dependency,</div> <div class="source">Ann. Stat</div> ., <div class="volume">29</div>, <div class="fpage">24</div>.</div></div></div></div><div content-id="dsx025-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Brion</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pflieger</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Souali-Crespo</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Friedrich</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Schacherer</div> <div class="given-names">J.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Differences in environmental stress response among yeasts is consistent with species-specific lifestyles,</div> <div class="source">Mol. Biol. Cell</div> , <div class="volume">27</div>, <div class="fpage">1694</div>–<div class="lpage">705</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differences%20in%20environmental%20stress%20response%20among%20yeasts%20is%20consistent%20with%20species-specific%20lifestyles%2C&amp;author=C.%20Brion&amp;author=D.%20Pflieger&amp;author=S.%20Souali-Crespo&amp;author=A.%20Friedrich&amp;author=J.%20Schacherer&amp;publication_year=2016&amp;journal=Mol.%20Biol.%20Cell&amp;volume=27&amp;pages=1694-705" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1091/mbc.E15-12-0816" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1091%2Fmbc.E15-12-0816" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1091%2Fmbc.E15-12-0816"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27009200" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Linde</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Duggan</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Weber</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Defining the transcriptomic landscape of <em>Candida glabrata</em> by RNA-Seq,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">1392</div>–<div class="lpage">406</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Defining%20the%20transcriptomic%20landscape%20of%20Candida%20glabrata%20by%20RNA-Seq%2C&amp;author=J.%20Linde&amp;author=S.%20Duggan&amp;author=M.%20Weber&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=1392-406" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku1357" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku1357" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku1357"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25586221" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Seoighe</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Yeast genome evolution in the post-genome era,</div> <div class="source">Curr. Opin. Microbiol</div> ., <div class="volume">2</div>, <div class="fpage">548</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Yeast%20genome%20evolution%20in%20the%20post-genome%20era%2C&amp;author=C.%20Seoighe&amp;author=K.H.%20Wolfe&amp;publication_year=1999&amp;journal=Curr.%20Opin.%20Microbiol&amp;volume=2&amp;pages=548-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1369-5274(99)00015-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1369-5274(99)00015-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1369-5274(99)00015-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10508730" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Aury</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Jaillon</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Duret</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">Global trends of whole-genome duplications revealed by the ciliate <em>Paramecium tetraurelia</em>,</div> <div class="source">Nature</div> , <div class="volume">444</div>, <div class="fpage">171</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20trends%20of%20whole-genome%20duplications%20revealed%20by%20the%20ciliate%20Paramecium%20tetraurelia%2C&amp;author=J.M.%20Aury&amp;author=O.%20Jaillon&amp;author=L.%20Duret&amp;publication_year=2006&amp;journal=Nature&amp;volume=444&amp;pages=171-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature05230" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature05230" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature05230"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17086204" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gout</div> <div class="given-names">J.F.</div></div>, <div class="name string-name"><div class="surname">Kahn</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Duret</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Paramecium Post-Genomics</div> <div class="given-names">C.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">The relationship among gene expression, the evolution of gene dosage, and the rate of protein evolution,</div> <div class="source">PLoS Genet</div> ., <div class="volume">6</div>, <div class="fpage">e1000944</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20relationship%20among%20gene%20expression%2C%20the%20evolution%20of%20gene%20dosage%2C%20and%20the%20rate%20of%20protein%20evolution%2C&amp;author=J.F.%20Gout&amp;author=D.%20Kahn&amp;author=L.%20Duret&amp;author=C.%20Paramecium%20Post-Genomics&amp;publication_year=2010&amp;journal=PLoS%20Genet&amp;volume=6&amp;pages=e1000944" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000944" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000944" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000944"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20485561" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">McGrath</div> <div class="given-names">C.L.</div></div>, <div class="name string-name"><div class="surname">Gout</div> <div class="given-names">J.F.</div></div>, <div class="name string-name"><div class="surname">Doak</div> <div class="given-names">T.G.</div></div>, <div class="name string-name"><div class="surname">Yanagi</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Lynch</div> <div class="given-names">M.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Insights into three whole-genome duplications gleaned from the <em>Paramecium caudatum</em> genome sequence,</div> <div class="source">Genetics</div> , <div class="volume">197</div>, <div class="fpage">1417</div>–<div class="lpage">28</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insights%20into%20three%20whole-genome%20duplications%20gleaned%20from%20the%20Paramecium%20caudatum%20genome%20sequence%2C&amp;author=C.L.%20McGrath&amp;author=J.F.%20Gout&amp;author=T.G.%20Doak&amp;author=A.%20Yanagi&amp;author=M.%20Lynch&amp;publication_year=2014&amp;journal=Genetics&amp;volume=197&amp;pages=1417-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.114.163287" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.114.163287" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.114.163287"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24840360" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">McGrath</div> <div class="given-names">C.L.</div></div>, <div class="name string-name"><div class="surname">Gout</div> <div class="given-names">J.F.</div></div>, <div class="name string-name"><div class="surname">Johri</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Doak</div> <div class="given-names">T.G.</div></div>, <div class="name string-name"><div class="surname">Lynch</div> <div class="given-names">M.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Differential retention and divergent resolution of duplicate genes following whole-genome duplication,</div> <div class="source">Genome Res</div> ., <div class="volume">24</div>, <div class="fpage">1665</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20retention%20and%20divergent%20resolution%20of%20duplicate%20genes%20following%20whole-genome%20duplication%2C&amp;author=C.L.%20McGrath&amp;author=J.F.%20Gout&amp;author=P.%20Johri&amp;author=T.G.%20Doak&amp;author=M.%20Lynch&amp;publication_year=2014&amp;journal=Genome%20Res&amp;volume=24&amp;pages=1665-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.173740.114" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.173740.114" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.173740.114"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25085612" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Albert</div> <div class="given-names">F.W.</div></div>, <div class="name string-name"><div class="surname">Muzzey</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Weissman</div> <div class="given-names">J.S.</div></div>, <div class="name string-name"><div class="surname">Kruglyak</div> <div class="given-names">L.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Genetic influences on translation in yeast,</div> <div class="source">PLoS Genet</div> ., <div class="volume">10</div>, <div class="fpage">e1004692</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20influences%20on%20translation%20in%20yeast%2C&amp;author=F.W.%20Albert&amp;author=D.%20Muzzey&amp;author=J.S.%20Weissman&amp;author=L.%20Kruglyak&amp;publication_year=2014&amp;journal=PLoS%20Genet&amp;volume=10&amp;pages=e1004692" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1004692" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1004692" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1004692"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25340754" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gout</div> <div class="given-names">J.F.</div></div>, <div class="name string-name"><div class="surname">Duret</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Kahn</div> <div class="given-names">D.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Differential retention of metabolic genes following whole-genome duplication,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">26</div>, <div class="fpage">1067</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20retention%20of%20metabolic%20genes%20following%20whole-genome%20duplication%2C&amp;author=J.F.%20Gout&amp;author=L.%20Duret&amp;author=D.%20Kahn&amp;publication_year=2009&amp;journal=Mol.%20Biol.%20Evol&amp;volume=26&amp;pages=1067-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msp026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsp026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsp026"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19218282" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Papp</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Pal</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Hurst</div> <div class="given-names">L.D.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Dosage sensitivity and the evolution of gene families in yeast,</div> <div class="source">Nature</div> , <div class="volume">424</div>, <div class="fpage">194</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dosage%20sensitivity%20and%20the%20evolution%20of%20gene%20families%20in%20yeast%2C&amp;author=B.%20Papp&amp;author=C.%20Pal&amp;author=L.D.%20Hurst&amp;publication_year=2003&amp;journal=Nature&amp;volume=424&amp;pages=194-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01771" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01771" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01771"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12853957" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Qian</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Liao</div> <div class="given-names">B.Y.</div></div>, <div class="name string-name"><div class="surname">Chang</div> <div class="given-names">A.Y.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Maintenance of duplicate genes and their functional redundancy by reduced expression,</div> <div class="source">Trends Genet</div> ., <div class="volume">26</div>, <div class="fpage">425</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Maintenance%20of%20duplicate%20genes%20and%20their%20functional%20redundancy%20by%20reduced%20expression%2C&amp;author=W.%20Qian&amp;author=B.Y.%20Liao&amp;author=A.Y.%20Chang&amp;author=J.%20Zhang&amp;publication_year=2010&amp;journal=Trends%20Genet&amp;volume=26&amp;pages=425-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2010.07.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2010.07.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2010.07.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20708291" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Birchler</div> <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Veitia</div> <div class="given-names">R.A.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Gene balance hypothesis: connecting issues of dosage sensitivity across biological disciplines,</div> <div class="source">Proc. Natl. Acad. Sci. USA</div> ., <div class="volume">109</div>, <div class="fpage">14746</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20balance%20hypothesis%3A%20connecting%20issues%20of%20dosage%20sensitivity%20across%20biological%20disciplines%2C&amp;author=J.A.%20Birchler&amp;author=R.A.%20Veitia&amp;publication_year=2012&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=109&amp;pages=14746-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1207726109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1207726109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1207726109"> </span></div> </div></div></div></div></div><div content-id="dsx025-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Wong</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Turner</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Cho</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Wodak</div> <div class="given-names">S.J.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Up-to-date catalogues of yeast protein complexes,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">37</div>, <div class="fpage">825</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Up-to-date%20catalogues%20of%20yeast%20protein%20complexes%2C&amp;author=S.%20Pu&amp;author=J.%20Wong&amp;author=B.%20Turner&amp;author=E.%20Cho&amp;author=S.J.%20Wodak&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res&amp;volume=37&amp;pages=825-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn1005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn1005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn1005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19095691" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Scannell</div> <div class="given-names">D.R.</div></div>, <div class="name string-name"><div class="surname">Byrne</div> <div class="given-names">K.P.</div></div>, <div class="name string-name"><div class="surname">Gordon</div> <div class="given-names">J.L.</div></div>, <div class="name string-name"><div class="surname">Wong</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Multiple rounds of speciation associated with reciprocal gene loss in polyploid yeasts,</div> <div class="source">Nature</div> , <div class="volume">440</div>, <div class="fpage">341</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20rounds%20of%20speciation%20associated%20with%20reciprocal%20gene%20loss%20in%20polyploid%20yeasts%2C&amp;author=D.R.%20Scannell&amp;author=K.P.%20Byrne&amp;author=J.L.%20Gordon&amp;author=S.%20Wong&amp;author=K.H.%20Wolfe&amp;publication_year=2006&amp;journal=Nature&amp;volume=440&amp;pages=341-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature04562" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature04562" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature04562"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16541074" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Landry</div> <div class="given-names">C.R.</div></div>, <div class="name string-name"><div class="surname">Lemos</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Rifkin</div> <div class="given-names">S.A.</div></div>, <div class="name string-name"><div class="surname">Dickinson</div> <div class="given-names">W.J.</div></div>, <div class="name string-name"><div class="surname">Hartl</div> <div class="given-names">D.L.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Genetic properties influencing the evolvability of gene expression,</div> <div class="source">Science</div> , <div class="volume">317</div>, <div class="fpage">118</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20properties%20influencing%20the%20evolvability%20of%20gene%20expression%2C&amp;author=C.R.%20Landry&amp;author=B.%20Lemos&amp;author=S.A.%20Rifkin&amp;author=W.J.%20Dickinson&amp;author=D.L.%20Hartl&amp;publication_year=2007&amp;journal=Science&amp;volume=317&amp;pages=118-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1140247" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1140247" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1140247"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17525304" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lehner</div> <div class="given-names">B.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Conflict between noise and plasticity in yeast,</div> <div class="source">PLoS Genet</div> ., <div class="volume">6</div>, <div class="fpage">e1001185</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conflict%20between%20noise%20and%20plasticity%20in%20yeast%2C&amp;author=B.%20Lehner&amp;publication_year=2010&amp;journal=PLoS%20Genet&amp;volume=6&amp;pages=e1001185" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1001185" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1001185" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1001185"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21079670" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mattenberger</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Sabater-Munoz</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Hallsworth</div> <div class="given-names">J.E.</div></div>, <div class="name string-name"><div class="surname">Fares</div> <div class="given-names">M.A.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Glycerol stress in Saccharomyces cerevisiae: cellular responses and evolved adaptations</div>, <div class="source">Environ. Microbiol</div> , <div class="volume">19</div>(<div class="issue">3</div>), <div class="fpage">990</div>–<div class="lpage">1007</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Glycerol%20stress%20in%20Saccharomyces%20cerevisiae%3A%20cellular%20responses%20and%20evolved%20adaptations&amp;author=F.%20Mattenberger&amp;author=B.%20Sabater-Munoz&amp;author=J.E.%20Hallsworth&amp;author=M.A.%20Fares&amp;publication_year=2017&amp;journal=Environ.%20Microbiol&amp;volume=19&amp;pages=990-1007" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/1462-2920.13603" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2F1462-2920.13603" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2F1462-2920.13603"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27871139" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mattenberger</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Sabater-Munoz</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Toft</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Fares</div> <div class="given-names">M.A.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">The phenotypic plasticity of duplicated genes in <em>Saccharomyces cerevisiae</em> and the origin of adaptations,</div> <div class="source">G3 (Bethesda)</div> , <div class="volume">7</div>, <div class="fpage">63</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20phenotypic%20plasticity%20of%20duplicated%20genes%20in%20Saccharomyces%20cerevisiae%20and%20the%20origin%20of%20adaptations%2C&amp;author=F.%20Mattenberger&amp;author=B.%20Sabater-Munoz&amp;author=C.%20Toft&amp;author=M.A.%20Fares&amp;publication_year=2017&amp;journal=G3%20(Bethesda)&amp;volume=7&amp;pages=63-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.116.035329" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.116.035329" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.116.035329"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27799339" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mattenberger</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Sabater-Munoz</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Hallsworth</div> <div class="given-names">J.E.</div></div>, <div class="name string-name"><div class="surname">Fares</div> <div class="given-names">M.A.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Glycerol stress in Saccharomyces cerevisiae: cellular responses and evolved adaptations,</div> <div class="source">Environ. Microbiol</div> ., <div class="volume">19</div>, <div class="fpage">990</div>–<div class="lpage">1007</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Glycerol%20stress%20in%20Saccharomyces%20cerevisiae%3A%20cellular%20responses%20and%20evolved%20adaptations%2C&amp;author=F.%20Mattenberger&amp;author=B.%20Sabater-Munoz&amp;author=J.E.%20Hallsworth&amp;author=M.A.%20Fares&amp;publication_year=2017&amp;journal=Environ.%20Microbiol&amp;volume=19&amp;pages=990-1007" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/1462-2920.13603" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2F1462-2920.13603" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2F1462-2920.13603"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27871139" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Blake</div> <div class="given-names">W.J.</div></div>, <div class="name string-name"><div class="surname">Balazsi</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Kohanski</div> <div class="given-names">M.A.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">Phenotypic consequences of promoter-mediated transcriptional noise,</div> <div class="source">Mol. Cell</div> , <div class="volume">24</div>, <div class="fpage">853</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phenotypic%20consequences%20of%20promoter-mediated%20transcriptional%20noise%2C&amp;author=W.J.%20Blake&amp;author=G.%20Balazsi&amp;author=M.A.%20Kohanski&amp;publication_year=2006&amp;journal=Mol.%20Cell&amp;volume=24&amp;pages=853-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2006.11.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2006.11.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2006.11.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17189188" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Raser</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">O'Shea</div> <div class="given-names">E.K.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Noise in gene expression: origins, consequences, and control,</div> <div class="source">Science</div> , <div class="volume">309</div>, <div class="fpage">2010</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Noise%20in%20gene%20expression%3A%20origins%2C%20consequences%2C%20and%20control%2C&amp;author=J.M.%20Raser&amp;author=E.K.%20O'Shea&amp;publication_year=2005&amp;journal=Science&amp;volume=309&amp;pages=2010-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1105891" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1105891" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1105891"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16179466" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Newman</div> <div class="given-names">J.R.</div></div>, <div class="name string-name"><div class="surname">Ghaemmaghami</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ihmels</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">Single-cell proteomic analysis of S. cerevisiae reveals the architecture of biological noise,</div> <div class="source">Nature</div> , <div class="volume">441</div>, <div class="fpage">840</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single-cell%20proteomic%20analysis%20of%20S.%20cerevisiae%20reveals%20the%20architecture%20of%20biological%20noise%2C&amp;author=J.R.%20Newman&amp;author=S.%20Ghaemmaghami&amp;author=J.%20Ihmels&amp;publication_year=2006&amp;journal=Nature&amp;volume=441&amp;pages=840-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature04785" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature04785" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature04785"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16699522" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx025-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx025-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tirosh</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Weinberger</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Carmi</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Barkai</div> <div class="given-names">N.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">A genetic signature of interspecies variations in gene expression,</div> <div class="source">Nat. Genet</div> ., <div class="volume">38</div>, <div class="fpage">830</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genetic%20signature%20of%20interspecies%20variations%20in%20gene%20expression%2C&amp;author=I.%20Tirosh&amp;author=A.%20Weinberger&amp;author=M.%20Carmi&amp;author=N.%20Barkai&amp;publication_year=2006&amp;journal=Nat.%20Genet&amp;volume=38&amp;pages=830-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1819" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1819" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1819"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16783381" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=3868561&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables1_dsx025.xls?Expires=2147483647&amp;Signature=nPaJXHYV6gQ53wdehHE7rsx-iQX0ETTZVzTjn5AmU2zebm8Vy102UjU827wCKDXjTAy9Rw5tHgpdB24X3VQfxnDGbDWNWgQqU-1RIWH87wtlk8HEB~nZXuEpFyi9FlceBEO7TR8mceWMN2rApEkpPf9k3IcugRkKGNWhyrPWUbrj6izwPoG0p7vTYZt13G7yv9G8dktdeUpRxvuF3OFB9UF9p4VQCrgDHJZFyxLMZ9lEkCUn9wD-295ni3IcoZ~85QXn7LHT~ELwTDUdqmWzq8lV~K9rIh7KsVPdATkyvhw2EVan6TBFpv5NPJQ5nSe1xtd976ljXnpi8-ywkvJfxw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S1</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables2_dsx025.xls?Expires=2147483647&amp;Signature=nzVQrockR3wyu0qAo2ciLBBoHXyCiMFz9iITggWbpEe6OfiL3sNeDX1fMXh5tHPpfbeL0l3x2Qih7p7owRicUe4Y~nZ4Tsc1WdgalxzdMGblT8ojWMrccQ~K~-~c6VsH8bDdjajZDd4nc6gYB-LjbJPEOksKWbNARUI-Xkt2FF1u4dU9LkjI0az~FTN1J3jO6YJ5f3UT0ap4WkmKKDaLszAbs4SmCmXwWlOzNUsEBWBYJVShiCzV1PJyZxIkv2v2YuJW6YYEQWNd0GQWrxqvV6fqcImtrj~heQe6Dq4UO5VJFrWB6MyQtdM6eS9sfNBTIyvIoaHVVhukcz8upTWNIw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S2</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables3_dsx025.xls?Expires=2147483647&amp;Signature=llkQGfG~w8wzukk92Mq0hOCkFc0nsnCCzBf9f3StqlSnhXxXBQ4lmLMjSXZz71vMIT-4vGExXWD-ziWvldHL-QgWkt-GlzjkctbDm18uTNNmVpxxpvmluZ5Os-muTRahSIhkyGWw~mVWg8-7~tcqonscpoMj5-eCAo6jylJV~08sh2pGRAUYmDkx37vkTnqJIa~Luf2cwLeBNh0Qa-b~HSVuDcvgEDwcCfJAPyfd0PmjX-jZwanaXfG5S1lZU9Sylp2fHrZx5bsZzFagJtYWLdkJtHXIwj6gJk9madl4xCwuGD6m-m32ql1MZBacuIbhxjD35kDRUvfh9UznFLpO0Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S3</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables4_dsx025.xls?Expires=2147483647&amp;Signature=du~ll4wTPKL5K5g7AHRPqwblsCMJEv5cjKBC8E9f5yBP2dbcp5q9V3tCnuGTKGHGNBYckv2c3MLFqNn3q50VNEd1jFW6VRapKUNr2cg42SN9NOuw2PTmX2cM0KMzK4Ym1hCXkzU9xobbhT22BqHzME0KwY4qycOpHXCMCKiZjdNK3wpC9PeywRu1tYW-6Ph0GfNsM2DR7~IWEFWSYmANoITLYCdkixyf9e4fuPfO5mOcJqG5o1k71a0LUuUpFNhRfiA8WZC-hZbQPzV591rP0vNpsrFvg34TEzL~je4gw~icswrgXqXCcqjWUsjs0JUmwA5sBb22VH52q1rbJjtqiA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S4</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables5_dsx025.xlsx?Expires=2147483647&amp;Signature=Lr9-K9bUaET87gTU2y6WeX8NfUeVV-VQNgqmGdQOMGKvTtoxNrDVu7~pwMVcGu6266LxmNS1PnEPXRk~YcKCmy0Hmqr6m9jmYH-19eQNKBgomnll9I7Xh-4ZTHz5lFwkf6GkrGAReeav9cDscICWxQD1K3lv35k7zdVB7XaKVhCPlpxNvkB4nvJgUU6oek~Q6CAS8p42OHNF3x4JXUiYiUDH~x~yVq8oJuEp2an0y7Rb4Yl6hRd2Dkx8rdoa6wKuEA~nfV7x6pWD8MLznu8Qi0~uI7lVkNZW-efuzu54Z7Ql7ei3dq7nSSnIrUC0G9Kvmi51y3wH7Mpo4ruisecmuQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S5</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables6_dsx025.xls?Expires=2147483647&amp;Signature=CLIUJaLyigYLq7rEkzJembnre1zrj5t~H19D4ZNWa6fM2o792BFtRXKqvcZJLL4BkoNkoHcBlqdG99wBzuKVC7K3sWWdZfjkmZEH2ODM-TXpafvfOkcr7BF7c9TB9VV3hWyQSyFoD412F6Wtcet8PgeJFbXM~EqMNVRzenq48d5EGAjrcAKk~vBauk0AMavGBVYK0-DrWcqsDGZfs9pZ35JUhYNU8~hx2S-svLd-i1BR8hm3O3pmYLGJ0Gd~iVtROIiCya~50esHtRpj8ADCKJxqZdNh0Awy1BfvlcVVQJ8Y3ts0sdrn3Qpo-YBcwJ9wrBoWmke0YLx7HLT0dyTviQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S6</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables7_dsx025.xls?Expires=2147483647&amp;Signature=uM1oaLXJJc-NU6zJaRE6-13I926wV0n-SBUHGQvAvPBXHhaySF-005YTQDKtMX~tZWB8-nCqZHi5IQNyR0e606QB1QQwSxq7yxVMN4badYRgE3srzksixUB3KNrXngDVq8ksfHxIhc7ViKona6uFhTkvmqLTqOQXydIODELSSUDg5dv2gFC7dZXxCsaXFxM6~UM2F~9xaPSXfN~AZ9hdfZI8dleB5529MoDi3o-KOq689yQk53m7jj1HcOlsZal8y~wzBDcXbPCz3yR-p9iMQpaEDRzvxWoOMh7twxkpg8REN5l-FsclrwqY-zY1GlYe2-uyn2tJLuuCcb09QYLH7g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S7</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables8_dsx025.xlsx?Expires=2147483647&amp;Signature=dvik8lEucGyEXr1zUF~bCcQUMbwKDkuCB5vdcUQWdjvHBDnz-u2e2m8tQgRe56XUOGfaTQfIPM3qPjRbsjb8lFKgNh6xHzdwfoVg3TFp5qHh-29uIzLg3dfATl~DQoZM~cqCKwFsuXvlLIuJTAv-G-0SwcsMUyAzOzVxx9BOtK8J777hgo40DwVzV6sdYSnkhhnhbQfKeM1XzpFKdmgz55uBYTT9LCSoDXrnqN0tHwDl0jpcngb6qUkDLoVavoTMmc1Qo2lCnHXxPxMHz3vUPNbHS5bv7paJeeE~RHJzwdISd~3aab-61XFvScdDxhepZ2BTtEn~AYgqi9uBhxTmIg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S8</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables9_dsx025.xls?Expires=2147483647&amp;Signature=JQZteoMM2HYroN2GkZ423jAtKrsmLYLi~NH1TLcUPlzudK4HtOHZd0QqNkPmiH3jNe85eDw~UFgqKAS~zJSJxo6txuAi65Em5aOw6xDcRJDrhWGRe1tvbC~A-Wlr-vmeZhHZwBaYoKmKamkbHcij9gNv2EHe9mlt0FvURfsnzqoJecEYTRk1q8eaFd2w4zYe8DLMTrhkO0xyCvPScqZ~8GcZcwOwkFyDtW1SqGsKS6LCp-9t4a-~TGd13JYw~jf3JngEhD5Czs5FaYnmK4lchm218WV0~VxjlbqYiZI~938tfnRxKqgLI~eA0YQI4zfdx8tagDStmXGpNRNmTSALiA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S9</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables10_dsx025.xls?Expires=2147483647&amp;Signature=debGbxRXYpaemCUHS4TWDVnJxtf4o8z9C4QjNxPvxzdAo0IOpIyzFUEL8pDQDq1bnnBY6ilAEYkIc0GhfB-DACWRRHoaMzTQ0EgHBL6OF-Y7oEKRPl9JRf0TrKobTpJXfhnOZkcpTD-jCql~hwxDrrVsK4GXg-g8725vQQPbeP783rs6uwubEtf4zsRrxaksiuNY4fGfTa10kbnjHyXc-kVVPQusJxXoR9vBRg6FfFB7em~m0aS349x-WFNmbY5k4zNJ7m29f1CvctqFnfhcS4HGjM-FonJZMijX7Na6A2dD0QaLCsrPL9Ckv35xaZa5~RTWExUOTis3WtOzGlj73w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S10</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables11_dsx025.xls?Expires=2147483647&amp;Signature=SGZwYk-31ltXxPadHG1Ia4NSpSn-XqbXexl8eXzkIJ7FWbKDa3XDbB5I-BCSAqyRKh16DcQaZrQQq0sV0HGe6dn-LqZqsauLLjkMoSC-~Vv7ZhgFmOGluvPlse9miT6-RLe-MfMt~AfsRvPe~eN~6uLX~2XMVQal~Rs~~Qh-Bs7alLX6Uyc8EGpUd1iR54~xmZ1lFyfLAmskB-1ar6aR-KT4u77SNeZW85G64srY0aXTQhJhuTxiCDK4WApBP6yn1P~sTSGs1JZLwrvtpK3495a6ekafQbucUBNNYZh3qu8UYWED3dHcrVoySXzs8oILlDE5WIsszGT1bSRJ6Exc5Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S11</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables12_dsx025.xls?Expires=2147483647&amp;Signature=K0JoQP6YXOfC3uHnFtUt8uDQ7ly4OMUnQD2svxQVGiIsNeWPiBF01~-vOdLR2vBBz7flJeOliD35saVg3gr-rWzinEkTzwElpKKQK9EmsHCUWKOXleVP6r1zepGImFBQfNXWl17cO0s4tZbSmKzPBahma7ELJZlO2tSQ0cft~BY5vtHlqpdA-PKbcUWTwxa02lUEkEJHJJBmkSSvhfs0SKF9GymiZ7dDVNFDjZAUWg1j9HpAdDw~vogPc-ad78GZj4K-WlbwlFSVFWfyevT93Ntf2iWYWgVpnOxiatrrijJSx4ZReb~ujoKMAUXabQ7XzQYa9FUSRlWN2dSOfVdbNw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S12</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables13_dsx025.xls?Expires=2147483647&amp;Signature=EL9yQt9SvNc8twLbVREamdzLP9oNy1k3f3lUSO6KTPeVoG3R6XP9yLbN7ND4~RO52VnjJhbw--IKJbsNzh1iocl3rRz4~fKolIqktt55MtoZ5lFrSmR~pWQ7fKWE9HpOmzhd92eNmpgyUk6u4-WLtjtqHDpdXULKRtWBDirnO2qQEf8JOXI7J0Ip2ka53KZVwiTxPY6L1l7Gln6gR6kryY~Rhje5MUZvvVkAshK19Np8JS81KL4eJ5LplLXv4Smj1hyxIyQVGQRLBxWJreRIZ6MwdB1~Q4WWiXdL3to~P9pF5t1Wm~f~PrXmVPFJn11fPer57FW52JHrJ3bmda6yPg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S13</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables14_dsx025.xls?Expires=2147483647&amp;Signature=AiIrZgBhu03z6s3Gu481WG0GCUHaCwV866Z1KVR5-zJ8n8kHkd24C7sc30y3JVdaj13HSK-qZbTV4FBcQvbJ5OYTaDys0p~QBehH4ztrirHjFoLiZz5Vf0Ft5uh2KLL-INUVahLecAkKZe3U2PLR-5-VWWQhKVi6rKzAqQ3QN3H2nJeD-yg7bZlG1k8hph6Ji4Z0UkKcECPR57ZfU9L28OEhfH5-HtSbv1Tb84dV~1K9hXp~1Td93Of2C23k-E3T5Bzk66JuJuiR9j2mj5ojnvVZ~vewQxy7h2hEirRsYMlfGF~lzjrO8mWJ3Kh5RVHMvOpLs5XYHofSI89pGg7Hrw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S14</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables15_dsx025.xls?Expires=2147483647&amp;Signature=vv5qBqYoxav5M7Rs8Hyr4BLN15iXokVtqjcPO-8ytilDjYo~-q0n~5pqrQ-MwhUtWNCfFElig1ggW5LUu~WYJ2AbtVYA7puWj8rx5SsQqRnCWGFkBCTPQYr4CSaROa4OtVbeuBUx2rT3~mJTrnn0S64NKmg-1LyyBVrDniImw21hf5UFwFqHgw9YQKMTz23vzhb~TeP6hzxXITYtQESJCDkXbl6K-P9oELUVkHdhEoV34JHM-UmuUNZfOhEl-Htf~bgclnsJTHLSKIRJguCifvNFAd8wLqap2TykID00ZfXmP6jX3Bfs-iSKasq3AZ48kUO-vUEgZoz94U2083984w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S15</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables16_dsx025.xls?Expires=2147483647&amp;Signature=p62uTG2usUIXo6VIS7nTR-KAYpowbseWoay2ljTNMs6J9vuot87WYbnvS73Lh-g3GAFEtuKT6NFkzfqyVZagOOhqDP3OsgU7aRi9-h6Y9RXPlBLzbh0y1QcLTgwsjWhhwRzbp48eYDbXFWe0c1qRQIozA-wROf1Vf7FNKkCQBIIcArWVMDZRWnkkQWYR8j287WuCfz9LHIclxMdS6e-FR6pXsDNr0MBi0T2YpK~yTkdSCdR9AF6Pef5PCFtA2MtMeB6UELQMt5LRaAjhr7laCx-m4JdrXVrEWUSlKW~3h1wYmiSTsHol~ltNqzVqh30P-PrBG20YnKOWPgBh8fcFFg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S16</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables17_dsx025.xls?Expires=2147483647&amp;Signature=yzOMYkwUG3F29nPbUttCAP8xftBd4KvIwAlCVp39i2yCLBheXQ8fcHjKrERzrsdxDDjpfpOPkAkjLLW6T4PJERiDuMx99k6vVDdCdAW-LwjNFDXSrASqggug6xcLbvnlGpw0Jso2E8Gs5IGde1rWus~Pu0tHdtMDkPz-~Gi3-tyKm8AF8Hhj1uRroh0YyT7xgL7oU2YOdCS-qgd3pUNJ-9eqvl8UCfaZX6vLlHzx6~RjUPfY7gNYs1RysQ5z5JyolemL3rtvFWbrEPD65Eh-dUGpWz01-DH13OGA9kkX2uxROPO4R98J3~~s1MO735y69iWWtIDUi5pjMwAxMSUzMA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S17</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables18_dsx025.xls?Expires=2147483647&amp;Signature=uDYYkwxVD1UDEcG1kjc~Lq3EOW3z~vG2ZRbAKiWzA5VbDSC5efwPjeJAyVmnZSJZgN1PFFgMLOgtqYUf-qKgHzJwYXNj47JLSe0HatFh-e6pTORs83fLis6MZsCpwsEtgrh1d3jGGmxdp5tx2UiZumW1I0VggvqbRxQ5giRhe4etVhaUoBcHBMwjiEnnLkX~x6LIckvrJL24PDqxZz0RMXI2i7OcexzXU~AXmQQni260GYODQvXex8ih107freGWt~2iS3mENUpF4-sfCp4uww0djZzJ~-J-eVCGEuPzad6849EE6pJfboYteqyuj4PMWq2yXfz6-9eHRLu3QtFNlQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S18</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables19_dsx025.xls?Expires=2147483647&amp;Signature=f6EVo1iiwEnJyOjPQYwBOhVtSRfpLUNk2loYMYf4GKNWmFM6iixOXDAb5W9MD11nWKm4wrRVcNi6tn-EWX6tkklIKg9xqiUClKZBPyY28inNei3oa3DvHxbVe4zDPIDLgBnOeqLRxDAROsyhePOnX3JpSTvUsfuspxQzLS3UC7zv2ZB4sLIFY1irGa2bl7zRiDufvy5068ogbc6T-QRsyaxMtCR2m60~vlhEHzniaNdA~5QQkQSAvHFOoLSFfRhipb2akw-q2X0QQpMRMQoCaV~7vwHsuVunqSjAcQxa5X9SNUWclx7m1OtuwTEkt5nTIHEoLc8aOYmrOCkLeZhwpA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S19</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables20_dsx025.xls?Expires=2147483647&amp;Signature=LTN23eCjhSUhwJZF8SuFlvny8j1ybHZ~tUrMs-j24CYJ5gBeuVQgxvF3y9uCD~6vRRfWIC6o5vQSNQ5fQSLewim7Nl5k~Ue7e~JsG6HnVitcgCV0Lo8STxMDaBx63UQadh0NYiN871Tu3naUWI0fABUITNqKFUZSeCujZyjYcDXz7pmQIvJRy29d3muDE8ksoDeGE~gPZd1kQeGutuyBPsD89x54jh1iLX9BbKrrpAqWLLMJtCdnA77Y4pKTIF5iPZUbkeTOn9cETX9lO71HYhNq29DpdMwSg0olUHGVk1rXWodcGgqJVTW5f8ibISGpcFIxTi~jiShL6O945uhcXQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S20</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables21_dsx025.xls?Expires=2147483647&amp;Signature=qzYuiuFhrQ2Cv6uTL8pRai4DhZe5XYtIlwRJxTlKLNZPnNhKFmmTNeVaEUtJsdRFWsWit82nj~w~5vOVXNtUawjZPY4K76brRyejQaEgdxE0x1Uc4TBDVslC9OzpbkXZjCGF0UhzsJp0ZXZt9oQWbwi1FsheziSOmKi5h6dYToV3AncM6tHGHAs5Vw25GTaZoL-YhawftUV8-PtbHE19J0S4hZi-RsuAHPxAhfA-1Que922XZaCQ6PDV4snR9~UhYUOaZiNrZdZqxmxsd92n~1~Swx3f3EpgOzbtMeuDYyrPxZtBfPzc7bUR4Bh~3di-X~fNS5hjEdjdu~1bX2lDqw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S21</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables22_dsx025.xls?Expires=2147483647&amp;Signature=N39Ubsj1Rk1lxFjREcICGO5f7lsWJhR5ZHC1r74p6gNsj-bZwB9rVL77WeoCnxFf7qgJVJztTbd5TmpV2LS4JLXMzz5KfeL1YfD5~chFd6MVjHFqX6ETsEIY2DLDUL40TMCaIxWt8IsbIr~g56odLhwLS1ZpD32UNX46joMkPVIHoEkFjYkGzlDPkfCDuXQWtU3z77eN-fWWGqqhS6kgeX1mKSS681EBTnf7b5XUjj0F4yrlow3CikzIdVZXF5EPaGxQPs-cWAmPmRzOoZ~CiJNkQLCX4ZRxURjCReyGn5q9PmepbUPIPrZPqyWxt5UsCl9iNK1k0NJ5DK0TLh~wHg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S22</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables23_dsx025.xls?Expires=2147483647&amp;Signature=FXT2Sjll3yv~P2enJyhILLejYLBUGRS0Cju9gvm3mP-KeoUr4uUWmzod1xSJuRdms7IDk-fC4eo~jTHysMY7iPN-nilzwp2algfjvJ1yl~6sjVNwxKcfCVjyRXN7AmfPp1eMf7CIai2L~Mypqap95OV~qOP~Z9WALECsnMHUmwaaNEWHI7g5TKfcis8yDV7Vm49a-7Qa~uroL0s4fDHmqmx64BFHgbaDk511PzdV8IxZVFI7aw8BbDg5sPZu4ve5VKeWDRHNRrBXzRPDaRmzX4plh2z0DLDgJ-L6mOCA0qQqSY2lp6lygMOUAImny4aCerQUq2KcdikNBMNsW-4JVw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S23</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables24_dsx025.xls?Expires=2147483647&amp;Signature=nqq4F7jz0oB7FhpZ0OcuQsoitxBN17clrtey-AVEj48RBf3zvRS7zad1hQitN2WRvhhDoH2PoSCO5GAZSoN6GZ7lOAvZpuINq5cFstpf6Aij-uPqguwhe4hWqokSBw1D7QVnQm36ozeYQUOZw9yU4O6MUlKpN-q25FshY65VC2plK8XLyVdg3BESMWQmWj1xwUGm9lIPj7uZUjbUWVBjnF0t3s-EWNj6E5rfFHf~GDqSNLKEbPrUEYrnZufXj9Ugv-Xkh4FwD578SjI~5o0Nqz2pN8WTnmk2HvXJE6p0XkK9sbTX4H21X0CN9yDLxL~GysJ7MiGevvghqmkejlVHyA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S24</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables25_dsx025.xls?Expires=2147483647&amp;Signature=n8eJtZkSGiqvbaqiSuw5aJkqUYh2Vrl4Om1uEOlwMtsY93U2A5isjKCqZoDLBzbAv00FceMrUXmG2EgNtzGonLIwg2EA7LC2j~136rWOM1~ZDOBWuCROvI0XbdKmsPmKhiUHS2mwOxU9lNPoQVoNzDcPf~riNeaiFYJH8GEodKcBpqPqFRB2ZTNfKjYp7OiFE9U40B1votVMtzjxOWuiQEwLvWrI2sDsPXlVJEPs1LdYto~J1BmE4-Avt2hGcOPDbfCJuby5k3I8tK9MLruJf7u3xheSXBcPYrcaT-QvNeWsv-AlKfmF5r14z5dKNqoqMJW6vvZ5O4lB0VyeUpuGdA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S25</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables26_dsx025.xls?Expires=2147483647&amp;Signature=v~YbrHGzefKER~B-yzFxe0~XsVrWGLvcx0iBC7GC6lws7l4xPgDdVRwrF1TNP2c8YnR~JRO4QtyWbu4oq7fJi-RtRpwY5g9Rx9s9rMXzyLIZGRMyiWnddxxyh0ncDZtal4nypYpVrC35fl32rI5mr4JTYdiBN5YOCwslqbVWjVCPRciHZerf42l9OY~HjEjRuJcayrmcv6HPtOC~Jmc19hLYzGr7-vdxA5reWv-6jHAihzTIU16SgkT-Cz8Y997GdoQttNxTIMqIJx5n-Eiq0G8XRYFSdFQa8emqRoeOJxUPvq5s9gi9ppw8desdYoz0OgaZ0b320U-tkS9u~QfKTw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S26</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementarytables27_dsx025.xls?Expires=2147483647&amp;Signature=YYBwrPZkt9Rh2q-qhR4YNI4Yp~Hmct-K0gz71WZ6KyvFZPUrFxlG8PL12xTVFZvdpSO0hD2xT2SczEgWpMIiD3XMUYoHbcBosNMWSEVk4zIH7jsJSETZxDE46WhIsI1qlJ1Ks6Zhn6qTpeBRLyB4jwGebm5jezle9RY6fYhbAkD0ZF~e5hIUY-zsIfGkoQzdkmvHIUDd8AamLs0qXdLJ9jyuXRdcmnWDN0DlJFjas9b0V7b6~77NoVHYmK-c39HAJGQFX2VwlgZeekoUPiM9i6-S7eQwoM70fOAFLf0R2wArjyc~Du6Hc1mujZcgAR~HUxCx~6Y3kPUrdYhEl8aLsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S27</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/figures1_dsx025.pdf?Expires=2147483647&amp;Signature=facQUC2BhmdpBrFQ5azyDi8U5YW-tOBKgyZ2xEuTjcQpc3kX0WubN-mflT6oGcChJmJbIpVPAgd6Wxya3DT8nEASg3yGKpUN~ZlNb0zkZRyhdLLNdzXf0GPnQadacKL-OJhqp3~qJBGliRKfnEHSaxiaKBH~tZUX7tycYycMehkjSsTodTCTgcmnEE7GTI4yiGfIb-TobrBhOHoVZBVQjyjx2asaPITZpSScyUBb74SnhUH30V2BjY-94nt2AE946Tm720WMhzbscD3c7bCmt-seeA1qzNxdTCTUKu3vKp7QY4mpp5lKE0UUrinrKal0-f8drIuhLW5UbgE80buifA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S1</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx025/1/supplementary_dataset1_dsx025.pdf?Expires=2147483647&amp;Signature=Yt5wmLlppgvB1A1B~TTHFlVJAyllbCJ3Rzi8NHV8THtyvLU5-NV-E22fGWHITi9lmzdwI0vrSk3DONaH9GbomwAduS7tRkh27WCTh1fX4Wcp8ZZ7nMg~XXC5zkX01sExxEITY58WCkkeeD6qnY49vPSCDwJnUmmb3cui2gjlY7GP4D59kZjs6YuFgGCA2UJg3-XGUbSSE8lHTbiLEO-yb6sgAGiAk9qc1YtiWIF6wf~-cNc0wPZJZY-FPimJ6yF~Moduqi-yIX1d4BIj64YRYt-mu0DRlznhbfw2lUUyAG1gio4sGXtPNqvyQevoPF8E5jEMkcJQSO163HQ8uXdcgw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data S1</div></a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">614</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx025"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="3868561"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f6%2f559%2f3868561">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F6%2F559%2F3868561" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/3868561'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Expression properties exhibit correlated patterns with the fate of duplicated genes, their divergence, and transcriptional plasticity in Saccharomycotina" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx025");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f559%2f3868561">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

