

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/6/583/420514","@type":"ScholarlyArticle","name":"Precise Sequential DNA Ligation on A Solid Substrate: Solid-Based Rapid Sequential Ligation of Multiple DNA Molecules","datePublished":"2013-07-29","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2013-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst032","keywords":["DNA ligation","cloning","multiple DNA fragment assembly","functional genomics"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/6/583/420514","author":[{"name":"Takita, Eiji","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan    2  Research Association for Biotechnology ,  Nishishinbashi Yasuda Union Bldg, 2-4-2 Nishi-shinbashi, Minato-ku, Tokyo 105-0003 ,  Japan    3  Graduate School of Biological Science ,  Nara Institute of Science and Technology ,  8916-5 Takayama, Ikoma, Nara 630-0192 ,  Japan","@type":"Person"},{"name":"Kohda, Katsunori","affiliation":"4  Biotechnology Laboratory ,  Toyota Central Research and Development Laboratories ,  41-1 Yokomichi, Nagakute, Aichi 480-1192 ,  Japan","@type":"Person"},{"name":"Tomatsu, Hajime","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Hanano, Shigeru","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Moriya, Kanami","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Hosouchi, Tsutomu","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Sakurai, Nozomu","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Suzuki, Hideyuki","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Shinmyo, Atsuhiko","affiliation":"3  Graduate School of Biological Science ,  Nara Institute of Science and Technology ,  8916-5 Takayama, Ikoma, Nara 630-0192 ,  Japan","@type":"Person"},{"name":"Shibata, Daisuke","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"}],"description":"Abstract.  Ligation, the joining of DNA fragments, is a fundamental procedure in molecular cloning and is indispensable to the production of genetically modifie","pageStart":"583","pageEnd":"592","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/6/1/m_cover.gif?Expires=2147483647&Signature=VPhijSxXdAPLpw4YhUouOC1tx3X8x9GMJFVJIau9z4hN5cdWK-jPPF1wEurkS1gh6aQR9TQYLHOUG1kzMtF4HcnoT9E-wJJg-rleKg0FU2PY4tefYkghKgj~tiS7jfzXHN~ky5UBYwxoXbHB1rhYpbzfxlAWCYEab07wd~Go~sCK2Z9WKtR5wEAnnMAXr1I8E45jivf904J4XT3ytlfCnSQKIVeT43OyeE29BJRkSUT0SEXeMt6-ucAdFaiaEOPvq~y1xtuGK9d7YWhoevE9wdrzD7fhQ5jWGawbllCtUUH0ojTdHlsKW2bYFxcwuER72KVIwWBylO57igmYb52w8A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Precise Sequential DNA Ligation on A Solid Substrate: Solid-Based Rapid Sequential Ligation of Multiple DNA Molecules","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/6/1/m_cover.gif?Expires=2147483647&Signature=VPhijSxXdAPLpw4YhUouOC1tx3X8x9GMJFVJIau9z4hN5cdWK-jPPF1wEurkS1gh6aQR9TQYLHOUG1kzMtF4HcnoT9E-wJJg-rleKg0FU2PY4tefYkghKgj~tiS7jfzXHN~ky5UBYwxoXbHB1rhYpbzfxlAWCYEab07wd~Go~sCK2Z9WKtR5wEAnnMAXr1I8E45jivf904J4XT3ytlfCnSQKIVeT43OyeE29BJRkSUT0SEXeMt6-ucAdFaiaEOPvq~y1xtuGK9d7YWhoevE9wdrzD7fhQ5jWGawbllCtUUH0ojTdHlsKW2bYFxcwuER72KVIwWBylO57igmYb52w8A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Precise Sequential DNA Ligation on A Solid Substrate: Solid-Based Rapid Sequential Ligation of Multiple DNA Molecules" />
    <meta property="og:description" content="Abstract.  Ligation, the joining of DNA fragments, is a fundamental procedure in molecular cloning and is indispensable to the production of genetically modifie" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/6/583/420514" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-07-29" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Takita, Eiji" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author_institution" content="Research Association for Biotechnology, Nishishinbashi Yasuda Union Bldg, 2-4-2 Nishi-shinbashi, Minato-ku, Tokyo 105-0003, Japan" /><meta name="citation_author_institution" content="Graduate School of Biological Science, Nara Institute of Science and Technology, 8916-5 Takayama, Ikoma, Nara 630-0192, Japan" /><meta name="citation_author" content="Kohda, Katsunori" /><meta name="citation_author_institution" content="Biotechnology Laboratory, Toyota Central Research and Development Laboratories, 41-1 Yokomichi, Nagakute, Aichi 480-1192, Japan" /><meta name="citation_author" content="Tomatsu, Hajime" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Hanano, Shigeru" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Moriya, Kanami" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Hosouchi, Tsutomu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Sakurai, Nozomu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Suzuki, Hideyuki" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Shinmyo, Atsuhiko" /><meta name="citation_author_institution" content="Graduate School of Biological Science, Nara Institute of Science and Technology, 8916-5 Takayama, Ikoma, Nara 630-0192, Japan" /><meta name="citation_author" content="Shibata, Daisuke" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_title" content="Precise Sequential DNA Ligation on A Solid Substrate: Solid-Based Rapid Sequential Ligation of Multiple DNA Molecules" /><meta name="citation_firstpage" content="583" /><meta name="citation_lastpage" content="592" /><meta name="citation_doi" content="10.1093/dnares/dst032" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2013/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Construction of biologically functional bacterial plasmids in vitro; Proc. Natl Acad. Sci. USA;  citation_year=1973;  citation_volume=70;  citation_pages=3240-4; " /><meta name="citation_reference" content="citation_title=Analysis of gene expression in Arabidopsis thaliana by array hybridization with genomic DNA fragments aligned along chromosomal regions; Plant J.;  citation_year=2002;  citation_volume=30;  citation_pages=247-55; " /><meta name="citation_reference" content="citation_title=ATTED-II: a database of co-expressed genes and cis elements for identifying co-regulated gene groups in Arabidopsis; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=D863-9; " /><meta name="citation_reference" content="citation_title=Distinctive features of plant organs characterized by global analysis of gene expression in Arabidopsis; DNA Res.;  citation_year=2004;  citation_volume=11;  citation_pages=11-25; " /><meta name="citation_reference" content="citation_title=CoP: a database for characterizing co-expressed gene modules with biological information in plants; Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=1267-8; " /><meta name="citation_reference" content="citation_title=KaPPA-View4: a metabolic pathway database for representation and analysis of correlation networks of gene co-expression and metabolite co-accumulation and omics data; Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=D677-84; " /><meta name="citation_reference" content="citation_title=KaPPA-view: a web-based analysis tool for integration of transcript and metabolite data on plant metabolic pathway maps; Plant Physiol.;  citation_year=2005;  citation_volume=138;  citation_pages=1289-300; " /><meta name="citation_reference" content="citation_title=Evidence for network evolution in an Arabidopsis interactome map; citation_author=Arabidopsis Interactome Mapping Consortium.; Science;  citation_year=2011;  citation_volume=333;  citation_pages=601-7; " /><meta name="citation_reference" content="citation_title=The predicted Arabidopsis interactome resource and network topology-based systems biology analyses; Plant Cell;  citation_year=2011;  citation_volume=23;  citation_pages=911-22; " /><meta name="citation_reference" content="citation_title=Accurate multiplex gene synthesis from programmable DNA microchips; Nature;  citation_year=2004;  citation_volume=432;  citation_pages=1050-4; " /><meta name="citation_reference" content="citation_title=In-fusion assembly: seamless engineering of multidomain fusion proteins, modular vectors, and mutations; Biotechniques;  citation_year=2007;  citation_volume=43;  citation_pages=354-9; " /><meta name="citation_reference" content="citation_title=Complete chemical synthesis, assembly, and cloning of a Mycoplasma genitalium genome; Science;  citation_year=2008;  citation_volume=319;  citation_pages=1215-20; " /><meta name="citation_reference" content="citation_title=Enzymatic assembly of DNA molecules up to several hundred kilobases; Nat. Methods;  citation_year=2009;  citation_volume=6;  citation_pages=343-5; " /><meta name="citation_reference" content="citation_title=The biotin-streptavidin interaction can be reversibly broken using water at elevated temperatures; Electrophoresis;  citation_year=2005;  citation_volume=26;  citation_pages=501-10; " /><meta name="citation_reference" content="citation_title=Pathway engineering of Brassica napus seeds using multiple key enzyme genes involved in ketocarotenoid formation; J. Exp. Bot.;  citation_year=2009;  citation_volume=60;  citation_pages=1319-32; " /><meta name="citation_reference" content="citation_title=Many paths to many clones: a comparative look at high-throughput cloning methods; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=2020-8; " /><meta name="citation_reference" content="citation_title=A novel strategy for rapid construction of libraries of full-length antibodies highly expressed on mammalian cell surfaces; Acta Biochim. Biophys. Sin.;  citation_year=2010;  citation_volume=42;  citation_pages=575-84; " /><meta name="citation_reference" content="citation_title=FLASH assembly of TALENs for high-throughput genome editing; Nat. Biotechnol.;  citation_year=2012;  citation_volume=30;  citation_pages=460-5; " /><meta name="citation_reference" content="citation_title=Creation of a bacterial cell controlled by a chemically synthesized genome; Science;  citation_year=2010;  citation_volume=329;  citation_pages=52-6; " /><meta name="citation_reference" content="citation_title=Modular construction of plasmids through ligation-free assembly of vector components with oligonucleotide linkers; Biotechniques;  citation_year=2008;  citation_volume=44;  citation_pages=924-6; " /><meta name="citation_reference" content="citation_title=Expression and inheritance of multiple transgenes in rice plants; Nat. Biotechnol.;  citation_year=1998;  citation_volume=16;  citation_pages=1060-4; " /><meta name="citation_reference" content="citation_title=Functional annotation of a full-length Arabidopsis cDNA collection; Science;  citation_year=2002;  citation_volume=296;  citation_pages=141-5; " /><meta name="citation_reference" content="citation_title=RIKEN Arabidopsis full-length (RAFL) cDNA and its applications for expression profiling under abiotic stress conditions; J. Exp. Bot.;  citation_year=2004;  citation_volume=55;  citation_pages=213-23; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/6/583/1165149/dst032.pdf" /><meta name="description" content="Abstract.  Ligation, the joining of DNA fragments, is a fundamental procedure in molecular cloning and is indispensable to the production of genetically modifie" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/6/583/420514" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/6/583/420514" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/20/6/583/420514");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst032';
    </script>
<title>
	Precise Sequential DNA Ligation on A Solid Substrate: Solid-Based Rapid Sequential Ligation of Multiple DNA Molecules | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./420514" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BblGPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYWtpdGEsIEVpamkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUmVzZWFyY2ggQXNzb2NpYXRpb24gZm9yIEJpb3RlY2hub2xvZ3ksIE5pc2hpc2hpbmJhc2hpIFlhc3VkYSBVbmlvbiBCbGRnLCAyLTQtMiBOaXNoaS1zaGluYmFzaGksIE1pbmF0by1rdSwgVG9reW8gMTA1LTAwMDMsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEJpb2xvZ2ljYWwgU2NpZW5jZSwgTmFyYSBJbnN0aXR1dGUgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgODkxNi01IFRha2F5YW1hLCBJa29tYSwgTmFyYSA2MzAtMDE5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLb2hkYSwgS2F0c3Vub3JpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvdGVjaG5vbG9neSBMYWJvcmF0b3J5LCBUb3lvdGEgQ2VudHJhbCBSZXNlYXJjaCBhbmQgRGV2ZWxvcG1lbnQgTGFib3JhdG9yaWVzLCA0MS0xIFlva29taWNoaSwgTmFnYWt1dGUsIEFpY2hpIDQ4MC0xMTkyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRvbWF0c3UsIEhhamltZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIYW5hbm8sIFNoaWdlcnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTW9yaXlhLCBLYW5hbWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSG9zb3VjaGksIFRzdXRvbXUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2FrdXJhaSwgTm96b211IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlN1enVraSwgSGlkZXl1a2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hpbm15bywgQXRzdWhpa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgQmlvbG9naWNhbCBTY2llbmNlLCBOYXJhIEluc3RpdHV0ZSBvZiBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCA4OTE2LTUgVGFrYXlhbWEsIElrb21hLCBOYXJhIDYzMC0wMTkyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNoaWJhdGEsIERhaXN1a2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJQcmVjaXNlIFNlcXVlbnRpYWwgRE5BIExpZ2F0aW9uIG9uIEEgU29saWQgU3Vic3RyYXRlOiBTb2xpZC1CYXNlZCBSYXBpZCBTZXF1ZW50aWFsIExpZ2F0aW9uIG9mIE11bHRpcGxlIEROQSBNb2xlY3VsZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI1ODMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjU5MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdDAzMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEzLzEyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIGJpb2xvZ2ljYWxseSBmdW5jdGlvbmFsIGJhY3RlcmlhbCBwbGFzbWlkcyBpbiB2aXRybzsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk3MzsgIGNpdGF0aW9uX3ZvbHVtZT03MDsgIGNpdGF0aW9uX3BhZ2VzPTMyNDAtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgZ2VuZSBleHByZXNzaW9uIGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hIGJ5IGFycmF5IGh5YnJpZGl6YXRpb24gd2l0aCBnZW5vbWljIEROQSBmcmFnbWVudHMgYWxpZ25lZCBhbG9uZyBjaHJvbW9zb21hbCByZWdpb25zOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTI0Ny01NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QVRURUQtSUk6IGEgZGF0YWJhc2Ugb2YgY28tZXhwcmVzc2VkIGdlbmVzIGFuZCBjaXMgZWxlbWVudHMgZm9yIGlkZW50aWZ5aW5nIGNvLXJlZ3VsYXRlZCBnZW5lIGdyb3VwcyBpbiBBcmFiaWRvcHNpczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9RDg2My05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXN0aW5jdGl2ZSBmZWF0dXJlcyBvZiBwbGFudCBvcmdhbnMgY2hhcmFjdGVyaXplZCBieSBnbG9iYWwgYW5hbHlzaXMgb2YgZ2VuZSBleHByZXNzaW9uIGluIEFyYWJpZG9wc2lzOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTExLTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db1A6IGEgZGF0YWJhc2UgZm9yIGNoYXJhY3Rlcml6aW5nIGNvLWV4cHJlc3NlZCBnZW5lIG1vZHVsZXMgd2l0aCBiaW9sb2dpY2FsIGluZm9ybWF0aW9uIGluIHBsYW50czsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz0xMjY3LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUthUFBBLVZpZXc0OiBhIG1ldGFib2xpYyBwYXRod2F5IGRhdGFiYXNlIGZvciByZXByZXNlbnRhdGlvbiBhbmQgYW5hbHlzaXMgb2YgY29ycmVsYXRpb24gbmV0d29ya3Mgb2YgZ2VuZSBjby1leHByZXNzaW9uIGFuZCBtZXRhYm9saXRlIGNvLWFjY3VtdWxhdGlvbiBhbmQgb21pY3MgZGF0YTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9RDY3Ny04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9S2FQUEEtdmlldzogYSB3ZWItYmFzZWQgYW5hbHlzaXMgdG9vbCBmb3IgaW50ZWdyYXRpb24gb2YgdHJhbnNjcmlwdCBhbmQgbWV0YWJvbGl0ZSBkYXRhIG9uIHBsYW50IG1ldGFib2xpYyBwYXRod2F5IG1hcHM7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEzODsgIGNpdGF0aW9uX3BhZ2VzPTEyODktMzAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmlkZW5jZSBmb3IgbmV0d29yayBldm9sdXRpb24gaW4gYW4gQXJhYmlkb3BzaXMgaW50ZXJhY3RvbWUgbWFwOyBjaXRhdGlvbl9hdXRob3I9QXJhYmlkb3BzaXMgSW50ZXJhY3RvbWUgTWFwcGluZyBDb25zb3J0aXVtLjsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzM7ICBjaXRhdGlvbl9wYWdlcz02MDEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHByZWRpY3RlZCBBcmFiaWRvcHNpcyBpbnRlcmFjdG9tZSByZXNvdXJjZSBhbmQgbmV0d29yayB0b3BvbG9neS1iYXNlZCBzeXN0ZW1zIGJpb2xvZ3kgYW5hbHlzZXM7IFBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz05MTEtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFjY3VyYXRlIG11bHRpcGxleCBnZW5lIHN5bnRoZXNpcyBmcm9tIHByb2dyYW1tYWJsZSBETkEgbWljcm9jaGlwczsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTQzMjsgIGNpdGF0aW9uX3BhZ2VzPTEwNTAtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW4tZnVzaW9uIGFzc2VtYmx5OiBzZWFtbGVzcyBlbmdpbmVlcmluZyBvZiBtdWx0aWRvbWFpbiBmdXNpb24gcHJvdGVpbnMsIG1vZHVsYXIgdmVjdG9ycywgYW5kIG11dGF0aW9uczsgQmlvdGVjaG5pcXVlczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTM1NC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBjaGVtaWNhbCBzeW50aGVzaXMsIGFzc2VtYmx5LCBhbmQgY2xvbmluZyBvZiBhIE15Y29wbGFzbWEgZ2VuaXRhbGl1bSBnZW5vbWU7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzE5OyAgY2l0YXRpb25fcGFnZXM9MTIxNS0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW56eW1hdGljIGFzc2VtYmx5IG9mIEROQSBtb2xlY3VsZXMgdXAgdG8gc2V2ZXJhbCBodW5kcmVkIGtpbG9iYXNlczsgTmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0zNDMtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGJpb3Rpbi1zdHJlcHRhdmlkaW4gaW50ZXJhY3Rpb24gY2FuIGJlIHJldmVyc2libHkgYnJva2VuIHVzaW5nIHdhdGVyIGF0IGVsZXZhdGVkIHRlbXBlcmF0dXJlczsgRWxlY3Ryb3Bob3Jlc2lzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9NTAxLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QYXRod2F5IGVuZ2luZWVyaW5nIG9mIEJyYXNzaWNhIG5hcHVzIHNlZWRzIHVzaW5nIG11bHRpcGxlIGtleSBlbnp5bWUgZ2VuZXMgaW52b2x2ZWQgaW4ga2V0b2Nhcm90ZW5vaWQgZm9ybWF0aW9uOyBKLiBFeHAuIEJvdC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NjA7ICBjaXRhdGlvbl9wYWdlcz0xMzE5LTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYW55IHBhdGhzIHRvIG1hbnkgY2xvbmVzOiBhIGNvbXBhcmF0aXZlIGxvb2sgYXQgaGlnaC10aHJvdWdocHV0IGNsb25pbmcgbWV0aG9kczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0yMDIwLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbm92ZWwgc3RyYXRlZ3kgZm9yIHJhcGlkIGNvbnN0cnVjdGlvbiBvZiBsaWJyYXJpZXMgb2YgZnVsbC1sZW5ndGggYW50aWJvZGllcyBoaWdobHkgZXhwcmVzc2VkIG9uIG1hbW1hbGlhbiBjZWxsIHN1cmZhY2VzOyBBY3RhIEJpb2NoaW0uIEJpb3BoeXMuIFNpbi47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz01NzUtODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZMQVNIIGFzc2VtYmx5IG9mIFRBTEVOcyBmb3IgaGlnaC10aHJvdWdocHV0IGdlbm9tZSBlZGl0aW5nOyBOYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9NDYwLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNyZWF0aW9uIG9mIGEgYmFjdGVyaWFsIGNlbGwgY29udHJvbGxlZCBieSBhIGNoZW1pY2FsbHkgc3ludGhlc2l6ZWQgZ2Vub21lOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTMyOTsgIGNpdGF0aW9uX3BhZ2VzPTUyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vZHVsYXIgY29uc3RydWN0aW9uIG9mIHBsYXNtaWRzIHRocm91Z2ggbGlnYXRpb24tZnJlZSBhc3NlbWJseSBvZiB2ZWN0b3IgY29tcG9uZW50cyB3aXRoIG9saWdvbnVjbGVvdGlkZSBsaW5rZXJzOyBCaW90ZWNobmlxdWVzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9OTI0LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cHJlc3Npb24gYW5kIGluaGVyaXRhbmNlIG9mIG11bHRpcGxlIHRyYW5zZ2VuZXMgaW4gcmljZSBwbGFudHM7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xMDYwLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bmN0aW9uYWwgYW5ub3RhdGlvbiBvZiBhIGZ1bGwtbGVuZ3RoIEFyYWJpZG9wc2lzIGNETkEgY29sbGVjdGlvbjsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOTY7ICBjaXRhdGlvbl9wYWdlcz0xNDEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UklLRU4gQXJhYmlkb3BzaXMgZnVsbC1sZW5ndGggKFJBRkwpIGNETkEgYW5kIGl0cyBhcHBsaWNhdGlvbnMgZm9yIGV4cHJlc3Npb24gcHJvZmlsaW5nIHVuZGVyIGFiaW90aWMgc3RyZXNzIGNvbmRpdGlvbnM7IEouIEV4cC4gQm90LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01NTsgIGNpdGF0aW9uX3BhZ2VzPTIxMy0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIwLzYvNTgzLzExNjUxNDkvZHN0MDMyLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBMaWdhdGlvbiwgdGhlIGpvaW5pbmcgb2YgRE5BIGZyYWdtZW50cywgaXMgYSBmdW5kYW1lbnRhbCBwcm9jZWR1cmUgaW4gbW9sZWN1bGFyIGNsb25pbmcgYW5kIGlzIGluZGlzcGVuc2FibGUgdG8gdGhlIHByb2R1Y3Rpb24gb2YgZ2VuZXRpY2FsbHkgbW9kaWZpZSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDIwNTE0JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZNDU0CgwumWog8LLsgagwr3MmdCB" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f6%2f583%2f420514" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f6%2f583%2f420514" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=16367" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/6/1/m_cover.gif?Expires=2147483647&amp;Signature=VPhijSxXdAPLpw4YhUouOC1tx3X8x9GMJFVJIau9z4hN5cdWK-jPPF1wEurkS1gh6aQR9TQYLHOUG1kzMtF4HcnoT9E-wJJg-rleKg0FU2PY4tefYkghKgj~tiS7jfzXHN~ky5UBYwxoXbHB1rhYpbzfxlAWCYEab07wd~Go~sCK2Z9WKtR5wEAnnMAXr1I8E45jivf904J4XT3ytlfCnSQKIVeT43OyeE29BJRkSUT0SEXeMt6-ucAdFaiaEOPvq~y1xtuGK9d7YWhoevE9wdrzD7fhQ5jWGawbllCtUUH0ojTdHlsKW2bYFxcwuER72KVIwWBylO57igmYb52w8A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/6">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6229383">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6229383">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6229387">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6229387">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6229401">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6229401">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6229427">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6229427">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6229441">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6229441">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6229449">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6229449">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6229451">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6229451">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6229453">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6229453">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6229455">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6229455">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/20/6/567/419919">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/20/6/593/420691">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Precise Sequential DNA Ligation on A Solid Substrate: Solid-Based Rapid Sequential Ligation of Multiple DNA Molecules
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eiji Takita</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Research Association for Biotechnology</div>, <div class="addr-line">Nishishinbashi Yasuda Union Bldg, 2-4-2 Nishi-shinbashi, Minato-ku, Tokyo 105-0003</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Graduate School of Biological Science</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5 Takayama, Ikoma, Nara 630-0192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eiji+Takita">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takita E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takita E&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Eiji Takita</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Katsunori Kohda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Biotechnology Laboratory</div>, <div class="insititution">Toyota Central Research and Development Laboratories</div>, <div class="addr-line">41-1 Yokomichi, Nagakute, Aichi 480-1192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Katsunori+Kohda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kohda K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kohda K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Katsunori Kohda</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hajime Tomatsu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hajime+Tomatsu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tomatsu H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tomatsu H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hajime Tomatsu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shigeru Hanano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shigeru+Hanano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hanano S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hanano S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shigeru Hanano</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kanami Moriya</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kanami+Moriya">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Moriya K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Moriya K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kanami Moriya</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tsutomu Hosouchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tsutomu+Hosouchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hosouchi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hosouchi T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tsutomu Hosouchi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nozomu Sakurai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nozomu+Sakurai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sakurai N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sakurai N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Nozomu Sakurai</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideyuki Suzuki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideyuki+Suzuki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suzuki H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suzuki H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hideyuki Suzuki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Atsuhiko Shinmyo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Graduate School of Biological Science</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5 Takayama, Ikoma, Nara 630-0192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Atsuhiko+Shinmyo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shinmyo A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shinmyo A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Atsuhiko Shinmyo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Daisuke Shibata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 0438-52-3947. Fax. +81 0438-52-3948. E-mail: <a href="mailto:shibata@kazusa.or.jp" target="_blank">shibata@kazusa.or.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Daisuke+Shibata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shibata D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shibata D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Daisuke Shibata</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="label -label">‡</div><div class="footnote-content">Present address: Advanced Technology Research Laboratories, Idemitsu Kosan Co., Ltd., 1280 Kami-izumi, Sodegaura, Chiba 299-0293, Japan.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN3"><div class="fn"><div class="label -label">§</div><div class="footnote-content">Present address: Graduate School of Pharmaceutical Sciences, Chiba University, Chuo-ku, Chiba, 260-8675, Japan.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 6, 1 December 2013, Pages 583–592, <a href='https://doi.org/10.1093/dnares/dst032'>https://doi.org/10.1093/dnares/dst032</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">29 July 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">11 February 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">18 June 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="420514" href="/dnaresearch/article-pdf/20/6/583/1165149/dst032.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Eiji Takita, Katsunori Kohda, Hajime Tomatsu, Shigeru Hanano, Kanami Moriya, Tsutomu Hosouchi, Nozomu Sakurai, Hideyuki Suzuki, Atsuhiko Shinmyo, Daisuke Shibata;  Precise Sequential DNA Ligation on A Solid Substrate: Solid-Based Rapid Sequential Ligation of Multiple DNA Molecules, <em>DNA Research</em>, Volume 20, Issue 6, 1 December 2013, Pages 583–592, <a href='https://doi.org/10.1093/dnares/dst032'>https://doi.org/10.1093/dnares/dst032</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/420514?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/420514?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/420514?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/420514?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/420514?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/420514?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/420514?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Precise%20Sequential%20DNA%20Ligation%20on%20A%20Solid%20Substrate%3A%20Solid-Based%20Rapid%20Sequential%20Ligation%20of%20Multiple%20DNA%20Molecules&amp;publicationDate=2013-07-29&amp;volumeNum=20&amp;issueNum=6&amp;author=Takita%2C%20Eiji%3B%20Kohda%2C%20Katsunori&amp;startPage=583&amp;endPage=592&amp;contentId=10.1093%2Fdnares%2Fdst032&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f6%2f583%2f420514" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6229383 id="6229383" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Ligation, the joining of DNA fragments, is a fundamental procedure in molecular cloning and is indispensable to the production of genetically modified organisms that can be used for basic research, the applied biosciences, or both. Given that many genes cooperate in various pathways, incorporating multiple gene cassettes in tandem in a transgenic DNA construct for the purpose of genetic modification is often necessary when generating organisms that produce multiple foreign gene products. Here, we describe a novel method, designated PRESSO (precise sequential DNA ligation on a solid substrate), for the tandem ligation of multiple DNA fragments. We amplified donor DNA fragments with non-palindromic ends, and ligated the fragment to acceptor DNA fragments on solid beads. After the final donor DNA fragments, which included vector sequences, were joined to the construct that contained the array of fragments, the ligation product (the construct) was thereby released from the beads via digestion with a rare-cut meganuclease; the freed linear construct was circularized via an intra-molecular ligation. PRESSO allowed us to rapidly and efficiently join multiple genes in an optimized order and orientation. This method can overcome many technical challenges in functional genomics during the post-sequencing generation.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;DNA ligation&quot;">DNA ligation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="cloning">cloning</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;multiple DNA fragment assembly&quot;">multiple DNA fragment assembly</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;functional genomics&quot;">functional genomics</a></div>                            <h2 scrollto-destination=6229387 id="6229387" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Since the first description of recombinant DNA by Cohen <em>et al.</em> in 1973,<sup><a reveal-id="DST032C1" data-open="DST032C1" class="link link-ref link-reveal xref-bibr">1</a></sup> scientists have dreamt of introducing novel characteristics into organisms for academic or industrial purposes, characteristics such as the production of desirable crops or consumable materials. Joining two or more DNA fragments via a ligase-catalyzed reaction (a ligation) is a fundamental technique in recombinant DNA technology and genetic engineering. The ligation of different DNA fragments can create new functional units. Many organisms can be genetically modified via recombinant DNA technology.</p><p>Recent advances in functional genomics have revealed that gene clusters are involved in certain biological pathways or metabolic processes.<sup><a reveal-id="DST032C2  DST032C9" data-open="DST032C2  DST032C9" class="link link-ref link-reveal xref-bibr">2–9</a></sup> For instance, a defined series of enzymatic reactions are often required in individual metabolic pathways. Therefore, introduction of a heterologous metabolic pathway into a host species will often required that multiple genes are expressed in the host organisms. However, despite many advances in molecular biology and genetic engineering technologies, a rapid, simple, and efficient method for joining a series of many gene cassettes is lacking.</p><p>Stepwise cloning of restriction fragments is a commonly used method for connecting multiple DNA fragments. However, this method is time-consuming; moreover, it is often difficult to devise a cloning strategy because the number of unique restriction sites is often too limited. To address these difficulties and to construct long DNA fragments, several methods have been developed; MultiSite Gateway cloning, polymerase assembly multiplexing (PAM) method, In-Fusion system, and chemical syntheses of genomes are some of these methods. The MultiSite Gateway cloning is based on DNA recombination; a maximum of four DNA fragments can be assembled into one construct simultaneously, because the number of unique <em>att</em>B recombination site is limited. However, when more than five DNA fragments need to be joined, the Gateway methods also require many sequential reactions. The PAM method requires three steps for the construction of multi-cassette transgenes: polymerase cycling assemble of multiple oligonucleotides, overlap extension and annealing, and repair and ligation.<sup><a reveal-id="DST032C10" data-open="DST032C10" class="link link-ref link-reveal xref-bibr">10</a></sup> The In-Fusion system is similar in principle to PAM; DNA fragments are amplified via PCR with oligonucleotide primers that carry extensions of &gt;15 bp, and the In-Fusion exonuclease is used to chew from 3′ to 5′ at the fragment ends to produce single-stranded ends; the ends are annealed, and the DNA fragments are repaired and ligated in <em>Escherichia coli</em>.<sup><a reveal-id="DST032C11" data-open="DST032C11" class="link link-ref link-reveal xref-bibr">11</a></sup> These methods for connecting multiple fragments proceed in a single step; therefore, the ligation reactions are faster than the stepwise cloning. But, these methods are often problematic when constructs with many gene cassettes are being built. It is difficult to ensure that the desired gene order and orientation and the correct nucleotide sequence are produced with any of these methods. Chemical syntheses of genome can be used to produce very large DNA constructs;<sup><a reveal-id="DST032C12" data-open="DST032C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DST032C13" data-open="DST032C13" class="link link-ref link-reveal xref-bibr">13</a></sup> however, this method often produces high error rates, and it is very expensive. Thus, the joining of multiple DNA fragments in a desired order is still labour intensive and time consuming.</p><p>Here, we describe an alternative method for the sequential ligation of multiple DNA fragments, the precise sequential DNA ligation on a solid substrate (PRESSO) (Fig. <a reveal-id="DST032F1" data-open="DST032F1" class="link link-reveal link-table xref-fig">1</a>). This method is cost-effective and produces DNA constructs more accurately than do previously described methods. </p>                        <a id="6229396" scrollto-destination="6229396"></a>
<div id="DST032F1" data-id="DST032F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst032/2/m_dst03201.png?Expires=2147483647&Signature=3BLBftK9813yvusEZJTg0dZcgnkVYpj5iZ9wxiBUgzlG-g7h2KDEMqdw5-SyznJ7Y806uzCFgJA4JSp1Sm6zRSiddsEyC2C-fdXfZS33gXqLkgZOHmu0wnWs~luWBBhJ~pvCQdOvBFbPQ-862pJZ5idbfh4AS2b-flRRb4mbyELnEe~u2eMC1ZFAyJLNkog8v3mktLTIedUntJQ04GLlhpq3m~WhY69UhEhlN4ZaMe7xhy69Q~l6ShVI4NItwOpMLfnWRwHJFAgmHdUn2jqnNFXSAjDcmWF8KLJdh7nf5io3Q~zUsLUMlnZMnvripXgbbkf36hABQ97AlHnAeKbvtA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of the PRESSO method. The 5′-biotin-labelled DNA molecule with recognition sequences for a meganuclease, e.g. I-SceI, and with a non-palindromic restriction site, e.g. an SfiI site, is used as the initial acceptor molecule for a series of sequential ligations. The donor DNA molecules for the subsequent ligations are prepared via PCR; the 5′-primers are biotinylated, the 3′-primers are unmodified for the initial acceptor and biotinylated for the donor DNAs, and all primers except for the 5′-primer of the initial acceptor contain an SfiI site. The amplified donor DNA fragments are digested with SfiI and purified. For convenience, ∼200 bp DNA fragments are used as a spacer DNA (linker) in this study. The initial linker DNA molecules are attached to streptavidin-coated beads via the biotin label. The 3′-end of the initial acceptor DNA (the linker) is designed to be complementary with the 5′-end of the initial donor molecule, which has non-palindromic sites in 5′- and 3′-ends. The 3′-end of the first donor molecule is designed to be complimentary to the 5′-end of next donor DNA. Each ligation step is performed individually; a donor DNA is ligated to the growing end of a construct that is attached to a solid bead via the opposite biotin-labelled end; the beads are then washed and collected. In subsequent sequential reactions, the 5′-end of the next donor DNA molecule is ligated to the 3′-end of bead-bound DNA. In this manner, each donor DNA is connected to an acceptor DNA molecule that is attached to a solid bead. The final donor DNA molecules contain a vector sequence and a meganuclease, I-SceI, recognition site. Therefore, the final products on the beads include an I-SceI site on each end. The ligated DNA fragments are released from the bead via I-SceI digestion; to construct a transformable plasmid, the newly freed linear fragments are circularized via an intra-molecular ligation. The circular plasmids are then used to transform E. coli; replicated plasmids can then be recovered from transformed cells." path-from-xml="dst03201.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6229396/dst03201.tiff" path-from-xml="dst03201.tiff" target="_blank">View large</a><a section="6229396" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst032/2/dst03201.png?Expires=2147483647&Signature=KSBKSdgmB8OtsAeHGHXSRq1LxDO6L4PMs6VlgzrxWrTnkmv4cNOF9nnbaokPkw1YrwgMxnokyj49gzAJ1E5yTFeCBIS~LIhm9C7aRcF9aP6ke-xVtQYdTT382NROQ2iAgWQdykL2cN74KV4X1L-R5cutXJMaipGDnbst5pH99~7K~PUcj1uB-qza1WWxZUWgjXEl-fIrNU7C~flYFUSc20RxAP9lGN331ohmvBar6s18NsXFeD36r3t3G7YL8O2m7YJTF0J1ko1Hx~5UEcGuLHiIyTyGPnGHbYdwHytW-qV5YPVnHEliZKyMJaONfHmmA3GNLUW7zyRru2VHbAWqtQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6229396&ar=420514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03201.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of the PRESSO method. The 5′-biotin-labelled DNA molecule with recognition sequences for a meganuclease, e.g. I-<em>Sce</em>I, and with a non-palindromic restriction site, e.g. an <em>Sfi</em>I site, is used as the initial acceptor molecule for a series of sequential ligations. The donor DNA molecules for the subsequent ligations are prepared via PCR; the 5′-primers are biotinylated, the 3′-primers are unmodified for the initial acceptor and biotinylated for the donor DNAs, and all primers except for the 5′-primer of the initial acceptor contain an <em>Sfi</em>I site. The amplified donor DNA fragments are digested with <em>Sfi</em>I and purified. For convenience, ∼200 bp DNA fragments are used as a spacer DNA (linker) in this study. The initial linker DNA molecules are attached to streptavidin-coated beads via the biotin label. The 3′-end of the initial acceptor DNA (the linker) is designed to be complementary with the 5′-end of the initial donor molecule, which has non-palindromic sites in 5′- and 3′-ends. The 3′-end of the first donor molecule is designed to be complimentary to the 5′-end of next donor DNA. Each ligation step is performed individually; a donor DNA is ligated to the growing end of a construct that is attached to a solid bead via the opposite biotin-labelled end; the beads are then washed and collected. In subsequent sequential reactions, the 5′-end of the next donor DNA molecule is ligated to the 3′-end of bead-bound DNA. In this manner, each donor DNA is connected to an acceptor DNA molecule that is attached to a solid bead. The final donor DNA molecules contain a vector sequence and a meganuclease, I-<em>Sce</em>I, recognition site. Therefore, the final products on the beads include an I-<em>Sce</em>I site on each end. The ligated DNA fragments are released from the bead via I-<em>Sce</em>I digestion; to construct a transformable plasmid, the newly freed linear fragments are circularized via an intra-molecular ligation. The circular plasmids are then used to transform <em>E. coli</em>; replicated plasmids can then be recovered from transformed cells.</p></div></div><div content-id="DST032F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst032/2/m_dst03201.png?Expires=2147483647&Signature=3BLBftK9813yvusEZJTg0dZcgnkVYpj5iZ9wxiBUgzlG-g7h2KDEMqdw5-SyznJ7Y806uzCFgJA4JSp1Sm6zRSiddsEyC2C-fdXfZS33gXqLkgZOHmu0wnWs~luWBBhJ~pvCQdOvBFbPQ-862pJZ5idbfh4AS2b-flRRb4mbyELnEe~u2eMC1ZFAyJLNkog8v3mktLTIedUntJQ04GLlhpq3m~WhY69UhEhlN4ZaMe7xhy69Q~l6ShVI4NItwOpMLfnWRwHJFAgmHdUn2jqnNFXSAjDcmWF8KLJdh7nf5io3Q~zUsLUMlnZMnvripXgbbkf36hABQ97AlHnAeKbvtA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of the PRESSO method. The 5′-biotin-labelled DNA molecule with recognition sequences for a meganuclease, e.g. I-SceI, and with a non-palindromic restriction site, e.g. an SfiI site, is used as the initial acceptor molecule for a series of sequential ligations. The donor DNA molecules for the subsequent ligations are prepared via PCR; the 5′-primers are biotinylated, the 3′-primers are unmodified for the initial acceptor and biotinylated for the donor DNAs, and all primers except for the 5′-primer of the initial acceptor contain an SfiI site. The amplified donor DNA fragments are digested with SfiI and purified. For convenience, ∼200 bp DNA fragments are used as a spacer DNA (linker) in this study. The initial linker DNA molecules are attached to streptavidin-coated beads via the biotin label. The 3′-end of the initial acceptor DNA (the linker) is designed to be complementary with the 5′-end of the initial donor molecule, which has non-palindromic sites in 5′- and 3′-ends. The 3′-end of the first donor molecule is designed to be complimentary to the 5′-end of next donor DNA. Each ligation step is performed individually; a donor DNA is ligated to the growing end of a construct that is attached to a solid bead via the opposite biotin-labelled end; the beads are then washed and collected. In subsequent sequential reactions, the 5′-end of the next donor DNA molecule is ligated to the 3′-end of bead-bound DNA. In this manner, each donor DNA is connected to an acceptor DNA molecule that is attached to a solid bead. The final donor DNA molecules contain a vector sequence and a meganuclease, I-SceI, recognition site. Therefore, the final products on the beads include an I-SceI site on each end. The ligated DNA fragments are released from the bead via I-SceI digestion; to construct a transformable plasmid, the newly freed linear fragments are circularized via an intra-molecular ligation. The circular plasmids are then used to transform E. coli; replicated plasmids can then be recovered from transformed cells." path-from-xml="dst03201.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6229396/dst03201.tiff" path-from-xml="dst03201.tiff" target="_blank">View large</a><a section="6229396" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst032/2/dst03201.png?Expires=2147483647&Signature=KSBKSdgmB8OtsAeHGHXSRq1LxDO6L4PMs6VlgzrxWrTnkmv4cNOF9nnbaokPkw1YrwgMxnokyj49gzAJ1E5yTFeCBIS~LIhm9C7aRcF9aP6ke-xVtQYdTT382NROQ2iAgWQdykL2cN74KV4X1L-R5cutXJMaipGDnbst5pH99~7K~PUcj1uB-qza1WWxZUWgjXEl-fIrNU7C~flYFUSc20RxAP9lGN331ohmvBar6s18NsXFeD36r3t3G7YL8O2m7YJTF0J1ko1Hx~5UEcGuLHiIyTyGPnGHbYdwHytW-qV5YPVnHEliZKyMJaONfHmmA3GNLUW7zyRru2VHbAWqtQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6229396&ar=420514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03201.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of the PRESSO method. The 5′-biotin-labelled DNA molecule with recognition sequences for a meganuclease, e.g. I-<em>Sce</em>I, and with a non-palindromic restriction site, e.g. an <em>Sfi</em>I site, is used as the initial acceptor molecule for a series of sequential ligations. The donor DNA molecules for the subsequent ligations are prepared via PCR; the 5′-primers are biotinylated, the 3′-primers are unmodified for the initial acceptor and biotinylated for the donor DNAs, and all primers except for the 5′-primer of the initial acceptor contain an <em>Sfi</em>I site. The amplified donor DNA fragments are digested with <em>Sfi</em>I and purified. For convenience, ∼200 bp DNA fragments are used as a spacer DNA (linker) in this study. The initial linker DNA molecules are attached to streptavidin-coated beads via the biotin label. The 3′-end of the initial acceptor DNA (the linker) is designed to be complementary with the 5′-end of the initial donor molecule, which has non-palindromic sites in 5′- and 3′-ends. The 3′-end of the first donor molecule is designed to be complimentary to the 5′-end of next donor DNA. Each ligation step is performed individually; a donor DNA is ligated to the growing end of a construct that is attached to a solid bead via the opposite biotin-labelled end; the beads are then washed and collected. In subsequent sequential reactions, the 5′-end of the next donor DNA molecule is ligated to the 3′-end of bead-bound DNA. In this manner, each donor DNA is connected to an acceptor DNA molecule that is attached to a solid bead. The final donor DNA molecules contain a vector sequence and a meganuclease, I-<em>Sce</em>I, recognition site. Therefore, the final products on the beads include an I-<em>Sce</em>I site on each end. The ligated DNA fragments are released from the bead via I-<em>Sce</em>I digestion; to construct a transformable plasmid, the newly freed linear fragments are circularized via an intra-molecular ligation. The circular plasmids are then used to transform <em>E. coli</em>; replicated plasmids can then be recovered from transformed cells.</p></div></div>                            <h2 scrollto-destination=6229401 id="6229401" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6229403 id="6229403" class="section-title" data-legacy-id=s2a>2.1. Materials</h3>
<p>The name, sequence, and restriction site information for each primer used in this study are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. In this study, a unique restriction enzyme site was added onto many of the primers, so that restriction digests could be used to confirm the final product (Fig. <a reveal-id="DST032F3" data-open="DST032F3" class="link link-reveal link-table xref-fig">3</a>). All restriction enzymes (including <em>Sfi</em>I, <em>Bst</em>XI, and I-<em>Sce</em>I), KOD—Plus—Ver. 2 DNA polymerase, which exhibits high PCR fidelity, and <em>E. coli</em> DH5-α competent cells were purchased from TOYOBO (Osaka, Japan). We used Dynabeads® M-280 Streptavidin-linked beads (Invitrogen, Carlsbad, California, USA) and DNA ligation kit &lt;Mighty mix&gt; (TAKARA BIO, Otsu, Japan) as the solid phase for the sequential ligation reactions. In our preliminary experiments, we found that the efficiency of transformation of <em>E. coli</em> with the resultant plasmid carrying sequentially ligated DNA fragments varied significantly among reaction series that were examined by using some ligation mixture or ligation kits. The DNA ligation kit &lt;Mighty mix&gt; (TAKARA BIO) exhibited high efficiency enough to connect multiple DNA fragments using the PRESSO protocol. Usually 10–100 of <em>E. coli</em> colonies were obtained when we transformed highly efficient competent <em>E. coli</em> cells with the plasmid carrying a 20-kb DNA insert connected with 10 DNA fragments of 2 kb (Fig. <a reveal-id="DST032F4" data-open="DST032F4" class="link link-reveal link-table xref-fig">4</a>). We recommend users to use the kit or to find a suitable one from available kits.</p>                            <h3 scrollto-destination=6229408 id="6229408" class="section-title" data-legacy-id=s2b>2.2. Synthesis of a spacer DNA</h3>
<p>A biotin-conjugated DNA molecule with an I-<em>Sce</em>I recognition site was synthesized as the spacer molecule that was attached to the streptavidin-linked beads. First, two complementary oligonucleotides, I-<em>Sce</em>I-F and I-<em>Sce</em>I-R, which each contained a I-<em>Sce</em>I recognition sequence, were annealed to one another to generate a double-stranded DNA molecule; this molecule was then inserted between the <em>Eco</em>RI and <em>Pst</em>I sites of the multi-cloning region of pUC19. This plasmid, pUC19-I-<em>Sce</em>I, and two biotin-labelled primers, PUC-N-u (5′) and PUC-CS (3′), were to PCR amplify a biotin-labelled DNA fragment (242 bp). The amplified products were purified with the QIAquick PCR purification Kit (QIAGEN, Germantown, MD, USA). The purified samples were dissolved in 100 μl of digestion buffer, and digested at 50°C with 50 U of <em>Sfi</em>I overnight. The digestion products were purified with the QIAquick PCR Purification Kit (QIAGEN) and precipitated with ethanol.</p>                            <h3 scrollto-destination=6229412 id="6229412" class="section-title" data-legacy-id=s2c>2.3. Preparation of the initial donor DNA–streptavidin beads</h3>
<p>Purified biotin-labelled linker DNA molecules (2 μg) were mixed with 30 μl of Dynabeads M-280 (Invitrogen) in binding solution [5 mM Tris–HCl (pH 7.5), 0.5 mM ethylenediaminetetraacetic acid (EDTA), and 1 M NaCl] for 2 h at room temperature; the beads were washed with the washing buffer [5 mM Tris–HCl (pH 7.5), 0.5 mM EDTA, and 1 M NaCl] according to the manufacture's protocol.</p>                            <h3 scrollto-destination=6229414 id="6229414" class="section-title" data-legacy-id=s2d>2.4. DNA fragments for the sequential ligation</h3>
<p>To demonstrate the utility of PRESSO, three sets of 10 DNA fragments were PCR amplified from <em>Arabidopsis thaliana</em> genomic clones (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>) with biotin-labelled oligonucleotide primers that each contained an <em>Sfi</em>I restriction enzyme (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The reactions were carried out for 30 cycles of 94°C for 30 s, 50°C for 30 s, and 72°C for 1 min. The amplification products were purified with the Wizard® SV Gel and PCR Clean-UP System (Promega, Madison, Wisconsin, USA), digested with <em>Sfi</em>I as described above, and then applied to streptavidin-linked magnetic beads to remove any undigested products (Fig. <a reveal-id="DST032F2" data-open="DST032F2" class="link link-reveal link-table xref-fig">2</a>). <em>Sfi</em>I-digested DNA (2 μg in 300 μl) was mixed with streptavidin-linked beads (in 300 μl) for 1 h with gentle stirring. As undigested DNA fragments bound to the beads, the supernatant fractions were recovered to collect the fully digested products, which were then used for the sequential ligation reactions (see below). </p>                        <a id="6229416" scrollto-destination="6229416"></a>
<div id="DST032F2" data-id="DST032F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst032/2/m_dst03202.png?Expires=2147483647&Signature=4rixRf3tXfKv4q3Km8xaUSJ9tuEfVArOUIExWBuyP-DgXl56e18zEw-UZvpDmz7ZHWRTFUP6yPchfdvEmVLjGGYMg4w72843Djz9i0CBbneLaHOomrtL5PGNJQicr4tClEYfU7Oz~2rLBIIbv-3-vyamzO1uj-ug5irJrLx6rnIoY~BXr~Ivpnl14Yy3iXnjAvRtAmsT98J2soaZw0GCNdjsAdZ96CAVrs8bJLPfDYmFaS25E9QlvQ0Q23s4-sqw0aCGaALIq23JO85GHeonqEJEN~f6DB9qqYAaUciLxKUx4Uw~60AKlkP3mU162LKB9mDKHGJ2C3d-zYoYFmYDpA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Purification of donor DNAs that were amplified with biotinylated primers. We purify donor DNA molecules to remove partially digested molecules. This step is often important to perform successful iterative ligations of multiple DNA fragments. Each donor molecule is amplified with biotinylated primers via PCR. The amplified fragments are digested with SfiI. When the digested fragments are mixed with streptavidin beads, partially digested products should interact with the beads, but the fully digested molecules should not. The supernatants are collected, because they contain the fully digested products with the appropriate overhang sequences; these products are then used for the sequential ligations." path-from-xml="dst03202.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6229416/dst03202.tiff" path-from-xml="dst03202.tiff" target="_blank">View large</a><a section="6229416" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst032/2/dst03202.png?Expires=2147483647&Signature=L0Oh~7pe9mQa9IVY77UGGANwP6pd95BazCw0MjuQ0sjDsuysbPdu3M8LIaTmDUqbKmoIwegBrgnzHgaUNBuXIE2~xHx6TKmgxw3pP~9nqC8i2rwjYh9QBn71YK4Qrx3nEy93NYylBqtlrtGEJWOU1xyDAqH8AnLmIVVm~ve9Cj4RBFmbue9DfBTObmYGUcSNNABjWYVs1aykcCGc~jULbQa14HSoovM~qJljEpI-so3QnF1SerWea5hSehUI10GQRKkrt3I7iPl6V9JMN6Ovbn0bqq576qoeO-bJqCMCdNGVOutFrQ7DFDWVvFBMLZA0sNtt1C0xUzbmS4DYh17vsA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6229416&ar=420514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03202.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Purification of donor DNAs that were amplified with biotinylated primers. We purify donor DNA molecules to remove partially digested molecules. This step is often important to perform successful iterative ligations of multiple DNA fragments. Each donor molecule is amplified with biotinylated primers via PCR. The amplified fragments are digested with <em>Sfi</em>I. When the digested fragments are mixed with streptavidin beads, partially digested products should interact with the beads, but the fully digested molecules should not. The supernatants are collected, because they contain the fully digested products with the appropriate overhang sequences; these products are then used for the sequential ligations.</p></div></div><div content-id="DST032F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst032/2/m_dst03202.png?Expires=2147483647&Signature=4rixRf3tXfKv4q3Km8xaUSJ9tuEfVArOUIExWBuyP-DgXl56e18zEw-UZvpDmz7ZHWRTFUP6yPchfdvEmVLjGGYMg4w72843Djz9i0CBbneLaHOomrtL5PGNJQicr4tClEYfU7Oz~2rLBIIbv-3-vyamzO1uj-ug5irJrLx6rnIoY~BXr~Ivpnl14Yy3iXnjAvRtAmsT98J2soaZw0GCNdjsAdZ96CAVrs8bJLPfDYmFaS25E9QlvQ0Q23s4-sqw0aCGaALIq23JO85GHeonqEJEN~f6DB9qqYAaUciLxKUx4Uw~60AKlkP3mU162LKB9mDKHGJ2C3d-zYoYFmYDpA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Purification of donor DNAs that were amplified with biotinylated primers. We purify donor DNA molecules to remove partially digested molecules. This step is often important to perform successful iterative ligations of multiple DNA fragments. Each donor molecule is amplified with biotinylated primers via PCR. The amplified fragments are digested with SfiI. When the digested fragments are mixed with streptavidin beads, partially digested products should interact with the beads, but the fully digested molecules should not. The supernatants are collected, because they contain the fully digested products with the appropriate overhang sequences; these products are then used for the sequential ligations." path-from-xml="dst03202.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6229416/dst03202.tiff" path-from-xml="dst03202.tiff" target="_blank">View large</a><a section="6229416" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst032/2/dst03202.png?Expires=2147483647&Signature=L0Oh~7pe9mQa9IVY77UGGANwP6pd95BazCw0MjuQ0sjDsuysbPdu3M8LIaTmDUqbKmoIwegBrgnzHgaUNBuXIE2~xHx6TKmgxw3pP~9nqC8i2rwjYh9QBn71YK4Qrx3nEy93NYylBqtlrtGEJWOU1xyDAqH8AnLmIVVm~ve9Cj4RBFmbue9DfBTObmYGUcSNNABjWYVs1aykcCGc~jULbQa14HSoovM~qJljEpI-so3QnF1SerWea5hSehUI10GQRKkrt3I7iPl6V9JMN6Ovbn0bqq576qoeO-bJqCMCdNGVOutFrQ7DFDWVvFBMLZA0sNtt1C0xUzbmS4DYh17vsA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6229416&ar=420514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03202.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Purification of donor DNAs that were amplified with biotinylated primers. We purify donor DNA molecules to remove partially digested molecules. This step is often important to perform successful iterative ligations of multiple DNA fragments. Each donor molecule is amplified with biotinylated primers via PCR. The amplified fragments are digested with <em>Sfi</em>I. When the digested fragments are mixed with streptavidin beads, partially digested products should interact with the beads, but the fully digested molecules should not. The supernatants are collected, because they contain the fully digested products with the appropriate overhang sequences; these products are then used for the sequential ligations.</p></div></div>                            <h3 scrollto-destination=6229417 id="6229417" class="section-title" data-legacy-id=s2e>2.5. Sequential ligation of DNA fragments</h3>
<p>We dissolved 600 ng of the first donor DNA molecule in 30 μl of the ligation buffer from the DNA ligation kit (TAKARA BIO); this solution was then mixed with 30 μl of the mixture that contained streptavidin beads coated with biotin-labelled digested spacer DNA from pUC19-I-<em>Sce</em>I; 30 μl of ligase solution from the kit was then added. The mixture was incubated at room temperature for 1 h with gentle agitation on a rotator. The beads were then washed twice with Tris–EDTA (TE) buffer (200 μl for each wash). The beads were then sequentially incubated with each DNA fragment, in the appropriate order; each incubation was carried out in ligation buffer from the kit.</p><p>For the last ligation reaction, we prepared a modified pHSG299 (TAKARA BIO) vector, designated pHSG299_CSPS. The vector pHSG299 was digested with <em>Eco</em>RI and <em>Sph</em>I, and the following sequence was inserted to introduce four meganuclease recognition sites—PI-<em>Sce</em>I, PI-<em>Psp</em>I, I-<em>Sce</em>I, and I-<em>Ceu</em>I—into the vector; in this sequence (GAATTCgcggccgcggccggcccgggcTGCCATTTCATTACCTCTTTCTCCGCACCCGACATAGATtcttctACCCATAATACCCATAATAGCTGTTTGCCAtcttctTAGGGATAACAGGGTAATtcttctTCGCTACCTTAGGACCGTTATAGTTAggcgcgcctgcaggGCATGC), the upper case characters indicate, respectively, the following restriction enzyme recognition sites—<em>Eco</em>RI, PI-<em>Sce</em>I, PI-<em>Psp</em>I, I-<em>Sce</em>I, I-<em>Ceu</em>I, and <em>Sph</em>I. The modified vector pHSG299_CSPS was amplified with two primers, CSPS-N2-CAA-u and CSPS-C2-u (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>), and digested with two enzymes, <em>Sfi</em>I and I-<em>Sce</em>I. The digestion product was used for the last sequential ligation.</p>                            <h3 scrollto-destination=6229420 id="6229420" class="section-title" data-legacy-id=s2f>2.6. Isolation of the ligation products</h3>
<p>The beads that contained the complete PRESSO product were first washed with 1.5 ml of TE buffer, and then rinsed with 200 μl of I-<em>Sce</em>I buffer. The beads were then incubated with 50 U of I-<em>Sce</em>I at 37°C for 2 h in 50 μl of I-<em>Sce</em>I buffer. To release the DNA construct containing the array of fragments from the beads, the samples were heated at 60°C for 10 min with sodium dodecyl sulphate (final concentration, 0.1%), and the supernatant was collected. The DNA was precipitated with ethanol, and then dissolved in 200 μl of buffer that contained 350 U of <em>E. coli</em> DNA ligase (TAKARA BIO). The intra-molecular ligation was performed at 16°C for 2 h. The ligated DNA was precipitated with ethanol, dissolved in 10 μl of distilled water, and used to transform <em>E. coli</em> DH5-α cells.</p>                            <h3 scrollto-destination=6229422 id="6229422" class="section-title" data-legacy-id=s2g>2.7. Antibiotic selection</h3>
<p>Transformed <em>E. coli</em> were selected on LB plates that contained chloramphenicol (Cm), and the QIAprep Spin Miniprep Kit (QIAGEN) was used to isolate the plasmid from cultures of transformed cells.</p>                            <h3 scrollto-destination=6229424 id="6229424" class="section-title" data-legacy-id=s2h>2.8. DNA digestions and sequencing</h3>
<p>The <em>Sfi</em>I and I-<em>Sce</em>I digestions have been described above. Many restriction enzymes—<em>Xba</em>I, <em>Bam</em>HI, <em>Eco</em>RI, <em>Hin</em>dIII, <em>Kpn</em>I, <em>Pst</em>I, <em>Sac</em>I, <em>Sal</em>I, <em>Sph</em>I, and PI-<em>Psp</em>I (TOYOBO)—were used to confirm structure of the resultant plasmid (Fig. <a reveal-id="DST032F3" data-open="DST032F3" class="link link-reveal link-table xref-fig">3</a>). The digestions were carried out according to the manufacturer's protocols. Each resultant plasmid was digested with I-<em>Sce</em>I and PI-<em>Psp</em>I to separate the insert DNA from the vector DNA. The insert DNA was then purified; aliquots of the purified insert DNA were then digested with each restriction enzyme in separate single-enzyme reactions (Fig. <a reveal-id="DST032F3" data-open="DST032F3" class="link link-reveal link-table xref-fig">3</a>). The products of each digestion reaction were resolved via gel electrophoresis. The length of each product was compared with two sets of DNA size markers—1 kb plus DNA ladder (Invitrogen) and HMW DNA marker (Invitrogen)—to confirm the structure of the insert DNA. The size of each genomic DNA fragment that was included in the PRESSO array is listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>. To estimate the sequence accuracy of insert DNA, four resultant plasmids with a 20-kb insert consisting of the array of 2.0 kb segments were sequenced. DNA sequences were determined using a Model 3730<em>xI</em> DNA analyzer (Applied Biosystems, Forster City, CA, USA). Sequencing reactions were performed with the BigDye® Terminator version 3.1 Cycle Sequencing Kit (Applied Biosystems) according to the manufacturer's instructions. </p>                        <a id="6229426" scrollto-destination="6229426"></a>
<div id="DST032F3" data-id="DST032F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst032/2/m_dst03203.png?Expires=2147483647&Signature=oO18a9l2DFbk4RX6bTwEKPuy49kjIUn~WepkfdwVgluQ1f1SUd7LlZ5OGxjlbJSxCROO4~BaKr8GubOPA~RRuVHIMz2tgUmvXpASFgaaKOpWvrNxS54y2xNb0aJyvWuzmN-GjpX~9GZYKPhpynwzkXgn0rgVPbjBxE5r76dR86mV-qq0bsNEKL0yXkrCK9M4RC7ikTEcc-lxKNIZHlh~nXkj0NF-yG1H1IAOh-ZfcEyZIb6z4dmo-zfgZxtgCIQLzb3l5cDzHU7iFdjETMWwLKGcuE1ig7vt6XhChBNMpksosjPT5lXZvgtChN9fLkHdFwse6Gn0fuXusMmV9m-e0Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Conformation of the PRESSO products. In each of three experiments, 10 different DNA fragments were ligated in an array along with a plasmid vector; the resulting plasmids were used to transform E. coli. Within each experiment; the lengths of all 10 DNA fragments were very similar 0.5, 1, or 2 kb; the inserts on resultant plasmids were 5, 10, or 20 kb, respectively, and were purified after the I-SceI and PI-PspI digestion; the isolated inserts were then used in 10 separate restriction digestion; each enzyme indicated was used to cut at 1 of the 10 segments in each of the three inserts. These restriction sites were designed in the primers for donor DNA amplification. The digested patterns were represented for the sequential ligation of 10 (A) 0.5, (B) 1, or (C) 2 kb DNA fragments. The lengths of digested DNA fragments were consistent with in the lengths of each individual donor DNA molecule listed in Supplementary Table S2. A SfiI site is also located at each connection between each pair of donor DNAs." path-from-xml="dst03203.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6229426/dst03203.tiff" path-from-xml="dst03203.tiff" target="_blank">View large</a><a section="6229426" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst032/2/dst03203.png?Expires=2147483647&Signature=i1rrk9IJN2pdqDMPq01hxYzzoETmlShvlK-2MG3noz7n4qJ~JcIcTbBNVYibbCTPUeGUQaenOw4Si0Kg3-NzInNxt6-zB46z3w0CHskfnXEI-kU2A0JQZfvyzkETQ087ylldAt6aorjnBtbNC957zkofhDW8UIuITHAP1foFF1vyh3pPe2X4ppyfkkMpcrV9gNz8FySC28nF9e~N32vvMTeBj1-Gve5wr5344E~so8NxG3PdRkxIaW8AVeB5SFRae6B236lMTXzOKZIAnEECxgbhQfEOYvIL8GZYEv0eNyRCQLxmCrF1quGAAqFpz1-9M7CMzBvtiMhO9GDsxXUzIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6229426&ar=420514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03203.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Conformation of the PRESSO products. In each of three experiments, 10 different DNA fragments were ligated in an array along with a plasmid vector; the resulting plasmids were used to transform <em>E. coli</em>. Within each experiment; the lengths of all 10 DNA fragments were very similar 0.5, 1, or 2 kb; the inserts on resultant plasmids were 5, 10, or 20 kb, respectively, and were purified after the I-<em>Sce</em>I and PI-<em>Psp</em>I digestion; the isolated inserts were then used in 10 separate restriction digestion; each enzyme indicated was used to cut at 1 of the 10 segments in each of the three inserts. These restriction sites were designed in the primers for donor DNA amplification. The digested patterns were represented for the sequential ligation of 10 (A) 0.5, (B) 1, or (C) 2 kb DNA fragments. The lengths of digested DNA fragments were consistent with in the lengths of each individual donor DNA molecule listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>. A <em>Sfi</em>I site is also located at each connection between each pair of donor DNAs.</p></div></div><div content-id="DST032F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst032/2/m_dst03203.png?Expires=2147483647&Signature=oO18a9l2DFbk4RX6bTwEKPuy49kjIUn~WepkfdwVgluQ1f1SUd7LlZ5OGxjlbJSxCROO4~BaKr8GubOPA~RRuVHIMz2tgUmvXpASFgaaKOpWvrNxS54y2xNb0aJyvWuzmN-GjpX~9GZYKPhpynwzkXgn0rgVPbjBxE5r76dR86mV-qq0bsNEKL0yXkrCK9M4RC7ikTEcc-lxKNIZHlh~nXkj0NF-yG1H1IAOh-ZfcEyZIb6z4dmo-zfgZxtgCIQLzb3l5cDzHU7iFdjETMWwLKGcuE1ig7vt6XhChBNMpksosjPT5lXZvgtChN9fLkHdFwse6Gn0fuXusMmV9m-e0Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Conformation of the PRESSO products. In each of three experiments, 10 different DNA fragments were ligated in an array along with a plasmid vector; the resulting plasmids were used to transform E. coli. Within each experiment; the lengths of all 10 DNA fragments were very similar 0.5, 1, or 2 kb; the inserts on resultant plasmids were 5, 10, or 20 kb, respectively, and were purified after the I-SceI and PI-PspI digestion; the isolated inserts were then used in 10 separate restriction digestion; each enzyme indicated was used to cut at 1 of the 10 segments in each of the three inserts. These restriction sites were designed in the primers for donor DNA amplification. The digested patterns were represented for the sequential ligation of 10 (A) 0.5, (B) 1, or (C) 2 kb DNA fragments. The lengths of digested DNA fragments were consistent with in the lengths of each individual donor DNA molecule listed in Supplementary Table S2. A SfiI site is also located at each connection between each pair of donor DNAs." path-from-xml="dst03203.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6229426/dst03203.tiff" path-from-xml="dst03203.tiff" target="_blank">View large</a><a section="6229426" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst032/2/dst03203.png?Expires=2147483647&Signature=i1rrk9IJN2pdqDMPq01hxYzzoETmlShvlK-2MG3noz7n4qJ~JcIcTbBNVYibbCTPUeGUQaenOw4Si0Kg3-NzInNxt6-zB46z3w0CHskfnXEI-kU2A0JQZfvyzkETQ087ylldAt6aorjnBtbNC957zkofhDW8UIuITHAP1foFF1vyh3pPe2X4ppyfkkMpcrV9gNz8FySC28nF9e~N32vvMTeBj1-Gve5wr5344E~so8NxG3PdRkxIaW8AVeB5SFRae6B236lMTXzOKZIAnEECxgbhQfEOYvIL8GZYEv0eNyRCQLxmCrF1quGAAqFpz1-9M7CMzBvtiMhO9GDsxXUzIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6229426&ar=420514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03203.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Conformation of the PRESSO products. In each of three experiments, 10 different DNA fragments were ligated in an array along with a plasmid vector; the resulting plasmids were used to transform <em>E. coli</em>. Within each experiment; the lengths of all 10 DNA fragments were very similar 0.5, 1, or 2 kb; the inserts on resultant plasmids were 5, 10, or 20 kb, respectively, and were purified after the I-<em>Sce</em>I and PI-<em>Psp</em>I digestion; the isolated inserts were then used in 10 separate restriction digestion; each enzyme indicated was used to cut at 1 of the 10 segments in each of the three inserts. These restriction sites were designed in the primers for donor DNA amplification. The digested patterns were represented for the sequential ligation of 10 (A) 0.5, (B) 1, or (C) 2 kb DNA fragments. The lengths of digested DNA fragments were consistent with in the lengths of each individual donor DNA molecule listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>. A <em>Sfi</em>I site is also located at each connection between each pair of donor DNAs.</p></div></div>                            <h2 scrollto-destination=6229427 id="6229427" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=6229428 id="6229428" class="section-title" data-legacy-id=s3a>3.1. Experimental design and overview of a sequential ligation of multiple DNA fragments</h3>
<p>We developed a PRESSO method that was based on non-palindromic sticky ends. In summary, there are three steps in PRESSO (Fig. <a reveal-id="DST032F1" data-open="DST032F1" class="link link-reveal link-table xref-fig">1</a>). First, a biotin-conjugated linker DNA with a meganuclease restriction site, e.g. the I-<em>Sce</em>I site, and a non-palindromic end cleaved by a restriction enzyme, e.g. <em>Sfi</em>I or <em>Bst</em>XI, is attached to streptavidin-linked beads. Secondly, each DNA fragment digested with <em>Sfi</em>I is then ligated, in an ordered sequence of iterative ligation reactions, to the growing end of the DNA construct, which is attached to the streptavidin-linked beads via a biotin label at the opposite end. Thirdly, after the vector sequences are added in the final sequential ligation step, the biotin-conjugated spacer DNA is removed from the construct via I-<em>Sce</em>I-catalyzed digestion, and the full-length linear DNA construct is circularized via an intra-molecular ligation to form a circular plasmid that is used to transform <em>E. coli</em>. Below, we describe the PRESSO method in detail.</p><p>Non-palindromic sequences recognized by some restriction enzymes, such as <em>Sfi</em>I and <em>Bst</em>XI, are used in PRESSO. Cleavage at such sites can generate any desire 3′-overhanging single-stranded sequences; these overhangs are derived from the bp in the middle of the recognition site. For instance, the restriction enzymes, <em>Sfi</em>I and <em>Bst</em>XI, recognize the sequences GGCCNNNNNGGCC and CCANNNNNNTGG, respectively, and produce 3′-overhanging ends comprising bases, which are denoted ‘N’, from the middle bp; any nucleotide can occupy any ‘N’ position in an <em>Sfi</em>I or <em>Bst</em>XI site, and these enzymes produced three or four ‘N’ base-overhang ends, respectively. Non-complementary overhanging bases at each end of each DNA molecule prevent an intra-molecular ligation. In contrast, if the overhanging end of the donor molecule is complementary to the overhanging end of the acceptor molecule, the unpaired bases can form Watson–Crick base pairs between the donor and acceptor DNA molecules, and iterative ligation of such donor–acceptor pairs can produce an array of many DNA fragments that are arranged in defined orientations within a defined array sequence.</p><p>PRESSO is carried out on a solid-phase, metallic streptavidin-conjugated beads, because these beads facilitate the removal of reaction intermediates after each ligation reaction, and this process precludes intra-molecular ligation (Fig. <a reveal-id="DST032F1" data-open="DST032F1" class="link link-reveal link-table xref-fig">1</a>). The solid phase permits repeated cycles of digestion, purification, and ligation. Streptavidin-conjugated magnetic beads were chosen as the solid phase, because streptavidin binds with high affinity to biotinylated oligonucleotides, and the interaction is sufficiently strong and stable during DNA manipulations.<sup><a reveal-id="DST032C14" data-open="DST032C14" class="link link-ref link-reveal xref-bibr">14</a></sup> The beads were also convenient for experimental handling, because they do not bind unlabelled DNA molecules. The initial double-strand DNA molecule (acceptor DNA) was designed to contain a biotinylated 5′-end and 3′-overhanging bases. The acceptor DNA was amplified with a 5′-biotinylated oligonucleotide and unlabelled primers, was cleaved with a non-palindromic restriction enzyme to generate a 3′-overhanging sequence that was complementary to the overhanging sequence on the first donor DNA molecules, and was then attached to the streptavidin beads.</p><p>The donor DNA can be prepared either by subcloning into a vector that contains non-palindromic restriction sites or by PCR amplification with appropriate primers. Subsequent digestion of the vector or PCR products with the non-palindromic enzyme, such as <em>Sfi</em>I, generates the donor DNA molecules with sticky ends. We purified donor DNA fragments by electrophoresis to remove undigested donor molecules and to remove any cleaved vector fragments that have overhang ends complementary to those of the donor molecules. These purification steps are especially critical for high yields when attempting to join multiple DNA fragments in a defined sequence, because ‘terminated’ molecules will accumulate as a result of incomplete digestion in each ligation cycle, and these ‘terminated’ molecules reduce the yield of the desired molecule. To avoid this problem, we used 5′-biotinylated primers to PCR amplify the donor molecules (Fig. <a reveal-id="DST032F2" data-open="DST032F2" class="link link-reveal link-table xref-fig">2</a>); the PCR products were digested with <em>Sfi</em>I, then mixed with streptavidin-linked beads, and finally, the unbound fully digested donor fragments were recovered from the supernatant; in contrast, ‘terminated’ partially digested molecules remained bound to the beads. This procedure allowed for the selection of completely digested non-palindromic ends and the successful sequential joining of 10 donor DNA molecules. The purified donor DNA was ligated to the 3′-end of acceptor DNA, which was bound to the beads. After each ligation, the liquid solution was removed and the beads were washed with buffer to accommodate addition of the next donor DNA molecules with non-palindromic ends. Repeated cycles of this procedure allowed the sequential addition of DNA fragments onto the growing acceptor molecules, which were fixed to the solid phase.</p><p>The release of the final composite DNA molecules from the solid support was achieved via digestion with the meganuclease I-<em>Sce</em>I, and an I-<em>Sce</em>I site had been engineered into the initial biotin-labelled linker acceptor fragment and into the final donor vector fragment. The I-<em>Sce</em>I restriction enzyme recognizes an 18-bp sequence, which is rarely present in any naturally occurring genome. Other commercially available meganucleases, such as I-<em>Ceu</em>I, could also be used for this purpose. Thus, we performed the sequential ligation on the solid phase. Here, as an example, we connected 10 DNA fragments derived from Chromosome 5 of the Arabidopsis genome.</p>                            <h3 scrollto-destination=6229434 id="6229434" class="section-title" data-legacy-id=s3b>3.2. Construction of a plasmid carrying 10 DNA fragments from Arabidopsis</h3>
<p>We conducted three prototypical experiments; for each experiment, 10 different donor DNA fragments all of approximately the same length (0.5, 1.0, or 2.0 kb in Experiments A, B, and C, respectively; Fig. <a reveal-id="DST032F3" data-open="DST032F3" class="link link-reveal link-table xref-fig">3</a>) were prepared, and each contained a unique restriction site so that the structure of each final plasmid construct could be determined (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). On Day 1, we amplified the biotin-conjugated spacer DNA, which included I-<em>Sce</em>I and <em>Sfi</em>I recognition sequences from the pUC19-I-<em>Sce</em>I plasmid and was used for all three experiments; this DNA was digested with <em>Sfi</em>I and attached to streptavidin-coated beads as the initial acceptor DNA. For each experiment, 10 unique fragments were prepared as donor DNA fragments; within an individual experiment, all 10 fragments were approximately the same length (0.5, 1.0, or 2.0 kb), but each had a distinct <em>Sfi</em>I site and a unique restriction site engineered into the primer. For all experiments, all amplified donor DNA fragments were digested with <em>Sfi</em>I, to produce distinct 3′-overhanging bases. Because undigested DNA fragments were removed from the solution by the streptavidin-linked beads, digested fragment could be purified from the supernatants and used as donor DNA molecules. On Day 2, five of each set of 10 DNA fragments were sequentially ligated to acceptor DNA on the streptavidin-linked beads; on Day 3, the remaining five donor molecules from each set of 10 were ligated to the acceptor arrays. After each sequential ligation reaction, the beads were washed, and the newly elongated construct was used as the acceptor molecule in the next ligation. The iterative reactions resulted in three separate arrays, each with 10 distinct DNA segments aligned in defined orientations and defined sequences. In each of the three experiments, the modified pHSG299 vector (pHSG299_CSPS), which encoded a chloramphenicol resistance gene and I-<em>Ceu</em>I, I-<em>Sce</em>I, PI-<em>Psp</em>I, and PI-<em>Sce</em>I recognition sequences, was used as the last donor molecule in the final sequential ligation.<sup><a reveal-id="DST032C15" data-open="DST032C15" class="link link-ref link-reveal xref-bibr">15</a></sup> The vector was digested with <em>Sfi</em>I and ligated to the 3′-overhang bases on final segment of the insert array, which was still attached to the streptavidin-linked beads. Each of the three final products of the sequential ligations was cleaved with I-<em>Sce</em>I to release it from the original biotin-labelled acceptor molecule and therefore, the streptavidin-linked beads. These free linear molecules, which each contained 10 distinct segments and a vector backbone, were circularized via an intra-molecular ligation; the circular constructs were then used to transform <em>E. coli</em>. The <em>E. coli</em> cells were cultured on LB plates that contained chloramphenicol overnight. On Day 4, chloramphenicol-resistant clones were isolated for plasmid preparation.</p>                            <h3 scrollto-destination=6229436 id="6229436" class="section-title" data-legacy-id=s3c>3.3. Confirmation of plasmids carrying 10 DNA fragments</h3>
<p>To determine whether the plasmids contain our expected sequences, we digested the plasmid with restriction enzymes specific for the 10 segments within each array. First, we separated the insert DNA, the array of 10 segments, from the pHSG299_CSPS backbone by digestion with I-<em>Sce</em>I and PI-<em>Psp</em>I, which was followed by elution via gel electrophoresis. To determine whether arrays were arranged as expected, we performed 10 separate digests for each of the three predicted inserts; each digest contained a distinct restriction enzyme that was targeted to a specific segment within an array (Fig. <a reveal-id="DST032F3" data-open="DST032F3" class="link link-reveal link-table xref-fig">3</a>). The predicted sizes of the DNA segment within the arrays containing 0.5, 1, or 2 kb segments are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>. The sizes of the DNA fragments that resulted from the restriction digests were consistent with these estimated segment lengths; therefore, each of the three inserts seemed to include the expected 10 segments in the expected order and orientation (Fig. <a reveal-id="DST032F3" data-open="DST032F3" class="link link-reveal link-table xref-fig">3</a>). As we were also anxious about the efficiency of the sequential ligation, we investigated how many plasmids contained all 10 DNA of the expected segments in a single experiment. To assess this, we isolated 12 plasmids carrying the inserts with an array of 2.0 kb segments, and the plasmids were digested with two restriction enzymes, I-<em>Sce</em>I and PI-<em>Psp</em>I, to separate the insert from the vector backbone. For arrays generated from the 2-kb fragments, the insert DNA from 9 of the 12 plasmids was ∼19.8 kb length, the insert DNA from one plasmid was ∼2 kb length, and other two plasmids carried no insert, suggesting that ∼75% of these transformants contained the expected constructs.</p><p>To estimate transformation efficiency of the plasmids obtained using the PRESSO method, we connected 2–10 DNA fragments to produce plasmids harbouring the inserts with an array of 0.5, 1.0, or 2.0 kb segments. In any case of PRESSO with &lt;10 segments with up to 2.0 kb inserts, we found more than tens of colonies, even though increasing size of DNA fragments and cycling reactions reduced the efficiency (Fig. <a reveal-id="DST032F4" data-open="DST032F4" class="link link-reveal link-table xref-fig">4</a>). Further, we isolated plasmids and investigated the insert length as described above. When we connected more than six DNA fragments using the PRESSO reactions, some plasmids included unexpected inserts. However, even when we connected 2 kb DNA fragments into an array of 10 segments, at least 10 colonies carrying our desired plasmid with correct size were isolated in our study. Thus, the efficiency of PRESSO is sufficient enough to connect multiple DNA fragments. </p>                        <a id="6229439" scrollto-destination="6229439"></a>
<div id="DST032F4" data-id="DST032F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst032/2/m_dst03204.png?Expires=2147483647&Signature=cDVjuiDmTX9GPeKz5EexA6mjBTFCpOrKw~fq7yno~UPLYX4slObrceZ0vbP7qwHRgVDvNOkcc78BS5qhLrrUnvwdvnOr4UaOcdFW1ucetXYVxTU68MnsjhvYMt199~Rte5wtZjK4pZ27-l6BA2bayZX-xgat2BVS3ZqlHPbE-SXDfgsQWIn6zzbSSDztxQzH08SJ0nK8cCd7nF2NRjssVYdgu-8reflT6FoCu-ktsh2dXEZOgam8VlBtolc9Lvn8RbxcgRU60ekOmx0YmCc2zQwhv3F4d-NT8MgeV3GYTNOe49~cTEnH4GCJ40-uvEWjGarEJQ6oRPqkxft1wf5QIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Effects of DNA length and ligation cycles. We ligated 2–10 DNA fragments of 0.5, 1.0, or 2.0 kb segments into the vector pHSG299_CSPS, and then transformed E. coli. The number of transformed colonies was shown. The error bars are standard deviations (SD)." path-from-xml="dst03204.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6229439/dst03204.tiff" path-from-xml="dst03204.tiff" target="_blank">View large</a><a section="6229439" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst032/2/dst03204.png?Expires=2147483647&Signature=FdiSAM28umVCAAvEMeuhq1vOxwH1p79mhW5Q-CXA~CX5iLu3E53ukmIk5QdExIMi4rbj4Ceu8RZQNIneOCEwSBlpbrTJCVeyyengFY9w3-hjMXFDKAwJtJSiv9vkfp4P2Hp1IGOlo7mVwYJprssQTmU2HPT0cS7qG6Nz3sL~V4cytpWOqB73XWt~3JidtH-2jpD76iIuG~gyaAbKdjFUDx5-BIbVfbqSHTk7kQFSX7N0pi65~dwKtKYeyNUDmjPeBh5n8APRhWDxx08TVcY-34bcq6KJZ8e-PhiCFHe4CRePkxIdAB4I65L4rUfsaOEKSr7uw7qB03nAYZufl1LKwQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6229439&ar=420514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03204.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Effects of DNA length and ligation cycles. We ligated 2–10 DNA fragments of 0.5, 1.0, or 2.0 kb segments into the vector pHSG299_CSPS, and then transformed <em>E. coli</em>. The number of transformed colonies was shown. The error bars are standard deviations (SD).</p></div></div><div content-id="DST032F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst032/2/m_dst03204.png?Expires=2147483647&Signature=cDVjuiDmTX9GPeKz5EexA6mjBTFCpOrKw~fq7yno~UPLYX4slObrceZ0vbP7qwHRgVDvNOkcc78BS5qhLrrUnvwdvnOr4UaOcdFW1ucetXYVxTU68MnsjhvYMt199~Rte5wtZjK4pZ27-l6BA2bayZX-xgat2BVS3ZqlHPbE-SXDfgsQWIn6zzbSSDztxQzH08SJ0nK8cCd7nF2NRjssVYdgu-8reflT6FoCu-ktsh2dXEZOgam8VlBtolc9Lvn8RbxcgRU60ekOmx0YmCc2zQwhv3F4d-NT8MgeV3GYTNOe49~cTEnH4GCJ40-uvEWjGarEJQ6oRPqkxft1wf5QIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Effects of DNA length and ligation cycles. We ligated 2–10 DNA fragments of 0.5, 1.0, or 2.0 kb segments into the vector pHSG299_CSPS, and then transformed E. coli. The number of transformed colonies was shown. The error bars are standard deviations (SD)." path-from-xml="dst03204.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6229439/dst03204.tiff" path-from-xml="dst03204.tiff" target="_blank">View large</a><a section="6229439" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst032/2/dst03204.png?Expires=2147483647&Signature=FdiSAM28umVCAAvEMeuhq1vOxwH1p79mhW5Q-CXA~CX5iLu3E53ukmIk5QdExIMi4rbj4Ceu8RZQNIneOCEwSBlpbrTJCVeyyengFY9w3-hjMXFDKAwJtJSiv9vkfp4P2Hp1IGOlo7mVwYJprssQTmU2HPT0cS7qG6Nz3sL~V4cytpWOqB73XWt~3JidtH-2jpD76iIuG~gyaAbKdjFUDx5-BIbVfbqSHTk7kQFSX7N0pi65~dwKtKYeyNUDmjPeBh5n8APRhWDxx08TVcY-34bcq6KJZ8e-PhiCFHe4CRePkxIdAB4I65L4rUfsaOEKSr7uw7qB03nAYZufl1LKwQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6229439&ar=420514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03204.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Effects of DNA length and ligation cycles. We ligated 2–10 DNA fragments of 0.5, 1.0, or 2.0 kb segments into the vector pHSG299_CSPS, and then transformed <em>E. coli</em>. The number of transformed colonies was shown. The error bars are standard deviations (SD).</p></div></div><p>Further, to investigate the accuracy of the PCR and the ligation reactions, we pick up 4 of 9 correct plasmids as examples, and determined the sequence of inserts within the arrays containing 2 kb segments. One of the four plasmids carried the insert with a correct sequence, and other three plasmids included a couple of errors in the insert sequences. We found total of five errors only in the junction sequences, which were originated as synthetic oligonucleotide PCR primers to amplify multiple DNA fragments. The errors found in the nucleotide sequence produced by the chemical synthesis are reportedly between 160 and 1394 bp per error.<sup><a reveal-id="DST032C10" data-open="DST032C10" class="link link-ref link-reveal xref-bibr">10</a></sup> In contrast, PRESSO resulted in a ∼15 840 correct bp for each incorrect bp based on the DNA sequence data; as total of five errors were detected in four clones of 19.8 kb inserts, and the error ratio was estimated as five errors/4 × 19.8 kb = 1/15 840. However, the errors in junction sequences ordinary will not affect the further analyses.</p>                            <h2 scrollto-destination=6229441 id="6229441" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>Here, we describe PRESSO, a method for arranging multiple DNA fragments into a single-defined array; this method is an up-to-date technology for handling multiple genes. The solid phase allows researchers to conduct iterative sets of reactions; each set includes a restriction digest, purification, and ligation; in this way, multiple fragments can be arranged into a single-defined array. In this study, we produced a large plasmid that contained a 5, 10, or 20 kb insert, which comprised 10 segments in defined orientations and a defined order. Ligation reactions that join sets of multiple DNA fragments are carried out as a routine task in our laboratory (data not shown). By combining PRESSO with standard cloning techniques, we were able to ligate 22 DNA fragments together in an array that encoded seven functional genes involved in astaxanthin synthesis, and confirmed that this construct indeed functions in transgenic plants<sup><a reveal-id="DST032C15" data-open="DST032C15" class="link link-ref link-reveal xref-bibr">15</a></sup> (also see below).</p><p>PRESSO has some advantages over other methods of DNA cloning; these advantages include high accuracy and efficiency. Although many methods for ligating multiple DNA fragments together have been developed previously, some of these methods that involve non-palindromic enzymes or Gateway systems can be used to connect only three or four DNA fragments together.<sup><a reveal-id="DST032C16" data-open="DST032C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DST032C17" data-open="DST032C17" class="link link-ref link-reveal xref-bibr">17</a></sup> These step-by-step methods are more time consuming than the PRESSO method. In methods, such as PAM or the In-Fusion system, all reactions were performed in a single tube without a solid phase;<sup><a reveal-id="DST032C10  DST032C12" data-open="DST032C10  DST032C12" class="link link-ref link-reveal xref-bibr">10–12</a></sup> consequently, all DNA fragments are ligated together simultaneously, and the methods are faster than PRESSO. However, these methods that lack stepwise reactions produce undesired DNA sequences, caused by disorder or mismatched ligations. In fact, with the In-Fusion system, the number of transformants recovered decreases as the number of DNA fragments increases.<sup><a reveal-id="DST032C11" data-open="DST032C11" class="link link-ref link-reveal xref-bibr">11</a></sup> In contrast, PRESSO, which depends on stepwise reactions on a solid phase, allows us to connect up to 10 cassettes precisely as shown in this study, and a longer DNA fragment in a plasmid can be prepared by ligating building blocks in which up to 10 DNA fragments are incorporated using the PRESSO protocol as shown in our previous research.<sup><a reveal-id="DST032C15" data-open="DST032C15" class="link link-ref link-reveal xref-bibr">15</a></sup> By using each DNA fragments with distinct overhang ends, we could avoid disorder or mismatched ligations as seen in the previous methods.<sup><a reveal-id="DST032C10  DST032C12" data-open="DST032C10  DST032C12" class="link link-ref link-reveal xref-bibr">10–12</a></sup> In our preliminary experiments, although we carried out the ligation reactions with same overhang sequences repeatedly in one assemble, we could not isolate our designed plasmids harbouring multiple DNA fragments; by skipping internal DNA fragments, many plasmids included less DNA segments than those that we desired. Using the distinct overhang ends is essential for the reaction. Recent reports about a fast ligation based on an automatable solid-phase high-throughput (FLASH) system for TALEffector nucleases (TALENs), which encode sequence-specific artificial nucleases, have also allowed for iterative ligations on a solid phase.<sup><a reveal-id="DST032C18" data-open="DST032C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Thus, PRESSO is effective for repeated ligation reactions.</p><p>The PRESSO allows us to connect multiple DNA fragments together with the high sequence accuracy. In many previous reports, chemical nucleotide syntheses coupled with PCR and ligation reactions have been used to produce giant DNAs.<sup><a reveal-id="DST032C13" data-open="DST032C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DST032C19" data-open="DST032C19" class="link link-ref link-reveal xref-bibr">19</a></sup> The nucleotide sequences produced by chemical syntheses contain more errors than standard PCR or cloning as described in our results and in a previous report.<sup><a reveal-id="DST032C10" data-open="DST032C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Such undesired mutations in the oligonucleotides are also seen in long-range PCR primers, which are synthesized for the polymerase cycling assembly or for production of the overhanging ends. As long-range PCR for overlap extensions in PAM requires a low-fidelity but high-efficiency polymerase, the amplification of &gt;20 kb of DNA often contains multiple errors.<sup><a reveal-id="DST032C12" data-open="DST032C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DST032C20" data-open="DST032C20" class="link link-ref link-reveal xref-bibr">20</a></sup> In contrast to the previous reports,<sup><a reveal-id="DST032C12" data-open="DST032C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DST032C20" data-open="DST032C20" class="link link-ref link-reveal xref-bibr">20</a></sup> the PRESSO system can suppress nucleotide substitutions as shown in our results, because DNA fragments can be designed to be short enough to be amplified with a high-fidelity polymerase.</p><p>An important goal in biotechnology is to transfer multiple genes into an organism, so that they function <em>in vivo</em>. Using PRESSO, we were able to link 24 DNA fragments to create a single construct comprising a plasmid vector, a single enhancer, and seven functional genes involved in astaxanthin synthesis each within a separate cassette that included a promoter and a operator.<sup><a reveal-id="DST032C15" data-open="DST032C15" class="link link-ref link-reveal xref-bibr">15</a></sup> We then introduced the constructed plasmid into plants and confirmed that astaxanthin was synthesized in the transgenic plants.<sup><a reveal-id="DST032C15" data-open="DST032C15" class="link link-ref link-reveal xref-bibr">15</a></sup> A decade ago, simultaneous introduction via co-bombardment of multiple independent DNA molecules (genes) into rice cells was described.<sup><a reveal-id="DST032C21" data-open="DST032C21" class="link link-ref link-reveal xref-bibr">21</a></sup> However, this co-bombardment approach required that many transformant populations had to be generated and screened to recover the desired transgenic plants for the presence of multiple genes, because the transgenic plants that harboured the desired combination of genes were very rare. This type of strategy is feasible for only some crops, such as rice or tobacco plants, that are readily transformed with high efficiency. In contrast, the PRESSO system described here will facilitate the introduction of a set of genes for a certain metabolic pathway from one species into another species even when the host species supports only low transformation rates. This capability system will open up new opportunities for the production of valuable materials in an easily manageable organism for both basic research and industrial purposes.</p><p>A RIKEN Arabidopsis full-length (RAFL) cDNA library (RIKEN, Japan) is a well-established library generated from the Arabidopsis genome; this library covered ∼17 000 of the protein-coding sequences from Arabidopsis, and each sequence was constructed with an <em>Sfi</em>I site.<sup><a reveal-id="DST032C22" data-open="DST032C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DST032C23" data-open="DST032C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Such full-length cDNA clones engineered with a non-palindromic restriction enzyme site can be used for PRESSO without PCR amplification of genes. By using available DNA clones, PRESSO becomes a powerful tool because it is quicker, more precise, and more cost-effective than the previously described methods.</p><p>To generate transgenic organisms that express multiple heterologous genes, adaptor cassettes that contain a promoter or reporter may be required. By coupling PRESSO with such cassettes that also include sequences complementally to the ends of cDNA clones, many cDNAs could be directly ligated together in tandem. In our PRESSO system described here, each donor DNA molecule contains designed linker sequences. We believe that this strategy is not problematic in most of the cases, especially for the joining functional genes (as demonstrated in the previous reports<sup><a reveal-id="DST032C15" data-open="DST032C15" class="link link-ref link-reveal xref-bibr">15</a></sup>). However, if a specific junction sequence is required, as in the case of in-frame ligation of certain protein-coding sequences, Type IIS restriction endonucleases, such as FokI, could be used. Such nucleases cut DNA at the outside sequence of their non-palindromic recognition sites. Site-directed mutagenesis is another approach that could be used to change a sequence as desired after a based plasmid has been constructed.</p><p>Multiple PRESSO reactions can be carried out in parallel. Currently, we are working on the simultaneous syntheses of multiple sequential ligation sets; these sets were designed based on network analyses, such as transcriptome or metabolome studies.<sup><a reveal-id="DST032C3" data-open="DST032C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DST032C4" data-open="DST032C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DST032C6" data-open="DST032C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DST032C7" data-open="DST032C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Our approach will greatly facilitate the comprehensive characterization of gene function that is currently underway in various organisms.</p>                            <h2 scrollto-destination=6229449 id="6229449" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=6229451 id="6229451" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by the ‘Innovation Technology for the Earth Program, Development of Transgenic Plants for Production of Industrial Materials Project’ from the New Energy and Industrial Technology Development Organization (NEDO) as a part of the ‘Green Biotechnology Program’, in coordination with the Ministry of Economy, Trade, and Industry (METI); it was also supported in part by the Kazusa DNA Research Institute Foundation.</p>                            <h2 scrollto-destination=6229453 id="6229453" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Yumiko Shirano for useful suggestions of connecting DNA molecules with sticky ends produced by the restriction enzyme <em>Sfi</em>I, and Junko Takatsu, Eri Okuyama, and Misa Ki for their technical assistance.</p>                            <h2 scrollto-destination=6229455 id="6229455" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST032C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST032C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cohen</div> <div class="given-names">S.N.</div></div>,  <div class="name"><div class="surname">Chang</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">Boyer</div> <div class="given-names">H.W.</div></div>,  <div class="name"><div class="surname">Helling</div> <div class="given-names">R.B.</div></div>. </div><div class="article-title">Construction of biologically functional bacterial plasmids in vitro</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1973</div>, vol. <div class="volume">70</div> (pg. <div class="fpage">3240</div>-<div class="lpage">4</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.70.11.3240" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.70.11.3240</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20biologically%20functional%20bacterial%20plasmids%20in%20vitro&amp;author=S.N.%20Cohen&amp;author=A.C.%20Chang&amp;author=H.W.%20Boyer&amp;author=R.B.%20Helling&amp;publication_year=1973&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=70&amp;pages=3240-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.70.11.3240" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.70.11.3240" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.70.11.3240"> </span></div> </div></div></div></div></div><div content-id="DST032C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST032C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hanano</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Amagai</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kaneko</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Analysis of gene expression in <em>Arabidopsis thaliana</em> by array hybridization with genomic DNA fragments aligned along chromosomal regions</div>, <div class="source">Plant J.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">247</div>-<div class="lpage">55</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1046/j.1365-313X.2002.01277.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1046/j.1365-313X.2002.01277.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20gene%20expression%20in%20Arabidopsis%20thaliana%20by%20array%20hybridization%20with%20genomic%20DNA%20fragments%20aligned%20along%20chromosomal%20regions&amp;author=S.%20Hanano&amp;author=M.%20Amagai&amp;author=T.%20Kaneko&amp;publication_year=2002&amp;journal=Plant%20J.&amp;volume=30&amp;pages=247-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.2002.01277.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.2002.01277.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.2002.01277.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12000460" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST032C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST032C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Obayashi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kinoshita</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nakai</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">ATTED-II: a database of co-expressed genes and cis elements for identifying co-regulated gene groups in <em>Arabidopsis</em></div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">D863</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkl783" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkl783</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ATTED-II%3A%20a%20database%20of%20co-expressed%20genes%20and%20cis%20elements%20for%20identifying%20co-regulated%20gene%20groups%20in%20Arabidopsis&amp;author=T.%20Obayashi&amp;author=K.%20Kinoshita&amp;author=K.%20Nakai&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=D863-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl783" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl783" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl783"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17130150" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST032C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST032C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Obayashi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Okegawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sasaki-Sekimoto</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Distinctive features of plant organs characterized by global analysis of gene expression in <em>Arabidopsis</em></div>, <div class="source">DNA Res.</div> , <div class="year">2004</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">11</div>-<div class="lpage">25</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/11.1.11" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/11.1.11</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distinctive%20features%20of%20plant%20organs%20characterized%20by%20global%20analysis%20of%20gene%20expression%20in%20Arabidopsis&amp;author=T.%20Obayashi&amp;author=T.%20Okegawa&amp;author=Y.%20Sasaki-Sekimoto&amp;publication_year=2004&amp;journal=DNA%20Res.&amp;volume=11&amp;pages=11-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/11.1.11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F11.1.11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F11.1.11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15141942" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST032C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST032C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ogata</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sakurai</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Shibata</div> <div class="given-names">D.</div></div>. </div><div class="article-title">CoP: a database for characterizing co-expressed gene modules with biological information in plants</div>, <div class="source">Bioinformatics</div> , <div class="year">2010</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">1267</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/btq121" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/btq121</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CoP%3A%20a%20database%20for%20characterizing%20co-expressed%20gene%20modules%20with%20biological%20information%20in%20plants&amp;author=Y.%20Ogata&amp;author=H.%20Suzuki&amp;author=N.%20Sakurai&amp;author=D.%20Shibata&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=1267-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq121" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20305269" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST032C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST032C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sakurai</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Ara</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ogata</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">KaPPA-View4: a metabolic pathway database for representation and analysis of correlation networks of gene co-expression and metabolite co-accumulation and omics data</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2011</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">D677</div>-<div class="lpage">84</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkq989" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkq989</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KaPPA-View4%3A%20a%20metabolic%20pathway%20database%20for%20representation%20and%20analysis%20of%20correlation%20networks%20of%20gene%20co-expression%20and%20metabolite%20co-accumulation%20and%20omics%20data&amp;author=N.%20Sakurai&amp;author=T.%20Ara&amp;author=Y.%20Ogata&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=D677-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq989" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq989" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq989"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21097783" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST032C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST032C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tokimatsu</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sakurai</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">KaPPA-view: a web-based analysis tool for integration of transcript and metabolite data on plant metabolic pathway maps</div>, <div class="source">Plant Physiol.</div> , <div class="year">2005</div>, vol. <div class="volume">138</div> (pg. <div class="fpage">1289</div>-<div class="lpage">300</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.105.060525" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.105.060525</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KaPPA-view%3A%20a%20web-based%20analysis%20tool%20for%20integration%20of%20transcript%20and%20metabolite%20data%20on%20plant%20metabolic%20pathway%20maps&amp;author=T.%20Tokimatsu&amp;author=N.%20Sakurai&amp;author=H.%20Suzuki&amp;publication_year=2005&amp;journal=Plant%20Physiol.&amp;volume=138&amp;pages=1289-300" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.105.060525" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.105.060525" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.105.060525"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16010003" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST032C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST032C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="collab">Arabidopsis Interactome Mapping Consortium.</div><div class="article-title">Evidence for network evolution in an Arabidopsis interactome map</div>, <div class="source">Science</div> , <div class="year">2011</div>, vol. <div class="volume">333</div> (pg. <div class="fpage">601</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1203877" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1203877</a></div><div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1203877" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1203877" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1203877"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21798944" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST032C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST032C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lin</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Shen</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Mao</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>. </div><div class="article-title">The predicted Arabidopsis interactome resource and network topology-based systems biology analyses</div>, <div class="source">Plant Cell</div> , <div class="year">2011</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">911</div>-<div class="lpage">22</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1105/tpc.110.082529" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1105/tpc.110.082529</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20predicted%20Arabidopsis%20interactome%20resource%20and%20network%20topology-based%20systems%20biology%20analyses&amp;author=M.%20Lin&amp;author=X.%20Zhou&amp;author=X.%20Shen&amp;author=C.%20Mao&amp;author=X.%20Chen&amp;publication_year=2011&amp;journal=Plant%20Cell&amp;volume=23&amp;pages=911-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.110.082529" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.110.082529" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.110.082529"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21441435" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST032C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST032C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tian</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Gong</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sheng</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Accurate multiplex gene synthesis from programmable DNA microchips</div>, <div class="source">Nature</div> , <div class="year">2004</div>, vol. <div class="volume">432</div> (pg. <div class="fpage">1050</div>-<div class="lpage">4</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature03151" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature03151</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accurate%20multiplex%20gene%20synthesis%20from%20programmable%20DNA%20microchips&amp;author=J.%20Tian&amp;author=H.%20Gong&amp;author=N.%20Sheng&amp;publication_year=2004&amp;journal=Nature&amp;volume=432&amp;pages=1050-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03151" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03151" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03151"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15616567" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST032C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST032C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhu</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Cai</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Hall</div> <div class="given-names">E.O.</div></div>,  <div class="name"><div class="surname">Freeman</div> <div class="given-names">G.J.</div></div>. </div><div class="article-title">In-fusion assembly: seamless engineering of multidomain fusion proteins, modular vectors, and mutations</div>, <div class="source">Biotechniques</div> , <div class="year">2007</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">354</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.2144/000112536" class="link link-pub-id-doi" target="_blank">https://doi.org/10.2144/000112536</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In-fusion%20assembly%3A%20seamless%20engineering%20of%20multidomain%20fusion%20proteins%2C%20modular%20vectors%2C%20and%20mutations&amp;author=B.%20Zhu&amp;author=G.%20Cai&amp;author=E.O.%20Hall&amp;author=G.J.%20Freeman&amp;publication_year=2007&amp;journal=Biotechniques&amp;volume=43&amp;pages=354-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2144/000112536" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2144%2F000112536" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2144%2F000112536"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17907578" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST032C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST032C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gibson</div> <div class="given-names">D.G.</div></div>,  <div class="name"><div class="surname">Benders</div> <div class="given-names">G.A.</div></div>,  <div class="name"><div class="surname">Andrews-Pfannkoch</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Complete chemical synthesis, assembly, and cloning of a <em>Mycoplasma genitalium</em> genome</div>, <div class="source">Science</div> , <div class="year">2008</div>, vol. <div class="volume">319</div> (pg. <div class="fpage">1215</div>-<div class="lpage">20</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1151721" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1151721</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20chemical%20synthesis%2C%20assembly%2C%20and%20cloning%20of%20a%20Mycoplasma%20genitalium%20genome&amp;author=D.G.%20Gibson&amp;author=G.A.%20Benders&amp;author=C.%20Andrews-Pfannkoch&amp;publication_year=2008&amp;journal=Science&amp;volume=319&amp;pages=1215-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1151721" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1151721" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1151721"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18218864" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST032C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST032C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gibson</div> <div class="given-names">D.G.</div></div>,  <div class="name"><div class="surname">Young</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Chuang</div> <div class="given-names">R.Y.</div></div>,  <div class="name"><div class="surname">Venter</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Hutchison</div> <div class="given-names">C.A.</div>III</div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">H.O.</div></div>. </div><div class="article-title">Enzymatic assembly of DNA molecules up to several hundred kilobases</div>, <div class="source">Nat. Methods</div> , <div class="year">2009</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">343</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nmeth.1318" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nmeth.1318</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enzymatic%20assembly%20of%20DNA%20molecules%20up%20to%20several%20hundred%20kilobases&amp;author=D.G.%20Gibson&amp;author=L.%20Young&amp;author=R.Y.%20Chuang&amp;author=J.C.%20Venter&amp;author=C.A.%20Hutchison&amp;author=H.O.%20Smith&amp;publication_year=2009&amp;journal=Nat.%20Methods&amp;volume=6&amp;pages=343-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1318" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1318" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1318"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19363495" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST032C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST032C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Holmberg</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Blomstergren</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Nord</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Lukacs</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Lundeberg</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Uhlén</div> <div class="given-names">M.</div></div>. </div><div class="article-title">The biotin-streptavidin interaction can be reversibly broken using water at elevated temperatures</div>, <div class="source">Electrophoresis</div> , <div class="year">2005</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">501</div>-<div class="lpage">10</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/elps.200410070" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/elps.200410070</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20biotin-streptavidin%20interaction%20can%20be%20reversibly%20broken%20using%20water%20at%20elevated%20temperatures&amp;author=A.%20Holmberg&amp;author=A.%20Blomstergren&amp;author=O.%20Nord&amp;author=M.%20Lukacs&amp;author=J.%20Lundeberg&amp;author=M.%20Uhl%C3%A9n&amp;publication_year=2005&amp;journal=Electrophoresis&amp;volume=26&amp;pages=501-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/elps.v26:3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Felps.v26:3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Felps.v26:3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15690449" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST032C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST032C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fujisawa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Takita</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Harada</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Pathway engineering of <em>Brassica napus</em> seeds using multiple key enzyme genes involved in ketocarotenoid formation</div>, <div class="source">J. Exp. Bot.</div> , <div class="year">2009</div>, vol. <div class="volume">60</div> (pg. <div class="fpage">1319</div>-<div class="lpage">32</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/jxb/erp006" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/jxb/erp006</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pathway%20engineering%20of%20Brassica%20napus%20seeds%20using%20multiple%20key%20enzyme%20genes%20involved%20in%20ketocarotenoid%20formation&amp;author=M.%20Fujisawa&amp;author=E.%20Takita&amp;author=H.%20Harada&amp;publication_year=2009&amp;journal=J.%20Exp.%20Bot.&amp;volume=60&amp;pages=1319-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erp006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferp006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferp006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19204032" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST032C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST032C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marsischky</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">LaBaer</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Many paths to many clones: a comparative look at high-throughput cloning methods</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">2020</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.2528804" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.2528804</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Many%20paths%20to%20many%20clones%3A%20a%20comparative%20look%20at%20high-throughput%20cloning%20methods&amp;author=G.%20Marsischky&amp;author=J.%20LaBaer&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=2020-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2528804" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2528804" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2528804"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15489321" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST032C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST032C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhou</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">Z.R.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">C.Z.</div></div>, et al. </div><div class="article-title">A novel strategy for rapid construction of libraries of full-length antibodies highly expressed on mammalian cell surfaces</div>, <div class="source">Acta Biochim. Biophys. Sin.</div> , <div class="year">2010</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">575</div>-<div class="lpage">84</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/abbs/gmq055" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/abbs/gmq055</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20strategy%20for%20rapid%20construction%20of%20libraries%20of%20full-length%20antibodies%20highly%20expressed%20on%20mammalian%20cell%20surfaces&amp;author=Y.%20Zhou&amp;author=Z.R.%20Chen&amp;author=C.Z.%20Li&amp;publication_year=2010&amp;journal=Acta%20Biochim.%20Biophys.%20Sin.&amp;volume=42&amp;pages=575-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/abbs/gmq055" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fabbs%2Fgmq055" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fabbs%2Fgmq055"> </span></div> </div></div></div></div></div><div content-id="DST032C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST032C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Reyon</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Tsai</div> <div class="given-names">S.Q.</div></div>,  <div class="name"><div class="surname">Khayter</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Foden</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Sander</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Joung</div> <div class="given-names">J.K.</div></div>. </div><div class="article-title">FLASH assembly of TALENs for high-throughput genome editing</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2012</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">460</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nbt.2170" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nbt.2170</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=FLASH%20assembly%20of%20TALENs%20for%20high-throughput%20genome%20editing&amp;author=D.%20Reyon&amp;author=S.Q.%20Tsai&amp;author=C.%20Khayter&amp;author=J.A.%20Foden&amp;author=J.D.%20Sander&amp;author=J.K.%20Joung&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol.&amp;volume=30&amp;pages=460-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2170" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2170" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2170"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22484455" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST032C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST032C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gibson</div> <div class="given-names">D.G.</div></div>,  <div class="name"><div class="surname">Glass</div> <div class="given-names">J.I.</div></div>,  <div class="name"><div class="surname">Lartigue</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Creation of a bacterial cell controlled by a chemically synthesized genome</div>, <div class="source">Science</div> , <div class="year">2010</div>, vol. <div class="volume">329</div> (pg. <div class="fpage">52</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1190719" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1190719</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Creation%20of%20a%20bacterial%20cell%20controlled%20by%20a%20chemically%20synthesized%20genome&amp;author=D.G.%20Gibson&amp;author=J.I.%20Glass&amp;author=C.%20Lartigue&amp;publication_year=2010&amp;journal=Science&amp;volume=329&amp;pages=52-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1190719" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1190719" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1190719"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20488990" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST032C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST032C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vroom</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">C.L.</div></div>. </div><div class="article-title">Modular construction of plasmids through ligation-free assembly of vector components with oligonucleotide linkers</div>, <div class="source">Biotechniques</div> , <div class="year">2008</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">924</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.2144/000112808" class="link link-pub-id-doi" target="_blank">https://doi.org/10.2144/000112808</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Modular%20construction%20of%20plasmids%20through%20ligation-free%20assembly%20of%20vector%20components%20with%20oligonucleotide%20linkers&amp;author=J.A.%20Vroom&amp;author=C.L.%20Wang&amp;publication_year=2008&amp;journal=Biotechniques&amp;volume=44&amp;pages=924-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2144/000112808" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2144%2F000112808" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2144%2F000112808"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18533903" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST032C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST032C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Marmey</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Taylor</div> <div class="given-names">N.J.</div></div>, et al. </div><div class="article-title">Expression and inheritance of multiple transgenes in rice plants</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">1998</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">1060</div>-<div class="lpage">4</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/3455" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/3455</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20and%20inheritance%20of%20multiple%20transgenes%20in%20rice%20plants&amp;author=L.%20Chen&amp;author=P.%20Marmey&amp;author=N.J.%20Taylor&amp;publication_year=1998&amp;journal=Nat.%20Biotechnol.&amp;volume=16&amp;pages=1060-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/3455" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F3455" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F3455"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9831036" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST032C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST032C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Seki</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Narusaka</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kamiya</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Functional annotation of a full-length Arabidopsis cDNA collection</div>, <div class="source">Science</div> , <div class="year">2002</div>, vol. <div class="volume">296</div> (pg. <div class="fpage">141</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1071006" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1071006</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20annotation%20of%20a%20full-length%20Arabidopsis%20cDNA%20collection&amp;author=M.%20Seki&amp;author=M.%20Narusaka&amp;author=A.%20Kamiya&amp;publication_year=2002&amp;journal=Science&amp;volume=296&amp;pages=141-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1071006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1071006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1071006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11910074" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST032C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST032C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Seki</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Satou</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sakurai</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">RIKEN Arabidopsis full-length (RAFL) cDNA and its applications for expression profiling under abiotic stress conditions</div>, <div class="source">J. Exp. Bot.</div> , <div class="year">2004</div>, vol. <div class="volume">55</div> (pg. <div class="fpage">213</div>-<div class="lpage">23</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/jxb/erh007" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/jxb/erh007</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RIKEN%20Arabidopsis%20full-length%20(RAFL)%20cDNA%20and%20its%20applications%20for%20expression%20profiling%20under%20abiotic%20stress%20conditions&amp;author=M.%20Seki&amp;author=M.%20Satou&amp;author=T.%20Sakurai&amp;publication_year=2004&amp;journal=J.%20Exp.%20Bot.&amp;volume=55&amp;pages=213-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erh007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferh007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferh007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14673034" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Prof. Masahira Hattori</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=420514&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst032/2/dst032-dst032supp_table2.pdf?Expires=2147483647&amp;Signature=263BhJOhqAtXkdYYOIrS6GNvtkpFHHyOsaXd2BUssHRYFyllE4s2FRXxEDUkWSCr7KDj1hi5LtAnM9nkcWs-rytM6wL9hnxw52MgyXnO1DbuEr73dGLy3l1w9LVifB9PcrbnZJLVrek6LY7mjtHh-zeoakXTbCwDepqOKjm6NLrcvPSGfbcVVotGAzvmIwZavJYDrSVYzI~wuOdEHgGotH-XtAA3q50pkWEpsZdkhSrT5tp~LuQRKgN5yEsRqhPkwRmRsziO1nsHRpgufAQ6Nzr8ITs-nxJlqfFGl~S9hrEXYJGaaIvLBwiA0~of9gHT9Miz~~Lvqe~dERh0YrPtAA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst032/2/dst032-dst032supp_table1.pdf?Expires=2147483647&amp;Signature=P1F55N6MK~sHft0X43CUa7rde85cpyqc-WfviiUmVNV1V0lDFDmtpfMJ3eqYlIrduCaGDDATw5O7FOUD42Jxu2STECfe6dH-AkQ5PdjZdc23Y4IYS4IMjyTohuWOxUjWo03dZfrZGGVNl00eri8sVQMfMH0xQE8nVHP5AZzTRUDPhSl2AsT4n9KRSGUYIWSU0Swqr-5Xse~jJepa7ria~7VFXjVx~2~MNvmm4CI7t4R6aQpRAVWyhaQL66yFCD1KOm9KEtrUXHAk7Egn2Rta0SuZsxXH-gWAcCnUS74L50A1cZmCz8BbH-WkrMNegiXVXBHB6YL8Y5smsaNTlyNulw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">317</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000330168100005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">4</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst032"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="420514"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000330168100005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f6%2f583%2f420514">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000330168100005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (4)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F6%2F583%2F420514" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/420514'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Precise Sequential DNA Ligation on A Solid Substrate: Solid-Based Rapid Sequential Ligation of Multiple DNA Molecules" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst032");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f6%2f583%2f420514">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

