

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/17/4/245/397387","@type":"ScholarlyArticle","name":"Comprehensive Analysis of the Palindromic Motif TCTCGCGAGA: A Regulatory Element of the  HNRNPK  Promoter","datePublished":"2010-06-29","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/17/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2010-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsq016","keywords":["HNRNPK promoter","palindromic motif","EMSA","mass spectrometry","chromatin immunoprecipitation"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/17/4/245/397387","author":[{"name":"Mikula, Michal","affiliation":"1  Department of Gastroenterology ,  Maria Sklodowska-Curie Memorial Cancer Center and Institute of Oncology ,  Roentgena 5, 02-781 Warsaw ,  Poland","@type":"Person"},{"name":"Gaj, Pawel","affiliation":"2  Department of Gastroenterology and Hepatology ,  Medical Center for Postgraduate Education ,  Warsaw ,  Poland","@type":"Person"},{"name":"Dzwonek, Karolina","affiliation":"1  Department of Gastroenterology ,  Maria Sklodowska-Curie Memorial Cancer Center and Institute of Oncology ,  Roentgena 5, 02-781 Warsaw ,  Poland","@type":"Person"},{"name":"Rubel, Tymon","affiliation":"3  Laboratory of Bioinformatics and Systems Biology ,  Maria Sklodowska-Curie Memorial Cancer Center and Institute of Oncology ,  Roentgena 5, 02-781 Warsaw ,  Poland","@type":"Person"},{"name":"Karczmarski, Jakub","affiliation":"1  Department of Gastroenterology ,  Maria Sklodowska-Curie Memorial Cancer Center and Institute of Oncology ,  Roentgena 5, 02-781 Warsaw ,  Poland","@type":"Person"},{"name":"Paziewska, Agnieszka","affiliation":"2  Department of Gastroenterology and Hepatology ,  Medical Center for Postgraduate Education ,  Warsaw ,  Poland","@type":"Person"},{"name":"Dzwonek, Artur","affiliation":"1  Department of Gastroenterology ,  Maria Sklodowska-Curie Memorial Cancer Center and Institute of Oncology ,  Roentgena 5, 02-781 Warsaw ,  Poland","@type":"Person"},{"name":"Bragoszewski, Piotr","affiliation":"2  Department of Gastroenterology and Hepatology ,  Medical Center for Postgraduate Education ,  Warsaw ,  Poland","@type":"Person"},{"name":"Dadlez, Michal","affiliation":"4  Institute of Biochemistry and Biophysics, Polish Academy of Sciences ,  Warsaw ,  Poland","@type":"Person"},{"name":"Ostrowski, Jerzy","affiliation":"1  Department of Gastroenterology ,  Maria Sklodowska-Curie Memorial Cancer Center and Institute of Oncology ,  Roentgena 5, 02-781 Warsaw ,  Poland    2  Department of Gastroenterology and Hepatology ,  Medical Center for Postgraduate Education ,  Warsaw ,  Poland","@type":"Person"}],"description":"Abstract.  Definitive identification of promoters, their cis-regulatory motifs, and their trans-acting proteins requires experimental analysis. To define the HN","pageStart":"245","pageEnd":"260","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/4/1/m_cover.gif?Expires=2147483647&Signature=gyespZcEz0u3yU5I1SxTvBT5xbOxPNOLYhwPsPR3aRNYFmHa5nka3C~O~XSM-xhnboR3fWGr2qZTxx4H6D5a1prxQNVRUky5teeRb6TGyzZOY1ts0mahn1IaVSV~iwF5Li2yIaOBrDGSrvkREQvnFxuFXLByOPU2xZqGFD3hkbuJfl8Zmq04LiLFB1KZ95YHjEAjg-AoOl5O~cQNH~modKmNtBOmwrss8u8onLmK7bX~3NRKpq-CeOZGFua0m5Jr7HSKoh2QGi3lGBNarjlvFa1Jq31MxWlp5EclYaSkUk41yA17dZtGSthlFGcVbtGK9W0l0-Af7ttPeQw2Y7YEBg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Comprehensive Analysis of the Palindromic Motif TCTCGCGAGA: A Regulatory Element of the  HNRNPK  Promoter","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/4/1/m_cover.gif?Expires=2147483647&Signature=gyespZcEz0u3yU5I1SxTvBT5xbOxPNOLYhwPsPR3aRNYFmHa5nka3C~O~XSM-xhnboR3fWGr2qZTxx4H6D5a1prxQNVRUky5teeRb6TGyzZOY1ts0mahn1IaVSV~iwF5Li2yIaOBrDGSrvkREQvnFxuFXLByOPU2xZqGFD3hkbuJfl8Zmq04LiLFB1KZ95YHjEAjg-AoOl5O~cQNH~modKmNtBOmwrss8u8onLmK7bX~3NRKpq-CeOZGFua0m5Jr7HSKoh2QGi3lGBNarjlvFa1Jq31MxWlp5EclYaSkUk41yA17dZtGSthlFGcVbtGK9W0l0-Af7ttPeQw2Y7YEBg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Comprehensive Analysis of the Palindromic Motif TCTCGCGAGA: A Regulatory Element of the  HNRNPK  Promoter" />
    <meta property="og:description" content="Abstract.  Definitive identification of promoters, their cis-regulatory motifs, and their trans-acting proteins requires experimental analysis. To define the HN" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/17/4/245/397387" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2010-06-29" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Mikula, Michal" /><meta name="citation_author_institution" content="Department of Gastroenterology, Maria Sklodowska-Curie Memorial Cancer Center and Institute of Oncology, Roentgena 5, 02-781 Warsaw, Poland" /><meta name="citation_author" content="Gaj, Pawel" /><meta name="citation_author_institution" content="Department of Gastroenterology and Hepatology, Medical Center for Postgraduate Education, Warsaw, Poland" /><meta name="citation_author" content="Dzwonek, Karolina" /><meta name="citation_author_institution" content="Department of Gastroenterology, Maria Sklodowska-Curie Memorial Cancer Center and Institute of Oncology, Roentgena 5, 02-781 Warsaw, Poland" /><meta name="citation_author" content="Rubel, Tymon" /><meta name="citation_author_institution" content="Laboratory of Bioinformatics and Systems Biology, Maria Sklodowska-Curie Memorial Cancer Center and Institute of Oncology, Roentgena 5, 02-781 Warsaw, Poland" /><meta name="citation_author" content="Karczmarski, Jakub" /><meta name="citation_author_institution" content="Department of Gastroenterology, Maria Sklodowska-Curie Memorial Cancer Center and Institute of Oncology, Roentgena 5, 02-781 Warsaw, Poland" /><meta name="citation_author" content="Paziewska, Agnieszka" /><meta name="citation_author_institution" content="Department of Gastroenterology and Hepatology, Medical Center for Postgraduate Education, Warsaw, Poland" /><meta name="citation_author" content="Dzwonek, Artur" /><meta name="citation_author_institution" content="Department of Gastroenterology, Maria Sklodowska-Curie Memorial Cancer Center and Institute of Oncology, Roentgena 5, 02-781 Warsaw, Poland" /><meta name="citation_author" content="Bragoszewski, Piotr" /><meta name="citation_author_institution" content="Department of Gastroenterology and Hepatology, Medical Center for Postgraduate Education, Warsaw, Poland" /><meta name="citation_author" content="Dadlez, Michal" /><meta name="citation_author_institution" content="Institute of Biochemistry and Biophysics, Polish Academy of Sciences, Warsaw, Poland" /><meta name="citation_author" content="Ostrowski, Jerzy" /><meta name="citation_author_institution" content="Department of Gastroenterology, Maria Sklodowska-Curie Memorial Cancer Center and Institute of Oncology, Roentgena 5, 02-781 Warsaw, Poland" /><meta name="citation_author_institution" content="Department of Gastroenterology and Hepatology, Medical Center for Postgraduate Education, Warsaw, Poland" /><meta name="citation_title" content="Comprehensive Analysis of the Palindromic Motif TCTCGCGAGA: A Regulatory Element of the HNRNPK Promoter" /><meta name="citation_firstpage" content="245" /><meta name="citation_lastpage" content="260" /><meta name="citation_doi" content="10.1093/dnares/dsq016" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="17" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2010/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=hnRNP K: one protein multiple processes; Bioessays;  citation_year=2004;  citation_volume=26;  citation_pages=629-38; " /><meta name="citation_reference" content="citation_title=ERK phosphorylation drives cytoplasmic accumulation of hnRNP-K and inhibition of mRNA translation; Nat. Cell biol.;  citation_year=2001;  citation_volume=3;  citation_pages=325-30; " /><meta name="citation_reference" content="citation_title=Heterogeneous nuclear ribonucleoprotein K is a transcription factor; Mol. Cell. Biol.;  citation_year=1996;  citation_volume=16;  citation_pages=2350-60; " /><meta name="citation_reference" content="citation_title=hnRNP K: an HDM2 target and transcriptional coactivator of p53 in response to DNA damage; Cell;  citation_year=2005;  citation_volume=123;  citation_pages=1065-78; " /><meta name="citation_reference" content="citation_title=Heterogeneous nuclear ribonucleoprotein K is over expressed, aberrantly localised and is associated with poor prognosis in colorectal cancer; Br. J. Cancer;  citation_year=2006;  citation_volume=95;  citation_pages=921-7; " /><meta name="citation_reference" content="citation_title=Heterogeneous ribonucleoprotein k and thymidine phosphorylase are independent prognostic and therapeutic markers for nasopharyngeal carcinoma; Clin. Cancer Res.;  citation_year=2008;  citation_volume=14;  citation_pages=3807-13; " /><meta name="citation_reference" content="citation_title=Protein clusters associated with carcinogenesis, histological differentiation and nodal metastasis in esophageal cancer; Proteomics;  citation_year=2006;  citation_volume=6;  citation_pages=6300-16; " /><meta name="citation_reference" content="citation_title=Altered patterns of expression of members of the heterogeneous nuclear ribonucleoprotein (hnRNP) family in lung cancer; Lung Cancer;  citation_year=2003;  citation_volume=41;  citation_pages=131-43; " /><meta name="citation_reference" content="citation_title=Evidence for heterogeneous nuclear ribonucleoprotein K overexpression in oral squamous cell carcinoma; citation_text=author reply 576.; Br. J. Cancer;  citation_year=2007;  citation_volume=97;  citation_pages=574-5; " /><meta name="citation_reference" content="citation_title=RNA and RNA binding proteins participate in early stages of cell spreading through spreading initiation centers; Cell;  citation_year=2004;  citation_volume=117;  citation_pages=649-62; " /><meta name="citation_reference" content="citation_title=Loss-of-function screening by randomized intracellular antibodies: identification of hnRNP-K as a potential target for metastasis; Proc. Natl. Acad. Sci. USA;  citation_year=2007;  citation_volume=104;  citation_pages=8983-8; " /><meta name="citation_reference" content="citation_title=A common cis-element in promoters of protein synthesis and cell cycle genes; Acta Biochim. Pol.;  citation_year=2007;  citation_volume=54;  citation_pages=89-98; " /><meta name="citation_reference" content="citation_title=A serine/threonine kinase activity is closely associated with a 65-kDa phosphoprotein specifically recognized by the kappa B enhancer element; J. Biol. Chem.;  citation_year=1991;  citation_volume=266;  citation_pages=12722-33; " /><meta name="citation_reference" content="citation_title=An integrated LC-ESI-MS platform for quantitation of serum peptide ladders. Application for colon carcinoma study; Proteomics Clin. Appl.;  citation_year=2009;  citation_volume=3;  citation_pages=932-46; " /><meta name="citation_reference" content="citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing; J. Roy. Stat. Soc. Ser. B;  citation_year=1995;  citation_volume=50;  citation_pages=289-300; " /><meta name="citation_reference" content="citation_title=Halogenated imidazole derivatives block RNA polymerase II elongation along mitogen inducible genes; BMC Mol. Biol.;  citation_year=2010;  citation_volume=11;  citation_pages=4" /><meta name="citation_reference" content="citation_title=Microplate-based chromatin immunoprecipitation method, Matrix ChIP: a platform to study signaling of complex genomic events; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=e17" /><meta name="citation_reference" content="citation_title=Treatment of mice with EGF and orthovanadate activates cytoplasmic and nuclear MAPK, p70S6k, and p90rsk in the liver; J. Hepatol.;  citation_year=2000;  citation_volume=32;  citation_pages=965-74; " /><meta name="citation_reference" content="citation_title=ENCODE whole-genome data in the UCSC Genome Browser; Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=D620-D625; " /><meta name="citation_reference" content="citation_title=Locating mammalian transcription factor binding sites: a survey of computational and experimental techniques; Genome Res.;  citation_year=2006;  citation_volume=16;  citation_pages=1455-64; " /><meta name="citation_reference" content="citation_title=A short ultraconserved sequence drives transcription from an alternate FBN1 promoter; Int. J. Biochem. Cell Biol.;  citation_year=2008;  citation_volume=40;  citation_pages=638-50; " /><meta name="citation_reference" content="citation_title=Molecular mechanisms of DNA double strand break repair; Trends Cell Biol.;  citation_year=1998;  citation_volume=8;  citation_pages=483-9; " /><meta name="citation_reference" content="citation_title=Methods for proteomic analysis of transcription factors; J. Chromatogr.;  citation_year=2009;  citation_volume=1216;  citation_pages=6881-9; " /><meta name="citation_reference" content="citation_title=Profiling the human protein–DNA interactome reveals ERK2 as a transcriptional repressor of interferon signaling; Cell;  citation_year=2009;  citation_volume=139;  citation_pages=610-22; " /><meta name="citation_reference" content="citation_title=hPDI: a database of experimental human protein–DNA interactions; Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=287-9; " /><meta name="citation_reference" content="citation_title=hDREF regulates cell proliferation and expression of ribosomal protein genes; Mol. Cell. Biol.;  citation_year=2007;  citation_volume=27;  citation_pages=2003-13; " /><meta name="citation_reference" content="citation_title=Chromatin modifications and their function; Cell;  citation_year=2007;  citation_volume=128;  citation_pages=693-705; " /><meta name="citation_reference" content="citation_title=The role of chromatin during transcription; Cell;  citation_year=2007;  citation_volume=128;  citation_pages=707-19; " /><meta name="citation_reference" content="citation_title=A glimpse into the epigenetic landscape of gene regulation; Curr. Opin. Genet. Dev.;  citation_year=2008;  citation_volume=18;  citation_pages=116-22; " /><meta name="citation_reference" content="citation_title=Transcriptional control by PARP-1: chromatin modulation, enhancer-binding, coregulation, and insulation; Curr. Opin. Cell Biol.;  citation_year=2008;  citation_volume=20;  citation_pages=294-302; " /><meta name="citation_reference" content="citation_title=Rapid, transcription-independent loss of nucleosomes over a large chromatin domain at Hsp70 loci; Cell;  citation_year=2008;  citation_volume=134;  citation_pages=74-84; " /><meta name="citation_reference" content="citation_title=Poly(ADP-ribose) polymerase 1 interacts with nuclear respiratory factor 1 (NRF-1) and plays a role in NRF-1 transcriptional regulation; J. Biol. Chem.;  citation_year=2009;  citation_volume=284;  citation_pages=8621-32; " /><meta name="citation_reference" content="citation_title=A topoisomerase IIbeta-mediated dsDNA break required for regulated transcription; Science;  citation_year=2006;  citation_volume=312;  citation_pages=1798-802; " /><meta name="citation_reference" content="citation_title=Dynamic binding of Ku80, Ku70 and NF90 to the IL-2 promoter in vivo in activated T-cells; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=2302-10; " /><meta name="citation_reference" content="citation_title=Specific binding of poly(ADP-ribose) polymerase-1 to cruciform hairpins; J. Mol. Biol.;  citation_year=2005;  citation_volume=348;  citation_pages=609-15; " /><meta name="citation_reference" content="citation_title=Transcriptional repression by binding of poly(ADP-ribose) polymerase to promoter sequences; J. Biol. Chem.;  citation_year=2002;  citation_volume=277;  citation_pages=665-70; " /><meta name="citation_reference" content="citation_title=NAD+-dependent modulation of chromatin structure and transcription by nucleosome binding properties of PARP-1; Cell;  citation_year=2004;  citation_volume=119;  citation_pages=803-14; " /><meta name="citation_reference" content="citation_title=The DNA binding and catalytic domains of poly(ADP-ribose) polymerase 1 cooperate in the regulation of chromatin structure and transcription; Mol. Cell. Biol.;  citation_year=2007;  citation_volume=27;  citation_pages=7475-85; " /><meta name="citation_reference" content="citation_title=Dynamic regulation of nucleosome positioning in the human genome; Cell;  citation_year=2008;  citation_volume=132;  citation_pages=887-98; " /><meta name="citation_reference" content="citation_title=Identifying positioned nucleosomes with epigenetic marks in human from ChIP-Seq; BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=537" /><meta name="citation_reference" content="citation_title=An exonuclease protection assay reveals heat-shock element and TATA box DNA-binding proteins in crude nuclear extracts; Nature;  citation_year=1985;  citation_volume=317;  citation_pages=84-7; " /><meta name="citation_reference" content="citation_title=Genome-wide mapping of in vivo protein–DNA interactions; Science;  citation_year=2007;  citation_volume=316;  citation_pages=1497-502; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of transcription factor binding sites based on ChIP-Seq data; Nat. Methods;  citation_year=2008;  citation_volume=5;  citation_pages=829-34; " /><meta name="citation_reference" content="citation_title=High-resolution mapping and characterization of open chromatin across the genome; Cell;  citation_year=2008;  citation_volume=132;  citation_pages=311-22; " /><meta name="citation_reference" content="citation_title=High-resolution profiling of histone methylations in the human genome; Cell;  citation_year=2007;  citation_volume=129;  citation_pages=823-37; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/17/4/245/1115269/dsq016.pdf" /><meta name="description" content="Abstract.  Definitive identification of promoters, their cis-regulatory motifs, and their trans-acting proteins requires experimental analysis. To define the HN" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/17/4/245/397387" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/17/4/245/397387" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/17/4/245/397387");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsq016';
    </script>
<title>
	Comprehensive Analysis of the Palindromic Motif TCTCGCGAGA: A Regulatory Element of the HNRNPK Promoter | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./397387" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bc1uPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNaWt1bGEsIE1pY2hhbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgR2FzdHJvZW50ZXJvbG9neSwgTWFyaWEgU2tsb2Rvd3NrYS1DdXJpZSBNZW1vcmlhbCBDYW5jZXIgQ2VudGVyIGFuZCBJbnN0aXR1dGUgb2YgT25jb2xvZ3ksIFJvZW50Z2VuYSA1LCAwMi03ODEgV2Fyc2F3LCBQb2xhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHYWosIFBhd2VsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBHYXN0cm9lbnRlcm9sb2d5IGFuZCBIZXBhdG9sb2d5LCBNZWRpY2FsIENlbnRlciBmb3IgUG9zdGdyYWR1YXRlIEVkdWNhdGlvbiwgV2Fyc2F3LCBQb2xhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJEendvbmVrLCBLYXJvbGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgR2FzdHJvZW50ZXJvbG9neSwgTWFyaWEgU2tsb2Rvd3NrYS1DdXJpZSBNZW1vcmlhbCBDYW5jZXIgQ2VudGVyIGFuZCBJbnN0aXR1dGUgb2YgT25jb2xvZ3ksIFJvZW50Z2VuYSA1LCAwMi03ODEgV2Fyc2F3LCBQb2xhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJSdWJlbCwgVHltb24iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IG9mIEJpb2luZm9ybWF0aWNzIGFuZCBTeXN0ZW1zIEJpb2xvZ3ksIE1hcmlhIFNrbG9kb3dza2EtQ3VyaWUgTWVtb3JpYWwgQ2FuY2VyIENlbnRlciBhbmQgSW5zdGl0dXRlIG9mIE9uY29sb2d5LCBSb2VudGdlbmEgNSwgMDItNzgxIFdhcnNhdywgUG9sYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2FyY3ptYXJza2ksIEpha3ViIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBHYXN0cm9lbnRlcm9sb2d5LCBNYXJpYSBTa2xvZG93c2thLUN1cmllIE1lbW9yaWFsIENhbmNlciBDZW50ZXIgYW5kIEluc3RpdHV0ZSBvZiBPbmNvbG9neSwgUm9lbnRnZW5hIDUsIDAyLTc4MSBXYXJzYXcsIFBvbGFuZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBhemlld3NrYSwgQWduaWVzemthIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBHYXN0cm9lbnRlcm9sb2d5IGFuZCBIZXBhdG9sb2d5LCBNZWRpY2FsIENlbnRlciBmb3IgUG9zdGdyYWR1YXRlIEVkdWNhdGlvbiwgV2Fyc2F3LCBQb2xhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJEendvbmVrLCBBcnR1ciIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgR2FzdHJvZW50ZXJvbG9neSwgTWFyaWEgU2tsb2Rvd3NrYS1DdXJpZSBNZW1vcmlhbCBDYW5jZXIgQ2VudGVyIGFuZCBJbnN0aXR1dGUgb2YgT25jb2xvZ3ksIFJvZW50Z2VuYSA1LCAwMi03ODEgV2Fyc2F3LCBQb2xhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCcmFnb3N6ZXdza2ksIFBpb3RyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBHYXN0cm9lbnRlcm9sb2d5IGFuZCBIZXBhdG9sb2d5LCBNZWRpY2FsIENlbnRlciBmb3IgUG9zdGdyYWR1YXRlIEVkdWNhdGlvbiwgV2Fyc2F3LCBQb2xhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJEYWRsZXosIE1pY2hhbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBCaW9jaGVtaXN0cnkgYW5kIEJpb3BoeXNpY3MsIFBvbGlzaCBBY2FkZW15IG9mIFNjaWVuY2VzLCBXYXJzYXcsIFBvbGFuZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9zdHJvd3NraSwgSmVyenkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEdhc3Ryb2VudGVyb2xvZ3ksIE1hcmlhIFNrbG9kb3dza2EtQ3VyaWUgTWVtb3JpYWwgQ2FuY2VyIENlbnRlciBhbmQgSW5zdGl0dXRlIG9mIE9uY29sb2d5LCBSb2VudGdlbmEgNSwgMDItNzgxIFdhcnNhdywgUG9sYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBHYXN0cm9lbnRlcm9sb2d5IGFuZCBIZXBhdG9sb2d5LCBNZWRpY2FsIENlbnRlciBmb3IgUG9zdGdyYWR1YXRlIEVkdWNhdGlvbiwgV2Fyc2F3LCBQb2xhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkNvbXByZWhlbnNpdmUgQW5hbHlzaXMgb2YgdGhlIFBhbGluZHJvbWljIE1vdGlmIFRDVENHQ0dBR0E6IEEgUmVndWxhdG9yeSBFbGVtZW50IG9mIHRoZSBITlJOUEsgUHJvbW90ZXIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyNDUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjI2MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzcTAxNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEwLzA4LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9aG5STlAgSzogb25lIHByb3RlaW4gbXVsdGlwbGUgcHJvY2Vzc2VzOyBCaW9lc3NheXM7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz02MjktMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVSSyBwaG9zcGhvcnlsYXRpb24gZHJpdmVzIGN5dG9wbGFzbWljIGFjY3VtdWxhdGlvbiBvZiBoblJOUC1LIGFuZCBpbmhpYml0aW9uIG9mIG1STkEgdHJhbnNsYXRpb247IE5hdC4gQ2VsbCBiaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MzI1LTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IZXRlcm9nZW5lb3VzIG51Y2xlYXIgcmlib251Y2xlb3Byb3RlaW4gSyBpcyBhIHRyYW5zY3JpcHRpb24gZmFjdG9yOyBNb2wuIENlbGwuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MjM1MC02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9aG5STlAgSzogYW4gSERNMiB0YXJnZXQgYW5kIHRyYW5zY3JpcHRpb25hbCBjb2FjdGl2YXRvciBvZiBwNTMgaW4gcmVzcG9uc2UgdG8gRE5BIGRhbWFnZTsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjM7ICBjaXRhdGlvbl9wYWdlcz0xMDY1LTc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IZXRlcm9nZW5lb3VzIG51Y2xlYXIgcmlib251Y2xlb3Byb3RlaW4gSyBpcyBvdmVyIGV4cHJlc3NlZCwgYWJlcnJhbnRseSBsb2NhbGlzZWQgYW5kIGlzIGFzc29jaWF0ZWQgd2l0aCBwb29yIHByb2dub3NpcyBpbiBjb2xvcmVjdGFsIGNhbmNlcjsgQnIuIEouIENhbmNlcjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT05NTsgIGNpdGF0aW9uX3BhZ2VzPTkyMS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IZXRlcm9nZW5lb3VzIHJpYm9udWNsZW9wcm90ZWluIGsgYW5kIHRoeW1pZGluZSBwaG9zcGhvcnlsYXNlIGFyZSBpbmRlcGVuZGVudCBwcm9nbm9zdGljIGFuZCB0aGVyYXBldXRpYyBtYXJrZXJzIGZvciBuYXNvcGhhcnluZ2VhbCBjYXJjaW5vbWE7IENsaW4uIENhbmNlciBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MzgwNy0xMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvdGVpbiBjbHVzdGVycyBhc3NvY2lhdGVkIHdpdGggY2FyY2lub2dlbmVzaXMsIGhpc3RvbG9naWNhbCBkaWZmZXJlbnRpYXRpb24gYW5kIG5vZGFsIG1ldGFzdGFzaXMgaW4gZXNvcGhhZ2VhbCBjYW5jZXI7IFByb3Rlb21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTYzMDAtMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsdGVyZWQgcGF0dGVybnMgb2YgZXhwcmVzc2lvbiBvZiBtZW1iZXJzIG9mIHRoZSBoZXRlcm9nZW5lb3VzIG51Y2xlYXIgcmlib251Y2xlb3Byb3RlaW4gKGhuUk5QKSBmYW1pbHkgaW4gbHVuZyBjYW5jZXI7IEx1bmcgQ2FuY2VyOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9MTMxLTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmlkZW5jZSBmb3IgaGV0ZXJvZ2VuZW91cyBudWNsZWFyIHJpYm9udWNsZW9wcm90ZWluIEsgb3ZlcmV4cHJlc3Npb24gaW4gb3JhbCBzcXVhbW91cyBjZWxsIGNhcmNpbm9tYTsgY2l0YXRpb25fdGV4dD1hdXRob3IgcmVwbHkgNTc2LjsgQnIuIEouIENhbmNlcjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT05NzsgIGNpdGF0aW9uX3BhZ2VzPTU3NC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1STkEgYW5kIFJOQSBiaW5kaW5nIHByb3RlaW5zIHBhcnRpY2lwYXRlIGluIGVhcmx5IHN0YWdlcyBvZiBjZWxsIHNwcmVhZGluZyB0aHJvdWdoIHNwcmVhZGluZyBpbml0aWF0aW9uIGNlbnRlcnM7IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTE3OyAgY2l0YXRpb25fcGFnZXM9NjQ5LTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Mb3NzLW9mLWZ1bmN0aW9uIHNjcmVlbmluZyBieSByYW5kb21pemVkIGludHJhY2VsbHVsYXIgYW50aWJvZGllczogaWRlbnRpZmljYXRpb24gb2YgaG5STlAtSyBhcyBhIHBvdGVudGlhbCB0YXJnZXQgZm9yIG1ldGFzdGFzaXM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTEwNDsgIGNpdGF0aW9uX3BhZ2VzPTg5ODMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb21tb24gY2lzLWVsZW1lbnQgaW4gcHJvbW90ZXJzIG9mIHByb3RlaW4gc3ludGhlc2lzIGFuZCBjZWxsIGN5Y2xlIGdlbmVzOyBBY3RhIEJpb2NoaW0uIFBvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz04OS05ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzZXJpbmUvdGhyZW9uaW5lIGtpbmFzZSBhY3Rpdml0eSBpcyBjbG9zZWx5IGFzc29jaWF0ZWQgd2l0aCBhIDY1LWtEYSBwaG9zcGhvcHJvdGVpbiBzcGVjaWZpY2FsbHkgcmVjb2duaXplZCBieSB0aGUga2FwcGEgQiBlbmhhbmNlciBlbGVtZW50OyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT0yNjY7ICBjaXRhdGlvbl9wYWdlcz0xMjcyMi0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gaW50ZWdyYXRlZCBMQy1FU0ktTVMgcGxhdGZvcm0gZm9yIHF1YW50aXRhdGlvbiBvZiBzZXJ1bSBwZXB0aWRlIGxhZGRlcnMuIEFwcGxpY2F0aW9uIGZvciBjb2xvbiBjYXJjaW5vbWEgc3R1ZHk7IFByb3Rlb21pY3MgQ2xpbi4gQXBwbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTkzMi00NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29udHJvbGxpbmcgdGhlIGZhbHNlIGRpc2NvdmVyeSByYXRlOiBhIHByYWN0aWNhbCBhbmQgcG93ZXJmdWwgYXBwcm9hY2ggdG8gbXVsdGlwbGUgdGVzdGluZzsgSi4gUm95LiBTdGF0LiBTb2MuIFNlci4gQjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT01MDsgIGNpdGF0aW9uX3BhZ2VzPTI4OS0zMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhhbG9nZW5hdGVkIGltaWRhem9sZSBkZXJpdmF0aXZlcyBibG9jayBSTkEgcG9seW1lcmFzZSBJSSBlbG9uZ2F0aW9uIGFsb25nIG1pdG9nZW4gaW5kdWNpYmxlIGdlbmVzOyBCTUMgTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb3BsYXRlLWJhc2VkIGNocm9tYXRpbiBpbW11bm9wcmVjaXBpdGF0aW9uIG1ldGhvZCwgTWF0cml4IENoSVA6IGEgcGxhdGZvcm0gdG8gc3R1ZHkgc2lnbmFsaW5nIG9mIGNvbXBsZXggZ2Vub21pYyBldmVudHM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPWUxNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyZWF0bWVudCBvZiBtaWNlIHdpdGggRUdGIGFuZCBvcnRob3ZhbmFkYXRlIGFjdGl2YXRlcyBjeXRvcGxhc21pYyBhbmQgbnVjbGVhciBNQVBLLCBwNzBTNmssIGFuZCBwOTByc2sgaW4gdGhlIGxpdmVyOyBKLiBIZXBhdG9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTk2NS03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RU5DT0RFIHdob2xlLWdlbm9tZSBkYXRhIGluIHRoZSBVQ1NDIEdlbm9tZSBCcm93c2VyOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz1ENjIwLUQ2MjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvY2F0aW5nIG1hbW1hbGlhbiB0cmFuc2NyaXB0aW9uIGZhY3RvciBiaW5kaW5nIHNpdGVzOiBhIHN1cnZleSBvZiBjb21wdXRhdGlvbmFsIGFuZCBleHBlcmltZW50YWwgdGVjaG5pcXVlczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xNDU1LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNob3J0IHVsdHJhY29uc2VydmVkIHNlcXVlbmNlIGRyaXZlcyB0cmFuc2NyaXB0aW9uIGZyb20gYW4gYWx0ZXJuYXRlIEZCTjEgcHJvbW90ZXI7IEludC4gSi4gQmlvY2hlbS4gQ2VsbCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPTYzOC01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIG1lY2hhbmlzbXMgb2YgRE5BIGRvdWJsZSBzdHJhbmQgYnJlYWsgcmVwYWlyOyBUcmVuZHMgQ2VsbCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9NDgzLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ldGhvZHMgZm9yIHByb3Rlb21pYyBhbmFseXNpcyBvZiB0cmFuc2NyaXB0aW9uIGZhY3RvcnM7IEouIENocm9tYXRvZ3IuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEyMTY7ICBjaXRhdGlvbl9wYWdlcz02ODgxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb2ZpbGluZyB0aGUgaHVtYW4gcHJvdGVpbuKAk0ROQSBpbnRlcmFjdG9tZSByZXZlYWxzIEVSSzIgYXMgYSB0cmFuc2NyaXB0aW9uYWwgcmVwcmVzc29yIG9mIGludGVyZmVyb24gc2lnbmFsaW5nOyBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEzOTsgIGNpdGF0aW9uX3BhZ2VzPTYxMC0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9aFBESTogYSBkYXRhYmFzZSBvZiBleHBlcmltZW50YWwgaHVtYW4gcHJvdGVpbuKAk0ROQSBpbnRlcmFjdGlvbnM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9Mjg3LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPWhEUkVGIHJlZ3VsYXRlcyBjZWxsIHByb2xpZmVyYXRpb24gYW5kIGV4cHJlc3Npb24gb2Ygcmlib3NvbWFsIHByb3RlaW4gZ2VuZXM7IE1vbC4gQ2VsbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0yMDAzLTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaHJvbWF0aW4gbW9kaWZpY2F0aW9ucyBhbmQgdGhlaXIgZnVuY3Rpb247IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTI4OyAgY2l0YXRpb25fcGFnZXM9NjkzLTcwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJvbGUgb2YgY2hyb21hdGluIGR1cmluZyB0cmFuc2NyaXB0aW9uOyBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTEyODsgIGNpdGF0aW9uX3BhZ2VzPTcwNy0xOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBnbGltcHNlIGludG8gdGhlIGVwaWdlbmV0aWMgbGFuZHNjYXBlIG9mIGdlbmUgcmVndWxhdGlvbjsgQ3Vyci4gT3Bpbi4gR2VuZXQuIERldi47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0xMTYtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb25hbCBjb250cm9sIGJ5IFBBUlAtMTogY2hyb21hdGluIG1vZHVsYXRpb24sIGVuaGFuY2VyLWJpbmRpbmcsIGNvcmVndWxhdGlvbiwgYW5kIGluc3VsYXRpb247IEN1cnIuIE9waW4uIENlbGwgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0yOTQtMzAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXBpZCwgdHJhbnNjcmlwdGlvbi1pbmRlcGVuZGVudCBsb3NzIG9mIG51Y2xlb3NvbWVzIG92ZXIgYSBsYXJnZSBjaHJvbWF0aW4gZG9tYWluIGF0IEhzcDcwIGxvY2k7IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTM0OyAgY2l0YXRpb25fcGFnZXM9NzQtODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvbHkoQURQLXJpYm9zZSkgcG9seW1lcmFzZSAxIGludGVyYWN0cyB3aXRoIG51Y2xlYXIgcmVzcGlyYXRvcnkgZmFjdG9yIDEgKE5SRi0xKSBhbmQgcGxheXMgYSByb2xlIGluIE5SRi0xIHRyYW5zY3JpcHRpb25hbCByZWd1bGF0aW9uOyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yODQ7ICBjaXRhdGlvbl9wYWdlcz04NjIxLTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHRvcG9pc29tZXJhc2UgSUliZXRhLW1lZGlhdGVkIGRzRE5BIGJyZWFrIHJlcXVpcmVkIGZvciByZWd1bGF0ZWQgdHJhbnNjcmlwdGlvbjsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zMTI7ICBjaXRhdGlvbl9wYWdlcz0xNzk4LTgwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RHluYW1pYyBiaW5kaW5nIG9mIEt1ODAsIEt1NzAgYW5kIE5GOTAgdG8gdGhlIElMLTIgcHJvbW90ZXIgaW4gdml2byBpbiBhY3RpdmF0ZWQgVC1jZWxsczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9MjMwMi0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3BlY2lmaWMgYmluZGluZyBvZiBwb2x5KEFEUC1yaWJvc2UpIHBvbHltZXJhc2UtMSB0byBjcnVjaWZvcm0gaGFpcnBpbnM7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzQ4OyAgY2l0YXRpb25fcGFnZXM9NjA5LTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uYWwgcmVwcmVzc2lvbiBieSBiaW5kaW5nIG9mIHBvbHkoQURQLXJpYm9zZSkgcG9seW1lcmFzZSB0byBwcm9tb3RlciBzZXF1ZW5jZXM7IEouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTI3NzsgIGNpdGF0aW9uX3BhZ2VzPTY2NS03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TkFEKy1kZXBlbmRlbnQgbW9kdWxhdGlvbiBvZiBjaHJvbWF0aW4gc3RydWN0dXJlIGFuZCB0cmFuc2NyaXB0aW9uIGJ5IG51Y2xlb3NvbWUgYmluZGluZyBwcm9wZXJ0aWVzIG9mIFBBUlAtMTsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTk7ICBjaXRhdGlvbl9wYWdlcz04MDMtMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBETkEgYmluZGluZyBhbmQgY2F0YWx5dGljIGRvbWFpbnMgb2YgcG9seShBRFAtcmlib3NlKSBwb2x5bWVyYXNlIDEgY29vcGVyYXRlIGluIHRoZSByZWd1bGF0aW9uIG9mIGNocm9tYXRpbiBzdHJ1Y3R1cmUgYW5kIHRyYW5zY3JpcHRpb247IE1vbC4gQ2VsbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz03NDc1LTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EeW5hbWljIHJlZ3VsYXRpb24gb2YgbnVjbGVvc29tZSBwb3NpdGlvbmluZyBpbiB0aGUgaHVtYW4gZ2Vub21lOyBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTEzMjsgIGNpdGF0aW9uX3BhZ2VzPTg4Ny05ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZnlpbmcgcG9zaXRpb25lZCBudWNsZW9zb21lcyB3aXRoIGVwaWdlbmV0aWMgbWFya3MgaW4gaHVtYW4gZnJvbSBDaElQLVNlcTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz01MzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBleG9udWNsZWFzZSBwcm90ZWN0aW9uIGFzc2F5IHJldmVhbHMgaGVhdC1zaG9jayBlbGVtZW50IGFuZCBUQVRBIGJveCBETkEtYmluZGluZyBwcm90ZWlucyBpbiBjcnVkZSBudWNsZWFyIGV4dHJhY3RzOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTE5ODU7ICBjaXRhdGlvbl92b2x1bWU9MzE3OyAgY2l0YXRpb25fcGFnZXM9ODQtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgbWFwcGluZyBvZiBpbiB2aXZvIHByb3RlaW7igJNETkEgaW50ZXJhY3Rpb25zOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTMxNjsgIGNpdGF0aW9uX3BhZ2VzPTE0OTctNTAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyBvZiB0cmFuc2NyaXB0aW9uIGZhY3RvciBiaW5kaW5nIHNpdGVzIGJhc2VkIG9uIENoSVAtU2VxIGRhdGE7IE5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ODI5LTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLXJlc29sdXRpb24gbWFwcGluZyBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBvcGVuIGNocm9tYXRpbiBhY3Jvc3MgdGhlIGdlbm9tZTsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMzI7ICBjaXRhdGlvbl9wYWdlcz0zMTEtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2gtcmVzb2x1dGlvbiBwcm9maWxpbmcgb2YgaGlzdG9uZSBtZXRoeWxhdGlvbnMgaW4gdGhlIGh1bWFuIGdlbm9tZTsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMjk7ICBjaXRhdGlvbl9wYWdlcz04MjMtMzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xNy80LzI0NS8xMTE1MjY5L2RzcTAxNi5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgRGVmaW5pdGl2ZSBpZGVudGlmaWNhdGlvbiBvZiBwcm9tb3RlcnMsIHRoZWlyIGNpcy1yZWd1bGF0b3J5IG1vdGlmcywgYW5kIHRoZWlyIHRyYW5zLWFjdGluZyBwcm90ZWlucyByZXF1aXJlcyBleHBlcmltZW50YWwgYW5hbHlzaXMuIFRvIGRlZmluZSB0aGUgSE4iIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM5NzM4NyZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGS4r12CYtAxBGF9TfyKb3jSdTMOHA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f4%2f245%2f397387" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f4%2f245%2f397387" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=15449" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/17/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/4/1/m_cover.gif?Expires=2147483647&amp;Signature=gyespZcEz0u3yU5I1SxTvBT5xbOxPNOLYhwPsPR3aRNYFmHa5nka3C~O~XSM-xhnboR3fWGr2qZTxx4H6D5a1prxQNVRUky5teeRb6TGyzZOY1ts0mahn1IaVSV~iwF5Li2yIaOBrDGSrvkREQvnFxuFXLByOPU2xZqGFD3hkbuJfl8Zmq04LiLFB1KZ95YHjEAjg-AoOl5O~cQNH~modKmNtBOmwrss8u8onLmK7bX~3NRKpq-CeOZGFua0m5Jr7HSKoh2QGi3lGBNarjlvFa1Jq31MxWlp5EclYaSkUk41yA17dZtGSthlFGcVbtGK9W0l0-Af7ttPeQw2Y7YEBg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/17/4">
                        <div class="volume trailing-comma">Volume 17</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2010        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5869548">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5869548">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5869554">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5869554">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5869563">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5869563">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5869635">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5869635">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5869736">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5869736">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5869740">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5869740">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5869742">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5869742">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5869746">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5869746">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/17/4/233/395899">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Comprehensive Analysis of the Palindromic Motif TCTCGCGAGA: A Regulatory Element of the <em>HNRNPK</em> Promoter
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Michal Mikula</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Gastroenterology</div>, <div class="insititution">Maria Sklodowska-Curie Memorial Cancer Center and Institute of Oncology</div>, <div class="addr-line">Roentgena 5, 02-781 Warsaw</div>, <div class="country">Poland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Michal+Mikula">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mikula M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mikula M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Michal Mikula</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Pawel Gaj</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Gastroenterology and Hepatology</div>, <div class="insititution">Medical Center for Postgraduate Education</div>, <div class="addr-line">Warsaw</div>, <div class="country">Poland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Pawel+Gaj">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gaj P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gaj P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Pawel Gaj</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Karolina Dzwonek</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Gastroenterology</div>, <div class="insititution">Maria Sklodowska-Curie Memorial Cancer Center and Institute of Oncology</div>, <div class="addr-line">Roentgena 5, 02-781 Warsaw</div>, <div class="country">Poland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Karolina+Dzwonek">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dzwonek K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Dzwonek K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Karolina Dzwonek</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tymon Rubel</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Laboratory of Bioinformatics and Systems Biology</div>, <div class="insititution">Maria Sklodowska-Curie Memorial Cancer Center and Institute of Oncology</div>, <div class="addr-line">Roentgena 5, 02-781 Warsaw</div>, <div class="country">Poland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tymon+Rubel">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Rubel T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Rubel T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tymon Rubel</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jakub Karczmarski</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Gastroenterology</div>, <div class="insititution">Maria Sklodowska-Curie Memorial Cancer Center and Institute of Oncology</div>, <div class="addr-line">Roentgena 5, 02-781 Warsaw</div>, <div class="country">Poland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jakub+Karczmarski">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Karczmarski J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Karczmarski J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jakub Karczmarski</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Agnieszka Paziewska</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Gastroenterology and Hepatology</div>, <div class="insititution">Medical Center for Postgraduate Education</div>, <div class="addr-line">Warsaw</div>, <div class="country">Poland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Agnieszka+Paziewska">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Paziewska A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Paziewska A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Agnieszka Paziewska</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Artur Dzwonek</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Gastroenterology</div>, <div class="insititution">Maria Sklodowska-Curie Memorial Cancer Center and Institute of Oncology</div>, <div class="addr-line">Roentgena 5, 02-781 Warsaw</div>, <div class="country">Poland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Artur+Dzwonek">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dzwonek A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Dzwonek A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Artur Dzwonek</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Piotr Bragoszewski</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Gastroenterology and Hepatology</div>, <div class="insititution">Medical Center for Postgraduate Education</div>, <div class="addr-line">Warsaw</div>, <div class="country">Poland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Piotr+Bragoszewski">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bragoszewski P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bragoszewski P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Piotr Bragoszewski</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Michal Dadlez</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">Institute of Biochemistry and Biophysics, Polish Academy of Sciences</div>, <div class="addr-line">Warsaw</div>, <div class="country">Poland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Michal+Dadlez">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dadlez M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Dadlez M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Michal Dadlez</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jerzy Ostrowski</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Gastroenterology</div>, <div class="insititution">Maria Sklodowska-Curie Memorial Cancer Center and Institute of Oncology</div>, <div class="addr-line">Roentgena 5, 02-781 Warsaw</div>, <div class="country">Poland</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Gastroenterology and Hepatology</div>, <div class="insititution">Medical Center for Postgraduate Education</div>, <div class="addr-line">Warsaw</div>, <div class="country">Poland</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +48 22-5462575. Fax. <div class="fax">+48 22-6440209</div>. E-mail: <a href="mailto:jostrow@warman.com.pl" target="_blank">jostrow@warman.com.pl</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jerzy+Ostrowski">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ostrowski J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ostrowski J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jerzy Ostrowski</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 17, Issue 4, 1 August 2010, Pages 245–260, <a href='https://doi.org/10.1093/dnares/dsq016'>https://doi.org/10.1093/dnares/dsq016</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">29 June 2010</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">26 March 2010</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">03 June 2010</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="397387" href="/dnaresearch/article-pdf/17/4/245/1115269/dsq016.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Michal Mikula, Pawel Gaj, Karolina Dzwonek, Tymon Rubel, Jakub Karczmarski, Agnieszka Paziewska, Artur Dzwonek, Piotr Bragoszewski, Michal Dadlez, Jerzy Ostrowski;  Comprehensive Analysis of the Palindromic Motif TCTCGCGAGA: A Regulatory Element of the <em>HNRNPK</em> Promoter, <em>DNA Research</em>, Volume 17, Issue 4, 1 August 2010, Pages 245–260, <a href='https://doi.org/10.1093/dnares/dsq016'>https://doi.org/10.1093/dnares/dsq016</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/397387?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/397387?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/397387?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/397387?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/397387?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/397387?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/397387?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Comprehensive%20Analysis%20of%20the%20Palindromic%20Motif%20TCTCGCGAGA%3A%20A%20Regulatory%20Element%20of%20the%20%26lt%3Bem%26gt%3BHNRNPK%26lt%3B%2Fem%26gt%3B%20Promoter&amp;publicationDate=2010-06-29&amp;volumeNum=17&amp;issueNum=4&amp;author=Mikula%2C%20Michal%3B%20Gaj%2C%20Pawel&amp;startPage=245&amp;endPage=260&amp;contentId=10.1093%2Fdnares%2Fdsq016&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f4%2f245%2f397387" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5869548 id="5869548" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Definitive identification of promoters, their cis-regulatory motifs, and their trans-acting proteins requires experimental analysis. To define the <em>HNRNPK</em> promoter and its cognate DNA–protein interactions, we performed a comprehensive study combining experimental approaches, including luciferase reporter gene assays, chromatin immunoprecipitations (ChIP), electrophoretic mobility shift assays (EMSA), and mass spectrometry (MS). We discovered that out of the four potential <em>HNRNPK</em> promoters tested, the one containing the palindromic motif TCTCGCGAGA exhibited the highest activity in a reporter system assay. Although further EMSA and MS analyses, performed to uncover the identity of the palindrome-binding transcription factor, did identify a complex of DNA-binding proteins, neither method unambiguously identified the pertinent direct trans-acting protein(s). ChIP revealed similar chromatin states at the promoters with the palindromic motif and at housekeeping gene promoters. A ChIP survey showed significantly higher recruitment of PARP1, a protein identified by MS as ubiquitously attached to DNA probes, within heterochromatin sites. Computational analyses indicated that this palindrome displays features that mark nucleosome boundaries, causing the surrounding DNA landscape to be constitutively open. Our strategy of diverse approaches facilitated the direct characterization of various molecular properties of <em>HNRNPK</em> promoter bearing the palindromic motif TCTCGCGAGA, despite the obstacles that accompany <em>in vitro</em> methods.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;HNRNPK promoter&quot;">HNRNPK promoter</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;palindromic motif&quot;">palindromic motif</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="EMSA">EMSA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;mass spectrometry&quot;">mass spectrometry</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;chromatin immunoprecipitation&quot;">chromatin immunoprecipitation</a></div>                            <h2 scrollto-destination=5869554 id="5869554" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>HNRNPK is an abundant protein factor found in the nucleus, cytoplasm, mitochondria, and plasma membrane that belongs to the family of heterogeneous nuclear ribonucleoproteins (hnRNPs). These hnRNPs are involved in a variety of biological processes, including telomere biogenesis, cellular signalling, DNA repair, and the regulation of expression on both the transcriptional and translational levels.<sup><a reveal-id="DSQ016C1" data-open="DSQ016C1" class="link link-ref link-reveal xref-bibr">1</a></sup> HNRNPK has been found to activate and repress gene expression, and its activity is mainly regulated by covalent modifications.<sup><a reveal-id="DSQ016C2  DSQ016C4" data-open="DSQ016C2  DSQ016C4" class="link link-ref link-reveal xref-bibr">2–4</a></sup> Through its binding to single-stranded (ss) and double-stranded (ds) DNA, HNRNPK regulates gene expression in both CT element-dependent and -independent fashions.</p><p>Several studies have demonstrated the aberrant increase in HNRNPK expression in cancers.<sup><a reveal-id="DSQ016C5  DSQ016C9" data-open="DSQ016C5  DSQ016C9" class="link link-ref link-reveal xref-bibr">5–9</a></sup> Other studies have shown that HNRNPK is a functional constituent of a cellular structure, the spreading initiation centre,<sup><a reveal-id="DSQ016C10" data-open="DSQ016C10" class="link link-ref link-reveal xref-bibr">10</a></sup> and that it is indispensable for cellular migration.<sup><a reveal-id="DSQ016C11" data-open="DSQ016C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Overall, HNRNPK has been implicated as a potential key player in carcinogenesis, making it an attractive target for anticancer therapies. Although HNRNPK has been the subject of numerous studies, the mechanisms regulating its expression are still largely unknown.</p><p>The palindromic motif TCTCGCGAGA has been found in TATA-less promoters, and is a potential cis-regulatory element in ∼5% of human genes including cell cycle, transcription regulators, chromatin structure modulators, translation initiation factors, and ribosomal protein genes.<sup><a reveal-id="DSQ016C12" data-open="DSQ016C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Here, we present experimental and computational evidence that the TCTCGCGAGA motif represents a critical element in the regulation of <em>HNRNPK</em> expression.</p>                            <h2 scrollto-destination=5869563 id="5869563" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5869565 id="5869565" class="section-title" data-legacy-id=s2a>2.1. Reporter plasmid construction</h3>
<p>Sequences obtained from annotated databases (GenBank and dbEST, available at the National Center for Biotechnology Information; <a class="link link-uri" href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</a>) were analysed by bioinformatic tools, leading to the identification of four potential <em>HNRNPK</em> promoters (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Promoter 1 contains the palindromic motif TCTCGCGAGA.</p><p>All analysed <em>HNRNPK</em> promoter sequences were amplified from human genomic DNA using Pfu Turbo DNA polymerase (Agilent Technologies, Inc., Santa Clara, CA, USA) and primers, as listed in Table <a reveal-id="DSQ016TB1" data-open="DSQ016TB1" class="link link-reveal link-table xref-fig">1</a>; graphical location of analysed fragments is shown in Fig. <a reveal-id="DSQ016F1" data-open="DSQ016F1" class="link link-reveal link-table xref-fig">1</a>A. For promoter fragments 1, 2, and 3, nested polymerase chain reactions (PCRs) were used, with pre-amplification of long fragments with the Promoter 1 forward and Promoter 3 reverse primers pair. For Fragment 4, a standard PCR reaction was used. The PCR-amplified DNA fragments were T/A subcloned into the pCR 2.1-TOPO vector using the TOPO TA Cloning kit (Invitrogen, Carlsbad, CA, USA), and the resulting vectors were transformed into chemically competent TOP10 cells. Plasmid DNA was isolated from randomly selected bacterial clones and sequenced using the ABI Prism 377 automated DNA sequencing system (Applied Biosystems, Foster City, CA, USA). Plasmids were digested with <em>Kpn</em>I and <em>EcoR</em>V (Promoters: 1, 2, 3) or with <em>Sac</em>I and <em>EcoR</em>V (Promoter 4), separated by agarose gel electrophoresis, and DNA fragments isolated from the gel were inserted into the corresponding restriction sites of the pGL4.10[<em>luc2</em>] luciferase reporter vector (Promega Corporation, Madison, WI, USA). Site-directed mutagenesis was performed using the QuikChange Multi Site-Directed Mutagenesis kit (Agilent Technologies, Inc.) in the context of pGL4.10[<em>luc2</em>]-promoter-1, according to the manufacturer's protocol. All final DNA constructs were confirmed by sequencing. </p>                        <a id="5869572" scrollto-destination="5869572"></a>
<div content-id="DSQ016TB1" class="table-modal"><div class="table-wrap"><div id="DSQ016TB1" class="table-wrap-title" data-id="DSQ016TB1"><div class="label -label">Table 1</div><div class="caption"><p>Oligonucleotide sequences used in this study</p></div> </div><div class="table-overflow"><table><tbody><tr><td colspan="3">Primer pairs used for amplification of promoter fragments </td></tr><tr><td> Promoter </td><td>Forward primer 5′–&gt;3′ </td><td>Reverse primer 5′–&gt;3′ </td></tr><tr><td> Promoter 1 </td><td>CTCCAACCCCAAAGGACTC* </td><td>ATAATGGCGTCTGCAGTGCT </td></tr><tr><td> Promoter 2 </td><td>GTTGGGGCTCCCTGAAAG </td><td>CGGTCGGAGCTAGACAGAAA </td></tr><tr><td> Promoter 3 </td><td>CCAGTTTTTCCGCTTGAGG </td><td>CCACTGCCCGAGACAAAG* </td></tr><tr><td> Promoter 4 </td><td>TTGGTGGTGTGGGAATTTTT </td><td>TGGCATTGTACTGCATTGGT </td></tr><tr><td colspan="3">Primer pairs used in ChIP </td></tr><tr><td> Gene name </td><td>Forward primer 5′–&gt;3′ </td><td>Reverse primer 5′–&gt;3′ </td></tr><tr><td> Intergenic </td><td>GCCACATTCAGTTCTCGCTTT </td><td>TTCTCCTTTCCCTGGCAACA </td></tr><tr><td> RHO_intron2 </td><td>TGGGTGGTGTCATCTGGTAA </td><td>GGATGGAATGGATCAGATGG </td></tr><tr><td> RHO_exon5 </td><td>GGACATCCACCAAGACCTACTG </td><td>ATATGTTCTCCCTTCCCATTCA </td></tr><tr><td> HBB </td><td>TTATGCTGGTCCTGTCCTCC </td><td>CTGAAAGAGATGCGGTGG </td></tr><tr><td> HNRNPK </td><td>TCGTTCGCGCACTCACTA </td><td>CGCGTGAACTAGGCAGTTGT </td></tr><tr><td> TUBB6 </td><td>AAGGGGCAGCGAAACGGTGG </td><td>GGAGCCGGGCGGAAAAGGAC </td></tr><tr><td> GAPDH </td><td>TGAGCAGTCCGGTGTCACTA </td><td>AGGACTTTGGGAACGACTGA </td></tr><tr><td> HNRNPH </td><td>ACAGCTGCGCAACCCACCAC </td><td>CGCGAGACGGGAAAGTCGGT </td></tr><tr><td> RPS7 </td><td>CCGCCTCTTGCCTTCGGACG </td><td>CCGAGCGCCGGCTTAGGAAG </td></tr><tr><td> HSP70 </td><td>GGCGAAACCCCTGGAATATTCCCGA </td><td>AGCCTTGGGACAACGGGAG </td></tr><tr><td> EGR-1 </td><td>GGTCCTGCCATATTAGGGCTT </td><td>TGGGATCTCTCGCGACTCC </td></tr><tr><td colspan="3">Primers used in mutagenesis </td></tr><tr><td> Primer name </td><td colspan="2">Primer sequence </td></tr><tr><td> P1M1 </td><td colspan="2">5′-CAGTTGTTAGA<span class="underline">TCTCG<strong>T</strong>GAGA</span>GGTTCGCCCCCTAG-3′ </td></tr><tr><td> P1M2 </td><td colspan="2">5′-CAGTTGTTAGA<span class="underline">TCTCG<strong>TA</strong>AGA</span>GGTTCGCCCCCTAG-3′ </td></tr><tr><td> P1M3 </td><td colspan="2">5′-CAGTTGTTAGA<span class="underline">TCTCG<strong>TAG</strong>GA</span>GGTTCGCCCCCTAG-3′ </td></tr><tr><td colspan="3">Probes used in EMSA and MS </td></tr><tr><td> P1-30-F </td><td colspan="2">5′-AGTTGTTCTA<span class="underline">TCTCGCGAGA</span>GGTTCGCCCC-3′ (+/− biotin) </td></tr><tr><td> P1-30-R </td><td colspan="2">5′-GGGGCGAACC<span class="underline">TCTCGCGAGA</span>TAGAACAACT-3′ </td></tr><tr><td> P1M-30-F </td><td colspan="2">5′-AGTTGTTCTA<span class="underline">TCTCG<strong>TAG</strong>GA</span>GGTTCGCCCC-3′ </td></tr><tr><td> P1M-30-R </td><td colspan="2">5′-GGGGCGAACC<span class="underline">TC<strong>CTA</strong>CGAGA</span>TAGAACAACT-3′ </td></tr><tr><td> P1-16-F </td><td colspan="2">5′-CTA<span class="underline">TCTCGCGAGA</span>GGT-3′ (+/− biotin) </td></tr><tr><td> P1-16-R </td><td colspan="2">5′-ACC<span class="underline">TCTCGCGAGA</span>TAG-3′ </td></tr><tr><td> P1M-16-F </td><td colspan="2">5′-CTA<span class="underline">TCTCG<strong>TAG</strong>GA</span>GG-3′ </td></tr><tr><td> P1M-16-R </td><td colspan="2">5′-ACC<span class="underline">TC<strong>CTA</strong>CGAGA</span>TAG-3′ </td></tr></tbody></table></div><div content-id="DSQ016TB1" class="table-modal"><table><tbody><tr><td colspan="3">Primer pairs used for amplification of promoter fragments </td></tr><tr><td> Promoter </td><td>Forward primer 5′–&gt;3′ </td><td>Reverse primer 5′–&gt;3′ </td></tr><tr><td> Promoter 1 </td><td>CTCCAACCCCAAAGGACTC* </td><td>ATAATGGCGTCTGCAGTGCT </td></tr><tr><td> Promoter 2 </td><td>GTTGGGGCTCCCTGAAAG </td><td>CGGTCGGAGCTAGACAGAAA </td></tr><tr><td> Promoter 3 </td><td>CCAGTTTTTCCGCTTGAGG </td><td>CCACTGCCCGAGACAAAG* </td></tr><tr><td> Promoter 4 </td><td>TTGGTGGTGTGGGAATTTTT </td><td>TGGCATTGTACTGCATTGGT </td></tr><tr><td colspan="3">Primer pairs used in ChIP </td></tr><tr><td> Gene name </td><td>Forward primer 5′–&gt;3′ </td><td>Reverse primer 5′–&gt;3′ </td></tr><tr><td> Intergenic </td><td>GCCACATTCAGTTCTCGCTTT </td><td>TTCTCCTTTCCCTGGCAACA </td></tr><tr><td> RHO_intron2 </td><td>TGGGTGGTGTCATCTGGTAA </td><td>GGATGGAATGGATCAGATGG </td></tr><tr><td> RHO_exon5 </td><td>GGACATCCACCAAGACCTACTG </td><td>ATATGTTCTCCCTTCCCATTCA </td></tr><tr><td> HBB </td><td>TTATGCTGGTCCTGTCCTCC </td><td>CTGAAAGAGATGCGGTGG </td></tr><tr><td> HNRNPK </td><td>TCGTTCGCGCACTCACTA </td><td>CGCGTGAACTAGGCAGTTGT </td></tr><tr><td> TUBB6 </td><td>AAGGGGCAGCGAAACGGTGG </td><td>GGAGCCGGGCGGAAAAGGAC </td></tr><tr><td> GAPDH </td><td>TGAGCAGTCCGGTGTCACTA </td><td>AGGACTTTGGGAACGACTGA </td></tr><tr><td> HNRNPH </td><td>ACAGCTGCGCAACCCACCAC </td><td>CGCGAGACGGGAAAGTCGGT </td></tr><tr><td> RPS7 </td><td>CCGCCTCTTGCCTTCGGACG </td><td>CCGAGCGCCGGCTTAGGAAG </td></tr><tr><td> HSP70 </td><td>GGCGAAACCCCTGGAATATTCCCGA </td><td>AGCCTTGGGACAACGGGAG </td></tr><tr><td> EGR-1 </td><td>GGTCCTGCCATATTAGGGCTT </td><td>TGGGATCTCTCGCGACTCC </td></tr><tr><td colspan="3">Primers used in mutagenesis </td></tr><tr><td> Primer name </td><td colspan="2">Primer sequence </td></tr><tr><td> P1M1 </td><td colspan="2">5′-CAGTTGTTAGA<span class="underline">TCTCG<strong>T</strong>GAGA</span>GGTTCGCCCCCTAG-3′ </td></tr><tr><td> P1M2 </td><td colspan="2">5′-CAGTTGTTAGA<span class="underline">TCTCG<strong>TA</strong>AGA</span>GGTTCGCCCCCTAG-3′ </td></tr><tr><td> P1M3 </td><td colspan="2">5′-CAGTTGTTAGA<span class="underline">TCTCG<strong>TAG</strong>GA</span>GGTTCGCCCCCTAG-3′ </td></tr><tr><td colspan="3">Probes used in EMSA and MS </td></tr><tr><td> P1-30-F </td><td colspan="2">5′-AGTTGTTCTA<span class="underline">TCTCGCGAGA</span>GGTTCGCCCC-3′ (+/− biotin) </td></tr><tr><td> P1-30-R </td><td colspan="2">5′-GGGGCGAACC<span class="underline">TCTCGCGAGA</span>TAGAACAACT-3′ </td></tr><tr><td> P1M-30-F </td><td colspan="2">5′-AGTTGTTCTA<span class="underline">TCTCG<strong>TAG</strong>GA</span>GGTTCGCCCC-3′ </td></tr><tr><td> P1M-30-R </td><td colspan="2">5′-GGGGCGAACC<span class="underline">TC<strong>CTA</strong>CGAGA</span>TAGAACAACT-3′ </td></tr><tr><td> P1-16-F </td><td colspan="2">5′-CTA<span class="underline">TCTCGCGAGA</span>GGT-3′ (+/− biotin) </td></tr><tr><td> P1-16-R </td><td colspan="2">5′-ACC<span class="underline">TCTCGCGAGA</span>TAG-3′ </td></tr><tr><td> P1M-16-F </td><td colspan="2">5′-CTA<span class="underline">TCTCG<strong>TAG</strong>GA</span>GG-3′ </td></tr><tr><td> P1M-16-R </td><td colspan="2">5′-ACC<span class="underline">TC<strong>CTA</strong>CGAGA</span>TAG-3′ </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>*Primers used for pre-amplification in nested PCR.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5869572" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ016TB1" class="table-wrap-title" data-id="DSQ016TB1"><div class="label -label">Table 1</div><div class="caption"><p>Oligonucleotide sequences used in this study</p></div> </div><div class="table-overflow"><table><tbody><tr><td colspan="3">Primer pairs used for amplification of promoter fragments </td></tr><tr><td> Promoter </td><td>Forward primer 5′–&gt;3′ </td><td>Reverse primer 5′–&gt;3′ </td></tr><tr><td> Promoter 1 </td><td>CTCCAACCCCAAAGGACTC* </td><td>ATAATGGCGTCTGCAGTGCT </td></tr><tr><td> Promoter 2 </td><td>GTTGGGGCTCCCTGAAAG </td><td>CGGTCGGAGCTAGACAGAAA </td></tr><tr><td> Promoter 3 </td><td>CCAGTTTTTCCGCTTGAGG </td><td>CCACTGCCCGAGACAAAG* </td></tr><tr><td> Promoter 4 </td><td>TTGGTGGTGTGGGAATTTTT </td><td>TGGCATTGTACTGCATTGGT </td></tr><tr><td colspan="3">Primer pairs used in ChIP </td></tr><tr><td> Gene name </td><td>Forward primer 5′–&gt;3′ </td><td>Reverse primer 5′–&gt;3′ </td></tr><tr><td> Intergenic </td><td>GCCACATTCAGTTCTCGCTTT </td><td>TTCTCCTTTCCCTGGCAACA </td></tr><tr><td> RHO_intron2 </td><td>TGGGTGGTGTCATCTGGTAA </td><td>GGATGGAATGGATCAGATGG </td></tr><tr><td> RHO_exon5 </td><td>GGACATCCACCAAGACCTACTG </td><td>ATATGTTCTCCCTTCCCATTCA </td></tr><tr><td> HBB </td><td>TTATGCTGGTCCTGTCCTCC </td><td>CTGAAAGAGATGCGGTGG </td></tr><tr><td> HNRNPK </td><td>TCGTTCGCGCACTCACTA </td><td>CGCGTGAACTAGGCAGTTGT </td></tr><tr><td> TUBB6 </td><td>AAGGGGCAGCGAAACGGTGG </td><td>GGAGCCGGGCGGAAAAGGAC </td></tr><tr><td> GAPDH </td><td>TGAGCAGTCCGGTGTCACTA </td><td>AGGACTTTGGGAACGACTGA </td></tr><tr><td> HNRNPH </td><td>ACAGCTGCGCAACCCACCAC </td><td>CGCGAGACGGGAAAGTCGGT </td></tr><tr><td> RPS7 </td><td>CCGCCTCTTGCCTTCGGACG </td><td>CCGAGCGCCGGCTTAGGAAG </td></tr><tr><td> HSP70 </td><td>GGCGAAACCCCTGGAATATTCCCGA </td><td>AGCCTTGGGACAACGGGAG </td></tr><tr><td> EGR-1 </td><td>GGTCCTGCCATATTAGGGCTT </td><td>TGGGATCTCTCGCGACTCC </td></tr><tr><td colspan="3">Primers used in mutagenesis </td></tr><tr><td> Primer name </td><td colspan="2">Primer sequence </td></tr><tr><td> P1M1 </td><td colspan="2">5′-CAGTTGTTAGA<span class="underline">TCTCG<strong>T</strong>GAGA</span>GGTTCGCCCCCTAG-3′ </td></tr><tr><td> P1M2 </td><td colspan="2">5′-CAGTTGTTAGA<span class="underline">TCTCG<strong>TA</strong>AGA</span>GGTTCGCCCCCTAG-3′ </td></tr><tr><td> P1M3 </td><td colspan="2">5′-CAGTTGTTAGA<span class="underline">TCTCG<strong>TAG</strong>GA</span>GGTTCGCCCCCTAG-3′ </td></tr><tr><td colspan="3">Probes used in EMSA and MS </td></tr><tr><td> P1-30-F </td><td colspan="2">5′-AGTTGTTCTA<span class="underline">TCTCGCGAGA</span>GGTTCGCCCC-3′ (+/− biotin) </td></tr><tr><td> P1-30-R </td><td colspan="2">5′-GGGGCGAACC<span class="underline">TCTCGCGAGA</span>TAGAACAACT-3′ </td></tr><tr><td> P1M-30-F </td><td colspan="2">5′-AGTTGTTCTA<span class="underline">TCTCG<strong>TAG</strong>GA</span>GGTTCGCCCC-3′ </td></tr><tr><td> P1M-30-R </td><td colspan="2">5′-GGGGCGAACC<span class="underline">TC<strong>CTA</strong>CGAGA</span>TAGAACAACT-3′ </td></tr><tr><td> P1-16-F </td><td colspan="2">5′-CTA<span class="underline">TCTCGCGAGA</span>GGT-3′ (+/− biotin) </td></tr><tr><td> P1-16-R </td><td colspan="2">5′-ACC<span class="underline">TCTCGCGAGA</span>TAG-3′ </td></tr><tr><td> P1M-16-F </td><td colspan="2">5′-CTA<span class="underline">TCTCG<strong>TAG</strong>GA</span>GG-3′ </td></tr><tr><td> P1M-16-R </td><td colspan="2">5′-ACC<span class="underline">TC<strong>CTA</strong>CGAGA</span>TAG-3′ </td></tr></tbody></table></div><div content-id="DSQ016TB1" class="table-modal"><table><tbody><tr><td colspan="3">Primer pairs used for amplification of promoter fragments </td></tr><tr><td> Promoter </td><td>Forward primer 5′–&gt;3′ </td><td>Reverse primer 5′–&gt;3′ </td></tr><tr><td> Promoter 1 </td><td>CTCCAACCCCAAAGGACTC* </td><td>ATAATGGCGTCTGCAGTGCT </td></tr><tr><td> Promoter 2 </td><td>GTTGGGGCTCCCTGAAAG </td><td>CGGTCGGAGCTAGACAGAAA </td></tr><tr><td> Promoter 3 </td><td>CCAGTTTTTCCGCTTGAGG </td><td>CCACTGCCCGAGACAAAG* </td></tr><tr><td> Promoter 4 </td><td>TTGGTGGTGTGGGAATTTTT </td><td>TGGCATTGTACTGCATTGGT </td></tr><tr><td colspan="3">Primer pairs used in ChIP </td></tr><tr><td> Gene name </td><td>Forward primer 5′–&gt;3′ </td><td>Reverse primer 5′–&gt;3′ </td></tr><tr><td> Intergenic </td><td>GCCACATTCAGTTCTCGCTTT </td><td>TTCTCCTTTCCCTGGCAACA </td></tr><tr><td> RHO_intron2 </td><td>TGGGTGGTGTCATCTGGTAA </td><td>GGATGGAATGGATCAGATGG </td></tr><tr><td> RHO_exon5 </td><td>GGACATCCACCAAGACCTACTG </td><td>ATATGTTCTCCCTTCCCATTCA </td></tr><tr><td> HBB </td><td>TTATGCTGGTCCTGTCCTCC </td><td>CTGAAAGAGATGCGGTGG </td></tr><tr><td> HNRNPK </td><td>TCGTTCGCGCACTCACTA </td><td>CGCGTGAACTAGGCAGTTGT </td></tr><tr><td> TUBB6 </td><td>AAGGGGCAGCGAAACGGTGG </td><td>GGAGCCGGGCGGAAAAGGAC </td></tr><tr><td> GAPDH </td><td>TGAGCAGTCCGGTGTCACTA </td><td>AGGACTTTGGGAACGACTGA </td></tr><tr><td> HNRNPH </td><td>ACAGCTGCGCAACCCACCAC </td><td>CGCGAGACGGGAAAGTCGGT </td></tr><tr><td> RPS7 </td><td>CCGCCTCTTGCCTTCGGACG </td><td>CCGAGCGCCGGCTTAGGAAG </td></tr><tr><td> HSP70 </td><td>GGCGAAACCCCTGGAATATTCCCGA </td><td>AGCCTTGGGACAACGGGAG </td></tr><tr><td> EGR-1 </td><td>GGTCCTGCCATATTAGGGCTT </td><td>TGGGATCTCTCGCGACTCC </td></tr><tr><td colspan="3">Primers used in mutagenesis </td></tr><tr><td> Primer name </td><td colspan="2">Primer sequence </td></tr><tr><td> P1M1 </td><td colspan="2">5′-CAGTTGTTAGA<span class="underline">TCTCG<strong>T</strong>GAGA</span>GGTTCGCCCCCTAG-3′ </td></tr><tr><td> P1M2 </td><td colspan="2">5′-CAGTTGTTAGA<span class="underline">TCTCG<strong>TA</strong>AGA</span>GGTTCGCCCCCTAG-3′ </td></tr><tr><td> P1M3 </td><td colspan="2">5′-CAGTTGTTAGA<span class="underline">TCTCG<strong>TAG</strong>GA</span>GGTTCGCCCCCTAG-3′ </td></tr><tr><td colspan="3">Probes used in EMSA and MS </td></tr><tr><td> P1-30-F </td><td colspan="2">5′-AGTTGTTCTA<span class="underline">TCTCGCGAGA</span>GGTTCGCCCC-3′ (+/− biotin) </td></tr><tr><td> P1-30-R </td><td colspan="2">5′-GGGGCGAACC<span class="underline">TCTCGCGAGA</span>TAGAACAACT-3′ </td></tr><tr><td> P1M-30-F </td><td colspan="2">5′-AGTTGTTCTA<span class="underline">TCTCG<strong>TAG</strong>GA</span>GGTTCGCCCC-3′ </td></tr><tr><td> P1M-30-R </td><td colspan="2">5′-GGGGCGAACC<span class="underline">TC<strong>CTA</strong>CGAGA</span>TAGAACAACT-3′ </td></tr><tr><td> P1-16-F </td><td colspan="2">5′-CTA<span class="underline">TCTCGCGAGA</span>GGT-3′ (+/− biotin) </td></tr><tr><td> P1-16-R </td><td colspan="2">5′-ACC<span class="underline">TCTCGCGAGA</span>TAG-3′ </td></tr><tr><td> P1M-16-F </td><td colspan="2">5′-CTA<span class="underline">TCTCG<strong>TAG</strong>GA</span>GG-3′ </td></tr><tr><td> P1M-16-R </td><td colspan="2">5′-ACC<span class="underline">TC<strong>CTA</strong>CGAGA</span>TAG-3′ </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>*Primers used for pre-amplification in nested PCR.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5869572" target="_blank">View Large</a></div></div>                        <a id="5869574" scrollto-destination="5869574"></a>
<div id="DSQ016F1" data-id="DSQ016F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/m_dsq01601.jpeg?Expires=2147483647&Signature=Aukx0px9oHs9UAqyQMbhcC7hAQFRfSD4M8MbV3nE6rxv6K36qfXu4ZhhwXbq3Su7don60rznTiHvcij96ovbhtKT0Sjp4uPlo3v2kYYbgyNoYQ3uBvnYcNmsf6GaQLV3Nje0-zfLymwzzbT8cOjFM0QtC9OKhNv-rIPzVjsjSTglhmhRra9bOE03dUNhjbPC72YUHlsRSK~sCEcv57XP2Rubjusz6rtD2o8hUVHaeU0IdIVQJ3NxmUR2G8hrnCT8twnaMw9z9i4dH7t6kRaOEwHATksMcwIpzUMHhj~XBfkmVaFV6XoAJWJHViZA4~8sLd5twzUPInPL7-rDnN09Mg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Graphical localization of HNRNPK promoters and their activity in HeLa cells. (A) Information from the annotated sequence databases GenBank and dbEST was analysed by computational tools, permitting the identification of four potential HNRNPK promoters. Localization of potential promoters sequences (vertical bars) are superimposed on three HNRNPK mRNA transcripts derived from the UCSC browser (hg18 genome assembly) http://genome.ucsc.edu (chr9:85772500-85786000). (B) HeLa cells were transiently transfected with constructs of four human HNRNPK promoters (P1, Promoter 1, containing the TCTCGCGAGA motif; P2, Promoter 2; P3, Promoter 3; P4, Promoter 4) fused to a luciferase reporter gene (pGL4.10) and cotransfected with the phrl-CMV plasmid encoding the Renilla luciferase gene as an internal control. (C) HeLa cells were transfected with mutated forms of the HNRNPK promoter 1 (P1M1, single mutation; P1M2, double mutation; P1M3, triple mutation; pGL4.10, null plasmid). Luciferase activities were measured 24 h after transfection. Mean firefly luciferase activities were normalized to Renilla luciferase activities measured for different promoter reporter plasmids. Data are expressed as a percentage of promoter fragment 1 activity, and represent the mean ± standard deviation of three independent experiments." path-from-xml="dsq01601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5869574/dsq01601.jpeg" path-from-xml="dsq01601.jpeg" target="_blank">View large</a><a section="5869574" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/dsq01601.jpeg?Expires=2147483647&Signature=DlAYnNxaNuxZ~Z2jLz7-Rz1LpzkKpRxW50ZtAexwh7U0NUZnuTAJBO0Xs3H2W9hFI7fk~VbfNHUypMgpG1Wo5iBN4ISY3vr1Yt-MNj7XJ17Tdofq-ymOMmwpp9eRNqmDD9Cxu3xlPbA~xjVYl9jmLB3eYpyrnkS5aY0wPdYzuTq6XojnKG8VdOk17sveaWqISzHXbVw5lxGXCiJWCWUgE6knQ1TcUVXkKXq8gVLF0YPkrML2foq4TUYs1Wo6Bu4fVP0z6ZMqCyFgBrDDZs2G17eR-eYPw6z2kF4Clq4~rJL2ehehGKdgb-UZHWoFiIi7SIFBrlpFr2FrlX0gYTcebA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5869574&ar=397387&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Graphical localization of <em>HNRNPK</em> promoters and their activity in HeLa cells. (A) Information from the annotated sequence databases GenBank and dbEST was analysed by computational tools, permitting the identification of four potential <em>HNRNPK</em> promoters. Localization of potential promoters sequences (vertical bars) are superimposed on three <em>HNRNPK</em> mRNA transcripts derived from the UCSC browser (hg18 genome assembly) <a class="link link-uri" href="http://genome.ucsc.edu">http://genome.ucsc.edu</a> (chr9:85772500-85786000). (B) HeLa cells were transiently transfected with constructs of four human <em>HNRNPK</em> promoters (P1, Promoter 1, containing the TCTCGCGAGA motif; P2, Promoter 2; P3, Promoter 3; P4, Promoter 4) fused to a luciferase reporter gene (pGL4.10) and cotransfected with the phrl-CMV plasmid encoding the <em>Renilla</em> luciferase gene as an internal control. (C) HeLa cells were transfected with mutated forms of the <em>HNRNPK</em> promoter 1 (P1M1, single mutation; P1M2, double mutation; P1M3, triple mutation; pGL4.10, null plasmid). Luciferase activities were measured 24 h after transfection. Mean firefly luciferase activities were normalized to <em>Renilla</em> luciferase activities measured for different promoter reporter plasmids. Data are expressed as a percentage of promoter fragment 1 activity, and represent the mean ± standard deviation of three independent experiments.</p></div></div><div content-id="DSQ016F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/m_dsq01601.jpeg?Expires=2147483647&Signature=Aukx0px9oHs9UAqyQMbhcC7hAQFRfSD4M8MbV3nE6rxv6K36qfXu4ZhhwXbq3Su7don60rznTiHvcij96ovbhtKT0Sjp4uPlo3v2kYYbgyNoYQ3uBvnYcNmsf6GaQLV3Nje0-zfLymwzzbT8cOjFM0QtC9OKhNv-rIPzVjsjSTglhmhRra9bOE03dUNhjbPC72YUHlsRSK~sCEcv57XP2Rubjusz6rtD2o8hUVHaeU0IdIVQJ3NxmUR2G8hrnCT8twnaMw9z9i4dH7t6kRaOEwHATksMcwIpzUMHhj~XBfkmVaFV6XoAJWJHViZA4~8sLd5twzUPInPL7-rDnN09Mg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Graphical localization of HNRNPK promoters and their activity in HeLa cells. (A) Information from the annotated sequence databases GenBank and dbEST was analysed by computational tools, permitting the identification of four potential HNRNPK promoters. Localization of potential promoters sequences (vertical bars) are superimposed on three HNRNPK mRNA transcripts derived from the UCSC browser (hg18 genome assembly) http://genome.ucsc.edu (chr9:85772500-85786000). (B) HeLa cells were transiently transfected with constructs of four human HNRNPK promoters (P1, Promoter 1, containing the TCTCGCGAGA motif; P2, Promoter 2; P3, Promoter 3; P4, Promoter 4) fused to a luciferase reporter gene (pGL4.10) and cotransfected with the phrl-CMV plasmid encoding the Renilla luciferase gene as an internal control. (C) HeLa cells were transfected with mutated forms of the HNRNPK promoter 1 (P1M1, single mutation; P1M2, double mutation; P1M3, triple mutation; pGL4.10, null plasmid). Luciferase activities were measured 24 h after transfection. Mean firefly luciferase activities were normalized to Renilla luciferase activities measured for different promoter reporter plasmids. Data are expressed as a percentage of promoter fragment 1 activity, and represent the mean ± standard deviation of three independent experiments." path-from-xml="dsq01601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5869574/dsq01601.jpeg" path-from-xml="dsq01601.jpeg" target="_blank">View large</a><a section="5869574" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/dsq01601.jpeg?Expires=2147483647&Signature=DlAYnNxaNuxZ~Z2jLz7-Rz1LpzkKpRxW50ZtAexwh7U0NUZnuTAJBO0Xs3H2W9hFI7fk~VbfNHUypMgpG1Wo5iBN4ISY3vr1Yt-MNj7XJ17Tdofq-ymOMmwpp9eRNqmDD9Cxu3xlPbA~xjVYl9jmLB3eYpyrnkS5aY0wPdYzuTq6XojnKG8VdOk17sveaWqISzHXbVw5lxGXCiJWCWUgE6knQ1TcUVXkKXq8gVLF0YPkrML2foq4TUYs1Wo6Bu4fVP0z6ZMqCyFgBrDDZs2G17eR-eYPw6z2kF4Clq4~rJL2ehehGKdgb-UZHWoFiIi7SIFBrlpFr2FrlX0gYTcebA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5869574&ar=397387&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Graphical localization of <em>HNRNPK</em> promoters and their activity in HeLa cells. (A) Information from the annotated sequence databases GenBank and dbEST was analysed by computational tools, permitting the identification of four potential <em>HNRNPK</em> promoters. Localization of potential promoters sequences (vertical bars) are superimposed on three <em>HNRNPK</em> mRNA transcripts derived from the UCSC browser (hg18 genome assembly) <a class="link link-uri" href="http://genome.ucsc.edu">http://genome.ucsc.edu</a> (chr9:85772500-85786000). (B) HeLa cells were transiently transfected with constructs of four human <em>HNRNPK</em> promoters (P1, Promoter 1, containing the TCTCGCGAGA motif; P2, Promoter 2; P3, Promoter 3; P4, Promoter 4) fused to a luciferase reporter gene (pGL4.10) and cotransfected with the phrl-CMV plasmid encoding the <em>Renilla</em> luciferase gene as an internal control. (C) HeLa cells were transfected with mutated forms of the <em>HNRNPK</em> promoter 1 (P1M1, single mutation; P1M2, double mutation; P1M3, triple mutation; pGL4.10, null plasmid). Luciferase activities were measured 24 h after transfection. Mean firefly luciferase activities were normalized to <em>Renilla</em> luciferase activities measured for different promoter reporter plasmids. Data are expressed as a percentage of promoter fragment 1 activity, and represent the mean ± standard deviation of three independent experiments.</p></div></div>                            <h3 scrollto-destination=5869578 id="5869578" class="section-title" data-legacy-id=s2b>2.2. Transient transfection of HeLa cells</h3>
<p>HeLa cells were grown at 37°C in 6% CO<sub>2</sub> humidified atmosphere in DME medium supplemented with 10% foetal bovine serum (FBS), 2 mM glutamine, 100 units/ml penicillin, and 0.01% streptomycin in plastic cell culture flasks. Cells were routinely subcultured using a trypsin solution. Prior to transfection, cells were plated in ViewPlate-96 White plates (PerkinElmer Wellesley, MA, USA) at 90% confluency and cultured overnight. Each well was transfected for 4 h using Lipofectamine 2000 Reagent (Invitrogen), with 170 ng of the pGL4.10-promoter construct or the empty pGL4.10 vector encoding the firefly luciferase gene, and co-transfected with 30 ng of the phrl-CMV plasmid (Promega) encoding the <em>Renilla</em> luciferase gene. To counterbalance minute differences in construct length and to retain constant particle numbers, the quantity of plasmid DNA was adjusted relative to construct size. The quantity of phrl-CMV plasmid DNA remained unchanged. Lipofectamine–DNA complexes were prepared according to the manufacturer's instructions, except that MEM was used instead of Opti-MEM medium. To maximize transfection efficiency during the 4 h incubation, the serum complement was reduced to 0.5%. Afterward the medium was changed to MEM containing 4% serum and all standard complements. Twenty-four hours following transfection, cells were harvested and assayed for luciferase activity in a Victor 2 luminometer (PerkinElmer) using the Dual-Glow Luciferase Assay System reagents (Promega) in accordance with the manufacturer's instructions. To normalize non-specific variations in transfection efficiency and cell number, all promoter activities were expressed as the ratio of firefly luciferase to <em>Renilla</em> luciferase luminescence in each well. Three independent transfection experiments were conducted with six replicates of each plasmid construct in each experiment. All values are presented as mean ± standard deviation.</p>                            <h3 scrollto-destination=5869583 id="5869583" class="section-title" data-legacy-id=s2c>2.3. Analyses of protein–DNA interactions</h3>
<p>Non-histone nuclear protein extracts (NEs) were isolated as previously reported.<sup><a reveal-id="DSQ016C13" data-open="DSQ016C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Protein Desalting Spin Columns (Pierce, Thermo Fisher Scientific, Waltham, MA, USA) were used for buffer exchange of protein samples. Two methods were applied for analysing interactions between the palindromic motif and DNA-binding proteins: electrophoretic mobility shift assay (EMSA) and mass spectrometry (MS). The sequences of oligonucleotides used in the study are shown in Table <a reveal-id="DSQ016TB1" data-open="DSQ016TB1" class="link link-reveal link-table xref-fig">1</a>.</p>                            <h4 scrollto-destination=5869588 id="5869588" class="section-title" data-legacy-id=s2c1>2.3.1. Ku80 immunodepletion</h4>
<p>NEs were immunodepleted of Ku80 protein using monoclonal anti-Ku80 antibody (Abcam, Cambridge, UK; ab3107) and Immunoprecipitation Kit—Dynabeads<sup>®</sup> Protein G (Invitrogen) according to the manufacturer's protocol.</p>                            <h4 scrollto-destination=5869592 id="5869592" class="section-title" data-legacy-id=s2c2>2.3.2. Electrophoretic mobility shift assay</h4>
<p>EMSA were performed as described previously.<sup><a reveal-id="DSQ016C12" data-open="DSQ016C12" class="link link-ref link-reveal xref-bibr">12</a></sup> DNA probes were phosphorylated with [γ-<sup>32</sup>P]ATP using T4 polynucleotide kinase (Fermentas International Inc., Burlington, Canada) according to the manufacturer's protocol. The reaction was carried out in a total volume of 15 µl (30 min at 25°C) containing 2.5 µg of NE proteins and 1.4 pmol of oligonucleotide in 100 mM NaCl-binding buffer (BB) (100 mM NaCl, 1 mM EDTA, 0.5 mM DTT, 10 mM Tris/HCl, pH 7.5). EMSA was performed using 4 µl of the product on an 8% non-denaturing polyacrylamide gel (37.5:1, Promega) for 1 h at 7.5 V/cm. Autoradiograms were obtained using Imaging screen K and Molecular Imager FX (BioRad, Hercules, CA, USA).</p>                            <h4 scrollto-destination=5869596 id="5869596" class="section-title" data-legacy-id=s2c3>2.3.3. Ms</h4>
<p>Fifty micrograms of NE diluted in 50 mM NaCl-BB [50 mM NaCl, 1 mM EDTA, 0.05% NP-40, 5 mM Tris–HCl, pH 7.5, containing protease inhibitors (Roche, Basel, Switzerland)] were incubated at 4°C for 60 min with rotation, in the presence of 200 pM biotinylated wild-type DNA probe with or without a 10-fold excess of competitor (non-biotinylated DNA probe containing the mutated motif). To precipitate the DNA–protein complexes, 1 mg of Dynabeads<sup>®</sup> M-280 Streptavidin (Invitrogen) was added and the mixture was incubated with rotation at 4°C for 15 min. After magnetic separation, the beads were extensively washed with 150 mM NaCl-BB, and the bound proteins were eluted from the beads with 300 mM NaCl, 500 mM NaHCO<sub>3</sub>, 1 mM EDTA, 5 mM Tris–HCl, pH 7.5. Eluted proteins were subjected to reduction, alkylation, tryptic digestion, and MS identification.</p>                            <h4 scrollto-destination=5869599 id="5869599" class="section-title" data-legacy-id=s2c4>2.3.4. LC-MS settings</h4>
<p>LC-MS analyses were carried out using the nano-Acquity (Waters Corporation, Milford, MA, USA) LC system coupled to an LTQ FTICR (Thermo Fisher Scientific, Waltham, MA, USA) mass spectrometer. Spectrometer parameters were as follows: capillary voltage—2.5 kV, cone—40 V, N<sub>2</sub> gas flow—0, and <em>m/z</em> range—300–2000. The spectrometer was calibrated on a weekly basis with Calmix (Thermo Fisher Scientific). Samples were loaded from the autosampler tray (cooled to 10°C) to the precolumn [Symmetry C18, 180 µm × 20 mm, 5 µm (Waters)] using a mobile phase of 100% MilliQ water acidified with 0.1% formic acid. Peptides were transferred to the nano-UPLC column [BEH130 C18, 75 µm × 250 mm, 1.7 µm (Waters)] by a gradient of 5–30% ACN in 0.1% formic acid in 45 min (250 nl/min), then directly eluted to the ion source of the mass spectrometer. Each LC run was preceded by a blank run to ensure lack of carry-over of the material from the previous run. For qualitative analyses (peptide and protein identification), the spectrometer was run in data-dependent MS to MS/MS switch mode, and up to ten MS/MS processes were allowed for each MS scan. Quantitative analyses were carried out in separate profile type survey scan LC-MS runs using the same ACN gradient.</p>                            <h4 scrollto-destination=5869604 id="5869604" class="section-title" data-legacy-id=s2c5>2.3.5. Qualitative analysis (LC-MS/MS)</h4>
<p>LTQ-FT MS raw data files were processed to peak lists with the Mascot Distiller software (version 2.2.1, Matrix Science, London, UK). The pre-processed parent and daughter ion lists were used to search the Swissprot (<a class="link link-uri" href="http://www.expasy.org/sprot/">http://www.expasy.org/sprot/</a>) protein database, with taxonomy restriction to human (20 332 sequences, 11 229 110 residues). The Mascot search engine (version 2.2.03, Matrix Science, London, UK) was used to search the database with the following parameters: enzyme specificity—semi-trypsin, permitted number of missed cleavages—1, fixed modification—carbamidomethylation (C), variable modifications—oxidation (M), phospho (ST), phospho (Y), and carbamidomethyl (K), protein mass—unrestricted. The peptide and fragment mass tolerance settings were established separately for individual experiments after measured mass recalibration (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Method 1</a>).</p><p>Statistical assessment of peptide assignments was based on the joined target/decoy database search strategy. This procedure (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Method 2</a>) provided <em>q</em>-value estimates for each peptide spectrum match (PSM) in the dataset. All PSMs with <em>q</em>-values &gt;0.001 were removed from further analysis. A protein was regarded as confidently identified if at least two peptides of this protein were found. Proteins identified by a subset of peptides from another protein were excluded from analysis. Proteins matching exactly the same set of peptides were joined into a single group.</p>                            <h4 scrollto-destination=5869611 id="5869611" class="section-title" data-legacy-id=s2c6>2.3.6. Quantitative feature extraction</h4>
<p>The lists of peptides matching the acceptance criteria from all the LC-MS/MS runs were merged into one common list, which was next overlaid onto 2-D heat maps generated from the LC-MS profile data. This list was used to tag the corresponding peptide-related ion spectra on the basis of mass difference, deviation from the predicted elution time, and the match between the theoretical and observed isotopic envelopes. A more detailed description of the quantitative feature extraction procedure implemented by our in-house software is available.<sup><a reveal-id="DSQ016C14" data-open="DSQ016C14" class="link link-ref link-reveal xref-bibr">14</a></sup> The relative abundance of each peptide was determined as the volume of a 2-D fit to the two most prominent peaks of the tagged isotopic envelope.</p><p>Multiple-charge states of the same peptide were combined by summing their relative abundances. Missing abundance values (signals below the detection level of the instrument) were replaced by a reference value equal to the smallest abundance observed in the entire LC-MS analysis. Normalization was carried out on the log-transformed peptide abundances by fitting a robust locally weighted regression smoother (LOESS) between the individual samples and a median pseudo-sample.</p>                            <h4 scrollto-destination=5869617 id="5869617" class="section-title" data-legacy-id=s2c7>2.3.7. Statistical analysis of quantitative MS results</h4>
<p>Principal components analysis (PCA) was used for graphical summarization and evaluation of the relationships among the studied samples. The statistical significance of protein abundance ratios was assessed by a one-sample <em>t</em>-test of the log-transformed peptide ratios for each protein. The resulting <em>P</em>-values were adjusted for multiple hypothesis testing using the Benjamini–Hochberg procedure that controls the false discovery rate (FDR).<sup><a reveal-id="DSQ016C15" data-open="DSQ016C15" class="link link-ref link-reveal xref-bibr">15</a></sup> All statistical analyses were performed using proprietary software implemented in the MATLAB (MathWorks) environment.</p>                            <h3 scrollto-destination=5869621 id="5869621" class="section-title" data-legacy-id=s2d>2.4. Chromatin immunoprecipitation</h3>
<p>Chromatin cross-linking and cell harvesting were performed as previously described.<sup><a reveal-id="DSQ016C16" data-open="DSQ016C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Chromatin was sheared for seven rounds of 20 s (1 s pulse, 0.5 s gap) on-off pulses with a 500-Watt Ultrasonic Processor (Cole-Parmer Instrument Co., Vernon Hills, IL, USA) equipped with a microtip and set to 30% of maximum power. Chromatin immunoprecipitation (ChIP) assays were performed using the Matrix-ChIP platform as previously described.<sup><a reveal-id="DSQ016C17" data-open="DSQ016C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Briefly, in-house prepared polypropylene 96-well PCR plates (washed once with 200 µl PBS/well) were incubated overnight at room temperature with 0.2 mg of Protein A in 100 µl PBS/well. After washing with 200 µl PBS/well, the wells were blocked with 200 µl blocking buffer for 30 min at room temperature. The wells were cleared, then incubated with 0.5 µg antibody [anti-RNAPII-CTD (Santa Cruz Biotechnology, Santa Cruz, CA, USA), sc-47701; anti-Histone H3, Abcam, ab1791; anti-Histone H3K4me3, Abcam, ab8580; anti-Histone H3K27me3 (Millipore, Billerica, MA, USA), 07-449; anti-Histone H3Ac(Lys9/18), Millipore, 07-593; anti-PARP, Abcam, ab6079] diluted in 100 µl blocking buffer/well for 60 min at room temperature. Chromatin samples (4 µl chromatin/100 µl blocking buffer) were added to the wells (100 µl/well), and the plates were floated in a 4°C ultrasonic water bath for 1 h in order to accelerate protein-antibody binding. Wells were washed three times with 150 µl IP buffer and once with 150 µl TE buffer. Wells were incubated with 100 µl elution buffers in a thermocycler for 15 min at 56°C, followed by 15 min at 95°C. DNA samples were stored at −20°C in the original Matrix-ChIP plates for repeated use.</p><p>The PCR mixture contained 2.5 µl 2× SYBR Green PCR master mix (SensiMix, Bioline, London, UK), 2.4 µl DNA template, and 0.1 µl primers (200 nM each) in 5 µl final volume. The reactions were run in 384-Well Optical Reaction Plates (Applied Biosystems). Amplification (two step, 40 cycles), data acquisition, and analysis were performed using the 7900HT Real-Time PCR system (Applied Biosystems). All PCR reactions were performed in triplicate. ChIP data are expressed as percent of input DNA as described previously,<sup><a reveal-id="DSQ016C17" data-open="DSQ016C17" class="link link-ref link-reveal xref-bibr">17</a></sup> or as an input ratio of modified histone to total histone H3. The RNAPII/H3Ac ratio was calculated for each site tested by dividing the averaged RNAPII percent of input and the H3Ac/H3 level for all time points. The ChIP data are deposited in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>. The primer sequences are listed in Table <a reveal-id="DSQ016TB1" data-open="DSQ016TB1" class="link link-reveal link-table xref-fig">1</a>.</p>                            <h3 scrollto-destination=5869626 id="5869626" class="section-title" data-legacy-id=s2e>2.5. Electrophoresis and immunoblotting</h3>
<p>Equal amounts of sample proteins (5 µg) were separated by 10% SDS–PAGE, electrotransfered to the Polyvinylidene Difluoride (PVDF) membrane and immunostained by standard methods as previously described.<sup><a reveal-id="DSQ016C18" data-open="DSQ016C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p>                            <h3 scrollto-destination=5869631 id="5869631" class="section-title" data-legacy-id=s2f>2.6. Extraction of an open chromatin ENCODE data for the promoters with the TCTCGCGAGA motif</h3>
<p>The DNase-seq tracks at <em>HNRNPK</em> promoter (chr9:85785291-85785515) for HepG2, HeLa, k562, and GM12878 cell lines were obtained from the ENCODE Open Chromatin, Duke DNase-seq Base Overlap Signal datasets<sup><a reveal-id="DSQ016C19" data-open="DSQ016C19" class="link link-ref link-reveal xref-bibr">19</a></sup> using the UCSC browser (hg18 genome assembly) <a class="link link-uri" href="http://genome.ucsc.edu">http://genome.ucsc.edu</a>. For acquiring an averaged open chromatin profile of 50 promoters bearing the palindromic motif the CisRED (<a class="link link-uri" href="http://www.cisred.org">http://www.cisred.org</a>) Human 9 database was queried with TCTCGCGAGA sequence and the resulting coordinates of 50 promoters were used to extract the DNase-seq tracks for HeLa cell line (The ENCODE Open Chromatin, Duke DNase-seq Base Overlap Signal dataset). The open chromatin profiles were superimposed over the TCTCGCGAGA sequence present in each of the 50 promoters and extended over a 150 bp window.</p>                            <h2 scrollto-destination=5869635 id="5869635" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5869637 id="5869637" class="section-title" data-legacy-id=s3a>3.1. TCTCGCGAGA is the primary regulatory element of the HNRNPK promoter</h3>
<p>Although computational approaches permit identification of gene regulatory and coding sequences, definitive characterization of promoters and their cis-regulatory motifs requires experimental investigation.<sup><a reveal-id="DSQ016C20" data-open="DSQ016C20" class="link link-ref link-reveal xref-bibr">20</a></sup> To identify which bioinformatically identified potential <em>HNRNPK</em> promoter(s) preferentially activates transcription, a series of promoter region fragments fused to a reporter gene were generated, and these constructs were transiently transfected into HeLa cells. Reporter plasmids contained four fragments of an <em>HNRNPK</em> promoter region (Fig. <a reveal-id="DSQ016F1" data-open="DSQ016F1" class="link link-reveal link-table xref-fig">1</a>A). Sequences were identical to the human reference sequence (GenBank, NC_000009.10), with the exception of two single nucleotide polymorphisms: rs7859578 in promoter fragment 1, and rs796 004 in promoter fragment 3. The highest luciferase activity resulted from the reporter with <em>HNRNPK</em> promoter fragment 1. Fragment 2 produced &lt;20% of the luciferase activity obtained with reporter fragment 1, while the luciferase activities generated by constructs containing Fragments 3 and 4 were marginal (Fig. <a reveal-id="DSQ016F1" data-open="DSQ016F1" class="link link-reveal link-table xref-fig">1</a>B).</p><p>Thus, promoter region 1 can be assumed to be the major <em>HNRNPK</em> promoter. It contains the palindromic motif TCTCGCGAGA, which has been recently grouped with the most-conserved motifs in a genome-wide human–mouse assessment of six to eight nucleotide segments.<sup><a reveal-id="DSQ016C12" data-open="DSQ016C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Palindromic motifs constitute an important group of regulatory elements. To further study the significance of this particular motif, we designed a series of Promoter 1 reporter constructs with mutated sequences at one, two, and three conserved nucleotide positions. Transcription assays revealed marked luciferase activity decreases in cells transfected with constructs containing the mutant motifs. A single point mutation in the palindromic motif resulted in a &gt;80% loss of promoter activity, whereas triple mutations reduced the activity to near basal levels, demonstrating that this single regulatory site is essential for the majority of <em>HNRNPK</em> promoter activity (Fig. <a reveal-id="DSQ016F1" data-open="DSQ016F1" class="link link-reveal link-table xref-fig">1</a>C). Similar results were demonstrated by Guo <em>et al</em>.,<sup><a reveal-id="DSQ016C21" data-open="DSQ016C21" class="link link-ref link-reveal xref-bibr">21</a></sup> who investigated the role of this motif in <em>FBN1</em> gene activity; the activity of a reporter gene decreased significantly when two middle nucleotides in the palindrome were replaced with two non-canonical nucleotides.</p>                            <h3 scrollto-destination=5869643 id="5869643" class="section-title" data-legacy-id=s3b>3.2. In vitro searches for the palindromic motif's interacting protein(s)</h3>
<p>Regulatory elements that are concentrated in promoters (cis-elements) are recognized by trans-acting proteins that control transcriptional efficiency. To analyse the interactions between nuclear proteins and the palindromic motif <em>in vitro</em>, we performed EMSA and MS-based analyses.</p><p>In order to characterize pertinent binding interactions, we used NEs, ds 16 and 30 bp DNA probes, and ss 16 and 30 oligonucleotide (nt) probes containing either the wild-type palindromic motif or a similar motif mutated at three nucleotide positions (Table <a reveal-id="DSQ016TB1" data-open="DSQ016TB1" class="link link-reveal link-table xref-fig">1</a>). As shown in Fig. <a reveal-id="DSQ016F2" data-open="DSQ016F2" class="link link-reveal link-table xref-fig">2</a>, a protein complex bound to 16 bp (Fig. <a reveal-id="DSQ016F2" data-open="DSQ016F2" class="link link-reveal link-table xref-fig">2</a>A) and 30 bp (Fig. <a reveal-id="DSQ016F2" data-open="DSQ016F2" class="link link-reveal link-table xref-fig">2</a>B) ds probes, containing either the wild-type or mutated motif, caused a similar DNA mobility shift in the form of a single band (Fig. <a reveal-id="DSQ016F2" data-open="DSQ016F2" class="link link-reveal link-table xref-fig">2</a>, left panels). The wild-type ss oligonucleotide probes, on the other hand, formed two main protein–DNA complexes represented by the typically shifted band and a slowly migrating band (Fig. <a reveal-id="DSQ016F2" data-open="DSQ016F2" class="link link-reveal link-table xref-fig">2</a>, right panels). When the wild-type 16 nt probe was replaced with a mutated oligonucleotide, the binding activity of the protein complex was substantially abolished (Fig. <a reveal-id="DSQ016F2" data-open="DSQ016F2" class="link link-reveal link-table xref-fig">2</a>C). In contrast, replacing the wild-type 30 nt sequence with a mutated probe did not change the intensity of the upper band, and significantly increased the intensity of the lower band (Fig. <a reveal-id="DSQ016F2" data-open="DSQ016F2" class="link link-reveal link-table xref-fig">2</a>D). </p>                        <a id="5869649" scrollto-destination="5869649"></a>
<div id="DSQ016F2" data-id="DSQ016F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/m_dsq01602.jpeg?Expires=2147483647&Signature=YKXTYVRCoSNYc3tHgKrXIBh-afzcE8yJYzyjEXyf-K41-ixudUFHfimjtDgtTmW0j36D7qQw9exB3ssfuUQO2KPk4vsTTCWNd9LJRRxCNOFdGy9hjfVwNJgezw3W7g5EU1ZLvWlOT2Ch-hR6r55~p7iYWj~yVHVzW0VS-~z6zJ5b4XFDLwh1lp3vcM39pKLdAkzKMkJWTIgo0Sobxf~MkExAgUj8OeK5RgTV-OUzltlYb0KD433nQgBSKpbVRa3~ro5z2hHdJORidc4w15AMhoQrWfFHXTI7kvBxcSgc2H7oUNz05~My5YPxIxPdwhXoLh3F8Y-gJes8x~~N4cnXiQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Electrophoretic mobility shift assay of in vitro binding of TCTCGCGAGA motif probes. Intact or Ku80-depleted NEs, prepared from proliferating HeLa cells, were incubated with a 32P-labelled (A) ds 16 bp probe, (B) ds 30 bp probe, (C) ss 16 nt probe, or (D) ss 30 nt probe and resolved on the gel. WT, wild-type probe, containing proper palindrome motif; MT, mutated probe." path-from-xml="dsq01602.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5869649/dsq01602.jpeg" path-from-xml="dsq01602.jpeg" target="_blank">View large</a><a section="5869649" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/dsq01602.jpeg?Expires=2147483647&Signature=IC0u3r9gL3BCKMJlNQwz93we94noluWb4XXyTSFC8wy2nfYimQTZU8SsdqD1lrkoMVxb7b6rPspaNXupgF~7v9dN2-MvIkWYbHf3RuJOVJn4pXnURha7oykG14xfRNfQ8yXb-BqyqIWco9tm5-Ux2limI7JbforWxhln3Xsh-eu~dl7nEXujqg5HWnnqLOT98w~ecVh-HkpwKxQr1Hlt8kJN7Uvd-PFbMZmrB1suViAtYIguLs2Jo0iDXFrHrVBU5IGJUzXb2d9eQNFXKBlPvbZyiaeDAeqxHtY5dMpBJuA2e~vBvJmkZACEWcAhKThlFkdW4sZX7Za2FBFNOZzOVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5869649&ar=397387&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01602.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Electrophoretic mobility shift assay of <em>in vitro</em> binding of TCTCGCGAGA motif probes. Intact or Ku80-depleted NEs, prepared from proliferating HeLa cells, were incubated with a <sup>32</sup>P-labelled (A) ds 16 bp probe, (B) ds 30 bp probe, (C) ss 16 nt probe, or (D) ss 30 nt probe and resolved on the gel. WT, wild-type probe, containing proper palindrome motif; MT, mutated probe.</p></div></div><div content-id="DSQ016F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/m_dsq01602.jpeg?Expires=2147483647&Signature=YKXTYVRCoSNYc3tHgKrXIBh-afzcE8yJYzyjEXyf-K41-ixudUFHfimjtDgtTmW0j36D7qQw9exB3ssfuUQO2KPk4vsTTCWNd9LJRRxCNOFdGy9hjfVwNJgezw3W7g5EU1ZLvWlOT2Ch-hR6r55~p7iYWj~yVHVzW0VS-~z6zJ5b4XFDLwh1lp3vcM39pKLdAkzKMkJWTIgo0Sobxf~MkExAgUj8OeK5RgTV-OUzltlYb0KD433nQgBSKpbVRa3~ro5z2hHdJORidc4w15AMhoQrWfFHXTI7kvBxcSgc2H7oUNz05~My5YPxIxPdwhXoLh3F8Y-gJes8x~~N4cnXiQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Electrophoretic mobility shift assay of in vitro binding of TCTCGCGAGA motif probes. Intact or Ku80-depleted NEs, prepared from proliferating HeLa cells, were incubated with a 32P-labelled (A) ds 16 bp probe, (B) ds 30 bp probe, (C) ss 16 nt probe, or (D) ss 30 nt probe and resolved on the gel. WT, wild-type probe, containing proper palindrome motif; MT, mutated probe." path-from-xml="dsq01602.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5869649/dsq01602.jpeg" path-from-xml="dsq01602.jpeg" target="_blank">View large</a><a section="5869649" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/dsq01602.jpeg?Expires=2147483647&Signature=IC0u3r9gL3BCKMJlNQwz93we94noluWb4XXyTSFC8wy2nfYimQTZU8SsdqD1lrkoMVxb7b6rPspaNXupgF~7v9dN2-MvIkWYbHf3RuJOVJn4pXnURha7oykG14xfRNfQ8yXb-BqyqIWco9tm5-Ux2limI7JbforWxhln3Xsh-eu~dl7nEXujqg5HWnnqLOT98w~ecVh-HkpwKxQr1Hlt8kJN7Uvd-PFbMZmrB1suViAtYIguLs2Jo0iDXFrHrVBU5IGJUzXb2d9eQNFXKBlPvbZyiaeDAeqxHtY5dMpBJuA2e~vBvJmkZACEWcAhKThlFkdW4sZX7Za2FBFNOZzOVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5869649&ar=397387&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01602.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Electrophoretic mobility shift assay of <em>in vitro</em> binding of TCTCGCGAGA motif probes. Intact or Ku80-depleted NEs, prepared from proliferating HeLa cells, were incubated with a <sup>32</sup>P-labelled (A) ds 16 bp probe, (B) ds 30 bp probe, (C) ss 16 nt probe, or (D) ss 30 nt probe and resolved on the gel. WT, wild-type probe, containing proper palindrome motif; MT, mutated probe.</p></div></div><p>With the exception of the ss 16 nt probes, the other probes predominantly captured proteins in a sequence-non-specific manner; most of the signal originating from the shifted band was supershifted by antibodies directed against ATP-dependent DNA helicase 2 subunit 2 (Ku80), one of the proteins forming DNA double-strand break repair complexes.<sup><a reveal-id="DSQ016C22" data-open="DSQ016C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Immunodepletion of Ku80 from the NEs had no effect on the supershift pattern, although the overall binding signal was weaker (Fig. <a reveal-id="DSQ016F2" data-open="DSQ016F2" class="link link-reveal link-table xref-fig">2</a>). On the other hand, the EMSA with a 16 nt ss probe produced quite a different picture of affinity to nuclear factors when compared with the activity of the other DNA probes. The supershift signal introduced by the anti-Ku80 antibody was much weaker in this assay, and competitor binding was diminished. This result suggests that the palindromic sequence in a single-stranded conformation may represent a functional cis-element. Since a palindromic motif may form a hairpin structure only when single-stranded, it is plausible that this state preferentially favours certain nuclear factors, or facilitates changes in local chromatin structure that drive transcription from promoters bearing that sequence.</p><p>To gain a broader view of the protein repertoire that might exist in complexes with the DNA probes, MS-based analyses were applied. We used NEs isolated from resting and proliferating cells in binding reactions with four 5′-biotinylated ss and ds probes, of two lengths, containing the palindromic motif sequence TCTCGCGAGA. Paired binding reactions were performed with and without a 10-fold excess of a competitor with similar characteristics (ss or ds, shorter or longer, non-biotinylated probe with a mutated motif). In addition, binding reactions were repeated using NEs that were depleted by immunoprecipitation with anti-Ku80 antibodies. Altogether, quantitative MS analyses of proteins bound to the DNA probes were carried out on 32 individual samples.</p><p>Database searches were performed on 135 592 MS/MS spectra, resulting in the identification of 15 150 PSMs with <em>q</em>-values ≤ 0.001 (corresponding to 2268 unique peptides). In total, 522 proteins were identified. Some proteins were represented by multiple peptides in all analytical MS runs, whereas other proteins appeared in individual runs as singular peptides; 297 proteins were identified by at least two peptides. Two proteins were detected by the MS/MS analysis only in the depleted samples, while 33 proteins were found only in the samples using crude NEs. Two hundred and sixty-two proteins were common to both experimental conditions. For statistical analysis we selected a subset of 132 proteins identified by at least three peptides that were annotated with the GO term nucleus (GO: 0005634), after exclusion of ribosomal proteins and histones (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>).</p><p>Normalized relative protein abundances were first transformed to corresponding principal-component scores. In the resulting plot (Fig. <a reveal-id="DSQ016F3" data-open="DSQ016F3" class="link link-reveal link-table xref-fig">3</a>), the first component divided the analysed samples into two distinct groups by discriminating between the binding of ss and ds sequences, although the second component discriminated based on probe length. </p>                        <a id="5869661" scrollto-destination="5869661"></a>
<div id="DSQ016F3" data-id="DSQ016F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/m_dsq01603.jpeg?Expires=2147483647&Signature=hzyoG~MgCGoGRtMx6cMYJlAoR84AlUjaQlkEP6ForlB87FwIUak34IEJhsZMnufl-wQrodmQLDijJlMuL177zHCST3J~8DGlTFR5WHsei-nN9sJcaGY-563oqxz2yXT71SnO8xyO7ZrTTUbpBrH2RJfwFKLSDqSt77WjOFfnNW3WoZDAepdifN4JPV~4OJaeaIVtKcWznnPvC91o8g8WJJLSPHVnBBYhxfZVu1GPEQWJa3lWQdHlKG22wk~F220AAdBU4HvHxGaJ7oFR02TUBhqzlO4O5FtgB1CwpZ1absb2SmlLrIsc8w02zLrnJhE5SnhK-lUKsanWfu24UfqxAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Principal component analysis of the relative abundances of 132 proteins selected for MS-based quantitative analysis. Experiments are colour-coded as follows: red, 16 nt single-stranded (ss) DNA probe, starving cells; blue, 16 bp ds DNA probe; green, 30 nt ss DNA probe; black, 30 bp ds DNA probe." path-from-xml="dsq01603.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5869661/dsq01603.jpeg" path-from-xml="dsq01603.jpeg" target="_blank">View large</a><a section="5869661" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/dsq01603.jpeg?Expires=2147483647&Signature=i7QUNscRdXuPggGB85x2agtjHrnd2NYKFSn~KtEM2JFlSGRAlQRZRkZOvmRT66c0PzI6baq6EwIcVTNtw6bV39M-bkuEaABc9JAnapTnPlSqiZLiFm-whgtDhwpLsVGWc3E6FwKwM3hUUnfIoDIpeO~Shr4loSmT3pU6xHY~XUe9UiK0fMz1hqob~n9do80oC5XsIiqsnMMb8AFeKzFQOXDhkc~LvzqSK6Qe6x5EXqwFIS1kjABp0Kjkzy2Y9meUCHZvIb2wmAkE1Q4d56DIJuEk5y~c5oSnEhb~IvSC-oxt0yWOWA~KvNqNrgpTX~FgRLKmj74o8YhyEy7~7KFl8w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5869661&ar=397387&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01603.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Principal component analysis of the relative abundances of 132 proteins selected for MS-based quantitative analysis. Experiments are colour-coded as follows: red, 16 nt single-stranded (ss) DNA probe, starving cells; blue, 16 bp ds DNA probe; green, 30 nt ss DNA probe; black, 30 bp ds DNA probe.</p></div></div><div content-id="DSQ016F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/m_dsq01603.jpeg?Expires=2147483647&Signature=hzyoG~MgCGoGRtMx6cMYJlAoR84AlUjaQlkEP6ForlB87FwIUak34IEJhsZMnufl-wQrodmQLDijJlMuL177zHCST3J~8DGlTFR5WHsei-nN9sJcaGY-563oqxz2yXT71SnO8xyO7ZrTTUbpBrH2RJfwFKLSDqSt77WjOFfnNW3WoZDAepdifN4JPV~4OJaeaIVtKcWznnPvC91o8g8WJJLSPHVnBBYhxfZVu1GPEQWJa3lWQdHlKG22wk~F220AAdBU4HvHxGaJ7oFR02TUBhqzlO4O5FtgB1CwpZ1absb2SmlLrIsc8w02zLrnJhE5SnhK-lUKsanWfu24UfqxAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Principal component analysis of the relative abundances of 132 proteins selected for MS-based quantitative analysis. Experiments are colour-coded as follows: red, 16 nt single-stranded (ss) DNA probe, starving cells; blue, 16 bp ds DNA probe; green, 30 nt ss DNA probe; black, 30 bp ds DNA probe." path-from-xml="dsq01603.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5869661/dsq01603.jpeg" path-from-xml="dsq01603.jpeg" target="_blank">View large</a><a section="5869661" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/dsq01603.jpeg?Expires=2147483647&Signature=i7QUNscRdXuPggGB85x2agtjHrnd2NYKFSn~KtEM2JFlSGRAlQRZRkZOvmRT66c0PzI6baq6EwIcVTNtw6bV39M-bkuEaABc9JAnapTnPlSqiZLiFm-whgtDhwpLsVGWc3E6FwKwM3hUUnfIoDIpeO~Shr4loSmT3pU6xHY~XUe9UiK0fMz1hqob~n9do80oC5XsIiqsnMMb8AFeKzFQOXDhkc~LvzqSK6Qe6x5EXqwFIS1kjABp0Kjkzy2Y9meUCHZvIb2wmAkE1Q4d56DIJuEk5y~c5oSnEhb~IvSC-oxt0yWOWA~KvNqNrgpTX~FgRLKmj74o8YhyEy7~7KFl8w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5869661&ar=397387&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01603.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Principal component analysis of the relative abundances of 132 proteins selected for MS-based quantitative analysis. Experiments are colour-coded as follows: red, 16 nt single-stranded (ss) DNA probe, starving cells; blue, 16 bp ds DNA probe; green, 30 nt ss DNA probe; black, 30 bp ds DNA probe.</p></div></div><p>Proteins may interact with a DNA sequence directly and/or indirectly, in both sequence-specific and -non-specific manners. Assuming that a competitor competes out non-specific DNA–protein interactions, we constructed a protein abundance ratios matrix. The elements of the matrix were equal to the ratios of the protein abundances measured in the paired binding reactions conducted with and without DNA competitor molecules. We assumed that lack of statistically significant changes in the protein abundance, or its enrichment within protein complexes existing in the presence of a competitor, may indicate that proteins are bound to DNA probes in a sequence-specific manner.</p><p>As already demonstrated by PCA, the relative abundances of DNA probe-associated proteins, and the protein compositions of DNA–bound complexes, were mostly dependent on probe sequence. Although the relative levels of 75 proteins, extracted from the nuclei of both resting and proliferating cells and associated with ss 16 nt and 30 nt probes, were not changed or were enriched in the binding reactions conducted in the presence of a competitor, the same was true only for 21 proteins associated with ds DNA probes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Further depletion of NEs by anti-Ku80 antibodies treatment additionally reduced the numbers of proteins which interaction with wild-type motif was unaffected by competitors to 40 and 8, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). Only three proteins (P46 013, Antigen KI-67; Q09666, Neuroblast differentiation-associated protein AHNAK; Q5SSJ5, Heterochromatin protein 1-binding protein 3) were common to all experimental settings.</p><p>Therefore, analysis by EMSA and MS of the <em>in vitro</em> formation of DNA–protein complexes revealed compositional complexity of proteins interacting with the DNA probes, but did not select trans-acting protein(s) that unambiguously directly recognize and bind the palindromic motif TCTCGCGAGA.</p><p>EMSA is a well-established method of monitoring the ability of a protein to bind to a DNA sequence <em>in vitro</em>. Despite the fact that it has been successfully employed to characterize transcription factors and the transcriptional regulation of various genes,<sup><a reveal-id="DSQ016C23" data-open="DSQ016C23" class="link link-ref link-reveal xref-bibr">23</a></sup> it also responds to non-specific protein binding, as in the case of the abundant DNA repair proteins. In particular, a heterodimer of Ku80 and Ku70 binds with high affinity to DNA ends, attracting the other components of the DNA double-strand break repair machinery.<sup><a reveal-id="DSQ016C22" data-open="DSQ016C22" class="link link-ref link-reveal xref-bibr">22</a></sup> We have encountered this inherent technical flaw during our EMSA of oligonucleotides containing the palindromic motif. Recently, Hu <em>et al</em>.<sup><a reveal-id="DSQ016C24" data-open="DSQ016C24" class="link link-ref link-reveal xref-bibr">24</a></sup> used a protein microarray-based strategy to systematically characterize the repertoire of proteins bound to 752 predicted DNA motifs from previously published studies; the palindrome TCTCGCGAGA was characterized among them. Using their data deposited in the Human Protein-DNA Interactome database,<sup><a reveal-id="DSQ016C25" data-open="DSQ016C25" class="link link-ref link-reveal xref-bibr">25</a></sup> we obtained a list of 21 proteins with reported <em>in vitro</em> affinity to this motif: LIG1, MTCP1, HOXB9, TRIP6 PHOX2A, TFAP2E, RBPMS2, NEIL2, MYOD1, ETV7, CREB5, RBM19, TCF7, C1orf25 LIG3, ACTL6B, MAFB, ZNF498, SORCS3, POU6F1, IRF3, and PARP3. We also generated a list of the 10 best matching proteins (RPS4X, PLG, H1FX, HES5, TFEB, USF2, GRHPR, TP73, ZMAT2, and LARP4) when the database was queried with the TCTCGCGAGA motif. Additionally, the hDREF protein, which was shown to regulate the expression of ribosomal proteins genes, is another protein with reported <em>in vitro</em> affinity to this motif.<sup><a reveal-id="DSQ016C26" data-open="DSQ016C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Surprisingly, none of these 32 proteins were found within the group of 132 proteins taken through quantitative statistical analysis of our MS data. This discrepancy again underlines the difficulties inherent in <em>in vitro</em> methods, where slightly different conditions (salt, detergent, protein, and DNA probe concentrations) may significantly affect the experimental readout and the ultimate data interpretation.</p>                            <h3 scrollto-destination=5869672 id="5869672" class="section-title" data-legacy-id=s3c>3.3. Cellular proliferation state increases non-specific DNA-binding</h3>
<p>We discovered that the composition of protein complexes interacting with the DNA probes relates to the proliferation status of the cells used for NE preparation. Using a FDR threshold equal to 0.05, our analysis identified 13 proteins that bind to ds 16 and 30 bp DNA probes with significantly higher affinities during proliferation, compared with quiescent cells (Table <a reveal-id="DSQ016TB2" data-open="DSQ016TB2" class="link link-reveal link-table xref-fig">2</a>). All these proteins are components of <em>DNA repair</em> process according to the GO annotation analysis (GO: 0006281). The fold change values of their relative abundances were significantly (<em>P</em> &lt; 0.02; Mann–Whitney test) decreased by a competitor and by anti-Ku80 antibody depletion of NEs used in the binding reactions. </p>                        <a id="5869677" scrollto-destination="5869677"></a>
<div content-id="DSQ016TB2" class="table-modal"><div class="table-wrap"><div id="DSQ016TB2" class="table-wrap-title" data-id="DSQ016TB2"><div class="label -label">Table 2</div><div class="caption"><p>Relative protein abundances in proliferating vs. resting cells, bound to ds DNA probes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="4">16 bp ds DNA probe<hr /> </th><th colspan="4">30 bp ds DNA probe<hr /> </th></tr><tr><th> </th><th> </th><th> </th><th colspan="2">Anti-Ku80 depletion<hr /> </th><th> </th><th> </th><th colspan="2">Anti-Ku80 depletion<hr /> </th></tr><tr><th> </th><th> </th><th>Competitor </th><th> </th><th>Competitor </th><th> </th><th>Competitor </th><th> </th><th>Competitor </th></tr><tr><th> </th><th>FC </th><th>FC </th><th>FC </th><th>FC </th><th>FC </th><th>FC </th><th>FC </th><th>FC </th></tr></thead><tbody><tr><td>P09874 Poly [ADP-ribose] polymerase 1 </td><td><strong>5.51</strong> </td><td><strong>1.66</strong> </td><td>1.10 </td><td>1.05 </td><td><strong>8.68</strong> </td><td><strong>2.30</strong> </td><td>0.93 </td><td>1.37 </td></tr><tr><td>P12956 ATP-dependent DNA helicase 2 subunit 1 </td><td><strong>3.66</strong> </td><td>1.35 </td><td>1.23 </td><td>1.65 </td><td><strong>2.60</strong> </td><td>1.67 </td><td>1.05 </td><td>ND </td></tr><tr><td>P13010 ATP-dependent DNA helicase 2 subunit 2 </td><td><strong>3.47</strong> </td><td>UP </td><td>1.06 </td><td>ND </td><td><strong>6.91</strong> </td><td>0.57 </td><td>1.58 </td><td>ND </td></tr><tr><td>P18887 DNA repair protein XRCC1 </td><td><strong>12.24</strong> </td><td><strong>4.46</strong> </td><td>1.58 </td><td>0.66 </td><td><strong>7.70</strong> </td><td><strong>5.80</strong> </td><td>1.41 </td><td>0.98 </td></tr><tr><td>P29372 DNA-3-methyladenine glycosylase </td><td><strong>3.26</strong> </td><td>1.05 </td><td><strong>1.57</strong> </td><td>0.96 </td><td><strong>3.19</strong> </td><td>ND </td><td>1.24 </td><td>ND </td></tr><tr><td>P35249 Replication factor C subunit 4 </td><td><strong>4.78</strong> </td><td><strong>1.95</strong> </td><td>ND </td><td>ND </td><td><strong>6.18</strong> </td><td>3.06 </td><td>ND </td><td>ND </td></tr><tr><td>P35250 Replication factor C subunit 2 </td><td>ND </td><td><strong>1.68</strong> </td><td>ND </td><td>ND </td><td><strong>11.88</strong> </td><td><strong>1.98</strong> </td><td>ND </td><td>ND </td></tr><tr><td>P35251 Replication factor C subunit 1 </td><td><strong>4.22</strong> </td><td><strong>2.58</strong> </td><td>ND </td><td>ND </td><td><strong>7.37</strong> </td><td><strong>2.88</strong> </td><td>ND </td><td>ND </td></tr><tr><td>P49916 DNA ligase 3 </td><td><strong>3.89</strong> </td><td><strong>2.45</strong> </td><td>1.19 </td><td>0.84 </td><td><strong>6.58</strong> </td><td><strong>3.04</strong> </td><td>0.58 </td><td>1.2 </td></tr><tr><td>P78527 DNA-dependent protein kinase catalytic subunit </td><td><strong>3.31</strong> </td><td>1.21 </td><td>1.39 </td><td>1.24 </td><td><strong>3.66</strong> </td><td>1.00 </td><td><strong>1.93</strong> </td><td>1.00 </td></tr><tr><td>P78549 Endonuclease III-like protein 1 </td><td><strong>5.01</strong> </td><td>ND </td><td>ND </td><td>ND </td><td><strong>8.35</strong> </td><td>ND </td><td>ND </td><td>ND </td></tr><tr><td>Q7Z2E3 Aprataxin </td><td><strong>7.48</strong> </td><td><strong>1.81</strong> </td><td>ND </td><td>ND </td><td><strong>7.64</strong> </td><td><strong>2.02</strong> </td><td>ND </td><td>ND </td></tr><tr><td>Q96T60 Bifunctional polynucleotide phosphatase/kinase </td><td><strong>7.85</strong> </td><td><strong>3.74</strong> </td><td>1.35 </td><td>1.06 </td><td><strong>9.89</strong> </td><td><strong>6.65</strong> </td><td>0.83 </td><td>ND </td></tr></tbody></table></div><div content-id="DSQ016TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="4">16 bp ds DNA probe<hr /> </th><th colspan="4">30 bp ds DNA probe<hr /> </th></tr><tr><th> </th><th> </th><th> </th><th colspan="2">Anti-Ku80 depletion<hr /> </th><th> </th><th> </th><th colspan="2">Anti-Ku80 depletion<hr /> </th></tr><tr><th> </th><th> </th><th>Competitor </th><th> </th><th>Competitor </th><th> </th><th>Competitor </th><th> </th><th>Competitor </th></tr><tr><th> </th><th>FC </th><th>FC </th><th>FC </th><th>FC </th><th>FC </th><th>FC </th><th>FC </th><th>FC </th></tr></thead><tbody><tr><td>P09874 Poly [ADP-ribose] polymerase 1 </td><td><strong>5.51</strong> </td><td><strong>1.66</strong> </td><td>1.10 </td><td>1.05 </td><td><strong>8.68</strong> </td><td><strong>2.30</strong> </td><td>0.93 </td><td>1.37 </td></tr><tr><td>P12956 ATP-dependent DNA helicase 2 subunit 1 </td><td><strong>3.66</strong> </td><td>1.35 </td><td>1.23 </td><td>1.65 </td><td><strong>2.60</strong> </td><td>1.67 </td><td>1.05 </td><td>ND </td></tr><tr><td>P13010 ATP-dependent DNA helicase 2 subunit 2 </td><td><strong>3.47</strong> </td><td>UP </td><td>1.06 </td><td>ND </td><td><strong>6.91</strong> </td><td>0.57 </td><td>1.58 </td><td>ND </td></tr><tr><td>P18887 DNA repair protein XRCC1 </td><td><strong>12.24</strong> </td><td><strong>4.46</strong> </td><td>1.58 </td><td>0.66 </td><td><strong>7.70</strong> </td><td><strong>5.80</strong> </td><td>1.41 </td><td>0.98 </td></tr><tr><td>P29372 DNA-3-methyladenine glycosylase </td><td><strong>3.26</strong> </td><td>1.05 </td><td><strong>1.57</strong> </td><td>0.96 </td><td><strong>3.19</strong> </td><td>ND </td><td>1.24 </td><td>ND </td></tr><tr><td>P35249 Replication factor C subunit 4 </td><td><strong>4.78</strong> </td><td><strong>1.95</strong> </td><td>ND </td><td>ND </td><td><strong>6.18</strong> </td><td>3.06 </td><td>ND </td><td>ND </td></tr><tr><td>P35250 Replication factor C subunit 2 </td><td>ND </td><td><strong>1.68</strong> </td><td>ND </td><td>ND </td><td><strong>11.88</strong> </td><td><strong>1.98</strong> </td><td>ND </td><td>ND </td></tr><tr><td>P35251 Replication factor C subunit 1 </td><td><strong>4.22</strong> </td><td><strong>2.58</strong> </td><td>ND </td><td>ND </td><td><strong>7.37</strong> </td><td><strong>2.88</strong> </td><td>ND </td><td>ND </td></tr><tr><td>P49916 DNA ligase 3 </td><td><strong>3.89</strong> </td><td><strong>2.45</strong> </td><td>1.19 </td><td>0.84 </td><td><strong>6.58</strong> </td><td><strong>3.04</strong> </td><td>0.58 </td><td>1.2 </td></tr><tr><td>P78527 DNA-dependent protein kinase catalytic subunit </td><td><strong>3.31</strong> </td><td>1.21 </td><td>1.39 </td><td>1.24 </td><td><strong>3.66</strong> </td><td>1.00 </td><td><strong>1.93</strong> </td><td>1.00 </td></tr><tr><td>P78549 Endonuclease III-like protein 1 </td><td><strong>5.01</strong> </td><td>ND </td><td>ND </td><td>ND </td><td><strong>8.35</strong> </td><td>ND </td><td>ND </td><td>ND </td></tr><tr><td>Q7Z2E3 Aprataxin </td><td><strong>7.48</strong> </td><td><strong>1.81</strong> </td><td>ND </td><td>ND </td><td><strong>7.64</strong> </td><td><strong>2.02</strong> </td><td>ND </td><td>ND </td></tr><tr><td>Q96T60 Bifunctional polynucleotide phosphatase/kinase </td><td><strong>7.85</strong> </td><td><strong>3.74</strong> </td><td>1.35 </td><td>1.06 </td><td><strong>9.89</strong> </td><td><strong>6.65</strong> </td><td>0.83 </td><td>ND </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The binding reactions with intact or Ku80-immunodepleted NEs from both proliferating and resting HeLa cells were performed with and without competitor. The eluted proteins were subjected to reduction, alkylation, tryptic digestion, and MS identification, followed by quantitative analysis of MS spectra. FC, fold change in protein abundance; ND, not detected; UP, detected only in proliferating HeLa cells, bold digits refer to statistically significant differences (adjusted <em>P</em> ≤ 0.05).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5869677" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ016TB2" class="table-wrap-title" data-id="DSQ016TB2"><div class="label -label">Table 2</div><div class="caption"><p>Relative protein abundances in proliferating vs. resting cells, bound to ds DNA probes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="4">16 bp ds DNA probe<hr /> </th><th colspan="4">30 bp ds DNA probe<hr /> </th></tr><tr><th> </th><th> </th><th> </th><th colspan="2">Anti-Ku80 depletion<hr /> </th><th> </th><th> </th><th colspan="2">Anti-Ku80 depletion<hr /> </th></tr><tr><th> </th><th> </th><th>Competitor </th><th> </th><th>Competitor </th><th> </th><th>Competitor </th><th> </th><th>Competitor </th></tr><tr><th> </th><th>FC </th><th>FC </th><th>FC </th><th>FC </th><th>FC </th><th>FC </th><th>FC </th><th>FC </th></tr></thead><tbody><tr><td>P09874 Poly [ADP-ribose] polymerase 1 </td><td><strong>5.51</strong> </td><td><strong>1.66</strong> </td><td>1.10 </td><td>1.05 </td><td><strong>8.68</strong> </td><td><strong>2.30</strong> </td><td>0.93 </td><td>1.37 </td></tr><tr><td>P12956 ATP-dependent DNA helicase 2 subunit 1 </td><td><strong>3.66</strong> </td><td>1.35 </td><td>1.23 </td><td>1.65 </td><td><strong>2.60</strong> </td><td>1.67 </td><td>1.05 </td><td>ND </td></tr><tr><td>P13010 ATP-dependent DNA helicase 2 subunit 2 </td><td><strong>3.47</strong> </td><td>UP </td><td>1.06 </td><td>ND </td><td><strong>6.91</strong> </td><td>0.57 </td><td>1.58 </td><td>ND </td></tr><tr><td>P18887 DNA repair protein XRCC1 </td><td><strong>12.24</strong> </td><td><strong>4.46</strong> </td><td>1.58 </td><td>0.66 </td><td><strong>7.70</strong> </td><td><strong>5.80</strong> </td><td>1.41 </td><td>0.98 </td></tr><tr><td>P29372 DNA-3-methyladenine glycosylase </td><td><strong>3.26</strong> </td><td>1.05 </td><td><strong>1.57</strong> </td><td>0.96 </td><td><strong>3.19</strong> </td><td>ND </td><td>1.24 </td><td>ND </td></tr><tr><td>P35249 Replication factor C subunit 4 </td><td><strong>4.78</strong> </td><td><strong>1.95</strong> </td><td>ND </td><td>ND </td><td><strong>6.18</strong> </td><td>3.06 </td><td>ND </td><td>ND </td></tr><tr><td>P35250 Replication factor C subunit 2 </td><td>ND </td><td><strong>1.68</strong> </td><td>ND </td><td>ND </td><td><strong>11.88</strong> </td><td><strong>1.98</strong> </td><td>ND </td><td>ND </td></tr><tr><td>P35251 Replication factor C subunit 1 </td><td><strong>4.22</strong> </td><td><strong>2.58</strong> </td><td>ND </td><td>ND </td><td><strong>7.37</strong> </td><td><strong>2.88</strong> </td><td>ND </td><td>ND </td></tr><tr><td>P49916 DNA ligase 3 </td><td><strong>3.89</strong> </td><td><strong>2.45</strong> </td><td>1.19 </td><td>0.84 </td><td><strong>6.58</strong> </td><td><strong>3.04</strong> </td><td>0.58 </td><td>1.2 </td></tr><tr><td>P78527 DNA-dependent protein kinase catalytic subunit </td><td><strong>3.31</strong> </td><td>1.21 </td><td>1.39 </td><td>1.24 </td><td><strong>3.66</strong> </td><td>1.00 </td><td><strong>1.93</strong> </td><td>1.00 </td></tr><tr><td>P78549 Endonuclease III-like protein 1 </td><td><strong>5.01</strong> </td><td>ND </td><td>ND </td><td>ND </td><td><strong>8.35</strong> </td><td>ND </td><td>ND </td><td>ND </td></tr><tr><td>Q7Z2E3 Aprataxin </td><td><strong>7.48</strong> </td><td><strong>1.81</strong> </td><td>ND </td><td>ND </td><td><strong>7.64</strong> </td><td><strong>2.02</strong> </td><td>ND </td><td>ND </td></tr><tr><td>Q96T60 Bifunctional polynucleotide phosphatase/kinase </td><td><strong>7.85</strong> </td><td><strong>3.74</strong> </td><td>1.35 </td><td>1.06 </td><td><strong>9.89</strong> </td><td><strong>6.65</strong> </td><td>0.83 </td><td>ND </td></tr></tbody></table></div><div content-id="DSQ016TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="4">16 bp ds DNA probe<hr /> </th><th colspan="4">30 bp ds DNA probe<hr /> </th></tr><tr><th> </th><th> </th><th> </th><th colspan="2">Anti-Ku80 depletion<hr /> </th><th> </th><th> </th><th colspan="2">Anti-Ku80 depletion<hr /> </th></tr><tr><th> </th><th> </th><th>Competitor </th><th> </th><th>Competitor </th><th> </th><th>Competitor </th><th> </th><th>Competitor </th></tr><tr><th> </th><th>FC </th><th>FC </th><th>FC </th><th>FC </th><th>FC </th><th>FC </th><th>FC </th><th>FC </th></tr></thead><tbody><tr><td>P09874 Poly [ADP-ribose] polymerase 1 </td><td><strong>5.51</strong> </td><td><strong>1.66</strong> </td><td>1.10 </td><td>1.05 </td><td><strong>8.68</strong> </td><td><strong>2.30</strong> </td><td>0.93 </td><td>1.37 </td></tr><tr><td>P12956 ATP-dependent DNA helicase 2 subunit 1 </td><td><strong>3.66</strong> </td><td>1.35 </td><td>1.23 </td><td>1.65 </td><td><strong>2.60</strong> </td><td>1.67 </td><td>1.05 </td><td>ND </td></tr><tr><td>P13010 ATP-dependent DNA helicase 2 subunit 2 </td><td><strong>3.47</strong> </td><td>UP </td><td>1.06 </td><td>ND </td><td><strong>6.91</strong> </td><td>0.57 </td><td>1.58 </td><td>ND </td></tr><tr><td>P18887 DNA repair protein XRCC1 </td><td><strong>12.24</strong> </td><td><strong>4.46</strong> </td><td>1.58 </td><td>0.66 </td><td><strong>7.70</strong> </td><td><strong>5.80</strong> </td><td>1.41 </td><td>0.98 </td></tr><tr><td>P29372 DNA-3-methyladenine glycosylase </td><td><strong>3.26</strong> </td><td>1.05 </td><td><strong>1.57</strong> </td><td>0.96 </td><td><strong>3.19</strong> </td><td>ND </td><td>1.24 </td><td>ND </td></tr><tr><td>P35249 Replication factor C subunit 4 </td><td><strong>4.78</strong> </td><td><strong>1.95</strong> </td><td>ND </td><td>ND </td><td><strong>6.18</strong> </td><td>3.06 </td><td>ND </td><td>ND </td></tr><tr><td>P35250 Replication factor C subunit 2 </td><td>ND </td><td><strong>1.68</strong> </td><td>ND </td><td>ND </td><td><strong>11.88</strong> </td><td><strong>1.98</strong> </td><td>ND </td><td>ND </td></tr><tr><td>P35251 Replication factor C subunit 1 </td><td><strong>4.22</strong> </td><td><strong>2.58</strong> </td><td>ND </td><td>ND </td><td><strong>7.37</strong> </td><td><strong>2.88</strong> </td><td>ND </td><td>ND </td></tr><tr><td>P49916 DNA ligase 3 </td><td><strong>3.89</strong> </td><td><strong>2.45</strong> </td><td>1.19 </td><td>0.84 </td><td><strong>6.58</strong> </td><td><strong>3.04</strong> </td><td>0.58 </td><td>1.2 </td></tr><tr><td>P78527 DNA-dependent protein kinase catalytic subunit </td><td><strong>3.31</strong> </td><td>1.21 </td><td>1.39 </td><td>1.24 </td><td><strong>3.66</strong> </td><td>1.00 </td><td><strong>1.93</strong> </td><td>1.00 </td></tr><tr><td>P78549 Endonuclease III-like protein 1 </td><td><strong>5.01</strong> </td><td>ND </td><td>ND </td><td>ND </td><td><strong>8.35</strong> </td><td>ND </td><td>ND </td><td>ND </td></tr><tr><td>Q7Z2E3 Aprataxin </td><td><strong>7.48</strong> </td><td><strong>1.81</strong> </td><td>ND </td><td>ND </td><td><strong>7.64</strong> </td><td><strong>2.02</strong> </td><td>ND </td><td>ND </td></tr><tr><td>Q96T60 Bifunctional polynucleotide phosphatase/kinase </td><td><strong>7.85</strong> </td><td><strong>3.74</strong> </td><td>1.35 </td><td>1.06 </td><td><strong>9.89</strong> </td><td><strong>6.65</strong> </td><td>0.83 </td><td>ND </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The binding reactions with intact or Ku80-immunodepleted NEs from both proliferating and resting HeLa cells were performed with and without competitor. The eluted proteins were subjected to reduction, alkylation, tryptic digestion, and MS identification, followed by quantitative analysis of MS spectra. FC, fold change in protein abundance; ND, not detected; UP, detected only in proliferating HeLa cells, bold digits refer to statistically significant differences (adjusted <em>P</em> ≤ 0.05).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5869677" target="_blank">View Large</a></div></div><p>To further characterize the changes in the DNA-binding affinities of non-specific binders in response to mitogenic signalling, quiescent cells were treated with 15% FBS for 0, 1, 6, and 24 h, after which NEs were prepared and used in EMSA. Again, the amount of the shifted ds probes increased with extension of time of serum induction of cells used for NE preparation. Results of a representative experiment with ds 16 bp DNA probes are presented in Fig. <a reveal-id="DSQ016F4" data-open="DSQ016F4" class="link link-reveal link-table xref-fig">4</a>A. These results indicate that either expression of the non-specifically interacting proteins in proliferating cells is increased, or that there is an increased interaction with non-specific DNA-binding proteins, or both. To differentiate between these possibilities, we analysed equal amounts of proteins from NEs by western blotting, using anti-Ku80, anti-PARP1, and anti-hnRNP K antibodies. This analysis revealed equal amounts of Ku80 and hnRNP K in extracts from both resting and proliferating cells and moderately increased signal of PARP1 after 6 h of serum stimulation (Fig. <a reveal-id="DSQ016F4" data-open="DSQ016F4" class="link link-reveal link-table xref-fig">4</a>B). </p>                        <a id="5869680" scrollto-destination="5869680"></a>
<div id="DSQ016F4" data-id="DSQ016F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/m_dsq01604.jpeg?Expires=2147483647&Signature=ArDMAImu8XCmUHYo-dfOwe~SkzNNJGCz07pJO1FajwPC0KikvSuo5gGVsnJN4plndRpFqbpMLijHNPs2edz4bGHVvQ79dW6LXRQHhEWNC9QcjX7mSuuyIHX8OMIclaUivqxaRcmpqjtliZ9vLZMJSKEtR7Fyt5fejSyDQnOw33NfdD~1OYiEWETerG~x6vfnG0AyMVXjjzwZLr0gw977HiVqASNbRyKY-8IsQj3dtmFVFUEfr19KcUYNQhq7QC5s5A~YYAmTe8bmLGRuKA3gNqYbZHtnbjQTUjtlwsEIQ1u4JECQ3P~zwSMCGJ7UvFn9-juLAh~O-Hy4cerbIoyTtg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Cellular proliferation state increases the affinity of non-specific DNA binders. (A) Electrophoretic mobility shift assay of in vitro binding of 16 bp ds DNA (P1-16), and a corresponding mutated probe (P1M-16), to HeLa NEs after 0, 1, 6, and 24 h of serum stimulation. (B) NEs (5 µg) as in (A) were resolved by SDS–PAGE and electrotransferred to a PVDF membrane. Blotted proteins were immunostained with antibodies against Ku80, PARP1, hnRNP K, and Histone H3." path-from-xml="dsq01604.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5869680/dsq01604.jpeg" path-from-xml="dsq01604.jpeg" target="_blank">View large</a><a section="5869680" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/dsq01604.jpeg?Expires=2147483647&Signature=KrhjJSy3EdL3Mi4eDFNqccid2CuVogAMH5NPdQGiTFGilwvglARWFNoXs8uRMQidWDTj38wEY-4QZZBkOlDujkzBFjg5PVbOGA4QOc8LsvS6vKMoaWYqovlfJPD3-TqhYqhUFx43TrWHQLJUQ1XG5UzemIdLRqcafYRVJePeClHdQGB6qHXecvSHpqc~wQyuSdtbDBdsagrUMWdjRl4JuxdQFqSoenH9Oym0CdzN1-qR317sJLfSbUqrAGi-0N25~b13ZSBdKC4ZUtlq7gGBZpfan7kAyg3DZCseEKst812tf2ODUK5OQGUt-jFhDeldVXjGv4J2iVt8mSed9ZT3hg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5869680&ar=397387&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01604.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Cellular proliferation state increases the affinity of non-specific DNA binders. (A) Electrophoretic mobility shift assay of <em>in vitro</em> binding of 16 bp ds DNA (P1-16), and a corresponding mutated probe (P1M-16), to HeLa NEs after 0, 1, 6, and 24 h of serum stimulation. (B) NEs (5 µg) as in (A) were resolved by SDS–PAGE and electrotransferred to a PVDF membrane. Blotted proteins were immunostained with antibodies against Ku80, PARP1, hnRNP K, and Histone H3.</p></div></div><div content-id="DSQ016F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/m_dsq01604.jpeg?Expires=2147483647&Signature=ArDMAImu8XCmUHYo-dfOwe~SkzNNJGCz07pJO1FajwPC0KikvSuo5gGVsnJN4plndRpFqbpMLijHNPs2edz4bGHVvQ79dW6LXRQHhEWNC9QcjX7mSuuyIHX8OMIclaUivqxaRcmpqjtliZ9vLZMJSKEtR7Fyt5fejSyDQnOw33NfdD~1OYiEWETerG~x6vfnG0AyMVXjjzwZLr0gw977HiVqASNbRyKY-8IsQj3dtmFVFUEfr19KcUYNQhq7QC5s5A~YYAmTe8bmLGRuKA3gNqYbZHtnbjQTUjtlwsEIQ1u4JECQ3P~zwSMCGJ7UvFn9-juLAh~O-Hy4cerbIoyTtg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Cellular proliferation state increases the affinity of non-specific DNA binders. (A) Electrophoretic mobility shift assay of in vitro binding of 16 bp ds DNA (P1-16), and a corresponding mutated probe (P1M-16), to HeLa NEs after 0, 1, 6, and 24 h of serum stimulation. (B) NEs (5 µg) as in (A) were resolved by SDS–PAGE and electrotransferred to a PVDF membrane. Blotted proteins were immunostained with antibodies against Ku80, PARP1, hnRNP K, and Histone H3." path-from-xml="dsq01604.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5869680/dsq01604.jpeg" path-from-xml="dsq01604.jpeg" target="_blank">View large</a><a section="5869680" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/dsq01604.jpeg?Expires=2147483647&Signature=KrhjJSy3EdL3Mi4eDFNqccid2CuVogAMH5NPdQGiTFGilwvglARWFNoXs8uRMQidWDTj38wEY-4QZZBkOlDujkzBFjg5PVbOGA4QOc8LsvS6vKMoaWYqovlfJPD3-TqhYqhUFx43TrWHQLJUQ1XG5UzemIdLRqcafYRVJePeClHdQGB6qHXecvSHpqc~wQyuSdtbDBdsagrUMWdjRl4JuxdQFqSoenH9Oym0CdzN1-qR317sJLfSbUqrAGi-0N25~b13ZSBdKC4ZUtlq7gGBZpfan7kAyg3DZCseEKst812tf2ODUK5OQGUt-jFhDeldVXjGv4J2iVt8mSed9ZT3hg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5869680&ar=397387&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01604.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Cellular proliferation state increases the affinity of non-specific DNA binders. (A) Electrophoretic mobility shift assay of <em>in vitro</em> binding of 16 bp ds DNA (P1-16), and a corresponding mutated probe (P1M-16), to HeLa NEs after 0, 1, 6, and 24 h of serum stimulation. (B) NEs (5 µg) as in (A) were resolved by SDS–PAGE and electrotransferred to a PVDF membrane. Blotted proteins were immunostained with antibodies against Ku80, PARP1, hnRNP K, and Histone H3.</p></div></div>                            <h3 scrollto-destination=5869684 id="5869684" class="section-title" data-legacy-id=s3d>3.4. Histone modification at promoters with the palindromic motif TCTCGCGAGA</h3>
<p>Chromatin is composed of nucleosomes, in which ∼147 bp of DNA are wrapped 1.7 times around a core of two copies each of histone proteins H2A, H2B, H3, and H4. <em>In vivo</em> positioning and covalent modifications of nucleosomes play an important role in transcriptional regulation. Post-translational covalent modifications of histone proteins include methylation, acetylation, phosphorylation, and ubiquitination. Such processes recruit and bind chromatin remodelling complexes which alter the configuration of the chromatin, and in turn, control gene transcription.<sup><a reveal-id="DSQ016C27" data-open="DSQ016C27" class="link link-ref link-reveal xref-bibr">27</a></sup> There is a high degree of complexity in the number of enzymes, histone modifications, and variant histones involved in chromatin regulation.<sup><a reveal-id="DSQ016C28" data-open="DSQ016C28" class="link link-ref link-reveal xref-bibr">28</a></sup> The spatial and temporal intricacies of the interactions of these factors with the genome suggest that chromatin regulation is at least as important to the transcriptional state of a gene as the DNA cis-elements and trans-factors present in its promoter. Thus, any complete study of the factors regulating gene expression must include the chromatin state at that gene's locus.</p><p>The promoters of expressed genes are characterized by decreased nucleosome occupancy (particularly a nucleosome-free region around the transcription start site), acetylation of H3 and H4 histones (H3Ac and H4Ac), trimethylation of histone H3 lysine 4 (H3K4me3), and the presence of histone destabilizing chromatin remodelling complexes.<sup><a reveal-id="DSQ016C29" data-open="DSQ016C29" class="link link-ref link-reveal xref-bibr">29</a></sup> The overall effect of these modifications is a general decreased presence and stability of nucleosomes, and as a result, increased accessibility to transcription factors and the RNA polymerase II (RNAPII) complex. On the other hand, silenced genes are characterized by a different set of histone modifications and, in general, by denser chromatin architecture. Histone H3 lysine 9 and lysine 27 trimethylation levels (H3K9me3 and H3K27me3) are highest at silent genes.<sup><a reveal-id="DSQ016C29" data-open="DSQ016C29" class="link link-ref link-reveal xref-bibr">29</a></sup></p><p>To characterize the <em>HNRNPK</em> promoter in greater detail, we compared it to other promoters bearing the palindromic motif TCTCGCGAGA (<em>HNRNPH</em>—heterogeneous ribonucleoprotein H and <em>RPS7</em>—ribosomal protein S7), well-established constitutively active housekeeping genes (HGs) (<em>TUBB6</em>—Tubulin beta-6, <em>GAPDH</em>—glyceraldehyde-3-phosphate dehydrogenase), an immediate early gene (<em>EGR1</em>—early growth response 1), a heat shock-induced gene (<em>HSP70</em>—heat shock 70 kDa protein), and to a silent gene (<em>HBB</em>—beta globin). We measured histone modification and RNAPII complex levels during serum stimulation in a ChIP assay (Fig. <a reveal-id="DSQ016F5" data-open="DSQ016F5" class="link link-reveal link-table xref-fig">5</a>). </p>                        <a id="5869693" scrollto-destination="5869693"></a>
<div id="DSQ016F5" data-id="DSQ016F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/m_dsq01605.jpeg?Expires=2147483647&Signature=To~grxQAGUrlZYh8Y5s5EMuSTvqdcA8FwsgJud8WZ-hJZY7KbVrV9HwbSTGesGLTxPE0P~5zRcS0p3jgSpQz2qSdWh8QuEgnURJnMboc1ABS8n05lp7DYxvw~Uc4VR6EjYvCB7JhcU4tCxs3q9l1wnh4NpTjKvFVKcCGoHTHgfinKcv0vr9PHtsVZikC6aQlNsnaSprTFSMBpATq7TukBI93XYrrSjsOQ7pRr54gedRq2IrbK1gBUul-A6jTp~6TwFEa-ABoDX2QbdBUGMdyRSgWmB5xBelmsEKyd81oIf2vmRrOFhc9IxBYci2jQy1PhT-I91zHjs7n0IYi3d~8Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Promoter levels of RNA polymerase II and histone modifications. HeLa resting (0 h) and serum-treated cells (1, 6, and 24 h) were used in ChIP assays with antibodies against RNAPII, H3, H3Ac(K9/K18), H3K4me3, and H3K27me3. Isolated DNA was used in real-time PCR with primers to the promoters of HBB, HNRNPK, TUBB6, GAPDH, HNRNPH, RPS7, HSP70, and EGR1. Data are expressed as percent of input chromatin for RNAPII, or as ratios of modified histone marks to total H3 ChIP signals. Data represent the mean ± sd of four independent experiments." path-from-xml="dsq01605.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5869693/dsq01605.jpeg" path-from-xml="dsq01605.jpeg" target="_blank">View large</a><a section="5869693" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/dsq01605.jpeg?Expires=2147483647&Signature=zj3RXduCmCiHczelgCiO3CT8YoUb9lHc0ZoF0nneGXDaquCL9iTQXxLF-Usj2avLO6TWEIDkVVxaTc2eVoFnw5E2wNf-bLd9ukYU5jHYWrrIaqrsx0dT4jFYcm~14IGtsOFtMxJciBxL6euHFDjZfAX5KSU-RUkeUfVV52TjpiLF4t9u~VvEwyN-sVgR4XKXp2kum9zz1DTn-5JLWddYPss3xJxxqX1-EgxnQZTK4N2ny~UfIH1t7gPMENbpdqgsShNFhanbDI7WGy2vddC8CSItW84Y4bwyLuLN6jPGO9XLkojtGbrHAa8KJIh-7g11MYuPvbb2XBth15WPr5LrqQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5869693&ar=397387&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01605.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Promoter levels of RNA polymerase II and histone modifications. HeLa resting (0 h) and serum-treated cells (1, 6, and 24 h) were used in ChIP assays with antibodies against RNAPII, H3, H3Ac(K9/K18), H3K4me3, and H3K27me3. Isolated DNA was used in real-time PCR with primers to the promoters of <em>HBB</em>, <em>HNRNPK</em>, <em>TUBB6</em>, <em>GAPDH</em>, <em>HNRNPH</em>, <em>RPS7</em>, <em>HSP70</em>, and <em>EGR1</em>. Data are expressed as percent of input chromatin for RNAPII, or as ratios of modified histone marks to total H3 ChIP signals. Data represent the mean ± <span class="small-caps">sd</span> of four independent experiments.</p></div></div><div content-id="DSQ016F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/m_dsq01605.jpeg?Expires=2147483647&Signature=To~grxQAGUrlZYh8Y5s5EMuSTvqdcA8FwsgJud8WZ-hJZY7KbVrV9HwbSTGesGLTxPE0P~5zRcS0p3jgSpQz2qSdWh8QuEgnURJnMboc1ABS8n05lp7DYxvw~Uc4VR6EjYvCB7JhcU4tCxs3q9l1wnh4NpTjKvFVKcCGoHTHgfinKcv0vr9PHtsVZikC6aQlNsnaSprTFSMBpATq7TukBI93XYrrSjsOQ7pRr54gedRq2IrbK1gBUul-A6jTp~6TwFEa-ABoDX2QbdBUGMdyRSgWmB5xBelmsEKyd81oIf2vmRrOFhc9IxBYci2jQy1PhT-I91zHjs7n0IYi3d~8Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Promoter levels of RNA polymerase II and histone modifications. HeLa resting (0 h) and serum-treated cells (1, 6, and 24 h) were used in ChIP assays with antibodies against RNAPII, H3, H3Ac(K9/K18), H3K4me3, and H3K27me3. Isolated DNA was used in real-time PCR with primers to the promoters of HBB, HNRNPK, TUBB6, GAPDH, HNRNPH, RPS7, HSP70, and EGR1. Data are expressed as percent of input chromatin for RNAPII, or as ratios of modified histone marks to total H3 ChIP signals. Data represent the mean ± sd of four independent experiments." path-from-xml="dsq01605.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5869693/dsq01605.jpeg" path-from-xml="dsq01605.jpeg" target="_blank">View large</a><a section="5869693" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/dsq01605.jpeg?Expires=2147483647&Signature=zj3RXduCmCiHczelgCiO3CT8YoUb9lHc0ZoF0nneGXDaquCL9iTQXxLF-Usj2avLO6TWEIDkVVxaTc2eVoFnw5E2wNf-bLd9ukYU5jHYWrrIaqrsx0dT4jFYcm~14IGtsOFtMxJciBxL6euHFDjZfAX5KSU-RUkeUfVV52TjpiLF4t9u~VvEwyN-sVgR4XKXp2kum9zz1DTn-5JLWddYPss3xJxxqX1-EgxnQZTK4N2ny~UfIH1t7gPMENbpdqgsShNFhanbDI7WGy2vddC8CSItW84Y4bwyLuLN6jPGO9XLkojtGbrHAa8KJIh-7g11MYuPvbb2XBth15WPr5LrqQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5869693&ar=397387&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01605.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Promoter levels of RNA polymerase II and histone modifications. HeLa resting (0 h) and serum-treated cells (1, 6, and 24 h) were used in ChIP assays with antibodies against RNAPII, H3, H3Ac(K9/K18), H3K4me3, and H3K27me3. Isolated DNA was used in real-time PCR with primers to the promoters of <em>HBB</em>, <em>HNRNPK</em>, <em>TUBB6</em>, <em>GAPDH</em>, <em>HNRNPH</em>, <em>RPS7</em>, <em>HSP70</em>, and <em>EGR1</em>. Data are expressed as percent of input chromatin for RNAPII, or as ratios of modified histone marks to total H3 ChIP signals. Data represent the mean ± <span class="small-caps">sd</span> of four independent experiments.</p></div></div><p>The RNAPII complex levels measured on <em>HNRNPK</em>, <em>HNRNPH</em>, and <em>RPS7</em> were similar to those at the HGs (∼10% of input), and remained relatively unchanged during serum stimulation. As expected, RNAPII levels at the <em>HBB</em> promoter were smallest, and levels at the <em>EGR1</em> promoter after 1 h of serum treatment were 2-fold higher than quiescent cells, followed by a return to basal levels at 6 and 24 h. We also observed progressive accumulation of RNAPII at the <em>HSP70</em> promoter during the serum time course, from 15.8% of input up to 24.1% in quiescent cells and after 24 h in serum, respectively (fold change = 1.52; Fig. <a reveal-id="DSQ016F5" data-open="DSQ016F5" class="link link-reveal link-table xref-fig">5</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>).</p><p>We subsequently analysed levels of acetylation of histone H3 lysine 9/18 (H3K9/18Ac), trimethylation of H3 lysine 4 (H3K4me3), and trimethylation of H3 lysine 27 (H3K27me3) at the various promoters. Acetylation of lysines eliminates their positive charge which, when it occurs on certain histone tails, has been shown to have a negative effect on the higher order structure of chromatin, essentially making it more open. H3K9/18Ac is associated with actively transcribed genes.<sup><a reveal-id="DSQ016C27" data-open="DSQ016C27" class="link link-ref link-reveal xref-bibr">27</a></sup> As expected, H3Ac levels were mirrored by the presence of RNAPII complexes, and remained similar during the serum stimulation on both palindromic and HG promoters. On the other hand, H3Ac levels measured at the <em>EGR1</em> and <em>HSP70</em> promoters were relatively smaller when compared with RNAPII amounts. The RNAPII/H3Ac ratios for <em>EGR1</em> and <em>HSP70</em> were 67:1 and 47:1, respectively, whereas the same measurement for the <em>HBB</em> promoter was 69:1. On the other hand, the ratios for both palindromic and HG promoters ranged from 11:1 for <em>HNRNPH</em> to 23:1 for <em>RPS7</em> (Fig. <a reveal-id="DSQ016F5" data-open="DSQ016F5" class="link link-reveal link-table xref-fig">5</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). H3K4me3 is associated with the 5′ ends of actively transcribed genes; H3K4me3 has been suggested to mark an open chromatin state by recruiting ATP-dependent chromatin remodelling complexes.<sup><a reveal-id="DSQ016C27" data-open="DSQ016C27" class="link link-ref link-reveal xref-bibr">27</a></sup> H3K27me3, however, marks silenced genes,<sup><a reveal-id="DSQ016C29" data-open="DSQ016C29" class="link link-ref link-reveal xref-bibr">29</a></sup> and appears to be dominant over other activating marks; genes that are marked with both K3K27me3 and H3K4me3 are typically silent. As expected, H3K4me3 was detected at the promoters of all analysed genes, including <em>HBB</em>. Also as expected, the H3K27me3 levels for all promoters were lower in comparison to levels at the transcriptionally silent <em>HBB</em>. In sum, these results indicate that RNAPII levels and histone modification levels at the promoters containing the palindromic motif TCTCGCGAGA are similar to those of constitutively active HG promoters.</p><p>We also wished to determine whether any of the proteins belonging to the DNA double-strand break repair complexes, as determined by EMSA and MS (Table <a reveal-id="DSQ016TB2" data-open="DSQ016TB2" class="link link-reveal link-table xref-fig">2</a>), were present at these promoters. Using ChIP, we measured the abundances of bound Ku80 and PARP1 at these promoters. PARP1 is responsible for the synthesis and attachment of polymers of ADP-ribose from nicotinamide adenine dinucleotide (NAD+) to the glutamic or aspartic acid residues of target proteins.<sup><a reveal-id="DSQ016C30" data-open="DSQ016C30" class="link link-ref link-reveal xref-bibr">30</a></sup> PARP1 binds to the <em>HSP70</em> promoter <em>in vivo</em>,<sup><a reveal-id="DSQ016C31" data-open="DSQ016C31" class="link link-ref link-reveal xref-bibr">31</a></sup> and therefore we used this gene as a ChIP positive control for PARP1 binding. Although Ku80 did not bind any of the studied promoters (data not shown), unexpectedly, our ChIP experiment demonstrated PARP1 binding not only to the <em>HSP70</em> promoter, but to all promoters tested, with the highest level of bound PARP1 protein at the <em>HBB</em> promoter (Fig. <a reveal-id="DSQ016F6" data-open="DSQ016F6" class="link link-reveal link-table xref-fig">6</a>A and B). Additionally, we tested an intergenic region and two sites within the <em>rhodopsin</em> (<em>RHO</em>) gene, where we found similar levels of PARP1 as measured at the <em>HBB</em> promoter (Fig. <a reveal-id="DSQ016F6" data-open="DSQ016F6" class="link link-reveal link-table xref-fig">6</a>B), along with the expected depletion of RNAPII abundance (data not shown). Overall, the ChIP assays revealed ubiquitous binding of PARP1 to various promoters, with significantly higher abundance at heterochromatin sites (Fig. <a reveal-id="DSQ016F6" data-open="DSQ016F6" class="link link-reveal link-table xref-fig">6</a>C). </p>                        <a id="5869703" scrollto-destination="5869703"></a>
<div id="DSQ016F6" data-id="DSQ016F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/m_dsq01606.jpeg?Expires=2147483647&Signature=d9fj4kyE8D87NTMKEwd~GFafunuVCruOjB3ZoySAIT9hblHvDAIWPNC5sJaKQOxTf4xkzF2XAnQEYIH8g7JJnY30NWqG7xRAZslwHpc7Qgxv7K3p5rECFoeMMJgXAaDRpOJ0-a2Qq6zXGl2~YTKMuA7qfjIzRCgBt-xadpnbyTdWVscTEAqIFUGLq5s4KclI4lFhwp89x-LfzHrLI76OF4InJjCVg92YVt5sWOhCMetpA~VsM~yUHe7oLPeolok2lXX8zE3BQTSbjsYkYNmwmjUpITEyhHLJSiTZLvfXJjLDiOSZE17QlF3VvC1F2uM8UpMZMUnNBWtHh8z9m0czyA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of PARP1 at heterochromatic and euchromatic sites. HeLa quiescent (0 h) and serum-treated cells (1, 6, and 24 h) were used in ChIP assays with antibodies against PARP1. Isolated DNA was used in real-time PCR with primers to the intergenic site, two sites within the RHO gene, and the promoter of HBB (A), as well as the promoters of HNRNPK, TUBB6, GAPDH, HNRNPH, RPS7, HSP70, and EGR1 (B). Data are expressed as percent of input chromatin and represent the mean ± sd of four independent experiments. (C) The comparison of PARP1 abundance at heterochromatic and euchromatic sites. The data represent an averaged % of input for all heterochromatin and euchromatin genes tested separately for each time point. The differences between heterochromatin and euchromatin in PARP1 occupancy were statistically significant (*P &lt; 0.005; unpaired t-test) for all time points tested." path-from-xml="dsq01606.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5869703/dsq01606.jpeg" path-from-xml="dsq01606.jpeg" target="_blank">View large</a><a section="5869703" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/dsq01606.jpeg?Expires=2147483647&Signature=PL~0gdoOuvaR7oFPbJ7DNhhzrShThld4s2YlVL-bAdxVLn7UddyZjqDDe1S4i8gkRt~Da~FxDFROiZRWAEkpuwjSfu~GfidBXu--jAUGr-0-uoIvxGUX2WCTCtRDZhov2pS8QAADTko0XVyBq1yuhzWm1hg6mrUBGbITKRZTohs77aHrz3~8DHYTXusk3Ysx75NUU1dGP93kYRApkC5RpTNnaE3NeC8moKGae0-OZzDcFn-jmfo2M-oL7~KiLTZOfGwZOT5~rDbQ~p-N5EaMG3zJ3lJfRytjgaQzxQ2hxw32fwndHyDjXN9cftNhM01VyXjekfGoNdAfxK4W33MCFw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5869703&ar=397387&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01606.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of PARP1 at heterochromatic and euchromatic sites. HeLa quiescent (0 h) and serum-treated cells (1, 6, and 24 h) were used in ChIP assays with antibodies against PARP1. Isolated DNA was used in real-time PCR with primers to the intergenic site, two sites within the <em>RHO</em> gene, and the promoter of <em>HBB</em> (A), as well as the promoters of <em>HNRNPK</em>, <em>TUBB6</em>, <em>GAPDH</em>, <em>HNRNPH</em>, <em>RPS7</em>, <em>HSP70</em>, and <em>EGR1</em> (B). Data are expressed as percent of input chromatin and represent the mean ± <span class="small-caps">sd</span> of four independent experiments. (C) The comparison of PARP1 abundance at heterochromatic and euchromatic sites. The data represent an averaged % of input for all heterochromatin and euchromatin genes tested separately for each time point. The differences between heterochromatin and euchromatin in PARP1 occupancy were statistically significant (*<em>P</em> &lt; 0.005; unpaired <em>t</em>-test) for all time points tested.</p></div></div><div content-id="DSQ016F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/m_dsq01606.jpeg?Expires=2147483647&Signature=d9fj4kyE8D87NTMKEwd~GFafunuVCruOjB3ZoySAIT9hblHvDAIWPNC5sJaKQOxTf4xkzF2XAnQEYIH8g7JJnY30NWqG7xRAZslwHpc7Qgxv7K3p5rECFoeMMJgXAaDRpOJ0-a2Qq6zXGl2~YTKMuA7qfjIzRCgBt-xadpnbyTdWVscTEAqIFUGLq5s4KclI4lFhwp89x-LfzHrLI76OF4InJjCVg92YVt5sWOhCMetpA~VsM~yUHe7oLPeolok2lXX8zE3BQTSbjsYkYNmwmjUpITEyhHLJSiTZLvfXJjLDiOSZE17QlF3VvC1F2uM8UpMZMUnNBWtHh8z9m0czyA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of PARP1 at heterochromatic and euchromatic sites. HeLa quiescent (0 h) and serum-treated cells (1, 6, and 24 h) were used in ChIP assays with antibodies against PARP1. Isolated DNA was used in real-time PCR with primers to the intergenic site, two sites within the RHO gene, and the promoter of HBB (A), as well as the promoters of HNRNPK, TUBB6, GAPDH, HNRNPH, RPS7, HSP70, and EGR1 (B). Data are expressed as percent of input chromatin and represent the mean ± sd of four independent experiments. (C) The comparison of PARP1 abundance at heterochromatic and euchromatic sites. The data represent an averaged % of input for all heterochromatin and euchromatin genes tested separately for each time point. The differences between heterochromatin and euchromatin in PARP1 occupancy were statistically significant (*P &lt; 0.005; unpaired t-test) for all time points tested." path-from-xml="dsq01606.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5869703/dsq01606.jpeg" path-from-xml="dsq01606.jpeg" target="_blank">View large</a><a section="5869703" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/dsq01606.jpeg?Expires=2147483647&Signature=PL~0gdoOuvaR7oFPbJ7DNhhzrShThld4s2YlVL-bAdxVLn7UddyZjqDDe1S4i8gkRt~Da~FxDFROiZRWAEkpuwjSfu~GfidBXu--jAUGr-0-uoIvxGUX2WCTCtRDZhov2pS8QAADTko0XVyBq1yuhzWm1hg6mrUBGbITKRZTohs77aHrz3~8DHYTXusk3Ysx75NUU1dGP93kYRApkC5RpTNnaE3NeC8moKGae0-OZzDcFn-jmfo2M-oL7~KiLTZOfGwZOT5~rDbQ~p-N5EaMG3zJ3lJfRytjgaQzxQ2hxw32fwndHyDjXN9cftNhM01VyXjekfGoNdAfxK4W33MCFw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5869703&ar=397387&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01606.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of PARP1 at heterochromatic and euchromatic sites. HeLa quiescent (0 h) and serum-treated cells (1, 6, and 24 h) were used in ChIP assays with antibodies against PARP1. Isolated DNA was used in real-time PCR with primers to the intergenic site, two sites within the <em>RHO</em> gene, and the promoter of <em>HBB</em> (A), as well as the promoters of <em>HNRNPK</em>, <em>TUBB6</em>, <em>GAPDH</em>, <em>HNRNPH</em>, <em>RPS7</em>, <em>HSP70</em>, and <em>EGR1</em> (B). Data are expressed as percent of input chromatin and represent the mean ± <span class="small-caps">sd</span> of four independent experiments. (C) The comparison of PARP1 abundance at heterochromatic and euchromatic sites. The data represent an averaged % of input for all heterochromatin and euchromatin genes tested separately for each time point. The differences between heterochromatin and euchromatin in PARP1 occupancy were statistically significant (*<em>P</em> &lt; 0.005; unpaired <em>t</em>-test) for all time points tested.</p></div></div><p>A large body of evidence has accumulated which unambiguously supports the involvement of double-strand break repair complex proteins in transcriptional regulation.<sup><a reveal-id="DSQ016C32  DSQ016C34" data-open="DSQ016C32  DSQ016C34" class="link link-ref link-reveal xref-bibr">32–34</a></sup> Ku80, Ku70, DNA-PK (DNA-dependent protein kinase), and PARP1 were among the most abundant proteins bound to DNA probes, as quantified by our MS assay (Table <a reveal-id="DSQ016TB2" data-open="DSQ016TB2" class="link link-reveal link-table xref-fig">2</a>).</p><p>PARP1, a ubiquitous and abundant nuclear protein, functions as part of the DNA repair pathway, but also orchestrates transcription through interactions with transcription factors, nucleosomes, histone-modifying proteins, and specific binding to promoters, enhancers, and insulators.<sup><a reveal-id="DSQ016C30" data-open="DSQ016C30" class="link link-ref link-reveal xref-bibr">30</a></sup> PARP1 was found to display an affinity not only to ds DNA ends, but also to DNA secondary structures like hairpins.<sup><a reveal-id="DSQ016C35" data-open="DSQ016C35" class="link link-ref link-reveal xref-bibr">35</a>,<a reveal-id="DSQ016C36" data-open="DSQ016C36" class="link link-ref link-reveal xref-bibr">36</a></sup> In particular, the <em>PARP1</em> promoter contains several imperfect repeats that are capable of forming a hairpin structure recognized by PARP1 itself, resulting in self-mediated transcriptional inhibition.<sup><a reveal-id="DSQ016C36" data-open="DSQ016C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Since the TCTCGCGAGA motif present in the <em>HNRNPK</em> promoter may form a hairpin structure <em>in vivo</em>, we speculate that such a spatial scaffold may attract PARP1. This possibility could explain the loss of signal in EMSA employing the mutated 16 nt ss probe, as the correct hairpin structure would be recognized by PARP1 both <em>in viv</em>o and <em>in vitro</em>. PARP1 was described as a structural component of heterochromatin that modulates transcription by self-modification in an NAD-dependent manner.<sup><a reveal-id="DSQ016C37" data-open="DSQ016C37" class="link link-ref link-reveal xref-bibr">37</a></sup> Furthermore, it was shown <em>in vitro</em> that PARP1 is able to promote the compaction of nucleosomes into higher order structures when NAD<sup>+</sup> is not present; on the contrary, saturating amounts of NAD<sup>+</sup> caused decondensation.<sup><a reveal-id="DSQ016C38" data-open="DSQ016C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Overall, PARP1 acts as either an activator or a repressor of specific genes, depending on the physiological conditions,<sup><a reveal-id="DSQ016C30" data-open="DSQ016C30" class="link link-ref link-reveal xref-bibr">30</a></sup> and we cannot rule out the possibility of its direct involvement in the regulation of promoters with the TCTCGCGAGA motif.</p>                            <h3 scrollto-destination=5869710 id="5869710" class="section-title" data-legacy-id=s3e>3.5. Chromatin accessibility at promoters with the palindromic motif TCTCGCGAGA</h3>
<p>During gene activation, nucleosomes are displaced and/or repositioned in a process involving histone modification, ATP-dependent nucleosome remodelling complexes, histone chaperones, and the shuffling of histone variants.<sup><a reveal-id="DSQ016C28" data-open="DSQ016C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Thus, characterizing global nucleosome positions and their chemical and compositional modifications is key to unravelling the mechanism of transcriptional regulation.</p><p>The advent of massively parallel DNA sequencing technology has allowed the localization of every nucleosome across the genomes with a high degree of accuracy.<sup><a reveal-id="DSQ016C39" data-open="DSQ016C39" class="link link-ref link-reveal xref-bibr">39</a></sup> We made use of publicly available databases to gain additional insight into chromatin structures surrounding promoters bearing the TCTCGCGAGA palindromic motif. We explored datasets provided by the ENCODE consortium within the Open Chromatin project, which, among other datasets, offers DNase I hypersensitive site (HS) tracks for multiple cell lines.<sup><a reveal-id="DSQ016C19" data-open="DSQ016C19" class="link link-ref link-reveal xref-bibr">19</a></sup> DNase I HS sites contain a mixture of cis-regulatory elements, and it has been shown that there is a striking co-occurrence of HS with regions of well-positioned nucleosomes.<sup><a reveal-id="DSQ016C40" data-open="DSQ016C40" class="link link-ref link-reveal xref-bibr">40</a></sup></p><p>First, we examined the DNase I HS profile within the <em>HNRNPK</em> promoter (Fig. <a reveal-id="DSQ016F7" data-open="DSQ016F7" class="link link-reveal link-table xref-fig">7</a>A), averaging the HS profile for four cell lines. This analysis revealed decreased values in the HS profile occurring at the palindromic motif, as well as an oscillating pattern of hypersensitive DNA that may represent a combination of trans-elements and positioned nucleosome occupancies within this region. Next, we extracted a HeLa dataset for 50 genes containing the palindromic motif TCTCGCGAGA, centred the dataset on that sequence, and superimposed all HS profiles averaged over a 150 bp window (Fig. <a reveal-id="DSQ016F7" data-open="DSQ016F7" class="link link-reveal link-table xref-fig">7</a>B). This analysis revealed that TCTCGCGAGA may define the boundaries of open chromatin, as it is flanked by two distinct HS peaks corresponding to positioned nucleosomes. On the basis of this observation, we believe that the palindrome itself may demarcate nucleosomal positions within promoters bearing it, exhibiting features that prevent nucleosomal occupancy of this motif. </p>                        <a id="5869719" scrollto-destination="5869719"></a>
<div id="DSQ016F7" data-id="DSQ016F7" class="fig fig-section"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/m_dsq01607.jpeg?Expires=2147483647&Signature=4fANQR9pfgJG3szFAe2Gbusv9Q~T19TXUZX7BGYSg4~tqZiXsQXlpDuPpdIRtFNEGIn28L43k8I~ibnntT6sN61MdX9x0IxPFXA9DZ-RbKN0IdPFd~1-gxjqz6v-zU3y67Y~3sYienJu2wuFdg68L6fd16K7jECyYB4vs8hLlwNeskQzOqbLE-DFwCZHbBndSK41yJAAlvmx00dcM-SPNVT7EfNYMwK-hqmTcSiLKXYZOO9yTSrVgRLKuGPtJx5EVoojKcWKe-Koi8zySZmaq0bs-83lpQSMbGuCcB0wjzodsBNiioWEWSa-fhTm18uvzfpydPOmNADQPni4KTekgg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="An open chromatin profile at the promoters with the TCTCGCGAGA motif. (A) Averaged profile of DNase-seq tracks surrounding the HNRNPK promoter (chr9:85785291-85785515) acquired for HepG2, HeLa, k562, and GM12878 cell lines, sd as whiskers. Data were obtained from the ENCODE Open Chromatin, Duke DNase-seq Base Overlap Signal datasets. The mammal conservation data were taken from Placental Mammal Basewise Conservation track deposited in the UCSC browser (hg18 genome assembly) http://genome.ucsc.edu. (B) Averaged HeLa cell profile of DNase-seq tracks surrounding 50 promoters bearing the TCTCGCGAGA motif. The CisRED (http://www.cisred.org) Human 9 database was queried with TCTCGCGAGA, and the resulting coordinates of 50 promoters were used to extract the DNase-seq tracks from the ENCODE Open Chromatin, Duke DNase-seq Base Overlap Signal dataset. The open chromatin profiles were superimposed over the TCTCGCGAGA sequence present in each of the 50 promoters and extended over a 150 bp window. The plot represents an averaged value of DNase-seq signal for 50 promoters." path-from-xml="dsq01607.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5869719/dsq01607.jpeg" path-from-xml="dsq01607.jpeg" target="_blank">View large</a><a section="5869719" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/dsq01607.jpeg?Expires=2147483647&Signature=zHicdozUYPd8RaLPkS97efbGxzmWFl5XEwR1ZHDZAC4ZOBXWcwMIqtG1RFtXgkIbCXSHKc6M4pxwFdmRoLJv8S10OBVonvl4nl1OksYvFs0jx2WONJgye6bwe7trYvXYO8jCF6OmU8nAJLuMpEnypfGzkfqwmjp9iBiFc-GqOlC4q-Uaj2~F734iseetRRC6R91ryKwBf0V~gR8etlLUfL4ZY0JmbV97ZC4P1wC5GL9AiVDjYJkMQEHsegVWoUSHoLJju-Y8rk7oJJPCAugIHhek6J8YTwf3DaNpgjNr2YCHQByBwVcalzxIcU9FpzP8jBuiFn1-2F00tCiVsZeRmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5869719&ar=397387&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01607.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>An open chromatin profile at the promoters with the TCTCGCGAGA motif. (A) Averaged profile of DNase-seq tracks surrounding the <em>HNRNPK</em> promoter (chr9:85785291-85785515) acquired for HepG2, HeLa, k562, and GM12878 cell lines, <span class="small-caps">sd</span> as whiskers. Data were obtained from the ENCODE Open Chromatin, Duke DNase-seq Base Overlap Signal datasets. The mammal conservation data were taken from Placental Mammal Basewise Conservation track deposited in the UCSC browser (hg18 genome assembly) <a class="link link-uri" href="http://genome.ucsc.edu">http://genome.ucsc.edu</a>. (B) Averaged HeLa cell profile of DNase-seq tracks surrounding 50 promoters bearing the TCTCGCGAGA motif. The CisRED (<a class="link link-uri" href="http://www.cisred.org">http://www.cisred.org</a>) Human 9 database was queried with TCTCGCGAGA, and the resulting coordinates of 50 promoters were used to extract the DNase-seq tracks from the ENCODE Open Chromatin, Duke DNase-seq Base Overlap Signal dataset. The open chromatin profiles were superimposed over the TCTCGCGAGA sequence present in each of the 50 promoters and extended over a 150 bp window. The plot represents an averaged value of DNase-seq signal for 50 promoters.</p></div></div><div content-id="DSQ016F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/m_dsq01607.jpeg?Expires=2147483647&Signature=4fANQR9pfgJG3szFAe2Gbusv9Q~T19TXUZX7BGYSg4~tqZiXsQXlpDuPpdIRtFNEGIn28L43k8I~ibnntT6sN61MdX9x0IxPFXA9DZ-RbKN0IdPFd~1-gxjqz6v-zU3y67Y~3sYienJu2wuFdg68L6fd16K7jECyYB4vs8hLlwNeskQzOqbLE-DFwCZHbBndSK41yJAAlvmx00dcM-SPNVT7EfNYMwK-hqmTcSiLKXYZOO9yTSrVgRLKuGPtJx5EVoojKcWKe-Koi8zySZmaq0bs-83lpQSMbGuCcB0wjzodsBNiioWEWSa-fhTm18uvzfpydPOmNADQPni4KTekgg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="An open chromatin profile at the promoters with the TCTCGCGAGA motif. (A) Averaged profile of DNase-seq tracks surrounding the HNRNPK promoter (chr9:85785291-85785515) acquired for HepG2, HeLa, k562, and GM12878 cell lines, sd as whiskers. Data were obtained from the ENCODE Open Chromatin, Duke DNase-seq Base Overlap Signal datasets. The mammal conservation data were taken from Placental Mammal Basewise Conservation track deposited in the UCSC browser (hg18 genome assembly) http://genome.ucsc.edu. (B) Averaged HeLa cell profile of DNase-seq tracks surrounding 50 promoters bearing the TCTCGCGAGA motif. The CisRED (http://www.cisred.org) Human 9 database was queried with TCTCGCGAGA, and the resulting coordinates of 50 promoters were used to extract the DNase-seq tracks from the ENCODE Open Chromatin, Duke DNase-seq Base Overlap Signal dataset. The open chromatin profiles were superimposed over the TCTCGCGAGA sequence present in each of the 50 promoters and extended over a 150 bp window. The plot represents an averaged value of DNase-seq signal for 50 promoters." path-from-xml="dsq01607.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5869719/dsq01607.jpeg" path-from-xml="dsq01607.jpeg" target="_blank">View large</a><a section="5869719" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/dsq01607.jpeg?Expires=2147483647&Signature=zHicdozUYPd8RaLPkS97efbGxzmWFl5XEwR1ZHDZAC4ZOBXWcwMIqtG1RFtXgkIbCXSHKc6M4pxwFdmRoLJv8S10OBVonvl4nl1OksYvFs0jx2WONJgye6bwe7trYvXYO8jCF6OmU8nAJLuMpEnypfGzkfqwmjp9iBiFc-GqOlC4q-Uaj2~F734iseetRRC6R91ryKwBf0V~gR8etlLUfL4ZY0JmbV97ZC4P1wC5GL9AiVDjYJkMQEHsegVWoUSHoLJju-Y8rk7oJJPCAugIHhek6J8YTwf3DaNpgjNr2YCHQByBwVcalzxIcU9FpzP8jBuiFn1-2F00tCiVsZeRmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5869719&ar=397387&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01607.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>An open chromatin profile at the promoters with the TCTCGCGAGA motif. (A) Averaged profile of DNase-seq tracks surrounding the <em>HNRNPK</em> promoter (chr9:85785291-85785515) acquired for HepG2, HeLa, k562, and GM12878 cell lines, <span class="small-caps">sd</span> as whiskers. Data were obtained from the ENCODE Open Chromatin, Duke DNase-seq Base Overlap Signal datasets. The mammal conservation data were taken from Placental Mammal Basewise Conservation track deposited in the UCSC browser (hg18 genome assembly) <a class="link link-uri" href="http://genome.ucsc.edu">http://genome.ucsc.edu</a>. (B) Averaged HeLa cell profile of DNase-seq tracks surrounding 50 promoters bearing the TCTCGCGAGA motif. The CisRED (<a class="link link-uri" href="http://www.cisred.org">http://www.cisred.org</a>) Human 9 database was queried with TCTCGCGAGA, and the resulting coordinates of 50 promoters were used to extract the DNase-seq tracks from the ENCODE Open Chromatin, Duke DNase-seq Base Overlap Signal dataset. The open chromatin profiles were superimposed over the TCTCGCGAGA sequence present in each of the 50 promoters and extended over a 150 bp window. The plot represents an averaged value of DNase-seq signal for 50 promoters.</p></div></div><p>The control of transcription is one of the most important and highly regulated steps in eukaryotic gene expression, and thus is instrumental in orchestrating the metabolic, growth, replicative, and differentiation states of cells. It is not surprising then, that transcription is carried out by a complex sequence of events, any combination of which may be regulated. Studies that investigate gene regulation are greatly facilitated by methods that characterize the interactions between proteins and DNA. Several methods have been developed to identify the direct binding of proteins to DNA sequences. In the classical DNA footprinting protocol, sequences that bind to a protein of interest are determined by identifying regions of DNA that are protected from digestion by exonuclease III or DNase I. The protected sequence corresponds to the ‘footprint’ the protein leaves on the DNA during digestion.<sup><a reveal-id="DSQ016C41" data-open="DSQ016C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Another classical method, EMSA, involves the retardation of a radiolabelled DNA probe by the binding of a protein of interest (or by a protein within a complex), indicating a molecular interaction.<sup><a reveal-id="DSQ016C23" data-open="DSQ016C23" class="link link-ref link-reveal xref-bibr">23</a></sup> All of these methods involve the <em>in vitro</em> interaction between a recombinant or NE-extracted protein with a naked segment of DNA, eliminating the chromatin context and the possible modification state(s) of the protein of interest. Similarly, assays utilizing reporter genes fail to take into account the local chromatin context, whereas potential cis-elements (promoters and enhancers) are tested using transient transfections, in which the test plasmid remains unintegrated in the nucleus.<sup><a reveal-id="DSQ016C20" data-open="DSQ016C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Despite improvements in both computational prediction of functional DNA sites and experimental verification techniques, current methods allow only snapshots of the transcription process, and as such favour indirect conclusions.</p><p>Recent advancements in high-throughput microarray (ChIP-chip) and sequencing methods (ChIP-Seq) have increased our understanding of chromatin composition, revealing transcription factor binding sites (TFBS),<sup><a reveal-id="DSQ016C42" data-open="DSQ016C42" class="link link-ref link-reveal xref-bibr">42</a>,<a reveal-id="DSQ016C43" data-open="DSQ016C43" class="link link-ref link-reveal xref-bibr">43</a></sup> chromatin accessibility,<sup><a reveal-id="DSQ016C44" data-open="DSQ016C44" class="link link-ref link-reveal xref-bibr">44</a></sup> histone modifications,<sup><a reveal-id="DSQ016C45" data-open="DSQ016C45" class="link link-ref link-reveal xref-bibr">45</a></sup> and nucleosome positions<sup><a reveal-id="DSQ016C39" data-open="DSQ016C39" class="link link-ref link-reveal xref-bibr">39</a></sup> on a genome-wide scale. These varied experimental datasets augment computational TFBS prediction methods, and aid in the description of transcriptional regulation on the scales of single genes to the entire genome.</p>                            <h3 scrollto-destination=5869728 id="5869728" class="section-title" data-legacy-id=s3f>3.6. Conclusions</h3>
<p>In the present study, we have characterized the molecular features of the palindromic motif TCTCGCGAGA, found not only in the <em>HNRNPK</em> promoter but also, as described in our previous work,<sup><a reveal-id="DSQ016C12" data-open="DSQ016C12" class="link link-ref link-reveal xref-bibr">12</a></sup> a potential cis-regulatory element of ∼5% of human genes. We used a reporter system assay to show that out of four identified <em>HNRNPK</em> promoters, the one containing the palindromic motif exhibits the highest expression of a reporter gene. Although EMSA and MS analyses, performed to discover the palindrome-binding transcription factor, did identify a complex of DNA-binding proteins, neither method unambiguously identified the relevant direct trans-acting protein(s). Moreover, for a collection of proteins functionally implicated in DNA repair processes we observed that changes in their affinity to ds DNA probes are related to the proliferation status of the cells.</p><p>To analyse the chromatin configuration at the <em>HNRNPK</em> promoter, we performed comparative ChIP experiments of RNAPII occupancy and histone modification density among the promoters of housekeeping, inducible, and silent genes. Our analysis revealed that promoters with the TCTCGCGAGA motif display properties of the promoters of HGs (<em>GAPDH</em> and <em>TUBB6</em>), containing similar RNAPII levels and relatively more H3Ac modifications than inducible and silent genes.</p><p>The analyses of open chromatin data provided by the ENCODE project revealed that the TCTCGCGAGA motif has features that promote constitutive openness of the chromatin and favor DNA accessibility. This function could be due to the binding of an unknown transcription factor, a hairpin spatial structure preventing nucleosome occupancy, or both. In sum, experimental observations using <em>in vitro</em> methods and analyses of open chromatin data support the function of this sequence in local nucleosomal organization, resulting in the regulation of transcription of the <em>HNRNPK</em> gene. Our findings suggest that the TCTCGCGAGA motif may play a critical role in regulating and establishing nucleosome patterns at the promoters containing it. Further advancements in analytical methods should elucidate the comprehensive picture of the function of this motif in gene expression regulation.</p>                            <h2 scrollto-destination=5869736 id="5869736" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=5869740 id="5869740" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by grants from the Polish Ministry of Science and Higher Education: N401193 32/4033 and R13010 03 (MS software development).</p>                            <h2 scrollto-destination=5869742 id="5869742" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Lucjan Wyrwicz for computational identification of potential <em>HNRNPK</em> promoters.</p>                            <h2 scrollto-destination=5869746 id="5869746" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSQ016C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bomsztyk</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Denisenko</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Ostrowski</div> <div class="given-names">J.</div></div>. </div><div class="article-title">hnRNP K: one protein multiple processes</div>, <div class="source">Bioessays</div> , <div class="year">2004</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">629</div>-<div class="lpage">38</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/bies.20048" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/bies.20048</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=hnRNP%20K%3A%20one%20protein%20multiple%20processes&amp;author=K.%20Bomsztyk&amp;author=O.%20Denisenko&amp;author=J.%20Ostrowski&amp;publication_year=2004&amp;journal=Bioessays&amp;volume=26&amp;pages=629-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1521-1878" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1521-1878" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1521-1878"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15170860" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Habelhah</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Shah</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">ERK phosphorylation drives cytoplasmic accumulation of hnRNP-K and inhibition of mRNA translation</div>, <div class="source">Nat. Cell biol.</div> , <div class="year">2001</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">325</div>-<div class="lpage">30</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/35060131" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/35060131</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ERK%20phosphorylation%20drives%20cytoplasmic%20accumulation%20of%20hnRNP-K%20and%20inhibition%20of%20mRNA%20translation&amp;author=H.%20Habelhah&amp;author=K.%20Shah&amp;author=L.%20Huang&amp;publication_year=2001&amp;journal=Nat.%20Cell%20biol.&amp;volume=3&amp;pages=325-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35060131" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35060131" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35060131"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11231586" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Michelotti</div> <div class="given-names">E.F.</div></div>,  <div class="name"><div class="surname">Michelotti</div> <div class="given-names">G.A.</div></div>,  <div class="name"><div class="surname">Aronsohn</div> <div class="given-names">A.I.</div></div>,  <div class="name"><div class="surname">Levens</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Heterogeneous nuclear ribonucleoprotein K is a transcription factor</div>, <div class="source">Mol. Cell. Biol.</div> , <div class="year">1996</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">2350</div>-<div class="lpage">60</div>)<div class="pub-id">
                [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8628302" class="link link-pub-id" target="_blank">PubMed</a>]
            </div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Heterogeneous%20nuclear%20ribonucleoprotein%20K%20is%20a%20transcription%20factor&amp;author=E.F.%20Michelotti&amp;author=G.A.%20Michelotti&amp;author=A.I.%20Aronsohn&amp;author=D.%20Levens&amp;publication_year=1996&amp;journal=Mol.%20Cell.%20Biol.&amp;volume=16&amp;pages=2350-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.16.5.2350" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.16.5.2350" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.16.5.2350"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8628302" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moumen</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Masterson</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">O'Connor</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Jackson</div> <div class="given-names">S.P.</div></div>. </div><div class="article-title">hnRNP K: an HDM2 target and transcriptional coactivator of p53 in response to DNA damage</div>, <div class="source">Cell</div> , <div class="year">2005</div>, vol. <div class="volume">123</div> (pg. <div class="fpage">1065</div>-<div class="lpage">78</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.cell.2005.09.032" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.cell.2005.09.032</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=hnRNP%20K%3A%20an%20HDM2%20target%20and%20transcriptional%20coactivator%20of%20p53%20in%20response%20to%20DNA%20damage&amp;author=A.%20Moumen&amp;author=P.%20Masterson&amp;author=M.J.%20O'Connor&amp;author=S.P.%20Jackson&amp;publication_year=2005&amp;journal=Cell&amp;volume=123&amp;pages=1065-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2005.09.032" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2005.09.032" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2005.09.032"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16360036" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Carpenter</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">McKay</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Dundas</div> <div class="given-names">S.R.</div></div>,  <div class="name"><div class="surname">Lawrie</div> <div class="given-names">L.C.</div></div>,  <div class="name"><div class="surname">Telfer</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Murray</div> <div class="given-names">G.I.</div></div>. </div><div class="article-title">Heterogeneous nuclear ribonucleoprotein K is over expressed, aberrantly localised and is associated with poor prognosis in colorectal cancer</div>, <div class="source">Br. J. Cancer</div> , <div class="year">2006</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">921</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/sj.bjc.6603349" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/sj.bjc.6603349</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Heterogeneous%20nuclear%20ribonucleoprotein%20K%20is%20over%20expressed%2C%20aberrantly%20localised%20and%20is%20associated%20with%20poor%20prognosis%20in%20colorectal%20cancer&amp;author=B.%20Carpenter&amp;author=M.%20McKay&amp;author=S.R.%20Dundas&amp;author=L.C.%20Lawrie&amp;author=C.%20Telfer&amp;author=G.I.%20Murray&amp;publication_year=2006&amp;journal=Br.%20J.%20Cancer&amp;volume=95&amp;pages=921-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.bjc.6603349" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.bjc.6603349" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.bjc.6603349"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16953238" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">L.C.</div></div>,  <div class="name"><div class="surname">Hsueh</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Tsang</div> <div class="given-names">N.M.</div></div>, et al. </div><div class="article-title">Heterogeneous ribonucleoprotein k and thymidine phosphorylase are independent prognostic and therapeutic markers for nasopharyngeal carcinoma</div>, <div class="source">Clin. Cancer Res.</div> , <div class="year">2008</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">3807</div>-<div class="lpage">13</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1158/1078-0432.CCR-08-0155" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1158/1078-0432.CCR-08-0155</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Heterogeneous%20ribonucleoprotein%20k%20and%20thymidine%20phosphorylase%20are%20independent%20prognostic%20and%20therapeutic%20markers%20for%20nasopharyngeal%20carcinoma&amp;author=L.C.%20Chen&amp;author=C.%20Hsueh&amp;author=N.M.%20Tsang&amp;publication_year=2008&amp;journal=Clin.%20Cancer%20Res.&amp;volume=14&amp;pages=3807-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1158/1078-0432.CCR-08-0155" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1158%2F1078-0432.CCR-08-0155" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1158%2F1078-0432.CCR-08-0155"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18559600" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hatakeyama</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kondo</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Fujii</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Protein clusters associated with carcinogenesis, histological differentiation and nodal metastasis in esophageal cancer</div>, <div class="source">Proteomics</div> , <div class="year">2006</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">6300</div>-<div class="lpage">16</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/pmic.200600488" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/pmic.200600488</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20clusters%20associated%20with%20carcinogenesis%2C%20histological%20differentiation%20and%20nodal%20metastasis%20in%20esophageal%20cancer&amp;author=H.%20Hatakeyama&amp;author=T.%20Kondo&amp;author=K.%20Fujii&amp;publication_year=2006&amp;journal=Proteomics&amp;volume=6&amp;pages=6300-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1615-9861" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1615-9861" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1615-9861"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17133371" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pino</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Pio</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Toledo</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">Altered patterns of expression of members of the heterogeneous nuclear ribonucleoprotein (hnRNP) family in lung cancer</div>, <div class="source">Lung Cancer</div> , <div class="year">2003</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">131</div>-<div class="lpage">43</div>)<div class="pub-id">
                [<a href="http://www.ncbi.nlm.nih.gov/pubmed/12871776" class="link link-pub-id" target="_blank">PubMed</a>]
            </div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Altered%20patterns%20of%20expression%20of%20members%20of%20the%20heterogeneous%20nuclear%20ribonucleoprotein%20(hnRNP)%20family%20in%20lung%20cancer&amp;author=I.%20Pino&amp;author=R.%20Pio&amp;author=G.%20Toledo&amp;publication_year=2003&amp;journal=Lung%20Cancer&amp;volume=41&amp;pages=131-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0169-5002(03)00193-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0169-5002(03)00193-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0169-5002(03)00193-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12871776" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Roychoudhury</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Chaudhuri</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Evidence for heterogeneous nuclear ribonucleoprotein K overexpression in oral squamous cell carcinoma</div>, <div class="source">Br. J. Cancer</div> , <div class="year">2007</div>, vol. <div class="volume">97</div> (pg. <div class="fpage">574</div>-<div class="lpage">5</div>) <div class="comment">author reply 576.</div><div class="pub-id-doi"><a href="https://doi.org/10.1038/sj.bjc.6603911" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/sj.bjc.6603911</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20heterogeneous%20nuclear%20ribonucleoprotein%20K%20overexpression%20in%20oral%20squamous%20cell%20carcinoma&amp;author=P.%20Roychoudhury&amp;author=K.%20Chaudhuri&amp;publication_year=2007&amp;journal=Br.%20J.%20Cancer&amp;volume=97&amp;pages=574-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.bjc.6603911" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.bjc.6603911" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.bjc.6603911"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17667925" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">de Hoog</div> <div class="given-names">C.L.</div></div>,  <div class="name"><div class="surname">Foster</div> <div class="given-names">L.J.</div></div>,  <div class="name"><div class="surname">Mann</div> <div class="given-names">M.</div></div>. </div><div class="article-title">RNA and RNA binding proteins participate in early stages of cell spreading through spreading initiation centers</div>, <div class="source">Cell</div> , <div class="year">2004</div>, vol. <div class="volume">117</div> (pg. <div class="fpage">649</div>-<div class="lpage">62</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S0092-8674(04)00456-8" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S0092-8674(04)00456-8</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20and%20RNA%20binding%20proteins%20participate%20in%20early%20stages%20of%20cell%20spreading%20through%20spreading%20initiation%20centers&amp;author=C.L.%20de%20Hoog&amp;author=L.J.%20Foster&amp;author=M.%20Mann&amp;publication_year=2004&amp;journal=Cell&amp;volume=117&amp;pages=649-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(04)00456-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(04)00456-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(04)00456-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15163412" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Inoue</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sawata</div> <div class="given-names">S.Y.</div></div>,  <div class="name"><div class="surname">Taira</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Wadhwa</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Loss-of-function screening by randomized intracellular antibodies: identification of hnRNP-K as a potential target for metastasis</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2007</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">8983</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.0607595104" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.0607595104</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Loss-of-function%20screening%20by%20randomized%20intracellular%20antibodies%3A%20identification%20of%20hnRNP-K%20as%20a%20potential%20target%20for%20metastasis&amp;author=A.%20Inoue&amp;author=S.Y.%20Sawata&amp;author=K.%20Taira&amp;author=R.%20Wadhwa&amp;publication_year=2007&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=104&amp;pages=8983-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0607595104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0607595104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0607595104"> </span></div> </div></div></div></div></div><div content-id="DSQ016C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wyrwicz</div> <div class="given-names">L.S.</div></div>,  <div class="name"><div class="surname">Gaj</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Hoffmann</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Rychlewski</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Ostrowski</div> <div class="given-names">J.</div></div>. </div><div class="article-title">A common cis-element in promoters of protein synthesis and cell cycle genes</div>, <div class="source">Acta Biochim. Pol.</div> , <div class="year">2007</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">89</div>-<div class="lpage">98</div>)<div class="pub-id">
                [<a href="http://www.ncbi.nlm.nih.gov/pubmed/17351670" class="link link-pub-id" target="_blank">PubMed</a>]
            </div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20common%20cis-element%20in%20promoters%20of%20protein%20synthesis%20and%20cell%20cycle%20genes&amp;author=L.S.%20Wyrwicz&amp;author=P.%20Gaj&amp;author=M.%20Hoffmann&amp;author=L.%20Rychlewski&amp;author=J.%20Ostrowski&amp;publication_year=2007&amp;journal=Acta%20Biochim.%20Pol.&amp;volume=54&amp;pages=89-98" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17351670" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ostrowski</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Sims</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Sibley</div> <div class="given-names">C.H.</div></div>, et al. </div><div class="article-title">A serine/threonine kinase activity is closely associated with a 65-kDa phosphoprotein specifically recognized by the kappa B enhancer element</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1991</div>, vol. <div class="volume">266</div> (pg. <div class="fpage">12722</div>-<div class="lpage">33</div>)<div class="pub-id">
                [<a href="http://www.ncbi.nlm.nih.gov/pubmed/1829460" class="link link-pub-id" target="_blank">PubMed</a>]
            </div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20serine%2Fthreonine%20kinase%20activity%20is%20closely%20associated%20with%20a%2065-kDa%20phosphoprotein%20specifically%20recognized%20by%20the%20kappa%20B%20enhancer%20element&amp;author=J.%20Ostrowski&amp;author=J.E.%20Sims&amp;author=C.H.%20Sibley&amp;publication_year=1991&amp;journal=J.%20Biol.%20Chem.&amp;volume=266&amp;pages=12722-33" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1829460" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bakun</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Karczmarski</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Poznanski</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">An integrated LC-ESI-MS platform for quantitation of serum peptide ladders. Application for colon carcinoma study</div>, <div class="source">Proteomics Clin. Appl.</div> , <div class="year">2009</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">932</div>-<div class="lpage">46</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/prca.200800111" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/prca.200800111</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20LC-ESI-MS%20platform%20for%20quantitation%20of%20serum%20peptide%20ladders.%20Application%20for%20colon%20carcinoma%20study&amp;author=M.%20Bakun&amp;author=J.%20Karczmarski&amp;author=J.%20Poznanski&amp;publication_year=2009&amp;journal=Proteomics%20Clin.%20Appl.&amp;volume=3&amp;pages=932-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/prca.v3:8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fprca.v3:8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fprca.v3:8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21136997" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Benjamini</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hochberg</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing</div>, <div class="source">J. Roy. Stat. Soc. Ser. B</div> , <div class="year">1995</div>, vol. <div class="volume">50</div> (pg. <div class="fpage">289</div>-<div class="lpage">300</div>)</div></div></div></div><div content-id="DSQ016C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mikula</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hanusek</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Paziewska</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Halogenated imidazole derivatives block RNA polymerase II elongation along mitogen inducible genes</div>, <div class="source">BMC Mol. Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">4</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2199-11-4" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2199-11-4</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Halogenated%20imidazole%20derivatives%20block%20RNA%20polymerase%20II%20elongation%20along%20mitogen%20inducible%20genes&amp;author=M.%20Mikula&amp;author=K.%20Hanusek&amp;author=A.%20Paziewska&amp;publication_year=2010&amp;journal=BMC%20Mol.%20Biol.&amp;volume=11&amp;pages=4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2199-11-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2199-11-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2199-11-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20078881" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Flanagin</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nelson</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Castner</div> <div class="given-names">D.G.</div></div>,  <div class="name"><div class="surname">Denisenko</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Bomsztyk</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Microplate-based chromatin immunoprecipitation method, Matrix ChIP: a platform to study signaling of complex genomic events</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> pg. <div class="fpage">e17</div> <div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkn001" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkn001</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microplate-based%20chromatin%20immunoprecipitation%20method%2C%20Matrix%20ChIP%3A%20a%20platform%20to%20study%20signaling%20of%20complex%20genomic%20events&amp;author=S.%20Flanagin&amp;author=J.D.%20Nelson&amp;author=D.G.%20Castner&amp;author=O.%20Denisenko&amp;author=K.%20Bomsztyk&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=e17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18203739" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ostrowski</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Woszczynski</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kowalczyk</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Trzeciak</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Hennig</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Bomsztyk</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Treatment of mice with EGF and orthovanadate activates cytoplasmic and nuclear MAPK, p70S6k, and p90rsk in the liver</div>, <div class="source">J. Hepatol.</div> , <div class="year">2000</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">965</div>-<div class="lpage">74</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S0168-8278(00)80101-4" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S0168-8278(00)80101-4</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Treatment%20of%20mice%20with%20EGF%20and%20orthovanadate%20activates%20cytoplasmic%20and%20nuclear%20MAPK%2C%20p70S6k%2C%20and%20p90rsk%20in%20the%20liver&amp;author=J.%20Ostrowski&amp;author=M.%20Woszczynski&amp;author=P.%20Kowalczyk&amp;author=L.%20Trzeciak&amp;author=E.%20Hennig&amp;author=K.%20Bomsztyk&amp;publication_year=2000&amp;journal=J.%20Hepatol.&amp;volume=32&amp;pages=965-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-8278(00)80101-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-8278(00)80101-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-8278(00)80101-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10898317" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rosenbloom</div> <div class="given-names">K.R.</div></div>,  <div class="name"><div class="surname">Dreszer</div> <div class="given-names">T.R.</div></div>,  <div class="name"><div class="surname">Pheasant</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">ENCODE whole-genome data in the UCSC Genome Browser</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2010</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">D620</div>-<div class="lpage">D625</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkp961" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkp961</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ENCODE%20whole-genome%20data%20in%20the%20UCSC%20Genome%20Browser&amp;author=K.R.%20Rosenbloom&amp;author=T.R.%20Dreszer&amp;author=M.%20Pheasant&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=D620-D625" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp961" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp961" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp961"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19920125" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Elnitski</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Jin</div> <div class="given-names">V.X.</div></div>,  <div class="name"><div class="surname">Farnham</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Jones</div> <div class="given-names">S.J.</div></div>. </div><div class="article-title">Locating mammalian transcription factor binding sites: a survey of computational and experimental techniques</div>, <div class="source">Genome Res.</div> , <div class="year">2006</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">1455</div>-<div class="lpage">64</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.4140006" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.4140006</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Locating%20mammalian%20transcription%20factor%20binding%20sites%3A%20a%20survey%20of%20computational%20and%20experimental%20techniques&amp;author=L.%20Elnitski&amp;author=V.X.%20Jin&amp;author=P.J.%20Farnham&amp;author=S.J.%20Jones&amp;publication_year=2006&amp;journal=Genome%20Res.&amp;volume=16&amp;pages=1455-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.4140006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.4140006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.4140006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17053094" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guo</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Bauer</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hecht</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Schulz</div> <div class="given-names">M.H.</div></div>,  <div class="name"><div class="surname">Busche</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Robinson</div> <div class="given-names">P.N.</div></div>. </div><div class="article-title">A short ultraconserved sequence drives transcription from an alternate FBN1 promoter</div>, <div class="source">Int. J. Biochem. Cell Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">638</div>-<div class="lpage">50</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.biocel.2007.09.004" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.biocel.2007.09.004</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20short%20ultraconserved%20sequence%20drives%20transcription%20from%20an%20alternate%20FBN1%20promoter&amp;author=G.%20Guo&amp;author=S.%20Bauer&amp;author=J.%20Hecht&amp;author=M.H.%20Schulz&amp;author=A.%20Busche&amp;author=P.N.%20Robinson&amp;publication_year=2008&amp;journal=Int.%20J.%20Biochem.%20Cell%20Biol.&amp;volume=40&amp;pages=638-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.biocel.2007.09.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.biocel.2007.09.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.biocel.2007.09.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17996480" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kanaar</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Hoeijmakers</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">van Gent</div> <div class="given-names">D.C.</div></div>. </div><div class="article-title">Molecular mechanisms of DNA double strand break repair</div>, <div class="source">Trends Cell Biol.</div> , <div class="year">1998</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">483</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S0962-8924(98)01383-X" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S0962-8924(98)01383-X</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20mechanisms%20of%20DNA%20double%20strand%20break%20repair&amp;author=R.%20Kanaar&amp;author=J.H.%20Hoeijmakers&amp;author=D.C.%20van%20Gent&amp;publication_year=1998&amp;journal=Trends%20Cell%20Biol.&amp;volume=8&amp;pages=483-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0962-8924(98)01383-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0962-8924(98)01383-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0962-8924(98)01383-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9861670" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jiang</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Jarrett</div> <div class="given-names">H.W.</div></div>,  <div class="name"><div class="surname">Haskins</div> <div class="given-names">W.E.</div></div>. </div><div class="article-title">Methods for proteomic analysis of transcription factors</div>, <div class="source">J. Chromatogr.</div> , <div class="year">2009</div>, vol. <div class="volume">1216</div> (pg. <div class="fpage">6881</div>-<div class="lpage">9</div>)<div class="pub-id">
                [<a href="http://www.ncbi.nlm.nih.gov/pubmed/19726046" class="link link-pub-id" target="_blank">PubMed</a>]
            </div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Methods%20for%20proteomic%20analysis%20of%20transcription%20factors&amp;author=D.%20Jiang&amp;author=H.W.%20Jarrett&amp;author=W.E.%20Haskins&amp;publication_year=2009&amp;journal=J.%20Chromatogr.&amp;volume=1216&amp;pages=6881-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.chroma.2009.08.044" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.chroma.2009.08.044" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.chroma.2009.08.044"> </span></div> </div></div></div></div></div><div content-id="DSQ016C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Xie</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Onishi</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Profiling the human protein–DNA interactome reveals ERK2 as a transcriptional repressor of interferon signaling</div>, <div class="source">Cell</div> , <div class="year">2009</div>, vol. <div class="volume">139</div> (pg. <div class="fpage">610</div>-<div class="lpage">22</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.cell.2009.08.037" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.cell.2009.08.037</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Profiling%20the%20human%20protein%E2%80%93DNA%20interactome%20reveals%20ERK2%20as%20a%20transcriptional%20repressor%20of%20interferon%20signaling&amp;author=S.%20Hu&amp;author=Z.%20Xie&amp;author=A.%20Onishi&amp;publication_year=2009&amp;journal=Cell&amp;volume=139&amp;pages=610-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2009.08.037" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2009.08.037" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2009.08.037"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19879846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xie</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Hu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Blackshaw</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Qian</div> <div class="given-names">J.</div></div>. </div><div class="article-title">hPDI: a database of experimental human protein–DNA interactions</div>, <div class="source">Bioinformatics</div> , <div class="year">2010</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">287</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/btp631" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/btp631</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=hPDI%3A%20a%20database%20of%20experimental%20human%20protein%E2%80%93DNA%20interactions&amp;author=Z.%20Xie&amp;author=S.%20Hu&amp;author=S.%20Blackshaw&amp;author=H.%20Zhu&amp;author=J.%20Qian&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=287-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp631" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp631" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp631"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19900953" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamashita</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Sano</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Adachi</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">hDREF regulates cell proliferation and expression of ribosomal protein genes</div>, <div class="source">Mol. Cell. Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">2003</div>-<div class="lpage">13</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1128/MCB.01462-06" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1128/MCB.01462-06</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=hDREF%20regulates%20cell%20proliferation%20and%20expression%20of%20ribosomal%20protein%20genes&amp;author=D.%20Yamashita&amp;author=Y.%20Sano&amp;author=Y.%20Adachi&amp;publication_year=2007&amp;journal=Mol.%20Cell.%20Biol.&amp;volume=27&amp;pages=2003-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.01462-06" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.01462-06" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.01462-06"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17220279" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kouzarides</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Chromatin modifications and their function</div>, <div class="source">Cell</div> , <div class="year">2007</div>, vol. <div class="volume">128</div> (pg. <div class="fpage">693</div>-<div class="lpage">705</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.cell.2007.02.005" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.cell.2007.02.005</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromatin%20modifications%20and%20their%20function&amp;author=T.%20Kouzarides&amp;publication_year=2007&amp;journal=Cell&amp;volume=128&amp;pages=693-705" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2007.02.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2007.02.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2007.02.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17320507" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Carey</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Workman</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">The role of chromatin during transcription</div>, <div class="source">Cell</div> , <div class="year">2007</div>, vol. <div class="volume">128</div> (pg. <div class="fpage">707</div>-<div class="lpage">19</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.cell.2007.01.015" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.cell.2007.01.015</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20chromatin%20during%20transcription&amp;author=B.%20Li&amp;author=M.%20Carey&amp;author=J.L.%20Workman&amp;publication_year=2007&amp;journal=Cell&amp;volume=128&amp;pages=707-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2007.01.015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2007.01.015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2007.01.015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17320508" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mellor</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Dudek</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Clynes</div> <div class="given-names">D.</div></div>. </div><div class="article-title">A glimpse into the epigenetic landscape of gene regulation</div>, <div class="source">Curr. Opin. Genet. Dev.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">116</div>-<div class="lpage">22</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.gde.2007.12.005" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.gde.2007.12.005</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20glimpse%20into%20the%20epigenetic%20landscape%20of%20gene%20regulation&amp;author=J.%20Mellor&amp;author=P.%20Dudek&amp;author=D.%20Clynes&amp;publication_year=2008&amp;journal=Curr.%20Opin.%20Genet.%20Dev.&amp;volume=18&amp;pages=116-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gde.2007.12.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gde.2007.12.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gde.2007.12.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18295475" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kraus</div> <div class="given-names">W.L.</div></div>. </div><div class="article-title">Transcriptional control by PARP-1: chromatin modulation, enhancer-binding, coregulation, and insulation</div>, <div class="source">Curr. Opin. Cell Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">294</div>-<div class="lpage">302</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.ceb.2008.03.006" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.ceb.2008.03.006</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20control%20by%20PARP-1%3A%20chromatin%20modulation%2C%20enhancer-binding%2C%20coregulation%2C%20and%20insulation&amp;author=W.L.%20Kraus&amp;publication_year=2008&amp;journal=Curr.%20Opin.%20Cell%20Biol.&amp;volume=20&amp;pages=294-302" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ceb.2008.03.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ceb.2008.03.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ceb.2008.03.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18450439" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Petesch</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Lis</div> <div class="given-names">J.T.</div></div>. </div><div class="article-title">Rapid, transcription-independent loss of nucleosomes over a large chromatin domain at Hsp70 loci</div>, <div class="source">Cell</div> , <div class="year">2008</div>, vol. <div class="volume">134</div> (pg. <div class="fpage">74</div>-<div class="lpage">84</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.cell.2008.05.029" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.cell.2008.05.029</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%2C%20transcription-independent%20loss%20of%20nucleosomes%20over%20a%20large%20chromatin%20domain%20at%20Hsp70%20loci&amp;author=S.J.%20Petesch&amp;author=J.T.%20Lis&amp;publication_year=2008&amp;journal=Cell&amp;volume=134&amp;pages=74-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2008.05.029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2008.05.029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2008.05.029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18614012" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hossain</div> <div class="given-names">M.B.</div></div>,  <div class="name"><div class="surname">Ji</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Anish</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Jacobson</div> <div class="given-names">R.H.</div></div>,  <div class="name"><div class="surname">Takada</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Poly(ADP-ribose) polymerase 1 interacts with nuclear respiratory factor 1 (NRF-1) and plays a role in NRF-1 transcriptional regulation</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2009</div>, vol. <div class="volume">284</div> (pg. <div class="fpage">8621</div>-<div class="lpage">32</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1074/jbc.M807198200" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1074/jbc.M807198200</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Poly(ADP-ribose)%20polymerase%201%20interacts%20with%20nuclear%20respiratory%20factor%201%20(NRF-1)%20and%20plays%20a%20role%20in%20NRF-1%20transcriptional%20regulation&amp;author=M.B.%20Hossain&amp;author=P.%20Ji&amp;author=R.%20Anish&amp;author=R.H.%20Jacobson&amp;author=S.%20Takada&amp;publication_year=2009&amp;journal=J.%20Biol.%20Chem.&amp;volume=284&amp;pages=8621-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M807198200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M807198200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M807198200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19181665" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ju</div> <div class="given-names">B.G.</div></div>,  <div class="name"><div class="surname">Lunyak</div> <div class="given-names">V.V.</div></div>,  <div class="name"><div class="surname">Perissi</div> <div class="given-names">V.</div></div>, et al. </div><div class="article-title">A topoisomerase IIbeta-mediated dsDNA break required for regulated transcription</div>, <div class="source">Science</div> , <div class="year">2006</div>, vol. <div class="volume">312</div> (pg. <div class="fpage">1798</div>-<div class="lpage">802</div>)<div class="pub-id">
                [<a href="http://www.ncbi.nlm.nih.gov/pubmed/16794079" class="link link-pub-id" target="_blank">PubMed</a>]
            </div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20topoisomerase%20IIbeta-mediated%20dsDNA%20break%20required%20for%20regulated%20transcription&amp;author=B.G.%20Ju&amp;author=V.V.%20Lunyak&amp;author=V.%20Perissi&amp;publication_year=2006&amp;journal=Science&amp;volume=312&amp;pages=1798-802" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1127196" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1127196" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1127196"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16794079" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shi</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Qiu</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">Dynamic binding of Ku80, Ku70 and NF90 to the IL-2 promoter in vivo in activated T-cells</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">2302</div>-<div class="lpage">10</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkm117" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkm117</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamic%20binding%20of%20Ku80%2C%20Ku70%20and%20NF90%20to%20the%20IL-2%20promoter%20in%20vivo%20in%20activated%20T-cells&amp;author=L.%20Shi&amp;author=D.%20Qiu&amp;author=G.%20Zhao&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=2302-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm117" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm117" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm117"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17389650" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Potaman</div> <div class="given-names">V.N.</div></div>,  <div class="name"><div class="surname">Shlyakhtenko</div> <div class="given-names">L.S.</div></div>,  <div class="name"><div class="surname">Oussatcheva</div> <div class="given-names">E.A.</div></div>,  <div class="name"><div class="surname">Lyubchenko</div> <div class="given-names">Y.L.</div></div>,  <div class="name"><div class="surname">Soldatenkov</div> <div class="given-names">V.A.</div></div>. </div><div class="article-title">Specific binding of poly(ADP-ribose) polymerase-1 to cruciform hairpins</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">348</div> (pg. <div class="fpage">609</div>-<div class="lpage">15</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.jmb.2005.03.010" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.jmb.2005.03.010</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Specific%20binding%20of%20poly(ADP-ribose)%20polymerase-1%20to%20cruciform%20hairpins&amp;author=V.N.%20Potaman&amp;author=L.S.%20Shlyakhtenko&amp;author=E.A.%20Oussatcheva&amp;author=Y.L.%20Lyubchenko&amp;author=V.A.%20Soldatenkov&amp;publication_year=2005&amp;journal=J.%20Mol.%20Biol.&amp;volume=348&amp;pages=609-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmb.2005.03.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2005.03.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jmb.2005.03.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15826658" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Soldatenkov</div> <div class="given-names">V.A.</div></div>,  <div class="name"><div class="surname">Chasovskikh</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Potaman</div> <div class="given-names">V.N.</div></div>,  <div class="name"><div class="surname">Trofimova</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Smulson</div> <div class="given-names">M.E.</div></div>,  <div class="name"><div class="surname">Dritschilo</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Transcriptional repression by binding of poly(ADP-ribose) polymerase to promoter sequences</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2002</div>, vol. <div class="volume">277</div> (pg. <div class="fpage">665</div>-<div class="lpage">70</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1074/jbc.M108551200" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1074/jbc.M108551200</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20repression%20by%20binding%20of%20poly(ADP-ribose)%20polymerase%20to%20promoter%20sequences&amp;author=V.A.%20Soldatenkov&amp;author=S.%20Chasovskikh&amp;author=V.N.%20Potaman&amp;author=I.%20Trofimova&amp;author=M.E.%20Smulson&amp;author=A.%20Dritschilo&amp;publication_year=2002&amp;journal=J.%20Biol.%20Chem.&amp;volume=277&amp;pages=665-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M108551200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M108551200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M108551200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11684688" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">M.Y.</div></div>,  <div class="name"><div class="surname">Mauro</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gevry</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Lis</div> <div class="given-names">J.T.</div></div>,  <div class="name"><div class="surname">Kraus</div> <div class="given-names">W.L.</div></div>. </div><div class="article-title">NAD+-dependent modulation of chromatin structure and transcription by nucleosome binding properties of PARP-1</div>, <div class="source">Cell</div> , <div class="year">2004</div>, vol. <div class="volume">119</div> (pg. <div class="fpage">803</div>-<div class="lpage">14</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.cell.2004.11.002" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.cell.2004.11.002</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NAD%2B-dependent%20modulation%20of%20chromatin%20structure%20and%20transcription%20by%20nucleosome%20binding%20properties%20of%20PARP-1&amp;author=M.Y.%20Kim&amp;author=S.%20Mauro&amp;author=N.%20Gevry&amp;author=J.T.%20Lis&amp;author=W.L.%20Kraus&amp;publication_year=2004&amp;journal=Cell&amp;volume=119&amp;pages=803-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2004.11.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2004.11.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2004.11.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15607977" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wacker</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Ruhl</div> <div class="given-names">D.D.</div></div>,  <div class="name"><div class="surname">Balagamwala</div> <div class="given-names">E.H.</div></div>,  <div class="name"><div class="surname">Hope</div> <div class="given-names">K.M.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kraus</div> <div class="given-names">W.L.</div></div>. </div><div class="article-title">The DNA binding and catalytic domains of poly(ADP-ribose) polymerase 1 cooperate in the regulation of chromatin structure and transcription</div>, <div class="source">Mol. Cell. Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">7475</div>-<div class="lpage">85</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1128/MCB.01314-07" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1128/MCB.01314-07</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20DNA%20binding%20and%20catalytic%20domains%20of%20poly(ADP-ribose)%20polymerase%201%20cooperate%20in%20the%20regulation%20of%20chromatin%20structure%20and%20transcription&amp;author=D.A.%20Wacker&amp;author=D.D.%20Ruhl&amp;author=E.H.%20Balagamwala&amp;author=K.M.%20Hope&amp;author=T.%20Zhang&amp;author=W.L.%20Kraus&amp;publication_year=2007&amp;journal=Mol.%20Cell.%20Biol.&amp;volume=27&amp;pages=7475-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.01314-07" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.01314-07" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.01314-07"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17785446" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schones</div> <div class="given-names">D.E.</div></div>,  <div class="name"><div class="surname">Cui</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Cuddapah</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Dynamic regulation of nucleosome positioning in the human genome</div>, <div class="source">Cell</div> , <div class="year">2008</div>, vol. <div class="volume">132</div> (pg. <div class="fpage">887</div>-<div class="lpage">98</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.cell.2008.02.022" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.cell.2008.02.022</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamic%20regulation%20of%20nucleosome%20positioning%20in%20the%20human%20genome&amp;author=D.E.%20Schones&amp;author=K.%20Cui&amp;author=S.%20Cuddapah&amp;publication_year=2008&amp;journal=Cell&amp;volume=132&amp;pages=887-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2008.02.022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2008.02.022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2008.02.022"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18329373" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Shin</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Lei</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">X.S.</div></div>. </div><div class="article-title">Identifying positioned nucleosomes with epigenetic marks in human from ChIP-Seq</div>, <div class="source">BMC Genomics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">537</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-9-537" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-9-537</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identifying%20positioned%20nucleosomes%20with%20epigenetic%20marks%20in%20human%20from%20ChIP-Seq&amp;author=Y.%20Zhang&amp;author=H.%20Shin&amp;author=J.S.%20Song&amp;author=Y.%20Lei&amp;author=X.S.%20Liu&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=537" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-537" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-537" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-537"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19014516" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wu</div> <div class="given-names">C.</div></div>. </div><div class="article-title">An exonuclease protection assay reveals heat-shock element and TATA box DNA-binding proteins in crude nuclear extracts</div>, <div class="source">Nature</div> , <div class="year">1985</div>, vol. <div class="volume">317</div> (pg. <div class="fpage">84</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/317084a0" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/317084a0</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20exonuclease%20protection%20assay%20reveals%20heat-shock%20element%20and%20TATA%20box%20DNA-binding%20proteins%20in%20crude%20nuclear%20extracts&amp;author=C.%20Wu&amp;publication_year=1985&amp;journal=Nature&amp;volume=317&amp;pages=84-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/317084a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F317084a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F317084a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2993917" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Johnson</div> <div class="given-names">D.S.</div></div>,  <div class="name"><div class="surname">Mortazavi</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Wold</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Genome-wide mapping of in vivo protein–DNA interactions</div>, <div class="source">Science</div> , <div class="year">2007</div>, vol. <div class="volume">316</div> (pg. <div class="fpage">1497</div>-<div class="lpage">502</div>)<div class="pub-id">
                [<a href="http://www.ncbi.nlm.nih.gov/pubmed/17540862" class="link link-pub-id" target="_blank">PubMed</a>]
            </div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20mapping%20of%20in%20vivo%20protein%E2%80%93DNA%20interactions&amp;author=D.S.%20Johnson&amp;author=A.%20Mortazavi&amp;author=R.M.%20Myers&amp;author=B.%20Wold&amp;publication_year=2007&amp;journal=Science&amp;volume=316&amp;pages=1497-502" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1141319" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1141319" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1141319"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17540862" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Valouev</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Johnson</div> <div class="given-names">D.S.</div></div>,  <div class="name"><div class="surname">Sundquist</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Genome-wide analysis of transcription factor binding sites based on ChIP-Seq data</div>, <div class="source">Nat. Methods</div> , <div class="year">2008</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">829</div>-<div class="lpage">34</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nmeth.1246" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nmeth.1246</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20transcription%20factor%20binding%20sites%20based%20on%20ChIP-Seq%20data&amp;author=A.%20Valouev&amp;author=D.S.%20Johnson&amp;author=A.%20Sundquist&amp;publication_year=2008&amp;journal=Nat.%20Methods&amp;volume=5&amp;pages=829-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1246" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1246" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1246"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19160518" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Boyle</div> <div class="given-names">A.P.</div></div>,  <div class="name"><div class="surname">Davis</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Shulha</div> <div class="given-names">H.P.</div></div>, et al. </div><div class="article-title">High-resolution mapping and characterization of open chromatin across the genome</div>, <div class="source">Cell</div> , <div class="year">2008</div>, vol. <div class="volume">132</div> (pg. <div class="fpage">311</div>-<div class="lpage">22</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.cell.2007.12.014" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.cell.2007.12.014</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-resolution%20mapping%20and%20characterization%20of%20open%20chromatin%20across%20the%20genome&amp;author=A.P.%20Boyle&amp;author=S.%20Davis&amp;author=H.P.%20Shulha&amp;publication_year=2008&amp;journal=Cell&amp;volume=132&amp;pages=311-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2007.12.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2007.12.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2007.12.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18243105" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ016C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ016C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barski</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Cuddapah</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Cui</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">High-resolution profiling of histone methylations in the human genome</div>, <div class="source">Cell</div> , <div class="year">2007</div>, vol. <div class="volume">129</div> (pg. <div class="fpage">823</div>-<div class="lpage">37</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.cell.2007.05.009" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.cell.2007.05.009</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-resolution%20profiling%20of%20histone%20methylations%20in%20the%20human%20genome&amp;author=A.%20Barski&amp;author=S.%20Cuddapah&amp;author=K.%20Cui&amp;publication_year=2007&amp;journal=Cell&amp;volume=129&amp;pages=823-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2007.05.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2007.05.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2007.05.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17512414" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Minoru Ko</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2010. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=397387&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/4/10.1093/dnares/dsq016/2/dsq016-dsq016supp.pdf?Expires=2147483647&amp;Signature=nKuaH~RImDJMUuKE9qjLhAAIY7E8NXWefunO9T-Cu1m0geQpa84vDLS~n0pT8nY6p5t3qRr0T31tqI0i2TWO14kUgOzdEh0S-C6i8z1wZa2UtBMSbgIGMKrtxVPcwfEibrXlsuO1oiO9v2cKXFAOCU5qcyMvV-OrmUyKlICqVfC6o9AN8NZlzO6JzsnGG-m3kB-82it~hv8CTxY2drH0Syxj6jYEB~PnpxJV96~PWmS4zCrpQwbzrhHaeuQg6p71KJDAWa4jIyZ7Q~FCbFowPizZm6Xd7Xf~FYUjI9-OPnpPYu~iLiBy10ATjC3rSB7ezPBWLb4ASY32meqySPR1Gg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">286</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000281469400004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">27</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsq016"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="397387"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000281469400004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f17%2f4%2f245%2f397387">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000281469400004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (27)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F17%2F4%2F245%2F397387" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/397387'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Comprehensive Analysis of the Palindromic Motif TCTCGCGAGA: A Regulatory Element of the HNRNPK Promoter" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsq016");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f17%2f4%2f245%2f397387">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

