

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/3/297/4838783","@type":"ScholarlyArticle","name":"Complete genome sequence and analysis of the industrial  Saccharomyces cerevisiae  strain N85 used in Chinese rice wine production","datePublished":"2018-02-05","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2018-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsy002","keywords":["rice wine yeast","genome sequence","annotation","transcriptomics"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/3/297/4838783","author":[{"name":"Zhang, Weiping","affiliation":"School of Biotechnology, Jiangnan University, Wuxi, Jiangsu 214443, China","@type":"Person"},{"name":"Li, Yudong","affiliation":"School of Biotechnology, Jiangnan University, Wuxi, Jiangsu 214443, China  Department of Bioengineering, School of Food Sciences and Biotechnology, Zhejiang Gongshang University, Hangzhou 310018, China","@type":"Person"},{"name":"Chen, Yiwang","affiliation":"Department of Bioengineering, School of Food Sciences and Biotechnology, Zhejiang Gongshang University, Hangzhou 310018, China","@type":"Person"},{"name":"Xu, Sha","affiliation":"School of Biotechnology, Jiangnan University, Wuxi, Jiangsu 214443, China","@type":"Person"},{"name":"Du, Guocheng","affiliation":"School of Biotechnology, Jiangnan University, Wuxi, Jiangsu 214443, China","@type":"Person"},{"name":"Shi, Huidong","affiliation":"Georgia Cancer Center, Augusta University, Augusta, GA 30912, USA","@type":"Person"},{"name":"Zhou, Jingwen","affiliation":"School of Biotechnology, Jiangnan University, Wuxi, Jiangsu 214443, China","@type":"Person"},{"name":"Chen, Jian","affiliation":"School of Biotechnology, Jiangnan University, Wuxi, Jiangsu 214443, China","@type":"Person"}],"description":"Abstract.  Chinese rice wine is a popular traditional alcoholic beverage in China, while its brewing processes have rarely been explored. We herein report the f","pageStart":"297","pageEnd":"306","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/3/1/m_dnares_25_3cover.png?Expires=2147483647&Signature=zOskx939n215x6LsaNaPxmag9A6Tt6e7bP~Kt5lfxwaH8FjX1dKLUJ0mdj-Sm0uq9hmCpw3OAnAQFLS-GTY4CsXXX4aRhm5J1IbPv51CcTKXj828y94I4tx-BuFB-EA4WwEvEot4P8hDcnFzu~xWV52k2Sm6IHcCABqAv5wSWve6oMrdsQXhpN0srJqhN5UIcLjzws~g9qv24XhgtRb~Nw7X9IiEo1EM5V47od3bkv5yHlkUT1xmh3tJ4-ZjHw-HyAXqIp8eeHds4Cfjy9xqh8e3n5ynt516a1ZMF0hjA3wYH5X0-y3lqTHnWmgUvQv1L6dLXdRWhki~KqxTetYsJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Complete genome sequence and analysis of the industrial  Saccharomyces cerevisiae  strain N85 used in Chinese rice wine production","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/3/1/m_dnares_25_3cover.png?Expires=2147483647&Signature=zOskx939n215x6LsaNaPxmag9A6Tt6e7bP~Kt5lfxwaH8FjX1dKLUJ0mdj-Sm0uq9hmCpw3OAnAQFLS-GTY4CsXXX4aRhm5J1IbPv51CcTKXj828y94I4tx-BuFB-EA4WwEvEot4P8hDcnFzu~xWV52k2Sm6IHcCABqAv5wSWve6oMrdsQXhpN0srJqhN5UIcLjzws~g9qv24XhgtRb~Nw7X9IiEo1EM5V47od3bkv5yHlkUT1xmh3tJ4-ZjHw-HyAXqIp8eeHds4Cfjy9xqh8e3n5ynt516a1ZMF0hjA3wYH5X0-y3lqTHnWmgUvQv1L6dLXdRWhki~KqxTetYsJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Complete genome sequence and analysis of the industrial  Saccharomyces cerevisiae  strain N85 used in Chinese rice wine production" />
    <meta property="og:description" content="Abstract.  Chinese rice wine is a popular traditional alcoholic beverage in China, while its brewing processes have rarely been explored. We herein report the f" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/3/297/4838783" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2018-02-05" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Zhang, Weiping" /><meta name="citation_author_institution" content="School of Biotechnology, Jiangnan University, Wuxi, Jiangsu 214443, China" /><meta name="citation_author" content="Li, Yudong" /><meta name="citation_author_institution" content="School of Biotechnology, Jiangnan University, Wuxi, Jiangsu 214443, China" /><meta name="citation_author_institution" content="Department of Bioengineering, School of Food Sciences and Biotechnology, Zhejiang Gongshang University, Hangzhou 310018, China" /><meta name="citation_author" content="Chen, Yiwang" /><meta name="citation_author_institution" content="Department of Bioengineering, School of Food Sciences and Biotechnology, Zhejiang Gongshang University, Hangzhou 310018, China" /><meta name="citation_author" content="Xu, Sha" /><meta name="citation_author_institution" content="School of Biotechnology, Jiangnan University, Wuxi, Jiangsu 214443, China" /><meta name="citation_author" content="Du, Guocheng" /><meta name="citation_author_institution" content="School of Biotechnology, Jiangnan University, Wuxi, Jiangsu 214443, China" /><meta name="citation_author" content="Shi, Huidong" /><meta name="citation_author_institution" content="Georgia Cancer Center, Augusta University, Augusta, GA 30912, USA" /><meta name="citation_author" content="Zhou, Jingwen" /><meta name="citation_author_institution" content="School of Biotechnology, Jiangnan University, Wuxi, Jiangsu 214443, China" /><meta name="citation_author" content="Chen, Jian" /><meta name="citation_author_institution" content="School of Biotechnology, Jiangnan University, Wuxi, Jiangsu 214443, China" /><meta name="citation_title" content="Complete genome sequence and analysis of the industrial Saccharomyces cerevisiae strain N85 used in Chinese rice wine production" /><meta name="citation_firstpage" content="297" /><meta name="citation_lastpage" content="306" /><meta name="citation_doi" content="10.1093/dnares/dsy002" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2018/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Fermented beverages of pre- and proto-historic China;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2004;  citation_volume=101;  citation_pages=17593-8; " /><meta name="citation_reference" content="citation_title=Functional constituents and health function of Chinese rice wine;  citation_journal_title=Liquor Making;  citation_year=2008;  citation_volume=35;  citation_pages=14-6; " /><meta name="citation_reference" content="citation_title=Progress in preventing the accumulation of ethyl carbamate in alcoholic beverages;  citation_journal_title=Trends Food Sci. Tech;  citation_year=2013;  citation_volume=32;  citation_pages=97-107; " /><meta name="citation_reference" content="citation_title=Constitutive expression of the DUR1, 2 gene in an industrial yeast strain to minimize ethyl carbamate production during Chinese rice wine fermentation;  citation_journal_title=FEMS Microbiol. Lett;  citation_year=2016;  citation_volume=363;  citation_pages=fnv214" /><meta name="citation_reference" content="citation_title=Functional enhancement of Sake yeast strains to minimize the production of ethyl carbamate in Sake wine;  citation_journal_title=J. Appl. Microbiol.;  citation_year=2010;  citation_volume=109;  citation_pages=963-73; " /><meta name="citation_reference" content="citation_title=Comparative proteomic analysis of Saccharomyces cerevisiae under different nitrogen sources;  citation_journal_title=J. Proteomics;  citation_year=2014;  citation_volume=101;  citation_pages=102-12; " /><meta name="citation_reference" content="citation_title=Nitrogen regulation involved in the accumulation of urea in Saccharomyces cerevisiae;  citation_journal_title=Yeast;  citation_year=2013;  citation_volume=30;  citation_pages=437-47; " /><meta name="citation_reference" content="citation_title=Metabolic engineering of the regulators in nitrogen catabolite repression to reduce the production of ethyl carbamate in a model rice wine system;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2014;  citation_volume=80;  citation_pages=392-8; " /><meta name="citation_reference" content="citation_title=Nitrogen catabolite repression in yeasts and filamentous fungi;  citation_journal_title=Adv. Microb. Physiol;  citation_year=1985;  citation_volume=26;  citation_pages=1-88; " /><meta name="citation_reference" content="citation_title=Nitrogen catabolite repression in Saccharomyces cerevisiae during wine fermentations;  citation_journal_title=FEMS Yeast Res;  citation_year=2004;  citation_volume=4;  citation_pages=625-32; " /><meta name="citation_reference" content="citation_title=General amino acid control and 14-3-3 proteins Bmh1/2 are required for nitrogen catabolite repression-sensitive regulation of Gln3 and Gat1 localization;  citation_journal_title=Genetics;  citation_year=2017;  citation_volume=205;  citation_pages=633-55; " /><meta name="citation_reference" content="citation_title=The modification of Gat1p in nitrogen catabolite repression to enhance non-preferred nitrogen utilization in Saccharomyces cerevisiae;  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=6;  citation_pages=21603" /><meta name="citation_reference" content="citation_title=Nitrogen availability and TOR regulate the Snf1 protein kinase in Saccharomyces cerevisiae;  citation_journal_title=Eukaryot. Cell;  citation_year=2006;  citation_volume=5;  citation_pages=1831-7; " /><meta name="citation_reference" content="citation_title=Amino-acid-induced signalling via the SPS-sensing pathway in yeast;  citation_journal_title=Biochem. Soc. Trans;  citation_year=2009;  citation_volume=37;  citation_pages=242-7; " /><meta name="citation_reference" content="citation_title=Genomic evolution of Saccharomyces cerevisiae under Chinese rice wine fermentation;  citation_journal_title=Genome Biol. Evol;  citation_year=2014;  citation_volume=6;  citation_pages=2516-26; " /><meta name="citation_reference" content="citation_title=Coming of age: ten years of next-generation sequencing technologies;  citation_journal_title=Nat. Rev. Genet;  citation_year=2016;  citation_volume=17;  citation_pages=333-51; " /><meta name="citation_reference" content="citation_title=High-quality draft assemblies of mammalian genomes from massively parallel sequence data;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2011;  citation_volume=108;  citation_pages=1513-8; " /><meta name="citation_reference" content="citation_title=Advantages of genome sequencing by long-read sequencer using SMRT technology in medical area;  citation_journal_title=Hum. Cell;  citation_year=2017;  citation_volume=30;  citation_pages=149-61; " /><meta name="citation_reference" content="citation_title=Whole genome complete resequencing of Bacillus subtilis natto by combining long reads with high-quality short reads;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e109999" /><meta name="citation_reference" content="citation_title=A5-miseq: an updated pipeline to assemble microbial genomes from Illumina MiSeq data;  citation_journal_title=Bioinformatics;  citation_year=2015;  citation_volume=31;  citation_pages=587-9; " /><meta name="citation_reference" content="citation_title=SSPACE-LongRead: scaffolding bacterial draft genomes using long read sequence information;  citation_journal_title=BMC Bioinformatics;  citation_year=2014;  citation_volume=15;  citation_pages=211" /><meta name="citation_reference" content="citation_title=SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing;  citation_journal_title=J. Comput. Biol;  citation_year=2012;  citation_volume=19;  citation_pages=455-77; " /><meta name="citation_reference" content="citation_title=Assembling large genomes with single-molecule sequencing and locality-sensitive hashing;  citation_journal_title=Nat. Biotechnol;  citation_year=2015;  citation_volume=33;  citation_pages=623-30; " /><meta name="citation_reference" content="citation_title=Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data;  citation_journal_title=Nat. Methods.;  citation_year=2013;  citation_volume=10;  citation_pages=563-9; " /><meta name="citation_reference" content="citation_title=Using native and syntenically mapped cDNA alignments to improve de novo gene finding;  citation_journal_title=Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=637-44; " /><meta name="citation_reference" content="citation_title=Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks;  citation_journal_title=Nat. Protoc;  citation_year=2012;  citation_volume=7;  citation_pages=562-78; " /><meta name="citation_reference" content="Hoff K.  2014 , Incorporating RNA-Seq into AUGUSTUS with TOPHAT. Avialable at: http://bioinf.uni-greifswald.de/bioinf/wiki/pmwiki.php? n=IncorporatingRNAseq.Tophat (19 January 2018, date last accessed). " /><meta name="citation_reference" content="citation_title=Trinity: reconstructing a full-length transcriptome without a genome from RNA-Seq data;  citation_journal_title=Nat. Biotechnol;  citation_year=2011;  citation_volume=29;  citation_pages=644-52; " /><meta name="citation_reference" content="citation_title=Automated generation of heuristics for biological sequence comparison;  citation_journal_title=BMC Bioinformatics;  citation_year=2005;  citation_volume=6;  citation_pages=31" /><meta name="citation_reference" content="citation_title=SpliceGrapher: detecting patterns of alternative splicing from RNA-Seq data in the context of gene models and EST data;  citation_journal_title=Genome Biol;  citation_year=2012;  citation_volume=13;  citation_pages=R4" /><meta name="citation_reference" content="citation_title=Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration;  citation_journal_title=Brief Bioinform;  citation_year=2013;  citation_volume=14;  citation_pages=178-92; " /><meta name="citation_reference" content="Li H.  2013 , Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXiv preprint arXiv: 1303.3997. " /><meta name="citation_reference" content="citation_title=BEDTools: a flexible suite of utilities for comparing genomic features;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=841-2; " /><meta name="citation_reference" content="citation_title=Integrative analysis of genome-wide loss of heterozygosity and monoallelic expression at nucleotide resolution reveals disrupted pathways in triple-negative breast cancer;  citation_journal_title=Genome Res;  citation_year=2012;  citation_volume=22;  citation_pages=1995-2007; " /><meta name="citation_reference" content="citation_title=Hybrid error correction and de novo assembly of single-molecule sequencing reads;  citation_journal_title=Nat. Biotechnol;  citation_year=2012;  citation_volume=30;  citation_pages=693-700; " /><meta name="citation_reference" content="citation_title=Computational methods for transcriptome annotation and quantification using RNA-seq;  citation_journal_title=Nat. Methods;  citation_year=2011;  citation_volume=8;  citation_pages=469-77; " /><meta name="citation_reference" content="citation_title=Molecular cloning and evolutionary analysis of the HOG-signaling pathway genes from Saccharomyces cerevisiae rice wine isolates;  citation_journal_title=Biochem. Genet;  citation_year=2013;  citation_volume=51;  citation_pages=296-305; " /><meta name="citation_reference" content="citation_title=Beer with reduced ethanol content produced using Saccharomyces cerevisiae yeasts deficient in various tricarboxylic acid cycle enzymes;  citation_journal_title=J. Inst. Brew;  citation_year=2008;  citation_volume=114;  citation_pages=97-101; " /><meta name="citation_reference" content="citation_title=Genetic determinants of volatile-thiol release by Saccharomyces cerevisiae during wine fermentation;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2005;  citation_volume=71;  citation_pages=5420-6; " /><meta name="citation_reference" content="citation_title=Antioxidant properties of Chinese yellow wine, its concentrate and volatiles;  citation_journal_title=LWT Food Sci. Technol;  citation_year=2006;  citation_volume=39;  citation_pages=111-7; " /><meta name="citation_reference" content="citation_title=Innovation Chinese rice wine brewing technology by bi-acidification to exclude rice soaking process;  citation_journal_title=J. Biosci. Bioeng;  citation_year=2017;  citation_volume=123;  citation_pages=460-5; " /><meta name="citation_reference" content="citation_title=Ethyl carbamate formation regulated by Saccharomyces cerevisiae ZJU in the processing of Chinese yellow rice wine;  citation_journal_title=Int. J. Food Sci. Technol;  citation_year=2015;  citation_volume=50;  citation_pages=626-32; " /><meta name="citation_reference" content="citation_title=Improving PacBio long read accuracy by short read alignment;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e46679" /><meta name="citation_reference" content="citation_title=Whole-genome sequencing of sake yeast Saccharomyces cerevisiae Kyokai no. 7;  citation_journal_title=DNA Res;  citation_year=2011;  citation_volume=18;  citation_pages=423-34; " /><meta name="citation_reference" content="citation_title=Genome regulation by long noncoding RNAs;  citation_journal_title=Annu. Rev. Biochem;  citation_year=2012;  citation_volume=81;  citation_pages=145-66; " /><meta name="citation_reference" content="citation_title=Long noncoding RNAs and microRNAs in cardiovascular pathophysiology;  citation_journal_title=Circ. Res;  citation_year=2015;  citation_volume=116;  citation_pages=751-62; " /><meta name="citation_reference" content="citation_title=Constitutive and nitrogen catabolite repression-sensitive production of Gat1 isoforms;  citation_journal_title=J. Biol. Chem;  citation_year=2014;  citation_volume=289;  citation_pages=2918-33; " /><meta name="citation_reference" content="citation_title=Domestication and divergence of Saccharomyces cerevisiae beer yeasts;  citation_journal_title=Cell;  citation_year=2016;  citation_volume=166;  citation_pages=1397" /><meta name="citation_reference" content="citation_title=Analysis of the Saccharomyces cerevisiae pan-genome reveals a pool of copy number variants distributed in diverse yeast strains from differing industrial environments;  citation_journal_title=Genome Res;  citation_year=2012;  citation_volume=22;  citation_pages=908-24; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/3/297/25072170/dsy002.pdf" /><meta name="description" content="Abstract.  Chinese rice wine is a popular traditional alcoholic beverage in China, while its brewing processes have rarely been explored. We herein report the f" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/3/297/4838783" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/3/297/4838783" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/3/297/4838783");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753&authnips=128.235.12.133&doi=10.1093/dnares/dsy002';
    </script>
<title>
	Complete genome sequence and analysis of the industrial Saccharomyces cerevisiae strain N85 used in Chinese rice wine production | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4838783" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Be14PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJaaGFuZywgV2VpcGluZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNjaG9vbCBvZiBCaW90ZWNobm9sb2d5LCBKaWFuZ25hbiBVbml2ZXJzaXR5LCBXdXhpLCBKaWFuZ3N1IDIxNDQ0MywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMaSwgWXVkb25nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU2Nob29sIG9mIEJpb3RlY2hub2xvZ3ksIEppYW5nbmFuIFVuaXZlcnNpdHksIFd1eGksIEppYW5nc3UgMjE0NDQzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvZW5naW5lZXJpbmcsIFNjaG9vbCBvZiBGb29kIFNjaWVuY2VzIGFuZCBCaW90ZWNobm9sb2d5LCBaaGVqaWFuZyBHb25nc2hhbmcgVW5pdmVyc2l0eSwgSGFuZ3pob3UgMzEwMDE4LCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNoZW4sIFlpd2FuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvZW5naW5lZXJpbmcsIFNjaG9vbCBvZiBGb29kIFNjaWVuY2VzIGFuZCBCaW90ZWNobm9sb2d5LCBaaGVqaWFuZyBHb25nc2hhbmcgVW5pdmVyc2l0eSwgSGFuZ3pob3UgMzEwMDE4LCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ilh1LCBTaGEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTY2hvb2wgb2YgQmlvdGVjaG5vbG9neSwgSmlhbmduYW4gVW5pdmVyc2l0eSwgV3V4aSwgSmlhbmdzdSAyMTQ0NDMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRHUsIEd1b2NoZW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU2Nob29sIG9mIEJpb3RlY2hub2xvZ3ksIEppYW5nbmFuIFVuaXZlcnNpdHksIFd1eGksIEppYW5nc3UgMjE0NDQzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNoaSwgSHVpZG9uZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikdlb3JnaWEgQ2FuY2VyIENlbnRlciwgQXVndXN0YSBVbml2ZXJzaXR5LCBBdWd1c3RhLCBHQSAzMDkxMiwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWmhvdSwgSmluZ3dlbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNjaG9vbCBvZiBCaW90ZWNobm9sb2d5LCBKaWFuZ25hbiBVbml2ZXJzaXR5LCBXdXhpLCBKaWFuZ3N1IDIxNDQ0MywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDaGVuLCBKaWFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU2Nob29sIG9mIEJpb3RlY2hub2xvZ3ksIEppYW5nbmFuIFVuaXZlcnNpdHksIFd1eGksIEppYW5nc3UgMjE0NDQzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQ29tcGxldGUgZ2Vub21lIHNlcXVlbmNlIGFuZCBhbmFseXNpcyBvZiB0aGUgaW5kdXN0cmlhbCBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWUgc3RyYWluIE44NSB1c2VkIGluIENoaW5lc2UgcmljZSB3aW5lIHByb2R1Y3Rpb24iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyOTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjMwNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzeTAwMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE4LzA2LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmVybWVudGVkIGJldmVyYWdlcyBvZiBwcmUtIGFuZCBwcm90by1oaXN0b3JpYyBDaGluYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVLiBTLiBBOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTE3NTkzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bmN0aW9uYWwgY29uc3RpdHVlbnRzIGFuZCBoZWFsdGggZnVuY3Rpb24gb2YgQ2hpbmVzZSByaWNlIHdpbmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUxpcXVvciBNYWtpbmc7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz0xNC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm9ncmVzcyBpbiBwcmV2ZW50aW5nIHRoZSBhY2N1bXVsYXRpb24gb2YgZXRoeWwgY2FyYmFtYXRlIGluIGFsY29ob2xpYyBiZXZlcmFnZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBGb29kIFNjaS4gVGVjaDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTk3LTEwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RpdHV0aXZlIGV4cHJlc3Npb24gb2YgdGhlIERVUjEsIDIgZ2VuZSBpbiBhbiBpbmR1c3RyaWFsIHllYXN0IHN0cmFpbiB0byBtaW5pbWl6ZSBldGh5bCBjYXJiYW1hdGUgcHJvZHVjdGlvbiBkdXJpbmcgQ2hpbmVzZSByaWNlIHdpbmUgZmVybWVudGF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRU1TIE1pY3JvYmlvbC4gTGV0dDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNjM7ICBjaXRhdGlvbl9wYWdlcz1mbnYyMTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5jdGlvbmFsIGVuaGFuY2VtZW50IG9mIFNha2UgeWVhc3Qgc3RyYWlucyB0byBtaW5pbWl6ZSB0aGUgcHJvZHVjdGlvbiBvZiBldGh5bCBjYXJiYW1hdGUgaW4gU2FrZSB3aW5lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBBcHBsLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEwOTsgIGNpdGF0aW9uX3BhZ2VzPTk2My03MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgcHJvdGVvbWljIGFuYWx5c2lzIG9mIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZSB1bmRlciBkaWZmZXJlbnQgbml0cm9nZW4gc291cmNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gUHJvdGVvbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDE7ICBjaXRhdGlvbl9wYWdlcz0xMDItMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5pdHJvZ2VuIHJlZ3VsYXRpb24gaW52b2x2ZWQgaW4gdGhlIGFjY3VtdWxhdGlvbiBvZiB1cmVhIGluIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9WWVhc3Q7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz00MzctNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ldGFib2xpYyBlbmdpbmVlcmluZyBvZiB0aGUgcmVndWxhdG9ycyBpbiBuaXRyb2dlbiBjYXRhYm9saXRlIHJlcHJlc3Npb24gdG8gcmVkdWNlIHRoZSBwcm9kdWN0aW9uIG9mIGV0aHlsIGNhcmJhbWF0ZSBpbiBhIG1vZGVsIHJpY2Ugd2luZSBzeXN0ZW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFwcGwuIEVudmlyb24uIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT04MDsgIGNpdGF0aW9uX3BhZ2VzPTM5Mi04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OaXRyb2dlbiBjYXRhYm9saXRlIHJlcHJlc3Npb24gaW4geWVhc3RzIGFuZCBmaWxhbWVudG91cyBmdW5naTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QWR2LiBNaWNyb2IuIFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTE5ODU7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz0xLTg4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OaXRyb2dlbiBjYXRhYm9saXRlIHJlcHJlc3Npb24gaW4gU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlIGR1cmluZyB3aW5lIGZlcm1lbnRhdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZFTVMgWWVhc3QgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz02MjUtMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmVyYWwgYW1pbm8gYWNpZCBjb250cm9sIGFuZCAxNC0zLTMgcHJvdGVpbnMgQm1oMS8yIGFyZSByZXF1aXJlZCBmb3Igbml0cm9nZW4gY2F0YWJvbGl0ZSByZXByZXNzaW9uLXNlbnNpdGl2ZSByZWd1bGF0aW9uIG9mIEdsbjMgYW5kIEdhdDEgbG9jYWxpemF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDU7ICBjaXRhdGlvbl9wYWdlcz02MzMtNTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtb2RpZmljYXRpb24gb2YgR2F0MXAgaW4gbml0cm9nZW4gY2F0YWJvbGl0ZSByZXByZXNzaW9uIHRvIGVuaGFuY2Ugbm9uLXByZWZlcnJlZCBuaXRyb2dlbiB1dGlsaXphdGlvbiBpbiBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gUmVwOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0yMTYwMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5pdHJvZ2VuIGF2YWlsYWJpbGl0eSBhbmQgVE9SIHJlZ3VsYXRlIHRoZSBTbmYxIHByb3RlaW4ga2luYXNlIGluIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXVrYXJ5b3QuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTE4MzEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW1pbm8tYWNpZC1pbmR1Y2VkIHNpZ25hbGxpbmcgdmlhIHRoZSBTUFMtc2Vuc2luZyBwYXRod2F5IGluIHllYXN0OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9jaGVtLiBTb2MuIFRyYW5zOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9MjQyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgZXZvbHV0aW9uIG9mIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZSB1bmRlciBDaGluZXNlIHJpY2Ugd2luZSBmZXJtZW50YXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0yNTE2LTI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21pbmcgb2YgYWdlOiB0ZW4geWVhcnMgb2YgbmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmcgdGVjaG5vbG9naWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0zMzMtNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2gtcXVhbGl0eSBkcmFmdCBhc3NlbWJsaWVzIG9mIG1hbW1hbGlhbiBnZW5vbWVzIGZyb20gbWFzc2l2ZWx5IHBhcmFsbGVsIHNlcXVlbmNlIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVS4gUy4gQTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDg7ICBjaXRhdGlvbl9wYWdlcz0xNTEzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFkdmFudGFnZXMgb2YgZ2Vub21lIHNlcXVlbmNpbmcgYnkgbG9uZy1yZWFkIHNlcXVlbmNlciB1c2luZyBTTVJUIHRlY2hub2xvZ3kgaW4gbWVkaWNhbCBhcmVhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1IdW0uIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0xNDktNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdob2xlIGdlbm9tZSBjb21wbGV0ZSByZXNlcXVlbmNpbmcgb2YgQmFjaWxsdXMgc3VidGlsaXMgbmF0dG8gYnkgY29tYmluaW5nIGxvbmcgcmVhZHMgd2l0aCBoaWdoLXF1YWxpdHkgc2hvcnQgcmVhZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1lMTA5OTk5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QTUtbWlzZXE6IGFuIHVwZGF0ZWQgcGlwZWxpbmUgdG8gYXNzZW1ibGUgbWljcm9iaWFsIGdlbm9tZXMgZnJvbSBJbGx1bWluYSBNaVNlcSBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTU4Ny05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TU1BBQ0UtTG9uZ1JlYWQ6IHNjYWZmb2xkaW5nIGJhY3RlcmlhbCBkcmFmdCBnZW5vbWVzIHVzaW5nIGxvbmcgcmVhZCBzZXF1ZW5jZSBpbmZvcm1hdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MjExIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U1BBZGVzOiBhIG5ldyBnZW5vbWUgYXNzZW1ibHkgYWxnb3JpdGhtIGFuZCBpdHMgYXBwbGljYXRpb25zIHRvIHNpbmdsZS1jZWxsIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIENvbXB1dC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTQ1NS03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzZW1ibGluZyBsYXJnZSBnZW5vbWVzIHdpdGggc2luZ2xlLW1vbGVjdWxlIHNlcXVlbmNpbmcgYW5kIGxvY2FsaXR5LXNlbnNpdGl2ZSBoYXNoaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz02MjMtMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vbmh5YnJpZCwgZmluaXNoZWQgbWljcm9iaWFsIGdlbm9tZSBhc3NlbWJsaWVzIGZyb20gbG9uZy1yZWFkIFNNUlQgc2VxdWVuY2luZyBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIE1ldGhvZHMuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NTYzLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVzaW5nIG5hdGl2ZSBhbmQgc3ludGVuaWNhbGx5IG1hcHBlZCBjRE5BIGFsaWdubWVudHMgdG8gaW1wcm92ZSBkZSBub3ZvIGdlbmUgZmluZGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz02MzctNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpZmZlcmVudGlhbCBnZW5lIGFuZCB0cmFuc2NyaXB0IGV4cHJlc3Npb24gYW5hbHlzaXMgb2YgUk5BLXNlcSBleHBlcmltZW50cyB3aXRoIFRvcEhhdCBhbmQgQ3VmZmxpbmtzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFByb3RvYzsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9NTYyLTc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJIb2ZmIEsuICAyMDE0ICwgSW5jb3Jwb3JhdGluZyBSTkEtU2VxIGludG8gQVVHVVNUVVMgd2l0aCBUT1BIQVQuIEF2aWFsYWJsZSBhdDogaHR0cDovL2Jpb2luZi51bmktZ3JlaWZzd2FsZC5kZS9iaW9pbmYvd2lraS9wbXdpa2kucGhwPyBuPUluY29ycG9yYXRpbmdSTkFzZXEuVG9waGF0ICgxOSBKYW51YXJ5IDIwMTgsIGRhdGUgbGFzdCBhY2Nlc3NlZCkuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyaW5pdHk6IHJlY29uc3RydWN0aW5nIGEgZnVsbC1sZW5ndGggdHJhbnNjcmlwdG9tZSB3aXRob3V0IGEgZ2Vub21lIGZyb20gUk5BLVNlcSBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz02NDQtNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUF1dG9tYXRlZCBnZW5lcmF0aW9uIG9mIGhldXJpc3RpY3MgZm9yIGJpb2xvZ2ljYWwgc2VxdWVuY2UgY29tcGFyaXNvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0zMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwbGljZUdyYXBoZXI6IGRldGVjdGluZyBwYXR0ZXJucyBvZiBhbHRlcm5hdGl2ZSBzcGxpY2luZyBmcm9tIFJOQS1TZXEgZGF0YSBpbiB0aGUgY29udGV4dCBvZiBnZW5lIG1vZGVscyBhbmQgRVNUIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9UjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlZ3JhdGl2ZSBHZW5vbWljcyBWaWV3ZXIgKElHVik6IGhpZ2gtcGVyZm9ybWFuY2UgZ2Vub21pY3MgZGF0YSB2aXN1YWxpemF0aW9uIGFuZCBleHBsb3JhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QnJpZWYgQmlvaW5mb3JtOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTc4LTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJMaSBILiAgMjAxMyAsIEFsaWduaW5nIHNlcXVlbmNlIHJlYWRzLCBjbG9uZSBzZXF1ZW5jZXMgYW5kIGFzc2VtYmx5IGNvbnRpZ3Mgd2l0aCBCV0EtTUVNLiBhclhpdiBwcmVwcmludCBhclhpdjogMTMwMy4zOTk3LiAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CRURUb29sczogYSBmbGV4aWJsZSBzdWl0ZSBvZiB1dGlsaXRpZXMgZm9yIGNvbXBhcmluZyBnZW5vbWljIGZlYXR1cmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTg0MS0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlZ3JhdGl2ZSBhbmFseXNpcyBvZiBnZW5vbWUtd2lkZSBsb3NzIG9mIGhldGVyb3p5Z29zaXR5IGFuZCBtb25vYWxsZWxpYyBleHByZXNzaW9uIGF0IG51Y2xlb3RpZGUgcmVzb2x1dGlvbiByZXZlYWxzIGRpc3J1cHRlZCBwYXRod2F5cyBpbiB0cmlwbGUtbmVnYXRpdmUgYnJlYXN0IGNhbmNlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTE5OTUtMjAwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SHlicmlkIGVycm9yIGNvcnJlY3Rpb24gYW5kIGRlIG5vdm8gYXNzZW1ibHkgb2Ygc2luZ2xlLW1vbGVjdWxlIHNlcXVlbmNpbmcgcmVhZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTY5My03MDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXB1dGF0aW9uYWwgbWV0aG9kcyBmb3IgdHJhbnNjcmlwdG9tZSBhbm5vdGF0aW9uIGFuZCBxdWFudGlmaWNhdGlvbiB1c2luZyBSTkEtc2VxOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTQ2OS03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGNsb25pbmcgYW5kIGV2b2x1dGlvbmFyeSBhbmFseXNpcyBvZiB0aGUgSE9HLXNpZ25hbGluZyBwYXRod2F5IGdlbmVzIGZyb20gU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlIHJpY2Ugd2luZSBpc29sYXRlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvY2hlbS4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NTE7ICBjaXRhdGlvbl9wYWdlcz0yOTYtMzA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CZWVyIHdpdGggcmVkdWNlZCBldGhhbm9sIGNvbnRlbnQgcHJvZHVjZWQgdXNpbmcgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlIHllYXN0cyBkZWZpY2llbnQgaW4gdmFyaW91cyB0cmljYXJib3h5bGljIGFjaWQgY3ljbGUgZW56eW1lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gSW5zdC4gQnJldzsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMTQ7ICBjaXRhdGlvbl9wYWdlcz05Ny0xMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgZGV0ZXJtaW5hbnRzIG9mIHZvbGF0aWxlLXRoaW9sIHJlbGVhc2UgYnkgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlIGR1cmluZyB3aW5lIGZlcm1lbnRhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXBwbC4gRW52aXJvbi4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTcxOyAgY2l0YXRpb25fcGFnZXM9NTQyMC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbnRpb3hpZGFudCBwcm9wZXJ0aWVzIG9mIENoaW5lc2UgeWVsbG93IHdpbmUsIGl0cyBjb25jZW50cmF0ZSBhbmQgdm9sYXRpbGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1MV1QgRm9vZCBTY2kuIFRlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz0xMTEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5ub3ZhdGlvbiBDaGluZXNlIHJpY2Ugd2luZSBicmV3aW5nIHRlY2hub2xvZ3kgYnkgYmktYWNpZGlmaWNhdGlvbiB0byBleGNsdWRlIHJpY2Ugc29ha2luZyBwcm9jZXNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCaW9zY2kuIEJpb2VuZzsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMjM7ICBjaXRhdGlvbl9wYWdlcz00NjAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXRoeWwgY2FyYmFtYXRlIGZvcm1hdGlvbiByZWd1bGF0ZWQgYnkgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlIFpKVSBpbiB0aGUgcHJvY2Vzc2luZyBvZiBDaGluZXNlIHllbGxvdyByaWNlIHdpbmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUludC4gSi4gRm9vZCBTY2kuIFRlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NTA7ICBjaXRhdGlvbl9wYWdlcz02MjYtMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcHJvdmluZyBQYWNCaW8gbG9uZyByZWFkIGFjY3VyYWN5IGJ5IHNob3J0IHJlYWQgYWxpZ25tZW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTQ2Njc5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hvbGUtZ2Vub21lIHNlcXVlbmNpbmcgb2Ygc2FrZSB5ZWFzdCBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWUgS3lva2FpIG5vLiA3OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9NDIzLTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgcmVndWxhdGlvbiBieSBsb25nIG5vbmNvZGluZyBSTkFzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIEJpb2NoZW07ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9ODE7ICBjaXRhdGlvbl9wYWdlcz0xNDUtNjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvbmcgbm9uY29kaW5nIFJOQXMgYW5kIG1pY3JvUk5BcyBpbiBjYXJkaW92YXNjdWxhciBwYXRob3BoeXNpb2xvZ3k7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNpcmMuIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTY7ICBjaXRhdGlvbl9wYWdlcz03NTEtNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0aXR1dGl2ZSBhbmQgbml0cm9nZW4gY2F0YWJvbGl0ZSByZXByZXNzaW9uLXNlbnNpdGl2ZSBwcm9kdWN0aW9uIG9mIEdhdDEgaXNvZm9ybXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJpb2wuIENoZW07ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9Mjg5OyAgY2l0YXRpb25fcGFnZXM9MjkxOC0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RG9tZXN0aWNhdGlvbiBhbmQgZGl2ZXJnZW5jZSBvZiBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWUgYmVlciB5ZWFzdHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTY2OyAgY2l0YXRpb25fcGFnZXM9MTM5NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIHRoZSBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWUgcGFuLWdlbm9tZSByZXZlYWxzIGEgcG9vbCBvZiBjb3B5IG51bWJlciB2YXJpYW50cyBkaXN0cmlidXRlZCBpbiBkaXZlcnNlIHllYXN0IHN0cmFpbnMgZnJvbSBkaWZmZXJpbmcgaW5kdXN0cmlhbCBlbnZpcm9ubWVudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz05MDgtMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNS8zLzI5Ny8yNTA3MjE3MC9kc3kwMDIucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIENoaW5lc2UgcmljZSB3aW5lIGlzIGEgcG9wdWxhciB0cmFkaXRpb25hbCBhbGNvaG9saWMgYmV2ZXJhZ2UgaW4gQ2hpbmEsIHdoaWxlIGl0cyBicmV3aW5nIHByb2Nlc3NlcyBoYXZlIHJhcmVseSBiZWVuIGV4cGxvcmVkLiBXZSBoZXJlaW4gcmVwb3J0IHRoZSBmIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00ODM4NzgzJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZAeNejp8NDBMsTYspys/BnxxAjBO" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f3%2f297%2f4838783" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f3%2f297%2f4838783" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=109141" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/3/1/m_dnares_25_3cover.png?Expires=2147483647&amp;Signature=zOskx939n215x6LsaNaPxmag9A6Tt6e7bP~Kt5lfxwaH8FjX1dKLUJ0mdj-Sm0uq9hmCpw3OAnAQFLS-GTY4CsXXX4aRhm5J1IbPv51CcTKXj828y94I4tx-BuFB-EA4WwEvEot4P8hDcnFzu~xWV52k2Sm6IHcCABqAv5wSWve6oMrdsQXhpN0srJqhN5UIcLjzws~g9qv24XhgtRb~Nw7X9IiEo1EM5V47od3bkv5yHlkUT1xmh3tJ4-ZjHw-HyAXqIp8eeHds4Cfjy9xqh8e3n5ynt516a1ZMF0hjA3wYH5X0-y3lqTHnWmgUvQv1L6dLXdRWhki~KqxTetYsJQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/3">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="118057328">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057328">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057330">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057330">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057335">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057335">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057354">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057354">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057377">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057377">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057382">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057382">5. Conclusion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057393">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057393">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/3/287/4818260">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/3/307/4831046">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Complete genome sequence and analysis of the industrial <em>Saccharomyces cerevisiae</em> strain N85 used in Chinese rice wine production
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Weiping Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">School of Biotechnology, Jiangnan University, Wuxi, Jiangsu 214443, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Weiping+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Weiping Zhang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yudong Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">School of Biotechnology, Jiangnan University, Wuxi, Jiangsu 214443, China</div><div class="aff">Department of Bioengineering, School of Food Sciences and Biotechnology, Zhejiang Gongshang University, Hangzhou 310018, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yudong+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yudong Li</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yiwang Chen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Bioengineering, School of Food Sciences and Biotechnology, Zhejiang Gongshang University, Hangzhou 310018, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yiwang+Chen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chen Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chen Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yiwang Chen</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sha Xu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">School of Biotechnology, Jiangnan University, Wuxi, Jiangsu 214443, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sha+Xu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xu S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Xu S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sha Xu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Guocheng Du</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">School of Biotechnology, Jiangnan University, Wuxi, Jiangsu 214443, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Guocheng+Du">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Du G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Du G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Guocheng Du</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Huidong Shi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Georgia Cancer Center, Augusta University, Augusta, GA 30912, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Huidong+Shi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shi H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shi H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Huidong Shi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jingwen Zhou</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">School of Biotechnology, Jiangnan University, Wuxi, Jiangsu 214443, China</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy002-cor1" data-open="dsy002-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +86 510 85918312. Fax. +86 510 85918309. Email: <a href="mailto:zhoujw1982@jiangnan.edu.cn" target="_blank">zhoujw1982@jiangnan.edu.cn</a> (J.W.Z.); Tel. +86 510 85913660. Fax. +86 510 85918309. <a href="mailto:jchen@jiangnan.edu.cn" target="_blank">jchen@jiangnan.edu.cn</a> (J.C.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jingwen+Zhou">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhou J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhou J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jingwen Zhou</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jian Chen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">School of Biotechnology, Jiangnan University, Wuxi, Jiangsu 214443, China</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy002-cor1" data-open="dsy002-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +86 510 85918312. Fax. +86 510 85918309. Email: <a href="mailto:zhoujw1982@jiangnan.edu.cn" target="_blank">zhoujw1982@jiangnan.edu.cn</a> (J.W.Z.); Tel. +86 510 85913660. Fax. +86 510 85918309. <a href="mailto:jchen@jiangnan.edu.cn" target="_blank">jchen@jiangnan.edu.cn</a> (J.C.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jian+Chen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chen J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chen J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jian Chen</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsy002-FM1"><div class="fn"><div class="footnote-content">Weiping Zhang and Yudong Li authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsy002-FM2"><div class="fn"><div class="footnote-content">Edited by Dr Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 3, 1 June 2018, Pages 297–306, <a href='https://doi.org/10.1093/dnares/dsy002'>https://doi.org/10.1093/dnares/dsy002</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">05 February 2018</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">24 May 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">10 January 2018</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4838783" href="/dnaresearch/article-pdf/25/3/297/25072170/dsy002.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Weiping Zhang, Yudong Li, Yiwang Chen, Sha Xu, Guocheng Du, Huidong Shi, Jingwen Zhou, Jian Chen;  Complete genome sequence and analysis of the industrial <em>Saccharomyces cerevisiae</em> strain N85 used in Chinese rice wine production, <em>DNA Research</em>, Volume 25, Issue 3, 1 June 2018, Pages 297–306, <a href='https://doi.org/10.1093/dnares/dsy002'>https://doi.org/10.1093/dnares/dsy002</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4838783?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4838783?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4838783?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4838783?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4838783?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4838783?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4838783?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Complete%20genome%20sequence%20and%20analysis%20of%20the%20industrial%20%26lt%3Bem%26gt%3BSaccharomyces%20cerevisiae%26lt%3B%2Fem%26gt%3B%20strain%20N85%20used%20in%20Chinese%20rice%20wine%20production&amp;publicationDate=2018-02-05&amp;volumeNum=25&amp;issueNum=3&amp;author=Zhang%2C%20Weiping%3B%20Li%2C%20Yudong&amp;startPage=297&amp;endPage=306&amp;contentId=10.1093%2Fdnares%2Fdsy002&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f3%2f297%2f4838783" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=118057328 id="118057328" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Chinese rice wine is a popular traditional alcoholic beverage in China, while its brewing processes have rarely been explored. We herein report the first gapless, near-finished genome sequence of the yeast strain <em>Saccharomyces cerevisiae</em> N85 for Chinese rice wine production. Several assembly methods were used to integrate Pacific Bioscience (PacBio) and Illumina sequencing data to achieve high-quality genome sequencing of the strain. The genome encodes more than 6,000 predicted proteins, and 238 long non-coding RNAs, which are validated by RNA-sequencing data. Moreover, our annotation predicts 171 novel genes that are not present in the reference S288c genome. We also identified 65,902 single nucleotide polymorphisms and small indels, many of which are located within genic regions. Dozens of larger copy-number variations and translocations were detected, mainly enriched in the subtelomeres, suggesting these regions may be related to genomic evolution. This study will serve as a milestone in studying of Chinese rice wine and related beverages in China and in other countries. It will help to develop more scientific and modern fermentation processes of Chinese rice wine, and explore metabolism pathways of desired and harmful components in Chinese rice wine to improve its taste and nutritional value.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;rice wine yeast&quot;">rice wine yeast</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome sequence&quot;">genome sequence</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="annotation">annotation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="transcriptomics">transcriptomics</a></div>                            <h2 scrollto-destination=118057330 id="118057330" class="section-title" >1. Introduction</h2>
<p>As early as 5,000 years ago, Chinese rice wine (Huangjiu) was being consumed by people as a fermented alcoholic beverage.<a reveal-id="dsy002-B1" data-open="dsy002-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> Chinese rice wine is made from sticky rice, which is different from other beverages made from malt or fruit. The streamed sticky rice was saccharified by raw wheat <em>koji</em> (a variety of <em>Aspergillus</em>) and fermented to generate alcohol by yeast strain. Nowadays, Chinese rice wine is still hugely popular in China due to its pleasant taste, and high nutritional and pharmacological value.<a reveal-id="dsy002-B2" data-open="dsy002-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> However, some harmful byproducts are generated during the fermentation and storage of Chinese rice wine, such as ethyl carbamate (EC),<a reveal-id="dsy002-B3" data-open="dsy002-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> a genotoxic carcinogen of widespread occurrence in fermented food and beverages, with particularly high concentrations in stone fruit spirits.<a reveal-id="dsy002-B4" data-open="dsy002-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsy002-B5" data-open="dsy002-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> In Chinese rice wine, EC is mainly generated from the reaction of ethanol and urea, and its concentration can reach 160 µg/kg.<a reveal-id="dsy002-B3" data-open="dsy002-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> As increasing attention is paid to food safety problems, some researchers have focused on the mechanism of EC formulation in Chinese rice wine, and attempts to reduce its concentration have been made.<a reveal-id="dsy002-B3" data-open="dsy002-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsy002-B6 dsy002-B7 dsy002-B8" data-open="dsy002-B6 dsy002-B7 dsy002-B8" class="link link-ref link-reveal xref-bibr"><sup>6–8</sup></a></p><p>Over the past few decades, researchers have identified proteins responsible for the uptake and catabolism of poorly utilized nitrogen sources such as proline and urea, and found that their genes are repressed in <em>Saccharomyces cerevisiae</em> cells cultured in a nitrogen-rich environment. Conversely, in nitrogen-deficient conditions, their transcription is derepressed. This regulatory phenomenon has been termed nitrogen catabolite repression (NCR).<a reveal-id="dsy002-B9 dsy002-B10 dsy002-B11" data-open="dsy002-B9 dsy002-B10 dsy002-B11" class="link link-ref link-reveal xref-bibr"><sup>9–11</sup></a> During the accumulation of EC in wine fermentation, NCR plays an important role in the accumulation urea, the major precursor of EC, by repressing the transcription of urea catabolism-related genes. NCR is reportedly controlled by complicated regulatory systems, such as global GATA family regulators,<a reveal-id="dsy002-B12" data-open="dsy002-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> the TOR pathway,<a reveal-id="dsy002-B13" data-open="dsy002-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> and the Ssy1p-Ptr3p-Ssy5p sensing pathway.<a reveal-id="dsy002-B14" data-open="dsy002-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> However, the exact regulatory mechanism of NCR is still unclear. Above all, a complete genome sequence is fundamental for understanding the genetic and regulatory systems of Chinese rice wine strains. The draft genome sequence of <em>S. cerevisiae</em> Chinese rice wine strain YHJ7 has been published,<a reveal-id="dsy002-B15" data-open="dsy002-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> which is closely related to the industrial N85 strain (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/dsy002_supplemental_materials.doc?Expires=2147483647&Signature=M4Zaqb0hkRb9QoBS1HCN63DxI~kdb6LDai3YWKMpeQHJWqCZmPrEV3ZfueyUStqu3bJJ4YBwOfVSEUVQ-7FuWaz0sFXAdThZ-9XhIWBZPczI~FzmnCookOMb5mYoI7rHiYukm-HrJ3yN86ah8CFhL72Yx6W5YJEMwUkBBYlsCGvW~DsAv06pLtyAOx28FSbPZCx3pb2Z3kS4y0V1shBrMPcsBCbfpJQUv2dSnTJRRlFPSkInshs3k6zr26leTimg2YZcRVMrI0r4t1OLiuGQQZjotNrwLsZQbjDhmEi3toscs0c7XgxXUJBxLBQ5WX7CQuBdab-fUF~ec06uNX8ivg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). However, there are still many gaps in the YHJ7 assembly, and detailed annotation of the genome is not available.</p><p>In the mid-2000s, the emergence of next-generation sequencing (NGS) platforms dramatically reduced the run time and cost (around US $1000 for the human genome) of genome sequencing, and increased the throughput to hundreds of Gbp per run<a reveal-id="dsy002-B16" data-open="dsy002-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> compared with first-generation sequencing platforms (Sanger sequencers). However, high/low G + C regions, tandem repeat regions, and interspersed repeat regions remain difficult to sequence using NGS platforms.<a reveal-id="dsy002-B16" data-open="dsy002-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Furthermore, the limited read length from NGS platforms can prohibit sequencing completeness and the accuracy of analysis, resulting in assemblies that are incomplete and fragmented into several thousand contigs.<a reveal-id="dsy002-B15" data-open="dsy002-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsy002-B17" data-open="dsy002-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> In 2011, PacBio RS II was developed as the first commercially available third-generation sequencer and it was marketed to address this issue. The system uses a novel and unique single molecule real-time (SMRT) technology which enables the generation of long reads (half of reads &gt;20 kb, maximum read length &gt;60 kb) and reduces the degree of bias (even coverage across regions of differing G + C content).<a reveal-id="dsy002-B18" data-open="dsy002-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> Because long reads can easily handle complex regions such as repeats, PacBio long reads have the potential to provide accurate and improved genome assemblies.<a reveal-id="dsy002-B19" data-open="dsy002-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a></p><p>In this study, a gapless, near-finished genome sequence of the <em>S. cerevisiae</em> Chinese rice wine N85 strain was achieved by combining second- and third-generation sequencing technologies. Complex approaches were performed to assemble and annotate the N85 genome, and over 6,000 proteins and 238 long non-coding RNAs (lncRNAs) were identified. Moreover, many genomic variants were identified that could be responsible for genomic evolution. The first complete genome sequence of Chinese rice wine brewing yeast, strain N85, is a milestone in studying Chinese rice wine, as well as other related beverages brewing. It will help to understand the evolutionary history of beverage brewing strains and improve the complicated brewing processes from a traditional and experiential way to a modern and scientific way. In addition, further detail analysis of the genome of strain N85 will provide more genetic information for improvement the general quality of Chinese rice wine.</p>                            <h2 scrollto-destination=118057335 id="118057335" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=118057336 id="118057336" class="section-title" >2.1. Yeast strain and growth conditions</h3>
<p><em>Saccharomyces cerevisiae</em> strain N85 (<em>MAT</em><strong>a</strong>/<strong><em>α</em></strong>) was the strain generally used for Chinese rice wine production and provided by Guyuelongshan wine company (Shaoxing, China), one of the biggest and oldest Chinese rice wine producer. The yeast strain was pre-cultured on YPD plates (10 g/l yeast extract, 20 g/l peptone, 20 g/l glucose, 20 g/l agar) at 30°C for 24 h. A single colony was activated in YPD media (30°C, 200 rpm) then grown in sole nitrogen medium (1.7 g/l yeast nitrogen base, 20 g/l glucose, 10 mM each of glutamine, arginine, and urea). All cultivations were performed in shake flasks (200 rpm) at 30°C and growth was monitored by determining the optical density at 600 nm (OD<sub>600</sub>).</p>                            <h3 scrollto-destination=118057338 id="118057338" class="section-title" >2.2. DNA and RNA sequencing</h3>
<p>Following a 1 h incubation under the conditions described above, genomic DNA and RNA was isolated as described previously in.<a reveal-id="dsy002-B15" data-open="dsy002-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> DNA/RNA quantity was determined using a Nanodrop 1000 (Thermo Scientific, MA, USA) and integrity was determined with an Agilent 2100 Bioanalyzer (Palo Alto, CA). For NGS of genomic DNA, the libraries were constructed using Nextera DNA sample preparation kit (Illumina, CA, USA). Next, libraries were sequenced using MiSeq reagent kit v3 (Illumina, CA, USA) on MiSeq platform, or TruSeq SBS kit v3-HS (Illumina, CA, USA) on HiSeq 2000 platform. For third generation DNA sequencing, the DNA sample was sequenced using the PacBio RS II technology with a C2 chemistry sequencing kits (Pacific Biosciences, Melon Park, CA). For RNA sequencing (RNA-Seq), all RNA samples were prepared as biological duplicates and subjected to removal of rRNA or Poly-A filtering before cDNA library generation. From these libraries, 100 bp paired-end or strand-specific reads were produced using an Illumina HiSeq 2000. Library preparation and sequencing of DNA and RNA on the Illumina platform was performed at BioMarker Inc. (Beijing, China), and the PacBio sequencing was performed at Shanghai Bohao Inc (Shanghai, China).</p>                            <h3 scrollto-destination=118057340 id="118057340" class="section-title" >2.3. Genome assembly</h3>
<p>The genome of strain N85 was assembled using two approaches, one using a classical reference assembly, and the other using a hybrid <em>de novo</em> assembly (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/dsy002_supplemental_materials.doc?Expires=2147483647&Signature=M4Zaqb0hkRb9QoBS1HCN63DxI~kdb6LDai3YWKMpeQHJWqCZmPrEV3ZfueyUStqu3bJJ4YBwOfVSEUVQ-7FuWaz0sFXAdThZ-9XhIWBZPczI~FzmnCookOMb5mYoI7rHiYukm-HrJ3yN86ah8CFhL72Yx6W5YJEMwUkBBYlsCGvW~DsAv06pLtyAOx28FSbPZCx3pb2Z3kS4y0V1shBrMPcsBCbfpJQUv2dSnTJRRlFPSkInshs3k6zr26leTimg2YZcRVMrI0r4t1OLiuGQQZjotNrwLsZQbjDhmEi3toscs0c7XgxXUJBxLBQ5WX7CQuBdab-fUF~ec06uNX8ivg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). The hybrid strategy for genome assembly was carried out in three distinct and contiguous steps: (i) <em>de novo</em> generation of contigs; (ii) ordering of the <em>de novo</em> contigs generated and their concatenation into supercontigs; (iii) closing of the remaining gaps using a <em>de novo</em> iterative approach. Specifically, regarding the reference-guided genome assembly, filtered short reads were used for direct alignment to the publically-available <em>S. cerevisiae</em> S288c (version Scer3) and YHJ7 genome sequences using the CLC Genomics Workbench version v8.0 (<a class="link link-uri" href="https://www.qiagenbioinformatics.com/products/clc-genomics-workbench/" target="_blank">https://www.qiagenbioinformatics.com/products/clc-genomics-workbench/</a> (19 January 2018, date last accessed)). Regarding the hybrid <em>de novo</em> assembly, <em>de novo</em> genome assembly was first performed using one or both of the PacBio and Illumina datasets by several methods, including the CLC Genomics Workbench version v8.0, A5-miseq v2,<a reveal-id="dsy002-B20" data-open="dsy002-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> SSPACE-LongRead v1.1,<a reveal-id="dsy002-B21" data-open="dsy002-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> SPAdes v3.5.0,<a reveal-id="dsy002-B22" data-open="dsy002-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> PBcR,<a reveal-id="dsy002-B23" data-open="dsy002-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> and hierarchical genome assembly process (HGAP) v2.0.<a reveal-id="dsy002-B24" data-open="dsy002-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> Second, the order and orientation of contigs from the <em>de novo</em> assembly were determined by aligning contigs to the reference-guided assembly with a minimum identity value of 90% between the <em>de novo</em> contigs and the reference genome using CLC Genomics Workbench version v8.0 software. In this manner, most <em>de novo</em> contigs could be concatenated into supercontigs by overlapping regions.</p>                            <h3 scrollto-destination=118057342 id="118057342" class="section-title" >2.4. Genome annotation</h3>
<p>Gene features were annotated in the high-quality genome sequence using three different approaches: <em>ab initio</em>, evidence-, and homology-based predictions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/dsy002_supplemental_materials.doc?Expires=2147483647&Signature=M4Zaqb0hkRb9QoBS1HCN63DxI~kdb6LDai3YWKMpeQHJWqCZmPrEV3ZfueyUStqu3bJJ4YBwOfVSEUVQ-7FuWaz0sFXAdThZ-9XhIWBZPczI~FzmnCookOMb5mYoI7rHiYukm-HrJ3yN86ah8CFhL72Yx6W5YJEMwUkBBYlsCGvW~DsAv06pLtyAOx28FSbPZCx3pb2Z3kS4y0V1shBrMPcsBCbfpJQUv2dSnTJRRlFPSkInshs3k6zr26leTimg2YZcRVMrI0r4t1OLiuGQQZjotNrwLsZQbjDhmEi3toscs0c7XgxXUJBxLBQ5WX7CQuBdab-fUF~ec06uNX8ivg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). For the <em>ab initio</em> prediction, AUGUSTUS v3.0.3<a reveal-id="dsy002-B25" data-open="dsy002-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> was employed with the predefined parameter set for the <em>S. cerevisiae</em> genome. For the evidence-based prediction of transcripts, two annotation methods were performed; one used TopHat2<a reveal-id="dsy002-B26" data-open="dsy002-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> to map RNA-Seq reads to the N85 genome with default parameters to identify the possible location of introns and exons<a reveal-id="dsy002-B27" data-open="dsy002-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> that were subsequently integrated in a hint-based run of AUGUSTUS; the other used TRINITY<a reveal-id="dsy002-B28" data-open="dsy002-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> to run mapping-free transcript assembly with the RNA-Seq data. For the homology-based prediction of transcripts, S288c open reading frames (ORFs) downloaded from <em>Saccharomyces</em> genome database (SGD), and exonerate<a reveal-id="dsy002-B29" data-open="dsy002-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> was employed to align the ORFs to the genome of strain N85.</p><p>To compare coding differences between N85 and S288c, local BLASTp searches were carried out using the amino acid sequences of the predicted ORFs as queries, and amino acid sequences of all ORFs in the S288c as database. The following threshold settings were used: <em>e</em>-value &lt; 1 × 10<sup>−5</sup>, identity &gt;80%, and an alignment length &gt;50%.</p>                            <h3 scrollto-destination=118057345 id="118057345" class="section-title" >2.5. Detection of isoforms and ncRNAs</h3>
<p>For the detection of lncRNAs, Tophat2 was used for spliced read mapping with the following non-standard parameters: ‘no-mixed’, ‘no-discordant’, ‘b2-very-sensitive’, ‘max-intron-length 10 100’, and for strand-specific samples ‘libtype fr-first strand’. The number of reads within exons and genes were calculated by using the Cufflinks pipeline,<a reveal-id="dsy002-B26" data-open="dsy002-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> used as gene expression values, and normalized using the number of reads per kilobase on exon regions per million mapped reads. Alternative splicing events (isoforms) were also predicted using the Cufflinks software,<a reveal-id="dsy002-B26" data-open="dsy002-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> and mapped reads were visualized with SpliceGrapher.<a reveal-id="dsy002-B30" data-open="dsy002-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> All identified isoforms and ncRNAs were manually checked using integrative genomics viewer.<a reveal-id="dsy002-B31" data-open="dsy002-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a></p>                            <h3 scrollto-destination=118057347 id="118057347" class="section-title" >2.6. Detection of single nucleotide polymorphisms and analysis of copy-number compare with S288c</h3>
<p>To estimate the genetic distance between the N85 and S288c reference strains, all Illumina reads were mapped to reference genomes using BWA-MEM.<a reveal-id="dsy002-B32" data-open="dsy002-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> Biallelic single nucleotide polymorphisms (SNPs) were called by the SAMtools v0.1.19 ‘mpileup’ command and the VCFutils ‘varFilter’ command with ‘-D 200 -d 5’.</p><p>Per-position read counts were calculated from BAM files of mapped reads using the ‘gennomecov’ utility of BEDTools v2.15.0.<a reveal-id="dsy002-B33" data-open="dsy002-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> Reads were counted using a sliding window (1 kb) and used to find copy-number variations (CNVs). CNVs were identified using HMMcopy,<a reveal-id="dsy002-B34" data-open="dsy002-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> based on the ReadDepth method. To control false-positives, only CNVs with a length &gt;2 kb were selected. The boundaries of CNVs were confirmed by visual inspection in the integrated genome viewer. We calculated the overlap of each gene with SNPs/CNVs from gene ontology (GO) gene sets using FungiFun2 (<a class="link link-uri" href="https://elbe.hki-jena.de/fungifun/fungifun.php" target="_blank">https://elbe.hki-jena.de/fungifun/fungifun.php</a> (19 January 2018, date last accessed)). Bonferroni adjusted <em>P</em>-values and the false discovery rate were recorded.</p>                            <h3 scrollto-destination=118057350 id="118057350" class="section-title" >2.7. Analysis of the genome variations between N85 and sake strains</h3>
<p>The genome sequences of sake strains K7 and K11 were downloaded from SGD (<a class="link link-uri" href="http://www.yeastgenome.org/" target="_blank">http://www.yeastgenome.org/</a> (19 January 2018, date last accessed)). The differences between Chinese rice wine strain N85 and sake strains K7 and K11 were analysed at contig and sequencing read levels. First, the genome sequences of strain K7 and K11 were compared with that of strain N85 through local BLASTn with parameters as follows: -outfmt 17 -evalue 1e-5 -num_threads 8 -parse_deflines. The results in SAM format were visualized and manually investigated in integrative genomics viewer.<a reveal-id="dsy002-B31" data-open="dsy002-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> Second, all Illumina sequencing reads of the genome of strain N85, which could be aligned to N85 genome, were mapped into the scaffolds of strain K7 and K11 through CLC Genomics Workbench version v8.0. The resulting unmapped reads were assembled again to detect the unique regions of N85 genome.</p>                            <h3 scrollto-destination=118057352 id="118057352" class="section-title" >2.8. Nucleotide sequence accession number</h3>
<p>Data from the whole-genome shotgun project have been deposited at the EMBL database under the accession number LN907784-LN907800.</p>                            <h2 scrollto-destination=118057354 id="118057354" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=118057355 id="118057355" class="section-title" >3.1. Genome sequencing and assembly of the N85 genome</h3>
<p>Genomic DNA from <em>S. cerevisiae</em> strain N85 was sequenced using PacBio RS and Illumina HiSeq/MiSeq platforms. The average read length and coverage value from both sequencing platforms are summarized in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/dsy002_supplemental_materials.doc?Expires=2147483647&Signature=M4Zaqb0hkRb9QoBS1HCN63DxI~kdb6LDai3YWKMpeQHJWqCZmPrEV3ZfueyUStqu3bJJ4YBwOfVSEUVQ-7FuWaz0sFXAdThZ-9XhIWBZPczI~FzmnCookOMb5mYoI7rHiYukm-HrJ3yN86ah8CFhL72Yx6W5YJEMwUkBBYlsCGvW~DsAv06pLtyAOx28FSbPZCx3pb2Z3kS4y0V1shBrMPcsBCbfpJQUv2dSnTJRRlFPSkInshs3k6zr26leTimg2YZcRVMrI0r4t1OLiuGQQZjotNrwLsZQbjDhmEi3toscs0c7XgxXUJBxLBQ5WX7CQuBdab-fUF~ec06uNX8ivg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>. <em>De novo</em> assembly of PacBio reads was performed using the RS HGAP<a reveal-id="dsy002-B24" data-open="dsy002-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> assembly protocol version 3.3 in SMRT analysis version 2.2.0 (Pacific Biosciences) and resulted in 324 contigs (each with a length &gt;500 bp, and an N50 value of 75,828 bp for all contigs). Subsequently, hybrid assembler PBcR<a reveal-id="dsy002-B35" data-open="dsy002-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> and SPAdes<a reveal-id="dsy002-B22" data-open="dsy002-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> were used to assemble the PacBio and Illumina reads together, and this generated 601 and 284 contigs, respectively. Additionally, we assembled the genome using only Illumina MiSeq and HiSeq reads with SPAdes, resulting in &gt;1,000 contigs, and the N50 value was &lt;10,000 bp when assembled with only paired-end MiSeq reads. The gap between contigs from each assembly method was further closed using SSPACE-LongReads.<a reveal-id="dsy002-B21" data-open="dsy002-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> The assembly processes and results are summarized in <a reveal-id="dsy002-T1" data-open="dsy002-T1" class="link link-reveal link-table xref-fig">Table 1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/dsy002_supplemental_materials.doc?Expires=2147483647&Signature=M4Zaqb0hkRb9QoBS1HCN63DxI~kdb6LDai3YWKMpeQHJWqCZmPrEV3ZfueyUStqu3bJJ4YBwOfVSEUVQ-7FuWaz0sFXAdThZ-9XhIWBZPczI~FzmnCookOMb5mYoI7rHiYukm-HrJ3yN86ah8CFhL72Yx6W5YJEMwUkBBYlsCGvW~DsAv06pLtyAOx28FSbPZCx3pb2Z3kS4y0V1shBrMPcsBCbfpJQUv2dSnTJRRlFPSkInshs3k6zr26leTimg2YZcRVMrI0r4t1OLiuGQQZjotNrwLsZQbjDhmEi3toscs0c7XgxXUJBxLBQ5WX7CQuBdab-fUF~ec06uNX8ivg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>. <div content-id="dsy002-T1" class="table-modal"><div class="table-wrap"><div id="dsy002-T1" data-id="dsy002-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Summary of the <em>de novo</em> hybrid assembly results</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Library type </th><th>No. of contigs<sup>a</sup> </th><th>Maximum contig size (bp) </th><th>N50 contig (bp) </th><th>Total length (bp) </th><th>Software </th></tr></thead><tbody><tr><td>HiSeq </td><td>1,379 </td><td>200,122 </td><td>50,511 </td><td>12,737,887 </td><td>CLC genomic workbench </td></tr><tr><td>MiSeq </td><td>2,382 </td><td>39,836 </td><td>7,243 </td><td>11,888,873 </td><td>A5-miseq pipeline </td></tr><tr><td>PacBio </td><td>324 </td><td>242,389 </td><td>75,828 </td><td>11,883,288 </td><td>HGAP </td></tr><tr><td>Hybrid<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td>601 </td><td>27,619 </td><td>6,928 </td><td>3,893,255 </td><td>PBcR </td></tr><tr><td>Hybrid </td><td>284 </td><td>862, 32 </td><td>201,497 </td><td>11,857,571 </td><td>SPAdes </td></tr><tr><td>Close Gap </td><td>204 </td><td>1,107,090 </td><td>477,098 </td><td>11,917,338 </td><td>SSPACE-LongRead </td></tr></tbody></table></div><div content-id="dsy002-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Library type </th><th>No. of contigs<sup>a</sup> </th><th>Maximum contig size (bp) </th><th>N50 contig (bp) </th><th>Total length (bp) </th><th>Software </th></tr></thead><tbody><tr><td>HiSeq </td><td>1,379 </td><td>200,122 </td><td>50,511 </td><td>12,737,887 </td><td>CLC genomic workbench </td></tr><tr><td>MiSeq </td><td>2,382 </td><td>39,836 </td><td>7,243 </td><td>11,888,873 </td><td>A5-miseq pipeline </td></tr><tr><td>PacBio </td><td>324 </td><td>242,389 </td><td>75,828 </td><td>11,883,288 </td><td>HGAP </td></tr><tr><td>Hybrid<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td>601 </td><td>27,619 </td><td>6,928 </td><td>3,893,255 </td><td>PBcR </td></tr><tr><td>Hybrid </td><td>284 </td><td>862, 32 </td><td>201,497 </td><td>11,857,571 </td><td>SPAdes </td></tr><tr><td>Close Gap </td><td>204 </td><td>1,107,090 </td><td>477,098 </td><td>11,917,338 </td><td>SSPACE-LongRead </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>For each assembly, only contigs &gt;500 bp in length were considered.</p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>b</sup>Hybrid represents the combination of HiSeq, MiSeq, and PacBio datasets.</p></div></div></div></div></div><div class="table-wrap"><div id="dsy002-T1" data-id="dsy002-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Summary of the <em>de novo</em> hybrid assembly results</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Library type </th><th>No. of contigs<sup>a</sup> </th><th>Maximum contig size (bp) </th><th>N50 contig (bp) </th><th>Total length (bp) </th><th>Software </th></tr></thead><tbody><tr><td>HiSeq </td><td>1,379 </td><td>200,122 </td><td>50,511 </td><td>12,737,887 </td><td>CLC genomic workbench </td></tr><tr><td>MiSeq </td><td>2,382 </td><td>39,836 </td><td>7,243 </td><td>11,888,873 </td><td>A5-miseq pipeline </td></tr><tr><td>PacBio </td><td>324 </td><td>242,389 </td><td>75,828 </td><td>11,883,288 </td><td>HGAP </td></tr><tr><td>Hybrid<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td>601 </td><td>27,619 </td><td>6,928 </td><td>3,893,255 </td><td>PBcR </td></tr><tr><td>Hybrid </td><td>284 </td><td>862, 32 </td><td>201,497 </td><td>11,857,571 </td><td>SPAdes </td></tr><tr><td>Close Gap </td><td>204 </td><td>1,107,090 </td><td>477,098 </td><td>11,917,338 </td><td>SSPACE-LongRead </td></tr></tbody></table></div><div content-id="dsy002-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Library type </th><th>No. of contigs<sup>a</sup> </th><th>Maximum contig size (bp) </th><th>N50 contig (bp) </th><th>Total length (bp) </th><th>Software </th></tr></thead><tbody><tr><td>HiSeq </td><td>1,379 </td><td>200,122 </td><td>50,511 </td><td>12,737,887 </td><td>CLC genomic workbench </td></tr><tr><td>MiSeq </td><td>2,382 </td><td>39,836 </td><td>7,243 </td><td>11,888,873 </td><td>A5-miseq pipeline </td></tr><tr><td>PacBio </td><td>324 </td><td>242,389 </td><td>75,828 </td><td>11,883,288 </td><td>HGAP </td></tr><tr><td>Hybrid<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </td><td>601 </td><td>27,619 </td><td>6,928 </td><td>3,893,255 </td><td>PBcR </td></tr><tr><td>Hybrid </td><td>284 </td><td>862, 32 </td><td>201,497 </td><td>11,857,571 </td><td>SPAdes </td></tr><tr><td>Close Gap </td><td>204 </td><td>1,107,090 </td><td>477,098 </td><td>11,917,338 </td><td>SSPACE-LongRead </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>For each assembly, only contigs &gt;500 bp in length were considered.</p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>b</sup>Hybrid represents the combination of HiSeq, MiSeq, and PacBio datasets.</p></div></div></div></div></p><p><em>De novo</em> genome assemblies were further improved by combining the different hybrid assembly contigs and reference-guided assembly (see Section 2.3). Illumina reads were mapped to the S288c reference genome sequence using CLC genomic workbench to generate consensus sequences, which were used to place these contigs into 16 chromosomal and 1 mitochondrial sequences. The final assembly of the strain N85 genome showed high collinearity and structural conservation with that of the S288c reference (<a reveal-id="dsy002-F1" data-open="dsy002-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>), excluding some repetitive or telomeric regions. Genome sequences that are derived from assembling reads potentially suffer from errors, especially around regions with repetitive sequences. Specifically, the ribosome DNA repeats in Chromosome XII appeared to be incorrectly assembled, and were manually adjusted based on the assembly of the closely related YHJ7 strain.<a reveal-id="dsy002-B15" data-open="dsy002-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> Finally, the gapless and near-finished genome sequence of <em>S. cerevisiae</em> strain N85 was obtained, which contains 16 chromosomal and one mitochondrial sequences and is 12.09 million bp in total (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/dsy002_supplemental_materials.doc?Expires=2147483647&Signature=M4Zaqb0hkRb9QoBS1HCN63DxI~kdb6LDai3YWKMpeQHJWqCZmPrEV3ZfueyUStqu3bJJ4YBwOfVSEUVQ-7FuWaz0sFXAdThZ-9XhIWBZPczI~FzmnCookOMb5mYoI7rHiYukm-HrJ3yN86ah8CFhL72Yx6W5YJEMwUkBBYlsCGvW~DsAv06pLtyAOx28FSbPZCx3pb2Z3kS4y0V1shBrMPcsBCbfpJQUv2dSnTJRRlFPSkInshs3k6zr26leTimg2YZcRVMrI0r4t1OLiuGQQZjotNrwLsZQbjDhmEi3toscs0c7XgxXUJBxLBQ5WX7CQuBdab-fUF~ec06uNX8ivg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). </p>                        <a id="118057358" scrollto-destination="118057358"></a>
<div id="dsy002-F1" data-id="dsy002-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/m_dsy002f1.png?Expires=2147483647&Signature=ga1tM5sxma-I7BOiAizSZvm8ivbQ6P7WwxV69PsRotow2r0GtysS8hmVEX0kksR8bHkNgUZqAS3RlhTvwpfxI33f7OPgR3ONst0pih45s07JYmsjVp8R0wEJTTTXWaRaOyc0VSYnIME4R8QikaQpeigSWmt1wANdLseQPnaBlvzUMslaK0RjyIiy1M6TBSgofuPGoZDkjxqGcWtJE~vPa-oRhYCgZaUP-g28jq9xEWOAI2-Lq-SMwc98wzXViCxPUoU1-0odCEoFwEukc4jNPeEFovaH7fC29hq-h3Yfruv6ZxVzSANsYf9K9~1-buinCjGciY1md19hqjJzVnVHXw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Dot plot of sequence similarity between the assembly scaffolds of the N85 and S288c strains. The majority of N85 assembly sequences are collinear with the chromosome of the reference S288c strain." path-from-xml="dsy002f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057358/dsy002f1.tif" path-from-xml="dsy002f1.tif" target="_blank">View large</a><a section="118057358" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/dsy002f1.png?Expires=2147483647&Signature=a-f4xmGtWixul0IPC1EEMsRxEPWe3ikr6Qv1XoBha8HxvHafhBeUdiyagx6nneqhFpEJzzT6L2G2fAdZj0euqdbVWp~UNMP5qAG4nT8pYpdoGfYnCAtI2nkifUT4so6wn1JuL-Ch8NFVkAVt84YL7UUV46m5rZIcy4U~rTulqL-hYiqJKb7oR~BKranad9Gf7~9JFe-DrA7mzXvIqC2nTi8OM0iJlkCke5DebmoWpMxILCWDLDLWUStcEeT97nDVBgO4gBKEoi-KQ0~NSofsLZhTZVtJkX5Le4vZsXeKilzhYC0c~5mFvtzipqQPiYfLwabGKlguQemxcWn8o8008w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057358&ar=4838783&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy002f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Dot plot of sequence similarity between the assembly scaffolds of the N85 and S288c strains. The majority of N85 assembly sequences are collinear with the chromosome of the reference S288c strain.</p></div></div><div content-id="dsy002-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/m_dsy002f1.png?Expires=2147483647&Signature=ga1tM5sxma-I7BOiAizSZvm8ivbQ6P7WwxV69PsRotow2r0GtysS8hmVEX0kksR8bHkNgUZqAS3RlhTvwpfxI33f7OPgR3ONst0pih45s07JYmsjVp8R0wEJTTTXWaRaOyc0VSYnIME4R8QikaQpeigSWmt1wANdLseQPnaBlvzUMslaK0RjyIiy1M6TBSgofuPGoZDkjxqGcWtJE~vPa-oRhYCgZaUP-g28jq9xEWOAI2-Lq-SMwc98wzXViCxPUoU1-0odCEoFwEukc4jNPeEFovaH7fC29hq-h3Yfruv6ZxVzSANsYf9K9~1-buinCjGciY1md19hqjJzVnVHXw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Dot plot of sequence similarity between the assembly scaffolds of the N85 and S288c strains. The majority of N85 assembly sequences are collinear with the chromosome of the reference S288c strain." path-from-xml="dsy002f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057358/dsy002f1.tif" path-from-xml="dsy002f1.tif" target="_blank">View large</a><a section="118057358" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/dsy002f1.png?Expires=2147483647&Signature=a-f4xmGtWixul0IPC1EEMsRxEPWe3ikr6Qv1XoBha8HxvHafhBeUdiyagx6nneqhFpEJzzT6L2G2fAdZj0euqdbVWp~UNMP5qAG4nT8pYpdoGfYnCAtI2nkifUT4so6wn1JuL-Ch8NFVkAVt84YL7UUV46m5rZIcy4U~rTulqL-hYiqJKb7oR~BKranad9Gf7~9JFe-DrA7mzXvIqC2nTi8OM0iJlkCke5DebmoWpMxILCWDLDLWUStcEeT97nDVBgO4gBKEoi-KQ0~NSofsLZhTZVtJkX5Le4vZsXeKilzhYC0c~5mFvtzipqQPiYfLwabGKlguQemxcWn8o8008w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057358&ar=4838783&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy002f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Dot plot of sequence similarity between the assembly scaffolds of the N85 and S288c strains. The majority of N85 assembly sequences are collinear with the chromosome of the reference S288c strain.</p></div></div>                            <h3 scrollto-destination=118057359 id="118057359" class="section-title" >3.2. Multiple approaches used to annotate the N85 genome</h3>
<p>In order to achieve a high-quality gene annotation of the <em>S. cerevisiae</em> strain N85 genome, four different annotation approaches were performed. First, all ORFs in the model <em>S. cerevisiae</em> S288c were downloaded from the SGD and aligned to the N85 genome using exonerate.<a reveal-id="dsy002-B29" data-open="dsy002-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> A total of 8,082 predicted ORFs were generated, and 6,236 of these were retained when the percentage coverage of the alignment cut-off was &gt;50%. Second, <em>ab initio</em> gene prediction was performed with AUGUSTUS,<a reveal-id="dsy002-B25" data-open="dsy002-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> which generated 6,786 draft predicted ORFs. Thirdly, the total reads from RNA-Seq experiments were mapped onto the genome of <em>S. cerevisiae</em> strain N85 using TOPHAT2 to generate exon and intron ‘hints’ in AUGUSTUS.<a reveal-id="dsy002-B25" data-open="dsy002-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> With the help of these hints, local AUSGUSTUS genome prediction was carried out, resulting in 5,320 hints-based predicted ORFs. Lastly, total RNA-Seq reads were used to create the unbiased mapping-free transcriptome assembly using TRINITY,<a reveal-id="dsy002-B36" data-open="dsy002-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> which generated 15,371 draft transcripts. The results of the final three methods were filtered by amino acid length (&gt;100), and 6,405, 5,339, and 10,487 ORFs were retained, respectively.</p><p>To compare coding differences between N85 and S288c, the amino acid sequences of the predicted ORFs were aligned to those of all S288c ORFs. 5,187, 4,968, and 5,267 S288c homologous genes were extracted from AUGUSTUS, AUGUSTUS-Tophat, and TRINITY predictions, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/dsy002_supplementary_table_s3.xlsx?Expires=2147483647&Signature=GoS2c0eYpQx89wzlHuUp03xO9EmUo5Qk5fV~13EB88KmlKdOBxFYpb8LHixy0780rl1PiKOlOMKKOMELoHLsurbgLVR7ypOM7hEBEO7vHy7vrhGZ2SgwXB3LZTBJZRqNSytPtxMYFhtz6Ry~s~fCGIr8ZGxm3Xs15cVEiG~NJl5zI-CA1HPCUMTbMpUMdaXlO1YjRg--cklOGvjoWA8aqU3cCcSjKiXvpP9rsG8SRXvgAl1D286bIA-00sYEBQfl8bdJMAu84VQVKCZGnUBlzo4ttrCGy7XFkAnBq-Ys~rz9C9xmKx4uoyMv8AJybV0IHvaFzJ49bPWQcTCGJmLcCA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). Combining these S288c homologous genes and those from the exonerate prediction, 6,464 S288c homologous genes were identified in strain N85. Among them, 4,694 S288c homologous genes were present in all four predictions (<a reveal-id="dsy002-F2" data-open="dsy002-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). Moreover, we identified 111, 86, and 82 non-S288c genes that were missing from the S288c genome annotation using AUGUSTUS, AUGUSTUS-Tophat, and TRINITY predictions, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/dsy002_supplementary_table_s3.xlsx?Expires=2147483647&Signature=GoS2c0eYpQx89wzlHuUp03xO9EmUo5Qk5fV~13EB88KmlKdOBxFYpb8LHixy0780rl1PiKOlOMKKOMELoHLsurbgLVR7ypOM7hEBEO7vHy7vrhGZ2SgwXB3LZTBJZRqNSytPtxMYFhtz6Ry~s~fCGIr8ZGxm3Xs15cVEiG~NJl5zI-CA1HPCUMTbMpUMdaXlO1YjRg--cklOGvjoWA8aqU3cCcSjKiXvpP9rsG8SRXvgAl1D286bIA-00sYEBQfl8bdJMAu84VQVKCZGnUBlzo4ttrCGy7XFkAnBq-Ys~rz9C9xmKx4uoyMv8AJybV0IHvaFzJ49bPWQcTCGJmLcCA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). In total, 171 non-S288c genes were identified by merging all these non-S288c genes, and 17 non-S288c genes were common to all three predictions (<a reveal-id="dsy002-F2" data-open="dsy002-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). Consistent with a previous study, we detected a large region (24 kb) in Chromosome XIV that includes three non-S288c genes (g5159, g5160, and g5161) that are only present in Asian yeast strains (<a reveal-id="dsy002-F2" data-open="dsy002-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>). In addition, we also detected some N85-specific regions that are not present in YHJ7, including the region in Chromosome XII that includes two novel genes g4241 and g4242 (<a reveal-id="dsy002-F2" data-open="dsy002-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>). The expression of these novel non-S288c genes was validated by RNA-Seq data, but their function requires further investigation. </p>                        <a id="118057362" scrollto-destination="118057362"></a>
<div id="dsy002-F2" data-id="dsy002-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/m_dsy002f2.png?Expires=2147483647&Signature=qnoUWd8AyBcX3Go8hXtZAnhkR9DvN3bUxFeMdZGmTukwKE1vyWRRItL7daGVOf4HnJo64F4V7rVCl8hLweuyh8gY9emasxQB~zWLCRUgrltjjVnkXASDIrQu9zPC92mcrCqSG469iNFq9rYvkeQjuCR8hTL6BuJ6nbdG8cwHPyxF87fpauyjrgeOdwGQSJQ4E~VakORuxF69lh4HpLd8PazsGTNv4bIAeBuR1wT8h2KjipT3-mTYQ7sFWsVNLVi3wD~G0wLC550bfAOCAz3fuXzymimdBRyHtDM-7ll~XUeJ50OZqyWIP7bcrf3LiOAt2axWVRe1KQd9MkKEvHgaaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Annotation of the S. cerevisiae N85 genome. Analysis of S288c homologous and non-S288c genes in S. cerevisiae N85 through different approaches. (A) Number of S288c homologous genes identified using exonerate (yellow), AUGUSTUS (red), AUGUSTUS-Tophat (blue), and TRINITY (green). (B) Number of non-S288c genes identified using AUGUSTUS (red), AUGUSTUS-Tophat (blue), and TRINITY (green). (C) Genomic architecture of non-S288c genes in N85. Gene locations are shown below each gene box. Color figures available in online version." path-from-xml="dsy002f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057362/dsy002f2.tif" path-from-xml="dsy002f2.tif" target="_blank">View large</a><a section="118057362" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/dsy002f2.png?Expires=2147483647&Signature=jIcyNG8BcVuGtNGzefOMbZrTB0I5jPvOIW0csKZVLp3bEzmgXQpEijW1G1wjbbMywY9pHRA64FjAccdXJGNp37vcPJ~tyjdXF8K3VWj5DhQyr~MtAbHC7Z0wXrnr2PUnI9EcosHLWsNJtGJWVh4wq4ouGIWDBhoDgUoKrOLKaXugrDICETK--8VdsH8zbBsGZH2ILrcS7H3Dtbx0k359SNs-xJ9JGCADzjt9BaEvythoHP2GkPYFMC9wi5y2BF~LMz9ocQZFTcWHkCGnyFMyUK6j2vzIX4IiMdEzBiKpe0oxp-KJp4cTJLU38-~utx2lq-L-s3xktmC05P50~quj9w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057362&ar=4838783&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy002f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Annotation of the <em>S. cerevisiae</em> N85 genome. Analysis of S288c homologous and non-S288c genes in <em>S. cerevisiae</em> N85 through different approaches. (A) Number of S288c homologous genes identified using exonerate (yellow), AUGUSTUS (red), AUGUSTUS-Tophat (blue), and TRINITY (green). (B) Number of non-S288c genes identified using AUGUSTUS (red), AUGUSTUS-Tophat (blue), and TRINITY (green). (C) Genomic architecture of non-S288c genes in N85. Gene locations are shown below each gene box. Color figures available in online version.</p></div></div><div content-id="dsy002-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/m_dsy002f2.png?Expires=2147483647&Signature=qnoUWd8AyBcX3Go8hXtZAnhkR9DvN3bUxFeMdZGmTukwKE1vyWRRItL7daGVOf4HnJo64F4V7rVCl8hLweuyh8gY9emasxQB~zWLCRUgrltjjVnkXASDIrQu9zPC92mcrCqSG469iNFq9rYvkeQjuCR8hTL6BuJ6nbdG8cwHPyxF87fpauyjrgeOdwGQSJQ4E~VakORuxF69lh4HpLd8PazsGTNv4bIAeBuR1wT8h2KjipT3-mTYQ7sFWsVNLVi3wD~G0wLC550bfAOCAz3fuXzymimdBRyHtDM-7ll~XUeJ50OZqyWIP7bcrf3LiOAt2axWVRe1KQd9MkKEvHgaaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Annotation of the S. cerevisiae N85 genome. Analysis of S288c homologous and non-S288c genes in S. cerevisiae N85 through different approaches. (A) Number of S288c homologous genes identified using exonerate (yellow), AUGUSTUS (red), AUGUSTUS-Tophat (blue), and TRINITY (green). (B) Number of non-S288c genes identified using AUGUSTUS (red), AUGUSTUS-Tophat (blue), and TRINITY (green). (C) Genomic architecture of non-S288c genes in N85. Gene locations are shown below each gene box. Color figures available in online version." path-from-xml="dsy002f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057362/dsy002f2.tif" path-from-xml="dsy002f2.tif" target="_blank">View large</a><a section="118057362" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/dsy002f2.png?Expires=2147483647&Signature=jIcyNG8BcVuGtNGzefOMbZrTB0I5jPvOIW0csKZVLp3bEzmgXQpEijW1G1wjbbMywY9pHRA64FjAccdXJGNp37vcPJ~tyjdXF8K3VWj5DhQyr~MtAbHC7Z0wXrnr2PUnI9EcosHLWsNJtGJWVh4wq4ouGIWDBhoDgUoKrOLKaXugrDICETK--8VdsH8zbBsGZH2ILrcS7H3Dtbx0k359SNs-xJ9JGCADzjt9BaEvythoHP2GkPYFMC9wi5y2BF~LMz9ocQZFTcWHkCGnyFMyUK6j2vzIX4IiMdEzBiKpe0oxp-KJp4cTJLU38-~utx2lq-L-s3xktmC05P50~quj9w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057362&ar=4838783&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy002f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Annotation of the <em>S. cerevisiae</em> N85 genome. Analysis of S288c homologous and non-S288c genes in <em>S. cerevisiae</em> N85 through different approaches. (A) Number of S288c homologous genes identified using exonerate (yellow), AUGUSTUS (red), AUGUSTUS-Tophat (blue), and TRINITY (green). (B) Number of non-S288c genes identified using AUGUSTUS (red), AUGUSTUS-Tophat (blue), and TRINITY (green). (C) Genomic architecture of non-S288c genes in N85. Gene locations are shown below each gene box. Color figures available in online version.</p></div></div>                            <h3 scrollto-destination=118057363 id="118057363" class="section-title" >3.3. Transcription of lncRNAs and isoforms</h3>
<p>RNA-Seq was used to determine lncRNAs in <em>S. cerevisiae</em> strain N85. Approximately, 20 million poly-adenylated RNA-Seq reads and 2 billion strand-specific ribosome-removed RNA-Seq reads were obtained, which allowed us to confirm the orientation of transcripts and predict anti-sense transcripts. After read mapping and transcript assembly, we classified all expressed transcripts longer than 200 nucleotides into coding genes and lncRNAs. Using these sequencing datasets, we detected 238 lncRNAs, most of which are novel lncRNAs not annotated in the databases (<a reveal-id="dsy002-F3" data-open="dsy002-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). Consistent with previous studies, lncRNAs were expressed at significantly lower levels than coding genes (<a reveal-id="dsy002-F3" data-open="dsy002-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>, Wilcoxon test, <em>P</em> &lt; 10<sup>−5</sup>). However, the function of these novel lncRNAs requires further investigation in the future. Besides lncRNAs, 619 transcript isoforms of N85 genes were also annotated using transcriptome analysis (<a reveal-id="dsy002-F3" data-open="dsy002-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>), and the expression levels of isoforms were slightly higher than those of genes without alternative splicing (<a reveal-id="dsy002-F3" data-open="dsy002-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). </p>                        <a id="118057365" scrollto-destination="118057365"></a>
<div id="dsy002-F3" data-id="dsy002-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/m_dsy002f3.png?Expires=2147483647&Signature=2X6LMwIW6ARNRGApyW25Nvxuip2UlQcMcppW36Hn~D34NSNNrgmq5AH1O-LWZm2HiU7ROsoRsYjDYYqqGaFYcEwT877r4J6s7MmfDg-AE0Eq54lFIwyJkuq4jSAVy~ECaaFg1k00Y6ChUUsR8GEaDFkUEVoAtikSml9tEfKAwQMBkfhPeFaHJ7kK9NeomJFnATPnxcYmANKzT4JzYLBzJnN~T70nna8xYTgLGIqNwCP5QaQlFlG0~w8mXR3hvIdWhC1vKBtia1xjbSc6Wzw1kaTc7XHjdpffEJbCZtEGUNB51TzkVhXuxE4wC5TmC0-o25JGT1VgUzzxJ2p5PI-Cqg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="General characteristics of coding and non-coding transcripts. (A) The number of different transcript variants in S. cerevisiae N85. (B) Box-plots of transcript expression levels in log2 (FPKM) units. FPKM, fragments per kilobase of exon per million reads mapped." path-from-xml="dsy002f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057365/dsy002f3.tif" path-from-xml="dsy002f3.tif" target="_blank">View large</a><a section="118057365" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/dsy002f3.png?Expires=2147483647&Signature=DLpuZy40wtop8tjQMRFPD8RA9ZzjltEKepxjSkif0goEJv2LpFqDkYhWU7uiV~5lsiEpAEqr1LxEVNT2QH2FLNl0a-PHXlzE8lWuJrEDpeh4rKotuZuQSLlpyb2HaFr5QJYGm8xGOopKYnxP42FXQfC-0gz0VIT~ycrbfL0Go0s8--g7ItZZZl228Js-xMF2UsKsDyD1X06T8aRFES1KZBMYs4rCSIXmapGxWBI2BdnXhtsgtBfAzhWeAWPxkSaR1V4xNJH8cqcPgHcWVvLxZmG~p-JxBBLUhShFMKhFKCrFAik-OPgxaXTuxocCQfznJ~MQFbOEYWXjQs4~sVEP3g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057365&ar=4838783&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy002f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>General characteristics of coding and non-coding transcripts. (A) The number of different transcript variants in <em>S. cerevisiae</em> N85. (B) Box-plots of transcript expression levels in log2 (FPKM) units. FPKM, fragments per kilobase of exon per million reads mapped.</p></div></div><div content-id="dsy002-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/m_dsy002f3.png?Expires=2147483647&Signature=2X6LMwIW6ARNRGApyW25Nvxuip2UlQcMcppW36Hn~D34NSNNrgmq5AH1O-LWZm2HiU7ROsoRsYjDYYqqGaFYcEwT877r4J6s7MmfDg-AE0Eq54lFIwyJkuq4jSAVy~ECaaFg1k00Y6ChUUsR8GEaDFkUEVoAtikSml9tEfKAwQMBkfhPeFaHJ7kK9NeomJFnATPnxcYmANKzT4JzYLBzJnN~T70nna8xYTgLGIqNwCP5QaQlFlG0~w8mXR3hvIdWhC1vKBtia1xjbSc6Wzw1kaTc7XHjdpffEJbCZtEGUNB51TzkVhXuxE4wC5TmC0-o25JGT1VgUzzxJ2p5PI-Cqg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="General characteristics of coding and non-coding transcripts. (A) The number of different transcript variants in S. cerevisiae N85. (B) Box-plots of transcript expression levels in log2 (FPKM) units. FPKM, fragments per kilobase of exon per million reads mapped." path-from-xml="dsy002f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057365/dsy002f3.tif" path-from-xml="dsy002f3.tif" target="_blank">View large</a><a section="118057365" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/dsy002f3.png?Expires=2147483647&Signature=DLpuZy40wtop8tjQMRFPD8RA9ZzjltEKepxjSkif0goEJv2LpFqDkYhWU7uiV~5lsiEpAEqr1LxEVNT2QH2FLNl0a-PHXlzE8lWuJrEDpeh4rKotuZuQSLlpyb2HaFr5QJYGm8xGOopKYnxP42FXQfC-0gz0VIT~ycrbfL0Go0s8--g7ItZZZl228Js-xMF2UsKsDyD1X06T8aRFES1KZBMYs4rCSIXmapGxWBI2BdnXhtsgtBfAzhWeAWPxkSaR1V4xNJH8cqcPgHcWVvLxZmG~p-JxBBLUhShFMKhFKCrFAik-OPgxaXTuxocCQfznJ~MQFbOEYWXjQs4~sVEP3g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057365&ar=4838783&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy002f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>General characteristics of coding and non-coding transcripts. (A) The number of different transcript variants in <em>S. cerevisiae</em> N85. (B) Box-plots of transcript expression levels in log2 (FPKM) units. FPKM, fragments per kilobase of exon per million reads mapped.</p></div></div>                            <h3 scrollto-destination=118057366 id="118057366" class="section-title" >3.4. Genome variations in strain N85 compared with the model strain S288c</h3>
<p>To examine genetic variation in N85, the Illumina short reads were mapped to the S288c reference genome, which identified 57,278 SNPs and 8,624 indels in N85 (<a reveal-id="dsy002-T2" data-open="dsy002-T2" class="link link-reveal link-table xref-fig">Table 2</a>). As expected, ∼99% of these SNPs have been observed in the closely related YHJ7 strain. Although N85 is almost homozygous, 2,131 heterozygous sites were found in the diploid sequenced N85 strain (<a reveal-id="dsy002-T2" data-open="dsy002-T2" class="link link-reveal link-table xref-fig">Table 2</a>). The SNP distribution was not random, and approximately one-third of all detected SNPs were intergenic (<a reveal-id="dsy002-T2" data-open="dsy002-T2" class="link link-reveal link-table xref-fig">Table 2</a>), even though only about 25% of the <em>S. cerevisiae</em> genome is noncoding. More than 30% of SNPs detected in coding regions are nonsynonymous, resulting in changes to the encoded protein sequence. Interestingly, nonsynonymous mutations are more frequent in homozygous SNPs than in heterozygous SNPs (24 vs. 22%, respectively; <em>P</em> &lt; 0.01, Chi-squared test). Moreover, several genes have gained or lost stop codons (<a reveal-id="dsy002-T2" data-open="dsy002-T2" class="link link-reveal link-table xref-fig">Table 2</a>). <div content-id="dsy002-T2" class="table-modal"><div class="table-wrap"><div id="dsy002-T2" data-id="dsy002-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Genetic variations identified in the <em>S. cerevisiae</em> strain N85</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="3">N85<hr /> </th><th colspan="3">YHJ7-homologues<hr /> </th></tr><tr><th>Total </th><th>Hom<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>Het </th><th>Total </th><th>Hom </th><th>Het </th></tr></thead><tbody><tr><td>Exonic </td><td>38,478 </td><td>37,564 </td><td>914 </td><td>214 </td><td>73 </td><td>141 </td></tr><tr><td> Synonymous </td><td>23,717 </td><td>23,350 </td><td>367 </td><td>75 </td><td>21 </td><td>54 </td></tr><tr><td> Nonsynonymous </td><td>12,380 </td><td>12,121 </td><td>259 </td><td>72 </td><td>26 </td><td>46 </td></tr><tr><td> Frameshift </td><td>274 </td><td>243 </td><td>31 </td><td>6 </td><td>2 </td><td>4 </td></tr><tr><td> Nonframeshift </td><td>1,996 </td><td>1,748 </td><td>248 </td><td>55 </td><td>20 </td><td>35 </td></tr><tr><td> Stop gain or loss </td><td>89 </td><td>82 </td><td>7 </td><td>6 </td><td>4 </td><td>2 </td></tr><tr><td>Intronic </td><td>428 </td><td>424 </td><td>4 </td><td>5 </td><td>3 </td><td>2 </td></tr><tr><td>Intergenic </td><td>26,996 </td><td>25,783 </td><td>1,213 </td><td>1,175 </td><td>852 </td><td>323 </td></tr></tbody></table></div><div content-id="dsy002-T2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="3">N85<hr /> </th><th colspan="3">YHJ7-homologues<hr /> </th></tr><tr><th>Total </th><th>Hom<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>Het </th><th>Total </th><th>Hom </th><th>Het </th></tr></thead><tbody><tr><td>Exonic </td><td>38,478 </td><td>37,564 </td><td>914 </td><td>214 </td><td>73 </td><td>141 </td></tr><tr><td> Synonymous </td><td>23,717 </td><td>23,350 </td><td>367 </td><td>75 </td><td>21 </td><td>54 </td></tr><tr><td> Nonsynonymous </td><td>12,380 </td><td>12,121 </td><td>259 </td><td>72 </td><td>26 </td><td>46 </td></tr><tr><td> Frameshift </td><td>274 </td><td>243 </td><td>31 </td><td>6 </td><td>2 </td><td>4 </td></tr><tr><td> Nonframeshift </td><td>1,996 </td><td>1,748 </td><td>248 </td><td>55 </td><td>20 </td><td>35 </td></tr><tr><td> Stop gain or loss </td><td>89 </td><td>82 </td><td>7 </td><td>6 </td><td>4 </td><td>2 </td></tr><tr><td>Intronic </td><td>428 </td><td>424 </td><td>4 </td><td>5 </td><td>3 </td><td>2 </td></tr><tr><td>Intergenic </td><td>26,996 </td><td>25,783 </td><td>1,213 </td><td>1,175 </td><td>852 </td><td>323 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p><sup>a</sup>Hom: homozygous; Het: heterozygous. YHJ7-homologs represents those SNPs that shared with YHJ7 were filtered from the total N85 SNPs.</p></div></div></div></div></div><div class="table-wrap"><div id="dsy002-T2" data-id="dsy002-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Genetic variations identified in the <em>S. cerevisiae</em> strain N85</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="3">N85<hr /> </th><th colspan="3">YHJ7-homologues<hr /> </th></tr><tr><th>Total </th><th>Hom<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>Het </th><th>Total </th><th>Hom </th><th>Het </th></tr></thead><tbody><tr><td>Exonic </td><td>38,478 </td><td>37,564 </td><td>914 </td><td>214 </td><td>73 </td><td>141 </td></tr><tr><td> Synonymous </td><td>23,717 </td><td>23,350 </td><td>367 </td><td>75 </td><td>21 </td><td>54 </td></tr><tr><td> Nonsynonymous </td><td>12,380 </td><td>12,121 </td><td>259 </td><td>72 </td><td>26 </td><td>46 </td></tr><tr><td> Frameshift </td><td>274 </td><td>243 </td><td>31 </td><td>6 </td><td>2 </td><td>4 </td></tr><tr><td> Nonframeshift </td><td>1,996 </td><td>1,748 </td><td>248 </td><td>55 </td><td>20 </td><td>35 </td></tr><tr><td> Stop gain or loss </td><td>89 </td><td>82 </td><td>7 </td><td>6 </td><td>4 </td><td>2 </td></tr><tr><td>Intronic </td><td>428 </td><td>424 </td><td>4 </td><td>5 </td><td>3 </td><td>2 </td></tr><tr><td>Intergenic </td><td>26,996 </td><td>25,783 </td><td>1,213 </td><td>1,175 </td><td>852 </td><td>323 </td></tr></tbody></table></div><div content-id="dsy002-T2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="3">N85<hr /> </th><th colspan="3">YHJ7-homologues<hr /> </th></tr><tr><th>Total </th><th>Hom<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>Het </th><th>Total </th><th>Hom </th><th>Het </th></tr></thead><tbody><tr><td>Exonic </td><td>38,478 </td><td>37,564 </td><td>914 </td><td>214 </td><td>73 </td><td>141 </td></tr><tr><td> Synonymous </td><td>23,717 </td><td>23,350 </td><td>367 </td><td>75 </td><td>21 </td><td>54 </td></tr><tr><td> Nonsynonymous </td><td>12,380 </td><td>12,121 </td><td>259 </td><td>72 </td><td>26 </td><td>46 </td></tr><tr><td> Frameshift </td><td>274 </td><td>243 </td><td>31 </td><td>6 </td><td>2 </td><td>4 </td></tr><tr><td> Nonframeshift </td><td>1,996 </td><td>1,748 </td><td>248 </td><td>55 </td><td>20 </td><td>35 </td></tr><tr><td> Stop gain or loss </td><td>89 </td><td>82 </td><td>7 </td><td>6 </td><td>4 </td><td>2 </td></tr><tr><td>Intronic </td><td>428 </td><td>424 </td><td>4 </td><td>5 </td><td>3 </td><td>2 </td></tr><tr><td>Intergenic </td><td>26,996 </td><td>25,783 </td><td>1,213 </td><td>1,175 </td><td>852 </td><td>323 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p><sup>a</sup>Hom: homozygous; Het: heterozygous. YHJ7-homologs represents those SNPs that shared with YHJ7 were filtered from the total N85 SNPs.</p></div></div></div></div></p><p>The nonsynonymous to synonymous substitution rate <em>(Ka/Ks</em>) relative to the S288c strain was assessed to identify fast-evolving genes. Genes with <em>Ka/Ks</em> values significantly &gt;1 were classified as under positive selection. We identified 232 genes with <em>Ka/Ks</em> values &gt; 1 (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/dsy002_supplementary_table_s4.xlsx?Expires=2147483647&Signature=iqZ02WaRRMKrdoaL596v1QY-WdrNtbObg7BbAB0HEM8yLb9V74RUeqlPkQf4JH-C8MEddvOVlqVZefWxwUcMtG8F8o84pInKQp1h0~gHrSuIheifDHxV11dr9ozcdd1UK5rj6BzsbkJ6CtkXS79KXfXr6XEk0XNmF8QOTF6Y2ikLu13eRJlsEnvAgyQyVE6NgfHf15aMqP5RHSJnI6g7qz7qq5h-9UDka7i4IMBBi9mX5kWopw7qJir3oQee907PQJHzHUOnATEfDmRRfTWki~6GuJAf08iNolaR3mdETVSlEEd7e5dZQArN8VpZdEbAhZ8jKHlDZqYAj99o~3L2mg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). GO term analysis revealed that these genes were mostly associated with transcriptional control. Several stress-responsive transcription factors appear to have evolved rapidly in N85, including the GATA zinc finger protein <em>GZF3</em>, which regulates nitrogen catabolic gene expression, and two genes (<em>HKR1</em>, <em>Ka/Ks</em> = 1.94; <em>MSB2</em>, <em>Ka/Ks</em> = 1.22) involved in the HOG pathway, which have been shown to play a key role in the adaption of Chinese rice wine strains.<a reveal-id="dsy002-B37" data-open="dsy002-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a></p><p>Sequencing of the N85 strain in its natural ploidy state allowed analysis of gross chromosomal rearrangements and aneuploidies. Based on the sequencing read depth, there were no gains or losses of whole chromosomes in the N85 genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/dsy002_supplemental_materials.doc?Expires=2147483647&Signature=M4Zaqb0hkRb9QoBS1HCN63DxI~kdb6LDai3YWKMpeQHJWqCZmPrEV3ZfueyUStqu3bJJ4YBwOfVSEUVQ-7FuWaz0sFXAdThZ-9XhIWBZPczI~FzmnCookOMb5mYoI7rHiYukm-HrJ3yN86ah8CFhL72Yx6W5YJEMwUkBBYlsCGvW~DsAv06pLtyAOx28FSbPZCx3pb2Z3kS4y0V1shBrMPcsBCbfpJQUv2dSnTJRRlFPSkInshs3k6zr26leTimg2YZcRVMrI0r4t1OLiuGQQZjotNrwLsZQbjDhmEi3toscs0c7XgxXUJBxLBQ5WX7CQuBdab-fUF~ec06uNX8ivg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>), discounting polyploidy or aneuploidy. A total of 76 amplification and deletion events were detected in N85, covering 1.57 Mb of the genome (<a reveal-id="dsy002-F4" data-open="dsy002-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). The size of these regions ranges from 1 to 18 kb, and most were detected in subtelomeric regions. GO enrichment analysis of CNV regions revealed that genes involved in cellular responses to nitrogen starvation, asparagine metabolic processes, and cellular aldehyde metabolic processes are most heavily influenced by CNVs. </p>                        <a id="118057370" scrollto-destination="118057370"></a>
<div id="dsy002-F4" data-id="dsy002-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/m_dsy002f4.png?Expires=2147483647&Signature=KZeWPOV66J6ydFQ33b86bSxI4ypTI2qxpC95i0wQxHfdoin~axyD9gNNGtReND7NfIV9263RVwKmwavpZ1DIrpdT~3Fogc2puFGe0vInMXtxOYjsPEUbVvz7b-e3fkq8ikjv2ujbubqE48BXZZMB17Qwmv2~U~QaipfVMcn6o-t2TvlopakXPj1LGQpNbZmJNybmVJ51ctpc145ZhJlLWlbqQmyaAuTcPn-LvLaf7jWgmYnPqcaYdTL0JFVMNqZX6OA4A9R~~EZwmb~NAyT1xd-OuSJwcVc1riqjqxV1qRhFcx10531zkwty03gC3CxQE2LlDuBOdGoCPsCeF7xK6Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic variation in the S. cerevisiae N85 genome. The first and second circles represent SNPs and INDELs relative to the S. cerevisiae S288c reference genome, in which the specific variation in N85 relative to YHJ7 is highlighted in red. The third and fourth cycles represent larger duplication/deletion or translocation events relative to YHJ7. Most of the structural rearrangements are localized in subtelomeric regions. Color figures available in online version." path-from-xml="dsy002f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057370/dsy002f4.tif" path-from-xml="dsy002f4.tif" target="_blank">View large</a><a section="118057370" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/dsy002f4.png?Expires=2147483647&Signature=y-naYOD2FIQvMWorp~CJRmYtJHemVJVtYBxwL6qAKKZSfRlH4qO2izC6R0fvzqUrjxlrF4teMZzqugm3ix~-ILMkQFyNNV21yy0AUea9OZ9-fCZmX0WDcfTKVYwynoC7Jy0yGY-1Y8lV6ubTudqC5HzUfeTCKkjUMT4FsO5MZoLPogsyi28hhsGd~OB8Dacklo3H23H8nhZ0Fe4j1EPX9cHCWPa3wqID7EwnL3IlzMVJD0eSZHgRB6muF9tABdAVRgcybDyOpiX1DVKqboIDnv4QMGy0OpzhoVba-30CXT0IDm4TYmcXI6Dai6yk3c4RYQ07LYD62I~UCAt9lnQ4ZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057370&ar=4838783&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy002f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic variation in the <em>S. cerevisiae</em> N85 genome. The first and second circles represent SNPs and INDELs relative to the <em>S. cerevisiae</em> S288c reference genome, in which the specific variation in N85 relative to YHJ7 is highlighted in red. The third and fourth cycles represent larger duplication/deletion or translocation events relative to YHJ7. Most of the structural rearrangements are localized in subtelomeric regions. Color figures available in online version.</p></div></div><div content-id="dsy002-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/m_dsy002f4.png?Expires=2147483647&Signature=KZeWPOV66J6ydFQ33b86bSxI4ypTI2qxpC95i0wQxHfdoin~axyD9gNNGtReND7NfIV9263RVwKmwavpZ1DIrpdT~3Fogc2puFGe0vInMXtxOYjsPEUbVvz7b-e3fkq8ikjv2ujbubqE48BXZZMB17Qwmv2~U~QaipfVMcn6o-t2TvlopakXPj1LGQpNbZmJNybmVJ51ctpc145ZhJlLWlbqQmyaAuTcPn-LvLaf7jWgmYnPqcaYdTL0JFVMNqZX6OA4A9R~~EZwmb~NAyT1xd-OuSJwcVc1riqjqxV1qRhFcx10531zkwty03gC3CxQE2LlDuBOdGoCPsCeF7xK6Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic variation in the S. cerevisiae N85 genome. The first and second circles represent SNPs and INDELs relative to the S. cerevisiae S288c reference genome, in which the specific variation in N85 relative to YHJ7 is highlighted in red. The third and fourth cycles represent larger duplication/deletion or translocation events relative to YHJ7. Most of the structural rearrangements are localized in subtelomeric regions. Color figures available in online version." path-from-xml="dsy002f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057370/dsy002f4.tif" path-from-xml="dsy002f4.tif" target="_blank">View large</a><a section="118057370" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/dsy002f4.png?Expires=2147483647&Signature=y-naYOD2FIQvMWorp~CJRmYtJHemVJVtYBxwL6qAKKZSfRlH4qO2izC6R0fvzqUrjxlrF4teMZzqugm3ix~-ILMkQFyNNV21yy0AUea9OZ9-fCZmX0WDcfTKVYwynoC7Jy0yGY-1Y8lV6ubTudqC5HzUfeTCKkjUMT4FsO5MZoLPogsyi28hhsGd~OB8Dacklo3H23H8nhZ0Fe4j1EPX9cHCWPa3wqID7EwnL3IlzMVJD0eSZHgRB6muF9tABdAVRgcybDyOpiX1DVKqboIDnv4QMGy0OpzhoVba-30CXT0IDm4TYmcXI6Dai6yk3c4RYQ07LYD62I~UCAt9lnQ4ZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057370&ar=4838783&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy002f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic variation in the <em>S. cerevisiae</em> N85 genome. The first and second circles represent SNPs and INDELs relative to the <em>S. cerevisiae</em> S288c reference genome, in which the specific variation in N85 relative to YHJ7 is highlighted in red. The third and fourth cycles represent larger duplication/deletion or translocation events relative to YHJ7. Most of the structural rearrangements are localized in subtelomeric regions. Color figures available in online version.</p></div></div>                            <h3 scrollto-destination=118057371 id="118057371" class="section-title" >3.5. The genome differences between Chinese rice wine strain N85 and sake strains</h3>
<p>In general, the genome of Chinese rice wine strain is quite similar with those of sake strains after manually reviewing the BLASTn results between strain N85 and K7 as well as K11. However, some genomic variations may explain the differences between Chinese rice wine and sake to some content. Four hundred indels were identified in the genome of strain N85 compared with both sake yeasts K7 and K11. Among them, 116 indels were found in gene coding regions. Some affected genes were found to involve in the production of organic acids (<em>KGD2</em>, <em>HSP31</em>, <em>BNA3</em>), amino acids catabolism (<em>STR2</em>, <em>HOM6</em>, <em>SPE2</em>, <em>CYS3</em>, <em>ARG2</em>). Previous study suggested that strain with deficient α-ketoglutarate dehydrogenase (<em>KGD2</em>) produced fewer ethanol during wine brewing.<a reveal-id="dsy002-B38" data-open="dsy002-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> The low ethanol content was compensated by an increase of organic acids, such citrate succinate, fumarate, and malate.<a reveal-id="dsy002-B38" data-open="dsy002-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a><sup>,</sup><em>BNA3</em> encodes a putative carbon-sulphur lyase responsible for volatile-thiol release. Deletion of <em>BNA3</em> reduced the release of 4-mercapto-4-methylpentan-2-one, which makes an important contribution to the aroma of wine, in both laboratory and wine yeast background.<a reveal-id="dsy002-B39" data-open="dsy002-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> Besides, the mutation of <em>CYS3</em>, which also encodes a putative carbon-sulphur lyase, led to over production of another sulphur compound, methyltetrathiophen-3-one, which was previously shown to contribute to wine aroma.<a reveal-id="dsy002-B39" data-open="dsy002-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a><em>ARG2</em> locates in arginine biosynthesis pathway and encodes acetylglutamate synthase, which catalyses the first step in the biosynthesis of the arginine precursor, ornithine.</p><p>In addition to indels, 25 regions in N85 genome were found to be absent either in the genome of strain K7 or that of K11, through mapping unmapped reads back to N85 genome. Five of these regions were identified to be absent in genomes of both sake strains. All of them are coding regions in the genome of strain N85, which encode the alpha-glucoside permease (<em>MPH3)</em>, the sorbitol dehydrogenase (<em>SOR1)</em>, a component of vacuolar cation channel (<em>YVC1)</em>, and two putative proteins with unknown function (YGL262W and YGL263W). <em>SOR1</em> and <em>MPH3</em> locate on the subtelomeric paralogous blocks of Chromosome X. However, the effects of the gain or loss of these genes on Chinese rice wine and sake are still unclear.</p>                            <h3 scrollto-destination=118057374 id="118057374" class="section-title" >3.6. Genome browser for N85</h3>
<p>To visualize gene sequences, annotated genes, and genetic variants in the genome of <em>S. cerevisiae</em> strain N85, a JBrowse-based genome browser was developed and deposited on the website (<a class="link link-uri" href="http://www.ligene.cn/hygd" target="_blank">http://www.ligene.cn/hygd</a> (19 January 2018, date last accessed)). As shown in <a reveal-id="dsy002-F5" data-open="dsy002-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>, the basic genome browser functionality can provide genome annotation views via an overhead bar that offers a visual indication of the chromosome position. Although only gene annotation and mutation information is currently available, further large-scale datasets will be integrated in the future. We also aim to implement additional analysis tools for BLAST searching, primer design, and sequence alignment, to help the scientific community to use the developed genome resources. </p>                        <a id="118057376" scrollto-destination="118057376"></a>
<div id="dsy002-F5" data-id="dsy002-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/m_dsy002f5.png?Expires=2147483647&Signature=NpbmnyzTQ~UEY~KjyBbBzEiZVDnIF7gnhXMF~6~Z4-FqtXxEXFysfOpR1YE920qPPCea1342dkw0HTXsSEEPH9D9gSWVlZdC2qyMUWE6BUyNSu9JBi5JhUwK67hEoTFZMEyu41LcXwkfnvj3wkiL0RwBDGCMEQikejbQm-yoUSyir6z~nsrq2X8G8o2p0qn8lVfGUWRIin6k2WuadAHeh4hf1A4WC7uslFKiCtzeg-ts1szimYNjGMbyB-0pyT6ApeDG3X-2dOaPeGc1xyIAKw2rN3feoJzWVdfGAr64FpvYbbzI1Rm-rzokI2spmWwGqHjicJHy1ZNAqV7s~3pQxw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Screenshot of the homepage of the Huangjiu Yeast Genome Database and genome browser displaying Chromosome 1 of S. cerevisiae N85. Gene regions are represented as a horizontal box, and genetic variants are represented as blue dots. Color figures available in online version." path-from-xml="dsy002f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057376/dsy002f5.tif" path-from-xml="dsy002f5.tif" target="_blank">View large</a><a section="118057376" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/dsy002f5.png?Expires=2147483647&Signature=KnrSNQ6eU4I47HgOk8ZsP41UR9P-U85WM5ToFR3tucfCEE5qIsWx7m-dYB6inBT7Wk8t2f0OJ9GYUhqBIloLaJVWY-k6sG3w~eBawCYsN2bma9Ld5S1j3~dttrFKKbckp8Vz4AMqO6iN6hpPtyggMR2el~oJmuSJAEDmOk31rLVkIAWa3JlpZppYuOXSdePkZBWZRqUNSoPlKYOsyXzFROb99HU9FE1MZKwsyKrHU474bRwdRHLo6z6xsErzxePM7AkJnbDktJSNL9zD6AQ21~ZAUbN9j6PDESOi1UE59jLbUyh0jmh4GqSGqJOeUZwT0-2-9f4egv~Nk-EH31NBOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057376&ar=4838783&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy002f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Screenshot of the homepage of the Huangjiu Yeast Genome Database and genome browser displaying Chromosome 1 of <em>S. cerevisiae</em> N85. Gene regions are represented as a horizontal box, and genetic variants are represented as blue dots. Color figures available in online version.</p></div></div><div content-id="dsy002-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/m_dsy002f5.png?Expires=2147483647&Signature=NpbmnyzTQ~UEY~KjyBbBzEiZVDnIF7gnhXMF~6~Z4-FqtXxEXFysfOpR1YE920qPPCea1342dkw0HTXsSEEPH9D9gSWVlZdC2qyMUWE6BUyNSu9JBi5JhUwK67hEoTFZMEyu41LcXwkfnvj3wkiL0RwBDGCMEQikejbQm-yoUSyir6z~nsrq2X8G8o2p0qn8lVfGUWRIin6k2WuadAHeh4hf1A4WC7uslFKiCtzeg-ts1szimYNjGMbyB-0pyT6ApeDG3X-2dOaPeGc1xyIAKw2rN3feoJzWVdfGAr64FpvYbbzI1Rm-rzokI2spmWwGqHjicJHy1ZNAqV7s~3pQxw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Screenshot of the homepage of the Huangjiu Yeast Genome Database and genome browser displaying Chromosome 1 of S. cerevisiae N85. Gene regions are represented as a horizontal box, and genetic variants are represented as blue dots. Color figures available in online version." path-from-xml="dsy002f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057376/dsy002f5.tif" path-from-xml="dsy002f5.tif" target="_blank">View large</a><a section="118057376" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/dsy002f5.png?Expires=2147483647&Signature=KnrSNQ6eU4I47HgOk8ZsP41UR9P-U85WM5ToFR3tucfCEE5qIsWx7m-dYB6inBT7Wk8t2f0OJ9GYUhqBIloLaJVWY-k6sG3w~eBawCYsN2bma9Ld5S1j3~dttrFKKbckp8Vz4AMqO6iN6hpPtyggMR2el~oJmuSJAEDmOk31rLVkIAWa3JlpZppYuOXSdePkZBWZRqUNSoPlKYOsyXzFROb99HU9FE1MZKwsyKrHU474bRwdRHLo6z6xsErzxePM7AkJnbDktJSNL9zD6AQ21~ZAUbN9j6PDESOi1UE59jLbUyh0jmh4GqSGqJOeUZwT0-2-9f4egv~Nk-EH31NBOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057376&ar=4838783&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy002f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Screenshot of the homepage of the Huangjiu Yeast Genome Database and genome browser displaying Chromosome 1 of <em>S. cerevisiae</em> N85. Gene regions are represented as a horizontal box, and genetic variants are represented as blue dots. Color figures available in online version.</p></div></div>                            <h2 scrollto-destination=118057377 id="118057377" class="section-title" >4. Discussion</h2>
<p>Chinese rice wine is a traditional fermented alcoholic beverage with many health benefits.<a reveal-id="dsy002-B40" data-open="dsy002-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> Recently, the annual consumption of Chinese rice wine has been steadily increasing, and now is more than 2 million kilolitres.<a reveal-id="dsy002-B41" data-open="dsy002-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a><sup>,</sup><a reveal-id="dsy002-B42" data-open="dsy002-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> However, the complete genome sequence of the Chinese rice wine strain used for fermentation has not been reported, which makes it difficult to further improve the quality of this popular product. The purpose of the current study was to genetically characterize the <em>S. cerevisiae</em> strain N85 by sequencing the genome using both PacBio and Illumina platforms. Assembly was achieved by combining the long but relatively low-quality PacBio reads<a reveal-id="dsy002-B43" data-open="dsy002-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> with the short but higher quality Illumina reads using a complex approach, and a high-quality genome of the N85 strain was generated. The assembled gapless and near-complete genome is equivalent in length to that of the model S288c strain. Due to the high integrity of the assembled genome, a larger number of protein coding genes were annotated through multiple annotation approaches compared with the gapped genome of the YHJ7 strain.<a reveal-id="dsy002-B15" data-open="dsy002-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> This comprehensive annotation also identified additional non-S288c genes,<a reveal-id="dsy002-B15" data-open="dsy002-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> which are important for functional and evolutionary analysis. Furthermore, the annotated genes in the N85-specific regions may be to explain the different evolutionary routes of Chinese rice wine strains.</p><p>In addition, genome comparison between Chinese strain and Japanese strains revealed a series of variations in coding regions, which encodes genes involved in the production of organic acids and the catabolism of amino acids. These variations might explain the difference of flavour and nutritional value between Chinese rice wine and sake. Notably, some variants were identified in the coding sequence of <em>ARG2</em>, who encodes acetylglutamate synthase in arginine biosynthesis pathway. Arginine is the precursor of urea, which is considered to be the major precursor of EC. The mutation in <em>ARG2</em> may lead to the higher concentration of EC in Chinese rice wine than that in sake. Besides, previous study revealed that two subtelomeric paralogous blocks in the genome of strain S288c, containing <em>HXT15</em>-<em>SOR2</em>-<em>MPH2</em> on Chromosome IV and <em>HXT16</em>-<em>SOR1</em>-<em>MPH3</em> on Chromosome X, were lost in the genome of strain K7.<a reveal-id="dsy002-B44" data-open="dsy002-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> However, a part of the second block was identified in the genome of strain N85, containing <em>SOR1</em>-<em>MPH3</em>. These differences among the genome of laboratory, Chinese rice wine, and sake strains may suggest strains underwent different evolutionary tracks to cope with each environment.</p><p>LncRNAs and alternative splicing are two important elements of transcription. LncRNAs are claimed to play an important role in the regulation of gene expression at transcriptional, post-transcriptional, and translational levels.<a reveal-id="dsy002-B45" data-open="dsy002-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a><sup>,</sup><a reveal-id="dsy002-B46" data-open="dsy002-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> Moreover, alternative splicing is a major contributor that determines the environmental fitness of an organism. Recently, various isoforms of the transcription factor <em>GAT1</em> were discovered in <em>S. cerevisiae</em> that are involved in nitrogen catabolite repression regulation.<a reveal-id="dsy002-B47" data-open="dsy002-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> In this work, numerous isoforms and novel lncRNAs were identified, enriching our existing knowledge in this area. Furthermore, future analysis of their functions could deepen our understanding of Chinese rice wine fermentation. It should be emphasized that current methods for annotating isoforms are not reliable; some annotated isoforms may be false-positives, and the accuracy of the results will be improved when the sequencing technology evolves and generates longer sequences.</p><p>The presence of highly conserved SNP sites shared in the N85 and YHJ7 genomes but not in the S288c genome indicated that the two strains originated from a common ancestor that diverged from the ancestor of the model S288c strain, and underwent adaptation during Chinese rice wine fermentation. However, the enrichment of SNPs in intergenic regions in the N85 genome might be due to reduced functional constraints in intergenic sequences during the evolutionary history of this strain. The enrichment of nonsynonymous mutations in homozygous genes suggests the function of heterozygous genes is much more stable, and homozygous SNPs appear to make a greater contribution to functional adaption in the N85 strain. During the fermentation of Chinese rice wine, yeast suffer high concentrations of sugars resulting from the digestion of rice starches in the fermentation mash. The enrichment of nonsynonymous SNPs in the HOG pathway is indicative of adaptive evolution of the strain over its long history in Chinese rice wine fermentation. Notably, this phenomenon was also observed in the genome of another Chinese rice wine strain, namely YJH7.<a reveal-id="dsy002-B15" data-open="dsy002-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> Similar enrichment was also identified in <em>GZF3</em>, one of four global transcriptional regulators of nitrogen catabolite repression, which may be responsible for the accumulation of EC in Chinese rice wine. Variations in genome structure, such as polyploidy, aneuploidy and copy number, have repeatedly been associated with domestication and adaptation to specific niches in experimentally evolved microbes.<a reveal-id="dsy002-B48" data-open="dsy002-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> Moreover, due to their plastic and dynamic nature, loss or gain of genes in subtelomeric regions occurs more frequently than in other regions, which may accelerate adaptive evolution.<a reveal-id="dsy002-B49" data-open="dsy002-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> Analysis of structural variation suggests that CNVs in the N85 genome may underlie niche adaptation.</p>                            <h2 scrollto-destination=118057382 id="118057382" class="section-title" >5. Conclusion</h2>
<p>We combined sequencing data from Illumina and PacBio sequencing platforms to generate the first gapless and near-complete genome assembly of the <em>S. cerevisiae</em> strain N85 industrial strain used in Chinese rice wine brewing. Our study revealed many genes and genetic variations that may help the strain to cope with the high glucose and ethanol concentrations in the fermentation environment. Industrial rice wine producers, even other beverages producers, will likely benefit from having access to a complete N85 genome to improve their production progresses and products quality. In addition, our findings provide a rich genetic resource for the <em>S. cerevisiae</em> fundamental and applied research communities.</p>                            <h3 scrollto-destination=118057385 id="118057385" class="section-title" >Funding</h3>
<p>This work was supported by The National Key Research and Development Programme of China (2017YFC1600403), the National Natural Science Foundation of China (31670095, 31770097), the Key Research and Development Programme of Jiangsu Province (BE2016689), the Fundamental Research Funds for the Central Universities (JUSRP51701A), the Six Talent Peaks Project in Jiangsu Province (2015-JY-005), the Distinguished Professor Project of Jiangsu Province, and a grant from Zhejiang Provincial Natural Science Foundation of China (LY14C060001) to Y. Li.</p>                            <h3 scrollto-destination=118057387 id="118057387" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=118057389 id="118057389" class="section-title" >Accession numbers</h3>
<p>EMBL:LN907784, LN907785, LN907786, LN907787, LN907788, LN907789, LN907790, LN907791, LN907792, LN907793, LN907794, LN907795, LN907796, LN907797, LN907798, LN907799, LN907800.</p>                            <h3 scrollto-destination=118057391 id="118057391" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/dsy002_supplemental_materials.doc?Expires=2147483647&Signature=M4Zaqb0hkRb9QoBS1HCN63DxI~kdb6LDai3YWKMpeQHJWqCZmPrEV3ZfueyUStqu3bJJ4YBwOfVSEUVQ-7FuWaz0sFXAdThZ-9XhIWBZPczI~FzmnCookOMb5mYoI7rHiYukm-HrJ3yN86ah8CFhL72Yx6W5YJEMwUkBBYlsCGvW~DsAv06pLtyAOx28FSbPZCx3pb2Z3kS4y0V1shBrMPcsBCbfpJQUv2dSnTJRRlFPSkInshs3k6zr26leTimg2YZcRVMrI0r4t1OLiuGQQZjotNrwLsZQbjDhmEi3toscs0c7XgxXUJBxLBQ5WX7CQuBdab-fUF~ec06uNX8ivg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h2 scrollto-destination=118057393 id="118057393" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsy002-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">McGovern</div> <div class="given-names">P. E.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2004</div>, 
<div class="article-title">Fermented beverages of pre- and proto-historic China</div>, <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">101</div>, <div class="fpage">17593</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fermented%20beverages%20of%20pre-%20and%20proto-historic%20China&amp;author=P.%20E.%20McGovern&amp;author=J.%20Zhang&amp;author=J.%20Tang&amp;publication_year=2004&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=101&amp;pages=17593-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0407921102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0407921102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0407921102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15590771" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guangfa</div> <div class="given-names">X.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Functional constituents and health function of Chinese rice wine</div>, <div class="source">Liquor Making</div> , <div class="volume">35</div>, <div class="fpage">14</div>–<div class="lpage">6</div>.</div></div></div></div><div content-id="dsy002-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Du</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Zou</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Fu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">J.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Progress in preventing the accumulation of ethyl carbamate in alcoholic beverages</div>, <div class="source">Trends Food Sci. Tech</div> ., <div class="volume">32</div>, <div class="fpage">97</div>–<div class="lpage">107</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Progress%20in%20preventing%20the%20accumulation%20of%20ethyl%20carbamate%20in%20alcoholic%20beverages&amp;author=X.%20Zhao&amp;author=G.%20Du&amp;author=H.%20Zou&amp;author=J.%20Fu&amp;author=J.%20Zhou&amp;author=J.%20Chen&amp;publication_year=2013&amp;journal=Trends%20Food%20Sci.%20Tech&amp;volume=32&amp;pages=97-107" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tifs.2013.05.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tifs.2013.05.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tifs.2013.05.009"> </span></div> </div></div></div></div></div><div content-id="dsy002-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Lu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Xie</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Poeggeler</div> <div class="given-names">S.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Constitutive expression of the <em>DUR1, 2</em> gene in an industrial yeast strain to minimize ethyl carbamate production during Chinese rice wine fermentation</div>, <div class="source">FEMS Microbiol. Lett</div> ., <div class="volume">363</div>, <div class="fpage">fnv214</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Constitutive%20expression%20of%20the%20DUR1%2C%202%20gene%20in%20an%20industrial%20yeast%20strain%20to%20minimize%20ethyl%20carbamate%20production%20during%20Chinese%20rice%20wine%20fermentation&amp;author=D.%20Wu&amp;author=X.%20Li&amp;author=J.%20Lu&amp;author=J.%20Chen&amp;author=L.%20Zhang&amp;author=G.%20Xie&amp;author=S.%20Poeggeler&amp;publication_year=2016&amp;journal=FEMS%20Microbiol.%20Lett&amp;volume=363&amp;pages=fnv214" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/femsle/fnv214" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Ffemsle%2Ffnv214" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Ffemsle%2Ffnv214"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26538578" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dahabieh</div> <div class="given-names">M. S.</div></div>, <div class="name string-name"><div class="surname">Husnik</div> <div class="given-names">J. I.</div></div>, <div class="name string-name"><div class="surname">Van Vuuren</div> <div class="given-names">H. J.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Functional enhancement of Sake yeast strains to minimize the production of ethyl carbamate in Sake wine</div>, <div class="source">J. Appl. Microbiol.</div>  , <div class="volume">109</div>, <div class="fpage">963</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20enhancement%20of%20Sake%20yeast%20strains%20to%20minimize%20the%20production%20of%20ethyl%20carbamate%20in%20Sake%20wine&amp;author=M.%20S.%20Dahabieh&amp;author=J.%20I.%20Husnik&amp;author=H.%20J.%20Van%20Vuuren&amp;publication_year=2010&amp;journal=J.%20Appl.%20Microbiol.&amp;volume=109&amp;pages=963-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jam.2010.109.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjam.2010.109.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjam.2010.109.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20408912" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Zou</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Fu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Du</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">J.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Comparative proteomic analysis of Saccharomyces cerevisiae under different nitrogen sources</div>, <div class="source">J. Proteomics</div> , <div class="volume">101</div>, <div class="fpage">102</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20proteomic%20analysis%20of%20Saccharomyces%20cerevisiae%20under%20different%20nitrogen%20sources&amp;author=S.%20Zhao&amp;author=X.%20Zhao&amp;author=H.%20Zou&amp;author=J.%20Fu&amp;author=G.%20Du&amp;author=J.%20Zhou&amp;author=J.%20Chen&amp;publication_year=2014&amp;journal=J.%20Proteomics&amp;volume=101&amp;pages=102-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jprot.2014.01.031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jprot.2014.01.031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jprot.2014.01.031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24530623" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Zou</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Fu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Du</div> <div class="given-names">G.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Nitrogen regulation involved in the accumulation of urea in <em>Saccharomyces cerevisiae</em></div>, <div class="source">Yeast</div> , <div class="volume">30</div>, <div class="fpage">437</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nitrogen%20regulation%20involved%20in%20the%20accumulation%20of%20urea%20in%20Saccharomyces%20cerevisiae&amp;author=X.%20Zhao&amp;author=H.%20Zou&amp;author=J.%20Fu&amp;author=J.%20Chen&amp;author=J.%20Zhou&amp;author=G.%20Du&amp;publication_year=2013&amp;journal=Yeast&amp;volume=30&amp;pages=437-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/yea.v30.11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fyea.v30.11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fyea.v30.11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23996237" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Zou</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Fu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Du</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">J.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Metabolic engineering of the regulators in nitrogen catabolite repression to reduce the production of ethyl carbamate in a model rice wine system</div>, <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">80</div>, <div class="fpage">392</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metabolic%20engineering%20of%20the%20regulators%20in%20nitrogen%20catabolite%20repression%20to%20reduce%20the%20production%20of%20ethyl%20carbamate%20in%20a%20model%20rice%20wine%20system&amp;author=X.%20Zhao&amp;author=H.%20Zou&amp;author=J.%20Fu&amp;author=J.%20Zhou&amp;author=G.%20Du&amp;author=J.%20Chen&amp;publication_year=2014&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=80&amp;pages=392-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.03055-13" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.03055-13" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.03055-13"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24185848" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wiame</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Grenson</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Arst</div> <div class="given-names">H. N.</div>Jr.</div></div> <div class="year">1985</div>, 
<div class="article-title">Nitrogen catabolite repression in yeasts and filamentous fungi</div>, <div class="source">Adv. Microb. Physiol</div> ., <div class="volume">26</div>, <div class="fpage">1</div>–<div class="lpage">88</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nitrogen%20catabolite%20repression%20in%20yeasts%20and%20filamentous%20fungi&amp;author=J.%20M.%20Wiame&amp;author=M.%20Grenson&amp;author=H.%20N.%20Arst&amp;publication_year=1985&amp;journal=Adv.%20Microb.%20Physiol&amp;volume=26&amp;pages=1-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0065-2911(08)60394-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0065-2911(08)60394-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0065-2911(08)60394-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2869649" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Beltran</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Novo</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Rozes</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Mas</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Guillamon</div> <div class="given-names">J. M.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Nitrogen catabolite repression in Saccharomyces cerevisiae during wine fermentations</div>, <div class="source">FEMS Yeast Res</div> ., <div class="volume">4</div>, <div class="fpage">625</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nitrogen%20catabolite%20repression%20in%20Saccharomyces%20cerevisiae%20during%20wine%20fermentations&amp;author=G.%20Beltran&amp;author=M.%20Novo&amp;author=N.%20Rozes&amp;author=A.%20Mas&amp;author=J.%20M.%20Guillamon&amp;publication_year=2004&amp;journal=FEMS%20Yeast%20Res&amp;volume=4&amp;pages=625-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.femsyr.2003.12.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.femsyr.2003.12.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.femsyr.2003.12.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15040951" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tate</div> <div class="given-names">J. J.</div></div>, <div class="name string-name"><div class="surname">Buford</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Rai</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Cooper</div> <div class="given-names">T. G.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">General amino acid control and 14-3-3 proteins Bmh1/2 are required for nitrogen catabolite repression-sensitive regulation of Gln3 and Gat1 localization</div>, <div class="source">Genetics</div> , <div class="volume">205</div>, <div class="fpage">633</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=General%20amino%20acid%20control%20and%2014-3-3%20proteins%20Bmh1%2F2%20are%20required%20for%20nitrogen%20catabolite%20repression-sensitive%20regulation%20of%20Gln3%20and%20Gat1%20localization&amp;author=J.%20J.%20Tate&amp;author=D.%20Buford&amp;author=R.%20Rai&amp;author=T.%20G.%20Cooper&amp;publication_year=2017&amp;journal=Genetics&amp;volume=205&amp;pages=633-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.116.195800" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.116.195800" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.116.195800"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28007891" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Zou</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Du</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">J.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">The modification of Gat1p in nitrogen catabolite repression to enhance non-preferred nitrogen utilization in <em>Saccharomyces cerevisiae</em></div>, <div class="source">Sci. Rep</div> ., <div class="volume">6</div>, <div class="fpage">21603</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20modification%20of%20Gat1p%20in%20nitrogen%20catabolite%20repression%20to%20enhance%20non-preferred%20nitrogen%20utilization%20in%20Saccharomyces%20cerevisiae&amp;author=X.%20Zhao&amp;author=H.%20Zou&amp;author=J.%20Chen&amp;author=G.%20Du&amp;author=J.%20Zhou&amp;publication_year=2016&amp;journal=Sci.%20Rep&amp;volume=6&amp;pages=21603" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep21603" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep21603" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep21603"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26899143" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Orlova</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kanter</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Krakovich</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Kuchin</div> <div class="given-names">S.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Nitrogen availability and TOR regulate the Snf1 protein kinase in <em>Saccharomyces cerevisiae</em></div>, <div class="source">Eukaryot. Cell</div> , <div class="volume">5</div>, <div class="fpage">1831</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nitrogen%20availability%20and%20TOR%20regulate%20the%20Snf1%20protein%20kinase%20in%20Saccharomyces%20cerevisiae&amp;author=M.%20Orlova&amp;author=E.%20Kanter&amp;author=D.%20Krakovich&amp;author=S.%20Kuchin&amp;publication_year=2006&amp;journal=Eukaryot.%20Cell&amp;volume=5&amp;pages=1831-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/EC.00110-06" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FEC.00110-06" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FEC.00110-06"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16980405" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ljungdahl</div> <div class="given-names">P. O.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Amino-acid-induced signalling via the SPS-sensing pathway in yeast</div>, <div class="source">Biochem. Soc. Trans</div> ., <div class="volume">37</div>, <div class="fpage">242</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Amino-acid-induced%20signalling%20via%20the%20SPS-sensing%20pathway%20in%20yeast&amp;author=P.%20O.%20Ljungdahl&amp;publication_year=2009&amp;journal=Biochem.%20Soc.%20Trans&amp;volume=37&amp;pages=242-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/BST0370242" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2FBST0370242" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2FBST0370242"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19143640" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Zheng</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Genomic evolution of Saccharomyces cerevisiae under Chinese rice wine fermentation</div>, <div class="source">Genome Biol. Evol</div> ., <div class="volume">6</div>, <div class="fpage">2516</div>–<div class="lpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20evolution%20of%20Saccharomyces%20cerevisiae%20under%20Chinese%20rice%20wine%20fermentation&amp;author=Y.%20Li&amp;author=W.%20Zhang&amp;author=D.%20Zheng&amp;publication_year=2014&amp;journal=Genome%20Biol.%20Evol&amp;volume=6&amp;pages=2516-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evu201" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevu201" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevu201"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25212861" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Goodwin</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">McPherson</div> <div class="given-names">J. D.</div></div>, <div class="name string-name"><div class="surname">McCombie</div> <div class="given-names">W. R.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Coming of age: ten years of next-generation sequencing technologies</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">17</div>, <div class="fpage">333</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coming%20of%20age%3A%20ten%20years%20of%20next-generation%20sequencing%20technologies&amp;author=S.%20Goodwin&amp;author=J.%20D.%20McPherson&amp;author=W.%20R.%20McCombie&amp;publication_year=2016&amp;journal=Nat.%20Rev.%20Genet&amp;volume=17&amp;pages=333-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg.2016.49" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg.2016.49" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg.2016.49"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27184599" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gnerre</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Maccallum</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Przybylski</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">High-quality draft assemblies of mammalian genomes from massively parallel sequence data</div>, <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">108</div>, <div class="fpage">1513</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-quality%20draft%20assemblies%20of%20mammalian%20genomes%20from%20massively%20parallel%20sequence%20data&amp;author=S.%20Gnerre&amp;author=I.%20Maccallum&amp;author=D.%20Przybylski&amp;publication_year=2011&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=108&amp;pages=1513-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1017351108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1017351108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1017351108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21187386" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nakano</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Shiroma</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Shimoji</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">Advantages of genome sequencing by long-read sequencer using SMRT technology in medical area</div>, <div class="source">Hum. Cell</div> ., <div class="volume">30</div>, <div class="fpage">149</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Advantages%20of%20genome%20sequencing%20by%20long-read%20sequencer%20using%20SMRT%20technology%20in%20medical%20area&amp;author=K.%20Nakano&amp;author=A.%20Shiroma&amp;author=M.%20Shimoji&amp;publication_year=2017&amp;journal=Hum.%20Cell&amp;volume=30&amp;pages=149-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s13577-017-0168-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs13577-017-0168-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs13577-017-0168-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28364362" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kamada</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hase</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sato</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Toyoda</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Fujiyama</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Sakakibara</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Schacherer</div> <div class="given-names">J.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Whole genome complete resequencing of <em>Bacillus subtilis</em> natto by combining long reads with high-quality short reads</div>, <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e109999</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole%20genome%20complete%20resequencing%20of%20Bacillus%20subtilis%20natto%20by%20combining%20long%20reads%20with%20high-quality%20short%20reads&amp;author=M.%20Kamada&amp;author=S.%20Hase&amp;author=K.%20Sato&amp;author=A.%20Toyoda&amp;author=A.%20Fujiyama&amp;author=Y.%20Sakakibara&amp;author=J.%20Schacherer&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e109999" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0109999" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0109999" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0109999"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25329997" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Coil</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Jospin</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Darling</div> <div class="given-names">A. E.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">A5-miseq: an updated pipeline to assemble microbial genomes from Illumina MiSeq data</div>, <div class="source">Bioinformatics</div> , <div class="volume">31</div>, <div class="fpage">587</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A5-miseq%3A%20an%20updated%20pipeline%20to%20assemble%20microbial%20genomes%20from%20Illumina%20MiSeq%20data&amp;author=D.%20Coil&amp;author=G.%20Jospin&amp;author=A.%20E.%20Darling&amp;publication_year=2015&amp;journal=Bioinformatics&amp;volume=31&amp;pages=587-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu661" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu661" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu661"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25338718" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Boetzer</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pirovano</div> <div class="given-names">W.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">SSPACE-LongRead: scaffolding bacterial draft genomes using long read sequence information</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">15</div>, <div class="fpage">211</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SSPACE-LongRead%3A%20scaffolding%20bacterial%20draft%20genomes%20using%20long%20read%20sequence%20information&amp;author=M.%20Boetzer&amp;author=W.%20Pirovano&amp;publication_year=2014&amp;journal=BMC%20Bioinformatics&amp;volume=15&amp;pages=211" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-15-211" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-15-211" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-15-211"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24950923" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bankevich</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Nurk</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Antipov</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing</div>, <div class="source">J. Comput. Biol</div> ., <div class="volume">19</div>, <div class="fpage">455</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SPAdes%3A%20a%20new%20genome%20assembly%20algorithm%20and%20its%20applications%20to%20single-cell%20sequencing&amp;author=A.%20Bankevich&amp;author=S.%20Nurk&amp;author=D.%20Antipov&amp;publication_year=2012&amp;journal=J.%20Comput.%20Biol&amp;volume=19&amp;pages=455-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/cmb.2012.0021" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fcmb.2012.0021" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2Fcmb.2012.0021"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22506599" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Berlin</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Koren</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Chin</div> <div class="given-names">C. S.</div></div>, <div class="name string-name"><div class="surname">Drake</div> <div class="given-names">J. P.</div></div>, <div class="name string-name"><div class="surname">Landolin</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Phillippy</div> <div class="given-names">A. M.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Assembling large genomes with single-molecule sequencing and locality-sensitive hashing</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">33</div>, <div class="fpage">623</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assembling%20large%20genomes%20with%20single-molecule%20sequencing%20and%20locality-sensitive%20hashing&amp;author=K.%20Berlin&amp;author=S.%20Koren&amp;author=C.%20S.%20Chin&amp;author=J.%20P.%20Drake&amp;author=J.%20M.%20Landolin&amp;author=A.%20M.%20Phillippy&amp;publication_year=2015&amp;journal=Nat.%20Biotechnol&amp;volume=33&amp;pages=623-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.3238" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.3238" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.3238"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26006009" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chin</div> <div class="given-names">C. S.</div></div>, <div class="name string-name"><div class="surname">Alexander</div> <div class="given-names">D. H.</div></div>, <div class="name string-name"><div class="surname">Marks</div> <div class="given-names">P.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data</div>, <div class="source">Nat. Methods.</div>  , <div class="volume">10</div>, <div class="fpage">563</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nonhybrid%2C%20finished%20microbial%20genome%20assemblies%20from%20long-read%20SMRT%20sequencing%20data&amp;author=C.%20S.%20Chin&amp;author=D.%20H.%20Alexander&amp;author=P.%20Marks&amp;publication_year=2013&amp;journal=Nat.%20Methods.&amp;volume=10&amp;pages=563-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.2474" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2474" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.2474"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23644548" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Diekhans</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Baertsch</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Haussler</div> <div class="given-names">D.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Using native and syntenically mapped cDNA alignments to improve de novo gene finding</div>, <div class="source">Bioinformatics</div> , <div class="volume">24</div>, <div class="fpage">637</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20native%20and%20syntenically%20mapped%20cDNA%20alignments%20to%20improve%20de%20novo%20gene%20finding&amp;author=M.%20Stanke&amp;author=M.%20Diekhans&amp;author=R.%20Baertsch&amp;author=D.%20Haussler&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=637-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18218656" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Roberts</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Goff</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks</div>, <div class="source">Nat. Protoc</div> ., <div class="volume">7</div>, <div class="fpage">562</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20gene%20and%20transcript%20expression%20analysis%20of%20RNA-seq%20experiments%20with%20TopHat%20and%20Cufflinks&amp;author=C.%20Trapnell&amp;author=A.%20Roberts&amp;author=L.%20Goff&amp;publication_year=2012&amp;journal=Nat.%20Protoc&amp;volume=7&amp;pages=562-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2012.016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2012.016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2012.016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22383036" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hoff</div> <div class="given-names">K.</div></div></div> <div class="year">2014</div>, Incorporating RNA-Seq into AUGUSTUS with TOPHAT. Avialable at: <a class="link link-uri" href="http://bioinf.uni-greifswald.de/bioinf/wiki/pmwiki.php? n=IncorporatingRNAseq.Tophat" target="_blank">http://bioinf.uni-greifswald.de/bioinf/wiki/pmwiki.php? n=IncorporatingRNAseq.Tophat</a> (19 January 2018, date last accessed).</div></div></div></div><div content-id="dsy002-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Grabherr</div> <div class="given-names">M. G.</div></div>, <div class="name string-name"><div class="surname">Haas</div> <div class="given-names">B. J.</div></div>, <div class="name string-name"><div class="surname">Yassour</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Trinity: reconstructing a full-length transcriptome without a genome from RNA-Seq data</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">29</div>, <div class="fpage">644</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Trinity%3A%20reconstructing%20a%20full-length%20transcriptome%20without%20a%20genome%20from%20RNA-Seq%20data&amp;author=M.%20G.%20Grabherr&amp;author=B.%20J.%20Haas&amp;author=M.%20Yassour&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol&amp;volume=29&amp;pages=644-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1883" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1883" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1883"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Slater</div> <div class="given-names">G. S.</div></div>, <div class="name string-name"><div class="surname">Birney</div> <div class="given-names">E.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Automated generation of heuristics for biological sequence comparison</div>, <div class="source">BMC Bioinformatics</div> ., <div class="volume">6</div>, <div class="fpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20generation%20of%20heuristics%20for%20biological%20sequence%20comparison&amp;author=G.%20S.%20Slater&amp;author=E.%20Birney&amp;publication_year=2005&amp;journal=BMC%20Bioinformatics&amp;volume=6&amp;pages=31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-6-31" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-6-31" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-6-31"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15713233" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rogers</div> <div class="given-names">M. F.</div></div>, <div class="name string-name"><div class="surname">Thomas</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Reddy</div> <div class="given-names">A. S.</div></div>, <div class="name string-name"><div class="surname">Ben-Hur</div> <div class="given-names">A.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">SpliceGrapher: detecting patterns of alternative splicing from RNA-Seq data in the context of gene models and EST data</div>, <div class="source">Genome Biol</div> ., <div class="volume">13</div>, <div class="fpage">R4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SpliceGrapher%3A%20detecting%20patterns%20of%20alternative%20splicing%20from%20RNA-Seq%20data%20in%20the%20context%20of%20gene%20models%20and%20EST%20data&amp;author=M.%20F.%20Rogers&amp;author=J.%20Thomas&amp;author=A.%20S.%20Reddy&amp;author=A.%20Ben-Hur&amp;publication_year=2012&amp;journal=Genome%20Biol&amp;volume=13&amp;pages=R4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2012-13-1-r4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2012-13-1-r4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2012-13-1-r4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22293517" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Thorvaldsdottir</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Robinson</div> <div class="given-names">J. T.</div></div>, <div class="name string-name"><div class="surname">Mesirov</div> <div class="given-names">J. P.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration</div>, <div class="source">Brief Bioinform</div> ., <div class="volume">14</div>, <div class="fpage">178</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrative%20Genomics%20Viewer%20(IGV)%3A%20high-performance%20genomics%20data%20visualization%20and%20exploration&amp;author=H.%20Thorvaldsdottir&amp;author=J.%20T.%20Robinson&amp;author=J.%20P.%20Mesirov&amp;publication_year=2013&amp;journal=Brief%20Bioinform&amp;volume=14&amp;pages=178-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/bbs017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2Fbbs017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbib%2Fbbs017"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22517427" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div></div> <div class="year">2013</div>, Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. <em>arXiv preprint arXiv: 1303.3997</em>.</div></div></div></div><div content-id="dsy002-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Quinlan</div> <div class="given-names">A. R.</div></div>, <div class="name string-name"><div class="surname">Hall</div> <div class="given-names">I. M.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">BEDTools: a flexible suite of utilities for comparing genomic features</div>, <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">841</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BEDTools%3A%20a%20flexible%20suite%20of%20utilities%20for%20comparing%20genomic%20features&amp;author=A.%20R.%20Quinlan&amp;author=I.%20M.%20Hall&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=841-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20110278" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ha</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Roth</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Lai</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Integrative analysis of genome-wide loss of heterozygosity and monoallelic expression at nucleotide resolution reveals disrupted pathways in triple-negative breast cancer</div>, <div class="source">Genome Res</div> ., <div class="volume">22</div>, <div class="fpage">1995</div>–<div class="lpage">2007</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrative%20analysis%20of%20genome-wide%20loss%20of%20heterozygosity%20and%20monoallelic%20expression%20at%20nucleotide%20resolution%20reveals%20disrupted%20pathways%20in%20triple-negative%20breast%20cancer&amp;author=G.%20Ha&amp;author=A.%20Roth&amp;author=D.%20Lai&amp;publication_year=2012&amp;journal=Genome%20Res&amp;volume=22&amp;pages=1995-2007" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.137570.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.137570.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.137570.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22637570" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Koren</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Schatz</div> <div class="given-names">M. C.</div></div>, <div class="name string-name"><div class="surname">Walenz</div> <div class="given-names">B. P.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Hybrid error correction and de novo assembly of single-molecule sequencing reads</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">30</div>, <div class="fpage">693</div>–<div class="lpage">700</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hybrid%20error%20correction%20and%20de%20novo%20assembly%20of%20single-molecule%20sequencing%20reads&amp;author=S.%20Koren&amp;author=M.%20C.%20Schatz&amp;author=B.%20P.%20Walenz&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol&amp;volume=30&amp;pages=693-700" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2280" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2280" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2280"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22750884" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Garber</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Grabherr</div> <div class="given-names">M. G.</div></div>, <div class="name string-name"><div class="surname">Guttman</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Computational methods for transcriptome annotation and quantification using RNA-seq</div>, <div class="source">Nat. Methods</div> , <div class="volume">8</div>, <div class="fpage">469</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20methods%20for%20transcriptome%20annotation%20and%20quantification%20using%20RNA-seq&amp;author=M.%20Garber&amp;author=M.%20G.%20Grabherr&amp;author=M.%20Guttman&amp;author=C.%20Trapnell&amp;publication_year=2011&amp;journal=Nat.%20Methods&amp;volume=8&amp;pages=469-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1613" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1613" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1613"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21623353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Shi</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Liang</div> <div class="given-names">X.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Molecular cloning and evolutionary analysis of the HOG-signaling pathway genes from <em>Saccharomyces cerevisiae</em> rice wine isolates</div>, <div class="source">Biochem. Genet</div> ., <div class="volume">51</div>, <div class="fpage">296</div>–<div class="lpage">305</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20cloning%20and%20evolutionary%20analysis%20of%20the%20HOG-signaling%20pathway%20genes%20from%20Saccharomyces%20cerevisiae%20rice%20wine%20isolates&amp;author=Y.%20Li&amp;author=W.%20Chen&amp;author=Y.%20Shi&amp;author=X.%20Liang&amp;publication_year=2013&amp;journal=Biochem.%20Genet&amp;volume=51&amp;pages=296-305" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10528-012-9563-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10528-012-9563-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10528-012-9563-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23338673" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Selecký</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Šmogrovičová</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Sulo</div> <div class="given-names">P.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Beer with reduced ethanol content produced using <em>Saccharomyces cerevisiae</em> yeasts deficient in various tricarboxylic acid cycle enzymes</div>, <div class="source">J. Inst. Brew</div> ., <div class="volume">114</div>, <div class="fpage">97</div>–<div class="lpage">101</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Beer%20with%20reduced%20ethanol%20content%20produced%20using%20Saccharomyces%20cerevisiae%20yeasts%20deficient%20in%20various%20tricarboxylic%20acid%20cycle%20enzymes&amp;author=R.%20Seleck%C3%BD&amp;author=D.%20%C5%A0mogrovi%C4%8Dov%C3%A1&amp;author=P.%20Sulo&amp;publication_year=2008&amp;journal=J.%20Inst.%20Brew&amp;volume=114&amp;pages=97-101" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)2050-0416" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)2050-0416" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)2050-0416"> </span></div> </div></div></div></div></div><div content-id="dsy002-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Howell</div> <div class="given-names">K. S.</div></div>, <div class="name string-name"><div class="surname">Klein</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Swiegers</div> <div class="given-names">J. H.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">Genetic determinants of volatile-thiol release by <em>Saccharomyces cerevisiae</em> during wine fermentation</div>, <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">71</div>, <div class="fpage">5420</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20determinants%20of%20volatile-thiol%20release%20by%20Saccharomyces%20cerevisiae%20during%20wine%20fermentation&amp;author=K.%20S.%20Howell&amp;author=M.%20Klein&amp;author=J.%20H.%20Swiegers&amp;publication_year=2005&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=71&amp;pages=5420-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.71.9.5420-5426.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.71.9.5420-5426.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.71.9.5420-5426.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16151133" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Que</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Mao</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Xie</div> <div class="given-names">G.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Antioxidant properties of Chinese yellow wine, its concentrate and volatiles</div>, <div class="source">LWT Food Sci. Technol</div> ., <div class="volume">39</div>, <div class="fpage">111</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Antioxidant%20properties%20of%20Chinese%20yellow%20wine%2C%20its%20concentrate%20and%20volatiles&amp;author=F.%20Que&amp;author=L.%20Mao&amp;author=C.%20Zhu&amp;author=G.%20Xie&amp;publication_year=2006&amp;journal=LWT%20Food%20Sci.%20Technol&amp;volume=39&amp;pages=111-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.lwt.2005.01.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.lwt.2005.01.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.lwt.2005.01.001"> </span></div> </div></div></div></div></div><div content-id="dsy002-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wei</div> <div class="given-names">X. L.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">S. P.</div></div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">J. S.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">Innovation Chinese rice wine brewing technology by bi-acidification to exclude rice soaking process</div>, <div class="source">J. Biosci. Bioeng</div> ., <div class="volume">123</div>, <div class="fpage">460</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Innovation%20Chinese%20rice%20wine%20brewing%20technology%20by%20bi-acidification%20to%20exclude%20rice%20soaking%20process&amp;author=X.%20L.%20Wei&amp;author=S.%20P.%20Liu&amp;author=J.%20S.%20Yu&amp;publication_year=2017&amp;journal=J.%20Biosci.%20Bioeng&amp;volume=123&amp;pages=460-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jbiosc.2016.11.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jbiosc.2016.11.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jbiosc.2016.11.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28043775" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fang</div> <div class="given-names">R. S.</div></div>, <div class="name string-name"><div class="surname">Dong</div> <div class="given-names">Y. C.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">H. J.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">Q. H.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Ethyl carbamate formation regulated by <em>Saccharomyces cerevisiae</em> ZJU in the processing of Chinese yellow rice wine</div>, <div class="source">Int. J. Food Sci. Technol</div> ., <div class="volume">50</div>, <div class="fpage">626</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ethyl%20carbamate%20formation%20regulated%20by%20Saccharomyces%20cerevisiae%20ZJU%20in%20the%20processing%20of%20Chinese%20yellow%20rice%20wine&amp;author=R.%20S.%20Fang&amp;author=Y.%20C.%20Dong&amp;author=H.%20J.%20Li&amp;author=Q.%20H.%20Chen&amp;publication_year=2015&amp;journal=Int.%20J.%20Food%20Sci.%20Technol&amp;volume=50&amp;pages=626-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/ijfs.2015.50.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fijfs.2015.50.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fijfs.2015.50.issue-3"> </span></div> </div></div></div></div></div><div content-id="dsy002-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Au</div> <div class="given-names">K. F.</div></div>, <div class="name string-name"><div class="surname">Underwood</div> <div class="given-names">J. G.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Wong</div> <div class="given-names">W. H.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Improving PacBio long read accuracy by short read alignment</div>, <div class="source">PLoS One</div> ., <div class="volume">7</div>, <div class="fpage">e46679</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improving%20PacBio%20long%20read%20accuracy%20by%20short%20read%20alignment&amp;author=K.%20F.%20Au&amp;author=J.%20G.%20Underwood&amp;author=L.%20Lee&amp;author=W.%20H.%20Wong&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e46679" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0046679" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0046679" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0046679"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23056399" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Akao</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Yashiro</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Hosoyama</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Whole-genome sequencing of sake yeast Saccharomyces cerevisiae Kyokai no. 7</div>, <div class="source">DNA Res</div> ., <div class="volume">18</div>, <div class="fpage">423</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20sequencing%20of%20sake%20yeast%20Saccharomyces%20cerevisiae%20Kyokai%20no.%207&amp;author=T.%20Akao&amp;author=I.%20Yashiro&amp;author=A.%20Hosoyama&amp;publication_year=2011&amp;journal=DNA%20Res&amp;volume=18&amp;pages=423-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsr029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsr029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsr029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21900213" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rinn</div> <div class="given-names">J. L.</div></div>, <div class="name string-name"><div class="surname">Chang</div> <div class="given-names">H. Y.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Genome regulation by long noncoding RNAs</div>, <div class="source">Annu. Rev. Biochem</div> ., <div class="volume">81</div>, <div class="fpage">145</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20regulation%20by%20long%20noncoding%20RNAs&amp;author=J.%20L.%20Rinn&amp;author=H.%20Y.%20Chang&amp;publication_year=2012&amp;journal=Annu.%20Rev.%20Biochem&amp;volume=81&amp;pages=145-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-biochem-051410-092902" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-biochem-051410-092902" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-biochem-051410-092902"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22663078" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Thum</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Condorelli</div> <div class="given-names">G.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Long noncoding RNAs and microRNAs in cardiovascular pathophysiology</div>, <div class="source">Circ. Res</div> ., <div class="volume">116</div>, <div class="fpage">751</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Long%20noncoding%20RNAs%20and%20microRNAs%20in%20cardiovascular%20pathophysiology&amp;author=T.%20Thum&amp;author=G.%20Condorelli&amp;publication_year=2015&amp;journal=Circ.%20Res&amp;volume=116&amp;pages=751-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1161/CIRCRESAHA.116.303549" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1161%2FCIRCRESAHA.116.303549" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1161%2FCIRCRESAHA.116.303549"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25677521" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rai</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Tate</div> <div class="given-names">J. J.</div></div>, <div class="name string-name"><div class="surname">Georis</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Dubois</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Cooper</div> <div class="given-names">T. G.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Constitutive and nitrogen catabolite repression-sensitive production of Gat1 isoforms</div>, <div class="source">J. Biol. Chem</div> ., <div class="volume">289</div>, <div class="fpage">2918</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Constitutive%20and%20nitrogen%20catabolite%20repression-sensitive%20production%20of%20Gat1%20isoforms&amp;author=R.%20Rai&amp;author=J.%20J.%20Tate&amp;author=I.%20Georis&amp;author=E.%20Dubois&amp;author=T.%20G.%20Cooper&amp;publication_year=2014&amp;journal=J.%20Biol.%20Chem&amp;volume=289&amp;pages=2918-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M113.516740" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M113.516740" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M113.516740"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24324255" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gallone</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Steensels</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Prahl</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Domestication and divergence of <em>Saccharomyces cerevisiae</em> beer yeasts</div>, <div class="source">Cell</div> , <div class="volume">166</div>, <div class="fpage">1397</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Domestication%20and%20divergence%20of%20Saccharomyces%20cerevisiae%20beer%20yeasts&amp;author=B.%20Gallone&amp;author=J.%20Steensels&amp;author=T.%20Prahl&amp;publication_year=2016&amp;journal=Cell&amp;volume=166&amp;pages=1397" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2016.08.020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2016.08.020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2016.08.020"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27610566" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy002-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy002-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dunn</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Richter</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Kvitek</div> <div class="given-names">D. J.</div></div>, <div class="name string-name"><div class="surname">Pugh</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Sherlock</div> <div class="given-names">G.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Analysis of the Saccharomyces cerevisiae pan-genome reveals a pool of copy number variants distributed in diverse yeast strains from differing industrial environments</div>, <div class="source">Genome Res</div> ., <div class="volume">22</div>, <div class="fpage">908</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20Saccharomyces%20cerevisiae%20pan-genome%20reveals%20a%20pool%20of%20copy%20number%20variants%20distributed%20in%20diverse%20yeast%20strains%20from%20differing%20industrial%20environments&amp;author=B.%20Dunn&amp;author=C.%20Richter&amp;author=D.%20J.%20Kvitek&amp;author=T.%20Pugh&amp;author=G.%20Sherlock&amp;publication_year=2012&amp;journal=Genome%20Res&amp;volume=22&amp;pages=908-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.130310.111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.130310.111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.130310.111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22369888" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2018. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4838783&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/dsy002_supplemental_materials.doc?Expires=2147483647&amp;Signature=M4Zaqb0hkRb9QoBS1HCN63DxI~kdb6LDai3YWKMpeQHJWqCZmPrEV3ZfueyUStqu3bJJ4YBwOfVSEUVQ-7FuWaz0sFXAdThZ-9XhIWBZPczI~FzmnCookOMb5mYoI7rHiYukm-HrJ3yN86ah8CFhL72Yx6W5YJEMwUkBBYlsCGvW~DsAv06pLtyAOx28FSbPZCx3pb2Z3kS4y0V1shBrMPcsBCbfpJQUv2dSnTJRRlFPSkInshs3k6zr26leTimg2YZcRVMrI0r4t1OLiuGQQZjotNrwLsZQbjDhmEi3toscs0c7XgxXUJBxLBQ5WX7CQuBdab-fUF~ec06uNX8ivg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Material</div></a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/dsy002_supplementary_table_s3.xlsx?Expires=2147483647&amp;Signature=GoS2c0eYpQx89wzlHuUp03xO9EmUo5Qk5fV~13EB88KmlKdOBxFYpb8LHixy0780rl1PiKOlOMKKOMELoHLsurbgLVR7ypOM7hEBEO7vHy7vrhGZ2SgwXB3LZTBJZRqNSytPtxMYFhtz6Ry~s~fCGIr8ZGxm3Xs15cVEiG~NJl5zI-CA1HPCUMTbMpUMdaXlO1YjRg--cklOGvjoWA8aqU3cCcSjKiXvpP9rsG8SRXvgAl1D286bIA-00sYEBQfl8bdJMAu84VQVKCZGnUBlzo4ttrCGy7XFkAnBq-Ys~rz9C9xmKx4uoyMv8AJybV0IHvaFzJ49bPWQcTCGJmLcCA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S3</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy002/1/dsy002_supplementary_table_s4.xlsx?Expires=2147483647&amp;Signature=iqZ02WaRRMKrdoaL596v1QY-WdrNtbObg7BbAB0HEM8yLb9V74RUeqlPkQf4JH-C8MEddvOVlqVZefWxwUcMtG8F8o84pInKQp1h0~gHrSuIheifDHxV11dr9ozcdd1UK5rj6BzsbkJ6CtkXS79KXfXr6XEk0XNmF8QOTF6Y2ikLu13eRJlsEnvAgyQyVE6NgfHf15aMqP5RHSJnI6g7qz7qq5h-9UDka7i4IMBBi9mX5kWopw7qJir3oQee907PQJHzHUOnATEfDmRRfTWki~6GuJAf08iNolaR3mdETVSlEEd7e5dZQArN8VpZdEbAhZ8jKHlDZqYAj99o~3L2mg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S4</div></a> - xlsx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,975</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsy002"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4838783"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f3%2f297%2f4838783">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F3%2F297%2F4838783" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4838783'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Complete genome sequence and analysis of the industrial Saccharomyces cerevisiae strain N85 used in Chinese rice wine production" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsy002");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f3%2f297%2f4838783">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

