

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/19/4/305/450433","@type":"ScholarlyArticle","name":"Evolution of Vertebrate Tissues Driven by Differential Modes of Gene Duplication","datePublished":"2012-04-05","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/19/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2012-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss012","keywords":["gene duplication","ohnologue","gene expression","tissue evolution"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/19/4/305/450433","author":[{"name":"Satake, Masanobu","affiliation":"1  Department of Molecular Immunology ,  Institute of Development, Aging and Cancer, Tohoku University ,  Seiryo-machi 4-1, Aoba-ku, Sendai 980-8575 ,  Japan","@type":"Person"},{"name":"Kawata, Masakado","affiliation":"2  Department of Ecology and Evolutionary Biology, Graduate School of Life Sciences ,  Tohoku University ,  Aramaki-Aoba 6-3, Aoba-ku, Sendai 980-8578 ,  Japan","@type":"Person"},{"name":"McLysaght, Aoife","affiliation":"3  Smurfit Institute of Genetics ,  University of Dublin, Trinity College ,  Dublin 2 ,  Ireland","@type":"Person"},{"name":"Makino, Takashi","affiliation":"2  Department of Ecology and Evolutionary Biology, Graduate School of Life Sciences ,  Tohoku University ,  Aramaki-Aoba 6-3, Aoba-ku, Sendai 980-8578 ,  Japan","@type":"Person"}],"description":"Abstract.  In this study, we investigated the evolution of vertebrate tissues by examining the potential association among gene expression, duplication, and bas","pageStart":"305","pageEnd":"316","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/4/1/m_cover.gif?Expires=2147483647&Signature=eprZ0s37mFZTW8q04XyXQPhslj5EsmNEiVd2xsNgTNSRMFCtDBeg3Jl-WpIZ04VSSHYdf4drJ9cM4eXR~wAjVX4Ka1lGx5jT4Aan8~ek1e8YFczc~efvkNtRr1j4OBLyYeTJMiA~vza484yAcFT5QrseynX-Et0Amq8jonZzJ4uDq5hZrvVMLdx8~LCdEtaG8K8Th5wGC0jLW5Dvd--ISvvgHuZMFgwQgbYRdBL~2OYDNhG3CqoT4aVgmKKF8B3aG~dK1DqjtB8y6Zeh2up0vNZSY74Ozsnuq5BVobk3Bk2WTZDMIWq3CmTTmKd9YkkZCnPAEOdYujN8-TWtB1D4Tw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Evolution of Vertebrate Tissues Driven by Differential Modes of Gene Duplication","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/4/1/m_cover.gif?Expires=2147483647&Signature=eprZ0s37mFZTW8q04XyXQPhslj5EsmNEiVd2xsNgTNSRMFCtDBeg3Jl-WpIZ04VSSHYdf4drJ9cM4eXR~wAjVX4Ka1lGx5jT4Aan8~ek1e8YFczc~efvkNtRr1j4OBLyYeTJMiA~vza484yAcFT5QrseynX-Et0Amq8jonZzJ4uDq5hZrvVMLdx8~LCdEtaG8K8Th5wGC0jLW5Dvd--ISvvgHuZMFgwQgbYRdBL~2OYDNhG3CqoT4aVgmKKF8B3aG~dK1DqjtB8y6Zeh2up0vNZSY74Ozsnuq5BVobk3Bk2WTZDMIWq3CmTTmKd9YkkZCnPAEOdYujN8-TWtB1D4Tw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Evolution of Vertebrate Tissues Driven by Differential Modes of Gene Duplication" />
    <meta property="og:description" content="Abstract.  In this study, we investigated the evolution of vertebrate tissues by examining the potential association among gene expression, duplication, and bas" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/19/4/305/450433" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2012-04-05" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Satake, Masanobu" /><meta name="citation_author_institution" content="Department of Molecular Immunology, Institute of Development, Aging and Cancer, Tohoku University, Seiryo-machi 4-1, Aoba-ku, Sendai 980-8575, Japan" /><meta name="citation_author" content="Kawata, Masakado" /><meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, Graduate School of Life Sciences, Tohoku University, Aramaki-Aoba 6-3, Aoba-ku, Sendai 980-8578, Japan" /><meta name="citation_author" content="McLysaght, Aoife" /><meta name="citation_author_institution" content="Smurfit Institute of Genetics, University of Dublin, Trinity College, Dublin 2, Ireland" /><meta name="citation_author" content="Makino, Takashi" /><meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, Graduate School of Life Sciences, Tohoku University, Aramaki-Aoba 6-3, Aoba-ku, Sendai 980-8578, Japan" /><meta name="citation_title" content="Evolution of Vertebrate Tissues Driven by Differential Modes of Gene Duplication" /><meta name="citation_firstpage" content="305" /><meta name="citation_lastpage" content="316" /><meta name="citation_doi" content="10.1093/dnares/dss012" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="19" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2012/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content=" citation_edition=2nd edition; citation_publisher=Sinauer Associates, Sunderland, MA; Fundamentals of Molecular Evolution;  citation_year=1999; " /><meta name="citation_reference" content="citation_title=Extensive genomic duplication during early chordate evolution; Nat. Genet.;  citation_year=2002;  citation_volume=31;  citation_pages=200-4; " /><meta name="citation_reference" content="citation_title=Reconstruction of the vertebrate ancestral genome reveals dynamic genome reorganization in early vertebrates; Genome Res.;  citation_year=2007;  citation_volume=17;  citation_pages=1254-65; " /><meta name="citation_reference" content="citation_title=Two rounds of whole genome duplication in the ancestral vertebrate; PLoS Biol.;  citation_year=2005;  citation_volume=3;  citation_pages=e314" /><meta name="citation_reference" content="citation_publisher=Springer, Berlin; Evolution by Gene Duplication;  citation_year=1970; " /><meta name="citation_reference" content="citation_title=Robustnessâ€“it's not where you think it is; Nat. Genet.;  citation_year=2000;  citation_volume=25;  citation_pages=3-4; " /><meta name="citation_reference" content="citation_title=Role of duplicate genes in genetic robustness against null mutations; Nature;  citation_year=2003;  citation_volume=421;  citation_pages=63-6; " /><meta name="citation_reference" content="citation_title=Duplicate genes and robustness to transient gene knock-downs in Caenorhabditis elegans; Proc. Biol. Sci.;  citation_year=2004;  citation_volume=271;  citation_pages=89-96; " /><meta name="citation_reference" content="citation_title=Gene essentiality, gene duplicability and protein connectivity in human and mouse; Trends Genet.;  citation_year=2007;  citation_volume=23;  citation_pages=375-8; " /><meta name="citation_reference" content="citation_title=Mouse duplicate genes are as essential as singletons; Trends Genet.;  citation_year=2007;  citation_volume=23;  citation_pages=378-81; " /><meta name="citation_reference" content="citation_title=The complex relationship of gene duplication and essentiality; Trends Genet.;  citation_year=2009;  citation_volume=25;  citation_pages=152-5; " /><meta name="citation_reference" content="citation_title=Exploring the etiology of haploinsufficiency; Bioessays;  citation_year=2002;  citation_volume=24;  citation_pages=175-84; " /><meta name="citation_reference" content="citation_title=Nonlinear effects in macromolecular assembly and dosage sensitivity; J. Theor. Biol.;  citation_year=2003;  citation_volume=220;  citation_pages=19-25; " /><meta name="citation_reference" content="citation_title=Gene dosage balance in cellular pathways: implications for dominance and gene duplicability; Genetics;  citation_year=2004;  citation_volume=168;  citation_pages=569-74; " /><meta name="citation_reference" content="citation_title=Ohnologs in the human genome are dosage balanced and frequently associated with disease; Proc. Natl Acad. Sci. USA;  citation_year=2010;  citation_volume=107;  citation_pages=9270-4; " /><meta name="citation_reference" content="citation_title=Low rates of expression profile divergence in highly expressed genes and tissue-specific genes during mammalian evolution; Mol. Biol. Evol.;  citation_year=2006;  citation_volume=23;  citation_pages=1119-28; " /><meta name="citation_reference" content="citation_title=Tissue-driven hypothesis of genomic evolution and sequence-expression correlations; Proc. Natl Acad. Sci. USA;  citation_year=2007;  citation_volume=104;  citation_pages=2779-84; " /><meta name="citation_reference" content="citation_title=Determinants of substitution rates in mammalian genes: expression pattern affects selection intensity but not mutation rate; Mol. Biol. Evol.;  citation_year=2000;  citation_volume=17;  citation_pages=68-74; " /><meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice; Nucleic Acids Res.;  citation_year=1994;  citation_volume=22;  citation_pages=4673-80; " /><meta name="citation_reference" content="citation_title=Estimating synonymous and nonsynonymous substitution rates under realistic evolutionary models; Mol. Biol. Evol.;  citation_year=2000;  citation_volume=17;  citation_pages=32-43; " /><meta name="citation_reference" content="citation_title=PAML: a program package for phylogenetic analysis by maximum likelihood; Comput. Appl. Biosci.;  citation_year=1997;  citation_volume=13;  citation_pages=555-6; " /><meta name="citation_reference" content="citation_title=Monoallelic expression and tissue specificity are associated with high crossover rates; Trends Genet.;  citation_year=2009;  citation_volume=25;  citation_pages=519-22; " /><meta name="citation_reference" content="citation_title=Functional constraints against variations on molecules from the tissue level: slowly evolving brain-specific genes demonstrated by protein kinase and immunoglobulin supergene families; Mol. Biol. Evol.;  citation_year=1995;  citation_volume=12;  citation_pages=123-30; " /><meta name="citation_reference" content="citation_title=Elevated rates of protein secretion, evolution, and disease among tissue-specific genes; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=54-61; " /><meta name="citation_reference" content="citation_title=Strong evolutionary conservation of broadly expressed protein isoforms in the troponin I gene family and other vertebrate gene families; J. Mol. Evol.;  citation_year=1996;  citation_volume=42;  citation_pages=631-40; " /><meta name="citation_reference" content="citation_title=Divergence of spatial gene expression profiles following species-specific gene duplications in human and mouse; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=1870-9; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/19/4/305/1231060/dss012.pdf" /><meta name="description" content="Abstract.  In this study, we investigated the evolution of vertebrate tissues by examining the potential association among gene expression, duplication, and bas" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/19/4/305/450433" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/19/4/305/450433" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/19/4/305/450433");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dss012';
    </script>
<title>
	Evolution of Vertebrate Tissues Driven by Differential Modes of Gene Duplication | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./450433" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZc+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXRha2UsIE1hc2Fub2J1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNb2xlY3VsYXIgSW1tdW5vbG9neSwgSW5zdGl0dXRlIG9mIERldmVsb3BtZW50LCBBZ2luZyBhbmQgQ2FuY2VyLCBUb2hva3UgVW5pdmVyc2l0eSwgU2VpcnlvLW1hY2hpIDQtMSwgQW9iYS1rdSwgU2VuZGFpIDk4MC04NTc1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikthd2F0YSwgTWFzYWthZG8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEVjb2xvZ3kgYW5kIEV2b2x1dGlvbmFyeSBCaW9sb2d5LCBHcmFkdWF0ZSBTY2hvb2wgb2YgTGlmZSBTY2llbmNlcywgVG9ob2t1IFVuaXZlcnNpdHksIEFyYW1ha2ktQW9iYSA2LTMsIEFvYmEta3UsIFNlbmRhaSA5ODAtODU3OCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNY0x5c2FnaHQsIEFvaWZlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU211cmZpdCBJbnN0aXR1dGUgb2YgR2VuZXRpY3MsIFVuaXZlcnNpdHkgb2YgRHVibGluLCBUcmluaXR5IENvbGxlZ2UsIER1YmxpbiAyLCBJcmVsYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTWFraW5vLCBUYWthc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBFY29sb2d5IGFuZCBFdm9sdXRpb25hcnkgQmlvbG9neSwgR3JhZHVhdGUgU2Nob29sIG9mIExpZmUgU2NpZW5jZXMsIFRvaG9rdSBVbml2ZXJzaXR5LCBBcmFtYWtpLUFvYmEgNi0zLCBBb2JhLWt1LCBTZW5kYWkgOTgwLTg1NzgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJFdm9sdXRpb24gb2YgVmVydGVicmF0ZSBUaXNzdWVzIERyaXZlbiBieSBEaWZmZXJlbnRpYWwgTW9kZXMgb2YgR2VuZSBEdXBsaWNhdGlvbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjMwNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMzE2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNzMDEyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTIvMDgvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSIgY2l0YXRpb25fZWRpdGlvbj0ybmQgZWRpdGlvbjsgY2l0YXRpb25fcHVibGlzaGVyPVNpbmF1ZXIgQXNzb2NpYXRlcywgU3VuZGVybGFuZCwgTUE7IEZ1bmRhbWVudGFscyBvZiBNb2xlY3VsYXIgRXZvbHV0aW9uOyAgY2l0YXRpb25feWVhcj0xOTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHRlbnNpdmUgZ2Vub21pYyBkdXBsaWNhdGlvbiBkdXJpbmcgZWFybHkgY2hvcmRhdGUgZXZvbHV0aW9uOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTIwMC00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWNvbnN0cnVjdGlvbiBvZiB0aGUgdmVydGVicmF0ZSBhbmNlc3RyYWwgZ2Vub21lIHJldmVhbHMgZHluYW1pYyBnZW5vbWUgcmVvcmdhbml6YXRpb24gaW4gZWFybHkgdmVydGVicmF0ZXM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MTI1NC02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHdvIHJvdW5kcyBvZiB3aG9sZSBnZW5vbWUgZHVwbGljYXRpb24gaW4gdGhlIGFuY2VzdHJhbCB2ZXJ0ZWJyYXRlOyBQTG9TIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz1lMzE0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPVNwcmluZ2VyLCBCZXJsaW47IEV2b2x1dGlvbiBieSBHZW5lIER1cGxpY2F0aW9uOyAgY2l0YXRpb25feWVhcj0xOTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Sb2J1c3RuZXNz4oCTaXQncyBub3Qgd2hlcmUgeW91IHRoaW5rIGl0IGlzOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTMtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Um9sZSBvZiBkdXBsaWNhdGUgZ2VuZXMgaW4gZ2VuZXRpYyByb2J1c3RuZXNzIGFnYWluc3QgbnVsbCBtdXRhdGlvbnM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT00MjE7ICBjaXRhdGlvbl9wYWdlcz02My02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EdXBsaWNhdGUgZ2VuZXMgYW5kIHJvYnVzdG5lc3MgdG8gdHJhbnNpZW50IGdlbmUga25vY2stZG93bnMgaW4gQ2Flbm9yaGFiZGl0aXMgZWxlZ2FuczsgUHJvYy4gQmlvbC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yNzE7ICBjaXRhdGlvbl9wYWdlcz04OS05NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBlc3NlbnRpYWxpdHksIGdlbmUgZHVwbGljYWJpbGl0eSBhbmQgcHJvdGVpbiBjb25uZWN0aXZpdHkgaW4gaHVtYW4gYW5kIG1vdXNlOyBUcmVuZHMgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9Mzc1LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vdXNlIGR1cGxpY2F0ZSBnZW5lcyBhcmUgYXMgZXNzZW50aWFsIGFzIHNpbmdsZXRvbnM7IFRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0zNzgtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb21wbGV4IHJlbGF0aW9uc2hpcCBvZiBnZW5lIGR1cGxpY2F0aW9uIGFuZCBlc3NlbnRpYWxpdHk7IFRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xNTItNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwbG9yaW5nIHRoZSBldGlvbG9neSBvZiBoYXBsb2luc3VmZmljaWVuY3k7IEJpb2Vzc2F5czsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTE3NS04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm9ubGluZWFyIGVmZmVjdHMgaW4gbWFjcm9tb2xlY3VsYXIgYXNzZW1ibHkgYW5kIGRvc2FnZSBzZW5zaXRpdml0eTsgSi4gVGhlb3IuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTIyMDsgIGNpdGF0aW9uX3BhZ2VzPTE5LTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIGRvc2FnZSBiYWxhbmNlIGluIGNlbGx1bGFyIHBhdGh3YXlzOiBpbXBsaWNhdGlvbnMgZm9yIGRvbWluYW5jZSBhbmQgZ2VuZSBkdXBsaWNhYmlsaXR5OyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjg7ICBjaXRhdGlvbl9wYWdlcz01NjktNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9obm9sb2dzIGluIHRoZSBodW1hbiBnZW5vbWUgYXJlIGRvc2FnZSBiYWxhbmNlZCBhbmQgZnJlcXVlbnRseSBhc3NvY2lhdGVkIHdpdGggZGlzZWFzZTsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDc7ICBjaXRhdGlvbl9wYWdlcz05MjcwLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvdyByYXRlcyBvZiBleHByZXNzaW9uIHByb2ZpbGUgZGl2ZXJnZW5jZSBpbiBoaWdobHkgZXhwcmVzc2VkIGdlbmVzIGFuZCB0aXNzdWUtc3BlY2lmaWMgZ2VuZXMgZHVyaW5nIG1hbW1hbGlhbiBldm9sdXRpb247IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xMTE5LTI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaXNzdWUtZHJpdmVuIGh5cG90aGVzaXMgb2YgZ2Vub21pYyBldm9sdXRpb24gYW5kIHNlcXVlbmNlLWV4cHJlc3Npb24gY29ycmVsYXRpb25zOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTEwNDsgIGNpdGF0aW9uX3BhZ2VzPTI3NzktODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldGVybWluYW50cyBvZiBzdWJzdGl0dXRpb24gcmF0ZXMgaW4gbWFtbWFsaWFuIGdlbmVzOiBleHByZXNzaW9uIHBhdHRlcm4gYWZmZWN0cyBzZWxlY3Rpb24gaW50ZW5zaXR5IGJ1dCBub3QgbXV0YXRpb24gcmF0ZTsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTY4LTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DTFVTVEFMIFc6IGltcHJvdmluZyB0aGUgc2Vuc2l0aXZpdHkgb2YgcHJvZ3Jlc3NpdmUgbXVsdGlwbGUgc2VxdWVuY2UgYWxpZ25tZW50IHRocm91Z2ggc2VxdWVuY2Ugd2VpZ2h0aW5nLCBwb3NpdGlvbi1zcGVjaWZpYyBnYXAgcGVuYWx0aWVzIGFuZCB3ZWlnaHQgbWF0cml4IGNob2ljZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9NDY3My04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXN0aW1hdGluZyBzeW5vbnltb3VzIGFuZCBub25zeW5vbnltb3VzIHN1YnN0aXR1dGlvbiByYXRlcyB1bmRlciByZWFsaXN0aWMgZXZvbHV0aW9uYXJ5IG1vZGVsczsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTMyLTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QQU1MOiBhIHByb2dyYW0gcGFja2FnZSBmb3IgcGh5bG9nZW5ldGljIGFuYWx5c2lzIGJ5IG1heGltdW0gbGlrZWxpaG9vZDsgQ29tcHV0LiBBcHBsLiBCaW9zY2kuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9NTU1LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbm9hbGxlbGljIGV4cHJlc3Npb24gYW5kIHRpc3N1ZSBzcGVjaWZpY2l0eSBhcmUgYXNzb2NpYXRlZCB3aXRoIGhpZ2ggY3Jvc3NvdmVyIHJhdGVzOyBUcmVuZHMgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9NTE5LTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5jdGlvbmFsIGNvbnN0cmFpbnRzIGFnYWluc3QgdmFyaWF0aW9ucyBvbiBtb2xlY3VsZXMgZnJvbSB0aGUgdGlzc3VlIGxldmVsOiBzbG93bHkgZXZvbHZpbmcgYnJhaW4tc3BlY2lmaWMgZ2VuZXMgZGVtb25zdHJhdGVkIGJ5IHByb3RlaW4ga2luYXNlIGFuZCBpbW11bm9nbG9idWxpbiBzdXBlcmdlbmUgZmFtaWxpZXM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xMjMtMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVsZXZhdGVkIHJhdGVzIG9mIHByb3RlaW4gc2VjcmV0aW9uLCBldm9sdXRpb24sIGFuZCBkaXNlYXNlIGFtb25nIHRpc3N1ZS1zcGVjaWZpYyBnZW5lczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz01NC02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3Ryb25nIGV2b2x1dGlvbmFyeSBjb25zZXJ2YXRpb24gb2YgYnJvYWRseSBleHByZXNzZWQgcHJvdGVpbiBpc29mb3JtcyBpbiB0aGUgdHJvcG9uaW4gSSBnZW5lIGZhbWlseSBhbmQgb3RoZXIgdmVydGVicmF0ZSBnZW5lIGZhbWlsaWVzOyBKLiBNb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9NjMxLTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXZlcmdlbmNlIG9mIHNwYXRpYWwgZ2VuZSBleHByZXNzaW9uIHByb2ZpbGVzIGZvbGxvd2luZyBzcGVjaWVzLXNwZWNpZmljIGdlbmUgZHVwbGljYXRpb25zIGluIGh1bWFuIGFuZCBtb3VzZTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0xODcwLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xOS80LzMwNS8xMjMxMDYwL2RzczAxMi5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgSW4gdGhpcyBzdHVkeSwgd2UgaW52ZXN0aWdhdGVkIHRoZSBldm9sdXRpb24gb2YgdmVydGVicmF0ZSB0aXNzdWVzIGJ5IGV4YW1pbmluZyB0aGUgcG90ZW50aWFsIGFzc29jaWF0aW9uIGFtb25nIGdlbmUgZXhwcmVzc2lvbiwgZHVwbGljYXRpb24sIGFuZCBiYXMiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQ1MDQzMyZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGTKVvtR6GDNkfavwnCagDoNnubGiw==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f4%2f305%2f450433" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f4%2f305%2f450433" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=17634" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/19/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/4/1/m_cover.gif?Expires=2147483647&amp;Signature=eprZ0s37mFZTW8q04XyXQPhslj5EsmNEiVd2xsNgTNSRMFCtDBeg3Jl-WpIZ04VSSHYdf4drJ9cM4eXR~wAjVX4Ka1lGx5jT4Aan8~ek1e8YFczc~efvkNtRr1j4OBLyYeTJMiA~vza484yAcFT5QrseynX-Et0Amq8jonZzJ4uDq5hZrvVMLdx8~LCdEtaG8K8Th5wGC0jLW5Dvd--ISvvgHuZMFgwQgbYRdBL~2OYDNhG3CqoT4aVgmKKF8B3aG~dK1DqjtB8y6Zeh2up0vNZSY74Ozsnuq5BVobk3Bk2WTZDMIWq3CmTTmKd9YkkZCnPAEOdYujN8-TWtB1D4Tw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/19/4">
                        <div class="volume trailing-comma">Volume 19</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6758413">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6758413">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6758417">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6758417">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6758428">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6758428">2. Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6758441">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6758441">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6758489">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6758489">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6758508">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6758508">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6758511">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6758511">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6758513">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6758513">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6758516">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6758516">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/19/4/317/450529">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Evolution of Vertebrate Tissues Driven by Differential Modes of Gene Duplication
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masanobu Satake</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Molecular Immunology</div>, <div class="insititution">Institute of Development, Aging and Cancer, Tohoku University</div>, <div class="addr-line">Seiryo-machi 4-1, Aoba-ku, Sendai 980-8575</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masanobu+Satake">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Satake M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Satake M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masanobu Satake</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masakado Kawata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Ecology and Evolutionary Biology, Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University</div>, <div class="addr-line">Aramaki-Aoba 6-3, Aoba-ku, Sendai 980-8578</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masakado+Kawata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kawata M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kawata M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masakado Kawata</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Aoife McLysaght</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Smurfit Institute of Genetics</div>, <div class="insititution">University of Dublin, Trinity College</div>, <div class="addr-line">Dublin 2</div>, <div class="country">Ireland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Aoife+McLysaght">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=McLysaght A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;McLysaght A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Aoife McLysaght</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takashi Makino</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Ecology and Evolutionary Biology, Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University</div>, <div class="addr-line">Aramaki-Aoba 6-3, Aoba-ku, Sendai 980-8578</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 22-795-6689. Fax. +81 22-795-6689. E-mail: <a href="mailto:tamakino@m.tohoku.ac.jp" target="_blank">tamakino@m.tohoku.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takashi+Makino">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Makino T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Makino T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takashi Makino</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Toshihiko Shiroishi</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 19, Issue 4, 1 August 2012, Pages 305â€“316, <a href='https://doi.org/10.1093/dnares/dss012'>https://doi.org/10.1093/dnares/dss012</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">05 April 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">29 December 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">04 March 2012</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="450433" href="/dnaresearch/article-pdf/19/4/305/1231060/dss012.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Masanobu Satake, Masakado Kawata, Aoife McLysaght, Takashi Makino;  Evolution of Vertebrate Tissues Driven by Differential Modes of Gene Duplication, <em>DNA Research</em>, Volume 19, Issue 4, 1 August 2012, Pages 305â€“316, <a href='https://doi.org/10.1093/dnares/dss012'>https://doi.org/10.1093/dnares/dss012</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/450433?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/450433?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/450433?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/450433?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/450433?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/450433?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/450433?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Evolution%20of%20Vertebrate%20Tissues%20Driven%20by%20Differential%20Modes%20of%20Gene%20Duplication&amp;publicationDate=2012-04-05&amp;volumeNum=19&amp;issueNum=4&amp;author=Satake%2C%20Masanobu%3B%20Kawata%2C%20Masakado&amp;startPage=305&amp;endPage=316&amp;contentId=10.1093%2Fdnares%2Fdss012&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f4%2f305%2f450433" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6758413 id="6758413" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>In this study, we investigated the evolution of vertebrate tissues by examining the potential association among gene expression, duplication, and base substitution patterns. In particular, we compared whole-genome duplication (WGD) with small-scale duplication (SSD), as well as tissue restricted with ubiquitously expressed genes. All patterns were also analysed in the light of gene evolutionary rates. Among those genes characterized by rapid evolution and expressed in a restricted range of tissues, SSD was represented in a larger proportion than WGD. Conversely, genes with ubiquitous expression were associated with slower evolutionary rates and a larger proportion of WGD. The results also show that evolutionary rates were faster in genes expressed in endodermal tissues and slower in ectodermal genes. Accordingly, the proportion of the SSD and WGD genes was highest in the endoderm and ectoderm, respectively. Therefore, quickly evolving SSD genes might have contributed to the faster evolution of endodermal tissues, whereas the comparatively slowly evolving WGD genes might have functioned to maintain the basic characteristics of ectodermal tissues. Mesenchymal tissues occupied an intermediate position in this regard, whereas the patterns observed for haemocytes were unique. Rapid tissue evolution could be related to a specific gene duplication mode (SSD) and faster molecular evolution in response to exposure to the external environment. These findings reveal general patterns underlying the evolution of tissues and their corresponding genes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene duplication&quot;">gene duplication</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="ohnologue">ohnologue</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene expression&quot;">gene expression</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;tissue evolution&quot;">tissue evolution</a></div>                            <h2 scrollto-destination=6758417 id="6758417" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Mutations are the main factors driving genome evolution and may happen within genes through base substitutions or involve their entire duplication.<sup><a reveal-id="DSS012C1" data-open="DSS012C1" class="link link-ref link-reveal xref-bibr">1</a></sup> In the latter case, two mechanisms have been recognized: whole-genome duplication (WGD) and small-scale duplication (SSD), which occurs in relatively a small region of the genome during evolution. For instance, the early vertebrate ancestor is thought to have undergone two rounds of WGD,<sup><a reveal-id="DSS012C2  DSS012C5" data-open="DSS012C2  DSS012C5" class="link link-ref link-reveal xref-bibr">2â€“5</a></sup> as suggested by four vertebrate Hox gene clusters located in different chromosomes. Paralogous genes originated from WGD are referred to as ohnologues.<sup><a reveal-id="DSS012C6" data-open="DSS012C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Singletons, in turn, are genes that did not undergo either WGD or SSD.</p><p>In general, duplicated genes are redundant and their functions overlap. Thus, in simple organisms such as yeasts and nematodes, the proportion of essential genes in duplicated genes is half as low as that in singletons.<sup><a reveal-id="DSS012C7" data-open="DSS012C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSS012C8" data-open="DSS012C8" class="link link-ref link-reveal xref-bibr">8</a></sup> In mice, however, the proportion of essential genes is comparable between duplicated genes and singletons.<sup><a reveal-id="DSS012C9" data-open="DSS012C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSS012C10" data-open="DSS012C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Furthermore, ohnologues are likely to contain a larger proportion of essential genes than SSD genes.<sup><a reveal-id="DSS012C11" data-open="DSS012C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Ohnologues are indeed associated with development, the regulation of transcription, and protein complex formation.<sup><a reveal-id="DSS012C11" data-open="DSS012C11" class="link link-ref link-reveal xref-bibr">11</a></sup> For these genes to function properly, the relative amount of their products must be balanced.<sup><a reveal-id="DSS012C12  DSS012C14" data-open="DSS012C12  DSS012C14" class="link link-ref link-reveal xref-bibr">12â€“14</a></sup> WGD typically involves the simultaneous duplication of all genes, therefore preserving the relative dosages of each gene. Because either the loss or the gain of ohnologues may lead to dosage imbalance, ohnologues are expected to be retained intact in the genome.<sup><a reveal-id="DSS012C15" data-open="DSS012C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Similarly, ancient WGD-derived ohnologues are expected to be more conservative than recently evolved SSD genes.</p><p>Gene expression can be tissue-specific, determining phenotypes such as the morphology and function of tissues, or ubiquitous. Previous research indicates that ubiquitously expressed genes are likely to evolve slowly.<sup><a reveal-id="DSS012C16  DSS012C18" data-open="DSS012C16  DSS012C18" class="link link-ref link-reveal xref-bibr">16â€“18</a></sup> For example, in humans and mice, the orthologous genes that are expressed in a limited number of tissues tend to evolve faster than ubiquitously expressed genes.<sup><a reveal-id="DSS012C18" data-open="DSS012C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Little is known, however, about the extent to which the evolution of tissues is influenced by the differential modes of gene duplication and expression. Indeed, there are no reports exploring the effects of gene duplication events, such as WGD and SSD, on tissue-restricted or ubiquitous gene expression.</p><p>In this study, the potential associations between gene evolutionary rates, duplication (WGD, SSD, and singletons), and gene expression breadth in different tissues (restricted or ubiquitous) were investigated. In addition, these parameters were also analysed in relation to the developmental origin of tissues (endodermal, mesenchymal, or ectodermal). The results support the notion that both base substitutions within genes and gene duplication are associated with gene expression breadth and that the nature of duplication (WGD or SSD) differs substantially depending on the germ-layer origin of the tissue. Tissue evolution is therefore discussed here as the outcome of a process involving the gene evolutionary rate, duplication, and expression.</p>                            <h2 scrollto-destination=6758428 id="6758428" class="section-title" data-legacy-id=s2>2. Materials and Methods</h2>
                            <h3 scrollto-destination=6758429 id="6758429" class="section-title" data-legacy-id=s2a>2.1. Classification of human genes based on the gene duplication mode</h3>
<p>Protein-coding genes of human origin were obtained from Ensembl release 52 (<a class="link link-uri" href="http://www.ensembl.org">http://www.ensembl.org</a>). A total of 7294 ohnologues and 9027 SSD genes were defined as described in Makino and McLysaght.<sup><a reveal-id="DSS012C15" data-open="DSS012C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Briefly, duplicated genes were so judged when the two aligned sequences showed homology in their &gt;30% length with <em>e</em> &lt; 10<sup>âˆ’7</sup> in BLAST search. Ohnologues were syntenic genes located on paralogous chromosomal regions and derived from WGD, whereas SSD genes were duplicated genes not experiencing WGD. Of the 9027 SSD genes, 1478 genes were classified as both ohnologues and SSD genes and, therefore, excluded from the analysis, resulting in 7549 pure SSD genes. An additional 6064 genes were classified as singletons. Thus, 20 907 genes (7294 + 7549 + 6064 = 20 907) were considered.</p><p>To define the origin of SSDs, a sequence similarity search was performed within protein-coding human genes using the all-against-all BLASTP program. Synonymous substitution rates (<em>K</em><sub>S</sub>) were estimated for each close paralogue. There were 2510 and 5039 SSD genes for which <em>K</em><sub>S</sub> were &lt;1 (recent SSD) and â‰¥1 (old SSD), respectively (2510 + 5039 = 7549).</p>                            <h3 scrollto-destination=6758432 id="6758432" class="section-title" data-legacy-id=s2b>2.2. Human orthologous genes and gene evolutionary rates</h3>
<p>Human genes with orthologues in mice and other species were obtained from Ensembl release 52. Orthologous sequences were aligned using CLUSTALW,<sup><a reveal-id="DSS012C19" data-open="DSS012C19" class="link link-ref link-reveal xref-bibr">19</a></sup> and <em>K</em><sub>S</sub> and non-synonymous substitution rates (<em>K</em><sub>A</sub>) were deduced for each orthologous pair using the method of Yang and Nielsen<sup><a reveal-id="DSS012C20" data-open="DSS012C20" class="link link-ref link-reveal xref-bibr">20</a></sup> implemented in PAML.<sup><a reveal-id="DSS012C21" data-open="DSS012C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Next, <em>Ï‰</em> values (<em>K</em><sub>A</sub>/<em>K</em><sub>S</sub>) were calculated.</p>                            <h3 scrollto-destination=6758434 id="6758434" class="section-title" data-legacy-id=s2c>2.3. Human EST data</h3>
<p>An expression sequence tag (EST) database<sup><a reveal-id="DSS012C22" data-open="DSS012C22" class="link link-ref link-reveal xref-bibr">22</a></sup> was used to determine the expression profile of human genes in various tissues. The data were registered at NCBI and included 3 199 559 reads from 47 different human tissues. Each tissue contained more than 10 000 ESTs (68 076 on an average). Using these 47 tissues, the breadth of expression of a gene was represented by the number of tissues in which the gene was identified, as based on the detection of ESTs.<sup><a reveal-id="DSS012C22" data-open="DSS012C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Thus, breadth varied from 1 to 47. Among the 20 907 human genes considered for analysis, there were 3871 for which ESTs were not identified in the EST database. These genes were excluded from further analyses, therefore resulting in a total number of genes of 17 036, among which 6 952 were ohnologues, 5505 were SSDs, and 4579 were singletons.</p>                            <h3 scrollto-destination=6758436 id="6758436" class="section-title" data-legacy-id=s2d>2.4. Classification of human tissues based on the developmental origin</h3>
<p>Forty-three tissues were classified into four subgroups based on their developmental origin, as follows: endoderm (nasopharynx, thymus, stomach, colon, bladder, liver, trachea, lung, pancreas, uterus, cervix, prostate, and intestine), ectoderm (breast, skin, caudate nucleus, hypothalamus, eye, thalamus, subthalamic nucleus, cerebellum, hippocampus, corpus callosum, nervous system, amygdala, and substantia nigra), mesenchymal (synovium, kidney, adipose tissue, bone, adrenal gland, cartilage, muscle, pericardium, and heart), and haemocytes (B cells, bone marrow, germinal centre B cells, blood cells, lymph node, spleen, blood vessels, and T lymphocytes). The developmental origin of four tissues, such as ovary, testis, amnion, and placenta, was not categorized into any of the above four subgroups.</p><p>One caveat of this classification, however, is that the intestine is classified as an endodermal tissue based on the presence of endoderm-derived intestinal epithelium. However, as a macroscopic organ, the intestine includes not only epithelium but also mesoderm-derived tissues such as the submucosa and muscles. Thus, in the above classification, an endodermal tissue usually includes both endoderm- and mesoderm-derived tissues, whereas an ectodermal tissue includes both ectoderm- and mesoderm-derived tissues.</p>                            <h3 scrollto-destination=6758439 id="6758439" class="section-title" data-legacy-id=s2e>2.5. Definition of tissue evolutionary rates</h3>
<p>The tissue evolutionary rate was originally calculated by Kuma <em>et al</em>.<sup><a reveal-id="DSS012C23" data-open="DSS012C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and this method has been adopted in the succeeding studies.<sup><a reveal-id="DSS012C18" data-open="DSS012C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSS012C24" data-open="DSS012C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Thus, we simply employed their definition in the present study. We assumed that a set of genes are expressed in a given tissue type and that the <em>Ï‰</em> values of expressing genes are <em>X</em><sub>1</sub>, <em>X</em><sub>2</sub>, â€¦, <em>X<sub>m</sub></em> (for ohnologues), <em>Y</em><sub>1</sub>, <em>Y</em><sub>2</sub>, â€¦, <em>Y<sub>n</sub></em> (for SSD genes), and <em>Z</em><sub>1</sub>, <em>Z</em><sub>2</sub>, â€¦, <em>Z<sub>o</sub></em> (for singletons). The evolutionary rate of this tissue is then given by: [(<em>X</em><sub>1</sub> + <em>X</em><sub>2</sub> + Â·Â·Â· + <em>X<sub>m</sub></em>) + (<em>Y</em><sub>1</sub> + <em>Y</em><sub>2</sub> + Â·Â·Â· + <em>Y<sub>n</sub></em>) + (<em>Z</em><sub>1</sub> + <em>Z</em><sub>2</sub> + Â·Â·Â· + <em>Z</em><sub>o</sub>)]/(<em>m</em> + <em>n</em> + <em>o</em>), where <em>m</em>, <em>n</em>, and <em>o</em> are the respective number of ohnologues, SSD genes, and singletons.</p>                            <h2 scrollto-destination=6758441 id="6758441" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=6758442 id="6758442" class="section-title" data-legacy-id=s3a>3.1. Association between expression breadth and duplication mode</h3>
<p>FigureÂ <a reveal-id="DSS012F1" data-open="DSS012F1" class="link link-reveal link-table xref-fig">1</a> shows the gene number distribution over various expression breadths (note that the actual number of genes expressed in each tissue at each expression breadth is shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). FigureÂ <a reveal-id="DSS012F1" data-open="DSS012F1" class="link link-reveal link-table xref-fig">1</a>Aâ€“C displays the numbers of ohnologues, SSD genes, and singletons, respectively. The average number of tissues in which each gene type was expressed was 18.7 for ohnologues (blue), 16.6 for SSD genes (red), and 18.0 for singletons (green). FigureÂ <a reveal-id="DSS012F1" data-open="DSS012F1" class="link link-reveal link-table xref-fig">1</a>D shows that the proportions of each gene type in each expression breadth were noticeably different. Specifically, the proportion of SSD genes was relatively higher among breadth-restricted genes (see <em>n</em> = 1, <em>n</em> â‰¤ 3, <em>n</em> â‰¤ 5, and <em>n</em> â‰¤ 10), whereas the proportion of ohnologues was increasingly higher among genes expressed in a larger number of tissues (see <em>n</em> &gt; 10, <em>n</em> &gt; 20, and <em>n</em> &gt; 40). </p>                        <a id="6758444" scrollto-destination="6758444"></a>
<div id="DSS012F1" data-id="DSS012F1" class="fig fig-section"><div class="label fig-label">FigureÂ 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/m_dss01201.jpeg?Expires=2147483647&Signature=4KOkbXgFIDSbofKjE0dXasA3cGr-lWLbpcRTAf-2ji4sNjJ6z5nRS5w1oAc4OiUQSss3bzQaNxYfBq4XbBKC-x75aua504S2X9cHuX0LG19A3a2rBR8c7qqlNorBrHVqPXWyHcbChStJISjsiW-rLO3qXv0kFPSsoGbuUcPmnSMRJBy5hKCDx6dRgNxsSYea~CdUuJkV583NPzWhiGP7WZO7N8LhitwGCi9VQpy7vxmCw1u5XpSTquOPuDMHNfJyfug1ycno8adeJBqsELXjkPAAoNvNqhsbGOx1LXQgLBpeFA53ZqNLzwOLPIeuqplhNpOpbp4IiirBV-VcFJld4A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The number and proportion of ohnologues, SSD genes, and singletons at each tissue expression breadth (as determined by the number of tissues where a corresponding EST was detected for a respective gene). The number of genes at each expression breadth was counted for ohnologues (A, blue), SSD genes (B, red), and singletons (C, green). (D) The percentage of ohnologues, SSD genes, and singletons at each breadth." path-from-xml="dss01201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6758444/dss01201.jpeg" path-from-xml="dss01201.jpeg" target="_blank">View large</a><a section="6758444" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/dss01201.jpeg?Expires=2147483647&Signature=zh26b92lk2nhX7BSSwMxY8s3~IZ09A8wtUr6A8grIWURNidQYxkIzolhMaAM89lq2ayprrHI1aTqAEUN1vVxu272lpgePY3si8PfW3gF13qDLAu1f73AdLCZ~JXDAhjn1OrJKu9rsZCZywts~dP1bMuJ2kSuH6VWy6YRshe6si8gai1s~G2oGyQdCCPowjqXT8SQY40kSMjjggm8WUyUPSC036K-1qcVBh2CzEoASiIGznEufHgHUogfIRFqBH1woURsqX~7Jg3NzlwDg5o1yilVQiTfcOPaGmWcljsn1R9GTs4D2ZjEOgCHkIxi~XDdGk5sPzaH9nH9Zf~4kOAeMw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6758444&ar=450433&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The number and proportion of ohnologues, SSD genes, and singletons at each tissue expression breadth (as determined by the number of tissues where a corresponding EST was detected for a respective gene). The number of genes at each expression breadth was counted for ohnologues (A, blue), SSD genes (B, red), and singletons (C, green). (D) The percentage of ohnologues, SSD genes, and singletons at each breadth.</p></div></div><div content-id="DSS012F1" class="fig fig-modal reveal-modal"><div class="label fig-label">FigureÂ 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/m_dss01201.jpeg?Expires=2147483647&Signature=4KOkbXgFIDSbofKjE0dXasA3cGr-lWLbpcRTAf-2ji4sNjJ6z5nRS5w1oAc4OiUQSss3bzQaNxYfBq4XbBKC-x75aua504S2X9cHuX0LG19A3a2rBR8c7qqlNorBrHVqPXWyHcbChStJISjsiW-rLO3qXv0kFPSsoGbuUcPmnSMRJBy5hKCDx6dRgNxsSYea~CdUuJkV583NPzWhiGP7WZO7N8LhitwGCi9VQpy7vxmCw1u5XpSTquOPuDMHNfJyfug1ycno8adeJBqsELXjkPAAoNvNqhsbGOx1LXQgLBpeFA53ZqNLzwOLPIeuqplhNpOpbp4IiirBV-VcFJld4A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The number and proportion of ohnologues, SSD genes, and singletons at each tissue expression breadth (as determined by the number of tissues where a corresponding EST was detected for a respective gene). The number of genes at each expression breadth was counted for ohnologues (A, blue), SSD genes (B, red), and singletons (C, green). (D) The percentage of ohnologues, SSD genes, and singletons at each breadth." path-from-xml="dss01201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6758444/dss01201.jpeg" path-from-xml="dss01201.jpeg" target="_blank">View large</a><a section="6758444" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/dss01201.jpeg?Expires=2147483647&Signature=zh26b92lk2nhX7BSSwMxY8s3~IZ09A8wtUr6A8grIWURNidQYxkIzolhMaAM89lq2ayprrHI1aTqAEUN1vVxu272lpgePY3si8PfW3gF13qDLAu1f73AdLCZ~JXDAhjn1OrJKu9rsZCZywts~dP1bMuJ2kSuH6VWy6YRshe6si8gai1s~G2oGyQdCCPowjqXT8SQY40kSMjjggm8WUyUPSC036K-1qcVBh2CzEoASiIGznEufHgHUogfIRFqBH1woURsqX~7Jg3NzlwDg5o1yilVQiTfcOPaGmWcljsn1R9GTs4D2ZjEOgCHkIxi~XDdGk5sPzaH9nH9Zf~4kOAeMw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6758444&ar=450433&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The number and proportion of ohnologues, SSD genes, and singletons at each tissue expression breadth (as determined by the number of tissues where a corresponding EST was detected for a respective gene). The number of genes at each expression breadth was counted for ohnologues (A, blue), SSD genes (B, red), and singletons (C, green). (D) The percentage of ohnologues, SSD genes, and singletons at each breadth.</p></div></div>                            <h3 scrollto-destination=6758445 id="6758445" class="section-title" data-legacy-id=s3b>3.2. Association between gene evolutionary rates and duplication mode</h3>
<p>Ohnologues are believed to be more conservative than SSD genes with respect to functional essentiality and dosage-balance requirement.<sup><a reveal-id="DSS012C11" data-open="DSS012C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSS012C15" data-open="DSS012C15" class="link link-ref link-reveal xref-bibr">15</a></sup> To determine whether ohnologues are also conservative in terms of molecular evolution, non-synonymous nucleotide divergence in the coding region (between humans and mice) were examined. TableÂ <a reveal-id="DSS012TB1" data-open="DSS012TB1" class="link link-reveal link-table xref-fig">1</a> shows that the average <em>Ï‰</em> value of ohnologues (0.11) was 0.55â€“0.57-fold lower than that of SSD genes (0.19; <em>P</em> &lt; 2.2 Ã— 10<sup>âˆ’16</sup>, the Mannâ€“Whitney <em>U</em>-test) and of singletons (0.20; <em>P</em> &lt; 2.2 Ã— 10<sup>âˆ’16</sup>). The result confirms the conservative nature of ohnologues in the evolution of coding regions. </p>                        <a id="6758447" scrollto-destination="6758447"></a>
<div content-id="DSS012TB1" class="table-modal"><div class="table-wrap"><div id="DSS012TB1" class="table-wrap-title" data-id="DSS012TB1"><div class="label -label">TableÂ 1.</div><div class="caption"><p>Average <em>K</em><sub>A</sub>/<em>K</em><sub>s</sub> values</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>CategoryÂ </th><th>SubcategoryÂ </th><th>Number of compared orthologous gene pairsÂ </th><th>Average <em>K</em><sub>A</sub>/<em>K</em><sub>s</sub>Â </th><th>SDÂ </th></tr></thead><tbody><tr><td rowspan="5">Expressed genesÂ </td><td>OhnologuesÂ </td><td>6952Â </td><td>0.11Â </td><td>0.11Â </td></tr><tr><td>Total SSD genesÂ </td><td>5505Â </td><td>0.19Â </td><td>0.16Â </td></tr><tr><td>â€ƒOld SSD genes (<em>K</em><sub>s</sub> â‰¥ 1)Â </td><td>4356Â </td><td>0.17Â </td><td>0.14Â </td></tr><tr><td>â€ƒRecent SSD genes (<em>K</em><sub>s</sub> &lt; 1)Â </td><td>1149Â </td><td>0.29Â </td><td>0.20Â </td></tr><tr><td>SingletonsÂ </td><td>4579Â </td><td>0.20Â </td><td>0.22Â </td></tr><tr><td rowspan="5">Narrowly expressed genes (<em>n</em> â‰¤ 10)Â </td><td>OhnologuesÂ </td><td>2151Â </td><td>0.13Â </td><td>0.12Â </td></tr><tr><td>Total SSD genesÂ </td><td>2008Â </td><td>0.26Â </td><td>0.18Â </td></tr><tr><td>â€ƒOld SSD genes (<em>K</em><sub>s</sub> â‰¥ 1)Â </td><td>1338Â </td><td>0.22Â </td><td>0.16Â </td></tr><tr><td>â€ƒRecent SSD genes (<em>K</em><sub>s</sub> &lt; 1)Â </td><td>670Â </td><td>0.34Â </td><td>0.20Â </td></tr><tr><td>SingletonsÂ </td><td>1101Â </td><td>0.29Â </td><td>0.19Â </td></tr><tr><td rowspan="5">Broadly expressed genes (<em>n</em> &gt; 10)Â </td><td>OhnologuesÂ </td><td>4801Â </td><td>0.10Â </td><td>0.11Â </td></tr><tr><td>Total SSD genesÂ </td><td>3497Â </td><td>0.15Â </td><td>0.14Â </td></tr><tr><td>â€ƒOld SSD genes (<em>K</em><sub>s</sub> â‰¥ 1)Â </td><td>3018Â </td><td>0.14Â </td><td>0.12Â </td></tr><tr><td>â€ƒRecent SSD genes (<em>K</em><sub>s</sub> &lt; 1)Â </td><td>479Â </td><td>0.23Â </td><td>0.19Â </td></tr><tr><td>SingletonsÂ </td><td>3478Â </td><td>0.17Â </td><td>0.23Â </td></tr></tbody></table></div><div content-id="DSS012TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>CategoryÂ </th><th>SubcategoryÂ </th><th>Number of compared orthologous gene pairsÂ </th><th>Average <em>K</em><sub>A</sub>/<em>K</em><sub>s</sub>Â </th><th>SDÂ </th></tr></thead><tbody><tr><td rowspan="5">Expressed genesÂ </td><td>OhnologuesÂ </td><td>6952Â </td><td>0.11Â </td><td>0.11Â </td></tr><tr><td>Total SSD genesÂ </td><td>5505Â </td><td>0.19Â </td><td>0.16Â </td></tr><tr><td>â€ƒOld SSD genes (<em>K</em><sub>s</sub> â‰¥ 1)Â </td><td>4356Â </td><td>0.17Â </td><td>0.14Â </td></tr><tr><td>â€ƒRecent SSD genes (<em>K</em><sub>s</sub> &lt; 1)Â </td><td>1149Â </td><td>0.29Â </td><td>0.20Â </td></tr><tr><td>SingletonsÂ </td><td>4579Â </td><td>0.20Â </td><td>0.22Â </td></tr><tr><td rowspan="5">Narrowly expressed genes (<em>n</em> â‰¤ 10)Â </td><td>OhnologuesÂ </td><td>2151Â </td><td>0.13Â </td><td>0.12Â </td></tr><tr><td>Total SSD genesÂ </td><td>2008Â </td><td>0.26Â </td><td>0.18Â </td></tr><tr><td>â€ƒOld SSD genes (<em>K</em><sub>s</sub> â‰¥ 1)Â </td><td>1338Â </td><td>0.22Â </td><td>0.16Â </td></tr><tr><td>â€ƒRecent SSD genes (<em>K</em><sub>s</sub> &lt; 1)Â </td><td>670Â </td><td>0.34Â </td><td>0.20Â </td></tr><tr><td>SingletonsÂ </td><td>1101Â </td><td>0.29Â </td><td>0.19Â </td></tr><tr><td rowspan="5">Broadly expressed genes (<em>n</em> &gt; 10)Â </td><td>OhnologuesÂ </td><td>4801Â </td><td>0.10Â </td><td>0.11Â </td></tr><tr><td>Total SSD genesÂ </td><td>3497Â </td><td>0.15Â </td><td>0.14Â </td></tr><tr><td>â€ƒOld SSD genes (<em>K</em><sub>s</sub> â‰¥ 1)Â </td><td>3018Â </td><td>0.14Â </td><td>0.12Â </td></tr><tr><td>â€ƒRecent SSD genes (<em>K</em><sub>s</sub> &lt; 1)Â </td><td>479Â </td><td>0.23Â </td><td>0.19Â </td></tr><tr><td>SingletonsÂ </td><td>3478Â </td><td>0.17Â </td><td>0.23Â </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6758447" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS012TB1" class="table-wrap-title" data-id="DSS012TB1"><div class="label -label">TableÂ 1.</div><div class="caption"><p>Average <em>K</em><sub>A</sub>/<em>K</em><sub>s</sub> values</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>CategoryÂ </th><th>SubcategoryÂ </th><th>Number of compared orthologous gene pairsÂ </th><th>Average <em>K</em><sub>A</sub>/<em>K</em><sub>s</sub>Â </th><th>SDÂ </th></tr></thead><tbody><tr><td rowspan="5">Expressed genesÂ </td><td>OhnologuesÂ </td><td>6952Â </td><td>0.11Â </td><td>0.11Â </td></tr><tr><td>Total SSD genesÂ </td><td>5505Â </td><td>0.19Â </td><td>0.16Â </td></tr><tr><td>â€ƒOld SSD genes (<em>K</em><sub>s</sub> â‰¥ 1)Â </td><td>4356Â </td><td>0.17Â </td><td>0.14Â </td></tr><tr><td>â€ƒRecent SSD genes (<em>K</em><sub>s</sub> &lt; 1)Â </td><td>1149Â </td><td>0.29Â </td><td>0.20Â </td></tr><tr><td>SingletonsÂ </td><td>4579Â </td><td>0.20Â </td><td>0.22Â </td></tr><tr><td rowspan="5">Narrowly expressed genes (<em>n</em> â‰¤ 10)Â </td><td>OhnologuesÂ </td><td>2151Â </td><td>0.13Â </td><td>0.12Â </td></tr><tr><td>Total SSD genesÂ </td><td>2008Â </td><td>0.26Â </td><td>0.18Â </td></tr><tr><td>â€ƒOld SSD genes (<em>K</em><sub>s</sub> â‰¥ 1)Â </td><td>1338Â </td><td>0.22Â </td><td>0.16Â </td></tr><tr><td>â€ƒRecent SSD genes (<em>K</em><sub>s</sub> &lt; 1)Â </td><td>670Â </td><td>0.34Â </td><td>0.20Â </td></tr><tr><td>SingletonsÂ </td><td>1101Â </td><td>0.29Â </td><td>0.19Â </td></tr><tr><td rowspan="5">Broadly expressed genes (<em>n</em> &gt; 10)Â </td><td>OhnologuesÂ </td><td>4801Â </td><td>0.10Â </td><td>0.11Â </td></tr><tr><td>Total SSD genesÂ </td><td>3497Â </td><td>0.15Â </td><td>0.14Â </td></tr><tr><td>â€ƒOld SSD genes (<em>K</em><sub>s</sub> â‰¥ 1)Â </td><td>3018Â </td><td>0.14Â </td><td>0.12Â </td></tr><tr><td>â€ƒRecent SSD genes (<em>K</em><sub>s</sub> &lt; 1)Â </td><td>479Â </td><td>0.23Â </td><td>0.19Â </td></tr><tr><td>SingletonsÂ </td><td>3478Â </td><td>0.17Â </td><td>0.23Â </td></tr></tbody></table></div><div content-id="DSS012TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>CategoryÂ </th><th>SubcategoryÂ </th><th>Number of compared orthologous gene pairsÂ </th><th>Average <em>K</em><sub>A</sub>/<em>K</em><sub>s</sub>Â </th><th>SDÂ </th></tr></thead><tbody><tr><td rowspan="5">Expressed genesÂ </td><td>OhnologuesÂ </td><td>6952Â </td><td>0.11Â </td><td>0.11Â </td></tr><tr><td>Total SSD genesÂ </td><td>5505Â </td><td>0.19Â </td><td>0.16Â </td></tr><tr><td>â€ƒOld SSD genes (<em>K</em><sub>s</sub> â‰¥ 1)Â </td><td>4356Â </td><td>0.17Â </td><td>0.14Â </td></tr><tr><td>â€ƒRecent SSD genes (<em>K</em><sub>s</sub> &lt; 1)Â </td><td>1149Â </td><td>0.29Â </td><td>0.20Â </td></tr><tr><td>SingletonsÂ </td><td>4579Â </td><td>0.20Â </td><td>0.22Â </td></tr><tr><td rowspan="5">Narrowly expressed genes (<em>n</em> â‰¤ 10)Â </td><td>OhnologuesÂ </td><td>2151Â </td><td>0.13Â </td><td>0.12Â </td></tr><tr><td>Total SSD genesÂ </td><td>2008Â </td><td>0.26Â </td><td>0.18Â </td></tr><tr><td>â€ƒOld SSD genes (<em>K</em><sub>s</sub> â‰¥ 1)Â </td><td>1338Â </td><td>0.22Â </td><td>0.16Â </td></tr><tr><td>â€ƒRecent SSD genes (<em>K</em><sub>s</sub> &lt; 1)Â </td><td>670Â </td><td>0.34Â </td><td>0.20Â </td></tr><tr><td>SingletonsÂ </td><td>1101Â </td><td>0.29Â </td><td>0.19Â </td></tr><tr><td rowspan="5">Broadly expressed genes (<em>n</em> &gt; 10)Â </td><td>OhnologuesÂ </td><td>4801Â </td><td>0.10Â </td><td>0.11Â </td></tr><tr><td>Total SSD genesÂ </td><td>3497Â </td><td>0.15Â </td><td>0.14Â </td></tr><tr><td>â€ƒOld SSD genes (<em>K</em><sub>s</sub> â‰¥ 1)Â </td><td>3018Â </td><td>0.14Â </td><td>0.12Â </td></tr><tr><td>â€ƒRecent SSD genes (<em>K</em><sub>s</sub> &lt; 1)Â </td><td>479Â </td><td>0.23Â </td><td>0.19Â </td></tr><tr><td>SingletonsÂ </td><td>3478Â </td><td>0.17Â </td><td>0.23Â </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6758447" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6758448 id="6758448" class="section-title" data-legacy-id=s3c>3.3. Tissue evolutionary rates in restricted expression breadths</h3>
<p>In Fig.Â <a reveal-id="DSS012F2" data-open="DSS012F2" class="link link-reveal link-table xref-fig">2</a>A, tissue evolutionary rates are plotted in an increasing order, for each of 47 tissues, and under a condition of <em>n</em> â‰¤ 10 (see â€˜Proportions of ohnologues and SSD genes in various expression breadthsâ€™ as for a rationale of the use of <em>n</em> â‰¤ 10). The <em>Ï‰</em> values gradually increased from 0.13 in a slowly evolving, leftward tissue to 0.29 in a fast evolving, rightward tissue. </p>                        <a id="6758450" scrollto-destination="6758450"></a>
<div id="DSS012F2" data-id="DSS012F2" class="fig fig-section"><div class="label fig-label">FigureÂ 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/m_dss01202.jpeg?Expires=2147483647&Signature=N2t2k5GSRCFvB8jI5s2Xe5~KDygz4nwHEv7~gWe2sKyF2nolZTGziNfRn~eit2XhmXbhHheGtbnEDb7a6UESV~Wn0ugKwGkTvV2heTvzuSVLk2tDxBsCqkDjMsWovhQBecIfGf~54WksPpmBAhSOW~mrl16FP68N6zbZ0wHVyyFyAYEfqD9hYJoT-n529R1pihYg9kE3I0BoshtqYlfGt7bMaIZSTa7ha4pa3NAg9vDVmP2b7~7GBFH1saWOqvDVgOvKaFmnU6ie~PFDYLl3ZnsbJQn4GeAQVmn5eh9xUGDOClsIEXhXxs-JJGN2s2UmYS3FDjmx9R1Kta7OQlaNAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Contribution of ohnologues, SSD genes, and singletons among breadth-restricted (n â‰¤ 10 tissues) and breadth-ubiquitous (n &gt; 10 tissues) genes. Blue, red, and green represent the ohnologues, SSD genes, and singletons, respectively. Tissues are aligned on the x-axis in the order of magnitude of the KA/KS (Ï‰) values of their expressed genes. Parameters used in the y-axis are as follows: (A and E) the average Ï‰ values of all genes expressed in a given tissue, which corresponds to the defined tissue evolutionary rate; (BÂ and F) the Ï‰ values of ohnologues, SSD genes, and singletons; (C and G) the proportion of ohnologues, SSD genes, and singletons; and (DÂ and H) the Ï‰ values and the proportions calculated for ohnologues, SSD genes, and singletons. The order of tissues aligned are as follows from the left to the right. In (A)â€“(D), the nervous system, subthalamic nucleus, amygdale, cerebellum, cartilage, substantia nigra, hypothalamus, hippocampus, pericardium, corpus callosum, thalamus, T lymphocytes, ovary, eye, heart, caudate nucleus, adipose tissue, pancreas, skin, muscle, adrenal gland, prostate, breast, lymph, kidney, colon, amnion, cervix, placenta, stomach, bladder, lung, uterus, bone, germinal centre B cell, intestine, trachea, bone marrow, B cells, testis, liver, spleen, thymus, blood vessels, synovium, blood, and nasopharynx. In (E)â€“(H), the amnion, cartilage, B cells, skin, muscle, nervous system, hypothalamus, cervix, adipose tissue, substantia nigra, caudate nucleus, corpus callosum, T lymphocytes, subthalamic nucleus, heart, bone marrow, adrenal gland, lymph, blood vessels, bone, bladder, amygdale, ovary, hippocampus, pancreas, breast, colon, blood, eye, pericardium, cerebellum, thalamus, prostate, stomach, germinal centre B cell, kidney, liver, spleen, lung, uterus, testis, placenta, intestine, thymus, synovium, trachea, and nasopharynx." path-from-xml="dss01202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6758450/dss01202.jpeg" path-from-xml="dss01202.jpeg" target="_blank">View large</a><a section="6758450" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/dss01202.jpeg?Expires=2147483647&Signature=UR9EMXXoK9ABlZ6ZNLSMLjDcsjbvu4EIoJvHJaTnCk7zNmJzwK5Wtj0Xpn6EvuWQ3lwfojdhSFmzpXQBOqW8cjq5V5HZOGAtuaswXb-mOMysbSQuZjKzaW5xyD7PyDzqYXAQOMr-cqsQMjetC92~Vh1ZgdU26Y4YApSNep7-HPArP7~CsnF0jNG07B-lIpWn6kWpANXmPoD4dxNJJsJwwKcRwhvKOSNnFs6Rlxc~IM-w3mHbg48JtFG~RQjSOW1Gwdq6xIiaHxL0JuJZ3LZFPBSY69pnyfbt87I4mFO1i33KLKRdGg2gZQYho8d1MdR4q1CVIn5OILz61ZCr2yAbCQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6758450&ar=450433&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Contribution of ohnologues, SSD genes, and singletons among breadth-restricted (<em>n</em> â‰¤ 10 tissues) and breadth-ubiquitous (<em>n</em> &gt; 10 tissues) genes. Blue, red, and green represent the ohnologues, SSD genes, and singletons, respectively. Tissues are aligned on the <em>x</em>-axis in the order of magnitude of the <em>K</em><sub>A</sub><em>/K</em><sub>S</sub> (<em>Ï‰</em>) values of their expressed genes. Parameters used in the <em>y</em>-axis are as follows: (A and E) the average <em>Ï‰</em> values of all genes expressed in a given tissue, which corresponds to the defined tissue evolutionary rate; (BÂ and F) the <em>Ï‰</em> values of ohnologues, SSD genes, and singletons; (C and G) the proportion of ohnologues, SSD genes, and singletons; and (DÂ and H) the <em>Ï‰</em> values and the proportions calculated for ohnologues, SSD genes, and singletons. The order of tissues aligned are as follows from the left to the right. In (A)â€“(D), the nervous system, subthalamic nucleus, amygdale, cerebellum, cartilage, substantia nigra, hypothalamus, hippocampus, pericardium, corpus callosum, thalamus, T lymphocytes, ovary, eye, heart, caudate nucleus, adipose tissue, pancreas, skin, muscle, adrenal gland, prostate, breast, lymph, kidney, colon, amnion, cervix, placenta, stomach, bladder, lung, uterus, bone, germinal centre B cell, intestine, trachea, bone marrow, B cells, testis, liver, spleen, thymus, blood vessels, synovium, blood, and nasopharynx. In (E)â€“(H), the amnion, cartilage, B cells, skin, muscle, nervous system, hypothalamus, cervix, adipose tissue, substantia nigra, caudate nucleus, corpus callosum, T lymphocytes, subthalamic nucleus, heart, bone marrow, adrenal gland, lymph, blood vessels, bone, bladder, amygdale, ovary, hippocampus, pancreas, breast, colon, blood, eye, pericardium, cerebellum, thalamus, prostate, stomach, germinal centre B cell, kidney, liver, spleen, lung, uterus, testis, placenta, intestine, thymus, synovium, trachea, and nasopharynx.</p></div></div><div content-id="DSS012F2" class="fig fig-modal reveal-modal"><div class="label fig-label">FigureÂ 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/m_dss01202.jpeg?Expires=2147483647&Signature=N2t2k5GSRCFvB8jI5s2Xe5~KDygz4nwHEv7~gWe2sKyF2nolZTGziNfRn~eit2XhmXbhHheGtbnEDb7a6UESV~Wn0ugKwGkTvV2heTvzuSVLk2tDxBsCqkDjMsWovhQBecIfGf~54WksPpmBAhSOW~mrl16FP68N6zbZ0wHVyyFyAYEfqD9hYJoT-n529R1pihYg9kE3I0BoshtqYlfGt7bMaIZSTa7ha4pa3NAg9vDVmP2b7~7GBFH1saWOqvDVgOvKaFmnU6ie~PFDYLl3ZnsbJQn4GeAQVmn5eh9xUGDOClsIEXhXxs-JJGN2s2UmYS3FDjmx9R1Kta7OQlaNAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Contribution of ohnologues, SSD genes, and singletons among breadth-restricted (n â‰¤ 10 tissues) and breadth-ubiquitous (n &gt; 10 tissues) genes. Blue, red, and green represent the ohnologues, SSD genes, and singletons, respectively. Tissues are aligned on the x-axis in the order of magnitude of the KA/KS (Ï‰) values of their expressed genes. Parameters used in the y-axis are as follows: (A and E) the average Ï‰ values of all genes expressed in a given tissue, which corresponds to the defined tissue evolutionary rate; (BÂ and F) the Ï‰ values of ohnologues, SSD genes, and singletons; (C and G) the proportion of ohnologues, SSD genes, and singletons; and (DÂ and H) the Ï‰ values and the proportions calculated for ohnologues, SSD genes, and singletons. The order of tissues aligned are as follows from the left to the right. In (A)â€“(D), the nervous system, subthalamic nucleus, amygdale, cerebellum, cartilage, substantia nigra, hypothalamus, hippocampus, pericardium, corpus callosum, thalamus, T lymphocytes, ovary, eye, heart, caudate nucleus, adipose tissue, pancreas, skin, muscle, adrenal gland, prostate, breast, lymph, kidney, colon, amnion, cervix, placenta, stomach, bladder, lung, uterus, bone, germinal centre B cell, intestine, trachea, bone marrow, B cells, testis, liver, spleen, thymus, blood vessels, synovium, blood, and nasopharynx. In (E)â€“(H), the amnion, cartilage, B cells, skin, muscle, nervous system, hypothalamus, cervix, adipose tissue, substantia nigra, caudate nucleus, corpus callosum, T lymphocytes, subthalamic nucleus, heart, bone marrow, adrenal gland, lymph, blood vessels, bone, bladder, amygdale, ovary, hippocampus, pancreas, breast, colon, blood, eye, pericardium, cerebellum, thalamus, prostate, stomach, germinal centre B cell, kidney, liver, spleen, lung, uterus, testis, placenta, intestine, thymus, synovium, trachea, and nasopharynx." path-from-xml="dss01202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6758450/dss01202.jpeg" path-from-xml="dss01202.jpeg" target="_blank">View large</a><a section="6758450" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/dss01202.jpeg?Expires=2147483647&Signature=UR9EMXXoK9ABlZ6ZNLSMLjDcsjbvu4EIoJvHJaTnCk7zNmJzwK5Wtj0Xpn6EvuWQ3lwfojdhSFmzpXQBOqW8cjq5V5HZOGAtuaswXb-mOMysbSQuZjKzaW5xyD7PyDzqYXAQOMr-cqsQMjetC92~Vh1ZgdU26Y4YApSNep7-HPArP7~CsnF0jNG07B-lIpWn6kWpANXmPoD4dxNJJsJwwKcRwhvKOSNnFs6Rlxc~IM-w3mHbg48JtFG~RQjSOW1Gwdq6xIiaHxL0JuJZ3LZFPBSY69pnyfbt87I4mFO1i33KLKRdGg2gZQYho8d1MdR4q1CVIn5OILz61ZCr2yAbCQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6758450&ar=450433&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Contribution of ohnologues, SSD genes, and singletons among breadth-restricted (<em>n</em> â‰¤ 10 tissues) and breadth-ubiquitous (<em>n</em> &gt; 10 tissues) genes. Blue, red, and green represent the ohnologues, SSD genes, and singletons, respectively. Tissues are aligned on the <em>x</em>-axis in the order of magnitude of the <em>K</em><sub>A</sub><em>/K</em><sub>S</sub> (<em>Ï‰</em>) values of their expressed genes. Parameters used in the <em>y</em>-axis are as follows: (A and E) the average <em>Ï‰</em> values of all genes expressed in a given tissue, which corresponds to the defined tissue evolutionary rate; (BÂ and F) the <em>Ï‰</em> values of ohnologues, SSD genes, and singletons; (C and G) the proportion of ohnologues, SSD genes, and singletons; and (DÂ and H) the <em>Ï‰</em> values and the proportions calculated for ohnologues, SSD genes, and singletons. The order of tissues aligned are as follows from the left to the right. In (A)â€“(D), the nervous system, subthalamic nucleus, amygdale, cerebellum, cartilage, substantia nigra, hypothalamus, hippocampus, pericardium, corpus callosum, thalamus, T lymphocytes, ovary, eye, heart, caudate nucleus, adipose tissue, pancreas, skin, muscle, adrenal gland, prostate, breast, lymph, kidney, colon, amnion, cervix, placenta, stomach, bladder, lung, uterus, bone, germinal centre B cell, intestine, trachea, bone marrow, B cells, testis, liver, spleen, thymus, blood vessels, synovium, blood, and nasopharynx. In (E)â€“(H), the amnion, cartilage, B cells, skin, muscle, nervous system, hypothalamus, cervix, adipose tissue, substantia nigra, caudate nucleus, corpus callosum, T lymphocytes, subthalamic nucleus, heart, bone marrow, adrenal gland, lymph, blood vessels, bone, bladder, amygdale, ovary, hippocampus, pancreas, breast, colon, blood, eye, pericardium, cerebellum, thalamus, prostate, stomach, germinal centre B cell, kidney, liver, spleen, lung, uterus, testis, placenta, intestine, thymus, synovium, trachea, and nasopharynx.</p></div></div>                            <h3 scrollto-destination=6758451 id="6758451" class="section-title" data-legacy-id=s3d>3.4. Gene evolutionary rate in restricted expression breadths</h3>
<p>The average <em>Ï‰</em> values were calculated separately for each type of gene as follows: (<em>X</em><sub>1</sub> + <em>X</em><sub>2</sub> + Â·Â·Â· + <em>X<sub>m</sub></em>)/<em>m</em> for ohnologues, (<em>Y</em><sub>1</sub> + <em>Y</em><sub>2</sub> + Â·Â·Â· + <em>Y<sub>n</sub></em>)/<em>n</em> for SSD genes, and (<em>Z</em><sub>1</sub> + <em>Z</em><sub>2</sub> + Â·Â·Â· + <em>Z<sub>o</sub></em>)/<em>o</em> for singletons. FigureÂ <a reveal-id="DSS012F2" data-open="DSS012F2" class="link link-reveal link-table xref-fig">2</a>B shows that the evolutionary rate (<em>Ï‰</em>) of ohnologues (blue), SSD genes (red), and singletons (green) increased gradually in line with increases in tissue evolutionary rates, indicating that all gene types evolved in parallel with the tissue evolution. Therefore, gene evolutionary rates were higher in the genes that are expressed in fast evolving tissues irrespective of the gene type, whereas those were lower in slowly evolving tissues regardless of the gene type. It is worth noticing, however, that the average <em>Ï‰</em> value of ohnologues was low (0.13), whereas that of SSD genes and singletons was high (0.26 and 0.29, respectively; Fig.Â <a reveal-id="DSS012F2" data-open="DSS012F2" class="link link-reveal link-table xref-fig">2</a>B and see <em>n</em> â‰¤ 10 in TableÂ <a reveal-id="DSS012TB1" data-open="DSS012TB1" class="link link-reveal link-table xref-fig">1</a>).</p>                            <h3 scrollto-destination=6758453 id="6758453" class="section-title" data-legacy-id=s3e>3.5. Proportions of gene type in restricted expression breadths</h3>
<p>FigureÂ <a reveal-id="DSS012F2" data-open="DSS012F2" class="link link-reveal link-table xref-fig">2</a>C shows that an increase in tissue evolutionary rates was accompanied by a corresponding decrease in the proportion of ohnologues and an increase in the proportion of SSD genes. This result indicates that ohnologues and SSD genes tended to be expressed in slowly and fast evolving tissues, respectively. The proportion of singletons also appeared to increase in parallel with tissue evolutionary rates.</p><p>FigureÂ <a reveal-id="DSS012F2" data-open="DSS012F2" class="link link-reveal link-table xref-fig">2</a>B and C also indicates that ohnologues and SSD genes behave differently. In the case of SSD genes, <em>Ï‰</em> values as well as proportions are positively associated with tissue evolutionary rates, suggesting that SSD genes contribute to faster tissue evolution. In the case of ohnologues, however, an increase in <em>Ï‰</em> was associated with a decrease in proportion, accompanying the tissue evolutionary rate. Therefore, further analysis is needed to examine the relative contribution of ohnologues to tissue evolution.</p>                            <h3 scrollto-destination=6758456 id="6758456" class="section-title" data-legacy-id=s3f>3.6. Differential contribution of ohnologues, SSD genes, and singletons among genes with restricted expression</h3>
<div class="block-child-p">Both the evolutionary rate of each gene type and their proportion are incorporated in the following formula:<a id="" class="jumplink-placeholder">Â </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/dss012ueq1.gif?Expires=2147483647&Signature=P-KTAqNYdQkxg3Cwce0nYF7BOOQNX6i~456VeTx-2f3P3TsFyqelQPjR2lqwSIIPgAmwmiSWw2il6rQz3XftTUp4TyYyc8Za2~nKqqkTqIOHvbFilkGOOrwVcPMD6iazM0BmInoeP0~csULH2Sbv4tEqmp9zXhsI~4Sk9Z-di5x0LzsaDxtQOzpoAowzndvTYPWO8v3bM2W5RS8-p-0A6cajXQnfgxyyCSpzBQp3BqUvPfM6-BZUJxlTJGPWr-3a4T3pPHahfA~TobLAr5ZQYybX4nMWWaFwMuIi26wNUlo34Kuv20nR2cvcnoXGxL2CI7qGlEJR4W~8g9vVG7fBRA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dss012ueq1.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/m_dss012ueq1.gif?Expires=2147483647&Signature=1ftjFsjlT4MRdhQNiS3OFmmcPyrn12IIQ1jjFR6RALARD-OCUSewGChDvCINI6oQoKvFtwbpnZbFVO1AQxukSx8w7pww7ZAmHLkAv1lUZgIHh2GJN5lnHd8IOPeMOXZVQbtkrs4Yo4dOEVsJppKySzpuzUUW5BMOfmjl5dcrTLfCSjOsZMkl~B82glfteTt5RvkjjJetc4Pl~eM3xDfmFdyPMiPvxu5GKPwNzrKzqiiF4dk4jbAX4nVtX2jzxmqlG7oJTorJcir6zCmwWhLgCfMacYDDH012gwcza5VjVsM5TAd2nBQ0wECyBl15Ju-PnO493XM3GrWppl6CbxCWrA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dss012ueq1.gif" /></a></div></div></div> where [(<em>X</em><sub>1</sub> + <em>X</em><sub>2</sub> + Â·Â·Â· + <em>X<sub>m</sub></em>)/<em>m</em>] Ã— [<em>m</em>/(<em>m</em> + <em>n</em> + <em>o</em>)] represents ohnologues, [(<em>Y</em><sub>1</sub> + <em>Y</em><sub>2</sub> + Â·Â·Â· + <em>Y<sub>n</sub></em>)/<em>n</em>] Ã— [<em>n</em>/(<em>m</em> + <em>n</em> + <em>o</em>)] represents SSD genes, and [(<em>Z</em><sub>1</sub> + <em>Z</em><sub>2</sub> + Â·Â·Â· + <em>Z<sub>o</sub></em>)/<em>o</em>] Ã— [<em>o</em>/(<em>m</em> + <em>n</em> + <em>o</em>)} represents singletons. FigureÂ <a reveal-id="DSS012F2" data-open="DSS012F2" class="link link-reveal link-table xref-fig">2</a>D shows each ohnologue, SSD, and singleton component in relation to tissue evolutionary rates. For SSD genes and singletons, there was a positive association between tissue and gene evolutionary rates. In contrast, ohnologues were almost flat irrespective of tissue evolution. This flat line for ohnologues suggests that ohnologues did not play a major role in tissue evolution, in agreement with the notion that ohnologues are conservative in nature. However, this does not necessarily mean a lack of any role for ohnologues in tissue evolution. In fact, the proportions of ohnologues were substantially reduced in fast evolving tissues. This effect and the increase in the <em>Ï‰</em> values cancelled out each other.</div>                            <h3 scrollto-destination=6758458 id="6758458" class="section-title" data-legacy-id=s3g>3.7. Equal contribution of ohnologues, SSD genes, and singletons among genes with ubiquitous expression</h3>
<p>FigureÂ <a reveal-id="DSS012F2" data-open="DSS012F2" class="link link-reveal link-table xref-fig">2</a>Eâ€“H represents the results obtained under a condition of <em>n</em> &gt; 10 tissues and show virtually no variation (flat lines) for all parameters measured and gene types among tissues. This is not surprising given the ubiquitous nature of the expression of each gene type in a broad range of tissues. The average <em>Ï‰</em> values when <em>n</em> &gt; 10 tissues were remarkably lower, particularly for SSD genes and singletons (0.15 and 0.17, respectively), than those in observed when <em>n</em> â‰¤ 10 tissues (0.26 and 0.29, respectively; Fig.Â <a reveal-id="DSS012F2" data-open="DSS012F2" class="link link-reveal link-table xref-fig">2</a>F and B and TableÂ <a reveal-id="DSS012TB1" data-open="DSS012TB1" class="link link-reveal link-table xref-fig">1</a>). In terms of the proportion of gene types (Fig.Â <a reveal-id="DSS012F2" data-open="DSS012F2" class="link link-reveal link-table xref-fig">2</a>G), the proportion of ohnologues (40%) was substantially higher than that of SSD genes and singletons (30% in both cases). This result is in sharp contrast to that found for expression breadths of <em>n</em> â‰¤ 10 tissues, indicating that among those genes that are expressed ubiquitously, there is a larger proportion of ohnologues and a lower proportion of SSD genes and singletons. Finally, the relative contribution of ohnologues, SSD genes, and singletons is nearly the same for all tissues (Fig.Â <a reveal-id="DSS012F2" data-open="DSS012F2" class="link link-reveal link-table xref-fig">2</a>H), which is in high contrast to the situation where <em>n</em> â‰¤ 10 tissues (Fig.Â <a reveal-id="DSS012F2" data-open="DSS012F2" class="link link-reveal link-table xref-fig">2</a>D).</p><p>Overall, the results show that the relative proportion of ohnologues, SSD genes, and singletons, as well as their evolutionary rates, are substantially different between genes with ubiquitous and restricted expression breadths.</p>                            <h3 scrollto-destination=6758461 id="6758461" class="section-title" data-legacy-id=s3h>3.8. Proportions of ohnologues and SSD genes in various expression breadths</h3>
<p>To confirm the previous findings, we analysed cases involving expression breadths other than <em>n</em> â‰¤ 10 and <em>n</em> &gt; 10 tissues. Similar results to those shown in Fig.Â <a reveal-id="DSS012F2" data-open="DSS012F2" class="link link-reveal link-table xref-fig">2</a>Aâ€“D were obtained for <em>n</em> = 1, <em>n</em> â‰¤ 3, <em>n</em> â‰¤ 5, <em>n</em> â‰¤ 20, and <em>n</em> â‰¤ 40 tissues, whereas similar observations as those indicated in Fig.Â <a reveal-id="DSS012F2" data-open="DSS012F2" class="link link-reveal link-table xref-fig">2</a>Eâ€“H were obtained for <em>n</em> &gt; 20 and <em>n</em> &gt; 40 tissues (data not shown). FigureÂ <a reveal-id="DSS012F3" data-open="DSS012F3" class="link link-reveal link-table xref-fig">3</a> shows the proportion of ohnologues (blue) and SSD genes (red) for each tissue under various expression breadths (<em>n</em> = 1, <em>n</em> â‰¤ 3, <em>n</em> â‰¤ 5, <em>n</em> â‰¤ 10, <em>n</em> â‰¤ 20, <em>n</em> â‰¤ 40, <em>n</em> &gt; 10, <em>n</em> &gt; 20, and <em>n</em> &gt; 40). The <em>x</em>-axis represents the tissue evolutionary rate, whereas the <em>y</em>-axis represents the proportions of ohnologues and SSD genes. In tissue-restricted expression (<em>n</em> = 1, <em>n</em> â‰¤ 3, <em>n</em> â‰¤ 5, <em>n</em> â‰¤ 20, and <em>n</em> â‰¤ 40), the faster the tissue evolutionary rates, the higher the proportion of SSD genes and the lower the proportion of ohnologues expressed (correlation coefficients and <em>P</em>-values are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). When the expression breadth was â‰¤10, a statistically significant correlation was detected for both the ohnologues and SSD genes (<em>P</em> = 1.6 Ã— 10<sup>âˆ’13</sup> and <em>P</em> = 2.1 Ã— 10<sup>âˆ’5</sup>, respectively). A similar correlation was observed when the expression breadth was set at â‰¤20 and â‰¤40. However, the correlation became rather weak in these cases. Therefore, at expression breadth â‰¤10, correlation became maximum and most reliable. </p>                        <a id="6758464" scrollto-destination="6758464"></a>
<div id="DSS012F3" data-id="DSS012F3" class="fig fig-section"><div class="label fig-label">FigureÂ 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/m_dss01203.jpeg?Expires=2147483647&Signature=Mpm1JgVSKM41RWEAT2LtwR0BcdeB~toL6h~EH56g-ZkYIHHdK~pfrcai3Qd~FhpBoYzYFar5sSs7-Y4Z0LajoJIkhmbKlaqx-ycrzZCw-E~D5Wsg3GKSfoENsHmQSIXBshSYYuUP5-RnE-1~BYkjxLjqkrCfqW7mflIrpsA3Feiy~edLMpqFD~702QVQT2MD0K6ROLh-zCsGm7aNKzwxvkcrdvjhgtSdLSRwvTsd7XOM7YhKweIXfgy3rK1oYQQCOFzOXqUPYgVuFT5extRw7Eojr4CcYZ9GmnmrZy-xXLZaW9n7qgpgnfL7blFkzMP0zE3mhqAlpT8WoNJw6SYsng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Proportion of ohnologues and SSD genes at various gene expression breadths. The y-axis represents the proportions of ohnologues (blue) and SSD genes (red), whereas the x-axis shows the tissue evolutionary rates [the average KA/KS (Ï‰) value of all genes expressed in a given tissue]. P-values and linear correlation coefficients are shown in Supplementary Table S2." path-from-xml="dss01203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6758464/dss01203.jpeg" path-from-xml="dss01203.jpeg" target="_blank">View large</a><a section="6758464" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/dss01203.jpeg?Expires=2147483647&Signature=o~ahgtYv5y~6n6ibjq99KyMxQmNUjSdzeulJhv6CLmu9PH6ELWwY2PeBBGpbjyEkje7ZWkRc0wK~7dh-6LbEXHR2tY8XE4ob6P5KNoF14RKxXYTfWUeI9nhxSq0PrTddTCK1GhgPu02J65JICAeKsOuLkdZMQ-HOQhRVq-lURzYgzWxkGP-iULMCGHChIXV5ryiYi2imKwbgMAxyPIKIhnK8tgqGbIuUiqw3TcIBHkUyoRIvzUrZkg~0VWRvgAUQDhrWUpV6pOfQpTW5eNl5YSnm7Limp72yaBx4BT52ZMq-jdSZ2~g66Vf6DBxsV8IHF84biXDME23ZW1hkvpm8wA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6758464&ar=450433&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Proportion of ohnologues and SSD genes at various gene expression breadths. The <em>y</em>-axis represents the proportions of ohnologues (blue) and SSD genes (red), whereas the <em>x</em>-axis shows the tissue evolutionary rates [the average <em>K</em><sub>A</sub><em>/K</em><sub>S</sub> (<em>Ï‰</em>) value of all genes expressed in a given tissue]. <em>P</em>-values and linear correlation coefficients are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>.</p></div></div><div content-id="DSS012F3" class="fig fig-modal reveal-modal"><div class="label fig-label">FigureÂ 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/m_dss01203.jpeg?Expires=2147483647&Signature=Mpm1JgVSKM41RWEAT2LtwR0BcdeB~toL6h~EH56g-ZkYIHHdK~pfrcai3Qd~FhpBoYzYFar5sSs7-Y4Z0LajoJIkhmbKlaqx-ycrzZCw-E~D5Wsg3GKSfoENsHmQSIXBshSYYuUP5-RnE-1~BYkjxLjqkrCfqW7mflIrpsA3Feiy~edLMpqFD~702QVQT2MD0K6ROLh-zCsGm7aNKzwxvkcrdvjhgtSdLSRwvTsd7XOM7YhKweIXfgy3rK1oYQQCOFzOXqUPYgVuFT5extRw7Eojr4CcYZ9GmnmrZy-xXLZaW9n7qgpgnfL7blFkzMP0zE3mhqAlpT8WoNJw6SYsng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Proportion of ohnologues and SSD genes at various gene expression breadths. The y-axis represents the proportions of ohnologues (blue) and SSD genes (red), whereas the x-axis shows the tissue evolutionary rates [the average KA/KS (Ï‰) value of all genes expressed in a given tissue]. P-values and linear correlation coefficients are shown in Supplementary Table S2." path-from-xml="dss01203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6758464/dss01203.jpeg" path-from-xml="dss01203.jpeg" target="_blank">View large</a><a section="6758464" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/dss01203.jpeg?Expires=2147483647&Signature=o~ahgtYv5y~6n6ibjq99KyMxQmNUjSdzeulJhv6CLmu9PH6ELWwY2PeBBGpbjyEkje7ZWkRc0wK~7dh-6LbEXHR2tY8XE4ob6P5KNoF14RKxXYTfWUeI9nhxSq0PrTddTCK1GhgPu02J65JICAeKsOuLkdZMQ-HOQhRVq-lURzYgzWxkGP-iULMCGHChIXV5ryiYi2imKwbgMAxyPIKIhnK8tgqGbIuUiqw3TcIBHkUyoRIvzUrZkg~0VWRvgAUQDhrWUpV6pOfQpTW5eNl5YSnm7Limp72yaBx4BT52ZMq-jdSZ2~g66Vf6DBxsV8IHF84biXDME23ZW1hkvpm8wA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6758464&ar=450433&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Proportion of ohnologues and SSD genes at various gene expression breadths. The <em>y</em>-axis represents the proportions of ohnologues (blue) and SSD genes (red), whereas the <em>x</em>-axis shows the tissue evolutionary rates [the average <em>K</em><sub>A</sub><em>/K</em><sub>S</sub> (<em>Ï‰</em>) value of all genes expressed in a given tissue]. <em>P</em>-values and linear correlation coefficients are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>.</p></div></div><p>On the other hand, in those cases where <em>n</em> &gt; 20 and <em>n</em> &gt; 40 tissues, the proportion of ohnologues was higher and the proportion of SSD genes was lower but there was no significant correlation between tissue evolutionary rates and the proportion of either gene type.</p>                            <h3 scrollto-destination=6758468 id="6758468" class="section-title" data-legacy-id=s3i>3.9. Orthologous genes between human and other vertebrates</h3>
<p>The previous analyses involved genes that are orthologous in human and mice. However, the origin of ohnologues can be traced back to the emergence of vertebrates. Therefore, human orthologues in various vertebrates, including rat, cow, dog, opossum, and chicken, were extracted from Ensembl. Using an expression breadth of <em>n</em> â‰¤ 10, analyses similar to those shown in Fig.Â <a reveal-id="DSS012F2" data-open="DSS012F2" class="link link-reveal link-table xref-fig">2</a>C were performed. In the results shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>, human genes were used as references for pairwise comparisons. Similar to the results shown in Fig.Â <a reveal-id="DSS012F2" data-open="DSS012F2" class="link link-reveal link-table xref-fig">2</a>C, the comparison between humans and other species (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>) shows that those tissues with higher <em>Ï‰</em> values were associated with a larger proportion of SSD genes than with ohnologues.</p>                            <h3 scrollto-destination=6758472 id="6758472" class="section-title" data-legacy-id=s3j>3.10. Recent SSD genes contribute to faster tissue evolution</h3>
<p>Approximate gene duplication times can be estimated based on <em>K</em><sub>S</sub> values, with lower and higher <em>K</em><sub>S</sub> values corresponding to recent and ancient duplication events, respectively. To examine the relative contribution of each SSD gene to tissue expression, a threshold value of <em>K</em><sub>S</sub> was set to 1.0 and the number and the percentage of each SSD gene at various expression breadths were counted (Fig.Â <a reveal-id="DSS012F4" data-open="DSS012F4" class="link link-reveal link-table xref-fig">4</a>Aâ€“C). Narrower expression breadths were associated with a larger percentage of recent SSD genes (<em>K</em><sub>S</sub> &lt; 1.0, red), whereas the percentage of ancient SSD genes (<em>K</em><sub>S</sub> â‰¥ 1.0, pink) was nearly constant regardless of expression breadth. These results indicate a higher contribution of recent SSD genes to tissue-restricted gene expression. </p>                        <a id="6758477" scrollto-destination="6758477"></a>
<div id="DSS012F4" data-id="DSS012F4" class="fig fig-section"><div class="label fig-label">FigureÂ 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/m_dss01204.jpeg?Expires=2147483647&Signature=gQbLHutzZsZf-W286jg2i9-o0JA9jdrw2Rjgkdw2hrgCc4jRTXkrkSM9MQbZdnTjtd~4DFMzv6h65TV1nSccU-luNE0lbR~hukzIQPu4hYH-PZcOFAS3kzjnbNGnlZEvpNgYbtCTFSo3Rr2g65v~cWzTo3lQpslRNquJahoAIfg-SOPXw4xhaD2OqB0mGOyP-dzVzHEVKe0r3ZD5OboXFOQ4dsCAvgS2kLopyhNDnOJu-CVSrloqZll-5NSTaH3QR~9Qfkl2kSgvS8iliIxqF7PXAa4hFoAIa0W~JyWZlaWgNzvseUEwFHLxGlD8uDMQbXiDYJcOBjuMzO5AmIHibA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Recent and ancient SSD genes. (Aâ€“C) The number and the percentage of ancient (pink) and recent (red) SSD genes are shown for each tissue expression breadth. (D) The proportions of ancient and recent SSD genes among genes with tissue-restricted expression limited to n â‰¤ 10 tissues." path-from-xml="dss01204.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6758477/dss01204.jpeg" path-from-xml="dss01204.jpeg" target="_blank">View large</a><a section="6758477" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/dss01204.jpeg?Expires=2147483647&Signature=gJaXcE11IHba6OGsOXok5Q149oSLdQFxCa0EoQb5LpVc1p2jEDBfj4XlZbRPRVa2TrtnX4iuEBKkA3Un4qT9n5Xz9kn~gcNwIfovRWaowRrrS7u2v95Fv~5OXGWnsbtoELc~IH7Sr780M47P9~6hi5w9dU4FItKDdk~reiHsc2mRyxFkaaw~Iq8VG3LBWoeliTM4cNWqYSjPz8f0q32ZnSCL32aGvmR1s8Q6ByGbVZ6uq2em8lLylRCvsyE07NExPWaVa-EpfI8Xv9U09L6zRI5v6VP8RSKv9TuWe87yV4HFEDAgxhirzjQ9AsLv307BycQrT4Ek2CNYb8oSfOiE7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6758477&ar=450433&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01204.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Recent and ancient SSD genes. (Aâ€“C) The number and the percentage of ancient (pink) and recent (red) SSD genes are shown for each tissue expression breadth. (D) The proportions of ancient and recent SSD genes among genes with tissue-restricted expression limited to <em>n</em> â‰¤ 10 tissues.</p></div></div><div content-id="DSS012F4" class="fig fig-modal reveal-modal"><div class="label fig-label">FigureÂ 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/m_dss01204.jpeg?Expires=2147483647&Signature=gQbLHutzZsZf-W286jg2i9-o0JA9jdrw2Rjgkdw2hrgCc4jRTXkrkSM9MQbZdnTjtd~4DFMzv6h65TV1nSccU-luNE0lbR~hukzIQPu4hYH-PZcOFAS3kzjnbNGnlZEvpNgYbtCTFSo3Rr2g65v~cWzTo3lQpslRNquJahoAIfg-SOPXw4xhaD2OqB0mGOyP-dzVzHEVKe0r3ZD5OboXFOQ4dsCAvgS2kLopyhNDnOJu-CVSrloqZll-5NSTaH3QR~9Qfkl2kSgvS8iliIxqF7PXAa4hFoAIa0W~JyWZlaWgNzvseUEwFHLxGlD8uDMQbXiDYJcOBjuMzO5AmIHibA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Recent and ancient SSD genes. (Aâ€“C) The number and the percentage of ancient (pink) and recent (red) SSD genes are shown for each tissue expression breadth. (D) The proportions of ancient and recent SSD genes among genes with tissue-restricted expression limited to n â‰¤ 10 tissues." path-from-xml="dss01204.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6758477/dss01204.jpeg" path-from-xml="dss01204.jpeg" target="_blank">View large</a><a section="6758477" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/dss01204.jpeg?Expires=2147483647&Signature=gJaXcE11IHba6OGsOXok5Q149oSLdQFxCa0EoQb5LpVc1p2jEDBfj4XlZbRPRVa2TrtnX4iuEBKkA3Un4qT9n5Xz9kn~gcNwIfovRWaowRrrS7u2v95Fv~5OXGWnsbtoELc~IH7Sr780M47P9~6hi5w9dU4FItKDdk~reiHsc2mRyxFkaaw~Iq8VG3LBWoeliTM4cNWqYSjPz8f0q32ZnSCL32aGvmR1s8Q6ByGbVZ6uq2em8lLylRCvsyE07NExPWaVa-EpfI8Xv9U09L6zRI5v6VP8RSKv9TuWe87yV4HFEDAgxhirzjQ9AsLv307BycQrT4Ek2CNYb8oSfOiE7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6758477&ar=450433&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01204.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Recent and ancient SSD genes. (Aâ€“C) The number and the percentage of ancient (pink) and recent (red) SSD genes are shown for each tissue expression breadth. (D) The proportions of ancient and recent SSD genes among genes with tissue-restricted expression limited to <em>n</em> â‰¤ 10 tissues.</p></div></div><p>An additional analysis similar to that shown in Fig.Â <a reveal-id="DSS012F2" data-open="DSS012F2" class="link link-reveal link-table xref-fig">2</a>C was performed using recent and ancient SSD genes as criteria. The results are shown in Fig.Â <a reveal-id="DSS012F4" data-open="DSS012F4" class="link link-reveal link-table xref-fig">4</a>D (expression breadth â‰¤10 tissues). Co-linearity between <em>Ï‰</em> values and gene proportions was 0.58 for recent (<em>P</em> = 1.7 Ã— 10<sup>âˆ’5</sup>) and 0.33 for ancient SSD genes (<em>P</em> = 0.024), indicating a larger contribution of recent SSD genes to the positive co-linearity among all SSD genes observed in Fig.Â <a reveal-id="DSS012F2" data-open="DSS012F2" class="link link-reveal link-table xref-fig">2</a>C. Yet, the slightly positive co-linearity observed for ancient SSD genes indicates that ancient SSD genes are distinct from ohnologues, which showed negative co-linearity. These results indicate that the evolution of SSD genes of both recent and ancient origins, as well as of ohnologues, is distinct in nature.</p>                            <h3 scrollto-destination=6758481 id="6758481" class="section-title" data-legacy-id=s3k>3.11. Association between gene evolutionary rates, duplication mode, and developmental origin of tissues</h3>
<p>In the analyses of the results shown in Fig.Â <a reveal-id="DSS012F2" data-open="DSS012F2" class="link link-reveal link-table xref-fig">2</a>, fast and slow tissue evolutionary rates were based on <em>Ï‰</em> values but we did not address tissue types. To examine how <em>Ï‰</em> values and the proportion of SSD genes and ohnologues are associated with the developmental origin of tissues, <em>Ï‰</em> values as well as the proportions of ohnologues, SSD genes, and singletons were calculated for each tissue type. Values calculated for the four subgroups with a breadth of <em>n</em> â‰¤ 10 tissues are presented in Fig.Â <a reveal-id="DSS012F5" data-open="DSS012F5" class="link link-reveal link-table xref-fig">5</a> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a> shows <em>P</em>-values). </p>                        <a id="6758483" scrollto-destination="6758483"></a>
<div id="DSS012F5" data-id="DSS012F5" class="fig fig-section"><div class="label fig-label">FigureÂ 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/m_dss01205.jpeg?Expires=2147483647&Signature=K7EApP8hImY6Qd65cuR00iUd9H~O-pIbGTIZtwBM6mGhBZtT1qv96KFJHHgBGcs9V8O8Ex5zthj1dNX4xfpIsuC8CddKVwR1ycruZ9Kt4Svwu9H4OFhhpRTCZKEwDoYPUwDax7Pr4seZOx6fUcefLQICfnaIvEC0w4KxHu2HyMy4DkwRoA8D0sfJanK6ZuyueVrd7rJNKSAV3Drxr9pNig4cAiREzzCWPRJE5~fcxnXFzzTPHix0~5kqTjY9adMUfE~H6G-j~kL6IHfbOBuXLci-95FWQvINMfsqlCUEBXEU2YZPr4wYNvRRbW6tpxndX2sqIqu66cKvslMau8mP4w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Contribution of ohnologues, SSD genes, and singletons to gene sets expressed in haemocytes, tissues of endodermal and ectodermal origin, and mesenchymal tissues. (A) The average KA/KS (Ï‰) values associated with each gene set. (B)â€“(D) The proportion of ohnologues, SSD genes, and singletons expressed in each tissue type. In each panel, averages and standard deviations are shown. P-values for pairwise comparisons are shown in Supplementary Table S4." path-from-xml="dss01205.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6758483/dss01205.jpeg" path-from-xml="dss01205.jpeg" target="_blank">View large</a><a section="6758483" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/dss01205.jpeg?Expires=2147483647&Signature=QX~u4j-TERhhxuXGDQzSX-h5CPVnnPVElX591dACxnT-fUvL7KBFIk4rRqE9Qwuff~SA9AcViY6NN~Iufh5Ptueg7GQ0ObdlkAhYWMmnCDQKpsbF9ShN-5P2gEuCRix22HJ-laygR3kpcoQfexmDOxym8y4zXtvycRcXm0Ustn2kHs-R8atF1eAbWDChO0Gjn8L5xKaoU7E2kMHGPvRIoqiGZFU3ISWwFMPvl4YAWco1ab3lw0IDmrG46XBubwzOqFD1WVz4jacMolq3BhBJvYDbZNltyG6okdUhbv~5RTMa8yXY5rNecCho3mfPsO3LD6tLIq4YXqSjnJQ7IkdZVg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6758483&ar=450433&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01205.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Contribution of ohnologues, SSD genes, and singletons to gene sets expressed in haemocytes, tissues of endodermal and ectodermal origin, and mesenchymal tissues. (A) The average <em>K</em><sub>A</sub><em>/K</em><sub>S</sub> (<em>Ï‰</em>) values associated with each gene set. (B)â€“(D) The proportion of ohnologues, SSD genes, and singletons expressed in each tissue type. In each panel, averages and standard deviations are shown. <em>P</em>-values for pairwise comparisons are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>.</p></div></div><div content-id="DSS012F5" class="fig fig-modal reveal-modal"><div class="label fig-label">FigureÂ 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/m_dss01205.jpeg?Expires=2147483647&Signature=K7EApP8hImY6Qd65cuR00iUd9H~O-pIbGTIZtwBM6mGhBZtT1qv96KFJHHgBGcs9V8O8Ex5zthj1dNX4xfpIsuC8CddKVwR1ycruZ9Kt4Svwu9H4OFhhpRTCZKEwDoYPUwDax7Pr4seZOx6fUcefLQICfnaIvEC0w4KxHu2HyMy4DkwRoA8D0sfJanK6ZuyueVrd7rJNKSAV3Drxr9pNig4cAiREzzCWPRJE5~fcxnXFzzTPHix0~5kqTjY9adMUfE~H6G-j~kL6IHfbOBuXLci-95FWQvINMfsqlCUEBXEU2YZPr4wYNvRRbW6tpxndX2sqIqu66cKvslMau8mP4w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Contribution of ohnologues, SSD genes, and singletons to gene sets expressed in haemocytes, tissues of endodermal and ectodermal origin, and mesenchymal tissues. (A) The average KA/KS (Ï‰) values associated with each gene set. (B)â€“(D) The proportion of ohnologues, SSD genes, and singletons expressed in each tissue type. In each panel, averages and standard deviations are shown. P-values for pairwise comparisons are shown in Supplementary Table S4." path-from-xml="dss01205.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6758483/dss01205.jpeg" path-from-xml="dss01205.jpeg" target="_blank">View large</a><a section="6758483" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/dss01205.jpeg?Expires=2147483647&Signature=QX~u4j-TERhhxuXGDQzSX-h5CPVnnPVElX591dACxnT-fUvL7KBFIk4rRqE9Qwuff~SA9AcViY6NN~Iufh5Ptueg7GQ0ObdlkAhYWMmnCDQKpsbF9ShN-5P2gEuCRix22HJ-laygR3kpcoQfexmDOxym8y4zXtvycRcXm0Ustn2kHs-R8atF1eAbWDChO0Gjn8L5xKaoU7E2kMHGPvRIoqiGZFU3ISWwFMPvl4YAWco1ab3lw0IDmrG46XBubwzOqFD1WVz4jacMolq3BhBJvYDbZNltyG6okdUhbv~5RTMa8yXY5rNecCho3mfPsO3LD6tLIq4YXqSjnJQ7IkdZVg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6758483&ar=450433&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01205.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Contribution of ohnologues, SSD genes, and singletons to gene sets expressed in haemocytes, tissues of endodermal and ectodermal origin, and mesenchymal tissues. (A) The average <em>K</em><sub>A</sub><em>/K</em><sub>S</sub> (<em>Ï‰</em>) values associated with each gene set. (B)â€“(D) The proportion of ohnologues, SSD genes, and singletons expressed in each tissue type. In each panel, averages and standard deviations are shown. <em>P</em>-values for pairwise comparisons are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>.</p></div></div><p>The average <em>Ï‰</em> values were significantly higher in endodermal, intermediate in mesenchymal, and lower in ectodermal tissues (Fig.Â <a reveal-id="DSS012F5" data-open="DSS012F5" class="link link-reveal link-table xref-fig">5</a>A). Conversely, the proportion of ohnologues was higher in ectodermal than in endodermal and mesenchymal tissues (Fig.Â <a reveal-id="DSS012F5" data-open="DSS012F5" class="link link-reveal link-table xref-fig">5</a>B), whereas the proportion of SSD genes was highest in endodermal, intermediate in mesenchymal, and lowest in ectodermal tissues (Fig.Â <a reveal-id="DSS012F5" data-open="DSS012F5" class="link link-reveal link-table xref-fig">5</a>C). The proportion of singletons did not vary among the three subgroups (Fig.Â <a reveal-id="DSS012F5" data-open="DSS012F5" class="link link-reveal link-table xref-fig">5</a>D). Therefore, it is possible that SSD genes with higher <em>Ï‰</em> values might have contributed to the faster evolution of endodermal tissues, whereas ohnologues with lower <em>Ï‰</em> values might have functioned to maintain the essential characteristics of ectodermal tissues. Mesenchymal tissues occupied an intermediate position.</p>                            <h3 scrollto-destination=6758486 id="6758486" class="section-title" data-legacy-id=s3l>3.12. The unique evolutionary position of haemocytes</h3>
<p>Haemocyte genes displayed unique features that were distinct from those of the three subgroups of tissues described previously. Specifically, the average <em>Ï‰</em> value of haemocyte genes was the highest, whereas the proportion of haemocyte ohnologues was the lowest. Furthermore, the proportion of haemocyte singletons was higher than that of the other three subtypes of tissues.</p>                            <h2 scrollto-destination=6758489 id="6758489" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=6758491 id="6758491" class="section-title" data-legacy-id=s4a>4.1. Gene evolutionary rate, duplication, and expression breadth</h3>
<p>In the present study, the putative association between gene evolutionary rates, duplication, and their expression breadth in different tissues was examined. Gene evolutionary rates are affected by various factors. For example, duplicated genes are believed to evolve relatively fast, whereas singletons evolve more slowly.<sup><a reveal-id="DSS012C1" data-open="DSS012C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Another important factor is the expression breadth of each gene. Genes expressed in a wide range of tissues tend to evolve more slowly, whereas those restricted to a narrow range of tissues evolve faster.<sup><a reveal-id="DSS012C18" data-open="DSS012C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSS012C25" data-open="DSS012C25" class="link link-ref link-reveal xref-bibr">25</a></sup> In addition, as the number of genes in a gene family increases, the number of tissues in which the corresponding gene is expressed decreases.<sup><a reveal-id="DSS012C26" data-open="DSS012C26" class="link link-ref link-reveal xref-bibr">26</a></sup> However, these findings were based on the analysis of only two parameters at a time, even though co-linearity among the three elements considered (gene evolutionary rate, gene duplication, and gene expression breadth) is also possible.</p><p>To analyse the involvement of duplicated genes on tissue evolution, we initially focused on the relationship between gene expression breadth and gene duplication. However, the percentages of singletons (Fig.Â <a reveal-id="DSS012F1" data-open="DSS012F1" class="link link-reveal link-table xref-fig">1</a>D, green) and duplicated genes (Fig.Â <a reveal-id="DSS012F1" data-open="DSS012F1" class="link link-reveal link-table xref-fig">1</a>D; red + blue) remained nearly constant at 30â€“35 and 65â€“70%, respectively, and were independent of gene expression breadth. This result suggests that the association between gene expression breadth and gene duplication is not as simple as reported previously.</p><p>The present study not only considered the analysis of gene evolutionary rates but also incorporated into it the distinction between SSD genes and ohnologues when considering gene duplication. Gene evolutionary rates were fast and slow for SSD genes and ohnologues, respectively (TableÂ <a reveal-id="DSS012TB1" data-open="DSS012TB1" class="link link-reveal link-table xref-fig">1</a>). Interestingly, SSD genes were represented in a larger proportion within narrow expression ranges, whereas the opposite was observed for ohnologues (Fig.Â <a reveal-id="DSS012F1" data-open="DSS012F1" class="link link-reveal link-table xref-fig">1</a>D). The present study is therefore the first to show co-linearity among gene evolutionary rate, duplication, and expression breadth.</p>                            <h3 scrollto-destination=6758498 id="6758498" class="section-title" data-legacy-id=s4b>4.2. Contribution of SSD genes, ohnologues, and singletons to expression breadths in tissues</h3>
<p>Genes that are expressed in a narrow range of tissues are considered to play significant roles in the establishment of the characteristic features of the corresponding tissues. Prior studies have shown that those genes that are expressed specifically in neural tissues evolve more slowly, whereas those expressed in other tissue types evolve faster.<sup><a reveal-id="DSS012C18" data-open="DSS012C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSS012C23" data-open="DSS012C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Alternatively, a substantial proportion of those genes with a tissue-specific expression tend to encode secreted polypeptides irrespective of tissue types, and there are few such genes in neural tissues.<sup><a reveal-id="DSS012C24" data-open="DSS012C24" class="link link-ref link-reveal xref-bibr">24</a></sup></p><p>In this study, we also examined if the nature of SSD genes depended on the type of tissue in which the corresponding genes were expressed. To this end, the evolutionary rate of a tissue was defined as the average <em>Ï‰</em> value of those genes expressed in that tissue, whereas the nature of the expressed genes was evaluated by their <em>Ï‰</em> values, proportions as well as by the product of these parameters. These parameters were calculated for each type of gene (SSD, ohnologues, and singletons) and plotted for each tissue (Fig.Â <a reveal-id="DSS012F2" data-open="DSS012F2" class="link link-reveal link-table xref-fig">2</a>). Among genes with a narrow spectrum of expression, SSD genes (and singletons) played a minor role in slowly evolving tissues with low <em>Ï‰</em> values. However, the evolutionary rates as well as the proportion of SSD genes were associated positively with the evolutionary rates of tissues. Thus, SSD genes (and singletons) seemed to play major roles in fast evolving tissues with high <em>Ï‰</em> values, indicating that the relative contribution of SSD genes differs substantially depending on the type of tissue in which the gene is expressed. In contrast, genes with a wide spectrum of expression are presumed to maintain basic cellular functions irrespective of the tissue type. Among these genes, the relative contribution of SSD genes and ohnologues was similar.</p><p>Our results indicate that ohnologues with slower rates of evolution and wide expression ranges are associated with slowly evolving tissues, whereas SSD genes with faster rates of evolution and narrow range of expression are associated with fast evolving tissues. Specifically, there were less ohnologues and more SSD genes in endodermal tissues such as the digestive tract. These tissues face the outer environment directly and therefore they may differentiate functionally and pleiotropically to adjust to environmental changes. Rapid tissue evolutionary rates could be therefore related to a specific mode of duplication (SSD genes) and faster molecular evolution (high <em>Ï‰</em> values) in response to this exposure to the outer environment.</p><p>Conversely, in ectoderm-derived tissues, such as the nervous system, there were more ohnologues and less SSD genes. Because these tissues are not exposed to the outer environment, their evolution may not be driven by functional differentiation but rather by the need to maintain basal functions. Ohnologues with a slow evolutionary rate and lower probability of further duplication were therefore predominant in slowly evolving tissues such as the ectoderm. Mesenchymal tissues were associated with an intermediate phenotype in terms of <em>Ï‰</em> values and the proportion of ohnologues and SSD genes.</p><p>Those genes with expression restricted to haemocytes exhibited very unique features. The percentage of ohnologues was the smallest among all tissue types, whereas singletons were the most representative, a result not observed for the other three tissue types. A lower proportion of conservative ohnologues might allow the relaxation of those constraints required for the formation and functionality of tissues. This flexibility might be further accelerated by an increase in the ratio of singletons. Such findings might be expected in light of the dramatic transitions of haematopoietic organs and tissues from the aorta, gonads, and mesonephros regions to the liver, spleen, and bone marrow during mammalian development and vertebrate evolution. Furthermore, the haemocytes used in this study included immune-competent cells and tissues, and immune-related genes are known to evolve fast (immunoglobulin and T cell receptor gene families were excluded from the analysis here). In this sense, endodermal tissues and haemocytes appear to adopt distinct strategies to increase the proportion of SSD genes and singletons, respectively, when adapting to environmental changes.</p>                            <h2 scrollto-destination=6758508 id="6758508" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at</a><a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=6758511 id="6758511" class="section-title" data-legacy-id=s6>Funding</h2>
<p>The work was supported by research grants from the Ministry of Education, Science, Sports, Culture and Technology of Japan and by the Creative Interdisciplinary Research Program of the Center for Interdisciplinary Research, Tohoku University, Japan. M.S. is supported by a GCOE program, â€˜Network Medicineâ€™, at Tohoku University.</p>                            <h2 scrollto-destination=6758513 id="6758513" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Koji Tamura for the valuable comments.</p>                            <h2 scrollto-destination=6758516 id="6758516" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSS012C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Graur</div> <div class="given-names">D.</div></div>,Â  <div class="name"><div class="surname">Li</div> <div class="given-names">W.-H.</div></div>.Â </div>,Â <div class="source">Fundamentals of Molecular Evolution</div>Â ,Â <div class="year">1999</div><div class="edition">2nd edition</div><div class="publisher-loc">Sunderland, MA</div><div class="publisher-name">Sinauer Associates</div></div></div></div></div><div content-id="DSS012C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McLysaght</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Hokamp</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div>.Â </div><div class="article-title">Extensive genomic duplication during early chordate evolution</div>,Â <div class="source">Nat. Genet.</div>Â ,Â <div class="year">2002</div>,Â vol.Â <div class="volume">31</div>Â (pg.Â <div class="fpage">200</div>-<div class="lpage">4</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/ng884" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/ng884</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensive%20genomic%20duplication%20during%20early%20chordate%20evolution&amp;author=A.%20McLysaght&amp;author=K.%20Hokamp&amp;author=K.H.%20Wolfe&amp;publication_year=2002&amp;journal=Nat.%20Genet.&amp;volume=31&amp;pages=200-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng884" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng884" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng884"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12032567" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS012C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakatani</div> <div class="given-names">Y.</div></div>,Â  <div class="name"><div class="surname">Takeda</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Kohara</div> <div class="given-names">Y.</div></div>,Â  <div class="name"><div class="surname">Morishita</div> <div class="given-names">S.</div></div>.Â </div><div class="article-title">Reconstruction of the vertebrate ancestral genome reveals dynamic genome reorganization in early vertebrates</div>,Â <div class="source">Genome Res.</div>Â ,Â <div class="year">2007</div>,Â vol.Â <div class="volume">17</div>Â (pg.Â <div class="fpage">1254</div>-<div class="lpage">65</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.6316407" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.6316407</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reconstruction%20of%20the%20vertebrate%20ancestral%20genome%20reveals%20dynamic%20genome%20reorganization%20in%20early%20vertebrates&amp;author=Y.%20Nakatani&amp;author=H.%20Takeda&amp;author=Y.%20Kohara&amp;author=S.%20Morishita&amp;publication_year=2007&amp;journal=Genome%20Res.&amp;volume=17&amp;pages=1254-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.6316407" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.6316407" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.6316407"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17652425" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS012C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dehal</div> <div class="given-names">P.</div></div>,Â  <div class="name"><div class="surname">Boore</div> <div class="given-names">J.L.</div></div>.Â </div><div class="article-title">Two rounds of whole genome duplication in the ancestral vertebrate</div>,Â <div class="source">PLoS Biol.</div>Â ,Â <div class="year">2005</div>,Â vol.Â <div class="volume">3</div>Â pg.Â <div class="fpage">e314</div>Â <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pbio.0030314" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pbio.0030314</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20rounds%20of%20whole%20genome%20duplication%20in%20the%20ancestral%20vertebrate&amp;author=P.%20Dehal&amp;author=J.L.%20Boore&amp;publication_year=2005&amp;journal=PLoS%20Biol.&amp;volume=3&amp;pages=e314" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0030314" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0030314" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0030314"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16128622" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS012C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohno</div> <div class="given-names">S.</div></div>.Â </div>,Â <div class="source">Evolution by Gene Duplication</div>Â ,Â <div class="year">1970</div><div class="publisher-loc">Berlin</div><div class="publisher-name">Springer</div></div></div></div></div><div content-id="DSS012C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wolfe</div> <div class="given-names">K.</div></div>.Â </div><div class="article-title">Robustnessâ€“it's not where you think it is</div>,Â <div class="source">Nat. Genet.</div>Â ,Â <div class="year">2000</div>,Â vol.Â <div class="volume">25</div>Â (pg.Â <div class="fpage">3</div>-<div class="lpage">4</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/75560" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/75560</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Robustness%E2%80%93it's%20not%20where%20you%20think%20it%20is&amp;author=K.%20Wolfe&amp;publication_year=2000&amp;journal=Nat.%20Genet.&amp;volume=25&amp;pages=3-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/75560" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F75560" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F75560"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10802639" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS012C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gu</div> <div class="given-names">Z.</div></div>,Â  <div class="name"><div class="surname">Steinmetz</div> <div class="given-names">L.M.</div></div>,Â  <div class="name"><div class="surname">Gu</div> <div class="given-names">X.</div></div>,Â  <div class="name"><div class="surname">Scharfe</div> <div class="given-names">C.</div></div>,Â  <div class="name"><div class="surname">Davis</div> <div class="given-names">R.W.</div></div>,Â  <div class="name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div>.Â </div><div class="article-title">Role of duplicate genes in genetic robustness against null mutations</div>,Â <div class="source">Nature</div>Â ,Â <div class="year">2003</div>,Â vol.Â <div class="volume">421</div>Â (pg.Â <div class="fpage">63</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature01198" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature01198</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20duplicate%20genes%20in%20genetic%20robustness%20against%20null%20mutations&amp;author=Z.%20Gu&amp;author=L.M.%20Steinmetz&amp;author=X.%20Gu&amp;author=C.%20Scharfe&amp;author=R.W.%20Davis&amp;author=W.H.%20Li&amp;publication_year=2003&amp;journal=Nature&amp;volume=421&amp;pages=63-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01198" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01198" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01198"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12511954" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS012C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Conant</div> <div class="given-names">G.C.</div></div>,Â  <div class="name"><div class="surname">Wagner</div> <div class="given-names">A.</div></div>.Â </div><div class="article-title">Duplicate genes and robustness to transient gene knock-downs in <em>Caenorhabditis elegans</em></div>,Â <div class="source">Proc. Biol. Sci.</div>Â ,Â <div class="year">2004</div>,Â vol.Â <div class="volume">271</div>Â (pg.Â <div class="fpage">89</div>-<div class="lpage">96</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1098/rspb.2003.2560" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1098/rspb.2003.2560</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Duplicate%20genes%20and%20robustness%20to%20transient%20gene%20knock-downs%20in%20Caenorhabditis%20elegans&amp;author=G.C.%20Conant&amp;author=A.%20Wagner&amp;publication_year=2004&amp;journal=Proc.%20Biol.%20Sci.&amp;volume=271&amp;pages=89-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rspb.2003.2560" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frspb.2003.2560" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frspb.2003.2560"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15002776" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS012C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liang</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div>.Â </div><div class="article-title">Gene essentiality, gene duplicability and protein connectivity in human and mouse</div>,Â <div class="source">Trends Genet.</div>Â ,Â <div class="year">2007</div>,Â vol.Â <div class="volume">23</div>Â (pg.Â <div class="fpage">375</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.tig.2007.04.005" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.tig.2007.04.005</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20essentiality%2C%20gene%20duplicability%20and%20protein%20connectivity%20in%20human%20and%20mouse&amp;author=H.%20Liang&amp;author=W.H.%20Li&amp;publication_year=2007&amp;journal=Trends%20Genet.&amp;volume=23&amp;pages=375-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2007.04.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2007.04.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2007.04.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17512629" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS012C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liao</div> <div class="given-names">B.Y.</div></div>,Â  <div class="name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>.Â </div><div class="article-title">Mouse duplicate genes are as essential as singletons</div>,Â <div class="source">Trends Genet.</div>Â ,Â <div class="year">2007</div>,Â vol.Â <div class="volume">23</div>Â (pg.Â <div class="fpage">378</div>-<div class="lpage">81</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.tig.2007.05.006" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.tig.2007.05.006</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mouse%20duplicate%20genes%20are%20as%20essential%20as%20singletons&amp;author=B.Y.%20Liao&amp;author=J.%20Zhang&amp;publication_year=2007&amp;journal=Trends%20Genet.&amp;volume=23&amp;pages=378-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2007.05.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2007.05.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2007.05.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17559966" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS012C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Makino</div> <div class="given-names">T.</div></div>,Â  <div class="name"><div class="surname">Hokamp</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">McLysaght</div> <div class="given-names">A.</div></div>.Â </div><div class="article-title">The complex relationship of gene duplication and essentiality</div>,Â <div class="source">Trends Genet.</div>Â ,Â <div class="year">2009</div>,Â vol.Â <div class="volume">25</div>Â (pg.Â <div class="fpage">152</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.tig.2009.03.001" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.tig.2009.03.001</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complex%20relationship%20of%20gene%20duplication%20and%20essentiality&amp;author=T.%20Makino&amp;author=K.%20Hokamp&amp;author=A.%20McLysaght&amp;publication_year=2009&amp;journal=Trends%20Genet.&amp;volume=25&amp;pages=152-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2009.03.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2009.03.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2009.03.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19285746" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS012C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Veitia</div> <div class="given-names">R.A.</div></div>.Â </div><div class="article-title">Exploring the etiology of haploinsufficiency</div>,Â <div class="source">Bioessays</div>Â ,Â <div class="year">2002</div>,Â vol.Â <div class="volume">24</div>Â (pg.Â <div class="fpage">175</div>-<div class="lpage">84</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/bies.10023" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/bies.10023</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploring%20the%20etiology%20of%20haploinsufficiency&amp;author=R.A.%20Veitia&amp;publication_year=2002&amp;journal=Bioessays&amp;volume=24&amp;pages=175-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1521-1878" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1521-1878" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1521-1878"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11835282" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS012C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Veitia</div> <div class="given-names">R.A.</div></div>.Â </div><div class="article-title">Nonlinear effects in macromolecular assembly and dosage sensitivity</div>,Â <div class="source">J. Theor. Biol.</div>Â ,Â <div class="year">2003</div>,Â vol.Â <div class="volume">220</div>Â (pg.Â <div class="fpage">19</div>-<div class="lpage">25</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1006/jtbi.2003.3105" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1006/jtbi.2003.3105</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nonlinear%20effects%20in%20macromolecular%20assembly%20and%20dosage%20sensitivity&amp;author=R.A.%20Veitia&amp;publication_year=2003&amp;journal=J.%20Theor.%20Biol.&amp;volume=220&amp;pages=19-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jtbi.2003.3105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjtbi.2003.3105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjtbi.2003.3105"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12453447" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS012C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Veitia</div> <div class="given-names">R.A.</div></div>.Â </div><div class="article-title">Gene dosage balance in cellular pathways: implications for dominance and gene duplicability</div>,Â <div class="source">Genetics</div>Â ,Â <div class="year">2004</div>,Â vol.Â <div class="volume">168</div>Â (pg.Â <div class="fpage">569</div>-<div class="lpage">74</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1534/genetics.104.029785" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1534/genetics.104.029785</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20dosage%20balance%20in%20cellular%20pathways%3A%20implications%20for%20dominance%20and%20gene%20duplicability&amp;author=R.A.%20Veitia&amp;publication_year=2004&amp;journal=Genetics&amp;volume=168&amp;pages=569-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.104.029785" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.104.029785" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.104.029785"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15454568" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS012C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Makino</div> <div class="given-names">T.</div></div>,Â  <div class="name"><div class="surname">McLysaght</div> <div class="given-names">A.</div></div>.Â </div><div class="article-title">Ohnologs in the human genome are dosage balanced and frequently associated with disease</div>,Â <div class="source">Proc. Natl Acad. Sci. USA</div>Â ,Â <div class="year">2010</div>,Â vol.Â <div class="volume">107</div>Â (pg.Â <div class="fpage">9270</div>-<div class="lpage">4</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.0914697107" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.0914697107</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ohnologs%20in%20the%20human%20genome%20are%20dosage%20balanced%20and%20frequently%20associated%20with%20disease&amp;author=T.%20Makino&amp;author=A.%20McLysaght&amp;publication_year=2010&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=9270-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0914697107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0914697107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0914697107"> </span></div>Â </div></div></div></div></div><div content-id="DSS012C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liao</div> <div class="given-names">B.Y.</div></div>,Â  <div class="name"><div class="surname">Zhang</div> <div class="given-names">J.Z.</div></div>.Â </div><div class="article-title">Low rates of expression profile divergence in highly expressed genes and tissue-specific genes during mammalian evolution</div>,Â <div class="source">Mol. Biol. Evol.</div>Â ,Â <div class="year">2006</div>,Â vol.Â <div class="volume">23</div>Â (pg.Â <div class="fpage">1119</div>-<div class="lpage">28</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/molbev/msj119" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/molbev/msj119</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Low%20rates%20of%20expression%20profile%20divergence%20in%20highly%20expressed%20genes%20and%20tissue-specific%20genes%20during%20mammalian%20evolution&amp;author=B.Y.%20Liao&amp;author=J.Z.%20Zhang&amp;publication_year=2006&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=23&amp;pages=1119-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msj119" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsj119" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsj119"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16520335" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS012C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gu</div> <div class="given-names">X.</div></div>,Â  <div class="name"><div class="surname">Su</div> <div class="given-names">Z.X.</div></div>.Â </div><div class="article-title">Tissue-driven hypothesis of genomic evolution and sequence-expression correlations</div>,Â <div class="source">Proc. Natl Acad. Sci. USA</div>Â ,Â <div class="year">2007</div>,Â vol.Â <div class="volume">104</div>Â (pg.Â <div class="fpage">2779</div>-<div class="lpage">84</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.0610797104" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.0610797104</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tissue-driven%20hypothesis%20of%20genomic%20evolution%20and%20sequence-expression%20correlations&amp;author=X.%20Gu&amp;author=Z.X.%20Su&amp;publication_year=2007&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=104&amp;pages=2779-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0610797104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0610797104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0610797104"> </span></div>Â </div></div></div></div></div><div content-id="DSS012C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Duret</div> <div class="given-names">L.</div></div>,Â  <div class="name"><div class="surname">Mouchiroud</div> <div class="given-names">D.</div></div>.Â </div><div class="article-title">Determinants of substitution rates in mammalian genes: expression pattern affects selection intensity but not mutation rate</div>,Â <div class="source">Mol. Biol. Evol.</div>Â ,Â <div class="year">2000</div>,Â vol.Â <div class="volume">17</div>Â (pg.Â <div class="fpage">68</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Determinants%20of%20substitution%20rates%20in%20mammalian%20genes%3A%20expression%20pattern%20affects%20selection%20intensity%20but%20not%20mutation%20rate&amp;author=L.%20Duret&amp;author=D.%20Mouchiroud&amp;publication_year=2000&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=17&amp;pages=68-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026239" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a026239" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a026239"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10666707" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS012C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thompson</div> <div class="given-names">J.D.</div></div>,Â  <div class="name"><div class="surname">Higgins</div> <div class="given-names">D.G.</div></div>,Â  <div class="name"><div class="surname">Gibson</div> <div class="given-names">T.J.</div></div>.Â </div><div class="article-title">CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice</div>,Â <div class="source">Nucleic Acids Res.</div>Â ,Â <div class="year">1994</div>,Â vol.Â <div class="volume">22</div>Â (pg.Â <div class="fpage">4673</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CLUSTAL%20W%3A%20improving%20the%20sensitivity%20of%20progressive%20multiple%20sequence%20alignment%20through%20sequence%20weighting%2C%20position-specific%20gap%20penalties%20and%20weight%20matrix%20choice&amp;author=J.D.%20Thompson&amp;author=D.G.%20Higgins&amp;author=T.J.%20Gibson&amp;publication_year=1994&amp;journal=Nucleic%20Acids%20Res.&amp;volume=22&amp;pages=4673-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F22.22.4673" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F22.22.4673"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7984417" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS012C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div>,Â  <div class="name"><div class="surname">Nielsen</div> <div class="given-names">R.</div></div>.Â </div><div class="article-title">Estimating synonymous and nonsynonymous substitution rates under realistic evolutionary models</div>,Â <div class="source">Mol. Biol. Evol.</div>Â ,Â <div class="year">2000</div>,Â vol.Â <div class="volume">17</div>Â (pg.Â <div class="fpage">32</div>-<div class="lpage">43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimating%20synonymous%20and%20nonsynonymous%20substitution%20rates%20under%20realistic%20evolutionary%20models&amp;author=Z.%20Yang&amp;author=R.%20Nielsen&amp;publication_year=2000&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=17&amp;pages=32-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026236" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a026236" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a026236"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10666704" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS012C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div>.Â </div><div class="article-title">PAML: a program package for phylogenetic analysis by maximum likelihood</div>,Â <div class="source">Comput. Appl. Biosci.</div>Â ,Â <div class="year">1997</div>,Â vol.Â <div class="volume">13</div>Â (pg.Â <div class="fpage">555</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PAML%3A%20a%20program%20package%20for%20phylogenetic%20analysis%20by%20maximum%20likelihood&amp;author=Z.%20Yang&amp;publication_year=1997&amp;journal=Comput.%20Appl.%20Biosci.&amp;volume=13&amp;pages=555-6" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9367129" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS012C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Necsulea</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Semon</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Duret</div> <div class="given-names">L.</div></div>,Â  <div class="name"><div class="surname">Hurst</div> <div class="given-names">L.D.</div></div>.Â </div><div class="article-title">Monoallelic expression and tissue specificity are associated with high crossover rates</div>,Â <div class="source">Trends Genet.</div>Â ,Â <div class="year">2009</div>,Â vol.Â <div class="volume">25</div>Â (pg.Â <div class="fpage">519</div>-<div class="lpage">22</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.tig.2009.10.001" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.tig.2009.10.001</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Monoallelic%20expression%20and%20tissue%20specificity%20are%20associated%20with%20high%20crossover%20rates&amp;author=A.%20Necsulea&amp;author=M.%20Semon&amp;author=L.%20Duret&amp;author=L.D.%20Hurst&amp;publication_year=2009&amp;journal=Trends%20Genet.&amp;volume=25&amp;pages=519-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2009.10.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2009.10.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2009.10.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19850368" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS012C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kuma</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Iwabe</div> <div class="given-names">N.</div></div>,Â  <div class="name"><div class="surname">Miyata</div> <div class="given-names">T.</div></div>.Â </div><div class="article-title">Functional constraints against variations on molecules from the tissue level: slowly evolving brain-specific genes demonstrated by protein kinase and immunoglobulin supergene families</div>,Â <div class="source">Mol. Biol. Evol.</div>Â ,Â <div class="year">1995</div>,Â vol.Â <div class="volume">12</div>Â (pg.Â <div class="fpage">123</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20constraints%20against%20variations%20on%20molecules%20from%20the%20tissue%20level%3A%20slowly%20evolving%20brain-specific%20genes%20demonstrated%20by%20protein%20kinase%20and%20immunoglobulin%20supergene%20families&amp;author=K.%20Kuma&amp;author=N.%20Iwabe&amp;author=T.%20Miyata&amp;publication_year=1995&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=12&amp;pages=123-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a040181" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a040181" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a040181"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7877487" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS012C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Winter</div> <div class="given-names">E.E.</div></div>,Â  <div class="name"><div class="surname">Goodstadt</div> <div class="given-names">L.</div></div>,Â  <div class="name"><div class="surname">Ponting</div> <div class="given-names">C.P.</div></div>.Â </div><div class="article-title">Elevated rates of protein secretion, evolution, and disease among tissue-specific genes</div>,Â <div class="source">Genome Res.</div>Â ,Â <div class="year">2004</div>,Â vol.Â <div class="volume">14</div>Â (pg.Â <div class="fpage">54</div>-<div class="lpage">61</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.1924004" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.1924004</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Elevated%20rates%20of%20protein%20secretion%2C%20evolution%2C%20and%20disease%20among%20tissue-specific%20genes&amp;author=E.E.%20Winter&amp;author=L.%20Goodstadt&amp;author=C.P.%20Ponting&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=54-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1924004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1924004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1924004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14707169" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS012C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hastings</div> <div class="given-names">K.E.</div></div>.Â </div><div class="article-title">Strong evolutionary conservation of broadly expressed protein isoforms in the troponin I gene family and other vertebrate gene families</div>,Â <div class="source">J. Mol. Evol.</div>Â ,Â <div class="year">1996</div>,Â vol.Â <div class="volume">42</div>Â (pg.Â <div class="fpage">631</div>-<div class="lpage">40</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/BF02338796" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/BF02338796</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strong%20evolutionary%20conservation%20of%20broadly%20expressed%20protein%20isoforms%20in%20the%20troponin%20I%20gene%20family%20and%20other%20vertebrate%20gene%20families&amp;author=K.E.%20Hastings&amp;publication_year=1996&amp;journal=J.%20Mol.%20Evol.&amp;volume=42&amp;pages=631-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02338796" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02338796" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02338796"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8662015" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS012C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS012C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huminiecki</div> <div class="given-names">L.</div></div>,Â  <div class="name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div>.Â </div><div class="article-title">Divergence of spatial gene expression profiles following species-specific gene duplications in human and mouse</div>,Â <div class="source">Genome Res.</div>Â ,Â <div class="year">2004</div>,Â vol.Â <div class="volume">14</div>Â (pg.Â <div class="fpage">1870</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.2705204" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.2705204</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Divergence%20of%20spatial%20gene%20expression%20profiles%20following%20species-specific%20gene%20duplications%20in%20human%20and%20mouse&amp;author=L.%20Huminiecki&amp;author=K.H.%20Wolfe&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=1870-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2705204" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2705204" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2705204"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15466287" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">Â© The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=450433&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/dss012-dss012supp_table4.xls?Expires=2147483647&amp;Signature=iVx0wFArnCUhq7jnloeskMI7dAmJBXnR42D4Cm9cky-wVnQNJJ8xrBNtx81Vf-pGzWAHU1oRnYAorV3GhKIL57kvcgY83h45oXtKvCGiXjcd-IOdeAxqMTipM~6OFOPvjEeHLepDw0QQOWHCxL-IWiHfXGKv3wsXrSz4UyR1NPnOmH4nMR62OFq1qbdbJndQWMlzkY9mncQtQxJTEGVDv5JmONNwVmbwEIIexg~kCAW3cL8ZSH3xHkYUNCWvoMJk~tKuD~fjEjsoWrJZSGX3rL6WZN58LEzl4B~pmnvqIStdShTRQ7MU7GGxK5HTyDKqPuRsDX-4t61Qj~sfA0AimQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/dss012-dss012supp_table3.xls?Expires=2147483647&amp;Signature=3YPUjgJEZkLWD57UirsVq1EPfOxW~jd9sMsY9hOEXEJULK-Kg2QV7wuyxviNSdYWK0QS3kzKzqnADh1qMhKiNTFbUECP9VbGK18kasyTQrop9WEGXLi-rmjz3jit4B6cwhOGHc5UnH~suUO9snv~oJKtST0VLRNcNaoQjMSNuEagF5Mpg8vuQu-YPZxVEDbsh7vnZWUb595eJ9H46zJ2VAljhT2mxAzE4s07KvNZquMdUPcX2y0q4yVLwA~DdroNA0xuBIG~gUa4LOngXNsTg9KwmRsIsqam33~PMXzqP0AXO9yeGDKePCZWyV-GsKIAy~k8A3xFBDGpePLZ-MYrLg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/dss012-dss012supp_table2.xls?Expires=2147483647&amp;Signature=5F6~Fh8WFx~fpEHFEN9j9asnQ97piezeZaXbTY6sFN85tEpPlq3XWQ4pQ2o9qNPE0wHbqISxcuCLr6RloZiHcXDb7YEJzj5A-nBGV2sFuoVypem~sVX7olhouhMmVvWL7vX-GCWOU80zb29U~EG4DK5ErlfVnW-2qsj9fTORAybAY7A2XdjGDhaE27s2FjIS6MrHFQYfXJKPJKyJ91tTXlD5SPRa9IpCFHf5ovP0NH~mJCkw5Y4U9HlaSiZDXUN1L1shBKTIcM8qQ8S60hErKWZ2yqoQRA2HP762CdVFp4b7TO8dt3AhgH4-BZ--S53tVgVv5ewkoGfu7Uf7m1Po8Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/dss012-dss012supp_table1.xls?Expires=2147483647&amp;Signature=pQvBkgeJ6M4DmIlScvkqqu~Cl7qWdpk~~gXaHj6RTDagHhpGRyjKZSawa4aE2KSJUtkZQ491nxWgQmdrvzzJsR1yCneMCEu0BBDBj0YIfaKPP9EpfpM2wVDxYqBkQlfoAc72GnFdBWXcOxQBnaidhVAX7NGct2iTs94cwdbZ8olYBAMHnLk1FmBtmoTtc~8AELX4RiqR22JdCt3lBWsMJyqygDzbA7uP0NKONv0R7MAFvE8o6EKhkYSLjrkV815Fh2beQsgMDqiG5PFB6nWoRIHURtgLCAX3zVU-N71KeUK5n3ykFW7ivbufOU922FyYKtBkagQv2olpMtGF0pdcSg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss012/2/dss012-dss012supp.doc?Expires=2147483647&amp;Signature=Vdtf4sSyAQ0zalVfbAyla2pKrrvaj6evPt5WUDSf2u8V2Xm21zJggLA7x2eAY7kzm8qsPfaof688-XZ1qnAAgeryImrhe2Ja2efvvUVvJzJi4Oej0TnN6KS-mHuoDAhxOWpNReI8GLxmK07MaIwR~0Mbty5qVDq-kMAmlM~-iiXV9Cw-5dF2VFZ3N9cTYpFV9XtRhP9prLKCSia--qV~nnfzpRmXeVTjSldjTb0wFIpW~eyvc~kKw1YsZUzjePk6XznsO5wVtK38s7Dpp32UEZDzMiJ1nTgNFyFxxO7YMMTcvXPpo2-QVrn-hxy2vh~wFaFJSc2eBnOsKXT4lc55rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">298</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000308385400001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">7</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dss012"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="450433"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000308385400001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f19%2f4%2f305%2f450433">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000308385400001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (7)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F19%2F4%2F305%2F450433" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/450433'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transpositionâ€“duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Evolution of Vertebrate Tissues Driven by Differential Modes of Gene Duplication" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dss012");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f19%2f4%2f305%2f450433">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

