

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":0,"applicationTime":1649,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/2/217/404545","@type":"ScholarlyArticle","name":"Design and Experimental Application of a Novel Non-Degenerate Universal Primer Set that Amplifies Prokaryotic 16S rRNA Genes with a Low Possibility to Amplify Eukaryotic rRNA Genes","datePublished":"2013-11-25","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2014-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst052","keywords":["16S rRNA","primer design","non-degenerate primer","microbial community"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/2/217/404545","author":[{"name":"Mori, Hiroshi","affiliation":"1  Department of Biological Information, Graduate School of Bioscience and Biotechnology ,  Tokyo Institute of Technology ,  4259 B-36, Nagatsuta-cho, Midori-ku, Yokohama 226-8501, Japan    2  Earth-Life Science Institute, Tokyo Institute of Technology ,  2-12-1-E3-10 Ookayama, Meguro-ku, Tokyo 152-8550, Japan","@type":"Person"},{"name":"Maruyama, Fumito","affiliation":"3  Section of Bacterial Pathogenesis, Graduate School of Medical and Dental Sciences ,  Tokyo Medical and Dental University ,  1-5-45, Yushima, Bunkyo-Ku, Tokyo 113-8510, Japan","@type":"Person"},{"name":"Kato, Hiromi","affiliation":"4  Graduate School of Life Sciences ,  Tohoku University ,  2-1-1 Katahira, Sendai 980-8577, Japan","@type":"Person"},{"name":"Toyoda, Atsushi","affiliation":"5  Center for Information Biology ,  National Institute of Genetics ,  Shizuoka 411-8540, Japan","@type":"Person"},{"name":"Dozono, Ayumi","affiliation":"1  Department of Biological Information, Graduate School of Bioscience and Biotechnology ,  Tokyo Institute of Technology ,  4259 B-36, Nagatsuta-cho, Midori-ku, Yokohama 226-8501, Japan","@type":"Person"},{"name":"Ohtsubo, Yoshiyuki","affiliation":"4  Graduate School of Life Sciences ,  Tohoku University ,  2-1-1 Katahira, Sendai 980-8577, Japan","@type":"Person"},{"name":"Nagata, Yuji","affiliation":"4  Graduate School of Life Sciences ,  Tohoku University ,  2-1-1 Katahira, Sendai 980-8577, Japan","@type":"Person"},{"name":"Fujiyama, Asao","affiliation":"5  Center for Information Biology ,  National Institute of Genetics ,  Shizuoka 411-8540, Japan    6  Principles of Informatics Research Division ,  National Institute of Informatics ,  Hitotsubashi, Chiyoda-ku, Tokyo 101-8430 ,  Japan","@type":"Person"},{"name":"Tsuda, Masataka","affiliation":"4  Graduate School of Life Sciences ,  Tohoku University ,  2-1-1 Katahira, Sendai 980-8577, Japan","@type":"Person"},{"name":"Kurokawa, Ken","affiliation":"2  Earth-Life Science Institute, Tokyo Institute of Technology ,  2-12-1-E3-10 Ookayama, Meguro-ku, Tokyo 152-8550, Japan","@type":"Person"}],"description":"Abstract.  The deep sequencing of 16S rRNA genes amplified by universal primers has revolutionized our understanding of microbial communities by allowing the ch","pageStart":"217","pageEnd":"227","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/2/1/m_cover.gif?Expires=2147483647&Signature=C2ZH5dkUQEctPPta7W831-4mGKGGgEq~Aqbfql5zal9lq-CH7ZJcQtnaLb-2~TFW5o1XuWK-If8uoA0C0x3E8Xqh7vx6q7XeJXntpLTv0xMKSiGPs8b5JT8WwzNVzF-MVRjRIurpCdYeMF6bhbaWGNZLajJ34JAWHyXvGj72zxR2I~0JkQXfyp1Fd2IS9rmTmm3EZ-Vtcf0zL2z0kWasBqYnVob3EwZTj2l448hvYoYYZgYfsgU2j1vdpw3GUsXrCjCI5GsZ1ht-EZrYHZzJ3zkCMOPIc~J0lFrmNiix6fEEbm~y5NNGWoTY3GIiDMTmJpXRxBN715elzocAQQfDUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Design and Experimental Application of a Novel Non-Degenerate Universal Primer Set that Amplifies Prokaryotic 16S rRNA Genes with a Low Possibility to Amplify Eukaryotic rRNA Genes","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/2/1/m_cover.gif?Expires=2147483647&Signature=C2ZH5dkUQEctPPta7W831-4mGKGGgEq~Aqbfql5zal9lq-CH7ZJcQtnaLb-2~TFW5o1XuWK-If8uoA0C0x3E8Xqh7vx6q7XeJXntpLTv0xMKSiGPs8b5JT8WwzNVzF-MVRjRIurpCdYeMF6bhbaWGNZLajJ34JAWHyXvGj72zxR2I~0JkQXfyp1Fd2IS9rmTmm3EZ-Vtcf0zL2z0kWasBqYnVob3EwZTj2l448hvYoYYZgYfsgU2j1vdpw3GUsXrCjCI5GsZ1ht-EZrYHZzJ3zkCMOPIc~J0lFrmNiix6fEEbm~y5NNGWoTY3GIiDMTmJpXRxBN715elzocAQQfDUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Design and Experimental Application of a Novel Non-Degenerate Universal Primer Set that Amplifies Prokaryotic 16S rRNA Genes with a Low Possibility to Amplify Eukaryotic rRNA Genes" />
    <meta property="og:description" content="Abstract.  The deep sequencing of 16S rRNA genes amplified by universal primers has revolutionized our understanding of microbial communities by allowing the ch" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/2/217/404545" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-11-25" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Mori, Hiroshi" /><meta name="citation_author_institution" content="Department of Biological Information, Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology, 4259 B-36, Nagatsuta-cho, Midori-ku, Yokohama 226-8501, Japan" /><meta name="citation_author_institution" content="Earth-Life Science Institute, Tokyo Institute of Technology, 2-12-1-E3-10 Ookayama, Meguro-ku, Tokyo 152-8550, Japan" /><meta name="citation_author" content="Maruyama, Fumito" /><meta name="citation_author_institution" content="Section of Bacterial Pathogenesis, Graduate School of Medical and Dental Sciences, Tokyo Medical and Dental University, 1-5-45, Yushima, Bunkyo-Ku, Tokyo 113-8510, Japan" /><meta name="citation_author" content="Kato, Hiromi" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan" /><meta name="citation_author" content="Toyoda, Atsushi" /><meta name="citation_author_institution" content="Center for Information Biology, National Institute of Genetics, Shizuoka 411-8540, Japan" /><meta name="citation_author" content="Dozono, Ayumi" /><meta name="citation_author_institution" content="Department of Biological Information, Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology, 4259 B-36, Nagatsuta-cho, Midori-ku, Yokohama 226-8501, Japan" /><meta name="citation_author" content="Ohtsubo, Yoshiyuki" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan" /><meta name="citation_author" content="Nagata, Yuji" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan" /><meta name="citation_author" content="Fujiyama, Asao" /><meta name="citation_author_institution" content="Center for Information Biology, National Institute of Genetics, Shizuoka 411-8540, Japan" /><meta name="citation_author_institution" content="Principles of Informatics Research Division, National Institute of Informatics, Hitotsubashi, Chiyoda-ku, Tokyo 101-8430, Japan" /><meta name="citation_author" content="Tsuda, Masataka" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan" /><meta name="citation_author" content="Kurokawa, Ken" /><meta name="citation_author_institution" content="Earth-Life Science Institute, Tokyo Institute of Technology, 2-12-1-E3-10 Ookayama, Meguro-ku, Tokyo 152-8550, Japan" /><meta name="citation_title" content="Design and Experimental Application of a Novel Non-Degenerate Universal Primer Set that Amplifies Prokaryotic 16S rRNA Genes with a Low Possibility to Amplify Eukaryotic rRNA Genes" /><meta name="citation_firstpage" content="217" /><meta name="citation_lastpage" content="227" /><meta name="citation_doi" content="10.1093/dnares/dst052" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2014/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Microbial community profiling for human microbiome projects: tools, techniques, and challenges,; Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=1141-52; " /><meta name="citation_reference" content="citation_title=The uncultured microbial majority; Annu. Rev. Microbiol.;  citation_year=2003;  citation_volume=57;  citation_pages=369-94; " /><meta name="citation_reference" content="citation_title=Error-correcting barcoded primers for pyrosequencing hundreds of samples in multiplex; Nat. Methods;  citation_year=2008;  citation_volume=5;  citation_pages=235-7; " /><meta name="citation_reference" content="citation_title=VITCOMIC: visualization tool for taxonomic compositions of microbial communities based on 16S rRNA gene sequences; BMC Bioinformatics;  citation_year=2010;  citation_volume=11;  citation_pages=332" /><meta name="citation_reference" content="citation_title=A core gut microbiome in obese and lean twins; Nature;  citation_year=2009;  citation_volume=457;  citation_pages=480-4; " /><meta name="citation_reference" content="citation_title=Genome sequencing in microfabricated high-density picolitre reactors; Nature;  citation_year=2005;  citation_volume=437;  citation_pages=376-80; " /><meta name="citation_reference" content="citation_title=Experimental and analytical tools for studying the human microbiome; Nat. Rev. Genet.;  citation_year=2012;  citation_volume=13;  citation_pages=47-58; " /><meta name="citation_reference" content="citation_title=Short pyrosequencing reads suffice for accurate microbial community analysis; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=e120" /><meta name="citation_reference" content="citation_title=Conservative fragments in bacterial 16S rRNA genes and primer design for 16S ribosomal DNA amplicons in metagenomic studies; PLoS ONE;  citation_year=2009;  citation_volume=4;  citation_pages=e7401" /><meta name="citation_reference" content="citation_title=Review and re-analysis of domain-specific 16S primers; J. Microbiol. Methods;  citation_year=2003;  citation_volume=55;  citation_pages=541-55; " /><meta name="citation_reference" content="citation_title=Robustness of gut microbiota of healthy adults in response to probiotic intervention revealed by high-throughput pyrosequencing; DNA Res.;  citation_year=2013;  citation_volume=20;  citation_pages=241-53; " /><meta name="citation_reference" content="citation_title=Exploring microbial diversity and taxonomy using SSU rRNA hypervariable tag sequencing; PLoS Genet.;  citation_year=2008;  citation_volume=4;  citation_pages=e1000255" /><meta name="citation_reference" content="citation_title=Global ecological patterns in uncultured Archaea; ISME J.;  citation_year=2010;  citation_volume=4;  citation_pages=182-90; " /><meta name="citation_reference" content="citation_title=Archaeal and bacterial communities respond differently to environmental gradients in anoxic sediments of a California hypersaline lake, the Salton Sea; Appl. Environ. Microbiol.;  citation_year=2010;  citation_volume=76;  citation_pages=757-68; " /><meta name="citation_reference" content="citation_title=Examining the global distribution of dominant archaeal populations in soil; ISME J.;  citation_year=2011;  citation_volume=5;  citation_pages=908-17; " /><meta name="citation_reference" content="citation_title=PrimerProspector: de novo design and taxonomic analysis of barcoded polymerase chain reaction primers; Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=1159-61; " /><meta name="citation_reference" content="citation_title=Evaluation of general 16S ribosomal RNA gene PCR primers for classical and next-generation sequencing-based diversity studies; Nucleic Acids Res.;  citation_year=2012;  citation_volume=41;  citation_pages=e1" /><meta name="citation_reference" content="citation_title=Coamplification of eukaryotic DNA with 16S rRNA gene-based PCR primers: possible consequences for population fingerprinting of complex microbial communities; Curr. Microbiol.;  citation_year=2008;  citation_volume=56;  citation_pages=553-7; " /><meta name="citation_reference" content="citation_title=Errors in ribosomal sequence datasets generated using PCR-coupled “panbacterial’ pyrosequencing, and the establishment of an improved approach; Mol. Cell Probes;  citation_year=2013;  citation_volume=27;  citation_pages=65-7; " /><meta name="citation_reference" content="citation_title=Cross-kingdom amplification using bacteria-specific primers: complications for studies of coral microbial ecology; Appl. Environ. Microbiol.;  citation_year=2008;  citation_volume=74;  citation_pages=7828-31; " /><meta name="citation_reference" content="citation_title=A framework for human microbiome research; citation_author=The Human Microbiome Project Consortium; Nature;  citation_year=2012;  citation_volume=486;  citation_pages=215-21; " /><meta name="citation_reference" content="citation_title=Target region selection is a critical determinant of community fingerprints generated by 16S pyrosequencing; PLoS ONE;  citation_year=2011;  citation_volume=6;  citation_pages=e20956" /><meta name="citation_reference" content="citation_title=Novel subgingival bacterial phylotypes detected using multiple universal polymerase chain reaction primer sets; Oral Microbiol. Immunol.;  citation_year=2006;  citation_volume=21;  citation_pages=61-8; " /><meta name="citation_reference" content="citation_title=Solexa Ltd; Pharmacogenomics;  citation_year=2004;  citation_volume=5;  citation_pages=433-8; " /><meta name="citation_reference" content="citation_title=RNAmmer: consistent and rapid annotation of ribosomal RNA genes; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=3100-8; " /><meta name="citation_reference" content="citation_title=The All-Species Living Tree project: a 16S rRNA-based phylogenetic tree of all sequenced type strains; Syst. Appl. Microbiol.;  citation_year=2008;  citation_volume=31;  citation_pages=241-50; " /><meta name="citation_reference" content="citation_title=Divergence and redundancy of 16S rRNA sequences in genomes with multiple rrn operons; J. Bacteriol.;  citation_year=2004;  citation_volume=186;  citation_pages=2629-35; " /><meta name="citation_reference" content="citation_title=Improved accuracy of multiple ncRNA alignment by incorporating structural information into a MAFFT-based framework; BMC Bioinformatics;  citation_year=2008;  citation_volume=9;  citation_pages=212" /><meta name="citation_reference" content="citation_title=The Ribosomal Database Project: improved alignments and new tools for rRNA analysis; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=D141-5; " /><meta name="citation_reference" content="citation_title=The comparative RNA web (CRW) site: an online database of comparative sequence and structure information for ribosomal, intron, and other RNAs; BMC Bioinformatics;  citation_year=2002;  citation_volume=3;  citation_pages=2" /><meta name="citation_reference" content="citation_title=Comparison of two next-generation sequencing technologies for resolving highly complex microbiota composition using tandem variable 16S rRNA gene regions; Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=e200" /><meta name="citation_reference" content="citation_title=Detection of bphAa gene expression of Rhodococcus sp. strain RHA1 in soil using a new method of RNA preparation from soil; Biosci. Biotechnol. Biochem.;  citation_year=2008;  citation_volume=72;  citation_pages=694-701; " /><meta name="citation_reference" content="citation_title=Introducing mothur: open-source, platform-independent, community-supported software for describing and comparing microbial communities; Appl. Environ. Microbiol.;  citation_year=2009;  citation_volume=75;  citation_pages=7537-41; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool; J. Mol. Biol.;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=Occurrence of introns in the 16S rRNA genes of members of the genus Thermoproteus; Arch. Microbiol.;  citation_year=1998;  citation_volume=170;  citation_pages=155-61; " /><meta name="citation_reference" content="citation_title=Coverage evaluation of universal bacterial primers using the metagenomic datasets; BMC Microbiol.;  citation_year=2012;  citation_volume=12;  citation_pages=66" /><meta name="citation_reference" content="citation_title=Critical evaluation of two primers commonly used for amplification of bacterial 16S rRNA genes; Appl. Environ. Microbiol.;  citation_year=2008;  citation_volume=74;  citation_pages=2461-70; " /><meta name="citation_reference" content="citation_title=Sequence logos: a new way to display consensus sequences; Nucleic Acids Res.;  citation_year=1990;  citation_volume=18;  citation_pages=6097-100; " /><meta name="citation_reference" content="citation_title=Analysis of stomach and gut microbiomes of the eastern oyster (Crassostrea virginica) from coastal Louisiana; PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e51475" /><meta name="citation_reference" content="citation_title=The origin and evolution of Archaea: a state of the art; Philos. Trans. R. Soc. Lond. B Biol. Sci.;  citation_year=2006;  citation_volume=361;  citation_pages=1007-22; " /><meta name="citation_reference" content="citation_title=Effects of experimental choices and analysis noise on surveys of the “rare biosphere”; Appl. Environ. Microbiol.;  citation_year=2009;  citation_volume=75;  citation_pages=3263-70; " /><meta name="citation_reference" content="citation_title=Genomic and functional adaptation in surface ocean planktonic prokaryotes; Nature;  citation_year=2010;  citation_volume=468;  citation_pages=60-6; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/2/217/1130388/dst052.pdf" /><meta name="description" content="Abstract.  The deep sequencing of 16S rRNA genes amplified by universal primers has revolutionized our understanding of microbial communities by allowing the ch" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/2/217/404545" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/2/217/404545" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/2/217/404545");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst052';
    </script>
<title>
	Design and Experimental Application of a Novel Non-Degenerate Universal Primer Set that Amplifies Prokaryotic 16S rRNA Genes with a Low Possibility to Amplify Eukaryotic rRNA Genes | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./404545" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BaxrPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNb3JpLCBIaXJvc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9sb2dpY2FsIEluZm9ybWF0aW9uLCBHcmFkdWF0ZSBTY2hvb2wgb2YgQmlvc2NpZW5jZSBhbmQgQmlvdGVjaG5vbG9neSwgVG9reW8gSW5zdGl0dXRlIG9mIFRlY2hub2xvZ3ksIDQyNTkgQi0zNiwgTmFnYXRzdXRhLWNobywgTWlkb3JpLWt1LCBZb2tvaGFtYSAyMjYtODUwMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJFYXJ0aC1MaWZlIFNjaWVuY2UgSW5zdGl0dXRlLCBUb2t5byBJbnN0aXR1dGUgb2YgVGVjaG5vbG9neSwgMi0xMi0xLUUzLTEwIE9va2F5YW1hLCBNZWd1cm8ta3UsIFRva3lvIDE1Mi04NTUwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1hcnV5YW1hLCBGdW1pdG8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTZWN0aW9uIG9mIEJhY3RlcmlhbCBQYXRob2dlbmVzaXMsIEdyYWR1YXRlIFNjaG9vbCBvZiBNZWRpY2FsIGFuZCBEZW50YWwgU2NpZW5jZXMsIFRva3lvIE1lZGljYWwgYW5kIERlbnRhbCBVbml2ZXJzaXR5LCAxLTUtNDUsIFl1c2hpbWEsIEJ1bmt5by1LdSwgVG9reW8gMTEzLTg1MTAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2F0bywgSGlyb21pIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIExpZmUgU2NpZW5jZXMsIFRvaG9rdSBVbml2ZXJzaXR5LCAyLTEtMSBLYXRhaGlyYSwgU2VuZGFpIDk4MC04NTc3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRveW9kYSwgQXRzdXNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNlbnRlciBmb3IgSW5mb3JtYXRpb24gQmlvbG9neSwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEdlbmV0aWNzLCBTaGl6dW9rYSA0MTEtODU0MCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJEb3pvbm8sIEF5dW1pIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9sb2dpY2FsIEluZm9ybWF0aW9uLCBHcmFkdWF0ZSBTY2hvb2wgb2YgQmlvc2NpZW5jZSBhbmQgQmlvdGVjaG5vbG9neSwgVG9reW8gSW5zdGl0dXRlIG9mIFRlY2hub2xvZ3ksIDQyNTkgQi0zNiwgTmFnYXRzdXRhLWNobywgTWlkb3JpLWt1LCBZb2tvaGFtYSAyMjYtODUwMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPaHRzdWJvLCBZb3NoaXl1a2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgTGlmZSBTY2llbmNlcywgVG9ob2t1IFVuaXZlcnNpdHksIDItMS0xIEthdGFoaXJhLCBTZW5kYWkgOTgwLTg1NzcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFnYXRhLCBZdWppIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIExpZmUgU2NpZW5jZXMsIFRvaG9rdSBVbml2ZXJzaXR5LCAyLTEtMSBLYXRhaGlyYSwgU2VuZGFpIDk4MC04NTc3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZ1aml5YW1hLCBBc2FvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudGVyIGZvciBJbmZvcm1hdGlvbiBCaW9sb2d5LCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgR2VuZXRpY3MsIFNoaXp1b2thIDQxMS04NTQwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlByaW5jaXBsZXMgb2YgSW5mb3JtYXRpY3MgUmVzZWFyY2ggRGl2aXNpb24sIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBJbmZvcm1hdGljcywgSGl0b3RzdWJhc2hpLCBDaGl5b2RhLWt1LCBUb2t5byAxMDEtODQzMCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUc3VkYSwgTWFzYXRha2EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgTGlmZSBTY2llbmNlcywgVG9ob2t1IFVuaXZlcnNpdHksIDItMS0xIEthdGFoaXJhLCBTZW5kYWkgOTgwLTg1NzcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS3Vyb2thd2EsIEtlbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkVhcnRoLUxpZmUgU2NpZW5jZSBJbnN0aXR1dGUsIFRva3lvIEluc3RpdHV0ZSBvZiBUZWNobm9sb2d5LCAyLTEyLTEtRTMtMTAgT29rYXlhbWEsIE1lZ3Vyby1rdSwgVG9reW8gMTUyLTg1NTAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJEZXNpZ24gYW5kIEV4cGVyaW1lbnRhbCBBcHBsaWNhdGlvbiBvZiBhIE5vdmVsIE5vbi1EZWdlbmVyYXRlIFVuaXZlcnNhbCBQcmltZXIgU2V0IHRoYXQgQW1wbGlmaWVzIFByb2thcnlvdGljIDE2UyByUk5BIEdlbmVzIHdpdGggYSBMb3cgUG9zc2liaWxpdHkgdG8gQW1wbGlmeSBFdWthcnlvdGljIHJSTkEgR2VuZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyMTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjIyNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdDA1MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE0LzA0LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9iaWFsIGNvbW11bml0eSBwcm9maWxpbmcgZm9yIGh1bWFuIG1pY3JvYmlvbWUgcHJvamVjdHM6IHRvb2xzLCB0ZWNobmlxdWVzLCBhbmQgY2hhbGxlbmdlcyw7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MTE0MS01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHVuY3VsdHVyZWQgbWljcm9iaWFsIG1ham9yaXR5OyBBbm51LiBSZXYuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NTc7ICBjaXRhdGlvbl9wYWdlcz0zNjktOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVycm9yLWNvcnJlY3RpbmcgYmFyY29kZWQgcHJpbWVycyBmb3IgcHlyb3NlcXVlbmNpbmcgaHVuZHJlZHMgb2Ygc2FtcGxlcyBpbiBtdWx0aXBsZXg7IE5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MjM1LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZJVENPTUlDOiB2aXN1YWxpemF0aW9uIHRvb2wgZm9yIHRheG9ub21pYyBjb21wb3NpdGlvbnMgb2YgbWljcm9iaWFsIGNvbW11bml0aWVzIGJhc2VkIG9uIDE2UyByUk5BIGdlbmUgc2VxdWVuY2VzOyBCTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0zMzIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvcmUgZ3V0IG1pY3JvYmlvbWUgaW4gb2Jlc2UgYW5kIGxlYW4gdHdpbnM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT00NTc7ICBjaXRhdGlvbl9wYWdlcz00ODAtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNpbmcgaW4gbWljcm9mYWJyaWNhdGVkIGhpZ2gtZGVuc2l0eSBwaWNvbGl0cmUgcmVhY3RvcnM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00Mzc7ICBjaXRhdGlvbl9wYWdlcz0zNzYtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cGVyaW1lbnRhbCBhbmQgYW5hbHl0aWNhbCB0b29scyBmb3Igc3R1ZHlpbmcgdGhlIGh1bWFuIG1pY3JvYmlvbWU7IE5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz00Ny01ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2hvcnQgcHlyb3NlcXVlbmNpbmcgcmVhZHMgc3VmZmljZSBmb3IgYWNjdXJhdGUgbWljcm9iaWFsIGNvbW11bml0eSBhbmFseXNpczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9ZTEyMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnNlcnZhdGl2ZSBmcmFnbWVudHMgaW4gYmFjdGVyaWFsIDE2UyByUk5BIGdlbmVzIGFuZCBwcmltZXIgZGVzaWduIGZvciAxNlMgcmlib3NvbWFsIEROQSBhbXBsaWNvbnMgaW4gbWV0YWdlbm9taWMgc3R1ZGllczsgUExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPWU3NDAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmV2aWV3IGFuZCByZS1hbmFseXNpcyBvZiBkb21haW4tc3BlY2lmaWMgMTZTIHByaW1lcnM7IEouIE1pY3JvYmlvbC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT01NTsgIGNpdGF0aW9uX3BhZ2VzPTU0MS01NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Um9idXN0bmVzcyBvZiBndXQgbWljcm9iaW90YSBvZiBoZWFsdGh5IGFkdWx0cyBpbiByZXNwb25zZSB0byBwcm9iaW90aWMgaW50ZXJ2ZW50aW9uIHJldmVhbGVkIGJ5IGhpZ2gtdGhyb3VnaHB1dCBweXJvc2VxdWVuY2luZzsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0yNDEtNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cGxvcmluZyBtaWNyb2JpYWwgZGl2ZXJzaXR5IGFuZCB0YXhvbm9teSB1c2luZyBTU1UgclJOQSBoeXBlcnZhcmlhYmxlIHRhZyBzZXF1ZW5jaW5nOyBQTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDAyNTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HbG9iYWwgZWNvbG9naWNhbCBwYXR0ZXJucyBpbiB1bmN1bHR1cmVkIEFyY2hhZWE7IElTTUUgSi47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTE4Mi05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXJjaGFlYWwgYW5kIGJhY3RlcmlhbCBjb21tdW5pdGllcyByZXNwb25kIGRpZmZlcmVudGx5IHRvIGVudmlyb25tZW50YWwgZ3JhZGllbnRzIGluIGFub3hpYyBzZWRpbWVudHMgb2YgYSBDYWxpZm9ybmlhIGh5cGVyc2FsaW5lIGxha2UsIHRoZSBTYWx0b24gU2VhOyBBcHBsLiBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTc2OyAgY2l0YXRpb25fcGFnZXM9NzU3LTY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeGFtaW5pbmcgdGhlIGdsb2JhbCBkaXN0cmlidXRpb24gb2YgZG9taW5hbnQgYXJjaGFlYWwgcG9wdWxhdGlvbnMgaW4gc29pbDsgSVNNRSBKLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9OTA4LTE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmltZXJQcm9zcGVjdG9yOiBkZSBub3ZvIGRlc2lnbiBhbmQgdGF4b25vbWljIGFuYWx5c2lzIG9mIGJhcmNvZGVkIHBvbHltZXJhc2UgY2hhaW4gcmVhY3Rpb24gcHJpbWVyczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0xMTU5LTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmFsdWF0aW9uIG9mIGdlbmVyYWwgMTZTIHJpYm9zb21hbCBSTkEgZ2VuZSBQQ1IgcHJpbWVycyBmb3IgY2xhc3NpY2FsIGFuZCBuZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZy1iYXNlZCBkaXZlcnNpdHkgc3R1ZGllczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9ZTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2FtcGxpZmljYXRpb24gb2YgZXVrYXJ5b3RpYyBETkEgd2l0aCAxNlMgclJOQSBnZW5lLWJhc2VkIFBDUiBwcmltZXJzOiBwb3NzaWJsZSBjb25zZXF1ZW5jZXMgZm9yIHBvcHVsYXRpb24gZmluZ2VycHJpbnRpbmcgb2YgY29tcGxleCBtaWNyb2JpYWwgY29tbXVuaXRpZXM7IEN1cnIuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NTY7ICBjaXRhdGlvbl9wYWdlcz01NTMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXJyb3JzIGluIHJpYm9zb21hbCBzZXF1ZW5jZSBkYXRhc2V0cyBnZW5lcmF0ZWQgdXNpbmcgUENSLWNvdXBsZWQg4oCccGFuYmFjdGVyaWFs4oCZIHB5cm9zZXF1ZW5jaW5nLCBhbmQgdGhlIGVzdGFibGlzaG1lbnQgb2YgYW4gaW1wcm92ZWQgYXBwcm9hY2g7IE1vbC4gQ2VsbCBQcm9iZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz02NS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Dcm9zcy1raW5nZG9tIGFtcGxpZmljYXRpb24gdXNpbmcgYmFjdGVyaWEtc3BlY2lmaWMgcHJpbWVyczogY29tcGxpY2F0aW9ucyBmb3Igc3R1ZGllcyBvZiBjb3JhbCBtaWNyb2JpYWwgZWNvbG9neTsgQXBwbC4gRW52aXJvbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT03NDsgIGNpdGF0aW9uX3BhZ2VzPTc4MjgtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZnJhbWV3b3JrIGZvciBodW1hbiBtaWNyb2Jpb21lIHJlc2VhcmNoOyBjaXRhdGlvbl9hdXRob3I9VGhlIEh1bWFuIE1pY3JvYmlvbWUgUHJvamVjdCBDb25zb3J0aXVtOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDg2OyAgY2l0YXRpb25fcGFnZXM9MjE1LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYXJnZXQgcmVnaW9uIHNlbGVjdGlvbiBpcyBhIGNyaXRpY2FsIGRldGVybWluYW50IG9mIGNvbW11bml0eSBmaW5nZXJwcmludHMgZ2VuZXJhdGVkIGJ5IDE2UyBweXJvc2VxdWVuY2luZzsgUExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUyMDk1NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vdmVsIHN1YmdpbmdpdmFsIGJhY3RlcmlhbCBwaHlsb3R5cGVzIGRldGVjdGVkIHVzaW5nIG11bHRpcGxlIHVuaXZlcnNhbCBwb2x5bWVyYXNlIGNoYWluIHJlYWN0aW9uIHByaW1lciBzZXRzOyBPcmFsIE1pY3JvYmlvbC4gSW1tdW5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz02MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Tb2xleGEgTHRkOyBQaGFybWFjb2dlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz00MzMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Uk5BbW1lcjogY29uc2lzdGVudCBhbmQgcmFwaWQgYW5ub3RhdGlvbiBvZiByaWJvc29tYWwgUk5BIGdlbmVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz0zMTAwLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBBbGwtU3BlY2llcyBMaXZpbmcgVHJlZSBwcm9qZWN0OiBhIDE2UyByUk5BLWJhc2VkIHBoeWxvZ2VuZXRpYyB0cmVlIG9mIGFsbCBzZXF1ZW5jZWQgdHlwZSBzdHJhaW5zOyBTeXN0LiBBcHBsLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MjQxLTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXZlcmdlbmNlIGFuZCByZWR1bmRhbmN5IG9mIDE2UyByUk5BIHNlcXVlbmNlcyBpbiBnZW5vbWVzIHdpdGggbXVsdGlwbGUgcnJuIG9wZXJvbnM7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTg2OyAgY2l0YXRpb25fcGFnZXM9MjYyOS0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1wcm92ZWQgYWNjdXJhY3kgb2YgbXVsdGlwbGUgbmNSTkEgYWxpZ25tZW50IGJ5IGluY29ycG9yYXRpbmcgc3RydWN0dXJhbCBpbmZvcm1hdGlvbiBpbnRvIGEgTUFGRlQtYmFzZWQgZnJhbWV3b3JrOyBCTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTIxMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBSaWJvc29tYWwgRGF0YWJhc2UgUHJvamVjdDogaW1wcm92ZWQgYWxpZ25tZW50cyBhbmQgbmV3IHRvb2xzIGZvciByUk5BIGFuYWx5c2lzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz1EMTQxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb21wYXJhdGl2ZSBSTkEgd2ViIChDUlcpIHNpdGU6IGFuIG9ubGluZSBkYXRhYmFzZSBvZiBjb21wYXJhdGl2ZSBzZXF1ZW5jZSBhbmQgc3RydWN0dXJlIGluZm9ybWF0aW9uIGZvciByaWJvc29tYWwsIGludHJvbiwgYW5kIG90aGVyIFJOQXM7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmlzb24gb2YgdHdvIG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIHRlY2hub2xvZ2llcyBmb3IgcmVzb2x2aW5nIGhpZ2hseSBjb21wbGV4IG1pY3JvYmlvdGEgY29tcG9zaXRpb24gdXNpbmcgdGFuZGVtIHZhcmlhYmxlIDE2UyByUk5BIGdlbmUgcmVnaW9uczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9ZTIwMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldGVjdGlvbiBvZiBicGhBYSBnZW5lIGV4cHJlc3Npb24gb2YgUmhvZG9jb2NjdXMgc3AuIHN0cmFpbiBSSEExIGluIHNvaWwgdXNpbmcgYSBuZXcgbWV0aG9kIG9mIFJOQSBwcmVwYXJhdGlvbiBmcm9tIHNvaWw7IEJpb3NjaS4gQmlvdGVjaG5vbC4gQmlvY2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NzI7ICBjaXRhdGlvbl9wYWdlcz02OTQtNzAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRyb2R1Y2luZyBtb3RodXI6IG9wZW4tc291cmNlLCBwbGF0Zm9ybS1pbmRlcGVuZGVudCwgY29tbXVuaXR5LXN1cHBvcnRlZCBzb2Z0d2FyZSBmb3IgZGVzY3JpYmluZyBhbmQgY29tcGFyaW5nIG1pY3JvYmlhbCBjb21tdW5pdGllczsgQXBwbC4gRW52aXJvbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT03NTsgIGNpdGF0aW9uX3BhZ2VzPTc1MzctNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhc2ljIGxvY2FsIGFsaWdubWVudCBzZWFyY2ggdG9vbDsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTU7ICBjaXRhdGlvbl9wYWdlcz00MDMtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9jY3VycmVuY2Ugb2YgaW50cm9ucyBpbiB0aGUgMTZTIHJSTkEgZ2VuZXMgb2YgbWVtYmVycyBvZiB0aGUgZ2VudXMgVGhlcm1vcHJvdGV1czsgQXJjaC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNzA7ICBjaXRhdGlvbl9wYWdlcz0xNTUtNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvdmVyYWdlIGV2YWx1YXRpb24gb2YgdW5pdmVyc2FsIGJhY3RlcmlhbCBwcmltZXJzIHVzaW5nIHRoZSBtZXRhZ2Vub21pYyBkYXRhc2V0czsgQk1DIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz02NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNyaXRpY2FsIGV2YWx1YXRpb24gb2YgdHdvIHByaW1lcnMgY29tbW9ubHkgdXNlZCBmb3IgYW1wbGlmaWNhdGlvbiBvZiBiYWN0ZXJpYWwgMTZTIHJSTkEgZ2VuZXM7IEFwcGwuIEVudmlyb24uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NzQ7ICBjaXRhdGlvbl9wYWdlcz0yNDYxLTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jZSBsb2dvczogYSBuZXcgd2F5IHRvIGRpc3BsYXkgY29uc2Vuc3VzIHNlcXVlbmNlczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9NjA5Ny0xMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIHN0b21hY2ggYW5kIGd1dCBtaWNyb2Jpb21lcyBvZiB0aGUgZWFzdGVybiBveXN0ZXIgKENyYXNzb3N0cmVhIHZpcmdpbmljYSkgZnJvbSBjb2FzdGFsIExvdWlzaWFuYTsgUExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWU1MTQ3NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBvcmlnaW4gYW5kIGV2b2x1dGlvbiBvZiBBcmNoYWVhOiBhIHN0YXRlIG9mIHRoZSBhcnQ7IFBoaWxvcy4gVHJhbnMuIFIuIFNvYy4gTG9uZC4gQiBCaW9sLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM2MTsgIGNpdGF0aW9uX3BhZ2VzPTEwMDctMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmVjdHMgb2YgZXhwZXJpbWVudGFsIGNob2ljZXMgYW5kIGFuYWx5c2lzIG5vaXNlIG9uIHN1cnZleXMgb2YgdGhlIOKAnHJhcmUgYmlvc3BoZXJl4oCdOyBBcHBsLiBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTc1OyAgY2l0YXRpb25fcGFnZXM9MzI2My03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBhbmQgZnVuY3Rpb25hbCBhZGFwdGF0aW9uIGluIHN1cmZhY2Ugb2NlYW4gcGxhbmt0b25pYyBwcm9rYXJ5b3RlczsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ2ODsgIGNpdGF0aW9uX3BhZ2VzPTYwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMS8yLzIxNy8xMTMwMzg4L2RzdDA1Mi5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgVGhlIGRlZXAgc2VxdWVuY2luZyBvZiAxNlMgclJOQSBnZW5lcyBhbXBsaWZpZWQgYnkgdW5pdmVyc2FsIHByaW1lcnMgaGFzIHJldm9sdXRpb25pemVkIG91ciB1bmRlcnN0YW5kaW5nIG9mIG1pY3JvYmlhbCBjb21tdW5pdGllcyBieSBhbGxvd2luZyB0aGUgY2giIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQwNDU0NSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGT+i77dBX9gY4nep7WBvJ7shWwSWw==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f2%2f217%2f404545" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f2%2f217%2f404545" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=15712" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/2/1/m_cover.gif?Expires=2147483647&amp;Signature=C2ZH5dkUQEctPPta7W831-4mGKGGgEq~Aqbfql5zal9lq-CH7ZJcQtnaLb-2~TFW5o1XuWK-If8uoA0C0x3E8Xqh7vx6q7XeJXntpLTv0xMKSiGPs8b5JT8WwzNVzF-MVRjRIurpCdYeMF6bhbaWGNZLajJ34JAWHyXvGj72zxR2I~0JkQXfyp1Fd2IS9rmTmm3EZ-Vtcf0zL2z0kWasBqYnVob3EwZTj2l448hvYoYYZgYfsgU2j1vdpw3GUsXrCjCI5GsZ1ht-EZrYHZzJ3zkCMOPIc~J0lFrmNiix6fEEbm~y5NNGWoTY3GIiDMTmJpXRxBN715elzocAQQfDUQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/2">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5985422">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5985422">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5985424">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5985424">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5985428">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5985428">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5985445">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5985445">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5985464">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5985464">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5985466">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5985466">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5985468">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5985468">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5985470">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5985470">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/2/207/404404">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/21/2/229/2754533">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Design and Experimental Application of a Novel Non-Degenerate Universal Primer Set that Amplifies Prokaryotic 16S rRNA Genes with a Low Possibility to Amplify Eukaryotic rRNA Genes
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroshi Mori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biological Information, Graduate School of Bioscience and Biotechnology</div>, <div class="insititution">Tokyo Institute of Technology</div>, <div class="addr-line">4259 B-36, Nagatsuta-cho, Midori-ku, Yokohama 226-8501, Japan</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Earth-Life Science Institute, Tokyo Institute of Technology</div>, <div class="addr-line">2-12-1-E3-10 Ookayama, Meguro-ku, Tokyo 152-8550, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroshi+Mori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mori H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mori H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hiroshi Mori</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fumito Maruyama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Section of Bacterial Pathogenesis, Graduate School of Medical and Dental Sciences</div>, <div class="insititution">Tokyo Medical and Dental University</div>, <div class="addr-line">1-5-45, Yushima, Bunkyo-Ku, Tokyo 113-8510, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fumito+Maruyama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Maruyama F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Maruyama F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Fumito Maruyama</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiromi Kato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University</div>, <div class="addr-line">2-1-1 Katahira, Sendai 980-8577, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiromi+Kato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kato H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kato H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hiromi Kato</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Atsushi Toyoda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Center for Information Biology</div>, <div class="insititution">National Institute of Genetics</div>, <div class="addr-line">Shizuoka 411-8540, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Atsushi+Toyoda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Toyoda A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Toyoda A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Atsushi Toyoda</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ayumi Dozono</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biological Information, Graduate School of Bioscience and Biotechnology</div>, <div class="insititution">Tokyo Institute of Technology</div>, <div class="addr-line">4259 B-36, Nagatsuta-cho, Midori-ku, Yokohama 226-8501, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ayumi+Dozono">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dozono A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Dozono A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ayumi Dozono</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshiyuki Ohtsubo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University</div>, <div class="addr-line">2-1-1 Katahira, Sendai 980-8577, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshiyuki+Ohtsubo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohtsubo Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohtsubo Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshiyuki Ohtsubo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuji Nagata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University</div>, <div class="addr-line">2-1-1 Katahira, Sendai 980-8577, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuji+Nagata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nagata Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nagata Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yuji Nagata</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Asao Fujiyama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Center for Information Biology</div>, <div class="insititution">National Institute of Genetics</div>, <div class="addr-line">Shizuoka 411-8540, Japan</div></div><div class="aff"><div class="label -label">6</div><div class="addr-line">Principles of Informatics Research Division</div>, <div class="insititution">National Institute of Informatics</div>, <div class="addr-line">Hitotsubashi, Chiyoda-ku, Tokyo 101-8430</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Asao+Fujiyama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fujiyama A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fujiyama A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Asao Fujiyama</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masataka Tsuda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University</div>, <div class="addr-line">2-1-1 Katahira, Sendai 980-8577, Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 -45-924-5139. Fax. +81 -45-924-5139. Email: <a href="mailto:ken@bio.titech.ac.jp" target="_blank">ken@bio.titech.ac.jp</a> (K.K.); Tel. +81 -22-217-5699. Fax. +81 -22-217-5699. Email: <a href="mailto:mtsuda@ige.tohoku.ac.jp" target="_blank">mtsuda@ige.tohoku.ac.jp</a> (M.T.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masataka+Tsuda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tsuda M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tsuda M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masataka Tsuda</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ken Kurokawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Earth-Life Science Institute, Tokyo Institute of Technology</div>, <div class="addr-line">2-12-1-E3-10 Ookayama, Meguro-ku, Tokyo 152-8550, Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 -45-924-5139. Fax. +81 -45-924-5139. Email: <a href="mailto:ken@bio.titech.ac.jp" target="_blank">ken@bio.titech.ac.jp</a> (K.K.); Tel. +81 -22-217-5699. Fax. +81 -22-217-5699. Email: <a href="mailto:mtsuda@ige.tohoku.ac.jp" target="_blank">mtsuda@ige.tohoku.ac.jp</a> (M.T.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ken+Kurokawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kurokawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kurokawa K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ken Kurokawa</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 2, 1 April 2014, Pages 217–227, <a href='https://doi.org/10.1093/dnares/dst052'>https://doi.org/10.1093/dnares/dst052</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">25 November 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">14 June 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">28 October 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="404545" href="/dnaresearch/article-pdf/21/2/217/1130388/dst052.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Hiroshi Mori, Fumito Maruyama, Hiromi Kato, Atsushi Toyoda, Ayumi Dozono, Yoshiyuki Ohtsubo, Yuji Nagata, Asao Fujiyama, Masataka Tsuda, Ken Kurokawa;  Design and Experimental Application of a Novel Non-Degenerate Universal Primer Set that Amplifies Prokaryotic 16S rRNA Genes with a Low Possibility to Amplify Eukaryotic rRNA Genes, <em>DNA Research</em>, Volume 21, Issue 2, 1 April 2014, Pages 217–227, <a href='https://doi.org/10.1093/dnares/dst052'>https://doi.org/10.1093/dnares/dst052</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/404545?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/404545?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/404545?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/404545?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/404545?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/404545?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/404545?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Design%20and%20Experimental%20Application%20of%20a%20Novel%20Non-Degenerate%20Universal%20Primer%20Set%20that%20Amplifies%20Prokaryotic%2016S%20rRNA%20Genes%20with%20a%20Low%20Possibility%20to%20Amplify%20Eukaryotic%20rRNA%20Genes&amp;publicationDate=2013-11-25&amp;volumeNum=21&amp;issueNum=2&amp;author=Mori%2C%20Hiroshi%3B%20Maruyama%2C%20Fumito&amp;startPage=217&amp;endPage=227&amp;contentId=10.1093%2Fdnares%2Fdst052&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f2%2f217%2f404545" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5985422 id="5985422" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The deep sequencing of 16S rRNA genes amplified by universal primers has revolutionized our understanding of microbial communities by allowing the characterization of the diversity of the uncultured majority. However, some universal primers also amplify eukaryotic rRNA genes, leading to a decrease in the efficiency of sequencing of prokaryotic 16S rRNA genes with possible mischaracterization of the diversity in the microbial community. In this study, we compared 16S rRNA gene sequences from genome-sequenced strains and identified candidates for non-degenerate universal primers that could be used for the amplification of prokaryotic 16S rRNA genes. The 50 identified candidates were investigated to calculate their coverage for prokaryotic and eukaryotic rRNA genes, including those from uncultured taxa and eukaryotic organelles, and a novel universal primer set, 342F-806R, covering many prokaryotic, but not eukaryotic, rRNA genes was identified. This primer set was validated by the amplification of 16S rRNA genes from a soil metagenomic sample and subsequent pyrosequencing using the Roche 454 platform. The same sample was also used for pyrosequencing of the amplicons by employing a commonly used primer set, 338F-533R, and for shotgun metagenomic sequencing using the Illumina platform. Our comparison of the taxonomic compositions inferred by the three sequencing experiments indicated that the non-degenerate 342F-806R primer set can characterize the taxonomic composition of the microbial community without substantial bias, and is highly expected to be applicable to the analysis of a wide variety of microbial communities.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;16S rRNA&quot;">16S rRNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;primer design&quot;">primer design</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;non-degenerate primer&quot;">non-degenerate primer</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;microbial community&quot;">microbial community</a></div>                            <h2 scrollto-destination=5985424 id="5985424" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Culture-independent and 16S rRNA gene-based analysis is widely used to profile microbial communities.<sup><a reveal-id="DST052C1" data-open="DST052C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Such profiling of 16S rRNA genes has revolutionized our understanding of microbial communities by allowing the characterization of the diversity of the uncultured majority.<sup><a reveal-id="DST052C2" data-open="DST052C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Moreover, ongoing development of massively parallel DNA sequencing technologies allows deep sequencing of a sample and simultaneous sequencing of many samples per run via low-cost bar-coding techniques.<sup><a reveal-id="DST052C3" data-open="DST052C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Because of its advantages over the traditional Sanger sequencing, the massively parallel sequencing has been applied to large-scale sequencing projects, including the analysis of 16S rRNA gene amplicons from diverse microbial communities.<sup><a reveal-id="DST052C4" data-open="DST052C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DST052C5" data-open="DST052C5" class="link link-ref link-reveal xref-bibr">5</a></sup></p><p>The Roche 454 Genome Sequencer FLX system<sup><a reveal-id="DST052C6" data-open="DST052C6" class="link link-ref link-reveal xref-bibr">6</a></sup> is commonly used for the massively parallel sequencing of the 16S rRNA gene amplicons.<sup><a reveal-id="DST052C5" data-open="DST052C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Several studies have suggested that short (450–600 nt) reads obtained by the 454 platform<sup><a reveal-id="DST052C7" data-open="DST052C7" class="link link-ref link-reveal xref-bibr">7</a></sup> are sufficient for the taxonomic assignment and the subsequent community comparison when the regions in 16S rRNA genes to be sequenced are carefully chosen.<sup><a reveal-id="DST052C8" data-open="DST052C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Notably, appropriate primer selection is a critical issue for analysing diverse microbial communities.<sup><a reveal-id="DST052C9" data-open="DST052C9" class="link link-ref link-reveal xref-bibr">9</a></sup> The bias arising from poor complementarity between primers and 16S rRNA genes in particular taxa leads to underestimation of these taxa in a microbial community.<sup><a reveal-id="DST052C10" data-open="DST052C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DST052C11" data-open="DST052C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Although the primer sets specific to <em>Bacteria</em> (e.g. the 338F and 533R set, which PCR-amplifies and covers the V3 regions of 16S rRNA genes)<sup><a reveal-id="DST052C12" data-open="DST052C12" class="link link-ref link-reveal xref-bibr">12</a></sup> have generally been used for community profiling,<sup><a reveal-id="DST052C5" data-open="DST052C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DST052C12" data-open="DST052C12" class="link link-ref link-reveal xref-bibr">12</a></sup> recent studies of <em>Archaea</em> have revealed its ubiquitous distribution in diverse environments and its important roles in various ecosystems (e.g. ammonia oxidization in soil and seawater).<sup><a reveal-id="DST052C13" data-open="DST052C13" class="link link-ref link-reveal xref-bibr">13</a></sup> To investigate the relative composition of <em>Bacteria</em> and <em>Archaea</em> and their ecological interaction in a microbial community,<sup><a reveal-id="DST052C14" data-open="DST052C14" class="link link-ref link-reveal xref-bibr">14</a></sup> universal primer sets that can amplify both bacterial and archaeal 16S rRNA genes are needed.<sup><a reveal-id="DST052C15" data-open="DST052C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Although many sets of universal primers have been designed for this purpose,<sup><a reveal-id="DST052C9" data-open="DST052C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DST052C10" data-open="DST052C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DST052C15  DST052C17" data-open="DST052C15  DST052C17" class="link link-ref link-reveal xref-bibr">15–17</a></sup> some such primers have led to the amplification of eukaryotic 18S rRNA genes and small-subunit rRNA genes from mitochondria and chloroplasts.<sup><a reveal-id="DST052C10" data-open="DST052C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DST052C18" data-open="DST052C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Moreover, it is often difficult to estimate the number and diversity of the eukaryotic rRNA genes present in the environment before conducting the sequence-based community analysis.<sup><a reveal-id="DST052C18" data-open="DST052C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Therefore, the use of primers that amplify rRNA genes of eukaryotic origins has led to mischaracterization of the diversity of microbial communities that were associated with eukaryotic cells.<sup><a reveal-id="DST052C19" data-open="DST052C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DST052C20" data-open="DST052C20" class="link link-ref link-reveal xref-bibr">20</a></sup> This unfavourable situation is very serious in the case of host-associated habitats (e.g. the human skin, oral cavity, and vagina).<sup><a reveal-id="DST052C21" data-open="DST052C21" class="link link-ref link-reveal xref-bibr">21</a></sup></p><p>The aim of this study is to overcome the problems associated with primer choice, and we attempted to identify the novel universal primers that complement only conserved bacterial and archaeal 16S rRNA gene sequences so that eukaryotic and organelle rRNA gene sequences would not be amplified. In general, the commonly used degenerate primers do not always achieve a better taxonomic coverage than the non-degenerate primers,<sup><a reveal-id="DST052C22" data-open="DST052C22" class="link link-ref link-reveal xref-bibr">22</a></sup> and other recently designed degenerate primers still introduce a level of bias into the community profile.<sup><a reveal-id="DST052C23" data-open="DST052C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Therefore, our present study was focused on only the non-degenerate universal primers. The primer set identified in this study was experimentally used to show the taxonomic composition of a soil microbial community by amplifying 16S rRNA genes from the microbial metagenomic sample and subsequent pyrosequencing using the Roche 454 system. To evaluate the appropriateness of our original primer set to the taxonomic bias that might result from the PCR amplification, the same metagenomic sample was also used for (i) pyrosequencing of the 16S rRNA gene fragments after their amplification by use of the 338F and 533R set and (ii) the Illumina Genome Analyzer IIx (GA IIx) platform-based shotgun<sup><a reveal-id="DST052C24" data-open="DST052C24" class="link link-ref link-reveal xref-bibr">24</a></sup> sequencing of the metagenome and subsequent finding and analysis of the 16S rRNA gene fragments, and the taxonomic compositions inferred by the three sequencing experiments were compared.</p>                            <h2 scrollto-destination=5985428 id="5985428" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5985429 id="5985429" class="section-title" data-legacy-id=s2a>2.1 Identification of universal primer candidates from genome-sequenced strains</h3>
<p>To identify the candidate primer sequences for the PCR amplification of prokaryotic 16S rRNA genes, such genes from genome-sequenced strains were used as references because they have been accurately sequenced, are full-length genes, have well-defined taxonomic information. Bacterial and archaeal genomic sequences were obtained from the NCBI Genome Database (<a class="link link-uri" href="http://ftp://ftp.ncbi.nih.gov/genbank/genomes/Bacteria/">ftp://ftp.ncbi.nih.gov/genbank/genomes/Bacteria/</a>, accessed on 11 November 2013) in November 2008. The 16S rRNA gene sequences in each strain were identified by RNAmmer.<sup><a reveal-id="DST052C25" data-open="DST052C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Then, one 16S rRNA gene sequence per species was randomly chosen because slight sequence differences exist among the 16S rRNA genes from strains of the same species,<sup><a reveal-id="DST052C26" data-open="DST052C26" class="link link-ref link-reveal xref-bibr">26</a></sup> and among the gene copies within a genome.<sup><a reveal-id="DST052C27" data-open="DST052C27" class="link link-ref link-reveal xref-bibr">27</a></sup> A total of 531 16S rRNA gene sequences were chosen. Their taxonomic information was obtained from the NCBI Taxonomy Database (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/Taxonomy/taxonomyhome.html/">http://www.ncbi.nlm.nih.gov/Taxonomy/taxonomyhome.html/</a>, accessed on 11 November 2013). A multiple sequence alignment of the 531 16S rRNA gene sequences was constructed using MAFFT version 6.713 with default parameters.<sup><a reveal-id="DST052C28" data-open="DST052C28" class="link link-ref link-reveal xref-bibr">28</a></sup></p><p>To find out the candidate sequences described above, highly conserved regions identified in the reference alignment were chosen as follows. Generally, the primer lengths for the PCR-amplification of 16S rRNA genes are more than 15 nt;<sup><a reveal-id="DST052C9" data-open="DST052C9" class="link link-ref link-reveal xref-bibr">9</a></sup> therefore, we used a sliding window of 15 nt with a step size of 1 nt across the reference alignment. For each window, we calculated the frequency of each 15-nt sequence with one mismatch allowed. The 15-nt sequences that included gaps were also considered when calculating the frequencies. The consensus sequence for each window was defined as the 15-nt sequence that was found most frequently within one mismatch among strains. The coverage rate for a consensus sequence in each phylum was defined as the percentage of matched sequences among genome-sequenced strains within one mismatch.</p>                            <h3 scrollto-destination=5985432 id="5985432" class="section-title" data-legacy-id=s2b>2.2 Evaluation of each candidate universal sequence using the databases of the Ribosomal Database Project and the comparative RNA Web site</h3>
<p>Each candidate sequence was evaluated using the Probe Match program in the Ribosomal Database Project (RDP) (release 10, update 22; <a class="link link-uri" href="http://rdp.cme.msu.edu/probematch/search.jsp">http://rdp.cme.msu.edu/probematch/search.jsp</a>, accessed on 11 November 2013).<sup><a reveal-id="DST052C29" data-open="DST052C29" class="link link-ref link-reveal xref-bibr">29</a></sup> The 615 916 16S rRNA gene sequences in the RDP were retrieved by using the parameters Strain = Both, Source = Both, Size ≥1200, and Quality = Good, and then these sequences were used as reference sequences of the Probe Match program for the evaluation of the candidate sequences. The coverage for a candidate sequence in each phylum of the reference sequences was calculated as the percentage of genera for which more than half of the member sequences in each genus were found to match the candidate sequence within one mismatch. Because the reference sequences contain many uncultured strains,<sup><a reveal-id="DST052C29" data-open="DST052C29" class="link link-ref link-reveal xref-bibr">29</a></sup> their taxonomic classifications are ambiguous, especially at the species level (e.g. description of the sp. as the taxa). Therefore, we used the genus, but not the species, to calculate the coverage for each candidate sequence. More than half of the 16S rRNA gene sequences in the reference sequences lacked their 5′- or 3′-terminal regions at the time of our analysis (<a class="link link-uri" href="http://rdp.cme.msu.edu/download/release10_22_containPosition.xls">http://rdp.cme.msu.edu/download/release10_22_containPosition.xls</a>, accessed on 11 November 2013). Therefore, the candidate se-quences corresponding to such regions were excluded from the subsequent analyses.</p><p>The possibility of amplifying eukaryotic and organelle rRNA genes was evaluated by calculating the coverage for each primer candidate and the eukaryotic and organelle rRNA genes that were obtained from the comparative RNA Web (CRW).<sup><a reveal-id="DST052C30" data-open="DST052C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Aligned rRNA sequences of chloroplasts, mitochondria, and eukaryotes were separately downloaded from the CRW (<a class="link link-uri" href="http://www.rna.ccbb.utexas.edu/DAT/3A/Summary/index.php">http://www.rna.ccbb.utexas.edu/DAT/3A/Summary/index.php</a>, accessed on 11 November 2013). The sequences containing ambiguous nucleotides (e.g. any nucleotide, N, or pyrimidine, Y) were discarded. One 16S rRNA sequence per genus was randomly chosen to reduce taxonomic bias in the dataset. After the conversion of uracil residues to thymine ones in the sequences, the coverage for each candidate sequence in the CRW-derived and subsequently processed sequences (76 chloroplastic, 480 mitochondrial, and 583 eukaryotic ones) was calculated as the percentage of matched sequences within two mismatches to eliminate the possible amplification of such rRNA genes. Based on the results of prior studies,<sup><a reveal-id="DST052C18" data-open="DST052C18" class="link link-ref link-reveal xref-bibr">18</a></sup> we assumed that three or more mismatches between a primer and template would not allow the successful PCR amplification. The candidate sequences that covered more than 50% of the mitochondrial and eukaryotic rRNA gene sequences were also excluded to avoid the amplification of such genes. To reduce the possibility of random and non-specific amplification of non-rRNA genes, we merged neighbouring candidate sequences to extend their sequence lengths while maintaining the coverage for each phylum.</p><p>The validity of our primers was compared with the validity of 17 other universal primers published in the literature (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>).<sup><a reveal-id="DST052C9" data-open="DST052C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DST052C10" data-open="DST052C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DST052C12" data-open="DST052C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DST052C31" data-open="DST052C31" class="link link-ref link-reveal xref-bibr">31</a></sup> We investigated the coverage of our and the 17 other primer sequences for (i) the 16S rRNA genes from bacterial and archaeal phyla in the reference sequences and (ii) the CRW-derived chloroplastic, mitochondrial, and eukaryotic rRNA genes described above. To avoid the difficulties of <em>in silico</em> evaluation, all ambiguous nucleotides in the 17 primers were converted to the nucleotides that gave rise to the best coverage for the reference sequences of bacterial and archaeal origins.</p>                            <h3 scrollto-destination=5985436 id="5985436" class="section-title" data-legacy-id=s2c>2.3 Sample processing and sequencing of a soil microbial community</h3>
<p>A brown forest soil sample was collected in April 2008 from farmland at the Ehime Research Institute of Agriculture, Forestry, and Fisheries, Matsuyama, Japan,<sup><a reveal-id="DST052C32" data-open="DST052C32" class="link link-ref link-reveal xref-bibr">32</a></sup> sieved through a 2-mm mesh to remove larger particles, and transferred to a sterilized glass pot with a loose lid. After its water content was adjusted to 60% of the maximum water-holding capacity, the soil was incubated at 28°C for 24 weeks. Total DNA was thereafter extracted from 10 g of soil using a PowerMax Soil DNA Isolation kit (MoBio, Carlsbad, CA, USA) according to the manufacturer's instructions. The DNA sample was concentrated by ethanol precipitation to obtain sufficient amount of DNA for PCR and metagenomic sequencing. PCR amplification of 16S rRNA genes was performed in 50 µl of 1 × <em>Ex Taq</em> buffer (2 mM Mg<sup>2+</sup> Plus; Takara Bio, Ohtsu, Japan) containing 0.2 mM dNTPs, 0.625 U of TaKaRa <em>Ex Taq</em> HS (Takara Bio), 0.2 µM each of the forward and reverse primers, 2% dimethylsulphoxide, 0.01% bovine serum albumin, and 30 ng template DNA. Two PCR primer sets, 342F-806R and 338F-533R,<sup><a reveal-id="DST052C12" data-open="DST052C12" class="link link-ref link-reveal xref-bibr">12</a></sup> were used. The PCR programme consisted of a single cycle of 95°C for 30 s, followed by 25 cycles of 95°C for 30 s, 55°C for 30 s, and 72°C for 15 s. A final extension was performed at 72°C for 7 min. PCR amplicons from eight parallel reactions were individually concentrated and purified by gel electrophoresis, and subsequently extracted to obtain more than 2 µg of amplicons. Pooled amplicons were sequenced on a 454 GS FLX Titanium one-sixteenth picotiter plate according to the manufacturer's protocols. In addition, 1-µg portions of metagenomic DNA were sheared to approximately 200-bp fragments using a Covaris-S instrument (Covaris, Woburn, MY, USA), and adapter sequences were ligated to both ends of the DNA fragments to generate a paired-end library. The library was subjected to 76 cycles of paired-end sequencing with an Illumina GA IIx instrument (one lane of an eight-lane flow cell) using reagents of an Illumina Sequencing kit, version 3.0, followed by base-calling using GA pipeline, version 1.5.0.</p>                            <h3 scrollto-destination=5985438 id="5985438" class="section-title" data-legacy-id=s2d>2.4 Sequence analysis</h3>
<p>We used the high-quality 454 platform reads after removal of the reads that (i) contained ambiguous nucleotides, (ii) contained &lt;150 or &gt;250 nt in the 338F-533R experiment and &lt;350 or &gt;550 nt in the 342F-806R experiment, and (iii) were associated with an average Phred-like quality score of less than 20 as calculated by the 454 sequencer. Both the forward and reverse primer sequences were removed using cross-match, version 1.090518 (-minmatch 10 -minscore 12) (<a class="link link-uri" href="http://www.phrap.org/phredphrapconsed.html">http://www.phrap.org/phredphrapconsed.html</a>, accessed on 11 November 2013). Reverse-complemented reads were converted so that all reads started according to standard-sense strand conventions. Chimera sequences were detected and removed by the ChimeraSlayer in mothur version 1.11,<sup><a reveal-id="DST052C33" data-open="DST052C33" class="link link-ref link-reveal xref-bibr">33</a></sup> using the Silva reference alignment (<a class="link link-uri" href="http://www.mothur.org/w/images/f/f1/Silva.gold.bacteria.zip">http://www.mothur.org/w/images/f/f1/Silva.gold.bacteria.zip</a>, accessed on 11 November 2013). We also discarded the Illumina reads that (i) were evaluated as quality ‘N’ by purity filtering using the GA pipeline version 1.5.0 and (ii) contained more than one nucleotide with the quality-flag ‘B’ in the first 60 nt.</p><p>From the 615 916 16S rRNA gene sequences that were used as the reference sequences (see above), we eliminated (i) the 117 059 sequences whose origins at the genus-level were designated as ‘unclassified,’ and (ii) forty 23S rRNA gene-derived sequences with lengths longer than 2000 nt. Six sequences (S001337843, S001337931, S001337937, S001338015, S001338173, and S001338177) that contained unusually long homopolymers were also discarded because such homopolymers have returned many hits in our sequence similarity searches of the Illumina reads. Consequently, we obtained a database consisting of a total of 498 811 high-quality 16S rRNA gene sequences whose origins at the phylum and genus are definite, and we designated this database the ‘in-house high-quality RDP’ database. This database for <em>Bacteria</em> and <em>Archaea</em> was merged with the CRW-derived and above-described rRNA gene sequence database for chloroplasts, mitochondria, and eukaryotes.</p><p>Taxonomic assignment of the high-quality reads that were obtained from the 454 amplicon sequencing by use of the 338F-533R and 342F-806R primer sets was performed by a BLASTN<sup><a reveal-id="DST052C34" data-open="DST052C34" class="link link-ref link-reveal xref-bibr">34</a></sup> search [e-value &lt;1e−8; Identity ≥94% (genus) or ≥85% (phylum); and Alignment length ≥100 (for 338F-533R) or ≥300 (for 342F-806R)] against our merged database described above. The thresholds for alignment length were necessary to eliminate inaccurate or false-positive alignments. Identification and taxonomic assignment of 16S or 18S rRNA gene fragments in the reads obtained by the Illumina metagenomic sequencing was performed by a BLASTN search [e-value &lt;1e−5; Identity ≥94% (genus) or ≥85% (phylum); and Alignment length ≥50] of our merged database.</p><p>When a best hit with the largest BLAST bit score existed, one was added to the taxon of the best hit for calculation of the taxonomic composition of the microbial community. When two or more database sequences had the same best BLAST bit score, one divided by the number of best hits was added to the corresponding taxa of the best hits. To improve the accuracy of the taxonomic assignments of the Illumina reads, we used only the read pairs whose sequences (i) were identified as those in a 16S rRNA gene from the same taxon and (ii) had a consistent alignment direction between the paired reads. The distance between the paired reads was not considered because the 16S rRNA genes in some taxa contain long insertions.<sup><a reveal-id="DST052C35" data-open="DST052C35" class="link link-ref link-reveal xref-bibr">35</a></sup></p>                            <h3 scrollto-destination=5985443 id="5985443" class="section-title" data-legacy-id=s2e>2.5 Nucleotide sequence accession numbers</h3>
<p>Sequences have been submitted to the DDBJ Sequence Read Archive under the following accession numbers: DRR001479 and DRR001475 for the 454-read data obtained using the 338F-533R and 342F-806R primer sets, respectively, and DRR001464 for the Illumina read data by metagenomic sequencing.</p>                            <h2 scrollto-destination=5985445 id="5985445" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5985446 id="5985446" class="section-title" data-legacy-id=s3a>3.1 Identification of universal primer candidates for 16S rRNA genes from genome-sequenced strains</h3>
<p>There is a large collection of 16S rRNA gene sequences in the databases,<sup><a reveal-id="DST052C26" data-open="DST052C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DST052C29" data-open="DST052C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DST052C30" data-open="DST052C30" class="link link-ref link-reveal xref-bibr">30</a></sup> and almost all such sequences were obtained by use of PCR. Therefore, the sequences that failed to match the universal primers do not exist in the database.<sup><a reveal-id="DST052C36" data-open="DST052C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Furthermore, many sequences in the database still contain the primer sequences used for PCR.<sup><a reveal-id="DST052C37" data-open="DST052C37" class="link link-ref link-reveal xref-bibr">37</a></sup> On the other hand, the 16S rRNA gene sequences from the genome-sequenced prokaryotic strains contain the complete 16S rRNA gene sequences, including both terminal regions, which are missing in many cases in the databases (e.g. see <a class="link link-uri" href="http://rdp.cme.msu.edu/download/release10_22_containPosition.xls">http://rdp.cme.msu.edu/download/release10_22_containPosition.xls</a>, accessed on 11 November 2013). Therefore, the 16S rRNA gene sequences from the genome-sequenced strains are expected to provide us accurate information on the variability of 16S rRNA genes from many different taxa.</p><p>Information concerning the consensus sequence for each 15-nt window and its coverage rate for 16S rRNA genes from the genome-sequenced strains of each phylum is shown in Figure <a reveal-id="DST052F1" data-open="DST052F1" class="link link-reveal link-table xref-fig">1</a> (some of the windows) and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figure S1</a> (all windows). The windows in which almost all of the dots for the archaeal (top four) and bacterial lines showed &gt;90% sequence identities (coloured red or yellow) were the universal primer candidate sequences. The candidates with conserved sequences for almost all bacterial and archaeal phyla were identified, and 50 such sequences are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>. Although the list was prepared based on the genomic data available in November 2008, the use of the updated data in July 2013 did not affect the conservation pattern of 50 universal primer candidate sequences across phyla (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Our 50 universal primer candidate sequences are well overlapped with the degenerate primer candidates proposed by Wang and Qian (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>),<sup><a reveal-id="DST052C9" data-open="DST052C9" class="link link-ref link-reveal xref-bibr">9</a></sup> in spite that their algorism and database used were different from ours. Since the maximum sequencing read by the 2nd-generation sequencers at the time of our study was up to 450 nt, 17 out of the 50 candidate universal primer sequences that are located at the 3′-terminal regions of 16S rRNA genes (e.g. sequences 1390–1395, 1491–1496, and 1525–1526, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>) were excluded for further analysis. Because the latest systems allow the determination of sequences up to 1100 nt in length (e.g. the PacBio sequencer<sup><a reveal-id="DST052C7" data-open="DST052C7" class="link link-ref link-reveal xref-bibr">7</a></sup> and 454 GS FLX+ system; <a class="link link-uri" href="http://454.com/products/gs-flx-system/index.asp">http://454.com/products/gs-flx-system/index.asp</a>, accessed on 11 November 2013), the excluded sequences might be useful as primer candidates in the near feature. </p>                        <a id="5985449" scrollto-destination="5985449"></a>
<div id="DST052F1" data-id="DST052F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/m_dst05201.png?Expires=2147483647&Signature=Jzw5bYc5KD22ousdn5aOnxVaM2B-n4pFMCmMF52f-Q6UBE3JFBQCwVORXbB5vRbmhiMgYQnO3d12qhmBvdDt22gLscq9r0V6UmEylAGC2S3ckiDSTMB8h9HlDib3xg~5mLfmA8ih8OC8FimwXYIw~WPvwtMhUYBtmhk~TE9d6LX7Z2-vbo6xb4R0J5~5thXtynamd1dncIHoLBkUBlQamoJcWHbbj9lzWP9o98MHoxi6qRyuL1dZM1pDNL~1ZhxBm2EKxA94xk7MPiHoG0n3paZcgoYI6E00CY7J4-G7TvxrtIVt5l-Qyzp6BErY41qmQJ5D5znQddUnQuYYsMk9eA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A partial plot of the consensus sequence in each window and its coverage rate for each phylum from 531 16S rRNA genes of genome-sequenced strains. The number of species in each phylum is indicated in parentheses. The start position of each window is represented according to the corresponding position in the 16S rRNA gene from Escherichia coli. The consensus sequence in each window is represented at the bottom of the figure. Each line indicate the coverage rate for the consensus sequence of each phylum by using the colours of dots: black &lt;50%, blue &lt;80%, green &lt;90%, yellow &lt;100%, and red = 100%. The bar graph at the top of the figure indicates the sequence variability of each window that is depicted by (i) calculating the relative entropy of four nucleotides and the gap (−) at each site38 and (ii) summing up the relative entropy of each site in the window." path-from-xml="dst05201.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5985449/dst05201.tiff" path-from-xml="dst05201.tiff" target="_blank">View large</a><a section="5985449" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/dst05201.png?Expires=2147483647&Signature=l2ZH~gEnYAnb~QYKpCaMG~qAFaxDjEfJl~CoxH0O9tfec8UweyPKtdpdVLBKwCtvz4D4k8yCaxEdy3i1rKOzIJglWyK1SEQlI44cjru0cdUcC~i95Cqo0LQB4LXgWDj4DAvbMwwtAEWMCTDaZww-AaWx92d1hzaYwAbqg82KCs5NKlGOv63lE9U1w2-pNh4iD5gTJQA7KPUNwCdALynY8Rf447i0apuXzN3ZMiEHtVZb83PPvBMdVxYnqHBrx3y6qQw3gm-GCxWvYGq~Wm1Rkm9pFkxG1okhqc821Nh910jkmT~41UnLNBowko8iMOuAZNkUnoLR2thAgLYIOatcvA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5985449&ar=404545&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05201.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A partial plot of the consensus sequence in each window and its coverage rate for each phylum from 531 16S rRNA genes of genome-sequenced strains. The number of species in each phylum is indicated in parentheses. The start position of each window is represented according to the corresponding position in the 16S rRNA gene from <em>Escherichia coli</em>. The consensus sequence in each window is represented at the bottom of the figure. Each line indicate the coverage rate for the consensus sequence of each phylum by using the colours of dots: black &lt;50%, blue &lt;80%, green &lt;90%, yellow &lt;100%, and red = 100%. The bar graph at the top of the figure indicates the sequence variability of each window that is depicted by (i) calculating the relative entropy of four nucleotides and the gap (−) at each site<sup><a reveal-id="DST052C38" data-open="DST052C38" class="link link-ref link-reveal xref-bibr">38</a></sup> and (ii) summing up the relative entropy of each site in the window.</p></div></div><div content-id="DST052F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/m_dst05201.png?Expires=2147483647&Signature=Jzw5bYc5KD22ousdn5aOnxVaM2B-n4pFMCmMF52f-Q6UBE3JFBQCwVORXbB5vRbmhiMgYQnO3d12qhmBvdDt22gLscq9r0V6UmEylAGC2S3ckiDSTMB8h9HlDib3xg~5mLfmA8ih8OC8FimwXYIw~WPvwtMhUYBtmhk~TE9d6LX7Z2-vbo6xb4R0J5~5thXtynamd1dncIHoLBkUBlQamoJcWHbbj9lzWP9o98MHoxi6qRyuL1dZM1pDNL~1ZhxBm2EKxA94xk7MPiHoG0n3paZcgoYI6E00CY7J4-G7TvxrtIVt5l-Qyzp6BErY41qmQJ5D5znQddUnQuYYsMk9eA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A partial plot of the consensus sequence in each window and its coverage rate for each phylum from 531 16S rRNA genes of genome-sequenced strains. The number of species in each phylum is indicated in parentheses. The start position of each window is represented according to the corresponding position in the 16S rRNA gene from Escherichia coli. The consensus sequence in each window is represented at the bottom of the figure. Each line indicate the coverage rate for the consensus sequence of each phylum by using the colours of dots: black &lt;50%, blue &lt;80%, green &lt;90%, yellow &lt;100%, and red = 100%. The bar graph at the top of the figure indicates the sequence variability of each window that is depicted by (i) calculating the relative entropy of four nucleotides and the gap (−) at each site38 and (ii) summing up the relative entropy of each site in the window." path-from-xml="dst05201.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5985449/dst05201.tiff" path-from-xml="dst05201.tiff" target="_blank">View large</a><a section="5985449" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/dst05201.png?Expires=2147483647&Signature=l2ZH~gEnYAnb~QYKpCaMG~qAFaxDjEfJl~CoxH0O9tfec8UweyPKtdpdVLBKwCtvz4D4k8yCaxEdy3i1rKOzIJglWyK1SEQlI44cjru0cdUcC~i95Cqo0LQB4LXgWDj4DAvbMwwtAEWMCTDaZww-AaWx92d1hzaYwAbqg82KCs5NKlGOv63lE9U1w2-pNh4iD5gTJQA7KPUNwCdALynY8Rf447i0apuXzN3ZMiEHtVZb83PPvBMdVxYnqHBrx3y6qQw3gm-GCxWvYGq~Wm1Rkm9pFkxG1okhqc821Nh910jkmT~41UnLNBowko8iMOuAZNkUnoLR2thAgLYIOatcvA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5985449&ar=404545&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05201.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A partial plot of the consensus sequence in each window and its coverage rate for each phylum from 531 16S rRNA genes of genome-sequenced strains. The number of species in each phylum is indicated in parentheses. The start position of each window is represented according to the corresponding position in the 16S rRNA gene from <em>Escherichia coli</em>. The consensus sequence in each window is represented at the bottom of the figure. Each line indicate the coverage rate for the consensus sequence of each phylum by using the colours of dots: black &lt;50%, blue &lt;80%, green &lt;90%, yellow &lt;100%, and red = 100%. The bar graph at the top of the figure indicates the sequence variability of each window that is depicted by (i) calculating the relative entropy of four nucleotides and the gap (−) at each site<sup><a reveal-id="DST052C38" data-open="DST052C38" class="link link-ref link-reveal xref-bibr">38</a></sup> and (ii) summing up the relative entropy of each site in the window.</p></div></div>                            <h3 scrollto-destination=5985450 id="5985450" class="section-title" data-legacy-id=s3b>3.2 Coverage for our candidate primer sequences and other universal primer sequences in bacterial, archaeal, eukaryotic, and organelle rRNA genes</h3>
<p>We calculated the coverage for each of the remaining 33 candidates for rRNA genes from each bacterial and archaeal phylum in our in-house high-quality RDP database and from chloroplasts, mitochondria, and eukaryotes in the CRW-derived and above-described database, and the results are depicted as a heat map (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). All of the candidate sequences covered most of the chloroplastic rRNA genes within two mismatches, indicating the difficulty in avoiding their amplification during microbial community profiling. Twenty-one out of the 33 primer candidates [514–522 (nine candidates), 781–788 (seven candidates), and 907–915 (five candidates)] had relatively large coverage in bacterial and archaeal 16S rRNA genes. However, these 21 candidate sequences also covered the mitochondrial and eukaryotic rRNA genes within two mismatches; therefore, these sequences were not further considered as candidates. In cases in which the targeted microbial community contains only a small fraction of <em>Eukaryota</em>, these primers might be a good choice for the community profiling.</p><p>The remaining 12 candidates [341–343 (three candidates), 789–792 (four candidates), 879–880 (two candidates), and 1060–1066 (three candidates)] had sequences that appeared to exclude possible amplification of mitochondrial and eukaryotic rRNA genes. Because (i) the maximum length of a high-quality read for the 454 GS FLX Titanium system was ∼450 nt and (ii) the longer sequences provide more accurate taxonomic assignments,<sup><a reveal-id="DST052C8" data-open="DST052C8" class="link link-ref link-reveal xref-bibr">8</a></sup> we chose candidates 341–343 and 789–792 as forward and reverse universal primer candidates, respectively. After merging the sequences of the neighbouring candidates, but maintaining the coverage in each phylum, we finally chose the sequence from 342 to 357 (5′-CTACGGGGGGCAGCAG-3′; 16 nt: 342F) and that from 790 to 806 (5′-GGACTACCGGGGTATCT-3′; 17 nt: 806R) as potential forward and reverse universal primers, respectively.</p><p>The coverage for our universal primer set and the 17 other universal primers (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>) in each of the bacterial and archaeal phyla rRNA genes, and the chloroplastic, mitochondrial, and eukaryotic rRNA genes is represented as a heat map (Fig. <a reveal-id="DST052F2" data-open="DST052F2" class="link link-reveal link-table xref-fig">2</a>). The previously designed primers,<sup><a reveal-id="DST052C9" data-open="DST052C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DST052C10" data-open="DST052C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DST052C12" data-open="DST052C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DST052C31" data-open="DST052C31" class="link link-ref link-reveal xref-bibr">31</a></sup> 519F–534R (four primers), 784F, 906F, and 926R, covered mitochondrial and/or eukaryotic rRNA genes within two mismatches. The coverage for our universal primer set was, when compared with those for the 17 other universal primers, relatively large for bacterial and archaeal rRNA genes; however, relatively poor coverage was observed for some phyla (e.g. <em>Crenarchaeota</em>, <em>Planctomycetes</em>, OD1, and <em>Chlamydiae</em>). Furthermore, our universal primers had four mismatches (on average) against mitochondrial and eukaryotic rRNA genes. Most of the 19 primers covered chloroplastic rRNA genes within two mismatches (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). This result and the data in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a> indicate that all of universal primer sequences examined in this study would be difficult to avoid the amplifications of chloroplastic rRNA genes. Finding a universal primer that avoids the amplification of chloroplastic rRNA genes is an important issue to be carried out in the future. </p>                        <a id="5985454" scrollto-destination="5985454"></a>
<div id="DST052F2" data-id="DST052F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/m_dst05202.png?Expires=2147483647&Signature=g93sG8ll6j4UOzEsD-xHnWLh3TLldcMr~HJUxvfJDqBZSpxVmWT5lOshoyd0cTV1fEgPIY6sepYgI3GZl~AGJDeeZnzEjkn2ov7KNQiYx70GdLnsQjox-4aCPvOrCSrPHM6k~U7Eh6r8WVXX0N1hgcXTWBWg-D2hWNf4~iv7NhqKUK9aEqMX-IXcuNzrHJs-559R2j6riKLWEZKQmE5dCSD6El3jsmIzUPnLQKTIP7ecFS7tGojKOmrMZBU04dVmjaY87ppB7ZBfHnOWa2K~WScR2gHFboahNHF~CuuCv4TaKtiGD2Pugm5zUJKXfQ0QQqM~aVRtvwSU7aKeSA3IaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The coverage for our universal primer set (red) and the 17 other universal primers in each of the bacterial and archaeal phyla rRNA genes, and the chloroplastic, mitochondrial, and eukaryotic rRNA genes. Each column indicates the coverage for a candidate sequence from each of the bacterial and archaeal phyla rRNA genes, and the chloroplastic, mitochondrial, and eukaryotic rRNA genes by colour intensity: blue = low coverage, and pale orange = high coverage. See Supplementary Table S1 for each sequence. The number of genera is provided in parentheses." path-from-xml="dst05202.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5985454/dst05202.tiff" path-from-xml="dst05202.tiff" target="_blank">View large</a><a section="5985454" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/dst05202.png?Expires=2147483647&Signature=T3ZA0AifZCNIY72knLObgDd8gDMuot-MhcIwJzvuUwMhYVTqeQs7ZjrhmB5LIjAwd~KxCeQlc8JAiQcF1BAhmF93v2EVjqe~NaM6yeKmkvqprtJGLQYjq7wL6mUKsqdqJGPrPVy~FArZ3Qn0PM2Fo2u~P7e2bFiN3bJNiiZwzYFg~3mUfHWclS5jRmg6Aa7rTdqMJ6IB424N4N5HphJQTmBReYFo90s-PUV9CjlknzNL6cJ~xxtF~wPwLTaLwdYZrV6OQqVNZKvAYe2i9pFPGIg2Pi4LVX4HS8FlLYviZM6xctIVIxTLTViGYpz5cV8E6UFMkLkfo49cdlLkZ8jBaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5985454&ar=404545&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05202.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The coverage for our universal primer set (red) and the 17 other universal primers in each of the bacterial and archaeal phyla rRNA genes, and the chloroplastic, mitochondrial, and eukaryotic rRNA genes. Each column indicates the coverage for a candidate sequence from each of the bacterial and archaeal phyla rRNA genes, and the chloroplastic, mitochondrial, and eukaryotic rRNA genes by colour intensity: blue = low coverage, and pale orange = high coverage. See <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a> for each sequence. The number of genera is provided in parentheses.</p></div></div><div content-id="DST052F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/m_dst05202.png?Expires=2147483647&Signature=g93sG8ll6j4UOzEsD-xHnWLh3TLldcMr~HJUxvfJDqBZSpxVmWT5lOshoyd0cTV1fEgPIY6sepYgI3GZl~AGJDeeZnzEjkn2ov7KNQiYx70GdLnsQjox-4aCPvOrCSrPHM6k~U7Eh6r8WVXX0N1hgcXTWBWg-D2hWNf4~iv7NhqKUK9aEqMX-IXcuNzrHJs-559R2j6riKLWEZKQmE5dCSD6El3jsmIzUPnLQKTIP7ecFS7tGojKOmrMZBU04dVmjaY87ppB7ZBfHnOWa2K~WScR2gHFboahNHF~CuuCv4TaKtiGD2Pugm5zUJKXfQ0QQqM~aVRtvwSU7aKeSA3IaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The coverage for our universal primer set (red) and the 17 other universal primers in each of the bacterial and archaeal phyla rRNA genes, and the chloroplastic, mitochondrial, and eukaryotic rRNA genes. Each column indicates the coverage for a candidate sequence from each of the bacterial and archaeal phyla rRNA genes, and the chloroplastic, mitochondrial, and eukaryotic rRNA genes by colour intensity: blue = low coverage, and pale orange = high coverage. See Supplementary Table S1 for each sequence. The number of genera is provided in parentheses." path-from-xml="dst05202.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5985454/dst05202.tiff" path-from-xml="dst05202.tiff" target="_blank">View large</a><a section="5985454" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/dst05202.png?Expires=2147483647&Signature=T3ZA0AifZCNIY72knLObgDd8gDMuot-MhcIwJzvuUwMhYVTqeQs7ZjrhmB5LIjAwd~KxCeQlc8JAiQcF1BAhmF93v2EVjqe~NaM6yeKmkvqprtJGLQYjq7wL6mUKsqdqJGPrPVy~FArZ3Qn0PM2Fo2u~P7e2bFiN3bJNiiZwzYFg~3mUfHWclS5jRmg6Aa7rTdqMJ6IB424N4N5HphJQTmBReYFo90s-PUV9CjlknzNL6cJ~xxtF~wPwLTaLwdYZrV6OQqVNZKvAYe2i9pFPGIg2Pi4LVX4HS8FlLYviZM6xctIVIxTLTViGYpz5cV8E6UFMkLkfo49cdlLkZ8jBaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5985454&ar=404545&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05202.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The coverage for our universal primer set (red) and the 17 other universal primers in each of the bacterial and archaeal phyla rRNA genes, and the chloroplastic, mitochondrial, and eukaryotic rRNA genes. Each column indicates the coverage for a candidate sequence from each of the bacterial and archaeal phyla rRNA genes, and the chloroplastic, mitochondrial, and eukaryotic rRNA genes by colour intensity: blue = low coverage, and pale orange = high coverage. See <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a> for each sequence. The number of genera is provided in parentheses.</p></div></div><p>In general, the broad-range primers targeting bacterial and archaeal 16S rRNA genes tended to also amplify eukaryotic rRNA genes.<sup><a reveal-id="DST052C10" data-open="DST052C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DST052C18" data-open="DST052C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DST052C20" data-open="DST052C20" class="link link-ref link-reveal xref-bibr">20</a></sup> When researchers analyse the microbial communities that contain many eukaryotic cells, the use of broad-range primers will lead to a decrease in the efficiency of sequencing and the mischaracterization of the microbial diversity in each community.<sup><a reveal-id="DST052C19" data-open="DST052C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Such communities are exemplified by eukaryotic host-associated ones: for example, the human microbiome project revealed that the average ratios of human DNA in the total metagenomic DNA extracted from several human body sites were about 50%.<sup><a reveal-id="DST052C21" data-open="DST052C21" class="link link-ref link-reveal xref-bibr">21</a></sup> A broad-range primer, 515F, is used as a standard universal primer to analyse microbial communities in several environments in the Earth Microbiome Project (<a class="link link-uri" href="http://www.earthmicrobiome.org/">http://www.earthmicrobiome.org/</a>, accessed on 11 November 2013). Since the 515F primer perfectly matches to many eukaryotic, mitochondrial, and chloroplastic rRNA genes, some studies that analysed animal-associated environments reported that the use of this primer also led to the amplification of many eukaryotic, mitochondrial, and chloroplastic rRNA genes.<sup><a reveal-id="DST052C39" data-open="DST052C39" class="link link-ref link-reveal xref-bibr">39</a></sup> We primarily aimed to identify novel non-degenerate universal primers that were expected to amplify most bacterial and archaeal rRNA genes, but not eukaryotic rRNA genes. Because of this aim, our novel universal primers could not easily amplify the 16S rRNA genes from some bacterial and archaeal taxa without bias. Examples of such underrepresented taxa are the archaeal phyla, <em>Crenarchaeota</em>, <em>Korarchaeota,</em> and <em>Nanoarchaeota</em>, and the bacterial phyla, <em>Chlamydiae</em>, <em>Lentisphaerae</em>, OD1, OP11, <em>Planctomycetes,</em> and SR1 (Fig. <a reveal-id="DST052F2" data-open="DST052F2" class="link link-reveal link-table xref-fig">2</a>). If these taxa are the major members in a microbial community of an environment, use of the 342F-806R primer set for the amplicon sequencing analysis will give rise to significantly biased taxonomic composition. To avoid misunderstanding of the compositions, the prior information related to the taxonomic composition of the community is very important. For example, <em>Crenarchaeota</em> is a major archaeal phylum;<sup><a reveal-id="DST052C40" data-open="DST052C40" class="link link-ref link-reveal xref-bibr">40</a></sup> thus, in the analysis of the microbial community rich in this phylum, the combined use of our universal primer set with an established <em>Archaea</em>-specific universal primer set<sup><a reveal-id="DST052C10" data-open="DST052C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DST052C17" data-open="DST052C17" class="link link-ref link-reveal xref-bibr">17</a></sup> will allow more accurate description of the composition of the community. Since both of the primer sets can amplify 16S rRNA genes from <em>Euryarchaeota</em>, the relative abundance of this phylum in the community can be inferred, allowing us to calibrate the relative abundance of <em>Crenarchaeota</em> to that of <em>Bacteria</em> in the community.</p>                            <h3 scrollto-destination=5985456 id="5985456" class="section-title" data-legacy-id=s3c>3.3 Comparison of soil microbial community profiling by pyrosequencing with the two sets of primer pairs and by Illumina metagenomic sequencing</h3>
<p>Experimental validation of our universal primers was conducted by 16S rRNA gene-based profiling of a soil microbial community with the 454 pyrosequencing. To assess whether our primer set enabled us to amplify most bacterial taxa in the soil sample in comparison with other primer sets, a commonly used bacteria-specific primer set, 338F-533R,<sup><a reveal-id="DST052C12" data-open="DST052C12" class="link link-ref link-reveal xref-bibr">12</a></sup> was also used for the 454 pyrosequencing. The use of 16S rRNA gene fragments obtained by shotgun metagenomic sequencing can eliminate any bias that might be associated with the initial PCR amplification of 16S rRNA genes for the pyrosequencing.<sup><a reveal-id="DST052C41" data-open="DST052C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Therefore, metagenomic sequencing was additionally performed using the Illumina platform with paired-end libraries of the same sample to examine whether use of our universal primers resulted in taxonomic bias. Our sequencing and subsequent filtering generated 15 165 (338F-533R) and 27 527 (342F-806R) high-quality 454 reads by 16S rRNA gene amplicon sequencing, and 12 825 888 high-quality Illumina reads (6 412 944 read pairs) by metagenomic sequencing. A total of 3026 Illumina reads were assigned to 16S rRNA genes.</p><p>Our use of the two primer sets generated no 454 reads that were assigned to mitochondrial or eukaryotic rRNA genes from our refined CRW database (see Materials and methods), although eight Illumina read pairs were assigned to 18S rRNA genes from some fungi. Therefore, our universal primers did not allow the detection of the amplification of mitochondrial or eukaryotic rRNA genes.</p><p>Figure <a reveal-id="DST052F3" data-open="DST052F3" class="link link-reveal link-table xref-fig">3</a> shows the results of a BLASTN search for phylum-level taxonomic assignment of each 454 and Illumina read against our merged in-house high-quality RDP and CRW sequence database. The numbers of taxonomically assigned reads at the phylum level were 13 591 (89.62% of the total high-quality 454 reads for the 338F-533R experiment), 26 738(97.13% of the total high-quality 454 reads for the 342F-806R experiment), and 1513 for the Illumina read pairs (3026 reads) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Among the three types of experiments, the taxonomic compositions at the phylum level exhibited good Spearman's correlations (<em>P</em>-value &lt; 0.05): illumina versus 338F-533R, <em>r</em> = 0.75; Illumina versus 342F-806R, <em>r</em> = 0.81; and 338F-533R versus 342F-806R, <em>r</em> = 0.86. These three experiments identified <em>Proteobacteria</em> and <em>Acidobacteria</em> as the major phyla in the soil microbial community. The compositional bias for each phylum among the three experiments was evaluated by investigating the rank abundance of each phylum in each experiment (Fig. <a reveal-id="DST052F4" data-open="DST052F4" class="link link-reveal link-table xref-fig">4</a>). In comparison with the Illumina metagenomic sequencing data, the 454 amplicon sequencing using the 338F-533R and 342F-806R primer sets gave rise to data in which the phyla <em>Verrucomicrobia</em>, <em>Planctomycetes</em>, <em>Crenarchaeota</em> and <em>Euryarchaeaota</em>, and the phyla <em>Planctomycetes</em>, <em>Crenarchaeota</em>, OD1, and <em>Chlamydiae</em>, respectively, were underrepresented. Such underrepresentation might have occurred because most 16S rRNA genes from the phyla had sequences with more than two mismatches with the 338F and 342F primer sequences. The use of the 338F-533R primer set generated no 454 reads that were assigned to <em>Euryarchaeota</em>, being consistent with the specific amplification of only bacterial 16S rRNA genes by this primer set. Conversely, eight 454 reads by the 342F-806R experiment and ten read pairs by the Illumina metagenomic sequencing were assigned to this phylum, indicating that 342F-806R can simultaneously amplify bacterial and archaeal 16S rRNA genes. </p>                        <a id="5985460" scrollto-destination="5985460"></a>
<div id="DST052F3" data-id="DST052F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/m_dst05203.png?Expires=2147483647&Signature=OKScO33jY1N3X0sk~Vu9aIe1gvwk~-Q8sthpuExewjcGeNdAk5xak92tmJO8t~Hq0del26NhZ7svOfrikZUNchF108cxDeaPWpR0aJa3CTeM7LPdEmgprAeM05czgKFUxOFrB1zWbpq-3uT7VSHomfsNNySHWVrMs9rGV3DbfD4hyfbwq2hB8kbs2lYferAZicvTZrZD4cz9hDUPobGgPXjCkObA59U99oq751giPDKe3EDtc0j8NfbflPp6gKrINuRBYGV3N4E5wAoUIJji81WIkBqgYg1hcPGecExCzGxqVJR4WsTtbwjR2YVqiqUwqsUfGDfBmtD3OSMkqkf6Ug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylum-level taxonomic compositions of prokaryotes in the soil metagenome on the basis of the 338F-533R and 342F-806R amplicon pyrosequencing and the 16S rRNA gene fragments from Illumina metagenomic sequencing. The top 18 abundant phyla from the results of Illumina metagenomic sequencing are listed." path-from-xml="dst05203.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5985460/dst05203.tiff" path-from-xml="dst05203.tiff" target="_blank">View large</a><a section="5985460" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/dst05203.png?Expires=2147483647&Signature=jhUTuA~dCR27NihPgTxle-6A5zdYJT~mRLyvVG3PWWZy1cogj2F2B7NdTLCZ5Ises-sY~y-2WUxNXfEf3cb0MAWAxaDkWr~ABesmLCkCzxPYEmD1quGrcLnFT92sm3ENKNPFmSrPad6iGduDapQEr0HqsxrWdgx0G7Pt9cktJR4NQzRfjwWrtXT47TPaQX0bcpu2QyvuGUQW67TqGaC2LzXC~UjYfNoOiTxCSC4XO2Pl6mUlXKXLc-9IjUmJChbOUXYTSRfNwLfPmgAFurGgjP5hn-CbxnIjtiCb9r2abM3Of~gikf31ZmREtTGs19ISvNiYQSnd8QQL3ZsfslW4vA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5985460&ar=404545&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05203.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylum-level taxonomic compositions of prokaryotes in the soil metagenome on the basis of the 338F-533R and 342F-806R amplicon pyrosequencing and the 16S rRNA gene fragments from Illumina metagenomic sequencing. The top 18 abundant phyla from the results of Illumina metagenomic sequencing are listed.</p></div></div><div content-id="DST052F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/m_dst05203.png?Expires=2147483647&Signature=OKScO33jY1N3X0sk~Vu9aIe1gvwk~-Q8sthpuExewjcGeNdAk5xak92tmJO8t~Hq0del26NhZ7svOfrikZUNchF108cxDeaPWpR0aJa3CTeM7LPdEmgprAeM05czgKFUxOFrB1zWbpq-3uT7VSHomfsNNySHWVrMs9rGV3DbfD4hyfbwq2hB8kbs2lYferAZicvTZrZD4cz9hDUPobGgPXjCkObA59U99oq751giPDKe3EDtc0j8NfbflPp6gKrINuRBYGV3N4E5wAoUIJji81WIkBqgYg1hcPGecExCzGxqVJR4WsTtbwjR2YVqiqUwqsUfGDfBmtD3OSMkqkf6Ug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylum-level taxonomic compositions of prokaryotes in the soil metagenome on the basis of the 338F-533R and 342F-806R amplicon pyrosequencing and the 16S rRNA gene fragments from Illumina metagenomic sequencing. The top 18 abundant phyla from the results of Illumina metagenomic sequencing are listed." path-from-xml="dst05203.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5985460/dst05203.tiff" path-from-xml="dst05203.tiff" target="_blank">View large</a><a section="5985460" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/dst05203.png?Expires=2147483647&Signature=jhUTuA~dCR27NihPgTxle-6A5zdYJT~mRLyvVG3PWWZy1cogj2F2B7NdTLCZ5Ises-sY~y-2WUxNXfEf3cb0MAWAxaDkWr~ABesmLCkCzxPYEmD1quGrcLnFT92sm3ENKNPFmSrPad6iGduDapQEr0HqsxrWdgx0G7Pt9cktJR4NQzRfjwWrtXT47TPaQX0bcpu2QyvuGUQW67TqGaC2LzXC~UjYfNoOiTxCSC4XO2Pl6mUlXKXLc-9IjUmJChbOUXYTSRfNwLfPmgAFurGgjP5hn-CbxnIjtiCb9r2abM3Of~gikf31ZmREtTGs19ISvNiYQSnd8QQL3ZsfslW4vA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5985460&ar=404545&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05203.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylum-level taxonomic compositions of prokaryotes in the soil metagenome on the basis of the 338F-533R and 342F-806R amplicon pyrosequencing and the 16S rRNA gene fragments from Illumina metagenomic sequencing. The top 18 abundant phyla from the results of Illumina metagenomic sequencing are listed.</p></div></div>                        <a id="5985461" scrollto-destination="5985461"></a>
<div id="DST052F4" data-id="DST052F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/m_dst05204.png?Expires=2147483647&Signature=zfZzmtkV2Cc1SSdwWjRVKqOzEGWyMkAIeHKG8GqwOmxJKPwaq31lbmLvRe5w9iYHcm6zWIcNAwWTxmCEC8M7X5-tOVw68IRhwKZQ-O3mMgrYOKhmCdGDWE0O5Q8DTKRYqJjm0gTRhu5DHKoYyVy8piD4GBmGEXJmHVq9nB8pjL0lX7v3MHKJbOIY1ClSIoerPufHpumyVYdu~tWatPEn7oNmHvMy9gGelu9vLlZNzBRjdcrIfO3b~mJMdOadBLqAre70xr-hMtkDsaRw3Dz-CzfiL3iLB5YCZY1poZUNS0K9yZBd8R7yIpY0yfiYQ~R~ijfW8Zn1Lkhg9Ud26GAnpw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of three experiments for rank abundances of phyla. Large circles indicate the rank abundances of each phylum in the three experiments, starting at the rank abundance 1, followed by 6, 11, 16, 21, and 26. Phyla are listed in a clockwise direction according to the rank abundance in the Illumina experiment." path-from-xml="dst05204.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5985461/dst05204.tiff" path-from-xml="dst05204.tiff" target="_blank">View large</a><a section="5985461" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/dst05204.png?Expires=2147483647&Signature=JKBBdWwtSf0dXTOj6OHoiO5ZQsCAfX0Rmu0F2GZvVwSkZykqPXwe9X1o6tByDaYR8yALYD1f1myNAfozZYttyTiufrcKc2~H-5p68su9wZvc9WjADcmu3BKi6h38t5LmxjgOwzB2nALkO0ghFQ6xhK3XMvb9FgPapuyB1Iqc9teSE6vyJJkMjxfokQNWEmJKLLZtlFM-YzlFqsk0WPVpE0rSQmhPG7Gahpo~pb-LFQXj3NPZ4yz8SnPLDyq2XwI7f-rXPDO2KqYVRerISCyS-2jaMlpMhT87~r7U7U3pnUDXXBoc12BMQehWQhjEctgKxwgUmJqssWynnJhZPxz7Pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5985461&ar=404545&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05204.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of three experiments for rank abundances of phyla. Large circles indicate the rank abundances of each phylum in the three experiments, starting at the rank abundance 1, followed by 6, 11, 16, 21, and 26. Phyla are listed in a clockwise direction according to the rank abundance in the Illumina experiment.</p></div></div><div content-id="DST052F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/m_dst05204.png?Expires=2147483647&Signature=zfZzmtkV2Cc1SSdwWjRVKqOzEGWyMkAIeHKG8GqwOmxJKPwaq31lbmLvRe5w9iYHcm6zWIcNAwWTxmCEC8M7X5-tOVw68IRhwKZQ-O3mMgrYOKhmCdGDWE0O5Q8DTKRYqJjm0gTRhu5DHKoYyVy8piD4GBmGEXJmHVq9nB8pjL0lX7v3MHKJbOIY1ClSIoerPufHpumyVYdu~tWatPEn7oNmHvMy9gGelu9vLlZNzBRjdcrIfO3b~mJMdOadBLqAre70xr-hMtkDsaRw3Dz-CzfiL3iLB5YCZY1poZUNS0K9yZBd8R7yIpY0yfiYQ~R~ijfW8Zn1Lkhg9Ud26GAnpw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of three experiments for rank abundances of phyla. Large circles indicate the rank abundances of each phylum in the three experiments, starting at the rank abundance 1, followed by 6, 11, 16, 21, and 26. Phyla are listed in a clockwise direction according to the rank abundance in the Illumina experiment." path-from-xml="dst05204.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5985461/dst05204.tiff" path-from-xml="dst05204.tiff" target="_blank">View large</a><a section="5985461" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/dst05204.png?Expires=2147483647&Signature=JKBBdWwtSf0dXTOj6OHoiO5ZQsCAfX0Rmu0F2GZvVwSkZykqPXwe9X1o6tByDaYR8yALYD1f1myNAfozZYttyTiufrcKc2~H-5p68su9wZvc9WjADcmu3BKi6h38t5LmxjgOwzB2nALkO0ghFQ6xhK3XMvb9FgPapuyB1Iqc9teSE6vyJJkMjxfokQNWEmJKLLZtlFM-YzlFqsk0WPVpE0rSQmhPG7Gahpo~pb-LFQXj3NPZ4yz8SnPLDyq2XwI7f-rXPDO2KqYVRerISCyS-2jaMlpMhT87~r7U7U3pnUDXXBoc12BMQehWQhjEctgKxwgUmJqssWynnJhZPxz7Pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5985461&ar=404545&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05204.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of three experiments for rank abundances of phyla. Large circles indicate the rank abundances of each phylum in the three experiments, starting at the rank abundance 1, followed by 6, 11, 16, 21, and 26. Phyla are listed in a clockwise direction according to the rank abundance in the Illumina experiment.</p></div></div><p>The genus-level taxonomic assignment for the 454 and Illumina reads is depicted in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figure S3</a>. The numbers of reads assigned at the genus level were 9956 (65.65% of the total high-quality 454 reads for the 338F-533R experiment), 20 431(74.22% of the total high-quality 454 reads for the 342F-806R experiment), and 1041 (Illumina read pairs) (<a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/lookup/suppl/doi:10.1093/dnares/dst052/-/DC1" target="_blank">Supple-mentary Table S7</a>). The top 20 abundant genera identified in each experiment accounted for more than 70% of the taxonomically assigned reads in all three experiments. Most of the top 20 most-abundant genera in each experiment (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>) were also categorized as those in the two other experiments, and a total of the top 26 most-abundant genera were used for comparative analysis. Compared with the Illumina metagenomic sequencing data, the 454 amplicon sequencing data led to an underrepresentation of three genera (the genera <em>Subdivision3</em> and <em>Spartobacteria</em> in the <em>Verrucomicrobia</em> phylum, and the genus <em>Zavarzinella</em> in the <em>Planctomycetes</em> phylum) and two genera (<em>Zavarzinella</em> and OD1 in the OD1 phylum) in the 338F-533R and 342F-806R experiments, respectively. Underrepresentation might have also occurred because the 16S rRNA genes from these genera had sequences with more than two mismatches for the 338F or 342F primer. The 16S rRNA genes in certain genera (e.g. <em>Conexibacter</em> and <em>Zavarzinella</em>) were underrepresented in the 342F-806R or 338F-533R data when compared with the Illumina metagenomic sequencing data, and their underrepresentation could not be explained by the primer mismatches. The formation of (a) specific secondary structure(s) or differences in the accuracies of the taxonomic assignments for the 454 and Illumina reads might have been responsible for the underrepresentation. The Spearman's correlations calculated for the abundance of the 26 genera between each pair of experiments were: 0.37 (Illumina versus 338F-533R), 0.71 (Illumina versus 342F-806R), and 0.60 (338F-533R versus 342F-806R). Comparison at the genus-levels indicated that the taxonomic composition inferred by the 338F-533R amplicon sequencing correlated weakly with that by the Illumina metagenomic sequencing. In contrast, the taxonomic composition by the 342F-806R amplicon sequencing correlated very well with that by the Illumina metagenomic sequencing, indicating that our universal primers can amplify a broad range of taxa with a low bias.</p><p>Since the Illumina sequencing of 16S rRNA gene fragments in the metagenome produced no chimera or bias that is associated with the initial PCR step of amplicon sequencing, the taxonomic composition of the microbial community based on this sequencing method is highly reliable.<sup><a reveal-id="DST052C41" data-open="DST052C41" class="link link-ref link-reveal xref-bibr">41</a></sup> However, the inference of taxonomic composition in a microbial community obtained using the metagenomic 16S rRNA gene fragments requires ultra-deep sequencing since such fragments represent only a very small fraction of the metagenome.<sup><a reveal-id="DST052C42" data-open="DST052C42" class="link link-ref link-reveal xref-bibr">42</a></sup> Indeed, our use of the Illumina GA IIx platform for metagenome sequencing of the soil microbial community gave rise to only thousands of reads as 16S rRNA gene fragments from &gt;10 million total reads. This observation implies that the quantification of relative abundance in a microbial community will be reliable for major taxa but unsuitable for minor taxa. Furthermore, the metagenomic 16S rRNA gene fragments corresponded randomly to a number of different regions within the 16S rRNA genes, indicating the difficulty in conducting a detailed comparison of metagenomic 16S rRNA gene fragments (e.g. construction of a phylogenetic tree). To characterize the taxonomic composition of a microbial community including its minor taxa at low cost, the amplification of 16S rRNA genes using appropriate universal primers is necessary. The taxonomic composition inferred by the 454 amplicon sequencing with our non-degenerate universal primer set correlated well with that obtained by Illumina metagenomic sequencing, indicating that our universal primers can characterize microbial communities without substantial bias and with low possibility to amplify eukaryotic rRNA genes. Although we did not carry out additional experimental validation of our novel and non-degenerate universal primer set by using other environmental samples rich in eukaryotic and mitochondrial rRNA genes, the results in this study indicate that our primer set is highly expected to be applicable to the analysis of many diverse microbial communities.</p>                            <h2 scrollto-destination=5985464 id="5985464" class="section-title" data-legacy-id=s4>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=5985466 id="5985466" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by a Grant-in-Aid for Scientific Research on Innovative Area ‘Genome Science’, Ministry of Education, Culture, Sports, Science and Technology in Japan, and a Grant-in-Aid for the Institute for Bioinformatics Research and Development, the Japan Science and Technology Agency (BIRD-JST).</p>                            <h2 scrollto-destination=5985468 id="5985468" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Dr T. Oomori at the Ehime Research Institute of Agriculture, Forestry, and Fisheries, Matsuyama, Japan for his permitting us to use the Institute soil sample for this study.</p>                            <h2 scrollto-destination=5985470 id="5985470" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST052C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hamady</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Knight</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Microbial community profiling for human microbiome projects: tools, techniques, and challenges,</div>, <div class="source">Genome Res.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">1141</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microbial%20community%20profiling%20for%20human%20microbiome%20projects%3A%20tools%2C%20techniques%2C%20and%20challenges%2C&amp;author=M.%20Hamady&amp;author=R.%20Knight&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=1141-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.085464.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.085464.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.085464.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19383763" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rappe</div> <div class="given-names">M.S.</div></div>,  <div class="name"><div class="surname">Giovannoni</div> <div class="given-names">S.J.</div></div>. </div><div class="article-title">The uncultured microbial majority</div>, <div class="source">Annu. Rev. Microbiol.</div> , <div class="year">2003</div>, vol. <div class="volume">57</div> (pg. <div class="fpage">369</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20uncultured%20microbial%20majority&amp;author=M.S.%20Rappe&amp;author=S.J.%20Giovannoni&amp;publication_year=2003&amp;journal=Annu.%20Rev.%20Microbiol.&amp;volume=57&amp;pages=369-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.micro.57.030502.090759" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.micro.57.030502.090759" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.micro.57.030502.090759"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14527284" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hamady</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Walker</div> <div class="given-names">J.J.</div></div>,  <div class="name"><div class="surname">Harris</div> <div class="given-names">J.K.</div></div>,  <div class="name"><div class="surname">Gold</div> <div class="given-names">N.J.</div></div>,  <div class="name"><div class="surname">Knight</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Error-correcting barcoded primers for pyrosequencing hundreds of samples in multiplex</div>, <div class="source">Nat. Methods</div> , <div class="year">2008</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">235</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Error-correcting%20barcoded%20primers%20for%20pyrosequencing%20hundreds%20of%20samples%20in%20multiplex&amp;author=M.%20Hamady&amp;author=J.J.%20Walker&amp;author=J.K.%20Harris&amp;author=N.J.%20Gold&amp;author=R.%20Knight&amp;publication_year=2008&amp;journal=Nat.%20Methods&amp;volume=5&amp;pages=235-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1184" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1184" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1184"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18264105" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mori</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Maruyama</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Kurokawa</div> <div class="given-names">K.</div></div>. </div><div class="article-title">VITCOMIC: visualization tool for taxonomic compositions of microbial communities based on 16S rRNA gene sequences</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">332</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=VITCOMIC%3A%20visualization%20tool%20for%20taxonomic%20compositions%20of%20microbial%20communities%20based%20on%2016S%20rRNA%20gene%20sequences&amp;author=H.%20Mori&amp;author=F.%20Maruyama&amp;author=K.%20Kurokawa&amp;publication_year=2010&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=332" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-11-332" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-11-332" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-11-332"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20565810" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Turnbaugh</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Hamady</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yatsunenko</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">A core gut microbiome in obese and lean twins</div>, <div class="source">Nature</div> , <div class="year">2009</div>, vol. <div class="volume">457</div> (pg. <div class="fpage">480</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20core%20gut%20microbiome%20in%20obese%20and%20lean%20twins&amp;author=P.J.%20Turnbaugh&amp;author=M.%20Hamady&amp;author=T.%20Yatsunenko&amp;publication_year=2009&amp;journal=Nature&amp;volume=457&amp;pages=480-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07540" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07540" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07540"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19043404" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Margulies</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Egholm</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Altman</div> <div class="given-names">W.E.</div></div>, et al. </div><div class="article-title">Genome sequencing in microfabricated high-density picolitre reactors</div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">437</div> (pg. <div class="fpage">376</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequencing%20in%20microfabricated%20high-density%20picolitre%20reactors&amp;author=M.%20Margulies&amp;author=M.%20Egholm&amp;author=W.E.%20Altman&amp;publication_year=2005&amp;journal=Nature&amp;volume=437&amp;pages=376-80" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16056220" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kuczynski</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lauber</div> <div class="given-names">C.L.</div></div>,  <div class="name"><div class="surname">Walters</div> <div class="given-names">W.A.</div></div>, et al. </div><div class="article-title">Experimental and analytical tools for studying the human microbiome</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">47</div>-<div class="lpage">58</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Experimental%20and%20analytical%20tools%20for%20studying%20the%20human%20microbiome&amp;author=J.%20Kuczynski&amp;author=C.L.%20Lauber&amp;author=W.A.%20Walters&amp;publication_year=2012&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=13&amp;pages=47-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3129" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3129" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3129"> </span></div> </div></div></div></div></div><div content-id="DST052C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Lozupone</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Hamady</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Bushman</div> <div class="given-names">F.D.</div></div>,  <div class="name"><div class="surname">Knight</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Short pyrosequencing reads suffice for accurate microbial community analysis</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> pg. <div class="fpage">e120</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Short%20pyrosequencing%20reads%20suffice%20for%20accurate%20microbial%20community%20analysis&amp;author=Z.%20Liu&amp;author=C.%20Lozupone&amp;author=M.%20Hamady&amp;author=F.D.%20Bushman&amp;author=R.%20Knight&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=e120" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm541" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm541" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm541"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17881377" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Qian</div> <div class="given-names">P.Y.</div></div>. </div><div class="article-title">Conservative fragments in bacterial 16S rRNA genes and primer design for 16S ribosomal DNA amplicons in metagenomic studies</div>, <div class="source">PLoS ONE</div> , <div class="year">2009</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e7401</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conservative%20fragments%20in%20bacterial%2016S%20rRNA%20genes%20and%20primer%20design%20for%2016S%20ribosomal%20DNA%20amplicons%20in%20metagenomic%20studies&amp;author=Y.%20Wang&amp;author=P.Y.%20Qian&amp;publication_year=2009&amp;journal=PLoS%20ONE&amp;volume=4&amp;pages=e7401" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0007401" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0007401" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0007401"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19816594" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baker</div> <div class="given-names">G.C.</div></div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">J.J.</div></div>,  <div class="name"><div class="surname">Cowan</div> <div class="given-names">D.A.</div></div>. </div><div class="article-title">Review and re-analysis of domain-specific 16S primers</div>, <div class="source">J. Microbiol. Methods</div> , <div class="year">2003</div>, vol. <div class="volume">55</div> (pg. <div class="fpage">541</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Review%20and%20re-analysis%20of%20domain-specific%2016S%20primers&amp;author=G.C.%20Baker&amp;author=J.J.%20Smith&amp;author=D.A.%20Cowan&amp;publication_year=2003&amp;journal=J.%20Microbiol.%20Methods&amp;volume=55&amp;pages=541-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mimet.2003.08.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mimet.2003.08.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mimet.2003.08.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14607398" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">S.W.</div></div>,  <div class="name"><div class="surname">Suda</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Robustness of gut microbiota of healthy adults in response to probiotic intervention revealed by high-throughput pyrosequencing</div>, <div class="source">DNA Res.</div> , <div class="year">2013</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">241</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Robustness%20of%20gut%20microbiota%20of%20healthy%20adults%20in%20response%20to%20probiotic%20intervention%20revealed%20by%20high-throughput%20pyrosequencing&amp;author=S.W.%20Kim&amp;author=W.%20Suda&amp;author=S.%20Kim&amp;publication_year=2013&amp;journal=DNA%20Res.&amp;volume=20&amp;pages=241-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23571675" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huse</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Dethlefsen</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Huber</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Mark</div> <div class="given-names">W.D.</div></div>,  <div class="name"><div class="surname">Relman</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Sogin</div> <div class="given-names">M.L.</div></div>. </div><div class="article-title">Exploring microbial diversity and taxonomy using SSU rRNA hypervariable tag sequencing</div>, <div class="source">PLoS Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e1000255</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploring%20microbial%20diversity%20and%20taxonomy%20using%20SSU%20rRNA%20hypervariable%20tag%20sequencing&amp;author=S.M.%20Huse&amp;author=L.%20Dethlefsen&amp;author=J.A.%20Huber&amp;author=W.D.%20Mark&amp;author=D.A.%20Relman&amp;author=M.L.%20Sogin&amp;publication_year=2008&amp;journal=PLoS%20Genet.&amp;volume=4&amp;pages=e1000255" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000255" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000255" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000255"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19023400" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Auguet</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Barberan</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Casamayor</div> <div class="given-names">E.O.</div></div>. </div><div class="article-title">Global ecological patterns in uncultured <em>Archaea</em></div>, <div class="source">ISME J.</div> , <div class="year">2010</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">182</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20ecological%20patterns%20in%20uncultured%20Archaea&amp;author=J.C.%20Auguet&amp;author=A.%20Barberan&amp;author=E.O.%20Casamayor&amp;publication_year=2010&amp;journal=ISME%20J.&amp;volume=4&amp;pages=182-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2009.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2009.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2009.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19847207" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Swan</div> <div class="given-names">B.K.</div></div>,  <div class="name"><div class="surname">Ehrhardt</div> <div class="given-names">C.J.</div></div>,  <div class="name"><div class="surname">Reifel</div> <div class="given-names">K.M.</div></div>,  <div class="name"><div class="surname">Moreno</div> <div class="given-names">L.I.</div></div>,  <div class="name"><div class="surname">Valentine</div> <div class="given-names">D.L.</div></div>. </div><div class="article-title">Archaeal and bacterial communities respond differently to environmental gradients in anoxic sediments of a California hypersaline lake, the Salton Sea</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2010</div>, vol. <div class="volume">76</div> (pg. <div class="fpage">757</div>-<div class="lpage">68</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Archaeal%20and%20bacterial%20communities%20respond%20differently%20to%20environmental%20gradients%20in%20anoxic%20sediments%20of%20a%20California%20hypersaline%20lake%2C%20the%20Salton%20Sea&amp;author=B.K.%20Swan&amp;author=C.J.%20Ehrhardt&amp;author=K.M.%20Reifel&amp;author=L.I.%20Moreno&amp;author=D.L.%20Valentine&amp;publication_year=2010&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=76&amp;pages=757-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.02409-09" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.02409-09" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.02409-09"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19948847" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bates</div> <div class="given-names">S.T.</div></div>,  <div class="name"><div class="surname">Berg-Lyons</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Caporaso</div> <div class="given-names">J.G.</div></div>,  <div class="name"><div class="surname">Walters</div> <div class="given-names">W.A.</div></div>,  <div class="name"><div class="surname">Knight</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Fierer</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Examining the global distribution of dominant archaeal populations in soil</div>, <div class="source">ISME J.</div> , <div class="year">2011</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">908</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Examining%20the%20global%20distribution%20of%20dominant%20archaeal%20populations%20in%20soil&amp;author=S.T.%20Bates&amp;author=D.%20Berg-Lyons&amp;author=J.G.%20Caporaso&amp;author=W.A.%20Walters&amp;author=R.%20Knight&amp;author=N.%20Fierer&amp;publication_year=2011&amp;journal=ISME%20J.&amp;volume=5&amp;pages=908-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2010.171" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2010.171" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2010.171"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21085198" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Walters</div> <div class="given-names">W.A.</div></div>,  <div class="name"><div class="surname">Caporaso</div> <div class="given-names">J.G.</div></div>,  <div class="name"><div class="surname">Lauber</div> <div class="given-names">C.L.</div></div>,  <div class="name"><div class="surname">Berg-Lyons</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Fierer</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Knight</div> <div class="given-names">R.</div></div>. </div><div class="article-title">PrimerProspector: de novo design and taxonomic analysis of barcoded polymerase chain reaction primers</div>, <div class="source">Bioinformatics</div> , <div class="year">2011</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">1159</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PrimerProspector%3A%20de%20novo%20design%20and%20taxonomic%20analysis%20of%20barcoded%20polymerase%20chain%20reaction%20primers&amp;author=W.A.%20Walters&amp;author=J.G.%20Caporaso&amp;author=C.L.%20Lauber&amp;author=D.%20Berg-Lyons&amp;author=N.%20Fierer&amp;author=R.%20Knight&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=1159-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr087" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr087" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr087"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21349862" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Klindworth</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Pruesse</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Schweer</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Evaluation of general 16S ribosomal RNA gene PCR primers for classical and next-generation sequencing-based diversity studies</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2012</div>, vol. <div class="volume">41</div> pg. <div class="fpage">e1</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evaluation%20of%20general%2016S%20ribosomal%20RNA%20gene%20PCR%20primers%20for%20classical%20and%20next-generation%20sequencing-based%20diversity%20studies&amp;author=A.%20Klindworth&amp;author=E.%20Pruesse&amp;author=T.%20Schweer&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=41&amp;pages=e1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks808" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks808" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks808"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22933715" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huys</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Vanhoutte</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Joossens</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Coamplification of eukaryotic DNA with 16S rRNA gene-based PCR primers: possible consequences for population fingerprinting of complex microbial communities</div>, <div class="source">Curr. Microbiol.</div> , <div class="year">2008</div>, vol. <div class="volume">56</div> (pg. <div class="fpage">553</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coamplification%20of%20eukaryotic%20DNA%20with%2016S%20rRNA%20gene-based%20PCR%20primers%3A%20possible%20consequences%20for%20population%20fingerprinting%20of%20complex%20microbial%20communities&amp;author=G.%20Huys&amp;author=T.%20Vanhoutte&amp;author=M.%20Joossens&amp;publication_year=2008&amp;journal=Curr.%20Microbiol.&amp;volume=56&amp;pages=553-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00284-008-9122-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00284-008-9122-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00284-008-9122-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18301945" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Prosdocimi</div> <div class="given-names">E.M.</div></div>,  <div class="name"><div class="surname">Novati</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bruno</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Errors in ribosomal sequence datasets generated using PCR-coupled “panbacterial’ pyrosequencing, and the establishment of an improved approach</div>, <div class="source">Mol. Cell Probes</div> , <div class="year">2013</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">65</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Errors%20in%20ribosomal%20sequence%20datasets%20generated%20using%20PCR-coupled%20%E2%80%9Cpanbacterial%E2%80%99%20pyrosequencing%2C%20and%20the%20establishment%20of%20an%20improved%20approach&amp;author=E.M.%20Prosdocimi&amp;author=S.%20Novati&amp;author=R.%20Bruno&amp;publication_year=2013&amp;journal=Mol.%20Cell%20Probes&amp;volume=27&amp;pages=65-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mcp.2012.07.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mcp.2012.07.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mcp.2012.07.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22824825" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Galkiewicz</div> <div class="given-names">J.P.</div></div>,  <div class="name"><div class="surname">Kellogg</div> <div class="given-names">C.A.</div></div>. </div><div class="article-title">Cross-kingdom amplification using bacteria-specific primers: complications for studies of coral microbial ecology</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2008</div>, vol. <div class="volume">74</div> (pg. <div class="fpage">7828</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cross-kingdom%20amplification%20using%20bacteria-specific%20primers%3A%20complications%20for%20studies%20of%20coral%20microbial%20ecology&amp;author=J.P.%20Galkiewicz&amp;author=C.A.%20Kellogg&amp;publication_year=2008&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=74&amp;pages=7828-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.01303-08" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.01303-08" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.01303-08"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18931299" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="collab">The Human Microbiome Project Consortium</div><div class="article-title">A framework for human microbiome research</div>, <div class="source">Nature</div> , <div class="year">2012</div>, vol. <div class="volume">486</div> (pg. <div class="fpage">215</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11209" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11209" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11209"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22699610" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kumar</div> <div class="given-names">P.S.</div></div>,  <div class="name"><div class="surname">Brooker</div> <div class="given-names">M.R.</div></div>,  <div class="name"><div class="surname">Dowd</div> <div class="given-names">S.E.</div></div>,  <div class="name"><div class="surname">Camerlengo</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Target region selection is a critical determinant of community fingerprints generated by 16S pyrosequencing</div>, <div class="source">PLoS ONE</div> , <div class="year">2011</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e20956</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Target%20region%20selection%20is%20a%20critical%20determinant%20of%20community%20fingerprints%20generated%20by%2016S%20pyrosequencing&amp;author=P.S.%20Kumar&amp;author=M.R.%20Brooker&amp;author=S.E.%20Dowd&amp;author=T.%20Camerlengo&amp;publication_year=2011&amp;journal=PLoS%20ONE&amp;volume=6&amp;pages=e20956" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0020956" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0020956" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0020956"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21738596" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">de Lillo</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ashley</div> <div class="given-names">F.P.</div></div>,  <div class="name"><div class="surname">Palmer</div> <div class="given-names">R.M.</div></div>, et al. </div><div class="article-title">Novel subgingival bacterial phylotypes detected using multiple universal polymerase chain reaction primer sets</div>, <div class="source">Oral Microbiol. Immunol.</div> , <div class="year">2006</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">61</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20subgingival%20bacterial%20phylotypes%20detected%20using%20multiple%20universal%20polymerase%20chain%20reaction%20primer%20sets&amp;author=A.%20de%20Lillo&amp;author=F.P.%20Ashley&amp;author=R.M.%20Palmer&amp;publication_year=2006&amp;journal=Oral%20Microbiol.%20Immunol.&amp;volume=21&amp;pages=61-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/omi.2006.21.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fomi.2006.21.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fomi.2006.21.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16390343" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bennett</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Solexa Ltd</div>, <div class="source">Pharmacogenomics</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">433</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Solexa%20Ltd&amp;author=S.%20Bennett&amp;publication_year=2004&amp;journal=Pharmacogenomics&amp;volume=5&amp;pages=433-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1517/14622416.5.4.433" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1517%2F14622416.5.4.433" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1517%2F14622416.5.4.433"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15165179" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lagesen</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Hallin</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Rodland</div> <div class="given-names">E.A.</div></div>,  <div class="name"><div class="surname">Staerfeldt</div> <div class="given-names">H.H.</div></div>,  <div class="name"><div class="surname">Rognes</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ussery</div> <div class="given-names">D.W.</div></div>. </div><div class="article-title">RNAmmer: consistent and rapid annotation of ribosomal RNA genes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">3100</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNAmmer%3A%20consistent%20and%20rapid%20annotation%20of%20ribosomal%20RNA%20genes&amp;author=K.%20Lagesen&amp;author=P.%20Hallin&amp;author=E.A.%20Rodland&amp;author=H.H.%20Staerfeldt&amp;author=T.%20Rognes&amp;author=D.W.%20Ussery&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=3100-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm160" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm160" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm160"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17452365" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yarza</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Richter</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Peplies</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">The All-Species Living Tree project: a 16S rRNA-based phylogenetic tree of all sequenced type strains</div>, <div class="source">Syst. Appl. Microbiol.</div> , <div class="year">2008</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">241</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20All-Species%20Living%20Tree%20project%3A%20a%2016S%20rRNA-based%20phylogenetic%20tree%20of%20all%20sequenced%20type%20strains&amp;author=P.%20Yarza&amp;author=M.%20Richter&amp;author=J.%20Peplies&amp;publication_year=2008&amp;journal=Syst.%20Appl.%20Microbiol.&amp;volume=31&amp;pages=241-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.syapm.2008.07.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.syapm.2008.07.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.syapm.2008.07.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18692976" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Acinas</div> <div class="given-names">S.G.</div></div>,  <div class="name"><div class="surname">Marcelino</div> <div class="given-names">L.A.</div></div>,  <div class="name"><div class="surname">Klepac-Ceraj</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Polz</div> <div class="given-names">M.F.</div></div>. </div><div class="article-title">Divergence and redundancy of 16S rRNA sequences in genomes with multiple <em>rrn</em> operons</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2004</div>, vol. <div class="volume">186</div> (pg. <div class="fpage">2629</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Divergence%20and%20redundancy%20of%2016S%20rRNA%20sequences%20in%20genomes%20with%20multiple%20rrn%20operons&amp;author=S.G.%20Acinas&amp;author=L.A.%20Marcelino&amp;author=V.%20Klepac-Ceraj&amp;author=M.F.%20Polz&amp;publication_year=2004&amp;journal=J.%20Bacteriol.&amp;volume=186&amp;pages=2629-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.186.9.2629-2635.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.186.9.2629-2635.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.186.9.2629-2635.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15090503" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Katoh</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Toh</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Improved accuracy of multiple ncRNA alignment by incorporating structural information into a MAFFT-based framework</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">212</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20accuracy%20of%20multiple%20ncRNA%20alignment%20by%20incorporating%20structural%20information%20into%20a%20MAFFT-based%20framework&amp;author=K.%20Katoh&amp;author=H.%20Toh&amp;publication_year=2008&amp;journal=BMC%20Bioinformatics&amp;volume=9&amp;pages=212" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-9-212" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-9-212" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-9-212"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18439255" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cole</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Cardenas</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">The Ribosomal Database Project: improved alignments and new tools for rRNA analysis</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">D141</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Ribosomal%20Database%20Project%3A%20improved%20alignments%20and%20new%20tools%20for%20rRNA%20analysis&amp;author=J.R.%20Cole&amp;author=Q.%20Wang&amp;author=E.%20Cardenas&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=D141-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn879" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn879" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn879"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19004872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cannone</div> <div class="given-names">J.J.</div></div>,  <div class="name"><div class="surname">Subramanian</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Schnare</div> <div class="given-names">M.N.</div></div>, et al. </div><div class="article-title">The comparative RNA web (CRW) site: an online database of comparative sequence and structure information for ribosomal, intron, and other RNAs</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2002</div>, vol. <div class="volume">3</div> pg. <div class="fpage">2</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20comparative%20RNA%20web%20(CRW)%20site%3A%20an%20online%20database%20of%20comparative%20sequence%20and%20structure%20information%20for%20ribosomal%2C%20intron%2C%20and%20other%20RNAs&amp;author=J.J.%20Cannone&amp;author=S.%20Subramanian&amp;author=M.N.%20Schnare&amp;publication_year=2002&amp;journal=BMC%20Bioinformatics&amp;volume=3&amp;pages=2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-3-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-3-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-3-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11869452" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Claesson</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">O'Sullivan</div> <div class="given-names">O.</div></div>, et al. </div><div class="article-title">Comparison of two next-generation sequencing technologies for resolving highly complex microbiota composition using tandem variable 16S rRNA gene regions</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2010</div>, vol. <div class="volume">38</div> pg. <div class="fpage">e200</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20two%20next-generation%20sequencing%20technologies%20for%20resolving%20highly%20complex%20microbiota%20composition%20using%20tandem%20variable%2016S%20rRNA%20gene%20regions&amp;author=M.J.%20Claesson&amp;author=Q.%20Wang&amp;author=O.%20O'Sullivan&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=e200" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq873" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq873" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq873"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20880993" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Shimodaira</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Miyasaka</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Detection of <em>bphAa</em> gene expression of <em>Rhodococcus</em> sp. strain RHA1 in soil using a new method of RNA preparation from soil</div>, <div class="source">Biosci. Biotechnol. Biochem.</div> , <div class="year">2008</div>, vol. <div class="volume">72</div> (pg. <div class="fpage">694</div>-<div class="lpage">701</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20of%20bphAa%20gene%20expression%20of%20Rhodococcus%20sp.%20strain%20RHA1%20in%20soil%20using%20a%20new%20method%20of%20RNA%20preparation%20from%20soil&amp;author=Y.%20Wang&amp;author=J.%20Shimodaira&amp;author=T.%20Miyasaka&amp;publication_year=2008&amp;journal=Biosci.%20Biotechnol.%20Biochem.&amp;volume=72&amp;pages=694-701" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1271/bbb.70493" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1271%2Fbbb.70493" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1271%2Fbbb.70493"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18323657" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schloss</div> <div class="given-names">P.D.</div></div>,  <div class="name"><div class="surname">Westcott</div> <div class="given-names">S.L.</div></div>,  <div class="name"><div class="surname">Ryabin</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Introducing mothur: open-source, platform-independent, community-supported software for describing and comparing microbial communities</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2009</div>, vol. <div class="volume">75</div> (pg. <div class="fpage">7537</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Introducing%20mothur%3A%20open-source%2C%20platform-independent%2C%20community-supported%20software%20for%20describing%20and%20comparing%20microbial%20communities&amp;author=P.D.%20Schloss&amp;author=S.L.%20Westcott&amp;author=T.%20Ryabin&amp;publication_year=2009&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=75&amp;pages=7537-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.01541-09" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.01541-09" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.01541-09"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19801464" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">E.W.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">Basic local alignment search tool</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1990</div>, vol. <div class="volume">215</div> (pg. <div class="fpage">403</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.W.%20Myers&amp;author=D.J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol.&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Itoh</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nakase</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Occurrence of introns in the 16S rRNA genes of members of the genus <em>Thermoproteus</em></div>, <div class="source">Arch. Microbiol.</div> , <div class="year">1998</div>, vol. <div class="volume">170</div> (pg. <div class="fpage">155</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Occurrence%20of%20introns%20in%20the%2016S%20rRNA%20genes%20of%20members%20of%20the%20genus%20Thermoproteus&amp;author=T.%20Itoh&amp;author=K.%20Suzuki&amp;author=T.%20Nakase&amp;publication_year=1998&amp;journal=Arch.%20Microbiol.&amp;volume=170&amp;pages=155-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s002030050628" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs002030050628" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs002030050628"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9683654" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mao</div> <div class="given-names">D.P.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">C.Y.</div></div>,  <div class="name"><div class="surname">Quan</div> <div class="given-names">Z.X.</div></div>. </div><div class="article-title">Coverage evaluation of universal bacterial primers using the metagenomic datasets</div>, <div class="source">BMC Microbiol.</div> , <div class="year">2012</div>, vol. <div class="volume">12</div> pg. <div class="fpage">66</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coverage%20evaluation%20of%20universal%20bacterial%20primers%20using%20the%20metagenomic%20datasets&amp;author=D.P.%20Mao&amp;author=Q.%20Zhou&amp;author=C.Y.%20Chen&amp;author=Z.X.%20Quan&amp;publication_year=2012&amp;journal=BMC%20Microbiol.&amp;volume=12&amp;pages=66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2180-12-66" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2180-12-66" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2180-12-66"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22554309" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Frank</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Reich</div> <div class="given-names">C.I.</div></div>,  <div class="name"><div class="surname">Sharma</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Weisbaum</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Wilson</div> <div class="given-names">B.A.</div></div>,  <div class="name"><div class="surname">Olsen</div> <div class="given-names">G.J.</div></div>. </div><div class="article-title">Critical evaluation of two primers commonly used for amplification of bacterial 16S rRNA genes</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2008</div>, vol. <div class="volume">74</div> (pg. <div class="fpage">2461</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Critical%20evaluation%20of%20two%20primers%20commonly%20used%20for%20amplification%20of%20bacterial%2016S%20rRNA%20genes&amp;author=J.A.%20Frank&amp;author=C.I.%20Reich&amp;author=S.%20Sharma&amp;author=J.S.%20Weisbaum&amp;author=B.A.%20Wilson&amp;author=G.J.%20Olsen&amp;publication_year=2008&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=74&amp;pages=2461-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.02272-07" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.02272-07" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.02272-07"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18296538" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schneider</div> <div class="given-names">T.D.</div></div>,  <div class="name"><div class="surname">Stephens</div> <div class="given-names">R.M.</div></div>. </div><div class="article-title">Sequence logos: a new way to display consensus sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1990</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">6097</div>-<div class="lpage">100</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20logos%3A%20a%20new%20way%20to%20display%20consensus%20sequences&amp;author=T.D.%20Schneider&amp;author=R.M.%20Stephens&amp;publication_year=1990&amp;journal=Nucleic%20Acids%20Res.&amp;volume=18&amp;pages=6097-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/18.20.6097" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F18.20.6097" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F18.20.6097"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2172928" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">King</div> <div class="given-names">G.M.</div></div>,  <div class="name"><div class="surname">Judd</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Kuske</div> <div class="given-names">C.R.</div></div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Analysis of stomach and gut microbiomes of the eastern oyster (<em>Crassostrea virginica</em>) from coastal Louisiana</div>, <div class="source">PLoS ONE</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e51475</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20stomach%20and%20gut%20microbiomes%20of%20the%20eastern%20oyster%20(Crassostrea%20virginica)%20from%20coastal%20Louisiana&amp;author=G.M.%20King&amp;author=C.%20Judd&amp;author=C.R.%20Kuske&amp;author=C.%20Smith&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e51475" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0051475" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0051475" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0051475"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23251548" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gribaldo</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Brochier-Armanet</div> <div class="given-names">C.</div></div>. </div><div class="article-title">The origin and evolution of <em>Archaea</em>: a state of the art</div>, <div class="source">Philos. Trans. R. Soc. Lond. B Biol. Sci.</div> , <div class="year">2006</div>, vol. <div class="volume">361</div> (pg. <div class="fpage">1007</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20origin%20and%20evolution%20of%20Archaea%3A%20a%20state%20of%20the%20art&amp;author=S.%20Gribaldo&amp;author=C.%20Brochier-Armanet&amp;publication_year=2006&amp;journal=Philos.%20Trans.%20R.%20Soc.%20Lond.%20B%20Biol.%20Sci.&amp;volume=361&amp;pages=1007-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rstb.2006.1841" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frstb.2006.1841" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frstb.2006.1841"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16754611" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hamp</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Jones</div> <div class="given-names">W.J.</div></div>,  <div class="name"><div class="surname">Fodor</div> <div class="given-names">A.A.</div></div>. </div><div class="article-title">Effects of experimental choices and analysis noise on surveys of the “rare biosphere”</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2009</div>, vol. <div class="volume">75</div> (pg. <div class="fpage">3263</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effects%20of%20experimental%20choices%20and%20analysis%20noise%20on%20surveys%20of%20the%20%E2%80%9Crare%20biosphere%E2%80%9D&amp;author=T.J.%20Hamp&amp;author=W.J.%20Jones&amp;author=A.A.%20Fodor&amp;publication_year=2009&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=75&amp;pages=3263-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.01931-08" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.01931-08" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.01931-08"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19270149" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST052C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST052C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yooseph</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nealson</div> <div class="given-names">K.H.</div></div>,  <div class="name"><div class="surname">Rusch</div> <div class="given-names">D.B.</div></div>, et al. </div><div class="article-title">Genomic and functional adaptation in surface ocean planktonic prokaryotes</div>, <div class="source">Nature</div> , <div class="year">2010</div>, vol. <div class="volume">468</div> (pg. <div class="fpage">60</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20and%20functional%20adaptation%20in%20surface%20ocean%20planktonic%20prokaryotes&amp;author=S.%20Yooseph&amp;author=K.H.%20Nealson&amp;author=D.B.%20Rusch&amp;publication_year=2010&amp;journal=Nature&amp;volume=468&amp;pages=60-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature09530" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature09530" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature09530"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21048761" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Prof. Masahira Hattori</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=404545&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/dst052-dst052supp_table7.doc?Expires=2147483647&amp;Signature=aYn6SeZh0PI5V31RMdJXMKQM-f-0sHnAnFfYPMElQQCmt4PQl4zQAiet4LyQTGogcD9484Cg9gnjpjzNJ05NgUCqRrhmuT-nWKq1Rq-yzllmVbZ9oMH1TIrICxOfGddJf24o5vMldjQoBzbvIA5yLwsBW5AVTN65KnvbAa3m~jlg1DeO3tNiKe3P5Dntz2IFBTjOOxtWbEgTa9cf~6ViW1sLP7hpWVmOwmFGPKVpeGc7xYJtbftNavBsluYYx9710zU53J~xSwKr8M3ktJYjFsUg4jsdtthnfEbi4qbMPTdHJayqko1GnjKZDPduy6kOgh3-LE7EV~1yfq5XZfGPig__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-12</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/dst052-dst052supp_table6.doc?Expires=2147483647&amp;Signature=MuDLDOTBwv9Zl04HlykoGXNIcUFjeApsNfFLVYownAHriqpLL0a-kLxGfJcmAKqpHtvldDlIXdx~SFR4ckhhk8SVI98RF6E-jqsI2Hx0pKjQg4LTZl-WbImJRPPefWP~69foEa2nejozrl-iHCclEQMEQ5~6Hkd1or~KNn4GExTLJEHPIzLDxzZ7Rk2dtFa5jn1T4QMzJNtx5R~0GVKEmzVfaef7u2p4rgvj1n1D2jmEtSfDyibFRmZ5wMXpLePEak0CaWilUEvZ1SmM0DIVQRIWDf0-6sQNorSFWs3PwTtubWKrQ~mi7JuWDjOAZoG7Xs6XXX59o~uPhwhysmDWOg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-11</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/dst052-dst052supp_table5.xls?Expires=2147483647&amp;Signature=m87dmvcHC~jQxHR9WzNp4DWOpJrOlgOc4P7koBYAiLgAJ-QmR-yczp2uOKIcMD18Eb7YWFvUffH7lFZDiwHTGx38gVMD~VCeWYiueI6~NdqqvEDp0zS~7IMa03A2sydSj7ViWIHHWnUsZ8m43cgNCfo-XhO4KPvhmxpLant6rtsPmPK3CTi9bY8AIM4159JCZjODwQR6OyMQjKoO5SVjEIBoouPbHgLB~hAd7MQASoIS-JLICSfoxUJ1qUM6ecf0jRV4E5d5iPq~IT7fSwams1T-GKx36SqYLCo-oVQLUEHgEWs76VZyud8tuMPaXoy0gh~zVrqA9VMsRKRJK24fPg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-10</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/dst052-dst052supp_table4.xls?Expires=2147483647&amp;Signature=S7qf6zafLuPwwGeIaPcqORNFtVOroch26llgAqWVY39I6YwQOZUcUFgibjXds59m0V8WEu3Ssd324NviPPnPc-ALpaPcGsgQJNVO2miy8ogktxPnhYB8zwdAYrfuLfnEiU7AY1fkk6QQQTn0MmukqUX4a5tcNXdGkY5sMAeYaAIQt3L22F5NjLsHKM4Ufx5V0VhdNx~5Mddb1t3JLtpm1wuxHWK6MCKth8GgJ6M0RI2eyon~IyORRdoNo0QWY9OJUlhi24HylezJ1ZhhzsrWQFDXFaoRc1vyU4VRIpqGpQzgmQreOkoN8E3Z0MUhGchcBZFcRfoUD2RuPLfJ0MdUeg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-9</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/dst052-dst052supp_table3.xls?Expires=2147483647&amp;Signature=Yt68WW2Y7mC7512hv1QEMGkk6CeAOu3oosXUkSPTfQv47g543xmXObu76OHRfaxuTmiH1yLLv-TAUtQUzshFhEit9M1usad49bmcPaSpJO51KCpprM2b4Qz6XIiGMBSGqq-Us4LlvLcIn1DAjVbKy8dM8RJRwyHkeY6QvjA3Q0HEqqatq0bmg-M8CLM--ISXS2rhRUFy7Y6tu6Bt~BmzWgA2Ms7OXCDA2w3hvqzNcGZO0E-OR1Qk-CaVkcGk37UosjsCvYiDyOoXfHnLhKW~4wmSXY6Xoi7UUtw6mKzn65krE9bwtkFxdeR6DWMpGd2VPJ7YbFZd4h-pe0GdYgFQWA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/dst052-dst052supp_table2.doc?Expires=2147483647&amp;Signature=2sxXBF0Ige2K0P4IRIfFRgDFQGnNIYyZbyeYL0FG7bnAQZHPAxWuAjCPeV4aZhh1Eg3kedLY4GZQeYl9RlOw4ay0MyN~31x~1BufM9Eju8d-RgBrz3c6GIAlhf8-tbpOmrqMpPZesLIG15MqeQu9AjN35~cNbWM1~2KGOnHGep9A4aPqAwSxBVhxYLHHyQ1DId7jE8lIfgAdZHxRuPg9aNbisfw-7BBTGR-96T~aNXBCf9Kcwvw6gZRDpKUSNvAlwhK1~tCaMXir2LPqd~IqiOtq9wEHBMu0G1LPank3m~hccWODpyJC2Lf4BjJmRMJFp7ezmZHDqVVVT-~QLlOdKQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/dst052-dst052supp_table1.doc?Expires=2147483647&amp;Signature=kia0vyfai5izuxosXAVoIpw-yzKEX~i8uNY26~fDJ6pAjFRaL0MXRWlIM~7iE8TXBA6MZJAPFvUURUi17dPRqZtlIjkWfmv7FxXkNM229XWvAjbYgXtCT82BBSBjGxMPUrd3wIV8ZiuAMIC4RHdQEdnZEuoycePWtFfHwdxIL7TLkdnyB4teaCpWeufz-iHuYw6ek2IdBoibbF-xRws096hwRTb8wl9zWfT7oc9FxTkyohREpTEKo9MA6mc6kHl7Zt7Q4WGHFisfBDOtS8nJhM6IGEXyZCwYyIhEoqsdMWkQC618rVCL-AMgn1gR41cV3xHfUewmTzPUni~uZ~K0cA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/dst052-dst052supp_fig4.pdf?Expires=2147483647&amp;Signature=1TvIuCEtNwW0igSM0frnK5m6fer8dg-KQ-mM1v2KcELRO3Lwtn7eTBBCCGCNISQttIopIAFIOA6KSQTZoyAALdLNEy~X78V0hrOT1rv7ZJkXBIpO~y-muHqaSIdNFkrLW8WLKfaSDYF~5kzC6G1SM1emIhhv4p9aaDEuE4tfiAlvrX~m~wkkD2kYWxnOTjk-E-eH9h9Rzbvpnqpns2yx4D4VgqPj1HyDp3nTcHG5xzTSzvEj3P9oQ9TQZR5t7ZVG74w4RTVcXEoeDj-M4OQ9UaOkQi5PcCTpBJw-LF1WEP23pz5jVjChHoUmOwTLtPXmJwFrEkiC-AVuOkkCX3LWfA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/dst052-dst052supp_fig3.pdf?Expires=2147483647&amp;Signature=eMH2HqFPpVkc7xmi0OcfC72BlWU444ynpKA5TufNKZr8tFhMy8pciUBZo~243sezQKmYoHv58Xm2cz8mq5dzC-Bw5fLLNkL-FmAC6GdAGnRMFtDQ-BhkXTN1Vc22I59j~xGq1SM4GvSw7Frk6uBWexJnhzluVJwyWQLL8BQSW89yo27bYgoHwa1PpWhUjYYKnVNFmfYjXlvh4jPoM~5bRMMA23uc31IzyIdijoHOVnoGkW7ZfMMDG3GQk-lTxh2YsuPaD36LoKK1sbus-pTEeuWRKYNFgiaW2pkptOZlJ0lYzuW1Jg82aEl7wPCxVZp6NnQUa~OxEsMCbqv0qfOO1Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/dst052-dst052supp_fig2.pdf?Expires=2147483647&amp;Signature=2H2IhwquY60zkvsTsoKi5j6xgq-tkolMxFgJ9D4NJx9ilxfHIt1-qWmWbtjJ9TsEtsZWf1eVvEUo2xL0xQvUkekzBwRjvNkxStGRQFoRKv-7U7ppNt1JbfwBLF5VOd1LznzJPweV04ZTOwH9Oo-onDBB3xbE3OQ02A~r0dJVCUZd4fln~2sFynjQLZVvnORA-Va982gb0LRcdVKe0kmf1CO4rGgq93lpBcV7IQYOXd6UZcsbg3AEaEpZeY5s6CmBvxtjyK8ylBewP-~pwYqFCq-rT5fhXmqg27OGewFJ4o2HyjEkWel95QF78QYsNFjCp56V5-Z5KJa9B2pmSpo7MQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/dst052-dst052supp_fig1.pdf?Expires=2147483647&amp;Signature=ioHkho7CTLSlzpw0B1ocUlb4dDAhDyAU3OfAEouGDADBCkqHW-2quZV~g00iIcRWYQW1IGNUNX~Z15Uy3p2EMS2bSiBho7IqZQMPBn9FxPYjB2r43y~aEg7VTFCD8oKc3Uyup5OAFgSQ2RhicYxN8RA4v2YK9c5QgUTo0vlcgpq-hbjRWqjjfyLSs9kUa6EGktXmbgq4-iDPUUriC8KcHil6ZPcauoQ5g2PSSi8p7u4VP5AHiv9KFmaQoOUCLiYtqJRyt0YKqgD520xOHgY9LcCieeQ3RCy1Mvz9pxb4b5w73yP5r-0pjLRB~UD7kjSt7KYEdRaSrF5LVli5jtl9iA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst052/2/dst052-dst052supp.xlsx?Expires=2147483647&amp;Signature=soYKiBwJlDR0unUgJzMv5ikXE1VbI6ZZLTGC9NUWjzY49VcV0A-O7GkA2jpThcjV7hcDsPhRq49SDnlpEyVzffLMZgL4d4ubl0grnL4-Iob7UH~rv2OJy0vH0fo1B0JAI2sCxTuxaz6eBIMdUltbZgOLAZdkn22zcpTggYcmUlZ4TNttuxr0vBtUq3mzLMrsJk-T~GEVaumvCJ837LFt4uRv2B-qHJQP~zX93r6rd80-DR0aR5E-TtMbiMz8rwXCXvEVChbpFRmItv3z26C7arsh1NPZvTeOXlim5PnFzcnjUO9XzDfIDuqAgDfVCInAFeeXZEwypBw8lgfYv6F6zQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - xlsx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">866</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897500010&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">40</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst052"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="404545"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897500010&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f2%2f217%2f404545">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000339897500010&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (40)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F2%2F217%2F404545" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/404545'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Design and Experimental Application of a Novel Non-Degenerate Universal Primer Set that Amplifies Prokaryotic 16S rRNA Genes with a Low Possibility to Amplify Eukaryotic rRNA Genes" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst052");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f2%2f217%2f404545">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

