

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/19/1/23/484949","@type":"ScholarlyArticle","name":"Discovery and mapping of a new expressed sequence tag-single nucleotide polymorphism and simple sequence repeat panel for large-scale genetic studies and breeding of  Theobroma cacao  L.","datePublished":"2011-12-30","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/19/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2012-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr039","keywords":["<italic>Theobroma cacao</italic>","genetic map","SNP","molecular marker"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/19/1/23/484949","author":[{"name":"Allegre, Mathilde","affiliation":"1  CIRAD, UMR 1334 AGAP ,  TA 108/03-34398, Montpellier Cedex 5 ,  France","@type":"Person"},{"name":"Argout, Xavier","affiliation":"1  CIRAD, UMR 1334 AGAP ,  TA 108/03-34398, Montpellier Cedex 5 ,  France","@type":"Person"},{"name":"Boccara, Michel","affiliation":"1  CIRAD, UMR 1334 AGAP ,  TA 108/03-34398, Montpellier Cedex 5 ,  France    2  University of the West Indies ,  Cocoa Research Unit (CRU) ,  St Augustine ,  Trinidad and Tobago","@type":"Person"},{"name":"Fouet, Olivier","affiliation":"1  CIRAD, UMR 1334 AGAP ,  TA 108/03-34398, Montpellier Cedex 5 ,  France","@type":"Person"},{"name":"Roguet, Yolande","affiliation":"1  CIRAD, UMR 1334 AGAP ,  TA 108/03-34398, Montpellier Cedex 5 ,  France","@type":"Person"},{"name":"Bérard, Aurélie","affiliation":"3  INRA, UR 1279 Etude du Polymorphisme des Génomes Végétaux, CEA Institut de Génomique, Centre National de Génotypage ,  2, rue Gaston Crémieux, CP5724, 91057 Evry ,  France","@type":"Person"},{"name":"Thévenin, Jean Marc","affiliation":"4  CIRAD, Biological Systems Department, UPR Bioagresseurs, 97387 Kourou Cedex ,  French Guiana","@type":"Person"},{"name":"Chauveau, Aurélie","affiliation":"3  INRA, UR 1279 Etude du Polymorphisme des Génomes Végétaux, CEA Institut de Génomique, Centre National de Génotypage ,  2, rue Gaston Crémieux, CP5724, 91057 Evry ,  France","@type":"Person"},{"name":"Rivallan, Ronan","affiliation":"1  CIRAD, UMR 1334 AGAP ,  TA 108/03-34398, Montpellier Cedex 5 ,  France","@type":"Person"},{"name":"Clement, Didier","affiliation":"1  CIRAD, UMR 1334 AGAP ,  TA 108/03-34398, Montpellier Cedex 5 ,  France    5  Comissão Executiva de Planejamento da Lavoura Cacaueira (CEPLAC) ,  Km 22 Rod. Ilheus Itabuna, Cx. postal 07, Itabuna 45600-00, Bahia ,  Brazil","@type":"Person"},{"name":"Courtois, Brigitte","affiliation":"1  CIRAD, UMR 1334 AGAP ,  TA 108/03-34398, Montpellier Cedex 5 ,  France","@type":"Person"},{"name":"Gramacho, Karina","affiliation":"5  Comissão Executiva de Planejamento da Lavoura Cacaueira (CEPLAC) ,  Km 22 Rod. Ilheus Itabuna, Cx. postal 07, Itabuna 45600-00, Bahia ,  Brazil","@type":"Person"},{"name":"Boland-Augé, Anne","affiliation":"3  INRA, UR 1279 Etude du Polymorphisme des Génomes Végétaux, CEA Institut de Génomique, Centre National de Génotypage ,  2, rue Gaston Crémieux, CP5724, 91057 Evry ,  France","@type":"Person"},{"name":"Tahi, Mathias","affiliation":"6  Centre National de la Recherche Agronomique (CNRA) ,  B.P. 808, Divo ,  Côte d'Ivoire","@type":"Person"},{"name":"Umaharan, Pathmanathan","affiliation":"2  University of the West Indies ,  Cocoa Research Unit (CRU) ,  St Augustine ,  Trinidad and Tobago","@type":"Person"},{"name":"Brunel, Dominique","affiliation":"3  INRA, UR 1279 Etude du Polymorphisme des Génomes Végétaux, CEA Institut de Génomique, Centre National de Génotypage ,  2, rue Gaston Crémieux, CP5724, 91057 Evry ,  France","@type":"Person"},{"name":"Lanaud, Claire","affiliation":"1  CIRAD, UMR 1334 AGAP ,  TA 108/03-34398, Montpellier Cedex 5 ,  France","@type":"Person"}],"description":"Abstract.  Theobroma cacao is an economically important tree of several tropical countries. Its genetic improvement is essential to provide protection against m","pageStart":"23","pageEnd":"35","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/1/1/m_cover.gif?Expires=2147483647&Signature=xaAwlSToJroE5fWhfpGVCZn3xBSUKDlAAPYmgFaxsPT0BgXTXG9-yQxL4l3AKE5wtgupce1wlaTfijDCbCNDlVGvYjkhmD3GS0NMSPpbODUXpCTjNxwFXDmBeN5OfByVvY3xkCQn2mF1aHyDolyWFOspDXh12ZtmIZRK9PLEHjkwQfY-Eas8nYfYkGa6~vbOZFKvI1uD0JmISa23PnI0WZCbFOdjoUUwej1eTi~tkPpOA3LkPujrg4-jHpsR-oUGENthSCY2yqVn9BonjQV~GPEsWHXi42gNXSDjuz3TQihicX2R9czd-tzArP3EMYC4WDAfSV92EKM50IA19YA9GA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Discovery and mapping of a new expressed sequence tag-single nucleotide polymorphism and simple sequence repeat panel for large-scale genetic studies and breeding of  Theobroma cacao  L.","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/1/1/m_cover.gif?Expires=2147483647&Signature=xaAwlSToJroE5fWhfpGVCZn3xBSUKDlAAPYmgFaxsPT0BgXTXG9-yQxL4l3AKE5wtgupce1wlaTfijDCbCNDlVGvYjkhmD3GS0NMSPpbODUXpCTjNxwFXDmBeN5OfByVvY3xkCQn2mF1aHyDolyWFOspDXh12ZtmIZRK9PLEHjkwQfY-Eas8nYfYkGa6~vbOZFKvI1uD0JmISa23PnI0WZCbFOdjoUUwej1eTi~tkPpOA3LkPujrg4-jHpsR-oUGENthSCY2yqVn9BonjQV~GPEsWHXi42gNXSDjuz3TQihicX2R9czd-tzArP3EMYC4WDAfSV92EKM50IA19YA9GA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Discovery and mapping of a new expressed sequence tag-single nucleotide polymorphism and simple sequence repeat panel for large-scale genetic studies and breeding of  Theobroma cacao  L." />
    <meta property="og:description" content="Abstract.  Theobroma cacao is an economically important tree of several tropical countries. Its genetic improvement is essential to provide protection against m" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/19/1/23/484949" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2011-12-30" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Allegre, Mathilde" /><meta name="citation_author_institution" content="CIRAD, UMR 1334 AGAP, TA 108/03-34398, Montpellier Cedex 5, France" /><meta name="citation_author" content="Argout, Xavier" /><meta name="citation_author_institution" content="CIRAD, UMR 1334 AGAP, TA 108/03-34398, Montpellier Cedex 5, France" /><meta name="citation_author" content="Boccara, Michel" /><meta name="citation_author_institution" content="CIRAD, UMR 1334 AGAP, TA 108/03-34398, Montpellier Cedex 5, France" /><meta name="citation_author_institution" content="University of the West Indies, Cocoa Research Unit (CRU), St Augustine, Trinidad and Tobago" /><meta name="citation_author" content="Fouet, Olivier" /><meta name="citation_author_institution" content="CIRAD, UMR 1334 AGAP, TA 108/03-34398, Montpellier Cedex 5, France" /><meta name="citation_author" content="Roguet, Yolande" /><meta name="citation_author_institution" content="CIRAD, UMR 1334 AGAP, TA 108/03-34398, Montpellier Cedex 5, France" /><meta name="citation_author" content="Bérard, Aurélie" /><meta name="citation_author_institution" content="INRA, UR 1279 Etude du Polymorphisme des Génomes Végétaux, CEA Institut de Génomique, Centre National de Génotypage, 2, rue Gaston Crémieux, CP5724, 91057 Evry, France" /><meta name="citation_author" content="Thévenin, Jean Marc" /><meta name="citation_author_institution" content="CIRAD, Biological Systems Department, UPR Bioagresseurs, 97387 Kourou Cedex, French Guiana" /><meta name="citation_author" content="Chauveau, Aurélie" /><meta name="citation_author_institution" content="INRA, UR 1279 Etude du Polymorphisme des Génomes Végétaux, CEA Institut de Génomique, Centre National de Génotypage, 2, rue Gaston Crémieux, CP5724, 91057 Evry, France" /><meta name="citation_author" content="Rivallan, Ronan" /><meta name="citation_author_institution" content="CIRAD, UMR 1334 AGAP, TA 108/03-34398, Montpellier Cedex 5, France" /><meta name="citation_author" content="Clement, Didier" /><meta name="citation_author_institution" content="CIRAD, UMR 1334 AGAP, TA 108/03-34398, Montpellier Cedex 5, France" /><meta name="citation_author_institution" content="Comissão Executiva de Planejamento da Lavoura Cacaueira (CEPLAC), Km 22 Rod. Ilheus Itabuna, Cx. postal 07, Itabuna 45600-00, Bahia, Brazil" /><meta name="citation_author" content="Courtois, Brigitte" /><meta name="citation_author_institution" content="CIRAD, UMR 1334 AGAP, TA 108/03-34398, Montpellier Cedex 5, France" /><meta name="citation_author" content="Gramacho, Karina" /><meta name="citation_author_institution" content="Comissão Executiva de Planejamento da Lavoura Cacaueira (CEPLAC), Km 22 Rod. Ilheus Itabuna, Cx. postal 07, Itabuna 45600-00, Bahia, Brazil" /><meta name="citation_author" content="Boland-Augé, Anne" /><meta name="citation_author_institution" content="INRA, UR 1279 Etude du Polymorphisme des Génomes Végétaux, CEA Institut de Génomique, Centre National de Génotypage, 2, rue Gaston Crémieux, CP5724, 91057 Evry, France" /><meta name="citation_author" content="Tahi, Mathias" /><meta name="citation_author_institution" content="Centre National de la Recherche Agronomique (CNRA), B.P. 808, Divo, Côte d'Ivoire" /><meta name="citation_author" content="Umaharan, Pathmanathan" /><meta name="citation_author_institution" content="University of the West Indies, Cocoa Research Unit (CRU), St Augustine, Trinidad and Tobago" /><meta name="citation_author" content="Brunel, Dominique" /><meta name="citation_author_institution" content="INRA, UR 1279 Etude du Polymorphisme des Génomes Végétaux, CEA Institut de Génomique, Centre National de Génotypage, 2, rue Gaston Crémieux, CP5724, 91057 Evry, France" /><meta name="citation_author" content="Lanaud, Claire" /><meta name="citation_author_institution" content="CIRAD, UMR 1334 AGAP, TA 108/03-34398, Montpellier Cedex 5, France" /><meta name="citation_title" content="Discovery and mapping of a new expressed sequence tag-single nucleotide polymorphism and simple sequence repeat panel for large-scale genetic studies and breeding of Theobroma cacao L." /><meta name="citation_firstpage" content="23" /><meta name="citation_lastpage" content="35" /><meta name="citation_doi" content="10.1093/dnares/dsr039" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="19" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2012/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The genome of Theobroma cacao; Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=101-8; " /><meta name="citation_reference" content="citation_title=Notes on the nomenclature, classification and possible relationships of cocoa populations; Trop. Agricult.;  citation_year=1944;  citation_volume=21;  citation_pages=144-59; " /><meta name="citation_reference" content="citation_title=A meta-QTL analysis of disease resistance traits of Theobroma cacao L.; Mol. Breed.;  citation_year=2009;  citation_volume=24;  citation_pages=361-74; " /><meta name="citation_reference" content="citation_title=A genetic-linkage map of Theobroma cacao L.; Theor. Appl. Genet.;  citation_year=1995;  citation_volume=91;  citation_pages=987-93; " /><meta name="citation_reference" content="citation_title=Theobroma cacao L.: a genetic linkage map and quantitative trait loci analysis; Theor. Appl. Genet.;  citation_year=1996;  citation_volume=93;  citation_pages=205-14; " /><meta name="citation_reference" content="citation_title=A high-density linkage map of Theobroma cacao L.; Theor. Appl. Genet.;  citation_year=2000;  citation_volume=101;  citation_pages=948-55; " /><meta name="citation_reference" content="citation_title=An overview of molecular marker methods for plants; Afr. J. Biotechnol.;  citation_year=2006;  citation_volume=5;  citation_pages=2540-68; " /><meta name="citation_reference" content="citation_title=A new cacao linkage map based on codominant markers: development and integration of 201 new microsatellite markers; Theor. Appl. Genet.;  citation_year=2004;  citation_volume=108;  citation_pages=1151-61; " /><meta name="citation_reference" content="citation_title=Structural characterization and mapping of functional EST-SSR markers in Theobroma cacao; Tree Genet. Genomes;  citation_year=2011;  citation_volume=99;  citation_pages=1-19; " /><meta name="citation_reference" content="citation_title=Large-scale identification and analysis of genome-wide single-nucleotide polymorphisms for mapping in Arabidopsis thaliana; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=1250-7; " /><meta name="citation_reference" content="citation_title=High throughput SNP discovery and genotyping in grapevine (Vitis vinifera L.) by combining a re-sequencing approach and SNPlex technology; BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=424" /><meta name="citation_reference" content="citation_title=High-throughput single nucleotide polymorphism genotyping in wheat (Triticum spp.); Plant Biotechnol. J.;  citation_year=2009;  citation_volume=7;  citation_pages=364-74; " /><meta name="citation_reference" content="citation_title=Automated SNP detection in expressed sequence tags: statistical considerations and application to maritime pine sequences; Plant Mol. Biol.;  citation_year=2004;  citation_volume=54;  citation_pages=461-70; " /><meta name="citation_reference" content="citation_title=Identification and validation of single nucleotide polymorphisms in poplar using publicly expressed sequence tags; J. Integr. Plant Biol.;  citation_year=2005;  citation_volume=47;  citation_pages=1493-9; " /><meta name="citation_reference" content="citation_title=Automated SNP detection from a large collection of white spruce expressed sequences: contributing factors and approaches for the categorization of SNPs; BMC Genomics;  citation_year=2006;  citation_volume=7;  citation_pages=174" /><meta name="citation_reference" content="citation_title=Genome-wide mapping with biallelic markers in Arabidopsis thaliana; Nat. Genet.;  citation_year=1999;  citation_volume=23;  citation_pages=203-7; " /><meta name="citation_reference" content="citation_title=Single nucleotide polymorphism markers for genetic mapping in Drosophila melanogaster; Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=1100-13; " /><meta name="citation_reference" content="citation_title=Analysis and application of European genetic substructure using 300 K SNP information; PLoS Genet.;  citation_year=2008;  citation_volume=4;  citation_pages=e4" /><meta name="citation_reference" content="citation_title=Genes mirror geography within Europe; Nature;  citation_year=2008;  citation_volume=456;  citation_pages=98-101; " /><meta name="citation_reference" content="citation_title=Estimation of the transition/transversion rate bias and species sampling; J. Mol. Evol.;  citation_year=1999;  citation_volume=48;  citation_pages=274-83; " /><meta name="citation_reference" content="citation_title=Phylogeny reconstruction; Mol. Syst.;  citation_year=1990;  citation_volume=411;  citation_pages=501" /><meta name="citation_reference" content="citation_title=Mitochondrial DNA sequences of primates: tempo and mode of evolution; J. Mol. Evol.;  citation_year=1982;  citation_volume=18;  citation_pages=225-39; " /><meta name="citation_reference" content="citation_title=Patterns of nucleotide substitution in pseudogenes and functional genes; J. Mol. Evol.;  citation_year=1982;  citation_volume=18;  citation_pages=360-9; " /><meta name="citation_reference" content="citation_title=Substitution-rate variation among sites and the estimation of transition bias; Mol. Biol. Evol.;  citation_year=1994;  citation_volume=11;  citation_pages=436-42; " /><meta name="citation_reference" content="citation_title=The excess of transitions among nucleotide substitutions: new methods of estimating transition bias underscore its significance; Trends Ecol. Evol. (Amst.);  citation_year=1996;  citation_volume=11;  citation_pages=158-62; " /><meta name="citation_reference" content="citation_title=Single-nucleotide polymorphisms detected in expressed sequence tags of melon (Cucumis melo L.); Genome;  citation_year=2004;  citation_volume=47;  citation_pages=352-60; " /><meta name="citation_reference" content="citation_title=A set of EST-SNPs for map saturation and cultivar identification in melon; BMC Plant Biol.;  citation_year=2009;  citation_volume=9; " /><meta name="citation_reference" content="citation_title=A Brassica rapa linkage map of EST-based SNP markers for identification of candidate genes controlling flowering time and leaf morphological traits; DNA Res.;  citation_year=2009;  citation_volume=16;  citation_pages=311-23; " /><meta name="citation_reference" content="citation_title=EST-derived single nucleotide polymorphism markers for assembling genetic and physical maps of the barley genome; Funct. Integr. Genomics;  citation_year=2008;  citation_volume=8;  citation_pages=223-33; " /><meta name="citation_reference" content="citation_title=Characterisation of single nucleotide polymorphisms in sugarcane ESTs; Theor. Appl. Genet.;  citation_year=2006;  citation_volume=113;  citation_pages=331-43; " /><meta name="citation_reference" content="citation_title=Single nucleotide polymorphisms from Theobroma cacao expressed sequence tags associated with witches’ broom disease in cacao; Genet. Mol. Res.;  citation_year=2009;  citation_volume=8;  citation_pages=799-808; " /><meta name="citation_reference" content="citation_title=Towards the understanding of the cocoa transcriptome: production and analysis of an exhaustive dataset of ESTs of Theobroma cacao L. generated from various tissues and under various conditions; BMC Genomics;  citation_year=2008; " /><meta name="citation_reference" content="citation_title=QualitySNP: a pipeline for detecting single nucleotide polymorphisms and insertions/deletions in EST data from diploid and polyploid species; BMC Bioinformatics;  citation_year=2006;  citation_volume=7;  citation_pages=438" /><meta name="citation_reference" content="citation_title=High-throughput genotyping with the GoldenGate assay in the complex genome of soybean; Theor. Appl. Genet.;  citation_year=2008;  citation_volume=116;  citation_pages=945-52; " /><meta name="citation_reference" content="citation_title=Single nucleotide polymorphism genotyping in polyploid wheat with the Illumina GoldenGate assay; Theor. Appl. Genet.;  citation_year=2009;  citation_volume=119;  citation_pages=507-17; " /><meta name="citation_reference" content="citation_title=Primer3 on the WWW for general users and for biologist programmers; Methods Mol. Biol.;  citation_year=2000;  citation_volume=132;  citation_pages=365-86; " /><meta name="citation_reference" content="citation_publisher=Wageningen, The Netherlands; Joinmap, Software for the Calculation of Genetic Linkage Maps;  citation_year=2006; " /><meta name="citation_reference" content="citation_title=The estimation of map distance from recombination values; Ann. Eugen.;  citation_year=1944;  citation_volume=12;  citation_pages=172-5; " /><meta name="citation_reference" content="citation_title=Mining for single nucleotide polymorphisms and insertions/deletions in maize expressed sequence tag data; Plant Physiol.;  citation_year=2003;  citation_volume=132;  citation_pages=84-91; " /><meta name="citation_reference" content="citation_title=Molecular characterization of an international cacao collection using microsatellite markers; Tree Genet. Genomes;  citation_year=2008;  citation_volume=5;  citation_pages=1-10; " /><meta name="citation_reference" content="citation_title=Genetic diversity and population structure in wild stands of cacao trees (Theobroma cacao L.), in French Guiana; Ann. For. Sci.;  citation_year=2008;  citation_volume=65;  citation_pages=7" /><meta name="citation_reference" content="citation_title=Genetic diversity and structure of managed and semi-natural populations of cocoa (Theobroma cacao) in the Huallaga and Ucayali Valleys of Peru; Ann. Bot.;  citation_year=2006;  citation_volume=98;  citation_pages=647-55; " /><meta name="citation_reference" content="citation_title=Geographic and genetic population differentiation of the Amazonian chocolate tree (Theobroma cacao L.); PLoS ONE;  citation_year=2008;  citation_volume=3;  citation_pages=e3311" /><meta name="citation_reference" content="citation_title=Contribution à l’étude de la domestication de la variété Nacional d'Equateur: recherche de la variété native et de ses ancêtres sauvages; citation_text=PhD Thesis, Montpellier SUPAGRO- ED SIBAGHE- spécialité: biologie intégrative des plantes;  citation_year=2007; " /><meta name="citation_reference" content="citation_title=Incompatibility in Theobroma cacao; Heredity;  citation_year=1955;  citation_volume=9;  citation_pages=69-77; " /><meta name="citation_reference" content="citation_title=Incompatibility in Theobroma cacao; Nature;  citation_year=1958;  citation_volume=181;  citation_pages=279" /><meta name="citation_reference" content="citation_title=A molecular marker-based linkage map of diploid bananas (Musa acuminata); Theor. Appl. Genetics;  citation_year=1993;  citation_volume=87;  citation_pages=517-26; " /><meta name="citation_reference" content="citation_title=Molecular dissection of a quantitative trait locus: a phenylalanine-to-tyrosine substitution in the transmembrane domain of the bovine growth hormone receptor is associated with a major effect on milk yield and composition; Genetics;  citation_year=2003;  citation_volume=163;  citation_pages=253-66; " /><meta name="citation_reference" content="citation_title=Association study designs for complex diseases; Nat. Rev. Genet.;  citation_year=2001;  citation_volume=2;  citation_pages=91-9; " /><meta name="citation_reference" content="citation_title=Candidate lung tumor susceptibility genes identified through whole-genome association analyses in inbred mice; Nat. Genet.;  citation_year=2006;  citation_volume=38;  citation_pages=888-95; " /><meta name="citation_reference" content="citation_title=Fine mapping of a quantitative trait locus for twinning rate using combined linkage and linkage disequilibrium mapping; Genetics;  citation_year=2002;  citation_volume=161;  citation_pages=373-9; " /><meta name="citation_reference" content="citation_title=Mapping quantitative trait loci using naturally occurring genetic variance among commercial inbred lines of maize (Zea mays L.); Genetics;  citation_year=2005;  citation_volume=169;  citation_pages=2267-75; " /><meta name="citation_reference" content="citation_title=Genome-wide association mapping in Arabidopsis identifies previously known flowering time and pathogen resistance genes; PLoS Genet.;  citation_year=2005;  citation_volume=1;  citation_pages=e60" /><meta name="citation_reference" content="citation_title=Linkage disequilibrium among modern sugarcane cultivars; Theor. Appl. Genet.;  citation_year=1999;  citation_volume=99;  citation_pages=1053-60; " /><meta name="citation_reference" content="citation_title=Associations between DNA markers and resistance to diseases in sugarcane and effects of population substructure; Theor. Appl. Genet.;  citation_year=2006;  citation_volume=114;  citation_pages=155-64; " /><meta name="citation_reference" content="citation_title=Association mapping of yield and its components in rice cultivars; Mol. Breed.;  citation_year=2007;  citation_volume=19;  citation_pages=341-56; " /><meta name="citation_reference" content="citation_title=Adding value to cocoa (Theobroma cacao L.) germplasm information with domestication history and admixture mapping; Theor. Appl. Genet.;  citation_year=2007;  citation_volume=114;  citation_pages=877-84; " /><meta name="citation_reference" content="citation_title=A genomewide admixture mapping study for yield factors and morphological traits in a cultivated cocoa (Theobroma cacao L.) population; Tree Genet. Genomes;  citation_year=2009;  citation_volume=5;  citation_pages=329-37; " /><meta name="citation_reference" content="citation_title=Evaluation and utilisation of cacao (Theobroma cacao L.) germplasm at the International Cocoa Genebank, Trinidad; Euphytica;  citation_year=2003;  citation_volume=130;  citation_pages=207-21; " /><meta name="citation_reference" content="citation_author=IBPGR Working Group on Genetics Resources of Cocoa; citation_publisher=Rome, Italy; Genetic Resources of Cocoa;  citation_year=1981;  citation_pages=25" /><meta name="citation_reference" content="citation_title=Development of a marker assisted selection program for cacao; Phytopathology;  citation_year=2007;  citation_volume=97;  citation_pages=1664-9; " /><meta name="citation_reference" content="citation_title=Prediction of total genetic value using genome-wide dense marker maps; Genetics;  citation_year=2001;  citation_volume=157;  citation_pages=1819-29; " /><meta name="citation_reference" content="citation_title=Genomic selection for crop improvement; Crop Sci.;  citation_year=2009;  citation_volume=49;  citation_pages=1-12; " /><meta name="citation_reference" content="citation_title=Prospects for genomewide selection for quantitative traits in maize; Crop Sci.;  citation_year=2007;  citation_volume=47;  citation_pages=1082" /><meta name="citation_reference" content="citation_title=Genomewide selection in oil palm: increasing selection gain per unit time and cost with small populations; Theor. Appl. Genet.;  citation_year=2008;  citation_volume=116;  citation_pages=815-24; " /><meta name="citation_reference" content="citation_title=Genomic selection in plant breeding: from theory to practice; Brief. Funct. Genomic.;  citation_year=2010;  citation_volume=9;  citation_pages=166" /><meta name="citation_reference" content="citation_title=Amélioration génétique du cacaoyer en Côte d'Ivoire-Présentation d'un nouveau schéma de selection;  citation_year=1993;  citation_pages=18-24; " /><meta name="citation_reference" content="citation_title=Le cacaoyer: stratégies de sélection; Plant. Recherche Dével.;  citation_year=1995;  citation_volume=2;  citation_pages=5-18; " /><meta name="citation_reference" content="citation_title=A proposal for cocoa breeding;  citation_year=1996;  citation_pages=17-23; " /><meta name="citation_reference" content="citation_title=Germplasm enhancement for resistance to black pod disease – final stages and achievements;  citation_year=2010;  citation_pages=493-499; " /><meta name="citation_reference" content="citation_title=Second cycle de selection recurrente du cacaoyer (Theobroma cacao L.) en Cote d'Ivoire: bilan a mi-parcours et propositions de sorties varietales;  citation_year=2010;  citation_pages=3-12; " /><meta name="citation_reference" content="citation_title=Cacao breeding in Bahia, Brazil—strategies and results; Crop Breed. Appl. Biotechnol.;  citation_year=2011;  citation_volume=1;  citation_pages=73-81; " /><meta name="citation_reference" content="citation_title=Functional genomics of Cacao; citation_publisher=Academic Press, Londres; Advances in Botanical Research;  citation_year=2010;  citation_volume=vol. 55;  citation_pages=119-77; " /><meta name="citation_reference" content="citation_title=The molecular genetics of crop domestication; Cell;  citation_year=2006;  citation_volume=127;  citation_pages=1309-21; " /><meta name="citation_reference" content="citation_title=Molecular signatures of natural selection; Annu. Rev. Genet.;  citation_year=2005;  citation_volume=39;  citation_pages=197-218; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/19/1/23/1311170/dsr039.pdf" /><meta name="description" content="Abstract.  Theobroma cacao is an economically important tree of several tropical countries. Its genetic improvement is essential to provide protection against m" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/19/1/23/484949" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/19/1/23/484949" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/19/1/23/484949");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsr039';
    </script>
<title>
	Discovery and mapping of a new expressed sequence tag-single nucleotide polymorphism and simple sequence repeat panel for large-scale genetic studies and breeding of Theobroma cacao L. | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./484949" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYWuATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQWxsZWdyZSwgTWF0aGlsZGUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDSVJBRCwgVU1SIDEzMzQgQUdBUCwgVEEgMTA4LzAzLTM0Mzk4LCBNb250cGVsbGllciBDZWRleCA1LCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBcmdvdXQsIFhhdmllciIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNJUkFELCBVTVIgMTMzNCBBR0FQLCBUQSAxMDgvMDMtMzQzOTgsIE1vbnRwZWxsaWVyIENlZGV4IDUsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkJvY2NhcmEsIE1pY2hlbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNJUkFELCBVTVIgMTMzNCBBR0FQLCBUQSAxMDgvMDMtMzQzOTgsIE1vbnRwZWxsaWVyIENlZGV4IDUsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVuaXZlcnNpdHkgb2YgdGhlIFdlc3QgSW5kaWVzLCBDb2NvYSBSZXNlYXJjaCBVbml0IChDUlUpLCBTdCBBdWd1c3RpbmUsIFRyaW5pZGFkIGFuZCBUb2JhZ28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGb3VldCwgT2xpdmllciIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNJUkFELCBVTVIgMTMzNCBBR0FQLCBUQSAxMDgvMDMtMzQzOTgsIE1vbnRwZWxsaWVyIENlZGV4IDUsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlJvZ3VldCwgWW9sYW5kZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNJUkFELCBVTVIgMTMzNCBBR0FQLCBUQSAxMDgvMDMtMzQzOTgsIE1vbnRwZWxsaWVyIENlZGV4IDUsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkLDqXJhcmQsIEF1csOpbGllIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSU5SQSwgVVIgMTI3OSBFdHVkZSBkdSBQb2x5bW9ycGhpc21lIGRlcyBHw6lub21lcyBWw6lnw6l0YXV4LCBDRUEgSW5zdGl0dXQgZGUgR8Opbm9taXF1ZSwgQ2VudHJlIE5hdGlvbmFsIGRlIEfDqW5vdHlwYWdlLCAyLCBydWUgR2FzdG9uIENyw6ltaWV1eCwgQ1A1NzI0LCA5MTA1NyBFdnJ5LCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUaMOpdmVuaW4sIEplYW4gTWFyYyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNJUkFELCBCaW9sb2dpY2FsIFN5c3RlbXMgRGVwYXJ0bWVudCwgVVBSIEJpb2FncmVzc2V1cnMsIDk3Mzg3IEtvdXJvdSBDZWRleCwgRnJlbmNoIEd1aWFuYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNoYXV2ZWF1LCBBdXLDqWxpZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IklOUkEsIFVSIDEyNzkgRXR1ZGUgZHUgUG9seW1vcnBoaXNtZSBkZXMgR8Opbm9tZXMgVsOpZ8OpdGF1eCwgQ0VBIEluc3RpdHV0IGRlIEfDqW5vbWlxdWUsIENlbnRyZSBOYXRpb25hbCBkZSBHw6lub3R5cGFnZSwgMiwgcnVlIEdhc3RvbiBDcsOpbWlldXgsIENQNTcyNCwgOTEwNTcgRXZyeSwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUml2YWxsYW4sIFJvbmFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ0lSQUQsIFVNUiAxMzM0IEFHQVAsIFRBIDEwOC8wMy0zNDM5OCwgTW9udHBlbGxpZXIgQ2VkZXggNSwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2xlbWVudCwgRGlkaWVyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ0lSQUQsIFVNUiAxMzM0IEFHQVAsIFRBIDEwOC8wMy0zNDM5OCwgTW9udHBlbGxpZXIgQ2VkZXggNSwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ29taXNzw6NvIEV4ZWN1dGl2YSBkZSBQbGFuZWphbWVudG8gZGEgTGF2b3VyYSBDYWNhdWVpcmEgKENFUExBQyksIEttIDIyIFJvZC4gSWxoZXVzIEl0YWJ1bmEsIEN4LiBwb3N0YWwgMDcsIEl0YWJ1bmEgNDU2MDAtMDAsIEJhaGlhLCBCcmF6aWwiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDb3VydG9pcywgQnJpZ2l0dGUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDSVJBRCwgVU1SIDEzMzQgQUdBUCwgVEEgMTA4LzAzLTM0Mzk4LCBNb250cGVsbGllciBDZWRleCA1LCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHcmFtYWNobywgS2FyaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ29taXNzw6NvIEV4ZWN1dGl2YSBkZSBQbGFuZWphbWVudG8gZGEgTGF2b3VyYSBDYWNhdWVpcmEgKENFUExBQyksIEttIDIyIFJvZC4gSWxoZXVzIEl0YWJ1bmEsIEN4LiBwb3N0YWwgMDcsIEl0YWJ1bmEgNDU2MDAtMDAsIEJhaGlhLCBCcmF6aWwiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCb2xhbmQtQXVnw6ksIEFubmUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJTlJBLCBVUiAxMjc5IEV0dWRlIGR1IFBvbHltb3JwaGlzbWUgZGVzIEfDqW5vbWVzIFbDqWfDqXRhdXgsIENFQSBJbnN0aXR1dCBkZSBHw6lub21pcXVlLCBDZW50cmUgTmF0aW9uYWwgZGUgR8Opbm90eXBhZ2UsIDIsIHJ1ZSBHYXN0b24gQ3LDqW1pZXV4LCBDUDU3MjQsIDkxMDU3IEV2cnksIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRhaGksIE1hdGhpYXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZW50cmUgTmF0aW9uYWwgZGUgbGEgUmVjaGVyY2hlIEFncm9ub21pcXVlIChDTlJBKSwgQi5QLiA4MDgsIERpdm8sIEPDtHRlIGQnSXZvaXJlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVW1haGFyYW4sIFBhdGhtYW5hdGhhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVuaXZlcnNpdHkgb2YgdGhlIFdlc3QgSW5kaWVzLCBDb2NvYSBSZXNlYXJjaCBVbml0IChDUlUpLCBTdCBBdWd1c3RpbmUsIFRyaW5pZGFkIGFuZCBUb2JhZ28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCcnVuZWwsIERvbWluaXF1ZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IklOUkEsIFVSIDEyNzkgRXR1ZGUgZHUgUG9seW1vcnBoaXNtZSBkZXMgR8Opbm9tZXMgVsOpZ8OpdGF1eCwgQ0VBIEluc3RpdHV0IGRlIEfDqW5vbWlxdWUsIENlbnRyZSBOYXRpb25hbCBkZSBHw6lub3R5cGFnZSwgMiwgcnVlIEdhc3RvbiBDcsOpbWlldXgsIENQNTcyNCwgOTEwNTcgRXZyeSwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGFuYXVkLCBDbGFpcmUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDSVJBRCwgVU1SIDEzMzQgQUdBUCwgVEEgMTA4LzAzLTM0Mzk4LCBNb250cGVsbGllciBDZWRleCA1LCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkRpc2NvdmVyeSBhbmQgbWFwcGluZyBvZiBhIG5ldyBleHByZXNzZWQgc2VxdWVuY2UgdGFnLXNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbSBhbmQgc2ltcGxlIHNlcXVlbmNlIHJlcGVhdCBwYW5lbCBmb3IgbGFyZ2Utc2NhbGUgZ2VuZXRpYyBzdHVkaWVzIGFuZCBicmVlZGluZyBvZiBUaGVvYnJvbWEgY2FjYW8gTC4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3IwMzkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMi8wMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgb2YgVGhlb2Jyb21hIGNhY2FvOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTEwMS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob3RlcyBvbiB0aGUgbm9tZW5jbGF0dXJlLCBjbGFzc2lmaWNhdGlvbiBhbmQgcG9zc2libGUgcmVsYXRpb25zaGlwcyBvZiBjb2NvYSBwb3B1bGF0aW9uczsgVHJvcC4gQWdyaWN1bHQuOyAgY2l0YXRpb25feWVhcj0xOTQ0OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MTQ0LTU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG1ldGEtUVRMIGFuYWx5c2lzIG9mIGRpc2Vhc2UgcmVzaXN0YW5jZSB0cmFpdHMgb2YgVGhlb2Jyb21hIGNhY2FvIEwuOyBNb2wuIEJyZWVkLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTM2MS03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBnZW5ldGljLWxpbmthZ2UgbWFwIG9mIFRoZW9icm9tYSBjYWNhbyBMLjsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT05MTsgIGNpdGF0aW9uX3BhZ2VzPTk4Ny05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlb2Jyb21hIGNhY2FvIEwuOiBhIGdlbmV0aWMgbGlua2FnZSBtYXAgYW5kIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2NpIGFuYWx5c2lzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTkzOyAgY2l0YXRpb25fcGFnZXM9MjA1LTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGhpZ2gtZGVuc2l0eSBsaW5rYWdlIG1hcCBvZiBUaGVvYnJvbWEgY2FjYW8gTC47IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTAxOyAgY2l0YXRpb25fcGFnZXM9OTQ4LTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBvdmVydmlldyBvZiBtb2xlY3VsYXIgbWFya2VyIG1ldGhvZHMgZm9yIHBsYW50czsgQWZyLiBKLiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MjU0MC02ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBuZXcgY2FjYW8gbGlua2FnZSBtYXAgYmFzZWQgb24gY29kb21pbmFudCBtYXJrZXJzOiBkZXZlbG9wbWVudCBhbmQgaW50ZWdyYXRpb24gb2YgMjAxIG5ldyBtaWNyb3NhdGVsbGl0ZSBtYXJrZXJzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwODsgIGNpdGF0aW9uX3BhZ2VzPTExNTEtNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cnVjdHVyYWwgY2hhcmFjdGVyaXphdGlvbiBhbmQgbWFwcGluZyBvZiBmdW5jdGlvbmFsIEVTVC1TU1IgbWFya2VycyBpbiBUaGVvYnJvbWEgY2FjYW87IFRyZWUgR2VuZXQuIEdlbm9tZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9OTk7ICBjaXRhdGlvbl9wYWdlcz0xLTE5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYXJnZS1zY2FsZSBpZGVudGlmaWNhdGlvbiBhbmQgYW5hbHlzaXMgb2YgZ2Vub21lLXdpZGUgc2luZ2xlLW51Y2xlb3RpZGUgcG9seW1vcnBoaXNtcyBmb3IgbWFwcGluZyBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0xMjUwLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2ggdGhyb3VnaHB1dCBTTlAgZGlzY292ZXJ5IGFuZCBnZW5vdHlwaW5nIGluIGdyYXBldmluZSAoVml0aXMgdmluaWZlcmEgTC4pIGJ5IGNvbWJpbmluZyBhIHJlLXNlcXVlbmNpbmcgYXBwcm9hY2ggYW5kIFNOUGxleCB0ZWNobm9sb2d5OyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTQyNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2gtdGhyb3VnaHB1dCBzaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc20gZ2Vub3R5cGluZyBpbiB3aGVhdCAoVHJpdGljdW0gc3BwLik7IFBsYW50IEJpb3RlY2hub2wuIEouOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0zNjQtNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUF1dG9tYXRlZCBTTlAgZGV0ZWN0aW9uIGluIGV4cHJlc3NlZCBzZXF1ZW5jZSB0YWdzOiBzdGF0aXN0aWNhbCBjb25zaWRlcmF0aW9ucyBhbmQgYXBwbGljYXRpb24gdG8gbWFyaXRpbWUgcGluZSBzZXF1ZW5jZXM7IFBsYW50IE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz00NjEtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIGFuZCB2YWxpZGF0aW9uIG9mIHNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbXMgaW4gcG9wbGFyIHVzaW5nIHB1YmxpY2x5IGV4cHJlc3NlZCBzZXF1ZW5jZSB0YWdzOyBKLiBJbnRlZ3IuIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTQ3OyAgY2l0YXRpb25fcGFnZXM9MTQ5My05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BdXRvbWF0ZWQgU05QIGRldGVjdGlvbiBmcm9tIGEgbGFyZ2UgY29sbGVjdGlvbiBvZiB3aGl0ZSBzcHJ1Y2UgZXhwcmVzc2VkIHNlcXVlbmNlczogY29udHJpYnV0aW5nIGZhY3RvcnMgYW5kIGFwcHJvYWNoZXMgZm9yIHRoZSBjYXRlZ29yaXphdGlvbiBvZiBTTlBzOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTE3NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIG1hcHBpbmcgd2l0aCBiaWFsbGVsaWMgbWFya2VycyBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0yMDMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtIG1hcmtlcnMgZm9yIGdlbmV0aWMgbWFwcGluZyBpbiBEcm9zb3BoaWxhIG1lbGFub2dhc3RlcjsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xMTAwLTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBhbmQgYXBwbGljYXRpb24gb2YgRXVyb3BlYW4gZ2VuZXRpYyBzdWJzdHJ1Y3R1cmUgdXNpbmcgMzAwIEsgU05QIGluZm9ybWF0aW9uOyBQTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9ZTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lcyBtaXJyb3IgZ2VvZ3JhcGh5IHdpdGhpbiBFdXJvcGU7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00NTY7ICBjaXRhdGlvbl9wYWdlcz05OC0xMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzdGltYXRpb24gb2YgdGhlIHRyYW5zaXRpb24vdHJhbnN2ZXJzaW9uIHJhdGUgYmlhcyBhbmQgc3BlY2llcyBzYW1wbGluZzsgSi4gTW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT00ODsgIGNpdGF0aW9uX3BhZ2VzPTI3NC04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW55IHJlY29uc3RydWN0aW9uOyBNb2wuIFN5c3QuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTQxMTsgIGNpdGF0aW9uX3BhZ2VzPTUwMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pdG9jaG9uZHJpYWwgRE5BIHNlcXVlbmNlcyBvZiBwcmltYXRlczogdGVtcG8gYW5kIG1vZGUgb2YgZXZvbHV0aW9uOyBKLiBNb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTgyOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MjI1LTM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QYXR0ZXJucyBvZiBudWNsZW90aWRlIHN1YnN0aXR1dGlvbiBpbiBwc2V1ZG9nZW5lcyBhbmQgZnVuY3Rpb25hbCBnZW5lczsgSi4gTW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk4MjsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTM2MC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdWJzdGl0dXRpb24tcmF0ZSB2YXJpYXRpb24gYW1vbmcgc2l0ZXMgYW5kIHRoZSBlc3RpbWF0aW9uIG9mIHRyYW5zaXRpb24gYmlhczsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTQzNi00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGV4Y2VzcyBvZiB0cmFuc2l0aW9ucyBhbW9uZyBudWNsZW90aWRlIHN1YnN0aXR1dGlvbnM6IG5ldyBtZXRob2RzIG9mIGVzdGltYXRpbmcgdHJhbnNpdGlvbiBiaWFzIHVuZGVyc2NvcmUgaXRzIHNpZ25pZmljYW5jZTsgVHJlbmRzIEVjb2wuIEV2b2wuIChBbXN0Lik7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xNTgtNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbmdsZS1udWNsZW90aWRlIHBvbHltb3JwaGlzbXMgZGV0ZWN0ZWQgaW4gZXhwcmVzc2VkIHNlcXVlbmNlIHRhZ3Mgb2YgbWVsb24gKEN1Y3VtaXMgbWVsbyBMLik7IEdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT00NzsgIGNpdGF0aW9uX3BhZ2VzPTM1Mi02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzZXQgb2YgRVNULVNOUHMgZm9yIG1hcCBzYXR1cmF0aW9uIGFuZCBjdWx0aXZhciBpZGVudGlmaWNhdGlvbiBpbiBtZWxvbjsgQk1DIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgQnJhc3NpY2EgcmFwYSBsaW5rYWdlIG1hcCBvZiBFU1QtYmFzZWQgU05QIG1hcmtlcnMgZm9yIGlkZW50aWZpY2F0aW9uIG9mIGNhbmRpZGF0ZSBnZW5lcyBjb250cm9sbGluZyBmbG93ZXJpbmcgdGltZSBhbmQgbGVhZiBtb3JwaG9sb2dpY2FsIHRyYWl0czsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0zMTEtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVTVC1kZXJpdmVkIHNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbSBtYXJrZXJzIGZvciBhc3NlbWJsaW5nIGdlbmV0aWMgYW5kIHBoeXNpY2FsIG1hcHMgb2YgdGhlIGJhcmxleSBnZW5vbWU7IEZ1bmN0LiBJbnRlZ3IuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0yMjMtMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3RlcmlzYXRpb24gb2Ygc2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtcyBpbiBzdWdhcmNhbmUgRVNUczsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTM7ICBjaXRhdGlvbl9wYWdlcz0zMzEtNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbXMgZnJvbSBUaGVvYnJvbWEgY2FjYW8gZXhwcmVzc2VkIHNlcXVlbmNlIHRhZ3MgYXNzb2NpYXRlZCB3aXRoIHdpdGNoZXPigJkgYnJvb20gZGlzZWFzZSBpbiBjYWNhbzsgR2VuZXQuIE1vbC4gUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9Nzk5LTgwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG93YXJkcyB0aGUgdW5kZXJzdGFuZGluZyBvZiB0aGUgY29jb2EgdHJhbnNjcmlwdG9tZTogcHJvZHVjdGlvbiBhbmQgYW5hbHlzaXMgb2YgYW4gZXhoYXVzdGl2ZSBkYXRhc2V0IG9mIEVTVHMgb2YgVGhlb2Jyb21hIGNhY2FvIEwuIGdlbmVyYXRlZCBmcm9tIHZhcmlvdXMgdGlzc3VlcyBhbmQgdW5kZXIgdmFyaW91cyBjb25kaXRpb25zOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVF1YWxpdHlTTlA6IGEgcGlwZWxpbmUgZm9yIGRldGVjdGluZyBzaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc21zIGFuZCBpbnNlcnRpb25zL2RlbGV0aW9ucyBpbiBFU1QgZGF0YSBmcm9tIGRpcGxvaWQgYW5kIHBvbHlwbG9pZCBzcGVjaWVzOyBCTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTQzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2gtdGhyb3VnaHB1dCBnZW5vdHlwaW5nIHdpdGggdGhlIEdvbGRlbkdhdGUgYXNzYXkgaW4gdGhlIGNvbXBsZXggZ2Vub21lIG9mIHNveWJlYW47IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTE2OyAgY2l0YXRpb25fcGFnZXM9OTQ1LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc20gZ2Vub3R5cGluZyBpbiBwb2x5cGxvaWQgd2hlYXQgd2l0aCB0aGUgSWxsdW1pbmEgR29sZGVuR2F0ZSBhc3NheTsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTk7ICBjaXRhdGlvbl9wYWdlcz01MDctMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByaW1lcjMgb24gdGhlIFdXVyBmb3IgZ2VuZXJhbCB1c2VycyBhbmQgZm9yIGJpb2xvZ2lzdCBwcm9ncmFtbWVyczsgTWV0aG9kcyBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTEzMjsgIGNpdGF0aW9uX3BhZ2VzPTM2NS04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPVdhZ2VuaW5nZW4sIFRoZSBOZXRoZXJsYW5kczsgSm9pbm1hcCwgU29mdHdhcmUgZm9yIHRoZSBDYWxjdWxhdGlvbiBvZiBHZW5ldGljIExpbmthZ2UgTWFwczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGVzdGltYXRpb24gb2YgbWFwIGRpc3RhbmNlIGZyb20gcmVjb21iaW5hdGlvbiB2YWx1ZXM7IEFubi4gRXVnZW4uOyAgY2l0YXRpb25feWVhcj0xOTQ0OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTcyLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pbmluZyBmb3Igc2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtcyBhbmQgaW5zZXJ0aW9ucy9kZWxldGlvbnMgaW4gbWFpemUgZXhwcmVzc2VkIHNlcXVlbmNlIHRhZyBkYXRhOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzI7ICBjaXRhdGlvbl9wYWdlcz04NC05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGNoYXJhY3Rlcml6YXRpb24gb2YgYW4gaW50ZXJuYXRpb25hbCBjYWNhbyBjb2xsZWN0aW9uIHVzaW5nIG1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnM7IFRyZWUgR2VuZXQuIEdlbm9tZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTEtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgZGl2ZXJzaXR5IGFuZCBwb3B1bGF0aW9uIHN0cnVjdHVyZSBpbiB3aWxkIHN0YW5kcyBvZiBjYWNhbyB0cmVlcyAoVGhlb2Jyb21hIGNhY2FvIEwuKSwgaW4gRnJlbmNoIEd1aWFuYTsgQW5uLiBGb3IuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NjU7ICBjaXRhdGlvbl9wYWdlcz03IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBkaXZlcnNpdHkgYW5kIHN0cnVjdHVyZSBvZiBtYW5hZ2VkIGFuZCBzZW1pLW5hdHVyYWwgcG9wdWxhdGlvbnMgb2YgY29jb2EgKFRoZW9icm9tYSBjYWNhbykgaW4gdGhlIEh1YWxsYWdhIGFuZCBVY2F5YWxpIFZhbGxleXMgb2YgUGVydTsgQW5uLiBCb3QuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTk4OyAgY2l0YXRpb25fcGFnZXM9NjQ3LTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW9ncmFwaGljIGFuZCBnZW5ldGljIHBvcHVsYXRpb24gZGlmZmVyZW50aWF0aW9uIG9mIHRoZSBBbWF6b25pYW4gY2hvY29sYXRlIHRyZWUgKFRoZW9icm9tYSBjYWNhbyBMLik7IFBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz1lMzMxMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnRyaWJ1dGlvbiDDoCBs4oCZw6l0dWRlIGRlIGxhIGRvbWVzdGljYXRpb24gZGUgbGEgdmFyacOpdMOpIE5hY2lvbmFsIGQnRXF1YXRldXI6IHJlY2hlcmNoZSBkZSBsYSB2YXJpw6l0w6kgbmF0aXZlIGV0IGRlIHNlcyBhbmPDqnRyZXMgc2F1dmFnZXM7IGNpdGF0aW9uX3RleHQ9UGhEIFRoZXNpcywgTW9udHBlbGxpZXIgU1VQQUdSTy0gRUQgU0lCQUdIRS0gc3DDqWNpYWxpdMOpOiBiaW9sb2dpZSBpbnTDqWdyYXRpdmUgZGVzIHBsYW50ZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluY29tcGF0aWJpbGl0eSBpbiBUaGVvYnJvbWEgY2FjYW87IEhlcmVkaXR5OyAgY2l0YXRpb25feWVhcj0xOTU1OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz02OS03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5jb21wYXRpYmlsaXR5IGluIFRoZW9icm9tYSBjYWNhbzsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTU4OyAgY2l0YXRpb25fdm9sdW1lPTE4MTsgIGNpdGF0aW9uX3BhZ2VzPTI3OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbW9sZWN1bGFyIG1hcmtlci1iYXNlZCBsaW5rYWdlIG1hcCBvZiBkaXBsb2lkIGJhbmFuYXMgKE11c2EgYWN1bWluYXRhKTsgVGhlb3IuIEFwcGwuIEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTg3OyAgY2l0YXRpb25fcGFnZXM9NTE3LTI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgZGlzc2VjdGlvbiBvZiBhIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2N1czogYSBwaGVueWxhbGFuaW5lLXRvLXR5cm9zaW5lIHN1YnN0aXR1dGlvbiBpbiB0aGUgdHJhbnNtZW1icmFuZSBkb21haW4gb2YgdGhlIGJvdmluZSBncm93dGggaG9ybW9uZSByZWNlcHRvciBpcyBhc3NvY2lhdGVkIHdpdGggYSBtYWpvciBlZmZlY3Qgb24gbWlsayB5aWVsZCBhbmQgY29tcG9zaXRpb247IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE2MzsgIGNpdGF0aW9uX3BhZ2VzPTI1My02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzb2NpYXRpb24gc3R1ZHkgZGVzaWducyBmb3IgY29tcGxleCBkaXNlYXNlczsgTmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9OTEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2FuZGlkYXRlIGx1bmcgdHVtb3Igc3VzY2VwdGliaWxpdHkgZ2VuZXMgaWRlbnRpZmllZCB0aHJvdWdoIHdob2xlLWdlbm9tZSBhc3NvY2lhdGlvbiBhbmFseXNlcyBpbiBpbmJyZWQgbWljZTsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz04ODgtOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZpbmUgbWFwcGluZyBvZiBhIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2N1cyBmb3IgdHdpbm5pbmcgcmF0ZSB1c2luZyBjb21iaW5lZCBsaW5rYWdlIGFuZCBsaW5rYWdlIGRpc2VxdWlsaWJyaXVtIG1hcHBpbmc7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE2MTsgIGNpdGF0aW9uX3BhZ2VzPTM3My05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBwaW5nIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2NpIHVzaW5nIG5hdHVyYWxseSBvY2N1cnJpbmcgZ2VuZXRpYyB2YXJpYW5jZSBhbW9uZyBjb21tZXJjaWFsIGluYnJlZCBsaW5lcyBvZiBtYWl6ZSAoWmVhIG1heXMgTC4pOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjk7ICBjaXRhdGlvbl9wYWdlcz0yMjY3LTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBtYXBwaW5nIGluIEFyYWJpZG9wc2lzIGlkZW50aWZpZXMgcHJldmlvdXNseSBrbm93biBmbG93ZXJpbmcgdGltZSBhbmQgcGF0aG9nZW4gcmVzaXN0YW5jZSBnZW5lczsgUExvUyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPWU2MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxpbmthZ2UgZGlzZXF1aWxpYnJpdW0gYW1vbmcgbW9kZXJuIHN1Z2FyY2FuZSBjdWx0aXZhcnM7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9OTk7ICBjaXRhdGlvbl9wYWdlcz0xMDUzLTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3NvY2lhdGlvbnMgYmV0d2VlbiBETkEgbWFya2VycyBhbmQgcmVzaXN0YW5jZSB0byBkaXNlYXNlcyBpbiBzdWdhcmNhbmUgYW5kIGVmZmVjdHMgb2YgcG9wdWxhdGlvbiBzdWJzdHJ1Y3R1cmU7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTE0OyAgY2l0YXRpb25fcGFnZXM9MTU1LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3NvY2lhdGlvbiBtYXBwaW5nIG9mIHlpZWxkIGFuZCBpdHMgY29tcG9uZW50cyBpbiByaWNlIGN1bHRpdmFyczsgTW9sLiBCcmVlZC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0zNDEtNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFkZGluZyB2YWx1ZSB0byBjb2NvYSAoVGhlb2Jyb21hIGNhY2FvIEwuKSBnZXJtcGxhc20gaW5mb3JtYXRpb24gd2l0aCBkb21lc3RpY2F0aW9uIGhpc3RvcnkgYW5kIGFkbWl4dHVyZSBtYXBwaW5nOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTExNDsgIGNpdGF0aW9uX3BhZ2VzPTg3Ny04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBnZW5vbWV3aWRlIGFkbWl4dHVyZSBtYXBwaW5nIHN0dWR5IGZvciB5aWVsZCBmYWN0b3JzIGFuZCBtb3JwaG9sb2dpY2FsIHRyYWl0cyBpbiBhIGN1bHRpdmF0ZWQgY29jb2EgKFRoZW9icm9tYSBjYWNhbyBMLikgcG9wdWxhdGlvbjsgVHJlZSBHZW5ldC4gR2Vub21lczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MzI5LTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmFsdWF0aW9uIGFuZCB1dGlsaXNhdGlvbiBvZiBjYWNhbyAoVGhlb2Jyb21hIGNhY2FvIEwuKSBnZXJtcGxhc20gYXQgdGhlIEludGVybmF0aW9uYWwgQ29jb2EgR2VuZWJhbmssIFRyaW5pZGFkOyBFdXBoeXRpY2E7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTMwOyAgY2l0YXRpb25fcGFnZXM9MjA3LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9hdXRob3I9SUJQR1IgV29ya2luZyBHcm91cCBvbiBHZW5ldGljcyBSZXNvdXJjZXMgb2YgQ29jb2E7IGNpdGF0aW9uX3B1Ymxpc2hlcj1Sb21lLCBJdGFseTsgR2VuZXRpYyBSZXNvdXJjZXMgb2YgQ29jb2E7ICBjaXRhdGlvbl95ZWFyPTE5ODE7ICBjaXRhdGlvbl9wYWdlcz0yNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIGEgbWFya2VyIGFzc2lzdGVkIHNlbGVjdGlvbiBwcm9ncmFtIGZvciBjYWNhbzsgUGh5dG9wYXRob2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9OTc7ICBjaXRhdGlvbl9wYWdlcz0xNjY0LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWRpY3Rpb24gb2YgdG90YWwgZ2VuZXRpYyB2YWx1ZSB1c2luZyBnZW5vbWUtd2lkZSBkZW5zZSBtYXJrZXIgbWFwczsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTU3OyAgY2l0YXRpb25fcGFnZXM9MTgxOS0yOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBzZWxlY3Rpb24gZm9yIGNyb3AgaW1wcm92ZW1lbnQ7IENyb3AgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT00OTsgIGNpdGF0aW9uX3BhZ2VzPTEtMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb3NwZWN0cyBmb3IgZ2Vub21ld2lkZSBzZWxlY3Rpb24gZm9yIHF1YW50aXRhdGl2ZSB0cmFpdHMgaW4gbWFpemU7IENyb3AgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00NzsgIGNpdGF0aW9uX3BhZ2VzPTEwODIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWV3aWRlIHNlbGVjdGlvbiBpbiBvaWwgcGFsbTogaW5jcmVhc2luZyBzZWxlY3Rpb24gZ2FpbiBwZXIgdW5pdCB0aW1lIGFuZCBjb3N0IHdpdGggc21hbGwgcG9wdWxhdGlvbnM7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTE2OyAgY2l0YXRpb25fcGFnZXM9ODE1LTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljIHNlbGVjdGlvbiBpbiBwbGFudCBicmVlZGluZzogZnJvbSB0aGVvcnkgdG8gcHJhY3RpY2U7IEJyaWVmLiBGdW5jdC4gR2Vub21pYy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTE2NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFtw6lsaW9yYXRpb24gZ8OpbsOpdGlxdWUgZHUgY2FjYW95ZXIgZW4gQ8O0dGUgZCdJdm9pcmUtUHLDqXNlbnRhdGlvbiBkJ3VuIG5vdXZlYXUgc2Now6ltYSBkZSBzZWxlY3Rpb247ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl9wYWdlcz0xOC0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGUgY2FjYW95ZXI6IHN0cmF0w6lnaWVzIGRlIHPDqWxlY3Rpb247IFBsYW50LiBSZWNoZXJjaGUgRMOpdmVsLjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9NS0xODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBwcm9wb3NhbCBmb3IgY29jb2EgYnJlZWRpbmc7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl9wYWdlcz0xNy0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VybXBsYXNtIGVuaGFuY2VtZW50IGZvciByZXNpc3RhbmNlIHRvIGJsYWNrIHBvZCBkaXNlYXNlIOKAkyBmaW5hbCBzdGFnZXMgYW5kIGFjaGlldmVtZW50czsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3BhZ2VzPTQ5My00OTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlY29uZCBjeWNsZSBkZSBzZWxlY3Rpb24gcmVjdXJyZW50ZSBkdSBjYWNhb3llciAoVGhlb2Jyb21hIGNhY2FvIEwuKSBlbiBDb3RlIGQnSXZvaXJlOiBiaWxhbiBhIG1pLXBhcmNvdXJzIGV0IHByb3Bvc2l0aW9ucyBkZSBzb3J0aWVzIHZhcmlldGFsZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl9wYWdlcz0zLTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DYWNhbyBicmVlZGluZyBpbiBCYWhpYSwgQnJhemls4oCUc3RyYXRlZ2llcyBhbmQgcmVzdWx0czsgQ3JvcCBCcmVlZC4gQXBwbC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTczLTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5jdGlvbmFsIGdlbm9taWNzIG9mIENhY2FvOyBjaXRhdGlvbl9wdWJsaXNoZXI9QWNhZGVtaWMgUHJlc3MsIExvbmRyZXM7IEFkdmFuY2VzIGluIEJvdGFuaWNhbCBSZXNlYXJjaDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT12b2wuIDU1OyAgY2l0YXRpb25fcGFnZXM9MTE5LTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbW9sZWN1bGFyIGdlbmV0aWNzIG9mIGNyb3AgZG9tZXN0aWNhdGlvbjsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjc7ICBjaXRhdGlvbl9wYWdlcz0xMzA5LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgc2lnbmF0dXJlcyBvZiBuYXR1cmFsIHNlbGVjdGlvbjsgQW5udS4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz0xOTctMjE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTkvMS8yMy8xMzExMTcwL2RzcjAzOS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgVGhlb2Jyb21hIGNhY2FvIGlzIGFuIGVjb25vbWljYWxseSBpbXBvcnRhbnQgdHJlZSBvZiBzZXZlcmFsIHRyb3BpY2FsIGNvdW50cmllcy4gSXRzIGdlbmV0aWMgaW1wcm92ZW1lbnQgaXMgZXNzZW50aWFsIHRvIHByb3ZpZGUgcHJvdGVjdGlvbiBhZ2FpbnN0IG0iIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQ4NDk0OSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGSHjogH/vEcbU+s+58FKuyLDKFGRg==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f1%2f23%2f484949" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f1%2f23%2f484949" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=19070" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/19/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/1/1/m_cover.gif?Expires=2147483647&amp;Signature=xaAwlSToJroE5fWhfpGVCZn3xBSUKDlAAPYmgFaxsPT0BgXTXG9-yQxL4l3AKE5wtgupce1wlaTfijDCbCNDlVGvYjkhmD3GS0NMSPpbODUXpCTjNxwFXDmBeN5OfByVvY3xkCQn2mF1aHyDolyWFOspDXh12ZtmIZRK9PLEHjkwQfY-Eas8nYfYkGa6~vbOZFKvI1uD0JmISa23PnI0WZCbFOdjoUUwej1eTi~tkPpOA3LkPujrg4-jHpsR-oUGENthSCY2yqVn9BonjQV~GPEsWHXi42gNXSDjuz3TQihicX2R9czd-tzArP3EMYC4WDAfSV92EKM50IA19YA9GA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/19/1">
                        <div class="volume trailing-comma">Volume 19</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7371005">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7371005">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7371007">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7371007">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7371014">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7371014">2. Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7371051">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7371051">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7371087">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7371087">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7371102">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7371102">5. Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7371104">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7371104">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7371106">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7371106">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7371108">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7371108">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7371110">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7371110">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/19/1/11/484721">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/19/1/37/485401">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Discovery and mapping of a new expressed sequence tag-single nucleotide polymorphism and simple sequence repeat panel for large-scale genetic studies and breeding of <em>Theobroma cacao</em> L.
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mathilde Allegre</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">CIRAD, UMR 1334 AGAP</div>, <div class="addr-line">TA 108/03-34398, Montpellier Cedex 5</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mathilde+Allegre">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Allegre M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Allegre M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Mathilde Allegre</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xavier Argout</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">CIRAD, UMR 1334 AGAP</div>, <div class="addr-line">TA 108/03-34398, Montpellier Cedex 5</div>, <div class="country">France</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Fax. +33 4-67-61-56-05. Email: <a href="mailto:xavier.argout@cirad.fr" target="_blank">xavier.argout@cirad.fr</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xavier+Argout">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Argout X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Argout X&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Xavier Argout</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Michel Boccara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">CIRAD, UMR 1334 AGAP</div>, <div class="addr-line">TA 108/03-34398, Montpellier Cedex 5</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">University of the West Indies</div>, <div class="insititution">Cocoa Research Unit (CRU)</div>, <div class="addr-line">St Augustine</div>, <div class="country">Trinidad and Tobago</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Michel+Boccara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Boccara M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Boccara M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Michel Boccara</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Olivier Fouet</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">CIRAD, UMR 1334 AGAP</div>, <div class="addr-line">TA 108/03-34398, Montpellier Cedex 5</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Olivier+Fouet">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fouet O">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fouet O&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Olivier Fouet</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yolande Roguet</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">CIRAD, UMR 1334 AGAP</div>, <div class="addr-line">TA 108/03-34398, Montpellier Cedex 5</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yolande+Roguet">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Roguet Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Roguet Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yolande Roguet</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Aur&#233;lie B&#233;rard</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">INRA, UR 1279 Etude du Polymorphisme des Génomes Végétaux, CEA Institut de Génomique, Centre National de Génotypage</div>, <div class="addr-line">2, rue Gaston Crémieux, CP5724, 91057 Evry</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Aur%c3%a9lie+B%c3%a9rard">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=B&#233;rard A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;B&#233;rard A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Aurélie Bérard</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jean Marc Th&#233;venin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">CIRAD, Biological Systems Department, UPR Bioagresseurs, 97387 Kourou Cedex</div>, <div class="country">French Guiana</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jean+Marc+Th%c3%a9venin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Th&#233;venin J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Th&#233;venin J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jean Marc Thévenin</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Aur&#233;lie Chauveau</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">INRA, UR 1279 Etude du Polymorphisme des Génomes Végétaux, CEA Institut de Génomique, Centre National de Génotypage</div>, <div class="addr-line">2, rue Gaston Crémieux, CP5724, 91057 Evry</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Aur%c3%a9lie+Chauveau">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chauveau A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chauveau A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Aurélie Chauveau</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ronan Rivallan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">CIRAD, UMR 1334 AGAP</div>, <div class="addr-line">TA 108/03-34398, Montpellier Cedex 5</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ronan+Rivallan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Rivallan R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Rivallan R&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ronan Rivallan</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Didier Clement</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">CIRAD, UMR 1334 AGAP</div>, <div class="addr-line">TA 108/03-34398, Montpellier Cedex 5</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">5</div><div class="insititution">Comissão Executiva de Planejamento da Lavoura Cacaueira (CEPLAC)</div>, <div class="addr-line">Km 22 Rod. Ilheus Itabuna, Cx. postal 07, Itabuna 45600-00, Bahia</div>, <div class="country">Brazil</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Didier+Clement">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Clement D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Clement D&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Didier Clement</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Brigitte Courtois</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">CIRAD, UMR 1334 AGAP</div>, <div class="addr-line">TA 108/03-34398, Montpellier Cedex 5</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Brigitte+Courtois">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Courtois B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Courtois B&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Brigitte Courtois</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Karina Gramacho</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="insititution">Comissão Executiva de Planejamento da Lavoura Cacaueira (CEPLAC)</div>, <div class="addr-line">Km 22 Rod. Ilheus Itabuna, Cx. postal 07, Itabuna 45600-00, Bahia</div>, <div class="country">Brazil</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Karina+Gramacho">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gramacho K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gramacho K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Karina Gramacho</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Anne Boland-Aug&#233;</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">INRA, UR 1279 Etude du Polymorphisme des Génomes Végétaux, CEA Institut de Génomique, Centre National de Génotypage</div>, <div class="addr-line">2, rue Gaston Crémieux, CP5724, 91057 Evry</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Anne+Boland-Aug%c3%a9">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Boland-Aug&#233; A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Boland-Aug&#233; A&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Anne Boland-Augé</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mathias Tahi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="insititution">Centre National de la Recherche Agronomique (CNRA)</div>, <div class="addr-line">B.P. 808, Divo</div>, <div class="country">Côte d'Ivoire</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mathias+Tahi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tahi M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tahi M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Mathias Tahi</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Pathmanathan Umaharan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">University of the West Indies</div>, <div class="insititution">Cocoa Research Unit (CRU)</div>, <div class="addr-line">St Augustine</div>, <div class="country">Trinidad and Tobago</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Pathmanathan+Umaharan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Umaharan P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Umaharan P&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Pathmanathan Umaharan</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Dominique Brunel</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">INRA, UR 1279 Etude du Polymorphisme des Génomes Végétaux, CEA Institut de Génomique, Centre National de Génotypage</div>, <div class="addr-line">2, rue Gaston Crémieux, CP5724, 91057 Evry</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Dominique+Brunel">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Brunel D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Brunel D&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Dominique Brunel</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Claire Lanaud</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">CIRAD, UMR 1334 AGAP</div>, <div class="addr-line">TA 108/03-34398, Montpellier Cedex 5</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Claire+Lanaud">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lanaud C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lanaud C&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Claire Lanaud</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr. Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 19, Issue 1, 1 February 2012, Pages 23–35, <a href='https://doi.org/10.1093/dnares/dsr039'>https://doi.org/10.1093/dnares/dsr039</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">30 December 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">30 June 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">11 October 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="484949" href="/dnaresearch/article-pdf/19/1/23/1311170/dsr039.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Mathilde Allegre, Xavier Argout, Michel Boccara, Olivier Fouet, Yolande Roguet, Aurélie Bérard, Jean Marc Thévenin, Aurélie Chauveau, Ronan Rivallan, Didier Clement, Brigitte Courtois, Karina Gramacho, Anne Boland-Augé, Mathias Tahi, Pathmanathan Umaharan, Dominique Brunel, Claire Lanaud;  Discovery and mapping of a new expressed sequence tag-single nucleotide polymorphism and simple sequence repeat panel for large-scale genetic studies and breeding of <em>Theobroma cacao</em> L., <em>DNA Research</em>, Volume 19, Issue 1, 1 February 2012, Pages 23–35, <a href='https://doi.org/10.1093/dnares/dsr039'>https://doi.org/10.1093/dnares/dsr039</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/484949?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/484949?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/484949?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/484949?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/484949?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/484949?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/484949?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Discovery%20and%20mapping%20of%20a%20new%20expressed%20sequence%20tag-single%20nucleotide%20polymorphism%20and%20simple%20sequence%20repeat%20panel%20for%20large-scale%20genetic%20studies%20and%20breeding%20of%20%26lt%3Bem%26gt%3BTheobroma%20cacao%26lt%3B%2Fem%26gt%3B%20L.&amp;publicationDate=2011-12-30&amp;volumeNum=19&amp;issueNum=1&amp;author=Allegre%2C%20Mathilde%3B%20Argout%2C%20Xavier&amp;startPage=23&amp;endPage=35&amp;contentId=10.1093%2Fdnares%2Fdsr039&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f1%2f23%2f484949" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7371005 id="7371005" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Theobroma cacao</em> is an economically important tree of several tropical countries. Its genetic improvement is essential to provide protection against major diseases and improve chocolate quality. We discovered and mapped new expressed sequence tag-single nucleotide polymorphism (EST-SNP) and simple sequence repeat (SSR) markers and constructed a high-density genetic map. By screening 149 650 ESTs, 5246 SNPs were detected <em>in silico</em>, of which 1536 corresponded to genes with a putative function, while 851 had a clear polymorphic pattern across a collection of genetic resources. In addition, 409 new SSR markers were detected on the Criollo genome. Lastly, 681 new EST-SNPs and 163 new SSRs were added to the pre-existing 418 co-dominant markers to construct a large consensus genetic map. This high-density map and the set of new genetic markers identified in this study are a milestone in cocoa genomics and for marker-assisted breeding. The data are available at <a class="link link-uri" href="http://tropgenedb.cirad.fr">http://tropgenedb.cirad.fr</a>.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Theobroma cacao&quot;"><em>Theobroma cacao</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genetic map&quot;">genetic map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SNP">SNP</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;molecular marker&quot;">molecular marker</a></div>                            <h2 scrollto-destination=7371007 id="7371007" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p><em>Theobroma cacao</em> L. is a diploid species (2<em>n</em> = 2<em>x</em> = 20) with a small genome ranging in size from 411 to 494 Mb.<sup><a reveal-id="DSR039C1" data-open="DSR039C1" class="link link-ref link-reveal xref-bibr">1</a></sup> According to Cheesman,<sup><a reveal-id="DSR039C2" data-open="DSR039C2" class="link link-ref link-reveal xref-bibr">2</a></sup> its centre of origin is at the lower eastern equatorial slopes of the Andes.</p><p><em>Theobroma cacao</em> is grown as a major cash crop that provides income to 14 million small-scale farmers in more than 50 tropical countries. However, cocoa production is markedly affected by a number of major diseases caused by several <em>Phytophthora</em> species, or by <em>Moniliophthora perniciosa</em> and <em>Moniliophthora roreri</em>. Several sources of disease resistance have been identified and the search for sustainable disease resistance by cumulating the different resistance genes is one of the major challenges facing <em>T. cacao</em>-breeding programmes.<sup><a reveal-id="DSR039C3" data-open="DSR039C3" class="link link-ref link-reveal xref-bibr">3</a></sup> The quality of chocolate is another important trait in cocoa breeding, and consumer demand for high-quality chocolate is increasing. A better understanding of the molecular and genetic bases of these traits is a key goal of cocoa genetic research.</p><p>High-density genetic maps are essential tools for trait genetic studies. Several molecular marker types have been developed in <em>T. cacao</em> in recent decades: restriction fragment length polymorphism (RFLP), microsatellites or simple sequence repeats (SSRs), random amplified polymorphic DNA, amplified fragment length polymorphism and isozymes.<sup><a reveal-id="DSR039C4  DSR039C6" data-open="DSR039C4  DSR039C6" class="link link-ref link-reveal xref-bibr">4–6</a></sup> Among them, only RFLP, SSR and single-nucleotide polymorphism (SNP) are co-dominant markers, and therefore more powerful for genetic analyses. Compared with RFLP, the advantage of SSR and SNP markers is that they can be revealed using high-throughput technologies with scant amounts of DNA. Semagn <em>et al.</em><sup><a reveal-id="DSR039C7" data-open="DSR039C7" class="link link-ref link-reveal xref-bibr">7</a></sup> made a detailed comparison of the characteristics of each kind of marker. A high-density cocoa linkage map enriched with SSR genomic markers, including only co-dominant markers, was developed by Pugh <em>et al.</em><sup><a reveal-id="DSR039C8" data-open="DSR039C8" class="link link-ref link-reveal xref-bibr">8</a></sup> More recently, that map was supplemented with 114 EST-SSRs.<sup><a reveal-id="DSR039C9" data-open="DSR039C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p><p>In recent years, the use of SNP markers has substantially increased in plant genetics such as in <em>Arabidopsis</em>,<sup><a reveal-id="DSR039C10" data-open="DSR039C10" class="link link-ref link-reveal xref-bibr">10</a></sup> grapevine,<sup><a reveal-id="DSR039C11" data-open="DSR039C11" class="link link-ref link-reveal xref-bibr">11</a></sup> wheat,<sup><a reveal-id="DSR039C12" data-open="DSR039C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and also a few woody perennial species.<sup><a reveal-id="DSR039C13  DSR039C15" data-open="DSR039C13  DSR039C15" class="link link-ref link-reveal xref-bibr">13–15</a></sup> SNP is one of the most abundant types of DNA sequence polymorphism and the SNP markers are suitable for large-scale genome analysis using high-throughput automated genotyping techniques. SNPs have been used to construct high-resolution genetic maps<sup><a reveal-id="DSR039C16" data-open="DSR039C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSR039C17" data-open="DSR039C17" class="link link-ref link-reveal xref-bibr">17</a></sup> or to trace evolution, particularly in the human genome, using large-scale SNP datasets.<sup><a reveal-id="DSR039C18" data-open="DSR039C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSR039C19" data-open="DSR039C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Knowledge of nucleotide substitution dynamics is an important basis for molecular evolutionary studies, phylogeny reconstruction and natural selection studies.<sup><a reveal-id="DSR039C20" data-open="DSR039C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSR039C21" data-open="DSR039C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Transitions are generally observed with higher frequencies than transversions. During natural selection, transitions are better tolerated because they generate more likely synonymous mutations in protein-coding sequences than transversions.<sup><a reveal-id="DSR039C22  DSR039C25" data-open="DSR039C22  DSR039C25" class="link link-ref link-reveal xref-bibr">22–25</a></sup></p><p>Of existing SNP markers, EST-SNPs (i.e. SNPs located within a gene expressed sequence) are of particular interest for studying functional genetic diversity and identifying candidate genes as the functional base of quantitative trait loci (QTLs). EST-SNPs have been developed for numerous plant models such as melon,<sup><a reveal-id="DSR039C26" data-open="DSR039C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSR039C27" data-open="DSR039C27" class="link link-ref link-reveal xref-bibr">27</a></sup><em>Brassica rapa</em>,<sup><a reveal-id="DSR039C28" data-open="DSR039C28" class="link link-ref link-reveal xref-bibr">28</a></sup> barley,<sup><a reveal-id="DSR039C29" data-open="DSR039C29" class="link link-ref link-reveal xref-bibr">29</a></sup> poplar,<sup><a reveal-id="DSR039C14" data-open="DSR039C14" class="link link-ref link-reveal xref-bibr">14</a></sup> and sugarcane<sup><a reveal-id="DSR039C30" data-open="DSR039C30" class="link link-ref link-reveal xref-bibr">30</a></sup> to detect QTLs for many traits and facilitate the selection of resistant and productive plants. In <em>T. cacao</em>, a few SNPs were detected in ESTs from expression libraries representing <em>T. cacao</em>/<em>M. perniciosa</em> interactions.<sup><a reveal-id="DSR039C31" data-open="DSR039C31" class="link link-ref link-reveal xref-bibr">31</a></sup></p><p>In our study, we discovered and mapped several hundred EST-SNP markers detected in an exhaustive collection of cocoa ESTs<sup><a reveal-id="DSR039C32" data-open="DSR039C32" class="link link-ref link-reveal xref-bibr">32</a></sup> homologous to genes with a known function. These SNP markers were supplemented by 163 new SSR markers to construct a very high-density genetic map suitable for large-scale genetic studies.</p>                            <h2 scrollto-destination=7371014 id="7371014" class="section-title" data-legacy-id=s2>2. Materials and Methods</h2>
                            <h3 scrollto-destination=7371015 id="7371015" class="section-title" data-legacy-id=s2a>2.1. Plant material</h3>
<p>SNP polymorphisms were screened in a collection of diverse germplasm representing the major part of the <em>T. cacao</em> diversity and two existing mapping populations denominated UPA402 × UF676 and F2.</p><p>The collection of diverse germplasm consisted of 249 genotypes from various genetic groups and geographical origins (Table <a reveal-id="DSR039TB1" data-open="DSR039TB1" class="link link-reveal link-table xref-fig">1</a>). Most of these accessions are maintained at the International Cocoa Genebank (ICG) at the Cocoa Research Unit (CRU), University of the West Indies, Trinidad and Tobago. </p>                        <a id="7371018" scrollto-destination="7371018"></a>
<div content-id="DSR039TB1" class="table-modal"><div class="table-wrap"><div id="DSR039TB1" class="table-wrap-title" data-id="DSR039TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Theobroma cacao genotypes of various geographical origins used to screen the polymorphism of the 1536 GoldenGate SNP panel</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Accession collection group name </th><th>Number of genotypes </th><th>Geographical origin </th></tr></thead><tbody><tr><td>AMAZ </td><td>2 </td><td>Ecuador </td></tr><tr><td>APA </td><td>1 </td><td>Colombia </td></tr><tr><td>Nacional </td><td>3 </td><td>Ecuador </td></tr><tr><td>Criollo </td><td>14 </td><td>Mexico-Belize </td></tr><tr><td>EBC </td><td>4 </td><td>Colombia </td></tr><tr><td>Trinitario </td><td>28 </td><td>Trinidad </td></tr><tr><td>GU </td><td>12 </td><td>French Guiana </td></tr><tr><td>IMC </td><td>19 </td><td>Peru </td></tr><tr><td>LCTEEN </td><td>46 </td><td>Ecuador </td></tr><tr><td>MORONA </td><td>3 </td><td>Peru </td></tr><tr><td>NANAY </td><td>49 </td><td>Peru </td></tr><tr><td>PARINARI </td><td>40 </td><td>Peru </td></tr><tr><td>POUND </td><td>6 </td><td>Peru </td></tr><tr><td>SC </td><td>5 </td><td>Colombia </td></tr><tr><td>SCAVINA </td><td>8 </td><td>Peru </td></tr><tr><td>Amelonado type </td><td>3 </td><td>Brazil </td></tr><tr><td>SPEC </td><td>6 </td><td>Colombia </td></tr><tr><td>Total </td><td>249 </td><td> </td></tr></tbody></table></div><div content-id="DSR039TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Accession collection group name </th><th>Number of genotypes </th><th>Geographical origin </th></tr></thead><tbody><tr><td>AMAZ </td><td>2 </td><td>Ecuador </td></tr><tr><td>APA </td><td>1 </td><td>Colombia </td></tr><tr><td>Nacional </td><td>3 </td><td>Ecuador </td></tr><tr><td>Criollo </td><td>14 </td><td>Mexico-Belize </td></tr><tr><td>EBC </td><td>4 </td><td>Colombia </td></tr><tr><td>Trinitario </td><td>28 </td><td>Trinidad </td></tr><tr><td>GU </td><td>12 </td><td>French Guiana </td></tr><tr><td>IMC </td><td>19 </td><td>Peru </td></tr><tr><td>LCTEEN </td><td>46 </td><td>Ecuador </td></tr><tr><td>MORONA </td><td>3 </td><td>Peru </td></tr><tr><td>NANAY </td><td>49 </td><td>Peru </td></tr><tr><td>PARINARI </td><td>40 </td><td>Peru </td></tr><tr><td>POUND </td><td>6 </td><td>Peru </td></tr><tr><td>SC </td><td>5 </td><td>Colombia </td></tr><tr><td>SCAVINA </td><td>8 </td><td>Peru </td></tr><tr><td>Amelonado type </td><td>3 </td><td>Brazil </td></tr><tr><td>SPEC </td><td>6 </td><td>Colombia </td></tr><tr><td>Total </td><td>249 </td><td> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7371018" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR039TB1" class="table-wrap-title" data-id="DSR039TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Theobroma cacao genotypes of various geographical origins used to screen the polymorphism of the 1536 GoldenGate SNP panel</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Accession collection group name </th><th>Number of genotypes </th><th>Geographical origin </th></tr></thead><tbody><tr><td>AMAZ </td><td>2 </td><td>Ecuador </td></tr><tr><td>APA </td><td>1 </td><td>Colombia </td></tr><tr><td>Nacional </td><td>3 </td><td>Ecuador </td></tr><tr><td>Criollo </td><td>14 </td><td>Mexico-Belize </td></tr><tr><td>EBC </td><td>4 </td><td>Colombia </td></tr><tr><td>Trinitario </td><td>28 </td><td>Trinidad </td></tr><tr><td>GU </td><td>12 </td><td>French Guiana </td></tr><tr><td>IMC </td><td>19 </td><td>Peru </td></tr><tr><td>LCTEEN </td><td>46 </td><td>Ecuador </td></tr><tr><td>MORONA </td><td>3 </td><td>Peru </td></tr><tr><td>NANAY </td><td>49 </td><td>Peru </td></tr><tr><td>PARINARI </td><td>40 </td><td>Peru </td></tr><tr><td>POUND </td><td>6 </td><td>Peru </td></tr><tr><td>SC </td><td>5 </td><td>Colombia </td></tr><tr><td>SCAVINA </td><td>8 </td><td>Peru </td></tr><tr><td>Amelonado type </td><td>3 </td><td>Brazil </td></tr><tr><td>SPEC </td><td>6 </td><td>Colombia </td></tr><tr><td>Total </td><td>249 </td><td> </td></tr></tbody></table></div><div content-id="DSR039TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Accession collection group name </th><th>Number of genotypes </th><th>Geographical origin </th></tr></thead><tbody><tr><td>AMAZ </td><td>2 </td><td>Ecuador </td></tr><tr><td>APA </td><td>1 </td><td>Colombia </td></tr><tr><td>Nacional </td><td>3 </td><td>Ecuador </td></tr><tr><td>Criollo </td><td>14 </td><td>Mexico-Belize </td></tr><tr><td>EBC </td><td>4 </td><td>Colombia </td></tr><tr><td>Trinitario </td><td>28 </td><td>Trinidad </td></tr><tr><td>GU </td><td>12 </td><td>French Guiana </td></tr><tr><td>IMC </td><td>19 </td><td>Peru </td></tr><tr><td>LCTEEN </td><td>46 </td><td>Ecuador </td></tr><tr><td>MORONA </td><td>3 </td><td>Peru </td></tr><tr><td>NANAY </td><td>49 </td><td>Peru </td></tr><tr><td>PARINARI </td><td>40 </td><td>Peru </td></tr><tr><td>POUND </td><td>6 </td><td>Peru </td></tr><tr><td>SC </td><td>5 </td><td>Colombia </td></tr><tr><td>SCAVINA </td><td>8 </td><td>Peru </td></tr><tr><td>Amelonado type </td><td>3 </td><td>Brazil </td></tr><tr><td>SPEC </td><td>6 </td><td>Colombia </td></tr><tr><td>Total </td><td>249 </td><td> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7371018" target="_blank">View Large</a></div></div><p>The UPA402 × UF676 mapping population consisted of 264 individuals derived from a cross of two unrelated heterozygous tree accessions; UPA402, an Upper Amazon Forastero from Peru, and UF676, a Trinitario (Forastero × Criollo hybrid) selected in Costa Rica. This progeny was maintained by Centre National de Recherche Agronomique (CNRA) in Bingerville and Divo, Côte d'Ivoire. It was used to establish genetic map as a reference in our laboratory.<sup><a reveal-id="DSR039C4" data-open="DSR039C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSR039C6" data-open="DSR039C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSR039C8" data-open="DSR039C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSR039C9" data-open="DSR039C9" class="link link-ref link-reveal xref-bibr">9</a></sup> We mapped the new SSR and SNP markers in this population.</p><p>The F2 second progeny of 132 individuals was obtained by selfing a hybrid between two heterozygous parents: Scavina 6, an Upper Amazon Forastero collected in Peru, and ICS1, a Trinitario selected in Trinidad. This progeny was produced by Comissão Executiva do Plano da Lavoura Cacaueira (CEPLAC) at Itabuna, Brazil.</p>                            <h4 scrollto-destination=7371021 id="7371021" class="section-title" data-legacy-id=s2a1>2.1.1. Genotypes used for EST-SNP detection and selection</h4>
<p>Most of the ESTs screened for SNPs had been obtained from the contrasting genotypes Scavina 6, an upper Amazon Forastero genotype from Peru and ICS1, a Trinitario selected in Trinidad, a hybrid between a Criollo from Central America and a Forastero from Lower Amazonia of Brazil.</p><p>These two genotypes, which represent the three distinct genetic origins, Upper Amazon Forastero, Lower Amazon Forastero, and Criollo, were also the parents of the F2 population from Brazil used to map SNPs.</p><p>Eleven other genotypes were involved in the construction of the cDNA libraries and SNP identification: B97-CC2, a Criollo from Belize, P7, IMC47, UPA 134, Upper Amazon Forastero genotypes from Peru, Jaca, an Upper Amazon Forastero from Brazil, GU255V, collected in French Guiana, B240 and 33–49, two Nacional genotypes from Ecuador, UF676, UF273, two Trinitario, and seedlings from a hybrid selected in Papua New Guinea.</p><p>SSRs were screened in three <em>T. cacao</em> genotypes: the two parents of the reference map (UPA402 and UF676), and the sequenced Criollo genotype (B97-61/B2).</p>                            <h3 scrollto-destination=7371026 id="7371026" class="section-title" data-legacy-id=s2b>2.2. DNA extraction and purification</h3>
<p>Genomic DNA was extracted according to a protocol using MATAB buffer already described for the isolation of genomic DNA.<sup><a reveal-id="DSR039C6" data-open="DSR039C6" class="link link-ref link-reveal xref-bibr">6</a></sup> DNA was resuspended with 1 ml of TE (10 mM Tris–HCl and 1 mM EDTA, pH 8.0).</p><p>DNA was purified with the Nucleobond® PC 20 kit (Macherey-Nagel, Cat. No. 740.571.100) with the modification that steps 1 and 2 were omitted and the DNA was purified directly after its isolation. A 1 ml mixture composed of 200 µl of crude DNA (20 µg DNA maximum), 450 µl water and 350 µl S3 buffer + RNAse (buffers provided with the kit) was passed through the column (step 3). This solution was homogenized on a rocking table for at least 1 h. After precipitation of the eluate with an equal volume of isopropyl alcohol, the pellet was resuspended in 70 µl of TE.</p><p>The quality and quantity of DNA were first checked on 0.8% agarose gel, compared with a standard range, and then the Quant-iT™ PicoGreen® dsDNA Assay from Invitrogen™ was used. A quality test was performed for each sample by amplifying microsatellite markers in a PCR mixture with a high DNA concentration (100 ng DNA in a 10 μl reaction volume). The purification step was repeated when the amplification failed.</p>                            <h3 scrollto-destination=7371030 id="7371030" class="section-title" data-legacy-id=s2c>2.3. In silico SNP discovery and verification</h3>
<p>A collection of 149 650 ESTs (EMBL accession number CU469588 to CU633156), corresponding to 48 594 unigenes, was produced after sequencing 56 cDNA libraries constructed from material collected from different organs, genotypes, and under different environmental conditions.<sup><a reveal-id="DSR039C32" data-open="DSR039C32" class="link link-ref link-reveal xref-bibr">32</a></sup></p><p>SNPs were detected <em>in silico</em> and quality checked using the QualitySNP pipeline,<sup><a reveal-id="DSR039C33" data-open="DSR039C33" class="link link-ref link-reveal xref-bibr">33</a></sup> as reported in Argout <em>et al.</em><sup><a reveal-id="DSR039C32" data-open="DSR039C32" class="link link-ref link-reveal xref-bibr">32</a></sup> and in ESTtik (<a class="link link-uri" href="http://esttik.cirad.fr">http://esttik.cirad.fr</a>).</p><p>QualitySNP uses quality information related to each EST and a haplotype-based strategy to predict reliable SNPs. In order to detect SNPs in known homologous coding sequences, we selected contigs displaying a significant similarity with proteins from a non-redundant protein sequence database (NR), with entries from GenPept, Swissprot, PIR, PDF, PDB, and NCBI RefSeq, and as described in Argout <em>et al.</em><sup><a reveal-id="DSR039C32" data-open="DSR039C32" class="link link-ref link-reveal xref-bibr">32</a></sup></p>                            <h3 scrollto-destination=7371034 id="7371034" class="section-title" data-legacy-id=s2d>2.4. Validation of SNPs via golden gate assay</h3>
<p>A total of 30–50 ng of genomic DNA per plant was used for Illumina SNP genotyping with the Illumina BeadArray platform at the French National Genotyping Centre (CNG, CEA-IG, Evry, France), according to the GoldenGate Assay manufacturer's protocol. Three 3-day assays were carried out to genotype the progeny samples for the 1536 SNP set revealed by the GoldenGate assay (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The protocol was similar to that briefly described by Hyten <em>et al.</em><sup><a reveal-id="DSR039C34" data-open="DSR039C34" class="link link-ref link-reveal xref-bibr">34</a></sup> except for the number of oligonucleotides involved in a single DNA reaction, thus comprising 4608 custom oligos assembled in the oligo pooled assays (OPA) designed by Illumina Inc. Raw hybridization, intensity data processing, clustering, and genotype calling were performed using the genotyping module in the BeadStudio/GenomeStudio package (Illumina, San Diego, CA, USA). Illumina has developed a self-normalization algorithm that relies on information contained in each array, as described by Akhunov <em>et al.</em><sup><a reveal-id="DSR039C35" data-open="DSR039C35" class="link link-ref link-reveal xref-bibr">35</a></sup></p><p>The clustering and genotype calling of each of the 1536 SNP markers were checked for their conformity and correct genotype distribution using known homozygous and heterozygous genotypes, included in the collection of diverse genotypes, as standards.</p>                            <h3 scrollto-destination=7371037 id="7371037" class="section-title" data-legacy-id=s2e>2.5. SSR in silico discovery and genotyping</h3>
<p>The MIcroSAtellite identification tool (MISA <a class="link link-uri" href="http://pgrc.ipk-gatersleben.de/misa">http://pgrc.ipk-gatersleben.de/misa</a>) was used to perform SSR searches, and primers were designed with Primer3 software.<sup><a reveal-id="DSR039C36" data-open="DSR039C36" class="link link-ref link-reveal xref-bibr">36</a></sup></p><p>Primers flanking microsatellite loci were designed at each end of the scaffolds to orient and anchor them to the genetic map.</p><p>SSRs identified in the scaffolds were mapped only on the reference map established from the UPA402 × UF676 cross.</p><p>A total of 409 primer pairs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>) were defined in the 100 larger non-anchored scaffolds using Primer3 software<sup><a reveal-id="DSR039C36" data-open="DSR039C36" class="link link-ref link-reveal xref-bibr">36</a></sup> and screened for their ability to segregate in the UPA402 × UF676 progeny.</p><p>For a given SSR locus, the forward primer was designed with a 5′-end M13 tail (5′-CACGACGTTGTAAAACGAC-3′). PCR amplifications were performed in a Mastercycler ep384 thermocycler (Eppendorf, Germany) with 5 ng of purified DNA in a 10 μl final volume of buffer containing 10 mM Tris–HCl (pH 8), 50 mM KCl, 0.001% glycerol, 2.0 mM MgCl<sub>2</sub>, 0.08 μM of the M13-tailed forward primer, 0.1 μM of the reverse primer, 200 μM of dNTP, 1 U of Taq DNA polymerase (Life Technologies, USA), 0.1 μM of M13 primer-fluorescent dye 6-FAM™, NED<sup>®</sup>, VIC<sup>®</sup>, or PET<sup>®</sup> (Applied Biosystems, CA, USA). The DNA and buffer were distributed in 384 plates using a Biomek NX automatic pipetting robot (Beckman Coulter, CA, USA). The touchdown PCR programme used was as follows: initial denaturation at 95°C for 5 min, followed by 10 cycles at 95°C for 45 s, Tm of 56–46°C (−1°C/cycle) for 1 min, and 72°C for 1 min 30 s. After these cycles, an additional round of 25 cycles were performed at 95°C for 45 s, Tm of 50°C for 1 min, and 72°C for 1 min, with a final elongation step at 72°C for 30 min.</p><p>PCR products were diluted specifically for each dye and pooled for multiplex SSR genotyping (revealing two SSRs having different sizes of amplified product per dye). A mixture of 15 µl of Hi-Di™ formamide (Applied Biosystems) and 0.12 µl of size marker GeneScan™ 600-LIZ-Size® Standard V2.0. (Applied Biosystems) was added to 2 μl of the diluted PCR pool. This pool was then analysed using the ABI 3500xL automatic sequencer (Applied Biosystems).</p><p>Images were analysed using Genemapper 4.0 software (Applied Biosystems) and exported as a data table.</p>                            <h3 scrollto-destination=7371045 id="7371045" class="section-title" data-legacy-id=s2f>2.6. Genetic mapping</h3>
<p>The UPA402 × UF676 population was the result of a cross between two heterozygous cocoa clones, UPA402 (♀) an Upper-Amazon Forastero and UF676 (♂) a Trinitario. In this case, there were three segregation possibilities: loci that were homozygous for one parent and heterozygous for the other, segregation (1:1), and those that segregated in both parents (1:2:1 or 1:1:1:1).</p><p>Segregations were checked for goodness-of-fit to the expected Mendelian ratio using a chi-square test at significance levels of 0.05 and 0.01.</p><p>Individual and consensus maps were constructed using Joinmap software, version 4.0.<sup><a reveal-id="DSR039C37" data-open="DSR039C37" class="link link-ref link-reveal xref-bibr">37</a></sup></p><p>Joinmap is able to combine data of several segregation types to construct a consensus genetic map. Here we used population type CP for the UPA402 × UF676 map, and population type F2 for the F2 population from Brazil. A lod score of 6 was used to identify 10 linkage groups (LGs) independently for each map. A consensus genetic map was established from the two distinct genetic maps. The corresponding groups were associated in pairs with JoinMap software. The Kosambi mapping function, with a lod score of 5 and a jump threshold of 3, was used to convert recombination frequencies into map distances.<sup><a reveal-id="DSR039C38" data-open="DSR039C38" class="link link-ref link-reveal xref-bibr">38</a></sup></p><p>This consensus map combined the new EST-SNPs and genomic SSRs defined from the scaffolds, in addition to the previously mapped markers.<sup><a reveal-id="DSR039C9" data-open="DSR039C9" class="link link-ref link-reveal xref-bibr">9</a></sup> This map contained only markers with a known nucleotide sequence.</p>                            <h2 scrollto-destination=7371051 id="7371051" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=7371052 id="7371052" class="section-title" data-legacy-id=s3a>3.1. Identification of SNPs and development of the golden gate assay</h3>
<p>The assembly made from the 149 650 T<em>. cacao</em> EST sequences (see Materials and Methods) generated 12692 T<em>. cacao</em> contigs. The number of ESTs per contig ranged from 2 to 5102. To detect good quality <em>in silico</em> SNPs, we assumed that contigs with more than 100 members contained paralogous sequences.<sup><a reveal-id="DSR039C13" data-open="DSR039C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSR039C39" data-open="DSR039C39" class="link link-ref link-reveal xref-bibr">39</a></sup> We therefore first selected 4818 contigs that contained at least 4 but no more than 100 EST members. A total of 5246 SNPs were identified <em>in silico</em> in 2012 contigs.</p><p>We selected 4150 <em>in silico</em> SNPs detected in 1834 contigs that had a significant BlastX annotation similarity with known proteins of the NCBI non-redundant protein sequence database (NR) with entries from GenPept, Swissprot, PIR, PDF, PDB, and NCBI RefSeq, and as described in Argout <em>et al.</em><sup><a reveal-id="DSR039C32" data-open="DSR039C32" class="link link-ref link-reveal xref-bibr">32</a></sup></p>                            <h3 scrollto-destination=7371055 id="7371055" class="section-title" data-legacy-id=s3b>3.2. SNP performance and quality</h3>
<p>The set of 4150 <em>in silico</em> SNPs was selected in the EST contigs and the SNP-harbouring sequences were then submitted to Illumina for processing using the Illumina<sup>®</sup> Assay Design Tool (ADT). ADT generates scores for each SNP that can range from 0 to 1; SNPs with scores &gt;0.6 have a high probability of being converted into a successful genotyping assay. In the set of 4150 submitted SNPs, 83.5% showed a high conversion success rate (&gt;0.6), 9.2% showed a moderate conversion success rate (between 0.4 and 0.6), and 7.3% showed either a low conversion success rate or no score. A total of 1536 SNP sites having ADT scores &gt;0.4 and without any other SNPs within the adjacent 60 bp was selected for the OPA design (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>).</p>                            <h3 scrollto-destination=7371057 id="7371057" class="section-title" data-legacy-id=s3c>3.3. Analysis of base changes</h3>
<p>One thousand and forty-four <em>in silico</em> SNPs (68%) were transitions and 462 (32%) were transversions (Table <a reveal-id="DSR039TB2" data-open="DSR039TB2" class="link link-reveal link-table xref-fig">2</a>). This ratio of transition/transversion SNPs tallies with the results observed in other plant species, where transition SNPs are always more frequent than transversion SNPs. </p>                        <a id="7371059" scrollto-destination="7371059"></a>
<div content-id="DSR039TB2" class="table-modal"><div class="table-wrap"><div id="DSR039TB2" class="table-wrap-title" data-id="DSR039TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Nucleotide substitution types of the 1536 selected <em>in silico</em> SNPs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Types </th><th>Number of SNPs </th><th>Percentage </th><th> </th><th>Percentage </th></tr></thead><tbody><tr><td>A ↔ C </td><td>126 </td><td>8 </td><td rowspan="4">Transversion </td><td rowspan="4">32 </td></tr><tr><td>A ↔ T </td><td>128 </td><td>8 </td></tr><tr><td>C ↔ G </td><td>112 </td><td>7 </td></tr><tr><td>T ↔ C </td><td>126 </td><td>8 </td></tr><tr><td colspan="5">  </td></tr><tr><td>T ↔ G </td><td>612 </td><td>40 </td><td rowspan="2">Transition </td><td rowspan="2">68 </td></tr><tr><td>A ↔ G </td><td>432 </td><td>29 </td></tr></tbody></table></div><div content-id="DSR039TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Types </th><th>Number of SNPs </th><th>Percentage </th><th> </th><th>Percentage </th></tr></thead><tbody><tr><td>A ↔ C </td><td>126 </td><td>8 </td><td rowspan="4">Transversion </td><td rowspan="4">32 </td></tr><tr><td>A ↔ T </td><td>128 </td><td>8 </td></tr><tr><td>C ↔ G </td><td>112 </td><td>7 </td></tr><tr><td>T ↔ C </td><td>126 </td><td>8 </td></tr><tr><td colspan="5">  </td></tr><tr><td>T ↔ G </td><td>612 </td><td>40 </td><td rowspan="2">Transition </td><td rowspan="2">68 </td></tr><tr><td>A ↔ G </td><td>432 </td><td>29 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7371059" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR039TB2" class="table-wrap-title" data-id="DSR039TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Nucleotide substitution types of the 1536 selected <em>in silico</em> SNPs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Types </th><th>Number of SNPs </th><th>Percentage </th><th> </th><th>Percentage </th></tr></thead><tbody><tr><td>A ↔ C </td><td>126 </td><td>8 </td><td rowspan="4">Transversion </td><td rowspan="4">32 </td></tr><tr><td>A ↔ T </td><td>128 </td><td>8 </td></tr><tr><td>C ↔ G </td><td>112 </td><td>7 </td></tr><tr><td>T ↔ C </td><td>126 </td><td>8 </td></tr><tr><td colspan="5">  </td></tr><tr><td>T ↔ G </td><td>612 </td><td>40 </td><td rowspan="2">Transition </td><td rowspan="2">68 </td></tr><tr><td>A ↔ G </td><td>432 </td><td>29 </td></tr></tbody></table></div><div content-id="DSR039TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Types </th><th>Number of SNPs </th><th>Percentage </th><th> </th><th>Percentage </th></tr></thead><tbody><tr><td>A ↔ C </td><td>126 </td><td>8 </td><td rowspan="4">Transversion </td><td rowspan="4">32 </td></tr><tr><td>A ↔ T </td><td>128 </td><td>8 </td></tr><tr><td>C ↔ G </td><td>112 </td><td>7 </td></tr><tr><td>T ↔ C </td><td>126 </td><td>8 </td></tr><tr><td colspan="5">  </td></tr><tr><td>T ↔ G </td><td>612 </td><td>40 </td><td rowspan="2">Transition </td><td rowspan="2">68 </td></tr><tr><td>A ↔ G </td><td>432 </td><td>29 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7371059" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=7371060 id="7371060" class="section-title" data-legacy-id=s3d>3.4. SNP polymorphism</h3>
<p>From the 1536 SNPs, 841 (55%) with a non-ambiguous polymorphic pattern across accessions were retained as true and verified SNPs and denominated TcSNP. Of the rest, 113 (7%) failed to be genotyped, 436 (28%) had a monomorphic pattern, and 146 (10%) were polymorphic but did not show any clear fluorescent pattern suitable for reliable genotype classification.</p><p>Of the 841 polymorphic SNPs, 461 segregated in the mapping population (UPA402 × UF676) and could be mapped on the reference map. Five hundred and thirty-one were polymorphic and mapped on the F2 population map. Two hundred and thirty-nine SNP markers were segregating in both maps, thus enabling construction of a consensus map between them.</p>                            <h3 scrollto-destination=7371063 id="7371063" class="section-title" data-legacy-id=s3e>3.5. SSR polymorphism</h3>
<p>A high-density genetic map is a key tool to order the scaffold assembly needed to generate a complete cocoa genome sequence. SSR markers were defined in the largest non-anchored scaffolds in order to improve anchoring of the <em>T. cacao</em> genome assembly provided by the International Cocoa Genome Sequencing consortium<sup><a reveal-id="DSR039C1" data-open="DSR039C1" class="link link-ref link-reveal xref-bibr">1</a></sup> on the genetic map.</p><p>From the 409 screened SSRs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>), 163 were polymorphic for the UPA402 × UF676 progeny and could be mapped.</p><p>The new SSR markers defined from scaffolds were named mTcCIR450 to mTcCIR613 to extend the previously identified SSR marker series; mTcCIR 1 to mTcCIR 291<sup><a reveal-id="DSR039C8" data-open="DSR039C8" class="link link-ref link-reveal xref-bibr">8</a></sup> from genomic DNA and mTcCIR 292 to mTcCIR 447<sup><a reveal-id="DSR039C9" data-open="DSR039C9" class="link link-ref link-reveal xref-bibr">9</a></sup> from ESTs.</p>                            <h3 scrollto-destination=7371067 id="7371067" class="section-title" data-legacy-id=s3f>3.6. Individual genetic linkage maps</h3>
                            <h4 scrollto-destination=7371068 id="7371068" class="section-title" data-legacy-id=s3f1>3.6.1. Map of the UPA402 × UF676 population</h4>
<p>A new set of 624 markers with their corresponding sequences, including 461 EST-SNP and 163 new SSR markers located on scaffolds of the genome assembly,<sup><a reveal-id="DSR039C1" data-open="DSR039C1" class="link link-ref link-reveal xref-bibr">1</a></sup> were added to the reference map (Fig. <a reveal-id="DSR039F1" data-open="DSR039F1" class="link link-reveal link-table xref-fig">1</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). </p>                        <a id="7371070" scrollto-destination="7371070"></a>
<div id="DSR039F1" data-id="DSR039F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr039/2/m_dsr03901.jpeg?Expires=2147483647&Signature=RQHtSi1LQe300avjLwOc1boO6-dRmeWXA-cuiQ9f0QfylwLl0ki8~1WhA6xwZ-lNT8zO1dZrzBH8i1IfJ6N9mCfJTt4bys4JSf6O1xsFRqtubn~HnAVThwy6rAwxKNM-~Lz6P9fNgwMFv72CLUnMGGJXTgGzj~ODJQ5aDkQVTrT7Oi-ToaAY7bcqjKafEDTc5CBXjtZuiaKzyHjABD1j5QNyrzEduUrmUeXUVRczYS0673X30D3LPTaTMu1sjVPpuRz39d3yM2nC7KefE-Bes6Lzqlh3kcnjdiw7qhq6DcNoNl1iO4EX3rPhlLQ8TLMJbA5IaXisqNnTXrFNkZYnEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic map constructed from an F1 progeny of 264 individuals (located in CNRA, Côte d'Ivoire) belonging to the UPA402 × UF676 cross. This map consists of 1043 markers of a known DNA sequence (461 SNPs, 524 SSRs, and 58 RFLPs), spanning 752 cM. The average distance between two markers is 0.7 cM. The new markers added to this map are printed in red." path-from-xml="dsr03901.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7371070/dsr03901.jpeg" path-from-xml="dsr03901.jpeg" target="_blank">View large</a><a section="7371070" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr039/2/dsr03901.jpeg?Expires=2147483647&Signature=R2KodbXM7hZZN6GlozRX5RBU-jI1jy57E-u2FjV7vlGxwRdIiZCxIIQKFSkAmsPUWbRExwdiW06HRMvoy-Cl3OtfioyZvYmXK4xRt-FbZVZF319iHu6myas6MTy~wT80mjNAHk0Edxcze0WyUh5Wci-ydaS0auPQ6zPJuh9rNfAnuWi3905zYZcypze8Pyn1xu4nVqWRZNHa4udo7Wx5bQxaqQ8nbolccfufy-N1DT-gcitnAQLrDp3i78wh3Ehzitxj7w6w1IMi9l~j2J8lWINdtqcm0nV6CsNqIvcH0Mjn91RnthMHp1vB5sQICDvp2SxRRRaAdWPpE4huhv9q4A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7371070&ar=484949&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03901.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic map constructed from an F1 progeny of 264 individuals (located in CNRA, Côte d'Ivoire) belonging to the UPA402 × UF676 cross. This map consists of 1043 markers of a known DNA sequence (461 SNPs, 524 SSRs, and 58 RFLPs), spanning 752 cM. The average distance between two markers is 0.7 cM. The new markers added to this map are printed in red.</p></div></div><div content-id="DSR039F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr039/2/m_dsr03901.jpeg?Expires=2147483647&Signature=RQHtSi1LQe300avjLwOc1boO6-dRmeWXA-cuiQ9f0QfylwLl0ki8~1WhA6xwZ-lNT8zO1dZrzBH8i1IfJ6N9mCfJTt4bys4JSf6O1xsFRqtubn~HnAVThwy6rAwxKNM-~Lz6P9fNgwMFv72CLUnMGGJXTgGzj~ODJQ5aDkQVTrT7Oi-ToaAY7bcqjKafEDTc5CBXjtZuiaKzyHjABD1j5QNyrzEduUrmUeXUVRczYS0673X30D3LPTaTMu1sjVPpuRz39d3yM2nC7KefE-Bes6Lzqlh3kcnjdiw7qhq6DcNoNl1iO4EX3rPhlLQ8TLMJbA5IaXisqNnTXrFNkZYnEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic map constructed from an F1 progeny of 264 individuals (located in CNRA, Côte d'Ivoire) belonging to the UPA402 × UF676 cross. This map consists of 1043 markers of a known DNA sequence (461 SNPs, 524 SSRs, and 58 RFLPs), spanning 752 cM. The average distance between two markers is 0.7 cM. The new markers added to this map are printed in red." path-from-xml="dsr03901.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7371070/dsr03901.jpeg" path-from-xml="dsr03901.jpeg" target="_blank">View large</a><a section="7371070" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr039/2/dsr03901.jpeg?Expires=2147483647&Signature=R2KodbXM7hZZN6GlozRX5RBU-jI1jy57E-u2FjV7vlGxwRdIiZCxIIQKFSkAmsPUWbRExwdiW06HRMvoy-Cl3OtfioyZvYmXK4xRt-FbZVZF319iHu6myas6MTy~wT80mjNAHk0Edxcze0WyUh5Wci-ydaS0auPQ6zPJuh9rNfAnuWi3905zYZcypze8Pyn1xu4nVqWRZNHa4udo7Wx5bQxaqQ8nbolccfufy-N1DT-gcitnAQLrDp3i78wh3Ehzitxj7w6w1IMi9l~j2J8lWINdtqcm0nV6CsNqIvcH0Mjn91RnthMHp1vB5sQICDvp2SxRRRaAdWPpE4huhv9q4A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7371070&ar=484949&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03901.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic map constructed from an F1 progeny of 264 individuals (located in CNRA, Côte d'Ivoire) belonging to the UPA402 × UF676 cross. This map consists of 1043 markers of a known DNA sequence (461 SNPs, 524 SSRs, and 58 RFLPs), spanning 752 cM. The average distance between two markers is 0.7 cM. The new markers added to this map are printed in red.</p></div></div><p>The new UPA402 × UF676 map contained 1043 markers, including 461 EST-SNPs, 524 SSRs and 58 RFLPs (Table <a reveal-id="DSR039TB3" data-open="DSR039TB3" class="link link-reveal link-table xref-fig">3</a>). Of the 1043 markers, 571 corresponded to gene markers. The length of this map was 751.7 cM having an average distance of 0.7 cM between adjacent markers. </p>                        <a id="7371072" scrollto-destination="7371072"></a>
<div content-id="DSR039TB3" class="table-modal"><div class="table-wrap"><div id="DSR039TB3" class="table-wrap-title" data-id="DSR039TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Distribution of each marker type in the LGs of the reference map (UPA402 × UF676)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LG </th><th>Length (cM) </th><th>Total number of markers </th><th>Average distance between markers (cM) </th><th>SNP </th><th>RFLP </th><th>Genomic SSR </th><th>SSR from scaffold </th><th>EST-SSR </th></tr></thead><tbody><tr><td>LG1 </td><td>90.3 </td><td>150 </td><td>0.6 </td><td>69 </td><td>8 </td><td>33 </td><td>22 </td><td>18 </td></tr><tr><td>LG2 </td><td>97.5 </td><td>126 </td><td>0.8 </td><td>55 </td><td>6 </td><td>26 </td><td>28 </td><td>11 </td></tr><tr><td>LG3 </td><td>74.4 </td><td>126 </td><td>0.6 </td><td>57 </td><td>6 </td><td>33 </td><td>18 </td><td>14 </td></tr><tr><td>LG4 </td><td>75.6 </td><td>120 </td><td>0.6 </td><td>61 </td><td>11 </td><td>27 </td><td>14 </td><td>8 </td></tr><tr><td>LG5 </td><td>77.4 </td><td>121 </td><td>0.6 </td><td>55 </td><td>6 </td><td>34 </td><td>15 </td><td>11 </td></tr><tr><td>LG6 </td><td>62.7 </td><td>73 </td><td>0.9 </td><td>28 </td><td>2 </td><td>19 </td><td>12 </td><td>12 </td></tr><tr><td>LG7 </td><td>48.9 </td><td>51 </td><td>1.0 </td><td>11 </td><td>6 </td><td>16 </td><td>16 </td><td>2 </td></tr><tr><td>LG8 </td><td>60.3 </td><td>64 </td><td>0.9 </td><td>30 </td><td>1 </td><td>17 </td><td>5 </td><td>11 </td></tr><tr><td>LG9 </td><td>103.2 </td><td>154 </td><td>0.7 </td><td>78 </td><td>6 </td><td>34 </td><td>17 </td><td>20 </td></tr><tr><td>LG10 </td><td>61.3 </td><td>54 </td><td>1.1 </td><td>17 </td><td>6 </td><td>12 </td><td>16 </td><td>3 </td></tr><tr><td>Total </td><td>751.7 </td><td>1043 </td><td>0.7 </td><td>461 </td><td>58 </td><td>251 </td><td>163 </td><td>110 </td></tr></tbody></table></div><div content-id="DSR039TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>LG </th><th>Length (cM) </th><th>Total number of markers </th><th>Average distance between markers (cM) </th><th>SNP </th><th>RFLP </th><th>Genomic SSR </th><th>SSR from scaffold </th><th>EST-SSR </th></tr></thead><tbody><tr><td>LG1 </td><td>90.3 </td><td>150 </td><td>0.6 </td><td>69 </td><td>8 </td><td>33 </td><td>22 </td><td>18 </td></tr><tr><td>LG2 </td><td>97.5 </td><td>126 </td><td>0.8 </td><td>55 </td><td>6 </td><td>26 </td><td>28 </td><td>11 </td></tr><tr><td>LG3 </td><td>74.4 </td><td>126 </td><td>0.6 </td><td>57 </td><td>6 </td><td>33 </td><td>18 </td><td>14 </td></tr><tr><td>LG4 </td><td>75.6 </td><td>120 </td><td>0.6 </td><td>61 </td><td>11 </td><td>27 </td><td>14 </td><td>8 </td></tr><tr><td>LG5 </td><td>77.4 </td><td>121 </td><td>0.6 </td><td>55 </td><td>6 </td><td>34 </td><td>15 </td><td>11 </td></tr><tr><td>LG6 </td><td>62.7 </td><td>73 </td><td>0.9 </td><td>28 </td><td>2 </td><td>19 </td><td>12 </td><td>12 </td></tr><tr><td>LG7 </td><td>48.9 </td><td>51 </td><td>1.0 </td><td>11 </td><td>6 </td><td>16 </td><td>16 </td><td>2 </td></tr><tr><td>LG8 </td><td>60.3 </td><td>64 </td><td>0.9 </td><td>30 </td><td>1 </td><td>17 </td><td>5 </td><td>11 </td></tr><tr><td>LG9 </td><td>103.2 </td><td>154 </td><td>0.7 </td><td>78 </td><td>6 </td><td>34 </td><td>17 </td><td>20 </td></tr><tr><td>LG10 </td><td>61.3 </td><td>54 </td><td>1.1 </td><td>17 </td><td>6 </td><td>12 </td><td>16 </td><td>3 </td></tr><tr><td>Total </td><td>751.7 </td><td>1043 </td><td>0.7 </td><td>461 </td><td>58 </td><td>251 </td><td>163 </td><td>110 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>SNP, single-nucleotide polymorphism; RFLP, restriction fragment polymorphism; SSR, simple sequence repeat; EST, expressed sequence tag.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7371072" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR039TB3" class="table-wrap-title" data-id="DSR039TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Distribution of each marker type in the LGs of the reference map (UPA402 × UF676)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LG </th><th>Length (cM) </th><th>Total number of markers </th><th>Average distance between markers (cM) </th><th>SNP </th><th>RFLP </th><th>Genomic SSR </th><th>SSR from scaffold </th><th>EST-SSR </th></tr></thead><tbody><tr><td>LG1 </td><td>90.3 </td><td>150 </td><td>0.6 </td><td>69 </td><td>8 </td><td>33 </td><td>22 </td><td>18 </td></tr><tr><td>LG2 </td><td>97.5 </td><td>126 </td><td>0.8 </td><td>55 </td><td>6 </td><td>26 </td><td>28 </td><td>11 </td></tr><tr><td>LG3 </td><td>74.4 </td><td>126 </td><td>0.6 </td><td>57 </td><td>6 </td><td>33 </td><td>18 </td><td>14 </td></tr><tr><td>LG4 </td><td>75.6 </td><td>120 </td><td>0.6 </td><td>61 </td><td>11 </td><td>27 </td><td>14 </td><td>8 </td></tr><tr><td>LG5 </td><td>77.4 </td><td>121 </td><td>0.6 </td><td>55 </td><td>6 </td><td>34 </td><td>15 </td><td>11 </td></tr><tr><td>LG6 </td><td>62.7 </td><td>73 </td><td>0.9 </td><td>28 </td><td>2 </td><td>19 </td><td>12 </td><td>12 </td></tr><tr><td>LG7 </td><td>48.9 </td><td>51 </td><td>1.0 </td><td>11 </td><td>6 </td><td>16 </td><td>16 </td><td>2 </td></tr><tr><td>LG8 </td><td>60.3 </td><td>64 </td><td>0.9 </td><td>30 </td><td>1 </td><td>17 </td><td>5 </td><td>11 </td></tr><tr><td>LG9 </td><td>103.2 </td><td>154 </td><td>0.7 </td><td>78 </td><td>6 </td><td>34 </td><td>17 </td><td>20 </td></tr><tr><td>LG10 </td><td>61.3 </td><td>54 </td><td>1.1 </td><td>17 </td><td>6 </td><td>12 </td><td>16 </td><td>3 </td></tr><tr><td>Total </td><td>751.7 </td><td>1043 </td><td>0.7 </td><td>461 </td><td>58 </td><td>251 </td><td>163 </td><td>110 </td></tr></tbody></table></div><div content-id="DSR039TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>LG </th><th>Length (cM) </th><th>Total number of markers </th><th>Average distance between markers (cM) </th><th>SNP </th><th>RFLP </th><th>Genomic SSR </th><th>SSR from scaffold </th><th>EST-SSR </th></tr></thead><tbody><tr><td>LG1 </td><td>90.3 </td><td>150 </td><td>0.6 </td><td>69 </td><td>8 </td><td>33 </td><td>22 </td><td>18 </td></tr><tr><td>LG2 </td><td>97.5 </td><td>126 </td><td>0.8 </td><td>55 </td><td>6 </td><td>26 </td><td>28 </td><td>11 </td></tr><tr><td>LG3 </td><td>74.4 </td><td>126 </td><td>0.6 </td><td>57 </td><td>6 </td><td>33 </td><td>18 </td><td>14 </td></tr><tr><td>LG4 </td><td>75.6 </td><td>120 </td><td>0.6 </td><td>61 </td><td>11 </td><td>27 </td><td>14 </td><td>8 </td></tr><tr><td>LG5 </td><td>77.4 </td><td>121 </td><td>0.6 </td><td>55 </td><td>6 </td><td>34 </td><td>15 </td><td>11 </td></tr><tr><td>LG6 </td><td>62.7 </td><td>73 </td><td>0.9 </td><td>28 </td><td>2 </td><td>19 </td><td>12 </td><td>12 </td></tr><tr><td>LG7 </td><td>48.9 </td><td>51 </td><td>1.0 </td><td>11 </td><td>6 </td><td>16 </td><td>16 </td><td>2 </td></tr><tr><td>LG8 </td><td>60.3 </td><td>64 </td><td>0.9 </td><td>30 </td><td>1 </td><td>17 </td><td>5 </td><td>11 </td></tr><tr><td>LG9 </td><td>103.2 </td><td>154 </td><td>0.7 </td><td>78 </td><td>6 </td><td>34 </td><td>17 </td><td>20 </td></tr><tr><td>LG10 </td><td>61.3 </td><td>54 </td><td>1.1 </td><td>17 </td><td>6 </td><td>12 </td><td>16 </td><td>3 </td></tr><tr><td>Total </td><td>751.7 </td><td>1043 </td><td>0.7 </td><td>461 </td><td>58 </td><td>251 </td><td>163 </td><td>110 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>SNP, single-nucleotide polymorphism; RFLP, restriction fragment polymorphism; SSR, simple sequence repeat; EST, expressed sequence tag.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7371072" target="_blank">View Large</a></div></div><p>Skewed segregation was observed for 118 markers (11.3%). The skewed markers were mainly located in LGs 2, 3, 6 and 10, as is shown in Fig. <a reveal-id="DSR039F3" data-open="DSR039F3" class="link link-reveal link-table xref-fig">3</a>.</p>                            <h4 scrollto-destination=7371074 id="7371074" class="section-title" data-legacy-id=s3f2>3.6.2. Map of the F2 population</h4>
<p>The F2 map (Fig. <a reveal-id="DSR039F2" data-open="DSR039F2" class="link link-reveal link-table xref-fig">2</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>) contained 531 EST-SNP markers. This map had a total length of 753.9 cM, with an average distance of 1.4 cM between neighboring markers. The marker density varied, with an average distance between neighboring markers ranging from 0.9 cM in LG 9 to 2.7 cM in LG7 (Table <a reveal-id="DSR039TB4" data-open="DSR039TB4" class="link link-reveal link-table xref-fig">4</a>). </p>                        <a id="7371076" scrollto-destination="7371076"></a>
<div content-id="DSR039TB4" class="table-modal"><div class="table-wrap"><div id="DSR039TB4" class="table-wrap-title" data-id="DSR039TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Distribution of SNP markers in the LGs of the F2 map (SCA6 × ICS1) selfing</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LG </th><th>Length (cM) </th><th>Number of SNP markers </th><th>Average distance between markers (cM) </th></tr></thead><tbody><tr><td>LG1 </td><td>98.6 </td><td>104 </td><td>0.9 </td></tr><tr><td>LG2 </td><td>102.8 </td><td>74 </td><td>1.4 </td></tr><tr><td>LG3 </td><td>78.7 </td><td>73 </td><td>1.1 </td></tr><tr><td>LG4 </td><td>69.1 </td><td>49 </td><td>1.4 </td></tr><tr><td>LG5 </td><td>85.4 </td><td>56 </td><td>1.5 </td></tr><tr><td>LG6 </td><td>71.0 </td><td>28 </td><td>2.5 </td></tr><tr><td>LG7 </td><td>48.7 </td><td>19 </td><td>2.6 </td></tr><tr><td>LG8 </td><td>43.3 </td><td>28 </td><td>1.5 </td></tr><tr><td>LG9 </td><td>104.9 </td><td>78 </td><td>1.3 </td></tr><tr><td>LG10 </td><td>51.5 </td><td>22 </td><td>2.3 </td></tr><tr><td>Total </td><td>753.9 </td><td>531 </td><td>1.4 </td></tr></tbody></table></div><div content-id="DSR039TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>LG </th><th>Length (cM) </th><th>Number of SNP markers </th><th>Average distance between markers (cM) </th></tr></thead><tbody><tr><td>LG1 </td><td>98.6 </td><td>104 </td><td>0.9 </td></tr><tr><td>LG2 </td><td>102.8 </td><td>74 </td><td>1.4 </td></tr><tr><td>LG3 </td><td>78.7 </td><td>73 </td><td>1.1 </td></tr><tr><td>LG4 </td><td>69.1 </td><td>49 </td><td>1.4 </td></tr><tr><td>LG5 </td><td>85.4 </td><td>56 </td><td>1.5 </td></tr><tr><td>LG6 </td><td>71.0 </td><td>28 </td><td>2.5 </td></tr><tr><td>LG7 </td><td>48.7 </td><td>19 </td><td>2.6 </td></tr><tr><td>LG8 </td><td>43.3 </td><td>28 </td><td>1.5 </td></tr><tr><td>LG9 </td><td>104.9 </td><td>78 </td><td>1.3 </td></tr><tr><td>LG10 </td><td>51.5 </td><td>22 </td><td>2.3 </td></tr><tr><td>Total </td><td>753.9 </td><td>531 </td><td>1.4 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>SNP, single-nucleotide.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7371076" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR039TB4" class="table-wrap-title" data-id="DSR039TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Distribution of SNP markers in the LGs of the F2 map (SCA6 × ICS1) selfing</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LG </th><th>Length (cM) </th><th>Number of SNP markers </th><th>Average distance between markers (cM) </th></tr></thead><tbody><tr><td>LG1 </td><td>98.6 </td><td>104 </td><td>0.9 </td></tr><tr><td>LG2 </td><td>102.8 </td><td>74 </td><td>1.4 </td></tr><tr><td>LG3 </td><td>78.7 </td><td>73 </td><td>1.1 </td></tr><tr><td>LG4 </td><td>69.1 </td><td>49 </td><td>1.4 </td></tr><tr><td>LG5 </td><td>85.4 </td><td>56 </td><td>1.5 </td></tr><tr><td>LG6 </td><td>71.0 </td><td>28 </td><td>2.5 </td></tr><tr><td>LG7 </td><td>48.7 </td><td>19 </td><td>2.6 </td></tr><tr><td>LG8 </td><td>43.3 </td><td>28 </td><td>1.5 </td></tr><tr><td>LG9 </td><td>104.9 </td><td>78 </td><td>1.3 </td></tr><tr><td>LG10 </td><td>51.5 </td><td>22 </td><td>2.3 </td></tr><tr><td>Total </td><td>753.9 </td><td>531 </td><td>1.4 </td></tr></tbody></table></div><div content-id="DSR039TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>LG </th><th>Length (cM) </th><th>Number of SNP markers </th><th>Average distance between markers (cM) </th></tr></thead><tbody><tr><td>LG1 </td><td>98.6 </td><td>104 </td><td>0.9 </td></tr><tr><td>LG2 </td><td>102.8 </td><td>74 </td><td>1.4 </td></tr><tr><td>LG3 </td><td>78.7 </td><td>73 </td><td>1.1 </td></tr><tr><td>LG4 </td><td>69.1 </td><td>49 </td><td>1.4 </td></tr><tr><td>LG5 </td><td>85.4 </td><td>56 </td><td>1.5 </td></tr><tr><td>LG6 </td><td>71.0 </td><td>28 </td><td>2.5 </td></tr><tr><td>LG7 </td><td>48.7 </td><td>19 </td><td>2.6 </td></tr><tr><td>LG8 </td><td>43.3 </td><td>28 </td><td>1.5 </td></tr><tr><td>LG9 </td><td>104.9 </td><td>78 </td><td>1.3 </td></tr><tr><td>LG10 </td><td>51.5 </td><td>22 </td><td>2.3 </td></tr><tr><td>Total </td><td>753.9 </td><td>531 </td><td>1.4 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>SNP, single-nucleotide.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7371076" target="_blank">View Large</a></div></div>                        <a id="7371077" scrollto-destination="7371077"></a>
<div id="DSR039F2" data-id="DSR039F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr039/2/m_dsr03902.jpeg?Expires=2147483647&Signature=YtOaRQ6dahCn4UJYJkiETFFQNwi31dZQKm0C7zzxwqA8KrV9L2VEEbge0VSpjsFUu0r3criSx5T9nTZZa98KDqZQUd9m7lc0bP2mxjayNgAf4yfCitYz~65WeUcWml3X8kCnBkkbch3e4f8k-fHqZl6JS7mev9nJjZ~SqqO6eQygGGP7oPEC2jp3KpsE0DovAES-8IbYwONVuuzL5LkwS7wk4mce6i2kxOJdmRxOJFB3WKuvrAcDKfZcwVDQktpBGMYcDxkfdekQB7YUJqBSCNw17R-TXjoDh7bm-RCHzkSxeB8o3ZAx~kSsRLYQoJxeNAFmYflQ~Zg7kCPRQk3G7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic map constructed from an F2 progeny of 132 individuals (located at CEPLAC, Brazil) obtained by selfing of a single (Scavina 6 × ICS1) selected hybrid. This map consists of 531 SNP markers, spanning 754 cM. The average distance between two markers is 1.4 cM." path-from-xml="dsr03902.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7371077/dsr03902.jpeg" path-from-xml="dsr03902.jpeg" target="_blank">View large</a><a section="7371077" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr039/2/dsr03902.jpeg?Expires=2147483647&Signature=J2zCNnmOFigicZjKGzGWKT4LeoS8VJ6EbIYP0~aNZyYjwFV0LbyZKySUpNG67bKyBGy7Ao8PzK9ETWG-krQeIHOIjDeIZUiBIUWC3ApX4Q83c2bognTzAr0xUId07fBkpde~9imJGh~Z5-QPnrNr2XZ1iuXK61T0AKlZIpY0ESOsZEaQpazxVzWhBwV9xsj1zCPx1NbnqFCsOb5goeetdPQRYAdxSacjD3~ZvCw3zDuNdjpFjrIMJ3~0JWnCzIvaPrzjOcFQWEun-ue6DIqvQ0Jf-Z0V~rSLHbR6WPhBYngYq7-a9Igfxp5cM~KbuUjTJ0ElVUYzqqQbDv6ae-MkQg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7371077&ar=484949&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03902.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic map constructed from an F2 progeny of 132 individuals (located at CEPLAC, Brazil) obtained by selfing of a single (Scavina 6 × ICS1) selected hybrid. This map consists of 531 SNP markers, spanning 754 cM. The average distance between two markers is 1.4 cM.</p></div></div><div content-id="DSR039F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr039/2/m_dsr03902.jpeg?Expires=2147483647&Signature=YtOaRQ6dahCn4UJYJkiETFFQNwi31dZQKm0C7zzxwqA8KrV9L2VEEbge0VSpjsFUu0r3criSx5T9nTZZa98KDqZQUd9m7lc0bP2mxjayNgAf4yfCitYz~65WeUcWml3X8kCnBkkbch3e4f8k-fHqZl6JS7mev9nJjZ~SqqO6eQygGGP7oPEC2jp3KpsE0DovAES-8IbYwONVuuzL5LkwS7wk4mce6i2kxOJdmRxOJFB3WKuvrAcDKfZcwVDQktpBGMYcDxkfdekQB7YUJqBSCNw17R-TXjoDh7bm-RCHzkSxeB8o3ZAx~kSsRLYQoJxeNAFmYflQ~Zg7kCPRQk3G7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic map constructed from an F2 progeny of 132 individuals (located at CEPLAC, Brazil) obtained by selfing of a single (Scavina 6 × ICS1) selected hybrid. This map consists of 531 SNP markers, spanning 754 cM. The average distance between two markers is 1.4 cM." path-from-xml="dsr03902.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7371077/dsr03902.jpeg" path-from-xml="dsr03902.jpeg" target="_blank">View large</a><a section="7371077" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr039/2/dsr03902.jpeg?Expires=2147483647&Signature=J2zCNnmOFigicZjKGzGWKT4LeoS8VJ6EbIYP0~aNZyYjwFV0LbyZKySUpNG67bKyBGy7Ao8PzK9ETWG-krQeIHOIjDeIZUiBIUWC3ApX4Q83c2bognTzAr0xUId07fBkpde~9imJGh~Z5-QPnrNr2XZ1iuXK61T0AKlZIpY0ESOsZEaQpazxVzWhBwV9xsj1zCPx1NbnqFCsOb5goeetdPQRYAdxSacjD3~ZvCw3zDuNdjpFjrIMJ3~0JWnCzIvaPrzjOcFQWEun-ue6DIqvQ0Jf-Z0V~rSLHbR6WPhBYngYq7-a9Igfxp5cM~KbuUjTJ0ElVUYzqqQbDv6ae-MkQg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7371077&ar=484949&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03902.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic map constructed from an F2 progeny of 132 individuals (located at CEPLAC, Brazil) obtained by selfing of a single (Scavina 6 × ICS1) selected hybrid. This map consists of 531 SNP markers, spanning 754 cM. The average distance between two markers is 1.4 cM.</p></div></div><p>Skewed segregation was observed in 97 markers (18.3%). The skewed markers were mainly located in LGs 1, 3 and 4, as is shown in Fig. <a reveal-id="DSR039F3" data-open="DSR039F3" class="link link-reveal link-table xref-fig">3</a>. </p>                        <a id="7371079" scrollto-destination="7371079"></a>
<div id="DSR039F3" data-id="DSR039F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr039/2/m_dsr03903.jpeg?Expires=2147483647&Signature=DwhxVk~5ATkdVVtzLDrXtsjo4j7oDGxpd~t4mPbaAvbJqP4ZISSsEylH85GMmlaOqN3F1onu~KBpgkQcgtJHtLCqYhcDvTa7RGedxNUgpwektlri0dkf8DcbVxQBz4uisE1lJoVIPZ7Fv9yioioib1WAogPzAhpeHdzUBZ2w8c1AUBKHShZcAelk0Ic3VKiL3ZS9N64YQUNaEUYMVfqvCYwZRObXK5kwFpyEPw9bcf2G9OvrKUYPUYhtO02FvKMNYFxREgADdEq9~ltQ87FAk9Cuee6Bz~ux4XEX1gyJNhGevPTwh8DmIQFk8GlUjSnxNSDUBY0ffpR9NymXYgVQuw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Consensus map of (UPA402 × UF676) and F2 progenies. Markers segregating in both progenies are indicated in black, those segregating only in (UPA402 × UF676) are printed in green (previously mapped markers) and blue (newly mapped markers). Markers segregating only in the F2 progeny are printed in pink. This consensus map consists of 1262 markers of a known DNA sequence, and it has a length of 734 cM. The average distance between two markers is 0.6 cM. Among the 1262 markers, 810 correspond to markers defined in expressed genes. Significant skewed segregations are indicated by asterisks (*P&lt; 0.05, **P&lt; 0.01) or dots (F2 population)." path-from-xml="dsr03903.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7371079/dsr03903.jpeg" path-from-xml="dsr03903.jpeg" target="_blank">View large</a><a section="7371079" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr039/2/dsr03903.jpeg?Expires=2147483647&Signature=XeMUmGWX94zAoZfHTLn974lovc-Te9xeZHidXq6CF~aee~o~NCp6n4Puc2GVW14JppOzIu4kF63F-4DcG2kmLpYOiUtHJ0NIqaZVybCkKUYoubKK4vJuiqWLppoqseyl7tWHTc5iI9~GfsBa6cpYFE6WvYgpwXlHZFMqpgF1b~A-0Skwb1MUD1GGbneb6-4VpLBoQrXoZ2nB~AvdySJA8GyHFvKxAJLpzKyDMiHGsBy9vnoo91e2ll9Mz3IHSm1WUDaw544tJLNJgM4-owvUIoVcpLMbF41EhyjfLXjaBQuLNAT95aObutbsfwSLnP~ZHPea3g4pJcRXhtO9uYKQaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7371079&ar=484949&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03903.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Consensus map of (UPA402 × UF676) and F2 progenies. Markers segregating in both progenies are indicated in black, those segregating only in (UPA402 × UF676) are printed in green (previously mapped markers) and blue (newly mapped markers). Markers segregating only in the F2 progeny are printed in pink. This consensus map consists of 1262 markers of a known DNA sequence, and it has a length of 734 cM. The average distance between two markers is 0.6 cM. Among the 1262 markers, 810 correspond to markers defined in expressed genes. Significant skewed segregations are indicated by asterisks (*<em>P</em>&lt; 0.05, **<em>P</em>&lt; 0.01) or dots (F2 population).</p></div></div><div content-id="DSR039F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr039/2/m_dsr03903.jpeg?Expires=2147483647&Signature=DwhxVk~5ATkdVVtzLDrXtsjo4j7oDGxpd~t4mPbaAvbJqP4ZISSsEylH85GMmlaOqN3F1onu~KBpgkQcgtJHtLCqYhcDvTa7RGedxNUgpwektlri0dkf8DcbVxQBz4uisE1lJoVIPZ7Fv9yioioib1WAogPzAhpeHdzUBZ2w8c1AUBKHShZcAelk0Ic3VKiL3ZS9N64YQUNaEUYMVfqvCYwZRObXK5kwFpyEPw9bcf2G9OvrKUYPUYhtO02FvKMNYFxREgADdEq9~ltQ87FAk9Cuee6Bz~ux4XEX1gyJNhGevPTwh8DmIQFk8GlUjSnxNSDUBY0ffpR9NymXYgVQuw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Consensus map of (UPA402 × UF676) and F2 progenies. Markers segregating in both progenies are indicated in black, those segregating only in (UPA402 × UF676) are printed in green (previously mapped markers) and blue (newly mapped markers). Markers segregating only in the F2 progeny are printed in pink. This consensus map consists of 1262 markers of a known DNA sequence, and it has a length of 734 cM. The average distance between two markers is 0.6 cM. Among the 1262 markers, 810 correspond to markers defined in expressed genes. Significant skewed segregations are indicated by asterisks (*P&lt; 0.05, **P&lt; 0.01) or dots (F2 population)." path-from-xml="dsr03903.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7371079/dsr03903.jpeg" path-from-xml="dsr03903.jpeg" target="_blank">View large</a><a section="7371079" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr039/2/dsr03903.jpeg?Expires=2147483647&Signature=XeMUmGWX94zAoZfHTLn974lovc-Te9xeZHidXq6CF~aee~o~NCp6n4Puc2GVW14JppOzIu4kF63F-4DcG2kmLpYOiUtHJ0NIqaZVybCkKUYoubKK4vJuiqWLppoqseyl7tWHTc5iI9~GfsBa6cpYFE6WvYgpwXlHZFMqpgF1b~A-0Skwb1MUD1GGbneb6-4VpLBoQrXoZ2nB~AvdySJA8GyHFvKxAJLpzKyDMiHGsBy9vnoo91e2ll9Mz3IHSm1WUDaw544tJLNJgM4-owvUIoVcpLMbF41EhyjfLXjaBQuLNAT95aObutbsfwSLnP~ZHPea3g4pJcRXhtO9uYKQaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7371079&ar=484949&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03903.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Consensus map of (UPA402 × UF676) and F2 progenies. Markers segregating in both progenies are indicated in black, those segregating only in (UPA402 × UF676) are printed in green (previously mapped markers) and blue (newly mapped markers). Markers segregating only in the F2 progeny are printed in pink. This consensus map consists of 1262 markers of a known DNA sequence, and it has a length of 734 cM. The average distance between two markers is 0.6 cM. Among the 1262 markers, 810 correspond to markers defined in expressed genes. Significant skewed segregations are indicated by asterisks (*<em>P</em>&lt; 0.05, **<em>P</em>&lt; 0.01) or dots (F2 population).</p></div></div>                            <h3 scrollto-destination=7371080 id="7371080" class="section-title" data-legacy-id=s3g>3.7. Consensus genetic linkage map</h3>
<p>Two hundred and thirty-nine SNP markers were mapped in both populations.</p><p>The complete consensus map (Table <a reveal-id="DSR039TB5" data-open="DSR039TB5" class="link link-reveal link-table xref-fig">5</a>) contained 1262 codominant markers including 681 EST-SNPs, 523 SSRs (163 scaffold-tagged-SSRs, 110 EST-SSRs, 250 SSRs from genomic DNA) and 58 RFLPs including 14 resistance gene analogues (Rgenes-RFLPs), arranged in 10 LGs corresponding to the haploid chromosome number of <em>T. cacao</em> (Fig. <a reveal-id="DSR039F3" data-open="DSR039F3" class="link link-reveal link-table xref-fig">3</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). Among the 1262 markers, 65% were gene-based markers, including SNPs, SSRs and RFLPs. </p>                        <a id="7371083" scrollto-destination="7371083"></a>
<div content-id="DSR039TB5" class="table-modal"><div class="table-wrap"><div id="DSR039TB5" class="table-wrap-title" data-id="DSR039TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Distribution of each marker type in the LGs of the consensus genetic map</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LG </th><th>Length (cM) </th><th>Total number of markers </th><th>Average distance between markers (cM) </th><th>SNP </th><th>RFLP </th><th>Genomic SSR </th><th>SSR from scaffold </th><th>EST-SSR </th></tr></thead><tbody><tr><td>LG1 </td><td>77.1 </td><td>201 </td><td>0.4 </td><td>120 </td><td>8 </td><td>33 </td><td>22 </td><td>18 </td></tr><tr><td>LG2 </td><td>101.1 </td><td>156 </td><td>0.6 </td><td>85 </td><td>6 </td><td>26 </td><td>28 </td><td>11 </td></tr><tr><td>LG3 </td><td>76.9 </td><td>162 </td><td>0.5 </td><td>91 </td><td>6 </td><td>33 </td><td>18 </td><td>14 </td></tr><tr><td>LG4 </td><td>64.2 </td><td>135 </td><td>0.5 </td><td>75 </td><td>11 </td><td>27 </td><td>14 </td><td>8 </td></tr><tr><td>LG5 </td><td>78.1 </td><td>147 </td><td>0.5 </td><td>81 </td><td>6 </td><td>34 </td><td>15 </td><td>11 </td></tr><tr><td>LG6 </td><td>64 </td><td>81 </td><td>0.8 </td><td>36 </td><td>2 </td><td>19 </td><td>12 </td><td>12 </td></tr><tr><td>LG7 </td><td>52.6 </td><td>62 </td><td>0.8 </td><td>22 </td><td>6 </td><td>16 </td><td>16 </td><td>2 </td></tr><tr><td>LG8 </td><td>59.2 </td><td>73 </td><td>0.8 </td><td>39 </td><td>1 </td><td>17 </td><td>5 </td><td>11 </td></tr><tr><td>LG9 </td><td>100.9 </td><td>182 </td><td>0.6 </td><td>106 </td><td>6 </td><td>33 </td><td>17 </td><td>20 </td></tr><tr><td>LG10 </td><td>59.5 </td><td>63 </td><td>0.9 </td><td>26 </td><td>6 </td><td>12 </td><td>16 </td><td>3 </td></tr><tr><td>Total </td><td>733.6 </td><td>1262 </td><td>0.6 </td><td>681 </td><td>58 </td><td>250 </td><td>163 </td><td>110 </td></tr></tbody></table></div><div content-id="DSR039TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>LG </th><th>Length (cM) </th><th>Total number of markers </th><th>Average distance between markers (cM) </th><th>SNP </th><th>RFLP </th><th>Genomic SSR </th><th>SSR from scaffold </th><th>EST-SSR </th></tr></thead><tbody><tr><td>LG1 </td><td>77.1 </td><td>201 </td><td>0.4 </td><td>120 </td><td>8 </td><td>33 </td><td>22 </td><td>18 </td></tr><tr><td>LG2 </td><td>101.1 </td><td>156 </td><td>0.6 </td><td>85 </td><td>6 </td><td>26 </td><td>28 </td><td>11 </td></tr><tr><td>LG3 </td><td>76.9 </td><td>162 </td><td>0.5 </td><td>91 </td><td>6 </td><td>33 </td><td>18 </td><td>14 </td></tr><tr><td>LG4 </td><td>64.2 </td><td>135 </td><td>0.5 </td><td>75 </td><td>11 </td><td>27 </td><td>14 </td><td>8 </td></tr><tr><td>LG5 </td><td>78.1 </td><td>147 </td><td>0.5 </td><td>81 </td><td>6 </td><td>34 </td><td>15 </td><td>11 </td></tr><tr><td>LG6 </td><td>64 </td><td>81 </td><td>0.8 </td><td>36 </td><td>2 </td><td>19 </td><td>12 </td><td>12 </td></tr><tr><td>LG7 </td><td>52.6 </td><td>62 </td><td>0.8 </td><td>22 </td><td>6 </td><td>16 </td><td>16 </td><td>2 </td></tr><tr><td>LG8 </td><td>59.2 </td><td>73 </td><td>0.8 </td><td>39 </td><td>1 </td><td>17 </td><td>5 </td><td>11 </td></tr><tr><td>LG9 </td><td>100.9 </td><td>182 </td><td>0.6 </td><td>106 </td><td>6 </td><td>33 </td><td>17 </td><td>20 </td></tr><tr><td>LG10 </td><td>59.5 </td><td>63 </td><td>0.9 </td><td>26 </td><td>6 </td><td>12 </td><td>16 </td><td>3 </td></tr><tr><td>Total </td><td>733.6 </td><td>1262 </td><td>0.6 </td><td>681 </td><td>58 </td><td>250 </td><td>163 </td><td>110 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>SNP, single nucleotide polymorphism; RFLP, restriction fragment polymorphism; SSR, simple sequence repeat; EST, expressed sequence tag.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7371083" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR039TB5" class="table-wrap-title" data-id="DSR039TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Distribution of each marker type in the LGs of the consensus genetic map</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LG </th><th>Length (cM) </th><th>Total number of markers </th><th>Average distance between markers (cM) </th><th>SNP </th><th>RFLP </th><th>Genomic SSR </th><th>SSR from scaffold </th><th>EST-SSR </th></tr></thead><tbody><tr><td>LG1 </td><td>77.1 </td><td>201 </td><td>0.4 </td><td>120 </td><td>8 </td><td>33 </td><td>22 </td><td>18 </td></tr><tr><td>LG2 </td><td>101.1 </td><td>156 </td><td>0.6 </td><td>85 </td><td>6 </td><td>26 </td><td>28 </td><td>11 </td></tr><tr><td>LG3 </td><td>76.9 </td><td>162 </td><td>0.5 </td><td>91 </td><td>6 </td><td>33 </td><td>18 </td><td>14 </td></tr><tr><td>LG4 </td><td>64.2 </td><td>135 </td><td>0.5 </td><td>75 </td><td>11 </td><td>27 </td><td>14 </td><td>8 </td></tr><tr><td>LG5 </td><td>78.1 </td><td>147 </td><td>0.5 </td><td>81 </td><td>6 </td><td>34 </td><td>15 </td><td>11 </td></tr><tr><td>LG6 </td><td>64 </td><td>81 </td><td>0.8 </td><td>36 </td><td>2 </td><td>19 </td><td>12 </td><td>12 </td></tr><tr><td>LG7 </td><td>52.6 </td><td>62 </td><td>0.8 </td><td>22 </td><td>6 </td><td>16 </td><td>16 </td><td>2 </td></tr><tr><td>LG8 </td><td>59.2 </td><td>73 </td><td>0.8 </td><td>39 </td><td>1 </td><td>17 </td><td>5 </td><td>11 </td></tr><tr><td>LG9 </td><td>100.9 </td><td>182 </td><td>0.6 </td><td>106 </td><td>6 </td><td>33 </td><td>17 </td><td>20 </td></tr><tr><td>LG10 </td><td>59.5 </td><td>63 </td><td>0.9 </td><td>26 </td><td>6 </td><td>12 </td><td>16 </td><td>3 </td></tr><tr><td>Total </td><td>733.6 </td><td>1262 </td><td>0.6 </td><td>681 </td><td>58 </td><td>250 </td><td>163 </td><td>110 </td></tr></tbody></table></div><div content-id="DSR039TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>LG </th><th>Length (cM) </th><th>Total number of markers </th><th>Average distance between markers (cM) </th><th>SNP </th><th>RFLP </th><th>Genomic SSR </th><th>SSR from scaffold </th><th>EST-SSR </th></tr></thead><tbody><tr><td>LG1 </td><td>77.1 </td><td>201 </td><td>0.4 </td><td>120 </td><td>8 </td><td>33 </td><td>22 </td><td>18 </td></tr><tr><td>LG2 </td><td>101.1 </td><td>156 </td><td>0.6 </td><td>85 </td><td>6 </td><td>26 </td><td>28 </td><td>11 </td></tr><tr><td>LG3 </td><td>76.9 </td><td>162 </td><td>0.5 </td><td>91 </td><td>6 </td><td>33 </td><td>18 </td><td>14 </td></tr><tr><td>LG4 </td><td>64.2 </td><td>135 </td><td>0.5 </td><td>75 </td><td>11 </td><td>27 </td><td>14 </td><td>8 </td></tr><tr><td>LG5 </td><td>78.1 </td><td>147 </td><td>0.5 </td><td>81 </td><td>6 </td><td>34 </td><td>15 </td><td>11 </td></tr><tr><td>LG6 </td><td>64 </td><td>81 </td><td>0.8 </td><td>36 </td><td>2 </td><td>19 </td><td>12 </td><td>12 </td></tr><tr><td>LG7 </td><td>52.6 </td><td>62 </td><td>0.8 </td><td>22 </td><td>6 </td><td>16 </td><td>16 </td><td>2 </td></tr><tr><td>LG8 </td><td>59.2 </td><td>73 </td><td>0.8 </td><td>39 </td><td>1 </td><td>17 </td><td>5 </td><td>11 </td></tr><tr><td>LG9 </td><td>100.9 </td><td>182 </td><td>0.6 </td><td>106 </td><td>6 </td><td>33 </td><td>17 </td><td>20 </td></tr><tr><td>LG10 </td><td>59.5 </td><td>63 </td><td>0.9 </td><td>26 </td><td>6 </td><td>12 </td><td>16 </td><td>3 </td></tr><tr><td>Total </td><td>733.6 </td><td>1262 </td><td>0.6 </td><td>681 </td><td>58 </td><td>250 </td><td>163 </td><td>110 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>SNP, single nucleotide polymorphism; RFLP, restriction fragment polymorphism; SSR, simple sequence repeat; EST, expressed sequence tag.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7371083" target="_blank">View Large</a></div></div><p>The total map length was 733.6 cM, i.e. slightly shorter than previously constructed maps (782.8 cM for Pugh <em>et al.</em><sup><a reveal-id="DSR039C8" data-open="DSR039C8" class="link link-ref link-reveal xref-bibr">8</a></sup> and 779.2 cM for Fouet <em>et al.</em><sup><a reveal-id="DSR039C9" data-open="DSR039C9" class="link link-ref link-reveal xref-bibr">9</a></sup>). The average distance between adjacent markers on this map was 0.6 cM, and thus shorter than the 1.3 cM of the map of Fouet <em>et al</em>.<sup><a reveal-id="DSR039C9" data-open="DSR039C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p><p>The number of mapped loci varied substantially between LGs on the consensus map; from 63 in LG10 to 201 in LG1. The average distance between two markers in the different LGs ranged from 0.4 cM in LG1 to 0.9 cM in LG10.</p><p>In total, 844 new markers (681 SNP markers and 163 SSR defined in scaffolds) were mapped. These new markers were well distributed over all chromosomes allowing to fill some gaps in the previous maps, for example on chromosome 10.</p>                            <h2 scrollto-destination=7371087 id="7371087" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>A large set of EST-SNP markers was generated and mapped in <em>T. cacao.</em> New SSR markers were added to these SNPs, providing an efficient tool for high-throughput genotyping of cocoa populations.</p><p>SSR markers are multiallelic and well adapted for fine analysis of population diversity structure.<sup><a reveal-id="DSR039C40  DSR039C43" data-open="DSR039C40  DSR039C43" class="link link-ref link-reveal xref-bibr">40–43</a></sup> In <em>T. cacao</em>, an average number of 5.8 alleles per SSR was observed by Loor Solorzano<sup><a reveal-id="DSR039C44" data-open="DSR039C44" class="link link-ref link-reveal xref-bibr">44</a></sup> after genotyping a collection of genetic resources of various genetic origins, and with a maximum of 15 alleles revealed by one SSR (mTcCIR322). This is not the case for SNPs that are only biallelic, but a higher number of SNP markers (several thousands) can be easily revealed at once using high-throughput technologies.</p><p>We used our new SNP and SSR markers to construct a very high-density genetic map. Sixty-five per cent of the markers were from within genes and the average distance between adjacent markers was 0.6 cM.</p><p>Several chromosome regions include markers with skewed segregations, particularly on LG 1, LG 3, LG 4, and LG 6. The region on LG 4 includes the locus for self-incompatibility previously identified by Crouzillat <em>et al.</em><sup><a reveal-id="DSR039C5" data-open="DSR039C5" class="link link-ref link-reveal xref-bibr">5</a></sup> The gameto-sporophytic incompatibility system existing in <em>T. cacao</em><sup><a reveal-id="DSR039C45" data-open="DSR039C45" class="link link-ref link-reveal xref-bibr">45</a>,<a reveal-id="DSR039C46" data-open="DSR039C46" class="link link-ref link-reveal xref-bibr">46</a></sup> could possibly explain the segregation distortion on this LG 4 region. Other factors which could explain segregation distortion, such as chromosome rearrangements in banana<sup><a reveal-id="DSR039C47" data-open="DSR039C47" class="link link-ref link-reveal xref-bibr">47</a></sup> which are responsible for highly skewed marker segregations, have not been reported in <em>T. cacao</em>.</p><p>This high-density genetic map can be used as a major tool for efficient genome-wide association studies (GWASs) in <em>T. cacao</em> populations. This method, first applied in human and animal genetics,<sup><a reveal-id="DSR039C48  DSR039C51" data-open="DSR039C48  DSR039C51" class="link link-ref link-reveal xref-bibr">48–51</a></sup> was also found to be highly effective for studying the determinism of useful traits in plants,<sup><a reveal-id="DSR039C52  DSR039C56" data-open="DSR039C52  DSR039C56" class="link link-ref link-reveal xref-bibr">52–56</a></sup> particularly in cocoa with the analysis of some recent hybrid populations.<sup><a reveal-id="DSR039C8" data-open="DSR039C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSR039C57" data-open="DSR039C57" class="link link-ref link-reveal xref-bibr">57</a>,<a reveal-id="DSR039C58" data-open="DSR039C58" class="link link-ref link-reveal xref-bibr">58</a></sup> GWAS is an alternative to QTL analyses in cross progenies for the purpose of studying genetic control of phenotypic traits in cocoa.</p><p>GWASs can be carried out on unrelated genetic resources such as wild or cultivated populations or germplasm collections. Large cocoa germplasm collections are maintained in many countries and characterized for useful traits. Two international cocoa collections are hosted at the International Cocoa Genebank, Trinidad (ICG,T, preserving 2300 accessions),<sup><a reveal-id="DSR039C59" data-open="DSR039C59" class="link link-ref link-reveal xref-bibr">59</a></sup> and at the Centro Agronomico Tropical de Investigacion y Ensenanza, Turrialba, Costa Rica (CATIE, preserving 1150 accessions).<sup><a reveal-id="DSR039C60" data-open="DSR039C60" class="link link-ref link-reveal xref-bibr">60</a></sup> The markers identified here will now certainly facilitate such GWASs, providing added-value to this wide characterization work, thus boosting knowledge on the genetic determinants of useful cocoa traits.</p><p>Another benefit of this large set of mapped markers is the possible integration of molecular information in conventional cocoa breeding schemes using marker-assisted selection (MAS).</p><p>In cocoa, few MAS experiments are currently underway.<sup><a reveal-id="DSR039C3" data-open="DSR039C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSR039C61" data-open="DSR039C61" class="link link-ref link-reveal xref-bibr">61</a></sup> The efficiency of MAS in selecting <em>P. palmivora</em>-resistant cocoa plants has been reported by Lanaud <em>et al</em>.<sup><a reveal-id="DSR039C3" data-open="DSR039C3" class="link link-ref link-reveal xref-bibr">3</a></sup></p><p>Until now, MAS studies have mainly been focused on traits controlled by a small number of genes, using only markers close to QTLs. However, such methods are of limited use for traits that are determined by a large number of genes of small effects.</p><p>Substantial genome-wide molecular data can now be generated at lower cost by high-throughput technologies, such as SNP genotyping. This progress has paved the way for the development of new methods to predict genotype value via MAS. The genome-wide selection or genomic selection (GS) method was recently successfully applied in animal or plant breeding<sup><a reveal-id="DSR039C62  DSR039C66" data-open="DSR039C62  DSR039C66" class="link link-ref link-reveal xref-bibr">62–66</a></sup> and allows to predict phenotypes using all marker information.</p><p>The integration of molecular markers in cocoa recurrent breeding programmes<sup><a reveal-id="DSR039C67  DSR039C72" data-open="DSR039C67  DSR039C72" class="link link-ref link-reveal xref-bibr">67–72</a></sup> could be facilitated by the GS approach in order to accelerate genetic gains. The GS strategy seems particularly suitable for the selection of multigenic traits such as yield and disease resistance. Cumulating a large number of resistance alleles is one of the main objectives of cocoa breeding for sustainable cocoa resistance. The large set of available SNP markers could facilitate the selection of resistant and high yielding cocoa trees via GS approaches enabling the use of all genome regions tagged by SNP markers, even those with very small effects.</p><p>The search for candidate genes underlying trait variation is another major challenge for plant biologists, with the aim of gaining further insight into the mechanisms underlying trait variation, and producing tools to efficiently screen and exploit genetic resources.</p><p>The consensus map produced in this work has been used efficiently for anchoring an assembly of <em>T. cacao</em> Criollo genome sequences, and for constituting pseudomolecules.<sup><a reveal-id="DSR039C1" data-open="DSR039C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Recently, two different cocoa varieties, i.e. Criollo<sup><a reveal-id="DSR039C1" data-open="DSR039C1" class="link link-ref link-reveal xref-bibr">1</a></sup> and Forastero from the Lower Amazon region (<a class="link link-uri" href="http://www.cacaogenomedb.org/">http://www.cacaogenomedb.org/</a>), were sequenced, with 28 798 and 35 000 annotated genes, respectively. These sequences will greatly facilitate the identification of candidate genes, allowing integration of both genetic and genomic (functional and structural) data. Overall, about 300 QTLs or marker/trait associations have already been identified in <em>T. cacao</em>. High-throughput genotyping associated with a high marker density will facilitate fine mapping of genes involved in trait variation (with GWAS or classical QTL analyses conducted on large progenies), thus allowing to refine the QTL position in the genome, while facilitating the search for candidate genes in corresponding genome sequences. Several functional studies have already been conducted in cocoa, focused mainly on genes generally expressed in specific physiological conditions or metabolisms.<sup><a reveal-id="DSR039C73" data-open="DSR039C73" class="link link-ref link-reveal xref-bibr">73</a></sup> It will be now possible to focus more specifically on the expression of genes directly responsible for trait variation after candidate gene validation.</p><p>Analysing genome evolution during domestication processes or adaptation to climate change can also help us to identify key genes underlying adaptive traits.<sup><a reveal-id="DSR039C74" data-open="DSR039C74" class="link link-ref link-reveal xref-bibr">74</a></sup> Loss of diversity generally occurs during genome evolution, and some genes are selectively involved in natural selection or domestication. A large set of SNPs defined in expressed genes, such as those identified in this study, provides a key tool for identifying selection signatures or adaptive substitutions, and then highlighting candidate genes potentially involved in the adaptation<sup><a reveal-id="DSR039C60" data-open="DSR039C60" class="link link-ref link-reveal xref-bibr">60</a></sup> or domestication processes and their corresponding molecular functions.<sup><a reveal-id="DSR039C75" data-open="DSR039C75" class="link link-ref link-reveal xref-bibr">75</a></sup> All SNPs reported in this paper were identified in orthologous genes or gene families, thus facilitating comparative genomic approaches, and benefiting from gene knowledge accumulated in other species to accelerate cocoa breeding.</p>                            <h2 scrollto-destination=7371102 id="7371102" class="section-title" data-legacy-id=s5>5. Availability</h2>
<p>Information on the consensus linkage map, molecular markers, and primers are available in the Map Study ‘SSR_SNP_consensus_map’ of the cocoa module of TropGeneDB database (<a class="link link-uri" href="http://tropgenedb.cirad.fr">http://tropgenedb.cirad.fr</a>).</p>                            <h2 scrollto-destination=7371104 id="7371104" class="section-title" data-legacy-id=s6>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=7371106 id="7371106" class="section-title" data-legacy-id=s7>Funding</h2>
<p>We thank the Region Languedoc-Roussillon, Valrhona and the Fondation Agropolis for their financial support in this work.</p>                            <h2 scrollto-destination=7371108 id="7371108" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We acknowledge Chantal Hamelin for TropGeneDB support. We thank Peter Biggins for improving the English in this paper, and the reviewers for their useful comments and editing job of this paper.</p>                            <h2 scrollto-destination=7371110 id="7371110" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSR039C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Argout</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Salse</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Aury</div> <div class="given-names">J.M.</div></div>, et al. </div><div class="article-title">The genome of <em>Theobroma cacao</em></div>, <div class="source">Nat. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">101</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20Theobroma%20cacao&amp;author=X.%20Argout&amp;author=J.%20Salse&amp;author=J.M.%20Aury&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=101-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.736" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.736" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.736"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21186351" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cheesman</div> <div class="given-names">E.E.</div></div>. </div><div class="article-title">Notes on the nomenclature, classification and possible relationships of cocoa populations</div>, <div class="source">Trop. Agricult.</div> , <div class="year">1944</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">144</div>-<div class="lpage">59</div>)</div></div></div></div><div content-id="DSR039C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lanaud</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Fouet</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Clément</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">A meta-QTL analysis of disease resistance traits of <em>Theobroma cacao</em> L.</div>, <div class="source">Mol. Breed.</div> , <div class="year">2009</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">361</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20meta-QTL%20analysis%20of%20disease%20resistance%20traits%20of%20Theobroma%20cacao%20L.&amp;author=C.%20Lanaud&amp;author=O.%20Fouet&amp;author=D.%20Cl%C3%A9ment&amp;publication_year=2009&amp;journal=Mol.%20Breed.&amp;volume=24&amp;pages=361-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-009-9297-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-009-9297-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-009-9297-4"> </span></div> </div></div></div></div></div><div content-id="DSR039C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lanaud</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Risterucci</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Ngoran</div> <div class="given-names">A.K.J.</div></div>, et al. </div><div class="article-title">A genetic-linkage map of <em>Theobroma cacao</em> L.</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1995</div>, vol. <div class="volume">91</div> (pg. <div class="fpage">987</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genetic-linkage%20map%20of%20Theobroma%20cacao%20L.&amp;author=C.%20Lanaud&amp;author=A.M.%20Risterucci&amp;author=A.K.J.%20Ngoran&amp;publication_year=1995&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=91&amp;pages=987-93" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24169987" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Crouzillat</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Lerceteau</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Petiard</div> <div class="given-names">V.</div></div>, et al. </div><div class="article-title"><em>Theobroma cacao</em> L.: a genetic linkage map and quantitative trait loci analysis</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1996</div>, vol. <div class="volume">93</div> (pg. <div class="fpage">205</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Theobroma%20cacao%20L.%3A%20a%20genetic%20linkage%20map%20and%20quantitative%20trait%20loci%20analysis&amp;author=D.%20Crouzillat&amp;author=E.%20Lerceteau&amp;author=V.%20Petiard&amp;publication_year=1996&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=93&amp;pages=205-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00225747" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00225747" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00225747"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24162219" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Risterucci</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Grivet</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">N'Goran</div> <div class="given-names">J.A.K.</div></div>,  <div class="name"><div class="surname">Pieretti</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Flament</div> <div class="given-names">M.H.</div></div>,  <div class="name"><div class="surname">Lanaud</div> <div class="given-names">C.</div></div>. </div><div class="article-title">A high-density linkage map of <em>Theobroma cacao L</em>.</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2000</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">948</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-density%20linkage%20map%20of%20Theobroma%20cacao%20L.&amp;author=A.M.%20Risterucci&amp;author=L.%20Grivet&amp;author=J.A.K.%20N'Goran&amp;author=I.%20Pieretti&amp;author=M.H.%20Flament&amp;author=C.%20Lanaud&amp;publication_year=2000&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=101&amp;pages=948-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220051566" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220051566" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220051566"> </span></div> </div></div></div></div></div><div content-id="DSR039C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Semagn</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Bjornstad</div> <div class="given-names">A.A.</div></div>,  <div class="name"><div class="surname">Ndjiondjop</div> <div class="given-names">M.N.</div></div>. </div><div class="article-title">An overview of molecular marker methods for plants</div>, <div class="source"><em>Afr. J. Biotechnol.</em></div> , <div class="year">2006</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">2540</div>-<div class="lpage">68</div>)</div></div></div></div><div content-id="DSR039C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pugh</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Fouet</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Risterucci</div> <div class="given-names">A.M.</div></div>, et al. </div><div class="article-title">A new cacao linkage map based on codominant markers: development and integration of 201 new microsatellite markers</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">108</div> (pg. <div class="fpage">1151</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20cacao%20linkage%20map%20based%20on%20codominant%20markers%3A%20development%20and%20integration%20of%20201%20new%20microsatellite%20markers&amp;author=T.%20Pugh&amp;author=O.%20Fouet&amp;author=A.M.%20Risterucci&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=108&amp;pages=1151-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1533-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1533-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1533-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14760486" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fouet</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Allegre</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Argout</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">Structural characterization and mapping of functional EST-SSR markers in <em>Theobroma cacao</em></div>, <div class="source">Tree Genet. Genomes</div> , <div class="year">2011</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">1</div>-<div class="lpage">19</div>)</div></div></div></div><div content-id="DSR039C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schmid KJ</div> <div class="given-names">R.S.T.</div></div>. </div><div class="article-title">Large-scale identification and analysis of genome-wide single-nucleotide polymorphisms for mapping in <em>Arabidopsis thaliana</em></div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">1250</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20identification%20and%20analysis%20of%20genome-wide%20single-nucleotide%20polymorphisms%20for%20mapping%20in%20Arabidopsis%20thaliana&amp;author=R.S.T.%20Schmid%20KJ&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=1250-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.728603" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.728603" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.728603"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12799357" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lijavetzky</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Cabezas</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Ibanez</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Rodriguez</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Martinez-Zapater</div> <div class="given-names">J.M.</div></div>. </div><div class="article-title">High throughput SNP discovery and genotyping in grapevine (<em>Vitis vinifera</em> L.) by combining a re-sequencing approach and SNPlex technology</div>, <div class="source">BMC Genomics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">424</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20throughput%20SNP%20discovery%20and%20genotyping%20in%20grapevine%20(Vitis%20vinifera%20L.)%20by%20combining%20a%20re-sequencing%20approach%20and%20SNPlex%20technology&amp;author=D.%20Lijavetzky&amp;author=J.A.%20Cabezas&amp;author=A.%20Ibanez&amp;author=V.%20Rodriguez&amp;author=J.M.%20Martinez-Zapater&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=424" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-424" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-424" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-424"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18021442" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Berard</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Le Paslier</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Dardevet</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">High-throughput single nucleotide polymorphism genotyping in wheat (<em>Triticum</em> spp.)</div>, <div class="source">Plant Biotechnol. J.</div> , <div class="year">2009</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">364</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20single%20nucleotide%20polymorphism%20genotyping%20in%20wheat%20(Triticum%20spp.)&amp;author=A.%20Berard&amp;author=M.C.%20Le%20Paslier&amp;author=M.%20Dardevet&amp;publication_year=2009&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=7&amp;pages=364-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.2009.7.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2009.7.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.2009.7.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19379285" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dantec</div> <div class="given-names">L.L.</div></div>,  <div class="name"><div class="surname">Chagne</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Pot</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Automated SNP detection in expressed sequence tags: statistical considerations and application to maritime pine sequences</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">461</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20SNP%20detection%20in%20expressed%20sequence%20tags%3A%20statistical%20considerations%20and%20application%20to%20maritime%20pine%20sequences&amp;author=L.L.%20Dantec&amp;author=D.%20Chagne&amp;author=D.%20Pot&amp;publication_year=2004&amp;journal=Plant%20Mol.%20Biol.&amp;volume=54&amp;pages=461-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/B:PLAN.0000036376.11710.6f" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FB:PLAN.0000036376.11710.6f" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FB:PLAN.0000036376.11710.6f"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15284499" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Zhuge</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">M.X.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">M.R.</div></div>. </div><div class="article-title">Identification and validation of single nucleotide polymorphisms in poplar using publicly expressed sequence tags</div>, <div class="source">J. Integr. Plant Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">1493</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20validation%20of%20single%20nucleotide%20polymorphisms%20in%20poplar%20using%20publicly%20expressed%20sequence%20tags&amp;author=B.%20Zhang&amp;author=Y.%20Zhou&amp;author=L.%20Zhang&amp;author=Q.%20Zhuge&amp;author=M.X.%20Wang&amp;author=M.R.%20Huang&amp;publication_year=2005&amp;journal=J.%20Integr.%20Plant%20Biol.&amp;volume=47&amp;pages=1493-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jipb.2005.47.issue-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjipb.2005.47.issue-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjipb.2005.47.issue-12"> </span></div> </div></div></div></div></div><div content-id="DSR039C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pavy</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Parsons</div> <div class="given-names">L.S.</div></div>,  <div class="name"><div class="surname">Paule</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">MacKay</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bousquet</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Automated SNP detection from a large collection of white spruce expressed sequences: contributing factors and approaches for the categorization of SNPs</div>, <div class="source">BMC Genomics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">174</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20SNP%20detection%20from%20a%20large%20collection%20of%20white%20spruce%20expressed%20sequences%3A%20contributing%20factors%20and%20approaches%20for%20the%20categorization%20of%20SNPs&amp;author=N.%20Pavy&amp;author=L.S.%20Parsons&amp;author=C.%20Paule&amp;author=J.%20MacKay&amp;author=J.%20Bousquet&amp;publication_year=2006&amp;journal=BMC%20Genomics&amp;volume=7&amp;pages=174" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-7-174" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-7-174" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-7-174"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16824208" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cho</div> <div class="given-names">R.J.</div></div>,  <div class="name"><div class="surname">Mindrinos</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Richards</div> <div class="given-names">D.R.</div></div>, et al. </div><div class="article-title">Genome-wide mapping with biallelic markers in <em>Arabidopsis thaliana</em></div>, <div class="source">Nat. Genet.</div> , <div class="year">1999</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">203</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20mapping%20with%20biallelic%20markers%20in%20Arabidopsis%20thaliana&amp;author=R.J.%20Cho&amp;author=M.%20Mindrinos&amp;author=D.R.%20Richards&amp;publication_year=1999&amp;journal=Nat.%20Genet.&amp;volume=23&amp;pages=203-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/13833" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F13833" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F13833"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10508518" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hoskins</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">Phan</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">Naeemuddin</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Single nucleotide polymorphism markers for genetic mapping in <em>Drosophila melanogaster</em></div>, <div class="source">Genome Res.</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1100</div>-<div class="lpage">13</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single%20nucleotide%20polymorphism%20markers%20for%20genetic%20mapping%20in%20Drosophila%20melanogaster&amp;author=R.A.%20Hoskins&amp;author=A.C.%20Phan&amp;author=M.%20Naeemuddin&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=1100-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.GR-1780R" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.GR-1780R" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.GR-1780R"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11381036" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tian</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Plenge</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Ransom</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Analysis and application of European genetic substructure using 300 K SNP information</div>, <div class="source">PLoS Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e4</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20and%20application%20of%20European%20genetic%20substructure%20using%20300%20K%20SNP%20information&amp;author=C.%20Tian&amp;author=R.M.%20Plenge&amp;author=M.%20Ransom&amp;publication_year=2008&amp;journal=PLoS%20Genet.&amp;volume=4&amp;pages=e4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.0040004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.0040004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.0040004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18208329" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Novembre</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Johnson</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Bryc</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Genes mirror geography within Europe</div>, <div class="source">Nature</div> , <div class="year">2008</div>, vol. <div class="volume">456</div> (pg. <div class="fpage">98</div>-<div class="lpage">101</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genes%20mirror%20geography%20within%20Europe&amp;author=J.%20Novembre&amp;author=T.%20Johnson&amp;author=K.%20Bryc&amp;publication_year=2008&amp;journal=Nature&amp;volume=456&amp;pages=98-101" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07331" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07331" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07331"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18758442" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Yoder</div> <div class="given-names">A.D.</div></div>. </div><div class="article-title">Estimation of the transition/transversion rate bias and species sampling</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1999</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">274</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimation%20of%20the%20transition%2Ftransversion%20rate%20bias%20and%20species%20sampling&amp;author=Z.%20Yang&amp;author=A.D.%20Yoder&amp;publication_year=1999&amp;journal=J.%20Mol.%20Evol.&amp;volume=48&amp;pages=274-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00006470" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00006470" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00006470"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10093216" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Swofford</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Olsen</div> <div class="given-names">G.J.</div></div>,  <div class="name"><div class="surname">Waddell</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Hillis</div> <div class="given-names">D.M.</div></div>. </div><div class="article-title">Phylogeny reconstruction</div>, <div class="source">Mol. Syst.</div> , <div class="year">1990</div>, vol. <div class="volume">411</div> pg. <div class="fpage">501</div> </div></div></div></div><div content-id="DSR039C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brown</div> <div class="given-names">W.M.</div></div>,  <div class="name"><div class="surname">Prager</div> <div class="given-names">E.M.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Wilson</div> <div class="given-names">A.C.</div></div>. </div><div class="article-title">Mitochondrial DNA sequences of primates: tempo and mode of evolution</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1982</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">225</div>-<div class="lpage">39</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mitochondrial%20DNA%20sequences%20of%20primates%3A%20tempo%20and%20mode%20of%20evolution&amp;author=W.M.%20Brown&amp;author=E.M.%20Prager&amp;author=A.%20Wang&amp;author=A.C.%20Wilson&amp;publication_year=1982&amp;journal=J.%20Mol.%20Evol.&amp;volume=18&amp;pages=225-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF01734101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01734101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF01734101"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6284948" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gojobori</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div>,  <div class="name"><div class="surname">Graur</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Patterns of nucleotide substitution in pseudogenes and functional genes</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1982</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">360</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Patterns%20of%20nucleotide%20substitution%20in%20pseudogenes%20and%20functional%20genes&amp;author=T.%20Gojobori&amp;author=W.H.%20Li&amp;author=D.%20Graur&amp;publication_year=1982&amp;journal=J.%20Mol.%20Evol.&amp;volume=18&amp;pages=360-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF01733904" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01733904" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF01733904"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7120431" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wakeley</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Substitution-rate variation among sites and the estimation of transition bias</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1994</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">436</div>-<div class="lpage">42</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Substitution-rate%20variation%20among%20sites%20and%20the%20estimation%20of%20transition%20bias&amp;author=J.%20Wakeley&amp;publication_year=1994&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=11&amp;pages=436-42" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8015437" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wakeley</div> <div class="given-names">J.</div></div>. </div><div class="article-title">The excess of transitions among nucleotide substitutions: new methods of estimating transition bias underscore its significance</div>, <div class="source">Trends Ecol. Evol. (Amst.)</div> , <div class="year">1996</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">158</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20excess%20of%20transitions%20among%20nucleotide%20substitutions%3A%20new%20methods%20of%20estimating%20transition%20bias%20underscore%20its%20significance&amp;author=J.%20Wakeley&amp;publication_year=1996&amp;journal=Trends%20Ecol.%20Evol.%20(Amst.)&amp;volume=11&amp;pages=158-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0169-5347(96)10009-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0169-5347(96)10009-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0169-5347(96)10009-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21237791" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morales</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Roig</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Monforte</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Arus</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Garcia-Mas</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Single-nucleotide polymorphisms detected in expressed sequence tags of melon (<em>Cucumis melo</em> L.)</div>, <div class="source">Genome</div> , <div class="year">2004</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">352</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single-nucleotide%20polymorphisms%20detected%20in%20expressed%20sequence%20tags%20of%20melon%20(Cucumis%20melo%20L.)&amp;author=M.%20Morales&amp;author=E.%20Roig&amp;author=A.J.%20Monforte&amp;author=P.%20Arus&amp;author=J.%20Garcia-Mas&amp;publication_year=2004&amp;journal=Genome&amp;volume=47&amp;pages=352-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g03-139" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg03-139" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg03-139"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15060588" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Deleu</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Esteras</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Roig</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">A set of EST-SNPs for map saturation and cultivar identification in melon</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">9</div> </div></div></div></div><div content-id="DSR039C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Kitashiba</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Inaba</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nishio</div> <div class="given-names">T.</div></div>. </div><div class="article-title">A <em>Brassica rapa</em> linkage map of EST-based SNP markers for identification of candidate genes controlling flowering time and leaf morphological traits</div>, <div class="source">DNA Res.</div> , <div class="year">2009</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">311</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20Brassica%20rapa%20linkage%20map%20of%20EST-based%20SNP%20markers%20for%20identification%20of%20candidate%20genes%20controlling%20flowering%20time%20and%20leaf%20morphological%20traits&amp;author=F.%20Li&amp;author=H.%20Kitashiba&amp;author=K.%20Inaba&amp;author=T.%20Nishio&amp;publication_year=2009&amp;journal=DNA%20Res.&amp;volume=16&amp;pages=311-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsp020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsp020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsp020"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19884167" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kota</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Varshney</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Prasad</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Stein</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Graner</div> <div class="given-names">A.</div></div>. </div><div class="article-title">EST-derived single nucleotide polymorphism markers for assembling genetic and physical maps of the barley genome</div>, <div class="source">Funct. Integr. Genomics</div> , <div class="year">2008</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">223</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EST-derived%20single%20nucleotide%20polymorphism%20markers%20for%20assembling%20genetic%20and%20physical%20maps%20of%20the%20barley%20genome&amp;author=R.%20Kota&amp;author=R.%20Varshney&amp;author=M.%20Prasad&amp;author=H.%20Zhang&amp;author=N.%20Stein&amp;author=A.%20Graner&amp;publication_year=2008&amp;journal=Funct.%20Integr.%20Genomics&amp;volume=8&amp;pages=223-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10142-007-0060-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10142-007-0060-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10142-007-0060-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17968603" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cordeiro</div> <div class="given-names">G.M.</div></div>,  <div class="name"><div class="surname">Eliott</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">McIntyre</div> <div class="given-names">C.L.</div></div>,  <div class="name"><div class="surname">Casu</div> <div class="given-names">R.E.</div></div>,  <div class="name"><div class="surname">Henry</div> <div class="given-names">R.J.</div></div>. </div><div class="article-title">Characterisation of single nucleotide polymorphisms in sugarcane ESTs</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">113</div> (pg. <div class="fpage">331</div>-<div class="lpage">43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterisation%20of%20single%20nucleotide%20polymorphisms%20in%20sugarcane%20ESTs&amp;author=G.M.%20Cordeiro&amp;author=F.%20Eliott&amp;author=C.L.%20McIntyre&amp;author=R.E.%20Casu&amp;author=R.J.%20Henry&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=113&amp;pages=331-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0300-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0300-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0300-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16791699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lima</div> <div class="given-names">L.S.</div></div>,  <div class="name"><div class="surname">Gramacho</div> <div class="given-names">K.P.</div></div>,  <div class="name"><div class="surname">Carels</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Single nucleotide polymorphisms from <em>Theobroma cacao</em> expressed sequence tags associated with witches’ broom disease in cacao</div>, <div class="source">Genet. Mol. Res.</div> , <div class="year">2009</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">799</div>-<div class="lpage">808</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single%20nucleotide%20polymorphisms%20from%20Theobroma%20cacao%20expressed%20sequence%20tags%20associated%20with%20witches%E2%80%99%20broom%20disease%20in%20cacao&amp;author=L.S.%20Lima&amp;author=K.P.%20Gramacho&amp;author=N.%20Carels&amp;publication_year=2009&amp;journal=Genet.%20Mol.%20Res.&amp;volume=8&amp;pages=799-808" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4238/vol8-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4238%2Fvol8-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4238%2Fvol8-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19681032" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Argout</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Fouet</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Wincker</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Towards the understanding of the cocoa transcriptome: production and analysis of an exhaustive dataset of ESTs of <em>Theobroma cacao</em> L. generated from various tissues and under various conditions</div>, <div class="source">BMC Genomics</div> , <div class="year">2008</div></div></div></div></div><div content-id="DSR039C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Vosman</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Voorrips</div> <div class="given-names">R.E.</div></div>,  <div class="name"><div class="surname">van der Linden</div> <div class="given-names">C.G.</div></div>,  <div class="name"><div class="surname">Leunissen</div> <div class="given-names">J.A.</div></div>. </div><div class="article-title">QualitySNP: a pipeline for detecting single nucleotide polymorphisms and insertions/deletions in EST data from diploid and polyploid species</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">438</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QualitySNP%3A%20a%20pipeline%20for%20detecting%20single%20nucleotide%20polymorphisms%20and%20insertions%2Fdeletions%20in%20EST%20data%20from%20diploid%20and%20polyploid%20species&amp;author=J.%20Tang&amp;author=B.%20Vosman&amp;author=R.E.%20Voorrips&amp;author=C.G.%20van%20der%20Linden&amp;author=J.A.%20Leunissen&amp;publication_year=2006&amp;journal=BMC%20Bioinformatics&amp;volume=7&amp;pages=438" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-7-438" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-7-438" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-7-438"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17029635" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hyten</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Choi</div> <div class="given-names">I.-Y.</div></div>, et al. </div><div class="article-title">High-throughput genotyping with the GoldenGate assay in the complex genome of soybean</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">116</div> (pg. <div class="fpage">945</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20genotyping%20with%20the%20GoldenGate%20assay%20in%20the%20complex%20genome%20of%20soybean&amp;author=D.L.%20Hyten&amp;author=Q.%20Song&amp;author=I.-Y.%20Choi&amp;publication_year=2008&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=116&amp;pages=945-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-008-0726-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-008-0726-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-008-0726-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18278477" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Akhunov</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Nicolet</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Dvorak</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Single nucleotide polymorphism genotyping in polyploid wheat with the Illumina GoldenGate assay</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">119</div> (pg. <div class="fpage">507</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single%20nucleotide%20polymorphism%20genotyping%20in%20polyploid%20wheat%20with%20the%20Illumina%20GoldenGate%20assay&amp;author=E.%20Akhunov&amp;author=C.%20Nicolet&amp;author=J.%20Dvorak&amp;publication_year=2009&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=119&amp;pages=507-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-009-1059-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-009-1059-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-009-1059-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19449174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rozen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Skaletsky</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Primer3 on the WWW for general users and for biologist programmers</div>, <div class="source">Methods Mol. Biol.</div> , <div class="year">2000</div>, vol. <div class="volume">132</div> (pg. <div class="fpage">365</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Primer3%20on%20the%20WWW%20for%20general%20users%20and%20for%20biologist%20programmers&amp;author=S.%20Rozen&amp;author=H.%20Skaletsky&amp;publication_year=2000&amp;journal=Methods%20Mol.%20Biol.&amp;volume=132&amp;pages=365-86" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10547847" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Van Ooijen</div> <div class="given-names">J.W.</div></div>. </div>, <div class="source">Joinmap, Software for the Calculation of Genetic Linkage Maps</div> , <div class="year">2006</div><div class="publisher-loc">The Netherlands</div><div class="publisher-name">Wageningen</div></div></div></div></div><div content-id="DSR039C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kosambi</div> <div class="given-names">D.D.</div></div>. </div><div class="article-title">The estimation of map distance from recombination values</div>, <div class="source">Ann. Eugen.</div> , <div class="year">1944</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">172</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20estimation%20of%20map%20distance%20from%20recombination%20values&amp;author=D.D.%20Kosambi&amp;publication_year=1944&amp;journal=Ann.%20Eugen.&amp;volume=12&amp;pages=172-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-1809.1943.tb02321.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-1809.1943.tb02321.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-1809.1943.tb02321.x"> </span></div> </div></div></div></div></div><div content-id="DSR039C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Batley</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Barker</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">O'Sullivan</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Edwards</div> <div class="given-names">K.J.</div></div>,  <div class="name"><div class="surname">Edwards</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Mining for single nucleotide polymorphisms and insertions/deletions in maize expressed sequence tag data</div>, <div class="source">Plant Physiol.</div> , <div class="year">2003</div>, vol. <div class="volume">132</div> (pg. <div class="fpage">84</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mining%20for%20single%20nucleotide%20polymorphisms%20and%20insertions%2Fdeletions%20in%20maize%20expressed%20sequence%20tag%20data&amp;author=J.%20Batley&amp;author=G.%20Barker&amp;author=H.%20O'Sullivan&amp;author=K.J.%20Edwards&amp;author=D.%20Edwards&amp;publication_year=2003&amp;journal=Plant%20Physiol.&amp;volume=132&amp;pages=84-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.102.019422" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.102.019422" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.102.019422"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12746514" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Mischke</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Johnson</div> <div class="given-names">E.S.</div></div>,  <div class="name"><div class="surname">Phillips-Mora</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Meinhardt</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Molecular characterization of an international cacao collection using microsatellite markers</div>, <div class="source">Tree Genet. Genomes</div> , <div class="year">2008</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">1</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20characterization%20of%20an%20international%20cacao%20collection%20using%20microsatellite%20markers&amp;author=D.%20Zhang&amp;author=S.%20Mischke&amp;author=E.S.%20Johnson&amp;author=W.%20Phillips-Mora&amp;author=L.%20Meinhardt&amp;publication_year=2008&amp;journal=Tree%20Genet.%20Genomes&amp;volume=5&amp;pages=1-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11295-008-0163-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11295-008-0163-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11295-008-0163-z"> </span></div> </div></div></div></div></div><div content-id="DSR039C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lachenaud</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Genetic diversity and population structure in wild stands of cacao trees (<em>Theobroma cacao</em> L.), in French Guiana</div>, <div class="source">Ann. For. Sci.</div> , <div class="year">2008</div>, vol. <div class="volume">65</div> pg. <div class="fpage">7</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20and%20population%20structure%20in%20wild%20stands%20of%20cacao%20trees%20(Theobroma%20cacao%20L.)%2C%20in%20French%20Guiana&amp;author=P.%20Lachenaud&amp;author=D.%20Zhang&amp;publication_year=2008&amp;journal=Ann.%20For.%20Sci.&amp;volume=65&amp;pages=7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1051/forest:2008011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1051%2Fforest:2008011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1051%2Fforest:2008011"> </span></div> </div></div></div></div></div><div content-id="DSR039C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Arevalo-Gardini</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Mischke</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Zúñiga-Cernades</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Barreto-Chavez</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Del Aguila</div> <div class="given-names">J.A.</div></div>. </div><div class="article-title">Genetic diversity and structure of managed and semi-natural populations of cocoa (<em>Theobroma cacao</em>) in the Huallaga and Ucayali Valleys of Peru</div>, <div class="source">Ann. Bot.</div> , <div class="year">2006</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">647</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20and%20structure%20of%20managed%20and%20semi-natural%20populations%20of%20cocoa%20(Theobroma%20cacao)%20in%20the%20Huallaga%20and%20Ucayali%20Valleys%20of%20Peru&amp;author=D.%20Zhang&amp;author=E.%20Arevalo-Gardini&amp;author=S.%20Mischke&amp;author=L.%20Z%C3%BA%C3%B1iga-Cernades&amp;author=A.%20Barreto-Chavez&amp;author=J.A.%20Del%20Aguila&amp;publication_year=2006&amp;journal=Ann.%20Bot.&amp;volume=98&amp;pages=647-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/aob/mcl146" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Faob%2Fmcl146" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Faob%2Fmcl146"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16845139" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Motamayor</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Lachenaud</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">da Silva E Mota</div> <div class="given-names">J.W.</div></div>, et al. </div><div class="article-title">Geographic and genetic population differentiation of the Amazonian chocolate tree (<em>Theobroma cacao</em> L.)</div>, <div class="source">PLoS ONE</div> , <div class="year">2008</div>, vol. <div class="volume">3</div> pg. <div class="fpage">e3311</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Geographic%20and%20genetic%20population%20differentiation%20of%20the%20Amazonian%20chocolate%20tree%20(Theobroma%20cacao%20L.)&amp;author=J.C.%20Motamayor&amp;author=P.%20Lachenaud&amp;author=J.W.%20da%20Silva%20E%20Mota&amp;publication_year=2008&amp;journal=PLoS%20ONE&amp;volume=3&amp;pages=e3311" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0003311" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0003311" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0003311"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18827930" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Loor Solorzano</div> <div class="given-names">R.G.</div></div>. </div><div class="article-title">Contribution à l’étude de la domestication de la variété Nacional d'Equateur: recherche de la variété native et de ses ancêtres sauvages</div>, <div class="year">2007</div> <div class="comment">PhD Thesis, Montpellier SUPAGRO- ED SIBAGHE- spécialité: biologie intégrative des plantes</div></div></div></div></div><div content-id="DSR039C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Knight</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Rogers</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Incompatibility in <em>Theobroma cacao</em></div>, <div class="source">Heredity</div> , <div class="year">1955</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">69</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Incompatibility%20in%20Theobroma%20cacao&amp;author=R.%20Knight&amp;author=H.%20Rogers&amp;publication_year=1955&amp;journal=Heredity&amp;volume=9&amp;pages=69-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/hdy.1955.3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fhdy.1955.3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fhdy.1955.3"> </span></div> </div></div></div></div></div><div content-id="DSR039C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cope</div> <div class="given-names">F.W.</div></div>. </div><div class="article-title">Incompatibility in <em>Theobroma cacao</em></div>, <div class="source">Nature</div> , <div class="year">1958</div>, vol. <div class="volume">181</div> pg. <div class="fpage">279</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Incompatibility%20in%20Theobroma%20cacao&amp;author=F.W.%20Cope&amp;publication_year=1958&amp;journal=Nature&amp;volume=181&amp;pages=279" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/181279a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F181279a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F181279a0"> </span></div> </div></div></div></div></div><div content-id="DSR039C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fauré</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Noyer</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Horry</div> <div class="given-names">J.P.</div></div>,  <div class="name"><div class="surname">Bakry</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Lanaud</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Gońzalez de León</div> <div class="given-names">D.</div></div>. </div><div class="article-title">A molecular marker-based linkage map of diploid bananas (<em>Musa acuminata</em>)</div>, <div class="source">Theor. Appl. Genetics</div> , <div class="year">1993</div>, vol. <div class="volume">87</div> (pg. <div class="fpage">517</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20molecular%20marker-based%20linkage%20map%20of%20diploid%20bananas%20(Musa%20acuminata)&amp;author=S.%20Faur%C3%A9&amp;author=J.L.%20Noyer&amp;author=J.P.%20Horry&amp;author=F.%20Bakry&amp;author=C.%20Lanaud&amp;author=D.%20Go%C5%84zalez%20de%20Le%C3%B3n&amp;publication_year=1993&amp;journal=Theor.%20Appl.%20Genetics&amp;volume=87&amp;pages=517-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00215098" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00215098" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00215098"> </span></div> </div></div></div></div></div><div content-id="DSR039C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Blott</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">J.-J.</div></div>,  <div class="name"><div class="surname">Moisio</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Molecular dissection of a quantitative trait locus: a phenylalanine-to-tyrosine substitution in the transmembrane domain of the bovine growth hormone receptor is associated with a major effect on milk yield and composition</div>, <div class="source">Genetics</div> , <div class="year">2003</div>, vol. <div class="volume">163</div> (pg. <div class="fpage">253</div>-<div class="lpage">66</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20dissection%20of%20a%20quantitative%20trait%20locus%3A%20a%20phenylalanine-to-tyrosine%20substitution%20in%20the%20transmembrane%20domain%20of%20the%20bovine%20growth%20hormone%20receptor%20is%20associated%20with%20a%20major%20effect%20on%20milk%20yield%20and%20composition&amp;author=S.%20Blott&amp;author=J.-J.%20Kim&amp;author=S.%20Moisio&amp;publication_year=2003&amp;journal=Genetics&amp;volume=163&amp;pages=253-66" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12586713" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cardon</div> <div class="given-names">L.R.</div></div>,  <div class="name"><div class="surname">Bell</div> <div class="given-names">J.I.</div></div>. </div><div class="article-title">Association study designs for complex diseases</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2001</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">91</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Association%20study%20designs%20for%20complex%20diseases&amp;author=L.R.%20Cardon&amp;author=J.I.%20Bell&amp;publication_year=2001&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=2&amp;pages=91-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35052543" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35052543" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35052543"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11253062" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Vikis</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Candidate lung tumor susceptibility genes identified through whole-genome association analyses in inbred mice</div>, <div class="source">Nat. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">888</div>-<div class="lpage">95</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Candidate%20lung%20tumor%20susceptibility%20genes%20identified%20through%20whole-genome%20association%20analyses%20in%20inbred%20mice&amp;author=P.%20Liu&amp;author=Y.%20Wang&amp;author=H.%20Vikis&amp;publication_year=2006&amp;journal=Nat.%20Genet.&amp;volume=38&amp;pages=888-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1849" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1849" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1849"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16862160" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Meuwissen</div> <div class="given-names">T.H.E.</div></div>,  <div class="name"><div class="surname">Karlsen</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lien</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Olsaker</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Goddard</div> <div class="given-names">M.E.</div></div>. </div><div class="article-title">Fine mapping of a quantitative trait locus for twinning rate using combined linkage and linkage disequilibrium mapping</div>, <div class="source">Genetics</div> , <div class="year">2002</div>, vol. <div class="volume">161</div> (pg. <div class="fpage">373</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fine%20mapping%20of%20a%20quantitative%20trait%20locus%20for%20twinning%20rate%20using%20combined%20linkage%20and%20linkage%20disequilibrium%20mapping&amp;author=T.H.E.%20Meuwissen&amp;author=A.%20Karlsen&amp;author=S.%20Lien&amp;author=I.%20Olsaker&amp;author=M.E.%20Goddard&amp;publication_year=2002&amp;journal=Genetics&amp;volume=161&amp;pages=373-9" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12019251" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.-M.</div></div>,  <div class="name"><div class="surname">Mao</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Xie</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Luo</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Mapping quantitative trait loci using naturally occurring genetic variance among commercial inbred lines of maize (<em>Zea mays</em> L.)</div>, <div class="source">Genetics</div> , <div class="year">2005</div>, vol. <div class="volume">169</div> (pg. <div class="fpage">2267</div>-<div class="lpage">75</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20quantitative%20trait%20loci%20using%20naturally%20occurring%20genetic%20variance%20among%20commercial%20inbred%20lines%20of%20maize%20(Zea%20mays%20L.)&amp;author=Y.-M.%20Zhang&amp;author=Y.%20Mao&amp;author=C.%20Xie&amp;author=H.%20Smith&amp;author=L.%20Luo&amp;author=S.%20Xu&amp;publication_year=2005&amp;journal=Genetics&amp;volume=169&amp;pages=2267-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.104.033217" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.104.033217" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.104.033217"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15716509" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aranzana</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Genome-wide association mapping in Arabidopsis identifies previously known flowering time and pathogen resistance genes</div>, <div class="source">PLoS Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">1</div> pg. <div class="fpage">e60</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20mapping%20in%20Arabidopsis%20identifies%20previously%20known%20flowering%20time%20and%20pathogen%20resistance%20genes&amp;author=M.J.%20Aranzana&amp;author=S.%20Kim&amp;author=K.%20Zhao&amp;publication_year=2005&amp;journal=PLoS%20Genet.&amp;volume=1&amp;pages=e60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.0010060" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.0010060" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.0010060"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16292355" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jannoo</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Grivet</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Dookun</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">D'hont</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Glaszmann</div> <div class="given-names">J.C.</div></div>. </div><div class="article-title">Linkage disequilibrium among modern sugarcane cultivars</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1999</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">1053</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Linkage%20disequilibrium%20among%20modern%20sugarcane%20cultivars&amp;author=N.%20Jannoo&amp;author=L.%20Grivet&amp;author=A.%20Dookun&amp;author=A.%20D'hont&amp;author=J.C.%20Glaszmann&amp;publication_year=1999&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=99&amp;pages=1053-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220051414" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220051414" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220051414"> </span></div> </div></div></div></div></div><div content-id="DSR039C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wei</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Jackson</div> <div class="given-names">P.A.</div></div>,  <div class="name"><div class="surname">McIntyre</div> <div class="given-names">C.L.</div></div>,  <div class="name"><div class="surname">Aitken</div> <div class="given-names">K.S.</div></div>,  <div class="name"><div class="surname">Croft</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Associations between DNA markers and resistance to diseases in sugarcane and effects of population substructure</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">114</div> (pg. <div class="fpage">155</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Associations%20between%20DNA%20markers%20and%20resistance%20to%20diseases%20in%20sugarcane%20and%20effects%20of%20population%20substructure&amp;author=X.%20Wei&amp;author=P.A.%20Jackson&amp;author=C.L.%20McIntyre&amp;author=K.S.%20Aitken&amp;author=B.%20Croft&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=114&amp;pages=155-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0418-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0418-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0418-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17047910" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Agrama</div> <div class="given-names">H.A.</div></div>,  <div class="name"><div class="surname">Eizenga</div> <div class="given-names">G.C.</div></div>,  <div class="name"><div class="surname">Yan</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Association mapping of yield and its components in rice cultivars</div>, <div class="source">Mol. Breed.</div> , <div class="year">2007</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">341</div>-<div class="lpage">56</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Association%20mapping%20of%20yield%20and%20its%20components%20in%20rice%20cultivars&amp;author=H.A.%20Agrama&amp;author=G.C.%20Eizenga&amp;author=W.%20Yan&amp;publication_year=2007&amp;journal=Mol.%20Breed.&amp;volume=19&amp;pages=341-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-006-9066-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-006-9066-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-006-9066-6"> </span></div> </div></div></div></div></div><div content-id="DSR039C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marcano</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Pugh</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Cros</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Adding value to cocoa (<em>Theobroma cacao</em> L.) germplasm information with domestication history and admixture mapping</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">114</div> (pg. <div class="fpage">877</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adding%20value%20to%20cocoa%20(Theobroma%20cacao%20L.)%20germplasm%20information%20with%20domestication%20history%20and%20admixture%20mapping&amp;author=M.%20Marcano&amp;author=T.%20Pugh&amp;author=E.%20Cros&amp;publication_year=2007&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=114&amp;pages=877-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0486-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0486-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0486-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17252253" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marcano</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Morales</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hoyer</div> <div class="given-names">M.T.</div></div>, et al. </div><div class="article-title">A genomewide admixture mapping study for yield factors and morphological traits in a cultivated cocoa (<em>Theobroma cacao</em> L.) population</div>, <div class="source">Tree Genet. Genomes</div> , <div class="year">2009</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">329</div>-<div class="lpage">37</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genomewide%20admixture%20mapping%20study%20for%20yield%20factors%20and%20morphological%20traits%20in%20a%20cultivated%20cocoa%20(Theobroma%20cacao%20L.)%20population&amp;author=M.%20Marcano&amp;author=S.%20Morales&amp;author=M.T.%20Hoyer&amp;publication_year=2009&amp;journal=Tree%20Genet.%20Genomes&amp;volume=5&amp;pages=329-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11295-008-0185-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11295-008-0185-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11295-008-0185-6"> </span></div> </div></div></div></div></div><div content-id="DSR039C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iwaro</div> <div class="given-names">A.D.</div></div>,  <div class="name"><div class="surname">Bekele</div> <div class="given-names">F.L.</div></div>,  <div class="name"><div class="surname">Butler</div> <div class="given-names">D.R.</div></div>. </div><div class="article-title">Evaluation and utilisation of cacao (<em>Theobroma cacao</em> L.) germplasm at the International Cocoa Genebank, Trinidad</div>, <div class="source">Euphytica</div> , <div class="year">2003</div>, vol. <div class="volume">130</div> (pg. <div class="fpage">207</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evaluation%20and%20utilisation%20of%20cacao%20(Theobroma%20cacao%20L.)%20germplasm%20at%20the%20International%20Cocoa%20Genebank%2C%20Trinidad&amp;author=A.D.%20Iwaro&amp;author=F.L.%20Bekele&amp;author=D.R.%20Butler&amp;publication_year=2003&amp;journal=Euphytica&amp;volume=130&amp;pages=207-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1022855131534" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1022855131534" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1022855131534"> </span></div> </div></div></div></div></div><div content-id="DSR039C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="collab">IBPGR Working Group on Genetics Resources of Cocoa</div>, <div class="source">Genetic Resources of Cocoa</div> , <div class="year">1981</div><div class="publisher-loc">Rome, Italy</div>pg. <div class="fpage">25</div> </div></div></div></div><div content-id="DSR039C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C61" class="ref-content "><div class="label -label">61</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schnell</div> <div class="given-names">R.J.</div></div>,  <div class="name"><div class="surname">Kuhn</div> <div class="given-names">D.N.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">J.S.</div></div>, et al. </div><div class="article-title">Development of a marker assisted selection program for cacao</div>, <div class="source">Phytopathology</div> , <div class="year">2007</div>, vol. <div class="volume">97</div> (pg. <div class="fpage">1664</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20marker%20assisted%20selection%20program%20for%20cacao&amp;author=R.J.%20Schnell&amp;author=D.N.%20Kuhn&amp;author=J.S.%20Brown&amp;publication_year=2007&amp;journal=Phytopathology&amp;volume=97&amp;pages=1664-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/PHYTO-97-12-1664" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FPHYTO-97-12-1664" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FPHYTO-97-12-1664"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18943731" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C62" class="ref-content "><div class="label -label">62</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Meuwissen</div> <div class="given-names">T.H.</div></div>,  <div class="name"><div class="surname">Hayes</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Goddard</div> <div class="given-names">M.E.</div></div>. </div><div class="article-title">Prediction of total genetic value using genome-wide dense marker maps</div>, <div class="source">Genetics</div> , <div class="year">2001</div>, vol. <div class="volume">157</div> (pg. <div class="fpage">1819</div>-<div class="lpage">29</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20total%20genetic%20value%20using%20genome-wide%20dense%20marker%20maps&amp;author=T.H.%20Meuwissen&amp;author=B.J.%20Hayes&amp;author=M.E.%20Goddard&amp;publication_year=2001&amp;journal=Genetics&amp;volume=157&amp;pages=1819-29" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11290733" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C63" class="ref-content "><div class="label -label">63</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Heffner</div> <div class="given-names">E.L.</div></div>,  <div class="name"><div class="surname">Sorrells</div> <div class="given-names">M.E.</div></div>,  <div class="name"><div class="surname">Jannink</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">Genomic selection for crop improvement</div>, <div class="source">Crop Sci.</div> , <div class="year">2009</div>, vol. <div class="volume">49</div> (pg. <div class="fpage">1</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20selection%20for%20crop%20improvement&amp;author=E.L.%20Heffner&amp;author=M.E.%20Sorrells&amp;author=J.L.%20Jannink&amp;publication_year=2009&amp;journal=Crop%20Sci.&amp;volume=49&amp;pages=1-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2008.08.0512" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2008.08.0512" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2008.08.0512"> </span></div> </div></div></div></div></div><div content-id="DSR039C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C64" class="ref-content "><div class="label -label">64</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bernardo</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Prospects for genomewide selection for quantitative traits in maize</div>, <div class="source">Crop Sci.</div> , <div class="year">2007</div>, vol. <div class="volume">47</div> pg. <div class="fpage">1082</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prospects%20for%20genomewide%20selection%20for%20quantitative%20traits%20in%20maize&amp;author=R.%20Bernardo&amp;author=J.%20Yu&amp;publication_year=2007&amp;journal=Crop%20Sci.&amp;volume=47&amp;pages=1082" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2006.11.0690" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2006.11.0690" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2006.11.0690"> </span></div> </div></div></div></div></div><div content-id="DSR039C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C65" class="ref-content "><div class="label -label">65</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wong</div> <div class="given-names">C.K.</div></div>,  <div class="name"><div class="surname">Bernardo</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Genomewide selection in oil palm: increasing selection gain per unit time and cost with small populations</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">116</div> (pg. <div class="fpage">815</div>-<div class="lpage">24</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomewide%20selection%20in%20oil%20palm%3A%20increasing%20selection%20gain%20per%20unit%20time%20and%20cost%20with%20small%20populations&amp;author=C.K.%20Wong&amp;author=R.%20Bernardo&amp;publication_year=2008&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=116&amp;pages=815-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-008-0715-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-008-0715-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-008-0715-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18219476" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C66" class="ref-content "><div class="label -label">66</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jannink</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Lorenz</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Iwata</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Genomic selection in plant breeding: from theory to practice</div>, <div class="source">Brief. Funct. Genomic.</div> , <div class="year">2010</div>, vol. <div class="volume">9</div> pg. <div class="fpage">166</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20selection%20in%20plant%20breeding%3A%20from%20theory%20to%20practice&amp;author=J.L.%20Jannink&amp;author=A.J.%20Lorenz&amp;author=H.%20Iwata&amp;publication_year=2010&amp;journal=Brief.%20Funct.%20Genomic.&amp;volume=9&amp;pages=166" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bfgp/elq001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbfgp%2Felq001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbfgp%2Felq001"> </span></div> </div></div></div></div></div><div content-id="DSR039C67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C67" class="ref-content "><div class="label -label">67</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Clement</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Eskes</div> <div class="given-names">A.B.</div></div>,  <div class="name"><div class="surname">Sounigo</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">N'Goran</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Amélioration génétique du cacaoyer en Côte d'Ivoire-Présentation d'un nouveau schéma de selection</div>, <div class="year">1993</div><div class="conf-name">Proceedings of the 11th International Cocoa Research Conference</div>(pg. <div class="fpage">18</div>-<div class="lpage">24</div>)</div></div></div></div><div content-id="DSR039C68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C68" class="ref-content "><div class="label -label">68</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Paulin</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Eskes</div> <div class="given-names">A.B.</div></div>. </div><div class="article-title">Le cacaoyer: stratégies de sélection</div>, <div class="source">Plant. Recherche Dével.</div> , <div class="year">1995</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">5</div>-<div class="lpage">18</div>)</div></div></div></div><div content-id="DSR039C69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C69" class="ref-content "><div class="label -label">69</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pires</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Monteiro</div> <div class="given-names">W.R.</div></div>,  <div class="name"><div class="surname">Pinto</div> <div class="given-names">L.R.M.</div></div>,  <div class="name"><div class="surname">Figueira</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Yamada</div> <div class="given-names">M.M.</div></div>,  <div class="name"><div class="surname">Ahnert</div> <div class="given-names">D.</div></div>. </div><div class="article-title">A proposal for cocoa breeding</div>, <div class="year">1996</div><div class="conf-name">Proceedings of the 12th International Cocoa Research Conference</div>(pg. <div class="fpage">17</div>-<div class="lpage">23</div>)</div></div></div></div><div content-id="DSR039C70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C70" class="ref-content "><div class="label -label">70</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iwaro</div> <div class="given-names">A.D.</div></div>,  <div class="name"><div class="surname">Singh</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Bharath</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Perez</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Ali</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Butler</div> <div class="given-names">D.R.</div></div>. </div><div class="article-title">Germplasm enhancement for resistance to black pod disease – final stages and achievements</div>, <div class="year">2010</div><div class="conf-name">Proceedings of the 16th International Cocoa Research Conference</div>(pg. <div class="fpage">493</div>-<div class="lpage">499</div>)</div></div></div></div><div content-id="DSR039C71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C71" class="ref-content "><div class="label -label">71</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tahi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Lachenaud</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">N'Goran</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Second cycle de selection recurrente du cacaoyer (<em>Theobroma cacao</em> L.) en Cote d'Ivoire: bilan a mi-parcours et propositions de sorties varietales</div>, <div class="year">2010</div><div class="conf-name">Proceedings of the 16th International Cocoa Research Conference</div>(pg. <div class="fpage">3</div>-<div class="lpage">12</div>)</div></div></div></div><div content-id="DSR039C72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C72" class="ref-content "><div class="label -label">72</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lopes</div> <div class="given-names">U.V.</div></div>,  <div class="name"><div class="surname">Monteiro</div> <div class="given-names">W.R.</div></div>,  <div class="name"><div class="surname">Pires</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Clement</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Yamada</div> <div class="given-names">M.M.</div></div>,  <div class="name"><div class="surname">Gramacho</div> <div class="given-names">K.P.</div></div>. </div><div class="article-title">Cacao breeding in Bahia, Brazil—strategies and results</div>, <div class="source">Crop Breed. Appl. Biotechnol.</div> , <div class="year">2011</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">73</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cacao%20breeding%20in%20Bahia%2C%20Brazil%E2%80%94strategies%20and%20results&amp;author=U.V.%20Lopes&amp;author=W.R.%20Monteiro&amp;author=J.L.%20Pires&amp;author=D.%20Clement&amp;author=M.M.%20Yamada&amp;author=K.P.%20Gramacho&amp;publication_year=2011&amp;journal=Crop%20Breed.%20Appl.%20Biotechnol.&amp;volume=1&amp;pages=73-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1590/S1984-70332011000500011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1590%2FS1984-70332011000500011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1590%2FS1984-70332011000500011"> </span></div> </div></div></div></div></div><div content-id="DSR039C73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C73" class="ref-content "><div class="label -label">73</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Micheli</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Guiltinan</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Gramacho</div> <div class="given-names">K.P.</div></div>, et al. </div><div class="person-group"><div class="name"><div class="surname">Jean-Claude</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Michel</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Functional genomics of Cacao</div>, <div class="source">Advances in Botanical Research</div> , <div class="year">2010</div>, vol. <div class="volume">vol. 55</div> <div class="publisher-loc">Londres</div><div class="publisher-name">Academic Press</div>(pg. <div class="fpage">119</div>-<div class="lpage">77</div>)</div></div></div></div><div content-id="DSR039C74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C74" class="ref-content "><div class="label -label">74</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Doebley</div> <div class="given-names">J.F.</div></div>,  <div class="name"><div class="surname">Gaut</div> <div class="given-names">B.S.</div></div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">B.D.</div></div>. </div><div class="article-title">The molecular genetics of crop domestication</div>, <div class="source">Cell</div> , <div class="year">2006</div>, vol. <div class="volume">127</div> (pg. <div class="fpage">1309</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20molecular%20genetics%20of%20crop%20domestication&amp;author=J.F.%20Doebley&amp;author=B.S.%20Gaut&amp;author=B.D.%20Smith&amp;publication_year=2006&amp;journal=Cell&amp;volume=127&amp;pages=1309-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2006.12.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2006.12.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2006.12.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17190597" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR039C75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR039C75" class="ref-content "><div class="label -label">75</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nielsen</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Molecular signatures of natural selection</div>, <div class="source">Annu. Rev. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">197</div>-<div class="lpage">218</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20signatures%20of%20natural%20selection&amp;author=R.%20Nielsen&amp;publication_year=2005&amp;journal=Annu.%20Rev.%20Genet.&amp;volume=39&amp;pages=197-218" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.39.073003.112420" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.39.073003.112420" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.39.073003.112420"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16285858" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2011. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=484949&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr039/2/dsr039-dsr039supp_table5.xls?Expires=2147483647&amp;Signature=CGxQ~qeN6migrR8hTTPle0Tt1zPFnmBgMbZe2pHxZvJ7Mbposc~qDkAQK3M6KvpvHbWWuWQbDKaaCBnJb-TsTWOwL0-6~5z~Bh4rnLyjtld3Q-bIjGjEGd5LHTocyYkByswPD847MRBm6UVLj6oQ9gq-zayhC0~1YIDwIKCWvCgcix17dOQCCKaHC9v~pg65dDTYFPi5CcvJvq3fy2B-vlo~BPPzbQSHpLoJwYxBisGSsaywzQu~iphNG8hozpnT552Gu7gbDp7rSxsXJTCv525nTyaTmemTsuAu9GZjrMu6OWmIaohdCTfKJe80Fob94n4fz39SScjzL4TRF~Q4Cw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr039/2/dsr039-dsr039supp_table4.xls?Expires=2147483647&amp;Signature=0wo3-~butPVDnB-kiMNz6waAam~r5TFtdJvMOIAe0-BYfOZWiD~wCvzt1aYUbXt3XvKL6o6uiAHjk7xfjKhbQ1i5tegYRRracEQwatBlA5A6DrEl0cL-rl7t2qrHvPVSccPUZ2kY6To1oArPLtp1IIbDkfmc1K-xbwXgn7kHjZ1VzTw0BSz2JDQxDPtdcw4-I1HE9K8QFBX5uagHKsOWq9BCWRQeEUr0nrc0lH5hngPUxJ2JqmBO6VLA7wJuiHSyzhH3XEsnIx1-Vzrwf0~~240aIJ~dSXQIFftR5iAbMKWVHFPdMhxhXzXTutURM7oegxmsKrclJ870ZRPIPKgG5w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr039/2/dsr039-dsr039supp_table3.xls?Expires=2147483647&amp;Signature=TRcPkBZRAX0xaiMYV84IOP4XMU5X1yoD-YhuyRguYGsf~~TBmRR~h7yM~MR-MZED2JuyO6u96T0PKhob5wRo-ZZ1wcgnu5N9T1fU2MhJXfKLHr6X53T2Q9yiFPVKpryjvnHGjaepsV2a4nSDNh8QpdoJZk9CqQhPQTRKw2vhAyeyzbjjmecPeInBxAr2c3L8~MQ-LD-0oH4cOcXd1dJ1NCYf4xa1JO2BqZjlTjIF-6Wk0pMwNFGxF30kWxx3Sb51pNNeerVDE4oorWdkfHq8QBsWLaOEjlGK~oMmW9uNMQH~vclEXo3vDY-REy1gNCn9UpBhs1t5q1JPXUYcqSW2Qg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr039/2/dsr039-dsr039supp_table2.xls?Expires=2147483647&amp;Signature=AxAD6-bc9jH3U129X1e-xBDLZaa2yRne1wPMQpoQkKHKixqZRKVHhkzYdZ3wPtoMTjBmx0f2dAJ-oYrsRZlAa8AJTnlCx4FFG8IK3dTThNLHKdxyOKyzSCP3dFqLYZ2nUdBrxKKEJO6SOhk1-ye~IbyFxdUwBgaeMojvABZ~QLaRyuJjenxigzZK8cKl~0uKHZI3qC3s84NpK5PK6DTRpuT7f~lFJ-oZkM4avOzCLNhCQe1TjdgxBfNuiX2fLPMKoQwJeQvlmyfT92cIX8Vl7tRQENbIZoWt535vwgJMR1uBppXM02709CkUeBVRbyABwY6yqHytE5KXGz6j8l8AFQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/1/10.1093/dnares/dsr039/2/dsr039-dsr039supp_table1.xls?Expires=2147483647&amp;Signature=jYB~Uv4yDTgpTCofgoJ-l5QlD2xhclzscFyJ3zG0rpp4OsKPbre2OZN1vmJ8Mm5MVn~jGNafObMEULjQOtxB7awePWwTyMXKnuDTNLEsTvth74LIkfb1I1lYuFtrEJdXFrP6hXg029gSd9VvXFAJHCrll~qPgWscM8d-7vu4BCx99XDrDlRwwNuUq3Q5JXctVA5UB1rchjw8IFjJzNPEqb~zyUO-z69cFRE-KzonhJ3rcNCxaRPXM9u--FN5DKsAhq9ffWUJvsWsuYenp2UvpFMO8IWPqfG-IhlI3lolNiZdlfDG-ZKNA3bR5V5NsFX1rVPl~2E8arPvdWso-5-8Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">389</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000300760000003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">25</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsr039"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="484949"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000300760000003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f19%2f1%2f23%2f484949">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000300760000003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (25)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F19%2F1%2F23%2F484949" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/484949'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Discovery and mapping of a new expressed sequence tag-single nucleotide polymorphism and simple sequence repeat panel for large-scale genetic studies and breeding of Theobroma cacao L." />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsr039");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f19%2f1%2f23%2f484949">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

