

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":1,"applicationTime":1803,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/1/11/4106313","@type":"ScholarlyArticle","name":"Comparative transcriptomics of cyprinid minnows and carp in a common wild setting: a resource for ecological genomics in freshwater communities","datePublished":"2017-09-06","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2018-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx034","keywords":["RNA-seq","essential genes","<italic>Cyprinus carpio</italic>","carp","gene silencing"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/1/11/4106313","author":[{"name":"Krabbenhoft, Trevor J","affiliation":"Department of Biology and Museum of Southwestern Biology, University of New Mexico, Albuquerque, NM 87131, USA","@type":"Person"},{"name":"Turner, Thomas F","affiliation":"Department of Biology and Museum of Southwestern Biology, University of New Mexico, Albuquerque, NM 87131, USA","@type":"Person"}],"description":"Abstract.  Comparative transcriptomics can now be conducted on organisms in natural settings, which has greatly enhanced understanding of genome–environment int","pageStart":"11","pageEnd":"23","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Comparative transcriptomics of cyprinid minnows and carp in a common wild setting: a resource for ecological genomics in freshwater communities","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Comparative transcriptomics of cyprinid minnows and carp in a common wild setting: a resource for ecological genomics in freshwater communities" />
    <meta property="og:description" content="Abstract.  Comparative transcriptomics can now be conducted on organisms in natural settings, which has greatly enhanced understanding of genome–environment int" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/1/11/4106313" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-09-06" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Krabbenhoft, Trevor J" /><meta name="citation_author_institution" content="Department of Biology and Museum of Southwestern Biology, University of New Mexico, Albuquerque, NM 87131, USA" /><meta name="citation_author" content="Turner, Thomas F" /><meta name="citation_author_institution" content="Department of Biology and Museum of Southwestern Biology, University of New Mexico, Albuquerque, NM 87131, USA" /><meta name="citation_title" content="Comparative transcriptomics of cyprinid minnows and carp in a common wild setting: a resource for ecological genomics in freshwater communities" /><meta name="citation_firstpage" content="11" /><meta name="citation_lastpage" content="23" /><meta name="citation_doi" content="10.1093/dnares/dsx034" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2018/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The impact of next-generation sequencing technology on genetics;  citation_journal_title=Trends Genet;  citation_year=2008;  citation_volume=24;  citation_pages=133-41; " /><meta name="citation_reference" content="citation_title=Next-generation DNA sequencing;  citation_journal_title=Nat. Biotechnol;  citation_year=2008;  citation_volume=26;  citation_pages=1135-45; " /><meta name="citation_reference" content="citation_title=Applications of next generation sequencing in molecular ecology of non-model organisms;  citation_journal_title=Heredity;  citation_year=2011;  citation_volume=107;  citation_pages=1-15; " /><meta name="citation_reference" content="citation_title=Ten years of transcriptomics in wild populations: what have we learned about their ecology and evolution?;  citation_journal_title=Mol. Ecol;  citation_year=2015;  citation_volume=24;  citation_pages=710-25; " /><meta name="citation_reference" content="citation_title=Harnessing the power of RADseq for ecological and evolutionary genomics;  citation_journal_title=Nat. Rev. Genet;  citation_year=2016;  citation_volume=17;  citation_pages=81-92; " /><meta name="citation_reference" content="citation_title=Genomic basis for coral resilience to climate change;  citation_journal_title=Proc. Natl. Acad. Sci;  citation_year=2013;  citation_volume=110;  citation_pages=1387-92; " /><meta name="citation_reference" content="citation_title=Transcriptome analyses and differential gene expression in a non-model fish species with alternative mating tactics;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=167" /><meta name="citation_reference" content="citation_title=De novo assembly of the Carcinus maenas transcriptome and characterization of innate immune system pathways;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=1" /><meta name="citation_reference" content="citation_title=What is needed for next-generation ecological and evolutionary genomics?;  citation_journal_title=Trends Ecol. Evol;  citation_year=2012;  citation_volume=27;  citation_pages=673-78; " /><meta name="citation_reference" content="citation_publisher=Oxford University Press, ;  citation_title=An Introduction to Ecological Genomics;  citation_year=2012; " /><meta name="citation_reference" content="citation_title=Recent advances in ecological genomics: from phenotypic plasticity to convergent and adaptive evolution and speciation;  citation_journal_title=Adv. Exp. Med. Biol;  citation_year=2014;  citation_volume=781;  citation_pages=1-5; " /><meta name="citation_reference" content="citation_title=Community transcriptomics reveals unexpected high microbial diversity in acidophilic biofilm communities;  citation_journal_title=ISME J;  citation_year=2015;  citation_volume=9;  citation_pages=1014-23; " /><meta name="citation_reference" content="citation_title=Plastic and evolved responses to global change: what can we learn from comparative transcriptomics?;  citation_journal_title=J. Hered;  citation_year=2016;  citation_volume=107;  citation_pages=71-81; " /><meta name="citation_reference" content="citation_title=Comparative transcriptomics and gene expression in larval tiger salamander (Ambystoma tigrinum) gill and lung tissues as revealed by pyrosequencing;  citation_journal_title=Gene;  citation_year=2012;  citation_volume=492;  citation_pages=329-38; " /><meta name="citation_reference" content="citation_title=Comparative transcriptomics implicates mechanisms of evolved pollution tolerance in a killifish population;  citation_journal_title=Mol. Ecol;  citation_year=2010;  citation_volume=19;  citation_pages=5186-203; " /><meta name="citation_reference" content="citation_title=Comparative transcriptomics of drought responses in Populus: a meta-analysis of genome-wide expression profiling in mature leaves and root apices across two genotypes;  citation_journal_title=BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=1" /><meta name="citation_reference" content="citation_title=Escape from adaptive conflict after duplication in an anthocyanin pathway gene;  citation_journal_title=Nature;  citation_year=2008;  citation_volume=454;  citation_pages=762-65; " /><meta name="citation_reference" content="citation_title=The genomic landscape of rapid repeated evolutionary adaptation to toxic pollution in wild fish;  citation_journal_title=Science;  citation_year=2016;  citation_volume=354;  citation_pages=1305-8; " /><meta name="citation_reference" content="citation_title=Annual killifish transcriptomics and candidate genes for metazoan diapause;  citation_journal_title=Mol. Biol. Evol;  citation_year=2016;  citation_volume=33;  citation_pages=2391-95; " /><meta name="citation_reference" content="citation_title=RNA-seq reveals transcriptomic shock involving transposable elements reactivation in hybrids of young lake whitefish species;  citation_journal_title=Mol. Biol. Evol;  citation_year=2014;  citation_volume=31;  citation_pages=1188-99; " /><meta name="citation_reference" content="citation_title=Genomics of adaptation to multiple concurrent stresses: insights from comparative transcriptomics of a Cichlid fish from one of earth’s most extreme environments, the Hypersaline Soda Lake Magadi in Kenya, East Africa;  citation_journal_title=J. Mol. Evol;  citation_year=2015;  citation_volume=81;  citation_pages=90-109; " /><meta name="citation_reference" content="citation_title=Genomic resources for a model in adaptation and speciation research: characterization of the Poecilia mexicana transcriptome;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=652" /><meta name="citation_reference" content="citation_title=Making the most of drought and salinity transcriptomics;  citation_journal_title=Plant Cell Environ;  citation_year=2010;  citation_volume=33;  citation_pages=648-54; " /><meta name="citation_reference" content="citation_title=Adaptation of Rhizobium leguminosarum to pea, alfalfa and sugar beet rhizospheres investigated by comparative transcriptomics;  citation_journal_title=Genome Biol;  citation_year=2011;  citation_volume=12;  citation_pages=1" /><meta name="citation_reference" content="citation_title=Functional genomics of adaptation to hypoxic cold-stress in high-altitude deer mice: transcriptomic plasticity and thermogenic performance;  citation_journal_title=Evolution;  citation_year=2014;  citation_volume=68;  citation_pages=48-62; " /><meta name="citation_reference" content="citation_title=Global transcriptomic profiling demonstrates induction of oxidative stress and of compensatory cellular stress responses in brown trout exposed to glyphosate and Roundup;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=1" /><meta name="citation_reference" content="citation_publisher=John Wiley &amp; Sons, Hoboken, NJ;  citation_title=Fishes of the World;  citation_year=2006; " /><meta name="citation_reference" content="citation_publisher=Academic Press, San Diego, CA;  citation_title=Freshwater fish distribution;  citation_year=2001; " /><meta name="citation_reference" content="citation_publisher=Springer, ;  citation_title=Cyprinid Fishes;  citation_year=1991;  citation_pages=552-71; " /><meta name="citation_reference" content="citation_title=Carbon, nitrogen, and phosphorous stoichiometry of cyprinid fishes;  citation_journal_title=Ecology;  citation_year=2000;  citation_volume=81;  citation_pages=127-40; " /><meta name="citation_reference" content="citation_title=Grazing minnows, piscivorous bass, and stream algae: dynamics of a strong interaction;  citation_journal_title=Ecology;  citation_year=1985;  citation_volume=66;  citation_pages=1448-56; " /><meta name="citation_reference" content="citation_title=Zebrafish hox clusters and vertebrate genome evolution;  citation_journal_title=Science;  citation_year=1998;  citation_volume=282;  citation_pages=1711-14; " /><meta name="citation_reference" content="citation_title=Vertebrate genome evolution and the zebrafish gene map;  citation_journal_title=Nature Genet;  citation_year=1998;  citation_volume=18;  citation_pages=345-49; " /><meta name="citation_reference" content="citation_title=From 2R to 3R: evidence for a fish-specific genome duplication (FSGD);  citation_journal_title=BioEssays;  citation_year=2005;  citation_volume=27;  citation_pages=937-45; " /><meta name="citation_reference" content="citation_title=Recent duplication of the common carp (Cyprinus carpio L.) genome as revealed by analyses of microsatellite loci;  citation_journal_title=Mol. Biol. Evol;  citation_year=2003;  citation_volume=20;  citation_pages=1425-34; " /><meta name="citation_reference" content="citation_title=Evolutionary history and speciation modes in the cyprinid genus Barbus;  citation_journal_title=Proc. Roy. Soc. Lond B Biol. Sci;  citation_year=2001;  citation_volume=268;  citation_pages=1297-306; " /><meta name="citation_reference" content="citation_title=Diploid-tetraploid relationship among old-world members of the fish family Cyprinidae;  citation_journal_title=Chromosoma;  citation_year=1967;  citation_volume=23;  citation_pages=1-9; " /><meta name="citation_reference" content="citation_title=Phylogeny and polyploidy: Resolving the classification of cyprinine fishes (Teleostei: Cypriniformes);  citation_journal_title=Mol. Phylogenet. Evol;  citation_year=2015;  citation_volume=85;  citation_pages=97-116; " /><meta name="citation_reference" content="citation_title=Transcriptome analysis reveals the time of the fourth round of genome duplication in common carp (Cyprinus carpio);  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=96" /><meta name="citation_reference" content="citation_title=Genome sequence and genetic diversity of the common carp, Cyprinus carpio;  citation_journal_title=Nature Genet;  citation_year=2014;  citation_volume=46;  citation_pages=1212-1219; " /><meta name="citation_reference" content="citation_title=High level of gene silencing in the tetraploid goldfish;  citation_journal_title=Biochem. Syst. Ecol;  citation_year=1984;  citation_volume=12;  citation_pages=415-21; " /><meta name="citation_reference" content="citation_title=The evolution of duplicate gene expression in the carp (Cyprinus carpio);  citation_journal_title=Experientia;  citation_year=1977;  citation_volume=33;  citation_pages=1299-1301; " /><meta name="citation_reference" content="citation_title=The zebrafish: a new model organism for integrative physiology;  citation_journal_title=Am. J. Physiol. Regul. Integr. Comp. Physiol;  citation_year=2002;  citation_volume=282;  citation_pages=R3-R9; " /><meta name="citation_reference" content="citation_title=Zebrafish: a model system for the study of human disease;  citation_journal_title=Curr. Opinion Genet. Dev;  citation_year=2000;  citation_volume=10;  citation_pages=252-56; " /><meta name="citation_reference" content="citation_title=Animal models of human disease: zebrafish swim into view;  citation_journal_title=Nat. Rev. Genet;  citation_year=2007;  citation_volume=8;  citation_pages=353-367; " /><meta name="citation_reference" content="citation_title=The zebrafish reference genome sequence and its relationship to the human genome;  citation_journal_title=Nature;  citation_year=2013;  citation_volume=496;  citation_pages=498-503; " /><meta name="citation_reference" content="citation_title=The fathead minnow in aquatic toxicology: past, present and future;  citation_journal_title=Aquatic Toxicol;  citation_year=2006;  citation_volume=78;  citation_pages=91-102; " /><meta name="citation_reference" content="citation_title=Use of a 15 k gene microarray to determine gene expression changes in response to acute and chronic methylmercury exposure in the fathead minnow Pimephales promelas Rafinesque;  citation_journal_title=J. Fish Biol;  citation_year=2008;  citation_volume=72;  citation_pages=2207-80; " /><meta name="citation_reference" content="citation_title=Sequencing and de novo draft assemblies of a fathead minnow (Pimephales promelas) reference genome;  citation_journal_title=Environ. Toxicol. Chem.;  citation_year=2016;  citation_volume=35;  citation_pages=212-217; " /><meta name="citation_reference" content="citation_author=FAO; citation_publisher=FAO, Rome, Italy;  citation_title=FISHSTAT: Global Aquaculture Production;  citation_year=2014; " /><meta name="citation_reference" content="citation_title=Characterization of common carp transcriptome: de novo sequencing, assembly, annotation and comparative genomics;  citation_journal_title=PloS One;  citation_year=2012;  citation_volume=7;  citation_pages=1-9; " /><meta name="citation_reference" content="citation_title=De novo assembly and characterization of the spleen transcriptome of common carp (Cyprinus carpio) using illumina paired-end sequencing;  citation_journal_title=Fish Shellfish Immunol;  citation_year=2015; " /><meta name="citation_reference" content="citation_publisher=W. &amp; A. White &amp; J. Visscher, Albany, NY;  citation_title=Zoology of New-York, or, the New-York fauna; comprising detailed descriptions of all the animals hitherto observed within the state of New-York, with brief notices of those occasionally found near its borders, and accompanied by appropriate illustrations. Part IV. Fishes;  citation_year=1842; " /><meta name="citation_reference" content="citation_title=Report of the commissioner of fish and fisheries for the fiscal years 1889-90 and 1890-91. Part XVII;  citation_journal_title=U.S. Commission of Fish and Fisheries. Washington, DC;  citation_year=1893;  citation_pages=1-96; " /><meta name="citation_reference" content="citation_title=Red shiner invasion and hybridization with blacktail shiner in the upper Coosa River, USA;  citation_journal_title=Biol. Invas;  citation_year=2008;  citation_volume=10;  citation_pages=1229-1242; " /><meta name="citation_reference" content="citation_title=Spatial variation in relative abundance of a widespread, numerically dominant fish species and its effect on fish assemblage structure;  citation_journal_title=Oecologia;  citation_year=2000;  citation_volume=125;  citation_pages=283-292; " /><meta name="citation_reference" content="citation_title=Historical fish assemblage flux in the Cheyenne River below Angostura Dam;  citation_journal_title=J. Freshwater Ecol;  citation_year=2007;  citation_volume=22;  citation_pages=219-29; " /><meta name="citation_reference" content="citation_title=Identification of 315 genes essential for early zebrafish development;  citation_journal_title=Proc. Natl. Acad. Sci USA;  citation_year=2004;  citation_volume=101;  citation_pages=12792-97; " /><meta name="citation_reference" content="citation_title=OGEE: an online gene essentiality database;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=D901-D906; " /><meta name="citation_reference" content="citation_title=Ensembl 2014;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=D749-755; " /><meta name="citation_reference" content="citation_title=Comparative genomics of centrality and essentiality in three eukaryotic protein-interaction networks;  citation_journal_title=Mol. Biol. Evol;  citation_year=2005;  citation_volume=22;  citation_pages=803-6; " /><meta name="citation_reference" content="citation_title=The evolution of gene duplications: classifying and distinguishing between models;  citation_journal_title=Nat. Rev. Genet;  citation_year=2010;  citation_volume=11;  citation_pages=97-108; " /><meta name="citation_reference" content="citation_title=The odds of duplicate gene persistence after polyploidization;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=1" /><meta name="citation_reference" content="Bolger A. M., Lohse M., Usadel B.  2014 , Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics, btu170. " /><meta name="citation_reference" content="citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome;  citation_journal_title=Nat. Biotechnol;  citation_year=2011;  citation_volume=29;  citation_pages=644-52; " /><meta name="citation_reference" content="citation_title=De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis;  citation_journal_title=Nat. Protocols;  citation_year=2013;  citation_volume=8;  citation_pages=1494-512; " /><meta name="citation_reference" content="citation_title=SILVA: a comprehensive online resource for quality checked and aligned ribosomal RNA sequence data compatible with ARB;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=7188-96; " /><meta name="citation_reference" content="citation_title=Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=3674-76; " /><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=357-59; " /><meta name="citation_reference" content="citation_title=RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome;  citation_journal_title=BMC Bioinform;  citation_year=2011;  citation_volume=12;  citation_pages=323" /><meta name="citation_reference" content="citation_title=WGCNA: an R package for weighted correlation network analysis;  citation_journal_title=BMC Bioinform;  citation_year=2008;  citation_volume=9;  citation_pages=559" /><meta name="citation_reference" content="citation_title=PANTHER in 2013: modeling the evolution of gene function, and other gene attributes, in the context of phylogenetic trees;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=D377-386; " /><meta name="citation_reference" content="citation_title=REVIGO summarizes and visualizes long lists of gene ontology terms;  citation_journal_title=PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e21800" /><meta name="citation_reference" content="citation_title=Ten years of next-generation sequencing technology;  citation_journal_title=Trends Genet;  citation_year=2014;  citation_volume=30;  citation_pages=418-26; " /><meta name="citation_reference" content="citation_title=On the maintenance of genetic variation and adaptation to environmental change: considerations from population genomics in fishes;  citation_journal_title=J. Fish Biol;  citation_year=2016;  citation_volume=89;  citation_pages=2519-56; " /><meta name="citation_reference" content="citation_title=Next-generation transcriptome assembly;  citation_journal_title=Nat. Rev. Genet;  citation_year=2011;  citation_volume=12;  citation_pages=671-82; " /><meta name="citation_reference" content="citation_title=Evaluation of de novo transcriptome assemblies from RNA-Seq data;  citation_journal_title=Genome Biol;  citation_year=2014;  citation_volume=15;  citation_pages=1" /><meta name="citation_reference" content="citation_title=Assessing De Novo transcriptome assembly metrics for consistency and utility;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=465" /><meta name="citation_reference" content="citation_title=SNP calling from RNA-seq data without a reference genome: identification, quantification, differential analysis and impact on the protein sequence;  citation_journal_title=Nucleic Acids Res;  citation_year=2016;  citation_volume=44;  citation_pages=e148-e148; " /><meta name="citation_reference" content="citation_title=Of contigs and quagmires: next-generation sequencing pitfalls associated with transcriptomic studies;  citation_journal_title=Mol. Ecol. Resour;  citation_year=2013;  citation_volume=13;  citation_pages=551-58; " /><meta name="citation_reference" content="citation_title=Principles of transcriptome analysis and gene expression quantification: an RNA-seq tutorial;  citation_journal_title=Mol. Ecol. Resour;  citation_year=2013;  citation_volume=13;  citation_pages=559-72; " /><meta name="citation_reference" content="citation_title=Challenges and strategies in transcriptome assembly and differential gene expression quantification. A comprehensive in silico assessment of RNA‐seq experiments;  citation_journal_title=Mol. Ecol;  citation_year=2013;  citation_volume=22;  citation_pages=620-34; " /><meta name="citation_reference" content="citation_title=Lineage-specific loss and divergence of functionally linked genes in eukaryotes;  citation_journal_title=Proc. Natl. Acad. Sci;  citation_year=2000;  citation_volume=97;  citation_pages=11319-24; " /><meta name="citation_reference" content="citation_title=Conservation and divergence of gene families encoding components of innate immune response systems in zebrafish;  citation_journal_title=Genome Biol;  citation_year=2007;  citation_volume=8;  citation_pages=1" /><meta name="citation_reference" content="citation_title=The genome of the Gulf pipefish enables understanding of evolutionary innovations;  citation_journal_title=Genome Biol;  citation_year=2016;  citation_volume=17;  citation_pages=258" /><meta name="citation_reference" content="citation_title=High throughput sequencing methods and analysis for microbiome research;  citation_journal_title=J. Microbiol. Methods;  citation_year=2013;  citation_volume=95;  citation_pages=401-14; " /><meta name="citation_reference" content="citation_title=PARSES: a pipeline for analysis of RNA-Seq exogenous sequences;  citation_journal_title=BICoB;  citation_year=2011;  citation_pages=196-200; " /><meta name="citation_reference" content="citation_title=Defining the role of essential genes in human disease;  citation_journal_title=PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e27368" /><meta name="citation_reference" content="citation_title=DEG 10, an update of the database of essential genes that includes both protein-coding genes and noncoding genomic elements;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=D574-D580; " /><meta name="citation_reference" content="citation_title=Genetic inviability is a major driver of type‐III survivorship in experimental families of a highly fecund marine bivalve;  citation_journal_title=Mol. Ecol;  citation_year=2016; " /><meta name="citation_reference" content="citation_title=Characterization of housekeeping genes in zebrafish: male-female differences and effects of tissue type, developmental stage and chemical treatment;  citation_journal_title=BMC Mol. Biol;  citation_year=2008;  citation_volume=9;  citation_pages=1" /><meta name="citation_reference" content="citation_title=Evidence from mitochondrial genomics supports the lower Mesozoic of South Asia as the time and place of basal divergence of cypriniform fishes (Actinopterygii: Ostariophysi);  citation_journal_title=Zool. J. Linnean Soc;  citation_year=2011;  citation_volume=161;  citation_pages=633-62; " /><meta name="citation_reference" content="citation_title=The zebrafish genome in context: ohnologs gone missing;  citation_journal_title=J. Exp. Zool. B Mol. Dev. Evol;  citation_year=2007;  citation_volume=308;  citation_pages=563-77; " /><meta name="citation_reference" content="citation_title=Loss of duplicate gene expression after polyploidisation;  citation_journal_title=Nature;  citation_year=1977;  citation_volume=265;  citation_pages=258-60; " /><meta name="citation_reference" content="citation_title=Molecular genetic aspects of tetraploidy in the common carp Cyprinus carpio;  citation_journal_title=Mol. Phylogenet. Evol;  citation_year=1994;  citation_volume=3;  citation_pages=59-68; " /><meta name="citation_reference" content="citation_publisher=Springer, New York, NY;  citation_title=Evolution by Gene Duplication;  citation_year=1970; " /><meta name="citation_reference" content="citation_title=The rainbow trout genome provides novel insights into evolution after whole-genome duplication in vertebrates;  citation_journal_title=Nat. Commun;  citation_year=2014;  citation_volume=5;  citation_pages=3657" /><meta name="citation_reference" content="citation_title=Genome evolution in the fish family salmonidae: generation of a brook charr genetic map and comparisons among charrs (Arctic charr and brook charr) with rainbow trout;  citation_journal_title=BMC Genet;  citation_year=2011;  citation_volume=12;  citation_pages=1-15; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/1/11/24151415/dsx034.pdf" /><meta name="description" content="Abstract.  Comparative transcriptomics can now be conducted on organisms in natural settings, which has greatly enhanced understanding of genome–environment int" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/1/11/4106313" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/1/11/4106313" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/1/11/4106313");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx034';
    </script>
<title>
	Comparative transcriptomics of cyprinid minnows and carp in a common wild setting: a resource for ecological genomics in freshwater communities | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4106313" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bd3OATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS3JhYmJlbmhvZnQsIFRyZXZvciBKIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9sb2d5IGFuZCBNdXNldW0gb2YgU291dGh3ZXN0ZXJuIEJpb2xvZ3ksIFVuaXZlcnNpdHkgb2YgTmV3IE1leGljbywgQWxidXF1ZXJxdWUsIE5NIDg3MTMxLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUdXJuZXIsIFRob21hcyBGIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9sb2d5IGFuZCBNdXNldW0gb2YgU291dGh3ZXN0ZXJuIEJpb2xvZ3ksIFVuaXZlcnNpdHkgb2YgTmV3IE1leGljbywgQWxidXF1ZXJxdWUsIE5NIDg3MTMxLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkNvbXBhcmF0aXZlIHRyYW5zY3JpcHRvbWljcyBvZiBjeXByaW5pZCBtaW5ub3dzIGFuZCBjYXJwIGluIGEgY29tbW9uIHdpbGQgc2V0dGluZzogYSByZXNvdXJjZSBmb3IgZWNvbG9naWNhbCBnZW5vbWljcyBpbiBmcmVzaHdhdGVyIGNvbW11bml0aWVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN4MDM0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTgvMDIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgaW1wYWN0IG9mIG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIHRlY2hub2xvZ3kgb24gZ2VuZXRpY3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTEzMy00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmV4dC1nZW5lcmF0aW9uIEROQSBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz0xMTM1LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcHBsaWNhdGlvbnMgb2YgbmV4dCBnZW5lcmF0aW9uIHNlcXVlbmNpbmcgaW4gbW9sZWN1bGFyIGVjb2xvZ3kgb2Ygbm9uLW1vZGVsIG9yZ2FuaXNtczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SGVyZWRpdHk7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTA3OyAgY2l0YXRpb25fcGFnZXM9MS0xNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGVuIHllYXJzIG9mIHRyYW5zY3JpcHRvbWljcyBpbiB3aWxkIHBvcHVsYXRpb25zOiB3aGF0IGhhdmUgd2UgbGVhcm5lZCBhYm91dCB0aGVpciBlY29sb2d5IGFuZCBldm9sdXRpb24/OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEVjb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz03MTAtMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhhcm5lc3NpbmcgdGhlIHBvd2VyIG9mIFJBRHNlcSBmb3IgZWNvbG9naWNhbCBhbmQgZXZvbHV0aW9uYXJ5IGdlbm9taWNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz04MS05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBiYXNpcyBmb3IgY29yYWwgcmVzaWxpZW5jZSB0byBjbGltYXRlIGNoYW5nZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTEzODctOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRvbWUgYW5hbHlzZXMgYW5kIGRpZmZlcmVudGlhbCBnZW5lIGV4cHJlc3Npb24gaW4gYSBub24tbW9kZWwgZmlzaCBzcGVjaWVzIHdpdGggYWx0ZXJuYXRpdmUgbWF0aW5nIHRhY3RpY3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTE2NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlIG5vdm8gYXNzZW1ibHkgb2YgdGhlIENhcmNpbnVzIG1hZW5hcyB0cmFuc2NyaXB0b21lIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGlubmF0ZSBpbW11bmUgc3lzdGVtIHBhdGh3YXlzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hhdCBpcyBuZWVkZWQgZm9yIG5leHQtZ2VuZXJhdGlvbiBlY29sb2dpY2FsIGFuZCBldm9sdXRpb25hcnkgZ2Vub21pY3M/OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgRWNvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTY3My03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPU94Zm9yZCBVbml2ZXJzaXR5IFByZXNzLCA7ICBjaXRhdGlvbl90aXRsZT1BbiBJbnRyb2R1Y3Rpb24gdG8gRWNvbG9naWNhbCBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVjZW50IGFkdmFuY2VzIGluIGVjb2xvZ2ljYWwgZ2Vub21pY3M6IGZyb20gcGhlbm90eXBpYyBwbGFzdGljaXR5IHRvIGNvbnZlcmdlbnQgYW5kIGFkYXB0aXZlIGV2b2x1dGlvbiBhbmQgc3BlY2lhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QWR2LiBFeHAuIE1lZC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT03ODE7ICBjaXRhdGlvbl9wYWdlcz0xLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbW11bml0eSB0cmFuc2NyaXB0b21pY3MgcmV2ZWFscyB1bmV4cGVjdGVkIGhpZ2ggbWljcm9iaWFsIGRpdmVyc2l0eSBpbiBhY2lkb3BoaWxpYyBiaW9maWxtIGNvbW11bml0aWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JU01FIEo7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTEwMTQtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsYXN0aWMgYW5kIGV2b2x2ZWQgcmVzcG9uc2VzIHRvIGdsb2JhbCBjaGFuZ2U6IHdoYXQgY2FuIHdlIGxlYXJuIGZyb20gY29tcGFyYXRpdmUgdHJhbnNjcmlwdG9taWNzPzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gSGVyZWQ7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTA3OyAgY2l0YXRpb25fcGFnZXM9NzEtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIHRyYW5zY3JpcHRvbWljcyBhbmQgZ2VuZSBleHByZXNzaW9uIGluIGxhcnZhbCB0aWdlciBzYWxhbWFuZGVyIChBbWJ5c3RvbWEgdGlncmludW0pIGdpbGwgYW5kIGx1bmcgdGlzc3VlcyBhcyByZXZlYWxlZCBieSBweXJvc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00OTI7ICBjaXRhdGlvbl9wYWdlcz0zMjktMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIHRyYW5zY3JpcHRvbWljcyBpbXBsaWNhdGVzIG1lY2hhbmlzbXMgb2YgZXZvbHZlZCBwb2xsdXRpb24gdG9sZXJhbmNlIGluIGEga2lsbGlmaXNoIHBvcHVsYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gRWNvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTUxODYtMjAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSB0cmFuc2NyaXB0b21pY3Mgb2YgZHJvdWdodCByZXNwb25zZXMgaW4gUG9wdWx1czogYSBtZXRhLWFuYWx5c2lzIG9mIGdlbm9tZS13aWRlIGV4cHJlc3Npb24gcHJvZmlsaW5nIGluIG1hdHVyZSBsZWF2ZXMgYW5kIHJvb3QgYXBpY2VzIGFjcm9zcyB0d28gZ2Vub3R5cGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXNjYXBlIGZyb20gYWRhcHRpdmUgY29uZmxpY3QgYWZ0ZXIgZHVwbGljYXRpb24gaW4gYW4gYW50aG9jeWFuaW4gcGF0aHdheSBnZW5lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NDU0OyAgY2l0YXRpb25fcGFnZXM9NzYyLTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21pYyBsYW5kc2NhcGUgb2YgcmFwaWQgcmVwZWF0ZWQgZXZvbHV0aW9uYXJ5IGFkYXB0YXRpb24gdG8gdG94aWMgcG9sbHV0aW9uIGluIHdpbGQgZmlzaDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNTQ7ICBjaXRhdGlvbl9wYWdlcz0xMzA1LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFubnVhbCBraWxsaWZpc2ggdHJhbnNjcmlwdG9taWNzIGFuZCBjYW5kaWRhdGUgZ2VuZXMgZm9yIG1ldGF6b2FuIGRpYXBhdXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz0yMzkxLTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1STkEtc2VxIHJldmVhbHMgdHJhbnNjcmlwdG9taWMgc2hvY2sgaW52b2x2aW5nIHRyYW5zcG9zYWJsZSBlbGVtZW50cyByZWFjdGl2YXRpb24gaW4gaHlicmlkcyBvZiB5b3VuZyBsYWtlIHdoaXRlZmlzaCBzcGVjaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0xMTg4LTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljcyBvZiBhZGFwdGF0aW9uIHRvIG11bHRpcGxlIGNvbmN1cnJlbnQgc3RyZXNzZXM6IGluc2lnaHRzIGZyb20gY29tcGFyYXRpdmUgdHJhbnNjcmlwdG9taWNzIG9mIGEgQ2ljaGxpZCBmaXNoIGZyb20gb25lIG9mIGVhcnRo4oCZcyBtb3N0IGV4dHJlbWUgZW52aXJvbm1lbnRzLCB0aGUgSHlwZXJzYWxpbmUgU29kYSBMYWtlIE1hZ2FkaSBpbiBLZW55YSwgRWFzdCBBZnJpY2E7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIE1vbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT04MTsgIGNpdGF0aW9uX3BhZ2VzPTkwLTEwOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyByZXNvdXJjZXMgZm9yIGEgbW9kZWwgaW4gYWRhcHRhdGlvbiBhbmQgc3BlY2lhdGlvbiByZXNlYXJjaDogY2hhcmFjdGVyaXphdGlvbiBvZiB0aGUgUG9lY2lsaWEgbWV4aWNhbmEgdHJhbnNjcmlwdG9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9NjUyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFraW5nIHRoZSBtb3N0IG9mIGRyb3VnaHQgYW5kIHNhbGluaXR5IHRyYW5zY3JpcHRvbWljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbCBFbnZpcm9uOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9NjQ4LTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZGFwdGF0aW9uIG9mIFJoaXpvYml1bSBsZWd1bWlub3NhcnVtIHRvIHBlYSwgYWxmYWxmYSBhbmQgc3VnYXIgYmVldCByaGl6b3NwaGVyZXMgaW52ZXN0aWdhdGVkIGJ5IGNvbXBhcmF0aXZlIHRyYW5zY3JpcHRvbWljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuY3Rpb25hbCBnZW5vbWljcyBvZiBhZGFwdGF0aW9uIHRvIGh5cG94aWMgY29sZC1zdHJlc3MgaW4gaGlnaC1hbHRpdHVkZSBkZWVyIG1pY2U6IHRyYW5zY3JpcHRvbWljIHBsYXN0aWNpdHkgYW5kIHRoZXJtb2dlbmljIHBlcmZvcm1hbmNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Fdm9sdXRpb247ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9Njg7ICBjaXRhdGlvbl9wYWdlcz00OC02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2xvYmFsIHRyYW5zY3JpcHRvbWljIHByb2ZpbGluZyBkZW1vbnN0cmF0ZXMgaW5kdWN0aW9uIG9mIG94aWRhdGl2ZSBzdHJlc3MgYW5kIG9mIGNvbXBlbnNhdG9yeSBjZWxsdWxhciBzdHJlc3MgcmVzcG9uc2VzIGluIGJyb3duIHRyb3V0IGV4cG9zZWQgdG8gZ2x5cGhvc2F0ZSBhbmQgUm91bmR1cDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1Kb2huIFdpbGV5ICZhbXA7IFNvbnMsIEhvYm9rZW4sIE5KOyAgY2l0YXRpb25fdGl0bGU9RmlzaGVzIG9mIHRoZSBXb3JsZDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUFjYWRlbWljIFByZXNzLCBTYW4gRGllZ28sIENBOyAgY2l0YXRpb25fdGl0bGU9RnJlc2h3YXRlciBmaXNoIGRpc3RyaWJ1dGlvbjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPVNwcmluZ2VyLCA7ICBjaXRhdGlvbl90aXRsZT1DeXByaW5pZCBGaXNoZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl9wYWdlcz01NTItNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNhcmJvbiwgbml0cm9nZW4sIGFuZCBwaG9zcGhvcm91cyBzdG9pY2hpb21ldHJ5IG9mIGN5cHJpbmlkIGZpc2hlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RWNvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT04MTsgIGNpdGF0aW9uX3BhZ2VzPTEyNy00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R3JhemluZyBtaW5ub3dzLCBwaXNjaXZvcm91cyBiYXNzLCBhbmQgc3RyZWFtIGFsZ2FlOiBkeW5hbWljcyBvZiBhIHN0cm9uZyBpbnRlcmFjdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RWNvbG9neTsgIGNpdGF0aW9uX3llYXI9MTk4NTsgIGNpdGF0aW9uX3ZvbHVtZT02NjsgIGNpdGF0aW9uX3BhZ2VzPTE0NDgtNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVplYnJhZmlzaCBob3ggY2x1c3RlcnMgYW5kIHZlcnRlYnJhdGUgZ2Vub21lIGV2b2x1dGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0yODI7ICBjaXRhdGlvbl9wYWdlcz0xNzExLTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WZXJ0ZWJyYXRlIGdlbm9tZSBldm9sdXRpb24gYW5kIHRoZSB6ZWJyYWZpc2ggZ2VuZSBtYXA7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZSBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTM0NS00OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnJvbSAyUiB0byAzUjogZXZpZGVuY2UgZm9yIGEgZmlzaC1zcGVjaWZpYyBnZW5vbWUgZHVwbGljYXRpb24gKEZTR0QpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9Fc3NheXM7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz05MzctNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlY2VudCBkdXBsaWNhdGlvbiBvZiB0aGUgY29tbW9uIGNhcnAgKEN5cHJpbnVzIGNhcnBpbyBMLikgZ2Vub21lIGFzIHJldmVhbGVkIGJ5IGFuYWx5c2VzIG9mIG1pY3Jvc2F0ZWxsaXRlIGxvY2k7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTE0MjUtMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbmFyeSBoaXN0b3J5IGFuZCBzcGVjaWF0aW9uIG1vZGVzIGluIHRoZSBjeXByaW5pZCBnZW51cyBCYXJidXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIFJveS4gU29jLiBMb25kIEIgQmlvbC4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI2ODsgIGNpdGF0aW9uX3BhZ2VzPTEyOTctMzA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXBsb2lkLXRldHJhcGxvaWQgcmVsYXRpb25zaGlwIGFtb25nIG9sZC13b3JsZCBtZW1iZXJzIG9mIHRoZSBmaXNoIGZhbWlseSBDeXByaW5pZGFlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DaHJvbW9zb21hOyAgY2l0YXRpb25feWVhcj0xOTY3OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlsb2dlbnkgYW5kIHBvbHlwbG9pZHk6IFJlc29sdmluZyB0aGUgY2xhc3NpZmljYXRpb24gb2YgY3lwcmluaW5lIGZpc2hlcyAoVGVsZW9zdGVpOiBDeXByaW5pZm9ybWVzKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBQaHlsb2dlbmV0LiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTg1OyAgY2l0YXRpb25fcGFnZXM9OTctMTE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0b21lIGFuYWx5c2lzIHJldmVhbHMgdGhlIHRpbWUgb2YgdGhlIGZvdXJ0aCByb3VuZCBvZiBnZW5vbWUgZHVwbGljYXRpb24gaW4gY29tbW9uIGNhcnAgKEN5cHJpbnVzIGNhcnBpbyk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTk2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNlIGFuZCBnZW5ldGljIGRpdmVyc2l0eSBvZiB0aGUgY29tbW9uIGNhcnAsIEN5cHJpbnVzIGNhcnBpbzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQ2OyAgY2l0YXRpb25fcGFnZXM9MTIxMi0xMjE5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoIGxldmVsIG9mIGdlbmUgc2lsZW5jaW5nIGluIHRoZSB0ZXRyYXBsb2lkIGdvbGRmaXNoOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9jaGVtLiBTeXN0LiBFY29sOyAgY2l0YXRpb25feWVhcj0xOTg0OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NDE1LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXZvbHV0aW9uIG9mIGR1cGxpY2F0ZSBnZW5lIGV4cHJlc3Npb24gaW4gdGhlIGNhcnAgKEN5cHJpbnVzIGNhcnBpbyk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV4cGVyaWVudGlhOyAgY2l0YXRpb25feWVhcj0xOTc3OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9MTI5OS0xMzAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgemVicmFmaXNoOiBhIG5ldyBtb2RlbCBvcmdhbmlzbSBmb3IgaW50ZWdyYXRpdmUgcGh5c2lvbG9neTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW0uIEouIFBoeXNpb2wuIFJlZ3VsLiBJbnRlZ3IuIENvbXAuIFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MjgyOyAgY2l0YXRpb25fcGFnZXM9UjMtUjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVplYnJhZmlzaDogYSBtb2RlbCBzeXN0ZW0gZm9yIHRoZSBzdHVkeSBvZiBodW1hbiBkaXNlYXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluaW9uIEdlbmV0LiBEZXY7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0yNTItNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuaW1hbCBtb2RlbHMgb2YgaHVtYW4gZGlzZWFzZTogemVicmFmaXNoIHN3aW0gaW50byB2aWV3OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTM1My0zNjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB6ZWJyYWZpc2ggcmVmZXJlbmNlIGdlbm9tZSBzZXF1ZW5jZSBhbmQgaXRzIHJlbGF0aW9uc2hpcCB0byB0aGUgaHVtYW4gZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDk2OyAgY2l0YXRpb25fcGFnZXM9NDk4LTUwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGZhdGhlYWQgbWlubm93IGluIGFxdWF0aWMgdG94aWNvbG9neTogcGFzdCwgcHJlc2VudCBhbmQgZnV0dXJlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcXVhdGljIFRveGljb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9Nzg7ICBjaXRhdGlvbl9wYWdlcz05MS0xMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVzZSBvZiBhIDE1IGsgZ2VuZSBtaWNyb2FycmF5IHRvIGRldGVybWluZSBnZW5lIGV4cHJlc3Npb24gY2hhbmdlcyBpbiByZXNwb25zZSB0byBhY3V0ZSBhbmQgY2hyb25pYyBtZXRoeWxtZXJjdXJ5IGV4cG9zdXJlIGluIHRoZSBmYXRoZWFkIG1pbm5vdyBQaW1lcGhhbGVzIHByb21lbGFzIFJhZmluZXNxdWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEZpc2ggQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT03MjsgIGNpdGF0aW9uX3BhZ2VzPTIyMDctODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNpbmcgYW5kIGRlIG5vdm8gZHJhZnQgYXNzZW1ibGllcyBvZiBhIGZhdGhlYWQgbWlubm93IChQaW1lcGhhbGVzIHByb21lbGFzKSByZWZlcmVuY2UgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FbnZpcm9uLiBUb3hpY29sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPTIxMi0yMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX2F1dGhvcj1GQU87IGNpdGF0aW9uX3B1Ymxpc2hlcj1GQU8sIFJvbWUsIEl0YWx5OyAgY2l0YXRpb25fdGl0bGU9RklTSFNUQVQ6IEdsb2JhbCBBcXVhY3VsdHVyZSBQcm9kdWN0aW9uOyAgY2l0YXRpb25feWVhcj0yMDE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIGNvbW1vbiBjYXJwIHRyYW5zY3JpcHRvbWU6IGRlIG5vdm8gc2VxdWVuY2luZywgYXNzZW1ibHksIGFubm90YXRpb24gYW5kIGNvbXBhcmF0aXZlIGdlbm9taWNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZSBub3ZvIGFzc2VtYmx5IGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIHRoZSBzcGxlZW4gdHJhbnNjcmlwdG9tZSBvZiBjb21tb24gY2FycCAoQ3lwcmludXMgY2FycGlvKSB1c2luZyBpbGx1bWluYSBwYWlyZWQtZW5kIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZpc2ggU2hlbGxmaXNoIEltbXVub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1XLiAmYW1wOyBBLiBXaGl0ZSAmYW1wOyBKLiBWaXNzY2hlciwgQWxiYW55LCBOWTsgIGNpdGF0aW9uX3RpdGxlPVpvb2xvZ3kgb2YgTmV3LVlvcmssIG9yLCB0aGUgTmV3LVlvcmsgZmF1bmE7IGNvbXByaXNpbmcgZGV0YWlsZWQgZGVzY3JpcHRpb25zIG9mIGFsbCB0aGUgYW5pbWFscyBoaXRoZXJ0byBvYnNlcnZlZCB3aXRoaW4gdGhlIHN0YXRlIG9mIE5ldy1Zb3JrLCB3aXRoIGJyaWVmIG5vdGljZXMgb2YgdGhvc2Ugb2NjYXNpb25hbGx5IGZvdW5kIG5lYXIgaXRzIGJvcmRlcnMsIGFuZCBhY2NvbXBhbmllZCBieSBhcHByb3ByaWF0ZSBpbGx1c3RyYXRpb25zLiBQYXJ0IElWLiBGaXNoZXM7ICBjaXRhdGlvbl95ZWFyPTE4NDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlcG9ydCBvZiB0aGUgY29tbWlzc2lvbmVyIG9mIGZpc2ggYW5kIGZpc2hlcmllcyBmb3IgdGhlIGZpc2NhbCB5ZWFycyAxODg5LTkwIGFuZCAxODkwLTkxLiBQYXJ0IFhWSUk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVUuUy4gQ29tbWlzc2lvbiBvZiBGaXNoIGFuZCBGaXNoZXJpZXMuIFdhc2hpbmd0b24sIERDOyAgY2l0YXRpb25feWVhcj0xODkzOyAgY2l0YXRpb25fcGFnZXM9MS05NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVkIHNoaW5lciBpbnZhc2lvbiBhbmQgaHlicmlkaXphdGlvbiB3aXRoIGJsYWNrdGFpbCBzaGluZXIgaW4gdGhlIHVwcGVyIENvb3NhIFJpdmVyLCBVU0E7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2wuIEludmFzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MTIyOS0xMjQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TcGF0aWFsIHZhcmlhdGlvbiBpbiByZWxhdGl2ZSBhYnVuZGFuY2Ugb2YgYSB3aWRlc3ByZWFkLCBudW1lcmljYWxseSBkb21pbmFudCBmaXNoIHNwZWNpZXMgYW5kIGl0cyBlZmZlY3Qgb24gZmlzaCBhc3NlbWJsYWdlIHN0cnVjdHVyZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9T2Vjb2xvZ2lhOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTEyNTsgIGNpdGF0aW9uX3BhZ2VzPTI4My0yOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpc3RvcmljYWwgZmlzaCBhc3NlbWJsYWdlIGZsdXggaW4gdGhlIENoZXllbm5lIFJpdmVyIGJlbG93IEFuZ29zdHVyYSBEYW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEZyZXNod2F0ZXIgRWNvbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTIxOS0yOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgMzE1IGdlbmVzIGVzc2VudGlhbCBmb3IgZWFybHkgemVicmFmaXNoIGRldmVsb3BtZW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kgVVNBOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTEyNzkyLTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PR0VFOiBhbiBvbmxpbmUgZ2VuZSBlc3NlbnRpYWxpdHkgZGF0YWJhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9RDkwMS1EOTA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbnNlbWJsIDIwMTQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9RDc0OS03NTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIGdlbm9taWNzIG9mIGNlbnRyYWxpdHkgYW5kIGVzc2VudGlhbGl0eSBpbiB0aHJlZSBldWthcnlvdGljIHByb3RlaW4taW50ZXJhY3Rpb24gbmV0d29ya3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTgwMy02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXZvbHV0aW9uIG9mIGdlbmUgZHVwbGljYXRpb25zOiBjbGFzc2lmeWluZyBhbmQgZGlzdGluZ3Vpc2hpbmcgYmV0d2VlbiBtb2RlbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTk3LTEwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG9kZHMgb2YgZHVwbGljYXRlIGdlbmUgcGVyc2lzdGVuY2UgYWZ0ZXIgcG9seXBsb2lkaXphdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IkJvbGdlciBBLiBNLiwgTG9oc2UgTS4sIFVzYWRlbCBCLiAgMjAxNCAsIFRyaW1tb21hdGljOiBhIGZsZXhpYmxlIHRyaW1tZXIgZm9yIElsbHVtaW5hIHNlcXVlbmNlIGRhdGEuIEJpb2luZm9ybWF0aWNzLCBidHUxNzAuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bGwtbGVuZ3RoIHRyYW5zY3JpcHRvbWUgYXNzZW1ibHkgZnJvbSBSTkEtU2VxIGRhdGEgd2l0aG91dCBhIHJlZmVyZW5jZSBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTY0NC01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGUgbm92byB0cmFuc2NyaXB0IHNlcXVlbmNlIHJlY29uc3RydWN0aW9uIGZyb20gUk5BLXNlcSB1c2luZyB0aGUgVHJpbml0eSBwbGF0Zm9ybSBmb3IgcmVmZXJlbmNlIGdlbmVyYXRpb24gYW5kIGFuYWx5c2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFByb3RvY29sczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MTQ5NC01MTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNJTFZBOiBhIGNvbXByZWhlbnNpdmUgb25saW5lIHJlc291cmNlIGZvciBxdWFsaXR5IGNoZWNrZWQgYW5kIGFsaWduZWQgcmlib3NvbWFsIFJOQSBzZXF1ZW5jZSBkYXRhIGNvbXBhdGlibGUgd2l0aCBBUkI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9NzE4OC05NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Qmxhc3QyR086IGEgdW5pdmVyc2FsIHRvb2wgZm9yIGFubm90YXRpb24sIHZpc3VhbGl6YXRpb24gYW5kIGFuYWx5c2lzIGluIGZ1bmN0aW9uYWwgZ2Vub21pY3MgcmVzZWFyY2g7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MzY3NC03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdCBnYXBwZWQtcmVhZCBhbGlnbm1lbnQgd2l0aCBCb3d0aWUgMjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0zNTctNTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJTRU06IGFjY3VyYXRlIHRyYW5zY3JpcHQgcXVhbnRpZmljYXRpb24gZnJvbSBSTkEtU2VxIGRhdGEgd2l0aCBvciB3aXRob3V0IGEgcmVmZXJlbmNlIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTMyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdHQ05BOiBhbiBSIHBhY2thZ2UgZm9yIHdlaWdodGVkIGNvcnJlbGF0aW9uIG5ldHdvcmsgYW5hbHlzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm07ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTU1OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBBTlRIRVIgaW4gMjAxMzogbW9kZWxpbmcgdGhlIGV2b2x1dGlvbiBvZiBnZW5lIGZ1bmN0aW9uLCBhbmQgb3RoZXIgZ2VuZSBhdHRyaWJ1dGVzLCBpbiB0aGUgY29udGV4dCBvZiBwaHlsb2dlbmV0aWMgdHJlZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9RDM3Ny0zODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJFVklHTyBzdW1tYXJpemVzIGFuZCB2aXN1YWxpemVzIGxvbmcgbGlzdHMgb2YgZ2VuZSBvbnRvbG9neSB0ZXJtczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUyMTgwMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRlbiB5ZWFycyBvZiBuZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZyB0ZWNobm9sb2d5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz00MTgtMjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9uIHRoZSBtYWludGVuYW5jZSBvZiBnZW5ldGljIHZhcmlhdGlvbiBhbmQgYWRhcHRhdGlvbiB0byBlbnZpcm9ubWVudGFsIGNoYW5nZTogY29uc2lkZXJhdGlvbnMgZnJvbSBwb3B1bGF0aW9uIGdlbm9taWNzIGluIGZpc2hlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gRmlzaCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTg5OyAgY2l0YXRpb25fcGFnZXM9MjUxOS01NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmV4dC1nZW5lcmF0aW9uIHRyYW5zY3JpcHRvbWUgYXNzZW1ibHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTY3MS04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZhbHVhdGlvbiBvZiBkZSBub3ZvIHRyYW5zY3JpcHRvbWUgYXNzZW1ibGllcyBmcm9tIFJOQS1TZXEgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzZXNzaW5nIERlIE5vdm8gdHJhbnNjcmlwdG9tZSBhc3NlbWJseSBtZXRyaWNzIGZvciBjb25zaXN0ZW5jeSBhbmQgdXRpbGl0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9NDY1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U05QIGNhbGxpbmcgZnJvbSBSTkEtc2VxIGRhdGEgd2l0aG91dCBhIHJlZmVyZW5jZSBnZW5vbWU6IGlkZW50aWZpY2F0aW9uLCBxdWFudGlmaWNhdGlvbiwgZGlmZmVyZW50aWFsIGFuYWx5c2lzIGFuZCBpbXBhY3Qgb24gdGhlIHByb3RlaW4gc2VxdWVuY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9ZTE0OC1lMTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PZiBjb250aWdzIGFuZCBxdWFnbWlyZXM6IG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIHBpdGZhbGxzIGFzc29jaWF0ZWQgd2l0aCB0cmFuc2NyaXB0b21pYyBzdHVkaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEVjb2wuIFJlc291cjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTU1MS01ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJpbmNpcGxlcyBvZiB0cmFuc2NyaXB0b21lIGFuYWx5c2lzIGFuZCBnZW5lIGV4cHJlc3Npb24gcXVhbnRpZmljYXRpb246IGFuIFJOQS1zZXEgdHV0b3JpYWw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gRWNvbC4gUmVzb3VyOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9NTU5LTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFsbGVuZ2VzIGFuZCBzdHJhdGVnaWVzIGluIHRyYW5zY3JpcHRvbWUgYXNzZW1ibHkgYW5kIGRpZmZlcmVudGlhbCBnZW5lIGV4cHJlc3Npb24gcXVhbnRpZmljYXRpb24uIEEgY29tcHJlaGVuc2l2ZSBpbiBzaWxpY28gYXNzZXNzbWVudCBvZiBSTkHigJBzZXEgZXhwZXJpbWVudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gRWNvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTYyMC0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGluZWFnZS1zcGVjaWZpYyBsb3NzIGFuZCBkaXZlcmdlbmNlIG9mIGZ1bmN0aW9uYWxseSBsaW5rZWQgZ2VuZXMgaW4gZXVrYXJ5b3RlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTk3OyAgY2l0YXRpb25fcGFnZXM9MTEzMTktMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnNlcnZhdGlvbiBhbmQgZGl2ZXJnZW5jZSBvZiBnZW5lIGZhbWlsaWVzIGVuY29kaW5nIGNvbXBvbmVudHMgb2YgaW5uYXRlIGltbXVuZSByZXNwb25zZSBzeXN0ZW1zIGluIHplYnJhZmlzaDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIG9mIHRoZSBHdWxmIHBpcGVmaXNoIGVuYWJsZXMgdW5kZXJzdGFuZGluZyBvZiBldm9sdXRpb25hcnkgaW5ub3ZhdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MjU4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaCB0aHJvdWdocHV0IHNlcXVlbmNpbmcgbWV0aG9kcyBhbmQgYW5hbHlzaXMgZm9yIG1pY3JvYmlvbWUgcmVzZWFyY2g7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIE1pY3JvYmlvbC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT05NTsgIGNpdGF0aW9uX3BhZ2VzPTQwMS0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UEFSU0VTOiBhIHBpcGVsaW5lIGZvciBhbmFseXNpcyBvZiBSTkEtU2VxIGV4b2dlbm91cyBzZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJJQ29COyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fcGFnZXM9MTk2LTIwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGVmaW5pbmcgdGhlIHJvbGUgb2YgZXNzZW50aWFsIGdlbmVzIGluIGh1bWFuIGRpc2Vhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1lMjczNjgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ERUcgMTAsIGFuIHVwZGF0ZSBvZiB0aGUgZGF0YWJhc2Ugb2YgZXNzZW50aWFsIGdlbmVzIHRoYXQgaW5jbHVkZXMgYm90aCBwcm90ZWluLWNvZGluZyBnZW5lcyBhbmQgbm9uY29kaW5nIGdlbm9taWMgZWxlbWVudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9RDU3NC1ENTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGludmlhYmlsaXR5IGlzIGEgbWFqb3IgZHJpdmVyIG9mIHR5cGXigJBJSUkgc3Vydml2b3JzaGlwIGluIGV4cGVyaW1lbnRhbCBmYW1pbGllcyBvZiBhIGhpZ2hseSBmZWN1bmQgbWFyaW5lIGJpdmFsdmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gRWNvbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbiBvZiBob3VzZWtlZXBpbmcgZ2VuZXMgaW4gemVicmFmaXNoOiBtYWxlLWZlbWFsZSBkaWZmZXJlbmNlcyBhbmQgZWZmZWN0cyBvZiB0aXNzdWUgdHlwZSwgZGV2ZWxvcG1lbnRhbCBzdGFnZSBhbmQgY2hlbWljYWwgdHJlYXRtZW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0xIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZpZGVuY2UgZnJvbSBtaXRvY2hvbmRyaWFsIGdlbm9taWNzIHN1cHBvcnRzIHRoZSBsb3dlciBNZXNvem9pYyBvZiBTb3V0aCBBc2lhIGFzIHRoZSB0aW1lIGFuZCBwbGFjZSBvZiBiYXNhbCBkaXZlcmdlbmNlIG9mIGN5cHJpbmlmb3JtIGZpc2hlcyAoQWN0aW5vcHRlcnlnaWk6IE9zdGFyaW9waHlzaSk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVpvb2wuIEouIExpbm5lYW4gU29jOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE2MTsgIGNpdGF0aW9uX3BhZ2VzPTYzMy02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHplYnJhZmlzaCBnZW5vbWUgaW4gY29udGV4dDogb2hub2xvZ3MgZ29uZSBtaXNzaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBFeHAuIFpvb2wuIEIgTW9sLiBEZXYuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzA4OyAgY2l0YXRpb25fcGFnZXM9NTYzLTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Mb3NzIG9mIGR1cGxpY2F0ZSBnZW5lIGV4cHJlc3Npb24gYWZ0ZXIgcG9seXBsb2lkaXNhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTc3OyAgY2l0YXRpb25fdm9sdW1lPTI2NTsgIGNpdGF0aW9uX3BhZ2VzPTI1OC02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGdlbmV0aWMgYXNwZWN0cyBvZiB0ZXRyYXBsb2lkeSBpbiB0aGUgY29tbW9uIGNhcnAgQ3lwcmludXMgY2FycGlvOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFBoeWxvZ2VuZXQuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTU5LTY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXIsIE5ldyBZb3JrLCBOWTsgIGNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBieSBHZW5lIER1cGxpY2F0aW9uOyAgY2l0YXRpb25feWVhcj0xOTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcmFpbmJvdyB0cm91dCBnZW5vbWUgcHJvdmlkZXMgbm92ZWwgaW5zaWdodHMgaW50byBldm9sdXRpb24gYWZ0ZXIgd2hvbGUtZ2Vub21lIGR1cGxpY2F0aW9uIGluIHZlcnRlYnJhdGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIENvbW11bjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MzY1NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBldm9sdXRpb24gaW4gdGhlIGZpc2ggZmFtaWx5IHNhbG1vbmlkYWU6IGdlbmVyYXRpb24gb2YgYSBicm9vayBjaGFyciBnZW5ldGljIG1hcCBhbmQgY29tcGFyaXNvbnMgYW1vbmcgY2hhcnJzIChBcmN0aWMgY2hhcnIgYW5kIGJyb29rIGNoYXJyKSB3aXRoIHJhaW5ib3cgdHJvdXQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTEtMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNS8xLzExLzI0MTUxNDE1L2RzeDAzNC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgQ29tcGFyYXRpdmUgdHJhbnNjcmlwdG9taWNzIGNhbiBub3cgYmUgY29uZHVjdGVkIG9uIG9yZ2FuaXNtcyBpbiBuYXR1cmFsIHNldHRpbmdzLCB3aGljaCBoYXMgZ3JlYXRseSBlbmhhbmNlZCB1bmRlcnN0YW5kaW5nIG9mIGdlbm9tZeKAk2Vudmlyb25tZW50IGludCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDEwNjMxMyZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGS5FCS9Z6zEwd0vVadpKlSFL8b4aw==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f11%2f4106313" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f11%2f4106313" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=107546" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&amp;Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/1">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="112481575">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112481575">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112481577">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112481577">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112481586">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112481586">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112481598">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112481598">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112481622">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112481622">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112481641">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112481641">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112481643">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112481643">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112481649">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112481649">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/1/1/4107259">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/1/25/4161430">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Comparative transcriptomics of cyprinid minnows and carp in a common wild setting: a resource for ecological genomics in freshwater communities
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Trevor J Krabbenhoft</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biology and Museum of Southwestern Biology, University of New Mexico, Albuquerque, NM 87131, USA</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx034-cor1" data-open="dsx034-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. 716-645-2363. Fax. 716-645-2975. Email: <a href="mailto:krabbent.j@gmail.com" target="_blank">krabbent.j@gmail.com</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Trevor+J+Krabbenhoft">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Krabbenhoft T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Krabbenhoft T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Trevor J Krabbenhoft</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Thomas F Turner</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biology and Museum of Southwestern Biology, University of New Mexico, Albuquerque, NM 87131, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Thomas+F+Turner">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Turner T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Turner T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Thomas F Turner</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx034-FM1"><div class="fn"><div class="footnote-content">Present address: Department of Biological Sciences and Research and Education in eNergy, Environment and Water Program (RENEW), University at Buffalo, Buffalo, NY 14260-1300, USA  Edited by Prof. Masahira Hattori</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 1, 1 February 2018, Pages 11–23, <a href='https://doi.org/10.1093/dnares/dsx034'>https://doi.org/10.1093/dnares/dsx034</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">06 September 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">18 January 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">12 August 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4106313" href="/dnaresearch/article-pdf/25/1/11/24151415/dsx034.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Trevor J Krabbenhoft, Thomas F Turner;  Comparative transcriptomics of cyprinid minnows and carp in a common wild setting: a resource for ecological genomics in freshwater communities, <em>DNA Research</em>, Volume 25, Issue 1, 1 February 2018, Pages 11–23, <a href='https://doi.org/10.1093/dnares/dsx034'>https://doi.org/10.1093/dnares/dsx034</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4106313?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4106313?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4106313?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4106313?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4106313?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4106313?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4106313?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Comparative%20transcriptomics%20of%20cyprinid%20minnows%20and%20carp%20in%20a%20common%20wild%20setting%3A%20a%20resource%20for%20ecological%20genomics%20in%20freshwater%20communities&amp;publicationDate=2017-09-06&amp;volumeNum=25&amp;issueNum=1&amp;author=Krabbenhoft%2C%20Trevor%20J%3B%20Turner%2C%20Thomas%20F&amp;startPage=11&amp;endPage=23&amp;contentId=10.1093%2Fdnares%2Fdsx034&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f11%2f4106313" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=112481575 id="112481575" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Comparative transcriptomics can now be conducted on organisms in natural settings, which has greatly enhanced understanding of genome–environment interactions. Here, we demonstrate the utility and potential pitfalls of comparative transcriptomics of wild organisms, with an example from three cyprinid fish species (Teleostei:Cypriniformes). We present extensively filtered and annotated transcriptome assemblies that provide a valuable resource for studies of genome evolution (e.g. polyploidy), ecological and morphological diversification, speciation, and shared and unique responses to environmental variation in cyprinid fishes. Our results and analyses address the following points: (i) ‘essential developmental genes’ are shown to be ubiquitously expressed in a diverse suite of tissues across later ontogenetic stages (i.e. juveniles and adults), making these genes are useful for assessing the quality of transcriptome assemblies, (ii) the influence of microbiomes and other exogenous DNA, (iii) potentially novel, species-specific genes, and (iv) genomic rearrangements (e.g. whole genome duplication). The data we present provide a resource for future comparative work in cypriniform fishes and other taxa across a variety of sub-disciplines, including stress response, morphological diversification, community ecology, ecotoxicology, and climate change.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="RNA-seq">RNA-seq</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;essential genes&quot;">essential genes</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Cyprinus carpio&quot;"><em>Cyprinus carpio</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="carp">carp</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene silencing&quot;">gene silencing</a></div>                            <h2 scrollto-destination=112481577 id="112481577" class="section-title" >1. Introduction</h2>
<p>High-throughput sequencing has dramatically accelerated the pace of genomic research.<a reveal-id="dsx034-B1" data-open="dsx034-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsx034-B2" data-open="dsx034-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> While once restricted to model species in laboratory settings, genomic methods are being widely applied to non-model species in nature,<a reveal-id="dsx034-B3 dsx034-B4 dsx034-B5 dsx034-B6 dsx034-B7 dsx034-B8" data-open="dsx034-B3 dsx034-B4 dsx034-B5 dsx034-B6 dsx034-B7 dsx034-B8" class="link link-ref link-reveal xref-bibr"><sup>3–8</sup></a> rapidly illuminating the black box of the genome and giving rise to the field of ecological genomics.<a reveal-id="dsx034-B9 dsx034-B10 dsx034-B11" data-open="dsx034-B9 dsx034-B10 dsx034-B11" class="link link-ref link-reveal xref-bibr"><sup>9–11</sup></a> Reduced sequencing costs have made it feasible to study transcriptomes of co-occurring species in a community ecology context (i.e. ‘community transcriptomics’),<a reveal-id="dsx034-B12" data-open="dsx034-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> as well as comparative studies of transcriptome evolution across diverse clades (i.e. ‘comparative transcriptomics’).<a reveal-id="dsx034-B13 dsx034-B14 dsx034-B15 dsx034-B16" data-open="dsx034-B13 dsx034-B14 dsx034-B15 dsx034-B16" class="link link-ref link-reveal xref-bibr"><sup>13–16</sup></a> While genomic data from model species can be informative for the biology of related organisms, not all species are the same in terms of their ecology, genetics, and morphology. For example, research on the zebrafish (<em>Danio rerio</em>, family Cyprinidae) can be relevant for closely related species, but cannot explain the tremendous ecological and morphological diversity in this clade, as studies of single species are insufficient for understanding dynamic interactions among species and their respective genomes in a macroevolutionary context. In order to understand the causes and consequences of those interactions, as well as the origin of ecological novelty (e.g. new genes<a reveal-id="dsx034-B17" data-open="dsx034-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a>), we must examine genomes across species that reflect that diversity. Recent studies across a diverse suite of teleost fish lineages have focused on functionally-important genetic variation in non-model species in nature.<a reveal-id="dsx034-B18 dsx034-B19 dsx034-B20 dsx034-B21 dsx034-B22" data-open="dsx034-B18 dsx034-B19 dsx034-B20 dsx034-B21 dsx034-B22" class="link link-ref link-reveal xref-bibr"><sup>18–22</sup></a></p><p>Transcriptomics of species in the wild has enormous potential to advance our understanding of mechanisms underlying molecular adaptation, evolutionary diversification, ecotoxicology, and community ecology.<a reveal-id="dsx034-B4" data-open="dsx034-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx034-B23 dsx034-B24 dsx034-B25 dsx034-B26" data-open="dsx034-B23 dsx034-B24 dsx034-B25 dsx034-B26" class="link link-ref link-reveal xref-bibr"><sup>23–26</sup></a> In this context, several important questions arise. For example, What are the proximate and ultimate mechanisms underlying phylogenetic, ecological, and morphological divergence? How have ancestral genomes been molded by divergent natural selection and other evolutionary forces into myriad forms that exist today? How does genomic architecture constrain or promote diversification? How important are genome duplication events in adaptive radiations? What role do genomes play in underlying the ecological dynamics of community assembly (e.g. competition, abundance, spatial and temporal dynamics, physiological constraint, etc.)? A necessary first step in addressing these questions is the generation of databases reflecting the genomic or transcriptomic variation among species, which we provide in the current study.</p><p>Here we present transcriptomic resources for three members of the freshwater fish family Cyprinidae (Teleostei: Cypriniformes), one of the most speciose vertebrate clades, with over 2,000 species.<a reveal-id="dsx034-B27" data-open="dsx034-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> In addition to remarkable species diversity, the clade includes extensive ecological, genetic, and morphological diversity. Cyprinid fishes (minnows and carp) comprise an important component of freshwater fish communities throughout North America, Asia, Europe and Africa.<a reveal-id="dsx034-B28" data-open="dsx034-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> They are often the dominant fish taxa in numerical abundance and biomass and play an important functional role in aquatic ecosystems.<a reveal-id="dsx034-B29 dsx034-B30 dsx034-B31" data-open="dsx034-B29 dsx034-B30 dsx034-B31" class="link link-ref link-reveal xref-bibr"><sup>29–31</sup></a></p><p>The ecological and taxonomic diversity of cyprinids is particularly interesting in light of the history of genome evolution in this clade. Cyprinids were part of the radiation that occurred after the teleost-specific genome duplication event, known as the ‘3R hypothesis’,<a reveal-id="dsx034-B32" data-open="dsx034-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a><sup>,</sup><a reveal-id="dsx034-B33" data-open="dsx034-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> that preceded and perhaps facilitated the diversification of teleost fishes.<a reveal-id="dsx034-B34" data-open="dsx034-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> In addition, several cyprinid lineages have independently undergone additional rounds of genome duplications.<a reveal-id="dsx034-B35 dsx034-B36 dsx034-B37 dsx034-B38" data-open="dsx034-B35 dsx034-B36 dsx034-B37 dsx034-B38" class="link link-ref link-reveal xref-bibr"><sup>35–38</sup></a> For example, the common carp (<em>Cyprinus carpio</em>) lineage had a fourth round of genome duplication approximately 5.6–11.3<a reveal-id="dsx034-B39" data-open="dsx034-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> or 8.2<a reveal-id="dsx034-B40" data-open="dsx034-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> million years ago (Ma), which we refer to as ‘Cc4R’. Pairs of genes arising from whole genome duplication, referred to as ‘Ohnologs’, were theoretically present for all genes immediately following the Cc4R genome duplication. Varying levels of subsequent gene-silencing and ‘re-diploidization’ have since occurred in polyploid lineages making cyprinids ideal for comparative studies of genome evolution.<a reveal-id="dsx034-B41" data-open="dsx034-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a><sup>,</sup><a reveal-id="dsx034-B42" data-open="dsx034-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a></p><p>Despite the ecological importance of cyprinids in freshwater systems worldwide and dynamic lineage-specific patterns of genomic expansions and contractions, most species have little or no genomic resources available for investigating their molecular ecology or genome evolution. Three notable exceptions are zebrafish (<em>Danio rerio</em>), fathead minnow (<em>Pimephales promelas</em>), and common carp (<em>Cyprinus carpio</em>). The family includes zebrafish (<em>Danio rerio</em>),<a reveal-id="dsx034-B43 dsx034-B44 dsx034-B45" data-open="dsx034-B43 dsx034-B44 dsx034-B45" class="link link-ref link-reveal xref-bibr"><sup>43–45</sup></a> a model species with a comprehensively-annotated genome.<a reveal-id="dsx034-B46" data-open="dsx034-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> Zebrafish is an important model organism in developmental biology and disease research,<a reveal-id="dsx034-B43 dsx034-B44 dsx034-B45" data-open="dsx034-B43 dsx034-B44 dsx034-B45" class="link link-ref link-reveal xref-bibr"><sup>43–45</sup></a> due to its semitransparent embryos and ease of laboratory culture, as well as its comprehensively annotated genome.<a reveal-id="dsx034-B46" data-open="dsx034-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> Fathead minnow is widely used as an indicator species in ecotoxicology studies for which microarrays have been developed<a reveal-id="dsx034-B47" data-open="dsx034-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a><sup>,</sup><a reveal-id="dsx034-B48" data-open="dsx034-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> and a draft genome sequence is now available.<a reveal-id="dsx034-B49" data-open="dsx034-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> Common carp is an important food fish, especially in Asia, and is produced extensively in aquaculture.<a reveal-id="dsx034-B50" data-open="dsx034-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> The carp transcriptome has been studied elsewhere<a reveal-id="dsx034-B39" data-open="dsx034-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a><sup>,</sup><a reveal-id="dsx034-B51" data-open="dsx034-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a><sup>,</sup><a reveal-id="dsx034-B52" data-open="dsx034-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> and recently a draft genome sequence was published.<a reveal-id="dsx034-B40" data-open="dsx034-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a></p><p>We used the extensive zebrafish genomic resources available to annotate transcriptomes of three evolutionarily related, but non-model species that co-occur in parts of the west-central United States: <em>Cyprinella lutrensis</em> (red shiner), <em>Platygobio gracilis</em> (flathead chub) and <em>Cyprinus carpio</em> (common carp). These species were selected to reflect phylogenetic breadth, but also because their distributions overlap and occupy identical dryland river habitats (i.e. the Rio Grande, New Mexico), where they are exposed to similar biotic and abiotic conditions. <em>Cyprinus carpio</em> is native to Asia and Europe, but was introduced into North America, perhaps as early as 1831,<a reveal-id="dsx034-B53" data-open="dsx034-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> and enthusiastically stocked throughout the US thereafter as a food fish, including New Mexico as early as 1889.<a reveal-id="dsx034-B54" data-open="dsx034-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a><sup>,</sup><em>Cyprinella lutrensis</em> and <em>Platygobio gracilis</em> are both native to central and western North America, from the Mississippi River basin to the Rio Grande in New Mexico. Both <em>C. lutrensis</em> and <em>C. carpio</em> are highly tolerant of a wide range of environmental conditions and are highly invasive in areas outside of their natural range,<a reveal-id="dsx034-B55" data-open="dsx034-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a><sup>,</sup><a reveal-id="dsx034-B56" data-open="dsx034-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> whereas <em>Platygobio gracilis</em> is sensitive to environmental disturbance and imperiled or declining in several parts of its range.<a reveal-id="dsx034-B57" data-open="dsx034-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a></p><p>Transcriptomes of <em>Cyprinella lutrensis</em> and <em>Platygobio gracilis</em> have not been published to our knowledge, whereas genomic and transcriptomic data are available for <em>Cyprinus carpio</em>.<a reveal-id="dsx034-B39" data-open="dsx034-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a><sup>,</sup><a reveal-id="dsx034-B40" data-open="dsx034-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a><sup>,</sup><a reveal-id="dsx034-B51" data-open="dsx034-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a><sup>,</sup><em>Cyprinella lutrensis</em> and <em>Platygobio gracilis</em> are diploid (2<em>n</em> = 50), while <em>Cyprinus carpio</em> is allotetraploid 2<em>n</em> = 100,<a reveal-id="dsx034-B37" data-open="dsx034-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> with some duplicated genes silenced after a lineage-specific whole genome duplication (i.e. Cc4R). Our aims in this study were to: (i) succinctly summarize and compare genes and functional annotation information obtained from various databases; (ii) test whether <em>Cyprinus carpio</em> expresses additional copies of particular genes compared to the two diploid species (<em>Cyprinella lutrensis</em> and <em>Platygobio gracilis</em>); (iii) identify potentially novel genes present in the three cyprinids that may underlie their unique ecological and morphological novelty and (iv) to assess evolutionary conservation of essential genes for development.</p><p>In zebrafish, 307 genes are known to be essential for development. Knockout mutations in these genes are embryonic lethal according to experiments by Amsterdam <em>et al.</em><a reveal-id="dsx034-B58" data-open="dsx034-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> with subsequent revisions by Chen <em>et al.</em><a reveal-id="dsx034-B59" data-open="dsx034-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> and updates to the ENSEMBL database.<a reveal-id="dsx034-B60" data-open="dsx034-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> These genes are highly conserved across extremely deep phylogenetic splits (e.g. yeast, fly, zebrafish, and human) due to their essential roles in development.<a reveal-id="dsx034-B58" data-open="dsx034-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> Despite their importance, essential genes have not been studied in the context of comparative molecular ecology or ecological genomics of co-occurring species. Using transcriptome data presented in this study, we assessed the evolutionary conservation of the 307 zebrafish essential genes across four cyprinid lineages. We predicted that these genes would be highly conserved across all species, consistent with their critical functional roles, as compared to non-essential genes.<a reveal-id="dsx034-B61" data-open="dsx034-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> If this is the case, then differences among species should be found in non-essential genes, such as lineage- or species-specific genes. We also tested whether both copies of duplicated genes in <em>C. carpio</em> (i.e. Cc4R Ohnologs) were retained and expressed in duplicate or whether one copy was evolutionarily lost.<a reveal-id="dsx034-B38" data-open="dsx034-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a><sup>,</sup><a reveal-id="dsx034-B42" data-open="dsx034-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> One mechanism for the loss of Ohnologs is ‘pseudogenization’,<a reveal-id="dsx034-B62" data-open="dsx034-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> wherein a gene accumulates one or more internal stop codons that prevent formation of a functional protein product and thus becomes a pseudogene.<a reveal-id="dsx034-B63" data-open="dsx034-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> If having redundant copies of essential genes were important for survival (e.g. due to loss-of-function mutations in one copy), then evolutionary retention of duplicates would likely be favored in <em>C. carpio</em>. Conversely, if regulation of proper gene expression levels were important in the context of functional pathways, then duplicated essential genes would likely be silenced at roughly the same rate as non-essential genes (although regulatory changes could also fine-tune expression patterns). We tested these hypotheses using expression data for the three cyprinid transcriptomes as compared to zebrafish. These sequences will provide resources for more detailed studies of the evolution and functional constraint of these critical genes, particularly in the context of genome expansions and reductions.</p>                            <h2 scrollto-destination=112481586 id="112481586" class="section-title" >2. Materials and methods</h2>
<p>Fish (<em>n</em> = 3 per species) were collected with a seine on 6 July 2012 from a field site on the Rio Grande, approximately 40 km south of Socorro, New Mexico (33.690556°N, 106.993042°W). Whole fish samples (juveniles or non-spawning adults) were immediately frozen in liquid nitrogen and transported to the laboratory. Skin, gill, gut, and kidney tissues were dissected and removed from frozen fish (outer layers were only slightly thawed by the time dissection was completed; &lt;5 minutes total time), placed in TRIzol (Invitrogen), and mechanically homogenized. Total RNA was isolated using Purelink RNA Mini kits (Ambion) following manufacturer’s protocol, along with DNase treatment to reduce genomic DNA contamination. Purified total RNA was sent to the National Center for Genome Resources (Santa Fe, New Mexico, USA) for quantification, quality assessment, cDNA library preparation and sequencing. RNA integrity and purity was assessed with a Bioanalyzer 2100 instrument (Agilent Technologies). Thirty-six Illumina libraries were constructed (3 species × 4 tissues × 3 biological replicates) from the total RNA samples using Illumina TruSeq DNA prep kits according to the manufacturer’s protocol. Libraries were barcoded using standard six base pair Illumina oligonucleotides, and six libraries were pooled for each lane of Illumina HiSeq 2000 (V<a reveal-id="dsx034-B3" data-open="dsx034-B3" class="link link-ref link-reveal xref-bibr">3</a> chemistry) for a total of six lanes of 2 × 100 bp paired-end sequencing.</p>                            <h3 scrollto-destination=112481588 id="112481588" class="section-title" >2.1. Bioinformatics</h3>
<p>We used the bioinformatics pipeline outlined in <a reveal-id="dsx034-F1" data-open="dsx034-F1" class="link link-reveal link-table xref-fig">Figure 1</a> for analyzing transcriptomic data in three main steps: <em>de novo</em> assembly, gene annotation, and analysis of expression of duplicated genes. Adapters and barcode sequences were removed from raw reads, and reads were trimmed using T<span class="small-caps">rimmomatic</span><a reveal-id="dsx034-B64" data-open="dsx034-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> with parameter settings as follows: leading quality = 5; trailing quality = 5; minimum trimmed read length = 36). Reads were normalized <em>in silico</em> to maximum read coverage of 50×. Clipped and trimmed reads were assembled, <em>de novo</em>, for each species separately using T<span class="small-caps">rinity</span> version 2014-04-13,<a reveal-id="dsx034-B65" data-open="dsx034-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a><sup>,</sup><a reveal-id="dsx034-B66" data-open="dsx034-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> with minimum contig length set to 200 bp. Libraries were pooled within a species for <em>de novo</em> assembly, to maximize the number of genes included. T<span class="small-caps">rinity</span> assembles reads into contigs (‘T<span class="small-caps">rinity</span> transcripts’), places similar transcripts in groups loosely referred to as ‘genes’, and groups similar ‘genes’ into gene clusters. </p>                        <a id="112481590" scrollto-destination="112481590"></a>
<div id="dsx034-F1" data-id="dsx034-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/m_dsx034f1.png?Expires=2147483647&Signature=lEB0h1xLDgSNu~Ee2VbPPR5WEzY2LaFOtO0BTgeqdMZCnw4RJ53WM0ltzXJZMbe26ClUylZuFj9g2AKxv-kQGLI4r9p0Ma2g7OzR1VcFFGM021EE6nFWLf~A6QoFNbjcn009Ylyartu3qFSBUaUMWLHDGLO5o~3KYzT1C8kReS07u6GcLt2YCZjHNhs-tLwrGmMHWN8gDVDNyPHcEm3CaqWyOGFFbyMv91WVOgtkijHA3Ibtmq8jIpvyblo~XKgKtgHfXJm6QPCYU5mGWd1sYv9iIAZDTwzCpsEHiA940KF1tvjbPIxKyGrY3U7O6GvylnmN9i-bXOSbCLnj7q3kSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Flow diagram illustrating our bioinformatic pipeline. Analyses consist of three main steps: assembly, annotation, and analysis of gene silencing patterns. Databases queried and software packages used are listed." path-from-xml="dsx034f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481590/dsx034f1.tif" path-from-xml="dsx034f1.tif" target="_blank">View large</a><a section="112481590" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/dsx034f1.png?Expires=2147483647&Signature=gMcLlu5iLq3yA~3E6ydNpvT-kcmmxRPVzr~eTNnwSegJSAvA~cSc6XXygcbKEDOp029S5GAbICoXxs5eFCqFv2mm4SSANFokzXPqAkjHo4i9AsionozYoJCorzyP1~dnk9lilzVOEtL6-Z~8O5snDFAww27xD1BA17oEqMwveYnKOdCCFJ6nphLKp4VH09gAaCLAEFzj13hGKk3B94R~qQy0TSPkV0eSAWGvRdpV~g1P04BUGbOCq0ClvvxtTN~VEKg6wdM5iwmt3Nuc0C1ufCEqW5Fpf1FTtAH~mfZ6WCL5HnYscPnRdQ82Rf3yBErNgB88yu-eR52SsFSSNeACYw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481590&ar=4106313&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx034f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flow diagram illustrating our bioinformatic pipeline. Analyses consist of three main steps: assembly, annotation, and analysis of gene silencing patterns. Databases queried and software packages used are listed.</p></div></div><div content-id="dsx034-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/m_dsx034f1.png?Expires=2147483647&Signature=lEB0h1xLDgSNu~Ee2VbPPR5WEzY2LaFOtO0BTgeqdMZCnw4RJ53WM0ltzXJZMbe26ClUylZuFj9g2AKxv-kQGLI4r9p0Ma2g7OzR1VcFFGM021EE6nFWLf~A6QoFNbjcn009Ylyartu3qFSBUaUMWLHDGLO5o~3KYzT1C8kReS07u6GcLt2YCZjHNhs-tLwrGmMHWN8gDVDNyPHcEm3CaqWyOGFFbyMv91WVOgtkijHA3Ibtmq8jIpvyblo~XKgKtgHfXJm6QPCYU5mGWd1sYv9iIAZDTwzCpsEHiA940KF1tvjbPIxKyGrY3U7O6GvylnmN9i-bXOSbCLnj7q3kSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Flow diagram illustrating our bioinformatic pipeline. Analyses consist of three main steps: assembly, annotation, and analysis of gene silencing patterns. Databases queried and software packages used are listed." path-from-xml="dsx034f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481590/dsx034f1.tif" path-from-xml="dsx034f1.tif" target="_blank">View large</a><a section="112481590" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/dsx034f1.png?Expires=2147483647&Signature=gMcLlu5iLq3yA~3E6ydNpvT-kcmmxRPVzr~eTNnwSegJSAvA~cSc6XXygcbKEDOp029S5GAbICoXxs5eFCqFv2mm4SSANFokzXPqAkjHo4i9AsionozYoJCorzyP1~dnk9lilzVOEtL6-Z~8O5snDFAww27xD1BA17oEqMwveYnKOdCCFJ6nphLKp4VH09gAaCLAEFzj13hGKk3B94R~qQy0TSPkV0eSAWGvRdpV~g1P04BUGbOCq0ClvvxtTN~VEKg6wdM5iwmt3Nuc0C1ufCEqW5Fpf1FTtAH~mfZ6WCL5HnYscPnRdQ82Rf3yBErNgB88yu-eR52SsFSSNeACYw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481590&ar=4106313&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx034f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flow diagram illustrating our bioinformatic pipeline. Analyses consist of three main steps: assembly, annotation, and analysis of gene silencing patterns. Databases queried and software packages used are listed.</p></div></div><p>Putative protein coding genes were also identified by BLASTx searches of contigs against zebrafish (<em>Danio rerio</em>) peptide sequences (database build Zv9) obtained from Ensembl 78.<a reveal-id="dsx034-B60" data-open="dsx034-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> Significant BLAST hits were identified based on the following parameter settings: <em>E</em>-value &lt; 0.0001; gap open penalty = 11; gap extend = 1; wordsize = 3. After extensive testing, this parameter combination was found to give the optimal balance between finding matches for large numbers of contigs, while minimizing spurious hits. For most genes a 1–1 match was expected between zebrafish versus <em>Platygobio gracilis</em> or <em>Cyprinella lutrensis</em>, whereas zebrafish and <em>Cyprinus carpio</em> should have either 1-2 or 1-1 due to partial diploidy in carp. We used this expectation in determining the threshold <em>E</em>-value (i.e. <em>E</em> &lt; 0.0001 in this study) to use. In practice, more stringent <em>E</em>-value thresholds (e.g. <em>E</em> &lt; 1e−6) had very little effect on the number of significant BLAST hits.</p><p>Contigs with no significant BLAST hits against the zebrafish transcriptome were subjected to a series of stepwise BLASTn searches until significant hits were found (or not) in order to identify the possible sources of those sequences (e.g. microbiome<a reveal-id="dsx034-B7" data-open="dsx034-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a>) or to identify novel genes not present in the zebrafish genome. First, remaining contigs lacking significant hits against the zebrafish transcriptome were queried against the rRNA silva database (SSU Ref 119 NR99 and LSU Parc 119), which contains bacterial and eukaryotic rRNA sequences.<a reveal-id="dsx034-B67" data-open="dsx034-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> Contigs with still no significant BLAST hits were then queried against a database containing all nine additional teleost fish transcriptomes (Amazon molly, <em>Poecilia formosa</em>; cavefish, <em>Astyanax mexicanus</em>; cod, <em>Gadus morhua</em>; fugu, <em>Takifugu rubripes</em>; medaka, <em>Oryzias latipes</em>; platyfish, <em>Xiphophorus maculates</em>; stickleback, <em>Gasterosteus aculeatus</em>; tetraodon, <em>Tetraodon nigroviridis</em>; tilapia, <em>Oreochromis niloticus</em>) from Ensembl 78. Contigs with no BLAST hits at this point were then BLASTed against the zebrafish genome (Zv9) using the ‘Top Level’ sequences from Ensembl to identify possible genomic DNA contamination. Remaining contigs with no significant blast hits in any of these databases were piped to T<span class="small-caps">rans</span>D<span class="small-caps">ecoder</span><a reveal-id="dsx034-B66" data-open="dsx034-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> to identify open reading frames (ORFs) that represent potentially novel genes. Default parameter settings were used with T<span class="small-caps">rans</span>D<span class="small-caps">ecoder</span>. The software generates predicted peptide sequences for contigs with ORFs. Predicted peptide sequences for the contigs with ORFs but no BLAST hits to the aforementioned databases were queried (BLASTp; <em>E</em>-value &lt; 0.001) against the NCBI nr database. B<span class="small-caps">last</span>2GO version 3.0,<a reveal-id="dsx034-B68" data-open="dsx034-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> was used to identify top species hits for those predicted proteins with significant hits against nr. The remaining sequences with no hits to databases and no ORFs were discarded as likely non-protein coding, genomic DNA contamination with sufficient divergence from zebrafish to render genomic BLASTn searches ineffective.</p>                            <h3 scrollto-destination=112481593 id="112481593" class="section-title" >2.2. Genome duplication, diploidization and gene silencing</h3>
<p>Trimmed sequence reads were mapped to T<span class="small-caps">rinity</span> contigs using B<span class="small-caps">owtie</span>2 version 2.2.2.3<a reveal-id="dsx034-B69" data-open="dsx034-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a> and corresponding gene expression was quantified with RSEM version 1.2.13.<a reveal-id="dsx034-B70" data-open="dsx034-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a> Because RSEM is incompatible with indel, local, and discordant alignments, parameter settings were chosen to avoid these alignments. The following RSEM parameters were used: –sensitive; –dpad 0; –gbar 99999999; –mp 1,1 –np 1 –score-min L,0,-0.1; –no-mixed; –no-discordant. Normalized expression for T<span class="small-caps">rinity</span> genes was calculated by standardizing by total mapped reads across libraries and summed across alternate T<span class="small-caps">rinity</span> transcripts (isoforms) for each locus. Networks of co-expressed genes were identified for the three species using the WGCNA package in R.<a reveal-id="dsx034-B71" data-open="dsx034-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> In order to assess the expression of duplicated genes in <em>Cyprinus carpio</em> arising from the Cc4R duplication event, we quantified the number of T<span class="small-caps">rinity</span> genes present in each species relative to zebrafish genes, as well as their expression levels. We used an arbitrary threshold of ten sequence reads per gene per tissue, summed across all three individuals, for a given gene to be considered ‘expressed’ in a particular tissue. Note that we are comparing whether or not a gene is expressed beyond a certain threshold, as opposed to quantifying levels of expression (i.e. RNA-seq). This approach was aimed at reducing the influence of unique reads (e.g. sequencing artifacts). Most of the contigs excluded as a result were contigs represented only by singleton reads in one library.</p><p>For <em>C. carpio</em>, we tested whether certain functional classes of genes were preferentially expressed in duplicate (i.e. the case where neither ohnolog is silenced). For this analysis, we used P<span class="small-caps">anther</span><a reveal-id="dsx034-B72" data-open="dsx034-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> to test for statistical overrepresentation of GO-slim Biological Processes, with Bonferroni correction. The test genes consisted of the list of <em>C. carpio</em> ohnologs expressed in duplicate, while the list of all <em>C. carpio</em> genes present in the assembly was used as the reference set. GO terminology was based on the zebrafish database. Results of the overrepresentation analysis were visualized with R<span class="small-caps">evigo</span>.<a reveal-id="dsx034-B73" data-open="dsx034-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a></p>                            <h3 scrollto-destination=112481596 id="112481596" class="section-title" >2.3. Essential genes</h3>
<p>To test the hypothesis of evolutionary conservation of essential genes among cyprinid fishes, we used zebrafish genes present in the Online Gene Essentiality Database OGEE;<a reveal-id="dsx034-B59" data-open="dsx034-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> and identified orthologs in the three transcriptomes from BLASTx searches described above. Of the 307 essential genes in zebrafish,<a reveal-id="dsx034-B58" data-open="dsx034-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a><sup>,</sup><a reveal-id="dsx034-B59" data-open="dsx034-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> one (ENSDARG00000038423) has been retired from ENSEMBL and one (ENSDARG00000045605) is an unprocessed pseudogene with no protein product. We searched for the remaining 305 genes in the three transcriptome assemblies to assess their conservation across cyprinids.<a reveal-id="dsx034-B58" data-open="dsx034-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a></p>                            <h2 scrollto-destination=112481598 id="112481598" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=112481599 id="112481599" class="section-title" >3.1. Sequencing and transcriptome assemblies</h3>
<p>Six lanes of Illumina sequencing produced more than 1.2 billion paired-end reads, including 420.5-, 413.9-, and 385.3-million sequences in <em>Cyprinus carpio</em>, <em>Cyprinella lutrensis</em>, and <em>Platygobio gracilis</em>, respectively. <em>De novo</em> assembly resulted in high quality transcriptomes for all three species (<a reveal-id="dsx034-T1" data-open="dsx034-T1" class="link link-reveal link-table xref-fig">Table 1</a>). The <em>C. carpio</em> assembly had the largest number of contigs (‘T<span class="small-caps">rinity</span> transcripts’) and genes (‘T<span class="small-caps">rinity</span> genes’), while <em>P. gracilis</em> had the fewest. In contrast, metrics for contig length (N25, N50, N75, median contig length, average contig length) were all longer in <em>P. gracilis</em> than the other two species (<a reveal-id="dsx034-T1" data-open="dsx034-T1" class="link link-reveal link-table xref-fig">Table 1</a>; <a reveal-id="dsx034-F2" data-open="dsx034-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). Overall, the <em>P. gracilis</em> transcriptome assembly was more complete despite fewer raw sequence reads. T<span class="small-caps">rans</span>D<span class="small-caps">ecoder</span> predicted ORFs in about half of all T<span class="small-caps">rinity</span> contigs (not shown), with the remainder comprised mainly of genomic DNA contamination that was filtered out of the final dataset. The N50 of predicted ORFs was only slightly shorter in the three species (i.e. 1,299–1,572 bp) than in zebrafish (CDS N50 = 2,037 bp), and similar to the recently published draft <em>C. carpio</em> genome 1,487 bp.<a reveal-id="dsx034-B40" data-open="dsx034-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> Removal of microbiome and genomic DNA contamination from the final assembly resulted in fewer, but longer contigs (see filtering of the final dataset, below), and an overall higher-quality assembly. <div content-id="dsx034-T1" class="table-modal"><div class="table-wrap"><div id="dsx034-T1" data-id="dsx034-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p><em>De novo</em> transcriptome assembly results. Zebrafish (<em>Danio rerio</em>) data is included as an example of a well-assembled and complete transcriptome based primarily on Sanger sequencing</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th><em>Cyprinus carpio</em> </th><th><em>Cyprinella lutrensis</em> </th><th><em>Platygobio gracilis</em> </th><th><em>Danio rerio</em> </th></tr></thead><tbody><tr><td>Trinity ‘genes’ (=Clusters of contigs) </td><td>309,921 </td><td>255,863 </td><td>180,130 </td><td>30,651 </td></tr><tr><td>Trinity ‘transcripts’ (=Assembled contigs) </td><td>440,696 </td><td>382,504 </td><td>262,969 </td><td>43,153 </td></tr><tr><td>GC content </td><td>42.45 </td><td>43.25 </td><td>42.67 </td><td>49.60 </td></tr><tr><td>N25 (bp) </td><td>3,327 </td><td>3,069 </td><td>3,644 </td><td>3,465 </td></tr><tr><td>N50 </td><td>1,841 </td><td>1,666 </td><td>1,972 </td><td>2,037 </td></tr><tr><td>N75 </td><td>704 </td><td>679 </td><td>788 </td><td>1,179 </td></tr><tr><td>Median contig length </td><td>418 </td><td>439 </td><td>450 </td><td>1,080 </td></tr><tr><td>Average contig length </td><td>907 </td><td>886 </td><td>978 </td><td>1,501 </td></tr><tr><td>Total assembled bases </td><td>399,790,412 </td><td>339,160,955 </td><td>257,217,466 </td><td>64,757,328 </td></tr></tbody></table></div><div content-id="dsx034-T1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th><em>Cyprinus carpio</em> </th><th><em>Cyprinella lutrensis</em> </th><th><em>Platygobio gracilis</em> </th><th><em>Danio rerio</em> </th></tr></thead><tbody><tr><td>Trinity ‘genes’ (=Clusters of contigs) </td><td>309,921 </td><td>255,863 </td><td>180,130 </td><td>30,651 </td></tr><tr><td>Trinity ‘transcripts’ (=Assembled contigs) </td><td>440,696 </td><td>382,504 </td><td>262,969 </td><td>43,153 </td></tr><tr><td>GC content </td><td>42.45 </td><td>43.25 </td><td>42.67 </td><td>49.60 </td></tr><tr><td>N25 (bp) </td><td>3,327 </td><td>3,069 </td><td>3,644 </td><td>3,465 </td></tr><tr><td>N50 </td><td>1,841 </td><td>1,666 </td><td>1,972 </td><td>2,037 </td></tr><tr><td>N75 </td><td>704 </td><td>679 </td><td>788 </td><td>1,179 </td></tr><tr><td>Median contig length </td><td>418 </td><td>439 </td><td>450 </td><td>1,080 </td></tr><tr><td>Average contig length </td><td>907 </td><td>886 </td><td>978 </td><td>1,501 </td></tr><tr><td>Total assembled bases </td><td>399,790,412 </td><td>339,160,955 </td><td>257,217,466 </td><td>64,757,328 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsx034-T1" data-id="dsx034-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p><em>De novo</em> transcriptome assembly results. Zebrafish (<em>Danio rerio</em>) data is included as an example of a well-assembled and complete transcriptome based primarily on Sanger sequencing</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th><em>Cyprinus carpio</em> </th><th><em>Cyprinella lutrensis</em> </th><th><em>Platygobio gracilis</em> </th><th><em>Danio rerio</em> </th></tr></thead><tbody><tr><td>Trinity ‘genes’ (=Clusters of contigs) </td><td>309,921 </td><td>255,863 </td><td>180,130 </td><td>30,651 </td></tr><tr><td>Trinity ‘transcripts’ (=Assembled contigs) </td><td>440,696 </td><td>382,504 </td><td>262,969 </td><td>43,153 </td></tr><tr><td>GC content </td><td>42.45 </td><td>43.25 </td><td>42.67 </td><td>49.60 </td></tr><tr><td>N25 (bp) </td><td>3,327 </td><td>3,069 </td><td>3,644 </td><td>3,465 </td></tr><tr><td>N50 </td><td>1,841 </td><td>1,666 </td><td>1,972 </td><td>2,037 </td></tr><tr><td>N75 </td><td>704 </td><td>679 </td><td>788 </td><td>1,179 </td></tr><tr><td>Median contig length </td><td>418 </td><td>439 </td><td>450 </td><td>1,080 </td></tr><tr><td>Average contig length </td><td>907 </td><td>886 </td><td>978 </td><td>1,501 </td></tr><tr><td>Total assembled bases </td><td>399,790,412 </td><td>339,160,955 </td><td>257,217,466 </td><td>64,757,328 </td></tr></tbody></table></div><div content-id="dsx034-T1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th><em>Cyprinus carpio</em> </th><th><em>Cyprinella lutrensis</em> </th><th><em>Platygobio gracilis</em> </th><th><em>Danio rerio</em> </th></tr></thead><tbody><tr><td>Trinity ‘genes’ (=Clusters of contigs) </td><td>309,921 </td><td>255,863 </td><td>180,130 </td><td>30,651 </td></tr><tr><td>Trinity ‘transcripts’ (=Assembled contigs) </td><td>440,696 </td><td>382,504 </td><td>262,969 </td><td>43,153 </td></tr><tr><td>GC content </td><td>42.45 </td><td>43.25 </td><td>42.67 </td><td>49.60 </td></tr><tr><td>N25 (bp) </td><td>3,327 </td><td>3,069 </td><td>3,644 </td><td>3,465 </td></tr><tr><td>N50 </td><td>1,841 </td><td>1,666 </td><td>1,972 </td><td>2,037 </td></tr><tr><td>N75 </td><td>704 </td><td>679 </td><td>788 </td><td>1,179 </td></tr><tr><td>Median contig length </td><td>418 </td><td>439 </td><td>450 </td><td>1,080 </td></tr><tr><td>Average contig length </td><td>907 </td><td>886 </td><td>978 </td><td>1,501 </td></tr><tr><td>Total assembled bases </td><td>399,790,412 </td><td>339,160,955 </td><td>257,217,466 </td><td>64,757,328 </td></tr></tbody></table></div></div></p>                        <a id="112481601" scrollto-destination="112481601"></a>
<div id="dsx034-F2" data-id="dsx034-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/m_dsx034f2.png?Expires=2147483647&Signature=vsykyW6yKRglrRLQarybfd26bbmvC1N4UCQGwqmcfd9A8-MpUHEaLhnxL0fUJCzB4ZF~WR4mQbcIi~EKe4yJ8t-dolHYIWHVxsYRcnB8pLhuJ~JaIWbe2btTaDwyEHU3mHYiVOu~HCgWeK2u0uKf2b5d-BpR7FB06O90VTrWKjBI-Jgp0dk8z1ggxHdRXTttxDhap3WYKYmBkXurTJboTkPB2pidK2kpid7q~oWqZIMbeaHVra7jTQ9inzCt9JwdRzj4Ep9rucjkwCqaYuGqTfDuD25VfAZxbTNGtImC13KRRspnLxecuXtYneyJaivQULdNXSodT80AO7ncHDPkfg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Contig length histogram of three cyprinids in this study and zebrafish, Danio rerio. By leveraging high throughput sequencing and bioinformatic filtering, we were able to generate high quality transcriptomes at a fraction of the cost and research effort used for zebrafish. As expected, de novo Trinity assemblies resulted in proportionally fewer contigs longer than 1000 bp, as compared to those of a well-assembled transcriptome, zebrafish (Danio rerio). However, note that we only used canonical transcripts for zebrafish and not the shorter isoforms, which skews the distribution toward longer transcripts for that species." path-from-xml="dsx034f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481601/dsx034f2.tif" path-from-xml="dsx034f2.tif" target="_blank">View large</a><a section="112481601" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/dsx034f2.png?Expires=2147483647&Signature=MKdcYg51jbVUUiiLUhZZ1egtcRWr92TwLekWvNjM3mQbEjLfEs8BBAwtpMJi0dy4CvAB0UrQAF9COxzsMw-r1XDk~o77Dv0FlfOcrCdG4jCGjmMhfCAn8yITPbXECyAYLEVMz4seeWW489Cd5T-MM-oxTDtI60Nevp5heOkAGbIfVzUX5cXOn8UWRTowAp~0vvZzx1-vcnNxbGJjXTd5gBH1N4ARaCY2LcLXVRMWdfgmlbwu8xqCEMvmEu2MgSHCTyJxmrUe7Bx4P2fQySEOvOloLZwv1WlA29unyZXe-zEJmvAb3nIEQ~-GzOeiAEdm6oiLRt9bOcNkpaJNiR1x4g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481601&ar=4106313&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx034f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Contig length histogram of three cyprinids in this study and zebrafish, <em>Danio rerio</em>. By leveraging high throughput sequencing and bioinformatic filtering, we were able to generate high quality transcriptomes at a fraction of the cost and research effort used for zebrafish. As expected, <em>de novo</em> T<span class="small-caps">rinity</span> assemblies resulted in proportionally fewer contigs longer than 1000 bp, as compared to those of a well-assembled transcriptome, zebrafish (<em>Danio rerio</em>). However, note that we only used canonical transcripts for zebrafish and not the shorter isoforms, which skews the distribution toward longer transcripts for that species.</p></div></div><div content-id="dsx034-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/m_dsx034f2.png?Expires=2147483647&Signature=vsykyW6yKRglrRLQarybfd26bbmvC1N4UCQGwqmcfd9A8-MpUHEaLhnxL0fUJCzB4ZF~WR4mQbcIi~EKe4yJ8t-dolHYIWHVxsYRcnB8pLhuJ~JaIWbe2btTaDwyEHU3mHYiVOu~HCgWeK2u0uKf2b5d-BpR7FB06O90VTrWKjBI-Jgp0dk8z1ggxHdRXTttxDhap3WYKYmBkXurTJboTkPB2pidK2kpid7q~oWqZIMbeaHVra7jTQ9inzCt9JwdRzj4Ep9rucjkwCqaYuGqTfDuD25VfAZxbTNGtImC13KRRspnLxecuXtYneyJaivQULdNXSodT80AO7ncHDPkfg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Contig length histogram of three cyprinids in this study and zebrafish, Danio rerio. By leveraging high throughput sequencing and bioinformatic filtering, we were able to generate high quality transcriptomes at a fraction of the cost and research effort used for zebrafish. As expected, de novo Trinity assemblies resulted in proportionally fewer contigs longer than 1000 bp, as compared to those of a well-assembled transcriptome, zebrafish (Danio rerio). However, note that we only used canonical transcripts for zebrafish and not the shorter isoforms, which skews the distribution toward longer transcripts for that species." path-from-xml="dsx034f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481601/dsx034f2.tif" path-from-xml="dsx034f2.tif" target="_blank">View large</a><a section="112481601" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/dsx034f2.png?Expires=2147483647&Signature=MKdcYg51jbVUUiiLUhZZ1egtcRWr92TwLekWvNjM3mQbEjLfEs8BBAwtpMJi0dy4CvAB0UrQAF9COxzsMw-r1XDk~o77Dv0FlfOcrCdG4jCGjmMhfCAn8yITPbXECyAYLEVMz4seeWW489Cd5T-MM-oxTDtI60Nevp5heOkAGbIfVzUX5cXOn8UWRTowAp~0vvZzx1-vcnNxbGJjXTd5gBH1N4ARaCY2LcLXVRMWdfgmlbwu8xqCEMvmEu2MgSHCTyJxmrUe7Bx4P2fQySEOvOloLZwv1WlA29unyZXe-zEJmvAb3nIEQ~-GzOeiAEdm6oiLRt9bOcNkpaJNiR1x4g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481601&ar=4106313&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx034f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Contig length histogram of three cyprinids in this study and zebrafish, <em>Danio rerio</em>. By leveraging high throughput sequencing and bioinformatic filtering, we were able to generate high quality transcriptomes at a fraction of the cost and research effort used for zebrafish. As expected, <em>de novo</em> T<span class="small-caps">rinity</span> assemblies resulted in proportionally fewer contigs longer than 1000 bp, as compared to those of a well-assembled transcriptome, zebrafish (<em>Danio rerio</em>). However, note that we only used canonical transcripts for zebrafish and not the shorter isoforms, which skews the distribution toward longer transcripts for that species.</p></div></div>                            <h3 scrollto-destination=112481602 id="112481602" class="section-title" >3.2. BLAST searches: zebrafish transcriptome</h3>
<p>Top BLASTx hits of T<span class="small-caps">rinity</span> contigs against zebrafish peptides included approximately 20,000 unique genes (ENSDARG) and 11,000 protein families (ENSFAM) present in each of the three species (<a reveal-id="dsx034-F3" data-open="dsx034-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>), suggesting similar annotation efficiency and transcriptome representation for each species. However, after pooling isoforms, the number of T<span class="small-caps">rinity</span> genes that significantly matched these ∼20,000 zebrafish genes varied among species: 66,447 in <em>Cyprinus carpio</em>, 60,990 in <em>Cyprinella lutrensis</em>, and 39,915 in <em>Platygobio gracilis</em> (<a reveal-id="dsx034-T2" data-open="dsx034-T2" class="link link-reveal link-table xref-fig">Table 2</a>, top row). Zebrafish genes were well covered, with more than 15,000 unique zebrafish genes covered over at least 70% of their length in corresponding contigs from each of the three cyprinids, consistent with the N50 data presented above. In general, zebrafish proteins were more completely covered by <em>P. gracilis</em> contigs than <em>C. carpio</em> or <em>C. lutrensis</em>. For example, zebrafish genes were more than 90% covered (i.e. the alignment covers &gt;90% of bases of a gene) by sequences in 50.3% (12,489 of 24,817 genes) of <em>P. gracilis</em> genes with significant zebrafish peptide hits, versus 49.9% (13,453 of 26,963) for <em>C. carpio</em>, and 46.8% (12,538 of 26,817) in <em>C. lutrensis</em>. A large number of T<span class="small-caps">rinity</span> contigs did not significantly match (BLASTx) zebrafish peptide sequences and were subsequently queried against several additional databases. <div content-id="dsx034-T2" class="table-modal"><div class="table-wrap"><div id="dsx034-T2" data-id="dsx034-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Significant BLAST hits for T<span class="small-caps">rinity</span> ‘genes’ versus various databases and number of ORFs present. BLAST searches were done in stepwise fashion: all T<span class="small-caps">rinity</span> genes were queried against zebrafish peptides but only genes without zebrafish peptide hits were queried against rRNA silva, and so on until all of the databases were queried. Summary of open reading frames (ORFs) identified in T<span class="small-caps">rinity</span> contigs with no significant BLAST hits against databases listed (‘No significant BLAST hits’). Some of the ORFs lacking similar proteins in the nr database may represent novel genes or genes with divergent sequences and function, while many are likely spurious results from the sequencing and assembly process or are from unidentified microbes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th><em>Cyprinus carpio</em> </th><th><em>Cyprinella lutrensis</em> </th><th><em>Platygobio gracilis</em> </th></tr></thead><tbody><tr><td>Zebrafish peptides </td><td>66,447 </td><td>60,990 </td><td>39,915 </td></tr><tr><td>rRNA Silva (microbiome) </td><td>140 </td><td>306 </td><td>87 </td></tr><tr><td>Teleost fish transcriptomes </td><td>4,572 </td><td>2,923 </td><td>1,561 </td></tr><tr><td>Zebrafish genome </td><td>48,527 </td><td>38,199 </td><td>31,955 </td></tr><tr><td>No significant BLAST hits </td><td>190,235 </td><td>153,445 </td><td>106,612 </td></tr><tr><td>Total contigs </td><td>309,921 </td><td>255,863 </td><td>180,130 </td></tr><tr><td>Predicted ORFs present </td><td>8,652 </td><td>9,215 </td><td>3,011 </td></tr><tr><td>ORFs with nr BLASTp hits </td><td>3,789 </td><td>4,154 </td><td>1,548 </td></tr><tr><td>ORFs without nr BLASTp hits (i.e. potentially novel genes) </td><td>4,863 </td><td>5,061 </td><td>1,463 </td></tr></tbody></table></div><div content-id="dsx034-T2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th><em>Cyprinus carpio</em> </th><th><em>Cyprinella lutrensis</em> </th><th><em>Platygobio gracilis</em> </th></tr></thead><tbody><tr><td>Zebrafish peptides </td><td>66,447 </td><td>60,990 </td><td>39,915 </td></tr><tr><td>rRNA Silva (microbiome) </td><td>140 </td><td>306 </td><td>87 </td></tr><tr><td>Teleost fish transcriptomes </td><td>4,572 </td><td>2,923 </td><td>1,561 </td></tr><tr><td>Zebrafish genome </td><td>48,527 </td><td>38,199 </td><td>31,955 </td></tr><tr><td>No significant BLAST hits </td><td>190,235 </td><td>153,445 </td><td>106,612 </td></tr><tr><td>Total contigs </td><td>309,921 </td><td>255,863 </td><td>180,130 </td></tr><tr><td>Predicted ORFs present </td><td>8,652 </td><td>9,215 </td><td>3,011 </td></tr><tr><td>ORFs with nr BLASTp hits </td><td>3,789 </td><td>4,154 </td><td>1,548 </td></tr><tr><td>ORFs without nr BLASTp hits (i.e. potentially novel genes) </td><td>4,863 </td><td>5,061 </td><td>1,463 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsx034-T2" data-id="dsx034-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Significant BLAST hits for T<span class="small-caps">rinity</span> ‘genes’ versus various databases and number of ORFs present. BLAST searches were done in stepwise fashion: all T<span class="small-caps">rinity</span> genes were queried against zebrafish peptides but only genes without zebrafish peptide hits were queried against rRNA silva, and so on until all of the databases were queried. Summary of open reading frames (ORFs) identified in T<span class="small-caps">rinity</span> contigs with no significant BLAST hits against databases listed (‘No significant BLAST hits’). Some of the ORFs lacking similar proteins in the nr database may represent novel genes or genes with divergent sequences and function, while many are likely spurious results from the sequencing and assembly process or are from unidentified microbes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th><em>Cyprinus carpio</em> </th><th><em>Cyprinella lutrensis</em> </th><th><em>Platygobio gracilis</em> </th></tr></thead><tbody><tr><td>Zebrafish peptides </td><td>66,447 </td><td>60,990 </td><td>39,915 </td></tr><tr><td>rRNA Silva (microbiome) </td><td>140 </td><td>306 </td><td>87 </td></tr><tr><td>Teleost fish transcriptomes </td><td>4,572 </td><td>2,923 </td><td>1,561 </td></tr><tr><td>Zebrafish genome </td><td>48,527 </td><td>38,199 </td><td>31,955 </td></tr><tr><td>No significant BLAST hits </td><td>190,235 </td><td>153,445 </td><td>106,612 </td></tr><tr><td>Total contigs </td><td>309,921 </td><td>255,863 </td><td>180,130 </td></tr><tr><td>Predicted ORFs present </td><td>8,652 </td><td>9,215 </td><td>3,011 </td></tr><tr><td>ORFs with nr BLASTp hits </td><td>3,789 </td><td>4,154 </td><td>1,548 </td></tr><tr><td>ORFs without nr BLASTp hits (i.e. potentially novel genes) </td><td>4,863 </td><td>5,061 </td><td>1,463 </td></tr></tbody></table></div><div content-id="dsx034-T2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th><em>Cyprinus carpio</em> </th><th><em>Cyprinella lutrensis</em> </th><th><em>Platygobio gracilis</em> </th></tr></thead><tbody><tr><td>Zebrafish peptides </td><td>66,447 </td><td>60,990 </td><td>39,915 </td></tr><tr><td>rRNA Silva (microbiome) </td><td>140 </td><td>306 </td><td>87 </td></tr><tr><td>Teleost fish transcriptomes </td><td>4,572 </td><td>2,923 </td><td>1,561 </td></tr><tr><td>Zebrafish genome </td><td>48,527 </td><td>38,199 </td><td>31,955 </td></tr><tr><td>No significant BLAST hits </td><td>190,235 </td><td>153,445 </td><td>106,612 </td></tr><tr><td>Total contigs </td><td>309,921 </td><td>255,863 </td><td>180,130 </td></tr><tr><td>Predicted ORFs present </td><td>8,652 </td><td>9,215 </td><td>3,011 </td></tr><tr><td>ORFs with nr BLASTp hits </td><td>3,789 </td><td>4,154 </td><td>1,548 </td></tr><tr><td>ORFs without nr BLASTp hits (i.e. potentially novel genes) </td><td>4,863 </td><td>5,061 </td><td>1,463 </td></tr></tbody></table></div></div></p>                        <a id="112481604" scrollto-destination="112481604"></a>
<div id="dsx034-F3" data-id="dsx034-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/m_dsx034f3.png?Expires=2147483647&Signature=vo4kJYIUv5ECs6kllNtCQ-M2QsbL18ftU4ocSV5s0nePxIqfQH4D2iC~nwHVa890HGCzGSlTUpEqKLkIaUdRx3WEFN3rR6a50ZBUqysMtp3ucgqUgkqVFzQ-1roy4hdLjY1zDvJPdSycTWwLkY47GyixGoUE7Z8luNWYyF7F7mQroqa0CmaWIe4scmZZlZ~Q9RbZasVuegdD8tERd8KLvfWVHvSrEgTqTHtESgBXdEVYw7fV9HMdZhonq9NFuZo3GJC0HtTEJrYc4eREvONU3RQ0OFKC81vHBBkF9EagXQEA3-gQB0XCJiYNH3M2OhND1hs9hpu4haRYD49js4RdXg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Unique genes and protein families from BLASTx searches (E-value threshold = 0.0001) against zebrafish (Danio rerio) peptide sequences." path-from-xml="dsx034f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481604/dsx034f3.tif" path-from-xml="dsx034f3.tif" target="_blank">View large</a><a section="112481604" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/dsx034f3.png?Expires=2147483647&Signature=WC4DFayvQG~YwJoUoScLffgph0LPbpNkH~DS29T9adUh88-EWgn2JU3P7wTDHHxsfgGHRszTE8fcw3b4VPzx8QmITM4yv9Cgyv-7Z7bjh9LHF2BpRclFnpHFnnOf~jSUzCyuRgghvk9M9lDGKg0I47nCH41JnDqXzJrT0d~27vf8TMYeNLJ9VxqCgrl2b7EhTrDFq7ZV5n4~ghytmNOL-kDmBFlYolwjHgPSrwVIUxFYh8V3NZOPclNNmQY3jR24V-TnGpuYwM5qCgCiecRlCV3aBb3Z3HfYhqyYeowVXh~tL4qv0F4GcDJhgjYB8p71QsXrTo0jMqhJFxnhbmceHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481604&ar=4106313&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx034f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Unique genes and protein families from BLASTx searches (E-value threshold = 0.0001) against zebrafish (<em>Danio rerio</em>) peptide sequences.</p></div></div><div content-id="dsx034-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/m_dsx034f3.png?Expires=2147483647&Signature=vo4kJYIUv5ECs6kllNtCQ-M2QsbL18ftU4ocSV5s0nePxIqfQH4D2iC~nwHVa890HGCzGSlTUpEqKLkIaUdRx3WEFN3rR6a50ZBUqysMtp3ucgqUgkqVFzQ-1roy4hdLjY1zDvJPdSycTWwLkY47GyixGoUE7Z8luNWYyF7F7mQroqa0CmaWIe4scmZZlZ~Q9RbZasVuegdD8tERd8KLvfWVHvSrEgTqTHtESgBXdEVYw7fV9HMdZhonq9NFuZo3GJC0HtTEJrYc4eREvONU3RQ0OFKC81vHBBkF9EagXQEA3-gQB0XCJiYNH3M2OhND1hs9hpu4haRYD49js4RdXg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Unique genes and protein families from BLASTx searches (E-value threshold = 0.0001) against zebrafish (Danio rerio) peptide sequences." path-from-xml="dsx034f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481604/dsx034f3.tif" path-from-xml="dsx034f3.tif" target="_blank">View large</a><a section="112481604" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/dsx034f3.png?Expires=2147483647&Signature=WC4DFayvQG~YwJoUoScLffgph0LPbpNkH~DS29T9adUh88-EWgn2JU3P7wTDHHxsfgGHRszTE8fcw3b4VPzx8QmITM4yv9Cgyv-7Z7bjh9LHF2BpRclFnpHFnnOf~jSUzCyuRgghvk9M9lDGKg0I47nCH41JnDqXzJrT0d~27vf8TMYeNLJ9VxqCgrl2b7EhTrDFq7ZV5n4~ghytmNOL-kDmBFlYolwjHgPSrwVIUxFYh8V3NZOPclNNmQY3jR24V-TnGpuYwM5qCgCiecRlCV3aBb3Z3HfYhqyYeowVXh~tL4qv0F4GcDJhgjYB8p71QsXrTo0jMqhJFxnhbmceHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481604&ar=4106313&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx034f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Unique genes and protein families from BLASTx searches (E-value threshold = 0.0001) against zebrafish (<em>Danio rerio</em>) peptide sequences.</p></div></div>                            <h3 scrollto-destination=112481605 id="112481605" class="section-title" >3.3. BLAST searches: other databases</h3>
<p>Contigs lacking significant BLASTx hits against zebrafish peptides were queried (BLASTn) iteratively against rRNA silva microbiome database, nine teleost transcriptomes, and the zebrafish genome databases (<a reveal-id="dsx034-T2" data-open="dsx034-T2" class="link link-reveal link-table xref-fig">Table 2</a>). For contigs lacking hits against zebrafish peptides, BLASTn searches versus the rRNA silva database revealed a small number of significant hits (i.e. &lt;400 contigs; <a reveal-id="dsx034-T2" data-open="dsx034-T2" class="link link-reveal link-table xref-fig">Table 2</a>). BLASTn searches of the remaining unmatched contigs versus the nine teleost fish transcriptomes identified approximately 1,500–4,500 additional hits (<a reveal-id="dsx034-T2" data-open="dsx034-T2" class="link link-reveal link-table xref-fig">Table 2</a>), far fewer than the evolutionarily more closely related zebrafish transcriptome. BLASTn searches of the remaining unidentified contigs against the zebrafish genome revealed a large number of significant hits (&gt;30,000 per species), suggesting these reads were the result of low levels of background genomic DNA contamination in the cDNA libraries, a common occurrence resulting from the hypersensitivity of Illumina sequencing. Conservation of sequences across deep evolutionary lineages suggests functional importance, such as regulatory regions.</p><p>Despite extensive BLAST searches, a large number of T<span class="small-caps">rinity</span> contigs (&gt;100,000 in each species or more than 50% of all contigs) did not have significant hits in any of the databases. These contigs are short in length (i.e. 200 bp) and have few reads mapping to them (e.g. single-read contigs). These could represent endogenous genomic DNA contamination of cDNA libraries and have sufficient evolutionary divergence from zebrafish to render BLASTn searches ineffective. A large number are also expected to be non-rRNA sequences from the microbiome, which were not present in target databases. Of contigs with no BLAST hits in the aforementioned databases, T<span class="small-caps">rans</span>D<span class="small-caps">ecoder</span> predicted ORFs in 8,652 (<em>Cyprinus carpio</em>), 9,215 (<em>Cyprinella lutrensis</em>), and 3,011 (<em>Platygobio gracilis</em>) contigs (<a reveal-id="dsx034-T2" data-open="dsx034-T2" class="link link-reveal link-table xref-fig">Table 2</a>). Roughly half of the predicted ORFs had significant BLASTp hits against the nr protein database (3,789, 4,154, and 1,548 contigs, respectively). Conversely, there were 4,863 (<em>C. carpio</em>), 5,061 (<em>C. lutrensis</em>), and 1,463 (<em>P. gracilis</em>) predicted ORFs had no significant hits against nr (<a reveal-id="dsx034-T2" data-open="dsx034-T2" class="link link-reveal link-table xref-fig">Table 2</a>). These ORFs could include novel genes not present in zebrafish or other teleost models, genes present in zebrafish but with significantly divergent sequences to cause BLAST searches to miss them, or could include genes from the microbiome that are not present in sequence databases.</p><p>For ORFs with nr hits, zebrafish was the top-hit species for a large portion (<a reveal-id="dsx034-F4" data-open="dsx034-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>), somewhat paradoxically given the lack of significant BLAST hits against zebrafish peptide and genome sequences discussed above. This appears to be due to the fact that T<span class="small-caps">rans</span>D<span class="small-caps">ecoder</span>-predicted ORFs exclude 5’ and 3’ untranslated regions (UTRs) which diverge more rapidly than ORFs over evolutionary time. In <em>C. carpio</em> and <em>C. lutrensis</em>, many of these ORFs are from a diverse microbiome with many sequences sharing significant similarity to cyclophyllid tapeworms (e.g. <em>Echinococcus</em>, <em>Hymenolepis</em>) and protozoans (e.g. <em>Tetrahymena</em>, <em>Parameceum</em>). Conversely, in <em>P. gracilis</em> the ORFs appear to be endogenous genes with high similarity to zebrafish (<a reveal-id="dsx034-F4" data-open="dsx034-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>), i.e. a less diverse microbiome is present. Contigs with predicted ORFs but no BLAST hits to any of the databases possibly represent novel or functionally divergent genes in these species that warrant further study. </p>                        <a id="112481609" scrollto-destination="112481609"></a>
<div id="dsx034-F4" data-id="dsx034-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/m_dsx034f4.png?Expires=2147483647&Signature=yeBvFIYJndlyhJxXJaPQ-DwCJ0efvhcO~a~SILFisGZtahlV0h~WFhGadgNy9ZV2DuFhRYa5g9mCq7JBgTDYxZXLg-mmorPYju6KOnUA6ENBxoZVyf1JWOo66wpNI5-LrLPuvN9eDHp8FWgAjuMIct7vvGXU6v53yiMB~b8ZkStgXMNOLeUt~hzjGDrw7SHTLbOpHgUlwOB9DG3U8Ew2blW7QohdYyz~PxXRAbqdpEbWospT80NkWBNPf8CClGMcNSWHi7Bk7vRCQrSqtnqLe8MUVffw7VCaH2tISkuNqrvtoKLvxlInCttkaLdTQzhtO4cS-FZ-nqqbhKoqI2594A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Top-species BLASTp hits for predicted open reading frame (ORF) peptide sequences queried against the nr database. Query sequences only included ORFs from contigs that lacked significant BLAST hits (see Table 2). Grey bars represent fish or other chordates, while black bars represent non-chordate taxa." path-from-xml="dsx034f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481609/dsx034f4.tif" path-from-xml="dsx034f4.tif" target="_blank">View large</a><a section="112481609" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/dsx034f4.png?Expires=2147483647&Signature=bvO~HZ8U89Y~ijBchZlYoGByXzTIf8TvYaBQ-S7jLNcgAL4W43lfTK1yjjL~02L0fbcpthETOMOtTKT46vgaVXqXnqrfATxMHUZGHtMjKR41aVJB7O7G1PR4ic22ZmXMK6JxQ3wyJAcZlu76YE38CXDYGizaWp0WOnI7VMRlxtDKa76nJLh04uDjIECYis3qjvKsbSAsZGRaO4EaNnuGfJWkjl-Iddcs5GOR3JJU3oGMVo7aei1V8AnVTji-D~rzQ10ggok8m9ISpI7dNJyAHRhDHNqwd7K3C~X7wHVtep-c5RxAC7x9~6u32dU2jfJzNwQTcTmlzukLR4l3f89u~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481609&ar=4106313&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx034f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Top-species BLASTp hits for predicted open reading frame (ORF) peptide sequences queried against the nr database. Query sequences only included ORFs from contigs that lacked significant BLAST hits (see <a reveal-id="dsx034-T2" data-open="dsx034-T2" class="link link-reveal link-table xref-fig">Table 2</a>). Grey bars represent fish or other chordates, while black bars represent non-chordate taxa.</p></div></div><div content-id="dsx034-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/m_dsx034f4.png?Expires=2147483647&Signature=yeBvFIYJndlyhJxXJaPQ-DwCJ0efvhcO~a~SILFisGZtahlV0h~WFhGadgNy9ZV2DuFhRYa5g9mCq7JBgTDYxZXLg-mmorPYju6KOnUA6ENBxoZVyf1JWOo66wpNI5-LrLPuvN9eDHp8FWgAjuMIct7vvGXU6v53yiMB~b8ZkStgXMNOLeUt~hzjGDrw7SHTLbOpHgUlwOB9DG3U8Ew2blW7QohdYyz~PxXRAbqdpEbWospT80NkWBNPf8CClGMcNSWHi7Bk7vRCQrSqtnqLe8MUVffw7VCaH2tISkuNqrvtoKLvxlInCttkaLdTQzhtO4cS-FZ-nqqbhKoqI2594A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Top-species BLASTp hits for predicted open reading frame (ORF) peptide sequences queried against the nr database. Query sequences only included ORFs from contigs that lacked significant BLAST hits (see Table 2). Grey bars represent fish or other chordates, while black bars represent non-chordate taxa." path-from-xml="dsx034f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481609/dsx034f4.tif" path-from-xml="dsx034f4.tif" target="_blank">View large</a><a section="112481609" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/dsx034f4.png?Expires=2147483647&Signature=bvO~HZ8U89Y~ijBchZlYoGByXzTIf8TvYaBQ-S7jLNcgAL4W43lfTK1yjjL~02L0fbcpthETOMOtTKT46vgaVXqXnqrfATxMHUZGHtMjKR41aVJB7O7G1PR4ic22ZmXMK6JxQ3wyJAcZlu76YE38CXDYGizaWp0WOnI7VMRlxtDKa76nJLh04uDjIECYis3qjvKsbSAsZGRaO4EaNnuGfJWkjl-Iddcs5GOR3JJU3oGMVo7aei1V8AnVTji-D~rzQ10ggok8m9ISpI7dNJyAHRhDHNqwd7K3C~X7wHVtep-c5RxAC7x9~6u32dU2jfJzNwQTcTmlzukLR4l3f89u~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481609&ar=4106313&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx034f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Top-species BLASTp hits for predicted open reading frame (ORF) peptide sequences queried against the nr database. Query sequences only included ORFs from contigs that lacked significant BLAST hits (see <a reveal-id="dsx034-T2" data-open="dsx034-T2" class="link link-reveal link-table xref-fig">Table 2</a>). Grey bars represent fish or other chordates, while black bars represent non-chordate taxa.</p></div></div>                            <h3 scrollto-destination=112481610 id="112481610" class="section-title" >3.4. Filtering and the final assembly datasets</h3>
<p>After filtering and removal of genomic DNA and microbiome reads, the final <em>de novo</em> assembly datasets contained only T<span class="small-caps">rinity</span> contigs falling into one of the following categories: (i) contigs with significant BLAST hits against zebrafish or the nine other teleost transcriptomes; or (ii) contigs with no matches against any of the databases but with predicted ORFs present, i.e. potentially novel genes. All other contigs were removed via bioinformatic filtering. While it is possible that some of the ‘microbiome’ hits are actually external contamination, we expect this to be a minor component given the diverse nature of these sequences in terms of top-hit organism (<a reveal-id="dsx034-F4" data-open="dsx034-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). It is also possible that some of the genes that significantly align against zebrafish are actually microbiome or contaminant reads, though these genes being target species DNA is a more parsimonious conclusion. The final datasets are significantly smaller than the raw <em>de novo</em> assembly but present much more reliable sequence information, i.e. transcriptome sequences rather than microbiome or genomic DNA contamination.</p>                            <h3 scrollto-destination=112481612 id="112481612" class="section-title" >3.5. Genome duplication, diploidization and gene silencing</h3>
<p>Transcriptome annotation and comparison with zebrafish revealed that <em>Cyprinus carpio</em> expresses more genes than <em>Cyprinella lutrensis</em> and <em>Platygobio gracilis</em>, due to the Cc4R duplication (<a reveal-id="dsx034-F5" data-open="dsx034-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). <em>Cyprinus carpio</em> expressed about 41% more genes overall than <em>P. gracilis</em> and 11% more than <em>C. lutrensis</em>. The number of duplicate genes expressed varied dramatically among tissue types (<a reveal-id="dsx034-F5" data-open="dsx034-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). In all tissues except skin, <em>C. carpio</em> expressed more genes than the other two species (i.e. 3–48% more). In skin, both <em>C. lutrensis</em> and <em>P. gracilis</em> expressed more genes than <em>C. carpio</em> (26 and 2%, respectively). Using higher thresholds for ‘expression’ had moderate impact on the inferred percentage of duplicates expressed: a threshold of 100 reads instead of 10 resulted in different estimates of duplicated genes expressed in <em>C. carpio</em> versus <em>P. gracilis</em> (18% more in <em>C. carpio</em>) and <em>C. lutrensis</em> (8% more in <em>C. carpio</em>), i.e. retained expression of Cc4R duplicates. The disparity in these results could be driven in part by different assembly qualities (e.g. a better assembled <em>P. gracilis</em> transcriptome). WGCNA analysis revealed broadly similar patterns of blocks of co-expressed genes across species, consistent with the phylogenetic relatedness of the three species (<a reveal-id="dsx034-F6" data-open="dsx034-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). </p>                        <a id="112481614" scrollto-destination="112481614"></a>
<div id="dsx034-F5" data-id="dsx034-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/m_dsx034f5.png?Expires=2147483647&Signature=IL4dXyU1UUbUjHV2k-9GaWsSzbWrVvXQok7OZQuoDLjTmtz4wRF3iJk0uNPDLDTHs-vK7DhabwoqCP-FtcLWW-pZjRB1WlOumWz9ECnfjOc~viQ~fcYpWdjOJ447W0YluCJ6IRjCNtLTzh3JG1F7RwPZm3BNQAccoq8wOVbeGBOI7FSwuVLaIapKVfwH6g5yCWxkmHo~6Mgych6F-h41EQ5Rckj5W3zPrABz3sP6zbbrv~xHEH4jEVkTvDb71vKAZvibyikuxx~DTZjCM8USB1dq9d2hMRFnBxYCTZ7RnNYlFx5YADaq0GVfBJUlDgBo3Zeh5SuEL8IE6MvAhRN8ww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Number of Trinity genes (contigs) expressed in each of four tissue types, as well as all tissues pooled. Contigs only include those with significant BLASTx hits versus zebrafish peptides." path-from-xml="dsx034f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481614/dsx034f5.tif" path-from-xml="dsx034f5.tif" target="_blank">View large</a><a section="112481614" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/dsx034f5.png?Expires=2147483647&Signature=mBKpHYm930pGEuJa6mqVqRO9xymDTxyOiSjL2cnIv26wTE6wVyZ9ChZoIPwckRDt6Bwn0XVngFth2KouXkHgHULy8jznieobJ~kHnFfMSEdPw-BHcZyjcWWwjZG8rz0heqCkxxCqHaI7NAHGumWscNbJvasrfIp33E3BId8R-ENZYpTuO~OtXD4Cj49hXhmTLsBPZ-oYjaUvgGwwMmzs3bTsWq8RikvjAKvZOqjh0g75dXxEH3l6K2k6uwSfQz8fKWyZdj1kDhx8xTGRiuYFe8Vls9gDCHkXP0R~VqqrBRIs2k9TmBeKjXG5zvlj-Y8pOGdcHY52YKLVqgSvDp3V9g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481614&ar=4106313&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx034f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of T<span class="small-caps">rinity</span> genes (contigs) expressed in each of four tissue types, as well as all tissues pooled. Contigs only include those with significant BLASTx hits versus zebrafish peptides.</p></div></div><div content-id="dsx034-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/m_dsx034f5.png?Expires=2147483647&Signature=IL4dXyU1UUbUjHV2k-9GaWsSzbWrVvXQok7OZQuoDLjTmtz4wRF3iJk0uNPDLDTHs-vK7DhabwoqCP-FtcLWW-pZjRB1WlOumWz9ECnfjOc~viQ~fcYpWdjOJ447W0YluCJ6IRjCNtLTzh3JG1F7RwPZm3BNQAccoq8wOVbeGBOI7FSwuVLaIapKVfwH6g5yCWxkmHo~6Mgych6F-h41EQ5Rckj5W3zPrABz3sP6zbbrv~xHEH4jEVkTvDb71vKAZvibyikuxx~DTZjCM8USB1dq9d2hMRFnBxYCTZ7RnNYlFx5YADaq0GVfBJUlDgBo3Zeh5SuEL8IE6MvAhRN8ww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Number of Trinity genes (contigs) expressed in each of four tissue types, as well as all tissues pooled. Contigs only include those with significant BLASTx hits versus zebrafish peptides." path-from-xml="dsx034f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481614/dsx034f5.tif" path-from-xml="dsx034f5.tif" target="_blank">View large</a><a section="112481614" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/dsx034f5.png?Expires=2147483647&Signature=mBKpHYm930pGEuJa6mqVqRO9xymDTxyOiSjL2cnIv26wTE6wVyZ9ChZoIPwckRDt6Bwn0XVngFth2KouXkHgHULy8jznieobJ~kHnFfMSEdPw-BHcZyjcWWwjZG8rz0heqCkxxCqHaI7NAHGumWscNbJvasrfIp33E3BId8R-ENZYpTuO~OtXD4Cj49hXhmTLsBPZ-oYjaUvgGwwMmzs3bTsWq8RikvjAKvZOqjh0g75dXxEH3l6K2k6uwSfQz8fKWyZdj1kDhx8xTGRiuYFe8Vls9gDCHkXP0R~VqqrBRIs2k9TmBeKjXG5zvlj-Y8pOGdcHY52YKLVqgSvDp3V9g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481614&ar=4106313&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx034f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of T<span class="small-caps">rinity</span> genes (contigs) expressed in each of four tissue types, as well as all tissues pooled. Contigs only include those with significant BLASTx hits versus zebrafish peptides.</p></div></div>                        <a id="112481615" scrollto-destination="112481615"></a>
<div id="dsx034-F6" data-id="dsx034-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/m_dsx034f6.png?Expires=2147483647&Signature=MLhRavEo4BFb9XlgUeYCd2yKRMqeGOAITpK24dQQQWw~hl35p6~ADl6Hhx4zwuLI1as6KbUQRdAZ99Ql1oF0U9PNzxOQhlhD7w0RDv7TkydHTIk2NfOcnS7xzLVbBKM9H11bpqpIu4HCTpSBojGPU9CfUyoqlJ1EtWXMVMx13naENf1CZbmCaKwk~oK0EyaG1vFM148cjDweo3qceC4KlWusAVPf4ATX8Z7snoprsB9tnV2e72qDFoQX8AGlzsRKkehr2GcttL5J8xpPEOA738~NgbV7MfORhCwWRq4l5LEE8omMQbgcqpYIj7SDXZIv86N6p2KLubppc2AKQVBJHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Results of WGCNA analysis for three cyprinid species. Dendrograms represent results of hierarchical cluster analysis of co-expression patterns of genes. Colored bars to the left and top of the heatmap show modules of co-expressed genes and pairs of genes with higher co-expression show (darker) coloration in the heatmap." path-from-xml="dsx034f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481615/dsx034f6.tif" path-from-xml="dsx034f6.tif" target="_blank">View large</a><a section="112481615" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/dsx034f6.png?Expires=2147483647&Signature=l8L6z~dK21L6qsLtX4EAyNdU8ws7HMQi5OdqHKE3ofpx8GZS5dohZa9gthQfb7ctZGXogXKt58xWI6bZkzR3WG7fHB-BJ~GWN0HtIGJxvm9lDG2POL8IGhQX8ceT02Z1dq~vlcHlIX7cn5scaWF80yrMw5jfl5SfEg9MXn-KJxjnaRnBHzekHS9eBJlcgv7~Jkco29iR13bg1Nb34Ut7Hyqu5fFkuWBxWGZYaK5VVEIwL5ddFSLIzivxriaNGTwX6ccIVta0vmC4XiG3LkzbaLm-IKMLX02UEqZ0WTKnsNEcEljVdUcNhJ84RePmW1TIpSyzsMlkUbqh9LxzUN4r4A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481615&ar=4106313&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx034f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Results of WGCNA analysis for three cyprinid species. Dendrograms represent results of hierarchical cluster analysis of co-expression patterns of genes. Colored bars to the left and top of the heatmap show modules of co-expressed genes and pairs of genes with higher co-expression show (darker) coloration in the heatmap.</p></div></div><div content-id="dsx034-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/m_dsx034f6.png?Expires=2147483647&Signature=MLhRavEo4BFb9XlgUeYCd2yKRMqeGOAITpK24dQQQWw~hl35p6~ADl6Hhx4zwuLI1as6KbUQRdAZ99Ql1oF0U9PNzxOQhlhD7w0RDv7TkydHTIk2NfOcnS7xzLVbBKM9H11bpqpIu4HCTpSBojGPU9CfUyoqlJ1EtWXMVMx13naENf1CZbmCaKwk~oK0EyaG1vFM148cjDweo3qceC4KlWusAVPf4ATX8Z7snoprsB9tnV2e72qDFoQX8AGlzsRKkehr2GcttL5J8xpPEOA738~NgbV7MfORhCwWRq4l5LEE8omMQbgcqpYIj7SDXZIv86N6p2KLubppc2AKQVBJHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Results of WGCNA analysis for three cyprinid species. Dendrograms represent results of hierarchical cluster analysis of co-expression patterns of genes. Colored bars to the left and top of the heatmap show modules of co-expressed genes and pairs of genes with higher co-expression show (darker) coloration in the heatmap." path-from-xml="dsx034f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481615/dsx034f6.tif" path-from-xml="dsx034f6.tif" target="_blank">View large</a><a section="112481615" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/dsx034f6.png?Expires=2147483647&Signature=l8L6z~dK21L6qsLtX4EAyNdU8ws7HMQi5OdqHKE3ofpx8GZS5dohZa9gthQfb7ctZGXogXKt58xWI6bZkzR3WG7fHB-BJ~GWN0HtIGJxvm9lDG2POL8IGhQX8ceT02Z1dq~vlcHlIX7cn5scaWF80yrMw5jfl5SfEg9MXn-KJxjnaRnBHzekHS9eBJlcgv7~Jkco29iR13bg1Nb34Ut7Hyqu5fFkuWBxWGZYaK5VVEIwL5ddFSLIzivxriaNGTwX6ccIVta0vmC4XiG3LkzbaLm-IKMLX02UEqZ0WTKnsNEcEljVdUcNhJ84RePmW1TIpSyzsMlkUbqh9LxzUN4r4A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481615&ar=4106313&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx034f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Results of WGCNA analysis for three cyprinid species. Dendrograms represent results of hierarchical cluster analysis of co-expression patterns of genes. Colored bars to the left and top of the heatmap show modules of co-expressed genes and pairs of genes with higher co-expression show (darker) coloration in the heatmap.</p></div></div><p>Genes with retained duplicate expression (i.e. Ohnologs) in <em>C. carpio</em> represented a suite of functional groups: gene ontology terms that were significantly enriched in the ‘retained duplicates’ list were diverse (<a reveal-id="dsx034-F7" data-open="dsx034-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>, top panel). One functional grouping that was a predominant contributor in the R<span class="small-caps">evigo</span> analysis was ‘anatomical structure morphogenesis,’ of interest because common carp attain much larger body size than the other two species (<a reveal-id="dsx034-F7" data-open="dsx034-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>, bottom panel). </p>                        <a id="112481617" scrollto-destination="112481617"></a>
<div id="dsx034-F7" data-id="dsx034-F7" class="fig fig-section"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/m_dsx034f7.png?Expires=2147483647&Signature=E3db7uKq~xjSeQujSdpuSS~n9FREWksiEPr8sI~10dk9jgjht8Wkzkpot6caDdsO7l~JPUP-l7jTFULi~73S4ZJmwqpMs0h-eBINCh-n4t6Av~XNbuPd-g19LtuEHuck3CV8~dCv2uRf8uweqyujGwXum~8ppdDAnf8HRXE1nMn71gGjX11lJmRYMKohzJ~5tv1quBKAXAMu5Aw8~McxE5CzbDkIpXpkEtXSK6gh1A0gfIny-xX7Y~LeHK1ZbTbhJZC-qXOH91xkPmDcTG6a6jycF-h6e5CRjYE-ZGjtRKPozd2Y7mgBbDHiS2aBWXBanhYnfRVCt5jL7jenfTtHLw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Top panel: Gene-ontology terms that are over- or under-represented (y-axis) in the list of genes retained as duplicates in the common carp transcriptome as compared to all expressed genes in common carp. Bottom panel: Summary of groups of biological processes overrepresented in the retained-duplicates in common carp. Box size is proportional to the number of genes with particular gene ontology terms, which may suggest a dosage effect in common carp." path-from-xml="dsx034f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481617/dsx034f7.tif" path-from-xml="dsx034f7.tif" target="_blank">View large</a><a section="112481617" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/dsx034f7.png?Expires=2147483647&Signature=rfhGhuhSbFcLjdY9JKb74yuiNABdbzqihalmFnSpRX46c9Guvx86m5qt~svqg39Yhzmuj0qUpXGynnflN1yvcF~LaD1TDac7vAqy~rfweYxaOdNg9IDwkEPVFJG4IIhGQD7nhcsSq0cyJ3zp4P-bhMOJvtCb38QLK~lUeWexDweEScSLqmHdmNe3zy23hh93Nt0wDkhoQc3rU32q6KY5gvb8p1nXY0lrF36wbzAEqVhxZ6SsT0WKnaQucz-hYeEF6agFUnIiygDh~RRuxhwjrNqJrIAIDlSlfA9fbm6GjPiuER2i5B8EACjNbp1VSAwx5A5DfRnai-ZLMe87LKnedQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481617&ar=4106313&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx034f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Top panel: Gene-ontology terms that are over- or under-represented (y-axis) in the list of genes retained as duplicates in the common carp transcriptome as compared to all expressed genes in common carp. Bottom panel: Summary of groups of biological processes overrepresented in the retained-duplicates in common carp. Box size is proportional to the number of genes with particular gene ontology terms, which may suggest a dosage effect in common carp.</p></div></div><div content-id="dsx034-F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/m_dsx034f7.png?Expires=2147483647&Signature=E3db7uKq~xjSeQujSdpuSS~n9FREWksiEPr8sI~10dk9jgjht8Wkzkpot6caDdsO7l~JPUP-l7jTFULi~73S4ZJmwqpMs0h-eBINCh-n4t6Av~XNbuPd-g19LtuEHuck3CV8~dCv2uRf8uweqyujGwXum~8ppdDAnf8HRXE1nMn71gGjX11lJmRYMKohzJ~5tv1quBKAXAMu5Aw8~McxE5CzbDkIpXpkEtXSK6gh1A0gfIny-xX7Y~LeHK1ZbTbhJZC-qXOH91xkPmDcTG6a6jycF-h6e5CRjYE-ZGjtRKPozd2Y7mgBbDHiS2aBWXBanhYnfRVCt5jL7jenfTtHLw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Top panel: Gene-ontology terms that are over- or under-represented (y-axis) in the list of genes retained as duplicates in the common carp transcriptome as compared to all expressed genes in common carp. Bottom panel: Summary of groups of biological processes overrepresented in the retained-duplicates in common carp. Box size is proportional to the number of genes with particular gene ontology terms, which may suggest a dosage effect in common carp." path-from-xml="dsx034f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481617/dsx034f7.tif" path-from-xml="dsx034f7.tif" target="_blank">View large</a><a section="112481617" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/dsx034f7.png?Expires=2147483647&Signature=rfhGhuhSbFcLjdY9JKb74yuiNABdbzqihalmFnSpRX46c9Guvx86m5qt~svqg39Yhzmuj0qUpXGynnflN1yvcF~LaD1TDac7vAqy~rfweYxaOdNg9IDwkEPVFJG4IIhGQD7nhcsSq0cyJ3zp4P-bhMOJvtCb38QLK~lUeWexDweEScSLqmHdmNe3zy23hh93Nt0wDkhoQc3rU32q6KY5gvb8p1nXY0lrF36wbzAEqVhxZ6SsT0WKnaQucz-hYeEF6agFUnIiygDh~RRuxhwjrNqJrIAIDlSlfA9fbm6GjPiuER2i5B8EACjNbp1VSAwx5A5DfRnai-ZLMe87LKnedQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481617&ar=4106313&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx034f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Top panel: Gene-ontology terms that are over- or under-represented (y-axis) in the list of genes retained as duplicates in the common carp transcriptome as compared to all expressed genes in common carp. Bottom panel: Summary of groups of biological processes overrepresented in the retained-duplicates in common carp. Box size is proportional to the number of genes with particular gene ontology terms, which may suggest a dosage effect in common carp.</p></div></div>                            <h3 scrollto-destination=112481618 id="112481618" class="section-title" >3.6. Expression of essential genes</h3>
<p>Genes that are essential for embryonic development in <em>D. rerio</em> were nearly all present in the three cyprinids: 285 (<em>Platygobio gracilis</em>), 301 (<em>Cyprinella lutrensis</em>), and 301 (<em>Cyprinus carpio</em>) genes were expressed out of 305 zebrafish essential genes (i.e. 93.4–97.8%). Of the 20 essential genes that we did not detect in <em>P. gracilis</em>, only one was also missing in <em>C. lutrensis</em>, and two were shared with <em>C. carpio</em>. No missing essential genes were shared between <em>C. lutrensis</em> and <em>C. carpio</em>, of the four missing in each species. Essential genes missing in one or more species were generally expressed at low levels in the other species. Essential genes were nearly ubiquitously expressed across all four tissue types (skin, gill, gut, kidney), with low levels of tissue specificity (<a reveal-id="dsx034-F8" data-open="dsx034-F8" class="link link-reveal link-table xref-fig">Fig. 8</a>), in contrast to non-essential genes which generally exhibited higher levels of tissue specificity. A few essential genes do exhibit patterns of tissue specificity or species-specificity. For example, <em>C. carpio</em> expresses more essential genes in the gut than the other two species, including genes such as <em>wdr46</em> and <em>exosc8</em>, which are missing in both of the other species. Normalized levels of expression were higher in <em>C. carpio</em> than <em>P. gracilis</em> and <em>C. lutrensis</em> for 165 and 204 out of 305 genes, respectively. This pattern was not due to <em>C. carpio</em> expressing more loci per zebrafish gene (e.g. Ohnologs) than the other two species. Only slightly more loci (e.g. <em>n</em> = 2 contigs) were expressed per essential gene in the recently duplicated <em>C. carpio</em> genome (<a reveal-id="dsx034-F9" data-open="dsx034-F9" class="link link-reveal link-table xref-fig">Fig. 9</a>) whereas most duplicated essential genes in <em>C. carpio</em> are not transcribed and have either been lost evolutionarily, e.g. pseudogenes, or are expressed in other developmental stages or tissues. </p>                        <a id="112481620" scrollto-destination="112481620"></a>
<div id="dsx034-F8" data-id="dsx034-F8" class="fig fig-section"><div class="label fig-label">Figure 8</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/m_dsx034f8.png?Expires=2147483647&Signature=sqkVLWkvrFAeIWniQK7FVAaNx4W5XC0hOo1GeF90wIMLWaLU04bW5vEPA9~CvGSGHHmmFrzTG2IbLDZf-LoQtxmj0i5J9rwA59wOCpIfshEoaqzT68uUNcVSE~ajnRcCFAUCQSZghCCgqgAbfZpGBHRQjvaoyd6grKoJSlipKcgPzVlj0vKs1NZoDJcV8Uvxj1y3~DzpXxbC9jXteFBF6If8Ya4M0hL8uHOj9HjBstACN-bbuMGKA88arqnpMBnvRQIas2mjHFtmB85MPIjrtFX76uY3d85p028n37GUk-lYC3zMnQYugKjsarvIFbinG9xC1mYn5xINI52SGfNbpA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression of essential developmental genes by tissue type in three cyprinid fishes compared to 305 essential genes expressed across all tissues in zebrafish (Danio rerio)." path-from-xml="dsx034f8.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481620/dsx034f8.tif" path-from-xml="dsx034f8.tif" target="_blank">View large</a><a section="112481620" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/dsx034f8.png?Expires=2147483647&Signature=djeiP56O5vK4iGcWq2suGUGqsGhh4492RRGylAsxIC9wz5SxFTVBsE8fQJLpajvcq5GYt3mJoTRGxHXoqCcYgRbdbq105AiafCAvs2nLj0-NtswTQqaLDhr70kvcS-ESczeff-xrK2BdYH1BJ3CrpG04LtfIK3LyNJ9UBPNMl-yRShifBuRPyiTUZpkj6ktYfx7VP3bMqB~If6CxYT-rnOU3-sqFJLYUyKtxdbZk9yJpZC3i9~zRtvrSedQgx3P7SE34jWyVm3PyMAboaaY~y6OlVdkHMJvFQQ9E3bAa8gFWCtArex2CxKgRUl9o2Kiwt3P5ATRQBwvVksNitAkfJA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481620&ar=4106313&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx034f8.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression of essential developmental genes by tissue type in three cyprinid fishes compared to 305 essential genes expressed across all tissues in zebrafish (<em>Danio rerio</em>).</p></div></div><div content-id="dsx034-F8" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 8</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/m_dsx034f8.png?Expires=2147483647&Signature=sqkVLWkvrFAeIWniQK7FVAaNx4W5XC0hOo1GeF90wIMLWaLU04bW5vEPA9~CvGSGHHmmFrzTG2IbLDZf-LoQtxmj0i5J9rwA59wOCpIfshEoaqzT68uUNcVSE~ajnRcCFAUCQSZghCCgqgAbfZpGBHRQjvaoyd6grKoJSlipKcgPzVlj0vKs1NZoDJcV8Uvxj1y3~DzpXxbC9jXteFBF6If8Ya4M0hL8uHOj9HjBstACN-bbuMGKA88arqnpMBnvRQIas2mjHFtmB85MPIjrtFX76uY3d85p028n37GUk-lYC3zMnQYugKjsarvIFbinG9xC1mYn5xINI52SGfNbpA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression of essential developmental genes by tissue type in three cyprinid fishes compared to 305 essential genes expressed across all tissues in zebrafish (Danio rerio)." path-from-xml="dsx034f8.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481620/dsx034f8.tif" path-from-xml="dsx034f8.tif" target="_blank">View large</a><a section="112481620" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/dsx034f8.png?Expires=2147483647&Signature=djeiP56O5vK4iGcWq2suGUGqsGhh4492RRGylAsxIC9wz5SxFTVBsE8fQJLpajvcq5GYt3mJoTRGxHXoqCcYgRbdbq105AiafCAvs2nLj0-NtswTQqaLDhr70kvcS-ESczeff-xrK2BdYH1BJ3CrpG04LtfIK3LyNJ9UBPNMl-yRShifBuRPyiTUZpkj6ktYfx7VP3bMqB~If6CxYT-rnOU3-sqFJLYUyKtxdbZk9yJpZC3i9~zRtvrSedQgx3P7SE34jWyVm3PyMAboaaY~y6OlVdkHMJvFQQ9E3bAa8gFWCtArex2CxKgRUl9o2Kiwt3P5ATRQBwvVksNitAkfJA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481620&ar=4106313&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx034f8.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression of essential developmental genes by tissue type in three cyprinid fishes compared to 305 essential genes expressed across all tissues in zebrafish (<em>Danio rerio</em>).</p></div></div>                        <a id="112481621" scrollto-destination="112481621"></a>
<div id="dsx034-F9" data-id="dsx034-F9" class="fig fig-section"><div class="label fig-label">Figure 9</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/m_dsx034f9.png?Expires=2147483647&Signature=OJ8~qgIWOQCouLfNvp-sGVYtcXrfRg-FMn7q5VzZ6KOZAgIDvIAIfJnHBtCoBDT~uz1~U3KA3lygg5KvbIRn4reKqY8kZD7P6IDvanltthwCPPwNsrB3~tC3GsfuHP6HCq9nsskK4V0rmSS2uDUW-LtLDJzM65j3AMURBLGcN2dAWLUULnqurgSAHuq5iQh8~og1s0YXWctAVpjVviJoLfmSN9Sl8H939aoaalaZzsi~o-KqzOHNdXyVOqkn5JPOi8fZGfdq3RDih7G31z2kPKV5EJNUtPfzn~AcbSNaDNFoqQM8wNqxAYbOt3NWsQ~fZtXfM4RsuaM3pIPq0pjQjg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Number of loci (Trinity genes) expressed per zebrafish essential gene. Only slightly more (e.g. n = 2) were expressed per essential gene in the recently duplicated genome of Cyprinus carpio. Most duplicated essential genes in C. carpio are not transcribed and have either been lost evolutionarily, e.g. pseudogenes, or are expressed in other developmental stages or tissues." path-from-xml="dsx034f9.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481621/dsx034f9.tif" path-from-xml="dsx034f9.tif" target="_blank">View large</a><a section="112481621" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/dsx034f9.png?Expires=2147483647&Signature=raHq2VthlZgyqaPSZ7nYG5vCTcOfBD~WOkzXDBZFqaPDCaRnI6~W2EUZXE~AKUj9cpLihbwM~lsO20lv8CSc~Daxz-8qX5~w-e3haP5UwhYmb0ayZGTnEGzUnpRnEZMwsJ9c6yFfqW~337VOpsI0SI7lbZtcUNNL5nXX53miRo61C-bf8v2XE9wcqucYir44KDMDodTNJK1ju6cK9aIDA9Fw9rfDDlfvx1pt52dKdVSVrTYF7NQrhgQYRO4GU1f9Fv~thEAowI6mozHvKZbhKC~JqSYXJYD7BvWfOvIKxgHrjMSC4lkaz6KGwkEubon~CMKSp0bzHRuO7Vl7PXq3DA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481621&ar=4106313&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx034f9.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of loci (T<span class="small-caps">rinity</span> genes) expressed per zebrafish essential gene. Only slightly more (e.g. <em>n</em> = 2) were expressed per essential gene in the recently duplicated genome of <em>Cyprinus carpio</em>. Most duplicated essential genes in <em>C. carpio</em> are not transcribed and have either been lost evolutionarily, e.g. pseudogenes, or are expressed in other developmental stages or tissues.</p></div></div><div content-id="dsx034-F9" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 9</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/m_dsx034f9.png?Expires=2147483647&Signature=OJ8~qgIWOQCouLfNvp-sGVYtcXrfRg-FMn7q5VzZ6KOZAgIDvIAIfJnHBtCoBDT~uz1~U3KA3lygg5KvbIRn4reKqY8kZD7P6IDvanltthwCPPwNsrB3~tC3GsfuHP6HCq9nsskK4V0rmSS2uDUW-LtLDJzM65j3AMURBLGcN2dAWLUULnqurgSAHuq5iQh8~og1s0YXWctAVpjVviJoLfmSN9Sl8H939aoaalaZzsi~o-KqzOHNdXyVOqkn5JPOi8fZGfdq3RDih7G31z2kPKV5EJNUtPfzn~AcbSNaDNFoqQM8wNqxAYbOt3NWsQ~fZtXfM4RsuaM3pIPq0pjQjg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Number of loci (Trinity genes) expressed per zebrafish essential gene. Only slightly more (e.g. n = 2) were expressed per essential gene in the recently duplicated genome of Cyprinus carpio. Most duplicated essential genes in C. carpio are not transcribed and have either been lost evolutionarily, e.g. pseudogenes, or are expressed in other developmental stages or tissues." path-from-xml="dsx034f9.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112481621/dsx034f9.tif" path-from-xml="dsx034f9.tif" target="_blank">View large</a><a section="112481621" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/dsx034f9.png?Expires=2147483647&Signature=raHq2VthlZgyqaPSZ7nYG5vCTcOfBD~WOkzXDBZFqaPDCaRnI6~W2EUZXE~AKUj9cpLihbwM~lsO20lv8CSc~Daxz-8qX5~w-e3haP5UwhYmb0ayZGTnEGzUnpRnEZMwsJ9c6yFfqW~337VOpsI0SI7lbZtcUNNL5nXX53miRo61C-bf8v2XE9wcqucYir44KDMDodTNJK1ju6cK9aIDA9Fw9rfDDlfvx1pt52dKdVSVrTYF7NQrhgQYRO4GU1f9Fv~thEAowI6mozHvKZbhKC~JqSYXJYD7BvWfOvIKxgHrjMSC4lkaz6KGwkEubon~CMKSp0bzHRuO7Vl7PXq3DA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112481621&ar=4106313&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx034f9.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of loci (T<span class="small-caps">rinity</span> genes) expressed per zebrafish essential gene. Only slightly more (e.g. <em>n</em> = 2) were expressed per essential gene in the recently duplicated genome of <em>Cyprinus carpio</em>. Most duplicated essential genes in <em>C. carpio</em> are not transcribed and have either been lost evolutionarily, e.g. pseudogenes, or are expressed in other developmental stages or tissues.</p></div></div>                            <h2 scrollto-destination=112481622 id="112481622" class="section-title" >4. Discussion</h2>
<p>Next-generation transcriptome sequencing has revolutionized the field of molecular ecology over the past decade.<a reveal-id="dsx034-B4" data-open="dsx034-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx034-B74" data-open="dsx034-B74" class="link link-ref link-reveal xref-bibr"><sup>74</sup></a> One outcome is increased appreciation for the molecular complexity underlying the evolution of basic ecological traits.<a reveal-id="dsx034-B75" data-open="dsx034-B75" class="link link-ref link-reveal xref-bibr"><sup>75</sup></a><sup>,</sup><a reveal-id="dsx034-B76" data-open="dsx034-B76" class="link link-ref link-reveal xref-bibr"><sup>76</sup></a> Here we present transcriptomic resources for comparative study of non-model cyprinid fishes in a natural ‘common-garden’ setting. Previous work, along with our bioinformatic analyses demonstrate that careful processing and filtering is needed to assess the sources of DNA fragments, which can be endogenous target transcriptome sequences, genomic DNA ‘contamination’ from the study organism, or DNA from the microbiome or diet items. Assessment of transcriptome quality also requires careful consideration.<a reveal-id="dsx034-B77" data-open="dsx034-B77" class="link link-ref link-reveal xref-bibr"><sup>77</sup></a><sup>,</sup><a reveal-id="dsx034-B78" data-open="dsx034-B78" class="link link-ref link-reveal xref-bibr"><sup>78</sup></a> Traditional measures of assembled read lengths such as N50 are largely meaningless for transcriptomes without additional context. We advocate combining N50 and/or histograms of contig lengths with explicit comparisons to well-studied transcriptomes of model organisms, when available. For example, we compared our <em>de novo</em> transcriptomes to zebrafish, which yielded valuable insight into progress made in our target species. Finally, positive identification of nearly all zebrafish essential genes in our transcriptomes provides additional evidence of the utility of our annotation procedures. Using the bioinformatics pipeline presented in <a reveal-id="dsx034-F1" data-open="dsx034-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, we obtained high quality transcriptome data from three species of cyprinid fishes with distinctly different evolutionary histories.</p><p>Our specific aims in this study were to sequence, annotate, and assemble the transcriptomes of co-occurring fishes with the goal of developing resources for ongoing studies of the evolution and molecular ecology of North American cyprinids. This comparative transcriptome dataset offers tools to construct assays to pose and test hypotheses related to differences in DNA sequences, functional pathways, and expression patterns among organisms that are more or less closely related (i.e. comparative approach), but that also co-occur in nature and experience similar biotic and abiotic conditions, including exposure to similar suites of pathogens and water quality conditions, for example. These data are also a resource for identifying single nucleotide polymorphisms (SNPs) in transcribed genes,<a reveal-id="dsx034-B79" data-open="dsx034-B79" class="link link-ref link-reveal xref-bibr"><sup>79</sup></a> which could be used to explore functional or phenotypic variation within and among species.</p><p>There are several key findings in this study, including: (i) high-quality transcriptome assemblies for cyprinid fishes that reveal broad similarities and evolutionary conservation of genes with zebrafish, but with some key differences; (ii) several potentially novel genes not identified in zebrafish that are candidates for studies of ecological and morphological novelty; (iii) diverse microbiomes that vary substantially among species, despite origin from a single collection locality; (iv) ubiquitous expression of essential genes for development in later ontogenetic stages (i.e. juveniles and adults) across a broad array of tissue types; (v) a large number of duplicate genes expressed in the tetraploid, <em>Cyprinus carpio</em>, representing a diverse suite of biological processes or gene ontologies. We discuss each of these findings in greater detail below.</p>                            <h3 scrollto-destination=112481626 id="112481626" class="section-title" >4.1. Assembly results</h3>
<p>There are important considerations associated with conducting transcriptome analysis in a non-laboratory setting and in species lacking high-quality, well-annotated genomes.<a reveal-id="dsx034-B4" data-open="dsx034-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx034-B80" data-open="dsx034-B80" class="link link-ref link-reveal xref-bibr"><sup>80</sup></a><sup>,</sup><a reveal-id="dsx034-B81" data-open="dsx034-B81" class="link link-ref link-reveal xref-bibr"><sup>81</sup></a> For example, it is necessary to identify ways to maximize the quality and completeness of <em>de novo</em> assemblies.<a reveal-id="dsx034-B77" data-open="dsx034-B77" class="link link-ref link-reveal xref-bibr"><sup>77</sup></a><sup>,</sup><a reveal-id="dsx034-B80" data-open="dsx034-B80" class="link link-ref link-reveal xref-bibr"><sup>80</sup></a><sup>,</sup><a reveal-id="dsx034-B82" data-open="dsx034-B82" class="link link-ref link-reveal xref-bibr"><sup>82</sup></a> Our assemblies are somewhat less complete than the zebrafish reference, but this was expected because zebrafish has been sequenced extensively at the genomic DNA level, empirically validated with RNA-seq, and refined by years of manual curation.</p><p>T<span class="small-caps">rinity</span> assemblies resulted in proportionally fewer long contigs (e.g. &gt; 1,000 bp) compared to zebrafish. Four factors account for this result. First, the microbiome is present in these sequences and many of the contigs are not endogenous, as reflected by top species hits in BLAST searches (<a reveal-id="dsx034-F4" data-open="dsx034-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). Second, a small amount of genomic DNA contamination persists despite DNase treatment during library preparation. Genomic contamination tends to be observed as short (e.g. 200 bp), shallow contigs often comprised of single-reads. Third, the <em>de novo</em> assemblies are more fragmented due to the short read technology employed, with multiple contigs often representing non-overlapping fragments of the same gene. This effect is particularly acute in genes with short sequence repeats (e.g. microsatellites). Finally, we only used the canonical zebrafish transcripts in this study, which excludes the shorter isoforms present in many genes and biases the zebrafish distribution toward longer sequences. Transcriptomes presented here represent an improvement (i.e. more sequences, higher coverage; longer relative N50) over earlier work on sequencing and assembling the common carp transcriptome using Roche 454 sequencing,<a reveal-id="dsx034-B39" data-open="dsx034-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a><sup>,</sup><a reveal-id="dsx034-B51" data-open="dsx034-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> due to the higher throughput, Illumina paired-end sequencing approach we employed. The bioinformatic approach we presented to identify and filter non-target sequences from the final dataset resulted in high quality and well annotated assemblies.</p>                            <h3 scrollto-destination=112481629 id="112481629" class="section-title" >4.2. Potentially novel genes</h3>
<p>Results of BLAST searches and ORF predictions helped us identify candidate genes that may represent novel species- or taxon-specific genes. Our interest in these genes lies in the idea that they may contain some of the functional elements responsible for extensive ecological and phylogenetic diversity present in the Cyprinidae, as in previous studies of lineage-specific gene family expansions.<a reveal-id="dsx034-B83 dsx034-B84 dsx034-B85" data-open="dsx034-B83 dsx034-B84 dsx034-B85" class="link link-ref link-reveal xref-bibr"><sup>83–85</sup></a> For example, expansion of the <em>patristacin</em> gene family in pipefish may be an important driver in the evolution of male pregnancy in that lineage.<a reveal-id="dsx034-B85" data-open="dsx034-B85" class="link link-ref link-reveal xref-bibr"><sup>85</sup></a> Many of the potentially novel genes we identified may prove to be false positives as more fish genomes are sequenced and annotated; however, these candidates would be an excellent starting point for researchers interested in targeted searches for genes or proteins underlying ecological novelty in cyprinids that may have arisen through local gene duplications, exon shuffling, horizontal transfer, or other mechanisms.</p>                            <h3 scrollto-destination=112481631 id="112481631" class="section-title" >4.3. Microbiome diversity</h3>
<p>Another valuable aspect of transcriptome sequencing of samples taken from nature is the simultaneous generation of quantifiable data on the microbiome.<a reveal-id="dsx034-B86" data-open="dsx034-B86" class="link link-ref link-reveal xref-bibr"><sup>86</sup></a> These data are applicable to study of host-parasite dynamics, immune response, paired comparative population genetics or phylogeographic analysis of host and microbiota. When generating <em>de novo</em> transcriptome assemblies for focal species, it is imperative that microbiome sequences are identified and filtered out of final assemblies.<a reveal-id="dsx034-B87" data-open="dsx034-B87" class="link link-ref link-reveal xref-bibr"><sup>87</sup></a> Genome-scale sequence data is often lacking for the bacterial and metazoan microbiota on vertebrate samples, which complicates attempts at removal. We used an iterative and successive filtering approach to address this issue (<a reveal-id="dsx034-F1" data-open="dsx034-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>) that provides valuable information on the likely source (e.g. exogenous or endogenous) of particular sequences or contigs. Transcriptome characterization studies often do not attempt to remove exogenous microbiome and genomic DNA contamination. Researchers should be cautious when using unfiltered sequence reads, particularly when they are compiled into massive databases that lack appropriate metadata.</p>                            <h3 scrollto-destination=112481633 id="112481633" class="section-title" >4.4. Conservation of essential genes</h3>
<p>Genes that are essential for embryonic development present interesting targets for studying genome evolution due to their critical functional importance.<a reveal-id="dsx034-B58" data-open="dsx034-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a><sup>,</sup><a reveal-id="dsx034-B88" data-open="dsx034-B88" class="link link-ref link-reveal xref-bibr"><sup>88</sup></a><sup>,</sup><a reveal-id="dsx034-B89" data-open="dsx034-B89" class="link link-ref link-reveal xref-bibr"><sup>89</sup></a> Essential genes also bear biomedical significance as many have been implicated in human diseases and developmental abnormalities.<a reveal-id="dsx034-B88" data-open="dsx034-B88" class="link link-ref link-reveal xref-bibr"><sup>88</sup></a> Our data demonstrate that essential ‘developmental’ genes previously identified in larval zebrafish<a reveal-id="dsx034-B58" data-open="dsx034-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a><sup>,</sup><a reveal-id="dsx034-B59" data-open="dsx034-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> are almost all ubiquitously expressed in juvenile or adults across a broad range of tissues, suggesting their importance is not simply limited to early ontogenetic stages or particular tissues. Previous work has shown that many of these genes are critical for basic cell function, which may underlie their ubiquitous expression.<a reveal-id="dsx034-B58" data-open="dsx034-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> Based on the critical functions they perform, these genes are candidates for future studies looking at the cause of high rates of genetic inviability and mortality in cyprinids and other organisms with type-III life histories.<a reveal-id="dsx034-B90" data-open="dsx034-B90" class="link link-ref link-reveal xref-bibr"><sup>90</sup></a> From a practical standpoint, however, we suggest that the ubiquitous expression of these genes makes their sequencing coverage and completeness useful metrics that should be used to assess the quality and completeness of <em>de novo</em> transcriptome assemblies, analogous to the use of ‘housekeeping’ genes as positive controls in qPCR studies.<a reveal-id="dsx034-B91" data-open="dsx034-B91" class="link link-ref link-reveal xref-bibr"><sup>91</sup></a> The presence of nearly all essential genes across these four cyprinid species (representing more than 100 million years of evolutionary divergence<a reveal-id="dsx034-B92" data-open="dsx034-B92" class="link link-ref link-reveal xref-bibr"><sup>92</sup></a>) is consistent with the hypothesis of broad evolutionary and functional conservation.<a reveal-id="dsx034-B58" data-open="dsx034-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a><sup>,</sup><a reveal-id="dsx034-B61" data-open="dsx034-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> The few essential genes not detected may still be present in the genome, but were missed due to assembly errors or are expressed transiently at larval or juvenile developmental stages. We propose that the number of essential genes expressed could be used as a metric to complement other measures of assembly quality and completeness,<a reveal-id="dsx034-B77" data-open="dsx034-B77" class="link link-ref link-reveal xref-bibr"><sup>77</sup></a> in addition to comparing transcript length histograms to closely related model species (see <a reveal-id="dsx034-F2" data-open="dsx034-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). While beyond the scope of the present study, future work should compare the utility of these metrics for assessing transcriptome assemblies.</p>                            <h3 scrollto-destination=112481635 id="112481635" class="section-title" >4.5. Tetraploidy and expression of duplicated genes</h3>
<p>Our results indicated that a large number of duplicate genes are expressed in <em>Cyprinus carpio</em>, representing a diverse suite of biological processes or gene ontologies, similar to previous studies.<a reveal-id="dsx034-B39" data-open="dsx034-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a><sup>,</sup><a reveal-id="dsx034-B40" data-open="dsx034-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a><sup>,</sup><a reveal-id="dsx034-B51" data-open="dsx034-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> For genes where both Ohnologs were expressed in <em>C. carpio</em>, there was enrichment in several different functional pathways, but many genes were associated with ‘anatomical structure morphogenesis’ in particular (<a reveal-id="dsx034-F7" data-open="dsx034-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>). Functional duplicates at these genes correlate with large body size and rapid growth in <em>C. carpio</em> as compared to <em>C. lutrensis</em> and <em>P. gracilis</em> and a potential dosage effect. Using a different set of tissues, Wang <em>et al.</em><a reveal-id="dsx034-B39" data-open="dsx034-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> identified enrichment of retained expression of duplicates in gene ontology pathways involved in metabolic and immune functions using 454 transcriptome sequencing and EST data mining. The availability of a (draft) genome for common carp<a reveal-id="dsx034-B40" data-open="dsx034-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> will eventually help identify Ohnologs that are silenced because pseudogenes of silenced genes may still be present in genomic DNA sequences; currently, the incomplete annotation of that genome precludes analysis of gene silencing at the genomic DNA level. Ultimately, knowledge of which genes are retained and expressed in duplicate in tetraploids as compared to related diploid species can provide insight into the role that whole genome duplication plays in the molecular ecology and phylogenetic diversification of organisms.<a reveal-id="dsx034-B93" data-open="dsx034-B93" class="link link-ref link-reveal xref-bibr"><sup>93</sup></a><sup>,</sup><a reveal-id="dsx034-B94" data-open="dsx034-B94" class="link link-ref link-reveal xref-bibr"><sup>94</sup></a> Note that our analyses and those of Wang <em>et al.</em><a reveal-id="dsx034-B39" data-open="dsx034-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> are based only on expressed genes in particular tissues at a single time point, rather than genomic DNA sequences and consequently would not include Ohnologs expressed only in different tissues or at different time points. The recent Cc4R allotetraploidy event<a reveal-id="dsx034-B35" data-open="dsx034-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a><sup>,</sup><a reveal-id="dsx034-B95" data-open="dsx034-B95" class="link link-ref link-reveal xref-bibr"><sup>95</sup></a><sup>,</sup><a reveal-id="dsx034-B96" data-open="dsx034-B96" class="link link-ref link-reveal xref-bibr"><sup>96</sup></a> complicates transcriptome assembly because there has been little time for divergence of Ohnologs e.g, 8.2 million years.<a reveal-id="dsx034-B40" data-open="dsx034-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> In autopolyploid salmonids, the fourth round of whole genome duplication is much older i.e. 90–102 ma;<a reveal-id="dsx034-B97" data-open="dsx034-B97" class="link link-ref link-reveal xref-bibr"><sup>97</sup></a> yet many Ohnologous loci are difficult to separate via bioinformatic approaches. Some loci even maintain tetrasomic inheritance because of the autopolyploid nature of the duplication.<a reveal-id="dsx034-B98" data-open="dsx034-B98" class="link link-ref link-reveal xref-bibr"><sup>98</sup></a> These factors need to be explicitly considered when conducting analyses that require orthologous alignments, such as RNA-seq and syntenic mapping, when working with polyploid or partially diploidized species.<a reveal-id="dsx034-B38" data-open="dsx034-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a></p>                            <h3 scrollto-destination=112481637 id="112481637" class="section-title" >4.6. Summary</h3>
<p>Results from short read sequences yield high-quality transcriptome resources for comparative study of cyprinids, a hyper-diverse clade of fishes. We used a variety of bioinformatic tools for assembly quality assessment, gene annotation, orthology assignment, and identification and partitioning of exogenous DNA in wild cyprinid fishes. This approach facilitates technology transfer from a model organism (zebrafish) to a group of related species that fill diverse and critical roles in these ecosystems and comprise an important component of biodiversity. Conserved expression of essential developmental genes across a broad phylogenetic scope, later ontogenetic stages, and array of tissue types, illustrates their utility as benchmarks for assessing coverage in <em>de novo</em> assemblies. Moreover, their ubiquitous expression further supports the hypothesis that these genes are required for the basic biology of cyprinid fish and are candidate loci for developmental abnormalities and disease. Finally, comparative transcriptomics must contend with genome duplications and other genomic ‘events’ that affect gene identity and expression. Nonetheless, comparative approaches could provide enormous power to identify shared and unique physiological pathways that respond to common environmental stressors in a natural setting.</p>                            <h3 scrollto-destination=112481639 id="112481639" class="section-title" >4.7. Data Availability</h3>
<p>Raw sequence reads were uploaded to the NCBI Sequence Read Archive (SRA: SRP107991: SRR5601334-SRR560133469. BIOPROJECT: PRJNA383604. BIOSAMPLES: SAMN07166458-SAMN0716493). T<span class="small-caps">rinity</span>-assembled transcriptomes are available via FigShare. BLAST results and the list of contigs corresponding to potentially novel genes are available in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/supplementarydata_dsx034.xlsx?Expires=2147483647&Signature=jZVn~I2JY8wrwNW2b2M3lDDIVKAlTUBD9SjHqxIN~bpSot8oZy4L1aIQlOxBJtCotU2197loDrnwre~22Me8F-~wXMIvHyWVltrAEne5slSO2Oud1RMbILcYKf3kRtCgnybZe55DAxKxPqfZ1l3DrV7rlmdhHsLWERTcf2KbDO8rDi-VAHI4zsCwMZRP1j5Ck6q-TjzZpu5cAnubHFLE2MbToJj~n8zGivxn~7u~Ji60eOqsC3GEoJrqsp6W9FipcrTIonY-CUqW3lgXItWB9XWnOh6RyyolP3vBulH6a~wcXjTaMqCQq81hqRS9BL~ajgd6dQfgAiVzymec-V6Bjg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span>.</p>                            <h2 scrollto-destination=112481641 id="112481641" class="section-title" >Supplementary data</h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/supplementarydata_dsx034.xlsx?Expires=2147483647&Signature=jZVn~I2JY8wrwNW2b2M3lDDIVKAlTUBD9SjHqxIN~bpSot8oZy4L1aIQlOxBJtCotU2197loDrnwre~22Me8F-~wXMIvHyWVltrAEne5slSO2Oud1RMbILcYKf3kRtCgnybZe55DAxKxPqfZ1l3DrV7rlmdhHsLWERTcf2KbDO8rDi-VAHI4zsCwMZRP1j5Ck6q-TjzZpu5cAnubHFLE2MbToJj~n8zGivxn~7u~Ji60eOqsC3GEoJrqsp6W9FipcrTIonY-CUqW3lgXItWB9XWnOh6RyyolP3vBulH6a~wcXjTaMqCQq81hqRS9BL~ajgd6dQfgAiVzymec-V6Bjg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h2 scrollto-destination=112481643 id="112481643" class="backacknowledgements-title" >Acknowledgements</h2>
<p>This project was supported by the National Institute of General Medical Sciences (8P20GM103451-12), New Mexico IDeA Networks of Biomedical Research Excellence (NMINBRE_A2_Jan_2013), and the Center for Evolutionary and Theoretical Immunology. Samples were collected under New Mexico Department of Game and Fish permit #3015. This research was approved by Institutional Animal Care and Use Committee Protocol #10-100468-MCC and #10-100492-MCC. We thank E. Loker, R. Miller, J. Kavka, and G. Rosenberg for research and technical support. Thanks to Z. Ren and L. Hao for assistance with the Database of Essential Genes. Fish images were provided T. Kennedy (red shiner, flathead chub) and C. Thomas (common carp). This research benefitted from insight and technical assistance provided by F. Schilkey, N. Devitt, P. Mena, T. Ramaraj, I. Lindquist, A. Snyder, M. Osborne, and C. Krabbenhoft. We thank the anonymous reviewers for helpful comments on the manuscript. The authors have no competing interests.</p>                            <h3 scrollto-destination=112481645 id="112481645" class="section-title" >Accession numbers</h3>
<p>NCBI Genbank SRA: SRP107991: SRR5601334-SRR560133469</p>                            <h3 scrollto-destination=112481647 id="112481647" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h2 scrollto-destination=112481649 id="112481649" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx034-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mardis</div> <div class="given-names">E. R.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">The impact of next-generation sequencing technology on genetics</div>. <div class="source">Trends Genet</div> ., <div class="volume">24</div>, <div class="fpage">133</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20impact%20of%20next-generation%20sequencing%20technology%20on%20genetics&amp;author=E.%20R.%20Mardis&amp;publication_year=2008&amp;journal=Trends%20Genet&amp;volume=24&amp;pages=133-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2007.12.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2007.12.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2007.12.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18262675" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shendure</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Ji</div> <div class="given-names">H.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Next-generation DNA sequencing</div>. <div class="source">Nat. Biotechnol</div> ., <div class="volume">26</div>, <div class="fpage">1135</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next-generation%20DNA%20sequencing&amp;author=J.%20Shendure&amp;author=H.%20Ji&amp;publication_year=2008&amp;journal=Nat.%20Biotechnol&amp;volume=26&amp;pages=1135-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1486" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1486" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1486"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18846087" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ekblom</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Galindo</div> <div class="given-names">J.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Applications of next generation sequencing in molecular ecology of non-model organisms</div>. <div class="source">Heredity</div> , <div class="volume">107</div>, <div class="fpage">1</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Applications%20of%20next%20generation%20sequencing%20in%20molecular%20ecology%20of%20non-model%20organisms&amp;author=R.%20Ekblom&amp;author=J.%20Galindo&amp;publication_year=2011&amp;journal=Heredity&amp;volume=107&amp;pages=1-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/hdy.2010.152" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fhdy.2010.152" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fhdy.2010.152"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21139633" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Alvarez</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Schrey</div> <div class="given-names">A. W.</div></div>, <div class="name string-name"><div class="surname">Richards</div> <div class="given-names">C. L.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Ten years of transcriptomics in wild populations: what have we learned about their ecology and evolution?</div> <div class="source">Mol. Ecol</div> ., <div class="volume">24</div>, <div class="fpage">710</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ten%20years%20of%20transcriptomics%20in%20wild%20populations%3A%20what%20have%20we%20learned%20about%20their%20ecology%20and%20evolution%3F&amp;author=M.%20Alvarez&amp;author=A.%20W.%20Schrey&amp;author=C.%20L.%20Richards&amp;publication_year=2015&amp;journal=Mol.%20Ecol&amp;volume=24&amp;pages=710-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.13055" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.13055" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.13055"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25604587" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Andrews</div> <div class="given-names">K. R.</div></div>, <div class="name string-name"><div class="surname">Good</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Miller</div> <div class="given-names">M. R.</div></div>, <div class="name string-name"><div class="surname">Luikart</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Hohenlohe</div> <div class="given-names">P. A.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Harnessing the power of RADseq for ecological and evolutionary genomics</div>. <div class="source">Nat. Rev. Genet</div> ., <div class="volume">17</div>, <div class="fpage">81</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Harnessing%20the%20power%20of%20RADseq%20for%20ecological%20and%20evolutionary%20genomics&amp;author=K.%20R.%20Andrews&amp;author=J.%20M.%20Good&amp;author=M.%20R.%20Miller&amp;author=G.%20Luikart&amp;author=P.%20A.%20Hohenlohe&amp;publication_year=2016&amp;journal=Nat.%20Rev.%20Genet&amp;volume=17&amp;pages=81-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg.2015.28" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg.2015.28" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg.2015.28"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26729255" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Barshis</div> <div class="given-names">D. J.</div></div>, <div class="name string-name"><div class="surname">Ladner</div> <div class="given-names">J. T.</div></div>, <div class="name string-name"><div class="surname">Oliver</div> <div class="given-names">T. A.</div></div>, <div class="name string-name"><div class="surname">Seneca</div> <div class="given-names">F. O.</div></div>, <div class="name string-name"><div class="surname">Traylor-Knowles</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Palumbi</div> <div class="given-names">S. R.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Genomic basis for coral resilience to climate change</div>. <div class="source">Proc. Natl. Acad. Sci</div> ., <div class="volume">110</div>, <div class="fpage">1387</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20basis%20for%20coral%20resilience%20to%20climate%20change&amp;author=D.%20J.%20Barshis&amp;author=J.%20T.%20Ladner&amp;author=T.%20A.%20Oliver&amp;author=F.%20O.%20Seneca&amp;author=N.%20Traylor-Knowles&amp;author=S.%20R.%20Palumbi&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci&amp;volume=110&amp;pages=1387-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1210224110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1210224110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1210224110"> </span></div> </div></div></div></div></div><div content-id="dsx034-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schunter</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Vollmer</div> <div class="given-names">S. V.</div></div>, <div class="name string-name"><div class="surname">Macpherson</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Pascual</div> <div class="given-names">M.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Transcriptome analyses and differential gene expression in a non-model fish species with alternative mating tactics</div>. <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">167</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20analyses%20and%20differential%20gene%20expression%20in%20a%20non-model%20fish%20species%20with%20alternative%20mating%20tactics&amp;author=C.%20Schunter&amp;author=S.%20V.%20Vollmer&amp;author=E.%20Macpherson&amp;author=M.%20Pascual&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=167" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-167" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-167" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-167"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24581002" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Verbruggen</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Bickley</div> <div class="given-names">L. K.</div></div>, <div class="name string-name"><div class="surname">Santos</div> <div class="given-names">E. M.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">De novo assembly of the <em>Carcinus maenas</em> transcriptome and characterization of innate immune system pathways</div>. <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20of%20the%20Carcinus%20maenas%20transcriptome%20and%20characterization%20of%20innate%20immune%20system%20pathways&amp;author=B.%20Verbruggen&amp;author=L.%20K.%20Bickley&amp;author=E.%20M.%20Santos&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1667-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1667-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1667-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25553907" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pavey</div> <div class="given-names">S. A.</div></div>, <div class="name string-name"><div class="surname">Bernatchez</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Aubin-Horth</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Landry</div> <div class="given-names">C. R.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">What is needed for next-generation ecological and evolutionary genomics?</div> <div class="source">Trends Ecol. Evol</div> ., <div class="volume">27</div>, <div class="fpage">673</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=What%20is%20needed%20for%20next-generation%20ecological%20and%20evolutionary%20genomics%3F&amp;author=S.%20A.%20Pavey&amp;author=L.%20Bernatchez&amp;author=N.%20Aubin-Horth&amp;author=C.%20R.%20Landry&amp;publication_year=2012&amp;journal=Trends%20Ecol.%20Evol&amp;volume=27&amp;pages=673-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tree.2012.07.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tree.2012.07.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tree.2012.07.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22902072" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Van Straalen</div> <div class="given-names">N. M.</div></div>, <div class="name string-name"><div class="surname">Roelofs</div> <div class="given-names">D.</div></div></div> <div class="year">2012</div>, <div class="source">An Introduction to Ecological Genomics</div> . 
<div class="publisher-name">Oxford University Press</div>.</div></div></div></div><div content-id="dsx034-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Landry</div> <div class="given-names">C. R.</div></div>, <div class="name string-name"><div class="surname">Aubin-Horth</div> <div class="given-names">N.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Recent advances in ecological genomics: from phenotypic plasticity to convergent and adaptive evolution and speciation</div>. <div class="source">Adv. Exp. Med. Biol</div> ., <div class="volume">781</div>, <div class="fpage">1</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recent%20advances%20in%20ecological%20genomics%3A%20from%20phenotypic%20plasticity%20to%20convergent%20and%20adaptive%20evolution%20and%20speciation&amp;author=C.%20R.%20Landry&amp;author=N.%20Aubin-Horth&amp;publication_year=2014&amp;journal=Adv.%20Exp.%20Med.%20Biol&amp;volume=781&amp;pages=1-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/978-94-007-7347-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2F978-94-007-7347-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2F978-94-007-7347-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24277292" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Goltsman</div> <div class="given-names">D. S. A.</div></div>, <div class="name string-name"><div class="surname">Comolli</div> <div class="given-names">L. R.</div></div>, <div class="name string-name"><div class="surname">Thomas</div> <div class="given-names">B. C.</div></div>, <div class="name string-name"><div class="surname">Banfield</div> <div class="given-names">J. F.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Community transcriptomics reveals unexpected high microbial diversity in acidophilic biofilm communities</div>. <div class="source">ISME J</div> ., <div class="volume">9</div>, <div class="fpage">1014</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Community%20transcriptomics%20reveals%20unexpected%20high%20microbial%20diversity%20in%20acidophilic%20biofilm%20communities&amp;author=D.%20S.%20A.%20Goltsman&amp;author=L.%20R.%20Comolli&amp;author=B.%20C.%20Thomas&amp;author=J.%20F.%20Banfield&amp;publication_year=2015&amp;journal=ISME%20J&amp;volume=9&amp;pages=1014-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2014.200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2014.200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2014.200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25361394" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">DeBiasse</div> <div class="given-names">M. B.</div></div>, <div class="name string-name"><div class="surname">Kelly</div> <div class="given-names">M. W.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Plastic and evolved responses to global change: what can we learn from comparative transcriptomics?</div> <div class="source">J. Hered</div> ., <div class="volume">107</div>, <div class="fpage">71</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plastic%20and%20evolved%20responses%20to%20global%20change%3A%20what%20can%20we%20learn%20from%20comparative%20transcriptomics%3F&amp;author=M.%20B.%20DeBiasse&amp;author=M.%20W.%20Kelly&amp;publication_year=2016&amp;journal=J.%20Hered&amp;volume=107&amp;pages=71-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/esv073" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2Fesv073" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2Fesv073"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26519514" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Eo</div> <div class="given-names">S. H.</div></div>, <div class="name string-name"><div class="surname">Doyle</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Hale</div> <div class="given-names">M. C.</div></div>, <div class="name string-name"><div class="surname">Marra</div> <div class="given-names">N. J.</div></div>, <div class="name string-name"><div class="surname">Ruhl</div> <div class="given-names">J. D.</div></div>, <div class="name string-name"><div class="surname">DeWoody</div> <div class="given-names">J. A.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Comparative transcriptomics and gene expression in larval tiger salamander (<em>Ambystoma tigrinum</em>) gill and lung tissues as revealed by pyrosequencing</div>. <div class="source">Gene</div> , <div class="volume">492</div>, <div class="fpage">329</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20transcriptomics%20and%20gene%20expression%20in%20larval%20tiger%20salamander%20(Ambystoma%20tigrinum)%20gill%20and%20lung%20tissues%20as%20revealed%20by%20pyrosequencing&amp;author=S.%20H.%20Eo&amp;author=J.%20M.%20Doyle&amp;author=M.%20C.%20Hale&amp;author=N.%20J.%20Marra&amp;author=J.%20D.%20Ruhl&amp;author=J.%20A.%20DeWoody&amp;publication_year=2012&amp;journal=Gene&amp;volume=492&amp;pages=329-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2011.11.018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2011.11.018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2011.11.018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22138480" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Whitehead</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Triant</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Champlin</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Nacci</div> <div class="given-names">D.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Comparative transcriptomics implicates mechanisms of evolved pollution tolerance in a killifish population</div>. <div class="source">Mol. Ecol</div> ., <div class="volume">19</div>, <div class="fpage">5186</div>–<div class="lpage">203</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20transcriptomics%20implicates%20mechanisms%20of%20evolved%20pollution%20tolerance%20in%20a%20killifish%20population&amp;author=A.%20Whitehead&amp;author=D.%20Triant&amp;author=D.%20Champlin&amp;author=D.%20Nacci&amp;publication_year=2010&amp;journal=Mol.%20Ecol&amp;volume=19&amp;pages=5186-203" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.2010.19.issue-23" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.2010.19.issue-23" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.2010.19.issue-23"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20874759" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cohen</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Bogeat-Triboulot</div> <div class="given-names">M.-B.</div></div>, <div class="name string-name"><div class="surname">Tisserant</div> <div class="given-names">E.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Comparative transcriptomics of drought responses in Populus: a meta-analysis of genome-wide expression profiling in mature leaves and root apices across two genotypes</div>. <div class="source">BMC Genomics</div> , <div class="volume">11</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20transcriptomics%20of%20drought%20responses%20in%20Populus%3A%20a%20meta-analysis%20of%20genome-wide%20expression%20profiling%20in%20mature%20leaves%20and%20root%20apices%20across%20two%20genotypes&amp;author=D.%20Cohen&amp;author=M.-B.%20Bogeat-Triboulot&amp;author=E.%20Tisserant&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-630" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-630" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-630"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20044946" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Des Marais</div> <div class="given-names">D. L.</div></div>, <div class="name string-name"><div class="surname">Rausher</div> <div class="given-names">M. D.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Escape from adaptive conflict after duplication in an anthocyanin pathway gene</div>. <div class="source">Nature</div> , <div class="volume">454</div>, <div class="fpage">762</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Escape%20from%20adaptive%20conflict%20after%20duplication%20in%20an%20anthocyanin%20pathway%20gene&amp;author=D.%20L.%20Des%20Marais&amp;author=M.%20D.%20Rausher&amp;publication_year=2008&amp;journal=Nature&amp;volume=454&amp;pages=762-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07092" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07092" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07092"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18594508" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Reid</div> <div class="given-names">N. M.</div></div>, <div class="name string-name"><div class="surname">Proestou</div> <div class="given-names">D. A.</div></div>, <div class="name string-name"><div class="surname">Clark</div> <div class="given-names">B. W.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">The genomic landscape of rapid repeated evolutionary adaptation to toxic pollution in wild fish</div>. <div class="source">Science</div> , <div class="volume">354</div>, <div class="fpage">1305</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genomic%20landscape%20of%20rapid%20repeated%20evolutionary%20adaptation%20to%20toxic%20pollution%20in%20wild%20fish&amp;author=N.%20M.%20Reid&amp;author=D.%20A.%20Proestou&amp;author=B.%20W.%20Clark&amp;publication_year=2016&amp;journal=Science&amp;volume=354&amp;pages=1305-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.aah4993" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.aah4993" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.aah4993"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27940876" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Thompson</div> <div class="given-names">A. W.</div></div>, <div class="name string-name"><div class="surname">Ortí</div> <div class="given-names">G.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Annual killifish transcriptomics and candidate genes for metazoan diapause</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">33</div>, <div class="fpage">2391</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Annual%20killifish%20transcriptomics%20and%20candidate%20genes%20for%20metazoan%20diapause&amp;author=A.%20W.%20Thompson&amp;author=G.%20Ort%C3%AD&amp;publication_year=2016&amp;journal=Mol.%20Biol.%20Evol&amp;volume=33&amp;pages=2391-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msw110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsw110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsw110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27297470" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dion-Côté</div> <div class="given-names">A.-M.</div></div>, <div class="name string-name"><div class="surname">Renaut</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Normandeau</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Bernatchez</div> <div class="given-names">L.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">RNA-seq reveals transcriptomic shock involving transposable elements reactivation in hybrids of young lake whitefish species</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">31</div>, <div class="fpage">1188</div>–<div class="lpage">99</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA-seq%20reveals%20transcriptomic%20shock%20involving%20transposable%20elements%20reactivation%20in%20hybrids%20of%20young%20lake%20whitefish%20species&amp;author=A.-M.%20Dion-C%C3%B4t%C3%A9&amp;author=S.%20Renaut&amp;author=E.%20Normandeau&amp;author=L.%20Bernatchez&amp;publication_year=2014&amp;journal=Mol.%20Biol.%20Evol&amp;volume=31&amp;pages=1188-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msu069" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsu069" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsu069"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24505119" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kavembe</div> <div class="given-names">G. D.</div></div>, <div class="name string-name"><div class="surname">Franchini</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Irisarri</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Machado-Schiaffino</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Meyer</div> <div class="given-names">A.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Genomics of adaptation to multiple concurrent stresses: insights from comparative transcriptomics of a Cichlid fish from one of earth’s most extreme environments, the Hypersaline Soda Lake Magadi in Kenya, East Africa</div>. <div class="source">J. Mol. Evol</div> ., <div class="volume">81</div>, <div class="fpage">90</div>–<div class="lpage">109</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomics%20of%20adaptation%20to%20multiple%20concurrent%20stresses%3A%20insights%20from%20comparative%20transcriptomics%20of%20a%20Cichlid%20fish%20from%20one%20of%20earth%E2%80%99s%20most%20extreme%20environments%2C%20the%20Hypersaline%20Soda%20Lake%20Magadi%20in%20Kenya%2C%20East%20Africa&amp;author=G.%20D.%20Kavembe&amp;author=P.%20Franchini&amp;author=I.%20Irisarri&amp;author=G.%20Machado-Schiaffino&amp;author=A.%20Meyer&amp;publication_year=2015&amp;journal=J.%20Mol.%20Evol&amp;volume=81&amp;pages=90-109" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-015-9696-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-015-9696-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-015-9696-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26345661" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kelley</div> <div class="given-names">J. L.</div></div>, <div class="name string-name"><div class="surname">Passow</div> <div class="given-names">C. N.</div></div>, <div class="name string-name"><div class="surname">Plath</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Arias Rodriguez</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Yee</div> <div class="given-names">M. C.</div></div>, <div class="name string-name"><div class="surname">Tobler</div> <div class="given-names">M.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Genomic resources for a model in adaptation and speciation research: characterization of the <em>Poecilia mexicana</em> transcriptome</div>. <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">652</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20resources%20for%20a%20model%20in%20adaptation%20and%20speciation%20research%3A%20characterization%20of%20the%20Poecilia%20mexicana%20transcriptome&amp;author=J.%20L.%20Kelley&amp;author=C.%20N.%20Passow&amp;author=M.%20Plath&amp;author=L.%20Arias%20Rodriguez&amp;author=M.%20C.%20Yee&amp;author=M.%20Tobler&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=652" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-652" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-652" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-652"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23170846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Deyholos</div> <div class="given-names">M. K.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Making the most of drought and salinity transcriptomics</div>. <div class="source">Plant Cell Environ</div> ., <div class="volume">33</div>, <div class="fpage">648</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Making%20the%20most%20of%20drought%20and%20salinity%20transcriptomics&amp;author=M.%20K.%20Deyholos&amp;publication_year=2010&amp;journal=Plant%20Cell%20Environ&amp;volume=33&amp;pages=648-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pce.2010.33.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpce.2010.33.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpce.2010.33.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20002333" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ramachandran</div> <div class="given-names">V. K.</div></div>, <div class="name string-name"><div class="surname">East</div> <div class="given-names">A. K.</div></div>, <div class="name string-name"><div class="surname">Karunakaran</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Downie</div> <div class="given-names">J. A.</div></div>, <div class="name string-name"><div class="surname">Poole</div> <div class="given-names">P. S.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Adaptation of <em>Rhizobium leguminosarum</em> to pea, alfalfa and sugar beet rhizospheres investigated by comparative transcriptomics</div>. <div class="source">Genome Biol</div> ., <div class="volume">12</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adaptation%20of%20Rhizobium%20leguminosarum%20to%20pea%2C%20alfalfa%20and%20sugar%20beet%20rhizospheres%20investigated%20by%20comparative%20transcriptomics&amp;author=V.%20K.%20Ramachandran&amp;author=A.%20K.%20East&amp;author=R.%20Karunakaran&amp;author=J.%20A.%20Downie&amp;author=P.%20S.%20Poole&amp;publication_year=2011&amp;journal=Genome%20Biol&amp;volume=12&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2011-12-10-r106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2011-12-10-r106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2011-12-10-r106"> </span></div> </div></div></div></div></div><div content-id="dsx034-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cheviron</div> <div class="given-names">Z. A.</div></div>, <div class="name string-name"><div class="surname">Connaty</div> <div class="given-names">A. D.</div></div>, <div class="name string-name"><div class="surname">McClelland</div> <div class="given-names">G. B.</div></div>, <div class="name string-name"><div class="surname">Storz</div> <div class="given-names">J. F.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Functional genomics of adaptation to hypoxic cold-stress in high-altitude deer mice: transcriptomic plasticity and thermogenic performance</div> <div class="source">Evolution</div> , <div class="volume">68</div>, <div class="fpage">48</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20genomics%20of%20adaptation%20to%20hypoxic%20cold-stress%20in%20high-altitude%20deer%20mice%3A%20transcriptomic%20plasticity%20and%20thermogenic%20performance&amp;author=Z.%20A.%20Cheviron&amp;author=A.%20D.%20Connaty&amp;author=G.%20B.%20McClelland&amp;author=J.%20F.%20Storz&amp;publication_year=2014&amp;journal=Evolution&amp;volume=68&amp;pages=48-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/evo.2014.68.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fevo.2014.68.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fevo.2014.68.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24102503" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Webster</div> <div class="given-names">T. M. U.</div></div>, <div class="name string-name"><div class="surname">Santos</div> <div class="given-names">E. M.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Global transcriptomic profiling demonstrates induction of oxidative stress and of compensatory cellular stress responses in brown trout exposed to glyphosate and Roundup</div>. <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20transcriptomic%20profiling%20demonstrates%20induction%20of%20oxidative%20stress%20and%20of%20compensatory%20cellular%20stress%20responses%20in%20brown%20trout%20exposed%20to%20glyphosate%20and%20Roundup&amp;author=T.%20M.%20U.%20Webster&amp;author=E.%20M.%20Santos&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1702-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1702-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1702-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25553907" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nelson</div> <div class="given-names">J. S.</div></div></div> <div class="year">2006</div>, <div class="source">Fishes of the World</div> . 
<div class="publisher-name">John Wiley &amp; Sons</div>: 
<div class="publisher-loc">Hoboken, NJ</div>.</div></div></div></div><div content-id="dsx034-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Berra</div> <div class="given-names">T. M.</div></div></div> <div class="year">2001</div>, <div class="source">Freshwater fish distribution</div> . 
<div class="publisher-name">Academic Press</div>: 
<div class="publisher-loc">San Diego, CA</div>.</div></div></div></div><div content-id="dsx034-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Winfield</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Townsend</div> <div class="given-names">C.</div></div></div> <div class="year">1991</div>, The role of cyprinids in ecosystems. In: <div class="person-group"><div class="name string-name"><div class="surname">Winfield</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Nelson</div> <div class="given-names">J.</div></div></div> (eds), <div class="source">Cyprinid Fishes</div> , 
<div class="publisher-name">Springer</div>, <div class="fpage">552</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cyprinid%20Fishes&amp;author=I.%20Winfield&amp;author=C.%20Townsend&amp;author=I.%20Winfield&amp;author=J.%20Nelson&amp;publication_year=1991&amp;book=Cyprinid%20Fishes" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/978-94-011-3092-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2F978-94-011-3092-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2F978-94-011-3092-9"> </span></div> </div></div></div></div></div><div content-id="dsx034-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sterner</div> <div class="given-names">R. W.</div></div>, <div class="name string-name"><div class="surname">George</div> <div class="given-names">N. B.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Carbon, nitrogen, and phosphorous stoichiometry of cyprinid fishes</div>. <div class="source">Ecology</div> , <div class="volume">81</div>, <div class="fpage">127</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Carbon%2C%20nitrogen%2C%20and%20phosphorous%20stoichiometry%20of%20cyprinid%20fishes&amp;author=R.%20W.%20Sterner&amp;author=N.%20B.%20George&amp;publication_year=2000&amp;journal=Ecology&amp;volume=81&amp;pages=127-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1890/0012-9658(2000)081[0127:CNAPSO]2.0.CO;2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1890%2F0012-9658(2000)081[0127:CNAPSO]2.0.CO;2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1890%2F0012-9658(2000)081[0127:CNAPSO]2.0.CO;2"> </span></div> </div></div></div></div></div><div content-id="dsx034-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Power</div> <div class="given-names">M. E.</div></div>, <div class="name string-name"><div class="surname">Matthews</div> <div class="given-names">W. J.</div></div>, <div class="name string-name"><div class="surname">Stewart</div> <div class="given-names">A. J.</div></div></div> <div class="year">1985</div>, 
<div class="article-title">Grazing minnows, piscivorous bass, and stream algae: dynamics of a strong interaction</div>. <div class="source">Ecology</div> , <div class="volume">66</div>, <div class="fpage">1448</div>–<div class="lpage">56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Grazing%20minnows%2C%20piscivorous%20bass%2C%20and%20stream%20algae%3A%20dynamics%20of%20a%20strong%20interaction&amp;author=M.%20E.%20Power&amp;author=W.%20J.%20Matthews&amp;author=A.%20J.%20Stewart&amp;publication_year=1985&amp;journal=Ecology&amp;volume=66&amp;pages=1448-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/1938007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F1938007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F1938007"> </span></div> </div></div></div></div></div><div content-id="dsx034-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Amores</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Force</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Yan</div> <div class="given-names">Y.-L.</div></div></div>, et al.   <div class="year">1998</div>, 
<div class="article-title">Zebrafish hox clusters and vertebrate genome evolution</div>. <div class="source">Science</div> , <div class="volume">282</div>, <div class="fpage">1711</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Zebrafish%20hox%20clusters%20and%20vertebrate%20genome%20evolution&amp;author=A.%20Amores&amp;author=A.%20Force&amp;author=Y.-L.%20Yan&amp;publication_year=1998&amp;journal=Science&amp;volume=282&amp;pages=1711-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.282.5394.1711" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.282.5394.1711" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.282.5394.1711"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9831563" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Postlethwait</div> <div class="given-names">J. H.</div></div>, <div class="name string-name"><div class="surname">Yan</div> <div class="given-names">Y.-L.</div></div>, <div class="name string-name"><div class="surname">Gates</div> <div class="given-names">M. A.</div></div></div>, et al.   <div class="year">1998</div>, 
<div class="article-title">Vertebrate genome evolution and the zebrafish gene map</div>. <div class="source">Nature Genet</div> ., <div class="volume">18</div>, <div class="fpage">345</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Vertebrate%20genome%20evolution%20and%20the%20zebrafish%20gene%20map&amp;author=J.%20H.%20Postlethwait&amp;author=Y.-L.%20Yan&amp;author=M.%20A.%20Gates&amp;publication_year=1998&amp;journal=Nature%20Genet&amp;volume=18&amp;pages=345-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng0498-345" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng0498-345" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng0498-345"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9537416" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Meyer</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Van de Peer</div> <div class="given-names">Y.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">From 2R to 3R: evidence for a fish-specific genome duplication (FSGD)</div>. <div class="source">BioEssays</div> , <div class="volume">27</div>, <div class="fpage">937</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=From%202R%20to%203R%3A%20evidence%20for%20a%20fish-specific%20genome%20duplication%20(FSGD)&amp;author=A.%20Meyer&amp;author=Y.%20Van%20de%20Peer&amp;publication_year=2005&amp;journal=BioEssays&amp;volume=27&amp;pages=937-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1521-1878" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1521-1878" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1521-1878"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16108068" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">David</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Blum</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Feldman</div> <div class="given-names">M. W.</div></div>, <div class="name string-name"><div class="surname">Lavi</div> <div class="given-names">U.</div></div>, <div class="name string-name"><div class="surname">Hillel</div> <div class="given-names">J.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Recent duplication of the common carp (<em>Cyprinus carpio</em> L.) genome as revealed by analyses of microsatellite loci</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">20</div>, <div class="fpage">1425</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recent%20duplication%20of%20the%20common%20carp%20(Cyprinus%20carpio%20L.)%20genome%20as%20revealed%20by%20analyses%20of%20microsatellite%20loci&amp;author=L.%20David&amp;author=S.%20Blum&amp;author=M.%20W.%20Feldman&amp;author=U.%20Lavi&amp;author=J.%20Hillel&amp;publication_year=2003&amp;journal=Mol.%20Biol.%20Evol&amp;volume=20&amp;pages=1425-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msg173" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsg173" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsg173"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12832638" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Machordom</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Doadrio</div> <div class="given-names">I.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Evolutionary history and speciation modes in the cyprinid genus <em>Barbus</em></div>. <div class="source">Proc. Roy. Soc. Lond B Biol. Sci</div> ., <div class="volume">268</div>, <div class="fpage">1297</div>–<div class="lpage">306</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20history%20and%20speciation%20modes%20in%20the%20cyprinid%20genus%20Barbus&amp;author=A.%20Machordom&amp;author=I.%20Doadrio&amp;publication_year=2001&amp;journal=Proc.%20Roy.%20Soc.%20Lond%20B%20Biol.%20Sci&amp;volume=268&amp;pages=1297-306" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rspb.2001.1654" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frspb.2001.1654" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frspb.2001.1654"> </span></div> </div></div></div></div></div><div content-id="dsx034-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ohno</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Muramoto</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Christian</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Atkin</div> <div class="given-names">N. B.</div></div></div> <div class="year">1967</div>, 
<div class="article-title">Diploid-tetraploid relationship among old-world members of the fish family Cyprinidae</div>. <div class="source">Chromosoma</div> , <div class="volume">23</div>, <div class="fpage">1</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diploid-tetraploid%20relationship%20among%20old-world%20members%20of%20the%20fish%20family%20Cyprinidae&amp;author=S.%20Ohno&amp;author=J.%20Muramoto&amp;author=L.%20Christian&amp;author=N.%20B.%20Atkin&amp;publication_year=1967&amp;journal=Chromosoma&amp;volume=23&amp;pages=1-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00293307" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00293307" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00293307"> </span></div> </div></div></div></div></div><div content-id="dsx034-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Sado</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Hirt</div> <div class="given-names">M. V.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Phylogeny and polyploidy: Resolving the classification of cyprinine fishes (Teleostei: Cypriniformes)</div>. <div class="source">Mol. Phylogenet. Evol</div> ., <div class="volume">85</div>, <div class="fpage">97</div>–<div class="lpage">116</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogeny%20and%20polyploidy%3A%20Resolving%20the%20classification%20of%20cyprinine%20fishes%20(Teleostei%3A%20Cypriniformes)&amp;author=L.%20Yang&amp;author=T.%20Sado&amp;author=M.%20V.%20Hirt&amp;publication_year=2015&amp;journal=Mol.%20Phylogenet.%20Evol&amp;volume=85&amp;pages=97-116" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ympev.2015.01.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2015.01.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ympev.2015.01.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25698355" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.-T.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">J.-T.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.-F.</div></div>, <div class="name string-name"><div class="surname">Sun</div> <div class="given-names">X.-W.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Transcriptome analysis reveals the time of the fourth round of genome duplication in common carp (<em>Cyprinus carpio</em>)</div>. <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20analysis%20reveals%20the%20time%20of%20the%20fourth%20round%20of%20genome%20duplication%20in%20common%20carp%20(Cyprinus%20carpio)&amp;author=J.-T.%20Wang&amp;author=J.-T.%20Li&amp;author=X.-F.%20Zhang&amp;author=X.-W.%20Sun&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-96" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-96" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-96"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22424280" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xu</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">X.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Genome sequence and genetic diversity of the common carp, <em>Cyprinus carpio</em></div>. <div class="source">Nature Genet</div> ., <div class="volume">46</div>, <div class="fpage">1212</div>–<div class="lpage">1219</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20and%20genetic%20diversity%20of%20the%20common%20carp%2C%20Cyprinus%20carpio&amp;author=P.%20Xu&amp;author=X.%20Zhang&amp;author=X.%20Wang&amp;publication_year=2014&amp;journal=Nature%20Genet&amp;volume=46&amp;pages=1212-1219" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.3098" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.3098" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.3098"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25240282" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Woods</div> <div class="given-names">T. D.</div></div>, <div class="name string-name"><div class="surname">Buth</div> <div class="given-names">D. G.</div></div></div> <div class="year">1984</div>, 
<div class="article-title">High level of gene silencing in the tetraploid goldfish</div>. <div class="source">Biochem. Syst. Ecol</div> ., <div class="volume">12</div>, <div class="fpage">415</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20level%20of%20gene%20silencing%20in%20the%20tetraploid%20goldfish&amp;author=T.%20D.%20Woods&amp;author=D.%20G.%20Buth&amp;publication_year=1984&amp;journal=Biochem.%20Syst.%20Ecol&amp;volume=12&amp;pages=415-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0305-1978(84)90074-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0305-1978(84)90074-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0305-1978(84)90074-7"> </span></div> </div></div></div></div></div><div content-id="dsx034-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ferris</div> <div class="given-names">S. D.</div></div>, <div class="name string-name"><div class="surname">Whitt</div> <div class="given-names">G. S.</div></div></div> <div class="year">1977</div>, 
<div class="article-title">The evolution of duplicate gene expression in the carp (<em>Cyprinus carpio</em>)</div>. <div class="source">Experientia</div> , <div class="volume">33</div>, <div class="fpage">1299</div>–<div class="lpage">1301</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20duplicate%20gene%20expression%20in%20the%20carp%20(Cyprinus%20carpio)&amp;author=S.%20D.%20Ferris&amp;author=G.%20S.%20Whitt&amp;publication_year=1977&amp;journal=Experientia&amp;volume=33&amp;pages=1299-1301" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF01920144" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01920144" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF01920144"> </span></div> </div></div></div></div></div><div content-id="dsx034-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Briggs</div> <div class="given-names">J. P.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">The zebrafish: a new model organism for integrative physiology</div>. <div class="source">Am. J. Physiol. Regul. Integr. Comp. Physiol</div> ., <div class="volume">282</div>, <div class="fpage">R3</div>–<div class="lpage">R9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20zebrafish%3A%20a%20new%20model%20organism%20for%20integrative%20physiology&amp;author=J.%20P.%20Briggs&amp;publication_year=2002&amp;journal=Am.%20J.%20Physiol.%20Regul.%20Integr.%20Comp.%20Physiol&amp;volume=282&amp;pages=R3-R9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1152/ajpregu.00589.2001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1152%2Fajpregu.00589.2001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1152%2Fajpregu.00589.2001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11742817" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dooley</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Zon</div> <div class="given-names">L. I.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Zebrafish: a model system for the study of human disease</div>. <div class="source">Curr. Opinion Genet. Dev</div> ., <div class="volume">10</div>, <div class="fpage">252</div>–<div class="lpage">56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Zebrafish%3A%20a%20model%20system%20for%20the%20study%20of%20human%20disease&amp;author=K.%20Dooley&amp;author=L.%20I.%20Zon&amp;publication_year=2000&amp;journal=Curr.%20Opinion%20Genet.%20Dev&amp;volume=10&amp;pages=252-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0959-437X(00)00074-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0959-437X(00)00074-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0959-437X(00)00074-5"> </span></div> </div></div></div></div></div><div content-id="dsx034-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lieschke</div> <div class="given-names">G. J.</div></div>, <div class="name string-name"><div class="surname">Currie</div> <div class="given-names">P. D.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Animal models of human disease: zebrafish swim into view</div>. <div class="source">Nat. Rev. Genet</div> ., <div class="volume">8</div>, <div class="fpage">353</div>–<div class="lpage">367</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Animal%20models%20of%20human%20disease%3A%20zebrafish%20swim%20into%20view&amp;author=G.%20J.%20Lieschke&amp;author=P.%20D.%20Currie&amp;publication_year=2007&amp;journal=Nat.%20Rev.%20Genet&amp;volume=8&amp;pages=353-367" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2091" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2091" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2091"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17440532" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Howe</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Clark</div> <div class="given-names">M. D.</div></div>, <div class="name string-name"><div class="surname">Torroja</div> <div class="given-names">C. F.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">The zebrafish reference genome sequence and its relationship to the human genome</div>. <div class="source">Nature</div> , <div class="volume">496</div>, <div class="fpage">498</div>–<div class="lpage">503</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20zebrafish%20reference%20genome%20sequence%20and%20its%20relationship%20to%20the%20human%20genome&amp;author=K.%20Howe&amp;author=M.%20D.%20Clark&amp;author=C.%20F.%20Torroja&amp;publication_year=2013&amp;journal=Nature&amp;volume=496&amp;pages=498-503" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature12111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature12111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature12111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23594743" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ankley</div> <div class="given-names">G. T.</div></div>, <div class="name string-name"><div class="surname">Villeneuve</div> <div class="given-names">D. L.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">The fathead minnow in aquatic toxicology: past, present and future</div>. <div class="source">Aquatic Toxicol</div> ., <div class="volume">78</div>, <div class="fpage">91</div>–<div class="lpage">102</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20fathead%20minnow%20in%20aquatic%20toxicology%3A%20past%2C%20present%20and%20future&amp;author=G.%20T.%20Ankley&amp;author=D.%20L.%20Villeneuve&amp;publication_year=2006&amp;journal=Aquatic%20Toxicol&amp;volume=78&amp;pages=91-102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.aquatox.2006.01.018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.aquatox.2006.01.018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.aquatox.2006.01.018"> </span></div> </div></div></div></div></div><div content-id="dsx034-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Klaper</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Carter</div> <div class="given-names">B. J.</div></div>, <div class="name string-name"><div class="surname">Richter</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Drevnick</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Sandheinrich</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Tillitt</div> <div class="given-names">D.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Use of a 15 k gene microarray to determine gene expression changes in response to acute and chronic methylmercury exposure in the fathead minnow <em>Pimephales promelas</em> Rafinesque</div>. <div class="source">J. Fish Biol</div> ., <div class="volume">72</div>, <div class="fpage">2207</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Use%20of%20a%2015%20k%20gene%20microarray%20to%20determine%20gene%20expression%20changes%20in%20response%20to%20acute%20and%20chronic%20methylmercury%20exposure%20in%20the%20fathead%20minnow%20Pimephales%20promelas%20Rafinesque&amp;author=R.%20Klaper&amp;author=B.%20J.%20Carter&amp;author=C.%20Richter&amp;author=P.%20Drevnick&amp;author=M.%20Sandheinrich&amp;author=D.%20Tillitt&amp;publication_year=2008&amp;journal=J.%20Fish%20Biol&amp;volume=72&amp;pages=2207-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1095-8649.2008.01899.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1095-8649.2008.01899.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1095-8649.2008.01899.x"> </span></div> </div></div></div></div></div><div content-id="dsx034-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Burns</div> <div class="given-names">F. R.</div></div>, <div class="name string-name"><div class="surname">Cogburn</div> <div class="given-names">A. L.</div></div>, <div class="name string-name"><div class="surname">Ankley</div> <div class="given-names">G. T.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Sequencing and de novo draft assemblies of a fathead minnow (<em>Pimephales promelas</em>) reference genome</div>. <div class="source">Environ. Toxicol. Chem.</div> , <div class="volume">35</div>, <div class="fpage">212</div>–<div class="lpage">217</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20and%20de%20novo%20draft%20assemblies%20of%20a%20fathead%20minnow%20(Pimephales%20promelas)%20reference%20genome&amp;author=F.%20R.%20Burns&amp;author=A.%20L.%20Cogburn&amp;author=G.%20T.%20Ankley&amp;publication_year=2016&amp;journal=Environ.%20Toxicol.%20Chem.&amp;volume=35&amp;pages=212-217" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/etc.3186" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fetc.3186" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fetc.3186"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26513338" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="collab">FAO</div> <div class="year">2014</div>, <div class="source">FISHSTAT: Global Aquaculture Production</div> . 
<div class="publisher-name">FAO</div>, 
<div class="publisher-loc">Rome, Italy</div>.<div class="citation-links"><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/NBK221497/ddd00073" class="link link-pub-id" target="_blank">PubMed</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3182704" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ji</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Characterization of common carp transcriptome: <em>de novo</em> sequencing, assembly, annotation and comparative genomics</div>. <div class="source">PloS One</div> , <div class="volume">7</div>, <div class="fpage">1</div>–<div class="lpage">9</div>.</div></div></div></div><div content-id="dsx034-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Z.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">De novo assembly and characterization of the spleen transcriptome of common carp (<em>Cyprinus carpio</em>) using illumina paired-end sequencing</div>. <div class="source">Fish Shellfish Immunol</div> .</div></div></div></div><div content-id="dsx034-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dekay</div> <div class="given-names">J. E.</div></div></div> <div class="year">1842</div>, <div class="source">Zoology of New-York, or, the New-York fauna; comprising detailed descriptions of all the animals hitherto observed within the state of New-York, with brief notices of those occasionally found near its borders, and accompanied by appropriate illustrations. Part IV. Fishes</div> . 
<div class="publisher-name">W. &amp; A. White &amp; J. Visscher</div>: 
<div class="publisher-loc">Albany, NY</div>.</div></div></div></div><div content-id="dsx034-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">McDonald</div> <div class="given-names">M.</div></div></div> <div class="year">1893</div>, 
<div class="article-title">Report of the commissioner of fish and fisheries for the fiscal years 1889-90 and 1890-91. Part XVII</div>. <div class="source">U.S. Commission of Fish and Fisheries. Washington, DC</div> , <div class="fpage">1</div>–<div class="lpage">96</div>.</div></div></div></div><div content-id="dsx034-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Walters</div> <div class="given-names">D. M.</div></div>, <div class="name string-name"><div class="surname">Blum</div> <div class="given-names">M. J.</div></div>, <div class="name string-name"><div class="surname">Rashleigh</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Freeman</div> <div class="given-names">B. J.</div></div>, <div class="name string-name"><div class="surname">Porter</div> <div class="given-names">B. A.</div></div>, <div class="name string-name"><div class="surname">Burkhead</div> <div class="given-names">N. M.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Red shiner invasion and hybridization with blacktail shiner in the upper Coosa River, USA</div>. <div class="source">Biol. Invas</div> ., <div class="volume">10</div>, <div class="fpage">1229</div>–<div class="lpage">1242</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Red%20shiner%20invasion%20and%20hybridization%20with%20blacktail%20shiner%20in%20the%20upper%20Coosa%20River%2C%20USA&amp;author=D.%20M.%20Walters&amp;author=M.%20J.%20Blum&amp;author=B.%20Rashleigh&amp;author=B.%20J.%20Freeman&amp;author=B.%20A.%20Porter&amp;author=N.%20M.%20Burkhead&amp;publication_year=2008&amp;journal=Biol.%20Invas&amp;volume=10&amp;pages=1229-1242" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10530-007-9198-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10530-007-9198-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10530-007-9198-6"> </span></div> </div></div></div></div></div><div content-id="dsx034-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marsh-Matthews</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Matthews</div> <div class="given-names">W. J.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Spatial variation in relative abundance of a widespread, numerically dominant fish species and its effect on fish assemblage structure</div>. <div class="source">Oecologia</div> , <div class="volume">125</div>, <div class="fpage">283</div>–<div class="lpage">292</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Spatial%20variation%20in%20relative%20abundance%20of%20a%20widespread%2C%20numerically%20dominant%20fish%20species%20and%20its%20effect%20on%20fish%20assemblage%20structure&amp;author=E.%20Marsh-Matthews&amp;author=W.%20J.%20Matthews&amp;publication_year=2000&amp;journal=Oecologia&amp;volume=125&amp;pages=283-292" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s004420000452" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004420000452" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs004420000452"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24595839" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hoagstrom</div> <div class="given-names">C. W.</div></div>, <div class="name string-name"><div class="surname">DeWitte</div> <div class="given-names">A. C.</div></div>, <div class="name string-name"><div class="surname">Gosch</div> <div class="given-names">N. J.</div></div>, <div class="name string-name"><div class="surname">Berry</div> <div class="given-names">C. R.</div>Jr</div></div> <div class="year">2007</div>, 
<div class="article-title">Historical fish assemblage flux in the Cheyenne River below Angostura Dam</div>. <div class="source">J. Freshwater Ecol</div> ., <div class="volume">22</div>, <div class="fpage">219</div>–<div class="lpage">29</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Historical%20fish%20assemblage%20flux%20in%20the%20Cheyenne%20River%20below%20Angostura%20Dam&amp;author=C.%20W.%20Hoagstrom&amp;author=A.%20C.%20DeWitte&amp;author=N.%20J.%20Gosch&amp;author=C.%20R.%20Berry&amp;publication_year=2007&amp;journal=J.%20Freshwater%20Ecol&amp;volume=22&amp;pages=219-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/02705060.2007.9665041" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F02705060.2007.9665041" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F02705060.2007.9665041"> </span></div> </div></div></div></div></div><div content-id="dsx034-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Amsterdam</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Nissen</div> <div class="given-names">R. M.</div></div>, <div class="name string-name"><div class="surname">Sun</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Swindell</div> <div class="given-names">E. C.</div></div>, <div class="name string-name"><div class="surname">Farrington</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Hopkins</div> <div class="given-names">N.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Identification of 315 genes essential for early zebrafish development</div>. <div class="source">Proc. Natl. Acad. Sci USA</div> , <div class="volume">101</div>, <div class="fpage">12792</div>–<div class="lpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20315%20genes%20essential%20for%20early%20zebrafish%20development&amp;author=A.%20Amsterdam&amp;author=R.%20M.%20Nissen&amp;author=Z.%20Sun&amp;author=E.%20C.%20Swindell&amp;author=S.%20Farrington&amp;author=N.%20Hopkins&amp;publication_year=2004&amp;journal=Proc.%20Natl.%20Acad.%20Sci%20USA&amp;volume=101&amp;pages=12792-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0403929101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0403929101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0403929101"> </span></div> </div></div></div></div></div><div content-id="dsx034-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">W.-H.</div></div>, <div class="name string-name"><div class="surname">Minguez</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Lercher</div> <div class="given-names">M. J.</div></div>, <div class="name string-name"><div class="surname">Bork</div> <div class="given-names">P.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">OGEE: an online gene essentiality database</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">D901</div>–<div class="lpage">D906</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=OGEE%3A%20an%20online%20gene%20essentiality%20database&amp;author=W.-H.%20Chen&amp;author=P.%20Minguez&amp;author=M.%20J.%20Lercher&amp;author=P.%20Bork&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=D901-D906" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr986" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr986" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr986"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22075992" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Flicek</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Amode</div> <div class="given-names">M. R.</div></div>, <div class="name string-name"><div class="surname">Barrell</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Ensembl 2014</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">D749</div>–<div class="lpage">755</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ensembl%202014&amp;author=P.%20Flicek&amp;author=M.%20R.%20Amode&amp;author=D.%20Barrell&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=D749-755" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1196" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1196" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1196"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24316576" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hahn</div> <div class="given-names">M. W.</div></div>, <div class="name string-name"><div class="surname">Kern</div> <div class="given-names">A. D.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Comparative genomics of centrality and essentiality in three eukaryotic protein-interaction networks</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">22</div>, <div class="fpage">803</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomics%20of%20centrality%20and%20essentiality%20in%20three%20eukaryotic%20protein-interaction%20networks&amp;author=M.%20W.%20Hahn&amp;author=A.%20D.%20Kern&amp;publication_year=2005&amp;journal=Mol.%20Biol.%20Evol&amp;volume=22&amp;pages=803-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msi072" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsi072" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsi072"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15616139" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Innan</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kondrashov</div> <div class="given-names">F.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">The evolution of gene duplications: classifying and distinguishing between models</div>. <div class="source">Nat. Rev. Genet</div> ., <div class="volume">11</div>, <div class="fpage">97</div>–<div class="lpage">108</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20gene%20duplications%3A%20classifying%20and%20distinguishing%20between%20models&amp;author=H.%20Innan&amp;author=F.%20Kondrashov&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Genet&amp;volume=11&amp;pages=97-108" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2689" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2689" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2689"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20051986" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chain</div> <div class="given-names">F. J.</div></div>, <div class="name string-name"><div class="surname">Dushoff</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Evans</div> <div class="given-names">B. J.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">The odds of duplicate gene persistence after polyploidization</div>. <div class="source">BMC Genomics</div> , <div class="volume">12</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20odds%20of%20duplicate%20gene%20persistence%20after%20polyploidization&amp;author=F.%20J.%20Chain&amp;author=J.%20Dushoff&amp;author=B.%20J.%20Evans&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-599" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-599" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-599"> </span></div> </div></div></div></div></div><div content-id="dsx034-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bolger</div> <div class="given-names">A. M.</div></div>, <div class="name string-name"><div class="surname">Lohse</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Usadel</div> <div class="given-names">B.</div></div></div> <div class="year">2014</div>, Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics, btu170.</div></div></div></div><div content-id="dsx034-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Grabherr</div> <div class="given-names">M. G.</div></div>, <div class="name string-name"><div class="surname">Haas</div> <div class="given-names">B. J.</div></div>, <div class="name string-name"><div class="surname">Yassour</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome</div>. <div class="source">Nat. Biotechnol</div> ., <div class="volume">29</div>, <div class="fpage">644</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Full-length%20transcriptome%20assembly%20from%20RNA-Seq%20data%20without%20a%20reference%20genome&amp;author=M.%20G.%20Grabherr&amp;author=B.%20J.%20Haas&amp;author=M.%20Yassour&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol&amp;volume=29&amp;pages=644-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1883" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1883" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1883"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Haas</div> <div class="given-names">B. J.</div></div>, <div class="name string-name"><div class="surname">Papanicolaou</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Yassour</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis</div>. <div class="source">Nat. Protocols</div> , <div class="volume">8</div>, <div class="fpage">1494</div>–<div class="lpage">512</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20transcript%20sequence%20reconstruction%20from%20RNA-seq%20using%20the%20Trinity%20platform%20for%20reference%20generation%20and%20analysis&amp;author=B.%20J.%20Haas&amp;author=A.%20Papanicolaou&amp;author=M.%20Yassour&amp;publication_year=2013&amp;journal=Nat.%20Protocols&amp;volume=8&amp;pages=1494-512" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2013.084" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2013.084" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2013.084"> </span></div> </div></div></div></div></div><div content-id="dsx034-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pruesse</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Quast</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Knittel</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2007</div>, 
<div class="article-title">SILVA: a comprehensive online resource for quality checked and aligned ribosomal RNA sequence data compatible with ARB</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">35</div>, <div class="fpage">7188</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SILVA%3A%20a%20comprehensive%20online%20resource%20for%20quality%20checked%20and%20aligned%20ribosomal%20RNA%20sequence%20data%20compatible%20with%20ARB&amp;author=E.%20Pruesse&amp;author=C.%20Quast&amp;author=K.%20Knittel&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=7188-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm864" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm864" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm864"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17947321" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Conesa</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Gotz</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Garcia-Gomez</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Terol</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Talon</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Robles</div> <div class="given-names">M.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research</div>. <div class="source">Bioinformatics</div> , <div class="volume">21</div>, <div class="fpage">3674</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Blast2GO%3A%20a%20universal%20tool%20for%20annotation%2C%20visualization%20and%20analysis%20in%20functional%20genomics%20research&amp;author=A.%20Conesa&amp;author=S.%20Gotz&amp;author=J.%20M.%20Garcia-Gomez&amp;author=J.%20Terol&amp;author=M.%20Talon&amp;author=M.%20Robles&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=3674-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti610" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti610" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti610"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16081474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Langmead</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Fast gapped-read alignment with Bowtie 2</div>. <div class="source">Nat. Methods</div> , <div class="volume">9</div>, <div class="fpage">357</div>–<div class="lpage">59</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;author=B.%20Langmead&amp;author=S.%20L.%20Salzberg&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=357-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Dewey</div> <div class="given-names">C. N.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome</div>. <div class="source">BMC Bioinform</div> ., <div class="volume">12</div>, <div class="fpage">323</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RSEM%3A%20accurate%20transcript%20quantification%20from%20RNA-Seq%20data%20with%20or%20without%20a%20reference%20genome&amp;author=B.%20Li&amp;author=C.%20N.%20Dewey&amp;publication_year=2011&amp;journal=BMC%20Bioinform&amp;volume=12&amp;pages=323" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-12-323" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-12-323" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-12-323"> </span></div> </div></div></div></div></div><div content-id="dsx034-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Langfelder</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Horvath</div> <div class="given-names">S.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">WGCNA: an R package for weighted correlation network analysis</div>. <div class="source">BMC Bioinform</div> ., <div class="volume">9</div>, <div class="fpage">559</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=WGCNA%3A%20an%20R%20package%20for%20weighted%20correlation%20network%20analysis&amp;author=P.%20Langfelder&amp;author=S.%20Horvath&amp;publication_year=2008&amp;journal=BMC%20Bioinform&amp;volume=9&amp;pages=559" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-9-559" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-9-559" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-9-559"> </span></div> </div></div></div></div></div><div content-id="dsx034-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mi</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Muruganujan</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Thomas</div> <div class="given-names">P. D.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">PANTHER in 2013: modeling the evolution of gene function, and other gene attributes, in the context of phylogenetic trees</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">D377</div>–<div class="lpage">386</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PANTHER%20in%202013%3A%20modeling%20the%20evolution%20of%20gene%20function%2C%20and%20other%20gene%20attributes%2C%20in%20the%20context%20of%20phylogenetic%20trees&amp;author=H.%20Mi&amp;author=A.%20Muruganujan&amp;author=P.%20D.%20Thomas&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=D377-386" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks1118" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1118" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks1118"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23193289" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Supek</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Bosnjak</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Skunca</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Smuc</div> <div class="given-names">T.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">REVIGO summarizes and visualizes long lists of gene ontology terms</div>. <div class="source">PLoS One</div> , <div class="volume">6</div>, <div class="fpage">e21800</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=REVIGO%20summarizes%20and%20visualizes%20long%20lists%20of%20gene%20ontology%20terms&amp;author=F.%20Supek&amp;author=M.%20Bosnjak&amp;author=N.%20Skunca&amp;author=T.%20Smuc&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e21800" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0021800" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0021800" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0021800"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21789182" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">van Dijk</div> <div class="given-names">E. L.</div></div>, <div class="name string-name"><div class="surname">Auger</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Jaszczyszyn</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Thermes</div> <div class="given-names">C.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Ten years of next-generation sequencing technology</div>. <div class="source">Trends Genet</div> ., <div class="volume">30</div>, <div class="fpage">418</div>–<div class="lpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ten%20years%20of%20next-generation%20sequencing%20technology&amp;author=E.%20L.%20van%20Dijk&amp;author=H.%20Auger&amp;author=Y.%20Jaszczyszyn&amp;author=C.%20Thermes&amp;publication_year=2014&amp;journal=Trends%20Genet&amp;volume=30&amp;pages=418-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2014.07.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2014.07.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2014.07.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25108476" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bernatchez</div> <div class="given-names">L.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">On the maintenance of genetic variation and adaptation to environmental change: considerations from population genomics in fishes</div>. <div class="source">J. Fish Biol</div> ., <div class="volume">89</div>, <div class="fpage">2519</div>–<div class="lpage">56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=On%20the%20maintenance%20of%20genetic%20variation%20and%20adaptation%20to%20environmental%20change%3A%20considerations%20from%20population%20genomics%20in%20fishes&amp;author=L.%20Bernatchez&amp;publication_year=2016&amp;journal=J.%20Fish%20Biol&amp;volume=89&amp;pages=2519-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jfb.2016.89.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjfb.2016.89.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjfb.2016.89.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27687146" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B76" class="ref-content "><div class="label -label">76</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Martin</div> <div class="given-names">J. A.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Next-generation transcriptome assembly</div>. <div class="source">Nat. Rev. Genet</div> ., <div class="volume">12</div>, <div class="fpage">671</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next-generation%20transcriptome%20assembly&amp;author=J.%20A.%20Martin&amp;author=Z.%20Wang&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet&amp;volume=12&amp;pages=671-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3068" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3068" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3068"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21897427" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B77" class="ref-content "><div class="label -label">77</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Fillmore</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Bai</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Evaluation of de novo transcriptome assemblies from RNA-Seq data</div>. <div class="source">Genome Biol</div> ., <div class="volume">15</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evaluation%20of%20de%20novo%20transcriptome%20assemblies%20from%20RNA-Seq%20data&amp;author=B.%20Li&amp;author=N.%20Fillmore&amp;author=Y.%20Bai&amp;publication_year=2014&amp;journal=Genome%20Biol&amp;volume=15&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2014-15-1-r1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2014-15-1-r1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2014-15-1-r1"> </span></div> </div></div></div></div></div><div content-id="dsx034-B78" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B78" class="ref-content "><div class="label -label">78</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">O'Neil</div> <div class="given-names">S. T.</div></div>, <div class="name string-name"><div class="surname">Emrich</div> <div class="given-names">S. J.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Assessing De Novo transcriptome assembly metrics for consistency and utility</div>. <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">465</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessing%20De%20Novo%20transcriptome%20assembly%20metrics%20for%20consistency%20and%20utility&amp;author=S.%20T.%20O'Neil&amp;author=S.%20J.%20Emrich&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=465" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-465" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-465" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-465"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23837739" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B79" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B79" class="ref-content "><div class="label -label">79</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lopez-Maestre</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Brinza</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Marchet</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">SNP calling from RNA-seq data without a reference genome: identification, quantification, differential analysis and impact on the protein sequence</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">44</div>, <div class="fpage">e148</div>–<div class="lpage">e148</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20calling%20from%20RNA-seq%20data%20without%20a%20reference%20genome%3A%20identification%2C%20quantification%2C%20differential%20analysis%20and%20impact%20on%20the%20protein%20sequence&amp;author=H.%20Lopez-Maestre&amp;author=L.%20Brinza&amp;author=C.%20Marchet&amp;publication_year=2016&amp;journal=Nucleic%20Acids%20Res&amp;volume=44&amp;pages=e148-e148" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27458203" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B80" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B80" class="ref-content "><div class="label -label">80</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">DeWoody</div> <div class="given-names">J. A.</div></div>, <div class="name string-name"><div class="surname">Abts</div> <div class="given-names">K. C.</div></div>, <div class="name string-name"><div class="surname">Fahey</div> <div class="given-names">A. L.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Of contigs and quagmires: next-generation sequencing pitfalls associated with transcriptomic studies</div>. <div class="source">Mol. Ecol. Resour</div> ., <div class="volume">13</div>, <div class="fpage">551</div>–<div class="lpage">58</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Of%20contigs%20and%20quagmires%3A%20next-generation%20sequencing%20pitfalls%20associated%20with%20transcriptomic%20studies&amp;author=J.%20A.%20DeWoody&amp;author=K.%20C.%20Abts&amp;author=A.%20L.%20Fahey&amp;publication_year=2013&amp;journal=Mol.%20Ecol.%20Resour&amp;volume=13&amp;pages=551-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/men.2013.13.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmen.2013.13.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmen.2013.13.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23615313" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B81" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B81" class="ref-content "><div class="label -label">81</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wolf</div> <div class="given-names">J. B.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Principles of transcriptome analysis and gene expression quantification: an RNA-seq tutorial</div>. <div class="source">Mol. Ecol. Resour</div> ., <div class="volume">13</div>, <div class="fpage">559</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Principles%20of%20transcriptome%20analysis%20and%20gene%20expression%20quantification%3A%20an%20RNA-seq%20tutorial&amp;author=J.%20B.%20Wolf&amp;publication_year=2013&amp;journal=Mol.%20Ecol.%20Resour&amp;volume=13&amp;pages=559-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/men.2013.13.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmen.2013.13.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmen.2013.13.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23621713" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B82" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B82" class="ref-content "><div class="label -label">82</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vijay</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Poelstra</div> <div class="given-names">J. W.</div></div>, <div class="name string-name"><div class="surname">Künstner</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Wolf</div> <div class="given-names">J. B.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Challenges and strategies in transcriptome assembly and differential gene expression quantification. A comprehensive in silico assessment of RNA‐seq experiments</div>. <div class="source">Mol. Ecol</div> ., <div class="volume">22</div>, <div class="fpage">620</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Challenges%20and%20strategies%20in%20transcriptome%20assembly%20and%20differential%20gene%20expression%20quantification.%20A%20comprehensive%20in%20silico%20assessment%20of%20RNA%E2%80%90seq%20experiments&amp;author=N.%20Vijay&amp;author=J.%20W.%20Poelstra&amp;author=A.%20K%C3%BCnstner&amp;author=J.%20B.%20Wolf&amp;publication_year=2013&amp;journal=Mol.%20Ecol&amp;volume=22&amp;pages=620-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.12014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.12014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.12014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22998089" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B83" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B83" class="ref-content "><div class="label -label">83</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Aravind</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Watanabe</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Lipman</div> <div class="given-names">D. J.</div></div>, <div class="name string-name"><div class="surname">Koonin</div> <div class="given-names">E. V.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Lineage-specific loss and divergence of functionally linked genes in eukaryotes</div>. <div class="source">Proc. Natl. Acad. Sci</div> ., <div class="volume">97</div>, <div class="fpage">11319</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lineage-specific%20loss%20and%20divergence%20of%20functionally%20linked%20genes%20in%20eukaryotes&amp;author=L.%20Aravind&amp;author=H.%20Watanabe&amp;author=D.%20J.%20Lipman&amp;author=E.%20V.%20Koonin&amp;publication_year=2000&amp;journal=Proc.%20Natl.%20Acad.%20Sci&amp;volume=97&amp;pages=11319-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.200346997" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.200346997" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.200346997"> </span></div> </div></div></div></div></div><div content-id="dsx034-B84" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B84" class="ref-content "><div class="label -label">84</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stein</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Caccamo</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Laird</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Leptin</div> <div class="given-names">M.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Conservation and divergence of gene families encoding components of innate immune response systems in zebrafish</div>. <div class="source">Genome Biol</div> ., <div class="volume">8</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conservation%20and%20divergence%20of%20gene%20families%20encoding%20components%20of%20innate%20immune%20response%20systems%20in%20zebrafish&amp;author=C.%20Stein&amp;author=M.%20Caccamo&amp;author=G.%20Laird&amp;author=M.%20Leptin&amp;publication_year=2007&amp;journal=Genome%20Biol&amp;volume=8&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2007-8-11-r251" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2007-8-11-r251" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2007-8-11-r251"> </span></div> </div></div></div></div></div><div content-id="dsx034-B85" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B85" class="ref-content "><div class="label -label">85</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Small</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Bassham</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Catchen</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">The genome of the Gulf pipefish enables understanding of evolutionary innovations</div>. <div class="source">Genome Biol</div> ., <div class="volume">17</div>, <div class="fpage">258</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20Gulf%20pipefish%20enables%20understanding%20of%20evolutionary%20innovations&amp;author=C.%20Small&amp;author=S.%20Bassham&amp;author=J.%20Catchen&amp;publication_year=2016&amp;journal=Genome%20Biol&amp;volume=17&amp;pages=258" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-016-1126-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-016-1126-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-016-1126-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27993155" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B86" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B86" class="ref-content "><div class="label -label">86</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Di Bella</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Bao</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Gloor</div> <div class="given-names">G. B.</div></div>, <div class="name string-name"><div class="surname">Burton</div> <div class="given-names">J. P.</div></div>, <div class="name string-name"><div class="surname">Reid</div> <div class="given-names">G.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">High throughput sequencing methods and analysis for microbiome research</div>. <div class="source">J. Microbiol. Methods</div> , <div class="volume">95</div>, <div class="fpage">401</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20throughput%20sequencing%20methods%20and%20analysis%20for%20microbiome%20research&amp;author=J.%20M.%20Di%20Bella&amp;author=Y.%20Bao&amp;author=G.%20B.%20Gloor&amp;author=J.%20P.%20Burton&amp;author=G.%20Reid&amp;publication_year=2013&amp;journal=J.%20Microbiol.%20Methods&amp;volume=95&amp;pages=401-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mimet.2013.08.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mimet.2013.08.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mimet.2013.08.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24029734" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B87" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B87" class="ref-content "><div class="label -label">87</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Coco</div> <div class="given-names">J. R.</div></div>, <div class="name string-name"><div class="surname">Flemington</div> <div class="given-names">E. K.</div></div>, <div class="name string-name"><div class="surname">Taylor</div> <div class="given-names">C. M.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">PARSES: a pipeline for analysis of RNA-Seq exogenous sequences</div>. <div class="source">BICoB</div> , <div class="fpage">196</div>–<div class="lpage">200</div>.</div></div></div></div><div content-id="dsx034-B88" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B88" class="ref-content "><div class="label -label">88</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dickerson</div> <div class="given-names">J. E.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Robertson</div> <div class="given-names">D. L.</div></div>, <div class="name string-name"><div class="surname">Hentges</div> <div class="given-names">K. E.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Defining the role of essential genes in human disease</div>. <div class="source">PLoS One</div> , <div class="volume">6</div>, <div class="fpage">e27368</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Defining%20the%20role%20of%20essential%20genes%20in%20human%20disease&amp;author=J.%20E.%20Dickerson&amp;author=A.%20Zhu&amp;author=D.%20L.%20Robertson&amp;author=K.%20E.%20Hentges&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e27368" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0027368" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0027368" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0027368"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22096564" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B89" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B89" class="ref-content "><div class="label -label">89</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Luo</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Lin</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Gao</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">C.-T.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">R.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">DEG 10, an update of the database of essential genes that includes both protein-coding genes and noncoding genomic elements</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">D574</div>–<div class="lpage">D580</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DEG%2010%2C%20an%20update%20of%20the%20database%20of%20essential%20genes%20that%20includes%20both%20protein-coding%20genes%20and%20noncoding%20genomic%20elements&amp;author=H.%20Luo&amp;author=Y.%20Lin&amp;author=F.%20Gao&amp;author=C.-T.%20Zhang&amp;author=R.%20Zhang&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=D574-D580" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1131" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1131" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1131"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24243843" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B90" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B90" class="ref-content "><div class="label -label">90</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Plough</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Shin</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Hedgecock</div> <div class="given-names">D.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Genetic inviability is a major driver of type‐III survivorship in experimental families of a highly fecund marine bivalve</div>. <div class="source">Mol. Ecol</div> .</div></div></div></div><div content-id="dsx034-B91" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B91" class="ref-content "><div class="label -label">91</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">McCurley</div> <div class="given-names">A. T.</div></div>, <div class="name string-name"><div class="surname">Callard</div> <div class="given-names">G. V.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Characterization of housekeeping genes in zebrafish: male-female differences and effects of tissue type, developmental stage and chemical treatment</div>. <div class="source">BMC Mol. Biol</div> ., <div class="volume">9</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20housekeeping%20genes%20in%20zebrafish%3A%20male-female%20differences%20and%20effects%20of%20tissue%20type%2C%20developmental%20stage%20and%20chemical%20treatment&amp;author=A.%20T.%20McCurley&amp;author=G.%20V.%20Callard&amp;publication_year=2008&amp;journal=BMC%20Mol.%20Biol&amp;volume=9&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2199-9-102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2199-9-102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2199-9-102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18177499" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B92" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B92" class="ref-content "><div class="label -label">92</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Saitoh</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Sado</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Doosey</div> <div class="given-names">M. H.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Evidence from mitochondrial genomics supports the lower Mesozoic of South Asia as the time and place of basal divergence of cypriniform fishes (Actinopterygii: Ostariophysi)</div>. <div class="source">Zool. J. Linnean Soc</div> ., <div class="volume">161</div>, <div class="fpage">633</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20from%20mitochondrial%20genomics%20supports%20the%20lower%20Mesozoic%20of%20South%20Asia%20as%20the%20time%20and%20place%20of%20basal%20divergence%20of%20cypriniform%20fishes%20(Actinopterygii%3A%20Ostariophysi)&amp;author=K.%20Saitoh&amp;author=T.%20Sado&amp;author=M.%20H.%20Doosey&amp;publication_year=2011&amp;journal=Zool.%20J.%20Linnean%20Soc&amp;volume=161&amp;pages=633-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/zoj.2011.161.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fzoj.2011.161.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fzoj.2011.161.issue-3"> </span></div> </div></div></div></div></div><div content-id="dsx034-B93" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B93" class="ref-content "><div class="label -label">93</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Postlethwait</div> <div class="given-names">J. H.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">The zebrafish genome in context: ohnologs gone missing</div>. <div class="source">J. Exp. Zool. B Mol. Dev. Evol</div> ., <div class="volume">308</div>, <div class="fpage">563</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20zebrafish%20genome%20in%20context%3A%20ohnologs%20gone%20missing&amp;author=J.%20H.%20Postlethwait&amp;publication_year=2007&amp;journal=J.%20Exp.%20Zool.%20B%20Mol.%20Dev.%20Evol&amp;volume=308&amp;pages=563-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1552-5015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1552-5015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1552-5015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17068775" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B94" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B94" class="ref-content "><div class="label -label">94</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ferris</div> <div class="given-names">S. D.</div></div>, <div class="name string-name"><div class="surname">Whitt</div> <div class="given-names">G. S.</div></div></div> <div class="year">1977</div>, 
<div class="article-title">Loss of duplicate gene expression after polyploidisation</div>. <div class="source">Nature</div> , <div class="volume">265</div>, <div class="fpage">258</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Loss%20of%20duplicate%20gene%20expression%20after%20polyploidisation&amp;author=S.%20D.%20Ferris&amp;author=G.%20S.%20Whitt&amp;publication_year=1977&amp;journal=Nature&amp;volume=265&amp;pages=258-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/265258a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F265258a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F265258a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/834271" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B95" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B95" class="ref-content "><div class="label -label">95</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Larhammar</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Risinger</div> <div class="given-names">C.</div></div></div> <div class="year">1994</div>, 
<div class="article-title">Molecular genetic aspects of tetraploidy in the common carp Cyprinus carpio</div>. <div class="source">Mol. Phylogenet. Evol</div> ., <div class="volume">3</div>, <div class="fpage">59</div>–<div class="lpage">68</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20genetic%20aspects%20of%20tetraploidy%20in%20the%20common%20carp%20Cyprinus%20carpio&amp;author=D.%20Larhammar&amp;author=C.%20Risinger&amp;publication_year=1994&amp;journal=Mol.%20Phylogenet.%20Evol&amp;volume=3&amp;pages=59-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/mpev.1994.1007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fmpev.1994.1007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fmpev.1994.1007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8025730" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B96" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B96" class="ref-content "><div class="label -label">96</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ohno</div> <div class="given-names">S.</div></div></div> <div class="year">1970</div>, <div class="source">Evolution by Gene Duplication</div> . 
<div class="publisher-name">Springer</div>: 
<div class="publisher-loc">New York, NY</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20by%20Gene%20Duplication&amp;author=S.%20Ohno&amp;publication_year=1970&amp;book=Evolution%20by%20Gene%20Duplication" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/978-3-642-86659-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2F978-3-642-86659-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2F978-3-642-86659-3"> </span></div> </div></div></div></div></div><div content-id="dsx034-B97" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B97" class="ref-content "><div class="label -label">97</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Berthelot</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Brunet</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Chalopin</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">The rainbow trout genome provides novel insights into evolution after whole-genome duplication in vertebrates</div>. <div class="source">Nat. Commun</div> ., <div class="volume">5</div>, <div class="fpage">3657</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20rainbow%20trout%20genome%20provides%20novel%20insights%20into%20evolution%20after%20whole-genome%20duplication%20in%20vertebrates&amp;author=C.%20Berthelot&amp;author=F.%20Brunet&amp;author=D.%20Chalopin&amp;publication_year=2014&amp;journal=Nat.%20Commun&amp;volume=5&amp;pages=3657" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms4657" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms4657" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms4657"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24755649" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx034-B98" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx034-B98" class="ref-content "><div class="label -label">98</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Timusk</div> <div class="given-names">E. R.</div></div>, <div class="name string-name"><div class="surname">Ferguson</div> <div class="given-names">M. M.</div></div>, <div class="name string-name"><div class="surname">Moghadam</div> <div class="given-names">H. K.</div></div>, <div class="name string-name"><div class="surname">Norman</div> <div class="given-names">J. D.</div></div>, <div class="name string-name"><div class="surname">Wilson</div> <div class="given-names">C. C.</div></div>, <div class="name string-name"><div class="surname">Danzmann</div> <div class="given-names">R. G.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Genome evolution in the fish family salmonidae: generation of a brook charr genetic map and comparisons among charrs (Arctic charr and brook charr) with rainbow trout</div>. <div class="source">BMC Genet</div> ., <div class="volume">12</div>, <div class="fpage">1</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20evolution%20in%20the%20fish%20family%20salmonidae%3A%20generation%20of%20a%20brook%20charr%20genetic%20map%20and%20comparisons%20among%20charrs%20(Arctic%20charr%20and%20brook%20charr)%20with%20rainbow%20trout&amp;author=E.%20R.%20Timusk&amp;author=M.%20M.%20Ferguson&amp;author=H.%20K.%20Moghadam&amp;author=J.%20D.%20Norman&amp;author=C.%20C.%20Wilson&amp;author=R.%20G.%20Danzmann&amp;publication_year=2011&amp;journal=BMC%20Genet&amp;volume=12&amp;pages=1-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2156-12-68" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2156-12-68" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2156-12-68"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21205287" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4106313&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx034/3/supplementarydata_dsx034.xlsx?Expires=2147483647&amp;Signature=jZVn~I2JY8wrwNW2b2M3lDDIVKAlTUBD9SjHqxIN~bpSot8oZy4L1aIQlOxBJtCotU2197loDrnwre~22Me8F-~wXMIvHyWVltrAEne5slSO2Oud1RMbILcYKf3kRtCgnybZe55DAxKxPqfZ1l3DrV7rlmdhHsLWERTcf2KbDO8rDi-VAHI4zsCwMZRP1j5Ck6q-TjzZpu5cAnubHFLE2MbToJj~n8zGivxn~7u~Ji60eOqsC3GEoJrqsp6W9FipcrTIonY-CUqW3lgXItWB9XWnOh6RyyolP3vBulH6a~wcXjTaMqCQq81hqRS9BL~ajgd6dQfgAiVzymec-V6Bjg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - xlsx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">886</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx034"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4106313"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f1%2f11%2f4106313">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F1%2F11%2F4106313" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4106313'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Comparative transcriptomics of cyprinid minnows and carp in a common wild setting: a resource for ecological genomics in freshwater communities" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx034");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f11%2f4106313">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

