

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/6/439/2462479","@type":"ScholarlyArticle","name":"AnABlast: a new  in silico  strategy for the genome-wide search of novel genes and fossil regions","datePublished":"2015-10-21","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2015-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv025","keywords":["<italic>Schizosaccharomyces pombe</italic>","new genes","<italic>in silico</italic> annotation tool","genome evolution","fossil DNA sequences"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/6/439/2462479","author":[{"name":"Jimenez, Juan","affiliation":"1  Centro Andaluz de Biología del Desarrollo ,  Universidad Pablo de Olavide de Sevilla/CSIC ,  Sevilla ,  Spain","@type":"Person"},{"name":"Duncan, Caia D. S.","affiliation":"2  Department of Biochemistry ,  University of Cambridge ,  Cambridge ,  UK","@type":"Person"},{"name":"Gallardo, María","affiliation":"1  Centro Andaluz de Biología del Desarrollo ,  Universidad Pablo de Olavide de Sevilla/CSIC ,  Sevilla ,  Spain","@type":"Person"},{"name":"Mata, Juan","affiliation":"2  Department of Biochemistry ,  University of Cambridge ,  Cambridge ,  UK","@type":"Person"},{"name":"Perez-Pulido, Antonio J.","affiliation":"1  Centro Andaluz de Biología del Desarrollo ,  Universidad Pablo de Olavide de Sevilla/CSIC ,  Sevilla ,  Spain","@type":"Person"}],"description":"Abstract.  Genome annotation, assisted by computer programs, is one of the great advances in modern biology. Nevertheless, the in silico identification of small","pageStart":"439","pageEnd":"449","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/6/1/m_cover.gif?Expires=2147483647&Signature=083aE4wPido26a6~EOoRT3Nb4fPpp5HV1MZzrjjay94a3evfhfRTvESRMhc7ROqyA2UIz2umFef89XL3Lu7hUZFod2Ix~QtkT4CP~8MDkYY61O0I1KLQQcYNpXVti4-0uwuVQWc0Ik7X46YCFzJAu2cTqwHZ2B0OBVFR-OZWKIwha86xHbqGB8R5L83t0zl8dqx1CkCPppI95mus6SUAdvZ0-wLyrVhsfEErb~4S7v8v0rjWvz1OZ1A2b6wWGyo1cjX3sRRxlgjkSlblmiowk2fs-R4agAPGw2bGiz2-0n1EnfsgKN8CSyFRqrRfdmccMFKJr5L5jh5C1s7L6OJB0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"AnABlast: a new  in silico  strategy for the genome-wide search of novel genes and fossil regions","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/6/1/m_cover.gif?Expires=2147483647&Signature=083aE4wPido26a6~EOoRT3Nb4fPpp5HV1MZzrjjay94a3evfhfRTvESRMhc7ROqyA2UIz2umFef89XL3Lu7hUZFod2Ix~QtkT4CP~8MDkYY61O0I1KLQQcYNpXVti4-0uwuVQWc0Ik7X46YCFzJAu2cTqwHZ2B0OBVFR-OZWKIwha86xHbqGB8R5L83t0zl8dqx1CkCPppI95mus6SUAdvZ0-wLyrVhsfEErb~4S7v8v0rjWvz1OZ1A2b6wWGyo1cjX3sRRxlgjkSlblmiowk2fs-R4agAPGw2bGiz2-0n1EnfsgKN8CSyFRqrRfdmccMFKJr5L5jh5C1s7L6OJB0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="AnABlast: a new  in silico  strategy for the genome-wide search of novel genes and fossil regions" />
    <meta property="og:description" content="Abstract.  Genome annotation, assisted by computer programs, is one of the great advances in modern biology. Nevertheless, the in silico identification of small" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/6/439/2462479" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-10-21" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Jimenez, Juan" /><meta name="citation_author_institution" content="Centro Andaluz de Biología del Desarrollo, Universidad Pablo de Olavide de Sevilla/CSIC, Sevilla, Spain" /><meta name="citation_author" content="Duncan, Caia D. S." /><meta name="citation_author_institution" content="Department of Biochemistry, University of Cambridge, Cambridge, UK" /><meta name="citation_author" content="Gallardo, María" /><meta name="citation_author_institution" content="Centro Andaluz de Biología del Desarrollo, Universidad Pablo de Olavide de Sevilla/CSIC, Sevilla, Spain" /><meta name="citation_author" content="Mata, Juan" /><meta name="citation_author_institution" content="Department of Biochemistry, University of Cambridge, Cambridge, UK" /><meta name="citation_author" content="Perez-Pulido, Antonio J." /><meta name="citation_author_institution" content="Centro Andaluz de Biología del Desarrollo, Universidad Pablo de Olavide de Sevilla/CSIC, Sevilla, Spain" /><meta name="citation_title" content="AnABlast: a new in silico strategy for the genome-wide search of novel genes and fossil regions" /><meta name="citation_firstpage" content="439" /><meta name="citation_lastpage" content="449" /><meta name="citation_doi" content="10.1093/dnares/dsv025" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2015/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Proteogenomics: concepts, applications and computational strategies;  citation_journal_title=Nat. Methods;  citation_year=2014;  citation_volume=11;  citation_pages=1114-25; " /><meta name="citation_reference" content="citation_title=Computational prediction of eukaryotic protein-coding genes;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2002;  citation_volume=3;  citation_pages=698-709; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs;  citation_journal_title=Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=Bioinformatics analysis identify novel OB fold protein coding genes in C. elegans;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e62204" /><meta name="citation_reference" content="citation_title=The Pfam protein families database;  citation_journal_title=Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=D211-22; " /><meta name="citation_reference" content="citation_title=Saturated BLAST: an automated multiple intermediate sequence search used to detect distant homology;  citation_journal_title=Bioinformatics;  citation_year=2000;  citation_volume=16;  citation_pages=1105-10; " /><meta name="citation_reference" content="citation_title=Protein sequence comparison and fold recognition: progress and good-practice benchmarking;  citation_journal_title=Curr. Opin. Struct. Biol.;  citation_year=2011;  citation_volume=21;  citation_pages=404-11; " /><meta name="citation_reference" content="citation_title=Using homology relations within a database markedly boosts protein sequence similarity search;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2015;  citation_volume=112;  citation_pages=7003-8; " /><meta name="citation_reference" content="citation_title=Application of Learning to Rank to protein remote homology detection;  citation_journal_title=Bioinformatics;  citation_year=2015; " /><meta name="citation_reference" content="citation_title=High-throughput functional genomics using CRISPR-Cas9;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2015;  citation_volume=16;  citation_pages=299-311; " /><meta name="citation_reference" content="citation_publisher=Springer, Berlin, Germany;  citation_title=The molecular biology of Schizosaccharomyces Pombe;  citation_year=2004; " /><meta name="citation_reference" content="citation_title=The genome sequence of Schizosaccharomyces pombe;  citation_journal_title=Nature;  citation_year=2002;  citation_volume=415;  citation_pages=871-80; " /><meta name="citation_reference" content="citation_title=Analysis of DNA strand-specific differential expression with high density tiling microarrays;  citation_journal_title=BMC Bioinformatics;  citation_year=2010;  citation_volume=11;  citation_pages=136" /><meta name="citation_reference" content="citation_title=Quantitative analysis of fission yeast transcriptomes and proteomes in proliferating and quiescent cells;  citation_journal_title=Cell;  citation_year=2012;  citation_volume=151;  citation_pages=671-83; " /><meta name="citation_reference" content="citation_title=Dynamic repertoire of a eukaryotic transcriptome surveyed at single-nucleotide resolution;  citation_journal_title=Nature;  citation_year=2008;  citation_volume=453;  citation_pages=1239-43; " /><meta name="citation_reference" content="citation_title=PomBase: a comprehensive online resource for fission yeast;  citation_journal_title=Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=D695-9; " /><meta name="citation_reference" content="citation_title=Augmented annotation of the Schizosaccharomyces pombe genome reveals additional genes required for growth and viability;  citation_journal_title=Genetics;  citation_year=2011;  citation_volume=187;  citation_pages=1207-17; " /><meta name="citation_reference" content="citation_title=The translational landscape of fission-yeast meiosis and sporulation;  citation_journal_title=Nat. Struct. Mol. Biol.;  citation_year=2014;  citation_volume=21;  citation_pages=641-7; " /><meta name="citation_reference" content="citation_title=Comparative functional genomics of the fission yeasts;  citation_journal_title=Science;  citation_year=2011;  citation_volume=332;  citation_pages=930-6; " /><meta name="citation_reference" content="citation_title=PomBase 2015: updates to the fission yeast database;  citation_journal_title=Nucleic Acids Res.;  citation_year=2015;  citation_volume=43;  citation_pages=D656-61; " /><meta name="citation_reference" content="citation_title=AnaGram:protein function assignment;  citation_journal_title=Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=291-2; " /><meta name="citation_reference" content="citation_title=Search for ancient patterns in protein sequences;  citation_journal_title=J. Mol. Evol.;  citation_year=1996;  citation_volume=42;  citation_pages=224-33; " /><meta name="citation_reference" content="citation_title=UniRef: comprehensive and non-redundant UniProt reference clusters;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=1282-8; " /><meta name="citation_reference" content="citation_title=Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration;  citation_journal_title=Brief Bioinform.;  citation_year=2013;  citation_volume=14;  citation_pages=178-92; " /><meta name="citation_reference" content="citation_title=Frameshift alignment: statistics and post-genomic applications;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=3575-82; " /><meta name="citation_reference" content="citation_title=The conversion of 3’ UTRs into coding regions;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=457-64; " /><meta name="citation_reference" content="citation_title=Alternative splicing in the human, mouse and rat genomes is associated with an increased frequency of exon creation and/or loss;  citation_journal_title=Nat. Genet.;  citation_year=2003;  citation_volume=34;  citation_pages=177-80; " /><meta name="citation_reference" content="citation_title=Origin and evolution of spliceosomal introns;  citation_journal_title=Biol. Direct;  citation_year=2012;  citation_volume=7;  citation_pages=11" /><meta name="citation_reference" content="citation_title=The evolution of transcription-initiation sites;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2005;  citation_volume=22;  citation_pages=1137-46; " /><meta name="citation_reference" content="citation_title=Genomic fossils as a snapshot of the human transcriptoma;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=1364-9; " /><meta name="citation_reference" content="citation_title=Genetic exchange between kingdoms;  citation_journal_title=Curr. Opin. Genet. Dev.;  citation_year=1991;  citation_volume=1;  citation_pages=530-3; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/6/439/17463186/dsv025.pdf" /><meta name="description" content="Abstract.  Genome annotation, assisted by computer programs, is one of the great advances in modern biology. Nevertheless, the in silico identification of small" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/6/439/2462479" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/6/439/2462479" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/6/439/2462479");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv025';
    </script>
<title>
	AnABlast: a new in silico strategy for the genome-wide search of novel genes and fossil regions | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2462479" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BepIPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJKaW1lbmV6LCBKdWFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudHJvIEFuZGFsdXogZGUgQmlvbG9nw61hIGRlbCBEZXNhcnJvbGxvLCBVbml2ZXJzaWRhZCBQYWJsbyBkZSBPbGF2aWRlIGRlIFNldmlsbGEvQ1NJQywgU2V2aWxsYSwgU3BhaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJEdW5jYW4sIENhaWEgRC4gUy4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2NoZW1pc3RyeSwgVW5pdmVyc2l0eSBvZiBDYW1icmlkZ2UsIENhbWJyaWRnZSwgVUsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHYWxsYXJkbywgTWFyw61hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudHJvIEFuZGFsdXogZGUgQmlvbG9nw61hIGRlbCBEZXNhcnJvbGxvLCBVbml2ZXJzaWRhZCBQYWJsbyBkZSBPbGF2aWRlIGRlIFNldmlsbGEvQ1NJQywgU2V2aWxsYSwgU3BhaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNYXRhLCBKdWFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9jaGVtaXN0cnksIFVuaXZlcnNpdHkgb2YgQ2FtYnJpZGdlLCBDYW1icmlkZ2UsIFVLIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUGVyZXotUHVsaWRvLCBBbnRvbmlvIEouIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudHJvIEFuZGFsdXogZGUgQmlvbG9nw61hIGRlbCBEZXNhcnJvbGxvLCBVbml2ZXJzaWRhZCBQYWJsbyBkZSBPbGF2aWRlIGRlIFNldmlsbGEvQ1NJQywgU2V2aWxsYSwgU3BhaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkFuQUJsYXN0OiBhIG5ldyBpbiBzaWxpY28gc3RyYXRlZ3kgZm9yIHRoZSBnZW5vbWUtd2lkZSBzZWFyY2ggb2Ygbm92ZWwgZ2VuZXMgYW5kIGZvc3NpbCByZWdpb25zIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNDM5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI0NDkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3YwMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNS8xMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb3Rlb2dlbm9taWNzOiBjb25jZXB0cywgYXBwbGljYXRpb25zIGFuZCBjb21wdXRhdGlvbmFsIHN0cmF0ZWdpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTExMTQtMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXB1dGF0aW9uYWwgcHJlZGljdGlvbiBvZiBldWthcnlvdGljIHByb3RlaW4tY29kaW5nIGdlbmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz02OTgtNzA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HYXBwZWQgQkxBU1QgYW5kIFBTSS1CTEFTVDogYSBuZXcgZ2VuZXJhdGlvbiBvZiBwcm90ZWluIGRhdGFiYXNlIHNlYXJjaCBwcm9ncmFtczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzM4OS00MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpb2luZm9ybWF0aWNzIGFuYWx5c2lzIGlkZW50aWZ5IG5vdmVsIE9CIGZvbGQgcHJvdGVpbiBjb2RpbmcgZ2VuZXMgaW4gQy4gZWxlZ2FuczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWU2MjIwNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBQZmFtIHByb3RlaW4gZmFtaWxpZXMgZGF0YWJhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zODsgIGNpdGF0aW9uX3BhZ2VzPUQyMTEtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNhdHVyYXRlZCBCTEFTVDogYW4gYXV0b21hdGVkIG11bHRpcGxlIGludGVybWVkaWF0ZSBzZXF1ZW5jZSBzZWFyY2ggdXNlZCB0byBkZXRlY3QgZGlzdGFudCBob21vbG9neTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xMTA1LTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm90ZWluIHNlcXVlbmNlIGNvbXBhcmlzb24gYW5kIGZvbGQgcmVjb2duaXRpb246IHByb2dyZXNzIGFuZCBnb29kLXByYWN0aWNlIGJlbmNobWFya2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gU3RydWN0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTQwNC0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXNpbmcgaG9tb2xvZ3kgcmVsYXRpb25zIHdpdGhpbiBhIGRhdGFiYXNlIG1hcmtlZGx5IGJvb3N0cyBwcm90ZWluIHNlcXVlbmNlIHNpbWlsYXJpdHkgc2VhcmNoOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTExMjsgIGNpdGF0aW9uX3BhZ2VzPTcwMDMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXBwbGljYXRpb24gb2YgTGVhcm5pbmcgdG8gUmFuayB0byBwcm90ZWluIHJlbW90ZSBob21vbG9neSBkZXRlY3Rpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLXRocm91Z2hwdXQgZnVuY3Rpb25hbCBnZW5vbWljcyB1c2luZyBDUklTUFItQ2FzOTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTI5OS0zMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1TcHJpbmdlciwgQmVybGluLCBHZXJtYW55OyAgY2l0YXRpb25fdGl0bGU9VGhlIG1vbGVjdWxhciBiaW9sb2d5IG9mIFNjaGl6b3NhY2NoYXJvbXljZXMgUG9tYmU7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgc2VxdWVuY2Ugb2YgU2NoaXpvc2FjY2hhcm9teWNlcyBwb21iZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTQxNTsgIGNpdGF0aW9uX3BhZ2VzPTg3MS04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgRE5BIHN0cmFuZC1zcGVjaWZpYyBkaWZmZXJlbnRpYWwgZXhwcmVzc2lvbiB3aXRoIGhpZ2ggZGVuc2l0eSB0aWxpbmcgbWljcm9hcnJheXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTEzNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVF1YW50aXRhdGl2ZSBhbmFseXNpcyBvZiBmaXNzaW9uIHllYXN0IHRyYW5zY3JpcHRvbWVzIGFuZCBwcm90ZW9tZXMgaW4gcHJvbGlmZXJhdGluZyBhbmQgcXVpZXNjZW50IGNlbGxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE1MTsgIGNpdGF0aW9uX3BhZ2VzPTY3MS04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RHluYW1pYyByZXBlcnRvaXJlIG9mIGEgZXVrYXJ5b3RpYyB0cmFuc2NyaXB0b21lIHN1cnZleWVkIGF0IHNpbmdsZS1udWNsZW90aWRlIHJlc29sdXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00NTM7ICBjaXRhdGlvbl9wYWdlcz0xMjM5LTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb21CYXNlOiBhIGNvbXByZWhlbnNpdmUgb25saW5lIHJlc291cmNlIGZvciBmaXNzaW9uIHllYXN0OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz1ENjk1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUF1Z21lbnRlZCBhbm5vdGF0aW9uIG9mIHRoZSBTY2hpem9zYWNjaGFyb215Y2VzIHBvbWJlIGdlbm9tZSByZXZlYWxzIGFkZGl0aW9uYWwgZ2VuZXMgcmVxdWlyZWQgZm9yIGdyb3d0aCBhbmQgdmlhYmlsaXR5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xODc7ICBjaXRhdGlvbl9wYWdlcz0xMjA3LTE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgdHJhbnNsYXRpb25hbCBsYW5kc2NhcGUgb2YgZmlzc2lvbi15ZWFzdCBtZWlvc2lzIGFuZCBzcG9ydWxhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBTdHJ1Y3QuIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz02NDEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgZnVuY3Rpb25hbCBnZW5vbWljcyBvZiB0aGUgZmlzc2lvbiB5ZWFzdHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MzMyOyAgY2l0YXRpb25fcGFnZXM9OTMwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvbUJhc2UgMjAxNTogdXBkYXRlcyB0byB0aGUgZmlzc2lvbiB5ZWFzdCBkYXRhYmFzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9RDY1Ni02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hR3JhbTpwcm90ZWluIGZ1bmN0aW9uIGFzc2lnbm1lbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MjkxLTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlYXJjaCBmb3IgYW5jaWVudCBwYXR0ZXJucyBpbiBwcm90ZWluIHNlcXVlbmNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTIyNC0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VW5pUmVmOiBjb21wcmVoZW5zaXZlIGFuZCBub24tcmVkdW5kYW50IFVuaVByb3QgcmVmZXJlbmNlIGNsdXN0ZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTEyODItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZWdyYXRpdmUgR2Vub21pY3MgVmlld2VyIChJR1YpOiBoaWdoLXBlcmZvcm1hbmNlIGdlbm9taWNzIGRhdGEgdmlzdWFsaXphdGlvbiBhbmQgZXhwbG9yYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJyaWVmIEJpb2luZm9ybS47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0xNzgtOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZyYW1lc2hpZnQgYWxpZ25tZW50OiBzdGF0aXN0aWNzIGFuZCBwb3N0LWdlbm9taWMgYXBwbGljYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTM1NzUtODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb252ZXJzaW9uIG9mIDPigJkgVVRScyBpbnRvIGNvZGluZyByZWdpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9NDU3LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbHRlcm5hdGl2ZSBzcGxpY2luZyBpbiB0aGUgaHVtYW4sIG1vdXNlIGFuZCByYXQgZ2Vub21lcyBpcyBhc3NvY2lhdGVkIHdpdGggYW4gaW5jcmVhc2VkIGZyZXF1ZW5jeSBvZiBleG9uIGNyZWF0aW9uIGFuZC9vciBsb3NzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPTE3Ny04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3JpZ2luIGFuZCBldm9sdXRpb24gb2Ygc3BsaWNlb3NvbWFsIGludHJvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2wuIERpcmVjdDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXZvbHV0aW9uIG9mIHRyYW5zY3JpcHRpb24taW5pdGlhdGlvbiBzaXRlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTExMzctNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgZm9zc2lscyBhcyBhIHNuYXBzaG90IG9mIHRoZSBodW1hbiB0cmFuc2NyaXB0b21hOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTEwMzsgIGNpdGF0aW9uX3BhZ2VzPTEzNjQtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBleGNoYW5nZSBiZXR3ZWVuIGtpbmdkb21zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBHZW5ldC4gRGV2LjsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9NTMwLTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMi82LzQzOS8xNzQ2MzE4Ni9kc3YwMjUucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEdlbm9tZSBhbm5vdGF0aW9uLCBhc3Npc3RlZCBieSBjb21wdXRlciBwcm9ncmFtcywgaXMgb25lIG9mIHRoZSBncmVhdCBhZHZhbmNlcyBpbiBtb2Rlcm4gYmlvbG9neS4gTmV2ZXJ0aGVsZXNzLCB0aGUgaW4gc2lsaWNvIGlkZW50aWZpY2F0aW9uIG9mIHNtYWxsIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0yNDYyNDc5JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZI7NdzV94aY492ksjOUPeDk1WK6f" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f439%2f2462479" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f439%2f2462479" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=83834" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/6/1/m_cover.gif?Expires=2147483647&amp;Signature=083aE4wPido26a6~EOoRT3Nb4fPpp5HV1MZzrjjay94a3evfhfRTvESRMhc7ROqyA2UIz2umFef89XL3Lu7hUZFod2Ix~QtkT4CP~8MDkYY61O0I1KLQQcYNpXVti4-0uwuVQWc0Ik7X46YCFzJAu2cTqwHZ2B0OBVFR-OZWKIwha86xHbqGB8R5L83t0zl8dqx1CkCPppI95mus6SUAdvZ0-wLyrVhsfEErb~4S7v8v0rjWvz1OZ1A2b6wWGyo1cjX3sRRxlgjkSlblmiowk2fs-R4agAPGw2bGiz2-0n1EnfsgKN8CSyFRqrRfdmccMFKJr5L5jh5C1s7L6OJB0g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/6">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87199832">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199832">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199834">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199834">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199839">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199839">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199848">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199848">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199887">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199887">Authors' contribution</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199889">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199889">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199891">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199891">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199893">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199893">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/6/425/2462458">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/22/6/451/2462493">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
<span class="article-flag"><a href="https://academic.oup.com/dnaresearch/search-results?f_OUPSeries=Editor's+Choice">Editor's Choice</a></span>            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">AnABlast: a new <em>in silico</em> strategy for the genome-wide search of novel genes and fossil regions
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Juan Jimenez</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Centro Andaluz de Biología del Desarrollo</div>, <div class="insititution">Universidad Pablo de Olavide de Sevilla/CSIC</div>, <div class="addr-line">Sevilla</div>, <div class="country">Spain</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel/Fax. +34 954-349-376. E-mail: <a href="mailto:jjimmar@upo.es" target="_blank">jjimmar@upo.es</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Juan+Jimenez">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jimenez J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jimenez J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Juan Jimenez</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Caia D. S. Duncan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Biochemistry</div>, <div class="insititution">University of Cambridge</div>, <div class="addr-line">Cambridge</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Caia+D.+S.+Duncan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Duncan C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Duncan C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Caia D. S. Duncan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mar&#237;a Gallardo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Centro Andaluz de Biología del Desarrollo</div>, <div class="insititution">Universidad Pablo de Olavide de Sevilla/CSIC</div>, <div class="addr-line">Sevilla</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mar%c3%ada+Gallardo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gallardo M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gallardo M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">María Gallardo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Juan Mata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Biochemistry</div>, <div class="insititution">University of Cambridge</div>, <div class="addr-line">Cambridge</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Juan+Mata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mata J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mata J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Juan Mata</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Antonio J. Perez-Pulido</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Centro Andaluz de Biología del Desarrollo</div>, <div class="insititution">Universidad Pablo de Olavide de Sevilla/CSIC</div>, <div class="addr-line">Sevilla</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Antonio+J.+Perez-Pulido">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Perez-Pulido A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Perez-Pulido A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Antonio J. Perez-Pulido</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Prof. Kenta Nakai</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 6, 1 December 2015, Pages 439–449, <a href='https://doi.org/10.1093/dnares/dsv025'>https://doi.org/10.1093/dnares/dsv025</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">21 October 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">23 July 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">25 September 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2462479" href="/dnaresearch/article-pdf/22/6/439/17463186/dsv025.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Juan Jimenez, Caia D. S. Duncan, María Gallardo, Juan Mata, Antonio J. Perez-Pulido;  AnABlast: a new <em>in silico</em> strategy for the genome-wide search of novel genes and fossil regions, <em>DNA Research</em>, Volume 22, Issue 6, 1 December 2015, Pages 439–449, <a href='https://doi.org/10.1093/dnares/dsv025'>https://doi.org/10.1093/dnares/dsv025</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2462479?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462479?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462479?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462479?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462479?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462479?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462479?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=AnABlast%3A%20a%20new%20%26lt%3Bem%26gt%3Bin%20silico%26lt%3B%2Fem%26gt%3B%20strategy%20for%20the%20genome-wide%20search%20of%20novel%20genes%20and%20fossil%20regions&amp;publicationDate=2015-10-21&amp;volumeNum=22&amp;issueNum=6&amp;author=Jimenez%2C%20Juan%3B%20Duncan%2C%20Caia%20D.%20S.&amp;startPage=439&amp;endPage=449&amp;contentId=10.1093%2Fdnares%2Fdsv025&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f439%2f2462479" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87199832 id="87199832" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Genome annotation, assisted by computer programs, is one of the great advances in modern biology. Nevertheless, the <em>in silico</em> identification of small and complex coding sequences is still challenging. We observed that amino acid sequences inferred from coding—but rarely from non-coding—DNA sequences accumulated alignments in low-stringency BLAST searches, suggesting that this alignments accumulation could be used to highlight coding regions in sequenced DNA. To investigate this possibility, we developed a computer program (AnABlast) that generates profiles of accumulated alignments in query amino acid sequences using a low-stringency BLAST strategy. To validate this approach, all six-frame translations of DNA sequences between every two annotated exons of the fission yeast genome were analysed with AnABlast. AnABlast-generated profiles identified three new copies of known genes, and four new genes supported by experimental evidence. New pseudogenes, ancestral carboxyl- and amino-terminal subtractions, complex gene rearrangements, and ancient fragments of mitDNA and of bacterial origin, were also inferred. Thus, this novel <em>in silico</em> approach provides a powerful tool to uncover new genes, as well as fossil-coding sequences, thus providing insight into the evolutionary history of annotated genomes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Schizosaccharomyces pombe&quot;"><em>Schizosaccharomyces pombe</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;new genes&quot;">new genes</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;in silico annotation tool&quot;"><em>in silico</em> annotation tool</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome evolution&quot;">genome evolution</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;fossil DNA sequences&quot;">fossil DNA sequences</a></div>                            <h2 scrollto-destination=87199834 id="87199834" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The development of new methods for identifying protein-coding genes and pseudogenes in genome sequences is one of the major challenges in the genomic age. In addition to the extraordinary support given by experimental approaches, computational analysis is an essential task for protein-coding gene identification in present genome biology.<sup><a reveal-id="DSV025C1" data-open="DSV025C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSV025C2" data-open="DSV025C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Current gene prediction algorithms perform a number of limited but standard steps, including identification of open reading frames (ORFs) and predicted sequence analysis to assign putative functions by similarity to known proteins, motifs, and/or structures in databases.<sup><a reveal-id="DSV025C3  DSV025C9" data-open="DSV025C3  DSV025C9" class="link link-ref link-reveal xref-bibr">3–9</a></sup> However, even though gene prediction and annotation software are increasingly efficient, small protein-coding genes and complex intron-containing genes may still escape detection by standard <em>in silico</em> analysis.</p><p>Recent experimental data from ‘omic-technologies’ generated in model organisms are creating the possibility to accurately update published genome annotations.<sup><a reveal-id="DSV025C10" data-open="DSV025C10" class="link link-ref link-reveal xref-bibr">10</a></sup> The fission yeast <em>Schizosaccharomyces pombe</em> is one of these model organisms.<sup><a reveal-id="DSV025C11" data-open="DSV025C11" class="link link-ref link-reveal xref-bibr">11</a></sup> The 12.5 Mb <em>S. pombe</em> genome was among the first genomes that were entirely sequenced. Based on standard computational prediction, together with experimental data, 4,824 genes and 31 pseudogenes were initially annotated.<sup><a reveal-id="DSV025C12" data-open="DSV025C12" class="link link-ref link-reveal xref-bibr">12</a></sup> However, while &gt;95% of the genome is transcribed,<sup><a reveal-id="DSV025C13  DSV025C15" data-open="DSV025C13  DSV025C15" class="link link-ref link-reveal xref-bibr">13–15</a></sup> only 55% of the genome corresponds to annotated coding sequences,<sup><a reveal-id="DSV025C12" data-open="DSV025C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSV025C16" data-open="DSV025C16" class="link link-ref link-reveal xref-bibr">16</a></sup> raising the possibility that more protein-coding genes await to be discovered. Accordingly, high-throughput cDNA sequencing (RNA-Seq) and mass spectrometry-based proteomic approaches<sup><a reveal-id="DSV025C14" data-open="DSV025C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSV025C17  DSV025C19" data-open="DSV025C17  DSV025C19" class="link link-ref link-reveal xref-bibr">17–19</a></sup> increased this initial number to the 5,123 protein-coding genes that are currently annotated in PomBase (<a class="link link-uri" href="http://www.pombase.org">http://www.pombase.org</a>), the <em>S. pombe</em> database.<sup><a reveal-id="DSV025C20" data-open="DSV025C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Some pseudogenes were also reassigned as genes, leaving 29 pseudogenes annotated at present. More recently, the use of ribosome profiling (Ribo-Seq) to investigate the translational landscape of meiosis added further 46 new protein-coding genes to the fission yeast genome.<sup><a reveal-id="DSV025C18" data-open="DSV025C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Most of these new findings correspond to genes coding for small polypeptides and/or coding sequences lacking significant alignments in databases, highlighting that the <em>in silico</em> identification of this set of coding sequences is still challenging in genome annotation.</p><p>The identification of similar proteins through BLAST analysis is one of the most useful strategies in annotation tasks. Finding significant alignments facilitates assignment of putative functions to query amino acid sequences through the identification of related proteins in databases. Interestingly, non-significant alignments (below the significant threshold) are often found in conventional BLAST searches. In polypeptides derived from the electronic translation of non-coding DNA, such alignments occur by chance. In polypeptides from coding DNA, however, in addition to random matches, non-significant alignments may also underlie small amino acid patterns representing random footprints of common ancestors.<sup><a reveal-id="DSV025C21" data-open="DSV025C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSV025C22" data-open="DSV025C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Since protein-coding genes mostly arise from previous ones during evolution, even highly divergent coding sequences may harbour ancient footprints of common ancestral proteins to be found among the millions of proteins available in databases. These footprints are not present in non-coding DNA sequences.<sup><a reveal-id="DSV025C22" data-open="DSV025C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Thus, alignments accumulated in predicted amino acid sequences might provide a method to discern coding from non-coding DNA, a new strategy that may overcome limitations of current <em>in silico</em> algorithms.</p><p>To use this strategy for the identification of coding regions at genome level, we developed a computer program, named AnABlast, which generates profiles of accumulated alignments in query amino acid sequences using a low-stringency BLAST strategy. To test its potential use as a gene-finder algorithm, a genome-wide search of coding sequences was carried out in fission yeast, and Ribo-Seq data used to validate protein-coding regions were highlighted by AnABlast profiles.<sup><a reveal-id="DSV025C18" data-open="DSV025C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Overall, several new genes and ancient coding sequences were efficiently identified by using AnABlast in the extremely well annotated <em>S. pombe</em> genome. The strategy was also successfully applied in a genomic region of the nematode <em>Caenorhabditis elegans</em>, suggesting that this novel tool may be applicable to the <em>in silico</em> analysis of any other sequenced genome.</p>                            <h2 scrollto-destination=87199839 id="87199839" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87199840 id="87199840" class="section-title" data-legacy-id=s2a>2.1. Search strategy</h3>
<p>We have designed a computational workflow, here named AnABlast (<strong>An</strong>cestral-patterns search through <strong>A BLAST</strong>-based strategy), to search for putative coding sequences in genomic regions. Briefly, to search for new coding regions in a genome, all inter-exon DNA sequences of the genome (intergenic and intronic regions) were translated in all six frames, masking stop codons with ‘X’ characters; in this way, the analysis become independent of traditional ORF signals. The algorithm then executes a BLASTP search with the six resultant amino acid sequences (as BLASTX, although it can be parallelized into six threads to accelerate the process) and the UniRef50 database (2014_02 version). The UniProt Reference Clusters (UniRef) provide clustered sets of sequences from virtually all the published protein sequences,<sup><a reveal-id="DSV025C23" data-open="DSV025C23" class="link link-ref link-reveal xref-bibr">23</a></sup> being the clusters from UniRef50 built with amino acid sequences having at least 50% sequence identity. AnABlast only uses the representative member from each UniRef50 cluster, and thus, it only considers non-similar sequences at 50% identity threshold (avoiding biases coming from sequence redundancy).</p><p>BLAST analysis efficiently identifies homologous sequences through highly significant alignments.<sup><a reveal-id="DSV025C3" data-open="DSV025C3" class="link link-ref link-reveal xref-bibr">3</a></sup> However, because a high frequency of low significant alignments may reveal stretches of ancestral regions,<sup><a reveal-id="DSV025C21" data-open="DSV025C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSV025C22" data-open="DSV025C22" class="link link-ref link-reveal xref-bibr">22</a></sup> we used a low-stringency BLAST strategy to include alignments beyond a significant similarity, a key step in our strategy to identify putative coding sequences with a biological and/or evolutionary significance (Fig. <a reveal-id="DSV025F1" data-open="DSV025F1" class="link link-reveal link-table xref-fig">1</a>). To this end, AnABlast uses non-restrictive parameters including a low bit score threshold for considering hits. Bit score represents a normalized value that allows comparison of results from different assays. Because bit score thresholds are not allowed in BLAST tools, low bit score values (from 25) were established by using high <em>E</em>-values (up to 2,00,000). Using the appropriate scoring matrix during the BLAST search (among available BLOSUM and PAM standard matrices) can improve AnABlast sensitivity. In this study, the BLOSUM90 matrix was selected to favour the search of high identity sequences in very short alignments, enough to be considered protomotifs<sup><a reveal-id="DSV025C21" data-open="DSV025C21" class="link link-ref link-reveal xref-bibr">21</a></sup> or ancient footprints of protein sequences.<sup><a reveal-id="DSV025C22" data-open="DSV025C22" class="link link-ref link-reveal xref-bibr">22</a></sup> The low-complexity filter SEG is also used to avoid sequence repeats, thereby improving specificity. Sequence alignments were trained with coding sequences from the analysed genome (data not shown), and a bit score threshold (given by the score matrix and the search space size) that better highlighted coding regions was used. In the <em>S. pombe</em> genome, a threshold of 30 in the score alignment was optimal in our search strategy, corresponding to an <em>E</em>-value of ∼12,000 (the default <em>E</em>-value for searching significant hits is usually 10). Finally, up to a maximum of 10,000 alignments by query is allowed. </p>                        <a id="87199843" scrollto-destination="87199843"></a>
<div id="DSV025F1" data-id="DSV025F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/m_dsv02501.png?Expires=2147483647&Signature=ZGZlPT1kS2T9vTnBm~2mOSo3X5SOQHoyg2vKoMu3ZKk3aG6BGZdGVrwdlt8Z8Srmz3Eda2m5SDUXOBvHxqF4uAlfxyYlOTJBWoEcoriqL66~lEzQcFBtY0q7HkQ70QGVkc6Wy-txd4MI82L5XFKWnQ-ptbTD0e333B3n3VbE8Yb2H~UMnyz5iBObYmuxMtcrPL8gDBp9eWB8fVe~Y0KNcFvb7l0~rHgewDQRZq90rg3XSwbDJ4vejdR6F26FW6ZpTiPqLdXdweB3Z8hRY6PoYc~AnN18f8mfWJkkzHkqFKqgesNdxLgtHLbnM3mw86J85MFscTw9AS4JMAgQm97Tig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of the AnABlast strategy for the identification of coding sequences. (A) In conserved proteins, conventional BLAST analysis of query sequences usually generates a number of significant alignments that allow the identification of coding sequences. The accumulation of these alignments along the query sequence (AnABlast profile) generates prominent peaks that also allow the easy identification of conserved coding regions. (B) In non-conserved sequences, BLAST search generates non-significant alignments, but AnABlast profiles highlight coding regions by the significant accumulation of these alignments, generated by random matches as well as from random footprints of common ancestors. (C) Non-coding sequences lack common ancestors in protein databases, and non-significant alignments may only occur by chance. In this case, alignments rarely accumulate in particular regions. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02501.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199843/dsv02501.tiff" path-from-xml="dsv02501.tiff" target="_blank">View large</a><a section="87199843" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv02501.png?Expires=2147483647&Signature=vBkF05tv7WTyXhKpxCnHG3uCI3lCr7HKd-g-7jr0YZHxI4BS2oY5bIbosoXtMKgaGCIEBcSH2AScAT5rWO6zCtG6VTs6oS35pBAZ7wuVZfkNgSaGf09EkE4Fu~u-SOdaEN1~iuQRDTN9eADeJexOugkD8IT4tktFHyvMla8XyJq4MbiTxmyNmCrAiwVLBrS0EFRqPfZ13VWMIo1uypohJ-se7ZFAHWQs4I36MWKt2CaJVhZY7FSMvzGLEiwUmAWvu80ypjrMgmATNdiDyvC8ZxyyFRpz1BP7Lcvt~7F12seZjcYftArvtPQYCh02lZmbUnGJYOcr7y2IoW1MSVPadA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199843&ar=2462479&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02501.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of the AnABlast strategy for the identification of coding sequences. (A) In conserved proteins, conventional BLAST analysis of query sequences usually generates a number of significant alignments that allow the identification of coding sequences. The accumulation of these alignments along the query sequence (AnABlast profile) generates prominent peaks that also allow the easy identification of conserved coding regions. (B) In non-conserved sequences, BLAST search generates non-significant alignments, but AnABlast profiles highlight coding regions by the significant accumulation of these alignments, generated by random matches as well as from random footprints of common ancestors. (C) Non-coding sequences lack common ancestors in protein databases, and non-significant alignments may only occur by chance. In this case, alignments rarely accumulate in particular regions. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV025F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/m_dsv02501.png?Expires=2147483647&Signature=ZGZlPT1kS2T9vTnBm~2mOSo3X5SOQHoyg2vKoMu3ZKk3aG6BGZdGVrwdlt8Z8Srmz3Eda2m5SDUXOBvHxqF4uAlfxyYlOTJBWoEcoriqL66~lEzQcFBtY0q7HkQ70QGVkc6Wy-txd4MI82L5XFKWnQ-ptbTD0e333B3n3VbE8Yb2H~UMnyz5iBObYmuxMtcrPL8gDBp9eWB8fVe~Y0KNcFvb7l0~rHgewDQRZq90rg3XSwbDJ4vejdR6F26FW6ZpTiPqLdXdweB3Z8hRY6PoYc~AnN18f8mfWJkkzHkqFKqgesNdxLgtHLbnM3mw86J85MFscTw9AS4JMAgQm97Tig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of the AnABlast strategy for the identification of coding sequences. (A) In conserved proteins, conventional BLAST analysis of query sequences usually generates a number of significant alignments that allow the identification of coding sequences. The accumulation of these alignments along the query sequence (AnABlast profile) generates prominent peaks that also allow the easy identification of conserved coding regions. (B) In non-conserved sequences, BLAST search generates non-significant alignments, but AnABlast profiles highlight coding regions by the significant accumulation of these alignments, generated by random matches as well as from random footprints of common ancestors. (C) Non-coding sequences lack common ancestors in protein databases, and non-significant alignments may only occur by chance. In this case, alignments rarely accumulate in particular regions. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02501.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199843/dsv02501.tiff" path-from-xml="dsv02501.tiff" target="_blank">View large</a><a section="87199843" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv02501.png?Expires=2147483647&Signature=vBkF05tv7WTyXhKpxCnHG3uCI3lCr7HKd-g-7jr0YZHxI4BS2oY5bIbosoXtMKgaGCIEBcSH2AScAT5rWO6zCtG6VTs6oS35pBAZ7wuVZfkNgSaGf09EkE4Fu~u-SOdaEN1~iuQRDTN9eADeJexOugkD8IT4tktFHyvMla8XyJq4MbiTxmyNmCrAiwVLBrS0EFRqPfZ13VWMIo1uypohJ-se7ZFAHWQs4I36MWKt2CaJVhZY7FSMvzGLEiwUmAWvu80ypjrMgmATNdiDyvC8ZxyyFRpz1BP7Lcvt~7F12seZjcYftArvtPQYCh02lZmbUnGJYOcr7y2IoW1MSVPadA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199843&ar=2462479&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02501.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of the AnABlast strategy for the identification of coding sequences. (A) In conserved proteins, conventional BLAST analysis of query sequences usually generates a number of significant alignments that allow the identification of coding sequences. The accumulation of these alignments along the query sequence (AnABlast profile) generates prominent peaks that also allow the easy identification of conserved coding regions. (B) In non-conserved sequences, BLAST search generates non-significant alignments, but AnABlast profiles highlight coding regions by the significant accumulation of these alignments, generated by random matches as well as from random footprints of common ancestors. (C) Non-coding sequences lack common ancestors in protein databases, and non-significant alignments may only occur by chance. In this case, alignments rarely accumulate in particular regions. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>In a second step, AnABlast takes the positions from the acquired hits and counts the number of alignments (belonging to different clusters) that matches each position. As a result of this analysis, AnABlast yields a profile where discrete peaks highlight specific regions in the genome. Independently of the level of significance of each particular alignment, the high number of sequence alignments accumulated at a specific strand and reading frame reveals a significant region in terms of biological function and/or evolution.<sup><a reveal-id="DSV025C21" data-open="DSV025C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSV025C22" data-open="DSV025C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Under the above parameters, randomized and non-coding sequences yielded AnABlast profiles with peaks rarely reaching a maximum of 70 accumulated sequences. In this study, a cut-off threshold of 70 accumulated sequences was applied to optimize the identification of putative present and fossil-coding regions. The AnABlast algorithm is summarized in Fig. <a reveal-id="DSV025F2" data-open="DSV025F2" class="link link-reveal link-table xref-fig">2</a> and can be tested by downloading its different modules from <a class="link link-uri" href="http://www.bioinfocabd.upo.es/ab">http://www.bioinfocabd.upo.es/ab</a>. </p>                        <a id="87199845" scrollto-destination="87199845"></a>
<div id="DSV025F2" data-id="DSV025F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/m_dsv02502.png?Expires=2147483647&Signature=v9J8UWXXdu0HiOLBSC4im0~MvP1R1~-F7s~uAY3NR6NHOjeeIf-UqbBsIn84PjPEJnZAsuf7IHVkWCQ68haXfGvCnfnJydfX52nRVoUkpwPockk5T90HTVJfuW-YvgxeYc0pH7u-hrZQjIMXbrS9774O-7o-FCnJXbWNSvN9dvhCtfJHF1~aqK2a1lOauf2wHPcvpDwFrHG91hhSzSg6D7oMZRQ39zOcaO3kIhqzFZI5TXqN2X3y7RKLcpb5gc-iGi8uSqRVqbihWfaZAuHJ2enr7Q~QiiJ7NO1j6XmHK94P8PdUZL7mTJvJBJ6tErSnaKID0yYIQJVBcX~uefbaJg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="AnABlast algorithm (schematized flux diagram) for coding sequence identification in a genome-wide search strategy. Amino acid sequences obtained from the six reading frames (RF) of each DNA inter-exon region are subjected to BLAST search against a protein database that minimizes redundancies (uniref50). After low-stringency alignment parameters (optimal bit score threshold: 30), hits are taken at each position of the query amino acid sequence, generating a profile of accumulated alignments along the sequence (AnABlast profile). Significant peaks in the generated profiles (higher than 70 in our S. pombe genome analysis) were selected, and the corresponding amino acid sequence further analysed by conventional BLAST and Pfam search. Ribo-Seq data in the identified genomic region of AnABlast peaks are also analysed. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02502.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199845/dsv02502.tiff" path-from-xml="dsv02502.tiff" target="_blank">View large</a><a section="87199845" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv02502.png?Expires=2147483647&Signature=ohsCbtkpmzNsK6tO2~sSRLx~qEPV5Xbzhs744-y857y5nypq8QgO7rzPat-EXZgmSejN6F96aE4hu2slwa5PrkscABq2~MoYqzn6UkdMSD3RDCARsNep7SUK0aXtkZW~h7k5nXsi-MuTaC~JaqApM1UwOGkvbNJNrIcXpZj6dXwdGEFc4Uu6zo5AEwM1~A~a2rujr7UeawI~eTw7mAUJIfyKQsCsbyXzDIYfg1i61wURrLlDignmcr9qH74yprGvglP7j1RAQfiQRCVz~rtQqLMBRjZ1aBEaCw5QKy9AFxn4h6Cgepj7c0wHSbtbj5xu5gCQFlGk4crHzU-oseZ9lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199845&ar=2462479&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02502.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>AnABlast algorithm (schematized flux diagram) for coding sequence identification in a genome-wide search strategy. Amino acid sequences obtained from the six reading frames (RF) of each DNA inter-exon region are subjected to BLAST search against a protein database that minimizes redundancies (uniref50). After low-stringency alignment parameters (optimal bit score threshold: 30), hits are taken at each position of the query amino acid sequence, generating a profile of accumulated alignments along the sequence (AnABlast profile). Significant peaks in the generated profiles (higher than 70 in our <em>S. pombe</em> genome analysis) were selected, and the corresponding amino acid sequence further analysed by conventional BLAST and Pfam search. Ribo-Seq data in the identified genomic region of AnABlast peaks are also analysed. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV025F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/m_dsv02502.png?Expires=2147483647&Signature=v9J8UWXXdu0HiOLBSC4im0~MvP1R1~-F7s~uAY3NR6NHOjeeIf-UqbBsIn84PjPEJnZAsuf7IHVkWCQ68haXfGvCnfnJydfX52nRVoUkpwPockk5T90HTVJfuW-YvgxeYc0pH7u-hrZQjIMXbrS9774O-7o-FCnJXbWNSvN9dvhCtfJHF1~aqK2a1lOauf2wHPcvpDwFrHG91hhSzSg6D7oMZRQ39zOcaO3kIhqzFZI5TXqN2X3y7RKLcpb5gc-iGi8uSqRVqbihWfaZAuHJ2enr7Q~QiiJ7NO1j6XmHK94P8PdUZL7mTJvJBJ6tErSnaKID0yYIQJVBcX~uefbaJg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="AnABlast algorithm (schematized flux diagram) for coding sequence identification in a genome-wide search strategy. Amino acid sequences obtained from the six reading frames (RF) of each DNA inter-exon region are subjected to BLAST search against a protein database that minimizes redundancies (uniref50). After low-stringency alignment parameters (optimal bit score threshold: 30), hits are taken at each position of the query amino acid sequence, generating a profile of accumulated alignments along the sequence (AnABlast profile). Significant peaks in the generated profiles (higher than 70 in our S. pombe genome analysis) were selected, and the corresponding amino acid sequence further analysed by conventional BLAST and Pfam search. Ribo-Seq data in the identified genomic region of AnABlast peaks are also analysed. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02502.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199845/dsv02502.tiff" path-from-xml="dsv02502.tiff" target="_blank">View large</a><a section="87199845" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv02502.png?Expires=2147483647&Signature=ohsCbtkpmzNsK6tO2~sSRLx~qEPV5Xbzhs744-y857y5nypq8QgO7rzPat-EXZgmSejN6F96aE4hu2slwa5PrkscABq2~MoYqzn6UkdMSD3RDCARsNep7SUK0aXtkZW~h7k5nXsi-MuTaC~JaqApM1UwOGkvbNJNrIcXpZj6dXwdGEFc4Uu6zo5AEwM1~A~a2rujr7UeawI~eTw7mAUJIfyKQsCsbyXzDIYfg1i61wURrLlDignmcr9qH74yprGvglP7j1RAQfiQRCVz~rtQqLMBRjZ1aBEaCw5QKy9AFxn4h6Cgepj7c0wHSbtbj5xu5gCQFlGk4crHzU-oseZ9lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199845&ar=2462479&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02502.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>AnABlast algorithm (schematized flux diagram) for coding sequence identification in a genome-wide search strategy. Amino acid sequences obtained from the six reading frames (RF) of each DNA inter-exon region are subjected to BLAST search against a protein database that minimizes redundancies (uniref50). After low-stringency alignment parameters (optimal bit score threshold: 30), hits are taken at each position of the query amino acid sequence, generating a profile of accumulated alignments along the sequence (AnABlast profile). Significant peaks in the generated profiles (higher than 70 in our <em>S. pombe</em> genome analysis) were selected, and the corresponding amino acid sequence further analysed by conventional BLAST and Pfam search. Ribo-Seq data in the identified genomic region of AnABlast peaks are also analysed. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87199846 id="87199846" class="section-title" data-legacy-id=s2b>2.2. Analysis of the selected sequences</h3>
<p>For all analysis, <em>S. pombe</em> annotations, protein and DNA sequences, and RNA expression data available from PomBase (<a class="link link-uri" href="http://www.pombase.org/">http://www.pombase.org/</a>) and <em>C. elegans</em> (<a class="link link-uri" href="https://www.wormbase.org/">https://www.wormbase.org/</a>) on 1 February 2014 were used. The amino acid sequence delimited by each AnABlast peak was further studied manually by using BLAST<sup><a reveal-id="DSV025C3" data-open="DSV025C3" class="link link-ref link-reveal xref-bibr">3</a></sup> and Pfam<sup><a reveal-id="DSV025C5" data-open="DSV025C5" class="link link-ref link-reveal xref-bibr">5</a></sup> from their web applications (<a class="link link-uri" href="https://blast.ncbi.nlm.nih.gov/">https://blast.ncbi.nlm.nih.gov/</a> and <a class="link link-uri" href="http://pfam.xfam.org/">http://pfam.xfam.org/</a>, respectively). To determine expression levels in AnABlast domains, genome-wide transcriptome Ribo-Seq data from proliferating and sporulating cells.<sup><a reveal-id="DSV025C18" data-open="DSV025C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Ribo-Seq and AnABlast data were visualized with the Integrated Genome Viewer.<sup><a reveal-id="DSV025C24" data-open="DSV025C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Genome-wide RNA-Seq data from cells subjected to temperature or nutritional stress<sup><a reveal-id="DSV025C14" data-open="DSV025C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSV025C19" data-open="DSV025C19" class="link link-ref link-reveal xref-bibr">19</a></sup> were also used (available at <a class="link link-uri" href="http://www.pombase.org/">http://www.pombase.org/</a>).</p>                            <h2 scrollto-destination=87199848 id="87199848" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=87199849 id="87199849" class="section-title" data-legacy-id=s3a>3.1. Setting up optimal AnABlast parameters</h3>
<p>The significant accumulation of amino acid sequence alignments produced by AnABlast analysis may be useful to identify DNA-coding sequences of biological and/or evolutionary significance (Figs <a reveal-id="DSV025F1" data-open="DSV025F1" class="link link-reveal link-table xref-fig">1</a> and <a reveal-id="DSV025F2" data-open="DSV025F2" class="link link-reveal link-table xref-fig">2</a>). To establish optimal AnABlast parameters for the accurate identification of coding sequences, we first analysed a genomic DNA region encoding known proteins. The <em>S. pombe</em> genome interval containing the universal protein kinase Cdc2 (encoded in the forward strand) and its flanking genes SPBC11B10.08 (forward strand), and <em>pht1</em> (reverse strand) was used to this purpose. All six-frame translations of this DNA interval were obtained. To allow the complete analysis of the corresponding amino acid sequence independently of existing ORFs, stop codons were ignored (masked with ‘X’). As expected, AnABlast analysis produced a profile of discrete peaks that matched regions of annotated exons and RNA expression patterns (Fig. <a reveal-id="DSV025F3" data-open="DSV025F3" class="link link-reveal link-table xref-fig">3</a>). Setting optimal thresholds in AnABlast parameters (bit score threshold in sequence alignments and cut-off threshold of accumulated sequences) is nonetheless a key question. The use of very low-stringency parameters to generate BLAST alignments (score = 28) would favour the identification of highly divergent coding sequences in our AnABlast strategy. However, the background noise caused by random matches at this permissive condition makes it difficult to discard false positives obtained by chance in non-coding sequences (Fig. <a reveal-id="DSV025F3" data-open="DSV025F3" class="link link-reveal link-table xref-fig">3</a>A). On the other hand, setting of high alignment cut-off thresholds (score = 40) resulted in the efficient highlighting of only those coding regions with a significant similarity to sequences in databases, a pool of genes that are easily identified by conventional methods. By using a threshold of 30 in the score alignment, AnABlast peaks (accumulated sequences) generated in non-coding regions rarely reached the value of 70, while coding regions accumulated hundreds of sequences, yielding easily identifiable peaks (Fig. <a reveal-id="DSV025F3" data-open="DSV025F3" class="link link-reveal link-table xref-fig">3</a>). Thus, a score alignment threshold of 30 in the BLAST search and a cut-off threshold of 70 in alignments accumulation were used as standard parameters for the selection of significant peaks identifying putative coding sequences. </p>                        <a id="87199851" scrollto-destination="87199851"></a>
<div id="DSV025F3" data-id="DSV025F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/m_dsv02503.png?Expires=2147483647&Signature=vT7-ntS21mH1KnY5-Gb4OFNca8TqnUM3GNHXoRhPvyB7tZBq9pTVx7VC~v2SUG-n4nuXjOxjuC3ZYpmIPUavQeqKZ8J87iSNlAwLOte2578z2CQ7LSi9A8QvqXLkAuHRSEPGwj8QQ4ieqk84ePqsh-STvEBWtkMQfptX0NfwSc9ljwMjnrZsaNloHg~MwTQFPNO2S~4eIAWn74B14Yih7vt9ensMDVAZXE7TNxZ2MpVKl~rxi8iUMwY89PfsW9TG3FRmbGM7Vf0XxyQxo75g25-0bRQF2h1WfnEvlHkrTP913vJqv5ojGDbOZ109hSEaQCTk23U0eqBAy2LECIgX7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="AnABlast profiles (accumulated alignments) in the Chr II: 1498400-1503600 genomic region containing the well-characterized cdc2 coding sequence and its flanking SPBC11B10.08 and pht1 genes (Pombase annotations). (A) Accumulation of alignments obtained in different score cut-off threshold from BLAST results (indicated) allows the establishment of optimal parameters for genome-wide search of coding sequences with AnABlast algorithms. Ribo-Seq data are used to confirm accuracy of AnABlast predictions from protein sequences encoded in the three possible reading frames in the forward (colour codes 1–3) and the reverse (colour codes 4–6) strand. (B) Representative AnABlast profile (score threshold: 30) obtained from a randomized DNA sequence of this genomic interval. (C) AnABlast profile (score threshold: 30) from the reverse DNA sequence (lacking biological significance in terms of protein coding) of this genomic interval. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02503.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199851/dsv02503.tiff" path-from-xml="dsv02503.tiff" target="_blank">View large</a><a section="87199851" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv02503.png?Expires=2147483647&Signature=KBZhjki3-nJng69yGC6UnV7EPv9qiaITMgyW1urCk4xzMe4CQj0D0Q6W5ZdLYGrl~XHZeR42nGPUPpKWauuza2Kz9lJMBBXPxi-un3m7UKHXR0hkLue1niCPWaHPjWIU33HzneZIP~255R9NDW6wNAUMuorZrWcD-lahjeqeOvzWd4w3Lhy8AmgsSIFjNRNmysntdZ4e-G--584nR1ubMvRyho1TbPhVnB-mf1xoDLC8aYZdH~4vGlnIZEj4GdNcVsTcZvlIzozE00wQVcmE6ZU09-CAsN8qNA7XqYO0cIv4pGzF1yh-NlQFcdr6YuiqMITh5tJPRO~9cu9070fjDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199851&ar=2462479&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02503.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>AnABlast profiles (accumulated alignments) in the Chr II: 1498400-1503600 genomic region containing the well-characterized <em>cdc2</em> coding sequence and its flanking SPBC11B10.08 and <em>pht1</em> genes (Pombase annotations). (A) Accumulation of alignments obtained in different score cut-off threshold from BLAST results (indicated) allows the establishment of optimal parameters for genome-wide search of coding sequences with AnABlast algorithms. Ribo-Seq data are used to confirm accuracy of AnABlast predictions from protein sequences encoded in the three possible reading frames in the forward (colour codes 1–3) and the reverse (colour codes 4–6) strand. (B) Representative AnABlast profile (score threshold: 30) obtained from a randomized DNA sequence of this genomic interval. (C) AnABlast profile (score threshold: 30) from the reverse DNA sequence (lacking biological significance in terms of protein coding) of this genomic interval. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV025F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/m_dsv02503.png?Expires=2147483647&Signature=vT7-ntS21mH1KnY5-Gb4OFNca8TqnUM3GNHXoRhPvyB7tZBq9pTVx7VC~v2SUG-n4nuXjOxjuC3ZYpmIPUavQeqKZ8J87iSNlAwLOte2578z2CQ7LSi9A8QvqXLkAuHRSEPGwj8QQ4ieqk84ePqsh-STvEBWtkMQfptX0NfwSc9ljwMjnrZsaNloHg~MwTQFPNO2S~4eIAWn74B14Yih7vt9ensMDVAZXE7TNxZ2MpVKl~rxi8iUMwY89PfsW9TG3FRmbGM7Vf0XxyQxo75g25-0bRQF2h1WfnEvlHkrTP913vJqv5ojGDbOZ109hSEaQCTk23U0eqBAy2LECIgX7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="AnABlast profiles (accumulated alignments) in the Chr II: 1498400-1503600 genomic region containing the well-characterized cdc2 coding sequence and its flanking SPBC11B10.08 and pht1 genes (Pombase annotations). (A) Accumulation of alignments obtained in different score cut-off threshold from BLAST results (indicated) allows the establishment of optimal parameters for genome-wide search of coding sequences with AnABlast algorithms. Ribo-Seq data are used to confirm accuracy of AnABlast predictions from protein sequences encoded in the three possible reading frames in the forward (colour codes 1–3) and the reverse (colour codes 4–6) strand. (B) Representative AnABlast profile (score threshold: 30) obtained from a randomized DNA sequence of this genomic interval. (C) AnABlast profile (score threshold: 30) from the reverse DNA sequence (lacking biological significance in terms of protein coding) of this genomic interval. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02503.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199851/dsv02503.tiff" path-from-xml="dsv02503.tiff" target="_blank">View large</a><a section="87199851" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv02503.png?Expires=2147483647&Signature=KBZhjki3-nJng69yGC6UnV7EPv9qiaITMgyW1urCk4xzMe4CQj0D0Q6W5ZdLYGrl~XHZeR42nGPUPpKWauuza2Kz9lJMBBXPxi-un3m7UKHXR0hkLue1niCPWaHPjWIU33HzneZIP~255R9NDW6wNAUMuorZrWcD-lahjeqeOvzWd4w3Lhy8AmgsSIFjNRNmysntdZ4e-G--584nR1ubMvRyho1TbPhVnB-mf1xoDLC8aYZdH~4vGlnIZEj4GdNcVsTcZvlIzozE00wQVcmE6ZU09-CAsN8qNA7XqYO0cIv4pGzF1yh-NlQFcdr6YuiqMITh5tJPRO~9cu9070fjDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199851&ar=2462479&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02503.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>AnABlast profiles (accumulated alignments) in the Chr II: 1498400-1503600 genomic region containing the well-characterized <em>cdc2</em> coding sequence and its flanking SPBC11B10.08 and <em>pht1</em> genes (Pombase annotations). (A) Accumulation of alignments obtained in different score cut-off threshold from BLAST results (indicated) allows the establishment of optimal parameters for genome-wide search of coding sequences with AnABlast algorithms. Ribo-Seq data are used to confirm accuracy of AnABlast predictions from protein sequences encoded in the three possible reading frames in the forward (colour codes 1–3) and the reverse (colour codes 4–6) strand. (B) Representative AnABlast profile (score threshold: 30) obtained from a randomized DNA sequence of this genomic interval. (C) AnABlast profile (score threshold: 30) from the reverse DNA sequence (lacking biological significance in terms of protein coding) of this genomic interval. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>Virtual translation of non-coding DNA generates random protein sequences. To better estimate false-positive rates under these parameters, we randomly shuffled the DNA sequence of the analysed genome interval. No significant peaks were found in 50 randomized sequences tested (see Fig. <a reveal-id="DSV025F3" data-open="DSV025F3" class="link link-reveal link-table xref-fig">3</a>B), suggesting that significant peaks rarely occur by chance with the selected settings. Similarly, no significant peaks were observed when analysing the reverse 3′-5′ sequence (Fig. <a reveal-id="DSV025F3" data-open="DSV025F3" class="link link-reveal link-table xref-fig">3</a>C).</p><p>Classic frameshift alignment analysis aids in the identification of frameshifts in DNA sequences.<sup><a reveal-id="DSV025C25" data-open="DSV025C25" class="link link-ref link-reveal xref-bibr">25</a></sup> As AnABlast profiles identify coding sequences in the different reading frames, coding frameshifts can be also easily inferred from this computational analysis (see Cdc2 exons in Fig. <a reveal-id="DSV025F3" data-open="DSV025F3" class="link link-reveal link-table xref-fig">3</a>A).</p>                            <h3 scrollto-destination=87199854 id="87199854" class="section-title" data-legacy-id=s3b>3.2. AnABlast analysis of inter-exon regions in the entire <em>S. pombe</em> genome</h3>
<p>To perform a genome-wide search of new coding regions in fission yeast, we analysed all nucleotide sequences between every two adjacent annotated exons, accounting for a total of 5,870 intergenic regions and 4,782 introns. AnABlast profiles were generated for each of the six-frame translation sequences of these inter-exon regions. Among them, 158 regions contained at least one significant peak and were selected for further analysis. As a negative control, we randomized the corresponding DNA sequence in each of the analysed inter-exon regions. With this data set, only 19 significant peaks were obtained (AnABlast profiles obtained both in intergenic sequences, and its randomized versions are available at <a class="link link-uri" href="http://www.bioinfocabd.upo.es/anablast/supplementary_data.html">http://www.bioinfocabd.upo.es/anablast/supplementary_data.html</a>). Thus, on average, &gt;85% of the selected AnABlast peaks are expected to identify true-positive coding sequences.</p><p>Interestingly, all the selected profiles, except one, were located in intergenic regions, suggesting that in general, <em>S. pombe</em> introns do not usually contain coding sequences. The only intronic peak was located in the first intron of the SPAC22F3.11c gene, <em>snu23</em> (Fig. <a reveal-id="DSV025F4" data-open="DSV025F4" class="link link-reveal link-table xref-fig">4</a>A). RNA-Seq profiles, the identification of zinc-finger motifs, and its high identity to <em>Schizosaccharomyces sps</em> genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv025_Supplementary_Data.zip?Expires=2147483647&Signature=1~jmJkVjh9nWQGIj2THUr5D9ZyIt-dff-I6YJG0D2WHfVKRiS1ULEF8wlV0~qsgibk4428hR8nTVJzlTk6AqCuZmrkL7RL9z~ZZxHzalAz43uaxTVm2M~i7SNOaxn3bja6jg-TLg9eJyoDg0ClySTnk6Yamj8pZn59POioyLbyBdL9MhSXahN9QyofilJXcuTB20yMCM2qcZ49WPg~C8vy1KkIaRIVtQe6SyNF8JAZ3lOlLDSUCSKXCXHxj1HVV8H7hEemFxddd9t0xQdjDWROOcsjpxn19QYay0xjvA5kdiz5eAUbmvOg95Yu2rzrHsH5dc7JXZyL9yqwya2DhrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1A</a></span>) indicate that the sequence identified by AnABlast in this annotated intron is in fact a coding sequence of the <em>snu23</em> gene (Fig. <a reveal-id="DSV025F4" data-open="DSV025F4" class="link link-reveal link-table xref-fig">4</a>A). A similar suggestion was made from <em>S. pombe</em> transcriptome analysis.<sup><a reveal-id="DSV025C19" data-open="DSV025C19" class="link link-ref link-reveal xref-bibr">19</a></sup></p>                        <a id="87199857" scrollto-destination="87199857"></a>
<div id="DSV025F4" data-id="DSV025F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/m_dsv02504.png?Expires=2147483647&Signature=CCOZHSefrMgggN0YoA1F1Re0WLVMCsgEhK~myfWEdnOX98TPI61N~PiyiH~nRxUfUv2Hqozy-O3iKRhxBoKdll6qFoo3T-vCtoCDG61v4J0sdjvA4pA9OyKHYhXiS3Ygtqu7K9nya3pOvOEkagw2Ch1t8tA2K9PhOoHYHXv0fpS4IEkmXZKglv8bBgCwYU7OJ8bWOPoSJfEisy-O~izIMg3ladlu13q69cXMGj29qQr~EjLpOZdPIpM~vDZyUgfNFjnSWoFWRk5mXl0Su1Dv1E6zHlCBPjD3sHX04of-hamwKHitAloVT-ws~Lm79eqhq4rkNiSUhVjNShbHm9piwg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="AnABlast profiles suggesting modifications in annotated intron and pseudogenes. (A) Profile generated in the first intron (Chr I: 683035-683098, forward strand) of the snu23 gene. Ribo-Seq data of the snu23 gene region are shown. (B) Profile identifying a DNA sequence encoding a putative dipeptide transmembrane transporter (Chr II: 4462621-4463890, reverse strand) similar to Schizosaccharomyces cryophilus EPY52281.1. RNA-Seq data of this region are shown. (C) Amino- and carboxyl-terminal expansions (forward strand in Chr II: 87554-87737 and Chr II: 89014-89176, respectively) highlighted by AnABlast analysis in the pseudogene SPBPB10D8.03 (arrows). The concatenated amino acid sequence of the different reading frame regions of this pseudogene, including carboxyl- and amino-terminal AnaBlast expansions, generated a protein 51% identical to a phthalate transporter of S. cryophilus. (D) AnABlast peaks (Chr I: 2955377-2955222 and Chr I: 29555220-29554983 respectively) (arrows) suggest changes in exons annotation in the pseudogene SPAPB24D3.05c (Chr I: 2955350-2955194 and Chr I: 2955191-2954983, respectively). Concatenated amino acid sequence of the reading frames predicted by AnABlast profiles produce a hypothetical pseudogene protein 65% identical to glyoxalase bacterial proteins. Pombase annotations and colour codes for reading frames of the analysed genomic intervals are shown. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02504.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199857/dsv02504.tiff" path-from-xml="dsv02504.tiff" target="_blank">View large</a><a section="87199857" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv02504.png?Expires=2147483647&Signature=MOTY--j6AKenT2k0ntQpc~7dYrekupDXYKoYv~iLA-7aaYPW5acQivqLIy9K-EsdHE~iWq~zBydnvZlPpPsxaclCW4ZYO5cIzp0UR40-jLY659gg8wIodwYqH7Ghx1fVquz3-4UAEpUEjH7WLDMal6klbH4jpY1kPm7VhDl2n56ILOljeMDWNsIdcacljMQp40wuTNiR1yLIUNoLvF693QJ09jDmbt7EsCpG4wYL72DGSAMSs-Q~gIoefyt~h0SAit1cFtvghmc0nQtq-DZPG1oyuy5J797lG4uWcIZ~UfK~ttbNUCndbOAr8dl5pQd8I1AIMHgv69NK-twCiGOkww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199857&ar=2462479&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02504.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>AnABlast profiles suggesting modifications in annotated intron and pseudogenes. (A) Profile generated in the first intron (Chr I: 683035-683098, forward strand) of the <em>snu23</em> gene. Ribo-Seq data of the <em>snu23</em> gene region are shown. (B) Profile identifying a DNA sequence encoding a putative dipeptide transmembrane transporter (Chr II: 4462621-4463890, reverse strand) similar to <em>Schizosaccharomyces cryophilus</em> EPY52281.1. RNA-Seq data of this region are shown. (C) Amino- and carboxyl-terminal expansions (forward strand in Chr II: 87554-87737 and Chr II: 89014-89176, respectively) highlighted by AnABlast analysis in the pseudogene SPBPB10D8.03 (arrows). The concatenated amino acid sequence of the different reading frame regions of this pseudogene, including carboxyl- and amino-terminal AnaBlast expansions, generated a protein 51% identical to a phthalate transporter of <em>S. cryophilus</em>. (D) AnABlast peaks (Chr I: 2955377-2955222 and Chr I: 29555220-29554983 respectively) (arrows) suggest changes in exons annotation in the pseudogene SPAPB24D3.05c (Chr I: 2955350-2955194 and Chr I: 2955191-2954983, respectively). Concatenated amino acid sequence of the reading frames predicted by AnABlast profiles produce a hypothetical pseudogene protein 65% identical to glyoxalase bacterial proteins. Pombase annotations and colour codes for reading frames of the analysed genomic intervals are shown. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV025F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/m_dsv02504.png?Expires=2147483647&Signature=CCOZHSefrMgggN0YoA1F1Re0WLVMCsgEhK~myfWEdnOX98TPI61N~PiyiH~nRxUfUv2Hqozy-O3iKRhxBoKdll6qFoo3T-vCtoCDG61v4J0sdjvA4pA9OyKHYhXiS3Ygtqu7K9nya3pOvOEkagw2Ch1t8tA2K9PhOoHYHXv0fpS4IEkmXZKglv8bBgCwYU7OJ8bWOPoSJfEisy-O~izIMg3ladlu13q69cXMGj29qQr~EjLpOZdPIpM~vDZyUgfNFjnSWoFWRk5mXl0Su1Dv1E6zHlCBPjD3sHX04of-hamwKHitAloVT-ws~Lm79eqhq4rkNiSUhVjNShbHm9piwg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="AnABlast profiles suggesting modifications in annotated intron and pseudogenes. (A) Profile generated in the first intron (Chr I: 683035-683098, forward strand) of the snu23 gene. Ribo-Seq data of the snu23 gene region are shown. (B) Profile identifying a DNA sequence encoding a putative dipeptide transmembrane transporter (Chr II: 4462621-4463890, reverse strand) similar to Schizosaccharomyces cryophilus EPY52281.1. RNA-Seq data of this region are shown. (C) Amino- and carboxyl-terminal expansions (forward strand in Chr II: 87554-87737 and Chr II: 89014-89176, respectively) highlighted by AnABlast analysis in the pseudogene SPBPB10D8.03 (arrows). The concatenated amino acid sequence of the different reading frame regions of this pseudogene, including carboxyl- and amino-terminal AnaBlast expansions, generated a protein 51% identical to a phthalate transporter of S. cryophilus. (D) AnABlast peaks (Chr I: 2955377-2955222 and Chr I: 29555220-29554983 respectively) (arrows) suggest changes in exons annotation in the pseudogene SPAPB24D3.05c (Chr I: 2955350-2955194 and Chr I: 2955191-2954983, respectively). Concatenated amino acid sequence of the reading frames predicted by AnABlast profiles produce a hypothetical pseudogene protein 65% identical to glyoxalase bacterial proteins. Pombase annotations and colour codes for reading frames of the analysed genomic intervals are shown. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02504.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199857/dsv02504.tiff" path-from-xml="dsv02504.tiff" target="_blank">View large</a><a section="87199857" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv02504.png?Expires=2147483647&Signature=MOTY--j6AKenT2k0ntQpc~7dYrekupDXYKoYv~iLA-7aaYPW5acQivqLIy9K-EsdHE~iWq~zBydnvZlPpPsxaclCW4ZYO5cIzp0UR40-jLY659gg8wIodwYqH7Ghx1fVquz3-4UAEpUEjH7WLDMal6klbH4jpY1kPm7VhDl2n56ILOljeMDWNsIdcacljMQp40wuTNiR1yLIUNoLvF693QJ09jDmbt7EsCpG4wYL72DGSAMSs-Q~gIoefyt~h0SAit1cFtvghmc0nQtq-DZPG1oyuy5J797lG4uWcIZ~UfK~ttbNUCndbOAr8dl5pQd8I1AIMHgv69NK-twCiGOkww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199857&ar=2462479&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02504.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>AnABlast profiles suggesting modifications in annotated intron and pseudogenes. (A) Profile generated in the first intron (Chr I: 683035-683098, forward strand) of the <em>snu23</em> gene. Ribo-Seq data of the <em>snu23</em> gene region are shown. (B) Profile identifying a DNA sequence encoding a putative dipeptide transmembrane transporter (Chr II: 4462621-4463890, reverse strand) similar to <em>Schizosaccharomyces cryophilus</em> EPY52281.1. RNA-Seq data of this region are shown. (C) Amino- and carboxyl-terminal expansions (forward strand in Chr II: 87554-87737 and Chr II: 89014-89176, respectively) highlighted by AnABlast analysis in the pseudogene SPBPB10D8.03 (arrows). The concatenated amino acid sequence of the different reading frame regions of this pseudogene, including carboxyl- and amino-terminal AnaBlast expansions, generated a protein 51% identical to a phthalate transporter of <em>S. cryophilus</em>. (D) AnABlast peaks (Chr I: 2955377-2955222 and Chr I: 29555220-29554983 respectively) (arrows) suggest changes in exons annotation in the pseudogene SPAPB24D3.05c (Chr I: 2955350-2955194 and Chr I: 2955191-2954983, respectively). Concatenated amino acid sequence of the reading frames predicted by AnABlast profiles produce a hypothetical pseudogene protein 65% identical to glyoxalase bacterial proteins. Pombase annotations and colour codes for reading frames of the analysed genomic intervals are shown. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87199858 id="87199858" class="section-title" data-legacy-id=s3c>3.3. AnABlast peaks highlighting pseudogenes</h3>
<p>Annotated transposons and pseudogenes were not excluded for intergenic regions analysis. Accordingly, a number of transposons and all 29 annotated pseudogenes were detected in AnABlast profiles (see examples in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv025_Supplementary_Data.zip?Expires=2147483647&Signature=1~jmJkVjh9nWQGIj2THUr5D9ZyIt-dff-I6YJG0D2WHfVKRiS1ULEF8wlV0~qsgibk4428hR8nTVJzlTk6AqCuZmrkL7RL9z~ZZxHzalAz43uaxTVm2M~i7SNOaxn3bja6jg-TLg9eJyoDg0ClySTnk6Yamj8pZn59POioyLbyBdL9MhSXahN9QyofilJXcuTB20yMCM2qcZ49WPg~C8vy1KkIaRIVtQe6SyNF8JAZ3lOlLDSUCSKXCXHxj1HVV8H7hEemFxddd9t0xQdjDWROOcsjpxn19QYay0xjvA5kdiz5eAUbmvOg95Yu2rzrHsH5dc7JXZyL9yqwya2DhrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). In addition, AnABlast peaks uncovered a new pseudogene (Fig. <a reveal-id="DSV025F4" data-open="DSV025F4" class="link link-reveal link-table xref-fig">4</a>B). Reading frames in this pseudogene contain numerous stop codons and at least five frameshifts, a complex structure that makes its identification by conventional methods difficult (see Fig. <a reveal-id="DSV025F4" data-open="DSV025F4" class="link link-reveal link-table xref-fig">4</a>B). BLAST analysis of predicted amino acid sequences indicated that this new pseudogene encodes a putative dipeptide transmembrane transporter protein (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv025_Supplementary_Data.zip?Expires=2147483647&Signature=1~jmJkVjh9nWQGIj2THUr5D9ZyIt-dff-I6YJG0D2WHfVKRiS1ULEF8wlV0~qsgibk4428hR8nTVJzlTk6AqCuZmrkL7RL9z~ZZxHzalAz43uaxTVm2M~i7SNOaxn3bja6jg-TLg9eJyoDg0ClySTnk6Yamj8pZn59POioyLbyBdL9MhSXahN9QyofilJXcuTB20yMCM2qcZ49WPg~C8vy1KkIaRIVtQe6SyNF8JAZ3lOlLDSUCSKXCXHxj1HVV8H7hEemFxddd9t0xQdjDWROOcsjpxn19QYay0xjvA5kdiz5eAUbmvOg95Yu2rzrHsH5dc7JXZyL9yqwya2DhrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1A</a></span>). A second new pseudogene uncovered within rearranged DNA regions will be described later.</p><p>Detailed analysis of AnABlast profiles generated in annotated pseudogenes suggested a number of corrections to their genomic annotations. Two additional coding domains (in different reading frames) flanking the SPBPB10D8.03 pseudogene suggest carboxyl- and amino-terminal expansions in the annotated region (Fig. <a reveal-id="DSV025F4" data-open="DSV025F4" class="link link-reveal link-table xref-fig">4</a>C and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv025_Supplementary_Data.zip?Expires=2147483647&Signature=1~jmJkVjh9nWQGIj2THUr5D9ZyIt-dff-I6YJG0D2WHfVKRiS1ULEF8wlV0~qsgibk4428hR8nTVJzlTk6AqCuZmrkL7RL9z~ZZxHzalAz43uaxTVm2M~i7SNOaxn3bja6jg-TLg9eJyoDg0ClySTnk6Yamj8pZn59POioyLbyBdL9MhSXahN9QyofilJXcuTB20yMCM2qcZ49WPg~C8vy1KkIaRIVtQe6SyNF8JAZ3lOlLDSUCSKXCXHxj1HVV8H7hEemFxddd9t0xQdjDWROOcsjpxn19QYay0xjvA5kdiz5eAUbmvOg95Yu2rzrHsH5dc7JXZyL9yqwya2DhrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1A</a></span>). Another correction is suggested in the two annotated exons of the SPAPB24D3.05c pseudogene. AnABlast also identified two regions separated by a single frameshift, although region intervals and the reading frame for the first region differ from those in the annotated exons (Fig. <a reveal-id="DSV025F4" data-open="DSV025F4" class="link link-reveal link-table xref-fig">4</a>D and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv025_Supplementary_Data.zip?Expires=2147483647&Signature=1~jmJkVjh9nWQGIj2THUr5D9ZyIt-dff-I6YJG0D2WHfVKRiS1ULEF8wlV0~qsgibk4428hR8nTVJzlTk6AqCuZmrkL7RL9z~ZZxHzalAz43uaxTVm2M~i7SNOaxn3bja6jg-TLg9eJyoDg0ClySTnk6Yamj8pZn59POioyLbyBdL9MhSXahN9QyofilJXcuTB20yMCM2qcZ49WPg~C8vy1KkIaRIVtQe6SyNF8JAZ3lOlLDSUCSKXCXHxj1HVV8H7hEemFxddd9t0xQdjDWROOcsjpxn19QYay0xjvA5kdiz5eAUbmvOg95Yu2rzrHsH5dc7JXZyL9yqwya2DhrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1A</a></span>). Aside corrections, AnABlast peaks matching Ribo-Seq data profiles in pseudogenes suggest that at least two of them could be reassigned as coding genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv025_Supplementary_Data.zip?Expires=2147483647&Signature=1~jmJkVjh9nWQGIj2THUr5D9ZyIt-dff-I6YJG0D2WHfVKRiS1ULEF8wlV0~qsgibk4428hR8nTVJzlTk6AqCuZmrkL7RL9z~ZZxHzalAz43uaxTVm2M~i7SNOaxn3bja6jg-TLg9eJyoDg0ClySTnk6Yamj8pZn59POioyLbyBdL9MhSXahN9QyofilJXcuTB20yMCM2qcZ49WPg~C8vy1KkIaRIVtQe6SyNF8JAZ3lOlLDSUCSKXCXHxj1HVV8H7hEemFxddd9t0xQdjDWROOcsjpxn19QYay0xjvA5kdiz5eAUbmvOg95Yu2rzrHsH5dc7JXZyL9yqwya2DhrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>).</p><p>From these results, we conclude that our <em>in silico</em> strategy efficiently highlights coding sequences in actual genes (Fig. <a reveal-id="DSV025F3" data-open="DSV025F3" class="link link-reveal link-table xref-fig">3</a>) as well as in pseudogenes (Fig. <a reveal-id="DSV025F4" data-open="DSV025F4" class="link link-reveal link-table xref-fig">4</a>). Since part of the pseudogenization process includes loss of initiating methionine, nonsense, and frameshift mutations, AnABlast might be useful to identify new genes, but also evolutionary degenerated coding sequences that escape to conventional searches.</p>                            <h3 scrollto-destination=87199862 id="87199862" class="section-title" data-legacy-id=s3d>3.4. Identification of new genes</h3>
<p>The <em>S. pombe</em> genome is extremely well annotated.<sup><a reveal-id="DSV025C16" data-open="DSV025C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSV025C20" data-open="DSV025C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Thus, a stringent test for AnABlast is the identification of new genes in the genome of this model organism. To this end, we initially looked for AnABlast peaks containing ORFs, and 18 of them were selected.</p><p>Pfam and BLAST searches found that three of the selected ORFs uncovered new copies of annotated genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv025_Supplementary_Data.zip?Expires=2147483647&Signature=1~jmJkVjh9nWQGIj2THUr5D9ZyIt-dff-I6YJG0D2WHfVKRiS1ULEF8wlV0~qsgibk4428hR8nTVJzlTk6AqCuZmrkL7RL9z~ZZxHzalAz43uaxTVm2M~i7SNOaxn3bja6jg-TLg9eJyoDg0ClySTnk6Yamj8pZn59POioyLbyBdL9MhSXahN9QyofilJXcuTB20yMCM2qcZ49WPg~C8vy1KkIaRIVtQe6SyNF8JAZ3lOlLDSUCSKXCXHxj1HVV8H7hEemFxddd9t0xQdjDWROOcsjpxn19QYay0xjvA5kdiz5eAUbmvOg95Yu2rzrHsH5dc7JXZyL9yqwya2DhrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). Importantly, four other ORF-containing AnABlast peaks identified putative new protein-coding genes, experimentally validated by Ribo-Seq expression patterns<sup><a reveal-id="DSV025C18" data-open="DSV025C18" class="link link-ref link-reveal xref-bibr">18</a></sup> (Fig. <a reveal-id="DSV025F5" data-open="DSV025F5" class="link link-reveal link-table xref-fig">5</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv025_Supplementary_Data.zip?Expires=2147483647&Signature=1~jmJkVjh9nWQGIj2THUr5D9ZyIt-dff-I6YJG0D2WHfVKRiS1ULEF8wlV0~qsgibk4428hR8nTVJzlTk6AqCuZmrkL7RL9z~ZZxHzalAz43uaxTVm2M~i7SNOaxn3bja6jg-TLg9eJyoDg0ClySTnk6Yamj8pZn59POioyLbyBdL9MhSXahN9QyofilJXcuTB20yMCM2qcZ49WPg~C8vy1KkIaRIVtQe6SyNF8JAZ3lOlLDSUCSKXCXHxj1HVV8H7hEemFxddd9t0xQdjDWROOcsjpxn19QYay0xjvA5kdiz5eAUbmvOg95Yu2rzrHsH5dc7JXZyL9yqwya2DhrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1B</a></span>). </p>                        <a id="87199865" scrollto-destination="87199865"></a>
<div id="DSV025F5" data-id="DSV025F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/m_dsv02505.png?Expires=2147483647&Signature=xJOyfsX6ho7qSGCwtyuKkJXRTo~sPrfProAjeRu8JEPICE7z~7hlW0DXDK653F8s4W6-ydFmlRTm-zufUPoF7G6OHCkGiuXC1RsaTUniL~9cblZpCc5S2fBw-GUZtSJzdlsH34yhDoHdK6CHT6CX6sg~FmX9BR02CA0v4Rg7hbzaZhw66gAziLJQkxQyfgVKMz7xj7LUZdsxk6Gp~ZLpCdzgqOAqtJkC-d0XqSqfkf20pjPn-0N0WXfvxAAHI7Zr1DjpxST-9eijUpmD5xiuJw6BzKwByxdAzPIb3hFLX~rhfd9hpYbUg1G~StpZfJfuZB3GFZHh5OcliG2O2C1ykA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="AnABlast profiles (accumulated alignments) identifying putative new genes. (A) AnaBlast peak at Chr I: 2975642-2975772 (forward strand) and (B) peak at Chr I: 127230-127316 (forward strand) (arrow) encode small peptides with no significant similarity to known proteins in databases. (C) Peak at Chr I: 5139023-5139195 (forward strand) identify a coding sequence similar to the SPOG_01629 protein of Schizosaccharomyces cryophilus. (D) Peak at Chr II: 3391178-3391413 (Chr II: 3391181-3391414, forward strand), uncovering a coding sequences with different degrees of similarity to protein SPOG_01213 from S. cryophilus and SOCG_06140 from Schizosaccharomyces octosporus, an hypothetical protein from a large number of filamentous fungi, and another one from the fresh water cyanobacteria Microcystis aeruginosa. (Schematic BLAST results are included). Ribo-Seq data, Pombase annotations, and colour codes for reading frames of the analysed genomic intervals are shown. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02505.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199865/dsv02505.tiff" path-from-xml="dsv02505.tiff" target="_blank">View large</a><a section="87199865" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv02505.png?Expires=2147483647&Signature=hUBz9GK-RVhmzcauHcXuiuUQrQNBYWwDoW8pE-VZx5UkghvzzohpE2urCI7DxKvWUsDhK0CK19vMgmVottkYLQR45bKZWfYdmi7sU5t1dbKWhkdiaSyo1pWNlXmmrguAixxaS-eXFmTAMcYO3UtklyOttMFIHi41D~xYpYC-r8JOeqzLrdieJSpfAfAdY4J1edBT6vzuwtaxbQDcA8DjVYzdxRlTxDKQpYgfcUDqNNnq46Lou7vIYzDQ0VM6EQingo6HNhgk~KzH3zbjj1phBYJjvXhq17Z5tB9Fs7RRTWjxdu6Iiz7V1bDoUUKxbTrbu~nTmxYudTEPLsb4OhUGeQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199865&ar=2462479&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02505.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>AnABlast profiles (accumulated alignments) identifying putative new genes. (A) AnaBlast peak at Chr I: 2975642-2975772 (forward strand) and (B) peak at Chr I: 127230-127316 (forward strand) (arrow) encode small peptides with no significant similarity to known proteins in databases. (C) Peak at Chr I: 5139023-5139195 (forward strand) identify a coding sequence similar to the SPOG_01629 protein of <em>Schizosaccharomyces cryophilus</em>. (D) Peak at Chr II: 3391178-3391413 (Chr II: 3391181-3391414, forward strand), uncovering a coding sequences with different degrees of similarity to protein SPOG_01213 from <em>S. cryophilus</em> and SOCG_06140 from <em>Schizosaccharomyces octosporus</em>, an hypothetical protein from a large number of filamentous fungi, and another one from the fresh water cyanobacteria <em>Microcystis aeruginosa</em>. (Schematic BLAST results are included). Ribo-Seq data, Pombase annotations, and colour codes for reading frames of the analysed genomic intervals are shown. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV025F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/m_dsv02505.png?Expires=2147483647&Signature=xJOyfsX6ho7qSGCwtyuKkJXRTo~sPrfProAjeRu8JEPICE7z~7hlW0DXDK653F8s4W6-ydFmlRTm-zufUPoF7G6OHCkGiuXC1RsaTUniL~9cblZpCc5S2fBw-GUZtSJzdlsH34yhDoHdK6CHT6CX6sg~FmX9BR02CA0v4Rg7hbzaZhw66gAziLJQkxQyfgVKMz7xj7LUZdsxk6Gp~ZLpCdzgqOAqtJkC-d0XqSqfkf20pjPn-0N0WXfvxAAHI7Zr1DjpxST-9eijUpmD5xiuJw6BzKwByxdAzPIb3hFLX~rhfd9hpYbUg1G~StpZfJfuZB3GFZHh5OcliG2O2C1ykA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="AnABlast profiles (accumulated alignments) identifying putative new genes. (A) AnaBlast peak at Chr I: 2975642-2975772 (forward strand) and (B) peak at Chr I: 127230-127316 (forward strand) (arrow) encode small peptides with no significant similarity to known proteins in databases. (C) Peak at Chr I: 5139023-5139195 (forward strand) identify a coding sequence similar to the SPOG_01629 protein of Schizosaccharomyces cryophilus. (D) Peak at Chr II: 3391178-3391413 (Chr II: 3391181-3391414, forward strand), uncovering a coding sequences with different degrees of similarity to protein SPOG_01213 from S. cryophilus and SOCG_06140 from Schizosaccharomyces octosporus, an hypothetical protein from a large number of filamentous fungi, and another one from the fresh water cyanobacteria Microcystis aeruginosa. (Schematic BLAST results are included). Ribo-Seq data, Pombase annotations, and colour codes for reading frames of the analysed genomic intervals are shown. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02505.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199865/dsv02505.tiff" path-from-xml="dsv02505.tiff" target="_blank">View large</a><a section="87199865" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv02505.png?Expires=2147483647&Signature=hUBz9GK-RVhmzcauHcXuiuUQrQNBYWwDoW8pE-VZx5UkghvzzohpE2urCI7DxKvWUsDhK0CK19vMgmVottkYLQR45bKZWfYdmi7sU5t1dbKWhkdiaSyo1pWNlXmmrguAixxaS-eXFmTAMcYO3UtklyOttMFIHi41D~xYpYC-r8JOeqzLrdieJSpfAfAdY4J1edBT6vzuwtaxbQDcA8DjVYzdxRlTxDKQpYgfcUDqNNnq46Lou7vIYzDQ0VM6EQingo6HNhgk~KzH3zbjj1phBYJjvXhq17Z5tB9Fs7RRTWjxdu6Iiz7V1bDoUUKxbTrbu~nTmxYudTEPLsb4OhUGeQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199865&ar=2462479&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02505.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>AnABlast profiles (accumulated alignments) identifying putative new genes. (A) AnaBlast peak at Chr I: 2975642-2975772 (forward strand) and (B) peak at Chr I: 127230-127316 (forward strand) (arrow) encode small peptides with no significant similarity to known proteins in databases. (C) Peak at Chr I: 5139023-5139195 (forward strand) identify a coding sequence similar to the SPOG_01629 protein of <em>Schizosaccharomyces cryophilus</em>. (D) Peak at Chr II: 3391178-3391413 (Chr II: 3391181-3391414, forward strand), uncovering a coding sequences with different degrees of similarity to protein SPOG_01213 from <em>S. cryophilus</em> and SOCG_06140 from <em>Schizosaccharomyces octosporus</em>, an hypothetical protein from a large number of filamentous fungi, and another one from the fresh water cyanobacteria <em>Microcystis aeruginosa</em>. (Schematic BLAST results are included). Ribo-Seq data, Pombase annotations, and colour codes for reading frames of the analysed genomic intervals are shown. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>The first novel gene partially overlaps the predicted poly(A)-bearing non-coding RNA SPNCRNA.31. Both the AnABlast ORF (42 amino acid length) and the annotated non-coding RNA are found in the forward strand, with the Ribo-Seq profile confirming the translation of the ORF (Fig. <a reveal-id="DSV025F5" data-open="DSV025F5" class="link link-reveal link-table xref-fig">5</a>A).</p><p>The second new gene contains a short 28 residues ORF that is highly expressed (Fig. <a reveal-id="DSV025F5" data-open="DSV025F5" class="link link-reveal link-table xref-fig">5</a>B). The corresponding AnABlast peak further extends the carboxyl-terminal end of this ORF (see in Fig. <a reveal-id="DSV025F5" data-open="DSV025F5" class="link link-reveal link-table xref-fig">5</a>B), suggesting that this extremely short gene could be originated through a carboxyl-terminal subtraction from a longer ancient one.<sup><a reveal-id="DSV025C26" data-open="DSV025C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Polypeptides encoded by these two new genes lack significant homologs or motifs in databases.</p><p>The other two AnABlast ORFs, in addition to being translated, encode polypeptides that share significant similarity to known proteins. A main AnABlast peak, flanked by two others in different reading frames, identifies one of these new genes (see in Fig. <a reveal-id="DSV025F5" data-open="DSV025F5" class="link link-reveal link-table xref-fig">5</a>C). Concatenated amino acid sequences from the three peaks render a 178-residue protein with a homologue in <em>Schizosaccharomyces octosporus</em> and eight in <em>Schizosaccharomyces cryophilus</em>, ranging in length from 102 to 190 residues (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv025_Supplementary_Data.zip?Expires=2147483647&Signature=1~jmJkVjh9nWQGIj2THUr5D9ZyIt-dff-I6YJG0D2WHfVKRiS1ULEF8wlV0~qsgibk4428hR8nTVJzlTk6AqCuZmrkL7RL9z~ZZxHzalAz43uaxTVm2M~i7SNOaxn3bja6jg-TLg9eJyoDg0ClySTnk6Yamj8pZn59POioyLbyBdL9MhSXahN9QyofilJXcuTB20yMCM2qcZ49WPg~C8vy1KkIaRIVtQe6SyNF8JAZ3lOlLDSUCSKXCXHxj1HVV8H7hEemFxddd9t0xQdjDWROOcsjpxn19QYay0xjvA5kdiz5eAUbmvOg95Yu2rzrHsH5dc7JXZyL9yqwya2DhrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1B</a></span>). RNA expression profile (see Ribo-Seq pattern in Fig. <a reveal-id="DSV025F5" data-open="DSV025F5" class="link link-reveal link-table xref-fig">5</a>C) and its homology to the short EPY50361.1 <em>S. cryophilus</em> variant (see alignments in Fig. <a reveal-id="DSV025F5" data-open="DSV025F5" class="link link-reveal link-table xref-fig">5</a>C) suggest that only the 56 residues ORF encoded by the first AnABlast peak is an active gene. Thus, the proposed <em>S. pombe</em> gene (and its equivalent in <em>Schizosaccharomyces cryophilus</em>) likely resulted from premature nonsense mutations, generating a fossil carboxyl-terminal sequence that in the remaining related proteins is still active.</p><p>Another AnABlast ORF encodes a 77-residue polypeptide sharing 29% identity (52% similarity) along the entire sequence of the WP_002758301.1 protein from <em>Microcystis aeruginosa</em>, an evolutionary distant freshwater cyanobacteria (Fig. <a reveal-id="DSV025F5" data-open="DSV025F5" class="link link-reveal link-table xref-fig">5</a>D). Identity increases to 35% with homolog proteins found in filamentous fungi, but homology length decreases to the first 52 residues in these closer microorganisms. Identity with homolog sequences found in <em>S. cryophilus</em> and <em>S. octosporus</em> rises to 76% but is limited to 31 amino-terminal residues. Since RNA expression is also observed at this 31 amino acids interval, we conclude that this short region is an active coding sequence in fission yeasts. According to BLAST alignments (Fig. <a reveal-id="DSV025F5" data-open="DSV025F5" class="link link-reveal link-table xref-fig">5</a>D), premature nonsense mutations could also result in the progressive carboxyl-terminal subtractions of this coding sequence over evolution.<sup><a reveal-id="DSV025C26" data-open="DSV025C26" class="link link-ref link-reveal xref-bibr">26</a></sup> However, in <em>S. pombe</em>, no stop codon is found at the 3′ end position of the DNA sequence coding for the 31-residue domain (see in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv025_Supplementary_Data.zip?Expires=2147483647&Signature=1~jmJkVjh9nWQGIj2THUr5D9ZyIt-dff-I6YJG0D2WHfVKRiS1ULEF8wlV0~qsgibk4428hR8nTVJzlTk6AqCuZmrkL7RL9z~ZZxHzalAz43uaxTVm2M~i7SNOaxn3bja6jg-TLg9eJyoDg0ClySTnk6Yamj8pZn59POioyLbyBdL9MhSXahN9QyofilJXcuTB20yMCM2qcZ49WPg~C8vy1KkIaRIVtQe6SyNF8JAZ3lOlLDSUCSKXCXHxj1HVV8H7hEemFxddd9t0xQdjDWROOcsjpxn19QYay0xjvA5kdiz5eAUbmvOg95Yu2rzrHsH5dc7JXZyL9yqwya2DhrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). In fact, RNA-Seq data suggest the existence of at least two exons in this region (see Fig. <a reveal-id="DSV025F5" data-open="DSV025F5" class="link link-reveal link-table xref-fig">5</a>D). Thus, in this particular case, subtraction likely raised in the <em>S. pombe</em> genome by the creation of a new splice site.<sup><a reveal-id="DSV025C27" data-open="DSV025C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DSV025C28" data-open="DSV025C28" class="link link-ref link-reveal xref-bibr">28</a></sup></p><p>We cannot rule out the existence of new genes in the remaining ORFs found by AnABlast search. However, neither significant translation levels nor sequence/motif conservation support their actual gene status (see examples in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv025_Supplementary_Data.zip?Expires=2147483647&Signature=1~jmJkVjh9nWQGIj2THUr5D9ZyIt-dff-I6YJG0D2WHfVKRiS1ULEF8wlV0~qsgibk4428hR8nTVJzlTk6AqCuZmrkL7RL9z~ZZxHzalAz43uaxTVm2M~i7SNOaxn3bja6jg-TLg9eJyoDg0ClySTnk6Yamj8pZn59POioyLbyBdL9MhSXahN9QyofilJXcuTB20yMCM2qcZ49WPg~C8vy1KkIaRIVtQe6SyNF8JAZ3lOlLDSUCSKXCXHxj1HVV8H7hEemFxddd9t0xQdjDWROOcsjpxn19QYay0xjvA5kdiz5eAUbmvOg95Yu2rzrHsH5dc7JXZyL9yqwya2DhrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>).</p><p>The <em>in silico</em> identification of new genes is particularly useful for genome annotations. To test the potential use of AnABlast as a gene-finger tool in other organisms, we analysed the Chromosome II:0-5 Mb interval of the <em>C. elegans</em> genome. Annotated ORFs were efficiently highlighted in AnABlast profiles using the settings trained for the <em>S. pombe</em> genome. Furthermore, a detailed analysis of intergenic regions in this interval revealed that AnABlast profiles can also be useful to identify new genes in the nematode genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv025_Supplementary_Data.zip?Expires=2147483647&Signature=1~jmJkVjh9nWQGIj2THUr5D9ZyIt-dff-I6YJG0D2WHfVKRiS1ULEF8wlV0~qsgibk4428hR8nTVJzlTk6AqCuZmrkL7RL9z~ZZxHzalAz43uaxTVm2M~i7SNOaxn3bja6jg-TLg9eJyoDg0ClySTnk6Yamj8pZn59POioyLbyBdL9MhSXahN9QyofilJXcuTB20yMCM2qcZ49WPg~C8vy1KkIaRIVtQe6SyNF8JAZ3lOlLDSUCSKXCXHxj1HVV8H7hEemFxddd9t0xQdjDWROOcsjpxn19QYay0xjvA5kdiz5eAUbmvOg95Yu2rzrHsH5dc7JXZyL9yqwya2DhrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>).</p>                            <h3 scrollto-destination=87199872 id="87199872" class="section-title" data-legacy-id=s3e>3.5. Ancestral gene subtractions</h3>
<p>As shown above, our coding sequences search strategy can both find new genes and uncover their associated evolutionary history, revealing fossil-coding sequences that have being subtracted by changes in stop positions (Figs <a reveal-id="DSV025F5" data-open="DSV025F5" class="link link-reveal link-table xref-fig">5</a>B and <a reveal-id="DSV025F3" data-open="DSV025F3" class="link link-reveal link-table xref-fig">3</a>C) or in splice sites (Fig. <a reveal-id="DSV025F5" data-open="DSV025F5" class="link link-reveal link-table xref-fig">5</a>D). To locate possible fossil coding sequences subtracted from present genes, we examined peaks located at the edge (exon boundaries) of AnABlast profiles at each inter-exon region. Peaks revealing amino acid sequences encoded in the same strand as adjacent exons were selected (putative ancient subtractions), and their corresponding amino acid sequences concatenated ignoring reading frameshifts and stop codons. Interestingly, BLAST searches identified two genes in which alignments to homologous proteins extended along the concatenated peptides, one by expanding the 5′ end coding DNA sequence and the other through a 3′ end extension (Fig. <a reveal-id="DSV025F6" data-open="DSV025F6" class="link link-reveal link-table xref-fig">6</a>A and <a reveal-id="DSV025F4" data-open="DSV025F4" class="link link-reveal link-table xref-fig">4</a>B, and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv025_Supplementary_Data.zip?Expires=2147483647&Signature=1~jmJkVjh9nWQGIj2THUr5D9ZyIt-dff-I6YJG0D2WHfVKRiS1ULEF8wlV0~qsgibk4428hR8nTVJzlTk6AqCuZmrkL7RL9z~ZZxHzalAz43uaxTVm2M~i7SNOaxn3bja6jg-TLg9eJyoDg0ClySTnk6Yamj8pZn59POioyLbyBdL9MhSXahN9QyofilJXcuTB20yMCM2qcZ49WPg~C8vy1KkIaRIVtQe6SyNF8JAZ3lOlLDSUCSKXCXHxj1HVV8H7hEemFxddd9t0xQdjDWROOcsjpxn19QYay0xjvA5kdiz5eAUbmvOg95Yu2rzrHsH5dc7JXZyL9yqwya2DhrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1C</a></span>). </p>                        <a id="87199874" scrollto-destination="87199874"></a>
<div id="DSV025F6" data-id="DSV025F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/m_dsv02506.png?Expires=2147483647&Signature=Mvj9rIRD9yP4~3BYstg2N95HlngH1ZPDjlkRsxW3j-r1yL6NaVKh5cU7a7lLHi89gclQgA3V-aCve6~rD6XmjP24C9GNYBHUJ-7ODDnPPkDkdVmFFrP19DSyO1xdulMcF8rWx~9Cmv-mSqw2OdZg4BBd88oeumld1nwW7tt6ZIkL~BC0jaovfyKZVUSqKGCSIyPXjHZtsKwmm0HqmGgH4pHFD6Bhtyvzgss5wbxZ-YRc8KcyH67lZpxwy9L3y~Qs4aj9bvNsMiK2vdxluDVeemUJlu26erK9XqBDd3oV-jfP2k4mrSzIXO~ZlKAuDfigZsW9eZXqN~ujmftb6yqr2g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="AnABlast peaks uncovering evolutionary carboxyl- and amino-terminal subtractions in present genes. (A) AnABlast peak (arrow) at Chr I: 127.165-128.049 (reverse strand) located in the 3′UTR of SPAC11D3.11c reveals a carboxyl-terminal subtraction of this gene. (B) Peak at Chr I: 5542072-5542417 (forward strand), partially overlapping the 5′ UTR of SPAC186.06, uncovers an evolutionary amino-terminal subtraction of the corresponding gene. Ribo-Seq data, Pombase annotations, and colour codes for reading frames of the analysed genomic intervals are shown. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02506.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199874/dsv02506.tiff" path-from-xml="dsv02506.tiff" target="_blank">View large</a><a section="87199874" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv02506.png?Expires=2147483647&Signature=go4kyjmjsrsfEn8tkMohetgP9g4jRrmNkpw~BK20wl38eOBhsqJd44Q-3pgCBOJoeWfiyBYwFuTFY~QnpbdpmpoMtE1QTDgAFJ0kFNKOpfPjuPLmZ~i5VomYRUdamKBmPWKmSKFvbkFMgwyvniLETC6VkixbQR0IID3NaGxGjzDgs3BsNO7QYrhILiVsbCYkCrPEllqlPYkq5wR80H7Td2fLNtu8-aLSQatzxInjbVT572vRTgsCqcLiTgKC0s582H2JV8~A0ua-AcEYKACpt5HR~ry8xA1weQLkRE68LiRBOsmlIbQ7jE8IbOwUIRntmCJ27q1YMU8cdsc6f7uWog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199874&ar=2462479&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02506.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>AnABlast peaks uncovering evolutionary carboxyl- and amino-terminal subtractions in present genes. (A) AnABlast peak (arrow) at Chr I: 127.165-128.049 (reverse strand) located in the 3′UTR of SPAC11D3.11c reveals a carboxyl-terminal subtraction of this gene. (B) Peak at Chr I: 5542072-5542417 (forward strand), partially overlapping the 5′ UTR of SPAC186.06, uncovers an evolutionary amino-terminal subtraction of the corresponding gene. Ribo-Seq data, Pombase annotations, and colour codes for reading frames of the analysed genomic intervals are shown. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV025F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/m_dsv02506.png?Expires=2147483647&Signature=Mvj9rIRD9yP4~3BYstg2N95HlngH1ZPDjlkRsxW3j-r1yL6NaVKh5cU7a7lLHi89gclQgA3V-aCve6~rD6XmjP24C9GNYBHUJ-7ODDnPPkDkdVmFFrP19DSyO1xdulMcF8rWx~9Cmv-mSqw2OdZg4BBd88oeumld1nwW7tt6ZIkL~BC0jaovfyKZVUSqKGCSIyPXjHZtsKwmm0HqmGgH4pHFD6Bhtyvzgss5wbxZ-YRc8KcyH67lZpxwy9L3y~Qs4aj9bvNsMiK2vdxluDVeemUJlu26erK9XqBDd3oV-jfP2k4mrSzIXO~ZlKAuDfigZsW9eZXqN~ujmftb6yqr2g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="AnABlast peaks uncovering evolutionary carboxyl- and amino-terminal subtractions in present genes. (A) AnABlast peak (arrow) at Chr I: 127.165-128.049 (reverse strand) located in the 3′UTR of SPAC11D3.11c reveals a carboxyl-terminal subtraction of this gene. (B) Peak at Chr I: 5542072-5542417 (forward strand), partially overlapping the 5′ UTR of SPAC186.06, uncovers an evolutionary amino-terminal subtraction of the corresponding gene. Ribo-Seq data, Pombase annotations, and colour codes for reading frames of the analysed genomic intervals are shown. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02506.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199874/dsv02506.tiff" path-from-xml="dsv02506.tiff" target="_blank">View large</a><a section="87199874" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv02506.png?Expires=2147483647&Signature=go4kyjmjsrsfEn8tkMohetgP9g4jRrmNkpw~BK20wl38eOBhsqJd44Q-3pgCBOJoeWfiyBYwFuTFY~QnpbdpmpoMtE1QTDgAFJ0kFNKOpfPjuPLmZ~i5VomYRUdamKBmPWKmSKFvbkFMgwyvniLETC6VkixbQR0IID3NaGxGjzDgs3BsNO7QYrhILiVsbCYkCrPEllqlPYkq5wR80H7Td2fLNtu8-aLSQatzxInjbVT572vRTgsCqcLiTgKC0s582H2JV8~A0ua-AcEYKACpt5HR~ry8xA1weQLkRE68LiRBOsmlIbQ7jE8IbOwUIRntmCJ27q1YMU8cdsc6f7uWog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199874&ar=2462479&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02506.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>AnABlast peaks uncovering evolutionary carboxyl- and amino-terminal subtractions in present genes. (A) AnABlast peak (arrow) at Chr I: 127.165-128.049 (reverse strand) located in the 3′UTR of SPAC11D3.11c reveals a carboxyl-terminal subtraction of this gene. (B) Peak at Chr I: 5542072-5542417 (forward strand), partially overlapping the 5′ UTR of SPAC186.06, uncovers an evolutionary amino-terminal subtraction of the corresponding gene. Ribo-Seq data, Pombase annotations, and colour codes for reading frames of the analysed genomic intervals are shown. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>Evolutionary changes in coding sequence length seem to be dynamic processes via shifts in the position of the start<sup><a reveal-id="DSV025C29" data-open="DSV025C29" class="link link-ref link-reveal xref-bibr">29</a></sup> and the stop codons.<sup><a reveal-id="DSV025C26" data-open="DSV025C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Our findings indicate that AnABlast may be useful to identify subtractions occurred during evolution. Remarkably, AnABlast can identify long terminal changes which, according to previous studies,<sup><a reveal-id="DSV025C26" data-open="DSV025C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSV025C29" data-open="DSV025C29" class="link link-ref link-reveal xref-bibr">29</a></sup> represent rare events among terminal subtractions and expansions. Since evolutionary expansions are part of present genes, AnABlast analysis of intergenic regions carried out in this study cannot uncover these evolutionary events.</p>                            <h3 scrollto-destination=87199876 id="87199876" class="section-title" data-legacy-id=s3f>3.6. Identification of gene fragments</h3>
<p>The vast majority of the selected AnABlast profiles identified single peak regions without putative ORFs (63%, excluding pseudogenes and transposons). Since AnABlast identifies coding sequences independently of ORFs, most of these peaks could represent gene fragments arisen from mutations and/or genome rearrangements. Accordingly, careful BLAST and Pfam analysis of predicted amino acid sequences in some of these peaks revealed fragments of tf1-retrotransposon (Fig. <a reveal-id="DSV025F7" data-open="DSV025F7" class="link link-reveal link-table xref-fig">7</a>A), of a MIP water channel protein (Fig. <a reveal-id="DSV025F7" data-open="DSV025F7" class="link link-reveal link-table xref-fig">7</a>B), and of an amino acid permease (Fig. <a reveal-id="DSV025F7" data-open="DSV025F7" class="link link-reveal link-table xref-fig">7</a>C). Interestingly, this later peak includes a possible fossil fragment of a bacterial transposase, fused to the amino acid permease sequence in the same reading frame (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv025_Supplementary_Data.zip?Expires=2147483647&Signature=1~jmJkVjh9nWQGIj2THUr5D9ZyIt-dff-I6YJG0D2WHfVKRiS1ULEF8wlV0~qsgibk4428hR8nTVJzlTk6AqCuZmrkL7RL9z~ZZxHzalAz43uaxTVm2M~i7SNOaxn3bja6jg-TLg9eJyoDg0ClySTnk6Yamj8pZn59POioyLbyBdL9MhSXahN9QyofilJXcuTB20yMCM2qcZ49WPg~C8vy1KkIaRIVtQe6SyNF8JAZ3lOlLDSUCSKXCXHxj1HVV8H7hEemFxddd9t0xQdjDWROOcsjpxn19QYay0xjvA5kdiz5eAUbmvOg95Yu2rzrHsH5dc7JXZyL9yqwya2DhrKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1D</a></span>). This finding casts light on how random combinatorial protein domains, including exogenous DNA, may contribute to the emergence of new traits during evolution. </p>                        <a id="87199878" scrollto-destination="87199878"></a>
<div id="DSV025F7" data-id="DSV025F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/m_dsv02507.png?Expires=2147483647&Signature=OYaZCG-RmzN1ETpHNwA8SATT628S2PHpD0H9xGBsrhcdq7kePM7~FMsLTfdhYyPT8tFLG-9AVvk~Q6ImNhJfa9xcOkfWKwOxgliQiEv49xuHDQYsSTHVmsJroGZJhr6wD~E5mSK7BIDC9-2HUpYs8RdJLBpCnZwcd7bj7NngI~HY400P3gdmpg1LcU1eFxQiAcUpsZbsEpC3WXE9m4HmMxqbQUFrkLeAfmnHULOAL~zEtqBp~c1BtlgMe1uspld-CvMzBEZT8FWidyE0387MdrMT4sdYTqSXqm7CRloxfXmlRHIDQfg9nCmbpWhT6wbk2NuQlmgMqq4c~32KhZPa9w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene fragments identified by AnABlast profiles. (A) Peak in the Chr I: 5543318-5543498 interval (forward strand) uncovers a fragment of Tf1 protein (S. pombe ref gb|AAA35339.1|, e-value: 2e−15). (B) Peak in the Chr I: 5536277-5538575 interval (reverse strand) encodes a polypeptide sequence belonging to a MIP water channel (S. pombe ref|NP_592788.1|, e-value: 5e−12). (C) Main AnABlast peak (arrow) at Chr II: 4452616-4452379 encodes a chimeric fragment of amino acid permeases (Chr II: 4452616-4452443, S. pombe ref|NP_596849.1|. e-value: 2e−20) fused in frame to another one sharing significant identity to bacterial transposases (Chr II: 4452444-4452379, Desulfobacter postgatei ref|WP_004074224.1|, 73% identity). (D) Peaks (arrows) at the Chr I: 5153233-5153041 and Chr I: E 5153042-5152866 intervals (reverse strand) encode fragments of cell surface glycoproteins (S. pombe ref|NP_588570.2|, e-values of concatenated sequence: 3e−72). (E and F) Genomic subtelomeric regions (Chr I: 5569804-5575195 and Chr II: 4514601-4519772, respectively) showing similar AnABlast profiles highlighting fragments of RecQ type DNA helicase coding sequences (Schizosaccharomyces pombe ref|NP_595040.1|, e-value: 0,0) and DUF999 family proteins (SPAC212.04c, e-value: 0,0). Pombase annotations and colour codes for reading frames of the analysed genomic intervals are shown. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02507.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199878/dsv02507.tiff" path-from-xml="dsv02507.tiff" target="_blank">View large</a><a section="87199878" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv02507.png?Expires=2147483647&Signature=MqQQejmou2ugygCuMPJP~P~UfH9w04TbLkft9-gDct1M6dprQgzKHUBMVWzgFvNg~Z6BTPzo0dc43BZ7i1bUxyLMVGJ4yF56f16heYywDHXOnOYaMgBAFcqElFRcv9XrjF6fevGm2V8WRfI3e4kN4Fdc6Cs9gMe8ZLjifuuAYlkr7vn5UAQf5tU6KnbGORcuZEZIbLXqAPDOB6Jq5ISj8Ii7Rns5d~F3TiBiZzb3mgLvz8s-H8Nw3BH-TcXPoe03SG8K-luBSz9Pk8Y5q4SCbLahAMkDDB5MLeqzS05PlIWfCmLiL-UCPB8yShILkW7zJNU6pi6brCvPmcdF-HVqLA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199878&ar=2462479&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02507.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene fragments identified by AnABlast profiles. (A) Peak in the Chr I: 5543318-5543498 interval (forward strand) uncovers a fragment of Tf1 protein (<em>S. pombe</em> ref gb|AAA35339.1|, e-value: 2e<sup>−15</sup>). (B) Peak in the Chr I: 5536277-5538575 interval (reverse strand) encodes a polypeptide sequence belonging to a MIP water channel (<em>S. pombe</em> ref|NP_592788.1|, e-value: 5e<sup>−12</sup>). (C) Main AnABlast peak (arrow) at Chr II: 4452616-4452379 encodes a chimeric fragment of amino acid permeases (Chr II: 4452616-4452443, <em>S. pombe</em> ref|NP_596849.1|. e-value: 2e<sup>−20</sup>) fused in frame to another one sharing significant identity to bacterial transposases (Chr II: 4452444-4452379, <em>Desulfobacter postgatei</em> ref|WP_004074224.1|, 73% identity). (D) Peaks (arrows) at the Chr I: 5153233-5153041 and Chr I: E 5153042-5152866 intervals (reverse strand) encode fragments of cell surface glycoproteins (<em>S. pombe</em> ref|NP_588570.2|, e-values of concatenated sequence: 3e<sup>−72</sup>). (E and F) Genomic subtelomeric regions (Chr I: 5569804-5575195 and Chr II: 4514601-4519772, respectively) showing similar AnABlast profiles highlighting fragments of RecQ type DNA helicase coding sequences (<em>Schizosaccharomyces pombe</em> ref|NP_595040.1|, e-value: 0,0) and DUF999 family proteins (SPAC212.04c, e-value: 0,0). Pombase annotations and colour codes for reading frames of the analysed genomic intervals are shown. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV025F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/m_dsv02507.png?Expires=2147483647&Signature=OYaZCG-RmzN1ETpHNwA8SATT628S2PHpD0H9xGBsrhcdq7kePM7~FMsLTfdhYyPT8tFLG-9AVvk~Q6ImNhJfa9xcOkfWKwOxgliQiEv49xuHDQYsSTHVmsJroGZJhr6wD~E5mSK7BIDC9-2HUpYs8RdJLBpCnZwcd7bj7NngI~HY400P3gdmpg1LcU1eFxQiAcUpsZbsEpC3WXE9m4HmMxqbQUFrkLeAfmnHULOAL~zEtqBp~c1BtlgMe1uspld-CvMzBEZT8FWidyE0387MdrMT4sdYTqSXqm7CRloxfXmlRHIDQfg9nCmbpWhT6wbk2NuQlmgMqq4c~32KhZPa9w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene fragments identified by AnABlast profiles. (A) Peak in the Chr I: 5543318-5543498 interval (forward strand) uncovers a fragment of Tf1 protein (S. pombe ref gb|AAA35339.1|, e-value: 2e−15). (B) Peak in the Chr I: 5536277-5538575 interval (reverse strand) encodes a polypeptide sequence belonging to a MIP water channel (S. pombe ref|NP_592788.1|, e-value: 5e−12). (C) Main AnABlast peak (arrow) at Chr II: 4452616-4452379 encodes a chimeric fragment of amino acid permeases (Chr II: 4452616-4452443, S. pombe ref|NP_596849.1|. e-value: 2e−20) fused in frame to another one sharing significant identity to bacterial transposases (Chr II: 4452444-4452379, Desulfobacter postgatei ref|WP_004074224.1|, 73% identity). (D) Peaks (arrows) at the Chr I: 5153233-5153041 and Chr I: E 5153042-5152866 intervals (reverse strand) encode fragments of cell surface glycoproteins (S. pombe ref|NP_588570.2|, e-values of concatenated sequence: 3e−72). (E and F) Genomic subtelomeric regions (Chr I: 5569804-5575195 and Chr II: 4514601-4519772, respectively) showing similar AnABlast profiles highlighting fragments of RecQ type DNA helicase coding sequences (Schizosaccharomyces pombe ref|NP_595040.1|, e-value: 0,0) and DUF999 family proteins (SPAC212.04c, e-value: 0,0). Pombase annotations and colour codes for reading frames of the analysed genomic intervals are shown. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02507.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199878/dsv02507.tiff" path-from-xml="dsv02507.tiff" target="_blank">View large</a><a section="87199878" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv02507.png?Expires=2147483647&Signature=MqQQejmou2ugygCuMPJP~P~UfH9w04TbLkft9-gDct1M6dprQgzKHUBMVWzgFvNg~Z6BTPzo0dc43BZ7i1bUxyLMVGJ4yF56f16heYywDHXOnOYaMgBAFcqElFRcv9XrjF6fevGm2V8WRfI3e4kN4Fdc6Cs9gMe8ZLjifuuAYlkr7vn5UAQf5tU6KnbGORcuZEZIbLXqAPDOB6Jq5ISj8Ii7Rns5d~F3TiBiZzb3mgLvz8s-H8Nw3BH-TcXPoe03SG8K-luBSz9Pk8Y5q4SCbLahAMkDDB5MLeqzS05PlIWfCmLiL-UCPB8yShILkW7zJNU6pi6brCvPmcdF-HVqLA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199878&ar=2462479&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02507.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene fragments identified by AnABlast profiles. (A) Peak in the Chr I: 5543318-5543498 interval (forward strand) uncovers a fragment of Tf1 protein (<em>S. pombe</em> ref gb|AAA35339.1|, e-value: 2e<sup>−15</sup>). (B) Peak in the Chr I: 5536277-5538575 interval (reverse strand) encodes a polypeptide sequence belonging to a MIP water channel (<em>S. pombe</em> ref|NP_592788.1|, e-value: 5e<sup>−12</sup>). (C) Main AnABlast peak (arrow) at Chr II: 4452616-4452379 encodes a chimeric fragment of amino acid permeases (Chr II: 4452616-4452443, <em>S. pombe</em> ref|NP_596849.1|. e-value: 2e<sup>−20</sup>) fused in frame to another one sharing significant identity to bacterial transposases (Chr II: 4452444-4452379, <em>Desulfobacter postgatei</em> ref|WP_004074224.1|, 73% identity). (D) Peaks (arrows) at the Chr I: 5153233-5153041 and Chr I: E 5153042-5152866 intervals (reverse strand) encode fragments of cell surface glycoproteins (<em>S. pombe</em> ref|NP_588570.2|, e-values of concatenated sequence: 3e<sup>−72</sup>). (E and F) Genomic subtelomeric regions (Chr I: 5569804-5575195 and Chr II: 4514601-4519772, respectively) showing similar AnABlast profiles highlighting fragments of RecQ type DNA helicase coding sequences (<em>Schizosaccharomyces pombe</em> ref|NP_595040.1|, e-value: 0,0) and DUF999 family proteins (SPAC212.04c, e-value: 0,0). Pombase annotations and colour codes for reading frames of the analysed genomic intervals are shown. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>Our search strategy also identified rearranged fragments of the same gene. Two adjacent genomic intervals coding for partial sequences of a predicted surface glycoprotein (in different reading frames) (Fig. <a reveal-id="DSV025F7" data-open="DSV025F7" class="link link-reveal link-table xref-fig">7</a>D), or up to three different subtelomeric regions encoding adjacent fragments of the RecA protein (encoded by the <em>tlh1</em>/<em>tlh2</em> genes) provide good examples of rearranged genes. As shown by AnABlast profiles, at least two regions containing <em>tlh1</em>/<em>tlh2</em> DNA fragments located in subtelomeric regions of Chromosomes I and II, respectively, harbour identical rearranged structures (see Fig. <a reveal-id="DSV025F7" data-open="DSV025F7" class="link link-reveal link-table xref-fig">7</a>E and F). A fragment of the SPAC212.04c gene (DUF999 family protein 1) can also be observed in both subtelomeric intervals (see in Fig. <a reveal-id="DSV025F7" data-open="DSV025F7" class="link link-reveal link-table xref-fig">7</a>), albeit only in the case of Chromosome II this gene fragment is annotated as pseudogene (SPBCPT2R1.05c). According to AnABlast peaks, this pseudogene should be also annotated as new pseudogene in the equivalent Chromosome I subtelomeric region.</p><p>Interestingly, the identical genomic gene fragments pattern revealed by AnABlast profiles in both subtelomeric intervals suggests that evolutionary events yielding such fragments likely occurred earlier than their translocations to the subtelomeric region. However, we cannot rule out that occasionally, these AnaBlast results could point to possible genome assembly errors rather than rearranged fragments and/or missing annotations.</p>                            <h3 scrollto-destination=87199881 id="87199881" class="section-title" data-legacy-id=s3g>3.7. Fossil-coding sequences</h3>
<p>As described above, BLAST analysis of a particular AnABlast peak identified a possible fossil transposase sequence, perhaps of bacterial origin (see in Fig. <a reveal-id="DSV025F7" data-open="DSV025F7" class="link link-reveal link-table xref-fig">7</a>C). This observation led us to investigate whether other AnABlast peaks could identify fossil sequences<sup><a reveal-id="DSV025C30" data-open="DSV025C30" class="link link-ref link-reveal xref-bibr">30</a></sup> transferred from other genomes.</p><p>Importantly, AnABlast identified a coding sequence containing a protein domain of the mitochondrial maturase protein-2 (SPMIT.03 mitochondrial gene), indicating that this genome fragment has a mitochondrial origin (Fig. <a reveal-id="DSV025F8" data-open="DSV025F8" class="link link-reveal link-table xref-fig">8</a>A). Furthermore, additional transposase and trehalose synthase fragments (Fig. <a reveal-id="DSV025F8" data-open="DSV025F8" class="link link-reveal link-table xref-fig">8</a>B and C, respectively) point to a possible horizontal transfer from bacteria as an evolutionary source for these intergenic sequences.<sup><a reveal-id="DSV025C31" data-open="DSV025C31" class="link link-ref link-reveal xref-bibr">31</a></sup> A short region encoding amino acid sequences highly similar to cytochrome c oxidase subunit III was also identified (Fig. <a reveal-id="DSV025F8" data-open="DSV025F8" class="link link-reveal link-table xref-fig">8</a>D). </p>                        <a id="87199884" scrollto-destination="87199884"></a>
<div id="DSV025F8" data-id="DSV025F8" class="fig fig-section"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/m_dsv02508.png?Expires=2147483647&Signature=RLU9fIM1MFgm4wHF7QdqEGm2yWhHtIKYlExVvMbiaaxmD1cS9THrDmIk9NTP4p2evJYK~dTa~mQwW6zL~cVUbAfhmENQGhhy1Z8xA89zuVcICxhzsfmQgMbEGkNL3Hhc8P3MavvLxcoDGMjn8zdcTY6WJuHHRQlaztkgwdOoVfHOGNYY3fUrJOnKXgaJxweRh3rae7tssnz-a9YEpVlpTDLJdxsNoEpvhoa5M5vODJwQFEe4mw9mNe31qoZtVz4~ROqJbturZCWLQ8wQHlnw6E8qNgstnObe1U5I2Mn6QV4BOawaoS2a99WVLIxj-euuLXN51G46R8UbH~U3VdGDYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Fossil-coding sequences predicted by AnABlast search. (A) AnABlast peak (Chr II: 36890-37298, forward strand) encodes a fragment of mitochondrial maturase protein 2 (fission yeast mitochondrion. Sequence ID: pir|S78197|. E-value: 5e−56). (B) Peak at Chr II: 4091456-4091554 (forward strand) codes for a polypeptide 58% identical to a domain of bacterial transposases (Bacillus mycoides ref: gb|KFN12866.1|). (C) Peak at 2958676-2958843 (reverse strand) encodes a partial sequence of bacterial trehalose synthase (Streptomyces chartreusis, ref|WP_010033287.1| e-value: 1e−17). (D) Peak at Chr II: 22433875-22433935 (forward strand) identifies a small peptide domain common to numerous cytochrome c oxidases (e-value for Hylarana sp: 1e−07). Pombase annotations and colour codes for reading frames of the analysed genomic intervals are shown. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02508.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199884/dsv02508.tiff" path-from-xml="dsv02508.tiff" target="_blank">View large</a><a section="87199884" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv02508.png?Expires=2147483647&Signature=uXI0t4vz5DWeMOLxhEcUbk5lytDrC52irLZXTk4NrCoGgU~mNBSYCA5uQxj-jOb1C2NyrqMI6SEs2V3UL3glgvuQ2B3ieY6aQWNjekOj3pd5C~7grbMDOFJxVLow~zX1vl6OcN9PAoQSa6TxQAP2MhCmplhrDu7DFLbjvcG~81yUguThByNXsIZtizEvNstXHMjhzsWCUZO8O2Ik1HFoprQwjuJE5-MGUaK5IpzMpyNLpwfJvFkTZGIwZ~YvLc4H1i57cfoVEyiDt2wwnKrOkEI3ZJ2ZPMy69xYYw2qjuqe~jsxz5m-nhAgYm~LeuEHnbi7R2Zy5oMy-ESGTQHWUbw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199884&ar=2462479&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02508.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Fossil-coding sequences predicted by AnABlast search. (A) AnABlast peak (Chr II: 36890-37298, forward strand) encodes a fragment of mitochondrial maturase protein 2 (fission yeast mitochondrion. Sequence ID: pir|S78197|. <em>E</em>-value: 5e<sup>−56</sup>). (B) Peak at Chr II: 4091456-4091554 (forward strand) codes for a polypeptide 58% identical to a domain of bacterial transposases (<em>Bacillus mycoides</em> ref: gb|KFN12866.1|). (C) Peak at 2958676-2958843 (reverse strand) encodes a partial sequence of bacterial trehalose synthase (<em>Streptomyces chartreusis</em>, ref|WP_010033287.1| <em>e</em>-value: 1e<sup>−17</sup>). (D) Peak at Chr II: 22433875-22433935 (forward strand) identifies a small peptide domain common to numerous cytochrome c oxidases (<em>e</em>-value for <em>Hylarana sp</em>: 1e<sup>−07</sup>). Pombase annotations and colour codes for reading frames of the analysed genomic intervals are shown. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV025F8" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/m_dsv02508.png?Expires=2147483647&Signature=RLU9fIM1MFgm4wHF7QdqEGm2yWhHtIKYlExVvMbiaaxmD1cS9THrDmIk9NTP4p2evJYK~dTa~mQwW6zL~cVUbAfhmENQGhhy1Z8xA89zuVcICxhzsfmQgMbEGkNL3Hhc8P3MavvLxcoDGMjn8zdcTY6WJuHHRQlaztkgwdOoVfHOGNYY3fUrJOnKXgaJxweRh3rae7tssnz-a9YEpVlpTDLJdxsNoEpvhoa5M5vODJwQFEe4mw9mNe31qoZtVz4~ROqJbturZCWLQ8wQHlnw6E8qNgstnObe1U5I2Mn6QV4BOawaoS2a99WVLIxj-euuLXN51G46R8UbH~U3VdGDYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Fossil-coding sequences predicted by AnABlast search. (A) AnABlast peak (Chr II: 36890-37298, forward strand) encodes a fragment of mitochondrial maturase protein 2 (fission yeast mitochondrion. Sequence ID: pir|S78197|. E-value: 5e−56). (B) Peak at Chr II: 4091456-4091554 (forward strand) codes for a polypeptide 58% identical to a domain of bacterial transposases (Bacillus mycoides ref: gb|KFN12866.1|). (C) Peak at 2958676-2958843 (reverse strand) encodes a partial sequence of bacterial trehalose synthase (Streptomyces chartreusis, ref|WP_010033287.1| e-value: 1e−17). (D) Peak at Chr II: 22433875-22433935 (forward strand) identifies a small peptide domain common to numerous cytochrome c oxidases (e-value for Hylarana sp: 1e−07). Pombase annotations and colour codes for reading frames of the analysed genomic intervals are shown. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02508.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199884/dsv02508.tiff" path-from-xml="dsv02508.tiff" target="_blank">View large</a><a section="87199884" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv02508.png?Expires=2147483647&Signature=uXI0t4vz5DWeMOLxhEcUbk5lytDrC52irLZXTk4NrCoGgU~mNBSYCA5uQxj-jOb1C2NyrqMI6SEs2V3UL3glgvuQ2B3ieY6aQWNjekOj3pd5C~7grbMDOFJxVLow~zX1vl6OcN9PAoQSa6TxQAP2MhCmplhrDu7DFLbjvcG~81yUguThByNXsIZtizEvNstXHMjhzsWCUZO8O2Ik1HFoprQwjuJE5-MGUaK5IpzMpyNLpwfJvFkTZGIwZ~YvLc4H1i57cfoVEyiDt2wwnKrOkEI3ZJ2ZPMy69xYYw2qjuqe~jsxz5m-nhAgYm~LeuEHnbi7R2Zy5oMy-ESGTQHWUbw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199884&ar=2462479&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02508.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Fossil-coding sequences predicted by AnABlast search. (A) AnABlast peak (Chr II: 36890-37298, forward strand) encodes a fragment of mitochondrial maturase protein 2 (fission yeast mitochondrion. Sequence ID: pir|S78197|. <em>E</em>-value: 5e<sup>−56</sup>). (B) Peak at Chr II: 4091456-4091554 (forward strand) codes for a polypeptide 58% identical to a domain of bacterial transposases (<em>Bacillus mycoides</em> ref: gb|KFN12866.1|). (C) Peak at 2958676-2958843 (reverse strand) encodes a partial sequence of bacterial trehalose synthase (<em>Streptomyces chartreusis</em>, ref|WP_010033287.1| <em>e</em>-value: 1e<sup>−17</sup>). (D) Peak at Chr II: 22433875-22433935 (forward strand) identifies a small peptide domain common to numerous cytochrome c oxidases (<em>e</em>-value for <em>Hylarana sp</em>: 1e<sup>−07</sup>). Pombase annotations and colour codes for reading frames of the analysed genomic intervals are shown. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87199885 id="87199885" class="section-title" data-legacy-id=s3h>3.8. Conclusion</h3>
<p>Here we report that low-stringency alignments accumulation in predicted amino acid sequences may discern coding from non-coding DNA, thereby providing a new <em>in silico</em> strategy (AnABlast) to reveal present and ancient coding regions in sequenced DNA. We have shown that the identification of rearranged and fossil sequences by AnABlast may be helpful to better understand the evolutionary history of the <em>S. pombe</em> genome. This property, together with the ability to uncover novel genes and pseudogenes, suggests that AnABlast provides a new strategy of general interest for the genome-wide search of present and fossil sequences in any genome.</p>                            <h2 scrollto-destination=87199887 id="87199887" class="section-title" data-legacy-id=s4>Authors' contribution</h2>
<p>J.J. and A.P.P. conceived the study and designed the experiments. A.P.P. developed the computer programme. A.P.P. and M.G. generated AnABlast profiles. C.D.S.D. and J.M. performed Ribo-Seq experiments. J.J. did the analyses of AnABlast peaks and wrote the manuscript. J.J., C.D.S.D., M.G., J.M., and A.P.P. critically read and approved the manuscript.</p>                            <h2 scrollto-destination=87199889 id="87199889" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This research was supported by the Ministry of Economy and Competitiveness of the Spanish Government grant BFU2013-46923-P (J.J.) and by Biotechnology and Biological Sciences Research Council (UK) research grant BB/J007153/1 (J.M.). Authors declare no competing interest. Funding to pay the Open Access publication charges for this article was provided by the Ministry of Economy and Competitiveness of the Spanish Government.</p>                            <h2 scrollto-destination=87199891 id="87199891" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Andrés Garzón, Manuel Muñoz. and Rafael R. Daga for comments on the manuscript.</p>                            <h2 scrollto-destination=87199893 id="87199893" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV025C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nesvizhskii</div>
               <div class="given-names">A.I.</div>
            </div>
          </div>
          <div class="year">2014</div>, <div class="article-title">Proteogenomics: concepts, applications and computational strategies</div>, <div class="source">Nat. Methods</div> , <div class="volume">11</div>, <div class="fpage">1114</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Proteogenomics%3A%20concepts%2C%20applications%20and%20computational%20strategies&amp;author=A.I.%20Nesvizhskii&amp;publication_year=2014&amp;journal=Nat.%20Methods&amp;volume=11&amp;pages=1114-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.3144" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.3144" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.3144"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25357241" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">M.Q.</div>
            </div>
          </div>
          <div class="year">2002</div>, <div class="article-title">Computational prediction of eukaryotic protein-coding genes</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">3</div>, <div class="fpage">698</div>–<div class="lpage">709</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20prediction%20of%20eukaryotic%20protein-coding%20genes&amp;author=M.Q.%20Zhang&amp;publication_year=2002&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=3&amp;pages=698-709" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg890" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg890" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg890"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12209144" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Altschul</div>
               <div class="given-names">S.F.</div>
            </div>, <div class="name string-name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>, <div class="name string-name"><div class="surname">Schäffer</div> <div class="given-names">A.A.</div></div>et al.  </div>. <div class="year">1997</div>, <div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">25</div>, <div class="fpage">3389</div>–<div class="lpage">402</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Sch%C3%A4ffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dargahi</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Baillie</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Pio</div> <div class="given-names">F.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Bioinformatics analysis identify novel OB fold protein coding genes in <em>C. elegans</em></div><em>,</em><div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e62204</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bioinformatics%20analysis%20identify%20novel%20OB%20fold%20protein%20coding%20genes%20in%20C.%20elegans&amp;author=D.%20Dargahi&amp;author=D.%20Baillie&amp;author=F.%20Pio&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e62204" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0062204" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0062204" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0062204"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23638006" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Finn</div>
               <div class="given-names">R.D.</div>
            </div>, <div class="name string-name"><div class="surname">Mistry</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Tate</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">The Pfam protein families database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">38</div>, <div class="fpage">D211</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Pfam%20protein%20families%20database&amp;author=R.D.%20Finn&amp;author=J.%20Mistry&amp;author=J.%20Tate&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=D211-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp985" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp985" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp985"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19920124" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">W.</div>
            </div>, <div class="name string-name"><div class="surname">Pio</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Pawlowski</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Godzik</div> <div class="given-names">A.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Saturated BLAST: an automated multiple intermediate sequence search used to detect distant homology</div>, <div class="source">Bioinformatics</div> , <div class="volume">16</div>, <div class="fpage">1105</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Saturated%20BLAST%3A%20an%20automated%20multiple%20intermediate%20sequence%20search%20used%20to%20detect%20distant%20homology&amp;author=W.%20Li&amp;author=F.%20Pio&amp;author=K.%20Pawlowski&amp;author=A.%20Godzik&amp;publication_year=2000&amp;journal=Bioinformatics&amp;volume=16&amp;pages=1105-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/16.12.1105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F16.12.1105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F16.12.1105"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11159329" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Soding</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Remmert</div> <div class="given-names">M.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Protein sequence comparison and fold recognition: progress and good-practice benchmarking</div>, <div class="source">Curr. Opin. Struct. Biol.</div> , <div class="volume">21</div>, <div class="fpage">404</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20sequence%20comparison%20and%20fold%20recognition%3A%20progress%20and%20good-practice%20benchmarking&amp;author=J.%20Soding&amp;author=M.%20Remmert&amp;publication_year=2011&amp;journal=Curr.%20Opin.%20Struct.%20Biol.&amp;volume=21&amp;pages=404-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.sbi.2011.03.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.sbi.2011.03.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.sbi.2011.03.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21458982" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tong</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Sadreyev</div> <div class="given-names">R.I.</div></div>, <div class="name string-name"><div class="surname">Pei</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Kinch</div> <div class="given-names">L.N.</div></div>, <div class="name string-name"><div class="surname">Grishin</div> <div class="given-names">N.V.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">Using homology relations within a database markedly boosts protein sequence similarity search</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">112</div>, <div class="fpage">7003</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20homology%20relations%20within%20a%20database%20markedly%20boosts%20protein%20sequence%20similarity%20search&amp;author=J.%20Tong&amp;author=R.I.%20Sadreyev&amp;author=J.%20Pei&amp;author=L.N.%20Kinch&amp;author=N.V.%20Grishin&amp;publication_year=2015&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=112&amp;pages=7003-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1424324112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1424324112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1424324112"> </span></div> </div></div></div></div></div><div content-id="DSV025C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">X.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">Application of Learning to Rank to protein remote homology detection</div>, <div class="source">Bioinformatics</div> , <div class="comment">pii: btv413 [Epub ahead of print]</div>.</div></div></div></div><div content-id="DSV025C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shalem</div>
               <div class="given-names">O.</div>
            </div>, <div class="name string-name"><div class="surname">Sanjana</div> <div class="given-names">N.E.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">F.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">High-throughput functional genomics using CRISPR-Cas9</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">16</div>, <div class="fpage">299</div>–<div class="lpage">311</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20functional%20genomics%20using%20CRISPR-Cas9&amp;author=O.%20Shalem&amp;author=N.E.%20Sanjana&amp;author=F.%20Zhang&amp;publication_year=2015&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=16&amp;pages=299-311" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3899" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3899" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3899"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25854182" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Egel</div>
               <div class="given-names">R.</div>
            </div>
          </div>, ed. <div class="year">2004</div>, <div class="source">The molecular biology of <em>Schizosaccharomyces Pombe</em></div> . <div class="publisher-name">Springer</div>: <div class="publisher-loc">Berlin, Germany</div>.</div></div></div></div><div content-id="DSV025C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wood</div>
               <div class="given-names">V.</div>
            </div>, <div class="name string-name"><div class="surname">Gwilliam</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Rajandream</div> <div class="given-names">M.A.</div></div>et al.  </div>. <div class="year">2002</div>, <div class="article-title">The genome sequence of <em>Schizosaccharomyces pombe</em></div>, <div class="source">Nature</div> , <div class="volume">415</div>, <div class="fpage">871</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20Schizosaccharomyces%20pombe&amp;author=V.%20Wood&amp;author=R.%20Gwilliam&amp;author=M.A.%20Rajandream&amp;publication_year=2002&amp;journal=Nature&amp;volume=415&amp;pages=871-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature724" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature724" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature724"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11859360" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Quintales</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Sanchez</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Antequera</div> <div class="given-names">F.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Analysis of DNA strand-specific differential expression with high density tiling microarrays</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">11</div>, <div class="fpage">136</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20DNA%20strand-specific%20differential%20expression%20with%20high%20density%20tiling%20microarrays&amp;author=L.%20Quintales&amp;author=M.%20Sanchez&amp;author=F.%20Antequera&amp;publication_year=2010&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=136" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-11-136" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-11-136" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-11-136"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20233458" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Marguerat</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Schmidt</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Codlin</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Aebersold</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Bähler</div> <div class="given-names">J.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Quantitative analysis of fission yeast transcriptomes and proteomes in proliferating and quiescent cells</div>, <div class="source">Cell</div> , <div class="volume">151</div>, <div class="fpage">671</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20analysis%20of%20fission%20yeast%20transcriptomes%20and%20proteomes%20in%20proliferating%20and%20quiescent%20cells&amp;author=S.%20Marguerat&amp;author=A.%20Schmidt&amp;author=S.%20Codlin&amp;author=W.%20Chen&amp;author=R.%20Aebersold&amp;author=J.%20B%C3%A4hler&amp;publication_year=2012&amp;journal=Cell&amp;volume=151&amp;pages=671-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2012.09.019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2012.09.019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2012.09.019"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23101633" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wilhelm</div>
               <div class="given-names">B.T.</div>
            </div>, <div class="name string-name"><div class="surname">Marguerat</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Watt</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2008</div>, <div class="article-title">Dynamic repertoire of a eukaryotic transcriptome surveyed at single-nucleotide resolution</div>, <div class="source">Nature</div> , <div class="volume">453</div>, <div class="fpage">1239</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamic%20repertoire%20of%20a%20eukaryotic%20transcriptome%20surveyed%20at%20single-nucleotide%20resolution&amp;author=B.T.%20Wilhelm&amp;author=S.%20Marguerat&amp;author=S.%20Watt&amp;publication_year=2008&amp;journal=Nature&amp;volume=453&amp;pages=1239-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18488015" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wood</div>
               <div class="given-names">V.</div>
            </div>, <div class="name string-name"><div class="surname">Harris</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">McDowall</div> <div class="given-names">M.D.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">PomBase: a comprehensive online resource for fission yeast</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">40</div>, <div class="fpage">D695</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PomBase%3A%20a%20comprehensive%20online%20resource%20for%20fission%20yeast&amp;author=V.%20Wood&amp;author=M.A.%20Harris&amp;author=M.D.%20McDowall&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=D695-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr853" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr853" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr853"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22039153" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bitton</div>
               <div class="given-names">D.A.</div>
            </div>, <div class="name string-name"><div class="surname">Wood</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Scutt</div> <div class="given-names">P.J.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Augmented annotation of the <em>Schizosaccharomyces pombe</em> genome reveals additional genes required for growth and viability</div>, <div class="source">Genetics</div> , <div class="volume">187</div>, <div class="fpage">1207</div>–<div class="lpage">17</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Augmented%20annotation%20of%20the%20Schizosaccharomyces%20pombe%20genome%20reveals%20additional%20genes%20required%20for%20growth%20and%20viability&amp;author=D.A.%20Bitton&amp;author=V.%20Wood&amp;author=P.J.%20Scutt&amp;publication_year=2011&amp;journal=Genetics&amp;volume=187&amp;pages=1207-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.110.123497" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.110.123497" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.110.123497"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21270388" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Duncan</div>
               <div class="given-names">C.D.</div>
            </div>, <div class="name string-name"><div class="surname">Mata</div> <div class="given-names">J.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">The translational landscape of fission-yeast meiosis and sporulation</div>, <div class="source">Nat. Struct. Mol. Biol.</div> , <div class="volume">21</div>, <div class="fpage">641</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20translational%20landscape%20of%20fission-yeast%20meiosis%20and%20sporulation&amp;author=C.D.%20Duncan&amp;author=J.%20Mata&amp;publication_year=2014&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;volume=21&amp;pages=641-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb.2843" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb.2843" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb.2843"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24929437" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rhind</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Yassour</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Comparative functional genomics of the fission yeasts</div>, <div class="source">Science</div> , <div class="volume">332</div>, <div class="fpage">930</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20functional%20genomics%20of%20the%20fission%20yeasts&amp;author=N.%20Rhind&amp;author=Z.%20Chen&amp;author=M.%20Yassour&amp;publication_year=2011&amp;journal=Science&amp;volume=332&amp;pages=930-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1203357" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1203357" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1203357"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21511999" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">McDowall</div>
               <div class="given-names">M.D.</div>
            </div>, <div class="name string-name"><div class="surname">Harris</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Lock</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">PomBase 2015: updates to the fission yeast database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">43</div>, <div class="fpage">D656</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PomBase%202015%3A%20updates%20to%20the%20fission%20yeast%20database&amp;author=M.D.%20McDowall&amp;author=M.A.%20Harris&amp;author=A.%20Lock&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res.&amp;volume=43&amp;pages=D656-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku1040" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku1040" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku1040"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25361970" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pérez</div>
               <div class="given-names">A.J.</div>
            </div>, <div class="name string-name"><div class="surname">Thode</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Trelles</div> <div class="given-names">O.</div></div></div>
          <div class="year">2004</div>, <div class="article-title">AnaGram:protein function assignment</div>, <div class="source">Bioinformatics</div> , <div class="volume">20</div>, <div class="fpage">291</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AnaGram%3Aprotein%20function%20assignment&amp;author=A.J.%20P%C3%A9rez&amp;author=G.%20Thode&amp;author=O.%20Trelles&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=291-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg414" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg414" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg414"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14734328" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Thode</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">García-Ranea</div> <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Jimenez</div> <div class="given-names">J.</div></div></div>
          <div class="year">1996</div>, <div class="article-title">Search for ancient patterns in protein sequences</div>, <div class="source">J. Mol. Evol.</div> , <div class="volume">42</div>, <div class="fpage">224</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Search%20for%20ancient%20patterns%20in%20protein%20sequences&amp;author=G.%20Thode&amp;author=J.A.%20Garc%C3%ADa-Ranea&amp;author=J.%20Jimenez&amp;publication_year=1996&amp;journal=J.%20Mol.%20Evol.&amp;volume=42&amp;pages=224-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02198848" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02198848" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02198848"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8919874" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Suzek</div>
               <div class="given-names">B.E.</div>
            </div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">McGarvey</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Mazumder</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">C.H.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">UniRef: comprehensive and non-redundant UniProt reference clusters</div>, <div class="source">Bioinformatics</div> , <div class="volume">23</div>, <div class="fpage">1282</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=UniRef%3A%20comprehensive%20and%20non-redundant%20UniProt%20reference%20clusters&amp;author=B.E.%20Suzek&amp;author=H.%20Huang&amp;author=P.%20McGarvey&amp;author=R.%20Mazumder&amp;author=C.H.%20Wu&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=1282-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm098" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm098" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm098"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17379688" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Thorvaldsdóttir</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Robinson</div> <div class="given-names">J.T.</div></div>, <div class="name string-name"><div class="surname">Mesirov</div> <div class="given-names">J.P.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration</div>, <div class="source">Brief Bioinform.</div> , <div class="volume">14</div>, <div class="fpage">178</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrative%20Genomics%20Viewer%20(IGV)%3A%20high-performance%20genomics%20data%20visualization%20and%20exploration&amp;author=H.%20Thorvaldsd%C3%B3ttir&amp;author=J.T.%20Robinson&amp;author=J.P.%20Mesirov&amp;publication_year=2013&amp;journal=Brief%20Bioinform.&amp;volume=14&amp;pages=178-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/bbs017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2Fbbs017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbib%2Fbbs017"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22517427" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sheetlin</div>
               <div class="given-names">S.L.</div>
            </div>, <div class="name string-name"><div class="surname">Park</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Frith</div> <div class="given-names">M.C.</div></div>, <div class="name string-name"><div class="surname">Spouge</div> <div class="given-names">J.L.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Frameshift alignment: statistics and post-genomic applications</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">3575</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Frameshift%20alignment%3A%20statistics%20and%20post-genomic%20applications&amp;author=S.L.%20Sheetlin&amp;author=Y.%20Park&amp;author=M.C.%20Frith&amp;author=J.L.%20Spouge&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=3575-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu576" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu576" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu576"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25172925" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Giacomelli</div>
               <div class="given-names">M.G.</div>
            </div>, <div class="name string-name"><div class="surname">Hancock</div> <div class="given-names">A.S.</div></div>, <div class="name string-name"><div class="surname">Masel</div> <div class="given-names">J.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">The conversion of 3’ UTRs into coding regions</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="volume">24</div>, <div class="fpage">457</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20conversion%20of%203%E2%80%99%20UTRs%20into%20coding%20regions&amp;author=M.G.%20Giacomelli&amp;author=A.S.%20Hancock&amp;author=J.%20Masel&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=24&amp;pages=457-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msl172" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsl172" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsl172"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17099057" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Modrek</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">C.J.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">Alternative splicing in the human, mouse and rat genomes is associated with an increased frequency of exon creation and/or loss</div>, <div class="source">Nat. Genet.</div> , <div class="volume">34</div>, <div class="fpage">177</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20in%20the%20human%2C%20mouse%20and%20rat%20genomes%20is%20associated%20with%20an%20increased%20frequency%20of%20exon%20creation%20and%2For%20loss&amp;author=B.%20Modrek&amp;author=C.J.%20Lee&amp;publication_year=2003&amp;journal=Nat.%20Genet.&amp;volume=34&amp;pages=177-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1159" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1159" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1159"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12730695" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rogozin</div>
               <div class="given-names">I.B.</div>
            </div>, <div class="name string-name"><div class="surname">Carmel</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Csuros</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Koonin</div> <div class="given-names">E.V.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Origin and evolution of spliceosomal introns</div>, <div class="source">Biol. Direct</div> , <div class="volume">7</div>, <div class="fpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origin%20and%20evolution%20of%20spliceosomal%20introns&amp;author=I.B.%20Rogozin&amp;author=L.%20Carmel&amp;author=M.%20Csuros&amp;author=E.V.%20Koonin&amp;publication_year=2012&amp;journal=Biol.%20Direct&amp;volume=7&amp;pages=11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1745-6150-7-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1745-6150-7-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1745-6150-7-11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22507701" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lynch</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Scofield</div> <div class="given-names">D.G.</div></div>, <div class="name string-name"><div class="surname">Hong</div> <div class="given-names">X.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">The evolution of transcription-initiation sites</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="volume">22</div>, <div class="fpage">1137</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20transcription-initiation%20sites&amp;author=M.%20Lynch&amp;author=D.G.%20Scofield&amp;author=X.%20Hong&amp;publication_year=2005&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=22&amp;pages=1137-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msi100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsi100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsi100"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15689525" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV025C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shemesh</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Novik</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Edelheit</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sorek</div> <div class="given-names">R.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">Genomic fossils as a snapshot of the human transcriptoma</div><em>,</em><div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">103</div>, <div class="fpage">1364</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20fossils%20as%20a%20snapshot%20of%20the%20human%20transcriptoma&amp;author=R.%20Shemesh&amp;author=A.%20Novik&amp;author=S.%20Edelheit&amp;author=R.%20Sorek&amp;publication_year=2006&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=1364-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0509330103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0509330103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0509330103"> </span></div> </div></div></div></div></div><div content-id="DSV025C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV025C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sprague</div>
               <div class="given-names">G.F.</div>
              Jr.
            </div>
          </div>
          <div class="year">1991</div>, <div class="article-title">Genetic exchange between kingdoms</div>, <div class="source">Curr. Opin. Genet. Dev.</div> , <div class="volume">1</div>, <div class="fpage">530</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20exchange%20between%20kingdoms&amp;author=G.F.%20Sprague&amp;publication_year=1991&amp;journal=Curr.%20Opin.%20Genet.%20Dev.&amp;volume=1&amp;pages=530-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0959-437X(05)80203-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0959-437X(05)80203-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0959-437X(05)80203-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1822285" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2462479&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv025/1/dsv025_Supplementary_Data.zip?Expires=2147483647&amp;Signature=1~jmJkVjh9nWQGIj2THUr5D9ZyIt-dff-I6YJG0D2WHfVKRiS1ULEF8wlV0~qsgibk4428hR8nTVJzlTk6AqCuZmrkL7RL9z~ZZxHzalAz43uaxTVm2M~i7SNOaxn3bja6jg-TLg9eJyoDg0ClySTnk6Yamj8pZn59POioyLbyBdL9MhSXahN9QyofilJXcuTB20yMCM2qcZ49WPg~C8vy1KkIaRIVtQe6SyNF8JAZ3lOlLDSUCSKXCXHxj1HVV8H7hEemFxddd9t0xQdjDWROOcsjpxn19QYay0xjvA5kdiz5eAUbmvOg95Yu2rzrHsH5dc7JXZyL9yqwya2DhrKw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">423</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000366631700005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">1</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv025"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2462479"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000366631700005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f6%2f439%2f2462479">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000366631700005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (1)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F6%2F439%2F2462479" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2462479'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="AnABlast: a new in silico strategy for the genome-wide search of novel genes and fossil regions" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv025");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f439%2f2462479">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

