

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":1,"applicationTime":1850,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/4/403/2469959","@type":"ScholarlyArticle","name":"Identifying the genome-wide genetic variation between precocious trifoliate orange and its wild type and developing new markers for genetics research","datePublished":"2016-04-21","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2016-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw017","keywords":["citrus","flowering","genome resequencing","genetic variation","SNP"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/4/403/2469959","author":[{"name":"Zhang, Jin-Zhi","affiliation":"Key Laboratory of Horticultural Plant Biology (Ministry of Education) ,  College of Horticulture and Forestry Science, Huazhong Agricultural University ,  Wuhan 430070 ,  China","@type":"Person"},{"name":"Liu, Sheng-Rui","affiliation":"Key Laboratory of Horticultural Plant Biology (Ministry of Education) ,  College of Horticulture and Forestry Science, Huazhong Agricultural University ,  Wuhan 430070 ,  China","@type":"Person"},{"name":"Hu, Chun-Gen","affiliation":"Key Laboratory of Horticultural Plant Biology (Ministry of Education) ,  College of Horticulture and Forestry Science, Huazhong Agricultural University ,  Wuhan 430070 ,  China","@type":"Person"}],"description":"Abstract.  To increase our understanding of the genes involved in flowering in citrus, we performed genome resequencing of an early flowering trifoliate orange ","pageStart":"403","pageEnd":"414","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/4/0/m_cover.gif?Expires=2147483647&Signature=XWSbFJdI5lGDaRny1g7BRzfZg~ebzc~OT20sUUxiNUi3-W~7uvtu5IDIgO3L5~QjmCQzUopLMGz-3gEHtIKUOFbRCHM2eH4h7l2Wu3nqAQC9YYHygJi-TLC9x3FpQRq3lTBcdWmpj185LJbB-1H-PKlUoEU9gxYyCNvjJUg~GUaHNRtVxbV0a7bDIfdyclZJ~w1BhGonEiYqh-IZSS0VekFWZ7mnLia~WbOEtL8AzSLsoIVak5-aADczuqIYGRkdfxcEh8zTdO-NlGhKxLoi5zlJy0Y~XuIxHZNxQxDFoVA3s~0J2fujGZsWX~ElpEU3I2Vma3M6MgFM80P2xPWpTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Identifying the genome-wide genetic variation between precocious trifoliate orange and its wild type and developing new markers for genetics research","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/4/0/m_cover.gif?Expires=2147483647&Signature=XWSbFJdI5lGDaRny1g7BRzfZg~ebzc~OT20sUUxiNUi3-W~7uvtu5IDIgO3L5~QjmCQzUopLMGz-3gEHtIKUOFbRCHM2eH4h7l2Wu3nqAQC9YYHygJi-TLC9x3FpQRq3lTBcdWmpj185LJbB-1H-PKlUoEU9gxYyCNvjJUg~GUaHNRtVxbV0a7bDIfdyclZJ~w1BhGonEiYqh-IZSS0VekFWZ7mnLia~WbOEtL8AzSLsoIVak5-aADczuqIYGRkdfxcEh8zTdO-NlGhKxLoi5zlJy0Y~XuIxHZNxQxDFoVA3s~0J2fujGZsWX~ElpEU3I2Vma3M6MgFM80P2xPWpTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Identifying the genome-wide genetic variation between precocious trifoliate orange and its wild type and developing new markers for genetics research" />
    <meta property="og:description" content="Abstract.  To increase our understanding of the genes involved in flowering in citrus, we performed genome resequencing of an early flowering trifoliate orange " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/4/403/2469959" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-04-21" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Zhang, Jin-Zhi" /><meta name="citation_author_institution" content="Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture and Forestry Science, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author" content="Liu, Sheng-Rui" /><meta name="citation_author_institution" content="Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture and Forestry Science, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author" content="Hu, Chun-Gen" /><meta name="citation_author_institution" content="Key Laboratory of Horticultural Plant Biology (Ministry of Education), College of Horticulture and Forestry Science, Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_title" content="Identifying the genome-wide genetic variation between precocious trifoliate orange and its wild type and developing new markers for genetics research" /><meta name="citation_firstpage" content="403" /><meta name="citation_lastpage" content="414" /><meta name="citation_doi" content="10.1093/dnares/dsw017" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2016/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Regulation of floral patterning by flowering time genes;  citation_journal_title=Dev. Cell;  citation_year=2009;  citation_volume=16;  citation_pages=711-22; " /><meta name="citation_reference" content="citation_title=Genetic regulation of flowering time in annual and perennial plants;  citation_journal_title=Wiley Interdiscip. Rev. RNA;  citation_year=2014;  citation_volume=5;  citation_pages=347-59; " /><meta name="citation_reference" content="citation_title=Control of flowering time interacting pathways as a basis for diversity;  citation_journal_title=Plant Cell;  citation_year=2002;  citation_volume=14;  citation_issue=(Suppl. 1);  citation_pages=S111-30; " /><meta name="citation_reference" content="citation_title=Multiple pathways in the decision to flower: enabling, promoting, and resetting;  citation_journal_title=Plant Cell;  citation_year=2004;  citation_volume=16;  citation_issue=(Suppl. 1);  citation_pages=S18-31; " /><meta name="citation_reference" content="citation_title=Conservation and diversity in flower land;  citation_journal_title=Curr. Opin. Plant Biol.;  citation_year=2004;  citation_volume=7;  citation_pages=84-91; " /><meta name="citation_reference" content="citation_title=Genetics of flower initiation and development in annual and perennial plants;  citation_journal_title=Physiol. Plant.;  citation_year=2006;  citation_volume=128;  citation_pages=8-17; " /><meta name="citation_reference" content="citation_title=Functional characterization of AP3, SOC1 and WUS homologues from citrus (Citrus sinensis);  citation_journal_title=Physiol. Plant.;  citation_year=2007;  citation_volume=131;  citation_pages=481-95; " /><meta name="citation_reference" content="citation_title=Citrus fruits; citation_publisher=Springer, Heidelberg;  citation_title=Fruits and Nuts, Genome Mapping &amp; Molecular Breeding in Plants;  citation_year=2007;  citation_pages=265-79; " /><meta name="citation_reference" content="citation_title=Sequencing of diverse mandarin, pummelo and orange genomes reveals complex history of admixture during citrus domestication;  citation_journal_title=Nat. Biotechnol.;  citation_year=2014;  citation_volume=32;  citation_pages=656-62; " /><meta name="citation_reference" content="citation_title=Precocious trifoliate orange (Poncirus trifoliata L. Raf.) biology characteristic and its stock experiment;  citation_journal_title=ZheJiang Citrus;  citation_year=1999;  citation_volume=16;  citation_pages=2-4; " /><meta name="citation_reference" content="citation_title=Transcriptome profile analysis of flowering molecular processes of early flowering trifoliate orange mutant and the wild-type [Poncirus trifoliata (L.) Raf.] by massively parallel signature sequencing;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=63" /><meta name="citation_reference" content="citation_title=Construction of AFLP fingerprint of 36 Poncirus accessions;  citation_journal_title=Acta Horticulturae Sinica;  citation_year=2003;  citation_volume=30;  citation_pages=394-8; " /><meta name="citation_reference" content="citation_title=Identification of flowering-related genes between early flowering trifoliate orange mutant and wild-type trifoliate orange (Poncirus trifoliata L. Raf.) by suppression subtraction hybridization (SSH) and macroarray;  citation_journal_title=Gene;  citation_year=2009;  citation_volume=430;  citation_pages=95-104; " /><meta name="citation_reference" content="citation_title=Citrus genomics;  citation_journal_title=Tree Genet. Genomes;  citation_year=2012;  citation_volume=8;  citation_pages=611-26; " /><meta name="citation_reference" content="citation_title=Sequencing technologies-the next generation;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2010;  citation_volume=11;  citation_pages=31-46; " /><meta name="citation_reference" content="citation_title=PtSVP, an SVP homolog from trifoliate orange (Poncirus trifoliata L. Raf.), shows seasonal periodicity of meristem determination and affects flower development in transgenic Arabidopsis and tobacco plants;  citation_journal_title=Plant Mol. Biol.;  citation_year=2010;  citation_volume=74;  citation_pages=129-42; " /><meta name="citation_reference" content="citation_title=An efficient protocol for genomic DNA extraction from Citrus species;  citation_journal_title=Plant Mol. Biol. Rep.;  citation_year=2003;  citation_volume=21;  citation_pages=177-8; " /><meta name="citation_reference" content="citation_title=Statistical analyses of next generation sequence data: a partial overview;  citation_journal_title=J. Proteomics Bioinform.;  citation_year=2010;  citation_volume=3;  citation_pages=183-90; " /><meta name="citation_reference" content="citation_title=Fast and accurate long-read alignment with Burrows–Wheeler transform;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=589-95; " /><meta name="citation_reference" content="citation_title=The sequence alignment/map format and SAMtools;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; " /><meta name="citation_reference" content="citation_title=SNP detection for massively parallel whole-genome resequencing;  citation_journal_title=Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=1124-32; " /><meta name="citation_reference" content="citation_title=SOAP2: an improved ultrafast tool for short read alignment;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1966-7; " /><meta name="citation_reference" content="citation_title=GeneWise and genomewise;  citation_journal_title=Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=988-95; " /><meta name="citation_reference" content="citation_title=Identification of early-flower-related ESTs in an early-flowering mutant of trifoliate orange (Poncirus trifoliata) by suppression subtractive hybridization and macroarray analysis;  citation_journal_title=Tree Physiol.;  citation_year=2008;  citation_volume=28;  citation_pages=1449-57; " /><meta name="citation_reference" content="citation_title=Characterization of transcriptional complexity during berry development in Vitis vinifera using RNA-Seq;  citation_journal_title=Plant Physiol.;  citation_year=2010;  citation_volume=152;  citation_pages=1787-95; " /><meta name="citation_reference" content="citation_title=annot8r: GO, EC and KEGG annotation of EST datasets;  citation_journal_title=BMC Bioinform.;  citation_year=2008;  citation_volume=9;  citation_pages=180" /><meta name="citation_reference" content="citation_title=Development and characterization of genomic and expressed SSRs in citrus by genome-wide analysis;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e75149" /><meta name="citation_reference" content="  Yeh F.C., Yang R., Boyle T., Ye Z., Mao J.X.  1999 , POPGENE, version 1.32: the user friendly software for population genetic analysis. Molecular Biology and Biotechnology Centre, University of Alberta, Edmonton, AB, Canada. " /><meta name="citation_reference" content="citation_title=The number of alleles that can be maintained in a finite population;  citation_journal_title=Genetics;  citation_year=1964;  citation_volume=49;  citation_pages=725-38; " /><meta name="citation_reference" content="citation_title=Testing the theory of natural selection;  citation_journal_title=Nature;  citation_year=1972;  citation_volume=236;  citation_pages=181-2; " /><meta name="citation_reference" content="citation_title=Analysis of gene diversity in subdivided populations;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=1973;  citation_volume=70;  citation_pages=3321-3; " /><meta name="citation_reference" content="citation_title=MEGA4: molecular evolutionary genetics analysis (MEGA) software version 4.0;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=1596-9; " /><meta name="citation_reference" content="citation_title=The use of AFLP markers for cultivar identification in apricot;  citation_journal_title=Plant Breed.;  citation_year=2003;  citation_volume=122;  citation_pages=526-31; " /><meta name="citation_reference" content="citation_title=Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm;  citation_journal_title=Nat. Protoc.;  citation_year=2009;  citation_volume=4;  citation_pages=1073-81; " /><meta name="citation_reference" content="citation_title=Genome-wide DNA polymorphisms in elite indica rice inbreds discovered by whole-genome sequencing;  citation_journal_title=Plant Biotechnol. J.;  citation_year=2012;  citation_volume=10;  citation_pages=623-34; " /><meta name="citation_reference" content="citation_title=Advances in research of citrus cultivars selected by bud mutation and the mechanism of formation of mutated characteristics;  citation_journal_title=J. Fruit Sci.;  citation_year=2006;  citation_volume=23;  citation_pages=871-6; " /><meta name="citation_reference" content="citation_title=Genome structural variation discovery and genotyping;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2011;  citation_volume=12;  citation_pages=363-76; " /><meta name="citation_reference" content="citation_title=Cancer genome landscapes;  citation_journal_title=Science;  citation_year=2013;  citation_volume=339;  citation_pages=1546-58; " /><meta name="citation_reference" content="citation_title=Frequency and characteristics of nucellar and zygotic seedlings in three cultivars of trifoliate orange;  citation_journal_title=J. Am. Soc. Hortic. Sci.;  citation_year=1988;  citation_volume=113;  citation_pages=105-10; " /><meta name="citation_reference" content="citation_title=Genome-wide characterization and expression analysis of genetic variants in sweet orange;  citation_journal_title=Plant J.;  citation_year=2013;  citation_volume=75;  citation_pages=954-64; " /><meta name="citation_reference" content="citation_title=The draft genome of sweet orange (Citrus sinensis);  citation_journal_title=Nat. Genet.;  citation_year=2013;  citation_volume=45;  citation_pages=59-66; " /><meta name="citation_reference" content="citation_title=Resequencing 50 accessions of cultivated and wild rice yields markers for identifying agronomically important genes;  citation_journal_title=Nat. Biotechnol.;  citation_year=2012;  citation_volume=30;  citation_pages=105-11; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of intraspecific DNA polymorphism in ‘Micro-Tom’, a model cultivar of tomato (Solanum lycopersicum);  citation_journal_title=Plant Cell Physiol.;  citation_year=2014;  citation_volume=55;  citation_pages=445-54; " /><meta name="citation_reference" content="citation_title=Common sequence polymorphisms shaping genetic diversity in Arabidopsis thaliana;  citation_journal_title=Science;  citation_year=2007;  citation_volume=317;  citation_pages=338-42; " /><meta name="citation_reference" content="citation_title=Genome-wide patterns of genetic variation among elite maize inbred lines;  citation_journal_title=Nat. Genet.;  citation_year=2010;  citation_volume=42;  citation_pages=1027-30; " /><meta name="citation_reference" content="citation_title=Natural genetic variation caused by small insertions and deletions in the human genome;  citation_journal_title=Genome Res.;  citation_year=2011;  citation_volume=21;  citation_pages=830-9; " /><meta name="citation_reference" content="citation_title=Genome-wide SNP validation and mantle tissue transcriptome analysis in the silver-lipped pearl oyster, Pinctada maxima;  citation_journal_title=Mar. Biotechnol.;  citation_year=2013;  citation_volume=15;  citation_pages=647-58; " /><meta name="citation_reference" content="citation_title=Discovery of genome-wide DNA polymorphisms in a landrace cultivar of japonica rice by whole-genome sequencing;  citation_journal_title=Plant Cell Physiol.;  citation_year=2011;  citation_volume=52;  citation_pages=274-82; " /><meta name="citation_reference" content="citation_title=A collection of INDEL markers for map-based cloning in seven Arabidopsis accessions;  citation_journal_title=J. Exp. Bot.;  citation_year=2012;  citation_volume=63;  citation_pages=2491-501; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/4/403/17462919/dsw017.pdf" /><meta name="description" content="Abstract.  To increase our understanding of the genes involved in flowering in citrus, we performed genome resequencing of an early flowering trifoliate orange " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/4/403/2469959" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/4/403/2469959" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/4/403/2469959");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw017';
    </script>
<title>
	Identifying the genome-wide genetic variation between precocious trifoliate orange and its wild type and developing new markers for genetics research | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2469959" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BchwPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJaaGFuZywgSmluLVpoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIEhvcnRpY3VsdHVyYWwgUGxhbnQgQmlvbG9neSAoTWluaXN0cnkgb2YgRWR1Y2F0aW9uKSwgQ29sbGVnZSBvZiBIb3J0aWN1bHR1cmUgYW5kIEZvcmVzdHJ5IFNjaWVuY2UsIEh1YXpob25nIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBXdWhhbiA0MzAwNzAsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGl1LCBTaGVuZy1SdWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBIb3J0aWN1bHR1cmFsIFBsYW50IEJpb2xvZ3kgKE1pbmlzdHJ5IG9mIEVkdWNhdGlvbiksIENvbGxlZ2Ugb2YgSG9ydGljdWx0dXJlIGFuZCBGb3Jlc3RyeSBTY2llbmNlLCBIdWF6aG9uZyBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgV3VoYW4gNDMwMDcwLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikh1LCBDaHVuLUdlbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIEhvcnRpY3VsdHVyYWwgUGxhbnQgQmlvbG9neSAoTWluaXN0cnkgb2YgRWR1Y2F0aW9uKSwgQ29sbGVnZSBvZiBIb3J0aWN1bHR1cmUgYW5kIEZvcmVzdHJ5IFNjaWVuY2UsIEh1YXpob25nIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBXdWhhbiA0MzAwNzAsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJJZGVudGlmeWluZyB0aGUgZ2Vub21lLXdpZGUgZ2VuZXRpYyB2YXJpYXRpb24gYmV0d2VlbiBwcmVjb2Npb3VzIHRyaWZvbGlhdGUgb3JhbmdlIGFuZCBpdHMgd2lsZCB0eXBlIGFuZCBkZXZlbG9waW5nIG5ldyBtYXJrZXJzIGZvciBnZW5ldGljcyByZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjQwMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNDE0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN3MDE3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTYvMDgvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWd1bGF0aW9uIG9mIGZsb3JhbCBwYXR0ZXJuaW5nIGJ5IGZsb3dlcmluZyB0aW1lIGdlbmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1EZXYuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz03MTEtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgcmVndWxhdGlvbiBvZiBmbG93ZXJpbmcgdGltZSBpbiBhbm51YWwgYW5kIHBlcmVubmlhbCBwbGFudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVdpbGV5IEludGVyZGlzY2lwLiBSZXYuIFJOQTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MzQ3LTU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db250cm9sIG9mIGZsb3dlcmluZyB0aW1lIGludGVyYWN0aW5nIHBhdGh3YXlzIGFzIGEgYmFzaXMgZm9yIGRpdmVyc2l0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX2lzc3VlPShTdXBwbC4gMSk7ICBjaXRhdGlvbl9wYWdlcz1TMTExLTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdWx0aXBsZSBwYXRod2F5cyBpbiB0aGUgZGVjaXNpb24gdG8gZmxvd2VyOiBlbmFibGluZywgcHJvbW90aW5nLCBhbmQgcmVzZXR0aW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25faXNzdWU9KFN1cHBsLiAxKTsgIGNpdGF0aW9uX3BhZ2VzPVMxOC0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc2VydmF0aW9uIGFuZCBkaXZlcnNpdHkgaW4gZmxvd2VyIGxhbmQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIE9waW4uIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz04NC05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpY3Mgb2YgZmxvd2VyIGluaXRpYXRpb24gYW5kIGRldmVsb3BtZW50IGluIGFubnVhbCBhbmQgcGVyZW5uaWFsIHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGh5c2lvbC4gUGxhbnQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTEyODsgIGNpdGF0aW9uX3BhZ2VzPTgtMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bmN0aW9uYWwgY2hhcmFjdGVyaXphdGlvbiBvZiBBUDMsIFNPQzEgYW5kIFdVUyBob21vbG9ndWVzIGZyb20gY2l0cnVzIChDaXRydXMgc2luZW5zaXMpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QaHlzaW9sLiBQbGFudC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTMxOyAgY2l0YXRpb25fcGFnZXM9NDgxLTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaXRydXMgZnJ1aXRzOyBjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXIsIEhlaWRlbGJlcmc7ICBjaXRhdGlvbl90aXRsZT1GcnVpdHMgYW5kIE51dHMsIEdlbm9tZSBNYXBwaW5nICZhbXA7IE1vbGVjdWxhciBCcmVlZGluZyBpbiBQbGFudHM7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl9wYWdlcz0yNjUtNzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNpbmcgb2YgZGl2ZXJzZSBtYW5kYXJpbiwgcHVtbWVsbyBhbmQgb3JhbmdlIGdlbm9tZXMgcmV2ZWFscyBjb21wbGV4IGhpc3Rvcnkgb2YgYWRtaXh0dXJlIGR1cmluZyBjaXRydXMgZG9tZXN0aWNhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTY1Ni02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlY29jaW91cyB0cmlmb2xpYXRlIG9yYW5nZSAoUG9uY2lydXMgdHJpZm9saWF0YSBMLiBSYWYuKSBiaW9sb2d5IGNoYXJhY3RlcmlzdGljIGFuZCBpdHMgc3RvY2sgZXhwZXJpbWVudDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9WmhlSmlhbmcgQ2l0cnVzOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9Mi00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0b21lIHByb2ZpbGUgYW5hbHlzaXMgb2YgZmxvd2VyaW5nIG1vbGVjdWxhciBwcm9jZXNzZXMgb2YgZWFybHkgZmxvd2VyaW5nIHRyaWZvbGlhdGUgb3JhbmdlIG11dGFudCBhbmQgdGhlIHdpbGQtdHlwZSBbUG9uY2lydXMgdHJpZm9saWF0YSAoTC4pIFJhZi5dIGJ5IG1hc3NpdmVseSBwYXJhbGxlbCBzaWduYXR1cmUgc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgQUZMUCBmaW5nZXJwcmludCBvZiAzNiBQb25jaXJ1cyBhY2Nlc3Npb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BY3RhIEhvcnRpY3VsdHVyYWUgU2luaWNhOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9Mzk0LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIGZsb3dlcmluZy1yZWxhdGVkIGdlbmVzIGJldHdlZW4gZWFybHkgZmxvd2VyaW5nIHRyaWZvbGlhdGUgb3JhbmdlIG11dGFudCBhbmQgd2lsZC10eXBlIHRyaWZvbGlhdGUgb3JhbmdlIChQb25jaXJ1cyB0cmlmb2xpYXRhIEwuIFJhZi4pIGJ5IHN1cHByZXNzaW9uIHN1YnRyYWN0aW9uIGh5YnJpZGl6YXRpb24gKFNTSCkgYW5kIG1hY3JvYXJyYXk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDMwOyAgY2l0YXRpb25fcGFnZXM9OTUtMTA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaXRydXMgZ2Vub21pY3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZWUgR2VuZXQuIEdlbm9tZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTYxMS0yNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2luZyB0ZWNobm9sb2dpZXMtdGhlIG5leHQgZ2VuZXJhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTMxLTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QdFNWUCwgYW4gU1ZQIGhvbW9sb2cgZnJvbSB0cmlmb2xpYXRlIG9yYW5nZSAoUG9uY2lydXMgdHJpZm9saWF0YSBMLiBSYWYuKSwgc2hvd3Mgc2Vhc29uYWwgcGVyaW9kaWNpdHkgb2YgbWVyaXN0ZW0gZGV0ZXJtaW5hdGlvbiBhbmQgYWZmZWN0cyBmbG93ZXIgZGV2ZWxvcG1lbnQgaW4gdHJhbnNnZW5pYyBBcmFiaWRvcHNpcyBhbmQgdG9iYWNjbyBwbGFudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NzQ7ICBjaXRhdGlvbl9wYWdlcz0xMjktNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGVmZmljaWVudCBwcm90b2NvbCBmb3IgZ2Vub21pYyBETkEgZXh0cmFjdGlvbiBmcm9tIENpdHJ1cyBzcGVjaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBNb2wuIEJpb2wuIFJlcC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0xNzctODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RhdGlzdGljYWwgYW5hbHlzZXMgb2YgbmV4dCBnZW5lcmF0aW9uIHNlcXVlbmNlIGRhdGE6IGEgcGFydGlhbCBvdmVydmlldzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gUHJvdGVvbWljcyBCaW9pbmZvcm0uOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0xODMtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3QgYW5kIGFjY3VyYXRlIGxvbmctcmVhZCBhbGlnbm1lbnQgd2l0aCBCdXJyb3dz4oCTV2hlZWxlciB0cmFuc2Zvcm07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9NTg5LTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc2VxdWVuY2UgYWxpZ25tZW50L21hcCBmb3JtYXQgYW5kIFNBTXRvb2xzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTIwNzgtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U05QIGRldGVjdGlvbiBmb3IgbWFzc2l2ZWx5IHBhcmFsbGVsIHdob2xlLWdlbm9tZSByZXNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MTEyNC0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U09BUDI6IGFuIGltcHJvdmVkIHVsdHJhZmFzdCB0b29sIGZvciBzaG9ydCByZWFkIGFsaWdubWVudDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xOTY2LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmVXaXNlIGFuZCBnZW5vbWV3aXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTk4OC05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgZWFybHktZmxvd2VyLXJlbGF0ZWQgRVNUcyBpbiBhbiBlYXJseS1mbG93ZXJpbmcgbXV0YW50IG9mIHRyaWZvbGlhdGUgb3JhbmdlIChQb25jaXJ1cyB0cmlmb2xpYXRhKSBieSBzdXBwcmVzc2lvbiBzdWJ0cmFjdGl2ZSBoeWJyaWRpemF0aW9uIGFuZCBtYWNyb2FycmF5IGFuYWx5c2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVlIFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MTQ0OS01NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbiBvZiB0cmFuc2NyaXB0aW9uYWwgY29tcGxleGl0eSBkdXJpbmcgYmVycnkgZGV2ZWxvcG1lbnQgaW4gVml0aXMgdmluaWZlcmEgdXNpbmcgUk5BLVNlcTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTUyOyAgY2l0YXRpb25fcGFnZXM9MTc4Ny05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9YW5ub3Q4cjogR08sIEVDIGFuZCBLRUdHIGFubm90YXRpb24gb2YgRVNUIGRhdGFzZXRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MTgwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgZ2Vub21pYyBhbmQgZXhwcmVzc2VkIFNTUnMgaW4gY2l0cnVzIGJ5IGdlbm9tZS13aWRlIGFuYWx5c2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTc1MTQ5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iICBZZWggRi5DLiwgWWFuZyBSLiwgQm95bGUgVC4sIFllIFouLCBNYW8gSi5YLiAgMTk5OSAsIFBPUEdFTkUsIHZlcnNpb24gMS4zMjogdGhlIHVzZXIgZnJpZW5kbHkgc29mdHdhcmUgZm9yIHBvcHVsYXRpb24gZ2VuZXRpYyBhbmFseXNpcy4gTW9sZWN1bGFyIEJpb2xvZ3kgYW5kIEJpb3RlY2hub2xvZ3kgQ2VudHJlLCBVbml2ZXJzaXR5IG9mIEFsYmVydGEsIEVkbW9udG9uLCBBQiwgQ2FuYWRhLiAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbnVtYmVyIG9mIGFsbGVsZXMgdGhhdCBjYW4gYmUgbWFpbnRhaW5lZCBpbiBhIGZpbml0ZSBwb3B1bGF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk2NDsgIGNpdGF0aW9uX3ZvbHVtZT00OTsgIGNpdGF0aW9uX3BhZ2VzPTcyNS0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGVzdGluZyB0aGUgdGhlb3J5IG9mIG5hdHVyYWwgc2VsZWN0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTE5NzI7ICBjaXRhdGlvbl92b2x1bWU9MjM2OyAgY2l0YXRpb25fcGFnZXM9MTgxLTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIGdlbmUgZGl2ZXJzaXR5IGluIHN1YmRpdmlkZWQgcG9wdWxhdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTczOyAgY2l0YXRpb25fdm9sdW1lPTcwOyAgY2l0YXRpb25fcGFnZXM9MzMyMS0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NRUdBNDogbW9sZWN1bGFyIGV2b2x1dGlvbmFyeSBnZW5ldGljcyBhbmFseXNpcyAoTUVHQSkgc29mdHdhcmUgdmVyc2lvbiA0LjA7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xNTk2LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB1c2Ugb2YgQUZMUCBtYXJrZXJzIGZvciBjdWx0aXZhciBpZGVudGlmaWNhdGlvbiBpbiBhcHJpY290OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBCcmVlZC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTIyOyAgY2l0YXRpb25fcGFnZXM9NTI2LTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW5nIHRoZSBlZmZlY3RzIG9mIGNvZGluZyBub24tc3lub255bW91cyB2YXJpYW50cyBvbiBwcm90ZWluIGZ1bmN0aW9uIHVzaW5nIHRoZSBTSUZUIGFsZ29yaXRobTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBQcm90b2MuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0xMDczLTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBETkEgcG9seW1vcnBoaXNtcyBpbiBlbGl0ZSBpbmRpY2EgcmljZSBpbmJyZWRzIGRpc2NvdmVyZWQgYnkgd2hvbGUtZ2Vub21lIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEJpb3RlY2hub2wuIEouOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NjIzLTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZHZhbmNlcyBpbiByZXNlYXJjaCBvZiBjaXRydXMgY3VsdGl2YXJzIHNlbGVjdGVkIGJ5IGJ1ZCBtdXRhdGlvbiBhbmQgdGhlIG1lY2hhbmlzbSBvZiBmb3JtYXRpb24gb2YgbXV0YXRlZCBjaGFyYWN0ZXJpc3RpY3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEZydWl0IFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz04NzEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHN0cnVjdHVyYWwgdmFyaWF0aW9uIGRpc2NvdmVyeSBhbmQgZ2Vub3R5cGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTM2My03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2FuY2VyIGdlbm9tZSBsYW5kc2NhcGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTMzOTsgIGNpdGF0aW9uX3BhZ2VzPTE1NDYtNTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZyZXF1ZW5jeSBhbmQgY2hhcmFjdGVyaXN0aWNzIG9mIG51Y2VsbGFyIGFuZCB6eWdvdGljIHNlZWRsaW5ncyBpbiB0aHJlZSBjdWx0aXZhcnMgb2YgdHJpZm9saWF0ZSBvcmFuZ2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEFtLiBTb2MuIEhvcnRpYy4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3ZvbHVtZT0xMTM7ICBjaXRhdGlvbl9wYWdlcz0xMDUtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGNoYXJhY3Rlcml6YXRpb24gYW5kIGV4cHJlc3Npb24gYW5hbHlzaXMgb2YgZ2VuZXRpYyB2YXJpYW50cyBpbiBzd2VldCBvcmFuZ2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEouOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTc1OyAgY2l0YXRpb25fcGFnZXM9OTU0LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZHJhZnQgZ2Vub21lIG9mIHN3ZWV0IG9yYW5nZSAoQ2l0cnVzIHNpbmVuc2lzKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz01OS02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVzZXF1ZW5jaW5nIDUwIGFjY2Vzc2lvbnMgb2YgY3VsdGl2YXRlZCBhbmQgd2lsZCByaWNlIHlpZWxkcyBtYXJrZXJzIGZvciBpZGVudGlmeWluZyBhZ3Jvbm9taWNhbGx5IGltcG9ydGFudCBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTEwNS0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgYW5hbHlzaXMgb2YgaW50cmFzcGVjaWZpYyBETkEgcG9seW1vcnBoaXNtIGluIOKAmE1pY3JvLVRvbeKAmSwgYSBtb2RlbCBjdWx0aXZhciBvZiB0b21hdG8gKFNvbGFudW0gbHljb3BlcnNpY3VtKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01NTsgIGNpdGF0aW9uX3BhZ2VzPTQ0NS01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tbW9uIHNlcXVlbmNlIHBvbHltb3JwaGlzbXMgc2hhcGluZyBnZW5ldGljIGRpdmVyc2l0eSBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTc7ICBjaXRhdGlvbl9wYWdlcz0zMzgtNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIHBhdHRlcm5zIG9mIGdlbmV0aWMgdmFyaWF0aW9uIGFtb25nIGVsaXRlIG1haXplIGluYnJlZCBsaW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz0xMDI3LTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OYXR1cmFsIGdlbmV0aWMgdmFyaWF0aW9uIGNhdXNlZCBieSBzbWFsbCBpbnNlcnRpb25zIGFuZCBkZWxldGlvbnMgaW4gdGhlIGh1bWFuIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz04MzAtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgU05QIHZhbGlkYXRpb24gYW5kIG1hbnRsZSB0aXNzdWUgdHJhbnNjcmlwdG9tZSBhbmFseXNpcyBpbiB0aGUgc2lsdmVyLWxpcHBlZCBwZWFybCBveXN0ZXIsIFBpbmN0YWRhIG1heGltYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWFyLiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTY0Ny01ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzY292ZXJ5IG9mIGdlbm9tZS13aWRlIEROQSBwb2x5bW9ycGhpc21zIGluIGEgbGFuZHJhY2UgY3VsdGl2YXIgb2YgamFwb25pY2EgcmljZSBieSB3aG9sZS1nZW5vbWUgc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT01MjsgIGNpdGF0aW9uX3BhZ2VzPTI3NC04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb2xsZWN0aW9uIG9mIElOREVMIG1hcmtlcnMgZm9yIG1hcC1iYXNlZCBjbG9uaW5nIGluIHNldmVuIEFyYWJpZG9wc2lzIGFjY2Vzc2lvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEV4cC4gQm90LjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT02MzsgIGNpdGF0aW9uX3BhZ2VzPTI0OTEtNTAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjMvNC80MDMvMTc0NjI5MTkvZHN3MDE3LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBUbyBpbmNyZWFzZSBvdXIgdW5kZXJzdGFuZGluZyBvZiB0aGUgZ2VuZXMgaW52b2x2ZWQgaW4gZmxvd2VyaW5nIGluIGNpdHJ1cywgd2UgcGVyZm9ybWVkIGdlbm9tZSByZXNlcXVlbmNpbmcgb2YgYW4gZWFybHkgZmxvd2VyaW5nIHRyaWZvbGlhdGUgb3JhbmdlICIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MjQ2OTk1OSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGRypudyvffgyKzlXn5qiMuqMBNvOQ==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f4%2f403%2f2469959" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f4%2f403%2f2469959" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=84049" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/4/0/m_cover.gif?Expires=2147483647&amp;Signature=XWSbFJdI5lGDaRny1g7BRzfZg~ebzc~OT20sUUxiNUi3-W~7uvtu5IDIgO3L5~QjmCQzUopLMGz-3gEHtIKUOFbRCHM2eH4h7l2Wu3nqAQC9YYHygJi-TLC9x3FpQRq3lTBcdWmpj185LJbB-1H-PKlUoEU9gxYyCNvjJUg~GUaHNRtVxbV0a7bDIfdyclZJ~w1BhGonEiYqh-IZSS0VekFWZ7mnLia~WbOEtL8AzSLsoIVak5-aADczuqIYGRkdfxcEh8zTdO-NlGhKxLoi5zlJy0Y~XuIxHZNxQxDFoVA3s~0J2fujGZsWX~ElpEU3I2Vma3M6MgFM80P2xPWpTw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/4">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87197432">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197432">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197434">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197434">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197438">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197438">2. Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197455">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197455">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197482">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197482">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197488">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197488">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197490">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197490">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197492">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197492">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/4/395/2469961">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Identifying the genome-wide genetic variation between precocious trifoliate orange and its wild type and developing new markers for genetics research
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jin-Zhi Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Horticultural Plant Biology (Ministry of Education)</div>, <div class="insititution">College of Horticulture and Forestry Science, Huazhong Agricultural University</div>, <div class="addr-line">Wuhan 430070</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jin-Zhi+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jin-Zhi Zhang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sheng-Rui Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Horticultural Plant Biology (Ministry of Education)</div>, <div class="insititution">College of Horticulture and Forestry Science, Huazhong Agricultural University</div>, <div class="addr-line">Wuhan 430070</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sheng-Rui+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liu S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sheng-Rui Liu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chun-Gen Hu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Key Laboratory of Horticultural Plant Biology (Ministry of Education)</div>, <div class="insititution">College of Horticulture and Forestry Science, Huazhong Agricultural University</div>, <div class="addr-line">Wuhan 430070</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 27-87281826. Fax. +86 27-87281826. E-mail: <a href="mailto:chungen@mail.hzau.edu.cn" target="_blank">chungen@mail.hzau.edu.cn</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chun-Gen+Hu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hu C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hu C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Chun-Gen Hu</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="TN1"><div class="fn"><div class="footnote-content">Edited by Dr Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 4, 1 August 2016, Pages 403–414, <a href='https://doi.org/10.1093/dnares/dsw017'>https://doi.org/10.1093/dnares/dsw017</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">21 April 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">13 December 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">21 March 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2469959" href="/dnaresearch/article-pdf/23/4/403/17462919/dsw017.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Jin-Zhi Zhang, Sheng-Rui Liu, Chun-Gen Hu;  Identifying the genome-wide genetic variation between precocious trifoliate orange and its wild type and developing new markers for genetics research, <em>DNA Research</em>, Volume 23, Issue 4, 1 August 2016, Pages 403–414, <a href='https://doi.org/10.1093/dnares/dsw017'>https://doi.org/10.1093/dnares/dsw017</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2469959?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469959?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469959?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469959?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469959?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469959?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469959?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Identifying%20the%20genome-wide%20genetic%20variation%20between%20precocious%20trifoliate%20orange%20and%20its%20wild%20type%20and%20developing%20new%20markers%20for%20genetics%20research&amp;publicationDate=2016-04-21&amp;volumeNum=23&amp;issueNum=4&amp;author=Zhang%2C%20Jin-Zhi%3B%20Liu%2C%20Sheng-Rui&amp;startPage=403&amp;endPage=414&amp;contentId=10.1093%2Fdnares%2Fdsw017&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f4%2f403%2f2469959" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87197432 id="87197432" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>To increase our understanding of the genes involved in flowering in citrus, we performed genome resequencing of an early flowering trifoliate orange mutant (<em>Poncirus trifoliata</em> L. Raf.) and its wild type. At the genome level, 3,932,628 single nucleotide polymorphisms (SNPs), 1,293,383 insertion/deletion polymorphisms (InDels), and 52,135 structural variations were identified between the mutant and its wild type based on the citrus reference genome. Based on integrative analysis of resequencing and transcriptome analysis, 233,998 SNPs and 75,836 InDels were also identified between the mutant and its wild type at the transcriptional level. Also, 272 citrus homologous flowering-time transcripts containing genetic variation were also identified. Gene Ontology and Kyoto Encyclopaedia of Genes and Genomes annotation revealed that the transcripts containing the mutant- and the wild-type-specific InDel were involved in diverse biological processes and molecular function. Among these transcripts, there were 131 transcripts that were expressed differently in the two genotypes. When 268 selected InDels were tested on 32 genotypes of the three genera of Rutaceae for the genetic diversity assessment, these InDel-based markers showed high transferability. This work provides important information that will allow a better understanding of the citrus genome and that will be helpful for dissecting the genetic basis of important traits in citrus.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="citrus">citrus</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="flowering">flowering</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome resequencing&quot;">genome resequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genetic variation&quot;">genetic variation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SNP">SNP</a></div>                            <h2 scrollto-destination=87197434 id="87197434" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Flower induction and initiation are the key developmental stages for flowering plants. Recently, extensive analyses performed in <em>Arabidopsis</em> have provided a complex picture of how these plants integrate environmental and endogenous signals to regulate the flowering transition.<sup><a reveal-id="DSW017C1" data-open="DSW017C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Several flowering regulatory pathways function to promote or repress flowering depending on the environmental and endogenous conditions identified, such as photoperiod, vernalization, gibberellins, autonomous, and ageing pathways.<sup><a reveal-id="DSW017C1" data-open="DSW017C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSW017C2" data-open="DSW017C2" class="link link-ref link-reveal xref-bibr">2</a></sup> The balance of signals from these flowering pathways is integrated by a common set of genes, such as <em>FLOWERING LOCUS T</em> (<em>FT</em>), <em>FLOWERING LOCUS C</em> (<em>FLC</em>), <em>LEAFY</em> (<em>LFY</em>), and <em>SUPPRESSOR OF OVEREXPRESSION OF CONSTANS 1</em> (<em>SOC1</em>), and could regulate the transition from the juvenile to the adult phase.<sup><a reveal-id="DSW017C3" data-open="DSW017C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSW017C4" data-open="DSW017C4" class="link link-ref link-reveal xref-bibr">4</a></sup> This knowledge has greatly accelerated research related to flowering in perennial plants.<sup><a reveal-id="DSW017C5" data-open="DSW017C5" class="link link-ref link-reveal xref-bibr">5</a></sup> However, when homologous genes from perennial plants were analysed, differences regarding expression patterns and phenotypes were detected when compared with <em>Arabidopsis</em>, and these differences may reflect roles distinct from those described in model plants.<sup><a reveal-id="DSW017C2" data-open="DSW017C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSW017C6" data-open="DSW017C6" class="link link-ref link-reveal xref-bibr">6</a></sup> These studies indicated that the underlying molecular mechanism of flowering time may differ between perennial and model plants because of different flowering characteristics, such as long juvenile phase and seasonal flowering; some novel genes may also play a key role in these processes. Therefore, an understanding of these different characteristics requires identification and characterization of novel genes related to these characteristics in woody plants.</p><p>Citrus is one of the most economically important evergreen crops for the production of fresh fruit and juice; its economic and social impact on our society is tremendous.<sup><a reveal-id="DSW017C7  DSW017C9" data-open="DSW017C7  DSW017C9" class="link link-ref link-reveal xref-bibr">7–9</a></sup> However, the flowering transition of most citrus plants is 6–8 yr, and for some species such as seedling oranges and grapefruit it is 8–10 yr.<sup><a reveal-id="DSW017C8" data-open="DSW017C8" class="link link-ref link-reveal xref-bibr">8</a></sup> However, commercially important traits of citrus are expressed primarily in fruit tissue. This requires the trees to be capable of flowering and producing fruit to evaluate functions of these genes. Precocious trifoliate orange, a spontaneous mutant with a short juvenile phase from WT trifoliate orange (<em>Poncirus trifoliata</em> L. Raf.) was found in Yichang, Hubei Province, China.<sup><a reveal-id="DSW017C10" data-open="DSW017C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Approximately 20% of the seedlings from the mutant seeds first flowered during the year after the spring planting; the juvenile period of the mutant has been greatly reduced to 1– 2 yr when compared with the WT plants, which have a juvenile period of 6– 8 yr.<sup><a reveal-id="DSW017C10" data-open="DSW017C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSW017C11" data-open="DSW017C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Precocious trifoliate orange and its wild type have nearly the same morphology, except for the flowering habit, and no DNA polymorphism has been detected between them based on some molecular markers.<sup><a reveal-id="DSW017C10  DSW017C12" data-open="DSW017C10  DSW017C12" class="link link-ref link-reveal xref-bibr">10–12</a></sup> Therefore, precocious trifoliate orange is speculated to be a direct variant of the wild type. Previous transcriptional studies including suppression subtraction hybridization combined with macroarray and massively parallel signature sequencing were used to investigate transcriptome changes between the mutant (MT) and its wild type (WT), and several differentially expressed genes were detected.<sup><a reveal-id="DSW017C11" data-open="DSW017C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSW017C13" data-open="DSW017C13" class="link link-ref link-reveal xref-bibr">13</a></sup> However, it remains largely unknown what kind of mechanism results in early flowering.</p><p>The recent release of the citrus genome<sup><a reveal-id="DSW017C9" data-open="DSW017C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSW017C14" data-open="DSW017C14" class="link link-ref link-reveal xref-bibr">14</a></sup> and next-generation DNA sequencing technology<sup><a reveal-id="DSW017C15" data-open="DSW017C15" class="link link-ref link-reveal xref-bibr">15</a></sup> will dramatically enhance the efficiency of functional and comparative genomics research in citrus. The alignment of the short reads obtained from the MT and the WT to the citrus reference genome has provided the perfect opportunity to identify a large number of genetic variations between the MT and the WT, including single-nucleotide polymorphisms (SNPs), insertion and deletion polymorphisms (InDels), and structural variations (SVs). Here, we sequenced the genome of the MT and the WT; several putative genetic variations including SNPs, InDels, and SVs were identified. The frequency and distribution of these genetic variations in different regions of gene were also identified. In addition, we developed a great deal of effective DNA markers for exploring genetic diversity in the citrus genus.</p>                            <h2 scrollto-destination=87197438 id="87197438" class="section-title" data-legacy-id=s2>2. Methods</h2>
                            <h3 scrollto-destination=87197439 id="87197439" class="section-title" data-legacy-id=s2a>2.1. Plant materials</h3>
<p>All samples from 32 citrus species (fortunella; kumquat; trifoliate orange; precocious trifoliate orange; flying dragon trifoliate orange; citranges; Fenghuang pummelo; Kaopan pummelo; grapefruit; Hirado Buntan pummelo; guoqing no. 1 satsuma; Qingjiang Ponkan; Bendiguangju mandarin; Miyagawa wase satsuma; red tangerine; calamondin; Clementine mandarin; Newhall navel orange; Valencia orange; Jingcheng sweet orange; Honganliu sweet orange; sour orange; Cara Cara navel orange; Washington navel orange; blood orange; Ichang papeda; papeda; citron; bergamot; Mexican lime; lemon; and rough lemon) were collected in the experiment fields of the National Citrus Breeding Center at Huazhong Agricultural University. To identify flowering-related genetic variation at the transcriptional level, the terminal bud and the five following buds (the major node position for flower formation) from spring flushes of the MT and the WT 2-yr-old trees were also collected at flower initiation (after self-pruning). Previous cytological studies revealed that the floral buds of spring shoots in 2-yr-old precocious trifoliate orange initiated differentiation immediately after self-pruning, but its WT did not.<sup><a reveal-id="DSW017C16" data-open="DSW017C16" class="link link-ref link-reveal xref-bibr">16</a></sup> To eliminate the influence of the genetic background, the materials used for genome re-sequencing and RNA-seq from the same 2-yr-old trees in this study. All materials for RNA and DNA extraction were collected from three individual plants. All plant tissues were sampled according to the demands of each experiment, immediately frozen in liquid nitrogen, and stored at −80°C until used.</p>                            <h3 scrollto-destination=87197441 id="87197441" class="section-title" data-legacy-id=s2b>2.2. DNA isolation and genome sequencing</h3>
<p>Plant DNA of all material was isolated from the leaf according to the cetyltrimethyl ammonium bromide method.<sup><a reveal-id="DSW017C17" data-open="DSW017C17" class="link link-ref link-reveal xref-bibr">17</a></sup> For genome resequencing, the DNA from the MT and the WT were randomly sheared. After electrophoresis, DNA fragments of the desired length were gel-purified. Adaptor ligation and DNA cluster preparation were performed and subjected to Solexa sequencing using Illumina Genome Analyzer II.<sup><a reveal-id="DSW017C18" data-open="DSW017C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Low-quality reads (&lt;20), reads with an adaptor sequence, and duplicated reads were filtered, and the remaining high-quality data were used for mapping. Raw sequence data obtained in our study have been deposited in the NCBI Short Read Archive with accession number SRP070975.</p>                            <h3 scrollto-destination=87197443 id="87197443" class="section-title" data-legacy-id=s2c>2.3. Detection of SNPs, InDels, and SVs</h3>
<p>The sequencing reads from the MT and the WT were aligned to the Clementine genome (<a class="link link-uri" href="http://www.phytozome.net/clementine.php">http://www.phytozome.net/clementine.php</a>) separately using Burrows–Wheeler Aligner (BWA) software.<sup><a reveal-id="DSW017C19" data-open="DSW017C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Reads that aligned to more than one position of the reference genome were filtered and used for determining reads mapping to multiple positions in the reference and unmapped reads. Average sequencing depth and coverage were calculated using the alignment results.<sup><a reveal-id="DSW017C20" data-open="DSW017C20" class="link link-ref link-reveal xref-bibr">20</a></sup> The mapped reads were then used to detect SNPs, InDels, and SVs using SOAPsnp and SOAPsv software with default parameter settings.<sup><a reveal-id="DSW017C20  DSW017C22" data-open="DSW017C20  DSW017C22" class="link link-ref link-reveal xref-bibr">20–22</a></sup> The SNPs identified were filtered based on the following stringent criteria: no less than two times for coverage depth (no less than three times in the heterozygous locus), no more than three times of average depth, distance of adjacent variation had to be &gt;5 bp, and target mapping quality had to be &gt;20. For the InDels, gaps supported by at least three pair-end (PE) reads were retained. For obtaining reliable SVs, the detected SVs must be returned to the PE alignments between resequencing genome and the reference genome and be validated under the following criteria: 2× to 100× for coverage depth and &gt;20 for SVs quality.</p>                            <h3 scrollto-destination=87197445 id="87197445" class="section-title" data-legacy-id=s2d>2.4. Annotation of SNPs, InDels, and SVs</h3>
<p>Localization of the SNPs, InDels, and SVs was based on the annotation of gene models of the Clementine genome (<a class="link link-uri" href="http://www.phytozome.net/clementine.php">http://www.phytozome.net/clementine.php</a>). The three types of polymorphisms in the gene region and other genome regions were annotated as genic and intergenic, respectively. The genic SNPs, InDels, and SVs were classified as CDS, UTR, and intron according to their localization. The SNPs in the CDS region were further separated into synonymous and non-synonymous amino substitutions using Genewise.<sup><a reveal-id="DSW017C23" data-open="DSW017C23" class="link link-ref link-reveal xref-bibr">23</a></sup> The Gene Ontology (GO)/PFAM annotation data were further used to functionally annotate each gene including non-synonymous SNPs with 1- to 10-bp lengths.</p>                            <h3 scrollto-destination=87197447 id="87197447" class="section-title" data-legacy-id=s2e>2.5. RNA-Seq data analyses</h3>
<p>To prepare a representative sample of total RNA, we pooled three individual plants from approximately equal numbers of the MT and the WT materials, respectively. Total RNA was extracted according to a previous protocol.<sup><a reveal-id="DSW017C24" data-open="DSW017C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Library construction and sequencing were performed as described by Zenoni et al.<sup><a reveal-id="DSW017C25" data-open="DSW017C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Low-quality reads (&lt;20), reads with adaptor sequence, and duplicated reads were filtered, and the remaining high-quality data were used in the mapping. The PE sequencing reads were aligned to the citrus reference genome sequence separately using BWA software algorithm under the default parameters as described for genome sequencing. To identify flowering-related genetic variation in the two genotypes, we mapped the reads against the Clementine genome<sup><a reveal-id="DSW017C9" data-open="DSW017C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSW017C14" data-open="DSW017C14" class="link link-ref link-reveal xref-bibr">14</a></sup> using the SOAP2 software with default parameter settings as described for genome sequencing.<sup><a reveal-id="DSW017C22" data-open="DSW017C22" class="link link-ref link-reveal xref-bibr">22</a></sup> These identified SNPs and InDels from RNA-Seq were annotated the genic genetic variation by identified of genome sequencing. A candidate genetic variation was identified by both RNA-Seq and genomic DNA reads, for which the available flanking sequence matched 100% over the entire length at a single location. It will be considered a real variation at the transcriptional level. The RNA-Seq data from this study have been submitted to Gene Expression Omnibus under accession number GSE78810.</p>                            <h3 scrollto-destination=87197449 id="87197449" class="section-title" data-legacy-id=s2f>2.6. Functional assignments of the transcripts containing specific InDels</h3>
<p>To assign putative functions to the MT and the WT containing specific InDel transcripts, Blast2Go program was run to BLAST against a reference database that stores UniProt entries, GO, Enzyme Commission (EC), and Kyoto Encyclopaedia of Genes and Genomes (KEGG) annotation.<sup><a reveal-id="DSW017C26" data-open="DSW017C26" class="link link-ref link-reveal xref-bibr">26</a></sup> The GO categorization results were expressed as three independent hierarchies for biological process, cellular component, and molecular function.<sup><a reveal-id="DSW017C26" data-open="DSW017C26" class="link link-ref link-reveal xref-bibr">26</a></sup></p>                            <h3 scrollto-destination=87197451 id="87197451" class="section-title" data-legacy-id=s2g>2.7. New marker screening and polymorphism survey</h3>
<p>For Sanger sequencing, primers spanning genomic regions predicted to contain InDels were used to amplify genomic DNA templates from the Clementine, the MT, and the WT. PCR amplification was conducted in 25-µl reactions containing 50 ng template DNA, 2.5 μM MgCl<sub>2</sub>, 2.5 μl 10× PCR buffer, 0.5 mM of each primer, 0.5 U Taq DNA polymerase, and 2.5 mM dNTPs. The PCR cycling profile was 94°C for 5 min, 35 cycles at 94°C for 45 s, 60°C for 45 s, 72°C for 45 s, and a final extension at 72°C for 10 min. Target sequences were recovered and sequenced. For new marker screening, the quality of the PCR product from 32 citrus species was checked by a 6% polyacrylamide sequencing gel containing 7 M urea in 0.5× TBE buffer. Three microliters of PCR product was mixed to an equal volume of loading buffer containing 95% formamide, 0.25% bromophenol blue, 0.25% xylen cyanol, and 10 mM EDTA. This mixture was heated for 5 min at 94°C to denature the DNA before loading. Gels were stained with silver nitrate following the protocol described<sup><a reveal-id="DSW017C27" data-open="DSW017C27" class="link link-ref link-reveal xref-bibr">27</a></sup> for gel electrophoresis analysis and for comparison with the 10-bp DNA standard ladders (Invitrogen).</p>                            <h3 scrollto-destination=87197453 id="87197453" class="section-title" data-legacy-id=s2h>2.8. Genetic diversity and data analysis</h3>
<p>POPgene (v1.32) was used to calculate the different statistical and genetic parameters,<sup><a reveal-id="DSW017C28" data-open="DSW017C28" class="link link-ref link-reveal xref-bibr">28</a></sup> such as the effective allele number,<sup><a reveal-id="DSW017C29" data-open="DSW017C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Shannon's information index,<sup><a reveal-id="DSW017C30" data-open="DSW017C30" class="link link-ref link-reveal xref-bibr">30</a></sup> and expected heterozygosity.<sup><a reveal-id="DSW017C31" data-open="DSW017C31" class="link link-ref link-reveal xref-bibr">31</a></sup> The genetic distance of the InDel genotype was calculated based on Nei's genetic distance measure, with pairwise distance calculated by MEGA 4.<sup><a reveal-id="DSW017C32" data-open="DSW017C32" class="link link-ref link-reveal xref-bibr">32</a></sup> The polymorphic information content (PIC) value for the InDel marker was calculated thereafter.<sup><a reveal-id="DSW017C33" data-open="DSW017C33" class="link link-ref link-reveal xref-bibr">33</a></sup> A dendrogram was constructed based on the unweighted pair-group method, with arithmetic mean determined by MEGA4.</p>                            <h2 scrollto-destination=87197455 id="87197455" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87197456 id="87197456" class="section-title" data-legacy-id=s3a>3.1. Genome sequencing and mapping onto the Clementine reference genome</h3>
<p>To identify the genome-wide genetic variation between precocious trifoliate orange and its WT, two resequencing libraries from the two genotypes of trifoliate orange were constructed. The genome size of the Clementine reference genome is 301.4 Mb.<sup><a reveal-id="DSW017C9" data-open="DSW017C9" class="link link-ref link-reveal xref-bibr">9</a></sup> We estimated that approximately 40-fold genome coverage should be sufficient for aligning most of the sequences. Therefore, 12.22 (143.78 million reads) and 13.16 Gb (159.56 million reads) of sequencing data were generated from the MT and the WT, respectively. Approximately 11.79 Gb (138.68 million clean reads) of filtered sequencing data for the MT and 12.64 Gb (153.23 million clean reads) of data for the WT were aligned to the Clementine genome using BWA software (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw017_Supplementary_Data.zip?Expires=2147483647&Signature=jOq1gW33zqr~sOP~x1PPMy7TWT-WE38Xi5v9CPg0rQt352NU0y9xhSjCw3WsXLLAtsK0jHDOQJxNEdZJ8jhPPjACEIohOU-hA3pzrL6uTBH6WIBYvQ8oVhrnSioL~584jG08OGDb~-gkgpObmdTHqZxFw7TMV9IHS2HvH027DpWDeNDJ2vAM0oKpco3dLlEx2TI1eoJA7J4IRbX1weIVLLuwo7rZasuqEKt08yl0IzQoaKVkpBnMh43yooTJkc-L5vb4rQUWbMWRf6a4vPo7xWgG~HzDTex4qGqvSpayOl3NU96NlSVyAVP0bG1~VX5H9F9Z~LooXz07P3ffj85Ryw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). The results showed that the genomic GC contents of the MT and the WT were 36.46 and 36.04%, respectively. A summary of the resequencing data is presented in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw017_Supplementary_Data.zip?Expires=2147483647&Signature=jOq1gW33zqr~sOP~x1PPMy7TWT-WE38Xi5v9CPg0rQt352NU0y9xhSjCw3WsXLLAtsK0jHDOQJxNEdZJ8jhPPjACEIohOU-hA3pzrL6uTBH6WIBYvQ8oVhrnSioL~584jG08OGDb~-gkgpObmdTHqZxFw7TMV9IHS2HvH027DpWDeNDJ2vAM0oKpco3dLlEx2TI1eoJA7J4IRbX1weIVLLuwo7rZasuqEKt08yl0IzQoaKVkpBnMh43yooTJkc-L5vb4rQUWbMWRf6a4vPo7xWgG~HzDTex4qGqvSpayOl3NU96NlSVyAVP0bG1~VX5H9F9Z~LooXz07P3ffj85Ryw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>. For the MT and the WT, 61.73 and 63.13% of clean reads were uniquely mapped to the reference genome and translated into 24- and 26-fold effective coverage of the reference sequences, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw017_Supplementary_Data.zip?Expires=2147483647&Signature=jOq1gW33zqr~sOP~x1PPMy7TWT-WE38Xi5v9CPg0rQt352NU0y9xhSjCw3WsXLLAtsK0jHDOQJxNEdZJ8jhPPjACEIohOU-hA3pzrL6uTBH6WIBYvQ8oVhrnSioL~584jG08OGDb~-gkgpObmdTHqZxFw7TMV9IHS2HvH027DpWDeNDJ2vAM0oKpco3dLlEx2TI1eoJA7J4IRbX1weIVLLuwo7rZasuqEKt08yl0IzQoaKVkpBnMh43yooTJkc-L5vb4rQUWbMWRf6a4vPo7xWgG~HzDTex4qGqvSpayOl3NU96NlSVyAVP0bG1~VX5H9F9Z~LooXz07P3ffj85Ryw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). These clean reads of the MT and the WT covered approximately 81.16 and 82.30% of the reference sequence, respectively, indicating that the generated data set was highly relevant to the reference genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw017_Supplementary_Data.zip?Expires=2147483647&Signature=jOq1gW33zqr~sOP~x1PPMy7TWT-WE38Xi5v9CPg0rQt352NU0y9xhSjCw3WsXLLAtsK0jHDOQJxNEdZJ8jhPPjACEIohOU-hA3pzrL6uTBH6WIBYvQ8oVhrnSioL~584jG08OGDb~-gkgpObmdTHqZxFw7TMV9IHS2HvH027DpWDeNDJ2vAM0oKpco3dLlEx2TI1eoJA7J4IRbX1weIVLLuwo7rZasuqEKt08yl0IzQoaKVkpBnMh43yooTJkc-L5vb4rQUWbMWRf6a4vPo7xWgG~HzDTex4qGqvSpayOl3NU96NlSVyAVP0bG1~VX5H9F9Z~LooXz07P3ffj85Ryw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Among citrus 33,929 transcripts, 6.95% and 7.01% reads of sequencing data were mapped 23,543 (69.38%) and 23,821 (70.21%) transcripts from the MT and the WT, respectively.</p>                            <h3 scrollto-destination=87197458 id="87197458" class="section-title" data-legacy-id=s3b>3.2. Detection and characteristics of genetic variations of the MT and the WT</h3>
<p>Genome-wide SNPs and InDels between the MT and the WT were identified by using SOAP on the basis of comparisons with the Clementine genome. In the MT and the WT, 3,237,862 and 3,422,013 SNPs were detected, respectively. Obviously, the number of detected SNPs in the latter comparison was more than that in the former because of the high fold coverage for the reference sequence. The SNP detected between two genomes and the reference genome was classified as transition (G/A and C/T) or transversion (A/C, C/G, G/T, and T/A) based on nucleotide substitution (Fig. <a reveal-id="DSW017F1" data-open="DSW017F1" class="link link-reveal link-table xref-fig">1</a>A). Both of the transition proportions were significantly higher than the transversion proportions in two genotypes (Fig. <a reveal-id="DSW017F1" data-open="DSW017F1" class="link link-reveal link-table xref-fig">1</a>A). Among these transitions, the G/A proportion was slightly more than that of C/T; among the transversions, the T/A proportion was the most and the G/C proportion was the least (Fig. <a reveal-id="DSW017F1" data-open="DSW017F1" class="link link-reveal link-table xref-fig">1</a>A). The transition-to-transversion ratios were 1.468 and 1.475 in the MT and the WT, respectively. </p>                        <a id="87197460" scrollto-destination="87197460"></a>
<div id="DSW017F1" data-id="DSW017F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/m_dsw01701.png?Expires=2147483647&Signature=isCanGv3UKhpxbZvWiYpJvHMAuz7WNEJsmKh0VTG8621QX9QZKNbqtimDEwqEvxYW16sVE7Wnku8SRP69DvXfdKtEzUZYuayTvyXx6TE5-ZYC1zUtSupYQJK2QMb1kkHSsMNy0gCI73u1v0ar~b8oJ4G9cy-SgvTAzqWFGJo43nvAg5d4vz0B-30bwjw49YAO5vRZDTqInaU7a8iFzSu4tHWZ6u~keuF85blrbys6zgxy9HsqOzuXUyVth7sociXks8dB80GvhEK-YsCXUE7Aq9EsrDg9IkLdRyV2-YW3kb3SVt79xn2R8-2sYIKOcxX1OubzGkL1vKiuC9ZXX-bmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Annotation analysis of SNPs, InDels, and SVs. (A) Frequency of different substitution types in the identified SNPs in MT/Clementine (precocious trifoliate orange compared with the C. clementina reference genome) and WT/Clementine (wild-type trifoliate orange compared with the C. clementina reference genome). (B) Length distribution of InDels in MT/Clementine and WT/Clementine. (C) The number of different types of SVs. DEL, deletion; INS, insertion; INV, inversion; ITX, translocation. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01701.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197460/dsw01701.png" path-from-xml="dsw01701.png" target="_blank">View large</a><a section="87197460" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw01701.png?Expires=2147483647&Signature=dB8IFPoSNcuD01CPT9EFYmQKn1XDTVg~o~TvacAvqpXbWzymm48YnZdbUi95jkhqoCU6AbOhhVP9swX2vo69bVEZC2Hjfq4urrINxavDtk2PSqjOP1nqKvxOLdl3fuToZtVw-TZOcwQUdrlLNgFZWFW8Q9w61GpD8tKKLRje-iw7KyklyUWBdFyRgurQX1~6nYH66~OiVsRzMRNKACF5wszN5p27A~NeBApXgZuamFgoVM~jwsDTzzkrICLmg0wHaAQ0vc9VN-jF6VuHSeXfdT9rE4JHJHQoKeKW-ZxvYueY2zr9B5BP4b-YUXLrBeG4uhvpbqOwoRQxH5tMe1ahpg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197460&ar=2469959&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01701.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Annotation analysis of SNPs, InDels, and SVs. (A) Frequency of different substitution types in the identified SNPs in MT/Clementine (precocious trifoliate orange compared with the <em>C. clementina</em> reference genome) and WT/Clementine (wild-type trifoliate orange compared with the <em>C. clementina</em> reference genome). (B) Length distribution of InDels in MT/Clementine and WT/Clementine. (C) The number of different types of SVs. DEL, deletion; INS, insertion; INV, inversion; ITX, translocation. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW017F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/m_dsw01701.png?Expires=2147483647&Signature=isCanGv3UKhpxbZvWiYpJvHMAuz7WNEJsmKh0VTG8621QX9QZKNbqtimDEwqEvxYW16sVE7Wnku8SRP69DvXfdKtEzUZYuayTvyXx6TE5-ZYC1zUtSupYQJK2QMb1kkHSsMNy0gCI73u1v0ar~b8oJ4G9cy-SgvTAzqWFGJo43nvAg5d4vz0B-30bwjw49YAO5vRZDTqInaU7a8iFzSu4tHWZ6u~keuF85blrbys6zgxy9HsqOzuXUyVth7sociXks8dB80GvhEK-YsCXUE7Aq9EsrDg9IkLdRyV2-YW3kb3SVt79xn2R8-2sYIKOcxX1OubzGkL1vKiuC9ZXX-bmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Annotation analysis of SNPs, InDels, and SVs. (A) Frequency of different substitution types in the identified SNPs in MT/Clementine (precocious trifoliate orange compared with the C. clementina reference genome) and WT/Clementine (wild-type trifoliate orange compared with the C. clementina reference genome). (B) Length distribution of InDels in MT/Clementine and WT/Clementine. (C) The number of different types of SVs. DEL, deletion; INS, insertion; INV, inversion; ITX, translocation. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01701.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197460/dsw01701.png" path-from-xml="dsw01701.png" target="_blank">View large</a><a section="87197460" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw01701.png?Expires=2147483647&Signature=dB8IFPoSNcuD01CPT9EFYmQKn1XDTVg~o~TvacAvqpXbWzymm48YnZdbUi95jkhqoCU6AbOhhVP9swX2vo69bVEZC2Hjfq4urrINxavDtk2PSqjOP1nqKvxOLdl3fuToZtVw-TZOcwQUdrlLNgFZWFW8Q9w61GpD8tKKLRje-iw7KyklyUWBdFyRgurQX1~6nYH66~OiVsRzMRNKACF5wszN5p27A~NeBApXgZuamFgoVM~jwsDTzzkrICLmg0wHaAQ0vc9VN-jF6VuHSeXfdT9rE4JHJHQoKeKW-ZxvYueY2zr9B5BP4b-YUXLrBeG4uhvpbqOwoRQxH5tMe1ahpg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197460&ar=2469959&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01701.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Annotation analysis of SNPs, InDels, and SVs. (A) Frequency of different substitution types in the identified SNPs in MT/Clementine (precocious trifoliate orange compared with the <em>C. clementina</em> reference genome) and WT/Clementine (wild-type trifoliate orange compared with the <em>C. clementina</em> reference genome). (B) Length distribution of InDels in MT/Clementine and WT/Clementine. (C) The number of different types of SVs. DEL, deletion; INS, insertion; INV, inversion; ITX, translocation. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>The MT and the WT yielded 867,854 and 895,643 InDels (including insertion/deletion), respectively. Among these InDels, 412,922 (47.58%) and 426,266 (47.59%) were insertions in the MT and the WT in comparison with the reference sequence, respectively (Fig. <a reveal-id="DSW017F1" data-open="DSW017F1" class="link link-reveal link-table xref-fig">1</a>B). The length of insertion ranged from 1 to 32 bp, whereas that of deletion was up to 46 bp between the MT and the reference (Fig. <a reveal-id="DSW017F1" data-open="DSW017F1" class="link link-reveal link-table xref-fig">1</a>B). More than half of the InDels (52.10%) were mononucleotides, 14.74% were dinucleotides, 19.98% were 3- to 6-bp nucleotides, 6.78% were 7- to 10-bp nucleotides, and 6.40% were nucleotides &gt;10 bp. Interestingly, the WT and the reference genome (Fig. <a reveal-id="DSW017F1" data-open="DSW017F1" class="link link-reveal link-table xref-fig">1</a>B) showed the same trend. Meanwhile, 27,257 and 29,066 SVs were also found in the MT and the WT compared with the citrus reference genome, respectively. A comparative analysis showed that a total of 52,135 were identified between the MT and the WT; 4,188 were common and 47,947 were specific to the two genotypes. The types of SVs were deletion (75.86%), insertion (17.24%), translocation (5.38%), and inversion (1.52%) in the MT (Fig. <a reveal-id="DSW017F1" data-open="DSW017F1" class="link link-reveal link-table xref-fig">1</a>C). In WT, the distribution trends were similar to those of MT. We also detected 16,487 SVs in two genotypes; 1,168 were present in both the MT and the WT, 7,600 were specific to the MT, and the other 7,719 were specific to the WT. The homozygous and heterozygous ratios were 4.12:1 for InDels and 2.22:1 for SVs in the MT, respectively. For the WT, the homozygous and heterozygous ratios were 4.00:1 and 2.12:1, respectively. The average densities of detected genetic variations between the two genotype genomes and the Clementine reference genome were 10,038.5 and 10,521.5 SNPs/Mb, 2,940.2 and 3,024.3 InDels/Mb, and 92.3 and 98.5 SVs/Mb in the MT and the WT, respectively.</p>                            <h3 scrollto-destination=87197462 id="87197462" class="section-title" data-legacy-id=s3c>3.3. Analysis of genetic variation differences between the MT and the WT</h3>
<p>We compared the SNPs between the MT and the WT based on Clementine reference genome, and the result showed that a total of 3,932,628 were identified between the MT and the WT, 2,727,248 were common and 1,117,089 were specific to the two genotypes at the genome level. To further evaluate the differential SNPs between the MT and the WT in the genic region, the data from the two genotypes were combined, and we obtained the following: a total of 1,897,842 SNPs located in the genic region; 1,589,456 were found in both the MT and the WT, 141,195 were specific to the MT distributed in 20,023 transcripts, and 167,182 were specific to the WT distributed in 21,684 transcripts. Among these genic SNPs, there were 1,422,343 homozygous and 475,499 hemizygous SNPs. The positions of specific SNP were identified in CDS, intron, 5′ UTR, and 3′ UTR regions according to the reference genome. Among 308,377 specific SNPs, 51,589 (MT: 23,169 and WT: 28,419) were located in UTR regions, 169,373 (MT: 76,885 and WT: 92,488) in intron regions, and 87,416 (MT: 41,141 and WT: 46,275) in CDS regions. Non-synonymous coding SNPs are believed to have the higher impact on phenotype.<sup><a reveal-id="DSW017C34" data-open="DSW017C34" class="link link-ref link-reveal xref-bibr">34</a></sup> We therefore paid attention to specific non-synonymous SNPs in CDS regions in two genotypes. The number of non-synonymous SNPs was 25,769 (29.14%) and 29,018 (27.68%) in CDS regions of the MT and the WT, respectively. Such information will greatly enhance our understanding of the flowering process in the MT, as these are more likely to have phenotypic effects.</p><p>A comparative analysis showed that a total of 1,293,383 InDels were identified between the MT and the WT; 649,514 were common and 397,741 were specific to the two genotypes. When increasing or decreasing nucleotides in a coding sequence, it's possible to produce a frameshift mutation followed by character variation.<sup><a reveal-id="DSW017C35" data-open="DSW017C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Therefore, the InDels of genic regions were also identified in this study. After being carefully filtered, 429,301 InDels were obtained in genic regions in the two genotypes; 267,756 were found in both the MT and the WT distributed in 23,488 transcripts, 63,119 were specific to the MT distributed in 10,092 transcripts, and 98,426 were specific to the WT distributed in 10,518 transcripts. Among 161,545 specific InDels, 37,720 (MT: 14,945 and WT: 22,775) were located in UTR regions, 114,118 (MT: 44,578 and WT: 69,540) in intron regions, and 9,707 (MT: 3,596 and WT: 6,111) in CDS regions. Interestingly, the number of specific genetic variations (including SNPs and InDels) from the MT is less than the MT in this study. Within the gene body, the maximum number of specific SNPs (MT: 54.45% and WT: 55.30%) and InDels (MT: 70.62% and WT: 70.65%) was present in the introns of two genotypes. Only a small part of specific SNPs (MT: 29.14% and WT: 27.68%) and InDels (MT: 5.69% and WT: 6.21%) occurred in the CDS region.</p>                            <h3 scrollto-destination=87197465 id="87197465" class="section-title" data-legacy-id=s3d>3.4. Transcriptome analysis of flowering-related genetic variation in two genotypes</h3>
<p>To reduce the identification of false-positive genetic variations at the transcriptional level, an integrative analysis of RNA-Seq and genome resequencing was performed for the two genotypes at the flowering transition stage (see Section 2.1). For the MT and the WT, 3.94 and 3.68 Gb of clean data were obtained, respectively. A total of 80.88% and 80.36% successful reads were perfectly matched to the Clementine genome for the MT and the WT, respectively. A summary of the resequencing data is presented in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw017_Supplementary_Data.zip?Expires=2147483647&Signature=jOq1gW33zqr~sOP~x1PPMy7TWT-WE38Xi5v9CPg0rQt352NU0y9xhSjCw3WsXLLAtsK0jHDOQJxNEdZJ8jhPPjACEIohOU-hA3pzrL6uTBH6WIBYvQ8oVhrnSioL~584jG08OGDb~-gkgpObmdTHqZxFw7TMV9IHS2HvH027DpWDeNDJ2vAM0oKpco3dLlEx2TI1eoJA7J4IRbX1weIVLLuwo7rZasuqEKt08yl0IzQoaKVkpBnMh43yooTJkc-L5vb4rQUWbMWRf6a4vPo7xWgG~HzDTex4qGqvSpayOl3NU96NlSVyAVP0bG1~VX5H9F9Z~LooXz07P3ffj85Ryw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>. A total of 195,495 and 180,449 SNPs were identified in the MT and the WT distributed in 24,849 and 24,328 transcripts, respectively. Finally, a total of 233,998 SNPs were identified between the MT and the WT based on integrative analysis of RNA-Seq and genome resequencing. Of these, 53,549 were only present in the MT distributed in 20,366 transcripts, and 38,503 were only observed in the WT distributed in 18,388 transcripts (Fig. <a reveal-id="DSW017F2" data-open="DSW017F2" class="link link-reveal link-table xref-fig">2</a>A). There were 202,222 homozygous SNPs and 95,986 synonymous SNPs among the total SNPs (Fig. <a reveal-id="DSW017F2" data-open="DSW017F2" class="link link-reveal link-table xref-fig">2</a>B and C). We also found 333 SNPs that might lead to premature starting or stopping of transcription (220 starts gained and 113 stops gained), 7 SNPs related to intron splicing, and 22 mutations in start/stop codons (12 in start codons and 10 in stop codons) between the MT and the WT. Compared with the Clementine genome, 75,836 InDels were uncovered in the two genotypes; 18,264 were only present in the MT distributed in 10,996 transcripts, and 12,266 were only observed in the WT distributed in 8,164 transcripts (Fig. <a reveal-id="DSW017F2" data-open="DSW017F2" class="link link-reveal link-table xref-fig">2</a>D). </p>                        <a id="87197467" scrollto-destination="87197467"></a>
<div id="DSW017F2" data-id="DSW017F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/m_dsw01702.png?Expires=2147483647&Signature=Z5wDkUnjQVjsgtxLbVneBJ4FXzwTdmCj78k1y0KKbRCfeocmO~mqKSw1kmpFKZD2X4CPqljDDJCxHSvHPTRuwMaV3ysUWnTbgfYIU5NBExLm~jpC6i-ea9Bx1oZ75NWMnHJQBRFH8QpzI6DYm19vOP7pwZadcbs~bJb5lp0h2uYHthQpfNe6RcBmZx0eXrp5CI0gbrTcYVUEYUYY1PJEVH-~O05L2R~EeHctWY~Gdu2OdAETGMB-zBoI0O-hgSqcDVmpgAC~WBB8R9TaJ5WG2G1CY51dUZ5I5w1bEI8Y4DrepsXATWFzlo9JTXS0VA42BdVf~xDYz-x8scwoxpzmwg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="SNPs and InDels from transcriptional level distribution. (A) WT-specific SNPs (these SNPs are present only in the wild-type trifoliate orange genome) and MT-specific SNPs (these SNPs are present only in precocious trifoliate orange genome). The remaining SNPs are present in both genotypes. (B) Numbers of homozygous and heterozygous SNPs. (C) Numbers of synonymous and non-synonymous SNPs. (D) WT-specific InDels (these InDels are present only in the wild-type trifoliate orange genome) and MT-specific InDels (these InDels are present only in precocious trifoliate orange genome). The remaining InDels are present in both genotypes. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01702.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197467/dsw01702.png" path-from-xml="dsw01702.png" target="_blank">View large</a><a section="87197467" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw01702.png?Expires=2147483647&Signature=uOxOzmPusb~H5KL4IaAnlqoY3IvAFJkj2KDeniARYJPongJhKi~PI8jf71zeA9IpgTfKgbKv1G9ahgKGncFzsTn3qm1luB2XKOjhkVfONm7mxcBQ-XLZ9SOT2y5jm6LItp5v50Ooi~kwk9AJr6C9lCSYDRj-0U78Zihk9VGLbSPoE-VWUJosxAaXWmsVyKc4GaoPRxHlKMo5qX2HrtQyqRQukaq0nF1eE-Nbub3-wWQXfdRIVRCgfUQQ0yda5Q4r~u1RRQ2XzDXhXXgHnsnKq7pljilxf3oBxZrWnHpaBeRMruEAVwaGD47Ndec9qWRi45s3vo0sq4oFq77-Muq-Xw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197467&ar=2469959&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01702.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>SNPs and InDels from transcriptional level distribution. (A) WT-specific SNPs (these SNPs are present only in the wild-type trifoliate orange genome) and MT-specific SNPs (these SNPs are present only in precocious trifoliate orange genome). The remaining SNPs are present in both genotypes. (B) Numbers of homozygous and heterozygous SNPs. (C) Numbers of synonymous and non-synonymous SNPs. (D) WT-specific InDels (these InDels are present only in the wild-type trifoliate orange genome) and MT-specific InDels (these InDels are present only in precocious trifoliate orange genome). The remaining InDels are present in both genotypes. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW017F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/m_dsw01702.png?Expires=2147483647&Signature=Z5wDkUnjQVjsgtxLbVneBJ4FXzwTdmCj78k1y0KKbRCfeocmO~mqKSw1kmpFKZD2X4CPqljDDJCxHSvHPTRuwMaV3ysUWnTbgfYIU5NBExLm~jpC6i-ea9Bx1oZ75NWMnHJQBRFH8QpzI6DYm19vOP7pwZadcbs~bJb5lp0h2uYHthQpfNe6RcBmZx0eXrp5CI0gbrTcYVUEYUYY1PJEVH-~O05L2R~EeHctWY~Gdu2OdAETGMB-zBoI0O-hgSqcDVmpgAC~WBB8R9TaJ5WG2G1CY51dUZ5I5w1bEI8Y4DrepsXATWFzlo9JTXS0VA42BdVf~xDYz-x8scwoxpzmwg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="SNPs and InDels from transcriptional level distribution. (A) WT-specific SNPs (these SNPs are present only in the wild-type trifoliate orange genome) and MT-specific SNPs (these SNPs are present only in precocious trifoliate orange genome). The remaining SNPs are present in both genotypes. (B) Numbers of homozygous and heterozygous SNPs. (C) Numbers of synonymous and non-synonymous SNPs. (D) WT-specific InDels (these InDels are present only in the wild-type trifoliate orange genome) and MT-specific InDels (these InDels are present only in precocious trifoliate orange genome). The remaining InDels are present in both genotypes. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01702.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197467/dsw01702.png" path-from-xml="dsw01702.png" target="_blank">View large</a><a section="87197467" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw01702.png?Expires=2147483647&Signature=uOxOzmPusb~H5KL4IaAnlqoY3IvAFJkj2KDeniARYJPongJhKi~PI8jf71zeA9IpgTfKgbKv1G9ahgKGncFzsTn3qm1luB2XKOjhkVfONm7mxcBQ-XLZ9SOT2y5jm6LItp5v50Ooi~kwk9AJr6C9lCSYDRj-0U78Zihk9VGLbSPoE-VWUJosxAaXWmsVyKc4GaoPRxHlKMo5qX2HrtQyqRQukaq0nF1eE-Nbub3-wWQXfdRIVRCgfUQQ0yda5Q4r~u1RRQ2XzDXhXXgHnsnKq7pljilxf3oBxZrWnHpaBeRMruEAVwaGD47Ndec9qWRi45s3vo0sq4oFq77-Muq-Xw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197467&ar=2469959&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01702.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>SNPs and InDels from transcriptional level distribution. (A) WT-specific SNPs (these SNPs are present only in the wild-type trifoliate orange genome) and MT-specific SNPs (these SNPs are present only in precocious trifoliate orange genome). The remaining SNPs are present in both genotypes. (B) Numbers of homozygous and heterozygous SNPs. (C) Numbers of synonymous and non-synonymous SNPs. (D) WT-specific InDels (these InDels are present only in the wild-type trifoliate orange genome) and MT-specific InDels (these InDels are present only in precocious trifoliate orange genome). The remaining InDels are present in both genotypes. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>For the MT, annotation analysis showed that 47,130 SNPs and 20,791 InDels were located in the intron regions based on the annotation of the reference gene models during the phase change stage. Meanwhile, 40,416 SNPs and 19,011 InDels were also located in the intron regions in the WT. These results indicated that alternative splicing and novel transcriptional events may have occurred during the gene expression process of the two genotypes. In the MT and the WT, InDels with a length of 1 bp accounted for &gt;50% of the whole genes. Most of the InDels in the coding sequence (CDS) regions were trinucleotides or hexanucleotides, which could not have been caused by frameshifts. However, mononucleotides were always the most common nucleotides in intergenic regions. Despite the minimally abundant distribution within critical sites, such as the CDS region and untranslated region (UTR) (12.1% of total InDels), these InDels can alter phenotypes through a variety of mechanisms.</p>                            <h3 scrollto-destination=87197469 id="87197469" class="section-title" data-legacy-id=s3e>3.5. Identification of genetic variation in flowering-related genes</h3>
<p>A total of 244 transcripts representing putative homologs to flowering-related genes were identified in Clementine genome (<a class="link link-uri" href="http://www.phytozome.net/clementine.php">http://www.phytozome.net/clementine.php</a>) by BLAST searches against the <em>Arabidopsis</em> Information Resource (TAIR) and NCBI data sets, and a total of 178 read-mapped flowering-related genes were observed in two genotypes in this study (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw017_Supplementary_Data.zip?Expires=2147483647&Signature=jOq1gW33zqr~sOP~x1PPMy7TWT-WE38Xi5v9CPg0rQt352NU0y9xhSjCw3WsXLLAtsK0jHDOQJxNEdZJ8jhPPjACEIohOU-hA3pzrL6uTBH6WIBYvQ8oVhrnSioL~584jG08OGDb~-gkgpObmdTHqZxFw7TMV9IHS2HvH027DpWDeNDJ2vAM0oKpco3dLlEx2TI1eoJA7J4IRbX1weIVLLuwo7rZasuqEKt08yl0IzQoaKVkpBnMh43yooTJkc-L5vb4rQUWbMWRf6a4vPo7xWgG~HzDTex4qGqvSpayOl3NU96NlSVyAVP0bG1~VX5H9F9Z~LooXz07P3ffj85Ryw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Annotation analysis showed that 1,073 SNPs, 382 InDels, and 50 SVs were located in 147, 97, and 29 transcripts (including UTR, CDS region, and intron region) related to flower induction, flower development, and flowering time, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw017_Supplementary_Data.zip?Expires=2147483647&Signature=jOq1gW33zqr~sOP~x1PPMy7TWT-WE38Xi5v9CPg0rQt352NU0y9xhSjCw3WsXLLAtsK0jHDOQJxNEdZJ8jhPPjACEIohOU-hA3pzrL6uTBH6WIBYvQ8oVhrnSioL~584jG08OGDb~-gkgpObmdTHqZxFw7TMV9IHS2HvH027DpWDeNDJ2vAM0oKpco3dLlEx2TI1eoJA7J4IRbX1weIVLLuwo7rZasuqEKt08yl0IzQoaKVkpBnMh43yooTJkc-L5vb4rQUWbMWRf6a4vPo7xWgG~HzDTex4qGqvSpayOl3NU96NlSVyAVP0bG1~VX5H9F9Z~LooXz07P3ffj85Ryw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). These genetic variations may directly represent functional changes in the gene products. Some of these genes were required for the photoperiod pathway and some encode regulatory proteins specifically involved in the control of flowering by vernalization pathway, whereas others encoded components of ageing pathways or were involved in the circadian clock function (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw017_Supplementary_Data.zip?Expires=2147483647&Signature=jOq1gW33zqr~sOP~x1PPMy7TWT-WE38Xi5v9CPg0rQt352NU0y9xhSjCw3WsXLLAtsK0jHDOQJxNEdZJ8jhPPjACEIohOU-hA3pzrL6uTBH6WIBYvQ8oVhrnSioL~584jG08OGDb~-gkgpObmdTHqZxFw7TMV9IHS2HvH027DpWDeNDJ2vAM0oKpco3dLlEx2TI1eoJA7J4IRbX1weIVLLuwo7rZasuqEKt08yl0IzQoaKVkpBnMh43yooTJkc-L5vb4rQUWbMWRf6a4vPo7xWgG~HzDTex4qGqvSpayOl3NU96NlSVyAVP0bG1~VX5H9F9Z~LooXz07P3ffj85Ryw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). These included <em>AGAMOUS</em>, <em>APETALA</em>, and <em>SEPALLATA</em> genes for flower development; <em>FRIGIDA</em>, <em>FRIGIDA INTERACTING PROTEIN1</em> gene, and <em>VERNALIZATION3/5-like</em> for the vernalization pathway; and <em>SQUAMOSA promoter binding protein–like</em> gene, GIGANTEA protein, and flowering-time control protein-related/FCA gamma-related for age and circadian clock developmental processes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw017_Supplementary_Data.zip?Expires=2147483647&Signature=jOq1gW33zqr~sOP~x1PPMy7TWT-WE38Xi5v9CPg0rQt352NU0y9xhSjCw3WsXLLAtsK0jHDOQJxNEdZJ8jhPPjACEIohOU-hA3pzrL6uTBH6WIBYvQ8oVhrnSioL~584jG08OGDb~-gkgpObmdTHqZxFw7TMV9IHS2HvH027DpWDeNDJ2vAM0oKpco3dLlEx2TI1eoJA7J4IRbX1weIVLLuwo7rZasuqEKt08yl0IzQoaKVkpBnMh43yooTJkc-L5vb4rQUWbMWRf6a4vPo7xWgG~HzDTex4qGqvSpayOl3NU96NlSVyAVP0bG1~VX5H9F9Z~LooXz07P3ffj85Ryw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Moreover, some additional flowering-time regulators containing genetic variations that have not been placed in any specific pathway were also identified, such as <em>EMBRYONIC FLOWER1</em> (<em>EMF1</em>), <em>RELATIVE OF EARLY FLOWERING 6</em>, <em>FLOWERING PROMOTING FACTOR 1</em>, and <em>RECEPTOR-LIKE KINASE IN FLOWERS 1</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw017_Supplementary_Data.zip?Expires=2147483647&Signature=jOq1gW33zqr~sOP~x1PPMy7TWT-WE38Xi5v9CPg0rQt352NU0y9xhSjCw3WsXLLAtsK0jHDOQJxNEdZJ8jhPPjACEIohOU-hA3pzrL6uTBH6WIBYvQ8oVhrnSioL~584jG08OGDb~-gkgpObmdTHqZxFw7TMV9IHS2HvH027DpWDeNDJ2vAM0oKpco3dLlEx2TI1eoJA7J4IRbX1weIVLLuwo7rZasuqEKt08yl0IzQoaKVkpBnMh43yooTJkc-L5vb4rQUWbMWRf6a4vPo7xWgG~HzDTex4qGqvSpayOl3NU96NlSVyAVP0bG1~VX5H9F9Z~LooXz07P3ffj85Ryw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). We also found several genes encoding putative photoreceptor proteins, including phytochrome, phytochrome interacting factor, phytochrome and flowering-time regulatory protein, phytochrome-associated protein, and cryptochrome. These variations may provide important genetic variations to explain the phenotypic differences between the MT and the WT (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw017_Supplementary_Data.zip?Expires=2147483647&Signature=jOq1gW33zqr~sOP~x1PPMy7TWT-WE38Xi5v9CPg0rQt352NU0y9xhSjCw3WsXLLAtsK0jHDOQJxNEdZJ8jhPPjACEIohOU-hA3pzrL6uTBH6WIBYvQ8oVhrnSioL~584jG08OGDb~-gkgpObmdTHqZxFw7TMV9IHS2HvH027DpWDeNDJ2vAM0oKpco3dLlEx2TI1eoJA7J4IRbX1weIVLLuwo7rZasuqEKt08yl0IzQoaKVkpBnMh43yooTJkc-L5vb4rQUWbMWRf6a4vPo7xWgG~HzDTex4qGqvSpayOl3NU96NlSVyAVP0bG1~VX5H9F9Z~LooXz07P3ffj85Ryw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>).</p>                            <h3 scrollto-destination=87197471 id="87197471" class="section-title" data-legacy-id=s3f>3.6. Functional clustering and expression analysis of the transcripts containing specific InDels from the flowering phase change stage</h3>
<p>InDels that occur in functionally important regions of genes (typically CDS region) could be seen affecting gene function by frameshifts and structural changes of protein. Annotation analysis showed that the MT- and WT-specific InDels from the phase change stage were located in the CDS region of 1,137 and 721 transcripts, respectively. To explore and summarize the functional categories of these transcripts, we used Blast2GO to obtain the GO terms for the representation of molecular function, cellular component, and biological process. Approximately 902 (79.4%) and 588 (80.5%) of the MT and the WT transcripts had BLAST hits, respectively. Fig. <a reveal-id="DSW017F3" data-open="DSW017F3" class="link link-reveal link-table xref-fig">3</a> shows the transcript distribution for three main categories. Based on molecular function, these genes were finally classified into 12 categories; the three most over-represented GO terms were binding, hydrolase activity, and transporter activity (Fig. <a reveal-id="DSW017F3" data-open="DSW017F3" class="link link-reveal link-table xref-fig">3</a>A). Categories based on biological processes revealed that the mutant genes were related to 14 biological processes (Fig. <a reveal-id="DSW017F3" data-open="DSW017F3" class="link link-reveal link-table xref-fig">3</a>B); the three most frequent terms were metabolic process, cellular process, and single-organism process, suggesting that these mutative genes were involved in a broad range of physiological functions (Fig. <a reveal-id="DSW017F3" data-open="DSW017F3" class="link link-reveal link-table xref-fig">3</a>B). As shown in Fig. <a reveal-id="DSW017F3" data-open="DSW017F3" class="link link-reveal link-table xref-fig">3</a>C in cellular component category, the analysis revealed a high percentage of cells and organelles in cellular components. The biological interpretation of these transcripts was further examined using KEGG pathway analysis. In the MT and the WT, 84 and 71 different pathways were found, respectively. Those pathways mainly correlated with development involved with metabolism, hormone signal transduction, and transcriptional regulation, suggesting that some mutative genes may affect the distinguishing traits of the MT and the WT. </p>                        <a id="87197473" scrollto-destination="87197473"></a>
<div id="DSW017F3" data-id="DSW017F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/m_dsw01703.png?Expires=2147483647&Signature=1psqeGolDFuSxY3kA2OrAIUTN6-yM~XsQSmjOH~~xzBRdikBEq71yc8196dlHEEI1cSAt6TwJ6fLwM9tAL4Y8EImCxXjMFgKwgNA4zYtxH3NCf-xYRXk3iMvisUz0yhpRtpJNSRDKMNh8nE9WM73duOHwUkmPVCJnNXNnsAZZdaRVXbR~KyIQnmnwD034o3NvHKz4O5av0fLcX8jAtlpdC~MyPWpnJYjNvD9~XmapVGj~gCeRWzPBEtupuzH8ZJl8eK-XL3Z5k0ELpG8HFOXCGRnmRu7vDA221RGcWKOE9obPmT-tl9vpcgJzz5FugyR2g9di5LyBtCBfBXaq3w0tw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional categorization of the transcripts containing InDels. These genes were categorized based on GO annotation, and the proportion of each category is displayed based on (A) cellular component, (B) molecular function, or (C) biological process. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01703.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197473/dsw01703.png" path-from-xml="dsw01703.png" target="_blank">View large</a><a section="87197473" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw01703.png?Expires=2147483647&Signature=qcdYePoeoidutti-wIGJq~0cpUJjtB5ZYMI6gHMPFRZaj41bBf-8i-6TkFwYhZ-chNOeewAK8eJ1lZR6fndQ8U394W1XL6cZBuEtRlV9vcFiwUc8EfImjzg6a9Z-GCmhWiIUsY5glANOWV7fDIpTuPL3r0N0PY~n8vleTDg9Tpeaisz3YkIm-~cy~HaQyAcyzfbmiUO2z-9LFko5t9uk~U5QkFovthnts5kYTqV15u-~n9tWzt1SPpoDvdC6RT6BckX7CvKaF9U1RfPicblCF75MQMHyNWVaAwSQ07Mzpjkf2QGSfDsxzDWhPblLfIdKjG7DEc~FKzF6nZq~pJesjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197473&ar=2469959&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01703.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional categorization of the transcripts containing InDels. These genes were categorized based on GO annotation, and the proportion of each category is displayed based on (A) cellular component, (B) molecular function, or (C) biological process. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW017F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/m_dsw01703.png?Expires=2147483647&Signature=1psqeGolDFuSxY3kA2OrAIUTN6-yM~XsQSmjOH~~xzBRdikBEq71yc8196dlHEEI1cSAt6TwJ6fLwM9tAL4Y8EImCxXjMFgKwgNA4zYtxH3NCf-xYRXk3iMvisUz0yhpRtpJNSRDKMNh8nE9WM73duOHwUkmPVCJnNXNnsAZZdaRVXbR~KyIQnmnwD034o3NvHKz4O5av0fLcX8jAtlpdC~MyPWpnJYjNvD9~XmapVGj~gCeRWzPBEtupuzH8ZJl8eK-XL3Z5k0ELpG8HFOXCGRnmRu7vDA221RGcWKOE9obPmT-tl9vpcgJzz5FugyR2g9di5LyBtCBfBXaq3w0tw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional categorization of the transcripts containing InDels. These genes were categorized based on GO annotation, and the proportion of each category is displayed based on (A) cellular component, (B) molecular function, or (C) biological process. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01703.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197473/dsw01703.png" path-from-xml="dsw01703.png" target="_blank">View large</a><a section="87197473" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw01703.png?Expires=2147483647&Signature=qcdYePoeoidutti-wIGJq~0cpUJjtB5ZYMI6gHMPFRZaj41bBf-8i-6TkFwYhZ-chNOeewAK8eJ1lZR6fndQ8U394W1XL6cZBuEtRlV9vcFiwUc8EfImjzg6a9Z-GCmhWiIUsY5glANOWV7fDIpTuPL3r0N0PY~n8vleTDg9Tpeaisz3YkIm-~cy~HaQyAcyzfbmiUO2z-9LFko5t9uk~U5QkFovthnts5kYTqV15u-~n9tWzt1SPpoDvdC6RT6BckX7CvKaF9U1RfPicblCF75MQMHyNWVaAwSQ07Mzpjkf2QGSfDsxzDWhPblLfIdKjG7DEc~FKzF6nZq~pJesjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197473&ar=2469959&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01703.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional categorization of the transcripts containing InDels. These genes were categorized based on GO annotation, and the proportion of each category is displayed based on (A) cellular component, (B) molecular function, or (C) biological process. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>Using all of our sequence reads, the expression levels of 1,858 transcripts containing specific InDels were estimated during the phase change stage. There were 131 transcripts that were differently expressed between the MT and the WT, with <em>P</em> ≤ 0.005 and the absolute value of |log<sub>2</sub> ratio| ≥ 0.5 used as the threshold. Of these, 97 were more abundant and 34 were less abundant in the MT compared with the WT, suggesting that many genes were enriched during the flowering transition (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw017_Supplementary_Data.zip?Expires=2147483647&Signature=jOq1gW33zqr~sOP~x1PPMy7TWT-WE38Xi5v9CPg0rQt352NU0y9xhSjCw3WsXLLAtsK0jHDOQJxNEdZJ8jhPPjACEIohOU-hA3pzrL6uTBH6WIBYvQ8oVhrnSioL~584jG08OGDb~-gkgpObmdTHqZxFw7TMV9IHS2HvH027DpWDeNDJ2vAM0oKpco3dLlEx2TI1eoJA7J4IRbX1weIVLLuwo7rZasuqEKt08yl0IzQoaKVkpBnMh43yooTJkc-L5vb4rQUWbMWRf6a4vPo7xWgG~HzDTex4qGqvSpayOl3NU96NlSVyAVP0bG1~VX5H9F9Z~LooXz07P3ffj85Ryw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). BLAST searches of the 131 transcripts showed that 102 transcripts (77.9%) were homologous to known genes, 9 transcripts (6.9%) were homologous to genes of unknown function, and 20 clones (15.3%) had no matches in the database. Among 102 homologous to known genes, many had high identity with known transcription and post-transcriptional regulatory genes, indicating that these genes may be key regulators controlling flower development by activating or repressing numerous genes. Moreover, some transcription factors, including MYB, MADS, and Zinc finger, were observed (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw017_Supplementary_Data.zip?Expires=2147483647&Signature=jOq1gW33zqr~sOP~x1PPMy7TWT-WE38Xi5v9CPg0rQt352NU0y9xhSjCw3WsXLLAtsK0jHDOQJxNEdZJ8jhPPjACEIohOU-hA3pzrL6uTBH6WIBYvQ8oVhrnSioL~584jG08OGDb~-gkgpObmdTHqZxFw7TMV9IHS2HvH027DpWDeNDJ2vAM0oKpco3dLlEx2TI1eoJA7J4IRbX1weIVLLuwo7rZasuqEKt08yl0IzQoaKVkpBnMh43yooTJkc-L5vb4rQUWbMWRf6a4vPo7xWgG~HzDTex4qGqvSpayOl3NU96NlSVyAVP0bG1~VX5H9F9Z~LooXz07P3ffj85Ryw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). In addition to transcription factors, differently expressed genes involving chromatin remodeling, hormone regulation, and other metabolic pathways were also observed.</p>                            <h3 scrollto-destination=87197475 id="87197475" class="section-title" data-legacy-id=s3g>3.7. Development of new DNA markers in citrus</h3>
<p>To test cross-species/genera transferability, 268 newly identified InDel markers were tested on a panel of 32 diverse citrus accessions (Fig. <a reveal-id="DSW017F4" data-open="DSW017F4" class="link link-reveal link-table xref-fig">4</a>). Of these primer pairs, 35 primer pairs cannot amplify any fragment, suggesting that these primers were not well designed; 72 pairs of primers amplify numerous non-target bands, suggesting that these primers were also problematic; and 87 pairs of primers did not exhibit any difference. Only 74 InDels were confirmed. The results indicated that a high level of genetic diversity in the vicinity of these InDels was discovered in 32 genotypes. The number of alleles per locus ranges from 2 to 10, with an average of 3.92 alleles. Ne ranges from 1.06 to 7.04, with an average of 2.34. Ho ranges from 0.0 to 0.97, with an average of 0.34. He ranges from 0.06 to 0.87, with an average of 0.51. PIC ranged from 0.06 for the Ciclev10029752m locus to 0.48 for the Ciclev10010584m locus (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw017_Supplementary_Data.zip?Expires=2147483647&Signature=jOq1gW33zqr~sOP~x1PPMy7TWT-WE38Xi5v9CPg0rQt352NU0y9xhSjCw3WsXLLAtsK0jHDOQJxNEdZJ8jhPPjACEIohOU-hA3pzrL6uTBH6WIBYvQ8oVhrnSioL~584jG08OGDb~-gkgpObmdTHqZxFw7TMV9IHS2HvH027DpWDeNDJ2vAM0oKpco3dLlEx2TI1eoJA7J4IRbX1weIVLLuwo7rZasuqEKt08yl0IzQoaKVkpBnMh43yooTJkc-L5vb4rQUWbMWRf6a4vPo7xWgG~HzDTex4qGqvSpayOl3NU96NlSVyAVP0bG1~VX5H9F9Z~LooXz07P3ffj85Ryw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). The mean PIC value for citrus genera was 0.45 (range, 0.40–0.47), which was higher than 0.43 and 0.44 observed for trifoliate orange genera (range, 0.43–0.48) and fortunella genera, respectively (Table <a reveal-id="DSW017TB1" data-open="DSW017TB1" class="link link-reveal link-table xref-fig">1</a>). The aforementioned InDel allelic data, when used to calculate the heterozygosity estimates, revealed highly significant differences between the observed and expected heterozygosity for citrus (mean Ho: 0.35; mean He: 0.43) and fortunella genera (mean Ho: 0.22; mean He: 0.23). Thus, the results suggest significant heterozygote deficiency among the three genera of Rutaceae (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw017_Supplementary_Data.zip?Expires=2147483647&Signature=jOq1gW33zqr~sOP~x1PPMy7TWT-WE38Xi5v9CPg0rQt352NU0y9xhSjCw3WsXLLAtsK0jHDOQJxNEdZJ8jhPPjACEIohOU-hA3pzrL6uTBH6WIBYvQ8oVhrnSioL~584jG08OGDb~-gkgpObmdTHqZxFw7TMV9IHS2HvH027DpWDeNDJ2vAM0oKpco3dLlEx2TI1eoJA7J4IRbX1weIVLLuwo7rZasuqEKt08yl0IzQoaKVkpBnMh43yooTJkc-L5vb4rQUWbMWRf6a4vPo7xWgG~HzDTex4qGqvSpayOl3NU96NlSVyAVP0bG1~VX5H9F9Z~LooXz07P3ffj85Ryw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). In addition, it is noteworthy that some band patterns from the above markers were almost the same between the MT and flying dragon trifoliate orange (Fig. <a reveal-id="DSW017F4" data-open="DSW017F4" class="link link-reveal link-table xref-fig">4</a>). However, precocious trifoliate orange cannot be derived from flying dragon trifoliate orange based on their huge morphological differences and previous studies.<sup><a reveal-id="DSW017C10" data-open="DSW017C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSW017C12" data-open="DSW017C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSW017C13" data-open="DSW017C13" class="link link-ref link-reveal xref-bibr">13</a></sup> There might be one possible explanation. Because precocious trifoliate orange and its WT have nearly the same morphology other than flowering regulation, these InDel markers from the MT and the WT may be related to flowering. Furthermore, flying dragon trifoliate orange shows a certain degree of early flowering compared with the mutant's WT. </p>                        <a id="87197477" scrollto-destination="87197477"></a>
<div content-id="DSW017TB1" class="table-modal"><div class="table-wrap"><div id="DSW017TB1" class="table-wrap-title" data-id="DSW017TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Cross-species transferability of new InDel marker among different genomes in three genera of Rutaceae</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Genus </th><th>Common name </th><th>Latin name </th><th>Ploidy </th><th>Cultivated/wild </th><th>No. of InDel </th><th>% of amplified </th><th>Absent of amplified </th><th>PIC </th></tr></thead><tbody><tr><td>Fortunella Swingle </td><td>Fortunella </td><td><em>F. japonica</em> </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.43 </td></tr><tr><td>Fortunella Swingle </td><td>Kumquat </td><td><em>F. hindsii</em> var<em>. chintou</em> Swing </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.43 </td></tr><tr><td>Trifoliate orange </td><td>Trifoliate orange </td><td><em>P. trifoliata</em> [L.] Raf. </td><td>2× </td><td>Cultivated </td><td>72 </td><td>97.4 </td><td>2 </td><td>0.45 </td></tr><tr><td>Trifoliate orange </td><td>Precocious trifoliate orange </td><td><em>P. trifoliata</em> [L.] Raf </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.43 </td></tr><tr><td>Trifoliate orange </td><td>Flyingdragon trifoliate orange </td><td><em>P. trifoliata Raf.</em> var<em>. monstrosa</em> </td><td>2× </td><td>Wild </td><td>74 </td><td>100 </td><td>0 </td><td>0.44 </td></tr><tr><td>Trifoliate orange </td><td>Citranges </td><td><em>P. trifoliata Raf x C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.48 </td></tr><tr><td>Citrus </td><td>‘Fenghuang’ pummelo </td><td><em>C. grandis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.43 </td></tr><tr><td>Citrus </td><td>‘Kaopan’ pummelo </td><td><em>C. grandis</em> </td><td>2× </td><td>Cultivated </td><td>71 </td><td>96.1 </td><td>3 </td><td>0.43 </td></tr><tr><td>Citrus </td><td>Grapefruit </td><td><em>C. paradisi</em> </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Hirado Buntan pummelo </td><td><em>C. grandis</em> </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.44 </td></tr><tr><td>Citrus </td><td>‘Guoqing no.1’ Satsuma </td><td><em>C. unshiu</em> </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.45 </td></tr><tr><td>Citrus </td><td>‘Qingjiang’ Ponkan </td><td><em>C. reticulata</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.44 </td></tr><tr><td>Citrus </td><td>Bendiguangju mandarin </td><td><em>C. reticulata cv. Succosa</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.44 </td></tr><tr><td>Citrus </td><td>‘Miyagawa wase’ Satsuma </td><td><em>C. unshiu</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.44 </td></tr><tr><td>Citrus </td><td>Red tangerine </td><td><em>C. reticulata</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.43 </td></tr><tr><td>Citrus </td><td>Clementine mandarin </td><td><em>C. clementina</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.46 </td></tr><tr><td>Citrus </td><td>Newhall Navel Orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Valencia orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>‘Jingcheng’ sweet orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>‘Honganliu’ sweet orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Sour orange </td><td><em>C. aurantium</em> </td><td>2× </td><td>Wild </td><td>74 </td><td>100 </td><td>0 </td><td>0.45 </td></tr><tr><td>Citrus </td><td>‘CaraCara’ navel orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Washington navel orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Blood orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Ichang papeda </td><td><em>C. ichangensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.44 </td></tr><tr><td>Citrus </td><td>Papeda </td><td><em>C. honghensis</em> </td><td>2× </td><td>Wild </td><td>68 </td><td>92.1 </td><td>6 </td><td>0.42 </td></tr><tr><td>Citrus </td><td>Calamondin </td><td><em>C. mitis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.43 </td></tr><tr><td>Citrus </td><td>Citron </td><td><em>C. medica</em> </td><td>2× </td><td>Wild </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.40 </td></tr><tr><td>Citrus </td><td>Bergamot </td><td><em>C. bergamia</em> </td><td>2× </td><td>Wild </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.42 </td></tr><tr><td>Citrus </td><td>Mexican lime </td><td><em>C. aurantifolia</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.46 </td></tr><tr><td>Citrus </td><td>lemon </td><td><em>C. limon</em> </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Rough lemon </td><td><em>C. jambhiri</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr></tbody></table></div><div content-id="DSW017TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Genus </th><th>Common name </th><th>Latin name </th><th>Ploidy </th><th>Cultivated/wild </th><th>No. of InDel </th><th>% of amplified </th><th>Absent of amplified </th><th>PIC </th></tr></thead><tbody><tr><td>Fortunella Swingle </td><td>Fortunella </td><td><em>F. japonica</em> </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.43 </td></tr><tr><td>Fortunella Swingle </td><td>Kumquat </td><td><em>F. hindsii</em> var<em>. chintou</em> Swing </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.43 </td></tr><tr><td>Trifoliate orange </td><td>Trifoliate orange </td><td><em>P. trifoliata</em> [L.] Raf. </td><td>2× </td><td>Cultivated </td><td>72 </td><td>97.4 </td><td>2 </td><td>0.45 </td></tr><tr><td>Trifoliate orange </td><td>Precocious trifoliate orange </td><td><em>P. trifoliata</em> [L.] Raf </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.43 </td></tr><tr><td>Trifoliate orange </td><td>Flyingdragon trifoliate orange </td><td><em>P. trifoliata Raf.</em> var<em>. monstrosa</em> </td><td>2× </td><td>Wild </td><td>74 </td><td>100 </td><td>0 </td><td>0.44 </td></tr><tr><td>Trifoliate orange </td><td>Citranges </td><td><em>P. trifoliata Raf x C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.48 </td></tr><tr><td>Citrus </td><td>‘Fenghuang’ pummelo </td><td><em>C. grandis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.43 </td></tr><tr><td>Citrus </td><td>‘Kaopan’ pummelo </td><td><em>C. grandis</em> </td><td>2× </td><td>Cultivated </td><td>71 </td><td>96.1 </td><td>3 </td><td>0.43 </td></tr><tr><td>Citrus </td><td>Grapefruit </td><td><em>C. paradisi</em> </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Hirado Buntan pummelo </td><td><em>C. grandis</em> </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.44 </td></tr><tr><td>Citrus </td><td>‘Guoqing no.1’ Satsuma </td><td><em>C. unshiu</em> </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.45 </td></tr><tr><td>Citrus </td><td>‘Qingjiang’ Ponkan </td><td><em>C. reticulata</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.44 </td></tr><tr><td>Citrus </td><td>Bendiguangju mandarin </td><td><em>C. reticulata cv. Succosa</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.44 </td></tr><tr><td>Citrus </td><td>‘Miyagawa wase’ Satsuma </td><td><em>C. unshiu</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.44 </td></tr><tr><td>Citrus </td><td>Red tangerine </td><td><em>C. reticulata</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.43 </td></tr><tr><td>Citrus </td><td>Clementine mandarin </td><td><em>C. clementina</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.46 </td></tr><tr><td>Citrus </td><td>Newhall Navel Orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Valencia orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>‘Jingcheng’ sweet orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>‘Honganliu’ sweet orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Sour orange </td><td><em>C. aurantium</em> </td><td>2× </td><td>Wild </td><td>74 </td><td>100 </td><td>0 </td><td>0.45 </td></tr><tr><td>Citrus </td><td>‘CaraCara’ navel orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Washington navel orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Blood orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Ichang papeda </td><td><em>C. ichangensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.44 </td></tr><tr><td>Citrus </td><td>Papeda </td><td><em>C. honghensis</em> </td><td>2× </td><td>Wild </td><td>68 </td><td>92.1 </td><td>6 </td><td>0.42 </td></tr><tr><td>Citrus </td><td>Calamondin </td><td><em>C. mitis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.43 </td></tr><tr><td>Citrus </td><td>Citron </td><td><em>C. medica</em> </td><td>2× </td><td>Wild </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.40 </td></tr><tr><td>Citrus </td><td>Bergamot </td><td><em>C. bergamia</em> </td><td>2× </td><td>Wild </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.42 </td></tr><tr><td>Citrus </td><td>Mexican lime </td><td><em>C. aurantifolia</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.46 </td></tr><tr><td>Citrus </td><td>lemon </td><td><em>C. limon</em> </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Rough lemon </td><td><em>C. jambhiri</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87197477" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSW017TB1" class="table-wrap-title" data-id="DSW017TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Cross-species transferability of new InDel marker among different genomes in three genera of Rutaceae</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Genus </th><th>Common name </th><th>Latin name </th><th>Ploidy </th><th>Cultivated/wild </th><th>No. of InDel </th><th>% of amplified </th><th>Absent of amplified </th><th>PIC </th></tr></thead><tbody><tr><td>Fortunella Swingle </td><td>Fortunella </td><td><em>F. japonica</em> </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.43 </td></tr><tr><td>Fortunella Swingle </td><td>Kumquat </td><td><em>F. hindsii</em> var<em>. chintou</em> Swing </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.43 </td></tr><tr><td>Trifoliate orange </td><td>Trifoliate orange </td><td><em>P. trifoliata</em> [L.] Raf. </td><td>2× </td><td>Cultivated </td><td>72 </td><td>97.4 </td><td>2 </td><td>0.45 </td></tr><tr><td>Trifoliate orange </td><td>Precocious trifoliate orange </td><td><em>P. trifoliata</em> [L.] Raf </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.43 </td></tr><tr><td>Trifoliate orange </td><td>Flyingdragon trifoliate orange </td><td><em>P. trifoliata Raf.</em> var<em>. monstrosa</em> </td><td>2× </td><td>Wild </td><td>74 </td><td>100 </td><td>0 </td><td>0.44 </td></tr><tr><td>Trifoliate orange </td><td>Citranges </td><td><em>P. trifoliata Raf x C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.48 </td></tr><tr><td>Citrus </td><td>‘Fenghuang’ pummelo </td><td><em>C. grandis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.43 </td></tr><tr><td>Citrus </td><td>‘Kaopan’ pummelo </td><td><em>C. grandis</em> </td><td>2× </td><td>Cultivated </td><td>71 </td><td>96.1 </td><td>3 </td><td>0.43 </td></tr><tr><td>Citrus </td><td>Grapefruit </td><td><em>C. paradisi</em> </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Hirado Buntan pummelo </td><td><em>C. grandis</em> </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.44 </td></tr><tr><td>Citrus </td><td>‘Guoqing no.1’ Satsuma </td><td><em>C. unshiu</em> </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.45 </td></tr><tr><td>Citrus </td><td>‘Qingjiang’ Ponkan </td><td><em>C. reticulata</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.44 </td></tr><tr><td>Citrus </td><td>Bendiguangju mandarin </td><td><em>C. reticulata cv. Succosa</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.44 </td></tr><tr><td>Citrus </td><td>‘Miyagawa wase’ Satsuma </td><td><em>C. unshiu</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.44 </td></tr><tr><td>Citrus </td><td>Red tangerine </td><td><em>C. reticulata</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.43 </td></tr><tr><td>Citrus </td><td>Clementine mandarin </td><td><em>C. clementina</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.46 </td></tr><tr><td>Citrus </td><td>Newhall Navel Orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Valencia orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>‘Jingcheng’ sweet orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>‘Honganliu’ sweet orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Sour orange </td><td><em>C. aurantium</em> </td><td>2× </td><td>Wild </td><td>74 </td><td>100 </td><td>0 </td><td>0.45 </td></tr><tr><td>Citrus </td><td>‘CaraCara’ navel orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Washington navel orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Blood orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Ichang papeda </td><td><em>C. ichangensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.44 </td></tr><tr><td>Citrus </td><td>Papeda </td><td><em>C. honghensis</em> </td><td>2× </td><td>Wild </td><td>68 </td><td>92.1 </td><td>6 </td><td>0.42 </td></tr><tr><td>Citrus </td><td>Calamondin </td><td><em>C. mitis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.43 </td></tr><tr><td>Citrus </td><td>Citron </td><td><em>C. medica</em> </td><td>2× </td><td>Wild </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.40 </td></tr><tr><td>Citrus </td><td>Bergamot </td><td><em>C. bergamia</em> </td><td>2× </td><td>Wild </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.42 </td></tr><tr><td>Citrus </td><td>Mexican lime </td><td><em>C. aurantifolia</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.46 </td></tr><tr><td>Citrus </td><td>lemon </td><td><em>C. limon</em> </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Rough lemon </td><td><em>C. jambhiri</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr></tbody></table></div><div content-id="DSW017TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Genus </th><th>Common name </th><th>Latin name </th><th>Ploidy </th><th>Cultivated/wild </th><th>No. of InDel </th><th>% of amplified </th><th>Absent of amplified </th><th>PIC </th></tr></thead><tbody><tr><td>Fortunella Swingle </td><td>Fortunella </td><td><em>F. japonica</em> </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.43 </td></tr><tr><td>Fortunella Swingle </td><td>Kumquat </td><td><em>F. hindsii</em> var<em>. chintou</em> Swing </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.43 </td></tr><tr><td>Trifoliate orange </td><td>Trifoliate orange </td><td><em>P. trifoliata</em> [L.] Raf. </td><td>2× </td><td>Cultivated </td><td>72 </td><td>97.4 </td><td>2 </td><td>0.45 </td></tr><tr><td>Trifoliate orange </td><td>Precocious trifoliate orange </td><td><em>P. trifoliata</em> [L.] Raf </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.43 </td></tr><tr><td>Trifoliate orange </td><td>Flyingdragon trifoliate orange </td><td><em>P. trifoliata Raf.</em> var<em>. monstrosa</em> </td><td>2× </td><td>Wild </td><td>74 </td><td>100 </td><td>0 </td><td>0.44 </td></tr><tr><td>Trifoliate orange </td><td>Citranges </td><td><em>P. trifoliata Raf x C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.48 </td></tr><tr><td>Citrus </td><td>‘Fenghuang’ pummelo </td><td><em>C. grandis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.43 </td></tr><tr><td>Citrus </td><td>‘Kaopan’ pummelo </td><td><em>C. grandis</em> </td><td>2× </td><td>Cultivated </td><td>71 </td><td>96.1 </td><td>3 </td><td>0.43 </td></tr><tr><td>Citrus </td><td>Grapefruit </td><td><em>C. paradisi</em> </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Hirado Buntan pummelo </td><td><em>C. grandis</em> </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.44 </td></tr><tr><td>Citrus </td><td>‘Guoqing no.1’ Satsuma </td><td><em>C. unshiu</em> </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.45 </td></tr><tr><td>Citrus </td><td>‘Qingjiang’ Ponkan </td><td><em>C. reticulata</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.44 </td></tr><tr><td>Citrus </td><td>Bendiguangju mandarin </td><td><em>C. reticulata cv. Succosa</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.44 </td></tr><tr><td>Citrus </td><td>‘Miyagawa wase’ Satsuma </td><td><em>C. unshiu</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.44 </td></tr><tr><td>Citrus </td><td>Red tangerine </td><td><em>C. reticulata</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.43 </td></tr><tr><td>Citrus </td><td>Clementine mandarin </td><td><em>C. clementina</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.46 </td></tr><tr><td>Citrus </td><td>Newhall Navel Orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Valencia orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>‘Jingcheng’ sweet orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>‘Honganliu’ sweet orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Sour orange </td><td><em>C. aurantium</em> </td><td>2× </td><td>Wild </td><td>74 </td><td>100 </td><td>0 </td><td>0.45 </td></tr><tr><td>Citrus </td><td>‘CaraCara’ navel orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Washington navel orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Blood orange </td><td><em>C. sinensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Ichang papeda </td><td><em>C. ichangensis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.44 </td></tr><tr><td>Citrus </td><td>Papeda </td><td><em>C. honghensis</em> </td><td>2× </td><td>Wild </td><td>68 </td><td>92.1 </td><td>6 </td><td>0.42 </td></tr><tr><td>Citrus </td><td>Calamondin </td><td><em>C. mitis</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.43 </td></tr><tr><td>Citrus </td><td>Citron </td><td><em>C. medica</em> </td><td>2× </td><td>Wild </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.40 </td></tr><tr><td>Citrus </td><td>Bergamot </td><td><em>C. bergamia</em> </td><td>2× </td><td>Wild </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.42 </td></tr><tr><td>Citrus </td><td>Mexican lime </td><td><em>C. aurantifolia</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.46 </td></tr><tr><td>Citrus </td><td>lemon </td><td><em>C. limon</em> </td><td>2× </td><td>Cultivated </td><td>73 </td><td>98.7 </td><td>1 </td><td>0.47 </td></tr><tr><td>Citrus </td><td>Rough lemon </td><td><em>C. jambhiri</em> </td><td>2× </td><td>Cultivated </td><td>74 </td><td>100 </td><td>0 </td><td>0.47 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87197477" target="_blank">View Large</a></div></div>                        <a id="87197478" scrollto-destination="87197478"></a>
<div id="DSW017F4" data-id="DSW017F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/m_dsw01704.png?Expires=2147483647&Signature=olKtxCLfcu1cvDYmfDrnER4mWzx9Iv9pWISbfCVolmXMHSgMTWSUIajtrjRuN4hRUrD6p9upTnc1Xbzhu~qI1uhI-mYzuSmbQQUSkyRIdbKvG~n26EYQ-~7M-8hqoo5Rn1ltfQhdlE0zypnAG7U5DaA7PTi459xLiB5L9bKgPTLtcAyNpPuphXj42dJvt~R2IMpDpVrWxPQNsZQERsFgDGqTkXKzsTjIhY7vX2f4l~tf2i-t5~zUClIZyium5A6u4uOnnp1agLZN-FpGOQ1HkzWwddKstQoMQxoOshjRcgubUz8R8ajZWiB2N2~FSh6b5HfKorto4xtwoM5kKs31eg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Polymorphism detected by 13 InDel markers among 32 genotypes of three genera of Rutaceae. 1: fortunella (F. japonica); 2: kumquat (F. hindsii var. chintou Swing); 3: trifoliate orange (P. trifoliata [L.] Raf.); 4: precocious trifoliate orange (P. trifoliata [L.] Raf.); 5: flying dragon trifoliate orange (P. trifoliata Raf. var. monstrosa); 6: cotranges (P. trifoliata Raf ́ C. sinensis); 7: Fenghuang pummelo (C. grandis); 8: Kaopan pummelo (C. grandis); 9: grapefruit (C. grandis); 10: Hirado Buntan pummelo (C. grandis); 11: Guoqing no. 1 satsuma (C. unshiu); 12: Qingjiang ponkan (C. reticulata); 13: Bendiguangju mandarin (C. reticulata cv. Succosa); 14: Miyagawa wase Satsuma (C. unshiu); 15: red tangerine (C. reticulata); 16: calamondin (C. mitis); 17: Clementine mandarin (C. clementina); 18: Newhall navel Orange (C. sinensis); 19: Valencia orange (C. sinensis); 20: Jingcheng sweet orange (C. sinensis); 21: Honganliu sweet orange (C. sinensis); 22: sour orange (C. aurantium); 23: Cara Cara navel orange (C. sinensis); 24: Washington navel orange (C. sinensis); 25: blood orange (C. sinensis); 26: Ichang papeda (C. ichangensis); 27: papeda (C. honghensis); 28: citron (C. medica); 29: bergamot (C. bergamia Risso); 30: lemon (C. limon); 31: Mexican lime (C. aurantifolia); 32: rough lemon (C. jambhiri). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01704.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197478/dsw01704.png" path-from-xml="dsw01704.png" target="_blank">View large</a><a section="87197478" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw01704.png?Expires=2147483647&Signature=GiFLF7oCx6pMYUnVCR~-EniynYOaJA1rDqZd~9rSErUqa~Qqjgzjeks3~AmnGNkiVQELqOc8-f0-gLmBnprNn6-MPD8IZYbih0ZfIWvr6Y~15aIdw2mv0ZymlPUN~cuUwSJxJd40nkdvoDXjR6TsRvkork0ieb~gs9y3xv0KG8hlifeyBvVcSxf-0VrLT0Fmb9Vpg9wur4V7xcz1CLpzwcbkhMTZJsxOzip2PpXZsTlEqlf00-hr9-Vfhw6vNMAab27Kevjjs4kXZ94H~ARpXsTau~SV8zLblesnv7lx2hi4w4YFnz2K~nPbZ08-OjSaJq7KS2aCzVMVmuabiOt1IA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197478&ar=2469959&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01704.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Polymorphism detected by 13 InDel markers among 32 genotypes of three genera of Rutaceae. 1: fortunella (<em>F. japonica</em>); 2: kumquat (<em>F. hindsii</em> var<em>. chintou</em> Swing); 3: trifoliate orange (<em>P. trifoliata</em> [L.] Raf.); 4: precocious trifoliate orange (<em>P. trifoliata</em> [L.] Raf.); 5: flying dragon trifoliate orange (<em>P. trifoliata Raf.</em> var<em>. monstrosa</em>); 6: cotranges (<em>P. trifoliata Raf</em> ́ <em>C. sinensis</em>); 7: Fenghuang pummelo (<em>C. grandis</em>); 8: Kaopan pummelo (<em>C. grandis</em>); 9: grapefruit (<em>C. grandis</em>); 10: Hirado Buntan pummelo (<em>C. grandis</em>); 11: Guoqing no. 1 satsuma (<em>C. unshiu</em>); 12: Qingjiang ponkan (<em>C. reticulata</em>); 13: Bendiguangju mandarin (<em>C. reticulata</em> cv. <em>Succosa</em>); 14: Miyagawa wase Satsuma (<em>C. unshiu</em>); 15: red tangerine (<em>C. reticulata</em>); 16: calamondin (<em>C. mitis</em>); 17: Clementine mandarin (<em>C. clementina</em>); 18: Newhall navel Orange (<em>C. sinensis</em>); 19: Valencia orange (<em>C. sinensis</em>); 20: Jingcheng sweet orange (<em>C. sinensis</em>); 21: Honganliu sweet orange (<em>C. sinensis</em>); 22: sour orange (<em>C. aurantium</em>); 23: Cara Cara navel orange (<em>C. sinensis</em>); 24: Washington navel orange (<em>C. sinensis</em>); 25: blood orange (<em>C. sinensis</em>); 26: Ichang papeda (<em>C. ichangensis</em>); 27: papeda (<em>C. honghensis</em>); 28: citron (<em>C. medica</em>); 29: bergamot (<em>C. bergamia</em> Risso); 30: lemon (<em>C. limon</em>); 31: Mexican lime (<em>C. aurantifolia</em>); 32: rough lemon (<em>C. jambhiri</em>). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW017F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/m_dsw01704.png?Expires=2147483647&Signature=olKtxCLfcu1cvDYmfDrnER4mWzx9Iv9pWISbfCVolmXMHSgMTWSUIajtrjRuN4hRUrD6p9upTnc1Xbzhu~qI1uhI-mYzuSmbQQUSkyRIdbKvG~n26EYQ-~7M-8hqoo5Rn1ltfQhdlE0zypnAG7U5DaA7PTi459xLiB5L9bKgPTLtcAyNpPuphXj42dJvt~R2IMpDpVrWxPQNsZQERsFgDGqTkXKzsTjIhY7vX2f4l~tf2i-t5~zUClIZyium5A6u4uOnnp1agLZN-FpGOQ1HkzWwddKstQoMQxoOshjRcgubUz8R8ajZWiB2N2~FSh6b5HfKorto4xtwoM5kKs31eg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Polymorphism detected by 13 InDel markers among 32 genotypes of three genera of Rutaceae. 1: fortunella (F. japonica); 2: kumquat (F. hindsii var. chintou Swing); 3: trifoliate orange (P. trifoliata [L.] Raf.); 4: precocious trifoliate orange (P. trifoliata [L.] Raf.); 5: flying dragon trifoliate orange (P. trifoliata Raf. var. monstrosa); 6: cotranges (P. trifoliata Raf ́ C. sinensis); 7: Fenghuang pummelo (C. grandis); 8: Kaopan pummelo (C. grandis); 9: grapefruit (C. grandis); 10: Hirado Buntan pummelo (C. grandis); 11: Guoqing no. 1 satsuma (C. unshiu); 12: Qingjiang ponkan (C. reticulata); 13: Bendiguangju mandarin (C. reticulata cv. Succosa); 14: Miyagawa wase Satsuma (C. unshiu); 15: red tangerine (C. reticulata); 16: calamondin (C. mitis); 17: Clementine mandarin (C. clementina); 18: Newhall navel Orange (C. sinensis); 19: Valencia orange (C. sinensis); 20: Jingcheng sweet orange (C. sinensis); 21: Honganliu sweet orange (C. sinensis); 22: sour orange (C. aurantium); 23: Cara Cara navel orange (C. sinensis); 24: Washington navel orange (C. sinensis); 25: blood orange (C. sinensis); 26: Ichang papeda (C. ichangensis); 27: papeda (C. honghensis); 28: citron (C. medica); 29: bergamot (C. bergamia Risso); 30: lemon (C. limon); 31: Mexican lime (C. aurantifolia); 32: rough lemon (C. jambhiri). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01704.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197478/dsw01704.png" path-from-xml="dsw01704.png" target="_blank">View large</a><a section="87197478" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw01704.png?Expires=2147483647&Signature=GiFLF7oCx6pMYUnVCR~-EniynYOaJA1rDqZd~9rSErUqa~Qqjgzjeks3~AmnGNkiVQELqOc8-f0-gLmBnprNn6-MPD8IZYbih0ZfIWvr6Y~15aIdw2mv0ZymlPUN~cuUwSJxJd40nkdvoDXjR6TsRvkork0ieb~gs9y3xv0KG8hlifeyBvVcSxf-0VrLT0Fmb9Vpg9wur4V7xcz1CLpzwcbkhMTZJsxOzip2PpXZsTlEqlf00-hr9-Vfhw6vNMAab27Kevjjs4kXZ94H~ARpXsTau~SV8zLblesnv7lx2hi4w4YFnz2K~nPbZ08-OjSaJq7KS2aCzVMVmuabiOt1IA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197478&ar=2469959&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01704.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Polymorphism detected by 13 InDel markers among 32 genotypes of three genera of Rutaceae. 1: fortunella (<em>F. japonica</em>); 2: kumquat (<em>F. hindsii</em> var<em>. chintou</em> Swing); 3: trifoliate orange (<em>P. trifoliata</em> [L.] Raf.); 4: precocious trifoliate orange (<em>P. trifoliata</em> [L.] Raf.); 5: flying dragon trifoliate orange (<em>P. trifoliata Raf.</em> var<em>. monstrosa</em>); 6: cotranges (<em>P. trifoliata Raf</em> ́ <em>C. sinensis</em>); 7: Fenghuang pummelo (<em>C. grandis</em>); 8: Kaopan pummelo (<em>C. grandis</em>); 9: grapefruit (<em>C. grandis</em>); 10: Hirado Buntan pummelo (<em>C. grandis</em>); 11: Guoqing no. 1 satsuma (<em>C. unshiu</em>); 12: Qingjiang ponkan (<em>C. reticulata</em>); 13: Bendiguangju mandarin (<em>C. reticulata</em> cv. <em>Succosa</em>); 14: Miyagawa wase Satsuma (<em>C. unshiu</em>); 15: red tangerine (<em>C. reticulata</em>); 16: calamondin (<em>C. mitis</em>); 17: Clementine mandarin (<em>C. clementina</em>); 18: Newhall navel Orange (<em>C. sinensis</em>); 19: Valencia orange (<em>C. sinensis</em>); 20: Jingcheng sweet orange (<em>C. sinensis</em>); 21: Honganliu sweet orange (<em>C. sinensis</em>); 22: sour orange (<em>C. aurantium</em>); 23: Cara Cara navel orange (<em>C. sinensis</em>); 24: Washington navel orange (<em>C. sinensis</em>); 25: blood orange (<em>C. sinensis</em>); 26: Ichang papeda (<em>C. ichangensis</em>); 27: papeda (<em>C. honghensis</em>); 28: citron (<em>C. medica</em>); 29: bergamot (<em>C. bergamia</em> Risso); 30: lemon (<em>C. limon</em>); 31: Mexican lime (<em>C. aurantifolia</em>); 32: rough lemon (<em>C. jambhiri</em>). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87197479 id="87197479" class="section-title" data-legacy-id=s3h>3.8. Genetic diversity and dendrogram in citrus genus by new DNA markers</h3>
<p>Despite a relatively low level of polymorphism, the 74 new markers from InDels were also examined for their potential use in genetic diversity analysis among 32 citrus species (Fig. <a reveal-id="DSW017F5" data-open="DSW017F5" class="link link-reveal link-table xref-fig">5</a>). The UPGMA phylogenetic tree revealed three major clusters including 4, 3, and 25 genotypes in each cluster (trifoliate orange, fortunella, and citrus, respectively). The UPGMA-based clustering showed the grouping of all 4 species (including the MT and the WT) of trifoliate orange in one group (trifoliate orange cluster), 3 species of fortunella, kumquat, and calamondin in one group (fortunella cluster), and 25 species in one group (citrus cluster). Traditionally, calamondin should be clustered citrus cluster. There might be two possible explanations: at first, calamondin with a short juvenile of 1–2 yr and flowered twice or three times per year similar to precocious trifoliate orange. Second, these new InDel markers may be related to flowering. In the citrus cluster, citrus limon, citrus medica, citrus grandis, and sweet orange formed one subcluster and were more closely related than the other subclusters of citrus. Nei's genetic distance values ranged from 0 (<em>C. sinesis</em> Newhall, Jingcheng, Cara Cara, Washington, and blood orange) to 0.41 (<em>P. trifoliata</em> Raf. × <em>C. sinensis</em> vs. <em>C. grandis</em> Kaopan), with an average value of 0.25 ± 0.07 (Fig. <a reveal-id="DSW017F5" data-open="DSW017F5" class="link link-reveal link-table xref-fig">5</a>). The out-group trifoliate orange and fortunella showed relatively larger amounts of average genetic distance (0.39 and 0.31) from all the genotypes of citrus. </p>                        <a id="87197481" scrollto-destination="87197481"></a>
<div id="DSW017F5" data-id="DSW017F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/m_dsw01705.png?Expires=2147483647&Signature=e8nUaVszhFD~Q25CU8UbzxqCEOK2FuLQagB3Bks4kD~iyN7xr3Q~Id4lGotuNiCt7IWU94rJNSGKclC1-iqbgs1hIBu8IcnBXkzggDdF6muHz4X53AZHpxLJlTAhIFJSstYDwv3Y~B~sY~0fL9TMclSWe6~MQOGo3Jk6Z7ISfLGxKcMXNaCB-dieS5OZqnFHne7YeBaake5FdPpFGCWqRf1L80w~ehiIbNRO0havtKBa7PhF8MAKZCCwMQzC0cwFmCGCaIGmIcfw7ul0LFKNpOclbZqF6h8sU6B5Q~VuAI~Ugm9ziMMGsTZw3dtOvZ9FRuIX-m2x8-ZYYQRJTJbn-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="UPGMA tree of 32 citrus genotypes of three genera of Rutaceae based on Nei's genetic distance using 74 new InDel markers. C. bergamia Risso: bergamot; C. medica: citron; C. honghensis: papeda; C. ichangensis: Ichang papeda; C. limon: lemon; C. aurantifolia: Mexican lime; C. jambhiri: rough lemon; C. unshiu-Miyagawa wase: Miyagawa wase Satsuma; C. grandis-Kaopan: Kaopan pummelo; C. grandis-Fenghuang: Fenghuang pummelo; C. grandis-Hirado: Hirado Buntan pummelo; C. reticulate-Qingjiang: Qingjiang ponkan; C. reticulate-red tangerine: red tangerine; C. unshiu-Guoqing no.1: Guoqing no. 1 satsuma fortunella; C. reticulate-Bendiguangju: Bendiguangju mandarin; C. aurantium: sour orange; C. grandis: grapefruit; C. clementina: Clementine mandarin; C. sinensis-Newhall: Newhall navel Orange; C. sinensis-Jingcheng: Jingcheng sweet orange; C. sinensis-blood orange: blood orange; C. sinensis-Cara Cara: Cara Cara navel orange; C. sinensis-Washington: Washington navel orange; C. sinensis-Valencia: Valencia orange; C. sinensis-Honganliu: Honganliu sweet orange; C. mitis: calamondin; F. hindsii var. chintou: kumquat; Fortunella: F. japonica; P. trifoliate: trifoliate orange; P. trifoliate var. precocious: precocious trifoliate orange; P. trifoliata var. monstrosa: flying dragon trifoliate orange; P. trifoliata Raf × C. sinensis: cotranges." path-from-xml="dsw01705.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197481/dsw01705.png" path-from-xml="dsw01705.png" target="_blank">View large</a><a section="87197481" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw01705.png?Expires=2147483647&Signature=XfoNcNjL6ZMQHK~r0Z3Dc151kUH4dGBanvJm1~dkpwhqCjcM76MBiB66EwSyCTmk2M8fe3GH0SxdG3QD7YVmKE8XQGDnv38K78mwtRgA00ierYOH~0tPgGcUWcxhZ-0ZSicUXwCjJwbMcLacXrvfj082l8IG0uwWXtZCH5dmefpuaUK2vmoME~QjWQcf~JwBUCOVFgke2QEQfNH~o6p4fyAjGnh1L8NldVI7iJn4M0X7yG7Z0YgiBX9Z8EeS53MX-eQgXnB3e7wwgIl5S9CWvq~~1SpD542t2RsgnT5a1ugbDrTrSIrZENgP3iTpqSsARDibdQMnLV-~nmvwWf0uSg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197481&ar=2469959&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01705.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>UPGMA tree of 32 citrus genotypes of three genera of Rutaceae based on Nei's genetic distance using 74 new InDel markers. <em>C. bergamia</em> Risso: bergamot; <em>C. medica</em>: citron; <em>C. honghensis</em>: papeda; <em>C. ichangensis</em>: Ichang papeda; <em>C. limon</em>: lemon; <em>C. aurantifolia</em>: Mexican lime; <em>C. jambhiri</em>: rough lemon; <em>C. unshiu-</em>Miyagawa wase: Miyagawa wase Satsuma; <em>C. grandis</em>-Kaopan: Kaopan pummelo; <em>C. grandis-</em>Fenghuang: Fenghuang pummelo; <em>C. grandis</em>-Hirado: Hirado Buntan pummelo; <em>C. reticulate</em>-Qingjiang: Qingjiang ponkan; <em>C. reticulate</em>-red tangerine: red tangerine; <em>C. unshiu-</em>Guoqing no.1: Guoqing no. 1 satsuma fortunella; <em>C. reticulate</em>-Bendiguangju: Bendiguangju mandarin; <em>C. aurantium</em>: sour orange; <em>C. grandis</em>: grapefruit; <em>C. clementina</em>: Clementine mandarin; <em>C. sinensis</em>-Newhall<em>:</em> Newhall navel Orange; <em>C. sinensis</em>-Jingcheng: Jingcheng sweet orange; <em>C. sinensis</em>-blood orange: blood orange; <em>C. sinensis</em>-Cara Cara: Cara Cara navel orange; <em>C. sinensis</em>-Washington: Washington navel orange; <em>C. sinensis</em>-Valencia: Valencia orange; <em>C. sinensis</em>-Honganliu: Honganliu sweet orange; <em>C. mitis</em>: calamondin; <em>F. hindsii</em> var<em>. chintou</em>: kumquat; Fortunella: <em>F. japonica</em>; <em>P. trifoliate</em>: trifoliate orange; <em>P. trifoliate</em> var<em>.</em> precocious: precocious trifoliate orange; <em>P. trifoliata</em> var<em>. monstrosa</em>: flying dragon trifoliate orange; <em>P. trifoliata</em> Raf × <em>C. sinensis</em>: cotranges.</p></div></div><div content-id="DSW017F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/m_dsw01705.png?Expires=2147483647&Signature=e8nUaVszhFD~Q25CU8UbzxqCEOK2FuLQagB3Bks4kD~iyN7xr3Q~Id4lGotuNiCt7IWU94rJNSGKclC1-iqbgs1hIBu8IcnBXkzggDdF6muHz4X53AZHpxLJlTAhIFJSstYDwv3Y~B~sY~0fL9TMclSWe6~MQOGo3Jk6Z7ISfLGxKcMXNaCB-dieS5OZqnFHne7YeBaake5FdPpFGCWqRf1L80w~ehiIbNRO0havtKBa7PhF8MAKZCCwMQzC0cwFmCGCaIGmIcfw7ul0LFKNpOclbZqF6h8sU6B5Q~VuAI~Ugm9ziMMGsTZw3dtOvZ9FRuIX-m2x8-ZYYQRJTJbn-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="UPGMA tree of 32 citrus genotypes of three genera of Rutaceae based on Nei's genetic distance using 74 new InDel markers. C. bergamia Risso: bergamot; C. medica: citron; C. honghensis: papeda; C. ichangensis: Ichang papeda; C. limon: lemon; C. aurantifolia: Mexican lime; C. jambhiri: rough lemon; C. unshiu-Miyagawa wase: Miyagawa wase Satsuma; C. grandis-Kaopan: Kaopan pummelo; C. grandis-Fenghuang: Fenghuang pummelo; C. grandis-Hirado: Hirado Buntan pummelo; C. reticulate-Qingjiang: Qingjiang ponkan; C. reticulate-red tangerine: red tangerine; C. unshiu-Guoqing no.1: Guoqing no. 1 satsuma fortunella; C. reticulate-Bendiguangju: Bendiguangju mandarin; C. aurantium: sour orange; C. grandis: grapefruit; C. clementina: Clementine mandarin; C. sinensis-Newhall: Newhall navel Orange; C. sinensis-Jingcheng: Jingcheng sweet orange; C. sinensis-blood orange: blood orange; C. sinensis-Cara Cara: Cara Cara navel orange; C. sinensis-Washington: Washington navel orange; C. sinensis-Valencia: Valencia orange; C. sinensis-Honganliu: Honganliu sweet orange; C. mitis: calamondin; F. hindsii var. chintou: kumquat; Fortunella: F. japonica; P. trifoliate: trifoliate orange; P. trifoliate var. precocious: precocious trifoliate orange; P. trifoliata var. monstrosa: flying dragon trifoliate orange; P. trifoliata Raf × C. sinensis: cotranges." path-from-xml="dsw01705.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197481/dsw01705.png" path-from-xml="dsw01705.png" target="_blank">View large</a><a section="87197481" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw01705.png?Expires=2147483647&Signature=XfoNcNjL6ZMQHK~r0Z3Dc151kUH4dGBanvJm1~dkpwhqCjcM76MBiB66EwSyCTmk2M8fe3GH0SxdG3QD7YVmKE8XQGDnv38K78mwtRgA00ierYOH~0tPgGcUWcxhZ-0ZSicUXwCjJwbMcLacXrvfj082l8IG0uwWXtZCH5dmefpuaUK2vmoME~QjWQcf~JwBUCOVFgke2QEQfNH~o6p4fyAjGnh1L8NldVI7iJn4M0X7yG7Z0YgiBX9Z8EeS53MX-eQgXnB3e7wwgIl5S9CWvq~~1SpD542t2RsgnT5a1ugbDrTrSIrZENgP3iTpqSsARDibdQMnLV-~nmvwWf0uSg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197481&ar=2469959&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01705.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>UPGMA tree of 32 citrus genotypes of three genera of Rutaceae based on Nei's genetic distance using 74 new InDel markers. <em>C. bergamia</em> Risso: bergamot; <em>C. medica</em>: citron; <em>C. honghensis</em>: papeda; <em>C. ichangensis</em>: Ichang papeda; <em>C. limon</em>: lemon; <em>C. aurantifolia</em>: Mexican lime; <em>C. jambhiri</em>: rough lemon; <em>C. unshiu-</em>Miyagawa wase: Miyagawa wase Satsuma; <em>C. grandis</em>-Kaopan: Kaopan pummelo; <em>C. grandis-</em>Fenghuang: Fenghuang pummelo; <em>C. grandis</em>-Hirado: Hirado Buntan pummelo; <em>C. reticulate</em>-Qingjiang: Qingjiang ponkan; <em>C. reticulate</em>-red tangerine: red tangerine; <em>C. unshiu-</em>Guoqing no.1: Guoqing no. 1 satsuma fortunella; <em>C. reticulate</em>-Bendiguangju: Bendiguangju mandarin; <em>C. aurantium</em>: sour orange; <em>C. grandis</em>: grapefruit; <em>C. clementina</em>: Clementine mandarin; <em>C. sinensis</em>-Newhall<em>:</em> Newhall navel Orange; <em>C. sinensis</em>-Jingcheng: Jingcheng sweet orange; <em>C. sinensis</em>-blood orange: blood orange; <em>C. sinensis</em>-Cara Cara: Cara Cara navel orange; <em>C. sinensis</em>-Washington: Washington navel orange; <em>C. sinensis</em>-Valencia: Valencia orange; <em>C. sinensis</em>-Honganliu: Honganliu sweet orange; <em>C. mitis</em>: calamondin; <em>F. hindsii</em> var<em>. chintou</em>: kumquat; Fortunella: <em>F. japonica</em>; <em>P. trifoliate</em>: trifoliate orange; <em>P. trifoliate</em> var<em>.</em> precocious: precocious trifoliate orange; <em>P. trifoliata</em> var<em>. monstrosa</em>: flying dragon trifoliate orange; <em>P. trifoliata</em> Raf × <em>C. sinensis</em>: cotranges.</p></div></div>                            <h2 scrollto-destination=87197482 id="87197482" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>To understand genetic variation involved in flowering development in citrus, we used genome resequencing technology to analyse the genetic variations between precocious trifoliate orange and its WT. Between the MT and the WT, 3,932,628 SNPs, 1,293,383 InDels, and 52,135 SVs were identified. In previous studies, precocious trifoliate orange is speculated to be a direct variant of the WT.<sup><a reveal-id="DSW017C10" data-open="DSW017C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSW017C12" data-open="DSW017C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSW017C13" data-open="DSW017C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Such a large amount of genetic variation is found owing to the following reasons. For example, naturally occurring mutants are extensively found under the influence of environmental factors in woody perennials and humanity<sup><a reveal-id="DSW017C36" data-open="DSW017C36" class="link link-ref link-reveal xref-bibr">36</a></sup>; it was reported that there was many genetic variation in different tissues of the human body such as liver and lung.<sup><a reveal-id="DSW017C37" data-open="DSW017C37" class="link link-ref link-reveal xref-bibr">37</a>,<a reveal-id="DSW017C38" data-open="DSW017C38" class="link link-ref link-reveal xref-bibr">38</a></sup> On the other hand, the trifoliate orange produces polyembryonic seeds containing both sexual and apomictic embryos. About 10–20% of seedlings from open-pollinated trifoliate orange seeds develop from zygotic embryos; some genetic variations may be generated. Therefore, the genetic variations gradually increase from generation to generation.<sup><a reveal-id="DSW017C39" data-open="DSW017C39" class="link link-ref link-reveal xref-bibr">39</a></sup> In this study, &gt;20% of the SNPs were heterozygous in two trifoliate oranges. Further supporting the observation that the InDels was also highly heterozygous, approximately 31.07 and 32.04% of the InDels were heterozygous in the MT and the WT, respectively. This observation suggested a notably high heterozygosity rate in the trifoliate orange genera. High levels of heterozygosity introduce additional challenges for the identification of various kinds of DNA polymorphisms, especially complex SVs.<sup><a reveal-id="DSW017C37" data-open="DSW017C37" class="link link-ref link-reveal xref-bibr">37</a></sup> Heterozygosity decreases the proportion of reads mapping to a unique genomic location in a manner dependent on the degree of heterozygosity, so only 61.73 and 63.13% of clean reads were uniquely mapped to the citrus reference genome. In previous studies, heterozygosity has been a common feature in most eukaryotic organisms and has shown important biological functions in woody plants.<sup><a reveal-id="DSW017C40" data-open="DSW017C40" class="link link-ref link-reveal xref-bibr">40</a>,<a reveal-id="DSW017C41" data-open="DSW017C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Although citrus is generally perceived to have highly heterozygous traits,<sup><a reveal-id="DSW017C14" data-open="DSW017C14" class="link link-ref link-reveal xref-bibr">14</a></sup> the amount of heterozygosity in the whole genome is not clear. The results presented in this study further characterize the extent of genome heterozygosity and its functional effects at both the whole genome and the global transcriptome levels for two trifoliate oranges.</p><p>Knowing genomic positions of genetic variations in genetic markers is important.<sup><a reveal-id="DSW017C42" data-open="DSW017C42" class="link link-ref link-reveal xref-bibr">42</a></sup> Many population genetic and genetic mapping applications rely on unlinked markers. In this study, these genetic variations showed only minimal distribution in CDS regions, consistent with the results of SNP in tomato.<sup><a reveal-id="DSW017C43" data-open="DSW017C43" class="link link-ref link-reveal xref-bibr">43</a></sup> This might be related to the increased size of the intron in citrus. Interestingly, there were more SNPs than InDels and SVs in the CDS regions. This difference can be explained by the fact that InDels and SVs are more deleterious than SNPs in the CDS regions, as indicated by InDels and SVs that cause frameshift mutations and amino acid substitutions that cause major changes in gene function.<sup><a reveal-id="DSW017C44" data-open="DSW017C44" class="link link-ref link-reveal xref-bibr">44</a>,<a reveal-id="DSW017C45" data-open="DSW017C45" class="link link-ref link-reveal xref-bibr">45</a></sup> However, SNPs often produce synonymous mutations that have little or no impact on gene expression and function.<sup><a reveal-id="DSW017C46" data-open="DSW017C46" class="link link-ref link-reveal xref-bibr">46</a></sup> In this study, 16,919 MT-specific and 12,513 WT-specific SNPs predicted to cause non-synonymous amino acid substitutions were identified at the transcriptional level based on integrative analysis. The ratio of non-synonymous to synonymous substitutions was 2.22 and 2.19 in the WT and the MT, respectively. These SNPs may represent causal genetic variation contributing to phenotype variation in the MT and the WT. Without additional experimental evidence, we cannot yet say whether these SNPs affect the regulation of flowering-related genes or have another direct effect on mutant phenotypes. Nevertheless, using this SNP set to perform genome-wide association in two genotype genomes would be more efficient than using a general SNP set to identify causal gene mutations.</p><p>Combining genome and transcriptome profiling data has been used as a powerful approach for the identification of functional genetic variations and candidate genes for traits of interest.<sup><a reveal-id="DSW017C47" data-open="DSW017C47" class="link link-ref link-reveal xref-bibr">47</a></sup> To investigate the candidate genetic variation responsible for early flowering, we integrated the RNA-Seq and genome resequencing of genetic variation data. A total of 362 large-effect SNPs were found between the MT and the WT and were found to affect the integrity of encoded proteins. These large-effect SNPs included disruption of splice sites, loss of translation initiation codon, introduction of premature stop codon, and loss of stop codon. Likewise, we identified 1,137 and 721 specific InDels in the CDS region of the MT and the WT, respectively, during the phase stage, which may cause frameshift, disruption of splice sites, or introduction of premature stop codon in the two genotypes of trifoliate orange. The significant proportion of genes containing large-effect SNPs and InDels suggested that these genes may represent differences in gene sequences between the MT and the WT. GO analysis indicated that these genes predicted to contain specific InDels were more commonly associated with binding than with other functionality in this study. This may suggest that proteins with the function of binding may play a significant role in the early flowering process of the MT. In addition, the significantly differentially expressed genes contain specific InDels (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw017_Supplementary_Data.zip?Expires=2147483647&Signature=jOq1gW33zqr~sOP~x1PPMy7TWT-WE38Xi5v9CPg0rQt352NU0y9xhSjCw3WsXLLAtsK0jHDOQJxNEdZJ8jhPPjACEIohOU-hA3pzrL6uTBH6WIBYvQ8oVhrnSioL~584jG08OGDb~-gkgpObmdTHqZxFw7TMV9IHS2HvH027DpWDeNDJ2vAM0oKpco3dLlEx2TI1eoJA7J4IRbX1weIVLLuwo7rZasuqEKt08yl0IzQoaKVkpBnMh43yooTJkc-L5vb4rQUWbMWRf6a4vPo7xWgG~HzDTex4qGqvSpayOl3NU96NlSVyAVP0bG1~VX5H9F9Z~LooXz07P3ffj85Ryw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>) that can be directly linked to trifoliate orange flowering transition. Many known transcription factors related to flowering regulation were also differentially expressed, including MADS-box, ring zinc finger, MYB, and GRAS (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw017_Supplementary_Data.zip?Expires=2147483647&Signature=jOq1gW33zqr~sOP~x1PPMy7TWT-WE38Xi5v9CPg0rQt352NU0y9xhSjCw3WsXLLAtsK0jHDOQJxNEdZJ8jhPPjACEIohOU-hA3pzrL6uTBH6WIBYvQ8oVhrnSioL~584jG08OGDb~-gkgpObmdTHqZxFw7TMV9IHS2HvH027DpWDeNDJ2vAM0oKpco3dLlEx2TI1eoJA7J4IRbX1weIVLLuwo7rZasuqEKt08yl0IzQoaKVkpBnMh43yooTJkc-L5vb4rQUWbMWRf6a4vPo7xWgG~HzDTex4qGqvSpayOl3NU96NlSVyAVP0bG1~VX5H9F9Z~LooXz07P3ffj85Ryw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). The transcript levels of these transcription factors were higher during the phase stage when floral buds began flowering compared with the WT, suggesting that its main role was associated with flowering time in the MT. These differentially expressed genes covered a broad range of genes related to flowering regulation, providing helpful information for understanding the genetic mechanisms underlying the signalling and regulation of the transition from the vegetative to the reproductive phase.</p><p>However, structural alterations of genes are generally believed to modify their function and expression. We found that at least 2,288 and 2,260 specific SVs showed changes in the CDS region of the MT and the WT, respectively. Although further studies are required to identify the specific gene(s) responsible for the early flowering trait, the genes could be interesting candidates for investigation in further studies. In addition, we determined whether the genes related to the flowering pathway controlling flowering time were significantly altered in the MT genome. Annotation analysis showed that 1,073 SNPs, 382 InDels, and 50 SVs were located in 272 flowering-related transcripts, respectively. This finding might suggest that the early flowering of the MT was correlated to genetic variations of these flowering pathway genes. An alternative hypothesis is that the phenotype was caused by the changes in the function of the genes through protein structure alterations. Differences in the expression patterns of flowering pathway genes were observed in the MT compared with the WT, and this might support this hypothesis. Although the exact function of these mutated genes, especially with respect to flowering, was not determined, these genes might be interesting candidates for further studies.</p><p>Recently, a large number of InDels, SNPs, and SVs have been generated using the genome resequencing platform in citrus.<sup><a reveal-id="DSW017C9" data-open="DSW017C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSW017C40" data-open="DSW017C40" class="link link-ref link-reveal xref-bibr">40</a>,<a reveal-id="DSW017C41" data-open="DSW017C41" class="link link-ref link-reveal xref-bibr">41</a></sup> These genetic variation markers may be assayed using the same separation and detection technologies as simple sequence repeat (SSR) markers. In fact, some InDels may be caused by SSRs. <em>Citrus trifoliata</em> (representing <em>P. trifoliata</em>) has consistently been one of the most important rootstock species used in the citrus industry, and it has even been used as a model species for citrus molecular biology and genomic studies. These new InDel markers are specific to trifoliate orange. Therefore, they will play important roles in marker-assisted selection and citrus breeding. In addition, these markers may be related to the flowering they ascribed to their high distribution, and whole-genome polymorphisms have been applied to high-resolution genetic mapping and map-based cloning in flowering genes.<sup><a reveal-id="DSW017C48" data-open="DSW017C48" class="link link-ref link-reveal xref-bibr">48</a>,<a reveal-id="DSW017C49" data-open="DSW017C49" class="link link-ref link-reveal xref-bibr">49</a></sup> However, the usefulness of genetic variation has not been explored in genetic and genomic research for citrus. To verify that these genetic variations were suitable for use as new DNA markers, they were used to successfully design PCR primers. In this study, we selected 268 InDels to develop PCR-based markers, of which 74 (26.7%) were polymorphic either between the two resequenced trifoliate orange or among 30 accessions from different subspecies of citrus, indicating that the bioinformatics tools were inadequate for detecting InDels in the trifoliate orange genome. However, the genetic variation will be useful to select candidate genetic variation that could be associated with unique phenotypes or agronomical traits in citrus. Furthermore, our collection of genetic variations that differentiate the MT from the WT can be used to guide the search for pure trifoliate orange types (or to recognize other cryptic species) among the hundreds of known cultivars and other germplasm accessions. More importantly, because the coordinates of these loci are known in relation to a reference genome, it is possible to develop genetic markers within specific genome regions to assist in efficient construction of genetic maps in the future.</p>                            <h2 scrollto-destination=87197488 id="87197488" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This research was supported financially by the National Natural Science Foundation of China (31130046, 31471863, 31372046, 31221062, and 31101528), the Fundamental Research Funds for the Central Universities (2013PY083), and the International Foundation for Science (C/5148-2). Funding to pay the Open Access publication charges for this article was provided by the National Science Foundation of China (grant no. 31130046).</p>                            <h2 scrollto-destination=87197490 id="87197490" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are grateful to Yong-Ping Li for his helpful discussion and help in bioinformatics analysis during revising this manuscript.</p>                            <h2 scrollto-destination=87197492 id="87197492" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSW017C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Xi</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Shen</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Tan</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">H.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Regulation of floral patterning by flowering time genes</div>, <div class="source">Dev. Cell</div> , <div class="volume">16</div>, <div class="fpage">711</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20floral%20patterning%20by%20flowering%20time%20genes&amp;author=C.%20Liu&amp;author=W.%20Xi&amp;author=L.%20Shen&amp;author=C.%20Tan&amp;author=H.%20Yu&amp;publication_year=2009&amp;journal=Dev.%20Cell&amp;volume=16&amp;pages=711-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.devcel.2009.03.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.devcel.2009.03.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.devcel.2009.03.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19460347" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Khan</div>
               <div class="given-names">M.R.G.</div>
            </div>, <div class="name string-name"><div class="surname">Ai</div> <div class="given-names">X.-Y.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">J.-Z.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Genetic regulation of flowering time in annual and perennial plants</div>, <div class="source">Wiley Interdiscip. Rev. RNA</div> , <div class="volume">5</div>, <div class="fpage">347</div>–<div class="lpage">59</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20regulation%20of%20flowering%20time%20in%20annual%20and%20perennial%20plants&amp;author=M.R.G.%20Khan&amp;author=X.-Y.%20Ai&amp;author=J.-Z.%20Zhang&amp;publication_year=2014&amp;journal=Wiley%20Interdiscip.%20Rev.%20RNA&amp;volume=5&amp;pages=347-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/wrna.2014.5.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fwrna.2014.5.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fwrna.2014.5.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24357620" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mouradov</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Cremer</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Coupland</div> <div class="given-names">G.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">Control of flowering time interacting pathways as a basis for diversity</div>, <div class="source">Plant Cell</div> , <div class="volume">14</div><div class="issue">(Suppl. 1)</div>, <div class="fpage">S111</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Control%20of%20flowering%20time%20interacting%20pathways%20as%20a%20basis%20for%20diversity&amp;author=A.%20Mouradov&amp;author=F.%20Cremer&amp;author=G.%20Coupland&amp;publication_year=2002&amp;journal=Plant%20Cell&amp;volume=14&amp;pages=S111-30" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12045273" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Boss</div>
               <div class="given-names">P.K.</div>
            </div>, <div class="name string-name"><div class="surname">Bastow</div> <div class="given-names">R.M.</div></div>, <div class="name string-name"><div class="surname">Mylne</div> <div class="given-names">J.S.</div></div>, <div class="name string-name"><div class="surname">Dean</div> <div class="given-names">C.</div></div></div>
          <div class="year">2004</div>, <div class="article-title">Multiple pathways in the decision to flower: enabling, promoting, and resetting</div>, <div class="source">Plant Cell</div> , <div class="volume">16</div><div class="issue">(Suppl. 1)</div>, <div class="fpage">S18</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20pathways%20in%20the%20decision%20to%20flower%3A%20enabling%2C%20promoting%2C%20and%20resetting&amp;author=P.K.%20Boss&amp;author=R.M.%20Bastow&amp;author=J.S.%20Mylne&amp;author=C.%20Dean&amp;publication_year=2004&amp;journal=Plant%20Cell&amp;volume=16&amp;pages=S18-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.015958" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.015958" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.015958"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15037730" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ferrario</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Immink</div> <div class="given-names">R.G.</div></div>, <div class="name string-name"><div class="surname">Angenent</div> <div class="given-names">G.C.</div></div></div>
          <div class="year">2004</div>, <div class="article-title">Conservation and diversity in flower land</div>, <div class="source">Curr. Opin. Plant Biol.</div> , <div class="volume">7</div>, <div class="fpage">84</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conservation%20and%20diversity%20in%20flower%20land&amp;author=S.%20Ferrario&amp;author=R.G.%20Immink&amp;author=G.C.%20Angenent&amp;publication_year=2004&amp;journal=Curr.%20Opin.%20Plant%20Biol.&amp;volume=7&amp;pages=84-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2003.11.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2003.11.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2003.11.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14732446" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tan</div>
               <div class="given-names">F.C.</div>
            </div>, <div class="name string-name"><div class="surname">Swain</div> <div class="given-names">S.M.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">Genetics of flower initiation and development in annual and perennial plants</div>, <div class="source">Physiol. Plant.</div> , <div class="volume">128</div>, <div class="fpage">8</div>–<div class="lpage">17</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetics%20of%20flower%20initiation%20and%20development%20in%20annual%20and%20perennial%20plants&amp;author=F.C.%20Tan&amp;author=S.M.%20Swain&amp;publication_year=2006&amp;journal=Physiol.%20Plant.&amp;volume=128&amp;pages=8-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/ppl.2006.128.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fppl.2006.128.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fppl.2006.128.issue-1"> </span></div> </div></div></div></div></div><div content-id="DSW017C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tan</div>
               <div class="given-names">F.C.</div>
            </div>, <div class="name string-name"><div class="surname">Swain</div> <div class="given-names">S.M.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Functional characterization of AP3, SOC1 and WUS homologues from citrus (Citrus sinensis)</div>, <div class="source">Physiol. Plant.</div> , <div class="volume">131</div>, <div class="fpage">481</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20characterization%20of%20AP3%2C%20SOC1%20and%20WUS%20homologues%20from%20citrus%20(Citrus%20sinensis)&amp;author=F.C.%20Tan&amp;author=S.M.%20Swain&amp;publication_year=2007&amp;journal=Physiol.%20Plant.&amp;volume=131&amp;pages=481-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/ppl.2007.131.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fppl.2007.131.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fppl.2007.131.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18251886" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gmitter</div>
               <div class="given-names">F.G.</div>
              Jr
            </div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Rao</div> <div class="given-names">M.N.</div></div>, <div class="name string-name"><div class="surname">Soneji</div> <div class="given-names">J.R.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Citrus fruits</div>. In: <div class="source">Fruits and Nuts, Genome Mapping &amp; Molecular Breeding in Plants</div> . <div class="publisher-name">Springer</div>: <div class="publisher-loc">Heidelberg</div>, pp. <div class="fpage">265</div>–<div class="lpage">79</div>.</div></div></div></div><div content-id="DSW017C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wu</div>
               <div class="given-names">G.A.</div>
            </div>, <div class="name string-name"><div class="surname">Prochnik</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Jenkins</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Sequencing of diverse mandarin, pummelo and orange genomes reveals complex history of admixture during citrus domestication</div>, <div class="source">Nat. Biotechnol.</div> , <div class="volume">32</div>, <div class="fpage">656</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20of%20diverse%20mandarin%2C%20pummelo%20and%20orange%20genomes%20reveals%20complex%20history%20of%20admixture%20during%20citrus%20domestication&amp;author=G.A.%20Wu&amp;author=S.%20Prochnik&amp;author=J.%20Jenkins&amp;publication_year=2014&amp;journal=Nat.%20Biotechnol.&amp;volume=32&amp;pages=656-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2906" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2906" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2906"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24908277" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Liang</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Xiang</div> <div class="given-names">W.</div></div></div>
          <div class="year">1999</div>, <div class="article-title">Precocious trifoliate orange (Poncirus trifoliata L. Raf.) biology characteristic and its stock experiment</div>, <div class="source">ZheJiang Citrus</div> , <div class="volume">16</div>, <div class="fpage">2</div>–<div class="lpage">4</div>.</div></div></div></div><div content-id="DSW017C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">J.-Z.</div>
            </div>, <div class="name string-name"><div class="surname">Ai</div> <div class="given-names">X.-Y.</div></div>, <div class="name string-name"><div class="surname">Sun</div> <div class="given-names">L.-M.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Transcriptome profile analysis of flowering molecular processes of early flowering trifoliate orange mutant and the wild-type [Poncirus trifoliata (L.) Raf.] by massively parallel signature sequencing</div>, <div class="source">BMC Genomics</div> , <div class="volume">12</div>, <div class="fpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20profile%20analysis%20of%20flowering%20molecular%20processes%20of%20early%20flowering%20trifoliate%20orange%20mutant%20and%20the%20wild-type%20%5BPoncirus%20trifoliata%20(L.)%20Raf.%5D%20by%20massively%20parallel%20signature%20sequencing&amp;author=J.-Z.%20Zhang&amp;author=X.-Y.%20Ai&amp;author=L.-M.%20Sun&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-63" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-63" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-63"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21269450" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pang</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Deng</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">C.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">Construction of AFLP fingerprint of 36 Poncirus accessions</div>, <div class="source">Acta Horticulturae Sinica</div> , <div class="volume">30</div>, <div class="fpage">394</div>–<div class="lpage">8</div>.</div></div></div></div><div content-id="DSW017C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">J.-Z.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Z.-M.</div></div>, <div class="name string-name"><div class="surname">Yao</div> <div class="given-names">J.-L.</div></div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">C.-G.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Identification of flowering-related genes between early flowering trifoliate orange mutant and wild-type trifoliate orange (Poncirus trifoliata L. Raf.) by suppression subtraction hybridization (SSH) and macroarray</div>, <div class="source">Gene</div> , <div class="volume">430</div>, <div class="fpage">95</div>–<div class="lpage">104</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20flowering-related%20genes%20between%20early%20flowering%20trifoliate%20orange%20mutant%20and%20wild-type%20trifoliate%20orange%20(Poncirus%20trifoliata%20L.%20Raf.)%20by%20suppression%20subtraction%20hybridization%20(SSH)%20and%20macroarray&amp;author=J.-Z.%20Zhang&amp;author=Z.-M.%20Li&amp;author=J.-L.%20Yao&amp;author=C.-G.%20Hu&amp;publication_year=2009&amp;journal=Gene&amp;volume=430&amp;pages=95-104" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2008.09.023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2008.09.023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2008.09.023"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18930791" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gmitter</div>
               <div class="given-names">F.G.</div>
              Jr
            </div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Machado</div> <div class="given-names">M.A.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Citrus genomics</div>, <div class="source">Tree Genet. Genomes</div> , <div class="volume">8</div>, <div class="fpage">611</div>–<div class="lpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Citrus%20genomics&amp;author=F.G.%20Gmitter&amp;author=C.%20Chen&amp;author=M.A.%20Machado&amp;publication_year=2012&amp;journal=Tree%20Genet.%20Genomes&amp;volume=8&amp;pages=611-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11295-012-0499-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11295-012-0499-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11295-012-0499-2"> </span></div> </div></div></div></div></div><div content-id="DSW017C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Metzker</div>
               <div class="given-names">M.L.</div>
            </div>
          </div>
          <div class="year">2010</div>, <div class="article-title">Sequencing technologies-the next generation</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">11</div>, <div class="fpage">31</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20technologies-the%20next%20generation&amp;author=M.L.%20Metzker&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=11&amp;pages=31-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2626" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2626" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2626"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19997069" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">Z.-M.</div>
            </div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">J.-Z.</div></div>, <div class="name string-name"><div class="surname">Mei</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Deng</div> <div class="given-names">X.-X.</div></div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">C.-G.</div></div>, <div class="name string-name"><div class="surname">Yao</div> <div class="given-names">J.-L.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">PtSVP, an SVP homolog from trifoliate orange (Poncirus trifoliata L. Raf.), shows seasonal periodicity of meristem determination and affects flower development in transgenic Arabidopsis and tobacco plants</div>, <div class="source">Plant Mol. Biol.</div> , <div class="volume">74</div>, <div class="fpage">129</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PtSVP%2C%20an%20SVP%20homolog%20from%20trifoliate%20orange%20(Poncirus%20trifoliata%20L.%20Raf.)%2C%20shows%20seasonal%20periodicity%20of%20meristem%20determination%20and%20affects%20flower%20development%20in%20transgenic%20Arabidopsis%20and%20tobacco%20plants&amp;author=Z.-M.%20Li&amp;author=J.-Z.%20Zhang&amp;author=L.%20Mei&amp;author=X.-X.%20Deng&amp;author=C.-G.%20Hu&amp;author=J.-L.%20Yao&amp;publication_year=2010&amp;journal=Plant%20Mol.%20Biol.&amp;volume=74&amp;pages=129-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-010-9660-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-010-9660-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-010-9660-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20602150" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cheng</div>
               <div class="given-names">Y.-J.</div>
            </div>, <div class="name string-name"><div class="surname">Guo</div> <div class="given-names">W.-W.</div></div>, <div class="name string-name"><div class="surname">Yi</div> <div class="given-names">H.-L.</div></div>, <div class="name string-name"><div class="surname">Pang</div> <div class="given-names">X.-M.</div></div>, <div class="name string-name"><div class="surname">Deng</div> <div class="given-names">X.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">An efficient protocol for genomic DNA extraction from Citrus species</div>, <div class="source">Plant Mol. Biol. Rep.</div> , <div class="volume">21</div>, <div class="fpage">177</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20efficient%20protocol%20for%20genomic%20DNA%20extraction%20from%20Citrus%20species&amp;author=Y.-J.%20Cheng&amp;author=W.-W.%20Guo&amp;author=H.-L.%20Yi&amp;author=X.-M.%20Pang&amp;author=X.%20Deng&amp;publication_year=2003&amp;journal=Plant%20Mol.%20Biol.%20Rep.&amp;volume=21&amp;pages=177-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02774246" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02774246" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02774246"> </span></div> </div></div></div></div></div><div content-id="DSW017C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Datta</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Datta</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Chakraborty</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Gill</div> <div class="given-names">R.S.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Statistical analyses of next generation sequence data: a partial overview</div>, <div class="source">J. Proteomics Bioinform.</div> , <div class="volume">3</div>, <div class="fpage">183</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistical%20analyses%20of%20next%20generation%20sequence%20data%3A%20a%20partial%20overview&amp;author=S.%20Datta&amp;author=S.%20Datta&amp;author=S.%20Kim&amp;author=S.%20Chakraborty&amp;author=R.S.%20Gill&amp;publication_year=2010&amp;journal=J.%20Proteomics%20Bioinform.&amp;volume=3&amp;pages=183-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4172/jpb" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4172%2Fjpb" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4172%2Fjpb"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21113236" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Fast and accurate long-read alignment with Burrows–Wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">589</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20long-read%20alignment%20with%20Burrows%E2%80%93Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=589-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp698" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp698" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp698"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20080505" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Handsaker</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Wysoker</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">The sequence alignment/map format and SAMtools</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">2078</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20sequence%20alignment%2Fmap%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Fang</div> <div class="given-names">X.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">SNP detection for massively parallel whole-genome resequencing</div>, <div class="source">Genome Res.</div> , <div class="volume">19</div>, <div class="fpage">1124</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20detection%20for%20massively%20parallel%20whole-genome%20resequencing&amp;author=R.%20Li&amp;author=Y.%20Li&amp;author=X.%20Fang&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=1124-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.088013.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.088013.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.088013.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19420381" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">SOAP2: an improved ultrafast tool for short read alignment</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1966</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SOAP2%3A%20an%20improved%20ultrafast%20tool%20for%20short%20read%20alignment&amp;author=R.%20Li&amp;author=C.%20Yu&amp;author=Y.%20Li&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1966-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp336" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp336" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp336"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19497933" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Birney</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Clamp</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div>
          <div class="year">2004</div>, <div class="article-title">GeneWise and genomewise</div>, <div class="source">Genome Res.</div> , <div class="volume">14</div>, <div class="fpage">988</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GeneWise%20and%20genomewise&amp;author=E.%20Birney&amp;author=M.%20Clamp&amp;author=R.%20Durbin&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=988-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1865504" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1865504" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1865504"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15123596" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">J.-Z.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Z.-M.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Mei</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Yao</div> <div class="given-names">J.-L.</div></div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">C.-G.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">Identification of early-flower-related ESTs in an early-flowering mutant of trifoliate orange (Poncirus trifoliata) by suppression subtractive hybridization and macroarray analysis</div>, <div class="source">Tree Physiol.</div> , <div class="volume">28</div>, <div class="fpage">1449</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20early-flower-related%20ESTs%20in%20an%20early-flowering%20mutant%20of%20trifoliate%20orange%20(Poncirus%20trifoliata)%20by%20suppression%20subtractive%20hybridization%20and%20macroarray%20analysis&amp;author=J.-Z.%20Zhang&amp;author=Z.-M.%20Li&amp;author=L.%20Liu&amp;author=L.%20Mei&amp;author=J.-L.%20Yao&amp;author=C.-G.%20Hu&amp;publication_year=2008&amp;journal=Tree%20Physiol.&amp;volume=28&amp;pages=1449-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/treephys/28.10.1449" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Ftreephys%2F28.10.1449" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Ftreephys%2F28.10.1449"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18708326" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zenoni</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Ferrarini</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Giacomelli</div> <div class="given-names">E.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Characterization of transcriptional complexity during berry development in Vitis vinifera using RNA-Seq</div>, <div class="source">Plant Physiol.</div> , <div class="volume">152</div>, <div class="fpage">1787</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20transcriptional%20complexity%20during%20berry%20development%20in%20Vitis%20vinifera%20using%20RNA-Seq&amp;author=S.%20Zenoni&amp;author=A.%20Ferrarini&amp;author=E.%20Giacomelli&amp;publication_year=2010&amp;journal=Plant%20Physiol.&amp;volume=152&amp;pages=1787-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.109.149716" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.149716" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.109.149716"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20118272" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schmid</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Blaxter</div> <div class="given-names">M.L.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">annot8r: GO, EC and KEGG annotation of EST datasets</div>, <div class="source">BMC Bioinform.</div> , <div class="volume">9</div>, <div class="fpage">180</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=annot8r%3A%20GO%2C%20EC%20and%20KEGG%20annotation%20of%20EST%20datasets&amp;author=R.%20Schmid&amp;author=M.L.%20Blaxter&amp;publication_year=2008&amp;journal=BMC%20Bioinform.&amp;volume=9&amp;pages=180" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-9-180" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-9-180" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-9-180"> </span></div> </div></div></div></div></div><div content-id="DSW017C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">S.-R.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">W.-Y.</div></div>, <div class="name string-name"><div class="surname">Long</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">C.-G.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">J.-Z.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Development and characterization of genomic and expressed SSRs in citrus by genome-wide analysis</div>, <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e75149</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20characterization%20of%20genomic%20and%20expressed%20SSRs%20in%20citrus%20by%20genome-wide%20analysis&amp;author=S.-R.%20Liu&amp;author=W.-Y.%20Li&amp;author=D.%20Long&amp;author=C.-G.%20Hu&amp;author=J.-Z.%20Zhang&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e75149" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0075149" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0075149" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0075149"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24204572" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yeh</div>
               <div class="given-names">F.C.</div>
            </div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Boyle</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Ye</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Mao</div> <div class="given-names">J.X.</div></div></div>
          <div class="year">1999</div>, <div class="comment">POPGENE, version 1.32: the user friendly software for population genetic analysis. Molecular Biology and Biotechnology Centre, University of Alberta, Edmonton, AB, Canada</div>.</div></div></div></div><div content-id="DSW017C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kimura</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Crow</div> <div class="given-names">J.F.</div></div></div>
          <div class="year">1964</div>, <div class="article-title">The number of alleles that can be maintained in a finite population</div>, <div class="source">Genetics</div> , <div class="volume">49</div>, <div class="fpage">725</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20number%20of%20alleles%20that%20can%20be%20maintained%20in%20a%20finite%20population&amp;author=M.%20Kimura&amp;author=J.F.%20Crow&amp;publication_year=1964&amp;journal=Genetics&amp;volume=49&amp;pages=725-38" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14156929" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lewontin</div>
               <div class="given-names">R.C.</div>
            </div>
          </div>
          <div class="year">1972</div>, <div class="article-title">Testing the theory of natural selection</div>, <div class="source">Nature</div> , <div class="volume">236</div>, <div class="fpage">181</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Testing%20the%20theory%20of%20natural%20selection&amp;author=R.C.%20Lewontin&amp;publication_year=1972&amp;journal=Nature&amp;volume=236&amp;pages=181-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/236181a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F236181a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F236181a0"> </span></div> </div></div></div></div></div><div content-id="DSW017C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nei</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="year">1973</div>, <div class="article-title">Analysis of gene diversity in subdivided populations</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">70</div>, <div class="fpage">3321</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20gene%20diversity%20in%20subdivided%20populations&amp;author=M.%20Nei&amp;publication_year=1973&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=70&amp;pages=3321-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.70.12.3321" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.70.12.3321" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.70.12.3321"> </span></div> </div></div></div></div></div><div content-id="DSW017C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tamura</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Dudley</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">MEGA4: molecular evolutionary genetics analysis (MEGA) software version 4.0</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="volume">24</div>, <div class="fpage">1596</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA4%3A%20molecular%20evolutionary%20genetics%20analysis%20(MEGA)%20software%20version%204.0&amp;author=K.%20Tamura&amp;author=J.%20Dudley&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=24&amp;pages=1596-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm092" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm092" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm092"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17488738" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Geuna</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Toschi</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Bassi</div> <div class="given-names">D.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">The use of AFLP markers for cultivar identification in apricot</div>, <div class="source">Plant Breed.</div> , <div class="volume">122</div>, <div class="fpage">526</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20use%20of%20AFLP%20markers%20for%20cultivar%20identification%20in%20apricot&amp;author=F.%20Geuna&amp;author=M.%20Toschi&amp;author=D.%20Bassi&amp;publication_year=2003&amp;journal=Plant%20Breed.&amp;volume=122&amp;pages=526-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbr.2003.122.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbr.2003.122.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbr.2003.122.issue-6"> </span></div> </div></div></div></div></div><div content-id="DSW017C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kumar</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Henikoff</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ng</div> <div class="given-names">P.C.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm</div>, <div class="source">Nat. Protoc.</div> , <div class="volume">4</div>, <div class="fpage">1073</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20the%20effects%20of%20coding%20non-synonymous%20variants%20on%20protein%20function%20using%20the%20SIFT%20algorithm&amp;author=P.%20Kumar&amp;author=S.%20Henikoff&amp;author=P.C.%20Ng&amp;publication_year=2009&amp;journal=Nat.%20Protoc.&amp;volume=4&amp;pages=1073-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2009.86" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2009.86" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2009.86"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19561590" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Subbaiyan</div>
               <div class="given-names">G.K.</div>
            </div>, <div class="name string-name"><div class="surname">Waters</div> <div class="given-names">D.L.</div></div>, <div class="name string-name"><div class="surname">Katiyar</div> <div class="given-names">S.K.</div></div>, <div class="name string-name"><div class="surname">Sadananda</div> <div class="given-names">A.R.</div></div>, <div class="name string-name"><div class="surname">Vaddadi</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Henry</div> <div class="given-names">R.J.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Genome-wide DNA polymorphisms in elite indica rice inbreds discovered by whole-genome sequencing</div>, <div class="source">Plant Biotechnol. J.</div> , <div class="volume">10</div>, <div class="fpage">623</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20DNA%20polymorphisms%20in%20elite%20indica%20rice%20inbreds%20discovered%20by%20whole-genome%20sequencing&amp;author=G.K.%20Subbaiyan&amp;author=D.L.%20Waters&amp;author=S.K.%20Katiyar&amp;author=A.R.%20Sadananda&amp;author=S.%20Vaddadi&amp;author=R.J.%20Henry&amp;publication_year=2012&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=10&amp;pages=623-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1467-7652.2011.00676.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1467-7652.2011.00676.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1467-7652.2011.00676.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22222031" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Deng</div> <div class="given-names">X.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">Advances in research of citrus cultivars selected by bud mutation and the mechanism of formation of mutated characteristics</div>, <div class="source">J. Fruit Sci.</div> , <div class="volume">23</div>, <div class="fpage">871</div>–<div class="lpage">6</div>.</div></div></div></div><div content-id="DSW017C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Alkan</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Coe</div> <div class="given-names">B.P.</div></div>, <div class="name string-name"><div class="surname">Eichler</div> <div class="given-names">E.E.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Genome structural variation discovery and genotyping</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">12</div>, <div class="fpage">363</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20structural%20variation%20discovery%20and%20genotyping&amp;author=C.%20Alkan&amp;author=B.P.%20Coe&amp;author=E.E.%20Eichler&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=12&amp;pages=363-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2958" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2958" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2958"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21358748" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Vogelstein</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Papadopoulos</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Velculescu</div> <div class="given-names">V.E.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Diaz</div> <div class="given-names">L.A.</div></div>, <div class="name string-name"><div class="surname">Kinzler</div> <div class="given-names">K.W.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Cancer genome landscapes</div>, <div class="source">Science</div> , <div class="volume">339</div>, <div class="fpage">1546</div>–<div class="lpage">58</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cancer%20genome%20landscapes&amp;author=B.%20Vogelstein&amp;author=N.%20Papadopoulos&amp;author=V.E.%20Velculescu&amp;author=S.%20Zhou&amp;author=L.A.%20Diaz&amp;author=K.W.%20Kinzler&amp;publication_year=2013&amp;journal=Science&amp;volume=339&amp;pages=1546-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1235122" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1235122" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1235122"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23539594" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Khan</div>
               <div class="given-names">I.</div>
            </div>, <div class="name string-name"><div class="surname">Roose</div> <div class="given-names">M.</div></div></div>
          <div class="year">1988</div>, <div class="article-title">Frequency and characteristics of nucellar and zygotic seedlings in three cultivars of trifoliate orange</div>, <div class="source">J. Am. Soc. Hortic. Sci.</div> , <div class="volume">113</div>, <div class="fpage">105</div>–<div class="lpage">10</div>.</div></div></div></div><div content-id="DSW017C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jiao</div>
               <div class="given-names">W.-B.</div>
            </div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Xing</div> <div class="given-names">F.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Genome-wide characterization and expression analysis of genetic variants in sweet orange</div>, <div class="source">Plant J.</div> , <div class="volume">75</div>, <div class="fpage">954</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20characterization%20and%20expression%20analysis%20of%20genetic%20variants%20in%20sweet%20orange&amp;author=W.-B.%20Jiao&amp;author=D.%20Huang&amp;author=F.%20Xing&amp;publication_year=2013&amp;journal=Plant%20J.&amp;volume=75&amp;pages=954-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.12254" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.12254" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.12254"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23738603" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Xu</div>
               <div class="given-names">Q.</div>
            </div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">L.-L.</div></div>, <div class="name string-name"><div class="surname">Ruan</div> <div class="given-names">X.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">The draft genome of sweet orange (Citrus sinensis)</div>, <div class="source">Nat. Genet.</div> , <div class="volume">45</div>, <div class="fpage">59</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20draft%20genome%20of%20sweet%20orange%20(Citrus%20sinensis)&amp;author=Q.%20Xu&amp;author=L.-L.%20Chen&amp;author=X.%20Ruan&amp;publication_year=2013&amp;journal=Nat.%20Genet.&amp;volume=45&amp;pages=59-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2472" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2472" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2472"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23179022" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Xu</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Ge</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Resequencing 50 accessions of cultivated and wild rice yields markers for identifying agronomically important genes</div>, <div class="source">Nat. Biotechnol.</div> , <div class="volume">30</div>, <div class="fpage">105</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Resequencing%2050%20accessions%20of%20cultivated%20and%20wild%20rice%20yields%20markers%20for%20identifying%20agronomically%20important%20genes&amp;author=X.%20Xu&amp;author=X.%20Liu&amp;author=S.%20Ge&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol.&amp;volume=30&amp;pages=105-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2050" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2050" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2050"> </span></div> </div></div></div></div></div><div content-id="DSW017C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kobayashi</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Nagasaki</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Garcia</div> <div class="given-names">V.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Genome-wide analysis of intraspecific DNA polymorphism in ‘Micro-Tom’, a model cultivar of tomato (Solanum lycopersicum)</div>, <div class="source">Plant Cell Physiol.</div> , <div class="volume">55</div>, <div class="fpage">445</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20intraspecific%20DNA%20polymorphism%20in%20%E2%80%98Micro-Tom%E2%80%99%2C%20a%20model%20cultivar%20of%20tomato%20(Solanum%20lycopersicum)&amp;author=M.%20Kobayashi&amp;author=H.%20Nagasaki&amp;author=V.%20Garcia&amp;publication_year=2014&amp;journal=Plant%20Cell%20Physiol.&amp;volume=55&amp;pages=445-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pct181" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpct181" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpct181"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24319074" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Clark</div>
               <div class="given-names">R.M.</div>
            </div>, <div class="name string-name"><div class="surname">Schweikert</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Toomajian</div> <div class="given-names">C.</div></div>et al.  </div>. <div class="year">2007</div>, <div class="article-title">Common sequence polymorphisms shaping genetic diversity in Arabidopsis thaliana</div>, <div class="source">Science</div> , <div class="volume">317</div>, <div class="fpage">338</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Common%20sequence%20polymorphisms%20shaping%20genetic%20diversity%20in%20Arabidopsis%20thaliana&amp;author=R.M.%20Clark&amp;author=G.%20Schweikert&amp;author=C.%20Toomajian&amp;publication_year=2007&amp;journal=Science&amp;volume=317&amp;pages=338-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1138632" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1138632" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1138632"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17641193" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lai</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">X.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Genome-wide patterns of genetic variation among elite maize inbred lines</div>, <div class="source">Nat. Genet.</div> , <div class="volume">42</div>, <div class="fpage">1027</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20patterns%20of%20genetic%20variation%20among%20elite%20maize%20inbred%20lines&amp;author=J.%20Lai&amp;author=R.%20Li&amp;author=X.%20Xu&amp;publication_year=2010&amp;journal=Nat.%20Genet.&amp;volume=42&amp;pages=1027-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.684" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.684" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.684"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20972441" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mills</div>
               <div class="given-names">R.E.</div>
            </div>, <div class="name string-name"><div class="surname">Pittard</div> <div class="given-names">W.S.</div></div>, <div class="name string-name"><div class="surname">Mullaney</div> <div class="given-names">J.M.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Natural genetic variation caused by small insertions and deletions in the human genome</div>, <div class="source">Genome Res.</div> , <div class="volume">21</div>, <div class="fpage">830</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Natural%20genetic%20variation%20caused%20by%20small%20insertions%20and%20deletions%20in%20the%20human%20genome&amp;author=R.E.%20Mills&amp;author=W.S.%20Pittard&amp;author=J.M.%20Mullaney&amp;publication_year=2011&amp;journal=Genome%20Res.&amp;volume=21&amp;pages=830-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.115907.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.115907.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.115907.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21460062" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jones</div>
               <div class="given-names">D.B.</div>
            </div>, <div class="name string-name"><div class="surname">Jerry</div> <div class="given-names">D.R.</div></div>, <div class="name string-name"><div class="surname">Forêt</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Konovalov</div> <div class="given-names">D.A.</div></div>, <div class="name string-name"><div class="surname">Zenger</div> <div class="given-names">K.R.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Genome-wide SNP validation and mantle tissue transcriptome analysis in the silver-lipped pearl oyster, Pinctada maxima</div>, <div class="source">Mar. Biotechnol.</div> , <div class="volume">15</div>, <div class="fpage">647</div>–<div class="lpage">58</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20SNP%20validation%20and%20mantle%20tissue%20transcriptome%20analysis%20in%20the%20silver-lipped%20pearl%20oyster%2C%20Pinctada%20maxima&amp;author=D.B.%20Jones&amp;author=D.R.%20Jerry&amp;author=S.%20For%C3%AAt&amp;author=D.A.%20Konovalov&amp;author=K.R.%20Zenger&amp;publication_year=2013&amp;journal=Mar.%20Biotechnol.&amp;volume=15&amp;pages=647-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10126-013-9514-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10126-013-9514-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10126-013-9514-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23715808" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Arai-Kichise</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Shiwa</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Nagasaki</div> <div class="given-names">H.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Discovery of genome-wide DNA polymorphisms in a landrace cultivar of japonica rice by whole-genome sequencing</div>, <div class="source">Plant Cell Physiol.</div> , <div class="volume">52</div>, <div class="fpage">274</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovery%20of%20genome-wide%20DNA%20polymorphisms%20in%20a%20landrace%20cultivar%20of%20japonica%20rice%20by%20whole-genome%20sequencing&amp;author=Y.%20Arai-Kichise&amp;author=Y.%20Shiwa&amp;author=H.%20Nagasaki&amp;publication_year=2011&amp;journal=Plant%20Cell%20Physiol.&amp;volume=52&amp;pages=274-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcr003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcr003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcr003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21258067" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW017C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW017C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Păcurar</div>
               <div class="given-names">D.I.</div>
            </div>, <div class="name string-name"><div class="surname">Păcurar</div> <div class="given-names">M.L.</div></div>, <div class="name string-name"><div class="surname">Street</div> <div class="given-names">N.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">A collection of INDEL markers for map-based cloning in seven Arabidopsis accessions</div>, <div class="source">J. Exp. Bot.</div> , <div class="volume">63</div>, <div class="fpage">2491</div>–<div class="lpage">501</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20collection%20of%20INDEL%20markers%20for%20map-based%20cloning%20in%20seven%20Arabidopsis%20accessions&amp;author=D.I.%20P%C4%83curar&amp;author=M.L.%20P%C4%83curar&amp;author=N.%20Street&amp;publication_year=2012&amp;journal=J.%20Exp.%20Bot.&amp;volume=63&amp;pages=2491-501" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/err422" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferr422" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferr422"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22282537" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2469959&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw017/2/dsw017_Supplementary_Data.zip?Expires=2147483647&amp;Signature=jOq1gW33zqr~sOP~x1PPMy7TWT-WE38Xi5v9CPg0rQt352NU0y9xhSjCw3WsXLLAtsK0jHDOQJxNEdZJ8jhPPjACEIohOU-hA3pzrL6uTBH6WIBYvQ8oVhrnSioL~584jG08OGDb~-gkgpObmdTHqZxFw7TMV9IHS2HvH027DpWDeNDJ2vAM0oKpco3dLlEx2TI1eoJA7J4IRbX1weIVLLuwo7rZasuqEKt08yl0IzQoaKVkpBnMh43yooTJkc-L5vb4rQUWbMWRf6a4vPo7xWgG~HzDTex4qGqvSpayOl3NU96NlSVyAVP0bG1~VX5H9F9Z~LooXz07P3ffj85Ryw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">467</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000384768700009&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">6</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw017"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2469959"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000384768700009&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f4%2f403%2f2469959">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000384768700009&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (6)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F4%2F403%2F2469959" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2469959'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Identifying the genome-wide genetic variation between precocious trifoliate orange and its wild type and developing new markers for genetics research" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw017");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f4%2f403%2f2469959">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

