

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/2/145/1745392","@type":"ScholarlyArticle","name":"Analytical workflow of double-digest restriction site-associated DNA sequencing based on empirical and  in silico  optimization in tomato","datePublished":"2016-02-29","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2016-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw004","keywords":["genetic linkage map","restriction-associated DNA sequencing","single nucleotide polymorphism","tomato (<italic>Solanum lycopersicum</italic>)","<italic>in silico</italic> prediction"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/2/145/1745392","author":[{"name":"Shirasawa, Kenta","affiliation":"Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari ,  Kisarazu, Chiba  292-0818 ,  Japan","@type":"Person"},{"name":"Hirakawa, Hideki","affiliation":"Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari ,  Kisarazu, Chiba  292-0818 ,  Japan","@type":"Person"},{"name":"Isobe, Sachiko","affiliation":"Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari ,  Kisarazu, Chiba  292-0818 ,  Japan","@type":"Person"}],"description":"Abstract.  Double-digest restriction site-associated DNA sequencing (ddRAD-Seq) enables high-throughput genome-wide genotyping with next-generation sequencing t","pageStart":"145","pageEnd":"153","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/2/1/m_cover.gif?Expires=2147483647&Signature=H~-rl2t0~4icOqxuxQciOuaILaWLPqFGH3A~nroLlKomW3yjgtZ~qWZv6XD0n7MjtydtUqNoJUBHP0IS9FeSlJ7e824tplkVC9mcToG2lma9nT3MqHOrgXg-FmGm8R1Hko1D-xKC7m9goSRdTV0PpCgY514~J0A53r05NXU-LixEnWtvnkrSQlIKg2cNSOr5m2suFRYKdAKYSGWqTMps1eFybyDDmECiZJwPFJr9hkzFGuT0qtGzqG0CshylrK0bC2jCz4U60gjwvUfzNmkXc68xyAxGKYJwvJo89NhQiuWScVg0wHNDvMxjguS6KYmoQK6GthtPkW1VFz6XLn-4lQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Analytical workflow of double-digest restriction site-associated DNA sequencing based on empirical and  in silico  optimization in tomato","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/2/1/m_cover.gif?Expires=2147483647&Signature=H~-rl2t0~4icOqxuxQciOuaILaWLPqFGH3A~nroLlKomW3yjgtZ~qWZv6XD0n7MjtydtUqNoJUBHP0IS9FeSlJ7e824tplkVC9mcToG2lma9nT3MqHOrgXg-FmGm8R1Hko1D-xKC7m9goSRdTV0PpCgY514~J0A53r05NXU-LixEnWtvnkrSQlIKg2cNSOr5m2suFRYKdAKYSGWqTMps1eFybyDDmECiZJwPFJr9hkzFGuT0qtGzqG0CshylrK0bC2jCz4U60gjwvUfzNmkXc68xyAxGKYJwvJo89NhQiuWScVg0wHNDvMxjguS6KYmoQK6GthtPkW1VFz6XLn-4lQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Analytical workflow of double-digest restriction site-associated DNA sequencing based on empirical and  in silico  optimization in tomato" />
    <meta property="og:description" content="Abstract.  Double-digest restriction site-associated DNA sequencing (ddRAD-Seq) enables high-throughput genome-wide genotyping with next-generation sequencing t" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/2/145/1745392" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-02-29" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Shirasawa, Kenta" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba292-0818, Japan" /><meta name="citation_author" content="Hirakawa, Hideki" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba292-0818, Japan" /><meta name="citation_author" content="Isobe, Sachiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba292-0818, Japan" /><meta name="citation_title" content="Analytical workflow of double-digest restriction site-associated DNA sequencing based on empirical and in silico optimization in tomato" /><meta name="citation_firstpage" content="145" /><meta name="citation_lastpage" content="153" /><meta name="citation_doi" content="10.1093/dnares/dsw004" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2016/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Detection of specific polymerase chain reaction product by utilizing the 5′ -&gt; 3′ exonuclease activity of Thermus aquaticus DNA polymerase;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=1991;  citation_volume=88;  citation_pages=7276-80; " /><meta name="citation_reference" content="citation_title=Array-based high-throughput DNA markers for crop improvement;  citation_journal_title=Heredity;  citation_year=2008;  citation_volume=101;  citation_pages=5-18; " /><meta name="citation_reference" content="citation_title=SNP array – a powerful platform to accelerate genetic studies and breeding;  citation_journal_title=J. Plant Biochem. Physiol.;  citation_year=2014;  citation_volume=2;  citation_pages=e119" /><meta name="citation_reference" content="citation_title=SNP discovery through next-generation sequencing and its applications;  citation_journal_title=Int. J. Plant Genomics;  citation_year=2012;  citation_volume=20,12;  citation_pages=831460" /><meta name="citation_reference" content="citation_title=Genome-wide genetic marker discovery and genotyping using next-generation sequencing;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2011;  citation_volume=12;  citation_pages=499-510; " /><meta name="citation_reference" content="citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species;  citation_journal_title=PLoS ONE;  citation_year=2011;  citation_volume=6;  citation_pages=e19379" /><meta name="citation_reference" content="citation_title=Genotyping-by-sequencing for plant breeding and genetics;  citation_journal_title=Plant Genome;  citation_year=2012;  citation_volume=5;  citation_pages=92-102; " /><meta name="citation_reference" content="citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers;  citation_journal_title=PLoS ONE;  citation_year=2008;  citation_volume=3;  citation_pages=e3376" /><meta name="citation_reference" content="citation_title=Single nucleotide polymorphism identification in polyploids: a review, example, and recommendations;  citation_journal_title=Mol. Plant;  citation_year=2015;  citation_volume=8;  citation_pages=831-46; " /><meta name="citation_reference" content="citation_title=Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species;  citation_journal_title=PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e37135" /><meta name="citation_reference" content="citation_title=Stacks: building and genotyping loci de novo from short-read sequences;  citation_journal_title=G3;  citation_year=2011;  citation_volume=1;  citation_pages=171-82; " /><meta name="citation_reference" content="citation_title=TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline;  citation_journal_title=PLoS ONE;  citation_year=2014;  citation_volume=9;  citation_pages=e90346" /><meta name="citation_reference" content="citation_title=Progress, challenges and the future of crop genomes;  citation_journal_title=Curr. Opin. Plant Biol.;  citation_year=2015;  citation_volume=24;  citation_pages=71-81; " /><meta name="citation_reference" content="citation_title=The tomato genome sequence provides insights into fleshy fruit evolution; citation_author=The Tomato Genome Consortium;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=485;  citation_pages=635-41; " /><meta name="citation_reference" content="citation_title=Genome-wide association studies using single nucleotide polymorphism markers developed by re-sequencing of the genomes of cultivated tomato;  citation_journal_title=DNA Res.;  citation_year=2013;  citation_volume=20;  citation_pages=593-603; " /><meta name="citation_reference" content="citation_title=Genome-wide survey of artificial mutations induced by ethyl methanesulfonate and gamma rays in tomato;  citation_journal_title=Plant Biotechnol. J.;  citation_year=2016;  citation_volume=14;  citation_pages=51-60; " /><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=357-9; " /><meta name="citation_reference" content="citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome;  citation_journal_title=Genome Biol.;  citation_year=2009;  citation_volume=10;  citation_pages=R25" /><meta name="citation_reference" content="citation_title=The sequence alignment/map format and SAMtools;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; " /><meta name="citation_reference" content="citation_title=BEDTools: a flexible suite of utilities for comparing genomic features;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=841-2; " /><meta name="citation_reference" content="citation_title=The variant call format and VCFtools;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=2156-8; " /><meta name="citation_reference" content="citation_title=Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering;  citation_journal_title=Am. J. Hum. Genet.;  citation_year=2007;  citation_volume=81;  citation_pages=1084-97; " /><meta name="citation_reference" content="citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3;  citation_journal_title=Fly;  citation_year=2012;  citation_volume=6;  citation_pages=80-92; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs;  citation_journal_title=Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_publisher=Kyazma BV, Wageningen;  citation_title=JoinMap®4, Software for the Calculation of Genetic Linkage Maps in Experimental Populations;  citation_year=2006; " /><meta name="citation_reference" content="citation_title=MapChart: software for the graphical presentation of linkage maps and QTLs;  citation_journal_title=J. Hered.;  citation_year=2002;  citation_volume=93;  citation_pages=77-8; " /><meta name="citation_reference" content="citation_title=Fluorescence in situ hybridization and optical mapping to correct scaffold arrangement in the tomato genome;  citation_journal_title=G3;  citation_year=2014;  citation_volume=4;  citation_pages=1395-405; " /><meta name="citation_reference" content="citation_title=Genome-wide SNP genotyping to infer the effects on gene functions in tomato;  citation_journal_title=DNA Res.;  citation_year=2013;  citation_volume=20;  citation_pages=221-33; " /><meta name="citation_reference" content="citation_title=Exploring genetic variation in the tomato (Solanum section Lycopersicon) clade by whole-genome sequencing; citation_author=The 100 Tomato Genome Sequencing Consortium;  citation_journal_title=Plant J.;  citation_year=2014;  citation_volume=80;  citation_pages=136-48; " /><meta name="citation_reference" content="citation_title=Patchwork sequencing of tomato San Marzano and Vesuviano varieties highlights genome-wide variations;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=138" /><meta name="citation_reference" content="citation_title=Target-enrichment strategies for next-generation sequencing;  citation_journal_title=Nat. Methods;  citation_year=2010;  citation_volume=7;  citation_pages=111-8; " /><meta name="citation_reference" content="citation_title=DNA marker applications to molecular genetics and genomics in tomato;  citation_journal_title=Breed. Sci.;  citation_year=2013;  citation_volume=63;  citation_pages=21-30; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/2/145/17463142/dsw004.pdf" /><meta name="description" content="Abstract.  Double-digest restriction site-associated DNA sequencing (ddRAD-Seq) enables high-throughput genome-wide genotyping with next-generation sequencing t" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/2/145/1745392" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/2/145/1745392" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/2/145/1745392");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw004';
    </script>
<title>
	Analytical workflow of double-digest restriction site-associated DNA sequencing based on empirical and in silico optimization in tomato | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./1745392" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYBMPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGlyYXNhd2EsIEtlbnRhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGlyYWthd2EsIEhpZGVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYTI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Iklzb2JlLCBTYWNoaWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJBbmFseXRpY2FsIHdvcmtmbG93IG9mIGRvdWJsZS1kaWdlc3QgcmVzdHJpY3Rpb24gc2l0ZS1hc3NvY2lhdGVkIEROQSBzZXF1ZW5jaW5nIGJhc2VkIG9uIGVtcGlyaWNhbCBhbmQgaW4gc2lsaWNvIG9wdGltaXphdGlvbiBpbiB0b21hdG8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxNDUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjE1MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdzAwNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE2LzA0LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV0ZWN0aW9uIG9mIHNwZWNpZmljIHBvbHltZXJhc2UgY2hhaW4gcmVhY3Rpb24gcHJvZHVjdCBieSB1dGlsaXppbmcgdGhlIDXigLIgLSZndDsgM+KAsiBleG9udWNsZWFzZSBhY3Rpdml0eSBvZiBUaGVybXVzIGFxdWF0aWN1cyBETkEgcG9seW1lcmFzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9ODg7ICBjaXRhdGlvbl9wYWdlcz03Mjc2LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcnJheS1iYXNlZCBoaWdoLXRocm91Z2hwdXQgRE5BIG1hcmtlcnMgZm9yIGNyb3AgaW1wcm92ZW1lbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUhlcmVkaXR5OyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTUtMTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNOUCBhcnJheSDigJMgYSBwb3dlcmZ1bCBwbGF0Zm9ybSB0byBhY2NlbGVyYXRlIGdlbmV0aWMgc3R1ZGllcyBhbmQgYnJlZWRpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIFBsYW50IEJpb2NoZW0uIFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz1lMTE5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U05QIGRpc2NvdmVyeSB0aHJvdWdoIG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIGFuZCBpdHMgYXBwbGljYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbnQuIEouIFBsYW50IEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTIwLDEyOyAgY2l0YXRpb25fcGFnZXM9ODMxNDYwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgZ2VuZXRpYyBtYXJrZXIgZGlzY292ZXJ5IGFuZCBnZW5vdHlwaW5nIHVzaW5nIG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NDk5LTUxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSByb2J1c3QsIHNpbXBsZSBnZW5vdHlwaW5nLWJ5LXNlcXVlbmNpbmcgKEdCUykgYXBwcm9hY2ggZm9yIGhpZ2ggZGl2ZXJzaXR5IHNwZWNpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1lMTkzNzkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vdHlwaW5nLWJ5LXNlcXVlbmNpbmcgZm9yIHBsYW50IGJyZWVkaW5nIGFuZCBnZW5ldGljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz05Mi0xMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhcGlkIFNOUCBkaXNjb3ZlcnkgYW5kIGdlbmV0aWMgbWFwcGluZyB1c2luZyBzZXF1ZW5jZWQgUkFEIG1hcmtlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz1lMzM3NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbSBpZGVudGlmaWNhdGlvbiBpbiBwb2x5cGxvaWRzOiBhIHJldmlldywgZXhhbXBsZSwgYW5kIHJlY29tbWVuZGF0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBQbGFudDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ODMxLTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Eb3VibGUgZGlnZXN0IFJBRHNlcTogYW4gaW5leHBlbnNpdmUgbWV0aG9kIGZvciBkZSBub3ZvIFNOUCBkaXNjb3ZlcnkgYW5kIGdlbm90eXBpbmcgaW4gbW9kZWwgYW5kIG5vbi1tb2RlbCBzcGVjaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTM3MTM1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RhY2tzOiBidWlsZGluZyBhbmQgZ2Vub3R5cGluZyBsb2NpIGRlIG5vdm8gZnJvbSBzaG9ydC1yZWFkIHNlcXVlbmNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RzM7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTE3MS04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VEFTU0VMLUdCUzogYSBoaWdoIGNhcGFjaXR5IGdlbm90eXBpbmcgYnkgc2VxdWVuY2luZyBhbmFseXNpcyBwaXBlbGluZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPWU5MDM0NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb2dyZXNzLCBjaGFsbGVuZ2VzIGFuZCB0aGUgZnV0dXJlIG9mIGNyb3AgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz03MS04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHRvbWF0byBnZW5vbWUgc2VxdWVuY2UgcHJvdmlkZXMgaW5zaWdodHMgaW50byBmbGVzaHkgZnJ1aXQgZXZvbHV0aW9uOyBjaXRhdGlvbl9hdXRob3I9VGhlIFRvbWF0byBHZW5vbWUgQ29uc29ydGl1bTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ4NTsgIGNpdGF0aW9uX3BhZ2VzPTYzNS00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgYXNzb2NpYXRpb24gc3R1ZGllcyB1c2luZyBzaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc20gbWFya2VycyBkZXZlbG9wZWQgYnkgcmUtc2VxdWVuY2luZyBvZiB0aGUgZ2Vub21lcyBvZiBjdWx0aXZhdGVkIHRvbWF0bzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz01OTMtNjAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBzdXJ2ZXkgb2YgYXJ0aWZpY2lhbCBtdXRhdGlvbnMgaW5kdWNlZCBieSBldGh5bCBtZXRoYW5lc3VsZm9uYXRlIGFuZCBnYW1tYSByYXlzIGluIHRvbWF0bzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQmlvdGVjaG5vbC4gSi47ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz01MS02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdCBnYXBwZWQtcmVhZCBhbGlnbm1lbnQgd2l0aCBCb3d0aWUgMjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0zNTctOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VWx0cmFmYXN0IGFuZCBtZW1vcnktZWZmaWNpZW50IGFsaWdubWVudCBvZiBzaG9ydCBETkEgc2VxdWVuY2VzIHRvIHRoZSBodW1hbiBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPVIyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBzZXF1ZW5jZSBhbGlnbm1lbnQvbWFwIGZvcm1hdCBhbmQgU0FNdG9vbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MjA3OC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CRURUb29sczogYSBmbGV4aWJsZSBzdWl0ZSBvZiB1dGlsaXRpZXMgZm9yIGNvbXBhcmluZyBnZW5vbWljIGZlYXR1cmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTg0MS0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgdmFyaWFudCBjYWxsIGZvcm1hdCBhbmQgVkNGdG9vbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MjE1Ni04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXBpZCBhbmQgYWNjdXJhdGUgaGFwbG90eXBlIHBoYXNpbmcgYW5kIG1pc3NpbmctZGF0YSBpbmZlcmVuY2UgZm9yIHdob2xlLWdlbm9tZSBhc3NvY2lhdGlvbiBzdHVkaWVzIGJ5IHVzZSBvZiBsb2NhbGl6ZWQgaGFwbG90eXBlIGNsdXN0ZXJpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFtLiBKLiBIdW0uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT04MTsgIGNpdGF0aW9uX3BhZ2VzPTEwODQtOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcHJvZ3JhbSBmb3IgYW5ub3RhdGluZyBhbmQgcHJlZGljdGluZyB0aGUgZWZmZWN0cyBvZiBzaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc21zLCBTbnBFZmY6IFNOUHMgaW4gdGhlIGdlbm9tZSBvZiBEcm9zb3BoaWxhIG1lbGFub2dhc3RlciBzdHJhaW4gdzExMTg7IGlzby0yOyBpc28tMzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Rmx5OyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz04MC05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2FwcGVkIEJMQVNUIGFuZCBQU0ktQkxBU1Q6IGEgbmV3IGdlbmVyYXRpb24gb2YgcHJvdGVpbiBkYXRhYmFzZSBzZWFyY2ggcHJvZ3JhbXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTMzODktNDAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9S3lhem1hIEJWLCBXYWdlbmluZ2VuOyAgY2l0YXRpb25fdGl0bGU9Sm9pbk1hcMKuNCwgU29mdHdhcmUgZm9yIHRoZSBDYWxjdWxhdGlvbiBvZiBHZW5ldGljIExpbmthZ2UgTWFwcyBpbiBFeHBlcmltZW50YWwgUG9wdWxhdGlvbnM7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcENoYXJ0OiBzb2Z0d2FyZSBmb3IgdGhlIGdyYXBoaWNhbCBwcmVzZW50YXRpb24gb2YgbGlua2FnZSBtYXBzIGFuZCBRVExzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBIZXJlZC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9OTM7ICBjaXRhdGlvbl9wYWdlcz03Ny04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GbHVvcmVzY2VuY2UgaW4gc2l0dSBoeWJyaWRpemF0aW9uIGFuZCBvcHRpY2FsIG1hcHBpbmcgdG8gY29ycmVjdCBzY2FmZm9sZCBhcnJhbmdlbWVudCBpbiB0aGUgdG9tYXRvIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RzM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTEzOTUtNDA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBTTlAgZ2Vub3R5cGluZyB0byBpbmZlciB0aGUgZWZmZWN0cyBvbiBnZW5lIGZ1bmN0aW9ucyBpbiB0b21hdG87ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MjIxLTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHBsb3JpbmcgZ2VuZXRpYyB2YXJpYXRpb24gaW4gdGhlIHRvbWF0byAoU29sYW51bSBzZWN0aW9uIEx5Y29wZXJzaWNvbikgY2xhZGUgYnkgd2hvbGUtZ2Vub21lIHNlcXVlbmNpbmc7IGNpdGF0aW9uX2F1dGhvcj1UaGUgMTAwIFRvbWF0byBHZW5vbWUgU2VxdWVuY2luZyBDb25zb3J0aXVtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT04MDsgIGNpdGF0aW9uX3BhZ2VzPTEzNi00ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGF0Y2h3b3JrIHNlcXVlbmNpbmcgb2YgdG9tYXRvIFNhbiBNYXJ6YW5vIGFuZCBWZXN1dmlhbm8gdmFyaWV0aWVzIGhpZ2hsaWdodHMgZ2Vub21lLXdpZGUgdmFyaWF0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MTM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFyZ2V0LWVucmljaG1lbnQgc3RyYXRlZ2llcyBmb3IgbmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MTExLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSBtYXJrZXIgYXBwbGljYXRpb25zIHRvIG1vbGVjdWxhciBnZW5ldGljcyBhbmQgZ2Vub21pY3MgaW4gdG9tYXRvOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CcmVlZC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT02MzsgIGNpdGF0aW9uX3BhZ2VzPTIxLTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjMvMi8xNDUvMTc0NjMxNDIvZHN3MDA0LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBEb3VibGUtZGlnZXN0IHJlc3RyaWN0aW9uIHNpdGUtYXNzb2NpYXRlZCBETkEgc2VxdWVuY2luZyAoZGRSQUQtU2VxKSBlbmFibGVzIGhpZ2gtdGhyb3VnaHB1dCBnZW5vbWUtd2lkZSBnZW5vdHlwaW5nIHdpdGggbmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmcgdCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MTc0NTM5MiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQxNw5hyk68gDNUKS+Kbmjj/MCeAg==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f2%2f145%2f1745392" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f2%2f145%2f1745392" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=69060" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/2/1/m_cover.gif?Expires=2147483647&amp;Signature=H~-rl2t0~4icOqxuxQciOuaILaWLPqFGH3A~nroLlKomW3yjgtZ~qWZv6XD0n7MjtydtUqNoJUBHP0IS9FeSlJ7e824tplkVC9mcToG2lma9nT3MqHOrgXg-FmGm8R1Hko1D-xKC7m9goSRdTV0PpCgY514~J0A53r05NXU-LixEnWtvnkrSQlIKg2cNSOr5m2suFRYKdAKYSGWqTMps1eFybyDDmECiZJwPFJr9hkzFGuT0qtGzqG0CshylrK0bC2jCz4U60gjwvUfzNmkXc68xyAxGKYJwvJo89NhQiuWScVg0wHNDvMxjguS6KYmoQK6GthtPkW1VFz6XLn-4lQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/2">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87199498">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199498">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199500">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199500">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199504">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199504">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199517">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199517">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199539">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199539">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199547">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199547">5. Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199549">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199549">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199551">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199551">Acknowledgments</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199553">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199553">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/2/135/1745371">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/2/155/1745431">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Analytical workflow of double-digest restriction site-associated DNA sequencing based on empirical and <em>in silico</em> optimization in tomato
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenta Shirasawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari</div>, <div class="addr-line">Kisarazu, Chiba</div><div class="addr-line">292-0818</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 438-52-3935. Fax. +81 438-52-3934. Email: <a href="mailto:shirasaw@kazusa.or.jp" target="_blank">shirasaw@kazusa.or.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenta+Shirasawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shirasawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shirasawa K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kenta Shirasawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Hirakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari</div>, <div class="addr-line">Kisarazu, Chiba</div><div class="addr-line">292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideki+Hirakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirakawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hirakawa H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hideki Hirakawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sachiko Isobe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari</div>, <div class="addr-line">Kisarazu, Chiba</div><div class="addr-line">292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sachiko+Isobe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Isobe S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Isobe S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sachiko Isobe</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Prof. Kazuhiro Sato</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 2, 1 April 2016, Pages 145–153, <a href='https://doi.org/10.1093/dnares/dsw004'>https://doi.org/10.1093/dnares/dsw004</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">29 February 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">07 December 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">26 January 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="1745392" href="/dnaresearch/article-pdf/23/2/145/17463142/dsw004.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Kenta Shirasawa, Hideki Hirakawa, Sachiko Isobe;  Analytical workflow of double-digest restriction site-associated DNA sequencing based on empirical and <em>in silico</em> optimization in tomato, <em>DNA Research</em>, Volume 23, Issue 2, 1 April 2016, Pages 145–153, <a href='https://doi.org/10.1093/dnares/dsw004'>https://doi.org/10.1093/dnares/dsw004</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/1745392?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745392?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745392?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745392?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745392?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745392?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745392?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Analytical%20workflow%20of%20double-digest%20restriction%20site-associated%20DNA%20sequencing%20based%20on%20empirical%20and%20%26lt%3Bem%26gt%3Bin%20silico%26lt%3B%2Fem%26gt%3B%20optimization%20in%20tomato&amp;publicationDate=2016-02-29&amp;volumeNum=23&amp;issueNum=2&amp;author=Shirasawa%2C%20Kenta%3B%20Hirakawa%2C%20Hideki&amp;startPage=145&amp;endPage=153&amp;contentId=10.1093%2Fdnares%2Fdsw004&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f2%2f145%2f1745392" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87199498 id="87199498" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Double-digest restriction site-associated DNA sequencing (ddRAD-Seq) enables high-throughput genome-wide genotyping with next-generation sequencing technology. Consequently, this method has become popular in plant genetics and breeding. Although computational <em>in silico</em> prediction of restriction sites from the genome sequence is recognized as an effective approach for choosing the restriction enzymes to be used, few reports have evaluated the <em>in silico</em> predictions in actual experimental data. In this study, we designed and demonstrated a workflow for <em>in silico</em> and empirical ddRAD-Seq analysis in tomato, as follows: (i) <em>in silico</em> prediction of optimum restriction enzymes from the reference genome, (ii) verification of the prediction by actual ddRAD-Seq data of four restriction enzyme combinations, (iii) establishment of a computational data processing pipeline for high-confidence single nucleotide polymorphism (SNP) calling, and (iv) validation of SNP accuracy by construction of genetic linkage maps. The quality of SNPs based on <em>de novo</em> assembly reference of the ddRAD-Seq reads was comparable with that of SNPs obtained using the published reference genome of tomato. Comparisons of SNP calls in diverse tomato lines revealed that SNP density in the genome influenced the detectability of SNPs by ddRAD-Seq. <em>In silico</em> prediction prior to actual analysis contributed to optimization of the experimental conditions for ddRAD-Seq, e.g. choices of enzymes and plant materials. Following optimization, this ddRAD-Seq pipeline could help accelerate genetics, genomics, and molecular breeding in both model and non-model plants, including crops.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genetic linkage map&quot;">genetic linkage map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;restriction-associated DNA sequencing&quot;">restriction-associated DNA sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;single nucleotide polymorphism&quot;">single nucleotide polymorphism</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;tomato (Solanum lycopersicum)&quot;">tomato (<em>Solanum lycopersicum</em>)</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;in silico prediction&quot;"><em>in silico</em> prediction</a></div>                            <h2 scrollto-destination=87199500 id="87199500" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>DNA markers are essential tools for molecular genetics and genomics. Simple sequence repeats (SSRs, also called microsatellites) and single nucleotide polymorphisms (SNPs) are the most powerful and widely used DNA markers. SSRs have the advantages of being both co-dominant and multi-allelic in nature, but they require time-consuming gel or capillary electrophoresis analyses. On the other hand, SNPs, most of which are co-dominant but bi-allelic, can be analysed using time-saving gel-free techniques, e.g. TaqMan assays,<sup><a reveal-id="DSW004C1" data-open="DSW004C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Kompetitive Allele-Specific PCR (KASP; LGC, London, UK), and high-resolution melting analysis (Idaho Technology, Salt Lake City, UT). Microarray-based SNP chip technologies, e.g. GoldenGate and Infinium (Illumina, San Diego, CA), and Axiom (Affymetrix, Santa Clara, CA), have enabled high-throughput SNP genotyping and thereby contributed to statistical genetic approaches, e.g. quantitative trait locus analyses and genome-wide association studies.<sup><a reveal-id="DSW004C2" data-open="DSW004C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSW004C3" data-open="DSW004C3" class="link link-ref link-reveal xref-bibr">3</a></sup> However, SNP microarrays have a disadvantage, namely, the lack of flexibility in experimental design. Progress in next-generation sequencing (NGS) technology has enabled the development of huge numbers of SNPs in both model and non-model plant species, including crops.<sup><a reveal-id="DSW004C4" data-open="DSW004C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Correspondingly, SNP genotyping by NGS, e.g. genotyping by sequencing (GBS) and restriction site-associated DNA sequencing (RAD-Seq), have recently become popular due to their flexibility and relatively low cost.<sup><a reveal-id="DSW004C5" data-open="DSW004C5" class="link link-ref link-reveal xref-bibr">5</a></sup></p><p>GBS was initially developed in maize<sup><a reveal-id="DSW004C6" data-open="DSW004C6" class="link link-ref link-reveal xref-bibr">6</a></sup> and subsequently applied to other crop species.<sup><a reveal-id="DSW004C7" data-open="DSW004C7" class="link link-ref link-reveal xref-bibr">7</a></sup> In the original GBS protocol, genomic DNA is digested with restriction enzymes, and adapters are ligated to the restriction ends.<sup><a reveal-id="DSW004C6" data-open="DSW004C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Sequencing data of single-end reads are always obtained from sites associated with the restriction ends, which is a great advantage in sequencing of identical loci across multiple samples. The RAD-Seq method,<sup><a reveal-id="DSW004C8" data-open="DSW004C8" class="link link-ref link-reveal xref-bibr">8</a></sup> which is similar to GBS, has been applied to several plant species.<sup><a reveal-id="DSW004C5" data-open="DSW004C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSW004C7" data-open="DSW004C7" class="link link-ref link-reveal xref-bibr">7</a></sup> In the original RAD-Seq protocol, genomic DNA is fragmented twice by different methods: first by a restriction enzyme, and second by physical shearing. The resultant DNA fragments, with restriction sites on one end and sheared ends on the other, are targeted for single-end sequencing analysis from the restriction ends. On the other hand, paired-end sequence reads can be more accurately mapped onto the reference genome than single-end reads, especially in plants, which often have large and complex polyploid genomes.<sup><a reveal-id="DSW004C9" data-open="DSW004C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Double-digest restriction site-associated DNA-Seq (ddRAD-Seq), in which a second restriction enzyme is employed for digestion of genome DNA to reduce cost and time to prepare the sequencing libraries, enables paired-end sequencing of identical loci across multiple samples.<sup><a reveal-id="DSW004C10" data-open="DSW004C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Therefore, from the point of view of high accuracy read mapping even in the complex plant genomes, ddRAD-Seq technology has the advantage over GBS and RAD-Seq. Along with the great advances in the sequencing technology, several data processing pipelines for GBS and RAD-Seq have been reported.<sup><a reveal-id="DSW004C11" data-open="DSW004C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSW004C12" data-open="DSW004C12" class="link link-ref link-reveal xref-bibr">12</a></sup> However, as mentioned, plants have complex genomes due to many types of ploidy, reproduction systems as well as various genome sizes. Therefore, data processing methods with flexibility in manipulation would be required.</p><p>Whole-genome sequencing (WGS) analysis of several plant species has been accelerated by NGS technology; as of June 2015, genome sequence data are available from &gt;100 plants.<sup><a reveal-id="DSW004C13" data-open="DSW004C13" class="link link-ref link-reveal xref-bibr">13</a></sup> This situation makes it possible to simulate ddRAD-Seq <em>in silico</em>, allowing prediction of the numbers, sizes, and genome positions of digested fragments. Based on <em>in silico</em> analysis, the optimal restriction enzymes for ddRAD-Seq analyses are chosen.<sup><a reveal-id="DSW004C10" data-open="DSW004C10" class="link link-ref link-reveal xref-bibr">10</a></sup> However, few reports have evaluated the <em>in silico</em> predictions by comparative experiments using several combinations of restriction enzymes and multiple samples with different SNP density. Moreover, it remains unclear what fraction of the SNPs in the whole genome can be detected by ddRAD-Seq. In this study, we performed <em>in silico</em> simulation of ddRAD-Seq analysis in tomato (<em>Solanum lycopersicum</em>) and validated the predictions by empirical ddRAD-Seq data using an optimized protocol. We selected tomato for this demonstration because of the richness of available genome information<sup><a reveal-id="DSW004C14" data-open="DSW004C14" class="link link-ref link-reveal xref-bibr">14</a></sup> and the diversity of available tomato lines.<sup><a reveal-id="DSW004C15" data-open="DSW004C15" class="link link-ref link-reveal xref-bibr">15</a></sup> In addition, we investigated the numbers of SNPs detected by ddRAD-Seq in six inbred tomato lines with different densities of genome-wide SNPs. To evaluate the quality of the SNPs, we performed linkage analyses of the SNPs identified in an F<sub>2</sub> mapping population and constructed genetic linkage maps. Finally, we proposed an analytical workflow for the ddRAD-Seq procedure including a pipeline for data processing.</p>                            <h2 scrollto-destination=87199504 id="87199504" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87199505 id="87199505" class="section-title" data-legacy-id=s2a>2.1. Processing data for whole-genome sequence of tomato</h3>
<p>Two tomato lines, Micro-Tom and Regina, were used as controls for empirical and <em>in silico</em> ddRAD-Seq and establishment of computational data processing pipelines. Published WGS data for Micro-Tom (accession number of DRX020765: Illumina data)<sup><a reveal-id="DSW004C16" data-open="DSW004C16" class="link link-ref link-reveal xref-bibr">16</a></sup> and Regina (accession numbers of DRX011585 and DRX011586: SOLiD data)<sup><a reveal-id="DSW004C15" data-open="DSW004C15" class="link link-ref link-reveal xref-bibr">15</a></sup> were used to generate a genome-wide SNP dataset. The WGS reads of the two lines were treated to remove low-quality reads and to trim adapters as described below (Computational processing for data from empirical ddRAD-Seq analysis), and mapped onto the tomato (cultivar Heinz 1706) reference genome sequences, version SL2.50, with Bowtie2 (version 2.2.3; parameters: -I 100 -X 500)<sup><a reveal-id="DSW004C17" data-open="DSW004C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and Bowtie (version 1.0; parameters: -l 15 -e 1,000),<sup><a reveal-id="DSW004C18" data-open="DSW004C18" class="link link-ref link-reveal xref-bibr">18</a></sup> respectively. Subsequent SNP calling was also performed as below (Computational processing for data from empirical ddRAD-Seq analysis).</p><p>The genome sequence of tomato (SL2.50; <a class="link link-uri" href="https://www.sgn.cornell.edu">https://www.sgn.cornell.edu</a>) as well as those of <em>Arabidopsis thaliana</em> (TAIR10; <a class="link link-uri" href="https://www.arabidopsis.org">https://www.arabidopsis.org</a>), <em>Lotus japonicus</em> (build 3.0; <a class="link link-uri" href="http://www.kazusa.or.jp/lotus">http://www.kazusa.or.jp/lotus</a>), and <em>Oryza sativa</em> (Os-Nipponbare-Reference-IRGSP-1.0; <a class="link link-uri" href="http://rapdb.dna.affrc.go.jp">http://rapdb.dna.affrc.go.jp</a>) were <em>in silico</em> treated with five restriction enzymes, e.g. <em>Eco</em>RI (recognition at site G↓AATTC), <em>Hin</em>dIII (A↓AGCTT), <em>Msp</em>I (C↓CGG), <em>Pst</em>I (CTGCA↓G), and <em>Sal</em>I (G↓TCGAC): the genome sequence was digested into restriction fragments at the points of the recognition sites of the enzymes, and information on sizes of each fragment was retained.</p>                            <h3 scrollto-destination=87199508 id="87199508" class="section-title" data-legacy-id=s2b>2.2. Plant materials</h3>
<p>Six inbred tomato lines (Ailsa Craig, Micro-Tom, M82, Moneymaker, Regina, and San Marzano) were used for ddRAD-Seq analysis. All lines except for Regina were obtained from the National BioResource Project through the University of Tsukuba, Japan (accession numbers: Micro-Tom, TOMJPF00001; Moneymaker, TOMJPF00002; Ailsa Craig, TOMJPF00004; and M82, TOMJPF00005) and the Tomato Genetic Resource Center, University of California, Davis, USA (San Marzano, LA3008). Regina was commercially available from Sakata Seed Corporation (Yokohama, Japan). An F<sub>2</sub> mapping population RMF2, consisting of 96 lines, was derived from a cross between Micro-Tom and Regina. Genomic DNAs were isolated from leaves of each line using the DNeasy Plant Mini Kit (Qiagen, Hilden, Germany), and quantitated using a Qubit fluorometer (Life Technologies, Carlsbad, CA, USA).</p>                            <h3 scrollto-destination=87199510 id="87199510" class="section-title" data-legacy-id=s2c>2.3. ddRAD-Seq analysis</h3>
<p>A total of 250 ng of genomic DNA for each line was double digested with <em>Sal</em>I and <em>Pst</em>I, <em>Pst</em>I and <em>Eco</em>RI, <em>Eco</em>RI and <em>Hin</em>dIII, or <em>Pst</em>I and <em>Msp</em>I (FastDigest restriction enzymes; Thermo Fisher Scientific, Waltham, MA, USA); ligated to adapters (Table <a reveal-id="DSW004TB1" data-open="DSW004TB1" class="link link-reveal link-table xref-fig">1</a>) using the LigaFast Rapid DNA Ligation System (Promega, Madison, WI, USA); and purified using Agencourt AMPure XP (Beckman Coulter, Brea, CA, USA) to eliminate short (&lt;300 bp) DNA fragments. Purified DNA was diluted with H<sub>2</sub>O and amplified by PCR with indexed primers (Table <a reveal-id="DSW004TB1" data-open="DSW004TB1" class="link link-reveal link-table xref-fig">1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw004_Supplementary_Data.zip?Expires=2147483647&Signature=qosUpqhIKaFSxMnA94HtqcgimoMsVGYfDdoaMWiECGocawcA8WW2-ahZyUM-mXfg8JhQtv9WUKzWh2kTBMBBGZtGAg9fpwYyaHF6TWEPVEkJG7PdrbIyJJbh0NsZ3nNrAiuMiFRYLkEAEEilmuSCroigb47KOYGz5RgGx5LKMu53R5HedlZeQ~BQZoGQATn4M-5aM6MVQ62vqSbBfb0dg2p~1NkL5XoLpKvHESzyIfNhpE9uFjCKPIUZrFmWSodLVBFe-mb~i6fK7c~NuTRBVGjjtPI3zXGwaq0~5WNw-Cr8UIwVnwKM0CIj9hhsNUpRedbO1gP0xxnOf0QNL8ErIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). The PCR mixture (50 µl) contained 0.4 ng of DNA, 0.2 µM of each indexed primer (one pair per mixture), 1× PCR buffer for KOD –plus– Ver. 2 (Toyobo, Osaka, Japan), 160 µM dNTPs, 1 mM MgSO<sub>4</sub>, and 1 U DNA polymerase (KOD –plus–; Toyobo). Thermal cycling conditions were as follows: a 3 min initial denaturation at 95°C; 20 cycles of 30 s of denaturation at 94°C, 30 s of annealing at 55°C, and a 60 s extension at 72°C; and a final 3 min extension at 72°C. Amplicons were pooled and separated on a BluePippin 1.5% agarose cassette (Sage Science, Beverly, MA, USA), and fragments of 300–900 bp were purified using the QIAGEN Mini Elute Kit (Qiagen). Concentrations of the resultant libraries were measured using the KAPA Library Quantification Kit (KAPA Biosystems, Wilmington, MA, USA) on an ABI-7900HT real-time PCR system (Life Technologies). Nucleotide sequences of the libraries were determined on a MiSeq (Illumina) in paired-end, 250 bp mode. </p>                        <a id="87199512" scrollto-destination="87199512"></a>
<div content-id="DSW004TB1" class="table-modal"><div class="table-wrap"><div id="DSW004TB1" class="table-wrap-title" data-id="DSW004TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Sequences of oligonucleotides used in ddRAD-Seq</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Names </th><th>Sequence (5′ – 3′) </th></tr></thead><tbody><tr><td>Restriction enzyme </td><td> </td></tr><tr><td rowspan="2"> <em>Pst</em>I </td><td>TCTTTCCCTACACGACGCTCTTCCGATCTGCA </td></tr><tr><td>GATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT </td></tr><tr><td rowspan="2"> <em>Eco</em>RI </td><td>CTGGAGTTCAGACGTGTGCTCTTCCGATCT </td></tr><tr><td>AATTAGATCGGAAGAGCACACGTCTGAACTCCAGTCAC </td></tr><tr><td rowspan="2"> <em>Hin</em>dIII </td><td>TCTTTCCCTACACGACGCTCTTCCGATCT </td></tr><tr><td>AGCTAGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT </td></tr><tr><td rowspan="2"> <em>Sal</em>I </td><td>CTGGAGTTCAGACGTGTGCTCTTCCGATC </td></tr><tr><td>TCGAGATCGGAAGAGCACACGTCTGAACTCCAGTCAC </td></tr><tr><td rowspan="2"> <em>Msp</em>I </td><td>CTGGAGTTCAGACGTGTGCTCTTCCGATCT </td></tr><tr><td>CGAGATCGGAAGAGCACACGTCTGAACTCCAGTCAC </td></tr><tr><td colspan="2">Indexed primers for PCR<sup>a</sup> </td></tr><tr><td> Forward primer </td><td>AATGATACGGCGACCACCGAGATCTACACXXXXXXXXACACTCTTTCCCTACACGACGCTCTTCC </td></tr><tr><td> Reverse primer </td><td>CAAGCAGAAGACGGCATACGAGATXXXXXXXXGTGACTGGAGTTCAGACGTGTGCTCTTC </td></tr></tbody></table></div><div content-id="DSW004TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Names </th><th>Sequence (5′ – 3′) </th></tr></thead><tbody><tr><td>Restriction enzyme </td><td> </td></tr><tr><td rowspan="2"> <em>Pst</em>I </td><td>TCTTTCCCTACACGACGCTCTTCCGATCTGCA </td></tr><tr><td>GATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT </td></tr><tr><td rowspan="2"> <em>Eco</em>RI </td><td>CTGGAGTTCAGACGTGTGCTCTTCCGATCT </td></tr><tr><td>AATTAGATCGGAAGAGCACACGTCTGAACTCCAGTCAC </td></tr><tr><td rowspan="2"> <em>Hin</em>dIII </td><td>TCTTTCCCTACACGACGCTCTTCCGATCT </td></tr><tr><td>AGCTAGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT </td></tr><tr><td rowspan="2"> <em>Sal</em>I </td><td>CTGGAGTTCAGACGTGTGCTCTTCCGATC </td></tr><tr><td>TCGAGATCGGAAGAGCACACGTCTGAACTCCAGTCAC </td></tr><tr><td rowspan="2"> <em>Msp</em>I </td><td>CTGGAGTTCAGACGTGTGCTCTTCCGATCT </td></tr><tr><td>CGAGATCGGAAGAGCACACGTCTGAACTCCAGTCAC </td></tr><tr><td colspan="2">Indexed primers for PCR<sup>a</sup> </td></tr><tr><td> Forward primer </td><td>AATGATACGGCGACCACCGAGATCTACACXXXXXXXXACACTCTTTCCCTACACGACGCTCTTCC </td></tr><tr><td> Reverse primer </td><td>CAAGCAGAAGACGGCATACGAGATXXXXXXXXGTGACTGGAGTTCAGACGTGTGCTCTTC </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>Index bases are indicated by X, which sequences are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw004_Supplementary_Data.zip?Expires=2147483647&Signature=qosUpqhIKaFSxMnA94HtqcgimoMsVGYfDdoaMWiECGocawcA8WW2-ahZyUM-mXfg8JhQtv9WUKzWh2kTBMBBGZtGAg9fpwYyaHF6TWEPVEkJG7PdrbIyJJbh0NsZ3nNrAiuMiFRYLkEAEEilmuSCroigb47KOYGz5RgGx5LKMu53R5HedlZeQ~BQZoGQATn4M-5aM6MVQ62vqSbBfb0dg2p~1NkL5XoLpKvHESzyIfNhpE9uFjCKPIUZrFmWSodLVBFe-mb~i6fK7c~NuTRBVGjjtPI3zXGwaq0~5WNw-Cr8UIwVnwKM0CIj9hhsNUpRedbO1gP0xxnOf0QNL8ErIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199512" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSW004TB1" class="table-wrap-title" data-id="DSW004TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Sequences of oligonucleotides used in ddRAD-Seq</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Names </th><th>Sequence (5′ – 3′) </th></tr></thead><tbody><tr><td>Restriction enzyme </td><td> </td></tr><tr><td rowspan="2"> <em>Pst</em>I </td><td>TCTTTCCCTACACGACGCTCTTCCGATCTGCA </td></tr><tr><td>GATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT </td></tr><tr><td rowspan="2"> <em>Eco</em>RI </td><td>CTGGAGTTCAGACGTGTGCTCTTCCGATCT </td></tr><tr><td>AATTAGATCGGAAGAGCACACGTCTGAACTCCAGTCAC </td></tr><tr><td rowspan="2"> <em>Hin</em>dIII </td><td>TCTTTCCCTACACGACGCTCTTCCGATCT </td></tr><tr><td>AGCTAGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT </td></tr><tr><td rowspan="2"> <em>Sal</em>I </td><td>CTGGAGTTCAGACGTGTGCTCTTCCGATC </td></tr><tr><td>TCGAGATCGGAAGAGCACACGTCTGAACTCCAGTCAC </td></tr><tr><td rowspan="2"> <em>Msp</em>I </td><td>CTGGAGTTCAGACGTGTGCTCTTCCGATCT </td></tr><tr><td>CGAGATCGGAAGAGCACACGTCTGAACTCCAGTCAC </td></tr><tr><td colspan="2">Indexed primers for PCR<sup>a</sup> </td></tr><tr><td> Forward primer </td><td>AATGATACGGCGACCACCGAGATCTACACXXXXXXXXACACTCTTTCCCTACACGACGCTCTTCC </td></tr><tr><td> Reverse primer </td><td>CAAGCAGAAGACGGCATACGAGATXXXXXXXXGTGACTGGAGTTCAGACGTGTGCTCTTC </td></tr></tbody></table></div><div content-id="DSW004TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Names </th><th>Sequence (5′ – 3′) </th></tr></thead><tbody><tr><td>Restriction enzyme </td><td> </td></tr><tr><td rowspan="2"> <em>Pst</em>I </td><td>TCTTTCCCTACACGACGCTCTTCCGATCTGCA </td></tr><tr><td>GATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT </td></tr><tr><td rowspan="2"> <em>Eco</em>RI </td><td>CTGGAGTTCAGACGTGTGCTCTTCCGATCT </td></tr><tr><td>AATTAGATCGGAAGAGCACACGTCTGAACTCCAGTCAC </td></tr><tr><td rowspan="2"> <em>Hin</em>dIII </td><td>TCTTTCCCTACACGACGCTCTTCCGATCT </td></tr><tr><td>AGCTAGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT </td></tr><tr><td rowspan="2"> <em>Sal</em>I </td><td>CTGGAGTTCAGACGTGTGCTCTTCCGATC </td></tr><tr><td>TCGAGATCGGAAGAGCACACGTCTGAACTCCAGTCAC </td></tr><tr><td rowspan="2"> <em>Msp</em>I </td><td>CTGGAGTTCAGACGTGTGCTCTTCCGATCT </td></tr><tr><td>CGAGATCGGAAGAGCACACGTCTGAACTCCAGTCAC </td></tr><tr><td colspan="2">Indexed primers for PCR<sup>a</sup> </td></tr><tr><td> Forward primer </td><td>AATGATACGGCGACCACCGAGATCTACACXXXXXXXXACACTCTTTCCCTACACGACGCTCTTCC </td></tr><tr><td> Reverse primer </td><td>CAAGCAGAAGACGGCATACGAGATXXXXXXXXGTGACTGGAGTTCAGACGTGTGCTCTTC </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>Index bases are indicated by X, which sequences are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw004_Supplementary_Data.zip?Expires=2147483647&Signature=qosUpqhIKaFSxMnA94HtqcgimoMsVGYfDdoaMWiECGocawcA8WW2-ahZyUM-mXfg8JhQtv9WUKzWh2kTBMBBGZtGAg9fpwYyaHF6TWEPVEkJG7PdrbIyJJbh0NsZ3nNrAiuMiFRYLkEAEEilmuSCroigb47KOYGz5RgGx5LKMu53R5HedlZeQ~BQZoGQATn4M-5aM6MVQ62vqSbBfb0dg2p~1NkL5XoLpKvHESzyIfNhpE9uFjCKPIUZrFmWSodLVBFe-mb~i6fK7c~NuTRBVGjjtPI3zXGwaq0~5WNw-Cr8UIwVnwKM0CIj9hhsNUpRedbO1gP0xxnOf0QNL8ErIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199512" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87199513 id="87199513" class="section-title" data-legacy-id=s2d>2.4. Computational processing for data from empirical ddRAD-Seq analysis</h3>
<p>In ddRAD-Seq data analysis as well as WGS ( Processing data for whole-genome sequence of tomato), low-quality sequences were removed and adapters were trimmed using PRINSEQ (-trim_right 1 -trim_qual_right 10 -min_len 100 -derep) and fastx_clipper (-a AGATCGGAAGAGC -l 100 -M 10 -n) in FASTX-Toolkit (<a class="link link-uri" href="http://hannonlab.cshl.edu/fastx_toolkit">http://hannonlab.cshl.edu/fastx_toolkit</a>; version 0.10.1). The filtered reads, or subsets of the reads randomly selected using seqtk (<a class="link link-uri" href="https://github.com/lh3/seqtk">https://github.com/lh3/seqtk</a>), were mapped onto the reference sequences of either contigs generated by assembly of the filtered reads using Newbler (version 3.0; parameters: off for extend low depth overlap; Roche, Basel, Switzerland) or the tomato genome sequence (SL2.50) using Bowtie 2 (version 2.1.0; parameters: --minins 100 --no-mixed).<sup><a reveal-id="DSW004C17" data-open="DSW004C17" class="link link-ref link-reveal xref-bibr">17</a></sup> The resultant sequence alignment/map format (SAM) files were converted to binary sequence alignment/map format files and subjected to SNP calling using the mpileup option of SAMtools (version 0.1.19; parameters: -Duf)<sup><a reveal-id="DSW004C19" data-open="DSW004C19" class="link link-ref link-reveal xref-bibr">19</a></sup> and the view option of BCFtools (parameters: -vcg). Lengths of genome regions covered with more than one read at least were calculated with genomeCoverage option of BEDtools (version 2.17.0; parameters: -d).<sup><a reveal-id="DSW004C20" data-open="DSW004C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Furthermore, variant call format (VCF) files were filtered with VCFtools (version 0.1.11; parameters: --minQ 10 --minDP 4 for the cultivars' data, or --minQ 10 --minDP 4 --max-missing 0.2 --remove-indels for the RMF2 data).<sup><a reveal-id="DSW004C21" data-open="DSW004C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Missing data were imputed using Beagle4.<sup><a reveal-id="DSW004C22" data-open="DSW004C22" class="link link-ref link-reveal xref-bibr">22</a></sup> The locations of SNPs in genic and intergenic regions were predicted using SnpEff (version 4.0e; parameters: -v SL2.50, -no-downstream and -no-upstream),<sup><a reveal-id="DSW004C23" data-open="DSW004C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and those in repetitive sequences and non-repetitive were classified in accordance with the annotation by International Tomato Annotation Group (ITAG2.4_repeats.gff3 available from Sol Genomics Network; <a class="link link-uri" href="https://www.sgn.cornell.edu">https://www.sgn.cornell.edu</a>). Similarity searches of marker-associated sequences against the SL2.50 tomato genome sequence were carried out using BlastN with default parameters.<sup><a reveal-id="DSW004C24" data-open="DSW004C24" class="link link-ref link-reveal xref-bibr">24</a></sup></p>                            <h3 scrollto-destination=87199515 id="87199515" class="section-title" data-legacy-id=s2e>2.5. Linkage analysis and construction of genetic linkage maps</h3>
<p>Linkage analysis was carried out with the imputed SNP dataset for RMF2. The segregated data were classified into groups using the grouping module of JoinMap4<sup><a reveal-id="DSW004C25" data-open="DSW004C25" class="link link-ref link-reveal xref-bibr">25</a></sup> with LOD scores of 3–6. The marker order and relative map distances were calculated using the regression-mapping algorithm with the following parameters: Haldane's mapping function, recombination frequency ≤0.35, and LOD score ≥2.0. The graphical maps were drawn using the MapChart program.<sup><a reveal-id="DSW004C26" data-open="DSW004C26" class="link link-ref link-reveal xref-bibr">26</a></sup></p>                            <h2 scrollto-destination=87199517 id="87199517" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87199518 id="87199518" class="section-title" data-legacy-id=s3a>3.1. Establishment of a genome-wide SNP dataset</h3>
<p>WGS data for Micro-Tom<sup><a reveal-id="DSW004C16" data-open="DSW004C16" class="link link-ref link-reveal xref-bibr">16</a></sup> and Regina<sup><a reveal-id="DSW004C15" data-open="DSW004C15" class="link link-ref link-reveal xref-bibr">15</a></sup> were used to generate a genome-wide SNP dataset by mapping the reads onto the tomato reference sequence, version SL2.50,<sup><a reveal-id="DSW004C14" data-open="DSW004C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSW004C27" data-open="DSW004C27" class="link link-ref link-reveal xref-bibr">27</a></sup> as described in Materials and Methods. Mapping rate and fraction of aligned regions of the SL2.50 were 97.9 and 99.2%, respectively, in Micro-Tom, while those were 61.1 and 98.8%, respectively, in Regina. A total of 1,187,941 high-quality SNPs between the two lines were discovered by filtering with the following parameters (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw004_Supplementary_Data.zip?Expires=2147483647&Signature=qosUpqhIKaFSxMnA94HtqcgimoMsVGYfDdoaMWiECGocawcA8WW2-ahZyUM-mXfg8JhQtv9WUKzWh2kTBMBBGZtGAg9fpwYyaHF6TWEPVEkJG7PdrbIyJJbh0NsZ3nNrAiuMiFRYLkEAEEilmuSCroigb47KOYGz5RgGx5LKMu53R5HedlZeQ~BQZoGQATn4M-5aM6MVQ62vqSbBfb0dg2p~1NkL5XoLpKvHESzyIfNhpE9uFjCKPIUZrFmWSodLVBFe-mb~i6fK7c~NuTRBVGjjtPI3zXGwaq0~5WNw-Cr8UIwVnwKM0CIj9hhsNUpRedbO1gP0xxnOf0QNL8ErIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>): SNP quality, &gt;10, and depth of coverage, ≥4. The SNP loci were unevenly distributed over the genome, with numbers ranging from 10,170 SNPs on chromosome 6 (chromosome length of 49.8 Mb in total) to 277,708 SNPs on chromosome 4 (66.5 Mb in total) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw004_Supplementary_Data.zip?Expires=2147483647&Signature=qosUpqhIKaFSxMnA94HtqcgimoMsVGYfDdoaMWiECGocawcA8WW2-ahZyUM-mXfg8JhQtv9WUKzWh2kTBMBBGZtGAg9fpwYyaHF6TWEPVEkJG7PdrbIyJJbh0NsZ3nNrAiuMiFRYLkEAEEilmuSCroigb47KOYGz5RgGx5LKMu53R5HedlZeQ~BQZoGQATn4M-5aM6MVQ62vqSbBfb0dg2p~1NkL5XoLpKvHESzyIfNhpE9uFjCKPIUZrFmWSodLVBFe-mb~i6fK7c~NuTRBVGjjtPI3zXGwaq0~5WNw-Cr8UIwVnwKM0CIj9hhsNUpRedbO1gP0xxnOf0QNL8ErIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Only 13.5 and 39.1% of the 1,187,941 SNPs were found on genic regions and non-repeat sequences (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw004_Supplementary_Data.zip?Expires=2147483647&Signature=qosUpqhIKaFSxMnA94HtqcgimoMsVGYfDdoaMWiECGocawcA8WW2-ahZyUM-mXfg8JhQtv9WUKzWh2kTBMBBGZtGAg9fpwYyaHF6TWEPVEkJG7PdrbIyJJbh0NsZ3nNrAiuMiFRYLkEAEEilmuSCroigb47KOYGz5RgGx5LKMu53R5HedlZeQ~BQZoGQATn4M-5aM6MVQ62vqSbBfb0dg2p~1NkL5XoLpKvHESzyIfNhpE9uFjCKPIUZrFmWSodLVBFe-mb~i6fK7c~NuTRBVGjjtPI3zXGwaq0~5WNw-Cr8UIwVnwKM0CIj9hhsNUpRedbO1gP0xxnOf0QNL8ErIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1 and Table S2</a></span>), respectively, both of which are biologically important sequences in the genome (see Distribution of SNPs in genic/intergenic regions and repeat/non-repeat sequences for details).</p>                            <h3 scrollto-destination=87199520 id="87199520" class="section-title" data-legacy-id=s3b>3.2. <em>In silico</em> restriction digestion to determine optimal restriction enzymes</h3>
<p>To identify the optimal restriction enzymes for experimental ddRAD-Seq analysis, we performed <em>in silico</em> restriction digestion. For this analysis, we selected five enzymes (<em>Sal</em>I, <em>Pst</em>I, <em>Eco</em>RI, <em>Hin</em>dIII, and <em>Msp</em>I) with different frequencies of recognition sites in the tomato genome, low in <em>Sal</em>I and <em>Pst</em>I, middle in <em>Eco</em>RI and <em>Hin</em>dIII, and high in <em>Msp</em>I (Fig. <a reveal-id="DSW004F1" data-open="DSW004F1" class="link link-reveal link-table xref-fig">1</a>A). Four combinations of the enzymes (a combination of low and low: <em>Sal</em>I/<em>Pst</em>I; low and middle: <em>Pst</em>I/<em>Eco</em>RI; middle and middle: <em>Eco</em>RI/<em>Hin</em>dIII; and middle and high: <em>Pst</em>I/<em>Msp</em>I) were used for <em>in silico</em> digestion of the genome sequence. The numbers of fragments with 300–900 bases, our target for experimental ddRAD-Seq experiment, covered the entire tomato genome evenly (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw004_Supplementary_Data.zip?Expires=2147483647&Signature=qosUpqhIKaFSxMnA94HtqcgimoMsVGYfDdoaMWiECGocawcA8WW2-ahZyUM-mXfg8JhQtv9WUKzWh2kTBMBBGZtGAg9fpwYyaHF6TWEPVEkJG7PdrbIyJJbh0NsZ3nNrAiuMiFRYLkEAEEilmuSCroigb47KOYGz5RgGx5LKMu53R5HedlZeQ~BQZoGQATn4M-5aM6MVQ62vqSbBfb0dg2p~1NkL5XoLpKvHESzyIfNhpE9uFjCKPIUZrFmWSodLVBFe-mb~i6fK7c~NuTRBVGjjtPI3zXGwaq0~5WNw-Cr8UIwVnwKM0CIj9hhsNUpRedbO1gP0xxnOf0QNL8ErIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>), but varied from 5,082 for <em>Sal</em>I/<em>Pst</em>I to 65,104 for <em>Eco</em>RI/<em>Hin</em>dIII (Fig. <a reveal-id="DSW004F1" data-open="DSW004F1" class="link link-reveal link-table xref-fig">1</a>B and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw004_Supplementary_Data.zip?Expires=2147483647&Signature=qosUpqhIKaFSxMnA94HtqcgimoMsVGYfDdoaMWiECGocawcA8WW2-ahZyUM-mXfg8JhQtv9WUKzWh2kTBMBBGZtGAg9fpwYyaHF6TWEPVEkJG7PdrbIyJJbh0NsZ3nNrAiuMiFRYLkEAEEilmuSCroigb47KOYGz5RgGx5LKMu53R5HedlZeQ~BQZoGQATn4M-5aM6MVQ62vqSbBfb0dg2p~1NkL5XoLpKvHESzyIfNhpE9uFjCKPIUZrFmWSodLVBFe-mb~i6fK7c~NuTRBVGjjtPI3zXGwaq0~5WNw-Cr8UIwVnwKM0CIj9hhsNUpRedbO1gP0xxnOf0QNL8ErIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2 and Table S2</a></span>). The distributions of SNPs on each chromosome corresponded to those obtained from WGS data, although the total numbers of the SNPs decreased drastically, to only 3,553 (0.3%) for <em>Sal</em>I/<em>Pst</em>I and 47,768 (4.0%) for <em>Eco</em>RI/<em>Hin</em>dIII (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw004_Supplementary_Data.zip?Expires=2147483647&Signature=qosUpqhIKaFSxMnA94HtqcgimoMsVGYfDdoaMWiECGocawcA8WW2-ahZyUM-mXfg8JhQtv9WUKzWh2kTBMBBGZtGAg9fpwYyaHF6TWEPVEkJG7PdrbIyJJbh0NsZ3nNrAiuMiFRYLkEAEEilmuSCroigb47KOYGz5RgGx5LKMu53R5HedlZeQ~BQZoGQATn4M-5aM6MVQ62vqSbBfb0dg2p~1NkL5XoLpKvHESzyIfNhpE9uFjCKPIUZrFmWSodLVBFe-mb~i6fK7c~NuTRBVGjjtPI3zXGwaq0~5WNw-Cr8UIwVnwKM0CIj9hhsNUpRedbO1gP0xxnOf0QNL8ErIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1 and Table S2</a></span>). While proportions of SNPs on genic regions to the detected SNPs were ranging from 16.3% (<em>Eco</em>RI/<em>Hin</em>dIII) to 29.0% (<em>Pst</em>I/<em>Eco</em>RI), those of SNPs in non-repeat sequences to whole genome were from 26.7% (<em>Sal</em>I/<em>Pst</em>I) to 45.0% (<em>Pst</em>I/<em>Eco</em>RI) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw004_Supplementary_Data.zip?Expires=2147483647&Signature=qosUpqhIKaFSxMnA94HtqcgimoMsVGYfDdoaMWiECGocawcA8WW2-ahZyUM-mXfg8JhQtv9WUKzWh2kTBMBBGZtGAg9fpwYyaHF6TWEPVEkJG7PdrbIyJJbh0NsZ3nNrAiuMiFRYLkEAEEilmuSCroigb47KOYGz5RgGx5LKMu53R5HedlZeQ~BQZoGQATn4M-5aM6MVQ62vqSbBfb0dg2p~1NkL5XoLpKvHESzyIfNhpE9uFjCKPIUZrFmWSodLVBFe-mb~i6fK7c~NuTRBVGjjtPI3zXGwaq0~5WNw-Cr8UIwVnwKM0CIj9hhsNUpRedbO1gP0xxnOf0QNL8ErIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1 and Table S2</a></span>). The <em>in silico</em> analysis was applied to other plant species, e.g. <em>A. thaliana</em>, <em>L. japonicus</em>, and <em>O. sativa</em>. The result indicated that the tendency was similar to those of <em>A. thaliana</em> and <em>L. japonicus</em> except for <em>O. sativa</em> in which number of <em>Pst</em>I/<em>Msp</em>I fragments were predominant (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw004_Supplementary_Data.zip?Expires=2147483647&Signature=qosUpqhIKaFSxMnA94HtqcgimoMsVGYfDdoaMWiECGocawcA8WW2-ahZyUM-mXfg8JhQtv9WUKzWh2kTBMBBGZtGAg9fpwYyaHF6TWEPVEkJG7PdrbIyJJbh0NsZ3nNrAiuMiFRYLkEAEEilmuSCroigb47KOYGz5RgGx5LKMu53R5HedlZeQ~BQZoGQATn4M-5aM6MVQ62vqSbBfb0dg2p~1NkL5XoLpKvHESzyIfNhpE9uFjCKPIUZrFmWSodLVBFe-mb~i6fK7c~NuTRBVGjjtPI3zXGwaq0~5WNw-Cr8UIwVnwKM0CIj9hhsNUpRedbO1gP0xxnOf0QNL8ErIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). </p>                        <a id="87199522" scrollto-destination="87199522"></a>
<div id="DSW004F1" data-id="DSW004F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/m_dsw00401.jpeg?Expires=2147483647&Signature=fevMl0okT9r8iUypzxpYGq9aPxCvmHOic08MuSPsEWd6F-mAo~sfcKI8kksXU21rJLEWFuVIxCBCbaKqZoAhUiuupmKmn8xz3pRY7bt6MPJkgvglmT7gyd4-Spazf~pO1evmxJmyl6dPzcAPIXNbZFHg1jVgOyfpwbBRAwWfM9wNNZPpkcJPPmiTJxEL50wtR7FpNkkuBE5N323cJUD8cvsz9RNF0dtWF8Yrgm0Z0zp6nppFBWf6Axk2OSHLmCOybd1TkeSSjpcqEj7N5BgD8YiKIVSqpt~9rj-nCaOII8BM6ZKc8SK1i~xwp4zgO4x~Y7tWqIP8NxIi0XQ-hBieKQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Numbers of restriction sites and restriction fragments in the tomato genome (SL2.50). Bars indicate the numbers of restriction sites (A) and 300–900 bp restriction fragments (B) predicted from the SL2.50 tomato genome sequence by in silico analysis." path-from-xml="dsw00401.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199522/dsw00401.jpeg" path-from-xml="dsw00401.jpeg" target="_blank">View large</a><a section="87199522" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw00401.jpeg?Expires=2147483647&Signature=wmmtsIuRnokqZ6xe5hADWv2jzF74PAmtkXBB0gLLvzctSOfKdnhDVO0UKbHxafbQIb1tlWGRjsP~7uWqeco~RfydthmkJueVtVaNfsi~2g8n7aAmyi8-rHdZK1NimQlDNokm14e8O6ND3aSIzCA9n9ECzwGPNGgnPTRS69Mb3vdcF1kl37ErJgOJJK1vC7bgmI5rX0w-1Olld-BoHabol2I2sUqcufBegsD5rxy-wHSbDdutUqOflN-E1qPftx4N~OSma4lv~T54vbU3zkYGpXbG3jP9mDDIWxOwp9tVS6ZCCax9dV-d6ji2W-IYlcqriEEDmEkONE61--4u7UrUPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199522&ar=1745392&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00401.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Numbers of restriction sites and restriction fragments in the tomato genome (SL2.50). Bars indicate the numbers of restriction sites (A) and 300–900 bp restriction fragments (B) predicted from the SL2.50 tomato genome sequence by <em>in silico</em> analysis.</p></div></div><div content-id="DSW004F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/m_dsw00401.jpeg?Expires=2147483647&Signature=fevMl0okT9r8iUypzxpYGq9aPxCvmHOic08MuSPsEWd6F-mAo~sfcKI8kksXU21rJLEWFuVIxCBCbaKqZoAhUiuupmKmn8xz3pRY7bt6MPJkgvglmT7gyd4-Spazf~pO1evmxJmyl6dPzcAPIXNbZFHg1jVgOyfpwbBRAwWfM9wNNZPpkcJPPmiTJxEL50wtR7FpNkkuBE5N323cJUD8cvsz9RNF0dtWF8Yrgm0Z0zp6nppFBWf6Axk2OSHLmCOybd1TkeSSjpcqEj7N5BgD8YiKIVSqpt~9rj-nCaOII8BM6ZKc8SK1i~xwp4zgO4x~Y7tWqIP8NxIi0XQ-hBieKQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Numbers of restriction sites and restriction fragments in the tomato genome (SL2.50). Bars indicate the numbers of restriction sites (A) and 300–900 bp restriction fragments (B) predicted from the SL2.50 tomato genome sequence by in silico analysis." path-from-xml="dsw00401.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199522/dsw00401.jpeg" path-from-xml="dsw00401.jpeg" target="_blank">View large</a><a section="87199522" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw00401.jpeg?Expires=2147483647&Signature=wmmtsIuRnokqZ6xe5hADWv2jzF74PAmtkXBB0gLLvzctSOfKdnhDVO0UKbHxafbQIb1tlWGRjsP~7uWqeco~RfydthmkJueVtVaNfsi~2g8n7aAmyi8-rHdZK1NimQlDNokm14e8O6ND3aSIzCA9n9ECzwGPNGgnPTRS69Mb3vdcF1kl37ErJgOJJK1vC7bgmI5rX0w-1Olld-BoHabol2I2sUqcufBegsD5rxy-wHSbDdutUqOflN-E1qPftx4N~OSma4lv~T54vbU3zkYGpXbG3jP9mDDIWxOwp9tVS6ZCCax9dV-d6ji2W-IYlcqriEEDmEkONE61--4u7UrUPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199522&ar=1745392&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00401.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Numbers of restriction sites and restriction fragments in the tomato genome (SL2.50). Bars indicate the numbers of restriction sites (A) and 300–900 bp restriction fragments (B) predicted from the SL2.50 tomato genome sequence by <em>in silico</em> analysis.</p></div></div>                            <h3 scrollto-destination=87199523 id="87199523" class="section-title" data-legacy-id=s3c>3.3. Establishment of data processing pipeline for ddRAD-Seq</h3>
<p>A data processing pipeline for SNP discovery was established using actual MiSeq reads of Micro-Tom and Regina ddRAD-Seq libraries generated using the <em>Pst</em>I/<em>Msp</em>I combination (PM libraries). Briefly, sequence reads were processed by removing low-quality reads and trimming adapters, and then mapped onto the reference sequence to detect SNP candidates (see Materials and Methods for details). When 1.9 and 2.2 M paired-reads for Micro-Tom and Regina, respectively, were analysed using this pipeline, 1.2 and 1.4 M high-quality reads were obtained, and 83,011 SNP candidates, including 20,689 homozygous and 62,322 loci with genotypes called as ‘heterozygous’, were detected prior to filtering. Because the two lines are inbred, the ‘heterozygous’ SNPs were excluded because they were likely to reflect sequencing or alignment errors. Of the 20,689 homozygous SNPs, 19,969 SNPs with quality values &gt;10 were selected as high-confidence SNP loci. Out of them, 15,746 SNPs (78.9%) were identical to those from WGS data, whereas the remaining 4,223 SNPs (22.1%) were not found due to sites of insufficient read coverage in the WGS data.</p>                            <h3 scrollto-destination=87199525 id="87199525" class="section-title" data-legacy-id=s3d>3.4. Experimental validation of SNP candidates detected by <em>in silico</em> analysis</h3>
<p>To validate the accuracy of the <em>in silico</em> predictions that the numbers of SNPs detected in ddRAD-Seq would be depending on choice of restriction enzymes, additional MiSeq reads were obtained from Micro-Tom and Regina ddRAD-Seq libraries generated using three more restriction enzyme combinations, <em>Sal</em>I/<em>Pst</em>I (SP), <em>Pst</em>I/<em>Eco</em>RI (PE), and <em>Eco</em>RI/<em>Hin</em>dIII (EH) as well as PM as above. After removal of low-quality sequences and trimming of adapters, a subset of 100k to 900k high-quality paired-end reads were generated for all four libraries. Each subset was mapped onto the reference genome sequence, SL2.50, and high-quality SNP candidates were selected by filtering using the criteria described above. As expected, the number of SNPs in each dataset increased as the number of reads increased (Fig. <a reveal-id="DSW004F2" data-open="DSW004F2" class="link link-reveal link-table xref-fig">2</a>A). However, this tendency differed considerably among the enzyme combinations. The number of SNPs of PM increased linearly up to ∼8,000 when 900k reads were used, whereas that of EH gradually reached ∼20,000. In contrast, despite their higher numbers of reads, the PE and SP libraries had far fewer SNPs: 4,000 for PE and &lt;100 for SP. As in the <em>in silico</em> prediction, the EH and PM libraries gave much more SNPs than the PE and SP. </p>                        <a id="87199527" scrollto-destination="87199527"></a>
<div id="DSW004F2" data-id="DSW004F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/m_dsw00402.jpeg?Expires=2147483647&Signature=4rNx0Q1tfJfhpBTtkMkedsqfJEX7wjOcv6KjVmfZWjfCyR1EGdEvxpSHOAn-PxiS0Ul91EBDJr7LUexHgesgGgU5Lvk3qxrMDtjsxc3ZPOnMDG5XuKtcjicoZ-UZmigHXe2lqBXVQLPWJLfmiZD06onhYmm8DyOYHMNO4~lfUhGfUKY~74TnSKUHdGA7pdarRBiLPsOCXjRFOZqcFUVIsbN9YkiI8DAmc3K2Dih89ajNtqx3~FNEx-ofV9~SsvV1Nq2licbwcZIWzqJLT1HfqD5dgUG9yj6WFrvZVgko06av1~hoY02qUlw4w~qAO9Uk0BSUFHs3itcPfqx-kBW4kA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Number of SNPs detected from empirical ddRAD-Seq analysis. Line chart indicates numbers of SNPs between Micro-Tom and Regina with four combinations of restriction enzymes (A) and SNPs of six cultivars with respect to SL2.50 using the PstI/MspI combination (B)." path-from-xml="dsw00402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199527/dsw00402.jpeg" path-from-xml="dsw00402.jpeg" target="_blank">View large</a><a section="87199527" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw00402.jpeg?Expires=2147483647&Signature=t85HqQ9ZRP9706TXqGosWCQXAuPfn7YQXbDLRZxNj2QAQP9bEDkfamAsdvlP-LXp5Ig2mV0a6OaLSN8ewnLAjmZSLV2lP8f5T2I0A6fS0Mq~erTv0x3TnZ1j5vidtCXtG4UYnpxcbte-oYaGylAxMsidcoIMiFsUCQ9uF9trBHvz74at01RFeQ6wPz1CZSIN5SANNtEstMhfXAmgJHU5kqLrAXousKRADoEVTnjsKSNo~aC8k9zrItRv34pm1JhAGA2oy9tuae3LehtXMRwpNPiZQX3o1IfxshjRWy3JlqDlNxXY52da9JwYTRAZBtCraXF9AvmyxpdbwlO7HR9u7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199527&ar=1745392&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of SNPs detected from empirical ddRAD-Seq analysis. Line chart indicates numbers of SNPs between Micro-Tom and Regina with four combinations of restriction enzymes (A) and SNPs of six cultivars with respect to SL2.50 using the <em>Pst</em>I/<em>Msp</em>I combination (B).</p></div></div><div content-id="DSW004F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/m_dsw00402.jpeg?Expires=2147483647&Signature=4rNx0Q1tfJfhpBTtkMkedsqfJEX7wjOcv6KjVmfZWjfCyR1EGdEvxpSHOAn-PxiS0Ul91EBDJr7LUexHgesgGgU5Lvk3qxrMDtjsxc3ZPOnMDG5XuKtcjicoZ-UZmigHXe2lqBXVQLPWJLfmiZD06onhYmm8DyOYHMNO4~lfUhGfUKY~74TnSKUHdGA7pdarRBiLPsOCXjRFOZqcFUVIsbN9YkiI8DAmc3K2Dih89ajNtqx3~FNEx-ofV9~SsvV1Nq2licbwcZIWzqJLT1HfqD5dgUG9yj6WFrvZVgko06av1~hoY02qUlw4w~qAO9Uk0BSUFHs3itcPfqx-kBW4kA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Number of SNPs detected from empirical ddRAD-Seq analysis. Line chart indicates numbers of SNPs between Micro-Tom and Regina with four combinations of restriction enzymes (A) and SNPs of six cultivars with respect to SL2.50 using the PstI/MspI combination (B)." path-from-xml="dsw00402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199527/dsw00402.jpeg" path-from-xml="dsw00402.jpeg" target="_blank">View large</a><a section="87199527" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw00402.jpeg?Expires=2147483647&Signature=t85HqQ9ZRP9706TXqGosWCQXAuPfn7YQXbDLRZxNj2QAQP9bEDkfamAsdvlP-LXp5Ig2mV0a6OaLSN8ewnLAjmZSLV2lP8f5T2I0A6fS0Mq~erTv0x3TnZ1j5vidtCXtG4UYnpxcbte-oYaGylAxMsidcoIMiFsUCQ9uF9trBHvz74at01RFeQ6wPz1CZSIN5SANNtEstMhfXAmgJHU5kqLrAXousKRADoEVTnjsKSNo~aC8k9zrItRv34pm1JhAGA2oy9tuae3LehtXMRwpNPiZQX3o1IfxshjRWy3JlqDlNxXY52da9JwYTRAZBtCraXF9AvmyxpdbwlO7HR9u7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199527&ar=1745392&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of SNPs detected from empirical ddRAD-Seq analysis. Line chart indicates numbers of SNPs between Micro-Tom and Regina with four combinations of restriction enzymes (A) and SNPs of six cultivars with respect to SL2.50 using the <em>Pst</em>I/<em>Msp</em>I combination (B).</p></div></div>                            <h3 scrollto-destination=87199528 id="87199528" class="section-title" data-legacy-id=s3e>3.5. Distribution of SNPs in genic/intergenic regions and repeat/non-repeat sequences</h3>
<p>Since SNPs in genes and non-repetitive sequences are biologically meaningful in comparison with those in intergenic and repetitive regions. Not only proportions of genic and intergenic SNPs detected in the empirical ddRAD-Seq but also those of unique and repeat sequences in the tomato genome were investigated. The result indicated remarkable differences of the proportions among the restriction enzyme combinations (Fig. <a reveal-id="DSW004F3" data-open="DSW004F3" class="link link-reveal link-table xref-fig">3</a>A). In the PE libraries, 70.1% of SNPs were derived from genic regions. This rate is much higher than those from the <em>in silico</em> prediction, which suggested that 29.0% of SNPs occurred in genic regions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw004_Supplementary_Data.zip?Expires=2147483647&Signature=qosUpqhIKaFSxMnA94HtqcgimoMsVGYfDdoaMWiECGocawcA8WW2-ahZyUM-mXfg8JhQtv9WUKzWh2kTBMBBGZtGAg9fpwYyaHF6TWEPVEkJG7PdrbIyJJbh0NsZ3nNrAiuMiFRYLkEAEEilmuSCroigb47KOYGz5RgGx5LKMu53R5HedlZeQ~BQZoGQATn4M-5aM6MVQ62vqSbBfb0dg2p~1NkL5XoLpKvHESzyIfNhpE9uFjCKPIUZrFmWSodLVBFe-mb~i6fK7c~NuTRBVGjjtPI3zXGwaq0~5WNw-Cr8UIwVnwKM0CIj9hhsNUpRedbO1gP0xxnOf0QNL8ErIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). Furthermore, the PM and SP libraries were enriched for genic SNPs. In contrast, the EH library had SNP frequencies comparable with those obtained from the prediction. The proportions of the SNPs in the repeat/unique sequences were also markedly different among the libraries (Fig. <a reveal-id="DSW004F3" data-open="DSW004F3" class="link link-reveal link-table xref-fig">3</a>B and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw004_Supplementary_Data.zip?Expires=2147483647&Signature=qosUpqhIKaFSxMnA94HtqcgimoMsVGYfDdoaMWiECGocawcA8WW2-ahZyUM-mXfg8JhQtv9WUKzWh2kTBMBBGZtGAg9fpwYyaHF6TWEPVEkJG7PdrbIyJJbh0NsZ3nNrAiuMiFRYLkEAEEilmuSCroigb47KOYGz5RgGx5LKMu53R5HedlZeQ~BQZoGQATn4M-5aM6MVQ62vqSbBfb0dg2p~1NkL5XoLpKvHESzyIfNhpE9uFjCKPIUZrFmWSodLVBFe-mb~i6fK7c~NuTRBVGjjtPI3zXGwaq0~5WNw-Cr8UIwVnwKM0CIj9hhsNUpRedbO1gP0xxnOf0QNL8ErIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>): SNPs from the SP, PE, and PM libraries were enriched in the unique sequences in comparison with the prediction, while the proportion of the EH library was comparable with the prediction. We concluded that the PM and PE libraries had advantages to detect SNPs in gene regions and non-repetitive sequences in the tomato genome. </p>                        <a id="87199530" scrollto-destination="87199530"></a>
<div id="DSW004F3" data-id="DSW004F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/m_dsw00403.jpeg?Expires=2147483647&Signature=ookrf2Ozn2EnwIBlXCaM9a8joxPAOSY5LkyODVSQHCNcagiPv3emhdion9P7ls9zs7ROQ15e8tiLtdtVQ5vwOqvqRfGPJO65cTAqpkAwQWMUwd1KNBhd~1zjUva71MRs7o-84vWox3lsguAorWDg4n1IU9S4GkvBpPEOEgLpMxfyhMjK3EedU-YYxUZ8RzqDmLL30~2cCcyF6mAPX6SgWueCv5xz6r721ip0d~CaSfknJSq3YQxj1rMTUGEl6DoNtgKxn8j4Fz~7EFvXYt7sigoytTprw9VFbLDBNiCP8~jbVuB6neBlrqsj0PkKZKfInFg56gK0N8NLU9k-peQbUw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Proportions of SNPs detected from empirical ddRAD-Seq analysis. SNPs from empirical ddRAD-Seq libraries are distributed in genic and intergenic regions (A) and repeat and non-repeat sequences (B). Proportions of SNPs between Micro-Tom and Regina (MT vs REG) detected from WGS data is shown as a control." path-from-xml="dsw00403.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199530/dsw00403.jpeg" path-from-xml="dsw00403.jpeg" target="_blank">View large</a><a section="87199530" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw00403.jpeg?Expires=2147483647&Signature=GUL1dCYncP8MYvutrWgkZ08H7fv3Xk3W~70s1BZ1M6uur0EpCA3V87VRJZ26jEqDPk9eBkDSVBRq9ELkDeVQWtrEN8w~jpp2aanWjbhDRlt3KTlcDzKEbsGPnfGkRPhQ-SdmRhL3F8Fx-hC-BUleYJQ1SJGj9D~RQi0enntoPec3ohwbkwSP0cBlKPu1ymOyX23rNPyI6UXX1bxrlozrzKV01YPng~rncWioKgnhzmHj23-ivf1BSc4ksewA4Mb461AVD-qJ7ubU12ao~SAstbPOb0O7JajECwFa50YBoSAof1h8FNRDwwxJl-sXkBIGYBrTKq3GtBzU1xA8azBVHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199530&ar=1745392&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00403.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Proportions of SNPs detected from empirical ddRAD-Seq analysis. SNPs from empirical ddRAD-Seq libraries are distributed in genic and intergenic regions (A) and repeat and non-repeat sequences (B). Proportions of SNPs between Micro-Tom and Regina (MT vs REG) detected from WGS data is shown as a control.</p></div></div><div content-id="DSW004F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/m_dsw00403.jpeg?Expires=2147483647&Signature=ookrf2Ozn2EnwIBlXCaM9a8joxPAOSY5LkyODVSQHCNcagiPv3emhdion9P7ls9zs7ROQ15e8tiLtdtVQ5vwOqvqRfGPJO65cTAqpkAwQWMUwd1KNBhd~1zjUva71MRs7o-84vWox3lsguAorWDg4n1IU9S4GkvBpPEOEgLpMxfyhMjK3EedU-YYxUZ8RzqDmLL30~2cCcyF6mAPX6SgWueCv5xz6r721ip0d~CaSfknJSq3YQxj1rMTUGEl6DoNtgKxn8j4Fz~7EFvXYt7sigoytTprw9VFbLDBNiCP8~jbVuB6neBlrqsj0PkKZKfInFg56gK0N8NLU9k-peQbUw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Proportions of SNPs detected from empirical ddRAD-Seq analysis. SNPs from empirical ddRAD-Seq libraries are distributed in genic and intergenic regions (A) and repeat and non-repeat sequences (B). Proportions of SNPs between Micro-Tom and Regina (MT vs REG) detected from WGS data is shown as a control." path-from-xml="dsw00403.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199530/dsw00403.jpeg" path-from-xml="dsw00403.jpeg" target="_blank">View large</a><a section="87199530" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw00403.jpeg?Expires=2147483647&Signature=GUL1dCYncP8MYvutrWgkZ08H7fv3Xk3W~70s1BZ1M6uur0EpCA3V87VRJZ26jEqDPk9eBkDSVBRq9ELkDeVQWtrEN8w~jpp2aanWjbhDRlt3KTlcDzKEbsGPnfGkRPhQ-SdmRhL3F8Fx-hC-BUleYJQ1SJGj9D~RQi0enntoPec3ohwbkwSP0cBlKPu1ymOyX23rNPyI6UXX1bxrlozrzKV01YPng~rncWioKgnhzmHj23-ivf1BSc4ksewA4Mb461AVD-qJ7ubU12ao~SAstbPOb0O7JajECwFa50YBoSAof1h8FNRDwwxJl-sXkBIGYBrTKq3GtBzU1xA8azBVHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199530&ar=1745392&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00403.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Proportions of SNPs detected from empirical ddRAD-Seq analysis. SNPs from empirical ddRAD-Seq libraries are distributed in genic and intergenic regions (A) and repeat and non-repeat sequences (B). Proportions of SNPs between Micro-Tom and Regina (MT vs REG) detected from WGS data is shown as a control.</p></div></div>                            <h3 scrollto-destination=87199531 id="87199531" class="section-title" data-legacy-id=s3f>3.6. ddRAD-Seq in genetically diverse tomato lines</h3>
<p>Micro-Tom and Regina show larger genetic distances to Heinz 1706 in comparison with the other cultivated tomato lines.<sup><a reveal-id="DSW004C28" data-open="DSW004C28" class="link link-ref link-reveal xref-bibr">28</a></sup> To assess the numbers of SNPs in genetically diverse samples, the six lines, i.e. Ailsa Craig, M82, Moneymaker, and San Marzano as well as Micro-Tom and Regina, were further analysed with ddRAD-Seq. The <em>Pst</em>I/<em>Msp</em>I combination was employed in accordance with the results of the validation test, expecting to gain as many SNPs in genes and unique sequences of the tomato genome as possible. The high-quality sequence data from the six PM libraries were divided into subsets of 100k–900k paired-end reads and mapped onto the reference genome. As expected, the numbers of SNPs with respect to Heinz 1706 (SL2.50) detected by experimental ddRAD-Seq in Regina and Micro-Tom increased linearly up to ∼25,000, whereas those in the other four cultivars reached 5,000 or less (Fig. <a reveal-id="DSW004F2" data-open="DSW004F2" class="link link-reveal link-table xref-fig">2</a>B), indicating that SNP density in the genome influenced the detectability of SNPs by ddRAD-Seq. A graphical genotypes based on the result from the ddRAD-Seq with 900k paired-end reads indicated that the distribution of the SNPs was highly biased on the genome as reported in our previous study (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw004_Supplementary_Data.zip?Expires=2147483647&Signature=qosUpqhIKaFSxMnA94HtqcgimoMsVGYfDdoaMWiECGocawcA8WW2-ahZyUM-mXfg8JhQtv9WUKzWh2kTBMBBGZtGAg9fpwYyaHF6TWEPVEkJG7PdrbIyJJbh0NsZ3nNrAiuMiFRYLkEAEEilmuSCroigb47KOYGz5RgGx5LKMu53R5HedlZeQ~BQZoGQATn4M-5aM6MVQ62vqSbBfb0dg2p~1NkL5XoLpKvHESzyIfNhpE9uFjCKPIUZrFmWSodLVBFe-mb~i6fK7c~NuTRBVGjjtPI3zXGwaq0~5WNw-Cr8UIwVnwKM0CIj9hhsNUpRedbO1gP0xxnOf0QNL8ErIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>).<sup><a reveal-id="DSW004C14" data-open="DSW004C14" class="link link-ref link-reveal xref-bibr">14</a></sup> The SNP densities relative to SL2.40, a previous version of the tomato genome sequence with the same base compositions to the SL2.50,<sup><a reveal-id="DSW004C27" data-open="DSW004C27" class="link link-ref link-reveal xref-bibr">27</a></sup> were estimated to be one SNP per 651 bp in Regina,<sup><a reveal-id="DSW004C15" data-open="DSW004C15" class="link link-ref link-reveal xref-bibr">15</a></sup> 803 bp in Micro-Tom,<sup><a reveal-id="DSW004C16" data-open="DSW004C16" class="link link-ref link-reveal xref-bibr">16</a></sup> 1,011 bp in M82,<sup><a reveal-id="DSW004C15" data-open="DSW004C15" class="link link-ref link-reveal xref-bibr">15</a></sup> 3,105 bp in Moneymaker,<sup><a reveal-id="DSW004C29" data-open="DSW004C29" class="link link-ref link-reveal xref-bibr">29</a></sup> 4,347 bp in San Marzano,<sup><a reveal-id="DSW004C30" data-open="DSW004C30" class="link link-ref link-reveal xref-bibr">30</a></sup> and 8,387 bp in Ailsa Craig.<sup><a reveal-id="DSW004C15" data-open="DSW004C15" class="link link-ref link-reveal xref-bibr">15</a></sup> The percentages of SNPs detected by ddRAD-Seq analysis per genome-wide SNPs by the WGS were almost even in these six lines: 2.0% genome-wide SNPs on average, ranging from 0.5% in M82 to 3.6% in San Marzano, and proportion of the SNPs in gene regions and repeat sequences from the six lines were similar to those from a combination of Micro-Tom and Regina (Fig. <a reveal-id="DSW004F3" data-open="DSW004F3" class="link link-reveal link-table xref-fig">3</a>A and B).</p>                            <h3 scrollto-destination=87199533 id="87199533" class="section-title" data-legacy-id=s3g>3.7. ddRAD-Seq in an F<sub>2</sub> mapping population to construct genetic maps</h3>
<p>Accuracy of SNP genotypes called from the ddRAD-Seq pipeline was validated by construction of genetic linkage maps. Because miss-called SNPs would be rejected from the maps, mapping rate of SNPs is an indicator of the accuracy.</p><p>For the same reason as above to obtain as many SNPs as possible from gene and non-repetitive regions, the <em>Pst</em>I/<em>Msp</em>I was selected as the optimal enzyme combination for library construction for the F<sub>2</sub> mapping population (<em>n</em> = 96), RMF2, derived from a cross between Micro-Tom and Regina. Ninety-six libraries of RMF2 with index tags distinguishing each line (Table <a reveal-id="DSW004TB1" data-open="DSW004TB1" class="link link-reveal link-table xref-fig">1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw004_Supplementary_Data.zip?Expires=2147483647&Signature=qosUpqhIKaFSxMnA94HtqcgimoMsVGYfDdoaMWiECGocawcA8WW2-ahZyUM-mXfg8JhQtv9WUKzWh2kTBMBBGZtGAg9fpwYyaHF6TWEPVEkJG7PdrbIyJJbh0NsZ3nNrAiuMiFRYLkEAEEilmuSCroigb47KOYGz5RgGx5LKMu53R5HedlZeQ~BQZoGQATn4M-5aM6MVQ62vqSbBfb0dg2p~1NkL5XoLpKvHESzyIfNhpE9uFjCKPIUZrFmWSodLVBFe-mb~i6fK7c~NuTRBVGjjtPI3zXGwaq0~5WNw-Cr8UIwVnwKM0CIj9hhsNUpRedbO1gP0xxnOf0QNL8ErIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) were pooled and sequenced on an Illumina MiSeq, yielding an average of 268k paired-end reads (=134 Mb, 0.14× genome coverage) per line. After removal of low-quality sequences and trimming of adapters, 226k high-quality reads on average in each line were mapped onto SL2.50 along with the reads from the parental lines. Mapping rates of Micro-Tom and Regina were 94.3 and 92.9%, respectively, while that in the F<sub>2</sub> population was 91.0% on average. Of 155,992 SNP candidates between the parental lines, 60,512 loci with quality of ≤10 and depth of &lt;4 were eliminated; furthermore, 89,241 ‘heterozygous’ SNPs, probably resulting from sequencing and/or alignment errors as noted above, were also removed. Ultimately, 6,239 loci were selected. By allowing 20% missing data for each SNP locus across the 96 F<sub>2</sub> lines, 1,845 positions (depth of coverage of 13.5 on average) of the 6,239 loci were selected as high-confidence segregating SNPs in the F<sub>2</sub> population. Prior to linkage analysis, the missing genotypes were imputed in accordance with genotype data from the parental lines. Subsequently, 528 genetic loci similar to others were eliminated. Of the remaining 1,317 non-redundant SNP loci, 1,297 (98.5%) were classified into 13 groups, each of which corresponded to one tomato chromosome (with the exception of chromosome 10, which was represented by two groups). Linkage analysis generated a genetic map consisting of 13 linkage groups, with 1,257 loci (95.4%) covering a total of 1,693.2 cM (Table <a reveal-id="DSW004TB2" data-open="DSW004TB2" class="link link-reveal link-table xref-fig">2</a> and Fig. <a reveal-id="DSW004F4" data-open="DSW004F4" class="link link-reveal link-table xref-fig">4</a>). The distributions of mapped loci were biased both inter- and intra-chromosomally, reflecting the biases in genome-wide SNP distributions. The order of the mapped loci were consistent with their physical positions in SL2.50 (Fig. <a reveal-id="DSW004F4" data-open="DSW004F4" class="link link-reveal link-table xref-fig">4</a>). </p>                        <a id="87199536" scrollto-destination="87199536"></a>
<div content-id="DSW004TB2" class="table-modal"><div class="table-wrap"><div id="DSW004TB2" class="table-wrap-title" data-id="DSW004TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Number of mapped loci and length of genetic linkage maps</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Linkage group </th><th colspan="2">Reference-based map<hr /> </th><th colspan="2"><em>De novo</em> map<hr /> </th></tr><tr><th>#Mapped loci </th><th>Map length (cM) </th><th>#Mapped loci </th><th>Map length (cM) </th></tr></thead><tbody><tr><td>1 </td><td>151 </td><td>230.1 </td><td>86 </td><td>253.6 </td></tr><tr><td>2 </td><td>58 </td><td>120.9 </td><td>32 </td><td>60.6 </td></tr><tr><td>3 </td><td>126 </td><td>176.7 </td><td>66 </td><td>177.8 </td></tr><tr><td>4 </td><td>240 </td><td>203.3 </td><td>139 </td><td>199.0 </td></tr><tr><td>5 </td><td>85 </td><td>98.7 </td><td>38 </td><td>103.0 </td></tr><tr><td>6 </td><td>25 </td><td>26.6 </td><td>13 </td><td>28.1 </td></tr><tr><td>7 </td><td>212 </td><td>176.4 </td><td>99 </td><td>169.4 </td></tr><tr><td>8 </td><td>25 </td><td>94.1 </td><td>11 </td><td>107.6 </td></tr><tr><td>9 </td><td>70 </td><td>145.5 </td><td>44 </td><td>174.6 </td></tr><tr><td>10 </td><td>68<sup>a</sup> </td><td>111.8<sup>a</sup> </td><td>43<sup>a</sup> </td><td>135.4<sup>a</sup> </td></tr><tr><td>11 </td><td>93 </td><td>147.7 </td><td>50<sup>a</sup> </td><td>130.2<sup>a</sup> </td></tr><tr><td>12 </td><td>104 </td><td>161.3 </td><td>65 </td><td>152.5 </td></tr><tr><td>Total </td><td>1,257 </td><td>1,693.2 </td><td>686 </td><td>1,691.8 </td></tr></tbody></table></div><div content-id="DSW004TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Linkage group </th><th colspan="2">Reference-based map<hr /> </th><th colspan="2"><em>De novo</em> map<hr /> </th></tr><tr><th>#Mapped loci </th><th>Map length (cM) </th><th>#Mapped loci </th><th>Map length (cM) </th></tr></thead><tbody><tr><td>1 </td><td>151 </td><td>230.1 </td><td>86 </td><td>253.6 </td></tr><tr><td>2 </td><td>58 </td><td>120.9 </td><td>32 </td><td>60.6 </td></tr><tr><td>3 </td><td>126 </td><td>176.7 </td><td>66 </td><td>177.8 </td></tr><tr><td>4 </td><td>240 </td><td>203.3 </td><td>139 </td><td>199.0 </td></tr><tr><td>5 </td><td>85 </td><td>98.7 </td><td>38 </td><td>103.0 </td></tr><tr><td>6 </td><td>25 </td><td>26.6 </td><td>13 </td><td>28.1 </td></tr><tr><td>7 </td><td>212 </td><td>176.4 </td><td>99 </td><td>169.4 </td></tr><tr><td>8 </td><td>25 </td><td>94.1 </td><td>11 </td><td>107.6 </td></tr><tr><td>9 </td><td>70 </td><td>145.5 </td><td>44 </td><td>174.6 </td></tr><tr><td>10 </td><td>68<sup>a</sup> </td><td>111.8<sup>a</sup> </td><td>43<sup>a</sup> </td><td>135.4<sup>a</sup> </td></tr><tr><td>11 </td><td>93 </td><td>147.7 </td><td>50<sup>a</sup> </td><td>130.2<sup>a</sup> </td></tr><tr><td>12 </td><td>104 </td><td>161.3 </td><td>65 </td><td>152.5 </td></tr><tr><td>Total </td><td>1,257 </td><td>1,693.2 </td><td>686 </td><td>1,691.8 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p><sup>a</sup>These numbers reflect the total values of divided linkage groups.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199536" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSW004TB2" class="table-wrap-title" data-id="DSW004TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Number of mapped loci and length of genetic linkage maps</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Linkage group </th><th colspan="2">Reference-based map<hr /> </th><th colspan="2"><em>De novo</em> map<hr /> </th></tr><tr><th>#Mapped loci </th><th>Map length (cM) </th><th>#Mapped loci </th><th>Map length (cM) </th></tr></thead><tbody><tr><td>1 </td><td>151 </td><td>230.1 </td><td>86 </td><td>253.6 </td></tr><tr><td>2 </td><td>58 </td><td>120.9 </td><td>32 </td><td>60.6 </td></tr><tr><td>3 </td><td>126 </td><td>176.7 </td><td>66 </td><td>177.8 </td></tr><tr><td>4 </td><td>240 </td><td>203.3 </td><td>139 </td><td>199.0 </td></tr><tr><td>5 </td><td>85 </td><td>98.7 </td><td>38 </td><td>103.0 </td></tr><tr><td>6 </td><td>25 </td><td>26.6 </td><td>13 </td><td>28.1 </td></tr><tr><td>7 </td><td>212 </td><td>176.4 </td><td>99 </td><td>169.4 </td></tr><tr><td>8 </td><td>25 </td><td>94.1 </td><td>11 </td><td>107.6 </td></tr><tr><td>9 </td><td>70 </td><td>145.5 </td><td>44 </td><td>174.6 </td></tr><tr><td>10 </td><td>68<sup>a</sup> </td><td>111.8<sup>a</sup> </td><td>43<sup>a</sup> </td><td>135.4<sup>a</sup> </td></tr><tr><td>11 </td><td>93 </td><td>147.7 </td><td>50<sup>a</sup> </td><td>130.2<sup>a</sup> </td></tr><tr><td>12 </td><td>104 </td><td>161.3 </td><td>65 </td><td>152.5 </td></tr><tr><td>Total </td><td>1,257 </td><td>1,693.2 </td><td>686 </td><td>1,691.8 </td></tr></tbody></table></div><div content-id="DSW004TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Linkage group </th><th colspan="2">Reference-based map<hr /> </th><th colspan="2"><em>De novo</em> map<hr /> </th></tr><tr><th>#Mapped loci </th><th>Map length (cM) </th><th>#Mapped loci </th><th>Map length (cM) </th></tr></thead><tbody><tr><td>1 </td><td>151 </td><td>230.1 </td><td>86 </td><td>253.6 </td></tr><tr><td>2 </td><td>58 </td><td>120.9 </td><td>32 </td><td>60.6 </td></tr><tr><td>3 </td><td>126 </td><td>176.7 </td><td>66 </td><td>177.8 </td></tr><tr><td>4 </td><td>240 </td><td>203.3 </td><td>139 </td><td>199.0 </td></tr><tr><td>5 </td><td>85 </td><td>98.7 </td><td>38 </td><td>103.0 </td></tr><tr><td>6 </td><td>25 </td><td>26.6 </td><td>13 </td><td>28.1 </td></tr><tr><td>7 </td><td>212 </td><td>176.4 </td><td>99 </td><td>169.4 </td></tr><tr><td>8 </td><td>25 </td><td>94.1 </td><td>11 </td><td>107.6 </td></tr><tr><td>9 </td><td>70 </td><td>145.5 </td><td>44 </td><td>174.6 </td></tr><tr><td>10 </td><td>68<sup>a</sup> </td><td>111.8<sup>a</sup> </td><td>43<sup>a</sup> </td><td>135.4<sup>a</sup> </td></tr><tr><td>11 </td><td>93 </td><td>147.7 </td><td>50<sup>a</sup> </td><td>130.2<sup>a</sup> </td></tr><tr><td>12 </td><td>104 </td><td>161.3 </td><td>65 </td><td>152.5 </td></tr><tr><td>Total </td><td>1,257 </td><td>1,693.2 </td><td>686 </td><td>1,691.8 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p><sup>a</sup>These numbers reflect the total values of divided linkage groups.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199536" target="_blank">View Large</a></div></div>                        <a id="87199537" scrollto-destination="87199537"></a>
<div id="DSW004F4" data-id="DSW004F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/m_dsw00404.jpeg?Expires=2147483647&Signature=PmlkmC8XCRZA5~eNeuiCaRLv5pDSXYGMYJWaR9zdqNz0ZWv66d9ORn4LgBPZjO5tVW5L-mr1OVsC1hdwW~4GKxVmwNZAe9K~rEWcGqZFGpRXLtaPq-QNugWxYCrHB3QLq0QjdMaUaphG4D81GYoYoXDWZKJip~Fo~b~8Bd783E4~us6xtMOg7WaFiHoQyMonWSmdZdpy~YE6HfY8xlYGwXcOjQM9vgILUS2KrQB0xgPvyF3Ca3psTL90nnjBLnq5qMYVjTfTj-Gr8QVvxusG811VMSaKBb8PfQRD8JAXgp4lmSb0pqEL1~~1ccTaT55-20NbrNpoNFTDF5qQosNMUg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic linkage maps of RMF2, an F2 population derived from a cross between Micro-Tom and Regina. Bars on the left and right sides indicate linkage group maps based on SNP loci detected in the tomato reference genome (red lines) and a de novo assembly of ddRAD-Seq data (blue lines). Bars between the two maps indicate the physical map of the tomato genome. The density of SNPs detected using WGS data for the two cultivars is indicated by the darkness of green lines. Loci that are identical between the genetic and physical maps are connected by lines." path-from-xml="dsw00404.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199537/dsw00404.jpeg" path-from-xml="dsw00404.jpeg" target="_blank">View large</a><a section="87199537" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw00404.jpeg?Expires=2147483647&Signature=QXU3Bfob7QdCsPvb11P6f8pCfrS~XrM9HJrEi-BDUw7l4NVNyiiSWq8PUlnjysVkpvcwyOdO35Kg7R-1xCt84E7BtVxTABAS5IjH2qXCzJAPq6RDh~vnzBeR1D3WP03PSKtgRX6UeIAmYyD~Qrk3fle~LfgJvAgJ0nEa2A9dvd4T-3PMe0i~jo~-qTm02ZcY2awdBG63zhnfW-I8vLKwxHh0Pp5nq-2SAmpqJeT3BDniLCQvRnQdXe9WeVINFN-rXUZ3dzU3FEnwzfI8BOwaF0rYHkSBTYkk3sM~~HwT16SggrCns4R-eeLAw0Zl64NJkncgIbZzBIBJ95aMcn3pyA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199537&ar=1745392&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00404.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic linkage maps of RMF2, an F<sub>2</sub> population derived from a cross between Micro-Tom and Regina. Bars on the left and right sides indicate linkage group maps based on SNP loci detected in the tomato reference genome (red lines) and a <em>de novo</em> assembly of ddRAD-Seq data (blue lines). Bars between the two maps indicate the physical map of the tomato genome. The density of SNPs detected using WGS data for the two cultivars is indicated by the darkness of green lines. Loci that are identical between the genetic and physical maps are connected by lines.</p></div></div><div content-id="DSW004F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/m_dsw00404.jpeg?Expires=2147483647&Signature=PmlkmC8XCRZA5~eNeuiCaRLv5pDSXYGMYJWaR9zdqNz0ZWv66d9ORn4LgBPZjO5tVW5L-mr1OVsC1hdwW~4GKxVmwNZAe9K~rEWcGqZFGpRXLtaPq-QNugWxYCrHB3QLq0QjdMaUaphG4D81GYoYoXDWZKJip~Fo~b~8Bd783E4~us6xtMOg7WaFiHoQyMonWSmdZdpy~YE6HfY8xlYGwXcOjQM9vgILUS2KrQB0xgPvyF3Ca3psTL90nnjBLnq5qMYVjTfTj-Gr8QVvxusG811VMSaKBb8PfQRD8JAXgp4lmSb0pqEL1~~1ccTaT55-20NbrNpoNFTDF5qQosNMUg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic linkage maps of RMF2, an F2 population derived from a cross between Micro-Tom and Regina. Bars on the left and right sides indicate linkage group maps based on SNP loci detected in the tomato reference genome (red lines) and a de novo assembly of ddRAD-Seq data (blue lines). Bars between the two maps indicate the physical map of the tomato genome. The density of SNPs detected using WGS data for the two cultivars is indicated by the darkness of green lines. Loci that are identical between the genetic and physical maps are connected by lines." path-from-xml="dsw00404.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199537/dsw00404.jpeg" path-from-xml="dsw00404.jpeg" target="_blank">View large</a><a section="87199537" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw00404.jpeg?Expires=2147483647&Signature=QXU3Bfob7QdCsPvb11P6f8pCfrS~XrM9HJrEi-BDUw7l4NVNyiiSWq8PUlnjysVkpvcwyOdO35Kg7R-1xCt84E7BtVxTABAS5IjH2qXCzJAPq6RDh~vnzBeR1D3WP03PSKtgRX6UeIAmYyD~Qrk3fle~LfgJvAgJ0nEa2A9dvd4T-3PMe0i~jo~-qTm02ZcY2awdBG63zhnfW-I8vLKwxHh0Pp5nq-2SAmpqJeT3BDniLCQvRnQdXe9WeVINFN-rXUZ3dzU3FEnwzfI8BOwaF0rYHkSBTYkk3sM~~HwT16SggrCns4R-eeLAw0Zl64NJkncgIbZzBIBJ95aMcn3pyA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199537&ar=1745392&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00404.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic linkage maps of RMF2, an F<sub>2</sub> population derived from a cross between Micro-Tom and Regina. Bars on the left and right sides indicate linkage group maps based on SNP loci detected in the tomato reference genome (red lines) and a <em>de novo</em> assembly of ddRAD-Seq data (blue lines). Bars between the two maps indicate the physical map of the tomato genome. The density of SNPs detected using WGS data for the two cultivars is indicated by the darkness of green lines. Loci that are identical between the genetic and physical maps are connected by lines.</p></div></div><p>Next, we investigated the accuracy of SNP calling without a reference genome sequence. The experimental ddRAD-Seq reads of the parental lines were assembled <em>de novo</em> into 44,764 contigs with a total length of 12,443,360 bases, and the high-quality ddRAD-Seq reads for the parents and the 96 F<sub>2</sub> lines were subsequently aligned onto the contigs with mapping rates of 65.0% in Micro-Tom, 62.8% in Regina, and 59.0% in the F<sub>2</sub> population. The genome positions of the marker loci on the tomato genome were determined by sequence similarity searches against the SL2.50 sequences. Using the same filtering process described above, a total of 1,017 high-confidence SNPs were selected between the parents, and 781 were identified as non-redundant SNP loci in RMF2. Linkage analysis of the 781 SNPs generated a genetic map comprising 14 linkage groups (Table <a reveal-id="DSW004TB2" data-open="DSW004TB2" class="link link-reveal link-table xref-fig">2</a> and Fig. <a reveal-id="DSW004F4" data-open="DSW004F4" class="link link-reveal link-table xref-fig">4</a>), each of which corresponded to one tomato chromosome (except for chromosomes 10 and 11, which were represented by two groups apiece). The resultant map consisted of 686 SNP loci (87.8%) covering a total of 1,691.8 cM, and the order of the loci were consistent with their physical positions in the reference genome (Fig. <a reveal-id="DSW004F4" data-open="DSW004F4" class="link link-reveal link-table xref-fig">4</a>). As for the SNPs identified on SL2.50, the distributions of mapped loci were highly biased both between chromosomes and within individual chromosomes. The two mapping studies indicated that the accuracy of SNPs from our ddRAD-Seq pipeline was ∼90% or more.</p>                            <h2 scrollto-destination=87199539 id="87199539" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>We propose an analytic workflow for the ddRAD-Seq procedure (Fig. <a reveal-id="DSW004F5" data-open="DSW004F5" class="link link-reveal link-table xref-fig">5</a>). During the establishment experimental and computational data processing pipelines, we found that the prediction of SNP detectability in ddRAD-Seq facilitated optimization of experimental conditions, e.g. choices of enzymes and the density of SNPs in the genome. Although all libraries contained the same amount of sequence data, the numbers of SNPs detected by experimental ddRAD-Seq varied depending on both the combination of restriction enzymes and the density of SNPs in the genome (Fig. <a reveal-id="DSW004F2" data-open="DSW004F2" class="link link-reveal link-table xref-fig">2</a>A and B). <em>In silico</em> prediction should be also useful for optimizing experimental conditions in other plant species for which reference genome sequences and sequencing data are available. On the other hand, in plant species for which less genomic information has accumulated, small-scale pilot experiments with several combinations of restriction enzymes should be performed to determine the optimal enzymes for ddRAD-Seq experiment. </p>                        <a id="87199541" scrollto-destination="87199541"></a>
<div id="DSW004F5" data-id="DSW004F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/m_dsw00405.jpeg?Expires=2147483647&Signature=AojK9kj9tailHz6zaln5mGHad2PHK3WRuuYwfDPP-06fVV6etgAFWyRiEwnMfrTR2YLirnNuIYWCtH9z8cz7F5khwTJ4I7E12MUjf7Pg~yiG~gfC9Gsxrx~Y8WLQfCcW~x6l32Plpe~ycXoPvX9awNhYlbdqxA8ngWA8NirC9JYsmBFlvKUybgI1aFOSXeFMn~yd1UNSWgeJbb5BmeBjpb8zBXwDGQ2Ddr0pHSF2iyD8-dnV0Yb9IlD8D2QfFYePGSVTIM7BD6BlKT0HlVBXIko0F~pPggFoEzB46EiMUvDB48SaesuA-EhgwanjHnqyQWSQZFhSODsnwY9TxXxR3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The ddRAD-Seq analytical workflow based on empirical and in silico optimization." path-from-xml="dsw00405.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199541/dsw00405.jpeg" path-from-xml="dsw00405.jpeg" target="_blank">View large</a><a section="87199541" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw00405.jpeg?Expires=2147483647&Signature=Q9~WlRhzs9ItNUWD9UQhLuHGU0Vio~IvihtKdxtwNPVIo~VP6kdIe8gXhbecH6ZB2C1WraFAyiO~fFQkHeNliLjBM3VSUblQyEbuu6YhMxnfsrHyBZf20K-0flD2kf7ivn7C1KKd0UQAgYXuUeJZBHlEg3-PWiU~IAGix~JRlD4bZjr3KkrrjQRffJ4vHKfrrb-qpEzl9Sf1d-Sc0D9BN0fM4IRwbNM20Qs~Wax9vJLM8pa7nUD-nLIATw~6pUhUbj9w1x2wYWrqDo~AzkvTgA5gifHpG4WmNT1VVQ7pP2J9dFC9CSnnV5UKVH0vh88E77XlruNqzDnW1TSqkWqw~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199541&ar=1745392&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00405.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The ddRAD-Seq analytical workflow based on empirical and <em>in silico</em> optimization.</p></div></div><div content-id="DSW004F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/m_dsw00405.jpeg?Expires=2147483647&Signature=AojK9kj9tailHz6zaln5mGHad2PHK3WRuuYwfDPP-06fVV6etgAFWyRiEwnMfrTR2YLirnNuIYWCtH9z8cz7F5khwTJ4I7E12MUjf7Pg~yiG~gfC9Gsxrx~Y8WLQfCcW~x6l32Plpe~ycXoPvX9awNhYlbdqxA8ngWA8NirC9JYsmBFlvKUybgI1aFOSXeFMn~yd1UNSWgeJbb5BmeBjpb8zBXwDGQ2Ddr0pHSF2iyD8-dnV0Yb9IlD8D2QfFYePGSVTIM7BD6BlKT0HlVBXIko0F~pPggFoEzB46EiMUvDB48SaesuA-EhgwanjHnqyQWSQZFhSODsnwY9TxXxR3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The ddRAD-Seq analytical workflow based on empirical and in silico optimization." path-from-xml="dsw00405.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199541/dsw00405.jpeg" path-from-xml="dsw00405.jpeg" target="_blank">View large</a><a section="87199541" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw00405.jpeg?Expires=2147483647&Signature=Q9~WlRhzs9ItNUWD9UQhLuHGU0Vio~IvihtKdxtwNPVIo~VP6kdIe8gXhbecH6ZB2C1WraFAyiO~fFQkHeNliLjBM3VSUblQyEbuu6YhMxnfsrHyBZf20K-0flD2kf7ivn7C1KKd0UQAgYXuUeJZBHlEg3-PWiU~IAGix~JRlD4bZjr3KkrrjQRffJ4vHKfrrb-qpEzl9Sf1d-Sc0D9BN0fM4IRwbNM20Qs~Wax9vJLM8pa7nUD-nLIATw~6pUhUbj9w1x2wYWrqDo~AzkvTgA5gifHpG4WmNT1VVQ7pP2J9dFC9CSnnV5UKVH0vh88E77XlruNqzDnW1TSqkWqw~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199541&ar=1745392&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00405.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The ddRAD-Seq analytical workflow based on empirical and <em>in silico</em> optimization.</p></div></div><p>Gene-associated SNPs located on non-repetitive sequences would be biologically meaningful, being beneficial for functional genomics, molecular genetics, and marker-assisted selection in breeding. Interestingly, the rate of gene SNPs detected by the empirical ddRAD-Seq was higher than the predicted rate when <em>Pst</em>I was employed for library construction, e.g. PE, PM, and SP libraries (Fig. <a reveal-id="DSW004F3" data-open="DSW004F3" class="link link-reveal link-table xref-fig">3</a>A, B and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw004_Supplementary_Data.zip?Expires=2147483647&Signature=qosUpqhIKaFSxMnA94HtqcgimoMsVGYfDdoaMWiECGocawcA8WW2-ahZyUM-mXfg8JhQtv9WUKzWh2kTBMBBGZtGAg9fpwYyaHF6TWEPVEkJG7PdrbIyJJbh0NsZ3nNrAiuMiFRYLkEAEEilmuSCroigb47KOYGz5RgGx5LKMu53R5HedlZeQ~BQZoGQATn4M-5aM6MVQ62vqSbBfb0dg2p~1NkL5XoLpKvHESzyIfNhpE9uFjCKPIUZrFmWSodLVBFe-mb~i6fK7c~NuTRBVGjjtPI3zXGwaq0~5WNw-Cr8UIwVnwKM0CIj9hhsNUpRedbO1gP0xxnOf0QNL8ErIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). Therefore, this point as well as the number of SNPs should be considered to select optimum restriction enzymes. It seems likely that the strong enrichment of euchromatic genes in the libraries is correlated to the methylation sensitivity of restriction enzymes.<sup><a reveal-id="DSW004C5" data-open="DSW004C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Whole-genome bisulphite sequencing analysis would be helpful to verify this hypothesis.</p><p>Genome complexity, i.e. ploidy and zygosity, is another important factor that influences the choice of restriction enzymes. For inbred lines and haploids without any heterozygous loci, SNP loci can in principle be correctly genotyped with coverage of at least one high-quality read. In such cases, to obtain as many SNPs as possible, a combination of enzymes should be selected that yields SNP numbers that increase linearly with the number of sequence reads (EH library in Fig. <a reveal-id="DSW004F2" data-open="DSW004F2" class="link link-reveal link-table xref-fig">2</a>A). In contrast, plants with highly heterozygous genomes, e.g. hybrid and polyploid lines, require deep read coverage for accurate SNP detection. Therefore, to distinguish homo- and heterozygous genotypes (or, for polyploids, homologous and homoeologous genotypes), an enzyme combination should be selected that yields a gradually increasing number of SNPs (PE library in Fig. <a reveal-id="DSW004F2" data-open="DSW004F2" class="link link-reveal link-table xref-fig">2</a>A).</p><p>Reference sequences are essential for SNP detection, but they remain unavailable for many plant species. In the absence of a reference sequence, <em>de novo</em> assembly of actual ddRAD-Seq reads should be used as a reference. To simulate this situation, we performed <em>de novo</em> assembly of the ddRAD-Seq reads generated in this study. The numbers of high-quality SNP loci, non-redundant segregated data, and SNPs located on genetic maps based on the <em>de novo</em> assemblies were ∼50% of those based on the SL2.50 reference (Table <a reveal-id="DSW004TB2" data-open="DSW004TB2" class="link link-reveal link-table xref-fig">2</a>). However, the total lengths of the resultant genetic maps were almost identical, indicating that both genetic maps were saturated. These results indicate that <em>de novo</em> assembly of the ddRAD-Seq reads is sufficient to establish saturated genetic maps. Alternatively, considering recent advances in NGS technologies, whole-genome sequence data from close relatives of a target species might be available,<sup><a reveal-id="DSW004C13" data-open="DSW004C13" class="link link-ref link-reveal xref-bibr">13</a></sup> and it is generally also possible to generate WGS of the target species itself.</p><p>The numbers of SNPs detected by ddRAD-Seq varied depending on SNP density in the genome (Fig. <a reveal-id="DSW004F2" data-open="DSW004F2" class="link link-reveal link-table xref-fig">2</a>B). In other words, SNP density is a key factor influencing SNP detectability by ddRAD-Seq. Unfortunately, a strong bias in distribution of SNPs over the genome was observed between Micro-Tom and Regina (Fig. <a reveal-id="DSW004F4" data-open="DSW004F4" class="link link-reveal link-table xref-fig">4</a> and Table <a reveal-id="DSW004TB2" data-open="DSW004TB2" class="link link-reveal link-table xref-fig">2</a>), the resultant genetic map with large gaps failed to cover the entire genome. Therefore, either the <em>in silico</em> ddRAD-Seq analysis or small-scale experiments with several combinations of restriction enzymes are recommended to predict SNP availability from actual large-scale ddRAD-Seq analysis before generating mapping populations. However, if this is impossible, increasing the variety of sequencing libraries is another possible way to increase the numbers of SNPs. For instance, although 0.3% (SP) to 4.6% (EH) of SNPs in the genome were theoretically detectable using a single sequencing library, this fraction reached a maximum of 7.6% when four libraries (SP, PE, EH, and PM) were analysed simultaneously. For plants with ultra-low SNP density in the genomes from which few SNPs are expected, alternatively, the SNP chip technologies and/or target capture or target amplicon sequencing technology,<sup><a reveal-id="DSW004C31" data-open="DSW004C31" class="link link-ref link-reveal xref-bibr">31</a></sup> which tags SNPs regardless of their distances from restriction sites, might be useful; however, this approach would be more costly than ddRAD-Seq. Therefore, prediction of the expected number of SNPs based on SNP density throughout the genome would be helpful to maximize the efficiency of ddRAD-Seq analysis.</p><p>In conclusion, the ddRAD-Seq technology has the potential to simultaneously genotype SNPs throughout the genome in multiple samples.<sup><a reveal-id="DSW004C5" data-open="DSW004C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSW004C6" data-open="DSW004C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSW004C8" data-open="DSW004C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSW004C10" data-open="DSW004C10" class="link link-ref link-reveal xref-bibr">10</a></sup> The ddRAD-Seq analytical workflow and the pipeline for the data processing developed in this study (Fig. <a reveal-id="DSW004F5" data-open="DSW004F5" class="link link-reveal link-table xref-fig">5</a>), including the empirical and <em>in silico</em> optimization processes, could be used to advance genetics, genomics, and molecular breeding in both model and non-model plant species, including crops.</p>                            <h2 scrollto-destination=87199547 id="87199547" class="section-title" data-legacy-id=s5>5. Availability</h2>
<p>All sequence data obtained in this study are available from the DDBJ Sequence Read Archive under accession number DRA003569 and Kazusa Tomato Genomics DataBase (KaTomicsDB: <a class="link link-uri" href="http://www.kazusa.or.jp/tomato">http://www.kazusa.or.jp/tomato</a>).<sup>32</sup></p>                            <h2 scrollto-destination=87199549 id="87199549" class="section-title" data-legacy-id=s7>Funding</h2>
<p>Funding to pay the Open Access publication charges for this article was provided by the Kazusa DNA Research Institute.</p>                            <h2 scrollto-destination=87199551 id="87199551" class="backacknowledgements-title" >Acknowledgments</h2>
<p>We are grateful to S. Sasamoto, C. Mimani, S. Nakayama, A. Watanabe, M. Kohara, and Y. Kishida (Kazusa DNA Research Institute) for their technical assistance. Plant materials were obtained from MEXT National BioResource Project, University of Tsukuba, Japan, and the Tomato Genetic Resource Center, University of California, Davis, CA, USA. This work was supported by JSPS KAKENHI (Grant Number 24710237) and the Kazusa DNA Research Institute Foundation.</p>                            <h2 scrollto-destination=87199553 id="87199553" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSW004C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Holland</div>
               <div class="given-names">P.M.</div>
            </div>, <div class="name string-name"><div class="surname">Abramson</div> <div class="given-names">R.D.</div></div>, <div class="name string-name"><div class="surname">Watson</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Gelfand</div> <div class="given-names">D.H.</div></div></div>
          <div class="year">1991</div>, <div class="article-title">Detection of specific polymerase chain reaction product by utilizing the 5′ -&gt; 3′ exonuclease activity of <em>Thermus aquaticus</em> DNA polymerase</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">88</div>, <div class="fpage">7276</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20of%20specific%20polymerase%20chain%20reaction%20product%20by%20utilizing%20the%205%E2%80%B2%20-%3E%203%E2%80%B2%20exonuclease%20activity%20of%20Thermus%20aquaticus%20DNA%20polymerase&amp;author=P.M.%20Holland&amp;author=R.D.%20Abramson&amp;author=R.%20Watson&amp;author=D.H.%20Gelfand&amp;publication_year=1991&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=88&amp;pages=7276-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.88.16.7276" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.88.16.7276" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.88.16.7276"> </span></div> </div></div></div></div></div><div content-id="DSW004C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gupta</div>
               <div class="given-names">P.K.</div>
            </div>, <div class="name string-name"><div class="surname">Rustgi</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Mir</div> <div class="given-names">R.R.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">Array-based high-throughput DNA markers for crop improvement</div>, <div class="source">Heredity</div> , <div class="volume">101</div>, <div class="fpage">5</div>–<div class="lpage">18</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Array-based%20high-throughput%20DNA%20markers%20for%20crop%20improvement&amp;author=P.K.%20Gupta&amp;author=S.%20Rustgi&amp;author=R.R.%20Mir&amp;publication_year=2008&amp;journal=Heredity&amp;volume=101&amp;pages=5-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/hdy.2008.35" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fhdy.2008.35" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fhdy.2008.35"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18461083" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Xing</div>
               <div class="given-names">Y.</div>
            </div>
          </div>
          <div class="year">2014</div>, <div class="article-title">SNP array – a powerful platform to accelerate genetic studies and breeding</div>, <div class="source">J. Plant Biochem. Physiol.</div> , <div class="volume">2</div>, <div class="fpage">e119</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20array%20%E2%80%93%20a%20powerful%20platform%20to%20accelerate%20genetic%20studies%20and%20breeding&amp;author=Y.%20Xing&amp;publication_year=2014&amp;journal=J.%20Plant%20Biochem.%20Physiol.&amp;volume=2&amp;pages=e119" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4172/2329-9029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4172%2F2329-9029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4172%2F2329-9029"> </span></div> </div></div></div></div></div><div content-id="DSW004C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kumar</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Banks</div> <div class="given-names">T.W.</div></div>, <div class="name string-name"><div class="surname">Cloutier</div> <div class="given-names">S.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">SNP discovery through next-generation sequencing and its applications</div>, <div class="source">Int. J. Plant Genomics</div> , <div class="volume">20,12</div>, <div class="fpage">831460</div>.</div></div></div></div><div content-id="DSW004C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Davey</div>
               <div class="given-names">J.W.</div>
            </div>, <div class="name string-name"><div class="surname">Hohenlohe</div> <div class="given-names">P.A.</div></div>, <div class="name string-name"><div class="surname">Etter</div> <div class="given-names">P.D.</div></div>, <div class="name string-name"><div class="surname">Boone</div> <div class="given-names">J.Q.</div></div>, <div class="name string-name"><div class="surname">Catchen</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Blaxter</div> <div class="given-names">M.L.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Genome-wide genetic marker discovery and genotyping using next-generation sequencing</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">12</div>, <div class="fpage">499</div>–<div class="lpage">510</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20genetic%20marker%20discovery%20and%20genotyping%20using%20next-generation%20sequencing&amp;author=J.W.%20Davey&amp;author=P.A.%20Hohenlohe&amp;author=P.D.%20Etter&amp;author=J.Q.%20Boone&amp;author=J.M.%20Catchen&amp;author=M.L.%20Blaxter&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=12&amp;pages=499-510" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21681211" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Elshire</div>
               <div class="given-names">R.J.</div>
            </div>, <div class="name string-name"><div class="surname">Glaubitz</div> <div class="given-names">J.C.</div></div>, <div class="name string-name"><div class="surname">Sun</div> <div class="given-names">Q.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species</div>, <div class="source">PLoS ONE</div> , <div class="volume">6</div>, <div class="fpage">e19379</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20robust%2C%20simple%20genotyping-by-sequencing%20(GBS)%20approach%20for%20high%20diversity%20species&amp;author=R.J.%20Elshire&amp;author=J.C.%20Glaubitz&amp;author=Q.%20Sun&amp;publication_year=2011&amp;journal=PLoS%20ONE&amp;volume=6&amp;pages=e19379" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0019379" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0019379" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0019379"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21573248" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Poland</div>
               <div class="given-names">J.A.</div>
            </div>, <div class="name string-name"><div class="surname">Rife</div> <div class="given-names">T.W.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Genotyping-by-sequencing for plant breeding and genetics</div>, <div class="source">Plant Genome</div> , <div class="volume">5</div>, <div class="fpage">92</div>–<div class="lpage">102</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genotyping-by-sequencing%20for%20plant%20breeding%20and%20genetics&amp;author=J.A.%20Poland&amp;author=T.W.%20Rife&amp;publication_year=2012&amp;journal=Plant%20Genome&amp;volume=5&amp;pages=92-102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3835/plantgenome2012.05.0005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3835%2Fplantgenome2012.05.0005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3835%2Fplantgenome2012.05.0005"> </span></div> </div></div></div></div></div><div content-id="DSW004C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Baird</div>
               <div class="given-names">N.A.</div>
            </div>, <div class="name string-name"><div class="surname">Etter</div> <div class="given-names">P.D.</div></div>, <div class="name string-name"><div class="surname">Atwood</div> <div class="given-names">T.S.</div></div>et al.  </div>. <div class="year">2008</div>, <div class="article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers</div>, <div class="source">PLoS ONE</div> , <div class="volume">3</div>, <div class="fpage">e3376</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20SNP%20discovery%20and%20genetic%20mapping%20using%20sequenced%20RAD%20markers&amp;author=N.A.%20Baird&amp;author=P.D.%20Etter&amp;author=T.S.%20Atwood&amp;publication_year=2008&amp;journal=PLoS%20ONE&amp;volume=3&amp;pages=e3376" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0003376" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0003376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0003376"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18852878" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Clevenger</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Chavarro</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pearl</div> <div class="given-names">S.A.</div></div>, <div class="name string-name"><div class="surname">Ozias-Akins</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Jackson</div> <div class="given-names">S.A.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">Single nucleotide polymorphism identification in polyploids: a review, example, and recommendations</div>, <div class="source">Mol. Plant</div> , <div class="volume">8</div>, <div class="fpage">831</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single%20nucleotide%20polymorphism%20identification%20in%20polyploids%3A%20a%20review%2C%20example%2C%20and%20recommendations&amp;author=J.%20Clevenger&amp;author=C.%20Chavarro&amp;author=S.A.%20Pearl&amp;author=P.%20Ozias-Akins&amp;author=S.A.%20Jackson&amp;publication_year=2015&amp;journal=Mol.%20Plant&amp;volume=8&amp;pages=831-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molp.2015.02.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molp.2015.02.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molp.2015.02.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25676455" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Peterson</div>
               <div class="given-names">B.K.</div>
            </div>, <div class="name string-name"><div class="surname">Weber</div> <div class="given-names">J.N.</div></div>, <div class="name string-name"><div class="surname">Kay</div> <div class="given-names">E.H.</div></div>, <div class="name string-name"><div class="surname">Fisher</div> <div class="given-names">H.S.</div></div>, <div class="name string-name"><div class="surname">Hoekstra</div> <div class="given-names">H.E.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species</div>, <div class="source">PLoS ONE</div> , <div class="volume">7</div>, <div class="fpage">e37135</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Double%20digest%20RADseq%3A%20an%20inexpensive%20method%20for%20de%20novo%20SNP%20discovery%20and%20genotyping%20in%20model%20and%20non-model%20species&amp;author=B.K.%20Peterson&amp;author=J.N.%20Weber&amp;author=E.H.%20Kay&amp;author=H.S.%20Fisher&amp;author=H.E.%20Hoekstra&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e37135" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0037135" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0037135" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0037135"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22675423" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Catchen</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Amores</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Hohenlohe</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Cresko</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Postlethwait</div> <div class="given-names">J.</div></div></div>
          <div class="year">2011</div>, <div class="article-title"><em>Stacks</em>: building and genotyping loci <em>de novo</em> from short-read sequences</div>, <div class="source">G3</div> , <div class="volume">1</div>, <div class="fpage">171</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stacks%3A%20building%20and%20genotyping%20loci%20de%20novo%20from%20short-read%20sequences&amp;author=J.%20Catchen&amp;author=A.%20Amores&amp;author=P.%20Hohenlohe&amp;author=W.%20Cresko&amp;author=J.%20Postlethwait&amp;publication_year=2011&amp;journal=G3&amp;volume=1&amp;pages=171-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.111.000240" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.111.000240" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.111.000240"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22384329" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Glaubitz</div>
               <div class="given-names">J.C.</div>
            </div>, <div class="name string-name"><div class="surname">Casstevens</div> <div class="given-names">T.M.</div></div>, <div class="name string-name"><div class="surname">Lu</div> <div class="given-names">F.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">TASSEL-GBS: a high capacity genotyping by sequencing analysis pipeline</div>, <div class="source">PLoS ONE</div> , <div class="volume">9</div>, <div class="fpage">e90346</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TASSEL-GBS%3A%20a%20high%20capacity%20genotyping%20by%20sequencing%20analysis%20pipeline&amp;author=J.C.%20Glaubitz&amp;author=T.M.%20Casstevens&amp;author=F.%20Lu&amp;publication_year=2014&amp;journal=PLoS%20ONE&amp;volume=9&amp;pages=e90346" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0090346" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0090346" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0090346"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24587335" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Michael</div>
               <div class="given-names">T.P.</div>
            </div>, <div class="name string-name"><div class="surname">VanBuren</div> <div class="given-names">R.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">Progress, challenges and the future of crop genomes</div>, <div class="source">Curr. Opin. Plant Biol.</div> , <div class="volume">24</div>, <div class="fpage">71</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Progress%2C%20challenges%20and%20the%20future%20of%20crop%20genomes&amp;author=T.P.%20Michael&amp;author=R.%20VanBuren&amp;publication_year=2015&amp;journal=Curr.%20Opin.%20Plant%20Biol.&amp;volume=24&amp;pages=71-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2015.02.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2015.02.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2015.02.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25703261" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="collab">The Tomato Genome Consortium</div>. <div class="year">2012</div>, <div class="article-title">The tomato genome sequence provides insights into fleshy fruit evolution</div>, <div class="source">Nature</div> , <div class="volume">485</div>, <div class="fpage">635</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11119" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11119" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11119"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22660326" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shirasawa</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Fukuoka</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Matsunaga</div> <div class="given-names">H.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Genome-wide association studies using single nucleotide polymorphism markers developed by re-sequencing of the genomes of cultivated tomato</div>, <div class="source">DNA Res.</div> , <div class="volume">20</div>, <div class="fpage">593</div>–<div class="lpage">603</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20studies%20using%20single%20nucleotide%20polymorphism%20markers%20developed%20by%20re-sequencing%20of%20the%20genomes%20of%20cultivated%20tomato&amp;author=K.%20Shirasawa&amp;author=H.%20Fukuoka&amp;author=H.%20Matsunaga&amp;publication_year=2013&amp;journal=DNA%20Res.&amp;volume=20&amp;pages=593-603" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23903436" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shirasawa</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Nunome</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Tabata</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Isobe</div> <div class="given-names">S.</div></div></div>
          <div class="year">2016</div>, <div class="article-title">Genome-wide survey of artificial mutations induced by ethyl methanesulfonate and gamma rays in tomato</div>, <div class="source">Plant Biotechnol. J.</div> , <div class="volume">14</div>, <div class="fpage">51</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20survey%20of%20artificial%20mutations%20induced%20by%20ethyl%20methanesulfonate%20and%20gamma%20rays%20in%20tomato&amp;author=K.%20Shirasawa&amp;author=H.%20Hirakawa&amp;author=T.%20Nunome&amp;author=S.%20Tabata&amp;author=S.%20Isobe&amp;publication_year=2016&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=14&amp;pages=51-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.12348" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.12348" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.12348"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25689669" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Langmead</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Fast gapped-read alignment with Bowtie 2</div>, <div class="source">Nat. Methods</div> , <div class="volume">9</div>, <div class="fpage">357</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;author=B.%20Langmead&amp;author=S.L.%20Salzberg&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=357-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Langmead</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pop</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</div>, <div class="source">Genome Biol.</div> , <div class="volume">10</div>, <div class="fpage">R25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ultrafast%20and%20memory-efficient%20alignment%20of%20short%20DNA%20sequences%20to%20the%20human%20genome&amp;author=B.%20Langmead&amp;author=C.%20Trapnell&amp;author=M.%20Pop&amp;author=S.L.%20Salzberg&amp;publication_year=2009&amp;journal=Genome%20Biol.&amp;volume=10&amp;pages=R25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-3-r25" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-3-r25" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-3-r25"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19261174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Handsaker</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Wysoker</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">The sequence alignment/map format and SAMtools</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">2078</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20sequence%20alignment%2Fmap%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Quinlan</div>
               <div class="given-names">A.R.</div>
            </div>, <div class="name string-name"><div class="surname">Hall</div> <div class="given-names">I.M.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">BEDTools: a flexible suite of utilities for comparing genomic features</div>, <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">841</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BEDTools%3A%20a%20flexible%20suite%20of%20utilities%20for%20comparing%20genomic%20features&amp;author=A.R.%20Quinlan&amp;author=I.M.%20Hall&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=841-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20110278" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Danecek</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Auton</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Abecasis</div> <div class="given-names">G.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">The variant call format and VCFtools</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">2156</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20variant%20call%20format%20and%20VCFtools&amp;author=P.%20Danecek&amp;author=A.%20Auton&amp;author=G.%20Abecasis&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2156-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr330" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr330" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr330"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21653522" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Browning</div>
               <div class="given-names">S.R.</div>
            </div>, <div class="name string-name"><div class="surname">Browning</div> <div class="given-names">B.L.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="volume">81</div>, <div class="fpage">1084</div>–<div class="lpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20and%20accurate%20haplotype%20phasing%20and%20missing-data%20inference%20for%20whole-genome%20association%20studies%20by%20use%20of%20localized%20haplotype%20clustering&amp;author=S.R.%20Browning&amp;author=B.L.%20Browning&amp;publication_year=2007&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=81&amp;pages=1084-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/521987" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F521987" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F521987"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17924348" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cingolani</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Platts</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">L.L.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of <em>Drosophila melanogaster</em> strain <em>w</em><sup>1118</sup>; <em>iso</em>-2; <em>iso</em>-3</div>, <div class="source">Fly</div> , <div class="volume">6</div>, <div class="fpage">80</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20program%20for%20annotating%20and%20predicting%20the%20effects%20of%20single%20nucleotide%20polymorphisms%2C%20SnpEff%3A%20SNPs%20in%20the%20genome%20of%20Drosophila%20melanogaster%20strain%20w1118%3B%20iso-2%3B%20iso-3&amp;author=P.%20Cingolani&amp;author=A.%20Platts&amp;author=L.L.%20Wang&amp;publication_year=2012&amp;journal=Fly&amp;volume=6&amp;pages=80-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/fly.19695" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Ffly.19695" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Ffly.19695"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22728672" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Altschul</div>
               <div class="given-names">S.F.</div>
            </div>, <div class="name string-name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>, <div class="name string-name"><div class="surname">Schaffer</div> <div class="given-names">A.A.</div></div>et al.  </div>. <div class="year">1997</div>, <div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">25</div>, <div class="fpage">3389</div>–<div class="lpage">402</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Van Ooijen</div>
               <div class="given-names">J.W.</div>
            </div>
          </div>
          <div class="year">2006</div>, <div class="source">JoinMap®4, Software for the Calculation of Genetic Linkage Maps in Experimental Populations</div> , <div class="publisher-name">Kyazma BV</div>, <div class="publisher-loc">Wageningen</div>, The <div class="publisher-loc">Netherlands</div>.</div></div></div></div><div content-id="DSW004C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Voorrips</div>
               <div class="given-names">R.E.</div>
            </div>
          </div>
          <div class="year">2002</div>, <div class="article-title">MapChart: software for the graphical presentation of linkage maps and QTLs</div>, <div class="source">J. Hered.</div> , <div class="volume">93</div>, <div class="fpage">77</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MapChart%3A%20software%20for%20the%20graphical%20presentation%20of%20linkage%20maps%20and%20QTLs&amp;author=R.E.%20Voorrips&amp;publication_year=2002&amp;journal=J.%20Hered.&amp;volume=93&amp;pages=77-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/93.1.77" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2F93.1.77" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2F93.1.77"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12011185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shearer</div>
               <div class="given-names">L.A.</div>
            </div>, <div class="name string-name"><div class="surname">Anderson</div> <div class="given-names">L.K.</div></div>, <div class="name string-name"><div class="surname">de Jong</div> <div class="given-names">H.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Fluorescence <em>in situ</em> hybridization and optical mapping to correct scaffold arrangement in the tomato genome</div>, <div class="source">G3</div> , <div class="volume">4</div>, <div class="fpage">1395</div>–<div class="lpage">405</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fluorescence%20in%20situ%20hybridization%20and%20optical%20mapping%20to%20correct%20scaffold%20arrangement%20in%20the%20tomato%20genome&amp;author=L.A.%20Shearer&amp;author=L.K.%20Anderson&amp;author=H.%20de%20Jong&amp;publication_year=2014&amp;journal=G3&amp;volume=4&amp;pages=1395-405" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.114.011197" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.114.011197" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.114.011197"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24879607" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hirakawa</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Ohyama</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Genome-wide SNP genotyping to infer the effects on gene functions in tomato</div>, <div class="source">DNA Res.</div> , <div class="volume">20</div>, <div class="fpage">221</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20SNP%20genotyping%20to%20infer%20the%20effects%20on%20gene%20functions%20in%20tomato&amp;author=H.%20Hirakawa&amp;author=K.%20Shirasawa&amp;author=A.%20Ohyama&amp;publication_year=2013&amp;journal=DNA%20Res.&amp;volume=20&amp;pages=221-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23482505" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="collab">The 100 Tomato Genome Sequencing Consortium</div>, <div class="person-group"><div class="name string-name"><div class="surname">Aflitos</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Schijlen</div> <div class="given-names">E.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Exploring genetic variation in the tomato (<em>Solanum</em> section <em>Lycopersicon</em>) clade by whole-genome sequencing</div>, <div class="source">Plant J.</div> , <div class="volume">80</div>, <div class="fpage">136</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploring%20genetic%20variation%20in%20the%20tomato%20(Solanum%20section%20Lycopersicon)%20clade%20by%20whole-genome%20sequencing&amp;author=S.%20Aflitos&amp;author=E.%20Schijlen&amp;publication_year=2014&amp;journal=Plant%20J.&amp;volume=80&amp;pages=136-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2014.80.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2014.80.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2014.80.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25039268" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ercolano</div>
               <div class="given-names">M.R.</div>
            </div>, <div class="name string-name"><div class="surname">Sacco</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ferriello</div> <div class="given-names">F.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Patchwork sequencing of tomato San Marzano and Vesuviano varieties highlights genome-wide variations</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">138</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Patchwork%20sequencing%20of%20tomato%20San%20Marzano%20and%20Vesuviano%20varieties%20highlights%20genome-wide%20variations&amp;author=M.R.%20Ercolano&amp;author=A.%20Sacco&amp;author=F.%20Ferriello&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=138" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-138" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-138" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-138"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24548308" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mamanova</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Coffey</div> <div class="given-names">A.J.</div></div>, <div class="name string-name"><div class="surname">Scott</div> <div class="given-names">C.E.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Target-enrichment strategies for next-generation sequencing</div>, <div class="source">Nat. Methods</div> , <div class="volume">7</div>, <div class="fpage">111</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Target-enrichment%20strategies%20for%20next-generation%20sequencing&amp;author=L.%20Mamanova&amp;author=A.J.%20Coffey&amp;author=C.E.%20Scott&amp;publication_year=2010&amp;journal=Nat.%20Methods&amp;volume=7&amp;pages=111-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1419" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1419" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1419"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20111037" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW004C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW004C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shirasawa</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">DNA marker applications to molecular genetics and genomics in tomato</div>, <div class="source">Breed. Sci.</div> , <div class="volume">63</div>, <div class="fpage">21</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20marker%20applications%20to%20molecular%20genetics%20and%20genomics%20in%20tomato&amp;author=K.%20Shirasawa&amp;author=H.%20Hirakawa&amp;publication_year=2013&amp;journal=Breed.%20Sci.&amp;volume=63&amp;pages=21-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.63.21" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.63.21" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.63.21"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23641178" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=1745392&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw004/2/dsw004_Supplementary_Data.zip?Expires=2147483647&amp;Signature=qosUpqhIKaFSxMnA94HtqcgimoMsVGYfDdoaMWiECGocawcA8WW2-ahZyUM-mXfg8JhQtv9WUKzWh2kTBMBBGZtGAg9fpwYyaHF6TWEPVEkJG7PdrbIyJJbh0NsZ3nNrAiuMiFRYLkEAEEilmuSCroigb47KOYGz5RgGx5LKMu53R5HedlZeQ~BQZoGQATn4M-5aM6MVQ62vqSbBfb0dg2p~1NkL5XoLpKvHESzyIfNhpE9uFjCKPIUZrFmWSodLVBFe-mb~i6fK7c~NuTRBVGjjtPI3zXGwaq0~5WNw-Cr8UIwVnwKM0CIj9hhsNUpRedbO1gP0xxnOf0QNL8ErIg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,767</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000375088800007&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">12</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw004"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="1745392"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000375088800007&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f2%2f145%2f1745392">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000375088800007&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (12)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F2%2F145%2F1745392" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/1745392'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Analytical workflow of double-digest restriction site-associated DNA sequencing based on empirical and in silico optimization in tomato" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw004");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f2%2f145%2f1745392">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

