

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/4/293/348350","@type":"ScholarlyArticle","name":"Epigenetic regulation of serotype expression antagonizes transcriptome dynamics in  Paramecium tetraurelia ","datePublished":"2015-07-31","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2015-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv014","keywords":["antigenic variation","epigenetics","ciliate","telomere position effect","heat-shock"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/4/293/348350","author":[{"name":"Cheaib, Miriam","affiliation":"1  Molecular Cell Dynamics, Centre for Human and Molecular Biology, Saarland University ,  Saarbrücken 66123 ,  Germany","@type":"Person"},{"name":"Dehghani Amirabad, Azim","affiliation":"1  Molecular Cell Dynamics, Centre for Human and Molecular Biology, Saarland University ,  Saarbrücken 66123 ,  Germany    2  Cluster of Excellence for Multimodal Computing and Interaction, Saarland University and Max Planck Institute for Informatics ,  Saarbrücken 66123 ,  Germany","@type":"Person"},{"name":"Nordström, Karl J. V.","affiliation":"3  Epigenetics Department ,  Centre for Human and Molecular Biology, Saarland University ,  Saarbrücken 66123 ,  Germany","@type":"Person"},{"name":"Schulz, Marcel H.","affiliation":"2  Cluster of Excellence for Multimodal Computing and Interaction, Saarland University and Max Planck Institute for Informatics ,  Saarbrücken 66123 ,  Germany","@type":"Person"},{"name":"Simon, Martin","affiliation":"1  Molecular Cell Dynamics, Centre for Human and Molecular Biology, Saarland University ,  Saarbrücken 66123 ,  Germany","@type":"Person"}],"description":"Abstract.  Phenotypic variation of a single genotype is achieved by alterations in gene expression patterns. Regulation of such alterations depends on their tim","pageStart":"293","pageEnd":"305","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/4/1/m_cover.gif?Expires=2147483647&Signature=Ufin3LLH~Iqb2kw-hetKqzdx2YgWiSwuSGu7FcRPyZ0YMXPxK67~EvZK~D~FaBMUMtkfx2C3nicLRXrG7LQSzXkxs1Qyu1rF6Hbn-vB-awwpaXOgx90CafYjmsJlX0TeG5UBY2DnHbv1s5YegTLh88o5c9kHiz4IICTc2SgUfnIM~hO2aQPqVNMfxHD9dsgu8Ly4qu3h4vkVcBZFQVgjv-cFeqoqOFVvCZqW6Vq3z-Hi91~oLc58RRI-PjbrcLYKmszwtDPbL2sZKRQky2J2HWAa2PZzNl-CMH92uFmvRLgkUeyANQZ-s6eHxAaArS5e4XiVxAKXHDRMAVOPl6~asA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Epigenetic regulation of serotype expression antagonizes transcriptome dynamics in  Paramecium tetraurelia ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/4/1/m_cover.gif?Expires=2147483647&Signature=Ufin3LLH~Iqb2kw-hetKqzdx2YgWiSwuSGu7FcRPyZ0YMXPxK67~EvZK~D~FaBMUMtkfx2C3nicLRXrG7LQSzXkxs1Qyu1rF6Hbn-vB-awwpaXOgx90CafYjmsJlX0TeG5UBY2DnHbv1s5YegTLh88o5c9kHiz4IICTc2SgUfnIM~hO2aQPqVNMfxHD9dsgu8Ly4qu3h4vkVcBZFQVgjv-cFeqoqOFVvCZqW6Vq3z-Hi91~oLc58RRI-PjbrcLYKmszwtDPbL2sZKRQky2J2HWAa2PZzNl-CMH92uFmvRLgkUeyANQZ-s6eHxAaArS5e4XiVxAKXHDRMAVOPl6~asA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Epigenetic regulation of serotype expression antagonizes transcriptome dynamics in  Paramecium tetraurelia " />
    <meta property="og:description" content="Abstract.  Phenotypic variation of a single genotype is achieved by alterations in gene expression patterns. Regulation of such alterations depends on their tim" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/4/293/348350" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-07-31" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Cheaib, Miriam" /><meta name="citation_author_institution" content="Molecular Cell Dynamics, Centre for Human and Molecular Biology, Saarland University, Saarbrücken 66123, Germany" /><meta name="citation_author" content="Dehghani Amirabad, Azim" /><meta name="citation_author_institution" content="Molecular Cell Dynamics, Centre for Human and Molecular Biology, Saarland University, Saarbrücken 66123, Germany" /><meta name="citation_author_institution" content="Cluster of Excellence for Multimodal Computing and Interaction, Saarland University and Max Planck Institute for Informatics, Saarbrücken 66123, Germany" /><meta name="citation_author" content="Nordström, Karl J. V." /><meta name="citation_author_institution" content="Epigenetics Department, Centre for Human and Molecular Biology, Saarland University, Saarbrücken 66123, Germany" /><meta name="citation_author" content="Schulz, Marcel H." /><meta name="citation_author_institution" content="Cluster of Excellence for Multimodal Computing and Interaction, Saarland University and Max Planck Institute for Informatics, Saarbrücken 66123, Germany" /><meta name="citation_author" content="Simon, Martin" /><meta name="citation_author_institution" content="Molecular Cell Dynamics, Centre for Human and Molecular Biology, Saarland University, Saarbrücken 66123, Germany" /><meta name="citation_title" content="Epigenetic regulation of serotype expression antagonizes transcriptome dynamics in Paramecium tetraurelia" /><meta name="citation_firstpage" content="293" /><meta name="citation_lastpage" content="305" /><meta name="citation_doi" content="10.1093/dnares/dsv014" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2015/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Phenotypic plasticity: molecular mechanisms and adaptive significance;  citation_journal_title=Compr. Physiol.;  citation_year=2012;  citation_volume=2;  citation_pages=1417-39; " /><meta name="citation_reference" content="citation_title=Epigenetic regulation of stem cells: the role of chromatin in cell differentiation;  citation_journal_title=Adv. Exp. Med. Biol.;  citation_year=2013;  citation_volume=786;  citation_pages=307-28; " /><meta name="citation_reference" content="citation_title=Protein folding in the cytoplasm and the heat shock response;  citation_journal_title=Cold Spring Harb. Perspect. Biol.;  citation_year=2010;  citation_volume=2;  citation_pages=a004390" /><meta name="citation_reference" content="citation_title=Phenotypic plasticity in development and evolution: facts and concepts;  citation_journal_title=Philos. T. Roy. Soc. B.;  citation_year=2010;  citation_volume=365;  citation_pages=547-56; " /><meta name="citation_reference" content="citation_title=Lipids and lipid metabolism in eukaryotic algae;  citation_journal_title=Prog. Lipid Res.;  citation_year=2006;  citation_volume=45;  citation_pages=160-86; " /><meta name="citation_reference" content="citation_title=New insights into the phylogeny of eukaryotes based on ciliate Hsp70 sequences;  citation_journal_title=Mol. Biol. Evol.;  citation_year=1998;  citation_volume=15;  citation_pages=943-56; " /><meta name="citation_reference" content="citation_title=Convergent evolution of heat-inducibility during subfunctionalization of the Hsp70 gene family;  citation_journal_title=BMC Evol. Biol.;  citation_year=2013;  citation_volume=13;  citation_pages=49" /><meta name="citation_reference" content="citation_title=Dynamic chromatin remodelling of ciliate macronuclear DNA as determined by an optimized chromatin immunoprecipitation (ChIP) method for Paramecium tetraurelia;  citation_journal_title=Appl. Microbiol. Biotechnol.;  citation_year=2013;  citation_volume=97;  citation_pages=2661-70; " /><meta name="citation_reference" content="citation_title=Genome-defence small RNAs exapted for epigenetic mating-type inheritance;  citation_journal_title=Nature;  citation_year=2014;  citation_volume=509;  citation_pages=447-52; " /><meta name="citation_reference" content="citation_title=The antigen system of Paramecium aurelia;  citation_journal_title=Int. Rev. Cytol.;  citation_year=1957;  citation_volume=6;  citation_pages=1-23; " /><meta name="citation_reference" content="citation_title=Distinct RNAdependent RNA polymerases are required for RNAi triggered by doublestranded RNA versus truncated transgenes in Paramecium tetraurelia;  citation_journal_title=Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=4092-107; " /><meta name="citation_reference" content="citation_title=Genomic characterization of variable surface antigens reveals a telomere position effect as a prerequisite for RNA interference-mediated silencing in Paramecium tetraurelia;  citation_journal_title=mBio.;  citation_year=2014;  citation_volume=5;  citation_pages=e01328-14; " /><meta name="citation_reference" content="citation_title=Unicellular eukaryotes as models in cell and molecular biology: critical appraisal of their past and future value;  citation_journal_title=Int. Rev. Cell Mol. Biol.;  citation_year=2014;  citation_volume=309;  citation_pages=141-98; " /><meta name="citation_reference" content="citation_publisher=Cambridge University Press, New York;  citation_title=The genetics of Paramecium Aurelia;  citation_year=1954; " /><meta name="citation_reference" content="citation_title=Cutadapt removes adapter sequences from high-throughput sequencing reads;  citation_journal_title=EMBnet J.;  citation_year=2011;  citation_volume=17;  citation_pages=10-2; " /><meta name="citation_reference" content="citation_title=The Pfam protein families database;  citation_journal_title=Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=D290-301; " /><meta name="citation_reference" content="citation_title=MEGA6: molecular evolutionary genetics analysis version 6.0.;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2013;  citation_volume=30;  citation_pages=2725-9; " /><meta name="citation_reference" content="citation_title=Sailfish enables alignment-free isoform quantification from RNA-seq reads using lightweight algorithms;  citation_journal_title=Nat. Biotechnol.;  citation_year=2014;  citation_volume=32;  citation_pages=462-4; " /><meta name="citation_reference" content="citation_title=Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2;  citation_journal_title=Genome Biol.;  citation_year=2014;  citation_volume=15;  citation_pages=550" /><meta name="citation_reference" content="citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing;  citation_journal_title=J. R. Stat. Soc. Ser. B. (Methodological);  citation_year=1995;  citation_volume=57;  citation_pages=289-300; " /><meta name="citation_reference" content="citation_title=Expression cartography of human tissues using self organizing maps;  citation_journal_title=BMC Bioinformatics;  citation_year=2011;  citation_volume=12;  citation_pages=306" /><meta name="citation_reference" content="citation_title=ParameciumDB in 2011: new tools and new data for functional and comparative genomics of the model ciliate Paramecium tetraurelia;  citation_journal_title=Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=D632-6; " /><meta name="citation_reference" content="citation_title=Ontologizer 2.0–a multifunctional tool for GO term enrichment analysis and data exploration;  citation_journal_title=Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=1650-1; " /><meta name="citation_reference" content="citation_title=Global trends of whole genome duplications revealed by the ciliate Paramecium tetraurelia;  citation_journal_title=Nature;  citation_year=2006;  citation_volume=444;  citation_pages=171-8; " /><meta name="citation_reference" content="citation_title=REVIGO summarizes and visualizes long lists of Gene Ontology terms;  citation_journal_title=PLoS ONE;  citation_year=2011;  citation_volume=6;  citation_pages=e21800" /><meta name="citation_reference" content="citation_title=The control of histone gene expression;  citation_journal_title=Biochem. Soc. Trans.;  citation_year=2012;  citation_volume=40;  citation_pages=880-5; " /><meta name="citation_reference" content="citation_title=Nucleosome loss leads to global transcriptional up-regulation and genomic instability during yeast aging;  citation_journal_title=Genes Dev.;  citation_year=2013;  citation_volume=28;  citation_pages=396-408; " /><meta name="citation_reference" content="citation_title=Histone supply regulates S phase timing and cell cycle progression;  citation_journal_title=eLife;  citation_year=2014;  citation_volume=3;  citation_pages=e02443" /><meta name="citation_reference" content="citation_title=Systems-wide analysis of acclimation responses to long-term heat stress and recovery in the photosynthetic model organism Chlamydomonas reinhardtii;  citation_journal_title=Plant Cell;  citation_year=2014;  citation_volume=26;  citation_pages=4270-97; " /><meta name="citation_reference" content="citation_title=Differential expression of histone H3 genes and selective association of the variant H3.7 with a specific sequence class in Stylonychia macronuclear development;  citation_journal_title=Epigenetics Chromatin;  citation_year=2014;  citation_volume=7;  citation_pages=1-15; " /><meta name="citation_reference" content="citation_title=Comparison of gene expression of Paramecium bursaria with and without Chlorella variabilis symbionts;  citation_journal_title=BMC Genomics.;  citation_year=2014;  citation_volume=15;  citation_pages=183" /><meta name="citation_reference" content="citation_title=The endosymbiotic bacterium Holospora obtusa enhances heat-shock gene expression of the host Paramecium caudatum;  citation_journal_title=J. Euk. Microbiol.;  citation_year=2003;  citation_volume=50;  citation_pages=293-8; " /><meta name="citation_reference" content="citation_title=Paramecium caudatum acquires heat-shock resisitance in ciliare movement by infection with the endonuclear symbiotic bacterium Holospora obtusa;  citation_journal_title=FEMS Microbiol. Lett.;  citation_year=2005;  citation_volume=243;  citation_pages=101-5; " /><meta name="citation_reference" content="citation_title=Micronucleus-specific bacterium Holospora elegans irreversibly enhances stress gene expression of the host Paramecium caudatum;  citation_journal_title=J. Euk. Microbiol.;  citation_year=2008;  citation_volume=55;  citation_pages=515-21; " /><meta name="citation_reference" content="citation_title=Antigenic variation in ciliates: antigen structure, function, expression;  citation_journal_title=J. Eukaryot. Microbiol.;  citation_year=2007;  citation_volume=54;  citation_pages=1-7; " /><meta name="citation_reference" content="citation_title=Structure and sequence of the H surface protein gene of Paramecium and comparison with related genes;  citation_journal_title=Mol. Gen. Genet.;  citation_year=1987;  citation_volume=208;  citation_pages=529-36; " /><meta name="citation_reference" content="citation_title=mRNA stability plays a major role in regulating the temperature-specific expression of a Tetrahymena thermophila surface protein;  citation_journal_title=Mol. Cell. Biol.;  citation_year=1988;  citation_volume=8;  citation_pages=427-32; " /><meta name="citation_reference" content="citation_title=Evidence for the requirement of protein synthesis and protein kinase activity in the temperature regulated stability of a Tetrahymena surface protein mRNA;  citation_journal_title=Nucleic Acids Res.;  citation_year=1995;  citation_volume=23;  citation_pages=942-8; " /><meta name="citation_reference" content="citation_title=Heterochromatin-mediated control of virulence gene expression;  citation_journal_title=Mol. Microbiol.;  citation_year=2006;  citation_volume=62;  citation_pages=612-20; " /><meta name="citation_reference" content="citation_title=Analysis of sequence variability in the macronuclear DNA of Paramecium tetraurelia: a somatic view of the germline;  citation_journal_title=Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=585-96; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/4/293/998318/dsv014.pdf" /><meta name="description" content="Abstract.  Phenotypic variation of a single genotype is achieved by alterations in gene expression patterns. Regulation of such alterations depends on their tim" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/4/293/348350" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/4/293/348350" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/4/293/348350");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv014';
    </script>
<title>
	Epigenetic regulation of serotype expression antagonizes transcriptome dynamics in Paramecium tetraurelia | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./348350" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BY1lPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDaGVhaWIsIE1pcmlhbSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik1vbGVjdWxhciBDZWxsIER5bmFtaWNzLCBDZW50cmUgZm9yIEh1bWFuIGFuZCBNb2xlY3VsYXIgQmlvbG9neSwgU2FhcmxhbmQgVW5pdmVyc2l0eSwgU2FhcmJyw7xja2VuIDY2MTIzLCBHZXJtYW55IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRGVoZ2hhbmkgQW1pcmFiYWQsIEF6aW0iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNb2xlY3VsYXIgQ2VsbCBEeW5hbWljcywgQ2VudHJlIGZvciBIdW1hbiBhbmQgTW9sZWN1bGFyIEJpb2xvZ3ksIFNhYXJsYW5kIFVuaXZlcnNpdHksIFNhYXJicsO8Y2tlbiA2NjEyMywgR2VybWFueSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNsdXN0ZXIgb2YgRXhjZWxsZW5jZSBmb3IgTXVsdGltb2RhbCBDb21wdXRpbmcgYW5kIEludGVyYWN0aW9uLCBTYWFybGFuZCBVbml2ZXJzaXR5IGFuZCBNYXggUGxhbmNrIEluc3RpdHV0ZSBmb3IgSW5mb3JtYXRpY3MsIFNhYXJicsO8Y2tlbiA2NjEyMywgR2VybWFueSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5vcmRzdHLDtm0sIEthcmwgSi4gVi4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJFcGlnZW5ldGljcyBEZXBhcnRtZW50LCBDZW50cmUgZm9yIEh1bWFuIGFuZCBNb2xlY3VsYXIgQmlvbG9neSwgU2FhcmxhbmQgVW5pdmVyc2l0eSwgU2FhcmJyw7xja2VuIDY2MTIzLCBHZXJtYW55IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2NodWx6LCBNYXJjZWwgSC4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDbHVzdGVyIG9mIEV4Y2VsbGVuY2UgZm9yIE11bHRpbW9kYWwgQ29tcHV0aW5nIGFuZCBJbnRlcmFjdGlvbiwgU2FhcmxhbmQgVW5pdmVyc2l0eSBhbmQgTWF4IFBsYW5jayBJbnN0aXR1dGUgZm9yIEluZm9ybWF0aWNzLCBTYWFyYnLDvGNrZW4gNjYxMjMsIEdlcm1hbnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaW1vbiwgTWFydGluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTW9sZWN1bGFyIENlbGwgRHluYW1pY3MsIENlbnRyZSBmb3IgSHVtYW4gYW5kIE1vbGVjdWxhciBCaW9sb2d5LCBTYWFybGFuZCBVbml2ZXJzaXR5LCBTYWFyYnLDvGNrZW4gNjYxMjMsIEdlcm1hbnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkVwaWdlbmV0aWMgcmVndWxhdGlvbiBvZiBzZXJvdHlwZSBleHByZXNzaW9uIGFudGFnb25pemVzIHRyYW5zY3JpcHRvbWUgZHluYW1pY3MgaW4gUGFyYW1lY2l1bSB0ZXRyYXVyZWxpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjI5MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMzA1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN2MDE0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTUvMDgvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaGVub3R5cGljIHBsYXN0aWNpdHk6IG1vbGVjdWxhciBtZWNoYW5pc21zIGFuZCBhZGFwdGl2ZSBzaWduaWZpY2FuY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNvbXByLiBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9MTQxNy0zOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXBpZ2VuZXRpYyByZWd1bGF0aW9uIG9mIHN0ZW0gY2VsbHM6IHRoZSByb2xlIG9mIGNocm9tYXRpbiBpbiBjZWxsIGRpZmZlcmVudGlhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QWR2LiBFeHAuIE1lZC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9Nzg2OyAgY2l0YXRpb25fcGFnZXM9MzA3LTI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm90ZWluIGZvbGRpbmcgaW4gdGhlIGN5dG9wbGFzbSBhbmQgdGhlIGhlYXQgc2hvY2sgcmVzcG9uc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNvbGQgU3ByaW5nIEhhcmIuIFBlcnNwZWN0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9YTAwNDM5MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoZW5vdHlwaWMgcGxhc3RpY2l0eSBpbiBkZXZlbG9wbWVudCBhbmQgZXZvbHV0aW9uOiBmYWN0cyBhbmQgY29uY2VwdHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBoaWxvcy4gVC4gUm95LiBTb2MuIEIuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM2NTsgIGNpdGF0aW9uX3BhZ2VzPTU0Ny01NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGlwaWRzIGFuZCBsaXBpZCBtZXRhYm9saXNtIGluIGV1a2FyeW90aWMgYWxnYWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2cuIExpcGlkIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz0xNjAtODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5ldyBpbnNpZ2h0cyBpbnRvIHRoZSBwaHlsb2dlbnkgb2YgZXVrYXJ5b3RlcyBiYXNlZCBvbiBjaWxpYXRlIEhzcDcwIHNlcXVlbmNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTk0My01NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29udmVyZ2VudCBldm9sdXRpb24gb2YgaGVhdC1pbmR1Y2liaWxpdHkgZHVyaW5nIHN1YmZ1bmN0aW9uYWxpemF0aW9uIG9mIHRoZSBIc3A3MCBnZW5lIGZhbWlseTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEV2b2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9NDkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EeW5hbWljIGNocm9tYXRpbiByZW1vZGVsbGluZyBvZiBjaWxpYXRlIG1hY3JvbnVjbGVhciBETkEgYXMgZGV0ZXJtaW5lZCBieSBhbiBvcHRpbWl6ZWQgY2hyb21hdGluIGltbXVub3ByZWNpcGl0YXRpb24gKENoSVApIG1ldGhvZCBmb3IgUGFyYW1lY2l1bSB0ZXRyYXVyZWxpYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXBwbC4gTWljcm9iaW9sLiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT05NzsgIGNpdGF0aW9uX3BhZ2VzPTI2NjEtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS1kZWZlbmNlIHNtYWxsIFJOQXMgZXhhcHRlZCBmb3IgZXBpZ2VuZXRpYyBtYXRpbmctdHlwZSBpbmhlcml0YW5jZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTUwOTsgIGNpdGF0aW9uX3BhZ2VzPTQ0Ny01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGFudGlnZW4gc3lzdGVtIG9mIFBhcmFtZWNpdW0gYXVyZWxpYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW50LiBSZXYuIEN5dG9sLjsgIGNpdGF0aW9uX3llYXI9MTk1NzsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MS0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzdGluY3QgUk5BZGVwZW5kZW50IFJOQSBwb2x5bWVyYXNlcyBhcmUgcmVxdWlyZWQgZm9yIFJOQWkgdHJpZ2dlcmVkIGJ5IGRvdWJsZXN0cmFuZGVkIFJOQSB2ZXJzdXMgdHJ1bmNhdGVkIHRyYW5zZ2VuZXMgaW4gUGFyYW1lY2l1bSB0ZXRyYXVyZWxpYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9NDA5Mi0xMDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgY2hhcmFjdGVyaXphdGlvbiBvZiB2YXJpYWJsZSBzdXJmYWNlIGFudGlnZW5zIHJldmVhbHMgYSB0ZWxvbWVyZSBwb3NpdGlvbiBlZmZlY3QgYXMgYSBwcmVyZXF1aXNpdGUgZm9yIFJOQSBpbnRlcmZlcmVuY2UtbWVkaWF0ZWQgc2lsZW5jaW5nIGluIFBhcmFtZWNpdW0gdGV0cmF1cmVsaWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPW1CaW8uOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lMDEzMjgtMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVuaWNlbGx1bGFyIGV1a2FyeW90ZXMgYXMgbW9kZWxzIGluIGNlbGwgYW5kIG1vbGVjdWxhciBiaW9sb2d5OiBjcml0aWNhbCBhcHByYWlzYWwgb2YgdGhlaXIgcGFzdCBhbmQgZnV0dXJlIHZhbHVlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbnQuIFJldi4gQ2VsbCBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTMwOTsgIGNpdGF0aW9uX3BhZ2VzPTE0MS05ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUNhbWJyaWRnZSBVbml2ZXJzaXR5IFByZXNzLCBOZXcgWW9yazsgIGNpdGF0aW9uX3RpdGxlPVRoZSBnZW5ldGljcyBvZiBQYXJhbWVjaXVtIEF1cmVsaWE7ICBjaXRhdGlvbl95ZWFyPTE5NTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUN1dGFkYXB0IHJlbW92ZXMgYWRhcHRlciBzZXF1ZW5jZXMgZnJvbSBoaWdoLXRocm91Z2hwdXQgc2VxdWVuY2luZyByZWFkczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RU1CbmV0IEouOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MTAtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFBmYW0gcHJvdGVpbiBmYW1pbGllcyBkYXRhYmFzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9RDI5MC0zMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1FR0E2OiBtb2xlY3VsYXIgZXZvbHV0aW9uYXJ5IGdlbmV0aWNzIGFuYWx5c2lzIHZlcnNpb24gNi4wLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTI3MjUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2FpbGZpc2ggZW5hYmxlcyBhbGlnbm1lbnQtZnJlZSBpc29mb3JtIHF1YW50aWZpY2F0aW9uIGZyb20gUk5BLXNlcSByZWFkcyB1c2luZyBsaWdodHdlaWdodCBhbGdvcml0aG1zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9NDYyLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vZGVyYXRlZCBlc3RpbWF0aW9uIG9mIGZvbGQgY2hhbmdlIGFuZCBkaXNwZXJzaW9uIGZvciBSTkEtc2VxIGRhdGEgd2l0aCBERVNlcTI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTU1MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnRyb2xsaW5nIHRoZSBmYWxzZSBkaXNjb3ZlcnkgcmF0ZTogYSBwcmFjdGljYWwgYW5kIHBvd2VyZnVsIGFwcHJvYWNoIHRvIG11bHRpcGxlIHRlc3Rpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIFIuIFN0YXQuIFNvYy4gU2VyLiBCLiAoTWV0aG9kb2xvZ2ljYWwpOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTU3OyAgY2l0YXRpb25fcGFnZXM9Mjg5LTMwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwcmVzc2lvbiBjYXJ0b2dyYXBoeSBvZiBodW1hbiB0aXNzdWVzIHVzaW5nIHNlbGYgb3JnYW5pemluZyBtYXBzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0zMDYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QYXJhbWVjaXVtREIgaW4gMjAxMTogbmV3IHRvb2xzIGFuZCBuZXcgZGF0YSBmb3IgZnVuY3Rpb25hbCBhbmQgY29tcGFyYXRpdmUgZ2Vub21pY3Mgb2YgdGhlIG1vZGVsIGNpbGlhdGUgUGFyYW1lY2l1bSB0ZXRyYXVyZWxpYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9RDYzMi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbnRvbG9naXplciAyLjDigJNhIG11bHRpZnVuY3Rpb25hbCB0b29sIGZvciBHTyB0ZXJtIGVucmljaG1lbnQgYW5hbHlzaXMgYW5kIGRhdGEgZXhwbG9yYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MTY1MC0xOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HbG9iYWwgdHJlbmRzIG9mIHdob2xlIGdlbm9tZSBkdXBsaWNhdGlvbnMgcmV2ZWFsZWQgYnkgdGhlIGNpbGlhdGUgUGFyYW1lY2l1bSB0ZXRyYXVyZWxpYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTQ0NDsgIGNpdGF0aW9uX3BhZ2VzPTE3MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SRVZJR08gc3VtbWFyaXplcyBhbmQgdmlzdWFsaXplcyBsb25nIGxpc3RzIG9mIEdlbmUgT250b2xvZ3kgdGVybXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1lMjE4MDAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY29udHJvbCBvZiBoaXN0b25lIGdlbmUgZXhwcmVzc2lvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvY2hlbS4gU29jLiBUcmFucy47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz04ODAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TnVjbGVvc29tZSBsb3NzIGxlYWRzIHRvIGdsb2JhbCB0cmFuc2NyaXB0aW9uYWwgdXAtcmVndWxhdGlvbiBhbmQgZ2Vub21pYyBpbnN0YWJpbGl0eSBkdXJpbmcgeWVhc3QgYWdpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmVzIERldi47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0zOTYtNDA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaXN0b25lIHN1cHBseSByZWd1bGF0ZXMgUyBwaGFzZSB0aW1pbmcgYW5kIGNlbGwgY3ljbGUgcHJvZ3Jlc3Npb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPWVMaWZlOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz1lMDI0NDMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeXN0ZW1zLXdpZGUgYW5hbHlzaXMgb2YgYWNjbGltYXRpb24gcmVzcG9uc2VzIHRvIGxvbmctdGVybSBoZWF0IHN0cmVzcyBhbmQgcmVjb3ZlcnkgaW4gdGhlIHBob3Rvc3ludGhldGljIG1vZGVsIG9yZ2FuaXNtIENobGFteWRvbW9uYXMgcmVpbmhhcmR0aWk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz00MjcwLTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWZmZXJlbnRpYWwgZXhwcmVzc2lvbiBvZiBoaXN0b25lIEgzIGdlbmVzIGFuZCBzZWxlY3RpdmUgYXNzb2NpYXRpb24gb2YgdGhlIHZhcmlhbnQgSDMuNyB3aXRoIGEgc3BlY2lmaWMgc2VxdWVuY2UgY2xhc3MgaW4gU3R5bG9ueWNoaWEgbWFjcm9udWNsZWFyIGRldmVsb3BtZW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FcGlnZW5ldGljcyBDaHJvbWF0aW47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTEtMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmlzb24gb2YgZ2VuZSBleHByZXNzaW9uIG9mIFBhcmFtZWNpdW0gYnVyc2FyaWEgd2l0aCBhbmQgd2l0aG91dCBDaGxvcmVsbGEgdmFyaWFiaWxpcyBzeW1iaW9udHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljcy47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xODMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZW5kb3N5bWJpb3RpYyBiYWN0ZXJpdW0gSG9sb3Nwb3JhIG9idHVzYSBlbmhhbmNlcyBoZWF0LXNob2NrIGdlbmUgZXhwcmVzc2lvbiBvZiB0aGUgaG9zdCBQYXJhbWVjaXVtIGNhdWRhdHVtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBFdWsuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NTA7ICBjaXRhdGlvbl9wYWdlcz0yOTMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGFyYW1lY2l1bSBjYXVkYXR1bSBhY3F1aXJlcyBoZWF0LXNob2NrIHJlc2lzaXRhbmNlIGluIGNpbGlhcmUgbW92ZW1lbnQgYnkgaW5mZWN0aW9uIHdpdGggdGhlIGVuZG9udWNsZWFyIHN5bWJpb3RpYyBiYWN0ZXJpdW0gSG9sb3Nwb3JhIG9idHVzYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RkVNUyBNaWNyb2Jpb2wuIExldHQuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTI0MzsgIGNpdGF0aW9uX3BhZ2VzPTEwMS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb251Y2xldXMtc3BlY2lmaWMgYmFjdGVyaXVtIEhvbG9zcG9yYSBlbGVnYW5zIGlycmV2ZXJzaWJseSBlbmhhbmNlcyBzdHJlc3MgZ2VuZSBleHByZXNzaW9uIG9mIHRoZSBob3N0IFBhcmFtZWNpdW0gY2F1ZGF0dW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEV1ay4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT01NTsgIGNpdGF0aW9uX3BhZ2VzPTUxNS0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW50aWdlbmljIHZhcmlhdGlvbiBpbiBjaWxpYXRlczogYW50aWdlbiBzdHJ1Y3R1cmUsIGZ1bmN0aW9uLCBleHByZXNzaW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBFdWthcnlvdC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT01NDsgIGNpdGF0aW9uX3BhZ2VzPTEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJlIGFuZCBzZXF1ZW5jZSBvZiB0aGUgSCBzdXJmYWNlIHByb3RlaW4gZ2VuZSBvZiBQYXJhbWVjaXVtIGFuZCBjb21wYXJpc29uIHdpdGggcmVsYXRlZCBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBHZW4uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDg7ICBjaXRhdGlvbl9wYWdlcz01MjktMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPW1STkEgc3RhYmlsaXR5IHBsYXlzIGEgbWFqb3Igcm9sZSBpbiByZWd1bGF0aW5nIHRoZSB0ZW1wZXJhdHVyZS1zcGVjaWZpYyBleHByZXNzaW9uIG9mIGEgVGV0cmFoeW1lbmEgdGhlcm1vcGhpbGEgc3VyZmFjZSBwcm90ZWluOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIENlbGwuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz00MjctMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2aWRlbmNlIGZvciB0aGUgcmVxdWlyZW1lbnQgb2YgcHJvdGVpbiBzeW50aGVzaXMgYW5kIHByb3RlaW4ga2luYXNlIGFjdGl2aXR5IGluIHRoZSB0ZW1wZXJhdHVyZSByZWd1bGF0ZWQgc3RhYmlsaXR5IG9mIGEgVGV0cmFoeW1lbmEgc3VyZmFjZSBwcm90ZWluIG1STkE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTk0Mi04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IZXRlcm9jaHJvbWF0aW4tbWVkaWF0ZWQgY29udHJvbCBvZiB2aXJ1bGVuY2UgZ2VuZSBleHByZXNzaW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NjI7ICBjaXRhdGlvbl9wYWdlcz02MTItMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIHNlcXVlbmNlIHZhcmlhYmlsaXR5IGluIHRoZSBtYWNyb251Y2xlYXIgRE5BIG9mIFBhcmFtZWNpdW0gdGV0cmF1cmVsaWE6IGEgc29tYXRpYyB2aWV3IG9mIHRoZSBnZXJtbGluZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz01ODUtOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMi80LzI5My85OTgzMTgvZHN2MDE0LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBQaGVub3R5cGljIHZhcmlhdGlvbiBvZiBhIHNpbmdsZSBnZW5vdHlwZSBpcyBhY2hpZXZlZCBieSBhbHRlcmF0aW9ucyBpbiBnZW5lIGV4cHJlc3Npb24gcGF0dGVybnMuIFJlZ3VsYXRpb24gb2Ygc3VjaCBhbHRlcmF0aW9ucyBkZXBlbmRzIG9uIHRoZWlyIHRpbSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzQ4MzUwJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZLt3bx76ELqfSx+NBeLcH3Q6nfsg" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f4%2f293%2f348350" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f4%2f293%2f348350" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13443" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/4/1/m_cover.gif?Expires=2147483647&amp;Signature=Ufin3LLH~Iqb2kw-hetKqzdx2YgWiSwuSGu7FcRPyZ0YMXPxK67~EvZK~D~FaBMUMtkfx2C3nicLRXrG7LQSzXkxs1Qyu1rF6Hbn-vB-awwpaXOgx90CafYjmsJlX0TeG5UBY2DnHbv1s5YegTLh88o5c9kHiz4IICTc2SgUfnIM~hO2aQPqVNMfxHD9dsgu8Ly4qu3h4vkVcBZFQVgjv-cFeqoqOFVvCZqW6Vq3z-Hi91~oLc58RRI-PjbrcLYKmszwtDPbL2sZKRQky2J2HWAa2PZzNl-CMH92uFmvRLgkUeyANQZ-s6eHxAaArS5e4XiVxAKXHDRMAVOPl6~asA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/4">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4944566">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4944566">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4944568">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4944568">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4944576">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4944576">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4944595">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4944595">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4944728">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4944728">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4944730">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4944730">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4944732">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4944732">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4944734">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4944734">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/4/279/348084">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Epigenetic regulation of serotype expression antagonizes transcriptome dynamics in <em>Paramecium tetraurelia</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Miriam Cheaib</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Molecular Cell Dynamics, Centre for Human and Molecular Biology, Saarland University</div>, <div class="addr-line">Saarbrücken 66123</div>, <div class="country">Germany</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Miriam+Cheaib">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cheaib M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cheaib M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Miriam Cheaib</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Azim Dehghani Amirabad</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Molecular Cell Dynamics, Centre for Human and Molecular Biology, Saarland University</div>, <div class="addr-line">Saarbrücken 66123</div>, <div class="country">Germany</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Cluster of Excellence for Multimodal Computing and Interaction, Saarland University and Max Planck Institute for Informatics</div>, <div class="addr-line">Saarbrücken 66123</div>, <div class="country">Germany</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Azim+Dehghani+Amirabad">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dehghani Amirabad A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Dehghani Amirabad A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Azim Dehghani Amirabad</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Karl J. V. Nordstr&#246;m</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Epigenetics Department</div>, <div class="insititution">Centre for Human and Molecular Biology, Saarland University</div>, <div class="addr-line">Saarbrücken 66123</div>, <div class="country">Germany</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Karl+J.+V.+Nordstr%c3%b6m">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nordstr&#246;m K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nordstr&#246;m K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Karl J. V. Nordström</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Marcel H. Schulz</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Cluster of Excellence for Multimodal Computing and Interaction, Saarland University and Max Planck Institute for Informatics</div>, <div class="addr-line">Saarbrücken 66123</div>, <div class="country">Germany</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Marcel+H.+Schulz">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Schulz M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Schulz M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Marcel H. Schulz</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Martin Simon</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Molecular Cell Dynamics, Centre for Human and Molecular Biology, Saarland University</div>, <div class="addr-line">Saarbrücken 66123</div>, <div class="country">Germany</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +49 681-302-4938. Fax. +49 681-302-2703. E-mail: <a href="mailto:martin.simon@uni-saarland.de" target="_blank">martin.simon@uni-saarland.de</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Martin+Simon">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Simon M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Simon M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Martin Simon</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="TN1"><div class="fn"><div class="footnote-content">Edited by Dr Minoru Yoshida</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 4, 1 August 2015, Pages 293–305, <a href='https://doi.org/10.1093/dnares/dsv014'>https://doi.org/10.1093/dnares/dsv014</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">31 July 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">06 April 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">28 June 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="348350" href="/dnaresearch/article-pdf/22/4/293/998318/dsv014.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Miriam Cheaib, Azim Dehghani Amirabad, Karl J. V. Nordström, Marcel H. Schulz, Martin Simon;  Epigenetic regulation of serotype expression antagonizes transcriptome dynamics in <em>Paramecium tetraurelia</em>, <em>DNA Research</em>, Volume 22, Issue 4, 1 August 2015, Pages 293–305, <a href='https://doi.org/10.1093/dnares/dsv014'>https://doi.org/10.1093/dnares/dsv014</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/348350?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/348350?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/348350?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/348350?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/348350?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/348350?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/348350?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Epigenetic%20regulation%20of%20serotype%20expression%20antagonizes%20transcriptome%20dynamics%20in%20%26lt%3Bem%26gt%3BParamecium%20tetraurelia%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2015-07-31&amp;volumeNum=22&amp;issueNum=4&amp;author=Cheaib%2C%20Miriam%3B%20Dehghani%20Amirabad%2C%20Azim&amp;startPage=293&amp;endPage=305&amp;contentId=10.1093%2Fdnares%2Fdsv014&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f4%2f293%2f348350" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4944566 id="4944566" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Phenotypic variation of a single genotype is achieved by alterations in gene expression patterns. Regulation of such alterations depends on their time scale, where short-time adaptations differ from permanently established gene expression patterns maintained by epigenetic mechanisms. In the ciliate <em>Paramecium</em>, serotypes were described for an epigenetically controlled gene expression pattern of an individual multigene family. Paradoxically, individual serotypes can be triggered in <em>Paramecium</em> by alternating environments but are then stabilized by epigenetic mechanisms, thus raising the question to which extend their expression follows environmental stimuli. To characterize environmental adaptation in the context of epigenetically controlled serotype expression, we used RNA-seq to characterize transcriptomes of serotype pure cultures. The resulting vegetative transcriptome resource is first analysed for genes involved in the adaptive response to the altered environment. Secondly, we identified groups of genes that do not follow the adaptive response but show co-regulation with the epigenetically controlled serotype system, suggesting that their gene expression pattern becomes manifested by similar mechanisms. In our experimental set-up, serotype expression and the entire group of co-regulated genes were stable among environmental changes and only heat-shock genes altered expression of these gene groups. The data suggest that the maintenance of these gene expression patterns in a lineage represents epigenetically controlled robustness counteracting short-time adaptation processes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;antigenic variation&quot;">antigenic variation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="epigenetics">epigenetics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="ciliate">ciliate</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;telomere position effect&quot;">telomere position effect</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="heat-shock">heat-shock</a></div>                            <h2 scrollto-destination=4944568 id="4944568" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Alterations in gene expression patterns are the regulatory basis for phenotypic plasticity: one single genome may produce quite different phenotypes, thus representing an important mechanism to adapt for alternating environmental conditions including morphological, physiological and behavioural changes.<sup><a reveal-id="DSV014C1" data-open="DSV014C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Similarly, differentiation processes in developmental stages of multicellular organisms allow one single genotype to differentiate into distinct tissues, representing the regulation of gene expression patterns in the meaning of phenotypic plasticity: epigenetic mechanisms have the aim to fix these gene expression patterns rather than to allow subsequent flexibility.<sup><a reveal-id="DSV014C2" data-open="DSV014C2" class="link link-ref link-reveal xref-bibr">2</a></sup></p><p>Among the diverse mechanisms regulating alterations in gene expression during phenotypic variation, one can distinguish mechanisms that evolved as a fast reaction to stresses, e.g. the heat-shock response, and adaptations to long-term changes in environments, e.g. such as temperature and food availability. The molecular regulation of the heat-shock proteins (HSPs) represents a tightly controlled transcriptional activation of molecular chaperones that assist proper folding of proteins.<sup><a reveal-id="DSV014C3" data-open="DSV014C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Usually, this transcriptional activation of HSPs occurs in minutes and only for a few minutes, thus being a rapid, programmed and reversible response to stress.</p><p>Although it is clear that short-time stress response and long-term phenotypic plasticity are key players for survival during short-time and long-term environmental changes, a general understanding of the complex genetic and epigenetic mechanisms controlling phenotypic plasticity as a response to environmental changes is still missing.<sup><a reveal-id="DSV014C4" data-open="DSV014C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Only few studies addressed the extent of phenotypic plasticity based on genome-wide transcriptomic profiling in response to altered environments.</p><p>Adaptation to different environmental temperatures represents one of the most important stress responses of organisms. As ectothermic species may use any environmental heat source to regulate their body temperature thus optimizing their metabolism, some organisms are poikilothermic, meaning that their body temperature is equal to the ambient environmental temperature which challenges the cellular metabolism to work properly in a huge range of temperatures. Depending on the level of temperature variation, adaptation processes have to control survival critical functions like efficient metabolic turnover. In the past, much attention was spend on the composition of membrane lipids during cold adaptation and heat-shocks,<sup><a reveal-id="DSV014C5" data-open="DSV014C5" class="link link-ref link-reveal xref-bibr">5</a></sup> but only few studies investigated transcriptomic changes to identify general metabolic pathways altered during temperature adaptation processes.</p><p>Paramecia are unicellular model organisms belonging to the ciliate clade. These are distributed all over the world in all climate areas. However, a single population may have to cope with drastic temperature alterations: on the one hand, these cells are so small that they cannot prevent poikilothermy by increasing body mass/volume ratio, and on the other hand, their natural ecosystem of small ponds undergoes drastic short-time temperature alterations. It is not surprising that paramecia in laboratory cultures can easily be cultivated at a broad temperature range from 4°C to 32°C. Therefore, <em>Paramecium</em> represents an excellent model to investigate mechanisms of temperature adaptation, and this ciliate has been subject of several studies investigating heat-shock response of individual genes,<sup><a reveal-id="DSV014C6  DSV014C8" data-open="DSV014C6  DSV014C8" class="link link-ref link-reveal xref-bibr">6–8</a></sup> but a genome-wide analysis is still missing.</p><p>This unicellular model organism has become even more attractive for the characterization of phenotypic plasticity as several epigenetic mechanisms have been described that allow differentiation of lineages into different cell states, which are reminiscent of developmental differentiation processes in multicellular organisms. One prominent example concerns the expression of the mating-type system,<sup><a reveal-id="DSV014C9" data-open="DSV014C9" class="link link-ref link-reveal xref-bibr">9</a></sup> and similarly, but not by the same mechanisms, the antigenic system was described to be epigenetically controlled a long time ago.<sup><a reveal-id="DSV014C10" data-open="DSV014C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Recent studies indicate that the antigenic system is regulated by complex mechanisms involving small RNAs and dynamic chromatin alterations<sup><a reveal-id="DSV014C11" data-open="DSV014C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSV014C12" data-open="DSV014C12" class="link link-ref link-reveal xref-bibr">12</a></sup> (see Cheaib et al., in preparation): <em>Paramecium</em> displays mutually exclusive expression of variable surface antigen genes similar to the phenomenon of antigenic variation in pathogens.<sup><a reveal-id="DSV014C13" data-open="DSV014C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Serotypes, meaning exclusive expression of one antigen gene of the surface antigen gene (SAg) multigene family, are initially triggered by different environmental conditions, e.g. temperature and food availability.<sup><a reveal-id="DSV014C14" data-open="DSV014C14" class="link link-ref link-reveal xref-bibr">14</a></sup> They are stabilized by mechanisms similar to epigenetic differentiation processes, which are under control of an RNAi mechanism likely involved in transcriptional silencing.<sup><a reveal-id="DSV014C11" data-open="DSV014C11" class="link link-ref link-reveal xref-bibr">11</a></sup> It was shown that a telomere position effect (TPE) is required for SAg regulation, suggesting that spreading of telomeric heterochromatin is involved in RNAi-mediated silencing.<sup><a reveal-id="DSV014C12" data-open="DSV014C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Differentiation into distinct serotypes therefore represents an epigenetically controlled manifestation of gene expression patterns, which interestingly can also be passed on to sexual progeny. In contrast to mating-type determination, serotype switches can occur anytime without the need for sexual recombination. Hence, they represent an important example of epigenetically controlled and heritable transcriptomic variations of a single genotype.<sup><a reveal-id="DSV014C13" data-open="DSV014C13" class="link link-ref link-reveal xref-bibr">13</a></sup></p><p>To understand the processes of long-term and shocked temperature adaptations in the context of serotype expression, we analysed the transcriptome information of serotype pure cultures to see to which extent the transcriptome becomes altered during temperature adaptation and how this is related to the epigenetically controlled SAg multigene family. We furthermore analysed the subtelomeres of SAg containing chromosomes to characterize a possible heterochromatic spreading from telomeric repeats by transcriptomic analysis.</p>                            <h2 scrollto-destination=4944576 id="4944576" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=4944577 id="4944577" class="section-title" data-legacy-id=s2a>2.1. <em>Paramecium</em> culture conditions, serotype analysis and heat-shocks</h3>
<p><em>Paramecium tetraurelia</em> strain 51 and d4-2 were grown in wheat grass powder (WGP, Pines International Co., Lawrence, KS, USA) infusion medium bacterized with <em>Klebsiella pneumoniae</em> supplemented with 0.8 µg/ml β-sitosterol, unless otherwise stated (serotype 51A, 51H of strain 51) and 1 : 1 diluted with Volvic<sup>®</sup> mineral water (serotype 51D, 51B of strain d4-2). To stabilize/trigger respective serotypes, different cultivation temperatures were chosen: 51A at 31°C, 51B at 24°C/6°C, 51D at 24°C and 51H at 14°C. Serotype expression was verified by immobilization with specific antibodies (1 : 100) (rabbit α-51A/-51B/-51D/-51H) in depression slides. Only serotype-pure cultures (refers to 100% immobilization) were used for further analysis. For heat-shock experiments, serotype-51D cultures were harvested, washed in Volvic<sup>®</sup> mineral water and exposed to 39°C for 20 min.<sup><a reveal-id="DSV014C8" data-open="DSV014C8" class="link link-ref link-reveal xref-bibr">8</a></sup> 51B cells were selected out of a 51D culture by immobilization with α-51D serum (1 : 100). Individual survived cells were transferred to fresh WGP-medium bacterized with <em>K. pneumoniae</em> after washing in Volvic<sup>®</sup> mineral water. One hundred per cent serotype 51B expressing cultures were cultivated at 24°C. They were then split, and the counterpart was cultivated at 14°C for 1 day before transfer to 6°C.</p>                            <h3 scrollto-destination=4944579 id="4944579" class="section-title" data-legacy-id=s2b>2.2. RNA isolation, cDNA library creation, Illumina sequencing</h3>
<p>A total of 150,000 <em>P. tetraurelia</em> cells per sample were harvested, washed in Volvic<sup>®</sup> mineral water and incubated for 20 min at the respective temperature in non-nutrient medium to reduce bacterial occurrence in food vacuoles. Total RNA was isolated using TriReagent<sup>®</sup> (Sigma-Aldrich, Seelze, Germany). After DNAse I (Invitrogen, Karlsruhe, Germany) digestion and subsequent purification with acid phenol, RNA integrity was verified on an Agilent Bioanalyzer 2100. Poly-A enrichment (NEBNext<sup>®</sup> Poly(A) mRNA Magnetic Isolation Module) out of 1 µg total RNA and library preparation (NEBNext<sup>®</sup> Ultra™ Directional RNA Library Prep Kit for Illumina) were carried out according to manufacturer's recommendation using 11 PCR cycles of library enrichment. Multiplexed libraries were 100 nt paired-end sequenced with an average of 14 Mio reads per sample on Illumina HiSeq 2500 platform. Reads were demultiplexed with bcl2fastq (v1.8.4) and trimmed for adaptor contamination and low-quality bases with the cutadapt (v1.4.1)<sup><a reveal-id="DSV014C15" data-open="DSV014C15" class="link link-ref link-reveal xref-bibr">15</a></sup> wrapper trim_galore (v0.3.3). All raw data were deposited at the European Nucleotide Archive (ENA, <a class="link link-uri" href="http://www.ebi.ac.uk/ena">http://www.ebi.ac.uk/ena</a>) under study accession no. PRJEB9464.</p>                            <h3 scrollto-destination=4944581 id="4944581" class="section-title" data-legacy-id=s2c>2.3. Annotation of HSP70 isoforms and phylogenetic reconstruction</h3>
<p>HSP70 candidates were identified by a proteome-wide search against the Pfam library of hidden Markov models.<sup><a reveal-id="DSV014C16" data-open="DSV014C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Hits were extracted for PF00012 (HSP70) using an E-value threshold of 3.6E−23. Amino acid sequences were aligned with ClustalW, and neighbour-joining trees were calculated with MEGA6<sup><a reveal-id="DSV014C17" data-open="DSV014C17" class="link link-ref link-reveal xref-bibr">17</a></sup> with 1,000 bootstrap replicates, based on the Poisson model and after complete deletion of gaps.</p>                            <h3 scrollto-destination=4944583 id="4944583" class="section-title" data-legacy-id=s2d>2.4. RNA-seq expression analysis</h3>
<p>We quantified gene expression levels using the Sailfish algorithm.<sup><a reveal-id="DSV014C18" data-open="DSV014C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Briefly, Sailfish uses transcript sequences to build a kmer index over all annotated transcript sequences. RNA-seq reads are decomposed into kmers and projected onto kmers of transcripts; then expression levels of all transcripts are deconvoluted. Finally, transcript expression levels of the same gene are summarized to give a gene expression estimate. As a normalized expression metric for plots, we used Reads per Kilobase per Million mapped reads (RPKM) reported by Saillfish. We used Sailfish version 0.6.2 with the following command lines:</p><p>sailfish index -t &lt;ref_transcripts&gt; -o &lt;out_dir&gt; -k 20 for building transcriptome index file and</p><p>sailfish quant -i indexfile -l ‘T = PE : O =&gt;&lt;:S = AS’ -1 sample_R1.fastq -2 R2.fastq -o Sample1 for strand-specific quantification of read files. We downloaded the <em>P. tetraurelia</em> cDNA file from the Ensembl protists database release 26 (assembly version GCA_000165425). The quantification was done for 39,635 genes.</p><p>After gene expression quantification, differential gene expression analysis was conducted using DESeq2 version 3<sup><a reveal-id="DSV014C19" data-open="DSV014C19" class="link link-ref link-reveal xref-bibr">19</a></sup> in R. Following the DESeq2 guidelines, absolute read counts per gene as obtained from Sailfish were normalized between samples to account for differences in sequencing depth. Linear models to assess DE status were fit for each gene with DESeq2 using a negative binomial distribution. After multiple testing correction,<sup><a reveal-id="DSV014C20" data-open="DSV014C20" class="link link-ref link-reveal xref-bibr">20</a></sup> genes with false discovery rate (FDR) &lt;0.01 were considered as differentially expressed.</p>                            <h3 scrollto-destination=4944588 id="4944588" class="section-title" data-legacy-id=s2e>2.5. Transcriptome expression landscape visualization and analysis</h3>
<p>To visualize and compare the transcriptome expression landscape for the different conditions, we applied a self-organizing map (SOM) approach using the oposSOM package.<sup><a reveal-id="DSV014C21" data-open="DSV014C21" class="link link-ref link-reveal xref-bibr">21</a></sup> An SOM is a machine learning technique for dimensionality reduction, which transfers a high-dimensional data matrix (expression of all genes across different samples) to a lower dimension.</p><p>The oposSOM package decomposes the data into clusters of correlated sets of genes, called metagenes, which are arranged in a fix order (here we use a lattice with dimension 20 by 20). The expression of a metagene represents the average expression of all genes in the cluster. Metagene expression values in the individual samples provide mosaic pictures visualizing condition specific over- and under-expression in terms of characteristic colour-coded textures, which allows the direct comparison of the expression of individual samples in a simple and intuitive way.</p><p>TPM scale expression levels were log-transformed as pre-processing step for training the SOM. After training, we had sample-specific transcriptome expression landscapes as well over- and under-expressed hot spots for every sample. Sample-specific transcriptome landscapes represent the expression level of the metagenes at the respective sample while preserving the topological order of the metagenes in high dimension. Metagenes are grouped into co-regulated hot spots of similar up- or down-regulation using the following criteria: if Δ<em>e</em> &gt; MaxEXP * 0.9, where Δ<em>e</em> is the expression level of the metagene under consideration and MaxEXP denotes the maximum expression of all metagenes, then the metagene is classified as over expressed. Analogously if Δ<em>e</em> &lt; MinEXP * 0.9, where MinEXP denotes the minimal expression of all metagenes, the metagene is classified as under-expressed. Metagenes with the same classification that group together in the SOM are called a regulated spot.</p><p>Hierarchical clustering (average linkage mode) of expression values was conducted using R, heatmap2 function. We used two different setups for the creation. The first was based on Euclidean distance computation, which measures the distance between all entries in the expression vectors. The second was based on 1-Pearson correlation, which measures if genes follow similar trends independent of their differences in expression values. In all cases, no scaling was used.</p>                            <h3 scrollto-destination=4944593 id="4944593" class="section-title" data-legacy-id=s2f>2.6. Gene ontology enrichment</h3>
<p>The gene ontology (GO) term association file has been created by processing the parameciumDB.gff3 file, downloaded from the <em>Paramecium</em>DB database.<sup><a reveal-id="DSV014C22" data-open="DSV014C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Differentially expressed genes (DEGs) for every pairwise condition were divided into up- and down-regulated genes. We conducted GO enrichment analysis using Ontologizer (version 2.1) software using the Parent-Child-Union method for each set of up- or down-regulated genes in each pairwise comparison.<sup><a reveal-id="DSV014C23" data-open="DSV014C23" class="link link-ref link-reveal xref-bibr">23</a></sup></p>                            <h2 scrollto-destination=4944595 id="4944595" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=4944596 id="4944596" class="section-title" data-legacy-id=s3a>3.1. Experimental design</h3>
<p>It was the aim of this study to characterize gene expression patterns following epigenetically stabilized serotype expression in <em>P. tetraurelia</em>. We therefore analysed the transcriptomes of four different serotype pure cultures (51A, 51B, 51D, 51H) and furthermore characterized transcriptome alteration during cold adaptation, starvation and heat-shocks to compare epigenetically controlled gene expression patterns to short-time adaptation processes.</p><p>All experiments were started with serotypically pure cultures, meaning that 100% of cells expressed one single surface antigen obviously by 100% immobilization to a specific antiserum.</p><p>In this study, we analysed seven distinct conditions, three biological replicates each (Fig. <a reveal-id="DSV014F1" data-open="DSV014F1" class="link link-reveal link-table xref-fig">1</a>): </p><ul class="roman-lower"><li><p>Serotype 51A expressing cultures cultivated at 31°C for &gt;6 months (A.31).</p></li><li><p>Serotype 51D expressing cultures cultivated at 24°C for &gt;6 months (D.24).</p></li><li><p>Serotype 51H expressing cultures cultivated at 14°C for &gt;6 months (H.14).</p></li><li><p>Three aliquots of the D.24 cultures subjected to a 20 min heat-shock at 39°C (D.HS).</p></li><li><p>Three cultures (B.24) were grown from 51B expressing individuals, which were selected from the D.24 cultures by immobilization with anti-51D serum. Surviving individuals were subjected to autogamy and further cultivated at 24°C for 3 weeks.</p></li><li><p>Three aliquots of the 51B cultures that were gradually adapted for lower temperatures (starting at 24°C, slow cool down to 14°C within 24 h, slow cool down to 6°C in 24 h) and then cultivated 3 weeks with permanent food supply at 6°C (B.6).</p></li><li><p>Three aliquots of the D.24 cultures were subjected to autogamy and subsequently starved at 24°C for 3 weeks (D.starv).</p></li></ul>                        <a id="4944604" scrollto-destination="4944604"></a>
<div id="DSV014F1" data-id="DSV014F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/m_dsv01401.png?Expires=2147483647&Signature=1Yz8tc7b2ZZWAYJRu3wgUOqLV~TczvFrIEU4ot30jZObLi7eb7~zENJ93~zO71AiKE~SNlJwIs0HHtp8BfPrvWTQaMtFbPcEP~sQUsG461cMBG5R5fOz1e5qp523~xPU-oSKiK~T-wVjiMVyFu9lyfntJ9e79zhmtB2Fw8UF5Bd6c9y0iwwSaCgPRYhwBxy7MsgpE2ROXfXNEOYw2gTrVOfdesL2VAN5NSB3JiKM2Q146bXB4kSqLD2GvXbog7dtj9Eip--hy~XYmxRga3q2VJx1nCv7bHbpxg0oWQpb1owMxkIhVfW~qQeipRHJB0JEY9A4SEn2PEKvasonrQ8hyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The experimental design and transcriptional landscape of individual cultures. The transcriptional landscape of analysed cultures is illustrated as a 3D heatmap of grouped genes. On the left side, the actual serotype is shown, meaning the surface antigen to which antiserum 100% of the cultures showed positive immobilization reaction at the time of RNA isolation. On the right side, the individual cultivation temperature is indicated (except the heat-shock cultures that were cultured at 24°C and were transferred to 39°C only for 20 min)." path-from-xml="dsv01401.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944604/dsv01401.tiff" path-from-xml="dsv01401.tiff" target="_blank">View large</a><a section="4944604" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/dsv01401.png?Expires=2147483647&Signature=xyoA4yfpG4jdGWt8IVyU8xZLMwn2alu8VSuvDudCHjtx2FkV9rgw5Ie4aSERTz55DPWQ9IDIOWk-xCJgyKOdkez6FCAXalQMb01CSz~kcD1Xq5AfD-jXcYzwmZMYW7yh7xECxG1d1W4pamxgPy5EuGfqEmF~YlG2f97i3MLcynIpkhrd6LQNb62A7e~6jDSUeLcbv4BqHdNYgvYOlF0AI205hc87uJfVWSMuVmrNBhl15NEDFudGg5NZ2sQnLquC5gIgLaADMdzi9EV5Jb1ugtallbSgkCqIVZsMKZ4h0aomit3wR2Et3ERxyL5jq~-BkDwPllky0XLlOC6Nywhjhw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944604&ar=348350&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01401.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The experimental design and transcriptional landscape of individual cultures. The transcriptional landscape of analysed cultures is illustrated as a 3D heatmap of grouped genes. On the left side, the actual serotype is shown, meaning the surface antigen to which antiserum 100% of the cultures showed positive immobilization reaction at the time of RNA isolation. On the right side, the individual cultivation temperature is indicated (except the heat-shock cultures that were cultured at 24°C and were transferred to 39°C only for 20 min).</p></div></div><div content-id="DSV014F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/m_dsv01401.png?Expires=2147483647&Signature=1Yz8tc7b2ZZWAYJRu3wgUOqLV~TczvFrIEU4ot30jZObLi7eb7~zENJ93~zO71AiKE~SNlJwIs0HHtp8BfPrvWTQaMtFbPcEP~sQUsG461cMBG5R5fOz1e5qp523~xPU-oSKiK~T-wVjiMVyFu9lyfntJ9e79zhmtB2Fw8UF5Bd6c9y0iwwSaCgPRYhwBxy7MsgpE2ROXfXNEOYw2gTrVOfdesL2VAN5NSB3JiKM2Q146bXB4kSqLD2GvXbog7dtj9Eip--hy~XYmxRga3q2VJx1nCv7bHbpxg0oWQpb1owMxkIhVfW~qQeipRHJB0JEY9A4SEn2PEKvasonrQ8hyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The experimental design and transcriptional landscape of individual cultures. The transcriptional landscape of analysed cultures is illustrated as a 3D heatmap of grouped genes. On the left side, the actual serotype is shown, meaning the surface antigen to which antiserum 100% of the cultures showed positive immobilization reaction at the time of RNA isolation. On the right side, the individual cultivation temperature is indicated (except the heat-shock cultures that were cultured at 24°C and were transferred to 39°C only for 20 min)." path-from-xml="dsv01401.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944604/dsv01401.tiff" path-from-xml="dsv01401.tiff" target="_blank">View large</a><a section="4944604" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/dsv01401.png?Expires=2147483647&Signature=xyoA4yfpG4jdGWt8IVyU8xZLMwn2alu8VSuvDudCHjtx2FkV9rgw5Ie4aSERTz55DPWQ9IDIOWk-xCJgyKOdkez6FCAXalQMb01CSz~kcD1Xq5AfD-jXcYzwmZMYW7yh7xECxG1d1W4pamxgPy5EuGfqEmF~YlG2f97i3MLcynIpkhrd6LQNb62A7e~6jDSUeLcbv4BqHdNYgvYOlF0AI205hc87uJfVWSMuVmrNBhl15NEDFudGg5NZ2sQnLquC5gIgLaADMdzi9EV5Jb1ugtallbSgkCqIVZsMKZ4h0aomit3wR2Et3ERxyL5jq~-BkDwPllky0XLlOC6Nywhjhw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944604&ar=348350&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01401.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The experimental design and transcriptional landscape of individual cultures. The transcriptional landscape of analysed cultures is illustrated as a 3D heatmap of grouped genes. On the left side, the actual serotype is shown, meaning the surface antigen to which antiserum 100% of the cultures showed positive immobilization reaction at the time of RNA isolation. On the right side, the individual cultivation temperature is indicated (except the heat-shock cultures that were cultured at 24°C and were transferred to 39°C only for 20 min).</p></div></div><p>Replicate experiments are denoted with numeric suffixes, for example, the three replicate cultures of B.6 are denoted B.6-1, B.6-2 and B.6-3. As serotypes are expressed throughout the life cycle and usually in most individuals of a population, all cultures analysed in this study represent vegetatively growing non-synchronous cultures, thus representing individuals of different cellular age.</p><p>To ensure that transcriptome differences do not result from activation of genes specifically activated during sexual recombination (autogamy), all cultures were checked for fragmented nuclei by DAPI staining prior to RNA isolation. In addition, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a> shows gene expression profiles of different (early, intermediate, late induced) autogamy-related genes showing no significant activation in any culture.</p>                            <h3 scrollto-destination=4944607 id="4944607" class="section-title" data-legacy-id=s3b>3.2. Analysis of DEGs</h3>
<p>As <em>P. tetraurelia</em> had at least three whole-genome duplications, there are a number of genes that appear in more than one highly similar copy in the genome.<sup><a reveal-id="DSV014C24" data-open="DSV014C24" class="link link-ref link-reveal xref-bibr">24</a></sup> This complicates gene expression estimates as reads may map to more than one location in the genome, potentially biasing estimates of non-expressed genomic copies of a gene. We used the Sailfish algorithm,<sup><a reveal-id="DSV014C18" data-open="DSV014C18" class="link link-ref link-reveal xref-bibr">18</a></sup> which uses a global approach to optimize read placements on genes, to avoid false-positive expression signals of silent genes. We did a qualitative investigation by looking at the two highly similar genes SAg 51D and 51J (92.2% sequence identity) that we had previously measured with qPCR.<sup><a reveal-id="DSV014C12" data-open="DSV014C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Whereas the 51D gene has the highest expression in D serotype cultures, the 51J gene is silent, despite the highly similar sequence, and the same pattern is observed for the Sailfish estimates and with qPCR.</p><p>After quantifying all gene expression values in the samples, we used a dimensionality reduction approach with SOMs to compare the expression landscapes between different conditions and their replicates (see Section 2). Figure <a reveal-id="DSV014F1" data-open="DSV014F1" class="link link-reveal link-table xref-fig">1</a> shows the transcriptional landscapes of all analysed cultures in 3D heatmaps. As expected, individual replicates are highly similar over the complete landscape, but may differ for smaller, individual gene groups, as seen for cultures A.31-3 and B.24-3. We decided to proceed the subsequent analysis also with these samples rather than to omit them as they represent biological fluctuations that cannot be neglected, but we are aware that some DEGs may become masked in the subsequent analyses merging all three replicates.</p><p>At first glance, a similar but not identical landscape can be observed in the maps of the two cold-adapted experiments cultured at 6°C and 14°C (B.6 and H.14). More surprisingly, the serotype pure B.24 and D.24 cultures show entirely different patterns although cultivated under identical environmental conditions. In contrast, the heat-shocked and the starved cultures still show similarity to their origin, the D.24 samples. To describe the relationship between the individual cultures and conditions, we analysed DEGs in more detail.</p><p>Using estimated gene read counts from the Sailfish analysis, we used the DESeq2 algorithm to compute pairwise DEGs between different conditions (see Section 2) and set a cut-off for DEGs at an FDR value of 0.01. Figure <a reveal-id="DSV014F2" data-open="DSV014F2" class="link link-reveal link-table xref-fig">2</a> shows complete numbers (Fig. <a reveal-id="DSV014F2" data-open="DSV014F2" class="link link-reveal link-table xref-fig">2</a>A) and MA plots (Fig. <a reveal-id="DSV014F2" data-open="DSV014F2" class="link link-reveal link-table xref-fig">2</a>B) of interesting comparisons as a detailed analysis of fold changes in relation to mean expression levels (further comparisons between conditions can be found in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). The plots support observed differences between the B.24 and D.24 cultures with 2,202 genes being significantly down-regulated in serotype 51D cells. As single 51B cells spontaneously appeared in the 51D cultures and were selected by immobilization, these differences suggest that the altered SAg, specific for the serotype, is not the only DEG but many more genes show altered gene expression. In contrast, the other two sets of variants of the D.24 condition, heat-shock and starvation, show less differences: in the former 587 up-regulated genes total in the latter case of starvation 3,123 genes; however, most of them showing a moderate down-regulation of ∼log-fold change −2. The cold-adapted transcriptome of the B.6 cultures shows 1,709 down- and 1,472 up-regulated genes and therefore obviously a higher degree of alteration during cold adaptation compared with starvation. However, there is still a difference between the two sets of cold cultures B.6 and the H.14, which were adapted for a much longer time. In addition, the MA plot comparing A.31 and B.24 shows many genes that are differentially regulated although the total number of significantly DEGs is lower (639 up and 1,058 down) compared with the B.24 and D.24 comparison. </p>                        <a id="4944612" scrollto-destination="4944612"></a>
<div id="DSV014F2" data-id="DSV014F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/m_dsv01402.png?Expires=2147483647&Signature=UXqHEVeuCRF1LFiVQFrEB5WNCHeYZX8Ti802aMuyu5OO4hcrIKRkGPCxppfJKcg0ZIxOrcyhg4dPpnWFFPHvUxGh7Qb8Y9ssfYpYrp3CWGYDf3eVKF2gpqF62NcKwqpPGb2J7E8~fHKyUNKdadzmv4fQpACZtNC-pnrZ3kRh0fglTBRgJmlYlJ-WcU4kXwJxVD7r72U4T8t5E8O3-QLybmQTXIdslFooRRIPRx~jpxzC-E7iz0NrKMsPddphhaZ0aem4Xrxyo2ysT6z~bGu9OPGHf10B0K0-gndlz00CLRQ~wFKPM8LaoWUQu0nhJWHs9IZamnbd~3CeoXfn-nUwfg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="MA plots of mean differential gene expression. (A) The number of DEGs between individual comparisons (FDR ≤ 0.01). (B) MA plots of fold change expression level (y-axis) against expression level (x-axis). Each point represents a transcript; those with significant differential expression (FDR ≤ 0.01) are indicated in red. The cultures compared are indicated above each plot. A summary of all possible comparison between cultures can be found in the Supplementary Fig. 2." path-from-xml="dsv01402.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944612/dsv01402.tiff" path-from-xml="dsv01402.tiff" target="_blank">View large</a><a section="4944612" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/dsv01402.png?Expires=2147483647&Signature=o2gT2~1fpo8G5Bc3360~Q3gf1jvVvV3e2XqS2kIEelIp6dKIKExoIH2NZ-FsacTYAgp368I8jr77lMrAogXo09CPNILRTZTpovioWVD0ZD6~jk1Lb1f-xgRVBOysnkuIXVsK8eUQ-CHsuFiCnOvtvT6yAw6GSg5TmZnO3HeXoXP6S1IO3WpFZghczp-sfsGtpreqwjxXkDLwt7XcDxuXsJcCNoLEw2okQDQk4M0lwf05jBryQ7IluMHu9U8SPKIYQOEkwvI-Sc-SVLgT~WsNHwxjUcUz4J0NhjYDV6oUGQwodLNyl5-6kA5dnMHCwINyQ858kP-4WKi8na62pgz3DQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944612&ar=348350&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01402.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>MA plots of mean differential gene expression. (A) The number of DEGs between individual comparisons (FDR ≤ 0.01). (B) MA plots of fold change expression level (<em>y</em>-axis) against expression level (<em>x</em>-axis). Each point represents a transcript; those with significant differential expression (FDR ≤ 0.01) are indicated in red. The cultures compared are indicated above each plot. A summary of all possible comparison between cultures can be found in the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. 2</a>.</p></div></div><div content-id="DSV014F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/m_dsv01402.png?Expires=2147483647&Signature=UXqHEVeuCRF1LFiVQFrEB5WNCHeYZX8Ti802aMuyu5OO4hcrIKRkGPCxppfJKcg0ZIxOrcyhg4dPpnWFFPHvUxGh7Qb8Y9ssfYpYrp3CWGYDf3eVKF2gpqF62NcKwqpPGb2J7E8~fHKyUNKdadzmv4fQpACZtNC-pnrZ3kRh0fglTBRgJmlYlJ-WcU4kXwJxVD7r72U4T8t5E8O3-QLybmQTXIdslFooRRIPRx~jpxzC-E7iz0NrKMsPddphhaZ0aem4Xrxyo2ysT6z~bGu9OPGHf10B0K0-gndlz00CLRQ~wFKPM8LaoWUQu0nhJWHs9IZamnbd~3CeoXfn-nUwfg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="MA plots of mean differential gene expression. (A) The number of DEGs between individual comparisons (FDR ≤ 0.01). (B) MA plots of fold change expression level (y-axis) against expression level (x-axis). Each point represents a transcript; those with significant differential expression (FDR ≤ 0.01) are indicated in red. The cultures compared are indicated above each plot. A summary of all possible comparison between cultures can be found in the Supplementary Fig. 2." path-from-xml="dsv01402.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944612/dsv01402.tiff" path-from-xml="dsv01402.tiff" target="_blank">View large</a><a section="4944612" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/dsv01402.png?Expires=2147483647&Signature=o2gT2~1fpo8G5Bc3360~Q3gf1jvVvV3e2XqS2kIEelIp6dKIKExoIH2NZ-FsacTYAgp368I8jr77lMrAogXo09CPNILRTZTpovioWVD0ZD6~jk1Lb1f-xgRVBOysnkuIXVsK8eUQ-CHsuFiCnOvtvT6yAw6GSg5TmZnO3HeXoXP6S1IO3WpFZghczp-sfsGtpreqwjxXkDLwt7XcDxuXsJcCNoLEw2okQDQk4M0lwf05jBryQ7IluMHu9U8SPKIYQOEkwvI-Sc-SVLgT~WsNHwxjUcUz4J0NhjYDV6oUGQwodLNyl5-6kA5dnMHCwINyQ858kP-4WKi8na62pgz3DQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944612&ar=348350&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01402.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>MA plots of mean differential gene expression. (A) The number of DEGs between individual comparisons (FDR ≤ 0.01). (B) MA plots of fold change expression level (<em>y</em>-axis) against expression level (<em>x</em>-axis). Each point represents a transcript; those with significant differential expression (FDR ≤ 0.01) are indicated in red. The cultures compared are indicated above each plot. A summary of all possible comparison between cultures can be found in the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. 2</a>.</p></div></div>                            <h3 scrollto-destination=4944613 id="4944613" class="section-title" data-legacy-id=s3c>3.3. Comparison of individual transcriptome similarity</h3>
<p>To get an overview of the global similarity between the different transcriptomes, we used hierarchical clustering (Euclidean distance, average linkage) on the log-transformed TPM estimates. Figure <a reveal-id="DSV014F3" data-open="DSV014F3" class="link link-reveal link-table xref-fig">3</a> shows the clustering result as a heatmap, indicating transcription levels of all genes clustered according to their common expression behaviour in all analysed libraries. The dendrogram above the heatmap reflects the overall similarity between the transcriptome profiles. Excluding the D.24 and D.starved cultures, this analysis clearly clusters the individual biological replicates although few replicates (e.g. H.14-2 or D.HS-2) show divergence as indicated by comparing transcriptomic landscapes in Fig. <a reveal-id="DSV014F1" data-open="DSV014F1" class="link link-reveal link-table xref-fig">1</a>. Comparing the D.24 and D.starv cultures, our analysis indicates that these still have similar transcriptomes: the D.starv cultures have been separated from the well-fed D.24 cultures and maintained without feeding bacteria for 3 weeks (Fig. <a reveal-id="DSV014F1" data-open="DSV014F1" class="link link-reveal link-table xref-fig">1</a>). </p>                        <a id="4944615" scrollto-destination="4944615"></a>
<div id="DSV014F3" data-id="DSV014F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/m_dsv01403.png?Expires=2147483647&Signature=er5oNADPfWEhmRqCcR0~Nsu296wdpavjqPkKWjQAGbFI7RRoN9OcTvnHE4Og2lQPQrA7PwO2VZtmOkx5LRbb2JoYIaGfo2u8SR6ClTrhks-8YLLlvwuxUGdLsGkM~B1fgDHPENh-adgQzPh4OITARotAyzzM5oAr7qMezhVcr0KrF6A1F~5ZNWWeHHWp0qmRB4jypMNfvnZH5SaawKyYDVJ2GpwwDVDkfvg5zmiGvGBB~24BOiioq9uMRRY7bjuNfCP7sP7sBanhclx83p~5v7AuPAU2sVxStc67NqMLrt04Iao5cve5XbTB7b0B325TG~S0A1Yanpjea-nQLXWr2g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relationship between transcriptomes of cultures and their replicates. Clustering heatmap showing expression levels (colour key on the left: green high and blue low expression) of the entire transcriptome per sample (bottom label). The dendrogram (top) indicates the global similarity of the transcriptome profiles using hierarchical clustering (Euclidean distance, average linkage) on the log-transformed expression values." path-from-xml="dsv01403.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944615/dsv01403.tiff" path-from-xml="dsv01403.tiff" target="_blank">View large</a><a section="4944615" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/dsv01403.png?Expires=2147483647&Signature=g~5G5~9IZ5J-oqNQdVl-jbi3y1OhUC02RqO~gJ1zvUmbnRh6o1Nxe2jdnnI58r2ZHAKVU6QPZ9IcCStOUJsyejlNJRm5Wd8DoQkzrNwfz4E1EI17E7D8sZ8MxLM8XoP6WIvQ2N9FN1E7yTAkLYjLXInMkuTuXj6fmNZbmkCHxMD6u3nqmeo0Fmh6hBQzty~KgQlYu8mrOftJY75SCHqHsvX2CEJrE9Dn436SeKDVj70K5AFmZWGlccf0KDd9u9HvijFM~XPyW4aPPlCJzccRP7D~9OeMcSvBx8EmBvL0oCyKETsCK4aZH0BLCP47g~BZojmBvUxkB1mHiKvp9tNwWw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944615&ar=348350&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01403.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relationship between transcriptomes of cultures and their replicates. Clustering heatmap showing expression levels (colour key on the left: green high and blue low expression) of the entire transcriptome per sample (bottom label). The dendrogram (top) indicates the global similarity of the transcriptome profiles using hierarchical clustering (Euclidean distance, average linkage) on the log-transformed expression values.</p></div></div><div content-id="DSV014F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/m_dsv01403.png?Expires=2147483647&Signature=er5oNADPfWEhmRqCcR0~Nsu296wdpavjqPkKWjQAGbFI7RRoN9OcTvnHE4Og2lQPQrA7PwO2VZtmOkx5LRbb2JoYIaGfo2u8SR6ClTrhks-8YLLlvwuxUGdLsGkM~B1fgDHPENh-adgQzPh4OITARotAyzzM5oAr7qMezhVcr0KrF6A1F~5ZNWWeHHWp0qmRB4jypMNfvnZH5SaawKyYDVJ2GpwwDVDkfvg5zmiGvGBB~24BOiioq9uMRRY7bjuNfCP7sP7sBanhclx83p~5v7AuPAU2sVxStc67NqMLrt04Iao5cve5XbTB7b0B325TG~S0A1Yanpjea-nQLXWr2g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relationship between transcriptomes of cultures and their replicates. Clustering heatmap showing expression levels (colour key on the left: green high and blue low expression) of the entire transcriptome per sample (bottom label). The dendrogram (top) indicates the global similarity of the transcriptome profiles using hierarchical clustering (Euclidean distance, average linkage) on the log-transformed expression values." path-from-xml="dsv01403.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944615/dsv01403.tiff" path-from-xml="dsv01403.tiff" target="_blank">View large</a><a section="4944615" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/dsv01403.png?Expires=2147483647&Signature=g~5G5~9IZ5J-oqNQdVl-jbi3y1OhUC02RqO~gJ1zvUmbnRh6o1Nxe2jdnnI58r2ZHAKVU6QPZ9IcCStOUJsyejlNJRm5Wd8DoQkzrNwfz4E1EI17E7D8sZ8MxLM8XoP6WIvQ2N9FN1E7yTAkLYjLXInMkuTuXj6fmNZbmkCHxMD6u3nqmeo0Fmh6hBQzty~KgQlYu8mrOftJY75SCHqHsvX2CEJrE9Dn436SeKDVj70K5AFmZWGlccf0KDd9u9HvijFM~XPyW4aPPlCJzccRP7D~9OeMcSvBx8EmBvL0oCyKETsCK4aZH0BLCP47g~BZojmBvUxkB1mHiKvp9tNwWw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944615&ar=348350&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01403.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relationship between transcriptomes of cultures and their replicates. Clustering heatmap showing expression levels (colour key on the left: green high and blue low expression) of the entire transcriptome per sample (bottom label). The dendrogram (top) indicates the global similarity of the transcriptome profiles using hierarchical clustering (Euclidean distance, average linkage) on the log-transformed expression values.</p></div></div><p>As a first insight in the general comparison of the transcriptomes, the indication from the MA plots can be confirmed, because the two serotypes B.24 and D.24 that were cultured both under identical environmental conditions show huge differences and are clearly separated in the dendrogram (Fig. <a reveal-id="DSV014F3" data-open="DSV014F3" class="link link-reveal link-table xref-fig">3</a>). This suggests indeed that the differences observed here are not due to the environmental temperature or other conditions, e.g. food or composition of medium, which were identical for both. In support of the latter conclusion, the two cold-adapted cultures B.6 and H.14 are clearly separated in Fig. <a reveal-id="DSV014F3" data-open="DSV014F3" class="link link-reveal link-table xref-fig">3</a> as the overall similarity of the transcriptome reveals large differences and clusters the B.6 cultures close to the B.24 conditions, from which they were derived.</p><p>At first glance, the separation of the B.6 and H.14 transcriptomes in the dendrogram appears to be in conflict with the fact that both share the lowest number of DEGs (Fig. <a reveal-id="DSV014F2" data-open="DSV014F2" class="link link-reveal link-table xref-fig">2</a>A). In comparison to the distance-based clustering in Fig. <a reveal-id="DSV014F3" data-open="DSV014F3" class="link link-reveal link-table xref-fig">3</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a> shows a clustering obtained using Pearson correlation of gene expression values (see Section 2.5). Using correlation allows comparing gene expression patterns, despite their absolute differences in expression. In agreement with the low number of DEGs in the B.6/H14 comparison, both transcriptomes are clustered together in the second cluster analysis. We therefore conclude that a common mechanism of cold adaptation occurred in both the H.14 and the B.6 cultures (indicated by the low number of DEGs and clustering by Pearson correlation), but that expression of many genes in the B.6 cultures remains similar to expression in the parental B.24 cultures (see further analysis of these genes below).</p><p>The lowest number of DEGs can be seen between D.24 and its heat-shocked portions D.HS. As the MA plot in Fig. <a reveal-id="DSV014F2" data-open="DSV014F2" class="link link-reveal link-table xref-fig">2</a> shows, the heat-shock is characterized mainly by strong activation of a certain number of genes, which explains the separation of the D.HS transcriptomes from the two other conditions of the serotype D cultures at 24°C in the dendrogram (Fig. <a reveal-id="DSV014F3" data-open="DSV014F3" class="link link-reveal link-table xref-fig">3</a>).</p><p>Figure <a reveal-id="DSV014F3" data-open="DSV014F3" class="link link-reveal link-table xref-fig">3</a> apparently shows that all cultures undergoing an environmental change (B.6, D.starv and D.HS) still cluster to their originating cultures. Although we did not necessarily expect a very high number of DEGs in the heat-shocked cultures, one would have expected greater differences in starvation and especially in the B.6 cultures. This suggests that these changes in the 3-week period led only to a partial rearrangement of the previously governing gene expression patterns.</p>                            <h3 scrollto-destination=4944620 id="4944620" class="section-title" data-legacy-id=s3d>3.4. Cold adaptation by alterations of metabolic, DNA metabolic and translational alterations</h3>
<p>Figure <a reveal-id="DSV014F4" data-open="DSV014F4" class="link link-reveal link-table xref-fig">4</a> shows enrichment of GO terms for biological processes and individual comparisons (complete lists of differential GOs for these comparisons can be found in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary File 5</a>). </p>                        <a id="4944622" scrollto-destination="4944622"></a>
<div id="DSV014F4" data-id="DSV014F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/m_dsv01404.png?Expires=2147483647&Signature=zQU1i909XdnbNqpP99gF78QxLKHgLNbvyRGf4mhuZ-PQ1SeYRO9MDO9-xovtrOYQxl9Fku5DJy8D6L2aSb25VgzVgGaPmRFFLbLp2Za13ybHctr-OBk6vzyqhfhAQ4peTO1eDxmhSMb1FaJFXqOtw8e2bBKB36YGTNaXxcEVjagQSPW85kbotpQIcFAg5EWg4PBvr5QpvSHyBlBpVB81eeR61Du8mPoYrUhZqyC1ESmeHH4CMuNjRwPkz8uNaOclKkBGxc9b1YD9oSW8l~4aBfXQe0uXIz6PjYesENOO5nkYZemiV3uX8Vhw6coAqzPqW7LxuhQ9Yf4HEI9qTc8xCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GO enrichment analysis results. Principal component analysis (PCA) scatter plots of GO representatives of biological processes generated by the REVIGO tool that summarizes the list of differential expressed GO terms (FDR ≤ 0.02) by removing redundant terms.25 The distance between circles (representing individual GO terms) indicates the relationship between terms: closer distance means closer relationship. Bubble colour indicates significance of differential expression of an individual GO term (red low and blue high); the size (in log10P-value) indicates the percentage of genes annotated with a term in the reference database (UniProt) and thus indicates more general terms (large) and more specific ones (small). Most significant terms are labelled in the individual plot as well as representatives for groups of terms with lower significance. Green and red lateral arrows indicate the direction of regulation (up or down)." path-from-xml="dsv01404.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944622/dsv01404.tiff" path-from-xml="dsv01404.tiff" target="_blank">View large</a><a section="4944622" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/dsv01404.png?Expires=2147483647&Signature=kfCQzBoHbjxBk2VAu2KhxLDY24SEsBgZ4vGcicsPoQXBNkMyD86e5Q6DzcpDpVcl68Rzpf~bp0H~j0NLj709cS0T74-KQQPB3RqAzg-8dhR37EoCfg86rlS8Gu1Kr6z97eRdB-cdfM3xFTBB~BQ8I-NAp829WvMFxYaweEU-Hetyv6UrqZFVuxhOrgk~5ujqw5TCDdN2TGWIq1As-Fqdlpy7yHPONAZcT3X1WSjXmmGS1vrqyfom-n8pmII8IVD-D3PP6xZMqtfVIIo7LpEuSRFfe9ZlVMPFgIPaq7AztfTLZr4XUymqTjLtz--L3DV3NHl80VoUnApEvDrlEDu4eQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944622&ar=348350&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01404.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GO enrichment analysis results. Principal component analysis (PCA) scatter plots of GO representatives of biological processes generated by the REVIGO tool that summarizes the list of differential expressed GO terms (FDR ≤ 0.02) by removing redundant terms.<sup><a reveal-id="DSV014C25" data-open="DSV014C25" class="link link-ref link-reveal xref-bibr">25</a></sup> The distance between circles (representing individual GO terms) indicates the relationship between terms: closer distance means closer relationship. Bubble colour indicates significance of differential expression of an individual GO term (red low and blue high); the size (in log<sub>10</sub><em>P</em>-value) indicates the percentage of genes annotated with a term in the reference database (UniProt) and thus indicates more general terms (large) and more specific ones (small). Most significant terms are labelled in the individual plot as well as representatives for groups of terms with lower significance. Green and red lateral arrows indicate the direction of regulation (up or down).</p></div></div><div content-id="DSV014F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/m_dsv01404.png?Expires=2147483647&Signature=zQU1i909XdnbNqpP99gF78QxLKHgLNbvyRGf4mhuZ-PQ1SeYRO9MDO9-xovtrOYQxl9Fku5DJy8D6L2aSb25VgzVgGaPmRFFLbLp2Za13ybHctr-OBk6vzyqhfhAQ4peTO1eDxmhSMb1FaJFXqOtw8e2bBKB36YGTNaXxcEVjagQSPW85kbotpQIcFAg5EWg4PBvr5QpvSHyBlBpVB81eeR61Du8mPoYrUhZqyC1ESmeHH4CMuNjRwPkz8uNaOclKkBGxc9b1YD9oSW8l~4aBfXQe0uXIz6PjYesENOO5nkYZemiV3uX8Vhw6coAqzPqW7LxuhQ9Yf4HEI9qTc8xCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GO enrichment analysis results. Principal component analysis (PCA) scatter plots of GO representatives of biological processes generated by the REVIGO tool that summarizes the list of differential expressed GO terms (FDR ≤ 0.02) by removing redundant terms.25 The distance between circles (representing individual GO terms) indicates the relationship between terms: closer distance means closer relationship. Bubble colour indicates significance of differential expression of an individual GO term (red low and blue high); the size (in log10P-value) indicates the percentage of genes annotated with a term in the reference database (UniProt) and thus indicates more general terms (large) and more specific ones (small). Most significant terms are labelled in the individual plot as well as representatives for groups of terms with lower significance. Green and red lateral arrows indicate the direction of regulation (up or down)." path-from-xml="dsv01404.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944622/dsv01404.tiff" path-from-xml="dsv01404.tiff" target="_blank">View large</a><a section="4944622" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/dsv01404.png?Expires=2147483647&Signature=kfCQzBoHbjxBk2VAu2KhxLDY24SEsBgZ4vGcicsPoQXBNkMyD86e5Q6DzcpDpVcl68Rzpf~bp0H~j0NLj709cS0T74-KQQPB3RqAzg-8dhR37EoCfg86rlS8Gu1Kr6z97eRdB-cdfM3xFTBB~BQ8I-NAp829WvMFxYaweEU-Hetyv6UrqZFVuxhOrgk~5ujqw5TCDdN2TGWIq1As-Fqdlpy7yHPONAZcT3X1WSjXmmGS1vrqyfom-n8pmII8IVD-D3PP6xZMqtfVIIo7LpEuSRFfe9ZlVMPFgIPaq7AztfTLZr4XUymqTjLtz--L3DV3NHl80VoUnApEvDrlEDu4eQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944622&ar=348350&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01404.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GO enrichment analysis results. Principal component analysis (PCA) scatter plots of GO representatives of biological processes generated by the REVIGO tool that summarizes the list of differential expressed GO terms (FDR ≤ 0.02) by removing redundant terms.<sup><a reveal-id="DSV014C25" data-open="DSV014C25" class="link link-ref link-reveal xref-bibr">25</a></sup> The distance between circles (representing individual GO terms) indicates the relationship between terms: closer distance means closer relationship. Bubble colour indicates significance of differential expression of an individual GO term (red low and blue high); the size (in log<sub>10</sub><em>P</em>-value) indicates the percentage of genes annotated with a term in the reference database (UniProt) and thus indicates more general terms (large) and more specific ones (small). Most significant terms are labelled in the individual plot as well as representatives for groups of terms with lower significance. Green and red lateral arrows indicate the direction of regulation (up or down).</p></div></div><p>In all three warm/cold comparisons made here (B.6 vs. B.24; A.31 vs. B.6; A.31 vs. H.14), DNA metabolic GO terms including DNA biosynthesis and replication (initiation) are strongly up-regulated in the respective cold cultures. In addition, also oxidation/reduction processes are up-regulated in all cold-adapted cultures (only in the B.6 vs. B.24 comparison slightly below our set threshold). As this indicates higher energy metabolism, maybe due to the cold temperatures, this goes along with increased lipid catabolism in the B.6 cultures and with increased carbohydrate catabolism in H.14 cultures. As the representation of GO terms in Fig. <a reveal-id="DSV014F4" data-open="DSV014F4" class="link link-reveal link-table xref-fig">4</a> often reduced individual processes to parental GO terms (e.g. biological process), this indicates indeed that temperature adaptation alters many different processes, which require further investigation by subsequent experiments.</p>                            <h3 scrollto-destination=4944624 id="4944624" class="section-title" data-legacy-id=s3e>3.5. GO term enrichment in starvation, heat-shock and serotype comparison</h3>
<p>The GO analysis indicates that starvation most significantly triggers microtubule-based processes and movement of cellular components, thus suggesting rearrangements of the cytoskeleton, probably because these starved cells do not undergo cell divisions. Up-regulated processes are DNA catabolism and lipid metabolism, indicating increased usage of lipids for energy metabolism.</p><p>In the heat-shocked cultures, the most significant up-regulated GO term is ‘protein folding’ (GO: 0006457, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary File 5</a>), which involves the heat-shock response chaperones in agreement with our expectations. The GO enrichment displayed in Fig. <a reveal-id="DSV014F4" data-open="DSV014F4" class="link link-reveal link-table xref-fig">4</a> clearly shows that many other global processes are down-regulated. In the special situation of the heat-shock, these down-regulations may not be necessarily considered as controlled regulation of gene expression but also a by-product of thermal instability of individual mRNA species as discussed later.</p><p>In general, the B.24 and D.24 comparison shows that many processes are down-regulated: the most significant one is again ‘oxidoreductase activity’ (GO: 0016491), indicating that the D.24 cultures have a lower energy metabolism compared with the B.24 cells, despite identical environmental cultivation conditions and thus food supply. In this comparison, also GO terms concerning ‘chromatin assembly’ become apparent in Fig. <a reveal-id="DSV014F4" data-open="DSV014F4" class="link link-reveal link-table xref-fig">4</a>. A closer look reveals that chromatin-modifying enzymes are differentially expressed, namely genes of three histone deacetylase isoforms, chromodomain containing proteins and nuclear assembly proteins, and surprisingly the core histones themselves (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>A). Several isoforms of histone genes H2A, H2B, H3 and H4 are down-regulated in the D.24 cultures. <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4B and C</a> show, for histone H3, that only those isoforms are down-regulated that are constitutively expressed throughout the life cycle and that the expression of developmental specific isoforms is not affected.</p><p>Further wet lab work has to clarify whether the global down-regulation of the core histones in our RNA-seq data indeed indicates a genome-wide loss of nucleosome occupancy: histone gene expression was described to be complex and to occur at several levels including post-transcriptional regulation.<sup><a reveal-id="DSV014C26" data-open="DSV014C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Therefore, our data do not allow for conclusions that indeed histone protein levels are strongly regulated here. However, if lower nucleosome occupancy would be true in the D.24 cultures, the high number of DEGs to the B.24 cultures might also be the result of a complex re-organization of the macronuclear chromatin.</p><p>In yeast, cellular ageing was described to be accompanied by a loss of nucleosomes and a global transcriptional up-regulation,<sup><a reveal-id="DSV014C27" data-open="DSV014C27" class="link link-ref link-reveal xref-bibr">27</a></sup> which is different to our observations as the D.24 cultures show more down-regulated genes. As the cells of the D.24 cultures were not synchronized in cellular age, we cannot rule out an age-dependent regulation of histones, although the down-regulation is maintained in the D.starv cultures, which represent 10–20 division old F1 individuals of the D.24 cultures.</p><p>A similar result of decreased expression of the core histones was recently observed in <em>P. tetraurelia</em> by microarray analysis during silencing of an RNA-dependent RNA polymerase 3 (RDR3), which resulted in a co-expression of SAgs accompanied with decreased expression of the core histones<sup><a reveal-id="DSV014C12" data-open="DSV014C12" class="link link-ref link-reveal xref-bibr">12</a></sup> (M. Simon, unpublished data). As these cultures were of the same cellular age as the controls, any age-dependent influence can be ruled out in these experiments. Surprisingly, we observe a similar phenomenon of low core histone expression here in wild-type cultures of different serotypes. As <em>RDR3</em> silencing is also accompanied with drastically decreased cells divisions,<sup><a reveal-id="DSV014C11" data-open="DSV014C11" class="link link-ref link-reveal xref-bibr">11</a></sup> this was also observed for D.24 cultures, which showed slightly fewer cell divisions per day compared with the B.24 cultures (data not shown). As reduced histone supply was described to extend the S phase in <em>Drosophila</em>, the delivery with core histones turns out to be a crucial component regulating the cell cycle.<sup><a reveal-id="DSV014C28" data-open="DSV014C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Although the ciliate macronucleus divides amitotically, the prolonged cell cycle fits to our observations in the D.24 cultures and <em>RDR3</em>-silencing cultures.</p><p>In support of this conclusion, the heat-shocked derivatives of the D.24 samples show a down-regulation of the core histones (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). Although we did not analyse the division rate of the heat-shocked cells, the down-regulation of the histones is consistent with a recent characterization of the heat-shock in the unicellular algae <em>Chlamydomonas</em> showing decreased histone expression by proteomic analysis; in this particular case, cells show an immediate stop of cell divisions.<sup><a reveal-id="DSV014C29" data-open="DSV014C29" class="link link-ref link-reveal xref-bibr">29</a></sup></p><p>It seems tempting to speculate here that <em>Paramecium</em> controls cell division also by histone supply although the underlying regulatory mechanisms and the involvement remain unclear, especially the involvement of the <em>RDR3-</em>associated RNAi machinery. In this context, recent work in the ciliate <em>Stylonychia</em> demonstrated that the knock down of the <em>Stylonychia</em> Piwi resulted in down-regulation of individual H3 isoforms, thus providing another example for the connection of RNAi and histone expression.<sup><a reveal-id="DSV014C30" data-open="DSV014C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Further studies have to clarify the role of histone gene expression and RNAi components as this would represent a powerful and new parameter of epigenetic influence to chromatin and genome integrity.</p>                            <h3 scrollto-destination=4944633 id="4944633" class="section-title" data-legacy-id=s3f>3.6. Constitutive and regulated expression of HSP70 isoforms</h3>
<p>Ten HSP70 isoforms have been described earlier for <em>P. tetraurelia,</em><sup><a reveal-id="DSV014C6" data-open="DSV014C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSV014C7" data-open="DSV014C7" class="link link-ref link-reveal xref-bibr">7</a></sup> and we added six more to the annotation, which were identified by a proteome-wide Pfam search (see Section 2 for details). Figure <a reveal-id="DSV014F5" data-open="DSV014F5" class="link link-reveal link-table xref-fig">5</a>A shows the phylogenetic relationship between these isoforms and their putative subcellular localization. Figure <a reveal-id="DSV014F5" data-open="DSV014F5" class="link link-reveal link-table xref-fig">5</a>B illustrates the TPM normalized expression levels of these HSP70 isoforms, where heat-inducible up-regulation can only be observed in the cytosolic group, in agreement with a previous analysis.<sup><a reveal-id="DSV014C7" data-open="DSV014C7" class="link link-ref link-reveal xref-bibr">7</a></sup> All other HSP70 isoforms show constitutive expression. </p>                        <a id="4944635" scrollto-destination="4944635"></a>
<div id="DSV014F5" data-id="DSV014F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/m_dsv01405.png?Expires=2147483647&Signature=Wgi3LOuR7One9a3D8HmOOLdP3fLg~JtIujI7WRq6qgJN9d9ZdmWZEYDuAnnZmsK-pZzzR4T-cEOAfH-FTKa4St-stI0fbl8ioNY6ygLbYRdGF04wNdj1XFO40rtJyBsXsoFPaw5qn579qZ-vDVblzK6boPADTB7SZ9c4kgkSLyuAx~Ikw7o13r0fE2GMQbpmJDWWLyRZcpVFVc2rkLVZZ1nIUyn7-c-n9nmcDX8xBFVVdhmQY3c~5-TvE6Pok6Ui5P5Rtql7ubUEbSvzwQTeyRmMXmbSJE7LFGORsQ-4ivKCfT~GrDN4dCb3mLbtH~hsweFa9kZ4~l1prDI5VZSx6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression level of heat-shock (HSP70) genes. (A) Neighbour-joining tree of HSP70 proteins of Paramecium tetraurelia (with 1,000 bootstrap replicates). The tree includes 10 previously described HSP70 genes (HSP70Pt01-10)6,7 and additionally identified HSP70 isoforms (HSP70Pt11-16). Putative localization of isoforms is based on similarity analysis to HSPs in other organisms and on the classification in Krenek et al.7 ER-endoplasmic reticulum, CY-cytosol. (B) TPM normalized expression values for all described HSP70 isoforms in all samples are shown, and cells are coloured according to high (red) or low (blue) expression." path-from-xml="dsv01405.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944635/dsv01405.tiff" path-from-xml="dsv01405.tiff" target="_blank">View large</a><a section="4944635" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/dsv01405.png?Expires=2147483647&Signature=3HoEdHHpltihmzfj9LYxo3TrfPwwIY3T4GPFW0US0CXPLf0tckFYS86vFZ5cHKhiFIggQuxqZ2uoehM~Yfhs~aQB72i2ugCUkvp36HkBteQ1uUWBecVmh5Qh~e2Eg4KJhvg4QIXTuh8wXbIW-A961hu-QlXfN8Ws-l82lD21Flt~Ukn54FuMyiXRO8iYFu3JAXanb~3~SqVs2xtzI1w5RMviLP82O0jQFf0cpqfjI7KwXOsY-nWSmfnMkbn-WseZEqdtVSHlwuTdtY06OOXMh9aQAog0Wh6yJ4mGwQlh6ov2NwxhUAkZH2gcBPHwsKQb5BEqVpZp62hm5f4995OjqA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944635&ar=348350&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01405.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression level of heat-shock (HSP70) genes. (A) Neighbour-joining tree of HSP70 proteins of <em>Paramecium tetraurelia</em> (with 1,000 bootstrap replicates). The tree includes 10 previously described HSP70 genes (HSP70Pt01-10)<sup><a reveal-id="DSV014C6" data-open="DSV014C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSV014C7" data-open="DSV014C7" class="link link-ref link-reveal xref-bibr">7</a></sup> and additionally identified HSP70 isoforms (HSP70Pt11-16). Putative localization of isoforms is based on similarity analysis to HSPs in other organisms and on the classification in Krenek et al.<sup><a reveal-id="DSV014C7" data-open="DSV014C7" class="link link-ref link-reveal xref-bibr">7</a></sup> ER-endoplasmic reticulum, CY-cytosol. (B) TPM normalized expression values for all described HSP70 isoforms in all samples are shown, and cells are coloured according to high (red) or low (blue) expression.</p></div></div><div content-id="DSV014F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/m_dsv01405.png?Expires=2147483647&Signature=Wgi3LOuR7One9a3D8HmOOLdP3fLg~JtIujI7WRq6qgJN9d9ZdmWZEYDuAnnZmsK-pZzzR4T-cEOAfH-FTKa4St-stI0fbl8ioNY6ygLbYRdGF04wNdj1XFO40rtJyBsXsoFPaw5qn579qZ-vDVblzK6boPADTB7SZ9c4kgkSLyuAx~Ikw7o13r0fE2GMQbpmJDWWLyRZcpVFVc2rkLVZZ1nIUyn7-c-n9nmcDX8xBFVVdhmQY3c~5-TvE6Pok6Ui5P5Rtql7ubUEbSvzwQTeyRmMXmbSJE7LFGORsQ-4ivKCfT~GrDN4dCb3mLbtH~hsweFa9kZ4~l1prDI5VZSx6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression level of heat-shock (HSP70) genes. (A) Neighbour-joining tree of HSP70 proteins of Paramecium tetraurelia (with 1,000 bootstrap replicates). The tree includes 10 previously described HSP70 genes (HSP70Pt01-10)6,7 and additionally identified HSP70 isoforms (HSP70Pt11-16). Putative localization of isoforms is based on similarity analysis to HSPs in other organisms and on the classification in Krenek et al.7 ER-endoplasmic reticulum, CY-cytosol. (B) TPM normalized expression values for all described HSP70 isoforms in all samples are shown, and cells are coloured according to high (red) or low (blue) expression." path-from-xml="dsv01405.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944635/dsv01405.tiff" path-from-xml="dsv01405.tiff" target="_blank">View large</a><a section="4944635" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/dsv01405.png?Expires=2147483647&Signature=3HoEdHHpltihmzfj9LYxo3TrfPwwIY3T4GPFW0US0CXPLf0tckFYS86vFZ5cHKhiFIggQuxqZ2uoehM~Yfhs~aQB72i2ugCUkvp36HkBteQ1uUWBecVmh5Qh~e2Eg4KJhvg4QIXTuh8wXbIW-A961hu-QlXfN8Ws-l82lD21Flt~Ukn54FuMyiXRO8iYFu3JAXanb~3~SqVs2xtzI1w5RMviLP82O0jQFf0cpqfjI7KwXOsY-nWSmfnMkbn-WseZEqdtVSHlwuTdtY06OOXMh9aQAog0Wh6yJ4mGwQlh6ov2NwxhUAkZH2gcBPHwsKQb5BEqVpZp62hm5f4995OjqA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944635&ar=348350&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01405.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression level of heat-shock (HSP70) genes. (A) Neighbour-joining tree of HSP70 proteins of <em>Paramecium tetraurelia</em> (with 1,000 bootstrap replicates). The tree includes 10 previously described HSP70 genes (HSP70Pt01-10)<sup><a reveal-id="DSV014C6" data-open="DSV014C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSV014C7" data-open="DSV014C7" class="link link-ref link-reveal xref-bibr">7</a></sup> and additionally identified HSP70 isoforms (HSP70Pt11-16). Putative localization of isoforms is based on similarity analysis to HSPs in other organisms and on the classification in Krenek et al.<sup><a reveal-id="DSV014C7" data-open="DSV014C7" class="link link-ref link-reveal xref-bibr">7</a></sup> ER-endoplasmic reticulum, CY-cytosol. (B) TPM normalized expression values for all described HSP70 isoforms in all samples are shown, and cells are coloured according to high (red) or low (blue) expression.</p></div></div><p>The only cytosolic HSP70 isoform that shows high expression in all cultures is HSP70Pt-01, and interestingly this isoform also shows a second expression peak in D.24 and D.starv cultures. This does not seem to be due to the cultivation temperature, as lower expression can be observed at the same cultivation temperature in B.24 cultures (Fig. <a reveal-id="DSV014F5" data-open="DSV014F5" class="link link-reveal link-table xref-fig">5</a>B). Similarly, also HSP70Pt-05 shows this activation in D.24 and D.starv cultures, however to a lower degree.</p><p>Interestingly, a continuous up-regulation of HSP70 isoforms was recently also reported in <em>Paramecium</em> in response to endosymbionts, another form of external stress to the cell. The transcriptome comparison of <em>Paramecium bursaria</em> without and with endosymbiotic <em>Chlorella</em> ssp. algae revealed individual HSP70 isoforms up-regulated,<sup><a reveal-id="DSV014C31" data-open="DSV014C31" class="link link-ref link-reveal xref-bibr">31</a></sup> and similar findings were also reported for infections with bacterial endosymbionts in <em>Paramecium caudatum</em>.<sup><a reveal-id="DSV014C32" data-open="DSV014C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Other studies suggest that bacterial symbiont bearing paramecia show an increased stress resistance,<sup><a reveal-id="DSV014C33" data-open="DSV014C33" class="link link-ref link-reveal xref-bibr">33</a></sup> and the authors concluded that permanent HSP70 activation awards increased stress tolerance. Interestingly, symbiont-induced expression of HSP70 isoforms is shown to be permanent even after the removal of bacteria by antibiotics, indicating an epigenetic manifestation of this acquired gene expression.<sup><a reveal-id="DSV014C34" data-open="DSV014C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Here, our results support a permanent activation of an individual HSP70 isoform (HSP70Pt-01) in D.24 as well as the D.starv cultures but not in other samples.</p>                            <h3 scrollto-destination=4944638 id="4944638" class="section-title" data-legacy-id=s3g>3.7. Expression pattern of the surface antigen multigene family</h3>
<p>As mentioned above, serotypes represent a special kind of differentiation process manifesting a gene expression pattern of the SAg family by epigenetic mechanisms: only one gene is expressed at a time.<sup><a reveal-id="DSV014C35" data-open="DSV014C35" class="link link-ref link-reveal xref-bibr">35</a></sup> The most dominant environmental factor triggering serotype expression was described to be the temperature, and once induced they can be transferred to different cultivation conditions without serotype switches, thus indicating a self-stabilizing gene expression mechanism. Recent studies demonstrated that the core family consists of eight genes, some of them with a certain number of isoforms, indicated by Greek letters.<sup><a reveal-id="DSV014C12" data-open="DSV014C12" class="link link-ref link-reveal xref-bibr">12</a></sup> In all our experiments we started with serotype pure cultures, meaning 100% of the cells responded to specific antiserum in the immobilization reaction. Figure <a reveal-id="DSV014F6" data-open="DSV014F6" class="link link-reveal link-table xref-fig">6</a>A shows the gene expression levels based on TPM normalization. The figure shows high expression levels of the individual SAg that corresponds to the antiserum with which the cells reacted. This is not surprising for the A.31, B.24, D.24 and H.14 cultures. However, the B.6 cultures, which represent an aliquot of the B.24 cultures, still retain high expression of the 51B gene. This means that the expression of the 51B gene shows stable expression, even after 3 weeks at a temperature of 6°C. Typically, <em>P. tetraurelia</em> stock 51 expresses the 51H gene at low temperatures.<sup><a reveal-id="DSV014C36" data-open="DSV014C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Our data show that the 51H gene indeed has increased expression in the B.6 cultures, nevertheless the isolated cells still responded 100% to the 51B antiserum. This means that the actual serotype persists even drastically temperature changes. From our experience, it is likely that some individuals of the B.6 cells will indeed shift to 51H after several weeks more of cultivation at 14°C or 6°C, but production of serotype pure cultures will require the selection of 51H expressing individuals. In such a case, serotype shifts can indeed be triggered by temperature alterations, but the selection of individual transformants is required. </p>                        <a id="4944645" scrollto-destination="4944645"></a>
<div id="DSV014F6" data-id="DSV014F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/m_dsv01406.png?Expires=2147483647&Signature=nn-e65tbzrTC7vKye5ZApNY6uStDfQB-R3QLggMTd68lN-rYnXxpIL9yBgRtNEIRqkgTa9eWcZChB3SVJaKBXi-59vJOfR3MEzSaSR0~1DjkR0-N3VRdhZfjnKHCO5rK8C0Q-86~XOoLIPepPmWS4UhfuZjJLBIkpTdCnwYnMR0Y5ve~4yJnaLAnAs8LuhqvKoUOb1h9t3z~5fo~6B2VRFySb2zEJI4KhuuFz9aiQL5-qQT7WBHPdRWRyjDLbtgFljkJNkTWOBk3BAB~Ah55BmrI-Ob5uMC~htwNpdpmNhHw34CimtGqEvXNBq7Vdu~O4wPWFFNZogrKq8Y8FqVAng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression level of SAgs. (A) TPM normalized expression values for members of the SAg multigene family are shown in the table, and colour shading indicates relative expression level (dark blue—low, red—high). (B) Identification of regulated spots (gene clusters showing co-regulation in all analysed samples) in serotype pure cultures. Plots on the left show the global profile, plots on the right show regulated spots. (blue—low expression, red—high expression). The samples are indicated above the plots, a representative replicate is shown. Black arrows indicate the regulated cluster containing exclusively the actual expressed SAg (SAg 51B in B.24; SAg 51D in D.24; SAg 51A in A.31). The open arrows in the second row (B.6, D.starv, D.HS) indicate the position of the regulated spot of the cultures origin (B.24; D.24 and D.24). (C) Fold change expression level of cytosolic HSP70 isoforms and the 51D surface antigen gene of the individual three replicates of the heat-shocked cultures." path-from-xml="dsv01406.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944645/dsv01406.tiff" path-from-xml="dsv01406.tiff" target="_blank">View large</a><a section="4944645" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/dsv01406.png?Expires=2147483647&Signature=Fw7bYZsELJobRVCRiKq-JekbnUGOphVXdhcej14aGPYxNIk-tRC-vSoq2W4Hl6MxTpZARtVZbDESCAXO-5pwfp~fO8tQ~hE~0Or58DMg4qVNgbnIPG21Jnvt8KafnSH1AsuEWw6ChHHUjFLtE7MSZuJCHZItIJXdC90TjWk3ALzTuH6HgpIEVgIDSqv-Wvf3J~QGa66hTxGiJbqHVrySpnS4eyvTK185gsBIBB~AquDNPWnFCJNg146Nuj5zNGx9NgBuVNFu~FWrDak5b36u9-o~h9RXPlrffwnqb72ha8YzdQu-Pxs9h89EmGTSdna0Z0rFRTq9smWMCW6yImd36A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944645&ar=348350&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01406.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression level of SAgs. (A) TPM normalized expression values for members of the SAg multigene family are shown in the table, and colour shading indicates relative expression level (dark blue—low, red—high). (B) Identification of regulated spots (gene clusters showing co-regulation in all analysed samples) in serotype pure cultures. Plots on the left show the global profile, plots on the right show regulated spots. (blue—low expression, red—high expression). The samples are indicated above the plots, a representative replicate is shown. Black arrows indicate the regulated cluster containing exclusively the actual expressed SAg (SAg 51B in B.24; SAg 51D in D.24; SAg 51A in A.31). The open arrows in the second row (B.6, D.starv, D.HS) indicate the position of the regulated spot of the cultures origin (B.24; D.24 and D.24). (C) Fold change expression level of cytosolic HSP70 isoforms and the 51D surface antigen gene of the individual three replicates of the heat-shocked cultures.</p></div></div><div content-id="DSV014F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/m_dsv01406.png?Expires=2147483647&Signature=nn-e65tbzrTC7vKye5ZApNY6uStDfQB-R3QLggMTd68lN-rYnXxpIL9yBgRtNEIRqkgTa9eWcZChB3SVJaKBXi-59vJOfR3MEzSaSR0~1DjkR0-N3VRdhZfjnKHCO5rK8C0Q-86~XOoLIPepPmWS4UhfuZjJLBIkpTdCnwYnMR0Y5ve~4yJnaLAnAs8LuhqvKoUOb1h9t3z~5fo~6B2VRFySb2zEJI4KhuuFz9aiQL5-qQT7WBHPdRWRyjDLbtgFljkJNkTWOBk3BAB~Ah55BmrI-Ob5uMC~htwNpdpmNhHw34CimtGqEvXNBq7Vdu~O4wPWFFNZogrKq8Y8FqVAng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression level of SAgs. (A) TPM normalized expression values for members of the SAg multigene family are shown in the table, and colour shading indicates relative expression level (dark blue—low, red—high). (B) Identification of regulated spots (gene clusters showing co-regulation in all analysed samples) in serotype pure cultures. Plots on the left show the global profile, plots on the right show regulated spots. (blue—low expression, red—high expression). The samples are indicated above the plots, a representative replicate is shown. Black arrows indicate the regulated cluster containing exclusively the actual expressed SAg (SAg 51B in B.24; SAg 51D in D.24; SAg 51A in A.31). The open arrows in the second row (B.6, D.starv, D.HS) indicate the position of the regulated spot of the cultures origin (B.24; D.24 and D.24). (C) Fold change expression level of cytosolic HSP70 isoforms and the 51D surface antigen gene of the individual three replicates of the heat-shocked cultures." path-from-xml="dsv01406.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944645/dsv01406.tiff" path-from-xml="dsv01406.tiff" target="_blank">View large</a><a section="4944645" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/dsv01406.png?Expires=2147483647&Signature=Fw7bYZsELJobRVCRiKq-JekbnUGOphVXdhcej14aGPYxNIk-tRC-vSoq2W4Hl6MxTpZARtVZbDESCAXO-5pwfp~fO8tQ~hE~0Or58DMg4qVNgbnIPG21Jnvt8KafnSH1AsuEWw6ChHHUjFLtE7MSZuJCHZItIJXdC90TjWk3ALzTuH6HgpIEVgIDSqv-Wvf3J~QGa66hTxGiJbqHVrySpnS4eyvTK185gsBIBB~AquDNPWnFCJNg146Nuj5zNGx9NgBuVNFu~FWrDak5b36u9-o~h9RXPlrffwnqb72ha8YzdQu-Pxs9h89EmGTSdna0Z0rFRTq9smWMCW6yImd36A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944645&ar=348350&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01406.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression level of SAgs. (A) TPM normalized expression values for members of the SAg multigene family are shown in the table, and colour shading indicates relative expression level (dark blue—low, red—high). (B) Identification of regulated spots (gene clusters showing co-regulation in all analysed samples) in serotype pure cultures. Plots on the left show the global profile, plots on the right show regulated spots. (blue—low expression, red—high expression). The samples are indicated above the plots, a representative replicate is shown. Black arrows indicate the regulated cluster containing exclusively the actual expressed SAg (SAg 51B in B.24; SAg 51D in D.24; SAg 51A in A.31). The open arrows in the second row (B.6, D.starv, D.HS) indicate the position of the regulated spot of the cultures origin (B.24; D.24 and D.24). (C) Fold change expression level of cytosolic HSP70 isoforms and the 51D surface antigen gene of the individual three replicates of the heat-shocked cultures.</p></div></div><p>Comparing the B.24 and B.6 transcriptomes in the heatmap representing the entire transcriptome, Fig. <a reveal-id="DSV014F3" data-open="DSV014F3" class="link link-reveal link-table xref-fig">3</a> indicates that many genes maintain their expression pattern, thus explaining the general high similarity. Additionally, the D.24 cultures are still highly similar to the D.starv cultures, and the serotype expression here indicates permanent expression of the 51D gene although slightly reduced (Fig. <a reveal-id="DSV014F6" data-open="DSV014F6" class="link link-reveal link-table xref-fig">6</a>A). For these two examples of environmental alterations (food limitation: D.24 vs. D.starv) and extreme temperature shift (B.24 vs. B.6), our data show that the transcriptomes are indeed much more similar compared with the selected individuals of spontaneous transformants, namely the selected 51B expressing cells from the D.24 cultures that were cultivated under identical conditions.</p><p>This suggests that spontaneously occurring transcriptome alterations, which may be regulated by epigenetic mechanisms similar to the serotype system, alter gene expression in <em>Paramecium</em> to a much higher degree compared with short-time adaptation to environmental changes.</p>                            <h3 scrollto-destination=4944665 id="4944665" class="section-title" data-legacy-id=s3h>3.8. Many genes show co-regulation with the dominant SAg and sensitivity to heat-shocks</h3>
<p>To see whether other genes are co-regulated with the surface antigens, we defined ‘regulated spots’ in the transcriptomes, which are characteristic sets of lowly or highly expressed genes per condition (Fig. <a reveal-id="DSV014F6" data-open="DSV014F6" class="link link-reveal link-table xref-fig">6</a>B). Genes that are part of a spot show not only coherent up- or down-regulation in one individual sample, but moreover they are forming a group of co-regulated genes that behave similarly among all analysed conditions and samples.</p><p>These data show that a huge number of genes are indeed co-regulated with individual antigens. These groups contain a large number of genes: on average between the replicates 2,468 genes in A.31, 1,180 in B.24 and 1,860 in D.24, thus representing a high percentage of the genome. In case of the B.24/D.24 comparison, the visualization of the regulated spots almost shows a mirror image of one another, thus explaining the clear separation of both states in the dendrogram (Fig. <a reveal-id="DSV014F3" data-open="DSV014F3" class="link link-reveal link-table xref-fig">3</a>); it is still surprising that this difference is not due to environmental alterations but more likely due to spontaneous alterations. The short-time derivatives of these cultures (B.6 and D.starv) still show these spots (open arrow), although the pattern seems slightly different (Fig. <a reveal-id="DSV014F6" data-open="DSV014F6" class="link link-reveal link-table xref-fig">6</a>B), thus indicating that a huge number of genes follows the expression behaviour of the SAg family and shows stability to environmental alterations within 3 weeks.</p><p>Surprisingly, only the heat-shocked cultures (D.HS) alter serotype expression and, moreover, show a loss of the highly expressed spot of regulated genes, which was characteristic for all 51D expressing cultures. This means that a heat-shock of 20 min alters the expression of genes, which we discussed to be co-regulated with the epigenetically controlled SAg family.</p><p>Comparing the transcript level of the cytosolic HSP70 isoforms per replicate to the expression levels of SAgs in Fig. <a reveal-id="DSV014F6" data-open="DSV014F6" class="link link-reveal link-table xref-fig">6</a>C, the 51D mRNA is drastically reduced after 20 min heat-shock. This is apparent in the D.HS-1 and -3 cultures, and only to a lesser extent in the D.24-2 culture (only ∼50% reduction). Further, comparing this with the HSP expression data, an inverse correlation becomes apparent: culture D.HS-2 also shows the lowest activation of the cytosolic HSP70 isoforms, thus indicating that this culture was only undergoing a moderate heat-shock but shows the lowest reduction of SAg 51D transcripts.</p><p>As a consequence, SAg expression levels seem to be sensitive for heat-shocks which is supported by studies in the close relative <em>Tetrahymena thermophila,</em> where mRNA of variable surface antigens (<em>Ser</em> genes) was shown to be sensitive to heat-shocks; however, not necessarily because of physical instability but by active degradation by a <em>de novo</em> synthesized protein factor.<sup><a reveal-id="DSV014C37" data-open="DSV014C37" class="link link-ref link-reveal xref-bibr">37</a>,<a reveal-id="DSV014C38" data-open="DSV014C38" class="link link-ref link-reveal xref-bibr">38</a></sup> This finding fits also very well to reports that a sudden change in temperature is much more efficient to induce serotype shifts compared with gradual changes.<sup><a reveal-id="DSV014C10" data-open="DSV014C10" class="link link-ref link-reveal xref-bibr">10</a></sup></p><p>In conclusion, we found that many genes in the <em>Paramecium</em> genome show expression patterns that correlate with serotype expression. Their expression behaviour is characterized by high expression levels, certain stability to environmental changes and the sensitivity to heat-shocks (Fig. <a reveal-id="DSV014F6" data-open="DSV014F6" class="link link-reveal link-table xref-fig">6</a>B).</p><p>As we know that heat-shocks induce serotype shifts more efficiently compared with more gradual changes and that the newly manifested serotype then becomes stabilized by epigenetic mechanisms, our finding that the entire spot of genes co-regulated with the SAg suggests that heat-shocks can induce widespread and long-term transcriptome alterations. In such case, the heat-shock could be interpreted as a kind of epigenetic reset on a large class of genes in the genome and would have an important influence on epigenetic memory.</p>                            <h3 scrollto-destination=4944703 id="4944703" class="section-title" data-legacy-id=s3i>3.9. Specific activation of SAgs in subtelomeric regions</h3>
<p>A TPE was recently reported to control SAg expression and silencing by RNA interference in <em>P. tetraurelia</em>.<sup><a reveal-id="DSV014C12" data-open="DSV014C12" class="link link-ref link-reveal xref-bibr">12</a></sup> The classical TPE is believed to silence genes by a spreading of the heterochromatic state from the telomere into the subtelomeric regions, and this is believed to be involved in the regulation of variant surface antigens in a series of pathogenic microbes.<sup><a reveal-id="DSV014C39" data-open="DSV014C39" class="link link-ref link-reveal xref-bibr">39</a></sup> However, it remains unknown how individual genes avoid silencing and become thus expressed. To see whether the activation of an SAg goes along with a general activation of genes in this particular subtelomere, we investigated these loci more in detail. Figure <a reveal-id="DSV014F7" data-open="DSV014F7" class="link link-reveal link-table xref-fig">7</a> shows the subtelomeric surrounding of macronuclear chromosomes harbouring the here investigated SAgs 51A, 51B, 51D and 51H. The position of telomeres is indicated by EOS (end of scaffolds) as predicted by the genome project resulting in a majority of closed scaffolds with telomeric repeats on both ends<sup><a reveal-id="DSV014C24" data-open="DSV014C24" class="link link-ref link-reveal xref-bibr">24</a></sup> or by the letter ‘T’ in a triangle indicating internal telomeric sites.<sup><a reveal-id="DSV014C12" data-open="DSV014C12" class="link link-ref link-reveal xref-bibr">12</a></sup> These internal telomeric sites are due to the ∼800n polyploidy macronuclear chromosomes that exist in different length versions because of imprecise DNA elimination processes during maturation of small macronuclear chromosomes from larger micronuclear chromosomes.<sup><a reveal-id="DSV014C40" data-open="DSV014C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Figure <a reveal-id="DSV014F7" data-open="DSV014F7" class="link link-reveal link-table xref-fig">7</a> gives the combined information about gene position and expression levels of genes surrounding the respective SAg in all four serotype pure states 51A, 51B, 51D, 51H. The data do not indicate any events of spreading of heterochromatic silencing: on the one hand, activation of a subtelomeric SAg does not involve a loss of silencing of downstream genes (between SAg and telomere); on the other hand, the on/off state of the SAg does not apparently influence gene expression level of any neighbouring gene: their expression appears independent. As a conclusion, the TPE associated with SAg silencing and activation cannot be explained by a simple spreading of telomeric heterochromatin into subtelomeric regions, rather it seems more likely that the TPE influences SAg regulation in combination with a gene specific factor to specifically regulate one individual locus, thus avoiding global spreading events. </p>                        <a id="4944714" scrollto-destination="4944714"></a>
<div id="DSV014F7" data-id="DSV014F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/m_dsv01407.png?Expires=2147483647&Signature=qerBLKR~NJwCyWa9b1IuvBDyuLeABIxCfMKbhBFz~I4bDd0yZcRIN3uWzL3EnRhIncnK9IqN4~sPCJiFPK3AVu5O4sc5WWhB3OBl9c4g6TAqbsuj4RXwwLS2J3TK4ZTQ1sUlHGqGKJ8PeFHfSQm4fN9gkmG05onKDFMJRiFaOpUblN72XKfU0GIN1zCDyzvRgbX7Ex3mt15KuFgcdGyl70j3aFyPl8c1zYefrB-4yI4NNg6BgCttWr3SSD0gKVkBzDYQ~RNN6WbEMxSNP15Q4DwehY15w5IaAQvJuq6yVHfeIYLvKGP8umNYWYk0MgWn7wlBphvOZBBL5U7M74L8wA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genomic map and expression of SAg and neighbouring genes in subtelomeric regions. Genomic maps of subtelomeric regions of scaffold 106 (SAg 51A), scaffold 142 (SAg 51H), scaffold 143 (SAg 51B) and scaffold 159 (51D) indicate individual ORFs by arrows (data from ParameciumDB22 and Baranasic et al.12). The position in the scaffold is indicated above in kbp (k). Expression level of the SAg and nearby genes (numbered by –X upstream and +X downstream relative to the SAg) are shown by TPM normalized data (logarithmic scale) in the graph below the genomic maps, respectively. Internal telomeric sites representing chromosome heterogeneity are indicated by the triangle containing the letter ‘T’; EOS, end of scaffold." path-from-xml="dsv01407.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944714/dsv01407.tiff" path-from-xml="dsv01407.tiff" target="_blank">View large</a><a section="4944714" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/dsv01407.png?Expires=2147483647&Signature=sp65Uz~tBm3GdvfQ5yfJEQKz-dCt4yvVOrWdamSrVakOnTAwQ0pz2Xpv2SeKAHrZgNskbXrPyQ~O-47OZ4g0Ze7sqr~K6ydPQQEK1YLDJd233h~dVHIfCD218l4XzDLq7gK~3Z1BQ3dchrOZLzJkr0cqMySuR4l0lHa3MurLFG~KyVMb35Ebknz1~wNvXF46ZX-KakLA7xZ4D3iOqRNIAhUGRAoyMDS2qyMCoq-y3Z9MKftF1WylmANqyARD9Hky4XF1cV3TtRKCfM8FU~c977M4m4~6Z20q7fSY7c50SMglTcyWZ9Sah6~0~8iRjsPqzHcmSJD3Ojm1lZQI51nt~w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944714&ar=348350&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01407.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genomic map and expression of SAg and neighbouring genes in subtelomeric regions. Genomic maps of subtelomeric regions of scaffold 106 (SAg 51A), scaffold 142 (SAg 51H), scaffold 143 (SAg 51B) and scaffold 159 (51D) indicate individual ORFs by arrows (data from <em>Paramecium</em>DB<sup><a reveal-id="DSV014C22" data-open="DSV014C22" class="link link-ref link-reveal xref-bibr">22</a></sup> and Baranasic et al.<sup><a reveal-id="DSV014C12" data-open="DSV014C12" class="link link-ref link-reveal xref-bibr">12</a></sup>). The position in the scaffold is indicated above in kbp (k). Expression level of the SAg and nearby genes (numbered by –X upstream and +X downstream relative to the SAg) are shown by TPM normalized data (logarithmic scale) in the graph below the genomic maps, respectively. Internal telomeric sites representing chromosome heterogeneity are indicated by the triangle containing the letter ‘T’; EOS, end of scaffold.</p></div></div><div content-id="DSV014F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/m_dsv01407.png?Expires=2147483647&Signature=qerBLKR~NJwCyWa9b1IuvBDyuLeABIxCfMKbhBFz~I4bDd0yZcRIN3uWzL3EnRhIncnK9IqN4~sPCJiFPK3AVu5O4sc5WWhB3OBl9c4g6TAqbsuj4RXwwLS2J3TK4ZTQ1sUlHGqGKJ8PeFHfSQm4fN9gkmG05onKDFMJRiFaOpUblN72XKfU0GIN1zCDyzvRgbX7Ex3mt15KuFgcdGyl70j3aFyPl8c1zYefrB-4yI4NNg6BgCttWr3SSD0gKVkBzDYQ~RNN6WbEMxSNP15Q4DwehY15w5IaAQvJuq6yVHfeIYLvKGP8umNYWYk0MgWn7wlBphvOZBBL5U7M74L8wA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genomic map and expression of SAg and neighbouring genes in subtelomeric regions. Genomic maps of subtelomeric regions of scaffold 106 (SAg 51A), scaffold 142 (SAg 51H), scaffold 143 (SAg 51B) and scaffold 159 (51D) indicate individual ORFs by arrows (data from ParameciumDB22 and Baranasic et al.12). The position in the scaffold is indicated above in kbp (k). Expression level of the SAg and nearby genes (numbered by –X upstream and +X downstream relative to the SAg) are shown by TPM normalized data (logarithmic scale) in the graph below the genomic maps, respectively. Internal telomeric sites representing chromosome heterogeneity are indicated by the triangle containing the letter ‘T’; EOS, end of scaffold." path-from-xml="dsv01407.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944714/dsv01407.tiff" path-from-xml="dsv01407.tiff" target="_blank">View large</a><a section="4944714" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/dsv01407.png?Expires=2147483647&Signature=sp65Uz~tBm3GdvfQ5yfJEQKz-dCt4yvVOrWdamSrVakOnTAwQ0pz2Xpv2SeKAHrZgNskbXrPyQ~O-47OZ4g0Ze7sqr~K6ydPQQEK1YLDJd233h~dVHIfCD218l4XzDLq7gK~3Z1BQ3dchrOZLzJkr0cqMySuR4l0lHa3MurLFG~KyVMb35Ebknz1~wNvXF46ZX-KakLA7xZ4D3iOqRNIAhUGRAoyMDS2qyMCoq-y3Z9MKftF1WylmANqyARD9Hky4XF1cV3TtRKCfM8FU~c977M4m4~6Z20q7fSY7c50SMglTcyWZ9Sah6~0~8iRjsPqzHcmSJD3Ojm1lZQI51nt~w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944714&ar=348350&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv01407.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genomic map and expression of SAg and neighbouring genes in subtelomeric regions. Genomic maps of subtelomeric regions of scaffold 106 (SAg 51A), scaffold 142 (SAg 51H), scaffold 143 (SAg 51B) and scaffold 159 (51D) indicate individual ORFs by arrows (data from <em>Paramecium</em>DB<sup><a reveal-id="DSV014C22" data-open="DSV014C22" class="link link-ref link-reveal xref-bibr">22</a></sup> and Baranasic et al.<sup><a reveal-id="DSV014C12" data-open="DSV014C12" class="link link-ref link-reveal xref-bibr">12</a></sup>). The position in the scaffold is indicated above in kbp (k). Expression level of the SAg and nearby genes (numbered by –X upstream and +X downstream relative to the SAg) are shown by TPM normalized data (logarithmic scale) in the graph below the genomic maps, respectively. Internal telomeric sites representing chromosome heterogeneity are indicated by the triangle containing the letter ‘T’; EOS, end of scaffold.</p></div></div>                            <h3 scrollto-destination=4944722 id="4944722" class="section-title" data-legacy-id=s3j>3.10. Conclusions</h3>
<p>Cell differentiation is mainly discussed in context of the ability of stem cells to differentiate into different tissues, i.e. an epigenetic mechanism manifests an individual gene expression pattern of the genome. Are these mechanisms for regulating and manifesting gene expression the only difference to phenotypic variation of unicellular eukaryotes?</p><p><em>Paramecium</em> was demonstrated in the past to differentiate its vegetative genome into different phenotypes with one prominent example in serotypes being regulated by epigenetically controlled heritable gene expression patterns whose regulation involves small RNA-induced chromatin regulation<sup><a reveal-id="DSV014C12" data-open="DSV014C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSV014C13" data-open="DSV014C13" class="link link-ref link-reveal xref-bibr">13</a></sup> (Cheaib et al., in preparation). Interpretation of our transcriptome resource here indicates that not only the surface antigen multigene family but also many more genes follow the expression pattern of SAgs and show persistence while transcriptome adaptation to environmental changes. This suggests that gene expression patterns of gene groups can be manifested by epigenetic mechanisms and consequently that also unicellular species can differentiate into distinct epigenetically controlled phenotypes. However, such differentiation should be interpreted as a kind of transcriptomic robustness and needs to be distinguished from regular adaptation to environmental changes representing short-time flexibility for adaptation.</p><p>It seems clear that our study can only estimate the extent of epigenetic differentiation by the above-discussed mechanisms, and the high number of genes co-regulated with the SAgs justifies future approaches to clarify the epigenetic mechanisms controlling and maintaining gene expression. Of special interest here is the heat-shock, as the data indicate that this might represent a trigger for chromatin remodelling, not only to activate individual HSP genes but for remodelling many more chromosomal loci.</p><p>Our data suggest that serotypes cannot be restricted to the activation of a single gene as we identified a certain number of co-regulated genes, thus indicating that SAg expression is representative for individual gene expression profiles. <em>Paramecium</em> serotypes can be triggered by the environment or occur spontaneously (e.g. 51B cells in 51D cultures). It seems tempting to speculate that the co-regulated genes can be interpreted as a pre-installed genome program running to achieve a complex predetermined phenotype. We therefore need to clarify the epigenetic mechanisms allowing gene silencing, activation and maintenance of expression, to understand the epigenetic network that acts to coordinate the co-regulation of so many genes. Here, small RNA pathways, as indicated for the serotype system, would be an attractive hypothesis.</p>                            <h2 scrollto-destination=4944728 id="4944728" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=4944730 id="4944730" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This study was supported by the starter grant by the Saarland University (61-cl/Anschub 2014/bew-Simon) and by grant SI 1397-2 by the Deutsche Forschungsgemeinschaft (DFG) to M.S. Funding to pay the Open Access publication charges for this article was provided by Saarland University.</p>                            <h2 scrollto-destination=4944732 id="4944732" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The entire work was carried out in the context of the CNRS supported research group ‘<em>Paramecium</em> Genome Dynamics and Evolution’ and the European COST action BM1102 Ciliates as model systems to study genome evolution, mechanisms of non-Mendelian inheritance and their roles in environmental adaptation. We thank Emanuel Staudt for assistance with serotype pure cultures, Simone Marker for correcting HSP70 gene models, Gilles Gasparoni for kind support in coordination of sequencing, and Martina Schrallhammer for critical comments to the manuscript.</p>                            <h2 scrollto-destination=4944734 id="4944734" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV014C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kelly</div>  <div class="given-names">S.A.</div></div>, <div class="name string-name"><div class="surname">Panhuis</div>  <div class="given-names">T.M.</div></div>, <div class="name string-name"><div class="surname">Stoehr</div>  <div class="given-names">A.M.</div></div></div> <div class="year">2012</div>, <div class="article-title">Phenotypic plasticity: molecular mechanisms and adaptive significance</div>, <div class="source">Compr. Physiol.</div> , <div class="volume">2</div>, <div class="fpage">1417</div>–<div class="lpage">39</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phenotypic%20plasticity%3A%20molecular%20mechanisms%20and%20adaptive%20significance&amp;author=S.A.%20Kelly&amp;author=T.M.%20Panhuis&amp;author=A.M.%20Stoehr&amp;publication_year=2012&amp;journal=Compr.%20Physiol.&amp;volume=2&amp;pages=1417-39" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23798305" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wutz</div>  <div class="given-names">A.</div></div></div> <div class="year">2013</div>, <div class="article-title">Epigenetic regulation of stem cells: the role of chromatin in cell differentiation</div>, <div class="source">Adv. Exp. Med. Biol.</div> , <div class="volume">786</div>, <div class="fpage">307</div>–<div class="lpage">28</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenetic%20regulation%20of%20stem%20cells%3A%20the%20role%20of%20chromatin%20in%20cell%20differentiation&amp;author=A.%20Wutz&amp;publication_year=2013&amp;journal=Adv.%20Exp.%20Med.%20Biol.&amp;volume=786&amp;pages=307-28" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23696364" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vabulas</div>  <div class="given-names">R.M.</div></div>, <div class="name string-name"><div class="surname">Raychaudhuri</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Hayer-Hartl</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hartl</div>  <div class="given-names">F.U.</div></div></div> <div class="year">2010</div>, <div class="article-title">Protein folding in the cytoplasm and the heat shock response</div>, <div class="source">Cold Spring Harb. Perspect. Biol.</div> , <div class="volume">2</div>, <div class="fpage">a004390</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20folding%20in%20the%20cytoplasm%20and%20the%20heat%20shock%20response&amp;author=R.M.%20Vabulas&amp;author=S.%20Raychaudhuri&amp;author=M.%20Hayer-Hartl&amp;author=F.U.%20Hartl&amp;publication_year=2010&amp;journal=Cold%20Spring%20Harb.%20Perspect.%20Biol.&amp;volume=2&amp;pages=a004390" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/cshperspect.a004390" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fcshperspect.a004390" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fcshperspect.a004390"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21123396" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fusco</div>  <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Minelli</div>  <div class="given-names">A.</div></div></div> <div class="year">2010</div>, <div class="article-title">Phenotypic plasticity in development and evolution: facts and concepts</div>, <div class="source">Philos. T. Roy. Soc. B.</div> , <div class="volume">365</div>, <div class="fpage">547</div>–<div class="lpage">56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phenotypic%20plasticity%20in%20development%20and%20evolution%3A%20facts%20and%20concepts&amp;author=G.%20Fusco&amp;author=A.%20Minelli&amp;publication_year=2010&amp;journal=Philos.%20T.%20Roy.%20Soc.%20B.&amp;volume=365&amp;pages=547-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rstb.2009.0267" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frstb.2009.0267" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frstb.2009.0267"> </span></div> </div></div></div></div></div><div content-id="DSV014C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guschina</div>  <div class="given-names">I.A.</div></div>, <div class="name string-name"><div class="surname">Harwood</div>  <div class="given-names">J.L.</div></div></div> <div class="year">2006</div>, <div class="article-title">Lipids and lipid metabolism in eukaryotic algae</div>, <div class="source">Prog. Lipid Res.</div> , <div class="volume">45</div>, <div class="fpage">160</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lipids%20and%20lipid%20metabolism%20in%20eukaryotic%20algae&amp;author=I.A.%20Guschina&amp;author=J.L.%20Harwood&amp;publication_year=2006&amp;journal=Prog.%20Lipid%20Res.&amp;volume=45&amp;pages=160-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.plipres.2006.01.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plipres.2006.01.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.plipres.2006.01.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16492482" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Budin</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Philippe</div>  <div class="given-names">H.</div></div></div> <div class="year">1998</div>, <div class="article-title">New insights into the phylogeny of eukaryotes based on ciliate Hsp70 sequences</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="volume">15</div>, <div class="fpage">943</div>–<div class="lpage">56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20insights%20into%20the%20phylogeny%20of%20eukaryotes%20based%20on%20ciliate%20Hsp70%20sequences&amp;author=K.%20Budin&amp;author=H.%20Philippe&amp;publication_year=1998&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=15&amp;pages=943-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a026010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a026010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9718722" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Krenek</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Schlegel</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Berendonk</div>  <div class="given-names">T.U.</div></div></div> <div class="year">2013</div>, <div class="article-title">Convergent evolution of heat-inducibility during subfunctionalization of the Hsp70 gene family</div>, <div class="source">BMC Evol. Biol.</div> , <div class="volume">13</div>, <div class="fpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Convergent%20evolution%20of%20heat-inducibility%20during%20subfunctionalization%20of%20the%20Hsp70%20gene%20family&amp;author=S.%20Krenek&amp;author=M.%20Schlegel&amp;author=T.U.%20Berendonk&amp;publication_year=2013&amp;journal=BMC%20Evol.%20Biol.&amp;volume=13&amp;pages=49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-13-49" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-13-49" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-13-49"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23433225" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cheaib</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Simon</div>  <div class="given-names">M.</div></div></div> <div class="year">2013</div>, <div class="article-title">Dynamic chromatin remodelling of ciliate macronuclear DNA as determined by an optimized chromatin immunoprecipitation (ChIP) method for <em>Paramecium tetraurelia</em></div>, <div class="source">Appl. Microbiol. Biotechnol.</div> , <div class="volume">97</div>, <div class="fpage">2661</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamic%20chromatin%20remodelling%20of%20ciliate%20macronuclear%20DNA%20as%20determined%20by%20an%20optimized%20chromatin%20immunoprecipitation%20(ChIP)%20method%20for%20Paramecium%20tetraurelia&amp;author=M.%20Cheaib&amp;author=M.%20Simon&amp;publication_year=2013&amp;journal=Appl.%20Microbiol.%20Biotechnol.&amp;volume=97&amp;pages=2661-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-013-4708-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-013-4708-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-013-4708-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23385475" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Singh</div>  <div class="given-names">D.P.</div></div>, <div class="name string-name"><div class="surname">Saudemont</div>  <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Guglielmi</div>  <div class="given-names">G.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Genome-defence small RNAs exapted for epigenetic mating-type inheritance</div>, <div class="source">Nature</div> , <div class="volume">509</div>, <div class="fpage">447</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-defence%20small%20RNAs%20exapted%20for%20epigenetic%20mating-type%20inheritance&amp;author=D.P.%20Singh&amp;author=B.%20Saudemont&amp;author=G.%20Guglielmi&amp;publication_year=2014&amp;journal=Nature&amp;volume=509&amp;pages=447-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature13318" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature13318" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature13318"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24805235" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Beale</div>  <div class="given-names">G.H.</div></div></div> <div class="year">1957</div>, <div class="article-title">The antigen system of <em>Paramecium aurelia</em></div>, <div class="source">Int. Rev. Cytol.</div> , <div class="volume">6</div>, <div class="fpage">1</div>–<div class="lpage">23</div>.</div></div></div></div><div content-id="DSV014C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marker</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Le Mouël</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Meyer</div>  <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Simon</div>  <div class="given-names">M.</div></div></div> <div class="year">2010</div>, <div class="article-title">Distinct RNAdependent RNA polymerases are required for RNAi triggered by doublestranded RNA versus truncated transgenes in <em>Paramecium tetraurelia</em></div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">38</div>, <div class="fpage">4092</div>–<div class="lpage">107</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distinct%20RNAdependent%20RNA%20polymerases%20are%20required%20for%20RNAi%20triggered%20by%20doublestranded%20RNA%20versus%20truncated%20transgenes%20in%20Paramecium%20tetraurelia&amp;author=S.%20Marker&amp;author=A.%20Le%20Mou%C3%ABl&amp;author=E.%20Meyer&amp;author=M.%20Simon&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=4092-107" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq131" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq131" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq131"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20200046" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Baranasic</div>  <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Oppermann</div>  <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Cheaib</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Cullum</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Schmidt</div>  <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Simon</div>  <div class="given-names">M.</div></div></div> <div class="year">2014</div>, <div class="article-title">Genomic characterization of variable surface antigens reveals a telomere position effect as a prerequisite for RNA interference-mediated silencing in <em>Paramecium tetraurelia</em></div>, <div class="source">mBio.</div> , <div class="volume">5</div>, <div class="fpage">e01328</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20characterization%20of%20variable%20surface%20antigens%20reveals%20a%20telomere%20position%20effect%20as%20a%20prerequisite%20for%20RNA%20interference-mediated%20silencing%20in%20Paramecium%20tetraurelia&amp;author=D.%20Baranasic&amp;author=T.%20Oppermann&amp;author=M.%20Cheaib&amp;author=J.%20Cullum&amp;author=H.%20Schmidt&amp;author=M.%20Simon&amp;publication_year=2014&amp;journal=mBio.&amp;volume=5&amp;pages=e01328-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/mBio.01328-14" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FmBio.01328-14" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FmBio.01328-14"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25389173" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Simon</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Plattner</div>  <div class="given-names">H.</div></div></div> <div class="year">2014</div>, <div class="article-title">Unicellular eukaryotes as models in cell and molecular biology: critical appraisal of their past and future value</div>, <div class="source">Int. Rev. Cell Mol. Biol.</div> , <div class="volume">309</div>, <div class="fpage">141</div>–<div class="lpage">98</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unicellular%20eukaryotes%20as%20models%20in%20cell%20and%20molecular%20biology%3A%20critical%20appraisal%20of%20their%20past%20and%20future%20value&amp;author=M.%20Simon&amp;author=H.%20Plattner&amp;publication_year=2014&amp;journal=Int.%20Rev.%20Cell%20Mol.%20Biol.&amp;volume=309&amp;pages=141-98" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24529723" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Beale</div>  <div class="given-names">G.H.</div></div></div> <div class="year">1954</div>, <div class="source">The genetics of <em>Paramecium Aurelia</em></div> , <div class="publisher-name">Cambridge University Press</div>: <div class="publisher-loc">New York</div>.</div></div></div></div><div content-id="DSV014C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Martin</div>  <div class="given-names">M.</div></div></div> <div class="year">2011</div>, <div class="article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads</div>, <div class="source">EMBnet J.</div> , <div class="volume">17</div>, <div class="fpage">10</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cutadapt%20removes%20adapter%20sequences%20from%20high-throughput%20sequencing%20reads&amp;author=M.%20Martin&amp;publication_year=2011&amp;journal=EMBnet%20J.&amp;volume=17&amp;pages=10-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.14806/ej.17.1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.14806%2Fej.17.1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.14806%2Fej.17.1"> </span></div> </div></div></div></div></div><div content-id="DSV014C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Punta</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Coggill</div>  <div class="given-names">P.C.</div></div>, <div class="name string-name"><div class="surname">Eberhardt</div>  <div class="given-names">R.Y.</div></div> et al.  </div>. <div class="year">2012</div>, <div class="article-title">The Pfam protein families database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">40</div>, <div class="fpage">D290</div>–<div class="lpage">301</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Pfam%20protein%20families%20database&amp;author=M.%20Punta&amp;author=P.C.%20Coggill&amp;author=R.Y.%20Eberhardt&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=D290-301" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1065" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1065" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1065"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22127870" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tamura</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Stecher</div>  <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Peterson</div>  <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Filipski</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Kumar</div>  <div class="given-names">S.</div></div></div> <div class="year">2013</div>, <div class="article-title">MEGA6: molecular evolutionary genetics analysis version 6.0.</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="volume">30</div>, <div class="fpage">2725</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA6%3A%20molecular%20evolutionary%20genetics%20analysis%20version%206.0.&amp;author=K.%20Tamura&amp;author=G.%20Stecher&amp;author=D.%20Peterson&amp;author=A.%20Filipski&amp;author=S.%20Kumar&amp;publication_year=2013&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=30&amp;pages=2725-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mst197" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmst197" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmst197"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24132122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Patro</div>  <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Mount</div>  <div class="given-names">S.M.</div></div>, <div class="name string-name"><div class="surname">Kingsford</div>  <div class="given-names">C.</div></div></div> <div class="year">2014</div>, <div class="article-title">Sailfish enables alignment-free isoform quantification from RNA-seq reads using lightweight algorithms</div>, <div class="source">Nat. Biotechnol.</div> , <div class="volume">32</div>, <div class="fpage">462</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sailfish%20enables%20alignment-free%20isoform%20quantification%20from%20RNA-seq%20reads%20using%20lightweight%20algorithms&amp;author=R.%20Patro&amp;author=S.M.%20Mount&amp;author=C.%20Kingsford&amp;publication_year=2014&amp;journal=Nat.%20Biotechnol.&amp;volume=32&amp;pages=462-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2862" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2862" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2862"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24752080" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Love</div>  <div class="given-names">M.I.</div></div>, <div class="name string-name"><div class="surname">Huber</div>  <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Anders</div>  <div class="given-names">S.</div></div></div> <div class="year">2014</div>, <div class="article-title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</div>, <div class="source">Genome Biol.</div> , <div class="volume">15</div>, <div class="fpage">550</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2&amp;author=M.I.%20Love&amp;author=W.%20Huber&amp;author=S.%20Anders&amp;publication_year=2014&amp;journal=Genome%20Biol.&amp;volume=15&amp;pages=550" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-014-0550-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-014-0550-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-014-0550-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25516281" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Benjamini</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Hochberg</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Roy</div>  <div class="given-names">J.</div></div></div> <div class="year">1995</div>, <div class="article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing</div>, <div class="source">J. R. Stat. Soc. Ser. B. (Methodological)</div> , <div class="volume">57</div>, <div class="fpage">289</div>–<div class="lpage">300</div>.</div></div></div></div><div content-id="DSV014C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wirth</div>  <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Loeffler</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Bergen</div>  <div class="given-names">M.V.</div></div>, <div class="name string-name"><div class="surname">Binder</div>  <div class="given-names">H.</div></div></div> <div class="year">2011</div>, <div class="article-title">Expression cartography of human tissues using self organizing maps</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">12</div>, <div class="fpage">306</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20cartography%20of%20human%20tissues%20using%20self%20organizing%20maps&amp;author=H.%20Wirth&amp;author=M.%20Loeffler&amp;author=M.V.%20Bergen&amp;author=H.%20Binder&amp;publication_year=2011&amp;journal=BMC%20Bioinformatics&amp;volume=12&amp;pages=306" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-12-306" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-12-306" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-12-306"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21794127" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Arnaiz</div>  <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Sperling</div>  <div class="given-names">L.</div></div></div> <div class="year">2011</div>, <div class="article-title"><em>Paramecium</em>DB in 2011: new tools and new data for functional and comparative genomics of the model ciliate <em>Paramecium tetraurelia</em></div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">39</div>, <div class="fpage">D632</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ParameciumDB%20in%202011%3A%20new%20tools%20and%20new%20data%20for%20functional%20and%20comparative%20genomics%20of%20the%20model%20ciliate%20Paramecium%20tetraurelia&amp;author=O.%20Arnaiz&amp;author=L.%20Sperling&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=D632-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq918" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq918" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq918"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20952411" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bauer</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Grossmann</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Vingron</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Robinson</div>  <div class="given-names">P.N.</div></div></div> <div class="year">2008</div>, <div class="article-title">Ontologizer 2.0–a multifunctional tool for GO term enrichment analysis and data exploration</div>, <div class="source">Bioinformatics</div> , <div class="volume">24</div>, <div class="fpage">1650</div>–<div class="lpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ontologizer%202.0%E2%80%93a%20multifunctional%20tool%20for%20GO%20term%20enrichment%20analysis%20and%20data%20exploration&amp;author=S.%20Bauer&amp;author=S.%20Grossmann&amp;author=M.%20Vingron&amp;author=P.N.%20Robinson&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=1650-1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn250" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn250" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn250"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18511468" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Aury</div>  <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Jaillon</div>  <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Duret</div>  <div class="given-names">L.</div></div> et al.  </div>. <div class="year">2006</div>, <div class="article-title">Global trends of whole genome duplications revealed by the ciliate <em>Paramecium tetraurelia</em></div>, <div class="source">Nature</div> , <div class="volume">444</div>, <div class="fpage">171</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20trends%20of%20whole%20genome%20duplications%20revealed%20by%20the%20ciliate%20Paramecium%20tetraurelia&amp;author=J.M.%20Aury&amp;author=O.%20Jaillon&amp;author=L.%20Duret&amp;publication_year=2006&amp;journal=Nature&amp;volume=444&amp;pages=171-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature05230" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature05230" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature05230"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17086204" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Supek</div>  <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Bošnjak</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Škunca</div>  <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Šmuc</div>  <div class="given-names">T.</div></div></div> <div class="year">2011</div>, <div class="article-title">REVIGO summarizes and visualizes long lists of Gene Ontology terms</div>, <div class="source">PLoS ONE</div> , <div class="volume">6</div>, <div class="fpage">e21800</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=REVIGO%20summarizes%20and%20visualizes%20long%20lists%20of%20Gene%20Ontology%20terms&amp;author=F.%20Supek&amp;author=M.%20Bo%C5%A1njak&amp;author=N.%20%C5%A0kunca&amp;author=T.%20%C5%A0muc&amp;publication_year=2011&amp;journal=PLoS%20ONE&amp;volume=6&amp;pages=e21800" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0021800" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0021800" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0021800"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21789182" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rattray</div>  <div class="given-names">A.M.J.</div></div>, <div class="name string-name"><div class="surname">Müller</div>  <div class="given-names">B.</div></div></div> <div class="year">2012</div>, <div class="article-title">The control of histone gene expression</div>, <div class="source">Biochem. Soc. Trans.</div> , <div class="volume">40</div>, <div class="fpage">880</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20control%20of%20histone%20gene%20expression&amp;author=A.M.J.%20Rattray&amp;author=B.%20M%C3%BCller&amp;publication_year=2012&amp;journal=Biochem.%20Soc.%20Trans.&amp;volume=40&amp;pages=880-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/BST20120065" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2FBST20120065" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2FBST20120065"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22817752" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hu</div>  <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Chen</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Yia</div>  <div class="given-names">Z.</div></div> et al.  </div>. <div class="year">2013</div>, <div class="article-title">Nucleosome loss leads to global transcriptional up-regulation and genomic instability during yeast aging</div>, <div class="source">Genes Dev.</div> , <div class="volume">28</div>, <div class="fpage">396</div>–<div class="lpage">408</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleosome%20loss%20leads%20to%20global%20transcriptional%20up-regulation%20and%20genomic%20instability%20during%20yeast%20aging&amp;author=Z.%20Hu&amp;author=K.%20Chen&amp;author=Z.%20Yia&amp;publication_year=2013&amp;journal=Genes%20Dev.&amp;volume=28&amp;pages=396-408" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.233221.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.233221.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.233221.113"> </span></div> </div></div></div></div></div><div content-id="DSV014C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Günesdogan</div>  <div class="given-names">U.</div></div>, <div class="name string-name"><div class="surname">Jäckle</div>  <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Herzig</div>  <div class="given-names">A.</div></div></div> <div class="year">2014</div>, <div class="article-title">Histone supply regulates S phase timing and cell cycle progression</div>, <div class="source">eLife</div> , <div class="volume">3</div>, <div class="fpage">e02443</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Histone%20supply%20regulates%20S%20phase%20timing%20and%20cell%20cycle%20progression&amp;author=U.%20G%C3%BCnesdogan&amp;author=H.%20J%C3%A4ckle&amp;author=A.%20Herzig&amp;publication_year=2014&amp;journal=eLife&amp;volume=3&amp;pages=e02443" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.7554/eLife.02443" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7554%2FeLife.02443" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.7554%2FeLife.02443"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25205668" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hemme</div>  <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Veyel</div>  <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Mühlhaus</div>  <div class="given-names">T.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Systems-wide analysis of acclimation responses to long-term heat stress and recovery in the photosynthetic model organism <em>Chlamydomonas reinhardtii</em></div>, <div class="source">Plant Cell</div> , <div class="volume">26</div>, <div class="fpage">4270</div>–<div class="lpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systems-wide%20analysis%20of%20acclimation%20responses%20to%20long-term%20heat%20stress%20and%20recovery%20in%20the%20photosynthetic%20model%20organism%20Chlamydomonas%20reinhardtii&amp;author=D.%20Hemme&amp;author=D.%20Veyel&amp;author=T.%20M%C3%BChlhaus&amp;publication_year=2014&amp;journal=Plant%20Cell&amp;volume=26&amp;pages=4270-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.114.130997" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.114.130997" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.114.130997"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25415976" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Forcob</div>  <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Bulic</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Jönsson</div>  <div class="given-names">F.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Differential expression of histone H3 genes and selective association of the variant H3.7 with a specific sequence class in <em>Stylonychia</em> macronuclear development</div>, <div class="source">Epigenetics Chromatin</div> , <div class="volume">7</div>, <div class="fpage">1</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20expression%20of%20histone%20H3%20genes%20and%20selective%20association%20of%20the%20variant%20H3.7%20with%20a%20specific%20sequence%20class%20in%20Stylonychia%20macronuclear%20development&amp;author=S.%20Forcob&amp;author=A.%20Bulic&amp;author=F.%20J%C3%B6nsson&amp;publication_year=2014&amp;journal=Epigenetics%20Chromatin&amp;volume=7&amp;pages=1-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1756-8935-7-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1756-8935-7-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1756-8935-7-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24393457" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kodama</div>  <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Suzuki</div>  <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Dohra</div>  <div class="given-names">H.</div></div> et al.  </div>. <div class="year">2014</div>, <div class="article-title">Comparison of gene expression of <em>Paramecium bursaria</em> with and without <em>Chlorella variabilis</em> symbionts</div>, <div class="source">BMC Genomics.</div> , <div class="volume">15</div>, <div class="fpage">183</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20gene%20expression%20of%20Paramecium%20bursaria%20with%20and%20without%20Chlorella%20variabilis%20symbionts&amp;author=Y.%20Kodama&amp;author=H.%20Suzuki&amp;author=H.%20Dohra&amp;publication_year=2014&amp;journal=BMC%20Genomics.&amp;volume=15&amp;pages=183" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-183" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-183" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-183"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24612690" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hori</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Fujishima</div>  <div class="given-names">M.</div></div></div> <div class="year">2003</div>, <div class="article-title">The endosymbiotic bacterium <em>Holospora obtusa</em> enhances heat-shock gene expression of the host <em>Paramecium caudatum</em></div>, <div class="source">J. Euk. Microbiol.</div> , <div class="volume">50</div>, <div class="fpage">293</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20endosymbiotic%20bacterium%20Holospora%20obtusa%20enhances%20heat-shock%20gene%20expression%20of%20the%20host%20Paramecium%20caudatum&amp;author=M.%20Hori&amp;author=M.%20Fujishima&amp;publication_year=2003&amp;journal=J.%20Euk.%20Microbiol.&amp;volume=50&amp;pages=293-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jeu.2003.50.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjeu.2003.50.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjeu.2003.50.issue-4"> </span></div> </div></div></div></div></div><div content-id="DSV014C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fujishima</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kawai</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Yamamoto</div>  <div class="given-names">R.</div></div></div> <div class="year">2005</div>, <div class="article-title"><em>Paramecium caudatum</em> acquires heat-shock resisitance in ciliare movement by infection with the endonuclear symbiotic bacterium <em>Holospora obtusa</em></div>, <div class="source">FEMS Microbiol. Lett.</div> , <div class="volume">243</div>, <div class="fpage">101</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Paramecium%20caudatum%20acquires%20heat-shock%20resisitance%20in%20ciliare%20movement%20by%20infection%20with%20the%20endonuclear%20symbiotic%20bacterium%20Holospora%20obtusa&amp;author=M.%20Fujishima&amp;author=M.%20Kawai&amp;author=R.%20Yamamoto&amp;publication_year=2005&amp;journal=FEMS%20Microbiol.%20Lett.&amp;volume=243&amp;pages=101-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.femsle.2004.11.053" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.femsle.2004.11.053" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.femsle.2004.11.053"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15668007" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hori</div>  <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Fujii</div>  <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Fujishima</div>  <div class="given-names">M.</div></div></div> <div class="year">2008</div>, <div class="article-title">Micronucleus-specific bacterium <em>Holospora elegans</em> irreversibly enhances stress gene expression of the host <em>Paramecium caudatum</em></div>, <div class="source">J. Euk. Microbiol.</div> , <div class="volume">55</div>, <div class="fpage">515</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Micronucleus-specific%20bacterium%20Holospora%20elegans%20irreversibly%20enhances%20stress%20gene%20expression%20of%20the%20host%20Paramecium%20caudatum&amp;author=M.%20Hori&amp;author=K.%20Fujii&amp;author=M.%20Fujishima&amp;publication_year=2008&amp;journal=J.%20Euk.%20Microbiol.&amp;volume=55&amp;pages=515-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jeu.2008.55.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjeu.2008.55.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjeu.2008.55.issue-6"> </span></div> </div></div></div></div></div><div content-id="DSV014C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Simon</div>  <div class="given-names">M.C.</div></div>, <div class="name string-name"><div class="surname">Schmidt</div>  <div class="given-names">H.J.</div></div></div> <div class="year">2007</div>, <div class="article-title">Antigenic variation in ciliates: antigen structure, function, expression</div>, <div class="source">J. Eukaryot. Microbiol.</div> , <div class="volume">54</div>, <div class="fpage">1</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Antigenic%20variation%20in%20ciliates%3A%20antigen%20structure%2C%20function%2C%20expression&amp;author=M.C.%20Simon&amp;author=H.J.%20Schmidt&amp;publication_year=2007&amp;journal=J.%20Eukaryot.%20Microbiol.&amp;volume=54&amp;pages=1-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jeu.2007.54.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjeu.2007.54.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjeu.2007.54.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17300509" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Godiska</div>  <div class="given-names">R.</div></div></div> <div class="year">1987</div>, <div class="article-title">Structure and sequence of the H surface protein gene of <em>Paramecium</em> and comparison with related genes</div>, <div class="source">Mol. Gen. Genet.</div> , <div class="volume">208</div>, <div class="fpage">529</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20and%20sequence%20of%20the%20H%20surface%20protein%20gene%20of%20Paramecium%20and%20comparison%20with%20related%20genes&amp;author=R.%20Godiska&amp;publication_year=1987&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=208&amp;pages=529-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00328151" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00328151" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00328151"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3478550" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Love</div>  <div class="given-names">H.D.</div>Jr</div>, <div class="name string-name"><div class="surname">Allen-Nash</div>  <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Zhao</div>  <div class="given-names">Q.A.</div></div>, <div class="name string-name"><div class="surname">Bannon</div>  <div class="given-names">G.A.</div></div></div> <div class="year">1988</div>, <div class="article-title">mRNA stability plays a major role in regulating the temperature-specific expression of a <em>Tetrahymena thermophila</em> surface protein</div>, <div class="source">Mol. Cell. Biol.</div> , <div class="volume">8</div>, <div class="fpage">427</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=mRNA%20stability%20plays%20a%20major%20role%20in%20regulating%20the%20temperature-specific%20expression%20of%20a%20Tetrahymena%20thermophila%20surface%20protein&amp;author=H.D.%20Love&amp;author=A.%20Allen-Nash&amp;author=Q.A.%20Zhao&amp;author=G.A.%20Bannon&amp;publication_year=1988&amp;journal=Mol.%20Cell.%20Biol.&amp;volume=8&amp;pages=427-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.8.1.427" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.8.1.427" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.8.1.427"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3336362" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">McMillan</div>  <div class="given-names">P.J.</div></div>, <div class="name string-name"><div class="surname">Stanley</div>  <div class="given-names">J.S.</div></div>, <div class="name string-name"><div class="surname">Bannon</div>  <div class="given-names">G.A.</div></div></div> <div class="year">1995</div>, <div class="article-title">Evidence for the requirement of protein synthesis and protein kinase activity in the temperature regulated stability of a <em>Tetrahymena</em> surface protein mRNA</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">23</div>, <div class="fpage">942</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20the%20requirement%20of%20protein%20synthesis%20and%20protein%20kinase%20activity%20in%20the%20temperature%20regulated%20stability%20of%20a%20Tetrahymena%20surface%20protein%20mRNA&amp;author=P.J.%20McMillan&amp;author=J.S.%20Stanley&amp;author=G.A.%20Bannon&amp;publication_year=1995&amp;journal=Nucleic%20Acids%20Res.&amp;volume=23&amp;pages=942-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/23.6.942" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F23.6.942" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F23.6.942"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7731807" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Merrick</div>  <div class="given-names">C.J.</div></div>, <div class="name string-name"><div class="surname">Duraisingh</div>  <div class="given-names">M.T.</div></div></div> <div class="year">2006</div>, <div class="article-title">Heterochromatin-mediated control of virulence gene expression</div>, <div class="source">Mol. Microbiol.</div> , <div class="volume">62</div>, <div class="fpage">612</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Heterochromatin-mediated%20control%20of%20virulence%20gene%20expression&amp;author=C.J.%20Merrick&amp;author=M.T.%20Duraisingh&amp;publication_year=2006&amp;journal=Mol.%20Microbiol.&amp;volume=62&amp;pages=612-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.2006.62.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.2006.62.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.2006.62.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17076663" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV014C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV014C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Duret</div>  <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Cohen</div>  <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Jubin</div>  <div class="given-names">C.</div></div> et al.  </div>. <div class="year">2008</div>, <div class="article-title">Analysis of sequence variability in the macronuclear DNA of <em>Paramecium tetraurelia</em>: a somatic view of the germline</div>, <div class="source">Genome Res.</div> , <div class="volume">18</div>, <div class="fpage">585</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20sequence%20variability%20in%20the%20macronuclear%20DNA%20of%20Paramecium%20tetraurelia%3A%20a%20somatic%20view%20of%20the%20germline&amp;author=L.%20Duret&amp;author=J.%20Cohen&amp;author=C.%20Jubin&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=585-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.074534.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.074534.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.074534.107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18256234" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=348350&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/dsv014-dsv014supp_fig5.pdf?Expires=2147483647&amp;Signature=XDbQWyHxsDNNGgR7lbzz46LTGDGeQTBTkljK1cmSOf-Uk31nCyqVZc04hz0lqTxp4Jv476mvgvphoXK2x5XbmdbQJlE7eCLqwQh9YQPaqsW8~rVivF2XtLfu2jPivaxldrgS2w7CYNU3vraR0W1~v4v9p9751gKwFnN3G~OPeLDjUByBPB4Rs~fXJcVdYTImdVQTFHv6NwJjxRxu0FTa23ySU6meCxv~EKQesEkVX5kHLseaLiJAEcVZySYnTavaxagTs~yPNRYB3-gcrt~r6nSx9WW72iZLtTUd2uGCUJu~OgFWy4uIM3yxuHb9cLxZdLYn5rFl~0fRr0HJdTEKeQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/dsv014-dsv014supp_fig4.ppt?Expires=2147483647&amp;Signature=ALZpFmaA6Wd3-~zh5p9Pc8SGd1qF5KwjiWeDRKCL6HAMDWHXLmmdO755zJIhCRDfiNm0JHMSdocjaXI9TdnB~eISd5U2U7jlgvinvd3pdvMuz-hmZRTsbsfUiEsMgVllV36EREQweOBTbkXa-xwP~xj1yNVP7SzrwM6gkA~kx~dPX0xwYJk2kKiSLWSaiP86Mgf~Jo7iQRCs~ZJgK0EwR~D9bJCxI9CXfR1zOoZWLE0aIodRjfpU0KQQhgASGllE5tq~qKs99z2KSFY7okMF4Q2IsBiB4POcEGcdp4nbh48pSUqZr8fPrWQlcmK0rTNGr-n~i~DJX5rblkIP3jwSPg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/dsv014-dsv014supp_fig3.ppt?Expires=2147483647&amp;Signature=g7gJqQkI99IdRtOO1svi5tHWYO~OXPDPe0Fk8cxZEEZJrATJ2NWG5w0~vdzixVFyRs4Lpia5j5--3aZG~RIeC2IsPvy-LHf8hUDyijukum3G22JXThXxiGteGNw2gCURxAr-JDwUyZKKI5-93F~e9O0Fh8lRRgAOzzSP50Ta5LSNexY2tIj11J3MdoTYqTqppQPS4cq-P6XqE0rr-Wd-gN5eAiosTNwIYLgt9NZYTzr1tkWXxqXfexy84QhBprxroAjWyAPO-XSxGf0AC9AkPPKJkq~u2XJYseEKPSH~5dHsAIMTTOfGzgb7RJqbeDrt1hhkFWK30HnUiVnCgyLbOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/dsv014-dsv014supp_fig2.pdf?Expires=2147483647&amp;Signature=VQ-8vhtwni13spVEFeItrn0sDV0e7EZu6LBIBCqzEezq3Wf4E9bDGiuBsyEJwW5EB8cgGzlg1k-v-2wWp1ADuEVf4HCDu98DJsdGPps0xxcCJuZl4RRjCXquyIYeXjlEBip2Wi46DOGECoN-Y~ykmEA1vJR7yGucE5-2pYt8h684QgIkX5rkMYCNsrm2XTVSofmZDfPA4yvqhdlb5x8FlOPk3faHUr~e5~fs45dNVmlJufR30KYXkQt9FVs8PmROR6TLlisQHi6X3j9Bg2wwfFC22m5MKie1XdPjqt1ryxlxuqDFc2vmVhMcWDimOlHt8w32ccJFz6ie2fkxLbOyRQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/dsv014-dsv014supp_fig1.ppt?Expires=2147483647&amp;Signature=b3fPoWpEe4Q9KaX1hZffg0x23zcZlqTfCrVJHE9e1ur9z2wIcvg8looygpmzlHc8CZjS78nrEVJP5mlj6WvKdFtSTCVWD4sipvERykEJLJHTJTY3WvsRY58VlP4nelr3zklU2HHnUhTHBD9pYvUJufNSftUDFYVlR3QSXwRUFFhPIekzw8IhpfDDJWXsObYmCsxlV2MAradq4OTL16APAirgeLXOHX-QDqXQhqqvX-kkRG5N16W27P0FNppWJtJuIZPHjXYGkDxUoMEvhJm-NOtdHXrpCudr4nPVQiDoo1ZkNOeEosXi2v3TKLdgKfPNXM1lYx27hSPJXMSlV9ZsIA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/dsv014-dsv014supp_data.txt?Expires=2147483647&amp;Signature=Ea6H~TO07X6-WT1mQc7BSkG8szw3xfDfga6SBev9p-2-UFvhlJLD9fHSXIEITRkkIHwrR~InOftwDJnJIGuFYKtOE-08n5plF0WaUK08za-opByUWZeOPOZUx5EmsN811-QvKRvJTz4F9KB2uO6wmlts27P~DrW-jovJUbOUMQXnVp1MNjARYS0Ys50MxA1N9Y4-yFlAAKCc1ZLXlhHmweERTWuOrf402U9-K-h24b2cdaBFV9v4BhlKr5oozq3ijQLQqiJgziT2Xqb4StHNWo7~c9I0veTJlFMbGXxjL4drqShVWKnYFcIgAhVrsLj~fWd94QKVQRDFdPyODAFWdA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - txt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/4/10.1093/dnares/dsv014/2/dsv014-dsv014supp.docx?Expires=2147483647&amp;Signature=PjFfjL8d9Wba72hhPWhKuNyS5B1VZX4ERzl72inQ6N13fgCYADBcFWb0FxND9XQc8NYq7Om8CHC-6-231g56nijY~-uVBhbm1zes8uh38VNdDvCwCc4T4-kmt6O3lktplNHVz0yJnn07anM5HRATzvIq3-3JVj77AiCbGOfDUOPPNP~gAkP5K3O~xO5M9fVQ7i-oGvmcyI6fghN7-ssM6rUjRQBK4HQfWM4-dRdcTThb-IGy5Gf~NfHdDEupyH-zml9YorXzs1UacuNhsthoQxinznTmkRzUWBr5V4LfNWX7ro-wNJcwWz4aIBbTrmMctBC7-yw~iVB8-Yqkahj-qw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - docx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">421</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000362346000005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">6</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv014"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="348350"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000362346000005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f4%2f293%2f348350">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000362346000005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (6)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F4%2F293%2F348350" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/348350'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Epigenetic regulation of serotype expression antagonizes transcriptome dynamics in Paramecium tetraurelia" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv014");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f4%2f293%2f348350">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

