

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/4/429/447212","@type":"ScholarlyArticle","name":" In Silico  Identification and Experimental Validation of Insertion–Deletion Polymorphisms in Tomato Genome","datePublished":"2014-03-11","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2014-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu008","keywords":["insertion–deletion","<italic>Solanum lycopersicum</italic>","genetic variation"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/4/429/447212","author":[{"name":"Yang, Jingjing","affiliation":"Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, Department of Vegetable Science ,  China Agricultural University ,  No. 2 Yuanmingyuan Xilu, Beijing 100193 ,  China","@type":"Person"},{"name":"Wang, Yuanyuan","affiliation":"Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, Department of Vegetable Science ,  China Agricultural University ,  No. 2 Yuanmingyuan Xilu, Beijing 100193 ,  China","@type":"Person"},{"name":"Shen, Huolin","affiliation":"Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, Department of Vegetable Science ,  China Agricultural University ,  No. 2 Yuanmingyuan Xilu, Beijing 100193 ,  China","@type":"Person"},{"name":"Yang, Wencai","affiliation":"Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, Department of Vegetable Science ,  China Agricultural University ,  No. 2 Yuanmingyuan Xilu, Beijing 100193 ,  China","@type":"Person"}],"description":"Abstract.  Comparative analysis of the genome sequences of Solanum lycopersicum variety Heinz 1706 and S. pimpinellifolium accession LA 1589 using MUGSY softwar","pageStart":"429","pageEnd":"438","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/4/1/m_cover.gif?Expires=2147483647&Signature=u1K0F1kMa-64NqKa4J2VQTk-ZgDNBNdy7xP9K68v94JN5WFnh6Of~dA8F~vuNuY-DWKv8sgKc6Tx8dCh~u-HxATm4iXS709rS-0HkL4VrV-yHYOvTEJMgBGMYLmkCZLHLnqJ4CE8~N9vWk3MlqGsd6CiX6Nq4~C6vim6BhKuTZhFWadSb~4gUcfc5RIhx0e6adpzN-8ic5w6tmizeNhHnsbot9fC0b5HPTtgY4T0sWivhNwSa56rOcmWTxrX6Ct6aeNXnMXGfCSv4nXY9sUZJB1uoWiHPNj1cxffLQO70T71y1MESr63u0ogFEZ8T3x1cUhck9QSbq-jL~iCjW~OcQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":" In Silico  Identification and Experimental Validation of Insertion–Deletion Polymorphisms in Tomato Genome","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/4/1/m_cover.gif?Expires=2147483647&Signature=u1K0F1kMa-64NqKa4J2VQTk-ZgDNBNdy7xP9K68v94JN5WFnh6Of~dA8F~vuNuY-DWKv8sgKc6Tx8dCh~u-HxATm4iXS709rS-0HkL4VrV-yHYOvTEJMgBGMYLmkCZLHLnqJ4CE8~N9vWk3MlqGsd6CiX6Nq4~C6vim6BhKuTZhFWadSb~4gUcfc5RIhx0e6adpzN-8ic5w6tmizeNhHnsbot9fC0b5HPTtgY4T0sWivhNwSa56rOcmWTxrX6Ct6aeNXnMXGfCSv4nXY9sUZJB1uoWiHPNj1cxffLQO70T71y1MESr63u0ogFEZ8T3x1cUhck9QSbq-jL~iCjW~OcQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content=" In Silico  Identification and Experimental Validation of Insertion–Deletion Polymorphisms in Tomato Genome" />
    <meta property="og:description" content="Abstract.  Comparative analysis of the genome sequences of Solanum lycopersicum variety Heinz 1706 and S. pimpinellifolium accession LA 1589 using MUGSY softwar" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/4/429/447212" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2014-03-11" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Yang, Jingjing" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, Department of Vegetable Science, China Agricultural University, No. 2 Yuanmingyuan Xilu, Beijing 100193, China" /><meta name="citation_author" content="Wang, Yuanyuan" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, Department of Vegetable Science, China Agricultural University, No. 2 Yuanmingyuan Xilu, Beijing 100193, China" /><meta name="citation_author" content="Shen, Huolin" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, Department of Vegetable Science, China Agricultural University, No. 2 Yuanmingyuan Xilu, Beijing 100193, China" /><meta name="citation_author" content="Yang, Wencai" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, Department of Vegetable Science, China Agricultural University, No. 2 Yuanmingyuan Xilu, Beijing 100193, China" /><meta name="citation_title" content="In Silico Identification and Experimental Validation of Insertion–Deletion Polymorphisms in Tomato Genome" /><meta name="citation_firstpage" content="429" /><meta name="citation_lastpage" content="438" /><meta name="citation_doi" content="10.1093/dnares/dsu008" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2014/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Marker-assisted selection in tomato breeding; Crit. Rev. Plant Sci.;  citation_year=2012;  citation_volume=31;  citation_pages=93-123; " /><meta name="citation_reference" content="citation_title=DNA marker applications to molecular genetics and genomics in tomato; Breeding Sci.;  citation_year=2013;  citation_volume=63;  citation_pages=21-30; " /><meta name="citation_reference" content="citation_title=RFLP analysis of phylogenetic relationships and genetic variation in the genus Lycopersicon; Theor. Appl. Genet.;  citation_year=1990;  citation_volume=80;  citation_pages=437-48; " /><meta name="citation_reference" content="citation_title=Discovery of single nucleotide polymorphisms in Lycopersicon esculentum by computer aided analysis of expressed sequence tags; Mol. Breeding;  citation_year=2004;  citation_volume=14;  citation_pages=21-34; " /><meta name="citation_reference" content="citation_title=Tomato SNP discovery by EST mining and resequencing; Mol. Breeding;  citation_year=2005;  citation_volume=16;  citation_pages=343-9; " /><meta name="citation_reference" content="citation_title=Expressed sequence tags from the laboratory-grown miniature tomato (Lycopersicon esculentum) cultivar Micro-Tom and mining for single nucleotide polymorphisms and insertions/deletions in tomato cultivars; Gene;  citation_year=2005;  citation_volume=56;  citation_pages=127-34; " /><meta name="citation_reference" content="citation_title=SNP discovery and linkage map construction in cultivated tomato; DNA Res.;  citation_year=2010;  citation_volume=17;  citation_pages=381-91; " /><meta name="citation_reference" content="citation_title=Diversity in conserved genes in tomato; BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=465" /><meta name="citation_reference" content="citation_title=Discovery of intron polymorphisms in cultivated tomato using both tomato and Arabidopsis genomic information; Theor. Appl. Genet.;  citation_year=2010;  citation_volume=121;  citation_pages=1199-207; " /><meta name="citation_reference" content="citation_title=Oligonucleotide array discovery of polymorphisms in cultivated tomato (Solanum lycopersicum L.) reveals patterns of SNP variation associated with breeding; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=466" /><meta name="citation_reference" content="citation_title=Single nucleotide polymorphism discovery in cultivated tomato via sequencing by synthesis; Plant Genome;  citation_year=2012;  citation_volume=5;  citation_pages=17-29; " /><meta name="citation_reference" content="citation_title=Tomato breeding in the genomics era: insights from a SNP array; BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=354" /><meta name="citation_reference" content="citation_title=Genome-wide association studies using single nucleotide polymorphism markers developed by re-sequencing of the genomes of cultivated tomato; DNA Res.;  citation_year=2013;  citation_volume=20;  citation_pages=593-603; " /><meta name="citation_reference" content="citation_title=Mapping and linkage disequilibrium analysis with a genome-wide collection of SNPs that detect polymorphism in cultivated tomato; J. Exp. Bot.;  citation_year=2011;  citation_volume=62;  citation_pages=1831-45; " /><meta name="citation_reference" content="citation_title=Genome-wide SNP genotyping to infer the effects on gene functions in tomato; DNA Res.;  citation_year=2013;  citation_volume=20;  citation_pages=221-33; " /><meta name="citation_reference" content="citation_title=Development of a large SNP genotyping array and generation of high-density genetic maps in tomato; PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e40563" /><meta name="citation_reference" content="citation_title=Genetic variation in tomato populations from four breeding programs revealed by single nucleotide polymorphism and simple sequence repeat markers; Sci. Hortic.;  citation_year=2009;  citation_volume=122;  citation_pages=6-16; " /><meta name="citation_reference" content="citation_title=Population structure and genetic differentiation associated with breeding history and selection in tomato (Solanum lycopersicum L.); Heredity;  citation_year=2011;  citation_volume=106;  citation_pages=927-35; " /><meta name="citation_reference" content="citation_title=High-density SNP genotyping of tomato (Solanum lycopersicum L.) reveals patterns of genetic variation due to breeding; PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e45520" /><meta name="citation_reference" content="citation_title=SNP genotyping reveals genetic diversity between cultivated landraces and contemporary varieties of tomato; BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=835" /><meta name="citation_reference" content="citation_title=Construction of a general human chromosome jumping library, with application to cystic fibrosis; Science;  citation_year=1987;  citation_volume=235;  citation_pages=1046-9; " /><meta name="citation_reference" content="citation_title=The biological effects of simple tandem repeats: lessons from the repeat expansion diseases; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=1011-9; " /><meta name="citation_reference" content="citation_title=Loss-of-function variants in the genomes of healthy humans; Hum. Mol. Genet.;  citation_year=2010;  citation_volume=19;  citation_pages=R125-130; " /><meta name="citation_reference" content="citation_title=The Human Gene Mutation Database: providing a comprehensive central mutation database for molecular diagnostics and personalized genomics; Hum. Genomics;  citation_year=2009;  citation_volume=4;  citation_pages=69-72; " /><meta name="citation_reference" content="citation_title=The origin, evolution, and functional impact of short insertion-deletion variants identified in 179 human genomes; Genome Res.;  citation_year=2013;  citation_volume=23;  citation_pages=749-61; " /><meta name="citation_reference" content="citation_title=An initial map of insertion and deletion (INDEL) variation in the human genome; Genome Res.;  citation_year=2006;  citation_volume=16;  citation_pages=1182-90; " /><meta name="citation_reference" content="citation_title=Small insertions and deletions (INDELs) in human genomes; Hum. Mol. Genet.;  citation_year=2010;  citation_volume=19;  citation_pages=R131-136; " /><meta name="citation_reference" content="citation_title=Building a genome analysis pipeline to predict disease risk and prevent disease; J. Mol. Biol.;  citation_year=2013;  citation_volume=425;  citation_pages=3993-4005; " /><meta name="citation_reference" content="citation_title=A platform of high-density INDEL/CAPS markers for map-based cloning in Arabidopsis; Plant J.;  citation_year=2010;  citation_volume=63;  citation_pages=880-8; " /><meta name="citation_reference" content="citation_title=A collection of INDEL markers for map-based cloning in seven Arabidopsis accessions; J. Exp. Bot.;  citation_year=2012;  citation_volume=63;  citation_pages=2491-501; " /><meta name="citation_reference" content="citation_title=Development of genome-wide DNA polymorphism database for map-based cloning of rice genes; Plant Physiol.;  citation_year=2004;  citation_volume=135;  citation_pages=1198-205; " /><meta name="citation_reference" content="citation_title=Development of 1047 insertion-deletion markers for rice genetic studies and breeding; Genet. Mol. Res.;  citation_year=2012;  citation_volume=12;  citation_pages=5226-35; " /><meta name="citation_reference" content="citation_title=Genome-wide InDel marker system for application in rice breeding and mapping studies; Euphytica;  citation_year=2013;  citation_volume=192;  citation_pages=131-43; " /><meta name="citation_reference" content="citation_title=The tomato genome sequence provides insights into fleshy fruit evolution; Nature;  citation_year=2012;  citation_volume=485;  citation_pages=635-41; " /><meta name="citation_reference" content="citation_title=Comparative transcriptomics reveals patterns of selection in domesticated and wild tomato; Proc. Natl. Acad. Sci. USA;  citation_year=2013;  citation_volume=110;  citation_pages=E2655-62; " /><meta name="citation_reference" content="citation_title=Two loci from Lycopersicon hirsutum LA407 confer resistance to strains of Clavibacter michiganensis subsp. Michiganensis; Phytopathology;  citation_year=2002;  citation_volume=92;  citation_pages=504-10; " /><meta name="citation_reference" content="citation_title=Mugsy: fast multiple alignment of closely related whole genomes; Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=334-42; " /><meta name="citation_reference" content="citation_title=Primer3 on the WWW for general users and for biologist programmers; Methods Mol. Biol.;  citation_year=2000;  citation_volume=132;  citation_pages=365-86; " /><meta name="citation_reference" content="citation_title=Identification of single-copy orthologous genes between Physalis and Solanum lycopersicum and analysis of genetic diversity in Physalis using molecular markers; PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e50164" /><meta name="citation_reference" content="citation_title=Genetic distance between populations; Am. Nat.;  citation_year=1972;  citation_volume=106;  citation_pages=283-92; " /><meta name="citation_reference" content="citation_title=Molecular breeding in developing countries: challenges and perspectives; Curr. Opin. Plant Biol.;  citation_year=2010;  citation_volume=13;  citation_pages=1-6; " /><meta name="citation_reference" content="citation_title=Plant marker-assisted breeding and conventional breeding: challenges and perspectives; Adv. Crop Sci. Technol.;  citation_year=2013;  citation_volume=1;  citation_pages=e106" /><meta name="citation_reference" content="citation_title=Human diallelic insertion/deletion polymorphisms; Am. J. Hum. Genet.;  citation_year=2002;  citation_volume=71;  citation_pages=854-62; " /><meta name="citation_reference" content="citation_title=Genotyping of 38 insertion/deletion polymorphisms for human identification using universal fluorescent PCR; Mol. Cell. Probes;  citation_year=2014;  citation_volume=28;  citation_pages=13-8; " /><meta name="citation_reference" content="citation_title=A new multiplex for human identification using insertion/deletion polymorphisms; Electrophoresis;  citation_year=2009;  citation_volume=30;  citation_pages=3682-90; " /><meta name="citation_reference" content="citation_title=Genetic analysis of 30 InDel markers for forensic use in five different Chinese populations; Genet. Mol. Res.;  citation_year=2011;  citation_volume=10;  citation_pages=964-79; " /><meta name="citation_reference" content="citation_title=Genetic polymorphism of 29 highly informative InDel markers for forensic use in the Chinese Han population; Forensic Sci. Int. Genet.;  citation_year=2011;  citation_volume=5;  citation_pages=e27-30; " /><meta name="citation_reference" content="citation_title=Forensic performance of two insertion-deletion marker assays; Int. J. Legal Med.;  citation_year=2012;  citation_volume=126;  citation_pages=725-37; " /><meta name="citation_reference" content="citation_title=Advances in tomato research in the post-genome era; Plant Biotechnol.;  citation_year=2013;  citation_volume=30;  citation_pages=243-56; " /><meta name="citation_reference" content="citation_title=Next-generation DNA sequencing; Nat. Biotechnol.;  citation_year=2008;  citation_volume=26;  citation_pages=1135-45; " /><meta name="citation_reference" content="citation_title=Whole genome resequencing in tomato reveals variation associated with introgression and breeding events; BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=791" /><meta name="citation_reference" content="citation_title=Domestication and breeding of tomatoes: what have we gained and what can we gain in the future?; Ann. Bot.;  citation_year=2007;  citation_volume=100;  citation_pages=1085-94; " /><meta name="citation_reference" content="citation_title=RAPD markers reveal narrowing genetic base of Indian tomato cultivars; Curr. Sci.;  citation_year=2002;  citation_volume=82;  citation_pages=1139-43; " /><meta name="citation_reference" content="citation_title=Genetic diversity of Argentina tomato varieties revealed by morphological traits, simple sequence repeat, and single nucleotide polymorphism markers; Pak. J. Bot.;  citation_year=2012;  citation_volume=44;  citation_pages=485-92; " /><meta name="citation_reference" content="citation_title=Allozyme variation in the cultivated tomato and closely related species; Bull. Torr. Bot. Club;  citation_year=1975;  citation_volume=102;  citation_pages=376-84; " /><meta name="citation_reference" content="citation_title=A clarified position for Solanum lycopersicum var. cerasiforme in the evolutionary history of tomatoes (solanaceae); BMC Plant Biol.;  citation_year=2008;  citation_volume=8;  citation_pages=130" /><meta name="citation_reference" content="citation_title=SIFT Indel: predictions for the functional effects of amino acid insertions/deletions in proteins; PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e77940" /><meta name="citation_reference" content="citation_title=A new family of Ty1-copia-like retrotransposons originated in the tomato genome by a recent horizontal transfer event; Genetics;  citation_year=2009;  citation_volume=181;  citation_pages=1183-93; " /><meta name="citation_reference" content="citation_title=A single-base deletion mutation in SlIAA9 gene causes tomato (Solanum lycopersicum) entire mutant; J. Plant Res.;  citation_year=2007;  citation_volume=120;  citation_pages=671-8; " /><meta name="citation_reference" content="citation_title=A-308 deletion of the tomato LAP promoters is able to direct flower-specific and MeJA-induced expression in transgenic plants; Plant Mol. Biol.;  citation_year=1998;  citation_volume=36;  citation_pages=639-48; " /><meta name="citation_reference" content="citation_title=Identification and analysis of resistance-like genes in the tomato genome; J. Phytopathol.;  citation_year=2013;  citation_volume=162;  citation_pages=137-46; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/4/429/1222482/dsu008.pdf" /><meta name="description" content="Abstract.  Comparative analysis of the genome sequences of Solanum lycopersicum variety Heinz 1706 and S. pimpinellifolium accession LA 1589 using MUGSY softwar" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/4/429/447212" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/4/429/447212" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/4/429/447212");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsu008';
    </script>
<title>
	In Silico Identification and Experimental Validation of Insertion–Deletion Polymorphisms in Tomato Genome | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./447212" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BfWBATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFuZywgSmluZ2ppbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCZWlqaW5nIEtleSBMYWJvcmF0b3J5IG9mIEdyb3d0aCBhbmQgRGV2ZWxvcG1lbnRhbCBSZWd1bGF0aW9uIGZvciBQcm90ZWN0ZWQgVmVnZXRhYmxlIENyb3BzLCBEZXBhcnRtZW50IG9mIFZlZ2V0YWJsZSBTY2llbmNlLCBDaGluYSBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgTm8uIDIgWXVhbm1pbmd5dWFuIFhpbHUsIEJlaWppbmcgMTAwMTkzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IldhbmcsIFl1YW55dWFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmVpamluZyBLZXkgTGFib3JhdG9yeSBvZiBHcm93dGggYW5kIERldmVsb3BtZW50YWwgUmVndWxhdGlvbiBmb3IgUHJvdGVjdGVkIFZlZ2V0YWJsZSBDcm9wcywgRGVwYXJ0bWVudCBvZiBWZWdldGFibGUgU2NpZW5jZSwgQ2hpbmEgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIE5vLiAyIFl1YW5taW5neXVhbiBYaWx1LCBCZWlqaW5nIDEwMDE5MywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGVuLCBIdW9saW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCZWlqaW5nIEtleSBMYWJvcmF0b3J5IG9mIEdyb3d0aCBhbmQgRGV2ZWxvcG1lbnRhbCBSZWd1bGF0aW9uIGZvciBQcm90ZWN0ZWQgVmVnZXRhYmxlIENyb3BzLCBEZXBhcnRtZW50IG9mIFZlZ2V0YWJsZSBTY2llbmNlLCBDaGluYSBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgTm8uIDIgWXVhbm1pbmd5dWFuIFhpbHUsIEJlaWppbmcgMTAwMTkzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IllhbmcsIFdlbmNhaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJlaWppbmcgS2V5IExhYm9yYXRvcnkgb2YgR3Jvd3RoIGFuZCBEZXZlbG9wbWVudGFsIFJlZ3VsYXRpb24gZm9yIFByb3RlY3RlZCBWZWdldGFibGUgQ3JvcHMsIERlcGFydG1lbnQgb2YgVmVnZXRhYmxlIFNjaWVuY2UsIENoaW5hIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBOby4gMiBZdWFubWluZ3l1YW4gWGlsdSwgQmVpamluZyAxMDAxOTMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJJbiBTaWxpY28gSWRlbnRpZmljYXRpb24gYW5kIEV4cGVyaW1lbnRhbCBWYWxpZGF0aW9uIG9mIEluc2VydGlvbuKAk0RlbGV0aW9uIFBvbHltb3JwaGlzbXMgaW4gVG9tYXRvIEdlbm9tZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjQyOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNDM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN1MDA4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTQvMDgvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXJrZXItYXNzaXN0ZWQgc2VsZWN0aW9uIGluIHRvbWF0byBicmVlZGluZzsgQ3JpdC4gUmV2LiBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9OTMtMTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgbWFya2VyIGFwcGxpY2F0aW9ucyB0byBtb2xlY3VsYXIgZ2VuZXRpY3MgYW5kIGdlbm9taWNzIGluIHRvbWF0bzsgQnJlZWRpbmcgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT02MzsgIGNpdGF0aW9uX3BhZ2VzPTIxLTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SRkxQIGFuYWx5c2lzIG9mIHBoeWxvZ2VuZXRpYyByZWxhdGlvbnNoaXBzIGFuZCBnZW5ldGljIHZhcmlhdGlvbiBpbiB0aGUgZ2VudXMgTHljb3BlcnNpY29uOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTgwOyAgY2l0YXRpb25fcGFnZXM9NDM3LTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXNjb3Zlcnkgb2Ygc2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtcyBpbiBMeWNvcGVyc2ljb24gZXNjdWxlbnR1bSBieSBjb21wdXRlciBhaWRlZCBhbmFseXNpcyBvZiBleHByZXNzZWQgc2VxdWVuY2UgdGFnczsgTW9sLiBCcmVlZGluZzsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTIxLTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ub21hdG8gU05QIGRpc2NvdmVyeSBieSBFU1QgbWluaW5nIGFuZCByZXNlcXVlbmNpbmc7IE1vbC4gQnJlZWRpbmc7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0zNDMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwcmVzc2VkIHNlcXVlbmNlIHRhZ3MgZnJvbSB0aGUgbGFib3JhdG9yeS1ncm93biBtaW5pYXR1cmUgdG9tYXRvIChMeWNvcGVyc2ljb24gZXNjdWxlbnR1bSkgY3VsdGl2YXIgTWljcm8tVG9tIGFuZCBtaW5pbmcgZm9yIHNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbXMgYW5kIGluc2VydGlvbnMvZGVsZXRpb25zIGluIHRvbWF0byBjdWx0aXZhcnM7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NTY7ICBjaXRhdGlvbl9wYWdlcz0xMjctMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNOUCBkaXNjb3ZlcnkgYW5kIGxpbmthZ2UgbWFwIGNvbnN0cnVjdGlvbiBpbiBjdWx0aXZhdGVkIHRvbWF0bzsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0zODEtOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpdmVyc2l0eSBpbiBjb25zZXJ2ZWQgZ2VuZXMgaW4gdG9tYXRvOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTQ2NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc2NvdmVyeSBvZiBpbnRyb24gcG9seW1vcnBoaXNtcyBpbiBjdWx0aXZhdGVkIHRvbWF0byB1c2luZyBib3RoIHRvbWF0byBhbmQgQXJhYmlkb3BzaXMgZ2Vub21pYyBpbmZvcm1hdGlvbjsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjE7ICBjaXRhdGlvbl9wYWdlcz0xMTk5LTIwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T2xpZ29udWNsZW90aWRlIGFycmF5IGRpc2NvdmVyeSBvZiBwb2x5bW9ycGhpc21zIGluIGN1bHRpdmF0ZWQgdG9tYXRvIChTb2xhbnVtIGx5Y29wZXJzaWN1bSBMLikgcmV2ZWFscyBwYXR0ZXJucyBvZiBTTlAgdmFyaWF0aW9uIGFzc29jaWF0ZWQgd2l0aCBicmVlZGluZzsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NDY2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtIGRpc2NvdmVyeSBpbiBjdWx0aXZhdGVkIHRvbWF0byB2aWEgc2VxdWVuY2luZyBieSBzeW50aGVzaXM7IFBsYW50IEdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTctMjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRvbWF0byBicmVlZGluZyBpbiB0aGUgZ2Vub21pY3MgZXJhOiBpbnNpZ2h0cyBmcm9tIGEgU05QIGFycmF5OyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0zNTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBzdHVkaWVzIHVzaW5nIHNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbSBtYXJrZXJzIGRldmVsb3BlZCBieSByZS1zZXF1ZW5jaW5nIG9mIHRoZSBnZW5vbWVzIG9mIGN1bHRpdmF0ZWQgdG9tYXRvOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTU5My02MDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgYW5kIGxpbmthZ2UgZGlzZXF1aWxpYnJpdW0gYW5hbHlzaXMgd2l0aCBhIGdlbm9tZS13aWRlIGNvbGxlY3Rpb24gb2YgU05QcyB0aGF0IGRldGVjdCBwb2x5bW9ycGhpc20gaW4gY3VsdGl2YXRlZCB0b21hdG87IEouIEV4cC4gQm90LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02MjsgIGNpdGF0aW9uX3BhZ2VzPTE4MzEtNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIFNOUCBnZW5vdHlwaW5nIHRvIGluZmVyIHRoZSBlZmZlY3RzIG9uIGdlbmUgZnVuY3Rpb25zIGluIHRvbWF0bzsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0yMjEtMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIGEgbGFyZ2UgU05QIGdlbm90eXBpbmcgYXJyYXkgYW5kIGdlbmVyYXRpb24gb2YgaGlnaC1kZW5zaXR5IGdlbmV0aWMgbWFwcyBpbiB0b21hdG87IFBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lNDA1NjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIHZhcmlhdGlvbiBpbiB0b21hdG8gcG9wdWxhdGlvbnMgZnJvbSBmb3VyIGJyZWVkaW5nIHByb2dyYW1zIHJldmVhbGVkIGJ5IHNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbSBhbmQgc2ltcGxlIHNlcXVlbmNlIHJlcGVhdCBtYXJrZXJzOyBTY2kuIEhvcnRpYy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTIyOyAgY2l0YXRpb25fcGFnZXM9Ni0xNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9wdWxhdGlvbiBzdHJ1Y3R1cmUgYW5kIGdlbmV0aWMgZGlmZmVyZW50aWF0aW9uIGFzc29jaWF0ZWQgd2l0aCBicmVlZGluZyBoaXN0b3J5IGFuZCBzZWxlY3Rpb24gaW4gdG9tYXRvIChTb2xhbnVtIGx5Y29wZXJzaWN1bSBMLik7IEhlcmVkaXR5OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEwNjsgIGNpdGF0aW9uX3BhZ2VzPTkyNy0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC1kZW5zaXR5IFNOUCBnZW5vdHlwaW5nIG9mIHRvbWF0byAoU29sYW51bSBseWNvcGVyc2ljdW0gTC4pIHJldmVhbHMgcGF0dGVybnMgb2YgZ2VuZXRpYyB2YXJpYXRpb24gZHVlIHRvIGJyZWVkaW5nOyBQTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTQ1NTIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U05QIGdlbm90eXBpbmcgcmV2ZWFscyBnZW5ldGljIGRpdmVyc2l0eSBiZXR3ZWVuIGN1bHRpdmF0ZWQgbGFuZHJhY2VzIGFuZCBjb250ZW1wb3JhcnkgdmFyaWV0aWVzIG9mIHRvbWF0bzsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9ODM1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIGEgZ2VuZXJhbCBodW1hbiBjaHJvbW9zb21lIGp1bXBpbmcgbGlicmFyeSwgd2l0aCBhcHBsaWNhdGlvbiB0byBjeXN0aWMgZmlicm9zaXM7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICBjaXRhdGlvbl92b2x1bWU9MjM1OyAgY2l0YXRpb25fcGFnZXM9MTA0Ni05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgYmlvbG9naWNhbCBlZmZlY3RzIG9mIHNpbXBsZSB0YW5kZW0gcmVwZWF0czogbGVzc29ucyBmcm9tIHRoZSByZXBlYXQgZXhwYW5zaW9uIGRpc2Vhc2VzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTEwMTEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG9zcy1vZi1mdW5jdGlvbiB2YXJpYW50cyBpbiB0aGUgZ2Vub21lcyBvZiBoZWFsdGh5IGh1bWFuczsgSHVtLiBNb2wuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPVIxMjUtMTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgSHVtYW4gR2VuZSBNdXRhdGlvbiBEYXRhYmFzZTogcHJvdmlkaW5nIGEgY29tcHJlaGVuc2l2ZSBjZW50cmFsIG11dGF0aW9uIGRhdGFiYXNlIGZvciBtb2xlY3VsYXIgZGlhZ25vc3RpY3MgYW5kIHBlcnNvbmFsaXplZCBnZW5vbWljczsgSHVtLiBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9NjktNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBvcmlnaW4sIGV2b2x1dGlvbiwgYW5kIGZ1bmN0aW9uYWwgaW1wYWN0IG9mIHNob3J0IGluc2VydGlvbi1kZWxldGlvbiB2YXJpYW50cyBpZGVudGlmaWVkIGluIDE3OSBodW1hbiBnZW5vbWVzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTc0OS02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gaW5pdGlhbCBtYXAgb2YgaW5zZXJ0aW9uIGFuZCBkZWxldGlvbiAoSU5ERUwpIHZhcmlhdGlvbiBpbiB0aGUgaHVtYW4gZ2Vub21lOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTExODItOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNtYWxsIGluc2VydGlvbnMgYW5kIGRlbGV0aW9ucyAoSU5ERUxzKSBpbiBodW1hbiBnZW5vbWVzOyBIdW0uIE1vbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9UjEzMS0xMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJ1aWxkaW5nIGEgZ2Vub21lIGFuYWx5c2lzIHBpcGVsaW5lIHRvIHByZWRpY3QgZGlzZWFzZSByaXNrIGFuZCBwcmV2ZW50IGRpc2Vhc2U7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDI1OyAgY2l0YXRpb25fcGFnZXM9Mzk5My00MDA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHBsYXRmb3JtIG9mIGhpZ2gtZGVuc2l0eSBJTkRFTC9DQVBTIG1hcmtlcnMgZm9yIG1hcC1iYXNlZCBjbG9uaW5nIGluIEFyYWJpZG9wc2lzOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT02MzsgIGNpdGF0aW9uX3BhZ2VzPTg4MC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbGxlY3Rpb24gb2YgSU5ERUwgbWFya2VycyBmb3IgbWFwLWJhc2VkIGNsb25pbmcgaW4gc2V2ZW4gQXJhYmlkb3BzaXMgYWNjZXNzaW9uczsgSi4gRXhwLiBCb3QuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTYzOyAgY2l0YXRpb25fcGFnZXM9MjQ5MS01MDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIGdlbm9tZS13aWRlIEROQSBwb2x5bW9ycGhpc20gZGF0YWJhc2UgZm9yIG1hcC1iYXNlZCBjbG9uaW5nIG9mIHJpY2UgZ2VuZXM7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEzNTsgIGNpdGF0aW9uX3BhZ2VzPTExOTgtMjA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiAxMDQ3IGluc2VydGlvbi1kZWxldGlvbiBtYXJrZXJzIGZvciByaWNlIGdlbmV0aWMgc3R1ZGllcyBhbmQgYnJlZWRpbmc7IEdlbmV0LiBNb2wuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz01MjI2LTM1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBJbkRlbCBtYXJrZXIgc3lzdGVtIGZvciBhcHBsaWNhdGlvbiBpbiByaWNlIGJyZWVkaW5nIGFuZCBtYXBwaW5nIHN0dWRpZXM7IEV1cGh5dGljYTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTI7ICBjaXRhdGlvbl9wYWdlcz0xMzEtNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB0b21hdG8gZ2Vub21lIHNlcXVlbmNlIHByb3ZpZGVzIGluc2lnaHRzIGludG8gZmxlc2h5IGZydWl0IGV2b2x1dGlvbjsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ4NTsgIGNpdGF0aW9uX3BhZ2VzPTYzNS00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgdHJhbnNjcmlwdG9taWNzIHJldmVhbHMgcGF0dGVybnMgb2Ygc2VsZWN0aW9uIGluIGRvbWVzdGljYXRlZCBhbmQgd2lsZCB0b21hdG87IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPUUyNjU1LTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ud28gbG9jaSBmcm9tIEx5Y29wZXJzaWNvbiBoaXJzdXR1bSBMQTQwNyBjb25mZXIgcmVzaXN0YW5jZSB0byBzdHJhaW5zIG9mIENsYXZpYmFjdGVyIG1pY2hpZ2FuZW5zaXMgc3Vic3AuIE1pY2hpZ2FuZW5zaXM7IFBoeXRvcGF0aG9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTkyOyAgY2l0YXRpb25fcGFnZXM9NTA0LTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdWdzeTogZmFzdCBtdWx0aXBsZSBhbGlnbm1lbnQgb2YgY2xvc2VseSByZWxhdGVkIHdob2xlIGdlbm9tZXM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MzM0LTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmltZXIzIG9uIHRoZSBXV1cgZm9yIGdlbmVyYWwgdXNlcnMgYW5kIGZvciBiaW9sb2dpc3QgcHJvZ3JhbW1lcnM7IE1ldGhvZHMgTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzI7ICBjaXRhdGlvbl9wYWdlcz0zNjUtODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIHNpbmdsZS1jb3B5IG9ydGhvbG9nb3VzIGdlbmVzIGJldHdlZW4gUGh5c2FsaXMgYW5kIFNvbGFudW0gbHljb3BlcnNpY3VtIGFuZCBhbmFseXNpcyBvZiBnZW5ldGljIGRpdmVyc2l0eSBpbiBQaHlzYWxpcyB1c2luZyBtb2xlY3VsYXIgbWFya2VyczsgUExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWU1MDE2NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgZGlzdGFuY2UgYmV0d2VlbiBwb3B1bGF0aW9uczsgQW0uIE5hdC47ICBjaXRhdGlvbl95ZWFyPTE5NzI7ICBjaXRhdGlvbl92b2x1bWU9MTA2OyAgY2l0YXRpb25fcGFnZXM9MjgzLTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgYnJlZWRpbmcgaW4gZGV2ZWxvcGluZyBjb3VudHJpZXM6IGNoYWxsZW5nZXMgYW5kIHBlcnNwZWN0aXZlczsgQ3Vyci4gT3Bpbi4gUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0xLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsYW50IG1hcmtlci1hc3Npc3RlZCBicmVlZGluZyBhbmQgY29udmVudGlvbmFsIGJyZWVkaW5nOiBjaGFsbGVuZ2VzIGFuZCBwZXJzcGVjdGl2ZXM7IEFkdi4gQ3JvcCBTY2kuIFRlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz1lMTA2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SHVtYW4gZGlhbGxlbGljIGluc2VydGlvbi9kZWxldGlvbiBwb2x5bW9ycGhpc21zOyBBbS4gSi4gSHVtLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NzE7ICBjaXRhdGlvbl9wYWdlcz04NTQtNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm90eXBpbmcgb2YgMzggaW5zZXJ0aW9uL2RlbGV0aW9uIHBvbHltb3JwaGlzbXMgZm9yIGh1bWFuIGlkZW50aWZpY2F0aW9uIHVzaW5nIHVuaXZlcnNhbCBmbHVvcmVzY2VudCBQQ1I7IE1vbC4gQ2VsbC4gUHJvYmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MTMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBuZXcgbXVsdGlwbGV4IGZvciBodW1hbiBpZGVudGlmaWNhdGlvbiB1c2luZyBpbnNlcnRpb24vZGVsZXRpb24gcG9seW1vcnBoaXNtczsgRWxlY3Ryb3Bob3Jlc2lzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MzY4Mi05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBhbmFseXNpcyBvZiAzMCBJbkRlbCBtYXJrZXJzIGZvciBmb3JlbnNpYyB1c2UgaW4gZml2ZSBkaWZmZXJlbnQgQ2hpbmVzZSBwb3B1bGF0aW9uczsgR2VuZXQuIE1vbC4gUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTk2NC03OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBwb2x5bW9ycGhpc20gb2YgMjkgaGlnaGx5IGluZm9ybWF0aXZlIEluRGVsIG1hcmtlcnMgZm9yIGZvcmVuc2ljIHVzZSBpbiB0aGUgQ2hpbmVzZSBIYW4gcG9wdWxhdGlvbjsgRm9yZW5zaWMgU2NpLiBJbnQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTI3LTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Gb3JlbnNpYyBwZXJmb3JtYW5jZSBvZiB0d28gaW5zZXJ0aW9uLWRlbGV0aW9uIG1hcmtlciBhc3NheXM7IEludC4gSi4gTGVnYWwgTWVkLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjY7ICBjaXRhdGlvbl9wYWdlcz03MjUtMzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFkdmFuY2VzIGluIHRvbWF0byByZXNlYXJjaCBpbiB0aGUgcG9zdC1nZW5vbWUgZXJhOyBQbGFudCBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTI0My01NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmV4dC1nZW5lcmF0aW9uIEROQSBzZXF1ZW5jaW5nOyBOYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9MTEzNS00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hvbGUgZ2Vub21lIHJlc2VxdWVuY2luZyBpbiB0b21hdG8gcmV2ZWFscyB2YXJpYXRpb24gYXNzb2NpYXRlZCB3aXRoIGludHJvZ3Jlc3Npb24gYW5kIGJyZWVkaW5nIGV2ZW50czsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9NzkxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RG9tZXN0aWNhdGlvbiBhbmQgYnJlZWRpbmcgb2YgdG9tYXRvZXM6IHdoYXQgaGF2ZSB3ZSBnYWluZWQgYW5kIHdoYXQgY2FuIHdlIGdhaW4gaW4gdGhlIGZ1dHVyZT87IEFubi4gQm90LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDA7ICBjaXRhdGlvbl9wYWdlcz0xMDg1LTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SQVBEIG1hcmtlcnMgcmV2ZWFsIG5hcnJvd2luZyBnZW5ldGljIGJhc2Ugb2YgSW5kaWFuIHRvbWF0byBjdWx0aXZhcnM7IEN1cnIuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9ODI7ICBjaXRhdGlvbl9wYWdlcz0xMTM5LTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGRpdmVyc2l0eSBvZiBBcmdlbnRpbmEgdG9tYXRvIHZhcmlldGllcyByZXZlYWxlZCBieSBtb3JwaG9sb2dpY2FsIHRyYWl0cywgc2ltcGxlIHNlcXVlbmNlIHJlcGVhdCwgYW5kIHNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbSBtYXJrZXJzOyBQYWsuIEouIEJvdC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz00ODUtOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsbG96eW1lIHZhcmlhdGlvbiBpbiB0aGUgY3VsdGl2YXRlZCB0b21hdG8gYW5kIGNsb3NlbHkgcmVsYXRlZCBzcGVjaWVzOyBCdWxsLiBUb3JyLiBCb3QuIENsdWI7ICBjaXRhdGlvbl95ZWFyPTE5NzU7ICBjaXRhdGlvbl92b2x1bWU9MTAyOyAgY2l0YXRpb25fcGFnZXM9Mzc2LTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNsYXJpZmllZCBwb3NpdGlvbiBmb3IgU29sYW51bSBseWNvcGVyc2ljdW0gdmFyLiBjZXJhc2lmb3JtZSBpbiB0aGUgZXZvbHV0aW9uYXJ5IGhpc3Rvcnkgb2YgdG9tYXRvZXMgKHNvbGFuYWNlYWUpOyBCTUMgUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTEzMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNJRlQgSW5kZWw6IHByZWRpY3Rpb25zIGZvciB0aGUgZnVuY3Rpb25hbCBlZmZlY3RzIG9mIGFtaW5vIGFjaWQgaW5zZXJ0aW9ucy9kZWxldGlvbnMgaW4gcHJvdGVpbnM7IFBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lNzc5NDAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5ldyBmYW1pbHkgb2YgVHkxLWNvcGlhLWxpa2UgcmV0cm90cmFuc3Bvc29ucyBvcmlnaW5hdGVkIGluIHRoZSB0b21hdG8gZ2Vub21lIGJ5IGEgcmVjZW50IGhvcml6b250YWwgdHJhbnNmZXIgZXZlbnQ7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE4MTsgIGNpdGF0aW9uX3BhZ2VzPTExODMtOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc2luZ2xlLWJhc2UgZGVsZXRpb24gbXV0YXRpb24gaW4gU2xJQUE5IGdlbmUgY2F1c2VzIHRvbWF0byAoU29sYW51bSBseWNvcGVyc2ljdW0pIGVudGlyZSBtdXRhbnQ7IEouIFBsYW50IFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTIwOyAgY2l0YXRpb25fcGFnZXM9NjcxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEtMzA4IGRlbGV0aW9uIG9mIHRoZSB0b21hdG8gTEFQIHByb21vdGVycyBpcyBhYmxlIHRvIGRpcmVjdCBmbG93ZXItc3BlY2lmaWMgYW5kIE1lSkEtaW5kdWNlZCBleHByZXNzaW9uIGluIHRyYW5zZ2VuaWMgcGxhbnRzOyBQbGFudCBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9NjM5LTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBhbmQgYW5hbHlzaXMgb2YgcmVzaXN0YW5jZS1saWtlIGdlbmVzIGluIHRoZSB0b21hdG8gZ2Vub21lOyBKLiBQaHl0b3BhdGhvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTYyOyAgY2l0YXRpb25fcGFnZXM9MTM3LTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjEvNC80MjkvMTIyMjQ4Mi9kc3UwMDgucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIENvbXBhcmF0aXZlIGFuYWx5c2lzIG9mIHRoZSBnZW5vbWUgc2VxdWVuY2VzIG9mIFNvbGFudW0gbHljb3BlcnNpY3VtIHZhcmlldHkgSGVpbnogMTcwNiBhbmQgUy4gcGltcGluZWxsaWZvbGl1bSBhY2Nlc3Npb24gTEEgMTU4OSB1c2luZyBNVUdTWSBzb2Z0d2FyIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00NDcyMTImeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkBq85vqda2MQ2xfNj8sqehvD1PSE=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f4%2f429%2f447212" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f4%2f429%2f447212" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=17419" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/4/1/m_cover.gif?Expires=2147483647&amp;Signature=u1K0F1kMa-64NqKa4J2VQTk-ZgDNBNdy7xP9K68v94JN5WFnh6Of~dA8F~vuNuY-DWKv8sgKc6Tx8dCh~u-HxATm4iXS709rS-0HkL4VrV-yHYOvTEJMgBGMYLmkCZLHLnqJ4CE8~N9vWk3MlqGsd6CiX6Nq4~C6vim6BhKuTZhFWadSb~4gUcfc5RIhx0e6adpzN-8ic5w6tmizeNhHnsbot9fC0b5HPTtgY4T0sWivhNwSa56rOcmWTxrX6Ct6aeNXnMXGfCSv4nXY9sUZJB1uoWiHPNj1cxffLQO70T71y1MESr63u0ogFEZ8T3x1cUhck9QSbq-jL~iCjW~OcQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/4">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6690645">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6690645">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6690647">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6690647">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6690652">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6690652">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6690668">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6690668">3. Result</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6690694">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6690694">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6690706">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6690706">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6690710">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6690710">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6690713">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6690713">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6690716">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6690716">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/4/417/446835">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/21/4/439/2754534">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main"><em>In Silico</em> Identification and Experimental Validation of Insertion–Deletion Polymorphisms in Tomato Genome
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jingjing Yang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, Department of Vegetable Science</div>, <div class="insititution">China Agricultural University</div>, <div class="addr-line">No. 2 Yuanmingyuan Xilu, Beijing 100193</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jingjing+Yang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yang J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yang J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jingjing Yang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuanyuan Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, Department of Vegetable Science</div>, <div class="insititution">China Agricultural University</div>, <div class="addr-line">No. 2 Yuanmingyuan Xilu, Beijing 100193</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuanyuan+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yuanyuan Wang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Huolin Shen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, Department of Vegetable Science</div>, <div class="insititution">China Agricultural University</div>, <div class="addr-line">No. 2 Yuanmingyuan Xilu, Beijing 100193</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Huolin+Shen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shen H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shen H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Huolin Shen</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wencai Yang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Beijing Key Laboratory of Growth and Developmental Regulation for Protected Vegetable Crops, Department of Vegetable Science</div>, <div class="insititution">China Agricultural University</div>, <div class="addr-line">No. 2 Yuanmingyuan Xilu, Beijing 100193</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div> To whom correspondence should be addressed. Tel. +86 10-62734136. Fax. +86 10-62733404. E-mail: <a href="mailto:yangwencai@cau.edu.cn" target="_blank">yangwencai@cau.edu.cn</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wencai+Yang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yang W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yang W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Wencai Yang</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="fn1"><div class="fn"><div class="footnote-content">Edited by Dr Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 4, 1 August 2014, Pages 429–438, <a href='https://doi.org/10.1093/dnares/dsu008'>https://doi.org/10.1093/dnares/dsu008</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">11 March 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">19 December 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">04 February 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="447212" href="/dnaresearch/article-pdf/21/4/429/1222482/dsu008.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Jingjing Yang, Yuanyuan Wang, Huolin Shen, Wencai Yang;  <em>In Silico</em> Identification and Experimental Validation of Insertion–Deletion Polymorphisms in Tomato Genome, <em>DNA Research</em>, Volume 21, Issue 4, 1 August 2014, Pages 429–438, <a href='https://doi.org/10.1093/dnares/dsu008'>https://doi.org/10.1093/dnares/dsu008</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/447212?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/447212?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/447212?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/447212?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/447212?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/447212?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/447212?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=%26lt%3Bem%26gt%3BIn%20Silico%26lt%3B%2Fem%26gt%3B%20Identification%20and%20Experimental%20Validation%20of%20Insertion%E2%80%93Deletion%20Polymorphisms%20in%20Tomato%20Genome&amp;publicationDate=2014-03-11&amp;volumeNum=21&amp;issueNum=4&amp;author=Yang%2C%20Jingjing%3B%20Wang%2C%20Yuanyuan&amp;startPage=429&amp;endPage=438&amp;contentId=10.1093%2Fdnares%2Fdsu008&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f4%2f429%2f447212" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6690645 id="6690645" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Comparative analysis of the genome sequences of <em>Solanum lycopersicum</em> variety Heinz 1706 and <em>S. pimpinellifolium</em> accession LA 1589 using MUGSY software identified 145 695 insertion–deletion (InDel) polymorphisms. A selected set of 3029 candidate InDels (≥2 bp) across the entire tomato genome were subjected to PCR validation, and 82.4% could be verified. Of 2272 polymorphic InDels between LA 1589 and Heinz 1706, 61.6, 45.2, and 31.6% were polymorphic in 8 accessions of <em>S. pimpinellifolium</em>, 4 accessions of <em>S. lycopersicum</em> var. <em>cerasiforme</em>, and 10 varieties of <em>S. lycopersicum</em>, respectively. Genetic distance was 0.216 in <em>S. pimpinellifolium</em>, 0.202 in <em>S. lycopersicum</em> var. <em>cerasiforme</em>, and 0.108 in <em>S. lycopersicum</em>. The data suggested a reduction of genetic variation from <em>S. pimpinellifolium</em> to <em>S. lycopersicum</em> var. <em>cerasiforme</em> and <em>S. lycopersicum</em>. Cluster analysis showed that the 8 accessions of <em>S. pimpinellifolium</em> were in one group, whereas 4 accessions of <em>S. lycopersicum</em> var. <em>cerasiforme</em> and 10 varieties of <em>S. lycopersicum</em> were in the same group.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="insertion–deletion">insertion–deletion</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Solanum lycopersicum&quot;"><em>Solanum lycopersicum</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genetic variation&quot;">genetic variation</a></div>                            <h2 scrollto-destination=6690647 id="6690647" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Tomato (<em>Solanum lycopersicum</em> L.) is an economically important vegetable crop worldwide and a pre-eminent plant genetic analysis system. Genetic marker development for tomato has been conducted over 30 years through various approaches, including restriction fragment length polymorphism (RFLP), random ampliﬁed polymorphic DNA (RAPD), ampliﬁed fragment length polymorphisms (AFLPs), simple sequence repeat (SSR), cleaved ampliﬁed polymorphisms (CAPs), and conserved ortholog sets (COSs). Most markers developed by these approaches are based on DNA or cDNA polymorphisms between wild species and cultivated tomato, which lead to the construction of the first generation reference linkage maps and isolation of genes of interests.<sup><a reveal-id="DSU008C1" data-open="DSU008C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSU008C2" data-open="DSU008C2" class="link link-ref link-reveal xref-bibr">2</a></sup> However, the ability of using these markers to detect polymorphisms in cultivated tomato is limited.<sup><a reveal-id="DSU008C3" data-open="DSU008C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Recent efforts to develop new markers in cultivated tomato have been focus on single-nucleotide polymorphisms (SNPs) using <em>in silico</em> mining of expressed sequence tag database and experimental validation,<sup><a reveal-id="DSU008C4  DSU008C7" data-open="DSU008C4  DSU008C7" class="link link-ref link-reveal xref-bibr">4–7</a></sup> amplicon sequencing of COS genes,<sup><a reveal-id="DSU008C8" data-open="DSU008C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSU008C9" data-open="DSU008C9" class="link link-ref link-reveal xref-bibr">9</a></sup> hybridization to oligonucleotide array,<sup><a reveal-id="DSU008C10" data-open="DSU008C10" class="link link-ref link-reveal xref-bibr">10</a></sup> and next-generation sequencing of transcriptome or re-sequencing of genome.<sup><a reveal-id="DSU008C11  DSU008C13" data-open="DSU008C11  DSU008C13" class="link link-ref link-reveal xref-bibr">11–13</a></sup> Owing to the abundance and wide distribution of SNPs in the whole genome and the availability of automatic large-scale genotyping platform, SNPs have widely been used in association analysis,<sup><a reveal-id="DSU008C13  DSU008C15" data-open="DSU008C13  DSU008C15" class="link link-ref link-reveal xref-bibr">13–15</a></sup> high-density SNP map construction,<sup><a reveal-id="DSU008C7" data-open="DSU008C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSU008C16" data-open="DSU008C16" class="link link-ref link-reveal xref-bibr">16</a></sup> as well as population structure and genetic variation analysis<sup><a reveal-id="DSU008C17  DSU008C20" data-open="DSU008C17  DSU008C20" class="link link-ref link-reveal xref-bibr">17–20</a></sup> in cultivated tomato.</p><p>Short insertion and deletion (InDel) polymorphisms are increasingly being received attention in human because they are the second abundant form of genetic variation and can influence multiple human phenotypes including diseases.<sup><a reveal-id="DSU008C21  DSU008C25" data-open="DSU008C21  DSU008C25" class="link link-ref link-reveal xref-bibr">21–25</a></sup> Therefore, great efforts have been put on identification, mapping, and functional analysis of InDels in the human genome.<sup><a reveal-id="DSU008C26  DSU008C28" data-open="DSU008C26  DSU008C28" class="link link-ref link-reveal xref-bibr">26–28</a></sup> Similar work has been done in other species, such as Arabidopsis and rice.<sup><a reveal-id="DSU008C29  DSU008C33" data-open="DSU008C29  DSU008C33" class="link link-ref link-reveal xref-bibr">29–33</a></sup> In tomato, a total of 749 966 putative InDels of 3–300 bp have been identified by comparing the genome sequences of <em>Solanum pimpinellifolium</em> accession LA 1589 and <em>S. lycopersicum</em> variety Heinz 1706,<sup><a reveal-id="DSU008C34" data-open="DSU008C34" class="link link-ref link-reveal xref-bibr">34</a></sup> and more than 80 000 putative InDels of 1–15 bp have been discovered by comparative analysis of transcriptome between wild species <em>S. galapagense</em> and cultivated tomato.<sup><a reveal-id="DSU008C35" data-open="DSU008C35" class="link link-ref link-reveal xref-bibr">35</a></sup> However, less work on discovery of InDels in cultivated tomato has been done.</p><p>The availability of the whole genome or transcriptome sequences provides a potential to identify InDels <em>in silico</em>. We here developed a pipeline to identify InDels by comparative analysis of the two available genome sequences of LA 1589 and Heinz 1706. A total of 3029 candidate InDels were subjected to experimental validation by PCR amplification of genomic DNA in a collection of 22 tomato lines. The main objective of this study was to develop easy-using markers for genetic study and marker-assisted selection in cultivated tomato.</p>                            <h2 scrollto-destination=6690652 id="6690652" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6690653 id="6690653" class="section-title" data-legacy-id=s2a>2.1. Plant materials and DNA isolation</h3>
<p>A panel of 22 tomato genotypes comprising of cultivated tomato (<em>S. lycopersicum</em>) and its wild relatives were used to validate InDel polymorphisms. These inbred lines were selected to represent a diverse collection including eight accessions of <em>S. pimpinellifolium</em>, five processing varieties, one greenhouse cultivar, four fresh market cultivars, and four <em>S. lycopersicum</em> var. <em>cerasiforme</em> accessions (Table <a reveal-id="DSU008TB1" data-open="DSU008TB1" class="link link-reveal link-table xref-fig">1</a>). Nine of them were used for SNP detection in our previous study.<sup><a reveal-id="DSU008C9" data-open="DSU008C9" class="link link-ref link-reveal xref-bibr">9</a></sup> The eight <em>S. pimpinellifolium</em> accessions were selected from the core collection or sources being used for genetic studies and were used to detect polymorphisms of candidate InDels within the species. Genomic DNA was isolated from fresh-collected young leaves of at least eight plants for each genotype using the modified CTAB method.<sup><a reveal-id="DSU008C36" data-open="DSU008C36" class="link link-ref link-reveal xref-bibr">36</a></sup></p>                        <a id="6690655" scrollto-destination="6690655"></a>
<div content-id="DSU008TB1" class="table-modal"><div class="table-wrap"><div id="DSU008TB1" class="table-wrap-title" data-id="DSU008TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Description of plant materials</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Genotype </th><th>Species </th><th>Market type </th><th>Origin </th><th>Note </th></tr></thead><tbody><tr><td>LA 1269 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Resistance source for late blight (<em>Ph-3</em>) </td></tr><tr><td>LA 1589 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Genome sequenced, widely used for genetic studies </td></tr><tr><td>PI 128216 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Bolivia </td><td>Resistance source for bacterial spot and bacterial speck </td></tr><tr><td>LA 0373 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Core collection </td></tr><tr><td>LA 0400 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Core collection </td></tr><tr><td>LA 0722 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Core collection </td></tr><tr><td>LA 1582 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Core collection </td></tr><tr><td>LA 2181 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Core collection </td></tr><tr><td>Heinz 1706 </td><td><em>Solanum lycopersicum</em> </td><td>Processing </td><td>USA </td><td>Genome sequenced </td></tr><tr><td>OH 88119 </td><td><em>Solanum lycopersicum</em> </td><td>Processing </td><td>USA </td><td>Early fruit set </td></tr><tr><td>OH 9242 </td><td><em>Solanum lycopersicum</em> </td><td>Processing </td><td>USA </td><td>High lycopene </td></tr><tr><td>Liger 87-5 </td><td><em>Solanum lycopersicum</em> </td><td>Processing </td><td>China </td><td>Current major variety in China </td></tr><tr><td>M 82 </td><td><em>Solanum lycopersicum</em> </td><td>Processing </td><td>Israel </td><td>Widely used in genetic studies </td></tr><tr><td>Money maker </td><td><em>Solanum lycopersicum</em> </td><td>Greenhouse </td><td>USA </td><td>Widely used in genetic studies </td></tr><tr><td>Fla.7600 </td><td><em>Solanum lycopersicum</em> </td><td>Fresh market </td><td>USA </td><td>Variety with multiple disease resistance genes </td></tr><tr><td>Baiguoqiangfeng </td><td><em>Solanum lycopersicum</em> </td><td>Fresh market </td><td>China </td><td>Previous major variety in China </td></tr><tr><td>Shijifeng </td><td><em>Solanum lycopersicum</em> </td><td>Fresh market </td><td>China </td><td>Previous major variety in China </td></tr><tr><td>Zhongshu 5 </td><td><em>Solanum lycopersicum</em> </td><td>Fresh market </td><td>China </td><td>Previous major variety in China </td></tr><tr><td>Black cherry </td><td><em>Solanum lycopersicum</em> var. <em>cerasiforme</em> </td><td>Cherry </td><td>USA </td><td>Brown fruit </td></tr><tr><td>LA 1310 </td><td><em>Solanum lycopersicum</em> var. <em>cerasiforme</em> </td><td>Cherry </td><td>Peru </td><td>Salt tolerance </td></tr><tr><td>LA 4133 </td><td><em>Solanum lycopersicum</em> var. <em>cerasiforme</em> </td><td>Cherry </td><td>USA </td><td>Core collection, salt tolerance </td></tr><tr><td>PI 114490 </td><td><em>Solanum lycopersicum</em> var. <em>cerasiforme</em> </td><td>Cherry </td><td>UK </td><td>Yellow fruit, resistance to bacterial spot </td></tr></tbody></table></div><div content-id="DSU008TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Genotype </th><th>Species </th><th>Market type </th><th>Origin </th><th>Note </th></tr></thead><tbody><tr><td>LA 1269 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Resistance source for late blight (<em>Ph-3</em>) </td></tr><tr><td>LA 1589 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Genome sequenced, widely used for genetic studies </td></tr><tr><td>PI 128216 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Bolivia </td><td>Resistance source for bacterial spot and bacterial speck </td></tr><tr><td>LA 0373 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Core collection </td></tr><tr><td>LA 0400 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Core collection </td></tr><tr><td>LA 0722 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Core collection </td></tr><tr><td>LA 1582 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Core collection </td></tr><tr><td>LA 2181 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Core collection </td></tr><tr><td>Heinz 1706 </td><td><em>Solanum lycopersicum</em> </td><td>Processing </td><td>USA </td><td>Genome sequenced </td></tr><tr><td>OH 88119 </td><td><em>Solanum lycopersicum</em> </td><td>Processing </td><td>USA </td><td>Early fruit set </td></tr><tr><td>OH 9242 </td><td><em>Solanum lycopersicum</em> </td><td>Processing </td><td>USA </td><td>High lycopene </td></tr><tr><td>Liger 87-5 </td><td><em>Solanum lycopersicum</em> </td><td>Processing </td><td>China </td><td>Current major variety in China </td></tr><tr><td>M 82 </td><td><em>Solanum lycopersicum</em> </td><td>Processing </td><td>Israel </td><td>Widely used in genetic studies </td></tr><tr><td>Money maker </td><td><em>Solanum lycopersicum</em> </td><td>Greenhouse </td><td>USA </td><td>Widely used in genetic studies </td></tr><tr><td>Fla.7600 </td><td><em>Solanum lycopersicum</em> </td><td>Fresh market </td><td>USA </td><td>Variety with multiple disease resistance genes </td></tr><tr><td>Baiguoqiangfeng </td><td><em>Solanum lycopersicum</em> </td><td>Fresh market </td><td>China </td><td>Previous major variety in China </td></tr><tr><td>Shijifeng </td><td><em>Solanum lycopersicum</em> </td><td>Fresh market </td><td>China </td><td>Previous major variety in China </td></tr><tr><td>Zhongshu 5 </td><td><em>Solanum lycopersicum</em> </td><td>Fresh market </td><td>China </td><td>Previous major variety in China </td></tr><tr><td>Black cherry </td><td><em>Solanum lycopersicum</em> var. <em>cerasiforme</em> </td><td>Cherry </td><td>USA </td><td>Brown fruit </td></tr><tr><td>LA 1310 </td><td><em>Solanum lycopersicum</em> var. <em>cerasiforme</em> </td><td>Cherry </td><td>Peru </td><td>Salt tolerance </td></tr><tr><td>LA 4133 </td><td><em>Solanum lycopersicum</em> var. <em>cerasiforme</em> </td><td>Cherry </td><td>USA </td><td>Core collection, salt tolerance </td></tr><tr><td>PI 114490 </td><td><em>Solanum lycopersicum</em> var. <em>cerasiforme</em> </td><td>Cherry </td><td>UK </td><td>Yellow fruit, resistance to bacterial spot </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6690655" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU008TB1" class="table-wrap-title" data-id="DSU008TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Description of plant materials</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Genotype </th><th>Species </th><th>Market type </th><th>Origin </th><th>Note </th></tr></thead><tbody><tr><td>LA 1269 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Resistance source for late blight (<em>Ph-3</em>) </td></tr><tr><td>LA 1589 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Genome sequenced, widely used for genetic studies </td></tr><tr><td>PI 128216 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Bolivia </td><td>Resistance source for bacterial spot and bacterial speck </td></tr><tr><td>LA 0373 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Core collection </td></tr><tr><td>LA 0400 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Core collection </td></tr><tr><td>LA 0722 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Core collection </td></tr><tr><td>LA 1582 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Core collection </td></tr><tr><td>LA 2181 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Core collection </td></tr><tr><td>Heinz 1706 </td><td><em>Solanum lycopersicum</em> </td><td>Processing </td><td>USA </td><td>Genome sequenced </td></tr><tr><td>OH 88119 </td><td><em>Solanum lycopersicum</em> </td><td>Processing </td><td>USA </td><td>Early fruit set </td></tr><tr><td>OH 9242 </td><td><em>Solanum lycopersicum</em> </td><td>Processing </td><td>USA </td><td>High lycopene </td></tr><tr><td>Liger 87-5 </td><td><em>Solanum lycopersicum</em> </td><td>Processing </td><td>China </td><td>Current major variety in China </td></tr><tr><td>M 82 </td><td><em>Solanum lycopersicum</em> </td><td>Processing </td><td>Israel </td><td>Widely used in genetic studies </td></tr><tr><td>Money maker </td><td><em>Solanum lycopersicum</em> </td><td>Greenhouse </td><td>USA </td><td>Widely used in genetic studies </td></tr><tr><td>Fla.7600 </td><td><em>Solanum lycopersicum</em> </td><td>Fresh market </td><td>USA </td><td>Variety with multiple disease resistance genes </td></tr><tr><td>Baiguoqiangfeng </td><td><em>Solanum lycopersicum</em> </td><td>Fresh market </td><td>China </td><td>Previous major variety in China </td></tr><tr><td>Shijifeng </td><td><em>Solanum lycopersicum</em> </td><td>Fresh market </td><td>China </td><td>Previous major variety in China </td></tr><tr><td>Zhongshu 5 </td><td><em>Solanum lycopersicum</em> </td><td>Fresh market </td><td>China </td><td>Previous major variety in China </td></tr><tr><td>Black cherry </td><td><em>Solanum lycopersicum</em> var. <em>cerasiforme</em> </td><td>Cherry </td><td>USA </td><td>Brown fruit </td></tr><tr><td>LA 1310 </td><td><em>Solanum lycopersicum</em> var. <em>cerasiforme</em> </td><td>Cherry </td><td>Peru </td><td>Salt tolerance </td></tr><tr><td>LA 4133 </td><td><em>Solanum lycopersicum</em> var. <em>cerasiforme</em> </td><td>Cherry </td><td>USA </td><td>Core collection, salt tolerance </td></tr><tr><td>PI 114490 </td><td><em>Solanum lycopersicum</em> var. <em>cerasiforme</em> </td><td>Cherry </td><td>UK </td><td>Yellow fruit, resistance to bacterial spot </td></tr></tbody></table></div><div content-id="DSU008TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Genotype </th><th>Species </th><th>Market type </th><th>Origin </th><th>Note </th></tr></thead><tbody><tr><td>LA 1269 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Resistance source for late blight (<em>Ph-3</em>) </td></tr><tr><td>LA 1589 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Genome sequenced, widely used for genetic studies </td></tr><tr><td>PI 128216 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Bolivia </td><td>Resistance source for bacterial spot and bacterial speck </td></tr><tr><td>LA 0373 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Core collection </td></tr><tr><td>LA 0400 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Core collection </td></tr><tr><td>LA 0722 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Core collection </td></tr><tr><td>LA 1582 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Core collection </td></tr><tr><td>LA 2181 </td><td><em>Solanum pimpinellifolium</em> </td><td>Wild </td><td>Peru </td><td>Core collection </td></tr><tr><td>Heinz 1706 </td><td><em>Solanum lycopersicum</em> </td><td>Processing </td><td>USA </td><td>Genome sequenced </td></tr><tr><td>OH 88119 </td><td><em>Solanum lycopersicum</em> </td><td>Processing </td><td>USA </td><td>Early fruit set </td></tr><tr><td>OH 9242 </td><td><em>Solanum lycopersicum</em> </td><td>Processing </td><td>USA </td><td>High lycopene </td></tr><tr><td>Liger 87-5 </td><td><em>Solanum lycopersicum</em> </td><td>Processing </td><td>China </td><td>Current major variety in China </td></tr><tr><td>M 82 </td><td><em>Solanum lycopersicum</em> </td><td>Processing </td><td>Israel </td><td>Widely used in genetic studies </td></tr><tr><td>Money maker </td><td><em>Solanum lycopersicum</em> </td><td>Greenhouse </td><td>USA </td><td>Widely used in genetic studies </td></tr><tr><td>Fla.7600 </td><td><em>Solanum lycopersicum</em> </td><td>Fresh market </td><td>USA </td><td>Variety with multiple disease resistance genes </td></tr><tr><td>Baiguoqiangfeng </td><td><em>Solanum lycopersicum</em> </td><td>Fresh market </td><td>China </td><td>Previous major variety in China </td></tr><tr><td>Shijifeng </td><td><em>Solanum lycopersicum</em> </td><td>Fresh market </td><td>China </td><td>Previous major variety in China </td></tr><tr><td>Zhongshu 5 </td><td><em>Solanum lycopersicum</em> </td><td>Fresh market </td><td>China </td><td>Previous major variety in China </td></tr><tr><td>Black cherry </td><td><em>Solanum lycopersicum</em> var. <em>cerasiforme</em> </td><td>Cherry </td><td>USA </td><td>Brown fruit </td></tr><tr><td>LA 1310 </td><td><em>Solanum lycopersicum</em> var. <em>cerasiforme</em> </td><td>Cherry </td><td>Peru </td><td>Salt tolerance </td></tr><tr><td>LA 4133 </td><td><em>Solanum lycopersicum</em> var. <em>cerasiforme</em> </td><td>Cherry </td><td>USA </td><td>Core collection, salt tolerance </td></tr><tr><td>PI 114490 </td><td><em>Solanum lycopersicum</em> var. <em>cerasiforme</em> </td><td>Cherry </td><td>UK </td><td>Yellow fruit, resistance to bacterial spot </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6690655" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6690656 id="6690656" class="section-title" data-legacy-id=s2b>2.2. Prediction of InDels between LA 1589 and Heinz 1706</h3>
<p>The genomic DNA sequences of <em>S. pimpinellifolium</em> accession LA 1589 (Spimpinellifolium_genome.contigs.fasta.gz) and <em>S. lycopersicum</em> variety Heinz 1706 (S_lycopersicum_chromosomes.2.40.fa.gz) were downloaded to a local computer from the SOL Genomics Network (SGN, <a class="link link-uri" href="http://solgenomics.net/">http://solgenomics.net/</a>, 19 February 2014, date last accessed). The genomic DNA sequence contigs of LA 1589 were assigned to Heinz 1706 genome using local MUGSY<sup><a reveal-id="DSU008C37" data-open="DSU008C37" class="link link-ref link-reveal xref-bibr">37</a></sup> downloaded from Sourceforge (<a class="link link-uri" href="http://mugsy.sourceforge.net/">http://mugsy.sourceforge.net/</a>, 19 February 2014, date last accessed). InDel polymorphisms referring to Heinz 1706 were mined from the alignments using custom PERL scripts. Flanking sequences of 100 bp from each side of candidate InDels were extracted from Heinz 1706 sequences for insertion and LA 1589 sequences for deletion. The flanking sequences were then blasted against LA 1589 sequences for deletion or Heinz 1706 sequences for insertion using local BLASTall with an <em>E</em>-value of <em>e</em><sup>−20</sup> to remove hits with low similarity. The types (insertion or deletion), lengths, nucleotides, and chromosomal positions of InDels were extracted using a PERL script with the highest score of blast search.</p>                            <h3 scrollto-destination=6690658 id="6690658" class="section-title" data-legacy-id=s2c>2.3. Selection of InDels for validation and primer design</h3>
<p>Our initial goal was to verify 3000 candidate InDels of 2 bp or longer evenly distributing on 12 chromosomes. Based on the genome sequenced for Heinz 1706 (760 Mb),<sup><a reveal-id="DSU008C34" data-open="DSU008C34" class="link link-ref link-reveal xref-bibr">34</a></sup> the average distance between two adjacent InDels would be ∼250 kb. The number of InDels to be validated was determined by the length of each chromosome (Table <a reveal-id="DSU008TB2" data-open="DSU008TB2" class="link link-reveal link-table xref-fig">2</a>). However, we found that the InDels were not always evenly distributed on chromosomes and hotspots have high levels of InDels than other regions. Therefore, we tried to acquire an InDel per 200 kb in each chromosome using a PERL script. If a region on a chromosome did not have InDel variation, the PERL script would make 200 plus 100 kb on circulation until it matched. </p>                        <a id="6690660" scrollto-destination="6690660"></a>
<div content-id="DSU008TB2" class="table-modal"><div class="table-wrap"><div id="DSU008TB2" class="table-wrap-title" data-id="DSU008TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary statistics for primer design, PCR amplification, and polymorphisms</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Chromosome </th><th rowspan="2">Sequence length (∼Mb)<sup>a</sup> </th><th rowspan="2">No. of primers designed </th><th rowspan="2">No. of primers without PCR amplification </th><th rowspan="2">No. of primers without polymorphism </th><th rowspan="2">No. of primers examined </th><th colspan="3">No. (percentage) of polymorphic InDels<hr /> </th></tr><tr><th><em>S. pimpinellifolium</em> </th><th><em>S. lycopersicum</em> var. <em>cerasiforme</em> </th><th><em>S. lycopersicum</em> </th></tr></thead><tbody><tr><td>chr01 </td><td>90.3 </td><td>362 </td><td>63 </td><td>89 </td><td>210 </td><td>132 (62.9) </td><td>38 (18.1) </td><td>22 (10.5) </td></tr><tr><td>chr02 </td><td>49.9 </td><td>207 </td><td>10 </td><td>22 </td><td>175 </td><td>98 (56.0) </td><td>134 (76.6) </td><td>75 (42.9) </td></tr><tr><td>chr03 </td><td>64.8 </td><td>254 </td><td>19 </td><td>31 </td><td>204 </td><td>123 (60.3) </td><td>128 (62.7) </td><td>32 (15.7) </td></tr><tr><td>chr04 </td><td>64.1 </td><td>254 </td><td>12 </td><td>24 </td><td>218 </td><td>120 (55.0) </td><td>128 (58.7) </td><td>144 (66.1) </td></tr><tr><td>chr05 </td><td>65.0 </td><td>262 </td><td>33 </td><td>53 </td><td>176 </td><td>112 (63.6) </td><td>125 (71.0) </td><td>127 (72.2) </td></tr><tr><td>chr06 </td><td>46.0 </td><td>181 </td><td>20 </td><td>40 </td><td>121 </td><td>99 (81.8) </td><td>83 (68.6) </td><td>73 (60.3) </td></tr><tr><td>chr07 </td><td>65.3 </td><td>259 </td><td>39 </td><td>15 </td><td>205 </td><td>94 (45.9) </td><td>26 (12.7) </td><td>17 (8.3) </td></tr><tr><td>chr08 </td><td>63.0 </td><td>252 </td><td>23 </td><td>21 </td><td>208 </td><td>160 (76.9) </td><td>12 (5.8) </td><td>11 (5.3) </td></tr><tr><td>chr09 </td><td>67.7 </td><td>267 </td><td>21 </td><td>34 </td><td>212 </td><td>135 (63.7) </td><td>80 (37.7) </td><td>66 (31.1) </td></tr><tr><td>chr10 </td><td>64.8 </td><td>255 </td><td>14 </td><td>35 </td><td>206 </td><td>131 (63.6) </td><td>111 (53.9) </td><td>11 (5.3) </td></tr><tr><td>chr11 </td><td>53.4 </td><td>214 </td><td>8 </td><td>35 </td><td>171 </td><td>80 (46.8) </td><td>123 (71.9) </td><td>109 (63.7) </td></tr><tr><td>chr12 </td><td>65.5 </td><td>262 </td><td>10 </td><td>86 </td><td>166 </td><td>116 (69.9) </td><td>40 (24.1) </td><td>30 (18.1) </td></tr><tr><td>Total </td><td>759.8 </td><td>3029 </td><td>272 </td><td>485 </td><td>2272 </td><td>1400 (61.6) </td><td>1028 (45.2) </td><td>717 (31.6) </td></tr></tbody></table></div><div content-id="DSU008TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Chromosome </th><th rowspan="2">Sequence length (∼Mb)<sup>a</sup> </th><th rowspan="2">No. of primers designed </th><th rowspan="2">No. of primers without PCR amplification </th><th rowspan="2">No. of primers without polymorphism </th><th rowspan="2">No. of primers examined </th><th colspan="3">No. (percentage) of polymorphic InDels<hr /> </th></tr><tr><th><em>S. pimpinellifolium</em> </th><th><em>S. lycopersicum</em> var. <em>cerasiforme</em> </th><th><em>S. lycopersicum</em> </th></tr></thead><tbody><tr><td>chr01 </td><td>90.3 </td><td>362 </td><td>63 </td><td>89 </td><td>210 </td><td>132 (62.9) </td><td>38 (18.1) </td><td>22 (10.5) </td></tr><tr><td>chr02 </td><td>49.9 </td><td>207 </td><td>10 </td><td>22 </td><td>175 </td><td>98 (56.0) </td><td>134 (76.6) </td><td>75 (42.9) </td></tr><tr><td>chr03 </td><td>64.8 </td><td>254 </td><td>19 </td><td>31 </td><td>204 </td><td>123 (60.3) </td><td>128 (62.7) </td><td>32 (15.7) </td></tr><tr><td>chr04 </td><td>64.1 </td><td>254 </td><td>12 </td><td>24 </td><td>218 </td><td>120 (55.0) </td><td>128 (58.7) </td><td>144 (66.1) </td></tr><tr><td>chr05 </td><td>65.0 </td><td>262 </td><td>33 </td><td>53 </td><td>176 </td><td>112 (63.6) </td><td>125 (71.0) </td><td>127 (72.2) </td></tr><tr><td>chr06 </td><td>46.0 </td><td>181 </td><td>20 </td><td>40 </td><td>121 </td><td>99 (81.8) </td><td>83 (68.6) </td><td>73 (60.3) </td></tr><tr><td>chr07 </td><td>65.3 </td><td>259 </td><td>39 </td><td>15 </td><td>205 </td><td>94 (45.9) </td><td>26 (12.7) </td><td>17 (8.3) </td></tr><tr><td>chr08 </td><td>63.0 </td><td>252 </td><td>23 </td><td>21 </td><td>208 </td><td>160 (76.9) </td><td>12 (5.8) </td><td>11 (5.3) </td></tr><tr><td>chr09 </td><td>67.7 </td><td>267 </td><td>21 </td><td>34 </td><td>212 </td><td>135 (63.7) </td><td>80 (37.7) </td><td>66 (31.1) </td></tr><tr><td>chr10 </td><td>64.8 </td><td>255 </td><td>14 </td><td>35 </td><td>206 </td><td>131 (63.6) </td><td>111 (53.9) </td><td>11 (5.3) </td></tr><tr><td>chr11 </td><td>53.4 </td><td>214 </td><td>8 </td><td>35 </td><td>171 </td><td>80 (46.8) </td><td>123 (71.9) </td><td>109 (63.7) </td></tr><tr><td>chr12 </td><td>65.5 </td><td>262 </td><td>10 </td><td>86 </td><td>166 </td><td>116 (69.9) </td><td>40 (24.1) </td><td>30 (18.1) </td></tr><tr><td>Total </td><td>759.8 </td><td>3029 </td><td>272 </td><td>485 </td><td>2272 </td><td>1400 (61.6) </td><td>1028 (45.2) </td><td>717 (31.6) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn1"><div class="fn"><p><sup>a</sup>The sequenced genome size was obtained from Sato <em>et al.</em><sup><a reveal-id="DSU008C34" data-open="DSU008C34" class="link link-ref link-reveal xref-bibr">34</a></sup></p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6690660" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU008TB2" class="table-wrap-title" data-id="DSU008TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary statistics for primer design, PCR amplification, and polymorphisms</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Chromosome </th><th rowspan="2">Sequence length (∼Mb)<sup>a</sup> </th><th rowspan="2">No. of primers designed </th><th rowspan="2">No. of primers without PCR amplification </th><th rowspan="2">No. of primers without polymorphism </th><th rowspan="2">No. of primers examined </th><th colspan="3">No. (percentage) of polymorphic InDels<hr /> </th></tr><tr><th><em>S. pimpinellifolium</em> </th><th><em>S. lycopersicum</em> var. <em>cerasiforme</em> </th><th><em>S. lycopersicum</em> </th></tr></thead><tbody><tr><td>chr01 </td><td>90.3 </td><td>362 </td><td>63 </td><td>89 </td><td>210 </td><td>132 (62.9) </td><td>38 (18.1) </td><td>22 (10.5) </td></tr><tr><td>chr02 </td><td>49.9 </td><td>207 </td><td>10 </td><td>22 </td><td>175 </td><td>98 (56.0) </td><td>134 (76.6) </td><td>75 (42.9) </td></tr><tr><td>chr03 </td><td>64.8 </td><td>254 </td><td>19 </td><td>31 </td><td>204 </td><td>123 (60.3) </td><td>128 (62.7) </td><td>32 (15.7) </td></tr><tr><td>chr04 </td><td>64.1 </td><td>254 </td><td>12 </td><td>24 </td><td>218 </td><td>120 (55.0) </td><td>128 (58.7) </td><td>144 (66.1) </td></tr><tr><td>chr05 </td><td>65.0 </td><td>262 </td><td>33 </td><td>53 </td><td>176 </td><td>112 (63.6) </td><td>125 (71.0) </td><td>127 (72.2) </td></tr><tr><td>chr06 </td><td>46.0 </td><td>181 </td><td>20 </td><td>40 </td><td>121 </td><td>99 (81.8) </td><td>83 (68.6) </td><td>73 (60.3) </td></tr><tr><td>chr07 </td><td>65.3 </td><td>259 </td><td>39 </td><td>15 </td><td>205 </td><td>94 (45.9) </td><td>26 (12.7) </td><td>17 (8.3) </td></tr><tr><td>chr08 </td><td>63.0 </td><td>252 </td><td>23 </td><td>21 </td><td>208 </td><td>160 (76.9) </td><td>12 (5.8) </td><td>11 (5.3) </td></tr><tr><td>chr09 </td><td>67.7 </td><td>267 </td><td>21 </td><td>34 </td><td>212 </td><td>135 (63.7) </td><td>80 (37.7) </td><td>66 (31.1) </td></tr><tr><td>chr10 </td><td>64.8 </td><td>255 </td><td>14 </td><td>35 </td><td>206 </td><td>131 (63.6) </td><td>111 (53.9) </td><td>11 (5.3) </td></tr><tr><td>chr11 </td><td>53.4 </td><td>214 </td><td>8 </td><td>35 </td><td>171 </td><td>80 (46.8) </td><td>123 (71.9) </td><td>109 (63.7) </td></tr><tr><td>chr12 </td><td>65.5 </td><td>262 </td><td>10 </td><td>86 </td><td>166 </td><td>116 (69.9) </td><td>40 (24.1) </td><td>30 (18.1) </td></tr><tr><td>Total </td><td>759.8 </td><td>3029 </td><td>272 </td><td>485 </td><td>2272 </td><td>1400 (61.6) </td><td>1028 (45.2) </td><td>717 (31.6) </td></tr></tbody></table></div><div content-id="DSU008TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Chromosome </th><th rowspan="2">Sequence length (∼Mb)<sup>a</sup> </th><th rowspan="2">No. of primers designed </th><th rowspan="2">No. of primers without PCR amplification </th><th rowspan="2">No. of primers without polymorphism </th><th rowspan="2">No. of primers examined </th><th colspan="3">No. (percentage) of polymorphic InDels<hr /> </th></tr><tr><th><em>S. pimpinellifolium</em> </th><th><em>S. lycopersicum</em> var. <em>cerasiforme</em> </th><th><em>S. lycopersicum</em> </th></tr></thead><tbody><tr><td>chr01 </td><td>90.3 </td><td>362 </td><td>63 </td><td>89 </td><td>210 </td><td>132 (62.9) </td><td>38 (18.1) </td><td>22 (10.5) </td></tr><tr><td>chr02 </td><td>49.9 </td><td>207 </td><td>10 </td><td>22 </td><td>175 </td><td>98 (56.0) </td><td>134 (76.6) </td><td>75 (42.9) </td></tr><tr><td>chr03 </td><td>64.8 </td><td>254 </td><td>19 </td><td>31 </td><td>204 </td><td>123 (60.3) </td><td>128 (62.7) </td><td>32 (15.7) </td></tr><tr><td>chr04 </td><td>64.1 </td><td>254 </td><td>12 </td><td>24 </td><td>218 </td><td>120 (55.0) </td><td>128 (58.7) </td><td>144 (66.1) </td></tr><tr><td>chr05 </td><td>65.0 </td><td>262 </td><td>33 </td><td>53 </td><td>176 </td><td>112 (63.6) </td><td>125 (71.0) </td><td>127 (72.2) </td></tr><tr><td>chr06 </td><td>46.0 </td><td>181 </td><td>20 </td><td>40 </td><td>121 </td><td>99 (81.8) </td><td>83 (68.6) </td><td>73 (60.3) </td></tr><tr><td>chr07 </td><td>65.3 </td><td>259 </td><td>39 </td><td>15 </td><td>205 </td><td>94 (45.9) </td><td>26 (12.7) </td><td>17 (8.3) </td></tr><tr><td>chr08 </td><td>63.0 </td><td>252 </td><td>23 </td><td>21 </td><td>208 </td><td>160 (76.9) </td><td>12 (5.8) </td><td>11 (5.3) </td></tr><tr><td>chr09 </td><td>67.7 </td><td>267 </td><td>21 </td><td>34 </td><td>212 </td><td>135 (63.7) </td><td>80 (37.7) </td><td>66 (31.1) </td></tr><tr><td>chr10 </td><td>64.8 </td><td>255 </td><td>14 </td><td>35 </td><td>206 </td><td>131 (63.6) </td><td>111 (53.9) </td><td>11 (5.3) </td></tr><tr><td>chr11 </td><td>53.4 </td><td>214 </td><td>8 </td><td>35 </td><td>171 </td><td>80 (46.8) </td><td>123 (71.9) </td><td>109 (63.7) </td></tr><tr><td>chr12 </td><td>65.5 </td><td>262 </td><td>10 </td><td>86 </td><td>166 </td><td>116 (69.9) </td><td>40 (24.1) </td><td>30 (18.1) </td></tr><tr><td>Total </td><td>759.8 </td><td>3029 </td><td>272 </td><td>485 </td><td>2272 </td><td>1400 (61.6) </td><td>1028 (45.2) </td><td>717 (31.6) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn1"><div class="fn"><p><sup>a</sup>The sequenced genome size was obtained from Sato <em>et al.</em><sup><a reveal-id="DSU008C34" data-open="DSU008C34" class="link link-ref link-reveal xref-bibr">34</a></sup></p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6690660" target="_blank">View Large</a></div></div><p>To design primers for PCR validation of InDels, flanking sequences of 100 bp for each side of candidate InDels were extracted. Primers were designed using local Primer3<sup><a reveal-id="DSU008C38" data-open="DSU008C38" class="link link-ref link-reveal xref-bibr">38</a></sup> downloaded from Sourceforge (<a class="link link-uri" href="http://sourceforge.net/project/showfiles.php?group_id=112461">http://sourceforge.net/project/showfiles.php?group_id=112461</a>, 19 February 2014, date last accessed) with PCR product length 100–200 bp and the optimal length of primer sequence of 20 bp. Several primer pairs were designed for each InDel. The best primer pair was selected based on the optimal GC content of 40–60% and the difference of GC content between forward and reverse primers &lt;10%. All the process was carried out using custom PERL scripts. Primers were synthesized at Sunbiotech Company (Beijing, China) or Sangong Company (Beijing, China).</p>                            <h3 scrollto-destination=6690662 id="6690662" class="section-title" data-legacy-id=s2d>2.4. Validation of InDels using PCR</h3>
<p>The PCR technique was adapted to validate the candidate InDels. All synthesized primers were first used to amplify genomic DNA of tomato lines LA 1589 and Heinz 1706. Only primers that successfully amplified a product and had length polymorphisms were then used to detect polymorphisms in the 22 tomato genotypes.</p><p>All PCRs were done in 10-μl reaction volume using the method described in Wei <em>et al.</em><sup><a reveal-id="DSU008C39" data-open="DSU008C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Reactions were heated at 95°C for 5 min, followed by 32 cycles of 30 s at 95°C, 30 s at 50–60°C depending on the <em>T</em><sub>m</sub> values of primer pairs, and 30 s at 72°C, with a final extension of 5 min at 72°C. The PCR products were subsequently separated in 8% polyacrylamide gel and visualized using the silver-staining approach.<sup><a reveal-id="DSU008C17" data-open="DSU008C17" class="link link-ref link-reveal xref-bibr">17</a></sup></p>                            <h3 scrollto-destination=6690665 id="6690665" class="section-title" data-legacy-id=s2e>2.5. Data collection and analysis</h3>
<p>The presence or absence of each allele for each InDel was coded by 1 or 0, respectively, and scored for a binary data matrix. Allele frequency of each InDel marker was calculated for each genotype. Nei's genetic distance<sup><a reveal-id="DSU008C40" data-open="DSU008C40" class="link link-ref link-reveal xref-bibr">40</a></sup> was calculated for each pair of tomato genotypes using the programme in the software package PHYLIP 3.695 (<a class="link link-uri" href="http://evolution.genetics.washington.edu/phylip.html">http://evolution.genetics.washington.edu/phylip.html</a>, 19 February 2014, date last accessed). An Unweighted Pair Group Method with Arithmetic Mean (UPGMA) cluster analysis was performed to develop a dendrogram.</p><p>The occurrences of InDels in coding regions of genes were examined by blasting the flanking sequences of 100 bp for each side of the InDel against the tomato ITAG2.3_cds.fasta downloaded from SGN using a PERL script.</p>                            <h2 scrollto-destination=6690668 id="6690668" class="section-title" data-legacy-id=s3>3. Result</h2>
                            <h3 scrollto-destination=6690669 id="6690669" class="section-title" data-legacy-id=s3a>3.1. Candidate InDels between LA 1589 and Heinz 1706</h3>
<p>A total of 145 695 candidate InDels were identified between the genome sequences of Heinz 1706 and LA 1589, of which 65 619 were insertions and 80 076 were deletions in Heinz 1706 (Table <a reveal-id="DSU008TB3" data-open="DSU008TB3" class="link link-reveal link-table xref-fig">3</a>). The average size of predicted InDels was 4.1 bp with a range of 1–94 bp, of which ∼54.0% were 1 bp, 42.3% were 2–20 bp, and 3.7% were longer than 20 bp. The average density of InDels was one per 5.22 kb with a range of 4.33–6.72 kb on 12 chromosomes. The highest density was on chromosome 6 and the lowest density was on chromosome 12 (Table <a reveal-id="DSU008TB3" data-open="DSU008TB3" class="link link-reveal link-table xref-fig">3</a>). The least difference of numbers for InDels between 1 bp and &gt;1 bp was observed on chromosome 2 (101), while the largest was on chromosome 10 (1496). </p>                        <a id="6690671" scrollto-destination="6690671"></a>
<div content-id="DSU008TB3" class="table-modal"><div class="table-wrap"><div id="DSU008TB3" class="table-wrap-title" data-id="DSU008TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Predicted number and frequency of InDels between Heinz 1706 and LA 1589</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Chromosome </th><th colspan="3">No. of predicted InDels<hr /> </th><th colspan="3">Frequency of InDels (kb/InDel)<hr /> </th></tr><tr><th>Total </th><th>1 bp </th><th>&gt;1 bp </th><th>Total </th><th>1 bp </th><th>&gt;1 bp </th></tr></thead><tbody><tr><td>chr01 </td><td>16 547 </td><td>8777 </td><td>7770 </td><td>5.46 </td><td>10.29 </td><td>11.62 </td></tr><tr><td>chr02 </td><td>10 695 </td><td>5398 </td><td>5297 </td><td>4.67 </td><td>9.24 </td><td>9.42 </td></tr><tr><td>chr03 </td><td>12 842 </td><td>6779 </td><td>6063 </td><td>5.05 </td><td>9.56 </td><td>10.69 </td></tr><tr><td>chr04 </td><td>11 495 </td><td>6112 </td><td>5383 </td><td>5.58 </td><td>10.49 </td><td>11.91 </td></tr><tr><td>chr05 </td><td>12 148 </td><td>6816 </td><td>5332 </td><td>5.35 </td><td>9.54 </td><td>12.19 </td></tr><tr><td>chr06 </td><td>10 619 </td><td>5540 </td><td>5079 </td><td>4.33 </td><td>8.30 </td><td>9.06 </td></tr><tr><td>chr07 </td><td>13 426 </td><td>7386 </td><td>6040 </td><td>4.86 </td><td>8.84 </td><td>10.81 </td></tr><tr><td>chr08 </td><td>13 776 </td><td>7591 </td><td>6185 </td><td>4.57 </td><td>8.30 </td><td>10.19 </td></tr><tr><td>chr09 </td><td>11 417 </td><td>6251 </td><td>5166 </td><td>5.93 </td><td>10.83 </td><td>13.10 </td></tr><tr><td>chr10 </td><td>13 390 </td><td>7443 </td><td>5947 </td><td>4.84 </td><td>8.71 </td><td>10.90 </td></tr><tr><td>chr11 </td><td>9587 </td><td>5221 </td><td>4366 </td><td>5.57 </td><td>10.23 </td><td>12.23 </td></tr><tr><td>chr12 </td><td>9753 </td><td>5432 </td><td>4321 </td><td>6.72 </td><td>12.06 </td><td>15.16 </td></tr><tr><td>Total </td><td>145 695 </td><td>78 746 </td><td>66 949 </td><td> </td><td> </td><td> </td></tr><tr><td>Average </td><td> </td><td> </td><td> </td><td>5.22 </td><td>9.65 </td><td>11.35 </td></tr></tbody></table></div><div content-id="DSU008TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Chromosome </th><th colspan="3">No. of predicted InDels<hr /> </th><th colspan="3">Frequency of InDels (kb/InDel)<hr /> </th></tr><tr><th>Total </th><th>1 bp </th><th>&gt;1 bp </th><th>Total </th><th>1 bp </th><th>&gt;1 bp </th></tr></thead><tbody><tr><td>chr01 </td><td>16 547 </td><td>8777 </td><td>7770 </td><td>5.46 </td><td>10.29 </td><td>11.62 </td></tr><tr><td>chr02 </td><td>10 695 </td><td>5398 </td><td>5297 </td><td>4.67 </td><td>9.24 </td><td>9.42 </td></tr><tr><td>chr03 </td><td>12 842 </td><td>6779 </td><td>6063 </td><td>5.05 </td><td>9.56 </td><td>10.69 </td></tr><tr><td>chr04 </td><td>11 495 </td><td>6112 </td><td>5383 </td><td>5.58 </td><td>10.49 </td><td>11.91 </td></tr><tr><td>chr05 </td><td>12 148 </td><td>6816 </td><td>5332 </td><td>5.35 </td><td>9.54 </td><td>12.19 </td></tr><tr><td>chr06 </td><td>10 619 </td><td>5540 </td><td>5079 </td><td>4.33 </td><td>8.30 </td><td>9.06 </td></tr><tr><td>chr07 </td><td>13 426 </td><td>7386 </td><td>6040 </td><td>4.86 </td><td>8.84 </td><td>10.81 </td></tr><tr><td>chr08 </td><td>13 776 </td><td>7591 </td><td>6185 </td><td>4.57 </td><td>8.30 </td><td>10.19 </td></tr><tr><td>chr09 </td><td>11 417 </td><td>6251 </td><td>5166 </td><td>5.93 </td><td>10.83 </td><td>13.10 </td></tr><tr><td>chr10 </td><td>13 390 </td><td>7443 </td><td>5947 </td><td>4.84 </td><td>8.71 </td><td>10.90 </td></tr><tr><td>chr11 </td><td>9587 </td><td>5221 </td><td>4366 </td><td>5.57 </td><td>10.23 </td><td>12.23 </td></tr><tr><td>chr12 </td><td>9753 </td><td>5432 </td><td>4321 </td><td>6.72 </td><td>12.06 </td><td>15.16 </td></tr><tr><td>Total </td><td>145 695 </td><td>78 746 </td><td>66 949 </td><td> </td><td> </td><td> </td></tr><tr><td>Average </td><td> </td><td> </td><td> </td><td>5.22 </td><td>9.65 </td><td>11.35 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6690671" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU008TB3" class="table-wrap-title" data-id="DSU008TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Predicted number and frequency of InDels between Heinz 1706 and LA 1589</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Chromosome </th><th colspan="3">No. of predicted InDels<hr /> </th><th colspan="3">Frequency of InDels (kb/InDel)<hr /> </th></tr><tr><th>Total </th><th>1 bp </th><th>&gt;1 bp </th><th>Total </th><th>1 bp </th><th>&gt;1 bp </th></tr></thead><tbody><tr><td>chr01 </td><td>16 547 </td><td>8777 </td><td>7770 </td><td>5.46 </td><td>10.29 </td><td>11.62 </td></tr><tr><td>chr02 </td><td>10 695 </td><td>5398 </td><td>5297 </td><td>4.67 </td><td>9.24 </td><td>9.42 </td></tr><tr><td>chr03 </td><td>12 842 </td><td>6779 </td><td>6063 </td><td>5.05 </td><td>9.56 </td><td>10.69 </td></tr><tr><td>chr04 </td><td>11 495 </td><td>6112 </td><td>5383 </td><td>5.58 </td><td>10.49 </td><td>11.91 </td></tr><tr><td>chr05 </td><td>12 148 </td><td>6816 </td><td>5332 </td><td>5.35 </td><td>9.54 </td><td>12.19 </td></tr><tr><td>chr06 </td><td>10 619 </td><td>5540 </td><td>5079 </td><td>4.33 </td><td>8.30 </td><td>9.06 </td></tr><tr><td>chr07 </td><td>13 426 </td><td>7386 </td><td>6040 </td><td>4.86 </td><td>8.84 </td><td>10.81 </td></tr><tr><td>chr08 </td><td>13 776 </td><td>7591 </td><td>6185 </td><td>4.57 </td><td>8.30 </td><td>10.19 </td></tr><tr><td>chr09 </td><td>11 417 </td><td>6251 </td><td>5166 </td><td>5.93 </td><td>10.83 </td><td>13.10 </td></tr><tr><td>chr10 </td><td>13 390 </td><td>7443 </td><td>5947 </td><td>4.84 </td><td>8.71 </td><td>10.90 </td></tr><tr><td>chr11 </td><td>9587 </td><td>5221 </td><td>4366 </td><td>5.57 </td><td>10.23 </td><td>12.23 </td></tr><tr><td>chr12 </td><td>9753 </td><td>5432 </td><td>4321 </td><td>6.72 </td><td>12.06 </td><td>15.16 </td></tr><tr><td>Total </td><td>145 695 </td><td>78 746 </td><td>66 949 </td><td> </td><td> </td><td> </td></tr><tr><td>Average </td><td> </td><td> </td><td> </td><td>5.22 </td><td>9.65 </td><td>11.35 </td></tr></tbody></table></div><div content-id="DSU008TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Chromosome </th><th colspan="3">No. of predicted InDels<hr /> </th><th colspan="3">Frequency of InDels (kb/InDel)<hr /> </th></tr><tr><th>Total </th><th>1 bp </th><th>&gt;1 bp </th><th>Total </th><th>1 bp </th><th>&gt;1 bp </th></tr></thead><tbody><tr><td>chr01 </td><td>16 547 </td><td>8777 </td><td>7770 </td><td>5.46 </td><td>10.29 </td><td>11.62 </td></tr><tr><td>chr02 </td><td>10 695 </td><td>5398 </td><td>5297 </td><td>4.67 </td><td>9.24 </td><td>9.42 </td></tr><tr><td>chr03 </td><td>12 842 </td><td>6779 </td><td>6063 </td><td>5.05 </td><td>9.56 </td><td>10.69 </td></tr><tr><td>chr04 </td><td>11 495 </td><td>6112 </td><td>5383 </td><td>5.58 </td><td>10.49 </td><td>11.91 </td></tr><tr><td>chr05 </td><td>12 148 </td><td>6816 </td><td>5332 </td><td>5.35 </td><td>9.54 </td><td>12.19 </td></tr><tr><td>chr06 </td><td>10 619 </td><td>5540 </td><td>5079 </td><td>4.33 </td><td>8.30 </td><td>9.06 </td></tr><tr><td>chr07 </td><td>13 426 </td><td>7386 </td><td>6040 </td><td>4.86 </td><td>8.84 </td><td>10.81 </td></tr><tr><td>chr08 </td><td>13 776 </td><td>7591 </td><td>6185 </td><td>4.57 </td><td>8.30 </td><td>10.19 </td></tr><tr><td>chr09 </td><td>11 417 </td><td>6251 </td><td>5166 </td><td>5.93 </td><td>10.83 </td><td>13.10 </td></tr><tr><td>chr10 </td><td>13 390 </td><td>7443 </td><td>5947 </td><td>4.84 </td><td>8.71 </td><td>10.90 </td></tr><tr><td>chr11 </td><td>9587 </td><td>5221 </td><td>4366 </td><td>5.57 </td><td>10.23 </td><td>12.23 </td></tr><tr><td>chr12 </td><td>9753 </td><td>5432 </td><td>4321 </td><td>6.72 </td><td>12.06 </td><td>15.16 </td></tr><tr><td>Total </td><td>145 695 </td><td>78 746 </td><td>66 949 </td><td> </td><td> </td><td> </td></tr><tr><td>Average </td><td> </td><td> </td><td> </td><td>5.22 </td><td>9.65 </td><td>11.35 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6690671" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6690672 id="6690672" class="section-title" data-legacy-id=s3b>3.2. Number of primers designed and success of PCR amplification</h3>
<p>Using the approach described in the section ‘Selection of InDels for validation and primer design’ of Materials and methods, 3029 candidate InDels were selected and primers were designed for PCR validation (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The average physical distance between two adjacent InDels was 250 kb with a range of 241 (chromosome 2) to 255 kb (chromosome 3) on 12 chromosomes. PCR results showed that 272 primer pairs could not generate PCR products from the genomic DNA of both Heinz 1706 and LA 1589 (Table <a reveal-id="DSU008TB2" data-open="DSU008TB2" class="link link-reveal link-table xref-fig">2</a>) . The PCR success rate was 91.0%, which was consistent with our previous finding of 91.9% for PCR amplification of genomic DNA in tomato.<sup><a reveal-id="DSU008C9" data-open="DSU008C9" class="link link-ref link-reveal xref-bibr">9</a></sup> The InDel sizes of PCR products amplified by most primer pairs (98.5%) were as predicted. However, 23 primer pairs showed smaller and 10 primer pairs showed larger sizes than predicted (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). In addition, 485 primer pairs did not show detectable polymorphisms between Heinz 1706 and LA 1589 (Table <a reveal-id="DSU008TB2" data-open="DSU008TB2" class="link link-reveal link-table xref-fig">2</a>). The InDel sizes between 6 and 30 bp had a high percentage (83.6%) of polymorphism validation, while InDels with sizes of &lt;6 bp and &gt;30 bp received 78.3 or 43.3% polymorphism validation, respectively. Particularly, only one of five InDels was validated when the size was &gt;50 bp (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). The primer pairs with PCR failure or non-detectable polymorphisms were excluded, and the remaining 2272 primer pairs were used for subsequent analysis. Therefore, the actual average distance between two adjacent InDels was 334 kb with a range of 285 (chromosome 2) to 430 kb (chromosome 1) on 12 chromosomes.</p><p>The 2272 InDel markers generated 5025 alleles in the whole collection of 22 tomato genotypes. The number of alleles generated for all InDels varied from 2 to 8 with an average of 2.2. Among the polymorphic InDels, most (85.3%) had two alleles, 10.7% had three alleles, and 2.7% had four alleles (Fig. <a reveal-id="DSU008F1" data-open="DSU008F1" class="link link-reveal link-table xref-fig">1</a>). Only three and two markers had seven and eight alleles, respectively. Similarly, 84.9% polymorphic InDels in <em>S. pimpinellifolium</em>, 94.7% in <em>S. lycopersicum</em> var. <em>cerasiforme</em>, and 95.8% in <em>S. lycopersicum</em> had two alleles (Fig. <a reveal-id="DSU008F1" data-open="DSU008F1" class="link link-reveal link-table xref-fig">1</a>). </p>                        <a id="6690675" scrollto-destination="6690675"></a>
<div id="DSU008F1" data-id="DSU008F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/m_dsu00801.png?Expires=2147483647&Signature=parOeQ~Glg0~th-De0nkh0H6ppMPY9JmYUgTwYKQ9P6zj5bHlbn3x~J5rUKUvpp19grRT2CWX4rUkX1HyWTOzPglIwG9yz8vr4l-8XHyWL~Fx3cN~aYaZhgx~re~8El0UMo6v~Xmuwb2FslrJSo-BBls42TJ7IT5YMB7eVAZkueOybsCV4NWlFdgGm3jVT1KxuXyxMDClA-tVZ5z~NxUSMRS~7F2OzTxZMlcmf5iC9GsnqZe~A1D6FFqt~MzgTFmAMuMUzy1i5S2a0iPXN3EnLIDebRasuY9FqA7L-nZKmzh6DQ8MHwAloZ-O6Ghc5IqjSMUTou3yrE3-gxALo1CCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Frequency distribution of InDels (≥2 bp) in Solanum pimpinellifolium, S. lycopersicum var. cerasiforme, and S. lycopersicum." path-from-xml="dsu00801.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6690675/dsu00801.tiff" path-from-xml="dsu00801.tiff" target="_blank">View large</a><a section="6690675" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/dsu00801.png?Expires=2147483647&Signature=yoPn-pTvxC~CV77gTA6I3kvN0x~D8~18prTyQ-V4katRVwfbdPMBEaFGuzTjN7z7bOgAJPk1QiJ8wpYmcAOs0matDOSW8MsZ0D1ndWrQTkdQmHttpVUwKdx9HmUas1Gk70zqfy9SA-4EkTl6td08lsJSRMnQ6swSp4dbx8VcEMONndbRxfz3L7aQPPGXp0VzHS5Zvo0UBL9ACbQgP-3wpPP-r78q4vcY~7lqMLzsoPv~p4mUhOCUmHhvJJSMvJo7p--rx6~iDqTHeXQ45jNeXtb1RtZ3bWlzEITpAsFpqUzrea78M~n7mcU1GxtO-QzKr5S~hthJtnYHhNa794i1rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6690675&ar=447212&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00801.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Frequency distribution of InDels (≥2 bp) in <em>Solanum pimpinellifolium</em>, <em>S. lycopersicum</em> var. <em>cerasiforme</em>, and <em>S. lycopersicum</em>.</p></div></div><div content-id="DSU008F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/m_dsu00801.png?Expires=2147483647&Signature=parOeQ~Glg0~th-De0nkh0H6ppMPY9JmYUgTwYKQ9P6zj5bHlbn3x~J5rUKUvpp19grRT2CWX4rUkX1HyWTOzPglIwG9yz8vr4l-8XHyWL~Fx3cN~aYaZhgx~re~8El0UMo6v~Xmuwb2FslrJSo-BBls42TJ7IT5YMB7eVAZkueOybsCV4NWlFdgGm3jVT1KxuXyxMDClA-tVZ5z~NxUSMRS~7F2OzTxZMlcmf5iC9GsnqZe~A1D6FFqt~MzgTFmAMuMUzy1i5S2a0iPXN3EnLIDebRasuY9FqA7L-nZKmzh6DQ8MHwAloZ-O6Ghc5IqjSMUTou3yrE3-gxALo1CCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Frequency distribution of InDels (≥2 bp) in Solanum pimpinellifolium, S. lycopersicum var. cerasiforme, and S. lycopersicum." path-from-xml="dsu00801.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6690675/dsu00801.tiff" path-from-xml="dsu00801.tiff" target="_blank">View large</a><a section="6690675" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/dsu00801.png?Expires=2147483647&Signature=yoPn-pTvxC~CV77gTA6I3kvN0x~D8~18prTyQ-V4katRVwfbdPMBEaFGuzTjN7z7bOgAJPk1QiJ8wpYmcAOs0matDOSW8MsZ0D1ndWrQTkdQmHttpVUwKdx9HmUas1Gk70zqfy9SA-4EkTl6td08lsJSRMnQ6swSp4dbx8VcEMONndbRxfz3L7aQPPGXp0VzHS5Zvo0UBL9ACbQgP-3wpPP-r78q4vcY~7lqMLzsoPv~p4mUhOCUmHhvJJSMvJo7p--rx6~iDqTHeXQ45jNeXtb1RtZ3bWlzEITpAsFpqUzrea78M~n7mcU1GxtO-QzKr5S~hthJtnYHhNa794i1rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6690675&ar=447212&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00801.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Frequency distribution of InDels (≥2 bp) in <em>Solanum pimpinellifolium</em>, <em>S. lycopersicum</em> var. <em>cerasiforme</em>, and <em>S. lycopersicum</em>.</p></div></div>                            <h3 scrollto-destination=6690676 id="6690676" class="section-title" data-legacy-id=s3c>3.3. Marker polymorphisms and distribution among three tomato species</h3>
<p>Of the 5025 alleles amplified by 2272 InDel markers, 1930 were shared by all three species. The total number of alleles in each species reduced from 3941 in <em>S. pimpinellifolium</em> to 3431 in <em>S. lycopersicum</em> var. <em>cerasiforme</em> and 3110 in <em>S. lycopersicum</em> (Fig. <a reveal-id="DSU008F2" data-open="DSU008F2" class="link link-reveal link-table xref-fig">2</a>). The number of alleles unique to each species also dramatically decreased from 1382 in <em>S. pimpinellifolium</em> to 56 in <em>S. lycopersicum</em> var. <em>cerasiforme</em> and 60 in <em>S. lycopersicum. Solanum pimpinellifolium</em> shared more alleles with <em>S. lycopersicum</em> var. <em>cerasiforme</em> than with <em>S. lycopersicum</em>. </p>                        <a id="6690678" scrollto-destination="6690678"></a>
<div id="DSU008F2" data-id="DSU008F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/m_dsu00802.png?Expires=2147483647&Signature=YL~Ql1iaerH2eVGKUzNq6D-x~IemeZMJkRPWhb~SjgTLzSYmamzyMa7MgWiM7erhGQAvYZXqPXYoGT5JluTbm7e~6CPT4GkuNPSvwe9HGtH6FfSpPp8HdXNF0TYqwA6V5voUX34dVPBhSWAMSv34E23tWU8Bx1yEP4QNBxInV97gRa3ulxSRlmjAD5ToQCu~gdwkOUR0eJ0p3kniS26aNpK8c0kEb3vAn7KJg5rOVih3LGo~ddqAZ8Gv1qKYilVOIcwjDtlRNFUjqb-71bN2-pYkyi76hj-bhhaZBbf15p5o-Vy2WCoG-r033F3RGJjL~KntyLnv6n8XUX7kNDUJAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagram shows the proportion of common alleles among Solanum pimpinellifolium, S. lycopersicum var. cerasiforme, and S. lycopersicum. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu00802.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6690678/dsu00802.tiff" path-from-xml="dsu00802.tiff" target="_blank">View large</a><a section="6690678" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/dsu00802.png?Expires=2147483647&Signature=r02bg1KiEMJgeZwsvCnLYuzVQ1Q2rjR8sPAygIYXsGl1F6Ypkw5pghvQJ5L4K3zUZqOD9VHIPfuQWBkC-4F1hmhl4kSPBEexDrOJwqM9t9-Mg-M2LM5DbYbcs~2NW0hIEmdEWUk7iOv~bVMwHLb9lAWuV0oZFALDDvo1ENif4vrW4zkXR13bKmqVLI6cjVeDTyf7bs1Prk1qc9lazRd9CafJn9104UhM7r-Bxi69kmM7mnm-3svLZ-sSLuvRIqVK5yrMHgx2ax-mggApKCpJgbBgO0HvIKczQd-Wf-VjMElRUSvsqMa137KTMk55m1dHqL7eaTOjaSM1MzIlV9FUog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6690678&ar=447212&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00802.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Venn diagram shows the proportion of common alleles among <em>Solanum pimpinellifolium</em>, <em>S. lycopersicum</em> var. <em>cerasiforme</em>, and <em>S. lycopersicum</em><strong>.</strong> This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><div content-id="DSU008F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/m_dsu00802.png?Expires=2147483647&Signature=YL~Ql1iaerH2eVGKUzNq6D-x~IemeZMJkRPWhb~SjgTLzSYmamzyMa7MgWiM7erhGQAvYZXqPXYoGT5JluTbm7e~6CPT4GkuNPSvwe9HGtH6FfSpPp8HdXNF0TYqwA6V5voUX34dVPBhSWAMSv34E23tWU8Bx1yEP4QNBxInV97gRa3ulxSRlmjAD5ToQCu~gdwkOUR0eJ0p3kniS26aNpK8c0kEb3vAn7KJg5rOVih3LGo~ddqAZ8Gv1qKYilVOIcwjDtlRNFUjqb-71bN2-pYkyi76hj-bhhaZBbf15p5o-Vy2WCoG-r033F3RGJjL~KntyLnv6n8XUX7kNDUJAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagram shows the proportion of common alleles among Solanum pimpinellifolium, S. lycopersicum var. cerasiforme, and S. lycopersicum. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu00802.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6690678/dsu00802.tiff" path-from-xml="dsu00802.tiff" target="_blank">View large</a><a section="6690678" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/dsu00802.png?Expires=2147483647&Signature=r02bg1KiEMJgeZwsvCnLYuzVQ1Q2rjR8sPAygIYXsGl1F6Ypkw5pghvQJ5L4K3zUZqOD9VHIPfuQWBkC-4F1hmhl4kSPBEexDrOJwqM9t9-Mg-M2LM5DbYbcs~2NW0hIEmdEWUk7iOv~bVMwHLb9lAWuV0oZFALDDvo1ENif4vrW4zkXR13bKmqVLI6cjVeDTyf7bs1Prk1qc9lazRd9CafJn9104UhM7r-Bxi69kmM7mnm-3svLZ-sSLuvRIqVK5yrMHgx2ax-mggApKCpJgbBgO0HvIKczQd-Wf-VjMElRUSvsqMa137KTMk55m1dHqL7eaTOjaSM1MzIlV9FUog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6690678&ar=447212&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00802.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Venn diagram shows the proportion of common alleles among <em>Solanum pimpinellifolium</em>, <em>S. lycopersicum</em> var. <em>cerasiforme</em>, and <em>S. lycopersicum</em><strong>.</strong> This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><p>Pairwise comparisons revealed that almost all InDel markers were polymorphic between <em>S. pimpinellifolium</em> and <em>S. lycopersicum</em> var. <em>cerasiforme</em> or <em>S. lycopersicum</em>. However, the proportion of polymorphic InDels reduced to 53.0% between <em>S. lycopersicum</em> var. <em>cerasiforme</em> and <em>S. lycopersicum</em>. There were 0.1–20.7% InDels had alleles alternatively fixed in paired species. In addition, 18.5–26.9% InDels had alleles shared by paired species. Proportions of InDels with alleles specific to one certain species varied from 6.1 to 44.0% (Fig. <a reveal-id="DSU008F3" data-open="DSU008F3" class="link link-reveal link-table xref-fig">3</a>). The proportion of polymorphic InDels was 61.4–100.0% (average 84.6%) between any accession in <em>S. pimpinellifolium</em> and any genotype in <em>S. lycopersicum</em>, 55.3–93.8% (average 71.5%) between any accession in <em>S. pimpinellifolium</em> and any line in <em>S. lycopersicum</em> var. <em>cerasiforme</em>, and 7.7–33.9% (average 19.2%) between any line in <em>S. lycopersicum</em> var. <em>cerasiforme</em> and any genotype in <em>S. lycopersicum</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). </p>                        <a id="6690680" scrollto-destination="6690680"></a>
<div id="DSU008F3" data-id="DSU008F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/m_dsu00803.png?Expires=2147483647&Signature=SahLhAHPEUKAjH8FmJw6sLR0YLqrbqGokygGzhlNYRKTJ8DdG~ewr5qt8alfzVxgHHYttAz2Ywe~f4f7k~sNHs~mrAw19MNWrhFifjk2fJ-rQfS74dmJS2c21BBEw2qhzjdcqqFS5tyuc3xX0hOdQab6k9qQazB0ekw3O5RvAtrlzNw6pjIpV~MSWjhmiC96E6rnBUKKiop3LIpjlGGNRkFI78BctcQ2HMbrBhN5f37jGVfjbh8oCTjTFaXT00k8i6qnxfIL9XLXSF4HHyDdIWDBzTEcaLqfD218nwWSxHNIRiWNmM05jbiPmB6fPPSwUAAZIXkUV4jeYgW8j-Xqfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Pairwise comparisons of allelic variation among Solanum pimpinellifolium, S. lycopersicum var. cerasiforme, and S. lycopersicum. Pie diagrams show the proportion of 2272 InDels that fell into five categories: (1) InDels where a monomorphic allele was shared by all members in the two species; (2) InDels where alleles were found among the members of the two species; (3) InDels where a unique allele was found among members of the first species listed, whereas an alternative allele (found in both groups) was fixed in the second species; (4) InDels where a unique allele was found among members of the second species listed, whereas an alternative allele (found in both species) was fixed in the first species; (5) InDels where the two species were fixed for alternative alleles." path-from-xml="dsu00803.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6690680/dsu00803.tiff" path-from-xml="dsu00803.tiff" target="_blank">View large</a><a section="6690680" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/dsu00803.png?Expires=2147483647&Signature=fYj5fToeIav0MWJt31LOFKe5grL87tuZbfAhJAtX0t985r0zJEIY3p7UIlFyVF1uzxiIvuGuDqIliSv9YKzbTjij6NsNbFV2YGAr3XMeLhZvm9uvMpyCyyPFK9A0al73YiPXYPExJRbo7lV1ZLJZvhcabGg~z8B~PkpIXNHhN~Qi3H254X6oiBPKPPStSMWlMdO281ufHPozUQjXHs6-4H~eyrLJSMwAUn1wD9U~Alnet7q3QJgamgbin0Nb8J3C5zY-A1LwkJvE-PhFFfzCyO1EWjwYdDgt1kuP5ZYnSfscqECB-DZh9dJUkrLUddVil~o9Av3FBwfhlCBINvLxDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6690680&ar=447212&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00803.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Pairwise comparisons of allelic variation among <em>Solanum pimpinellifolium</em>, <em>S. lycopersicum</em> var. <em>cerasiforme</em>, and <em>S. lycopersicum</em>. Pie diagrams show the proportion of 2272 InDels that fell into five categories: (1) InDels where a monomorphic allele was shared by all members in the two species; (2) InDels where alleles were found among the members of the two species; (3) InDels where a unique allele was found among members of the first species listed, whereas an alternative allele (found in both groups) was fixed in the second species; (4) InDels where a unique allele was found among members of the second species listed, whereas an alternative allele (found in both species) was fixed in the first species; (5) InDels where the two species were fixed for alternative alleles.</p></div></div><div content-id="DSU008F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/m_dsu00803.png?Expires=2147483647&Signature=SahLhAHPEUKAjH8FmJw6sLR0YLqrbqGokygGzhlNYRKTJ8DdG~ewr5qt8alfzVxgHHYttAz2Ywe~f4f7k~sNHs~mrAw19MNWrhFifjk2fJ-rQfS74dmJS2c21BBEw2qhzjdcqqFS5tyuc3xX0hOdQab6k9qQazB0ekw3O5RvAtrlzNw6pjIpV~MSWjhmiC96E6rnBUKKiop3LIpjlGGNRkFI78BctcQ2HMbrBhN5f37jGVfjbh8oCTjTFaXT00k8i6qnxfIL9XLXSF4HHyDdIWDBzTEcaLqfD218nwWSxHNIRiWNmM05jbiPmB6fPPSwUAAZIXkUV4jeYgW8j-Xqfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Pairwise comparisons of allelic variation among Solanum pimpinellifolium, S. lycopersicum var. cerasiforme, and S. lycopersicum. Pie diagrams show the proportion of 2272 InDels that fell into five categories: (1) InDels where a monomorphic allele was shared by all members in the two species; (2) InDels where alleles were found among the members of the two species; (3) InDels where a unique allele was found among members of the first species listed, whereas an alternative allele (found in both groups) was fixed in the second species; (4) InDels where a unique allele was found among members of the second species listed, whereas an alternative allele (found in both species) was fixed in the first species; (5) InDels where the two species were fixed for alternative alleles." path-from-xml="dsu00803.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6690680/dsu00803.tiff" path-from-xml="dsu00803.tiff" target="_blank">View large</a><a section="6690680" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/dsu00803.png?Expires=2147483647&Signature=fYj5fToeIav0MWJt31LOFKe5grL87tuZbfAhJAtX0t985r0zJEIY3p7UIlFyVF1uzxiIvuGuDqIliSv9YKzbTjij6NsNbFV2YGAr3XMeLhZvm9uvMpyCyyPFK9A0al73YiPXYPExJRbo7lV1ZLJZvhcabGg~z8B~PkpIXNHhN~Qi3H254X6oiBPKPPStSMWlMdO281ufHPozUQjXHs6-4H~eyrLJSMwAUn1wD9U~Alnet7q3QJgamgbin0Nb8J3C5zY-A1LwkJvE-PhFFfzCyO1EWjwYdDgt1kuP5ZYnSfscqECB-DZh9dJUkrLUddVil~o9Av3FBwfhlCBINvLxDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6690680&ar=447212&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00803.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Pairwise comparisons of allelic variation among <em>Solanum pimpinellifolium</em>, <em>S. lycopersicum</em> var. <em>cerasiforme</em>, and <em>S. lycopersicum</em>. Pie diagrams show the proportion of 2272 InDels that fell into five categories: (1) InDels where a monomorphic allele was shared by all members in the two species; (2) InDels where alleles were found among the members of the two species; (3) InDels where a unique allele was found among members of the first species listed, whereas an alternative allele (found in both groups) was fixed in the second species; (4) InDels where a unique allele was found among members of the second species listed, whereas an alternative allele (found in both species) was fixed in the first species; (5) InDels where the two species were fixed for alternative alleles.</p></div></div><p>Although the 2272 InDels almost evenly distributed across all 12 chromosomes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>), the distribution of polymorphic markers varied for three species (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). <em>Solanum pimpinellifolium</em> had a relatively even distribution of polymorphic InDels on all 12 chromosomes. <em>Solanum lycopersicum</em> var. <em>cerasiforme</em> had the similar distribution pattern of polymorphic InDels as <em>S. pimpinellifolium</em> on chromosomes 2, 3, 4, 5, 6, 9, 10, and 11, but clusters of polymorphic InDels occurred at some regions on chromosomes 1, 7, and 12. The distribution of polymorphic InDels varied across and within chromosomes in <em>S. lycopersicum</em>. Among six chromosomes with less polymorphic InDels, chromosomes 1, 8, 10, and 12 had relatively even distribution, while the long-arm ends of chromosomes 3 and 7 had more InDels than other regions. There were less InDels at one end of chromosomes 2, 4, 5, 9, and 11. However, chromosomes 5, 9, and 11 showed relatively even distribution. On chromosome 6, the short arm had more polymorphic InDels than the long arm.</p><p>The proportion of polymorphic InDels on 12 chromosomes ranged from 45.9 to 81.8% in <em>S. pimpinellifolium</em>, 5.8 to 76.6% in <em>S. lycopersicum</em> var. <em>cerasiforme</em>, and 5.3 to 72.2% in <em>S. lycopersicum</em> (Fig. <a reveal-id="DSU008F4" data-open="DSU008F4" class="link link-reveal link-table xref-fig">4</a>). The numbers of polymorphic InDels considerably decreased on four chromosomes 1, 7, 8, and 12 in <em>S. lycopersicum</em> var. <em>cerasiforme</em> and <em>S. lycopersicum</em> (Table <a reveal-id="DSU008TB2" data-open="DSU008TB2" class="link link-reveal link-table xref-fig">2</a>). Furthermore, the proportions of polymorphic InDels on chromosomes 3 and 10 were close between <em>S. pimpinellifolium</em> and <em>S. lycopersicum</em> var. <em>cerasiforme</em>, but significantly decreased in <em>S. lycopersicum</em> (Fig. <a reveal-id="DSU008F4" data-open="DSU008F4" class="link link-reveal link-table xref-fig">4</a>). Interestingly, increases of InDel polymorphisms were observed on chromosomes 4, 5, and 11 in <em>S. lycopersicum</em> var. <em>cerasiforme</em> and <em>S. lycopersicum</em>. The proportions of polymorphic InDels also increased on chromosomes 2 and 3 in <em>S. lycopersicum</em> var. <em>cerasiforme</em>. </p>                        <a id="6690683" scrollto-destination="6690683"></a>
<div id="DSU008F4" data-id="DSU008F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/m_dsu00804.png?Expires=2147483647&Signature=v5recCt~vosHWYwvIdy0VSVh2-KdlJRQPnFaIRAIfSCOFjVkHlj7qPP7M3-AiEWPpId~YFFidkerHrof6X-ydBkWZjDvgge2qu~DhZyJslD-h9YWJLHL-UVRmNhTrGyzYAE4aNqtYzbzN~1SiqUXwY2JRZUI~HLrKqnAA48sY32R5pj4g-9TvXp8GifJzLnDBHSCNbsPibzDEZAJm2PHFQMmWbD4hn6toaJe1jlmK2GBrsk4NMbsQbxXrwXvvVjcvGdbAyqU0Jjr5lrjVel0-L0zonghI4B2Cvfei9qaQ9TSl9W6enUgb04JErnvJYxjVA1p6huJ--DRaegtgZT2bA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of the proportion of polymorphic InDels on 12 chromosomes in Solanum pimpinellifolium, S. lycopersicum var. cerasiforme, and S. lycopersicum." path-from-xml="dsu00804.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6690683/dsu00804.tiff" path-from-xml="dsu00804.tiff" target="_blank">View large</a><a section="6690683" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/dsu00804.png?Expires=2147483647&Signature=LSMafZD9powCoFXLs7wCxGm4nXtlUllWru0gDEM86fw8dkk8aXR9RIJDSWenxbaAmQcBwCzyC6Rv6vbNQBNMtg-P3B-iNiyicVMGoJ3mYHIRXuroYDAsnQISP4kefZZYUGJyehEA84Ji4HBzlP0S-feLMKSZ1GAZTZNDBR53-A1mArlIESG8o0fkj~CuehVX-xxGuQuwCKk7Q2aYdQ24ZI7bLb8qdp73DBbGHEqbJBzZqloHzEqt~YkZJnW~vCED8mylj-KBJKSRCoHg2meMAGtQfcdqp0MCEL6VOmVNsxPCCZnfrbYCUciTwF~~tUtOP7Onp7XGut2l0MZfQDLA0Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6690683&ar=447212&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00804.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of the proportion of polymorphic InDels on 12 chromosomes in <em>Solanum pimpinellifolium</em>, <em>S. lycopersicum</em> var. <em>cerasiforme</em>, and <em>S. lycopersicum</em>.</p></div></div><div content-id="DSU008F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/m_dsu00804.png?Expires=2147483647&Signature=v5recCt~vosHWYwvIdy0VSVh2-KdlJRQPnFaIRAIfSCOFjVkHlj7qPP7M3-AiEWPpId~YFFidkerHrof6X-ydBkWZjDvgge2qu~DhZyJslD-h9YWJLHL-UVRmNhTrGyzYAE4aNqtYzbzN~1SiqUXwY2JRZUI~HLrKqnAA48sY32R5pj4g-9TvXp8GifJzLnDBHSCNbsPibzDEZAJm2PHFQMmWbD4hn6toaJe1jlmK2GBrsk4NMbsQbxXrwXvvVjcvGdbAyqU0Jjr5lrjVel0-L0zonghI4B2Cvfei9qaQ9TSl9W6enUgb04JErnvJYxjVA1p6huJ--DRaegtgZT2bA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of the proportion of polymorphic InDels on 12 chromosomes in Solanum pimpinellifolium, S. lycopersicum var. cerasiforme, and S. lycopersicum." path-from-xml="dsu00804.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6690683/dsu00804.tiff" path-from-xml="dsu00804.tiff" target="_blank">View large</a><a section="6690683" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/dsu00804.png?Expires=2147483647&Signature=LSMafZD9powCoFXLs7wCxGm4nXtlUllWru0gDEM86fw8dkk8aXR9RIJDSWenxbaAmQcBwCzyC6Rv6vbNQBNMtg-P3B-iNiyicVMGoJ3mYHIRXuroYDAsnQISP4kefZZYUGJyehEA84Ji4HBzlP0S-feLMKSZ1GAZTZNDBR53-A1mArlIESG8o0fkj~CuehVX-xxGuQuwCKk7Q2aYdQ24ZI7bLb8qdp73DBbGHEqbJBzZqloHzEqt~YkZJnW~vCED8mylj-KBJKSRCoHg2meMAGtQfcdqp0MCEL6VOmVNsxPCCZnfrbYCUciTwF~~tUtOP7Onp7XGut2l0MZfQDLA0Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6690683&ar=447212&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00804.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of the proportion of polymorphic InDels on 12 chromosomes in <em>Solanum pimpinellifolium</em>, <em>S. lycopersicum</em> var. <em>cerasiforme</em>, and <em>S. lycopersicum</em>.</p></div></div>                            <h3 scrollto-destination=6690684 id="6690684" class="section-title" data-legacy-id=s3d>3.4. Marker polymorphisms and genetic vitiation within three tomato species</h3>
<p>The proportion of polymorphic InDels was 61.6% in 8 <em>S. pimpinellifolium</em> accessions, 45.2% in 4 <em>S. lycopersicum</em> var. <em>cerasiforme</em> accessions, and 31.6% in 10 cultivated tomato varieties (Table <a reveal-id="DSU008TB2" data-open="DSU008TB2" class="link link-reveal link-table xref-fig">2</a>). However, the rate of polymorphic InDels between any two genotypes was low with a range of 14.3–33.6% in <em>S. pimpinellifolium</em>, 17.5–31.5% in <em>S. lycopersicum</em> var. <em>cerasiforme</em>, and 1.5–19.8% in <em>S. lycopersicum</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>).</p><p>Not surprisingly, the eight accessions of <em>S. pimpinellifolium</em> had the largest genetic variation among three species. The average genetic distance was 0.216 with a range from 0.178 (PI 128216) to 0.244 (LA1589). Accessions LA 1589 and LA 2181 had the greatest genetic distance with 0.394, whereas accessions PI 128216 and LA 0373 had the least genetic distance with 0.137. The average genetic distance slightly reduced to 0.202 with a range from 0.162 (LA 4133) to 0.237 (PI 114490) in four <em>S. lycopersicum</em> var. <em>cerasiforme</em> lines, but significantly decreased to 0.108 with a range of 0.086 (Baiguoqiangfeng) to 0.139 (M 82) in 10 varieties of <em>S. lycopersicum</em>. The minimum genetic distance was 0.012 between varieties Liger 87-5 and M 82, followed by 0.015 between varieties Baiguoqiangfeng and Zhongshu 5, while the largest genetic distance was 0.214 between Shijifeng and M 82.</p><p>The dendrogram was constructed from the pairwise genetic distance matrices based on Nei's distance for 22 genotypes. Two distinct groups, A and B, were obtained (Fig. <a reveal-id="DSU008F5" data-open="DSU008F5" class="link link-reveal link-table xref-fig">5</a>). All 8 accessions of <em>S. pimpinellifolium</em> were in Group A, and 10 <em>S. lycopersicum</em> var. <em>cerasiforme</em> cultivars and 4 <em>S. lycopersicum</em> var. <em>cerasiforme</em> accessions were in Group B. The four fresh market cultivars clustered together. However, five processing varieties, one greenhouse variety, and four <em>S. lycopersicum</em> var. <em>cerasiforme</em> accessions did not form their own clades. Of the four <em>S. lycopersicum</em> var. <em>cerasiforme</em> lines, LA 4133 clustered to three processing and one greenhouse varieties, Black cherry clustered to two processing varieties, while PI 114490 and LA 1310 stood alone. </p>                        <a id="6690690" scrollto-destination="6690690"></a>
<div id="DSU008F5" data-id="DSU008F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/m_dsu00805.png?Expires=2147483647&Signature=MbqwRcHpxiZL-C9XZ6S8P2qeHoor6uxxCpCHi5GekE53KSgNY1FQB4KkfynBBRHN9NP4xbxMQsupGNBJDpaPvPo~NdQtES4u5eLYtmTfgH9c8lsWr08xyYflSK0N0e1pt0-PLbxxnq5Jwywvzl9pvXcXub7AaN8wTqaLF~nKaOhcBTqPFRqWTyw4GRMZwBkF-Iqe3NzhTEhsd57P6OukgghVsVIaOaHmHolS1ZVkTKhhpIEQXHZW5WsBX5swEDUyOZILqAmhUgibwAXdHqBpoYCc6fKw62tE1X39s6zOBvwvzrkBBoj~qQk~BVu4chMF4xz8rSxzPQrz3DLMpOZDQA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The dendrogram of 22 tomato genotypes based on 2272 InDel marker data, and generated from Nei's genetic distance matrix by UPGMA in PHYLIP 3.695." path-from-xml="dsu00805.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6690690/dsu00805.tiff" path-from-xml="dsu00805.tiff" target="_blank">View large</a><a section="6690690" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/dsu00805.png?Expires=2147483647&Signature=lZuISwkJQvVy7MoLQRar9uEMUbrMwhuCnMp-9ugNDOahsRrrwGmxYqprjIT9o003e57RkiQoNnGucaln5HUvj13N-cgsg~w86al0XNbSbwTIGRiIZ0xSP9Iou02SwPoWDmYQsL6tOqXwYpy-uHqau7CxRwHqW9N9OJcVb-dgUUhATvnunWcwohGdqdcw1Vbo95oy~y-5mLgu5WZ9weNgRCUd~6e6ZTMJBS4pBhWTzCl3NHW2KfE5L5Y~uHHNoVl1Z2QuZDt8tA6eCZYzllAiunJtSZ-mfDMQxgclfvJYO2v2SB-X9VUxBynfRQE4MYSM584jKU1P0yUkL5kR0iFe~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6690690&ar=447212&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00805.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The dendrogram of 22 tomato genotypes based on 2272 InDel marker data, and generated from Nei's genetic distance matrix by UPGMA in PHYLIP 3.695.</p></div></div><div content-id="DSU008F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/m_dsu00805.png?Expires=2147483647&Signature=MbqwRcHpxiZL-C9XZ6S8P2qeHoor6uxxCpCHi5GekE53KSgNY1FQB4KkfynBBRHN9NP4xbxMQsupGNBJDpaPvPo~NdQtES4u5eLYtmTfgH9c8lsWr08xyYflSK0N0e1pt0-PLbxxnq5Jwywvzl9pvXcXub7AaN8wTqaLF~nKaOhcBTqPFRqWTyw4GRMZwBkF-Iqe3NzhTEhsd57P6OukgghVsVIaOaHmHolS1ZVkTKhhpIEQXHZW5WsBX5swEDUyOZILqAmhUgibwAXdHqBpoYCc6fKw62tE1X39s6zOBvwvzrkBBoj~qQk~BVu4chMF4xz8rSxzPQrz3DLMpOZDQA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The dendrogram of 22 tomato genotypes based on 2272 InDel marker data, and generated from Nei's genetic distance matrix by UPGMA in PHYLIP 3.695." path-from-xml="dsu00805.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6690690/dsu00805.tiff" path-from-xml="dsu00805.tiff" target="_blank">View large</a><a section="6690690" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/dsu00805.png?Expires=2147483647&Signature=lZuISwkJQvVy7MoLQRar9uEMUbrMwhuCnMp-9ugNDOahsRrrwGmxYqprjIT9o003e57RkiQoNnGucaln5HUvj13N-cgsg~w86al0XNbSbwTIGRiIZ0xSP9Iou02SwPoWDmYQsL6tOqXwYpy-uHqau7CxRwHqW9N9OJcVb-dgUUhATvnunWcwohGdqdcw1Vbo95oy~y-5mLgu5WZ9weNgRCUd~6e6ZTMJBS4pBhWTzCl3NHW2KfE5L5Y~uHHNoVl1Z2QuZDt8tA6eCZYzllAiunJtSZ-mfDMQxgclfvJYO2v2SB-X9VUxBynfRQE4MYSM584jKU1P0yUkL5kR0iFe~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6690690&ar=447212&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00805.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The dendrogram of 22 tomato genotypes based on 2272 InDel marker data, and generated from Nei's genetic distance matrix by UPGMA in PHYLIP 3.695.</p></div></div>                            <h3 scrollto-destination=6690692 id="6690692" class="section-title" data-legacy-id=s3e>3.5. Genes with InDels in the coding region</h3>
<p>Blast search of flanking sequences of 2272 validated InDels against the tomato ITAG2.3_cds.fasta data identified 56 InDels in coding regions of annotated genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>), of which 64.3% were deletions in Heinz 1706 and 35.7% were insertion in Heinz 1706. Based on the sizes of InDels, 28.6% of InDels were frame-shift mutations, because the numbers of nucleotides in the InDels were indivisible by three. The remaining 71.4% InDels did not result in frame-shift, but would cause insertion or deletion of some amino acids.</p>                            <h2 scrollto-destination=6690694 id="6690694" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>Molecular markers are important to genetic study and marker-assisted selection. Large-scale discovery combining high-throughput genotyping of SNPs have shown its power in gene identification and breeding in tomato.<sup><a reveal-id="DSU008C12" data-open="DSU008C12" class="link link-ref link-reveal xref-bibr">12</a></sup> However, high costs and technical or equipment demands will still be a major obstacle for large-scale use of SNPs in the developing countries.<sup><a reveal-id="DSU008C41" data-open="DSU008C41" class="link link-ref link-reveal xref-bibr">41</a>,<a reveal-id="DSU008C42" data-open="DSU008C42" class="link link-ref link-reveal xref-bibr">42</a></sup> On the contrary, the genotyping of short InDels is relatively easy and inexpensive with a simple PCR and electrophoresis. Short InDels can also be analysed with high-throughput technologies<sup><a reveal-id="DSU008C26" data-open="DSU008C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSU008C43" data-open="DSU008C43" class="link link-ref link-reveal xref-bibr">43</a>,<a reveal-id="DSU008C44" data-open="DSU008C44" class="link link-ref link-reveal xref-bibr">44</a></sup> and in large-scale multiplexing.<sup><a reveal-id="DSU008C45" data-open="DSU008C45" class="link link-ref link-reveal xref-bibr">45</a></sup> As a type of genetic markers, InDels have been successfully used for forensic analysis<sup><a reveal-id="DSU008C46  DSU008C48" data-open="DSU008C46  DSU008C48" class="link link-ref link-reveal xref-bibr">46–48</a></sup> and individual identification<sup><a reveal-id="DSU008C44" data-open="DSU008C44" class="link link-ref link-reveal xref-bibr">44</a>,<a reveal-id="DSU008C45" data-open="DSU008C45" class="link link-ref link-reveal xref-bibr">45</a></sup> in human, as well as genetic studies in several plant species including rice, wheat, citrus, and Arabidopsis.<sup><a reveal-id="DSU008C33" data-open="DSU008C33" class="link link-ref link-reveal xref-bibr">33</a></sup> Although the tomato genome sequences have been widely used in various purposes including SNP discovery, genetic mapping, gene prediction, gene expression, genetic diversity, comparative genomics, and epigenetics since their release,<sup><a reveal-id="DSU008C49" data-open="DSU008C49" class="link link-ref link-reveal xref-bibr">49</a></sup> identification of InDels has so far been confined to detect polymorphisms between wild species and cultivated tomato.<sup><a reveal-id="DSU008C34" data-open="DSU008C34" class="link link-ref link-reveal xref-bibr">34</a>,<a reveal-id="DSU008C35" data-open="DSU008C35" class="link link-ref link-reveal xref-bibr">35</a></sup> In this study, we identified InDels by comparative analysis of genome sequences between <em>S. pimpinellifolium</em> and <em>S. lycopersicum</em>, and then validated them in 10 cultivated tomato lines via PCR amplification. Of 2272 InDels polymorphic between LA 1589 and Heinz 1706, 31.6% were polymorphic among the 10 cultivated tomato varieties and 1.5–19.8% were polymorphic between any 2 of the 10 cultivated tomato varieties. Based on the total number of InDels (145 695) between LA 1589 and Heinz 1706, we estimated that there were 2100–28 800 InDels between any two cultivated tomato varieties, suggesting that there were abundant InDels for genetic study and marker-assisted selection in the cultivated tomato.</p><p>Precise identification of InDels in sequence databases depends on the strategy and the parameters used for data mining as well as the quality of sequence data. Since InDels are the dominant error type generated by 454 pyrosequencing<sup><a reveal-id="DSU008C50" data-open="DSU008C50" class="link link-ref link-reveal xref-bibr">50</a></sup> and an InDel error rate of one per 6.4 kb was observed in tomato,<sup><a reveal-id="DSU008C34" data-open="DSU008C34" class="link link-ref link-reveal xref-bibr">34</a></sup> the initial work on identification of InDels between the genomes of LA 1589 and Heinz 1706 did not count InDels of 1 and 2 bp to avoid overestimation of small InDels due to sequencing errors.<sup><a reveal-id="DSU008C34" data-open="DSU008C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Using a bioinformatic pipeline involving various comparative genomics tools, 9474 InDels of 15–100 bp were identified between LA 1589 and Heinz 1706, and &gt;80% could be verified by PCR (Jiang <em>et al.</em> unpublished data, acquired from <a class="link link-uri" href="http://ftp://ftp.solgenomics.net/maps_and_markers/LippmanZ/">ftp://ftp.solgenomics.net/maps_and_markers/LippmanZ/</a>, 19 February 2014, date last accessed). In this study, a total of 145 695 InDels were predicted between LA 1589 and Heinz 1796, which was approximate one-fifth of 749 966 InDels Identified in Sato <em>et al</em>.<sup><a reveal-id="DSU008C34" data-open="DSU008C34" class="link link-ref link-reveal xref-bibr">34</a></sup> The overall frequency of InDels (one per 5.22 kb) was also much lower than one per 110 bp in Sato <em>et al</em>.<sup><a reveal-id="DSU008C34" data-open="DSU008C34" class="link link-ref link-reveal xref-bibr">34</a></sup> However, the number (9137) of InDels of 15–94 bp was close to the results of Jiang <em>et al.</em>, though the strategies used for InDels identification were different. Owing to the lack of methodology description in Sato <em>et al.</em>,<sup><a reveal-id="DSU008C34" data-open="DSU008C34" class="link link-ref link-reveal xref-bibr">34</a></sup> we were not able to determine the cause of the difference between two studies. Two points might be worthy of notice. First, the lengths of putative InDels identified in two studies were different with ranges of 3–300 bp in Sato <em>et al.</em><sup><a reveal-id="DSU008C34" data-open="DSU008C34" class="link link-ref link-reveal xref-bibr">34</a></sup> and 1–94 bp in this study. We could not identify any InDels &gt;94 bp using our methodology. Secondly, the rate of validation (82.4%) was close to 81.7% obtained in Koenig <em>et al.</em>,<sup><a reveal-id="DSU008C35" data-open="DSU008C35" class="link link-ref link-reveal xref-bibr">35</a></sup> though the comparisons involved in different wild species and cultivated varieties, indicating that ∼20% of predicted InDels (≥2 bp) were false due to sequencing error. All these suggested that our prediction might be more close to the real number of InDels in the currently available genome sequences of LA 1589 and Heinz 1706.</p><p>The polymorphic InDels evenly distributed across all 12 chromosomes in <em>S. pimpinellifolium</em>, but appeared non-randomly distributed across and within chromosomes in <em>S. lycopersicum</em> var. <em>cerasiforme</em> and <em>S. lycopersicum</em>. Domestication and selection could be one causal of this difference. For example, there were 38 and 35 polymorphic InDels at the bottom (∼11 Mb) of chromosome 2 in <em>S. pimpinellifolium</em> and <em>S. lycopersicum</em> var. <em>cerasiforme</em>, respectively, but only two InDels were polymorphic in <em>S. lycopersicum</em>. This might be due to the existence of quantitative trait loci for fruit weight and selection for large fruit in <em>S. lycopersicum</em>.<sup><a reveal-id="DSU008C12" data-open="DSU008C12" class="link link-ref link-reveal xref-bibr">12</a></sup> In addition, several studies have proved that the introgression of disease resistance genes in many cultivars has strong influence on SNP patterns.<sup><a reveal-id="DSU008C19" data-open="DSU008C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSU008C51" data-open="DSU008C51" class="link link-ref link-reveal xref-bibr">51</a></sup> This kind of introgression could also cause the difference of polymorphic InDels distribution among three species.</p><p>It has been suggested that domestication and inbreeding dramatically reduced the genetic variation<sup><a reveal-id="DSU008C52" data-open="DSU008C52" class="link link-ref link-reveal xref-bibr">52</a></sup> and modern cultivars have less genetic variation than old ones in tomato.<sup><a reveal-id="DSU008C53" data-open="DSU008C53" class="link link-ref link-reveal xref-bibr">53</a>,<a reveal-id="DSU008C54" data-open="DSU008C54" class="link link-ref link-reveal xref-bibr">54</a></sup> In this study, genetic variation of three species was investigated using the same large set of InDel markers, which allowed us to compare genetic polymorphisms among and within species at the same time. The number of polymorphic InDels, the total number of alleles amplified by InDel markers, and the average genetic distance in 10 <em>S. lycopersicum</em> varieties significantly reduced comparing with those in 8 <em>S. pimpinellifolium</em> accessions, supported the reduction of genetic variation in cultivated tomato. The four <em>S. lycopersicum</em> var. <em>cerasiforme</em> accessions showed an intermediate amount of genetic diversity between <em>S. lycopersicum</em> and <em>S. pimpinellifolium</em>, which was consistent with previous findings.<sup><a reveal-id="DSU008C55" data-open="DSU008C55" class="link link-ref link-reveal xref-bibr">55</a>,<a reveal-id="DSU008C56" data-open="DSU008C56" class="link link-ref link-reveal xref-bibr">56</a></sup> However, some novel alleles occurred in both <em>S. lycopersicum</em> var. <em>cerasiforme</em> and <em>S. lycopersicum</em>, suggesting that domestication and selection could also generate new variation.</p><p>The occurrence of InDels in coding regions of a gene can either cause frame-shift or amino acid InDels, which most likely alternates the gene function and results in phenotype change.<sup><a reveal-id="DSU008C57" data-open="DSU008C57" class="link link-ref link-reveal xref-bibr">57</a></sup> A <em>Rider</em> mutational insertion event occurring in the first exon of the <em>Psy1</em> gene causes the early termination of <em>Psy1</em> transcription that results in yellow flesh in the tomato <em>r</em> mutant.<sup><a reveal-id="DSU008C58" data-open="DSU008C58" class="link link-ref link-reveal xref-bibr">58</a></sup> A single-base deletion mutation in the coding region of <em>SlIAA9</em> gene, an <em>Aux/IAA</em> gene involving in tomato leaf morphology, converts tomato compound leaves to simple leaves.<sup><a reveal-id="DSU008C59" data-open="DSU008C59" class="link link-ref link-reveal xref-bibr">59</a></sup> InDels occurring in the promoter region can also affect the gene expression.<sup><a reveal-id="DSU008C60" data-open="DSU008C60" class="link link-ref link-reveal xref-bibr">60</a></sup> Here, we identified 145 695 InDels between LA 1589 and Heinz 1706, and 31.6% of them were polymorphic in cultivated tomatoes. The percentage of InDels (2.5%) occurring in coding regions of genes identified in this study was much lower than our recent work (19.7%) on comparative analysis of resistance-like genes between LA 1589 and Heinz 1706.<sup><a reveal-id="DSU008C61" data-open="DSU008C61" class="link link-ref link-reveal xref-bibr">61</a></sup> Identification of specific genes in our previous work other than a random sample in this study could cause the different proportions of InDels in coding regions.</p><p>In conclusion, there are abundant short InDels in cultivated tomato. Identification and validation of this kind of short InDels will not only provide molecular markers for genetic study and marker-assisted selection in breeding, but also provide useful information for gene cloning and functional analysis.</p>                            <h2 scrollto-destination=6690706 id="6690706" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=6690710 id="6690710" class="section-title" data-legacy-id=s6>Funding</h2>
<p>The work was partially supported by the National Program on Key Basic Research Projects (The 973 Program: 2012CB113900) and the National Natural Science Foundation of China (31171973).</p>                            <h2 scrollto-destination=6690713 id="6690713" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Dr David M. Francis at the Ohio State University (USA), and Tomato Genetics Resource Center at the University of California (Davis, USA) for providing seeds of some tomato lines used in this study.</p>                            <h2 scrollto-destination=6690716 id="6690716" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSU008C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Foolad</div> <div class="given-names">M.R.</div></div>,  <div class="name"><div class="surname">Panthee</div> <div class="given-names">R.D.</div></div>. </div><div class="article-title">Marker-assisted selection in tomato breeding</div>, <div class="source">Crit. Rev. Plant Sci.</div> , <div class="year">2012</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">93</div>-<div class="lpage">123</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Marker-assisted%20selection%20in%20tomato%20breeding&amp;author=M.R.%20Foolad&amp;author=R.D.%20Panthee&amp;publication_year=2012&amp;journal=Crit.%20Rev.%20Plant%20Sci.&amp;volume=31&amp;pages=93-123" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/07352689.2011.616057" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F07352689.2011.616057" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F07352689.2011.616057"> </span></div> </div></div></div></div></div><div content-id="DSU008C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>. </div><div class="article-title">DNA marker applications to molecular genetics and genomics in tomato</div>, <div class="source">Breeding Sci.</div> , <div class="year">2013</div>, vol. <div class="volume">63</div> (pg. <div class="fpage">21</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20marker%20applications%20to%20molecular%20genetics%20and%20genomics%20in%20tomato&amp;author=K.%20Shirasawa&amp;author=H.%20Hirakawa&amp;publication_year=2013&amp;journal=Breeding%20Sci.&amp;volume=63&amp;pages=21-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.63.21" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.63.21" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.63.21"> </span></div> </div></div></div></div></div><div content-id="DSU008C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miller</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Tanksley</div> <div class="given-names">S.D.</div></div>. </div><div class="article-title">RFLP analysis of phylogenetic relationships and genetic variation in the genus <em>Lycopersicon</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1990</div>, vol. <div class="volume">80</div> (pg. <div class="fpage">437</div>-<div class="lpage">48</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RFLP%20analysis%20of%20phylogenetic%20relationships%20and%20genetic%20variation%20in%20the%20genus%20Lycopersicon&amp;author=J.C.%20Miller&amp;author=S.D.%20Tanksley&amp;publication_year=1990&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=80&amp;pages=437-48" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24221000" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">W.C.</div></div>,  <div class="name"><div class="surname">Bai</div> <div class="given-names">X.D.</div></div>,  <div class="name"><div class="surname">Kabelka</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Discovery of single nucleotide polymorphisms in <em>Lycopersicon esculentum</em> by computer aided analysis of expressed sequence tags</div>, <div class="source">Mol. Breeding</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">21</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovery%20of%20single%20nucleotide%20polymorphisms%20in%20Lycopersicon%20esculentum%20by%20computer%20aided%20analysis%20of%20expressed%20sequence%20tags&amp;author=W.C.%20Yang&amp;author=X.D.%20Bai&amp;author=E.%20Kabelka&amp;publication_year=2004&amp;journal=Mol.%20Breeding&amp;volume=14&amp;pages=21-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/B:MOLB.0000037992.03731.a5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FB:MOLB.0000037992.03731.a5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FB:MOLB.0000037992.03731.a5"> </span></div> </div></div></div></div></div><div content-id="DSU008C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Labate</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Baldo</div> <div class="given-names">A.M.</div></div>. </div><div class="article-title">Tomato SNP discovery by EST mining and resequencing</div>, <div class="source">Mol. Breeding</div> , <div class="year">2005</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">343</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tomato%20SNP%20discovery%20by%20EST%20mining%20and%20resequencing&amp;author=J.A.%20Labate&amp;author=A.M.%20Baldo&amp;publication_year=2005&amp;journal=Mol.%20Breeding&amp;volume=16&amp;pages=343-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-005-1911-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-005-1911-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-005-1911-5"> </span></div> </div></div></div></div></div><div content-id="DSU008C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamamoto</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Tsugane</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Watanabe</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Expressed sequence tags from the laboratory-grown miniature tomato (<em>Lycopersicon esculentum</em>) cultivar Micro-Tom and mining for single nucleotide polymorphisms and insertions/deletions in tomato cultivars</div>, <div class="source">Gene</div> , <div class="year">2005</div>, vol. <div class="volume">56</div> (pg. <div class="fpage">127</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expressed%20sequence%20tags%20from%20the%20laboratory-grown%20miniature%20tomato%20(Lycopersicon%20esculentum)%20cultivar%20Micro-Tom%20and%20mining%20for%20single%20nucleotide%20polymorphisms%20and%20insertions%2Fdeletions%20in%20tomato%20cultivars&amp;author=N.%20Yamamoto&amp;author=T.%20Tsugane&amp;author=M.%20Watanabe&amp;publication_year=2005&amp;journal=Gene&amp;volume=56&amp;pages=127-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2005.04.026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2005.04.026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2005.04.026"> </span></div> </div></div></div></div></div><div content-id="DSU008C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Isobe</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">SNP discovery and linkage map construction in cultivated tomato</div>, <div class="source">DNA Res.</div> , <div class="year">2010</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">381</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20discovery%20and%20linkage%20map%20construction%20in%20cultivated%20tomato&amp;author=K.%20Shirasawa&amp;author=S.%20Isobe&amp;author=H.%20Hirakawa&amp;publication_year=2010&amp;journal=DNA%20Res.&amp;volume=17&amp;pages=381-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsq024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsq024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsq024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21044984" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">van Deynze</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Stoffel</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Buell</div> <div class="given-names">C.R.</div></div>, et al. </div><div class="article-title">Diversity in conserved genes in tomato</div>, <div class="source">BMC Genomics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">465</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversity%20in%20conserved%20genes%20in%20tomato&amp;author=A.%20van%20Deynze&amp;author=K.%20Stoffel&amp;author=C.R.%20Buell&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=465" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-465" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-465" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-465"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18088428" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">Y.Y.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Francis</div> <div class="given-names">D.M.</div></div>,  <div class="name"><div class="surname">Shen</div> <div class="given-names">H.L.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">T.T.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">W.C.</div></div>. </div><div class="article-title">Discovery of intron polymorphisms in cultivated tomato using both tomato and Arabidopsis genomic information</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">121</div> (pg. <div class="fpage">1199</div>-<div class="lpage">207</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovery%20of%20intron%20polymorphisms%20in%20cultivated%20tomato%20using%20both%20tomato%20and%20Arabidopsis%20genomic%20information&amp;author=Y.Y.%20Wang&amp;author=J.%20Chen&amp;author=D.M.%20Francis&amp;author=H.L.%20Shen&amp;author=T.T.%20Wu&amp;author=W.C.%20Yang&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=121&amp;pages=1199-207" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-010-1381-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-010-1381-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-010-1381-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20552324" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sim</div> <div class="given-names">S.-C.</div></div>,  <div class="name"><div class="surname">Robbins</div> <div class="given-names">M.D.</div></div>,  <div class="name"><div class="surname">Chilcott</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Francis</div> <div class="given-names">D.M.</div></div>. </div><div class="article-title">Oligonucleotide array discovery of polymorphisms in cultivated tomato (<em>Solanum lycopersicum</em> L.) reveals patterns of SNP variation associated with breeding</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">466</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Oligonucleotide%20array%20discovery%20of%20polymorphisms%20in%20cultivated%20tomato%20(Solanum%20lycopersicum%20L.)%20reveals%20patterns%20of%20SNP%20variation%20associated%20with%20breeding&amp;author=S.-C.%20Sim&amp;author=M.D.%20Robbins&amp;author=C.%20Chilcott&amp;author=T.%20Zhu&amp;author=D.M.%20Francis&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=466" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-466" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-466" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-466"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19818135" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hamilton</div> <div class="given-names">J.P.</div></div>,  <div class="name"><div class="surname">Sim</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Stoffel</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Van Deynze</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Buell</div> <div class="given-names">C.R.</div></div>,  <div class="name"><div class="surname">Francis</div> <div class="given-names">D.M.</div></div>. </div><div class="article-title">Single nucleotide polymorphism discovery in cultivated tomato via sequencing by synthesis</div>, <div class="source">Plant Genome</div> , <div class="year">2012</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">17</div>-<div class="lpage">29</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single%20nucleotide%20polymorphism%20discovery%20in%20cultivated%20tomato%20via%20sequencing%20by%20synthesis&amp;author=J.P.%20Hamilton&amp;author=S.C.%20Sim&amp;author=K.%20Stoffel&amp;author=A.%20Van%20Deynze&amp;author=C.R.%20Buell&amp;author=D.M.%20Francis&amp;publication_year=2012&amp;journal=Plant%20Genome&amp;volume=5&amp;pages=17-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3835/plantgenome2011.12.0033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3835%2Fplantgenome2011.12.0033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3835%2Fplantgenome2011.12.0033"> </span></div> </div></div></div></div></div><div content-id="DSU008C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Víquez-Zamora</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Vosman</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">van de Geest</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Tomato breeding in the genomics era: insights from a SNP array</div>, <div class="source">BMC Genomics</div> , <div class="year">2013</div>, vol. <div class="volume">14</div> pg. <div class="fpage">354</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tomato%20breeding%20in%20the%20genomics%20era%3A%20insights%20from%20a%20SNP%20array&amp;author=M.%20V%C3%ADquez-Zamora&amp;author=B.%20Vosman&amp;author=H.%20van%20de%20Geest&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=354" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-354" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-354" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-354"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23711327" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Fukuoka</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Matsunaga</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Genome-wide association studies using single nucleotide polymorphism markers developed by re-sequencing of the genomes of cultivated tomato</div>, <div class="source">DNA Res.</div> , <div class="year">2013</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">593</div>-<div class="lpage">603</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20studies%20using%20single%20nucleotide%20polymorphism%20markers%20developed%20by%20re-sequencing%20of%20the%20genomes%20of%20cultivated%20tomato&amp;author=K.%20Shirasawa&amp;author=H.%20Fukuoka&amp;author=H.%20Matsunaga&amp;publication_year=2013&amp;journal=DNA%20Res.&amp;volume=20&amp;pages=593-603" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23903436" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Robbins</div> <div class="given-names">M.D.</div></div>,  <div class="name"><div class="surname">Sim</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">W.C.</div></div>, et al. </div><div class="article-title">Mapping and linkage disequilibrium analysis with a genome-wide collection of SNPs that detect polymorphism in cultivated tomato</div>, <div class="source">J. Exp. Bot.</div> , <div class="year">2011</div>, vol. <div class="volume">62</div> (pg. <div class="fpage">1831</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20and%20linkage%20disequilibrium%20analysis%20with%20a%20genome-wide%20collection%20of%20SNPs%20that%20detect%20polymorphism%20in%20cultivated%20tomato&amp;author=M.D.%20Robbins&amp;author=S.C.%20Sim&amp;author=W.C.%20Yang&amp;publication_year=2011&amp;journal=J.%20Exp.%20Bot.&amp;volume=62&amp;pages=1831-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erq367" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferq367" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferq367"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21193580" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ohyama</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Genome-wide SNP genotyping to infer the effects on gene functions in tomato</div>, <div class="source">DNA Res.</div> , <div class="year">2013</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">221</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20SNP%20genotyping%20to%20infer%20the%20effects%20on%20gene%20functions%20in%20tomato&amp;author=H.%20Hirakawa&amp;author=K.%20Shirasawa&amp;author=A.%20Ohyama&amp;publication_year=2013&amp;journal=DNA%20Res.&amp;volume=20&amp;pages=221-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23482505" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sim</div> <div class="given-names">S.-C.</div></div>,  <div class="name"><div class="surname">Durstewitz</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Plieske</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Development of a large SNP genotyping array and generation of high-density genetic maps in tomato</div>, <div class="source">PLoS ONE</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e40563</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20large%20SNP%20genotyping%20array%20and%20generation%20of%20high-density%20genetic%20maps%20in%20tomato&amp;author=S.-C.%20Sim&amp;author=G.%20Durstewitz&amp;author=J.%20Plieske&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e40563" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0040563" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0040563" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0040563"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22802968" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Shen</div> <div class="given-names">H.L.</div></div>, et al. </div><div class="article-title">Genetic variation in tomato populations from four breeding programs revealed by single nucleotide polymorphism and simple sequence repeat markers</div>, <div class="source">Sci. Hortic.</div> , <div class="year">2009</div>, vol. <div class="volume">122</div> (pg. <div class="fpage">6</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20variation%20in%20tomato%20populations%20from%20four%20breeding%20programs%20revealed%20by%20single%20nucleotide%20polymorphism%20and%20simple%20sequence%20repeat%20markers&amp;author=J.%20Chen&amp;author=H.%20Wang&amp;author=H.L.%20Shen&amp;publication_year=2009&amp;journal=Sci.%20Hortic.&amp;volume=122&amp;pages=6-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.scienta.2009.03.025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.scienta.2009.03.025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.scienta.2009.03.025"> </span></div> </div></div></div></div></div><div content-id="DSU008C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sim</div> <div class="given-names">S.-C.</div></div>,  <div class="name"><div class="surname">Robbins</div> <div class="given-names">M.D.</div></div>,  <div class="name"><div class="surname">van Deynze</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Michel</div> <div class="given-names">A.P.</div></div>,  <div class="name"><div class="surname">Francis</div> <div class="given-names">D.M.</div></div>. </div><div class="article-title">Population structure and genetic differentiation associated with breeding history and selection in tomato (<em>Solanum lycopersicum</em> L.)</div>, <div class="source">Heredity</div> , <div class="year">2011</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">927</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population%20structure%20and%20genetic%20differentiation%20associated%20with%20breeding%20history%20and%20selection%20in%20tomato%20(Solanum%20lycopersicum%20L.)&amp;author=S.-C.%20Sim&amp;author=M.D.%20Robbins&amp;author=A.%20van%20Deynze&amp;author=A.P.%20Michel&amp;author=D.M.%20Francis&amp;publication_year=2011&amp;journal=Heredity&amp;volume=106&amp;pages=927-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/hdy.2010.139" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fhdy.2010.139" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fhdy.2010.139"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21081965" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sim</div> <div class="given-names">S.-C.</div></div>,  <div class="name"><div class="surname">van Deynze</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Stoffel</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">High-density SNP genotyping of tomato (<em>Solanum lycopersicum</em> L.) reveals patterns of genetic variation due to breeding</div>, <div class="source">PLoS ONE</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e45520</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-density%20SNP%20genotyping%20of%20tomato%20(Solanum%20lycopersicum%20L.)%20reveals%20patterns%20of%20genetic%20variation%20due%20to%20breeding&amp;author=S.-C.%20Sim&amp;author=A.%20van%20Deynze&amp;author=K.%20Stoffel&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e45520" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0045520" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0045520" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0045520"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23029069" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Corrado</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Piffanelli</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Caramante</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Coppola</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Rao</div> <div class="given-names">R.</div></div>. </div><div class="article-title">SNP genotyping reveals genetic diversity between cultivated landraces and contemporary varieties of tomato</div>, <div class="source">BMC Genomics</div> , <div class="year">2013</div>, vol. <div class="volume">14</div> pg. <div class="fpage">835</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20genotyping%20reveals%20genetic%20diversity%20between%20cultivated%20landraces%20and%20contemporary%20varieties%20of%20tomato&amp;author=G.%20Corrado&amp;author=P.%20Piffanelli&amp;author=M.%20Caramante&amp;author=M.%20Coppola&amp;author=R.%20Rao&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=835" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-835" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-835" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-835"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24279304" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Collins</div> <div class="given-names">F.S.</div></div>,  <div class="name"><div class="surname">Drumm</div> <div class="given-names">M.L.</div></div>,  <div class="name"><div class="surname">Cole</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Lockwood</div> <div class="given-names">W.K.</div></div>,  <div class="name"><div class="surname">Vande Woude</div> <div class="given-names">G.F.</div></div>,  <div class="name"><div class="surname">Iannuzzi</div> <div class="given-names">M.C.</div></div>. </div><div class="article-title">Construction of a general human chromosome jumping library, with application to cystic fibrosis</div>, <div class="source">Science</div> , <div class="year">1987</div>, vol. <div class="volume">235</div> (pg. <div class="fpage">1046</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20general%20human%20chromosome%20jumping%20library%2C%20with%20application%20to%20cystic%20fibrosis&amp;author=F.S.%20Collins&amp;author=M.L.%20Drumm&amp;author=J.L.%20Cole&amp;author=W.K.%20Lockwood&amp;author=G.F.%20Vande%20Woude&amp;author=M.C.%20Iannuzzi&amp;publication_year=1987&amp;journal=Science&amp;volume=235&amp;pages=1046-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.2950591" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.2950591" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.2950591"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2950591" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Usdin</div> <div class="given-names">K.</div></div>. </div><div class="article-title">The biological effects of simple tandem repeats: lessons from the repeat expansion diseases</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">1011</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20biological%20effects%20of%20simple%20tandem%20repeats%3A%20lessons%20from%20the%20repeat%20expansion%20diseases&amp;author=K.%20Usdin&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=1011-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.070409.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.070409.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.070409.107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18593815" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">MacArthur</div> <div class="given-names">D.G.</div></div>,  <div class="name"><div class="surname">Tyler-Smith</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Loss-of-function variants in the genomes of healthy humans</div>, <div class="source">Hum. Mol. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">R125</div>-<div class="lpage">130</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Loss-of-function%20variants%20in%20the%20genomes%20of%20healthy%20humans&amp;author=D.G.%20MacArthur&amp;author=C.%20Tyler-Smith&amp;publication_year=2010&amp;journal=Hum.%20Mol.%20Genet.&amp;volume=19&amp;pages=R125-130" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/hmg/ddq365" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fhmg%2Fddq365" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fhmg%2Fddq365"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20805107" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stenson</div> <div class="given-names">P.D.</div></div>,  <div class="name"><div class="surname">Ball</div> <div class="given-names">E.V.</div></div>,  <div class="name"><div class="surname">Howells</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Phillips</div> <div class="given-names">A.D.</div></div>,  <div class="name"><div class="surname">Mort</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Cooper</div> <div class="given-names">D.N.</div></div>. </div><div class="article-title">The Human Gene Mutation Database: providing a comprehensive central mutation database for molecular diagnostics and personalized genomics</div>, <div class="source">Hum. Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">69</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Human%20Gene%20Mutation%20Database%3A%20providing%20a%20comprehensive%20central%20mutation%20database%20for%20molecular%20diagnostics%20and%20personalized%20genomics&amp;author=P.D.%20Stenson&amp;author=E.V.%20Ball&amp;author=K.%20Howells&amp;author=A.D.%20Phillips&amp;author=M.%20Mort&amp;author=D.N.%20Cooper&amp;publication_year=2009&amp;journal=Hum.%20Genomics&amp;volume=4&amp;pages=69-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1479-7364-4-2-69" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1479-7364-4-2-69" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1479-7364-4-2-69"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20038494" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Montgomery</div> <div class="given-names">S.B.</div></div>,  <div class="name"><div class="surname">Goode</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Kvikstad</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">The origin, evolution, and functional impact of short insertion-deletion variants identified in 179 human genomes</div>, <div class="source">Genome Res.</div> , <div class="year">2013</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">749</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20origin%2C%20evolution%2C%20and%20functional%20impact%20of%20short%20insertion-deletion%20variants%20identified%20in%20179%20human%20genomes&amp;author=S.B.%20Montgomery&amp;author=D.L.%20Goode&amp;author=E.%20Kvikstad&amp;publication_year=2013&amp;journal=Genome%20Res.&amp;volume=23&amp;pages=749-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.148718.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.148718.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.148718.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23478400" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mills</div> <div class="given-names">R.E.</div></div>,  <div class="name"><div class="surname">Luttig</div> <div class="given-names">C.T.</div></div>,  <div class="name"><div class="surname">Larkins</div> <div class="given-names">C.E.</div></div>, et al. </div><div class="article-title">An initial map of insertion and deletion (INDEL) variation in the human genome</div>, <div class="source">Genome Res.</div> , <div class="year">2006</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">1182</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20initial%20map%20of%20insertion%20and%20deletion%20(INDEL)%20variation%20in%20the%20human%20genome&amp;author=R.E.%20Mills&amp;author=C.T.%20Luttig&amp;author=C.E.%20Larkins&amp;publication_year=2006&amp;journal=Genome%20Res.&amp;volume=16&amp;pages=1182-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.4565806" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.4565806" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.4565806"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16902084" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mullaney</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Mills</div> <div class="given-names">R.E.</div></div>,  <div class="name"><div class="surname">Pittard</div> <div class="given-names">W.S.</div></div>,  <div class="name"><div class="surname">Devine</div> <div class="given-names">S.E.</div></div>. </div><div class="article-title">Small insertions and deletions (INDELs) in human genomes</div>, <div class="source">Hum. Mol. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">R131</div>-<div class="lpage">136</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Small%20insertions%20and%20deletions%20(INDELs)%20in%20human%20genomes&amp;author=J.M.%20Mullaney&amp;author=R.E.%20Mills&amp;author=W.S.%20Pittard&amp;author=S.E.%20Devine&amp;publication_year=2010&amp;journal=Hum.%20Mol.%20Genet.&amp;volume=19&amp;pages=R131-136" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/hmg/ddq400" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fhmg%2Fddq400" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fhmg%2Fddq400"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20858594" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bromberg</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Building a genome analysis pipeline to predict disease risk and prevent disease</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2013</div>, vol. <div class="volume">425</div> (pg. <div class="fpage">3993</div>-<div class="lpage">4005</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Building%20a%20genome%20analysis%20pipeline%20to%20predict%20disease%20risk%20and%20prevent%20disease&amp;author=Y.%20Bromberg&amp;publication_year=2013&amp;journal=J.%20Mol.%20Biol.&amp;volume=425&amp;pages=3993-4005" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmb.2013.07.038" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2013.07.038" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jmb.2013.07.038"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23928561" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hou</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Peng</div> <div class="given-names">Z.</div></div>, et al. </div><div class="article-title">A platform of high-density INDEL/CAPS markers for map-based cloning in Arabidopsis</div>, <div class="source">Plant J.</div> , <div class="year">2010</div>, vol. <div class="volume">63</div> (pg. <div class="fpage">880</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20platform%20of%20high-density%20INDEL%2FCAPS%20markers%20for%20map-based%20cloning%20in%20Arabidopsis&amp;author=X.%20Hou&amp;author=L.%20Li&amp;author=Z.%20Peng&amp;publication_year=2010&amp;journal=Plant%20J.&amp;volume=63&amp;pages=880-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2010.63.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2010.63.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2010.63.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20561258" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pacurar</div> <div class="given-names">D.I.</div></div>,  <div class="name"><div class="surname">Pacurar</div> <div class="given-names">M.L.</div></div>,  <div class="name"><div class="surname">Street</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">A collection of INDEL markers for map-based cloning in seven Arabidopsis accessions</div>, <div class="source">J. Exp. Bot.</div> , <div class="year">2012</div>, vol. <div class="volume">63</div> (pg. <div class="fpage">2491</div>-<div class="lpage">501</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20collection%20of%20INDEL%20markers%20for%20map-based%20cloning%20in%20seven%20Arabidopsis%20accessions&amp;author=D.I.%20Pacurar&amp;author=M.L.%20Pacurar&amp;author=N.%20Street&amp;publication_year=2012&amp;journal=J.%20Exp.%20Bot.&amp;volume=63&amp;pages=2491-501" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/err422" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferr422" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferr422"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22282537" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shen</div> <div class="given-names">Y.J.</div></div>,  <div class="name"><div class="surname">Jiang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Jin</div> <div class="given-names">J.P.</div></div>, et al. </div><div class="article-title">Development of genome-wide DNA polymorphism database for map-based cloning of rice genes</div>, <div class="source">Plant Physiol.</div> , <div class="year">2004</div>, vol. <div class="volume">135</div> (pg. <div class="fpage">1198</div>-<div class="lpage">205</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20genome-wide%20DNA%20polymorphism%20database%20for%20map-based%20cloning%20of%20rice%20genes&amp;author=Y.J.%20Shen&amp;author=H.%20Jiang&amp;author=J.P.%20Jin&amp;publication_year=2004&amp;journal=Plant%20Physiol.&amp;volume=135&amp;pages=1198-205" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.103.038463" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.103.038463" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.103.038463"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15266053" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zeng</div> <div class="given-names">Y.X.</div></div>,  <div class="name"><div class="surname">Wen</div> <div class="given-names">Z.H.</div></div>,  <div class="name"><div class="surname">Ma</div> <div class="given-names">L.Y.</div></div>,  <div class="name"><div class="surname">Ji</div> <div class="given-names">Z.J.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">X.M.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">C.D.</div></div>. </div><div class="article-title">Development of 1047 insertion-deletion markers for rice genetic studies and breeding</div>, <div class="source">Genet. Mol. Res.</div> , <div class="year">2012</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">5226</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%201047%20insertion-deletion%20markers%20for%20rice%20genetic%20studies%20and%20breeding&amp;author=Y.X.%20Zeng&amp;author=Z.H.%20Wen&amp;author=L.Y.%20Ma&amp;author=Z.J.%20Ji&amp;author=X.M.%20Li&amp;author=C.D.%20Yang&amp;publication_year=2012&amp;journal=Genet.%20Mol.%20Res.&amp;volume=12&amp;pages=5226-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4238/2013.October.30.7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4238%2F2013.October.30.7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4238%2F2013.October.30.7"> </span></div> </div></div></div></div></div><div content-id="DSU008C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wu</div> <div class="given-names">D.H.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">H.P.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">C.S.</div></div>,  <div class="name"><div class="surname">Tseng</div> <div class="given-names">H.Y.</div></div>,  <div class="name"><div class="surname">Hwu</div> <div class="given-names">K.K.</div></div>. </div><div class="article-title">Genome-wide InDel marker system for application in rice breeding and mapping studies</div>, <div class="source">Euphytica</div> , <div class="year">2013</div>, vol. <div class="volume">192</div> (pg. <div class="fpage">131</div>-<div class="lpage">43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20InDel%20marker%20system%20for%20application%20in%20rice%20breeding%20and%20mapping%20studies&amp;author=D.H.%20Wu&amp;author=H.P.%20Wu&amp;author=C.S.%20Wang&amp;author=H.Y.%20Tseng&amp;author=K.K.%20Hwu&amp;publication_year=2013&amp;journal=Euphytica&amp;volume=192&amp;pages=131-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10681-013-0925-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10681-013-0925-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10681-013-0925-z"> </span></div> </div></div></div></div></div><div content-id="DSU008C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tabata</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">The tomato genome sequence provides insights into fleshy fruit evolution</div>, <div class="source">Nature</div> , <div class="year">2012</div>, vol. <div class="volume">485</div> (pg. <div class="fpage">635</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20tomato%20genome%20sequence%20provides%20insights%20into%20fleshy%20fruit%20evolution&amp;author=S.%20Sato&amp;author=S.%20Tabata&amp;author=H.%20Hirakawa&amp;publication_year=2012&amp;journal=Nature&amp;volume=485&amp;pages=635-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11119" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11119" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11119"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22660326" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koenig</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Jimenez-Gomez</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Kimura</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Comparative transcriptomics reveals patterns of selection in domesticated and wild tomato</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2013</div>, vol. <div class="volume">110</div> (pg. <div class="fpage">E2655</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20transcriptomics%20reveals%20patterns%20of%20selection%20in%20domesticated%20and%20wild%20tomato&amp;author=D.%20Koenig&amp;author=J.M.%20Jimenez-Gomez&amp;author=S.%20Kimura&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=110&amp;pages=E2655-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1309606110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1309606110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1309606110"> </span></div> </div></div></div></div></div><div content-id="DSU008C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kabelka</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Franchino</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Francis</div> <div class="given-names">D.M.</div></div>. </div><div class="article-title">Two loci from <em>Lycopersicon hirsutum</em> LA407 confer resistance to strains of <em>Clavibacter michiganensis</em> subsp. <em>Michiganensis</em></div>, <div class="source">Phytopathology</div> , <div class="year">2002</div>, vol. <div class="volume">92</div> (pg. <div class="fpage">504</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20loci%20from%20Lycopersicon%20hirsutum%20LA407%20confer%20resistance%20to%20strains%20of%20Clavibacter%20michiganensis%20subsp.%20Michiganensis&amp;author=E.%20Kabelka&amp;author=B.%20Franchino&amp;author=D.M.%20Francis&amp;publication_year=2002&amp;journal=Phytopathology&amp;volume=92&amp;pages=504-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/PHYTO.2002.92.5.504" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FPHYTO.2002.92.5.504" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FPHYTO.2002.92.5.504"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18943024" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Angiuoli</div> <div class="given-names">S.V.</div></div>,  <div class="name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div>. </div><div class="article-title">Mugsy: fast multiple alignment of closely related whole genomes</div>, <div class="source">Bioinformatics</div> , <div class="year">2011</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">334</div>-<div class="lpage">42</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mugsy%3A%20fast%20multiple%20alignment%20of%20closely%20related%20whole%20genomes&amp;author=S.V.%20Angiuoli&amp;author=S.L.%20Salzberg&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=334-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq665" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq665" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq665"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21148543" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rozen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Skaletsky</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Primer3 on the WWW for general users and for biologist programmers</div>, <div class="source">Methods Mol. Biol.</div> , <div class="year">2000</div>, vol. <div class="volume">132</div> (pg. <div class="fpage">365</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Primer3%20on%20the%20WWW%20for%20general%20users%20and%20for%20biologist%20programmers&amp;author=S.%20Rozen&amp;author=H.%20Skaletsky&amp;publication_year=2000&amp;journal=Methods%20Mol.%20Biol.&amp;volume=132&amp;pages=365-86" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10547847" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wei</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Hu</div> <div class="given-names">X.R.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">J.J.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">W.C.</div></div>. </div><div class="article-title">Identification of single-copy orthologous genes between <em>Physalis</em> and <em>Solanum lycopersicum</em> and analysis of genetic diversity in <em>Physalis</em> using molecular markers</div>, <div class="source">PLoS ONE</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e50164</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20single-copy%20orthologous%20genes%20between%20Physalis%20and%20Solanum%20lycopersicum%20and%20analysis%20of%20genetic%20diversity%20in%20Physalis%20using%20molecular%20markers&amp;author=J.L.%20Wei&amp;author=X.R.%20Hu&amp;author=J.J.%20Yang&amp;author=W.C.%20Yang&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e50164" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0050164" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0050164" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0050164"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23166835" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Genetic distance between populations</div>, <div class="source">Am. Nat.</div> , <div class="year">1972</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">283</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20distance%20between%20populations&amp;author=M.%20Nei&amp;publication_year=1972&amp;journal=Am.%20Nat.&amp;volume=106&amp;pages=283-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/282771" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F282771" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F282771"> </span></div> </div></div></div></div></div><div content-id="DSU008C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ribaut</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">de Vicente</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Delannay</div> <div class="given-names">X.</div></div>. </div><div class="article-title">Molecular breeding in developing countries: challenges and perspectives</div>, <div class="source">Curr. Opin. Plant Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">1</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20breeding%20in%20developing%20countries%3A%20challenges%20and%20perspectives&amp;author=J.M.%20Ribaut&amp;author=M.C.%20de%20Vicente&amp;author=X.%20Delannay&amp;publication_year=2010&amp;journal=Curr.%20Opin.%20Plant%20Biol.&amp;volume=13&amp;pages=1-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2009.12.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2009.12.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2009.12.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20047852" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jiang</div> <div class="given-names">G.L.</div></div>. </div><div class="article-title">Plant marker-assisted breeding and conventional breeding: challenges and perspectives</div>, <div class="source">Adv. Crop Sci. Technol.</div> , <div class="year">2013</div>, vol. <div class="volume">1</div> pg. <div class="fpage">e106</div> </div></div></div></div><div content-id="DSU008C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Weber</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">David</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Heil</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Fan</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Marth</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Human diallelic insertion/deletion polymorphisms</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">71</div> (pg. <div class="fpage">854</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20diallelic%20insertion%2Fdeletion%20polymorphisms&amp;author=J.L.%20Weber&amp;author=D.%20David&amp;author=J.%20Heil&amp;author=Y.%20Fan&amp;author=C.%20Zhao&amp;author=G.%20Marth&amp;publication_year=2002&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=71&amp;pages=854-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/342727" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F342727" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F342727"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12205564" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Oka</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Asari</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Omura</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Genotyping of 38 insertion/deletion polymorphisms for human identification using universal fluorescent PCR</div>, <div class="source">Mol. Cell. Probes</div> , <div class="year">2014</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">13</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genotyping%20of%2038%20insertion%2Fdeletion%20polymorphisms%20for%20human%20identification%20using%20universal%20fluorescent%20PCR&amp;author=K.%20Oka&amp;author=M.%20Asari&amp;author=T.%20Omura&amp;publication_year=2014&amp;journal=Mol.%20Cell.%20Probes&amp;volume=28&amp;pages=13-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mcp.2013.09.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mcp.2013.09.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mcp.2013.09.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24075877" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pereira</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Phillips</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Alves</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Amorim</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Carracedo</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Gusmão</div> <div class="given-names">L.</div></div>. </div><div class="article-title">A new multiplex for human identification using insertion/deletion polymorphisms</div>, <div class="source">Electrophoresis</div> , <div class="year">2009</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">3682</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20multiplex%20for%20human%20identification%20using%20insertion%2Fdeletion%20polymorphisms&amp;author=R.%20Pereira&amp;author=C.%20Phillips&amp;author=C.%20Alves&amp;author=A.%20Amorim&amp;author=A.%20Carracedo&amp;author=L.%20Gusm%C3%A3o&amp;publication_year=2009&amp;journal=Electrophoresis&amp;volume=30&amp;pages=3682-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/elps.v30:21" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Felps.v30:21" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Felps.v30:21"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19862748" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">C.T.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">S.H.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">S.M.</div></div>. </div><div class="article-title">Genetic analysis of 30 InDel markers for forensic use in five different Chinese populations</div>, <div class="source">Genet. Mol. Res.</div> , <div class="year">2011</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">964</div>-<div class="lpage">79</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20analysis%20of%2030%20InDel%20markers%20for%20forensic%20use%20in%20five%20different%20Chinese%20populations&amp;author=C.T.%20Li&amp;author=S.H.%20Zhang&amp;author=S.M.%20Zhao&amp;publication_year=2011&amp;journal=Genet.%20Mol.%20Res.&amp;volume=10&amp;pages=964-79" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4238/vol10-2gmr1082" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4238%2Fvol10-2gmr1082" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4238%2Fvol10-2gmr1082"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21644214" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Genetic polymorphism of 29 highly informative InDel markers for forensic use in the Chinese Han population</div>, <div class="source">Forensic Sci. Int. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">e27</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20polymorphism%20of%2029%20highly%20informative%20InDel%20markers%20for%20forensic%20use%20in%20the%20Chinese%20Han%20population&amp;author=C.%20Li&amp;author=S.%20Zhao&amp;author=S.%20Zhang&amp;publication_year=2011&amp;journal=Forensic%20Sci.%20Int.%20Genet.&amp;volume=5&amp;pages=e27-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fsigen.2010.03.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fsigen.2010.03.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fsigen.2010.03.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20457100" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fondevila</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Phillips</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Santos</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Forensic performance of two insertion-deletion marker assays</div>, <div class="source">Int. J. Legal Med.</div> , <div class="year">2012</div>, vol. <div class="volume">126</div> (pg. <div class="fpage">725</div>-<div class="lpage">37</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Forensic%20performance%20of%20two%20insertion-deletion%20marker%20assays&amp;author=M.%20Fondevila&amp;author=C.%20Phillips&amp;author=C.%20Santos&amp;publication_year=2012&amp;journal=Int.%20J.%20Legal%20Med.&amp;volume=126&amp;pages=725-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00414-012-0721-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00414-012-0721-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00414-012-0721-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22714117" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Menda</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Strickler</div> <div class="given-names">S.R.</div></div>,  <div class="name"><div class="surname">Mueller</div> <div class="given-names">L.A</div></div>. </div><div class="article-title">Advances in tomato research in the post-genome era</div>, <div class="source">Plant Biotechnol.</div> , <div class="year">2013</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">243</div>-<div class="lpage">56</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Advances%20in%20tomato%20research%20in%20the%20post-genome%20era&amp;author=N.%20Menda&amp;author=S.R.%20Strickler&amp;author=L.A%20Mueller&amp;publication_year=2013&amp;journal=Plant%20Biotechnol.&amp;volume=30&amp;pages=243-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.5511/plantbiotechnology.13.0904a" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.5511%2Fplantbiotechnology.13.0904a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.5511%2Fplantbiotechnology.13.0904a"> </span></div> </div></div></div></div></div><div content-id="DSU008C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shendure</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Ji</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Next-generation DNA sequencing</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2008</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">1135</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next-generation%20DNA%20sequencing&amp;author=J.%20Shendure&amp;author=H.%20Ji&amp;publication_year=2008&amp;journal=Nat.%20Biotechnol.&amp;volume=26&amp;pages=1135-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1486" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1486" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1486"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18846087" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Causse</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Desplat</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Pascual</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Whole genome resequencing in tomato reveals variation associated with introgression and breeding events</div>, <div class="source">BMC Genomics</div> , <div class="year">2013</div>, vol. <div class="volume">14</div> pg. <div class="fpage">791</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole%20genome%20resequencing%20in%20tomato%20reveals%20variation%20associated%20with%20introgression%20and%20breeding%20events&amp;author=M.%20Causse&amp;author=M.%20Desplat&amp;author=L.%20Pascual&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=791" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-791" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-791" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-791"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24228636" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bai</div> <div class="given-names">Y.L.</div></div>,  <div class="name"><div class="surname">Lindhout</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Domestication and breeding of tomatoes: what have we gained and what can we gain in the future?</div>, <div class="source">Ann. Bot.</div> , <div class="year">2007</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">1085</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Domestication%20and%20breeding%20of%20tomatoes%3A%20what%20have%20we%20gained%20and%20what%20can%20we%20gain%20in%20the%20future%3F&amp;author=Y.L.%20Bai&amp;author=P.%20Lindhout&amp;publication_year=2007&amp;journal=Ann.%20Bot.&amp;volume=100&amp;pages=1085-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/aob/mcm150" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Faob%2Fmcm150" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Faob%2Fmcm150"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17717024" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Archak</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Karihaloo</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Jain</div> <div class="given-names">A.</div></div>. </div><div class="article-title">RAPD markers reveal narrowing genetic base of Indian tomato cultivars</div>, <div class="source">Curr. Sci.</div> , <div class="year">2002</div>, vol. <div class="volume">82</div> (pg. <div class="fpage">1139</div>-<div class="lpage">43</div>)</div></div></div></div><div content-id="DSU008C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hu</div> <div class="given-names">X.R.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">W.C.</div></div>. </div><div class="article-title">Genetic diversity of Argentina tomato varieties revealed by morphological traits, simple sequence repeat, and single nucleotide polymorphism markers</div>, <div class="source">Pak. J. Bot.</div> , <div class="year">2012</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">485</div>-<div class="lpage">92</div>)</div></div></div></div><div content-id="DSU008C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rick</div> <div class="given-names">C.M.</div></div>,  <div class="name"><div class="surname">Fobes</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Allozyme variation in the cultivated tomato and closely related species</div>, <div class="source">Bull. Torr. Bot. Club</div> , <div class="year">1975</div>, vol. <div class="volume">102</div> (pg. <div class="fpage">376</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Allozyme%20variation%20in%20the%20cultivated%20tomato%20and%20closely%20related%20species&amp;author=C.M.%20Rick&amp;author=F.%20Fobes&amp;publication_year=1975&amp;journal=Bull.%20Torr.%20Bot.%20Club&amp;volume=102&amp;pages=376-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2484764" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2484764" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2484764"> </span></div> </div></div></div></div></div><div content-id="DSU008C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ranc</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Muños</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Santoni</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Causse</div> <div class="given-names">M.</div></div>. </div><div class="article-title">A clarified position for <em>Solanum lycopersicum</em> var. <em>cerasiforme</em> in the evolutionary history of tomatoes (solanaceae)</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">8</div> pg. <div class="fpage">130</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20clarified%20position%20for%20Solanum%20lycopersicum%20var.%20cerasiforme%20in%20the%20evolutionary%20history%20of%20tomatoes%20(solanaceae)&amp;author=N.%20Ranc&amp;author=S.%20Mu%C3%B1os&amp;author=S.%20Santoni&amp;author=M.%20Causse&amp;publication_year=2008&amp;journal=BMC%20Plant%20Biol.&amp;volume=8&amp;pages=130" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-8-130" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-8-130" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-8-130"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19099601" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Ng</div> <div class="given-names">P.C.</div></div>. </div><div class="article-title">SIFT Indel: predictions for the functional effects of amino acid insertions/deletions in proteins</div>, <div class="source">PLoS ONE</div> , <div class="year">2013</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e77940</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SIFT%20Indel%3A%20predictions%20for%20the%20functional%20effects%20of%20amino%20acid%20insertions%2Fdeletions%20in%20proteins&amp;author=J.%20Hu&amp;author=P.C.%20Ng&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e77940" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0077940" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0077940" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0077940"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24194902" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cheng</div> <div class="given-names">X.D.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">D.F.</div></div>,  <div class="name"><div class="surname">Cheng</div> <div class="given-names">Z.K.</div></div>,  <div class="name"><div class="surname">Keller</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Ling</div> <div class="given-names">H.Q.</div></div>. </div><div class="article-title">A new family of Ty1-copia-like retrotransposons originated in the tomato genome by a recent horizontal transfer event</div>, <div class="source">Genetics</div> , <div class="year">2009</div>, vol. <div class="volume">181</div> (pg. <div class="fpage">1183</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20family%20of%20Ty1-copia-like%20retrotransposons%20originated%20in%20the%20tomato%20genome%20by%20a%20recent%20horizontal%20transfer%20event&amp;author=X.D.%20Cheng&amp;author=D.F.%20Zhang&amp;author=Z.K.%20Cheng&amp;author=B.%20Keller&amp;author=H.Q.%20Ling&amp;publication_year=2009&amp;journal=Genetics&amp;volume=181&amp;pages=1183-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.108.099150" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.108.099150" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.108.099150"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19153256" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Xiao</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">A single-base deletion mutation in <em>SlIAA9</em> gene causes tomato (<em>Solanum lycopersicum</em>) entire mutant</div>, <div class="source">J. Plant Res.</div> , <div class="year">2007</div>, vol. <div class="volume">120</div> (pg. <div class="fpage">671</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20single-base%20deletion%20mutation%20in%20SlIAA9%20gene%20causes%20tomato%20(Solanum%20lycopersicum)%20entire%20mutant&amp;author=J.%20Zhang&amp;author=R.%20Chen&amp;author=J.%20Xiao&amp;publication_year=2007&amp;journal=J.%20Plant%20Res.&amp;volume=120&amp;pages=671-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10265-007-0109-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10265-007-0109-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10265-007-0109-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17955175" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ruíz-Rivero</div> <div class="given-names">O.J.</div></div>,  <div class="name"><div class="surname">Prat</div> <div class="given-names">S.</div></div>. </div><div class="article-title">A-308 deletion of the tomato LAP promoters is able to direct flower-specific and MeJA-induced expression in transgenic plants</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">1998</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">639</div>-<div class="lpage">48</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A-308%20deletion%20of%20the%20tomato%20LAP%20promoters%20is%20able%20to%20direct%20flower-specific%20and%20MeJA-induced%20expression%20in%20transgenic%20plants&amp;author=O.J.%20Ru%C3%ADz-Rivero&amp;author=S.%20Prat&amp;publication_year=1998&amp;journal=Plant%20Mol.%20Biol.&amp;volume=36&amp;pages=639-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1005980028203" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1005980028203" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1005980028203"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9526496" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU008C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU008C61" class="ref-content "><div class="label -label">61</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ni</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Sun</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Identification and analysis of resistance-like genes in the tomato genome</div>, <div class="source">J. Phytopathol.</div> , <div class="year">2013</div>, vol. <div class="volume">162</div> (pg. <div class="fpage">137</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20analysis%20of%20resistance-like%20genes%20in%20the%20tomato%20genome&amp;author=X.%20Ni&amp;author=J.%20Yang&amp;author=S.%20Sun&amp;author=W.%20Yang&amp;publication_year=2013&amp;journal=J.%20Phytopathol.&amp;volume=162&amp;pages=137-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jph.2014.162.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjph.2014.162.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjph.2014.162.issue-3"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=447212&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/dsu008-dsu008supp_table4.xls?Expires=2147483647&amp;Signature=TIgSQ1Pq27N31lG0HQwcF2OEudiomZM5V0oNl1QyXL8DewHfKSc2sN5SyiUF0p9tGInbGdDgQD1V2Ue6vag6ar6wbndxKJbDjZ1TiIm6PmJsQSqlMeWsyxPINYv05gdQ2bQPT-bs5btQ24xRnfFTT2z5lsWATdiUGoo-Ut3v1PvB5Y3qfH~5RlMiucL4l9OOCL0YJI4MMqqK1qf5fKvtp4ViWONhPj56TvuhjCMETLY4seZTBpuO-n~hhkHD44f08W5s2LZ4aOueeWK9XUC0kXicw-i-ppi4tAicoKu-x6QzkIUbUW2xJE1xwAP-VwihkEsXYLyE7hYJg72GfpuFQA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/dsu008-dsu008supp_table3.xls?Expires=2147483647&amp;Signature=IHbCNqzKftT2nXU2Iudc3r~~Fu5FHNZtELGBBgXUqXGDCsf7aK~MrnFeIQA8beuR6i-YDpGBjFCVUfwtxAcL-oOgeeivnX9ye5YZIYyPy1NvdUN7~mUwcq9WVz860j1tNAcDFSfyvxbWsVfz0XCTJkaVhwT5BcVSuUVwWcEStT8E052SMHOXd~xYJBYtglPWBC6rLfKgp42dzrbsr-eDDm5bT3wMA0Ocej5RRLuNpqE0xli~YYH6cgvM-JG1Rc1a4g429eefARNw5ucga1pgfvHP9chz4KgPvoP1pMWDqYjEEl0vOC2PIvcBmrLeURT3Uhe4UWbUi7Xljxcj7esrtQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/dsu008-dsu008supp_table2.xls?Expires=2147483647&amp;Signature=bKzWtvM4dTIH5UZ~0wKatUYKVdg7ZfYSGEpq4hMd2IURnmFiYLqH2rpr6mUjJgi-JAtW~fBe6mu7Fydz1EMzUmKUOXqXq~F7x8eDRfS8wVMKudgT07jMQiLhR3hh2-2LrEW2dg-waNOYmW2QNP-78DIXD-ZYgRuSS-dunoQ45HhfZIBkWD9RonGME9ay-qVuwKnsAmtCal~Eh573EwhRhd58R4b1pKkKuXne-fG8ZND9QfXsmFqcFojkVORgAjme2y7pM18EOngR3ievfe4bhTiq9ZprbP9d4ti1kP5f62L4uY-YZEfuiGkCstcQdFZlBhPA-AbmpljxiHGNBs8ZUw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/dsu008-dsu008supp_table1.xls?Expires=2147483647&amp;Signature=axldv7m1DC80VAEODrP1GNTc-RVflSW2vIEhN6h5mWs7DSCDJbseh5TxVTSFNrkiexZUJEMPfuECWrddzjFoh2p6qztBn4jezkjGRSbTtpjkQxMJ7xG1l2oOW7NPPNpynAGyLv4cz-SP7WuRAkVzfEWLjaifQynH8vCYA-ISuwaTI8-A~DFtlQmSk7Td-UUyO8Tlb8SPJzBEDW8~ED3SraHiMCZJeQrPq5u29KkiOcL4WxSJo0CNPu7MvjVL8b0LnjoBWY3ZdqMcsbeM0Z3qHD3ldDIqmMj5Mg3MoUahfRV4bFCp1QGB-z2zVCEogOIrvMUCWYs7S3qCeUkeK45jJg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/dsu008-dsu008supp_fig2.pdf?Expires=2147483647&amp;Signature=lNZOFNJhAFo9PAK8eZCoOu28a2kZw7Zkn0CxKiQlF9LqfvRY800fxf0UAyw30ZfMOHdibnPzggh-dJb1JMq9ePCOdKEGk9ae-Kvp5IQztfaIf128EFqGxc6NoeWeeKzH0-8Bt8cEZPTW5a8AxyWv6WsJVJY3RB-O84xVqCtejEjumYNBUZUkQoX5oghQn5a3PP82qdbtKgtks5C66K3lfaAld9hw4Kooql-XIROsrTTdIObSDOODaSUYJhP1AkIJWEhqzHbi9QMNyqkm0WgKU8nUnAr4MlfSCMt6yZJSvEV-M-k60cOXn2Xxdg3j1llOcqkncc8ej6AvXPGSdAXu8w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/dsu008-dsu008supp_fig1.png?Expires=2147483647&amp;Signature=sfaQZQDPa7QOQ~~dw6QaXaCq9wiYfDQVkk7wFGbJnQf3DAxtxcr8auDsF70s3DfSCL8~HsHFViQXZLWQx3yqn7iQogjO8v0vrTKgDzZx4AE38eTASLxXhTOjrSPU~Ejjscf6~lo9yV5uVjLgbeP4EhC9J5umvghLAg8PPF97AG3xSHExNidKpPQdQcU2MSsc7waQxKfnQuDd0nsmqGskmG-smPRMFYaVBRMErxfohi1anr2Nv2JvpFKQsD8Vsz8QNovZUog3CJ9iR2XagLAq9nDHJRamYBIamurRMFbIO5h~mhv2o73EEqtlCj61aDrX3lxA7GFgmU54n5vyaf~aXA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu008/2/dsu008-dsu008supp.doc?Expires=2147483647&amp;Signature=JQ1OKzh6aAjsyrCxazouiQWG1IexLM7zG-l3y5SwBPouaN7ldaoEAd8bTOo1ds7kknpFEK6AU-09UoqsVQB4eI7LBRJSScyQmu4Qglh36wgtzzI37UZ8Jsf7xbd6d~Zrke1~~tdNi~JO66CS4MLxcVihKVBnvxUYDvQWWa6UDtY2B9yAflJHxHAEMUKMIvUFIXaopGzMLSrbMfGzuwtO3EA7Ieo-2vsOyh-pfF6Hh6AX3YWfIC4lQzrnGYvBcIvmyL4eCBG8EVCqWN0WkiAxVYQTYrHaqZROUVjA7hLCX4-rh-2UYqc3dRzngw54S4iC6Noc2S8Y5euhzKXApCE~4A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">385</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000344243900008&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">15</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsu008"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="447212"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000344243900008&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f4%2f429%2f447212">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000344243900008&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (15)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F4%2F429%2F447212" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/447212'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="In Silico Identification and Experimental Validation of Insertion–Deletion Polymorphisms in Tomato Genome" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsu008");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f4%2f429%2f447212">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

