

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/17/3/123/353305","@type":"ScholarlyArticle","name":"Bacterial Lifestyle in a Deep-sea Hydrothermal Vent Chimney Revealed by the Genome Sequence of the Thermophilic Bacterium  Deferribacter desulfuricans  SSM1","datePublished":"2010-02-26","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/17/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2010-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsq005","keywords":["<italic>Deferribacter desulfuricans</italic>","whole genome sequence","hydrothermal vent","chemolithotroph"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/17/3/123/353305","author":[{"name":"Takaki, Yoshihiro","affiliation":"1  Microbial Genome Research Group, Extremobiosphere Research Program, Institute of Biogeosciences, Japan Agency for Marine-Earth Science and Technology (JAMSTEC) ,  Yokosuka, Kanagawa 237-0061 ,  Japan","@type":"Person"},{"name":"Shimamura, Shigeru","affiliation":"2  Subsurface Geobiology Advanced Research Team (SUGAR) Team, Extremobiosphere Research Program, Institute of Biogeosciences, JAMSTEC ,  Yokosuka, Kanagawa 237-0061 ,  Japan","@type":"Person"},{"name":"Nakagawa, Satoshi","affiliation":"2  Subsurface Geobiology Advanced Research Team (SUGAR) Team, Extremobiosphere Research Program, Institute of Biogeosciences, JAMSTEC ,  Yokosuka, Kanagawa 237-0061 ,  Japan    3  Laboratory of Microbiology, Faculty of Fisheries ,  Hokkaido University ,  3-1-1 Minato-cho, Hakodate 041-8611 ,  Japan","@type":"Person"},{"name":"Fukuhara, Yasuo","affiliation":"4  NITE Genome Analysis Center, Department of Biotechnology ,  National Institute of Technology and Evolution (NITE) ,  2-10-49 Nishihara, Shibuya-ku, Tokyo 151-0066 ,  Japan","@type":"Person"},{"name":"Horikawa, Hiroshi","affiliation":"4  NITE Genome Analysis Center, Department of Biotechnology ,  National Institute of Technology and Evolution (NITE) ,  2-10-49 Nishihara, Shibuya-ku, Tokyo 151-0066 ,  Japan","@type":"Person"},{"name":"Ankai, Akiho","affiliation":"4  NITE Genome Analysis Center, Department of Biotechnology ,  National Institute of Technology and Evolution (NITE) ,  2-10-49 Nishihara, Shibuya-ku, Tokyo 151-0066 ,  Japan","@type":"Person"},{"name":"Harada, Takeshi","affiliation":"4  NITE Genome Analysis Center, Department of Biotechnology ,  National Institute of Technology and Evolution (NITE) ,  2-10-49 Nishihara, Shibuya-ku, Tokyo 151-0066 ,  Japan","@type":"Person"},{"name":"Hosoyama, Akira","affiliation":"4  NITE Genome Analysis Center, Department of Biotechnology ,  National Institute of Technology and Evolution (NITE) ,  2-10-49 Nishihara, Shibuya-ku, Tokyo 151-0066 ,  Japan","@type":"Person"},{"name":"Oguchi, Akio","affiliation":"4  NITE Genome Analysis Center, Department of Biotechnology ,  National Institute of Technology and Evolution (NITE) ,  2-10-49 Nishihara, Shibuya-ku, Tokyo 151-0066 ,  Japan","@type":"Person"},{"name":"Fukui, Shigehiro","affiliation":"4  NITE Genome Analysis Center, Department of Biotechnology ,  National Institute of Technology and Evolution (NITE) ,  2-10-49 Nishihara, Shibuya-ku, Tokyo 151-0066 ,  Japan","@type":"Person"},{"name":"Fujita, Nobuyuki","affiliation":"4  NITE Genome Analysis Center, Department of Biotechnology ,  National Institute of Technology and Evolution (NITE) ,  2-10-49 Nishihara, Shibuya-ku, Tokyo 151-0066 ,  Japan","@type":"Person"},{"name":"Takami, Hideto","affiliation":"1  Microbial Genome Research Group, Extremobiosphere Research Program, Institute of Biogeosciences, Japan Agency for Marine-Earth Science and Technology (JAMSTEC) ,  Yokosuka, Kanagawa 237-0061 ,  Japan","@type":"Person"},{"name":"Takai, Ken","affiliation":"2  Subsurface Geobiology Advanced Research Team (SUGAR) Team, Extremobiosphere Research Program, Institute of Biogeosciences, JAMSTEC ,  Yokosuka, Kanagawa 237-0061 ,  Japan","@type":"Person"}],"description":"Abstract.  The complete genome sequence of the thermophilic sulphur-reducing bacterium, Deferribacter desulfuricans SMM1, isolated from a hydrothermal vent chim","pageStart":"123","pageEnd":"137","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/3/1/m_cover.gif?Expires=2147483647&Signature=zrlqOYlr8puhWaI06gf8sYMMP6iQcSaE6WnxD18CI1vM-vSYmV7zw1fL1UUm1LAo6OR8YaC2ow7Gg3gFyOuRxjrsW2e9seZ-Fc4W0zpKOKeLDqycZiol4B5gPhMqbsAY9k5dM2sXst3uhEwLyf9p6Clk-Cv9XLljwpdq37Ye9OThamAMiMSAoDxynPtt~5t-2ainSI295gFWPk3t9zFxKYx4zJXORsGXGnkmtct97jCcoDB0mHB0FSIfJFRj9SkO~ccPVDFJ5bkP0CQlUseB~SQw04UmN2w0van9EOcwauxu2Sl9Oe1AFiN92gYP2rZEbuX5qeGuIN6geNJvEhApPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Bacterial Lifestyle in a Deep-sea Hydrothermal Vent Chimney Revealed by the Genome Sequence of the Thermophilic Bacterium  Deferribacter desulfuricans  SSM1","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/3/1/m_cover.gif?Expires=2147483647&Signature=zrlqOYlr8puhWaI06gf8sYMMP6iQcSaE6WnxD18CI1vM-vSYmV7zw1fL1UUm1LAo6OR8YaC2ow7Gg3gFyOuRxjrsW2e9seZ-Fc4W0zpKOKeLDqycZiol4B5gPhMqbsAY9k5dM2sXst3uhEwLyf9p6Clk-Cv9XLljwpdq37Ye9OThamAMiMSAoDxynPtt~5t-2ainSI295gFWPk3t9zFxKYx4zJXORsGXGnkmtct97jCcoDB0mHB0FSIfJFRj9SkO~ccPVDFJ5bkP0CQlUseB~SQw04UmN2w0van9EOcwauxu2Sl9Oe1AFiN92gYP2rZEbuX5qeGuIN6geNJvEhApPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Bacterial Lifestyle in a Deep-sea Hydrothermal Vent Chimney Revealed by the Genome Sequence of the Thermophilic Bacterium  Deferribacter desulfuricans  SSM1" />
    <meta property="og:description" content="Abstract.  The complete genome sequence of the thermophilic sulphur-reducing bacterium, Deferribacter desulfuricans SMM1, isolated from a hydrothermal vent chim" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/17/3/123/353305" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2010-02-26" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Takaki, Yoshihiro" /><meta name="citation_author_institution" content="Microbial Genome Research Group, Extremobiosphere Research Program, Institute of Biogeosciences, Japan Agency for Marine-Earth Science and Technology (JAMSTEC), Yokosuka, Kanagawa 237-0061, Japan" /><meta name="citation_author" content="Shimamura, Shigeru" /><meta name="citation_author_institution" content="Subsurface Geobiology Advanced Research Team (SUGAR) Team, Extremobiosphere Research Program, Institute of Biogeosciences, JAMSTEC, Yokosuka, Kanagawa 237-0061, Japan" /><meta name="citation_author" content="Nakagawa, Satoshi" /><meta name="citation_author_institution" content="Subsurface Geobiology Advanced Research Team (SUGAR) Team, Extremobiosphere Research Program, Institute of Biogeosciences, JAMSTEC, Yokosuka, Kanagawa 237-0061, Japan" /><meta name="citation_author_institution" content="Laboratory of Microbiology, Faculty of Fisheries, Hokkaido University, 3-1-1 Minato-cho, Hakodate 041-8611, Japan" /><meta name="citation_author" content="Fukuhara, Yasuo" /><meta name="citation_author_institution" content="NITE Genome Analysis Center, Department of Biotechnology, National Institute of Technology and Evolution (NITE), 2-10-49 Nishihara, Shibuya-ku, Tokyo 151-0066, Japan" /><meta name="citation_author" content="Horikawa, Hiroshi" /><meta name="citation_author_institution" content="NITE Genome Analysis Center, Department of Biotechnology, National Institute of Technology and Evolution (NITE), 2-10-49 Nishihara, Shibuya-ku, Tokyo 151-0066, Japan" /><meta name="citation_author" content="Ankai, Akiho" /><meta name="citation_author_institution" content="NITE Genome Analysis Center, Department of Biotechnology, National Institute of Technology and Evolution (NITE), 2-10-49 Nishihara, Shibuya-ku, Tokyo 151-0066, Japan" /><meta name="citation_author" content="Harada, Takeshi" /><meta name="citation_author_institution" content="NITE Genome Analysis Center, Department of Biotechnology, National Institute of Technology and Evolution (NITE), 2-10-49 Nishihara, Shibuya-ku, Tokyo 151-0066, Japan" /><meta name="citation_author" content="Hosoyama, Akira" /><meta name="citation_author_institution" content="NITE Genome Analysis Center, Department of Biotechnology, National Institute of Technology and Evolution (NITE), 2-10-49 Nishihara, Shibuya-ku, Tokyo 151-0066, Japan" /><meta name="citation_author" content="Oguchi, Akio" /><meta name="citation_author_institution" content="NITE Genome Analysis Center, Department of Biotechnology, National Institute of Technology and Evolution (NITE), 2-10-49 Nishihara, Shibuya-ku, Tokyo 151-0066, Japan" /><meta name="citation_author" content="Fukui, Shigehiro" /><meta name="citation_author_institution" content="NITE Genome Analysis Center, Department of Biotechnology, National Institute of Technology and Evolution (NITE), 2-10-49 Nishihara, Shibuya-ku, Tokyo 151-0066, Japan" /><meta name="citation_author" content="Fujita, Nobuyuki" /><meta name="citation_author_institution" content="NITE Genome Analysis Center, Department of Biotechnology, National Institute of Technology and Evolution (NITE), 2-10-49 Nishihara, Shibuya-ku, Tokyo 151-0066, Japan" /><meta name="citation_author" content="Takami, Hideto" /><meta name="citation_author_institution" content="Microbial Genome Research Group, Extremobiosphere Research Program, Institute of Biogeosciences, Japan Agency for Marine-Earth Science and Technology (JAMSTEC), Yokosuka, Kanagawa 237-0061, Japan" /><meta name="citation_author" content="Takai, Ken" /><meta name="citation_author_institution" content="Subsurface Geobiology Advanced Research Team (SUGAR) Team, Extremobiosphere Research Program, Institute of Biogeosciences, JAMSTEC, Yokosuka, Kanagawa 237-0061, Japan" /><meta name="citation_title" content="Bacterial Lifestyle in a Deep-sea Hydrothermal Vent Chimney Revealed by the Genome Sequence of the Thermophilic Bacterium Deferribacter desulfuricans SSM1" /><meta name="citation_firstpage" content="123" /><meta name="citation_lastpage" content="137" /><meta name="citation_doi" content="10.1093/dnares/dsq005" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="17" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2010/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_publisher=Copernicus Books, New York; The Deep Hot Biosphere.;  citation_year=2001;  citation_pages=19-22; " /><meta name="citation_reference" content="citation_title=Deferribacter desulfuricans sp. nov., a novel sulfur-, nitrate- and arsenate-reducing thermophile isolated from a deep-sea hydrothermal vent; Int. J. Syst. Evol. Microbiol.;  citation_year=2003;  citation_volume=53;  citation_pages=839-46; " /><meta name="citation_reference" content="citation_title=Deferribacter abyssi sp. nov., an anaerobic thermophile from deep-sea hydrothermal vents of the Mid-Atlantic Ridge; Int. J. Syst. Evol. Microbiol.;  citation_year=2003;  citation_volume=53;  citation_pages=1637-41; " /><meta name="citation_reference" content="citation_title=Deferribacter thermophilus gen. nov., sp. nov., a novel thermophilic manganese- and iron-reducing bacterium isolated from a petroleum reservoir; Int. J. Syst. Bacteriol.;  citation_year=1997;  citation_volume=47;  citation_pages=505-9; " /><meta name="citation_reference" content="citation_title=Deferribacter autotrophicus sp. nov., an iron(III)-reducing bacterium from a deep-sea hydrothermal vent; Int. J. Syst. Evol. Microbiol.;  citation_year=2009;  citation_volume=59;  citation_pages=1508-12; " /><meta name="citation_reference" content="citation_title=Distribution, phylogenetic diversity and physiological characteristics of epsilon-Proteobacteria in a deep-sea hydrothermal field; Environ. Microbiol.;  citation_year=2005;  citation_volume=7;  citation_pages=1619-32; " /><meta name="citation_reference" content="citation_title=Evidence for the presence of thermophilic Fe(III)-reducing microorganisms in deep-sea hydrothermal vents at 13 degrees N (East Pacific Rise); FEMS Microbiol. Ecol.;  citation_year=2001;  citation_volume=36;  citation_pages=235-43; " /><meta name="citation_reference" content="citation_title=Variability in the microbial communities and hydrothermal fluid chemistry at the newly discovered Mariner hydrothermal field, southern Lau Basin; J. Geophys. Res.-Biogeosci.;  citation_year=2008;  citation_volume=113;  citation_pages=G02031" /><meta name="citation_reference" content="citation_title=The genome of deep-sea vent chemolithoautotroph Thiomicrospira crunogena XCL-2; PLoS Biol.;  citation_year=2006;  citation_volume=4;  citation_pages=e383" /><meta name="citation_reference" content="citation_title=Adaptations to submarine hydrothermal environments exemplified by the genome of Nautilia profundicola; PLoS Genet.;  citation_year=2009;  citation_volume=5;  citation_pages=e1000362" /><meta name="citation_reference" content="citation_title=Deep-sea vent ε-proteobacterial genomes provide insights into emergence of pathogens; Proc. Natl. Acad. Sci. USA;  citation_year=2007;  citation_volume=104;  citation_pages=12146-50; " /><meta name="citation_reference" content="citation_title=Complete and draft genome sequences of six members of the Aquificales; J. Bacteriol.;  citation_year=2009;  citation_volume=191;  citation_pages=1992-3; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of the soil actinomycete Kocuria rhizophila; J. Bacteriol.;  citation_year=2008;  citation_volume=190;  citation_pages=4139-46; " /><meta name="citation_reference" content="citation_title=Base-calling of automated sequencer traces using phred. I. Accuracy assessment; Genome Res.;  citation_year=1998;  citation_volume=8;  citation_pages=175-85; " /><meta name="citation_reference" content="citation_title=Rfam: an RNA family database; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=439-41; " /><meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=955-64; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=Improved microbial gene identification with GLIMMER; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=4636-41; " /><meta name="citation_reference" content="citation_title=GeneMarkS: a self-training method for prediction of gene starts in microbial genomes. Implications for finding sequence motifs in regulatory regions; Nucleic Acids Res.;  citation_year=2001;  citation_volume=29;  citation_pages=2607-18; " /><meta name="citation_reference" content="citation_title=GenBank; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=D34-8; " /><meta name="citation_reference" content="citation_title=The Universal Protein Resource (UniProt); Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=D154-9; " /><meta name="citation_reference" content="citation_title=The KEGG resource for deciphering the genome; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=D277-80; " /><meta name="citation_reference" content="citation_title=The Pfam protein families database; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D281-8; " /><meta name="citation_reference" content="citation_title=The MetaCyc Database of metabolic pathways and enzymes and the BioCyc collection of Pathway/Genome Databases; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D623-31; " /><meta name="citation_reference" content="citation_title=Improved prediction of signal peptides: SignalP 3.0; J. Mol. Biol.;  citation_year=2004;  citation_volume=340;  citation_pages=783-95; " /><meta name="citation_reference" content="citation_title=SOSUI: classification and secondary structure prediction system for membrane proteins; Bioinformatics;  citation_year=1998;  citation_volume=14;  citation_pages=378-9; " /><meta name="citation_reference" content="citation_title=CRISPRFinder: a web tool to identify clustered regularly interspaced short palindromic repeats; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=W52-7; " /><meta name="citation_reference" content="citation_title=Automatic clustering of orthologs and in-paralogs from pairwise species comparisons; J. Mol. Biol.;  citation_year=2001;  citation_volume=314;  citation_pages=1041-52; " /><meta name="citation_reference" content="citation_title=The COG database: an updated version includes eukaryotes; BMC Bioinformatics;  citation_year=2003;  citation_volume=4;  citation_pages=41" /><meta name="citation_reference" content="citation_title=Genome trees and the tree of life; Trends Genet.;  citation_year=2002;  citation_volume=18;  citation_pages=472-9; " /><meta name="citation_reference" content="citation_title=Pathogenicity islands and the evolution of microbes; Annu. Rev. Microbiol.;  citation_year=2000;  citation_volume=54;  citation_pages=641-79; " /><meta name="citation_reference" content="citation_title=Prokaryotic toxin–antitoxin stress response loci; Nat. Rev. Microbiol.;  citation_year=2005;  citation_volume=3;  citation_pages=371-82; " /><meta name="citation_reference" content="citation_title=CRISPR provides acquired resistance against viruses in prokaryotes; Science;  citation_year=2007;  citation_volume=315;  citation_pages=1709-12; " /><meta name="citation_reference" content="citation_title=Enzymology of the Wood–Ljungdahl pathway of acetogenesis; Ann. N. Y. Acad. Sci.;  citation_year=2008;  citation_volume=1125;  citation_pages=129-36; " /><meta name="citation_reference" content="citation_title=Characterization and transcription of the genes encoding enzymes involved in butyrate production in Butyrivibrio fibrisolvens; Curr. Microbiol.;  citation_year=2003;  citation_volume=47;  citation_pages=203-7; " /><meta name="citation_reference" content="citation_title=Evidence of reversed electron transport in syntrophic butyrate or benzoate oxidation by Syntrophomonas wolfei and Syntrophus buswellii; Arch. Microbiol.;  citation_year=1994;  citation_volume=162;  citation_pages=136-42; " /><meta name="citation_reference" content="citation_title=Genes required for rapid expression of nitrogenase activity in Azotobacter vinelandii; Proc. Natl. Acad. Sci. USA;  citation_year=2005;  citation_volume=102;  citation_pages=6291-6; " /><meta name="citation_reference" content="citation_title=Electron transport in the pathway of acetate conversion to methane in the marine archaeon Methanosarcina acetivorans; J. Bacteriol.;  citation_year=2006;  citation_volume=188;  citation_pages=702-10; " /><meta name="citation_reference" content="citation_title=Discovery of a ferredoxin:NAD + -oxidoreductase (Rnf) in Acetobacterium woodii: a novel potential coupling site in acetogens; Ann. N. Y. Acad. Sci.;  citation_year=2008;  citation_volume=1125;  citation_pages=137-46; " /><meta name="citation_reference" content="citation_title=Membrane localization, topology, and mutual stabilization of the rnfABC gene products in Rhodobacter capsulatus and implications for a new family of energy-coupling NADH oxidoreductases; Biochemistry;  citation_year=1997;  citation_volume=36;  citation_pages=5509-21; " /><meta name="citation_reference" content="citation_title=Energy conservation via electron-transferring flavoprotein in anaerobic bacteria; J. Bacteriol.;  citation_year=2008;  citation_volume=190;  citation_pages=784-91; " /><meta name="citation_reference" content="citation_title=Prokaryotic nitrate reduction: molecular properties and functional distinction among bacterial nitrate reductases; J. Bacteriol.;  citation_year=1999;  citation_volume=181;  citation_pages=6573-84; " /><meta name="citation_reference" content="citation_title=Evolution of the soluble nitrate reductase: defining the monomeric periplasmic nitrate reductase subgroup; Biochem. Soc. Trans.;  citation_year=2006;  citation_volume=34;  citation_pages=122-6; " /><meta name="citation_reference" content="citation_title=The function of Wolinella succinogenes psr genes in electron transport with polysulphide as the terminal electron acceptor; Eur. J. Biochem.;  citation_year=1995;  citation_volume=230;  citation_pages=601-6; " /><meta name="citation_reference" content="citation_title=The phs gene and hydrogen sulfide production by Salmonella typhimurium; J. Bacteriol.;  citation_year=1987;  citation_volume=169;  citation_pages=2391-7; " /><meta name="citation_reference" content="citation_title=A membrane-bound multienzyme, hydrogen-oxidizing, and sulfur-reducing complex from the hyperthermophilic bacterium Aquifex aeolicus; J. Biol. Chem.;  citation_year=2005;  citation_volume=280;  citation_pages=42004-15; " /><meta name="citation_reference" content="citation_title=Membrane-bound hydrogenase and sulfur reductase of the hyperthermophilic and acidophilic archaeon Acidianus ambivalens; Microbiology;  citation_year=2003;  citation_volume=149;  citation_pages=2357-71; " /><meta name="citation_reference" content="citation_title=Growth and phylogenetic properties of novel bacteria belonging to the epsilon subdivision of the Proteobacteria enriched from Alvinella pompejana and deep-sea hydrothermal vents; Appl. Environ. Microbiol.;  citation_year=2001;  citation_volume=67;  citation_pages=4566-72; " /><meta name="citation_reference" content="citation_title=The function of the periplasmic Sud protein in polysulfide respiration of Wolinella succinogenes; Eur. J. Biochem.;  citation_year=1998;  citation_volume=253;  citation_pages=263-9; " /><meta name="citation_reference" content="citation_title=Overexpression in Escherichia coli and functional analysis of a novel PPi-selective porin, oprO, from Pseudomonas aeruginosa; J. Bacteriol.;  citation_year=1992;  citation_volume=174;  citation_pages=471-6; " /><meta name="citation_reference" content="citation_title=Respiration of metal (hydr)oxides by Shewanella and Geobacter: a key role for multihaem c-type cytochromes; Mol. Microbiol.;  citation_year=2007;  citation_volume=65;  citation_pages=12-20; " /><meta name="citation_reference" content="citation_title=Variability in microbial communities in black smoker chimneys at the NW caldera vent field, Brothers volcano, Kermadec arc; Geomicrobiol. J.;  citation_year=2009;  citation_volume=26;  citation_pages=552-569; " /><meta name="citation_reference" content="citation_title=Bacterial signal transduction network in a genomic perspective; Environ. Microbiol.;  citation_year=2004;  citation_volume=6;  citation_pages=552-67; " /><meta name="citation_reference" content="citation_title=PAS domain S-boxes in Archaea, bacteria and sensors for oxygen and redox; Trends Biochem. Sci.;  citation_year=1997;  citation_volume=22;  citation_pages=331-3; " /><meta name="citation_reference" content="citation_title=Unique regulation of carbohydrate chemotaxis in Bacillus subtilis by the phosphoenolpyruvate-dependent phosphotransferase system and the methyl-accepting chemotaxis protein McpC; J. Bacteriol.;  citation_year=1998;  citation_volume=180;  citation_pages=4475-80; " /><meta name="citation_reference" content="citation_title=A chemosensory system that regulates biofilm formation through modulation of cyclic diguanylate levels; Proc. Natl. Acad. Sci. USA;  citation_year=2005;  citation_volume=102;  citation_pages=14422-7; " /><meta name="citation_reference" content="citation_title=Chemosensory pathways, motility and development in Myxococcus xanthus; Nat. Rev. Microbiol.;  citation_year=2007;  citation_volume=5;  citation_pages=862-72; " /><meta name="citation_reference" content="citation_title=Involvement of a Che-like signal transduction cascade in regulating cyst cell development in Rhodospirillum centenum; Mol. Microbiol.;  citation_year=2005;  citation_volume=56;  citation_pages=1457-66; " /><meta name="citation_reference" content="citation_title=Merging genomes with geochemistry in hydrothermal ecosystems; Science;  citation_year=2002;  citation_volume=296;  citation_pages=1077-82; " /><meta name="citation_reference" content="citation_title=Thermophilic prokaryotes have characteristic patterns of codon usage, amino acid composition and nucleotide content; Gene;  citation_year=2003;  citation_volume=317;  citation_pages=39-47; " /><meta name="citation_reference" content="citation_title=Differences in amino acids composition and coupling patterns between mesophilic and thermophilic proteins; Amino Acids;  citation_year=2008;  citation_volume=34;  citation_pages=25-33; " /><meta name="citation_reference" content="citation_title=Thermoadaptation trait revealed by the genome sequence of thermophilic Geobacillus kaustophilus; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=6292-303; " /><meta name="citation_reference" content="citation_title=Cloning and characterization of tRNA (m1A58) methyltransferase (TrmI) from Thermus thermophilus HB27, a protein required for cell growth at extreme temperatures; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=2148-56; " /><meta name="citation_reference" content="citation_title=Biomolecular stability and life at high temperatures; Cell. Mol. Life Sci.;  citation_year=2000;  citation_volume=57;  citation_pages=250-64; " /><meta name="citation_reference" content="citation_title=Effects of polyamines on the thermal stability and formation kinetics of DNA duplexes with abnormal structure; Nucleic Acids Res.;  citation_year=2001;  citation_volume=29;  citation_pages=5121-8; " /><meta name="citation_reference" content="citation_title=Rubrerythrin from the hyperthermophilic archaeon Pyrococcus furiosus is a rubredoxin-dependent, iron-containing peroxidase; J. Bacteriol.;  citation_year=2004;  citation_volume=186;  citation_pages=7888-95; " /><meta name="citation_reference" content="citation_title=Rubredoxin:oxygen oxidoreductase enhances survival of Desulfovibrio vulgaris Hildenborough under microaerophilic conditions; J. Bacteriol.;  citation_year=2006;  citation_volume=188;  citation_pages=6253-60; " /><meta name="citation_reference" content="citation_title=Light regulation of type IV pilus-dependent motility by chemosensor-like elements in Synechocystis PCC6803; Proc. Natl. Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=7540-5; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/17/3/123/1010345/dsq005.pdf" /><meta name="description" content="Abstract.  The complete genome sequence of the thermophilic sulphur-reducing bacterium, Deferribacter desulfuricans SMM1, isolated from a hydrothermal vent chim" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/17/3/123/353305" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/17/3/123/353305" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/17/3/123/353305");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsq005';
    </script>
<title>
	Bacterial Lifestyle in a Deep-sea Hydrothermal Vent Chimney Revealed by the Genome Sequence of the Thermophilic Bacterium Deferribacter desulfuricans SSM1 | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./353305" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZenATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFrYWtpLCBZb3NoaWhpcm8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNaWNyb2JpYWwgR2Vub21lIFJlc2VhcmNoIEdyb3VwLCBFeHRyZW1vYmlvc3BoZXJlIFJlc2VhcmNoIFByb2dyYW0sIEluc3RpdHV0ZSBvZiBCaW9nZW9zY2llbmNlcywgSmFwYW4gQWdlbmN5IGZvciBNYXJpbmUtRWFydGggU2NpZW5jZSBhbmQgVGVjaG5vbG9neSAoSkFNU1RFQyksIFlva29zdWthLCBLYW5hZ2F3YSAyMzctMDA2MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGltYW11cmEsIFNoaWdlcnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTdWJzdXJmYWNlIEdlb2Jpb2xvZ3kgQWR2YW5jZWQgUmVzZWFyY2ggVGVhbSAoU1VHQVIpIFRlYW0sIEV4dHJlbW9iaW9zcGhlcmUgUmVzZWFyY2ggUHJvZ3JhbSwgSW5zdGl0dXRlIG9mIEJpb2dlb3NjaWVuY2VzLCBKQU1TVEVDLCBZb2tvc3VrYSwgS2FuYWdhd2EgMjM3LTAwNjEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFrYWdhd2EsIFNhdG9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTdWJzdXJmYWNlIEdlb2Jpb2xvZ3kgQWR2YW5jZWQgUmVzZWFyY2ggVGVhbSAoU1VHQVIpIFRlYW0sIEV4dHJlbW9iaW9zcGhlcmUgUmVzZWFyY2ggUHJvZ3JhbSwgSW5zdGl0dXRlIG9mIEJpb2dlb3NjaWVuY2VzLCBKQU1TVEVDLCBZb2tvc3VrYSwgS2FuYWdhd2EgMjM3LTAwNjEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yeSBvZiBNaWNyb2Jpb2xvZ3ksIEZhY3VsdHkgb2YgRmlzaGVyaWVzLCBIb2trYWlkbyBVbml2ZXJzaXR5LCAzLTEtMSBNaW5hdG8tY2hvLCBIYWtvZGF0ZSAwNDEtODYxMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGdWt1aGFyYSwgWWFzdW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOSVRFIEdlbm9tZSBBbmFseXNpcyBDZW50ZXIsIERlcGFydG1lbnQgb2YgQmlvdGVjaG5vbG9neSwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIFRlY2hub2xvZ3kgYW5kIEV2b2x1dGlvbiAoTklURSksIDItMTAtNDkgTmlzaGloYXJhLCBTaGlidXlhLWt1LCBUb2t5byAxNTEtMDA2NiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIb3Jpa2F3YSwgSGlyb3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5JVEUgR2Vub21lIEFuYWx5c2lzIENlbnRlciwgRGVwYXJ0bWVudCBvZiBCaW90ZWNobm9sb2d5LCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgVGVjaG5vbG9neSBhbmQgRXZvbHV0aW9uIChOSVRFKSwgMi0xMC00OSBOaXNoaWhhcmEsIFNoaWJ1eWEta3UsIFRva3lvIDE1MS0wMDY2LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkFua2FpLCBBa2lobyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5JVEUgR2Vub21lIEFuYWx5c2lzIENlbnRlciwgRGVwYXJ0bWVudCBvZiBCaW90ZWNobm9sb2d5LCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgVGVjaG5vbG9neSBhbmQgRXZvbHV0aW9uIChOSVRFKSwgMi0xMC00OSBOaXNoaWhhcmEsIFNoaWJ1eWEta3UsIFRva3lvIDE1MS0wMDY2LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhhcmFkYSwgVGFrZXNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5JVEUgR2Vub21lIEFuYWx5c2lzIENlbnRlciwgRGVwYXJ0bWVudCBvZiBCaW90ZWNobm9sb2d5LCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgVGVjaG5vbG9neSBhbmQgRXZvbHV0aW9uIChOSVRFKSwgMi0xMC00OSBOaXNoaWhhcmEsIFNoaWJ1eWEta3UsIFRva3lvIDE1MS0wMDY2LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikhvc295YW1hLCBBa2lyYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5JVEUgR2Vub21lIEFuYWx5c2lzIENlbnRlciwgRGVwYXJ0bWVudCBvZiBCaW90ZWNobm9sb2d5LCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgVGVjaG5vbG9neSBhbmQgRXZvbHV0aW9uIChOSVRFKSwgMi0xMC00OSBOaXNoaWhhcmEsIFNoaWJ1eWEta3UsIFRva3lvIDE1MS0wMDY2LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9ndWNoaSwgQWtpbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5JVEUgR2Vub21lIEFuYWx5c2lzIENlbnRlciwgRGVwYXJ0bWVudCBvZiBCaW90ZWNobm9sb2d5LCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgVGVjaG5vbG9neSBhbmQgRXZvbHV0aW9uIChOSVRFKSwgMi0xMC00OSBOaXNoaWhhcmEsIFNoaWJ1eWEta3UsIFRva3lvIDE1MS0wMDY2LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZ1a3VpLCBTaGlnZWhpcm8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOSVRFIEdlbm9tZSBBbmFseXNpcyBDZW50ZXIsIERlcGFydG1lbnQgb2YgQmlvdGVjaG5vbG9neSwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIFRlY2hub2xvZ3kgYW5kIEV2b2x1dGlvbiAoTklURSksIDItMTAtNDkgTmlzaGloYXJhLCBTaGlidXlhLWt1LCBUb2t5byAxNTEtMDA2NiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGdWppdGEsIE5vYnV5dWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTklURSBHZW5vbWUgQW5hbHlzaXMgQ2VudGVyLCBEZXBhcnRtZW50IG9mIEJpb3RlY2hub2xvZ3ksIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBUZWNobm9sb2d5IGFuZCBFdm9sdXRpb24gKE5JVEUpLCAyLTEwLTQ5IE5pc2hpaGFyYSwgU2hpYnV5YS1rdSwgVG9reW8gMTUxLTAwNjYsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFrYW1pLCBIaWRldG8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNaWNyb2JpYWwgR2Vub21lIFJlc2VhcmNoIEdyb3VwLCBFeHRyZW1vYmlvc3BoZXJlIFJlc2VhcmNoIFByb2dyYW0sIEluc3RpdHV0ZSBvZiBCaW9nZW9zY2llbmNlcywgSmFwYW4gQWdlbmN5IGZvciBNYXJpbmUtRWFydGggU2NpZW5jZSBhbmQgVGVjaG5vbG9neSAoSkFNU1RFQyksIFlva29zdWthLCBLYW5hZ2F3YSAyMzctMDA2MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYWthaSwgS2VuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU3Vic3VyZmFjZSBHZW9iaW9sb2d5IEFkdmFuY2VkIFJlc2VhcmNoIFRlYW0gKFNVR0FSKSBUZWFtLCBFeHRyZW1vYmlvc3BoZXJlIFJlc2VhcmNoIFByb2dyYW0sIEluc3RpdHV0ZSBvZiBCaW9nZW9zY2llbmNlcywgSkFNU1RFQywgWW9rb3N1a2EsIEthbmFnYXdhIDIzNy0wMDYxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQmFjdGVyaWFsIExpZmVzdHlsZSBpbiBhIERlZXAtc2VhIEh5ZHJvdGhlcm1hbCBWZW50IENoaW1uZXkgUmV2ZWFsZWQgYnkgdGhlIEdlbm9tZSBTZXF1ZW5jZSBvZiB0aGUgVGhlcm1vcGhpbGljIEJhY3Rlcml1bSBEZWZlcnJpYmFjdGVyIGRlc3VsZnVyaWNhbnMgU1NNMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjEyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTM3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNxMDA1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTAvMDYvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Q29wZXJuaWN1cyBCb29rcywgTmV3IFlvcms7IFRoZSBEZWVwIEhvdCBCaW9zcGhlcmUuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fcGFnZXM9MTktMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlZmVycmliYWN0ZXIgZGVzdWxmdXJpY2FucyBzcC4gbm92LiwgYSBub3ZlbCBzdWxmdXItLCBuaXRyYXRlLSBhbmQgYXJzZW5hdGUtcmVkdWNpbmcgdGhlcm1vcGhpbGUgaXNvbGF0ZWQgZnJvbSBhIGRlZXAtc2VhIGh5ZHJvdGhlcm1hbCB2ZW50OyBJbnQuIEouIFN5c3QuIEV2b2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NTM7ICBjaXRhdGlvbl9wYWdlcz04MzktNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlZmVycmliYWN0ZXIgYWJ5c3NpIHNwLiBub3YuLCBhbiBhbmFlcm9iaWMgdGhlcm1vcGhpbGUgZnJvbSBkZWVwLXNlYSBoeWRyb3RoZXJtYWwgdmVudHMgb2YgdGhlIE1pZC1BdGxhbnRpYyBSaWRnZTsgSW50LiBKLiBTeXN0LiBFdm9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTUzOyAgY2l0YXRpb25fcGFnZXM9MTYzNy00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGVmZXJyaWJhY3RlciB0aGVybW9waGlsdXMgZ2VuLiBub3YuLCBzcC4gbm92LiwgYSBub3ZlbCB0aGVybW9waGlsaWMgbWFuZ2FuZXNlLSBhbmQgaXJvbi1yZWR1Y2luZyBiYWN0ZXJpdW0gaXNvbGF0ZWQgZnJvbSBhIHBldHJvbGV1bSByZXNlcnZvaXI7IEludC4gSi4gU3lzdC4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT00NzsgIGNpdGF0aW9uX3BhZ2VzPTUwNS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZWZlcnJpYmFjdGVyIGF1dG90cm9waGljdXMgc3AuIG5vdi4sIGFuIGlyb24oSUlJKS1yZWR1Y2luZyBiYWN0ZXJpdW0gZnJvbSBhIGRlZXAtc2VhIGh5ZHJvdGhlcm1hbCB2ZW50OyBJbnQuIEouIFN5c3QuIEV2b2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NTk7ICBjaXRhdGlvbl9wYWdlcz0xNTA4LTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXN0cmlidXRpb24sIHBoeWxvZ2VuZXRpYyBkaXZlcnNpdHkgYW5kIHBoeXNpb2xvZ2ljYWwgY2hhcmFjdGVyaXN0aWNzIG9mIGVwc2lsb24tUHJvdGVvYmFjdGVyaWEgaW4gYSBkZWVwLXNlYSBoeWRyb3RoZXJtYWwgZmllbGQ7IEVudmlyb24uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTE2MTktMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2aWRlbmNlIGZvciB0aGUgcHJlc2VuY2Ugb2YgdGhlcm1vcGhpbGljIEZlKElJSSktcmVkdWNpbmcgbWljcm9vcmdhbmlzbXMgaW4gZGVlcC1zZWEgaHlkcm90aGVybWFsIHZlbnRzIGF0IDEzIGRlZ3JlZXMgTiAoRWFzdCBQYWNpZmljIFJpc2UpOyBGRU1TIE1pY3JvYmlvbC4gRWNvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz0yMzUtNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZhcmlhYmlsaXR5IGluIHRoZSBtaWNyb2JpYWwgY29tbXVuaXRpZXMgYW5kIGh5ZHJvdGhlcm1hbCBmbHVpZCBjaGVtaXN0cnkgYXQgdGhlIG5ld2x5IGRpc2NvdmVyZWQgTWFyaW5lciBoeWRyb3RoZXJtYWwgZmllbGQsIHNvdXRoZXJuIExhdSBCYXNpbjsgSi4gR2VvcGh5cy4gUmVzLi1CaW9nZW9zY2kuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTExMzsgIGNpdGF0aW9uX3BhZ2VzPUcwMjAzMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgb2YgZGVlcC1zZWEgdmVudCBjaGVtb2xpdGhvYXV0b3Ryb3BoIFRoaW9taWNyb3NwaXJhIGNydW5vZ2VuYSBYQ0wtMjsgUExvUyBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9ZTM4MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFkYXB0YXRpb25zIHRvIHN1Ym1hcmluZSBoeWRyb3RoZXJtYWwgZW52aXJvbm1lbnRzIGV4ZW1wbGlmaWVkIGJ5IHRoZSBnZW5vbWUgb2YgTmF1dGlsaWEgcHJvZnVuZGljb2xhOyBQTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDAzNjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZWVwLXNlYSB2ZW50IM61LXByb3Rlb2JhY3RlcmlhbCBnZW5vbWVzIHByb3ZpZGUgaW5zaWdodHMgaW50byBlbWVyZ2VuY2Ugb2YgcGF0aG9nZW5zOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDQ7ICBjaXRhdGlvbl9wYWdlcz0xMjE0Ni01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxldGUgYW5kIGRyYWZ0IGdlbm9tZSBzZXF1ZW5jZXMgb2Ygc2l4IG1lbWJlcnMgb2YgdGhlIEFxdWlmaWNhbGVzOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE5MTsgIGNpdGF0aW9uX3BhZ2VzPTE5OTItMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxldGUgZ2Vub21lIHNlcXVlbmNlIG9mIHRoZSBzb2lsIGFjdGlub215Y2V0ZSBLb2N1cmlhIHJoaXpvcGhpbGE7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTkwOyAgY2l0YXRpb25fcGFnZXM9NDEzOS00NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFzZS1jYWxsaW5nIG9mIGF1dG9tYXRlZCBzZXF1ZW5jZXIgdHJhY2VzIHVzaW5nIHBocmVkLiBJLiBBY2N1cmFjeSBhc3Nlc3NtZW50OyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MTc1LTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZmFtOiBhbiBSTkEgZmFtaWx5IGRhdGFiYXNlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz00MzktNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPXRSTkFzY2FuLVNFOiBhIHByb2dyYW0gZm9yIGltcHJvdmVkIGRldGVjdGlvbiBvZiB0cmFuc2ZlciBSTkEgZ2VuZXMgaW4gZ2Vub21pYyBzZXF1ZW5jZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9OTU1LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HYXBwZWQgQkxBU1QgYW5kIFBTSS1CTEFTVDogYSBuZXcgZ2VuZXJhdGlvbiBvZiBwcm90ZWluIGRhdGFiYXNlIHNlYXJjaCBwcm9ncmFtczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzM4OS00MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcHJvdmVkIG1pY3JvYmlhbCBnZW5lIGlkZW50aWZpY2F0aW9uIHdpdGggR0xJTU1FUjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9NDYzNi00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZU1hcmtTOiBhIHNlbGYtdHJhaW5pbmcgbWV0aG9kIGZvciBwcmVkaWN0aW9uIG9mIGdlbmUgc3RhcnRzIGluIG1pY3JvYmlhbCBnZW5vbWVzLiBJbXBsaWNhdGlvbnMgZm9yIGZpbmRpbmcgc2VxdWVuY2UgbW90aWZzIGluIHJlZ3VsYXRvcnkgcmVnaW9uczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9MjYwNy0xODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuQmFuazsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9RDM0LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBVbml2ZXJzYWwgUHJvdGVpbiBSZXNvdXJjZSAoVW5pUHJvdCk7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPUQxNTQtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEtFR0cgcmVzb3VyY2UgZm9yIGRlY2lwaGVyaW5nIHRoZSBnZW5vbWU7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPUQyNzctODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBQZmFtIHByb3RlaW4gZmFtaWxpZXMgZGF0YWJhc2U7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPUQyODEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIE1ldGFDeWMgRGF0YWJhc2Ugb2YgbWV0YWJvbGljIHBhdGh3YXlzIGFuZCBlbnp5bWVzIGFuZCB0aGUgQmlvQ3ljIGNvbGxlY3Rpb24gb2YgUGF0aHdheS9HZW5vbWUgRGF0YWJhc2VzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz1ENjIzLTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbXByb3ZlZCBwcmVkaWN0aW9uIG9mIHNpZ25hbCBwZXB0aWRlczogU2lnbmFsUCAzLjA7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzQwOyAgY2l0YXRpb25fcGFnZXM9NzgzLTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TT1NVSTogY2xhc3NpZmljYXRpb24gYW5kIHNlY29uZGFyeSBzdHJ1Y3R1cmUgcHJlZGljdGlvbiBzeXN0ZW0gZm9yIG1lbWJyYW5lIHByb3RlaW5zOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTM3OC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DUklTUFJGaW5kZXI6IGEgd2ViIHRvb2wgdG8gaWRlbnRpZnkgY2x1c3RlcmVkIHJlZ3VsYXJseSBpbnRlcnNwYWNlZCBzaG9ydCBwYWxpbmRyb21pYyByZXBlYXRzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz1XNTItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXV0b21hdGljIGNsdXN0ZXJpbmcgb2Ygb3J0aG9sb2dzIGFuZCBpbi1wYXJhbG9ncyBmcm9tIHBhaXJ3aXNlIHNwZWNpZXMgY29tcGFyaXNvbnM7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MzE0OyAgY2l0YXRpb25fcGFnZXM9MTA0MS01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIENPRyBkYXRhYmFzZTogYW4gdXBkYXRlZCB2ZXJzaW9uIGluY2x1ZGVzIGV1a2FyeW90ZXM7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9NDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgdHJlZXMgYW5kIHRoZSB0cmVlIG9mIGxpZmU7IFRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz00NzItOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGF0aG9nZW5pY2l0eSBpc2xhbmRzIGFuZCB0aGUgZXZvbHV0aW9uIG9mIG1pY3JvYmVzOyBBbm51LiBSZXYuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz02NDEtNzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb2thcnlvdGljIHRveGlu4oCTYW50aXRveGluIHN0cmVzcyByZXNwb25zZSBsb2NpOyBOYXQuIFJldi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MzcxLTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DUklTUFIgcHJvdmlkZXMgYWNxdWlyZWQgcmVzaXN0YW5jZSBhZ2FpbnN0IHZpcnVzZXMgaW4gcHJva2FyeW90ZXM7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzE1OyAgY2l0YXRpb25fcGFnZXM9MTcwOS0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW56eW1vbG9neSBvZiB0aGUgV29vZOKAk0xqdW5nZGFobCBwYXRod2F5IG9mIGFjZXRvZ2VuZXNpczsgQW5uLiBOLiBZLiBBY2FkLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTExMjU7ICBjaXRhdGlvbl9wYWdlcz0xMjktMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gYW5kIHRyYW5zY3JpcHRpb24gb2YgdGhlIGdlbmVzIGVuY29kaW5nIGVuenltZXMgaW52b2x2ZWQgaW4gYnV0eXJhdGUgcHJvZHVjdGlvbiBpbiBCdXR5cml2aWJyaW8gZmlicmlzb2x2ZW5zOyBDdXJyLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQ3OyAgY2l0YXRpb25fcGFnZXM9MjAzLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2aWRlbmNlIG9mIHJldmVyc2VkIGVsZWN0cm9uIHRyYW5zcG9ydCBpbiBzeW50cm9waGljIGJ1dHlyYXRlIG9yIGJlbnpvYXRlIG94aWRhdGlvbiBieSBTeW50cm9waG9tb25hcyB3b2xmZWkgYW5kIFN5bnRyb3BodXMgYnVzd2VsbGlpOyBBcmNoLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTE2MjsgIGNpdGF0aW9uX3BhZ2VzPTEzNi00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXMgcmVxdWlyZWQgZm9yIHJhcGlkIGV4cHJlc3Npb24gb2Ygbml0cm9nZW5hc2UgYWN0aXZpdHkgaW4gQXpvdG9iYWN0ZXIgdmluZWxhbmRpaTsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTAyOyAgY2l0YXRpb25fcGFnZXM9NjI5MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbGVjdHJvbiB0cmFuc3BvcnQgaW4gdGhlIHBhdGh3YXkgb2YgYWNldGF0ZSBjb252ZXJzaW9uIHRvIG1ldGhhbmUgaW4gdGhlIG1hcmluZSBhcmNoYWVvbiBNZXRoYW5vc2FyY2luYSBhY2V0aXZvcmFuczsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xODg7ICBjaXRhdGlvbl9wYWdlcz03MDItMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc2NvdmVyeSBvZiBhIGZlcnJlZG94aW46TkFEICsgLW94aWRvcmVkdWN0YXNlIChSbmYpIGluIEFjZXRvYmFjdGVyaXVtIHdvb2RpaTogYSBub3ZlbCBwb3RlbnRpYWwgY291cGxpbmcgc2l0ZSBpbiBhY2V0b2dlbnM7IEFubi4gTi4gWS4gQWNhZC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMTI1OyAgY2l0YXRpb25fcGFnZXM9MTM3LTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZW1icmFuZSBsb2NhbGl6YXRpb24sIHRvcG9sb2d5LCBhbmQgbXV0dWFsIHN0YWJpbGl6YXRpb24gb2YgdGhlIHJuZkFCQyBnZW5lIHByb2R1Y3RzIGluIFJob2RvYmFjdGVyIGNhcHN1bGF0dXMgYW5kIGltcGxpY2F0aW9ucyBmb3IgYSBuZXcgZmFtaWx5IG9mIGVuZXJneS1jb3VwbGluZyBOQURIIG94aWRvcmVkdWN0YXNlczsgQmlvY2hlbWlzdHJ5OyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9NTUwOS0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW5lcmd5IGNvbnNlcnZhdGlvbiB2aWEgZWxlY3Ryb24tdHJhbnNmZXJyaW5nIGZsYXZvcHJvdGVpbiBpbiBhbmFlcm9iaWMgYmFjdGVyaWE7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTkwOyAgY2l0YXRpb25fcGFnZXM9Nzg0LTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm9rYXJ5b3RpYyBuaXRyYXRlIHJlZHVjdGlvbjogbW9sZWN1bGFyIHByb3BlcnRpZXMgYW5kIGZ1bmN0aW9uYWwgZGlzdGluY3Rpb24gYW1vbmcgYmFjdGVyaWFsIG5pdHJhdGUgcmVkdWN0YXNlczsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0xODE7ICBjaXRhdGlvbl9wYWdlcz02NTczLTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb24gb2YgdGhlIHNvbHVibGUgbml0cmF0ZSByZWR1Y3Rhc2U6IGRlZmluaW5nIHRoZSBtb25vbWVyaWMgcGVyaXBsYXNtaWMgbml0cmF0ZSByZWR1Y3Rhc2Ugc3ViZ3JvdXA7IEJpb2NoZW0uIFNvYy4gVHJhbnMuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9MTIyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBmdW5jdGlvbiBvZiBXb2xpbmVsbGEgc3VjY2lub2dlbmVzIHBzciBnZW5lcyBpbiBlbGVjdHJvbiB0cmFuc3BvcnQgd2l0aCBwb2x5c3VscGhpZGUgYXMgdGhlIHRlcm1pbmFsIGVsZWN0cm9uIGFjY2VwdG9yOyBFdXIuIEouIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTIzMDsgIGNpdGF0aW9uX3BhZ2VzPTYwMS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcGhzIGdlbmUgYW5kIGh5ZHJvZ2VuIHN1bGZpZGUgcHJvZHVjdGlvbiBieSBTYWxtb25lbGxhIHR5cGhpbXVyaXVtOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTE2OTsgIGNpdGF0aW9uX3BhZ2VzPTIzOTEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBtZW1icmFuZS1ib3VuZCBtdWx0aWVuenltZSwgaHlkcm9nZW4tb3hpZGl6aW5nLCBhbmQgc3VsZnVyLXJlZHVjaW5nIGNvbXBsZXggZnJvbSB0aGUgaHlwZXJ0aGVybW9waGlsaWMgYmFjdGVyaXVtIEFxdWlmZXggYWVvbGljdXM7IEouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTI4MDsgIGNpdGF0aW9uX3BhZ2VzPTQyMDA0LTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZW1icmFuZS1ib3VuZCBoeWRyb2dlbmFzZSBhbmQgc3VsZnVyIHJlZHVjdGFzZSBvZiB0aGUgaHlwZXJ0aGVybW9waGlsaWMgYW5kIGFjaWRvcGhpbGljIGFyY2hhZW9uIEFjaWRpYW51cyBhbWJpdmFsZW5zOyBNaWNyb2Jpb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTQ5OyAgY2l0YXRpb25fcGFnZXM9MjM1Ny03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R3Jvd3RoIGFuZCBwaHlsb2dlbmV0aWMgcHJvcGVydGllcyBvZiBub3ZlbCBiYWN0ZXJpYSBiZWxvbmdpbmcgdG8gdGhlIGVwc2lsb24gc3ViZGl2aXNpb24gb2YgdGhlIFByb3Rlb2JhY3RlcmlhIGVucmljaGVkIGZyb20gQWx2aW5lbGxhIHBvbXBlamFuYSBhbmQgZGVlcC1zZWEgaHlkcm90aGVybWFsIHZlbnRzOyBBcHBsLiBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTY3OyAgY2l0YXRpb25fcGFnZXM9NDU2Ni03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGZ1bmN0aW9uIG9mIHRoZSBwZXJpcGxhc21pYyBTdWQgcHJvdGVpbiBpbiBwb2x5c3VsZmlkZSByZXNwaXJhdGlvbiBvZiBXb2xpbmVsbGEgc3VjY2lub2dlbmVzOyBFdXIuIEouIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTI1MzsgIGNpdGF0aW9uX3BhZ2VzPTI2My05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PdmVyZXhwcmVzc2lvbiBpbiBFc2NoZXJpY2hpYSBjb2xpIGFuZCBmdW5jdGlvbmFsIGFuYWx5c2lzIG9mIGEgbm92ZWwgUFBpLXNlbGVjdGl2ZSBwb3Jpbiwgb3ByTywgZnJvbSBQc2V1ZG9tb25hcyBhZXJ1Z2lub3NhOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTE3NDsgIGNpdGF0aW9uX3BhZ2VzPTQ3MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXNwaXJhdGlvbiBvZiBtZXRhbCAoaHlkcilveGlkZXMgYnkgU2hld2FuZWxsYSBhbmQgR2VvYmFjdGVyOiBhIGtleSByb2xlIGZvciBtdWx0aWhhZW0gYy10eXBlIGN5dG9jaHJvbWVzOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NjU7ICBjaXRhdGlvbl9wYWdlcz0xMi0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmFyaWFiaWxpdHkgaW4gbWljcm9iaWFsIGNvbW11bml0aWVzIGluIGJsYWNrIHNtb2tlciBjaGltbmV5cyBhdCB0aGUgTlcgY2FsZGVyYSB2ZW50IGZpZWxkLCBCcm90aGVycyB2b2xjYW5vLCBLZXJtYWRlYyBhcmM7IEdlb21pY3JvYmlvbC4gSi47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz01NTItNTY5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYWN0ZXJpYWwgc2lnbmFsIHRyYW5zZHVjdGlvbiBuZXR3b3JrIGluIGEgZ2Vub21pYyBwZXJzcGVjdGl2ZTsgRW52aXJvbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9NTUyLTY3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QQVMgZG9tYWluIFMtYm94ZXMgaW4gQXJjaGFlYSwgYmFjdGVyaWEgYW5kIHNlbnNvcnMgZm9yIG94eWdlbiBhbmQgcmVkb3g7IFRyZW5kcyBCaW9jaGVtLiBTY2kuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MzMxLTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVuaXF1ZSByZWd1bGF0aW9uIG9mIGNhcmJvaHlkcmF0ZSBjaGVtb3RheGlzIGluIEJhY2lsbHVzIHN1YnRpbGlzIGJ5IHRoZSBwaG9zcGhvZW5vbHB5cnV2YXRlLWRlcGVuZGVudCBwaG9zcGhvdHJhbnNmZXJhc2Ugc3lzdGVtIGFuZCB0aGUgbWV0aHlsLWFjY2VwdGluZyBjaGVtb3RheGlzIHByb3RlaW4gTWNwQzsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xODA7ICBjaXRhdGlvbl9wYWdlcz00NDc1LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNoZW1vc2Vuc29yeSBzeXN0ZW0gdGhhdCByZWd1bGF0ZXMgYmlvZmlsbSBmb3JtYXRpb24gdGhyb3VnaCBtb2R1bGF0aW9uIG9mIGN5Y2xpYyBkaWd1YW55bGF0ZSBsZXZlbHM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEwMjsgIGNpdGF0aW9uX3BhZ2VzPTE0NDIyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoZW1vc2Vuc29yeSBwYXRod2F5cywgbW90aWxpdHkgYW5kIGRldmVsb3BtZW50IGluIE15eG9jb2NjdXMgeGFudGh1czsgTmF0LiBSZXYuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTg2Mi03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW52b2x2ZW1lbnQgb2YgYSBDaGUtbGlrZSBzaWduYWwgdHJhbnNkdWN0aW9uIGNhc2NhZGUgaW4gcmVndWxhdGluZyBjeXN0IGNlbGwgZGV2ZWxvcG1lbnQgaW4gUmhvZG9zcGlyaWxsdW0gY2VudGVudW07IE1vbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT01NjsgIGNpdGF0aW9uX3BhZ2VzPTE0NTctNjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lcmdpbmcgZ2Vub21lcyB3aXRoIGdlb2NoZW1pc3RyeSBpbiBoeWRyb3RoZXJtYWwgZWNvc3lzdGVtczsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOTY7ICBjaXRhdGlvbl9wYWdlcz0xMDc3LTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGVybW9waGlsaWMgcHJva2FyeW90ZXMgaGF2ZSBjaGFyYWN0ZXJpc3RpYyBwYXR0ZXJucyBvZiBjb2RvbiB1c2FnZSwgYW1pbm8gYWNpZCBjb21wb3NpdGlvbiBhbmQgbnVjbGVvdGlkZSBjb250ZW50OyBHZW5lOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxNzsgIGNpdGF0aW9uX3BhZ2VzPTM5LTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWZmZXJlbmNlcyBpbiBhbWlubyBhY2lkcyBjb21wb3NpdGlvbiBhbmQgY291cGxpbmcgcGF0dGVybnMgYmV0d2VlbiBtZXNvcGhpbGljIGFuZCB0aGVybW9waGlsaWMgcHJvdGVpbnM7IEFtaW5vIEFjaWRzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9MjUtMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZXJtb2FkYXB0YXRpb24gdHJhaXQgcmV2ZWFsZWQgYnkgdGhlIGdlbm9tZSBzZXF1ZW5jZSBvZiB0aGVybW9waGlsaWMgR2VvYmFjaWxsdXMga2F1c3RvcGhpbHVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz02MjkyLTMwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2xvbmluZyBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiB0Uk5BIChtMUE1OCkgbWV0aHlsdHJhbnNmZXJhc2UgKFRybUkpIGZyb20gVGhlcm11cyB0aGVybW9waGlsdXMgSEIyNywgYSBwcm90ZWluIHJlcXVpcmVkIGZvciBjZWxsIGdyb3d0aCBhdCBleHRyZW1lIHRlbXBlcmF0dXJlczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MjE0OC01NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmlvbW9sZWN1bGFyIHN0YWJpbGl0eSBhbmQgbGlmZSBhdCBoaWdoIHRlbXBlcmF0dXJlczsgQ2VsbC4gTW9sLiBMaWZlIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NTc7ICBjaXRhdGlvbl9wYWdlcz0yNTAtNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmVjdHMgb2YgcG9seWFtaW5lcyBvbiB0aGUgdGhlcm1hbCBzdGFiaWxpdHkgYW5kIGZvcm1hdGlvbiBraW5ldGljcyBvZiBETkEgZHVwbGV4ZXMgd2l0aCBhYm5vcm1hbCBzdHJ1Y3R1cmU7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTUxMjEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UnVicmVyeXRocmluIGZyb20gdGhlIGh5cGVydGhlcm1vcGhpbGljIGFyY2hhZW9uIFB5cm9jb2NjdXMgZnVyaW9zdXMgaXMgYSBydWJyZWRveGluLWRlcGVuZGVudCwgaXJvbi1jb250YWluaW5nIHBlcm94aWRhc2U7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTg2OyAgY2l0YXRpb25fcGFnZXM9Nzg4OC05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UnVicmVkb3hpbjpveHlnZW4gb3hpZG9yZWR1Y3Rhc2UgZW5oYW5jZXMgc3Vydml2YWwgb2YgRGVzdWxmb3ZpYnJpbyB2dWxnYXJpcyBIaWxkZW5ib3JvdWdoIHVuZGVyIG1pY3JvYWVyb3BoaWxpYyBjb25kaXRpb25zOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE4ODsgIGNpdGF0aW9uX3BhZ2VzPTYyNTMtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxpZ2h0IHJlZ3VsYXRpb24gb2YgdHlwZSBJViBwaWx1cy1kZXBlbmRlbnQgbW90aWxpdHkgYnkgY2hlbW9zZW5zb3ItbGlrZSBlbGVtZW50cyBpbiBTeW5lY2hvY3lzdGlzIFBDQzY4MDM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTk4OyAgY2l0YXRpb25fcGFnZXM9NzU0MC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTcvMy8xMjMvMTAxMDM0NS9kc3EwMDUucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFRoZSBjb21wbGV0ZSBnZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIHRoZXJtb3BoaWxpYyBzdWxwaHVyLXJlZHVjaW5nIGJhY3Rlcml1bSwgRGVmZXJyaWJhY3RlciBkZXN1bGZ1cmljYW5zIFNNTTEsIGlzb2xhdGVkIGZyb20gYSBoeWRyb3RoZXJtYWwgdmVudCBjaGltIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNTMzMDUmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2Rkp2waWVDRYmITAJLXh2/AsHXz404=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f3%2f123%2f353305" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f3%2f123%2f353305" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13687" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/17/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/3/1/m_cover.gif?Expires=2147483647&amp;Signature=zrlqOYlr8puhWaI06gf8sYMMP6iQcSaE6WnxD18CI1vM-vSYmV7zw1fL1UUm1LAo6OR8YaC2ow7Gg3gFyOuRxjrsW2e9seZ-Fc4W0zpKOKeLDqycZiol4B5gPhMqbsAY9k5dM2sXst3uhEwLyf9p6Clk-Cv9XLljwpdq37Ye9OThamAMiMSAoDxynPtt~5t-2ainSI295gFWPk3t9zFxKYx4zJXORsGXGnkmtct97jCcoDB0mHB0FSIfJFRj9SkO~ccPVDFJ5bkP0CQlUseB~SQw04UmN2w0van9EOcwauxu2Sl9Oe1AFiN92gYP2rZEbuX5qeGuIN6geNJvEhApPQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/17/3">
                        <div class="volume trailing-comma">Volume 17</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2010        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5027256">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5027256">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5027258">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5027258">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5027262">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5027262">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5027271">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5027271">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5027312">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5027312">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5027314">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5027314">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/17/3/139/354144">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Bacterial Lifestyle in a Deep-sea Hydrothermal Vent Chimney Revealed by the Genome Sequence of the Thermophilic Bacterium <em>Deferribacter desulfuricans</em> SSM1
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshihiro Takaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Microbial Genome Research Group, Extremobiosphere Research Program, Institute of Biogeosciences, Japan Agency for Marine-Earth Science and Technology (JAMSTEC)</div>, <div class="addr-line">Yokosuka, Kanagawa 237-0061</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 46-867-9711. Fax. <div class="fax">+81 46-867-9645.</div> Email: <a href="mailto:takakiy@jamstec.go.jp" target="_blank">takakiy@jamstec.go.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshihiro+Takaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takaki Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takaki Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yoshihiro Takaki</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shigeru Shimamura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Subsurface Geobiology Advanced Research Team (SUGAR) Team, Extremobiosphere Research Program, Institute of Biogeosciences, JAMSTEC</div>, <div class="addr-line">Yokosuka, Kanagawa 237-0061</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shigeru+Shimamura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shimamura S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shimamura S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shigeru Shimamura</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Nakagawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Subsurface Geobiology Advanced Research Team (SUGAR) Team, Extremobiosphere Research Program, Institute of Biogeosciences, JAMSTEC</div>, <div class="addr-line">Yokosuka, Kanagawa 237-0061</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Laboratory of Microbiology, Faculty of Fisheries</div>, <div class="insititution">Hokkaido University</div>, <div class="addr-line">3-1-1 Minato-cho, Hakodate 041-8611</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Nakagawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakagawa S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakagawa S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Satoshi Nakagawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasuo Fukuhara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">NITE Genome Analysis Center, Department of Biotechnology</div>, <div class="insititution">National Institute of Technology and Evolution (NITE)</div>, <div class="addr-line">2-10-49 Nishihara, Shibuya-ku, Tokyo 151-0066</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasuo+Fukuhara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fukuhara Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fukuhara Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yasuo Fukuhara</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroshi Horikawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">NITE Genome Analysis Center, Department of Biotechnology</div>, <div class="insititution">National Institute of Technology and Evolution (NITE)</div>, <div class="addr-line">2-10-49 Nishihara, Shibuya-ku, Tokyo 151-0066</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroshi+Horikawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Horikawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Horikawa H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hiroshi Horikawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akiho Ankai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">NITE Genome Analysis Center, Department of Biotechnology</div>, <div class="insititution">National Institute of Technology and Evolution (NITE)</div>, <div class="addr-line">2-10-49 Nishihara, Shibuya-ku, Tokyo 151-0066</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akiho+Ankai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ankai A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ankai A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Akiho Ankai</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takeshi Harada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">NITE Genome Analysis Center, Department of Biotechnology</div>, <div class="insititution">National Institute of Technology and Evolution (NITE)</div>, <div class="addr-line">2-10-49 Nishihara, Shibuya-ku, Tokyo 151-0066</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takeshi+Harada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Harada T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Harada T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Takeshi Harada</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akira Hosoyama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">NITE Genome Analysis Center, Department of Biotechnology</div>, <div class="insititution">National Institute of Technology and Evolution (NITE)</div>, <div class="addr-line">2-10-49 Nishihara, Shibuya-ku, Tokyo 151-0066</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akira+Hosoyama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hosoyama A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hosoyama A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Akira Hosoyama</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akio Oguchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">NITE Genome Analysis Center, Department of Biotechnology</div>, <div class="insititution">National Institute of Technology and Evolution (NITE)</div>, <div class="addr-line">2-10-49 Nishihara, Shibuya-ku, Tokyo 151-0066</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akio+Oguchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Oguchi A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Oguchi A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Akio Oguchi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shigehiro Fukui</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">NITE Genome Analysis Center, Department of Biotechnology</div>, <div class="insititution">National Institute of Technology and Evolution (NITE)</div>, <div class="addr-line">2-10-49 Nishihara, Shibuya-ku, Tokyo 151-0066</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shigehiro+Fukui">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fukui S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fukui S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shigehiro Fukui</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nobuyuki Fujita</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">NITE Genome Analysis Center, Department of Biotechnology</div>, <div class="insititution">National Institute of Technology and Evolution (NITE)</div>, <div class="addr-line">2-10-49 Nishihara, Shibuya-ku, Tokyo 151-0066</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nobuyuki+Fujita">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fujita N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fujita N&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Nobuyuki Fujita</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideto Takami</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Microbial Genome Research Group, Extremobiosphere Research Program, Institute of Biogeosciences, Japan Agency for Marine-Earth Science and Technology (JAMSTEC)</div>, <div class="addr-line">Yokosuka, Kanagawa 237-0061</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideto+Takami">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takami H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takami H&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Hideto Takami</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ken Takai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Subsurface Geobiology Advanced Research Team (SUGAR) Team, Extremobiosphere Research Program, Institute of Biogeosciences, JAMSTEC</div>, <div class="addr-line">Yokosuka, Kanagawa 237-0061</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ken+Takai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takai K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takai K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Ken Takai</a>
                                        </span>
                                </div>
                            </div>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 17, Issue 3, 1 June 2010, Pages 123–137, <a href='https://doi.org/10.1093/dnares/dsq005'>https://doi.org/10.1093/dnares/dsq005</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">26 February 2010</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">26 October 2009</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">31 January 2010</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="353305" href="/dnaresearch/article-pdf/17/3/123/1010345/dsq005.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yoshihiro Takaki, Shigeru Shimamura, Satoshi Nakagawa, Yasuo Fukuhara, Hiroshi Horikawa, Akiho Ankai, Takeshi Harada, Akira Hosoyama, Akio Oguchi, Shigehiro Fukui, Nobuyuki Fujita, Hideto Takami, Ken Takai;  Bacterial Lifestyle in a Deep-sea Hydrothermal Vent Chimney Revealed by the Genome Sequence of the Thermophilic Bacterium <em>Deferribacter desulfuricans</em> SSM1, <em>DNA Research</em>, Volume 17, Issue 3, 1 June 2010, Pages 123–137, <a href='https://doi.org/10.1093/dnares/dsq005'>https://doi.org/10.1093/dnares/dsq005</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/353305?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/353305?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/353305?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/353305?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/353305?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/353305?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/353305?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Bacterial%20Lifestyle%20in%20a%20Deep-sea%20Hydrothermal%20Vent%20Chimney%20Revealed%20by%20the%20Genome%20Sequence%20of%20the%20Thermophilic%20Bacterium%20%26lt%3Bem%26gt%3BDeferribacter%20desulfuricans%26lt%3B%2Fem%26gt%3B%20SSM1&amp;publicationDate=2010-02-26&amp;volumeNum=17&amp;issueNum=3&amp;author=Takaki%2C%20Yoshihiro%3B%20Shimamura%2C%20Shigeru&amp;startPage=123&amp;endPage=137&amp;contentId=10.1093%2Fdnares%2Fdsq005&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f3%2f123%2f353305" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5027256 id="5027256" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The complete genome sequence of the thermophilic sulphur-reducing bacterium, <em>Deferribacter desulfuricans</em> SMM1, isolated from a hydrothermal vent chimney has been determined. The genome comprises a single circular chromosome of 2 234 389 bp and a megaplasmid of 308 544 bp. Many genes encoded in the genome are most similar to the genes of sulphur- or sulphate-reducing bacterial species within <em>Deltaproteobacteria</em>. The reconstructed central metabolisms showed a heterotrophic lifestyle primarily driven by C1 to C3 organics, e.g. formate, acetate, and pyruvate, and also suggested that the inability of autotrophy via a reductive tricarboxylic acid cycle may be due to the lack of ATP-dependent citrate lyase. In addition, the genome encodes numerous genes for chemoreceptors, chemotaxis-like systems, and signal transduction machineries. These signalling networks may be linked to this bacterium's versatile energy metabolisms and may provide ecophysiological advantages for <em>D. desulfuricans</em> SSM1 thriving in the physically and chemically fluctuating environments near hydrothermal vents. This is the first genome sequence from the phylum <em>Deferribacteres</em>.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Deferribacter desulfuricans&quot;"><em>Deferribacter desulfuricans</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;whole genome sequence&quot;">whole genome sequence</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;hydrothermal vent&quot;">hydrothermal vent</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="chemolithotroph">chemolithotroph</a></div>                            <h2 scrollto-destination=5027258 id="5027258" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Generally, the greatest challenge to thermophilic microorganisms living on hydrothermal vents may be posed by the risk of being swept out of the range of the vent and thereby losing the temperature range and necessary chemical supplies. They solve this problem by clinging to rocks in communal mats or swimming with a whip-like flagellum as sensing temperature or chemical stimuli to guide their directional movements.<sup><a reveal-id="DSQ005C1" data-open="DSQ005C1" class="link link-ref link-reveal xref-bibr">1</a></sup><em>Deferribacter desulfuricans</em> SMM1<sup>T</sup> (DSM 14783<sup>T</sup>) has been isolated from a deep-sea hydrothermal vent chimney at the Suiyo Seamount in the Izu-Bonin Arc, Japan.<sup><a reveal-id="DSQ005C2" data-open="DSQ005C2" class="link link-ref link-reveal xref-bibr">2</a></sup> The strain SMM1 is thermophilic (optimal temperature, 60–65°C) and a strictly anaerobic heterotroph capable of using complex organic compounds such as yeast extract and tryptone, ethanol, and various organic acids as sources of energy and carbon. There are three other species in the genus <em>Deferribacter</em>—<em>D. abyssi</em> and <em>D. autotrophicus</em>, which have been isolated from deep-sea hydrothermal vent environments, and <em>D. thermophilus</em>, which has been isolated from a subseafloor petroleum reservoir.<sup><a reveal-id="DSQ005C3  DSQ005C5" data-open="DSQ005C3  DSQ005C5" class="link link-ref link-reveal xref-bibr">3–5</a></sup> These bacteria are strictly anaerobic chemolithotrophs utilizing various organic compounds and H<sub>2</sub> as electron donors and nitrate, S<sup>0</sup> (<em>D. desulfuricans</em>, <em>D. abyssi</em>, and <em>D. autotrophicus</em>), and Fe (III) and Mn (IV) (<em>D. thermophilus</em>, <em>D. abyssi</em>, and <em>D. autotrophicus</em>) as electron acceptors. Such versatility for energy generation may provide an ecological advantage for deep-sea vent-dominating chemolithotrophs as has been proposed for members of <em>Aquificales</em> and <em>Epsilonproteobacteria</em>.<sup><a reveal-id="DSQ005C6" data-open="DSQ005C6" class="link link-ref link-reveal xref-bibr">6</a></sup> It has been shown, depending on the hydrothermal vent chimney environment, that <em>Deferribacter</em>-related species are the dominant species in these locations.<sup><a reveal-id="DSQ005C7" data-open="DSQ005C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSQ005C8" data-open="DSQ005C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Thus, it is intriguing to compare the genomes of <em>Deferribacter</em>-related species with other deep-sea vent chemolithotrophic species such as <em>Thiomicrospira crunogena</em>,<sup><a reveal-id="DSQ005C9" data-open="DSQ005C9" class="link link-ref link-reveal xref-bibr">9</a></sup><em>Nautilia profundicola</em>,<sup><a reveal-id="DSQ005C10" data-open="DSQ005C10" class="link link-ref link-reveal xref-bibr">10</a></sup><em>Sulfurovum</em> sp. NBC37-1,<sup><a reveal-id="DSQ005C11" data-open="DSQ005C11" class="link link-ref link-reveal xref-bibr">11</a></sup><em>Nitratiruptor</em> sp. SB155-2,<sup><a reveal-id="DSQ005C11" data-open="DSQ005C11" class="link link-ref link-reveal xref-bibr">11</a></sup> and <em>Persephonella marina</em><sup><a reveal-id="DSQ005C12" data-open="DSQ005C12" class="link link-ref link-reveal xref-bibr">12</a></sup> in order to highlight the genomic features that reflect their lifestyle in the environment of deep-sea hydrothermal vent chimneys. Here, we report the complete genome sequence of thermophilic <em>D. desulfuricans</em> SMM1 determined as the first published bacterial genome sequence from the phylum <em>Deferribacteres</em>. We provide a comparative analysis of the genome of <em>D. desulfuricans</em> SMM1 with those of five other chemolithotrophs isolated from deep-sea hydrothermal vent chimneys.</p>                            <h2 scrollto-destination=5027262 id="5027262" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5027263 id="5027263" class="section-title" data-legacy-id=s2a>2.1. Genome sequencing and assembly</h3>
<p><em>Deferribacter desulfuricans</em> SSM1 was grown heterotrophically as described previously,<sup><a reveal-id="DSQ005C2" data-open="DSQ005C2" class="link link-ref link-reveal xref-bibr">2</a></sup> and total DNA was isolated using proteinase K treatment followed by phenol extraction. The DNA was fragmented by HydroShear (Genomic Solutions), and three shotgun libraries were generated: small- and medium-insert (1.5 and 5 kb) plasmid libraries and a large-insert fosmid (35 kb) library as described previously.<sup><a reveal-id="DSQ005C13" data-open="DSQ005C13" class="link link-ref link-reveal xref-bibr">13</a></sup> All three libraries provided 15 times coverage of the genome. Assemblies were accomplished using the PHRED/PHRAP/CONSED suite (<a class="link link-uri" href="http://www.pharap.org">http://www.pharap.org</a>).<sup><a reveal-id="DSQ005C14" data-open="DSQ005C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Gaps between the assembled sequences were primarily closed by primer walking on gap-spanning library clones or with PCR products from genomic DNA. Quality assessment of the final assembly was performed as described previously.<sup><a reveal-id="DSQ005C13" data-open="DSQ005C13" class="link link-ref link-reveal xref-bibr">13</a></sup></p>                            <h3 scrollto-destination=5027265 id="5027265" class="section-title" data-legacy-id=s2b>2.2. Gene identification and annotation</h3>
<p>Putative non-translated genes were identified by using the Rfam<sup><a reveal-id="DSQ005C15" data-open="DSQ005C15" class="link link-ref link-reveal xref-bibr">15</a></sup> and tRNAscan-SE<sup><a reveal-id="DSQ005C16" data-open="DSQ005C16" class="link link-ref link-reveal xref-bibr">16</a></sup> programs, whereas rRNA genes were identified using the BLASTN program.<sup><a reveal-id="DSQ005C17" data-open="DSQ005C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Protein-coding sequences (CDSs) were predicted using a combination of GLIMMER<sup><a reveal-id="DSQ005C18" data-open="DSQ005C18" class="link link-ref link-reveal xref-bibr">18</a></sup> and GeneMarkS.<sup><a reveal-id="DSQ005C19" data-open="DSQ005C19" class="link link-ref link-reveal xref-bibr">19</a></sup> The remaining parts of the genome were screened further to find missed CDSs by a BLASTX homology search against protein databases. The predicted CDSs were translated and submitted for BLAST analysis against the NCBI non-redundant (nr),<sup><a reveal-id="DSQ005C20" data-open="DSQ005C20" class="link link-ref link-reveal xref-bibr">20</a></sup> UniProt,<sup><a reveal-id="DSQ005C21" data-open="DSQ005C21" class="link link-ref link-reveal xref-bibr">21</a></sup> and Kyoto Encyclopedia of Genes and Genomes (KEGG) GENES<sup><a reveal-id="DSQ005C22" data-open="DSQ005C22" class="link link-ref link-reveal xref-bibr">22</a></sup> databases. The results were collated and presented via a home-made browser (GBROWN) for manual verification. The start sites were manually inspected and altered in an alignment comparison with the best match. The revised CDS set was searched against Pfam<sup><a reveal-id="DSQ005C23" data-open="DSQ005C23" class="link link-ref link-reveal xref-bibr">23</a></sup> in addition to the previously mentioned databases. Specific functional assignments suggested by matches with these databases were only accepted when they had a minimum 30% amino acid sequence identity over 70% of the gene length and were supported by Pfam domain assignments, or were consistent with gene context in the genome (e.g. membership in a potential operon with other genes with convincing matches to curated databases). The KEGG PATHWAY and MetaCyc<sup><a reveal-id="DSQ005C24" data-open="DSQ005C24" class="link link-ref link-reveal xref-bibr">24</a></sup> databases were used for pathway reconstruction. Signal peptides were predicted with SignalP,<sup><a reveal-id="DSQ005C25" data-open="DSQ005C25" class="link link-ref link-reveal xref-bibr">25</a></sup> whereas transmembrane helices were predicted with SOSUI.<sup><a reveal-id="DSQ005C26" data-open="DSQ005C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Clusters of regularly interspaced repeats [clustered regularly interspaced short palindromic repeats (CRISPR)] were identified using the CRISPRFinder.<sup><a reveal-id="DSQ005C27" data-open="DSQ005C27" class="link link-ref link-reveal xref-bibr">27</a></sup></p>                            <h3 scrollto-destination=5027267 id="5027267" class="section-title" data-legacy-id=s2c>2.3. Comparative genomics analyses</h3>
<p>Protein sequence data for bacterial genomes were obtained from the KEGG database.<sup><a reveal-id="DSQ005C22" data-open="DSQ005C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Pairwise ortholog families were identified with the InParanoid program.<sup><a reveal-id="DSQ005C28" data-open="DSQ005C28" class="link link-ref link-reveal xref-bibr">28</a></sup> In order to examine genome-wide relationships of <em>D. desulfuricans</em> in terms of gene repertories, the ordination of bacterial genomes was generated using the non-metric multidimensional scaling (NMDS) method, which is a major branch of multivariate analysis. NMDS ordinations attempt to place all samples in an arbitrary two-dimensional space, in which the relative distances between the samples indicate the corresponding pairwise similarity. Hence, closely related organisms in the NMDS ordination would have similar gene repertoires. The gene context of each genome, based on clusters of orthologous groups (COGs),<sup><a reveal-id="DSQ005C29" data-open="DSQ005C29" class="link link-ref link-reveal xref-bibr">29</a></sup> was constructed with the following procedure. Each complete proteome was compared with the COG profile database (4450 families) using the RPSBLAST program.<sup><a reveal-id="DSQ005C17" data-open="DSQ005C17" class="link link-ref link-reveal xref-bibr">17</a></sup> A significant match (<em>E</em>-value of 10<sup>−5</sup>) was determined for each COG, so that any outlier detected by the Smirnov–Grubbs test at the 0.01 significance level was excluded from the COG profile. Any proteome was thus converted into a binary character matrix indicating the presence (1) or absence (0) of the COG families. The genome distance between species (A and B) was calculated by the formula: <em>D</em><sub>AB</sub> = 1 − <em>J</em><sub>AB</sub>, where <em>J</em><sub>AB</sub> is the Jaccard's coefficient, which reflects the similarity of gene content between A and B.<sup><a reveal-id="DSQ005C30" data-open="DSQ005C30" class="link link-ref link-reveal xref-bibr">30</a></sup> NMDS ordination was calculated using metaMDS in the vegan library running with the R programming language and environment (R package version 2.7.2: R Development Core Team, 2007, <a class="link link-uri" href="http://www.R.project.org">http://www.R.project.org</a>).</p>                            <h3 scrollto-destination=5027269 id="5027269" class="section-title" data-legacy-id=s2d>2.4. Nucleotide sequence accession numbers</h3>
<p>The sequences of the <em>D. desulfuricans</em> genome, consisting of the chromosome and plasmid sequences, are available at GenBank/EMBL/DDBJ under accession numbers AP011529 and AP011530, respectively.</p>                            <h2 scrollto-destination=5027271 id="5027271" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5027272 id="5027272" class="section-title" data-legacy-id=s3a>3.1. General genome features</h3>
<p>The genome of <em>Deferribacter desulfuricans</em> SSM1 contains a single circular chromosome of 2 234 389 bp and a megaplasmid (pDF308) of 308 544 bp (Table <a reveal-id="DSQ005TB1" data-open="DSQ005TB1" class="link link-reveal link-table xref-fig">1</a>). Their average G + C contents are 31.1% and 24.5%, respectively. The chromosome displays two clear GC skew transitions that likely correspond to the DNA replication origin and terminus (Fig. <a reveal-id="DSQ005F1" data-open="DSQ005F1" class="link link-reveal link-table xref-fig">1</a>). Annotation of the chromosomal sequence reveals 2117 CDSs, of which 1404 (66%) can be functionally assigned. The megaplasmid encodes 257 CDSs, whereas more than two-thirds are unique and exhibit no apparent similarity with any of the CDSs present in the database. Interestingly, 17 copies of gene cluster encoding two transposases belonging to the IS<em>200</em> and IS<em>605</em> family present in the megaplasmid but none were found in the chromosome. The chromosome contains two ribosomal RNA operons with a 16S–23S–5S rRNA gene alignment. In all, 43 tRNA genes were identified (Table <a reveal-id="DSQ005TB1" data-open="DSQ005TB1" class="link link-reveal link-table xref-fig">1</a>). In addition to the regular tRNA genes, the <em>D. desulfuricans</em> genome also contains <em>selC</em> for the selenocysteine tRNA. The other components necessary for the selenocysteine system, e.g. selenocysteine synthetase (SelA), the specific elongation factor (SelB), selenophosphate synthetase (SelD), and seryl-tRNA synthetase (SerS), are also present in the genome. As for small RNA-encoding genes, potential genes for RNase P RNA (RnpB) and 6S RNA (SsrS) were assigned by using the Rfam database. </p>                        <a id="5027274" scrollto-destination="5027274"></a>
<div content-id="DSQ005TB1" class="table-modal"><div class="table-wrap"><div id="DSQ005TB1" class="table-wrap-title" data-id="DSQ005TB1"><div class="label -label">Table 1</div><div class="caption"><p>Summary of genome features of <em>D. desulfuricans</em> SSM1</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Characteristics </th><th>Chromosome </th><th>Plasmid </th></tr></thead><tbody><tr><td>Size (bp) </td><td>2 234 389 </td><td>308 544 </td></tr><tr><td>G + C content (%) </td><td>31.1 </td><td>24.5 </td></tr><tr><td>No. of identified protein-coding genes </td><td> </td><td> </td></tr><tr><td> Total </td><td>2117 </td><td>257 </td></tr><tr><td> Functionally assigned </td><td>1404 </td><td>63 </td></tr><tr><td> Conserved hypothetical </td><td>494 </td><td>20 </td></tr><tr><td> Hypothetical </td><td>219 </td><td>174 </td></tr><tr><td> Pseudogenes </td><td>17 </td><td>9 </td></tr><tr><td>Average gene length (bp) </td><td>976 </td><td>993 </td></tr><tr><td>Coding density (%) </td><td>93.7 </td><td>88.2 </td></tr><tr><td colspan="3">No. of identified RNA genes </td></tr><tr><td> rRNA operons </td><td>2 </td><td>— </td></tr><tr><td> tRNA </td><td>43 </td><td>— </td></tr><tr><td> Small RNA<sup>a</sup> </td><td>2 </td><td>— </td></tr><tr><td colspan="3">Size (bp) of genomic islands (% G + C content) </td></tr><tr><td> DDGI-1 </td><td>35 072 (31.4%) </td><td>— </td></tr><tr><td> DDGI-2 </td><td>20 995 (28.2%) </td><td> </td></tr><tr><td> DDGI-3 </td><td>36 900 (31.0%) </td><td> </td></tr><tr><td> CRISPR elements </td><td>2 </td><td>2 </td></tr></tbody></table></div><div content-id="DSQ005TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Characteristics </th><th>Chromosome </th><th>Plasmid </th></tr></thead><tbody><tr><td>Size (bp) </td><td>2 234 389 </td><td>308 544 </td></tr><tr><td>G + C content (%) </td><td>31.1 </td><td>24.5 </td></tr><tr><td>No. of identified protein-coding genes </td><td> </td><td> </td></tr><tr><td> Total </td><td>2117 </td><td>257 </td></tr><tr><td> Functionally assigned </td><td>1404 </td><td>63 </td></tr><tr><td> Conserved hypothetical </td><td>494 </td><td>20 </td></tr><tr><td> Hypothetical </td><td>219 </td><td>174 </td></tr><tr><td> Pseudogenes </td><td>17 </td><td>9 </td></tr><tr><td>Average gene length (bp) </td><td>976 </td><td>993 </td></tr><tr><td>Coding density (%) </td><td>93.7 </td><td>88.2 </td></tr><tr><td colspan="3">No. of identified RNA genes </td></tr><tr><td> rRNA operons </td><td>2 </td><td>— </td></tr><tr><td> tRNA </td><td>43 </td><td>— </td></tr><tr><td> Small RNA<sup>a</sup> </td><td>2 </td><td>— </td></tr><tr><td colspan="3">Size (bp) of genomic islands (% G + C content) </td></tr><tr><td> DDGI-1 </td><td>35 072 (31.4%) </td><td>— </td></tr><tr><td> DDGI-2 </td><td>20 995 (28.2%) </td><td> </td></tr><tr><td> DDGI-3 </td><td>36 900 (31.0%) </td><td> </td></tr><tr><td> CRISPR elements </td><td>2 </td><td>2 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Includes genes for RNase P RNA and 6S RNA.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5027274" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ005TB1" class="table-wrap-title" data-id="DSQ005TB1"><div class="label -label">Table 1</div><div class="caption"><p>Summary of genome features of <em>D. desulfuricans</em> SSM1</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Characteristics </th><th>Chromosome </th><th>Plasmid </th></tr></thead><tbody><tr><td>Size (bp) </td><td>2 234 389 </td><td>308 544 </td></tr><tr><td>G + C content (%) </td><td>31.1 </td><td>24.5 </td></tr><tr><td>No. of identified protein-coding genes </td><td> </td><td> </td></tr><tr><td> Total </td><td>2117 </td><td>257 </td></tr><tr><td> Functionally assigned </td><td>1404 </td><td>63 </td></tr><tr><td> Conserved hypothetical </td><td>494 </td><td>20 </td></tr><tr><td> Hypothetical </td><td>219 </td><td>174 </td></tr><tr><td> Pseudogenes </td><td>17 </td><td>9 </td></tr><tr><td>Average gene length (bp) </td><td>976 </td><td>993 </td></tr><tr><td>Coding density (%) </td><td>93.7 </td><td>88.2 </td></tr><tr><td colspan="3">No. of identified RNA genes </td></tr><tr><td> rRNA operons </td><td>2 </td><td>— </td></tr><tr><td> tRNA </td><td>43 </td><td>— </td></tr><tr><td> Small RNA<sup>a</sup> </td><td>2 </td><td>— </td></tr><tr><td colspan="3">Size (bp) of genomic islands (% G + C content) </td></tr><tr><td> DDGI-1 </td><td>35 072 (31.4%) </td><td>— </td></tr><tr><td> DDGI-2 </td><td>20 995 (28.2%) </td><td> </td></tr><tr><td> DDGI-3 </td><td>36 900 (31.0%) </td><td> </td></tr><tr><td> CRISPR elements </td><td>2 </td><td>2 </td></tr></tbody></table></div><div content-id="DSQ005TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Characteristics </th><th>Chromosome </th><th>Plasmid </th></tr></thead><tbody><tr><td>Size (bp) </td><td>2 234 389 </td><td>308 544 </td></tr><tr><td>G + C content (%) </td><td>31.1 </td><td>24.5 </td></tr><tr><td>No. of identified protein-coding genes </td><td> </td><td> </td></tr><tr><td> Total </td><td>2117 </td><td>257 </td></tr><tr><td> Functionally assigned </td><td>1404 </td><td>63 </td></tr><tr><td> Conserved hypothetical </td><td>494 </td><td>20 </td></tr><tr><td> Hypothetical </td><td>219 </td><td>174 </td></tr><tr><td> Pseudogenes </td><td>17 </td><td>9 </td></tr><tr><td>Average gene length (bp) </td><td>976 </td><td>993 </td></tr><tr><td>Coding density (%) </td><td>93.7 </td><td>88.2 </td></tr><tr><td colspan="3">No. of identified RNA genes </td></tr><tr><td> rRNA operons </td><td>2 </td><td>— </td></tr><tr><td> tRNA </td><td>43 </td><td>— </td></tr><tr><td> Small RNA<sup>a</sup> </td><td>2 </td><td>— </td></tr><tr><td colspan="3">Size (bp) of genomic islands (% G + C content) </td></tr><tr><td> DDGI-1 </td><td>35 072 (31.4%) </td><td>— </td></tr><tr><td> DDGI-2 </td><td>20 995 (28.2%) </td><td> </td></tr><tr><td> DDGI-3 </td><td>36 900 (31.0%) </td><td> </td></tr><tr><td> CRISPR elements </td><td>2 </td><td>2 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Includes genes for RNase P RNA and 6S RNA.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5027274" target="_blank">View Large</a></div></div>                        <a id="5027275" scrollto-destination="5027275"></a>
<div id="DSQ005F1" data-id="DSQ005F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq005/2/m_dsq00501.jpeg?Expires=2147483647&Signature=j0Uzv1yG7vwpgNmHT~kGdlF8TDL7ugr4QfIi9JEcTOF39yLqNlQDel3JbdIaBAdwE3j0B2JdwTK4Y18AyNzg-6yJsVn1OlDFL9VTzda2iTjFvgdOFpycN4SCVBOduJmaK0oBngGMKf2y88BlsQaoiXmRdjRGSAbGLfFyM9jVcnbvxVHNw5XwriXNcd5E1hmkzdf6fl0pMprASEQIs~jjdUg6zDHTFV8nBy4LMy3vkc~upr-1aFIBUdvdHatkQ54ZjnlP9t0EqCjzoHoi3OVPxY68ZKX4y5D-czIIg58u-D0tT2OmiQkORI-F~aijPLZAMophhFjPDYgxRLtvIIOcEA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Circular representation of the D. desulfuricans SSM1 genome. (A) Chromosome. (B) Megaplasmid pDF308. From the inside, the first and second circles show the GC skew (values greater than or less than zero are indicated in green and pink, respectively) and the G + C percent content (values greater or smaller than the average percentage in the overall chromosome or plasmid are shown in blue and sky blue, respectively) in a 10-kb window with 100-bp step, respectively. The third and fourth circles show the presence of RNAs (rRNA, tRNA, and small RNA genes); CDSs aligned in the clockwise and counterclockwise directions are indicated in the upper and lower sides of the circle, respectively. Different colours indicate different functional categories: red for information storage and processing; green for metabolism; blue for cellular processes and signalling; grey for poorly characterized function; and purple for RNA genes. The outermost circle shows the location of genomic islands (red) and CRISPR/Cas systems (blue). The ‘0’ marked on the outmost circles corresponds to the putative replication origin, and the putative replication termination site of the chromosome is at 1.23 Mb." path-from-xml="dsq00501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5027275/dsq00501.jpeg" path-from-xml="dsq00501.jpeg" target="_blank">View large</a><a section="5027275" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq005/2/dsq00501.jpeg?Expires=2147483647&Signature=O8x9fhIZSJ3ql9CzuZ6uk1iw08hjOpmmzfIdVcnRuuyO04QlmBdrYPcso~nmUBDLb4A7f27LyMTZj2SBvXd4sF2xXVg9myDRKUDnLI9ptzwctjFNaPo727WU0oUun7~0ugKHOYhC0YyOpqmOdFEZgj86RGTOYNG5m3CcjN707cQffp4oa6ESdKKEQ4Avk12h7ZLKiuZaamEjUTBJxR-vohjP0JvdjrwySEgwSQ7zO59DyqWU-THg422Z746AIGv-EXpvrpsD3xysWy4eQFtLkmkQHqjCbfnLJVaP5hcA5S~G4st9tYR45qEIqSqQ0qvxvx3JDMsMc5PJzi1OK5N0lw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5027275&ar=353305&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Circular representation of the <em>D. desulfuricans</em> SSM1 genome. (A) Chromosome. (B) Megaplasmid pDF308. From the inside, the first and second circles show the GC skew (values greater than or less than zero are indicated in green and pink, respectively) and the G + C percent content (values greater or smaller than the average percentage in the overall chromosome or plasmid are shown in blue and sky blue, respectively) in a 10-kb window with 100-bp step, respectively. The third and fourth circles show the presence of RNAs (rRNA, tRNA, and small RNA genes); CDSs aligned in the clockwise and counterclockwise directions are indicated in the upper and lower sides of the circle, respectively. Different colours indicate different functional categories: red for information storage and processing; green for metabolism; blue for cellular processes and signalling; grey for poorly characterized function; and purple for RNA genes. The outermost circle shows the location of genomic islands (red) and CRISPR/Cas systems (blue). The ‘0’ marked on the outmost circles corresponds to the putative replication origin, and the putative replication termination site of the chromosome is at 1.23 Mb.</p></div></div><div content-id="DSQ005F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq005/2/m_dsq00501.jpeg?Expires=2147483647&Signature=j0Uzv1yG7vwpgNmHT~kGdlF8TDL7ugr4QfIi9JEcTOF39yLqNlQDel3JbdIaBAdwE3j0B2JdwTK4Y18AyNzg-6yJsVn1OlDFL9VTzda2iTjFvgdOFpycN4SCVBOduJmaK0oBngGMKf2y88BlsQaoiXmRdjRGSAbGLfFyM9jVcnbvxVHNw5XwriXNcd5E1hmkzdf6fl0pMprASEQIs~jjdUg6zDHTFV8nBy4LMy3vkc~upr-1aFIBUdvdHatkQ54ZjnlP9t0EqCjzoHoi3OVPxY68ZKX4y5D-czIIg58u-D0tT2OmiQkORI-F~aijPLZAMophhFjPDYgxRLtvIIOcEA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Circular representation of the D. desulfuricans SSM1 genome. (A) Chromosome. (B) Megaplasmid pDF308. From the inside, the first and second circles show the GC skew (values greater than or less than zero are indicated in green and pink, respectively) and the G + C percent content (values greater or smaller than the average percentage in the overall chromosome or plasmid are shown in blue and sky blue, respectively) in a 10-kb window with 100-bp step, respectively. The third and fourth circles show the presence of RNAs (rRNA, tRNA, and small RNA genes); CDSs aligned in the clockwise and counterclockwise directions are indicated in the upper and lower sides of the circle, respectively. Different colours indicate different functional categories: red for information storage and processing; green for metabolism; blue for cellular processes and signalling; grey for poorly characterized function; and purple for RNA genes. The outermost circle shows the location of genomic islands (red) and CRISPR/Cas systems (blue). The ‘0’ marked on the outmost circles corresponds to the putative replication origin, and the putative replication termination site of the chromosome is at 1.23 Mb." path-from-xml="dsq00501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5027275/dsq00501.jpeg" path-from-xml="dsq00501.jpeg" target="_blank">View large</a><a section="5027275" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq005/2/dsq00501.jpeg?Expires=2147483647&Signature=O8x9fhIZSJ3ql9CzuZ6uk1iw08hjOpmmzfIdVcnRuuyO04QlmBdrYPcso~nmUBDLb4A7f27LyMTZj2SBvXd4sF2xXVg9myDRKUDnLI9ptzwctjFNaPo727WU0oUun7~0ugKHOYhC0YyOpqmOdFEZgj86RGTOYNG5m3CcjN707cQffp4oa6ESdKKEQ4Avk12h7ZLKiuZaamEjUTBJxR-vohjP0JvdjrwySEgwSQ7zO59DyqWU-THg422Z746AIGv-EXpvrpsD3xysWy4eQFtLkmkQHqjCbfnLJVaP5hcA5S~G4st9tYR45qEIqSqQ0qvxvx3JDMsMc5PJzi1OK5N0lw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5027275&ar=353305&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Circular representation of the <em>D. desulfuricans</em> SSM1 genome. (A) Chromosome. (B) Megaplasmid pDF308. From the inside, the first and second circles show the GC skew (values greater than or less than zero are indicated in green and pink, respectively) and the G + C percent content (values greater or smaller than the average percentage in the overall chromosome or plasmid are shown in blue and sky blue, respectively) in a 10-kb window with 100-bp step, respectively. The third and fourth circles show the presence of RNAs (rRNA, tRNA, and small RNA genes); CDSs aligned in the clockwise and counterclockwise directions are indicated in the upper and lower sides of the circle, respectively. Different colours indicate different functional categories: red for information storage and processing; green for metabolism; blue for cellular processes and signalling; grey for poorly characterized function; and purple for RNA genes. The outermost circle shows the location of genomic islands (red) and CRISPR/Cas systems (blue). The ‘0’ marked on the outmost circles corresponds to the putative replication origin, and the putative replication termination site of the chromosome is at 1.23 Mb.</p></div></div><p>The <em>D. desulfuricans</em> chromosome contains three genomic islands, termed DDGI-1 (coordinates 278 054–313 125), DDGI-2 (coordinates 687 244–708 238), and DDGI-3 (coordinates 864 851–901 750), possibly acquired via horizontal gene transfer (Fig. <a reveal-id="DSQ005F1" data-open="DSQ005F1" class="link link-reveal link-table xref-fig">1</a> and Table <a reveal-id="DSQ005TB1" data-open="DSQ005TB1" class="link link-reveal link-table xref-fig">1</a>). They have many specific features of genomic island, such as the tRNA gene locus at junctions and the presence of direct repeats and phage integrases (DEFDS_0277, DEFDS_0708, and DEFDS_0871), but no anomalous GC content.<sup><a reveal-id="DSQ005C31" data-open="DSQ005C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Most of the genes in the genomic islands encode hypothetical proteins, but some of those appeared to encode the functional proteins related to the adaptation mechanisms for the hydrothermal vent environments. For example, the DDGI-1 includes two heavy-metal transporting P-type ATPases (DEFDS_0300 and DEFDS_0301). The heavy-metal transporters could be responsible for heavy metal tolerance in the hydrothermal vent environment. The DDGI-2 has a toxin–antitoxin system (DEFDS_0716 and DEFDS_0717) involved in phage defence and the stress response.<sup><a reveal-id="DSQ005C32" data-open="DSQ005C32" class="link link-ref link-reveal xref-bibr">32</a></sup></p><p>Similar to most other thermophiles, <em>D. desulfuricans</em> has the CRISPR elements together with their associated genes (<em>cas</em>), which would serve as immunity against phages, possibly by an RNA-interference-like mechanism.<sup><a reveal-id="DSQ005C33" data-open="DSQ005C33" class="link link-ref link-reveal xref-bibr">33</a></sup> The CRISPR/Cas systems were identified in the chromosome (coordinates 77 878– 86 387) and the plasmid (coordinates 267 868–276 377). Both systems have an identical repeat, and the <em>cas</em> genes also show a high relevance with each other. Regarding the repeat sequence and structure of the <em>cas</em> genes, the systems of <em>D. desulfuricans</em> were closely related with those observed in other thermophiles, such as <em>Sulfurihydrogenibium azorense</em>, <em>Thermoanaerobacter pseudethanolicus</em>, and the homo-acetogen <em>Clostridium thermocellum</em>. Therefore, the <em>D. desulfuricans</em> CRISPR/Cas systems may have been acquired horizontally via the megaplasmid.</p>                            <h3 scrollto-destination=5027278 id="5027278" class="section-title" data-legacy-id=s3b>3.2. Orthologous relationships among the bacterial species</h3>
<p>Among the proteins identified in the <em>D. desulfuricans</em> genome, ∼30% of them showed the highest similarity to those species from <em>Deltaproteobacteria</em>, especially <em>Geobacter</em>, <em>Pelobacter</em>, <em>Desulfovibrio</em>, and <em>Syntrophobacter</em> spp., and ∼14% were found to be most similar to those from clostridial species. The remaining proteins showed the highest similarities to several species from <em>Aquificae</em> (5.3%), <em>Gammaproteobacteria</em> (4.8%), and <em>Epsilonproteobacteria</em> (3.7%). Approximately half of the proteins were shared between <em>D. desulfuricans</em> and deltaproteobacterial species, such as <em>Geobacter sulfurreducens</em> PCA (1106 orthologs), <em>Pelobacter carbinolicus</em> (1046 orthologs), <em>Syntrophobacter fumaroxidans</em> (985 orthologs), and <em>Desulfovibrio vulgaris</em> subsp. <em>vulgaris</em> Hildenborough (936 orthologs), although <em>D. desulfuricans</em> is phylogenetically distant from <em>Deltaproteobacteria</em>.</p><p>For the next step of orthologous analysis, we performed multivariate analyses on the basis of the gene repertories from 51 bacterial species in order to schematically express the orthologous relationships of <em>D. desulfuricans</em> among other bacterial species (Fig. <a reveal-id="DSQ005F2" data-open="DSQ005F2" class="link link-reveal link-table xref-fig">2</a>A). As expected, the correlation map indicated that <em>D. desulfuricans</em> and the deltaproteobacterial species, which have strong orthologous relationships, gathered around the centre of map. The same analysis to focus on six species of the class <em>Deltaproteobacteria</em> with <em>D. desulfuricans</em> showed that <em>D. desulfuricans</em> is especially related to three species, <em>G. sulfurreducens</em> (gsu), <em>P. carbinolicus</em> (pca), and <em>D. vulgaris</em> (dvu), possessing sulphur- or sulphate-reducing properties (Fig. <a reveal-id="DSQ005F2" data-open="DSQ005F2" class="link link-reveal link-table xref-fig">2</a>B). This result seems to attribute that these four species share many genes involved in physiological and metabolic properties such as anaerobiosis and assimilations system for small organic molecules (e.g. acetate, pyruvate, and lactate) as carbon and energy source.</p>                        <a id="5027281" scrollto-destination="5027281"></a>
<div id="DSQ005F2" data-id="DSQ005F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq005/2/m_dsq00502.jpeg?Expires=2147483647&Signature=tNTQMhbl6G2AXfj~h4RyVOc3~K~48wGv5lHE6s~ZJOOuKkqLH3Cz51k5XwNNTw8i7JW~BYypQklYf76tt5GgjF7UrSxdo7fNSq-XpTAM9NZKlQVJ7sUvrAQOZEkmF6~e56HPggutEYNgLTxyPR9c8NxutTxL67vAx3fVcci~iJZWswSrJbu-XaNX-R5X~yXArl5CRTxxkhf3rhOIAfD~8a8WRWvI4PcalcCYFf6-pK4PjVjRlsl19jdLZgPF9lKDq~ijmvUrlwg46m0o0zdygUYZzPzyW5HguxBgEUot2nPGJ~p5X5b2ySUQGo5ARPD~2e7-HxJY4n1CNuRn3J~urA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Ordination plot of bacterial genomes using NMDS. (A) Analysis with 50 species belonging to 12 phyla and 5 classes, and D. desulfuricans SSM1. (B) Analysis with six species within Deltaproteobacteria and D. desulfuricans SSM1. Distances were calculated from gene profiles based on COG families. The abbreviation corresponding to the KEGG organism code is used as the label for the species name (detailed explanations are described in Supplementary Table S1). The labels are colour-coded according to their taxonomic groups (phylum/class): red, D. desulfuricans SSM1 (def); orange, Deltaproteobacteria; yellow, Epsilonproteobacteria; blue, other Proteobacteria; purple, Firmicutes; green, Chlorobi; black, Aquificae, Thermotogae, and Deinococcus-Thermus; and grey, other bacteria." path-from-xml="dsq00502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5027281/dsq00502.jpeg" path-from-xml="dsq00502.jpeg" target="_blank">View large</a><a section="5027281" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq005/2/dsq00502.jpeg?Expires=2147483647&Signature=YsW9zc2TK12OAa6T11-0eUQ7C253vfBfmPsPonO5gDtBDOV4UeLXyFaylws-khRwW~ThQ4PMuFKnLx0IPI5FlflzCoaFzeijhg4q2uwzw6JYl9RoRlg0df569~barsbSbTZXYMX2a3DOhxYICTGJY17II1DF-ULBwad8x5tje87z0q3fhmU~XEb6IgPxfH3GurW3Q76U6CgFLNC58KbegREoZssCuyNQKKpufojbGuVwknZQS3~jHmfYle-D4I26vSFKo6~1fzhwSj93hicuHVdE~mrTz~oSrjJxF2aud74G2JTxUYeNUVQjISxXLUiIEc145Ex1vhbmiyHfikWJng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5027281&ar=353305&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Ordination plot of bacterial genomes using NMDS. (A) Analysis with 50 species belonging to 12 phyla and 5 classes, and <em>D. desulfuricans</em> SSM1. (B) Analysis with six species within <em>Deltaproteobacteria</em> and <em>D. desulfuricans</em> SSM1. Distances were calculated from gene profiles based on COG families. The abbreviation corresponding to the KEGG organism code is used as the label for the species name (detailed explanations are described in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The labels are colour-coded according to their taxonomic groups (phylum/class): red, <em>D. desulfuricans</em> SSM1 (def); orange, <em>Deltaproteobacteria</em>; yellow, <em>Epsilonproteobacteria</em>; blue, other <em>Proteobacteria</em>; purple, <em>Firmicutes</em>; green, <em>Chlorobi</em>; black, <em>Aquificae</em>, <em>Thermotogae</em>, and <em>Deinococcus-Thermus</em>; and grey, other bacteria.</p></div></div><div content-id="DSQ005F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq005/2/m_dsq00502.jpeg?Expires=2147483647&Signature=tNTQMhbl6G2AXfj~h4RyVOc3~K~48wGv5lHE6s~ZJOOuKkqLH3Cz51k5XwNNTw8i7JW~BYypQklYf76tt5GgjF7UrSxdo7fNSq-XpTAM9NZKlQVJ7sUvrAQOZEkmF6~e56HPggutEYNgLTxyPR9c8NxutTxL67vAx3fVcci~iJZWswSrJbu-XaNX-R5X~yXArl5CRTxxkhf3rhOIAfD~8a8WRWvI4PcalcCYFf6-pK4PjVjRlsl19jdLZgPF9lKDq~ijmvUrlwg46m0o0zdygUYZzPzyW5HguxBgEUot2nPGJ~p5X5b2ySUQGo5ARPD~2e7-HxJY4n1CNuRn3J~urA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Ordination plot of bacterial genomes using NMDS. (A) Analysis with 50 species belonging to 12 phyla and 5 classes, and D. desulfuricans SSM1. (B) Analysis with six species within Deltaproteobacteria and D. desulfuricans SSM1. Distances were calculated from gene profiles based on COG families. The abbreviation corresponding to the KEGG organism code is used as the label for the species name (detailed explanations are described in Supplementary Table S1). The labels are colour-coded according to their taxonomic groups (phylum/class): red, D. desulfuricans SSM1 (def); orange, Deltaproteobacteria; yellow, Epsilonproteobacteria; blue, other Proteobacteria; purple, Firmicutes; green, Chlorobi; black, Aquificae, Thermotogae, and Deinococcus-Thermus; and grey, other bacteria." path-from-xml="dsq00502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5027281/dsq00502.jpeg" path-from-xml="dsq00502.jpeg" target="_blank">View large</a><a section="5027281" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq005/2/dsq00502.jpeg?Expires=2147483647&Signature=YsW9zc2TK12OAa6T11-0eUQ7C253vfBfmPsPonO5gDtBDOV4UeLXyFaylws-khRwW~ThQ4PMuFKnLx0IPI5FlflzCoaFzeijhg4q2uwzw6JYl9RoRlg0df569~barsbSbTZXYMX2a3DOhxYICTGJY17II1DF-ULBwad8x5tje87z0q3fhmU~XEb6IgPxfH3GurW3Q76U6CgFLNC58KbegREoZssCuyNQKKpufojbGuVwknZQS3~jHmfYle-D4I26vSFKo6~1fzhwSj93hicuHVdE~mrTz~oSrjJxF2aud74G2JTxUYeNUVQjISxXLUiIEc145Ex1vhbmiyHfikWJng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5027281&ar=353305&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Ordination plot of bacterial genomes using NMDS. (A) Analysis with 50 species belonging to 12 phyla and 5 classes, and <em>D. desulfuricans</em> SSM1. (B) Analysis with six species within <em>Deltaproteobacteria</em> and <em>D. desulfuricans</em> SSM1. Distances were calculated from gene profiles based on COG families. The abbreviation corresponding to the KEGG organism code is used as the label for the species name (detailed explanations are described in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The labels are colour-coded according to their taxonomic groups (phylum/class): red, <em>D. desulfuricans</em> SSM1 (def); orange, <em>Deltaproteobacteria</em>; yellow, <em>Epsilonproteobacteria</em>; blue, other <em>Proteobacteria</em>; purple, <em>Firmicutes</em>; green, <em>Chlorobi</em>; black, <em>Aquificae</em>, <em>Thermotogae</em>, and <em>Deinococcus-Thermus</em>; and grey, other bacteria.</p></div></div>                            <h3 scrollto-destination=5027282 id="5027282" class="section-title" data-legacy-id=s3c>3.3. Central metabolism</h3>
<p><em>Deferribacter desulfuricans</em> grows heterotrophically using a variety of organic acids (formate, acetate, propionate, pyruvate, and lactate) with nitrate or S<sup>0</sup> as a primary electron acceptor.<sup><a reveal-id="DSQ005C2" data-open="DSQ005C2" class="link link-ref link-reveal xref-bibr">2</a></sup> As shown in Fig. <a reveal-id="DSQ005F3" data-open="DSQ005F3" class="link link-reveal link-table xref-fig">3</a>, the reconstructed central metabolic pathways from the <em>D. desulfuricans</em> genome certainly showed that these organic acids could be utilized as energy and carbon sources via the oxidative tricarboxylic acid (TCA) cycle and various anaplerotic pathways. Acetate used as a <em>D. desulfuricans</em> SSM1 growth substrate can be activated to acetyl-CoA either via a single-step reaction by an acetyl-CoA synthetase (Acs: DEFDS_0854) or a two-step reaction by acetate kinase (AckA: DEFDS_1816) and phosphate acetyltransferase (Pta: DEFDS_1815). The AckA–Pta reaction can also operate in acetate production from acetyl-CoA in a fermentative metabolism of pyruvate, and the enzymatic action of AckA results in ATP production by substrate-level phosphorylation. Lactate may be oxidized to pyruvate by a malate dehydrogenase with a broad substrate specificity. In contrast to acetate and lactate, propionate is fed directly into the TCA cycle via succinyl-CoA. Propionyl-CoA, produced by the activation of propionate by acetyl-CoA synthetase, is converted to succinyl-CoA through a methylmalonyl pathway, including a propionyl-CoA carboxylase (DEFDS_1227-8), a methylmalonyl-CoA epimerase (DEFDS_1935), and a methylmalonyl-CoA mutase (MutA: DEFDS_2077 and MutB: DEFDS_1225). The catabolism of pyruvate reflects the anaerobic nature of <em>D. desulfuricans</em>. Conversion of pyruvate to acetyl-CoA is performed by either pyruvate ferredoxin oxidoreductase (POR: DEFDS_0568) or pyruvate-formate lyase (PFL: DEFDS_2103).</p>                        <a id="5027284" scrollto-destination="5027284"></a>
<div id="DSQ005F3" data-id="DSQ005F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq005/2/m_dsq00503.jpeg?Expires=2147483647&Signature=CCuFngyRq6~euS4bmstlYeeCAcPnd64ixF~GQq7XOxS4YOS6uurXoxEVyE~dxVZIdcjQoBtLA~NagfPsr7L7N8XYQLEk9ZONPxByOcSCqweDr5NE56SZreRNrd4bImQDAFiS80Crr44UYidhYrVSxBY21OpI~g9r-Gd-tkguSJhf6ziBAPfLHtM2jSQKwcWB1JsTK1N6kk~RtZFyEbLC9a4ykZl8INtVwBoUTPHgCksLQjsTS8HiL2wF9ZEivXibY2-529drf93y3seLelc2AgJNWDW8k5mwmedmHypLLeFnoDoUKL9ybPiD32mZPXmghvAdEk8DrQBeH8~x08ByvA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Central metabolism based on potential growth substrates and metabolic capacities reconstructed from the D. desulfuricans genome. This figure displays the flow of carbon in the metabolism of various organic acids (acetate, propionate, butyrate, lactate, and glycerol) predicted from the genome information of D. desulfuricans SSM1. The reversible and irreversible reactions catalysed by enzymes are indicated with both and single arrowhead, respectively. POR, pyruvate ferredoxin oxidoreductase; PFL, pyruvate formate-lyase; Pyc, pyruvate carboxylase; Pck, phosphoenolpyruvate carboxykinase; PykA, pyruvate kinase; Ppd, pyruvate phosphate dikinase; MaeB, malate dehydrogenase (oxaloacetate-decarboxylating); Mdh, malate dehydrogenase; SucCD, succinyl-CoA synthase; Sdh, succinate dehydrogenase; OOR, 2-oxogultarate ferredoxin oxidoreductase; Fba, fructose-bisphosphatase; Pfk, 6-phosphofructokinase; Acs, acetyl-CoA synthetase; Ack, acetate kinase; Pta, phosphate acetyltransferase; AOR, aldehyde ferredoxin oxidoreductase; GlpK, glycerol kinase; GlpD, glycerol-3-phosphate dehydrogenase; Thl, acetoacetyl-CoA thiolase; Hbd, 3-hydroxybutyryl-CoA dehydrogenase; Crt, 3-hydroxybutyryl-CoA dehydratase (crotonase); Bcd, butyryl-CoA dehydrogenase; Ptb, phosphate butyryltransferase; Buk, butyrate kinase; Rnf, Rnf-type ion-translocating electron transport complex; Etf, electron transfer flavoprotein complex; PccAB, propionyl-CoA carboxylase; MutAB, methylmalonyl-CoA mutase; 2Pi, diphosphate; and Fdox/Fdred, ferredoxin, oxidized and reduced forms respectively." path-from-xml="dsq00503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5027284/dsq00503.jpeg" path-from-xml="dsq00503.jpeg" target="_blank">View large</a><a section="5027284" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq005/2/dsq00503.jpeg?Expires=2147483647&Signature=UwGvEDyI6tdYnjXfr9aZQMzykYF2NKMKUrEw-h1kjyQCaYDNJrVx04fYfbTXahdL3b5XzOV~Nx6j-gwb4DqF2ogwTzVpyXoyiYFhY1lA2CTnyCgRlt2HwU4ZLmGdBnXjefSyNuKb80EbNNKYyL3Mjg4nZJnju2uPCWeUFViJYLSv4MNAfR25NXUKpVHsODUHOMAozy~kJkGHIEPQVfVKoawziEwsPx-pB1CYctSKn-kVP7i-y3qkHtk9x-fx8eXnkcIwF8tVixKFlBUENN1Hy9yMZGGdZf6gpp7ImWyvsT3~UYPqmmJuUl3bo7SMv7aZf1GuiR7Xsevxw4NxAUkwLA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5027284&ar=353305&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Central metabolism based on potential growth substrates and metabolic capacities reconstructed from the <em>D. desulfuricans</em> genome. This figure displays the flow of carbon in the metabolism of various organic acids (acetate, propionate, butyrate, lactate, and glycerol) predicted from the genome information of <em>D. desulfuricans</em> SSM1. The reversible and irreversible reactions catalysed by enzymes are indicated with both and single arrowhead, respectively. POR, pyruvate ferredoxin oxidoreductase; PFL, pyruvate formate-lyase; Pyc, pyruvate carboxylase; Pck, phosphoenolpyruvate carboxykinase; PykA, pyruvate kinase; Ppd, pyruvate phosphate dikinase; MaeB, malate dehydrogenase (oxaloacetate-decarboxylating); Mdh, malate dehydrogenase; SucCD, succinyl-CoA synthase; Sdh, succinate dehydrogenase; OOR, 2-oxogultarate ferredoxin oxidoreductase; Fba, fructose-bisphosphatase; Pfk, 6-phosphofructokinase; Acs, acetyl-CoA synthetase; Ack, acetate kinase; Pta, phosphate acetyltransferase; AOR, aldehyde ferredoxin oxidoreductase; GlpK, glycerol kinase; GlpD, glycerol-3-phosphate dehydrogenase; Thl, acetoacetyl-CoA thiolase; Hbd, 3-hydroxybutyryl-CoA dehydrogenase; Crt, 3-hydroxybutyryl-CoA dehydratase (crotonase); Bcd, butyryl-CoA dehydrogenase; Ptb, phosphate butyryltransferase; Buk, butyrate kinase; Rnf, Rnf-type ion-translocating electron transport complex; Etf, electron transfer flavoprotein complex; PccAB, propionyl-CoA carboxylase; MutAB, methylmalonyl-CoA mutase; 2Pi, diphosphate; and Fdox/Fdred, ferredoxin, oxidized and reduced forms respectively.</p></div></div><div content-id="DSQ005F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq005/2/m_dsq00503.jpeg?Expires=2147483647&Signature=CCuFngyRq6~euS4bmstlYeeCAcPnd64ixF~GQq7XOxS4YOS6uurXoxEVyE~dxVZIdcjQoBtLA~NagfPsr7L7N8XYQLEk9ZONPxByOcSCqweDr5NE56SZreRNrd4bImQDAFiS80Crr44UYidhYrVSxBY21OpI~g9r-Gd-tkguSJhf6ziBAPfLHtM2jSQKwcWB1JsTK1N6kk~RtZFyEbLC9a4ykZl8INtVwBoUTPHgCksLQjsTS8HiL2wF9ZEivXibY2-529drf93y3seLelc2AgJNWDW8k5mwmedmHypLLeFnoDoUKL9ybPiD32mZPXmghvAdEk8DrQBeH8~x08ByvA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Central metabolism based on potential growth substrates and metabolic capacities reconstructed from the D. desulfuricans genome. This figure displays the flow of carbon in the metabolism of various organic acids (acetate, propionate, butyrate, lactate, and glycerol) predicted from the genome information of D. desulfuricans SSM1. The reversible and irreversible reactions catalysed by enzymes are indicated with both and single arrowhead, respectively. POR, pyruvate ferredoxin oxidoreductase; PFL, pyruvate formate-lyase; Pyc, pyruvate carboxylase; Pck, phosphoenolpyruvate carboxykinase; PykA, pyruvate kinase; Ppd, pyruvate phosphate dikinase; MaeB, malate dehydrogenase (oxaloacetate-decarboxylating); Mdh, malate dehydrogenase; SucCD, succinyl-CoA synthase; Sdh, succinate dehydrogenase; OOR, 2-oxogultarate ferredoxin oxidoreductase; Fba, fructose-bisphosphatase; Pfk, 6-phosphofructokinase; Acs, acetyl-CoA synthetase; Ack, acetate kinase; Pta, phosphate acetyltransferase; AOR, aldehyde ferredoxin oxidoreductase; GlpK, glycerol kinase; GlpD, glycerol-3-phosphate dehydrogenase; Thl, acetoacetyl-CoA thiolase; Hbd, 3-hydroxybutyryl-CoA dehydrogenase; Crt, 3-hydroxybutyryl-CoA dehydratase (crotonase); Bcd, butyryl-CoA dehydrogenase; Ptb, phosphate butyryltransferase; Buk, butyrate kinase; Rnf, Rnf-type ion-translocating electron transport complex; Etf, electron transfer flavoprotein complex; PccAB, propionyl-CoA carboxylase; MutAB, methylmalonyl-CoA mutase; 2Pi, diphosphate; and Fdox/Fdred, ferredoxin, oxidized and reduced forms respectively." path-from-xml="dsq00503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5027284/dsq00503.jpeg" path-from-xml="dsq00503.jpeg" target="_blank">View large</a><a section="5027284" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq005/2/dsq00503.jpeg?Expires=2147483647&Signature=UwGvEDyI6tdYnjXfr9aZQMzykYF2NKMKUrEw-h1kjyQCaYDNJrVx04fYfbTXahdL3b5XzOV~Nx6j-gwb4DqF2ogwTzVpyXoyiYFhY1lA2CTnyCgRlt2HwU4ZLmGdBnXjefSyNuKb80EbNNKYyL3Mjg4nZJnju2uPCWeUFViJYLSv4MNAfR25NXUKpVHsODUHOMAozy~kJkGHIEPQVfVKoawziEwsPx-pB1CYctSKn-kVP7i-y3qkHtk9x-fx8eXnkcIwF8tVixKFlBUENN1Hy9yMZGGdZf6gpp7ImWyvsT3~UYPqmmJuUl3bo7SMv7aZf1GuiR7Xsevxw4NxAUkwLA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5027284&ar=353305&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Central metabolism based on potential growth substrates and metabolic capacities reconstructed from the <em>D. desulfuricans</em> genome. This figure displays the flow of carbon in the metabolism of various organic acids (acetate, propionate, butyrate, lactate, and glycerol) predicted from the genome information of <em>D. desulfuricans</em> SSM1. The reversible and irreversible reactions catalysed by enzymes are indicated with both and single arrowhead, respectively. POR, pyruvate ferredoxin oxidoreductase; PFL, pyruvate formate-lyase; Pyc, pyruvate carboxylase; Pck, phosphoenolpyruvate carboxykinase; PykA, pyruvate kinase; Ppd, pyruvate phosphate dikinase; MaeB, malate dehydrogenase (oxaloacetate-decarboxylating); Mdh, malate dehydrogenase; SucCD, succinyl-CoA synthase; Sdh, succinate dehydrogenase; OOR, 2-oxogultarate ferredoxin oxidoreductase; Fba, fructose-bisphosphatase; Pfk, 6-phosphofructokinase; Acs, acetyl-CoA synthetase; Ack, acetate kinase; Pta, phosphate acetyltransferase; AOR, aldehyde ferredoxin oxidoreductase; GlpK, glycerol kinase; GlpD, glycerol-3-phosphate dehydrogenase; Thl, acetoacetyl-CoA thiolase; Hbd, 3-hydroxybutyryl-CoA dehydrogenase; Crt, 3-hydroxybutyryl-CoA dehydratase (crotonase); Bcd, butyryl-CoA dehydrogenase; Ptb, phosphate butyryltransferase; Buk, butyrate kinase; Rnf, Rnf-type ion-translocating electron transport complex; Etf, electron transfer flavoprotein complex; PccAB, propionyl-CoA carboxylase; MutAB, methylmalonyl-CoA mutase; 2Pi, diphosphate; and Fdox/Fdred, ferredoxin, oxidized and reduced forms respectively.</p></div></div><p><em>Deferribacter desulfuricans</em> is capable of growing using formate as its sole source of carbon and energy. According to the reconstructed metabolic pathways from the <em>D. desulfuricans</em> genome, formate is oxidized to CO<sub>2</sub> by membrane-bound formate dehydrogenase (Fdh: DEFDS_1329-31), which is energetically coupled with the respiratory nitrate or S<sup>0</sup> reduction. Methanogens and homo-acetogens are known to assimilate formate to acetyl-CoA via the Wood–Ljungdahl pathway (reductive acetyl-CoA pathway).<sup><a reveal-id="DSQ005C34" data-open="DSQ005C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Since the reductive acetyl-CoA pathway is absent in the <em>D. desulfuricans</em> genome, formate must be assimilated by another pathway. A potential reaction for formate assimilation is the direct or indirect conversion of formate and acetyl-CoA to pyruvate, the reverse reaction of pyruvate-formate lyase or the POR-catalysing carboxylation of CO<sub>2</sub> coupled with the formate oxidation by the Fdh protein. The pyruvate can then be converted to various biosynthetic intermediates by the TCA cycle and anaplerotic pathways (Fig. <a reveal-id="DSQ005F3" data-open="DSQ005F3" class="link link-reveal link-table xref-fig">3</a>).</p><p>The <em>D. desulfuricans</em> genome has a variety of anaplerotic pathways, which include the three enzymes, malic enzyme (MaeB: DEFDS_1074), pyruvate carboxylase (Pyc: DEFDS_1275), and phosphoenolpyruvate carboxykinase (PckA: DEFDS_1461). These pathways replenish the intermediates of the TCA cycle for gluconeogenesis and amino acid biosynthesis, and their anaplerotic CO<sub>2</sub> fixation may be an important function for <em>D. desulfuricans</em> living heterotrophically in a limited organic carbon such as hydrothermal vent environment. In the gluconeogenesis pathway, pyruvate may be dominantly produced by POR from acetyl-CoA and further converted to phosphoenolpyruvate by pyruvate phosphate dikinase (PpdK: DEFDS_0235).</p><p>The genome possesses genes for the Embden–Meyerhof–Parnas (EMP) pathway and the non-oxidative branch of the pentose phosphate pathway (Fig. <a reveal-id="DSQ005F3" data-open="DSQ005F3" class="link link-reveal link-table xref-fig">3</a>). The EMP pathway includes fructose-1,6-bisphosphatase (DEFDS_1408), which is a key enzyme in gluconeogenesis. The TCA cycle of <em>D. desulfuricans</em> includes 2-oxoglutarate ferredoxin oxidoreductases (OOR: DEFDS_0922-25 and DEFDS_0804-5) instead of 2-oxoglutarate dehydrogenases that are typically found in aerobic bacteria. These enzymes catalyse the reversible oxidative decarboxylation of 2-oxoglutarate to form succinyl-CoA, while they are also key enzymes of the reductive TCA (rTCA) cycle. Therefore, the TCA cycle has the potential to proceed in the reverse direction. Although no ATP-dependent citrate lyase (AclAB) is present, the presence of other key enzymes of the rTCA cycle suggests that the rTCA cycle could be partially operative in <em>D. desulfuricans</em>. Actually, oxaloacetate is produced from pyruvate and phosphoenolpyruvate by the anaplerotic enzymes, Pyc and PckA, respectively, and then oxaloacetate is converted to 2-oxoglutarate by operating the TCA cycle in the reductive direction. Among the previously characterized members of the genus <em>Deferribacter</em>, <em>D. abyssi</em> and <em>D. autotrophicus</em> grow autotrophically.<sup><a reveal-id="DSQ005C3" data-open="DSQ005C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSQ005C5" data-open="DSQ005C5" class="link link-ref link-reveal xref-bibr">5</a></sup> We have also isolated several strictly autotrophic strains of <em>D. desulfuricans</em> from different deep-sea hydrothermal environments (data not shown). Thus, the autotrophy may not be an unusual feature within the genus <em>Deferribacter</em>. The lack of an autotrophic phenotype in the strain SMM1 is presumably caused by the lack of the <em>aclAB</em> genes.</p><p>Genome analysis suggested the capability of butyrate-fermentation in <em>D. desulfuricans</em>, although it has not been confirmed experimentally. In this pathway, as shown in Fig. <a reveal-id="DSQ005F3" data-open="DSQ005F3" class="link link-reveal link-table xref-fig">3</a>, butyryl-CoA is converted from acetyl-CoA by the four enzymes—acetoacetyl-CoA thiolase (Thl; DEFDS_1837), 3-hydroxybutyryl-CoA dehydrogenase (Hbd; DEFDS_1836), 3-hydroxybutyryl-CoA dehydratase (Crt; DEFDS_1835), and butyryl-CoA dehydrogenase (Bcd; DEFDS_1834)—and the electron transfer flavoprotein complex (Etf; DEFDS_1831-1832). The order of this gene cluster is conserved in the genomes of <em>Geobacter metallireducens</em>, <em>Geobacter uraniireducens</em>, and a butyrate-producing clostridial species, <em>Butyrivibrio fibrisolvens</em>.<sup><a reveal-id="DSQ005C35" data-open="DSQ005C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Butyryl-CoA is converted to butyrate by phosphate butyryltransferase (DEFDS_1240) and butyrate kinase (DEFDS_1241) resulting in the generation of ATP by the substrate-level phosphorylation. This process can be reversible under a certain condition<sup><a reveal-id="DSQ005C36" data-open="DSQ005C36" class="link link-ref link-reveal xref-bibr">36</a></sup> and may thus serve for the butyrate utilization.</p><p>As shown in Fig. <a reveal-id="DSQ005F3" data-open="DSQ005F3" class="link link-reveal link-table xref-fig">3</a>, the <em>D. desulfuricans</em> genome also contains a set of six genes (<em>rnfCDGEAB</em>, DEFDS_0487-92) related to the potential membrane-bound electron transport complex (Rnf) recently found in various bacteria and archaea.<sup><a reveal-id="DSQ005C37  DSQ005C39" data-open="DSQ005C37  DSQ005C39" class="link link-ref link-reveal xref-bibr">37–39</a></sup> In nitrogen-fixing bacteria, the Rnf complex transports electrons from NADH to ferredoxin, which donates electrons to nitrogenase.<sup><a reveal-id="DSQ005C40" data-open="DSQ005C40" class="link link-ref link-reveal xref-bibr">40</a></sup> It is further proposed that the same enzyme apparently runs in the reverse direction: electron transfer from reduced ferredoxin to NAD<sup>+</sup> driving the electrogenic pumping of Na<sup>+</sup> out of the cell.<sup><a reveal-id="DSQ005C39" data-open="DSQ005C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Regional synteny for the <em>rnf</em> genes in the <em>D. desulfuricans</em> genome was found in several clostridial genomes, such as <em>Clostridium tetani</em>, <em>Clostridium kluyveri</em>, and <em>Halothermothrix orenii</em>. It is supposed that the Rnf complex is involved in the regeneration of NADH for the fermentation of butyrate.<sup><a reveal-id="DSQ005C41" data-open="DSQ005C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Since the <em>D. desulfuricans</em> genome indicated the possible fermentation of butyrate and the requirement of reduced ferredoxin for the anabolic reactions, the Rnf complex may be operative in either the forward or reverse flow of electrons, or both. In addition, it is suggested that glycerol can be utilized by glycerol kinase (GlpK, DEFDS_1230), <em>sn</em>-glycerol-3-phosphate dehydrogenase (GlpD, DEFDS_1577), and triosephosphate isomerase (DEFDS_0129). However, no growth was observed when glycerol was used as the sole source of energy and carbon,<sup><a reveal-id="DSQ005C2" data-open="DSQ005C2" class="link link-ref link-reveal xref-bibr">2</a></sup> and this phenomenon is presumably explained by the lack of a glycerol-uptake-facilitator protein.</p><p>Supporting the heterotrophic ability of <em>D. desulfuricans</em>, a variety of transporters for mono-, di-, and tricarboxylates are encoded within the genome. Actually, the genes encoding two sodium:solute symporter (SSS) family proteins (DEFDS_1547 and DEFDS_1574) and a YaaH family protein, which are presumably involved in acetate uptake, were identified in the genome. In addition, the genes encoding five transporter systems of tripartite ATP-independent periplasmic (TRAP) family (DEFDS_0682-4, DEFDS_0770-2, DEFDS_1399-1400, DEFDS_1846-7, and DEFDS_2070-1) may be involved in uptake of various dicarboxylates such as fumarate, malate, and succinate. In terms of tricarboxylate transporters, the genes for two citrate transporters (DEFDS_1180 and DEFDS_2162) were identified in the genome. On the other hand, the genome encodes two phosphotransferase system sugar transporters for fructose and mannose, although <em>D. desulfuricans</em> could not grow by using sugar.</p>                            <h3 scrollto-destination=5027291 id="5027291" class="section-title" data-legacy-id=s3d>3.4. Respiration</h3>
<p>Since <em>D. desulfuricans</em> SSM1 is capable of using molecular hydrogen (H<sub>2</sub>) as an energy source,<sup><a reveal-id="DSQ005C2" data-open="DSQ005C2" class="link link-ref link-reveal xref-bibr">2</a></sup> a membrane-bound NiFe-hydrogenase (Hyd) (DEFDS_0075-77) would function as an H<sub>2</sub>-uptake hydrogenase. As shown in Fig. <a reveal-id="DSQ005F4" data-open="DSQ005F4" class="link link-reveal link-table xref-fig">4</a>, the oxidation of molecular hydrogen is coupled to the reduction of menaquinone. The genome has two distinctive nitrate-reducing enzymes for respiration, which are coupled to the electron transfer chains (Fig. <a reveal-id="DSQ005F4" data-open="DSQ005F4" class="link link-reveal link-table xref-fig">4</a>). One is a membrane-bound (Nar: DEFDS_2086-89) and the other is a periplasmic (Nap: DEFDS_1819-23) nitrate reductase, which does not directly contribute to the generation of a proton motive force but contributes to redox balancing.<sup><a reveal-id="DSQ005C42" data-open="DSQ005C42" class="link link-ref link-reveal xref-bibr">42</a></sup> It is generally known that bacteria have Nar-type nitrate reductases, but some bacterial species have a Nap-type or both types of nitrate reductase. Nap-type nitrate reductase usually functions as a two-subunit enzyme comprising a catalytic subunit (NapA) that binds a bis-MGD cofactor and a [4Fe-4S] cluster, and an electron transfer subunit (NapB) that binds two <em>c</em>-type haems. However, since <em>D. desulfuricans</em> does not possess the <em>napB</em> gene, the nitrate reductase of <em>D. desulfuricans</em> may be monomeric, which is differentiated from the heterodimeric NapAB structure typically found in bacteria. Actually, NapB-independent periplasmic nitrate reductases have been reported for some species in <em>Deltaproteobacteria</em> and <em>Clostridia</em>.<sup><a reveal-id="DSQ005C43" data-open="DSQ005C43" class="link link-ref link-reveal xref-bibr">43</a></sup> The organization of Nap genes in the <em>D. desulfuricans</em> genome is similar to that of <em>Desulfovibrio desulfuricans</em>. Although nitrate is sequentially reduced to N<sub>2</sub> (denitrification) or NH<sub>4</sub> (ammonification) in many cases, no enzyme for nitrite reduction was identified in the <em>D. desulfuricans</em> genome. This is consistent with experimental results, because the accumulation of nitrite is observed during culture of this organism.<sup><a reveal-id="DSQ005C2" data-open="DSQ005C2" class="link link-ref link-reveal xref-bibr">2</a></sup></p>                        <a id="5027293" scrollto-destination="5027293"></a>
<div id="DSQ005F4" data-id="DSQ005F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq005/2/m_dsq00504.jpeg?Expires=2147483647&Signature=IugBFptDorx~IT~6QBh0azlUX6f3gVcZE9f0MrzNoBBrTUvLcJ9qKlTHief~eEwA0uDa~8aaO6Av1qjE1Ky~9ow8bpGpgRjU~oCufaAxn7b9IFiy9y4C3KL4fwC63Wb-oxaxHG60-MY1u7biJZesOApLZo3IA5UiSXHPIQSUbZNm4fmjVI2sw~Zo9UQC07AkcqtZosrG4gwgN3YeG56Ym0ZTVSa3hGYOoSX0OejVoHSi-Q5lu2iH1zf~EgxkTKzgfsuwRXOucFfzyw-zeHRH5i9--VjamtLOFEwcx9yJwrER8TyOZLshgy9g5t6sHkOUkYdRCzZ9SWWD16YmA0hrIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome-based models for the energy-conserving electron-transport pathways of D. desulfuricans SSM1. Reducing power acquired by catabolic metabolism (NADH, succinate, and sn-glycerol-3-phosphate) or by oxidation of hydrogen and/or formate is used to reduce sulphur compounds and nitrate, and potentially iron ions via the quinol pool for energy conservation or dissipation. Membrane-binding components are indicated with cylinders or cones, where the upper and the lower reactions are catalysed on the periplasmic side and the cytoplasmic side, respectively. Periplasmic components are indicated with ellipsoids or spheres. Hyd, membrane-binding NiFe-hydrogenase; Nar, respiratory membrane-bound nitrate reductase; Nap, periplasmic nitrate reductase; Psr, polysulphide reductase; Phs, thiosulphate reductase; Ttr, tetrathionate reductase; Fdh, formate dehydrogenase; Nuo, proton-pumping NADH dehydrogenase; Nqr, sodium-translocating NADH:quinone oxidoreductase; Sdh, succinate dehydrogenase; Cyt bd, cytochrome bd quinol oxidase; Cyt bc, cytochrome bc complex; Glp, sn-glycerol-3-phosphate dehydrogenase; G3P, sn-glycerol-3-phosphate; and DHAP, dihydroxyacetone phosphate." path-from-xml="dsq00504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5027293/dsq00504.jpeg" path-from-xml="dsq00504.jpeg" target="_blank">View large</a><a section="5027293" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq005/2/dsq00504.jpeg?Expires=2147483647&Signature=PbLepyPDaxnkf-3iSiMBQbOgeOKmL7bKquOOWaXoO4~Uiw-Oat6pJNf-6gDeTG6e0gbXrnw1pCBLb19o7Yzp6-WTzwpUW1ANLLrjDwVK3N72ES6odicz2tN-21oyyzllYfBtEHuFKeYWo8eOkrdSN86I8K-sCW8k1dhwjwyIb1nr0JFB3ngVJ1oHIncjL2lSyf1Kc~~Z-ub7VjC3ohvNCeyrK7uELZnVfcliI0LczozE8wYkSpEzojgQ1IsuYF~ZUk7RoISrk6kTGMKNLOiUYJz5fZwIwB3Zw4jNLR7TjS9QWvkz2GE15Dg0V6oKYunoAOvsGbVhYGaTDiaEo9FpWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5027293&ar=353305&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome-based models for the energy-conserving electron-transport pathways of <em>D. desulfuricans</em> SSM1. Reducing power acquired by catabolic metabolism (NADH, succinate, and <em>sn</em>-glycerol-3-phosphate) or by oxidation of hydrogen and/or formate is used to reduce sulphur compounds and nitrate, and potentially iron ions via the quinol pool for energy conservation or dissipation. Membrane-binding components are indicated with cylinders or cones, where the upper and the lower reactions are catalysed on the periplasmic side and the cytoplasmic side, respectively. Periplasmic components are indicated with ellipsoids or spheres. Hyd, membrane-binding NiFe-hydrogenase; Nar, respiratory membrane-bound nitrate reductase; Nap, periplasmic nitrate reductase; Psr, polysulphide reductase; Phs, thiosulphate reductase; Ttr, tetrathionate reductase; Fdh, formate dehydrogenase; Nuo, proton-pumping NADH dehydrogenase; Nqr, sodium-translocating NADH:quinone oxidoreductase; Sdh, succinate dehydrogenase; Cyt bd, cytochrome <em>bd</em> quinol oxidase; Cyt bc, cytochrome <em>bc</em> complex; Glp, <em>sn</em>-glycerol-3-phosphate dehydrogenase; G3P, <em>sn</em>-glycerol-3-phosphate; and DHAP, dihydroxyacetone phosphate.</p></div></div><div content-id="DSQ005F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq005/2/m_dsq00504.jpeg?Expires=2147483647&Signature=IugBFptDorx~IT~6QBh0azlUX6f3gVcZE9f0MrzNoBBrTUvLcJ9qKlTHief~eEwA0uDa~8aaO6Av1qjE1Ky~9ow8bpGpgRjU~oCufaAxn7b9IFiy9y4C3KL4fwC63Wb-oxaxHG60-MY1u7biJZesOApLZo3IA5UiSXHPIQSUbZNm4fmjVI2sw~Zo9UQC07AkcqtZosrG4gwgN3YeG56Ym0ZTVSa3hGYOoSX0OejVoHSi-Q5lu2iH1zf~EgxkTKzgfsuwRXOucFfzyw-zeHRH5i9--VjamtLOFEwcx9yJwrER8TyOZLshgy9g5t6sHkOUkYdRCzZ9SWWD16YmA0hrIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome-based models for the energy-conserving electron-transport pathways of D. desulfuricans SSM1. Reducing power acquired by catabolic metabolism (NADH, succinate, and sn-glycerol-3-phosphate) or by oxidation of hydrogen and/or formate is used to reduce sulphur compounds and nitrate, and potentially iron ions via the quinol pool for energy conservation or dissipation. Membrane-binding components are indicated with cylinders or cones, where the upper and the lower reactions are catalysed on the periplasmic side and the cytoplasmic side, respectively. Periplasmic components are indicated with ellipsoids or spheres. Hyd, membrane-binding NiFe-hydrogenase; Nar, respiratory membrane-bound nitrate reductase; Nap, periplasmic nitrate reductase; Psr, polysulphide reductase; Phs, thiosulphate reductase; Ttr, tetrathionate reductase; Fdh, formate dehydrogenase; Nuo, proton-pumping NADH dehydrogenase; Nqr, sodium-translocating NADH:quinone oxidoreductase; Sdh, succinate dehydrogenase; Cyt bd, cytochrome bd quinol oxidase; Cyt bc, cytochrome bc complex; Glp, sn-glycerol-3-phosphate dehydrogenase; G3P, sn-glycerol-3-phosphate; and DHAP, dihydroxyacetone phosphate." path-from-xml="dsq00504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5027293/dsq00504.jpeg" path-from-xml="dsq00504.jpeg" target="_blank">View large</a><a section="5027293" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq005/2/dsq00504.jpeg?Expires=2147483647&Signature=PbLepyPDaxnkf-3iSiMBQbOgeOKmL7bKquOOWaXoO4~Uiw-Oat6pJNf-6gDeTG6e0gbXrnw1pCBLb19o7Yzp6-WTzwpUW1ANLLrjDwVK3N72ES6odicz2tN-21oyyzllYfBtEHuFKeYWo8eOkrdSN86I8K-sCW8k1dhwjwyIb1nr0JFB3ngVJ1oHIncjL2lSyf1Kc~~Z-ub7VjC3ohvNCeyrK7uELZnVfcliI0LczozE8wYkSpEzojgQ1IsuYF~ZUk7RoISrk6kTGMKNLOiUYJz5fZwIwB3Zw4jNLR7TjS9QWvkz2GE15Dg0V6oKYunoAOvsGbVhYGaTDiaEo9FpWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5027293&ar=353305&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome-based models for the energy-conserving electron-transport pathways of <em>D. desulfuricans</em> SSM1. Reducing power acquired by catabolic metabolism (NADH, succinate, and <em>sn</em>-glycerol-3-phosphate) or by oxidation of hydrogen and/or formate is used to reduce sulphur compounds and nitrate, and potentially iron ions via the quinol pool for energy conservation or dissipation. Membrane-binding components are indicated with cylinders or cones, where the upper and the lower reactions are catalysed on the periplasmic side and the cytoplasmic side, respectively. Periplasmic components are indicated with ellipsoids or spheres. Hyd, membrane-binding NiFe-hydrogenase; Nar, respiratory membrane-bound nitrate reductase; Nap, periplasmic nitrate reductase; Psr, polysulphide reductase; Phs, thiosulphate reductase; Ttr, tetrathionate reductase; Fdh, formate dehydrogenase; Nuo, proton-pumping NADH dehydrogenase; Nqr, sodium-translocating NADH:quinone oxidoreductase; Sdh, succinate dehydrogenase; Cyt bd, cytochrome <em>bd</em> quinol oxidase; Cyt bc, cytochrome <em>bc</em> complex; Glp, <em>sn</em>-glycerol-3-phosphate dehydrogenase; G3P, <em>sn</em>-glycerol-3-phosphate; and DHAP, dihydroxyacetone phosphate.</p></div></div><p>It has been reported that the reduction of energy-yielding sulphur compounds (sulphur-respiration) is catalysed by some key enzymes: a polysulphide reductase (Psr) in <em>Wolinella succinogenes</em> (<em>Epsilonproteobacteria</em>),<sup><a reveal-id="DSQ005C44" data-open="DSQ005C44" class="link link-ref link-reveal xref-bibr">44</a></sup> a thiosulphate reductase (Phs) in <em>Salmonella enterica</em> (<em>Gammaproteobacteria</em>),<sup><a reveal-id="DSQ005C45" data-open="DSQ005C45" class="link link-ref link-reveal xref-bibr">45</a></sup> and a hydrogenase-sulphur reductase multienzyme (Sre) in <em>Acidianus ambivalens</em> (<em>Crenarchaeota</em>) or <em>Aquifex aerolicus</em> (<em>Aquificae</em>).<sup><a reveal-id="DSQ005C46" data-open="DSQ005C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DSQ005C47" data-open="DSQ005C47" class="link link-ref link-reveal xref-bibr">47</a></sup> Psr and Phs reduce soluble polysulphide derived from S<sup>0</sup> to sulphide, whereas Sre is considered to reduce S<sup>0</sup> directly to sulphide. Genome analysis revealed that <em>D. desulfuricans</em> has three gene clusters encoding Psr/Phs enzymes, DEFDS_0670-72, DEFDS_1691-93, and DEFDS_1697-99. The genes in each cluster encode three subunits that construct the following membrane-bound complex enzyme: a catalytic subunit with molybdopterin (PsrA/PhsA), an electron transfer subunit with the [Fe–S] cluster (PsrB/PhsB), and a membrane anchor subunit (PsrC/PhsC). The amino acid sequences of the PsrA/PhsA subunits contain the motif necessary for translocation of the protein towards the periplasmic space via the twin arginine-translocation systems, suggesting that the polysulphide reduction should occur in the periplasmic space. Moreover, an alternative molybdopterin-containing enzyme, tetrathionate reductase (Ttr), is encoded by DEFDS_1446-47, but it is unclear whether this enzyme is involved in energy conservation because of the absence of a membrane anchor subunit. <em>Deferribacter desulfuricans</em> SSM1 presumably uses the polysulphides chemically formed from the reaction of S<sup>0</sup> and sulphide, which readily occurs in hydrothermal vents,<sup><a reveal-id="DSQ005C48" data-open="DSQ005C48" class="link link-ref link-reveal xref-bibr">48</a></sup> and therefore it needs to transfer polysulphide to the periplasmic space across the outer membrane (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1A</a>). The periplasmic Sud protein in <em>W. succinogenes</em> has been proposed to serve as a polysulphide-binding protein and to transfer polysulphide-sulphur to the active site of polysulphide reductase.<sup><a reveal-id="DSQ005C49" data-open="DSQ005C49" class="link link-ref link-reveal xref-bibr">49</a></sup> A Psr/Phs gene cluster (DEFDS_0670-72) in the <em>D. desulfuricans</em> genome is associated with the genes coding an outer membrane porin and lipoprotein (DEFDS_0667-68). The amino acid sequence of DEFDS_0667 represents the functional domain of porin_O_P (phosphate-selective porins O and P) that are conserved in anion-specific porins.<sup><a reveal-id="DSQ005C50" data-open="DSQ005C50" class="link link-ref link-reveal xref-bibr">50</a></sup> The lipoprotein DEFDS_0668 contains a rhodanese (thiosulphate sulphurtransferase) domain as well as the Sud protein, although there is no homology between the two proteins. Thus, these two proteins may facilitate sulphur respiration in <em>D. desulfuricans</em> as a polysulphide-specific porin in the outer membrane and a polysulphide-binding protein in the periplasmic space, respectively. This locus is adjacent to other genes related to respiration, two multihaem <em>c</em>-type cytochromes (DEFDS_0665-66) and a cytochrome <em>bc</em> complex (DEFDS_0674-77). Interestingly, a similar locus without the Psr/Phs genes was found in the genomes of the iron- and sulphur-reducing bacteria, <em>Geobacter</em> spp. and <em>Desulfuromonas acetoxidans</em> DSM 684 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1B</a>).</p><p>The <em>D. desulfuricans</em> genome has a gene cluster (DEFDS_0741-60) that includes many genes encoding membrane-bound or periplasmic multihaem <em>c</em>-type cytochromes, which are potentially distributed in the inner or outer membranes or in the periplasmic space (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2A</a>). As demonstrated in <em>G. sulfurreducens</em> and <em>Shewanella oneidensis</em>, multihaem <em>c</em>-type cytochromes may be involved in the potential dissimilatory reduction of metal ions.<sup><a reveal-id="DSQ005C51" data-open="DSQ005C51" class="link link-ref link-reveal xref-bibr">51</a></sup> Although <em>D. desulfuricans</em> SSM1 cannot grow by iron reduction, the capability of iron reduction is a symbolic physiological trait of the genus <em>Deferribacter</em>.<sup><a reveal-id="DSQ005C3  DSQ005C5" data-open="DSQ005C3  DSQ005C5" class="link link-ref link-reveal xref-bibr">3–5</a></sup> On the basis of cultivation-dependent estimations, the members of the genus <em>Deferribacter</em> have been predicted to be the most numerically abundant iron-reducing prokaryotes in deep-sea vents.<sup><a reveal-id="DSQ005C7" data-open="DSQ005C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSQ005C8" data-open="DSQ005C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSQ005C52" data-open="DSQ005C52" class="link link-ref link-reveal xref-bibr">52</a></sup> Therefore, the multihaem <em>c</em>-type cytochromes in <em>D. desulfuricans</em> may be a remnant of the metal reduction system.</p><p>As shown in Fig. <a reveal-id="DSQ005F4" data-open="DSQ005F4" class="link link-reveal link-table xref-fig">4</a>, reductants produced from the oxidation of organic compounds would be oxidized by a proton-pumping NADH dehydrogenase (Nuo: DEFDS_1972-85) and succinate dehydrogenase (Sdh: DEFDS_0685-88) to provide reducing equivalents for the nitrate- and polysulphide reductions. The <em>D. desulfuricans</em> genome has an alternative complex for NADH oxidation, sodium-translocating NADH:quinone oxidoreductase (Nqr: DEFDS_1913-18). This complex possibly allows the generation of a sodium gradient, which could energize an Na<sup>+</sup>-dependent symporter of the organic substrates. This organism found to possess many sodium gradient-dependent symporters such as two solute:sodium symporters (SSS family), a nucleobase:cation symporter-2 (NCS2) family protein (DEFDS_1105), an alanine or glycine:cation symporter (AGCS) family protein (DEFDS_1601), and phosphate:sodium symporter (PNaS) family protein (DEFDS_0081).</p>                            <h3 scrollto-destination=5027297 id="5027297" class="section-title" data-legacy-id=s3e>3.5. Signal transduction, motility, and chemotaxis</h3>
<p><em>Deferribacter desulfuricans</em> SSM1 and its relatives inhabit the ever-changing steep physical and chemical gradients of hydrothermal vent chimney. The thermophilic microorganisms living on hydrothermal vents are generally sensing temperature or chemical stimuli to guide their directional movements by swimming with a whip-like flagellum. Thus, the <em>D. desulfuricans</em> genome should have the molecular machinery for sensing and responding to environmental changes. Indeed, <em>D. desulfuricans</em> has genes for various signalling systems, as other vent chemolithotrophs do<sup><a reveal-id="DSQ005C9  DSQ005C11" data-open="DSQ005C9  DSQ005C11" class="link link-ref link-reveal xref-bibr">9–11</a></sup> (Table <a reveal-id="DSQ005TB2" data-open="DSQ005TB2" class="link link-reveal link-table xref-fig">2</a>). More than 70 genes for two-component signal transduction (TCS) systems, including 23 histidine kinases, 41 response regulator genes, and 13 hybrid histidine kinases with regulator domains, were found in the <em>D. desulfuricans</em> genome. The frequency of TCS systems found in the <em>D. desulfuricans</em> genome is the highest (34 genes per megabase) among vent bacteria (ranging from 11 to 22 genes per megabase). Although most of them could not be functionally assigned due to very low homologies with well-defined systems, some functions were assumed to be phosphate regulation (PhoBR) and ammonium assimilation (NtrBC). </p>                        <a id="5027299" scrollto-destination="5027299"></a>
<div content-id="DSQ005TB2" class="table-modal"><div class="table-wrap"><div id="DSQ005TB2" class="table-wrap-title" data-id="DSQ005TB2"><div class="label -label">Table 2</div><div class="caption"><p>Abundance of genetic components for signal transduction systems in the genomes of representative deep-sea vent chemolithotrophs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Signal transduction systems </th><th>Domains (Pfam) </th><th><em>Deferribacter desulfuricans</em> </th><th><em>Thiomicrospira crunogena</em> </th><th><em>Nautilia profundicola</em> </th><th><em>Nitratiruptor</em> sp. SB155-2 </th><th><em>Sulfurovum</em> sp. NBC37-1 </th><th><em>Persephonella marina</em> </th></tr></thead><tbody><tr><td>Genome size (bp) </td><td> </td><td>2 234 389 </td><td>2 427 734 </td><td>1 676 444 </td><td>1 877 931 </td><td>2 562 277 </td><td>1 930 284 </td></tr><tr><td colspan="8">Two-component signal transduction systems </td></tr><tr><td> Sensor histidine kinase </td><td>PF00512/PF02518 </td><td>23 </td><td>13 </td><td>11 </td><td>15 </td><td>20 </td><td>8 </td></tr><tr><td> Response regulator </td><td>PF00072 </td><td>41 </td><td>24 </td><td>16 </td><td>19 </td><td>28 </td><td>14 </td></tr><tr><td> Hybrid </td><td> </td><td>13 </td><td>7 </td><td>1 </td><td>1 </td><td>9 </td><td>0 </td></tr><tr><td> Total </td><td> </td><td>77 </td><td>44 </td><td>28 </td><td>35 </td><td>57 </td><td>22 </td></tr><tr><td colspan="8">Cyclic diGMP signalling systems </td></tr><tr><td> GGDEF </td><td>PF00990 </td><td>12 </td><td>20 </td><td>17 </td><td>11 </td><td>5 </td><td>12 </td></tr><tr><td> EAL </td><td>PF00563 </td><td>1 </td><td>5 </td><td>3 </td><td>2 </td><td>1 </td><td>4 </td></tr><tr><td> HD-GYP </td><td>PF01966 </td><td>3 </td><td>3 </td><td>2 </td><td>0 </td><td>0 </td><td>2 </td></tr><tr><td> GGDEF-EAL </td><td> </td><td>3 </td><td>16 </td><td>11 </td><td>19 </td><td>12 </td><td>12 </td></tr><tr><td> Total </td><td> </td><td>19 </td><td>44 </td><td>33 </td><td>32 </td><td>18 </td><td>30 </td></tr><tr><td colspan="8">Chemotaxis </td></tr><tr><td> Methyl-accepting chemotaxis proteins </td><td>PF00015 </td><td>14 </td><td>14 </td><td>11 </td><td>8 </td><td>0 </td><td>5 </td></tr><tr><td> Chemotaxis proteins </td><td> </td><td>32 </td><td>13 </td><td>3 </td><td>13 </td><td>0 </td><td>6 </td></tr><tr><td> Total </td><td> </td><td>46 </td><td>27 </td><td>14 </td><td>21 </td><td>0 </td><td>11 </td></tr></tbody></table></div><div content-id="DSQ005TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Signal transduction systems </th><th>Domains (Pfam) </th><th><em>Deferribacter desulfuricans</em> </th><th><em>Thiomicrospira crunogena</em> </th><th><em>Nautilia profundicola</em> </th><th><em>Nitratiruptor</em> sp. SB155-2 </th><th><em>Sulfurovum</em> sp. NBC37-1 </th><th><em>Persephonella marina</em> </th></tr></thead><tbody><tr><td>Genome size (bp) </td><td> </td><td>2 234 389 </td><td>2 427 734 </td><td>1 676 444 </td><td>1 877 931 </td><td>2 562 277 </td><td>1 930 284 </td></tr><tr><td colspan="8">Two-component signal transduction systems </td></tr><tr><td> Sensor histidine kinase </td><td>PF00512/PF02518 </td><td>23 </td><td>13 </td><td>11 </td><td>15 </td><td>20 </td><td>8 </td></tr><tr><td> Response regulator </td><td>PF00072 </td><td>41 </td><td>24 </td><td>16 </td><td>19 </td><td>28 </td><td>14 </td></tr><tr><td> Hybrid </td><td> </td><td>13 </td><td>7 </td><td>1 </td><td>1 </td><td>9 </td><td>0 </td></tr><tr><td> Total </td><td> </td><td>77 </td><td>44 </td><td>28 </td><td>35 </td><td>57 </td><td>22 </td></tr><tr><td colspan="8">Cyclic diGMP signalling systems </td></tr><tr><td> GGDEF </td><td>PF00990 </td><td>12 </td><td>20 </td><td>17 </td><td>11 </td><td>5 </td><td>12 </td></tr><tr><td> EAL </td><td>PF00563 </td><td>1 </td><td>5 </td><td>3 </td><td>2 </td><td>1 </td><td>4 </td></tr><tr><td> HD-GYP </td><td>PF01966 </td><td>3 </td><td>3 </td><td>2 </td><td>0 </td><td>0 </td><td>2 </td></tr><tr><td> GGDEF-EAL </td><td> </td><td>3 </td><td>16 </td><td>11 </td><td>19 </td><td>12 </td><td>12 </td></tr><tr><td> Total </td><td> </td><td>19 </td><td>44 </td><td>33 </td><td>32 </td><td>18 </td><td>30 </td></tr><tr><td colspan="8">Chemotaxis </td></tr><tr><td> Methyl-accepting chemotaxis proteins </td><td>PF00015 </td><td>14 </td><td>14 </td><td>11 </td><td>8 </td><td>0 </td><td>5 </td></tr><tr><td> Chemotaxis proteins </td><td> </td><td>32 </td><td>13 </td><td>3 </td><td>13 </td><td>0 </td><td>6 </td></tr><tr><td> Total </td><td> </td><td>46 </td><td>27 </td><td>14 </td><td>21 </td><td>0 </td><td>11 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Signal transduction proteins were identified by querying the predicted gene products to the Pfam and COG databases. Genes with significant matches (<em>E</em>-value of less than 1E−5) were assigned a product description and classified using a set of rules based on the domain architecture of the protein. The final results were manually verified.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5027299" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ005TB2" class="table-wrap-title" data-id="DSQ005TB2"><div class="label -label">Table 2</div><div class="caption"><p>Abundance of genetic components for signal transduction systems in the genomes of representative deep-sea vent chemolithotrophs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Signal transduction systems </th><th>Domains (Pfam) </th><th><em>Deferribacter desulfuricans</em> </th><th><em>Thiomicrospira crunogena</em> </th><th><em>Nautilia profundicola</em> </th><th><em>Nitratiruptor</em> sp. SB155-2 </th><th><em>Sulfurovum</em> sp. NBC37-1 </th><th><em>Persephonella marina</em> </th></tr></thead><tbody><tr><td>Genome size (bp) </td><td> </td><td>2 234 389 </td><td>2 427 734 </td><td>1 676 444 </td><td>1 877 931 </td><td>2 562 277 </td><td>1 930 284 </td></tr><tr><td colspan="8">Two-component signal transduction systems </td></tr><tr><td> Sensor histidine kinase </td><td>PF00512/PF02518 </td><td>23 </td><td>13 </td><td>11 </td><td>15 </td><td>20 </td><td>8 </td></tr><tr><td> Response regulator </td><td>PF00072 </td><td>41 </td><td>24 </td><td>16 </td><td>19 </td><td>28 </td><td>14 </td></tr><tr><td> Hybrid </td><td> </td><td>13 </td><td>7 </td><td>1 </td><td>1 </td><td>9 </td><td>0 </td></tr><tr><td> Total </td><td> </td><td>77 </td><td>44 </td><td>28 </td><td>35 </td><td>57 </td><td>22 </td></tr><tr><td colspan="8">Cyclic diGMP signalling systems </td></tr><tr><td> GGDEF </td><td>PF00990 </td><td>12 </td><td>20 </td><td>17 </td><td>11 </td><td>5 </td><td>12 </td></tr><tr><td> EAL </td><td>PF00563 </td><td>1 </td><td>5 </td><td>3 </td><td>2 </td><td>1 </td><td>4 </td></tr><tr><td> HD-GYP </td><td>PF01966 </td><td>3 </td><td>3 </td><td>2 </td><td>0 </td><td>0 </td><td>2 </td></tr><tr><td> GGDEF-EAL </td><td> </td><td>3 </td><td>16 </td><td>11 </td><td>19 </td><td>12 </td><td>12 </td></tr><tr><td> Total </td><td> </td><td>19 </td><td>44 </td><td>33 </td><td>32 </td><td>18 </td><td>30 </td></tr><tr><td colspan="8">Chemotaxis </td></tr><tr><td> Methyl-accepting chemotaxis proteins </td><td>PF00015 </td><td>14 </td><td>14 </td><td>11 </td><td>8 </td><td>0 </td><td>5 </td></tr><tr><td> Chemotaxis proteins </td><td> </td><td>32 </td><td>13 </td><td>3 </td><td>13 </td><td>0 </td><td>6 </td></tr><tr><td> Total </td><td> </td><td>46 </td><td>27 </td><td>14 </td><td>21 </td><td>0 </td><td>11 </td></tr></tbody></table></div><div content-id="DSQ005TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Signal transduction systems </th><th>Domains (Pfam) </th><th><em>Deferribacter desulfuricans</em> </th><th><em>Thiomicrospira crunogena</em> </th><th><em>Nautilia profundicola</em> </th><th><em>Nitratiruptor</em> sp. SB155-2 </th><th><em>Sulfurovum</em> sp. NBC37-1 </th><th><em>Persephonella marina</em> </th></tr></thead><tbody><tr><td>Genome size (bp) </td><td> </td><td>2 234 389 </td><td>2 427 734 </td><td>1 676 444 </td><td>1 877 931 </td><td>2 562 277 </td><td>1 930 284 </td></tr><tr><td colspan="8">Two-component signal transduction systems </td></tr><tr><td> Sensor histidine kinase </td><td>PF00512/PF02518 </td><td>23 </td><td>13 </td><td>11 </td><td>15 </td><td>20 </td><td>8 </td></tr><tr><td> Response regulator </td><td>PF00072 </td><td>41 </td><td>24 </td><td>16 </td><td>19 </td><td>28 </td><td>14 </td></tr><tr><td> Hybrid </td><td> </td><td>13 </td><td>7 </td><td>1 </td><td>1 </td><td>9 </td><td>0 </td></tr><tr><td> Total </td><td> </td><td>77 </td><td>44 </td><td>28 </td><td>35 </td><td>57 </td><td>22 </td></tr><tr><td colspan="8">Cyclic diGMP signalling systems </td></tr><tr><td> GGDEF </td><td>PF00990 </td><td>12 </td><td>20 </td><td>17 </td><td>11 </td><td>5 </td><td>12 </td></tr><tr><td> EAL </td><td>PF00563 </td><td>1 </td><td>5 </td><td>3 </td><td>2 </td><td>1 </td><td>4 </td></tr><tr><td> HD-GYP </td><td>PF01966 </td><td>3 </td><td>3 </td><td>2 </td><td>0 </td><td>0 </td><td>2 </td></tr><tr><td> GGDEF-EAL </td><td> </td><td>3 </td><td>16 </td><td>11 </td><td>19 </td><td>12 </td><td>12 </td></tr><tr><td> Total </td><td> </td><td>19 </td><td>44 </td><td>33 </td><td>32 </td><td>18 </td><td>30 </td></tr><tr><td colspan="8">Chemotaxis </td></tr><tr><td> Methyl-accepting chemotaxis proteins </td><td>PF00015 </td><td>14 </td><td>14 </td><td>11 </td><td>8 </td><td>0 </td><td>5 </td></tr><tr><td> Chemotaxis proteins </td><td> </td><td>32 </td><td>13 </td><td>3 </td><td>13 </td><td>0 </td><td>6 </td></tr><tr><td> Total </td><td> </td><td>46 </td><td>27 </td><td>14 </td><td>21 </td><td>0 </td><td>11 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Signal transduction proteins were identified by querying the predicted gene products to the Pfam and COG databases. Genes with significant matches (<em>E</em>-value of less than 1E−5) were assigned a product description and classified using a set of rules based on the domain architecture of the protein. The final results were manually verified.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5027299" target="_blank">View Large</a></div></div><p>As an alternative signalling system, the <em>D. desulfuricans</em> genome has many genes relevant to a cyclic diguanylate (c-diGMP) signalling system,<sup><a reveal-id="DSQ005C53" data-open="DSQ005C53" class="link link-ref link-reveal xref-bibr">53</a></sup> although these were less abundant than in other deep-sea vent bacteria (Table <a reveal-id="DSQ005TB2" data-open="DSQ005TB2" class="link link-reveal link-table xref-fig">2</a>). This system is characterized by either diguanylate cyclase (GGDEF domain) or phosphodiesterase (EAL and HD-GYP domains) activity and is relevant to biofilm formation, motility, and virulence.<sup><a reveal-id="DSQ005C53" data-open="DSQ005C53" class="link link-ref link-reveal xref-bibr">53</a></sup> In addition, at least one PAS domain was found in seven histidine kinase proteins and two c-diGMP signalling proteins as additional input modules. PAS domains are known to sense changes in the redox conditions inside or outside cells.<sup><a reveal-id="DSQ005C54" data-open="DSQ005C54" class="link link-ref link-reveal xref-bibr">54</a></sup> It is worthy to note that the <em>D. desulfuricans</em> genome has a higher proportion of intracellular signalling systems than those of other deep-sea vent chemolithotrophs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). These sensors may be essential for the versatile energy acquisition, carbon metabolisms, and the chemotaxis and redoxtaxis of <em>D. desulfuricans</em> SSM1 that are necessary under highly variable environmental conditions.</p><p>This genome encodes a large number of methyl-accepting chemotaxis proteins (14 MCPs) as observed in other deep-sea vent bacteria (5–14 MCPs), except for the immotile <em>Sulfurovum</em> sp. NBC37-1 (Table <a reveal-id="DSQ005TB2" data-open="DSQ005TB2" class="link link-reveal link-table xref-fig">2</a>). These chemotaxis-specific receptors would, in part, contribute to the versatile sensing capabilities of <em>D. desulfuricans</em>. The MCP encoded by DEFDS_1855 possesses a Cache (Ca<sup>2+</sup> channels and chemotaxis receptors) domain that is involved in the binding of amino acids and carbohydrates in the MCP of <em>Bacillus subtilis</em>.<sup><a reveal-id="DSQ005C55" data-open="DSQ005C55" class="link link-ref link-reveal xref-bibr">55</a></sup> It is also notable that multiple <em>che</em> clusters (six clusters of <em>che-1</em> to <em>che-6</em>) were identified in the genome of <em>D. desulfuricans</em>, whereas most deep-sea vent microorganisms usually have one or two <em>che</em> gene clusters (Table <a reveal-id="DSQ005TB2" data-open="DSQ005TB2" class="link link-reveal link-table xref-fig">2</a> and Fig. <a reveal-id="DSQ005F5" data-open="DSQ005F5" class="link link-reveal link-table xref-fig">5</a>). The <em>che-2</em> cluster is associated with genes for flagellar biosynthesis, suggesting that flagellar formation and motility may be linked with the chemotaxis pathway. <em>Deferribacter desulfuricans</em> SSM1 has almost all of the genes necessary for the flagellar apparatus although motility has not yet been observed in this strain.<sup><a reveal-id="DSQ005C2" data-open="DSQ005C2" class="link link-ref link-reveal xref-bibr">2</a></sup> On the other hand, since motility has been confirmed in several strains of <em>D. desulfuricans</em> recently isolated by us, flagellar-based motility could be an intrinsic genetic potential in <em>D. desulfuricans</em>. Moreover, all <em>che</em> clusters, except for <em>che-4</em>, contain genes that encode a fusion protein (CheA/Y) consisting of histidine kinase (CheA) and response regulator (CheY) domains. In other bacteria, the gene clusters containing <em>cheA/Y</em> are involved in specific functions other than chemotaxis. For instances, the <em>Pseudomonas aeruginosa</em> Wsp cluster is involved in biofilm formation,<sup><a reveal-id="DSQ005C56" data-open="DSQ005C56" class="link link-ref link-reveal xref-bibr">56</a></sup> and the <em>Myxococcus xanthus</em> Frz cluster functionally controls type IV pili-based motility for cellular aggregation.<sup><a reveal-id="DSQ005C57" data-open="DSQ005C57" class="link link-ref link-reveal xref-bibr">57</a></sup> Also, the gene cluster 3 of <em>Rhodospirillum centenum</em> is involved in cyst cell development.<sup><a reveal-id="DSQ005C58" data-open="DSQ005C58" class="link link-ref link-reveal xref-bibr">58</a></sup> As shown in Fig. <a reveal-id="DSQ005F5" data-open="DSQ005F5" class="link link-reveal link-table xref-fig">5</a>, the <em>che-3</em> and <em>che-5</em> gene clusters of <em>D. desulfuricans</em>, in particular, have a similar gene organization to each other. There are significant homologies (more than 22% identity at the level of amino acid sequences) between CheA/Y proteins in <em>D. desulfuricans</em> and other organisms. Thus, various <em>D. desulfuricans</em> chemotaxis-like systems may be responsible not only for chemotaxis but also for other cellular functions involved in cell–cell interactions. In fact, this genome contains several gene clusters potentially involved in the formation of a type IV pilus that plays a role in the adhesion of bacteria to host cells and solid surfaces, and mediates bacterial twitching motility. Some genes encoding pilus-related proteins such as PilA of the major prepilin (DEFDS_1243), PilB of the assembly ATPase (DEFDS_1109 and DEFDS_1256), PilQ of secretin (DEFDS_1255), and PilD of prepilin peptidase (DEFDS_0112) were detected. Considering the presence of these genes, it is possible to propose a lifestyle of clinging to the chimney surface for survival under a constantly variable environment.<sup><a reveal-id="DSQ005C59" data-open="DSQ005C59" class="link link-ref link-reveal xref-bibr">59</a></sup></p>                        <a id="5027302" scrollto-destination="5027302"></a>
<div id="DSQ005F5" data-id="DSQ005F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq005/2/m_dsq00505.jpeg?Expires=2147483647&Signature=IkAjk1SYvGpW~AuRG9t1Bmno6fUo4IRdxKnEV9L1U8j2COeBzvU~QgSLCMH0F9b6QHypHnL5ae3~ootTv4YWC1kuaaQZYxcbtzU1x8JCuw0p7XS6D~Cq3h2sFx1yqoHmlYhAMkzmHUv~3x71-CYY04oaL9dlvMuZbzF4rseuOoK7zzmt8zHCJtNtX0NbqiJ3YNwRDOMBmT68I3nh5ZQLnJOfhqaGhCjblm9CaNAgI6Ha9L~mx8hC6N7llwviL~Yt1a3-3N8RUxV2RNqA9sSaMn-F8aDGyOnHiq8GI1uJseCpG4mCuOaXp-tQHmkPZgL2yVSVeRKKxztWN204G-FDAg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene arrangement of the representative chemotaxis-like gene clusters in the genomes of D. desulfuricans and related bacteria. Six chemotaxis-like gene clusters (che-1 to che-6) identified in the D. desulfuricans genome are compared with those CheA/Y-containing clusters that have been experimentally verified: M. xanthus Frz,57P. aeruginosa Wsp,56R. centenum Cluster 3,58 and Synechocystis sp. PCC 6803 Tax2.68" path-from-xml="dsq00505.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5027302/dsq00505.jpeg" path-from-xml="dsq00505.jpeg" target="_blank">View large</a><a section="5027302" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq005/2/dsq00505.jpeg?Expires=2147483647&Signature=E~lCu6mDG54H-ise8YrQn-1cJ1LbXzWWPSWDtUjw11AOxEyxqFGFPAdKCmm4nlUv0kz9-lMVbgY~2olhf6lvnC3D6o2qlE4ID4j6rXHzfAgJi9V5JzYuGCXdAc63z4SYatWzxpxDhNxr2ntd786isP7TQ3ZnInHQqM0p2wJKtOQArxhMYAScnHb6MnnLu7Nx4BWrQ3YvmFI3t3wL2z9~5F9AV3INASkiy6CuUGQHHI5umiOiO9zfbhoCVhKZzs2Tr~IaqZyJ0M0T6h-SG2gQuVKb5zoAVAuQpxy5X~tTc-QXDXwnUHQjMOE~rVa4d3M5TmxIJNLJ6N6cnUAdjLhrXw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5027302&ar=353305&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00505.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene arrangement of the representative chemotaxis-like gene clusters in the genomes of <em>D. desulfuricans</em> and related bacteria. Six chemotaxis-like gene clusters (<em>che-1</em> to <em>che-6</em>) identified in the <em>D. desulfuricans</em> genome are compared with those CheA/Y-containing clusters that have been experimentally verified: <em>M. xanthus</em> Frz,<sup><a reveal-id="DSQ005C57" data-open="DSQ005C57" class="link link-ref link-reveal xref-bibr">57</a></sup><em>P. aeruginosa</em> Wsp,<sup><a reveal-id="DSQ005C56" data-open="DSQ005C56" class="link link-ref link-reveal xref-bibr">56</a></sup><em>R. centenum</em> Cluster 3,<sup><a reveal-id="DSQ005C58" data-open="DSQ005C58" class="link link-ref link-reveal xref-bibr">58</a></sup> and <em>Synechocystis</em> sp. PCC 6803 Tax2.<sup><a reveal-id="DSQ005C68" data-open="DSQ005C68" class="link link-ref link-reveal xref-bibr">68</a></sup></p></div></div><div content-id="DSQ005F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq005/2/m_dsq00505.jpeg?Expires=2147483647&Signature=IkAjk1SYvGpW~AuRG9t1Bmno6fUo4IRdxKnEV9L1U8j2COeBzvU~QgSLCMH0F9b6QHypHnL5ae3~ootTv4YWC1kuaaQZYxcbtzU1x8JCuw0p7XS6D~Cq3h2sFx1yqoHmlYhAMkzmHUv~3x71-CYY04oaL9dlvMuZbzF4rseuOoK7zzmt8zHCJtNtX0NbqiJ3YNwRDOMBmT68I3nh5ZQLnJOfhqaGhCjblm9CaNAgI6Ha9L~mx8hC6N7llwviL~Yt1a3-3N8RUxV2RNqA9sSaMn-F8aDGyOnHiq8GI1uJseCpG4mCuOaXp-tQHmkPZgL2yVSVeRKKxztWN204G-FDAg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene arrangement of the representative chemotaxis-like gene clusters in the genomes of D. desulfuricans and related bacteria. Six chemotaxis-like gene clusters (che-1 to che-6) identified in the D. desulfuricans genome are compared with those CheA/Y-containing clusters that have been experimentally verified: M. xanthus Frz,57P. aeruginosa Wsp,56R. centenum Cluster 3,58 and Synechocystis sp. PCC 6803 Tax2.68" path-from-xml="dsq00505.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5027302/dsq00505.jpeg" path-from-xml="dsq00505.jpeg" target="_blank">View large</a><a section="5027302" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq005/2/dsq00505.jpeg?Expires=2147483647&Signature=E~lCu6mDG54H-ise8YrQn-1cJ1LbXzWWPSWDtUjw11AOxEyxqFGFPAdKCmm4nlUv0kz9-lMVbgY~2olhf6lvnC3D6o2qlE4ID4j6rXHzfAgJi9V5JzYuGCXdAc63z4SYatWzxpxDhNxr2ntd786isP7TQ3ZnInHQqM0p2wJKtOQArxhMYAScnHb6MnnLu7Nx4BWrQ3YvmFI3t3wL2z9~5F9AV3INASkiy6CuUGQHHI5umiOiO9zfbhoCVhKZzs2Tr~IaqZyJ0M0T6h-SG2gQuVKb5zoAVAuQpxy5X~tTc-QXDXwnUHQjMOE~rVa4d3M5TmxIJNLJ6N6cnUAdjLhrXw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5027302&ar=353305&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq00505.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene arrangement of the representative chemotaxis-like gene clusters in the genomes of <em>D. desulfuricans</em> and related bacteria. Six chemotaxis-like gene clusters (<em>che-1</em> to <em>che-6</em>) identified in the <em>D. desulfuricans</em> genome are compared with those CheA/Y-containing clusters that have been experimentally verified: <em>M. xanthus</em> Frz,<sup><a reveal-id="DSQ005C57" data-open="DSQ005C57" class="link link-ref link-reveal xref-bibr">57</a></sup><em>P. aeruginosa</em> Wsp,<sup><a reveal-id="DSQ005C56" data-open="DSQ005C56" class="link link-ref link-reveal xref-bibr">56</a></sup><em>R. centenum</em> Cluster 3,<sup><a reveal-id="DSQ005C58" data-open="DSQ005C58" class="link link-ref link-reveal xref-bibr">58</a></sup> and <em>Synechocystis</em> sp. PCC 6803 Tax2.<sup><a reveal-id="DSQ005C68" data-open="DSQ005C68" class="link link-ref link-reveal xref-bibr">68</a></sup></p></div></div>                            <h3 scrollto-destination=5027303 id="5027303" class="section-title" data-legacy-id=s3f>3.6. Adaptation to thermal environment</h3>
<p>As in other thermophilic microbes, the proteome of <em>D. desulfuricans</em>, which can grow up to 70°C should be stable at least at the maximum growth temperature. Generally, it is known that a protein's amino acid composition has a great influence on its thermostability, and the proteins of thermophiles show a tendency to possess fewer non-charged amino acids and more charged amino acids than those of mesophiles.<sup><a reveal-id="DSQ005C60" data-open="DSQ005C60" class="link link-ref link-reveal xref-bibr">60</a></sup> Actually, all of the proteins encoded in the <em>D. desulfuricans</em> genome were found to have fewer histidine, glutamine, and threonine residues, but more glutamic acid and lysine residues than mesophiles (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). Both charged amino acids presumably contribute to increase the thermostability of the protein by enhancing the occurrence of salt bridges and ion pairs.<sup><a reveal-id="DSQ005C61" data-open="DSQ005C61" class="link link-ref link-reveal xref-bibr">61</a></sup></p><p>Also, the stabilization of DNA and RNA at a high temperature is indispensable for survival in the hydrothermal environment. It has been suggested that polyamines, RNA methyltransferases, and protamine P1 could contribute to the thermoadaptation of <em>Geobacillus kaustophilus</em>, which has a similar maximum temperature for growth (74°C) as <em>D. desulfuricans</em>.<sup><a reveal-id="DSQ005C62" data-open="DSQ005C62" class="link link-ref link-reveal xref-bibr">62</a></sup> In <em>Thermus thermophilus</em>, which can grow up to 85°C, the inactivation of genes for polyamine biosynthesis or tRNA (adenine-N1)-methyltransferase (TrmI) results in a thermolabile phenotype.<sup><a reveal-id="DSQ005C63" data-open="DSQ005C63" class="link link-ref link-reveal xref-bibr">63</a></sup> The <em>D. desulfuricans</em> genome encodes genes necessary for putrescine and spermidine synthesis, <em>speA</em> (DEFDS_1288) and <em>speBDE</em> (DEFDS_0967-9). In addition, we identified two genes for norspermidine synthesis—carboxynorspermidine dehydrogenase (DEFDS_1287) and carboxynorspermidine decarboxylase (DEFDS_1286). Norspermidine and norspermine are commonly found in hyperthermophilic bacteria.<sup><a reveal-id="DSQ005C64" data-open="DSQ005C64" class="link link-ref link-reveal xref-bibr">64</a></sup> These polyamines stabilize DNA by binding to nucleic acids and can induce aggregation or conformational changes of DNA.<sup><a reveal-id="DSQ005C65" data-open="DSQ005C65" class="link link-ref link-reveal xref-bibr">65</a></sup> On the other hand, a homolog (DEFDS_0605) of TrmI found in <em>T. thermophilus</em>, and four tRNA and three tRNA/rRNA methyltransferase genes in total were identified in <em>D. desulfuricans</em>. The genes described above appear to contribute to the thermoadaptation of <em>D. desulfuricans</em> as well as other thermophilic bacteria.</p><p><em>Deferribacter desulfuricans</em> possesses many molecular chaperons used for protein folding and unfolding such as DnaJ-DnaK-GrpE (DEFDS_2113-4) and the HrcA repressor (DEFDS_2113-6), GroEL-GroES (DEFDS_0240-39), and small heat shock proteins categorized into Hsp20 family (DEFDS_1576 and DEFDS_1707). This organism also possesses ATP-dependent heat shock-responsive proteases, which is thought to be concerned in providing thermotolerance to cell on exposure to heat stress, such as HslVU (DEFDS_2155-6), ClpPX (DEFDS_0193-4), and Lon (DEFDS_0195 and DEFDS_1706).</p>                            <h3 scrollto-destination=5027307 id="5027307" class="section-title" data-legacy-id=s3g>3.7. Antioxidant system</h3>
<p><em>Deferribacter desulfuricans</em> SSM1 is a strict anaerobe: its genome possesses many genes potentially associated with resistance to oxidative stress. DEFDS_1019 and DEFDS_0573 encode a superoxide reductase (SOR) and a rubredoxin (Rd), respectively. The SOR–Rd proteins can catalyse the reduction of superoxide to hydrogen peroxide. Putative rubrerythrins (Rbr) encoded by DEFDS_0019, DEFDS_1568, and DEFDS_1971 further reduce hydrogen peroxide to water.<sup><a reveal-id="DSQ005C66" data-open="DSQ005C66" class="link link-ref link-reveal xref-bibr">66</a></sup> DEFDS_1568 is located adjacent to the gene encoding the peroxide repressor (<em>perR</em>) (DEFDS_1567). A series of these genes presumably construct the antioxidant system of <em>D. desulfuricans</em> SSM1 inhabiting the oxic–anoxic transition zone in the deep-sea hydrothermal environment. Additionally, genes for peroxiredoxins (DEFDS_0018 and DEFDS_1350), thioredoxin (DEFDS_1167), and thioredoxin reductase (DEFDS_0503) may be involved in the antioxidant system.</p><p>On the other hand, <em>D. desulfuricans</em> possesses genes encoding two potential dioxygen scavenging systems. The first one is a cytochrome <em>bd</em> quinol oxidase (DEFDS_1619-1620), which is a respiratory terminal oxidase, and the second one is a rubredoxin-oxygen oxidoreductase (Roo), whose gene is located next to the rubredoxin gene (DEFDS_0573). In the sulphate-reducing <em>Desulfovibrio</em>-related species, the latter enzyme is proposed to be the cytoplasmic terminal oxidase for the non-energy-conserving respiratory reduction of O<sub>2</sub> or nitric oxide.<sup><a reveal-id="DSQ005C67" data-open="DSQ005C67" class="link link-ref link-reveal xref-bibr">67</a></sup> Thus, the <em>bd</em>-type oxidase and Roo of <em>D. desulfuricans</em> may also serve to relieve the environmental and intracellular oxidative stresses.</p>                            <h3 scrollto-destination=5027310 id="5027310" class="section-title" data-legacy-id=s3h>3.8. Conclusions</h3>
<p>Genome analysis of <em>D. desulfuricans</em> SSM1 revealed its versatile energy and carbon metabolisms and its machineries for sensing and responding to the environmental changes in hydrothermal vent habitats. We showed that the molecular systems such as the multihaem <em>c</em>-type cytochrome clusters, two-component signal transducers, and abundant chemotaxis components could be tightly linked to the adaptation mechanisms required to adapt in physically and chemically variable environments. The multiple signal transduction systems for sensing dynamic changes in carbon source and temperature, and the type IV pili, which are likely to be useful for clinging to the chimney, allow <em>D. desulfuricans</em> to survive in such a harsh environment. The genome sequence of <em>D. desulfuricans</em> SSM1 should provide many clues for the better understanding of bacterial life in environments around hydrothermal vents from the ecological and evolutionary points of view. The sequences, as well as the gene information shown in this paper, are available in the web databases, ExtremoBase (<a class="link link-uri" href="http://www.jamstec.go.jp/gbrowser/cgi-bin/top.cgi">http://www.jamstec.go.jp/gbrowser/cgi-bin/top.cgi</a>) and DOGAN (<a class="link link-uri" href="http://www.bio.nite.g.o.jp/dogan/Top">http://www.bio.nite.g.o.jp/dogan/Top</a>).</p>                            <h2 scrollto-destination=5027312 id="5027312" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5027314 id="5027314" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSQ005C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gold</div> <div class="given-names">T.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Gold</div> <div class="given-names">T.</div></div>. </div>, <div class="source">The Deep Hot Biosphere.</div> , <div class="year">2001</div><div class="publisher-loc">New York</div><div class="publisher-name">Copernicus Books</div>(pg. <div class="fpage">19</div>-<div class="lpage">22</div>)</div></div></div></div><div content-id="DSQ005C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takai</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kobayashi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Nealson</div> <div class="given-names">K.H.</div></div>,  <div class="name"><div class="surname">Horikoshi</div> <div class="given-names">K.</div></div>. </div><div class="article-title"><em>Deferribacter desulfuricans</em> sp. nov., a novel sulfur-, nitrate- and arsenate-reducing thermophile isolated from a deep-sea hydrothermal vent</div>, <div class="source">Int. J. Syst. Evol. Microbiol.</div> , <div class="year">2003</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">839</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deferribacter%20desulfuricans%20sp.%20nov.%2C%20a%20novel%20sulfur-%2C%20nitrate-%20and%20arsenate-reducing%20thermophile%20isolated%20from%20a%20deep-sea%20hydrothermal%20vent&amp;author=K.%20Takai&amp;author=H.%20Kobayashi&amp;author=K.H.%20Nealson&amp;author=K.%20Horikoshi&amp;publication_year=2003&amp;journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&amp;volume=53&amp;pages=839-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/ijs.0.02479-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fijs.0.02479-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fijs.0.02479-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12807210" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miroshnichenko</div> <div class="given-names">M.L.</div></div>,  <div class="name"><div class="surname">Slobodkin</div> <div class="given-names">A.I.</div></div>,  <div class="name"><div class="surname">Kostrikina</div> <div class="given-names">N.A.</div></div>, et al. </div><div class="article-title"><em>Deferribacter abyssi</em> sp. nov., an anaerobic thermophile from deep-sea hydrothermal vents of the Mid-Atlantic Ridge</div>, <div class="source">Int. J. Syst. Evol. Microbiol.</div> , <div class="year">2003</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">1637</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deferribacter%20abyssi%20sp.%20nov.%2C%20an%20anaerobic%20thermophile%20from%20deep-sea%20hydrothermal%20vents%20of%20the%20Mid-Atlantic%20Ridge&amp;author=M.L.%20Miroshnichenko&amp;author=A.I.%20Slobodkin&amp;author=N.A.%20Kostrikina&amp;publication_year=2003&amp;journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&amp;volume=53&amp;pages=1637-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/ijs.0.02673-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fijs.0.02673-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fijs.0.02673-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/13130062" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Greene</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">Patel</div> <div class="given-names">B.K.</div></div>,  <div class="name"><div class="surname">Sheehy</div> <div class="given-names">A.J.</div></div>. </div><div class="article-title"><em>Deferribacter thermophilus</em> gen. nov., sp. nov., a novel thermophilic manganese- and iron-reducing bacterium isolated from a petroleum reservoir</div>, <div class="source">Int. J. Syst. Bacteriol.</div> , <div class="year">1997</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">505</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deferribacter%20thermophilus%20gen.%20nov.%2C%20sp.%20nov.%2C%20a%20novel%20thermophilic%20manganese-%20and%20iron-reducing%20bacterium%20isolated%20from%20a%20petroleum%20reservoir&amp;author=A.C.%20Greene&amp;author=B.K.%20Patel&amp;author=A.J.%20Sheehy&amp;publication_year=1997&amp;journal=Int.%20J.%20Syst.%20Bacteriol.&amp;volume=47&amp;pages=505-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00207713-47-2-505" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00207713-47-2-505" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00207713-47-2-505"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9103640" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Slobodkina</div> <div class="given-names">G.B.</div></div>,  <div class="name"><div class="surname">Kolganova</div> <div class="given-names">T.V.</div></div>,  <div class="name"><div class="surname">Chernyh</div> <div class="given-names">N.A.</div></div>,  <div class="name"><div class="surname">Querellou</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bonch-Osmolovskaya</div> <div class="given-names">E.A.</div></div>,  <div class="name"><div class="surname">Slobodkin</div> <div class="given-names">A.I.</div></div>. </div><div class="article-title"><em>Deferribacter autotrophicus</em> sp. nov., an iron(III)-reducing bacterium from a deep-sea hydrothermal vent</div>, <div class="source">Int. J. Syst. Evol. Microbiol.</div> , <div class="year">2009</div>, vol. <div class="volume">59</div> (pg. <div class="fpage">1508</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deferribacter%20autotrophicus%20sp.%20nov.%2C%20an%20iron(III)-reducing%20bacterium%20from%20a%20deep-sea%20hydrothermal%20vent&amp;author=G.B.%20Slobodkina&amp;author=T.V.%20Kolganova&amp;author=N.A.%20Chernyh&amp;author=J.%20Querellou&amp;author=E.A.%20Bonch-Osmolovskaya&amp;author=A.I.%20Slobodkin&amp;publication_year=2009&amp;journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&amp;volume=59&amp;pages=1508-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/ijs.0.006767-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fijs.0.006767-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fijs.0.006767-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19502344" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakagawa</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Takai</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Inagaki</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">Distribution, phylogenetic diversity and physiological characteristics of epsilon-<em>Proteobacteria</em> in a deep-sea hydrothermal field</div>, <div class="source">Environ. Microbiol.</div> , <div class="year">2005</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">1619</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distribution%2C%20phylogenetic%20diversity%20and%20physiological%20characteristics%20of%20epsilon-Proteobacteria%20in%20a%20deep-sea%20hydrothermal%20field&amp;author=S.%20Nakagawa&amp;author=K.%20Takai&amp;author=F.%20Inagaki&amp;publication_year=2005&amp;journal=Environ.%20Microbiol.&amp;volume=7&amp;pages=1619-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/emi.2005.7.issue-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Femi.2005.7.issue-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Femi.2005.7.issue-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16156735" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Slobodkin</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Campbell</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Cary</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Bonch-Osmolovskaya</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Jeanthon</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Evidence for the presence of thermophilic Fe(III)-reducing microorganisms in deep-sea hydrothermal vents at 13 degrees N (East Pacific Rise)</div>, <div class="source">FEMS Microbiol. Ecol.</div> , <div class="year">2001</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">235</div>-<div class="lpage">43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20the%20presence%20of%20thermophilic%20Fe(III)-reducing%20microorganisms%20in%20deep-sea%20hydrothermal%20vents%20at%2013%20degrees%20N%20(East%20Pacific%20Rise)&amp;author=A.%20Slobodkin&amp;author=B.%20Campbell&amp;author=S.C.%20Cary&amp;author=E.%20Bonch-Osmolovskaya&amp;author=C.%20Jeanthon&amp;publication_year=2001&amp;journal=FEMS%20Microbiol.%20Ecol.&amp;volume=36&amp;pages=235-43" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11451528" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takai</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nunoura</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ishibashi</div> <div class="given-names">J.-I.</div></div>, et al. </div><div class="article-title">Variability in the microbial communities and hydrothermal fluid chemistry at the newly discovered Mariner hydrothermal field, southern Lau Basin</div>, <div class="source">J. Geophys. Res.-Biogeosci.</div> , <div class="year">2008</div>, vol. <div class="volume">113</div> pg. <div class="fpage">G02031</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variability%20in%20the%20microbial%20communities%20and%20hydrothermal%20fluid%20chemistry%20at%20the%20newly%20discovered%20Mariner%20hydrothermal%20field%2C%20southern%20Lau%20Basin&amp;author=K.%20Takai&amp;author=T.%20Nunoura&amp;author=J.-I.%20Ishibashi&amp;publication_year=2008&amp;journal=J.%20Geophys.%20Res.-Biogeosci.&amp;volume=113&amp;pages=G02031" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1029/2007JG000636" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1029%2F2007JG000636" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1029%2F2007JG000636"> </span></div> </div></div></div></div></div><div content-id="DSQ005C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Scott</div> <div class="given-names">K.M.</div></div>,  <div class="name"><div class="surname">Sievert</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Abril</div> <div class="given-names">F.N.</div></div>, et al. </div><div class="article-title">The genome of deep-sea vent chemolithoautotroph <em>Thiomicrospira crunogena</em> XCL-2</div>, <div class="source">PLoS Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e383</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20deep-sea%20vent%20chemolithoautotroph%20Thiomicrospira%20crunogena%20XCL-2&amp;author=K.M.%20Scott&amp;author=S.M.%20Sievert&amp;author=F.N.%20Abril&amp;publication_year=2006&amp;journal=PLoS%20Biol.&amp;volume=4&amp;pages=e383" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0040383" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0040383" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0040383"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17105352" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Campbell</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Hanson</div> <div class="given-names">T.E.</div></div>, et al. </div><div class="article-title">Adaptations to submarine hydrothermal environments exemplified by the genome of <em>Nautilia profundicola</em></div>, <div class="source">PLoS Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e1000362</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adaptations%20to%20submarine%20hydrothermal%20environments%20exemplified%20by%20the%20genome%20of%20Nautilia%20profundicola&amp;author=B.J.%20Campbell&amp;author=J.L.%20Smith&amp;author=T.E.%20Hanson&amp;publication_year=2009&amp;journal=PLoS%20Genet.&amp;volume=5&amp;pages=e1000362" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000362" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000362" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000362"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19197347" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakagawa</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Takaki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Shimamura</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Reysenbach</div> <div class="given-names">A.-L.</div></div>,  <div class="name"><div class="surname">Takai</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Horikoshi</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Deep-sea vent ε-proteobacterial genomes provide insights into emergence of pathogens</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2007</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">12146</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deep-sea%20vent%20%CE%B5-proteobacterial%20genomes%20provide%20insights%20into%20emergence%20of%20pathogens&amp;author=S.%20Nakagawa&amp;author=Y.%20Takaki&amp;author=S.%20Shimamura&amp;author=A.-L.%20Reysenbach&amp;author=K.%20Takai&amp;author=K.%20Horikoshi&amp;publication_year=2007&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=104&amp;pages=12146-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0700687104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0700687104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0700687104"> </span></div> </div></div></div></div></div><div content-id="DSQ005C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Reysenbach</div> <div class="given-names">A.-L.</div></div>,  <div class="name"><div class="surname">Hamamura</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Podar</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Complete and draft genome sequences of six members of the <em>Aquificales</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2009</div>, vol. <div class="volume">191</div> (pg. <div class="fpage">1992</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20and%20draft%20genome%20sequences%20of%20six%20members%20of%20the%20Aquificales&amp;author=A.-L.%20Reysenbach&amp;author=N.%20Hamamura&amp;author=M.%20Podar&amp;publication_year=2009&amp;journal=J.%20Bacteriol.&amp;volume=191&amp;pages=1992-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.01645-08" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.01645-08" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.01645-08"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19136599" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takarada</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sekine</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kosugi</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Complete genome sequence of the soil actinomycete <em>Kocuria rhizophila</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2008</div>, vol. <div class="volume">190</div> (pg. <div class="fpage">4139</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20the%20soil%20actinomycete%20Kocuria%20rhizophila&amp;author=H.%20Takarada&amp;author=M.%20Sekine&amp;author=H.%20Kosugi&amp;publication_year=2008&amp;journal=J.%20Bacteriol.&amp;volume=190&amp;pages=4139-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.01853-07" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.01853-07" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.01853-07"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18408034" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ewing</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Hillier</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Wendl</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Base-calling of automated sequencer traces using phred. I. Accuracy assessment</div>, <div class="source">Genome Res.</div> , <div class="year">1998</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">175</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Base-calling%20of%20automated%20sequencer%20traces%20using%20phred.%20I.%20Accuracy%20assessment&amp;author=B.%20Ewing&amp;author=L.%20Hillier&amp;author=M.C.%20Wendl&amp;author=P.%20Green&amp;publication_year=1998&amp;journal=Genome%20Res.&amp;volume=8&amp;pages=175-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.8.3.175" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.8.3.175" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.8.3.175"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9521921" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Griffiths-Jones</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bateman</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Marshall</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Khanna</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">Rfam: an RNA family database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">439</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rfam%3A%20an%20RNA%20family%20database&amp;author=S.%20Griffiths-Jones&amp;author=A.%20Bateman&amp;author=M.%20Marshall&amp;author=A.%20Khanna&amp;author=S.R.%20Eddy&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=439-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12520045" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lowe</div> <div class="given-names">T.M.</div></div>,  <div class="name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">955</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%3A%20a%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence&amp;author=T.M.%20Lowe&amp;author=S.R.%20Eddy&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=955-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.5.0955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.5.0955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.5.0955"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9023104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Schäffer</div> <div class="given-names">A.A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Sch%C3%A4ffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Delcher</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Harmon</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Kasif</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">White</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div>. </div><div class="article-title">Improved microbial gene identification with GLIMMER</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1999</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">4636</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20microbial%20gene%20identification%20with%20GLIMMER&amp;author=A.L.%20Delcher&amp;author=D.%20Harmon&amp;author=S.%20Kasif&amp;author=O.%20White&amp;author=S.L.%20Salzberg&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=4636-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.23.4636" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.23.4636" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.23.4636"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10556321" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Besemer</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lomsadze</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Borodovsky</div> <div class="given-names">M.</div></div>. </div><div class="article-title">GeneMarkS: a self-training method for prediction of gene starts in microbial genomes. Implications for finding sequence motifs in regulatory regions</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2001</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">2607</div>-<div class="lpage">18</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GeneMarkS%3A%20a%20self-training%20method%20for%20prediction%20of%20gene%20starts%20in%20microbial%20genomes.%20Implications%20for%20finding%20sequence%20motifs%20in%20regulatory%20regions&amp;author=J.%20Besemer&amp;author=A.%20Lomsadze&amp;author=M.%20Borodovsky&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res.&amp;volume=29&amp;pages=2607-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/29.12.2607" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.12.2607" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F29.12.2607"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11410670" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Benson</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Karsch-Mizrachi</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Ostell</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wheeler</div> <div class="given-names">D.L.</div></div>. </div><div class="article-title">GenBank</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">D34</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GenBank&amp;author=D.A.%20Benson&amp;author=I.%20Karsch-Mizrachi&amp;author=D.J.%20Lipman&amp;author=J.%20Ostell&amp;author=D.L.%20Wheeler&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=D34-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki063" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki063" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki063"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15608212" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bairoch</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Apweiler</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">C.H.</div></div>, et al. </div><div class="article-title">The Universal Protein Resource (UniProt)</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">D154</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Universal%20Protein%20Resource%20(UniProt)&amp;author=A.%20Bairoch&amp;author=R.%20Apweiler&amp;author=C.H.%20Wu&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=D154-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki070" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki070" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki070"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15608167" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kanehisa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Goto</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kawashima</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Okuno</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hattori</div> <div class="given-names">M.</div></div>. </div><div class="article-title">The KEGG resource for deciphering the genome</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">D277</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20KEGG%20resource%20for%20deciphering%20the%20genome&amp;author=M.%20Kanehisa&amp;author=S.%20Goto&amp;author=S.%20Kawashima&amp;author=Y.%20Okuno&amp;author=M.%20Hattori&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=D277-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh063" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh063" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh063"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14681412" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Finn</div> <div class="given-names">R.D.</div></div>,  <div class="name"><div class="surname">Tate</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Mistry</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">The Pfam protein families database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D281</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Pfam%20protein%20families%20database&amp;author=R.D.%20Finn&amp;author=J.%20Tate&amp;author=J.%20Mistry&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D281-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm960" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm960" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm960"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18039703" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Caspi</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Foerster</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Fulcher</div> <div class="given-names">C.A.</div></div>, et al. </div><div class="article-title">The MetaCyc Database of metabolic pathways and enzymes and the BioCyc collection of Pathway/Genome Databases</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D623</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20MetaCyc%20Database%20of%20metabolic%20pathways%20and%20enzymes%20and%20the%20BioCyc%20collection%20of%20Pathway%2FGenome%20Databases&amp;author=R.%20Caspi&amp;author=H.%20Foerster&amp;author=C.A.%20Fulcher&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D623-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm900" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm900" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm900"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17965431" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bendtsen</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Nielsen</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">von Heijne</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Brunak</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Improved prediction of signal peptides: SignalP 3.0</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">340</div> (pg. <div class="fpage">783</div>-<div class="lpage">95</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20prediction%20of%20signal%20peptides%3A%20SignalP%203.0&amp;author=J.D.%20Bendtsen&amp;author=H.%20Nielsen&amp;author=G.%20von%20Heijne&amp;author=S.%20Brunak&amp;publication_year=2004&amp;journal=J.%20Mol.%20Biol.&amp;volume=340&amp;pages=783-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmb.2004.05.028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2004.05.028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jmb.2004.05.028"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15223320" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hirokawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Boon-Chieng</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Mitaku</div> <div class="given-names">S.</div></div>. </div><div class="article-title">SOSUI: classification and secondary structure prediction system for membrane proteins</div>, <div class="source">Bioinformatics</div> , <div class="year">1998</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">378</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SOSUI%3A%20classification%20and%20secondary%20structure%20prediction%20system%20for%20membrane%20proteins&amp;author=T.%20Hirokawa&amp;author=S.%20Boon-Chieng&amp;author=S.%20Mitaku&amp;publication_year=1998&amp;journal=Bioinformatics&amp;volume=14&amp;pages=378-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/14.4.378" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F14.4.378" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F14.4.378"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9632836" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grissa</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Vergnaud</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Pourcel</div> <div class="given-names">C.</div></div>. </div><div class="article-title">CRISPRFinder: a web tool to identify clustered regularly interspaced short palindromic repeats</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">W52</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CRISPRFinder%3A%20a%20web%20tool%20to%20identify%20clustered%20regularly%20interspaced%20short%20palindromic%20repeats&amp;author=I.%20Grissa&amp;author=G.%20Vergnaud&amp;author=C.%20Pourcel&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=W52-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm360" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm360" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm360"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17537822" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Remm</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Storm</div> <div class="given-names">C.E.</div></div>,  <div class="name"><div class="surname">Sonnhammer</div> <div class="given-names">E.L.</div></div>. </div><div class="article-title">Automatic clustering of orthologs and in-paralogs from pairwise species comparisons</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2001</div>, vol. <div class="volume">314</div> (pg. <div class="fpage">1041</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automatic%20clustering%20of%20orthologs%20and%20in-paralogs%20from%20pairwise%20species%20comparisons&amp;author=M.%20Remm&amp;author=C.E.%20Storm&amp;author=E.L.%20Sonnhammer&amp;publication_year=2001&amp;journal=J.%20Mol.%20Biol.&amp;volume=314&amp;pages=1041-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.2000.5197" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.2000.5197" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.2000.5197"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11743721" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tatusov</div> <div class="given-names">R.L.</div></div>,  <div class="name"><div class="surname">Fedorova</div> <div class="given-names">N.D.</div></div>,  <div class="name"><div class="surname">Jackson</div> <div class="given-names">J.D.</div></div>, et al. </div><div class="article-title">The COG database: an updated version includes eukaryotes</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2003</div>, vol. <div class="volume">4</div> pg. <div class="fpage">41</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20COG%20database%3A%20an%20updated%20version%20includes%20eukaryotes&amp;author=R.L.%20Tatusov&amp;author=N.D.%20Fedorova&amp;author=J.D.%20Jackson&amp;publication_year=2003&amp;journal=BMC%20Bioinformatics&amp;volume=4&amp;pages=41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-4-41" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-4-41" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-4-41"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12969510" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wolf</div> <div class="given-names">Y.I.</div></div>,  <div class="name"><div class="surname">Rogozin</div> <div class="given-names">I.B.</div></div>,  <div class="name"><div class="surname">Grishin</div> <div class="given-names">N.V.</div></div>,  <div class="name"><div class="surname">Koonin</div> <div class="given-names">E.V.</div></div>. </div><div class="article-title">Genome trees and the tree of life</div>, <div class="source">Trends Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">472</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20trees%20and%20the%20tree%20of%20life&amp;author=Y.I.%20Wolf&amp;author=I.B.%20Rogozin&amp;author=N.V.%20Grishin&amp;author=E.V.%20Koonin&amp;publication_year=2002&amp;journal=Trends%20Genet.&amp;volume=18&amp;pages=472-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(02)02744-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(02)02744-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(02)02744-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12175808" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hacker</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kaper</div> <div class="given-names">J.B.</div></div>. </div><div class="article-title">Pathogenicity islands and the evolution of microbes</div>, <div class="source">Annu. Rev. Microbiol.</div> , <div class="year">2000</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">641</div>-<div class="lpage">79</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pathogenicity%20islands%20and%20the%20evolution%20of%20microbes&amp;author=J.%20Hacker&amp;author=J.B.%20Kaper&amp;publication_year=2000&amp;journal=Annu.%20Rev.%20Microbiol.&amp;volume=54&amp;pages=641-79" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.micro.54.1.641" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.micro.54.1.641" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.micro.54.1.641"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11018140" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gerdes</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Christensen</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Løbner-Olesen</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Prokaryotic toxin–antitoxin stress response loci</div>, <div class="source">Nat. Rev. Microbiol.</div> , <div class="year">2005</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">371</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prokaryotic%20toxin%E2%80%93antitoxin%20stress%20response%20loci&amp;author=K.%20Gerdes&amp;author=S.K.%20Christensen&amp;author=A.%20L%C3%B8bner-Olesen&amp;publication_year=2005&amp;journal=Nat.%20Rev.%20Microbiol.&amp;volume=3&amp;pages=371-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro1147" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro1147" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrmicro1147"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15864262" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barrangou</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Fremaux</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Deveau</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">CRISPR provides acquired resistance against viruses in prokaryotes</div>, <div class="source">Science</div> , <div class="year">2007</div>, vol. <div class="volume">315</div> (pg. <div class="fpage">1709</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CRISPR%20provides%20acquired%20resistance%20against%20viruses%20in%20prokaryotes&amp;author=R.%20Barrangou&amp;author=C.%20Fremaux&amp;author=H.%20Deveau&amp;publication_year=2007&amp;journal=Science&amp;volume=315&amp;pages=1709-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1138140" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1138140" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1138140"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17379808" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ragsdale</div> <div class="given-names">S.W.</div></div>. </div><div class="article-title">Enzymology of the Wood–Ljungdahl pathway of acetogenesis</div>, <div class="source">Ann. N. Y. Acad. Sci.</div> , <div class="year">2008</div>, vol. <div class="volume">1125</div> (pg. <div class="fpage">129</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enzymology%20of%20the%20Wood%E2%80%93Ljungdahl%20pathway%20of%20acetogenesis&amp;author=S.W.%20Ragsdale&amp;publication_year=2008&amp;journal=Ann.%20N.%20Y.%20Acad.%20Sci.&amp;volume=1125&amp;pages=129-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1196/annals.1419.015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1196%2Fannals.1419.015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1196%2Fannals.1419.015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18378591" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Asanuma</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Kawato</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ohkawara</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hino</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Characterization and transcription of the genes encoding enzymes involved in butyrate production in <em>Butyrivibrio fibrisolvens</em></div>, <div class="source">Curr. Microbiol.</div> , <div class="year">2003</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">203</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20and%20transcription%20of%20the%20genes%20encoding%20enzymes%20involved%20in%20butyrate%20production%20in%20Butyrivibrio%20fibrisolvens&amp;author=N.%20Asanuma&amp;author=M.%20Kawato&amp;author=S.%20Ohkawara&amp;author=T.%20Hino&amp;publication_year=2003&amp;journal=Curr.%20Microbiol.&amp;volume=47&amp;pages=203-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00284-002-3976-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00284-002-3976-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00284-002-3976-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14570270" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wallrabenstein</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Schink</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Evidence of reversed electron transport in syntrophic butyrate or benzoate oxidation by <em>Syntrophomonas wolfei</em> and <em>Syntrophus buswellii</em></div>, <div class="source">Arch. Microbiol.</div> , <div class="year">1994</div>, vol. <div class="volume">162</div> (pg. <div class="fpage">136</div>-<div class="lpage">42</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20of%20reversed%20electron%20transport%20in%20syntrophic%20butyrate%20or%20benzoate%20oxidation%20by%20Syntrophomonas%20wolfei%20and%20Syntrophus%20buswellii&amp;author=C.%20Wallrabenstein&amp;author=B.%20Schink&amp;publication_year=1994&amp;journal=Arch.%20Microbiol.&amp;volume=162&amp;pages=136-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00264387" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00264387" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00264387"> </span></div> </div></div></div></div></div><div content-id="DSQ005C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Curatti</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">C.S.</div></div>,  <div class="name"><div class="surname">Ludden</div> <div class="given-names">P.W.</div></div>,  <div class="name"><div class="surname">Rubio</div> <div class="given-names">L.M.</div></div>. </div><div class="article-title">Genes required for rapid expression of nitrogenase activity in <em>Azotobacter vinelandii</em></div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2005</div>, vol. <div class="volume">102</div> (pg. <div class="fpage">6291</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genes%20required%20for%20rapid%20expression%20of%20nitrogenase%20activity%20in%20Azotobacter%20vinelandii&amp;author=L.%20Curatti&amp;author=C.S.%20Brown&amp;author=P.W.%20Ludden&amp;author=L.M.%20Rubio&amp;publication_year=2005&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=6291-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0501216102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0501216102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0501216102"> </span></div> </div></div></div></div></div><div content-id="DSQ005C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Rejtar</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Lessner</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Karger</div> <div class="given-names">B.L.</div></div>,  <div class="name"><div class="surname">Ferry</div> <div class="given-names">J.G.</div></div>. </div><div class="article-title">Electron transport in the pathway of acetate conversion to methane in the marine archaeon <em>Methanosarcina acetivorans</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2006</div>, vol. <div class="volume">188</div> (pg. <div class="fpage">702</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Electron%20transport%20in%20the%20pathway%20of%20acetate%20conversion%20to%20methane%20in%20the%20marine%20archaeon%20Methanosarcina%20acetivorans&amp;author=Q.%20Li&amp;author=L.%20Li&amp;author=T.%20Rejtar&amp;author=D.J.%20Lessner&amp;author=B.L.%20Karger&amp;author=J.G.%20Ferry&amp;publication_year=2006&amp;journal=J.%20Bacteriol.&amp;volume=188&amp;pages=702-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.188.2.702-710.2006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.188.2.702-710.2006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.188.2.702-710.2006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16385060" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Müller</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Imkamp</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Biegel</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Schmidt</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Dilling</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Discovery of a ferredoxin:NAD + -oxidoreductase (Rnf) in <em>Acetobacterium woodii</em>: a novel potential coupling site in acetogens</div>, <div class="source">Ann. N. Y. Acad. Sci.</div> , <div class="year">2008</div>, vol. <div class="volume">1125</div> (pg. <div class="fpage">137</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovery%20of%20a%20ferredoxin%3ANAD%20%2B%20-oxidoreductase%20(Rnf)%20in%20Acetobacterium%20woodii%3A%20a%20novel%20potential%20coupling%20site%20in%20acetogens&amp;author=V.%20M%C3%BCller&amp;author=F.%20Imkamp&amp;author=E.%20Biegel&amp;author=S.%20Schmidt&amp;author=S.%20Dilling&amp;publication_year=2008&amp;journal=Ann.%20N.%20Y.%20Acad.%20Sci.&amp;volume=1125&amp;pages=137-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1196/annals.1419.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1196%2Fannals.1419.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1196%2Fannals.1419.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18378592" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kumagai</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Fujiwara</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Matsubara</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Saeki</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Membrane localization, topology, and mutual stabilization of the <em>rnfABC</em> gene products in <em>Rhodobacter capsulatus</em> and implications for a new family of energy-coupling NADH oxidoreductases</div>, <div class="source">Biochemistry</div> , <div class="year">1997</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">5509</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Membrane%20localization%2C%20topology%2C%20and%20mutual%20stabilization%20of%20the%20rnfABC%20gene%20products%20in%20Rhodobacter%20capsulatus%20and%20implications%20for%20a%20new%20family%20of%20energy-coupling%20NADH%20oxidoreductases&amp;author=H.%20Kumagai&amp;author=T.%20Fujiwara&amp;author=H.%20Matsubara&amp;author=K.%20Saeki&amp;publication_year=1997&amp;journal=Biochemistry&amp;volume=36&amp;pages=5509-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/bi970014q" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fbi970014q" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fbi970014q"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9154934" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Herrmann</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Jayamani</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Mai</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Buckel</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Energy conservation via electron-transferring flavoprotein in anaerobic bacteria</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2008</div>, vol. <div class="volume">190</div> (pg. <div class="fpage">784</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Energy%20conservation%20via%20electron-transferring%20flavoprotein%20in%20anaerobic%20bacteria&amp;author=G.%20Herrmann&amp;author=E.%20Jayamani&amp;author=G.%20Mai&amp;author=W.%20Buckel&amp;publication_year=2008&amp;journal=J.%20Bacteriol.&amp;volume=190&amp;pages=784-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.01422-07" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.01422-07" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.01422-07"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18039764" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moreno-Vivián</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Cabello</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Martínez-Luque</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Blasco</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Castillo</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Prokaryotic nitrate reduction: molecular properties and functional distinction among bacterial nitrate reductases</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1999</div>, vol. <div class="volume">181</div> (pg. <div class="fpage">6573</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prokaryotic%20nitrate%20reduction%3A%20molecular%20properties%20and%20functional%20distinction%20among%20bacterial%20nitrate%20reductases&amp;author=C.%20Moreno-Vivi%C3%A1n&amp;author=P.%20Cabello&amp;author=M.%20Mart%C3%ADnez-Luque&amp;author=R.%20Blasco&amp;author=F.%20Castillo&amp;publication_year=1999&amp;journal=J.%20Bacteriol.&amp;volume=181&amp;pages=6573-84" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10542156" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jepson</div> <div class="given-names">B.J.N.</div></div>,  <div class="name"><div class="surname">Marietou</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Mohan</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Cole</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Butler</div> <div class="given-names">C.S.</div></div>,  <div class="name"><div class="surname">Richardson</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">Evolution of the soluble nitrate reductase: defining the monomeric periplasmic nitrate reductase subgroup</div>, <div class="source">Biochem. Soc. Trans.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">122</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20the%20soluble%20nitrate%20reductase%3A%20defining%20the%20monomeric%20periplasmic%20nitrate%20reductase%20subgroup&amp;author=B.J.N.%20Jepson&amp;author=A.%20Marietou&amp;author=S.%20Mohan&amp;author=J.A.%20Cole&amp;author=C.S.%20Butler&amp;author=D.J.%20Richardson&amp;publication_year=2006&amp;journal=Biochem.%20Soc.%20Trans.&amp;volume=34&amp;pages=122-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/BST0340122" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2FBST0340122" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2FBST0340122"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16417499" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Krafft</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Gross</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Kröger</div> <div class="given-names">A.</div></div>. </div><div class="article-title">The function of <em>Wolinella succinogenes psr</em> genes in electron transport with polysulphide as the terminal electron acceptor</div>, <div class="source">Eur. J. Biochem.</div> , <div class="year">1995</div>, vol. <div class="volume">230</div> (pg. <div class="fpage">601</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20function%20of%20Wolinella%20succinogenes%20psr%20genes%20in%20electron%20transport%20with%20polysulphide%20as%20the%20terminal%20electron%20acceptor&amp;author=T.%20Krafft&amp;author=R.%20Gross&amp;author=A.%20Kr%C3%B6ger&amp;publication_year=1995&amp;journal=Eur.%20J.%20Biochem.&amp;volume=230&amp;pages=601-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/ejb.1995.230.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fejb.1995.230.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fejb.1995.230.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7607234" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Clark</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Barrett</div> <div class="given-names">E.L.</div></div>. </div><div class="article-title">The <em>phs</em> gene and hydrogen sulfide production by <em>Salmonella typhimurium</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">1987</div>, vol. <div class="volume">169</div> (pg. <div class="fpage">2391</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20phs%20gene%20and%20hydrogen%20sulfide%20production%20by%20Salmonella%20typhimurium&amp;author=M.A.%20Clark&amp;author=E.L.%20Barrett&amp;publication_year=1987&amp;journal=J.%20Bacteriol.&amp;volume=169&amp;pages=2391-7" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3108233" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guiral</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tron</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Aubert</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Gloter</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Iobbi-Nivol</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Giudici-Orticoni</div> <div class="given-names">T.</div></div>. </div><div class="article-title">A membrane-bound multienzyme, hydrogen-oxidizing, and sulfur-reducing complex from the hyperthermophilic bacterium <em>Aquifex aeolicus</em></div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2005</div>, vol. <div class="volume">280</div> (pg. <div class="fpage">42004</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20membrane-bound%20multienzyme%2C%20hydrogen-oxidizing%2C%20and%20sulfur-reducing%20complex%20from%20the%20hyperthermophilic%20bacterium%20Aquifex%20aeolicus&amp;author=M.%20Guiral&amp;author=P.%20Tron&amp;author=C.%20Aubert&amp;author=A.%20Gloter&amp;author=C.%20Iobbi-Nivol&amp;author=T.%20Giudici-Orticoni&amp;publication_year=2005&amp;journal=J.%20Biol.%20Chem.&amp;volume=280&amp;pages=42004-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M508034200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M508034200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M508034200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16236714" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Laska</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Lottspeich</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Kletzin</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Membrane-bound hydrogenase and sulfur reductase of the hyperthermophilic and acidophilic archaeon <em>Acidianus ambivalens</em></div>, <div class="source">Microbiology</div> , <div class="year">2003</div>, vol. <div class="volume">149</div> (pg. <div class="fpage">2357</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Membrane-bound%20hydrogenase%20and%20sulfur%20reductase%20of%20the%20hyperthermophilic%20and%20acidophilic%20archaeon%20Acidianus%20ambivalens&amp;author=S.%20Laska&amp;author=F.%20Lottspeich&amp;author=A.%20Kletzin&amp;publication_year=2003&amp;journal=Microbiology&amp;volume=149&amp;pages=2357-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.26455-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.26455-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.26455-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12949162" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Campbell</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Jeanthon</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Kostka</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Luther</div> <div class="given-names">G.W.</div></div>,  <div class="name"><div class="surname">Cary</div> <div class="given-names">S.C.</div></div>. </div><div class="article-title">Growth and phylogenetic properties of novel bacteria belonging to the epsilon subdivision of the <em>Proteobacteria</em> enriched from <em>Alvinella pompejana</em> and deep-sea hydrothermal vents</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2001</div>, vol. <div class="volume">67</div> (pg. <div class="fpage">4566</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Growth%20and%20phylogenetic%20properties%20of%20novel%20bacteria%20belonging%20to%20the%20epsilon%20subdivision%20of%20the%20Proteobacteria%20enriched%20from%20Alvinella%20pompejana%20and%20deep-sea%20hydrothermal%20vents&amp;author=B.J.%20Campbell&amp;author=C.%20Jeanthon&amp;author=J.E.%20Kostka&amp;author=G.W.%20Luther&amp;author=S.C.%20Cary&amp;publication_year=2001&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=67&amp;pages=4566-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.67.10.4566-4572.2001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.67.10.4566-4572.2001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.67.10.4566-4572.2001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11571157" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Klimmek</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Kreis</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Klein</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Simon</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wittershagen</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kröger</div> <div class="given-names">A.</div></div>. </div><div class="article-title">The function of the periplasmic Sud protein in polysulfide respiration of <em>Wolinella succinogenes</em></div>, <div class="source">Eur. J. Biochem.</div> , <div class="year">1998</div>, vol. <div class="volume">253</div> (pg. <div class="fpage">263</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20function%20of%20the%20periplasmic%20Sud%20protein%20in%20polysulfide%20respiration%20of%20Wolinella%20succinogenes&amp;author=O.%20Klimmek&amp;author=V.%20Kreis&amp;author=C.%20Klein&amp;author=J.%20Simon&amp;author=A.%20Wittershagen&amp;author=A.%20Kr%C3%B6ger&amp;publication_year=1998&amp;journal=Eur.%20J.%20Biochem.&amp;volume=253&amp;pages=263-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1432-1327.1998.2530263.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1432-1327.1998.2530263.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1432-1327.1998.2530263.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9578485" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hancock</div> <div class="given-names">R.E.</div></div>,  <div class="name"><div class="surname">Egli</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Benz</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Siehnel</div> <div class="given-names">R.J.</div></div>. </div><div class="article-title">Overexpression in <em>Escherichia coli</em> and functional analysis of a novel PPi-selective porin, oprO, from <em>Pseudomonas aeruginosa</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">1992</div>, vol. <div class="volume">174</div> (pg. <div class="fpage">471</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Overexpression%20in%20Escherichia%20coli%20and%20functional%20analysis%20of%20a%20novel%20PPi-selective%20porin%2C%20oprO%2C%20from%20Pseudomonas%20aeruginosa&amp;author=R.E.%20Hancock&amp;author=C.%20Egli&amp;author=R.%20Benz&amp;author=R.J.%20Siehnel&amp;publication_year=1992&amp;journal=J.%20Bacteriol.&amp;volume=174&amp;pages=471-6" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1370289" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shi</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Squier</div> <div class="given-names">T.C.</div></div>,  <div class="name"><div class="surname">Zachara</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Fredrickson</div> <div class="given-names">J.K.</div></div>. </div><div class="article-title">Respiration of metal (hydr)oxides by <em>Shewanella</em> and <em>Geobacter</em>: a key role for multihaem <em>c</em>-type cytochromes</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2007</div>, vol. <div class="volume">65</div> (pg. <div class="fpage">12</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Respiration%20of%20metal%20(hydr)oxides%20by%20Shewanella%20and%20Geobacter%3A%20a%20key%20role%20for%20multihaem%20c-type%20cytochromes&amp;author=L.%20Shi&amp;author=T.C.%20Squier&amp;author=J.M.%20Zachara&amp;author=J.K.%20Fredrickson&amp;publication_year=2007&amp;journal=Mol.%20Microbiol.&amp;volume=65&amp;pages=12-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.2007.65.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.2007.65.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.2007.65.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17581116" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takai</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nunoura</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Variability in microbial communities in black smoker chimneys at the NW caldera vent field, Brothers volcano, Kermadec arc</div>, <div class="source">Geomicrobiol. J.</div> , <div class="year">2009</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">552</div>-<div class="lpage">569</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variability%20in%20microbial%20communities%20in%20black%20smoker%20chimneys%20at%20the%20NW%20caldera%20vent%20field%2C%20Brothers%20volcano%2C%20Kermadec%20arc&amp;author=K.%20Takai&amp;author=T.%20Nunoura&amp;author=Y.%20Suzuki&amp;publication_year=2009&amp;journal=Geomicrobiol.%20J.&amp;volume=26&amp;pages=552-569" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/01490450903304949" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F01490450903304949" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F01490450903304949"> </span></div> </div></div></div></div></div><div content-id="DSQ005C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Galperin</div> <div class="given-names">M.Y.</div></div>. </div><div class="article-title">Bacterial signal transduction network in a genomic perspective</div>, <div class="source">Environ. Microbiol.</div> , <div class="year">2004</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">552</div>-<div class="lpage">67</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacterial%20signal%20transduction%20network%20in%20a%20genomic%20perspective&amp;author=M.Y.%20Galperin&amp;publication_year=2004&amp;journal=Environ.%20Microbiol.&amp;volume=6&amp;pages=552-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/emi.2004.6.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Femi.2004.6.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Femi.2004.6.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15142243" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhulin</div> <div class="given-names">I.B.</div></div>,  <div class="name"><div class="surname">Taylor</div> <div class="given-names">B.L.</div></div>,  <div class="name"><div class="surname">Dixon</div> <div class="given-names">R.</div></div>. </div><div class="article-title">PAS domain S-boxes in Archaea, bacteria and sensors for oxygen and redox</div>, <div class="source">Trends Biochem. Sci.</div> , <div class="year">1997</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">331</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PAS%20domain%20S-boxes%20in%20Archaea%2C%20bacteria%20and%20sensors%20for%20oxygen%20and%20redox&amp;author=I.B.%20Zhulin&amp;author=B.L.%20Taylor&amp;author=R.%20Dixon&amp;publication_year=1997&amp;journal=Trends%20Biochem.%20Sci.&amp;volume=22&amp;pages=331-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0968-0004(97)01110-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0968-0004(97)01110-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0968-0004(97)01110-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9301332" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garrity</div> <div class="given-names">L.F.</div></div>,  <div class="name"><div class="surname">Schiel</div> <div class="given-names">S.L.</div></div>,  <div class="name"><div class="surname">Merrill</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Reizer</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Saier</div> <div class="given-names">M.H.</div></div>,  <div class="name"><div class="surname">Ordal</div> <div class="given-names">G.W.</div></div>. </div><div class="article-title">Unique regulation of carbohydrate chemotaxis in <em>Bacillus subtilis</em> by the phosphoenolpyruvate-dependent phosphotransferase system and the methyl-accepting chemotaxis protein McpC</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1998</div>, vol. <div class="volume">180</div> (pg. <div class="fpage">4475</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unique%20regulation%20of%20carbohydrate%20chemotaxis%20in%20Bacillus%20subtilis%20by%20the%20phosphoenolpyruvate-dependent%20phosphotransferase%20system%20and%20the%20methyl-accepting%20chemotaxis%20protein%20McpC&amp;author=L.F.%20Garrity&amp;author=S.L.%20Schiel&amp;author=R.%20Merrill&amp;author=J.%20Reizer&amp;author=M.H.%20Saier&amp;author=G.W.%20Ordal&amp;publication_year=1998&amp;journal=J.%20Bacteriol.&amp;volume=180&amp;pages=4475-80" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9721285" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hickman</div> <div class="given-names">J.W.</div></div>,  <div class="name"><div class="surname">Tifrea</div> <div class="given-names">D.F.</div></div>,  <div class="name"><div class="surname">Harwood</div> <div class="given-names">C.S.</div></div>. </div><div class="article-title">A chemosensory system that regulates biofilm formation through modulation of cyclic diguanylate levels</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2005</div>, vol. <div class="volume">102</div> (pg. <div class="fpage">14422</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20chemosensory%20system%20that%20regulates%20biofilm%20formation%20through%20modulation%20of%20cyclic%20diguanylate%20levels&amp;author=J.W.%20Hickman&amp;author=D.F.%20Tifrea&amp;author=C.S.%20Harwood&amp;publication_year=2005&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=14422-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0507170102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0507170102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0507170102"> </span></div> </div></div></div></div></div><div content-id="DSQ005C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zusman</div> <div class="given-names">D.R.</div></div>,  <div class="name"><div class="surname">Scott</div> <div class="given-names">A.E.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Kirby</div> <div class="given-names">J.R.</div></div>. </div><div class="article-title">Chemosensory pathways, motility and development in <em>Myxococcus xanthus</em></div>, <div class="source">Nat. Rev. Microbiol.</div> , <div class="year">2007</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">862</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chemosensory%20pathways%2C%20motility%20and%20development%20in%20Myxococcus%20xanthus&amp;author=D.R.%20Zusman&amp;author=A.E.%20Scott&amp;author=Z.%20Yang&amp;author=J.R.%20Kirby&amp;publication_year=2007&amp;journal=Nat.%20Rev.%20Microbiol.&amp;volume=5&amp;pages=862-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro1770" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro1770" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrmicro1770"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17922045" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Berleman</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Bauer</div> <div class="given-names">C.E.</div></div>. </div><div class="article-title">Involvement of a Che-like signal transduction cascade in regulating cyst cell development in <em>Rhodospirillum centenum</em></div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2005</div>, vol. <div class="volume">56</div> (pg. <div class="fpage">1457</div>-<div class="lpage">66</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Involvement%20of%20a%20Che-like%20signal%20transduction%20cascade%20in%20regulating%20cyst%20cell%20development%20in%20Rhodospirillum%20centenum&amp;author=J.E.%20Berleman&amp;author=C.E.%20Bauer&amp;publication_year=2005&amp;journal=Mol.%20Microbiol.&amp;volume=56&amp;pages=1457-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-2958.2005.04646.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-2958.2005.04646.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-2958.2005.04646.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15916598" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Reysenbach</div> <div class="given-names">A.-L.</div></div>,  <div class="name"><div class="surname">Shock</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Merging genomes with geochemistry in hydrothermal ecosystems</div>, <div class="source">Science</div> , <div class="year">2002</div>, vol. <div class="volume">296</div> (pg. <div class="fpage">1077</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Merging%20genomes%20with%20geochemistry%20in%20hydrothermal%20ecosystems&amp;author=A.-L.%20Reysenbach&amp;author=E.%20Shock&amp;publication_year=2002&amp;journal=Science&amp;volume=296&amp;pages=1077-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1072483" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1072483" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1072483"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12004120" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Singer</div> <div class="given-names">G.A.C.</div></div>,  <div class="name"><div class="surname">Hickey</div> <div class="given-names">D.A.</div></div>. </div><div class="article-title">Thermophilic prokaryotes have characteristic patterns of codon usage, amino acid composition and nucleotide content</div>, <div class="source">Gene</div> , <div class="year">2003</div>, vol. <div class="volume">317</div> (pg. <div class="fpage">39</div>-<div class="lpage">47</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Thermophilic%20prokaryotes%20have%20characteristic%20patterns%20of%20codon%20usage%2C%20amino%20acid%20composition%20and%20nucleotide%20content&amp;author=G.A.C.%20Singer&amp;author=D.A.%20Hickey&amp;publication_year=2003&amp;journal=Gene&amp;volume=317&amp;pages=39-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(03)00660-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(03)00660-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(03)00660-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14604790" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C61" class="ref-content "><div class="label -label">61</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhou</div> <div class="given-names">X.-X.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Y.-B.</div></div>,  <div class="name"><div class="surname">Pan</div> <div class="given-names">Y.-J.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Differences in amino acids composition and coupling patterns between mesophilic and thermophilic proteins</div>, <div class="source">Amino Acids</div> , <div class="year">2008</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">25</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differences%20in%20amino%20acids%20composition%20and%20coupling%20patterns%20between%20mesophilic%20and%20thermophilic%20proteins&amp;author=X.-X.%20Zhou&amp;author=Y.-B.%20Wang&amp;author=Y.-J.%20Pan&amp;author=F.%20Li&amp;publication_year=2008&amp;journal=Amino%20Acids&amp;volume=34&amp;pages=25-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00726-007-0589-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00726-007-0589-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00726-007-0589-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17710363" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C62" class="ref-content "><div class="label -label">62</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takami</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Takaki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Chee</div> <div class="given-names">G.-J.</div></div>, et al. </div><div class="article-title">Thermoadaptation trait revealed by the genome sequence of thermophilic <em>Geobacillus kaustophilus</em></div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">6292</div>-<div class="lpage">303</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Thermoadaptation%20trait%20revealed%20by%20the%20genome%20sequence%20of%20thermophilic%20Geobacillus%20kaustophilus&amp;author=H.%20Takami&amp;author=Y.%20Takaki&amp;author=G.-J.%20Chee&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=6292-303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh970" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh970" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh970"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15576355" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C63" class="ref-content "><div class="label -label">63</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Droogmans</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Roovers</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Bujnicki</div> <div class="given-names">J.M.</div></div>, et al. </div><div class="article-title">Cloning and characterization of tRNA (m1A58) methyltransferase (TrmI) from <em>Thermus thermophilus</em> HB27, a protein required for cell growth at extreme temperatures</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">2148</div>-<div class="lpage">56</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%20and%20characterization%20of%20tRNA%20(m1A58)%20methyltransferase%20(TrmI)%20from%20Thermus%20thermophilus%20HB27%2C%20a%20protein%20required%20for%20cell%20growth%20at%20extreme%20temperatures&amp;author=L.%20Droogmans&amp;author=M.%20Roovers&amp;author=J.M.%20Bujnicki&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=2148-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg314" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg314" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg314"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12682365" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C64" class="ref-content "><div class="label -label">64</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Daniel</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Cowan</div> <div class="given-names">D.A.</div></div>. </div><div class="article-title">Biomolecular stability and life at high temperatures</div>, <div class="source">Cell. Mol. Life Sci.</div> , <div class="year">2000</div>, vol. <div class="volume">57</div> (pg. <div class="fpage">250</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biomolecular%20stability%20and%20life%20at%20high%20temperatures&amp;author=R.M.%20Daniel&amp;author=D.A.%20Cowan&amp;publication_year=2000&amp;journal=Cell.%20Mol.%20Life%20Sci.&amp;volume=57&amp;pages=250-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00000688" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00000688" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00000688"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10766021" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C65" class="ref-content "><div class="label -label">65</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hou</div> <div class="given-names">M.H.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">S.B.</div></div>,  <div class="name"><div class="surname">Yuann</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">W.C.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">A.H.</div></div>,  <div class="name"><div class="surname">Kan Ls</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Effects of polyamines on the thermal stability and formation kinetics of DNA duplexes with abnormal structure</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2001</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">5121</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effects%20of%20polyamines%20on%20the%20thermal%20stability%20and%20formation%20kinetics%20of%20DNA%20duplexes%20with%20abnormal%20structure&amp;author=M.H.%20Hou&amp;author=S.B.%20Lin&amp;author=J.M.%20Yuann&amp;author=W.C.%20Lin&amp;author=A.H.%20Wang&amp;author=L.%20Kan%20Ls&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res.&amp;volume=29&amp;pages=5121-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/29.24.5121" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.24.5121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F29.24.5121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11812845" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C66" class="ref-content "><div class="label -label">66</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Weinberg</div> <div class="given-names">M.V.</div></div>,  <div class="name"><div class="surname">Jenney</div> <div class="given-names">F.E.</div></div>,  <div class="name"><div class="surname">Cui</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Adams</div> <div class="given-names">M.W.W.</div></div>. </div><div class="article-title">Rubrerythrin from the hyperthermophilic archaeon <em>Pyrococcus furiosus</em> is a rubredoxin-dependent, iron-containing peroxidase</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2004</div>, vol. <div class="volume">186</div> (pg. <div class="fpage">7888</div>-<div class="lpage">95</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rubrerythrin%20from%20the%20hyperthermophilic%20archaeon%20Pyrococcus%20furiosus%20is%20a%20rubredoxin-dependent%2C%20iron-containing%20peroxidase&amp;author=M.V.%20Weinberg&amp;author=F.E.%20Jenney&amp;author=X.%20Cui&amp;author=M.W.W.%20Adams&amp;publication_year=2004&amp;journal=J.%20Bacteriol.&amp;volume=186&amp;pages=7888-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.186.23.7888-7895.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.186.23.7888-7895.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.186.23.7888-7895.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15547260" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C67" class="ref-content "><div class="label -label">67</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wildschut</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Lang</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Voordouw</div> <div class="given-names">J.K.</div></div>,  <div class="name"><div class="surname">Voordouw</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Rubredoxin:oxygen oxidoreductase enhances survival of <em>Desulfovibrio vulgaris</em> Hildenborough under microaerophilic conditions</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2006</div>, vol. <div class="volume">188</div> (pg. <div class="fpage">6253</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rubredoxin%3Aoxygen%20oxidoreductase%20enhances%20survival%20of%20Desulfovibrio%20vulgaris%20Hildenborough%20under%20microaerophilic%20conditions&amp;author=J.D.%20Wildschut&amp;author=R.M.%20Lang&amp;author=J.K.%20Voordouw&amp;author=G.%20Voordouw&amp;publication_year=2006&amp;journal=J.%20Bacteriol.&amp;volume=188&amp;pages=6253-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.00425-06" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.00425-06" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.00425-06"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16923892" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ005C68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ005C68" class="ref-content "><div class="label -label">68</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bhaya</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Takahashi</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Grossman</div> <div class="given-names">A.R.</div></div>. </div><div class="article-title">Light regulation of type IV pilus-dependent motility by chemosensor-like elements in <em>Synechocystis</em> PCC6803</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2001</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">7540</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Light%20regulation%20of%20type%20IV%20pilus-dependent%20motility%20by%20chemosensor-like%20elements%20in%20Synechocystis%20PCC6803&amp;author=D.%20Bhaya&amp;author=A.%20Takahashi&amp;author=A.R.%20Grossman&amp;publication_year=2001&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=7540-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.131201098" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.131201098" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.131201098"> </span></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Naotake Ogasawara</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2010. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=353305&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq005/2/dsq005-dsq005supp.pdf?Expires=2147483647&amp;Signature=IhivZJGZ1FwNfOoRyEY-v3U8QSR3N57HAb-jByqu5ZeuFb8CrS9ySIyufVEblvFcMYb49vpRGkpzOHVXUpIA5qlH~IHI3U65zYplbOAIqHquB6S5bUZ7vhPwJ1u8O6Y9HDuPeCfcNfOWw8KjDpi5ZiRTkJcI1YPycPlrv0p2bJGRHWdvKugKYx8YLg72PwKcgXduL5eaWe15ykrCxKvCIAEFNbLoTLu3vPHD9tpHq5jAL0VsdXB0LDOx2GJOA3PKDW1O326CWKs79wfjmB555xsc99jdDs9MyU1gGS75zaVyAYTCdmnCTOE5dMwBa2ELbBb1ZfyqOUgRVQp9Pt8RKA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">256</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000279411300001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">22</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsq005"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="353305"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000279411300001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f17%2f3%2f123%2f353305">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000279411300001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (22)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F17%2F3%2F123%2F353305" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/353305'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Bacterial Lifestyle in a Deep-sea Hydrothermal Vent Chimney Revealed by the Genome Sequence of the Thermophilic Bacterium Deferribacter desulfuricans SSM1" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsq005");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f17%2f3%2f123%2f353305">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

