

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/2/179/2960054","@type":"ScholarlyArticle","name":"The draft genome of  Ruellia speciosa  (Beautiful Wild Petunia: Acanthaceae)","datePublished":"2017-01-27","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2017-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw054","keywords":["anthocyanin","Asterid","evolution","Ka/Ks","phylogenetic"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/2/179/2960054","author":[{"name":"Zhuang, Yongbin","affiliation":"1 Department of Ecology and Evolutionary Biology, University of Colorado, UCB 334, Boulder, CO 80309, USA   2 Museum of Natural History, University of Colorado, UCB 350, Boulder, CO 80309, USA","@type":"Person"},{"name":"Tripp, Erin A.","affiliation":"1 Department of Ecology and Evolutionary Biology, University of Colorado, UCB 334, Boulder, CO 80309, USA   2 Museum of Natural History, University of Colorado, UCB 350, Boulder, CO 80309, USA","@type":"Person"}],"description":"Abstract.  The genus Ruellia (Wild Petunias; Acanthaceae) is characterized by an enormous diversity of floral shapes and colours manifested among closely relate","pageStart":"179","pageEnd":"192","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/2/2/m_dnares_24_2cover.png?Expires=2147483647&Signature=C8sLyIyfepNN-Oy5BR2xLoVJLx2W1jQBt~48a2lQ~mjys3EWTAKP5k2kV~t7b8sG~Qk1B4iVdz8BINFP5bR-Hc0gvwqekGHM8qavh8kDaLAfN~qNxrDj7NNeLHJ-IURMi5v1ATu23u5wB01wP27BHknbA3W9JgGnq6AVGPqeLfhK7AJcTzOYzKpGYwaedh5crkYXYHp~9GCcHuTjoInf3sKaYC2Ew-HnuA-yIL45j8ocXrNQuIxJV8rFXwx1Tt0b7I1ZJg09R4-7m-GZB9vPgdehgAXUMUNQ4gDatBY22xUt9COqnlb7RZKNUXFDo7Qe0LNJgSHwNUOJvsh-SLpXkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"The draft genome of  Ruellia speciosa  (Beautiful Wild Petunia: Acanthaceae)","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/2/2/m_dnares_24_2cover.png?Expires=2147483647&Signature=C8sLyIyfepNN-Oy5BR2xLoVJLx2W1jQBt~48a2lQ~mjys3EWTAKP5k2kV~t7b8sG~Qk1B4iVdz8BINFP5bR-Hc0gvwqekGHM8qavh8kDaLAfN~qNxrDj7NNeLHJ-IURMi5v1ATu23u5wB01wP27BHknbA3W9JgGnq6AVGPqeLfhK7AJcTzOYzKpGYwaedh5crkYXYHp~9GCcHuTjoInf3sKaYC2Ew-HnuA-yIL45j8ocXrNQuIxJV8rFXwx1Tt0b7I1ZJg09R4-7m-GZB9vPgdehgAXUMUNQ4gDatBY22xUt9COqnlb7RZKNUXFDo7Qe0LNJgSHwNUOJvsh-SLpXkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="draft genome of  Ruellia speciosa  (Beautiful Wild Petunia: Acanthaceae)" />
    <meta property="og:description" content="Abstract.  The genus Ruellia (Wild Petunias; Acanthaceae) is characterized by an enormous diversity of floral shapes and colours manifested among closely relate" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/2/179/2960054" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-01-27" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Zhuang, Yongbin" /><meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Colorado, UCB 334, Boulder, CO 80309, USA" /><meta name="citation_author_institution" content="Museum of Natural History, University of Colorado, UCB 350, Boulder, CO 80309, USA" /><meta name="citation_author" content="Tripp, Erin A." /><meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Colorado, UCB 334, Boulder, CO 80309, USA" /><meta name="citation_author_institution" content="Museum of Natural History, University of Colorado, UCB 350, Boulder, CO 80309, USA" /><meta name="citation_title" content="The draft genome of Ruellia speciosa (Beautiful Wild Petunia: Acanthaceae)" /><meta name="citation_firstpage" content="179" /><meta name="citation_lastpage" content="192" /><meta name="citation_doi" content="10.1093/dnares/dsw054" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2017/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="Stevens P.  2001 , Angiosperm phylogeny website. Version 12, July 2012 [and more or less continuously updated since]. " /><meta name="citation_reference" content="Funk V., Susanna A., Stuessy T., Bayer R. (Editors). 2009 , Systematics, evolution and biogeography of the Compositae, International Association of Plant Taxonomy, Vienna, Austria. " /><meta name="citation_reference" content="citation_title=PlantGDB: a resource for comparative plant genomics,;  citation_journal_title=Nucleic Acids Res;  citation_year=2008;  citation_volume=36;  citation_pages=D959-65; " /><meta name="citation_reference" content="citation_title=The Solanum commersonii genome sequence provides insights into adaptation to stress conditions and genome evolution of wild potato relatives,;  citation_journal_title=Plant Cell;  citation_year=2015;  citation_volume=27;  citation_pages=954-68; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of eggplant (Solanum melongena L.): the representative Solanum species indigenous to the old world;  citation_journal_title=DNA Res;  citation_year=2014;  citation_volume=21;  citation_pages=649-60; " /><meta name="citation_reference" content="citation_title=The tomato genome sequence provides insights into fleshy fruit evolution; citation_author=Tomato Genome Consortium;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=485;  citation_pages=635-41; " /><meta name="citation_reference" content="citation_title=Genome sequence and analysis of the tuber crop potato,; citation_author=Potato Genome Sequencing Consortium;  citation_journal_title=Nature;  citation_year=2011;  citation_volume=475;  citation_pages=189-95; " /><meta name="citation_reference" content="citation_title=Reference genomes and transcriptomes of Nicotiana sylvestris and Nicotiana tomentosiformis,;  citation_journal_title=Genome Biol;  citation_year=2013;  citation_volume=14;  citation_pages=1" /><meta name="citation_reference" content="citation_title=A draft genome sequence of Nicotiana benthamiana to enhance molecular plant-microbe biology research,;  citation_journal_title=Mol. Plant Microbe Interact;  citation_year=2012;  citation_volume=25;  citation_pages=1523-30; " /><meta name="citation_reference" content="citation_title=Genome sequence of the hot pepper provides insights into the evolution of pungency in Capsicum species,;  citation_journal_title=Nat. Genet;  citation_year=2014;  citation_volume=46;  citation_pages=270-8; " /><meta name="citation_reference" content="citation_title=Whole-genome sequencing of cultivated and wild peppers provides insights into Capsicum domestication and specialization,;  citation_journal_title=Proc. Natl. Acad. Sci;  citation_year=2014;  citation_volume=111;  citation_pages=5135-40; " /><meta name="citation_reference" content="citation_title=Insight into the evolution of the Solanaceae from the parental genomes of Petunia hybrida,;  citation_journal_title=Nat. Plants;  citation_year=2016;  citation_volume=2;  citation_pages=16074" /><meta name="citation_reference" content="citation_title=Fine-scale variation in meiotic recombination in Mimulus inferred from population shotgun sequencing,;  citation_journal_title=Proc. Natl. Acad. Sci;  citation_year=2013;  citation_volume=110;  citation_pages=19478-82; " /><meta name="citation_reference" content="citation_title=Architecture and evolution of a minute plant genome,;  citation_journal_title=Nature;  citation_year=2013;  citation_volume=498;  citation_pages=94-8; " /><meta name="citation_reference" content="citation_title=Evolutionary relationships within the species-rich genus Ruellia (Acanthaceae);  citation_journal_title=Syst. Botany;  citation_year=2007;  citation_volume=32;  citation_pages=628-49; " /><meta name="citation_reference" content="citation_title=Taxonomic revision of Ruellia section Chiropterophila (Acanthaceae): a lineage of rare and endemic species from Mexico,;  citation_journal_title=Syst. Botany;  citation_year=2010;  citation_volume=35;  citation_pages=629-61; " /><meta name="citation_reference" content="citation_title=Is floral specialization an evolutionary dead‐end? Pollination system transitions in Ruellia (Acanthaceae),;  citation_journal_title=Evolution;  citation_year=2008;  citation_volume=62;  citation_pages=1712-37; " /><meta name="citation_reference" content="citation_title=Time-calibrated phylogenies of hummingbirds and hummingbird-pollinated plants reject a hypothesis of diffuse co-evolution,;  citation_journal_title=Aliso. Discipline · Botany;  citation_year=2013;  citation_volume=31;  citation_pages=89-103; " /><meta name="citation_reference" content="citation_title=BD DACSDiVA Option, BD Biosciences, Pharmingen; Becton, Dickson and Company;  citation_journal_title=White Paper;  citation_year=2002;  citation_pages=1-22; " /><meta name="citation_reference" content="citation_title=The MaSuRCA genome assembler,;  citation_journal_title=Bioinformatics;  citation_year=2013;  citation_volume=29;  citation_pages=2669-77; " /><meta name="citation_reference" content="citation_title=SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler,;  citation_journal_title=Gigascience;  citation_year=2012;  citation_volume=1;  citation_pages=1" /><meta name="citation_reference" content="citation_title=ABySS: a parallel assembler for short read sequence data,;  citation_journal_title=Genome Res;  citation_year=2009;  citation_volume=19;  citation_pages=1117-23; " /><meta name="citation_reference" content="citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=2114-2120; " /><meta name="citation_reference" content="citation_title=Scaffolding pre-assembled contigs using SSPACE,;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=578-9; " /><meta name="citation_reference" content="citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs;  citation_journal_title=Bioinformatics;  citation_year=2015;  citation_volume=31;  citation_pages=3210-3212; " /><meta name="citation_reference" content="citation_title=The rise and fall of the Phytophthora infestans lineage that triggered the Irish potato famine,;  citation_journal_title=Elife;  citation_year=2013;  citation_volume=2;  citation_pages=e00731" /><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows–Wheeler transform,;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; " /><meta name="citation_reference" content="citation_title=The sequence alignment/map format and SAMtools,;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; " /><meta name="citation_reference" content="citation_title=EvoPipes. net: bioinformatic tools for ecological and evolutionary genomics;  citation_journal_title=Evol. Bioinform;  citation_year=2010;  citation_volume=6;  citation_pages=143" /><meta name="citation_reference" content="citation_title=PAML 4: phylogenetic analysis by maximum likelihood,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2007;  citation_volume=24;  citation_pages=1586-91; " /><meta name="citation_reference" content="citation_title=SiZer for exploration of structures in curves,;  citation_journal_title=J. Am. Stat. Assoc;  citation_year=1999;  citation_volume=94;  citation_pages=807-23; " /><meta name="citation_reference" content="citation_title=A fast, lock-free approach for efficient parallel counting of occurrences of k-mers,;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=764-70; " /><meta name="citation_reference" content="citation_title=MAKER-P: a tool kit for the rapid creation, management, and quality control of plant genome annotations,;  citation_journal_title=Plant Physiol;  citation_year=2014;  citation_volume=164;  citation_pages=513-24; " /><meta name="citation_reference" content="citation_title=Repbase Update, a database of eukaryotic repetitive elements,;  citation_journal_title=Cytogenet. Genome Res;  citation_year=2005;  citation_volume=110;  citation_pages=462-7; " /><meta name="citation_reference" content="Smit A.F., Hubley R., Green P.  1996 , RepeatMasker. http://www.repeatmasker.org. (06 Decemebr 2016, date last accessed). " /><meta name="citation_reference" content="citation_title=Gene finding in novel genomes;  citation_journal_title=BMC Bioinformatics;  citation_year=2004;  citation_volume=5;  citation_pages=1" /><meta name="citation_reference" content="citation_title=Gene prediction with a hidden Markov model and a new intron submodel,;  citation_journal_title=Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=ii215-25; " /><meta name="citation_reference" content="citation_title=PLAZA: a comparative genomics resource to study gene and genome evolution in plants,;  citation_journal_title=Plant Cell;  citation_year=2009;  citation_volume=21;  citation_pages=3718-31; " /><meta name="citation_reference" content="citation_title=Extensive error in the number of genes inferred from draft genome assemblies,;  citation_journal_title=PLoS Comput. Biol;  citation_year=2014;  citation_volume=10;  citation_pages=e1003998" /><meta name="citation_reference" content="citation_title=Genome sequencing of herb Tulsi (Ocimum tenuiflorum) unravels key genes behind its strong medicinal properties,;  citation_journal_title=BMC Plant Biol;  citation_year=2015;  citation_volume=15;  citation_pages=1" /><meta name="citation_reference" content="citation_title=GMATo: a novel tool for the identification and analysis of microsatellites in large genomes,;  citation_journal_title=Bioinformation;  citation_year=2013;  citation_volume=9;  citation_pages=541-4; " /><meta name="citation_reference" content="citation_title=A comprehensive characterization of simple sequence repeats in pepper genomes provides valuable resources for marker development in Capsicum,;  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=6;  citation_pages=18919" /><meta name="citation_reference" content="citation_title=BLAST+: architecture and applications,;  citation_journal_title=BMC Bioinformatics;  citation_year=2009;  citation_volume=10;  citation_pages=1" /><meta name="citation_reference" content="citation_title=The SWISS-PROT protein knowledgebase and its supplement TrEMBL in 2003,;  citation_journal_title=Nucleic Acids Res;  citation_year=2003;  citation_volume=31;  citation_pages=365-70; " /><meta name="citation_reference" content="citation_title=UniProt: a hub for protein information; citation_author=UniProt Consortium;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=43;  citation_pages=D204-D212; " /><meta name="citation_reference" content="citation_title=The Gene Ontology (GO) database and informatics resource; citation_author=Gene Ontology Consortium;  citation_journal_title=Nucleic Acids Res;  citation_year=2004;  citation_volume=32;  citation_pages=D258-61; " /><meta name="citation_reference" content="citation_title=UniProtKB/Swiss-Prot: The manually annotated section of the UniProt KnowledgeBase;  citation_journal_title=Methods Mol. Biol;  citation_year=2007;  citation_volume=406;  citation_pages=89-112; " /><meta name="citation_reference" content="citation_title=AgriGO: a GO analysis toolkit for the agricultural community;  citation_journal_title=Nucleic Acids Res;  citation_year=2010;  citation_volume=38;  citation_pages=W64-W70; " /><meta name="citation_reference" content="citation_title=WEGO: a web tool for plotting GO annotations,;  citation_journal_title=Nucleic Acids Res;  citation_year=2006;  citation_volume=34;  citation_pages=W293-7; " /><meta name="citation_reference" content="citation_title=OrthoDB: a hierarchical catalog of animal, fungal and bacterial orthologs,;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=D358-65; " /><meta name="citation_reference" content="citation_title=Unique features of the loblolly pine (Pinus taeda L.) megagenome revealed through sequence annotation,;  citation_journal_title=Genetics;  citation_year=2014;  citation_volume=196;  citation_pages=891-909; " /><meta name="citation_reference" content="citation_title=CD-HIT: a fast program for clustering and comparing large sets of protein or nucleotide sequences,;  citation_journal_title=Bioinformatics;  citation_year=2006;  citation_volume=22;  citation_pages=1658-9; " /><meta name="citation_reference" content="citation_title=Proteinortho: detection of (co-) orthologs in large-scale analysis,;  citation_journal_title=BMC Bioinformatics;  citation_year=2011;  citation_volume=12;  citation_pages=1" /><meta name="citation_reference" content="citation_title=MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform,;  citation_journal_title=Nucleic Acids Res;  citation_year=2002;  citation_volume=30;  citation_pages=3059-66; " /><meta name="citation_reference" content="citation_publisher=Wiley-Liss, New York;  citation_title=Molecular Zoology: Advances, Strategies, and Protocols;  citation_year=1996;  citation_pages=19-45; " /><meta name="citation_reference" content="citation_title=An improved general amino acid replacement matrix,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2008;  citation_volume=25;  citation_pages=1307-20; " /><meta name="citation_reference" content="citation_title=MEGA6: molecular evolutionary genetics analysis version 6.0,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2013;  citation_volume=30;  citation_pages=2725-9; " /><meta name="citation_reference" content="citation_title=A simple covarion‐based approach to analyse nucleotide substitution rates,;  citation_journal_title=J. Evol. Biol;  citation_year=2002;  citation_volume=15;  citation_pages=588-94; " /><meta name="citation_reference" content="citation_title=The Arabidopsis information resource (TAIR): a model organism database providing a centralized, curated gateway to Arabidopsis biology, research materials and community,;  citation_journal_title=Nucleic Acids Res;  citation_year=2003;  citation_volume=31;  citation_pages=224-8; " /><meta name="citation_reference" content="citation_title=Pfam: clans, web tools and services,;  citation_journal_title=Nucleic Acids Res;  citation_year=2006;  citation_volume=34;  citation_pages=D247-51; " /><meta name="citation_reference" content="citation_title=A guard-cell-specific MYB transcription factor regulates stomatal movements and plant drought tolerance,;  citation_journal_title=Curr. Biol;  citation_year=2005;  citation_volume=15;  citation_pages=1196-200; " /><meta name="citation_reference" content="citation_title=Overexpression of the wounding‐responsive gene AtMYB15 activates the shikimate pathway in Arabidopsis,;  citation_journal_title=J. Integr. Plant Biol;  citation_year=2006;  citation_volume=48;  citation_pages=1084-95; " /><meta name="citation_reference" content="citation_title=The transcription factor HIG1/MYB51 regulates indolic glucosinolate biosynthesis in Arabidopsis thaliana,;  citation_journal_title=Plant J;  citation_year=2007;  citation_volume=50;  citation_pages=886-901; " /><meta name="citation_reference" content="citation_title=Omics-based identification of Arabidopsis Myb transcription factors regulating aliphatic glucosinolate biosynthesis,;  citation_journal_title=Proc. Natl. Acad. Sci;  citation_year=2007;  citation_volume=104;  citation_pages=6478-83; " /><meta name="citation_reference" content="citation_title=HAG2/MYB76 and HAG3/MYB29 exert a specific and coordinated control on the regulation of aliphatic glucosinolate biosynthesis in Arabidopsis thaliana,;  citation_journal_title=New Phytol;  citation_year=2008;  citation_volume=177;  citation_pages=627-42; " /><meta name="citation_reference" content="citation_title=Analysis of the grape MYB R2R3 subfamily reveals expanded wine quality-related clades and conserved gene structure organization across Vitis and Arabidopsis genomes,;  citation_journal_title=BMC Plant Biol;  citation_year=2008;  citation_volume=8;  citation_pages=83" /><meta name="citation_reference" content="citation_title=MYB72 is required in early signaling steps of rhizobacteria-induced systemic resistance in Arabidopsis,;  citation_journal_title=Plant Physiol;  citation_year=2008;  citation_volume=146;  citation_pages=1293-304; " /><meta name="citation_reference" content="citation_title=Arabidopsis MYB30 is a direct target of BES1 and cooperates with BES1 to regulate brassinosteroid‐induced gene expression,;  citation_journal_title=Plant J;  citation_year=2009;  citation_volume=58;  citation_pages=275-86; " /><meta name="citation_reference" content="citation_title=MYB72, a node of convergence in induced systemic resistance triggered by a fungal and a bacterial beneficial microbe,;  citation_journal_title=Plant Biol;  citation_year=2009;  citation_volume=11;  citation_pages=90-6; " /><meta name="citation_reference" content="citation_title=Auxin homeostasis during lateral root development under drought condition,;  citation_journal_title=Plant Signal. Behav;  citation_year=2009;  citation_volume=4;  citation_pages=1002-4; " /><meta name="citation_reference" content="citation_title=Characterization of Arabidopsis MYB transcription factor gene AtMYB17 and its possible regulation by LEAFY and AGL15,;  citation_journal_title=J. Genet. Genomics;  citation_year=2009;  citation_volume=36;  citation_pages=99-107; " /><meta name="citation_reference" content="citation_title=MYB transcription factors in Arabidopsis,;  citation_journal_title=Trends Plant Sci;  citation_year=2010;  citation_volume=15;  citation_pages=573-81; " /><meta name="citation_reference" content="citation_title=MYB transcription factor genes as regulators for plant responses: an overview,;  citation_journal_title=Physiol. Mol. Biol. Plants;  citation_year=2013;  citation_volume=19;  citation_pages=307-21; " /><meta name="citation_reference" content="citation_title=Novel bioresources for studies of Brassica oleracea: identification of a kale MYB transcription factor responsible for glucosinolate production,;  citation_journal_title=Plant Biotech. J;  citation_year=2013;  citation_volume=11;  citation_pages=1017-27; " /><meta name="citation_reference" content="citation_title=AtMYB14 regulates cold tolerance in Arabidopsis,;  citation_journal_title=Plant Mol. Biol. Rep;  citation_year=2013;  citation_volume=31;  citation_pages=87-97; " /><meta name="citation_reference" content="citation_title=A chemical complementation approach reveals genes and interactions of flavonoids with other pathways,;  citation_journal_title=Plant J;  citation_year=2013;  citation_volume=74;  citation_pages=383-97; " /><meta name="citation_reference" content="citation_title=AtMYB93 is a novel negative regulator of lateral root development in Arabidopsis,;  citation_journal_title=New Phytol;  citation_year=2014;  citation_volume=203;  citation_pages=1194-207; " /><meta name="citation_reference" content="citation_title=The role of R2R3MYB transcription factors in plant stress tolerance;  citation_journal_title=J. Anim. Plant Sci;  citation_year=2014;  citation_volume=24;  citation_pages=1821-33; " /><meta name="citation_reference" content="citation_title=Fragment assignment in the cloud with eXpress-D,;  citation_journal_title=BMC Bioinformatics;  citation_year=2013;  citation_volume=14;  citation_pages=1" /><meta name="citation_reference" content="citation_title=gplots: various R programming tools for plotting data;  citation_journal_title=R package version;  citation_year=2009;  citation_pages=2" /><meta name="citation_reference" content="citation_title=BLAT—the BLAST-like alignment tool,;  citation_journal_title=Genome Res;  citation_year=2002;  citation_volume=12;  citation_pages=656-64; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs,;  citation_journal_title=Nucleic Acids Res;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2,;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=357-9; " /><meta name="citation_reference" content="Bushnell B.  2016 , BBMap short read aligner. http://sourceforge.net/projects/bbmap (17 June 2016, date last accessed). " /><meta name="citation_reference" content="citation_title=SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing,;  citation_journal_title=J. Comp. Biol;  citation_year=2012;  citation_volume=19;  citation_pages=455-77; " /><meta name="citation_reference" content="citation_title=CpGAVAS, an integrated web server for the annotation, visualization, analysis, and GenBank submission of completely sequenced chloroplast genome sequences,;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=715" /><meta name="citation_reference" content="citation_title=The draft genome of Primula veris yields insights into the molecular basis of heterostyly,;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=1" /><meta name="citation_reference" content="citation_title=The amaranth genome: genome, transcriptome, and physical map assembly,;  citation_journal_title=Plant Genome;  citation_year=2016;  citation_volume=9; " /><meta name="citation_reference" content="citation_title=Why assembling plant genome sequences is so challenging,;  citation_journal_title=Biology;  citation_year=2012;  citation_volume=1;  citation_pages=439-59; " /><meta name="citation_reference" content="citation_title=The first 50 plant genomes,;  citation_journal_title=Plant Genome;  citation_year=2013;  citation_volume=6; " /><meta name="citation_reference" content="citation_title=The draft genome and transcriptome of Cannabis sativa,;  citation_journal_title=Genome Biol;  citation_year=2011;  citation_volume=12;  citation_pages=1" /><meta name="citation_reference" content="citation_title=De novo genome assembly of the economically important weed horseweed using integrated data from multiple sequencing platforms,;  citation_journal_title=Plant Physiol;  citation_year=2014;  citation_volume=166;  citation_pages=1241-54; " /><meta name="citation_reference" content="citation_title=The first draft genome of the aquatic model plant Lemna minor opens the route for future stress physiology research and biotechnological applications,;  citation_journal_title=Biotechnol. Biofuels;  citation_year=2015;  citation_volume=8;  citation_pages=1" /><meta name="citation_reference" content="citation_title=Draft genome sequence of the rubber tree Hevea brasiliensis,;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=75" /><meta name="citation_reference" content="citation_title=Aegilops tauschii draft genome sequence reveals a gene repertoire for wheat adaptation,;  citation_journal_title=Nature;  citation_year=2013;  citation_volume=496;  citation_pages=91-5; " /><meta name="citation_reference" content="citation_title=Draft genome of the wheat A-genome progenitor Triticum urartu,;  citation_journal_title=Nature;  citation_year=2013;  citation_volume=496;  citation_pages=87-90; " /><meta name="citation_reference" content="citation_title=The evolutionary consequences of polyploidy,;  citation_journal_title=Cell;  citation_year=2007;  citation_volume=131;  citation_pages=452-62; " /><meta name="citation_reference" content="citation_title=TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions,;  citation_journal_title=Genome Biol;  citation_year=2013;  citation_volume=14;  citation_pages=1" /><meta name="citation_reference" content="citation_title=A survey of best practices for RNA-seq data analysis,;  citation_journal_title=Genome Biol;  citation_year=2016;  citation_volume=17;  citation_pages=1" /><meta name="citation_reference" content="citation_title=Sexually-dimorphic targeting of functionally-related genes in COPD,;  citation_journal_title=BMC Syst. Biol;  citation_year=2014;  citation_volume=8;  citation_pages=118" /><meta name="citation_reference" content="citation_title=Genome comparison using Gene Ontology (GO) with statistical testing;  citation_journal_title=BMC Bioinformatics;  citation_year=2006;  citation_volume=7;  citation_pages=1" /><meta name="citation_reference" content="citation_title=Repetitive DNA elements as a major component of plant genomes,;  citation_journal_title=Ann. Bot;  citation_year=1998;  citation_volume=82;  citation_pages=45-55; " /><meta name="citation_reference" content="citation_title=Analysis of the bread wheat genome using whole-genome shotgun sequencing,;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=491;  citation_pages=705-10; " /><meta name="citation_reference" content="citation_title=Repetitive sequences in plant nuclear DNA: types, distribution, evolution and function,;  citation_journal_title=Genomics Proteomics Bioinformatics;  citation_year=2014;  citation_volume=12;  citation_pages=164-71; " /><meta name="citation_reference" content="citation_title=Microsatellites as tools for genetic diversity analysis;  citation_title=Genet Divers Microorganisms;  citation_year=2012;  citation_volume=6;  citation_pages=149-170; " /><meta name="citation_reference" content="citation_title=A review of microsatellite markers and their applications in rice breeding programs to improve blast disease resistance,;  citation_journal_title=Int. J. Mol. Sci;  citation_year=2013;  citation_volume=14;  citation_pages=22499-528; " /><meta name="citation_reference" content="citation_title=Unravelling the genome of Holy basil: an “incomparable” “elixir of life” of traditional Indian medicine,;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=1" /><meta name="citation_reference" content="citation_title=Why are bird-visited flowers predominantly red?,;  citation_journal_title=Evolution;  citation_year=1972;  citation_volume=26;  citation_pages=674" /><meta name="citation_reference" content="citation_title=Adaptive significance of flower color and inter-trait correlations in an Ipomopsis hybrid zone,;  citation_journal_title=Evolution;  citation_year=1998;  citation_pages=1293-303; " /><meta name="citation_reference" content="citation_title=Floral isolation between Aquilegia formosa and Aquilegia pubescens,;  citation_journal_title=Proc. R. Soc. Lond;  citation_year=1999;  citation_volume=266;  citation_pages=2247-52; " /><meta name="citation_reference" content="citation_title=Allele substitution at a flower colour locus produces a pollinator shift in monkeyflowers,;  citation_journal_title=Nature;  citation_year=2003;  citation_volume=426;  citation_pages=176-8; " /><meta name="citation_reference" content="citation_title=Flower color microevolution in wild radish: evolutionary response to pollinator‐mediated selection,;  citation_journal_title=Am. Nat;  citation_year=2005;  citation_volume=165;  citation_pages=225-37; " /><meta name="citation_reference" content="citation_title=Flavanols and anthocyanins in cardiovascular health: a review of current evidence,;  citation_journal_title=Int. J. Mol. Sci;  citation_year=2010;  citation_volume=11;  citation_pages=1679-703; " /><meta name="citation_reference" content="citation_title=Expansion and subfunctionalisation of flavonoid 3′, 5′-hydroxylases in the grapevine lineage,;  citation_journal_title=BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=562" /><meta name="citation_reference" content="citation_title=Ecological transition predictably associated with gene degeneration,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2015;  citation_volume=32;  citation_pages=347-54; " /><meta name="citation_reference" content="citation_title=A phylogenetic examination of the primary anthocyanin production pathway of the Plantae,;  citation_journal_title=Bot. Stud;  citation_year=2014;  citation_volume=55;  citation_pages=1" /><meta name="citation_reference" content="citation_title=Maize anthocyanin regulatory gene pl is a duplicate of c1 that functions in the plant,;  citation_journal_title=Plant Cell;  citation_year=1993;  citation_volume=5;  citation_pages=1795-805; " /><meta name="citation_reference" content="citation_title=Multiple evolution of flavonoid 3′, 5′-hydroxylase,;  citation_journal_title=Planta;  citation_year=2015;  citation_volume=242;  citation_pages=561-73; " /><meta name="citation_reference" content="citation_publisher=Springer, New York, ;  citation_title=The Science of Flavonoids;  citation_year=2006;  citation_pages=71-95; " /><meta name="citation_reference" content="citation_title=Alteration of a single amino acid changes the substrate specificity of dihydroflavonol 4‐reductase,;  citation_journal_title=Plant J;  citation_year=2001;  citation_volume=25;  citation_pages=325-33; " /><meta name="citation_reference" content="citation_title=Functional evolution of an anthocyanin pathway enzyme during a flower color transition,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2013;  citation_volume=30;  citation_pages=602-12; " /><meta name="citation_reference" content="citation_title=Dihydroflavonol 4-reductase genes encode enzymes with contrasting substrate specificity and show divergent gene expression profiles in Fragaria Species,;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e112707" /><meta name="citation_reference" content="citation_title=Analysis of bHLH and MYB domain proteins: species‐specific regulatory differences are caused by divergent evolution of target anthocyanin genes;  citation_journal_title=Plant J;  citation_year=1998;  citation_volume=13;  citation_pages=475-88; " /><meta name="citation_reference" content="citation_title=Molecular analysis of the anthocyanin2 gene of Petunia and its role in the evolution of flower color,;  citation_journal_title=Plant Cell;  citation_year=1999;  citation_volume=11;  citation_pages=1433-44; " /><meta name="citation_reference" content="citation_title=Members of an R2R3‐MYB transcription factor family in Petunia are developmentally and environmentally regulated to control complex floral and vegetative pigmentation patterning,;  citation_journal_title=Plant J;  citation_year=2011;  citation_volume=65;  citation_pages=771-84; " /><meta name="citation_reference" content="citation_title=Complete chloroplast genome sequences of Praxelis (Eupatorium catarium Veldkamp), an important invasive species,;  citation_journal_title=Gene;  citation_year=2014;  citation_volume=549;  citation_pages=58-69; " /><meta name="citation_reference" content="citation_title=Rates and patterns of chloroplast DNA evolution,;  citation_journal_title=Proc. Natl. Acad. Sci;  citation_year=1994;  citation_volume=91;  citation_pages=6795-801; " /><meta name="citation_reference" content="citation_title=The complete chloroplast genome of banana (Musa acuminata, Zingiberales): insight into plastid monocotyledon evolution,;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e67350" /><meta name="citation_reference" content="citation_title=Phylogenetic and evolutionary implications of complete chloroplast genome sequences of four early-diverging angiosperms: Buxus (Buxaceae), Chloranthus (Chloranthaceae), Dioscorea (Dioscoreaceae), and Illicium (Schisandraceae),;  citation_journal_title=Mol. Phylogenet. Evol;  citation_year=2007;  citation_volume=45;  citation_pages=547-63; " /><meta name="citation_reference" content="citation_title=Adaptive radiation of reproductive characteristics in angiosperms, I: pollination mechanisms;  citation_journal_title=Annu. Rev. Ecol. Evol. Syst;  citation_year=1970;  citation_volume=1;  citation_pages=307-26; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/2/179/13688899/dsw054.pdf" /><meta name="description" content="Abstract.  The genus Ruellia (Wild Petunias; Acanthaceae) is characterized by an enormous diversity of floral shapes and colours manifested among closely relate" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/2/179/2960054" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/2/179/2960054" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/2/179/2960054");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw054';
    </script>
<title>
	draft genome of Ruellia speciosa (Beautiful Wild Petunia: Acanthaceae) | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2960054" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BeqKAjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWmh1YW5nLCBZb25nYmluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBFY29sb2d5IGFuZCBFdm9sdXRpb25hcnkgQmlvbG9neSwgVW5pdmVyc2l0eSBvZiBDb2xvcmFkbywgVUNCIDMzNCwgQm91bGRlciwgQ08gODAzMDksIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik11c2V1bSBvZiBOYXR1cmFsIEhpc3RvcnksIFVuaXZlcnNpdHkgb2YgQ29sb3JhZG8sIFVDQiAzNTAsIEJvdWxkZXIsIENPIDgwMzA5LCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUcmlwcCwgRXJpbiBBLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgRWNvbG9neSBhbmQgRXZvbHV0aW9uYXJ5IEJpb2xvZ3ksIFVuaXZlcnNpdHkgb2YgQ29sb3JhZG8sIFVDQiAzMzQsIEJvdWxkZXIsIENPIDgwMzA5LCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNdXNldW0gb2YgTmF0dXJhbCBIaXN0b3J5LCBVbml2ZXJzaXR5IG9mIENvbG9yYWRvLCBVQ0IgMzUwLCBCb3VsZGVyLCBDTyA4MDMwOSwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJUaGUgZHJhZnQgZ2Vub21lIG9mIFJ1ZWxsaWEgc3BlY2lvc2EgKEJlYXV0aWZ1bCBXaWxkIFBldHVuaWE6IEFjYW50aGFjZWFlKSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjE3OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTkyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN3MDU0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTcvMDQvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJTdGV2ZW5zIFAuICAyMDAxICwgQW5naW9zcGVybSBwaHlsb2dlbnkgd2Vic2l0ZS4gVmVyc2lvbiAxMiwgSnVseSAyMDEyIFthbmQgbW9yZSBvciBsZXNzIGNvbnRpbnVvdXNseSB1cGRhdGVkIHNpbmNlXS4gIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iRnVuayBWLiwgU3VzYW5uYSBBLiwgU3R1ZXNzeSBULiwgQmF5ZXIgUi4gKEVkaXRvcnMpLiAyMDA5ICwgU3lzdGVtYXRpY3MsIGV2b2x1dGlvbiBhbmQgYmlvZ2VvZ3JhcGh5IG9mIHRoZSBDb21wb3NpdGFlLCBJbnRlcm5hdGlvbmFsIEFzc29jaWF0aW9uIG9mIFBsYW50IFRheG9ub215LCBWaWVubmEsIEF1c3RyaWEuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsYW50R0RCOiBhIHJlc291cmNlIGZvciBjb21wYXJhdGl2ZSBwbGFudCBnZW5vbWljcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9RDk1OS02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFNvbGFudW0gY29tbWVyc29uaWkgZ2Vub21lIHNlcXVlbmNlIHByb3ZpZGVzIGluc2lnaHRzIGludG8gYWRhcHRhdGlvbiB0byBzdHJlc3MgY29uZGl0aW9ucyBhbmQgZ2Vub21lIGV2b2x1dGlvbiBvZiB3aWxkIHBvdGF0byByZWxhdGl2ZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9OTU0LTY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EcmFmdCBnZW5vbWUgc2VxdWVuY2Ugb2YgZWdncGxhbnQgKFNvbGFudW0gbWVsb25nZW5hIEwuKTogdGhlIHJlcHJlc2VudGF0aXZlIFNvbGFudW0gc3BlY2llcyBpbmRpZ2Vub3VzIHRvIHRoZSBvbGQgd29ybGQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz02NDktNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB0b21hdG8gZ2Vub21lIHNlcXVlbmNlIHByb3ZpZGVzIGluc2lnaHRzIGludG8gZmxlc2h5IGZydWl0IGV2b2x1dGlvbjsgY2l0YXRpb25fYXV0aG9yPVRvbWF0byBHZW5vbWUgQ29uc29ydGl1bTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ4NTsgIGNpdGF0aW9uX3BhZ2VzPTYzNS00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNlIGFuZCBhbmFseXNpcyBvZiB0aGUgdHViZXIgY3JvcCBwb3RhdG8sOyBjaXRhdGlvbl9hdXRob3I9UG90YXRvIEdlbm9tZSBTZXF1ZW5jaW5nIENvbnNvcnRpdW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00NzU7ICBjaXRhdGlvbl9wYWdlcz0xODktOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZmVyZW5jZSBnZW5vbWVzIGFuZCB0cmFuc2NyaXB0b21lcyBvZiBOaWNvdGlhbmEgc3lsdmVzdHJpcyBhbmQgTmljb3RpYW5hIHRvbWVudG9zaWZvcm1pcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZHJhZnQgZ2Vub21lIHNlcXVlbmNlIG9mIE5pY290aWFuYSBiZW50aGFtaWFuYSB0byBlbmhhbmNlIG1vbGVjdWxhciBwbGFudC1taWNyb2JlIGJpb2xvZ3kgcmVzZWFyY2gsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFBsYW50IE1pY3JvYmUgSW50ZXJhY3Q7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xNTIzLTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIGhvdCBwZXBwZXIgcHJvdmlkZXMgaW5zaWdodHMgaW50byB0aGUgZXZvbHV0aW9uIG9mIHB1bmdlbmN5IGluIENhcHNpY3VtIHNwZWNpZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQ2OyAgY2l0YXRpb25fcGFnZXM9MjcwLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdob2xlLWdlbm9tZSBzZXF1ZW5jaW5nIG9mIGN1bHRpdmF0ZWQgYW5kIHdpbGQgcGVwcGVycyBwcm92aWRlcyBpbnNpZ2h0cyBpbnRvIENhcHNpY3VtIGRvbWVzdGljYXRpb24gYW5kIHNwZWNpYWxpemF0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTExMTsgIGNpdGF0aW9uX3BhZ2VzPTUxMzUtNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluc2lnaHQgaW50byB0aGUgZXZvbHV0aW9uIG9mIHRoZSBTb2xhbmFjZWFlIGZyb20gdGhlIHBhcmVudGFsIGdlbm9tZXMgb2YgUGV0dW5pYSBoeWJyaWRhLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBQbGFudHM7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTE2MDc0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmluZS1zY2FsZSB2YXJpYXRpb24gaW4gbWVpb3RpYyByZWNvbWJpbmF0aW9uIGluIE1pbXVsdXMgaW5mZXJyZWQgZnJvbSBwb3B1bGF0aW9uIHNob3RndW4gc2VxdWVuY2luZyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTA7ICBjaXRhdGlvbl9wYWdlcz0xOTQ3OC04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXJjaGl0ZWN0dXJlIGFuZCBldm9sdXRpb24gb2YgYSBtaW51dGUgcGxhbnQgZ2Vub21lLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQ5ODsgIGNpdGF0aW9uX3BhZ2VzPTk0LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbmFyeSByZWxhdGlvbnNoaXBzIHdpdGhpbiB0aGUgc3BlY2llcy1yaWNoIGdlbnVzIFJ1ZWxsaWEgKEFjYW50aGFjZWFlKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U3lzdC4gQm90YW55OyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9NjI4LTQ5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYXhvbm9taWMgcmV2aXNpb24gb2YgUnVlbGxpYSBzZWN0aW9uIENoaXJvcHRlcm9waGlsYSAoQWNhbnRoYWNlYWUpOiBhIGxpbmVhZ2Ugb2YgcmFyZSBhbmQgZW5kZW1pYyBzcGVjaWVzIGZyb20gTWV4aWNvLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U3lzdC4gQm90YW55OyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9NjI5LTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JcyBmbG9yYWwgc3BlY2lhbGl6YXRpb24gYW4gZXZvbHV0aW9uYXJ5IGRlYWTigJBlbmQ/IFBvbGxpbmF0aW9uIHN5c3RlbSB0cmFuc2l0aW9ucyBpbiBSdWVsbGlhIChBY2FudGhhY2VhZSksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Fdm9sdXRpb247ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NjI7ICBjaXRhdGlvbl9wYWdlcz0xNzEyLTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaW1lLWNhbGlicmF0ZWQgcGh5bG9nZW5pZXMgb2YgaHVtbWluZ2JpcmRzIGFuZCBodW1taW5nYmlyZC1wb2xsaW5hdGVkIHBsYW50cyByZWplY3QgYSBoeXBvdGhlc2lzIG9mIGRpZmZ1c2UgY28tZXZvbHV0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QWxpc28uIERpc2NpcGxpbmUgwrcgQm90YW55OyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9ODktMTAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CRCBEQUNTRGlWQSBPcHRpb24sIEJEIEJpb3NjaWVuY2VzLCBQaGFybWluZ2VuOyBCZWN0b24sIERpY2tzb24gYW5kIENvbXBhbnk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVdoaXRlIFBhcGVyOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fcGFnZXM9MS0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIE1hU3VSQ0EgZ2Vub21lIGFzc2VtYmxlciw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9MjY2OS03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U09BUGRlbm92bzI6IGFuIGVtcGlyaWNhbGx5IGltcHJvdmVkIG1lbW9yeS1lZmZpY2llbnQgc2hvcnQtcmVhZCBkZSBub3ZvIGFzc2VtYmxlciw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdpZ2FzY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0xIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QUJ5U1M6IGEgcGFyYWxsZWwgYXNzZW1ibGVyIGZvciBzaG9ydCByZWFkIHNlcXVlbmNlIGRhdGEsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MTExNy0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJpbW1vbWF0aWM6IGEgZmxleGlibGUgdHJpbW1lciBmb3IgSWxsdW1pbmEgc2VxdWVuY2UgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0yMTE0LTIxMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNjYWZmb2xkaW5nIHByZS1hc3NlbWJsZWQgY29udGlncyB1c2luZyBTU1BBQ0UsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTU3OC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CVVNDTzogYXNzZXNzaW5nIGdlbm9tZSBhc3NlbWJseSBhbmQgYW5ub3RhdGlvbiBjb21wbGV0ZW5lc3Mgd2l0aCBzaW5nbGUtY29weSBvcnRob2xvZ3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MzIxMC0zMjEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcmlzZSBhbmQgZmFsbCBvZiB0aGUgUGh5dG9waHRob3JhIGluZmVzdGFucyBsaW5lYWdlIHRoYXQgdHJpZ2dlcmVkIHRoZSBJcmlzaCBwb3RhdG8gZmFtaW5lLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RWxpZmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPWUwMDczMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3QgYW5kIGFjY3VyYXRlIHNob3J0IHJlYWQgYWxpZ25tZW50IHdpdGggQnVycm93c+KAk1doZWVsZXIgdHJhbnNmb3JtLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xNzU0LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc2VxdWVuY2UgYWxpZ25tZW50L21hcCBmb3JtYXQgYW5kIFNBTXRvb2xzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0yMDc4LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b1BpcGVzLiBuZXQ6IGJpb2luZm9ybWF0aWMgdG9vbHMgZm9yIGVjb2xvZ2ljYWwgYW5kIGV2b2x1dGlvbmFyeSBnZW5vbWljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXZvbC4gQmlvaW5mb3JtOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0xNDMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QQU1MIDQ6IHBoeWxvZ2VuZXRpYyBhbmFseXNpcyBieSBtYXhpbXVtIGxpa2VsaWhvb2QsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xNTg2LTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaVplciBmb3IgZXhwbG9yYXRpb24gb2Ygc3RydWN0dXJlcyBpbiBjdXJ2ZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBBbS4gU3RhdC4gQXNzb2M7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9OTQ7ICBjaXRhdGlvbl9wYWdlcz04MDctMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZmFzdCwgbG9jay1mcmVlIGFwcHJvYWNoIGZvciBlZmZpY2llbnQgcGFyYWxsZWwgY291bnRpbmcgb2Ygb2NjdXJyZW5jZXMgb2Ygay1tZXJzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz03NjQtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1BS0VSLVA6IGEgdG9vbCBraXQgZm9yIHRoZSByYXBpZCBjcmVhdGlvbiwgbWFuYWdlbWVudCwgYW5kIHF1YWxpdHkgY29udHJvbCBvZiBwbGFudCBnZW5vbWUgYW5ub3RhdGlvbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE2NDsgIGNpdGF0aW9uX3BhZ2VzPTUxMy0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVwYmFzZSBVcGRhdGUsIGEgZGF0YWJhc2Ugb2YgZXVrYXJ5b3RpYyByZXBldGl0aXZlIGVsZW1lbnRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3l0b2dlbmV0LiBHZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTQ2Mi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJTbWl0IEEuRi4sIEh1YmxleSBSLiwgR3JlZW4gUC4gIDE5OTYgLCBSZXBlYXRNYXNrZXIuIGh0dHA6Ly93d3cucmVwZWF0bWFza2VyLm9yZy4gKDA2IERlY2VtZWJyIDIwMTYsIGRhdGUgbGFzdCBhY2Nlc3NlZCkuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgZmluZGluZyBpbiBub3ZlbCBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIHByZWRpY3Rpb24gd2l0aCBhIGhpZGRlbiBNYXJrb3YgbW9kZWwgYW5kIGEgbmV3IGludHJvbiBzdWJtb2RlbCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9aWkyMTUtMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBMQVpBOiBhIGNvbXBhcmF0aXZlIGdlbm9taWNzIHJlc291cmNlIHRvIHN0dWR5IGdlbmUgYW5kIGdlbm9tZSBldm9sdXRpb24gaW4gcGxhbnRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTM3MTgtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4dGVuc2l2ZSBlcnJvciBpbiB0aGUgbnVtYmVyIG9mIGdlbmVzIGluZmVycmVkIGZyb20gZHJhZnQgZ2Vub21lIGFzc2VtYmxpZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIENvbXB1dC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAzOTk4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNpbmcgb2YgaGVyYiBUdWxzaSAoT2NpbXVtIHRlbnVpZmxvcnVtKSB1bnJhdmVscyBrZXkgZ2VuZXMgYmVoaW5kIGl0cyBzdHJvbmcgbWVkaWNpbmFsIHByb3BlcnRpZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgUGxhbnQgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HTUFUbzogYSBub3ZlbCB0b29sIGZvciB0aGUgaWRlbnRpZmljYXRpb24gYW5kIGFuYWx5c2lzIG9mIG1pY3Jvc2F0ZWxsaXRlcyBpbiBsYXJnZSBnZW5vbWVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpb247ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTU0MS00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbXByZWhlbnNpdmUgY2hhcmFjdGVyaXphdGlvbiBvZiBzaW1wbGUgc2VxdWVuY2UgcmVwZWF0cyBpbiBwZXBwZXIgZ2Vub21lcyBwcm92aWRlcyB2YWx1YWJsZSByZXNvdXJjZXMgZm9yIG1hcmtlciBkZXZlbG9wbWVudCBpbiBDYXBzaWN1bSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gUmVwOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0xODkxOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJMQVNUKzogYXJjaGl0ZWN0dXJlIGFuZCBhcHBsaWNhdGlvbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFNXSVNTLVBST1QgcHJvdGVpbiBrbm93bGVkZ2ViYXNlIGFuZCBpdHMgc3VwcGxlbWVudCBUckVNQkwgaW4gMjAwMyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MzY1LTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbmlQcm90OiBhIGh1YiBmb3IgcHJvdGVpbiBpbmZvcm1hdGlvbjsgY2l0YXRpb25fYXV0aG9yPVVuaVByb3QgQ29uc29ydGl1bTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz1EMjA0LUQyMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBHZW5lIE9udG9sb2d5IChHTykgZGF0YWJhc2UgYW5kIGluZm9ybWF0aWNzIHJlc291cmNlOyBjaXRhdGlvbl9hdXRob3I9R2VuZSBPbnRvbG9neSBDb25zb3J0aXVtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPUQyNTgtNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVuaVByb3RLQi9Td2lzcy1Qcm90OiBUaGUgbWFudWFsbHkgYW5ub3RhdGVkIHNlY3Rpb24gb2YgdGhlIFVuaVByb3QgS25vd2xlZGdlQmFzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWV0aG9kcyBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDA2OyAgY2l0YXRpb25fcGFnZXM9ODktMTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZ3JpR086IGEgR08gYW5hbHlzaXMgdG9vbGtpdCBmb3IgdGhlIGFncmljdWx0dXJhbCBjb21tdW5pdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9VzY0LVc3MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V0VHTzogYSB3ZWIgdG9vbCBmb3IgcGxvdHRpbmcgR08gYW5ub3RhdGlvbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPVcyOTMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3J0aG9EQjogYSBoaWVyYXJjaGljYWwgY2F0YWxvZyBvZiBhbmltYWwsIGZ1bmdhbCBhbmQgYmFjdGVyaWFsIG9ydGhvbG9ncyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9RDM1OC02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VW5pcXVlIGZlYXR1cmVzIG9mIHRoZSBsb2Jsb2xseSBwaW5lIChQaW51cyB0YWVkYSBMLikgbWVnYWdlbm9tZSByZXZlYWxlZCB0aHJvdWdoIHNlcXVlbmNlIGFubm90YXRpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xOTY7ICBjaXRhdGlvbl9wYWdlcz04OTEtOTA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DRC1ISVQ6IGEgZmFzdCBwcm9ncmFtIGZvciBjbHVzdGVyaW5nIGFuZCBjb21wYXJpbmcgbGFyZ2Ugc2V0cyBvZiBwcm90ZWluIG9yIG51Y2xlb3RpZGUgc2VxdWVuY2VzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0xNjU4LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb3RlaW5vcnRobzogZGV0ZWN0aW9uIG9mIChjby0pIG9ydGhvbG9ncyBpbiBsYXJnZS1zY2FsZSBhbmFseXNpcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NQUZGVDogYSBub3ZlbCBtZXRob2QgZm9yIHJhcGlkIG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCBiYXNlZCBvbiBmYXN0IEZvdXJpZXIgdHJhbnNmb3JtLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0zMDU5LTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9V2lsZXktTGlzcywgTmV3IFlvcms7ICBjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgWm9vbG9neTogQWR2YW5jZXMsIFN0cmF0ZWdpZXMsIGFuZCBQcm90b2NvbHM7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl9wYWdlcz0xOS00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gaW1wcm92ZWQgZ2VuZXJhbCBhbWlubyBhY2lkIHJlcGxhY2VtZW50IG1hdHJpeCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTEzMDctMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1FR0E2OiBtb2xlY3VsYXIgZXZvbHV0aW9uYXJ5IGdlbmV0aWNzIGFuYWx5c2lzIHZlcnNpb24gNi4wLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MjcyNS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNpbXBsZSBjb3ZhcmlvbuKAkGJhc2VkIGFwcHJvYWNoIHRvIGFuYWx5c2UgbnVjbGVvdGlkZSBzdWJzdGl0dXRpb24gcmF0ZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBFdm9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9NTg4LTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQXJhYmlkb3BzaXMgaW5mb3JtYXRpb24gcmVzb3VyY2UgKFRBSVIpOiBhIG1vZGVsIG9yZ2FuaXNtIGRhdGFiYXNlIHByb3ZpZGluZyBhIGNlbnRyYWxpemVkLCBjdXJhdGVkIGdhdGV3YXkgdG8gQXJhYmlkb3BzaXMgYmlvbG9neSwgcmVzZWFyY2ggbWF0ZXJpYWxzIGFuZCBjb21tdW5pdHksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTIyNC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QZmFtOiBjbGFucywgd2ViIHRvb2xzIGFuZCBzZXJ2aWNlcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9RDI0Ny01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBndWFyZC1jZWxsLXNwZWNpZmljIE1ZQiB0cmFuc2NyaXB0aW9uIGZhY3RvciByZWd1bGF0ZXMgc3RvbWF0YWwgbW92ZW1lbnRzIGFuZCBwbGFudCBkcm91Z2h0IHRvbGVyYW5jZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xMTk2LTIwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3ZlcmV4cHJlc3Npb24gb2YgdGhlIHdvdW5kaW5n4oCQcmVzcG9uc2l2ZSBnZW5lIEF0TVlCMTUgYWN0aXZhdGVzIHRoZSBzaGlraW1hdGUgcGF0aHdheSBpbiBBcmFiaWRvcHNpcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEludGVnci4gUGxhbnQgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT00ODsgIGNpdGF0aW9uX3BhZ2VzPTEwODQtOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB0cmFuc2NyaXB0aW9uIGZhY3RvciBISUcxL01ZQjUxIHJlZ3VsYXRlcyBpbmRvbGljIGdsdWNvc2lub2xhdGUgYmlvc3ludGhlc2lzIGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgSjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT01MDsgIGNpdGF0aW9uX3BhZ2VzPTg4Ni05MDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9taWNzLWJhc2VkIGlkZW50aWZpY2F0aW9uIG9mIEFyYWJpZG9wc2lzIE15YiB0cmFuc2NyaXB0aW9uIGZhY3RvcnMgcmVndWxhdGluZyBhbGlwaGF0aWMgZ2x1Y29zaW5vbGF0ZSBiaW9zeW50aGVzaXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTA0OyAgY2l0YXRpb25fcGFnZXM9NjQ3OC04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SEFHMi9NWUI3NiBhbmQgSEFHMy9NWUIyOSBleGVydCBhIHNwZWNpZmljIGFuZCBjb29yZGluYXRlZCBjb250cm9sIG9uIHRoZSByZWd1bGF0aW9uIG9mIGFsaXBoYXRpYyBnbHVjb3Npbm9sYXRlIGJpb3N5bnRoZXNpcyBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5ldyBQaHl0b2w7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTc3OyAgY2l0YXRpb25fcGFnZXM9NjI3LTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiB0aGUgZ3JhcGUgTVlCIFIyUjMgc3ViZmFtaWx5IHJldmVhbHMgZXhwYW5kZWQgd2luZSBxdWFsaXR5LXJlbGF0ZWQgY2xhZGVzIGFuZCBjb25zZXJ2ZWQgZ2VuZSBzdHJ1Y3R1cmUgb3JnYW5pemF0aW9uIGFjcm9zcyBWaXRpcyBhbmQgQXJhYmlkb3BzaXMgZ2Vub21lcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBQbGFudCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz04MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ZQjcyIGlzIHJlcXVpcmVkIGluIGVhcmx5IHNpZ25hbGluZyBzdGVwcyBvZiByaGl6b2JhY3RlcmlhLWluZHVjZWQgc3lzdGVtaWMgcmVzaXN0YW5jZSBpbiBBcmFiaWRvcHNpcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTQ2OyAgY2l0YXRpb25fcGFnZXM9MTI5My0zMDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFyYWJpZG9wc2lzIE1ZQjMwIGlzIGEgZGlyZWN0IHRhcmdldCBvZiBCRVMxIGFuZCBjb29wZXJhdGVzIHdpdGggQkVTMSB0byByZWd1bGF0ZSBicmFzc2lub3N0ZXJvaWTigJBpbmR1Y2VkIGdlbmUgZXhwcmVzc2lvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEo7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NTg7ICBjaXRhdGlvbl9wYWdlcz0yNzUtODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ZQjcyLCBhIG5vZGUgb2YgY29udmVyZ2VuY2UgaW4gaW5kdWNlZCBzeXN0ZW1pYyByZXNpc3RhbmNlIHRyaWdnZXJlZCBieSBhIGZ1bmdhbCBhbmQgYSBiYWN0ZXJpYWwgYmVuZWZpY2lhbCBtaWNyb2JlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTkwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUF1eGluIGhvbWVvc3Rhc2lzIGR1cmluZyBsYXRlcmFsIHJvb3QgZGV2ZWxvcG1lbnQgdW5kZXIgZHJvdWdodCBjb25kaXRpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBTaWduYWwuIEJlaGF2OyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0xMDAyLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgQXJhYmlkb3BzaXMgTVlCIHRyYW5zY3JpcHRpb24gZmFjdG9yIGdlbmUgQXRNWUIxNyBhbmQgaXRzIHBvc3NpYmxlIHJlZ3VsYXRpb24gYnkgTEVBRlkgYW5kIEFHTDE1LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gR2VuZXQuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9OTktMTA3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NWUIgdHJhbnNjcmlwdGlvbiBmYWN0b3JzIGluIEFyYWJpZG9wc2lzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIFBsYW50IFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTU3My04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TVlCIHRyYW5zY3JpcHRpb24gZmFjdG9yIGdlbmVzIGFzIHJlZ3VsYXRvcnMgZm9yIHBsYW50IHJlc3BvbnNlczogYW4gb3ZlcnZpZXcsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QaHlzaW9sLiBNb2wuIEJpb2wuIFBsYW50czsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTMwNy0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm92ZWwgYmlvcmVzb3VyY2VzIGZvciBzdHVkaWVzIG9mIEJyYXNzaWNhIG9sZXJhY2VhOiBpZGVudGlmaWNhdGlvbiBvZiBhIGthbGUgTVlCIHRyYW5zY3JpcHRpb24gZmFjdG9yIHJlc3BvbnNpYmxlIGZvciBnbHVjb3Npbm9sYXRlIHByb2R1Y3Rpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBCaW90ZWNoLiBKOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MTAxNy0yNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXRNWUIxNCByZWd1bGF0ZXMgY29sZCB0b2xlcmFuY2UgaW4gQXJhYmlkb3BzaXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBNb2wuIEJpb2wuIFJlcDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTg3LTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNoZW1pY2FsIGNvbXBsZW1lbnRhdGlvbiBhcHByb2FjaCByZXZlYWxzIGdlbmVzIGFuZCBpbnRlcmFjdGlvbnMgb2YgZmxhdm9ub2lkcyB3aXRoIG90aGVyIHBhdGh3YXlzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgSjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT03NDsgIGNpdGF0aW9uX3BhZ2VzPTM4My05NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXRNWUI5MyBpcyBhIG5vdmVsIG5lZ2F0aXZlIHJlZ3VsYXRvciBvZiBsYXRlcmFsIHJvb3QgZGV2ZWxvcG1lbnQgaW4gQXJhYmlkb3BzaXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OZXcgUGh5dG9sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTIwMzsgIGNpdGF0aW9uX3BhZ2VzPTExOTQtMjA3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcm9sZSBvZiBSMlIzTVlCIHRyYW5zY3JpcHRpb24gZmFjdG9ycyBpbiBwbGFudCBzdHJlc3MgdG9sZXJhbmNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBBbmltLiBQbGFudCBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xODIxLTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GcmFnbWVudCBhc3NpZ25tZW50IGluIHRoZSBjbG91ZCB3aXRoIGVYcHJlc3MtRCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ncGxvdHM6IHZhcmlvdXMgUiBwcm9ncmFtbWluZyB0b29scyBmb3IgcGxvdHRpbmcgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UiBwYWNrYWdlIHZlcnNpb247ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl9wYWdlcz0yIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QkxBVOKAlHRoZSBCTEFTVC1saWtlIGFsaWdubWVudCB0b29sLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTY1Ni02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2FwcGVkIEJMQVNUIGFuZCBQU0ktQkxBU1Q6IGEgbmV3IGdlbmVyYXRpb24gb2YgcHJvdGVpbiBkYXRhYmFzZSBzZWFyY2ggcHJvZ3JhbXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTMzODktNDAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GYXN0IGdhcHBlZC1yZWFkIGFsaWdubWVudCB3aXRoIEJvd3RpZSAyLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0zNTctOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iQnVzaG5lbGwgQi4gIDIwMTYgLCBCQk1hcCBzaG9ydCByZWFkIGFsaWduZXIuIGh0dHA6Ly9zb3VyY2Vmb3JnZS5uZXQvcHJvamVjdHMvYmJtYXAgKDE3IEp1bmUgMjAxNiwgZGF0ZSBsYXN0IGFjY2Vzc2VkKS4gIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U1BBZGVzOiBhIG5ldyBnZW5vbWUgYXNzZW1ibHkgYWxnb3JpdGhtIGFuZCBpdHMgYXBwbGljYXRpb25zIHRvIHNpbmdsZS1jZWxsIHNlcXVlbmNpbmcsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBDb21wLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9NDU1LTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DcEdBVkFTLCBhbiBpbnRlZ3JhdGVkIHdlYiBzZXJ2ZXIgZm9yIHRoZSBhbm5vdGF0aW9uLCB2aXN1YWxpemF0aW9uLCBhbmFseXNpcywgYW5kIEdlbkJhbmsgc3VibWlzc2lvbiBvZiBjb21wbGV0ZWx5IHNlcXVlbmNlZCBjaGxvcm9wbGFzdCBnZW5vbWUgc2VxdWVuY2VzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9NzE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGRyYWZ0IGdlbm9tZSBvZiBQcmltdWxhIHZlcmlzIHlpZWxkcyBpbnNpZ2h0cyBpbnRvIHRoZSBtb2xlY3VsYXIgYmFzaXMgb2YgaGV0ZXJvc3R5bHksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgYW1hcmFudGggZ2Vub21lOiBnZW5vbWUsIHRyYW5zY3JpcHRvbWUsIGFuZCBwaHlzaWNhbCBtYXAgYXNzZW1ibHksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBHZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2h5IGFzc2VtYmxpbmcgcGxhbnQgZ2Vub21lIHNlcXVlbmNlcyBpcyBzbyBjaGFsbGVuZ2luZyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTQzOS01OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGZpcnN0IDUwIHBsYW50IGdlbm9tZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBHZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGRyYWZ0IGdlbm9tZSBhbmQgdHJhbnNjcmlwdG9tZSBvZiBDYW5uYWJpcyBzYXRpdmEsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZSBub3ZvIGdlbm9tZSBhc3NlbWJseSBvZiB0aGUgZWNvbm9taWNhbGx5IGltcG9ydGFudCB3ZWVkIGhvcnNld2VlZCB1c2luZyBpbnRlZ3JhdGVkIGRhdGEgZnJvbSBtdWx0aXBsZSBzZXF1ZW5jaW5nIHBsYXRmb3Jtcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTY2OyAgY2l0YXRpb25fcGFnZXM9MTI0MS01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGZpcnN0IGRyYWZ0IGdlbm9tZSBvZiB0aGUgYXF1YXRpYyBtb2RlbCBwbGFudCBMZW1uYSBtaW5vciBvcGVucyB0aGUgcm91dGUgZm9yIGZ1dHVyZSBzdHJlc3MgcGh5c2lvbG9neSByZXNlYXJjaCBhbmQgYmlvdGVjaG5vbG9naWNhbCBhcHBsaWNhdGlvbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW90ZWNobm9sLiBCaW9mdWVsczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyYWZ0IGdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgcnViYmVyIHRyZWUgSGV2ZWEgYnJhc2lsaWVuc2lzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9NzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZWdpbG9wcyB0YXVzY2hpaSBkcmFmdCBnZW5vbWUgc2VxdWVuY2UgcmV2ZWFscyBhIGdlbmUgcmVwZXJ0b2lyZSBmb3Igd2hlYXQgYWRhcHRhdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00OTY7ICBjaXRhdGlvbl9wYWdlcz05MS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EcmFmdCBnZW5vbWUgb2YgdGhlIHdoZWF0IEEtZ2Vub21lIHByb2dlbml0b3IgVHJpdGljdW0gdXJhcnR1LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQ5NjsgIGNpdGF0aW9uX3BhZ2VzPTg3LTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXZvbHV0aW9uYXJ5IGNvbnNlcXVlbmNlcyBvZiBwb2x5cGxvaWR5LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzE7ICBjaXRhdGlvbl9wYWdlcz00NTItNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRvcEhhdDI6IGFjY3VyYXRlIGFsaWdubWVudCBvZiB0cmFuc2NyaXB0b21lcyBpbiB0aGUgcHJlc2VuY2Ugb2YgaW5zZXJ0aW9ucywgZGVsZXRpb25zIGFuZCBnZW5lIGZ1c2lvbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHN1cnZleSBvZiBiZXN0IHByYWN0aWNlcyBmb3IgUk5BLXNlcSBkYXRhIGFuYWx5c2lzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0xIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2V4dWFsbHktZGltb3JwaGljIHRhcmdldGluZyBvZiBmdW5jdGlvbmFsbHktcmVsYXRlZCBnZW5lcyBpbiBDT1BELDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIFN5c3QuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTExOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBjb21wYXJpc29uIHVzaW5nIEdlbmUgT250b2xvZ3kgKEdPKSB3aXRoIHN0YXRpc3RpY2FsIHRlc3Rpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlcGV0aXRpdmUgRE5BIGVsZW1lbnRzIGFzIGEgbWFqb3IgY29tcG9uZW50IG9mIHBsYW50IGdlbm9tZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm4uIEJvdDsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT04MjsgIGNpdGF0aW9uX3BhZ2VzPTQ1LTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiB0aGUgYnJlYWQgd2hlYXQgZ2Vub21lIHVzaW5nIHdob2xlLWdlbm9tZSBzaG90Z3VuIHNlcXVlbmNpbmcsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDkxOyAgY2l0YXRpb25fcGFnZXM9NzA1LTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXBldGl0aXZlIHNlcXVlbmNlcyBpbiBwbGFudCBudWNsZWFyIEROQTogdHlwZXMsIGRpc3RyaWJ1dGlvbiwgZXZvbHV0aW9uIGFuZCBmdW5jdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9taWNzIFByb3Rlb21pY3MgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xNjQtNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3Jvc2F0ZWxsaXRlcyBhcyB0b29scyBmb3IgZ2VuZXRpYyBkaXZlcnNpdHkgYW5hbHlzaXM7ICBjaXRhdGlvbl90aXRsZT1HZW5ldCBEaXZlcnMgTWljcm9vcmdhbmlzbXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTE0OS0xNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcmV2aWV3IG9mIG1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnMgYW5kIHRoZWlyIGFwcGxpY2F0aW9ucyBpbiByaWNlIGJyZWVkaW5nIHByb2dyYW1zIHRvIGltcHJvdmUgYmxhc3QgZGlzZWFzZSByZXNpc3RhbmNlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW50LiBKLiBNb2wuIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTIyNDk5LTUyODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VW5yYXZlbGxpbmcgdGhlIGdlbm9tZSBvZiBIb2x5IGJhc2lsOiBhbiDigJxpbmNvbXBhcmFibGXigJ0g4oCcZWxpeGlyIG9mIGxpZmXigJ0gb2YgdHJhZGl0aW9uYWwgSW5kaWFuIG1lZGljaW5lLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdoeSBhcmUgYmlyZC12aXNpdGVkIGZsb3dlcnMgcHJlZG9taW5hbnRseSByZWQ/LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXZvbHV0aW9uOyAgY2l0YXRpb25feWVhcj0xOTcyOyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9Njc0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWRhcHRpdmUgc2lnbmlmaWNhbmNlIG9mIGZsb3dlciBjb2xvciBhbmQgaW50ZXItdHJhaXQgY29ycmVsYXRpb25zIGluIGFuIElwb21vcHNpcyBoeWJyaWQgem9uZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV2b2x1dGlvbjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3BhZ2VzPTEyOTMtMzAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GbG9yYWwgaXNvbGF0aW9uIGJldHdlZW4gQXF1aWxlZ2lhIGZvcm1vc2EgYW5kIEFxdWlsZWdpYSBwdWJlc2NlbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBSLiBTb2MuIExvbmQ7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MjY2OyAgY2l0YXRpb25fcGFnZXM9MjI0Ny01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWxsZWxlIHN1YnN0aXR1dGlvbiBhdCBhIGZsb3dlciBjb2xvdXIgbG9jdXMgcHJvZHVjZXMgYSBwb2xsaW5hdG9yIHNoaWZ0IGluIG1vbmtleWZsb3dlcnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDI2OyAgY2l0YXRpb25fcGFnZXM9MTc2LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZsb3dlciBjb2xvciBtaWNyb2V2b2x1dGlvbiBpbiB3aWxkIHJhZGlzaDogZXZvbHV0aW9uYXJ5IHJlc3BvbnNlIHRvIHBvbGxpbmF0b3LigJBtZWRpYXRlZCBzZWxlY3Rpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BbS4gTmF0OyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE2NTsgIGNpdGF0aW9uX3BhZ2VzPTIyNS0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmxhdmFub2xzIGFuZCBhbnRob2N5YW5pbnMgaW4gY2FyZGlvdmFzY3VsYXIgaGVhbHRoOiBhIHJldmlldyBvZiBjdXJyZW50IGV2aWRlbmNlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW50LiBKLiBNb2wuIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTE2NzktNzAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHBhbnNpb24gYW5kIHN1YmZ1bmN0aW9uYWxpc2F0aW9uIG9mIGZsYXZvbm9pZCAz4oCyLCA14oCyLWh5ZHJveHlsYXNlcyBpbiB0aGUgZ3JhcGV2aW5lIGxpbmVhZ2UsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz01NjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FY29sb2dpY2FsIHRyYW5zaXRpb24gcHJlZGljdGFibHkgYXNzb2NpYXRlZCB3aXRoIGdlbmUgZGVnZW5lcmF0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9MzQ3LTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHBoeWxvZ2VuZXRpYyBleGFtaW5hdGlvbiBvZiB0aGUgcHJpbWFyeSBhbnRob2N5YW5pbiBwcm9kdWN0aW9uIHBhdGh3YXkgb2YgdGhlIFBsYW50YWUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Cb3QuIFN0dWQ7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NTU7ICBjaXRhdGlvbl9wYWdlcz0xIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFpemUgYW50aG9jeWFuaW4gcmVndWxhdG9yeSBnZW5lIHBsIGlzIGEgZHVwbGljYXRlIG9mIGMxIHRoYXQgZnVuY3Rpb25zIGluIHRoZSBwbGFudCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTE3OTUtODA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdWx0aXBsZSBldm9sdXRpb24gb2YgZmxhdm9ub2lkIDPigLIsIDXigLItaHlkcm94eWxhc2UsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudGE7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjQyOyAgY2l0YXRpb25fcGFnZXM9NTYxLTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXIsIE5ldyBZb3JrLCA7ICBjaXRhdGlvbl90aXRsZT1UaGUgU2NpZW5jZSBvZiBGbGF2b25vaWRzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fcGFnZXM9NzEtOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsdGVyYXRpb24gb2YgYSBzaW5nbGUgYW1pbm8gYWNpZCBjaGFuZ2VzIHRoZSBzdWJzdHJhdGUgc3BlY2lmaWNpdHkgb2YgZGloeWRyb2ZsYXZvbm9sIDTigJByZWR1Y3Rhc2UsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzI1LTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5jdGlvbmFsIGV2b2x1dGlvbiBvZiBhbiBhbnRob2N5YW5pbiBwYXRod2F5IGVuenltZSBkdXJpbmcgYSBmbG93ZXIgY29sb3IgdHJhbnNpdGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTYwMi0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGloeWRyb2ZsYXZvbm9sIDQtcmVkdWN0YXNlIGdlbmVzIGVuY29kZSBlbnp5bWVzIHdpdGggY29udHJhc3Rpbmcgc3Vic3RyYXRlIHNwZWNpZmljaXR5IGFuZCBzaG93IGRpdmVyZ2VudCBnZW5lIGV4cHJlc3Npb24gcHJvZmlsZXMgaW4gRnJhZ2FyaWEgU3BlY2llcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1lMTEyNzA3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgYkhMSCBhbmQgTVlCIGRvbWFpbiBwcm90ZWluczogc3BlY2llc+KAkHNwZWNpZmljIHJlZ3VsYXRvcnkgZGlmZmVyZW5jZXMgYXJlIGNhdXNlZCBieSBkaXZlcmdlbnQgZXZvbHV0aW9uIG9mIHRhcmdldCBhbnRob2N5YW5pbiBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgSjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTQ3NS04ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGFuYWx5c2lzIG9mIHRoZSBhbnRob2N5YW5pbjIgZ2VuZSBvZiBQZXR1bmlhIGFuZCBpdHMgcm9sZSBpbiB0aGUgZXZvbHV0aW9uIG9mIGZsb3dlciBjb2xvciw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xNDMzLTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZW1iZXJzIG9mIGFuIFIyUjPigJBNWUIgdHJhbnNjcmlwdGlvbiBmYWN0b3IgZmFtaWx5IGluIFBldHVuaWEgYXJlIGRldmVsb3BtZW50YWxseSBhbmQgZW52aXJvbm1lbnRhbGx5IHJlZ3VsYXRlZCB0byBjb250cm9sIGNvbXBsZXggZmxvcmFsIGFuZCB2ZWdldGF0aXZlIHBpZ21lbnRhdGlvbiBwYXR0ZXJuaW5nLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgSjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02NTsgIGNpdGF0aW9uX3BhZ2VzPTc3MS04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxldGUgY2hsb3JvcGxhc3QgZ2Vub21lIHNlcXVlbmNlcyBvZiBQcmF4ZWxpcyAoRXVwYXRvcml1bSBjYXRhcml1bSBWZWxka2FtcCksIGFuIGltcG9ydGFudCBpbnZhc2l2ZSBzcGVjaWVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01NDk7ICBjaXRhdGlvbl9wYWdlcz01OC02OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmF0ZXMgYW5kIHBhdHRlcm5zIG9mIGNobG9yb3BsYXN0IEROQSBldm9sdXRpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2k7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9OTE7ICBjaXRhdGlvbl9wYWdlcz02Nzk1LTgwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNvbXBsZXRlIGNobG9yb3BsYXN0IGdlbm9tZSBvZiBiYW5hbmEgKE11c2EgYWN1bWluYXRhLCBaaW5naWJlcmFsZXMpOiBpbnNpZ2h0IGludG8gcGxhc3RpZCBtb25vY290eWxlZG9uIGV2b2x1dGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lNjczNTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlsb2dlbmV0aWMgYW5kIGV2b2x1dGlvbmFyeSBpbXBsaWNhdGlvbnMgb2YgY29tcGxldGUgY2hsb3JvcGxhc3QgZ2Vub21lIHNlcXVlbmNlcyBvZiBmb3VyIGVhcmx5LWRpdmVyZ2luZyBhbmdpb3NwZXJtczogQnV4dXMgKEJ1eGFjZWFlKSwgQ2hsb3JhbnRodXMgKENobG9yYW50aGFjZWFlKSwgRGlvc2NvcmVhIChEaW9zY29yZWFjZWFlKSwgYW5kIElsbGljaXVtIChTY2hpc2FuZHJhY2VhZSksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFBoeWxvZ2VuZXQuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz01NDctNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFkYXB0aXZlIHJhZGlhdGlvbiBvZiByZXByb2R1Y3RpdmUgY2hhcmFjdGVyaXN0aWNzIGluIGFuZ2lvc3Blcm1zLCBJOiBwb2xsaW5hdGlvbiBtZWNoYW5pc21zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIEVjb2wuIEV2b2wuIFN5c3Q7ICBjaXRhdGlvbl95ZWFyPTE5NzA7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTMwNy0yNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI0LzIvMTc5LzEzNjg4ODk5L2RzdzA1NC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgVGhlIGdlbnVzIFJ1ZWxsaWEgKFdpbGQgUGV0dW5pYXM7IEFjYW50aGFjZWFlKSBpcyBjaGFyYWN0ZXJpemVkIGJ5IGFuIGVub3Jtb3VzIGRpdmVyc2l0eSBvZiBmbG9yYWwgc2hhcGVzIGFuZCBjb2xvdXJzIG1hbmlmZXN0ZWQgYW1vbmcgY2xvc2VseSByZWxhdGUiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTI5NjAwNTQmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkOmP1dcd0rof4+4aUs07qKpE2YZ4=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f179%2f2960054" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f179%2f2960054" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=97104" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/2/2/m_dnares_24_2cover.png?Expires=2147483647&amp;Signature=C8sLyIyfepNN-Oy5BR2xLoVJLx2W1jQBt~48a2lQ~mjys3EWTAKP5k2kV~t7b8sG~Qk1B4iVdz8BINFP5bR-Hc0gvwqekGHM8qavh8kDaLAfN~qNxrDj7NNeLHJ-IURMi5v1ATu23u5wB01wP27BHknbA3W9JgGnq6AVGPqeLfhK7AJcTzOYzKpGYwaedh5crkYXYHp~9GCcHuTjoInf3sKaYC2Ew-HnuA-yIL45j8ocXrNQuIxJV8rFXwx1Tt0b7I1ZJg09R4-7m-GZB9vPgdehgAXUMUNQ4gDatBY22xUt9COqnlb7RZKNUXFDo7Qe0LNJgSHwNUOJvsh-SLpXkg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/2">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="78055039">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055039">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055041">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055041">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055046">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055046">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055069">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055069">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055087">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055087">4. Conclusion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055089">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055089">Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055091">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055091">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055099">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055099">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/2/169/2870503">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/2/193/2731026">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">The draft genome of <em>Ruellia speciosa</em> (Beautiful Wild Petunia: Acanthaceae)
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yongbin Zhuang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Ecology and Evolutionary Biology, University of Colorado, UCB 334, Boulder, CO 80309, USA</div><div class="aff"><div class="label -label">2</div>Museum of Natural History, University of Colorado, UCB 350, Boulder, CO 80309, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yongbin+Zhuang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhuang Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhuang Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yongbin Zhuang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Erin A. Tripp</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Ecology and Evolutionary Biology, University of Colorado, UCB 334, Boulder, CO 80309, USA</div><div class="aff"><div class="label -label">2</div>Museum of Natural History, University of Colorado, UCB 350, Boulder, CO 80309, USA</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw054-cor1" data-open="dsw054-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Email: <a href="mailto:erin.tripp@colorado.edu" target="_blank">erin.tripp@colorado.edu</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Erin+A.+Tripp">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tripp E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tripp E&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Erin A. Tripp</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw054-FM1"><div class="fn"><div class="footnote-content">Edited by Dr. Sachiko Isobe</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 2, 1 April 2017, Pages 179–192, <a href='https://doi.org/10.1093/dnares/dsw054'>https://doi.org/10.1093/dnares/dsw054</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">27 January 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">18 July 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">16 November 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">17 November 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2960054" href="/dnaresearch/article-pdf/24/2/179/13688899/dsw054.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yongbin Zhuang, Erin A. Tripp;  The draft genome of <em>Ruellia speciosa</em> (Beautiful Wild Petunia: Acanthaceae), <em>DNA Research</em>, Volume 24, Issue 2, 1 April 2017, Pages 179–192, <a href='https://doi.org/10.1093/dnares/dsw054'>https://doi.org/10.1093/dnares/dsw054</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2960054?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2960054?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2960054?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2960054?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2960054?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2960054?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2960054?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=The%20draft%20genome%20of%20%26lt%3Bem%26gt%3BRuellia%20speciosa%26lt%3B%2Fem%26gt%3B%20(Beautiful%20Wild%20Petunia%3A%20Acanthaceae)&amp;publicationDate=2017-01-27&amp;volumeNum=24&amp;issueNum=2&amp;author=Zhuang%2C%20Yongbin%3B%20Tripp%2C%20Erin%20A.&amp;startPage=179&amp;endPage=192&amp;contentId=10.1093%2Fdnares%2Fdsw054&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f179%2f2960054" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=78055039 id="78055039" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The genus <em>Ruellia</em> (Wild Petunias; Acanthaceae) is characterized by an enormous diversity of floral shapes and colours manifested among closely related species. Using Illumina platform, we reconstructed the draft genome of <em>Ruellia speciosa</em>, with a scaffold size of 1,021 Mb (or ∼1.02 Gb) and an N50 size of 17,908 bp, spanning ∼93% of the estimated genome (∼1.1 Gb). The draft assembly predicted 40,124 gene models and phylogenetic analyses of four key enzymes involved in anthocyanin colour production [flavanone 3-hydroxylase (F3H), flavonoid 3′-hydroxylase (F3′H), flavonoid 3′,5′-hydroxylase (F3′5′H), and dihydroflavonol 4-reductase (DFR)] found that most angiosperms here sampled harboured at least one copy of F3H, F3′H, and DFR. In contrast, fewer than one-half (but including <em>R. speciosa</em>) harboured a copy of F3′5′H, supporting observations that blue flowers and/or fruits, which this enzyme is required for, are less common among flowering plants. Ka/Ks analyses of duplicated copies of F3′H and DFR in <em>R. speciosa</em> suggested purifying selection in the former but detected evidence of positive selection in the latter. The genome sequence and annotation of <em>R. speciosa</em> represents only one of only four families sequenced in the large and important Asterid clade of flowering plants and, as such, will facilitate extensive future research on this diverse group, particularly with respect to floral evolution.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="anthocyanin">anthocyanin</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Asterid">Asterid</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="evolution">evolution</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Ka/Ks">Ka/Ks</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="phylogenetic">phylogenetic</a></div>                            <h2 scrollto-destination=78055041 id="78055041" class="section-title" data-legacy-id=SEC1>1. Introduction</h2>
<p>The Asterid clade is one of the most species-rich lineages of flowering plants, containing over a quarter of all known species of angiosperms (∼80,000 of 300,000). In this clade are four of the 10 most diverse families of flowering plants: Asteraceae (Sunflowers: ∼24,000 spp.), Rubiaceae (Coffee Family: ∼11,200 spp.), Lamiaceae (Mint Family: ∼7,200 spp.), and Acanthaceae (Acanthus Family: 4,200 spp.). Asterids are of tremendous economic and ecological significance, with cultivated and economically important members including coffee, mints (basil, oregano, rosemary, sage, thyme, lavender, teak), tomatoes and relatives (potatoes, tobacco, chilies, peppers, petunias), and carrots and relatives (carrots, cumin, cilantro, dill, fennel, parsley, celery), as well as blueberries, olives, sunflowers, sweet potatoes, snapdragons, dogwoods, ashes, milkweeds, and hollies.<a reveal-id="dsw054-B1" data-open="dsw054-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> Members of the Asterid clade furthermore dominate ecosystems worldwide.<a reveal-id="dsw054-B2" data-open="dsw054-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a></p><p>Despite a revolution in sequencing plant genomes over the last decade, there exists only a handful of plants in the Asterid clade with nuclear reference genomes (<a class="link link-uri" href="http://www.plantgdb.org/" target="_blank">http://www.plantgdb.org/</a>).<a reveal-id="dsw054-B3" data-open="dsw054-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> These genomes derive primarily from the tomato family (Solanaceae) in which nine species have been sequenced (<em>Solanum</em>, four species: ∼838–900 Mb;<a reveal-id="dsw054-B4 dsw054-B5 dsw054-B6 dsw054-B7" data-open="dsw054-B4 dsw054-B5 dsw054-B6 dsw054-B7" class="link link-ref link-reveal xref-bibr"><sup>4–7</sup></a><sup>,</sup><em>Nicotiana</em>, three species: ∼3 Gb;<a reveal-id="dsw054-B8" data-open="dsw054-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsw054-B9" data-open="dsw054-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><em>Capsicum</em>, three species: ∼3.48 Gb;<a reveal-id="dsw054-B10" data-open="dsw054-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsw054-B11" data-open="dsw054-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a>and <em>Petunia</em>, two species: ∼1.4 Gb<a reveal-id="dsw054-B12" data-open="dsw054-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a>). Beyond this, there exists only two additional reference genomes in the Asterid clade, only one of which is &gt;100 Mb in size: the monkeyflower <em>Mimulus guttatus</em>, ∼430 Mb;<a reveal-id="dsw054-B13" data-open="dsw054-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> the other derives from the insectivorous bladderwort <em>Utricularia gibba</em>, which is only ∼81.9 Mb.<a reveal-id="dsw054-B14" data-open="dsw054-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> Thus, only three of 100+ families that comprise Asterids have representative nuclear genomes.</p><div class="block-child-p">To build genomic resources for Asterids, we here contribute new nuclear genome reference from a fourth family in this clade: Acanthaceae. The genus <em>Ruellia</em> (Wild Petunias) contains ∼400 species that are distributed primarily in the Neotropics and Paleotropics. With very few exceptions, all species of <em>Ruellia</em> (∼50 species counted to date) are thought to be diploid and share a somatic chromosome count of <em>2n = 34</em> (reviewed by Tripp<a reveal-id="dsw054-B15" data-open="dsw054-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> and Tripp<a reveal-id="dsw054-B16" data-open="dsw054-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a>). <em>Ruellia</em> is marked by tremendous diversity in both flower shape and colour, with very closely related species (e.g. sister taxa) often divergent in floral form (<a reveal-id="dsw054-F1" data-open="dsw054-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>).<a reveal-id="dsw054-B17" data-open="dsw054-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> Flowers of <em>Ruellia</em> range from blue to purple, pink, red, green, yellow, and white and are pollinated by bees, butterflies, hawkmoths, bats, hummingbirds, and sunbirds.<a reveal-id="dsw054-B15" data-open="dsw054-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsw054-B18" data-open="dsw054-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> Owing to the floral diversity and species richness in the genus, <em>Ruellia</em> has potential as a future model system for floral evolution in angiosperms. <div data-id="dsw054-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/m_dsw054f1.png?Expires=2147483647&Signature=kF6BHngulmyCPYCNQyHezNY4VBmBNPiRflQE2HgotJh8uPis88gQmFfDRGDftjcVlcCrPZ4m1WtqOtpCRsehMPQudpyiTSEWZr9RlotCwEEbaWvmwuqQGtAeHGTBExo8ABblv6RbiGHIXqOZTfUYCtujrwsNrlFUbqubA5X9gSkIbjUsRecMnfAvpLGVzlvHmw5jhIU7aMyj4Z9fli1NKD5Z0j2wj~uqNzhIIpzjj4o-SwuachAl~D~AhcthsR~lAZKZmkVIev3wPuFbysjIyWxUc0unpMIAcXI5JDweHOzv1L2xuK7HKrXJlhp5dvKoFTIVVwh1SW-e1pg5czWpuQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Floral colour and floral shape diversity present within Ruellia. (A) R. speciosa. (B) Ruellia macrantha. (C) Ruellia longipedunculata. (D) Ruellia hirsuto-glandulosa. (E) Ruellia bourgaei. (F) Ruellia saccata. (G) Ruellia biolleyi. (H) R. breedlovei. (I) Ruellia californica. (J) Ruellia noctiflora." path-from-xml="dsw054f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055044/dsw054f1.tif" path-from-xml="dsw054f1.tif" target="_blank">View large</a><a section="78055044" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054f1.png?Expires=2147483647&Signature=xu8VX92OlxlC9mt1U4dKBWNU~HoQ5H~MJgnwxk1vVJ49Wn3uM6eB9TPTST70hG2IRmHZCE7TKNA9QwY3AVCCyZIF6tqJIXL10x2v1rTVGE1DrZpfe8yLESSpA~vEO4WiV57kdW3rbPmrADg5uXs2thUO76lLnWjOVQABcEhr1e--z~~TOQm5KMNxrxg8lRkMm08vxpBpjVTR~ZB5rV8XvT8nxFXaurWxwV6lZq9xZZ9Vf8zSp~klIphCmw5P5ggGc~IuSUt2sW8CJNORJcIFfSF1Mx-L~xgKrK01EbfoskA3qtIEeSA5G4HkWNatakOwDzzvho6lMUWJuFjnVTivzg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055044&ar=2960054&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw054f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Floral colour and floral shape diversity present within Ruellia. (A) <em>R. speciosa</em>. (B) <em>Ruellia macrantha</em>. (C) <em>Ruellia longipedunculata</em>. (D) <em>Ruellia hirsuto-glandulosa</em>. (E) <em>Ruellia bourgaei</em>. (F) <em>Ruellia saccata</em>. (G) <em>Ruellia biolleyi</em>. (H) <em>R. breedlovei</em>. (I) <em>Ruellia californica</em>. (J) <em>Ruellia noctiflora</em>.</p></div></div><div content-id="dsw054-F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/m_dsw054f1.png?Expires=2147483647&Signature=kF6BHngulmyCPYCNQyHezNY4VBmBNPiRflQE2HgotJh8uPis88gQmFfDRGDftjcVlcCrPZ4m1WtqOtpCRsehMPQudpyiTSEWZr9RlotCwEEbaWvmwuqQGtAeHGTBExo8ABblv6RbiGHIXqOZTfUYCtujrwsNrlFUbqubA5X9gSkIbjUsRecMnfAvpLGVzlvHmw5jhIU7aMyj4Z9fli1NKD5Z0j2wj~uqNzhIIpzjj4o-SwuachAl~D~AhcthsR~lAZKZmkVIev3wPuFbysjIyWxUc0unpMIAcXI5JDweHOzv1L2xuK7HKrXJlhp5dvKoFTIVVwh1SW-e1pg5czWpuQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Floral colour and floral shape diversity present within Ruellia. (A) R. speciosa. (B) Ruellia macrantha. (C) Ruellia longipedunculata. (D) Ruellia hirsuto-glandulosa. (E) Ruellia bourgaei. (F) Ruellia saccata. (G) Ruellia biolleyi. (H) R. breedlovei. (I) Ruellia californica. (J) Ruellia noctiflora." path-from-xml="dsw054f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055044/dsw054f1.tif" path-from-xml="dsw054f1.tif" target="_blank">View large</a><a section="78055044" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054f1.png?Expires=2147483647&Signature=xu8VX92OlxlC9mt1U4dKBWNU~HoQ5H~MJgnwxk1vVJ49Wn3uM6eB9TPTST70hG2IRmHZCE7TKNA9QwY3AVCCyZIF6tqJIXL10x2v1rTVGE1DrZpfe8yLESSpA~vEO4WiV57kdW3rbPmrADg5uXs2thUO76lLnWjOVQABcEhr1e--z~~TOQm5KMNxrxg8lRkMm08vxpBpjVTR~ZB5rV8XvT8nxFXaurWxwV6lZq9xZZ9Vf8zSp~klIphCmw5P5ggGc~IuSUt2sW8CJNORJcIFfSF1Mx-L~xgKrK01EbfoskA3qtIEeSA5G4HkWNatakOwDzzvho6lMUWJuFjnVTivzg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055044&ar=2960054&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw054f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Floral colour and floral shape diversity present within Ruellia. (A) <em>R. speciosa</em>. (B) <em>Ruellia macrantha</em>. (C) <em>Ruellia longipedunculata</em>. (D) <em>Ruellia hirsuto-glandulosa</em>. (E) <em>Ruellia bourgaei</em>. (F) <em>Ruellia saccata</em>. (G) <em>Ruellia biolleyi</em>. (H) <em>R. breedlovei</em>. (I) <em>Ruellia californica</em>. (J) <em>Ruellia noctiflora</em>.</p></div></div></div><p><em>Ruellia speciosa</em> (Beautiful Wild Petunia) is a pale yellow-flowered species native to a narrow portion of the southern Sierra Madre Oriental of Mexico and is nearly extinct in the wild due to habitat destruction.<a reveal-id="dsw054-B16" data-open="dsw054-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Because of its tiny population sizes and isolation from related species, the genome is expected to be highly homozygous and suitable for sequencing as a reference.</p>                            <h2 scrollto-destination=78055046 id="78055046" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=78055047 id="78055047" class="section-title" >2.1. DNA isolation, library preparation, and sequencing</h3>
<p>Plant material for this study was collected in the wild by E. Tripp and S. Acosta (voucher # 175, housed at the Duke University Herbarium; in living cultivation in the University of Colorado Greenhouses), in a small canyon of a mountain that lies within the city limits of Oaxaca City, Oaxaca, Mexico. Total genomic DNA was extracted from young leaf tissue of <em>R.</em><em>speciosa</em> using the DNeasy Plant Mini Kit (QIAGEN) following the manufacturer’s instructions. Presence of high molecular weight DNA was confirmed by 1% agarose-gel electrophoresis stained with SYBR Safe (Invitrogen). DNA concentration was quantified using a Qubit 2.0 Fluorimeter spectrophotometer (Life Technology). DNA that passed quality control was used for mate-pair library preparation followed by whole-genome shotgun sequencing. Small fragment libraries with insert sizes of 250, 350, 450, and 550 bp were prepared using a TruSeq DNA PCR-Free Library Prep Kit (Illumina). Two mate-pair libraries with average insert sizes of 2 and 5 kbp were also built using a Nextera Mate Pair Library Preparation Kit (Illumina). Final libraries were quantified using a Qubit and qualities were assessed using a Bioanalyzer. Libraries were sequenced on an Illumina HiSeq2500 using either 2 × 125 bp paired-end or 2 × 150 bp paired-end chemistry at the Genomics and Microarray Core, University of Colorado–Anschutz Medical Campus.</p>                            <h3 scrollto-destination=78055049 id="78055049" class="section-title" >2.2. Flow cytometry</h3>
<p>Prior to sequencing, we conducted flow cytometry to estimate the genome size of <em>R.</em><em>speciosa</em>. Samples were transferred to the Iowa State University Flow Cytometry Facility and analysed on a BD Biosciences FACSCanto Flow Cytometer using <em>Solanum lycopersicum</em> as an internal standard. All cytometry protocols followed in house methods at that facility. The software package BD FACSDiva v.6.1.3<a reveal-id="dsw054-B19" data-open="dsw054-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> was used to analyse the data.</p>                            <h3 scrollto-destination=78055051 id="78055051" class="section-title" >2.3. <em>De novo</em> nuclear genome assembly</h3>
<p><em>De</em><em>novo</em> assembly of the <em>R.</em><em>speciosa</em> genome into contigs was conducted using MaSuRCA v3.1.0.<a reveal-id="dsw054-B20" data-open="dsw054-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> All raw untrimmed sequenced reads were used as input for MaSuRCA as instructed in the manual and run with default settings except the memory usage option (NUM_THREADS and JF_SIZE). Gap closing was conducted via BGI’s GapCloser v1.6 of SOAPdenovo.<a reveal-id="dsw054-B21" data-open="dsw054-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> Following this, we used the package SSPACE v2.0 (scaffolding pre-assemblies after contig extension)<a reveal-id="dsw054-B22" data-open="dsw054-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> to conduct the final round of scaffolding. Reads trimmed with Trimmomatic v0.33<a reveal-id="dsw054-B23" data-open="dsw054-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> (ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:50) were also assembled with SOAPdenovo2<a reveal-id="dsw054-B21" data-open="dsw054-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> (‘Multi-kmer’ method was used, kmer ranges from 65 to 80) and Abyss v1.9.0<a reveal-id="dsw054-B24" data-open="dsw054-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> (<em>k</em> = 71, <em>b</em> = 1,000, <em>P</em> = 0.95 and <em>s</em> = 500). Assembly generated by MaSuRCA was used for all downstream analysis as it yielded the best statistics based on scaffold N50. Completeness of genome assembly was evaluated using BUSCO v1.2.<a reveal-id="dsw054-B25" data-open="dsw054-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a></p>                            <h3 scrollto-destination=78055053 id="78055053" class="section-title" >2.4. Ploidy estimation and heterozygosity analysis</h3>
<p>To estimate ploidy of <em>R.</em><em>speciosa</em>, we used an approach similar to that employed by Kentaro Yoshida’s group.<a reveal-id="dsw054-B26" data-open="dsw054-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> After quality trimming and error correction, reads were mapped backed to the assembled genome using BWA v0.7.13<sup>27</sup> with default settings allowing two gaps and without seeding. Samtools was used to convert the sam file containing the alignment information into bam format and the resulting bam file was sorted and indexed.<a reveal-id="dsw054-B28" data-open="dsw054-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> SNP calling was conducted using both samtools mpileup and bcftools.<a reveal-id="dsw054-B28" data-open="dsw054-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> To reliably calculate base and minor allele frequencies to enable identification of heterozygous alleles, only positions with read coverages ≥ ×50 (approximately one-half of the estimated overall read coverage) and minor allele frequencies ≥ 0.2 were considered. Ploidy is determined by the ratio of alleles at heterozygous positions. In a diploid genome, a ratio of 1:1 is expected for the majority of the heterozygous alleles; in triploid and tetraploid genomes, these ratios should ∼1:2 and 1:3, respectively. Using Marker-predicted genes as input, prediction of recent or ancient polyploidization was secondarily tested by plotting the distribution of ages of duplicated sequences using DupPipe.<a reveal-id="dsw054-B29" data-open="dsw054-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> Synonymous divergence (Ks) was estimated using PAML v4.9c and the F3X4 model of evolution.<a reveal-id="dsw054-B30" data-open="dsw054-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> As polyploidy or whole-genome duplication (WGD) yields a large and enduring signature in such plots, significant features (<em>α</em> = 0.05) in age distributions were identified with SiZer v0.1-4.<a reveal-id="dsw054-B31" data-open="dsw054-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> To estimate genome heterozygosity, the number of distinct k-mers in our reads was calculated using Jellyfish v2.2.5.<a reveal-id="dsw054-B32" data-open="dsw054-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> k-mers were plotted as frequencies, and the number of peaks was used as an indicator of genome heterozygosity.</p>                            <h3 scrollto-destination=78055055 id="78055055" class="section-title" >2.5. Genome annotation with MAKER</h3>
<p>Genome annotation was conducted using the MAKER-P annotation pipeline and its dependent programmes.<a reveal-id="dsw054-B33" data-open="dsw054-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> Two rounds of running the pipeline were completed to yield the final annotation file. Prior to annotation of the protein-coding genes, a <em>R.</em><em>speciosa-</em>specific repeat library was constructed following the tutorial included in MAKER-P tool kit<a reveal-id="dsw054-B33" data-open="dsw054-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> and RepBase v21.09.<a reveal-id="dsw054-B34" data-open="dsw054-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> This library served to mask repeat elements in the assembled genome using RepeatMasker v3.2.0.<a reveal-id="dsw054-B35" data-open="dsw054-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> The repeat-masked assembled genome of <em>R. speciosa</em> (scaffold length ≥ 1,000 bp) was then processed through the MAKER annotation pipeline. Gene prediction was conducted using SNAP<a reveal-id="dsw054-B36" data-open="dsw054-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> and AUGUSTUS v3.2.2,<a reveal-id="dsw054-B37" data-open="dsw054-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> trained on <em>Arabidopsis thaliana</em> and <em>S.</em><em>lycopersicum</em> as models. RNA-seq data obtained from corolla and leaf samples of <em>R. speciosa</em> (Zhuang and Tripp, in rev.; SRA accession: SRP075855) served as EST evidence to refine gene models. High confidence annotation of protein sequences of 31 dicot species were retrieved from the PLAZA database v3.0<a reveal-id="dsw054-B38" data-open="dsw054-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> to serve as protein evidence for refining gene predictions. The annotation file from the first-round run was used to create the HMM training file for SNAP. The resulting HMM training file together with first-round GFF annotation file were further processed by MAKER-P to generate a final annotation file. Reads generated from previous RNA-seq study (Zhuang and Tripp, in rev.) were mapped against the <em>R. speciosa</em> genome assembly using BWA<a reveal-id="dsw054-B27" data-open="dsw054-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> with default parameters. Paired-end reads where each end mapped to gene-containing regions of different scaffolds were then used to combine the fragments of a single gene.<a reveal-id="dsw054-B39" data-open="dsw054-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> All predicted protein sequences were further validated following methods described by Upadhyay et al.<a reveal-id="dsw054-B40" data-open="dsw054-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a></p>                            <h3 scrollto-destination=78055057 id="78055057" class="section-title" >2.6. SSR identification and maker predication</h3>
<p>We used GMATo (genome-wide microsatellite analysing tool; <a class="link link-uri" href="https://sourceforge.net/projects/gmato/files/" target="_blank">https://sourceforge.net/projects/gmato/files/</a>)<a reveal-id="dsw054-B41" data-open="dsw054-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> to characterize the masked SSR loci. Masked SSR sequences were extracted using an in-house Perl script prior to analysis with GMATo. To enable comparison of our SSR statistics to those reported from other species,<a reveal-id="dsw054-B42" data-open="dsw054-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> we employed similar GMATo parameters to those used in prior studies. These included retaining SSR motifs that were 2–8 bp in length and had ≥4 repeats with a minimum of 10 bp length (−r 5 −m 2 −× 10 −s 0).</p>                            <h3 scrollto-destination=78055059 id="78055059" class="section-title" ><em>2.7.</em> Gene Ontology <em>database annotation</em></h3>
<p>Functional annotation of predicted gene models was conducted using the Trinotate pipeline v3.0.0 (<a class="link link-uri" href="http://trinotate.sourceforge.net/" target="_blank">http://trinotate.sourceforge.net/</a>), with a blast <em>e</em>-value threshold of 1 × 10<sup>−</sup><a reveal-id="dsw054-B5" data-open="dsw054-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a>. To assign function annotations to the predicted genes, local NCBI-BLAST v2.2.31<a reveal-id="dsw054-B43" data-open="dsw054-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> was used to search for homologies between the gene and SwissProt,<a reveal-id="dsw054-B44" data-open="dsw054-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> UniProt,<a reveal-id="dsw054-B45" data-open="dsw054-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> and eggNOG/Gene Ontology (GO) databases.<a reveal-id="dsw054-B46" data-open="dsw054-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> In addition, all green plant entries integrated into UniProtKB/TrEMBL (Taxonomy: Viridiplantae)<a reveal-id="dsw054-B47" data-open="dsw054-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> were also retrieved to serve as a custom database for blast searches. To enable comparison and visualization of GO annotations in <em>R.</em><em>speciosa</em> to other plants, the annotated GO terms of <em>Ruellia</em> were combined with GO annotation files of <em>A.</em><em>thaliana</em> and <em>Solanum tuberosum</em>, which were downloaded from AgriGO (<a class="link link-uri" href="http://bioinfo.cau.edu.cn/agriGO/dowload" target="_blank">http://bioinfo.cau.edu.cn/agriGO/dowload</a>).<a reveal-id="dsw054-B48" data-open="dsw054-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> This combined file was analysed using BGI WEGO (<a class="link link-uri" href="http://wego.genomics.org.cn" target="_blank">http://wego.genomics.org.cn</a>)<a reveal-id="dsw054-B49" data-open="dsw054-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> to classify GO terms into Level-2 sub categories.</p>                            <h3 scrollto-destination=78055061 id="78055061" class="section-title" >2.8. Phylogenetic analysis of genes involved in the anthocyanin biosynthesis pathway</h3>
<p>To enable phylogenetic investigation of key structural genes involved in the production of anthocyanin pigments that occur in flowers, fruits, and leaves, amino acid sequences representing genes from several angiosperm species were downloaded from PLAZA<a reveal-id="dsw054-B38" data-open="dsw054-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> and OrthoDB v9.<a reveal-id="dsw054-B50" data-open="dsw054-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> Sequences from <em>Pinus taeda</em> were also download from PINREFSEQ (<a class="link link-uri" href="http://pinegenome.org" target="_blank">http://pinegenome.org</a>) to facilitate tree rooting.<a reveal-id="dsw054-B51" data-open="dsw054-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> In total, four structural genes from 25 dicots plus three monocots were downloaded then subject to phylogenetic analysis along with orthologues retrieved from <em>R.</em><em>speciosa</em> and <em>P.</em><em>taeda</em> (ntax = 30 in total). Based on Trinotate annotation, genes annotated as flavanone 3-hydroxylase (F3H), flavonoid 3′-hydroxylase (F3′H), flavonoid 3′,5′-hydroxylase (F3′5′H), and dihydroflavonol 4-reductase (DFR) were extracted, and CD-HIT<a reveal-id="dsw054-B52" data-open="dsw054-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> was used to retrieve non-redundant copies (at a 99% similarity cutoff). To accurately estimate copy numbers of family genes studied, SNPs located within target gene-containing regions identified from polyploidy analysis were further examined; the gene was considered to have unidentified paralogues if the percentage of SNPs were ≥90% of gene length and averaged ≥10 reads mapped to the SNP-containing region. We searched for orthologues of these genes in queried plants using the package Proteinortho v5.15.<a reveal-id="dsw054-B53" data-open="dsw054-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> Identified sequences were parsed and extracted with an in-house Perl script. Two alignment matrices were constructed: the first containing all F3H, F3′H, and F3′5′H sequences and the second containing all DFR sequences. Sequence alignment was conducted using the ‘linsi’ option of MAFFT v7.305.<a reveal-id="dsw054-B54" data-open="dsw054-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> Resulting alignments were truncated to exclude regions of extremely high sequence divergence (namely: autapomorphic divergence), which can interfere with phylogenetic signal owing to phenomena such as long branch attraction.<a reveal-id="dsw054-B55" data-open="dsw054-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> Evolutionary histories were inferred using maximum likelihood methods, employing the LG+ F<a reveal-id="dsw054-B56" data-open="dsw054-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> amino acid model as the best-fitting model determined by MEGA6,<a reveal-id="dsw054-B57" data-open="dsw054-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> and final resulting phylogenetic trees were rooted using <em>P.</em><em>taeda.</em> Branch support was assessed via 200 ML bootstrap replicates. To test selection on and assess potential functional divergence of duplicated genes, ratios of non-synonymous (Ka) to synonymous (Ks) nucleotide substitutions were estimated using the Ka/Ks web service at the computational biology unit, hosted at The University of Bergen.<a reveal-id="dsw054-B58" data-open="dsw054-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> Phylogenetic trees from the earlier analyses were used as reference trees for the Ka/Ks analysis.</p>                            <h3 scrollto-destination=78055063 id="78055063" class="section-title" >2.9. Genome-wide characterization of MYB3R and R2R3 type MYB gene family</h3>
<p>To characterize the MYB gene family in <em>R.</em><em>speciosa</em>, protein sequences of 125 typical R2R3-MYB proteins and six atypical MYB proteins (AtMYBCDC5, AtMYB3R-like, and AtMYB4R1) were downloaded from the <em>Arabidopsis</em> information resource (<a class="link link-uri" href="https://www.arabidopsis.org/browse/genefamily/MYB.jsp" target="_blank">https://www.arabidopsis.org/browse/genefamily/MYB.jsp</a>).<a reveal-id="dsw054-B59" data-open="dsw054-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> Three <em>Petunia</em> MYBs regulate anthocyanin biosynthesis were also downloaded from NCBI database (GI:673536265, GI:321688260, GI:321688230) to facilitate function annotation. Proteinortho was used to conduct homologue searches of <em>Arabidopsis</em> MYBs against protein sequence of all gene models annotated by Maker-P with default parameters. Proteins without homologues among <em>Arabidopsis</em> MYBs were removed from subsequent analysis. The resulting MYB proteins were then assessed for the presence of Myb domains using Pfam v26.0 on the Pfam server with default parameters (<a class="link link-uri" href="http://pfam.sanger.ac.uk" target="_blank">http://pfam.sanger.ac.uk</a>).<a reveal-id="dsw054-B60" data-open="dsw054-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> Proteins containing Myb domains were extracted and subjected to phylogenetic analysis using methods as earlier employing the LG + G<a reveal-id="dsw054-B56" data-open="dsw054-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> amino acid model as the best-fitting model. Function assignments were conducted based on known functions of <em>Arabidopsis</em> MYBs.<a reveal-id="dsw054-B61 dsw054-B62 dsw054-B63 dsw054-B64 dsw054-B65 dsw054-B66 dsw054-B67 dsw054-B68 dsw054-B69 dsw054-B70 dsw054-B71 dsw054-B72 dsw054-B73 dsw054-B74 dsw054-B75 dsw054-B76 dsw054-B77 dsw054-B78" data-open="dsw054-B61 dsw054-B62 dsw054-B63 dsw054-B64 dsw054-B65 dsw054-B66 dsw054-B67 dsw054-B68 dsw054-B69 dsw054-B70 dsw054-B71 dsw054-B72 dsw054-B73 dsw054-B74 dsw054-B75 dsw054-B76 dsw054-B77 dsw054-B78" class="link link-ref link-reveal xref-bibr"><sup>61–78</sup></a> MYBs with bootstrap values ≥ 70% were considered to belong to the same subgroup.</p>                            <h3 scrollto-destination=78055065 id="78055065" class="section-title" >2.10. Gene expression analysis</h3>
<p>To measure expression levels of selected anthocyanin genes and identified MYBs, reads from previous tissue-specific RNA-seq libraries (Zhuang and Tripp, in rev.; SRA accession: SRP075855) were trimmed with Trimmomatic v0.33<sup>23</sup>(parameters used: ILLUMINACLIP:TruSeq3-PE.fa:2:30:10 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36) and mapped back to Maker-P predicted gene sequences using BWA with default parameters. eXpress v1.5.1<a reveal-id="dsw054-B79" data-open="dsw054-B79" class="link link-ref link-reveal xref-bibr"><sup>79</sup></a> was used to measure gene abundance with default parameters. The ‘fpkm’ column of tissue-specific eXpress output was extracted and used for building expression heatmaps using the heatmap.2 program within the gplots<a reveal-id="dsw054-B80" data-open="dsw054-B80" class="link link-ref link-reveal xref-bibr"><sup>80</sup></a> package for R.</p>                            <h3 scrollto-destination=78055067 id="78055067" class="section-title" >2.11. Reference-based chloroplast assembly</h3>
<p>We additionally used resulting reads to assemble the <em>R.</em><em>speciosa</em> chloroplast genome. First, we retrieved all available plant chloroplast sequences that were 100,000–200,000 bp in size from NCBI. Blat was used to compared our contigs generated from MaSuRCA<a reveal-id="dsw054-B20" data-open="dsw054-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> to these reference chloroplast sequences to find nucleotide sequences with high similarity.<a reveal-id="dsw054-B81" data-open="dsw054-B81" class="link link-ref link-reveal xref-bibr"><sup>81</sup></a> Five contigs (length ≥ 10,000 bp; query coverage ≥ 80%) were identified. We used Blast<a reveal-id="dsw054-B82" data-open="dsw054-B82" class="link link-ref link-reveal xref-bibr"><sup>82</sup></a> to compare these five contigs against reference chloroplasts and identified two with the highest <em>e</em>-values and query coverages: <em>Lindenbergia philippensis</em> (Accession: HG530133) and <em>Andrographis paniculata</em> (Accession: KF150644). These, together with a chloroplast reference from closed related species <em>Ruellia breedlovei</em> (Accession: KP300014) were used as references for our <em>R. speciosa</em> assembly. Quality trimmed paired-end reads were aligned to the references using Bowtie v2-2.2.3.<a reveal-id="dsw054-B83" data-open="dsw054-B83" class="link link-ref link-reveal xref-bibr"><sup>83</sup></a> Reads with either end mapped to the references were extracted using Samtools and an in-house Perl script. Mapped reads were randomly subsampled from ×100 to ×500 coverage of the estimated chloroplast genome size; reads with an apparent depth under ×5 were discarded using BBmap v36.02.<a reveal-id="dsw054-B84" data-open="dsw054-B84" class="link link-ref link-reveal xref-bibr"><sup>84</sup></a> The resulting aligned reads were assembled using SPAdes v3.5.0<a reveal-id="dsw054-B85" data-open="dsw054-B85" class="link link-ref link-reveal xref-bibr"><sup>85</sup></a> and scaffolded using SSPACE v2.0<a reveal-id="dsw054-B22" data-open="dsw054-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> against all sequenced libraries. Scaffolds were gap closed using Gapcloser v1.6.<a reveal-id="dsw054-B21" data-open="dsw054-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> The best assemblies containing three scaffolds ≥ 300 bp were selected and subjected to second-round scaffolding and gap closing, thereby generating a circular chloroplast genome. The chloroplast sequence of the closely related <em>R.</em><em>breedlovei</em> was used to correct the orientation of the region between the inverted repeats (IRs). CpGAVAS<a reveal-id="dsw054-B86" data-open="dsw054-B86" class="link link-ref link-reveal xref-bibr"><sup>86</sup></a>was used for chloroplast annotation and visualization.</p>                            <h2 scrollto-destination=78055069 id="78055069" class="section-title" >3. Results and discussion</h2>
                            <h3 scrollto-destination=78055070 id="78055070" class="section-title" >3.1. Genome sequencing and assembly</h3>
<div class="block-child-p">From a whole-genome perspective, the successful sequencing, assembly, and annotation of the <em>R.</em><em>speciosa</em> genome contributes important new data about one of the largest yet understudied clades of flowering plants—the Asterids. Our targeting <em>Ruellia</em> initiates the process of building a new model system for studies of floral evolution. We herein estimate the genome size of <em>R.</em><em>speciosa</em> to be ∼1.2 Gb based on flow cytometry or ∼1.1 Gb based on Kmer analysis (<a reveal-id="dsw054-F2" data-open="dsw054-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). For the nuclear genome assembly, nine paired-end libraries with insert sizes ranging from 250 to 550 bp and two mate-pair libraries with insert sizes of 3–6 kbp were constructed. After quality trimming, we generated &gt;118.9 Gb of sequence data representing &gt;×104 coverage of the predicted genome size (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054_Supp.zip?Expires=2147483647&amp;Signature=OzwUbdrbll8GWrzlrYMohmjg2aIkCFsGZYTFXY~Kt6KuVc2twDadH8zYRPTVlbeirvm1zVBvwJJ62AMcnZZQRT6IkIiVdHs8RgqOLfsAeIdh1ClQTbbNSYHc87Of2VzfRrqiJSBJfZsk8gGsQOEMVqZ87ObAsjdKIt1D-l2pplHdY-9Bvx3S5C84jcZZN7spkpVLyp1Amn0LUMm0Z6563bsp1Ely9Dp~rQnN63w02~k0kobcBfHFjTanMPAC6QAKpc2tzoYt3l5iwjOgcp3pwV1K2ooGe9cXit7wNNM1qehybTZkN05Ngz~w8giL74bxPcg2lPmrHbg5qzF0W5XtaQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Before scaffolding, short reads were assembled into 808,710 contigs with an N50 of 1,551 bp that totaled 733 Mb (spanning 67.25% of the predicted genome size; <a reveal-id="dsw054-T1" data-open="dsw054-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Additional MP reads improved the N50 by ∼10 and the additional gap closing step increased genome coverage to ∼90.07% of its predicted size without Ns (<a reveal-id="dsw054-T1" data-open="dsw054-T1" class="link link-reveal link-table xref-fig">Table 1</a>). The success of a draft genome assembly is strongly dependent on the genetic complexity of the specific organism and its genome size.<a reveal-id="dsw054-B87" data-open="dsw054-B87" class="link link-ref link-reveal xref-bibr"><sup>87</sup></a> Of the two methods we used for genome size estimation, the Kmer protocol estimated a smaller genome size, which may suggest a substantial repeat fraction in the genome that is difficult to account for using this type of analysis.<a reveal-id="dsw054-B88" data-open="dsw054-B88" class="link link-ref link-reveal xref-bibr"><sup>88</sup></a> Our repeat DNA analysis suggested that ∼70% of the genome is repetitive (<a reveal-id="dsw054-T2" data-open="dsw054-T2" class="link link-reveal link-table xref-fig">Table 2</a>). Repeat sequences are difficult to assemble because high-identity reads can derive from different portions of the genome, generating gaps, ambiguities, and collapses in alignment and assembly.<a reveal-id="dsw054-B89" data-open="dsw054-B89" class="link link-ref link-reveal xref-bibr"><sup>89</sup></a><sup>,</sup><a reveal-id="dsw054-B90" data-open="dsw054-B90" class="link link-ref link-reveal xref-bibr"><sup>90</sup></a> This high repeat DNA content is likely to explain the relatively low scaffold N50 (17,908 bp), even though the sequencing depth exceeds ×100. Nonetheless, our scaffold N50 is comparable to those derived from genome assemblies of <em>Cannabis sativa</em> (N50: 16.7 kb),<a reveal-id="dsw054-B91" data-open="dsw054-B91" class="link link-ref link-reveal xref-bibr"><sup>91</sup></a><sup>,</sup><em>Conyza canadensis</em> (N50: 33.5 kb, repeat content 6.25%),<a reveal-id="dsw054-B92" data-open="dsw054-B92" class="link link-ref link-reveal xref-bibr"><sup>92</sup></a><sup>,</sup><em>Ocimum tenuiflorum</em> (N50: 27.1 kb; repeat content: 42.9%),<a reveal-id="dsw054-B40" data-open="dsw054-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a><sup>,</sup><em>Lemna minor</em> (N50: 23.6 kb, repeat content: 61.5%),<a reveal-id="dsw054-B93" data-open="dsw054-B93" class="link link-ref link-reveal xref-bibr"><sup>93</sup></a><sup>,</sup><em>Solanum commersonii</em> (N50: 44.3 kb, repeat content: 44.5%),<a reveal-id="dsw054-B4" data-open="dsw054-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><em>Hevea brasiliensis</em> (N50: 3 kb, repeat content: 72%),<a reveal-id="dsw054-B94" data-open="dsw054-B94" class="link link-ref link-reveal xref-bibr"><sup>94</sup></a><sup>,</sup><em>Aegilops tauschii</em> (N50: 57.6 kb, repeat content: 66%),<a reveal-id="dsw054-B95" data-open="dsw054-B95" class="link link-ref link-reveal xref-bibr"><sup>95</sup></a> and <em>Triticum urartu</em> (N50: 63.7 kb, repeat content: 67%).<a reveal-id="dsw054-B96" data-open="dsw054-B96" class="link link-ref link-reveal xref-bibr"><sup>96</sup></a> Although data from additional sequencing libraries or a combination of existing and additional sequencing technologies may be needed to make further improvements to the quality of the <em>R. speciosa</em> genome for purposes of whole-genome synteny analyses, our current assembly enables us to characterize numerous aspects of this genome ranging from repeat elements, gene content, gene annotation, and phylogenetic history of specific pathways of interest. <div data-id="dsw054-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/m_dsw054f2.png?Expires=2147483647&amp;Signature=yEOOirXF1BSbbUifS0DPU5Mx3s7vm27~oHXAl6zqPJlYCViEURT~RYNkCGMnT~jdEy6Cr4hJELob3X1rYwU8ypUMAubaIHiP4SfwznPx8YUV9J4~YWg0kwzyDJC92e5daITg1TtMtO-DrTugh6JefMmpsChQrd-xH6lA6FOgyQptb9RB2R5qm~4ScPYpv35PVCA3NyFKavCuA06XJ-18Y1~pnXY~~s8BA~N42WxJlNc-HRoHZnk7WQYHOvBN6gUNGgJFdaZkTi5hMPRZl~88uGdqZpTq9MuV~IyhCX7zVAs0xr-HY3qAKNORF4AqWYEjWm4ITlY3O4BPhCxu4C4ZSw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Ploidy analysis of assembled R. speciosa genome. A major peak around 0.5 supports hypothesis of diploidy in R. speciosa. (B) Twenty-one k-mer depth distribution of whole-genome Illumina reads; the single peak observed indicates high level of genome homozygosity." path-from-xml="dsw054f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055071/dsw054f2.tif" path-from-xml="dsw054f2.tif" target="_blank">View large</a><a section="78055071" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054f2.png?Expires=2147483647&amp;Signature=FyAdkVK9f71ulwuEmgBYsZ7M7VCv6uYTQccBOqdwO~DYPjXK-5OLVCiI~6pPQnwO0BgtOInfNtr9RqcJbc9~d2rdYPdkWDDe0-~VqeeZxHPj8krDVfHaN1Mq8OeNvgFYnWGdD594xy3o4KRgtvmJFTxEemISzO7snVF3BcvFX6g46dBybn2TRAejMQ5egbhQMqzG2I8mo0TvmmsUNvzNBzJfhumpMQVmp4LbgaWlZX8QlE1UcigdtahZkf2vCKHjUWO6LzA4Cda6NDZenCHmwh1UPgn4Go7yN5vt2WnE1gSH96S3kVUawVF7SA5d-wnfVkiLUk-E31zWZAtjplm2iQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055071&amp;ar=2960054&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw054f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Ploidy analysis of assembled <em>R. speciosa</em> genome. A major peak around 0.5 supports hypothesis of diploidy in <em>R. speciosa</em>. (B) Twenty-one k-mer depth distribution of whole-genome Illumina reads; the single peak observed indicates high level of genome homozygosity.</p></div></div><div content-id="dsw054-F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/m_dsw054f2.png?Expires=2147483647&amp;Signature=yEOOirXF1BSbbUifS0DPU5Mx3s7vm27~oHXAl6zqPJlYCViEURT~RYNkCGMnT~jdEy6Cr4hJELob3X1rYwU8ypUMAubaIHiP4SfwznPx8YUV9J4~YWg0kwzyDJC92e5daITg1TtMtO-DrTugh6JefMmpsChQrd-xH6lA6FOgyQptb9RB2R5qm~4ScPYpv35PVCA3NyFKavCuA06XJ-18Y1~pnXY~~s8BA~N42WxJlNc-HRoHZnk7WQYHOvBN6gUNGgJFdaZkTi5hMPRZl~88uGdqZpTq9MuV~IyhCX7zVAs0xr-HY3qAKNORF4AqWYEjWm4ITlY3O4BPhCxu4C4ZSw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Ploidy analysis of assembled R. speciosa genome. A major peak around 0.5 supports hypothesis of diploidy in R. speciosa. (B) Twenty-one k-mer depth distribution of whole-genome Illumina reads; the single peak observed indicates high level of genome homozygosity." path-from-xml="dsw054f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055071/dsw054f2.tif" path-from-xml="dsw054f2.tif" target="_blank">View large</a><a section="78055071" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054f2.png?Expires=2147483647&amp;Signature=FyAdkVK9f71ulwuEmgBYsZ7M7VCv6uYTQccBOqdwO~DYPjXK-5OLVCiI~6pPQnwO0BgtOInfNtr9RqcJbc9~d2rdYPdkWDDe0-~VqeeZxHPj8krDVfHaN1Mq8OeNvgFYnWGdD594xy3o4KRgtvmJFTxEemISzO7snVF3BcvFX6g46dBybn2TRAejMQ5egbhQMqzG2I8mo0TvmmsUNvzNBzJfhumpMQVmp4LbgaWlZX8QlE1UcigdtahZkf2vCKHjUWO6LzA4Cda6NDZenCHmwh1UPgn4Go7yN5vt2WnE1gSH96S3kVUawVF7SA5d-wnfVkiLUk-E31zWZAtjplm2iQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055071&amp;ar=2960054&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw054f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Ploidy analysis of assembled <em>R. speciosa</em> genome. A major peak around 0.5 supports hypothesis of diploidy in <em>R. speciosa</em>. (B) Twenty-one k-mer depth distribution of whole-genome Illumina reads; the single peak observed indicates high level of genome homozygosity.</p></div></div><div content-id="dsw054-T1" class="table-modal"><div class="table-wrap"><div id="dsw054-T1" data-id="dsw054-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of the <em>de novo</em> genome assembly of <em>R.</em><em>speciosa</em></p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Category </th><th>Contigs </th><th>Scaffolds (all) </th><th>Scaffold (&gt;1 k) </th></tr></thead><tbody><tr><td>Number of contigs </td><td>808,710 </td><td>341,168 </td><td>84,681 </td></tr><tr><td>N50 (bp) </td><td>1,551 </td><td>17,908 </td><td>19,156 </td></tr><tr><td>L50 </td><td>92,735 </td><td>14,124 </td><td>10,074 </td></tr><tr><td>Largest contig/scaffolds (bp) </td><td>34,954 </td><td>345,229 </td><td>345,229 </td></tr><tr><td>GC content (%) </td><td>39.27 </td><td>38.80 </td><td>38.77 </td></tr><tr><td><em>N</em>’s percentage </td><td> </td><td>3.62 </td><td>3.82 </td></tr><tr><td>Total span (M) </td><td>733 </td><td>1,021 </td><td>892 </td></tr><tr><td>Estimated coverage (%) </td><td>67.25 </td><td>93.69 </td><td>81.89 </td></tr><tr><td>Number of gene models </td><td> </td><td> </td><td>40,124 </td></tr><tr><td>Mean transcript length </td><td> </td><td> </td><td>1,131 bp </td></tr><tr><td>Number of genes annotated </td><td> </td><td> </td><td>37,848 (94.32%) </td></tr></tbody></table></div><div content-id="dsw054-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Category </th><th>Contigs </th><th>Scaffolds (all) </th><th>Scaffold (&gt;1 k) </th></tr></thead><tbody><tr><td>Number of contigs </td><td>808,710 </td><td>341,168 </td><td>84,681 </td></tr><tr><td>N50 (bp) </td><td>1,551 </td><td>17,908 </td><td>19,156 </td></tr><tr><td>L50 </td><td>92,735 </td><td>14,124 </td><td>10,074 </td></tr><tr><td>Largest contig/scaffolds (bp) </td><td>34,954 </td><td>345,229 </td><td>345,229 </td></tr><tr><td>GC content (%) </td><td>39.27 </td><td>38.80 </td><td>38.77 </td></tr><tr><td><em>N</em>’s percentage </td><td> </td><td>3.62 </td><td>3.82 </td></tr><tr><td>Total span (M) </td><td>733 </td><td>1,021 </td><td>892 </td></tr><tr><td>Estimated coverage (%) </td><td>67.25 </td><td>93.69 </td><td>81.89 </td></tr><tr><td>Number of gene models </td><td> </td><td> </td><td>40,124 </td></tr><tr><td>Mean transcript length </td><td> </td><td> </td><td>1,131 bp </td></tr><tr><td>Number of genes annotated </td><td> </td><td> </td><td>37,848 (94.32%) </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsw054-T1" data-id="dsw054-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of the <em>de novo</em> genome assembly of <em>R.</em><em>speciosa</em></p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Category </th><th>Contigs </th><th>Scaffolds (all) </th><th>Scaffold (&gt;1 k) </th></tr></thead><tbody><tr><td>Number of contigs </td><td>808,710 </td><td>341,168 </td><td>84,681 </td></tr><tr><td>N50 (bp) </td><td>1,551 </td><td>17,908 </td><td>19,156 </td></tr><tr><td>L50 </td><td>92,735 </td><td>14,124 </td><td>10,074 </td></tr><tr><td>Largest contig/scaffolds (bp) </td><td>34,954 </td><td>345,229 </td><td>345,229 </td></tr><tr><td>GC content (%) </td><td>39.27 </td><td>38.80 </td><td>38.77 </td></tr><tr><td><em>N</em>’s percentage </td><td> </td><td>3.62 </td><td>3.82 </td></tr><tr><td>Total span (M) </td><td>733 </td><td>1,021 </td><td>892 </td></tr><tr><td>Estimated coverage (%) </td><td>67.25 </td><td>93.69 </td><td>81.89 </td></tr><tr><td>Number of gene models </td><td> </td><td> </td><td>40,124 </td></tr><tr><td>Mean transcript length </td><td> </td><td> </td><td>1,131 bp </td></tr><tr><td>Number of genes annotated </td><td> </td><td> </td><td>37,848 (94.32%) </td></tr></tbody></table></div><div content-id="dsw054-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Category </th><th>Contigs </th><th>Scaffolds (all) </th><th>Scaffold (&gt;1 k) </th></tr></thead><tbody><tr><td>Number of contigs </td><td>808,710 </td><td>341,168 </td><td>84,681 </td></tr><tr><td>N50 (bp) </td><td>1,551 </td><td>17,908 </td><td>19,156 </td></tr><tr><td>L50 </td><td>92,735 </td><td>14,124 </td><td>10,074 </td></tr><tr><td>Largest contig/scaffolds (bp) </td><td>34,954 </td><td>345,229 </td><td>345,229 </td></tr><tr><td>GC content (%) </td><td>39.27 </td><td>38.80 </td><td>38.77 </td></tr><tr><td><em>N</em>’s percentage </td><td> </td><td>3.62 </td><td>3.82 </td></tr><tr><td>Total span (M) </td><td>733 </td><td>1,021 </td><td>892 </td></tr><tr><td>Estimated coverage (%) </td><td>67.25 </td><td>93.69 </td><td>81.89 </td></tr><tr><td>Number of gene models </td><td> </td><td> </td><td>40,124 </td></tr><tr><td>Mean transcript length </td><td> </td><td> </td><td>1,131 bp </td></tr><tr><td>Number of genes annotated </td><td> </td><td> </td><td>37,848 (94.32%) </td></tr></tbody></table></div></div><div content-id="dsw054-T2" class="table-modal"><div class="table-wrap"><div id="dsw054-T2" data-id="dsw054-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p><em>De novo</em> identification of sequence repeats in the genome of <em>R.</em><em>speciosa</em></p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Class </th><th>Number </th><th>Total length (bp) </th><th>Percentage of genome </th></tr></thead><tbody><tr><td colspan="4">Retrotransposons </td></tr><tr><td> LINE </td><td>29,246 </td><td>11,469,068 </td><td>1.23 </td></tr><tr><td> SINE </td><td>276 </td><td>14,095 </td><td>1.50<em>e</em><sup>−</sup><sup>05</sup> </td></tr><tr><td> LTR <em>Copia</em> </td><td>53,812 </td><td>23,369,721 </td><td>2.50 </td></tr><tr><td> LTR <em>Gypsy</em> </td><td>342,944 </td><td>158,185,560 </td><td>16.92 </td></tr><tr><td> LTR other </td><td>26,556 </td><td>9,772,864 </td><td>1.05 </td></tr><tr><td> DNA transposons </td><td>113,878 </td><td>35,584,790 </td><td>3.81 </td></tr><tr><td colspan="4">Tandem repeats </td></tr><tr><td> Satellite </td><td>1,213 </td><td>176,347 </td><td>0.02 </td></tr><tr><td> Low complexity </td><td>64,404 </td><td>3,277,475 </td><td>0.35 </td></tr><tr><td> Simple repeats </td><td>352,081 </td><td>19,601,079 </td><td>2.10 </td></tr><tr><td>Unknown </td><td>1,158,125 </td><td>384,916,724 </td><td>41.17 </td></tr><tr><td>Total </td><td>2,142,535 </td><td>646,367,723 </td><td>69.13 </td></tr></tbody></table></div><div content-id="dsw054-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Class </th><th>Number </th><th>Total length (bp) </th><th>Percentage of genome </th></tr></thead><tbody><tr><td colspan="4">Retrotransposons </td></tr><tr><td> LINE </td><td>29,246 </td><td>11,469,068 </td><td>1.23 </td></tr><tr><td> SINE </td><td>276 </td><td>14,095 </td><td>1.50<em>e</em><sup>−</sup><sup>05</sup> </td></tr><tr><td> LTR <em>Copia</em> </td><td>53,812 </td><td>23,369,721 </td><td>2.50 </td></tr><tr><td> LTR <em>Gypsy</em> </td><td>342,944 </td><td>158,185,560 </td><td>16.92 </td></tr><tr><td> LTR other </td><td>26,556 </td><td>9,772,864 </td><td>1.05 </td></tr><tr><td> DNA transposons </td><td>113,878 </td><td>35,584,790 </td><td>3.81 </td></tr><tr><td colspan="4">Tandem repeats </td></tr><tr><td> Satellite </td><td>1,213 </td><td>176,347 </td><td>0.02 </td></tr><tr><td> Low complexity </td><td>64,404 </td><td>3,277,475 </td><td>0.35 </td></tr><tr><td> Simple repeats </td><td>352,081 </td><td>19,601,079 </td><td>2.10 </td></tr><tr><td>Unknown </td><td>1,158,125 </td><td>384,916,724 </td><td>41.17 </td></tr><tr><td>Total </td><td>2,142,535 </td><td>646,367,723 </td><td>69.13 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsw054-T2" data-id="dsw054-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p><em>De novo</em> identification of sequence repeats in the genome of <em>R.</em><em>speciosa</em></p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Class </th><th>Number </th><th>Total length (bp) </th><th>Percentage of genome </th></tr></thead><tbody><tr><td colspan="4">Retrotransposons </td></tr><tr><td> LINE </td><td>29,246 </td><td>11,469,068 </td><td>1.23 </td></tr><tr><td> SINE </td><td>276 </td><td>14,095 </td><td>1.50<em>e</em><sup>−</sup><sup>05</sup> </td></tr><tr><td> LTR <em>Copia</em> </td><td>53,812 </td><td>23,369,721 </td><td>2.50 </td></tr><tr><td> LTR <em>Gypsy</em> </td><td>342,944 </td><td>158,185,560 </td><td>16.92 </td></tr><tr><td> LTR other </td><td>26,556 </td><td>9,772,864 </td><td>1.05 </td></tr><tr><td> DNA transposons </td><td>113,878 </td><td>35,584,790 </td><td>3.81 </td></tr><tr><td colspan="4">Tandem repeats </td></tr><tr><td> Satellite </td><td>1,213 </td><td>176,347 </td><td>0.02 </td></tr><tr><td> Low complexity </td><td>64,404 </td><td>3,277,475 </td><td>0.35 </td></tr><tr><td> Simple repeats </td><td>352,081 </td><td>19,601,079 </td><td>2.10 </td></tr><tr><td>Unknown </td><td>1,158,125 </td><td>384,916,724 </td><td>41.17 </td></tr><tr><td>Total </td><td>2,142,535 </td><td>646,367,723 </td><td>69.13 </td></tr></tbody></table></div><div content-id="dsw054-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Class </th><th>Number </th><th>Total length (bp) </th><th>Percentage of genome </th></tr></thead><tbody><tr><td colspan="4">Retrotransposons </td></tr><tr><td> LINE </td><td>29,246 </td><td>11,469,068 </td><td>1.23 </td></tr><tr><td> SINE </td><td>276 </td><td>14,095 </td><td>1.50<em>e</em><sup>−</sup><sup>05</sup> </td></tr><tr><td> LTR <em>Copia</em> </td><td>53,812 </td><td>23,369,721 </td><td>2.50 </td></tr><tr><td> LTR <em>Gypsy</em> </td><td>342,944 </td><td>158,185,560 </td><td>16.92 </td></tr><tr><td> LTR other </td><td>26,556 </td><td>9,772,864 </td><td>1.05 </td></tr><tr><td> DNA transposons </td><td>113,878 </td><td>35,584,790 </td><td>3.81 </td></tr><tr><td colspan="4">Tandem repeats </td></tr><tr><td> Satellite </td><td>1,213 </td><td>176,347 </td><td>0.02 </td></tr><tr><td> Low complexity </td><td>64,404 </td><td>3,277,475 </td><td>0.35 </td></tr><tr><td> Simple repeats </td><td>352,081 </td><td>19,601,079 </td><td>2.10 </td></tr><tr><td>Unknown </td><td>1,158,125 </td><td>384,916,724 </td><td>41.17 </td></tr><tr><td>Total </td><td>2,142,535 </td><td>646,367,723 </td><td>69.13 </td></tr></tbody></table></div></div></div>                            <h3 scrollto-destination=78055072 id="78055072" class="section-title" >3.2. Ploidy and heterozygosity of R. speciosa</h3>
<p>Polyploidization is rampant in flowering plants and is an important contributor to genomic diversity and function; as such, polyploidization can facilitate novel ecological transitions and substantially impact evolutionary trajectories.<a reveal-id="dsw054-B97" data-open="dsw054-B97" class="link link-ref link-reveal xref-bibr"><sup>97</sup></a> As discussed by Yoshida et al.,<a reveal-id="dsw054-B26" data-open="dsw054-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> for diploid species, we expect a single peak at 0.5 for the mean of read counts at heterozygous positions. To characterize the ploidy of <em>R. speciosa</em>, the distribution of read counts of biallelic SNPs was calculated from high coverage genomic regions. <a reveal-id="dsw054-F2" data-open="dsw054-F2" class="link link-reveal link-table xref-fig">Figure 2A</a> depicts a major peak around 0.5, supporting a hypothesis of diploidy in <em>R. speciosa</em>. This ploidy was further confirmed by Ks analysis using DupPipe: peaks of gene duplication serve as evidence of ancient WGDs, but none was identified through Sizer analysis in our Ks plot (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054_Supp.zip?Expires=2147483647&Signature=OzwUbdrbll8GWrzlrYMohmjg2aIkCFsGZYTFXY~Kt6KuVc2twDadH8zYRPTVlbeirvm1zVBvwJJ62AMcnZZQRT6IkIiVdHs8RgqOLfsAeIdh1ClQTbbNSYHc87Of2VzfRrqiJSBJfZsk8gGsQOEMVqZ87ObAsjdKIt1D-l2pplHdY-9Bvx3S5C84jcZZN7spkpVLyp1Amn0LUMm0Z6563bsp1Ely9Dp~rQnN63w02~k0kobcBfHFjTanMPAC6QAKpc2tzoYt3l5iwjOgcp3pwV1K2ooGe9cXit7wNNM1qehybTZkN05Ngz~w8giL74bxPcg2lPmrHbg5qzF0W5XtaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>) thus supporting our earlier conclusion that <em>R. speciosa</em> genome is diploid.</p><p>Kmer analysis (<a reveal-id="dsw054-F2" data-open="dsw054-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>) revealed the presence of only a single peak, consistent with our prediction that <em>R.</em><em>speciosa</em> is highly homozygous. A very minor peak immediately adjacent to the major peak could be detected only when Kmer size  ≤ 31, which represents regions with higher copy numbers such as repeats. To further explore heterozygosity and polymorphism,we calculated the percentage of total insertions/deletions and SNPs using SAMtools and BCFtools.<a reveal-id="dsw054-B28" data-open="dsw054-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> We found a total of 2,431,330 SNPs (∼0.20%) with relatively relaxed search parameters (depth ≥ 3) or 299,334 SNPs (0.02%) of high confidence (depth ≥ 30), further supporting our hypothesis that the <em>R. speciosa</em> genome is highly homozygous.</p>                            <h3 scrollto-destination=78055075 id="78055075" class="section-title" >3.3. Genome annotation and quality assessment</h3>
<p>Of the 84,681 scaffolds (length &gt; 1,000 bp) used for gene annotation, 25,256 (29.82%) were found to contain annotated genes. Our final Maker-P<a reveal-id="dsw054-B33" data-open="dsw054-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> annotation predicted 40,124 protein-coding genes with an average length of 1,131 bp. This number exceeds the number of genes annotated for two relatively closely related species that have comparable genome sizes: tomato (gene number: 34,727, genome size ∼900 Mb)<a reveal-id="dsw054-B6" data-open="dsw054-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> and potato (gene number: 35,004, genome size ∼844 Mb).<a reveal-id="dsw054-B7" data-open="dsw054-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> To assess completeness of our <em>R. speciosa</em> genome assembly, 966 genes present as single-copy orthologues in at least 90% of plant species in OrthoDB were compiled. Using BUSCO, we retrieved orthologues for 741 (77.52%) conserved genes in <em>Ruellia</em>, suggesting a relatively high level of completeness for draft plant genomes generated only from Illumina reads. Additionally, mapping reads from previous RNA-seq experiment on <em>R. speciosa</em> (Zhuang and Tripp, in rev.) back to our genome assembly demonstrated that 76.12% of the quality trimmed reads could be successfully mapped using Tophat2<a reveal-id="dsw054-B98" data-open="dsw054-B98" class="link link-ref link-reveal xref-bibr"><sup>98</sup></a> with default parameters. We interpret this as further evidence of the completeness of our genome assembly given this number is on par with expected mapping rates for RNA-seq data from very well assembled genomes such as the human genome, which has a 70–90% RNA-seq read mapping rate.<a reveal-id="dsw054-B99" data-open="dsw054-B99" class="link link-ref link-reveal xref-bibr"><sup>99</sup></a></p><p>Gene annotation databases are commonly used to evaluate functional properties of experimentally derived gene sets.<a reveal-id="dsw054-B100" data-open="dsw054-B100" class="link link-ref link-reveal xref-bibr"><sup>100</sup></a> Comparison of completed sets of genes from different genomes helps to reveal the genetic basis of biological traits as well as differences among species.<a reveal-id="dsw054-B101" data-open="dsw054-B101" class="link link-ref link-reveal xref-bibr"><sup>101</sup></a> Using the Trinotate pipeline, we assigned GO functional terms to 30,852 (76.89%) genes. Genome-wide comparative analyses among <em>R. speciosa</em>, <em>S. tuberosum</em>, and <em>A. thaliana</em> revealed distinct patterns of enriched Level 2 GO terms (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054_Supp.zip?Expires=2147483647&Signature=OzwUbdrbll8GWrzlrYMohmjg2aIkCFsGZYTFXY~Kt6KuVc2twDadH8zYRPTVlbeirvm1zVBvwJJ62AMcnZZQRT6IkIiVdHs8RgqOLfsAeIdh1ClQTbbNSYHc87Of2VzfRrqiJSBJfZsk8gGsQOEMVqZ87ObAsjdKIt1D-l2pplHdY-9Bvx3S5C84jcZZN7spkpVLyp1Amn0LUMm0Z6563bsp1Ely9Dp~rQnN63w02~k0kobcBfHFjTanMPAC6QAKpc2tzoYt3l5iwjOgcp3pwV1K2ooGe9cXit7wNNM1qehybTZkN05Ngz~w8giL74bxPcg2lPmrHbg5qzF0W5XtaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). Although potato is more closely related to <em>R. speciosa</em> than is <em>Arabidopsis</em>, gene enrichment patterns were more similar between <em>R. speciosa</em> and <em>Arabidopsis</em> than between <em>R. speciosa</em> and <em>S. tuberosum</em> for two of the three categories [cellular component and biological process categories (BP)]. GO terms categorized under molecular function were more consistent among the three species. Across all functions, fewer GO terms were enriched in <em>S. tuberosum</em> compared with in <em>R. speciosa</em> and <em>A. thaliana</em>, with the exception of GO terms involved metabolic processes (BP).</p>                            <h3 scrollto-destination=78055078 id="78055078" class="section-title" >3.4. Analysis of repetitive elements</h3>
<div class="block-child-p">Repetitive DNA elements generally compose the majority of nuclear genomes in plants,<a reveal-id="dsw054-B102" data-open="dsw054-B102" class="link link-ref link-reveal xref-bibr"><sup>102</sup></a> but this content varies tremendously, ranging from 3% (<em>U.</em><em>gibba</em>)<a reveal-id="dsw054-B14" data-open="dsw054-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> to 80% of the total genome (<em>Triticum aestivum</em>).<a reveal-id="dsw054-B103" data-open="dsw054-B103" class="link link-ref link-reveal xref-bibr"><sup>103</sup></a> In this study, a combination of homologue-based searching and <em>de novo</em> analyses identified 417,698 (2.47%) tandem repeats in our assembled <em>R. speciosa</em> genome, which can be parsed into 352,081 simple repeats (2.1%), 64,404 low-complexity elements (0.35%), and 1,213 satellite elements (0.02%). Various types of retrotransposons comprising ∼25.51% of the whole assembled genome were also recovered in this analysis; these include 29,246 (1.23%) long interspersed nuclear elements (LINE), 276 (1.50<em>e</em><sup>−</sup><sup>05</sup>%) short interspersed nuclear elements (SINE), 424,312 (20.47%) long terminal repeats (LTR), and 113,878 (3.81%) DNA transposons. Overall, using homologue-based and <em>de novo</em> approaches, we estimate that ∼70% of the <em>R. speciosa</em> genome is composed of repetitive DNA (<a reveal-id="dsw054-T2" data-open="dsw054-T2" class="link link-reveal link-table xref-fig">Table 2</a>). However, because repeat elements are difficult to assemble, our use of assembly dependent methods for repeat identification may still underestimate total repeat content in the genome. Repetitive sequences possess high sequence homogeneity but, over the course of evolution, accumulate variations in both sequence and copy numbers.<a reveal-id="dsw054-B104" data-open="dsw054-B104" class="link link-ref link-reveal xref-bibr"><sup>104</sup></a> Because of this, repetitive elements are very useful for markers for many downstream applications, ranging from plant population genetics to breeding studies.<a reveal-id="dsw054-B105" data-open="dsw054-B105" class="link link-ref link-reveal xref-bibr"><sup>105</sup></a> In particular, molecular makers developed from simple sequence repeats (or microsatellites) have been utilized most extensively as these can be readily amplified by PCR and contain a large amount of allelic variation at each locus.<a reveal-id="dsw054-B106" data-open="dsw054-B106" class="link link-ref link-reveal xref-bibr"><sup>106</sup></a> We identified 181,793 SSR loci using GMATo. These repetitive elements are depicted via motif type in a frequency distribution plot shown in <a reveal-id="dsw054-F3" data-open="dsw054-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>. The most abundant repeat motifs are AT and TA, together accounting for ∼53.37% of the total SSRs identified (<a reveal-id="dsw054-F3" data-open="dsw054-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). Dinucleotide repeat motifs were the greatest in number (<a reveal-id="dsw054-F3" data-open="dsw054-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>), consistent with results from SSR study in holy basil<a reveal-id="dsw054-B107" data-open="dsw054-B107" class="link link-ref link-reveal xref-bibr"><sup>107</sup></a> but differing from patterns in most other plants including tomato, potato, cucumber, and rice where trinucleotide SSRs represent the majority (not including mononucleotide SSRs)<a reveal-id="dsw054-B42" data-open="dsw054-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> of repeat type. Taken together, these data suggest that SSRs constitute a relatively unique and important aspect of the <em>R. speciosa</em> genome. <div data-id="dsw054-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/m_dsw054f3.png?Expires=2147483647&Signature=rgLey92fcoWYqrvGbxB5obTA8q8bMwnHTqeDaKw--FbnJLQy2MqvURhjae1HPWnXBYJzi635Z-9q4j6MwFN-PbfkI5f0~9HfZvs8RXmElaRDlV1lasIjgJ9GkG8UHJy2kGKKmHkjWLtIS5Ep7suf7iGHEkG~PB-x-uTENmdWecpxXL58QJKMqzoy0o3sasvw1BJAIpfDdTRyAuFmPaUelytwqz049oVen6W93zWFsPzISBypgugvnm3hsmJMItawLLVR76yp2YpW7yCNwgGX2pGJcK8mM8Sn~7A3D8IIlcNyRCDNNNw-WSDDV1fnYvMasxBWnRJz9Ui72Kwq5sR2jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Simple sequence repeat (SSR or microsatellite) analysis. (A) Bar plot of top 15 repeat motifs with highest abundance. (B) Pie chart showing distribution of identified SSRs based on motif type." path-from-xml="dsw054f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055079/dsw054f3.tif" path-from-xml="dsw054f3.tif" target="_blank">View large</a><a section="78055079" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054f3.png?Expires=2147483647&Signature=4cbf2xT01HzuWrIeUuWG4sHPRM2q5eWH2QO6S5I~UE9sZ60Ikb~Yhy3~t3c5Byw429Q-z9ri6HdK6e6UxYBK3az5twCDg9Jcj89~-EtYjsDRvOiNaDAZIH64eIryJ5sNXOMkx7FfrqjrsEdi4Xr~zYboCvQO3g3dSjV0aCFw~Zs7H9WSn0GRAz4ynyxOYDS9RHgXGwVBI01jeDgb~E~ujsXrLUel3qcmJKJ4OXvg-pWAHLvA9JNzcNPufe4pZbZG9z4z-zlBfwWjQVqjQoPntSnEYS2GqlwcYhGq2ODO1zm-rfHngPtJaqidSe8aen6ww3AQ95M4pj2oHR415n8Rwg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055079&ar=2960054&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw054f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Simple sequence repeat (SSR or microsatellite) analysis. (A) Bar plot of top 15 repeat motifs with highest abundance. (B) Pie chart showing distribution of identified SSRs based on motif type.</p></div></div><div content-id="dsw054-F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/m_dsw054f3.png?Expires=2147483647&Signature=rgLey92fcoWYqrvGbxB5obTA8q8bMwnHTqeDaKw--FbnJLQy2MqvURhjae1HPWnXBYJzi635Z-9q4j6MwFN-PbfkI5f0~9HfZvs8RXmElaRDlV1lasIjgJ9GkG8UHJy2kGKKmHkjWLtIS5Ep7suf7iGHEkG~PB-x-uTENmdWecpxXL58QJKMqzoy0o3sasvw1BJAIpfDdTRyAuFmPaUelytwqz049oVen6W93zWFsPzISBypgugvnm3hsmJMItawLLVR76yp2YpW7yCNwgGX2pGJcK8mM8Sn~7A3D8IIlcNyRCDNNNw-WSDDV1fnYvMasxBWnRJz9Ui72Kwq5sR2jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Simple sequence repeat (SSR or microsatellite) analysis. (A) Bar plot of top 15 repeat motifs with highest abundance. (B) Pie chart showing distribution of identified SSRs based on motif type." path-from-xml="dsw054f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055079/dsw054f3.tif" path-from-xml="dsw054f3.tif" target="_blank">View large</a><a section="78055079" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054f3.png?Expires=2147483647&Signature=4cbf2xT01HzuWrIeUuWG4sHPRM2q5eWH2QO6S5I~UE9sZ60Ikb~Yhy3~t3c5Byw429Q-z9ri6HdK6e6UxYBK3az5twCDg9Jcj89~-EtYjsDRvOiNaDAZIH64eIryJ5sNXOMkx7FfrqjrsEdi4Xr~zYboCvQO3g3dSjV0aCFw~Zs7H9WSn0GRAz4ynyxOYDS9RHgXGwVBI01jeDgb~E~ujsXrLUel3qcmJKJ4OXvg-pWAHLvA9JNzcNPufe4pZbZG9z4z-zlBfwWjQVqjQoPntSnEYS2GqlwcYhGq2ODO1zm-rfHngPtJaqidSe8aen6ww3AQ95M4pj2oHR415n8Rwg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055079&ar=2960054&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw054f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Simple sequence repeat (SSR or microsatellite) analysis. (A) Bar plot of top 15 repeat motifs with highest abundance. (B) Pie chart showing distribution of identified SSRs based on motif type.</p></div></div></div>                            <h3 scrollto-destination=78055080 id="78055080" class="section-title" >3.5. Analysis of genes involved in the anthocyanin biosynthesis pathway</h3>
<div class="block-child-p">Flower colour is among the key traits that serve to signal rewards to pollinators and thus helps determine reproductive success of both plants and pollinators.<a reveal-id="dsw054-B108 dsw054-B109 dsw054-B110 dsw054-B111 dsw054-B112" data-open="dsw054-B108 dsw054-B109 dsw054-B110 dsw054-B111 dsw054-B112" class="link link-ref link-reveal xref-bibr"><sup>108–112</sup></a> The anthocyanin biosynthesis pathway (ABP), largely responsible for flower colour, has furthermore been extensively studied for its beneficial contributions to human health.<a reveal-id="dsw054-B113" data-open="dsw054-B113" class="link link-ref link-reveal xref-bibr"><sup>113</sup></a> Using data from the <em>R.</em><em>speciosa</em> genome as well as orthologues in other plants with annotated nuclear reference genomes, we reconstructed evolutionary histories of four structural enzymes in the ABP. In <em>R.</em><em>speciosa</em>, we found two putative copies of F3H, eight of F3′H, one of F3′5′H, and three of DFR. These enzymes were present in variable duplicated copy numbers in other reference genomes as well as in <em>Ruellia</em> (<a reveal-id="dsw054-F4" data-open="dsw054-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054_Supp.zip?Expires=2147483647&amp;Signature=OzwUbdrbll8GWrzlrYMohmjg2aIkCFsGZYTFXY~Kt6KuVc2twDadH8zYRPTVlbeirvm1zVBvwJJ62AMcnZZQRT6IkIiVdHs8RgqOLfsAeIdh1ClQTbbNSYHc87Of2VzfRrqiJSBJfZsk8gGsQOEMVqZ87ObAsjdKIt1D-l2pplHdY-9Bvx3S5C84jcZZN7spkpVLyp1Amn0LUMm0Z6563bsp1Ely9Dp~rQnN63w02~k0kobcBfHFjTanMPAC6QAKpc2tzoYt3l5iwjOgcp3pwV1K2ooGe9cXit7wNNM1qehybTZkN05Ngz~w8giL74bxPcg2lPmrHbg5qzF0W5XtaQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S3 and S4</a></span>; see also ref. <a reveal-id="dsw054-B114" data-open="dsw054-B114" class="link link-ref link-reveal xref-bibr">114</a>), although studies in other systems have reported these enzymes as single copy (12 species of <em>Penstemon</em>).<a reveal-id="dsw054-B114" data-open="dsw054-B114" class="link link-ref link-reveal xref-bibr"><sup>114</sup></a><div data-id="dsw054-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/m_dsw054f4.png?Expires=2147483647&amp;Signature=khVgamXbEpnovi20Gi3NJALVYzujhhqVZBelw-Lf7m2Duf5sM4V7uzJbmEB6EsvLfWWbyaKwQnEpRw7M~RFkHP7PMHCmbdZObLqKI~C3Esve0~1CZidwbKHc9As639~N20er8J4PKDVCf7G5RIx4LJ42LzvmMG7fQrmuhfVJvqHLeHYgxnpksF19bymT7hzqxVUwKTZLiAf~rA4LFrefso7SP8ZO0D9dBFIun81M8Gl664mae75mW9-A3aY3ZRf-zpwMpqN9omV~wL1L21lUCFue4maQKYzNTOcotdE4ppDsZowcfZPGWNTx2pZEadVVwQcuSX0fXzugpcZtq4gIYw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Summary phylogenies for (A) F3H, F3′H, and F3′5′H and (B) DFR. Phylogenetic analyses incorporated all copies of these four genes available from published angiosperm genomes (these collapased into clades depicted with triangles) as well as all copies from one gymnosperm, P. taeda. (A) Analyses recovered two major clades of F3H, three of F3′H, and one of F3′5′H in angiosperms (A) as well as three major clades of DFR in angiosperms (B). All Pinus copies depicted as individual lineages. Numbers of copies of genes recovered in R. speciosa genome shown in brackets. Numbers above branches indicate maximum likelihood bootstrap support (only values at or above 70% shown except for DFR-2). Full phylogenetic detail provided in Supplementary Figs S3 and S4." path-from-xml="dsw054f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055081/dsw054f4.tif" path-from-xml="dsw054f4.tif" target="_blank">View large</a><a section="78055081" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054f4.png?Expires=2147483647&amp;Signature=3XgwjVvLTiU7rFF~nEFGQIyGrMh-76EIPLcSfI2GT64MdGwNk2piw8Z9brHauZXqeEhBRo2J2uFa-03v4hyO5AQcNpb2P4f~aLvXz9rqKHku79LQc89-8umxbtNR-7rTbzXTR14IX7~jUv-cTVRdqXx03cxcQOuc7DGwb6NqJU0SUoSWS5qAxSxKraNBILhBeWzcalrs2RSIBjVffWU41z5SebdeD~8UjC6DphzutNzUXk~Rj8L2eD4hwmDHTQre6fTZDLgu~CgFYuXVRMU8iRcAG4RjzFseQXXzsJH8Ekt6~YPdNTeutauYMESfT4ub0WeiKevZR1G46xhHP3gN-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055081&amp;ar=2960054&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw054f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Summary phylogenies for (A) F3H, F3′H, and F3′5′H and (B) DFR. Phylogenetic analyses incorporated all copies of these four genes available from published angiosperm genomes (these collapased into clades depicted with triangles) as well as all copies from one gymnosperm, <em>P. taeda</em>. (A) Analyses recovered two major clades of F3H, three of F3′H, and one of F3′5′H in angiosperms (A) as well as three major clades of DFR in angiosperms (B). All Pinus copies depicted as individual lineages. Numbers of copies of genes recovered in <em>R. speciosa</em> genome shown in brackets. Numbers above branches indicate maximum likelihood bootstrap support (only values at or above 70% shown except for DFR-2). Full phylogenetic detail provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054_Supp.zip?Expires=2147483647&amp;Signature=OzwUbdrbll8GWrzlrYMohmjg2aIkCFsGZYTFXY~Kt6KuVc2twDadH8zYRPTVlbeirvm1zVBvwJJ62AMcnZZQRT6IkIiVdHs8RgqOLfsAeIdh1ClQTbbNSYHc87Of2VzfRrqiJSBJfZsk8gGsQOEMVqZ87ObAsjdKIt1D-l2pplHdY-9Bvx3S5C84jcZZN7spkpVLyp1Amn0LUMm0Z6563bsp1Ely9Dp~rQnN63w02~k0kobcBfHFjTanMPAC6QAKpc2tzoYt3l5iwjOgcp3pwV1K2ooGe9cXit7wNNM1qehybTZkN05Ngz~w8giL74bxPcg2lPmrHbg5qzF0W5XtaQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S3 and S4</a></span>.</p></div></div><div content-id="dsw054-F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/m_dsw054f4.png?Expires=2147483647&amp;Signature=khVgamXbEpnovi20Gi3NJALVYzujhhqVZBelw-Lf7m2Duf5sM4V7uzJbmEB6EsvLfWWbyaKwQnEpRw7M~RFkHP7PMHCmbdZObLqKI~C3Esve0~1CZidwbKHc9As639~N20er8J4PKDVCf7G5RIx4LJ42LzvmMG7fQrmuhfVJvqHLeHYgxnpksF19bymT7hzqxVUwKTZLiAf~rA4LFrefso7SP8ZO0D9dBFIun81M8Gl664mae75mW9-A3aY3ZRf-zpwMpqN9omV~wL1L21lUCFue4maQKYzNTOcotdE4ppDsZowcfZPGWNTx2pZEadVVwQcuSX0fXzugpcZtq4gIYw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Summary phylogenies for (A) F3H, F3′H, and F3′5′H and (B) DFR. Phylogenetic analyses incorporated all copies of these four genes available from published angiosperm genomes (these collapased into clades depicted with triangles) as well as all copies from one gymnosperm, P. taeda. (A) Analyses recovered two major clades of F3H, three of F3′H, and one of F3′5′H in angiosperms (A) as well as three major clades of DFR in angiosperms (B). All Pinus copies depicted as individual lineages. Numbers of copies of genes recovered in R. speciosa genome shown in brackets. Numbers above branches indicate maximum likelihood bootstrap support (only values at or above 70% shown except for DFR-2). Full phylogenetic detail provided in Supplementary Figs S3 and S4." path-from-xml="dsw054f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055081/dsw054f4.tif" path-from-xml="dsw054f4.tif" target="_blank">View large</a><a section="78055081" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054f4.png?Expires=2147483647&amp;Signature=3XgwjVvLTiU7rFF~nEFGQIyGrMh-76EIPLcSfI2GT64MdGwNk2piw8Z9brHauZXqeEhBRo2J2uFa-03v4hyO5AQcNpb2P4f~aLvXz9rqKHku79LQc89-8umxbtNR-7rTbzXTR14IX7~jUv-cTVRdqXx03cxcQOuc7DGwb6NqJU0SUoSWS5qAxSxKraNBILhBeWzcalrs2RSIBjVffWU41z5SebdeD~8UjC6DphzutNzUXk~Rj8L2eD4hwmDHTQre6fTZDLgu~CgFYuXVRMU8iRcAG4RjzFseQXXzsJH8Ekt6~YPdNTeutauYMESfT4ub0WeiKevZR1G46xhHP3gN-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055081&amp;ar=2960054&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw054f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Summary phylogenies for (A) F3H, F3′H, and F3′5′H and (B) DFR. Phylogenetic analyses incorporated all copies of these four genes available from published angiosperm genomes (these collapased into clades depicted with triangles) as well as all copies from one gymnosperm, <em>P. taeda</em>. (A) Analyses recovered two major clades of F3H, three of F3′H, and one of F3′5′H in angiosperms (A) as well as three major clades of DFR in angiosperms (B). All Pinus copies depicted as individual lineages. Numbers of copies of genes recovered in <em>R. speciosa</em> genome shown in brackets. Numbers above branches indicate maximum likelihood bootstrap support (only values at or above 70% shown except for DFR-2). Full phylogenetic detail provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054_Supp.zip?Expires=2147483647&amp;Signature=OzwUbdrbll8GWrzlrYMohmjg2aIkCFsGZYTFXY~Kt6KuVc2twDadH8zYRPTVlbeirvm1zVBvwJJ62AMcnZZQRT6IkIiVdHs8RgqOLfsAeIdh1ClQTbbNSYHc87Of2VzfRrqiJSBJfZsk8gGsQOEMVqZ87ObAsjdKIt1D-l2pplHdY-9Bvx3S5C84jcZZN7spkpVLyp1Amn0LUMm0Z6563bsp1Ely9Dp~rQnN63w02~k0kobcBfHFjTanMPAC6QAKpc2tzoYt3l5iwjOgcp3pwV1K2ooGe9cXit7wNNM1qehybTZkN05Ngz~w8giL74bxPcg2lPmrHbg5qzF0W5XtaQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S3 and S4</a></span>.</p></div></div></div><div class="block-child-p">Phylogenetic reconstruction yielded two clades of F3H, three of F3′H, one of F3′5′H, and three of DFR, and strong bootstrap support (≥70%) was recovered for some but not all key branches in our topologies (<a reveal-id="dsw054-F4" data-open="dsw054-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054_Supp.zip?Expires=2147483647&amp;Signature=OzwUbdrbll8GWrzlrYMohmjg2aIkCFsGZYTFXY~Kt6KuVc2twDadH8zYRPTVlbeirvm1zVBvwJJ62AMcnZZQRT6IkIiVdHs8RgqOLfsAeIdh1ClQTbbNSYHc87Of2VzfRrqiJSBJfZsk8gGsQOEMVqZ87ObAsjdKIt1D-l2pplHdY-9Bvx3S5C84jcZZN7spkpVLyp1Amn0LUMm0Z6563bsp1Ely9Dp~rQnN63w02~k0kobcBfHFjTanMPAC6QAKpc2tzoYt3l5iwjOgcp3pwV1K2ooGe9cXit7wNNM1qehybTZkN05Ngz~w8giL74bxPcg2lPmrHbg5qzF0W5XtaQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S3 and S4</a></span>). Our trees in part contradict those in Campanella et al.,<a reveal-id="dsw054-B115" data-open="dsw054-B115" class="link link-ref link-reveal xref-bibr"><sup>115</sup></a> which depicted clades of F3′H and DFR that correspond specifically to monocots vs. dicots (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054_Supp.zip?Expires=2147483647&amp;Signature=OzwUbdrbll8GWrzlrYMohmjg2aIkCFsGZYTFXY~Kt6KuVc2twDadH8zYRPTVlbeirvm1zVBvwJJ62AMcnZZQRT6IkIiVdHs8RgqOLfsAeIdh1ClQTbbNSYHc87Of2VzfRrqiJSBJfZsk8gGsQOEMVqZ87ObAsjdKIt1D-l2pplHdY-9Bvx3S5C84jcZZN7spkpVLyp1Amn0LUMm0Z6563bsp1Ely9Dp~rQnN63w02~k0kobcBfHFjTanMPAC6QAKpc2tzoYt3l5iwjOgcp3pwV1K2ooGe9cXit7wNNM1qehybTZkN05Ngz~w8giL74bxPcg2lPmrHbg5qzF0W5XtaQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S3 and S4</a></span>). Instead, <a reveal-id="dsw054-F4" data-open="dsw054-F4" class="link link-reveal link-table xref-fig">Fig. 4</a> suggests the evolution of different copies of F3H, F3′H, F3′5′H, and DFR predates the divergence of monocots from Eudicots.<a reveal-id="dsw054-B1" data-open="dsw054-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> With a given gene genealogy in our analyses (e.g. <a reveal-id="dsw054-F4" data-open="dsw054-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>), there has been subsequent lineage-specific duplication. Thus, ABP genes are marked by a history of early duplication as well as subsequent duplication events, products of which have in some cases facilitated the evolution of tissue-specific functions.<a reveal-id="dsw054-B116" data-open="dsw054-B116" class="link link-ref link-reveal xref-bibr"><sup>116</sup></a> Consistent with this, our expression analysis of putative F3′H genes showed tissue-specific expression patterns: among 5 putative F3′H copies with detectable expression levels (fpkm &gt; 1), two copies appeared to be leaf specific and two were corolla specific (<a reveal-id="dsw054-F5" data-open="dsw054-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). However, Ka/Ks analysis of <em>R.</em><em>speciosa</em> copies yielded ratios consistently &lt;1, suggesting an overall signature of purifying selection or constraint on these copies (<a reveal-id="dsw054-F6" data-open="dsw054-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054_Supp.zip?Expires=2147483647&amp;Signature=OzwUbdrbll8GWrzlrYMohmjg2aIkCFsGZYTFXY~Kt6KuVc2twDadH8zYRPTVlbeirvm1zVBvwJJ62AMcnZZQRT6IkIiVdHs8RgqOLfsAeIdh1ClQTbbNSYHc87Of2VzfRrqiJSBJfZsk8gGsQOEMVqZ87ObAsjdKIt1D-l2pplHdY-9Bvx3S5C84jcZZN7spkpVLyp1Amn0LUMm0Z6563bsp1Ely9Dp~rQnN63w02~k0kobcBfHFjTanMPAC6QAKpc2tzoYt3l5iwjOgcp3pwV1K2ooGe9cXit7wNNM1qehybTZkN05Ngz~w8giL74bxPcg2lPmrHbg5qzF0W5XtaQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Our phylogenetic results also suggest that F3′5′H is more closely related to one of the three clades of F3′H than it is to F3H, corroborating Seitz et al.<a reveal-id="dsw054-B117" data-open="dsw054-B117" class="link link-ref link-reveal xref-bibr"><sup>117</sup></a> who documented multiple evolutionary origins of F3′5′H from an F3′H ancestor in Asteraceae. F3′5′H is an enzyme required for production of blue delphinidins and, across all land plants, is evolutionarily younger than F3H and F3′H.<a reveal-id="dsw054-B115" data-open="dsw054-B115" class="link link-ref link-reveal xref-bibr"><sup>115</sup></a> Our finding that only one-half of all reference genomes sampled harbour a copy of this enzyme corroborates observations that blue anthocyanins are less common in plants than are purple, pink, and red cyanidins and pelargonidins that derive from other branches of the ABP pathway. Finally, DFR plays a crucial role in production of all three branches of the ABP pathway and was traditionally thought of as a substrate generalist, having the capacity to metabolize precursors to all three branches.<a reveal-id="dsw054-B118" data-open="dsw054-B118" class="link link-ref link-reveal xref-bibr"><sup>118</sup></a> However, substrate specificity has evolved in numerous groups of plants, in some cases driven by a single amino acid change.<a reveal-id="dsw054-B118 dsw054-B119 dsw054-B120 dsw054-B121" data-open="dsw054-B118 dsw054-B119 dsw054-B120 dsw054-B121" class="link link-ref link-reveal xref-bibr"><sup>118–121</sup></a> We recovered three copies of DFR in the <em>R.</em><em>speciosa</em> genome, each resolved in three separate clades (<a reveal-id="dsw054-F4" data-open="dsw054-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>). RNA-seq analysis detected expression of two of these three copies: DFR.1 and DFR.3 as labelled in <a reveal-id="dsw054-F4" data-open="dsw054-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>. Although both copies were expressed in leaf as well as corolla tissue, DFR.1 was up-regulated in leaves (a 2.56-fold increase) whereas DFR.3 was up-regulated in corollas (a 3.50-fold increase) (<a reveal-id="dsw054-F5" data-open="dsw054-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). In contrast to F3′H copies, Ka/Ks analysis of DFR copies present in <em>R.</em><em>speciosa</em> yielded ratios &gt; 1 in one of the three copies (<a reveal-id="dsw054-F6" data-open="dsw054-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054_Supp.zip?Expires=2147483647&amp;Signature=OzwUbdrbll8GWrzlrYMohmjg2aIkCFsGZYTFXY~Kt6KuVc2twDadH8zYRPTVlbeirvm1zVBvwJJ62AMcnZZQRT6IkIiVdHs8RgqOLfsAeIdh1ClQTbbNSYHc87Of2VzfRrqiJSBJfZsk8gGsQOEMVqZ87ObAsjdKIt1D-l2pplHdY-9Bvx3S5C84jcZZN7spkpVLyp1Amn0LUMm0Z6563bsp1Ely9Dp~rQnN63w02~k0kobcBfHFjTanMPAC6QAKpc2tzoYt3l5iwjOgcp3pwV1K2ooGe9cXit7wNNM1qehybTZkN05Ngz~w8giL74bxPcg2lPmrHbg5qzF0W5XtaQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). These data suggest positive selection and are consistent with a hypothesis of adaptive evolution of this copy but functional assays are needed to understand the role of duplicated ABP loci in <em>R.</em><em>speciosa</em>. <div data-id="dsw054-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/m_dsw054f5.png?Expires=2147483647&amp;Signature=k0FuIlqBFhINOGKY0I-U1Aj2ivGIvUhySFw9YjkLT4lxuiPY4HIX50pHSSdGnBp6XeLAlCoTAi3OhaZ5z6OqnY-~p4Utuwk55dRVf7gSTXJAgTH9v0ewgaHaHkoNnmbSTdLbLnuJAYWFGXoPUViXXXLTMAiAl3DVWBvXThPNkDwHa8y4SvEDYc6kNNkYiQEUJUtKXO7wkd0tg6ak1yLlTpsAEmE3cvx7O9Gz429mE~JivODTxQjUcqha3eDF94Eo9~CfPjnSwNBTrPGaFdsUoSQdyQ6jRy~PWSQ6k9rdOEsHT4GF~fuNqQLxt0BapfWsXfV-ElnrJoR~qm2TiTwSYw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Heatmap showing expression patterns of putative structural genes functional in ABP and MYBs identified in Ruellia. Only genes with detectable expression level (fpkm &gt; 1) were shown." path-from-xml="dsw054f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055082/dsw054f5.tif" path-from-xml="dsw054f5.tif" target="_blank">View large</a><a section="78055082" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054f5.png?Expires=2147483647&amp;Signature=ktNIz3CswVlaTT-4q2BJthyHM8s8sXdkfvxmt5uLGsYht6KCQIb979Xwe0qaGBJKD7HJ4FQ74GE~NXmlOORZq95jx1fPMIg7pNIc7z3FyLJYsc72-sjGS~z6p5mDT8tuRNlmS4FUZ8PybArOa8zdEeOjIDG-qFzfin15GHSQ2OgqTLa17jWruDlRkKpwUIiQFefRrcPtEX5l7dTr8bwSzx9i6~VRJ0NI9G-VICleTyEWPgZjRTyHaOmavlryyDUp93EU-8gbPg94c4y~WC3go3w6dIexADi~~p9XTd53YAOTJx9pPGpEd608qv0xB5pEt9f6XD9y5XEGQlbqmKBLtA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055082&amp;ar=2960054&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw054f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Heatmap showing expression patterns of putative structural genes functional in ABP and MYBs identified in Ruellia. Only genes with detectable expression level (fpkm &gt; 1) were shown.</p></div></div><div content-id="dsw054-F5" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/m_dsw054f5.png?Expires=2147483647&amp;Signature=k0FuIlqBFhINOGKY0I-U1Aj2ivGIvUhySFw9YjkLT4lxuiPY4HIX50pHSSdGnBp6XeLAlCoTAi3OhaZ5z6OqnY-~p4Utuwk55dRVf7gSTXJAgTH9v0ewgaHaHkoNnmbSTdLbLnuJAYWFGXoPUViXXXLTMAiAl3DVWBvXThPNkDwHa8y4SvEDYc6kNNkYiQEUJUtKXO7wkd0tg6ak1yLlTpsAEmE3cvx7O9Gz429mE~JivODTxQjUcqha3eDF94Eo9~CfPjnSwNBTrPGaFdsUoSQdyQ6jRy~PWSQ6k9rdOEsHT4GF~fuNqQLxt0BapfWsXfV-ElnrJoR~qm2TiTwSYw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Heatmap showing expression patterns of putative structural genes functional in ABP and MYBs identified in Ruellia. Only genes with detectable expression level (fpkm &gt; 1) were shown." path-from-xml="dsw054f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055082/dsw054f5.tif" path-from-xml="dsw054f5.tif" target="_blank">View large</a><a section="78055082" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054f5.png?Expires=2147483647&amp;Signature=ktNIz3CswVlaTT-4q2BJthyHM8s8sXdkfvxmt5uLGsYht6KCQIb979Xwe0qaGBJKD7HJ4FQ74GE~NXmlOORZq95jx1fPMIg7pNIc7z3FyLJYsc72-sjGS~z6p5mDT8tuRNlmS4FUZ8PybArOa8zdEeOjIDG-qFzfin15GHSQ2OgqTLa17jWruDlRkKpwUIiQFefRrcPtEX5l7dTr8bwSzx9i6~VRJ0NI9G-VICleTyEWPgZjRTyHaOmavlryyDUp93EU-8gbPg94c4y~WC3go3w6dIexADi~~p9XTd53YAOTJx9pPGpEd608qv0xB5pEt9f6XD9y5XEGQlbqmKBLtA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055082&amp;ar=2960054&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw054f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Heatmap showing expression patterns of putative structural genes functional in ABP and MYBs identified in Ruellia. Only genes with detectable expression level (fpkm &gt; 1) were shown.</p></div></div><div data-id="dsw054-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/m_dsw054f6.png?Expires=2147483647&amp;Signature=o~9Z4lW9mrzZzYG5c4BoxrSYONUcmSzs4Dn9hfJfjxFW9ABSMHGX7iDdVwCwosESeLIjFSe7ndb~tGcLsHDEiTIQnASpJhMl~O6XsFhgXaAZkZ-CyLq1-2QwfTLaRghBs5GkjP18bZLLPyD3m0M27jp21ZgYnxf-0HIdvbIaXpnZo2MsjaBtB0qpsxuC9xATQsD3FtN81lS7tYzGQ81BuBHPW2fqaXY9p4lz6vJMmq0a3eURvaVlqgASe2WPnagUgwwUucDBdDbS2MqHWe6zKrbnxxleES8H4oIz~nAhZIPRwXJsQ8zzcF25fkevJjV8h~4wZ88fdpllSQ5i06jIGg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic hypothesis for F3'H (A) and DFR (B) identified in R speciosa. Ka/Ks values shown above branches (values &gt;1 in red). Node IDs (1 through 7 for each gene) reflect Ka and Ks values in Supplementary Tables 3 &amp; 4." path-from-xml="dsw054f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055082/dsw054f6.tif" path-from-xml="dsw054f6.tif" target="_blank">View large</a><a section="78055082" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054f6.png?Expires=2147483647&amp;Signature=PW2-JkL4zQAu5pXXpXxxuPRyrOkflwp1AicHmmwykseJxdL4p-RMCdqVo08BAcPvNyecuC8k28VVVguqKYSgcA9g1Pbf9MX3Y-Z0T2SPobxSsLm~a2y6PJR6HP43MO83VZ265ouUYgeythE6enQLEgu48FBinl~hsXEygr0hWssY5A01HdfPf5LcAM49cvTQl8P4POLy0OT6J70uqHPkUmE8VTfGBDO1RD0djX2lKKFxhhOnpqkxyWDKwl31RlWDXVSP8hom~wdHHEFSLVSNUnzi1uJdfzre1lr1kQp0B7r9yhqcKxw8FEvK6XmhXObUkdP8gzcEhNVe6GqS7olD6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055082&amp;ar=2960054&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw054f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic hypothesis for F3'H (A) and DFR (B) identified in R speciosa. Ka/Ks values shown above branches (values &gt;1 in red). Node IDs (1 through 7 for each gene) reflect Ka and Ks values in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054_Supp.zip?Expires=2147483647&amp;Signature=OzwUbdrbll8GWrzlrYMohmjg2aIkCFsGZYTFXY~Kt6KuVc2twDadH8zYRPTVlbeirvm1zVBvwJJ62AMcnZZQRT6IkIiVdHs8RgqOLfsAeIdh1ClQTbbNSYHc87Of2VzfRrqiJSBJfZsk8gGsQOEMVqZ87ObAsjdKIt1D-l2pplHdY-9Bvx3S5C84jcZZN7spkpVLyp1Amn0LUMm0Z6563bsp1Ely9Dp~rQnN63w02~k0kobcBfHFjTanMPAC6QAKpc2tzoYt3l5iwjOgcp3pwV1K2ooGe9cXit7wNNM1qehybTZkN05Ngz~w8giL74bxPcg2lPmrHbg5qzF0W5XtaQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables 3 &amp; 4</a></span>.</p></div></div><div content-id="dsw054-F6" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/m_dsw054f6.png?Expires=2147483647&amp;Signature=o~9Z4lW9mrzZzYG5c4BoxrSYONUcmSzs4Dn9hfJfjxFW9ABSMHGX7iDdVwCwosESeLIjFSe7ndb~tGcLsHDEiTIQnASpJhMl~O6XsFhgXaAZkZ-CyLq1-2QwfTLaRghBs5GkjP18bZLLPyD3m0M27jp21ZgYnxf-0HIdvbIaXpnZo2MsjaBtB0qpsxuC9xATQsD3FtN81lS7tYzGQ81BuBHPW2fqaXY9p4lz6vJMmq0a3eURvaVlqgASe2WPnagUgwwUucDBdDbS2MqHWe6zKrbnxxleES8H4oIz~nAhZIPRwXJsQ8zzcF25fkevJjV8h~4wZ88fdpllSQ5i06jIGg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic hypothesis for F3'H (A) and DFR (B) identified in R speciosa. Ka/Ks values shown above branches (values &gt;1 in red). Node IDs (1 through 7 for each gene) reflect Ka and Ks values in Supplementary Tables 3 &amp; 4." path-from-xml="dsw054f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055082/dsw054f6.tif" path-from-xml="dsw054f6.tif" target="_blank">View large</a><a section="78055082" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054f6.png?Expires=2147483647&amp;Signature=PW2-JkL4zQAu5pXXpXxxuPRyrOkflwp1AicHmmwykseJxdL4p-RMCdqVo08BAcPvNyecuC8k28VVVguqKYSgcA9g1Pbf9MX3Y-Z0T2SPobxSsLm~a2y6PJR6HP43MO83VZ265ouUYgeythE6enQLEgu48FBinl~hsXEygr0hWssY5A01HdfPf5LcAM49cvTQl8P4POLy0OT6J70uqHPkUmE8VTfGBDO1RD0djX2lKKFxhhOnpqkxyWDKwl31RlWDXVSP8hom~wdHHEFSLVSNUnzi1uJdfzre1lr1kQp0B7r9yhqcKxw8FEvK6XmhXObUkdP8gzcEhNVe6GqS7olD6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055082&amp;ar=2960054&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw054f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic hypothesis for F3'H (A) and DFR (B) identified in R speciosa. Ka/Ks values shown above branches (values &gt;1 in red). Node IDs (1 through 7 for each gene) reflect Ka and Ks values in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054_Supp.zip?Expires=2147483647&amp;Signature=OzwUbdrbll8GWrzlrYMohmjg2aIkCFsGZYTFXY~Kt6KuVc2twDadH8zYRPTVlbeirvm1zVBvwJJ62AMcnZZQRT6IkIiVdHs8RgqOLfsAeIdh1ClQTbbNSYHc87Of2VzfRrqiJSBJfZsk8gGsQOEMVqZ87ObAsjdKIt1D-l2pplHdY-9Bvx3S5C84jcZZN7spkpVLyp1Amn0LUMm0Z6563bsp1Ely9Dp~rQnN63w02~k0kobcBfHFjTanMPAC6QAKpc2tzoYt3l5iwjOgcp3pwV1K2ooGe9cXit7wNNM1qehybTZkN05Ngz~w8giL74bxPcg2lPmrHbg5qzF0W5XtaQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables 3 &amp; 4</a></span>.</p></div></div></div>                            <h3 scrollto-destination=78055083 id="78055083" class="section-title" >3.6. Phylogenetic analysis of MYB gene family</h3>
<div class="block-child-p">MYB transcription factors, marked by a conserved DNA-binding domain,<a reveal-id="dsw054-B72" data-open="dsw054-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> comprise the largest transcription factor family in plants.<a reveal-id="dsw054-B78" data-open="dsw054-B78" class="link link-ref link-reveal xref-bibr"><sup>78</sup></a> R2R3 type MYBs are specific to the plant kingdom and are involved in the transcriptional control of plant-specific processes.<a reveal-id="dsw054-B71" data-open="dsw054-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> Based on homologue searches to known MYB3R and R2R3 type MYBs in <em>A.</em><em>thaliana</em>, we identified 96 homologous MYBs in the <em>R.</em><em>speciosa</em> genome then assigned putative functions to 90 of them based on 87 function-annotated MYBs in <em>Arabidopsis</em> (<a reveal-id="dsw054-F7" data-open="dsw054-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>). Our phylogenetic tree is generally consistent with prior data from <em>Arabidopsis</em> alone.<a reveal-id="dsw054-B66" data-open="dsw054-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> Among classified MYBs, of particular interest are three subgroups that function in flavonoid biosynthesis (highlighted in blue in <a reveal-id="dsw054-F7" data-open="dsw054-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>), a large pathway that includes the ABP branch. In most species, the anthocyanin branch is controlled by a ternary complex of MYB-bHLH-WD40 transcription factors and the specificity of each function appears to be the result of a particular R2R3-MYB protein that joins the complex.<a reveal-id="dsw054-B71 dsw054-B72 dsw054-B73" data-open="dsw054-B71 dsw054-B72 dsw054-B73" class="link link-ref link-reveal xref-bibr"><sup>71–73</sup></a><sup>,</sup><a reveal-id="dsw054-B78" data-open="dsw054-B78" class="link link-ref link-reveal xref-bibr"><sup>78</sup></a> A total of nine <em>Ruellia</em> MYBs (RsMYBs; all highlighted in red in <a reveal-id="dsw054-F7" data-open="dsw054-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>) belonging to two of the three flavonoid subgroups were identified (<a reveal-id="dsw054-F7" data-open="dsw054-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>). Phylogenetic analyses resolved RsMYB71 in the same clade as several flavonoid MYBs from <em>Arabidopsis</em> as well as the R2R3-MYB members anthocyanin2, deep purple, and purple haze of <em>Petunia</em>, which are responsible for full petal colour, flower tube venation/bud-blush, and vegetative pigmentation.<a reveal-id="dsw054-B122 dsw054-B123 dsw054-B124" data-open="dsw054-B122 dsw054-B123 dsw054-B124" class="link link-ref link-reveal xref-bibr"><sup>122–124</sup></a> Transcriptome data demonstrated corolla-specific expression of RsMYB71 in <em>Ruellia</em> (<a reveal-id="dsw054-F5" data-open="dsw054-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>), suggesting that this regulatory factor may be a key candidate in regulating flower colour in <em>Ruellia</em>. In contrast, our transcriptome data failed to detect any expression of the remaining eight RsMYBs, all of which belong to a second subgroup (<a reveal-id="dsw054-F5 dsw054-F7" data-open="dsw054-F5 dsw054-F7" class="link link-reveal link-table xref-fig">Figs 5 and 7</a>). Comparative analysis of these MYBs across multiple species of <em>Ruellia</em> followed by functional assays are needed to advance knowledge of flavonoid biosynthesis and flower colour production in this group. <div data-id="dsw054-F7" class="fig fig-section"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/m_dsw054f7.png?Expires=2147483647&Signature=qOTGJq5lFcFtLrxw0S9JlctAIqgUp6WliYO486xHiuuLpRcAk4rCY1wKkGotERhWUYkstf1hS43aX-u0O03bRqm3hiNmh3nimiUssOUDGGmgwuBwLMuXiNUSp2lwbnLnAJbeKQPEweEkQm7TgD1fBeG~Ua0NIkDGQLAt8-Mp~Q2D4o9nyxtj6mjjzHclNaZVX5iTndsVuqBWAWa1ulEeMSYoOC~gSuSu6pL8yVcTxR04~ylLMkc5jcM3bOA3T34PYDx0-mMa8xvQpC9LiPXraKJKemQ3wQ0rUwflz3GXwTe5nkh4-GbijLyHQXSpoWs6Ve6-rS076Lh9mXQT9KnPcQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic analysis of MYB genes present in Ruellia speciosa, Petunia hybrida, and Arabidopsis thaliana. MYBs of R. speciosa are in red text, MYBs of A. thaliana are in blue text, and MYBs of P. hybrida are in black text. Functional annotations based on experimental confirmation in Arabidopsis are marked by asterisks. Branches supported by ML bootstrap values &gt; 70% are labeled to the right of the supported node. Clades highlighted in orange and yellow depict genes involved in non-flavonoid functions; clades highlighted in blue depict genes involved in flavonoid biosynthesis." path-from-xml="dsw054f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055084/dsw054f7.tif" path-from-xml="dsw054f7.tif" target="_blank">View large</a><a section="78055084" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054f7.png?Expires=2147483647&Signature=3FPpKdhp0rsPwEO7FB0jgPjCn2cLvvFM7exMSN2v6XcNBe6xCjMkauD38zaNmo3FPu2MvI0r9qidbG9bf0lUAVLwIHOjDiU~y8KV8QQ2qc~xiOo6GZDaYW--AKw6PI3WYJikE~pXQZGU0CK10usoa4Ja1vBwTqcQycZQ6pDibDB7Mjw3XiS8qIYn172Rb54hHNNDQE7Y-yU8qL4iVaywlvPIOORbykBo7HH3zeg84rmcYEUwxbFcp5f3J39~LofUxxXslDch~1Uxq5aDLsZbwk2VZGmQWoqfP-F3N9E9ir9dmm~1OW4OwdZB5ppOcvuBCNmoiIi7fiMi1oXh8RT4FA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055084&ar=2960054&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw054f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic analysis of MYB genes present in Ruellia speciosa, Petunia hybrida, and Arabidopsis thaliana. MYBs of R. speciosa are in red text, MYBs of A. thaliana are in blue text, and MYBs of P. hybrida are in black text. Functional annotations based on experimental confirmation in Arabidopsis are marked by asterisks. Branches supported by ML bootstrap values &gt; 70% are labeled to the right of the supported node. Clades highlighted in orange and yellow depict genes involved in non-flavonoid functions; clades highlighted in blue depict genes involved in flavonoid biosynthesis.</p></div></div><div content-id="dsw054-F7" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/m_dsw054f7.png?Expires=2147483647&Signature=qOTGJq5lFcFtLrxw0S9JlctAIqgUp6WliYO486xHiuuLpRcAk4rCY1wKkGotERhWUYkstf1hS43aX-u0O03bRqm3hiNmh3nimiUssOUDGGmgwuBwLMuXiNUSp2lwbnLnAJbeKQPEweEkQm7TgD1fBeG~Ua0NIkDGQLAt8-Mp~Q2D4o9nyxtj6mjjzHclNaZVX5iTndsVuqBWAWa1ulEeMSYoOC~gSuSu6pL8yVcTxR04~ylLMkc5jcM3bOA3T34PYDx0-mMa8xvQpC9LiPXraKJKemQ3wQ0rUwflz3GXwTe5nkh4-GbijLyHQXSpoWs6Ve6-rS076Lh9mXQT9KnPcQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic analysis of MYB genes present in Ruellia speciosa, Petunia hybrida, and Arabidopsis thaliana. MYBs of R. speciosa are in red text, MYBs of A. thaliana are in blue text, and MYBs of P. hybrida are in black text. Functional annotations based on experimental confirmation in Arabidopsis are marked by asterisks. Branches supported by ML bootstrap values &gt; 70% are labeled to the right of the supported node. Clades highlighted in orange and yellow depict genes involved in non-flavonoid functions; clades highlighted in blue depict genes involved in flavonoid biosynthesis." path-from-xml="dsw054f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055084/dsw054f7.tif" path-from-xml="dsw054f7.tif" target="_blank">View large</a><a section="78055084" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054f7.png?Expires=2147483647&Signature=3FPpKdhp0rsPwEO7FB0jgPjCn2cLvvFM7exMSN2v6XcNBe6xCjMkauD38zaNmo3FPu2MvI0r9qidbG9bf0lUAVLwIHOjDiU~y8KV8QQ2qc~xiOo6GZDaYW--AKw6PI3WYJikE~pXQZGU0CK10usoa4Ja1vBwTqcQycZQ6pDibDB7Mjw3XiS8qIYn172Rb54hHNNDQE7Y-yU8qL4iVaywlvPIOORbykBo7HH3zeg84rmcYEUwxbFcp5f3J39~LofUxxXslDch~1Uxq5aDLsZbwk2VZGmQWoqfP-F3N9E9ir9dmm~1OW4OwdZB5ppOcvuBCNmoiIi7fiMi1oXh8RT4FA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055084&ar=2960054&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw054f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic analysis of MYB genes present in Ruellia speciosa, Petunia hybrida, and Arabidopsis thaliana. MYBs of R. speciosa are in red text, MYBs of A. thaliana are in blue text, and MYBs of P. hybrida are in black text. Functional annotations based on experimental confirmation in Arabidopsis are marked by asterisks. Branches supported by ML bootstrap values &gt; 70% are labeled to the right of the supported node. Clades highlighted in orange and yellow depict genes involved in non-flavonoid functions; clades highlighted in blue depict genes involved in flavonoid biosynthesis.</p></div></div></div>                            <h3 scrollto-destination=78055085 id="78055085" class="section-title" >3.7. De novo assembly of chloroplast</h3>
<div class="block-child-p">Chloroplasts are essential photosynthetic organelles of plant cells that manufacture energy in the presence of sunlight.<a reveal-id="dsw054-B125" data-open="dsw054-B125" class="link link-ref link-reveal xref-bibr"><sup>125</sup></a> Because the chloroplast genome is small, relatively easily sequenced and assembled, and generally uniparentally inherited, this molecule can provide abundant molecular information to support comparative evolutionary research, especially for species without a whole nuclear genome reference available. Furthermore, nucleotide substitution rates of chloroplast are relatively slow and therefore provide an appropriate window of resolution to study plant phylogeny at deep evolutionary time scales.<a reveal-id="dsw054-B126" data-open="dsw054-B126" class="link link-ref link-reveal xref-bibr"><sup>126</sup></a> In angiosperms, most assembled chloroplast genomes range from 120 to 160 kb in length and exist as circular molecules.<a reveal-id="dsw054-B127" data-open="dsw054-B127" class="link link-ref link-reveal xref-bibr"><sup>127</sup></a> Employing reference-based assembly methods, a subset of chloroplasts reads were extracted from our whole-genome shotgun dataset and successfully assembled into a circular contig with a length of 148,941 bp. Function and structure annotations conducted using CPGAVAS<a reveal-id="dsw054-B86" data-open="dsw054-B86" class="link link-ref link-reveal xref-bibr"><sup>86</sup></a> show that the obtained <em>R.</em><em>speciosa</em> chloroplast genome has a standard quadripartite organization (<a reveal-id="dsw054-F8" data-open="dsw054-F8" class="link link-reveal link-table xref-fig">Fig. 8</a>), comprising two copies of IRs, a large single copy region, and a small single copy region, typical of other plants.<a reveal-id="dsw054-B125" data-open="dsw054-B125" class="link link-ref link-reveal xref-bibr"><sup>125</sup></a><sup>,</sup><a reveal-id="dsw054-B127" data-open="dsw054-B127" class="link link-ref link-reveal xref-bibr"><sup>127</sup></a><sup>,</sup><a reveal-id="dsw054-B128" data-open="dsw054-B128" class="link link-ref link-reveal xref-bibr"><sup>128</sup></a> In total, five genes involved in photosynthesis and 38 genes involved in self-replication were identified in the <em>R</em>. <em>speciosa</em> chloroplast genome, these with an average intron length of 659 bp (<a reveal-id="dsw054-F8" data-open="dsw054-F8" class="link link-reveal link-table xref-fig">Fig. 8</a>). The whole chloroplast genome assembled in this study will be useful to future phylogenetic research both in Acanthaceae and across angiosperms. <div data-id="dsw054-F8" class="fig fig-section"><div class="label fig-label">Figure 8</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/m_dsw054f8.png?Expires=2147483647&Signature=n-SZIZLPHqSFMx1f8yi4krchi9s7JwvHKr-AEnHf8TDHKe~5HUwbjJy6pdmiWyx2Gr2vnirPguJcDkRhpxOFvnoi0Sv6Pdgc2OcVlwQ--d8sC1L6mwszo2vBCVyWbVIdVDxCb7TRMeiId-KCQrpePHfmAUtJV4PUY6a7hnUzZoL74OPbHCKy1phgOwCkW3MKenU4d8Zy5FyVtgrC7L3w7qKvIiMTgZjvMwYTmLyY~TroeF3cN96hc-SDtgaQ06MxpjrAxJa-W3YLkEw2c0iweB1X4YLb-LKC~2-vWlnb~jLeNCbTZJvb6Dz3CTIjCysWgRbApHXWt7Aef6EwxYIcGw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene organization of the R. speciosa chloroplast genome. Genes drawn inside the circle are transcribed clockwise while those drawn outside are transcribed counterclockwise. Different gene functional groups are colour coded. The map was drawn using CpGAVAS." path-from-xml="dsw054f8.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055086/dsw054f8.tif" path-from-xml="dsw054f8.tif" target="_blank">View large</a><a section="78055086" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054f8.png?Expires=2147483647&Signature=zj14fy35-~uajgz~DpitzURXy~1ljO-3jX-YJE1cUN-hyYd1ADXZd6KNL0ihNvypD1ZDD8ux1MDKmm9Mrv8k3iMDSsfpsbqKPs8V6OcSB3EnfuKOqrl0LhRSbAs-QM4r4cCQYBQD~Wa3xWrRbpyKhSZpMbvETXxYlndv4z4QDbgSklz6aDSONyWU8SpXyzZVGpsU6NAcGqGZ3Csu4JjH1aqFBucmuUxJ6DhHn4-tseApQqOrvVIXCDd2N~QRUcxvfXk5xTINiFn3d3eesp8n6jdsW6aRe~yZv-D6LG-36O0F0CH5LCFqOkofjS6yfyfU4YvbnU9A~-JTfPDXxYt0Rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055086&ar=2960054&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw054f8.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene organization of the <em>R. speciosa</em> chloroplast genome. Genes drawn inside the circle are transcribed clockwise while those drawn outside are transcribed counterclockwise. Different gene functional groups are colour coded. The map was drawn using CpGAVAS.</p></div></div><div content-id="dsw054-F8" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 8</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/m_dsw054f8.png?Expires=2147483647&Signature=n-SZIZLPHqSFMx1f8yi4krchi9s7JwvHKr-AEnHf8TDHKe~5HUwbjJy6pdmiWyx2Gr2vnirPguJcDkRhpxOFvnoi0Sv6Pdgc2OcVlwQ--d8sC1L6mwszo2vBCVyWbVIdVDxCb7TRMeiId-KCQrpePHfmAUtJV4PUY6a7hnUzZoL74OPbHCKy1phgOwCkW3MKenU4d8Zy5FyVtgrC7L3w7qKvIiMTgZjvMwYTmLyY~TroeF3cN96hc-SDtgaQ06MxpjrAxJa-W3YLkEw2c0iweB1X4YLb-LKC~2-vWlnb~jLeNCbTZJvb6Dz3CTIjCysWgRbApHXWt7Aef6EwxYIcGw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene organization of the R. speciosa chloroplast genome. Genes drawn inside the circle are transcribed clockwise while those drawn outside are transcribed counterclockwise. Different gene functional groups are colour coded. The map was drawn using CpGAVAS." path-from-xml="dsw054f8.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055086/dsw054f8.tif" path-from-xml="dsw054f8.tif" target="_blank">View large</a><a section="78055086" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054f8.png?Expires=2147483647&Signature=zj14fy35-~uajgz~DpitzURXy~1ljO-3jX-YJE1cUN-hyYd1ADXZd6KNL0ihNvypD1ZDD8ux1MDKmm9Mrv8k3iMDSsfpsbqKPs8V6OcSB3EnfuKOqrl0LhRSbAs-QM4r4cCQYBQD~Wa3xWrRbpyKhSZpMbvETXxYlndv4z4QDbgSklz6aDSONyWU8SpXyzZVGpsU6NAcGqGZ3Csu4JjH1aqFBucmuUxJ6DhHn4-tseApQqOrvVIXCDd2N~QRUcxvfXk5xTINiFn3d3eesp8n6jdsW6aRe~yZv-D6LG-36O0F0CH5LCFqOkofjS6yfyfU4YvbnU9A~-JTfPDXxYt0Rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055086&ar=2960054&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw054f8.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene organization of the <em>R. speciosa</em> chloroplast genome. Genes drawn inside the circle are transcribed clockwise while those drawn outside are transcribed counterclockwise. Different gene functional groups are colour coded. The map was drawn using CpGAVAS.</p></div></div></div>                            <h2 scrollto-destination=78055087 id="78055087" class="section-title" >4. Conclusion</h2>
<p>Asterids contain a quarter of all known species of flowering plants and four of the 10 most diverse families of flowering plants. We have contributed a new nuclear genome sequence that serves as only the third family represented by such a sequence within this important clade. By building new genomic resources for <em>R.</em><em>speciosa</em>, this study facilitates future investigation of Asterid evolution, particular with respect to flower colour production. Additionally, this new resource may facilitate future investigation of the genomic architecture of evolutionary intermediates (sensu Stebbins<a reveal-id="dsw054-B129" data-open="dsw054-B129" class="link link-ref link-reveal xref-bibr"><sup>129</sup></a>), as <em>R.</em><em>speciosa</em> is member to a clade that demonstrates a clear transition from bee to hummingbird to bat pollination and has features intermediate between the latter two stages.<a reveal-id="dsw054-B18" data-open="dsw054-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a></p>                            <h2 scrollto-destination=78055089 id="78055089" class="section-title" >Availability</h2>
<p>Raw nucleotide sequence data are available in the NCBI sequence read archive database (BioProject PRJNA326965) under the accession number SRP077633. The draft assembly is deposited in the NCBI whole-genome shotgun database (BioProject PRJNA326965) under the submission number SUB1661288 (released upon article acceptance).</p>                            <h2 scrollto-destination=78055091 id="78055091" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors thank Travis Glenn, Troy Kieren, Swarnali Louha, the Georgia Genomics Facility, and Brant Faircloth for generating and sequencing five DNA libraries that were used in this study. We additionally thank Nolan Kane for allowing us access to the DupPipe software and Mathew Sharples for assembling and annotating the Ruellia breedlovei chloroplast genome, which facilitated our cp assembly of Ruellia speciosa.</p>                            <h3 scrollto-destination=78055093 id="78055093" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=78055095 id="78055095" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054_Supp.zip?Expires=2147483647&Signature=OzwUbdrbll8GWrzlrYMohmjg2aIkCFsGZYTFXY~Kt6KuVc2twDadH8zYRPTVlbeirvm1zVBvwJJ62AMcnZZQRT6IkIiVdHs8RgqOLfsAeIdh1ClQTbbNSYHc87Of2VzfRrqiJSBJfZsk8gGsQOEMVqZ87ObAsjdKIt1D-l2pplHdY-9Bvx3S5C84jcZZN7spkpVLyp1Amn0LUMm0Z6563bsp1Ely9Dp~rQnN63w02~k0kobcBfHFjTanMPAC6QAKpc2tzoYt3l5iwjOgcp3pwV1K2ooGe9cXit7wNNM1qehybTZkN05Ngz~w8giL74bxPcg2lPmrHbg5qzF0W5XtaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h3 scrollto-destination=78055097 id="78055097" class="section-title" >Funding</h3>
<p>This work was supported by National Science Foundation’s Division of Environmental Biology (Award #1354963 and #1355138) to E.A.T. and Lucinda McDade.</p>                            <h2 scrollto-destination=78055099 id="78055099" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw054-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stevens</div> <div class="given-names">P.</div></div></div> <div class="year">2001</div>, Angiosperm phylogeny website. Version 12, July 2012 [and more or less continuously updated since].</div></div></div></div><div content-id="dsw054-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Funk</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Susanna</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Stuessy</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Bayer</div> <div class="given-names">R.</div></div></div> (Editors). <div class="year">2009</div>, Systematics, evolution and biogeography of the Compositae, International Association of Plant Taxonomy, Vienna, Austria.</div></div></div></div><div content-id="dsw054-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Duvick</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Fu</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Muppirala</div> <div class="given-names">U.</div></div></div>, et al.  . <div class="year">2008</div>, 
<div class="article-title">PlantGDB: a resource for comparative plant genomics,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">36</div>, <div class="fpage">D959</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PlantGDB%3A%20a%20resource%20for%20comparative%20plant%20genomics%2C&amp;author=J.%20Duvick&amp;author=A.%20Fu&amp;author=U.%20Muppirala&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res&amp;volume=36&amp;pages=D959-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm1041" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm1041" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm1041"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18063570" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Aversano</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Contaldi</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Ercolano</div> <div class="given-names">M.R.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">The <em>Solanum commersonii</em> genome sequence provides insights into adaptation to stress conditions and genome evolution of wild potato relatives,</div> <div class="source">Plant Cell</div> , <div class="volume">27</div>, <div class="fpage">954</div>–<div class="lpage">68</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Solanum%20commersonii%20genome%20sequence%20provides%20insights%20into%20adaptation%20to%20stress%20conditions%20and%20genome%20evolution%20of%20wild%20potato%20relatives%2C&amp;author=R.%20Aversano&amp;author=F.%20Contaldi&amp;author=M.R.%20Ercolano&amp;publication_year=2015&amp;journal=Plant%20Cell&amp;volume=27&amp;pages=954-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.114.135954" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.114.135954" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.114.135954"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25873387" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Miyatake</div> <div class="given-names">K.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Draft genome sequence of eggplant (<em>Solanum melongena</em> L.): the representative Solanum species indigenous to the old world</div>, <div class="source">DNA Res</div> ., <div class="volume">21</div>, <div class="fpage">649</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20eggplant%20(Solanum%20melongena%20L.)%3A%20the%20representative%20Solanum%20species%20indigenous%20to%20the%20old%20world&amp;author=H.%20Hirakawa&amp;author=K.%20Shirasawa&amp;author=K.%20Miyatake&amp;publication_year=2014&amp;journal=DNA%20Res&amp;volume=21&amp;pages=649-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsu027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsu027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsu027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25233906" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="collab">Tomato Genome Consortium</div>. <div class="year">2012</div>, 
<div class="article-title">The tomato genome sequence provides insights into fleshy fruit evolution</div>, <div class="source">Nature</div> , <div class="volume">485</div>, <div class="fpage">635</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11119" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11119" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11119"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22660326" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="collab">Potato Genome Sequencing Consortium</div>. <div class="year">2011</div>, 
<div class="article-title">Genome sequence and analysis of the tuber crop potato,</div> <div class="source">Nature</div> , <div class="volume">475</div>, <div class="fpage">189</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10158" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10158" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10158"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21743474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sierro</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Battey</div> <div class="given-names">J.N.</div></div>, <div class="name string-name"><div class="surname">Ouadi</div> <div class="given-names">S.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">Reference genomes and transcriptomes of <em>Nicotiana sylvestris</em> and <em>Nicotiana tomentosiformis</em>,</div> <div class="source">Genome Biol</div> ., <div class="volume">14</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reference%20genomes%20and%20transcriptomes%20of%20Nicotiana%20sylvestris%20and%20Nicotiana%20tomentosiformis%2C&amp;author=N.%20Sierro&amp;author=J.N.%20Battey&amp;author=S.%20Ouadi&amp;publication_year=2013&amp;journal=Genome%20Biol&amp;volume=14&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2013-14-6-r60" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-6-r60" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2013-14-6-r60"> </span></div> </div></div></div></div></div><div content-id="dsw054-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bombarely</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Rosli</div> <div class="given-names">H.G.</div></div>, <div class="name string-name"><div class="surname">Vrebalov</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Moffett</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Mueller</div> <div class="given-names">L.A.</div></div>, <div class="name string-name"><div class="surname">Martin</div> <div class="given-names">G.B.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">A draft genome sequence of <em>Nicotiana benthamiana</em> to enhance molecular plant-microbe biology research,</div> <div class="source">Mol. Plant Microbe Interact</div> ., <div class="volume">25</div>, <div class="fpage">1523</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20draft%20genome%20sequence%20of%20Nicotiana%20benthamiana%20to%20enhance%20molecular%20plant-microbe%20biology%20research%2C&amp;author=A.%20Bombarely&amp;author=H.G.%20Rosli&amp;author=J.%20Vrebalov&amp;author=P.%20Moffett&amp;author=L.A.%20Mueller&amp;author=G.B.%20Martin&amp;publication_year=2012&amp;journal=Mol.%20Plant%20Microbe%20Interact&amp;volume=25&amp;pages=1523-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-06-12-0148-TA" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-06-12-0148-TA" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-06-12-0148-TA"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22876960" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kim</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Park</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Yeom</div> <div class="given-names">S.-I.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Genome sequence of the hot pepper provides insights into the evolution of pungency in <em>Capsicum</em> species,</div> <div class="source">Nat. Genet</div> ., <div class="volume">46</div>, <div class="fpage">270</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20hot%20pepper%20provides%20insights%20into%20the%20evolution%20of%20pungency%20in%20Capsicum%20species%2C&amp;author=S.%20Kim&amp;author=M.%20Park&amp;author=S.-I.%20Yeom&amp;publication_year=2014&amp;journal=Nat.%20Genet&amp;volume=46&amp;pages=270-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2877" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2877" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2877"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24441736" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Qin</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Shen</div> <div class="given-names">Y.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Whole-genome sequencing of cultivated and wild peppers provides insights into <em>Capsicum</em> domestication and specialization,</div> <div class="source">Proc. Natl. Acad. Sci</div> ., <div class="volume">111</div>, <div class="fpage">5135</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20sequencing%20of%20cultivated%20and%20wild%20peppers%20provides%20insights%20into%20Capsicum%20domestication%20and%20specialization%2C&amp;author=C.%20Qin&amp;author=C.%20Yu&amp;author=Y.%20Shen&amp;publication_year=2014&amp;journal=Proc.%20Natl.%20Acad.%20Sci&amp;volume=111&amp;pages=5135-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1400975111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1400975111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1400975111"> </span></div> </div></div></div></div></div><div content-id="dsw054-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bombarely</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Moser</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Amrad</div> <div class="given-names">A.</div></div></div>, et al.  . <div class="year">2016</div>, 
<div class="article-title">Insight into the evolution of the <em>Solanaceae</em> from the parental genomes of <em>Petunia</em> hybrida,</div> <div class="source">Nat. Plants</div> , <div class="volume">2</div>, <div class="fpage">16074</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insight%20into%20the%20evolution%20of%20the%20Solanaceae%20from%20the%20parental%20genomes%20of%20Petunia%20hybrida%2C&amp;author=A.%20Bombarely&amp;author=M.%20Moser&amp;author=A.%20Amrad&amp;publication_year=2016&amp;journal=Nat.%20Plants&amp;volume=2&amp;pages=16074" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nplants.2016.74" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnplants.2016.74" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnplants.2016.74"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27255838" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hellsten</div> <div class="given-names">U.</div></div>, <div class="name string-name"><div class="surname">Wright</div> <div class="given-names">K.M.</div></div>, <div class="name string-name"><div class="surname">Jenkins</div> <div class="given-names">J.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">Fine-scale variation in meiotic recombination in <em>Mimulus</em> inferred from population shotgun sequencing,</div> <div class="source">Proc. Natl. Acad. Sci</div> ., <div class="volume">110</div>, <div class="fpage">19478</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fine-scale%20variation%20in%20meiotic%20recombination%20in%20Mimulus%20inferred%20from%20population%20shotgun%20sequencing%2C&amp;author=U.%20Hellsten&amp;author=K.M.%20Wright&amp;author=J.%20Jenkins&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci&amp;volume=110&amp;pages=19478-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1319032110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1319032110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1319032110"> </span></div> </div></div></div></div></div><div content-id="dsw054-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ibarra-Laclette</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Lyons</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Hernández-Guzmán</div> <div class="given-names">G.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">Architecture and evolution of a minute plant genome,</div> <div class="source">Nature</div> , <div class="volume">498</div>, <div class="fpage">94</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Architecture%20and%20evolution%20of%20a%20minute%20plant%20genome%2C&amp;author=E.%20Ibarra-Laclette&amp;author=E.%20Lyons&amp;author=G.%20Hern%C3%A1ndez-Guzm%C3%A1n&amp;publication_year=2013&amp;journal=Nature&amp;volume=498&amp;pages=94-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature12132" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature12132" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature12132"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23665961" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tripp</div> <div class="given-names">E.A.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Evolutionary relationships within the species-rich genus <em>Ruellia</em> (Acanthaceae)</div>, <div class="source">Syst. Botany</div> , <div class="volume">32</div>, <div class="fpage">628</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20relationships%20within%20the%20species-rich%20genus%20Ruellia%20(Acanthaceae)&amp;author=E.A.%20Tripp&amp;publication_year=2007&amp;journal=Syst.%20Botany&amp;volume=32&amp;pages=628-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1600/036364407782250625" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1600%2F036364407782250625" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1600%2F036364407782250625"> </span></div> </div></div></div></div></div><div content-id="dsw054-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tripp</div> <div class="given-names">E.A.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Taxonomic revision of Ruellia section <em>Chiropterophila</em> (Acanthaceae): a lineage of rare and endemic species from Mexico,</div> <div class="source">Syst. Botany</div> , <div class="volume">35</div>, <div class="fpage">629</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Taxonomic%20revision%20of%20Ruellia%20section%20Chiropterophila%20(Acanthaceae)%3A%20a%20lineage%20of%20rare%20and%20endemic%20species%20from%20Mexico%2C&amp;author=E.A.%20Tripp&amp;publication_year=2010&amp;journal=Syst.%20Botany&amp;volume=35&amp;pages=629-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1600/036364410792495845" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1600%2F036364410792495845" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1600%2F036364410792495845"> </span></div> </div></div></div></div></div><div content-id="dsw054-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tripp</div> <div class="given-names">E.A.</div></div>, <div class="name string-name"><div class="surname">Manos</div> <div class="given-names">P.S.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Is floral specialization an evolutionary dead‐end? Pollination system transitions in Ruellia (Acanthaceae),</div> <div class="source">Evolution</div> , <div class="volume">62</div>, <div class="fpage">1712</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Is%20floral%20specialization%20an%20evolutionary%20dead%E2%80%90end%3F%20Pollination%20system%20transitions%20in%20Ruellia%20(Acanthaceae)%2C&amp;author=E.A.%20Tripp&amp;author=P.S.%20Manos&amp;publication_year=2008&amp;journal=Evolution&amp;volume=62&amp;pages=1712-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/evo.2008.62.issue-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fevo.2008.62.issue-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fevo.2008.62.issue-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18410536" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tripp</div> <div class="given-names">E.A.</div></div>, <div class="name string-name"><div class="surname">McDade</div> <div class="given-names">L.A.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Time-calibrated phylogenies of hummingbirds and hummingbird-pollinated plants reject a hypothesis of diffuse co-evolution,</div> <div class="source">Aliso. Discipline · Botany</div> , <div class="volume">31</div>, <div class="fpage">89</div>–<div class="lpage">103</div>.</div></div></div></div><div content-id="dsw054-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Verwer</div> <div class="given-names">B.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">BD DACSDiVA Option, BD Biosciences, Pharmingen; Becton, Dickson and Company</div>, <div class="source">White Paper</div> , <div class="fpage">1</div>–<div class="lpage">22</div>.</div></div></div></div><div content-id="dsw054-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zimin</div> <div class="given-names">A.V.</div></div>, <div class="name string-name"><div class="surname">Marçais</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Puiu</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Roberts</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div>, <div class="name string-name"><div class="surname">Yorke</div> <div class="given-names">J.A.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">The MaSuRCA genome assembler,</div> <div class="source">Bioinformatics</div> , <div class="volume">29</div>, <div class="fpage">2669</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20MaSuRCA%20genome%20assembler%2C&amp;author=A.V.%20Zimin&amp;author=G.%20Mar%C3%A7ais&amp;author=D.%20Puiu&amp;author=M.%20Roberts&amp;author=S.L.%20Salzberg&amp;author=J.A.%20Yorke&amp;publication_year=2013&amp;journal=Bioinformatics&amp;volume=29&amp;pages=2669-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btt476" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtt476" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtt476"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23990416" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Luo</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Xie</div> <div class="given-names">Y.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler,</div> <div class="source">Gigascience</div> , <div class="volume">1</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SOAPdenovo2%3A%20an%20empirically%20improved%20memory-efficient%20short-read%20de%20novo%20assembler%2C&amp;author=R.%20Luo&amp;author=B.%20Liu&amp;author=Y.%20Xie&amp;publication_year=2012&amp;journal=Gigascience&amp;volume=1&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/2047-217X-1-18" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F2047-217X-1-18" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F2047-217X-1-18"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23587310" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Simpson</div> <div class="given-names">J.T.</div></div>, <div class="name string-name"><div class="surname">Wong</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Jackman</div> <div class="given-names">S.D.</div></div>, <div class="name string-name"><div class="surname">Schein</div> <div class="given-names">J.E.</div></div>, <div class="name string-name"><div class="surname">Jones</div> <div class="given-names">S.J.</div></div>, <div class="name string-name"><div class="surname">Birol</div> <div class="given-names">I.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">ABySS: a parallel assembler for short read sequence data,</div> <div class="source">Genome Res</div> ., <div class="volume">19</div>, <div class="fpage">1117</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ABySS%3A%20a%20parallel%20assembler%20for%20short%20read%20sequence%20data%2C&amp;author=J.T.%20Simpson&amp;author=K.%20Wong&amp;author=S.D.%20Jackman&amp;author=J.E.%20Schein&amp;author=S.J.%20Jones&amp;author=I.%20Birol&amp;publication_year=2009&amp;journal=Genome%20Res&amp;volume=19&amp;pages=1117-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.089532.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.089532.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.089532.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19251739" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bolger</div> <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">Lohse</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Usadel</div> <div class="given-names">B.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">2114</div>–<div class="lpage">2120</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;author=A.M.%20Bolger&amp;author=M.%20Lohse&amp;author=B.%20Usadel&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=2114-2120" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu170" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu170" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu170"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24695404" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Boetzer</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Henkel</div> <div class="given-names">C.V.</div></div>, <div class="name string-name"><div class="surname">Jansen</div> <div class="given-names">H.J.</div></div>, <div class="name string-name"><div class="surname">Butler</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Pirovano</div> <div class="given-names">W.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Scaffolding pre-assembled contigs using SSPACE,</div> <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">578</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Scaffolding%20pre-assembled%20contigs%20using%20SSPACE%2C&amp;author=M.%20Boetzer&amp;author=C.V.%20Henkel&amp;author=H.J.%20Jansen&amp;author=D.%20Butler&amp;author=W.%20Pirovano&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=578-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq683" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq683"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21149342" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Simão</div> <div class="given-names">F.A.</div></div>, <div class="name string-name"><div class="surname">Waterhouse</div> <div class="given-names">R.M.</div></div>, <div class="name string-name"><div class="surname">Ioannidis</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Kriventseva</div> <div class="given-names">E.V.</div></div>, <div class="name string-name"><div class="surname">Zdobnov</div> <div class="given-names">E.M.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs</div>, <div class="source">Bioinformatics</div> , <div class="volume">31</div>, <div class="fpage">3210</div>–<div class="lpage">3212</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BUSCO%3A%20assessing%20genome%20assembly%20and%20annotation%20completeness%20with%20single-copy%20orthologs&amp;author=F.A.%20Sim%C3%A3o&amp;author=R.M.%20Waterhouse&amp;author=P.%20Ioannidis&amp;author=E.V.%20Kriventseva&amp;author=E.M.%20Zdobnov&amp;publication_year=2015&amp;journal=Bioinformatics&amp;volume=31&amp;pages=3210-3212" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btv351" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtv351" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtv351"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26059717" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yoshida</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Schuenemann</div> <div class="given-names">V.J.</div></div>, <div class="name string-name"><div class="surname">Cano</div> <div class="given-names">L.M.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">The rise and fall of the <em>Phytophthora infestans</em> lineage that triggered the Irish potato famine,</div> <div class="source">Elife</div> , <div class="volume">2</div>, <div class="fpage">e00731</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20rise%20and%20fall%20of%20the%20Phytophthora%20infestans%20lineage%20that%20triggered%20the%20Irish%20potato%20famine%2C&amp;author=K.%20Yoshida&amp;author=V.J.%20Schuenemann&amp;author=L.M.%20Cano&amp;publication_year=2013&amp;journal=Elife&amp;volume=2&amp;pages=e00731" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23741619" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Fast and accurate short read alignment with Burrows–Wheeler transform,</div> <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1754</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows%E2%80%93Wheeler%20transform%2C&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Handsaker</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Wysoker</div> <div class="given-names">A.</div></div></div>, et al.  . <div class="year">2009</div>, 
<div class="article-title">The sequence alignment/map format and SAMtools,</div> <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">2078</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20sequence%20alignment%2Fmap%20format%20and%20SAMtools%2C&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Barker</div> <div class="given-names">M.S.</div></div>, <div class="name string-name"><div class="surname">Dlugosch</div> <div class="given-names">K.M.</div></div>, <div class="name string-name"><div class="surname">Dinh</div> <div class="given-names">L.</div></div></div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">EvoPipes. net: bioinformatic tools for ecological and evolutionary genomics</div>, <div class="source">Evol. Bioinform</div> ., <div class="volume">6</div>, <div class="fpage">143</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EvoPipes.%20net%3A%20bioinformatic%20tools%20for%20ecological%20and%20evolutionary%20genomics&amp;author=M.S.%20Barker&amp;author=K.M.%20Dlugosch&amp;author=L.%20Dinh&amp;publication_year=2010&amp;journal=Evol.%20Bioinform&amp;volume=6&amp;pages=143" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4137/EBO" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4137%2FEBO" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4137%2FEBO"> </span></div> </div></div></div></div></div><div content-id="dsw054-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">PAML 4: phylogenetic analysis by maximum likelihood,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">24</div>, <div class="fpage">1586</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PAML%204%3A%20phylogenetic%20analysis%20by%20maximum%20likelihood%2C&amp;author=Z.%20Yang&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol&amp;volume=24&amp;pages=1586-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm088" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm088" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm088"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17483113" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chaudhuri</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Marron</div> <div class="given-names">J.S.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">SiZer for exploration of structures in curves,</div> <div class="source">J. Am. Stat. Assoc</div> ., <div class="volume">94</div>, <div class="fpage">807</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SiZer%20for%20exploration%20of%20structures%20in%20curves%2C&amp;author=P.%20Chaudhuri&amp;author=J.S.%20Marron&amp;publication_year=1999&amp;journal=J.%20Am.%20Stat.%20Assoc&amp;volume=94&amp;pages=807-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/01621459.1999.10474186" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F01621459.1999.10474186" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F01621459.1999.10474186"> </span></div> </div></div></div></div></div><div content-id="dsw054-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marçais</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Kingsford</div> <div class="given-names">C.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">A fast, lock-free approach for efficient parallel counting of occurrences of k-mers,</div> <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">764</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20fast%2C%20lock-free%20approach%20for%20efficient%20parallel%20counting%20of%20occurrences%20of%20k-mers%2C&amp;author=G.%20Mar%C3%A7ais&amp;author=C.%20Kingsford&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=764-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Campbell</div> <div class="given-names">M.S.</div></div>, <div class="name string-name"><div class="surname">Law</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Holt</div> <div class="given-names">C.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">MAKER-P: a tool kit for the rapid creation, management, and quality control of plant genome annotations,</div> <div class="source">Plant Physiol</div> ., <div class="volume">164</div>, <div class="fpage">513</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MAKER-P%3A%20a%20tool%20kit%20for%20the%20rapid%20creation%2C%20management%2C%20and%20quality%20control%20of%20plant%20genome%20annotations%2C&amp;author=M.S.%20Campbell&amp;author=M.%20Law&amp;author=C.%20Holt&amp;publication_year=2014&amp;journal=Plant%20Physiol&amp;volume=164&amp;pages=513-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.113.230144" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.113.230144" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.113.230144"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24306534" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jurka</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Kapitonov</div> <div class="given-names">V.V.</div></div>, <div class="name string-name"><div class="surname">Pavlicek</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Klonowski</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Kohany</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Walichiewicz</div> <div class="given-names">J.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Repbase Update, a database of eukaryotic repetitive elements,</div> <div class="source">Cytogenet. Genome Res</div> ., <div class="volume">110</div>, <div class="fpage">462</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repbase%20Update%2C%20a%20database%20of%20eukaryotic%20repetitive%20elements%2C&amp;author=J.%20Jurka&amp;author=V.V.%20Kapitonov&amp;author=A.%20Pavlicek&amp;author=P.%20Klonowski&amp;author=O.%20Kohany&amp;author=J.%20Walichiewicz&amp;publication_year=2005&amp;journal=Cytogenet.%20Genome%20Res&amp;volume=110&amp;pages=462-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000084979" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000084979" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000084979"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16093699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Smit</div> <div class="given-names">A.F.</div></div>, <div class="name string-name"><div class="surname">Hubley</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Green</div> <div class="given-names">P.</div></div></div> <div class="year">1996</div>, RepeatMasker. <a class="link link-uri" href="http://www.repeatmasker.org" target="_blank">http://www.repeatmasker.org</a>. (06 Decemebr 2016, date last accessed).</div></div></div></div><div content-id="dsw054-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Korf</div> <div class="given-names">I.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Gene finding in novel genomes</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">5</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20finding%20in%20novel%20genomes&amp;author=I.%20Korf&amp;publication_year=2004&amp;journal=BMC%20Bioinformatics&amp;volume=5&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-5-59" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-5-59" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-5-59"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14706121" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Waack</div> <div class="given-names">S.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Gene prediction with a hidden Markov model and a new intron submodel,</div> <div class="source">Bioinformatics</div> , <div class="volume">19</div>, <div class="fpage">ii215</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20prediction%20with%20a%20hidden%20Markov%20model%20and%20a%20new%20intron%20submodel%2C&amp;author=M.%20Stanke&amp;author=S.%20Waack&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=ii215-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg1080" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg1080" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg1080"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14534192" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Proost</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Van Bel</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Sterck</div> <div class="given-names">L.</div></div></div>, et al.  . <div class="year">2009</div>, 
<div class="article-title">PLAZA: a comparative genomics resource to study gene and genome evolution in plants,</div> <div class="source">Plant Cell</div> , <div class="volume">21</div>, <div class="fpage">3718</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PLAZA%3A%20a%20comparative%20genomics%20resource%20to%20study%20gene%20and%20genome%20evolution%20in%20plants%2C&amp;author=S.%20Proost&amp;author=M.%20Van%20Bel&amp;author=L.%20Sterck&amp;publication_year=2009&amp;journal=Plant%20Cell&amp;volume=21&amp;pages=3718-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.109.071506" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.109.071506" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.109.071506"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20040540" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Denton</div> <div class="given-names">J.F.</div></div>, <div class="name string-name"><div class="surname">Lugo-Martinez</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Tucker</div> <div class="given-names">A.E.</div></div>, <div class="name string-name"><div class="surname">Schrider</div> <div class="given-names">D.R.</div></div>, <div class="name string-name"><div class="surname">Warren</div> <div class="given-names">W.C.</div></div>, <div class="name string-name"><div class="surname">Hahn</div> <div class="given-names">M.W.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Extensive error in the number of genes inferred from draft genome assemblies,</div> <div class="source">PLoS Comput. Biol</div> ., <div class="volume">10</div>, <div class="fpage">e1003998</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensive%20error%20in%20the%20number%20of%20genes%20inferred%20from%20draft%20genome%20assemblies%2C&amp;author=J.F.%20Denton&amp;author=J.%20Lugo-Martinez&amp;author=A.E.%20Tucker&amp;author=D.R.%20Schrider&amp;author=W.C.%20Warren&amp;author=M.W.%20Hahn&amp;publication_year=2014&amp;journal=PLoS%20Comput.%20Biol&amp;volume=10&amp;pages=e1003998" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1003998" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1003998" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1003998"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25474019" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Upadhyay</div> <div class="given-names">A.K.</div></div>, <div class="name string-name"><div class="surname">Chacko</div> <div class="given-names">A.R.</div></div>, <div class="name string-name"><div class="surname">Gandhimathi</div> <div class="given-names">A.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Genome sequencing of herb Tulsi (<em>Ocimum tenuiflorum</em>) unravels key genes behind its strong medicinal properties,</div> <div class="source">BMC Plant Biol</div> ., <div class="volume">15</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequencing%20of%20herb%20Tulsi%20(Ocimum%20tenuiflorum)%20unravels%20key%20genes%20behind%20its%20strong%20medicinal%20properties%2C&amp;author=A.K.%20Upadhyay&amp;author=A.R.%20Chacko&amp;author=A.%20Gandhimathi&amp;publication_year=2015&amp;journal=BMC%20Plant%20Biol&amp;volume=15&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12870-015-0562-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12870-015-0562-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12870-015-0562-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25592487" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Lu</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Luo</div> <div class="given-names">Z.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">GMATo: a novel tool for the identification and analysis of microsatellites in large genomes,</div> <div class="source">Bioinformation</div> , <div class="volume">9</div>, <div class="fpage">541</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GMATo%3A%20a%20novel%20tool%20for%20the%20identification%20and%20analysis%20of%20microsatellites%20in%20large%20genomes%2C&amp;author=X.%20Wang&amp;author=P.%20Lu&amp;author=Z.%20Luo&amp;publication_year=2013&amp;journal=Bioinformation&amp;volume=9&amp;pages=541-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.6026/bioinformation" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.6026%2Fbioinformation" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.6026%2Fbioinformation"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23861572" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cheng</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">B.</div></div></div>, et al.  . <div class="year">2016</div>, 
<div class="article-title">A comprehensive characterization of simple sequence repeats in pepper genomes provides valuable resources for marker development in Capsicum,</div> <div class="source">Sci. Rep</div> ., <div class="volume">6</div>, <div class="fpage">18919</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comprehensive%20characterization%20of%20simple%20sequence%20repeats%20in%20pepper%20genomes%20provides%20valuable%20resources%20for%20marker%20development%20in%20Capsicum%2C&amp;author=J.%20Cheng&amp;author=Z.%20Zhao&amp;author=B.%20Li&amp;publication_year=2016&amp;journal=Sci.%20Rep&amp;volume=6&amp;pages=18919" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep18919" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep18919" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep18919"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26739748" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Camacho</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Coulouris</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Avagyan</div> <div class="given-names">V.</div></div></div>, et al.  . <div class="year">2009</div>, 
<div class="article-title">BLAST+: architecture and applications,</div> <div class="source">BMC Bioinformatics</div> , <div class="volume">10</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BLAST%2B%3A%20architecture%20and%20applications%2C&amp;author=C.%20Camacho&amp;author=G.%20Coulouris&amp;author=V.%20Avagyan&amp;publication_year=2009&amp;journal=BMC%20Bioinformatics&amp;volume=10&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-10-421" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-10-421" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-10-421"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19118496" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Boeckmann</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Bairoch</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Apweiler</div> <div class="given-names">R.</div></div></div>, et al.  . <div class="year">2003</div>, 
<div class="article-title">The SWISS-PROT protein knowledgebase and its supplement TrEMBL in 2003,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">31</div>, <div class="fpage">365</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20SWISS-PROT%20protein%20knowledgebase%20and%20its%20supplement%20TrEMBL%20in%202003%2C&amp;author=B.%20Boeckmann&amp;author=A.%20Bairoch&amp;author=R.%20Apweiler&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res&amp;volume=31&amp;pages=365-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg095" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg095" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg095"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12520024" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="collab">UniProt Consortium</div>. <div class="year">2014</div>, 
<div class="article-title">UniProt: a hub for protein information</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">D204</div>–<div class="lpage">D212</div>.<div class="citation-links"><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25348405" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="collab">Gene Ontology Consortium</div>. <div class="year">2004</div>, 
<div class="article-title">The Gene Ontology (GO) database and informatics resource</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">32</div>, <div class="fpage">D258</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh036" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh036" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh036"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14681407" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Boutet</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Lieberherr</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Tognolli</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Schneider</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Bairoch</div> <div class="given-names">A.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">UniProtKB/Swiss-Prot: The manually annotated section of the UniProt KnowledgeBase</div>. <div class="source">Methods Mol. Biol</div> , <div class="volume">406</div>, <div class="fpage">89</div>–<div class="lpage">112</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=UniProtKB%2FSwiss-Prot%3A%20The%20manually%20annotated%20section%20of%20the%20UniProt%20KnowledgeBase&amp;author=E.%20Boutet&amp;author=D.%20Lieberherr&amp;author=M.%20Tognolli&amp;author=M.%20Schneider&amp;author=A.%20Bairoch&amp;publication_year=2007&amp;journal=Methods%20Mol.%20Biol&amp;volume=406&amp;pages=89-112" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18287689" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Du</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Ling</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Su</div> <div class="given-names">Z.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">AgriGO: a GO analysis toolkit for the agricultural community</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">38</div>, <div class="fpage">W64</div>–<div class="lpage">W70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AgriGO%3A%20a%20GO%20analysis%20toolkit%20for%20the%20agricultural%20community&amp;author=Z.%20Du&amp;author=X.%20Zhou&amp;author=Y.%20Ling&amp;author=Z.%20Zhang&amp;author=Z.%20Su&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res&amp;volume=38&amp;pages=W64-W70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq310" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq310" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq310"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20435677" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ye</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Fang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Zheng</div> <div class="given-names">H.</div></div></div>, et al.  . <div class="year">2006</div>, 
<div class="article-title">WEGO: a web tool for plotting GO annotations,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">34</div>, <div class="fpage">W293</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=WEGO%3A%20a%20web%20tool%20for%20plotting%20GO%20annotations%2C&amp;author=J.%20Ye&amp;author=L.%20Fang&amp;author=H.%20Zheng&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res&amp;volume=34&amp;pages=W293-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16845012" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Waterhouse</div> <div class="given-names">R.M.</div></div>, <div class="name string-name"><div class="surname">Tegenfeldt</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zdobnov</div> <div class="given-names">E.M.</div></div>, <div class="name string-name"><div class="surname">Kriventseva</div> <div class="given-names">E.V.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">OrthoDB: a hierarchical catalog of animal, fungal and bacterial orthologs,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">D358</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=OrthoDB%3A%20a%20hierarchical%20catalog%20of%20animal%2C%20fungal%20and%20bacterial%20orthologs%2C&amp;author=R.M.%20Waterhouse&amp;author=F.%20Tegenfeldt&amp;author=J.%20Li&amp;author=E.M.%20Zdobnov&amp;author=E.V.%20Kriventseva&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=D358-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks1116" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1116" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks1116"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23180791" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wegrzyn</div> <div class="given-names">J.L.</div></div>, <div class="name string-name"><div class="surname">Liechty</div> <div class="given-names">J.D.</div></div>, <div class="name string-name"><div class="surname">Stevens</div> <div class="given-names">K.A.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Unique features of the loblolly pine (<em>Pinus taeda</em> L.) megagenome revealed through sequence annotation,</div> <div class="source">Genetics</div> , <div class="volume">196</div>, <div class="fpage">891</div>–<div class="lpage">909</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unique%20features%20of%20the%20loblolly%20pine%20(Pinus%20taeda%20L.)%20megagenome%20revealed%20through%20sequence%20annotation%2C&amp;author=J.L.%20Wegrzyn&amp;author=J.D.%20Liechty&amp;author=K.A.%20Stevens&amp;publication_year=2014&amp;journal=Genetics&amp;volume=196&amp;pages=891-909" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.113.159996" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.113.159996" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.113.159996"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24653211" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Godzik</div> <div class="given-names">A.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">CD-HIT: a fast program for clustering and comparing large sets of protein or nucleotide sequences,</div> <div class="source">Bioinformatics</div> , <div class="volume">22</div>, <div class="fpage">1658</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CD-HIT%3A%20a%20fast%20program%20for%20clustering%20and%20comparing%20large%20sets%20of%20protein%20or%20nucleotide%20sequences%2C&amp;author=W.%20Li&amp;author=A.%20Godzik&amp;publication_year=2006&amp;journal=Bioinformatics&amp;volume=22&amp;pages=1658-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btl158" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl158" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtl158"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16731699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lechner</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Findeiß</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Steiner</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Marz</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Stadler</div> <div class="given-names">P.F.</div></div>, <div class="name string-name"><div class="surname">Prohaska</div> <div class="given-names">S.J.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Proteinortho: detection of (co-) orthologs in large-scale analysis,</div> <div class="source">BMC Bioinformatics</div> , <div class="volume">12</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Proteinortho%3A%20detection%20of%20(co-)%20orthologs%20in%20large-scale%20analysis%2C&amp;author=M.%20Lechner&amp;author=S.%20Findei%C3%9F&amp;author=L.%20Steiner&amp;author=M.%20Marz&amp;author=P.F.%20Stadler&amp;author=S.J.%20Prohaska&amp;publication_year=2011&amp;journal=BMC%20Bioinformatics&amp;volume=12&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-12-124" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-12-124" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-12-124"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21199577" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Katoh</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Misawa</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Kuma</div> <div class="given-names">K.I.</div></div>, <div class="name string-name"><div class="surname">Miyata</div> <div class="given-names">T.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">30</div>, <div class="fpage">3059</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MAFFT%3A%20a%20novel%20method%20for%20rapid%20multiple%20sequence%20alignment%20based%20on%20fast%20Fourier%20transform%2C&amp;author=K.%20Katoh&amp;author=K.%20Misawa&amp;author=K.I.%20Kuma&amp;author=T.%20Miyata&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res&amp;volume=30&amp;pages=3059-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkf436" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkf436" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkf436"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12136088" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huelsenbeck</div> <div class="given-names">J.P.</div></div>, <div class="name string-name"><div class="surname">Hillis</div> <div class="given-names">D.M.</div></div>, <div class="name string-name"><div class="surname">Jones</div> <div class="given-names">R.</div></div></div> <div class="year">1996</div>, Parametric bootstrapping in molecular phylogenetics: applications and performance. In: <div class="source">Molecular Zoology: Advances, Strategies, and Protocols</div> . 
<div class="publisher-name">Wiley-Liss</div>, 
<div class="publisher-loc">New York</div>, pp. <div class="fpage">19</div>–<div class="lpage">45</div>.</div></div></div></div><div content-id="dsw054-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Le</div> <div class="given-names">S.Q.</div></div>, <div class="name string-name"><div class="surname">Gascuel</div> <div class="given-names">O.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">An improved general amino acid replacement matrix,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">25</div>, <div class="fpage">1307</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20improved%20general%20amino%20acid%20replacement%20matrix%2C&amp;author=S.Q.%20Le&amp;author=O.%20Gascuel&amp;publication_year=2008&amp;journal=Mol.%20Biol.%20Evol&amp;volume=25&amp;pages=1307-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msn067" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsn067" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsn067"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18367465" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Stecher</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Peterson</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Filipski</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">MEGA6: molecular evolutionary genetics analysis version 6.0,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">30</div>, <div class="fpage">2725</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA6%3A%20molecular%20evolutionary%20genetics%20analysis%20version%206.0%2C&amp;author=K.%20Tamura&amp;author=G.%20Stecher&amp;author=D.%20Peterson&amp;author=A.%20Filipski&amp;author=S.%20Kumar&amp;publication_year=2013&amp;journal=Mol.%20Biol.%20Evol&amp;volume=30&amp;pages=2725-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mst197" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmst197" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmst197"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24132122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Siltberg</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Liberles</div> <div class="given-names">D.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">A simple covarion‐based approach to analyse nucleotide substitution rates,</div> <div class="source">J. Evol. Biol</div> ., <div class="volume">15</div>, <div class="fpage">588</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%20covarion%E2%80%90based%20approach%20to%20analyse%20nucleotide%20substitution%20rates%2C&amp;author=J.%20Siltberg&amp;author=D.%20Liberles&amp;publication_year=2002&amp;journal=J.%20Evol.%20Biol&amp;volume=15&amp;pages=588-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1420-9101.2002.00416.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1420-9101.2002.00416.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1420-9101.2002.00416.x"> </span></div> </div></div></div></div></div><div content-id="dsw054-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rhee</div> <div class="given-names">S.Y.</div></div>, <div class="name string-name"><div class="surname">Beavis</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Berardini</div> <div class="given-names">T.Z.</div></div></div>, et al.  . <div class="year">2003</div>, 
<div class="article-title">The Arabidopsis information resource (TAIR): a model organism database providing a centralized, curated gateway to Arabidopsis biology, research materials and community,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">31</div>, <div class="fpage">224</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Arabidopsis%20information%20resource%20(TAIR)%3A%20a%20model%20organism%20database%20providing%20a%20centralized%2C%20curated%20gateway%20to%20Arabidopsis%20biology%2C%20research%20materials%20and%20community%2C&amp;author=S.Y.%20Rhee&amp;author=W.%20Beavis&amp;author=T.Z.%20Berardini&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res&amp;volume=31&amp;pages=224-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg076" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg076" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg076"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12519987" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Finn</div> <div class="given-names">R.D.</div></div>, <div class="name string-name"><div class="surname">Mistry</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Schuster-Böckler</div> <div class="given-names">B.</div></div></div>, et al.  . <div class="year">2006</div>, 
<div class="article-title">Pfam: clans, web tools and services,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">34</div>, <div class="fpage">D247</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pfam%3A%20clans%2C%20web%20tools%20and%20services%2C&amp;author=R.D.%20Finn&amp;author=J.%20Mistry&amp;author=B.%20Schuster-B%C3%B6ckler&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res&amp;volume=34&amp;pages=D247-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj149" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj149" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj149"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16381856" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cominelli</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Galbiati</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Vavasseur</div> <div class="given-names">A.</div></div></div>, et al.  . <div class="year">2005</div>, 
<div class="article-title">A guard-cell-specific MYB transcription factor regulates stomatal movements and plant drought tolerance,</div> <div class="source">Curr. Biol</div> ., <div class="volume">15</div>, <div class="fpage">1196</div>–<div class="lpage">200</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20guard-cell-specific%20MYB%20transcription%20factor%20regulates%20stomatal%20movements%20and%20plant%20drought%20tolerance%2C&amp;author=E.%20Cominelli&amp;author=M.%20Galbiati&amp;author=A.%20Vavasseur&amp;publication_year=2005&amp;journal=Curr.%20Biol&amp;volume=15&amp;pages=1196-200" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cub.2005.05.048" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cub.2005.05.048" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cub.2005.05.048"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16005291" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Gu</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Qu</div> <div class="given-names">L.J.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Overexpression of the wounding‐responsive gene AtMYB15 activates the shikimate pathway in Arabidopsis,</div> <div class="source">J. Integr. Plant Biol</div> ., <div class="volume">48</div>, <div class="fpage">1084</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Overexpression%20of%20the%20wounding%E2%80%90responsive%20gene%20AtMYB15%20activates%20the%20shikimate%20pathway%20in%20Arabidopsis%2C&amp;author=Y.%20Chen&amp;author=X.%20Zhang&amp;author=W.%20Wu&amp;author=Z.%20Chen&amp;author=H.%20Gu&amp;author=L.J.%20Qu&amp;publication_year=2006&amp;journal=J.%20Integr.%20Plant%20Biol&amp;volume=48&amp;pages=1084-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jipb.2006.48.issue-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjipb.2006.48.issue-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjipb.2006.48.issue-9"> </span></div> </div></div></div></div></div><div content-id="dsw054-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gigolashvili</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Berger</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Mock</div> <div class="given-names">H.P.</div></div>, <div class="name string-name"><div class="surname">Müller</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Weisshaar</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Flügge</div> <div class="given-names">U.I.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">The transcription factor HIG1/MYB51 regulates indolic glucosinolate biosynthesis in <em>Arabidopsis thaliana</em>,</div> <div class="source">Plant J</div> ., <div class="volume">50</div>, <div class="fpage">886</div>–<div class="lpage">901</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20transcription%20factor%20HIG1%2FMYB51%20regulates%20indolic%20glucosinolate%20biosynthesis%20in%20Arabidopsis%20thaliana%2C&amp;author=T.%20Gigolashvili&amp;author=B.%20Berger&amp;author=H.P.%20Mock&amp;author=C.%20M%C3%BCller&amp;author=B.%20Weisshaar&amp;author=U.I.%20Fl%C3%BCgge&amp;publication_year=2007&amp;journal=Plant%20J&amp;volume=50&amp;pages=886-901" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-313X.2007.03099.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.2007.03099.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-313X.2007.03099.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17461791" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hirai</div> <div class="given-names">M.Y.</div></div>, <div class="name string-name"><div class="surname">Sugiyama</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Sawada</div> <div class="given-names">Y.</div></div></div>, et al.  . <div class="year">2007</div>, 
<div class="article-title">Omics-based identification of Arabidopsis Myb transcription factors regulating aliphatic glucosinolate biosynthesis,</div> <div class="source">Proc. Natl. Acad. Sci</div> ., <div class="volume">104</div>, <div class="fpage">6478</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Omics-based%20identification%20of%20Arabidopsis%20Myb%20transcription%20factors%20regulating%20aliphatic%20glucosinolate%20biosynthesis%2C&amp;author=M.Y.%20Hirai&amp;author=K.%20Sugiyama&amp;author=Y.%20Sawada&amp;publication_year=2007&amp;journal=Proc.%20Natl.%20Acad.%20Sci&amp;volume=104&amp;pages=6478-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0611629104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0611629104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0611629104"> </span></div> </div></div></div></div></div><div content-id="dsw054-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gigolashvili</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Engqvist</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Yatusevich</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Müller</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Flügge</div> <div class="given-names">U.I.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">HAG2/MYB76 and HAG3/MYB29 exert a specific and coordinated control on the regulation of aliphatic glucosinolate biosynthesis in <em>Arabidopsis thaliana</em>,</div> <div class="source">New Phytol</div> ., <div class="volume">177</div>, <div class="fpage">627</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HAG2%2FMYB76%20and%20HAG3%2FMYB29%20exert%20a%20specific%20and%20coordinated%20control%20on%20the%20regulation%20of%20aliphatic%20glucosinolate%20biosynthesis%20in%20Arabidopsis%20thaliana%2C&amp;author=T.%20Gigolashvili&amp;author=M.%20Engqvist&amp;author=R.%20Yatusevich&amp;author=C.%20M%C3%BCller&amp;author=U.I.%20Fl%C3%BCgge&amp;publication_year=2008&amp;journal=New%20Phytol&amp;volume=177&amp;pages=627-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.2008.177.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.2008.177.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.2008.177.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18042203" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Matus</div> <div class="given-names">J.T.</div></div>, <div class="name string-name"><div class="surname">Aquea</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Arce-Johnson</div> <div class="given-names">P.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Analysis of the grape MYB R2R3 subfamily reveals expanded wine quality-related clades and conserved gene structure organization across Vitis and Arabidopsis genomes,</div> <div class="source">BMC Plant Biol</div> ., <div class="volume">8</div>, <div class="fpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20grape%20MYB%20R2R3%20subfamily%20reveals%20expanded%20wine%20quality-related%20clades%20and%20conserved%20gene%20structure%20organization%20across%20Vitis%20and%20Arabidopsis%20genomes%2C&amp;author=J.T.%20Matus&amp;author=F.%20Aquea&amp;author=P.%20Arce-Johnson&amp;publication_year=2008&amp;journal=BMC%20Plant%20Biol&amp;volume=8&amp;pages=83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-8-83" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-8-83" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-8-83"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18647406" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Van der Ent</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Verhagen</div> <div class="given-names">B.W.</div></div>, <div class="name string-name"><div class="surname">Van Doorn</div> <div class="given-names">R.</div></div></div>, et al.  . <div class="year">2008</div>, 
<div class="article-title">MYB72 is required in early signaling steps of rhizobacteria-induced systemic resistance in Arabidopsis,</div> <div class="source">Plant Physiol</div> ., <div class="volume">146</div>, <div class="fpage">1293</div>–<div class="lpage">304</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MYB72%20is%20required%20in%20early%20signaling%20steps%20of%20rhizobacteria-induced%20systemic%20resistance%20in%20Arabidopsis%2C&amp;author=S.%20Van%20der%20Ent&amp;author=B.W.%20Verhagen&amp;author=R.%20Van%20Doorn&amp;publication_year=2008&amp;journal=Plant%20Physiol&amp;volume=146&amp;pages=1293-304" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.107.113829" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.107.113829" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.107.113829"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18218967" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Thompson</div> <div class="given-names">A.</div></div></div>, et al.  . <div class="year">2009</div>, 
<div class="article-title">Arabidopsis MYB30 is a direct target of BES1 and cooperates with BES1 to regulate brassinosteroid‐induced gene expression,</div> <div class="source">Plant J</div> ., <div class="volume">58</div>, <div class="fpage">275</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20MYB30%20is%20a%20direct%20target%20of%20BES1%20and%20cooperates%20with%20BES1%20to%20regulate%20brassinosteroid%E2%80%90induced%20gene%20expression%2C&amp;author=L.%20Li&amp;author=X.%20Yu&amp;author=A.%20Thompson&amp;publication_year=2009&amp;journal=Plant%20J&amp;volume=58&amp;pages=275-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2009.58.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2009.58.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2009.58.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19170933" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Segarra</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Van der Ent</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Trillas</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Pieterse</div> <div class="given-names">C.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">MYB72, a node of convergence in induced systemic resistance triggered by a fungal and a bacterial beneficial microbe,</div> <div class="source">Plant Biol</div> ., <div class="volume">11</div>, <div class="fpage">90</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MYB72%2C%20a%20node%20of%20convergence%20in%20induced%20systemic%20resistance%20triggered%20by%20a%20fungal%20and%20a%20bacterial%20beneficial%20microbe%2C&amp;author=G.%20Segarra&amp;author=S.%20Van%20der%20Ent&amp;author=I.%20Trillas&amp;author=C.%20Pieterse&amp;publication_year=2009&amp;journal=Plant%20Biol&amp;volume=11&amp;pages=90-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/plb.2008.11.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fplb.2008.11.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fplb.2008.11.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19121118" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Seo</div> <div class="given-names">P.J.</div></div>, <div class="name string-name"><div class="surname">Park</div> <div class="given-names">C.-M.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Auxin homeostasis during lateral root development under drought condition,</div> <div class="source">Plant Signal. Behav</div> ., <div class="volume">4</div>, <div class="fpage">1002</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Auxin%20homeostasis%20during%20lateral%20root%20development%20under%20drought%20condition%2C&amp;author=P.J.%20Seo&amp;author=C.-M.%20Park&amp;publication_year=2009&amp;journal=Plant%20Signal.%20Behav&amp;volume=4&amp;pages=1002-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/psb.4.10.9716" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Fpsb.4.10.9716" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Fpsb.4.10.9716"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19826230" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Cao</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Qu</div> <div class="given-names">L.-J.</div></div>, <div class="name string-name"><div class="surname">Gu</div> <div class="given-names">H.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Characterization of Arabidopsis MYB transcription factor gene AtMYB17 and its possible regulation by LEAFY and AGL15,</div> <div class="source">J. Genet. Genomics</div> , <div class="volume">36</div>, <div class="fpage">99</div>–<div class="lpage">107</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20Arabidopsis%20MYB%20transcription%20factor%20gene%20AtMYB17%20and%20its%20possible%20regulation%20by%20LEAFY%20and%20AGL15%2C&amp;author=Y.%20Zhang&amp;author=G.%20Cao&amp;author=L.-J.%20Qu&amp;author=H.%20Gu&amp;publication_year=2009&amp;journal=J.%20Genet.%20Genomics&amp;volume=36&amp;pages=99-107" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1673-8527(08)60096-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1673-8527(08)60096-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1673-8527(08)60096-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19232308" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dubos</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Stracke</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Grotewold</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Weisshaar</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Martin</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Lepiniec</div> <div class="given-names">L.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">MYB transcription factors in Arabidopsis,</div> <div class="source">Trends Plant Sci</div> ., <div class="volume">15</div>, <div class="fpage">573</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MYB%20transcription%20factors%20in%20Arabidopsis%2C&amp;author=C.%20Dubos&amp;author=R.%20Stracke&amp;author=E.%20Grotewold&amp;author=B.%20Weisshaar&amp;author=C.%20Martin&amp;author=L.%20Lepiniec&amp;publication_year=2010&amp;journal=Trends%20Plant%20Sci&amp;volume=15&amp;pages=573-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2010.06.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2010.06.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2010.06.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20674465" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ambawat</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Yadav</div> <div class="given-names">N.R.</div></div>, <div class="name string-name"><div class="surname">Yadav</div> <div class="given-names">R.C.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">MYB transcription factor genes as regulators for plant responses: an overview,</div> <div class="source">Physiol. Mol. Biol. Plants</div> , <div class="volume">19</div>, <div class="fpage">307</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MYB%20transcription%20factor%20genes%20as%20regulators%20for%20plant%20responses%3A%20an%20overview%2C&amp;author=S.%20Ambawat&amp;author=P.%20Sharma&amp;author=N.R.%20Yadav&amp;author=R.C.%20Yadav&amp;publication_year=2013&amp;journal=Physiol.%20Mol.%20Biol.%20Plants&amp;volume=19&amp;pages=307-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s12298-013-0179-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12298-013-0179-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs12298-013-0179-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24431500" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Araki</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Hasumi</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Nishizawa</div> <div class="given-names">O.I.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">Novel bioresources for studies of <em>Brassica oleracea</em>: identification of a kale MYB transcription factor responsible for glucosinolate production,</div> <div class="source">Plant Biotech. J</div> ., <div class="volume">11</div>, <div class="fpage">1017</div>–<div class="lpage">27</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20bioresources%20for%20studies%20of%20Brassica%20oleracea%3A%20identification%20of%20a%20kale%20MYB%20transcription%20factor%20responsible%20for%20glucosinolate%20production%2C&amp;author=R.%20Araki&amp;author=A.%20Hasumi&amp;author=O.I.%20Nishizawa&amp;publication_year=2013&amp;journal=Plant%20Biotech.%20J&amp;volume=11&amp;pages=1017-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.12095" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.12095" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.12095"> </span></div> </div></div></div></div></div><div content-id="dsw054-B75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Kang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Kang</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Gu</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Qin</div> <div class="given-names">G.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">AtMYB14 regulates cold tolerance in Arabidopsis,</div> <div class="source">Plant Mol. Biol. Rep</div> ., <div class="volume">31</div>, <div class="fpage">87</div>–<div class="lpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AtMYB14%20regulates%20cold%20tolerance%20in%20Arabidopsis%2C&amp;author=Y.%20Chen&amp;author=Z.%20Chen&amp;author=J.%20Kang&amp;author=D.%20Kang&amp;author=H.%20Gu&amp;author=G.%20Qin&amp;publication_year=2013&amp;journal=Plant%20Mol.%20Biol.%20Rep&amp;volume=31&amp;pages=87-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11105-012-0481-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11105-012-0481-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11105-012-0481-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24415840" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B76" class="ref-content "><div class="label -label">76</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pourcel</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Irani</div> <div class="given-names">N.G.</div></div>, <div class="name string-name"><div class="surname">Koo</div> <div class="given-names">A.J.</div></div>, <div class="name string-name"><div class="surname">Bohorquez‐Restrepo</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Howe</div> <div class="given-names">G.A.</div></div>, <div class="name string-name"><div class="surname">Grotewold</div> <div class="given-names">E.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">A chemical complementation approach reveals genes and interactions of flavonoids with other pathways,</div> <div class="source">Plant J</div> ., <div class="volume">74</div>, <div class="fpage">383</div>–<div class="lpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20chemical%20complementation%20approach%20reveals%20genes%20and%20interactions%20of%20flavonoids%20with%20other%20pathways%2C&amp;author=L.%20Pourcel&amp;author=N.G.%20Irani&amp;author=A.J.%20Koo&amp;author=A.%20Bohorquez%E2%80%90Restrepo&amp;author=G.A.%20Howe&amp;author=E.%20Grotewold&amp;publication_year=2013&amp;journal=Plant%20J&amp;volume=74&amp;pages=383-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2013.74.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2013.74.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2013.74.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23360095" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B77" class="ref-content "><div class="label -label">77</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gibbs</div> <div class="given-names">D.J.</div></div>, <div class="name string-name"><div class="surname">Voß</div> <div class="given-names">U.</div></div>, <div class="name string-name"><div class="surname">Harding</div> <div class="given-names">S.A.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">AtMYB93 is a novel negative regulator of lateral root development in Arabidopsis,</div> <div class="source">New Phytol</div> ., <div class="volume">203</div>, <div class="fpage">1194</div>–<div class="lpage">207</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AtMYB93%20is%20a%20novel%20negative%20regulator%20of%20lateral%20root%20development%20in%20Arabidopsis%2C&amp;author=D.J.%20Gibbs&amp;author=U.%20Vo%C3%9F&amp;author=S.A.%20Harding&amp;publication_year=2014&amp;journal=New%20Phytol&amp;volume=203&amp;pages=1194-207" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.12879" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.12879" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.12879"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24902892" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B78" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B78" class="ref-content "><div class="label -label">78</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xie</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Zheng</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Deng</div> <div class="given-names">L.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">The role of R2R3MYB transcription factors in plant stress tolerance</div>, <div class="source">J. Anim. Plant Sci</div> ., <div class="volume">24</div>, <div class="fpage">1821</div>–<div class="lpage">33</div>.</div></div></div></div><div content-id="dsw054-B79" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B79" class="ref-content "><div class="label -label">79</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Roberts</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Feng</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Pachter</div> <div class="given-names">L.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Fragment assignment in the cloud with eXpress-D,</div> <div class="source">BMC Bioinformatics</div> , <div class="volume">14</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fragment%20assignment%20in%20the%20cloud%20with%20eXpress-D%2C&amp;author=A.%20Roberts&amp;author=H.%20Feng&amp;author=L.%20Pachter&amp;publication_year=2013&amp;journal=BMC%20Bioinformatics&amp;volume=14&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-14-358" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-14-358" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-14-358"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23323762" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B80" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B80" class="ref-content "><div class="label -label">80</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Warnes</div> <div class="given-names">G.R.</div></div>, <div class="name string-name"><div class="surname">Bolker</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Bonebakker</div> <div class="given-names">L.</div></div></div>, et al.  . <div class="year">2009</div>, 
<div class="article-title">gplots: various R programming tools for plotting data</div>. <div class="source">R package version</div> , <div class="fpage">2</div>.</div></div></div></div><div content-id="dsw054-B81" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B81" class="ref-content "><div class="label -label">81</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kent</div> <div class="given-names">W.J.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">BLAT—the BLAST-like alignment tool,</div> <div class="source">Genome Res</div> ., <div class="volume">12</div>, <div class="fpage">656</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BLAT%E2%80%94the%20BLAST-like%20alignment%20tool%2C&amp;author=W.J.%20Kent&amp;publication_year=2002&amp;journal=Genome%20Res&amp;volume=12&amp;pages=656-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.229202" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.229202" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.229202"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11932250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B82" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B82" class="ref-content "><div class="label -label">82</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>, <div class="name string-name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>, <div class="name string-name"><div class="surname">Schäffer</div> <div class="given-names">A.A.</div></div></div>, et al.  . <div class="year">1997</div>, 
<div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">25</div>, <div class="fpage">3389</div>–<div class="lpage">402</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs%2C&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Sch%C3%A4ffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B83" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B83" class="ref-content "><div class="label -label">83</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Langmead</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Fast gapped-read alignment with Bowtie 2,</div> <div class="source">Nat. Methods</div> , <div class="volume">9</div>, <div class="fpage">357</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20Bowtie%202%2C&amp;author=B.%20Langmead&amp;author=S.L.%20Salzberg&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=357-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B84" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B84" class="ref-content "><div class="label -label">84</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bushnell</div> <div class="given-names">B.</div></div></div> <div class="year">2016</div>, BBMap short read aligner. <a class="link link-uri" href="http://sourceforge.net/projects/bbmap" target="_blank">http://sourceforge.net/projects/bbmap</a> (17 June 2016, date last accessed).</div></div></div></div><div content-id="dsw054-B85" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B85" class="ref-content "><div class="label -label">85</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bankevich</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Nurk</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Antipov</div> <div class="given-names">D.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing,</div> <div class="source">J. Comp. Biol</div> ., <div class="volume">19</div>, <div class="fpage">455</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SPAdes%3A%20a%20new%20genome%20assembly%20algorithm%20and%20its%20applications%20to%20single-cell%20sequencing%2C&amp;author=A.%20Bankevich&amp;author=S.%20Nurk&amp;author=D.%20Antipov&amp;publication_year=2012&amp;journal=J.%20Comp.%20Biol&amp;volume=19&amp;pages=455-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/cmb.2012.0021" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fcmb.2012.0021" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2Fcmb.2012.0021"> </span></div> </div></div></div></div></div><div content-id="dsw054-B86" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B86" class="ref-content "><div class="label -label">86</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Shi</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">Y.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">CpGAVAS, an integrated web server for the annotation, visualization, analysis, and GenBank submission of completely sequenced chloroplast genome sequences,</div> <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">715</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CpGAVAS%2C%20an%20integrated%20web%20server%20for%20the%20annotation%2C%20visualization%2C%20analysis%2C%20and%20GenBank%20submission%20of%20completely%20sequenced%20chloroplast%20genome%20sequences%2C&amp;author=C.%20Liu&amp;author=L.%20Shi&amp;author=Y.%20Zhu&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=715" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-715" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-715" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-715"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23256920" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B87" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B87" class="ref-content "><div class="label -label">87</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nowak</div> <div class="given-names">M.D.</div></div>, <div class="name string-name"><div class="surname">Russo</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Schlapbach</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Huu</div> <div class="given-names">C.N.</div></div>, <div class="name string-name"><div class="surname">Lenhard</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Conti</div> <div class="given-names">E.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">The draft genome of <em>Primula veris</em> yields insights into the molecular basis of heterostyly,</div> <div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20draft%20genome%20of%20Primula%20veris%20yields%20insights%20into%20the%20molecular%20basis%20of%20heterostyly%2C&amp;author=M.D.%20Nowak&amp;author=G.%20Russo&amp;author=R.%20Schlapbach&amp;author=C.N.%20Huu&amp;author=M.%20Lenhard&amp;author=E.%20Conti&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-014-0567-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-014-0567-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-014-0567-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25583448" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B88" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B88" class="ref-content "><div class="label -label">88</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Clouse</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Adhikary</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Page</div> <div class="given-names">J.</div></div></div>, et al.  . <div class="year">2016</div>, 
<div class="article-title">The amaranth genome: genome, transcriptome, and physical map assembly,</div> <div class="source">Plant Genome</div> , <div class="volume">9</div>.</div></div></div></div><div content-id="dsw054-B89" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B89" class="ref-content "><div class="label -label">89</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Claros</div> <div class="given-names">M.G.</div></div>, <div class="name string-name"><div class="surname">Bautista</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Guerrero-Fernández</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Benzerki</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Seoane</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Fernández-Pozo</div> <div class="given-names">N.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Why assembling plant genome sequences is so challenging,</div> <div class="source">Biology</div> , <div class="volume">1</div>, <div class="fpage">439</div>–<div class="lpage">59</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Why%20assembling%20plant%20genome%20sequences%20is%20so%20challenging%2C&amp;author=M.G.%20Claros&amp;author=R.%20Bautista&amp;author=D.%20Guerrero-Fern%C3%A1ndez&amp;author=H.%20Benzerki&amp;author=P.%20Seoane&amp;author=N.%20Fern%C3%A1ndez-Pozo&amp;publication_year=2012&amp;journal=Biology&amp;volume=1&amp;pages=439-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3390/biology1020439" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3390%2Fbiology1020439" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3390%2Fbiology1020439"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24832233" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B90" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B90" class="ref-content "><div class="label -label">90</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Michael</div> <div class="given-names">T.P.</div></div>, <div class="name string-name"><div class="surname">Jackson</div> <div class="given-names">S.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">The first 50 plant genomes,</div> <div class="source">Plant Genome</div> , <div class="volume">6</div>.</div></div></div></div><div content-id="dsw054-B91" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B91" class="ref-content "><div class="label -label">91</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Van Bakel</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Stout</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Cote</div> <div class="given-names">A.G.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">The draft genome and transcriptome of <em>Cannabis sativa</em>,</div> <div class="source">Genome Biol</div> ., <div class="volume">12</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20draft%20genome%20and%20transcriptome%20of%20Cannabis%20sativa%2C&amp;author=H.%20Van%20Bakel&amp;author=J.M.%20Stout&amp;author=A.G.%20Cote&amp;publication_year=2011&amp;journal=Genome%20Biol&amp;volume=12&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2011-12-10-r102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2011-12-10-r102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2011-12-10-r102"> </span></div> </div></div></div></div></div><div content-id="dsw054-B92" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B92" class="ref-content "><div class="label -label">92</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Peng</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Lai</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Lane</div> <div class="given-names">T.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">De novo genome assembly of the economically important weed horseweed using integrated data from multiple sequencing platforms,</div> <div class="source">Plant Physiol</div> ., <div class="volume">166</div>, <div class="fpage">1241</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20genome%20assembly%20of%20the%20economically%20important%20weed%20horseweed%20using%20integrated%20data%20from%20multiple%20sequencing%20platforms%2C&amp;author=Y.%20Peng&amp;author=Z.%20Lai&amp;author=T.%20Lane&amp;publication_year=2014&amp;journal=Plant%20Physiol&amp;volume=166&amp;pages=1241-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.114.247668" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.114.247668" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.114.247668"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25209985" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B93" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B93" class="ref-content "><div class="label -label">93</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Van Hoeck</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Horemans</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Monsieurs</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Cao</div> <div class="given-names">H.X.</div></div>, <div class="name string-name"><div class="surname">Vandenhove</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Blust</div> <div class="given-names">R.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">The first draft genome of the aquatic model plant <em>Lemna minor</em> opens the route for future stress physiology research and biotechnological applications,</div> <div class="source">Biotechnol. Biofuels</div> , <div class="volume">8</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20first%20draft%20genome%20of%20the%20aquatic%20model%20plant%20Lemna%20minor%20opens%20the%20route%20for%20future%20stress%20physiology%20research%20and%20biotechnological%20applications%2C&amp;author=A.%20Van%20Hoeck&amp;author=N.%20Horemans&amp;author=P.%20Monsieurs&amp;author=H.X.%20Cao&amp;author=H.%20Vandenhove&amp;author=R.%20Blust&amp;publication_year=2015&amp;journal=Biotechnol.%20Biofuels&amp;volume=8&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13068-015-0381-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13068-015-0381-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13068-015-0381-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25642283" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B94" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B94" class="ref-content "><div class="label -label">94</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rahman</div> <div class="given-names">A.Y.A.</div></div>, <div class="name string-name"><div class="surname">Usharraj</div> <div class="given-names">A.O.</div></div>, <div class="name string-name"><div class="surname">Misra</div> <div class="given-names">B.B.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">Draft genome sequence of the rubber tree <em>Hevea brasiliensis</em>,</div> <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20the%20rubber%20tree%20Hevea%20brasiliensis%2C&amp;author=A.Y.A.%20Rahman&amp;author=A.O.%20Usharraj&amp;author=B.B.%20Misra&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-75" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-75" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-75"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23375136" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B95" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B95" class="ref-content "><div class="label -label">95</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jia</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kong</div> <div class="given-names">X.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title"><em>Aegilops tauschii</em> draft genome sequence reveals a gene repertoire for wheat adaptation,</div> <div class="source">Nature</div> , <div class="volume">496</div>, <div class="fpage">91</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aegilops%20tauschii%20draft%20genome%20sequence%20reveals%20a%20gene%20repertoire%20for%20wheat%20adaptation%2C&amp;author=J.%20Jia&amp;author=S.%20Zhao&amp;author=X.%20Kong&amp;publication_year=2013&amp;journal=Nature&amp;volume=496&amp;pages=91-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature12028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature12028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature12028"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23535592" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B96" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B96" class="ref-content "><div class="label -label">96</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ling</div> <div class="given-names">H.-Q.</div></div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">D.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">Draft genome of the wheat A-genome progenitor <em>Triticum urartu</em>,</div> <div class="source">Nature</div> , <div class="volume">496</div>, <div class="fpage">87</div>-<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20of%20the%20wheat%20A-genome%20progenitor%20Triticum%20urartu%2C&amp;author=H.-Q.%20Ling&amp;author=S.%20Zhao&amp;author=D.%20Liu&amp;publication_year=2013&amp;journal=Nature&amp;volume=496&amp;pages=87-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11997" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11997" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11997"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23535596" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B97" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B97" class="ref-content "><div class="label -label">97</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Otto</div> <div class="given-names">S.P.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">The evolutionary consequences of polyploidy,</div> <div class="source">Cell</div> , <div class="volume">131</div>, <div class="fpage">452</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolutionary%20consequences%20of%20polyploidy%2C&amp;author=S.P.%20Otto&amp;publication_year=2007&amp;journal=Cell&amp;volume=131&amp;pages=452-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2007.10.022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2007.10.022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2007.10.022"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17981114" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B98" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B98" class="ref-content "><div class="label -label">98</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kim</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Pertea</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pimentel</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kelley</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions,</div> <div class="source">Genome Biol</div> ., <div class="volume">14</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TopHat2%3A%20accurate%20alignment%20of%20transcriptomes%20in%20the%20presence%20of%20insertions%2C%20deletions%20and%20gene%20fusions%2C&amp;author=D.%20Kim&amp;author=G.%20Pertea&amp;author=C.%20Trapnell&amp;author=H.%20Pimentel&amp;author=R.%20Kelley&amp;author=S.L.%20Salzberg&amp;publication_year=2013&amp;journal=Genome%20Biol&amp;volume=14&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2013-14-1-r1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-1-r1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2013-14-1-r1"> </span></div> </div></div></div></div></div><div content-id="dsw054-B99" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B99" class="ref-content "><div class="label -label">99</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Conesa</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Madrigal</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Tarazona</div> <div class="given-names">S.</div></div></div>, et al.  . <div class="year">2016</div>, 
<div class="article-title">A survey of best practices for RNA-seq data analysis,</div> <div class="source">Genome Biol</div> ., <div class="volume">17</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20survey%20of%20best%20practices%20for%20RNA-seq%20data%20analysis%2C&amp;author=A.%20Conesa&amp;author=P.%20Madrigal&amp;author=S.%20Tarazona&amp;publication_year=2016&amp;journal=Genome%20Biol&amp;volume=17&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-015-0866-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-015-0866-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-015-0866-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26753840" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B100" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B100" class="ref-content "><div class="label -label">100</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Glass</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Quackenbush</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Silverman</div> <div class="given-names">E.K.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Sexually-dimorphic targeting of functionally-related genes in COPD,</div> <div class="source">BMC Syst. Biol</div> ., <div class="volume">8</div>, <div class="fpage">118</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sexually-dimorphic%20targeting%20of%20functionally-related%20genes%20in%20COPD%2C&amp;author=K.%20Glass&amp;author=J.%20Quackenbush&amp;author=E.K.%20Silverman&amp;publication_year=2014&amp;journal=BMC%20Syst.%20Biol&amp;volume=8&amp;pages=118" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12918-014-0118-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12918-014-0118-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12918-014-0118-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25431000" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B101" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B101" class="ref-content "><div class="label -label">101</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cai</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Mao</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Wei</div> <div class="given-names">L.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Genome comparison using Gene Ontology (GO) with statistical testing</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">7</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20comparison%20using%20Gene%20Ontology%20(GO)%20with%20statistical%20testing&amp;author=Z.%20Cai&amp;author=X.%20Mao&amp;author=S.%20Li&amp;author=L.%20Wei&amp;publication_year=2006&amp;journal=BMC%20Bioinformatics&amp;volume=7&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-7-374" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-7-374" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-7-374"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16393334" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B102" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B102" class="ref-content "><div class="label -label">102</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kubis</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Schmidt</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Heslop-Harrison</div> <div class="given-names">J.S.P.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Repetitive DNA elements as a major component of plant genomes,</div> <div class="source">Ann. Bot</div> ., <div class="volume">82</div>, <div class="fpage">45</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repetitive%20DNA%20elements%20as%20a%20major%20component%20of%20plant%20genomes%2C&amp;author=S.%20Kubis&amp;author=T.%20Schmidt&amp;author=J.S.P.%20Heslop-Harrison&amp;publication_year=1998&amp;journal=Ann.%20Bot&amp;volume=82&amp;pages=45-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/anbo.1998.0779" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fanbo.1998.0779" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fanbo.1998.0779"> </span></div> </div></div></div></div></div><div content-id="dsw054-B103" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B103" class="ref-content "><div class="label -label">103</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Brenchley</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Spannagl</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pfeifer</div> <div class="given-names">M.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Analysis of the bread wheat genome using whole-genome shotgun sequencing,</div> <div class="source">Nature</div> , <div class="volume">491</div>, <div class="fpage">705</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20bread%20wheat%20genome%20using%20whole-genome%20shotgun%20sequencing%2C&amp;author=R.%20Brenchley&amp;author=M.%20Spannagl&amp;author=M.%20Pfeifer&amp;publication_year=2012&amp;journal=Nature&amp;volume=491&amp;pages=705-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11650" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11650" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11650"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23192148" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B104" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B104" class="ref-content "><div class="label -label">104</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mehrotra</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Goyal</div> <div class="given-names">V.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Repetitive sequences in plant nuclear DNA: types, distribution, evolution and function,</div> <div class="source">Genomics Proteomics Bioinformatics</div> , <div class="volume">12</div>, <div class="fpage">164</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repetitive%20sequences%20in%20plant%20nuclear%20DNA%3A%20types%2C%20distribution%2C%20evolution%20and%20function%2C&amp;author=S.%20Mehrotra&amp;author=V.%20Goyal&amp;publication_year=2014&amp;journal=Genomics%20Proteomics%20Bioinformatics&amp;volume=12&amp;pages=164-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gpb.2014.07.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gpb.2014.07.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gpb.2014.07.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25132181" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B105" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B105" class="ref-content "><div class="label -label">105</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hoshino</div> <div class="given-names">A.A.</div></div>, <div class="name string-name"><div class="surname">Bravo</div> <div class="given-names">J.P.</div></div>, <div class="name string-name"><div class="surname">Morelli</div> <div class="given-names">K.A.</div></div>, <div class="name string-name"><div class="surname">Nobile</div> <div class="given-names">P.M.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Microsatellites as tools for genetic diversity analysis</div>, <div class="source">Genet Divers Microorganisms</div> ., <div class="volume">6</div>, <div class="fpage">149</div>–<div class="lpage">170</div>.</div></div></div></div><div content-id="dsw054-B106" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B106" class="ref-content "><div class="label -label">106</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Miah</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Rafii</div> <div class="given-names">M.Y.</div></div>, <div class="name string-name"><div class="surname">Ismail</div> <div class="given-names">M.R.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">A review of microsatellite markers and their applications in rice breeding programs to improve blast disease resistance,</div> <div class="source">Int. J. Mol. Sci</div> ., <div class="volume">14</div>, <div class="fpage">22499</div>–<div class="lpage">528</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20review%20of%20microsatellite%20markers%20and%20their%20applications%20in%20rice%20breeding%20programs%20to%20improve%20blast%20disease%20resistance%2C&amp;author=G.%20Miah&amp;author=M.Y.%20Rafii&amp;author=M.R.%20Ismail&amp;publication_year=2013&amp;journal=Int.%20J.%20Mol.%20Sci&amp;volume=14&amp;pages=22499-528" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3390/ijms141122499" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3390%2Fijms141122499" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3390%2Fijms141122499"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24240810" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B107" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B107" class="ref-content "><div class="label -label">107</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rastogi</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kalra</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Gupta</div> <div class="given-names">V.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Unravelling the genome of Holy basil: an “incomparable” “elixir of life” of traditional Indian medicine,</div> <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unravelling%20the%20genome%20of%20Holy%20basil%3A%20an%20%E2%80%9Cincomparable%E2%80%9D%20%E2%80%9Celixir%20of%20life%E2%80%9D%20of%20traditional%20Indian%20medicine%2C&amp;author=S.%20Rastogi&amp;author=A.%20Kalra&amp;author=V.%20Gupta&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1640-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1640-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1640-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25553907" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B108" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B108" class="ref-content "><div class="label -label">108</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Raven</div> <div class="given-names">P.H.</div></div></div> <div class="year">1972</div>, 
<div class="article-title">Why are bird-visited flowers predominantly red?,</div> <div class="source">Evolution</div> , <div class="volume">26</div>, <div class="fpage">674</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Why%20are%20bird-visited%20flowers%20predominantly%20red%3F%2C&amp;author=P.H.%20Raven&amp;publication_year=1972&amp;journal=Evolution&amp;volume=26&amp;pages=674" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2407064" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2407064" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2407064"> </span></div> </div></div></div></div></div><div content-id="dsw054-B109" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B109" class="ref-content "><div class="label -label">109</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Melendez-Ackerman</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Campbell</div> <div class="given-names">D.R.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Adaptive significance of flower color and inter-trait correlations in an Ipomopsis hybrid zone,</div> <div class="source">Evolution</div> , <div class="fpage">1293</div>–<div class="lpage">303</div>.</div></div></div></div><div content-id="dsw054-B110" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B110" class="ref-content "><div class="label -label">110</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fulton</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hodges</div> <div class="given-names">S.A.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Floral isolation between <em>Aquilegia formosa</em> and <em>Aquilegia pubescens</em>,</div> <div class="source">Proc. R. Soc. Lond</div> ., <div class="volume">266</div>, <div class="fpage">2247</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Floral%20isolation%20between%20Aquilegia%20formosa%20and%20Aquilegia%20pubescens%2C&amp;author=M.%20Fulton&amp;author=S.A.%20Hodges&amp;publication_year=1999&amp;journal=Proc.%20R.%20Soc.%20Lond&amp;volume=266&amp;pages=2247-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rspb.1999.0915" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frspb.1999.0915" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frspb.1999.0915"> </span></div> </div></div></div></div></div><div content-id="dsw054-B111" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B111" class="ref-content "><div class="label -label">111</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bradshaw</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Schemske</div> <div class="given-names">D.W.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Allele substitution at a flower colour locus produces a pollinator shift in monkeyflowers,</div> <div class="source">Nature</div> , <div class="volume">426</div>, <div class="fpage">176</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Allele%20substitution%20at%20a%20flower%20colour%20locus%20produces%20a%20pollinator%20shift%20in%20monkeyflowers%2C&amp;author=H.%20Bradshaw&amp;author=D.W.%20Schemske&amp;publication_year=2003&amp;journal=Nature&amp;volume=426&amp;pages=176-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature02106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature02106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature02106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14614505" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B112" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B112" class="ref-content "><div class="label -label">112</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Irwin</div> <div class="given-names">R.E.</div></div>, <div class="name string-name"><div class="surname">Strauss</div> <div class="given-names">S.Y.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Flower color microevolution in wild radish: evolutionary response to pollinator‐mediated selection,</div> <div class="source">Am. Nat</div> ., <div class="volume">165</div>, <div class="fpage">225</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Flower%20color%20microevolution%20in%20wild%20radish%3A%20evolutionary%20response%20to%20pollinator%E2%80%90mediated%20selection%2C&amp;author=R.E.%20Irwin&amp;author=S.Y.%20Strauss&amp;publication_year=2005&amp;journal=Am.%20Nat&amp;volume=165&amp;pages=225-37" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15729652" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B113" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B113" class="ref-content "><div class="label -label">113</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">de Pascual-Teresa</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Moreno</div> <div class="given-names">D.A.</div></div>, <div class="name string-name"><div class="surname">García-Viguera</div> <div class="given-names">C.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Flavanols and anthocyanins in cardiovascular health: a review of current evidence,</div> <div class="source">Int. J. Mol. Sci</div> ., <div class="volume">11</div>, <div class="fpage">1679</div>–<div class="lpage">703</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Flavanols%20and%20anthocyanins%20in%20cardiovascular%20health%3A%20a%20review%20of%20current%20evidence%2C&amp;author=S.%20de%20Pascual-Teresa&amp;author=D.A.%20Moreno&amp;author=C.%20Garc%C3%ADa-Viguera&amp;publication_year=2010&amp;journal=Int.%20J.%20Mol.%20Sci&amp;volume=11&amp;pages=1679-703" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3390/ijms11041679" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3390%2Fijms11041679" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3390%2Fijms11041679"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20480037" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B114" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B114" class="ref-content "><div class="label -label">114</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Falginella</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Castellarin</div> <div class="given-names">S.D.</div></div>, <div class="name string-name"><div class="surname">Testolin</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Gambetta</div> <div class="given-names">G.A.</div></div>, <div class="name string-name"><div class="surname">Morgante</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Di Gaspero</div> <div class="given-names">G.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Expansion and subfunctionalisation of flavonoid 3′, 5′-hydroxylases in the grapevine lineage,</div> <div class="source">BMC Genomics</div> , <div class="volume">11</div>, <div class="fpage">562</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expansion%20and%20subfunctionalisation%20of%20flavonoid%203%E2%80%B2%2C%205%E2%80%B2-hydroxylases%20in%20the%20grapevine%20lineage%2C&amp;author=L.%20Falginella&amp;author=S.D.%20Castellarin&amp;author=R.%20Testolin&amp;author=G.A.%20Gambetta&amp;author=M.%20Morgante&amp;author=G.%20Di%20Gaspero&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=562" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-562" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-562" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-562"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20939908" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B115" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B115" class="ref-content "><div class="label -label">115</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wessinger</div> <div class="given-names">C.A.</div></div>, <div class="name string-name"><div class="surname">Rausher</div> <div class="given-names">M.D.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Ecological transition predictably associated with gene degeneration,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">32</div>, <div class="fpage">347</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ecological%20transition%20predictably%20associated%20with%20gene%20degeneration%2C&amp;author=C.A.%20Wessinger&amp;author=M.D.%20Rausher&amp;publication_year=2015&amp;journal=Mol.%20Biol.%20Evol&amp;volume=32&amp;pages=347-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msu298" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsu298" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsu298"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25371436" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B116" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B116" class="ref-content "><div class="label -label">116</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Campanella</div> <div class="given-names">J.J.</div></div>, <div class="name string-name"><div class="surname">Smalley</div> <div class="given-names">J.V.</div></div>, <div class="name string-name"><div class="surname">Dempsey</div> <div class="given-names">M.E.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">A phylogenetic examination of the primary anthocyanin production pathway of the Plantae,</div> <div class="source">Bot. Stud</div> ., <div class="volume">55</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20phylogenetic%20examination%20of%20the%20primary%20anthocyanin%20production%20pathway%20of%20the%20Plantae%2C&amp;author=J.J.%20Campanella&amp;author=J.V.%20Smalley&amp;author=M.E.%20Dempsey&amp;publication_year=2014&amp;journal=Bot.%20Stud&amp;volume=55&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1999-3110-55-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1999-3110-55-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1999-3110-55-10"> </span></div> </div></div></div></div></div><div content-id="dsw054-B117" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B117" class="ref-content "><div class="label -label">117</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cone</div> <div class="given-names">K.C.</div></div>, <div class="name string-name"><div class="surname">Cocciolone</div> <div class="given-names">S.M.</div></div>, <div class="name string-name"><div class="surname">Burr</div> <div class="given-names">F.A.</div></div>, <div class="name string-name"><div class="surname">Burr</div> <div class="given-names">B.</div></div></div> <div class="year">1993</div>, 
<div class="article-title">Maize anthocyanin regulatory gene <em>pl</em> is a duplicate of <em>c1</em> that functions in the plant,</div> <div class="source">Plant Cell</div> , <div class="volume">5</div>, <div class="fpage">1795</div>–<div class="lpage">805</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Maize%20anthocyanin%20regulatory%20gene%20pl%20is%20a%20duplicate%20of%20c1%20that%20functions%20in%20the%20plant%2C&amp;author=K.C.%20Cone&amp;author=S.M.%20Cocciolone&amp;author=F.A.%20Burr&amp;author=B.%20Burr&amp;publication_year=1993&amp;journal=Plant%20Cell&amp;volume=5&amp;pages=1795-805" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.5.12.1795" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.5.12.1795" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.5.12.1795"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8305872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B118" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B118" class="ref-content "><div class="label -label">118</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Seitz</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Ameres</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Schlangen</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Forkmann</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Halbwirth</div> <div class="given-names">H.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Multiple evolution of flavonoid 3′, 5′-hydroxylase,</div> <div class="source">Planta</div> , <div class="volume">242</div>, <div class="fpage">561</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20evolution%20of%20flavonoid%203%E2%80%B2%2C%205%E2%80%B2-hydroxylase%2C&amp;author=C.%20Seitz&amp;author=S.%20Ameres&amp;author=K.%20Schlangen&amp;author=G.%20Forkmann&amp;author=H.%20Halbwirth&amp;publication_year=2015&amp;journal=Planta&amp;volume=242&amp;pages=561-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00425-015-2293-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00425-015-2293-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00425-015-2293-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25916309" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B119" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B119" class="ref-content "><div class="label -label">119</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Winkel</div> <div class="given-names">B.S.</div></div></div> <div class="year">2006</div>, The biosynthesis of flavonoids. In: <div class="source">The Science of Flavonoids</div> . 
<div class="publisher-name">Springer, New York</div>, pp. <div class="fpage">71</div>–<div class="lpage">95</div>.</div></div></div></div><div content-id="dsw054-B120" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B120" class="ref-content "><div class="label -label">120</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Johnson</div> <div class="given-names">E.T.</div></div>, <div class="name string-name"><div class="surname">Ryu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Yi</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Shin</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Cheong</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Choi</div> <div class="given-names">G.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Alteration of a single amino acid changes the substrate specificity of dihydroflavonol 4‐reductase,</div> <div class="source">Plant J</div> ., <div class="volume">25</div>, <div class="fpage">325</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alteration%20of%20a%20single%20amino%20acid%20changes%20the%20substrate%20specificity%20of%20dihydroflavonol%204%E2%80%90reductase%2C&amp;author=E.T.%20Johnson&amp;author=S.%20Ryu&amp;author=H.%20Yi&amp;author=B.%20Shin&amp;author=H.%20Cheong&amp;author=G.%20Choi&amp;publication_year=2001&amp;journal=Plant%20J&amp;volume=25&amp;pages=325-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313x.2001.00962.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313x.2001.00962.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313x.2001.00962.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11208024" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B121" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B121" class="ref-content "><div class="label -label">121</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Smith</div> <div class="given-names">S.D.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Rausher</div> <div class="given-names">M.D.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Functional evolution of an anthocyanin pathway enzyme during a flower color transition,</div> <div class="source">Mol. Biol. Evol</div> ., <div class="volume">30</div>, <div class="fpage">602</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20evolution%20of%20an%20anthocyanin%20pathway%20enzyme%20during%20a%20flower%20color%20transition%2C&amp;author=S.D.%20Smith&amp;author=S.%20Wang&amp;author=M.D.%20Rausher&amp;publication_year=2013&amp;journal=Mol.%20Biol.%20Evol&amp;volume=30&amp;pages=602-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mss255" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmss255" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmss255"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23155005" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B122" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B122" class="ref-content "><div class="label -label">122</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Miosic</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Thill</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Milosevic</div> <div class="given-names">M.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Dihydroflavonol 4-reductase genes encode enzymes with contrasting substrate specificity and show divergent gene expression profiles in <em>Fragaria</em> Species,</div> <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e112707</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dihydroflavonol%204-reductase%20genes%20encode%20enzymes%20with%20contrasting%20substrate%20specificity%20and%20show%20divergent%20gene%20expression%20profiles%20in%20Fragaria%20Species%2C&amp;author=S.%20Miosic&amp;author=J.%20Thill&amp;author=M.%20Milosevic&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e112707" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0112707" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0112707" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0112707"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25393679" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B123" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B123" class="ref-content "><div class="label -label">123</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Quattrocchio</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Wing</div> <div class="given-names">J.F.</div></div>, <div class="name string-name"><div class="surname">Va</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Mol</div> <div class="given-names">J.N.</div></div>, <div class="name string-name"><div class="surname">Koes</div> <div class="given-names">R.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Analysis of bHLH and MYB domain proteins: species‐specific regulatory differences are caused by divergent evolution of target anthocyanin genes</div>, <div class="source">Plant J</div> ., <div class="volume">13</div>, <div class="fpage">475</div>–<div class="lpage">88</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20bHLH%20and%20MYB%20domain%20proteins%3A%20species%E2%80%90specific%20regulatory%20differences%20are%20caused%20by%20divergent%20evolution%20of%20target%20anthocyanin%20genes&amp;author=F.%20Quattrocchio&amp;author=J.F.%20Wing&amp;author=K.%20Va&amp;author=J.N.%20Mol&amp;author=R.%20Koes&amp;publication_year=1998&amp;journal=Plant%20J&amp;volume=13&amp;pages=475-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.1998.00046.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.1998.00046.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.1998.00046.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9680994" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B124" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B124" class="ref-content "><div class="label -label">124</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Quattrocchio</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Wing</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">van der Woude</div> <div class="given-names">K.</div></div></div>, et al.  . <div class="year">1999</div>, 
<div class="article-title">Molecular analysis of the anthocyanin2 gene of Petunia and its role in the evolution of flower color,</div> <div class="source">Plant Cell</div> , <div class="volume">11</div>, <div class="fpage">1433</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20analysis%20of%20the%20anthocyanin2%20gene%20of%20Petunia%20and%20its%20role%20in%20the%20evolution%20of%20flower%20color%2C&amp;author=F.%20Quattrocchio&amp;author=J.%20Wing&amp;author=K.%20van%20der%20Woude&amp;publication_year=1999&amp;journal=Plant%20Cell&amp;volume=11&amp;pages=1433-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.11.8.1433" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.11.8.1433" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.11.8.1433"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10449578" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B125" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B125" class="ref-content "><div class="label -label">125</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Albert</div> <div class="given-names">N.W.</div></div>, <div class="name string-name"><div class="surname">Lewis</div> <div class="given-names">D.H.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Schwinn</div> <div class="given-names">K.E.</div></div>, <div class="name string-name"><div class="surname">Jameson</div> <div class="given-names">P.E.</div></div>, <div class="name string-name"><div class="surname">Davies</div> <div class="given-names">K.M.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Members of an R2R3‐MYB transcription factor family in Petunia are developmentally and environmentally regulated to control complex floral and vegetative pigmentation patterning,</div> <div class="source">Plant J</div> ., <div class="volume">65</div>, <div class="fpage">771</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Members%20of%20an%20R2R3%E2%80%90MYB%20transcription%20factor%20family%20in%20Petunia%20are%20developmentally%20and%20environmentally%20regulated%20to%20control%20complex%20floral%20and%20vegetative%20pigmentation%20patterning%2C&amp;author=N.W.%20Albert&amp;author=D.H.%20Lewis&amp;author=H.%20Zhang&amp;author=K.E.%20Schwinn&amp;author=P.E.%20Jameson&amp;author=K.M.%20Davies&amp;publication_year=2011&amp;journal=Plant%20J&amp;volume=65&amp;pages=771-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2011.65.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2011.65.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2011.65.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21235651" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B126" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B126" class="ref-content "><div class="label -label">126</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Yan</div> <div class="given-names">T.L.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Q.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Complete chloroplast genome sequences of <em>Praxelis</em> (<em>Eupatorium catarium Veldkamp</em>), an important invasive species,</div> <div class="source">Gene</div> , <div class="volume">549</div>, <div class="fpage">58</div>-<div class="lpage">69</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20chloroplast%20genome%20sequences%20of%20Praxelis%20(Eupatorium%20catarium%20Veldkamp)%2C%20an%20important%20invasive%20species%2C&amp;author=Y.%20Zhang&amp;author=L.%20Li&amp;author=T.L.%20Yan&amp;author=Q.%20Liu&amp;publication_year=2014&amp;journal=Gene&amp;volume=549&amp;pages=58-69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2014.07.041" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2014.07.041" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2014.07.041"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25042453" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B127" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B127" class="ref-content "><div class="label -label">127</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Clegg</div> <div class="given-names">M.T.</div></div>, <div class="name string-name"><div class="surname">Gaut</div> <div class="given-names">B.S.</div></div>, <div class="name string-name"><div class="surname">Learn</div> <div class="given-names">G.H.</div></div>, <div class="name string-name"><div class="surname">Morton</div> <div class="given-names">B.R.</div></div></div> <div class="year">1994</div>, 
<div class="article-title">Rates and patterns of chloroplast DNA evolution,</div> <div class="source">Proc. Natl. Acad. Sci</div> ., <div class="volume">91</div>, <div class="fpage">6795</div>–<div class="lpage">801</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rates%20and%20patterns%20of%20chloroplast%20DNA%20evolution%2C&amp;author=M.T.%20Clegg&amp;author=B.S.%20Gaut&amp;author=G.H.%20Learn&amp;author=B.R.%20Morton&amp;publication_year=1994&amp;journal=Proc.%20Natl.%20Acad.%20Sci&amp;volume=91&amp;pages=6795-801" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.91.15.6795" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.91.15.6795" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.91.15.6795"> </span></div> </div></div></div></div></div><div content-id="dsw054-B128" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B128" class="ref-content "><div class="label -label">128</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Martin</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Baurens</div> <div class="given-names">F.-C.</div></div>, <div class="name string-name"><div class="surname">Cardi</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Aury</div> <div class="given-names">J.-M.</div></div>, <div class="name string-name"><div class="surname">D’Hont</div> <div class="given-names">A.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">The complete chloroplast genome of banana (<em>Musa acuminata</em>, Zingiberales): insight into plastid monocotyledon evolution,</div> <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e67350</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20chloroplast%20genome%20of%20banana%20(Musa%20acuminata%2C%20Zingiberales)%3A%20insight%20into%20plastid%20monocotyledon%20evolution%2C&amp;author=G.%20Martin&amp;author=F.-C.%20Baurens&amp;author=C.%20Cardi&amp;author=J.-M.%20Aury&amp;author=A.%20D%E2%80%99Hont&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e67350" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0067350" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0067350" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0067350"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23840670" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B129" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B129" class="ref-content "><div class="label -label">129</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hansen</div> <div class="given-names">D.R.</div></div>, <div class="name string-name"><div class="surname">Dastidar</div> <div class="given-names">S.G.</div></div>, <div class="name string-name"><div class="surname">Cai</div> <div class="given-names">Z.</div></div></div>, et al.  . <div class="year">2007</div>, 
<div class="article-title">Phylogenetic and evolutionary implications of complete chloroplast genome sequences of four early-diverging angiosperms: <em>Buxus</em> (Buxaceae), <em>Chloranthus</em> (Chloranthaceae), <em>Dioscorea</em> (Dioscoreaceae), and <em>Illicium</em> (Schisandraceae),</div> <div class="source">Mol. Phylogenet. Evol</div> ., <div class="volume">45</div>, <div class="fpage">547</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20and%20evolutionary%20implications%20of%20complete%20chloroplast%20genome%20sequences%20of%20four%20early-diverging%20angiosperms%3A%20Buxus%20(Buxaceae)%2C%20Chloranthus%20(Chloranthaceae)%2C%20Dioscorea%20(Dioscoreaceae)%2C%20and%20Illicium%20(Schisandraceae)%2C&amp;author=D.R.%20Hansen&amp;author=S.G.%20Dastidar&amp;author=Z.%20Cai&amp;publication_year=2007&amp;journal=Mol.%20Phylogenet.%20Evol&amp;volume=45&amp;pages=547-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ympev.2007.06.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2007.06.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ympev.2007.06.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17644003" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw054-B130" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw054-B130" class="ref-content "><div class="label -label">130</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stebbins</div> <div class="given-names">G.L.</div></div></div> <div class="year">1970</div>, 
<div class="article-title">Adaptive radiation of reproductive characteristics in angiosperms, I: pollination mechanisms</div>, <div class="source">Annu. Rev. Ecol. Evol. Syst</div> ., <div class="volume">1</div>, <div class="fpage">307</div>–<div class="lpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adaptive%20radiation%20of%20reproductive%20characteristics%20in%20angiosperms%2C%20I%3A%20pollination%20mechanisms&amp;author=G.L.%20Stebbins&amp;publication_year=1970&amp;journal=Annu.%20Rev.%20Ecol.%20Evol.%20Syst&amp;volume=1&amp;pages=307-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.es.01.110170.001515" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.es.01.110170.001515" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.es.01.110170.001515"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute. </div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2960054&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw054/2/dsw054_Supp.zip?Expires=2147483647&amp;Signature=OzwUbdrbll8GWrzlrYMohmjg2aIkCFsGZYTFXY~Kt6KuVc2twDadH8zYRPTVlbeirvm1zVBvwJJ62AMcnZZQRT6IkIiVdHs8RgqOLfsAeIdh1ClQTbbNSYHc87Of2VzfRrqiJSBJfZsk8gGsQOEMVqZ87ObAsjdKIt1D-l2pplHdY-9Bvx3S5C84jcZZN7spkpVLyp1Amn0LUMm0Z6563bsp1Ely9Dp~rQnN63w02~k0kobcBfHFjTanMPAC6QAKpc2tzoYt3l5iwjOgcp3pwV1K2ooGe9cXit7wNNM1qehybTZkN05Ngz~w8giL74bxPcg2lPmrHbg5qzF0W5XtaQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">779</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000400913000007&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">5</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw054"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2960054"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000400913000007&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f2%2f179%2f2960054">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000400913000007&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (5)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F2%2F179%2F2960054" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2960054'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="draft genome of Ruellia speciosa (Beautiful Wild Petunia: Acanthaceae)" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw054");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f179%2f2960054">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

