

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":0,"applicationTime":2080,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/18/6/435/477266","@type":"ScholarlyArticle","name":"An Integrative Method for Identifying the Over-Annotated Protein-Coding Genes in Microbial Genomes","datePublished":"2011-09-08","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/18/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2011-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr030","keywords":["protein-coding gene","microbial genome","re-annotation","horizontal gene transfer"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/18/6/435/477266","author":[{"name":"Yu, Jia-Feng","affiliation":"1  State Key Laboratory of Bioelectronics, School of Biological Science and Medical Engineering, Southeast University ,  Nanjing 210096 ,  China    2  Shandong Province Key Laboratory of Biophysics for Functional Macromolecules, Department of Physics ,  Dezhou University ,  Dezhou 253023 ,  China","@type":"Person"},{"name":"Xiao, Ke","affiliation":"1  State Key Laboratory of Bioelectronics, School of Biological Science and Medical Engineering, Southeast University ,  Nanjing 210096 ,  China","@type":"Person"},{"name":"Jiang, Dong-Ke","affiliation":"1  State Key Laboratory of Bioelectronics, School of Biological Science and Medical Engineering, Southeast University ,  Nanjing 210096 ,  China","@type":"Person"},{"name":"Guo, Jing","affiliation":"1  State Key Laboratory of Bioelectronics, School of Biological Science and Medical Engineering, Southeast University ,  Nanjing 210096 ,  China","@type":"Person"},{"name":"Wang, Ji-Hua","affiliation":"2  Shandong Province Key Laboratory of Biophysics for Functional Macromolecules, Department of Physics ,  Dezhou University ,  Dezhou 253023 ,  China","@type":"Person"},{"name":"Sun, Xiao","affiliation":"1  State Key Laboratory of Bioelectronics, School of Biological Science and Medical Engineering, Southeast University ,  Nanjing 210096 ,  China","@type":"Person"}],"description":"Abstract.  The falsely annotated protein-coding genes have been deemed one of the major causes accounting for the annotating errors in public databases. Althoug","pageStart":"435","pageEnd":"449","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/6/1/m_cover.gif?Expires=2147483647&Signature=qL9inMhepM2zFUhwbU8mgSKjtZVxpg~PmNu8fbxg-9GXAGTJaQrhv6qnZRDeMk7qu~VNHreHxR8Oz-VBGUn7klik4EpJ16UF40231HSss3FWSUANYRXx6t042t9kZJxg5tVYoWTO8VtSK1XYiJ1~Hb4XzfVloDJNBsISl1abmgO6rLxcEn6ZsFnsGHwCs2gNKgd6or0KznZwXt8UzZ8lxgCsA3E4dHjQXfOwyJhuIWr89hghDvrxBGqblSwMEBtNAQaF~zOz6WymUDfO7ME02xpTq6iZ5QfAMCaAMZasP1hnDFuY5z9kxUJ-TfMyLTyjyLEQfP4lam1wVdxuX5tJEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"An Integrative Method for Identifying the Over-Annotated Protein-Coding Genes in Microbial Genomes","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/6/1/m_cover.gif?Expires=2147483647&Signature=qL9inMhepM2zFUhwbU8mgSKjtZVxpg~PmNu8fbxg-9GXAGTJaQrhv6qnZRDeMk7qu~VNHreHxR8Oz-VBGUn7klik4EpJ16UF40231HSss3FWSUANYRXx6t042t9kZJxg5tVYoWTO8VtSK1XYiJ1~Hb4XzfVloDJNBsISl1abmgO6rLxcEn6ZsFnsGHwCs2gNKgd6or0KznZwXt8UzZ8lxgCsA3E4dHjQXfOwyJhuIWr89hghDvrxBGqblSwMEBtNAQaF~zOz6WymUDfO7ME02xpTq6iZ5QfAMCaAMZasP1hnDFuY5z9kxUJ-TfMyLTyjyLEQfP4lam1wVdxuX5tJEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Integrative Method for Identifying the Over-Annotated Protein-Coding Genes in Microbial Genomes" />
    <meta property="og:description" content="Abstract.  The falsely annotated protein-coding genes have been deemed one of the major causes accounting for the annotating errors in public databases. Althoug" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/18/6/435/477266" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2011-09-08" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Yu, Jia-Feng" /><meta name="citation_author_institution" content="State Key Laboratory of Bioelectronics, School of Biological Science and Medical Engineering, Southeast University, Nanjing 210096, China" /><meta name="citation_author_institution" content="Shandong Province Key Laboratory of Biophysics for Functional Macromolecules, Department of Physics, Dezhou University, Dezhou 253023, China" /><meta name="citation_author" content="Xiao, Ke" /><meta name="citation_author_institution" content="State Key Laboratory of Bioelectronics, School of Biological Science and Medical Engineering, Southeast University, Nanjing 210096, China" /><meta name="citation_author" content="Jiang, Dong-Ke" /><meta name="citation_author_institution" content="State Key Laboratory of Bioelectronics, School of Biological Science and Medical Engineering, Southeast University, Nanjing 210096, China" /><meta name="citation_author" content="Guo, Jing" /><meta name="citation_author_institution" content="State Key Laboratory of Bioelectronics, School of Biological Science and Medical Engineering, Southeast University, Nanjing 210096, China" /><meta name="citation_author" content="Wang, Ji-Hua" /><meta name="citation_author_institution" content="Shandong Province Key Laboratory of Biophysics for Functional Macromolecules, Department of Physics, Dezhou University, Dezhou 253023, China" /><meta name="citation_author" content="Sun, Xiao" /><meta name="citation_author_institution" content="State Key Laboratory of Bioelectronics, School of Biological Science and Medical Engineering, Southeast University, Nanjing 210096, China" /><meta name="citation_title" content="An Integrative Method for Identifying the Over-Annotated Protein-Coding Genes in Microbial Genomes" /><meta name="citation_firstpage" content="435" /><meta name="citation_lastpage" content="449" /><meta name="citation_doi" content="10.1093/dnares/dsr030" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="18" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2011/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Errors in genome annotation; Trends Genet.;  citation_year=1999;  citation_volume=15;  citation_pages=132-3; " /><meta name="citation_reference" content="citation_title=Computational methods for gene annotation: the Arabidopsis genome; Curr. Opin. Biotech.;  citation_year=2001;  citation_volume=12;  citation_pages=126-30; " /><meta name="citation_reference" content="citation_title=On the total number of genes and their length distribution in complete microbial genomes; Trends Genet.;  citation_year=2001;  citation_volume=17;  citation_pages=425-8; " /><meta name="citation_reference" content="citation_title=Intrinsic errors in genome annotation; Trends Genet.;  citation_year=2001;  citation_volume=17;  citation_pages=429-31; " /><meta name="citation_reference" content="citation_title=Comprehensive analysis of pseudogenes in prokaryotes: widespread gene decay and failure of putative horizontally transferred genes; Genome Biol.;  citation_year=2004;  citation_volume=5;  citation_pages=R64" /><meta name="citation_reference" content="citation_title=Genome update: annotation quality in sequenced microbial genomes; Microbiol. SGM;  citation_year=2004;  citation_volume=150;  citation_pages=2015-7; " /><meta name="citation_reference" content="citation_title=Estimating the annotation error rate of curated GO database sequence annotations; BMC Bioinformatics;  citation_year=2007;  citation_volume=8;  citation_pages=170" /><meta name="citation_reference" content="citation_title=Genome re-annotation: a wiki solution?; Genome Biol.;  citation_year=2007;  citation_volume=8;  citation_pages=102" /><meta name="citation_reference" content="citation_title=Large gene overlaps in prokaryotic genomes: result of functional constraints or mispredictions?; BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=335" /><meta name="citation_reference" content="citation_title=Identification and correction of abnormal, incomplete and mispredicted proteins in public databases; BMC Bioinformatics;  citation_year=2008;  citation_volume=9;  citation_pages=353" /><meta name="citation_reference" content="citation_title=Evaluation of three automated genome annotations for Halorhabdus utahensis; PLoS One;  citation_year=2009;  citation_volume=4;  citation_pages=e6291" /><meta name="citation_reference" content="citation_title=Using comparative genome analysis to identify problems in annotated microbial genomes; Microbiol. SGM;  citation_year=2010;  citation_volume=156;  citation_pages=1909-17; " /><meta name="citation_reference" content="citation_title=Recognition of protein coding genes in the yeast genome at better than 95% accuracy based on the Z curve; Nucleic Acids Res.;  citation_year=2000;  citation_volume=28;  citation_pages=2804-14; " /><meta name="citation_reference" content="citation_title=Towards understanding the first genome sequence of a crenarchaeon by genome annotation using clusters of orthologous groups of proteins (COGs); citation_text=research0009.; Genome Biol.;  citation_year=2000;  citation_volume=1;  citation_pages=1-19; " /><meta name="citation_reference" content="citation_title=Identification of protein-coding genes in the genome of Vibriocholerae with more than 98% accuracy using occurrence frequencies of single nucleotides; Eur. J. Biochem.;  citation_year=2001;  citation_volume=268;  citation_pages=4261-8; " /><meta name="citation_reference" content="citation_title=Re-annotation of the genome sequence of Mycobacterium tuberculosis H37Rv; Microbiology;  citation_year=2002;  citation_volume=148;  citation_pages=2967-73; " /><meta name="citation_reference" content="citation_title=Re-annotation of genome microbial CoDing-Sequences: finding new genes and inaccurately annotated genes; BMC Bioinformatics;  citation_year=2002;  citation_volume=3;  citation_pages=5" /><meta name="citation_reference" content="citation_title=Distinguishing the ORFs from the ELFs:short bacterial genes and the annotation of genomes; Trends Genet.;  citation_year=2002;  citation_volume=18;  citation_pages=335-7; " /><meta name="citation_reference" content="citation_title=Gene recognition from questionable ORFs in bacterial and archaeal genomes; J. Biomol. Struct. Dyn.;  citation_year=2003;  citation_volume=21;  citation_pages=99-109; " /><meta name="citation_reference" content="citation_title=NCBI reference sequence project: update and current status; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=34-7; " /><meta name="citation_reference" content="citation_title=Gene recognition based on nucleotide distribution of ORFs in a hyper-thermophilic crenarchaeon, Aeropyrum pernix K1; DNA Res.;  citation_year=2004;  citation_volume=11;  citation_pages=361-70; " /><meta name="citation_reference" content="citation_title=Using purine skews to predict genes in AT-rich poxviruses; BMC Genomics;  citation_year=2005;  citation_volume=6;  citation_pages=22" /><meta name="citation_reference" content="citation_title=Re-prediction of protein coding genes in the genome of Amsacta moorei entomopoxvirus; J. Virol. Methods;  citation_year=2007;  citation_volume=146;  citation_pages=389-92; " /><meta name="citation_reference" content="citation_title=Reannotation of hypothetical ORFs in plant pathogen Erwinia carotovora subsp. atroseptica SCRI1043; FEBS J.;  citation_year=2008;  citation_volume=275;  citation_pages=198-206; " /><meta name="citation_reference" content="citation_title=Genome reannotation of Escherichia coli CFT073 with new insights into virulence; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=552" /><meta name="citation_reference" content="citation_title=Identify protein coding genes in the genomes of Aeropyrum pernix K1 and Chlorobium tepidum TLS; J. Biomol. Struct. Dyn.;  citation_year=2009;  citation_volume=26;  citation_pages=413-20; " /><meta name="citation_reference" content="citation_title=Missing genes in the annotation of prokaryotic genomes; BMC Bioinformatics;  citation_year=2010;  citation_volume=11;  citation_pages=131" /><meta name="citation_reference" content="citation_title=Re-annotation and re-analysis of the Campylobacter jejuni NCTC11168 genome sequence; BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=162" /><meta name="citation_reference" content="citation_title=The past, present and future of genome-wide re-annotation; Genome Biol.;  citation_year=2002;  citation_volume=3;  citation_pages=comment2001.1-6; " /><meta name="citation_reference" content="citation_title=YACOP: enhanced gene prediction obtained by a combination of existing methods; In Silico Biol.;  citation_year=2003;  citation_volume=3;  citation_pages=441-51; " /><meta name="citation_reference" content="citation_title=Development of joint application strategies for two microbial gene finders; Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=1622-31; " /><meta name="citation_reference" content="citation_title=ZCURVE: a new system for recognizing protein coding genes in bacterial and archaeal genomes; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=1780-9; " /><meta name="citation_reference" content="citation_title=H curves, a novel method of representation of nucleotide series especially suited for long DNA sequences; J. Biol. Chem.;  citation_year=1983;  citation_volume=258;  citation_pages=1318-27; " /><meta name="citation_reference" content="citation_title=A novel construction of genome space with biological geometry; DNA Res.;  citation_year=2010;  citation_volume=17;  citation_pages=155-68; " /><meta name="citation_reference" content="citation_title=TN curve: a novel 3D graphical representation of DNA sequence based on trinucleotides and its applications; J. Theor. Biol.;  citation_year=2009;  citation_volume=261;  citation_pages=459-68; " /><meta name="citation_reference" content="citation_title=Reannotation of protein coding genes based on an improved graphical representation of DNA sequence; J. Comput. Chem.;  citation_year=2010;  citation_volume=31;  citation_pages=2126-35; " /><meta name="citation_reference" content="citation_title=Analysis of distribution of bases in the coding sequences by a diagrammatic technique; Nucleic Acids Res.;  citation_year=1991;  citation_volume=19;  citation_pages=6313-7; " /><meta name="citation_reference" content="citation_title=Analysis of similarities/dissimilarities of DNA sequences based on a novel graphical representation; MATCH Commun. Math. Comput. Chem.;  citation_year=2010;  citation_volume=63;  citation_pages=493-512; " /><meta name="citation_reference" content="citation_title=Comparison of various algorithms for recognizing short coding sequences of human genes; Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=673-81; " /><meta name="citation_reference" content="citation_title=Studies on the relationships between the synonymous codon usage and protein secondary structural units; Biochem. Biophys. Res. Commun.;  citation_year=2000;  citation_volume=269;  citation_pages=692-6; " /><meta name="citation_reference" content="citation_title=Second codon positions of genes and the secondary structures of proteins. Relationships and implications for the origin of the genetic code; Gene;  citation_year=2000;  citation_volume=261;  citation_pages=63-9; " /><meta name="citation_reference" content="citation_title=Assessment of protein coding measures; Nucleic Acids Res.;  citation_year=1992;  citation_volume=20;  citation_pages=6441-50; " /><meta name="citation_reference" content="citation_title=Evaluation of gene structure prediction programs; Genomics;  citation_year=1996;  citation_volume=34;  citation_pages=353-7; " /><meta name="citation_reference" content="citation_title=Chargaff's legacy; Gene;  citation_year=2000;  citation_volume=261;  citation_pages=127-37; " /><meta name="citation_reference" content="citation_title=The guanine and cytosine content of genomic DNA and bacterial evolution; Proc. Natl Acad. Sci. USA;  citation_year=1987;  citation_volume=84;  citation_pages=166-9; " /><meta name="citation_reference" content="citation_title=The evolution of base composition and phylogenetic inference; Trends Ecol. Evol.;  citation_year=2000;  citation_volume=9;  citation_pages=365-9; " /><meta name="citation_reference" content="citation_title=DNA sequence of both chromosomes of the cholera pathogen Vibrio cholerae; Nature;  citation_year=2000;  citation_volume=406;  citation_pages=477-83; " /><meta name="citation_reference" content="citation_title=Codon usage in regulatory genes in Escherichia coli does not reflect selection for ‘rare’ codons; Nucleic Acids Res.;  citation_year=1986;  citation_volume=14;  citation_pages=7737-49; " /><meta name="citation_reference" content="citation_title=Translation framing code and frame-monitoring mechanism as suggested by the analysis of mRNA and 16 S rRNA nucleotide sequences; J. Mol. Biol.;  citation_year=1987;  citation_volume=194;  citation_pages=643-52; " /><meta name="citation_reference" content="citation_title=NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=61-5; " /><meta name="citation_reference" content="citation_title=The COG database: a tool for genome-scale analysis of protein functions and evolution; Nucleic Acids Res.;  citation_year=2000;  citation_volume=28;  citation_pages=33-6; " /><meta name="citation_reference" content="citation_title=Large-scale prokaryotic gene prediction and comparison to genome annotation; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=4322-9; " /><meta name="citation_reference" content="citation_title=Genomics of bacteria and archaea: the emerging dynamic view of the prokaryotic world; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=6688-719; " /><meta name="citation_reference" content="citation_title=HGT-DB: a database of putative horizontally transferred genes in prokaryotic complete genomes; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=187-9; " /><meta name="citation_reference" content="citation_title=Do different surrogate methods detect lateral genetic transfer events of different relative ages?; Trends Microbiol.;  citation_year=2006;  citation_volume=14;  citation_pages=4-8; " /><meta name="citation_reference" content="citation_title=Genetic flux over time in the Salmonella lineage; Genome Biol.;  citation_year=2007;  citation_volume=8;  citation_pages=R100" /><meta name="citation_reference" content="citation_title=Horizontal gene transfer and nucleotide compositional anomaly in large DNA viruses; BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=456" /><meta name="citation_reference" content="citation_title=Detecting laterally transferred genes: use of entropic clustering methods and genome position; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=4629-39; " /><meta name="citation_reference" content="citation_title=Horizontal gene transfer in bacterial and archaeal complete genomes; Genome Res.;  citation_year=2000;  citation_volume=10;  citation_pages=1719-25; " /><meta name="citation_reference" content="citation_title=Improved microbial gene identification with GLIMMER; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=4636-41; " /><meta name="citation_reference" content="citation_title=GeneMark.hmm: new solutions for gene finding; Nucleic Acids Res.;  citation_year=1998;  citation_volume=26;  citation_pages=1107-15; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/18/6/435/1290281/dsr030.pdf" /><meta name="description" content="Abstract.  The falsely annotated protein-coding genes have been deemed one of the major causes accounting for the annotating errors in public databases. Althoug" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/18/6/435/477266" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/18/6/435/477266" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/18/6/435/477266");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsr030';
    </script>
<title>
	Integrative Method for Identifying the Over-Annotated Protein-Coding Genes in Microbial Genomes | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./477266" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BfmAATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWXUsIEppYS1GZW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU3RhdGUgS2V5IExhYm9yYXRvcnkgb2YgQmlvZWxlY3Ryb25pY3MsIFNjaG9vbCBvZiBCaW9sb2dpY2FsIFNjaWVuY2UgYW5kIE1lZGljYWwgRW5naW5lZXJpbmcsIFNvdXRoZWFzdCBVbml2ZXJzaXR5LCBOYW5qaW5nIDIxMDA5NiwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTaGFuZG9uZyBQcm92aW5jZSBLZXkgTGFib3JhdG9yeSBvZiBCaW9waHlzaWNzIGZvciBGdW5jdGlvbmFsIE1hY3JvbW9sZWN1bGVzLCBEZXBhcnRtZW50IG9mIFBoeXNpY3MsIERlemhvdSBVbml2ZXJzaXR5LCBEZXpob3UgMjUzMDIzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlhpYW8sIEtlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU3RhdGUgS2V5IExhYm9yYXRvcnkgb2YgQmlvZWxlY3Ryb25pY3MsIFNjaG9vbCBvZiBCaW9sb2dpY2FsIFNjaWVuY2UgYW5kIE1lZGljYWwgRW5naW5lZXJpbmcsIFNvdXRoZWFzdCBVbml2ZXJzaXR5LCBOYW5qaW5nIDIxMDA5NiwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJKaWFuZywgRG9uZy1LZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlN0YXRlIEtleSBMYWJvcmF0b3J5IG9mIEJpb2VsZWN0cm9uaWNzLCBTY2hvb2wgb2YgQmlvbG9naWNhbCBTY2llbmNlIGFuZCBNZWRpY2FsIEVuZ2luZWVyaW5nLCBTb3V0aGVhc3QgVW5pdmVyc2l0eSwgTmFuamluZyAyMTAwOTYsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR3VvLCBKaW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU3RhdGUgS2V5IExhYm9yYXRvcnkgb2YgQmlvZWxlY3Ryb25pY3MsIFNjaG9vbCBvZiBCaW9sb2dpY2FsIFNjaWVuY2UgYW5kIE1lZGljYWwgRW5naW5lZXJpbmcsIFNvdXRoZWFzdCBVbml2ZXJzaXR5LCBOYW5qaW5nIDIxMDA5NiwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXYW5nLCBKaS1IdWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTaGFuZG9uZyBQcm92aW5jZSBLZXkgTGFib3JhdG9yeSBvZiBCaW9waHlzaWNzIGZvciBGdW5jdGlvbmFsIE1hY3JvbW9sZWN1bGVzLCBEZXBhcnRtZW50IG9mIFBoeXNpY3MsIERlemhvdSBVbml2ZXJzaXR5LCBEZXpob3UgMjUzMDIzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlN1biwgWGlhbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlN0YXRlIEtleSBMYWJvcmF0b3J5IG9mIEJpb2VsZWN0cm9uaWNzLCBTY2hvb2wgb2YgQmlvbG9naWNhbCBTY2llbmNlIGFuZCBNZWRpY2FsIEVuZ2luZWVyaW5nLCBTb3V0aGVhc3QgVW5pdmVyc2l0eSwgTmFuamluZyAyMTAwOTYsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJBbiBJbnRlZ3JhdGl2ZSBNZXRob2QgZm9yIElkZW50aWZ5aW5nIHRoZSBPdmVyLUFubm90YXRlZCBQcm90ZWluLUNvZGluZyBHZW5lcyBpbiBNaWNyb2JpYWwgR2Vub21lcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjQzNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNDQ5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNyMDMwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTEvMTIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FcnJvcnMgaW4gZ2Vub21lIGFubm90YXRpb247IFRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xMzItMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcHV0YXRpb25hbCBtZXRob2RzIGZvciBnZW5lIGFubm90YXRpb246IHRoZSBBcmFiaWRvcHNpcyBnZW5vbWU7IEN1cnIuIE9waW4uIEJpb3RlY2guOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTI2LTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbiB0aGUgdG90YWwgbnVtYmVyIG9mIGdlbmVzIGFuZCB0aGVpciBsZW5ndGggZGlzdHJpYnV0aW9uIGluIGNvbXBsZXRlIG1pY3JvYmlhbCBnZW5vbWVzOyBUcmVuZHMgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9NDI1LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludHJpbnNpYyBlcnJvcnMgaW4gZ2Vub21lIGFubm90YXRpb247IFRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz00MjktMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXByZWhlbnNpdmUgYW5hbHlzaXMgb2YgcHNldWRvZ2VuZXMgaW4gcHJva2FyeW90ZXM6IHdpZGVzcHJlYWQgZ2VuZSBkZWNheSBhbmQgZmFpbHVyZSBvZiBwdXRhdGl2ZSBob3Jpem9udGFsbHkgdHJhbnNmZXJyZWQgZ2VuZXM7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9UjY0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHVwZGF0ZTogYW5ub3RhdGlvbiBxdWFsaXR5IGluIHNlcXVlbmNlZCBtaWNyb2JpYWwgZ2Vub21lczsgTWljcm9iaW9sLiBTR007ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTUwOyAgY2l0YXRpb25fcGFnZXM9MjAxNS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fc3RpbWF0aW5nIHRoZSBhbm5vdGF0aW9uIGVycm9yIHJhdGUgb2YgY3VyYXRlZCBHTyBkYXRhYmFzZSBzZXF1ZW5jZSBhbm5vdGF0aW9uczsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0xNzAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgcmUtYW5ub3RhdGlvbjogYSB3aWtpIHNvbHV0aW9uPzsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0xMDIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYXJnZSBnZW5lIG92ZXJsYXBzIGluIHByb2thcnlvdGljIGdlbm9tZXM6IHJlc3VsdCBvZiBmdW5jdGlvbmFsIGNvbnN0cmFpbnRzIG9yIG1pc3ByZWRpY3Rpb25zPzsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0zMzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBhbmQgY29ycmVjdGlvbiBvZiBhYm5vcm1hbCwgaW5jb21wbGV0ZSBhbmQgbWlzcHJlZGljdGVkIHByb3RlaW5zIGluIHB1YmxpYyBkYXRhYmFzZXM7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MzUzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZhbHVhdGlvbiBvZiB0aHJlZSBhdXRvbWF0ZWQgZ2Vub21lIGFubm90YXRpb25zIGZvciBIYWxvcmhhYmR1cyB1dGFoZW5zaXM7IFBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz1lNjI5MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVzaW5nIGNvbXBhcmF0aXZlIGdlbm9tZSBhbmFseXNpcyB0byBpZGVudGlmeSBwcm9ibGVtcyBpbiBhbm5vdGF0ZWQgbWljcm9iaWFsIGdlbm9tZXM7IE1pY3JvYmlvbC4gU0dNOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE1NjsgIGNpdGF0aW9uX3BhZ2VzPTE5MDktMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlY29nbml0aW9uIG9mIHByb3RlaW4gY29kaW5nIGdlbmVzIGluIHRoZSB5ZWFzdCBnZW5vbWUgYXQgYmV0dGVyIHRoYW4gOTUlIGFjY3VyYWN5IGJhc2VkIG9uIHRoZSBaIGN1cnZlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0yODA0LTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ub3dhcmRzIHVuZGVyc3RhbmRpbmcgdGhlIGZpcnN0IGdlbm9tZSBzZXF1ZW5jZSBvZiBhIGNyZW5hcmNoYWVvbiBieSBnZW5vbWUgYW5ub3RhdGlvbiB1c2luZyBjbHVzdGVycyBvZiBvcnRob2xvZ291cyBncm91cHMgb2YgcHJvdGVpbnMgKENPR3MpOyBjaXRhdGlvbl90ZXh0PXJlc2VhcmNoMDAwOS47IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MS0xOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgcHJvdGVpbi1jb2RpbmcgZ2VuZXMgaW4gdGhlIGdlbm9tZSBvZiBWaWJyaW9jaG9sZXJhZSB3aXRoIG1vcmUgdGhhbiA5OCUgYWNjdXJhY3kgdXNpbmcgb2NjdXJyZW5jZSBmcmVxdWVuY2llcyBvZiBzaW5nbGUgbnVjbGVvdGlkZXM7IEV1ci4gSi4gQmlvY2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MjY4OyAgY2l0YXRpb25fcGFnZXM9NDI2MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZS1hbm5vdGF0aW9uIG9mIHRoZSBnZW5vbWUgc2VxdWVuY2Ugb2YgTXljb2JhY3Rlcml1bSB0dWJlcmN1bG9zaXMgSDM3UnY7IE1pY3JvYmlvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDg7ICBjaXRhdGlvbl9wYWdlcz0yOTY3LTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZS1hbm5vdGF0aW9uIG9mIGdlbm9tZSBtaWNyb2JpYWwgQ29EaW5nLVNlcXVlbmNlczogZmluZGluZyBuZXcgZ2VuZXMgYW5kIGluYWNjdXJhdGVseSBhbm5vdGF0ZWQgZ2VuZXM7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc3Rpbmd1aXNoaW5nIHRoZSBPUkZzIGZyb20gdGhlIEVMRnM6c2hvcnQgYmFjdGVyaWFsIGdlbmVzIGFuZCB0aGUgYW5ub3RhdGlvbiBvZiBnZW5vbWVzOyBUcmVuZHMgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MzM1LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgcmVjb2duaXRpb24gZnJvbSBxdWVzdGlvbmFibGUgT1JGcyBpbiBiYWN0ZXJpYWwgYW5kIGFyY2hhZWFsIGdlbm9tZXM7IEouIEJpb21vbC4gU3RydWN0LiBEeW4uOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9OTktMTA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OQ0JJIHJlZmVyZW5jZSBzZXF1ZW5jZSBwcm9qZWN0OiB1cGRhdGUgYW5kIGN1cnJlbnQgc3RhdHVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0zNC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIHJlY29nbml0aW9uIGJhc2VkIG9uIG51Y2xlb3RpZGUgZGlzdHJpYnV0aW9uIG9mIE9SRnMgaW4gYSBoeXBlci10aGVybW9waGlsaWMgY3JlbmFyY2hhZW9uLCBBZXJvcHlydW0gcGVybml4IEsxOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTM2MS03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXNpbmcgcHVyaW5lIHNrZXdzIHRvIHByZWRpY3QgZ2VuZXMgaW4gQVQtcmljaCBwb3h2aXJ1c2VzOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmUtcHJlZGljdGlvbiBvZiBwcm90ZWluIGNvZGluZyBnZW5lcyBpbiB0aGUgZ2Vub21lIG9mIEFtc2FjdGEgbW9vcmVpIGVudG9tb3BveHZpcnVzOyBKLiBWaXJvbC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDY7ICBjaXRhdGlvbl9wYWdlcz0zODktOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlYW5ub3RhdGlvbiBvZiBoeXBvdGhldGljYWwgT1JGcyBpbiBwbGFudCBwYXRob2dlbiBFcndpbmlhIGNhcm90b3ZvcmEgc3Vic3AuIGF0cm9zZXB0aWNhIFNDUkkxMDQzOyBGRUJTIEouOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI3NTsgIGNpdGF0aW9uX3BhZ2VzPTE5OC0yMDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSByZWFubm90YXRpb24gb2YgRXNjaGVyaWNoaWEgY29saSBDRlQwNzMgd2l0aCBuZXcgaW5zaWdodHMgaW50byB2aXJ1bGVuY2U7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTU1MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZ5IHByb3RlaW4gY29kaW5nIGdlbmVzIGluIHRoZSBnZW5vbWVzIG9mIEFlcm9weXJ1bSBwZXJuaXggSzEgYW5kIENobG9yb2JpdW0gdGVwaWR1bSBUTFM7IEouIEJpb21vbC4gU3RydWN0LiBEeW4uOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9NDEzLTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaXNzaW5nIGdlbmVzIGluIHRoZSBhbm5vdGF0aW9uIG9mIHByb2thcnlvdGljIGdlbm9tZXM7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTEzMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlLWFubm90YXRpb24gYW5kIHJlLWFuYWx5c2lzIG9mIHRoZSBDYW1weWxvYmFjdGVyIGplanVuaSBOQ1RDMTExNjggZ2Vub21lIHNlcXVlbmNlOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTE2MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBwYXN0LCBwcmVzZW50IGFuZCBmdXR1cmUgb2YgZ2Vub21lLXdpZGUgcmUtYW5ub3RhdGlvbjsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz1jb21tZW50MjAwMS4xLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVlBQ09QOiBlbmhhbmNlZCBnZW5lIHByZWRpY3Rpb24gb2J0YWluZWQgYnkgYSBjb21iaW5hdGlvbiBvZiBleGlzdGluZyBtZXRob2RzOyBJbiBTaWxpY28gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTQ0MS01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2Ygam9pbnQgYXBwbGljYXRpb24gc3RyYXRlZ2llcyBmb3IgdHdvIG1pY3JvYmlhbCBnZW5lIGZpbmRlcnM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTYyMi0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9WkNVUlZFOiBhIG5ldyBzeXN0ZW0gZm9yIHJlY29nbml6aW5nIHByb3RlaW4gY29kaW5nIGdlbmVzIGluIGJhY3RlcmlhbCBhbmQgYXJjaGFlYWwgZ2Vub21lczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MTc4MC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IIGN1cnZlcywgYSBub3ZlbCBtZXRob2Qgb2YgcmVwcmVzZW50YXRpb24gb2YgbnVjbGVvdGlkZSBzZXJpZXMgZXNwZWNpYWxseSBzdWl0ZWQgZm9yIGxvbmcgRE5BIHNlcXVlbmNlczsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5ODM7ICBjaXRhdGlvbl92b2x1bWU9MjU4OyAgY2l0YXRpb25fcGFnZXM9MTMxOC0yNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBub3ZlbCBjb25zdHJ1Y3Rpb24gb2YgZ2Vub21lIHNwYWNlIHdpdGggYmlvbG9naWNhbCBnZW9tZXRyeTsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0xNTUtNjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVROIGN1cnZlOiBhIG5vdmVsIDNEIGdyYXBoaWNhbCByZXByZXNlbnRhdGlvbiBvZiBETkEgc2VxdWVuY2UgYmFzZWQgb24gdHJpbnVjbGVvdGlkZXMgYW5kIGl0cyBhcHBsaWNhdGlvbnM7IEouIFRoZW9yLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNjE7ICBjaXRhdGlvbl9wYWdlcz00NTktNjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlYW5ub3RhdGlvbiBvZiBwcm90ZWluIGNvZGluZyBnZW5lcyBiYXNlZCBvbiBhbiBpbXByb3ZlZCBncmFwaGljYWwgcmVwcmVzZW50YXRpb24gb2YgRE5BIHNlcXVlbmNlOyBKLiBDb21wdXQuIENoZW0uOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MjEyNi0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgZGlzdHJpYnV0aW9uIG9mIGJhc2VzIGluIHRoZSBjb2Rpbmcgc2VxdWVuY2VzIGJ5IGEgZGlhZ3JhbW1hdGljIHRlY2huaXF1ZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9NjMxMy03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBzaW1pbGFyaXRpZXMvZGlzc2ltaWxhcml0aWVzIG9mIEROQSBzZXF1ZW5jZXMgYmFzZWQgb24gYSBub3ZlbCBncmFwaGljYWwgcmVwcmVzZW50YXRpb247IE1BVENIIENvbW11bi4gTWF0aC4gQ29tcHV0LiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT02MzsgIGNpdGF0aW9uX3BhZ2VzPTQ5My01MTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmlzb24gb2YgdmFyaW91cyBhbGdvcml0aG1zIGZvciByZWNvZ25pemluZyBzaG9ydCBjb2Rpbmcgc2VxdWVuY2VzIG9mIGh1bWFuIGdlbmVzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTY3My04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3R1ZGllcyBvbiB0aGUgcmVsYXRpb25zaGlwcyBiZXR3ZWVuIHRoZSBzeW5vbnltb3VzIGNvZG9uIHVzYWdlIGFuZCBwcm90ZWluIHNlY29uZGFyeSBzdHJ1Y3R1cmFsIHVuaXRzOyBCaW9jaGVtLiBCaW9waHlzLiBSZXMuIENvbW11bi47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MjY5OyAgY2l0YXRpb25fcGFnZXM9NjkyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlY29uZCBjb2RvbiBwb3NpdGlvbnMgb2YgZ2VuZXMgYW5kIHRoZSBzZWNvbmRhcnkgc3RydWN0dXJlcyBvZiBwcm90ZWlucy4gUmVsYXRpb25zaGlwcyBhbmQgaW1wbGljYXRpb25zIGZvciB0aGUgb3JpZ2luIG9mIHRoZSBnZW5ldGljIGNvZGU7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MjYxOyAgY2l0YXRpb25fcGFnZXM9NjMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzZXNzbWVudCBvZiBwcm90ZWluIGNvZGluZyBtZWFzdXJlczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9NjQ0MS01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZhbHVhdGlvbiBvZiBnZW5lIHN0cnVjdHVyZSBwcmVkaWN0aW9uIHByb2dyYW1zOyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPTM1My03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyZ2FmZidzIGxlZ2FjeTsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjE7ICBjaXRhdGlvbl9wYWdlcz0xMjctMzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBndWFuaW5lIGFuZCBjeXRvc2luZSBjb250ZW50IG9mIGdlbm9taWMgRE5BIGFuZCBiYWN0ZXJpYWwgZXZvbHV0aW9uOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTg0OyAgY2l0YXRpb25fcGFnZXM9MTY2LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBldm9sdXRpb24gb2YgYmFzZSBjb21wb3NpdGlvbiBhbmQgcGh5bG9nZW5ldGljIGluZmVyZW5jZTsgVHJlbmRzIEVjb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0zNjUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIHNlcXVlbmNlIG9mIGJvdGggY2hyb21vc29tZXMgb2YgdGhlIGNob2xlcmEgcGF0aG9nZW4gVmlicmlvIGNob2xlcmFlOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NDA2OyAgY2l0YXRpb25fcGFnZXM9NDc3LTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiB1c2FnZSBpbiByZWd1bGF0b3J5IGdlbmVzIGluIEVzY2hlcmljaGlhIGNvbGkgZG9lcyBub3QgcmVmbGVjdCBzZWxlY3Rpb24gZm9yIOKAmHJhcmXigJkgY29kb25zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5ODY7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz03NzM3LTQ5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2xhdGlvbiBmcmFtaW5nIGNvZGUgYW5kIGZyYW1lLW1vbml0b3JpbmcgbWVjaGFuaXNtIGFzIHN1Z2dlc3RlZCBieSB0aGUgYW5hbHlzaXMgb2YgbVJOQSBhbmQgMTYgUyByUk5BIG51Y2xlb3RpZGUgc2VxdWVuY2VzOyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTE5NDsgIGNpdGF0aW9uX3BhZ2VzPTY0My01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TkNCSSByZWZlcmVuY2Ugc2VxdWVuY2VzIChSZWZTZXEpOiBhIGN1cmF0ZWQgbm9uLXJlZHVuZGFudCBzZXF1ZW5jZSBkYXRhYmFzZSBvZiBnZW5vbWVzLCB0cmFuc2NyaXB0cyBhbmQgcHJvdGVpbnM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPTYxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBDT0cgZGF0YWJhc2U6IGEgdG9vbCBmb3IgZ2Vub21lLXNjYWxlIGFuYWx5c2lzIG9mIHByb3RlaW4gZnVuY3Rpb25zIGFuZCBldm9sdXRpb247IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTMzLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxhcmdlLXNjYWxlIHByb2thcnlvdGljIGdlbmUgcHJlZGljdGlvbiBhbmQgY29tcGFyaXNvbiB0byBnZW5vbWUgYW5ub3RhdGlvbjsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz00MzIyLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWNzIG9mIGJhY3RlcmlhIGFuZCBhcmNoYWVhOiB0aGUgZW1lcmdpbmcgZHluYW1pYyB2aWV3IG9mIHRoZSBwcm9rYXJ5b3RpYyB3b3JsZDsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9NjY4OC03MTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhHVC1EQjogYSBkYXRhYmFzZSBvZiBwdXRhdGl2ZSBob3Jpem9udGFsbHkgdHJhbnNmZXJyZWQgZ2VuZXMgaW4gcHJva2FyeW90aWMgY29tcGxldGUgZ2Vub21lczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MTg3LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURvIGRpZmZlcmVudCBzdXJyb2dhdGUgbWV0aG9kcyBkZXRlY3QgbGF0ZXJhbCBnZW5ldGljIHRyYW5zZmVyIGV2ZW50cyBvZiBkaWZmZXJlbnQgcmVsYXRpdmUgYWdlcz87IFRyZW5kcyBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9NC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGZsdXggb3ZlciB0aW1lIGluIHRoZSBTYWxtb25lbGxhIGxpbmVhZ2U7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9UjEwMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhvcml6b250YWwgZ2VuZSB0cmFuc2ZlciBhbmQgbnVjbGVvdGlkZSBjb21wb3NpdGlvbmFsIGFub21hbHkgaW4gbGFyZ2UgRE5BIHZpcnVzZXM7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9NDU2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV0ZWN0aW5nIGxhdGVyYWxseSB0cmFuc2ZlcnJlZCBnZW5lczogdXNlIG9mIGVudHJvcGljIGNsdXN0ZXJpbmcgbWV0aG9kcyBhbmQgZ2Vub21lIHBvc2l0aW9uOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz00NjI5LTM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ib3Jpem9udGFsIGdlbmUgdHJhbnNmZXIgaW4gYmFjdGVyaWFsIGFuZCBhcmNoYWVhbCBjb21wbGV0ZSBnZW5vbWVzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTE3MTktMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcHJvdmVkIG1pY3JvYmlhbCBnZW5lIGlkZW50aWZpY2F0aW9uIHdpdGggR0xJTU1FUjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9NDYzNi00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZU1hcmsuaG1tOiBuZXcgc29sdXRpb25zIGZvciBnZW5lIGZpbmRpbmc7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTExMDctMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xOC82LzQzNS8xMjkwMjgxL2RzcjAzMC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgVGhlIGZhbHNlbHkgYW5ub3RhdGVkIHByb3RlaW4tY29kaW5nIGdlbmVzIGhhdmUgYmVlbiBkZWVtZWQgb25lIG9mIHRoZSBtYWpvciBjYXVzZXMgYWNjb3VudGluZyBmb3IgdGhlIGFubm90YXRpbmcgZXJyb3JzIGluIHB1YmxpYyBkYXRhYmFzZXMuIEFsdGhvdWciIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQ3NzI2NiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQjbDiy7lx9coMNw/ALaKwrY8sRlA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f6%2f435%2f477266" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f6%2f435%2f477266" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=18719" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/18/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/6/1/m_cover.gif?Expires=2147483647&amp;Signature=qL9inMhepM2zFUhwbU8mgSKjtZVxpg~PmNu8fbxg-9GXAGTJaQrhv6qnZRDeMk7qu~VNHreHxR8Oz-VBGUn7klik4EpJ16UF40231HSss3FWSUANYRXx6t042t9kZJxg5tVYoWTO8VtSK1XYiJ1~Hb4XzfVloDJNBsISl1abmgO6rLxcEn6ZsFnsGHwCs2gNKgd6or0KznZwXt8UzZ8lxgCsA3E4dHjQXfOwyJhuIWr89hghDvrxBGqblSwMEBtNAQaF~zOz6WymUDfO7ME02xpTq6iZ5QfAMCaAMZasP1hnDFuY5z9kxUJ-TfMyLTyjyLEQfP4lam1wVdxuX5tJEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/18/6">
                        <div class="volume trailing-comma">Volume 18</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2011        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7207916">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7207916">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7207918">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7207918">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7207921">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7207921">2. Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7207939">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7207939">3. Results and discussions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7207969">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7207969">4. Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7207972">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7207972">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7207974">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7207974">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7207976">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7207976">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/18/6/423/477017">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/18/6/451/477488">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">An Integrative Method for Identifying the Over-Annotated Protein-Coding Genes in Microbial Genomes
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jia-Feng Yu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">State Key Laboratory of Bioelectronics, School of Biological Science and Medical Engineering, Southeast University</div>, <div class="addr-line">Nanjing 210096</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Shandong Province Key Laboratory of Biophysics for Functional Macromolecules, Department of Physics</div>, <div class="insititution">Dezhou University</div>, <div class="addr-line">Dezhou 253023</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 25-83795174. Fax. +86 25-83792349. Email: <a href="mailto:jfyu1979@126.com" target="_blank">jfyu1979@126.com</a> (J.-F.Y.); <a href="mailto:xsun@seu.edu.cn" target="_blank">xsun@seu.edu.cn</a> (X.S.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jia-Feng+Yu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yu J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yu J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jia-Feng Yu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ke Xiao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">State Key Laboratory of Bioelectronics, School of Biological Science and Medical Engineering, Southeast University</div>, <div class="addr-line">Nanjing 210096</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ke+Xiao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xiao K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Xiao K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ke Xiao</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Dong-Ke Jiang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">State Key Laboratory of Bioelectronics, School of Biological Science and Medical Engineering, Southeast University</div>, <div class="addr-line">Nanjing 210096</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Dong-Ke+Jiang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jiang D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jiang D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Dong-Ke Jiang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jing Guo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">State Key Laboratory of Bioelectronics, School of Biological Science and Medical Engineering, Southeast University</div>, <div class="addr-line">Nanjing 210096</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jing+Guo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Guo J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Guo J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jing Guo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ji-Hua Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Shandong Province Key Laboratory of Biophysics for Functional Macromolecules, Department of Physics</div>, <div class="insititution">Dezhou University</div>, <div class="addr-line">Dezhou 253023</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ji-Hua+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ji-Hua Wang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiao Sun</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">State Key Laboratory of Bioelectronics, School of Biological Science and Medical Engineering, Southeast University</div>, <div class="addr-line">Nanjing 210096</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 25-83795174. Fax. +86 25-83792349. Email: <a href="mailto:jfyu1979@126.com" target="_blank">jfyu1979@126.com</a> (J.-F.Y.); <a href="mailto:xsun@seu.edu.cn" target="_blank">xsun@seu.edu.cn</a> (X.S.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiao+Sun">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sun X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sun X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xiao Sun</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Kenta Nakai</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 18, Issue 6, 1 December 2011, Pages 435–449, <a href='https://doi.org/10.1093/dnares/dsr030'>https://doi.org/10.1093/dnares/dsr030</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">08 September 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">29 April 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">01 August 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="477266" href="/dnaresearch/article-pdf/18/6/435/1290281/dsr030.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Jia-Feng Yu, Ke Xiao, Dong-Ke Jiang, Jing Guo, Ji-Hua Wang, Xiao Sun;  An Integrative Method for Identifying the Over-Annotated Protein-Coding Genes in Microbial Genomes, <em>DNA Research</em>, Volume 18, Issue 6, 1 December 2011, Pages 435–449, <a href='https://doi.org/10.1093/dnares/dsr030'>https://doi.org/10.1093/dnares/dsr030</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/477266?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/477266?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/477266?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/477266?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/477266?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/477266?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/477266?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=An%20Integrative%20Method%20for%20Identifying%20the%20Over-Annotated%20Protein-Coding%20Genes%20in%20Microbial%20Genomes&amp;publicationDate=2011-09-08&amp;volumeNum=18&amp;issueNum=6&amp;author=Yu%2C%20Jia-Feng%3B%20Xiao%2C%20Ke&amp;startPage=435&amp;endPage=449&amp;contentId=10.1093%2Fdnares%2Fdsr030&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f6%2f435%2f477266" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7207916 id="7207916" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The falsely annotated protein-coding genes have been deemed one of the major causes accounting for the annotating errors in public databases. Although many filtering approaches have been designed for the over-annotated protein-coding genes, some are questionable due to the resultant increase in false negative. Furthermore, there is no webserver or software specifically devised for the problem of over-annotation. In this study, we propose an integrative algorithm for detecting the over-annotated protein-coding genes in microorganisms. Overall, an average accuracy of 99.94% is achieved over 61 microbial genomes. The extremely high accuracy indicates that the presented algorithm is efficient to differentiate the protein-coding genes from the non-coding open reading frames. Abundant analyses show that the predicting results are reliable and the integrative algorithm is robust and convenient. Our analysis also indicates that the over-annotated protein-coding genes can cause the false positive of horizontal gene transfers detection. The webserver of the proposed algorithm can be freely accessible from <a class="link link-uri" href="http://www.cbi.seu.edu.cn/RPGM">www.cbi.seu.edu.cn/RPGM</a>.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;protein-coding gene&quot;">protein-coding gene</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;microbial genome&quot;">microbial genome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="re-annotation">re-annotation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;horizontal gene transfer&quot;">horizontal gene transfer</a></div>                            <h2 scrollto-destination=7207918 id="7207918" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Up to now, thousands of microbial genomes have been published in public databases. The explosive growth number of available genomic sequences presents unprecedented opportunities for probing the secret of life and extract biological information on genetics, which is highly dependent on the annotation quality of each genome. In most cases, many people think that gene finding in prokaryotic genomes is relatively easy due to lack of introns, and they usually deem the genes deposited in public databases such as GenBank or EMBL are correctly annotated. However, more and more researches indicate that the issue of gene finding in microbial genomes is far from thoroughly resolved; the annotation quality of microbial genomes has been questioned continuously in the past several years.<sup><a reveal-id="DSR030C1  DSR030C29" data-open="DSR030C1  DSR030C29" class="link link-ref link-reveal xref-bibr">1–29</a></sup> Many studies implied that some annotated protein-coding genes in most completely sequenced microbial genomes do not encode any proteins actually, but random open reading frames (ORFs) occurring by chance.<sup><a reveal-id="DSR030C1  DSR030C9" data-open="DSR030C1  DSR030C9" class="link link-ref link-reveal xref-bibr">1–9</a>,<a reveal-id="DSR030C11  DSR030C26" data-open="DSR030C11  DSR030C26" class="link link-ref link-reveal xref-bibr">11–26</a></sup> In current public databases, the careful annotators have marked those questionable ORFs as hypothetical, then which of them encode protein, and which do not? As many users may take it for granted that all the annotated genes are true protein coding, this can easily lead to wrong conclusions. Then the researchers need to be aware of the existing errors in the annotation of even well-studied genomes and consider additional quality control for their results. Although many groups have performed re-annotation on different microbial genomes, some filtering strategies for the over-annotated ORFs seemed to be questionable due to the resultant increase in false negative, and then there is still much room for improvement. Moreover, it is regretful that there is no webserver or software reported for addressing the problem of over-annotated genes in public databases. Then the update speed of current databases based on the one-by-one re-annotating strategy is far from practical applications and the increasing rate of novel sequenced genomes, which will decrease greatly the value of public databases. Therefore, the problem is still open for protein-coding genes identification in microbial genomes.</p><p>Recent researches show that development of techniques by combination of multiple programs may produce a higher performance in prediction problems.<sup><a reveal-id="DSR030C30  DSR030C32" data-open="DSR030C30  DSR030C32" class="link link-ref link-reveal xref-bibr">30–32</a></sup> In this paper, we put forward a meta-approach for identifying the falsely annotated protein-coding genes in microbial genomes. The algorithm is evaluated by 61 microbial genomes and an average accuracy of 99.94% is obtained. The extremely high accuracy shows that the integrated method can grasp universal information of protein-coding genes. Subsequent analysis indicates that the predicting results are much reliable. In order to facilitate the potential users, we exploited an interface-friendly webserver aiming to NCBI's RefSeq resources. This platform only needs to input the accession number of the queried microbial genome, and then it is much convenient and easy to operate.</p>                            <h2 scrollto-destination=7207921 id="7207921" class="section-title" data-legacy-id=s2>2. Methods</h2>
                            <h3 scrollto-destination=7207922 id="7207922" class="section-title" data-legacy-id=s2a>2.1. Numerical descriptors</h3>
<p>The walking model method is such a kind of graphical representation that can transform biological sequences into visual patterns based on different encoding strategies. Since the first walking model was proposed by Hamori and Ruskin,<sup><a reveal-id="DSR030C33" data-open="DSR030C33" class="link link-ref link-reveal xref-bibr">33</a></sup> many graphical approaches have been reported for DNA sequences, which can provide intuitive pictures or useful insights for helping analysing complicated relations in biological systems.<sup><a reveal-id="DSR030C34" data-open="DSR030C34" class="link link-ref link-reveal xref-bibr">34</a></sup> The methodology proposed in this work is based on the TN curve and <em>Z</em> curve, from which we derive 75 numerical parameters to exhibit the intrinsic properties of protein-coding genes.</p><p>The TN curve is a recently proposed walking model by us, with which one can inspect information of trinucleotides both qualitatively and quantitatively.<sup><a reveal-id="DSR030C35" data-open="DSR030C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Consideration of trinucleotides instead of individual and dual nucleotides has superior advantages for protein-coding genes. In this paper, we derive 54 parameters based on the encoding strategy of the TN curve to exhibit the specific structures of protein genes numerically, which are briefly introduced below.</p><p>(i) According to the encoding strategy of the TN curve, each kind of trinucleotide can be represented by a 2D Cartesian coordinates (<em>x</em>, <em>y</em>). Here, we determine the signs of <em>x</em> and <em>y</em> by the base at the first position ({+, +}→A, {−, +}→G, {−, −}→C, {+, −}→T) and decide the absolute values of <em>x</em> and <em>y</em> by the bases at the second and third positions (1→A, 2→G, 3→C, 4→T), respectively. Taking GCG as an example, the base ‘G’ at the first position denotes that the signs of <em>x</em> and <em>y</em> are negative and positive, respectively; the base ‘C’ at the second position and the base ‘G’ at the third position imply that the absolute values of <em>x</em> and <em>y</em> are 3 and 2, respectively. Therefore, GCG is represented by (−3, 2). In this way, other kinds of trinucleotides can also be denoted numerically. According to the definition, <em>x</em>&gt; 0 or <em>x</em>&lt; 0 mean that the base at the first position is an element of (A, T) or (G, C), which corresponds to week-H bond (A, T)/strong-H bond (G, C) groups, <em>y</em>&gt; 0 or <em>y</em>&lt; 0 mean that the first base is (A, G) or (C, T), which corresponds to purine (A, G)/pyrimidine (C, T) groups. Letting <em>z</em>= <em>x</em>× <em>y</em>, it is noted that when <em>z</em>&gt; 0, <em>x</em> and <em>y</em> are positive or negative simultaneously, the second base must be A or C, which corresponds to the amino group, when <em>z</em>&lt; 0, the sign of <em>x</em> is opposite to <em>y</em>, the first base must be G or T, which corresponds to the keto group. Thus, the 64 kinds of trinucleotides can be classified into two groups in three ways based on <em>x</em>, <em>y</em> and <em>z</em>, respectively. On the other hand, <em>x</em> links the first and second bases of a trinucleotide, and this can be used as an approximate descriptor of dual nucleotide. Similar results can be obtained for <em>y</em> which links the first and third bases of a nucleotide triplet. Then, we can obtain more information from these parameters.<sup><a reveal-id="DSR030C35" data-open="DSR030C35" class="link link-ref link-reveal xref-bibr">35</a></sup></p><div class="block-child-p">In a protein-coding sequence, there are three forward and three reverse reading frames, of which usually only one can encode protein sequence. Supposing <em>S</em>= <em>s</em><sub>1</sub><em>s</em><sub>2</sub><em>s</em><sub>3</sub><em>s</em><sub>4</sub><em>s</em><sub>5</sub><em>s</em><sub>6</sub><em>s</em><sub>7</sub><em>s</em><sub>8,</sub>… <sub>,</sub><em>s<sub>N</sub></em><sub>−5</sub><em>s<sub>N</sub></em><sub>−4</sub><em>s<sub>N</sub></em><sub>−3</sub><em>s<sub>N</sub></em><sub>−2</sub><em>s<sub>N</sub></em><sub>−1</sub><em>s<sub>N</sub></em> is a protein-coding sequence, the three forward frames {<em>s</em><sub>1</sub><em>s</em><sub>2</sub><em>s</em><sub>3</sub>, <em>s</em><sub>4</sub><em>s</em><sub>5</sub><em>s</em><sub>6</sub>, <em>s</em><sub>7</sub><em>s</em><sub>8</sub>…}, {<em>s</em><sub>2</sub><em>s</em><sub>3</sub><em>s</em><sub>4</sub>, <em>s</em><sub>5</sub><em>s</em><sub>6</sub><em>s</em><sub>7</sub>, <em>s</em><sub>8</sub>…} and {<em>s</em><sub>3</sub><em>s</em><sub>4</sub><em>s</em><sub>5</sub>, <em>s</em><sub>6</sub><em>s</em><sub>7</sub><em>s</em><sub>8</sub>, …} are denoted by +0, +1 and +2, respectively. For frame +0, we can map it into a plot set <em>ϕ</em>(<em>S</em><sub>+0</sub>) = {<em>ϕ</em>(<em>s</em><sub>1</sub><em>s</em><sub>2</sub><em>s</em><sub>3</sub>) <em>ϕ</em>(<em>s</em><sub>4</sub><em>s</em><sub>5</sub><em>s</em><sub>6</sub>)<sub>,</sub> … <sub>,</sub><em>ϕ</em>(<em>s<sub>n</sub>s<sub>n</sub></em><sub>+1</sub><em>s<sub>n</sub></em><sub>+2</sub>) …}, where <em>ϕ</em>(<em>s<sub>n</sub>s<sub>n</sub></em><sub>+1</sub><em>s<sub>n</sub></em><sub>+2</sub>) = (<em>x<sub>n</sub></em>, <em>y<sub>n</sub></em>, <em>z<sub>n</sub></em>), <em>x<sub>n</sub></em>, <em>y<sub>n</sub></em> and <em>z<sub>n</sub></em> are the initial assignment introduced above, <em>n</em>= 1,4,7, …. Letting<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsr030ueq1.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/m_dsr030ueq1.gif?Expires=2147483647&amp;Signature=N99dmrpjV5hTBkezEK~ufIy3JtY6uX4qco~IrAk5fgJwjlV40MtEQsmXnyL44miTKYmrPPyMHNK7TmHz908BL4NrApkRk00IEcr7jOsKtKtT108lYRbR6Ft3ezrXeGoN1SP6-0arkl9gYkcq1bByNnJybep8KQ0hEQ86j84NHWnxrK6pHYWIJdMYNnjdbsQqymOaVOBi1BFS0sS8rkCWmO~Pi-~iVGfsdmOL3GW6MjuJEY7GOIsyE0bdKfp1vmYfHBtUzfax2QfT9K8XydklzLHbW4RG37iW4ZJ-ay7dzk9k-jlRXITteW4N5fn9HnpCfLoMr-Lt-rNPpuD2x0yJCQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr030ueq1.gif" /></a></div></div></div> to represent the cumulative effects of <em>x<sub>n</sub></em>, <em>y<sub>n</sub></em> and <em>z<sub>n</sub></em> in frame +0, respectively, where <em>i</em> ∈ [1, 2, 3, … , <em>N</em><sup>+0</sup>], <em>N</em><sup>+0</sup> is the total number of trinucleotides in frame +0. We can transform frame +0 into six sets of 2D curves by {<em>x<sub>i</sub></em>, <em>y<sub>i</sub></em>, <em>z<sub>i</sub></em>, <em>x′</em>, <em>y′</em>, <em>z′</em>} vs. <em>i</em>, respectively. Then, six numerical descriptors corresponding to the six geometric centre of each 2D curve can be derived to describe the corresponding reading frame. In this way, we have a 3 × 6 = 18D vector <em>V</em><sub>1</sub>= [<em>v</em><sub>1</sub>, <em>v</em><sub>2</sub>, <em>v</em><sub>3</sub>, … , <em>v</em><sub>18</sub>] as a quantitative descriptor for a complete protein-coding sequence, which corresponds to the geometric centre of each 2D curve of frames +0, +1 and +2, i.e.<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsr030ueq2.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/m_dsr030ueq2.gif?Expires=2147483647&amp;Signature=y-5oUdg6FIlFnljDfTFGn1tLin1aQ4Hr6jLcuYyi8wJTSfYPuKL9~DE9rj5NJJ6Kvci58RXZJ-lYnXbyYcTed8G0~R7Jnjct58TkuP9ousa1C~LrWYjXeZbaa1kkkffA28UpX5u7JzrjfamZVRIvhFn6p-~Mu2-4fqSSrWHj-nzVyLZDaZ42mZiRPkrKVok1rqeLUi5NfIUxh6-cD1epKVYLoLgDf0Rw8oxZD54-3N250YihvIE-K5HbQVaEnmiwGGztW5ujxDVPoNoyMqMSq9pjOlxpzPVNdgyUAWq-q4o~nAxnWy3-qCWybdCL6CCC0cn9Jc8j3MvNGsJ5UYHoMw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr030ueq2.gif" /></a></div></div></div> Where, <em>N</em><sup>{+0}</sup>, <em>N</em><sup>{+1}</sup> and <em>N</em><sup>{+2}</sup> denotes the total number of trinucleotides in the three forward reading frames, respectively. To differentiate the present encoding strategy from the subsequent encoding strategies, an ‘I’ is marked at the top right corner.</div><div class="block-child-p">(ii) Following the similar encoding strategy abovementioned, we can also determine the signs of <em>x</em> and <em>y</em> according to the base at the second codon position, and determine the absolute values of <em>x</em> and <em>y</em> by the bases at the first and third positions, respectively. Still taking GCG as an example, the base ‘C’ at the second position implies that the signs of <em>x</em> and <em>y</em> are both negative. The bases ‘G’ at the first and third positions imply that the absolute values of <em>x</em> and <em>y</em> are 2, respectively. Therefore, GCG is numerically represented by (−2, −2). Following the same steps introduced in (i), we can also derive an 18D vector, which corresponds to the geometric centres of the corresponding 2D curves of the three forward reading frames,<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/dsr030ueq3.gif?Expires=2147483647&Signature=ty3CoZlckn0xlBlysyvN4qp1dGpSQ8-F80qDfke7u4-3PM8I4TsZj-AWkFuSX5Qh22PKr8ts77-pJRNr9p45Wc8EGCp4T7MHGcgdXq~rZiFuz5KTA5harH~FMbvssHvb-m5i5WtpvYWOr8cg2E8GpERhz5DkEBI4dGxZMaLP3hNC5Xkbb-tlu9ph5Vkld~hSwun6F4LBITBY8vdKS5fwZVOEOys-mU1v9rw7o3w6wTumvKwBMVMlcSuqRGqiT8c1jbRNKVC8OA~sNUxXnzDbjng8UNZwgC4Vj2oPLG4s4G8niLaLfgEU4QeVBAKY4S8HjDBjHiAMQLmrFQ7ifoP2eQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsr030ueq3.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/m_dsr030ueq3.gif?Expires=2147483647&Signature=SpFAU~-f46yvLXxGky9O7y0LzQoj2l-KXHzpfJvVpypk7Bub7Pvv055Mge8mbyoIxpIxyjBhvFHUkoMsIJK0tn-Igc-zcqmF253IWwhwZ47AsZhswgFVa-zrYgpu22zd39uV0QOcwn7NR6aJi65IETQREDQeog92pvE4deHPPXSZIH2-J5Rwv0OhqTTXfaLNaFRzc1RmYJsQmZurz7~rEQaSj9QQ9SGD61uw1i~MmzUSZUm8MIMImC4h6oRC5hM3Q3IhjzCoFlWKW8ee-2yaxQtGzK3YNzNtz0ePkF9ubBoCN43mjlln2zCVfCF9Ym9Lv98x1fk0yUbLAPQmdU3R0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr030ueq3.gif" /></a></div></div></div>  where, <em>l</em>= 19, 20, 21, … , 36, Ω ∈ {<em>x</em>, <em>y</em>, <em>z</em>, <em>x′</em>, <em>y′</em>, <em>z′</em>}, <em>j</em>= +0, +1, +2 represents the three forward reading frames, <em>N</em><sup>{<em>j</em>}</sup> is the total number of trinucleotides in each reading frame.</div><div class="block-child-p">(iii) In the third encoding strategy, the signs of <em>x</em> and <em>y</em> are determined by the category of the base at the third codon position, while the absolute values of <em>x</em> and <em>y</em> are decided by the bases at the first and second positions, respectively. Therefore, the trinucleotide GCG can be numerically represented by (−2, 3). Following the similar way in (i) and (ii), we have another 18D vector as numerical descriptors, i.e.<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/dsr030ueq4.gif?Expires=2147483647&Signature=L-h67qIzdpsAmkBkNRkUyvSSbPvP8jnfoOnVBeqhkTB2uja3vnkLWJd5t3~Euzk5eT-SlnpMRsxd7MoXSxm6wI77ICYKF7M3cgNliVv~wtApB1ubmGSuruhMzM4NGaBdsh5Bcykz9ktrWy4Zm873lJDracrRo1xJ8IBnZxXmKfwbX56cUGJy-lB8I~6cHTg82NaT3GqzQK69PlKTKyFsA3-wo74gAKO1IDXHBgaw37967z1HSDNaE1R~ZHFvl~MRdM7dTndmkmONzbzqJ02j~q96P6IrIYNMFq7NKbp-kqcy-OpoOsKJNLHfN~xW6B34hhxwolopkl-uo0og-KG26A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsr030ueq4.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/m_dsr030ueq4.gif?Expires=2147483647&Signature=Z~y7zJhkDM~Ua9YcJpgk5ZTHZugaEDfZyHsgkZq4u0TGLgUAXwhyH2D7AtPaWfwUhh6oWYRk6KJkfIYDb6ZxYjxw3zVWTetfBkUOjBMPfxnNbQwc314jUFP-g9M2NAW0Io-6IUn4AroG-AasVSzC9zqFRw54L0twXdnCJgQaMZtJihTtmP67xhMhY~IdsWzXFOsg-~5XXMqpcEjlbnhspncdGHmXTPs6KZUT1oOWOu2jvTUlWOyN5ow58jlnnrU~RuWpj5DF88tSNEsras0Awsmknnmf~gzonbFaiTyf-1CK1gNMQUogPNW2x0LO6qiZrK4t38TllAor8eD1BPLgOQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr030ueq4.gif" /></a></div></div></div>  where <em>m</em>= 37, 38, 39, … , 54.</div><p>To explain the implications of the derived 54 numerical descriptors, we present three short sequences that have the same trinucleotide compositions but different trinucleotide order as examples in Table <a reveal-id="DSR030TB1" data-open="DSR030TB1" class="link link-reveal link-table xref-fig">1</a>. In our previous work, we have demonstrated that one can obtain intuitive information of trinucleotides both compositions and distributions based on <em>x<sub>i</sub></em>, <em>y<sub>i</sub></em>, <em>z<sub>i</sub></em> and their derivants <em>x′</em>, <em>y′</em>, <em>z′</em>, respectively.<sup><a reveal-id="DSR030C35" data-open="DSR030C35" class="link link-ref link-reveal xref-bibr">35</a></sup> As can be seen from Table <a reveal-id="DSR030TB1" data-open="DSR030TB1" class="link link-reveal link-table xref-fig">1</a>, <em>u</em><sub>1</sub>, <em>u</em><sub>2</sub> and <em>u</em><sub>3</sub> have equal values, while the values of <em>u</em><sub>4</sub>, <em>u</em><sub>5</sub> and <em>u</em><sub>6</sub> differ greatly. This indicates that <em>u</em><sub>1</sub>, <em>u</em><sub>2</sub> and <em>u</em><sub>3</sub> reflect the information of trinucleotide compositions and <em>u</em><sub>4</sub>, <em>u</em><sub>5</sub> and <em>u</em><sub>6</sub> are sensitive to the trinucleotides orders along the corresponding sequences. For other numerical descriptors, the same results can also be observed. Then, we can obtain sufficient information from the two groups numerical descriptors <em>u</em><sub>1</sub>, <em>u</em><sub>2</sub>, <em>u</em><sub>3</sub>, <em>u</em><sub>7</sub>, <em>u</em><sub>8</sub>, <em>u</em><sub>9</sub>, … and <em>u</em><sub>4</sub>, <em>u</em><sub>5</sub>, <em>u</em><sub>6</sub>, <em>u</em><sub>10</sub>, <em>u</em><sub>11</sub>, <em>u</em><sub>12</sub>, … . Recently, we have re-annotated the genome of <em>Amsacta moorei entomopoxvirus</em> using an 18D vector according to encoding strategy III.<sup><a reveal-id="DSR030C36" data-open="DSR030C36" class="link link-ref link-reveal xref-bibr">36</a></sup> In the present work, the number of numerical descriptors is extended to 54. As we discussed above, based on each kind of encoding strategy, the 64 kinds of trinucleotides can be divided into two groups in three ways. That is, we can provide sufficient information with the 54D vector based on the three encoding strategies, which can reveal more universal properties for protein-coding genes than the 18D vector. </p>                        <a id="7207930" scrollto-destination="7207930"></a>
<div content-id="DSR030TB1" class="table-modal"><div class="table-wrap"><div id="DSR030TB1" class="table-wrap-title" data-id="DSR030TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Numerical descriptors for two short sequence (a) ATG CAT TTA, (b) CAT ATG TTA and (c) ATG TTA CAT</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Numerical descriptors </th><th colspan="3">Encoding strategy I<hr /> </th><th rowspan="2">Numerical descriptors </th><th colspan="3">Encoding strategy II<hr /> </th><th rowspan="2">Numerical descriptors </th><th colspan="3">Encoding strategy III<hr /> </th></tr><tr><th>Seq. a </th><th>Seq. b </th><th>Seq. c </th><th>Seq. a </th><th>Seq. b </th><th>Seq. c </th><th>Seq. a </th><th>Seq. b </th><th>Seq. c </th></tr></thead><tbody><tr><td><em>u</em><sub>1</sub> </td><td>7/3 </td><td>7/3 </td><td>7/3 </td><td><em>u</em><sub>19</sub> </td><td>8/3 </td><td>8/3 </td><td>8/3 </td><td><em>u</em><sub>37</sub> </td><td>2 </td><td>2 </td><td>2 </td></tr><tr><td><em>u</em><sub>2</sub> </td><td>−1 </td><td>−1 </td><td>−1 </td><td><em>u</em><sub>20</sub> </td><td>1/3 </td><td>1/3 </td><td>1/3 </td><td><em>u</em><sub>38</sub> </td><td>7/3 </td><td>7/3 </td><td>7/3 </td></tr><tr><td><em>u</em><sub>3</sub> </td><td>8/3 </td><td>8/3 </td><td>8/3 </td><td><em>u</em><sub>21</sub> </td><td>2 </td><td>2 </td><td>2 </td><td><em>u</em><sub>39</sub> </td><td>3 </td><td>3 </td><td>3 </td></tr><tr><td><em>u</em><sub>4</sub> </td><td>14/3 </td><td>3 </td><td>19/3 </td><td><em>u</em><sub>22</sub> </td><td>13/3 </td><td>5 </td><td>14/3 </td><td><em>u</em><sub>40</sub> </td><td>7/3 </td><td>11/3 </td><td>8/3 </td></tr><tr><td><em>u</em><sub>5</sub> </td><td>−1 </td><td>−3 </td><td>0 </td><td><em>u</em><sub>23</sub> </td><td>1/3 </td><td>7/3 </td><td>4/3 </td><td><em>u</em><sub>41</sub> </td><td>14/3 </td><td>3 </td><td>19/3 </td></tr><tr><td><em>u</em><sub>6</sub> </td><td>28/3 </td><td>8 </td><td>20/3 </td><td><em>u</em><sub>24</sub> </td><td>14/3 </td><td>28/3 </td><td>−2/3 </td><td><em>u</em><sub>42</sub> </td><td>−2/3 </td><td>−1/3 </td><td>17/3 </td></tr></tbody></table></div><div content-id="DSR030TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Numerical descriptors </th><th colspan="3">Encoding strategy I<hr /> </th><th rowspan="2">Numerical descriptors </th><th colspan="3">Encoding strategy II<hr /> </th><th rowspan="2">Numerical descriptors </th><th colspan="3">Encoding strategy III<hr /> </th></tr><tr><th>Seq. a </th><th>Seq. b </th><th>Seq. c </th><th>Seq. a </th><th>Seq. b </th><th>Seq. c </th><th>Seq. a </th><th>Seq. b </th><th>Seq. c </th></tr></thead><tbody><tr><td><em>u</em><sub>1</sub> </td><td>7/3 </td><td>7/3 </td><td>7/3 </td><td><em>u</em><sub>19</sub> </td><td>8/3 </td><td>8/3 </td><td>8/3 </td><td><em>u</em><sub>37</sub> </td><td>2 </td><td>2 </td><td>2 </td></tr><tr><td><em>u</em><sub>2</sub> </td><td>−1 </td><td>−1 </td><td>−1 </td><td><em>u</em><sub>20</sub> </td><td>1/3 </td><td>1/3 </td><td>1/3 </td><td><em>u</em><sub>38</sub> </td><td>7/3 </td><td>7/3 </td><td>7/3 </td></tr><tr><td><em>u</em><sub>3</sub> </td><td>8/3 </td><td>8/3 </td><td>8/3 </td><td><em>u</em><sub>21</sub> </td><td>2 </td><td>2 </td><td>2 </td><td><em>u</em><sub>39</sub> </td><td>3 </td><td>3 </td><td>3 </td></tr><tr><td><em>u</em><sub>4</sub> </td><td>14/3 </td><td>3 </td><td>19/3 </td><td><em>u</em><sub>22</sub> </td><td>13/3 </td><td>5 </td><td>14/3 </td><td><em>u</em><sub>40</sub> </td><td>7/3 </td><td>11/3 </td><td>8/3 </td></tr><tr><td><em>u</em><sub>5</sub> </td><td>−1 </td><td>−3 </td><td>0 </td><td><em>u</em><sub>23</sub> </td><td>1/3 </td><td>7/3 </td><td>4/3 </td><td><em>u</em><sub>41</sub> </td><td>14/3 </td><td>3 </td><td>19/3 </td></tr><tr><td><em>u</em><sub>6</sub> </td><td>28/3 </td><td>8 </td><td>20/3 </td><td><em>u</em><sub>24</sub> </td><td>14/3 </td><td>28/3 </td><td>−2/3 </td><td><em>u</em><sub>42</sub> </td><td>−2/3 </td><td>−1/3 </td><td>17/3 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7207930" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR030TB1" class="table-wrap-title" data-id="DSR030TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Numerical descriptors for two short sequence (a) ATG CAT TTA, (b) CAT ATG TTA and (c) ATG TTA CAT</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Numerical descriptors </th><th colspan="3">Encoding strategy I<hr /> </th><th rowspan="2">Numerical descriptors </th><th colspan="3">Encoding strategy II<hr /> </th><th rowspan="2">Numerical descriptors </th><th colspan="3">Encoding strategy III<hr /> </th></tr><tr><th>Seq. a </th><th>Seq. b </th><th>Seq. c </th><th>Seq. a </th><th>Seq. b </th><th>Seq. c </th><th>Seq. a </th><th>Seq. b </th><th>Seq. c </th></tr></thead><tbody><tr><td><em>u</em><sub>1</sub> </td><td>7/3 </td><td>7/3 </td><td>7/3 </td><td><em>u</em><sub>19</sub> </td><td>8/3 </td><td>8/3 </td><td>8/3 </td><td><em>u</em><sub>37</sub> </td><td>2 </td><td>2 </td><td>2 </td></tr><tr><td><em>u</em><sub>2</sub> </td><td>−1 </td><td>−1 </td><td>−1 </td><td><em>u</em><sub>20</sub> </td><td>1/3 </td><td>1/3 </td><td>1/3 </td><td><em>u</em><sub>38</sub> </td><td>7/3 </td><td>7/3 </td><td>7/3 </td></tr><tr><td><em>u</em><sub>3</sub> </td><td>8/3 </td><td>8/3 </td><td>8/3 </td><td><em>u</em><sub>21</sub> </td><td>2 </td><td>2 </td><td>2 </td><td><em>u</em><sub>39</sub> </td><td>3 </td><td>3 </td><td>3 </td></tr><tr><td><em>u</em><sub>4</sub> </td><td>14/3 </td><td>3 </td><td>19/3 </td><td><em>u</em><sub>22</sub> </td><td>13/3 </td><td>5 </td><td>14/3 </td><td><em>u</em><sub>40</sub> </td><td>7/3 </td><td>11/3 </td><td>8/3 </td></tr><tr><td><em>u</em><sub>5</sub> </td><td>−1 </td><td>−3 </td><td>0 </td><td><em>u</em><sub>23</sub> </td><td>1/3 </td><td>7/3 </td><td>4/3 </td><td><em>u</em><sub>41</sub> </td><td>14/3 </td><td>3 </td><td>19/3 </td></tr><tr><td><em>u</em><sub>6</sub> </td><td>28/3 </td><td>8 </td><td>20/3 </td><td><em>u</em><sub>24</sub> </td><td>14/3 </td><td>28/3 </td><td>−2/3 </td><td><em>u</em><sub>42</sub> </td><td>−2/3 </td><td>−1/3 </td><td>17/3 </td></tr></tbody></table></div><div content-id="DSR030TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Numerical descriptors </th><th colspan="3">Encoding strategy I<hr /> </th><th rowspan="2">Numerical descriptors </th><th colspan="3">Encoding strategy II<hr /> </th><th rowspan="2">Numerical descriptors </th><th colspan="3">Encoding strategy III<hr /> </th></tr><tr><th>Seq. a </th><th>Seq. b </th><th>Seq. c </th><th>Seq. a </th><th>Seq. b </th><th>Seq. c </th><th>Seq. a </th><th>Seq. b </th><th>Seq. c </th></tr></thead><tbody><tr><td><em>u</em><sub>1</sub> </td><td>7/3 </td><td>7/3 </td><td>7/3 </td><td><em>u</em><sub>19</sub> </td><td>8/3 </td><td>8/3 </td><td>8/3 </td><td><em>u</em><sub>37</sub> </td><td>2 </td><td>2 </td><td>2 </td></tr><tr><td><em>u</em><sub>2</sub> </td><td>−1 </td><td>−1 </td><td>−1 </td><td><em>u</em><sub>20</sub> </td><td>1/3 </td><td>1/3 </td><td>1/3 </td><td><em>u</em><sub>38</sub> </td><td>7/3 </td><td>7/3 </td><td>7/3 </td></tr><tr><td><em>u</em><sub>3</sub> </td><td>8/3 </td><td>8/3 </td><td>8/3 </td><td><em>u</em><sub>21</sub> </td><td>2 </td><td>2 </td><td>2 </td><td><em>u</em><sub>39</sub> </td><td>3 </td><td>3 </td><td>3 </td></tr><tr><td><em>u</em><sub>4</sub> </td><td>14/3 </td><td>3 </td><td>19/3 </td><td><em>u</em><sub>22</sub> </td><td>13/3 </td><td>5 </td><td>14/3 </td><td><em>u</em><sub>40</sub> </td><td>7/3 </td><td>11/3 </td><td>8/3 </td></tr><tr><td><em>u</em><sub>5</sub> </td><td>−1 </td><td>−3 </td><td>0 </td><td><em>u</em><sub>23</sub> </td><td>1/3 </td><td>7/3 </td><td>4/3 </td><td><em>u</em><sub>41</sub> </td><td>14/3 </td><td>3 </td><td>19/3 </td></tr><tr><td><em>u</em><sub>6</sub> </td><td>28/3 </td><td>8 </td><td>20/3 </td><td><em>u</em><sub>24</sub> </td><td>14/3 </td><td>28/3 </td><td>−2/3 </td><td><em>u</em><sub>42</sub> </td><td>−2/3 </td><td>−1/3 </td><td>17/3 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7207930" target="_blank">View Large</a></div></div><p>The <em>Z</em> curve is another graphical representation proposed for displaying information of individual nucleotides,<sup><a reveal-id="DSR030C37" data-open="DSR030C37" class="link link-ref link-reveal xref-bibr">37</a>,<a reveal-id="DSR030C38" data-open="DSR030C38" class="link link-ref link-reveal xref-bibr">38</a></sup> which has been applied to some protein-coding genes re-annotation works.<sup><a reveal-id="DSR030C15" data-open="DSR030C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSR030C19" data-open="DSR030C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSR030C21" data-open="DSR030C21" class="link link-ref link-reveal xref-bibr">21</a></sup> In this work, the 21 statistical numerical descriptors derived from the Z curve method are adopted to perfect the presented algorithm, which is introduced briefly below, for details refer to the work by Gao and Zhang.<sup><a reveal-id="DSR030C39" data-open="DSR030C39" class="link link-ref link-reveal xref-bibr">39</a></sup></p><p>Supposing <em>a</em><sub>1</sub>, <em>c</em><sub>1</sub>, <em>g</em><sub>1</sub>, <em>t</em><sub>1</sub>; <em>a</em><sub>2</sub>, <em>c</em><sub>2</sub>, <em>g</em><sub>2</sub>, <em>t</em><sub>2</sub>; <em>a</em><sub>3</sub>, <em>c</em><sub>3</sub>, <em>g</em><sub>3</sub>, <em>t</em><sub>3</sub> denote the occurring frequencies of A, C, G and T at different codon positions 1, 4, 7, …; 2, 5, 8, …; 3, 6, 9, … , respectively, then, <em>a<sub>i</sub></em>, <em>c<sub>i</sub></em>, <em>g<sub>i</sub></em>, <em>t<sub>i</sub></em> (<em>i</em> = 1, 2, 3) can be mapped onto a point <em>P<sub>i</sub></em> in a 3D space, the coordinates of which are calculated by <em>Z</em>-transform</p><div class="block-child-p"><span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/m_dsr030ileq1.gif?Expires=2147483647&amp;Signature=J1FjQOlahb8f1ifWqVxjz~ZiyKRMIAzBOCwfFl5zabMuMg-vyR1bXhkXMYHOlEB2ut3PhrTJoSO5Blg8ZEYkVSOD2C3Jgwgu7dgEQjGetDTaQyb993g5KYRi-axMZxx975fMjzYtqAa-PeDvBHAG-1IBXCBNzIeTwNyYUm904H8kxI1SxB0VcRj3qlcHeKh5wuwQJzbShCH-qSuAzINk6XVjV1Ij8AH~D09vV1Oq2uFyFB7pr7JGjbX6dvUmSV9Uh4tgDmXmcFkBuyRzTRqZlVz3cXNQWxrzd5PMSAjCtdKzLTAFSp0EcaYpTHKCDwu3H-4A2zxK2mGiHvswabX~vw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr030ileq1.gif" /></span>Obviously, <em>x<sub>i</sub></em>, <em>y<sub>i</sub></em> and <em>z<sub>i</sub></em> display the statistical features of the base compositions at different codon positions, then nine numerical descriptors can be obtained,<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsr030ueq5.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/m_dsr030ueq5.gif?Expires=2147483647&amp;Signature=hQgFJgGkJPb8giupS9rM3vLgtuHuocqv8kOi7nNek9W9CHg6WXZ2UBrASupfEOhj-mG06x1LNbFI1cTilGe6Tc99u0BIxIuvkPJZnfgwuqGWNNAQrtD0Tj3ldF-PCUG37Rbt1imGo-PFQJSsXuJNwtCgJmiI8jY0JfjcqleUuHuX93w8x0dVngcfIR-Mlzx4qBtVQMTS4l2WawSk1hH6iCU8KE~Rm~Otv3gLrPcGQ9CoUPGkHEpTdGJin~Vxt5ZKwJQGXCpxt9tj6M4Vd7A2gdjvahfmzwE00ZAtm1LGBCuUnvFpuFsUqc2LgfVWXH35949Vl9URtxY4nkFmPUmDcg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr030ueq5.gif" /></a></div></div></div> In addition to the nine codon position-dependent parameters, 12 phase-specific dinucleotides were also considered. Let the occurring frequencies of the 16 dinucleotides AA, AC, … , and TT be denoted by <em>p</em>(AA), <em>p</em>(AC), … , <em>p</em>(TT), respectively. Using the <em>Z</em>-transform,<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsr030ueq6.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/m_dsr030ueq6.gif?Expires=2147483647&amp;Signature=WxPPa~A3KfjKeIeEbGJ3A2uzwG7KmhcA9Vr1oCbVoW65Dae-no33LYVByTP2Y-6fFaw1uw1evWCGA7HzpeqZrbQ1mgna7B7uC7PcAafXsT240roEHOuOCMgw0tlsk9T80CB2fbKX33Expsr6H8wLP~2HA1kLRvcENdsEyaaubz923aJiY~0erpZjxUqmbbacxHRPh8QgdNI9uDOldf0drhV95Fvs3YPcTeQGabWpAMK8jmCJ4fsL-QD7VAMWOeofZO1Y6XvcnSQsJA55UV2PoyIbDq-GHY2xgf6l9mDD9dKOxTYJQf3o2Sb7pDqUI3WYpUk0GAPIYXOmOgDrtaiVog__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr030ueq6.gif" /></a></div></div></div> where <em>X</em> = A, C, G and T. Then, an additional 12D vector can be obtained, which is written as follows,<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsr030ueq7.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/m_dsr030ueq7.gif?Expires=2147483647&amp;Signature=sXd-DeHTPNqTOx9-fCJyF5P8Cgmcj2ej1M55aAu1eGWnrSAgx8qRiJnvObKQQT9SDU~z3B~0fh2pCRvPVZkr0gjm9iP209MsWhjaX0CELxcbeaWhbWhmtVLOEDgq3EDTOFEubyWtU5atY3tY6onGKeCm~JDEWaSaUH7HhClc41zh5HiPgPFJz4j7w77KlyVXj3z4ASmsTOwuqV2aK0dwzMPlFTGqm9w3MkFBLXkCNWja-3psleUegFPfSlBZKy2qWlgRmi6VHtHQsvzD9J75ur-rBRjv~qwEPbEHF1X2~9KSJJSc6B-ewF3STbxuFJUpyxgKuxtfKQR3geHQQX7U6A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr030ueq7.gif" /></a></div></div></div> Comparing with the 54D vector based on the TN curve, the 21 numerical descriptors provide statistical information of the base compositions at different codon positions and adjacent nucleotides. The critical differences between protein-coding genes and non-coding sequences exist, in that the former has regularly specific features such as asymmetric nucleotide distributions at the three codon positions and codon usage bias, while the latter does not. Then, how to propose sufficient numerical descriptors to exhibit the specific features of protein-coding genes is the core for gene prediction programs. Previous research showed that the first and second bases determine the category of translated amino acid, while the third base is associated with a synonymous codon.<sup><a reveal-id="DSR030C40" data-open="DSR030C40" class="link link-ref link-reveal xref-bibr">40</a>,<a reveal-id="DSR030C41" data-open="DSR030C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Because of the uneven distribution of synonymous codons, protein-coding genes are different from non-coding sequences in gene structure, which can be used to find protein-coding sequences.<sup><a reveal-id="DSR030C42" data-open="DSR030C42" class="link link-ref link-reveal xref-bibr">42</a></sup> In this paper, the outlined 75-component vector can be represented by the direct combinations of the subspaces, i.e. <em>V</em>Þ{<em>V</em><sup>I</sup>, <em>V</em><sup>II</sup>, <em>V</em><sup>III</sup>, <em>V</em><sup>Z</sup>}, the former three items reflect the compositions and distributions of trinucleotides along the DNA sequences and the latter item provides statistical significances of protein-coding genes. Therefore, the two groups of parameters can complement each other, which provide sufficient information for protein-coding genes from different angles.</div>                            <h3 scrollto-destination=7207934 id="7207934" class="section-title" data-legacy-id=s2b>2.2. The Fisher discriminant algorithm</h3>
<p>The Fisher discriminant algorithm is a simple method that has been extensively used in gene prediction. For detail introductions, refer to the work by Zhang and Wang.<sup><a reveal-id="DSR030C13" data-open="DSR030C13" class="link link-ref link-reveal xref-bibr">13</a></sup> To accomplish the presented algorithm, two sets of samples are required to train the discriminant coefficients, i.e. positive samples corresponding to true protein-coding genes and negative samples corresponding to non-coding ORFs. For each queried genome, its annotated known functional genes are used as the positive training set and the negative training set is composed of the shuffled complementary sequences of corresponding known functional genes, which has been initially introduced by Guo <em>et al</em>.<sup><a reveal-id="DSR030C32" data-open="DSR030C32" class="link link-ref link-reveal xref-bibr">32</a></sup> It was found that the 75 parameters are sensitive to the specific gene structures. Then we shuffle the primary sequences only 50 times, hence the runtime can be shorten remarkably. The Fisher linear equation for discriminating the positive and negative samples in the 75D space <em>V</em> represents a super-plane, described by a vector <em>C</em> that has 75 components. To avoid loss of generality, the vector <em>C</em> was determined according to the criterion |<em>C</em>|<sup>2</sup>= 1. Besides, an appropriate threshold <em>C</em><sub>0</sub> is obtained by strictly letting the false-negative rate and the false-positive rate to be identical. Once the vector <em>C</em> and the threshold <em>C</em><sub>0</sub> are determined, each sequence is assigned a <em>T</em>_score= <em>CV</em>− <em>C</em><sub>0</sub>. Then the decision of coding/non-coding for each genes in the test set is simply performed by the criterion of <em>T</em>_score &gt; 0 or <em>T</em>_score &lt; 0, where <em>C</em>= (<em>C</em><sub>1</sub>, <em>C</em><sub>2</sub>, … , <em>C</em><sub>75</sub>) and <em>V</em>= (<em>u</em><sub>1</sub>, <em>u</em><sub>2</sub> … , <em>u</em><sub>75</sub>).</p>                            <h3 scrollto-destination=7207936 id="7207936" class="section-title" data-legacy-id=s2c>2.3. Evaluation index</h3>
<div class="block-child-p">The accuracy (<em>A</em><sub>c</sub>), sensitivity (<em>s</em><sub>n</sub>) and specificity (<em>s</em><sub>p</sub>) proposed by Burset and Guigo<sup><a reveal-id="DSR030C43" data-open="DSR030C43" class="link link-ref link-reveal xref-bibr">43</a></sup> are used to evaluate the performance of the presented method<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/dsr030ueq8.gif?Expires=2147483647&Signature=HvX8cAjFcvR~QMwasDsRuzc5tM5mkA3nb3Zg22D6B3AbOc7gsDAvcwC-fw73Fq-tXXWEGTwobEy-P7WgF5tmVWR6s0SgBqRAnrr6QYK4kbUQQyIwOe-I-hnLYDZOcMXn~EjvgdMP-DrwxvdwHRSwwk63SWs7EfyHmiPoQrdVEsalN-vNFVJl75bOV~Fn6rvVKyJULBljn2f50TtJtja-h7QRwLj69~kbjQO197XCdY4XNOjdf6msy8TPa20VXpthq5yCSNgnr8KphYFkeSI6otXbhIk4RpQ2WJkerEYbAVx11XHu8AEFpELXQYWUvn7J0chSqn~DOqkObyfLH5Szkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsr030ueq8.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/m_dsr030ueq8.gif?Expires=2147483647&Signature=WRRypQ6c5TvWneiZ~~FE4RCcXRsg4GduF1rgYWvkMgI8gdkD7EucpzVvZABaIHBKjpdRv~PT3avwQitULbnM-0vI2krS3MI9UhGYe0PVWMT0uNmrhzDpUYxRz5VqxBwtKDzT~xTAnwh-LJfaowh3cfTk5nsIPKDjrect74MPR2kMiH4O1EO-z4M~4vlDictFX~SAJ6ixYpDi0eJ~gU5zWOl0v4CIw7MPRGepq1i~~1XqGBb1b0wU3EP-UE3F6fGnUS7DlAYYYzdo0b-F8E-WlaeQlRRkUZDpGNWsRhdb0KOyZo9Ry~8tueuLg2lbLz6elmMy8cI3NENAIkwyBBZv7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr030ueq8.gif" /></a></div></div></div> Where, TP and FN denote the number of coding ORFs that have been predicted as coding and non-coding sequences, respectively. Then, <em>s</em><sub>n</sub> is the proportion of the coding ORFs that have been predicted correctly as coding sequences. Similarly, TN and FP denote the number of non-coding sequences that have been predicted as non-coding and coding sequences, respectively. Then, <em>s</em><sub>p</sub> is the proportion of the non-coding sequences that have been correctly predicted as non-coding.</div><div class="block-child-p">The Matthew's correlation coefficient (MCC) is also used to describe the agreement of predictions and annotation with a single value in the range of [−1, 1], where,<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/dsr030ueq9.gif?Expires=2147483647&Signature=4ccs~fj9-fCHo1AZT6gA-kWBZvhfmF6ZGy0MwK-cdGWEuDjzIrIwDDv~HkhektB9hw1JQrKRSKhA1XpkEHzko~k80BFWfIPx1-gvMKJ-~7Z8HTl9HOXmZy-Pi2tXqaJSLtWjOl9gpoPHi~VqQBthmPdNNJmhUjzmO-3YAFj1Ui~bHoMG7AcpOH5W3rN-bKURSNsSK37rXbYUQbz-sfbcNO~h7Cd3HAD~87kf9LCBw7jmrdciTiKk7F1nKiaNkk7pXT9X2PtaBNZ5IH-SgTIG6hJomZfK2cBMaOyuna997kwkulkBC56tEEVw~c6K0XWnGjFdUoW7r7Yp8cPu~gA8Mw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsr030ueq9.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/m_dsr030ueq9.gif?Expires=2147483647&Signature=N9LyUZLtpSv13ril6QxepOUpyPu9f-ysefwLw4lerUTeYLIYAOI-iNPrz1vu36hOk-JvrNxXINWmgMNf0wKxktDfcba95UWbcSabFrHk7DIIEi2feUpBGe7UWlAUuMnVignL227U7S0iBuDZAWDsKgksJiGynlteksM23zDO-OkQW-vaz6S6rWBVXxZQLcu7XWNf8AUDfLloUfo2EjcfKxHyDcL5Zo2L0MQ0A8~fRMtfITtqOSULMW0cFVDDwYkDHNhJP~qW1BacmMYpR6vouv8Rr3xrK6yThtjYDy0nIbP0kj6reKXZ6hEXF-OnZu9xdlWm3fWUeN34K66q1QCq0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr030ueq9.gif" /></a></div></div></div></div>                            <h2 scrollto-destination=7207939 id="7207939" class="section-title" data-legacy-id=s3>3. Results and discussions</h2>
                            <h3 scrollto-destination=7207940 id="7207940" class="section-title" data-legacy-id=s3a>3.1. Performance of the presented algorithm</h3>
<p>According to the annotation files, the protein-coding genes in public databases can be classified into two groups, i.e. the genes with known function and those marked with different prefixes, such as ‘putative’, ‘probable’, ‘possible’, ‘possibly’, ‘similar’, ‘alternate’, ‘uncharacterized’, ‘unknown’, ‘predicted’, ‘conserved hypothetical’ and ‘hypothetical’ genes. In this study, we divide all the annotated protein-coding genes into four classes. The first class includes the known functional genes, while those marked ‘conserved hypothetical’ and ‘hypothetical’ genes are assigned to the third and fourth classes, respectively, the rest marked ‘putative’, ‘probable’, ‘possible’, ‘possibly’, ‘similar’, ‘alternate’, ‘uncharacterized’, ‘unknown’, and ‘predicted’ belong to the second class. It is noted that the genes in the first class are genuine protein-coding with validated functions, while some ORFs in other classes may be random sequences that are falsely predicted as protein-coding genes. In the presented re-annotating algorithm, the known functional genes in the first class and their corresponding shuffled sequences are used to train the Fisher coefficients and evaluate the predicting performance, this can also regarded as self-test. In addition, the 10-fold cross-validation is also employed to evaluate our algorithm. In the 10-fold cross-validation, the positive and negative samples composed of the known functional genes and their shuffled sequences are randomly divided into three groups averagely, one is used as the training set and the others are used as the testing set. Here, 61 bacterial and archaeal genomes (listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>) are taken as examples to accomplish the algorithm. For convenience, the abbreviation names are used, for example, <em>Candidatus Phytoplasma mali</em> is abbreviated as <em>C. Phytoplasma</em>. The genome size of the 61 species ranges from 412 348 to 7 036 071 bp, and the G + C content ranges from 21 to 72%. In <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>, we present the sensitivity (s<sub>n</sub>), specificity (s<sub>p</sub>), accuracy (A<sub>c</sub>) and the MCC of self-test and the 10-fold cross-validation. Since the proposed algorithm is a supervised learning strategy, which may be impacted by the shuffling negative samples, then the program is performed five times over each genome, and the mean evaluating indices are calculated. Because the threshold <em>C</em><sub>0</sub> is strictly demined by the false-positive rate equal the false-negative rate, then, for self-test, s<sub>n</sub> is identical to s<sub>p</sub>, and <em>A</em><sub>c</sub>= <em>s</em><sub>n</sub> = <em>s</em><sub>p</sub>. In addition, some ORFs with lengths that cannot be divided by three integrally are excluded. For comparison, the results by 54- and 21-vectors are also presented in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>. As can be seen, the overall average accuracies of self-test over the 61 microbial genomes are 99.62, 99.79 and 99.94% for the 54, 21 and 75D vectors, respectively. As for the 10-fold cross-validation, overall average accuracies of 99.55, 99.75 and 99.88% are obtained by 54, 21 and 75D vectors, respectively. The overall average MCCs over the 61 microbial genomes for the self-test are 0.9925, 0.9958 and 0.9989, for the 10-fold cross-validation, 0.9910, 0.9951 and 0.9977 obtained by 54, 21 and 75D vectors, respectively. Although the integrated algorithm achieves an extremely high performance, the parameters also increased. Then the main question remains open—why can this so-called combined methodology achieve such higher accuracy? Can we obtain more information about the specific protein-coding genes, in other words, can the integrated 75D vector grasp the universal features of protein-coding genes different from non-coding? To give explicit answer, we should perform sufficient analysis in the following sections.</p>                            <h3 scrollto-destination=7207942 id="7207942" class="section-title" data-legacy-id=s3b>3.2. Correlation between the G + C content and the Fisher coefficients</h3>
<p>The ratio of G + C to the total bases appears to be constant in particular microbial genomes, but varies between species, which is found somehow related to phylogeny, as well as the genomic components, such as protein-coding genes, stable RNA genes and spacers including various signals.<sup><a reveal-id="DSR030C44" data-open="DSR030C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Muto and Osawa's<sup><a reveal-id="DSR030C45" data-open="DSR030C45" class="link link-ref link-reveal xref-bibr">45</a></sup> researches indicated that the G + C content of each kind of genomic component positively but differentially correlate to the genomic G + C content for a given bacterium. When the genetic code was deciphered in the early 1960s, it was observed to be universal for most organisms, whereas there are no universal gene-finding parameters suitable for any organism, which can be reflected that the 61 sets of Fisher coefficients are dissimilar from each other. The 75 numerical descriptors in our algorithm are proposed to demonstrate the general features of protein-coding genes, and then it is interesting to explore the correlation between the genomic G + C content and the trained Fisher coefficients among different genomes.</p><p>To accomplish the analysis, the following steps are proposed. (i) Six species with different G + C contents, <em>Buchnera</em> (26%), <em>Streptococcus uberis</em> (36%), <em>Yersinia pestis</em> (47%), <em>Brucella melitensis</em> (57%), <em>Pseudomonas aeruginosa</em> (66%) and <em>Clavibacter michiganensis</em> (72%) are selected discretionarily. (ii) Based on the six genomes, six sets of the Fisher coefficients are obtained, each of which is trained by its known functional genes. (iii) Each set of the Fisher coefficients is singled out in turn to identify the known functional genes in other genomes. For convenience, we list the discriminating results in Table <a reveal-id="DSR030TB2" data-open="DSR030TB2" class="link link-reveal link-table xref-fig">2</a>. </p>                        <a id="7207945" scrollto-destination="7207945"></a>
<div content-id="DSR030TB2" class="table-modal"><div class="table-wrap"><div id="DSR030TB2" class="table-wrap-title" data-id="DSR030TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Accuracies of mutual validations for the genomes with different G + C content based on the 75D vector</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th><em>Buchnera</em> (%) </th><th><em>S. uberis</em> (%) </th><th><em>Y. pestis</em> (%) </th><th><em>B. melitensis</em> (%) </th><th><em>P. aeruginosa</em> (%) </th><th><em>C. michiganensis</em> (%) </th></tr></thead><tbody><tr><td><em>Buchnera</em> </td><td>100 </td><td>88.79 </td><td>50.25 </td><td>9.01 </td><td>0.97 </td><td>0 </td></tr><tr><td><em>S. uberis</em> </td><td>99.79 </td><td>99.87 </td><td>87.11 </td><td>60.66 </td><td>12.84 </td><td>2.15 </td></tr><tr><td><em>Y. pestis</em> </td><td>99.38 </td><td>99.53 </td><td>99.85 </td><td>99.47 </td><td>99.32 </td><td>96.97 </td></tr><tr><td><em>B. melitensis</em> </td><td>91.99 </td><td>96.86 </td><td>96.59 </td><td>100 </td><td>99.46 </td><td>99.24 </td></tr><tr><td><em>P. aeruginosa</em> </td><td>98.36 </td><td>99.13 </td><td>99.75 </td><td>100 </td><td>99.78 </td><td>100 </td></tr><tr><td><em>C. michiganensis</em> </td><td>2.67 </td><td>13.35 </td><td>70.06 </td><td>99.34 </td><td>99.18 </td><td>100 </td></tr></tbody></table></div><div content-id="DSR030TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th><em>Buchnera</em> (%) </th><th><em>S. uberis</em> (%) </th><th><em>Y. pestis</em> (%) </th><th><em>B. melitensis</em> (%) </th><th><em>P. aeruginosa</em> (%) </th><th><em>C. michiganensis</em> (%) </th></tr></thead><tbody><tr><td><em>Buchnera</em> </td><td>100 </td><td>88.79 </td><td>50.25 </td><td>9.01 </td><td>0.97 </td><td>0 </td></tr><tr><td><em>S. uberis</em> </td><td>99.79 </td><td>99.87 </td><td>87.11 </td><td>60.66 </td><td>12.84 </td><td>2.15 </td></tr><tr><td><em>Y. pestis</em> </td><td>99.38 </td><td>99.53 </td><td>99.85 </td><td>99.47 </td><td>99.32 </td><td>96.97 </td></tr><tr><td><em>B. melitensis</em> </td><td>91.99 </td><td>96.86 </td><td>96.59 </td><td>100 </td><td>99.46 </td><td>99.24 </td></tr><tr><td><em>P. aeruginosa</em> </td><td>98.36 </td><td>99.13 </td><td>99.75 </td><td>100 </td><td>99.78 </td><td>100 </td></tr><tr><td><em>C. michiganensis</em> </td><td>2.67 </td><td>13.35 </td><td>70.06 </td><td>99.34 </td><td>99.18 </td><td>100 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7207945" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR030TB2" class="table-wrap-title" data-id="DSR030TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Accuracies of mutual validations for the genomes with different G + C content based on the 75D vector</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th><em>Buchnera</em> (%) </th><th><em>S. uberis</em> (%) </th><th><em>Y. pestis</em> (%) </th><th><em>B. melitensis</em> (%) </th><th><em>P. aeruginosa</em> (%) </th><th><em>C. michiganensis</em> (%) </th></tr></thead><tbody><tr><td><em>Buchnera</em> </td><td>100 </td><td>88.79 </td><td>50.25 </td><td>9.01 </td><td>0.97 </td><td>0 </td></tr><tr><td><em>S. uberis</em> </td><td>99.79 </td><td>99.87 </td><td>87.11 </td><td>60.66 </td><td>12.84 </td><td>2.15 </td></tr><tr><td><em>Y. pestis</em> </td><td>99.38 </td><td>99.53 </td><td>99.85 </td><td>99.47 </td><td>99.32 </td><td>96.97 </td></tr><tr><td><em>B. melitensis</em> </td><td>91.99 </td><td>96.86 </td><td>96.59 </td><td>100 </td><td>99.46 </td><td>99.24 </td></tr><tr><td><em>P. aeruginosa</em> </td><td>98.36 </td><td>99.13 </td><td>99.75 </td><td>100 </td><td>99.78 </td><td>100 </td></tr><tr><td><em>C. michiganensis</em> </td><td>2.67 </td><td>13.35 </td><td>70.06 </td><td>99.34 </td><td>99.18 </td><td>100 </td></tr></tbody></table></div><div content-id="DSR030TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th><em>Buchnera</em> (%) </th><th><em>S. uberis</em> (%) </th><th><em>Y. pestis</em> (%) </th><th><em>B. melitensis</em> (%) </th><th><em>P. aeruginosa</em> (%) </th><th><em>C. michiganensis</em> (%) </th></tr></thead><tbody><tr><td><em>Buchnera</em> </td><td>100 </td><td>88.79 </td><td>50.25 </td><td>9.01 </td><td>0.97 </td><td>0 </td></tr><tr><td><em>S. uberis</em> </td><td>99.79 </td><td>99.87 </td><td>87.11 </td><td>60.66 </td><td>12.84 </td><td>2.15 </td></tr><tr><td><em>Y. pestis</em> </td><td>99.38 </td><td>99.53 </td><td>99.85 </td><td>99.47 </td><td>99.32 </td><td>96.97 </td></tr><tr><td><em>B. melitensis</em> </td><td>91.99 </td><td>96.86 </td><td>96.59 </td><td>100 </td><td>99.46 </td><td>99.24 </td></tr><tr><td><em>P. aeruginosa</em> </td><td>98.36 </td><td>99.13 </td><td>99.75 </td><td>100 </td><td>99.78 </td><td>100 </td></tr><tr><td><em>C. michiganensis</em> </td><td>2.67 </td><td>13.35 </td><td>70.06 </td><td>99.34 </td><td>99.18 </td><td>100 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7207945" target="_blank">View Large</a></div></div><p>As can be seen from Table <a reveal-id="DSR030TB2" data-open="DSR030TB2" class="link link-reveal link-table xref-fig">2</a>, 88.79% known functional genes of <em>S. uberis</em> genome can be correctly identified using the Fisher coefficients and threshold trained in <em>Buchnera</em>. With the increase in the G + C content, the discriminating accuracy obtained by the Fisher coefficients of <em>Buchnera</em> drops quickly. Observing the results obtained by the Fisher coefficients of <em>C. michiganensis</em>, it was found that 99.18% known functional genes of <em>P. aeruginosa</em> are correctly identified, with the decrease in the G + C content, the accuracy drops to an extremely low level (2.67% for <em>Buchnera</em>). As for the results obtained by <em>S. uberis</em>, it was found that 99.79 and 87.11% known functional genes of <em>Buchnera</em> and <em>Y. pestis</em> can be correctly identified, respectively, but the accuracy for genomes with significantly different G + C content is much lower. The similar phenomenon that high accuracies can be achieved among genomes with similar G + C contents can be observed in other species. To give an exhaustive interpretation, we also perform the similar steps based on the 54 and 21D vectors (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S3</a> and <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/lookup/suppl/doi:10.1093/dnares/dsr030/-/DC1" target="_blank">S4</a>), respectively. Meaningfully, the similar phenomenon to the above analysis can be found in both tables. In some sense, the genomic G + C content can reflect the phylogenetic relationship among bacteria and archaea,<sup><a reveal-id="DSR030C45" data-open="DSR030C45" class="link link-ref link-reveal xref-bibr">45</a>,<a reveal-id="DSR030C46" data-open="DSR030C46" class="link link-ref link-reveal xref-bibr">46</a></sup> and it was suggested that the divergence of genomic G + C content of various bacterial species in one genus is &lt;10%.<sup><a reveal-id="DSR030C19" data-open="DSR030C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Therefore, we infer that genomes with similar G + C content may share similar gene-finding parameters.</p><p>Among the six researched genomes, <em>Y. pestis</em> has a medium G + C content. Using the Fisher coefficients and threshold of <em>Y. pestis</em>, over 97% known functional genes of the other five genomes can be correctly identified. The similar results can also be observed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S3</a> and <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/lookup/suppl/doi:10.1093/dnares/dsr030/-/DC1" target="_blank">S4</a>, which imply that the Fisher coefficients and thresholds trained by genomes with medium G + C content (∼50%) are much universal, this is consistent with previous works by Chen <em>et al</em>.<sup><a reveal-id="DSR030C19" data-open="DSR030C19" class="link link-ref link-reveal xref-bibr">19</a></sup> In Table <a reveal-id="DSR030TB2" data-open="DSR030TB2" class="link link-reveal link-table xref-fig">2</a>, the results obtained by <em>P. aeruginosa</em> are much conspicuous, for this genome has a high G + C content of 66%. As can be seen, much high accuracies over the other five tested genomes are achieved (≥98%) by this genome. By contrast, the results listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S3</a> and <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/lookup/suppl/doi:10.1093/dnares/dsr030/-/DC1" target="_blank">S4</a> for <em>P. aeruginosa</em> is found to be much different in some cases. In <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>, only 21.36% known functional genes of <em>Buchnera</em> are correctly predicted by the coefficients of <em>P. aeruginosa</em>, while in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>, the accuracy is 95.69%. Similarly, the accuracy on <em>S. uberis</em> based on the 54D vector is also lower than that of 21D vector. Then, we divide the 75 trained Fisher coefficients into three subcomponents, i.e. <em>W</em><sub>1</sub>= [<em>C</em><sub>1</sub>–<em>C</em><sub>3</sub>, <em>C</em><sub>7</sub>–<em>C</em><sub>9</sub>, <em>C</em><sub>13</sub>–<em>C</em><sub>15</sub>, <em>C</em><sub>19</sub>–<em>C</em><sub>21</sub>, <em>C</em><sub>25</sub>–<em>C</em><sub>27</sub>, <em>C</em><sub>31</sub>–<em>C</em><sub>33</sub>, <em>C</em><sub>37</sub>–<em>C</em><sub>39</sub>, <em>C</em><sub>43</sub>–<em>C</em><sub>45</sub>, <em>C</em><sub>49</sub>–<em>C</em><sub>51</sub>], <em>W</em><sub>2</sub>= [<em>C</em><sub>4</sub>–<em>C</em><sub>6</sub>, <em>C</em><sub>10</sub>–<em>C</em><sub>12</sub>, <em>C</em><sub>16</sub>–<em>C</em><sub>18</sub>, <em>C</em><sub>22</sub>–<em>C</em><sub>24</sub>, <em>C</em><sub>28</sub>–<em>C</em><sub>30</sub>, <em>C</em><sub>34</sub>–<em>C</em><sub>36</sub>, <em>C</em><sub>40</sub>–<em>C</em><sub>42</sub>, <em>C</em><sub>46</sub>–<em>C</em><sub>48</sub>, <em>C</em><sub>52</sub>–<em>C</em><sub>54</sub>], <em>W</em><sub>3</sub>= [<em>C</em><sub>55</sub>–<em>C</em><sub>75</sub>]. According to the definitions, <em>W</em><sub>1</sub> and <em>W</em><sub>2</sub> correspond to information of trinucleotides and their corresponding cumulative effects, and <em>W</em><sub>3</sub> represents the information of base compositions and dinucleotides, respectively. For comparison, the three subcomponents for <em>P. aeruginosa</em> and <em>Buchnera</em> are intuitively plotted in Fig. <a reveal-id="DSR030F1" data-open="DSR030F1" class="link link-reveal link-table xref-fig">1</a>. Observing the distribution of <em>W</em><sub>3</sub>, it seems that <em>P. aeruginosa</em> and <em>Buchnera</em> have similar patterns in the regions of 10–21, while in the regions of 1–9, the patterns of the two genomes seem to be much different. <em>W</em><sub>3</sub> represents the Fisher coefficients that corresponding to the 21 statistical parameters, which reflect the statistical properties of base distributions in the three codon positions and phase-specific dinucleotides. The G + C content of <em>P. aeruginosa</em> is much different from that of <em>Buchnera</em>, which can account for the differences in regions 1–9 that correspond to base distributions at the three codon positions. While in regions 10–21, the similar tendency imply that there are some universal properties on the phase-specific dinucleotides in <em>P. aeruginosa</em> genome that accounts for the high accuracy on other genomes as shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>, which needs further experimental validations. <em>W</em><sub>1</sub> and <em>W</em><sub>2</sub> represent the Fisher coefficients corresponding to the 54D vector that derived from the trinucleotide-based walking model, which reflects the distribution and compositions of trinucleotides along the DNA sequences. It was found that codon usages in protein-coding genes are highly dependent on the G + C content, then the remarkable differences in the G + C content between <em>P. aeruginosa</em> and <em>Buchnera</em> cause the low accuracy in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>. From the bar graphs in Fig. <a reveal-id="DSR030F1" data-open="DSR030F1" class="link link-reveal link-table xref-fig">1</a>, the accordant patterns of <em>W</em><sub>1</sub> and <em>W</em><sub>2</sub> can be observed, for both exhibit much different tendency between the two species. Then, the results of Fig. <a reveal-id="DSR030F1" data-open="DSR030F1" class="link link-reveal link-table xref-fig">1</a> indicate that the 75 integrative parameters in the presented re-annotating algorithm can demonstrate more underlying information of protein-coding genes. </p>                        <a id="7207948" scrollto-destination="7207948"></a>
<div id="DSR030F1" data-id="DSR030F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/m_dsr03001.jpeg?Expires=2147483647&Signature=GkUcq0WYqDGA7tAHw9L33-86Ociwm4IjF4S9g1Op1nw9kOpw4yZdd3nyc4TDxiwSZBxZzMCRfXG4rgph-bp8ZbC1T0r6V0E3eNyA78jgNcQsI6iQq4xOHxghKNaYuPcqgaTMD27ySmOQztrzcS-V0LTJOWknzuxRyIhRNQxpV1~QH52B653gZyrkaO83EjtPc2I6TRPMtuZK-mY9YAlf3nEAmS2TBK330lel4L15WgDIyy28sUWs45aRNZTkLn4a2aCV-Z46yjFoAzlJ10C1iedtzfNO8dEASOGCXDizNXGOJrLvh7bqHQxr0jLVmg8tV47rivQJhBaDN2OmDmWVcA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparing the Fisher coefficients (C) between P. aeruginosa and Buchnera." path-from-xml="dsr03001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7207948/dsr03001.jpeg" path-from-xml="dsr03001.jpeg" target="_blank">View large</a><a section="7207948" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/dsr03001.jpeg?Expires=2147483647&Signature=1t-uzgGXrEVErCEGmY-UTvFpcEajiuhy-DsXfrnvD4CbRqa~vZ2cS1ED9Zyp2l9XR0tdWetqC5rOxNEVbXTUEY5e6C0uw~0--jxhimE7UZRLN5A715~gxk1c7nAnzYm36sSvvvucB8OCg8BGYgMZnodaEfe~Vxx0GSW5mXdyYCFcdGIUGHNfCBeMZ-pDfiRaQ1qCY36Y-uv83ZWT22wPSoTPJbPG34sWg4z4wtiitpMy6xXqcx6Aa34egNLsE2pEbgfmNg~Ac5o1elEOZj4VNgoLGYyLLZZxSc1YVdApHOgRCsUd3Kks1-lyE-STxcapXTnegloZ79DG3DUX3K8qow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7207948&ar=477266&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparing the Fisher coefficients (<em>C</em>) between <em>P. aeruginosa</em> and <em>Buchnera</em>.</p></div></div><div content-id="DSR030F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/m_dsr03001.jpeg?Expires=2147483647&Signature=GkUcq0WYqDGA7tAHw9L33-86Ociwm4IjF4S9g1Op1nw9kOpw4yZdd3nyc4TDxiwSZBxZzMCRfXG4rgph-bp8ZbC1T0r6V0E3eNyA78jgNcQsI6iQq4xOHxghKNaYuPcqgaTMD27ySmOQztrzcS-V0LTJOWknzuxRyIhRNQxpV1~QH52B653gZyrkaO83EjtPc2I6TRPMtuZK-mY9YAlf3nEAmS2TBK330lel4L15WgDIyy28sUWs45aRNZTkLn4a2aCV-Z46yjFoAzlJ10C1iedtzfNO8dEASOGCXDizNXGOJrLvh7bqHQxr0jLVmg8tV47rivQJhBaDN2OmDmWVcA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparing the Fisher coefficients (C) between P. aeruginosa and Buchnera." path-from-xml="dsr03001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7207948/dsr03001.jpeg" path-from-xml="dsr03001.jpeg" target="_blank">View large</a><a section="7207948" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/dsr03001.jpeg?Expires=2147483647&Signature=1t-uzgGXrEVErCEGmY-UTvFpcEajiuhy-DsXfrnvD4CbRqa~vZ2cS1ED9Zyp2l9XR0tdWetqC5rOxNEVbXTUEY5e6C0uw~0--jxhimE7UZRLN5A715~gxk1c7nAnzYm36sSvvvucB8OCg8BGYgMZnodaEfe~Vxx0GSW5mXdyYCFcdGIUGHNfCBeMZ-pDfiRaQ1qCY36Y-uv83ZWT22wPSoTPJbPG34sWg4z4wtiitpMy6xXqcx6Aa34egNLsE2pEbgfmNg~Ac5o1elEOZj4VNgoLGYyLLZZxSc1YVdApHOgRCsUd3Kks1-lyE-STxcapXTnegloZ79DG3DUX3K8qow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7207948&ar=477266&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparing the Fisher coefficients (<em>C</em>) between <em>P. aeruginosa</em> and <em>Buchnera</em>.</p></div></div>                            <h3 scrollto-destination=7207949 id="7207949" class="section-title" data-legacy-id=s3c>3.3. Correlation between the genome size and the Fisher coefficients</h3>
<p>Although it has been validated that the Fisher coefficients is much universal for genomes with medium G + C content, the high accuracy obtained by <em>P. aeruginosa</em> presented in Table <a reveal-id="DSR030TB2" data-open="DSR030TB2" class="link link-reveal link-table xref-fig">2</a> seems to be much unexpected, for its G + C content is up to 66%. On the other hand, the genome size of <em>P. aeruginosa</em> is 6 588 339 bp with 6286 annotated protein-coding genes. Then, another issue is questioned that whether the high accuracy is caused by its huge genomic contents. To explore the correlation between the genome size and the trained Fisher coefficients, four additional species with different genome size, <em>Stenotrophomonas maltophilia</em> (4 851 126 bp), <em>Burkholderia cenocepacia</em> (875 977 bp), <em>Deinococcus radiodurans c</em>2 (412 348 bp) and <em>D. radiodurans c</em>1 (2 648 638 bp) are selected, which have similar G + C content (66%) with <em>P. aeruginosa</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Whereupon we use the Fisher coefficients and threshold trained by <em>P. aeruginosa</em>, <em>S. maltophilia</em>, <em>B. cenocepacia</em>, <em>D. radiodurans c</em>2 and <em>D. radiodurans c</em>1 to identify the protein-coding genes in the genomes with different G + C content, where the other five species listed in Table <a reveal-id="DSR030TB2" data-open="DSR030TB2" class="link link-reveal link-table xref-fig">2</a> are employed, respectively. The predicting results are presented in Table <a reveal-id="DSR030TB3" data-open="DSR030TB3" class="link link-reveal link-table xref-fig">3</a>. Among the four selected species, <em>S. maltophilia</em> has the biggest genome size, but it seems that its Fisher coefficients are not as universal as that of <em>P. aeruginosa</em>. Based on the coefficients and threshold trained by <em>S. maltophilia</em>, 98.99% known functional genes in <em>C. michiganensis</em> are correctly predicted, while the accuracy drops to 0.21% with the decrease in the G + C content. <em>Deinococcus radiodurans c</em>2 is the smaller chromosome of <em>D. radiodurans</em> R1, which has only 368 protein-coding genes, while its performance is comparable with that of <em>P. aeruginosa</em>. As for the other two species, the average accuracy of <em>B. cenocepacia</em> is higher than that of <em>D. radiodurans c</em>1, although the latter has a much bigger genome size. </p>                        <a id="7207951" scrollto-destination="7207951"></a>
<div content-id="DSR030TB3" class="table-modal"><div class="table-wrap"><div id="DSR030TB3" class="table-wrap-title" data-id="DSR030TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Predicting results based on genomes with different sizes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th><em>Buchnera</em> (%) </th><th><em>S. uberis</em> (%) </th><th><em>Y. pestis</em> (%) </th><th><em>B. melitensis</em> (%) </th><th><em>C. michiganensis</em> (%) </th></tr></thead><tbody><tr><td><em>P. aeruginosa</em> </td><td>98.36 </td><td>99.13 </td><td>99.75 </td><td>100 </td><td>100 </td></tr><tr><td><em>S. maltophilia</em> </td><td>0.21 </td><td>0.80 </td><td>41.62 </td><td>91.91 </td><td>98.99 </td></tr><tr><td><em>B. cenocepacia</em> </td><td>71.46 </td><td>81.58 </td><td>93.68 </td><td>99.47 </td><td>99.12 </td></tr><tr><td><em>D. radiodurans c</em>2 </td><td>98.15 </td><td>99.53 </td><td>99.80 </td><td>100 </td><td>100 </td></tr><tr><td><em>D. radiodurans c</em>1 </td><td>7.60 </td><td>71.09 </td><td>95.64 </td><td>99.54 </td><td>99.87 </td></tr></tbody></table></div><div content-id="DSR030TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th><em>Buchnera</em> (%) </th><th><em>S. uberis</em> (%) </th><th><em>Y. pestis</em> (%) </th><th><em>B. melitensis</em> (%) </th><th><em>C. michiganensis</em> (%) </th></tr></thead><tbody><tr><td><em>P. aeruginosa</em> </td><td>98.36 </td><td>99.13 </td><td>99.75 </td><td>100 </td><td>100 </td></tr><tr><td><em>S. maltophilia</em> </td><td>0.21 </td><td>0.80 </td><td>41.62 </td><td>91.91 </td><td>98.99 </td></tr><tr><td><em>B. cenocepacia</em> </td><td>71.46 </td><td>81.58 </td><td>93.68 </td><td>99.47 </td><td>99.12 </td></tr><tr><td><em>D. radiodurans c</em>2 </td><td>98.15 </td><td>99.53 </td><td>99.80 </td><td>100 </td><td>100 </td></tr><tr><td><em>D. radiodurans c</em>1 </td><td>7.60 </td><td>71.09 </td><td>95.64 </td><td>99.54 </td><td>99.87 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7207951" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR030TB3" class="table-wrap-title" data-id="DSR030TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Predicting results based on genomes with different sizes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th><em>Buchnera</em> (%) </th><th><em>S. uberis</em> (%) </th><th><em>Y. pestis</em> (%) </th><th><em>B. melitensis</em> (%) </th><th><em>C. michiganensis</em> (%) </th></tr></thead><tbody><tr><td><em>P. aeruginosa</em> </td><td>98.36 </td><td>99.13 </td><td>99.75 </td><td>100 </td><td>100 </td></tr><tr><td><em>S. maltophilia</em> </td><td>0.21 </td><td>0.80 </td><td>41.62 </td><td>91.91 </td><td>98.99 </td></tr><tr><td><em>B. cenocepacia</em> </td><td>71.46 </td><td>81.58 </td><td>93.68 </td><td>99.47 </td><td>99.12 </td></tr><tr><td><em>D. radiodurans c</em>2 </td><td>98.15 </td><td>99.53 </td><td>99.80 </td><td>100 </td><td>100 </td></tr><tr><td><em>D. radiodurans c</em>1 </td><td>7.60 </td><td>71.09 </td><td>95.64 </td><td>99.54 </td><td>99.87 </td></tr></tbody></table></div><div content-id="DSR030TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th><em>Buchnera</em> (%) </th><th><em>S. uberis</em> (%) </th><th><em>Y. pestis</em> (%) </th><th><em>B. melitensis</em> (%) </th><th><em>C. michiganensis</em> (%) </th></tr></thead><tbody><tr><td><em>P. aeruginosa</em> </td><td>98.36 </td><td>99.13 </td><td>99.75 </td><td>100 </td><td>100 </td></tr><tr><td><em>S. maltophilia</em> </td><td>0.21 </td><td>0.80 </td><td>41.62 </td><td>91.91 </td><td>98.99 </td></tr><tr><td><em>B. cenocepacia</em> </td><td>71.46 </td><td>81.58 </td><td>93.68 </td><td>99.47 </td><td>99.12 </td></tr><tr><td><em>D. radiodurans c</em>2 </td><td>98.15 </td><td>99.53 </td><td>99.80 </td><td>100 </td><td>100 </td></tr><tr><td><em>D. radiodurans c</em>1 </td><td>7.60 </td><td>71.09 </td><td>95.64 </td><td>99.54 </td><td>99.87 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7207951" target="_blank">View Large</a></div></div><p><em>Deinococcus radiodurans c</em>2 and <em>D. radiodurans c</em>1 are the smaller and larger chromosomes of <em>D. radiodurans</em> R1, respectively. In previous works, some authors deemed the bases distribution patterns in different chromosomes for one bacterium to be similar, then they do not have independent origins.<sup><a reveal-id="DSR030C15" data-open="DSR030C15" class="link link-ref link-reveal xref-bibr">15</a></sup> While some other works conjectured that the smaller chromosome was originally a megaplasmid captured by an ancestral species.<sup><a reveal-id="DSR030C47" data-open="DSR030C47" class="link link-ref link-reveal xref-bibr">47</a></sup> Both the two chromosomes share similar G + C content. In the larger chromosome, the G + C contents of the annotated protein-coding genes range from 34.1 to 77.1%, with the average of 67.44%, the values of GC3 range from 28.2 to 96.3%, with the average of 83.54%. In the smaller chromosome, the G + C contents of the annotated protein-coding genes range from 37 to 76.1%, with the average of 67.14%, the values of GC3 range from 28.4 to 93.1%, with the average of 82.34%. Nevertheless, from Table <a reveal-id="DSR030TB3" data-open="DSR030TB3" class="link link-reveal link-table xref-fig">3</a>, we note that there are much differences between the results obtained by <em>D. radiodurans c</em>2 and <em>D. radiodurans c</em>1, which indicates that their Fisher coefficients display discrepant properties. We speculate that the differences can be accounted by their intrinsic genes features. For protein-coding genes, the relative synonymous codon usage (RSCU) is an effective index used to examine synonymous codon usage without the confounding influence of amino acid composition of different gene samples.<sup><a reveal-id="DSR030C48" data-open="DSR030C48" class="link link-ref link-reveal xref-bibr">48</a></sup> Correspondence analysis (COA) can be used to investigate the major trend in codon usage variation among protein-coding genes. After performing COA on the RSCU values of the annotated protein-coding genes, it was found that axes 1 and 2 of COA account for 24.59 and 5.31% for <em>D. radiodurans c</em>2, and those for <em>D. radiodurans c</em>1 account for 15.6 and 5.69%, respectively. The prominent weight of the first principle suggests strong codon bias trend in both chromosomes. To assess the factors that affect the codon usage bias, we plotted axis 1 against their codon adaptation index (CAI) values in Fig. <a reveal-id="DSR030F2" data-open="DSR030F2" class="link link-reveal link-table xref-fig">2</a>. CAI is used to measure the gene expression level,<sup><a reveal-id="DSR030C43" data-open="DSR030C43" class="link link-ref link-reveal xref-bibr">43</a></sup> highly expressed genes are presumed to have high CAI values. In Fig. <a reveal-id="DSR030F2" data-open="DSR030F2" class="link link-reveal link-table xref-fig">2</a>, the scatter plot for <em>D. radiodurans c</em>1 indicates a significant positive correlation with the position of the genes on axis 1 and their corresponding CAI values (<em>r</em> = 0.5275, <em>P</em> &lt; 0.01). Further analysis suggests that the coordinates of axis 1 also show a significant positive correlation with the GC3 (<em>r</em> = 0.9090, <em>P</em> &lt; 0.01). On the contrary, from the scatter plot of <em>D. radiodurans c</em>2, a significant negative correlation between axis 1 and their CAI values (<em>r</em> = −0.5712, <em>P</em> &lt; 0.01) can be observed. Further analysis shows that axis 1 is also significantly negatively correlate with the GC3 (<em>r</em> = −0.9267, <em>P</em> &lt; 0.01). Furthermore, the CAI value shows positive correlation with GC3 in both <em>D. radiodurans c</em>1 (<em>r</em> = 0.3437, <em>P</em> &lt; 0.01) and <em>D. radiodurans c</em>2 (<em>r</em> = 0.4726, <em>P</em> &lt; 0.01). These results show that gene expression level and base compositions play an important role to shape the codon usage patterns in both chromosomes; the highly expressed genes prefer higher GC content at their synonymous third codon position. However, the opposite trends exhibit different evolutionary pressures on them, which seem to support the speculation that the smaller chromosome was originally a megaplasmid. Therefore, our analysis shows that there is no causality between the genome size and the Fisher coefficients, but the highly universal Fisher coefficients in some species imply that there may be general properties, which is worthy of further researching in the future. On the other hand, the present analysis implies that the integrated 75D vector is sensitive to the specific gene structures, which may provide novel clues for gene-finding algorithms. </p>                        <a id="7207953" scrollto-destination="7207953"></a>
<div id="DSR030F2" data-id="DSR030F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/m_dsr03002.jpeg?Expires=2147483647&Signature=O-37SpA1ocx9GMFh4d9f60XV1~32JhHYIJK2AMkosWSUu~cJhOfj6Q0YkatqA1jatehSWGwmUYnV6Kd77flYnUM30S83UGl9uygGx6S2gq3iMb5wEkfyEt92~21QQJ738KVpzxUVLuyBCaoCaPci4n2hwHe65K9f2psGTzLWPUza6bg6E2uG9cbKz4NRO3ih6NQIc1BvThFj6lr64mMaEpeBpyss4jFSohZ0TUso8bydHYlPTdjs5AqzJilRvcErOLeR3S97fPkHObi6WFA3zlke~-wE81pI56QdcPqxL0BEKyV1vCJr-ZVUZr4bSJ20rCRy5vyp1cQWNcd1FqBDxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plot of axis 1 against CAI." path-from-xml="dsr03002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7207953/dsr03002.jpeg" path-from-xml="dsr03002.jpeg" target="_blank">View large</a><a section="7207953" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/dsr03002.jpeg?Expires=2147483647&Signature=wFZpAP4IC4EmZpOyYSXDm0P26OjyjQo5TTL38ypeThLsU6hmcQb4JgGe~idHsK656I-ic9SoMYyVcNp755LWjXzszxTAr9yJ1PTCmF5jrKSNB53yrP-SqOOrJlXQNXoa29eKRKDJ18wz79lrVuNIBXfdOP98rs5leIkAm3bZkRUJYJWJ0qp5fEmnBuVTkeXugQQKSAIUeikspLEajplVnpBDahfxn6rNYmWlLP8pxIML9RpjjL9HYa7JNzNhU1E6R-MYQh-Vx5zAspVsMtozFspAOxcwtyJHYz1N8eTV7pC8UYPKc6cgH2qJxZkPPNs26130t92TDphSecKBhGv36Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7207953&ar=477266&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plot of axis 1 against CAI.</p></div></div><div content-id="DSR030F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/m_dsr03002.jpeg?Expires=2147483647&Signature=O-37SpA1ocx9GMFh4d9f60XV1~32JhHYIJK2AMkosWSUu~cJhOfj6Q0YkatqA1jatehSWGwmUYnV6Kd77flYnUM30S83UGl9uygGx6S2gq3iMb5wEkfyEt92~21QQJ738KVpzxUVLuyBCaoCaPci4n2hwHe65K9f2psGTzLWPUza6bg6E2uG9cbKz4NRO3ih6NQIc1BvThFj6lr64mMaEpeBpyss4jFSohZ0TUso8bydHYlPTdjs5AqzJilRvcErOLeR3S97fPkHObi6WFA3zlke~-wE81pI56QdcPqxL0BEKyV1vCJr-ZVUZr4bSJ20rCRy5vyp1cQWNcd1FqBDxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plot of axis 1 against CAI." path-from-xml="dsr03002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7207953/dsr03002.jpeg" path-from-xml="dsr03002.jpeg" target="_blank">View large</a><a section="7207953" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/dsr03002.jpeg?Expires=2147483647&Signature=wFZpAP4IC4EmZpOyYSXDm0P26OjyjQo5TTL38ypeThLsU6hmcQb4JgGe~idHsK656I-ic9SoMYyVcNp755LWjXzszxTAr9yJ1PTCmF5jrKSNB53yrP-SqOOrJlXQNXoa29eKRKDJ18wz79lrVuNIBXfdOP98rs5leIkAm3bZkRUJYJWJ0qp5fEmnBuVTkeXugQQKSAIUeikspLEajplVnpBDahfxn6rNYmWlLP8pxIML9RpjjL9HYa7JNzNhU1E6R-MYQh-Vx5zAspVsMtozFspAOxcwtyJHYz1N8eTV7pC8UYPKc6cgH2qJxZkPPNs26130t92TDphSecKBhGv36Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7207953&ar=477266&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plot of axis 1 against CAI.</p></div></div>                            <h3 scrollto-destination=7207954 id="7207954" class="section-title" data-legacy-id=s3d>3.4. Why the filtered ORFs do not encode proteins</h3>
<p>Using the presented re-annotating algorithm, we re-identified the protein-coding genes in the 61 genomes. For objectivity, the algorithm is performed five times on each genome, those ORFs with <em>T_</em>score &lt;0 occurring equal or more than three times are recognized as non-coding. Consequently, an average of 99.94% over the 61 microbial genomes is achieved (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). From the predicting results listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>, we can find that different numbers of ORFs are predicted as non-coding in most species. Taking the genomes of <em>Pyrococcus horikoshii</em> and <em>Caulobacter crescentus</em> as examples, 72 and 76 hypothetical genes are filtered as non-coding, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>).</p><p>Previous analysis of protein-coding genes showed that there are severe restrictions on bases distributions at different codon positions because they are associated with different biological functions. It was found that the restrictions are universal that the first codon position prefers purine bases.<sup><a reveal-id="DSR030C49" data-open="DSR030C49" class="link link-ref link-reveal xref-bibr">49</a></sup> After calculating the purine–pyrimidine disparities at the first codon position, an average of −0.026 (−2.6%) is obtained for the 71 filtered hypothetical ORFs in <em>P. horikoshii</em> genome, while an average value of 0.3785 (37.85%) is obtained for the protein-coding genes. In <em>C. crescentus</em> genome, an average of 0.0425 (4.25%) for the 76 filtered hypothetical ORFs is obtained, while for those protein-coding genes, an average of 0.2131 (21.31%) is obtained. These results show that purine bases are dominant at the first codon position for the protein-coding genes, while the purine bases are equivalent to the pyrimidine bases for the recognized non-coding ORFs, which indicate that they are likely random sequences that are falsely predicted as protein-coding genes. The differences between coding and non-coding ORFs can also be displayed by the principal component analysis (PCA). PCA defines the correlation among the variables of given data. We have demonstrated that the integrative 75D vector carries sufficient information to exhibit the specific properties of protein-coding genes. After performing PCA, we project the 75 parameters of each ORF into a 2D coordinates by the first two principal components, which is shown in Fig. <a reveal-id="DSR030F3" data-open="DSR030F3" class="link link-reveal link-table xref-fig">3</a>. It can be seen that the filtered non-coding ORFs are clustered far from the core of coding sequences. The different regions reflect that there are intrinsic differences between the recognized ORFs and coding sequences. Then, the results based on PCA also indicate that the filtered ORFs are unlikely to encode proteins. </p>                        <a id="7207957" scrollto-destination="7207957"></a>
<div id="DSR030F3" data-id="DSR030F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/m_dsr03003.jpeg?Expires=2147483647&Signature=yVh62FYhn6RywZfuIahoaXFKTMc8Felac~W31JdA-La50vjGAt38J8O8FFTWVKP6tqA5jSNi90MvK2c8j2JtynGrZw4GrEe6DakPApzXBaxlXttiaL6gbMfIwmjS2z1u~soUp-6R2Z7B2vWzkah1i9KhSAKByIyktuemh5ZV4g3ZNieiOZfyTtuGB0llIclixsgELDPkynR1jIkSdyE-wieJqfjfhupZ-52ehruZhJofJWsw4QNTOe8giW1XmNdH2W3DOvsQPiqdphFwAjqrcZJ2lQiBqxDamgjXyz7jzaUyubZ21w9~1nKWjdxpns2aaOiJeUW8tKNZlv0eFI7DOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Projecting the annotated ORFs into 2D coordinates by PCA." path-from-xml="dsr03003.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7207957/dsr03003.jpeg" path-from-xml="dsr03003.jpeg" target="_blank">View large</a><a section="7207957" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/dsr03003.jpeg?Expires=2147483647&Signature=E2fccI-u1sPyR0aCGvo8bSjs--vCJeBgHfjGRUIWtalz5IvvWFfwkJ4qYJuFLdIAVpnzQvfwmTA0yRgR8ZPFFroj2lQP182QxtOyLdmwWJV3DMfh9tHJ-A4lgZyU2XG3F8k-kLkuXMKpwhSArJDPaYlER5OKXWf8ixljCIu~b-ZYuagN3reK78c2m8ZO~cPwpze-7xXbebS2PUlW75ycKgbdj-17X99krqC0s70ufSu5SXwjld817IpeoQ3XQdbeelhiBjftW1xzrHaywjauzOhtuEl~8XcaQZrc-NkfacKw04nvA8eRlMufHme-0Vuc218jAHCxSlpE~ia0gCLZzg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7207957&ar=477266&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03003.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Projecting the annotated ORFs into 2D coordinates by PCA.</p></div></div><div content-id="DSR030F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/m_dsr03003.jpeg?Expires=2147483647&Signature=yVh62FYhn6RywZfuIahoaXFKTMc8Felac~W31JdA-La50vjGAt38J8O8FFTWVKP6tqA5jSNi90MvK2c8j2JtynGrZw4GrEe6DakPApzXBaxlXttiaL6gbMfIwmjS2z1u~soUp-6R2Z7B2vWzkah1i9KhSAKByIyktuemh5ZV4g3ZNieiOZfyTtuGB0llIclixsgELDPkynR1jIkSdyE-wieJqfjfhupZ-52ehruZhJofJWsw4QNTOe8giW1XmNdH2W3DOvsQPiqdphFwAjqrcZJ2lQiBqxDamgjXyz7jzaUyubZ21w9~1nKWjdxpns2aaOiJeUW8tKNZlv0eFI7DOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Projecting the annotated ORFs into 2D coordinates by PCA." path-from-xml="dsr03003.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7207957/dsr03003.jpeg" path-from-xml="dsr03003.jpeg" target="_blank">View large</a><a section="7207957" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/dsr03003.jpeg?Expires=2147483647&Signature=E2fccI-u1sPyR0aCGvo8bSjs--vCJeBgHfjGRUIWtalz5IvvWFfwkJ4qYJuFLdIAVpnzQvfwmTA0yRgR8ZPFFroj2lQP182QxtOyLdmwWJV3DMfh9tHJ-A4lgZyU2XG3F8k-kLkuXMKpwhSArJDPaYlER5OKXWf8ixljCIu~b-ZYuagN3reK78c2m8ZO~cPwpze-7xXbebS2PUlW75ycKgbdj-17X99krqC0s70ufSu5SXwjld817IpeoQ3XQdbeelhiBjftW1xzrHaywjauzOhtuEl~8XcaQZrc-NkfacKw04nvA8eRlMufHme-0Vuc218jAHCxSlpE~ia0gCLZzg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7207957&ar=477266&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03003.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Projecting the annotated ORFs into 2D coordinates by PCA.</p></div></div><p>In the RefSeq database,<sup><a reveal-id="DSR030C50" data-open="DSR030C50" class="link link-ref link-reveal xref-bibr">50</a></sup> clusters of orthologous groups (COG)<sup><a reveal-id="DSR030C51" data-open="DSR030C51" class="link link-ref link-reveal xref-bibr">51</a></sup> are used to shape the potential functions of proteins produced by the annotated ORFs. Each COG is a group of three or more proteins that have evolved from a common ancestor. Then, these ORFs assigned with a COG are highly likely to be true protein-coding genes. According to the annotation file of <em>P. horikoshii</em>, 98.69% (678 out of 687) known functional genes and 94.87% (37 out of 39) putative genes are marked COG, while among the 1229 hypothetical genes, only 740 (56.97%) are marked COG. In the annotation file of <em>C. crescentus</em>, the percentages of the marked COG of known functional genes, putative genes and hypothetical genes are 96.25% (2107 out of 2189), 100% (15 out of 15) and 51.21% (785 out of 1533), respectively. Among the recognized 71 + 76 = 147 non-coding ORFs in genomes of <em>P. horikoshii</em> and <em>C. crescentus</em>, none has been assigned with COG tags. According to the results in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>, up to 925 hypothetical ORFs are recognized as non-coding, among which only 50 (5.41%) are marked with COG. Some previous works show that a significant fraction of annotated short ORFs may be not true genes, which is one of the major causes that account for the over-annotation of microbial genomes.<sup><a reveal-id="DSR030C3" data-open="DSR030C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSR030C52" data-open="DSR030C52" class="link link-ref link-reveal xref-bibr">52</a></sup> In <em>P. horikoshii</em> genome, the average length of the 71 annotated non-coding sequences is 391 bp, which is much shorter than that of these recognized protein-coding genes (858 bp). The similar result is also obtained in the genome of <em>C. crescentus</em>, in which the average length of the 76 annotated non-coding ORFs is 466 bp, while that of these protein-coding genes is 981 bp. In Table <a reveal-id="DSR030TB4" data-open="DSR030TB4" class="link link-reveal link-table xref-fig">4</a>, we present the statistical results of the length distributions among all the 925 recognized non-coding ORFs. Then, from the above analysis, sufficient evidences suggest that most of these filtered ORFs are random sequences that are falsely predicted as protein-coding genes. </p>                        <a id="7207959" scrollto-destination="7207959"></a>
<div content-id="DSR030TB4" class="table-modal"><div class="table-wrap"><div id="DSR030TB4" class="table-wrap-title" data-id="DSR030TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Distribution of sequence length among the 925 recognized non-coding ORFs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="2"><em>L</em> ≤ 300 bp<hr /> </th><th colspan="2">300 bp &lt; <em>L</em> &lt; 500 bp<hr /> </th><th colspan="2"><em>L</em> &gt; 500 bp<hr /> </th><th rowspan="2">Average (bp) </th></tr><tr><th>Number </th><th>Percentage </th><th>Number </th><th>Percentage </th><th>Number </th><th>Percentage </th></tr></thead><tbody><tr><td>367 </td><td>39.68 </td><td>335 </td><td>36.22 </td><td>223 </td><td>24.11 </td><td>411 </td></tr></tbody></table></div><div content-id="DSR030TB4" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="2"><em>L</em> ≤ 300 bp<hr /> </th><th colspan="2">300 bp &lt; <em>L</em> &lt; 500 bp<hr /> </th><th colspan="2"><em>L</em> &gt; 500 bp<hr /> </th><th rowspan="2">Average (bp) </th></tr><tr><th>Number </th><th>Percentage </th><th>Number </th><th>Percentage </th><th>Number </th><th>Percentage </th></tr></thead><tbody><tr><td>367 </td><td>39.68 </td><td>335 </td><td>36.22 </td><td>223 </td><td>24.11 </td><td>411 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7207959" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR030TB4" class="table-wrap-title" data-id="DSR030TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Distribution of sequence length among the 925 recognized non-coding ORFs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="2"><em>L</em> ≤ 300 bp<hr /> </th><th colspan="2">300 bp &lt; <em>L</em> &lt; 500 bp<hr /> </th><th colspan="2"><em>L</em> &gt; 500 bp<hr /> </th><th rowspan="2">Average (bp) </th></tr><tr><th>Number </th><th>Percentage </th><th>Number </th><th>Percentage </th><th>Number </th><th>Percentage </th></tr></thead><tbody><tr><td>367 </td><td>39.68 </td><td>335 </td><td>36.22 </td><td>223 </td><td>24.11 </td><td>411 </td></tr></tbody></table></div><div content-id="DSR030TB4" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="2"><em>L</em> ≤ 300 bp<hr /> </th><th colspan="2">300 bp &lt; <em>L</em> &lt; 500 bp<hr /> </th><th colspan="2"><em>L</em> &gt; 500 bp<hr /> </th><th rowspan="2">Average (bp) </th></tr><tr><th>Number </th><th>Percentage </th><th>Number </th><th>Percentage </th><th>Number </th><th>Percentage </th></tr></thead><tbody><tr><td>367 </td><td>39.68 </td><td>335 </td><td>36.22 </td><td>223 </td><td>24.11 </td><td>411 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7207959" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=7207960 id="7207960" class="section-title" data-legacy-id=s3e>3.5. Influence of horizontal gene transfer</h3>
<p>Horizontal gene transfer (HGT) is one of the most important driving forces in prokaryotic evolution that can drift among bacteria not only from similar strains, but also from distantly related species.<sup><a reveal-id="DSR030C53" data-open="DSR030C53" class="link link-ref link-reveal xref-bibr">53</a></sup> In the cases of <em>P. horikoshii</em> and <em>C. crescentus</em>, 9 (PH0055, PH0216.1n, PH0221, PH0428, PH1184, PH1187, PH1741, PH1861, PHs009) and 16 (CC_0370, CC_0605, CC_0853, CC_1246, CC_1274, CC_1712, CC_2413, CC_2699, CC_2731, CC_2732, CC_2737, CC_3515, CC_3516, CC_3517, CC_3520, CC_3548) of the recognized non-coding ORFs listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a> are detected as HGTs by the HGT database,<sup><a reveal-id="DSR030C54" data-open="DSR030C54" class="link link-ref link-reveal xref-bibr">54</a></sup> respectively. The predicting procedure employed in the HGT database is based on the parametric methods, which identify anomalous sequence signatures of all the genes and derived protein sequences for each organism. The parametric approaches are based on the hypothesis that sequence features are similar within a genome but differ significantly between genomes. However, just as the authors of the HGT database have pointed out, these predicting results should be used with caution, for other forces may be responsible for the codon usage or G + C content heterogeneity of a genome. A recent comparison of different HGT prediction programs showed that these sequence composition-based methods could predict very different classes of genes.<sup><a reveal-id="DSR030C55" data-open="DSR030C55" class="link link-ref link-reveal xref-bibr">55</a></sup> In addition, very recently acquired prophage elements tended to have sequence compositions that are more similar to the host genome, not representing amelioration but rather specialization and adaptation to their hosts.<sup><a reveal-id="DSR030C56" data-open="DSR030C56" class="link link-ref link-reveal xref-bibr">56</a></sup> A recent study of large viruses further supported that some genes with atypical sequence composition are not horizontally acquired but are likely related to certain functions and gene features.<sup><a reveal-id="DSR030C57" data-open="DSR030C57" class="link link-ref link-reveal xref-bibr">57</a></sup> This means that there are a considerable number of false negatives and false positives among the detecting results. Although very strict methodology has been outlined to improve the performance, recent researches show that the mean error of these HGT predicting methods is up to 39.96%, even for the most efficient programs.<sup><a reveal-id="DSR030C58" data-open="DSR030C58" class="link link-ref link-reveal xref-bibr">58</a></sup> In most cases, the false positives are attributed to pseudogenes, segments of fossilized DNA, and the compositional asymmetries between genes lying on the leading versus lagging strand, selection for translational efficiency, mutation biases and random drift.<sup><a reveal-id="DSR030C59" data-open="DSR030C59" class="link link-ref link-reveal xref-bibr">59</a></sup> In Section 3.4, we have verified that the filtered ORFs by our re-annotating algorithm are highly likely non-coding sequences that are over-annotated as protein-coding genes. On the other hand, our proposed algorithm is trained by the positive samples composed of those genuine protein-coding genes and the negative samples composed of the shuffled random sequences. Thereupon we infer that the 9 + 16 = 25 ORFs in <em>P. horikoshii</em> and <em>C. crescentus</em> that have been detected as HGTs by the HGT database are not laterally transferred in fact. That is, the falsely predicted protein-coding genes should also be taken into account for the false positives in HGT predictions in future studies.</p>                            <h3 scrollto-destination=7207962 id="7207962" class="section-title" data-legacy-id=s3f>3.6. Comparing the presented algorithm with other programs</h3>
<p>In the past decade, many <em>de novo</em> gene-finding algorithms have been proposed for discriminating the protein-coding genes in prokaryotic genomes. Thereinto, the HMM-based methods such as Glimmer<sup><a reveal-id="DSR030C60" data-open="DSR030C60" class="link link-ref link-reveal xref-bibr">60</a></sup> and GeneMark<sup><a reveal-id="DSR030C61" data-open="DSR030C61" class="link link-ref link-reveal xref-bibr">61</a></sup> are two most popular gene-finding programs that have been used in some re-annotating works.<sup><a reveal-id="DSR030C17" data-open="DSR030C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSR030C25" data-open="DSR030C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Our re-annotating algorithm has been shown to enhance the predicting performance extremely. The initial annotations of the 61 genomes listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a> did not use these HMM-based programs. Then it is interesting to compare the presented method with the two prevalent gene-finding programs. Glimmer 3.02 and GeneMark.hmm 2.4 are performed on the 61 species, and the performance is evaluated based on the known functional genes. In <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>, we present the predicting results. As a result, Glimmer 3.02 and GeneMark.hmm 2.4 achieve the average accuracies of 98.89 and 99.14%, respectively. It is noted that no records or appropriate reference species were found for <em>S. maltophilia</em>, <em>Acidovorax citrulli</em> and <em>C. michiganensis</em> genomes when performing the GeneMark.hmm program. Using the models trained by other species to annotate these genomes, the accuracies are much lower (&lt;50% in most cases). By contrast, the presented integrative method achieves an average accuracy of 99.94%, which is ∼1% higher than the two other programs. Analysing the predicting results of Glimmer 3.02 and GeneMark.hmm 2.4, we found that although similar accuracy is obtained, the results differ greatly in most cases. Taking the genome of <em>Escherichia coli CFT073</em> as an example, Glimmer 3.02, GeneMark.hmm 2.4 and our algorithm miss 21, 20 and 3 items of the 2580 annotated known functional protein-coding genes, respectively. Then highly similar accuracies are obtained by Glimmer 3.02 (99.19%) and GeneMark 2.4 (99.22%). Based on the two programs, 5155 (Glimmer) and 5015 (GeneMark) ORFs are predicted as protein-coding genes in <em>E. coli CFT073</em> genome, respectively. Among the 5338 potential protein-coding genes annotated in the current RefSeq database, there are 4803 common items identified by Glimmer 3.02 and 4701 common items identified by GeneMark.hmm 2.4, which mean that 535 and 637 ORFs are excluded as non-coding by the two programs, respectively. Besides, there are only 4803 common items observed by Glimmer and GeneMark. These comparisons indicate that there are significant differences among the predicting results by Glimmer and GeneMark.hmm even they share similar accuracy. Then, it is difficult to determine which one is superior to another, and additional bioinformatics analysis is necessary to avoid the increase in both false positives and false negatives when using these programs. Recently, Luo <em>et al</em>.<sup><a reveal-id="DSR030C25" data-open="DSR030C25" class="link link-ref link-reveal xref-bibr">25</a></sup> excluded 608 annotated protein-coding genes in <em>E. coli CFT073</em> genome from the RefSeq database. In their work, four gene-finding programs were adopted and the ORFs that are co-predicted less than three were deemed as non-coding. However, according to our analysis, this kind of filtering strategy is not rigorous enough, and some genuine genes can be lost, which results in the increase in false negatives. For comparison, we also re-annotated the protein-coding genes in <em>E. coli CFT073</em> genome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>), and 77 hypothetical genes are recognized as non-coding. We have demonstrated that the presented integrative algorithm is reliable for the problem of over-annotation of protein-coding genes in microbial genomes, then we hope our work can provide an efficient platform for future re-annotation researches.</p>                            <h3 scrollto-destination=7207964 id="7207964" class="section-title" data-legacy-id=s3g>3.7. Stability of the re-annotating algorithm</h3>
<p>In the presented re-annotating algorithm, the negative samples used as the training set are generated by the randomly shuffled sequences. Then it is necessary to investigate the influence of the shuffled negative samples on the output of the re-annotating program. To accomplish the analysis, the known functional genes of <em>P. horikoshii</em> and <em>C. crescentus</em> genomes are employed following the subsequent steps. First, the re-annotating algorithm is performed on each genome 10 times, which correspond to 10 sets of negative samples. Then, for each genome, 10 vectors composed of the <em>T_</em>scores of these known functional genes are obtained. Finally, the correlations among the 10 vectors are calculated, and a 10 × 10 matrix can be obtained for each genome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S7</a> and <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/lookup/suppl/doi:10.1093/dnares/dsr030/-/DC1" target="_blank">S8</a>). In the obtained 10 × 10 matrix, the elements listed in the diagonal equal to 1 (omitted in corresponding tables), which represent the self-correlation of each vector. The underlying assumption is that the more significantly correlated among the 10 vectors, the more stable the re-annnotating program is. The results can also be visualized in Fig. <a reveal-id="DSR030F4" data-open="DSR030F4" class="link link-reveal link-table xref-fig">4</a>, in which we present the standard deviation and the average values of each column. The average correlation coefficients are bigger than 0.97 for both genomes. Then the correlations among the 10 vectors are overall significantly positively correlated (<em>P</em> &lt; 0.01). This can be validated by normalizing the <em>T_</em>score vectors with the following equation, <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/m_dsr030ileq2.gif?Expires=2147483647&Signature=hljMED9tz0PKLCpm06UylQM52XWehgcqyA2pULW1S760ieAZrvjN2sWNqfrXwRlDurQ17yd24AY8vjh2BhsOcDhjfZcMmO4FvhDWGseYhkW3hUP~XnV0vtfVSEWS9b4odMDJ6NHONFMlPhUQglj2UY9wXpYZ-UbhAl1mWBM5Roi3EbhUF8t52bSWIlth6SA4Ssbn7MFgNdogQy26ntZJ4GuMuGQ-QhYxVz6tUlpqq6NI98v-8d0N-6LPgNtf537YE2FX7hA94cElDfhyyxOO2vz4AEKLsPhPhUkbUj1bhFz9Sp2cG3AxYFXuIYJPQZ~KVtP0~WYKESnB1p1m6ox5rA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr030ileq2.gif" />⁠</span>, where <em>T<sub>n</sub></em> is the initial <em>T</em>_score of corresponding ORF. For convenience, we present the normalized results in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>, from which one can find the results by the 10 running times are much consistent with each other. Therefore, our re-annotating algorithm is robust. Even so, the <em>T</em>_score of some individual ORFs may slightly fluctuate in some cases, hence we advise the users to perform the program more than one times (five in the present paper) and determine those ORFs with <em>T</em>_score &lt;0 whose occurring times equal or more than a given threshold (three in the present paper) as non-coding. </p>                        <a id="7207966" scrollto-destination="7207966"></a>
<div id="DSR030F4" data-id="DSR030F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/m_dsr03004.jpeg?Expires=2147483647&Signature=owiCRG0Pav-J~de4dcHNiD0VNIPMcysTFQU-yYkuUzUGOpJIIUD5~4bVOu5Q37TcNahKwDtvoJnTzxHWox86I-PqllstB7Dsr~hCTmIWwlD6Zo4WWDzZ79d8QMim8v~kUErG1Ev8F18LEsvJ~qgGVE-8b7mW8bTPCZfY0kMbFZy7R~p4HcVRQvocqCWHLRz49C6bbR5dtrv8PCci6c0NZudmFQYJd7TjWXJAy6kR~FAbnS~a7iG7kc0JVtEXW4s4A4hBYJpfKrqwD3OxIJ1oyymMXx~DS-nS0xqJffj9HYapY4Xp6~PmM-2fqymPGkFBvXQ0Og4FqKeL9qz4o8ph1A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlations among the 10 vectors composed of each genome." path-from-xml="dsr03004.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7207966/dsr03004.jpeg" path-from-xml="dsr03004.jpeg" target="_blank">View large</a><a section="7207966" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/dsr03004.jpeg?Expires=2147483647&Signature=aDM1H-aKxNfV0InGGwzqrAApSYsSnT~0O0VLh-OtbSnSk84LJHBIq35yR0iDbWDx0YUMAo5HSIcsapvY0dL0Aem6Vk6Boj4kzcct9hObBuB-3n7x5tr1hDYHYa8UaAvvTevHCX0HGNIpUiV~3ZZOKQs7uQUl8m8ROTgB-ve27sVaVQqZX~DQtJb9eX-h6Vuw-30IumlqPiVlrdM6X-WLIdcEomLU7~URA~e2qPjduohN9IdXHcps0jlJCQXzF7M49PmyRZncKw4IqNUNL-u2FB2rpFHAlxRIdSJ7tJfWbSkB9EZ-NNS63Vn01ju41Vmb6Kusf8ni~VPvk6HMEkWSwg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7207966&ar=477266&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03004.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlations among the 10 vectors composed of each genome.</p></div></div><div content-id="DSR030F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/m_dsr03004.jpeg?Expires=2147483647&Signature=owiCRG0Pav-J~de4dcHNiD0VNIPMcysTFQU-yYkuUzUGOpJIIUD5~4bVOu5Q37TcNahKwDtvoJnTzxHWox86I-PqllstB7Dsr~hCTmIWwlD6Zo4WWDzZ79d8QMim8v~kUErG1Ev8F18LEsvJ~qgGVE-8b7mW8bTPCZfY0kMbFZy7R~p4HcVRQvocqCWHLRz49C6bbR5dtrv8PCci6c0NZudmFQYJd7TjWXJAy6kR~FAbnS~a7iG7kc0JVtEXW4s4A4hBYJpfKrqwD3OxIJ1oyymMXx~DS-nS0xqJffj9HYapY4Xp6~PmM-2fqymPGkFBvXQ0Og4FqKeL9qz4o8ph1A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlations among the 10 vectors composed of each genome." path-from-xml="dsr03004.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7207966/dsr03004.jpeg" path-from-xml="dsr03004.jpeg" target="_blank">View large</a><a section="7207966" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/dsr03004.jpeg?Expires=2147483647&Signature=aDM1H-aKxNfV0InGGwzqrAApSYsSnT~0O0VLh-OtbSnSk84LJHBIq35yR0iDbWDx0YUMAo5HSIcsapvY0dL0Aem6Vk6Boj4kzcct9hObBuB-3n7x5tr1hDYHYa8UaAvvTevHCX0HGNIpUiV~3ZZOKQs7uQUl8m8ROTgB-ve27sVaVQqZX~DQtJb9eX-h6Vuw-30IumlqPiVlrdM6X-WLIdcEomLU7~URA~e2qPjduohN9IdXHcps0jlJCQXzF7M49PmyRZncKw4IqNUNL-u2FB2rpFHAlxRIdSJ7tJfWbSkB9EZ-NNS63Vn01ju41Vmb6Kusf8ni~VPvk6HMEkWSwg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7207966&ar=477266&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr03004.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlations among the 10 vectors composed of each genome.</p></div></div>                            <h3 scrollto-destination=7207967 id="7207967" class="section-title" data-legacy-id=s3h>3.8. Conclusion</h3>
<p>Genome annotation is a multi-level process and annotating errors can emerge at different stages.<sup><a reveal-id="DSR030C12" data-open="DSR030C12" class="link link-ref link-reveal xref-bibr">12</a></sup> As have been reported in many works, most of the over-annotated protein-coding genes are generated by the <em>de novo</em> gene-finding programs. However, it is difficult to validate these predicted genes by database searches or ‘wet’ experiments one by one, because the excessive computational and expensive cost. In addition, recent re-annotation works showed that only a limited proportion of hypothetical genes can be validated through database searches method.<sup><a reveal-id="DSR030C24" data-open="DSR030C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSR030C25" data-open="DSR030C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Therefore, the deposit of experimental information cannot meet the update speed of explosively increased number of microbial genomes. In this work, we propose an integrative method for filtering the falsely predicted protein-coding genes in microbial genomes. After testing the re-annotating algorithm on 61 microbial genomes, we demonstrate that the re-annotating algorithm is efficient and robust based on sufficient bioinformatics analysis. Our study indicates that the phenomenon of over-annotated protein-coding genes exists in most microorganisms in different degree. Although many re-annotating works have been conducted on some functional microbial genomes in recent years, our research shows that precise analysis is necessary to avoid increasing the false-negative rate of protein-coding genes.</p>                            <h2 scrollto-destination=7207969 id="7207969" class="section-title" data-legacy-id=s4>4. Availability</h2>
<p>Based on our integrative re-annotating algorithm, webserver with a user-friendly interface was developed, which can be freely accessible from <a class="link link-uri" href="http://www.cbi.seu.edu.cn/RPGM">www.cbi.seu.edu.cn/RPGM</a>. This web-based platform aims to the genomic sources in RefSeq databse, which is easy to operate. For convenience, we introduce the platform as follows. </p><ul class="roman-lower"><li><p>Input</p><p>The users are only required to input the accession number of their queried genome, for example, NC_000919.</p></li><li><p>Options</p><p>The self-test is default in the program. To facilitate the users for evaluating the predicting results, the 10-fold cross-validation is also provided.</p></li><li><p>The Job ID</p><p>When running the program every time, a Job ID will be assigned randomly, with which the users can retrieve the predicting results anytime within 3 days by input of the assigned ID.</p></li><li><p>Output</p><p>In the output interface, the predicting results of the protein-coding genes in the four classes are listed and the trained Fisher coefficients can also be downloaded.</p></li><li><p>TN_curve Num 2.0: a generator for the 75-D vector</p><p>We exploit a software package titled TN_curve Num 2.0 that can generate the 75D vector of a given DNA sequence, with which one can also generate the 75D vector of the corresponding shuffled sequence. TN_curve Num 2.0 can be downloaded from the webserver.</p></li></ul>                            <h2 scrollto-destination=7207972 id="7207972" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=7207974 id="7207974" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This study is supported by National Natural Science Foundation of China (Projects No. 61073141, 30970561), Shandong Natural Science Foundation (Project No. ZR2010CQ041) and the Scientific Research Foundation of Graduate School of Southeast University (Project No. YBJJ1010).</p>                            <h2 scrollto-destination=7207976 id="7207976" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSR030C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brenner</div> <div class="given-names">S.E.</div></div>. </div><div class="article-title">Errors in genome annotation</div>, <div class="source">Trends Genet.</div> , <div class="year">1999</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">132</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Errors%20in%20genome%20annotation&amp;author=S.E.%20Brenner&amp;publication_year=1999&amp;journal=Trends%20Genet.&amp;volume=15&amp;pages=132-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(99)01706-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(99)01706-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(99)01706-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10203816" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yangrae</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Virginia</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Computational methods for gene annotation: the Arabidopsis genome</div>, <div class="source">Curr. Opin. Biotech.</div> , <div class="year">2001</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">126</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20methods%20for%20gene%20annotation%3A%20the%20Arabidopsis%20genome&amp;author=C.%20Yangrae&amp;author=W.%20Virginia&amp;publication_year=2001&amp;journal=Curr.%20Opin.%20Biotech.&amp;volume=12&amp;pages=126-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0958-1669(00)00185-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0958-1669(00)00185-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0958-1669(00)00185-3"> </span></div> </div></div></div></div></div><div content-id="DSR030C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Skovgaard</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Jensen</div> <div class="given-names">L.J.</div></div>,  <div class="name"><div class="surname">Brunak</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ussery</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Krogh</div> <div class="given-names">A.</div></div>. </div><div class="article-title">On the total number of genes and their length distribution in complete microbial genomes</div>, <div class="source">Trends Genet.</div> , <div class="year">2001</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">425</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=On%20the%20total%20number%20of%20genes%20and%20their%20length%20distribution%20in%20complete%20microbial%20genomes&amp;author=M.%20Skovgaard&amp;author=L.J.%20Jensen&amp;author=S.%20Brunak&amp;author=D.%20Ussery&amp;author=A.%20Krogh&amp;publication_year=2001&amp;journal=Trends%20Genet.&amp;volume=17&amp;pages=425-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(01)02372-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(01)02372-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(01)02372-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11485798" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Devos</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Valencia</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Intrinsic errors in genome annotation</div>, <div class="source">Trends Genet.</div> , <div class="year">2001</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">429</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intrinsic%20errors%20in%20genome%20annotation&amp;author=D.%20Devos&amp;author=A.%20Valencia&amp;publication_year=2001&amp;journal=Trends%20Genet.&amp;volume=17&amp;pages=429-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(01)02348-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(01)02348-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(01)02348-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11485799" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Harrison</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Kunin</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Gerstein</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Comprehensive analysis of pseudogenes in prokaryotes: widespread gene decay and failure of putative horizontally transferred genes</div>, <div class="source">Genome Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> pg. <div class="fpage">R64</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20analysis%20of%20pseudogenes%20in%20prokaryotes%3A%20widespread%20gene%20decay%20and%20failure%20of%20putative%20horizontally%20transferred%20genes&amp;author=Y.%20Liu&amp;author=P.M.%20Harrison&amp;author=V.%20Kunin&amp;author=M.%20Gerstein&amp;publication_year=2004&amp;journal=Genome%20Biol.&amp;volume=5&amp;pages=R64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2004-5-9-r64" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2004-5-9-r64" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2004-5-9-r64"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15345048" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ussery</div> <div class="given-names">D.W.</div></div>,  <div class="name"><div class="surname">Hallin</div> <div class="given-names">P.F.</div></div>. </div><div class="article-title">Genome update: annotation quality in sequenced microbial genomes</div>, <div class="source">Microbiol. SGM</div> , <div class="year">2004</div>, vol. <div class="volume">150</div> (pg. <div class="fpage">2015</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20update%3A%20annotation%20quality%20in%20sequenced%20microbial%20genomes&amp;author=D.W.%20Ussery&amp;author=P.F.%20Hallin&amp;publication_year=2004&amp;journal=Microbiol.%20SGM&amp;volume=150&amp;pages=2015-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.27338-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.27338-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.27338-0"> </span></div> </div></div></div></div></div><div content-id="DSR030C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jones</div> <div class="given-names">C.E.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Baumann</div> <div class="given-names">U.</div></div>. </div><div class="article-title">Estimating the annotation error rate of curated GO database sequence annotations</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">170</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimating%20the%20annotation%20error%20rate%20of%20curated%20GO%20database%20sequence%20annotations&amp;author=C.E.%20Jones&amp;author=A.L.%20Brown&amp;author=U.%20Baumann&amp;publication_year=2007&amp;journal=BMC%20Bioinformatics&amp;volume=8&amp;pages=170" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-8-170" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-8-170" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-8-170"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17519041" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div>. </div><div class="article-title">Genome re-annotation: a wiki solution?</div>, <div class="source">Genome Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">102</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20re-annotation%3A%20a%20wiki%20solution%3F&amp;author=S.L.%20Salzberg&amp;publication_year=2007&amp;journal=Genome%20Biol.&amp;volume=8&amp;pages=102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2007-8-6-r102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2007-8-6-r102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2007-8-6-r102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17274839" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pallejà</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Harrington</div> <div class="given-names">E.D.</div></div>,  <div class="name"><div class="surname">Bork</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Large gene overlaps in prokaryotic genomes: result of functional constraints or mispredictions?</div>, <div class="source">BMC Genomics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">335</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large%20gene%20overlaps%20in%20prokaryotic%20genomes%3A%20result%20of%20functional%20constraints%20or%20mispredictions%3F&amp;author=A.%20Pallej%C3%A0&amp;author=E.D.%20Harrington&amp;author=P.%20Bork&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=335" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-335" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-335" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-335"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18627618" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nagy</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hegyi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Farkas</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Identification and correction of abnormal, incomplete and mispredicted proteins in public databases</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">353</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20correction%20of%20abnormal%2C%20incomplete%20and%20mispredicted%20proteins%20in%20public%20databases&amp;author=A.%20Nagy&amp;author=H.%20Hegyi&amp;author=K.%20Farkas&amp;publication_year=2008&amp;journal=BMC%20Bioinformatics&amp;volume=9&amp;pages=353" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-9-353" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-9-353" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-9-353"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18752676" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bakke</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Carney</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">DeLoache</div> <div class="given-names">W.</div></div>, et al. </div><div class="article-title">Evaluation of three automated genome annotations for <em>Halorhabdus utahensis</em></div>, <div class="source">PLoS One</div> , <div class="year">2009</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e6291</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evaluation%20of%20three%20automated%20genome%20annotations%20for%20Halorhabdus%20utahensis&amp;author=P.%20Bakke&amp;author=N.%20Carney&amp;author=W.%20DeLoache&amp;publication_year=2009&amp;journal=PLoS%20One&amp;volume=4&amp;pages=e6291" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0006291" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0006291" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0006291"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19617911" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Poptsova</div> <div class="given-names">M.S.</div></div>,  <div class="name"><div class="surname">Gogarten</div> <div class="given-names">J.P.</div></div>. </div><div class="article-title">Using comparative genome analysis to identify problems in annotated microbial genomes</div>, <div class="source">Microbiol. SGM</div> , <div class="year">2010</div>, vol. <div class="volume">156</div> (pg. <div class="fpage">1909</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20comparative%20genome%20analysis%20to%20identify%20problems%20in%20annotated%20microbial%20genomes&amp;author=M.S.%20Poptsova&amp;author=J.P.%20Gogarten&amp;publication_year=2010&amp;journal=Microbiol.%20SGM&amp;volume=156&amp;pages=1909-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.033811-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.033811-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.033811-0"> </span></div> </div></div></div></div></div><div content-id="DSR030C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">C.T.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Recognition of protein coding genes in the yeast genome at better than 95% accuracy based on the Z curve</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2000</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">2804</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recognition%20of%20protein%20coding%20genes%20in%20the%20yeast%20genome%20at%20better%20than%2095%25%20accuracy%20based%20on%20the%20Z%20curve&amp;author=C.T.%20Zhang&amp;author=J.%20Wang&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res.&amp;volume=28&amp;pages=2804-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.14.2804" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.14.2804" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.14.2804"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10908339" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Natale</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Shankavaram</div> <div class="given-names">U.T.</div></div>,  <div class="name"><div class="surname">Galperin</div> <div class="given-names">M.Y.</div></div>,  <div class="name"><div class="surname">Wolf</div> <div class="given-names">Y.I.</div></div>,  <div class="name"><div class="surname">Aravind</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Koonin</div> <div class="given-names">E.V.</div></div>. </div><div class="article-title">Towards understanding the first genome sequence of a crenarchaeon by genome annotation using clusters of orthologous groups of proteins (COGs)</div>, <div class="source">Genome Biol.</div> , <div class="year">2000</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">1</div>-<div class="lpage">19</div>) <div class="comment">research0009.</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Towards%20understanding%20the%20first%20genome%20sequence%20of%20a%20crenarchaeon%20by%20genome%20annotation%20using%20clusters%20of%20orthologous%20groups%20of%20proteins%20(COGs)&amp;author=D.A.%20Natale&amp;author=U.T.%20Shankavaram&amp;author=M.Y.%20Galperin&amp;author=Y.I.%20Wolf&amp;author=L.%20Aravind&amp;author=E.V.%20Koonin&amp;publication_year=2000&amp;journal=Genome%20Biol.&amp;volume=1&amp;pages=1-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2000-1-5-research0009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2000-1-5-research0009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2000-1-5-research0009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11178226" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">C.T.</div></div>. </div><div class="article-title">Identification of protein-coding genes in the genome of Vibriocholerae with more than 98% accuracy using occurrence frequencies of single nucleotides</div>, <div class="source">Eur. J. Biochem.</div> , <div class="year">2001</div>, vol. <div class="volume">268</div> (pg. <div class="fpage">4261</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20protein-coding%20genes%20in%20the%20genome%20of%20Vibriocholerae%20with%20more%20than%2098%25%20accuracy%20using%20occurrence%20frequencies%20of%20single%20nucleotides&amp;author=J.%20Wang&amp;author=C.T.%20Zhang&amp;publication_year=2001&amp;journal=Eur.%20J.%20Biochem.&amp;volume=268&amp;pages=4261-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1432-1327.2001.02341.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1432-1327.2001.02341.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1432-1327.2001.02341.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11488920" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Camus</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Pryor</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Médigue</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Cole</div> <div class="given-names">S.T.</div></div>. </div><div class="article-title">Re-annotation of the genome sequence of <em>Mycobacterium tuberculosis</em> H37Rv</div>, <div class="source">Microbiology</div> , <div class="year">2002</div>, vol. <div class="volume">148</div> (pg. <div class="fpage">2967</div>-<div class="lpage">73</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Re-annotation%20of%20the%20genome%20sequence%20of%20Mycobacterium%20tuberculosis%20H37Rv&amp;author=J.C.%20Camus&amp;author=M.J.%20Pryor&amp;author=C.%20M%C3%A9digue&amp;author=S.T.%20Cole&amp;publication_year=2002&amp;journal=Microbiology&amp;volume=148&amp;pages=2967-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00221287-148-10-2967" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00221287-148-10-2967" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00221287-148-10-2967"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12368430" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bocs</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Danchin</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Medigue</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Re-annotation of genome microbial CoDing-Sequences: finding new genes and inaccurately annotated genes</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2002</div>, vol. <div class="volume">3</div> pg. <div class="fpage">5</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Re-annotation%20of%20genome%20microbial%20CoDing-Sequences%3A%20finding%20new%20genes%20and%20inaccurately%20annotated%20genes&amp;author=S.%20Bocs&amp;author=A.%20Danchin&amp;author=C.%20Medigue&amp;publication_year=2002&amp;journal=BMC%20Bioinformatics&amp;volume=3&amp;pages=5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-3-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-3-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-3-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11879526" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ochman</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Distinguishing the ORFs from the ELFs:short bacterial genes and the annotation of genomes</div>, <div class="source">Trends Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">335</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distinguishing%20the%20ORFs%20from%20the%20ELFs%3Ashort%20bacterial%20genes%20and%20the%20annotation%20of%20genomes&amp;author=H.%20Ochman&amp;publication_year=2002&amp;journal=Trends%20Genet.&amp;volume=18&amp;pages=335-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(02)02668-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(02)02668-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(02)02668-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12127765" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">L.L.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">C.T.</div></div>. </div><div class="article-title">Gene recognition from questionable ORFs in bacterial and archaeal genomes</div>, <div class="source">J. Biomol. Struct. Dyn.</div> , <div class="year">2003</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">99</div>-<div class="lpage">109</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20recognition%20from%20questionable%20ORFs%20in%20bacterial%20and%20archaeal%20genomes&amp;author=L.L.%20Chen&amp;author=C.T.%20Zhang&amp;publication_year=2003&amp;journal=J.%20Biomol.%20Struct.%20Dyn.&amp;volume=21&amp;pages=99-109" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/07391102.2003.10506908" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F07391102.2003.10506908" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F07391102.2003.10506908"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12854962" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pruitt</div> <div class="given-names">K.D.</div></div>,  <div class="name"><div class="surname">Tatusova</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Maglott</div> <div class="given-names">D.R.</div></div>. </div><div class="article-title">NCBI reference sequence project: update and current status</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">34</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NCBI%20reference%20sequence%20project%3A%20update%20and%20current%20status&amp;author=K.D.%20Pruitt&amp;author=T.%20Tatusova&amp;author=D.R.%20Maglott&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=34-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12519942" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guo</div> <div class="given-names">F.B.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">C.T.</div></div>. </div><div class="article-title">Gene recognition based on nucleotide distribution of ORFs in a hyper-thermophilic crenarchaeon, <em>Aeropyrum pernix</em> K1</div>, <div class="source">DNA Res.</div> , <div class="year">2004</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">361</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20recognition%20based%20on%20nucleotide%20distribution%20of%20ORFs%20in%20a%20hyper-thermophilic%20crenarchaeon%2C%20Aeropyrum%20pernix%20K1&amp;author=F.B.%20Guo&amp;author=J.%20Wang&amp;author=C.T.%20Zhang&amp;publication_year=2004&amp;journal=DNA%20Res.&amp;volume=11&amp;pages=361-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/11.6.361" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F11.6.361" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F11.6.361"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15871459" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Silva</div> <div class="given-names">M.D.</div></div>,  <div class="name"><div class="surname">Upton</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Using purine skews to predict genes in AT-rich poxviruses</div>, <div class="source">BMC Genomics</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> pg. <div class="fpage">22</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20purine%20skews%20to%20predict%20genes%20in%20AT-rich%20poxviruses&amp;author=M.D.%20Silva&amp;author=C.%20Upton&amp;publication_year=2005&amp;journal=BMC%20Genomics&amp;volume=6&amp;pages=22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-6-22" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-6-22" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-6-22"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15720717" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guo</div> <div class="given-names">F.B.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">X.J.</div></div>. </div><div class="article-title">Re-prediction of protein coding genes in the genome of <em>Amsacta moorei entomopoxvirus</em></div>, <div class="source">J. Virol. Methods</div> , <div class="year">2007</div>, vol. <div class="volume">146</div> (pg. <div class="fpage">389</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Re-prediction%20of%20protein%20coding%20genes%20in%20the%20genome%20of%20Amsacta%20moorei%20entomopoxvirus&amp;author=F.B.%20Guo&amp;author=X.J.%20Yu&amp;publication_year=2007&amp;journal=J.%20Virol.%20Methods&amp;volume=146&amp;pages=389-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jviromet.2007.07.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jviromet.2007.07.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jviromet.2007.07.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17716751" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">L.L.</div></div>,  <div class="name"><div class="surname">Ma</div> <div class="given-names">B.G.</div></div>,  <div class="name"><div class="surname">Gao</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Reannotation of hypothetical ORFs in plant pathogen <em>Erwinia carotovora</em> subsp. <em>atroseptica</em> SCRI1043</div>, <div class="source">FEBS J.</div> , <div class="year">2008</div>, vol. <div class="volume">275</div> (pg. <div class="fpage">198</div>-<div class="lpage">206</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reannotation%20of%20hypothetical%20ORFs%20in%20plant%20pathogen%20Erwinia%20carotovora%20subsp.%20atroseptica%20SCRI1043&amp;author=L.L.%20Chen&amp;author=B.G.%20Ma&amp;author=N.%20Gao&amp;publication_year=2008&amp;journal=FEBS%20J.&amp;volume=275&amp;pages=198-206" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1742-4658.2007.06190.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1742-4658.2007.06190.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1742-4658.2007.06190.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18067578" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Luo</div> <div class="given-names">C.W.</div></div>,  <div class="name"><div class="surname">Hu</div> <div class="given-names">G.Q.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">H.Q.</div></div>. </div><div class="article-title">Genome reannotation of <em>Escherichia coli</em> CFT073 with new insights into virulence</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">552</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20reannotation%20of%20Escherichia%20coli%20CFT073%20with%20new%20insights%20into%20virulence&amp;author=C.W.%20Luo&amp;author=G.Q.%20Hu&amp;author=H.Q.%20Zhu&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=552" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-552" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-552" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-552"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19930606" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guo</div> <div class="given-names">F.B.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Identify protein coding genes in the genomes of <em>Aeropyrum pernix</em> K1 and <em>Chlorobium tepidum</em> TLS</div>, <div class="source">J. Biomol. Struct. Dyn.</div> , <div class="year">2009</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">413</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identify%20protein%20coding%20genes%20in%20the%20genomes%20of%20Aeropyrum%20pernix%20K1%20and%20Chlorobium%20tepidum%20TLS&amp;author=F.B.%20Guo&amp;author=Y.%20Lin&amp;publication_year=2009&amp;journal=J.%20Biomol.%20Struct.%20Dyn.&amp;volume=26&amp;pages=413-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/07391102.2009.10507256" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F07391102.2009.10507256" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F07391102.2009.10507256"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19108580" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Warren</div> <div class="given-names">A.S.</div></div>,  <div class="name"><div class="surname">Archuleta</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Feng</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Setubal</div> <div class="given-names">J.C.</div></div>. </div><div class="article-title">Missing genes in the annotation of prokaryotic genomes</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">131</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Missing%20genes%20in%20the%20annotation%20of%20prokaryotic%20genomes&amp;author=A.S.%20Warren&amp;author=J.%20Archuleta&amp;author=W.%20Feng&amp;author=J.C.%20Setubal&amp;publication_year=2010&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=131" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-11-131" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-11-131" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-11-131"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20230630" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gundogdu</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Bentley</div> <div class="given-names">S.D.</div></div>,  <div class="name"><div class="surname">Holden</div> <div class="given-names">M.T.</div></div>,  <div class="name"><div class="surname">Parkhill</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Dorrell</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Wren</div> <div class="given-names">B.W.</div></div>. </div><div class="article-title">Re-annotation and re-analysis of the <em>Campylobacter jejuni</em> NCTC11168 genome sequence</div>, <div class="source">BMC Genomics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">162</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Re-annotation%20and%20re-analysis%20of%20the%20Campylobacter%20jejuni%20NCTC11168%20genome%20sequence&amp;author=O.%20Gundogdu&amp;author=S.D.%20Bentley&amp;author=M.T.%20Holden&amp;author=J.%20Parkhill&amp;author=N.%20Dorrell&amp;author=B.W.%20Wren&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=162" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-162" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-162" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-162"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17565669" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ouzounis</div> <div class="given-names">C.A.</div></div>,  <div class="name"><div class="surname">Karp</div> <div class="given-names">P.D.</div></div>. </div><div class="article-title">The past, present and future of genome-wide re-annotation</div>, <div class="source">Genome Biol.</div> , <div class="year">2002</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">comment2001.1</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20past%2C%20present%20and%20future%20of%20genome-wide%20re-annotation&amp;author=C.A.%20Ouzounis&amp;author=P.D.%20Karp&amp;publication_year=2002&amp;journal=Genome%20Biol.&amp;volume=3&amp;pages=comment2001.1-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2002-3-2-comment2001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2002-3-2-comment2001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2002-3-2-comment2001"> </span></div> </div></div></div></div></div><div content-id="DSR030C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tech</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Merkl</div> <div class="given-names">R.</div></div>. </div><div class="article-title">YACOP: enhanced gene prediction obtained by a combination of existing methods</div>, <div class="source">In Silico Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">441</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=YACOP%3A%20enhanced%20gene%20prediction%20obtained%20by%20a%20combination%20of%20existing%20methods&amp;author=M.%20Tech&amp;author=R.%20Merkl&amp;publication_year=2003&amp;journal=In%20Silico%20Biol.&amp;volume=3&amp;pages=441-51" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14965344" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McHardy</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">Goesmann</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Pühler</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Meyer</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Development of joint application strategies for two microbial gene finders</div>, <div class="source">Bioinformatics</div> , <div class="year">2004</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1622</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20joint%20application%20strategies%20for%20two%20microbial%20gene%20finders&amp;author=A.C.%20McHardy&amp;author=A.%20Goesmann&amp;author=A.%20P%C3%BChler&amp;author=F.%20Meyer&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=1622-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bth137" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbth137" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbth137"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14988122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guo</div> <div class="given-names">F.B.</div></div>,  <div class="name"><div class="surname">Ou</div> <div class="given-names">H.Y.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">C.T.</div></div>. </div><div class="article-title">ZCURVE: a new system for recognizing protein coding genes in bacterial and archaeal genomes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">1780</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ZCURVE%3A%20a%20new%20system%20for%20recognizing%20protein%20coding%20genes%20in%20bacterial%20and%20archaeal%20genomes&amp;author=F.B.%20Guo&amp;author=H.Y.%20Ou&amp;author=C.T.%20Zhang&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=1780-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg254" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg254" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg254"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12626720" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hamori</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Ruskin</div> <div class="given-names">J.</div></div>. </div><div class="article-title">H curves, a novel method of representation of nucleotide series especially suited for long DNA sequences</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1983</div>, vol. <div class="volume">258</div> (pg. <div class="fpage">1318</div>-<div class="lpage">27</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=H%20curves%2C%20a%20novel%20method%20of%20representation%20of%20nucleotide%20series%20especially%20suited%20for%20long%20DNA%20sequences&amp;author=E.%20Hamori&amp;author=J.%20Ruskin&amp;publication_year=1983&amp;journal=J.%20Biol.%20Chem.&amp;volume=258&amp;pages=1318-27" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6822501" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">C.L.</div></div>,  <div class="name"><div class="surname">Liang</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Yin</div> <div class="given-names">C.C.</div></div>, et al. </div><div class="article-title">A novel construction of genome space with biological geometry</div>, <div class="source">DNA Res.</div> , <div class="year">2010</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">155</div>-<div class="lpage">68</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20construction%20of%20genome%20space%20with%20biological%20geometry&amp;author=C.L.%20Yu&amp;author=Q.%20Liang&amp;author=C.C.%20Yin&amp;publication_year=2010&amp;journal=DNA%20Res.&amp;volume=17&amp;pages=155-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsq008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsq008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsq008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20360268" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">J.F.</div></div>,  <div class="name"><div class="surname">Sun</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.H.</div></div>. </div><div class="article-title">TN curve: a novel 3D graphical representation of DNA sequence based on trinucleotides and its applications</div>, <div class="source">J. Theor. Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">261</div> (pg. <div class="fpage">459</div>-<div class="lpage">68</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TN%20curve%3A%20a%20novel%203D%20graphical%20representation%20of%20DNA%20sequence%20based%20on%20trinucleotides%20and%20its%20applications&amp;author=J.F.%20Yu&amp;author=X.%20Sun&amp;author=J.H.%20Wang&amp;publication_year=2009&amp;journal=J.%20Theor.%20Biol.&amp;volume=261&amp;pages=459-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jtbi.2009.08.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jtbi.2009.08.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jtbi.2009.08.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19679137" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">J. F.</div></div>,  <div class="name"><div class="surname">Sun</div> <div class="given-names">X.</div></div>. </div><div class="article-title">Reannotation of protein coding genes based on an improved graphical representation of DNA sequence</div>, <div class="source">J. Comput. Chem.</div> , <div class="year">2010</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">2126</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reannotation%20of%20protein%20coding%20genes%20based%20on%20an%20improved%20graphical%20representation%20of%20DNA%20sequence&amp;author=J.%20F.%20Yu&amp;author=X.%20Sun&amp;publication_year=2010&amp;journal=J.%20Comput.%20Chem.&amp;volume=31&amp;pages=2126-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/jcc.v31:11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fjcc.v31:11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fjcc.v31:11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20175214" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">C.T.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Analysis of distribution of bases in the coding sequences by a diagrammatic technique</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1991</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">6313</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20distribution%20of%20bases%20in%20the%20coding%20sequences%20by%20a%20diagrammatic%20technique&amp;author=C.T.%20Zhang&amp;author=R.%20Zhang&amp;publication_year=1991&amp;journal=Nucleic%20Acids%20Res.&amp;volume=19&amp;pages=6313-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/19.22.6313" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F19.22.6313" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F19.22.6313"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1956790" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">J.F.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">Sun</div> <div class="given-names">X.</div></div>. </div><div class="article-title">Analysis of similarities/dissimilarities of DNA sequences based on a novel graphical representation</div>, <div class="source">MATCH Commun. Math. Comput. Chem.</div> , <div class="year">2010</div>, vol. <div class="volume">63</div> (pg. <div class="fpage">493</div>-<div class="lpage">512</div>)</div></div></div></div><div content-id="DSR030C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gao</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">C.T.</div></div>. </div><div class="article-title">Comparison of various algorithms for recognizing short coding sequences of human genes</div>, <div class="source">Bioinformatics</div> , <div class="year">2004</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">673</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20various%20algorithms%20for%20recognizing%20short%20coding%20sequences%20of%20human%20genes&amp;author=F.%20Gao&amp;author=C.T.%20Zhang&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=673-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg467" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg467" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg467"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14764563" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gupta</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Majumdar</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bhattacharya</div> <div class="given-names">T.K.</div></div>,  <div class="name"><div class="surname">Ghosh</div> <div class="given-names">T.C.</div></div>. </div><div class="article-title">Studies on the relationships between the synonymous codon usage and protein secondary structural units</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">2000</div>, vol. <div class="volume">269</div> (pg. <div class="fpage">692</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Studies%20on%20the%20relationships%20between%20the%20synonymous%20codon%20usage%20and%20protein%20secondary%20structural%20units&amp;author=S.K.%20Gupta&amp;author=S.%20Majumdar&amp;author=T.K.%20Bhattacharya&amp;author=T.C.%20Ghosh&amp;publication_year=2000&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=269&amp;pages=692-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/bbrc.2000.2351" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fbbrc.2000.2351" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fbbrc.2000.2351"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10720478" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chiusano</div> <div class="given-names">M.L.</div></div>,  <div class="name"><div class="surname">Alvarez-Valin</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Giulio</div> <div class="given-names">M.D.</div></div>, et al. </div><div class="article-title">Second codon positions of genes and the secondary structures of proteins. Relationships and implications for the origin of the genetic code</div>, <div class="source">Gene</div> , <div class="year">2000</div>, vol. <div class="volume">261</div> (pg. <div class="fpage">63</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Second%20codon%20positions%20of%20genes%20and%20the%20secondary%20structures%20of%20proteins.%20Relationships%20and%20implications%20for%20the%20origin%20of%20the%20genetic%20code&amp;author=M.L.%20Chiusano&amp;author=F.%20Alvarez-Valin&amp;author=M.D.%20Giulio&amp;publication_year=2000&amp;journal=Gene&amp;volume=261&amp;pages=63-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(00)00521-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(00)00521-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(00)00521-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11164038" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fickett</div> <div class="given-names">J.W.</div></div>,  <div class="name"><div class="surname">Tung</div> <div class="given-names">C.S.</div></div>. </div><div class="article-title">Assessment of protein coding measures</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1992</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">6441</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessment%20of%20protein%20coding%20measures&amp;author=J.W.%20Fickett&amp;author=C.S.%20Tung&amp;publication_year=1992&amp;journal=Nucleic%20Acids%20Res.&amp;volume=20&amp;pages=6441-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/20.24.6441" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F20.24.6441" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F20.24.6441"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1480466" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Burset</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Guigo</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Evaluation of gene structure prediction programs</div>, <div class="source">Genomics</div> , <div class="year">1996</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">353</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evaluation%20of%20gene%20structure%20prediction%20programs&amp;author=M.%20Burset&amp;author=R.%20Guigo&amp;publication_year=1996&amp;journal=Genomics&amp;volume=34&amp;pages=353-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/geno.1996.0298" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fgeno.1996.0298" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fgeno.1996.0298"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8786136" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Forsdyke</div> <div class="given-names">D.R.</div></div>,  <div class="name"><div class="surname">Mortimer</div> <div class="given-names">J.R.</div></div>. </div><div class="article-title">Chargaff's legacy</div>, <div class="source">Gene</div> , <div class="year">2000</div>, vol. <div class="volume">261</div> (pg. <div class="fpage">127</div>-<div class="lpage">37</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chargaff's%20legacy&amp;author=D.R.%20Forsdyke&amp;author=J.R.%20Mortimer&amp;publication_year=2000&amp;journal=Gene&amp;volume=261&amp;pages=127-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(00)00472-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(00)00472-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(00)00472-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11164044" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Muto</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Osawa</div> <div class="given-names">S.</div></div>. </div><div class="article-title">The guanine and cytosine content of genomic DNA and bacterial evolution</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1987</div>, vol. <div class="volume">84</div> (pg. <div class="fpage">166</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20guanine%20and%20cytosine%20content%20of%20genomic%20DNA%20and%20bacterial%20evolution&amp;author=A.%20Muto&amp;author=S.%20Osawa&amp;publication_year=1987&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=84&amp;pages=166-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.84.1.166" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.84.1.166" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.84.1.166"> </span></div> </div></div></div></div></div><div content-id="DSR030C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mooers</div> <div class="given-names">A.Ø.</div></div>,  <div class="name"><div class="surname">Holmes</div> <div class="given-names">E.C.</div></div>. </div><div class="article-title">The evolution of base composition and phylogenetic inference</div>, <div class="source">Trends Ecol. Evol.</div> , <div class="year">2000</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">365</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20base%20composition%20and%20phylogenetic%20inference&amp;author=A.%C3%98.%20Mooers&amp;author=E.C.%20Holmes&amp;publication_year=2000&amp;journal=Trends%20Ecol.%20Evol.&amp;volume=9&amp;pages=365-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0169-5347(00)01934-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0169-5347(00)01934-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0169-5347(00)01934-0"> </span></div> </div></div></div></div></div><div content-id="DSR030C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Heidelberg</div> <div class="given-names">J.F.</div></div>,  <div class="name"><div class="surname">Eisen</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Neison</div> <div class="given-names">W.C.</div></div>, et al. </div><div class="article-title">DNA sequence of both chromosomes of the cholera pathogen <em>Vibrio cholerae</em></div>, <div class="source">Nature</div> , <div class="year">2000</div>, vol. <div class="volume">406</div> (pg. <div class="fpage">477</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20sequence%20of%20both%20chromosomes%20of%20the%20cholera%20pathogen%20Vibrio%20cholerae&amp;author=J.F.%20Heidelberg&amp;author=J.A.%20Eisen&amp;author=W.C.%20Neison&amp;publication_year=2000&amp;journal=Nature&amp;volume=406&amp;pages=477-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35020000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35020000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35020000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10952301" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharp</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div>. </div><div class="article-title">Codon usage in regulatory genes in <em>Escherichia coli</em> does not reflect selection for ‘rare’ codons</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1986</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">7737</div>-<div class="lpage">49</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20in%20regulatory%20genes%20in%20Escherichia%20coli%20does%20not%20reflect%20selection%20for%20%E2%80%98rare%E2%80%99%20codons&amp;author=P.M.%20Sharp&amp;author=W.H.%20Li&amp;publication_year=1986&amp;journal=Nucleic%20Acids%20Res.&amp;volume=14&amp;pages=7737-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/14.19.7737" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F14.19.7737" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F14.19.7737"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3534792" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Trifonov</div> <div class="given-names">E.N.</div></div>. </div><div class="article-title">Translation framing code and frame-monitoring mechanism as suggested by the analysis of mRNA and 16 S rRNA nucleotide sequences</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1987</div>, vol. <div class="volume">194</div> (pg. <div class="fpage">643</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translation%20framing%20code%20and%20frame-monitoring%20mechanism%20as%20suggested%20by%20the%20analysis%20of%20mRNA%20and%2016%20S%20rRNA%20nucleotide%20sequences&amp;author=E.N.%20Trifonov&amp;publication_year=1987&amp;journal=J.%20Mol.%20Biol.&amp;volume=194&amp;pages=643-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(87)90241-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(87)90241-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(87)90241-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2443708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pruitt</div> <div class="given-names">K.D.</div></div>,  <div class="name"><div class="surname">Tatusova</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Maglott</div> <div class="given-names">D.R.</div></div>. </div><div class="article-title">NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">61</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NCBI%20reference%20sequences%20(RefSeq)%3A%20a%20curated%20non-redundant%20sequence%20database%20of%20genomes%2C%20transcripts%20and%20proteins&amp;author=K.D.%20Pruitt&amp;author=T.%20Tatusova&amp;author=D.R.%20Maglott&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=61-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl842" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl842" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl842"> </span></div> </div></div></div></div></div><div content-id="DSR030C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tatusov</div> <div class="given-names">R.L.</div></div>,  <div class="name"><div class="surname">Galperin</div> <div class="given-names">M.Y.</div></div>,  <div class="name"><div class="surname">Natale</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Koonin</div> <div class="given-names">E.V.</div></div>. </div><div class="article-title">The COG database: a tool for genome-scale analysis of protein functions and evolution</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2000</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">33</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20COG%20database%3A%20a%20tool%20for%20genome-scale%20analysis%20of%20protein%20functions%20and%20evolution&amp;author=R.L.%20Tatusov&amp;author=M.Y.%20Galperin&amp;author=D.A.%20Natale&amp;author=E.V.%20Koonin&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res.&amp;volume=28&amp;pages=33-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.1.33" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.1.33" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.1.33"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10592175" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nielsen</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Krogh</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Large-scale prokaryotic gene prediction and comparison to genome annotation</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">4322</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20prokaryotic%20gene%20prediction%20and%20comparison%20to%20genome%20annotation&amp;author=P.%20Nielsen&amp;author=A.%20Krogh&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=4322-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti701" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti701" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti701"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16249266" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koonin</div> <div class="given-names">E.V.</div></div>,  <div class="name"><div class="surname">Wolf</div> <div class="given-names">Y.I.</div></div>. </div><div class="article-title">Genomics of bacteria and archaea: the emerging dynamic view of the prokaryotic world</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">6688</div>-<div class="lpage">719</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomics%20of%20bacteria%20and%20archaea%3A%20the%20emerging%20dynamic%20view%20of%20the%20prokaryotic%20world&amp;author=E.V.%20Koonin&amp;author=Y.I.%20Wolf&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=6688-719" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn668" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn668" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn668"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18948295" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garcia-Vallve</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Guzman</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Montero</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Romeu</div> <div class="given-names">A.</div></div>. </div><div class="article-title">HGT-DB: a database of putative horizontally transferred genes in prokaryotic complete genomes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">187</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HGT-DB%3A%20a%20database%20of%20putative%20horizontally%20transferred%20genes%20in%20prokaryotic%20complete%20genomes&amp;author=S.%20Garcia-Vallve&amp;author=E.%20Guzman&amp;author=M.A.%20Montero&amp;author=A.%20Romeu&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=187-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12519978" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ragan</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Harlow</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Beiko</div> <div class="given-names">R.G.</div></div>. </div><div class="article-title">Do different surrogate methods detect lateral genetic transfer events of different relative ages?</div>, <div class="source">Trends Microbiol.</div> , <div class="year">2006</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">4</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Do%20different%20surrogate%20methods%20detect%20lateral%20genetic%20transfer%20events%20of%20different%20relative%20ages%3F&amp;author=M.A.%20Ragan&amp;author=T.J.%20Harlow&amp;author=R.G.%20Beiko&amp;publication_year=2006&amp;journal=Trends%20Microbiol.&amp;volume=14&amp;pages=4-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tim.2005.11.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tim.2005.11.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tim.2005.11.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16356716" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vernikos</div> <div class="given-names">G.S.</div></div>,  <div class="name"><div class="surname">Thomson</div> <div class="given-names">N.R.</div></div>,  <div class="name"><div class="surname">Parkhill</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Genetic flux over time in the Salmonella lineage</div>, <div class="source">Genome Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">R100</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20flux%20over%20time%20in%20the%20Salmonella%20lineage&amp;author=G.S.%20Vernikos&amp;author=N.R.%20Thomson&amp;author=J.%20Parkhill&amp;publication_year=2007&amp;journal=Genome%20Biol.&amp;volume=8&amp;pages=R100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2007-8-6-r100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2007-8-6-r100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2007-8-6-r100"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17547764" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Monier</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Claverie</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Ogata</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Horizontal gene transfer and nucleotide compositional anomaly in large DNA viruses</div>, <div class="source">BMC Genomics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">456</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Horizontal%20gene%20transfer%20and%20nucleotide%20compositional%20anomaly%20in%20large%20DNA%20viruses&amp;author=A.%20Monier&amp;author=J.%20Claverie&amp;author=H.%20Ogata&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=456" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-456" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-456" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-456"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18070355" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Azad</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Lawrence</div> <div class="given-names">J.G.</div></div>. </div><div class="article-title">Detecting laterally transferred genes: use of entropic clustering methods and genome position</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">4629</div>-<div class="lpage">39</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detecting%20laterally%20transferred%20genes%3A%20use%20of%20entropic%20clustering%20methods%20and%20genome%20position&amp;author=R.K.%20Azad&amp;author=J.G.%20Lawrence&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=4629-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm204" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm204" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm204"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17591616" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garcia-Vallve</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Romeu</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Palau</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Horizontal gene transfer in bacterial and archaeal complete genomes</div>, <div class="source">Genome Res.</div> , <div class="year">2000</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">1719</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Horizontal%20gene%20transfer%20in%20bacterial%20and%20archaeal%20complete%20genomes&amp;author=S.%20Garcia-Vallve&amp;author=A.%20Romeu&amp;author=J.%20Palau&amp;publication_year=2000&amp;journal=Genome%20Res.&amp;volume=10&amp;pages=1719-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.130000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.130000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.130000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11076857" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Delcher</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Harmon</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Kasif</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">White</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div>. </div><div class="article-title">Improved microbial gene identification with GLIMMER</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1999</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">4636</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20microbial%20gene%20identification%20with%20GLIMMER&amp;author=A.L.%20Delcher&amp;author=D.%20Harmon&amp;author=S.%20Kasif&amp;author=O.%20White&amp;author=S.L.%20Salzberg&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=4636-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.23.4636" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.23.4636" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.23.4636"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10556321" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR030C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR030C61" class="ref-content "><div class="label -label">61</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lukashin</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Borodovsky</div> <div class="given-names">M.</div></div>. </div><div class="article-title">GeneMark.hmm: new solutions for gene finding</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1998</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">1107</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GeneMark.hmm%3A%20new%20solutions%20for%20gene%20finding&amp;author=A.%20Lukashin&amp;author=M.%20Borodovsky&amp;publication_year=1998&amp;journal=Nucleic%20Acids%20Res.&amp;volume=26&amp;pages=1107-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.4.1107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.4.1107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.4.1107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9461475" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2011. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=477266&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/dsr030-dsr030supp_table9.xls?Expires=2147483647&amp;Signature=gHX~WAC42ILuoLi~WUuLRiV2EJSmtvHh4KcUzq0I1dIGG0ZtTRmgGjjrJgk97YW-CURTNyMbOwQsOh7lpM-w87zEUQz173g6GU86Bt2kT7W4dPhW~yqjmmggUY08X15MCSOli-ldDfGGVXp13HcsbCiElQZU3QPRMdUcmN77-Ml6xFYhiFYk4wd9L~QabuzqUHl61aWKHeLXYg27NNVtY-WwzMOBe~zX2VlTsPzdV9PV4jv7Yr7bJKqgkVzybYfd~v2eHLoHwdvaR2OWln~aC8iCNkmFSPFJJPiwI7Ndef21dFLu-xHA50BfyLePoBOpeEIuw5VsLqakOfZ6UOT3tg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-9</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/dsr030-dsr030supp_table8.doc?Expires=2147483647&amp;Signature=RjygNXf9C6UKGjQey0t2jqTgdxsQZTzJls5pz8yG6zLKWdGKx9DU5sXsP71ljpMPNkKE55E8NlfzmCToXvoIFPmNVo5FGCqzI9-vhBp4YTxfWF96v~tvc5FjtQih0T4qg1gcuRtsqARvJVpP4~nc-SYIP6gYDQ1OEwmZ4f3SL-pGjWPWmtpnQIjF3~lfrv5vGRb0nACuJRtOg84BVsCHfoz-SluPxrlgCmJBhdWbZmp4wYDro5X3Fg8AG-xOdc1Q9bP8i1dkMgyHqpP8-mZwCgQB51DopAbbLgqAIcVnAjMyu~nlqVCpdNNJTcR3qI5vS7L1l8XJWriuNEgKBBmJjw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/dsr030-dsr030supp_table7.doc?Expires=2147483647&amp;Signature=vQJ6mx8XLnXI91EUmRy62WLVaG2r3UcHZ2WcE15w1Ll5HcyxsNQ82f-wEinLuo5DHZy9KFSrM1hskETn2L9TVJP9aSinzM8cOE0PQ0bEluSS0x7WZT706defZG3u8D-wIBQEvyVg7L6BwZGbHlDXbjA1K3sNTxsT~k3DAU0CWzmK~I9GbFo2q8AlgHcJC28QqLdU6qsSgvdRjXy-~Yz8PQdjeelJQ8jVd9DiBMjQ1xQFA3eTCmW78F5Vmt5NTRUpYp10joYbnkSf9RK9L68OWqlK7g0Fq-SsdzICX5PbLiw6x38NWMKtNCTxaQ0YpYVZMuPPwbvlUkTdhSD~goX0vg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/dsr030-dsr030supp_table6.doc?Expires=2147483647&amp;Signature=aFKY6rmwWK8AZtXgbsJMRPt-Y0WGtmlPKOJCQ7RVfvsI0nm-qCesfZoAEPiUKiSFHn8FO-3o89ZLwLElXT80zNt3pyXUtuEeW1bdS00WounA~ekP7jyor0A9xdaO9W3F2IlS-hvj6lqwODleUKxUvO46IeQzMwuVmd0zJFZozrRthAzNlAbdq9dK~UVsWE3u1fLFT81nt63vkpUAbkB9jmmtmTSXsZGDHOHyd2yhfUfJnqoFAdSbRkVgsNEE3~yDAosij7HFMHrL4Nkwgv966iFE4hH-QzNGDa2-C5nO7gq--ai81pAq4yLLII~IR-haONN6YuVziO6z8Is7cj59iA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/dsr030-dsr030supp_table5.doc?Expires=2147483647&amp;Signature=QlYzKj1oIA2BEbtYRDXd5DLapPs9PdVPrRKz42c3rzD3aD0FShzn3ZOF6lMvQUGfJt7JXIKAaMF7mfzSgal20vHrP8JRIySdPIzoZeb4QT-o-zgp2eE4btsW~E7yNWYkqzlyYfy1fyOKDdsPH~tZ1kGdo2TWwVNcH9kLL0irHhWSx9kPI2~QV3iuL4HhXuOWZwV6Ennnx48kWSWS6sAIqsyi0lCKYZyHXnYvuz6DR5va6HWGO6bsqAsc8SPJzr4VigncqGDTDv7QyXSFzFNkzKJJKqlGGFFd94oMTEli5ItxQ1-Msz919bTf4GG4gOrtNjI9xEGveHCnaZKPh3hSXw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/dsr030-dsr030supp_table4.doc?Expires=2147483647&amp;Signature=amXRUvJ6EgSmM1rNIPIb7rV8Qb3JJbwijnn6ewIHwXgQ4ttOajZt7i1HO0XFfafxtGRSM4VFvK3o0~prpsl3hUiw1M5h05v2zJM5444psdLxL-y4~AbLkiVzfvFOR2yCu9P~JjkGUr90hy6P~HjFFUHO92LpLtUmKWcbW4RQUZBZ3Ko6WKQHEVvUGyHY0bOxrJm9DPGeosE6wwZOMo-3gm03qm2pQSVFJiUaclhHS20ulapieJMTzDJC50HpxcvvTJ2gZO9t4bV8Z~ZSzp9iZ9EfB46h~bk8ZftgtFF4my3gOFsbEA4mDSf6-1PlD8tOIMt4CuVD218VRtxnN8RtEA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/dsr030-dsr030supp_table3.doc?Expires=2147483647&amp;Signature=v9K-Z3nfyOkdQwOJkqCI1uwl2wuGp7yP-xHKlxJXju8Mw9m32PHo1x9GqX5grnGAXH8nJzD~0AQebm1cOhmvkuqZmdtAJ6PBUkwQsUBOZsvqVhdyLnP~Tpld2t30AnkpmefZvlwa77Pvh8xgqrJGV1DsAsvV-bpEwcxvkBk7f-OpHM8Eegf41Wo70jfYkvQcaTCQei-jwI0jl89zxyBjSkYVwctq1Txw4FIXobSZX0bplKVaxiEij-DjsTpHgjkCH5fKqDUOWmNnNhcGz~Q5tpE1Gmki1C~6y7Iv8LJbv7uJ-neyl85SVFgXNFEaLyRcnp0Y-8O6rmALlrhHimrweA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/dsr030-dsr030supp_table2.xls?Expires=2147483647&amp;Signature=T0pVNez700FL51o~GkuBXassFejEnDO5hnXdOXIkGGsKEo7XAuLl8ufyNDEgf-wrrYxD5aZ~7sfdNkkVbo~eXhSJhu7qpMREE6OREgw91ph5cFpNJ49168yqUz-iZyT~BLa6un3qobHNpkSvFDFkP2pWyTvhRp7W5oSyExSWUUWsYrK2A0ofFDa4s4-a8QvlVMrySp7Q2S8pnvB2sEk5ZdYKiVPCEn8fdFcvNe8KjBBuIS3WO4s2MH4ZAwNGX6WaetEeGmG7LhKfzc4Wd7d2E9tvJUebt9ZUA4nYnhbnRpvJjkEKUAPKQKNSO1PB146MP7w413PC59RKZBkMFyqo4Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr030/2/dsr030-dsr030supp_table1.doc?Expires=2147483647&amp;Signature=wiPxaGOddNy~kKdNrgR0RohGhYpQxU9FJzyxuIJFDuAcVPRP9lBVDqRPppwzWVlzmyQ-F8smvs27EQhX6YU~iDdtWwqKsQwWsRDQa1WC-mT3KxSBdAL1kojO~xdrmHmvL8G6OKSAscXo2IaHAIrsmvxTkfC2HgmTp5yjNxLeFHAns20kDiaUasx94Os1RP0xm-5Dc-MeVuFOt0owe0f3VMFMJfPu2wl447~vKnXu~g4mvw4UgYd05qn~vBMYAn2uPQt8G7osA5kj6zX2C~TUnLDp2Vp9zu~FpHjpchegT7tCs6LpleNNrTzxMWaVjeZMf5q0Tv5BDRE~h~2sBcpDeA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">150</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000297990700003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">14</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsr030"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="477266"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000297990700003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f18%2f6%2f435%2f477266">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000297990700003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (14)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F18%2F6%2F435%2F477266" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/477266'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Integrative Method for Identifying the Over-Annotated Protein-Coding Genes in Microbial Genomes" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsr030");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f18%2f6%2f435%2f477266">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

