

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/2/169/404005","@type":"ScholarlyArticle","name":"Dissection of the Octoploid Strawberry Genome by Deep Sequencing of the Genomes of  Fragaria  Species","datePublished":"2013-11-26","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2014-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst049","keywords":["<italic>Fragaria</italic>x <italic>ananassa</italic>","wild <italic>Fragaria</italic> species","genome sequence assembly","comparative analysis","polyploidy"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/2/169/404005","author":[{"name":"Hirakawa, Hideki","affiliation":"1  Kazusa DNA Research Institute ,  Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Shirasawa, Kenta","affiliation":"1  Kazusa DNA Research Institute ,  Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kosugi, Shunichi","affiliation":"1  Kazusa DNA Research Institute ,  Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Tashiro, Kosuke","affiliation":"2  Department of Bioscience and Biotechnology, Faculty of Agriculture ,  Kyushu University ,  Higashi-ku, Fukuoka 812-8581 ,  Japan","@type":"Person"},{"name":"Nakayama, Shinobu","affiliation":"1  Kazusa DNA Research Institute ,  Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Yamada, Manabu","affiliation":"1  Kazusa DNA Research Institute ,  Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kohara, Mistuyo","affiliation":"1  Kazusa DNA Research Institute ,  Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Watanabe, Akiko","affiliation":"1  Kazusa DNA Research Institute ,  Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kishida, Yoshie","affiliation":"1  Kazusa DNA Research Institute ,  Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Fujishiro, Tsunakazu","affiliation":"1  Kazusa DNA Research Institute ,  Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Tsuruoka, Hisano","affiliation":"1  Kazusa DNA Research Institute ,  Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Minami, Chiharu","affiliation":"1  Kazusa DNA Research Institute ,  Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Sasamoto, Shigemi","affiliation":"1  Kazusa DNA Research Institute ,  Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kato, Midori","affiliation":"1  Kazusa DNA Research Institute ,  Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Nanri, Keiko","affiliation":"1  Kazusa DNA Research Institute ,  Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Komaki, Akiko","affiliation":"1  Kazusa DNA Research Institute ,  Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Yanagi, Tomohiro","affiliation":"3  Faculty of Agriculture ,  Kagawa University ,  2393 Ikenobe, Miki, Kita, Kagawa 761-0795 ,  Japan","@type":"Person"},{"name":"Guoxin, Qin","affiliation":"4  Chongqing Nanshan Botanical Garden, No. 101, Nanshan Garden Road, Nan'an District, Chongqing 400065 ,  China","@type":"Person"},{"name":"Maeda, Fumi","affiliation":"5  Chiba Prefectural Agriculture and Forestry Research Center ,  Crop Breeding Institute ,  Chousei, Daizenno-Cyou 808, Midori, Chiba 299-4335 ,  Japan","@type":"Person"},{"name":"Ishikawa, Masami","affiliation":"5  Chiba Prefectural Agriculture and Forestry Research Center ,  Crop Breeding Institute ,  Chousei, Daizenno-Cyou 808, Midori, Chiba 299-4335 ,  Japan","@type":"Person"},{"name":"Kuhara, Satoru","affiliation":"2  Department of Bioscience and Biotechnology, Faculty of Agriculture ,  Kyushu University ,  Higashi-ku, Fukuoka 812-8581 ,  Japan","@type":"Person"},{"name":"Sato, Shusei","affiliation":"1  Kazusa DNA Research Institute ,  Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818 ,  Japan    6  Graduate School of Life Sciences ,  Tohoku University ,  Katahira 2-1-1, Aoba, Sendai, Miyagi 980-8577 ,  Japan","@type":"Person"},{"name":"Tabata, Satoshi","affiliation":"1  Kazusa DNA Research Institute ,  Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Isobe, Sachiko N.","affiliation":"1  Kazusa DNA Research Institute ,  Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"}],"description":"Abstract.  Cultivated strawberry (Fragaria x ananassa) is octoploid and shows allogamous behaviour. The present study aims at dissecting this octoploid genome t","pageStart":"169","pageEnd":"181","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/2/1/m_cover.gif?Expires=2147483647&Signature=C2ZH5dkUQEctPPta7W831-4mGKGGgEq~Aqbfql5zal9lq-CH7ZJcQtnaLb-2~TFW5o1XuWK-If8uoA0C0x3E8Xqh7vx6q7XeJXntpLTv0xMKSiGPs8b5JT8WwzNVzF-MVRjRIurpCdYeMF6bhbaWGNZLajJ34JAWHyXvGj72zxR2I~0JkQXfyp1Fd2IS9rmTmm3EZ-Vtcf0zL2z0kWasBqYnVob3EwZTj2l448hvYoYYZgYfsgU2j1vdpw3GUsXrCjCI5GsZ1ht-EZrYHZzJ3zkCMOPIc~J0lFrmNiix6fEEbm~y5NNGWoTY3GIiDMTmJpXRxBN715elzocAQQfDUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Dissection of the Octoploid Strawberry Genome by Deep Sequencing of the Genomes of  Fragaria  Species","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/2/1/m_cover.gif?Expires=2147483647&Signature=C2ZH5dkUQEctPPta7W831-4mGKGGgEq~Aqbfql5zal9lq-CH7ZJcQtnaLb-2~TFW5o1XuWK-If8uoA0C0x3E8Xqh7vx6q7XeJXntpLTv0xMKSiGPs8b5JT8WwzNVzF-MVRjRIurpCdYeMF6bhbaWGNZLajJ34JAWHyXvGj72zxR2I~0JkQXfyp1Fd2IS9rmTmm3EZ-Vtcf0zL2z0kWasBqYnVob3EwZTj2l448hvYoYYZgYfsgU2j1vdpw3GUsXrCjCI5GsZ1ht-EZrYHZzJ3zkCMOPIc~J0lFrmNiix6fEEbm~y5NNGWoTY3GIiDMTmJpXRxBN715elzocAQQfDUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Dissection of the Octoploid Strawberry Genome by Deep Sequencing of the Genomes of  Fragaria  Species" />
    <meta property="og:description" content="Abstract.  Cultivated strawberry (Fragaria x ananassa) is octoploid and shows allogamous behaviour. The present study aims at dissecting this octoploid genome t" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/2/169/404005" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-11-26" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Hirakawa, Hideki" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Shirasawa, Kenta" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kosugi, Shunichi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Tashiro, Kosuke" /><meta name="citation_author_institution" content="Department of Bioscience and Biotechnology, Faculty of Agriculture, Kyushu University, Higashi-ku, Fukuoka 812-8581, Japan" /><meta name="citation_author" content="Nakayama, Shinobu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Yamada, Manabu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kohara, Mistuyo" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Watanabe, Akiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kishida, Yoshie" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Fujishiro, Tsunakazu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Tsuruoka, Hisano" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Minami, Chiharu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Sasamoto, Shigemi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kato, Midori" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Nanri, Keiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Komaki, Akiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Yanagi, Tomohiro" /><meta name="citation_author_institution" content="Faculty of Agriculture, Kagawa University, 2393 Ikenobe, Miki, Kita, Kagawa 761-0795, Japan" /><meta name="citation_author" content="Guoxin, Qin" /><meta name="citation_author_institution" content="Chongqing Nanshan Botanical Garden, No. 101, Nanshan Garden Road, Nan'an District, Chongqing 400065, China" /><meta name="citation_author" content="Maeda, Fumi" /><meta name="citation_author_institution" content="Chiba Prefectural Agriculture and Forestry Research Center, Crop Breeding Institute, Chousei, Daizenno-Cyou 808, Midori, Chiba 299-4335, Japan" /><meta name="citation_author" content="Ishikawa, Masami" /><meta name="citation_author_institution" content="Chiba Prefectural Agriculture and Forestry Research Center, Crop Breeding Institute, Chousei, Daizenno-Cyou 808, Midori, Chiba 299-4335, Japan" /><meta name="citation_author" content="Kuhara, Satoru" /><meta name="citation_author_institution" content="Department of Bioscience and Biotechnology, Faculty of Agriculture, Kyushu University, Higashi-ku, Fukuoka 812-8581, Japan" /><meta name="citation_author" content="Sato, Shusei" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, Katahira 2-1-1, Aoba, Sendai, Miyagi 980-8577, Japan" /><meta name="citation_author" content="Tabata, Satoshi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Isobe, Sachiko N." /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_title" content="Dissection of the Octoploid Strawberry Genome by Deep Sequencing of the Genomes of Fragaria Species" /><meta name="citation_firstpage" content="169" /><meta name="citation_lastpage" content="181" /><meta name="citation_doi" content="10.1093/dnares/dst049" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2014/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="FAOSTAT 2011 http://faostat3.fao.org " /><meta name="citation_reference" content="citation_title=Estimation of the nuclear DNA content of strawberries (Fragaria spp.) compared with Arabidopsis thaliana by using dual system flow cytometry; Cytologia.;  citation_year=2001;  citation_volume=66;  citation_pages=431-6; " /><meta name="citation_reference" content="citation_title=Strawberry; citation_publisher=Springer-Verlag, Berlin; Genome Mapping and Molecular Breeding Inplants. vol. 4. Fruit and Nuts;  citation_year=2007;  citation_pages=189-205; " /><meta name="citation_reference" content="citation_title=Cross ability and phylogenetic relationships in the main European species of Fragaria; Natl. Acad. Sci. USSR;  citation_year=1946;  citation_volume=52;  citation_pages=545-7; " /><meta name="citation_reference" content="citation_title=Origin of the Fragaria polyploids. I. Cytological evidence; Am. J. Bot.;  citation_year=1967;  citation_volume=54;  citation_pages=221-8; " /><meta name="citation_reference" content="citation_title=Cytogenetics and evolution in American Fragaria; HortScience;  citation_year=1990;  citation_volume=25;  citation_pages=879-81; " /><meta name="citation_reference" content="citation_title=Studies using DNA markers in F. x ananassa: genetic analysis, genome structure, and cultivar identification; J. Jpn. Soc. Horticultural Sci..;  citation_year=2011;  citation_volume=80;  citation_pages=231-43; " /><meta name="citation_reference" content="citation_title=A microsatellite linkage map for the cultivated strawberry (Fragaria x ananassa) suggests extensive regions of homozygosity in the genome that may have resulted from breeding and selection; Theor. Appl. Genet.;  citation_year=2012;  citation_volume=124;  citation_pages=1229-40; " /><meta name="citation_reference" content="citation_title=Construction of an integrated high density simple sequence repeat linkage map in cultivated strawberry (Fragaria x ananassa) and its applicability; DNA Res.;  citation_year=2013;  citation_volume=20;  citation_pages=79-92; " /><meta name="citation_reference" content="citation_title=Strawberry biogeography, genetics and systematics; Acta Hort.;  citation_year=2009;  citation_volume=842;  citation_pages=71-84; " /><meta name="citation_reference" content="citation_title=Decaploidy in Fragaria iturupensis (Rosaceae); Am. J. Bot.;  citation_year=2009;  citation_volume=96;  citation_pages=713-6; " /><meta name="citation_reference" content="citation_publisher=Holt, Rinehart and Winston, New York; The Strawberry. History, Breeding and Physiology;  citation_year=1966; " /><meta name="citation_reference" content="citation_title=Phylogenetic relationships among species of Fragaria (Rosaceae) inferred from non-coding nuclear and chloroplast DNA sequences; Syst. Bot.;  citation_year=2000;  citation_volume=25;  citation_pages=337-48; " /><meta name="citation_reference" content="citation_title=Tracking the evolutionary history of polyploidy in Fragaria L. (strawberry): new insights from phylogenetic analyses of low-copy nuclear genes; Mol. Phylogenet. Evol.;  citation_year=2009;  citation_volume=51;  citation_pages=515-30; " /><meta name="citation_reference" content="citation_title=The genome of woodland strawberry (Fragaria vesca); Nat, Genet.;  citation_year=2010;  citation_volume=43;  citation_pages=109-16; " /><meta name="citation_reference" content="Fragaria vesca Whole Genome v1.1 Assembly &amp; Annotation Genome Database for Rosaceae (GDR) http://www.rosaceae.org/species/fragaria/fragaria_vesca/genome_v1.1 " /><meta name="citation_reference" content="citation_title=Mutation in TERMINAL FLOWER1 reverses the photoperiodic requirement for flowering in the wild strawberry Fragaria vesca; Plant Physiol.;  citation_year=2012;  citation_volume=159;  citation_pages=1043-54; " /><meta name="citation_reference" content="citation_title=Targeted sequence capture provides insight into genome structure and genetics of male sterility in a gynodioecious diploid strawberry, Fragaria vesca ssp. bracteata (Rosaceae); G3 (Bethesda);  citation_year=2013;  citation_volume=3;  citation_pages=1341-51; " /><meta name="citation_reference" content="citation_title=Genome-wide identification and comparative expression analysis of NBS-LRR-encoding genes upon Colletotrichum gloeosporioides infection in two ecotypes of Fragaria vesca; Gene.;  citation_year=2013;  citation_volume=527;  citation_pages=215-27; " /><meta name="citation_reference" content="citation_title=Current challenges in de novo plant genome sequencing and assembly; Genome Biol.;  citation_year=2012;  citation_volume=13;  citation_pages=243" /><meta name="citation_reference" content="citation_title=Advances in plant genome sequencing; Plant J.;  citation_year=2012;  citation_volume=70;  citation_pages=177-90; " /><meta name="citation_reference" content="citation_title=The genome of the domesticated apple (Malus domestica Borkh.); Nat. Genet.;  citation_year=2010;  citation_volume=42;  citation_pages=833-9; " /><meta name="citation_reference" content="citation_title=The genome of the pear (Pyrus bretschneideri Rehd.); Genome Res.;  citation_year=2012;  citation_volume=23;  citation_pages=396-408; " /><meta name="citation_reference" content="citation_title=The high-quality draft genome of peach (Prunus persica) identifies unique patterns of genetic diversity, domestication and genome evolution; citation_author=International Peach Genome Initiative; Nat. Genet.;  citation_year=2013;  citation_volume=45;  citation_pages=487-94; " /><meta name="citation_reference" content="citation_title=The genome of Prunus mume; Nat. Commun.;  citation_year=2012;  citation_volume=3;  citation_pages=1318" /><meta name="citation_reference" content="citation_title=Genome sequence and analysis of the tuber crop potato; citation_author=The Potato Genome Sequencing Consortium; Nature;  citation_year=2011;  citation_volume=475;  citation_pages=189-95; " /><meta name="citation_reference" content="citation_title=The draft genome of a diploid cotton Gossypium raimondii; Nat. Genet.;  citation_year=2012;  citation_volume=44;  citation_pages=1098-103; " /><meta name="citation_reference" content="citation_title=The banana (Musa acuminata) genome and the evolution of monocotyledonous plants; Nature;  citation_year=2012;  citation_volume=488;  citation_pages=213-7; " /><meta name="citation_reference" content="citation_title=GGT 2.0: Versatile software for visualization and analysis of genetic data; J. Heredity.;  citation_year=2008;  citation_volume=99;  citation_pages=232-6; " /><meta name="citation_reference" content="citation_title=Pvclust: an R package for assessing the uncertainty in hierarchical clustering; Bioinformatics.;  citation_year=2006;  citation_volume=22;  citation_pages=1540-2; " /><meta name="citation_reference" content="citation_title=A fast, lock-free approach for efficient parallel counting of occurrences of k-mers; Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=764-70; " /><meta name="citation_reference" content="citation_title=De novo assembly of human genomes with massively parallel short read sequencing; Genome Res.;  citation_year=2010;  citation_volume=20;  citation_pages=265-72; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=The genome of the fire ant Solenopsis invicta; Proc. Natl. Acad. Sci. USA;  citation_year=2011;  citation_volume=108;  citation_pages=5679-84; " /><meta name="citation_reference" content="citation_title=De novo identification of repeat families in large genomes;  citation_year=2005; " /><meta name="citation_reference" content="citation_title=Repeats in genomic dna: mining and meaning; Curr. Opin. Struct. Biol.;  citation_year=1998;  citation_volume=8;  citation_pages=333-7; " /><meta name="citation_reference" content="citation_title=SciRoKo: a new tool for whole genome microsatellite search and investigation; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=1683-5; " /><meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=955-64; " /><meta name="citation_reference" content="citation_title=Gene prediction with a hidden markov model and a new intron submodel; Bioinformatics.;  citation_year=2003;  citation_volume=19;  citation_issue=Suppl 2);  citation_pages=ii215-25; " /><meta name="citation_reference" content="citation_title=New developments in the InterPro database; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=D224-8; " /><meta name="citation_reference" content="citation_title=InterProScan: protein domains identifier; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=W116-20; " /><meta name="citation_reference" content="citation_title=Accelerated profile HMM searches; PLoS Comput. Biol.;  citation_year=2011;  citation_volume=7;  citation_pages=e1002195" /><meta name="citation_reference" content="citation_title=The Gypsy Database (GyDB) of mobile genetic elements: release 2.0; citation_author=Covelli; Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=D70-4; " /><meta name="citation_reference" content="citation_title=BLAT—the BLAST-like alignment tool; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=656-64; " /><meta name="citation_reference" content="citation_title=Improved tools for biological sequence comparison; Proc. Natl. Acad. Sci. USA;  citation_year=1988;  citation_volume=85;  citation_pages=2444-8; " /><meta name="citation_reference" content="citation_title=Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences; Bioinformatics;  citation_year=2006;  citation_volume=22;  citation_pages=1658-9; " /><meta name="citation_reference" content="citation_title=Mastering seeds for genomic size nucleotide BLAST searches; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=6935-41; " /><meta name="citation_reference" content="citation_title=Assembly algorithms for next-generation sequencing data; Genomics;  citation_year=2010;  citation_volume=95;  citation_pages=315-27; " /><meta name="citation_reference" content="citation_title=Review of general algorithmic features for genome assemblers for next generation sequencers; Genomics Proteomics Bioinformatics;  citation_year=2012;  citation_volume=10;  citation_pages=58-73; " /><meta name="citation_reference" content="citation_title=Conservation and loss of ribosomal RNA gene sites in diploid and polyploid Fragaria (Rosaceae); BMC Plant Biol.;  citation_year=2011;  citation_volume=11;  citation_pages=157" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/2/169/1128573/dst049.pdf" /><meta name="description" content="Abstract.  Cultivated strawberry (Fragaria x ananassa) is octoploid and shows allogamous behaviour. The present study aims at dissecting this octoploid genome t" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/2/169/404005" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/2/169/404005" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/2/169/404005");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst049';
    </script>
<title>
	Dissection of the Octoploid Strawberry Genome by Deep Sequencing of the Genomes of Fragaria Species | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./404005" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbqGATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGlyYWthd2EsIEhpZGVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCBLYXp1c2EtS2FtYXRhcmkgMi02LTcsIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGlyYXNhd2EsIEtlbnRhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIEthenVzYS1LYW1hdGFyaSAyLTYtNywgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Iktvc3VnaSwgU2h1bmljaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgS2F6dXNhLUthbWF0YXJpIDItNi03LCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFzaGlybywgS29zdWtlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9zY2llbmNlIGFuZCBCaW90ZWNobm9sb2d5LCBGYWN1bHR5IG9mIEFncmljdWx0dXJlLCBLeXVzaHUgVW5pdmVyc2l0eSwgSGlnYXNoaS1rdSwgRnVrdW9rYSA4MTItODU4MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWtheWFtYSwgU2hpbm9idSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCBLYXp1c2EtS2FtYXRhcmkgMi02LTcsIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZYW1hZGEsIE1hbmFidSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCBLYXp1c2EtS2FtYXRhcmkgMi02LTcsIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLb2hhcmEsIE1pc3R1eW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgS2F6dXNhLUthbWF0YXJpIDItNi03LCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iV2F0YW5hYmUsIEFraWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIEthenVzYS1LYW1hdGFyaSAyLTYtNywgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Iktpc2hpZGEsIFlvc2hpZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCBLYXp1c2EtS2FtYXRhcmkgMi02LTcsIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGdWppc2hpcm8sIFRzdW5ha2F6dSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCBLYXp1c2EtS2FtYXRhcmkgMi02LTcsIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUc3VydW9rYSwgSGlzYW5vIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIEthenVzYS1LYW1hdGFyaSAyLTYtNywgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1pbmFtaSwgQ2hpaGFydSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCBLYXp1c2EtS2FtYXRhcmkgMi02LTcsIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXNhbW90bywgU2hpZ2VtaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCBLYXp1c2EtS2FtYXRhcmkgMi02LTcsIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYXRvLCBNaWRvcmkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgS2F6dXNhLUthbWF0YXJpIDItNi03LCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFucmksIEtlaWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIEthenVzYS1LYW1hdGFyaSAyLTYtNywgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktvbWFraSwgQWtpa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgS2F6dXNhLUthbWF0YXJpIDItNi03LCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFuYWdpLCBUb21vaGlybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZhY3VsdHkgb2YgQWdyaWN1bHR1cmUsIEthZ2F3YSBVbml2ZXJzaXR5LCAyMzkzIElrZW5vYmUsIE1pa2ksIEtpdGEsIEthZ2F3YSA3NjEtMDc5NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHdW94aW4sIFFpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNob25ncWluZyBOYW5zaGFuIEJvdGFuaWNhbCBHYXJkZW4sIE5vLiAxMDEsIE5hbnNoYW4gR2FyZGVuIFJvYWQsIE5hbidhbiBEaXN0cmljdCwgQ2hvbmdxaW5nIDQwMDA2NSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNYWVkYSwgRnVtaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNoaWJhIFByZWZlY3R1cmFsIEFncmljdWx0dXJlIGFuZCBGb3Jlc3RyeSBSZXNlYXJjaCBDZW50ZXIsIENyb3AgQnJlZWRpbmcgSW5zdGl0dXRlLCBDaG91c2VpLCBEYWl6ZW5uby1DeW91IDgwOCwgTWlkb3JpLCBDaGliYSAyOTktNDMzNSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJc2hpa2F3YSwgTWFzYW1pIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2hpYmEgUHJlZmVjdHVyYWwgQWdyaWN1bHR1cmUgYW5kIEZvcmVzdHJ5IFJlc2VhcmNoIENlbnRlciwgQ3JvcCBCcmVlZGluZyBJbnN0aXR1dGUsIENob3VzZWksIERhaXplbm5vLUN5b3UgODA4LCBNaWRvcmksIENoaWJhIDI5OS00MzM1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikt1aGFyYSwgU2F0b3J1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9zY2llbmNlIGFuZCBCaW90ZWNobm9sb2d5LCBGYWN1bHR5IG9mIEFncmljdWx0dXJlLCBLeXVzaHUgVW5pdmVyc2l0eSwgSGlnYXNoaS1rdSwgRnVrdW9rYSA4MTItODU4MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXRvLCBTaHVzZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgS2F6dXNhLUthbWF0YXJpIDItNi03LCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIExpZmUgU2NpZW5jZXMsIFRvaG9rdSBVbml2ZXJzaXR5LCBLYXRhaGlyYSAyLTEtMSwgQW9iYSwgU2VuZGFpLCBNaXlhZ2kgOTgwLTg1NzcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFiYXRhLCBTYXRvc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIEthenVzYS1LYW1hdGFyaSAyLTYtNywgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Iklzb2JlLCBTYWNoaWtvIE4uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIEthenVzYS1LYW1hdGFyaSAyLTYtNywgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iRGlzc2VjdGlvbiBvZiB0aGUgT2N0b3Bsb2lkIFN0cmF3YmVycnkgR2Vub21lIGJ5IERlZXAgU2VxdWVuY2luZyBvZiB0aGUgR2Vub21lcyBvZiBGcmFnYXJpYSBTcGVjaWVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMTY5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIxODEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3QwNDkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNC8wNC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IkZBT1NUQVQgMjAxMSBodHRwOi8vZmFvc3RhdDMuZmFvLm9yZyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fc3RpbWF0aW9uIG9mIHRoZSBudWNsZWFyIEROQSBjb250ZW50IG9mIHN0cmF3YmVycmllcyAoRnJhZ2FyaWEgc3BwLikgY29tcGFyZWQgd2l0aCBBcmFiaWRvcHNpcyB0aGFsaWFuYSBieSB1c2luZyBkdWFsIHN5c3RlbSBmbG93IGN5dG9tZXRyeTsgQ3l0b2xvZ2lhLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT02NjsgIGNpdGF0aW9uX3BhZ2VzPTQzMS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJhd2JlcnJ5OyBjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXItVmVybGFnLCBCZXJsaW47IEdlbm9tZSBNYXBwaW5nIGFuZCBNb2xlY3VsYXIgQnJlZWRpbmcgSW5wbGFudHMuIHZvbC4gNC4gRnJ1aXQgYW5kIE51dHM7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl9wYWdlcz0xODktMjA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Dcm9zcyBhYmlsaXR5IGFuZCBwaHlsb2dlbmV0aWMgcmVsYXRpb25zaGlwcyBpbiB0aGUgbWFpbiBFdXJvcGVhbiBzcGVjaWVzIG9mIEZyYWdhcmlhOyBOYXRsLiBBY2FkLiBTY2kuIFVTU1I7ICBjaXRhdGlvbl95ZWFyPTE5NDY7ICBjaXRhdGlvbl92b2x1bWU9NTI7ICBjaXRhdGlvbl9wYWdlcz01NDUtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3JpZ2luIG9mIHRoZSBGcmFnYXJpYSBwb2x5cGxvaWRzLiBJLiBDeXRvbG9naWNhbCBldmlkZW5jZTsgQW0uIEouIEJvdC47ICBjaXRhdGlvbl95ZWFyPTE5Njc7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz0yMjEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3l0b2dlbmV0aWNzIGFuZCBldm9sdXRpb24gaW4gQW1lcmljYW4gRnJhZ2FyaWE7IEhvcnRTY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9ODc5LTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHVkaWVzIHVzaW5nIEROQSBtYXJrZXJzIGluIEYuIHggYW5hbmFzc2E6IGdlbmV0aWMgYW5hbHlzaXMsIGdlbm9tZSBzdHJ1Y3R1cmUsIGFuZCBjdWx0aXZhciBpZGVudGlmaWNhdGlvbjsgSi4gSnBuLiBTb2MuIEhvcnRpY3VsdHVyYWwgU2NpLi47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9ODA7ICBjaXRhdGlvbl9wYWdlcz0yMzEtNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbWljcm9zYXRlbGxpdGUgbGlua2FnZSBtYXAgZm9yIHRoZSBjdWx0aXZhdGVkIHN0cmF3YmVycnkgKEZyYWdhcmlhIHggYW5hbmFzc2EpIHN1Z2dlc3RzIGV4dGVuc2l2ZSByZWdpb25zIG9mIGhvbW96eWdvc2l0eSBpbiB0aGUgZ2Vub21lIHRoYXQgbWF5IGhhdmUgcmVzdWx0ZWQgZnJvbSBicmVlZGluZyBhbmQgc2VsZWN0aW9uOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEyNDsgIGNpdGF0aW9uX3BhZ2VzPTEyMjktNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBhbiBpbnRlZ3JhdGVkIGhpZ2ggZGVuc2l0eSBzaW1wbGUgc2VxdWVuY2UgcmVwZWF0IGxpbmthZ2UgbWFwIGluIGN1bHRpdmF0ZWQgc3RyYXdiZXJyeSAoRnJhZ2FyaWEgeCBhbmFuYXNzYSkgYW5kIGl0cyBhcHBsaWNhYmlsaXR5OyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTc5LTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJhd2JlcnJ5IGJpb2dlb2dyYXBoeSwgZ2VuZXRpY3MgYW5kIHN5c3RlbWF0aWNzOyBBY3RhIEhvcnQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTg0MjsgIGNpdGF0aW9uX3BhZ2VzPTcxLTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZWNhcGxvaWR5IGluIEZyYWdhcmlhIGl0dXJ1cGVuc2lzIChSb3NhY2VhZSk7IEFtLiBKLiBCb3QuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTk2OyAgY2l0YXRpb25fcGFnZXM9NzEzLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1Ib2x0LCBSaW5laGFydCBhbmQgV2luc3RvbiwgTmV3IFlvcms7IFRoZSBTdHJhd2JlcnJ5LiBIaXN0b3J5LCBCcmVlZGluZyBhbmQgUGh5c2lvbG9neTsgIGNpdGF0aW9uX3llYXI9MTk2NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW5ldGljIHJlbGF0aW9uc2hpcHMgYW1vbmcgc3BlY2llcyBvZiBGcmFnYXJpYSAoUm9zYWNlYWUpIGluZmVycmVkIGZyb20gbm9uLWNvZGluZyBudWNsZWFyIGFuZCBjaGxvcm9wbGFzdCBETkEgc2VxdWVuY2VzOyBTeXN0LiBCb3QuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzM3LTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFja2luZyB0aGUgZXZvbHV0aW9uYXJ5IGhpc3Rvcnkgb2YgcG9seXBsb2lkeSBpbiBGcmFnYXJpYSBMLiAoc3RyYXdiZXJyeSk6IG5ldyBpbnNpZ2h0cyBmcm9tIHBoeWxvZ2VuZXRpYyBhbmFseXNlcyBvZiBsb3ctY29weSBudWNsZWFyIGdlbmVzOyBNb2wuIFBoeWxvZ2VuZXQuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTUxOyAgY2l0YXRpb25fcGFnZXM9NTE1LTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIG9mIHdvb2RsYW5kIHN0cmF3YmVycnkgKEZyYWdhcmlhIHZlc2NhKTsgTmF0LCBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz0xMDktMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IkZyYWdhcmlhIHZlc2NhIFdob2xlIEdlbm9tZSB2MS4xIEFzc2VtYmx5ICZhbXA7IEFubm90YXRpb24gR2Vub21lIERhdGFiYXNlIGZvciBSb3NhY2VhZSAoR0RSKSBodHRwOi8vd3d3LnJvc2FjZWFlLm9yZy9zcGVjaWVzL2ZyYWdhcmlhL2ZyYWdhcmlhX3Zlc2NhL2dlbm9tZV92MS4xICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11dGF0aW9uIGluIFRFUk1JTkFMIEZMT1dFUjEgcmV2ZXJzZXMgdGhlIHBob3RvcGVyaW9kaWMgcmVxdWlyZW1lbnQgZm9yIGZsb3dlcmluZyBpbiB0aGUgd2lsZCBzdHJhd2JlcnJ5IEZyYWdhcmlhIHZlc2NhOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xNTk7ICBjaXRhdGlvbl9wYWdlcz0xMDQzLTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYXJnZXRlZCBzZXF1ZW5jZSBjYXB0dXJlIHByb3ZpZGVzIGluc2lnaHQgaW50byBnZW5vbWUgc3RydWN0dXJlIGFuZCBnZW5ldGljcyBvZiBtYWxlIHN0ZXJpbGl0eSBpbiBhIGd5bm9kaW9lY2lvdXMgZGlwbG9pZCBzdHJhd2JlcnJ5LCBGcmFnYXJpYSB2ZXNjYSBzc3AuIGJyYWN0ZWF0YSAoUm9zYWNlYWUpOyBHMyAoQmV0aGVzZGEpOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0xMzQxLTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBpZGVudGlmaWNhdGlvbiBhbmQgY29tcGFyYXRpdmUgZXhwcmVzc2lvbiBhbmFseXNpcyBvZiBOQlMtTFJSLWVuY29kaW5nIGdlbmVzIHVwb24gQ29sbGV0b3RyaWNodW0gZ2xvZW9zcG9yaW9pZGVzIGluZmVjdGlvbiBpbiB0d28gZWNvdHlwZXMgb2YgRnJhZ2FyaWEgdmVzY2E7IEdlbmUuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTUyNzsgIGNpdGF0aW9uX3BhZ2VzPTIxNS0yNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3VycmVudCBjaGFsbGVuZ2VzIGluIGRlIG5vdm8gcGxhbnQgZ2Vub21lIHNlcXVlbmNpbmcgYW5kIGFzc2VtYmx5OyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0yNDMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZHZhbmNlcyBpbiBwbGFudCBnZW5vbWUgc2VxdWVuY2luZzsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzA7ICBjaXRhdGlvbl9wYWdlcz0xNzctOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgb2YgdGhlIGRvbWVzdGljYXRlZCBhcHBsZSAoTWFsdXMgZG9tZXN0aWNhIEJvcmtoLik7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9ODMzLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgb2YgdGhlIHBlYXIgKFB5cnVzIGJyZXRzY2huZWlkZXJpIFJlaGQuKTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0zOTYtNDA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgaGlnaC1xdWFsaXR5IGRyYWZ0IGdlbm9tZSBvZiBwZWFjaCAoUHJ1bnVzIHBlcnNpY2EpIGlkZW50aWZpZXMgdW5pcXVlIHBhdHRlcm5zIG9mIGdlbmV0aWMgZGl2ZXJzaXR5LCBkb21lc3RpY2F0aW9uIGFuZCBnZW5vbWUgZXZvbHV0aW9uOyBjaXRhdGlvbl9hdXRob3I9SW50ZXJuYXRpb25hbCBQZWFjaCBHZW5vbWUgSW5pdGlhdGl2ZTsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz00ODctOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgb2YgUHJ1bnVzIG11bWU7IE5hdC4gQ29tbXVuLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MTMxOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZSBhbmQgYW5hbHlzaXMgb2YgdGhlIHR1YmVyIGNyb3AgcG90YXRvOyBjaXRhdGlvbl9hdXRob3I9VGhlIFBvdGF0byBHZW5vbWUgU2VxdWVuY2luZyBDb25zb3J0aXVtOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDc1OyAgY2l0YXRpb25fcGFnZXM9MTg5LTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZHJhZnQgZ2Vub21lIG9mIGEgZGlwbG9pZCBjb3R0b24gR29zc3lwaXVtIHJhaW1vbmRpaTsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz0xMDk4LTEwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGJhbmFuYSAoTXVzYSBhY3VtaW5hdGEpIGdlbm9tZSBhbmQgdGhlIGV2b2x1dGlvbiBvZiBtb25vY290eWxlZG9ub3VzIHBsYW50czsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ4ODsgIGNpdGF0aW9uX3BhZ2VzPTIxMy03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HR1QgMi4wOiBWZXJzYXRpbGUgc29mdHdhcmUgZm9yIHZpc3VhbGl6YXRpb24gYW5kIGFuYWx5c2lzIG9mIGdlbmV0aWMgZGF0YTsgSi4gSGVyZWRpdHkuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk5OyAgY2l0YXRpb25fcGFnZXM9MjMyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVB2Y2x1c3Q6IGFuIFIgcGFja2FnZSBmb3IgYXNzZXNzaW5nIHRoZSB1bmNlcnRhaW50eSBpbiBoaWVyYXJjaGljYWwgY2x1c3RlcmluZzsgQmlvaW5mb3JtYXRpY3MuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MTU0MC0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGZhc3QsIGxvY2stZnJlZSBhcHByb2FjaCBmb3IgZWZmaWNpZW50IHBhcmFsbGVsIGNvdW50aW5nIG9mIG9jY3VycmVuY2VzIG9mIGstbWVyczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz03NjQtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlIG5vdm8gYXNzZW1ibHkgb2YgaHVtYW4gZ2Vub21lcyB3aXRoIG1hc3NpdmVseSBwYXJhbGxlbCBzaG9ydCByZWFkIHNlcXVlbmNpbmc7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MjY1LTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HYXBwZWQgQkxBU1QgYW5kIFBTSS1CTEFTVDogYSBuZXcgZ2VuZXJhdGlvbiBvZiBwcm90ZWluIGRhdGFiYXNlIHNlYXJjaCBwcm9ncmFtczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzM4OS00MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgb2YgdGhlIGZpcmUgYW50IFNvbGVub3BzaXMgaW52aWN0YTsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTA4OyAgY2l0YXRpb25fcGFnZXM9NTY3OS04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGUgbm92byBpZGVudGlmaWNhdGlvbiBvZiByZXBlYXQgZmFtaWxpZXMgaW4gbGFyZ2UgZ2Vub21lczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVwZWF0cyBpbiBnZW5vbWljIGRuYTogbWluaW5nIGFuZCBtZWFuaW5nOyBDdXJyLiBPcGluLiBTdHJ1Y3QuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0zMzMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2NpUm9LbzogYSBuZXcgdG9vbCBmb3Igd2hvbGUgZ2Vub21lIG1pY3Jvc2F0ZWxsaXRlIHNlYXJjaCBhbmQgaW52ZXN0aWdhdGlvbjsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xNjgzLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPXRSTkFzY2FuLVNFOiBhIHByb2dyYW0gZm9yIGltcHJvdmVkIGRldGVjdGlvbiBvZiB0cmFuc2ZlciBSTkEgZ2VuZXMgaW4gZ2Vub21pYyBzZXF1ZW5jZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9OTU1LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIHByZWRpY3Rpb24gd2l0aCBhIGhpZGRlbiBtYXJrb3YgbW9kZWwgYW5kIGEgbmV3IGludHJvbiBzdWJtb2RlbDsgQmlvaW5mb3JtYXRpY3MuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25faXNzdWU9U3VwcGwgMik7ICBjaXRhdGlvbl9wYWdlcz1paTIxNS0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmV3IGRldmVsb3BtZW50cyBpbiB0aGUgSW50ZXJQcm8gZGF0YWJhc2U7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPUQyMjQtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZXJQcm9TY2FuOiBwcm90ZWluIGRvbWFpbnMgaWRlbnRpZmllcjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9VzExNi0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWNjZWxlcmF0ZWQgcHJvZmlsZSBITU0gc2VhcmNoZXM7IFBMb1MgQ29tcHV0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDIxOTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgR3lwc3kgRGF0YWJhc2UgKEd5REIpIG9mIG1vYmlsZSBnZW5ldGljIGVsZW1lbnRzOiByZWxlYXNlIDIuMDsgY2l0YXRpb25fYXV0aG9yPUNvdmVsbGk7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPUQ3MC00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CTEFU4oCUdGhlIEJMQVNULWxpa2UgYWxpZ25tZW50IHRvb2w7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NjU2LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbXByb3ZlZCB0b29scyBmb3IgYmlvbG9naWNhbCBzZXF1ZW5jZSBjb21wYXJpc29uOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3ZvbHVtZT04NTsgIGNpdGF0aW9uX3BhZ2VzPTI0NDQtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2QtaGl0OiBhIGZhc3QgcHJvZ3JhbSBmb3IgY2x1c3RlcmluZyBhbmQgY29tcGFyaW5nIGxhcmdlIHNldHMgb2YgcHJvdGVpbiBvciBudWNsZW90aWRlIHNlcXVlbmNlczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0xNjU4LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hc3RlcmluZyBzZWVkcyBmb3IgZ2Vub21pYyBzaXplIG51Y2xlb3RpZGUgQkxBU1Qgc2VhcmNoZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTY5MzUtNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc2VtYmx5IGFsZ29yaXRobXMgZm9yIG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIGRhdGE7IEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTk1OyAgY2l0YXRpb25fcGFnZXM9MzE1LTI3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXZpZXcgb2YgZ2VuZXJhbCBhbGdvcml0aG1pYyBmZWF0dXJlcyBmb3IgZ2Vub21lIGFzc2VtYmxlcnMgZm9yIG5leHQgZ2VuZXJhdGlvbiBzZXF1ZW5jZXJzOyBHZW5vbWljcyBQcm90ZW9taWNzIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NTgtNzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnNlcnZhdGlvbiBhbmQgbG9zcyBvZiByaWJvc29tYWwgUk5BIGdlbmUgc2l0ZXMgaW4gZGlwbG9pZCBhbmQgcG9seXBsb2lkIEZyYWdhcmlhIChSb3NhY2VhZSk7IEJNQyBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTE1NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMS8yLzE2OS8xMTI4NTczL2RzdDA0OS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgQ3VsdGl2YXRlZCBzdHJhd2JlcnJ5IChGcmFnYXJpYSB4IGFuYW5hc3NhKSBpcyBvY3RvcGxvaWQgYW5kIHNob3dzIGFsbG9nYW1vdXMgYmVoYXZpb3VyLiBUaGUgcHJlc2VudCBzdHVkeSBhaW1zIGF0IGRpc3NlY3RpbmcgdGhpcyBvY3RvcGxvaWQgZ2Vub21lIHQiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQwNDAwNSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGR1oLXuuOB0BMS22s8V0BfAd3biCg==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f2%2f169%2f404005" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f2%2f169%2f404005" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=15712" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/2/1/m_cover.gif?Expires=2147483647&amp;Signature=C2ZH5dkUQEctPPta7W831-4mGKGGgEq~Aqbfql5zal9lq-CH7ZJcQtnaLb-2~TFW5o1XuWK-If8uoA0C0x3E8Xqh7vx6q7XeJXntpLTv0xMKSiGPs8b5JT8WwzNVzF-MVRjRIurpCdYeMF6bhbaWGNZLajJ34JAWHyXvGj72zxR2I~0JkQXfyp1Fd2IS9rmTmm3EZ-Vtcf0zL2z0kWasBqYnVob3EwZTj2l448hvYoYYZgYfsgU2j1vdpw3GUsXrCjCI5GsZ1ht-EZrYHZzJ3zkCMOPIc~J0lFrmNiix6fEEbm~y5NNGWoTY3GIiDMTmJpXRxBN715elzocAQQfDUQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/2">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5971246">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5971246">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5971248">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5971248">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5971255">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5971255">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5971281">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5971281">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5971310">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5971310">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5971319">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5971319">5. Database</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5971321">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5971321">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5971323">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5971323">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/2/153/403337">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/21/2/183/403825">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
<span class="article-flag"><a href="https://academic.oup.com/dnaresearch/search-results?f_OUPSeries=Editor's+Choice">Editor's Choice</a></span>            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Dissection of the Octoploid Strawberry Genome by Deep Sequencing of the Genomes of <em>Fragaria</em> Species
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Hirakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideki+Hirakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirakawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hirakawa H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hideki Hirakawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenta Shirasawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenta+Shirasawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shirasawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shirasawa K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kenta Shirasawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shunichi Kosugi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shunichi+Kosugi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kosugi S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kosugi S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shunichi Kosugi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kosuke Tashiro</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Bioscience and Biotechnology, Faculty of Agriculture</div>, <div class="insititution">Kyushu University</div>, <div class="addr-line">Higashi-ku, Fukuoka 812-8581</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kosuke+Tashiro">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tashiro K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tashiro K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kosuke Tashiro</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shinobu Nakayama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shinobu+Nakayama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakayama S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakayama S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shinobu Nakayama</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Manabu Yamada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Manabu+Yamada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamada M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamada M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Manabu Yamada</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mistuyo Kohara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mistuyo+Kohara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kohara M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kohara M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Mistuyo Kohara</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akiko Watanabe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akiko+Watanabe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Watanabe A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Watanabe A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Akiko Watanabe</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshie Kishida</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshie+Kishida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kishida Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kishida Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yoshie Kishida</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tsunakazu Fujishiro</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tsunakazu+Fujishiro">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fujishiro T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fujishiro T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Tsunakazu Fujishiro</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hisano Tsuruoka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hisano+Tsuruoka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tsuruoka H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tsuruoka H&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Hisano Tsuruoka</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chiharu Minami</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chiharu+Minami">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Minami C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Minami C&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Chiharu Minami</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shigemi Sasamoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shigemi+Sasamoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sasamoto S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sasamoto S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Shigemi Sasamoto</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Midori Kato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Midori+Kato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kato M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kato M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Midori Kato</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Keiko Nanri</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Keiko+Nanri">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nanri K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nanri K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Keiko Nanri</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akiko Komaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akiko+Komaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Komaki A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Komaki A&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Akiko Komaki</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tomohiro Yanagi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Faculty of Agriculture</div>, <div class="insititution">Kagawa University</div>, <div class="addr-line">2393 Ikenobe, Miki, Kita, Kagawa 761-0795</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tomohiro+Yanagi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yanagi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yanagi T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tomohiro Yanagi</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Qin Guoxin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Chongqing Nanshan Botanical Garden, No. 101, Nanshan Garden Road, Nan'an District, Chongqing 400065</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Qin+Guoxin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Guoxin Q">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Guoxin Q&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Qin Guoxin</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fumi Maeda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Chiba Prefectural Agriculture and Forestry Research Center</div>, <div class="insititution">Crop Breeding Institute</div>, <div class="addr-line">Chousei, Daizenno-Cyou 808, Midori, Chiba 299-4335</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fumi+Maeda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Maeda F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Maeda F&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Fumi Maeda</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masami Ishikawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Chiba Prefectural Agriculture and Forestry Research Center</div>, <div class="insititution">Crop Breeding Institute</div>, <div class="addr-line">Chousei, Daizenno-Cyou 808, Midori, Chiba 299-4335</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masami+Ishikawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ishikawa M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ishikawa M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Masami Ishikawa</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoru Kuhara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Bioscience and Biotechnology, Faculty of Agriculture</div>, <div class="insititution">Kyushu University</div>, <div class="addr-line">Higashi-ku, Fukuoka 812-8581</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoru+Kuhara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kuhara S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kuhara S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Satoru Kuhara</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shusei Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">6</div><div class="addr-line">Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University</div>, <div class="addr-line">Katahira 2-1-1, Aoba, Sendai, Miyagi 980-8577</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shusei+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Shusei Sato</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Tabata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Tabata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tabata S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tabata S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Satoshi Tabata</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sachiko N. Isobe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kazusa-Kamatari 2-6-7, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel: +81 438-52-3928. Fax: +81 438-52-3934. Email: <a href="mailto:sisobe@kazusa.or.jp" target="_blank">sisobe@kazusa.or.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sachiko+N.+Isobe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Isobe S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Isobe S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Sachiko N. Isobe</a>
                                        </span>
                                </div>
                            </div>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 2, 1 April 2014, Pages 169–181, <a href='https://doi.org/10.1093/dnares/dst049'>https://doi.org/10.1093/dnares/dst049</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">26 November 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">05 September 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">21 October 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="404005" href="/dnaresearch/article-pdf/21/2/169/1128573/dst049.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Hideki Hirakawa, Kenta Shirasawa, Shunichi Kosugi, Kosuke Tashiro, Shinobu Nakayama, Manabu Yamada, Mistuyo Kohara, Akiko Watanabe, Yoshie Kishida, Tsunakazu Fujishiro, Hisano Tsuruoka, Chiharu Minami, Shigemi Sasamoto, Midori Kato, Keiko Nanri, Akiko Komaki, Tomohiro Yanagi, Qin Guoxin, Fumi Maeda, Masami Ishikawa, Satoru Kuhara, Shusei Sato, Satoshi Tabata, Sachiko N. Isobe;  Dissection of the Octoploid Strawberry Genome by Deep Sequencing of the Genomes of <em>Fragaria</em> Species, <em>DNA Research</em>, Volume 21, Issue 2, 1 April 2014, Pages 169–181, <a href='https://doi.org/10.1093/dnares/dst049'>https://doi.org/10.1093/dnares/dst049</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/404005?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/404005?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/404005?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/404005?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/404005?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/404005?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/404005?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Dissection%20of%20the%20Octoploid%20Strawberry%20Genome%20by%20Deep%20Sequencing%20of%20the%20Genomes%20of%20%26lt%3Bem%26gt%3BFragaria%26lt%3B%2Fem%26gt%3B%20Species&amp;publicationDate=2013-11-26&amp;volumeNum=21&amp;issueNum=2&amp;author=Hirakawa%2C%20Hideki%3B%20Shirasawa%2C%20Kenta&amp;startPage=169&amp;endPage=181&amp;contentId=10.1093%2Fdnares%2Fdst049&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f2%2f169%2f404005" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5971246 id="5971246" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Cultivated strawberry (<em>Fragaria</em> x <em>ananassa</em>) is octoploid and shows allogamous behaviour. The present study aims at dissecting this octoploid genome through comparison with its wild relatives, <em>F. iinumae</em>, <em>F. nipponica</em>, <em>F. nubicola</em>, and <em>F. orientalis</em> by <em>de novo</em> whole-genome sequencing on an Illumina and Roche 454 platforms. The total length of the assembled Illumina genome sequences obtained was 698 Mb for <em>F.</em> x <em>ananassa</em>, and ∼200 Mb each for the four wild species. Subsequently, a virtual reference genome termed FANhybrid_r1.2 was constructed by integrating the sequences of the four homoeologous subgenomes of <em>F</em>. x <em>ananassa</em>, from which heterozygous regions in the Roche 454 and Illumina genome sequences were eliminated. The total length of FANhybrid_r1.2 thus created was 173.2 Mb with the N50 length of 5137 bp. The Illumina-assembled genome sequences of <em>F</em>. x <em>ananassa</em> and the four wild species were then mapped onto the reference genome, along with the previously published <em>F. vesca</em> genome sequence to establish the subgenomic structure of <em>F.</em> x <em>ananassa</em>. The strategy adopted in this study has turned out to be successful in dissecting the genome of octoploid <em>F</em>. x <em>ananassa</em> and appears promising when applied to the analysis of other polyploid plant species.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Fragariax ananassa&quot;"><em>Fragaria</em>x <em>ananassa</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;wild Fragaria species&quot;">wild <em>Fragaria</em> species</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome sequence assembly&quot;">genome sequence assembly</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;comparative analysis&quot;">comparative analysis</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="polyploidy">polyploidy</a></div>                            <h2 scrollto-destination=5971248 id="5971248" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The cultivated strawberry (<em>Fragaria</em> x <em>ananassa</em>) is a globally consumed crop species that is grown around the world; in the USA (37.5%), Europe (36.7%), Asia (16.3%), Africa (8.6%), and Oceania (0.8%).<sup><a reveal-id="DST049C1" data-open="DST049C1" class="link link-ref link-reveal xref-bibr">1</a></sup><em>Fragaria</em> x <em>ananassa</em> is an octoploid species (2<em>n</em> = 8<em>×</em> = 56) with an estimated genome size of 1C = 708–720 Mb.<sup><a reveal-id="DST049C2" data-open="DST049C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DST049C3" data-open="DST049C3" class="link link-ref link-reveal xref-bibr">3</a></sup> In addition to its polyploidy, allogamous behaviour in <em>F.</em> x <em>ananassa</em> contributes further complexity to the genome structure<em>.</em> Until now, three genome composition models, namely AABBBBCC,<sup><a reveal-id="DST049C4" data-open="DST049C4" class="link link-ref link-reveal xref-bibr">4</a></sup> AAA′A′BBBB,<sup><a reveal-id="DST049C5" data-open="DST049C5" class="link link-ref link-reveal xref-bibr">5</a></sup> and AAA′A′BBB′B′,<sup><a reveal-id="DST049C6" data-open="DST049C6" class="link link-ref link-reveal xref-bibr">6</a></sup> have been proposed based on cytological and genetic evidence. Of the three models, the most recently proposed model, AAA′A′BBB′B′,<sup><a reveal-id="DST049C6" data-open="DST049C6" class="link link-ref link-reveal xref-bibr">6</a></sup> is considered the most probable candidate, since several studies have reported the disomic inheritance of large numbers of DNA markers,<sup><a reveal-id="DST049C7  DST049C9" data-open="DST049C7  DST049C9" class="link link-ref link-reveal xref-bibr">7–9</a></sup> which suggests an allopolyploid genome composition in <em>F.</em>x <em>ananassa</em>.</p><p>The genus <em>Fragaria</em> belongs to the family Rosaceae and is comprised of one cultivated (<em>F.</em>x <em>ananassa</em>) and 24 wild species, including 13 diploids, 5 tetraploids, 1 hexaploid, 4 octoploids, and 1 decaploid.<sup><a reveal-id="DST049C10" data-open="DST049C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DST049C11" data-open="DST049C11" class="link link-ref link-reveal xref-bibr">11</a></sup> The geographic origins of the wild species are distributed throughout Eurasia, North and South America, and Japan. <em>Fragaria</em> x <em>ananassa</em> originated in the 1700s from a natural hybridization between two octoploids, <em>F. virginiana</em> and <em>F. chiloensis</em>.<sup><a reveal-id="DST049C12" data-open="DST049C12" class="link link-ref link-reveal xref-bibr">12</a></sup> However, the history of evolution from diploid to octoploid species in the genus <em>Fragaria</em> remains controversial. Davis <em>et al.</em><sup><a reveal-id="DST049C3" data-open="DST049C3" class="link link-ref link-reveal xref-bibr">3</a></sup> proposed that <em>F. vesca</em>, <em>F. nubicola</em>, and <em>F. orientalis</em> were possible progenitors to octoploids, based on the polygenic analysis of internal transcribed spacer sequences of rDNA and chloroplast DNA performed by Potter <em>et al.</em><sup><a reveal-id="DST049C13" data-open="DST049C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Rousseau-Gueutin <em>et al.</em><sup><a reveal-id="DST049C14" data-open="DST049C14" class="link link-ref link-reveal xref-bibr">14</a></sup> hypothesized that the evolutionary history was based on five subgenomic entries, X1, X2, Y1, Y2, and Z, classified according to two nuclear genes, <em>GBSSI-2</em> and <em>DHAR</em>. They proposed that the genomes of wild octoploids consisted of Y1′Y1′Y1″Y1″ZZZZ or Y1Y1Y1Y1ZZZZ genomes. The Y1 genomes were considered to be derived from two diploid species, <em>F. vesca</em> or <em>F. mandshurica</em>, via a tetraploid, <em>F. orientalis</em>, whereas <em>F. iinumae</em> was presumed to be the progenitor of the Z genome.</p><p><em>Fragaria vesca</em>, which is the most plausible progenitor of <em>F.</em>x <em>ananassa</em>, was selected as the genomic reference for <em>Fragaria</em>. Using a fourth-generation inbred line and the Roche 454, Illumina Solexa, and Life Technologies SOLiD platforms, the whole-genome sequence of <em>F. vesca</em> was published in 2011.<sup><a reveal-id="DST049C15" data-open="DST049C15" class="link link-ref link-reveal xref-bibr">15</a></sup> According to the report, a total of 209.8 Mb were assembled into 272 scaffolds, and 34 809 candidate genes were identified by gene prediction. Later, the original v1.0 pseudomolecule assembly was updated to v1.1.<sup><a reveal-id="DST049C16" data-open="DST049C16" class="link link-ref link-reveal xref-bibr">16</a></sup></p><p>The <em>F. vesca</em> genome sequences have contributed greatly to advances in molecular genetic analysis in <em>F.</em> x <em>ananassa</em><sup><a reveal-id="DST049C8" data-open="DST049C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DST049C9" data-open="DST049C9" class="link link-ref link-reveal xref-bibr">9</a></sup> and are expected to assist in the identification of genes related to agriculturally important traits in <em>F.</em> x <em>ananassa</em>, such as flowering time,<sup><a reveal-id="DST049C17" data-open="DST049C17" class="link link-ref link-reveal xref-bibr">17</a></sup> male sterility,<sup><a reveal-id="DST049C18" data-open="DST049C18" class="link link-ref link-reveal xref-bibr">18</a></sup> and stress resistance.<sup><a reveal-id="DST049C19" data-open="DST049C19" class="link link-ref link-reveal xref-bibr">19</a></sup> However, consultation of the <em>F. vesca</em> genome has limitations, particularly for <em>F.</em> x <em>ananassa</em>-specific genome regions or genome regions derived from other progenitors. Therefore, whole-genome sequencing of <em>F.</em> x <em>ananassa</em> is needed for a comprehensive understanding of the genome structure of the species, in parallel with reference to the <em>F. vesca</em> genome.</p><p>Next-generation sequencing (NGS) technologies have revolutionized <em>de novo</em> whole-genome sequencing in most species. This is especially true in plant species, for which genomes are relatively large and complex in structure compared with non-plant genomes.<sup><a reveal-id="DST049C20" data-open="DST049C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DST049C21" data-open="DST049C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Up to now, whole-genome sequences have been published in more than 50 plant species, including five species in Rosaceae: <em>F. vesca</em>,<sup><a reveal-id="DST049C15" data-open="DST049C15" class="link link-ref link-reveal xref-bibr">15</a></sup> apple,<sup><a reveal-id="DST049C22" data-open="DST049C22" class="link link-ref link-reveal xref-bibr">22</a></sup> pear,<sup><a reveal-id="DST049C23" data-open="DST049C23" class="link link-ref link-reveal xref-bibr">23</a></sup> peach,<sup><a reveal-id="DST049C24" data-open="DST049C24" class="link link-ref link-reveal xref-bibr">24</a></sup> and Chinese plum.<sup><a reveal-id="DST049C25" data-open="DST049C25" class="link link-ref link-reveal xref-bibr">25</a></sup> However, to our knowledge, no reports of <em>de novo</em> whole-genome sequencing have been published for any polyploid species. As with strawberry, diploid progenitor species have been employed to advance the genomic understanding of other polyploid crops, such as potato,<sup><a reveal-id="DST049C26" data-open="DST049C26" class="link link-ref link-reveal xref-bibr">26</a></sup> cotton,<sup><a reveal-id="DST049C27" data-open="DST049C27" class="link link-ref link-reveal xref-bibr">27</a></sup> and banana.<sup><a reveal-id="DST049C28" data-open="DST049C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Genome sequencing of polyploid species has been avoided, because the homoeology of subgenomes made sequence assembly difficult. To overcome this difficulty, we attempted to construct a virtual ‘reference genome’ in octoploid strawberry, <em>F.</em> x <em>ananassa</em>, as a first step for whole-genome sequencing of the species.</p><p>In this study, <em>de novo</em> whole-genome sequencing was performed in <em>F.</em> x <em>ananassa</em> using the Illumina (Illumina, Inc., CA, USA) and Roche 454 sequencing platforms (Roche Diagnostics, IN, USA). A virtual reference genome, which integrated genome sequences of homoeologous chromosomes, was constructed by eliminating heterozygous bases in the process of sequence assembly. In a previous study, macrosynteny at the chromosome level was observed between the linkage groups (LGs) in an <em>F.</em>x <em>ananassa</em> genetic map and the genome of <em>F. vesca.</em><sup><a reveal-id="DST049C9" data-open="DST049C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Therefore, we aligned the generated <em>F.</em>x <em>ananassa</em> scaffolds with the pseudomolecules of <em>F. vesca</em>.<sup><a reveal-id="DST049C15" data-open="DST049C15" class="link link-ref link-reveal xref-bibr">15</a></sup> In parallel, four wild <em>Fragaria</em> species, representing genetic diversity in the genus <em>Fragaria</em>, were selected based on simple sequence repeat (SSR) markers and were subjected to whole-genome sequencing using an Illumina platform. The assembled sequences of the wild species, along with the heterozygous <em>F.</em> x <em>ananassa</em> sequences, were mapped onto the <em>F.</em> x <em>ananassa</em> reference genome and used to elucidate the genome structures of the <em>Fragaria</em> species.</p>                            <h2 scrollto-destination=5971255 id="5971255" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5971256 id="5971256" class="section-title" data-legacy-id=s2a>2.1. Plant materials</h3>
<p>A Japanese variety, ‘Reikou’, and its progeny (S<sub>1</sub>) were subjected to genome sequencing as representatives of <em>F.</em> x <em>ananassa.</em> ‘Reikou’ was bred in the Chiba Prefectural Agriculture and Forestry Research Center. Like other strawberry varieties, ‘Reikou’ maintains heterozygosity in the genome. The S<sub>1</sub> progenies were sequenced for SNP discovery within ‘Reikou’ for further analysis (data were not shown in this study). A total of 20 wild <em>Fragaria</em> species with a diverse range of polyploidy were used in phylogenetic analyses with SSR markers (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Along with the 20 wild species, the following four Japanese <em>F.</em> x <em>ananassa</em> varieties were subjected to the phylogenic analysis: ‘Reikou’, ‘Hokowase’, ‘Sachinoka’, and ‘Akihime’. Whole-genome sequencing was subsequently performed for four wild species, <em>F. iinumae</em>, <em>F. nipponica</em>, <em>F. nubicola</em>, and <em>F. orientalis</em>. The genomic DNA was extracted from young leaves using a DNeasy Plant Mini Kit (Qiagen, Inc., CA, USA). DNA quantification and quality checks were performed using a Nanodrop ND1000 spectrophotometer (Nanodrop Technologies, DE, USA) and 0.8% agarose gel electrophoresis, respectively.</p>                            <h3 scrollto-destination=5971258 id="5971258" class="section-title" data-legacy-id=s2b>2.2. Phylogenetic analysis</h3>
<p>Genomic polymorphisms from 24 <em>Fragaria</em> accessions (listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>) were analysed using 632 <em>F.</em> x <em>ananassa</em> and <em>F. vesca</em> expressed sequence tag derived SSR markers that randomly mapped onto the 28 LGs of the consensus genetic map of <em>F.</em> x <em>ananassa</em><sup><a reveal-id="DST049C9" data-open="DST049C9" class="link link-ref link-reveal xref-bibr">9</a></sup> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). PCR was performed as described in Isobe <em>et al.</em><sup><a reveal-id="DST049C9" data-open="DST049C9" class="link link-ref link-reveal xref-bibr">9</a></sup> The PCR products were separated using an ABI 3730xl fluorescent fragment analyser (Applied Biosystems, CA, USA). Polymorphisms were investigated using the GeneMapper software (Applied Biosystems). The genetic distances and Jaccard's similarity coefficients of all combinations of any two samples were calculated from the genotypic data using the GGT2 software.<sup><a reveal-id="DST049C29" data-open="DST049C29" class="link link-ref link-reveal xref-bibr">29</a></sup> The hierarchical cluster analysis with the multiscale bootstrap was performed for 1000 replications with Ward's method by pvclust package<sup><a reveal-id="DST049C30" data-open="DST049C30" class="link link-ref link-reveal xref-bibr">30</a></sup> for the R software (<a class="link link-uri" href="http://www.r-project.org/">http://www.r-project.org/</a>).</p>                            <h3 scrollto-destination=5971260 id="5971260" class="section-title" data-legacy-id=s2c>2.3. Genome sequencing</h3>
<p>Whole-genome shotgun sequencing was carried out using the Roche 454 GS FLX+ and Illumina GAIIx/Hiseq 1000 platforms (Roche Diagnostics; Illumina, Inc., Fig. <a reveal-id="DST049F1" data-open="DST049F1" class="link link-reveal link-table xref-fig">1</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Total cellular DNA of the ‘Reikou’ variety of <em>F.</em>x <em>ananassa</em> was used for the construction of single-end (SE) and paired-end (PE) libraries for the Roche 454 sequencing platform according to the instructions provided by the manufacturer. Insert sizes of the PE libraries were 3, 5, 8, and 20 kb. </p>                        <a id="5971262" scrollto-destination="5971262"></a>
<div id="DST049F1" data-id="DST049F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst049/2/m_dst04901.png?Expires=2147483647&Signature=djZ98nZ-LmMio9EvOTRtbv6yvW7zRymzJPU4HBfIU-sfLMB-PsOunO82Ya-4Dz6Xg32vGIk52aBGKG04shsPbNoQpteMaDKTW74fj39vQRphWW4bdjFQai9jXsYsD~TEwqIdNBL9bQnZ585WPro8rc-4N3lSVmrN~u3o0fcIMZVRtsfpitNfI6HPMOk-8NVo8Zva44LH~3P2nDBWge~ymBGkac9GHgVu6k6yGk3IkeWvM-kalDzfj2K1T76l~Z6CxENz0EpMiJU3fYgcbW6oAKBzCZ9Fh9~9-LCR7dKAVJyx3wX50tX5ZzCAsdL52CgLVmZADpTiBPDwnqOmu~3UMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The strategy and status of sequencing and assembly of the reference genome of F. x ananassa (FANhybrid_r1.2). PE, SE, and MP represent PE, SE, and MP reads, respectively. The FANhybrid_r1.2 sequences (a black box) are consisted with the 454 scaffolds gap-closed with Illumina reads and Illumina-specific assembled sequences (gray boxes)." path-from-xml="dst04901.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5971262/dst04901.tiff" path-from-xml="dst04901.tiff" target="_blank">View large</a><a section="5971262" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst049/2/dst04901.png?Expires=2147483647&Signature=CZPracc62GgK4B1qXmq8soA6KLLHdjMnJzmHC1hvoW1XMM~b7jOqfcyfgXGOdtqerB6uHb1n8bWTmhZeeoKpYv~26RF4EUUDsZu55N-1sVk6-A1cH7xIiH6KQ2j8z67XChjHQtskuzScrykRNsVFyoTSI8GZOOmTtRCKGyNBSL1DFoJ0gOXAC95bbpV-yVPF45WU641NhoST-DvJGltGgeU4BfGGku7gEoR6d2MBy0BS37dtNiqeFZFJ-evf9UpWsj~OmMmnxFUAbSvNqDVb15a3DE~xPmfaTmZzm8eJ16JJMAIhJEa2igZsl3JvHFC056CEc9fLzjb7Y-svJjdZMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5971262&ar=404005&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04901.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The strategy and status of sequencing and assembly of the reference genome of <em>F.</em> x <em>ananassa</em> (FANhybrid_r1.2). PE, SE, and MP represent PE, SE, and MP reads, respectively. The FANhybrid_r1.2 sequences (a black box) are consisted with the 454 scaffolds gap-closed with Illumina reads and Illumina-specific assembled sequences (gray boxes).</p></div></div><div content-id="DST049F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst049/2/m_dst04901.png?Expires=2147483647&Signature=djZ98nZ-LmMio9EvOTRtbv6yvW7zRymzJPU4HBfIU-sfLMB-PsOunO82Ya-4Dz6Xg32vGIk52aBGKG04shsPbNoQpteMaDKTW74fj39vQRphWW4bdjFQai9jXsYsD~TEwqIdNBL9bQnZ585WPro8rc-4N3lSVmrN~u3o0fcIMZVRtsfpitNfI6HPMOk-8NVo8Zva44LH~3P2nDBWge~ymBGkac9GHgVu6k6yGk3IkeWvM-kalDzfj2K1T76l~Z6CxENz0EpMiJU3fYgcbW6oAKBzCZ9Fh9~9-LCR7dKAVJyx3wX50tX5ZzCAsdL52CgLVmZADpTiBPDwnqOmu~3UMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The strategy and status of sequencing and assembly of the reference genome of F. x ananassa (FANhybrid_r1.2). PE, SE, and MP represent PE, SE, and MP reads, respectively. The FANhybrid_r1.2 sequences (a black box) are consisted with the 454 scaffolds gap-closed with Illumina reads and Illumina-specific assembled sequences (gray boxes)." path-from-xml="dst04901.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5971262/dst04901.tiff" path-from-xml="dst04901.tiff" target="_blank">View large</a><a section="5971262" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst049/2/dst04901.png?Expires=2147483647&Signature=CZPracc62GgK4B1qXmq8soA6KLLHdjMnJzmHC1hvoW1XMM~b7jOqfcyfgXGOdtqerB6uHb1n8bWTmhZeeoKpYv~26RF4EUUDsZu55N-1sVk6-A1cH7xIiH6KQ2j8z67XChjHQtskuzScrykRNsVFyoTSI8GZOOmTtRCKGyNBSL1DFoJ0gOXAC95bbpV-yVPF45WU641NhoST-DvJGltGgeU4BfGGku7gEoR6d2MBy0BS37dtNiqeFZFJ-evf9UpWsj~OmMmnxFUAbSvNqDVb15a3DE~xPmfaTmZzm8eJ16JJMAIhJEa2igZsl3JvHFC056CEc9fLzjb7Y-svJjdZMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5971262&ar=404005&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04901.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The strategy and status of sequencing and assembly of the reference genome of <em>F.</em> x <em>ananassa</em> (FANhybrid_r1.2). PE, SE, and MP represent PE, SE, and MP reads, respectively. The FANhybrid_r1.2 sequences (a black box) are consisted with the 454 scaffolds gap-closed with Illumina reads and Illumina-specific assembled sequences (gray boxes).</p></div></div><p>Illumina PE and mate-pair (MP) libraries were constructed from total cellular DNAs of ‘Reikou’, five S<sub>1</sub> progenies of ‘Reikou’, and four wild species, <em>F. iinumae</em>, <em>F. nipponica</em>, <em>F. nubicola</em>, and <em>F. orientalis</em>, according to the instructions provided by the manufacturer (Fig. <a reveal-id="DST049F2" data-open="DST049F2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). The expected insert sizes and read lengths of the libraries ranged from 290 bp to 2 kb, and 51–101 bp, respectively. Of the developed Illumina libraries, two ‘Reikou’ PE libraries (with insertion sizes of 600 and 400 bp) were subjected to genome sequencing by the Illumina GAIIx, whereas other Illumina libraries were analysed by the Illumina HiSeq 1000. The mixed sequences generated from the five S<sub>1</sub> progenies were regarded as equivalent to the parental ‘Reikou’ genome and were assembled together with sequence data from the ‘Reikou’ libraries. </p>                        <a id="5971264" scrollto-destination="5971264"></a>
<div id="DST049F2" data-id="DST049F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst049/2/m_dst04902.png?Expires=2147483647&Signature=1gGijIGmgFkldZ8JGeESuBaui0W34-JH1eMkHb9M54L3OtpLc6xYfcvh-mw~dZbLW3pSDW6JjeapJsJN3TYHzDeUWy0SuVYusd9RyATzrWn239L9fuBqlY80Gt-5xCVpzo1JVqoQl59jHJ4n3fh4aUuQbCCVt3tkw4OLcCRLE1gOK30IknBHqEsBADxnFJL8ghWQoYxsjHbeRy~oKxYKfsXmM0PPaUs9ndFIvbXsF9saWKrmg9hM8MNzvbED9YdA1UUDDY-4fp10ntarcizkCOce~oDHqOmSm-Xhrt9tRxTlvCgP6ZZh1k-WhTzVjp6AgCrg5g0QBbLwAq7ytON9bw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The strategy and status of Illumina sequence assembly of the genomes of F.x ananassa (FAN_r1.1) and four wild species, F. iinumae (FII_r1.1), F. nipponica (FNI_r1.1), F. nubicola (FNU_r1.1), and F. orientalis (FOR_r1.1), based on Illumina reads." path-from-xml="dst04902.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5971264/dst04902.tiff" path-from-xml="dst04902.tiff" target="_blank">View large</a><a section="5971264" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst049/2/dst04902.png?Expires=2147483647&Signature=OIPjEH5389zVDEpvk6zxEnIxcSd-9oB2PXkOMZp4X8oev0ZK9PP9hv2BEYx~mgbnNCDMQSc67Z-JqZ1g6IhJ8lpznZSY0iohqsJSuGmQjcISxL-mu9geRF6WfOtaDSo9XExU4T8w1G-DQOZRayTFsRexNx~fzo2HBcBwnsebnhqMQyTjRGx~A~4qU-WXQtuPpHx4Spu6D5VFkCfAalSl8wk7iAlm9lsDgyn90H1r0P6NM2QuZV38r2kei55PsmXG3fCLujXuZofZaqN94P42vRnCA-nEEzZoynkNym~KipsuOJ7lxG0WV-N4Py~UF0U4qPjXn8IBaTgNqTi0XKnweA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5971264&ar=404005&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04902.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The strategy and status of Illumina sequence assembly of the genomes of <em>F.</em>x <em>ananassa</em> (FAN_r1.1) and four wild species, <em>F. iinumae</em> (FII_r1.1), <em>F. nipponica</em> (FNI_r1.1), <em>F. nubicola</em> (FNU_r1.1), and <em>F. orientalis</em> (FOR_r1.1), based on Illumina reads.</p></div></div><div content-id="DST049F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst049/2/m_dst04902.png?Expires=2147483647&Signature=1gGijIGmgFkldZ8JGeESuBaui0W34-JH1eMkHb9M54L3OtpLc6xYfcvh-mw~dZbLW3pSDW6JjeapJsJN3TYHzDeUWy0SuVYusd9RyATzrWn239L9fuBqlY80Gt-5xCVpzo1JVqoQl59jHJ4n3fh4aUuQbCCVt3tkw4OLcCRLE1gOK30IknBHqEsBADxnFJL8ghWQoYxsjHbeRy~oKxYKfsXmM0PPaUs9ndFIvbXsF9saWKrmg9hM8MNzvbED9YdA1UUDDY-4fp10ntarcizkCOce~oDHqOmSm-Xhrt9tRxTlvCgP6ZZh1k-WhTzVjp6AgCrg5g0QBbLwAq7ytON9bw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The strategy and status of Illumina sequence assembly of the genomes of F.x ananassa (FAN_r1.1) and four wild species, F. iinumae (FII_r1.1), F. nipponica (FNI_r1.1), F. nubicola (FNU_r1.1), and F. orientalis (FOR_r1.1), based on Illumina reads." path-from-xml="dst04902.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5971264/dst04902.tiff" path-from-xml="dst04902.tiff" target="_blank">View large</a><a section="5971264" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst049/2/dst04902.png?Expires=2147483647&Signature=OIPjEH5389zVDEpvk6zxEnIxcSd-9oB2PXkOMZp4X8oev0ZK9PP9hv2BEYx~mgbnNCDMQSc67Z-JqZ1g6IhJ8lpznZSY0iohqsJSuGmQjcISxL-mu9geRF6WfOtaDSo9XExU4T8w1G-DQOZRayTFsRexNx~fzo2HBcBwnsebnhqMQyTjRGx~A~4qU-WXQtuPpHx4Spu6D5VFkCfAalSl8wk7iAlm9lsDgyn90H1r0P6NM2QuZV38r2kei55PsmXG3fCLujXuZofZaqN94P42vRnCA-nEEzZoynkNym~KipsuOJ7lxG0WV-N4Py~UF0U4qPjXn8IBaTgNqTi0XKnweA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5971264&ar=404005&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04902.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The strategy and status of Illumina sequence assembly of the genomes of <em>F.</em>x <em>ananassa</em> (FAN_r1.1) and four wild species, <em>F. iinumae</em> (FII_r1.1), <em>F. nipponica</em> (FNI_r1.1), <em>F. nubicola</em> (FNU_r1.1), and <em>F. orientalis</em> (FOR_r1.1), based on Illumina reads.</p></div></div>                            <h3 scrollto-destination=5971265 id="5971265" class="section-title" data-legacy-id=s2d>2.4. Quality control of Illumina reads</h3>
<p>The Illumina reads were preprocessed for quality control using the FastX-toolkit, ver. 0.0.13 (<a class="link link-uri" href="http://hannonlab.cshl.edu/fastx_toolkit/">http://hannonlab.cshl.edu/fastx_toolkit/</a>). After quality filtering, the sequences that included an N or were trimmed according to the following criteria were excluded from further analysis: (i) bases with quality value &lt;10; (ii) probable artefact reads; and (iii) adaptor sequences consisting of more than five bases at the 3′ terminal. The genome size of <em>F.</em>x <em>ananassa</em> and the four wild relatives were estimated, based on a <em>k</em>-mer = 17 frequency of the Illumina reads, by using Jellyfish ver. 1.1.6.<sup><a reveal-id="DST049C31" data-open="DST049C31" class="link link-ref link-reveal xref-bibr">31</a></sup></p>                            <h3 scrollto-destination=5971267 id="5971267" class="section-title" data-legacy-id=s2e>2.5. Assembly of the <em>F.</em><em>x</em><em>ananassa</em> reference genome</h3>
<p>The 454 reads were assembled using Newbler 2.7 (Roche Diagnostics) in a heterozygotic mode (Fig. <a reveal-id="DST049F1" data-open="DST049F1" class="link link-reveal link-table xref-fig">1</a>). The heterozygous bases between the homoeologous or heterozygous genomes were eliminated by the overlap layout consensus method, with sequence identity of 90%. The gaps on the scaffolds were closed by GapCloser 1.10 (<em>p</em> = 31) (<a class="link link-uri" href="http://soap.genomics.org.cn/soapdenovo.html">http://soap.genomics.org.cn/soapdenovo.html</a>) for the Illumina reads. In parallel, all the <em>F.</em>x <em>ananassa</em> Illumina reads were assembled using SOAPdenovo v1.05<sup><a reveal-id="DST049C32" data-open="DST049C32" class="link link-ref link-reveal xref-bibr">32</a></sup> with <em>k</em>-mer = 75. After the assembly of the Illumina reads, the gaps on the scaffolds were closed by GapCloser 1.10 (<em>p</em> = 31). Illumina-specific scaffolds and contigs, when compared with the 454 scaffolds, were selected by conducting BLASTN<sup><a reveal-id="DST049C33" data-open="DST049C33" class="link link-ref link-reveal xref-bibr">33</a></sup> searches with an <em>E</em>-value cut-off of 1<em>E</em>−10. To eliminate heterozygous bases, the Illumina-specific scaffolds and contigs were re-assembled according to the approach employed to assemble the fire ant genome.<sup><a reveal-id="DST049C34" data-open="DST049C34" class="link link-ref link-reveal xref-bibr">34</a></sup> First, the sequences of Illumina-specific scaffolds and unassembled contigs were divided into 300-bp lengths with 200-bp overlaps. The divided sequences were then re-assembled using Newbler 2.7 in the heterozygotic mode. In cases when the divided sequences were not joined by Newbler 2.7, they were merged based on the overlapping sequences.</p><p>Probable contaminating sequences on the 454 scaffolds and the Illumina-specific sequences were identified and removed using BLASTN searches against the chloroplast genome sequence of <em>F. vesca</em> (accession number: NC_015206.1), mitochondrial genome sequence of <em>Arabidopsis thaliana</em> (accession number: NC_001284.2), bacterial genome sequences registered in NCBI (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</a>), and vector sequences from UniVec (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/tools/vecscreen/univec/">http://www.ncbi.nlm.nih.gov/tools/vecscreen/univec/</a>) with an <em>E</em>-value cut-off of 1<em>E</em>−10 and length coverage of ≥10%. The cleaned Illumina scaffolds, contigs, and singlets were integrated with the 454 scaffolds and designated as FANhybrid_r1.2.</p>                            <h3 scrollto-destination=5971270 id="5971270" class="section-title" data-legacy-id=s2f>2.6. Assembly of Illumina reads from <em>F. x ananassa</em> and wild species</h3>
<p>The assembly of Illumina reads from <em>F.</em>x <em>ananassa</em> was performed along with that of reads from the four wild species, <em>F. iinumae</em>, <em>F. nipponica</em>, <em>F. nubicola</em>, and <em>F. orientalis</em> (Fig. <a reveal-id="DST049F2" data-open="DST049F2" class="link link-reveal link-table xref-fig">2</a>). The Illumina reads in each species were assembled using SOAPdenovo v1.05 with <em>k</em>-mer = 75. The gaps on the scaffolds in each species were closed by GapCloser 1.10 (<em>p</em> = 31). Probable contaminating sequences were identified and removed as described in the above section. The cleaned scaffolds and contigs ≥300 bp in length were designated FAN_r1.1 (<em>F.</em>x <em>ananassa</em>), FII_r1.1 (<em>F. iinumae</em>), FNI_r1.1 (<em>F. nipponica</em>), FNU_r1.1 (<em>F. nubicola</em>), and FOR_r1.1 (<em>F. orientalis</em>).</p>                            <h3 scrollto-destination=5971272 id="5971272" class="section-title" data-legacy-id=s2g>2.7. Repetitive sequences</h3>
<p>Probable repetitive sequences in the assembled genome of the <em>Fragaria</em> species were identified by RepeatScout<sup><a reveal-id="DST049C35" data-open="DST049C35" class="link link-ref link-reveal xref-bibr">35</a></sup> with default parameters based on the discovery of repetitive substrings in the sequences. In parallel, similarity searches and repeat masking were performed by RepeatMasker (<a class="link link-uri" href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>) on the assembled genome sequences of the <em>Fragaria</em> species against known repetitive sequences registered in RepBase.<sup><a reveal-id="DST049C36" data-open="DST049C36" class="link link-ref link-reveal xref-bibr">36</a></sup> To identify novel repetitive sequences, the assembled genome sequences, with known repetitive sequences masked, were subjected to second-round similarity search by RepeatMasker against probable repetitive sequences detected by RepeatScout. The SSR motifs with equal or greater numbers of defined repeats were searched for in exons, introns, and entire regions on the assembled genomes using SciRoKo with misa mode.<sup><a reveal-id="DST049C37" data-open="DST049C37" class="link link-ref link-reveal xref-bibr">37</a></sup> The defined numbers of repeats in mono-, di-, tri-, tetra-, penta-, and hexa-motifs were 12, 6, 7, 5, 5, and 5, respectively.</p>                            <h3 scrollto-destination=5971274 id="5971274" class="section-title" data-legacy-id=s2h>2.8. Assignment of RNA-encoding genes</h3>
<p>Transfer RNA genes were predicted by using tRNAscan-SE ver. 1.23<sup><a reveal-id="DST049C38" data-open="DST049C38" class="link link-ref link-reveal xref-bibr">38</a></sup> with default parameters. Ribosomal RNA genes were identified by BLAT searches in BLAST output format, with an <em>E</em>-value cut-off of 1<em>E</em>−10 and length cut-off of 70 bp, using 5.8S rRNA (accession number: X15589.1), 18S rRNA (accession number: X15590.1), and 26S rRNA (accession number: X58118.1), in <em>F.</em>x <em>ananassa</em>.</p>                            <h3 scrollto-destination=5971276 id="5971276" class="section-title" data-legacy-id=s2i>2.9. Gene prediction and annotation</h3>
<p>The FANhybrid_r1.2 and the five Illumina-assembled genomes were subjected to gene prediction and modelling using Augustus 2.7<sup><a reveal-id="DST049C39" data-open="DST049C39" class="link link-ref link-reveal xref-bibr">39</a></sup> with a training set from <em>A. thaliana</em> and the following parameters; gene model = partial; protein, introns, start, stop, cds, coding seq, gff3, and UTR = on, and alternatives-from-evidence and alternatives-from-sampling = true. The genes related to transposable elements (TEs) were predicted according to the domain and product names of genes identified by similarity search in two databases, the Interpro database<sup><a reveal-id="DST049C40" data-open="DST049C40" class="link link-ref link-reveal xref-bibr">40</a></sup> and the NCBI's non-redundant protein sequence (NR) database (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</a>). Similarity searches in the former and later databases were performed by InterProScan<sup><a reveal-id="DST049C41" data-open="DST049C41" class="link link-ref link-reveal xref-bibr">41</a></sup> and BLASTX<sup><a reveal-id="DST049C33" data-open="DST049C33" class="link link-ref link-reveal xref-bibr">33</a></sup> with an <em>E</em>-value cut-off of 1.0 and 1<em>E</em>−10, respectively. The Gypsy-type TEs were further identified by a search using the hmmscan module in HMMER 3.0<sup><a reveal-id="DST049C42" data-open="DST049C42" class="link link-ref link-reveal xref-bibr">42</a></sup> against the hidden Markov model in the Gypsy Database 2.0.<sup><a reveal-id="DST049C43" data-open="DST049C43" class="link link-ref link-reveal xref-bibr">43</a></sup></p>                            <h3 scrollto-destination=5971278 id="5971278" class="section-title" data-legacy-id=s2j>2.10. Comparative analysis between <em>F. x ananassa</em> and wild species</h3>
<p>BLAT searches<sup><a reveal-id="DST049C44" data-open="DST049C44" class="link link-ref link-reveal xref-bibr">44</a></sup> were performed for the sequences in FANhyblid_r1.2 against the pseudomolecules of <em>F. vesca</em> (v1.1) with −minscore = 100 and −minIdentity = 95. Subsequently, the sequences in FANhybrid_r1.2 were aligned with the genomic sequences of <em>F. vesca</em> based on the results of the BLAT searches. Furthermore, the hit regions were selected according to the following criteria: (match score + number of gaps in query) × 100/query length (%) ≥ 80, hit region in query × 100/query length (%) ≥ 80, match score/query length ≥ 25, 80 ≤ (hit region in subject × 100/query length) ≤ 120. The best alignments in each query sequence were identified approximately roughly identified using the pslReps program<sup><a reveal-id="DST049C44" data-open="DST049C44" class="link link-ref link-reveal xref-bibr">44</a></sup> in BLAT with –minCover = 0.20. Validation and modification of the results were manually performed by eye. Furthermore, similarity searches were performed for the five Illumina-assembled genome sequences against FANhybrid_r1.2 by BLAT, and these sequences were aligned with the genome sequences of FANhybrid_r1.2 using the pslReps program as described above. In parallel, the BLASTN searches with an <em>E</em>-value cut-off of 1<em>E</em>−10 were performed between each possible pair of assembled genomes to investigate sequence similarity across the assembled genome sequences.</p><p>BLAT searches were also performed for the five assembled genomes against the 34 809 candidate genes estimated on the <em>F. vesca</em> genome<sup><a reveal-id="DST049C15" data-open="DST049C15" class="link link-ref link-reveal xref-bibr">15</a></sup> with the same parameters as described above. In addition, a total of 104 annotated candidate genes in <em>F. vesca</em><sup><a reveal-id="DST049C13" data-open="DST049C13" class="link link-ref link-reveal xref-bibr">13</a></sup> were selected for further investigation of degree of duplication in the assembled <em>Fragaria</em> genome sequences. Similarity searches between these 104 genes and the assembled sequences were performed by the FASTA36 program,<sup><a reveal-id="DST049C45" data-open="DST049C45" class="link link-ref link-reveal xref-bibr">45</a></sup> based on a cut-off value of 90% identity. In parallel, Illumina-assembled genes predicted by Augustus 2.7 were subjected to cluster analysis with the 34 809 <em>F. vesca</em> candidate genes by using CD-hit v4.6.1<sup><a reveal-id="DST049C46" data-open="DST049C46" class="link link-ref link-reveal xref-bibr">46</a></sup> with the following parameters: <em>c</em> = 0.8, aS = 0.1.</p>                            <h2 scrollto-destination=5971281 id="5971281" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=5971282 id="5971282" class="section-title" data-legacy-id=s3a>3.1. Phylogenetic analysis of <em>Fragaria</em> species with SSR markers</h3>
<p>A total of 632 SSR markers, which generated 954 loci mapped onto the consensus map of <em>F.</em>x <em>ananassa</em>,<sup><a reveal-id="DST049C9" data-open="DST049C9" class="link link-ref link-reveal xref-bibr">9</a></sup> were employed for phylogenetic analysis of 20 wild <em>Fragaria</em> species and 4 <em>F.</em>x <em>ananassa</em> varieties. The average number of observed allelic peaks per marker was 11.1, ranging from 1 to 57 in all the tested species, with an average of 1.8 in each line, ranging from 1.0 in <em>Fragaria daltoniana</em> to 3.2 in <em>F.</em>x <em>ananassa</em> ‘Hokowase’. The genetic distances between all combinations of any two lines were calculated based on the presence or absence of 10 830 allelic peaks derived from the 632 markers. The genetic distances among the 24 lines ranged from 0.07 to 0.33 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). On the constructed phylogenetic tree, the 24 lines were classified into four clusters (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>A). The octoploids were clearly distinguished from <em>Fragaria</em> species of other ploidy (cluster C). Among the octoploids, two wild species, <em>F. virginiana</em> and <em>F. chiloensis</em>, were distinguishable from the cultivated species, <em>F.</em>x <em>ananassa</em>. The clusters A, B, and D consisted of six <em>Fragaria</em> species each. The relative genetic relations did not show large differences between a phylogenetic trees constructed with all samples and di-, tetra-, or octoploid-specific trees. Based on the phylogenetic analysis, the following four wild species were selected as representatives of Clusters A, B, and D for subsequent whole-genome sequencing: <em>F. nipponica</em> (Cluster A), <em>F. iinumae</em> (Cluster B), <em>F. nubicola</em> (Cluster D), and <em>F. orientalis</em> (Cluster D). Plural species were selected from Cluster D, because the cluster showed the closest genetic distance to Cluster C.</p><p>Genetic distances were investigated for each of the LGs on the consensus map according to the locations of the tested markers<sup><a reveal-id="DST049C9" data-open="DST049C9" class="link link-ref link-reveal xref-bibr">9</a></sup> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). The mean genetic distances between <em>F.</em>x <em>ananassa</em> ‘Reikou’ and the 20 wild relatives ranged from 0.24 (LG3A) to 0.34 (LG3D). In comparison, among the homoeologous groups (HGs), HG5 showed the least variation between the corresponding LGs (0.28–0.30), whereas HG3 showed the greatest variation between the corresponding LGs (0.24–0.34). <em>Fragaria vesca</em> and <em>F. chinensis</em> showed closer genetic distances to <em>F.</em>x <em>ananassa</em> than any other diploid wild species for every LG except LG5D, for which the closest diploid species was <em>F. iinumae</em>.</p>                            <h3 scrollto-destination=5971285 id="5971285" class="section-title" data-legacy-id=s3b>3.2. Genome sequencing and quality control</h3>
<p>A total of 6 211 718 reads were obtained from <em>F.</em>x <em>ananassa</em> SE and PE libraries using the Roche 454 GS FLX+ platform (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). The number of obtained Illumina reads in <em>F.</em>x <em>ananassa</em>, <em>F. iinumae</em>, <em>F. nipponica</em>, <em>F. nubicola</em>, and <em>F. orientalis</em> were 4 219 377 380, 1 078 824 432, 1 075 841 020, 1 074 353 610, and 1 050 815 546, respectively. After quality control, 33% of the Illumina reads were excluded from further analysis. Consequently, the total bases that were subjected to subsequent analysis were 276.6, 70.8, 72.2, 74.3, and 74.5 Gb, in <em>F.</em>x <em>ananassa</em>, <em>F. iinumae</em>, <em>F. nipponica</em>, <em>F. nubicola</em>, and <em>F. orientalis</em>, respectively.</p>                            <h3 scrollto-destination=5971287 id="5971287" class="section-title" data-legacy-id=s3c>3.3. Estimation of genome size based on the Illumina reads</h3>
<p>The genome sizes of <em>F.</em>x <em>ananassa</em> and the four wild relatives were estimated based on a <em>k</em>-mer = 17 frequency of the Illumina reads (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). The genome sizes of the five <em>Fragaria</em> species were estimated as follows based on the distributions of distinct <em>k</em>mers along with consideration of the <em>F. vesca</em> genome size (209.8 Mb)<sup><a reveal-id="DST049C13" data-open="DST049C13" class="link link-ref link-reveal xref-bibr">13</a></sup> and polyploidy in each species: <em>F.</em>x <em>ananassa</em> = 692 Mb, <em>F. iinumae</em> = 221 Mb, <em>F. nipponica</em> = 208 Mb, <em>F. nubicola</em> = 202 Mb, and <em>F. orientalis</em><em>=</em> 349.3 Mb.</p>                            <h3 scrollto-destination=5971289 id="5971289" class="section-title" data-legacy-id=s3d>3.4. Assembly of the <em>F. x ananassa</em> reference genome</h3>
<p>All 454 reads were assembled using Newbler 2.7 in the heterozygotic mode (Fig. <a reveal-id="DST049F1" data-open="DST049F1" class="link link-reveal link-table xref-fig">1</a>). During assembly, heterozygous bases on the 454 reads were adopted based on the comparison of depth and accuracy between the bases. After the gaps were closed by GapCloser 1.0, a total of 7598 scaffolds consisting of 101 218 723 bp with 19 520 552 N were obtained (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). The longest contig was 348 406 bp, and N50 was 46 803 bp.</p><p>In parallel, a total of 4 602 723 scaffolds and contigs consisting of 1 301 006 845 bp were generated using SOAPdenovo v1.05 and GapCloser 1.10. After the BLASTN search of the Illumina sequences against the 454 scaffolds, 1 153 521 Illumina-specific contigs were obtained, comprising 213.5 Mb. To assemble the Illumina-specific sequences using Newbler 2.7, the sequences that were longer than 300 bp were divided into 300-bp long sequences with 200-bp overlaps. As a result, a total of 1 437 769 Illumina sequences were re-assembled onto 40 416 215 bp (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). In addition, 137 608 unassembled singlets were obtained, including 10 330 repeats and 44 686 outliers. The total length of Illumina-specific sequences was 72 010 849 bp, and the N50 was 411 bp. The integrated 454 scaffolds and Illumina-specific sequences were qualified as a reference genome for <em>F.</em>x <em>ananassa</em>, designated FANhybrid_r1.2. The number of sequences, total length, N50, and GC% of the FANhybrid_r1.2 were: 211 588 sequences, 173 229 572 bp, 5137 bp, and 38.4%, respectively (Table <a reveal-id="DST049TB1" data-open="DST049TB1" class="link link-reveal link-table xref-fig">1</a>). </p>                        <a id="5971292" scrollto-destination="5971292"></a>
<div content-id="DST049TB1" class="table-modal"><div class="table-wrap"><div id="DST049TB1" class="table-wrap-title" data-id="DST049TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Statistics regarding the assembled genome sequences for <em>F.</em>x <em>ananassa</em> and four wild <em>Fragaria</em> species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>FANhybrid_r1.2 </th><th>FAN_r1.1 </th><th>FII_r1.1 </th><th>FNI_r1.1 </th><th>FNU_r1.1 </th><th>FOR_r1.1 </th></tr></thead><tbody><tr><td>Number of sequences </td><td>211 588 </td><td>625 966 </td><td>117 822 </td><td>215 024 </td><td>210 780 </td><td>323 163 </td></tr><tr><td>Total length (bp) </td><td>173 229 572 </td><td>697 765 214 </td><td>199 627 645 </td><td>206 414 979 </td><td>203 686 576 </td><td>214 184 046 </td></tr><tr><td>Average length (bp) </td><td>819 </td><td>1115 </td><td>1694 </td><td>960 </td><td>966 </td><td>663 </td></tr><tr><td>Maximum length (bp) </td><td>348 406 </td><td>51 398 </td><td>36 387 </td><td>32 113 </td><td>27 253 </td><td>12 564 </td></tr><tr><td>N50 length (bp) </td><td>5137 </td><td>2201 </td><td>3309 </td><td>1275 </td><td>1291 </td><td>722 </td></tr><tr><td>A </td><td>47 398 175 </td><td>201 607 452 </td><td>60 846 197 </td><td>63 581 246 </td><td>62 914 664 </td><td>66 367 041 </td></tr><tr><td>T </td><td>47 314 079 </td><td>201 232 679 </td><td>60 764 847 </td><td>63 483 905 </td><td>62 877 129 </td><td>66 179 649 </td></tr><tr><td>G </td><td>29 454 229 </td><td>127 672 886 </td><td>38 950 399 </td><td>39 626 127 </td><td>38 930 937 </td><td>40 738 935 </td></tr><tr><td>C </td><td>29 542 534 </td><td>127 932 068 </td><td>39 045 461 </td><td>39 709 257 </td><td>38 949 864 </td><td>40 883 595 </td></tr><tr><td>N </td><td>19 520 555 </td><td> 39 320 129 </td><td>   20 741 </td><td>   14 444 </td><td>   13 982 </td><td>   14 826 </td></tr><tr><td>Total (A + T + C + G) </td><td>153 709 017 </td><td>658 445 085 </td><td>199 606 904 </td><td>206 400 535 </td><td>203 672 594 </td><td>214 169 220 </td></tr><tr><td>GC% (GC/ATGC) </td><td>38.4 </td><td>38.8 </td><td>39.1 </td><td>38.4 </td><td>38.2 </td><td>38.1 </td></tr></tbody></table></div><div content-id="DST049TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>FANhybrid_r1.2 </th><th>FAN_r1.1 </th><th>FII_r1.1 </th><th>FNI_r1.1 </th><th>FNU_r1.1 </th><th>FOR_r1.1 </th></tr></thead><tbody><tr><td>Number of sequences </td><td>211 588 </td><td>625 966 </td><td>117 822 </td><td>215 024 </td><td>210 780 </td><td>323 163 </td></tr><tr><td>Total length (bp) </td><td>173 229 572 </td><td>697 765 214 </td><td>199 627 645 </td><td>206 414 979 </td><td>203 686 576 </td><td>214 184 046 </td></tr><tr><td>Average length (bp) </td><td>819 </td><td>1115 </td><td>1694 </td><td>960 </td><td>966 </td><td>663 </td></tr><tr><td>Maximum length (bp) </td><td>348 406 </td><td>51 398 </td><td>36 387 </td><td>32 113 </td><td>27 253 </td><td>12 564 </td></tr><tr><td>N50 length (bp) </td><td>5137 </td><td>2201 </td><td>3309 </td><td>1275 </td><td>1291 </td><td>722 </td></tr><tr><td>A </td><td>47 398 175 </td><td>201 607 452 </td><td>60 846 197 </td><td>63 581 246 </td><td>62 914 664 </td><td>66 367 041 </td></tr><tr><td>T </td><td>47 314 079 </td><td>201 232 679 </td><td>60 764 847 </td><td>63 483 905 </td><td>62 877 129 </td><td>66 179 649 </td></tr><tr><td>G </td><td>29 454 229 </td><td>127 672 886 </td><td>38 950 399 </td><td>39 626 127 </td><td>38 930 937 </td><td>40 738 935 </td></tr><tr><td>C </td><td>29 542 534 </td><td>127 932 068 </td><td>39 045 461 </td><td>39 709 257 </td><td>38 949 864 </td><td>40 883 595 </td></tr><tr><td>N </td><td>19 520 555 </td><td> 39 320 129 </td><td>   20 741 </td><td>   14 444 </td><td>   13 982 </td><td>   14 826 </td></tr><tr><td>Total (A + T + C + G) </td><td>153 709 017 </td><td>658 445 085 </td><td>199 606 904 </td><td>206 400 535 </td><td>203 672 594 </td><td>214 169 220 </td></tr><tr><td>GC% (GC/ATGC) </td><td>38.4 </td><td>38.8 </td><td>39.1 </td><td>38.4 </td><td>38.2 </td><td>38.1 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5971292" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST049TB1" class="table-wrap-title" data-id="DST049TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Statistics regarding the assembled genome sequences for <em>F.</em>x <em>ananassa</em> and four wild <em>Fragaria</em> species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>FANhybrid_r1.2 </th><th>FAN_r1.1 </th><th>FII_r1.1 </th><th>FNI_r1.1 </th><th>FNU_r1.1 </th><th>FOR_r1.1 </th></tr></thead><tbody><tr><td>Number of sequences </td><td>211 588 </td><td>625 966 </td><td>117 822 </td><td>215 024 </td><td>210 780 </td><td>323 163 </td></tr><tr><td>Total length (bp) </td><td>173 229 572 </td><td>697 765 214 </td><td>199 627 645 </td><td>206 414 979 </td><td>203 686 576 </td><td>214 184 046 </td></tr><tr><td>Average length (bp) </td><td>819 </td><td>1115 </td><td>1694 </td><td>960 </td><td>966 </td><td>663 </td></tr><tr><td>Maximum length (bp) </td><td>348 406 </td><td>51 398 </td><td>36 387 </td><td>32 113 </td><td>27 253 </td><td>12 564 </td></tr><tr><td>N50 length (bp) </td><td>5137 </td><td>2201 </td><td>3309 </td><td>1275 </td><td>1291 </td><td>722 </td></tr><tr><td>A </td><td>47 398 175 </td><td>201 607 452 </td><td>60 846 197 </td><td>63 581 246 </td><td>62 914 664 </td><td>66 367 041 </td></tr><tr><td>T </td><td>47 314 079 </td><td>201 232 679 </td><td>60 764 847 </td><td>63 483 905 </td><td>62 877 129 </td><td>66 179 649 </td></tr><tr><td>G </td><td>29 454 229 </td><td>127 672 886 </td><td>38 950 399 </td><td>39 626 127 </td><td>38 930 937 </td><td>40 738 935 </td></tr><tr><td>C </td><td>29 542 534 </td><td>127 932 068 </td><td>39 045 461 </td><td>39 709 257 </td><td>38 949 864 </td><td>40 883 595 </td></tr><tr><td>N </td><td>19 520 555 </td><td> 39 320 129 </td><td>   20 741 </td><td>   14 444 </td><td>   13 982 </td><td>   14 826 </td></tr><tr><td>Total (A + T + C + G) </td><td>153 709 017 </td><td>658 445 085 </td><td>199 606 904 </td><td>206 400 535 </td><td>203 672 594 </td><td>214 169 220 </td></tr><tr><td>GC% (GC/ATGC) </td><td>38.4 </td><td>38.8 </td><td>39.1 </td><td>38.4 </td><td>38.2 </td><td>38.1 </td></tr></tbody></table></div><div content-id="DST049TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>FANhybrid_r1.2 </th><th>FAN_r1.1 </th><th>FII_r1.1 </th><th>FNI_r1.1 </th><th>FNU_r1.1 </th><th>FOR_r1.1 </th></tr></thead><tbody><tr><td>Number of sequences </td><td>211 588 </td><td>625 966 </td><td>117 822 </td><td>215 024 </td><td>210 780 </td><td>323 163 </td></tr><tr><td>Total length (bp) </td><td>173 229 572 </td><td>697 765 214 </td><td>199 627 645 </td><td>206 414 979 </td><td>203 686 576 </td><td>214 184 046 </td></tr><tr><td>Average length (bp) </td><td>819 </td><td>1115 </td><td>1694 </td><td>960 </td><td>966 </td><td>663 </td></tr><tr><td>Maximum length (bp) </td><td>348 406 </td><td>51 398 </td><td>36 387 </td><td>32 113 </td><td>27 253 </td><td>12 564 </td></tr><tr><td>N50 length (bp) </td><td>5137 </td><td>2201 </td><td>3309 </td><td>1275 </td><td>1291 </td><td>722 </td></tr><tr><td>A </td><td>47 398 175 </td><td>201 607 452 </td><td>60 846 197 </td><td>63 581 246 </td><td>62 914 664 </td><td>66 367 041 </td></tr><tr><td>T </td><td>47 314 079 </td><td>201 232 679 </td><td>60 764 847 </td><td>63 483 905 </td><td>62 877 129 </td><td>66 179 649 </td></tr><tr><td>G </td><td>29 454 229 </td><td>127 672 886 </td><td>38 950 399 </td><td>39 626 127 </td><td>38 930 937 </td><td>40 738 935 </td></tr><tr><td>C </td><td>29 542 534 </td><td>127 932 068 </td><td>39 045 461 </td><td>39 709 257 </td><td>38 949 864 </td><td>40 883 595 </td></tr><tr><td>N </td><td>19 520 555 </td><td> 39 320 129 </td><td>   20 741 </td><td>   14 444 </td><td>   13 982 </td><td>   14 826 </td></tr><tr><td>Total (A + T + C + G) </td><td>153 709 017 </td><td>658 445 085 </td><td>199 606 904 </td><td>206 400 535 </td><td>203 672 594 </td><td>214 169 220 </td></tr><tr><td>GC% (GC/ATGC) </td><td>38.4 </td><td>38.8 </td><td>39.1 </td><td>38.4 </td><td>38.2 </td><td>38.1 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5971292" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5971293 id="5971293" class="section-title" data-legacy-id=s3e>3.5. Illumina genome assembly in <em>F. x ananassa</em> and wild <em>Fragaria</em> species</h3>
<p>A total of 276.6 Gb of <em>F.</em>x <em>ananassa</em> Illumina reads were assembled by SOAPdenovo v1.05 and GapCloser 1.10. After excluding sequences of contaminating DNA, the number of sequences and total length of the Illumina-assembled sequences were 4 366 193 sequences and 1 264 283 529 bp, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>). The assembled sequences that were shorter than 300 bp in length accounted for 86% of the total were excluded from further analysis. The selected sequences were totalled 697 765 214 bp with 39 320 129 Ns, and designated as FAN_r1.1. N50 and maximum length of the FAN_r1.1 were 2201 and 51 398 bp, respectively (Table <a reveal-id="DST049TB1" data-open="DST049TB1" class="link link-reveal link-table xref-fig">1</a>).</p><p>In parallel, <em>de novo</em> genome sequence assembly was performed for Illumina reads of the four wild species using SOAPdenovo v1.05 and GapCloser 1.10. As with <em>F.</em>x <em>ananassa</em>, assembled sequences of &lt;300 bp in length were excluded from further analysis. The remaining sequences of <em>F. iinumae</em>, <em>F. nipponica</em>, <em>F. nubicola</em>, and <em>F. orientalis</em> were designated as FII_r1.1, FNI_r1.1, FNU_r1.1, and FOR_r1.1, respectively (Table <a reveal-id="DST049TB1" data-open="DST049TB1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>). The total length and N50 in the assembled genome sequences of the four wild species ranged from 199.6 (FII_r1.1) to 214.2 Mb (FOR_r1.1), and 722 (FOR_r1.1) to 3309 bp (FII_r1.1), respectively. GC% ranged from 38.1% (FOR_1.1) to 39.1% (FII_r1.1).</p>                            <h3 scrollto-destination=5971296 id="5971296" class="section-title" data-legacy-id=s3f>3.6. Repetitive sequences</h3>
<p>The total length of repetitive sequences identified by RepeatMasker was 8 697 730 bp in FANhydrid_r1.2 (5.0% of the total length); 328 305 437 bp in FAN_r1.1 (47.1%); 63 285 682 bp in FII_r1.1 (31.7%); 52 580 277 bp in FNI_r1.1 (25.5%); 49 943 156 bp in FNU_r1.1 (24.5%); and 56 272 506 bp in FOR_r1.1 (26.3%) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). When the same approach was applied to the <em>F. vesca</em> genome (v1.1), 51 223 702 bp (24.8%) was comprised of repeat sequences. Most of the identified repeats in the assembled genomes, except FANhybrid_r1.2, were novel repeats that were not registered in RepBase. The ratios of the novel repeats to the total lengths of Illumina-assembled genome sequences ranged from 22.6% (FNU_r1.1) to 44.9% (FAN_r1.1), whereas that in the FANhybrid_r1.2 was 3.9%. The ratio of the total length of known interspersed repeats to the total genome length ranged from 0.35% (FANhybrid_r1.2) to 0.94% (FAN_r1.1). The Class 1 long terminal repeat retrotransposons, including <em>Copia</em> and <em>Gypsy</em> types, were the most frequency observed in the known interspersed repeats.</p><p>The total numbers of identified di- to hexa-nucleotide SSRs were 22 456 (FANhybrid_r1.2), 110 251 (FAN_r1.1), 32 487 (FII_r1.1), 35 188 (FNI_r1.1), 34 556 (FNU_r1.1), and 32 340 (FOR_r1.1) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>). In each genome, di-nucleotide motifs were the most frequently observed type of SSR, ranging from 57.6% (FANhybrid_r1.2) to 69.6% (FII_r1.1). The (AT)<em><sub>n</sub></em> motif was the most abundantly observed in the Illumina-assembled genomes, whereas (AG)<em><sub>n</sub></em> was the most frequent motif in FANhybrid_r1.2.</p>                            <h3 scrollto-destination=5971299 id="5971299" class="section-title" data-legacy-id=s3g>3.7. RNA-encoding genes</h3>
<p>The total number of putative tRNA-encoding genes in FANhybrid_r1.2 was 300, which was less than in the <em>F. vesca</em> scaffolds (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S10</a>). Large numbers of putative tRNA-encoding genes (1720) were identified in FAN_r1.1, while 424–514 tRNA-encoding genes were predicted in the four wild Illumina-assembled genomes. The ratios of putative tRNA genes encoding amino acids ranged from 83.7% (FOR_r1.1) to 92.4% (FII_r1.1). The total number of predicted rRNA-encoding genes in FANhybrid_r1.2, FAN_r1.1, FII_r1.1, FNI_r1.1, FNU_r1.1, and FOR_r1.1 were 18, 86, 37, 53, 19, and 45, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S11</a>).</p>                            <h3 scrollto-destination=5971301 id="5971301" class="section-title" data-legacy-id=s3h>3.8. Gene prediction and annotation</h3>
<p>The total number of predicted genes in FANhybrid_r1.2 was 45 377; more genes than in <em>F. vesca</em> (v1.0) but less than in other assembled genome sequences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S12</a>). The total sequence length and N50 were 32 959 863 and 1290 bp, respectively. Total numbers of predicted genes in the wild <em>Fragaria</em> sequences ranged from 76 760 (FII_r1.1) to 99 674 (FOR_r1.1). Approximately 3.7-fold more total sequence length were identified in FAN_r1.1 compared with FANhybrid_r1.2. The N50 of the five Illumina-assembled genomes ranged from 484 (FOR_r1.1) to 948 bp (FII_r1.1).</p><p>The predicted genes were classified into TE genes versus non-TE genes (designated as intrinsic genes, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S13</a>). The following three gene categories were subsequently notated: partial genes (without start or stop codons), pseudogenes (with in-frame stop codons), and short genes (encoding fewer than 49 aa). No notations were given for predicted full-length gene sequences framed by start and stop codons. In FANhybrid_r1.2, 41 730 sequences (92% of the total sequences) were predicted to be intrinsic genes, while the other 3647 (8%) were annotated as TEs. Among the intrinsic genes in FANhybrid_r1.2, 41% did not fall neatly into any of the three categories. This ratio was higher than that in the Illumina-assembled genome sequences.</p>                            <h3 scrollto-destination=5971304 id="5971304" class="section-title" data-legacy-id=s3i>3.9. Comparative analysis between the genomes of cultivated and wild <em>Fragaria</em> species</h3>
<p>Of the 211 588 sequences in FANhybrid_r1.2, 120 703 showed significant similarity to pseudomolecules of <em>F. vesca</em> (v1.1) by BLAT search (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S14</a>). The lengths of the mapped FANhybrid_r1.2 sequences in each of the seven chromosomes ranged from 13 922 608 (Chr1) to 26 401 015 bp (Chr6). The FANhybrid_r1.2 sequences showing significant similarity according to the BLAT search were well aligned across the <em>F. vesca</em> pseudomolecules (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). The mapped FANhybrid_r1.2 sequences included 145 387 993 bp of <em>F. vesca</em> pseudomolecules, which covered 70.3% of the total length (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S14</a>). Furthermore, the MEGABLAST<sup><a reveal-id="DST049C47" data-open="DST049C47" class="link link-ref link-reveal xref-bibr">47</a></sup> search (identity% ≥90 and <em>E</em>-value cut-off of 1<em>E</em>−50) was performed for the 90 885 FANhybrid_r1.2 sequences that did not align with the <em>F. vesca</em> pseudomolecules, against the wild <em>Fragaria</em> genome sequences. Of the 90 885 FANhybrid_r1.2 sequences, 22 145 showed significant similarity against the <em>F. vesca</em> genome sequences, whereas 68 740 did not (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S15</a>). A total of 39 692 FANhybrid_r1.2 sequences (total length: 10 742 467 bp) showed no significant similarity to any of the genome sequences, and were concluded to be <em>F.</em> x <em>ananassa-</em>specific. In parallel, BLASTN searches were performed between all pairwise combinations of the assembled genomes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S16</a>). The total length of non-homologous sequences in FANhybrid_r1.2 compared with FAN_r1.1 was 1.4 Mb, whereas that in FAN_r1.1 compared with FANhybrid_r1.2 was 281.7 kb. Comparison with each of the five wild species revealed that the ratios of non-homologous sequences to total sequences of FANhybrid_r1.2 and FAN_r1.1 ranged from 4.7% (FNU_r1.1) to 5.6% (FOR_r1.1), and from 1.3% (FNU_r1.1) to 1.8% <em>(F. vesca</em> v1.1), respectively. In comparisons between the wild species, the smallest ratio of non-homologous sequences to the total was observed in FNU_r1.1 against FNI_r1.1 (0.66%), whereas the largest ratio was observed in FNI_r1.1 against FII_r1.1 (2.61%).</p><p>A BLAT search mapped 142 699 FAN_r1.1 sequences (61.8% of the total) onto the FANhybrid_r1.2 sequence (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S17</a>). Multiple FAN_r1.1 sequences were often mapped onto the same FANhybrid_r1.2 sequences. The numbers of single, double, triple, and quadruple FAN_r1.1 sequences that mapped onto the same FANhybrid_r1.2 sequences were 64 828 (45.4% of the mapped total), 31 922 (22.4%), 13 656 (9.6%), and 6289 (4.4%), respectively. The FANhybrid_r1.2 sequences that mapped onto five or more FAN_r1.1 sequences tended to be classified as repetitive sequences (Fig. <a reveal-id="DST049F3" data-open="DST049F3" class="link link-reveal link-table xref-fig">3</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). The sequence of each wild species was then mapped onto the entire genomic sequence of FANhybrid_r1.2. The ratios of single and double sequences from the wild Illumina-assembled genomes that mapped onto the same FANhybrid_r1.2 sequences ranged from 62.4% (FOR_r1.1) to 87.3% (FII_r1.1), and from 9.5% (FII_r1.1) to 18.5% (FOR_r1.1), respectively. The ratios of single and double sequences from the <em>F. vesca</em> genome that mapped onto the same FANhybrid_r1.2 sequences were 72.7 and 4.8%, respectively. <em>Fragaria vesca</em> generated the largest number of top-hit sequences in each chromosome, followed by <em>F. iinumae</em> (Fig. <a reveal-id="DST049F3" data-open="DST049F3" class="link link-reveal link-table xref-fig">3</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>). The BLASTN search was furthermore performed for the FAN_r1.1 against the wild <em>Fragaria</em> genome sequences. Of the 625 966 FAN_r1.1 sequences, 5447 (0.9%) were not identified significant similarity to the wild <em>Fragaria</em> sequences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S18</a>). The ratios of top-hit sequences in FII_r1.1, FNI_r1.1, FNU_r1.1, FOR_r1.1, and <em>F. vesca</em> (v1.1) to the FAN_r1.1 sequences were 39.1, 8.9, 8.1, 8.2, and 34.8%, respectively. </p>                        <a id="5971307" scrollto-destination="5971307"></a>
<div id="DST049F3" data-id="DST049F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst049/2/m_dst04903.png?Expires=2147483647&Signature=bb55IrtXfkADTjkbjmCtWaqAlZx5Qu3hFealtcuWBQSwtWHir6AfRulY4cnhaMnFL0pMfXgS9maFpGoz--1dCsUtDygPFB6fEiS0fD9k-Kckb8q8wWTYZX9iIOfaTsAc5zHR7xr3gEMxA~07MD0PhLnqGCM3zg8jGsUdGpWyLpuEUqCG8gad0b-Aj1zzzFZZZDYjxacC-q53Tw0DkbQOPBii3xPmfsJBcHK5uHSxg53OoBwoIhoBcSoNsv9EJQ7OZMLIcS-E~neqM~RUMlqlJQZgW4-w9scOIRQQM4~Obm3sV6G1MZKcFsOLjcVGyFw~Df9o0KJjv0m4TcBycC58bQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Position and coverage of the Illumina-assembled genome sequences mapped onto the FANhybrid_r1.2 sequence (left), and frequency of the top-hit sequences from the five wild Fragaria species against FANhybrid_r1.2 (right). The central black and white bars indicate the genome sequences of FANhybrid_r1.2 aligned based on the homologous sequence positions on F. vesca (v1.1) Chr1. Details are described in Supplementary Figs S3 and S4." path-from-xml="dst04903.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5971307/dst04903.tiff" path-from-xml="dst04903.tiff" target="_blank">View large</a><a section="5971307" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst049/2/dst04903.png?Expires=2147483647&Signature=eXdk6LnQ8V~DSSfuOtD6hxXJwueTCosuXioZoA9ghykRXylxmxYPwY1fcNjF6Muj7uZrUcH7h7IbBqXJeilaDS9gdXOVgZrRdPw8sbdbc9eIxQR5mlSJrSARARAEr4cCGumrwJXXmABQGdbbLMCNixeFPeiCYXw47znbv0tvENv9ZIyrYPVBxunI4w0W9ZNUAXoYngTWdCZ1x-D5Xg33r7~ysWekuFRx2dqYady0z7UHNMOEg8JvitZ-vcHhBq0AyFWSmzG~Mn5Yk~WgjKgKuTBYmi10eNR8-Hb6c4P5OSYlfZR6bd7k~eLIH2P8TVaZhBntAiwXew~XBAuyhcT7cw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5971307&ar=404005&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04903.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Position and coverage of the Illumina-assembled genome sequences mapped onto the FANhybrid_r1.2 sequence (left), and frequency of the top-hit sequences from the five wild <em>Fragaria</em> species against FANhybrid_r1.2 (right). The central black and white bars indicate the genome sequences of FANhybrid_r1.2 aligned based on the homologous sequence positions on <em>F. vesca</em> (v1.1) Chr1. Details are described in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figs S3 and S4</a>.</p></div></div><div content-id="DST049F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst049/2/m_dst04903.png?Expires=2147483647&Signature=bb55IrtXfkADTjkbjmCtWaqAlZx5Qu3hFealtcuWBQSwtWHir6AfRulY4cnhaMnFL0pMfXgS9maFpGoz--1dCsUtDygPFB6fEiS0fD9k-Kckb8q8wWTYZX9iIOfaTsAc5zHR7xr3gEMxA~07MD0PhLnqGCM3zg8jGsUdGpWyLpuEUqCG8gad0b-Aj1zzzFZZZDYjxacC-q53Tw0DkbQOPBii3xPmfsJBcHK5uHSxg53OoBwoIhoBcSoNsv9EJQ7OZMLIcS-E~neqM~RUMlqlJQZgW4-w9scOIRQQM4~Obm3sV6G1MZKcFsOLjcVGyFw~Df9o0KJjv0m4TcBycC58bQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Position and coverage of the Illumina-assembled genome sequences mapped onto the FANhybrid_r1.2 sequence (left), and frequency of the top-hit sequences from the five wild Fragaria species against FANhybrid_r1.2 (right). The central black and white bars indicate the genome sequences of FANhybrid_r1.2 aligned based on the homologous sequence positions on F. vesca (v1.1) Chr1. Details are described in Supplementary Figs S3 and S4." path-from-xml="dst04903.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5971307/dst04903.tiff" path-from-xml="dst04903.tiff" target="_blank">View large</a><a section="5971307" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst049/2/dst04903.png?Expires=2147483647&Signature=eXdk6LnQ8V~DSSfuOtD6hxXJwueTCosuXioZoA9ghykRXylxmxYPwY1fcNjF6Muj7uZrUcH7h7IbBqXJeilaDS9gdXOVgZrRdPw8sbdbc9eIxQR5mlSJrSARARAEr4cCGumrwJXXmABQGdbbLMCNixeFPeiCYXw47znbv0tvENv9ZIyrYPVBxunI4w0W9ZNUAXoYngTWdCZ1x-D5Xg33r7~ysWekuFRx2dqYady0z7UHNMOEg8JvitZ-vcHhBq0AyFWSmzG~Mn5Yk~WgjKgKuTBYmi10eNR8-Hb6c4P5OSYlfZR6bd7k~eLIH2P8TVaZhBntAiwXew~XBAuyhcT7cw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5971307&ar=404005&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04903.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Position and coverage of the Illumina-assembled genome sequences mapped onto the FANhybrid_r1.2 sequence (left), and frequency of the top-hit sequences from the five wild <em>Fragaria</em> species against FANhybrid_r1.2 (right). The central black and white bars indicate the genome sequences of FANhybrid_r1.2 aligned based on the homologous sequence positions on <em>F. vesca</em> (v1.1) Chr1. Details are described in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figs S3 and S4</a>.</p></div></div>                            <h3 scrollto-destination=5971308 id="5971308" class="section-title" data-legacy-id=s3j>3.10. Comparative analysis between the genes in <em>F. vesca</em> and the assembled sequences in the <em>Fragaria</em> species</h3>
<p>To investigate the degree of gene duplication in each assembled genome, BLAT searches were performed for the six assembled genome sequences against the 34 809 candidate genes identified on the <em>F. vesca</em> genome.<sup><a reveal-id="DST049C15" data-open="DST049C15" class="link link-ref link-reveal xref-bibr">15</a></sup> The numbers of BLAT hit sequences in FANhybrid_r1.2, FAN_r1.1, FII_r1.1, FNI_r1.1, FNU_r1.1, and FOR_r1.1 were 27 718 (79.6% of the 34 809 candidate genes), 32 120 (92.3%), 27 560 (79.2%), 29 323 (84.2%), 31 869 (91.6%), and 31 901 (91.6%), respectively. Single hits were the most frequently observed in all the assembled genome sequences except FAN_r1.1 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6a</a>). The distribution of FAN_r1.1 differed from those of the other assembled genome sequences; the peak in frequency was quite broad, and triple hits were the most frequently observed. Of the four wild species, the <em>F. orientalis</em> (FOR_r1.1) genome showed broader peaks than the others, whereas sharper peaks were observed in the <em>F. iinumae</em> genome (FII_r1.1). Cluster analysis was performed on the gene sequences predicted as ‘intrinsic gene’ and ‘intrinsic gene/partial’ (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S13</a>) in the Illumina-assembled genome sequences of <em>F.</em>x <em>ananassa</em> and the diploids (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7</a>) against 34 809 <em>F. vesca</em> candidate genes. The numbers of gene sequences that did not cluster in any other species were 24 596 (14.2% of the total sequences) in FAN_r1.1; 3989 (6.7%) in FII_r1.1; 6351 (8.8%) in FNI_r1.1; 3399 (4.8%) in FNU_r1.1; and 1492 (4.6%) in <em>F. vesca</em>. The BLAT search results were further investigated for 104 genes that were annotated as agriculturally important traits in a previous study<sup><a reveal-id="DST049C15" data-open="DST049C15" class="link link-ref link-reveal xref-bibr">15</a></sup> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S19</a>). The numbers of genes that had no hits to any of the assembled genomes were as follows: 17 (FANhybrid_r1.2), 10 (FII_r1.1), 3 (FNI_r1.1), 7 (FNU_r1.1), and 6 (FOR_r1.1). All the <em>F. vesca</em> genes showed hits to the FAN_r1.1 sequences. The distributions of the hit sequences were similar to the BLAT search results against the 34 809 genes, except the highest numbers of hit sequences in FNI_r1.1, and FOR_r1.1 were 2 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6b</a>).</p>                            <h2 scrollto-destination=5971310 id="5971310" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>The phylogenetic analysis with 632 markers revealed that <em>F. vesca</em> was the most closely related diploid species to <em>F.</em>x <em>ananassa.</em> The constructed phylogenetic tree suggested that <em>F. vesca</em> and <em>F. nubicola</em> were genetically closer to <em>F.</em>x <em>ananassa</em> than <em>F. iinumae</em> and <em>F. nipponica.</em> A comparison of the genetic distances between all pairs of the three diploids, <em>F. iinumae</em>, <em>F. nipponica</em>, and <em>F. nubicola</em>, indicated that <em>F. iinumae</em> and <em>F. nipponica</em> were more closely related than the other pairs. This result diverged from that of a previous report,<sup><a reveal-id="DST049C10" data-open="DST049C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DST049C14" data-open="DST049C14" class="link link-ref link-reveal xref-bibr">14</a></sup> which distinguished the genome of <em>F. iinumae</em> from those of <em>F. nipponica</em> and <em>F. nubicola</em>. It is considered that differences in investigated regions and scales may be responsible for the inconsistency to the previous studies. We concluded that the four wild species, <em>F. iinumae</em>, <em>F. nipponica</em>, <em>F. nubicola</em>, and <em>F. orientalis</em>, were good representatives of the genetic diversity in the genus <em>Fragaria</em> and were subjected to further genome sequencing analysis.</p><p>The genome size of <em>F.</em>x <em>ananassa</em> was estimated as 692 Mb, based on the multiplicity of distinct <em>k</em>-mers in the Illumina reads. Despite the similarity in size to that determined in previous studies (708–720 Mb),<sup><a reveal-id="DST049C2" data-open="DST049C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DST049C3" data-open="DST049C3" class="link link-ref link-reveal xref-bibr">3</a></sup> it was less than 4-fold the size of the <em>F. vesca</em> genome (209.8 Mb).<sup><a reveal-id="DST049C13" data-open="DST049C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Similarly, the genome size of <em>F. orientalis</em> (349.3 Mb) was less than 2-fold the size of the <em>F. vesca</em> genome. Therefore, we suspected that the genome sizes of <em>F.</em>x <em>ananassa</em> and <em>F. orientalis</em> were underestimated. The estimated genome size of the three diploids, <em>F. iinumae</em>, <em>F. nipponica</em>, and <em>F. nubiclola</em>, was similar to that of <em>F. vesca</em>, and we therefore concluded that the estimated values were close to the true values. Most of the species, except <em>F. orientalis</em>, showed two peaks in the distribution of the number of distinct <em>k</em>-mers, and the distinct peaks detected on larger multiplicity were employed for the genome size estimation. Because not all the sequenced materials were from homozygous lines, the peaks identified on smaller multiplicity were assumed to stem from heterozygous sequences. The peaks employed for genome size estimations in <em>F.</em>x <em>ananassa</em> and <em>F. orientalis</em> were quite broad. The broad shape of the peaks may have resulted from the homoeologous nature of the sequences, and this could also have led to the underestimation of the genome sizes.</p><p>The largest obstacle to genome sequence assembly of octoploid <em>F.</em>x <em>ananassa</em> was the homoeology of the subgenomes. In addition, outcrossing behaviour of <em>F.</em>x <em>ananassa</em> generates allelic heterozygosity within pairs of homoeologous genomes. Since up to eight heterozygous sequences exist in the <em>F.</em>x <em>ananassa</em> genome, it was predicted that this heterozygosity would create difficulty in assembling chromosome-specific sequences. Therefore, we tried to construct a virtual reference genome that could integrate genome sequences of homoeologous or heterozygous chromosomes by eliminating heterozygous bases. A number of assembling programmes were developed for the assembly of sequences generated by the NGS platforms. The base strategies were mainly classified into two methods: the overlap graph method and the de Bruijn graph method.<sup><a reveal-id="DST049C48" data-open="DST049C48" class="link link-ref link-reveal xref-bibr">48</a>,<a reveal-id="DST049C49" data-open="DST049C49" class="link link-ref link-reveal xref-bibr">49</a></sup> The overlap graph method lays out a consensus paradigm based on overlapping sequences, whereas the de Bruijn graph method segments sequence reads into <em>k</em>-mers, and then assembles <em>k</em>-mers based on paths represented on the graphed reads. Because the overlap graph method assumes similarities between the reads in sequence assembly, it is able to eliminate heterozygous bases. The Newbler 2.7 and SOAPdenovo v1.05 are typical assemblers employing the overlap graph method and the de Bruijn graph method, respectively. Therefore, Newbler 2.7 was used for the assembly of the reference genome sequences, whereas SOAPdenovo v1.05 was used for the assembly of the sequence that maintained heterozygosity. The total length of the assembled genome, FANhybrid_r1.2, was 173.2 Mb, which was shorter than that of <em>F. vesca</em>. The N50 of the 454 scaffolds was 46 803 bp, whereas that of the Illumina scaffolds and contigs was 411 bp. Therefore, FANhybrid_r1.2 consisted of long 454 scaffolds and short Illumina sequences.</p><p>The total length of all Illumina-assembled genome sequences in <em>F.</em>x <em>ananassa</em> was 1264.3 Mb, which was ∼2-fold the genome size estimated by the Jellyfish program, and more than 6-fold the genome size of <em>F. vesca</em>. The N50 was quite short, 406 bp. The long total length and short N50 suggested that assembly of the Illumina reads was obstructed by large numbers of heterozygous sequences. Hence, contigs shorter than 300 bp were excluded from further analysis. As a result, the total length of FAN_r1.1 was reduced to 697.8 Mb, which was close to the length estimated by Jellyfish. The result of the similarity search against the candidate genes in <em>F. vesca</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>) indicated that high heterozygosity was maintained in FAN_r1.1 sequences. As with FAN_r1.1, the total lengths of Illumina-assembled genome sequences in the three diploids were similar to the genome size estimated by Jellyfish. However, the total length of <em>F. orientalis</em> (FOR_r1.1) was 214.2 Mb, which was ∼61% of the genome size as estimated by Jellyfish. This result indicated a possibility of over-exclusion of sequences caused by the removal of sequences &lt;300 bp in length.</p><p>The total lengths of repeat sequences in the four wild Illumina-assembled genome sequences were close to that of <em>F. vesca</em>, whereas the FAN_r1.1 sequence was ∼6.4 times that of <em>F. vesca</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). This result implied that the total length of repeat sequences in FAN_r1.1 was overestimated due to the high heterozygosity in the sequences. In contrast, the total length of the repeat sequences in FANhybrid_r1.2 was quite short, 0.17 that of <em>F. vesca</em>. We considered that excessive integration had occurred as a result of the elimination of heterozygous bases through the use of the heterozygotic mode in Newbler 2.7. Similar patterns were observed in the numbers of tRNAs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S10</a>) and the length and numbers of candidate genes predicted by Augustus 2.7 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S12</a>). The frequency of SSRs ranged from 12.7 (FANhybrid_r1.2) to 16.8 (FNI_r1.1) per 100 kb across the five assembled genome sequences. This agreement in observed SSR frequency suggested that SSR identification was not affected by the heterozygosity in the assembled genome sequences. An SSR frequency in intron regions was 2- to 3-fold that in exon regions. Liu <em>et al.</em><sup><a reveal-id="DST049C50" data-open="DST049C50" class="link link-ref link-reveal xref-bibr">50</a></sup> reported loss of rDNA site numbers in octoploids by fluorescence <em>in situ</em> hybridization using 5S and 25S rDNA probes. However, we found no clear evidence of structural changes in the genomes during the evolutionary transition from diploids to octoploids based on the features of the assembled genome sequences.</p><p>The BLAST analysis between the whole assembled genome sequences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S16</a>) showed that sequences in FAN_r1.1 covered larger regions in the <em>F.</em> x <em>ananassa</em> genome than those in FANhybrid_r1.2. The total length of non-homologous sequences was only 281 769 bp in FAN_r1.1 against FANhybrid_r1.2, whereas it was 1.4 Mb in FANhybrid_r1.2 against FAN_r1.1. The ratio of non-homologous sequence to total sequence did not exceed 3% in comparisons between any two pairs of assembled sequences. It was predicted that the percentages of non-homologous sequence between the four subgenomes in <em>F.</em>x <em>ananassa</em> (A, A′, B, and B′) would reflect those between the ancestral species. Therefore, we estimated that the percentage of non-homologous sequence between the four subgenomes did not exceed 3%.</p><p>Based on the results of the BLAT analysis, 57% of the sequences in FANhybrid_r1.2 aligned with <em>F. vesca</em> pseudomolecules, while the other 43% of the sequences did not. The large number of non-aligning sequences in the BLAT analysis may be related to the strict thresholds used (i.e. the total length of the subject sequence must show significant similarity over 80–120% of the length of the query sequence). In the Illumina-assembled genomes, multiple sequences were often mapped onto the same FANhybrid_r1.2 sequences, and this might reflect the heterozygosity of the corresponding regions. This multiple mapping was observed across the genome sequence, and we concluded that the heterozygosity was randomly distributed in <em>F.</em> x <em>ananassa</em> and the four wild species sequenced in this study. In comparisons of the numbers of top-hit sequences of the five wild species, slight difference was observed between the results of BLAT analysis against FANhybrid_r1.2 and BLASTN search against FAN_r1.1. In the result of BLAT analysis, <em>F. vesca</em> showed the highest numbers of the top-hit sequences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>), while nearly equal ratio was observed in <em>F. iinumae</em> (39.1%) and <em>F. vesca</em> (34.8%) in the BLASTN search (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S18</a>). We considered that the result of BLASTN search was more accurate, because the result of BLAT analysis was affected by the length of assembled sequences in the subject (wild species) sequences. The result of BLASTN search suggested that the genomes of <em>F. vesca</em> and <em>F. iinumae</em> equally contributed as progenitors of <em>F.</em> x <em>ananassa</em>, and agreed with previous studies.<sup><a reveal-id="DST049C3" data-open="DST049C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DST049C10" data-open="DST049C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DST049C14" data-open="DST049C14" class="link link-ref link-reveal xref-bibr">14</a></sup> On the other hand, 14.2% of the FAN_r1.1 intrinsic genes were predicted as unique genes in <em>F.</em> x <em>ananassa</em>. These results suggested that the genome of <em>F.</em> x <em>ananassa</em> has diverged from the other <em>Fragaria</em> species during the process of evolution.</p><p>In this study, we dissected the octoploid <em>F.</em> x <em>ananassa</em> genome through comparisons between reference and heterozygous genome sequences in <em>F.</em> x <em>ananassa</em> and sequencing analysis of the genomes of wild <em>Fragaria</em> relatives. To our knowledge, this is the first genomic analysis of a polyploid species, and we expect that this approach will be applied to the genomic analysis of other polyploid species. On the other hand, one remarkable issue remained; that is, the distinction between homoeologous and allelic heterozygous sequences. The small percentages of non-homologous sequence across the assembled genomes suggested that most of the genes in <em>F.</em> x <em>ananassa</em> have homoeologous sequences in the genome. To distinguish subgenome-specific sequences, we considered that introducing segregation analysis into the genome sequence assembly would provide a more effective approach. Therefore, S<sub>1</sub> progenies of ‘Reikou’ were subjected to further analysis. The results obtained should contribute to progress in the genomic and genetic analyses of the octopolyploid species, <em>F.</em> x <em>ananassa</em>.</p>                            <h2 scrollto-destination=5971319 id="5971319" class="section-title" data-legacy-id=s5>5. Database</h2>
<p>The genome assembly data, annotations, and gene models of <em>F.</em>x <em>ananassa</em> and wild <em>Fragaria</em> species are available at the Strawberry GARDEN (<a class="link link-uri" href="http://strawberry-garden.kazusa.or.jp">http://strawberry-garden.kazusa.or.jp</a>). All sequence data (assembled sequences and genome sequence reads by NGSs) are available through the international databases (DDBJ/GenBank/EMBL) under the umbrella project number PRJDB1445. The accession numbers of assembled sequences, FANhybrid_r1.2, FAN_r1.1, FII_r1.1, FNI_r1.1, FNU_r1.1, and FOR_r1.1 are BATS01000001-BATS01220286, BATT01000001-BATT01714282, BATU01000001-BATU01118549, BATV01000001-BATV01215530, BATW01000001-BATW01211274, and BATX01000001-BATX01323675, respectively. The genome sequence reads obtained by Roche 454 GS FLX+ and Illumina GAII/HiSeq are available from the DDBJ Sequence Read Archive (DRA) under the accession number DRA001114.</p>                            <h2 scrollto-destination=5971321 id="5971321" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by the Kazusa DNA Research Institute Foundation.</p>                            <h2 scrollto-destination=5971323 id="5971323" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST049C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="collab">FAOSTAT</div>, <div class="year">2011</div> <div class="comment"><a class="link link-uri" href="http://faostat3.fao.org">http://faostat3.fao.org</a></div></div></div></div></div><div content-id="DST049C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Akiyama</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yamamoto</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ohmido</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Oshima</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Fukui</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Estimation of the nuclear DNA content of strawberries (<em>Fragaria</em> spp.) compared with <em>Arabidopsis thaliana</em> by using dual system flow cytometry</div>, <div class="source">Cytologia.</div> , <div class="year">2001</div>, vol. <div class="volume">66</div> (pg. <div class="fpage">431</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimation%20of%20the%20nuclear%20DNA%20content%20of%20strawberries%20(Fragaria%20spp.)%20compared%20with%20Arabidopsis%20thaliana%20by%20using%20dual%20system%20flow%20cytometry&amp;author=Y.%20Akiyama&amp;author=Y.%20Yamamoto&amp;author=N.%20Ohmido&amp;author=M.%20Oshima&amp;author=K.%20Fukui&amp;publication_year=2001&amp;journal=Cytologia.&amp;volume=66&amp;pages=431-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1508/cytologia.66.431" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1508%2Fcytologia.66.431" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1508%2Fcytologia.66.431"> </span></div> </div></div></div></div></div><div content-id="DST049C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Davis</div> <div class="given-names">T.M.</div></div>,  <div class="name"><div class="surname">Denoyes-Rothan</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Lerceteau-Köhler</div> <div class="given-names">E.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Kole</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Strawberry</div>, <div class="source">Genome Mapping and Molecular Breeding Inplants. vol. 4. Fruit and Nuts</div> , <div class="year">2007</div><div class="publisher-loc">Berlin</div><div class="publisher-name">Springer-Verlag</div>(pg. <div class="fpage">189</div>-<div class="lpage">205</div>)</div></div></div></div><div content-id="DST049C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Federova</div> <div class="given-names">N.J.</div></div>. </div><div class="article-title">Cross ability and phylogenetic relationships in the main European species of <em>Fragaria</em></div>, <div class="source">Natl. Acad. Sci. USSR</div> , <div class="year">1946</div>, vol. <div class="volume">52</div> (pg. <div class="fpage">545</div>-<div class="lpage">7</div>)</div></div></div></div><div content-id="DST049C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Senanayake</div> <div class="given-names">Y.D.A.</div></div>,  <div class="name"><div class="surname">Bringhurst</div> <div class="given-names">R.S.</div></div>. </div><div class="article-title">Origin of the <em>Fragaria</em> polyploids. I. Cytological evidence</div>, <div class="source">Am. J. Bot.</div> , <div class="year">1967</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">221</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origin%20of%20the%20Fragaria%20polyploids.%20I.%20Cytological%20evidence&amp;author=Y.D.A.%20Senanayake&amp;author=R.S.%20Bringhurst&amp;publication_year=1967&amp;journal=Am.%20J.%20Bot.&amp;volume=54&amp;pages=221-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2440801" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2440801" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2440801"> </span></div> </div></div></div></div></div><div content-id="DST049C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bringhurst</div> <div class="given-names">R.S.</div></div>. </div><div class="article-title">Cytogenetics and evolution in American <em>Fragaria</em></div>, <div class="source">HortScience</div> , <div class="year">1990</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">879</div>-<div class="lpage">81</div>)</div></div></div></div><div content-id="DST049C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kunihisa</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Studies using DNA markers in <em>F.</em> x <em>ananassa</em>: genetic analysis, genome structure, and cultivar identification</div>, <div class="source">J. Jpn. Soc. Horticultural Sci..</div> , <div class="year">2011</div>, vol. <div class="volume">80</div> (pg. <div class="fpage">231</div>-<div class="lpage">43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Studies%20using%20DNA%20markers%20in%20F.%20x%20ananassa%3A%20genetic%20analysis%2C%20genome%20structure%2C%20and%20cultivar%20identification&amp;author=M.%20Kunihisa&amp;publication_year=2011&amp;journal=J.%20Jpn.%20Soc.%20Horticultural%20Sci..&amp;volume=80&amp;pages=231-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2503/jjshs1.80.231" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2503%2Fjjshs1.80.231" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2503%2Fjjshs1.80.231"> </span></div> </div></div></div></div></div><div content-id="DST049C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sargent</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Passey</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Šurbanovski</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">A microsatellite linkage map for the cultivated strawberry (<em>Fragaria</em> x <em>ananassa</em>) suggests extensive regions of homozygosity in the genome that may have resulted from breeding and selection</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">124</div> (pg. <div class="fpage">1229</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20microsatellite%20linkage%20map%20for%20the%20cultivated%20strawberry%20(Fragaria%20x%20ananassa)%20suggests%20extensive%20regions%20of%20homozygosity%20in%20the%20genome%20that%20may%20have%20resulted%20from%20breeding%20and%20selection&amp;author=D.J.%20Sargent&amp;author=T.%20Passey&amp;author=N.%20%C5%A0urbanovski&amp;publication_year=2012&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=124&amp;pages=1229-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-011-1782-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-011-1782-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-011-1782-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22218676" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Isobe</div> <div class="given-names">S.N.</div></div>,  <div class="name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Construction of an integrated high density simple sequence repeat linkage map in cultivated strawberry (<em>Fragaria</em> x <em>ananassa</em>) and its applicability</div>, <div class="source">DNA Res.</div> , <div class="year">2013</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">79</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20an%20integrated%20high%20density%20simple%20sequence%20repeat%20linkage%20map%20in%20cultivated%20strawberry%20(Fragaria%20x%20ananassa)%20and%20its%20applicability&amp;author=S.N.%20Isobe&amp;author=H.%20Hirakawa&amp;author=S.%20Sato&amp;publication_year=2013&amp;journal=DNA%20Res.&amp;volume=20&amp;pages=79-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dss035" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdss035" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdss035"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23248204" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Staudt</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Strawberry biogeography, genetics and systematics</div>, <div class="source">Acta Hort.</div> , <div class="year">2009</div>, vol. <div class="volume">842</div> (pg. <div class="fpage">71</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strawberry%20biogeography%2C%20genetics%20and%20systematics&amp;author=G.%20Staudt&amp;publication_year=2009&amp;journal=Acta%20Hort.&amp;volume=842&amp;pages=71-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.17660/ActaHortic.2009.842.1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.17660%2FActaHortic.2009.842.1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.17660%2FActaHortic.2009.842.1"> </span></div> </div></div></div></div></div><div content-id="DST049C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hummer</div> <div class="given-names">K.E.</div></div>,  <div class="name"><div class="surname">Nathewet</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Yanagi</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Decaploidy in <em>Fragaria iturupensis</em> (Rosaceae)</div>, <div class="source">Am. J. Bot.</div> , <div class="year">2009</div>, vol. <div class="volume">96</div> (pg. <div class="fpage">713</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Decaploidy%20in%20Fragaria%20iturupensis%20(Rosaceae)&amp;author=K.E.%20Hummer&amp;author=P.%20Nathewet&amp;author=T.%20Yanagi&amp;publication_year=2009&amp;journal=Am.%20J.%20Bot.&amp;volume=96&amp;pages=713-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3732/ajb.0800285" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3732%2Fajb.0800285" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3732%2Fajb.0800285"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21628226" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Darrow</div> <div class="given-names">G.</div></div>. </div>, <div class="source">The Strawberry. History, Breeding and Physiology</div> , <div class="year">1966</div><div class="publisher-loc">New York</div><div class="publisher-name">Holt, Rinehart and Winston</div></div></div></div></div><div content-id="DST049C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Potter</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Luby</div> <div class="given-names">J.J.</div></div>,  <div class="name"><div class="surname">Harrison</div> <div class="given-names">R.E.</div></div>. </div><div class="article-title">Phylogenetic relationships among species of <em>Fragaria</em> (<em>Rosaceae</em>) inferred from non-coding nuclear and chloroplast DNA sequences</div>, <div class="source">Syst. Bot.</div> , <div class="year">2000</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">337</div>-<div class="lpage">48</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20relationships%20among%20species%20of%20Fragaria%20(Rosaceae)%20inferred%20from%20non-coding%20nuclear%20and%20chloroplast%20DNA%20sequences&amp;author=D.%20Potter&amp;author=J.J.%20Luby&amp;author=R.E.%20Harrison&amp;publication_year=2000&amp;journal=Syst.%20Bot.&amp;volume=25&amp;pages=337-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2666646" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2666646" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2666646"> </span></div> </div></div></div></div></div><div content-id="DST049C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rousseau-Gueutin</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Gaston</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Aïnouche</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Tracking the evolutionary history of polyploidy in <em>Fragaria</em> L. (strawberry): new insights from phylogenetic analyses of low-copy nuclear genes</div>, <div class="source">Mol. Phylogenet. Evol.</div> , <div class="year">2009</div>, vol. <div class="volume">51</div> (pg. <div class="fpage">515</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tracking%20the%20evolutionary%20history%20of%20polyploidy%20in%20Fragaria%20L.%20(strawberry)%3A%20new%20insights%20from%20phylogenetic%20analyses%20of%20low-copy%20nuclear%20genes&amp;author=M.%20Rousseau-Gueutin&amp;author=A.%20Gaston&amp;author=S.%20A%C3%AFnouche&amp;publication_year=2009&amp;journal=Mol.%20Phylogenet.%20Evol.&amp;volume=51&amp;pages=515-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ympev.2008.12.024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2008.12.024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ympev.2008.12.024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19166953" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shulaev</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Sargent</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Crowhurst</div> <div class="given-names">R.N.</div></div>, et al. </div><div class="article-title">The genome of woodland strawberry (<em>Fragaria vesca</em>)</div>, <div class="source">Nat, Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">109</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20woodland%20strawberry%20(Fragaria%20vesca)&amp;author=V.%20Shulaev&amp;author=D.J.%20Sargent&amp;author=R.N.%20Crowhurst&amp;publication_year=2010&amp;journal=Nat%2C%20Genet.&amp;volume=43&amp;pages=109-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.740" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.740" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.740"> </span></div> </div></div></div></div></div><div content-id="DST049C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="comment"><em>Fragaria vesca</em> Whole Genome v1.1 Assembly &amp; Annotation Genome Database for Rosaceae (GDR) <a class="link link-uri" href="http://www.rosaceae.org/species/fragaria/fragaria_vesca/genome_v1.1">http://www.rosaceae.org/species/fragaria/fragaria_vesca/genome_v1.1</a></div></div></div></div></div><div content-id="DST049C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koskela</div> <div class="given-names">E.A.</div></div>,  <div class="name"><div class="surname">Mouhu</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Albani</div> <div class="given-names">M.C.</div></div>, et al. </div><div class="article-title">Mutation in TERMINAL FLOWER1 reverses the photoperiodic requirement for flowering in the wild strawberry <em>Fragaria vesca</em></div>, <div class="source">Plant Physiol.</div> , <div class="year">2012</div>, vol. <div class="volume">159</div> (pg. <div class="fpage">1043</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutation%20in%20TERMINAL%20FLOWER1%20reverses%20the%20photoperiodic%20requirement%20for%20flowering%20in%20the%20wild%20strawberry%20Fragaria%20vesca&amp;author=E.A.%20Koskela&amp;author=K.%20Mouhu&amp;author=M.C.%20Albani&amp;publication_year=2012&amp;journal=Plant%20Physiol.&amp;volume=159&amp;pages=1043-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.112.196659" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.112.196659" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.112.196659"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22566495" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tennessen</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Govindarajulu</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Liston</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ashman</div> <div class="given-names">T.L.</div></div>. </div><div class="article-title">Targeted sequence capture provides insight into genome structure and genetics of male sterility in a gynodioecious diploid strawberry, <em>Fragaria vesca</em> ssp. bracteata (<em>Rosaceae</em>)</div>, <div class="source"><em>G3</em> (Bethesda)</div> , <div class="year">2013</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">1341</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20sequence%20capture%20provides%20insight%20into%20genome%20structure%20and%20genetics%20of%20male%20sterility%20in%20a%20gynodioecious%20diploid%20strawberry%2C%20Fragaria%20vesca%20ssp.%20bracteata%20(Rosaceae)&amp;author=J.A.%20Tennessen&amp;author=R.%20Govindarajulu&amp;author=A.%20Liston&amp;author=T.L.%20Ashman&amp;publication_year=2013&amp;journal=G3%20(Bethesda)&amp;volume=3&amp;pages=1341-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.113.006288" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.113.006288" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.113.006288"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23749450" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Q.Y.</div></div>,  <div class="name"><div class="surname">Gao</div> <div class="given-names">Z.H.</div></div>, et al. </div><div class="article-title">Genome-wide identification and comparative expression analysis of NBS-LRR-encoding genes upon <em>Colletotrichum gloeosporioides</em> infection in two ecotypes of <em>Fragaria vesca</em></div>, <div class="source">Gene.</div> , <div class="year">2013</div>, vol. <div class="volume">527</div> (pg. <div class="fpage">215</div>-<div class="lpage">27</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20identification%20and%20comparative%20expression%20analysis%20of%20NBS-LRR-encoding%20genes%20upon%20Colletotrichum%20gloeosporioides%20infection%20in%20two%20ecotypes%20of%20Fragaria%20vesca&amp;author=J.%20Li&amp;author=Q.Y.%20Zhang&amp;author=Z.H.%20Gao&amp;publication_year=2013&amp;journal=Gene.&amp;volume=527&amp;pages=215-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2013.06.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2013.06.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2013.06.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23806759" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schatz</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Witkowski</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">McCombie</div> <div class="given-names">W.R.</div></div>. </div><div class="article-title">Current challenges in <em>de novo</em> plant genome sequencing and assembly</div>, <div class="source">Genome Biol.</div> , <div class="year">2012</div>, vol. <div class="volume">13</div> pg. <div class="fpage">243</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Current%20challenges%20in%20de%20novo%20plant%20genome%20sequencing%20and%20assembly&amp;author=M.C.%20Schatz&amp;author=J.%20Witkowski&amp;author=W.R.%20McCombie&amp;publication_year=2012&amp;journal=Genome%20Biol.&amp;volume=13&amp;pages=243" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2012-13-4-243" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2012-13-4-243" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2012-13-4-243"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22546054" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hamilton</div> <div class="given-names">J.P.</div></div>,  <div class="name"><div class="surname">Buell</div> <div class="given-names">C.R.</div></div>. </div><div class="article-title">Advances in plant genome sequencing</div>, <div class="source">Plant J.</div> , <div class="year">2012</div>, vol. <div class="volume">70</div> (pg. <div class="fpage">177</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Advances%20in%20plant%20genome%20sequencing&amp;author=J.P.%20Hamilton&amp;author=C.R.%20Buell&amp;publication_year=2012&amp;journal=Plant%20J.&amp;volume=70&amp;pages=177-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-313X.2012.04894.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.2012.04894.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-313X.2012.04894.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22449051" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Velasco</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Zharkikh</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Affourtit</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">The genome of the domesticated apple (<em>Malus domestica</em> Borkh.)</div>, <div class="source">Nat. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">833</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20domesticated%20apple%20(Malus%20domestica%20Borkh.)&amp;author=R.%20Velasco&amp;author=A.%20Zharkikh&amp;author=J.%20Affourtit&amp;publication_year=2010&amp;journal=Nat.%20Genet.&amp;volume=42&amp;pages=833-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.654" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.654" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.654"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20802477" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Shi</div> <div class="given-names">Z.</div></div>, et al. </div><div class="article-title">The genome of the pear (<em>Pyrus bretschneideri</em> Rehd.)</div>, <div class="source">Genome Res.</div> , <div class="year">2012</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">396</div>-<div class="lpage">408</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20pear%20(Pyrus%20bretschneideri%20Rehd.)&amp;author=J.%20Wu&amp;author=Z.%20Wang&amp;author=Z.%20Shi&amp;publication_year=2012&amp;journal=Genome%20Res.&amp;volume=23&amp;pages=396-408" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.144311.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.144311.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.144311.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23149293" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="collab">International Peach Genome Initiative</div><div class="article-title">The high-quality draft genome of peach (<em>Prunus persica</em>) identifies unique patterns of genetic diversity, domestication and genome evolution</div>, <div class="source">Nat. Genet.</div> , <div class="year">2013</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">487</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2586" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2586" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2586"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23525075" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Sun</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">The genome of <em>Prunus mume</em></div>, <div class="source">Nat. Commun.</div> , <div class="year">2012</div>, vol. <div class="volume">3</div> pg. <div class="fpage">1318</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20Prunus%20mume&amp;author=Q.%20Zhang&amp;author=W.%20Chen&amp;author=L.%20Sun&amp;publication_year=2012&amp;journal=Nat.%20Commun.&amp;volume=3&amp;pages=1318" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms2290" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms2290" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms2290"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23271652" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="collab">The Potato Genome Sequencing Consortium</div><div class="article-title">Genome sequence and analysis of the tuber crop potato</div>, <div class="source">Nature</div> , <div class="year">2011</div>, vol. <div class="volume">475</div> (pg. <div class="fpage">189</div>-<div class="lpage">95</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10158" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10158" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10158"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21743474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">The draft genome of a diploid cotton <em>Gossypium raimondii</em></div>, <div class="source">Nat. Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">1098</div>-<div class="lpage">103</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20draft%20genome%20of%20a%20diploid%20cotton%20Gossypium%20raimondii&amp;author=K.%20Wang&amp;author=Z.%20Wang&amp;author=F.%20Li&amp;publication_year=2012&amp;journal=Nat.%20Genet.&amp;volume=44&amp;pages=1098-103" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2371" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2371" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2371"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22922876" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">D'Hont</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Denoeud</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Aury</div> <div class="given-names">J.M.</div></div>, et al. </div><div class="article-title">The banana (<em>Musa acuminata</em>) genome and the evolution of monocotyledonous plants</div>, <div class="source">Nature</div> , <div class="year">2012</div>, vol. <div class="volume">488</div> (pg. <div class="fpage">213</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20banana%20(Musa%20acuminata)%20genome%20and%20the%20evolution%20of%20monocotyledonous%20plants&amp;author=A.%20D'Hont&amp;author=F.%20Denoeud&amp;author=J.M.%20Aury&amp;publication_year=2012&amp;journal=Nature&amp;volume=488&amp;pages=213-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11241" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11241" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11241"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22801500" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Van Berloo</div> <div class="given-names">R.</div></div>. </div><div class="article-title">GGT 2.0: Versatile software for visualization and analysis of genetic data</div>, <div class="source">J. Heredity.</div> , <div class="year">2008</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">232</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GGT%202.0%3A%20Versatile%20software%20for%20visualization%20and%20analysis%20of%20genetic%20data&amp;author=R.%20Van%20Berloo&amp;publication_year=2008&amp;journal=J.%20Heredity.&amp;volume=99&amp;pages=232-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/esm109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2Fesm109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2Fesm109"> </span></div> </div></div></div></div></div><div content-id="DST049C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Shimodaira</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Pvclust: an R package for assessing the uncertainty in hierarchical clustering</div>, <div class="source">Bioinformatics.</div> , <div class="year">2006</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">1540</div>-<div class="lpage">2</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pvclust%3A%20an%20R%20package%20for%20assessing%20the%20uncertainty%20in%20hierarchical%20clustering&amp;author=R.%20Suzuki&amp;author=H.%20Shimodaira&amp;publication_year=2006&amp;journal=Bioinformatics.&amp;volume=22&amp;pages=1540-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btl117" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl117" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtl117"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16595560" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marçais</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Kingsford</div> <div class="given-names">C.</div></div>. </div><div class="article-title">A fast, lock-free approach for efficient parallel counting of occurrences of k-mers</div>, <div class="source">Bioinformatics</div> , <div class="year">2011</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">764</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20fast%2C%20lock-free%20approach%20for%20efficient%20parallel%20counting%20of%20occurrences%20of%20k-mers&amp;author=G.%20Mar%C3%A7ais&amp;author=C.%20Kingsford&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=764-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ruan</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">De novo assembly of human genomes with massively parallel short read sequencing</div>, <div class="source">Genome Res.</div> , <div class="year">2010</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">265</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20of%20human%20genomes%20with%20massively%20parallel%20short%20read%20sequencing&amp;author=R.%20Li&amp;author=H.%20Zhu&amp;author=J.%20Ruan&amp;publication_year=2010&amp;journal=Genome%20Res.&amp;volume=20&amp;pages=265-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.097261.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.097261.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.097261.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20019144" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Schäffer</div> <div class="given-names">A.A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Sch%C3%A4ffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wurm</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Riba-Grognuz</div> <div class="given-names">O.</div></div>, et al. </div><div class="article-title">The genome of the fire ant <em>Solenopsis invicta</em></div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2011</div>, vol. <div class="volume">108</div> (pg. <div class="fpage">5679</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20fire%20ant%20Solenopsis%20invicta&amp;author=Y.%20Wurm&amp;author=J.%20Wang&amp;author=O.%20Riba-Grognuz&amp;publication_year=2011&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=108&amp;pages=5679-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1009690108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1009690108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1009690108"> </span></div> </div></div></div></div></div><div content-id="DST049C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Price</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Jones</div> <div class="given-names">N.C.</div></div>,  <div class="name"><div class="surname">Pevzner</div> <div class="given-names">P.A.</div></div>. </div><div class="article-title">De novo identification of repeat families in large genomes</div>, <div class="year">2005</div><div class="conf-name">To appear in Proceedings of the 13 Annual International conference on Intelligent Systems for Molecular Biology (ISMB-05)</div><div class="conf-loc">Detroit, Michigan</div></div></div></div></div><div content-id="DST049C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jurka</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Repeats in genomic dna: mining and meaning</div>, <div class="source">Curr. Opin. Struct. Biol.</div> , <div class="year">1998</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">333</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repeats%20in%20genomic%20dna%3A%20mining%20and%20meaning&amp;author=J.%20Jurka&amp;publication_year=1998&amp;journal=Curr.%20Opin.%20Struct.%20Biol.&amp;volume=8&amp;pages=333-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0959-440X(98)80067-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0959-440X(98)80067-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0959-440X(98)80067-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9666329" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kofler</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Schlötterer</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Lelley</div> <div class="given-names">T.</div></div>. </div><div class="article-title">SciRoKo: a new tool for whole genome microsatellite search and investigation</div>, <div class="source">Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">1683</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SciRoKo%3A%20a%20new%20tool%20for%20whole%20genome%20microsatellite%20search%20and%20investigation&amp;author=R.%20Kofler&amp;author=C.%20Schl%C3%B6tterer&amp;author=T.%20Lelley&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=1683-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm157" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm157" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm157"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17463017" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lowe</div> <div class="given-names">T.M.</div></div>,  <div class="name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">955</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%3A%20a%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence&amp;author=T.M.%20Lowe&amp;author=S.R.%20Eddy&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=955-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.5.0955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.5.0955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.5.0955"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9023104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Waack</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Gene prediction with a hidden markov model and a new intron submodel</div>, <div class="source">Bioinformatics.</div> , <div class="year">2003</div>, vol. <div class="volume">19</div> <div class="issue">Suppl 2)</div>(pg. <div class="fpage">ii215</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20prediction%20with%20a%20hidden%20markov%20model%20and%20a%20new%20intron%20submodel&amp;author=M.%20Stanke&amp;author=S.%20Waack&amp;publication_year=2003&amp;journal=Bioinformatics.&amp;volume=19&amp;pages=ii215-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg1080" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg1080" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg1080"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14534192" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mulder</div> <div class="given-names">N.J.</div></div>,  <div class="name"><div class="surname">Apweiler</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Attwood</div> <div class="given-names">T.K.</div></div>, et al. </div><div class="article-title">New developments in the InterPro database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">D224</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20developments%20in%20the%20InterPro%20database&amp;author=N.J.%20Mulder&amp;author=R.%20Apweiler&amp;author=T.K.%20Attwood&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=D224-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl841" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl841" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl841"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17202162" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Quevillon</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Silventoinen</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Pillai</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">InterProScan: protein domains identifier</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">W116</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterProScan%3A%20protein%20domains%20identifier&amp;author=E.%20Quevillon&amp;author=V.%20Silventoinen&amp;author=S.%20Pillai&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=W116-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki442" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki442" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki442"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15980438" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">Accelerated profile HMM searches</div>, <div class="source">PLoS Comput. Biol.</div> , <div class="year">2011</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e1002195</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accelerated%20profile%20HMM%20searches&amp;author=S.R.%20Eddy&amp;publication_year=2011&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=7&amp;pages=e1002195" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1002195" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1002195" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1002195"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22039361" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Llorens</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Futami</div> <div class="given-names">R.</div></div>. </div><div class="collab">Covelli</div><div class="article-title">The Gypsy Database (GyDB) of mobile genetic elements: release 2.0</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2011</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">D70</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Gypsy%20Database%20(GyDB)%20of%20mobile%20genetic%20elements%3A%20release%202.0&amp;author=C.%20Llorens&amp;author=R.%20Futami&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=D70-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq1061" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq1061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq1061"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21036865" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kent</div> <div class="given-names">W.J.</div></div>. </div><div class="article-title">BLAT—the BLAST-like alignment tool</div>, <div class="source">Genome Res.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">656</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BLAT%E2%80%94the%20BLAST-like%20alignment%20tool&amp;author=W.J.%20Kent&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=656-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.229202. Article published online before March 2002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.229202. Article published online before March 2002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.229202. Article published online before March 2002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11932250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pearson</div> <div class="given-names">W.R.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">Improved tools for biological sequence comparison</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1988</div>, vol. <div class="volume">85</div> (pg. <div class="fpage">2444</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20tools%20for%20biological%20sequence%20comparison&amp;author=W.R.%20Pearson&amp;author=D.J.%20Lipman&amp;publication_year=1988&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=85&amp;pages=2444-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.85.8.2444" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.85.8.2444" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.85.8.2444"> </span></div> </div></div></div></div></div><div content-id="DST049C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Godzik</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences</div>, <div class="source">Bioinformatics</div> , <div class="year">2006</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">1658</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cd-hit%3A%20a%20fast%20program%20for%20clustering%20and%20comparing%20large%20sets%20of%20protein%20or%20nucleotide%20sequences&amp;author=W.%20Li&amp;author=A.%20Godzik&amp;publication_year=2006&amp;journal=Bioinformatics&amp;volume=22&amp;pages=1658-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btl158" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl158" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtl158"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16731699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gotea</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Veeramachaneni</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Makałowski</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Mastering seeds for genomic size nucleotide BLAST searches</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">6935</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mastering%20seeds%20for%20genomic%20size%20nucleotide%20BLAST%20searches&amp;author=V.%20Gotea&amp;author=V.%20Veeramachaneni&amp;author=W.%20Maka%C5%82owski&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=6935-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg886" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg886" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg886"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14627826" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miller</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">Koren</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sutton</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Assembly algorithms for next-generation sequencing data</div>, <div class="source">Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">315</div>-<div class="lpage">27</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assembly%20algorithms%20for%20next-generation%20sequencing%20data&amp;author=J.R.%20Miller&amp;author=S.%20Koren&amp;author=G.%20Sutton&amp;publication_year=2010&amp;journal=Genomics&amp;volume=95&amp;pages=315-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ygeno.2010.03.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygeno.2010.03.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ygeno.2010.03.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20211242" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wajid</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Serpedin</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Review of general algorithmic features for genome assemblers for next generation sequencers</div>, <div class="source">Genomics Proteomics Bioinformatics</div> , <div class="year">2012</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">58</div>-<div class="lpage">73</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Review%20of%20general%20algorithmic%20features%20for%20genome%20assemblers%20for%20next%20generation%20sequencers&amp;author=B.%20Wajid&amp;author=E.%20Serpedin&amp;publication_year=2012&amp;journal=Genomics%20Proteomics%20Bioinformatics&amp;volume=10&amp;pages=58-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gpb.2012.05.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gpb.2012.05.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gpb.2012.05.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22768980" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST049C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST049C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Davis</div> <div class="given-names">T.M.</div></div>. </div><div class="article-title">Conservation and loss of ribosomal RNA gene sites in diploid and polyploid <em>Fragaria</em> (Rosaceae)</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2011</div>, vol. <div class="volume">11</div> pg. <div class="fpage">157</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conservation%20and%20loss%20of%20ribosomal%20RNA%20gene%20sites%20in%20diploid%20and%20polyploid%20Fragaria%20(Rosaceae)&amp;author=B.%20Liu&amp;author=T.M.%20Davis&amp;publication_year=2011&amp;journal=BMC%20Plant%20Biol.&amp;volume=11&amp;pages=157" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-11-157" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-11-157" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-11-157"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22074487" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Katsumi Isono</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=404005&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst049/2/dst049-dst049supp_tables.xls?Expires=2147483647&amp;Signature=FSdi-KtLqBhGdyRV6V-Ue-QCFMPWfSIZ54HpC-R4y0rFbJftA7LPEWADa9kOXY8TvWzIwQz1LkVaghJDqsbc2w9yEieS~mipDDG~sbM4W69gM1fuDQKTroEuUl4pIASf39Bpbw1mgeJHLPzMmYa6tunCf7~huQ6bnPcc8zsZ9f4F3fCxGSc0zL5ZQKBSu-s2AOOLhB3Uqpp72KUDKmImEG-Zdy37wgGvWkbQO1h40lYGLazj4gs~yhtSGOKMzmPKJbL2WCZb44lFSnZ5NKK2YrgU8D5geWCGbdGTtdB7IxcAWJHfeMsipB7weY47Y-u0Vq0HuCUxhTtMGVm6YEd7nQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst049/2/dst049-dst049supp_fig7.ppt?Expires=2147483647&amp;Signature=xwvuD9qEKgJG2zjf8~bRnVOx8alNrFsvIUnP4T~QkHA0gf7HZ3HOTVmnhMAWDAExLI5sanZ9ZS0DwuD59N0LKBq60U25fAidH98FBjuRF~vhfIyloRiggb7kolL~W3-Xwt2vb3Dj0FMCTTytC0ZPmcYPgXCEuoqZ5f1qTw7gBR1nTnIA5HwEZDdUT3ZkViuapSUrjuEYE7BB-wpA-IQ-SFUB4zZ-8dX6Wcihm0qDaVKr12WlNUD6dLVyhIthI0MTsMlaM0PwQbPgp54cr7EUB8jvbZEZpRWCdRBvEXLSNn~qXtSpTWoVmKymTIb~wUX8j1f6OcWMYWwqnQqF2F78ow__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst049/2/dst049-dst049supp_fig6.ppt?Expires=2147483647&amp;Signature=K6vkq2pfKfYhHb824wPwn9uMeOX2S7bEt4SJj-9Fyh7mN2IHYOWvr8KyBkZ5c5s6LyVuVUbTQpT0eDZXfuf7ElrrWIpvuRZT5BCixVz8ye2S31W7AecOdb0MwWN18aDSrgZ4UTYFE~D2PNAdHYRTbRF3yd4Rxgah7OPmDLwcHXXoHoKAG-nPzW2Hvf~8ZoZJpolXCJcSSYQ7axjG7qfzqlG4D80j0Dkt3SfPqaB1pKk5C-52hhQNRgAONWHZHsPGi845wr1yga4bLQDJDWNVmX1bqK3WRsr-PwYDSVZWTx9yGmhHx5zzNFVKjez1IoM8cHkxdCGSqwwpIwPKK9901A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst049/2/dst049-dst049supp_fig5.pdf?Expires=2147483647&amp;Signature=x8BMG343IiKz-qGDmJ7JFWczFHKTR5Y5Gy5Hqg24ZRreFCp-UxdGCGm57L5Wj6QG37D6G3LGy~yBf6d4-SWhnB~qBQ8qqjsiKQBoIw3WS1Raw88Nf3swVdjHzy-12SOtSKAJ7ByqsEi397pbZyTkP6lngAALmCRFJ3eTvfq6v9cjZXzZul28ARiy71uFGMfG5Mxzyu4gUAos-8ejmCpd0ONVicL~lLuMwQBRwBuoME-~7z3zTa-n3D24x71SLMoP-T~JfaOFYc7HHCLGJjuMTLagG7E28nFjbHaAmWNZUW9HkZYm~oTuLtVfs1ZWx-LrhbnXaAaVlUVL11MQZwfT0Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst049/2/dst049-dst049supp_fig4.pdf?Expires=2147483647&amp;Signature=LD0WPnMAxDIajwwN-2es7MCh3OI~mg~C~iGjCTc-ndE66w~ObpiMturIMbRu7ugj7JqX0O1W58iM-d7COzjZM4U1pN9RimNWRrLdbBgbKwIxZcz6IOR0oE0OhBi72D~wROtO6MzzqQMpoNqAQv~oiRUhwEs5I6qRcaIani2cWmsVwkO1oFR1Gx4yAn-MQjLJKc2C~kVTdyhMSMqFkeErsK030rCuMVAVUp-5rIL8kXN86tLuWbPwMwkA~p15ilXTpMTiYi3ptazG7h2E7dw5OqroEdthFxitlpcHQ3V0rDA0rlvYkW5f6pBC8cSF-mAfTX5ozfygV4CZo4~IMcA2fQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst049/2/dst049-dst049supp_fig3.ppt?Expires=2147483647&amp;Signature=2kf4BHo0KH-ulruEYFtLSBxXk3TFJbAsJLbvxQl5lZmLbnkq2vZe~ZVWt9s1nU~68vTn16LaosSH1Yg3bOTH3Y8uVySExYLSVm30jbuqeuH2DR-7MPpCb4KZ4JNy3Hw2qK~2thHegqkXn04g2bmEj4NkVRBLEM6iVaczXfrOefi0ReNWH~9bmLY1XPJ-YYmIFlC~86-UP6nNfTJzjkhRQNmRuKyGdHbtW9MKTUlaszpNmsVMrekM2kpseIm7UVWn0OeorTPgW223uarKWBIAO7~PXynjvX6DWu3U8mPG9arfbMKjJu17nvVQUmQBvHGKMztfqM6ESr9JjY01gq~2iA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst049/2/dst049-dst049supp_fig2.ppt?Expires=2147483647&amp;Signature=R0l9dVCLFuAeJJw2Ns9Gkg0cUiDS0qw61hQof4pI83cPJoJYMkUTL6UAaDvqIxGoiJ43etyB5PXjsQK05m1dIfWwR-iIOwEw969Rgeal4A1NgOd77JG7sDhQhR4RMx8SCUhtzp-O7C9rfbx~BNZIzj0g8Lha3wnMa40qZJsfTsx0UVeuZxcIxamVUj5sswY2Hr6EvhJ5mK96ovOvaSbT8PIWmIclyDyzp1HMhvnm13rvVSxVa51j2mR~QDVCQWCxASQIlCndhQqDOfBCONL7VmizUmTE3zyZCeMJMBCFE33gdDuaoGptoJBgecprG5IQmHO4zgboqRBBF8kqp-FhGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst049/2/dst049-dst049supp_fig1.ppt?Expires=2147483647&amp;Signature=pZLhnHFxpdcZ48o42BXpthYnG8f7iK0xZtQBZAAYtdQcZTWKxxGhOnwVScb2mpHGMaWH5yfIzWHBDpZXLJ3QVEr~o80UAV9Mqn8fjzVBmHwKLLSZoFZTv~FvDtgtwKxMxb9qY6sKfX-ARyWAESqImH~ddlrs9C64b9H-jabdPqPeovDdfUzLi9HrXL3HgNMfrl6eYXrQj4Q4ILgeDc~9xJ1X8qyX9qvBWae-Id6wkE-V7loeXVMVXv2dzO-m0d8r8abkBlRqQvZoFXHi9O6AM4HJrvrA8FYtXcSObWYYx6HzrGvYWw9qYzdOBfJLFRL~4hzO6d8RJOjbDc8Iv5Fuqw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - ppt file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">3,210</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897500006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">74</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst049"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="404005"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897500006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f2%2f169%2f404005">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000339897500006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (74)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F2%2F169%2F404005" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/404005'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Dissection of the Octoploid Strawberry Genome by Deep Sequencing of the Genomes of Fragaria Species" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst049");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f2%2f169%2f404005">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

