

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/13/3/111/347374","@type":"ScholarlyArticle","name":"Whole-Genome Microarray in  Arabidopsis  Facilitates Global Analysis of Retained Introns","datePublished":"2006-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/13/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2006-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsl003","keywords":["microarray","TILING","alternative splicing","<italic>Arabidopsis</italic>","intron retention","NPR-1","GIGANTEA"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/13/3/111/347374","author":[{"name":"Ner-Gaon, Hadas","affiliation":"Department of Plant Sciences, Weizmann Institute of Science  Rehovot 76100, Israel","@type":"Person"},{"name":"Fluhr, Robert","affiliation":"Department of Plant Sciences, Weizmann Institute of Science  Rehovot 76100, Israel","@type":"Person"}],"description":"Abstract.  Alternative splicing (AS) is an important post-transcriptional regulatory mechanism that can increase protein diversity and affect mRNA stability. Di","pageStart":"111","pageEnd":"121","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/3/0/m_dnaresearch13_3.cover.gif?Expires=2147483647&Signature=anx2J~8v0IVYzlePG5tUbncUtEx2PsYUKWb~Qpjo1jhMHlu0islS2eGWU9i~f0nzFtaqNwAgI5-no5gavB~5K2oqu5oELPuFU9PB0RbXx30rT5CtFH7Z7Tak8QE4~Qv5CaQ-S-laJYNwnkzL6WJ228M7M2qJZAZbrbUpUHT7RG37R-U5f~bJ7ij8Hr1oUW8kL1EexLVBqSK~E8j95q-vzg7TEqouIZZVhlOy-gxfExmOp8ieedz1ReyVM4Msqvq9iUEHn2KT3PqF7FeNdH-Lqt5xtcPFEJYTrL4VFii2sM~Knj8tJAAT5R0NldIHOqOze-~nk5Mk6oyjGclZXc9hCA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Whole-Genome Microarray in  Arabidopsis  Facilitates Global Analysis of Retained Introns","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/3/0/m_dnaresearch13_3.cover.gif?Expires=2147483647&Signature=anx2J~8v0IVYzlePG5tUbncUtEx2PsYUKWb~Qpjo1jhMHlu0islS2eGWU9i~f0nzFtaqNwAgI5-no5gavB~5K2oqu5oELPuFU9PB0RbXx30rT5CtFH7Z7Tak8QE4~Qv5CaQ-S-laJYNwnkzL6WJ228M7M2qJZAZbrbUpUHT7RG37R-U5f~bJ7ij8Hr1oUW8kL1EexLVBqSK~E8j95q-vzg7TEqouIZZVhlOy-gxfExmOp8ieedz1ReyVM4Msqvq9iUEHn2KT3PqF7FeNdH-Lqt5xtcPFEJYTrL4VFii2sM~Knj8tJAAT5R0NldIHOqOze-~nk5Mk6oyjGclZXc9hCA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Whole-Genome Microarray in  Arabidopsis  Facilitates Global Analysis of Retained Introns" />
    <meta property="og:description" content="Abstract.  Alternative splicing (AS) is an important post-transcriptional regulatory mechanism that can increase protein diversity and affect mRNA stability. Di" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/13/3/111/347374" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2006-01-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Ner-Gaon, Hadas" /><meta name="citation_author_institution" content="Department of Plant Sciences, Weizmann Institute of ScienceRehovot 76100, Israel" /><meta name="citation_author" content="Fluhr, Robert" /><meta name="citation_author_institution" content="Department of Plant Sciences, Weizmann Institute of ScienceRehovot 76100, Israel" /><meta name="citation_title" content="Whole-Genome Microarray in Arabidopsis Facilitates Global Analysis of Retained Introns" /><meta name="citation_firstpage" content="111" /><meta name="citation_lastpage" content="121" /><meta name="citation_doi" content="10.1093/dnares/dsl003" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="13" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2006/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Selecting for functional alternative splices in ESTs; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=1837-1845; " /><meta name="citation_reference" content="citation_title=The transcriptional landscape of the mammalian genome; Science;  citation_year=2005;  citation_volume=309;  citation_pages=1559-1563; " /><meta name="citation_reference" content="citation_title=Species-specific variation of alternative splicing and transcriptional initiation in six eukaryotes; Gene;  citation_year=2005;  citation_volume=364;  citation_pages=53-62; " /><meta name="citation_reference" content="citation_title=A genomic view of alternative splicing; Nat. Genet.;  citation_year=2002;  citation_volume=30;  citation_pages=13-19; " /><meta name="citation_reference" content="citation_title=Multiple intron retention occurs in tumor cell CD44 mRNA processing; Am. J. Pathol.;  citation_year=1998;  citation_volume=153;  citation_pages=1221-1228; " /><meta name="citation_reference" content="citation_title=Developmental regulation of a proinsulin messenger RNA generated by intron retention; EMBO Rep.;  citation_year=2005;  citation_volume=6;  citation_pages=1182-1187; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of alternative pre-mRNA splicing in Arabidopsis thaliana based on full-length cDNA sequences; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=5096-5103; " /><meta name="citation_reference" content="citation_title=Genomewide comparative analysis of alternative splicing in plants; Proc. Natl. Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=7175-7180; " /><meta name="citation_reference" content="citation_title=Intron retention is a major phenomenon in alternative splicing in Arabidopsis; Plant J.;  citation_year=2004;  citation_volume=39;  citation_pages=877-885; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSIBLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-3402; " /><meta name="citation_reference" content="citation_title=Epigenomic mapping in Arabidopsis using tiling microarrays; Chromosome Res.;  citation_year=2005;  citation_volume=13;  citation_pages=299-308; " /><meta name="citation_reference" content="citation_title=Large-scale transcriptional activity in chromosomes 21 and 22; Science;  citation_year=2002;  citation_volume=296;  citation_pages=916-919; " /><meta name="citation_reference" content="citation_title=Novel RNAs identified from an in-depth analysis of the transcriptome of human chromosomes 21 and 22; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=331-342; " /><meta name="citation_reference" content="citation_title=Empirical analysis of transcriptional activity in the Arabidopsis genome; Science;  citation_year=2003;  citation_volume=302;  citation_pages=842-846; " /><meta name="citation_reference" content="citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing; J. R. Stat. Soc. Ser.;  citation_year=1995;  citation_volume=B57;  citation_pages=289-300; " /><meta name="citation_reference" content="citation_title=Alternative splicing modulation by a LAMMER kinase impinges on developmental and transcriptome expression; Plant Cell;  citation_year=2003;  citation_volume=15;  citation_pages=926-938; " /><meta name="citation_reference" content="citation_title=Analysis of the genome sequence of the flowering plant Arabidopsis thaliana; citation_author=Arabidopsis Genome Initiative.; Nature;  citation_year=2000;  citation_volume=408;  citation_pages=796-815; " /><meta name="citation_reference" content="citation_title=Refined annotation of the Arabidopsis genome by complete expressed sequence tag mapping; Plant Physiol.;  citation_year=2003;  citation_volume=132;  citation_pages=469-484; " /><meta name="citation_reference" content="citation_title=Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=5654-5666; " /><meta name="citation_reference" content="citation_title=Analysis of evolution of exon–intron structure of eukaryotic genes; Brief Bioinform.;  citation_year=2005;  citation_volume=6;  citation_pages=118-134; " /><meta name="citation_reference" content="citation_title=The architecture of pre-mRNAs affects mechanisms of splice-site pairing; Proc. Natl Acad. Sci. USA;  citation_year=2005;  citation_volume=102;  citation_pages=16176-16181; " /><meta name="citation_reference" content="citation_title=Arabidopsis intron mutations and pre-mRNA splicing; Plant J.;  citation_year=1996;  citation_volume=10;  citation_pages=771-780; " /><meta name="citation_reference" content="citation_title=Splicing of precursors to mRNA in higher plants: mechanism, regulation and sub-nuclear organisation of the spliceosomal machinery; Plant Mol. Biol.;  citation_year=1996;  citation_volume=32;  citation_pages=1-41; " /><meta name="citation_reference" content="citation_title=A rule for termination-codon position within intron-containing genes: when nonsense affects RNA abundance; Trends Biochem. Sci.;  citation_year=1998;  citation_volume=23;  citation_pages=198-199; " /><meta name="citation_reference" content="citation_title=Nonsense-mediated mRNA decay: terminating erroneous gene expression; Curr. Opin. Cell Biol.;  citation_year=2004;  citation_volume=16;  citation_pages=293-299; " /><meta name="citation_reference" content="citation_title=UPF3 suppresses aberrant spliced mRNA in Arabidopsis; Plant J.;  citation_year=2005;  citation_volume=43;  citation_pages=530-540; " /><meta name="citation_reference" content="citation_title=Mechanistic links between nonsense-mediated mRNA decay and pre-mRNA splicing in mammalian cells; Curr. Opin. Cell Biol.;  citation_year=2005;  citation_volume=17;  citation_pages=309-315; " /><meta name="citation_reference" content="citation_title=The evolving roles of alternative splicing; Curr. Opin. Struct. Biol.;  citation_year=2004;  citation_volume=14;  citation_pages=273-282; " /><meta name="citation_reference" content="citation_title=Arabidopsis UPF1 RNA helicase for nonsense-mediated mRNA decay is involved in seed size control and is essential for growth; Plant Cell. Physiol.;  citation_year=2006;  citation_volume=47;  citation_pages=572-580; " /><meta name="citation_reference" content="citation_title=Plant serine/arginine-rich proteins and their role in pre-mRNA splicing; Trends Plant Sci.;  citation_year=2004;  citation_volume=9;  citation_pages=541-547; " /><meta name="citation_reference" content="citation_title=Survey of conserved alternative splicing events of mRNAs encoding SR proteins in land plants; citation_text=[Please update reference 31 if now it has been published.]; Mol Biol Evol.;  citation_year=2006; " /><meta name="citation_reference" content="citation_title=The Arabidopsis splicing factor SR1 is regulated by alternative splicing; Plant Mol. Biol.;  citation_year=2000;  citation_volume=42;  citation_pages=571-581; " /><meta name="citation_reference" content="citation_title=Genome-wide ORFeome cloning and analysis of Arabidopsis transcription factor genes; Plant Physiol.;  citation_year=2004;  citation_volume=135;  citation_pages=773-782; " /><meta name="citation_reference" content="citation_title=The splicing factor SRp20 modifies splicing of its own mRNA and ASF/SF2 antagonizes this regulation; EMBO J.;  citation_year=1997;  citation_volume=16;  citation_pages=5077-5085; " /><meta name="citation_reference" content="citation_title=SR proteins and splicing control; Genes Dev.;  citation_year=1996;  citation_volume=10;  citation_pages=1569-1579; " /><meta name="citation_reference" content="citation_title=The Arabidopsis NPR1 gene that controls systemic acquired resistance encodes a novel protein containing ankyrin repeats; Cell;  citation_year=1997;  citation_volume=88;  citation_pages=57-63; " /><meta name="citation_reference" content="citation_title=The Arabidopsis NIM1 protein shows homology to the mammalian transcription factor inhibitor I kappa B; Plant Cell;  citation_year=1997;  citation_volume=9;  citation_pages=425-439; " /><meta name="citation_reference" content="citation_title=Monogenic recessive mutations causing both late floral initiation and excess starch accumulation in Arabidopsis; Plant Cell;  citation_year=1995;  citation_volume=7;  citation_pages=1703-1712; " /><meta name="citation_reference" content="citation_title=Evidence for the widespread coupling of alternative splicing and nonsense-mediated mRNA decay in humans; Proc. Natl Acad. Sci. USA;  citation_year=2003;  citation_volume=100;  citation_pages=189-192; " /><meta name="citation_reference" content="citation_title=Functional annotation of the Arabidopsis genome using controlled vocabularies; Plant Physiol.;  citation_year=2004;  citation_volume=135;  citation_pages=745-755; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/13/3/111/17462396/dsl003.pdf" /><meta name="description" content="Abstract.  Alternative splicing (AS) is an important post-transcriptional regulatory mechanism that can increase protein diversity and affect mRNA stability. Di" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/13/3/111/347374" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/13/3/111/347374" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/13/3/111/347374");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsl003';
    </script>
<title>
	Whole-Genome Microarray in Arabidopsis Facilitates Global Analysis of Retained Introns | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./347374" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbJSPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOZXItR2FvbiwgSGFkYXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFBsYW50IFNjaWVuY2VzLCBXZWl6bWFubiBJbnN0aXR1dGUgb2YgU2NpZW5jZVJlaG92b3QgNzYxMDAsIElzcmFlbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZsdWhyLCBSb2JlcnQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFBsYW50IFNjaWVuY2VzLCBXZWl6bWFubiBJbnN0aXR1dGUgb2YgU2NpZW5jZVJlaG92b3QgNzYxMDAsIElzcmFlbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iV2hvbGUtR2Vub21lIE1pY3JvYXJyYXkgaW4gQXJhYmlkb3BzaXMgRmFjaWxpdGF0ZXMgR2xvYmFsIEFuYWx5c2lzIG9mIFJldGFpbmVkIEludHJvbnMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxMTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjEyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzbDAwMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDA2LzAxLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VsZWN0aW5nIGZvciBmdW5jdGlvbmFsIGFsdGVybmF0aXZlIHNwbGljZXMgaW4gRVNUczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xODM3LTE4NDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB0cmFuc2NyaXB0aW9uYWwgbGFuZHNjYXBlIG9mIHRoZSBtYW1tYWxpYW4gZ2Vub21lOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMwOTsgIGNpdGF0aW9uX3BhZ2VzPTE1NTktMTU2MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3BlY2llcy1zcGVjaWZpYyB2YXJpYXRpb24gb2YgYWx0ZXJuYXRpdmUgc3BsaWNpbmcgYW5kIHRyYW5zY3JpcHRpb25hbCBpbml0aWF0aW9uIGluIHNpeCBldWthcnlvdGVzOyBHZW5lOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTM2NDsgIGNpdGF0aW9uX3BhZ2VzPTUzLTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbm9taWMgdmlldyBvZiBhbHRlcm5hdGl2ZSBzcGxpY2luZzsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0xMy0xOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXVsdGlwbGUgaW50cm9uIHJldGVudGlvbiBvY2N1cnMgaW4gdHVtb3IgY2VsbCBDRDQ0IG1STkEgcHJvY2Vzc2luZzsgQW0uIEouIFBhdGhvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTUzOyAgY2l0YXRpb25fcGFnZXM9MTIyMS0xMjI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudGFsIHJlZ3VsYXRpb24gb2YgYSBwcm9pbnN1bGluIG1lc3NlbmdlciBSTkEgZ2VuZXJhdGVkIGJ5IGludHJvbiByZXRlbnRpb247IEVNQk8gUmVwLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MTE4Mi0xMTg3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyBvZiBhbHRlcm5hdGl2ZSBwcmUtbVJOQSBzcGxpY2luZyBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYSBiYXNlZCBvbiBmdWxsLWxlbmd0aCBjRE5BIHNlcXVlbmNlczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9NTA5Ni01MTAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWV3aWRlIGNvbXBhcmF0aXZlIGFuYWx5c2lzIG9mIGFsdGVybmF0aXZlIHNwbGljaW5nIGluIHBsYW50czsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTAzOyAgY2l0YXRpb25fcGFnZXM9NzE3NS03MTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRyb24gcmV0ZW50aW9uIGlzIGEgbWFqb3IgcGhlbm9tZW5vbiBpbiBhbHRlcm5hdGl2ZSBzcGxpY2luZyBpbiBBcmFiaWRvcHNpczsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz04NzctODg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HYXBwZWQgQkxBU1QgYW5kIFBTSUJMQVNUOiBhIG5ldyBnZW5lcmF0aW9uIG9mIHByb3RlaW4gZGF0YWJhc2Ugc2VhcmNoIHByb2dyYW1zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zMzg5LTM0MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVwaWdlbm9taWMgbWFwcGluZyBpbiBBcmFiaWRvcHNpcyB1c2luZyB0aWxpbmcgbWljcm9hcnJheXM7IENocm9tb3NvbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTI5OS0zMDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxhcmdlLXNjYWxlIHRyYW5zY3JpcHRpb25hbCBhY3Rpdml0eSBpbiBjaHJvbW9zb21lcyAyMSBhbmQgMjI7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9Mjk2OyAgY2l0YXRpb25fcGFnZXM9OTE2LTkxOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm92ZWwgUk5BcyBpZGVudGlmaWVkIGZyb20gYW4gaW4tZGVwdGggYW5hbHlzaXMgb2YgdGhlIHRyYW5zY3JpcHRvbWUgb2YgaHVtYW4gY2hyb21vc29tZXMgMjEgYW5kIDIyOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTMzMS0zNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVtcGlyaWNhbCBhbmFseXNpcyBvZiB0cmFuc2NyaXB0aW9uYWwgYWN0aXZpdHkgaW4gdGhlIEFyYWJpZG9wc2lzIGdlbm9tZTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDI7ICBjaXRhdGlvbl9wYWdlcz04NDItODQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db250cm9sbGluZyB0aGUgZmFsc2UgZGlzY292ZXJ5IHJhdGU6IGEgcHJhY3RpY2FsIGFuZCBwb3dlcmZ1bCBhcHByb2FjaCB0byBtdWx0aXBsZSB0ZXN0aW5nOyBKLiBSLiBTdGF0LiBTb2MuIFNlci47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9QjU3OyAgY2l0YXRpb25fcGFnZXM9Mjg5LTMwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWx0ZXJuYXRpdmUgc3BsaWNpbmcgbW9kdWxhdGlvbiBieSBhIExBTU1FUiBraW5hc2UgaW1waW5nZXMgb24gZGV2ZWxvcG1lbnRhbCBhbmQgdHJhbnNjcmlwdG9tZSBleHByZXNzaW9uOyBQbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9OTI2LTkzODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgdGhlIGdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgZmxvd2VyaW5nIHBsYW50IEFyYWJpZG9wc2lzIHRoYWxpYW5hOyBjaXRhdGlvbl9hdXRob3I9QXJhYmlkb3BzaXMgR2Vub21lIEluaXRpYXRpdmUuOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NDA4OyAgY2l0YXRpb25fcGFnZXM9Nzk2LTgxNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVmaW5lZCBhbm5vdGF0aW9uIG9mIHRoZSBBcmFiaWRvcHNpcyBnZW5vbWUgYnkgY29tcGxldGUgZXhwcmVzc2VkIHNlcXVlbmNlIHRhZyBtYXBwaW5nOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzI7ICBjaXRhdGlvbl9wYWdlcz00NjktNDg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbXByb3ZpbmcgdGhlIEFyYWJpZG9wc2lzIGdlbm9tZSBhbm5vdGF0aW9uIHVzaW5nIG1heGltYWwgdHJhbnNjcmlwdCBhbGlnbm1lbnQgYXNzZW1ibGllczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9NTY1NC01NjY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBldm9sdXRpb24gb2YgZXhvbuKAk2ludHJvbiBzdHJ1Y3R1cmUgb2YgZXVrYXJ5b3RpYyBnZW5lczsgQnJpZWYgQmlvaW5mb3JtLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MTE4LTEzNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGFyY2hpdGVjdHVyZSBvZiBwcmUtbVJOQXMgYWZmZWN0cyBtZWNoYW5pc21zIG9mIHNwbGljZS1zaXRlIHBhaXJpbmc7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTAyOyAgY2l0YXRpb25fcGFnZXM9MTYxNzYtMTYxODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFyYWJpZG9wc2lzIGludHJvbiBtdXRhdGlvbnMgYW5kIHByZS1tUk5BIHNwbGljaW5nOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTc3MS03ODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwbGljaW5nIG9mIHByZWN1cnNvcnMgdG8gbVJOQSBpbiBoaWdoZXIgcGxhbnRzOiBtZWNoYW5pc20sIHJlZ3VsYXRpb24gYW5kIHN1Yi1udWNsZWFyIG9yZ2FuaXNhdGlvbiBvZiB0aGUgc3BsaWNlb3NvbWFsIG1hY2hpbmVyeTsgUGxhbnQgTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTEtNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcnVsZSBmb3IgdGVybWluYXRpb24tY29kb24gcG9zaXRpb24gd2l0aGluIGludHJvbi1jb250YWluaW5nIGdlbmVzOiB3aGVuIG5vbnNlbnNlIGFmZmVjdHMgUk5BIGFidW5kYW5jZTsgVHJlbmRzIEJpb2NoZW0uIFNjaS47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xOTgtMTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob25zZW5zZS1tZWRpYXRlZCBtUk5BIGRlY2F5OiB0ZXJtaW5hdGluZyBlcnJvbmVvdXMgZ2VuZSBleHByZXNzaW9uOyBDdXJyLiBPcGluLiBDZWxsIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MjkzLTI5OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VVBGMyBzdXBwcmVzc2VzIGFiZXJyYW50IHNwbGljZWQgbVJOQSBpbiBBcmFiaWRvcHNpczsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz01MzAtNTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZWNoYW5pc3RpYyBsaW5rcyBiZXR3ZWVuIG5vbnNlbnNlLW1lZGlhdGVkIG1STkEgZGVjYXkgYW5kIHByZS1tUk5BIHNwbGljaW5nIGluIG1hbW1hbGlhbiBjZWxsczsgQ3Vyci4gT3Bpbi4gQ2VsbCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTMwOS0zMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBldm9sdmluZyByb2xlcyBvZiBhbHRlcm5hdGl2ZSBzcGxpY2luZzsgQ3Vyci4gT3Bpbi4gU3RydWN0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTI3My0yODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFyYWJpZG9wc2lzIFVQRjEgUk5BIGhlbGljYXNlIGZvciBub25zZW5zZS1tZWRpYXRlZCBtUk5BIGRlY2F5IGlzIGludm9sdmVkIGluIHNlZWQgc2l6ZSBjb250cm9sIGFuZCBpcyBlc3NlbnRpYWwgZm9yIGdyb3d0aDsgUGxhbnQgQ2VsbC4gUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NDc7ICBjaXRhdGlvbl9wYWdlcz01NzItNTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFudCBzZXJpbmUvYXJnaW5pbmUtcmljaCBwcm90ZWlucyBhbmQgdGhlaXIgcm9sZSBpbiBwcmUtbVJOQSBzcGxpY2luZzsgVHJlbmRzIFBsYW50IFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTU0MS01NDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN1cnZleSBvZiBjb25zZXJ2ZWQgYWx0ZXJuYXRpdmUgc3BsaWNpbmcgZXZlbnRzIG9mIG1STkFzIGVuY29kaW5nIFNSIHByb3RlaW5zIGluIGxhbmQgcGxhbnRzOyBjaXRhdGlvbl90ZXh0PVtQbGVhc2UgdXBkYXRlIHJlZmVyZW5jZSAzMSBpZiBub3cgaXQgaGFzIGJlZW4gcHVibGlzaGVkLl07IE1vbCBCaW9sIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQXJhYmlkb3BzaXMgc3BsaWNpbmcgZmFjdG9yIFNSMSBpcyByZWd1bGF0ZWQgYnkgYWx0ZXJuYXRpdmUgc3BsaWNpbmc7IFBsYW50IE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz01NzEtNTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBPUkZlb21lIGNsb25pbmcgYW5kIGFuYWx5c2lzIG9mIEFyYWJpZG9wc2lzIHRyYW5zY3JpcHRpb24gZmFjdG9yIGdlbmVzOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzU7ICBjaXRhdGlvbl9wYWdlcz03NzMtNzgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc3BsaWNpbmcgZmFjdG9yIFNScDIwIG1vZGlmaWVzIHNwbGljaW5nIG9mIGl0cyBvd24gbVJOQSBhbmQgQVNGL1NGMiBhbnRhZ29uaXplcyB0aGlzIHJlZ3VsYXRpb247IEVNQk8gSi47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz01MDc3LTUwODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNSIHByb3RlaW5zIGFuZCBzcGxpY2luZyBjb250cm9sOyBHZW5lcyBEZXYuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MTU2OS0xNTc5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQXJhYmlkb3BzaXMgTlBSMSBnZW5lIHRoYXQgY29udHJvbHMgc3lzdGVtaWMgYWNxdWlyZWQgcmVzaXN0YW5jZSBlbmNvZGVzIGEgbm92ZWwgcHJvdGVpbiBjb250YWluaW5nIGFua3lyaW4gcmVwZWF0czsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT04ODsgIGNpdGF0aW9uX3BhZ2VzPTU3LTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQXJhYmlkb3BzaXMgTklNMSBwcm90ZWluIHNob3dzIGhvbW9sb2d5IHRvIHRoZSBtYW1tYWxpYW4gdHJhbnNjcmlwdGlvbiBmYWN0b3IgaW5oaWJpdG9yIEkga2FwcGEgQjsgUGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9NDI1LTQzOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9ub2dlbmljIHJlY2Vzc2l2ZSBtdXRhdGlvbnMgY2F1c2luZyBib3RoIGxhdGUgZmxvcmFsIGluaXRpYXRpb24gYW5kIGV4Y2VzcyBzdGFyY2ggYWNjdW11bGF0aW9uIGluIEFyYWJpZG9wc2lzOyBQbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0xNzAzLTE3MTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2aWRlbmNlIGZvciB0aGUgd2lkZXNwcmVhZCBjb3VwbGluZyBvZiBhbHRlcm5hdGl2ZSBzcGxpY2luZyBhbmQgbm9uc2Vuc2UtbWVkaWF0ZWQgbVJOQSBkZWNheSBpbiBodW1hbnM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTAwOyAgY2l0YXRpb25fcGFnZXM9MTg5LTE5MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuY3Rpb25hbCBhbm5vdGF0aW9uIG9mIHRoZSBBcmFiaWRvcHNpcyBnZW5vbWUgdXNpbmcgY29udHJvbGxlZCB2b2NhYnVsYXJpZXM7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEzNTsgIGNpdGF0aW9uX3BhZ2VzPTc0NS03NTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xMy8zLzExMS8xNzQ2MjM5Ni9kc2wwMDMucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEFsdGVybmF0aXZlIHNwbGljaW5nIChBUykgaXMgYW4gaW1wb3J0YW50IHBvc3QtdHJhbnNjcmlwdGlvbmFsIHJlZ3VsYXRvcnkgbWVjaGFuaXNtIHRoYXQgY2FuIGluY3JlYXNlIHByb3RlaW4gZGl2ZXJzaXR5IGFuZCBhZmZlY3QgbVJOQSBzdGFiaWxpdHkuIERpIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNDczNzQmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkfO8cwfCCHft0MioclrZaXNKHoBc=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f3%2f111%2f347374" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f3%2f111%2f347374" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13406" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/13/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/3/0/m_dnaresearch13_3.cover.gif?Expires=2147483647&amp;Signature=anx2J~8v0IVYzlePG5tUbncUtEx2PsYUKWb~Qpjo1jhMHlu0islS2eGWU9i~f0nzFtaqNwAgI5-no5gavB~5K2oqu5oELPuFU9PB0RbXx30rT5CtFH7Z7Tak8QE4~Qv5CaQ-S-laJYNwnkzL6WJ228M7M2qJZAZbrbUpUHT7RG37R-U5f~bJ7ij8Hr1oUW8kL1EexLVBqSK~E8j95q-vzg7TEqouIZZVhlOy-gxfExmOp8ieedz1ReyVM4Msqvq9iUEHn2KT3PqF7FeNdH-Lqt5xtcPFEJYTrL4VFii2sM~Knj8tJAAT5R0NldIHOqOze-~nk5Mk6oyjGclZXc9hCA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/13/3">
                        <div class="volume trailing-comma">Volume 13</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
2006        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87193714">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87193714">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87193716">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87193716">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87193721">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87193721">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87193733">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87193733">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87193763">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87193763">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87193774">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87193774">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/13/3/103/347233">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/13/3/123/347581">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Whole-Genome Microarray in <em>Arabidopsis</em> Facilitates Global Analysis of Retained Introns
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hadas Ner-Gaon</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Department of Plant Sciences, Weizmann Institute of Science</div><div class="addr-line">Rehovot 76100, Israel</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hadas+Ner-Gaon">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ner-Gaon H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ner-Gaon H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hadas Ner-Gaon</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Robert Fluhr</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Department of Plant Sciences, Weizmann Institute of Science</div><div class="addr-line">Rehovot 76100, Israel</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <sup>*</sup>To whom correspondence should be addressed. Tel. +972-8-9342175. Fax. +972-8-9344181, E-mail: <a href="mailto:robert.fluhr@weizmann.ac.il" target="_blank">robert.fluhr@weizmann.ac.il</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Robert+Fluhr">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fluhr R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fluhr R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Robert Fluhr</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Communicated by Kazuo Shinozaki</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 13, Issue 3, 1 January 2006, Pages 111–121, <a href='https://doi.org/10.1093/dnares/dsl003'>https://doi.org/10.1093/dnares/dsl003</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 January 2006</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">02 May 2006</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">19 July 2006</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="347374" href="/dnaresearch/article-pdf/13/3/111/17462396/dsl003.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Hadas Ner-Gaon, Robert Fluhr;  Whole-Genome Microarray in <em>Arabidopsis</em> Facilitates Global Analysis of Retained Introns, <em>DNA Research</em>, Volume 13, Issue 3, 1 January 2006, Pages 111–121, <a href='https://doi.org/10.1093/dnares/dsl003'>https://doi.org/10.1093/dnares/dsl003</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/347374?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/347374?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/347374?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/347374?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/347374?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/347374?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/347374?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Whole-Genome%20Microarray%20in%20%26lt%3Bem%26gt%3BArabidopsis%26lt%3B%2Fem%26gt%3B%20Facilitates%20Global%20Analysis%20of%20Retained%20Introns&amp;publicationDate=2006-01-01&amp;volumeNum=13&amp;issueNum=3&amp;author=Ner-Gaon%2C%20Hadas%3B%20Fluhr%2C%20Robert&amp;startPage=111&amp;endPage=121&amp;contentId=10.1093%2Fdnares%2Fdsl003&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f3%2f111%2f347374" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87193714 id="87193714" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Alternative splicing (AS) is an important post-transcriptional regulatory mechanism that can increase protein diversity and affect mRNA stability. Different types of AS have been observed; these include exon skipping, alternative donor or acceptor site and intron retention. In humans, exon skipping is the most common type while intron retention is rare. In contrast, in <em>Arabidopsis</em>, intron retention is the most prevalent AS type (∼40%). Here we show that direct transcript expression analysis using high-density oligonucleotide-based whole-genome microarrays (WGAs) is particularly amenable for assessing global intron retention in <em>Arabidopsis</em>. By applying a novel algorithm retained introns are detected in 8% of the transcripts examined. A sampling of 14 transcripts showed that 86% can be confirmed by RT–PCR. This rate of detection predicts an overall total AS rate of 20% for <em>Arabidopsis</em> compared with 10–22% based on EST/cDNA-based analysis. These findings will facilitate monitoring constitutive and dynamic whole-genome splicing on the next generation WGA slides.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="microarray">microarray</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="TILING">TILING</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;alternative splicing&quot;">alternative splicing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Arabidopsis"><em>Arabidopsis</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;intron retention&quot;">intron retention</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="NPR-1">NPR-1</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="GIGANTEA">GIGANTEA</a></div>                            <h2 scrollto-destination=87193716 id="87193716" class="section-title" >1. Introduction</h2>
<p>Alternative RNA processing pathways are a result of combining different splice junctions that are present in pre-mRNA transcripts. In this way, a variety of mRNA and proteins can be created from the same gene. Alternative splicing (AS) is thought to play a major role in expanding the potential informational content of eukaryotic genomes. Recent evidence indicates a high incidence (32–60%) of AS in the human genome, predominantly in the form of exon skip while a minor form was intron retention (5–16%).<a reveal-id="b1  b4" data-open="b1  b4" class="link link-ref link-reveal xref-bibr">1–4</a> Although rare, intron retention can play an important biological role. It has been linked to tumor growth<a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> and is part of developmental regulation of a proinsulin expression.<a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a></p><p>Plants are thought to exhibit less AS (10–22%).<a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="b7" data-open="b7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> Computational analysis of AS in <em>Arabidopsis</em> by EST-pair alignment identified 436 alternatively spliced genes.<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a> Unexpectedly, intron retention was found to be the most common type (45%). Retained introns were shown to be present in RNA derived from polyribosomes, demonstrating that these intron retention events are not the byproduct of incomplete splicing but are found in a translatable context in the cytoplasm.<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a> Iida et al.<a reveal-id="b7" data-open="b7" class="link link-ref link-reveal xref-bibr">7</a> aligned 248 514 RAFL (RIKEN <em>Arabidopsis</em> Full-Length) cDNA/EST sequences to the <em>Arabidopsis</em> genome using a BLAST-based method.<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a> They identified 15 214 transcription units (TUs) containing at least two sequences each and observed AS for 11.6% of these TUs of which 44% were retained introns.<a reveal-id="b7" data-open="b7" class="link link-ref link-reveal xref-bibr">7</a> In a recent study using a large collection of EST/cDNA, 4707 (21.8%) of the transcripts showed 8264 AS events. Approximately 56% of these events were of the intron retention type.<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> These studies confirmed that a low percentage of genes (10–22%) are alternatively spliced and that intron retention is the most prevalent AS type in <em>Arabidopsis</em>.</p><p>Calculating the rate of AS using expressed sequence data (ESTs and cDNAs) can underestimate the amount of AS for the following reasons. Collectively, the EST and cDNA database in plants is relatively small and, thus, may lack low-abundance transcripts. In addition, AS predictions based on cDNA data are in many cases biased towards the termini of transcripts owing to the preponderance of end-sequence reads among ESTs and oligo(dT)-based priming for reverse transcription. Recently, high-density oligonucleotide-based whole-genome microarrays (WGAs) have emerged as a novel platform for genomic analysis beyond gene expression profiling. Unbiased WGAs can be used for ‘epigenomic’ mapping<a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> and have been used to monitor exon skipping in humans.<a reveal-id="b12" data-open="b12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> By comparing the expression levels of all the exons within genes, it was shown that a large fraction (∼80%) of expressed genes on chromosomes 21 and 22 exhibit exon skipping. This estimate is significantly higher than previous predictions, which was based on computational analysis of ESTs and cDNAs.<a reveal-id="b4" data-open="b4" class="link link-ref link-reveal xref-bibr">4</a> The array-based approach to analyzing AS also has limitations. For example, the detection of relatively rare splicing events is difficult. Furthermore, in a manner analogous to obtaining multiple EST libraries from different tissue, a large number of distinct tissues or cell populations must be surveyed to obtain sufficient confidence to call a splice variant at a specific splice junction.</p><p>In this work by using published WGA information, we monitor intron retention in <em>Arabidopsis</em> and show that it is readily detectable, generally leading to transcripts containing new reading frames that would result in truncated gene products.</p>                            <h2 scrollto-destination=87193721 id="87193721" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=87193722 id="87193722" class="section-title" >2.1. Data sources</h3>
<p><em>Arabidopsis</em> genes, introns and coding sequences annotations and their genomic location were obtained from TAIR 6.0 release databases <a class="link link-uri" href="ftp://ftp.arabidopsis.org/home/tair/Sequences/blast_datasets/TAIR6_seq_20051108" target="_blank"></a><a class="link link-uri" href="ftp://ftp.arabidopsis.org/home/tair/Sequences/blast_datasets/TAIR6_intron_20060221" target="_blank"></a><a class="link link-uri" href="ftp://ftp.arabidopsis.org/home/tair/Sequences/blast_datasets/TAIR6_cds_20051108" target="_blank"></a><a class="link link-uri" href="ftp://ftp.arabidopsis.org/home/tair/home/tair/Sequences/blast_datasets/TAIR6_3_UTR_20060126" target="_blank"></a><a class="link link-uri" href="ftp://ftp.arabidopsis.org/home/tair/home/tair/Sequences/blast_datasets/TAIR6_5_UTR_20060126" target="_blank"></a><a class="link link-uri" href="ftp://ftp.arabidopsis.org/home/tair/Maps/seqviewer_data/sv_gene_feature.data" target="_blank"></a> In order to allocate each probe to its TUs, programs were written in PERL. Those programs analyzed the BLAST results of the probes and the different databases. Programs are available upon request from H.N-G.</p>                            <h3 scrollto-destination=87193724 id="87193724" class="section-title" >2.2. Normalization of the whole-genome array-WGA</h3>
<p>The hybridization value from a set of 12 oligonucleotide arrays representing 94% of the <em>Arabidopsis</em> genome sequence (110 Mb) was used.<a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> Each array contains 835 000 oligos of 25 mer size. Four RNA populations were hybridized to these arrays.<a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> The probe sequences and the hybridization values for the 12 slides were taken from GEO Accession viewer <a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/projects/geo/query/acc.cgi?" target="_blank"></a> using the GEO accession GPL slide numbers (GPL432-GPL443). Hybridization data were normalized by taking the log2 of the data of all probes and then by dividing each probe intensity value in a given experiment by the median intensity value of all probes in that experiment. As reported, the median intensity value of all probes were determined to be a good estimate of the background noise level in a given experiment.<a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a></p>                            <h3 scrollto-destination=87193726 id="87193726" class="section-title" >2.3. Retained-intron analysis</h3>
<p>An intron was defined as candidate constitutive or as a retained intron based on its combined probes hybridization values relative to the mean hybridization value of the other introns and exons in the transcript. In order to define retained introns in an expressed transcript the probes of its exons were treated as one group, while each intron, which contained at least three probes, was treated as different group. A balanced one-way ANOVA for comparing the means of all the groups in a transcript was performed. The ANOVA test evaluates the hypothesis that the samples all have the same mean against the alternative hypothesis that the means are not all the same. To insure a False Discovery Rate (FDR) of at most 0.05, the Benjamini-Hochberg (BH) method was used.<a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a> Transcripts in which the exons mean was at least 1.09 and the introns mean was less than 1.09 were ranked according to their <em>P</em> ANOVA value. The <em>P</em> ANOVA value was then compared with the BH critical value to select significant values. The BH critical values and the P-value for each ANOVA test are shown in Supplementary Table S2 available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a>. Using these results a test was then performed that determines which pairs of means are significantly different and which are not. This test employs the Tukey's least significant difference procedure. PERL programs were written in order to differentiate between groups that have the same mean to groups that are different from each other. Programs are available upon request from H.N-G.</p>                            <h3 scrollto-destination=87193728 id="87193728" class="section-title" >2.4. Gene ontology</h3>
<p>Gene ontology classification of transcripts was carried out using the first level categories of the Gene Ontology classification of the biological processes of their protein products (<a class="link link-uri" href="ftp://ftp.arabidopsis.org/home/tair/Ontologies/Gene_Ontology/ATH_GO_GOSLIM.20060121.txt" target="_blank"></a>). Gene ontology categories that deviate from cumulative binomial distribution with the probability for retained-intron genes <em>P</em> = 0.08 and 1 − <em>P</em> for constitutive introns genes, with significant <em>P</em>-value set at 0.05, are summarized.</p>                            <h3 scrollto-destination=87193730 id="87193730" class="section-title" >2.5. Growth of plants, cell culture, RNA extraction and RT–PCR</h3>
<p><em>Arabidopsis thaliana</em> (ecotype Columbia) seeds were surface sterilized and placed onto sterilized MS containing Petri dishes. The seeds were cold-treated at 4°C for 2 days and then incubated 7 days at 16 h light/8 h dark cycle. Flower buds were collected from mature <em>Arabidopsis</em> plants (5–7 weeks old) grown on soil. <em>Arabidopsis</em> suspension-culture cells were a generous gift from Dr Gideon Grafi. The culture was maintained at 16 h light/8 h dark cycle with constant shaking and was subcultured every week and harvested then. Root tissue was obtained by incubating surface-sterilized seeds in flasks containing 100 ml Gamborg's liquid media. The seedlings were incubated for 2 weeks with constant agitation under 16 h light/8 h dark cycle, and root tissue was collected and frozen.</p><p>RNA was extracted from the four different tissues, using RNeasy (Qiagen, Hilden, Germany). RT–PCR was conducted as previously published.<a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a> Oligonucleotide primers were designed for simultaneous amplification of both the spliced and unspliced variants or by using an internal intron probe in which the opposing primer was from the next non-adjacent exon. A complete list of the PCR primers used in this study is shown as Supplementary Table S6 available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a>.</p>                            <h2 scrollto-destination=87193733 id="87193733" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=87193734 id="87193734" class="section-title" >3.1. Division of probes in WGA to accommodate gene structure-criteria</h3>
<p>A unique set of whole-genome arrays representing ∼94% of the <em>Arabidopsis</em> genome sequence are composed of a set of 12 microarrays (6 for the forward strand and 6 for the reverse complement), each containing ∼835 000 oligos of 25 mer size. Each array contains an end-to-end ‘tile’ of oligonucleotides (from head to tail) with no overlap in their DNA sequences. In this array, only ‘perfect match’ probes were included in the whole-genome array design. The standard Affymetrix <em>Arabidopsis</em> gene expression controls are present on each of these chips. Recently, RNA from four sources were used to probe this array to determine by empirical analysis the transcriptional activity of the <em>Arabidopsis</em> genome.<a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> Tissues sampled included seedlings, flowers, suspension-culture cells and roots.</p><p>We wished to ascertain the possibility of refining from this data knowledge about alternative transcripts. In order to parse the data into established annotation of genes and their exons and introns structure the genes structures predicted by TAIR (The <em>Arabidopsis</em> Information Resource; <a class="link link-uri" href="http://www.arabidopsis.org/" target="_blank"></a>) were used. The gene structure includes the genomic location and sequence of each exon and intron. In cases in which one gene includes more then 1 transcript (e.g. known AS), the transcript structure that included the most exons was used so that a gene was divided into as many distinct groups as possible. The oligos were aligned to the exons and introns sequences in order to allocate oligos to exons and introns. The rest of the oligos that include control probes, probes with more than 1 hit in the genome, intergenic probes as well as probes that fall on intron/exon borders were removed. Thus, in total, 1 682 698 unique probes of the introns or exons were found to be amenable to analysis (Supplementary Table S1 is available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a>).</p>                            <h3 scrollto-destination=87193737 id="87193737" class="section-title" >3.2. Criteria for differentiating introns and exons in the whole-genome screen</h3>
<p>Our aim was to devise statistical tests for each transcriptional unit so that aberrant behavior of introns and exons could be registered. Hybridization values were normalized by dividing each probe intensity value in a given experiment by the median intensity value of all probes in that experiment as done previously.<a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> As hybridization values are distributed normally in a log scale, we transformed all the values to log2 and then normalized by dividing each probe intensity value in a given experiment by the median intensity value of all probes in that experiment.</p><p>We first examined the possibility of assessing retained introns by comparing the expression data of individual introns compared with a transcripts exon hybridization value. An intron was defined as ‘candidate constitutive’ or as ‘retained’ intron based on its mean hybridization value relative to the mean hybridization value of all introns and exons in a TU. To establish global statistical parameters that can be used to distinguish between intron and exon scores in a transcriptional unit, we first explored the hybridization values of introns and exons in the full genome expression database. Genes that do not contain introns or that have multiple non-unique probes in their introns are not included in this analysis. The hybridization values of 18 275 genes were used (<a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>). The mean hybridization values of all the probes that belong to the exons of a specific gene were calculated, as well as the mean hybridization values of all the probes that belong to the gene introns. This assumes that the vast majority of plant introns are constitutively spliced. The mean hybridization values distribution of all the transcripts exons and introns is shown in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>. The two samples are statistically different based on the goodness-of-fit hypothesis test (Two-sample Kolmogorov–Smirnov, <em>P</em> &lt; 10<sup>−100</sup>). A cut-off minimum for the mean hybridization value of exons of a transcript of 1.09 was chosen for analysis as at that level 58% (10 574) of the transcripts could be analyzed but only 1.5% of the introns mean hybridization values were above this level.</p>                        <a id="87193740" scrollto-destination="87193740"></a>
<div id="fig1" data-id="fig1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl003/1/m_dsl003f1.jpeg?Expires=2147483647&Signature=ks3QAHZXEWdyDwqY7oqmu4envrK89NbTz1bwhHzO9s674cREBF9S4QmbNbZSFUrq1wxCnE1s1onciSkiF7emaoJ-b3SsxuUKK6xF648QY1zswxZ9D~3~wpCXdFQxCXKY1FNHvwc6cLRN0F3sfKqQ2gZRIcTceySv2ggETWn-RuD94dMAdvGRUu6SdbFHAWG0GLNf7YzIOhFouo14s-nVGcle8uVTvptmQ9tekNco5mYkuCnbZaA4J9ofCMTrJ6wuTBxUIegekzBjI06wRPwMZ4sL6GP7CfGzCm0qDSb9X~0s8ykM~NSL1FXSuk3hJsXunhzn7JwON7tybfOsOFqgtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of the mean hybridization values of the probes in exons and introns transcripts. Probes were divided into exons and introns. The mean hybridization value of all the exons in a transcript as well as the mean hybridization value of all its introns were calculated." path-from-xml="dsl003f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193740/dsl003f1.jpeg" path-from-xml="dsl003f1.jpeg" target="_blank">View large</a><a section="87193740" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl003/1/dsl003f1.jpeg?Expires=2147483647&Signature=eagcO2mUXw9gT~raCb~C2SiaC4Pa4fZQWI~q~j9LzEBqiudOpf2A6ihn8zbkEk9S-DiXZuLw~kwMxbFMalFI5VaLh0lRJtBpd13ixAY8w0tV0OLkSsPZSuo5GIuIxHSwXR-vK8l~GMrYFrZ-XSuLUv7hQejM99YXgYHVhTP3~DwIFtyvrjodsJgVSY8Q0TJ81hjJmdoy6L~YxsRUK1UpyzwE8czn7lfV2V-RfDUg5u959yUPjSetnugvLhS7btUo22IyhV4gIX2Uz9xpzhdzBhco3f2fGYGDn75tjXgXhXrq5cs0LehUiRIVk1bVecT-ZN9g1SNS45ddh90kXBR8hQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87193740&ar=347374&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl003f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of the mean hybridization values of the probes in exons and introns transcripts. Probes were divided into exons and introns. The mean hybridization value of all the exons in a transcript as well as the mean hybridization value of all its introns were calculated.</p></div></div><div content-id="fig1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl003/1/m_dsl003f1.jpeg?Expires=2147483647&Signature=ks3QAHZXEWdyDwqY7oqmu4envrK89NbTz1bwhHzO9s674cREBF9S4QmbNbZSFUrq1wxCnE1s1onciSkiF7emaoJ-b3SsxuUKK6xF648QY1zswxZ9D~3~wpCXdFQxCXKY1FNHvwc6cLRN0F3sfKqQ2gZRIcTceySv2ggETWn-RuD94dMAdvGRUu6SdbFHAWG0GLNf7YzIOhFouo14s-nVGcle8uVTvptmQ9tekNco5mYkuCnbZaA4J9ofCMTrJ6wuTBxUIegekzBjI06wRPwMZ4sL6GP7CfGzCm0qDSb9X~0s8ykM~NSL1FXSuk3hJsXunhzn7JwON7tybfOsOFqgtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of the mean hybridization values of the probes in exons and introns transcripts. Probes were divided into exons and introns. The mean hybridization value of all the exons in a transcript as well as the mean hybridization value of all its introns were calculated." path-from-xml="dsl003f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193740/dsl003f1.jpeg" path-from-xml="dsl003f1.jpeg" target="_blank">View large</a><a section="87193740" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl003/1/dsl003f1.jpeg?Expires=2147483647&Signature=eagcO2mUXw9gT~raCb~C2SiaC4Pa4fZQWI~q~j9LzEBqiudOpf2A6ihn8zbkEk9S-DiXZuLw~kwMxbFMalFI5VaLh0lRJtBpd13ixAY8w0tV0OLkSsPZSuo5GIuIxHSwXR-vK8l~GMrYFrZ-XSuLUv7hQejM99YXgYHVhTP3~DwIFtyvrjodsJgVSY8Q0TJ81hjJmdoy6L~YxsRUK1UpyzwE8czn7lfV2V-RfDUg5u959yUPjSetnugvLhS7btUo22IyhV4gIX2Uz9xpzhdzBhco3f2fGYGDn75tjXgXhXrq5cs0LehUiRIVk1bVecT-ZN9g1SNS45ddh90kXBR8hQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87193740&ar=347374&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl003f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of the mean hybridization values of the probes in exons and introns transcripts. Probes were divided into exons and introns. The mean hybridization value of all the exons in a transcript as well as the mean hybridization value of all its introns were calculated.</p></div></div>                        <a id="87193741" scrollto-destination="87193741"></a>
<div content-id="tbl1" class="table-modal"><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Intron and exon classes in the WGA survey</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Type </th><th>Genes 1 </th><th>Introns/exons<sup>a</sup> </th><th>Analyzed transcripts<sup>b</sup> </th><th>Class 1<sup>c</sup> </th><th>Class 2<sup>c</sup> </th><th>Class 3<sup>c</sup> </th><th>Undifferentiated<sup>c</sup> </th><th>%IR/%ES<sup>d</sup> </th></tr></thead><tbody><tr><td>Intron </td><td>18 275 </td><td>59 510 </td><td>10 270 </td><td>8123 </td><td>644 </td><td>59 </td><td>1444 </td><td char=".">7.97 </td></tr><tr><td>Exon </td><td>19 782 </td><td>92 456 </td><td>10 725 </td><td>6149 </td><td>452 </td><td>125 </td><td>− </td><td char=".">8.6 </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th>Type </th><th>Genes 1 </th><th>Introns/exons<sup>a</sup> </th><th>Analyzed transcripts<sup>b</sup> </th><th>Class 1<sup>c</sup> </th><th>Class 2<sup>c</sup> </th><th>Class 3<sup>c</sup> </th><th>Undifferentiated<sup>c</sup> </th><th>%IR/%ES<sup>d</sup> </th></tr></thead><tbody><tr><td>Intron </td><td>18 275 </td><td>59 510 </td><td>10 270 </td><td>8123 </td><td>644 </td><td>59 </td><td>1444 </td><td char=".">7.97 </td></tr><tr><td>Exon </td><td>19 782 </td><td>92 456 </td><td>10 725 </td><td>6149 </td><td>452 </td><td>125 </td><td>− </td><td char=".">8.6 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>All exons and introns probes in Table S1 available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a> were parsed to the specific transcript that they belong to. Of these 59 510 introns and 92 456 exons contained at least three probes/intron or exon. Those introns or exons are part of 18 275 or 19 782 genes, respectively.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Values from the four different RNA samples were combined. Transcripts were used for further analysis if the mean hybridization value of its exons was ≥1.09 and the mean hybridization value of its introns was &lt;1.09. Transcripts in which the <em>P</em>-value of the ANOVA test was above the BH critical value were also excluded. The BH critical values and the <em>P</em>-value for each ANOVA test are shown in Supplementary Table S2 available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a>. In the exon analysis 727 and 449 exons that were the first or last exon skipped in class 2 and 3, respectively, were not included as their veracity could not be determined.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Transcripts classes are as described in the text; Class 1, constitutive; Class 2, retained or skipped; Class 3, intermediate.</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>Intron retention and Exon skipped rate are computed by dividing the total number of Class 2 and 3 transcripts by the number of statistically significant cases (analyzed transcripts less undifferentiated transcripts).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87193741" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Intron and exon classes in the WGA survey</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Type </th><th>Genes 1 </th><th>Introns/exons<sup>a</sup> </th><th>Analyzed transcripts<sup>b</sup> </th><th>Class 1<sup>c</sup> </th><th>Class 2<sup>c</sup> </th><th>Class 3<sup>c</sup> </th><th>Undifferentiated<sup>c</sup> </th><th>%IR/%ES<sup>d</sup> </th></tr></thead><tbody><tr><td>Intron </td><td>18 275 </td><td>59 510 </td><td>10 270 </td><td>8123 </td><td>644 </td><td>59 </td><td>1444 </td><td char=".">7.97 </td></tr><tr><td>Exon </td><td>19 782 </td><td>92 456 </td><td>10 725 </td><td>6149 </td><td>452 </td><td>125 </td><td>− </td><td char=".">8.6 </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th>Type </th><th>Genes 1 </th><th>Introns/exons<sup>a</sup> </th><th>Analyzed transcripts<sup>b</sup> </th><th>Class 1<sup>c</sup> </th><th>Class 2<sup>c</sup> </th><th>Class 3<sup>c</sup> </th><th>Undifferentiated<sup>c</sup> </th><th>%IR/%ES<sup>d</sup> </th></tr></thead><tbody><tr><td>Intron </td><td>18 275 </td><td>59 510 </td><td>10 270 </td><td>8123 </td><td>644 </td><td>59 </td><td>1444 </td><td char=".">7.97 </td></tr><tr><td>Exon </td><td>19 782 </td><td>92 456 </td><td>10 725 </td><td>6149 </td><td>452 </td><td>125 </td><td>− </td><td char=".">8.6 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>All exons and introns probes in Table S1 available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a> were parsed to the specific transcript that they belong to. Of these 59 510 introns and 92 456 exons contained at least three probes/intron or exon. Those introns or exons are part of 18 275 or 19 782 genes, respectively.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Values from the four different RNA samples were combined. Transcripts were used for further analysis if the mean hybridization value of its exons was ≥1.09 and the mean hybridization value of its introns was &lt;1.09. Transcripts in which the <em>P</em>-value of the ANOVA test was above the BH critical value were also excluded. The BH critical values and the <em>P</em>-value for each ANOVA test are shown in Supplementary Table S2 available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a>. In the exon analysis 727 and 449 exons that were the first or last exon skipped in class 2 and 3, respectively, were not included as their veracity could not be determined.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Transcripts classes are as described in the text; Class 1, constitutive; Class 2, retained or skipped; Class 3, intermediate.</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>Intron retention and Exon skipped rate are computed by dividing the total number of Class 2 and 3 transcripts by the number of statistically significant cases (analyzed transcripts less undifferentiated transcripts).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87193741" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87193742 id="87193742" class="section-title" >3.3. Examining intron retention in whole-genome arrays</h3>
<p>In order to identify retained introns in the 10 574 transcripts, the probes of the exons in each transcript were treated as one group, while each intron, which contained at least three probes, was treated as a different group. The three probes limitation was used to increase the significance of the statistical tests. A balanced one-way ANOVA for comparing the means of all the groups in a transcript was performed. To insure a FDR of at most 0.05, the BH method was used.<a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a> The <em>P</em>-value of the ANOVA tests was found to be statistically significant in 10 270 transcripts (analyzed transcripts, <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>) and used to define three different intron classes.</p><p>In Class 1 transcripts, all the introns hybridization values are different from the exon group (<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). In the example shown, the AT2G36530 transcript is defined as containing only constitutive introns as the exons group is significantly different from all the introns. In Class 2 transcripts, one intron is statistically similar to the exons group. Thus, in the case of transcript AT2G47470, intron 5 is defined as a retained intron (<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). Intermediate cases are exemplified in Class 3 (intermediate retained). In this case the exceptional intron is expressed at a level that is between the exons group and all the other introns. In the example shown in <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2C</a>, the AT5G11200 transcript shows that intron 6 hybridization values are between the exon and intron groups.</p>                        <a id="87193745" scrollto-destination="87193745"></a>
<div id="fig2" data-id="fig2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl003/1/m_dsl003f2.jpeg?Expires=2147483647&Signature=uLHOQIAhLEAHpGWDzzVfWlf9L-Vp6miv6X6MSda9nUYo96Mz3Pjt7QMEBJcX7Wf87ake53XObkVtXvhEct1DqWcxYSVccQo4LLpEt-8zRip~8Q-ygqi4pzHA5Mr2-4XvFRV0g-3FRn7cuQta26V9VkL-Hi5pWLSxYFok2xlXE8n4WhwdN2p~oQRAsONK0FGvvXyuCdI3AZE769LBviRKhIyJ9M5DFdP0YJ5FthrQl0auue2UjxJMppkh42XiIcN1jDKX5iL9oFjRugTREYNxnZGLQC1EAq3ooWaKay5vlE5Ag0vvbggQ4muWFph35uH~uyy6igo~kwGzL757YQHyjg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic expression profiles of introns and exons. Each circle represents one intron or exon group defined by at least three probes. The circle represents the mean hybridization values of the group and the line borders are the 95% confidence level of the mean. (A) Transcript hybridization values of AT2G36530. (B) Transcript hybridization values of AT2G47470. (C) Transcript hybridization values of AT5G11200." path-from-xml="dsl003f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193745/dsl003f2.jpeg" path-from-xml="dsl003f2.jpeg" target="_blank">View large</a><a section="87193745" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl003/1/dsl003f2.jpeg?Expires=2147483647&Signature=PY~H84DzdjGCWtMtYu7PDsTDfyaeX-2-jEJ9lHr9Qd2h6pP5wr-e6E4dYWAkutvKw-mBgiyKbFwlAMYrhRBBGgReDi5CmOqM4Ufw~~meNEjvWX2UCFuaxBFgGwxa2dMR6c9GFMHM~JXPsNFfYDwaaM-8ypbbPMZgI1Cy7LnjmKHUOlKgkkTVxlDUuT~tYOdLnTzXWD2-jJb6uFrvRsm~DkDUjmcg4UJTFPfElxhXcq0XPvsQ6rnttl2psPhPgfAGGhhCkPYuho6EgbxBf5mufMmfGG8ZW3FImXxJP855bGR5z-Wm~J7ryCM54-15fRQaJnOpnhYb3gj-5cCWOgK2ZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87193745&ar=347374&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl003f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic expression profiles of introns and exons. Each circle represents one intron or exon group defined by at least three probes. The circle represents the mean hybridization values of the group and the line borders are the 95% confidence level of the mean. (<strong>A</strong>) Transcript hybridization values of AT2G36530. (<strong>B</strong>) Transcript hybridization values of AT2G47470. (<strong>C</strong>) Transcript hybridization values of AT5G11200.</p></div></div><div content-id="fig2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl003/1/m_dsl003f2.jpeg?Expires=2147483647&Signature=uLHOQIAhLEAHpGWDzzVfWlf9L-Vp6miv6X6MSda9nUYo96Mz3Pjt7QMEBJcX7Wf87ake53XObkVtXvhEct1DqWcxYSVccQo4LLpEt-8zRip~8Q-ygqi4pzHA5Mr2-4XvFRV0g-3FRn7cuQta26V9VkL-Hi5pWLSxYFok2xlXE8n4WhwdN2p~oQRAsONK0FGvvXyuCdI3AZE769LBviRKhIyJ9M5DFdP0YJ5FthrQl0auue2UjxJMppkh42XiIcN1jDKX5iL9oFjRugTREYNxnZGLQC1EAq3ooWaKay5vlE5Ag0vvbggQ4muWFph35uH~uyy6igo~kwGzL757YQHyjg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic expression profiles of introns and exons. Each circle represents one intron or exon group defined by at least three probes. The circle represents the mean hybridization values of the group and the line borders are the 95% confidence level of the mean. (A) Transcript hybridization values of AT2G36530. (B) Transcript hybridization values of AT2G47470. (C) Transcript hybridization values of AT5G11200." path-from-xml="dsl003f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193745/dsl003f2.jpeg" path-from-xml="dsl003f2.jpeg" target="_blank">View large</a><a section="87193745" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl003/1/dsl003f2.jpeg?Expires=2147483647&Signature=PY~H84DzdjGCWtMtYu7PDsTDfyaeX-2-jEJ9lHr9Qd2h6pP5wr-e6E4dYWAkutvKw-mBgiyKbFwlAMYrhRBBGgReDi5CmOqM4Ufw~~meNEjvWX2UCFuaxBFgGwxa2dMR6c9GFMHM~JXPsNFfYDwaaM-8ypbbPMZgI1Cy7LnjmKHUOlKgkkTVxlDUuT~tYOdLnTzXWD2-jJb6uFrvRsm~DkDUjmcg4UJTFPfElxhXcq0XPvsQ6rnttl2psPhPgfAGGhhCkPYuho6EgbxBf5mufMmfGG8ZW3FImXxJP855bGR5z-Wm~J7ryCM54-15fRQaJnOpnhYb3gj-5cCWOgK2ZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87193745&ar=347374&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl003f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic expression profiles of introns and exons. Each circle represents one intron or exon group defined by at least three probes. The circle represents the mean hybridization values of the group and the line borders are the 95% confidence level of the mean. (<strong>A</strong>) Transcript hybridization values of AT2G36530. (<strong>B</strong>) Transcript hybridization values of AT2G47470. (<strong>C</strong>) Transcript hybridization values of AT5G11200.</p></div></div><p>Examination of the rest of the transcripts shows that 8206 out of 10 270 (80%) fall into one of the intron classes. In the remaining 2064 cases, there is no clear statistical differentiation of the intron and exon groups. In these cases, the transcript exon and intron groups are compared again using a lower confidence level (<em>a</em> = 0.1 instead of 0.05). This causes the interval around the mean of each group to contract. At this probability level conclusions could be drawn for an additional 620 cases. In the remaining 14% of transcripts the statistical variance was too great and no clear definition of the intron/exon expression boundaries could be made. In these cases, the transcripts were termed ‘undifferentiated’. The algorithms above as applied to the whole-genome arrays are summarized in the <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>. Thus, ∼8% rate of the transcripts that could be measured contained retained introns. This is slightly higher than previous EST and cDNA-derived estimates for retained introns of between 3 to 6% of total transcripts.<a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a> The detailed analysis of all the transcripts are tabulated in Table S2 available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a>.</p>                            <h3 scrollto-destination=87193747 id="87193747" class="section-title" >3.4. Confirmation of retained introns by RT–PCR analysis</h3>
<p>To test the veracity of using WGA expression data to characterize AS, a sample of 14 AS events and 5 constitutive-type introns was examined by RT–PCR. The genes were selected such that they represent low- and high-expression value transcripts, retained introns in different parts of the transcripts and contain intermediate and retained-type introns. Each PCR was carried out using RNA that was extracted from four different samples including; seedlings, flowers, suspension-culture cells and roots that were combined together. Retained introns were traced either by primers that flank the intron or by using an internal intron probe in which the opposing primer was from the next non-adjacent exon. In this way, retained (I) and the smaller spliced fragment (S) could be recovered. However, if a fragment larger than that predicted by ‘I’ was detected, i.e. a fragment that included the non-adjacent but normally constitutively spliced intron was detected, this indicated that the RT–PCR amplification used pre-mRNA as its template and was rejected. All fragments containing putative introns were confirmed by sequencing (data not shown). The results shown in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3</a> and summarized in <a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a> indicate that 12 out of 14 introns (86%) that were predicted to be retained by the expression analysis were detected by RT–PCR. In a similar manner, when constitutive introns were examined 4 out of 5 could be verified as being constitutive.</p>                        <a id="87193749" scrollto-destination="87193749"></a>
<div id="fig3" data-id="fig3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl003/1/m_dsl003f3.jpeg?Expires=2147483647&Signature=3lOMIXpHYZTTaHMtBJoZUlMqT0dt5SBOtLoBhDydHatOsY6P1AttObvepatqVtgwmOF4YM6rlWeQFmZXLigSbjk1MjsknkuaSkwP~8RWwIgfjGR08WjFPvhupxfZN1XvmcHncUXYGSMHKRmmEvOuVtNw9CxJ3oi3kKUuTmYV-xYHbjNtpC0ktdhH2FFdxik6QDyti7biMQccdcJqaGa9GEXULKi~Uc94cmxI4fs1QOAlYFLxX3K5ORIrviZKgSVdgT8UK-QZJ-wCozufPzj9MF2bR3NMeppJNTqW8F4seLl3cAIRMTudCnjbFQiKmqF5z2XoIGqXHrKVhZaTcz7dVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Composite of gene structure, primer choices and gel fractionation of RT–PCR products. The gene structure is from TIGR View (=). Arrows indicate the position of primers used. In cases where flanking intron probes showed two migrating fragments one gel insert is shown. In cases using intron probes, the appropriate controls are also shown. The results are summarized in Table 2. I and S indicates retained or spliced fragment, P indicate pre-mRNA, m indicates marker lane." path-from-xml="dsl003f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193749/dsl003f3.jpeg" path-from-xml="dsl003f3.jpeg" target="_blank">View large</a><a section="87193749" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl003/1/dsl003f3.jpeg?Expires=2147483647&Signature=s07v3gDR-u7pPegGYenCidXaD7mLbTCaWn4XUoKepcks-PAAq6HSY4T1uYVoUtS8L8Rd5Uq6Ls~SYSfGYhYU6emHZCboBveoojIUbkc9r-iLflzW6tdOWcw58G4U0g6yy2gvg7xABisamnsq~yGhgkVsEDHuM-1RumNxxSrKsznino24Lg1og620OJ487GKX9p1XDuuzjKMw376C9Wi7q3whz9q2sf7WIG1XvoA36qTOqmcV1VUIR0d-HhuWRKgxjh-GgRwd01XEyyN5IdlYIS68D0UBK8GAiIf9BM19Pz8T26LmjNBo34Y5SP2Y-~SwevrBdDJ~2zzI6SBpyEPzIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87193749&ar=347374&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl003f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Composite of gene structure, primer choices and gel fractionation of RT–PCR products. The gene structure is from TIGR View (<a class="link link-uri" href="http://www.tigr.org/tigr-scripts/tgi/T_index.cgi?species" target="_blank"></a>=<a class="link link-uri" href="http://arab" target="_blank"></a>). Arrows indicate the position of primers used. In cases where flanking intron probes showed two migrating fragments one gel insert is shown. In cases using intron probes, the appropriate controls are also shown. The results are summarized in <a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a>. I and S indicates retained or spliced fragment, P indicate pre-mRNA, m indicates marker lane.</p></div></div><div content-id="fig3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl003/1/m_dsl003f3.jpeg?Expires=2147483647&Signature=3lOMIXpHYZTTaHMtBJoZUlMqT0dt5SBOtLoBhDydHatOsY6P1AttObvepatqVtgwmOF4YM6rlWeQFmZXLigSbjk1MjsknkuaSkwP~8RWwIgfjGR08WjFPvhupxfZN1XvmcHncUXYGSMHKRmmEvOuVtNw9CxJ3oi3kKUuTmYV-xYHbjNtpC0ktdhH2FFdxik6QDyti7biMQccdcJqaGa9GEXULKi~Uc94cmxI4fs1QOAlYFLxX3K5ORIrviZKgSVdgT8UK-QZJ-wCozufPzj9MF2bR3NMeppJNTqW8F4seLl3cAIRMTudCnjbFQiKmqF5z2XoIGqXHrKVhZaTcz7dVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Composite of gene structure, primer choices and gel fractionation of RT–PCR products. The gene structure is from TIGR View (=). Arrows indicate the position of primers used. In cases where flanking intron probes showed two migrating fragments one gel insert is shown. In cases using intron probes, the appropriate controls are also shown. The results are summarized in Table 2. I and S indicates retained or spliced fragment, P indicate pre-mRNA, m indicates marker lane." path-from-xml="dsl003f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193749/dsl003f3.jpeg" path-from-xml="dsl003f3.jpeg" target="_blank">View large</a><a section="87193749" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl003/1/dsl003f3.jpeg?Expires=2147483647&Signature=s07v3gDR-u7pPegGYenCidXaD7mLbTCaWn4XUoKepcks-PAAq6HSY4T1uYVoUtS8L8Rd5Uq6Ls~SYSfGYhYU6emHZCboBveoojIUbkc9r-iLflzW6tdOWcw58G4U0g6yy2gvg7xABisamnsq~yGhgkVsEDHuM-1RumNxxSrKsznino24Lg1og620OJ487GKX9p1XDuuzjKMw376C9Wi7q3whz9q2sf7WIG1XvoA36qTOqmcV1VUIR0d-HhuWRKgxjh-GgRwd01XEyyN5IdlYIS68D0UBK8GAiIf9BM19Pz8T26LmjNBo34Y5SP2Y-~SwevrBdDJ~2zzI6SBpyEPzIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87193749&ar=347374&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl003f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Composite of gene structure, primer choices and gel fractionation of RT–PCR products. The gene structure is from TIGR View (<a class="link link-uri" href="http://www.tigr.org/tigr-scripts/tgi/T_index.cgi?species" target="_blank"></a>=<a class="link link-uri" href="http://arab" target="_blank"></a>). Arrows indicate the position of primers used. In cases where flanking intron probes showed two migrating fragments one gel insert is shown. In cases using intron probes, the appropriate controls are also shown. The results are summarized in <a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a>. I and S indicates retained or spliced fragment, P indicate pre-mRNA, m indicates marker lane.</p></div></div>                        <a id="87193750" scrollto-destination="87193750"></a>
<div content-id="tbl2" class="table-modal"><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Detection of retained introns by RT–PCR</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene identifier </th><th>Exons mean </th><th>Introns mean </th><th>α<sup>a</sup> </th><th>Type of intron<sup>b</sup> </th><th>Flanking primers<sup>c</sup> </th><th>Intron primers<sup>d</sup> </th><th>Location of intron </th><th>Intron results<sup>e,f</sup> </th></tr></thead><tbody><tr><td>AT4G03200 </td><td char=".">1.10 </td><td char=".">1.00 </td><td char=".">0.05 </td><td>Retained </td><td>S </td><td>Pre RNA </td><td>Intron 9 </td><td>− </td></tr><tr><td>AT1G18160 </td><td char=".">1.11 </td><td char=".">1.00 </td><td char=".">0.1 </td><td>Retained </td><td>S </td><td>I </td><td>Intron 2 </td><td>+ </td></tr><tr><td>AT1G27320 </td><td char=".">1.12 </td><td char=".">1.01 </td><td char=".">0.05 </td><td>Retained </td><td>I, S </td><td> </td><td>Intron 10–3′-UTR </td><td>+ </td></tr><tr><td>AT2G19620 </td><td char=".">1.13 </td><td char=".">1.02 </td><td char=".">0.05 </td><td>Retained </td><td>S </td><td>I </td><td>Intron 3 </td><td>+ </td></tr><tr><td>AT1G22930 </td><td char=".">1.13 </td><td char=".">1.00 </td><td char=".">0.05 </td><td>Retained </td><td> </td><td>Pre RNA </td><td>Intron 7 </td><td>− </td></tr><tr><td>AT1G22770 </td><td char=".">1.14 </td><td char=".">1.03 </td><td char=".">0.05 </td><td>Retained </td><td>S </td><td>I </td><td>Intron 5 </td><td>+ </td></tr><tr><td>AT1G02410 </td><td char=".">1.14 </td><td char=".">1.02 </td><td char=".">0.05 </td><td>Retained </td><td>S </td><td>I </td><td>Intron 1–5′-UTR </td><td>+ </td></tr><tr><td>AT2G19940 </td><td char=".">1.16 </td><td char=".">1.00 </td><td char=".">0.1 </td><td>Retained </td><td>I, S </td><td>I </td><td>Intron 14 </td><td>+ </td></tr><tr><td>AT1G76140 </td><td char=".">1.16 </td><td char=".">1.00 </td><td char=".">0.05 </td><td>Retained </td><td>S </td><td>I </td><td>Intron 6 </td><td>+ </td></tr><tr><td>AT3G48440 </td><td char=".">1.17 </td><td char=".">1.03 </td><td char=".">0.05 </td><td>Retained </td><td>I, S </td><td>I </td><td>Intron 2 </td><td>+ </td></tr><tr><td>AT3G13300 </td><td char=".">1.20 </td><td char=".">1.03 </td><td char=".">0.05 </td><td>Retained </td><td>I, S </td><td>ND </td><td>Intron 1 </td><td>+ </td></tr><tr><td>AT1G32230 </td><td char=".">1.25 </td><td char=".">1.05 </td><td char=".">0.05 </td><td>Intermediate </td><td>I, S </td><td>ND </td><td>Intron 4 </td><td>+ </td></tr><tr><td>AT1G76030 </td><td char=".">1.31 </td><td char=".">1.00 </td><td char=".">0.05 </td><td>Intermediate </td><td>S </td><td>I </td><td>Intron 5 </td><td>+ </td></tr><tr><td>AT1G48920 </td><td char=".">1.37 </td><td char=".">1.01 </td><td char=".">0.05 </td><td>Intermediate </td><td>S </td><td>I </td><td>Intron 1 </td><td>+ </td></tr><tr><td>AT1G53310 </td><td char=".">1.24 </td><td char=".">1.00 </td><td char=".">0.05 </td><td>Constitutive </td><td>S </td><td>Pre RNA </td><td>Intron 2 </td><td>+ </td></tr><tr><td>AT4G04910 </td><td char=".">1.16 </td><td char=".">0.99 </td><td char=".">0.05 </td><td>Constitutive </td><td>S </td><td>Pre RNA </td><td>Intron 11 </td><td>+ </td></tr><tr><td>AT2G23070 </td><td char=".">1.28 </td><td char=".">1.02 </td><td char=".">0.05 </td><td>Constitutive </td><td>S </td><td>ND </td><td>Intron 3 </td><td>+ </td></tr><tr><td>AT3G21215 </td><td char=".">1.25 </td><td char=".">1.02 </td><td char=".">0.05 </td><td>Constitutive </td><td>S </td><td>ND </td><td>Intron 3 </td><td>+ </td></tr><tr><td>AT5G20380 </td><td char=".">1.17 </td><td char=".">1.00 </td><td char=".">0.05 </td><td>Constitutive </td><td>I,S </td><td>Pre RNA, I </td><td>Intron 11 </td><td>− </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene identifier </th><th>Exons mean </th><th>Introns mean </th><th>α<sup>a</sup> </th><th>Type of intron<sup>b</sup> </th><th>Flanking primers<sup>c</sup> </th><th>Intron primers<sup>d</sup> </th><th>Location of intron </th><th>Intron results<sup>e,f</sup> </th></tr></thead><tbody><tr><td>AT4G03200 </td><td char=".">1.10 </td><td char=".">1.00 </td><td char=".">0.05 </td><td>Retained </td><td>S </td><td>Pre RNA </td><td>Intron 9 </td><td>− </td></tr><tr><td>AT1G18160 </td><td char=".">1.11 </td><td char=".">1.00 </td><td char=".">0.1 </td><td>Retained </td><td>S </td><td>I </td><td>Intron 2 </td><td>+ </td></tr><tr><td>AT1G27320 </td><td char=".">1.12 </td><td char=".">1.01 </td><td char=".">0.05 </td><td>Retained </td><td>I, S </td><td> </td><td>Intron 10–3′-UTR </td><td>+ </td></tr><tr><td>AT2G19620 </td><td char=".">1.13 </td><td char=".">1.02 </td><td char=".">0.05 </td><td>Retained </td><td>S </td><td>I </td><td>Intron 3 </td><td>+ </td></tr><tr><td>AT1G22930 </td><td char=".">1.13 </td><td char=".">1.00 </td><td char=".">0.05 </td><td>Retained </td><td> </td><td>Pre RNA </td><td>Intron 7 </td><td>− </td></tr><tr><td>AT1G22770 </td><td char=".">1.14 </td><td char=".">1.03 </td><td char=".">0.05 </td><td>Retained </td><td>S </td><td>I </td><td>Intron 5 </td><td>+ </td></tr><tr><td>AT1G02410 </td><td char=".">1.14 </td><td char=".">1.02 </td><td char=".">0.05 </td><td>Retained </td><td>S </td><td>I </td><td>Intron 1–5′-UTR </td><td>+ </td></tr><tr><td>AT2G19940 </td><td char=".">1.16 </td><td char=".">1.00 </td><td char=".">0.1 </td><td>Retained </td><td>I, S </td><td>I </td><td>Intron 14 </td><td>+ </td></tr><tr><td>AT1G76140 </td><td char=".">1.16 </td><td char=".">1.00 </td><td char=".">0.05 </td><td>Retained </td><td>S </td><td>I </td><td>Intron 6 </td><td>+ </td></tr><tr><td>AT3G48440 </td><td char=".">1.17 </td><td char=".">1.03 </td><td char=".">0.05 </td><td>Retained </td><td>I, S </td><td>I </td><td>Intron 2 </td><td>+ </td></tr><tr><td>AT3G13300 </td><td char=".">1.20 </td><td char=".">1.03 </td><td char=".">0.05 </td><td>Retained </td><td>I, S </td><td>ND </td><td>Intron 1 </td><td>+ </td></tr><tr><td>AT1G32230 </td><td char=".">1.25 </td><td char=".">1.05 </td><td char=".">0.05 </td><td>Intermediate </td><td>I, S </td><td>ND </td><td>Intron 4 </td><td>+ </td></tr><tr><td>AT1G76030 </td><td char=".">1.31 </td><td char=".">1.00 </td><td char=".">0.05 </td><td>Intermediate </td><td>S </td><td>I </td><td>Intron 5 </td><td>+ </td></tr><tr><td>AT1G48920 </td><td char=".">1.37 </td><td char=".">1.01 </td><td char=".">0.05 </td><td>Intermediate </td><td>S </td><td>I </td><td>Intron 1 </td><td>+ </td></tr><tr><td>AT1G53310 </td><td char=".">1.24 </td><td char=".">1.00 </td><td char=".">0.05 </td><td>Constitutive </td><td>S </td><td>Pre RNA </td><td>Intron 2 </td><td>+ </td></tr><tr><td>AT4G04910 </td><td char=".">1.16 </td><td char=".">0.99 </td><td char=".">0.05 </td><td>Constitutive </td><td>S </td><td>Pre RNA </td><td>Intron 11 </td><td>+ </td></tr><tr><td>AT2G23070 </td><td char=".">1.28 </td><td char=".">1.02 </td><td char=".">0.05 </td><td>Constitutive </td><td>S </td><td>ND </td><td>Intron 3 </td><td>+ </td></tr><tr><td>AT3G21215 </td><td char=".">1.25 </td><td char=".">1.02 </td><td char=".">0.05 </td><td>Constitutive </td><td>S </td><td>ND </td><td>Intron 3 </td><td>+ </td></tr><tr><td>AT5G20380 </td><td char=".">1.17 </td><td char=".">1.00 </td><td char=".">0.05 </td><td>Constitutive </td><td>I,S </td><td>Pre RNA, I </td><td>Intron 11 </td><td>− </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Chosen significance level value.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Intron types were selected from <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Flanking primers (Supplementary Table S6 is available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a>) yielded either ‘I’ which indicates fragments that includes the intron or ‘S’ which indicates fragment size indicative of a spliced transcript.</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>In the cases in which ‘I’ was not detected by using flanking primers, another PCR was done, using intron primers. ‘I’ indicates transcripts that include the retained intron but in which the other intron was spliced. ‘Pre-RNA’ indicates that the slower migrating fragment of the constitutive and retained intron was detected. ND, not detected.</p></div></div><div content-id=""><div class="fn"><p><sup>e</sup>Intron that was indicated in the bioinformatics analysis as retained and was detected by RT–PCR was marked as ‘+’. Intron that was supposed to be retained but was not detected was marked ‘−’.</p></div></div><div content-id=""><div class="fn"><p><sup>f</sup>Intron that was indicated in the bioinformatics analysis as constitutively spliced and was not detected in the PCR was marked as ‘+’. Intron that was supposed to be spliced out but was detected was marked ‘−’.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87193750" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Detection of retained introns by RT–PCR</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene identifier </th><th>Exons mean </th><th>Introns mean </th><th>α<sup>a</sup> </th><th>Type of intron<sup>b</sup> </th><th>Flanking primers<sup>c</sup> </th><th>Intron primers<sup>d</sup> </th><th>Location of intron </th><th>Intron results<sup>e,f</sup> </th></tr></thead><tbody><tr><td>AT4G03200 </td><td char=".">1.10 </td><td char=".">1.00 </td><td char=".">0.05 </td><td>Retained </td><td>S </td><td>Pre RNA </td><td>Intron 9 </td><td>− </td></tr><tr><td>AT1G18160 </td><td char=".">1.11 </td><td char=".">1.00 </td><td char=".">0.1 </td><td>Retained </td><td>S </td><td>I </td><td>Intron 2 </td><td>+ </td></tr><tr><td>AT1G27320 </td><td char=".">1.12 </td><td char=".">1.01 </td><td char=".">0.05 </td><td>Retained </td><td>I, S </td><td> </td><td>Intron 10–3′-UTR </td><td>+ </td></tr><tr><td>AT2G19620 </td><td char=".">1.13 </td><td char=".">1.02 </td><td char=".">0.05 </td><td>Retained </td><td>S </td><td>I </td><td>Intron 3 </td><td>+ </td></tr><tr><td>AT1G22930 </td><td char=".">1.13 </td><td char=".">1.00 </td><td char=".">0.05 </td><td>Retained </td><td> </td><td>Pre RNA </td><td>Intron 7 </td><td>− </td></tr><tr><td>AT1G22770 </td><td char=".">1.14 </td><td char=".">1.03 </td><td char=".">0.05 </td><td>Retained </td><td>S </td><td>I </td><td>Intron 5 </td><td>+ </td></tr><tr><td>AT1G02410 </td><td char=".">1.14 </td><td char=".">1.02 </td><td char=".">0.05 </td><td>Retained </td><td>S </td><td>I </td><td>Intron 1–5′-UTR </td><td>+ </td></tr><tr><td>AT2G19940 </td><td char=".">1.16 </td><td char=".">1.00 </td><td char=".">0.1 </td><td>Retained </td><td>I, S </td><td>I </td><td>Intron 14 </td><td>+ </td></tr><tr><td>AT1G76140 </td><td char=".">1.16 </td><td char=".">1.00 </td><td char=".">0.05 </td><td>Retained </td><td>S </td><td>I </td><td>Intron 6 </td><td>+ </td></tr><tr><td>AT3G48440 </td><td char=".">1.17 </td><td char=".">1.03 </td><td char=".">0.05 </td><td>Retained </td><td>I, S </td><td>I </td><td>Intron 2 </td><td>+ </td></tr><tr><td>AT3G13300 </td><td char=".">1.20 </td><td char=".">1.03 </td><td char=".">0.05 </td><td>Retained </td><td>I, S </td><td>ND </td><td>Intron 1 </td><td>+ </td></tr><tr><td>AT1G32230 </td><td char=".">1.25 </td><td char=".">1.05 </td><td char=".">0.05 </td><td>Intermediate </td><td>I, S </td><td>ND </td><td>Intron 4 </td><td>+ </td></tr><tr><td>AT1G76030 </td><td char=".">1.31 </td><td char=".">1.00 </td><td char=".">0.05 </td><td>Intermediate </td><td>S </td><td>I </td><td>Intron 5 </td><td>+ </td></tr><tr><td>AT1G48920 </td><td char=".">1.37 </td><td char=".">1.01 </td><td char=".">0.05 </td><td>Intermediate </td><td>S </td><td>I </td><td>Intron 1 </td><td>+ </td></tr><tr><td>AT1G53310 </td><td char=".">1.24 </td><td char=".">1.00 </td><td char=".">0.05 </td><td>Constitutive </td><td>S </td><td>Pre RNA </td><td>Intron 2 </td><td>+ </td></tr><tr><td>AT4G04910 </td><td char=".">1.16 </td><td char=".">0.99 </td><td char=".">0.05 </td><td>Constitutive </td><td>S </td><td>Pre RNA </td><td>Intron 11 </td><td>+ </td></tr><tr><td>AT2G23070 </td><td char=".">1.28 </td><td char=".">1.02 </td><td char=".">0.05 </td><td>Constitutive </td><td>S </td><td>ND </td><td>Intron 3 </td><td>+ </td></tr><tr><td>AT3G21215 </td><td char=".">1.25 </td><td char=".">1.02 </td><td char=".">0.05 </td><td>Constitutive </td><td>S </td><td>ND </td><td>Intron 3 </td><td>+ </td></tr><tr><td>AT5G20380 </td><td char=".">1.17 </td><td char=".">1.00 </td><td char=".">0.05 </td><td>Constitutive </td><td>I,S </td><td>Pre RNA, I </td><td>Intron 11 </td><td>− </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene identifier </th><th>Exons mean </th><th>Introns mean </th><th>α<sup>a</sup> </th><th>Type of intron<sup>b</sup> </th><th>Flanking primers<sup>c</sup> </th><th>Intron primers<sup>d</sup> </th><th>Location of intron </th><th>Intron results<sup>e,f</sup> </th></tr></thead><tbody><tr><td>AT4G03200 </td><td char=".">1.10 </td><td char=".">1.00 </td><td char=".">0.05 </td><td>Retained </td><td>S </td><td>Pre RNA </td><td>Intron 9 </td><td>− </td></tr><tr><td>AT1G18160 </td><td char=".">1.11 </td><td char=".">1.00 </td><td char=".">0.1 </td><td>Retained </td><td>S </td><td>I </td><td>Intron 2 </td><td>+ </td></tr><tr><td>AT1G27320 </td><td char=".">1.12 </td><td char=".">1.01 </td><td char=".">0.05 </td><td>Retained </td><td>I, S </td><td> </td><td>Intron 10–3′-UTR </td><td>+ </td></tr><tr><td>AT2G19620 </td><td char=".">1.13 </td><td char=".">1.02 </td><td char=".">0.05 </td><td>Retained </td><td>S </td><td>I </td><td>Intron 3 </td><td>+ </td></tr><tr><td>AT1G22930 </td><td char=".">1.13 </td><td char=".">1.00 </td><td char=".">0.05 </td><td>Retained </td><td> </td><td>Pre RNA </td><td>Intron 7 </td><td>− </td></tr><tr><td>AT1G22770 </td><td char=".">1.14 </td><td char=".">1.03 </td><td char=".">0.05 </td><td>Retained </td><td>S </td><td>I </td><td>Intron 5 </td><td>+ </td></tr><tr><td>AT1G02410 </td><td char=".">1.14 </td><td char=".">1.02 </td><td char=".">0.05 </td><td>Retained </td><td>S </td><td>I </td><td>Intron 1–5′-UTR </td><td>+ </td></tr><tr><td>AT2G19940 </td><td char=".">1.16 </td><td char=".">1.00 </td><td char=".">0.1 </td><td>Retained </td><td>I, S </td><td>I </td><td>Intron 14 </td><td>+ </td></tr><tr><td>AT1G76140 </td><td char=".">1.16 </td><td char=".">1.00 </td><td char=".">0.05 </td><td>Retained </td><td>S </td><td>I </td><td>Intron 6 </td><td>+ </td></tr><tr><td>AT3G48440 </td><td char=".">1.17 </td><td char=".">1.03 </td><td char=".">0.05 </td><td>Retained </td><td>I, S </td><td>I </td><td>Intron 2 </td><td>+ </td></tr><tr><td>AT3G13300 </td><td char=".">1.20 </td><td char=".">1.03 </td><td char=".">0.05 </td><td>Retained </td><td>I, S </td><td>ND </td><td>Intron 1 </td><td>+ </td></tr><tr><td>AT1G32230 </td><td char=".">1.25 </td><td char=".">1.05 </td><td char=".">0.05 </td><td>Intermediate </td><td>I, S </td><td>ND </td><td>Intron 4 </td><td>+ </td></tr><tr><td>AT1G76030 </td><td char=".">1.31 </td><td char=".">1.00 </td><td char=".">0.05 </td><td>Intermediate </td><td>S </td><td>I </td><td>Intron 5 </td><td>+ </td></tr><tr><td>AT1G48920 </td><td char=".">1.37 </td><td char=".">1.01 </td><td char=".">0.05 </td><td>Intermediate </td><td>S </td><td>I </td><td>Intron 1 </td><td>+ </td></tr><tr><td>AT1G53310 </td><td char=".">1.24 </td><td char=".">1.00 </td><td char=".">0.05 </td><td>Constitutive </td><td>S </td><td>Pre RNA </td><td>Intron 2 </td><td>+ </td></tr><tr><td>AT4G04910 </td><td char=".">1.16 </td><td char=".">0.99 </td><td char=".">0.05 </td><td>Constitutive </td><td>S </td><td>Pre RNA </td><td>Intron 11 </td><td>+ </td></tr><tr><td>AT2G23070 </td><td char=".">1.28 </td><td char=".">1.02 </td><td char=".">0.05 </td><td>Constitutive </td><td>S </td><td>ND </td><td>Intron 3 </td><td>+ </td></tr><tr><td>AT3G21215 </td><td char=".">1.25 </td><td char=".">1.02 </td><td char=".">0.05 </td><td>Constitutive </td><td>S </td><td>ND </td><td>Intron 3 </td><td>+ </td></tr><tr><td>AT5G20380 </td><td char=".">1.17 </td><td char=".">1.00 </td><td char=".">0.05 </td><td>Constitutive </td><td>I,S </td><td>Pre RNA, I </td><td>Intron 11 </td><td>− </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Chosen significance level value.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Intron types were selected from <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Flanking primers (Supplementary Table S6 is available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a>) yielded either ‘I’ which indicates fragments that includes the intron or ‘S’ which indicates fragment size indicative of a spliced transcript.</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>In the cases in which ‘I’ was not detected by using flanking primers, another PCR was done, using intron primers. ‘I’ indicates transcripts that include the retained intron but in which the other intron was spliced. ‘Pre-RNA’ indicates that the slower migrating fragment of the constitutive and retained intron was detected. ND, not detected.</p></div></div><div content-id=""><div class="fn"><p><sup>e</sup>Intron that was indicated in the bioinformatics analysis as retained and was detected by RT–PCR was marked as ‘+’. Intron that was supposed to be retained but was not detected was marked ‘−’.</p></div></div><div content-id=""><div class="fn"><p><sup>f</sup>Intron that was indicated in the bioinformatics analysis as constitutively spliced and was not detected in the PCR was marked as ‘+’. Intron that was supposed to be spliced out but was detected was marked ‘−’.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87193750" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87193751 id="87193751" class="section-title" >3.5. Feasibility of detecting ‘exon skip’ in whole-genome arrays</h3>
<p>In theory, a modification of the same algorithm should be applicable to estimating exon skip. However inspection of <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a> shows that the distribution of the introns mean hybridization value is much narrower than the exons mean hybridization value distribution. Thus, the lower hybridization values expected from potentially skipped exons would fall well within the normal exons distribution values, making the direct application of WGA difficult. The limitation of this application is exemplified when the same statistical methods used for detection of retained introns are applied to detect exon skip. Thus, to detect exon skip in a transcript, the probes of its introns were treated as one group, while each exon, which contained at least three probes, was treated as a different group. Exon skip of the first or last exons were excluded as these are probably due to 5′ and 3′ transcription start/finish differences and do not necessarily arise from splicing. Furthermore in that case, any data retrieved would be impossible to verify owing to the lack of flanking sequence for RT–PCR priming. The results of the search for exons skips were divided as for the introns. Examination of internal exons using the statistical tests and parameters as those applied for examination of the introns showed an exon skip level of more than 8% (<a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>). A sample of previously un-annotated AS events in <em>Arabidopsis</em> was experimentally tested by RT–PCR. Each PCR was done using RNA that was extracted from four different samples as carried out for intron retention analysis. However, in this case only a low rate verification was achieved, 3 out of 8 (37.5%; Supplementary Table S3 available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a>), Thus, owing to the difficulty of separating the exons distribution from the introns distribution in the available database, the detection of exon skip is unreliable.</p>                            <h3 scrollto-destination=87193753 id="87193753" class="section-title" >3.6. Comparison of the results for intron retention to EST-derived estimates</h3>
<p>Identification of alternative transcripts have relied on comparison of sequences from EST and cDNA databases, a method that is inherently different from the type of analysis carried out here, and, therefore, it is worthy to compare the results. The TAIR6 release contains 31 407 putative genes of which 3159 (10%) have annotated splice variants. Using TAIR annotation (<a class="link link-uri" href="ftp://ftp.arabidopsis.org/home/tair/Maps/seqviewer_data/SV_gene_feature.data" target="_blank"></a>) the type of each splice variant was determined. Out of the 3159 genes, 2251 can be analyzed in the WGA experiments and were composed of two transcripts that could be compared to one another. Of these 468 were shown to be of the retained-intron type, a rate that is considerably lower than the more than 40% that was previously observed.<a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="b7  b9" data-open="b7  b9" class="link link-ref link-reveal xref-bibr">7–9</a> In order to compare the efficiency of both methods the limitations applied to the WGA analysis were applied to the retained introns in the TAIR database. Of the 468 retained introns, 257 met the criteria employed to analyze the existing WGA data. Of these 45 or 18% were detected in common by the WGA method (Supplementary Table S4 is available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a>). These results indicate the overlapping but complementary nature of both approaches as a tool to determine AS.</p>                            <h3 scrollto-destination=87193755 id="87193755" class="section-title" >3.7. Distribution of biological functionalities of transcripts with retained introns</h3>
<p>Examining the assigned global functional activity of transcripts with alternatively spliced retained introns will contribute to understanding their function in the biology of the plant. Genes that are analyzed in this study are specific to flowers, root, cell culture or seedling and as a result their gene ontology will be different from the total genes ontology. Therefore, in our case, changes in gene ontology distribution were sought by comparing the retained introns group with the constitutive introns group analyzed in this work rather than to total genes ontology. As nearly 8% of the genes contain retained introns we expect a similar distribution to be emulated in the gene ontology. Deviations from the expected binomial distribution with the <em>P</em>-value set at 0.05 are summarized in <a reveal-id="tbl3" data-open="tbl3" class="link link-reveal link-table xref-fig">Table 3</a>. Strikingly, the groups that are enriched in the retained introns group include mostly RNA processing and signal transduction. In contrast, transcripts related to protein metabolism are particularly underrepresented in the intron retention group.</p>                        <a id="87193757" scrollto-destination="87193757"></a>
<div content-id="tbl3" class="table-modal"><div class="table-wrap"><div id="tbl3" class="table-wrap-title" data-id="tbl3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Distribution of assigned gene ontology in genes with retained introns and constitutive introns</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>GO </th><th>Annotation </th><th>Constitutive </th><th>Retained </th><th><em>P</em>-value </th></tr></thead><tbody><tr><td colspan="5">Over-representation </td></tr><tr><td>    GO:0000398 </td><td>Nuclear mRNA splicing, via spliceosome </td><td>11 </td><td>5 </td><td char=".">0.006 </td></tr><tr><td>    GO:0007165 </td><td>Signal transduction </td><td>33 </td><td>7 </td><td char=".">0.024 </td></tr><tr><td>    GO:0006468 </td><td>Protein amino acid phosphorylation </td><td>210 </td><td>28 </td><td char=".">0.010 </td></tr><tr><td colspan="5">Under-representation </td></tr><tr><td>    GO:0006412 </td><td>Protein biosynthesis </td><td>272 </td><td>13 </td><td char=".">0.008 </td></tr><tr><td>    GO:0005975 </td><td>Carbohydrate metabolism </td><td>134 </td><td>6 </td><td char=".">0.035 </td></tr></tbody></table></div><div content-id="tbl3" class="table-modal"><table><thead name="thead" align=""><tr><th>GO </th><th>Annotation </th><th>Constitutive </th><th>Retained </th><th><em>P</em>-value </th></tr></thead><tbody><tr><td colspan="5">Over-representation </td></tr><tr><td>    GO:0000398 </td><td>Nuclear mRNA splicing, via spliceosome </td><td>11 </td><td>5 </td><td char=".">0.006 </td></tr><tr><td>    GO:0007165 </td><td>Signal transduction </td><td>33 </td><td>7 </td><td char=".">0.024 </td></tr><tr><td>    GO:0006468 </td><td>Protein amino acid phosphorylation </td><td>210 </td><td>28 </td><td char=".">0.010 </td></tr><tr><td colspan="5">Under-representation </td></tr><tr><td>    GO:0006412 </td><td>Protein biosynthesis </td><td>272 </td><td>13 </td><td char=".">0.008 </td></tr><tr><td>    GO:0005975 </td><td>Carbohydrate metabolism </td><td>134 </td><td>6 </td><td char=".">0.035 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The transcripts with intron retention from the WGA database were classified by the biological processes of their protein products ontology in relation to the constitutive introns from this database. (<a class="link link-uri" href="ftp://ftp.arabidopsis.org/home/tair/Ontologies/Gene_Ontology/ATH_GO_GOSLIM.20060121.txt" target="_blank"></a><a reveal-id="b40" data-open="b40" class="link link-ref link-reveal xref-bibr">40</a> (Details in Supplementary Table S5 available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a>). Transcripts listed as ‘biological process unknown’ (30% of retained introns) were not included. Note that gene ontology may lead to the classification of one gene into multiple categories. Only classifications containing at least 5 retained introns or 64 constitutive transcripts were included (i.e. at least 0.9% of the expressed transcripts).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87193757" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl3" class="table-wrap-title" data-id="tbl3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Distribution of assigned gene ontology in genes with retained introns and constitutive introns</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>GO </th><th>Annotation </th><th>Constitutive </th><th>Retained </th><th><em>P</em>-value </th></tr></thead><tbody><tr><td colspan="5">Over-representation </td></tr><tr><td>    GO:0000398 </td><td>Nuclear mRNA splicing, via spliceosome </td><td>11 </td><td>5 </td><td char=".">0.006 </td></tr><tr><td>    GO:0007165 </td><td>Signal transduction </td><td>33 </td><td>7 </td><td char=".">0.024 </td></tr><tr><td>    GO:0006468 </td><td>Protein amino acid phosphorylation </td><td>210 </td><td>28 </td><td char=".">0.010 </td></tr><tr><td colspan="5">Under-representation </td></tr><tr><td>    GO:0006412 </td><td>Protein biosynthesis </td><td>272 </td><td>13 </td><td char=".">0.008 </td></tr><tr><td>    GO:0005975 </td><td>Carbohydrate metabolism </td><td>134 </td><td>6 </td><td char=".">0.035 </td></tr></tbody></table></div><div content-id="tbl3" class="table-modal"><table><thead name="thead" align=""><tr><th>GO </th><th>Annotation </th><th>Constitutive </th><th>Retained </th><th><em>P</em>-value </th></tr></thead><tbody><tr><td colspan="5">Over-representation </td></tr><tr><td>    GO:0000398 </td><td>Nuclear mRNA splicing, via spliceosome </td><td>11 </td><td>5 </td><td char=".">0.006 </td></tr><tr><td>    GO:0007165 </td><td>Signal transduction </td><td>33 </td><td>7 </td><td char=".">0.024 </td></tr><tr><td>    GO:0006468 </td><td>Protein amino acid phosphorylation </td><td>210 </td><td>28 </td><td char=".">0.010 </td></tr><tr><td colspan="5">Under-representation </td></tr><tr><td>    GO:0006412 </td><td>Protein biosynthesis </td><td>272 </td><td>13 </td><td char=".">0.008 </td></tr><tr><td>    GO:0005975 </td><td>Carbohydrate metabolism </td><td>134 </td><td>6 </td><td char=".">0.035 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The transcripts with intron retention from the WGA database were classified by the biological processes of their protein products ontology in relation to the constitutive introns from this database. (<a class="link link-uri" href="ftp://ftp.arabidopsis.org/home/tair/Ontologies/Gene_Ontology/ATH_GO_GOSLIM.20060121.txt" target="_blank"></a><a reveal-id="b40" data-open="b40" class="link link-ref link-reveal xref-bibr">40</a> (Details in Supplementary Table S5 available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a>). Transcripts listed as ‘biological process unknown’ (30% of retained introns) were not included. Note that gene ontology may lead to the classification of one gene into multiple categories. Only classifications containing at least 5 retained introns or 64 constitutive transcripts were included (i.e. at least 0.9% of the expressed transcripts).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87193757" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87193758 id="87193758" class="section-title" >3.8. Physical characterization of retained introns</h3>
<p>Differences in the physical characteristics of introns, such as size or flanking sequence information, may serve to characterize this class of AS. The characteristics that were measured include; position of the intron in the transcript, i.e. present in the UTR or CDS, whether it contains an open reading frame (ORF), the intron size, its GC content and borders of the intron. Interestingly as a group, the median size of the retained introns is significantly smaller when compared with the size of the constitutive introns irrespective of the presence of an ORF (117 and 176 bp, respectively). The size distributions are statistically different based on the goodness-of-fit hypothesis test (Two-sample Kolmogorov–Smirnov, <em>P</em> = 9.5 × 10<sup>−44</sup>).</p><p>We note that the median size of introns calculated from the TAIR database (<a class="link link-uri" href="ftp://ftp.arabidopsis.org/home/tair/Sequences/blast_datasets/TAIR6_intron_20060221" target="_blank"></a>) is 99 bp. The higher means in all types of introns measured here is a direct result of the requirement that introns be represented by at least three probes (i.e. 3 probes × 25 bp size/probe). Interestingly, the GC content of retained introns is higher than that of constitutive introns (37% compare to 33%) but lower than the GC content of coding regions (44%).<a reveal-id="b17" data-open="b17" class="link link-ref link-reveal xref-bibr">17</a> The two GC content distributions are statistically different based on the goodness-of-fit hypothesis test (Two-sample Kolmogorov–Smirnov <em>P</em> = 1.5 × 10<sup>−139</sup>; Supplementary Figure S1 is available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a>). Higher GC contents have been recently detected in retained introns that were identified by genome EST/cDNA alignments.<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> The possibility exists that the higher GC content reflects an artifact that is associated with enhanced non-specific hybridization of probes, i.e. they may be more sensitive to background noise. In order to assess this possibility, we examined the distribution of the GC content of the intron probes whose hybridization values were above or equal 1.09 (Supplementary Figure S2 is available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a>) In this case, the <em>R</em><sup>2</sup> for the correlation of GC content and hybridization value was found to be 0.06. Hence for the range of hybridization values used in our survey non-specific hybridization is not correlated to GC content and the values measured are a result of true specific hybridization.</p><p>Additional qualities of note in the characterization of retained introns are that, relatively more retained introns originate from the 5′- and 3′-UTR region compared with the constitutive intron distribution (13.4% compared with 7.3%; <a reveal-id="tbl4" data-open="tbl4" class="link link-reveal link-table xref-fig">Table 4</a>). By comparison, the global analysis predicts that at least 9% of <em>Arabidopsis</em> genes have introns in their UTRs.<a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> Strikingly, inspection of <a reveal-id="tbl4" data-open="tbl4" class="link link-reveal link-table xref-fig">Table 4</a> shows that 609 (86.6%) of the retained introns are inside the CDS. Yet of these, only 4.6% maintain an in-frame ORF (<a reveal-id="tbl4" data-open="tbl4" class="link link-reveal link-table xref-fig">Table 4</a>). Interestingly this rate, while low, is still more than 3-fold higher than that noted for constitutive introns (1.5%). Finally, Intron borders can play a crucial role in splicing efficiency.<a reveal-id="b20" data-open="b20" class="link link-ref link-reveal xref-bibr">20</a> However a comparison between the borders of the constitutive and retained introns showed no differences (Supplementary Figure S3 is available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a>). Thus, border sequence cannot be used to predict alternative intron splicing.</p>                        <a id="87193762" scrollto-destination="87193762"></a>
<div content-id="tbl4" class="table-modal"><div class="table-wrap"><div id="tbl4" class="table-wrap-title" data-id="tbl4"><div class="label -label"><strong>Table 4</strong></div><div class="caption"><p>Characterization of retained and constitutive introns</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Intron type </th><th>Position </th><th>ORF </th><th>Number of introns </th><th>% ORF </th><th>% position </th></tr></thead><tbody><tr><td>Constitutive </td><td>CDS </td><td>0 </td><td>21 310 </td><td>98.5 </td><td> </td></tr><tr><td>Constitutive </td><td>CDS </td><td>1 </td><td>325 </td><td>1.5 </td><td> </td></tr><tr><td>Summary </td><td> </td><td> </td><td>21 635 </td><td> </td><td>92.7 </td></tr><tr><td>    Constitutive </td><td>3′-UTR </td><td> </td><td>249 </td><td> </td><td>1.1 </td></tr><tr><td>    Constitutive </td><td>5′-UTR </td><td> </td><td>1445 </td><td> </td><td>6.2 </td></tr><tr><td>    Retained<sup>a</sup> </td><td>CDS </td><td>0 </td><td>581 </td><td>95.4 </td><td> </td></tr><tr><td>    Retained </td><td>CDS </td><td>1 </td><td>28 </td><td>4.6 </td><td> </td></tr><tr><td>Summary </td><td>CDS </td><td> </td><td>609 </td><td> </td><td>86.6 </td></tr><tr><td>    Retained </td><td>3′-UTR </td><td> </td><td>31 </td><td> </td><td>4.4 </td></tr><tr><td>    Retained </td><td>5′-UTR </td><td>0 </td><td>63 </td><td> </td><td>9 </td></tr></tbody></table></div><div content-id="tbl4" class="table-modal"><table><thead name="thead" align=""><tr><th>Intron type </th><th>Position </th><th>ORF </th><th>Number of introns </th><th>% ORF </th><th>% position </th></tr></thead><tbody><tr><td>Constitutive </td><td>CDS </td><td>0 </td><td>21 310 </td><td>98.5 </td><td> </td></tr><tr><td>Constitutive </td><td>CDS </td><td>1 </td><td>325 </td><td>1.5 </td><td> </td></tr><tr><td>Summary </td><td> </td><td> </td><td>21 635 </td><td> </td><td>92.7 </td></tr><tr><td>    Constitutive </td><td>3′-UTR </td><td> </td><td>249 </td><td> </td><td>1.1 </td></tr><tr><td>    Constitutive </td><td>5′-UTR </td><td> </td><td>1445 </td><td> </td><td>6.2 </td></tr><tr><td>    Retained<sup>a</sup> </td><td>CDS </td><td>0 </td><td>581 </td><td>95.4 </td><td> </td></tr><tr><td>    Retained </td><td>CDS </td><td>1 </td><td>28 </td><td>4.6 </td><td> </td></tr><tr><td>Summary </td><td>CDS </td><td> </td><td>609 </td><td> </td><td>86.6 </td></tr><tr><td>    Retained </td><td>3′-UTR </td><td> </td><td>31 </td><td> </td><td>4.4 </td></tr><tr><td>    Retained </td><td>5′-UTR </td><td>0 </td><td>63 </td><td> </td><td>9 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Retained introns are Class 1 and Class 2 retained introns.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87193762" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl4" class="table-wrap-title" data-id="tbl4"><div class="label -label"><strong>Table 4</strong></div><div class="caption"><p>Characterization of retained and constitutive introns</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Intron type </th><th>Position </th><th>ORF </th><th>Number of introns </th><th>% ORF </th><th>% position </th></tr></thead><tbody><tr><td>Constitutive </td><td>CDS </td><td>0 </td><td>21 310 </td><td>98.5 </td><td> </td></tr><tr><td>Constitutive </td><td>CDS </td><td>1 </td><td>325 </td><td>1.5 </td><td> </td></tr><tr><td>Summary </td><td> </td><td> </td><td>21 635 </td><td> </td><td>92.7 </td></tr><tr><td>    Constitutive </td><td>3′-UTR </td><td> </td><td>249 </td><td> </td><td>1.1 </td></tr><tr><td>    Constitutive </td><td>5′-UTR </td><td> </td><td>1445 </td><td> </td><td>6.2 </td></tr><tr><td>    Retained<sup>a</sup> </td><td>CDS </td><td>0 </td><td>581 </td><td>95.4 </td><td> </td></tr><tr><td>    Retained </td><td>CDS </td><td>1 </td><td>28 </td><td>4.6 </td><td> </td></tr><tr><td>Summary </td><td>CDS </td><td> </td><td>609 </td><td> </td><td>86.6 </td></tr><tr><td>    Retained </td><td>3′-UTR </td><td> </td><td>31 </td><td> </td><td>4.4 </td></tr><tr><td>    Retained </td><td>5′-UTR </td><td>0 </td><td>63 </td><td> </td><td>9 </td></tr></tbody></table></div><div content-id="tbl4" class="table-modal"><table><thead name="thead" align=""><tr><th>Intron type </th><th>Position </th><th>ORF </th><th>Number of introns </th><th>% ORF </th><th>% position </th></tr></thead><tbody><tr><td>Constitutive </td><td>CDS </td><td>0 </td><td>21 310 </td><td>98.5 </td><td> </td></tr><tr><td>Constitutive </td><td>CDS </td><td>1 </td><td>325 </td><td>1.5 </td><td> </td></tr><tr><td>Summary </td><td> </td><td> </td><td>21 635 </td><td> </td><td>92.7 </td></tr><tr><td>    Constitutive </td><td>3′-UTR </td><td> </td><td>249 </td><td> </td><td>1.1 </td></tr><tr><td>    Constitutive </td><td>5′-UTR </td><td> </td><td>1445 </td><td> </td><td>6.2 </td></tr><tr><td>    Retained<sup>a</sup> </td><td>CDS </td><td>0 </td><td>581 </td><td>95.4 </td><td> </td></tr><tr><td>    Retained </td><td>CDS </td><td>1 </td><td>28 </td><td>4.6 </td><td> </td></tr><tr><td>Summary </td><td>CDS </td><td> </td><td>609 </td><td> </td><td>86.6 </td></tr><tr><td>    Retained </td><td>3′-UTR </td><td> </td><td>31 </td><td> </td><td>4.4 </td></tr><tr><td>    Retained </td><td>5′-UTR </td><td>0 </td><td>63 </td><td> </td><td>9 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Retained introns are Class 1 and Class 2 retained introns.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87193762" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=87193763 id="87193763" class="section-title" >4. Discussion</h2>
<p>We demonstrate that direct transcript expression analysis using 'high-density oligonucleotide-based WGA is particularly amenable for assessing global intron retention in <em>Arabidopsis</em>. However, the detection of exon skip is unreliable owing to the difficulty in separating the distribution of the expression values that distinguish exons from introns. We note that detection of alternative 5′ and 3′ splicing sites by analysis of WGA would be limited to those alternative sites that comprise a number of probe lengths (in the present case a minimum of 75 bp). As alternative 5′ and 3′ splice choices are usually of shorter length, they would not generally be detected.</p><p>Genome-wide detection of retained introns in <em>Arabidopsis</em> reveals that at least 8% of genes that can be surveyed by microarray include a transcript with a retained intron. A sample of these transcripts was tested using RT–PCR and 86% could be verified. In our analysis, results of the WGA data were combined to enable the generation of statistically reliable outcomes. Thus, tissue-specific intron retention may be lost as combining RNA from diverse biological material can obscure the AS signal. In the future, the use of sample repeats from the same tissue will undoubtedly improve the statistical veracity of the results and optimize the detection of aberrant intron behavior. Nonetheless, as intron retention comprises above 40% of all AS events<a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="b7" data-open="b7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a> this indicates that overall the rate of AS in <em>Arabidopsis</em> is 20%. This rate is considerably higher than 10–14%, which was garnered from compilation of EST and cDNA data from different sources<a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="b7" data-open="b7" class="link link-ref link-reveal xref-bibr">7</a> but similar to the one that was recently reported from a large collection of EST/cDNA.<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> Similarly, the recent application of WGA to human chromosome 21 and 22 showed a much higher rate of AS than previously detected.<a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a></p><p>The retained-intron type is distinct from the other AS types because it represents an absence or failure of spliceosome action rather than alternative choice of splice junctions. Our analysis detects several features that are specific to retained introns. They are smaller, have higher GC content, are relatively enriched in the UTR and those in the CDS tend to have a slightly higher chance of containing an ORF when compared with a constitutive-type introns. The size of introns has been shown to play a role in AS biology. In <em>Drosophila</em>, the size and location of the flanking introns control the frequency and the type of AS that a pre-mRNA transcript undergoes.<a reveal-id="b21" data-open="b21" class="link link-ref link-reveal xref-bibr">21</a> Furthermore, it is possible that the attribute of higher GC content could contribute to a weaker U-rich splicing signal as a result of less efficient AU protein binding.<a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> This is because intronic U-rich or AU-rich elements can influence splice site selection and splicing efficiency particularly in plants (reviewed in Simpson and Filipowicz<a reveal-id="b23" data-open="b23" class="link link-ref link-reveal xref-bibr">23</a>). These sequences are likely to bind proteins (analogous to animal hnRNP proteins), which may permit intron recognition and recruit or allow access to spliceosomal components and the formation of commitment splicing complexes.</p><p>Of particular note is that 82.6% (581 out of 703) of the transcripts containing retained introns are in the CDS and would shift reading frames and have an early stop codon. Of these, in 68% (477 out of 703) of the cases the intron that is retained is not the last one. (data not shown). The position of these stop codons is important in the context of potential nonsense-mediated decay (NMD) effects. NMD serves as a surveillance mechanism that removes erroneous mRNAs containing a premature termination codon.<a reveal-id="b24" data-open="b24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="b25" data-open="b25" class="link link-ref link-reveal xref-bibr">25</a> Thus, if NMD mechanism exists in plants, its selective power of degradation is incomplete as the transcripts with AS events are readily detected. The existence of limited use of NMD in plants has recently been shown in the analysis of transcripts containing premature termination codon.<a reveal-id="b26" data-open="b26" class="link link-ref link-reveal xref-bibr">26</a> However, the coupling of AS and NMD could be an important post-transcriptional regulation mechanism to adjust the level of transcript isoforms.<a reveal-id="b27  b29" data-open="b27  b29" class="link link-ref link-reveal xref-bibr">27–29</a></p><p>Inspection of the intron retention database provides biological insight. RNA processing and signal transduction transcripts appear to be differentially enriched in the retained-introns group. A serine/arginine-rich (SR) family of proteins is implicated in constitutive and AS of pre-mRNAs. They appear in alternatively spliced forms<a reveal-id="b30" data-open="b30" class="link link-ref link-reveal xref-bibr">30</a>,<a reveal-id="b31" data-open="b31" class="link link-ref link-reveal xref-bibr">31</a> and at least one of them, SR1, contains a retained intron.<a reveal-id="b32" data-open="b32" class="link link-ref link-reveal xref-bibr">32</a> Transcription factors that show AS variants were described for five genes and in one of them, At1g26260, the first intron is retained.<a reveal-id="b33" data-open="b33" class="link link-ref link-reveal xref-bibr">33</a> Numerous studies have contributed to the view that SR proteins play a general role in splicing and can modulate splice site selection in a concentration-dependent manner (Ref. <a reveal-id="b34" data-open="b34" class="link link-ref link-reveal xref-bibr">34</a> and reviewed by Manley and Tacke<a reveal-id="b35" data-open="b35" class="link link-ref link-reveal xref-bibr">35</a>). One imaginable consequence of this is that cells may regulate the expression or activity of individual SR proteins, or their antagonists, to control the expression of one or more target genes in a tissue-specific and/or developmentally regulated fashion. Transcript that contains intron may serve as a negative regulator of the SR protein concentration.</p><p>The potential regulatory power of transcripts with retained introns can be gleaned from examination of the database (Supplementary Table S2 is available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a>). Two examples are NPR1 (At1g64280) and GIGANTEA (At1g22770). NPR1 is a key regulator of the salicylic acid (SA)-mediated systemic acquired resistance (SAR) pathway and it confers resistance to pathogens. Known mutations of this gene cause loss of SAR induction, loss of expression of PR genes and increased susceptibility to infections.<a reveal-id="b36" data-open="b36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="b37" data-open="b37" class="link link-ref link-reveal xref-bibr">37</a> WGA analysis predicts that the last intron of this gene is retained and contains an in-frame stop codon. GIGANTEA promotes flowering under long days in a circadian clock-controlled flowering pathway, together with CONSTANTS (CO) and FLOWERING LOCUS T (FT). Mutations in this gene cause late-flowering phenotype.<a reveal-id="b38" data-open="b38" class="link link-ref link-reveal xref-bibr">38</a> WGA analysis predicts intron 10 (out of 13) to be retained and to contain a stop codon.</p><p>The biological function of retained introns remains an enigma. A recent survey of human transcripts shows that one-third of the AS contained premature termination codons that are the apparent targets of NMD. It was proposed that regulated unproductive splicing may be a means of modifying the level of protein expression or of introducing modified gene products.<a reveal-id="b39" data-open="b39" class="link link-ref link-reveal xref-bibr">39</a> This work provides a novel database of retained introns and their genomic sites and lays a framework for future application of WGA for dynamic screening of changes in splicing during the plants' life and as a result of environmental input.</p><p>This research was supported by the Israel Science Foundation grant No 388/02 and by the BARD United States-Israel Binational Agricultural Research and Development Fund Grant number IS-3454-03.</p><p><strong>Supplementary Data:</strong> Supplementary data is available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a></p>                            <h2 scrollto-destination=87193774 id="87193774" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="b1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kan</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">States</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Selecting for functional alternative splices in ESTs</div>, <div class="source">Genome Res.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">1837</div>-<div class="lpage">1845</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selecting%20for%20functional%20alternative%20splices%20in%20ESTs&amp;author=Z.%20Kan&amp;author=D.%20States&amp;author=W.%20Gish&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=1837-1845" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.764102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.764102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.764102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12466287" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Carninci</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Kasukawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Katayama</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">The transcriptional landscape of the mammalian genome</div>, <div class="source">Science</div> , <div class="year">2005</div>, vol. <div class="volume">309</div> (pg. <div class="fpage">1559</div>-<div class="lpage">1563</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20transcriptional%20landscape%20of%20the%20mammalian%20genome&amp;author=P.%20Carninci&amp;author=T.%20Kasukawa&amp;author=S.%20Katayama&amp;publication_year=2005&amp;journal=Science&amp;volume=309&amp;pages=1559-1563" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1112014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1112014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1112014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16141072" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nagasaki</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Arita</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Nishizawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Suwa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Gotoh</div> <div class="given-names">O.</div></div>. </div><div class="article-title">Species-specific variation of alternative splicing and transcriptional initiation in six eukaryotes</div>, <div class="source">Gene</div> , <div class="year">2005</div>, vol. <div class="volume">364</div> (pg. <div class="fpage">53</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Species-specific%20variation%20of%20alternative%20splicing%20and%20transcriptional%20initiation%20in%20six%20eukaryotes&amp;author=H.%20Nagasaki&amp;author=M.%20Arita&amp;author=T.%20Nishizawa&amp;author=M.%20Suwa&amp;author=O.%20Gotoh&amp;publication_year=2005&amp;journal=Gene&amp;volume=364&amp;pages=53-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2005.07.027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2005.07.027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2005.07.027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16219431" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Modrek</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">C.</div></div>. </div><div class="article-title">A genomic view of alternative splicing</div>, <div class="source">Nat. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">13</div>-<div class="lpage">19</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genomic%20view%20of%20alternative%20splicing&amp;author=B.%20Modrek&amp;author=C.%20Lee&amp;publication_year=2002&amp;journal=Nat.%20Genet.&amp;volume=30&amp;pages=13-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng0102-13" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng0102-13" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng0102-13"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11753382" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Goodison</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yoshida</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Churchman</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tarin</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Multiple intron retention occurs in tumor cell CD44 mRNA processing</div>, <div class="source">Am. J. Pathol.</div> , <div class="year">1998</div>, vol. <div class="volume">153</div> (pg. <div class="fpage">1221</div>-<div class="lpage">1228</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20intron%20retention%20occurs%20in%20tumor%20cell%20CD44%20mRNA%20processing&amp;author=S.%20Goodison&amp;author=K.%20Yoshida&amp;author=M.%20Churchman&amp;author=D.%20Tarin&amp;publication_year=1998&amp;journal=Am.%20J.%20Pathol.&amp;volume=153&amp;pages=1221-1228" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0002-9440(10)65666-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0002-9440(10)65666-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0002-9440(10)65666-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9777953" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mansilla</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lopez-Sanchez</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">de la Rosa</div> <div class="given-names">E.J.</div></div>, et al. </div><div class="article-title">Developmental regulation of a proinsulin messenger RNA generated by intron retention</div>, <div class="source">EMBO Rep.</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">1182</div>-<div class="lpage">1187</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Developmental%20regulation%20of%20a%20proinsulin%20messenger%20RNA%20generated%20by%20intron%20retention&amp;author=A.%20Mansilla&amp;author=C.%20Lopez-Sanchez&amp;author=E.J.%20de%20la%20Rosa&amp;publication_year=2005&amp;journal=EMBO%20Rep.&amp;volume=6&amp;pages=1182-1187" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.embor.7400539" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.embor.7400539" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.embor.7400539"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16179943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iida</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Seki</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sakurai</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Genome-wide analysis of alternative pre-mRNA splicing in <em>Arabidopsis thaliana</em> based on full-length cDNA sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">5096</div>-<div class="lpage">5103</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20alternative%20pre-mRNA%20splicing%20in%20Arabidopsis%20thaliana%20based%20on%20full-length%20cDNA%20sequences&amp;author=K.%20Iida&amp;author=M.%20Seki&amp;author=T.%20Sakurai&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=5096-5103" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh845" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh845" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh845"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15452276" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">B.B.</div></div>,  <div class="name"><div class="surname">Brendel</div> <div class="given-names">V.</div></div>. </div><div class="article-title">Genomewide comparative analysis of alternative splicing in plants</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2006</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">7175</div>-<div class="lpage">7180</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomewide%20comparative%20analysis%20of%20alternative%20splicing%20in%20plants&amp;author=B.B.%20Wang&amp;author=V.%20Brendel&amp;publication_year=2006&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=7175-7180" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0602039103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0602039103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0602039103"> </span></div> </div></div></div></div></div><div content-id="b9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ner-Gaon</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Halachmi</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Savaldi-Goldstein</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Rubin</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Ophir</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Fluhr</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Intron retention is a major phenomenon in alternative splicing in <em>Arabidopsis</em></div>, <div class="source">Plant J.</div> , <div class="year">2004</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">877</div>-<div class="lpage">885</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intron%20retention%20is%20a%20major%20phenomenon%20in%20alternative%20splicing%20in%20Arabidopsis&amp;author=H.%20Ner-Gaon&amp;author=R.%20Halachmi&amp;author=S.%20Savaldi-Goldstein&amp;author=E.%20Rubin&amp;author=R.%20Ophir&amp;author=R.%20Fluhr&amp;publication_year=2004&amp;journal=Plant%20J.&amp;volume=39&amp;pages=877-885" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2004.39.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2004.39.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2004.39.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15341630" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Schaffer</div> <div class="given-names">A.A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSIBLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">3402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSIBLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-3402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Martienssen</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">Doerge</div> <div class="given-names">R.W.</div></div>,  <div class="name"><div class="surname">Colot</div> <div class="given-names">V.</div></div>. </div><div class="article-title">Epigenomic mapping in <em>Arabidopsis</em> using tiling microarrays</div>, <div class="source">Chromosome Res.</div> , <div class="year">2005</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">299</div>-<div class="lpage">308</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenomic%20mapping%20in%20Arabidopsis%20using%20tiling%20microarrays&amp;author=R.A.%20Martienssen&amp;author=R.W.%20Doerge&amp;author=V.%20Colot&amp;publication_year=2005&amp;journal=Chromosome%20Res.&amp;volume=13&amp;pages=299-308" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10577-005-1507-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10577-005-1507-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10577-005-1507-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15868423" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kapranov</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Cawley</div> <div class="given-names">S.E.</div></div>,  <div class="name"><div class="surname">Drenkow</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Large-scale transcriptional activity in chromosomes 21 and 22</div>, <div class="source">Science</div> , <div class="year">2002</div>, vol. <div class="volume">296</div> (pg. <div class="fpage">916</div>-<div class="lpage">919</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20transcriptional%20activity%20in%20chromosomes%2021%20and%2022&amp;author=P.%20Kapranov&amp;author=S.E.%20Cawley&amp;author=J.%20Drenkow&amp;publication_year=2002&amp;journal=Science&amp;volume=296&amp;pages=916-919" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1068597" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1068597" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1068597"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11988577" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kampa</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Cheng</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kapranov</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Novel RNAs identified from an in-depth analysis of the transcriptome of human chromosomes 21 and 22</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">331</div>-<div class="lpage">342</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20RNAs%20identified%20from%20an%20in-depth%20analysis%20of%20the%20transcriptome%20of%20human%20chromosomes%2021%20and%2022&amp;author=D.%20Kampa&amp;author=J.%20Cheng&amp;author=P.%20Kapranov&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=331-342" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2094104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2094104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2094104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14993201" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamada</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Lim</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Dale</div> <div class="given-names">J.M.</div></div>, et al. </div><div class="article-title">Empirical analysis of transcriptional activity in the <em>Arabidopsis</em> genome</div>, <div class="source">Science</div> , <div class="year">2003</div>, vol. <div class="volume">302</div> (pg. <div class="fpage">842</div>-<div class="lpage">846</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Empirical%20analysis%20of%20transcriptional%20activity%20in%20the%20Arabidopsis%20genome&amp;author=K.%20Yamada&amp;author=J.%20Lim&amp;author=J.M.%20Dale&amp;publication_year=2003&amp;journal=Science&amp;volume=302&amp;pages=842-846" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1088305" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1088305" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1088305"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14593172" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Benjamini</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hochberg</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing</div>, <div class="source">J. R. Stat. Soc. Ser.</div> , <div class="year">1995</div>, vol. <div class="volume">B57</div> (pg. <div class="fpage">289</div>-<div class="lpage">300</div>)</div></div></div></div><div content-id="b16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Savaldi-Goldstein</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Aviv</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Davydov</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Fluhr</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Alternative splicing modulation by a LAMMER kinase impinges on developmental and transcriptome expression</div>, <div class="source">Plant Cell</div> , <div class="year">2003</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">926</div>-<div class="lpage">938</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20modulation%20by%20a%20LAMMER%20kinase%20impinges%20on%20developmental%20and%20transcriptome%20expression&amp;author=S.%20Savaldi-Goldstein&amp;author=D.%20Aviv&amp;author=O.%20Davydov&amp;author=R.%20Fluhr&amp;publication_year=2003&amp;journal=Plant%20Cell&amp;volume=15&amp;pages=926-938" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.011056" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.011056" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.011056"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12671088" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="collab">Arabidopsis Genome Initiative.</div><div class="article-title">Analysis of the genome sequence of the flowering plant <em>Arabidopsis thaliana</em></div>, <div class="source">Nature</div> , <div class="year">2000</div>, vol. <div class="volume">408</div> (pg. <div class="fpage">796</div>-<div class="lpage">815</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35048692" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35048692" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35048692"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11130711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhu</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Schlueter</div> <div class="given-names">S.D.</div></div>,  <div class="name"><div class="surname">Brendel</div> <div class="given-names">V.</div></div>. </div><div class="article-title">Refined annotation of the <em>Arabidopsis</em> genome by complete expressed sequence tag mapping</div>, <div class="source">Plant Physiol.</div> , <div class="year">2003</div>, vol. <div class="volume">132</div> (pg. <div class="fpage">469</div>-<div class="lpage">484</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Refined%20annotation%20of%20the%20Arabidopsis%20genome%20by%20complete%20expressed%20sequence%20tag%20mapping&amp;author=W.%20Zhu&amp;author=S.D.%20Schlueter&amp;author=V.%20Brendel&amp;publication_year=2003&amp;journal=Plant%20Physiol.&amp;volume=132&amp;pages=469-484" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.102.018101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.102.018101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.102.018101"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12805580" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Haas</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Delcher</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Mount</div> <div class="given-names">S.M.</div></div>, et al. </div><div class="article-title">Improving the <em>Arabidopsis</em> genome annotation using maximal transcript alignment assemblies</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">5654</div>-<div class="lpage">5666</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improving%20the%20Arabidopsis%20genome%20annotation%20using%20maximal%20transcript%20alignment%20assemblies&amp;author=B.J.%20Haas&amp;author=A.L.%20Delcher&amp;author=S.M.%20Mount&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=5654-5666" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg770" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg770" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg770"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14500829" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rogozin</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Sverdlov</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Babenko</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Koonin</div> <div class="given-names">V.</div></div>. </div><div class="article-title">Analysis of evolution of exon–intron structure of eukaryotic genes</div>, <div class="source">Brief Bioinform.</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">118</div>-<div class="lpage">134</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20evolution%20of%20exon%E2%80%93intron%20structure%20of%20eukaryotic%20genes&amp;author=B.%20Rogozin&amp;author=V.%20Sverdlov&amp;author=N.%20Babenko&amp;author=V.%20Koonin&amp;publication_year=2005&amp;journal=Brief%20Bioinform.&amp;volume=6&amp;pages=118-134" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/6.2.118" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2F6.2.118" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbib%2F6.2.118"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15975222" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fox-Walsh</div> <div class="given-names">K.L.</div></div>,  <div class="name"><div class="surname">Dou</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Lam</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Hung</div> <div class="given-names">S.P.</div></div>,  <div class="name"><div class="surname">Baldi</div> <div class="given-names">P.F.</div></div>,  <div class="name"><div class="surname">Hertel</div> <div class="given-names">K.J.</div></div>. </div><div class="article-title">The architecture of pre-mRNAs affects mechanisms of splice-site pairing</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2005</div>, vol. <div class="volume">102</div> (pg. <div class="fpage">16176</div>-<div class="lpage">16181</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20architecture%20of%20pre-mRNAs%20affects%20mechanisms%20of%20splice-site%20pairing&amp;author=K.L.%20Fox-Walsh&amp;author=Y.%20Dou&amp;author=B.J.%20Lam&amp;author=S.P.%20Hung&amp;author=P.F.%20Baldi&amp;author=K.J.%20Hertel&amp;publication_year=2005&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=16176-16181" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0508489102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0508489102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0508489102"> </span></div> </div></div></div></div></div><div content-id="b22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brown</div> <div class="given-names">J.W.</div></div>. </div><div class="article-title"><em>Arabidopsis</em> intron mutations and pre-mRNA splicing</div>, <div class="source">Plant J.</div> , <div class="year">1996</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">771</div>-<div class="lpage">780</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20intron%20mutations%20and%20pre-mRNA%20splicing&amp;author=J.W.%20Brown&amp;publication_year=1996&amp;journal=Plant%20J.&amp;volume=10&amp;pages=771-780" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.1996.10050771.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.1996.10050771.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.1996.10050771.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8953241" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Simpson</div> <div class="given-names">G.G.</div></div>,  <div class="name"><div class="surname">Filipowicz</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Splicing of precursors to mRNA in higher plants: mechanism, regulation and sub-nuclear organisation of the spliceosomal machinery</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">1996</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">1</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Splicing%20of%20precursors%20to%20mRNA%20in%20higher%20plants%3A%20mechanism%2C%20regulation%20and%20sub-nuclear%20organisation%20of%20the%20spliceosomal%20machinery&amp;author=G.G.%20Simpson&amp;author=W.%20Filipowicz&amp;publication_year=1996&amp;journal=Plant%20Mol.%20Biol.&amp;volume=32&amp;pages=1-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00039375" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00039375" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00039375"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8980472" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nagy</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Maquat</div> <div class="given-names">L.E.</div></div>. </div><div class="article-title">A rule for termination-codon position within intron-containing genes: when nonsense affects RNA abundance</div>, <div class="source">Trends Biochem. Sci.</div> , <div class="year">1998</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">198</div>-<div class="lpage">199</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20rule%20for%20termination-codon%20position%20within%20intron-containing%20genes%3A%20when%20nonsense%20affects%20RNA%20abundance&amp;author=E.%20Nagy&amp;author=L.E.%20Maquat&amp;publication_year=1998&amp;journal=Trends%20Biochem.%20Sci.&amp;volume=23&amp;pages=198-199" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0968-0004(98)01208-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0968-0004(98)01208-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0968-0004(98)01208-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9644970" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baker</div> <div class="given-names">K.E.</div></div>,  <div class="name"><div class="surname">Parker</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Nonsense-mediated mRNA decay: terminating erroneous gene expression</div>, <div class="source">Curr. Opin. Cell Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">293</div>-<div class="lpage">299</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nonsense-mediated%20mRNA%20decay%3A%20terminating%20erroneous%20gene%20expression&amp;author=K.E.%20Baker&amp;author=R.%20Parker&amp;publication_year=2004&amp;journal=Curr.%20Opin.%20Cell%20Biol.&amp;volume=16&amp;pages=293-299" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ceb.2004.03.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ceb.2004.03.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ceb.2004.03.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15145354" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hori</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Watanabe</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">UPF3 suppresses aberrant spliced mRNA in Arabidopsis</div>, <div class="source">Plant J.</div> , <div class="year">2005</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">530</div>-<div class="lpage">540</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=UPF3%20suppresses%20aberrant%20spliced%20mRNA%20in%20Arabidopsis&amp;author=K.%20Hori&amp;author=Y.%20Watanabe&amp;publication_year=2005&amp;journal=Plant%20J.&amp;volume=43&amp;pages=530-540" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2005.43.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2005.43.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2005.43.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16098107" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lejeune</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Maquat</div> <div class="given-names">L.E.</div></div>. </div><div class="article-title">Mechanistic links between nonsense-mediated mRNA decay and pre-mRNA splicing in mammalian cells</div>, <div class="source">Curr. Opin. Cell Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">309</div>-<div class="lpage">315</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanistic%20links%20between%20nonsense-mediated%20mRNA%20decay%20and%20pre-mRNA%20splicing%20in%20mammalian%20cells&amp;author=F.%20Lejeune&amp;author=L.E.%20Maquat&amp;publication_year=2005&amp;journal=Curr.%20Opin.%20Cell%20Biol.&amp;volume=17&amp;pages=309-315" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ceb.2005.03.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ceb.2005.03.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ceb.2005.03.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15901502" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lareau</div> <div class="given-names">L.F.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">R.E.</div></div>,  <div class="name"><div class="surname">Bhatnagar</div> <div class="given-names">R.S.</div></div>,  <div class="name"><div class="surname">Brenner</div> <div class="given-names">S.E.</div></div>. </div><div class="article-title">The evolving roles of alternative splicing</div>, <div class="source">Curr. Opin. Struct. Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">273</div>-<div class="lpage">282</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolving%20roles%20of%20alternative%20splicing&amp;author=L.F.%20Lareau&amp;author=R.E.%20Green&amp;author=R.S.%20Bhatnagar&amp;author=S.E.%20Brenner&amp;publication_year=2004&amp;journal=Curr.%20Opin.%20Struct.%20Biol.&amp;volume=14&amp;pages=273-282" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.sbi.2004.05.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.sbi.2004.05.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.sbi.2004.05.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15193306" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yoine</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Nishii</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">K.</div></div>. </div><div class="article-title"><em>Arabidopsis</em> UPF1 RNA helicase for nonsense-mediated mRNA decay is involved in seed size control and is essential for growth</div>, <div class="source">Plant Cell. Physiol.</div> , <div class="year">2006</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">572</div>-<div class="lpage">580</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20UPF1%20RNA%20helicase%20for%20nonsense-mediated%20mRNA%20decay%20is%20involved%20in%20seed%20size%20control%20and%20is%20essential%20for%20growth&amp;author=M.%20Yoine&amp;author=T.%20Nishii&amp;author=K.%20Nakamura&amp;publication_year=2006&amp;journal=Plant%20Cell.%20Physiol.&amp;volume=47&amp;pages=572-580" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcj035" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcj035" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcj035"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16540482" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Reddy</div> <div class="given-names">A.S.</div></div>. </div><div class="article-title">Plant serine/arginine-rich proteins and their role in pre-mRNA splicing</div>, <div class="source">Trends Plant Sci.</div> , <div class="year">2004</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">541</div>-<div class="lpage">547</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20serine%2Farginine-rich%20proteins%20and%20their%20role%20in%20pre-mRNA%20splicing&amp;author=A.S.%20Reddy&amp;publication_year=2004&amp;journal=Trends%20Plant%20Sci.&amp;volume=9&amp;pages=541-547" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2004.09.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2004.09.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2004.09.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15501179" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iida</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Go</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Survey of conserved alternative splicing events of mRNAs encoding SR proteins in land plants</div>, <div class="source">Mol Biol Evol.</div> , <div class="year">2006</div> <div class="comment">[Please update reference 31 if now it has been published.]</div></div></div></div></div><div content-id="b32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lazar</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Goodman</div> <div class="given-names">H.M.</div></div>. </div><div class="article-title">The <em>Arabidopsis</em> splicing factor SR1 is regulated by alternative splicing</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">2000</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">571</div>-<div class="lpage">581</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Arabidopsis%20splicing%20factor%20SR1%20is%20regulated%20by%20alternative%20splicing&amp;author=G.%20Lazar&amp;author=H.M.%20Goodman&amp;publication_year=2000&amp;journal=Plant%20Mol.%20Biol.&amp;volume=42&amp;pages=571-581" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1006394207479" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1006394207479" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1006394207479"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10809003" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gong</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Shen</div> <div class="given-names">Y.P.</div></div>,  <div class="name"><div class="surname">Ma</div> <div class="given-names">L.G.</div></div>, et al. </div><div class="article-title">Genome-wide ORFeome cloning and analysis of <em>Arabidopsis</em> transcription factor genes</div>, <div class="source">Plant Physiol.</div> , <div class="year">2004</div>, vol. <div class="volume">135</div> (pg. <div class="fpage">773</div>-<div class="lpage">782</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20ORFeome%20cloning%20and%20analysis%20of%20Arabidopsis%20transcription%20factor%20genes&amp;author=W.%20Gong&amp;author=Y.P.%20Shen&amp;author=L.G.%20Ma&amp;publication_year=2004&amp;journal=Plant%20Physiol.&amp;volume=135&amp;pages=773-782" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.104.042176" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.104.042176" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.104.042176"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15208423" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jumaa</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Nielsen</div> <div class="given-names">P.J.</div></div>. </div><div class="article-title">The splicing factor SRp20 modifies splicing of its own mRNA and ASF/SF2 antagonizes this regulation</div>, <div class="source">EMBO J.</div> , <div class="year">1997</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">5077</div>-<div class="lpage">5085</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20splicing%20factor%20SRp20%20modifies%20splicing%20of%20its%20own%20mRNA%20and%20ASF%2FSF2%20antagonizes%20this%20regulation&amp;author=H.%20Jumaa&amp;author=P.J.%20Nielsen&amp;publication_year=1997&amp;journal=EMBO%20J.&amp;volume=16&amp;pages=5077-5085" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/emboj/16.16.5077" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Femboj%2F16.16.5077" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Femboj%2F16.16.5077"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9305649" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Manley</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Tacke</div> <div class="given-names">R.</div></div>. </div><div class="article-title">SR proteins and splicing control</div>, <div class="source">Genes Dev.</div> , <div class="year">1996</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">1569</div>-<div class="lpage">1579</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SR%20proteins%20and%20splicing%20control&amp;author=J.L.%20Manley&amp;author=R.%20Tacke&amp;publication_year=1996&amp;journal=Genes%20Dev.&amp;volume=10&amp;pages=1569-1579" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.10.13.1569" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.10.13.1569" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.10.13.1569"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8682289" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cao</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Glazebrook</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Clarke</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Volko</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Dong</div> <div class="given-names">X.</div></div>. </div><div class="article-title">The <em>Arabidopsis</em> NPR1 gene that controls systemic acquired resistance encodes a novel protein containing ankyrin repeats</div>, <div class="source">Cell</div> , <div class="year">1997</div>, vol. <div class="volume">88</div> (pg. <div class="fpage">57</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Arabidopsis%20NPR1%20gene%20that%20controls%20systemic%20acquired%20resistance%20encodes%20a%20novel%20protein%20containing%20ankyrin%20repeats&amp;author=H.%20Cao&amp;author=J.%20Glazebrook&amp;author=J.D.%20Clarke&amp;author=S.%20Volko&amp;author=X.%20Dong&amp;publication_year=1997&amp;journal=Cell&amp;volume=88&amp;pages=57-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(00)81858-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(00)81858-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(00)81858-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9019406" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ryals</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Weymann</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Lawton</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">The <em>Arabidopsis</em> NIM1 protein shows homology to the mammalian transcription factor inhibitor I kappa B</div>, <div class="source">Plant Cell</div> , <div class="year">1997</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">425</div>-<div class="lpage">439</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Arabidopsis%20NIM1%20protein%20shows%20homology%20to%20the%20mammalian%20transcription%20factor%20inhibitor%20I%20kappa%20B&amp;author=J.%20Ryals&amp;author=K.%20Weymann&amp;author=K.%20Lawton&amp;publication_year=1997&amp;journal=Plant%20Cell&amp;volume=9&amp;pages=425-439" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9090885" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eimert</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Lue</div> <div class="given-names">W.I.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Monogenic recessive mutations causing both late floral initiation and excess starch accumulation in <em>Arabidopsis</em></div>, <div class="source">Plant Cell</div> , <div class="year">1995</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">1703</div>-<div class="lpage">1712</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Monogenic%20recessive%20mutations%20causing%20both%20late%20floral%20initiation%20and%20excess%20starch%20accumulation%20in%20Arabidopsis&amp;author=K.%20Eimert&amp;author=S.M.%20Wang&amp;author=W.I.%20Lue&amp;author=J.%20Chen&amp;publication_year=1995&amp;journal=Plant%20Cell&amp;volume=7&amp;pages=1703-1712" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.7.10.1703" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.7.10.1703" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.7.10.1703"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12242359" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lewis</div> <div class="given-names">B.P.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">R.E.</div></div>,  <div class="name"><div class="surname">Brenner</div> <div class="given-names">S.E.</div></div>. </div><div class="article-title">Evidence for the widespread coupling of alternative splicing and nonsense-mediated mRNA decay in humans</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2003</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">189</div>-<div class="lpage">192</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20the%20widespread%20coupling%20of%20alternative%20splicing%20and%20nonsense-mediated%20mRNA%20decay%20in%20humans&amp;author=B.P.%20Lewis&amp;author=R.E.%20Green&amp;author=S.E.%20Brenner&amp;publication_year=2003&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=100&amp;pages=189-192" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0136770100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0136770100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0136770100"> </span></div> </div></div></div></div></div><div content-id="b40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Berardini</div> <div class="given-names">T.Z.</div></div>,  <div class="name"><div class="surname">Mundodi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Reiser</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Functional annotation of the <em>Arabidopsis</em> genome using controlled vocabularies</div>, <div class="source">Plant Physiol.</div> , <div class="year">2004</div>, vol. <div class="volume">135</div> (pg. <div class="fpage">745</div>-<div class="lpage">755</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20annotation%20of%20the%20Arabidopsis%20genome%20using%20controlled%20vocabularies&amp;author=T.Z.%20Berardini&amp;author=S.%20Mundodi&amp;author=L.%20Reiser&amp;publication_year=2004&amp;journal=Plant%20Physiol.&amp;volume=135&amp;pages=745-755" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.104.040071" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.104.040071" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.104.040071"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15173566" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2006. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=347374&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/3/10.1093_dnares_dsl003/1/dsl003_Supplementary_Data.zip?Expires=2147483647&amp;Signature=wGVFk5kWDyr13jtPKnXNS6KEDktRWh3UCArwKM9CJXOpXjFu37yYiojk2nI1HEy5wTyR9PD9coTfOkVE5OG3-a5EOnRkBItmBaywJSB3ErE1-~RUR5Ug6Q1Tv5ApeKcjoFtmzVGioElFmduY8K6z3LVQWaKsMWuZ0mJg-KuUpU~g2yP5mXRD6Q9ItSwtkV3RY4YatOGJkKKxJTMUp93rMdtLor1WIfLuA3z4K7nFOi-ozwJ8A59wl7EM-MPmy4hIwM1yT68xWahFrO1v~cLb0DcA5jtVv2g4JX-9MRaG3X-J7t4n-G6Pawc-NmNbzw8ngpTXn6E7cK3148jAaAKpkA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">156</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000241913100003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">35</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsl003"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="347374"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000241913100003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f13%2f3%2f111%2f347374">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000241913100003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (35)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F13%2F3%2F111%2F347374" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/347374'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Whole-Genome Microarray in Arabidopsis Facilitates Global Analysis of Retained Introns" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsl003");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f13%2f3%2f111%2f347374">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

