

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/5/477/5035914","@type":"ScholarlyArticle","name":"Optimization of single strand DNA incorporation reaction by Moloney murine leukaemia virus reverse transcriptase","datePublished":"2018-06-11","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2018-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsy018","keywords":["adaptor ligation","template-independent DNA addition","DNA polymerase","reverse transcriptase"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/5/477/5035914","author":[{"name":"Ohtsubo, Yoshiyuki","affiliation":"Department of Molecular and Chemical Life Sciences, Graduate School of Life Sciences, Tohoku University, Sendai, Japan","@type":"Person"},{"name":"Sasaki, Haruna","affiliation":"Department of Molecular and Chemical Life Sciences, Graduate School of Life Sciences, Tohoku University, Sendai, Japan","@type":"Person"},{"name":"Nagata, Yuji","affiliation":"Department of Molecular and Chemical Life Sciences, Graduate School of Life Sciences, Tohoku University, Sendai, Japan","@type":"Person"},{"name":"Tsuda, Masataka","affiliation":"Department of Molecular and Chemical Life Sciences, Graduate School of Life Sciences, Tohoku University, Sendai, Japan","@type":"Person"}],"description":"Abstract.  In this study, we investigated CIS reaction (clamping-mediated incorporation of single-stranded DNA with concomitant DNA syntheses) of Moloney murine","pageStart":"477","pageEnd":"487","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Optimization of single strand DNA incorporation reaction by Moloney murine leukaemia virus reverse transcriptase","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Optimization of single strand DNA incorporation reaction by Moloney murine leukaemia virus reverse transcriptase" />
    <meta property="og:description" content="Abstract.  In this study, we investigated CIS reaction (clamping-mediated incorporation of single-stranded DNA with concomitant DNA syntheses) of Moloney murine" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/5/477/5035914" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2018-06-11" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Ohtsubo, Yoshiyuki" /><meta name="citation_author_institution" content="Department of Molecular and Chemical Life Sciences, Graduate School of Life Sciences, Tohoku University, Sendai, Japan" /><meta name="citation_author" content="Sasaki, Haruna" /><meta name="citation_author_institution" content="Department of Molecular and Chemical Life Sciences, Graduate School of Life Sciences, Tohoku University, Sendai, Japan" /><meta name="citation_author" content="Nagata, Yuji" /><meta name="citation_author_institution" content="Department of Molecular and Chemical Life Sciences, Graduate School of Life Sciences, Tohoku University, Sendai, Japan" /><meta name="citation_author" content="Tsuda, Masataka" /><meta name="citation_author_institution" content="Department of Molecular and Chemical Life Sciences, Graduate School of Life Sciences, Tohoku University, Sendai, Japan" /><meta name="citation_title" content="Optimization of single strand DNA incorporation reaction by Moloney murine leukaemia virus reverse transcriptase" /><meta name="citation_firstpage" content="477" /><meta name="citation_lastpage" content="487" /><meta name="citation_doi" content="10.1093/dnares/dsy018" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2018/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_publisher=Cold Spring Harbor Laboratory, Cold Spring Harbor, NY, USA;  citation_title=Molecular Cloning: A Laboratory Manual;  citation_year=1982; " /><meta name="citation_reference" content="citation_title=Nucleotide sequence of Moloney murine leukaemia virus;  citation_journal_title=Nature;  citation_year=1981;  citation_volume=293;  citation_pages=543-8; " /><meta name="citation_reference" content="citation_title=Reverse transcriptases can clamp together nucleic acids strands with two complementary bases at their 3′-termini for initiating DNA synthesis;  citation_journal_title=Nucleic Acids Res;  citation_year=2011;  citation_volume=39;  citation_pages=1042-53; " /><meta name="citation_reference" content="citation_title=Substrate variations that affect the nucleic acid clamp activity of reverse transcriptases;  citation_journal_title=FEBS J;  citation_year=2012;  citation_volume=279;  citation_pages=1894-903; " /><meta name="citation_reference" content="citation_title=Strand selections resulting from the combined template-independent DNA synthesis and clamp activities of HIV-1 reverse transcriptase;  citation_journal_title=Biochem. Biophys. Res. Commun;  citation_year=2011;  citation_volume=408;  citation_pages=482-8; " /><meta name="citation_reference" content="citation_publisher=Eaton Publishing, Natick, MA, USA;  citation_title=Gene Cloning and Analysis by RT-PCR;  citation_year=1998;  citation_pages=305-19; " /><meta name="citation_reference" content="citation_title=Reverse transcriptase template switching: a SMART approach for full-length cDNA library construction;  citation_journal_title=BioTechniques;  citation_year=2001;  citation_volume=30;  citation_pages=892-7; " /><meta name="citation_reference" content="citation_title=Incorporation of non-natural nucleotides into template-switching oligonucleotides reduces background and improves cDNA synthesis from very small RNA samples;  citation_journal_title=BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=413" /><meta name="citation_reference" content="citation_title=Efficient N-tailing of blunt DNA ends by Moloney murine leukemia virus reverse transcriptase;  citation_journal_title=Sci. Rep;  citation_year=2010;  citation_volume=11;  citation_pages=41769" /><meta name="citation_reference" content="citation_title=Compounds that enhance the tailing activity of Moloney murine leukemia virus reverse transcriptase;  citation_journal_title=Sci. Rep;  citation_year=2017;  citation_volume=7;  citation_pages=6520" /><meta name="citation_reference" content="citation_title=Universal cloning method by TA strategy;  citation_journal_title=BioTechniques;  citation_year=1995;  citation_volume=19;  citation_pages=34-5; " /><meta name="citation_reference" content="citation_title=Comparison of RNA- or LNA-hybrid oligonucleotides in template-switching reactions for high-speed sequencing library preparation;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=665" /><meta name="citation_reference" content="citation_title=CapSelect: a highly sensitive method for 5′ CAP-dependent enrichment of full-length cDNA in PCR-mediated analysis of mRNAs;  citation_journal_title=Nucleic Acids Res;  citation_year=1999;  citation_volume=27;  citation_pages=31e." /><meta name="citation_reference" content="citation_title=Determination of the site of first strand transfer during Moloney murine leukemia virus reverse transcription and identification of strand transfer-associated reverse transcriptase errors;  citation_journal_title=EMBO J;  citation_year=1997;  citation_volume=16;  citation_pages=856-65; " /><meta name="citation_reference" content="citation_title=Extension of mismatched 3′ termini of DNA is a major determinant of the infidelity of human immunodeficiency virus type 1 reverse transcriptase;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=1989;  citation_volume=86;  citation_pages=8343-7; " /><meta name="citation_reference" content="citation_title=Quantitation of next generation sequencing library preparation protocol efficiencies using droplet digital PCR assays - a systematic comparison of DNA library preparation kits for Illumina sequencing;  citation_journal_title=BMC Genomics;  citation_year=2016;  citation_volume=17;  citation_pages=458" /><meta name="citation_reference" content="citation_title=PacBio sequencing and its applications;  citation_journal_title=Genomics Proteomics Bioinformatics;  citation_year=2015;  citation_volume=13;  citation_pages=278-89; " /><meta name="citation_reference" content="citation_title=Accurate whole human genome sequencing using reversible terminator chemistry;  citation_journal_title=Nature;  citation_year=2008;  citation_volume=456;  citation_pages=53-9; " /><meta name="citation_reference" content="citation_title=A complete bacterial genome assembled de novo using only nanopore sequencing data;  citation_journal_title=Nat. Methods;  citation_year=2015;  citation_volume=12;  citation_pages=733-5; " /><meta name="citation_reference" content="citation_title=Transposon sequencing: methods and expanding applications;  citation_journal_title=Appl. Microbiol. Biotechnol;  citation_year=2016;  citation_volume=100;  citation_pages=31-43; " /><meta name="citation_reference" content="citation_title=Practical innovations for high-throughput amplicon sequencing;  citation_journal_title=Nat. Methods;  citation_year=2013;  citation_volume=10;  citation_pages=999-1002; " /><meta name="citation_reference" content="citation_title=Massively systematic transcript end readout, “MASTER”: transcription start site selection, transcriptional slippage, and transcript yields;  citation_journal_title=Mol. Cell;  citation_year=2015;  citation_volume=60;  citation_pages=953-65; " /><meta name="citation_reference" content="citation_title=Je, a versatile suite to handle multiplexed NGS libraries with unique molecular identifiers;  citation_journal_title=BMC Bioinformatics;  citation_year=2016;  citation_volume=17;  citation_pages=419" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/5/477/26077021/dsy018.pdf" /><meta name="description" content="Abstract.  In this study, we investigated CIS reaction (clamping-mediated incorporation of single-stranded DNA with concomitant DNA syntheses) of Moloney murine" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/5/477/5035914" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/5/477/5035914" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/5/477/5035914");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753&authnips=128.235.12.133&doi=10.1093/dnares/dsy018';
    </script>
<title>
	Optimization of single strand DNA incorporation reaction by Moloney murine leukaemia virus reverse transcriptase | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./5035914" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bbo/PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPaHRzdWJvLCBZb3NoaXl1a2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1vbGVjdWxhciBhbmQgQ2hlbWljYWwgTGlmZSBTY2llbmNlcywgR3JhZHVhdGUgU2Nob29sIG9mIExpZmUgU2NpZW5jZXMsIFRvaG9rdSBVbml2ZXJzaXR5LCBTZW5kYWksIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2FzYWtpLCBIYXJ1bmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1vbGVjdWxhciBhbmQgQ2hlbWljYWwgTGlmZSBTY2llbmNlcywgR3JhZHVhdGUgU2Nob29sIG9mIExpZmUgU2NpZW5jZXMsIFRvaG9rdSBVbml2ZXJzaXR5LCBTZW5kYWksIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFnYXRhLCBZdWppIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNb2xlY3VsYXIgYW5kIENoZW1pY2FsIExpZmUgU2NpZW5jZXMsIEdyYWR1YXRlIFNjaG9vbCBvZiBMaWZlIFNjaWVuY2VzLCBUb2hva3UgVW5pdmVyc2l0eSwgU2VuZGFpLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRzdWRhLCBNYXNhdGFrYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTW9sZWN1bGFyIGFuZCBDaGVtaWNhbCBMaWZlIFNjaWVuY2VzLCBHcmFkdWF0ZSBTY2hvb2wgb2YgTGlmZSBTY2llbmNlcywgVG9ob2t1IFVuaXZlcnNpdHksIFNlbmRhaSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ik9wdGltaXphdGlvbiBvZiBzaW5nbGUgc3RyYW5kIEROQSBpbmNvcnBvcmF0aW9uIHJlYWN0aW9uIGJ5IE1vbG9uZXkgbXVyaW5lIGxldWthZW1pYSB2aXJ1cyByZXZlcnNlIHRyYW5zY3JpcHRhc2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI0NzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjQ4NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzeTAxOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE4LzEwLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUNvbGQgU3ByaW5nIEhhcmJvciBMYWJvcmF0b3J5LCBDb2xkIFNwcmluZyBIYXJib3IsIE5ZLCBVU0E7ICBjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgQ2xvbmluZzogQSBMYWJvcmF0b3J5IE1hbnVhbDsgIGNpdGF0aW9uX3llYXI9MTk4MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TnVjbGVvdGlkZSBzZXF1ZW5jZSBvZiBNb2xvbmV5IG11cmluZSBsZXVrYWVtaWEgdmlydXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MTk4MTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTM7ICBjaXRhdGlvbl9wYWdlcz01NDMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmV2ZXJzZSB0cmFuc2NyaXB0YXNlcyBjYW4gY2xhbXAgdG9nZXRoZXIgbnVjbGVpYyBhY2lkcyBzdHJhbmRzIHdpdGggdHdvIGNvbXBsZW1lbnRhcnkgYmFzZXMgYXQgdGhlaXIgM+KAsi10ZXJtaW5pIGZvciBpbml0aWF0aW5nIEROQSBzeW50aGVzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9MTA0Mi01MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3Vic3RyYXRlIHZhcmlhdGlvbnMgdGhhdCBhZmZlY3QgdGhlIG51Y2xlaWMgYWNpZCBjbGFtcCBhY3Rpdml0eSBvZiByZXZlcnNlIHRyYW5zY3JpcHRhc2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRUJTIEo7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Mjc5OyAgY2l0YXRpb25fcGFnZXM9MTg5NC05MDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cmFuZCBzZWxlY3Rpb25zIHJlc3VsdGluZyBmcm9tIHRoZSBjb21iaW5lZCB0ZW1wbGF0ZS1pbmRlcGVuZGVudCBETkEgc3ludGhlc2lzIGFuZCBjbGFtcCBhY3Rpdml0aWVzIG9mIEhJVi0xIHJldmVyc2UgdHJhbnNjcmlwdGFzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvY2hlbS4gQmlvcGh5cy4gUmVzLiBDb21tdW47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDA4OyAgY2l0YXRpb25fcGFnZXM9NDgyLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1FYXRvbiBQdWJsaXNoaW5nLCBOYXRpY2ssIE1BLCBVU0E7ICBjaXRhdGlvbl90aXRsZT1HZW5lIENsb25pbmcgYW5kIEFuYWx5c2lzIGJ5IFJULVBDUjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3BhZ2VzPTMwNS0xOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmV2ZXJzZSB0cmFuc2NyaXB0YXNlIHRlbXBsYXRlIHN3aXRjaGluZzogYSBTTUFSVCBhcHByb2FjaCBmb3IgZnVsbC1sZW5ndGggY0ROQSBsaWJyYXJ5IGNvbnN0cnVjdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvVGVjaG5pcXVlczsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTg5Mi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmNvcnBvcmF0aW9uIG9mIG5vbi1uYXR1cmFsIG51Y2xlb3RpZGVzIGludG8gdGVtcGxhdGUtc3dpdGNoaW5nIG9saWdvbnVjbGVvdGlkZXMgcmVkdWNlcyBiYWNrZ3JvdW5kIGFuZCBpbXByb3ZlcyBjRE5BIHN5bnRoZXNpcyBmcm9tIHZlcnkgc21hbGwgUk5BIHNhbXBsZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTQxMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmljaWVudCBOLXRhaWxpbmcgb2YgYmx1bnQgRE5BIGVuZHMgYnkgTW9sb25leSBtdXJpbmUgbGV1a2VtaWEgdmlydXMgcmV2ZXJzZSB0cmFuc2NyaXB0YXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2kuIFJlcDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTQxNzY5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcG91bmRzIHRoYXQgZW5oYW5jZSB0aGUgdGFpbGluZyBhY3Rpdml0eSBvZiBNb2xvbmV5IG11cmluZSBsZXVrZW1pYSB2aXJ1cyByZXZlcnNlIHRyYW5zY3JpcHRhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gUmVwOyAgY2l0YXRpb25feWVhcj0yMDE3OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz02NTIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VW5pdmVyc2FsIGNsb25pbmcgbWV0aG9kIGJ5IFRBIHN0cmF0ZWd5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9UZWNobmlxdWVzOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MzQtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyaXNvbiBvZiBSTkEtIG9yIExOQS1oeWJyaWQgb2xpZ29udWNsZW90aWRlcyBpbiB0ZW1wbGF0ZS1zd2l0Y2hpbmcgcmVhY3Rpb25zIGZvciBoaWdoLXNwZWVkIHNlcXVlbmNpbmcgbGlicmFyeSBwcmVwYXJhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9NjY1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2FwU2VsZWN0OiBhIGhpZ2hseSBzZW5zaXRpdmUgbWV0aG9kIGZvciA14oCyIENBUC1kZXBlbmRlbnQgZW5yaWNobWVudCBvZiBmdWxsLWxlbmd0aCBjRE5BIGluIFBDUi1tZWRpYXRlZCBhbmFseXNpcyBvZiBtUk5BczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0zMWUuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV0ZXJtaW5hdGlvbiBvZiB0aGUgc2l0ZSBvZiBmaXJzdCBzdHJhbmQgdHJhbnNmZXIgZHVyaW5nIE1vbG9uZXkgbXVyaW5lIGxldWtlbWlhIHZpcnVzIHJldmVyc2UgdHJhbnNjcmlwdGlvbiBhbmQgaWRlbnRpZmljYXRpb24gb2Ygc3RyYW5kIHRyYW5zZmVyLWFzc29jaWF0ZWQgcmV2ZXJzZSB0cmFuc2NyaXB0YXNlIGVycm9yczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RU1CTyBKOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9ODU2LTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHRlbnNpb24gb2YgbWlzbWF0Y2hlZCAz4oCyIHRlcm1pbmkgb2YgRE5BIGlzIGEgbWFqb3IgZGV0ZXJtaW5hbnQgb2YgdGhlIGluZmlkZWxpdHkgb2YgaHVtYW4gaW1tdW5vZGVmaWNpZW5jeSB2aXJ1cyB0eXBlIDEgcmV2ZXJzZSB0cmFuc2NyaXB0YXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUuIFMuIEE7ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICBjaXRhdGlvbl92b2x1bWU9ODY7ICBjaXRhdGlvbl9wYWdlcz04MzQzLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVF1YW50aXRhdGlvbiBvZiBuZXh0IGdlbmVyYXRpb24gc2VxdWVuY2luZyBsaWJyYXJ5IHByZXBhcmF0aW9uIHByb3RvY29sIGVmZmljaWVuY2llcyB1c2luZyBkcm9wbGV0IGRpZ2l0YWwgUENSIGFzc2F5cyAtIGEgc3lzdGVtYXRpYyBjb21wYXJpc29uIG9mIEROQSBsaWJyYXJ5IHByZXBhcmF0aW9uIGtpdHMgZm9yIElsbHVtaW5hIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTQ1OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBhY0JpbyBzZXF1ZW5jaW5nIGFuZCBpdHMgYXBwbGljYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWljcyBQcm90ZW9taWNzIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9Mjc4LTg5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY2N1cmF0ZSB3aG9sZSBodW1hbiBnZW5vbWUgc2VxdWVuY2luZyB1c2luZyByZXZlcnNpYmxlIHRlcm1pbmF0b3IgY2hlbWlzdHJ5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NDU2OyAgY2l0YXRpb25fcGFnZXM9NTMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb21wbGV0ZSBiYWN0ZXJpYWwgZ2Vub21lIGFzc2VtYmxlZCBkZSBub3ZvIHVzaW5nIG9ubHkgbmFub3BvcmUgc2VxdWVuY2luZyBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz03MzMtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNwb3NvbiBzZXF1ZW5jaW5nOiBtZXRob2RzIGFuZCBleHBhbmRpbmcgYXBwbGljYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcHBsLiBNaWNyb2Jpb2wuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTAwOyAgY2l0YXRpb25fcGFnZXM9MzEtNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByYWN0aWNhbCBpbm5vdmF0aW9ucyBmb3IgaGlnaC10aHJvdWdocHV0IGFtcGxpY29uIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTk5OS0xMDAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXNzaXZlbHkgc3lzdGVtYXRpYyB0cmFuc2NyaXB0IGVuZCByZWFkb3V0LCDigJxNQVNURVLigJ06IHRyYW5zY3JpcHRpb24gc3RhcnQgc2l0ZSBzZWxlY3Rpb24sIHRyYW5zY3JpcHRpb25hbCBzbGlwcGFnZSwgYW5kIHRyYW5zY3JpcHQgeWllbGRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NjA7ICBjaXRhdGlvbl9wYWdlcz05NTMtNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUplLCBhIHZlcnNhdGlsZSBzdWl0ZSB0byBoYW5kbGUgbXVsdGlwbGV4ZWQgTkdTIGxpYnJhcmllcyB3aXRoIHVuaXF1ZSBtb2xlY3VsYXIgaWRlbnRpZmllcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTQxOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNS81LzQ3Ny8yNjA3NzAyMS9kc3kwMTgucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEluIHRoaXMgc3R1ZHksIHdlIGludmVzdGlnYXRlZCBDSVMgcmVhY3Rpb24gKGNsYW1waW5nLW1lZGlhdGVkIGluY29ycG9yYXRpb24gb2Ygc2luZ2xlLXN0cmFuZGVkIEROQSB3aXRoIGNvbmNvbWl0YW50IEROQSBzeW50aGVzZXMpIG9mIE1vbG9uZXkgbXVyaW5lIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj01MDM1OTE0JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZO8/mOmKk7LaP04ih0JAB2yOm0EN" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f5%2f477%2f5035914" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f5%2f477%2f5035914" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=110919" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/5">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="122671367">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122671367">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122671369">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122671369">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122671376">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122671376">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122671390">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122671390">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122671422">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122671422">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122671439">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122671439">5. Conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122671441">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122671441">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122671447">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122671447">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/5/465/5033008">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/5/489/5036360">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Optimization of single strand DNA incorporation reaction by Moloney murine leukaemia virus reverse transcriptase
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshiyuki Ohtsubo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Molecular and Chemical Life Sciences, Graduate School of Life Sciences, Tohoku University, Sendai, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy018-cor1" data-open="dsy018-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel./Fax. +81 22 217 5696. Email: <a href="mailto:yohtsubo@ige.tohoku.ac.jp" target="_blank">yohtsubo@ige.tohoku.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshiyuki+Ohtsubo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohtsubo Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohtsubo Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshiyuki Ohtsubo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Haruna Sasaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Molecular and Chemical Life Sciences, Graduate School of Life Sciences, Tohoku University, Sendai, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Haruna+Sasaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sasaki H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sasaki H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Haruna Sasaki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuji Nagata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Molecular and Chemical Life Sciences, Graduate School of Life Sciences, Tohoku University, Sendai, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuji+Nagata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nagata Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nagata Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yuji Nagata</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masataka Tsuda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Molecular and Chemical Life Sciences, Graduate School of Life Sciences, Tohoku University, Sendai, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masataka+Tsuda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tsuda M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tsuda M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masataka Tsuda</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 5, 1 October 2018, Pages 477–487, <a href='https://doi.org/10.1093/dnares/dsy018'>https://doi.org/10.1093/dnares/dsy018</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">11 June 2018</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">29 January 2018</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">10 May 2018</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5035914" href="/dnaresearch/article-pdf/25/5/477/26077021/dsy018.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yoshiyuki Ohtsubo, Haruna Sasaki, Yuji Nagata, Masataka Tsuda;  Optimization of single strand DNA incorporation reaction by Moloney murine leukaemia virus reverse transcriptase, <em>DNA Research</em>, Volume 25, Issue 5, 1 October 2018, Pages 477–487, <a href='https://doi.org/10.1093/dnares/dsy018'>https://doi.org/10.1093/dnares/dsy018</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/5035914?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/5035914?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/5035914?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/5035914?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/5035914?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/5035914?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/5035914?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Optimization%20of%20single%20strand%20DNA%20incorporation%20reaction%20by%20Moloney%20murine%20leukaemia%20virus%20reverse%20transcriptase&amp;publicationDate=2018-06-11&amp;volumeNum=25&amp;issueNum=5&amp;author=Ohtsubo%2C%20Yoshiyuki%3B%20Sasaki%2C%20Haruna&amp;startPage=477&amp;endPage=487&amp;contentId=10.1093%2Fdnares%2Fdsy018&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f5%2f477%2f5035914" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=122671367 id="122671367" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>In this study, we investigated CIS reaction (<span class="underline">c</span>lamping-mediated <span class="underline">i</span>ncorporation of single-stranded DNA with concomitant DNA <span class="underline">s</span>yntheses) of Moloney murine leukaemia virus reverse transcriptase (MMLV-RT), and established a set of conditions with which single-stranded DNA is ligated to a G-tailed model substrate DNA at efficiencies close to 100%. Prior to the CIS reaction, a target blunt-end DNA was 3′ G-tailed by MMLV-RT in the presence of a tailing enhancer, deoxycytidine. In the CIS reaction, the G-tail reacted with a single-stranded DNA carrying a stretch of Cs on its 3′ end (termed as GAO for <span class="underline">g</span>uide <span class="underline">a</span>daptor <span class="underline">o</span>ligonucleotide), and MMLV-RT performed DNA polymerization, starting from the 3′ overhang, using the GAO as a template. We could append a given nucleotide sequence of as long as 72 nucleotides, which would be sufficient for various NGS-sequencing platforms. The high efficiency and the unique features of this MMLV-RT activity that enables the labelling of each DNA molecule with a unique degenerate sequence as a molecular identifier has many potential uses in biotechnology.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;adaptor ligation&quot;">adaptor ligation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;template-independent DNA addition&quot;">template-independent DNA addition</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;DNA polymerase&quot;">DNA polymerase</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;reverse transcriptase&quot;">reverse transcriptase</a></div>                            <h2 scrollto-destination=122671369 id="122671369" class="section-title" >1. Introduction</h2>
<p>Various fundamental DNA-handling techniques using the enzymatic activities of restriction enzymes, ligases, kinases, phosphatases, and DNA polymerases have been developed and combined for different biotechnological purposes.<a reveal-id="dsy018-B1" data-open="dsy018-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> The addition of an enzymatic activity that catalyses a novel reaction would increase the applicability of the currently available genetic tools and establish a base on which a number of new methodologies could be founded.</p><p>Reverse transcriptases (RT) from different origins such as human immunodeficiency virus Type-1 and Moloney murine leukaemia virus (MMLV),<a reveal-id="dsy018-B2" data-open="dsy018-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> has been shown to have clamping activity, which stabilizes a short duplex of dinucleotide at the 3′ tail of a double-stranded DNA and the 3′ end of a single-stranded DNA.<a reveal-id="dsy018-B3 dsy018-B4 dsy018-B5" data-open="dsy018-B3 dsy018-B4 dsy018-B5" class="link link-ref link-reveal xref-bibr"><sup>3–5</sup></a> The clamping activity enables RT-mediated DNA synthesis of the complementary strand of the single-stranded DNA, as well as the strand displacement synthesis originating from the single-stranded DNA.<a reveal-id="dsy018-B3" data-open="dsy018-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> However, the efficiency of the overall reaction (hereafter CIS reaction; <span class="underline">c</span>lamping-mediated <span class="underline">i</span>ncorporation of single-stranded DNA with concomitant DNA <span class="underline">s</span>yntheses) have been low, leaving a high fraction of DNA unreacted,<a reveal-id="dsy018-B3" data-open="dsy018-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> and thereby leaving its potential as a biotechnological tool unrealized.</p><p>MMLV-RT has been shown to catalyse a range of unique reactions; one such being the ‘template switching’ reaction that has been utilized for the efficient cloning of cDNA.<a reveal-id="dsy018-B6" data-open="dsy018-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsy018-B7" data-open="dsy018-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> The template switching occurs when the reverse transcribing MMLV-RT reaches the 5′ end of the RNA where it preferentially adds, in a template-independent fashion, a few dCMPs to the 3′ end of the newly synthesized cDNA. Then, in the presence of a single-stranded DNA (ssDNA) with a stretch of guanosine monophosphates (GMPs), referred to as a template switching oligo (TSO), MMLV-RT switches the template from RNA to the TSO and continues to incorporate deoxyribonucleotides that are complementary to the TSO, thereby extending the cDNA strand. Upon reaching the TSO end, MMLV-RT appends another 3′ tail consisting of dCMPs, to which another TSO (carrying 3′ GMPs) is incorporated, resulting in a concatenated end.<a reveal-id="dsy018-B8" data-open="dsy018-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a></p><p>MMLV-RT also has unusually strong tailing activity. In a previous study, we had optimized the tailing reaction conditions and shown that MMLV-RT appends to blunt double-stranded DNA ends with a 3′ tail of A, C, G, or T residues in a template-independent fashion.<a reveal-id="dsy018-B9" data-open="dsy018-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> We also identified specific compounds that enhance C-, G-, and T-tailing reactions, thus enabling the appending of a tail composed of ∼3 Cs and Ts and 4–5 Gs.<a reveal-id="dsy018-B10" data-open="dsy018-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> For the A-tailing reaction, specific enhancers have not been found but MMLV-RT inherently appends up to ∼4 As. Importantly, most of the DNA molecules in the reaction mixture are tailed, apparently leaving no DNA substrates unreacted.</p><p>It has not been addressed whether the template switching reaction of MMLV-RT involves a dynamic process, in which MMLV-RT molecule that has just finished reverse transcription (or DNA-dependent DNA polymerization) with its specific protein dynamics exerts the tailing and concomitant switching reaction. In addition, it has not been clear whether the performing of the tailing reaction by an MMLV-RT molecule is prerequisite for it to conduct subsequent template switching. The low efficiency of the reported CIS-reaction,<a reveal-id="dsy018-B3" data-open="dsy018-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> might be due to the lack of those dynamic processes, or to lack of stable interaction between the tail and incoming single-stranded DNA molecule (hereafter GAO; guide adaptor oligonucleotide), or to the intrinsic nature of MMLV-RT that can efficiently conduct CIS reaction on RNA-DNA hybrid end but not on double-stranded DNA end.</p><p>As we could extend the tail lengths,<a reveal-id="dsy018-B9" data-open="dsy018-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsy018-B10" data-open="dsy018-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> we tested, in this study, the possibility that the extended tail increase the CIS reaction efficiency. Since this was the case, to establish a fundamental DNA manipulation technique to append a DNA of given nucleotide sequence to a blunt-end DNA, we sought for the ideal CIS reaction conditions including the best combination of the 3′ tail and GAO nucleotides, and investigated whether longer tails result in higher efficiency, and investigated the length of GAO that can be efficiently incorporated. The efficiency of the CIS reaction using a model DNA substrate reached almost 100% suggesting it to be useful for different biotechnological applications.</p>                            <h2 scrollto-destination=122671376 id="122671376" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=122671377 id="122671377" class="section-title" >2.1. Tailing reactions</h3>
<p>The tailing reactions were conducted based on our previous reports.<a reveal-id="dsy018-B9" data-open="dsy018-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsy018-B10" data-open="dsy018-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> Wild-type MMLV-RT (200 U/µl) was purchased from Nippon Gene (Japan). The reaction mixture contained, in a total volume of 10 µl, 100 fmols to 1 pmol substrate DNA; 50 mM Tris–HCl, pH 8.3; 75 mM KCl; 6 mM MgCl<sub>2</sub>; 2 mM DTT; 4 mM dATP, dCTP, dGTP, or dTTP; 4 mM MnCl<sub>2</sub>; and 50 U MMLV-RT. Reactions were carried out in PCR tubes using a thermal cycler (C1000; Bio-Rad, USA) at 30°C. To enhance tailing, 200 mM deoxycytidine (for G-tailing), 40% saturated concentrations of deoxyguanosine (∼5 mM; for C-tailing), or deoxyadenosine (∼20 mM; for T-tailing) was added. We always added MnCl<sub>2</sub> just before adding the MMLV-RT and used fresh DTT. The tailing reaction was conducted at 30°C for 5–90 min and terminated by phenol/chloroform/isoamylalcohol extraction or by adding a salt solution for the subsequent purification of DNA using a DNA-binding column.</p>                            <h3 scrollto-destination=122671379 id="122671379" class="section-title" >2.2. CIS reaction</h3>
<p>The CIS reaction mixture contained, in a total volume of 10 µl, 100 fmol tailed DNA substrate, 0.25 mM each of dNTPs, 2 pmol GAO, 50 units of MMLV-RT, 50 mM Tris–HCl pH 8.3, 75 mM KCl, 6 mM MgCl<sub>2</sub>, and 2 mM DTT. The solutions for the DNA substrate and GAO were mixed and kept at room temperature (25°C) and added to the rest of the reaction mixture kept at the same temperature. After it was mixed by tapping the tube, the reaction mixture was incubated at 37°C.</p>                            <h3 scrollto-destination=122671381 id="122671381" class="section-title" >2.3. Reagents</h3>
<p>Streptavidin was purchased from Nacalai Tesuque (Japan) and dissolved in PBS buffer. T4 DNA polymerase was purchased from Takara (Japan).</p>                            <h3 scrollto-destination=122671383 id="122671383" class="section-title" >2.4. Preparation of FAM70 and 33G0 to 33G4</h3>
<p>FAM70 DNA was prepared as described previously.<a reveal-id="dsy018-B9" data-open="dsy018-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Briefly, a 5′-FAM-labelled primer was used to amplify a 300-bp fragment, and the fragment was digested by PvuII to generate a 70-bp blunt-end fragment bearing FAM at one 5′ end; the fragment was then purified by 15% polyacrylamide gel electrophoresis. The opposite 3′ end of the FAM-labelled strand was the tailing target, and the 5′ end adjacent to the tailing target carried a phosphate group.</p><p>Five types of oligonucleotides (SA606 and SA677-SA680, see <a reveal-id="dsy018-F3" data-open="dsy018-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>) bearing FAM at the 5′ end were annealed with SA659, purified as described earlier, and designated as 33G0 to 33G4. The concentrations of FAM70 and 33G0 to 33G4 were determined by measuring FAM fluorescence with an Infinite 200 Fluorescence Spectrophotometer (Tecan, Switzerland) using a FAM-labelled oligonucleotide (SA560) as a standard.</p>                            <h3 scrollto-destination=122671386 id="122671386" class="section-title" >2.5. DNA length analysis using a capillary sequencer and data analysis</h3>
<p>The length analysis was performed as described previously.<a reveal-id="dsy018-B9" data-open="dsy018-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsy018-B10" data-open="dsy018-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> To 1 ml of HiDi formamide, 10 µl of GeneScan-500 LIZ Size Standard (Thermo Fisher Scientific, USA), which contains 16 fragments of known sizes, was added to prepare HiDi-LIZ500. To 12.5 µl of HiDi-LIZ500, 0.5 µl of a reaction mixture was added, or when purified DNA samples were analysed, DNA samples were diluted with water, and DNA not exceeding 0.5 fmol was added. The samples were heat denatured at 96°C for 1 min and analysed with a 3130xl Genetic Analyzer (Thermo Fisher Scientific, USA) with a 50-cm capillary array and POP7 polymer. The data obtained were analysed using TraceViewer software,<a reveal-id="dsy018-B9" data-open="dsy018-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> and two LIZ bands were chosen to calibrate the electropherogram. Peak areas were determined using TraceViewer software, and GAO incorporation rates were calculated by dividing sum of peak area of CIS reaction products by total peak area. When peaks corresponding to concatenated products were observed, those peaks were also counted as CIS reaction products.</p>                            <h3 scrollto-destination=122671388 id="122671388" class="section-title" >2.6. Preparation of a size control</h3>
<p>FAM70 DNA and SA682 (5′-GACGTGTGCTCTTCCGATCT<span class="underline">CCCCCC</span><strong>CTGTCTCTTATACACATCTGACGC</strong>-3′; nucleotides complementary to the FAM70 end are in bold type, and intervening Cs are underlined) were mixed and annealed by heating and cooling and then extended by ExTaq DNA polymerase (TAKARA, Japan) for 10 min at 50°C. The product was used as a template for PCR amplification using the primers SA560 (FAM-AATGATACGGCGACCACCGAGATCTACAC-3′) and SA574 (5′-GACGTGTGCTCTTCCGATCTCCCCCC-3′) to amplify a 96-bp DNA fragment, which served as a size control.</p>                            <h2 scrollto-destination=122671390 id="122671390" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=122671391 id="122671391" class="section-title" >3.1. Schematic representation of CIS reaction</h3>
<p>In this study, we explored the optimal CIS reaction conditions, and the highest efficiency close to 100% was observed when extensively G-tailed dsDNA was reacted with a GAO carrying a 3′ stretch of Cs, which is contrasted to the template switching reaction (see above). <a reveal-id="dsy018-F1" data-open="dsy018-F1" class="link link-reveal link-table xref-fig">Figure 1a</a> shows an example of the CIS reaction that was conducted with the optimized protocol, and <a reveal-id="dsy018-F1" data-open="dsy018-F1" class="link link-reveal link-table xref-fig">Fig. 1b</a> shows a schematic representation of the reaction, with the horizontal scale applicable to both panels. In this example, 70-bp blunt-end DNA, carrying the FAM fluorophore on one of its 5′ ends (FAM70), was subjected to G-tailing, resulting in a DNA product that mostly carried a tail containing four Gs (see the major peak denoted as ‘FAM70 + 4G’ with two minor accompanying peaks of +3G and +5G). In the CIS reaction, a GAO annealed with the 4G-tail, and MMLV-RT catalysed DNA polymerization to synthesize the complementary strand of the GAO. Upon reaching the GAO end, MMLV-RT appended a 3′ tail through its tailing activity, resulting in a 3′-overhang in a template-independent fashion (see the two peaks marked with ‘+1’ and ‘+2’ that are 1 and 2 nucleotides larger than the size control, respectively). To the newly appended tail, another GAO was incorporated, resulting in a concatenated end (marked with *). In the reaction, the 3′ end of the GAO was used for strand-displacing synthesis. </p>                        <a id="122671393" scrollto-destination="122671393"></a>
<div id="dsy018-F1" data-id="dsy018-F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/m_dsy018f1.png?Expires=2147483647&Signature=2IxoDdyrPC0zs7pcznB-0yin1VeG5Khk5HcU-VBcnPebesbrhl8xsNBjopY5t~boVEK66co3HYGzDlynp36XhW2zE4TmYYvx85RhRvmxDtBdqxXqFFY0RNXGvC1uIWRmZqJL~~JusLqMu37IKZf6NzRNVQN7TKWDDGYv8rSRQWMe02XG6gjeEJBNvADIV0BGwQjCBomZnrCG7H2Zx9nczfTXGiQPMRbXwnf3XFQt7DYSN-mgWQK2nvVQB7WrplJswi2XuoqecTl8itdAk0be5Kf1HDgNYPR7R0IBLCip-u-lLHF1BXJhzkS5cBuqjhRtAmelTr8olqVk8L7Bdi7HCw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="An example of CIS reaction and schematic interpretation. (a) Electropherograms obtained by analysing the samples by a capillary sequencer are shown; x-axis represents the retention time and y-axis represents the fluorescence signal intensity. FAM70 DNA was G-tailed, and reacted with GAO SA574, and the product was further subjected to blunting by KOD DNA polymerase. The size control was prepared by PCR (See materials and methods). (b) Schematic interpretation of the CIS reaction in panel (a). Asterisks indicate FAM label and ‘N’ indicate a nucleotide. (i) Before reaction, (ii) GAO annealing to the 3′ tail consisted of Gs, (iii) extension from the 3′ end of the G-tail by MMLV-RT, (iv) template-independent addition of nucleotides, and strand-displacing synthesis by the MMLV-RT, and (v) concatemer formation. The two panels share the same x-axis." path-from-xml="dsy018f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671393/dsy018f1.tif" path-from-xml="dsy018f1.tif" target="_blank">View large</a><a section="122671393" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/dsy018f1.png?Expires=2147483647&Signature=mRn3IDhDx-fjje0SKEeWN916aEIxOxbDwqVgjayP3Dtml8rB3hgOsw5jwbMeztaHtlOyS3Tbtet5dx7OXEjJZVsptqOVQRaDruuFD~UiW~SWKANRVYbMQZuZSP9OziF7~yQvUtp9pP~0uQS4qQLD9EJ~OdLZKQY9hQveCyxQ2Tx7fXjoQu1j3y1ovJKJCjHNRn~EFPAhXuMkuMG7vwFHUJwgegpjtKl~6NYaYIBgrLB~~y5hXbIRgScdxwvjjoPbb5X6EjOS61ZjqihFuuhe~--bjNhnt18vPqSpywsCV8Z7rQr1k6XEC0XNTgaZcgqRHedud3dpWvSYA6s9te~AzA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671393&ar=5035914&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy018f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>An example of CIS reaction and schematic interpretation. (a) Electropherograms obtained by analysing the samples by a capillary sequencer are shown; <em>x</em>-axis represents the retention time and <em>y</em>-axis represents the fluorescence signal intensity. FAM70 DNA was G-tailed, and reacted with GAO SA574, and the product was further subjected to blunting by KOD DNA polymerase. The size control was prepared by PCR (See materials and methods). (b) Schematic interpretation of the CIS reaction in panel (a). Asterisks indicate FAM label and ‘N’ indicate a nucleotide. (i) Before reaction, (ii) GAO annealing to the 3′ tail consisted of Gs, (iii) extension from the 3′ end of the G-tail by MMLV-RT, (iv) template-independent addition of nucleotides, and strand-displacing synthesis by the MMLV-RT, and (v) concatemer formation. The two panels share the same <em>x</em>-axis.</p></div></div><div content-id="dsy018-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/m_dsy018f1.png?Expires=2147483647&Signature=2IxoDdyrPC0zs7pcznB-0yin1VeG5Khk5HcU-VBcnPebesbrhl8xsNBjopY5t~boVEK66co3HYGzDlynp36XhW2zE4TmYYvx85RhRvmxDtBdqxXqFFY0RNXGvC1uIWRmZqJL~~JusLqMu37IKZf6NzRNVQN7TKWDDGYv8rSRQWMe02XG6gjeEJBNvADIV0BGwQjCBomZnrCG7H2Zx9nczfTXGiQPMRbXwnf3XFQt7DYSN-mgWQK2nvVQB7WrplJswi2XuoqecTl8itdAk0be5Kf1HDgNYPR7R0IBLCip-u-lLHF1BXJhzkS5cBuqjhRtAmelTr8olqVk8L7Bdi7HCw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="An example of CIS reaction and schematic interpretation. (a) Electropherograms obtained by analysing the samples by a capillary sequencer are shown; x-axis represents the retention time and y-axis represents the fluorescence signal intensity. FAM70 DNA was G-tailed, and reacted with GAO SA574, and the product was further subjected to blunting by KOD DNA polymerase. The size control was prepared by PCR (See materials and methods). (b) Schematic interpretation of the CIS reaction in panel (a). Asterisks indicate FAM label and ‘N’ indicate a nucleotide. (i) Before reaction, (ii) GAO annealing to the 3′ tail consisted of Gs, (iii) extension from the 3′ end of the G-tail by MMLV-RT, (iv) template-independent addition of nucleotides, and strand-displacing synthesis by the MMLV-RT, and (v) concatemer formation. The two panels share the same x-axis." path-from-xml="dsy018f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671393/dsy018f1.tif" path-from-xml="dsy018f1.tif" target="_blank">View large</a><a section="122671393" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/dsy018f1.png?Expires=2147483647&Signature=mRn3IDhDx-fjje0SKEeWN916aEIxOxbDwqVgjayP3Dtml8rB3hgOsw5jwbMeztaHtlOyS3Tbtet5dx7OXEjJZVsptqOVQRaDruuFD~UiW~SWKANRVYbMQZuZSP9OziF7~yQvUtp9pP~0uQS4qQLD9EJ~OdLZKQY9hQveCyxQ2Tx7fXjoQu1j3y1ovJKJCjHNRn~EFPAhXuMkuMG7vwFHUJwgegpjtKl~6NYaYIBgrLB~~y5hXbIRgScdxwvjjoPbb5X6EjOS61ZjqihFuuhe~--bjNhnt18vPqSpywsCV8Z7rQr1k6XEC0XNTgaZcgqRHedud3dpWvSYA6s9te~AzA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671393&ar=5035914&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy018f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>An example of CIS reaction and schematic interpretation. (a) Electropherograms obtained by analysing the samples by a capillary sequencer are shown; <em>x</em>-axis represents the retention time and <em>y</em>-axis represents the fluorescence signal intensity. FAM70 DNA was G-tailed, and reacted with GAO SA574, and the product was further subjected to blunting by KOD DNA polymerase. The size control was prepared by PCR (See materials and methods). (b) Schematic interpretation of the CIS reaction in panel (a). Asterisks indicate FAM label and ‘N’ indicate a nucleotide. (i) Before reaction, (ii) GAO annealing to the 3′ tail consisted of Gs, (iii) extension from the 3′ end of the G-tail by MMLV-RT, (iv) template-independent addition of nucleotides, and strand-displacing synthesis by the MMLV-RT, and (v) concatemer formation. The two panels share the same <em>x</em>-axis.</p></div></div><p>After blunting of the product, the peaks shifted to the left, and the size of the largest peak matched that of the size control, demonstrating that a tail 1–2 nucleotides long had formed (see discussion for the other accompanying peaks). Notably, most of the DNA ends were appended with the GAO, as indicated by a tiny peak at the position of FAM70 (see the products of the blunting reaction in panel A).</p>                            <h3 scrollto-destination=122671395 id="122671395" class="section-title" >3.2. CIS reaction conditions</h3>
<p>The CIS reaction was initiated by mixing two solutions at room temperature (25°C); one contained pre-tailed 100 fmol of DNA substrate and 2 pmol GAO and the other contained the rest of the reaction components, including MMLV-RT, dNTPs, and DTT. After mixing, the samples were transferred to 37°C and incubated. This protocol gave satisfactory results (see below) and was used as a standard protocol throughout this study.</p>                            <h3 scrollto-destination=122671397 id="122671397" class="section-title" >3.3. G- and C-tailed DNA ends are good CIS reaction substrates</h3>
<p>To find the best combination of nucleotides for dsDNA 3′ tail and GAO 3′ end, we evaluated the efficiency with which A-, C-, G-, or T-tailed dsDNA was appended with a GAO carrying a stretch of Ts, Gs, Cs, or As at its 3′ end, respectively. In the first step of the experiment, a 70-bp blunt-end DNA carrying the FAM fluorophore at one of its 5′ ends (FAM70; the first line in each panel of <a reveal-id="dsy018-F2" data-open="dsy018-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>) was subjected to A-, C-, G-, or T-tailing (see the second line in each panel of <a reveal-id="dsy018-F2" data-open="dsy018-F2" class="link link-reveal link-table xref-fig">Fig. 2</a> for tail length distributions). Each tailing reaction was terminated by phenol/chloroform/isoamylalcohol extraction and ethanol precipitation. In the second step, the tailed-DNA was reacted with the GAO carrying a stretch of nucleotides complementary to the tail. The lengths of the complementary nucleotides were 4 for A, T, and G and 6 for C. The longer C-stretch was used to respond to the longer G-tail that exceeded four nucleotides long. Each reaction product was analysed with a capillary sequencer, and the resulting electropherogram was analysed using TraceViewer software. </p>                        <a id="122671399" scrollto-destination="122671399"></a>
<div id="dsy018-F2" data-id="dsy018-F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/m_dsy018f2.png?Expires=2147483647&Signature=AtmgESyX7IduZqlpdvhQuPNlpxQe0pqD5lP76mcF0ADIkgOfpP16RVsTMFq6Vmrf13Hk3zA1JRyBZ6Lx1xKlwIZJdPK3zA3VyuYCT4pDwcl8L5iEnBHX~tbe~NJoZ8UDfstahTYWXmAzanlw1jnQ16cESF5Qt2pHwLk2b5xMGMpUOKPuyJiNg7uuFrspaxMDIRJ9Zus3A6X7D6p15NbBn9GllazCeLQTp1tclHDVQ~rFWmQOS-CAPpWrzjk-e3DvCrhJwYlYEF2UKzxPXVzQ~hVlg3hGQswCLpNwSbPkdCAk6EWr5flk9NMEQtBR9ALDmvCgmLhc2J7oBcEhDSvrIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="In the CIS reaction, G- or C-tailed DNA acquired the GAO efficiently. FAM70 DNA (a 70-bp dsDNA carrying FAM at one 5′ end) was subjected to A-, C-, G-, or T-tailing and then purified by phenol/chloroform/isoamylalcohol extraction and ethanol precipitation. The tailing reactions produced tails of different lengths; some of the peaks are labelled with the number of nucleotides added. Each of the tailed DNAs was reacted with the indicated GAO carrying a stretch of nucleotides complementary to the tail (underlined). At different time points, a portion of the reaction mixture was taken for analysis under denaturing conditions to investigate the length of the FAM-labelled strand. The panels present the results of A-tailing (a), C-tailing (b), G-tailing (c), and T-tailing (d) of FAM70 DNA and the subsequent CIS reaction with SA597 (a), SA593 (b), SA574 (c), and SA601 (d). In each panel, the electropherogram of unreacted FAM70 DNA and the tailing reaction product are also shown. Single asterisks (*) indicate the CIS reaction products, and double asterisks (**) indicate concatenated products. The CIS reaction products were observed as multiple peaks, which resulted from additional tailing reactions. Note that the sampling time points are different in panels (a, d) and (b, c)." path-from-xml="dsy018f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671399/dsy018f2.tif" path-from-xml="dsy018f2.tif" target="_blank">View large</a><a section="122671399" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/dsy018f2.png?Expires=2147483647&Signature=2ebKs1xb3ndzc05ERtDuq742dPOl00EEGJrEQe9rgGZlNkalBVh90oYZ7zhQy2iFqWb1PlrpwPOTCtHCWMQ-pc7DcPVFZMLwj7ALfLnrqF0IHfotq1Ort16ad0JpB-sEosFBp59nbdM82WnsSTvJisQwSEsToxmQ~OGBpj6wkRp~bfHp85HauM6~fiFdKujjoUREnooY5~geEUSl8JrRlyLWCFTmu~NGhXW7h2mk1GF8PwUwyGBZT~FjIA0J9fSTt-30DWorx13UWsnelRpmsmuqGqXn-r8zcRH2xh9aiz8JUa4S1F81cMbKrlMaKR3DGO4d~JxgMflP8QPxoNRoVg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671399&ar=5035914&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy018f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>In the CIS reaction, G- or C-tailed DNA acquired the GAO efficiently. FAM70 DNA (a 70-bp dsDNA carrying FAM at one 5′ end) was subjected to A-, C-, G-, or T-tailing and then purified by phenol/chloroform/isoamylalcohol extraction and ethanol precipitation. The tailing reactions produced tails of different lengths; some of the peaks are labelled with the number of nucleotides added. Each of the tailed DNAs was reacted with the indicated GAO carrying a stretch of nucleotides complementary to the tail (underlined). At different time points, a portion of the reaction mixture was taken for analysis under denaturing conditions to investigate the length of the FAM-labelled strand. The panels present the results of A-tailing (a), C-tailing (b), G-tailing (c), and T-tailing (d) of FAM70 DNA and the subsequent CIS reaction with SA597 (a), SA593 (b), SA574 (c), and SA601 (d). In each panel, the electropherogram of unreacted FAM70 DNA and the tailing reaction product are also shown. Single asterisks (*) indicate the CIS reaction products, and double asterisks (**) indicate concatenated products. The CIS reaction products were observed as multiple peaks, which resulted from additional tailing reactions. Note that the sampling time points are different in panels (a, d) and (b, c).</p></div></div><div content-id="dsy018-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/m_dsy018f2.png?Expires=2147483647&Signature=AtmgESyX7IduZqlpdvhQuPNlpxQe0pqD5lP76mcF0ADIkgOfpP16RVsTMFq6Vmrf13Hk3zA1JRyBZ6Lx1xKlwIZJdPK3zA3VyuYCT4pDwcl8L5iEnBHX~tbe~NJoZ8UDfstahTYWXmAzanlw1jnQ16cESF5Qt2pHwLk2b5xMGMpUOKPuyJiNg7uuFrspaxMDIRJ9Zus3A6X7D6p15NbBn9GllazCeLQTp1tclHDVQ~rFWmQOS-CAPpWrzjk-e3DvCrhJwYlYEF2UKzxPXVzQ~hVlg3hGQswCLpNwSbPkdCAk6EWr5flk9NMEQtBR9ALDmvCgmLhc2J7oBcEhDSvrIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="In the CIS reaction, G- or C-tailed DNA acquired the GAO efficiently. FAM70 DNA (a 70-bp dsDNA carrying FAM at one 5′ end) was subjected to A-, C-, G-, or T-tailing and then purified by phenol/chloroform/isoamylalcohol extraction and ethanol precipitation. The tailing reactions produced tails of different lengths; some of the peaks are labelled with the number of nucleotides added. Each of the tailed DNAs was reacted with the indicated GAO carrying a stretch of nucleotides complementary to the tail (underlined). At different time points, a portion of the reaction mixture was taken for analysis under denaturing conditions to investigate the length of the FAM-labelled strand. The panels present the results of A-tailing (a), C-tailing (b), G-tailing (c), and T-tailing (d) of FAM70 DNA and the subsequent CIS reaction with SA597 (a), SA593 (b), SA574 (c), and SA601 (d). In each panel, the electropherogram of unreacted FAM70 DNA and the tailing reaction product are also shown. Single asterisks (*) indicate the CIS reaction products, and double asterisks (**) indicate concatenated products. The CIS reaction products were observed as multiple peaks, which resulted from additional tailing reactions. Note that the sampling time points are different in panels (a, d) and (b, c)." path-from-xml="dsy018f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671399/dsy018f2.tif" path-from-xml="dsy018f2.tif" target="_blank">View large</a><a section="122671399" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/dsy018f2.png?Expires=2147483647&Signature=2ebKs1xb3ndzc05ERtDuq742dPOl00EEGJrEQe9rgGZlNkalBVh90oYZ7zhQy2iFqWb1PlrpwPOTCtHCWMQ-pc7DcPVFZMLwj7ALfLnrqF0IHfotq1Ort16ad0JpB-sEosFBp59nbdM82WnsSTvJisQwSEsToxmQ~OGBpj6wkRp~bfHp85HauM6~fiFdKujjoUREnooY5~geEUSl8JrRlyLWCFTmu~NGhXW7h2mk1GF8PwUwyGBZT~FjIA0J9fSTt-30DWorx13UWsnelRpmsmuqGqXn-r8zcRH2xh9aiz8JUa4S1F81cMbKrlMaKR3DGO4d~JxgMflP8QPxoNRoVg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671399&ar=5035914&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy018f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>In the CIS reaction, G- or C-tailed DNA acquired the GAO efficiently. FAM70 DNA (a 70-bp dsDNA carrying FAM at one 5′ end) was subjected to A-, C-, G-, or T-tailing and then purified by phenol/chloroform/isoamylalcohol extraction and ethanol precipitation. The tailing reactions produced tails of different lengths; some of the peaks are labelled with the number of nucleotides added. Each of the tailed DNAs was reacted with the indicated GAO carrying a stretch of nucleotides complementary to the tail (underlined). At different time points, a portion of the reaction mixture was taken for analysis under denaturing conditions to investigate the length of the FAM-labelled strand. The panels present the results of A-tailing (a), C-tailing (b), G-tailing (c), and T-tailing (d) of FAM70 DNA and the subsequent CIS reaction with SA597 (a), SA593 (b), SA574 (c), and SA601 (d). In each panel, the electropherogram of unreacted FAM70 DNA and the tailing reaction product are also shown. Single asterisks (*) indicate the CIS reaction products, and double asterisks (**) indicate concatenated products. The CIS reaction products were observed as multiple peaks, which resulted from additional tailing reactions. Note that the sampling time points are different in panels (a, d) and (b, c).</p></div></div><p>As shown in <a reveal-id="dsy018-F2" data-open="dsy018-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>, all types of tails analysed reacted with the corresponding GAO; however, regarding the reaction speed and yield, C- and G-tailed dsDNAs reacted well with the GAO, and A- and T-tailed dsDNAs reacted far less efficiently. For example, in our experimental setup using 100 fmol G-tailed DNA and 50 units of MMLV-RT in a 10-µl reaction, the GAO was appended to nearly 100% of G-tailed dsDNAs within a minute. In contrast, 30% of A-tailed DNAs remained unreacted even after 50 min of incubation (sum of multiple peak areas corresponding to the CIS products was divided by the total peak area). Moreover, G-tailed DNA reacted slightly better than C-tailed DNA [see weak signals representing unreacted substrates in panel (b)], and yielded less concatenated products [marked with double asterisks (**) in <a reveal-id="dsy018-F2" data-open="dsy018-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>].</p>                            <h3 scrollto-destination=122671401 id="122671401" class="section-title" >3.4. A longer 3′-tail is favourable for the CIS reaction</h3>
<p>To determine whether a longer 3′ tail is favourable for the CIS reaction, 33-bp dsDNAs with 3′ G-tails 0–4 nucleotides long (33G0 to 33G4; see <a reveal-id="dsy018-F3" data-open="dsy018-F3" class="link link-reveal link-table xref-fig">Fig. 3a</a>) were reacted with GAO SA574. The results shown in <a reveal-id="dsy018-F3" data-open="dsy018-F3" class="link link-reveal link-table xref-fig">Fig. 3a</a> clearly indicate that the longer the tail was, the higher the efficiency. To further clarify this, FAM70 DNA was G- or C-tailed in the presence (marked FAM70G+ and FAM70C+ in <a reveal-id="dsy018-F3" data-open="dsy018-F3" class="link link-reveal link-table xref-fig">Fig. 3b and c</a>, respectively) or absence of a cognate tailing-enhancer,<a reveal-id="dsy018-B10" data-open="dsy018-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> and the tailed product was reacted with a corresponding GAO (<a reveal-id="dsy018-F3" data-open="dsy018-F3" class="link link-reveal link-table xref-fig">Fig. 3b and c</a>). Again, longer 3′ tails were more favourable for CIS reaction. </p>                        <a id="122671403" scrollto-destination="122671403"></a>
<div id="dsy018-F3" data-id="dsy018-F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/m_dsy018f3.png?Expires=2147483647&Signature=NdqfjgrhlkW5hQYIGeV8~MQqnWo778~EuNgDEOWi15YaQoMtW2erb6IqB7Z42CBInJ-WGxNpkCMpLPj1DL5YgUN3IxIWznbqFHGqQhpBkuKsJkOE1vefvVPX9m9OxTtGpCt6WUmb2vPjK1xKmirLsGt8imt8L4gJeshJy8kPOiyYz5RilkstOAT5Q1ZgHELboGfDoGfQkZfSvswgJeDNPhqo0fppLxae3o1qeMD6~8P-NK8CbQks8gXJVgccEgv8oif4t4P77sQzecd~8bE3Fi-dslQ1virHo1cSdsYiAw-cgSD2xcEWi~03oA7ODRCkweSo8VmrXQj6HmD5Dxu53g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Longer tails are favourable for the CIS reaction. (a) Double-stranded DNAs 33G0 to 33G4, which are 33 base pairs long plus a 3′ tail of 0–4 Gs (carries FAM at the 5′ end of the G-tail strand), were subjected to the CIS reaction with GAO SA574. At the different time points, a portion of the reaction mixture was taken for analysis using a capillary sequencer, and the GAO incorporation rate was calculated as described in the materials and methods. (b) FAM70 DNA was G-tailed in the presence (FAM70G+) or absence (FAM70G) of the tailing enhancer deoxycytidine, purified by phenol/chloroform/isoamylalcohol extraction and ethanol precipitation, and then reacted with SA574. (c) FAM70 DNA was C-tailed in the presence (FAM70C+) or absence (FAM70C) of the tailing enhancer deoxyguanosine, purified, and reacted with GAO SA593. Values are the average of triplicate experiments, and standard deviations are shown." path-from-xml="dsy018f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671403/dsy018f3.tif" path-from-xml="dsy018f3.tif" target="_blank">View large</a><a section="122671403" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/dsy018f3.png?Expires=2147483647&Signature=bvQQ8kRAk7kL6vv1Z7hUUrHZS7dUgM673UHnG75EoKwkj1O5yoEqCgCg9PJKiO-O3mhlzRHT3XxM8gBskwYkMOtozHu~31p3riMVCBXOwCkyW3JyRVS5bC1S8jhdTJROjYDxpaSj8jRHLlZhlFd9dS1WmaAlNiXE1mf106I-YAnkta6fl2bkgTSb-FCAb2kCRzMLVAIRaf-SMepaCT8gIJmWCvqWI65zA2JbHNufBS~YkBKB2NtcEKurmwYgkTZvnNMFjWaXwzxksXGr3IMimYIFkO~mjC88vqziugk6HfvjdJuHMS-VZ7JRaN7DVjszktbcPLjCMJOlqwB6V7RUfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671403&ar=5035914&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy018f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Longer tails are favourable for the CIS reaction. (a) Double-stranded DNAs 33G0 to 33G4, which are 33 base pairs long plus a 3′ tail of 0–4 Gs (carries FAM at the 5′ end of the G-tail strand), were subjected to the CIS reaction with GAO SA574. At the different time points, a portion of the reaction mixture was taken for analysis using a capillary sequencer, and the GAO incorporation rate was calculated as described in the materials and methods. (b) FAM70 DNA was G-tailed in the presence (FAM70G<sup>+</sup>) or absence (FAM70G) of the tailing enhancer deoxycytidine, purified by phenol/chloroform/isoamylalcohol extraction and ethanol precipitation, and then reacted with SA574. (c) FAM70 DNA was C-tailed in the presence (FAM70C<sup>+</sup>) or absence (FAM70C) of the tailing enhancer deoxyguanosine, purified, and reacted with GAO SA593. Values are the average of triplicate experiments, and standard deviations are shown.</p></div></div><div content-id="dsy018-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/m_dsy018f3.png?Expires=2147483647&Signature=NdqfjgrhlkW5hQYIGeV8~MQqnWo778~EuNgDEOWi15YaQoMtW2erb6IqB7Z42CBInJ-WGxNpkCMpLPj1DL5YgUN3IxIWznbqFHGqQhpBkuKsJkOE1vefvVPX9m9OxTtGpCt6WUmb2vPjK1xKmirLsGt8imt8L4gJeshJy8kPOiyYz5RilkstOAT5Q1ZgHELboGfDoGfQkZfSvswgJeDNPhqo0fppLxae3o1qeMD6~8P-NK8CbQks8gXJVgccEgv8oif4t4P77sQzecd~8bE3Fi-dslQ1virHo1cSdsYiAw-cgSD2xcEWi~03oA7ODRCkweSo8VmrXQj6HmD5Dxu53g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Longer tails are favourable for the CIS reaction. (a) Double-stranded DNAs 33G0 to 33G4, which are 33 base pairs long plus a 3′ tail of 0–4 Gs (carries FAM at the 5′ end of the G-tail strand), were subjected to the CIS reaction with GAO SA574. At the different time points, a portion of the reaction mixture was taken for analysis using a capillary sequencer, and the GAO incorporation rate was calculated as described in the materials and methods. (b) FAM70 DNA was G-tailed in the presence (FAM70G+) or absence (FAM70G) of the tailing enhancer deoxycytidine, purified by phenol/chloroform/isoamylalcohol extraction and ethanol precipitation, and then reacted with SA574. (c) FAM70 DNA was C-tailed in the presence (FAM70C+) or absence (FAM70C) of the tailing enhancer deoxyguanosine, purified, and reacted with GAO SA593. Values are the average of triplicate experiments, and standard deviations are shown." path-from-xml="dsy018f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671403/dsy018f3.tif" path-from-xml="dsy018f3.tif" target="_blank">View large</a><a section="122671403" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/dsy018f3.png?Expires=2147483647&Signature=bvQQ8kRAk7kL6vv1Z7hUUrHZS7dUgM673UHnG75EoKwkj1O5yoEqCgCg9PJKiO-O3mhlzRHT3XxM8gBskwYkMOtozHu~31p3riMVCBXOwCkyW3JyRVS5bC1S8jhdTJROjYDxpaSj8jRHLlZhlFd9dS1WmaAlNiXE1mf106I-YAnkta6fl2bkgTSb-FCAb2kCRzMLVAIRaf-SMepaCT8gIJmWCvqWI65zA2JbHNufBS~YkBKB2NtcEKurmwYgkTZvnNMFjWaXwzxksXGr3IMimYIFkO~mjC88vqziugk6HfvjdJuHMS-VZ7JRaN7DVjszktbcPLjCMJOlqwB6V7RUfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671403&ar=5035914&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy018f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Longer tails are favourable for the CIS reaction. (a) Double-stranded DNAs 33G0 to 33G4, which are 33 base pairs long plus a 3′ tail of 0–4 Gs (carries FAM at the 5′ end of the G-tail strand), were subjected to the CIS reaction with GAO SA574. At the different time points, a portion of the reaction mixture was taken for analysis using a capillary sequencer, and the GAO incorporation rate was calculated as described in the materials and methods. (b) FAM70 DNA was G-tailed in the presence (FAM70G<sup>+</sup>) or absence (FAM70G) of the tailing enhancer deoxycytidine, purified by phenol/chloroform/isoamylalcohol extraction and ethanol precipitation, and then reacted with SA574. (c) FAM70 DNA was C-tailed in the presence (FAM70C<sup>+</sup>) or absence (FAM70C) of the tailing enhancer deoxyguanosine, purified, and reacted with GAO SA593. Values are the average of triplicate experiments, and standard deviations are shown.</p></div></div>                            <h3 scrollto-destination=122671404 id="122671404" class="section-title" >3.5. Ribonucleotides at the GAO 3′ end facilitate the CIS reaction</h3>
<p>In template switching reactions, oligonucleotides with a few ribonucleotides<a reveal-id="dsy018-B11" data-open="dsy018-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> or locked-nucleotides<a reveal-id="dsy018-B12" data-open="dsy018-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> at their 3′ end are often used to improve the reaction efficiency, possibly by increasing the thermostability between the tail and the 3′ end of the oligonucleotides. To assess the effect of replacing the 3′ deoxyribonucleotides of GAO with ribonucleotides on the CIS reaction, FAM70 DNA tailed with C or G in the absence of enhancers was reacted with a GAO that carried rGrGrG or rCrCrC at its 3′ end (the lower case ‘r’ indicates ‘ribonucleotide’).</p><p><a reveal-id="dsy018-F4" data-open="dsy018-F4" class="link link-reveal link-table xref-fig">Figure 4a</a> shows that GAO with ribonucleotides reacted more efficiently than its deoxynucleotidyl counterpart, i.e. G-tailed DNA (FAM70G) reacted faster with SA671 (carrying dCrCrCrC at its 3′ end) than with SA673 (carrying dCdCdCdC at its 3′ end), and C-tailed DNA (FAM70C) reacted faster with SA670 (carrying rGrGrG at its 3′ end) than with SA672 (carrying dGdGdG at its 3′ end). The superiority of the ribonucleotidyl GAO to deoxyribonucleotidyl GAO was also observed when a dsDNA with a longer G-tail was tested (<a reveal-id="dsy018-F4" data-open="dsy018-F4" class="link link-reveal link-table xref-fig">Fig. 4b</a>). A tailed 33G4 DNA (carrying four Gs as a tail) reacted faster with SA671 than with SA673. Likewise, FAM70 DNA subjected to C-tailing in the presence of an enhancer (C3, 55%; C4, 44%) reacted faster with SA670 than with SA672 (<a reveal-id="dsy018-F4" data-open="dsy018-F4" class="link link-reveal link-table xref-fig">Fig. 4c</a>). </p>                        <a id="122671407" scrollto-destination="122671407"></a>
<div id="dsy018-F4" data-id="dsy018-F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/m_dsy018f4.png?Expires=2147483647&Signature=jxJXBRBCHc~Bq5AYdIOTzpkktWcDkOEBrbugheVifPJHTPRL4Y-HBgY7SaW1hK2GTQALQYWWVqudvTb8C3zGwMje0YIzyjaPYKYcrdxOFRURFe-gmSOQ0CIYbRrYYOb8pRG7TkGlxC8VYu4ZRryMxGcicKeGO05vwgjsaRiZzAiy-VWS~v-fSpx6fcPYdFAL-bH4KHPW1b353Z0SZvVjHD2OAUtkzlKlcrhyrNEeqm1ssJVLikG84p7NJ1SqX45s7TBA3avyAXxcS7v0~N3RxBinbXe~bUZuQJmBq5ggNkNQwja6uO1bklv~vVu0FWekwwMei2zCOldRlomoV0vPqQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GAOs carrying 3′ ribonucleotides react more efficiently. G-tailed FAM70 DNA (G298%) or C-tailed FAM70 DNA (C182%C212%C35%) prepared by a tailing reaction conducted in the absence of a tailing enhancer (a), 33G4 DNA (b), and C-tailed FAM70 DNA (C356%C444%) in the presence of the tailing enhancer deoxyguanosine (c) were reacted with the indicated GAOs. Values are the average of triplicate experiments, and standard deviations are shown. SA670 and SA672 are equivalent to TS-oligo-rG3 and TS-oligo-dG3, respectively, in reference.6" path-from-xml="dsy018f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671407/dsy018f4.tif" path-from-xml="dsy018f4.tif" target="_blank">View large</a><a section="122671407" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/dsy018f4.png?Expires=2147483647&Signature=q9WtnvdA35b2dROwyO~hTzztzvUudlAouOXy4PeFKVUaS07Yiw97-JFF~kB8NHWTuBR2KkxaeYRVezsRftZqItH4i1Sgmy7d8GmeoJProF8ybl-Sy-3bbpzNObFkmzbvsqlm97YhnXmoXXG2B9Sle4NUXZpABNR42v7ONACEPW0B1NH~OEq5MXTsesAaykrAxWYkujz8EnMY9VxcS3A0L8FKfmPGnng2aQdGbtkJiWm8y-ieBVkgwoce~vBf7tC~mrNvmwaRWtw9YZ9OJu4OBB8jdhE8uPj5qdH69~qtIdczu4VH5gypQ2wQ~l2COF3tDz0rK6FJiJQOeh77F7jeng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671407&ar=5035914&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy018f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GAOs carrying 3′ ribonucleotides react more efficiently. G-tailed FAM70 DNA (G2<sup>98%</sup>) or C-tailed FAM70 DNA (C1<sup>82%</sup>C2<sup>12%</sup>C3<sup>5%</sup>) prepared by a tailing reaction conducted in the absence of a tailing enhancer (a), 33G4 DNA (b), and C-tailed FAM70 DNA (C3<sup>56%</sup>C4<sup>44%</sup>) in the presence of the tailing enhancer deoxyguanosine (c) were reacted with the indicated GAOs. Values are the average of triplicate experiments, and standard deviations are shown. SA670 and SA672 are equivalent to TS-oligo-rG3 and TS-oligo-dG3, respectively, in reference.<a reveal-id="dsy018-B6" data-open="dsy018-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a></p></div></div><div content-id="dsy018-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/m_dsy018f4.png?Expires=2147483647&Signature=jxJXBRBCHc~Bq5AYdIOTzpkktWcDkOEBrbugheVifPJHTPRL4Y-HBgY7SaW1hK2GTQALQYWWVqudvTb8C3zGwMje0YIzyjaPYKYcrdxOFRURFe-gmSOQ0CIYbRrYYOb8pRG7TkGlxC8VYu4ZRryMxGcicKeGO05vwgjsaRiZzAiy-VWS~v-fSpx6fcPYdFAL-bH4KHPW1b353Z0SZvVjHD2OAUtkzlKlcrhyrNEeqm1ssJVLikG84p7NJ1SqX45s7TBA3avyAXxcS7v0~N3RxBinbXe~bUZuQJmBq5ggNkNQwja6uO1bklv~vVu0FWekwwMei2zCOldRlomoV0vPqQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GAOs carrying 3′ ribonucleotides react more efficiently. G-tailed FAM70 DNA (G298%) or C-tailed FAM70 DNA (C182%C212%C35%) prepared by a tailing reaction conducted in the absence of a tailing enhancer (a), 33G4 DNA (b), and C-tailed FAM70 DNA (C356%C444%) in the presence of the tailing enhancer deoxyguanosine (c) were reacted with the indicated GAOs. Values are the average of triplicate experiments, and standard deviations are shown. SA670 and SA672 are equivalent to TS-oligo-rG3 and TS-oligo-dG3, respectively, in reference.6" path-from-xml="dsy018f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671407/dsy018f4.tif" path-from-xml="dsy018f4.tif" target="_blank">View large</a><a section="122671407" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/dsy018f4.png?Expires=2147483647&Signature=q9WtnvdA35b2dROwyO~hTzztzvUudlAouOXy4PeFKVUaS07Yiw97-JFF~kB8NHWTuBR2KkxaeYRVezsRftZqItH4i1Sgmy7d8GmeoJProF8ybl-Sy-3bbpzNObFkmzbvsqlm97YhnXmoXXG2B9Sle4NUXZpABNR42v7ONACEPW0B1NH~OEq5MXTsesAaykrAxWYkujz8EnMY9VxcS3A0L8FKfmPGnng2aQdGbtkJiWm8y-ieBVkgwoce~vBf7tC~mrNvmwaRWtw9YZ9OJu4OBB8jdhE8uPj5qdH69~qtIdczu4VH5gypQ2wQ~l2COF3tDz0rK6FJiJQOeh77F7jeng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671407&ar=5035914&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy018f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GAOs carrying 3′ ribonucleotides react more efficiently. G-tailed FAM70 DNA (G2<sup>98%</sup>) or C-tailed FAM70 DNA (C1<sup>82%</sup>C2<sup>12%</sup>C3<sup>5%</sup>) prepared by a tailing reaction conducted in the absence of a tailing enhancer (a), 33G4 DNA (b), and C-tailed FAM70 DNA (C3<sup>56%</sup>C4<sup>44%</sup>) in the presence of the tailing enhancer deoxyguanosine (c) were reacted with the indicated GAOs. Values are the average of triplicate experiments, and standard deviations are shown. SA670 and SA672 are equivalent to TS-oligo-rG3 and TS-oligo-dG3, respectively, in reference.<a reveal-id="dsy018-B6" data-open="dsy018-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a></p></div></div>                            <h3 scrollto-destination=122671408 id="122671408" class="section-title" >3.6. 5′ biotin on GAO inhibits concatenation</h3>
<p>Because GAO concatenation might compromize the downstream application of CIS reaction products, its suppression is desirable. GAO concatenation might also arise through the annealing of two GAOs and complementary strand synthesis by MMLV-RT and subsequent tailing and the CIS reaction.</p><p>We found that 5′-biotinylated GAO (SA686) suppressed the concatenation (<a reveal-id="dsy018-F5" data-open="dsy018-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). In <a reveal-id="dsy018-F5" data-open="dsy018-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>, when unbiotinylated GAO (SA574) was used, peaks representing the concatenated products were observed (marked with ** in <a reveal-id="dsy018-F5" data-open="dsy018-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). In contrast, when biotinylated GAO (SA686) was used, no concatenated products were observed. Originally, we expected that the addition of streptavidin to the reaction mixture containing biotinylated GAO would lead to the formation of a macromolecular complex that might exclude the MMLV-RT from the 5′ end of the GAO by steric hindrance, thereby inhibiting the concatenation. Contrary to our expectation, even in the absence of streptavidin, biotinylated GAO suppressed the concatenation. However, the addition of streptavidin to the reaction using SA686 changed the peak profile of the CIS reaction product (<a reveal-id="dsy018-F5" data-open="dsy018-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>), indicating that the strand synthesis of the last few bases was suppressed by the biotin-streptavidin complex. </p>                        <a id="122671411" scrollto-destination="122671411"></a>
<div id="dsy018-F5" data-id="dsy018-F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/m_dsy018f5.png?Expires=2147483647&Signature=LQ475bcukCFLJw41RLLG4KxalmDxAuVIBzQ~q-G1dBF5-uYhoPruppdudCuQD~54zvM0iHcHsh-dp533aoQrtGXl7wQGas0dXsupX1q2I3-9ZiG07r0SMyoQPe8YECkNnSLLmM5enzVIsNJlW0mBryqwHCaRxxkRYIm3boJg-~gTtsZQgH01TUihQlFmVwI7l8lmevmkjCVVojYeO30BhzUMhHx4LFYsO0ql5g8XhlauMk0E2g3dXOBMS0q4dgTBTVCYCblrz4ndzI6sXZ~zA1vvPvV4rF~UdW4p6Vjh8Ge9ttRKym6iYETyYwth8Iu7lXUhSCFxA1ESKZMTeyHStg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Biotin labelling at the GAO 5′ end prevented concatenation. FAM70 DNA was G-tailed (G480%G520%) and purified using a DNA purification column. Two types of GAOs, SA574 and SA686, with SA686 bearing a biotin moiety at its 5′ end, were reacted with the G-tailed DNA for 20 min in the presence or absence of 10 pmol streptavidin. The single asterisk (*) indicates the CIS reaction product, and parts of the data were magnified 10-fold to show the concatenated products (**). Note that, under the denaturing conditions of the capillary sequencer, the FAM-labelled strand is dissociated from the biotinylated strand." path-from-xml="dsy018f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671411/dsy018f5.tif" path-from-xml="dsy018f5.tif" target="_blank">View large</a><a section="122671411" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/dsy018f5.png?Expires=2147483647&Signature=CiBHx7Aus0pzsF88fL4AnfT-nTj5akj53YQlELUSF4i-LAMiNDMk77r1LxmSlq-BPooXrnHazEqyGgfCOpilHXl-eYSuPdOlYydhKqBfyq4F05LVDdlvy1kUAQXMACPtkMB9m29qoNl5E-2HZrOqZfKRhNNDssfCKAuFfba-67utRyOt93~cHSz463sGielAa3TAzCDKYHMDUwbox-ebNCndggY6aHHaVKLW~58t7dmkVDHPo8zOA3TpCkd4c-JPkvlr6Li6UxVloe99EBhoAzMfZ~zj2SRkJcZ1nQ~9nA3KZH8NLvrZzntO9gDvmGK-s7GdIfELVAQn4j3mAqXDXw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671411&ar=5035914&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy018f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Biotin labelling at the GAO 5′ end prevented concatenation. FAM70 DNA was G-tailed (G4<sup>80%</sup>G5<sup>20%</sup>) and purified using a DNA purification column. Two types of GAOs, SA574 and SA686, with SA686 bearing a biotin moiety at its 5′ end, were reacted with the G-tailed DNA for 20 min in the presence or absence of 10 pmol streptavidin. The single asterisk (*) indicates the CIS reaction product, and parts of the data were magnified 10-fold to show the concatenated products (**). Note that, under the denaturing conditions of the capillary sequencer, the FAM-labelled strand is dissociated from the biotinylated strand.</p></div></div><div content-id="dsy018-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/m_dsy018f5.png?Expires=2147483647&Signature=LQ475bcukCFLJw41RLLG4KxalmDxAuVIBzQ~q-G1dBF5-uYhoPruppdudCuQD~54zvM0iHcHsh-dp533aoQrtGXl7wQGas0dXsupX1q2I3-9ZiG07r0SMyoQPe8YECkNnSLLmM5enzVIsNJlW0mBryqwHCaRxxkRYIm3boJg-~gTtsZQgH01TUihQlFmVwI7l8lmevmkjCVVojYeO30BhzUMhHx4LFYsO0ql5g8XhlauMk0E2g3dXOBMS0q4dgTBTVCYCblrz4ndzI6sXZ~zA1vvPvV4rF~UdW4p6Vjh8Ge9ttRKym6iYETyYwth8Iu7lXUhSCFxA1ESKZMTeyHStg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Biotin labelling at the GAO 5′ end prevented concatenation. FAM70 DNA was G-tailed (G480%G520%) and purified using a DNA purification column. Two types of GAOs, SA574 and SA686, with SA686 bearing a biotin moiety at its 5′ end, were reacted with the G-tailed DNA for 20 min in the presence or absence of 10 pmol streptavidin. The single asterisk (*) indicates the CIS reaction product, and parts of the data were magnified 10-fold to show the concatenated products (**). Note that, under the denaturing conditions of the capillary sequencer, the FAM-labelled strand is dissociated from the biotinylated strand." path-from-xml="dsy018f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671411/dsy018f5.tif" path-from-xml="dsy018f5.tif" target="_blank">View large</a><a section="122671411" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/dsy018f5.png?Expires=2147483647&Signature=CiBHx7Aus0pzsF88fL4AnfT-nTj5akj53YQlELUSF4i-LAMiNDMk77r1LxmSlq-BPooXrnHazEqyGgfCOpilHXl-eYSuPdOlYydhKqBfyq4F05LVDdlvy1kUAQXMACPtkMB9m29qoNl5E-2HZrOqZfKRhNNDssfCKAuFfba-67utRyOt93~cHSz463sGielAa3TAzCDKYHMDUwbox-ebNCndggY6aHHaVKLW~58t7dmkVDHPo8zOA3TpCkd4c-JPkvlr6Li6UxVloe99EBhoAzMfZ~zj2SRkJcZ1nQ~9nA3KZH8NLvrZzntO9gDvmGK-s7GdIfELVAQn4j3mAqXDXw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671411&ar=5035914&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy018f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Biotin labelling at the GAO 5′ end prevented concatenation. FAM70 DNA was G-tailed (G4<sup>80%</sup>G5<sup>20%</sup>) and purified using a DNA purification column. Two types of GAOs, SA574 and SA686, with SA686 bearing a biotin moiety at its 5′ end, were reacted with the G-tailed DNA for 20 min in the presence or absence of 10 pmol streptavidin. The single asterisk (*) indicates the CIS reaction product, and parts of the data were magnified 10-fold to show the concatenated products (**). Note that, under the denaturing conditions of the capillary sequencer, the FAM-labelled strand is dissociated from the biotinylated strand.</p></div></div>                            <h3 scrollto-destination=122671412 id="122671412" class="section-title" >3.7. Fate of the GAO in the CIS reaction</h3>
<p>In this study, we confirmed the strand displacement synthesis by RTs that originates from the 3′ end of GAO followed by the clamping of the 3′ tail of dsDNA and 3′-end nucleotides of the single-stranded DNA.<a reveal-id="dsy018-B3" data-open="dsy018-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> A GAO (SA701) with a Cy3-labelled 5′-end was reacted with a G-tailed DNA and then analysed to see the extended Cy3-labelled fragments. To exclude the unreacted GAO, the reaction product was subjected to polyacrylamide-gel electrophoresis, and the band with the expected size of the CIS reaction product was excised, purified, and analysed with a capillary sequencer under denaturing conditions. As shown in <a reveal-id="dsy018-F6" data-open="dsy018-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>, the GAO strand was extended as demonstrated by the appearance of peaks that were located close to the peaks for the CIS reaction product of the FAM-labelled upper strand. The slight difference in the electrophoretic mobilities of FAM and the Cy3-labelled strand might reflect the different nucleotide compositions of the strands. The peak area ratio of the Cy3 to FAM signals was close to the ratio of 0.25 obtained when running equimolar Cy3 or FAM-labelled single-stranded DNA (data not shown), indicating that most of the GAO molecules were incorporated into the target DNA that underwent strand extension. Because MMLV-RT has not been reported to have 5′ -&gt; 3′ exonuclease activity, this extension might represent the strand-displacing synthesis that MMLV-RT catalyses during cDNA synthesis. </p>                        <a id="122671414" scrollto-destination="122671414"></a>
<div id="dsy018-F6" data-id="dsy018-F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/m_dsy018f6.png?Expires=2147483647&Signature=O7x2Kovrsoc-lvCXIOjR1oGIk1zfyt9ahDiIWZ-dsMMZK4iSDtpVjp~SctEwCYQ8to9z54hk-Ui6wzaZ9kjLt0-jjfiaEaR4WCAWFddvAEo6yVhIHgbRqzsRBwQaCA0DlTScpHeSA~mI-AqGiY79tf3pf3PkIe3SOHqtvXeau6US4Js6NYZSKmIgwzAvrFP7B8rvoZT4X8i-LUirKdJsYXYbayUjLe8q0trIPFuafx025En9K60iE7XrIe-58GPGyldCpYk74RQX12Cjb6TX2dUayCohJ6-TXO~fXe6T8weKXT-TUXUlioUcnsVqodxjcjReUMkiCA2pyXV0aXMtVg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GAO is incorporated into the target DNA end. FAM70 DNA (a) was G-tailed (G489%G511%) in the presence of the enhancer deoxycytidine (b) and then reacted in a CIS reaction with GAO SA701, which carries a Cy3 label at its 5′ end. The CIS reaction product was purified by polyacrylamide gel electrophoresis and then analysed (c). As a size control, SA701 was also analysed (d). The Cy3 signals are indicated by asterisks." path-from-xml="dsy018f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671414/dsy018f6.tif" path-from-xml="dsy018f6.tif" target="_blank">View large</a><a section="122671414" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/dsy018f6.png?Expires=2147483647&Signature=y0-LKX74K0aV-ZaYloynqnyyiG67yZdWh6Cs3A5Z9UGnOe7opHiFQgtQkTmivxCGC4XoGWfMcyeGT4iQs0fg6VFWyc73S7fK0TXri2ASJAiExXqKPz8BS1p6ydLFOnyMEb1d5R9EJ3W0RtYOUSi3iIF~agRhz9Ub~UfYpNRjZ-V538AFPUy42RxjoHDue-eL-WCUOCAQ2FiU49mOgeppLZwXdmNLYDV8QrQ8UEpaNFB7NEOrhJKw~X-r1Lwl3vBUMRb7BXH5V6AXdTx5i88KG4nQRp9SjZcuq8gi7YGiCkSP6TITWc9sm~qBTCaBmKZObs2COLvZtmzQt5GIgexodg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671414&ar=5035914&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy018f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GAO is incorporated into the target DNA end. FAM70 DNA (a) was G-tailed (G4<sup>89%</sup>G5<sup>11%</sup>) in the presence of the enhancer deoxycytidine (b) and then reacted in a CIS reaction with GAO SA701, which carries a Cy3 label at its 5′ end. The CIS reaction product was purified by polyacrylamide gel electrophoresis and then analysed (c). As a size control, SA701 was also analysed (d). The Cy3 signals are indicated by asterisks.</p></div></div><div content-id="dsy018-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/m_dsy018f6.png?Expires=2147483647&Signature=O7x2Kovrsoc-lvCXIOjR1oGIk1zfyt9ahDiIWZ-dsMMZK4iSDtpVjp~SctEwCYQ8to9z54hk-Ui6wzaZ9kjLt0-jjfiaEaR4WCAWFddvAEo6yVhIHgbRqzsRBwQaCA0DlTScpHeSA~mI-AqGiY79tf3pf3PkIe3SOHqtvXeau6US4Js6NYZSKmIgwzAvrFP7B8rvoZT4X8i-LUirKdJsYXYbayUjLe8q0trIPFuafx025En9K60iE7XrIe-58GPGyldCpYk74RQX12Cjb6TX2dUayCohJ6-TXO~fXe6T8weKXT-TUXUlioUcnsVqodxjcjReUMkiCA2pyXV0aXMtVg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GAO is incorporated into the target DNA end. FAM70 DNA (a) was G-tailed (G489%G511%) in the presence of the enhancer deoxycytidine (b) and then reacted in a CIS reaction with GAO SA701, which carries a Cy3 label at its 5′ end. The CIS reaction product was purified by polyacrylamide gel electrophoresis and then analysed (c). As a size control, SA701 was also analysed (d). The Cy3 signals are indicated by asterisks." path-from-xml="dsy018f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671414/dsy018f6.tif" path-from-xml="dsy018f6.tif" target="_blank">View large</a><a section="122671414" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/dsy018f6.png?Expires=2147483647&Signature=y0-LKX74K0aV-ZaYloynqnyyiG67yZdWh6Cs3A5Z9UGnOe7opHiFQgtQkTmivxCGC4XoGWfMcyeGT4iQs0fg6VFWyc73S7fK0TXri2ASJAiExXqKPz8BS1p6ydLFOnyMEb1d5R9EJ3W0RtYOUSi3iIF~agRhz9Ub~UfYpNRjZ-V538AFPUy42RxjoHDue-eL-WCUOCAQ2FiU49mOgeppLZwXdmNLYDV8QrQ8UEpaNFB7NEOrhJKw~X-r1Lwl3vBUMRb7BXH5V6AXdTx5i88KG4nQRp9SjZcuq8gi7YGiCkSP6TITWc9sm~qBTCaBmKZObs2COLvZtmzQt5GIgexodg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671414&ar=5035914&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy018f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GAO is incorporated into the target DNA end. FAM70 DNA (a) was G-tailed (G4<sup>89%</sup>G5<sup>11%</sup>) in the presence of the enhancer deoxycytidine (b) and then reacted in a CIS reaction with GAO SA701, which carries a Cy3 label at its 5′ end. The CIS reaction product was purified by polyacrylamide gel electrophoresis and then analysed (c). As a size control, SA701 was also analysed (d). The Cy3 signals are indicated by asterisks.</p></div></div><p>The extension from the 3′ end of the GAO is not necessary for the CIS reaction because a GAO with a 2′, 3′-dideoxy end was incorporated at a rate very similar to that of a GAO with a 2′ deoxy end (data not shown).</p>                            <h3 scrollto-destination=122671416 id="122671416" class="section-title" >3.8. GAO lengths and efficiencies</h3>
<p>In practical use, GAOs with different lengths might be to be incorporated. To investigate the effect of GAO length on CIS reaction efficiency, eight types of GAOs with different lengths ranging from 19 to 72 nucleotides long were reacted with G-tailed FAM70 DNA (G4<sup>80%</sup>G5<sup>20%</sup>). As shown in <a reveal-id="dsy018-F7" data-open="dsy018-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>, GAO length did not seem to affect the reaction efficiency, and 98% of the DNA ends underwent the CIS reaction. </p>                        <a id="122671418" scrollto-destination="122671418"></a>
<div id="dsy018-F7" data-id="dsy018-F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/m_dsy018f7.png?Expires=2147483647&Signature=fQGlK3K35iMKaa7JMTTLWu5fAELdeMiHPt2qR08uiu5y6ryQE4jCn85VmwDNpaL6n3y4~iNku211YhPNeS1ijG223rJzHcC2rmL~QIxFF2YoLo1WCmYDcD8-nX0gZMx5rimUNayzhhyT~xomePWKja8AgVNyZB2mRylMmrnHY2TLEwa0hA8O-Zbxua7HHy~YjRHIQLrp0otYOrfc~6YORQCt~sTfvyg0xsW-WJQ~BbNyQ~MLpjWW8dlcq8-kXBA2RguRANR42nBEEL2nrh56pWnL9ybnpg8rE1AW5MvSVFqGyKfMlH4DVb~z4PAnsRIjUCsudChG-79fL6wd-wEh~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="G-tailed DNA end efficiently acquired GAOs of different lengths. G-tailed FAM70 DNA (G480%G520%) was reacted with GAOs SA725 to SA732 of different lengths for 5 min and then purified using DNA purification columns. The purification product was further subjected to blunting by T4 DNA polymerase and then analysed. Asterisks indicate the concatenated products, and the arrow indicates the peaks of the original size (70 nucleotides)." path-from-xml="dsy018f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671418/dsy018f7.tif" path-from-xml="dsy018f7.tif" target="_blank">View large</a><a section="122671418" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/dsy018f7.png?Expires=2147483647&Signature=yfJ7tCoUmHnUeD~YtOKuGsmDxr4AlmfFTmfM6JmfjwfG~YvWS~Ko7JSdMgUE0pA2UxVfUXkRX5iD1KM4K9XYVad2jLxUX4pVQKkCSRmARA9J9S~Go7LD6TMRfzJtSXO5T5LrByBT5xKTpg0Rd0JPMjvk5ct-d~dGAgT6nrhESOtS6GiGmnwR0vO6scy1Dd5OVjPtXS08r3GqoCfN68m4t7BC2Jq7lRWb8Oh6p6X5AN18HZW3El5QtqSHAaw9-s~eW15BkhmPm6-yJJ3MtDOzQiWkek5bjVWhCRMtzNnwY28pcw~fyBcT~AyrkH90lKjLRk8AUrLLIpNcoCVtVD5sMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671418&ar=5035914&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy018f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>G-tailed DNA end efficiently acquired GAOs of different lengths. G-tailed FAM70 DNA (G4<sup>80%</sup>G5<sup>20%</sup>) was reacted with GAOs SA725 to SA732 of different lengths for 5 min and then purified using DNA purification columns. The purification product was further subjected to blunting by T4 DNA polymerase and then analysed. Asterisks indicate the concatenated products, and the arrow indicates the peaks of the original size (70 nucleotides).</p></div></div><div content-id="dsy018-F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/m_dsy018f7.png?Expires=2147483647&Signature=fQGlK3K35iMKaa7JMTTLWu5fAELdeMiHPt2qR08uiu5y6ryQE4jCn85VmwDNpaL6n3y4~iNku211YhPNeS1ijG223rJzHcC2rmL~QIxFF2YoLo1WCmYDcD8-nX0gZMx5rimUNayzhhyT~xomePWKja8AgVNyZB2mRylMmrnHY2TLEwa0hA8O-Zbxua7HHy~YjRHIQLrp0otYOrfc~6YORQCt~sTfvyg0xsW-WJQ~BbNyQ~MLpjWW8dlcq8-kXBA2RguRANR42nBEEL2nrh56pWnL9ybnpg8rE1AW5MvSVFqGyKfMlH4DVb~z4PAnsRIjUCsudChG-79fL6wd-wEh~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="G-tailed DNA end efficiently acquired GAOs of different lengths. G-tailed FAM70 DNA (G480%G520%) was reacted with GAOs SA725 to SA732 of different lengths for 5 min and then purified using DNA purification columns. The purification product was further subjected to blunting by T4 DNA polymerase and then analysed. Asterisks indicate the concatenated products, and the arrow indicates the peaks of the original size (70 nucleotides)." path-from-xml="dsy018f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671418/dsy018f7.tif" path-from-xml="dsy018f7.tif" target="_blank">View large</a><a section="122671418" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/dsy018f7.png?Expires=2147483647&Signature=yfJ7tCoUmHnUeD~YtOKuGsmDxr4AlmfFTmfM6JmfjwfG~YvWS~Ko7JSdMgUE0pA2UxVfUXkRX5iD1KM4K9XYVad2jLxUX4pVQKkCSRmARA9J9S~Go7LD6TMRfzJtSXO5T5LrByBT5xKTpg0Rd0JPMjvk5ct-d~dGAgT6nrhESOtS6GiGmnwR0vO6scy1Dd5OVjPtXS08r3GqoCfN68m4t7BC2Jq7lRWb8Oh6p6X5AN18HZW3El5QtqSHAaw9-s~eW15BkhmPm6-yJJ3MtDOzQiWkek5bjVWhCRMtzNnwY28pcw~fyBcT~AyrkH90lKjLRk8AUrLLIpNcoCVtVD5sMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671418&ar=5035914&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy018f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>G-tailed DNA end efficiently acquired GAOs of different lengths. G-tailed FAM70 DNA (G4<sup>80%</sup>G5<sup>20%</sup>) was reacted with GAOs SA725 to SA732 of different lengths for 5 min and then purified using DNA purification columns. The purification product was further subjected to blunting by T4 DNA polymerase and then analysed. Asterisks indicate the concatenated products, and the arrow indicates the peaks of the original size (70 nucleotides).</p></div></div>                            <h3 scrollto-destination=122671419 id="122671419" class="section-title" >3.9. Concentrations of G-tailed substrate and efficiencies</h3>
<p>A series of concentrations of a G-tailed DNA ranging from 0.2 nM (2 fmol in 10 µl reaction) to 5 nM (50 fmol in 10 µl reaction) was reacted with 0.5 or 2 pmol of GAO, for 2 min, and the reaction products were analysed. For all the samples, we observed only CIS reaction product peaks, demonstrating that the CIS reaction could be carried out efficiently under a wide range of DNA concentrations (<a reveal-id="dsy018-F8" data-open="dsy018-F8" class="link link-reveal link-table xref-fig">Fig. 8</a>). </p>                        <a id="122671421" scrollto-destination="122671421"></a>
<div id="dsy018-F8" data-id="dsy018-F8" class="fig fig-section"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/m_dsy018f8.png?Expires=2147483647&Signature=USPRMJ8M8mqwql4fEoONDve0zNSzr8Vy8tAM9EDcWKhcj2At4xuyu~ImkwCLz~QlgZNnJzUcezeyxKXHp3ZqzTHDPjqKS7LDK~J4kcQ2B8zYGPGZ7bVi0YZQLU3TiIg7ybhXsFyjaPwiJcgq1V2iXI7UUjy~19N-CF7PgIzK6acnGwjU9TAXltQU4agjMtGsGCa8Pzfxy0JmPeipzd1AXp2tuOOORry4SSwszRdFEUpDUvHcqn4fV2oObs1~BiY9xtT-b8BwQ5my5Mmd1qU8admLU5L0AbGcIwKgmGf1h1OtbYqU4sp1pIg6uRlehCFPWr43hdMFabb5t6U16vNIrw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CIS-reactions conducted under different substrate DNA concentrations. (a) FAM70 DNA and FAM70 DNA subjected for G-tailing (G450%G546%G64%) used as a substrate in panels (b) and (c). In total, 2–50 fmol (0.2–5 nM) of G-tailed FAM70 DNA were reacted with 2 pmol (b) or 0.5 pmol (c) of GAO SA698, which carries N9D2 representing a molecular identifier (N: A or C or G or T; D: A or G or T) for 2 min and then analysed. In panels (b) and (c), electropherograms were y-axis scaled to enable to see the absence of signals from unreacted CIS reaction substrate. Three panels share the same x-axis." path-from-xml="dsy018f8.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671421/dsy018f8.tif" path-from-xml="dsy018f8.tif" target="_blank">View large</a><a section="122671421" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/dsy018f8.png?Expires=2147483647&Signature=JDidp6A8AihmRmHhP6tNyKRe41Ene6Kf-X6Jn2adQlt5yG9WgBUamzVfIhWBlUcYjnNT~mm8uxod-JVfo~3Z~e0~EZYkuJThyOtq4YHqyUO5yefOV5MdSPNHtt1k-HBOlCUq2hu9DupAHa1juqVaOOndk8lQgHoeplVliniAAbfZU6qr-USrGiS7JiVNKpJ4ZTflIMOc-pk1RowvSg4LdFmZpCZPVrku4WPysEl1Y8w6XaeURcyAK2DPLM4DWvmDI2iktXTeDcm91MecZC04G-AnJtsH2cl5X6fu7p0opli61xlhcN7BtLmapCaIu9punkn3YTtubEW1HESFppRVFw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671421&ar=5035914&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy018f8.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>CIS-reactions conducted under different substrate DNA concentrations. (a) FAM70 DNA and FAM70 DNA subjected for G-tailing (G4<sup>50%</sup>G5<sup>46%</sup>G6<sup>4%</sup>) used as a substrate in panels (b) and (c). In total, 2–50 fmol (0.2–5 nM) of G-tailed FAM70 DNA were reacted with 2 pmol (b) or 0.5 pmol (c) of GAO SA698, which carries N9D2 representing a molecular identifier (N: A or C or G or T; D: A or G or T) for 2 min and then analysed. In panels (b) and (c), electropherograms were <em>y</em>-axis scaled to enable to see the absence of signals from unreacted CIS reaction substrate. Three panels share the same <em>x</em>-axis.</p></div></div><div content-id="dsy018-F8" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/m_dsy018f8.png?Expires=2147483647&Signature=USPRMJ8M8mqwql4fEoONDve0zNSzr8Vy8tAM9EDcWKhcj2At4xuyu~ImkwCLz~QlgZNnJzUcezeyxKXHp3ZqzTHDPjqKS7LDK~J4kcQ2B8zYGPGZ7bVi0YZQLU3TiIg7ybhXsFyjaPwiJcgq1V2iXI7UUjy~19N-CF7PgIzK6acnGwjU9TAXltQU4agjMtGsGCa8Pzfxy0JmPeipzd1AXp2tuOOORry4SSwszRdFEUpDUvHcqn4fV2oObs1~BiY9xtT-b8BwQ5my5Mmd1qU8admLU5L0AbGcIwKgmGf1h1OtbYqU4sp1pIg6uRlehCFPWr43hdMFabb5t6U16vNIrw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CIS-reactions conducted under different substrate DNA concentrations. (a) FAM70 DNA and FAM70 DNA subjected for G-tailing (G450%G546%G64%) used as a substrate in panels (b) and (c). In total, 2–50 fmol (0.2–5 nM) of G-tailed FAM70 DNA were reacted with 2 pmol (b) or 0.5 pmol (c) of GAO SA698, which carries N9D2 representing a molecular identifier (N: A or C or G or T; D: A or G or T) for 2 min and then analysed. In panels (b) and (c), electropherograms were y-axis scaled to enable to see the absence of signals from unreacted CIS reaction substrate. Three panels share the same x-axis." path-from-xml="dsy018f8.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671421/dsy018f8.tif" path-from-xml="dsy018f8.tif" target="_blank">View large</a><a section="122671421" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy018/1/dsy018f8.png?Expires=2147483647&Signature=JDidp6A8AihmRmHhP6tNyKRe41Ene6Kf-X6Jn2adQlt5yG9WgBUamzVfIhWBlUcYjnNT~mm8uxod-JVfo~3Z~e0~EZYkuJThyOtq4YHqyUO5yefOV5MdSPNHtt1k-HBOlCUq2hu9DupAHa1juqVaOOndk8lQgHoeplVliniAAbfZU6qr-USrGiS7JiVNKpJ4ZTflIMOc-pk1RowvSg4LdFmZpCZPVrku4WPysEl1Y8w6XaeURcyAK2DPLM4DWvmDI2iktXTeDcm91MecZC04G-AnJtsH2cl5X6fu7p0opli61xlhcN7BtLmapCaIu9punkn3YTtubEW1HESFppRVFw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671421&ar=5035914&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy018f8.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>CIS-reactions conducted under different substrate DNA concentrations. (a) FAM70 DNA and FAM70 DNA subjected for G-tailing (G4<sup>50%</sup>G5<sup>46%</sup>G6<sup>4%</sup>) used as a substrate in panels (b) and (c). In total, 2–50 fmol (0.2–5 nM) of G-tailed FAM70 DNA were reacted with 2 pmol (b) or 0.5 pmol (c) of GAO SA698, which carries N9D2 representing a molecular identifier (N: A or C or G or T; D: A or G or T) for 2 min and then analysed. In panels (b) and (c), electropherograms were <em>y</em>-axis scaled to enable to see the absence of signals from unreacted CIS reaction substrate. Three panels share the same <em>x</em>-axis.</p></div></div>                            <h2 scrollto-destination=122671422 id="122671422" class="section-title" >4. Discussion</h2>
<p>In this study, upon our finding of tailing enhancers,<a reveal-id="dsy018-B10" data-open="dsy018-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> we optimized the CIS reaction, which has been reported but could not be conducted at a high efficiency. By applying the best set of conditions, adaptor DNA sequence could be appended to the double-stranded blunt DNA end at efficiency close to 100% that no other DNA-handling techniques have achieved.</p>                            <h3 scrollto-destination=122671424 id="122671424" class="section-title" >4.1. Accompanying peaks</h3>
<p>When the CIS reaction products were subjected to treatment with T4 DNA polymerase, one major peak was associated with the peaks of products that were a few nucleotides shorter or one nucleotide longer (<a reveal-id="dsy018-F7" data-open="dsy018-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>). The shorter peaks might represent insufficient extension by T4 DNA polymerase. To support this idea, although at a lower level, 3′ recessed ends were observed for the product of the PCR reaction using KOD DNA polymerase, which is supposed to generate blunt ends (see the size control in <a reveal-id="dsy018-F1" data-open="dsy018-F1" class="link link-reveal link-table xref-fig">Fig. 1a</a>). The other possibility is the ‘erosion’ of the GAO 3′ end during storage (we stored GAOs at 4°C in TE buffer). To assess this possibility, after long-term storage at 4°C for 6 months, some, although significant, fractions of the GAO were found to have undergone erosion of a few nucleotides, which was determined by measuring the mass spectra of the stored oligonucleotides (data not shown). Another possibility is that a proportion of the GAO was not synthesized as its full length. In contrast, the longer product might be the result of the gapped-annealing of GAO that increased the size of the product by the length of the gap. Although the gaps between GAO and dsDNA have been shown to decrease the reaction efficiency considerably,<a reveal-id="dsy018-B4" data-open="dsy018-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> extended G-tail of c.a. four nucleotides long might stabilize the gapped annealing.</p>                            <h3 scrollto-destination=122671426 id="122671426" class="section-title" >4.2. Importance of tailing enhancers</h3>
<p>In our previous study, we identified enhancers for C-, G-, and T-tailing reactions.<a reveal-id="dsy018-B10" data-open="dsy018-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> The results of this study clearly showed that the length of the tail is important for the efficiency of the CIS reaction (<a reveal-id="dsy018-F3" data-open="dsy018-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). Because MMLV-RT can append up to four dAMPs as a tail even in the absence of an enhancer,<a reveal-id="dsy018-B9" data-open="dsy018-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> when the G-tail length of the target DNA is short and CIS reaction was started in the presence of dNTPs, MMLV-RT can append the G-tail with a few additional deoxyadenosine monophosphates, resulting in a mosaic tail that is not complementary, and hence not reactive to the GAO that carry Cs at it 3′ end. In this regard, the addition of four or more Gs is desirable because no additional A would be further appended to such a long G-tail. In addition, the removal of enhancers as well as Mn<sup>2+</sup> ions, which are also enhancers,<a reveal-id="dsy018-B8" data-open="dsy018-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsy018-B9" data-open="dsy018-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsy018-B13" data-open="dsy018-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> after the tailing reaction is important for a successful CIS reaction.</p><p>The removal of enhancers also seems to be important for preventing concatemer formation. Concatemer formation was noted in the experiment shown in <a reveal-id="dsy018-F2" data-open="dsy018-F2" class="link link-reveal link-table xref-fig">Fig. 2c</a> but not in that shown in <a reveal-id="dsy018-F5" data-open="dsy018-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>, and this difference might have resulted from the different methods used to purify DNA after the tailing reaction (phenol/chloroform/isoamylalcohol extraction and ethanol precipitation in <a reveal-id="dsy018-F2" data-open="dsy018-F2" class="link link-reveal link-table xref-fig">Fig. 2</a> and a DNA purification column in <a reveal-id="dsy018-F5" data-open="dsy018-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>).</p>                            <h3 scrollto-destination=122671429 id="122671429" class="section-title" >4.3. Use of a GAO with 3′ ribonucleotides</h3>
<p>In this study, we showed that even with a long tail of Gs up to four nucleotides long, a GAO carrying 3′ ribonucleotides reacted more efficiently than a GAO carrying the deoxyribonucleotidyl counterpart. However, in practical use, the advantage of a ribonucleotidyl GAO seems to be limited, especially when extensively G-tailed DNA is subjected to the CIS reaction. In <a reveal-id="dsy018-F4" data-open="dsy018-F4" class="link link-reveal link-table xref-fig">Fig. 4a</a>, substrates G-tailed in the absence of enhancer (mostly +G2) was used, and the rG-carrying GAO exhibited only ∼5% higher efficiency when compared with GAO that carry dG. In addition, 98% of reaction yield can be achieved even when deoxyribonucleotidyl GAOs are used when extensively G-taild DNA (mostly +G4) was a substrate (<a reveal-id="dsy018-F7" data-open="dsy018-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>). The chemical instability as well as the increased cost for ribonucleotidyl GAO preparation also supports the use of deoxyribonucleotidyl GAO.</p><p>From a mechanistic point of view, the 3′ ribonucleotides of the GAO appear to not only increase the thermostability with the deoxyribonucleotidyl tail but also stably interact with MMLV-RT, thereby increasing the efficiency. This idea is consistent with the fact that the physiological substrate of MMLV-RT is not DNA but RNA.</p>                            <h3 scrollto-destination=122671432 id="122671432" class="section-title" >4.4. Effect of the GAO sequence on CIS reaction efficiency</h3>
<p>We used two types of GAOs: one represents the adaptor sequence of the Illumina sequencer (used here except in <a reveal-id="dsy018-F4" data-open="dsy018-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>)<a reveal-id="dsy018-B6" data-open="dsy018-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> and the other represents single-stranded DNAs as used by Chenchik et al.<a reveal-id="dsy018-B6" data-open="dsy018-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> (used in <a reveal-id="dsy018-F4" data-open="dsy018-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>; SA670-SA673). Not only the 3′ end of the GAO but also the entire nucleotide sequence affect the CIS reaction efficiency (for example, compare the SA673 data in <a reveal-id="dsy018-F4" data-open="dsy018-F4" class="link link-reveal link-table xref-fig">Fig. 4b</a> and the SA574 data in <a reveal-id="dsy018-F3" data-open="dsy018-F3" class="link link-reveal link-table xref-fig">Fig. 3a</a>). The structure that each GAO can have, e.g. dimers formed between two GAO molecules at the reaction temperature, may affect the efficiency. We also observed a low CIS reaction efficiency with a GAO with six Gs at its 3′ end, although the reason for this was unclear (data not shown). Keeping these observations in mind, each GAO to be incorporated should be examined prior to practical use in a system like the one we used in this study.</p>                            <h3 scrollto-destination=122671434 id="122671434" class="section-title" >4.5. Use of a biotin-labelled GAO and streptavidin to prevent concatenation</h3>
<p>Concatemer formation on the cDNA ends of template-switching oligonucleotides has been demonstrated and proposed as a reason for the high background and low cDNA yield, especially when a small amount of RNA is used for cDNA synthesis.<a reveal-id="dsy018-B8" data-open="dsy018-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> In their study, Kapteyn et al. demonstrated the usefulness of nucleotide isomers that form non-standard base pairs to reduce concatemer formation. Our findings indicated that more common 5′ end modifications on the GAO, e.g. biotinylation, prevented concatenation and that combining with streptavidin further prevented additional tailing (<a reveal-id="dsy018-F5" data-open="dsy018-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>), suggesting that steric hindrance is sufficient to prevent concatenation.</p>                            <h3 scrollto-destination=122671436 id="122671436" class="section-title" >4.6. CIS reaction efficiency</h3>
<p>When we analysed the CIS reaction products, we often observed no peaks representing the initial DNA targets (see <a reveal-id="dsy018-F1" data-open="dsy018-F1" class="link link-reveal link-table xref-fig">Fig. 1a</a> for an example). However, after T4 DNA polymerase treatment, a small but distinct peak with the size of the untailed DNA substrate was always observed. This peak indicated that some fractions of MMLV-RT failed to complete the synthesis of the strand complementary to the GAO, leaving an extended tail resulting from the incomplete synthesis that was removed by T4 DNA polymerase. This failure might have been because the dGTP solution used for tailing was not 100% pure, resulting in a base-mismatch between the tail and 3′ end of the GAO. In support of this conjecture, RT’s high error rate has been suggested to be due to its efficiency at extending mismatches.<a reveal-id="dsy018-B14" data-open="dsy018-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsy018-B15" data-open="dsy018-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a></p><p>Nonetheless, CIS reaction efficiency is very high and might replace the traditional single-A-tailing followed by adaptor DNA ligation by DNA ligase, for which efficiencies were analysed by droplet digital PCR assays and reported to be 3–20%.<a reveal-id="dsy018-B16" data-open="dsy018-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a></p>                            <h2 scrollto-destination=122671439 id="122671439" class="section-title" >5. Conclusions</h2>
<p>Adaptor ligation to DNA ends is used as a fundamental DNA technique in library preparations for PacBio,<a reveal-id="dsy018-B17" data-open="dsy018-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> Illumina,<a reveal-id="dsy018-B18" data-open="dsy018-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> and NanoPore<a reveal-id="dsy018-B19" data-open="dsy018-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> sequencing technologies for various research objectives such as <em>de novo</em> genome sequencing, re-sequencing for mutation mapping, and transposon mutant library analysis.<a reveal-id="dsy018-B20" data-open="dsy018-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Our method termed the CIS reaction is advantageous over existing methods in that the CIS reaction can label each DNA at a high efficiency, thereby facilitating analyses for which only limited amount of DNA is available. In addition, when using the CIS reaction with a GAO carrying a random sequence, it is easy to label each DNA fragment with a unique molecular identifier<a reveal-id="dsy018-B21 dsy018-B22 dsy018-B23" data-open="dsy018-B21 dsy018-B22 dsy018-B23" class="link link-ref link-reveal xref-bibr"><sup>21–23</sup></a> before the PCR-amplification step, thereby enabling the quantitative processing of NGS reads.</p>                            <h2 scrollto-destination=122671441 id="122671441" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are grateful to Shouta Nonoyama for experimental support. We also thank the anonymous reviewers for valuable comments.</p>                            <h3 scrollto-destination=122671443 id="122671443" class="section-title" >Funding</h3>
<p>This work was supported by Grants-in-Aid from the Ministry of Education, Culture, Sports, Science, and Technology, Japan (15H04471 and 17H03781) and by a grant from the Institute for Fermentation, Osaka (IFO).</p>                            <h3 scrollto-destination=122671445 id="122671445" class="section-title" >Conflict of interest</h3>
<p>Tohoku University has filed patent applications on some techniques described in this study.</p>                            <h2 scrollto-destination=122671447 id="122671447" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsy018-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy018-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Maniatists</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Fritsch</div> <div class="given-names">E. F.</div></div>, <div class="name string-name"><div class="surname">Sambrook</div> <div class="given-names">J.</div></div></div> <div class="year">1982</div>, <div class="source">Molecular Cloning: A Laboratory Manual</div> . 
<div class="publisher-name">Cold Spring Harbor Laboratory</div>, 
<div class="publisher-loc">Cold Spring Harbor, NY, USA</div>.</div></div></div></div><div content-id="dsy018-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy018-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shinnick</div> <div class="given-names">T. M.</div></div>, <div class="name string-name"><div class="surname">Lerner</div> <div class="given-names">R. A.</div></div>, <div class="name string-name"><div class="surname">Sutcliffe</div> <div class="given-names">J. G.</div></div></div> <div class="year">1981</div>, 
<div class="article-title">Nucleotide sequence of Moloney murine leukaemia virus</div>, <div class="source">Nature</div> , <div class="volume">293</div>, <div class="fpage">543</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleotide%20sequence%20of%20Moloney%20murine%20leukaemia%20virus&amp;author=T.%20M.%20Shinnick&amp;author=R.%20A.%20Lerner&amp;author=J.%20G.%20Sutcliffe&amp;publication_year=1981&amp;journal=Nature&amp;volume=293&amp;pages=543-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/293543a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F293543a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F293543a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6169994" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy018-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy018-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Oz-Gleenberg</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Herschhorn</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Hizi</div> <div class="given-names">A.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Reverse transcriptases can clamp together nucleic acids strands with two complementary bases at their 3′-termini for initiating DNA synthesis</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">39</div>, <div class="fpage">1042</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reverse%20transcriptases%20can%20clamp%20together%20nucleic%20acids%20strands%20with%20two%20complementary%20bases%20at%20their%203%E2%80%B2-termini%20for%20initiating%20DNA%20synthesis&amp;author=I.%20Oz-Gleenberg&amp;author=A.%20Herschhorn&amp;author=A.%20Hizi&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res&amp;volume=39&amp;pages=1042-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq786" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq786" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq786"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20876692" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy018-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy018-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Oz-Gleenberg</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Herzig</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Voronin</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Hizi</div> <div class="given-names">A.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Substrate variations that affect the nucleic acid clamp activity of reverse transcriptases</div>, <div class="source">FEBS J</div> ., <div class="volume">279</div>, <div class="fpage">1894</div>–<div class="lpage">903</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Substrate%20variations%20that%20affect%20the%20nucleic%20acid%20clamp%20activity%20of%20reverse%20transcriptases&amp;author=I.%20Oz-Gleenberg&amp;author=E.%20Herzig&amp;author=N.%20Voronin&amp;author=A.%20Hizi&amp;publication_year=2012&amp;journal=FEBS%20J&amp;volume=279&amp;pages=1894-903" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1742-4658.2012.08570.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1742-4658.2012.08570.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1742-4658.2012.08570.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22443410" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy018-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy018-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Oz-Gleenberg</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Hizi</div> <div class="given-names">A.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Strand selections resulting from the combined template-independent DNA synthesis and clamp activities of HIV-1 reverse transcriptase</div>, <div class="source">Biochem. Biophys. Res. Commun</div> ., <div class="volume">408</div>, <div class="fpage">482</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strand%20selections%20resulting%20from%20the%20combined%20template-independent%20DNA%20synthesis%20and%20clamp%20activities%20of%20HIV-1%20reverse%20transcriptase&amp;author=I.%20Oz-Gleenberg&amp;author=A.%20Hizi&amp;publication_year=2011&amp;journal=Biochem.%20Biophys.%20Res.%20Commun&amp;volume=408&amp;pages=482-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2011.04.063" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2011.04.063" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2011.04.063"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21527243" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy018-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy018-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chenchik</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">Y. Y.</div></div>, <div class="name string-name"><div class="surname">Diatchenko</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Hill</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Siebert</div> <div class="given-names">P. D.</div></div></div> <div class="year">1998</div>, Generation and use of high quality cDNA from small amounts of total RNA by SMART PCR. In: <div class="person-group"><div class="name string-name"><div class="surname">Siebert</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Larrick</div> <div class="given-names">J.</div></div></div>, eds., <div class="source">Gene Cloning and Analysis by RT-PCR</div> , pp. <div class="fpage">305</div>–<div class="lpage">19</div>, 
<div class="publisher-name">Eaton Publishing</div>, 
<div class="publisher-loc">Natick, MA, USA</div>.</div></div></div></div><div content-id="dsy018-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy018-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">Y. Y.</div></div>, <div class="name string-name"><div class="surname">Machleder</div> <div class="given-names">E. M.</div></div>, <div class="name string-name"><div class="surname">Chenchik</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Siebert</div> <div class="given-names">P. D.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Reverse transcriptase template switching: a SMART approach for full-length cDNA library construction</div>, <div class="source">BioTechniques</div> , <div class="volume">30</div>, <div class="fpage">892</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reverse%20transcriptase%20template%20switching%3A%20a%20SMART%20approach%20for%20full-length%20cDNA%20library%20construction&amp;author=Y.%20Y.%20Zhu&amp;author=E.%20M.%20Machleder&amp;author=A.%20Chenchik&amp;author=R.%20Li&amp;author=P.%20D.%20Siebert&amp;publication_year=2001&amp;journal=BioTechniques&amp;volume=30&amp;pages=892-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2144/01304pf02" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2144%2F01304pf02" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2144%2F01304pf02"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11314272" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy018-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy018-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kapteyn</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">He</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">McDowell</div> <div class="given-names">E. T.</div></div>, <div class="name string-name"><div class="surname">Gang</div> <div class="given-names">D. R.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Incorporation of non-natural nucleotides into template-switching oligonucleotides reduces background and improves cDNA synthesis from very small RNA samples</div>, <div class="source">BMC Genomics</div> , <div class="volume">11</div>, <div class="fpage">413</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Incorporation%20of%20non-natural%20nucleotides%20into%20template-switching%20oligonucleotides%20reduces%20background%20and%20improves%20cDNA%20synthesis%20from%20very%20small%20RNA%20samples&amp;author=J.%20Kapteyn&amp;author=R.%20He&amp;author=E.%20T.%20McDowell&amp;author=D.%20R.%20Gang&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=413" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-413" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-413" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-413"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20598146" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy018-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy018-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kapteyn</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">He</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">McDowell</div> <div class="given-names">E. T.</div></div>, <div class="name string-name"><div class="surname">Gang</div> <div class="given-names">D. R.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Efficient N-tailing of blunt DNA ends by Moloney murine leukemia virus reverse transcriptase</div>, <div class="source">Sci. Rep</div> ., <div class="volume">11</div>, <div class="fpage">41769</div>.</div></div></div></div><div content-id="dsy018-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy018-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ohtsubo</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Nagata</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Tsuda</div> <div class="given-names">M.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Compounds that enhance the tailing activity of Moloney murine leukemia virus reverse transcriptase</div>, <div class="source">Sci. Rep</div> ., <div class="volume">7</div>, <div class="fpage">6520</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Compounds%20that%20enhance%20the%20tailing%20activity%20of%20Moloney%20murine%20leukemia%20virus%20reverse%20transcriptase&amp;author=Y.%20Ohtsubo&amp;author=Y.%20Nagata&amp;author=M.%20Tsuda&amp;publication_year=2017&amp;journal=Sci.%20Rep&amp;volume=7&amp;pages=6520" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/s41598-017-04765-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fs41598-017-04765-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fs41598-017-04765-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28747695" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy018-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy018-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">M. Y.</div></div>, <div class="name string-name"><div class="surname">Clark</div> <div class="given-names">S. E.</div></div>, <div class="name string-name"><div class="surname">Gomez-Sanchez</div> <div class="given-names">C. E.</div></div></div> <div class="year">1995</div>, 
<div class="article-title">Universal cloning method by TA strategy</div>, <div class="source">BioTechniques</div> , <div class="volume">19</div>, <div class="fpage">34</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Universal%20cloning%20method%20by%20TA%20strategy&amp;author=M.%20Y.%20Zhou&amp;author=S.%20E.%20Clark&amp;author=C.%20E.%20Gomez-Sanchez&amp;publication_year=1995&amp;journal=BioTechniques&amp;volume=19&amp;pages=34-5" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7669292" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy018-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy018-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Harbers</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kato</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">de Hoon</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hayashizaki</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Carninci</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Plessy</div> <div class="given-names">C.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Comparison of RNA- or LNA-hybrid oligonucleotides in template-switching reactions for high-speed sequencing library preparation</div>, <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">665</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20RNA-%20or%20LNA-hybrid%20oligonucleotides%20in%20template-switching%20reactions%20for%20high-speed%20sequencing%20library%20preparation&amp;author=M.%20Harbers&amp;author=S.%20Kato&amp;author=M.%20de%20Hoon&amp;author=Y.%20Hayashizaki&amp;author=P.%20Carninci&amp;author=C.%20Plessy&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=665" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-665" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-665" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-665"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24079827" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy018-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy018-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schmidt</div> <div class="given-names">W. M.</div></div>, <div class="name string-name"><div class="surname">Mueller</div> <div class="given-names">M. W.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">CapSelect: a highly sensitive method for 5′ CAP-dependent enrichment of full-length cDNA in PCR-mediated analysis of mRNAs</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">27</div>, <div class="fpage">31e.</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CapSelect%3A%20a%20highly%20sensitive%20method%20for%205%E2%80%B2%20CAP-dependent%20enrichment%20of%20full-length%20cDNA%20in%20PCR-mediated%20analysis%20of%20mRNAs&amp;author=W.%20M.%20Schmidt&amp;author=M.%20W.%20Mueller&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res&amp;volume=27&amp;pages=31e." target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.21.e31" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.21.e31" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.21.e31"> </span></div> </div></div></div></div></div><div content-id="dsy018-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy018-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kulpa</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Topping</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Telesnitsky</div> <div class="given-names">A.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">Determination of the site of first strand transfer during Moloney murine leukemia virus reverse transcription and identification of strand transfer-associated reverse transcriptase errors</div>, <div class="source">EMBO J</div> ., <div class="volume">16</div>, <div class="fpage">856</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Determination%20of%20the%20site%20of%20first%20strand%20transfer%20during%20Moloney%20murine%20leukemia%20virus%20reverse%20transcription%20and%20identification%20of%20strand%20transfer-associated%20reverse%20transcriptase%20errors&amp;author=D.%20Kulpa&amp;author=R.%20Topping&amp;author=A.%20Telesnitsky&amp;publication_year=1997&amp;journal=EMBO%20J&amp;volume=16&amp;pages=856-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/emboj/16.4.856" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Femboj%2F16.4.856" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Femboj%2F16.4.856"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9049314" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy018-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy018-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Perrino</div> <div class="given-names">F. W.</div></div>, <div class="name string-name"><div class="surname">Preston</div> <div class="given-names">B. D.</div></div>, <div class="name string-name"><div class="surname">Sandell</div> <div class="given-names">L. L.</div></div>, <div class="name string-name"><div class="surname">Loeb</div> <div class="given-names">L. A.</div></div></div> <div class="year">1989</div>, 
<div class="article-title">Extension of mismatched 3′ termini of DNA is a major determinant of the infidelity of human immunodeficiency virus type 1 reverse transcriptase</div>, <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">86</div>, <div class="fpage">8343</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extension%20of%20mismatched%203%E2%80%B2%20termini%20of%20DNA%20is%20a%20major%20determinant%20of%20the%20infidelity%20of%20human%20immunodeficiency%20virus%20type%201%20reverse%20transcriptase&amp;author=F.%20W.%20Perrino&amp;author=B.%20D.%20Preston&amp;author=L.%20L.%20Sandell&amp;author=L.%20A.%20Loeb&amp;publication_year=1989&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=86&amp;pages=8343-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.86.21.8343" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.86.21.8343" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.86.21.8343"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2479023" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy018-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy018-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Aigrain</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Gu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Quail</div> <div class="given-names">M. A.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Quantitation of next generation sequencing library preparation protocol efficiencies using droplet digital PCR assays - a systematic comparison of DNA library preparation kits for Illumina sequencing</div>, <div class="source">BMC Genomics</div> , <div class="volume">17</div>, <div class="fpage">458</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitation%20of%20next%20generation%20sequencing%20library%20preparation%20protocol%20efficiencies%20using%20droplet%20digital%20PCR%20assays%20-%20a%20systematic%20comparison%20of%20DNA%20library%20preparation%20kits%20for%20Illumina%20sequencing&amp;author=L.%20Aigrain&amp;author=Y.%20Gu&amp;author=M.%20A.%20Quail&amp;publication_year=2016&amp;journal=BMC%20Genomics&amp;volume=17&amp;pages=458" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-016-2757-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-016-2757-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-016-2757-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27297323" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy018-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy018-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rhoads</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Au</div> <div class="given-names">K. F.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">PacBio sequencing and its applications</div>, <div class="source">Genomics Proteomics Bioinformatics</div> , <div class="volume">13</div>, <div class="fpage">278</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PacBio%20sequencing%20and%20its%20applications&amp;author=A.%20Rhoads&amp;author=K.%20F.%20Au&amp;publication_year=2015&amp;journal=Genomics%20Proteomics%20Bioinformatics&amp;volume=13&amp;pages=278-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gpb.2015.08.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gpb.2015.08.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gpb.2015.08.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26542840" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy018-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy018-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bentley</div> <div class="given-names">D. R.</div></div>, <div class="name string-name"><div class="surname">Balasubramanian</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Swerdlow</div> <div class="given-names">H. P.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Accurate whole human genome sequencing using reversible terminator chemistry</div>, <div class="source">Nature</div> , <div class="volume">456</div>, <div class="fpage">53</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accurate%20whole%20human%20genome%20sequencing%20using%20reversible%20terminator%20chemistry&amp;author=D.%20R.%20Bentley&amp;author=S.%20Balasubramanian&amp;author=H.%20P.%20Swerdlow&amp;publication_year=2008&amp;journal=Nature&amp;volume=456&amp;pages=53-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07517" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07517" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07517"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18987734" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy018-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy018-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Loman</div> <div class="given-names">N. J.</div></div>, <div class="name string-name"><div class="surname">Quick</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Simpson</div> <div class="given-names">J. T.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">A complete bacterial genome assembled de novo using only nanopore sequencing data</div>, <div class="source">Nat. Methods</div> , <div class="volume">12</div>, <div class="fpage">733</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20complete%20bacterial%20genome%20assembled%20de%20novo%20using%20only%20nanopore%20sequencing%20data&amp;author=N.%20J.%20Loman&amp;author=J.%20Quick&amp;author=J.%20T.%20Simpson&amp;publication_year=2015&amp;journal=Nat.%20Methods&amp;volume=12&amp;pages=733-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.3444" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.3444" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.3444"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26076426" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy018-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy018-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kwon</div> <div class="given-names">Y. M.</div></div>, <div class="name string-name"><div class="surname">Ricke</div> <div class="given-names">S. C.</div></div>, <div class="name string-name"><div class="surname">Mandal</div> <div class="given-names">R. K.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Transposon sequencing: methods and expanding applications</div>, <div class="source">Appl. Microbiol. Biotechnol</div> ., <div class="volume">100</div>, <div class="fpage">31</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transposon%20sequencing%3A%20methods%20and%20expanding%20applications&amp;author=Y.%20M.%20Kwon&amp;author=S.%20C.%20Ricke&amp;author=R.%20K.%20Mandal&amp;publication_year=2016&amp;journal=Appl.%20Microbiol.%20Biotechnol&amp;volume=100&amp;pages=31-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-015-7037-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-015-7037-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-015-7037-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26476650" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy018-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy018-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lundberg</div> <div class="given-names">D. S.</div></div>, <div class="name string-name"><div class="surname">Yourstone</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Mieczkowski</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Jones</div> <div class="given-names">C. D.</div></div>, <div class="name string-name"><div class="surname">Dangl</div> <div class="given-names">J. L.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Practical innovations for high-throughput amplicon sequencing</div>, <div class="source">Nat. Methods</div> , <div class="volume">10</div>, <div class="fpage">999</div>–<div class="lpage">1002</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Practical%20innovations%20for%20high-throughput%20amplicon%20sequencing&amp;author=D.%20S.%20Lundberg&amp;author=S.%20Yourstone&amp;author=P.%20Mieczkowski&amp;author=C.%20D.%20Jones&amp;author=J.%20L.%20Dangl&amp;publication_year=2013&amp;journal=Nat.%20Methods&amp;volume=10&amp;pages=999-1002" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.2634" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2634" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.2634"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23995388" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy018-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy018-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vvedenskaya</div> <div class="given-names">I. O.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Goldman</div> <div class="given-names">S. R.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Massively systematic transcript end readout, “MASTER”: transcription start site selection, transcriptional slippage, and transcript yields</div>, <div class="source">Mol. Cell</div> , <div class="volume">60</div>, <div class="fpage">953</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Massively%20systematic%20transcript%20end%20readout%2C%20%E2%80%9CMASTER%E2%80%9D%3A%20transcription%20start%20site%20selection%2C%20transcriptional%20slippage%2C%20and%20transcript%20yields&amp;author=I.%20O.%20Vvedenskaya&amp;author=Y.%20Zhang&amp;author=S.%20R.%20Goldman&amp;publication_year=2015&amp;journal=Mol.%20Cell&amp;volume=60&amp;pages=953-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2015.10.029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2015.10.029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2015.10.029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26626484" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy018-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy018-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Girardot</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Scholtalbers</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Sauer</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Su</div> <div class="given-names">S. Y.</div></div>, <div class="name string-name"><div class="surname">Furlong</div> <div class="given-names">E. E.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Je, a versatile suite to handle multiplexed NGS libraries with unique molecular identifiers</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">17</div>, <div class="fpage">419</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Je%2C%20a%20versatile%20suite%20to%20handle%20multiplexed%20NGS%20libraries%20with%20unique%20molecular%20identifiers&amp;author=C.%20Girardot&amp;author=J.%20Scholtalbers&amp;author=S.%20Sauer&amp;author=S.%20Y.%20Su&amp;author=E.%20E.%20Furlong&amp;publication_year=2016&amp;journal=BMC%20Bioinformatics&amp;volume=17&amp;pages=419" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12859-016-1284-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12859-016-1284-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12859-016-1284-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27717304" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2018. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
        <div class="wi-editors">
            <div class="al-authors-list">

                    <div class="editor-tag-label">
Editor:
                            <span class="al-author-name">
                                <span id="info-18743871" class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_Article_EditorInformation_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masahira Hattori</div>
            <div class="info-card-role">
                Editor
            </div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masahira+Hattori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hattori M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hattori M&quot;">Google Scholar</a>
            </div>

</div>

                                </span>
                                <a class="linked-name" href="javascript:;">Masahira Hattori</a>
                            </span>
                        <br/>
                    </div>
            </div>
        </div>

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=5035914&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">487</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsy018"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5035914"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f5%2f477%2f5035914">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F5%2F477%2F5035914" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/5035914'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Optimization of single strand DNA incorporation reaction by Moloney murine leukaemia virus reverse transcriptase" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsy018");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f5%2f477%2f5035914">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

