

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":1,"applicationTime":2110,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/5/425/359087","@type":"ScholarlyArticle","name":"Identification of a Novel  PNMA-MS1  Gene in Marsupials Suggests the LTR Retrotransposon-Derived  PNMA  Genes Evolved Differently in Marsupials and Eutherians","datePublished":"2013-05-23","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2013-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst020","keywords":["LTR retrotransposons","<italic>PNMA</italic> family","marsupial-specific genes","mammalian evolution"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/5/425/359087","author":[{"name":"Iwasaki, Sawa","affiliation":"1  Department of Epigenetics, Medical Research Institute ,  Tokyo Medical and Dental University ,  1-5-45 Yushima, Bunkyo-ku, Tokyo 113-8510 ,  Japan    2  School of Health Sciences ,  Tokai University ,  Bohseidai, Isehara, Kanagawa 259-1193 ,  Japan","@type":"Person"},{"name":"Suzuki, Shunsuke","affiliation":"3  Department of Zoology ,  The University of Melbourne ,  Melbourne, Victoria 3010 ,  Australia    4  Epigenomics Division, Faculty of Agriculture, Frontier Agriscience and Technology Center ,  Shinshu University ,  8304 Minami-minowa, Kami-ina, Nagano 399-4598 ,  Japan","@type":"Person"},{"name":"Pelekanos, Matthew","affiliation":"3  Department of Zoology ,  The University of Melbourne ,  Melbourne, Victoria 3010 ,  Australia","@type":"Person"},{"name":"Clark, Helen","affiliation":"3  Department of Zoology ,  The University of Melbourne ,  Melbourne, Victoria 3010 ,  Australia","@type":"Person"},{"name":"Ono, Ryuichi","affiliation":"1  Department of Epigenetics, Medical Research Institute ,  Tokyo Medical and Dental University ,  1-5-45 Yushima, Bunkyo-ku, Tokyo 113-8510 ,  Japan","@type":"Person"},{"name":"Shaw, Geoff","affiliation":"3  Department of Zoology ,  The University of Melbourne ,  Melbourne, Victoria 3010 ,  Australia","@type":"Person"},{"name":"Renfree, Marilyn B.","affiliation":"3  Department of Zoology ,  The University of Melbourne ,  Melbourne, Victoria 3010 ,  Australia","@type":"Person"},{"name":"Kaneko-Ishino, Tomoko","affiliation":"2  School of Health Sciences ,  Tokai University ,  Bohseidai, Isehara, Kanagawa 259-1193 ,  Japan","@type":"Person"},{"name":"Ishino, Fumitoshi","affiliation":"1  Department of Epigenetics, Medical Research Institute ,  Tokyo Medical and Dental University ,  1-5-45 Yushima, Bunkyo-ku, Tokyo 113-8510 ,  Japan    5  Global Center of Excellence Program for International Research Center for Molecular Science in Tooth and Bone Diseases, Tokyo Medical and Dental University ,  1-5-45 Yushima, Bunkyo-ku, Tokyo 113-8510 ,  Japan","@type":"Person"}],"description":"Abstract.  Two major gene families derived from Ty3/Gypsy long terminal repeat (LTR) retrotransposons were recently identified in mammals. The sushi-ichi retrot","pageStart":"425","pageEnd":"436","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/5/1/m_cover.gif?Expires=2147483647&Signature=Lk49EY44ST5kshtHpZ8RK~2R1A7PgJCg08uzNU5E4ewaw5su36z91fGFzyjKSwM7FA8vqYVOBewtYyaa49VDZyPXl9Uk3UXnVDma3euMHlznALvLA3cmgbZS4rUQVsL9pDxMqMdxSm2soKO-Fi0OyE0yb48n7~-liMKL-OGMbIZgQadhhuxmJniONIYojHw~acVvxnAWpMlYOg7NZ41neYIpgkm8f0MVUTZPG1rOzVvSkGGn2mgtSfwCsvVKdAnNTIkShF7ickxRfq678V912uPQ-4xXwjlGVd5JPSDhJq0sRXahBacmVvr-s125nZapbB7I5xsPcGLkoW6cY527iw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Identification of a Novel  PNMA-MS1  Gene in Marsupials Suggests the LTR Retrotransposon-Derived  PNMA  Genes Evolved Differently in Marsupials and Eutherians","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/5/1/m_cover.gif?Expires=2147483647&Signature=Lk49EY44ST5kshtHpZ8RK~2R1A7PgJCg08uzNU5E4ewaw5su36z91fGFzyjKSwM7FA8vqYVOBewtYyaa49VDZyPXl9Uk3UXnVDma3euMHlznALvLA3cmgbZS4rUQVsL9pDxMqMdxSm2soKO-Fi0OyE0yb48n7~-liMKL-OGMbIZgQadhhuxmJniONIYojHw~acVvxnAWpMlYOg7NZ41neYIpgkm8f0MVUTZPG1rOzVvSkGGn2mgtSfwCsvVKdAnNTIkShF7ickxRfq678V912uPQ-4xXwjlGVd5JPSDhJq0sRXahBacmVvr-s125nZapbB7I5xsPcGLkoW6cY527iw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Identification of a Novel  PNMA-MS1  Gene in Marsupials Suggests the LTR Retrotransposon-Derived  PNMA  Genes Evolved Differently in Marsupials and Eutherians" />
    <meta property="og:description" content="Abstract.  Two major gene families derived from Ty3/Gypsy long terminal repeat (LTR) retrotransposons were recently identified in mammals. The sushi-ichi retrot" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/5/425/359087" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-05-23" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Iwasaki, Sawa" /><meta name="citation_author_institution" content="Department of Epigenetics, Medical Research Institute, Tokyo Medical and Dental University, 1-5-45 Yushima, Bunkyo-ku, Tokyo 113-8510, Japan" /><meta name="citation_author_institution" content="School of Health Sciences, Tokai University, Bohseidai, Isehara, Kanagawa 259-1193, Japan" /><meta name="citation_author" content="Suzuki, Shunsuke" /><meta name="citation_author_institution" content="Department of Zoology, The University of Melbourne, Melbourne, Victoria 3010, Australia" /><meta name="citation_author_institution" content="Epigenomics Division, Faculty of Agriculture, Frontier Agriscience and Technology Center, Shinshu University, 8304 Minami-minowa, Kami-ina, Nagano 399-4598, Japan" /><meta name="citation_author" content="Pelekanos, Matthew" /><meta name="citation_author_institution" content="Department of Zoology, The University of Melbourne, Melbourne, Victoria 3010, Australia" /><meta name="citation_author" content="Clark, Helen" /><meta name="citation_author_institution" content="Department of Zoology, The University of Melbourne, Melbourne, Victoria 3010, Australia" /><meta name="citation_author" content="Ono, Ryuichi" /><meta name="citation_author_institution" content="Department of Epigenetics, Medical Research Institute, Tokyo Medical and Dental University, 1-5-45 Yushima, Bunkyo-ku, Tokyo 113-8510, Japan" /><meta name="citation_author" content="Shaw, Geoff" /><meta name="citation_author_institution" content="Department of Zoology, The University of Melbourne, Melbourne, Victoria 3010, Australia" /><meta name="citation_author" content="Renfree, Marilyn B." /><meta name="citation_author_institution" content="Department of Zoology, The University of Melbourne, Melbourne, Victoria 3010, Australia" /><meta name="citation_author" content="Kaneko-Ishino, Tomoko" /><meta name="citation_author_institution" content="School of Health Sciences, Tokai University, Bohseidai, Isehara, Kanagawa 259-1193, Japan" /><meta name="citation_author" content="Ishino, Fumitoshi" /><meta name="citation_author_institution" content="Department of Epigenetics, Medical Research Institute, Tokyo Medical and Dental University, 1-5-45 Yushima, Bunkyo-ku, Tokyo 113-8510, Japan" /><meta name="citation_author_institution" content="Global Center of Excellence Program for International Research Center for Molecular Science in Tooth and Bone Diseases, Tokyo Medical and Dental University, 1-5-45 Yushima, Bunkyo-ku, Tokyo 113-8510, Japan" /><meta name="citation_title" content="Identification of a Novel PNMA-MS1 Gene in Marsupials Suggests the LTR Retrotransposon-Derived PNMA Genes Evolved Differently in Marsupials and Eutherians" /><meta name="citation_firstpage" content="425" /><meta name="citation_lastpage" content="436" /><meta name="citation_doi" content="10.1093/dnares/dst020" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2013/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Initial sequencing and analysis of the human genome; Nature;  citation_year=2001;  citation_volume=409;  citation_pages=860-921; " /><meta name="citation_reference" content="citation_title=Initial sequencing and comparative analysis of the mouse genome; citation_author=Mouse Genome Sequencing Consortium; Nature;  citation_year=2002;  citation_volume=420;  citation_pages=520-62; " /><meta name="citation_reference" content="citation_title=Genome analysis of the platypus reveals unique signatures of evolution; Nature;  citation_year=2008;  citation_volume=453;  citation_pages=175-83; " /><meta name="citation_reference" content="citation_title=Genome of the marsupial Monodelphis domestica reveals innovation in non-coding sequences; Nature;  citation_year=2007;  citation_volume=447;  citation_pages=167-77; " /><meta name="citation_reference" content="citation_title=Genome sequence of an Australian kangaroo, Macropus eugenii, provides insight into the evolution of mammalian reproduction and development; Genome Biol.;  citation_year=2011;  citation_volume=12;  citation_pages=R81" /><meta name="citation_reference" content="citation_title=Identification of multiple Gypsy LTR-retrotransposon lineages in vertebrate genomes; J. Mol. Evol.;  citation_year=1999;  citation_volume=49;  citation_pages=358-66; " /><meta name="citation_reference" content="citation_title=Deletion of Peg10, an imprinted gene acquired from a retrotransposon, causes early embryonic lethality; Nat. Genet.;  citation_year=2006;  citation_volume=38;  citation_pages=101-6; " /><meta name="citation_reference" content="citation_title=Role of retrotransposon-derived imprinted gene, Rtl1, in the feto-maternal interface of mouse placenta; Nat. Genet.;  citation_year=2008;  citation_volume=40;  citation_pages=243-8; " /><meta name="citation_reference" content="citation_title=A family of neofunctionalized Ty3/gypsy retrotransposon genes in mammalian genomes; Cytogenet. Genome Res.;  citation_year=2005;  citation_volume=110;  citation_pages=307-17; " /><meta name="citation_reference" content="citation_title=A small family of sushi-class retrotransposon-derived genes in mammals and their relation to genomic imprinting; J. Mol. Evol.;  citation_year=2005;  citation_volume=61;  citation_pages=481-90; " /><meta name="citation_reference" content="citation_title=Identification of tammar wallaby SIRH12, derived from a marsupial-specific retrotransposition event; DNA Res.;  citation_year=2011;  citation_volume=18;  citation_pages=211-9; " /><meta name="citation_reference" content="citation_title=The role of genes domesticated from LTR retrotransposons and retroviruses in mammals; Front Microbiol.;  citation_year=2012;  citation_volume=3;  citation_pages=262" /><meta name="citation_reference" content="citation_title=Computational characterization of multiple Gag-like human proteins; Trends Genet.;  citation_year=2006;  citation_volume=22;  citation_pages=585-9; " /><meta name="citation_reference" content="citation_title=Paraneoplastic antigen-like 5 gene (PNMA5) is preferentially expressed in the association areas in a primate specific manner; Cereb. Cortex;  citation_year=2009;  citation_volume=19;  citation_pages=2865-79; " /><meta name="citation_reference" content="citation_title=Molecular and clinical diversity in paraneoplastic immunity to Ma proteins; Ann. Neurol.;  citation_year=2001;  citation_volume=50;  citation_pages=339-48; " /><meta name="citation_reference" content="citation_title=The human PNMA family: novel neuronal proteins implicated in paraneoplastic neurological disease; J. Neuroimmunol.;  citation_year=2005;  citation_volume=169;  citation_pages=172-6; " /><meta name="citation_reference" content="citation_text=Technical Note; citation_publisher=Division of Wildlife and Ecology, CSIRO, Canberra, Australia; Tables for Age Determination of the Kangaroo Island Wallaby (Tammar), Macropus eugenii, from Body Measurements;  citation_year=1991; " /><meta name="citation_reference" content="citation_author=NHMRC;  citation_edition=7th edition; citation_publisher=Australian Government, Canberra; Australian Code of Practice for the Care and Use of Animals for Scientific Purposes;  citation_year=2004; " /><meta name="citation_reference" content="citation_title=The mouse olfactory receptor gene family; Proc. Natl Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=2156-61; " /><meta name="citation_reference" content="citation_title=The human olfactory receptor gene family; Proc. Natl Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=2584-9; " /><meta name="citation_reference" content="citation_title=Retrotransposon silencing by DNA methylation can drive mammalian genomic imprinting; PLoS Genet.;  citation_year=2007;  citation_volume=3;  citation_pages=e55" /><meta name="citation_reference" content="Ashwell K. 2010 Cambridge University Press 95 6 The neurobiology of Australian marsupials: brain evolution in the other mammalian radiation " /><meta name="citation_reference" content="citation_title=Structure/function mapping of amino acids in the N-terminal zinc finger of the human immunodeficiency virus type 1 nucleocapsid protein: residues responsible for nucleic acid helix destabilizing activity; Biochemistry;  citation_year=2006;  citation_volume=45;  citation_pages=12617-28; " /><meta name="citation_reference" content="citation_title=RNA recognition mechanism of the minimal active domain of the human immunodeficiency virus type-2 nucleocapsid protein; J. Biochem.;  citation_year=2007;  citation_volume=141;  citation_pages=269-77; " /><meta name="citation_reference" content="citation_title=A CCHC metal-binding domain in Nanos is essential for translational regulation; EMBO J.;  citation_year=1997;  citation_volume=16;  citation_pages=834-43; " /><meta name="citation_reference" content="citation_title=The zinc-finger protein CNBP is required for forebrain formation in the mouse; Development;  citation_year=2003;  citation_volume=130;  citation_pages=1367-79; " /><meta name="citation_reference" content="citation_title=Identification of a zinc finger protein that binds to the sterol regulatory element; Science;  citation_year=1989;  citation_volume=245;  citation_pages=640-3; " /><meta name="citation_reference" content="citation_title=Novel CNBP- and La-based translation control systems for mammalian cells; Biotechnol. Bioeng.;  citation_year=2003;  citation_volume=81;  citation_pages=1-12; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/5/425/1027885/dst020.pdf" /><meta name="description" content="Abstract.  Two major gene families derived from Ty3/Gypsy long terminal repeat (LTR) retrotransposons were recently identified in mammals. The sushi-ichi retrot" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/5/425/359087" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/5/425/359087" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/20/5/425/359087");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst020';
    </script>
<title>
	Identification of a Novel PNMA-MS1 Gene in Marsupials Suggests the LTR Retrotransposon-Derived PNMA Genes Evolved Differently in Marsupials and Eutherians | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./359087" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bf5PPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJd2FzYWtpLCBTYXdhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBFcGlnZW5ldGljcywgTWVkaWNhbCBSZXNlYXJjaCBJbnN0aXR1dGUsIFRva3lvIE1lZGljYWwgYW5kIERlbnRhbCBVbml2ZXJzaXR5LCAxLTUtNDUgWXVzaGltYSwgQnVua3lvLWt1LCBUb2t5byAxMTMtODUxMCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTY2hvb2wgb2YgSGVhbHRoIFNjaWVuY2VzLCBUb2thaSBVbml2ZXJzaXR5LCBCb2hzZWlkYWksIElzZWhhcmEsIEthbmFnYXdhIDI1OS0xMTkzLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlN1enVraSwgU2h1bnN1a2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFpvb2xvZ3ksIFRoZSBVbml2ZXJzaXR5IG9mIE1lbGJvdXJuZSwgTWVsYm91cm5lLCBWaWN0b3JpYSAzMDEwLCBBdXN0cmFsaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJFcGlnZW5vbWljcyBEaXZpc2lvbiwgRmFjdWx0eSBvZiBBZ3JpY3VsdHVyZSwgRnJvbnRpZXIgQWdyaXNjaWVuY2UgYW5kIFRlY2hub2xvZ3kgQ2VudGVyLCBTaGluc2h1IFVuaXZlcnNpdHksIDgzMDQgTWluYW1pLW1pbm93YSwgS2FtaS1pbmEsIE5hZ2FubyAzOTktNDU5OCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQZWxla2Fub3MsIE1hdHRoZXciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFpvb2xvZ3ksIFRoZSBVbml2ZXJzaXR5IG9mIE1lbGJvdXJuZSwgTWVsYm91cm5lLCBWaWN0b3JpYSAzMDEwLCBBdXN0cmFsaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDbGFyaywgSGVsZW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFpvb2xvZ3ksIFRoZSBVbml2ZXJzaXR5IG9mIE1lbGJvdXJuZSwgTWVsYm91cm5lLCBWaWN0b3JpYSAzMDEwLCBBdXN0cmFsaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPbm8sIFJ5dWljaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEVwaWdlbmV0aWNzLCBNZWRpY2FsIFJlc2VhcmNoIEluc3RpdHV0ZSwgVG9reW8gTWVkaWNhbCBhbmQgRGVudGFsIFVuaXZlcnNpdHksIDEtNS00NSBZdXNoaW1hLCBCdW5reW8ta3UsIFRva3lvIDExMy04NTEwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNoYXcsIEdlb2ZmIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBab29sb2d5LCBUaGUgVW5pdmVyc2l0eSBvZiBNZWxib3VybmUsIE1lbGJvdXJuZSwgVmljdG9yaWEgMzAxMCwgQXVzdHJhbGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUmVuZnJlZSwgTWFyaWx5biBCLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgWm9vbG9neSwgVGhlIFVuaXZlcnNpdHkgb2YgTWVsYm91cm5lLCBNZWxib3VybmUsIFZpY3RvcmlhIDMwMTAsIEF1c3RyYWxpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkthbmVrby1Jc2hpbm8sIFRvbW9rbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNjaG9vbCBvZiBIZWFsdGggU2NpZW5jZXMsIFRva2FpIFVuaXZlcnNpdHksIEJvaHNlaWRhaSwgSXNlaGFyYSwgS2FuYWdhd2EgMjU5LTExOTMsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSXNoaW5vLCBGdW1pdG9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEVwaWdlbmV0aWNzLCBNZWRpY2FsIFJlc2VhcmNoIEluc3RpdHV0ZSwgVG9reW8gTWVkaWNhbCBhbmQgRGVudGFsIFVuaXZlcnNpdHksIDEtNS00NSBZdXNoaW1hLCBCdW5reW8ta3UsIFRva3lvIDExMy04NTEwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikdsb2JhbCBDZW50ZXIgb2YgRXhjZWxsZW5jZSBQcm9ncmFtIGZvciBJbnRlcm5hdGlvbmFsIFJlc2VhcmNoIENlbnRlciBmb3IgTW9sZWN1bGFyIFNjaWVuY2UgaW4gVG9vdGggYW5kIEJvbmUgRGlzZWFzZXMsIFRva3lvIE1lZGljYWwgYW5kIERlbnRhbCBVbml2ZXJzaXR5LCAxLTUtNDUgWXVzaGltYSwgQnVua3lvLWt1LCBUb2t5byAxMTMtODUxMCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IklkZW50aWZpY2F0aW9uIG9mIGEgTm92ZWwgUE5NQS1NUzEgR2VuZSBpbiBNYXJzdXBpYWxzIFN1Z2dlc3RzIHRoZSBMVFIgUmV0cm90cmFuc3Bvc29uLURlcml2ZWQgUE5NQSBHZW5lcyBFdm9sdmVkIERpZmZlcmVudGx5IGluIE1hcnN1cGlhbHMgYW5kIEV1dGhlcmlhbnMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI0MjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjQzNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdDAyMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEzLzEwLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5pdGlhbCBzZXF1ZW5jaW5nIGFuZCBhbmFseXNpcyBvZiB0aGUgaHVtYW4gZ2Vub21lOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NDA5OyAgY2l0YXRpb25fcGFnZXM9ODYwLTkyMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5pdGlhbCBzZXF1ZW5jaW5nIGFuZCBjb21wYXJhdGl2ZSBhbmFseXNpcyBvZiB0aGUgbW91c2UgZ2Vub21lOyBjaXRhdGlvbl9hdXRob3I9TW91c2UgR2Vub21lIFNlcXVlbmNpbmcgQ29uc29ydGl1bTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTQyMDsgIGNpdGF0aW9uX3BhZ2VzPTUyMC02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIGFuYWx5c2lzIG9mIHRoZSBwbGF0eXB1cyByZXZlYWxzIHVuaXF1ZSBzaWduYXR1cmVzIG9mIGV2b2x1dGlvbjsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ1MzsgIGNpdGF0aW9uX3BhZ2VzPTE3NS04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIG9mIHRoZSBtYXJzdXBpYWwgTW9ub2RlbHBoaXMgZG9tZXN0aWNhIHJldmVhbHMgaW5ub3ZhdGlvbiBpbiBub24tY29kaW5nIHNlcXVlbmNlczsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ0NzsgIGNpdGF0aW9uX3BhZ2VzPTE2Ny03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNlIG9mIGFuIEF1c3RyYWxpYW4ga2FuZ2Fyb28sIE1hY3JvcHVzIGV1Z2VuaWksIHByb3ZpZGVzIGluc2lnaHQgaW50byB0aGUgZXZvbHV0aW9uIG9mIG1hbW1hbGlhbiByZXByb2R1Y3Rpb24gYW5kIGRldmVsb3BtZW50OyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz1SODEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBtdWx0aXBsZSBHeXBzeSBMVFItcmV0cm90cmFuc3Bvc29uIGxpbmVhZ2VzIGluIHZlcnRlYnJhdGUgZ2Vub21lczsgSi4gTW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT00OTsgIGNpdGF0aW9uX3BhZ2VzPTM1OC02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGVsZXRpb24gb2YgUGVnMTAsIGFuIGltcHJpbnRlZCBnZW5lIGFjcXVpcmVkIGZyb20gYSByZXRyb3RyYW5zcG9zb24sIGNhdXNlcyBlYXJseSBlbWJyeW9uaWMgbGV0aGFsaXR5OyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zODsgIGNpdGF0aW9uX3BhZ2VzPTEwMS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Sb2xlIG9mIHJldHJvdHJhbnNwb3Nvbi1kZXJpdmVkIGltcHJpbnRlZCBnZW5lLCBSdGwxLCBpbiB0aGUgZmV0by1tYXRlcm5hbCBpbnRlcmZhY2Ugb2YgbW91c2UgcGxhY2VudGE7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9MjQzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZmFtaWx5IG9mIG5lb2Z1bmN0aW9uYWxpemVkIFR5My9neXBzeSByZXRyb3RyYW5zcG9zb24gZ2VuZXMgaW4gbWFtbWFsaWFuIGdlbm9tZXM7IEN5dG9nZW5ldC4gR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTEwOyAgY2l0YXRpb25fcGFnZXM9MzA3LTE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNtYWxsIGZhbWlseSBvZiBzdXNoaS1jbGFzcyByZXRyb3RyYW5zcG9zb24tZGVyaXZlZCBnZW5lcyBpbiBtYW1tYWxzIGFuZCB0aGVpciByZWxhdGlvbiB0byBnZW5vbWljIGltcHJpbnRpbmc7IEouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NjE7ICBjaXRhdGlvbl9wYWdlcz00ODEtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIHRhbW1hciB3YWxsYWJ5IFNJUkgxMiwgZGVyaXZlZCBmcm9tIGEgbWFyc3VwaWFsLXNwZWNpZmljIHJldHJvdHJhbnNwb3NpdGlvbiBldmVudDsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0yMTEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJvbGUgb2YgZ2VuZXMgZG9tZXN0aWNhdGVkIGZyb20gTFRSIHJldHJvdHJhbnNwb3NvbnMgYW5kIHJldHJvdmlydXNlcyBpbiBtYW1tYWxzOyBGcm9udCBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0yNjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wdXRhdGlvbmFsIGNoYXJhY3Rlcml6YXRpb24gb2YgbXVsdGlwbGUgR2FnLWxpa2UgaHVtYW4gcHJvdGVpbnM7IFRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz01ODUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGFyYW5lb3BsYXN0aWMgYW50aWdlbi1saWtlIDUgZ2VuZSAoUE5NQTUpIGlzIHByZWZlcmVudGlhbGx5IGV4cHJlc3NlZCBpbiB0aGUgYXNzb2NpYXRpb24gYXJlYXMgaW4gYSBwcmltYXRlIHNwZWNpZmljIG1hbm5lcjsgQ2VyZWIuIENvcnRleDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTI4NjUtNzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBhbmQgY2xpbmljYWwgZGl2ZXJzaXR5IGluIHBhcmFuZW9wbGFzdGljIGltbXVuaXR5IHRvIE1hIHByb3RlaW5zOyBBbm4uIE5ldXJvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NTA7ICBjaXRhdGlvbl9wYWdlcz0zMzktNDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBodW1hbiBQTk1BIGZhbWlseTogbm92ZWwgbmV1cm9uYWwgcHJvdGVpbnMgaW1wbGljYXRlZCBpbiBwYXJhbmVvcGxhc3RpYyBuZXVyb2xvZ2ljYWwgZGlzZWFzZTsgSi4gTmV1cm9pbW11bm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjk7ICBjaXRhdGlvbl9wYWdlcz0xNzItNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGV4dD1UZWNobmljYWwgTm90ZTsgY2l0YXRpb25fcHVibGlzaGVyPURpdmlzaW9uIG9mIFdpbGRsaWZlIGFuZCBFY29sb2d5LCBDU0lSTywgQ2FuYmVycmEsIEF1c3RyYWxpYTsgVGFibGVzIGZvciBBZ2UgRGV0ZXJtaW5hdGlvbiBvZiB0aGUgS2FuZ2Fyb28gSXNsYW5kIFdhbGxhYnkgKFRhbW1hciksIE1hY3JvcHVzIGV1Z2VuaWksIGZyb20gQm9keSBNZWFzdXJlbWVudHM7ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX2F1dGhvcj1OSE1SQzsgIGNpdGF0aW9uX2VkaXRpb249N3RoIGVkaXRpb247IGNpdGF0aW9uX3B1Ymxpc2hlcj1BdXN0cmFsaWFuIEdvdmVybm1lbnQsIENhbmJlcnJhOyBBdXN0cmFsaWFuIENvZGUgb2YgUHJhY3RpY2UgZm9yIHRoZSBDYXJlIGFuZCBVc2Ugb2YgQW5pbWFscyBmb3IgU2NpZW50aWZpYyBQdXJwb3NlczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG1vdXNlIG9sZmFjdG9yeSByZWNlcHRvciBnZW5lIGZhbWlseTsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDE7ICBjaXRhdGlvbl9wYWdlcz0yMTU2LTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgaHVtYW4gb2xmYWN0b3J5IHJlY2VwdG9yIGdlbmUgZmFtaWx5OyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTI1ODQtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmV0cm90cmFuc3Bvc29uIHNpbGVuY2luZyBieSBETkEgbWV0aHlsYXRpb24gY2FuIGRyaXZlIG1hbW1hbGlhbiBnZW5vbWljIGltcHJpbnRpbmc7IFBMb1MgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz1lNTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJBc2h3ZWxsIEsuIDIwMTAgQ2FtYnJpZGdlIFVuaXZlcnNpdHkgUHJlc3MgOTUgNiBUaGUgbmV1cm9iaW9sb2d5IG9mIEF1c3RyYWxpYW4gbWFyc3VwaWFsczogYnJhaW4gZXZvbHV0aW9uIGluIHRoZSBvdGhlciBtYW1tYWxpYW4gcmFkaWF0aW9uICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cnVjdHVyZS9mdW5jdGlvbiBtYXBwaW5nIG9mIGFtaW5vIGFjaWRzIGluIHRoZSBOLXRlcm1pbmFsIHppbmMgZmluZ2VyIG9mIHRoZSBodW1hbiBpbW11bm9kZWZpY2llbmN5IHZpcnVzIHR5cGUgMSBudWNsZW9jYXBzaWQgcHJvdGVpbjogcmVzaWR1ZXMgcmVzcG9uc2libGUgZm9yIG51Y2xlaWMgYWNpZCBoZWxpeCBkZXN0YWJpbGl6aW5nIGFjdGl2aXR5OyBCaW9jaGVtaXN0cnk7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz0xMjYxNy0yODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Uk5BIHJlY29nbml0aW9uIG1lY2hhbmlzbSBvZiB0aGUgbWluaW1hbCBhY3RpdmUgZG9tYWluIG9mIHRoZSBodW1hbiBpbW11bm9kZWZpY2llbmN5IHZpcnVzIHR5cGUtMiBudWNsZW9jYXBzaWQgcHJvdGVpbjsgSi4gQmlvY2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTQxOyAgY2l0YXRpb25fcGFnZXM9MjY5LTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIENDSEMgbWV0YWwtYmluZGluZyBkb21haW4gaW4gTmFub3MgaXMgZXNzZW50aWFsIGZvciB0cmFuc2xhdGlvbmFsIHJlZ3VsYXRpb247IEVNQk8gSi47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz04MzQtNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB6aW5jLWZpbmdlciBwcm90ZWluIENOQlAgaXMgcmVxdWlyZWQgZm9yIGZvcmVicmFpbiBmb3JtYXRpb24gaW4gdGhlIG1vdXNlOyBEZXZlbG9wbWVudDsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzA7ICBjaXRhdGlvbl9wYWdlcz0xMzY3LTc5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBhIHppbmMgZmluZ2VyIHByb3RlaW4gdGhhdCBiaW5kcyB0byB0aGUgc3Rlcm9sIHJlZ3VsYXRvcnkgZWxlbWVudDsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MTk4OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNDU7ICBjaXRhdGlvbl9wYWdlcz02NDAtMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm92ZWwgQ05CUC0gYW5kIExhLWJhc2VkIHRyYW5zbGF0aW9uIGNvbnRyb2wgc3lzdGVtcyBmb3IgbWFtbWFsaWFuIGNlbGxzOyBCaW90ZWNobm9sLiBCaW9lbmcuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTgxOyAgY2l0YXRpb25fcGFnZXM9MS0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIwLzUvNDI1LzEwMjc4ODUvZHN0MDIwLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBUd28gbWFqb3IgZ2VuZSBmYW1pbGllcyBkZXJpdmVkIGZyb20gVHkzL0d5cHN5IGxvbmcgdGVybWluYWwgcmVwZWF0IChMVFIpIHJldHJvdHJhbnNwb3NvbnMgd2VyZSByZWNlbnRseSBpZGVudGlmaWVkIGluIG1hbW1hbHMuIFRoZSBzdXNoaS1pY2hpIHJldHJvdCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzU5MDg3JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZE/xZag8MAtIY5OQTqaHdKQa5dQZ" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f5%2f425%2f359087" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f5%2f425%2f359087" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13963" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/5/1/m_cover.gif?Expires=2147483647&amp;Signature=Lk49EY44ST5kshtHpZ8RK~2R1A7PgJCg08uzNU5E4ewaw5su36z91fGFzyjKSwM7FA8vqYVOBewtYyaa49VDZyPXl9Uk3UXnVDma3euMHlznALvLA3cmgbZS4rUQVsL9pDxMqMdxSm2soKO-Fi0OyE0yb48n7~-liMKL-OGMbIZgQadhhuxmJniONIYojHw~acVvxnAWpMlYOg7NZ41neYIpgkm8f0MVUTZPG1rOzVvSkGGn2mgtSfwCsvVKdAnNTIkShF7ickxRfq678V912uPQ-4xXwjlGVd5JPSDhJq0sRXahBacmVvr-s125nZapbB7I5xsPcGLkoW6cY527iw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/5">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5174502">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5174502">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5174504">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5174504">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5174507">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5174507">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5174521">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5174521">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5174545">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5174545">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5174554">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5174554">5. Conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5174556">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5174556">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5174558">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5174558">Accession Numbers</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5174560">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5174560">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5174562">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5174562">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5174564">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5174564">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/20/5/437/359202">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Identification of a Novel <em>PNMA-MS1</em> Gene in Marsupials Suggests the LTR Retrotransposon-Derived <em>PNMA</em> Genes Evolved Differently in Marsupials and Eutherians
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sawa Iwasaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Epigenetics, Medical Research Institute</div>, <div class="insititution">Tokyo Medical and Dental University</div>, <div class="addr-line">1-5-45 Yushima, Bunkyo-ku, Tokyo 113-8510</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">School of Health Sciences</div>, <div class="insititution">Tokai University</div>, <div class="addr-line">Bohseidai, Isehara, Kanagawa 259-1193</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sawa+Iwasaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Iwasaki S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Iwasaki S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sawa Iwasaki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shunsuke Suzuki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Zoology</div>, <div class="insititution">The University of Melbourne</div>, <div class="addr-line">Melbourne, Victoria 3010</div>, <div class="country">Australia</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">Epigenomics Division, Faculty of Agriculture, Frontier Agriscience and Technology Center</div>, <div class="insititution">Shinshu University</div>, <div class="addr-line">8304 Minami-minowa, Kami-ina, Nagano 399-4598</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shunsuke+Suzuki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suzuki S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suzuki S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shunsuke Suzuki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Matthew Pelekanos</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Zoology</div>, <div class="insititution">The University of Melbourne</div>, <div class="addr-line">Melbourne, Victoria 3010</div>, <div class="country">Australia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Matthew+Pelekanos">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pelekanos M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Pelekanos M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Matthew Pelekanos</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Helen Clark</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Zoology</div>, <div class="insititution">The University of Melbourne</div>, <div class="addr-line">Melbourne, Victoria 3010</div>, <div class="country">Australia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Helen+Clark">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Clark H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Clark H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Helen Clark</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ryuichi Ono</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Epigenetics, Medical Research Institute</div>, <div class="insititution">Tokyo Medical and Dental University</div>, <div class="addr-line">1-5-45 Yushima, Bunkyo-ku, Tokyo 113-8510</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ryuichi+Ono">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ono R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ono R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ryuichi Ono</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Geoff Shaw</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Zoology</div>, <div class="insititution">The University of Melbourne</div>, <div class="addr-line">Melbourne, Victoria 3010</div>, <div class="country">Australia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Geoff+Shaw">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shaw G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shaw G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Geoff Shaw</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Marilyn B. Renfree</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Zoology</div>, <div class="insititution">The University of Melbourne</div>, <div class="addr-line">Melbourne, Victoria 3010</div>, <div class="country">Australia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Marilyn+B.+Renfree">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Renfree M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Renfree M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Marilyn B. Renfree</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tomoko Kaneko-Ishino</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">School of Health Sciences</div>, <div class="insititution">Tokai University</div>, <div class="addr-line">Bohseidai, Isehara, Kanagawa 259-1193</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tomoko+Kaneko-Ishino">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kaneko-Ishino T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kaneko-Ishino T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tomoko Kaneko-Ishino</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fumitoshi Ishino</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Epigenetics, Medical Research Institute</div>, <div class="insititution">Tokyo Medical and Dental University</div>, <div class="addr-line">1-5-45 Yushima, Bunkyo-ku, Tokyo 113-8510</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">5</div><div class="insititution">Global Center of Excellence Program for International Research Center for Molecular Science in Tooth and Bone Diseases, Tokyo Medical and Dental University</div>, <div class="addr-line">1-5-45 Yushima, Bunkyo-ku, Tokyo 113-8510</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81-3-5803-4862. Fax. +81-3-5803-4863. Email: <a href="mailto:fishino.epgn@mri.tmd.ac.jp" target="_blank">fishino.epgn@mri.tmd.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fumitoshi+Ishino">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ishino F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ishino F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Fumitoshi Ishino</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 5, 1 October 2013, Pages 425–436, <a href='https://doi.org/10.1093/dnares/dst020'>https://doi.org/10.1093/dnares/dst020</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">23 May 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">05 November 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">24 April 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="359087" href="/dnaresearch/article-pdf/20/5/425/1027885/dst020.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Sawa Iwasaki, Shunsuke Suzuki, Matthew Pelekanos, Helen Clark, Ryuichi Ono, Geoff Shaw, Marilyn B. Renfree, Tomoko Kaneko-Ishino, Fumitoshi Ishino;  Identification of a Novel <em>PNMA-MS1</em> Gene in Marsupials Suggests the LTR Retrotransposon-Derived <em>PNMA</em> Genes Evolved Differently in Marsupials and Eutherians, <em>DNA Research</em>, Volume 20, Issue 5, 1 October 2013, Pages 425–436, <a href='https://doi.org/10.1093/dnares/dst020'>https://doi.org/10.1093/dnares/dst020</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/359087?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/359087?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/359087?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/359087?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/359087?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/359087?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/359087?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Identification%20of%20a%20Novel%20%26lt%3Bem%26gt%3BPNMA-MS1%26lt%3B%2Fem%26gt%3B%20Gene%20in%20Marsupials%20Suggests%20the%20LTR%20Retrotransposon-Derived%20%26lt%3Bem%26gt%3BPNMA%26lt%3B%2Fem%26gt%3B%20Genes%20Evolved%20Differently%20in%20Marsupials%20and%20Eutherians&amp;publicationDate=2013-05-23&amp;volumeNum=20&amp;issueNum=5&amp;author=Iwasaki%2C%20Sawa%3B%20Suzuki%2C%20Shunsuke&amp;startPage=425&amp;endPage=436&amp;contentId=10.1093%2Fdnares%2Fdst020&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f5%2f425%2f359087" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5174502 id="5174502" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Two major gene families derived from Ty3/Gypsy long terminal repeat (LTR) retrotransposons were recently identified in mammals. The sushi-ichi retrotransposon homologue (<em>SIRH</em>) family comprises 12 genes: 11 in eutherians including <em>Peg10</em> and <em>Peg11</em>/<em>Rtl1</em> that have essential roles in the eutherian placenta and 1 that is marsupial specific. Fifteen and 12 genes were reported in the second gene family, para-neoplastic antigen MA (<em>PNMA</em>), in humans and mice, respectively, although their biological functions and evolutionary history remain largely unknown. Here, we identified two novel candidate <em>PNMA</em> genes, <em>PNMA-MS1</em> and <em>-MS2</em> in marsupials. Like all eutherian-specific <em>PNMA</em> genes, they exhibit the highest homology to a Gypsy12_DR (DR, <em>Danio rerio</em>) Gag protein. <em>PNMA-MS1</em> is conserved in both Australian and South American marsupial species, the tammar wallaby and grey short-tailed opossum. However, no <em>PNMA-MS1</em> orthologue was found in eutherians, monotremes or non-mammalian vertebrates. <em>PNMA-MS1</em> was expressed in the ovary, mammary gland and brain during development and growth in the tammar, suggesting that <em>PNMA-MS1</em> may have acquired a marsupial-specific function. However, <em>PNMA-MS2</em> seems to be a pseudogene. The absence of marsupial orthologues of eutherian <em>PNMA</em> genes suggests that the retrotransposition events of the Gypsy12_DR-related retrotransposons that gave rise to the <em>PNMA</em> family occurred after the divergence of marsupials and eutherians.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;LTR retrotransposons&quot;">LTR retrotransposons</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;PNMA family&quot;"><em>PNMA</em> family</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;marsupial-specific genes&quot;">marsupial-specific genes</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;mammalian evolution&quot;">mammalian evolution</a></div>                            <h2 scrollto-destination=5174504 id="5174504" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Approximately 40–50% of the mammalian genome is derived from transposable elements, such as retrotransposons and DNA transposons.<sup><a reveal-id="DST020C1  DST020C5" data-open="DST020C1  DST020C5" class="link link-ref link-reveal xref-bibr">1–5</a></sup> The Ty3/Gypsy long terminal repeat (LTR) retrotransposons have been detected in various eukaryotic organisms including fungi, plants, insects, tunicates and echinoderms as well as in several vertebrates, such as fish, amphibians and reptiles, but not in mammals and birds.<sup><a reveal-id="DST020C6" data-open="DST020C6" class="link link-ref link-reveal xref-bibr">6</a></sup> However, discrete regions within these elements have acquired new functions as novel endogenous genes and are highly conserved in marsupials and eutherians.<sup><a reveal-id="DST020C7  DST020C12" data-open="DST020C7  DST020C12" class="link link-ref link-reveal xref-bibr">7–12</a></sup> Two major gene families derived from the Ty3/Gypsy LTR retrotransposons are the sushi-ichi retrotransposon homologue (<em>SIRH</em>) family (also called the <em>MART</em> or <em>SUSHI</em> family) comprising 12 genes encoding a Gag-like protein, each of which has 20–30% similarity to the sushi-ichi retrotransposon Gag in pufferfish (<em>Takifugu rubripes</em>),<sup><a reveal-id="DST020C9  DST020C11" data-open="DST020C9  DST020C11" class="link link-ref link-reveal xref-bibr">9–11</a></sup> and the para-neoplastic antigen MA (<em>PNMA</em>) family also encoding the Gag-like protein homologous to the Gypsy12_DR retrotransposon Gag in zebrafish (DR, <em>Danio rerio</em>)<sup><a reveal-id="DST020C13" data-open="DST020C13" class="link link-ref link-reveal xref-bibr">13</a></sup> comprising 15 and 11 genes in humans and mice, respectively. It should be noted that the homology between these two LTR retrotransposons is only 6.5% and 13.6% along with the entire Gag and Pol regions, respectively. <em>PEG10/SIRH1</em> is a therian-specific gene, and <em>PEG11/SIHR2</em> and the remaining <em>SIRH3–11</em> seem eutherian specific, while <em>SIRH12</em> was derived from a marsupial-specific retrotransposition event. We previously demonstrated that <em>Peg10/Sirh1</em> and <em>Peg11/Sirh2</em> are essential for placental formation and function in mice.<sup><a reveal-id="DST020C7" data-open="DST020C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DST020C8" data-open="DST020C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Most <em>PNMA</em> genes are expressed in the brains of macaques and mice and their functions remain unknown.<sup><a reveal-id="DST020C14" data-open="DST020C14" class="link link-ref link-reveal xref-bibr">14</a></sup><em>PNMA1</em>–<em>3</em> were first identified as genes encoding neuronal auto-antigens using sera from patients with para-neoplastic neurological syndromes.<sup><a reveal-id="DST020C15" data-open="DST020C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Schüller <em>et al.</em><sup><a reveal-id="DST020C16" data-open="DST020C16" class="link link-ref link-reveal xref-bibr">16</a></sup> and Campillos <em>et al.</em><sup><a reveal-id="DST020C13" data-open="DST020C13" class="link link-ref link-reveal xref-bibr">13</a></sup> performed genome-wide analyses and identified additional 12 family genes in humans among which <em>PNMA6</em> has no mouse orthologue. No Gypsy12_DR Gag-derived sequences were reported in birds,<sup><a reveal-id="DST020C13" data-open="DST020C13" class="link link-ref link-reveal xref-bibr">13</a></sup> and thus, it is probable that the <em>PNMA</em> genes are also mammal specific. However, the search has been limited in several eutherian species and the existence of marsupial orthologues and/or marsupial-specific <em>PNMA</em> genes remained unknown.</p><p>Here, we conducted comprehensive <em>in silico</em> screening for the <em>PNMA</em> genes using the whole-genome shotgun (WGS) sequences of the grey short-tailed opossum<sup><a reveal-id="DST020C4" data-open="DST020C4" class="link link-ref link-reveal xref-bibr">4</a></sup> and the tammar wallaby<sup><a reveal-id="DST020C5" data-open="DST020C5" class="link link-ref link-reveal xref-bibr">5</a></sup> and identified a novel <em>PNMA-MS1</em> gene as the first marsupial-specific <em>PNMA</em> gene.</p>                            <h2 scrollto-destination=5174507 id="5174507" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5174508 id="5174508" class="section-title" data-legacy-id=s2a>2.1. Animals and tissue collection</h3>
<p>Tammar wallabies (<em>Macropus eugenii</em>) of Kangaroo Island, South Australia origin, were maintained in The University of Melbourne marsupial breeding colony in grassy outdoor enclosures. Lucerne cubes, grass and water were provided <em>ad libitum</em> and supplemented with fresh vegetables. The day of birth of pouch young was designated as d0. When the day of birth was unknown, their age was estimated using the head length.<sup><a reveal-id="DST020C17" data-open="DST020C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Foetal tissues including the head and body were collected from two foetuses at Day 23 and 26 of gestation, and the yolk sac placenta (YSP) from four foetuses sampled between Day 23 and 26 of gestation. Tissues including the brain, liver, lung, kidney, ovary and testis were collected from two pouch young aged Day 60–70 after birth. The liver, lung, pancreas, stomach, bladder, heart, kidney, adrenal, spleen and brain (cerebrum and cerebellum) were collected from Day 152 and 162 pouch young. Adult female tissues, including the brain (thalamus, hypothalamus and pituitary), ovary (ovary with active corpus luteum, corpus luteum, ovary with developing follicle and ovary with primary or secondary follicle), endometrium (gravid endometrium and non-gravid endometrium) and mammary gland (sucked gland and non-sucked gland) were also collected from two adults. Grey short-tailed opossums (<em>Monodelphis domestica</em>) were purchased from a breeding colony in the Department of Physiology at the University of Melbourne. The brain, liver, spleen, pituitary and ovary were collected from five adult opossums. Experimental procedures conformed to the Australian National Health and Medical Research Council guidelines<sup><a reveal-id="DST020C18" data-open="DST020C18" class="link link-ref link-reveal xref-bibr">18</a></sup> and were approved by the Animal Experimentation Ethics Committees of the University of Melbourne.</p>                            <h3 scrollto-destination=5174510 id="5174510" class="section-title" data-legacy-id=s2b>2.2. Reverse transcriptase polymerase chain reaction</h3>
<p>Genomic DNA and total RNA from tissues were prepared by using TRIZOL (Invitrogen), as described in the manufacturer's protocol. cDNA was synthesized from 1 μg of total RNA using Superscript III reverse transcriptase (Invitrogen) with an oligo dT primer. Polymerase chain reaction (PCR) amplification for gene expression profiles were carried out using 10–100 ng of cDNA in a 25-μl reaction mixture containing 1 × <em>ExTaq</em> buffer, 2.5 mM deoxynucleotide triphosphasteses, 10 pmol primers and 1.25 U <em>ExTaq</em> HS (TaKaRa) and were subjected to 30–35 PCR cycles; 96°C for 15 s, 60–65°C for 30 s and 72°C for 15–120 s depending on the length of PCR products at the ratio of 1 min/kb. PCR products were visualized by agarose gel electrophoresis with ethidium bromide staining. The primers used for the expression profiles were as follows: <em>PNMAMS1</em>-F1 (5′-AAC ATG GTG GAG GAG TCT GGA T-3′), <em>PNMAMS1</em>-R1 (5′-CAA CGG TAA GGT GAC CTC TTG G-3′), <em>wGAPDH</em>-F1 (5′-AGA AAG TGG TGA AGC AGG CAT-3′), <em>wGAPDH</em>-R1 (5′-TGG AGG ACA TGT AGA CCA TGA G-3′), <em>wGAPDH</em>-F2 (5′-CCT ACT CCA ATG TAT CTG TGT-3′), <em>wGAPDH</em>-R2 (5′-GGT GGA ACT CCT TTT TTG ACT G-3′), <em>LAMA3</em>-F1 (5′-ACT CTG CAA AGA TCA GCA CAC C-3′), <em>LAMA3</em>-R1 (5′-CTC CTG CCT TCA GCA AGA AGA T-3′); <em>PNMAMS2</em>-F1 (5′-GGC TAA TGG AAA GTC ATA AGA AAG C-3′), <em>PNMAMS2</em>-R1 (5′-GAT TCC TTG ATA CAA ATG GTT GTC C-3′), <em>PNMAMS2</em>-F2 (5′-TTG ATG CAT TGT CTG AAA CCA G-3′), <em>PNMAMS2</em>-R2 (5′-ATC TAT CAA CCA AGC GCC AAC T-3′); <em>oOAZ1</em>-F1 (5′-ATA AAC CCA GCA CCA CCG TCC ACG-3′), <em>oOAZ1</em>-R1 (5′-GGT CTC ACA ATC TCA AAG CCC AAA AAG-3′).</p>                            <h3 scrollto-destination=5174512 id="5174512" class="section-title" data-legacy-id=s2c>2.3. 5′- and 3′-RACE</h3>
<p>Rapid amplification of cDNA ends (RACE) reactions were performed with the tammar liver using the RNA SMARTER RACE cDNA Amplification kit (Clontech) according to the manufacturer's recommendations. The 5′- and 3′-RACE fragments were generated with the following gene-specific primers: <em>PNMAMS1</em>-5′RACE-GSP1 (5′-TGC GTA TGG AGG GGA GAG TGA GCA AG-3′) and <em>PNMAMS1</em>-3′RACE-GSP1 (5′-GAC TGT GCC ATC GGG AGA AGG TGA AC-3′), and nested PCR was performed with following primers: <em>PNMAMS1</em>-5′RACE-GSP2 (5′-CAG ACA AGG TGG GGT CTG TCT CTT C-3′) and <em>PNMAMS1</em>-3′-RACE-GSP2 (5′-TTC CTG TGA AGG TCT CCC TCT C-3′), respectively.</p>                            <h3 scrollto-destination=5174514 id="5174514" class="section-title" data-legacy-id=s2d>2.4. Detection and prediction of the Gypsy12_DR Gag-derived genes</h3>
<p>For the detection of <em>PNMA</em> family genes, we performed TBLASTN searches (<em>e</em>-value &lt;1.0E−9) using the NCBI server (<a class="link link-uri" href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">http://blast.ncbi.nlm.nih.gov/Blast.cgi</a>) against eutherian reference genomic sequences and marsupial WGS sequences using the Gypsy12-I_DR Gag protein sequence from Repbase (<a class="link link-uri" href="http://www.girinst.org/">http://www.girinst.org/</a>) as a query . After TBLASTN searches, sequences that encoded open reading frames (ORFs) with &gt;100 aa (amino acids) were selected for the next analysis. Secondary screening was performed with all the sequences that were selected by first screening as a query. In addition, only sequences encoding proteins with &gt;100 aa were considered to be candidate <em>PNMA</em> family genes and those with &lt;100 aa were considered as <em>PNMA</em> pseudogenes. Genome resources used were: <em>Homo sapiens</em> (GRCh37.p5), <em>Mus musculus</em> (MGSCv37), <em>M. eugenii</em> (Meug_1.1) and <em>M. domestica</em> (MonDom5). ORF prediction was performed using an ORF finder (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/gorf/gorf.html">http://www.ncbi.nlm.nih.gov/gorf/gorf.html</a>).</p>                            <h3 scrollto-destination=5174516 id="5174516" class="section-title" data-legacy-id=s2e>2.5. Multiple alignment and phylogenetic tree</h3>
<p><em>PNMA</em> family genes and retrotransposons from Repbase were aligned using the MEGA 5.0 (Molecular Evolutionary Genetics Analysis). Phylogenetic tree analysis was also performed using the MEGA 5.0. The tree was inferred using the neighbour-joining method with the bootstrap test (1000 replicates). The evolutionary distances were computed using the <em>p</em>-distance method and are in the units of the number of amino acid differences per site. All ambiguous positions were removed for each sequence pair.</p>                            <h3 scrollto-destination=5174518 id="5174518" class="section-title" data-legacy-id=s2f>2.6. Comparative genomic analysis</h3>
<p>For comparison of the marsupial <em>PNMA-MS1</em> and <em>-MS2</em> genomic regions with the corresponding regions in eutherian species, we extracted the following sequences from Ensembl (<a class="link link-uri" href="http://www.ensembl.org">http://www.ensembl.org</a>); <em>PNMA-MS1</em>; <em>M. eugenii</em> (GeneScaffold: Meug_1.0: 503: 24 720–39 177), <em>M. domestica</em> (Chromosome: MonDom5: 3: 260 858 116–260 881 268), <em>H. sapiens</em> (Chromosome: GRCh37: 18: 21 343 369–21 355 887), <em>M. musculus</em> (Chromosome: GRCm37: 18: 12 572 248–12 578 437), <em>Ornithorhynchus anatinus</em> (Chromosome: OANA5: 7: 17 676 195–17 684 806), <em>Gallus gallus</em> (Chromosome: WASHUC2: 2: 106 303 674–106 308 473), <em>Xenopus tropicalis</em> (Scaffold: JGI4.1: 84: 2 861 161–2 864 246), <em>T. rubripes</em> (Scaffold: FUGU4: 285: 98 142–99 106) and for <em>PNMA-MS2</em>; <em>M. domestica</em> (Chromosome: MonDom5: 1: 416 106 001–416 736 671), Tasmanian devil <em>Sarcophilus harrisii</em> (Scaffold: DEVIL_7.0: GL834637.1: 1–311 016), <em>H. sapiens</em> (Chromosome: GRCh37: 9: 125 122 856–125 594 315), <em>M. musculus</em> (Chromosome: GRCm37: 2: 36 078 175–37 218 455), <em>O. anatinus</em> (Chromosome: OANA5 : Ultra70 : 222 463–282 938), <em>G. gallus</em> (Chromosome: WASHUC2: 17: 9 467 383–9 508 331), <em>X. tropicalis</em> (Scaffold: JGI_4.2: GL173356.1: 228 227–292 347) and <em>T. rubripes</em> (Scaffold: FUGU4: scaffold_49: 144 872–154 417).</p><p>Alignments were obtained using the VISTA Web server (<a class="link link-uri" href="http://genome.lbl.gov/vista/">http://genome.lbl.gov/vista/</a>). <em>PNMA-MS1</em> syntenic regions of several species identified above were aligned using the default setting (&gt;70% identity and &gt;100 bp in length) of mVISTA using the LAGAN global multiple alignment option.</p>                            <h2 scrollto-destination=5174521 id="5174521" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=5174522 id="5174522" class="section-title" data-legacy-id=s3a>3.1. Novel candidate <em>PNMA</em> genes in humans and mice</h3>
<p>We validated our approach to search for candidate <em>PNMA</em> genes in marsupials by performing TBLASTN analysis against human and mouse reference genomic sequences using the Gypsy12_DR Gag protein as a query. With a cut-off <em>e</em>-value of &lt;1.0E−9, this screening resulted in 19 and 15 candidates in the human and mouse genomes, respectively. In humans, 15 of the 19 were known <em>PNMA</em> genes and the remaining 4 were novel putative <em>PNMA</em> genes, <em>PNMA7/LOC649201</em>, <em>PNMA8/LOC649238</em>, <em>PNMA9/LOC100128960</em> and <em>PNMA16</em> (Table <a reveal-id="DST020TB1" data-open="DST020TB1" class="link link-reveal link-table xref-fig">1</a>, Humans). In mice, 12 of the 15 were known, with two novel putative <em>PNMA</em> genes, <em>Pnma7/Gm7028</em> and <em>Pnma9/Gm6858</em>, and one pseudogene, <em>Gm 1832215</em> identified (Table <a reveal-id="DST020TB1" data-open="DST020TB1" class="link link-reveal link-table xref-fig">1</a>, Mouse). </p>                        <a id="5174524" scrollto-destination="5174524"></a>
<div content-id="DST020TB1" class="table-modal"><div class="table-wrap"><div id="DST020TB1" class="table-wrap-title" data-id="DST020TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Candidate list for PNMA family genes in two eutherian and two marsupial species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>PNMA number </th><th>Gene name </th><th>Accession number </th><th>Location </th></tr></thead><tbody><tr><td rowspan="19">Human </td><td>hsPNMA1 </td><td>PNMA1 </td><td>NM_006029.4 </td><td>chr.14:74178486-74181128 </td></tr><tr><td>hsPNMA2 </td><td>PNMA2 </td><td>NM_007257.5 </td><td>chr.8:26362196-26371483 </td></tr><tr><td>hsPNMA3 </td><td>PNMA3 </td><td>NM_013364.4 </td><td>chr.X:152224766-152228827 </td></tr><tr><td>hsPNMA4 </td><td>PNMA4 </td><td>NM_022151.4 </td><td>chr.14:93648541-93651249 </td></tr><tr><td>hsPNMA5 </td><td>PNMA5 </td><td>NM_052926.2 </td><td>chr.X:152157368-152162671 </td></tr><tr><td>hsPNMA6A </td><td>PNMA6A </td><td>NM_032882.4 </td><td>chr.X:152338301-152340107 </td></tr><tr><td>hsPNMA6B </td><td>PNMA6B </td><td>XM_002343859.2 </td><td>chr.X:152341614-152342813 </td></tr><tr><td>hsPNMA6D </td><td>PNMA6D </td><td>XM_002343858.2 </td><td>chr.X:152244152-152246070 </td></tr><tr><td>hsPNMA6C </td><td>PNMA6C </td><td>NM_001170944.1 </td><td>chr.X:152240819-152243402 </td></tr><tr><td>hsPNMA7 </td><td>LOC649201 </td><td>XP_001127211 </td><td>chr.X:152584221-152587591 </td></tr><tr><td>hsPNMA8 </td><td>LOC649238 </td><td>XM_938309.4 </td><td>chr.X:152662364-152663269 </td></tr><tr><td>hsPNMA9 </td><td>LOC100128960 </td><td>— </td><td>chr.X:152197130-152200901 </td></tr><tr><td>hsPNMA10 </td><td>ZCCHC12 </td><td>NM_173798.2 </td><td>chr.X:117957787-117960931 </td></tr><tr><td>hsPNMA11 </td><td>ZCCHC18 </td><td>NM_001143978.1 </td><td>chr.X:103357107-103360533 </td></tr><tr><td>hsPNMA12 </td><td>PNMAL1 </td><td>NM_001103149.1 </td><td>chr.19:46969748-46974820 </td></tr><tr><td>hsPNMA13 </td><td>PNMAL2 </td><td>NM_020709.1 </td><td>chr.19:46994448-46999169 </td></tr><tr><td>hsPNMA14 </td><td>CCDC8 </td><td>NM_032040.3 </td><td>chr.19:46913586-46916919 </td></tr><tr><td>hsPNMA15 </td><td>— </td><td>— </td><td>chr.19:46931182-46931595 </td></tr><tr><td>hsPNMA16 </td><td>— </td><td>— </td><td>chr.19:47036933-47037357 </td></tr><tr><td rowspan="15">Mouse </td><td>mmPNMA1 </td><td>Pnma1 </td><td>NM_027438.3 </td><td>chr.12:85487081-85489439 </td></tr><tr><td>mmPNMA2 </td><td>Pnma2 </td><td>NM_175498.4 </td><td>chr.14:67530045-67538898 </td></tr><tr><td>mmPNMA3 </td><td>Pnma3 </td><td>NM_153169.2 </td><td>chr.X:70310126-70313530 </td></tr><tr><td>mmPNMA4 </td><td>Pnma4 </td><td>NM_001142937.1 </td><td>chr.12:103978040-103981870 </td></tr><tr><td>mmPNMA5 </td><td>Pnma5 </td><td>NM_001100461.3 </td><td>chr.X:70279327-70282442 </td></tr><tr><td>mmPNMA7 </td><td>Gm7028 </td><td>NG_005480.3 </td><td>chr.X;70580917-70581762 </td></tr><tr><td>mmPNMA8 </td><td>LOC100416956 </td><td>NG_017874 </td><td>chr.X:70642228-70644051 </td></tr><tr><td>mmPNMA9 </td><td>Gm6858 </td><td>NG_005479.2 </td><td>chr.X:70295221-70295900 </td></tr><tr><td>mmPNMA10 </td><td>Zcchc12 </td><td>NM_028325.3 </td><td>chr.X:33735899-33739153 </td></tr><tr><td>mmPNMA11 </td><td>Zcchc18 </td><td>NM_001035509.1 </td><td>chr.X:133527694-133531462 </td></tr><tr><td>mmPNMA12 </td><td>PNMAL1 </td><td>NM_001007569.1 </td><td>chr.7:17545144-17547669 </td></tr><tr><td>mmPNMA13 </td><td>PNMAL2 </td><td>NM_001099636.2 </td><td>chr.7:17530031-17532427 </td></tr><tr><td>mmPNMA14 </td><td>CCDC8 </td><td>NM_001101535.1 </td><td>chr.7:17579937-17581994 </td></tr><tr><td>mmPNMA15 </td><td>— </td><td>— </td><td>chr.7:17568964-17569311 </td></tr><tr><td>mmPNMA pseudo1 </td><td>Gm18322 </td><td>NC_000084.5 </td><td>chr.18:57308641-57309445 </td></tr><tr><td rowspan="20">Tammar </td><td>mePNMA-MS1 </td><td>mePNMA-MS1 </td><td>— </td><td>GeneScaffold_503:27168-31803 </td></tr><tr><td>mePNMA pseudo1 </td><td>— </td><td>— </td><td>Scaffold94060:2914-6134 </td></tr><tr><td>mePNMA pseudo2 </td><td>— </td><td>— </td><td>Scaffold1032:54408-58815 </td></tr><tr><td>mePNMA pseudo3 </td><td>— </td><td>— </td><td>Scaffold385831:1-1432 </td></tr><tr><td>mePNMA pseudo4 </td><td>— </td><td>— </td><td>Scaffold57604:6970-11278 </td></tr><tr><td>mePNMA pseudo5 </td><td>— </td><td>— </td><td>Scaffold428007:1-986 </td></tr><tr><td>mePNMA pseudo6 </td><td>— </td><td>— </td><td>Scaffold1439:20136-24450 </td></tr><tr><td>mePNMA pseudo7 </td><td>— </td><td>— </td><td>Scaffold391804:1-1797 </td></tr><tr><td>mePNMA pseudo8 </td><td>— </td><td>— </td><td>Scaffold46963:6594-10851 </td></tr><tr><td>mePNMA pseudo9 </td><td>— </td><td>— </td><td>Scaffold3242:41001-45246 </td></tr><tr><td>mePNMA pseudo10 </td><td>— </td><td>— </td><td>Scaffold407990:1-2479 </td></tr><tr><td>mePNMA pseudo11 </td><td>— </td><td>— </td><td>Scaffold799:60812-65099 </td></tr><tr><td>mePNMA pseudo12 </td><td>— </td><td>— </td><td>Scaffold492911:1-878 </td></tr><tr><td>mePNMA pseudo13 </td><td>— </td><td>— </td><td>Scaffold111753:1-2597 </td></tr><tr><td>mePNMA pseudo14 </td><td>— </td><td>— </td><td>Scaffold92201:2820-7152 </td></tr><tr><td>mePNMA pseudo15 </td><td>— </td><td>— </td><td>Scaffold9921:21555-26131 </td></tr><tr><td>mePNMA pseudo16 </td><td>— </td><td>— </td><td>Scaffold1103:4422-8826 </td></tr><tr><td>mePNMA pseudo17 </td><td>— </td><td>— </td><td>Scaffold27816:17763-21999 </td></tr><tr><td>mePNMA pseudo18 </td><td>— </td><td>— </td><td>Scaffold134268:3226-6573 </td></tr><tr><td>mePNMA pseudo19 </td><td>— </td><td>— </td><td>GeneScaffold_10085:47646-51858 </td></tr><tr><td rowspan="14">Opossum </td><td>mdPNMA-MS1 </td><td>mdPNMA-MS1 </td><td>— </td><td>chr.3:260874625-260879998 </td></tr><tr><td>mdPNMA-MS2 </td><td>mdPNMA-MS2 </td><td>— </td><td>chr.1:416409687-416414127 </td></tr><tr><td>mdPNMA pseudo1 </td><td>— </td><td>— </td><td>chr.3:15383915-15388463 </td></tr><tr><td>mdPNMA pseudo2 </td><td>— </td><td>— </td><td>chr.3:239729898-239734143 </td></tr><tr><td>mdPNMA pseudo3 </td><td>— </td><td>— </td><td>chr.1:692451545-692456282 </td></tr><tr><td>mdPNMA pseudo4 </td><td>— </td><td>— </td><td>chr.1:451204818-451209222 </td></tr><tr><td>mdPNMA pseudo5 </td><td>— </td><td>— </td><td>chr.1:331217342-331221920 </td></tr><tr><td>mdPNMA pseudo6 </td><td>— </td><td>— </td><td>chr.1:342376567-342381307 </td></tr><tr><td>mdPNMA pseudo7 </td><td>— </td><td>— </td><td>chr.X:6503247-6507903 </td></tr><tr><td>mdPNMA pseudo8 </td><td>— </td><td>— </td><td>chr.2:529194963-529199310 </td></tr><tr><td>mdPNMA pseudo9 </td><td>— </td><td>— </td><td>chr.2:271702858-271707121 </td></tr><tr><td>mdPNMA pseudo10 </td><td>— </td><td>— </td><td>chr.5:205686598-205690882 </td></tr><tr><td>mdPNMA pseudo11 </td><td>— </td><td>— </td><td>chr.5:229359947-229364330 </td></tr><tr><td>mdPNMA pseudo12 </td><td>— </td><td>— </td><td>chr.6:103527713-103532099 </td></tr></tbody></table></div><div content-id="DST020TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>PNMA number </th><th>Gene name </th><th>Accession number </th><th>Location </th></tr></thead><tbody><tr><td rowspan="19">Human </td><td>hsPNMA1 </td><td>PNMA1 </td><td>NM_006029.4 </td><td>chr.14:74178486-74181128 </td></tr><tr><td>hsPNMA2 </td><td>PNMA2 </td><td>NM_007257.5 </td><td>chr.8:26362196-26371483 </td></tr><tr><td>hsPNMA3 </td><td>PNMA3 </td><td>NM_013364.4 </td><td>chr.X:152224766-152228827 </td></tr><tr><td>hsPNMA4 </td><td>PNMA4 </td><td>NM_022151.4 </td><td>chr.14:93648541-93651249 </td></tr><tr><td>hsPNMA5 </td><td>PNMA5 </td><td>NM_052926.2 </td><td>chr.X:152157368-152162671 </td></tr><tr><td>hsPNMA6A </td><td>PNMA6A </td><td>NM_032882.4 </td><td>chr.X:152338301-152340107 </td></tr><tr><td>hsPNMA6B </td><td>PNMA6B </td><td>XM_002343859.2 </td><td>chr.X:152341614-152342813 </td></tr><tr><td>hsPNMA6D </td><td>PNMA6D </td><td>XM_002343858.2 </td><td>chr.X:152244152-152246070 </td></tr><tr><td>hsPNMA6C </td><td>PNMA6C </td><td>NM_001170944.1 </td><td>chr.X:152240819-152243402 </td></tr><tr><td>hsPNMA7 </td><td>LOC649201 </td><td>XP_001127211 </td><td>chr.X:152584221-152587591 </td></tr><tr><td>hsPNMA8 </td><td>LOC649238 </td><td>XM_938309.4 </td><td>chr.X:152662364-152663269 </td></tr><tr><td>hsPNMA9 </td><td>LOC100128960 </td><td>— </td><td>chr.X:152197130-152200901 </td></tr><tr><td>hsPNMA10 </td><td>ZCCHC12 </td><td>NM_173798.2 </td><td>chr.X:117957787-117960931 </td></tr><tr><td>hsPNMA11 </td><td>ZCCHC18 </td><td>NM_001143978.1 </td><td>chr.X:103357107-103360533 </td></tr><tr><td>hsPNMA12 </td><td>PNMAL1 </td><td>NM_001103149.1 </td><td>chr.19:46969748-46974820 </td></tr><tr><td>hsPNMA13 </td><td>PNMAL2 </td><td>NM_020709.1 </td><td>chr.19:46994448-46999169 </td></tr><tr><td>hsPNMA14 </td><td>CCDC8 </td><td>NM_032040.3 </td><td>chr.19:46913586-46916919 </td></tr><tr><td>hsPNMA15 </td><td>— </td><td>— </td><td>chr.19:46931182-46931595 </td></tr><tr><td>hsPNMA16 </td><td>— </td><td>— </td><td>chr.19:47036933-47037357 </td></tr><tr><td rowspan="15">Mouse </td><td>mmPNMA1 </td><td>Pnma1 </td><td>NM_027438.3 </td><td>chr.12:85487081-85489439 </td></tr><tr><td>mmPNMA2 </td><td>Pnma2 </td><td>NM_175498.4 </td><td>chr.14:67530045-67538898 </td></tr><tr><td>mmPNMA3 </td><td>Pnma3 </td><td>NM_153169.2 </td><td>chr.X:70310126-70313530 </td></tr><tr><td>mmPNMA4 </td><td>Pnma4 </td><td>NM_001142937.1 </td><td>chr.12:103978040-103981870 </td></tr><tr><td>mmPNMA5 </td><td>Pnma5 </td><td>NM_001100461.3 </td><td>chr.X:70279327-70282442 </td></tr><tr><td>mmPNMA7 </td><td>Gm7028 </td><td>NG_005480.3 </td><td>chr.X;70580917-70581762 </td></tr><tr><td>mmPNMA8 </td><td>LOC100416956 </td><td>NG_017874 </td><td>chr.X:70642228-70644051 </td></tr><tr><td>mmPNMA9 </td><td>Gm6858 </td><td>NG_005479.2 </td><td>chr.X:70295221-70295900 </td></tr><tr><td>mmPNMA10 </td><td>Zcchc12 </td><td>NM_028325.3 </td><td>chr.X:33735899-33739153 </td></tr><tr><td>mmPNMA11 </td><td>Zcchc18 </td><td>NM_001035509.1 </td><td>chr.X:133527694-133531462 </td></tr><tr><td>mmPNMA12 </td><td>PNMAL1 </td><td>NM_001007569.1 </td><td>chr.7:17545144-17547669 </td></tr><tr><td>mmPNMA13 </td><td>PNMAL2 </td><td>NM_001099636.2 </td><td>chr.7:17530031-17532427 </td></tr><tr><td>mmPNMA14 </td><td>CCDC8 </td><td>NM_001101535.1 </td><td>chr.7:17579937-17581994 </td></tr><tr><td>mmPNMA15 </td><td>— </td><td>— </td><td>chr.7:17568964-17569311 </td></tr><tr><td>mmPNMA pseudo1 </td><td>Gm18322 </td><td>NC_000084.5 </td><td>chr.18:57308641-57309445 </td></tr><tr><td rowspan="20">Tammar </td><td>mePNMA-MS1 </td><td>mePNMA-MS1 </td><td>— </td><td>GeneScaffold_503:27168-31803 </td></tr><tr><td>mePNMA pseudo1 </td><td>— </td><td>— </td><td>Scaffold94060:2914-6134 </td></tr><tr><td>mePNMA pseudo2 </td><td>— </td><td>— </td><td>Scaffold1032:54408-58815 </td></tr><tr><td>mePNMA pseudo3 </td><td>— </td><td>— </td><td>Scaffold385831:1-1432 </td></tr><tr><td>mePNMA pseudo4 </td><td>— </td><td>— </td><td>Scaffold57604:6970-11278 </td></tr><tr><td>mePNMA pseudo5 </td><td>— </td><td>— </td><td>Scaffold428007:1-986 </td></tr><tr><td>mePNMA pseudo6 </td><td>— </td><td>— </td><td>Scaffold1439:20136-24450 </td></tr><tr><td>mePNMA pseudo7 </td><td>— </td><td>— </td><td>Scaffold391804:1-1797 </td></tr><tr><td>mePNMA pseudo8 </td><td>— </td><td>— </td><td>Scaffold46963:6594-10851 </td></tr><tr><td>mePNMA pseudo9 </td><td>— </td><td>— </td><td>Scaffold3242:41001-45246 </td></tr><tr><td>mePNMA pseudo10 </td><td>— </td><td>— </td><td>Scaffold407990:1-2479 </td></tr><tr><td>mePNMA pseudo11 </td><td>— </td><td>— </td><td>Scaffold799:60812-65099 </td></tr><tr><td>mePNMA pseudo12 </td><td>— </td><td>— </td><td>Scaffold492911:1-878 </td></tr><tr><td>mePNMA pseudo13 </td><td>— </td><td>— </td><td>Scaffold111753:1-2597 </td></tr><tr><td>mePNMA pseudo14 </td><td>— </td><td>— </td><td>Scaffold92201:2820-7152 </td></tr><tr><td>mePNMA pseudo15 </td><td>— </td><td>— </td><td>Scaffold9921:21555-26131 </td></tr><tr><td>mePNMA pseudo16 </td><td>— </td><td>— </td><td>Scaffold1103:4422-8826 </td></tr><tr><td>mePNMA pseudo17 </td><td>— </td><td>— </td><td>Scaffold27816:17763-21999 </td></tr><tr><td>mePNMA pseudo18 </td><td>— </td><td>— </td><td>Scaffold134268:3226-6573 </td></tr><tr><td>mePNMA pseudo19 </td><td>— </td><td>— </td><td>GeneScaffold_10085:47646-51858 </td></tr><tr><td rowspan="14">Opossum </td><td>mdPNMA-MS1 </td><td>mdPNMA-MS1 </td><td>— </td><td>chr.3:260874625-260879998 </td></tr><tr><td>mdPNMA-MS2 </td><td>mdPNMA-MS2 </td><td>— </td><td>chr.1:416409687-416414127 </td></tr><tr><td>mdPNMA pseudo1 </td><td>— </td><td>— </td><td>chr.3:15383915-15388463 </td></tr><tr><td>mdPNMA pseudo2 </td><td>— </td><td>— </td><td>chr.3:239729898-239734143 </td></tr><tr><td>mdPNMA pseudo3 </td><td>— </td><td>— </td><td>chr.1:692451545-692456282 </td></tr><tr><td>mdPNMA pseudo4 </td><td>— </td><td>— </td><td>chr.1:451204818-451209222 </td></tr><tr><td>mdPNMA pseudo5 </td><td>— </td><td>— </td><td>chr.1:331217342-331221920 </td></tr><tr><td>mdPNMA pseudo6 </td><td>— </td><td>— </td><td>chr.1:342376567-342381307 </td></tr><tr><td>mdPNMA pseudo7 </td><td>— </td><td>— </td><td>chr.X:6503247-6507903 </td></tr><tr><td>mdPNMA pseudo8 </td><td>— </td><td>— </td><td>chr.2:529194963-529199310 </td></tr><tr><td>mdPNMA pseudo9 </td><td>— </td><td>— </td><td>chr.2:271702858-271707121 </td></tr><tr><td>mdPNMA pseudo10 </td><td>— </td><td>— </td><td>chr.5:205686598-205690882 </td></tr><tr><td>mdPNMA pseudo11 </td><td>— </td><td>— </td><td>chr.5:229359947-229364330 </td></tr><tr><td>mdPNMA pseudo12 </td><td>— </td><td>— </td><td>chr.6:103527713-103532099 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Candidates in humans, mouse, tammar wallaby and opossum. Newly identified <em>PNMA</em> family genes in this study are coloured in grey. ‘Pseudo’ denotes putative ORFs from sequences detected by TBLASTN, which encode &lt;100 aa.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5174524" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST020TB1" class="table-wrap-title" data-id="DST020TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Candidate list for PNMA family genes in two eutherian and two marsupial species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>PNMA number </th><th>Gene name </th><th>Accession number </th><th>Location </th></tr></thead><tbody><tr><td rowspan="19">Human </td><td>hsPNMA1 </td><td>PNMA1 </td><td>NM_006029.4 </td><td>chr.14:74178486-74181128 </td></tr><tr><td>hsPNMA2 </td><td>PNMA2 </td><td>NM_007257.5 </td><td>chr.8:26362196-26371483 </td></tr><tr><td>hsPNMA3 </td><td>PNMA3 </td><td>NM_013364.4 </td><td>chr.X:152224766-152228827 </td></tr><tr><td>hsPNMA4 </td><td>PNMA4 </td><td>NM_022151.4 </td><td>chr.14:93648541-93651249 </td></tr><tr><td>hsPNMA5 </td><td>PNMA5 </td><td>NM_052926.2 </td><td>chr.X:152157368-152162671 </td></tr><tr><td>hsPNMA6A </td><td>PNMA6A </td><td>NM_032882.4 </td><td>chr.X:152338301-152340107 </td></tr><tr><td>hsPNMA6B </td><td>PNMA6B </td><td>XM_002343859.2 </td><td>chr.X:152341614-152342813 </td></tr><tr><td>hsPNMA6D </td><td>PNMA6D </td><td>XM_002343858.2 </td><td>chr.X:152244152-152246070 </td></tr><tr><td>hsPNMA6C </td><td>PNMA6C </td><td>NM_001170944.1 </td><td>chr.X:152240819-152243402 </td></tr><tr><td>hsPNMA7 </td><td>LOC649201 </td><td>XP_001127211 </td><td>chr.X:152584221-152587591 </td></tr><tr><td>hsPNMA8 </td><td>LOC649238 </td><td>XM_938309.4 </td><td>chr.X:152662364-152663269 </td></tr><tr><td>hsPNMA9 </td><td>LOC100128960 </td><td>— </td><td>chr.X:152197130-152200901 </td></tr><tr><td>hsPNMA10 </td><td>ZCCHC12 </td><td>NM_173798.2 </td><td>chr.X:117957787-117960931 </td></tr><tr><td>hsPNMA11 </td><td>ZCCHC18 </td><td>NM_001143978.1 </td><td>chr.X:103357107-103360533 </td></tr><tr><td>hsPNMA12 </td><td>PNMAL1 </td><td>NM_001103149.1 </td><td>chr.19:46969748-46974820 </td></tr><tr><td>hsPNMA13 </td><td>PNMAL2 </td><td>NM_020709.1 </td><td>chr.19:46994448-46999169 </td></tr><tr><td>hsPNMA14 </td><td>CCDC8 </td><td>NM_032040.3 </td><td>chr.19:46913586-46916919 </td></tr><tr><td>hsPNMA15 </td><td>— </td><td>— </td><td>chr.19:46931182-46931595 </td></tr><tr><td>hsPNMA16 </td><td>— </td><td>— </td><td>chr.19:47036933-47037357 </td></tr><tr><td rowspan="15">Mouse </td><td>mmPNMA1 </td><td>Pnma1 </td><td>NM_027438.3 </td><td>chr.12:85487081-85489439 </td></tr><tr><td>mmPNMA2 </td><td>Pnma2 </td><td>NM_175498.4 </td><td>chr.14:67530045-67538898 </td></tr><tr><td>mmPNMA3 </td><td>Pnma3 </td><td>NM_153169.2 </td><td>chr.X:70310126-70313530 </td></tr><tr><td>mmPNMA4 </td><td>Pnma4 </td><td>NM_001142937.1 </td><td>chr.12:103978040-103981870 </td></tr><tr><td>mmPNMA5 </td><td>Pnma5 </td><td>NM_001100461.3 </td><td>chr.X:70279327-70282442 </td></tr><tr><td>mmPNMA7 </td><td>Gm7028 </td><td>NG_005480.3 </td><td>chr.X;70580917-70581762 </td></tr><tr><td>mmPNMA8 </td><td>LOC100416956 </td><td>NG_017874 </td><td>chr.X:70642228-70644051 </td></tr><tr><td>mmPNMA9 </td><td>Gm6858 </td><td>NG_005479.2 </td><td>chr.X:70295221-70295900 </td></tr><tr><td>mmPNMA10 </td><td>Zcchc12 </td><td>NM_028325.3 </td><td>chr.X:33735899-33739153 </td></tr><tr><td>mmPNMA11 </td><td>Zcchc18 </td><td>NM_001035509.1 </td><td>chr.X:133527694-133531462 </td></tr><tr><td>mmPNMA12 </td><td>PNMAL1 </td><td>NM_001007569.1 </td><td>chr.7:17545144-17547669 </td></tr><tr><td>mmPNMA13 </td><td>PNMAL2 </td><td>NM_001099636.2 </td><td>chr.7:17530031-17532427 </td></tr><tr><td>mmPNMA14 </td><td>CCDC8 </td><td>NM_001101535.1 </td><td>chr.7:17579937-17581994 </td></tr><tr><td>mmPNMA15 </td><td>— </td><td>— </td><td>chr.7:17568964-17569311 </td></tr><tr><td>mmPNMA pseudo1 </td><td>Gm18322 </td><td>NC_000084.5 </td><td>chr.18:57308641-57309445 </td></tr><tr><td rowspan="20">Tammar </td><td>mePNMA-MS1 </td><td>mePNMA-MS1 </td><td>— </td><td>GeneScaffold_503:27168-31803 </td></tr><tr><td>mePNMA pseudo1 </td><td>— </td><td>— </td><td>Scaffold94060:2914-6134 </td></tr><tr><td>mePNMA pseudo2 </td><td>— </td><td>— </td><td>Scaffold1032:54408-58815 </td></tr><tr><td>mePNMA pseudo3 </td><td>— </td><td>— </td><td>Scaffold385831:1-1432 </td></tr><tr><td>mePNMA pseudo4 </td><td>— </td><td>— </td><td>Scaffold57604:6970-11278 </td></tr><tr><td>mePNMA pseudo5 </td><td>— </td><td>— </td><td>Scaffold428007:1-986 </td></tr><tr><td>mePNMA pseudo6 </td><td>— </td><td>— </td><td>Scaffold1439:20136-24450 </td></tr><tr><td>mePNMA pseudo7 </td><td>— </td><td>— </td><td>Scaffold391804:1-1797 </td></tr><tr><td>mePNMA pseudo8 </td><td>— </td><td>— </td><td>Scaffold46963:6594-10851 </td></tr><tr><td>mePNMA pseudo9 </td><td>— </td><td>— </td><td>Scaffold3242:41001-45246 </td></tr><tr><td>mePNMA pseudo10 </td><td>— </td><td>— </td><td>Scaffold407990:1-2479 </td></tr><tr><td>mePNMA pseudo11 </td><td>— </td><td>— </td><td>Scaffold799:60812-65099 </td></tr><tr><td>mePNMA pseudo12 </td><td>— </td><td>— </td><td>Scaffold492911:1-878 </td></tr><tr><td>mePNMA pseudo13 </td><td>— </td><td>— </td><td>Scaffold111753:1-2597 </td></tr><tr><td>mePNMA pseudo14 </td><td>— </td><td>— </td><td>Scaffold92201:2820-7152 </td></tr><tr><td>mePNMA pseudo15 </td><td>— </td><td>— </td><td>Scaffold9921:21555-26131 </td></tr><tr><td>mePNMA pseudo16 </td><td>— </td><td>— </td><td>Scaffold1103:4422-8826 </td></tr><tr><td>mePNMA pseudo17 </td><td>— </td><td>— </td><td>Scaffold27816:17763-21999 </td></tr><tr><td>mePNMA pseudo18 </td><td>— </td><td>— </td><td>Scaffold134268:3226-6573 </td></tr><tr><td>mePNMA pseudo19 </td><td>— </td><td>— </td><td>GeneScaffold_10085:47646-51858 </td></tr><tr><td rowspan="14">Opossum </td><td>mdPNMA-MS1 </td><td>mdPNMA-MS1 </td><td>— </td><td>chr.3:260874625-260879998 </td></tr><tr><td>mdPNMA-MS2 </td><td>mdPNMA-MS2 </td><td>— </td><td>chr.1:416409687-416414127 </td></tr><tr><td>mdPNMA pseudo1 </td><td>— </td><td>— </td><td>chr.3:15383915-15388463 </td></tr><tr><td>mdPNMA pseudo2 </td><td>— </td><td>— </td><td>chr.3:239729898-239734143 </td></tr><tr><td>mdPNMA pseudo3 </td><td>— </td><td>— </td><td>chr.1:692451545-692456282 </td></tr><tr><td>mdPNMA pseudo4 </td><td>— </td><td>— </td><td>chr.1:451204818-451209222 </td></tr><tr><td>mdPNMA pseudo5 </td><td>— </td><td>— </td><td>chr.1:331217342-331221920 </td></tr><tr><td>mdPNMA pseudo6 </td><td>— </td><td>— </td><td>chr.1:342376567-342381307 </td></tr><tr><td>mdPNMA pseudo7 </td><td>— </td><td>— </td><td>chr.X:6503247-6507903 </td></tr><tr><td>mdPNMA pseudo8 </td><td>— </td><td>— </td><td>chr.2:529194963-529199310 </td></tr><tr><td>mdPNMA pseudo9 </td><td>— </td><td>— </td><td>chr.2:271702858-271707121 </td></tr><tr><td>mdPNMA pseudo10 </td><td>— </td><td>— </td><td>chr.5:205686598-205690882 </td></tr><tr><td>mdPNMA pseudo11 </td><td>— </td><td>— </td><td>chr.5:229359947-229364330 </td></tr><tr><td>mdPNMA pseudo12 </td><td>— </td><td>— </td><td>chr.6:103527713-103532099 </td></tr></tbody></table></div><div content-id="DST020TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>PNMA number </th><th>Gene name </th><th>Accession number </th><th>Location </th></tr></thead><tbody><tr><td rowspan="19">Human </td><td>hsPNMA1 </td><td>PNMA1 </td><td>NM_006029.4 </td><td>chr.14:74178486-74181128 </td></tr><tr><td>hsPNMA2 </td><td>PNMA2 </td><td>NM_007257.5 </td><td>chr.8:26362196-26371483 </td></tr><tr><td>hsPNMA3 </td><td>PNMA3 </td><td>NM_013364.4 </td><td>chr.X:152224766-152228827 </td></tr><tr><td>hsPNMA4 </td><td>PNMA4 </td><td>NM_022151.4 </td><td>chr.14:93648541-93651249 </td></tr><tr><td>hsPNMA5 </td><td>PNMA5 </td><td>NM_052926.2 </td><td>chr.X:152157368-152162671 </td></tr><tr><td>hsPNMA6A </td><td>PNMA6A </td><td>NM_032882.4 </td><td>chr.X:152338301-152340107 </td></tr><tr><td>hsPNMA6B </td><td>PNMA6B </td><td>XM_002343859.2 </td><td>chr.X:152341614-152342813 </td></tr><tr><td>hsPNMA6D </td><td>PNMA6D </td><td>XM_002343858.2 </td><td>chr.X:152244152-152246070 </td></tr><tr><td>hsPNMA6C </td><td>PNMA6C </td><td>NM_001170944.1 </td><td>chr.X:152240819-152243402 </td></tr><tr><td>hsPNMA7 </td><td>LOC649201 </td><td>XP_001127211 </td><td>chr.X:152584221-152587591 </td></tr><tr><td>hsPNMA8 </td><td>LOC649238 </td><td>XM_938309.4 </td><td>chr.X:152662364-152663269 </td></tr><tr><td>hsPNMA9 </td><td>LOC100128960 </td><td>— </td><td>chr.X:152197130-152200901 </td></tr><tr><td>hsPNMA10 </td><td>ZCCHC12 </td><td>NM_173798.2 </td><td>chr.X:117957787-117960931 </td></tr><tr><td>hsPNMA11 </td><td>ZCCHC18 </td><td>NM_001143978.1 </td><td>chr.X:103357107-103360533 </td></tr><tr><td>hsPNMA12 </td><td>PNMAL1 </td><td>NM_001103149.1 </td><td>chr.19:46969748-46974820 </td></tr><tr><td>hsPNMA13 </td><td>PNMAL2 </td><td>NM_020709.1 </td><td>chr.19:46994448-46999169 </td></tr><tr><td>hsPNMA14 </td><td>CCDC8 </td><td>NM_032040.3 </td><td>chr.19:46913586-46916919 </td></tr><tr><td>hsPNMA15 </td><td>— </td><td>— </td><td>chr.19:46931182-46931595 </td></tr><tr><td>hsPNMA16 </td><td>— </td><td>— </td><td>chr.19:47036933-47037357 </td></tr><tr><td rowspan="15">Mouse </td><td>mmPNMA1 </td><td>Pnma1 </td><td>NM_027438.3 </td><td>chr.12:85487081-85489439 </td></tr><tr><td>mmPNMA2 </td><td>Pnma2 </td><td>NM_175498.4 </td><td>chr.14:67530045-67538898 </td></tr><tr><td>mmPNMA3 </td><td>Pnma3 </td><td>NM_153169.2 </td><td>chr.X:70310126-70313530 </td></tr><tr><td>mmPNMA4 </td><td>Pnma4 </td><td>NM_001142937.1 </td><td>chr.12:103978040-103981870 </td></tr><tr><td>mmPNMA5 </td><td>Pnma5 </td><td>NM_001100461.3 </td><td>chr.X:70279327-70282442 </td></tr><tr><td>mmPNMA7 </td><td>Gm7028 </td><td>NG_005480.3 </td><td>chr.X;70580917-70581762 </td></tr><tr><td>mmPNMA8 </td><td>LOC100416956 </td><td>NG_017874 </td><td>chr.X:70642228-70644051 </td></tr><tr><td>mmPNMA9 </td><td>Gm6858 </td><td>NG_005479.2 </td><td>chr.X:70295221-70295900 </td></tr><tr><td>mmPNMA10 </td><td>Zcchc12 </td><td>NM_028325.3 </td><td>chr.X:33735899-33739153 </td></tr><tr><td>mmPNMA11 </td><td>Zcchc18 </td><td>NM_001035509.1 </td><td>chr.X:133527694-133531462 </td></tr><tr><td>mmPNMA12 </td><td>PNMAL1 </td><td>NM_001007569.1 </td><td>chr.7:17545144-17547669 </td></tr><tr><td>mmPNMA13 </td><td>PNMAL2 </td><td>NM_001099636.2 </td><td>chr.7:17530031-17532427 </td></tr><tr><td>mmPNMA14 </td><td>CCDC8 </td><td>NM_001101535.1 </td><td>chr.7:17579937-17581994 </td></tr><tr><td>mmPNMA15 </td><td>— </td><td>— </td><td>chr.7:17568964-17569311 </td></tr><tr><td>mmPNMA pseudo1 </td><td>Gm18322 </td><td>NC_000084.5 </td><td>chr.18:57308641-57309445 </td></tr><tr><td rowspan="20">Tammar </td><td>mePNMA-MS1 </td><td>mePNMA-MS1 </td><td>— </td><td>GeneScaffold_503:27168-31803 </td></tr><tr><td>mePNMA pseudo1 </td><td>— </td><td>— </td><td>Scaffold94060:2914-6134 </td></tr><tr><td>mePNMA pseudo2 </td><td>— </td><td>— </td><td>Scaffold1032:54408-58815 </td></tr><tr><td>mePNMA pseudo3 </td><td>— </td><td>— </td><td>Scaffold385831:1-1432 </td></tr><tr><td>mePNMA pseudo4 </td><td>— </td><td>— </td><td>Scaffold57604:6970-11278 </td></tr><tr><td>mePNMA pseudo5 </td><td>— </td><td>— </td><td>Scaffold428007:1-986 </td></tr><tr><td>mePNMA pseudo6 </td><td>— </td><td>— </td><td>Scaffold1439:20136-24450 </td></tr><tr><td>mePNMA pseudo7 </td><td>— </td><td>— </td><td>Scaffold391804:1-1797 </td></tr><tr><td>mePNMA pseudo8 </td><td>— </td><td>— </td><td>Scaffold46963:6594-10851 </td></tr><tr><td>mePNMA pseudo9 </td><td>— </td><td>— </td><td>Scaffold3242:41001-45246 </td></tr><tr><td>mePNMA pseudo10 </td><td>— </td><td>— </td><td>Scaffold407990:1-2479 </td></tr><tr><td>mePNMA pseudo11 </td><td>— </td><td>— </td><td>Scaffold799:60812-65099 </td></tr><tr><td>mePNMA pseudo12 </td><td>— </td><td>— </td><td>Scaffold492911:1-878 </td></tr><tr><td>mePNMA pseudo13 </td><td>— </td><td>— </td><td>Scaffold111753:1-2597 </td></tr><tr><td>mePNMA pseudo14 </td><td>— </td><td>— </td><td>Scaffold92201:2820-7152 </td></tr><tr><td>mePNMA pseudo15 </td><td>— </td><td>— </td><td>Scaffold9921:21555-26131 </td></tr><tr><td>mePNMA pseudo16 </td><td>— </td><td>— </td><td>Scaffold1103:4422-8826 </td></tr><tr><td>mePNMA pseudo17 </td><td>— </td><td>— </td><td>Scaffold27816:17763-21999 </td></tr><tr><td>mePNMA pseudo18 </td><td>— </td><td>— </td><td>Scaffold134268:3226-6573 </td></tr><tr><td>mePNMA pseudo19 </td><td>— </td><td>— </td><td>GeneScaffold_10085:47646-51858 </td></tr><tr><td rowspan="14">Opossum </td><td>mdPNMA-MS1 </td><td>mdPNMA-MS1 </td><td>— </td><td>chr.3:260874625-260879998 </td></tr><tr><td>mdPNMA-MS2 </td><td>mdPNMA-MS2 </td><td>— </td><td>chr.1:416409687-416414127 </td></tr><tr><td>mdPNMA pseudo1 </td><td>— </td><td>— </td><td>chr.3:15383915-15388463 </td></tr><tr><td>mdPNMA pseudo2 </td><td>— </td><td>— </td><td>chr.3:239729898-239734143 </td></tr><tr><td>mdPNMA pseudo3 </td><td>— </td><td>— </td><td>chr.1:692451545-692456282 </td></tr><tr><td>mdPNMA pseudo4 </td><td>— </td><td>— </td><td>chr.1:451204818-451209222 </td></tr><tr><td>mdPNMA pseudo5 </td><td>— </td><td>— </td><td>chr.1:331217342-331221920 </td></tr><tr><td>mdPNMA pseudo6 </td><td>— </td><td>— </td><td>chr.1:342376567-342381307 </td></tr><tr><td>mdPNMA pseudo7 </td><td>— </td><td>— </td><td>chr.X:6503247-6507903 </td></tr><tr><td>mdPNMA pseudo8 </td><td>— </td><td>— </td><td>chr.2:529194963-529199310 </td></tr><tr><td>mdPNMA pseudo9 </td><td>— </td><td>— </td><td>chr.2:271702858-271707121 </td></tr><tr><td>mdPNMA pseudo10 </td><td>— </td><td>— </td><td>chr.5:205686598-205690882 </td></tr><tr><td>mdPNMA pseudo11 </td><td>— </td><td>— </td><td>chr.5:229359947-229364330 </td></tr><tr><td>mdPNMA pseudo12 </td><td>— </td><td>— </td><td>chr.6:103527713-103532099 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Candidates in humans, mouse, tammar wallaby and opossum. Newly identified <em>PNMA</em> family genes in this study are coloured in grey. ‘Pseudo’ denotes putative ORFs from sequences detected by TBLASTN, which encode &lt;100 aa.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5174524" target="_blank">View Large</a></div></div><p>The putative human <em>PNMA7</em>–<em>9</em> genes were located near <em>PNMA6A–D</em> cluster on Chromosome Xq28. There is a sequence gap between <em>PNMA6A–B</em> and <em>6C</em>–<em>D</em>, so additional <em>PNMA</em> genes may exist in this region (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). The high homology (47–57%) between the putative amino acid sequences of the <em>PNMA7–9</em> and <em>PNMA6A–D</em> genes suggested that they share a common ancestor and evolved by gene duplication (see Fig. <a reveal-id="DST020F3" data-open="DST020F3" class="link link-reveal link-table xref-fig">3</a>). The putative murine <em>Pnma7</em>, <em>8</em> and 9 are all located in the orthologous region on the X chromosome. However, the <em>PNMA6</em> cluster is absent from the mouse genome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). This region is occupied by the X-linked leucocyte-regulated complex (<em>Xlr</em>) gene cluster.</p>                            <h3 scrollto-destination=5174526 id="5174526" class="section-title" data-legacy-id=s3b>3.2. Identification of novel PNMA genes in marsupials</h3>
<p>A comprehensive search of the tammar wallaby (Meug_1.1) and opossum WGS (MonDom5) for <em>PNMA</em> genes was then undertaken using the same method as the human and mouse above. Twenty and 14 hits were returned for TBLASTN searches of the tammar and opossum genomes, respectively (Table <a reveal-id="DST020TB1" data-open="DST020TB1" class="link link-reveal link-table xref-fig">1</a>, Tammar wallaby and Opossum). However, most of the sequences were predicted to be pseudogenes or remnants of the original retrotransposons (&lt;100 aa). Only ORFs predicted to encode &gt;100 aa were considered to be marsupial <em>PNMA</em> candidate genes. One candidate exhibited the highest homology to the Gypsy12_DR Gag protein along with matrix (MA), N- and C-terminal parts of capsid like (N- and C-CA) and cys-cys-his-cys (CCHC) zinc finger domains and had a putative ORF consisting of 456 and 458 aa in the tammar and opossum, respectively (Fig. <a reveal-id="DST020F1" data-open="DST020F1" class="link link-reveal link-table xref-fig">1</a>). Therefore, we named it <em>PNMA-MS1</em> as a novel marsupial-specific <em>PNMA</em> gene. The marsupial <em>PNMA-MS1</em> gene was located on a syntenic segment in the tammar (Gene scaffold_503:27168-31803) and the opossum (Chr.3: 260 874 625–260 879 998). A second <em>PNMA</em> candidate was identified in the opossum, <em>PNMA-MS2</em>, that had a putative ORF encoding 112 aa with high homology only to a central part of the capsid-like domain of the Gypsy12_DR Gag. It was located on Chromosome 1: 416 409 687–416 414 127, where an olfactory receptor (OR) gene cluster exists (see below). The presence of <em>PNMA-MS2</em> in the tammar was inconclusive due to the incomplete assembly of the corresponding region of the genome. </p>                        <a id="5174528" scrollto-destination="5174528"></a>
<div id="DST020F1" data-id="DST020F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/m_dst02001.jpeg?Expires=2147483647&Signature=JVsdssdHotTqStp-QUuoP5--7fsZMcx1sG0r2GyJ6JUQGU1864wGu3XoXc~~ILYvwux1W1VG6yOUSCPiMgGw0l9-EujwHAYJurP2wJSJBGuTucuI6wg4HcIIY8-hBG6TQdAR4Ix6w36psych4Rh~YjrNyVPUuP9~F1-8ZHbuiUFxcee4otuzP5E0faHaqgb6wPtv2pLgzODxaWyno9sotymMRtbspK06NdRkrv0xS8QWXonwoh9rMQryCr6LYcoASzHr7s8CAqKvulrUO3N2lZ03-YCs06F09j9LvU3KFHHvCaPeMrdPL2iLUpunmu~uKNSekHFi2M98~pFlViuuSg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="PNMA-MS1 and -MS2 have high homology to the Gypsy12_DR retrotransposon Gag protein. Boxes represent ORFs. Regions with significant similarity to a Gag protein of a Gypsy12_DR retrotransposon are shown in colours. PNMA-MS1 encodes a Gag-like protein including several typical domains and motifs, but lacks a Pol-like protein and LTR sequences attached to either end of the retrotransposons. Although PNMA-MS2 has a possible protein-coding frame corresponding to a central part of the capsid-like domain of the Gypsy12_DR Gag, it seems to be a pseudogene because no expression was confirmed. MA: matrix domain; N-CA: an N-terminal part of capsid-like domain; C-CA: a C-terminal part of capsid-like domain; CCHC: a CCHC zinc finger motif for RNA-binding site; RT: reverse transcriptase domain; RNaseH: RNase H domain; INT: integrase domain, me: M. eugenii (tammar wallaby); md: M. domestica (grey short-tailed opossum)." path-from-xml="dst02001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5174528/dst02001.jpeg" path-from-xml="dst02001.jpeg" target="_blank">View large</a><a section="5174528" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/dst02001.jpeg?Expires=2147483647&Signature=4HKrbSn2rdxp5PgAMsIIHvRAEnVtmwZcLxXLEnbhaUPcMQs82y7z34ZraB5OGqEQVFchZLHOpByfXc1skuisfnAoYm4wEIHFLvUdKHKuSFmf13b9GCDmcAc89MkxKvtMHLsjtNElQYZeQN-EH8jId56owLbVVc4J~ZhWmHpWkjoIZaBomh4TMcnlTdhDqgTNfjzHEmPCqIoBEG52kp3IoNro8l7YJdVbx6WNC9dhai2jVpKMLrEFxFPObCU5ypRSR5EU~e1UX5evUFFbuUo6cPxHOxc2IDFydmH-SD44ybodpMLxM9TV6ctJe2QHFMQRJIwxiBqru107mRngYZhzRQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5174528&ar=359087&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>PNMA-MS1</em> and <em>-MS2</em> have high homology to the Gypsy12_DR retrotransposon Gag protein. Boxes represent ORFs. Regions with significant similarity to a Gag protein of a Gypsy12_DR retrotransposon are shown in colours. <em>PNMA-MS1</em> encodes a Gag-like protein including several typical domains and motifs, but lacks a Pol-like protein and LTR sequences attached to either end of the retrotransposons. Although <em>PNMA-MS2</em> has a possible protein-coding frame corresponding to a central part of the capsid-like domain of the Gypsy12_DR Gag, it seems to be a pseudogene because no expression was confirmed. MA: matrix domain; N-CA: an N-terminal part of capsid-like domain; C-CA: a C-terminal part of capsid-like domain; CCHC: a CCHC zinc finger motif for RNA-binding site; RT: reverse transcriptase domain; RNaseH: RNase H domain; INT: integrase domain, <em>me</em>: <em>M. eugenii</em> (tammar wallaby); <em>md</em>: <em>M. domestica</em> (grey short-tailed opossum).</p></div></div><div content-id="DST020F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/m_dst02001.jpeg?Expires=2147483647&Signature=JVsdssdHotTqStp-QUuoP5--7fsZMcx1sG0r2GyJ6JUQGU1864wGu3XoXc~~ILYvwux1W1VG6yOUSCPiMgGw0l9-EujwHAYJurP2wJSJBGuTucuI6wg4HcIIY8-hBG6TQdAR4Ix6w36psych4Rh~YjrNyVPUuP9~F1-8ZHbuiUFxcee4otuzP5E0faHaqgb6wPtv2pLgzODxaWyno9sotymMRtbspK06NdRkrv0xS8QWXonwoh9rMQryCr6LYcoASzHr7s8CAqKvulrUO3N2lZ03-YCs06F09j9LvU3KFHHvCaPeMrdPL2iLUpunmu~uKNSekHFi2M98~pFlViuuSg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="PNMA-MS1 and -MS2 have high homology to the Gypsy12_DR retrotransposon Gag protein. Boxes represent ORFs. Regions with significant similarity to a Gag protein of a Gypsy12_DR retrotransposon are shown in colours. PNMA-MS1 encodes a Gag-like protein including several typical domains and motifs, but lacks a Pol-like protein and LTR sequences attached to either end of the retrotransposons. Although PNMA-MS2 has a possible protein-coding frame corresponding to a central part of the capsid-like domain of the Gypsy12_DR Gag, it seems to be a pseudogene because no expression was confirmed. MA: matrix domain; N-CA: an N-terminal part of capsid-like domain; C-CA: a C-terminal part of capsid-like domain; CCHC: a CCHC zinc finger motif for RNA-binding site; RT: reverse transcriptase domain; RNaseH: RNase H domain; INT: integrase domain, me: M. eugenii (tammar wallaby); md: M. domestica (grey short-tailed opossum)." path-from-xml="dst02001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5174528/dst02001.jpeg" path-from-xml="dst02001.jpeg" target="_blank">View large</a><a section="5174528" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/dst02001.jpeg?Expires=2147483647&Signature=4HKrbSn2rdxp5PgAMsIIHvRAEnVtmwZcLxXLEnbhaUPcMQs82y7z34ZraB5OGqEQVFchZLHOpByfXc1skuisfnAoYm4wEIHFLvUdKHKuSFmf13b9GCDmcAc89MkxKvtMHLsjtNElQYZeQN-EH8jId56owLbVVc4J~ZhWmHpWkjoIZaBomh4TMcnlTdhDqgTNfjzHEmPCqIoBEG52kp3IoNro8l7YJdVbx6WNC9dhai2jVpKMLrEFxFPObCU5ypRSR5EU~e1UX5evUFFbuUo6cPxHOxc2IDFydmH-SD44ybodpMLxM9TV6ctJe2QHFMQRJIwxiBqru107mRngYZhzRQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5174528&ar=359087&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>PNMA-MS1</em> and <em>-MS2</em> have high homology to the Gypsy12_DR retrotransposon Gag protein. Boxes represent ORFs. Regions with significant similarity to a Gag protein of a Gypsy12_DR retrotransposon are shown in colours. <em>PNMA-MS1</em> encodes a Gag-like protein including several typical domains and motifs, but lacks a Pol-like protein and LTR sequences attached to either end of the retrotransposons. Although <em>PNMA-MS2</em> has a possible protein-coding frame corresponding to a central part of the capsid-like domain of the Gypsy12_DR Gag, it seems to be a pseudogene because no expression was confirmed. MA: matrix domain; N-CA: an N-terminal part of capsid-like domain; C-CA: a C-terminal part of capsid-like domain; CCHC: a CCHC zinc finger motif for RNA-binding site; RT: reverse transcriptase domain; RNaseH: RNase H domain; INT: integrase domain, <em>me</em>: <em>M. eugenii</em> (tammar wallaby); <em>md</em>: <em>M. domestica</em> (grey short-tailed opossum).</p></div></div>                            <h3 scrollto-destination=5174529 id="5174529" class="section-title" data-legacy-id=s3c>3.3. Genomic structure of PNMA-MS1 in the tammar wallaby</h3>
<p>The full-length sequence of tammar <em>PNMA-MS1</em> consisting of 4290 bp was determined by 5′- and 3′-RACE. It has two exons and encodes a putative ORF encoding a 456 aa sequence (Fig. <a reveal-id="DST020F2" data-open="DST020F2" class="link link-reveal link-table xref-fig">2</a>). The PNMA-MS1 putative ORF shared 28% similarity at the amino acid level with the Gag protein of Gypsy12_DR retrotransposon (Fig. <a reveal-id="DST020F3" data-open="DST020F3" class="link link-reveal link-table xref-fig">3</a>A). Multiple alignment and phylogenetic tree analyses of the putative PNMA-MS1 and -MS2 amino acid sequences are shown in Fig. <a reveal-id="DST020F3" data-open="DST020F3" class="link link-reveal link-table xref-fig">3</a>A and B. The Pol protein and LTR regions are absent from PNMA-MS1 and -MS2, suggesting that they no longer have retrotranspositional activity (Fig. <a reveal-id="DST020F3" data-open="DST020F3" class="link link-reveal link-table xref-fig">3</a>A). The marsupial PNMA-MS1 and -MS2 protein sequences were grouped together and more closely related to the zebrafish Gypsy12_DR Gag than to the mouse and human proteins (Fig. <a reveal-id="DST020F3" data-open="DST020F3" class="link link-reveal link-table xref-fig">3</a>B). </p>                        <a id="5174531" scrollto-destination="5174531"></a>
<div id="DST020F2" data-id="DST020F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/m_dst02002.jpeg?Expires=2147483647&Signature=jYCU56MrdxShL-6EGC8CCK4QwySSq04INJfljYMyXV61sJz0J~ALVMfA6RZrFifiFfI~PhtQx0mca9L58F5tV5JGzon0lBPZRiB~v78~tIZ3BFEzUtx549V1vc8oDtRXupAEtohV7yECeO2g3W8XwCOz14iTXMj5VkE7zH62yYAh9IDzIVu~E7DfpcXZZCl-PCF1BoY0Ua22SDTvPPTWDw6PlZ4MeblPdm~5aG8bOhglQxdRVpS3aiXoi2V1ts1z0x~irUT6DTI~YlcevQmqWRD1S~c0oZgrYSwDhfFlL7F10BQYD3fSpHn6qd8M05KKkXPT2xaNzOQ4bf0EHBSfaw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genomic structure of full-length tammar wallaby PNMA-MS1. An arrow represents the direction of PNMA-MS1 transcription. UTR and ORF are indicated by light blue and dark blue boxes, respectively. There are no supporting data that the promoter of tammar PNMA-MS1 was derived from an LTR sequence of the original retrotransposon. UTR: untranslated region; ORF: open reading frame." path-from-xml="dst02002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5174531/dst02002.jpeg" path-from-xml="dst02002.jpeg" target="_blank">View large</a><a section="5174531" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/dst02002.jpeg?Expires=2147483647&Signature=gjYjfoLeYZn74upqUlj-rELWPE-tujaZWRAI888AJY064UZ2jyHFMDyVqMvJMU~wcnIt4aCZzKTFLBfJ49wI3jcEHtLYtPUA0paVzIsoejSguY8NxvY2ubqaBJZsg7cmqxbU~qsPauTnbZBZwJlhDWx2nkyWGErx36f9det473ErUn406yzXJzkd~N5hTSMWoAyuSaF3vuDSUHvwnpdIteKBQKMR~X1AAmsn7DoEI-DU2RF1QeVN04FE2A87kgqxFk-nqQ7EbztICBZll37c59njZgasrVwcjtGv-fKw7Fmfxc7f-kQLE3gOXLAEZOC078SvjyqAv4hPI1gkHfYObg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5174531&ar=359087&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genomic structure of full-length tammar wallaby <em>PNMA-MS1</em>. An arrow represents the direction of <em>PNMA-MS1</em> transcription. UTR and ORF are indicated by light blue and dark blue boxes, respectively. There are no supporting data that the promoter of tammar PNMA-MS1 was derived from an LTR sequence of the original retrotransposon. UTR: untranslated region; ORF: open reading frame.</p></div></div><div content-id="DST020F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/m_dst02002.jpeg?Expires=2147483647&Signature=jYCU56MrdxShL-6EGC8CCK4QwySSq04INJfljYMyXV61sJz0J~ALVMfA6RZrFifiFfI~PhtQx0mca9L58F5tV5JGzon0lBPZRiB~v78~tIZ3BFEzUtx549V1vc8oDtRXupAEtohV7yECeO2g3W8XwCOz14iTXMj5VkE7zH62yYAh9IDzIVu~E7DfpcXZZCl-PCF1BoY0Ua22SDTvPPTWDw6PlZ4MeblPdm~5aG8bOhglQxdRVpS3aiXoi2V1ts1z0x~irUT6DTI~YlcevQmqWRD1S~c0oZgrYSwDhfFlL7F10BQYD3fSpHn6qd8M05KKkXPT2xaNzOQ4bf0EHBSfaw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genomic structure of full-length tammar wallaby PNMA-MS1. An arrow represents the direction of PNMA-MS1 transcription. UTR and ORF are indicated by light blue and dark blue boxes, respectively. There are no supporting data that the promoter of tammar PNMA-MS1 was derived from an LTR sequence of the original retrotransposon. UTR: untranslated region; ORF: open reading frame." path-from-xml="dst02002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5174531/dst02002.jpeg" path-from-xml="dst02002.jpeg" target="_blank">View large</a><a section="5174531" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/dst02002.jpeg?Expires=2147483647&Signature=gjYjfoLeYZn74upqUlj-rELWPE-tujaZWRAI888AJY064UZ2jyHFMDyVqMvJMU~wcnIt4aCZzKTFLBfJ49wI3jcEHtLYtPUA0paVzIsoejSguY8NxvY2ubqaBJZsg7cmqxbU~qsPauTnbZBZwJlhDWx2nkyWGErx36f9det473ErUn406yzXJzkd~N5hTSMWoAyuSaF3vuDSUHvwnpdIteKBQKMR~X1AAmsn7DoEI-DU2RF1QeVN04FE2A87kgqxFk-nqQ7EbztICBZll37c59njZgasrVwcjtGv-fKw7Fmfxc7f-kQLE3gOXLAEZOC078SvjyqAv4hPI1gkHfYObg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5174531&ar=359087&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genomic structure of full-length tammar wallaby <em>PNMA-MS1</em>. An arrow represents the direction of <em>PNMA-MS1</em> transcription. UTR and ORF are indicated by light blue and dark blue boxes, respectively. There are no supporting data that the promoter of tammar PNMA-MS1 was derived from an LTR sequence of the original retrotransposon. UTR: untranslated region; ORF: open reading frame.</p></div></div>                        <a id="5174532" scrollto-destination="5174532"></a>
<div id="DST020F3" data-id="DST020F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/m_dst02003a.jpeg?Expires=2147483647&amp;Signature=ZGbEh8pP72aghSY1fN495QQgcPnEou5IJ0Ivyl2jS6UEJV0RfPWPHW3AJx7P0rGBy~wpqbZSIAFTeVE7J9MgVslaozBA9WsCl3LyJtUeRfivr~nBFc6gTxC8~OKN9k0h7hWXhR3XfL~ViiM3vJ1Oy5738mJ2ctPLmkpLaXjek-cgexyBKiR6Z-df9ZbOSuGWDjSpWPQvR1AR9tMc-kDgLgc9~bXSs0APzwJX7bpklClV0FTIoh6AOV8v5SdnhRr3Keawobgib-q-bHrBtfUuHUO2FDby95ZaJdolfKHwMjtvFXFZ8~HieHA4ehWpkJs7h2T7Ktd7LUvy5zD0KTcEbg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Multiple sequence alignment and phylogenetic tree of the PNMA family. (A) Multiple sequence alignment of the amino acid sequence of the Gag-like regions of marsupial PNMA-MS1, the human and mouse PNMA genes and Gypsy12_I_DR Gag. An evolutionarily conserved Gag-derived CX2CX4HX4C zinc finger motif is indicated by yellow shading. Residues conserved in all sequences are shaded black and highly conserved ones in grey. (B) A phylogenetic tree of PNMA family genes was constructed by the neighbour-joining method using the multiple alignment shown in Fig. 2. Bootstrap support (%) is shown for branches. hs: human:; mm: mouse; md: opossum; me: wallaby." path-from-xml="dst02003a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5174532/dst02003a.jpeg" path-from-xml="dst02003a.jpeg" target="_blank">View large</a><a section="5174532" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/dst02003a.jpeg?Expires=2147483647&amp;Signature=SNLUtKmn81gBh5Kdw5XBqAIEdtoWdlKLKwAwrkmvq47Y4a7xR02qME~-V8MjlzxpaVcumSvHrb6fcOoyzWN5t6~bI9PFdVQ1DsrLZ7wKP8uCmFVLVyenKCP-pPcgXrBiwAZZNFD~8moJkEQZpJDyKDimFWpqjDasvuAXsAsw-Bsm~kzgzXnFkQ9-WdzPQkj9AQ1VRZ6T-T2SKiaPKCBnfqeh3snx77I6I-2jbaGPaeS8wFI5yw7MWXrPU1FD7X6h2uqeDwba-94CUWQIKxYMjpifBMrHl56~UkmjsPsQ2P1mM~dK4z526IJKHPQrn3njrc92LYW-QpGetcebz8blLA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5174532&amp;ar=359087&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dst02003a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/m_dst02003b.jpeg?Expires=2147483647&amp;Signature=dgzM4zgz3z4-omEjd-DAcJmGmRssCV0lTzRnhbZrBJ~o6FYqDtyPm6v5KetC7qJAzmIsa-wzpetig7beM2WQAwXH1C1zladU8x1s9MWJ7iJQkGNSthXEjxu-LM1g-E7KIaXpeN2lnPIzeck5d7SFxFCMYfyU1NUb~OYdX6O5262CiPg7ScihqagCbFWETipCN5B2zOOQBb3rVmclytI7qXvzUzlPgIeKrAU~lOv9WLh62bjxjaXbyE0UFnooXjUedLSJttp19WldVYYNz4AH1kR1lEt-4q2oRoP67BgUTEk6UoY-wfY8RVyJN4j9Pwb5efy3wCehMDQpBCm1VEZ6Yg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Multiple sequence alignment and phylogenetic tree of the PNMA family. (A) Multiple sequence alignment of the amino acid sequence of the Gag-like regions of marsupial PNMA-MS1, the human and mouse PNMA genes and Gypsy12_I_DR Gag. An evolutionarily conserved Gag-derived CX2CX4HX4C zinc finger motif is indicated by yellow shading. Residues conserved in all sequences are shaded black and highly conserved ones in grey. (B) A phylogenetic tree of PNMA family genes was constructed by the neighbour-joining method using the multiple alignment shown in Fig. 2. Bootstrap support (%) is shown for branches. hs: human:; mm: mouse; md: opossum; me: wallaby." path-from-xml="dst02003b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5174532/dst02003b.jpeg" path-from-xml="dst02003b.jpeg" target="_blank">View large</a><a section="5174532" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/dst02003b.jpeg?Expires=2147483647&amp;Signature=gXzQNRIGduTGZ9y-fPFN0jdVg51Tj8doCtMThsUAuQOjxWlVXpamKYQSocJrlHjm3IoZoac9LIivPCtcu9WEEbdOd0Yq9VO~DP-PqK-9qIij6A87HnM~lHjENrF-z~E-vwFYcli0A4jFUHv7ZJaqBTXgR989nVN55hYUfBo41U1VicY1M6RwTXMqEwAeeYVqXmPdJ7x~5WTHAp5g-F0A16GnDc~67OTUtUJEbCaIHjH0Ea6I2-oANfEG4~8W0QbCc0MPz64Vl-RFxK~sdPyk~PI57wMRbp91Z1X5JKM86A~WCb0k4nSrDSFfS5mTxxAKtCP-JNs2i-I9X1-9sMW04g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5174532&amp;ar=359087&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dst02003b.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Multiple sequence alignment and phylogenetic tree of the <em>PNMA</em> family. (A) Multiple sequence alignment of the amino acid sequence of the Gag-like regions of marsupial <em>PNMA-MS1</em>, the human and mouse <em>PNMA</em> genes and Gypsy12_I_DR Gag. An evolutionarily conserved Gag-derived CX2CX4HX4C zinc finger motif is indicated by yellow shading. Residues conserved in all sequences are shaded black and highly conserved ones in grey. (B) A phylogenetic tree of <em>PNMA</em> family genes was constructed by the neighbour-joining method using the multiple alignment shown in Fig. <a reveal-id="DST020F2" data-open="DST020F2" class="link link-reveal link-table xref-fig">2</a>. Bootstrap support (%) is shown for branches. <em>hs</em>: human:; <em>mm</em>: mouse; <em>md</em>: opossum; <em>me</em>: wallaby.</p></div></div><div content-id="DST020F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/m_dst02003a.jpeg?Expires=2147483647&amp;Signature=ZGbEh8pP72aghSY1fN495QQgcPnEou5IJ0Ivyl2jS6UEJV0RfPWPHW3AJx7P0rGBy~wpqbZSIAFTeVE7J9MgVslaozBA9WsCl3LyJtUeRfivr~nBFc6gTxC8~OKN9k0h7hWXhR3XfL~ViiM3vJ1Oy5738mJ2ctPLmkpLaXjek-cgexyBKiR6Z-df9ZbOSuGWDjSpWPQvR1AR9tMc-kDgLgc9~bXSs0APzwJX7bpklClV0FTIoh6AOV8v5SdnhRr3Keawobgib-q-bHrBtfUuHUO2FDby95ZaJdolfKHwMjtvFXFZ8~HieHA4ehWpkJs7h2T7Ktd7LUvy5zD0KTcEbg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Multiple sequence alignment and phylogenetic tree of the PNMA family. (A) Multiple sequence alignment of the amino acid sequence of the Gag-like regions of marsupial PNMA-MS1, the human and mouse PNMA genes and Gypsy12_I_DR Gag. An evolutionarily conserved Gag-derived CX2CX4HX4C zinc finger motif is indicated by yellow shading. Residues conserved in all sequences are shaded black and highly conserved ones in grey. (B) A phylogenetic tree of PNMA family genes was constructed by the neighbour-joining method using the multiple alignment shown in Fig. 2. Bootstrap support (%) is shown for branches. hs: human:; mm: mouse; md: opossum; me: wallaby." path-from-xml="dst02003a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5174532/dst02003a.jpeg" path-from-xml="dst02003a.jpeg" target="_blank">View large</a><a section="5174532" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/dst02003a.jpeg?Expires=2147483647&amp;Signature=SNLUtKmn81gBh5Kdw5XBqAIEdtoWdlKLKwAwrkmvq47Y4a7xR02qME~-V8MjlzxpaVcumSvHrb6fcOoyzWN5t6~bI9PFdVQ1DsrLZ7wKP8uCmFVLVyenKCP-pPcgXrBiwAZZNFD~8moJkEQZpJDyKDimFWpqjDasvuAXsAsw-Bsm~kzgzXnFkQ9-WdzPQkj9AQ1VRZ6T-T2SKiaPKCBnfqeh3snx77I6I-2jbaGPaeS8wFI5yw7MWXrPU1FD7X6h2uqeDwba-94CUWQIKxYMjpifBMrHl56~UkmjsPsQ2P1mM~dK4z526IJKHPQrn3njrc92LYW-QpGetcebz8blLA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5174532&amp;ar=359087&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dst02003a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/m_dst02003b.jpeg?Expires=2147483647&amp;Signature=dgzM4zgz3z4-omEjd-DAcJmGmRssCV0lTzRnhbZrBJ~o6FYqDtyPm6v5KetC7qJAzmIsa-wzpetig7beM2WQAwXH1C1zladU8x1s9MWJ7iJQkGNSthXEjxu-LM1g-E7KIaXpeN2lnPIzeck5d7SFxFCMYfyU1NUb~OYdX6O5262CiPg7ScihqagCbFWETipCN5B2zOOQBb3rVmclytI7qXvzUzlPgIeKrAU~lOv9WLh62bjxjaXbyE0UFnooXjUedLSJttp19WldVYYNz4AH1kR1lEt-4q2oRoP67BgUTEk6UoY-wfY8RVyJN4j9Pwb5efy3wCehMDQpBCm1VEZ6Yg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Multiple sequence alignment and phylogenetic tree of the PNMA family. (A) Multiple sequence alignment of the amino acid sequence of the Gag-like regions of marsupial PNMA-MS1, the human and mouse PNMA genes and Gypsy12_I_DR Gag. An evolutionarily conserved Gag-derived CX2CX4HX4C zinc finger motif is indicated by yellow shading. Residues conserved in all sequences are shaded black and highly conserved ones in grey. (B) A phylogenetic tree of PNMA family genes was constructed by the neighbour-joining method using the multiple alignment shown in Fig. 2. Bootstrap support (%) is shown for branches. hs: human:; mm: mouse; md: opossum; me: wallaby." path-from-xml="dst02003b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5174532/dst02003b.jpeg" path-from-xml="dst02003b.jpeg" target="_blank">View large</a><a section="5174532" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/dst02003b.jpeg?Expires=2147483647&amp;Signature=gXzQNRIGduTGZ9y-fPFN0jdVg51Tj8doCtMThsUAuQOjxWlVXpamKYQSocJrlHjm3IoZoac9LIivPCtcu9WEEbdOd0Yq9VO~DP-PqK-9qIij6A87HnM~lHjENrF-z~E-vwFYcli0A4jFUHv7ZJaqBTXgR989nVN55hYUfBo41U1VicY1M6RwTXMqEwAeeYVqXmPdJ7x~5WTHAp5g-F0A16GnDc~67OTUtUJEbCaIHjH0Ea6I2-oANfEG4~8W0QbCc0MPz64Vl-RFxK~sdPyk~PI57wMRbp91Z1X5JKM86A~WCb0k4nSrDSFfS5mTxxAKtCP-JNs2i-I9X1-9sMW04g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5174532&amp;ar=359087&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dst02003b.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Multiple sequence alignment and phylogenetic tree of the <em>PNMA</em> family. (A) Multiple sequence alignment of the amino acid sequence of the Gag-like regions of marsupial <em>PNMA-MS1</em>, the human and mouse <em>PNMA</em> genes and Gypsy12_I_DR Gag. An evolutionarily conserved Gag-derived CX2CX4HX4C zinc finger motif is indicated by yellow shading. Residues conserved in all sequences are shaded black and highly conserved ones in grey. (B) A phylogenetic tree of <em>PNMA</em> family genes was constructed by the neighbour-joining method using the multiple alignment shown in Fig. <a reveal-id="DST020F2" data-open="DST020F2" class="link link-reveal link-table xref-fig">2</a>. Bootstrap support (%) is shown for branches. <em>hs</em>: human:; <em>mm</em>: mouse; <em>md</em>: opossum; <em>me</em>: wallaby.</p></div></div>                            <h3 scrollto-destination=5174533 id="5174533" class="section-title" data-legacy-id=s3d>3.4. Comparative genomic analysis of PNMA-MS1 and -MS2</h3>
<p>To elucidate whether <em>PNMA-MS1</em> is a marsupial-specific <em>PNMA</em> gene, comparative genomic analysis was performed using the VISTA tool with several vertebrate genomic sequences. <em>PNMA-MS1</em> was located in the intron 8 of the laminin alpha 3 (<em>LAMA3</em>) gene that is highly conserved in vertebrates. No <em>PNMA-MS1</em> orthologue was found in the syntenic region of any eutherian species, platypus (monotreme mammals), chicken (birds), frog (amphibian) and fugu (fish), demonstrating that <em>PNMA-MS1</em> is marsupial specific (Fig. <a reveal-id="DST020F4" data-open="DST020F4" class="link link-reveal link-table xref-fig">4</a>A). It indicates that <em>PNMA-MS1</em> retrotransposition occurred only in the marsupial lineage after their divergence from eutherians (Fig. <a reveal-id="DST020F5" data-open="DST020F5" class="link link-reveal link-table xref-fig">5</a>). </p>                        <a id="5174535" scrollto-destination="5174535"></a>
<div id="DST020F4" data-id="DST020F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/m_dst02004.jpeg?Expires=2147483647&Signature=ypRRUlbfDhakt6CrK1M7dmrLgZxsx1tntmHqj2QEIbsut4n4mP4uy2CTK4TnXg8th~Kfp0~UYjHcml4vTpOAuegWA26yHYVymXB-otYzF6wIgH3Cw6BtYGsUr5YvkrVBgWVWRmfALz19fcNK3bLzZOgzADTA0FVlHpgBfOIJ2t0TL0ALXEDA9m7dkKLYig4l0vppm5BjaGTr55nb6~9oh5ldvxsjDIXIjgrvpufMc5umJ4J6v12CwfkqBrfrwpPcFTLhAALdoktIxPzvEtaY5ObKe91pLcDInmLeMys61FM9T2SCUARCRarKD8quCOpGXkeG1EQF-iU9DlL8ap5RzA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparative genomic analysis of the PNMA-MS1 and -MS2 regions in vertebrates. (A) PNMA-MS1. mLAGAN alignment of the tammar wallaby, opossum, human, mouse, platypus, chicken, frog and fugu LAMA3 exons 8–10 region produced by mVISTA using the tammar sequence as the basis for comparison. Default parameters for mVISTA were used (conservation level, 70%, 100 bp window). Conserved regions appear as peaks highlighted in pink (&gt;70% identity). Where these regions coincide with ORF sequences of PNMA-MS1 or LAMA3, the peaks are shaded in purple. Where these regions coincide with the UTR region of PNMA-MS1, the peaks are shaded in light blue. The tammar genome sequence has a gap of about 200 bp that located at the 5′-side of exon 9 (shown by a green bar). (B) The region from PTGS1 to PDCL in the opossum, human, mouse, platypus, chicken, frog and fugu genomes is shown (upper). PNMA-MS2 is located between the OR1Q1 gene and an OR1J2-like pseudgene (ENSMODG00000019710) in the opossum (red bar), but is absent in the Tasmanian devil, human and mouse genomes (red bars indicate the equivalent region in these species. An mVISTA alignment of the OR1Q1-ENSMODG00000019710 region in the opossum (lower) confirms that PNMA-MS2 is not present in the Tasmanian devil, human or mouse." path-from-xml="dst02004.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5174535/dst02004.jpeg" path-from-xml="dst02004.jpeg" target="_blank">View large</a><a section="5174535" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/dst02004.jpeg?Expires=2147483647&Signature=CCQOGzmaAFVW2KNqwLstdfgS29urS4sulgSx2HmQfnagGzcAnct3y6DBQRRPcCBETIgNqZyAbDzg2osD3rhIQEBqtJMo2IETV8T-mjSrIJpjYIrhKGsuhBXaLt6mvIz1FoObIn7jR-~nFVM7cOUyiP1mbVmonE6A41dwSF6WHb0auu14mzpyD4yuVRizrN89dA-fThmblHHhOHFVaWNp-gTWNdBkfQA60zC7Zn5J17EU17Zd0EzmMuq5~0SxPsji8sF-gYg0-aKKTKxO0dMCoEmjET6HDdO9xWpQwEab8mJG87miBC3PmF2hdruhhsvwow~WLkhg2fkPrNp4cLhh4g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5174535&ar=359087&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02004.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparative genomic analysis of the <em>PNMA-MS1</em> and <em>-MS2</em> regions in vertebrates. (A) <em>PNMA-MS1.</em> mLAGAN alignment of the tammar wallaby, opossum, human, mouse, platypus, chicken, frog and fugu <em>LAMA3</em> exons 8–10 region produced by mVISTA using the tammar sequence as the basis for comparison. Default parameters for mVISTA were used (conservation level, 70%, 100 bp window). Conserved regions appear as peaks highlighted in pink (&gt;70% identity). Where these regions coincide with ORF sequences of <em>PNMA-MS1</em> or <em>LAMA3</em>, the peaks are shaded in purple. Where these regions coincide with the UTR region of <em>PNMA-MS1</em>, the peaks are shaded in light blue. The tammar genome sequence has a gap of about 200 bp that located at the 5′-side of exon 9 (shown by a green bar). (B) The region from <em>PTGS1</em> to <em>PDCL</em> in the opossum, human, mouse, platypus, chicken, frog and fugu genomes is shown (upper). <em>PNMA-MS2</em> is located between the <em>OR1Q1</em> gene and an <em>OR1J2</em>-like pseudgene (<em>ENSMODG00000019710</em>) in the opossum (red bar), but is absent in the Tasmanian devil, human and mouse genomes (red bars indicate the equivalent region in these species. An mVISTA alignment of the <em>OR1Q1-ENSMODG00000019710</em> region in the opossum (lower) confirms that <em>PNMA-MS2</em> is not present in the Tasmanian devil, human or mouse.</p></div></div><div content-id="DST020F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/m_dst02004.jpeg?Expires=2147483647&Signature=ypRRUlbfDhakt6CrK1M7dmrLgZxsx1tntmHqj2QEIbsut4n4mP4uy2CTK4TnXg8th~Kfp0~UYjHcml4vTpOAuegWA26yHYVymXB-otYzF6wIgH3Cw6BtYGsUr5YvkrVBgWVWRmfALz19fcNK3bLzZOgzADTA0FVlHpgBfOIJ2t0TL0ALXEDA9m7dkKLYig4l0vppm5BjaGTr55nb6~9oh5ldvxsjDIXIjgrvpufMc5umJ4J6v12CwfkqBrfrwpPcFTLhAALdoktIxPzvEtaY5ObKe91pLcDInmLeMys61FM9T2SCUARCRarKD8quCOpGXkeG1EQF-iU9DlL8ap5RzA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparative genomic analysis of the PNMA-MS1 and -MS2 regions in vertebrates. (A) PNMA-MS1. mLAGAN alignment of the tammar wallaby, opossum, human, mouse, platypus, chicken, frog and fugu LAMA3 exons 8–10 region produced by mVISTA using the tammar sequence as the basis for comparison. Default parameters for mVISTA were used (conservation level, 70%, 100 bp window). Conserved regions appear as peaks highlighted in pink (&gt;70% identity). Where these regions coincide with ORF sequences of PNMA-MS1 or LAMA3, the peaks are shaded in purple. Where these regions coincide with the UTR region of PNMA-MS1, the peaks are shaded in light blue. The tammar genome sequence has a gap of about 200 bp that located at the 5′-side of exon 9 (shown by a green bar). (B) The region from PTGS1 to PDCL in the opossum, human, mouse, platypus, chicken, frog and fugu genomes is shown (upper). PNMA-MS2 is located between the OR1Q1 gene and an OR1J2-like pseudgene (ENSMODG00000019710) in the opossum (red bar), but is absent in the Tasmanian devil, human and mouse genomes (red bars indicate the equivalent region in these species. An mVISTA alignment of the OR1Q1-ENSMODG00000019710 region in the opossum (lower) confirms that PNMA-MS2 is not present in the Tasmanian devil, human or mouse." path-from-xml="dst02004.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5174535/dst02004.jpeg" path-from-xml="dst02004.jpeg" target="_blank">View large</a><a section="5174535" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/dst02004.jpeg?Expires=2147483647&Signature=CCQOGzmaAFVW2KNqwLstdfgS29urS4sulgSx2HmQfnagGzcAnct3y6DBQRRPcCBETIgNqZyAbDzg2osD3rhIQEBqtJMo2IETV8T-mjSrIJpjYIrhKGsuhBXaLt6mvIz1FoObIn7jR-~nFVM7cOUyiP1mbVmonE6A41dwSF6WHb0auu14mzpyD4yuVRizrN89dA-fThmblHHhOHFVaWNp-gTWNdBkfQA60zC7Zn5J17EU17Zd0EzmMuq5~0SxPsji8sF-gYg0-aKKTKxO0dMCoEmjET6HDdO9xWpQwEab8mJG87miBC3PmF2hdruhhsvwow~WLkhg2fkPrNp4cLhh4g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5174535&ar=359087&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02004.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparative genomic analysis of the <em>PNMA-MS1</em> and <em>-MS2</em> regions in vertebrates. (A) <em>PNMA-MS1.</em> mLAGAN alignment of the tammar wallaby, opossum, human, mouse, platypus, chicken, frog and fugu <em>LAMA3</em> exons 8–10 region produced by mVISTA using the tammar sequence as the basis for comparison. Default parameters for mVISTA were used (conservation level, 70%, 100 bp window). Conserved regions appear as peaks highlighted in pink (&gt;70% identity). Where these regions coincide with ORF sequences of <em>PNMA-MS1</em> or <em>LAMA3</em>, the peaks are shaded in purple. Where these regions coincide with the UTR region of <em>PNMA-MS1</em>, the peaks are shaded in light blue. The tammar genome sequence has a gap of about 200 bp that located at the 5′-side of exon 9 (shown by a green bar). (B) The region from <em>PTGS1</em> to <em>PDCL</em> in the opossum, human, mouse, platypus, chicken, frog and fugu genomes is shown (upper). <em>PNMA-MS2</em> is located between the <em>OR1Q1</em> gene and an <em>OR1J2</em>-like pseudgene (<em>ENSMODG00000019710</em>) in the opossum (red bar), but is absent in the Tasmanian devil, human and mouse genomes (red bars indicate the equivalent region in these species. An mVISTA alignment of the <em>OR1Q1-ENSMODG00000019710</em> region in the opossum (lower) confirms that <em>PNMA-MS2</em> is not present in the Tasmanian devil, human or mouse.</p></div></div>                        <a id="5174536" scrollto-destination="5174536"></a>
<div id="DST020F5" data-id="DST020F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/m_dst02005.jpeg?Expires=2147483647&Signature=GCfIa~wlLs3pajpS~2fUD~-cgjp5BSBdpvI6xEOxw5X8iWIZUKzKZ-mlnIXx-tUlT7osV0jG92spwg5j1hKCivFSNTw-17Tm2IitN7loAVdBj8M4L55LS7KVJVFGtIj8zbzIqqVascCyw6QDOGy4RpnDnDL~YEJig9sqaqs2OPVvcMuMbN2miyzZrW2kageaw63Okr15Zeo463C0yQuUrwNMqe-vMua-anDuyj0-9QOO2vVw3pN13KBVqASWcmS4HE70dqIDTTfCeM6Q4WJP5WuQh8j42g2oNdYEPFT-TS0tsw9wJC7HzD6El4dEormBZAJl4xuHf8V4ka0H9oWgTQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evolutionary pathway of the PNMA family in mammals. PNMA-MS1 insertion occurred in a marsupial ancestor prior to the radiation of marsupial species. PNMA-MS2 was only found in the opossum and not in the Tasmanian devil, human and mouse, which suggests that PNMA-MS2 was acquired only in the opossum lineage after the divergence of the Australian and South American marsupials, but it does not show active transcription. The insertion of eutherian PNMA family genes occurred in a eutherian ancestor prior to the radiation of eutherian species. There are no PNMA family genes (or pseudogenes) in the platypus (monotremes) and chicken (birds)." path-from-xml="dst02005.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5174536/dst02005.jpeg" path-from-xml="dst02005.jpeg" target="_blank">View large</a><a section="5174536" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/dst02005.jpeg?Expires=2147483647&Signature=sjijMeZULrPwGtrPjDTYCE2RVrqwTxl0kqkbpxzNC7AIjbaqhUoUMrmkK63RZi8yvDiYfyDHahMTFY2r87IvSh8MQIjzVaE88wICZtaZhmVq6kYKIII8RHrDk9QaGjpOj1-JcEFBSKqIJhVXIOXHXAYjlxmix7V5nvKyArbUGa927K8ZhYmi6gbl65vuwHCsW~0J~c8x4WQ-p4hdP6UNZbqL3sG90-XUaVQBhInbiO68lc8Zm9BYdOaFrbolA7B~7KyBTO0icZHKjYh51FEiWYY3Xb6HDJ9oPkKQ4Qz6Oz7d5lJgfQLJ6F-783XOKhKd1baquBeoItgJSjrkrklQ5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5174536&ar=359087&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02005.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evolutionary pathway of the <em>PNMA</em> family in mammals. <em>PNMA-MS1</em> insertion occurred in a marsupial ancestor prior to the radiation of marsupial species. <em>PNMA-MS2</em> was only found in the opossum and not in the Tasmanian devil, human and mouse, which suggests that <em>PNMA-MS2</em> was acquired only in the opossum lineage after the divergence of the Australian and South American marsupials, but it does not show active transcription. The insertion of eutherian <em>PNMA</em> family genes occurred in a eutherian ancestor prior to the radiation of eutherian species. There are no <em>PNMA</em> family genes (or pseudogenes) in the platypus (monotremes) and chicken (birds).</p></div></div><div content-id="DST020F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/m_dst02005.jpeg?Expires=2147483647&Signature=GCfIa~wlLs3pajpS~2fUD~-cgjp5BSBdpvI6xEOxw5X8iWIZUKzKZ-mlnIXx-tUlT7osV0jG92spwg5j1hKCivFSNTw-17Tm2IitN7loAVdBj8M4L55LS7KVJVFGtIj8zbzIqqVascCyw6QDOGy4RpnDnDL~YEJig9sqaqs2OPVvcMuMbN2miyzZrW2kageaw63Okr15Zeo463C0yQuUrwNMqe-vMua-anDuyj0-9QOO2vVw3pN13KBVqASWcmS4HE70dqIDTTfCeM6Q4WJP5WuQh8j42g2oNdYEPFT-TS0tsw9wJC7HzD6El4dEormBZAJl4xuHf8V4ka0H9oWgTQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evolutionary pathway of the PNMA family in mammals. PNMA-MS1 insertion occurred in a marsupial ancestor prior to the radiation of marsupial species. PNMA-MS2 was only found in the opossum and not in the Tasmanian devil, human and mouse, which suggests that PNMA-MS2 was acquired only in the opossum lineage after the divergence of the Australian and South American marsupials, but it does not show active transcription. The insertion of eutherian PNMA family genes occurred in a eutherian ancestor prior to the radiation of eutherian species. There are no PNMA family genes (or pseudogenes) in the platypus (monotremes) and chicken (birds)." path-from-xml="dst02005.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5174536/dst02005.jpeg" path-from-xml="dst02005.jpeg" target="_blank">View large</a><a section="5174536" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/dst02005.jpeg?Expires=2147483647&Signature=sjijMeZULrPwGtrPjDTYCE2RVrqwTxl0kqkbpxzNC7AIjbaqhUoUMrmkK63RZi8yvDiYfyDHahMTFY2r87IvSh8MQIjzVaE88wICZtaZhmVq6kYKIII8RHrDk9QaGjpOj1-JcEFBSKqIJhVXIOXHXAYjlxmix7V5nvKyArbUGa927K8ZhYmi6gbl65vuwHCsW~0J~c8x4WQ-p4hdP6UNZbqL3sG90-XUaVQBhInbiO68lc8Zm9BYdOaFrbolA7B~7KyBTO0icZHKjYh51FEiWYY3Xb6HDJ9oPkKQ4Qz6Oz7d5lJgfQLJ6F-783XOKhKd1baquBeoItgJSjrkrklQ5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5174536&ar=359087&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02005.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evolutionary pathway of the <em>PNMA</em> family in mammals. <em>PNMA-MS1</em> insertion occurred in a marsupial ancestor prior to the radiation of marsupial species. <em>PNMA-MS2</em> was only found in the opossum and not in the Tasmanian devil, human and mouse, which suggests that <em>PNMA-MS2</em> was acquired only in the opossum lineage after the divergence of the Australian and South American marsupials, but it does not show active transcription. The insertion of eutherian <em>PNMA</em> family genes occurred in a eutherian ancestor prior to the radiation of eutherian species. There are no <em>PNMA</em> family genes (or pseudogenes) in the platypus (monotremes) and chicken (birds).</p></div></div><p><em>PNMA-MS2</em> was located between an <em>OR1Q1</em> gene and an <em>OR1J2</em>-like pseudogene (<em>ENSMODG00000019710</em>) that lies 12-kb upstream of the former, in an OR gene cluster located between prostaglandin-endoperoxide synthase 1 (<em>PTGS1</em>) and phosducin-like (<em>PDCL</em>) genes<sup><a reveal-id="DST020C19" data-open="DST020C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DST020C20" data-open="DST020C20" class="link link-ref link-reveal xref-bibr">20</a></sup> on opossum Chromosome 1. At present, we cannot confirm the presence or absence of <em>PNMA-MS2</em> in the tammar because the corresponding regions encompassing the <em>PTGS1</em> and <em>PDCL</em> genes are yet to be completely assembled. Recently, another Australian marsupial genome of the Tasmanian devil has been sequenced, and the region syntenic to that between opossum <em>OR1N2</em> and <em>PDCL</em> became available. It contains some gaps but none between opossum <em>OR1Q1</em> and <em>ENSMODG00000019710</em> (<em>OR1J2</em>-like pseudogene). A search of this region clearly demonstrated that the <em>PNMA-MS2</em> orthologue is absent from the Tasmanian devil genome.</p><p>The syntenic OR cluster lies between <em>PTGS1</em> and <em>PDCL</em> in the human Chromosome 9 and mouse Chromosome 2, respectively, but neither the number nor the order of OR genes and pseudogenes are conserved. As the human and mouse genome sequences in this region are complete and contain no gaps, the absence of the <em>PNMA-MS2</em> orthologue was confirmed (Fig. <a reveal-id="DST020F4" data-open="DST020F4" class="link link-reveal link-table xref-fig">4</a>B). For the platypus, the <em>PTGS1</em> and <em>PDCL</em> genes are located next to each other, with no OR gene cluster and no <em>PNMA-MS2</em> orthologue, like in the chicken and fugu (Fig. <a reveal-id="DST020F4" data-open="DST020F4" class="link link-reveal link-table xref-fig">4</a>B).</p><p>These results suggest that the integration of selected OR genes occurred between the <em>PTGS1</em> and <em>PDCL</em> genes in a common therian ancestor, and that the opossum-specific insertion of <em>PNMA-MS2</em> occurred after the divergence of eutherians and marsupials and the geographic separation of Australian and South American marsupials (Fig. <a reveal-id="DST020F5" data-open="DST020F5" class="link link-reveal link-table xref-fig">5</a>). However, the possibility that the <em>PNMA-MS2</em> orthologue exists in some of Australian marsupial species cannot be excluded. It is possible that <em>PNMA</em>-<em>MS2</em> was deleted from Australian marsupial species after integration in a common marsupial ancestor (Fig. <a reveal-id="DST020F5" data-open="DST020F5" class="link link-reveal link-table xref-fig">5</a>).</p>                            <h3 scrollto-destination=5174540 id="5174540" class="section-title" data-legacy-id=s3e>3.5. Expressions of PNMA-MS1 in the tammar wallaby and PNMA-MS2 in the opossum</h3>
<p><em>PNMA</em>-<em>MS1</em> expression was investigated in several tissues in four different stages of the tammar wallaby, including foetal and pouch young stages. Human <em>LAMA3</em> is expressed ubiquitously (EST profile Hs.436367). To exclude the possibility that heterogenous nuclear RNA (hnRNA) was detected between exons 8 and 9 of the <em>LAMA3</em> gene rather than <em>PNMA-MS1</em>, we amplified <em>PNMA-MS1</em> using PCR primers designed within exons 1 and 2, respectively. Thus, the PCR product was shorter than its genomic sequence corresponding to hnRNAs of <em>LAMA3</em> and <em>PNMA-MS1. LAMA3</em> expression was analysed using primers designed to exons 79 and 81, near the 3′-UTR, due to the poor genome sequence quality in introns 84–90 of tammar <em>LAMA3.</em> Tammar <em>LAMA3</em> expression was almost ubiquitous, with the exception of several pouch young tissues.</p><p>From Day 23 to 26 pregnancy, <em>PNMA-MS1</em> expression was detected in the foetal head and body, but there was no expression in the YSP (Fig. <a reveal-id="DST020F6" data-open="DST020F6" class="link link-reveal link-table xref-fig">6</a>A). In pouch young aged Day 60–70, <em>PNMA-MS1</em> was detected only in the brain, kidney and ovary, but not in the liver, lung or testis (Fig. <a reveal-id="DST020F6" data-open="DST020F6" class="link link-reveal link-table xref-fig">6</a>B). In Day 152 and 162 pouch young, changes in <em>PNMA-MS1</em> expression were minimal, with expression detected in the kidney, liver, pancreas, heart, spleen and stomach, but not in the lung, bladder, adrenal, cerebrum or cerebellum (Fig. <a reveal-id="DST020F6" data-open="DST020F6" class="link link-reveal link-table xref-fig">6</a>C). In the adult female, the brain (thalamus, hypothalamus and pituitary), ovary (ovary with active corpus luteum, corpus luteum alone, ovary with enlarged developing follicle and ovary with primary or secondary follicles), endometrium (gravid endometrium and non-gravid endometrium) and mammary gland (sucked gland and non-sucked gland) were examined. <em>PNMA-MS1</em> expression was detected in the ovary (all four stages), mammary gland (sucked and non-sucked) and thalamus, but not in the hypothalamus or pituitary (Fig. <a reveal-id="DST020F6" data-open="DST020F6" class="link link-reveal link-table xref-fig">6</a>D). There was no expression in either the gravid, or non-gravid, endometrium. </p>                        <a id="5174543" scrollto-destination="5174543"></a>
<div id="DST020F6" data-id="DST020F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/m_dst02006.jpeg?Expires=2147483647&Signature=hRcmyiMzHtUjurUwBJ5c82b1FCfz6ha~qO9nKB8B7NIzzKoVqm8NuL7vZZ3A8Sz2lZqPQ6-RvpSLHc2eOOsvgrGhiLAxJcD8BJHZVVmpENfDax6Kb1guHLwEPiuYbpHj~KJeo1GPtuvnu6OGrekzpq4LT03jt-FD-bSePx9DywHOyLqO5OCjIQ4k~XvMKOHJuaer1bArErgb2QE-8IHWYjTPIsuYshPaku01Ma1elF-Fh-9ZeoWGxP5w57yxbmDy9u1gp1UW6Kp7WxzK8GRf~69SnHqx24HBeU1lhq2477KwK6DmqZj2nVCisoTzEpodmNNDsNaGMZ~gd9UY-6Gemg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression profiles of PNMA-MS1 in the tammar wallaby. Expression analysis of tammar wallaby PNMA-MS1 in (A) foetal tissues and YSP at Day 23–26 of pregnancy, in several tissues, from (B) Day 60–70 and (C) Day 152–162 pouch young and (D) the adult female brain, ovary, endometrium and mammary gland. Expression of GAPDH and LAMA3 for each sample is shown as a control. hypothal: hypothalamus; ov CL: ovary with active corpus luteum; CL: corpus luteum; ov follicle: ovary with developing follicle; ov non CL: ovary with primary or secondary follicle; gr endo: gravid endometrium; ngr endo: non-gravid endometrium; ns mg: non-sucked mammary gland; s mg: sucked (lactating) mammary gland." path-from-xml="dst02006.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5174543/dst02006.jpeg" path-from-xml="dst02006.jpeg" target="_blank">View large</a><a section="5174543" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/dst02006.jpeg?Expires=2147483647&Signature=0a93QK1Gsie95WCdqM8x~b8ZgP2Thq1iVrntOzP3VVR3noICD2kUUZWp-7JARYe2qjdYX6sMHYkMrdboaI935vsK4OBmqKv-3azKq5SjTo41OnKY0d4~mSqCl60nomTAxwOLdJi835kXKbO2U7721n7aMdIcwbyb5uCSyPnPzXngqCyvCgiHaSM3o0FO-Sh6tORgqaOyCGMucQjN6DTUUE~4HG2eUWNBPPaarD8VB1doI7tDMXe7UVkKXV5k-qVGa6MfaR5vpdIjgqKR7q0g-q5igtqA3zi9ClkYYmBw0wDmsVpkVhJSYs6QKTcIANSD4kFnRpicK7UjaU~ed1ewAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5174543&ar=359087&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02006.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression profiles of <em>PNMA-MS1</em> in the tammar wallaby. Expression analysis of tammar wallaby <em>PNMA-MS1</em> in (A) foetal tissues and YSP at Day 23–26 of pregnancy, in several tissues, from (B) Day 60–70 and (C) Day 152–162 pouch young and (D) the adult female brain, ovary, endometrium and mammary gland. Expression of <em>GAPDH</em> and <em>LAMA3</em> for each sample is shown as a control. hypothal: hypothalamus; ov CL: ovary with active corpus luteum; CL: corpus luteum; ov follicle: ovary with developing follicle; ov non CL: ovary with primary or secondary follicle; gr endo: gravid endometrium; ngr endo: non-gravid endometrium; ns mg: non-sucked mammary gland; s mg: sucked (lactating) mammary gland.</p></div></div><div content-id="DST020F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/m_dst02006.jpeg?Expires=2147483647&Signature=hRcmyiMzHtUjurUwBJ5c82b1FCfz6ha~qO9nKB8B7NIzzKoVqm8NuL7vZZ3A8Sz2lZqPQ6-RvpSLHc2eOOsvgrGhiLAxJcD8BJHZVVmpENfDax6Kb1guHLwEPiuYbpHj~KJeo1GPtuvnu6OGrekzpq4LT03jt-FD-bSePx9DywHOyLqO5OCjIQ4k~XvMKOHJuaer1bArErgb2QE-8IHWYjTPIsuYshPaku01Ma1elF-Fh-9ZeoWGxP5w57yxbmDy9u1gp1UW6Kp7WxzK8GRf~69SnHqx24HBeU1lhq2477KwK6DmqZj2nVCisoTzEpodmNNDsNaGMZ~gd9UY-6Gemg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression profiles of PNMA-MS1 in the tammar wallaby. Expression analysis of tammar wallaby PNMA-MS1 in (A) foetal tissues and YSP at Day 23–26 of pregnancy, in several tissues, from (B) Day 60–70 and (C) Day 152–162 pouch young and (D) the adult female brain, ovary, endometrium and mammary gland. Expression of GAPDH and LAMA3 for each sample is shown as a control. hypothal: hypothalamus; ov CL: ovary with active corpus luteum; CL: corpus luteum; ov follicle: ovary with developing follicle; ov non CL: ovary with primary or secondary follicle; gr endo: gravid endometrium; ngr endo: non-gravid endometrium; ns mg: non-sucked mammary gland; s mg: sucked (lactating) mammary gland." path-from-xml="dst02006.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5174543/dst02006.jpeg" path-from-xml="dst02006.jpeg" target="_blank">View large</a><a section="5174543" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/dst02006.jpeg?Expires=2147483647&Signature=0a93QK1Gsie95WCdqM8x~b8ZgP2Thq1iVrntOzP3VVR3noICD2kUUZWp-7JARYe2qjdYX6sMHYkMrdboaI935vsK4OBmqKv-3azKq5SjTo41OnKY0d4~mSqCl60nomTAxwOLdJi835kXKbO2U7721n7aMdIcwbyb5uCSyPnPzXngqCyvCgiHaSM3o0FO-Sh6tORgqaOyCGMucQjN6DTUUE~4HG2eUWNBPPaarD8VB1doI7tDMXe7UVkKXV5k-qVGa6MfaR5vpdIjgqKR7q0g-q5igtqA3zi9ClkYYmBw0wDmsVpkVhJSYs6QKTcIANSD4kFnRpicK7UjaU~ed1ewAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5174543&ar=359087&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst02006.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression profiles of <em>PNMA-MS1</em> in the tammar wallaby. Expression analysis of tammar wallaby <em>PNMA-MS1</em> in (A) foetal tissues and YSP at Day 23–26 of pregnancy, in several tissues, from (B) Day 60–70 and (C) Day 152–162 pouch young and (D) the adult female brain, ovary, endometrium and mammary gland. Expression of <em>GAPDH</em> and <em>LAMA3</em> for each sample is shown as a control. hypothal: hypothalamus; ov CL: ovary with active corpus luteum; CL: corpus luteum; ov follicle: ovary with developing follicle; ov non CL: ovary with primary or secondary follicle; gr endo: gravid endometrium; ngr endo: non-gravid endometrium; ns mg: non-sucked mammary gland; s mg: sucked (lactating) mammary gland.</p></div></div><p><em>PNMA-MS2</em> expression was analysed in five tissues, the brain, liver, spleen, pituitary and ovary, using three primer sets designed in the putative coding frame. However, there was no expression in these tissues, suggesting that <em>PNMA-MS2</em> is not active in the opossum (data not shown). Although we cannot exclude the possibility that it may be expressed in a stage or tissue-specific manner, we conclude that <em>PNMA-MS2</em> is a pseudogene.</p>                            <h2 scrollto-destination=5174545 id="5174545" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=5174546 id="5174546" class="section-title" data-legacy-id=s4a>4.1. <em>PNMA-MS1</em> is a marsupial-specific <em>PNMA</em> gene</h3>
<p>In this study, we have identified <em>PNMA-MS1</em> as a novel Ty3/Gypsy LTR retrotransposon-derived gene. Comparative genomic analysis showed that <em>PNMA-MS1</em> was present only in the marsupial lineage and was absent in the eutherian and monotreme mammals and in the non-mammalian vertebrates. <em>PEG10/SIRH1</em> and <em>SIRH12</em> are the only Ty3/Gypsy LTR retrotransposon-derived genes (derived from sushi-ichi-related retrotranposons) reported in marsupials so far.<sup><a reveal-id="DST020C11" data-open="DST020C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DST020C21" data-open="DST020C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Therefore, the <em>PNMA-MS1</em>, which is also a Ty3/gypsy LTR retrotransposon-derived gene, is the first and only member of the <em>PNMA</em> gene family in the marsupials.</p><p>The sushi-ichi-related retrotranposon that gave rise to the <em>SIRH</em> family was probably active around the time of the divergence between marsupials and eutherians, because <em>PEG10/SIRH1</em> is conserved between the eutherians and marsupials,<sup><a reveal-id="DST020C21" data-open="DST020C21" class="link link-ref link-reveal xref-bibr">21</a></sup> while <em>SIRH3–11</em> seems to be eutherian-specific and <em>SIRH12</em> evolved from a marsupial-specific retrotransposition event.<sup><a reveal-id="DST020C11" data-open="DST020C11" class="link link-ref link-reveal xref-bibr">11</a></sup> We did not detect any orthologues of <em>PNMA-MS1</em> or <em>-MS2</em> in the eutherian genome, nor any orthologues of eutherian <em>PNMA1–16</em> genes in the marsupial genomes (data not shown). Due to many sequence gaps in the tammar wallaby, Tasmanian devil and opossum genomes, we cannot exclude the possibility that some marsupial orthologues of eutherian <em>PNMA</em> genes exist in such gap regions. Thus, it is possible that some retrotransposition events of Gypsy12_DR-related retrotranposon occurred in the common ancestor of the marsupials and eutherians. However, the higher similarity of <em>PNMA-MS1</em> and <em>-MS2</em> to Gypsy12_DR Gag than any other eutherian <em>PNMAs</em> suggests that their insertions in the marsupial genome were recent events. Taken together, these results suggest that the retrotransposition of Gypsy12_DR-related retrotransposon occurred after the divergence of the marsupials and eutherians. The <em>PNMA</em> genes then evolved independently in these two lineages (Fig. <a reveal-id="DST020F5" data-open="DST020F5" class="link link-reveal link-table xref-fig">5</a>).</p><p>In our analysis, only <em>PNMA-MS1</em> was detected in the tammar and opossum in contrast to 19 and 14 <em>PNMA</em> genes in humans and mice, respectively. We also observed the same trend in the <em>SIRH</em> genes: 11 genes in both humans and mice, while there are only 2 genes (<em>PEG10</em> and <em>SIRH12</em>) in the tammar genome.<sup><a reveal-id="DST020C11" data-open="DST020C11" class="link link-ref link-reveal xref-bibr">11</a></sup> This implies that the eutherian genome has a greater ability of exaptation as more Ty3/Gypsy types of LTR retrotransposons were incorporated into the genomes as endogenous genes than in the marsupial genomes.</p>                            <h3 scrollto-destination=5174550 id="5174550" class="section-title" data-legacy-id=s4b>4.2. The possible role of <em>PNMA-MS1</em> genes in marsupial development</h3>
<p>In rare cases, some retrotransposons have been incorporated as novel acquired genes into the host genomes and have contributed to the innovation of some eutherian-specific characteristics. Two such advantageous genes, <em>PEG10/SIRH1</em> and <em>PEG11/SIRH2</em>, play essential roles in the placental development in mice.<sup><a reveal-id="DST020C7" data-open="DST020C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DST020C8" data-open="DST020C8" class="link link-ref link-reveal xref-bibr">8</a></sup></p><p>The role of <em>PNMA-MS1</em> in marsupial development and growth is less clear. <em>PNMA-MS1</em> expression was detected in the tammar brain, consistent with the expression of eutherian <em>PNMA</em> genes in brain. Interestingly, <em>PNMA-MS1</em> expression was confirmed only in the thalamus, but not in the hypothalamus, and pituitary in the adult brain. The thalamus has multiple functions including relaying sensation, spatial sense and motor signals to the cerebral cortex,<sup><a reveal-id="DST020C22" data-open="DST020C22" class="link link-ref link-reveal xref-bibr">22</a></sup> so it is possible that <em>PNMA-MS1</em> is involved in the transmission of marsupial-specific sensations and signals. <em>PNMA-MS1</em> expression in the Day 60–70 pouch young ovary and adult female ovary suggests the gene may have a role in ovarian function. These issues will be addressed in a future study.</p><p><em>PNMA-MS1</em> protein has a conserved CCHC zinc finger domain. In retroviruses, this domain forms a part of the nucleocapsid protein that functions in virus genome packaging and the early infection process.<sup><a reveal-id="DST020C23" data-open="DST020C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Proteins containing the CCHC zinc finger domain are commonly known to interact with single-stranded DNAs (ssDNAs) and RNAs.<sup><a reveal-id="DST020C24" data-open="DST020C24" class="link link-ref link-reveal xref-bibr">24</a></sup> The <em>Drosophila</em> Nanos protein is required for <em>hunchback</em> mRNA translational regulation in the early embryo to the establishment of the anterior–posterior body axis.<sup><a reveal-id="DST020C25" data-open="DST020C25" class="link link-ref link-reveal xref-bibr">25</a></sup> The mammalian cellular nucleic acid-binding protein (CNBP) containing seven CCHC domains is involved in neural crest development, affecting forebrain and craniofacial development.<sup><a reveal-id="DST020C26" data-open="DST020C26" class="link link-ref link-reveal xref-bibr">26</a></sup> CNBP has a single-stranded nucleic acid-binding ability and is also implicated in both transcriptional and translational regulations.<sup><a reveal-id="DST020C27" data-open="DST020C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DST020C28" data-open="DST020C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Therefore, <em>PNMA-MS1</em> may also be involved in a specific-transcriptional or translational regulation by binding ssDNAs or RNAs.</p>                            <h2 scrollto-destination=5174554 id="5174554" class="section-title" data-legacy-id=s5>5. Conclusions</h2>
<p>We have identified one novel Ty3/Gypsy LTR retrotransposon-derived gene, <em>PNMA-MS1</em> in marsupials as the first marsupial-specific <em>PNMA</em> gene reported. The high <em>PNMA-MS1</em> expression levels in the thalamus, ovary and mammary gland provide intriguing questions as to its functions in marsupial development and growth as well as its role in marsupial evolution. Our data suggest that, in most of the cases, Ty3/Gypsy LTR retrotransposons have been independently incorporated into the marsupial and eutherian lineages. Consequently, the marsupials and eutherians have completely different sets of <em>PNMA</em> and <em>SIRH</em> genes, with the exception of <em>PEG10.</em> Thus, it is highly likely that these genes have evolved lineage-specific functions in the reproduction and development and contributed in establishing marsupial- or eutherian-specific traits, leading to the diversification of these two viviparous mammalian groups.</p>                            <h2 scrollto-destination=5174556 id="5174556" class="section-title" data-legacy-id=s6>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at</a><a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5174558 id="5174558" class="section-title" data-legacy-id=s7>Accession Numbers</h2>
<p>The National Center for Biotechnology Information GenBank (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/Genbank">http://www.ncbi.nlm.nih.gov/Genbank</a>) sequence accession number for tammar PNMA-MS1 is AB646689.</p>                            <h2 scrollto-destination=5174560 id="5174560" class="section-title" data-legacy-id=s8>Funding</h2>
<p>This work was supported by grants from the Australian Research Council Centre of Excellence in Kangaroo Genomics to M.B.R., Japan Society for the Promotion of Science (JSPS), Bilateral Program on Joint Research Project to F.I. and T.K-I. and Funding Program for Next Generation World-Leading Researchers (NEXT Program) to T.K-I., the Ministry of Education, Culture, Sports, Science, and Technology (MEXT) of Japan, Grants-in-Aid for Scientific Research (S) and the Global Center for Excellence (GCOE) Program for International Research Center for Molecular Science in Tooth and Bone Diseases and Medical Research Institute, Tokyo Medical and Dental University, Joint Usage/Research Program to F.I.</p>                            <h2 scrollto-destination=5174562 id="5174562" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Scott Brownlees, Bonnie Dopheide, Hongshi Yu, Jessica Stringer, Stephen Frankenberg and the Wallaby research group for the assistance with the animals and tissue collections, and Soichi Ogishima for his advice on phylogenetic analysis.</p>                            <h2 scrollto-destination=5174564 id="5174564" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST020C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lander</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Linton</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Birren</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Initial sequencing and analysis of the human genome</div>, <div class="source">Nature</div> , <div class="year">2001</div>, vol. <div class="volume">409</div> (pg. <div class="fpage">860</div>-<div class="lpage">921</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Initial%20sequencing%20and%20analysis%20of%20the%20human%20genome&amp;author=E.%20Lander&amp;author=L.%20Linton&amp;author=B.%20Birren&amp;publication_year=2001&amp;journal=Nature&amp;volume=409&amp;pages=860-921" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35057062" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35057062" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35057062"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11237011" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST020C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Waterston</div> <div class="given-names">R.H.</div></div>,  <div class="name"><div class="surname">Lindblad-Toh</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Birney</div> <div class="given-names">E.</div></div>, et al. </div><div class="collab">Mouse Genome Sequencing Consortium</div><div class="article-title">Initial sequencing and comparative analysis of the mouse genome</div>, <div class="source">Nature</div> , <div class="year">2002</div>, vol. <div class="volume">420</div> (pg. <div class="fpage">520</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Initial%20sequencing%20and%20comparative%20analysis%20of%20the%20mouse%20genome&amp;author=R.H.%20Waterston&amp;author=K.%20Lindblad-Toh&amp;author=E.%20Birney&amp;publication_year=2002&amp;journal=Nature&amp;volume=420&amp;pages=520-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01262" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01262" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01262"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12466850" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST020C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Warren</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Hillier</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Marshall Graves</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Genome analysis of the platypus reveals unique signatures of evolution</div>, <div class="source">Nature</div> , <div class="year">2008</div>, vol. <div class="volume">453</div> (pg. <div class="fpage">175</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20analysis%20of%20the%20platypus%20reveals%20unique%20signatures%20of%20evolution&amp;author=W.%20Warren&amp;author=L.%20Hillier&amp;author=J.%20Marshall%20Graves&amp;publication_year=2008&amp;journal=Nature&amp;volume=453&amp;pages=175-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06936" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06936" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06936"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18464734" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST020C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mikkelsen</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Wakefield</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Aken</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Genome of the marsupial <em>Monodelphis domestica</em> reveals innovation in non-coding sequences</div>, <div class="source">Nature</div> , <div class="year">2007</div>, vol. <div class="volume">447</div> (pg. <div class="fpage">167</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20of%20the%20marsupial%20Monodelphis%20domestica%20reveals%20innovation%20in%20non-coding%20sequences&amp;author=T.%20Mikkelsen&amp;author=M.%20Wakefield&amp;author=B.%20Aken&amp;publication_year=2007&amp;journal=Nature&amp;volume=447&amp;pages=167-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature05805" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature05805" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature05805"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17495919" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST020C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Renfree</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Papenfuss</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Deakin</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Genome sequence of an Australian kangaroo, <em>Macropus eugenii</em>, provides insight into the evolution of mammalian reproduction and development</div>, <div class="source">Genome Biol.</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">R81</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20an%20Australian%20kangaroo%2C%20Macropus%20eugenii%2C%20provides%20insight%20into%20the%20evolution%20of%20mammalian%20reproduction%20and%20development&amp;author=M.%20Renfree&amp;author=A.%20Papenfuss&amp;author=J.%20Deakin&amp;publication_year=2011&amp;journal=Genome%20Biol.&amp;volume=12&amp;pages=R81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2011-12-8-r81" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2011-12-8-r81" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2011-12-8-r81"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21854559" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST020C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miller</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Lynch</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Martin</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Herniou</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Tristem</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Identification of multiple Gypsy LTR-retrotransposon lineages in vertebrate genomes</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1999</div>, vol. <div class="volume">49</div> (pg. <div class="fpage">358</div>-<div class="lpage">66</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20multiple%20Gypsy%20LTR-retrotransposon%20lineages%20in%20vertebrate%20genomes&amp;author=K.%20Miller&amp;author=C.%20Lynch&amp;author=J.%20Martin&amp;author=E.%20Herniou&amp;author=M.%20Tristem&amp;publication_year=1999&amp;journal=J.%20Mol.%20Evol.&amp;volume=49&amp;pages=358-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00006559" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00006559" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00006559"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10473777" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST020C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ono</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Inoue</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Deletion of Peg10, an imprinted gene acquired from a retrotransposon, causes early embryonic lethality</div>, <div class="source">Nat. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">101</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deletion%20of%20Peg10%2C%20an%20imprinted%20gene%20acquired%20from%20a%20retrotransposon%2C%20causes%20early%20embryonic%20lethality&amp;author=R.%20Ono&amp;author=K.%20Nakamura&amp;author=K.%20Inoue&amp;publication_year=2006&amp;journal=Nat.%20Genet.&amp;volume=38&amp;pages=101-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1699" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1699" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1699"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16341224" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST020C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sekita</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Wagatsuma</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Role of retrotransposon-derived imprinted gene, Rtl1, in the feto-maternal interface of mouse placenta</div>, <div class="source">Nat. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">243</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20retrotransposon-derived%20imprinted%20gene%2C%20Rtl1%2C%20in%20the%20feto-maternal%20interface%20of%20mouse%20placenta&amp;author=Y.%20Sekita&amp;author=H.%20Wagatsuma&amp;author=K.%20Nakamura&amp;publication_year=2008&amp;journal=Nat.%20Genet.&amp;volume=40&amp;pages=243-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2007.51" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2007.51" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2007.51"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18176565" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST020C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brandt</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Veith</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Volff</div> <div class="given-names">J.</div></div>. </div><div class="article-title">A family of neofunctionalized Ty3/gypsy retrotransposon genes in mammalian genomes</div>, <div class="source">Cytogenet. Genome Res.</div> , <div class="year">2005</div>, vol. <div class="volume">110</div> (pg. <div class="fpage">307</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20family%20of%20neofunctionalized%20Ty3%2Fgypsy%20retrotransposon%20genes%20in%20mammalian%20genomes&amp;author=J.%20Brandt&amp;author=A.%20Veith&amp;author=J.%20Volff&amp;publication_year=2005&amp;journal=Cytogenet.%20Genome%20Res.&amp;volume=110&amp;pages=307-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000084963" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000084963" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000084963"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16093683" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST020C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Youngson</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Kocialkowski</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Peel</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Ferguson-Smith</div> <div class="given-names">A.</div></div>. </div><div class="article-title">A small family of sushi-class retrotransposon-derived genes in mammals and their relation to genomic imprinting</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">2005</div>, vol. <div class="volume">61</div> (pg. <div class="fpage">481</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20small%20family%20of%20sushi-class%20retrotransposon-derived%20genes%20in%20mammals%20and%20their%20relation%20to%20genomic%20imprinting&amp;author=N.%20Youngson&amp;author=S.%20Kocialkowski&amp;author=N.%20Peel&amp;author=A.%20Ferguson-Smith&amp;publication_year=2005&amp;journal=J.%20Mol.%20Evol.&amp;volume=61&amp;pages=481-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-004-0332-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-004-0332-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-004-0332-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16155747" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST020C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ono</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Kuroki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Naruse</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Identification of tammar wallaby SIRH12, derived from a marsupial-specific retrotransposition event</div>, <div class="source">DNA Res.</div> , <div class="year">2011</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">211</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20tammar%20wallaby%20SIRH12%2C%20derived%20from%20a%20marsupial-specific%20retrotransposition%20event&amp;author=R.%20Ono&amp;author=Y.%20Kuroki&amp;author=M.%20Naruse&amp;publication_year=2011&amp;journal=DNA%20Res.&amp;volume=18&amp;pages=211-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsr012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsr012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsr012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21636603" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST020C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kaneko-Ishino</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ishino</div> <div class="given-names">F.</div></div>. </div><div class="article-title">The role of genes domesticated from LTR retrotransposons and retroviruses in mammals</div>, <div class="source">Front Microbiol.</div> , <div class="year">2012</div>, vol. <div class="volume">3</div> pg. <div class="fpage">262</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20genes%20domesticated%20from%20LTR%20retrotransposons%20and%20retroviruses%20in%20mammals&amp;author=T.%20Kaneko-Ishino&amp;author=F.%20Ishino&amp;publication_year=2012&amp;journal=Front%20Microbiol.&amp;volume=3&amp;pages=262" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fmicb.2012.00262" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffmicb.2012.00262" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffmicb.2012.00262"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22866050" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST020C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Campillos</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Doerks</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Shah</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Bork</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Computational characterization of multiple Gag-like human proteins</div>, <div class="source">Trends Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">585</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20characterization%20of%20multiple%20Gag-like%20human%20proteins&amp;author=M.%20Campillos&amp;author=T.%20Doerks&amp;author=P.%20Shah&amp;author=P.%20Bork&amp;publication_year=2006&amp;journal=Trends%20Genet.&amp;volume=22&amp;pages=585-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2006.09.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2006.09.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2006.09.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16979784" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST020C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takaji</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Komatsu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Watakabe</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hashikawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yamamori</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Paraneoplastic antigen-like 5 gene (PNMA5) is preferentially expressed in the association areas in a primate specific manner</div>, <div class="source">Cereb. Cortex</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">2865</div>-<div class="lpage">79</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Paraneoplastic%20antigen-like%205%20gene%20(PNMA5)%20is%20preferentially%20expressed%20in%20the%20association%20areas%20in%20a%20primate%20specific%20manner&amp;author=M.%20Takaji&amp;author=Y.%20Komatsu&amp;author=A.%20Watakabe&amp;author=T.%20Hashikawa&amp;author=T.%20Yamamori&amp;publication_year=2009&amp;journal=Cereb.%20Cortex&amp;volume=19&amp;pages=2865-79" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/cercor/bhp062" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fcercor%2Fbhp062" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fcercor%2Fbhp062"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19366867" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST020C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rosenfeld</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Eichen</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wade</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Posner</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Dalmau</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Molecular and clinical diversity in paraneoplastic immunity to Ma proteins</div>, <div class="source">Ann. Neurol.</div> , <div class="year">2001</div>, vol. <div class="volume">50</div> (pg. <div class="fpage">339</div>-<div class="lpage">48</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20and%20clinical%20diversity%20in%20paraneoplastic%20immunity%20to%20Ma%20proteins&amp;author=M.%20Rosenfeld&amp;author=J.%20Eichen&amp;author=D.%20Wade&amp;author=J.%20Posner&amp;author=J.%20Dalmau&amp;publication_year=2001&amp;journal=Ann.%20Neurol.&amp;volume=50&amp;pages=339-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1531-8249" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1531-8249" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1531-8249"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11558790" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST020C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schüller</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Jenne</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Voltz</div> <div class="given-names">R.</div></div>. </div><div class="article-title">The human PNMA family: novel neuronal proteins implicated in paraneoplastic neurological disease</div>, <div class="source">J. Neuroimmunol.</div> , <div class="year">2005</div>, vol. <div class="volume">169</div> (pg. <div class="fpage">172</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20human%20PNMA%20family%3A%20novel%20neuronal%20proteins%20implicated%20in%20paraneoplastic%20neurological%20disease&amp;author=M.%20Sch%C3%BCller&amp;author=D.%20Jenne&amp;author=R.%20Voltz&amp;publication_year=2005&amp;journal=J.%20Neuroimmunol.&amp;volume=169&amp;pages=172-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jneuroim.2005.08.019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jneuroim.2005.08.019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jneuroim.2005.08.019"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16214224" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST020C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Poole</div> <div class="given-names">W.E.</div></div>,  <div class="name"><div class="surname">Simms</div> <div class="given-names">N.G.</div></div>,  <div class="name"><div class="surname">Wood</div> <div class="given-names">J.T.</div></div>,  <div class="name"><div class="surname">Lubulwa</div> <div class="given-names">M.</div></div>. </div>, <div class="source">Tables for Age Determination of the Kangaroo Island Wallaby (Tammar), <em>Macropus eugenii</em>, from Body Measurements</div> , <div class="year">1991</div><div class="publisher-loc">Canberra, Australia</div><div class="publisher-name">Division of Wildlife and Ecology, CSIRO</div> <div class="comment">Technical Note</div></div></div></div></div><div content-id="DST020C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="collab">NHMRC</div>, <div class="source">Australian Code of Practice for the Care and Use of Animals for Scientific Purposes</div> , <div class="year">2004</div><div class="edition">7th edition</div><div class="publisher-loc">Canberra</div><div class="publisher-name">Australian Government</div></div></div></div></div><div content-id="DST020C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Godfrey</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Malnic</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Buck</div> <div class="given-names">L.</div></div>. </div><div class="article-title">The mouse olfactory receptor gene family</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">2156</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20mouse%20olfactory%20receptor%20gene%20family&amp;author=P.%20Godfrey&amp;author=B.%20Malnic&amp;author=L.%20Buck&amp;publication_year=2004&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=2156-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0308051100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0308051100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0308051100"> </span></div> </div></div></div></div></div><div content-id="DST020C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Malnic</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Godfrey</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Buck</div> <div class="given-names">L.</div></div>. </div><div class="article-title">The human olfactory receptor gene family</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">2584</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20human%20olfactory%20receptor%20gene%20family&amp;author=B.%20Malnic&amp;author=P.%20Godfrey&amp;author=L.%20Buck&amp;publication_year=2004&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=2584-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0307882100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0307882100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0307882100"> </span></div> </div></div></div></div></div><div content-id="DST020C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ono</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Narita</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Retrotransposon silencing by DNA methylation can drive mammalian genomic imprinting</div>, <div class="source">PLoS Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">3</div> pg. <div class="fpage">e55</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Retrotransposon%20silencing%20by%20DNA%20methylation%20can%20drive%20mammalian%20genomic%20imprinting&amp;author=S.%20Suzuki&amp;author=R.%20Ono&amp;author=T.%20Narita&amp;publication_year=2007&amp;journal=PLoS%20Genet.&amp;volume=3&amp;pages=e55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.0030055" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.0030055" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.0030055"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17432937" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST020C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ashwell</div> <div class="given-names">K.</div></div>. </div>, <div class="year">2010</div><div class="publisher-name">Cambridge University Press</div>(pg. <div class="fpage">95</div>-<div class="lpage">6</div>) <div class="comment">The neurobiology of Australian marsupials: brain evolution in the other mammalian radiation</div></div></div></div></div><div content-id="DST020C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Narayanan</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Gorelick</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">DeStefano</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Structure/function mapping of amino acids in the N-terminal zinc finger of the human immunodeficiency virus type 1 nucleocapsid protein: residues responsible for nucleic acid helix destabilizing activity</div>, <div class="source">Biochemistry</div> , <div class="year">2006</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">12617</div>-<div class="lpage">28</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%2Ffunction%20mapping%20of%20amino%20acids%20in%20the%20N-terminal%20zinc%20finger%20of%20the%20human%20immunodeficiency%20virus%20type%201%20nucleocapsid%20protein%3A%20residues%20responsible%20for%20nucleic%20acid%20helix%20destabilizing%20activity&amp;author=N.%20Narayanan&amp;author=R.%20Gorelick&amp;author=J.%20DeStefano&amp;publication_year=2006&amp;journal=Biochemistry&amp;volume=45&amp;pages=12617-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/bi060925c" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fbi060925c" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fbi060925c"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17029416" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST020C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Matsui</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kodera</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Endoh</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">RNA recognition mechanism of the minimal active domain of the human immunodeficiency virus type-2 nucleocapsid protein</div>, <div class="source">J. Biochem.</div> , <div class="year">2007</div>, vol. <div class="volume">141</div> (pg. <div class="fpage">269</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20recognition%20mechanism%20of%20the%20minimal%20active%20domain%20of%20the%20human%20immunodeficiency%20virus%20type-2%20nucleocapsid%20protein&amp;author=T.%20Matsui&amp;author=Y.%20Kodera&amp;author=H.%20Endoh&amp;publication_year=2007&amp;journal=J.%20Biochem.&amp;volume=141&amp;pages=269-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jb/mvm037" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjb%2Fmvm037" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjb%2Fmvm037"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17202191" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST020C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Curtis</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Treiber</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Tao</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Zamore</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Williamson</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lehmann</div> <div class="given-names">R.</div></div>. </div><div class="article-title">A CCHC metal-binding domain in Nanos is essential for translational regulation</div>, <div class="source">EMBO J.</div> , <div class="year">1997</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">834</div>-<div class="lpage">43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20CCHC%20metal-binding%20domain%20in%20Nanos%20is%20essential%20for%20translational%20regulation&amp;author=D.%20Curtis&amp;author=D.%20Treiber&amp;author=F.%20Tao&amp;author=P.%20Zamore&amp;author=J.%20Williamson&amp;author=R.%20Lehmann&amp;publication_year=1997&amp;journal=EMBO%20J.&amp;volume=16&amp;pages=834-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/emboj/16.4.834" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Femboj%2F16.4.834" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Femboj%2F16.4.834"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9049312" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST020C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Liang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Deng</div> <div class="given-names">W.</div></div>, et al. </div><div class="article-title">The zinc-finger protein CNBP is required for forebrain formation in the mouse</div>, <div class="source">Development</div> , <div class="year">2003</div>, vol. <div class="volume">130</div> (pg. <div class="fpage">1367</div>-<div class="lpage">79</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20zinc-finger%20protein%20CNBP%20is%20required%20for%20forebrain%20formation%20in%20the%20mouse&amp;author=W.%20Chen&amp;author=Y.%20Liang&amp;author=W.%20Deng&amp;publication_year=2003&amp;journal=Development&amp;volume=130&amp;pages=1367-79" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1242/dev.00349" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1242%2Fdev.00349" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1242%2Fdev.00349"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12588852" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST020C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rajavashisth</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Taylor</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Andalibi</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Svenson</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Lusis</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Identification of a zinc finger protein that binds to the sterol regulatory element</div>, <div class="source">Science</div> , <div class="year">1989</div>, vol. <div class="volume">245</div> (pg. <div class="fpage">640</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20a%20zinc%20finger%20protein%20that%20binds%20to%20the%20sterol%20regulatory%20element&amp;author=T.%20Rajavashisth&amp;author=A.%20Taylor&amp;author=A.%20Andalibi&amp;author=K.%20Svenson&amp;author=A.%20Lusis&amp;publication_year=1989&amp;journal=Science&amp;volume=245&amp;pages=640-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.2562787" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.2562787" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.2562787"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2562787" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST020C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST020C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schlatter</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Fussenegger</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Novel CNBP- and La-based translation control systems for mammalian cells</div>, <div class="source">Biotechnol. Bioeng.</div> , <div class="year">2003</div>, vol. <div class="volume">81</div> (pg. <div class="fpage">1</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20CNBP-%20and%20La-based%20translation%20control%20systems%20for%20mammalian%20cells&amp;author=S.%20Schlatter&amp;author=M.%20Fussenegger&amp;publication_year=2003&amp;journal=Biotechnol.%20Bioeng.&amp;volume=81&amp;pages=1-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0290" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0290" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0290"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12432575" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Toshihiko Shiroishi</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=359087&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/5/10.1093/dnares/dst020/2/dst020-dst020supp.doc?Expires=2147483647&amp;Signature=gHeznmhlPble-Zz-aGu0Y~D5~rGCUMsCluERYx-qgq2vdKzcm-TnTz0A3ZiFeYZpKIV7teuTvHBsWmr2BRKGmCL3V-QhMqLUaCfIHGU5x54yvBxkrNJLb1AtXFDolqgwwpCBWiut5cLuLni-bCJdauYkpHqJVQqzlOBp5lNf-LaKrHHUCFN6oMYBeoZ2zfOC1L-UupoU-sA0m6B901KpTsZsmGvjsC1IBIHVuWZQ7cQUfF57w3Y~mDMuIuZui~f4jDla~fOaO67o7~msZvMKcZGFSJtM0qN2l7TuKVg9tqYoyaW28Z51~FW-g3hp0-SVh8JTNryYneDzIAO99MMLFg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">239</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000330167500001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">8</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst020"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="359087"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000330167500001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f5%2f425%2f359087">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000330167500001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (8)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F5%2F425%2F359087" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/359087'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Identification of a Novel PNMA-MS1 Gene in Marsupials Suggests the LTR Retrotransposon-Derived PNMA Genes Evolved Differently in Marsupials and Eutherians" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst020");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f5%2f425%2f359087">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

