

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/19/5/423/629170","@type":"ScholarlyArticle","name":"Efficient Detection, Quantification and Enrichment of Subtle Allelic Alterations","datePublished":"2012-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/19/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2012-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss023","keywords":["gene targeting","heteroduplex analysis","zinc finger nucleases","transcription activator-like effector nucleases","polyacrylamide gel electrophoresis"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/19/5/423/629170","author":[{"name":"Chen, Jianbin","affiliation":"1  Department of Biological Sciences ,  National University of Singapore ,  14 Science Drive 4 ,  Singapore  117543 ,  Singapore","@type":"Person"},{"name":"Zhang, Xi","affiliation":"1  Department of Biological Sciences ,  National University of Singapore ,  14 Science Drive 4 ,  Singapore  117543 ,  Singapore","@type":"Person"},{"name":"Wang, Tiansu","affiliation":"1  Department of Biological Sciences ,  National University of Singapore ,  14 Science Drive 4 ,  Singapore  117543 ,  Singapore","@type":"Person"},{"name":"Li, Zhendong","affiliation":"1  Department of Biological Sciences ,  National University of Singapore ,  14 Science Drive 4 ,  Singapore  117543 ,  Singapore","@type":"Person"},{"name":"Guan, Guijun","affiliation":"1  Department of Biological Sciences ,  National University of Singapore ,  14 Science Drive 4 ,  Singapore  117543 ,  Singapore    2  Department of Bioresources ,  National Institute for Basic Biology ,  Okazaki, Aichi 444-8585 ,  Japan","@type":"Person"},{"name":"Hong, Yunhan","affiliation":"1  Department of Biological Sciences ,  National University of Singapore ,  14 Science Drive 4 ,  Singapore  117543 ,  Singapore","@type":"Person"}],"description":"Abstract.  Gene targeting (GT) can introduce subtle alterations into a particular locus and represents a powerful tool for genome editing. Engineered zinc finge","pageStart":"423","pageEnd":"433","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/5/1/m_cover.gif?Expires=2147483647&Signature=oCF62cOF-RHSHHzNh6fqRIkUeFdGFZWfw3pKB6h-XrXTUmjPkfpfn8w~8bchsCbKMP2r~GVw22lZ1MNx0q-s71-AcQyMN0lvE0QjuCe~oMOmvkISrjvYiWPgB2qMv~4jOVZX-tDdrzrIeyUlp6-dE-d1BMbDpzvRnY6ImlMPnjamQ8tNFPiRg84G~Uknop7I8711Uikreov-Syq2nbs4ZI3p39Ios3nBrhdKb3cghqgrrGVSW3I0iXxFPXkjQY1cpzGh8rn-oZc~FVxVdjoBppF4WPcNdaRzbNDAFK0kliJ6Rs5rfCQy64JYiBPIa6tUr3wfBbp66X2fB6q0zAaV3g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Efficient Detection, Quantification and Enrichment of Subtle Allelic Alterations","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/5/1/m_cover.gif?Expires=2147483647&Signature=oCF62cOF-RHSHHzNh6fqRIkUeFdGFZWfw3pKB6h-XrXTUmjPkfpfn8w~8bchsCbKMP2r~GVw22lZ1MNx0q-s71-AcQyMN0lvE0QjuCe~oMOmvkISrjvYiWPgB2qMv~4jOVZX-tDdrzrIeyUlp6-dE-d1BMbDpzvRnY6ImlMPnjamQ8tNFPiRg84G~Uknop7I8711Uikreov-Syq2nbs4ZI3p39Ios3nBrhdKb3cghqgrrGVSW3I0iXxFPXkjQY1cpzGh8rn-oZc~FVxVdjoBppF4WPcNdaRzbNDAFK0kliJ6Rs5rfCQy64JYiBPIa6tUr3wfBbp66X2fB6q0zAaV3g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Efficient Detection, Quantification and Enrichment of Subtle Allelic Alterations" />
    <meta property="og:description" content="Abstract.  Gene targeting (GT) can introduce subtle alterations into a particular locus and represents a powerful tool for genome editing. Engineered zinc finge" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/19/5/423/629170" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2012-10-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Chen, Jianbin" /><meta name="citation_author_institution" content="Department of Biological Sciences, National University of Singapore, 14 Science Drive 4, Singapore117543, Singapore" /><meta name="citation_author" content="Zhang, Xi" /><meta name="citation_author_institution" content="Department of Biological Sciences, National University of Singapore, 14 Science Drive 4, Singapore117543, Singapore" /><meta name="citation_author" content="Wang, Tiansu" /><meta name="citation_author_institution" content="Department of Biological Sciences, National University of Singapore, 14 Science Drive 4, Singapore117543, Singapore" /><meta name="citation_author" content="Li, Zhendong" /><meta name="citation_author_institution" content="Department of Biological Sciences, National University of Singapore, 14 Science Drive 4, Singapore117543, Singapore" /><meta name="citation_author" content="Guan, Guijun" /><meta name="citation_author_institution" content="Department of Biological Sciences, National University of Singapore, 14 Science Drive 4, Singapore117543, Singapore" /><meta name="citation_author_institution" content="Department of Bioresources, National Institute for Basic Biology, Okazaki, Aichi 444-8585, Japan" /><meta name="citation_author" content="Hong, Yunhan" /><meta name="citation_author_institution" content="Department of Biological Sciences, National University of Singapore, 14 Science Drive 4, Singapore117543, Singapore" /><meta name="citation_title" content="Efficient Detection, Quantification and Enrichment of Subtle Allelic Alterations" /><meta name="citation_firstpage" content="423" /><meta name="citation_lastpage" content="433" /><meta name="citation_doi" content="10.1093/dnares/dss023" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="19" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2012/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Gene targeting in mice: functional analysis of the mammalian genome for the twenty-first century; Nat. Rev. Genet.;  citation_year=2005;  citation_volume=6;  citation_pages=507-12; " /><meta name="citation_reference" content="citation_title=Impaired DNA damage response, genome instability, and tumorigenesis in SIRT1 mutant mice; Cancer Cell;  citation_year=2008;  citation_volume=14;  citation_pages=312-23; " /><meta name="citation_reference" content="citation_title=A mouse for all reasons; Cell;  citation_year=2007;  citation_volume=128;  citation_pages=9-13; " /><meta name="citation_reference" content="citation_title=Production of p53 gene knockout rats by homologous recombination in embryonic stem cells; Nature;  citation_year=2010;  citation_volume=467;  citation_pages=211-3; " /><meta name="citation_reference" content="citation_title=A TALE of two nucleases: gene targeting for the masses?; Zebrafish;  citation_year=2011;  citation_volume=8;  citation_pages=147-9; " /><meta name="citation_reference" content="citation_title=Heritable targeted gene disruption in zebrafish using designed zinc-finger nucleases; Nat. Biotechol.;  citation_year=2008;  citation_volume=26;  citation_pages=702-8; " /><meta name="citation_reference" content="citation_title=Targeted disruption of exogenous EGFP gene in medaka using zinc-finger nucleases; Dev. Growth Differ.;  citation_year=2012;  citation_volume=54;  citation_pages=546-56; " /><meta name="citation_reference" content="citation_title=Heritable gene targeting in zebrafish using customized TALENs; Nat. Biotechnol.;  citation_year=2011;  citation_volume=29;  citation_pages=699-700; " /><meta name="citation_reference" content="citation_title=Targeted gene disruption in somatic zebrafish cells using engineered TALENs; Nat. Biotechnol.;  citation_year=2011;  citation_volume=29;  citation_pages=697-8; " /><meta name="citation_reference" content="citation_title=Double-strand-break-induced homologous recombination in mammalian cells; Biochem. Soc. Trans.;  citation_year=2001;  citation_volume=29;  citation_pages=196-201; " /><meta name="citation_reference" content="citation_title=Engineered zinc finger nickases induce homology-directed repair with reduced mutagenic effects; Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=5560-8; " /><meta name="citation_reference" content="citation_title=Selection-independent generation of gene knockout mouse embryonic stem cells using zinc-finger nucleases; PLoS One;  citation_year=2012;  citation_volume=6;  citation_pages=e28911" /><meta name="citation_reference" content="citation_title=Monomeric site-specific nucleases for genome editing; Proc. Natl Acad. Sci. USA;  citation_year=2012;  citation_volume=109;  citation_pages=8061-6; " /><meta name="citation_reference" content="citation_title=Targeted mutagenesis in the silkworm Bombyx mori using zinc finger nuclease mRNA injection; Insect Biochem. Mol. Biol.;  citation_year=2010;  citation_volume=40;  citation_pages=759-65; " /><meta name="citation_reference" content="citation_title=Targeted genome editing in human cells with zinc finger nucleases constructed via modular assembly; Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=1279-88; " /><meta name="citation_reference" content="citation_title=Rapid ‘open-source’ engineering of customized zinc-finger nucleases for highly efficient gene modification; Mol. Cell;  citation_year=2008;  citation_volume=31;  citation_pages=294-301; " /><meta name="citation_reference" content="citation_title=The use of resolvases T4 endonuclease VII and T7 endonuclease I in mutation detection; Mol. Biotechnol.;  citation_year=2003;  citation_volume=23;  citation_pages=73-81; " /><meta name="citation_reference" content="citation_title=Two-step total gene synthesis method; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=e59" /><meta name="citation_reference" content="citation_title=Combined SSCP/duplex analysis by capillary electrophoresis for more efficient mutation detection; Nucleic Acids Res.;  citation_year=2001;  citation_volume=29;  citation_pages=e71" /><meta name="citation_reference" content="citation_title=Single-molecule DNA amplification and analysis in an integrated microfluidic device; Anal. Chem.;  citation_year=2001;  citation_volume=73;  citation_pages=565-70; " /><meta name="citation_reference" content="citation_title=High-resolution genotyping by amplicon melting analysis using LCGreen; Clin. Chem.;  citation_year=2003;  citation_volume=49;  citation_pages=853-60; " /><meta name="citation_reference" content="citation_title=High-resolution melting curve analysis for rapid detection of mutations in a Medaka TILLING library; BMC Mol. Biol.;  citation_year=2010;  citation_volume=11;  citation_pages=70" /><meta name="citation_reference" content="citation_title=Direct detection and quantification of methylation in nucleic acid sequences using high-resolution melting analysis; Anal. Chem.;  citation_year=2010;  citation_volume=82;  citation_pages=9100-8; " /><meta name="citation_reference" content="citation_title=Genetic polymorphism in normal human fibroblasts as analyzed by two-dimensional polyacrylamide gel electrophoresis; J. Biol. Chem.;  citation_year=1979;  citation_volume=254;  citation_pages=7951-60; " /><meta name="citation_reference" content="citation_title=Human MutSalpha specifically binds to DNA containing aminofluorene and acetylaminofluorene adducts; J. Biol. Chem.;  citation_year=1996;  citation_volume=271;  citation_pages=24084-8; " /><meta name="citation_reference" content="citation_title=Fish stem cell cultures; Int. J. Biol. Sci.;  citation_year=2011;  citation_volume=7;  citation_pages=392-402; " /><meta name="citation_reference" content="citation_title=Production of medakafish chimeras from a stable embryonic stem cell line; Proc. Natl Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=3679-84; " /><meta name="citation_reference" content="citation_title=Modeling of heteroduplex formation during PCR from mixtures of DNA templates; PCR Methods Appl.;  citation_year=1992;  citation_volume=2;  citation_pages=112-6; " /><meta name="citation_reference" content="citation_title=Use of a DNA toolbox for the characterization of mutation scanning methods. I: construction of the toolbox and evaluation of heteroduplex analysis; Electrophoresis;  citation_year=1999;  citation_volume=20;  citation_pages=1186-94; " /><meta name="citation_reference" content="citation_title=A rapid alkaline extraction procedure for screening recombinant plasmid DNA; Nucleic Acids Res.;  citation_year=1979;  citation_volume=7;  citation_pages=1513-23; " /><meta name="citation_reference" content="citation_title=Medaka vasa is required for migration but not survival of primordial germ cells; Mech. Dev.;  citation_year=2009;  citation_volume=126;  citation_pages=366-381; " /><meta name="citation_reference" content="citation_title=Isolation of DNA fragments from polyacrylamide gels by the crush and soak method; Cold Spring Harb Protoc.;  citation_year=2006; " /><meta name="citation_reference" content="citation_title=Single-strand conformation polymorphism and heteroduplex analysis for gel-based mutation detection; Electrophoresis;  citation_year=1999;  citation_volume=20;  citation_pages=1177-85; " /><meta name="citation_reference" content="citation_title=Targeted mutagenesis in zebrafish using customized zinc-finger nucleases; Nat. Protoc.;  citation_year=2009;  citation_volume=4;  citation_pages=1855-68; " /><meta name="citation_reference" content="citation_title=Targeted gene inactivation in zebrafish using engineered zinc-finger nucleases; Nat. Biotechnol.;  citation_year=2008;  citation_volume=26;  citation_pages=695-701; " /><meta name="citation_reference" content="citation_title=Pluripotency and differentiation of embryonic stem cell lines from the medakafish (Oryzias latipes); Mech. Dev.;  citation_year=1996;  citation_volume=60;  citation_pages=33-44; " /><meta name="citation_reference" content="citation_title=Generation of medaka fish haploid embryonic stem cells; Science;  citation_year=2009;  citation_volume=326;  citation_pages=430-3; " /><meta name="citation_reference" content="citation_title=Establishment of a normal medakafish spermatogonial cell line capable of sperm production in vitro; Proc. Natl Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=8011-6; " /><meta name="citation_reference" content="citation_title=Accessibility of host cell lineages to medaka stem cells depends on genetic background and irradiation of recipient embryos; Cell. Mol. Life Sci.;  citation_year=2010;  citation_volume=67;  citation_pages=1189-202; " /><meta name="citation_reference" content="citation_title=Sperm nuclear transfer and transgenic production in the fish Medaka; Int. J. Biol. Sci.;  citation_year=2011;  citation_volume=7;  citation_pages=469-75; " /><meta name="citation_reference" content="citation_title=Expression of gonadal soma derived factor (GSDF) is spatially and temporally correlated with early testicular differentiation in medaka; Gene Expr. Patterns;  citation_year=2010;  citation_volume=10;  citation_pages=283-9; " /><meta name="citation_reference" content="citation_title=Targeted integration in rat and mouse embryos with zinc-finger nucleases; Nat. Biotechnol.;  citation_year=2011;  citation_volume=29;  citation_pages=64-7; " /><meta name="citation_reference" content="citation_title=Comparison of heteroduplex and single-strand conformation analyses, followed by ethidium fluorescence visualization, for the detection of mutations in four human genes; Mol. Cell. Probes;  citation_year=1995;  citation_volume=9;  citation_pages=195-200; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/19/5/423/1753024/dss023.pdf" /><meta name="description" content="Abstract.  Gene targeting (GT) can introduce subtle alterations into a particular locus and represents a powerful tool for genome editing. Engineered zinc finge" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/19/5/423/629170" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/19/5/423/629170" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/19/5/423/629170");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dss023';
    </script>
<title>
	Efficient Detection, Quantification and Enrichment of Subtle Allelic Alterations | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./629170" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BeNgPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDaGVuLCBKaWFuYmluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9sb2dpY2FsIFNjaWVuY2VzLCBOYXRpb25hbCBVbml2ZXJzaXR5IG9mIFNpbmdhcG9yZSwgMTQgU2NpZW5jZSBEcml2ZSA0LCBTaW5nYXBvcmUxMTc1NDMsIFNpbmdhcG9yZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlpoYW5nLCBYaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9naWNhbCBTY2llbmNlcywgTmF0aW9uYWwgVW5pdmVyc2l0eSBvZiBTaW5nYXBvcmUsIDE0IFNjaWVuY2UgRHJpdmUgNCwgU2luZ2Fwb3JlMTE3NTQzLCBTaW5nYXBvcmUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXYW5nLCBUaWFuc3UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMsIE5hdGlvbmFsIFVuaXZlcnNpdHkgb2YgU2luZ2Fwb3JlLCAxNCBTY2llbmNlIERyaXZlIDQsIFNpbmdhcG9yZTExNzU0MywgU2luZ2Fwb3JlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGksIFpoZW5kb25nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9sb2dpY2FsIFNjaWVuY2VzLCBOYXRpb25hbCBVbml2ZXJzaXR5IG9mIFNpbmdhcG9yZSwgMTQgU2NpZW5jZSBEcml2ZSA0LCBTaW5nYXBvcmUxMTc1NDMsIFNpbmdhcG9yZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikd1YW4sIEd1aWp1biIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9naWNhbCBTY2llbmNlcywgTmF0aW9uYWwgVW5pdmVyc2l0eSBvZiBTaW5nYXBvcmUsIDE0IFNjaWVuY2UgRHJpdmUgNCwgU2luZ2Fwb3JlMTE3NTQzLCBTaW5nYXBvcmUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb3Jlc291cmNlcywgTmF0aW9uYWwgSW5zdGl0dXRlIGZvciBCYXNpYyBCaW9sb2d5LCBPa2F6YWtpLCBBaWNoaSA0NDQtODU4NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIb25nLCBZdW5oYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMsIE5hdGlvbmFsIFVuaXZlcnNpdHkgb2YgU2luZ2Fwb3JlLCAxNCBTY2llbmNlIERyaXZlIDQsIFNpbmdhcG9yZTExNzU0MywgU2luZ2Fwb3JlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJFZmZpY2llbnQgRGV0ZWN0aW9uLCBRdWFudGlmaWNhdGlvbiBhbmQgRW5yaWNobWVudCBvZiBTdWJ0bGUgQWxsZWxpYyBBbHRlcmF0aW9ucyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjQyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNDMzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNzMDIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTIvMTAvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIHRhcmdldGluZyBpbiBtaWNlOiBmdW5jdGlvbmFsIGFuYWx5c2lzIG9mIHRoZSBtYW1tYWxpYW4gZ2Vub21lIGZvciB0aGUgdHdlbnR5LWZpcnN0IGNlbnR1cnk7IE5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTUwNy0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1wYWlyZWQgRE5BIGRhbWFnZSByZXNwb25zZSwgZ2Vub21lIGluc3RhYmlsaXR5LCBhbmQgdHVtb3JpZ2VuZXNpcyBpbiBTSVJUMSBtdXRhbnQgbWljZTsgQ2FuY2VyIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0zMTItMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbW91c2UgZm9yIGFsbCByZWFzb25zOyBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTEyODsgIGNpdGF0aW9uX3BhZ2VzPTktMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb2R1Y3Rpb24gb2YgcDUzIGdlbmUga25vY2tvdXQgcmF0cyBieSBob21vbG9nb3VzIHJlY29tYmluYXRpb24gaW4gZW1icnlvbmljIHN0ZW0gY2VsbHM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00Njc7ICBjaXRhdGlvbl9wYWdlcz0yMTEtMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBUQUxFIG9mIHR3byBudWNsZWFzZXM6IGdlbmUgdGFyZ2V0aW5nIGZvciB0aGUgbWFzc2VzPzsgWmVicmFmaXNoOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0xNDctOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGVyaXRhYmxlIHRhcmdldGVkIGdlbmUgZGlzcnVwdGlvbiBpbiB6ZWJyYWZpc2ggdXNpbmcgZGVzaWduZWQgemluYy1maW5nZXIgbnVjbGVhc2VzOyBOYXQuIEJpb3RlY2hvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz03MDItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFyZ2V0ZWQgZGlzcnVwdGlvbiBvZiBleG9nZW5vdXMgRUdGUCBnZW5lIGluIG1lZGFrYSB1c2luZyB6aW5jLWZpbmdlciBudWNsZWFzZXM7IERldi4gR3Jvd3RoIERpZmZlci47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz01NDYtNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhlcml0YWJsZSBnZW5lIHRhcmdldGluZyBpbiB6ZWJyYWZpc2ggdXNpbmcgY3VzdG9taXplZCBUQUxFTnM7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz02OTktNzAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYXJnZXRlZCBnZW5lIGRpc3J1cHRpb24gaW4gc29tYXRpYyB6ZWJyYWZpc2ggY2VsbHMgdXNpbmcgZW5naW5lZXJlZCBUQUxFTnM7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz02OTctODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RG91YmxlLXN0cmFuZC1icmVhay1pbmR1Y2VkIGhvbW9sb2dvdXMgcmVjb21iaW5hdGlvbiBpbiBtYW1tYWxpYW4gY2VsbHM7IEJpb2NoZW0uIFNvYy4gVHJhbnMuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9MTk2LTIwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW5naW5lZXJlZCB6aW5jIGZpbmdlciBuaWNrYXNlcyBpbmR1Y2UgaG9tb2xvZ3ktZGlyZWN0ZWQgcmVwYWlyIHdpdGggcmVkdWNlZCBtdXRhZ2VuaWMgZWZmZWN0czsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9NTU2MC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWxlY3Rpb24taW5kZXBlbmRlbnQgZ2VuZXJhdGlvbiBvZiBnZW5lIGtub2Nrb3V0IG1vdXNlIGVtYnJ5b25pYyBzdGVtIGNlbGxzIHVzaW5nIHppbmMtZmluZ2VyIG51Y2xlYXNlczsgUExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUyODkxMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbm9tZXJpYyBzaXRlLXNwZWNpZmljIG51Y2xlYXNlcyBmb3IgZ2Vub21lIGVkaXRpbmc7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTA5OyAgY2l0YXRpb25fcGFnZXM9ODA2MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYXJnZXRlZCBtdXRhZ2VuZXNpcyBpbiB0aGUgc2lsa3dvcm0gQm9tYnl4IG1vcmkgdXNpbmcgemluYyBmaW5nZXIgbnVjbGVhc2UgbVJOQSBpbmplY3Rpb247IEluc2VjdCBCaW9jaGVtLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9NzU5LTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYXJnZXRlZCBnZW5vbWUgZWRpdGluZyBpbiBodW1hbiBjZWxscyB3aXRoIHppbmMgZmluZ2VyIG51Y2xlYXNlcyBjb25zdHJ1Y3RlZCB2aWEgbW9kdWxhciBhc3NlbWJseTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0xMjc5LTg4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXBpZCDigJhvcGVuLXNvdXJjZeKAmSBlbmdpbmVlcmluZyBvZiBjdXN0b21pemVkIHppbmMtZmluZ2VyIG51Y2xlYXNlcyBmb3IgaGlnaGx5IGVmZmljaWVudCBnZW5lIG1vZGlmaWNhdGlvbjsgTW9sLiBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9Mjk0LTMwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHVzZSBvZiByZXNvbHZhc2VzIFQ0IGVuZG9udWNsZWFzZSBWSUkgYW5kIFQ3IGVuZG9udWNsZWFzZSBJIGluIG11dGF0aW9uIGRldGVjdGlvbjsgTW9sLiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTczLTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ud28tc3RlcCB0b3RhbCBnZW5lIHN5bnRoZXNpcyBtZXRob2Q7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPWU1OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbWJpbmVkIFNTQ1AvZHVwbGV4IGFuYWx5c2lzIGJ5IGNhcGlsbGFyeSBlbGVjdHJvcGhvcmVzaXMgZm9yIG1vcmUgZWZmaWNpZW50IG11dGF0aW9uIGRldGVjdGlvbjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9ZTcxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2luZ2xlLW1vbGVjdWxlIEROQSBhbXBsaWZpY2F0aW9uIGFuZCBhbmFseXNpcyBpbiBhbiBpbnRlZ3JhdGVkIG1pY3JvZmx1aWRpYyBkZXZpY2U7IEFuYWwuIENoZW0uOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTczOyAgY2l0YXRpb25fcGFnZXM9NTY1LTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLXJlc29sdXRpb24gZ2Vub3R5cGluZyBieSBhbXBsaWNvbiBtZWx0aW5nIGFuYWx5c2lzIHVzaW5nIExDR3JlZW47IENsaW4uIENoZW0uOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQ5OyAgY2l0YXRpb25fcGFnZXM9ODUzLTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLXJlc29sdXRpb24gbWVsdGluZyBjdXJ2ZSBhbmFseXNpcyBmb3IgcmFwaWQgZGV0ZWN0aW9uIG9mIG11dGF0aW9ucyBpbiBhIE1lZGFrYSBUSUxMSU5HIGxpYnJhcnk7IEJNQyBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NzAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXJlY3QgZGV0ZWN0aW9uIGFuZCBxdWFudGlmaWNhdGlvbiBvZiBtZXRoeWxhdGlvbiBpbiBudWNsZWljIGFjaWQgc2VxdWVuY2VzIHVzaW5nIGhpZ2gtcmVzb2x1dGlvbiBtZWx0aW5nIGFuYWx5c2lzOyBBbmFsLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT04MjsgIGNpdGF0aW9uX3BhZ2VzPTkxMDAtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBwb2x5bW9ycGhpc20gaW4gbm9ybWFsIGh1bWFuIGZpYnJvYmxhc3RzIGFzIGFuYWx5emVkIGJ5IHR3by1kaW1lbnNpb25hbCBwb2x5YWNyeWxhbWlkZSBnZWwgZWxlY3Ryb3Bob3Jlc2lzOyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk3OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTQ7ICBjaXRhdGlvbl9wYWdlcz03OTUxLTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IdW1hbiBNdXRTYWxwaGEgc3BlY2lmaWNhbGx5IGJpbmRzIHRvIEROQSBjb250YWluaW5nIGFtaW5vZmx1b3JlbmUgYW5kIGFjZXR5bGFtaW5vZmx1b3JlbmUgYWRkdWN0czsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MjcxOyAgY2l0YXRpb25fcGFnZXM9MjQwODQtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmlzaCBzdGVtIGNlbGwgY3VsdHVyZXM7IEludC4gSi4gQmlvbC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MzkyLTQwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvZHVjdGlvbiBvZiBtZWRha2FmaXNoIGNoaW1lcmFzIGZyb20gYSBzdGFibGUgZW1icnlvbmljIHN0ZW0gY2VsbCBsaW5lOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTk1OyAgY2l0YXRpb25fcGFnZXM9MzY3OS04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9kZWxpbmcgb2YgaGV0ZXJvZHVwbGV4IGZvcm1hdGlvbiBkdXJpbmcgUENSIGZyb20gbWl4dHVyZXMgb2YgRE5BIHRlbXBsYXRlczsgUENSIE1ldGhvZHMgQXBwbC47ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTExMi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Vc2Ugb2YgYSBETkEgdG9vbGJveCBmb3IgdGhlIGNoYXJhY3Rlcml6YXRpb24gb2YgbXV0YXRpb24gc2Nhbm5pbmcgbWV0aG9kcy4gSTogY29uc3RydWN0aW9uIG9mIHRoZSB0b29sYm94IGFuZCBldmFsdWF0aW9uIG9mIGhldGVyb2R1cGxleCBhbmFseXNpczsgRWxlY3Ryb3Bob3Jlc2lzOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTE4Ni05NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSByYXBpZCBhbGthbGluZSBleHRyYWN0aW9uIHByb2NlZHVyZSBmb3Igc2NyZWVuaW5nIHJlY29tYmluYW50IHBsYXNtaWQgRE5BOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5Nzk7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTE1MTMtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lZGFrYSB2YXNhIGlzIHJlcXVpcmVkIGZvciBtaWdyYXRpb24gYnV0IG5vdCBzdXJ2aXZhbCBvZiBwcmltb3JkaWFsIGdlcm0gY2VsbHM7IE1lY2guIERldi47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTI2OyAgY2l0YXRpb25fcGFnZXM9MzY2LTM4MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uIG9mIEROQSBmcmFnbWVudHMgZnJvbSBwb2x5YWNyeWxhbWlkZSBnZWxzIGJ5IHRoZSBjcnVzaCBhbmQgc29hayBtZXRob2Q7IENvbGQgU3ByaW5nIEhhcmIgUHJvdG9jLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2luZ2xlLXN0cmFuZCBjb25mb3JtYXRpb24gcG9seW1vcnBoaXNtIGFuZCBoZXRlcm9kdXBsZXggYW5hbHlzaXMgZm9yIGdlbC1iYXNlZCBtdXRhdGlvbiBkZXRlY3Rpb247IEVsZWN0cm9waG9yZXNpczsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTExNzctODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRhcmdldGVkIG11dGFnZW5lc2lzIGluIHplYnJhZmlzaCB1c2luZyBjdXN0b21pemVkIHppbmMtZmluZ2VyIG51Y2xlYXNlczsgTmF0LiBQcm90b2MuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0xODU1LTY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYXJnZXRlZCBnZW5lIGluYWN0aXZhdGlvbiBpbiB6ZWJyYWZpc2ggdXNpbmcgZW5naW5lZXJlZCB6aW5jLWZpbmdlciBudWNsZWFzZXM7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz02OTUtNzAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbHVyaXBvdGVuY3kgYW5kIGRpZmZlcmVudGlhdGlvbiBvZiBlbWJyeW9uaWMgc3RlbSBjZWxsIGxpbmVzIGZyb20gdGhlIG1lZGFrYWZpc2ggKE9yeXppYXMgbGF0aXBlcyk7IE1lY2guIERldi47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9NjA7ICBjaXRhdGlvbl9wYWdlcz0zMy00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXJhdGlvbiBvZiBtZWRha2EgZmlzaCBoYXBsb2lkIGVtYnJ5b25pYyBzdGVtIGNlbGxzOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTMyNjsgIGNpdGF0aW9uX3BhZ2VzPTQzMC0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fc3RhYmxpc2htZW50IG9mIGEgbm9ybWFsIG1lZGFrYWZpc2ggc3Blcm1hdG9nb25pYWwgY2VsbCBsaW5lIGNhcGFibGUgb2Ygc3Blcm0gcHJvZHVjdGlvbiBpbiB2aXRybzsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDE7ICBjaXRhdGlvbl9wYWdlcz04MDExLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFjY2Vzc2liaWxpdHkgb2YgaG9zdCBjZWxsIGxpbmVhZ2VzIHRvIG1lZGFrYSBzdGVtIGNlbGxzIGRlcGVuZHMgb24gZ2VuZXRpYyBiYWNrZ3JvdW5kIGFuZCBpcnJhZGlhdGlvbiBvZiByZWNpcGllbnQgZW1icnlvczsgQ2VsbC4gTW9sLiBMaWZlIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Njc7ICBjaXRhdGlvbl9wYWdlcz0xMTg5LTIwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3Blcm0gbnVjbGVhciB0cmFuc2ZlciBhbmQgdHJhbnNnZW5pYyBwcm9kdWN0aW9uIGluIHRoZSBmaXNoIE1lZGFrYTsgSW50LiBKLiBCaW9sLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz00NjktNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cHJlc3Npb24gb2YgZ29uYWRhbCBzb21hIGRlcml2ZWQgZmFjdG9yIChHU0RGKSBpcyBzcGF0aWFsbHkgYW5kIHRlbXBvcmFsbHkgY29ycmVsYXRlZCB3aXRoIGVhcmx5IHRlc3RpY3VsYXIgZGlmZmVyZW50aWF0aW9uIGluIG1lZGFrYTsgR2VuZSBFeHByLiBQYXR0ZXJuczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTI4My05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYXJnZXRlZCBpbnRlZ3JhdGlvbiBpbiByYXQgYW5kIG1vdXNlIGVtYnJ5b3Mgd2l0aCB6aW5jLWZpbmdlciBudWNsZWFzZXM7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz02NC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJpc29uIG9mIGhldGVyb2R1cGxleCBhbmQgc2luZ2xlLXN0cmFuZCBjb25mb3JtYXRpb24gYW5hbHlzZXMsIGZvbGxvd2VkIGJ5IGV0aGlkaXVtIGZsdW9yZXNjZW5jZSB2aXN1YWxpemF0aW9uLCBmb3IgdGhlIGRldGVjdGlvbiBvZiBtdXRhdGlvbnMgaW4gZm91ciBodW1hbiBnZW5lczsgTW9sLiBDZWxsLiBQcm9iZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTE5NS0yMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xOS81LzQyMy8xNzUzMDI0L2RzczAyMy5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgR2VuZSB0YXJnZXRpbmcgKEdUKSBjYW4gaW50cm9kdWNlIHN1YnRsZSBhbHRlcmF0aW9ucyBpbnRvIGEgcGFydGljdWxhciBsb2N1cyBhbmQgcmVwcmVzZW50cyBhIHBvd2VyZnVsIHRvb2wgZm9yIGdlbm9tZSBlZGl0aW5nLiBFbmdpbmVlcmVkIHppbmMgZmluZ2UiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTYyOTE3MCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGR9rQreqR+kaz2ukZsrs3ymCjwnaA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f5%2f423%2f629170" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f5%2f423%2f629170" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=25687" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/19/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/5/1/m_cover.gif?Expires=2147483647&amp;Signature=oCF62cOF-RHSHHzNh6fqRIkUeFdGFZWfw3pKB6h-XrXTUmjPkfpfn8w~8bchsCbKMP2r~GVw22lZ1MNx0q-s71-AcQyMN0lvE0QjuCe~oMOmvkISrjvYiWPgB2qMv~4jOVZX-tDdrzrIeyUlp6-dE-d1BMbDpzvRnY6ImlMPnjamQ8tNFPiRg84G~Uknop7I8711Uikreov-Syq2nbs4ZI3p39Ios3nBrhdKb3cghqgrrGVSW3I0iXxFPXkjQY1cpzGh8rn-oZc~FVxVdjoBppF4WPcNdaRzbNDAFK0kliJ6Rs5rfCQy64JYiBPIa6tUr3wfBbp66X2fB6q0zAaV3g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/19/5">
                        <div class="volume trailing-comma">Volume 19</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="10646292">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10646292">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10646294">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10646294">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10646300">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10646300">2. Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10646329">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10646329">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10646351">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10646351">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10646359">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10646359">Authors’ contributions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10646361">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10646361">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10646363">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10646363">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10646365">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10646365">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10646367">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10646367">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/19/5/407/629039">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Efficient Detection, Quantification and Enrichment of Subtle Allelic Alterations
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jianbin Chen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biological Sciences</div>, <div class="insititution">National University of Singapore</div>, <div class="addr-line">14 Science Drive 4</div>, <div class="country">Singapore</div><div class="addr-line">117543</div>, <div class="country">Singapore</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jianbin+Chen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chen J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chen J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jianbin Chen</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xi Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biological Sciences</div>, <div class="insititution">National University of Singapore</div>, <div class="addr-line">14 Science Drive 4</div>, <div class="country">Singapore</div><div class="addr-line">117543</div>, <div class="country">Singapore</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xi+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xi Zhang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tiansu Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biological Sciences</div>, <div class="insititution">National University of Singapore</div>, <div class="addr-line">14 Science Drive 4</div>, <div class="country">Singapore</div><div class="addr-line">117543</div>, <div class="country">Singapore</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tiansu+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tiansu Wang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zhendong Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biological Sciences</div>, <div class="insititution">National University of Singapore</div>, <div class="addr-line">14 Science Drive 4</div>, <div class="country">Singapore</div><div class="addr-line">117543</div>, <div class="country">Singapore</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zhendong+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li Z&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Zhendong Li</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Guijun Guan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biological Sciences</div>, <div class="insititution">National University of Singapore</div>, <div class="addr-line">14 Science Drive 4</div>, <div class="country">Singapore</div><div class="addr-line">117543</div>, <div class="country">Singapore</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Bioresources</div>, <div class="insititution">National Institute for Basic Biology</div>, <div class="addr-line">Okazaki, Aichi 444-8585</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +65 6516-2915. Fax. +65 6779-2486. E-mail: <a href="mailto:dbshyh@nus.edu.sg" target="_blank">dbshyh@nus.edu.sg</a> (Y.H.), <a href="mailto:kan@nibb.ac.jp" target="_blank">kan@nibb.ac.jp</a> (G.G.).
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Guijun+Guan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Guan G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Guan G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Guijun Guan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yunhan Hong</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biological Sciences</div>, <div class="insititution">National University of Singapore</div>, <div class="addr-line">14 Science Drive 4</div>, <div class="country">Singapore</div><div class="addr-line">117543</div>, <div class="country">Singapore</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +65 6516-2915. Fax. +65 6779-2486. E-mail: <a href="mailto:dbshyh@nus.edu.sg" target="_blank">dbshyh@nus.edu.sg</a> (Y.H.), <a href="mailto:kan@nibb.ac.jp" target="_blank">kan@nibb.ac.jp</a> (G.G.).
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yunhan+Hong">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hong Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hong Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yunhan Hong</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 19, Issue 5, 1 October 2012, Pages 423–433, <a href='https://doi.org/10.1093/dnares/dss023'>https://doi.org/10.1093/dnares/dss023</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 October 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">18 June 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">03 September 2012</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="629170" href="/dnaresearch/article-pdf/19/5/423/1753024/dss023.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Jianbin Chen, Xi Zhang, Tiansu Wang, Zhendong Li, Guijun Guan, Yunhan Hong;  Efficient Detection, Quantification and Enrichment of Subtle Allelic Alterations, <em>DNA Research</em>, Volume 19, Issue 5, 1 October 2012, Pages 423–433, <a href='https://doi.org/10.1093/dnares/dss023'>https://doi.org/10.1093/dnares/dss023</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/629170?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/629170?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/629170?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/629170?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/629170?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/629170?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/629170?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Efficient%20Detection%2C%20Quantification%20and%20Enrichment%20of%20Subtle%20Allelic%20Alterations&amp;publicationDate=2012-10-01&amp;volumeNum=19&amp;issueNum=5&amp;author=Chen%2C%20Jianbin%3B%20Zhang%2C%20Xi&amp;startPage=423&amp;endPage=433&amp;contentId=10.1093%2Fdnares%2Fdss023&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f5%2f423%2f629170" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=10646292 id="10646292" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Gene targeting (GT) can introduce subtle alterations into a particular locus and represents a powerful tool for genome editing. Engineered zinc finger nucleases (ZFNs) are effective for generating minor allelic alterations. Efficient detection of such minor alterations remains one of the challenges in ZFN-mediated GT experiments. Here, we report the establishment of procedures allowing for efficient detection, quantification and enrichment of such subtle alterations. In a biallelic model, polyacrylamide gel electrophoresis (PAGE) is capable of detecting rare allelic variations in the form of DNA heteroduplexes at a high efficiency of ∼0.4% compared with ∼6.3% by the traditional T7 endonuclease I-digestion and agarose gel electrophoresis. In a multiple allelic model, PAGE could discriminate different alleles bearing addition or deletion of 1–18 bp as distinct bands that were easily quantifiable by densitometry. Furthermore, PAGE enables enrichment for rare alleles. We show for the first time that direct endogenous GT is possible in medaka by ZFN RNA injection, whereas PAGE allows for detection and cloning of ZFN-targeted alleles in adults arising from ZFN-injected medaka embryos. Therefore, PAGE is effective for detection, quantification and enrichment of multiple fine allelic differences and thus offers a versatile tool for screening targeted subtle gene alterations.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene targeting&quot;">gene targeting</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;heteroduplex analysis&quot;">heteroduplex analysis</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;zinc finger nucleases&quot;">zinc finger nucleases</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcription activator-like effector nucleases&quot;">transcription activator-like effector nucleases</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;polyacrylamide gel electrophoresis&quot;">polyacrylamide gel electrophoresis</a></div>                            <h2 scrollto-destination=10646294 id="10646294" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Genetic modifications at specific sites of animal genomes represent a powerful method for elucidating molecular mechanisms underlying normal and pathological processes.<sup><a reveal-id="DSS023C1" data-open="DSS023C1" class="link link-ref link-reveal xref-bibr">1</a></sup> In mice, gene targeting (GT) via homologous recombination (HR) in embryonic stem (ES) cells followed by the formation of germline chimeras has been a routine tool, leading to the production of numerous knockout animals to explore the gene functions in the particular processes of normal development and disease models.<sup><a reveal-id="DSS023C1" data-open="DSS023C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSS023C2" data-open="DSS023C2" class="link link-ref link-reveal xref-bibr">2</a></sup> This classical approach of knockout production has been limited to mouse and rat,<sup><a reveal-id="DSS023C3" data-open="DSS023C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSS023C4" data-open="DSS023C4" class="link link-ref link-reveal xref-bibr">4</a></sup> mainly due to the availability of pluripotent ES cell lines capable of whole animal production via germline transmission.</p><p>Engineered sequence-specific endonucleases, namely zinc finger nucleases (ZFNs) and transcription activator-like effector nucleases (TALENs), have recently attracted considerable attention,<sup><a reveal-id="DSS023C5" data-open="DSS023C5" class="link link-ref link-reveal xref-bibr">5</a></sup> because they can mediate high efficiencies of GT and thus allow for allelic alterations directly in developing embryos without the intermediate steps such as HR in ES cells, chimera formation from targeted ES cells and germline transmission, which are required in traditional GT experiments. ZFNs have successfully led to the targeted gene disruption (GD) in zebrafish and exogenous GD in medaka.<sup><a reveal-id="DSS023C6" data-open="DSS023C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSS023C7" data-open="DSS023C7" class="link link-ref link-reveal xref-bibr">7</a></sup> More recently, TALENs have also been utilized successfully in zebrafish somatic cell cultures and embryos.<sup><a reveal-id="DSS023C8" data-open="DSS023C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSS023C9" data-open="DSS023C9" class="link link-ref link-reveal xref-bibr">9</a></sup> These endonucleases can induce double-strand DNA breaks to increase the HR frequency, allowing for gene knockin directly in developing embryos.<sup><a reveal-id="DSS023C10" data-open="DSS023C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Recent progresses include the development of engineered zinc finger nickases inducing homology-directed repair with reduced mutagenic effects,<sup><a reveal-id="DSS023C11" data-open="DSS023C11" class="link link-ref link-reveal xref-bibr">11</a></sup> selection-independent generation of gene knockout mouse ES cells using ZFNs<sup><a reveal-id="DSS023C12" data-open="DSS023C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and monomeric site-specific nucleases for genome editing.<sup><a reveal-id="DSS023C13" data-open="DSS023C13" class="link link-ref link-reveal xref-bibr">13</a></sup></p><p>One of the challenges in ZFN- or TALEN-mediated GD experiments is the ability and efficiency to detect the desirable disrupted alleles, which are rare and usually contain minor differences compared with wild-type (WT) alleles,<sup><a reveal-id="DSS023C6" data-open="DSS023C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSS023C8" data-open="DSS023C8" class="link link-ref link-reveal xref-bibr">8</a></sup> because ZFN- or TALEN-mediated GD may occur independently and generate various alterations in different cells and embryos and the individuals derived from them may be highly heterogeneous. These minor allelic alterations may be hardly detectable under the standard agarose gel electrophoresis (AGE). Traditionally, three methods are used to detect GD lesions without any requirement for special equipment. These are phenotype selection,<sup><a reveal-id="DSS023C6" data-open="DSS023C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSS023C14" data-open="DSS023C14" class="link link-ref link-reveal xref-bibr">14</a></sup> restriction site selection<sup><a reveal-id="DSS023C8" data-open="DSS023C8" class="link link-ref link-reveal xref-bibr">8</a></sup> and electrophoresis after enzymatic mismatch cleavage.<sup><a reveal-id="DSS023C15" data-open="DSS023C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSS023C16" data-open="DSS023C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Phenotype selection is not applicable to many genes that do not show an apparent phenotype after disruption. Restriction site selection requires a specific restriction site within the region of detection: upon disruption, a gene or its fragment may lose the recognition site for a restriction enzyme, leading to a change in the restriction pattern. Enzyme mismatch cleavage is a versatile approach that is widely used in mutation and polymorphism detection,<sup><a reveal-id="DSS023C17" data-open="DSS023C17" class="link link-ref link-reveal xref-bibr">17</a></sup> because it enables the detection of any allelic variations independent of the corresponding phenotype and restriction site. Allelic sequences can form homoduplex (Hm) or heteroduplex (Ht) when they are subject to repeated cycles of denaturing and annealing as in PCR. The T4 endonuclease VII and T7 endonuclease I (T7) specifically recognize and cut an Ht DNA at the site of mismatch, generating two shorter DNA fragments detectable on ordinary AGE.<sup><a reveal-id="DSS023C18" data-open="DSS023C18" class="link link-ref link-reveal xref-bibr">18</a></sup> This approach makes use of T7 digestion and AGE, and is thus referred to TAGE subsequently for simplicity.</p><p>Several other approaches of detecting minor sequence differences require the use of special equipment. Capillary electrophoresis<sup><a reveal-id="DSS023C19" data-open="DSS023C19" class="link link-ref link-reveal xref-bibr">19</a></sup> uses particular devices and can identify single-molecule differences.<sup><a reveal-id="DSS023C20" data-open="DSS023C20" class="link link-ref link-reveal xref-bibr">20</a></sup> High-resolution melting analysis<sup><a reveal-id="DSS023C21" data-open="DSS023C21" class="link link-ref link-reveal xref-bibr">21</a></sup> has recently been used to detect chemically induced point mutations in medaka<sup><a reveal-id="DSS023C22" data-open="DSS023C22" class="link link-ref link-reveal xref-bibr">22</a></sup> as well as DNA methylation.<sup><a reveal-id="DSS023C23" data-open="DSS023C23" class="link link-ref link-reveal xref-bibr">23</a></sup> However, specialized instruments designed to monitor fluorescence changes during heating are required.</p><p>Polyacrylamide gel electrophoresis (PAGE) provides high-resolution detection of polymorphisms<sup><a reveal-id="DSS023C24" data-open="DSS023C24" class="link link-ref link-reveal xref-bibr">24</a></sup> and it has also been used to study DNA mismatch repair.<sup><a reveal-id="DSS023C25" data-open="DSS023C25" class="link link-ref link-reveal xref-bibr">25</a></sup> The usefulness of PAGE for detecting allelic alterations in GD experiments remains to be determined. This study was aimed at the development of procedures for the detection of allelic alterations at high efficiency and resolution. In a biallelic model system, TAGE and PAGE were found to be able to detect minor DNA differences at an efficiency of 6 and 0.4%, respectively. In a multi-allelic model system, PAGE but not TAGE allowed for rapid detection, quantification and enrichment of multiple rare alleles. Most importantly, in a real experiment for ZFN-mediated GT through direct microinjection in medaka embryos, PAGE proved its effectiveness to screen for GT at a particular locus.</p>                            <h2 scrollto-destination=10646300 id="10646300" class="section-title" data-legacy-id=s2>2. Materials and Methods</h2>
                            <h3 scrollto-destination=10646302 id="10646302" class="section-title" data-legacy-id=s2a>2.1 Fish and chemicals</h3>
<p>Work with fish followed the guidelines on the care and use of animals for scientific purposes of the National Advisory Committee for Laboratory Animal Research in Singapore and approved by this committee (permit number 27/09). Medaka (strains <em>af</em>, <em>HB32C</em>, <em>HdrR</em>, <em>i1</em>, <em>i3</em> and <em>Orange</em>) was maintained under an artificial photoperiod of 14-h/10-h light/darkness at 26°C as described.<sup><a reveal-id="DSS023C26" data-open="DSS023C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Chemicals and enzymes were from Sigma-aldrich and Promega unless otherwise indicated.</p>                            <h3 scrollto-destination=10646307 id="10646307" class="section-title" data-legacy-id=s2b>2.2 Plasmids</h3>
<p>Plasmid DNA was extracted by using the Qiagen Maxi and Midi preparation kits (Qiagen). WT1 and D18 are 321 and 303 bp in length, respectively, which were PCR-amplified by using primers ATGGTTGAGTCCCAATCTTTTG plus TCAATATCGCTCTGAAACCCAG from two plasmids containing the medaka <em>nanog</em> cDNAs. pWT2 contains a 346-bp insert of the first exon of medaka <em>gsdf</em> gene. The insert was PCR amplified by using primers GGTTCCCAGGCCCCGGGCTAGCAG plus GAAAAGGATCGTTGCAATTCGC and cloned in pGEM-T Easy vector. pD1, pD6, pD7, pA18 and pA3D7 were similar to pWT2 except for deletions (D) or additions (A) of 1, 6, 7 and 18 bp in the insert. These deletions or additions were generated from ZFN-mediated GD in medaka embryos (see below).</p>                            <h3 scrollto-destination=10646309 id="10646309" class="section-title" data-legacy-id=s2c>2.3 Genomic DNA extraction</h3>
<p>Caudal fin clips were collected from growing or adult fish and incubated overnight at 55°C in 1.5-ml Eppendorf tubes containing 100 μl of lysis buffer (100 mM Tris–HCl, pH 8.0, 200 mM NaCl, 0.2% SDS, 5 mM EDTA, pH 8.0 and 100 μg/ml proteinase K). Following heat-inactivation of proteinase K for 10 min at 70°C, DNA was precipitated with 2.5 volume of ethanol and dissolved in 50 μl of TE (10 mM Tris–HCl, 1 mM EDTA, pH 8.0). DNA isolation from whole fish and embryos was performed as described.<sup><a reveal-id="DSS023C27" data-open="DSS023C27" class="link link-ref link-reveal xref-bibr">27</a></sup></p>                            <h3 scrollto-destination=10646311 id="10646311" class="section-title" data-legacy-id=s2d>2.4 Genomic PCR and successive PCR</h3>
<p>Genomic PCR was run in a 20-μl volume containing 1 μl of fin clips DNA, 50 ng of medaka genomic DNA or 50 pg of linearized plasmid DNA in the absence or presence of 50 ng of medaka genomic DNA for 35 cycles (94°C for 15 s, 60°C for 15 s and 72°C for 30 s) with Takara Ex-Taq DNA polymerase and the supplied buffer.</p><p>In the case of consecutive rounds of PCR for enrichment for particular fractions, 1 μl of the gel-purified PCR products was used as template for 30 cycles of PCR at the same conditions.</p>                            <h3 scrollto-destination=10646314 id="10646314" class="section-title" data-legacy-id=s2e>2.5 Heteroduplex formation</h3>
<p>When DNA mixtures containing two or more different sequences are used as templates for PCR, heteroduplexes are formed automatically during cycles of denaturing and re-annealing.<sup><a reveal-id="DSS023C28" data-open="DSS023C28" class="link link-ref link-reveal xref-bibr">28</a></sup> However, to ensure full heteroduplex formation or to generate heteroduplexes between PCR products from two separate reactions, a denaturing and re-annealing procedure was performed. PCR products or mixture of PCR products from separate reactions were heated to 94°C for 3 min and slowly returned to room temperature to form heteroduplexes.<sup><a reveal-id="DSS023C29" data-open="DSS023C29" class="link link-ref link-reveal xref-bibr">29</a></sup></p>                            <h3 scrollto-destination=10646316 id="10646316" class="section-title" data-legacy-id=s2f>2.6 T7 endonuclease I digestion and agarose gel electrophoresis</h3>
<p>For the traditional TAGE detection method, 3 μl of PCR product was subjected to digestion with 0.3 U T7 endonuclease I (New England BioLabs) in a 5-μl volume at 37°C for 1.5 h. Then, 5 μl of digested products, together with 3 μl undigested PCR product as control, were mixed with 5× loading dye, and was loaded on the 2% (w/v) agarose gel with 0.05 μg/ml of ethidium bromide to run the electrophoresis under 130 V for 15 min.</p>                            <h3 scrollto-destination=10646318 id="10646318" class="section-title" data-legacy-id=s2g>2.7 Polyacrylamide gel electrophoresis</h3>
<p>For our PAGE detection method, 4 μl of PCR products were separated on 8% polyacrylamide gels in 1× TBE buffer using a Mini-Protean electrophoresis unit (Bio-Rad Laboratories) at 100 V for 2 h at room temperature. Gels were submerged in 1× TBE buffer containing 0.05 µg/ml of ethidium bromide for 20 min. Gels were documented on a bioimaging system (Vilber Lourmat).</p>                            <h3 scrollto-destination=10646320 id="10646320" class="section-title" data-legacy-id=s2h>2.8 Gel recovery, cloning and sequencing</h3>
<p>DNA bands were cut from the gel under UV light and smashed in 10–20 μl TE. After incubation overnight at room temperature, the supernatant containing DNA was used for successive PCR (see above).</p><p>TA cloning was performed with pGEM-T Easy vectors. DNA mini-preps were prepared according to the standard alkaline lysis protocol<sup><a reveal-id="DSS023C30" data-open="DSS023C30" class="link link-ref link-reveal xref-bibr">30</a></sup> and used for sequencing on the 3130xl Sequencer (Applied Biosystems). Sequence alignment and analysis were done by using Vector NTI and DNAman.</p>                            <h3 scrollto-destination=10646323 id="10646323" class="section-title" data-legacy-id=s2i>2.9 ZFN-mediated GD</h3>
<p>Expression vectors pZN1gsdf and pZN2gsdf (see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data</a>) were supplied commercially (Toolgene, South Korea). They encode a pair of ZFNs that target the first exon of the medaka <em>gsdf</em> gene (unpublished data). After linearization with <em>Apa</em>I, both vectors were used for mRNA synthesis by using the T7 RNA polymerase (Invitrogen) as described.<sup><a reveal-id="DSS023C31" data-open="DSS023C31" class="link link-ref link-reveal xref-bibr">31</a></sup> The mRNAs were mixed at 1:1 ratio and microinjected at 20–100 ng/μl into embryos of <em>af</em> and <em>HdrR</em> medaka at the 1-cell stage as described.<sup><a reveal-id="DSS023C31" data-open="DSS023C31" class="link link-ref link-reveal xref-bibr">31</a></sup> The injected embryos were analysed or allowed to develop into adulthood.</p>                            <h3 scrollto-destination=10646325 id="10646325" class="section-title" data-legacy-id=s2j>2.10 Step-by-step procedure</h3>
<p>We are interested in versatile procedures for use in most ordinary laboratories without specialized equipment. Therefore, we developed a PAGE detection procedure, as is indicated in Fig. <a reveal-id="DSS023F1" data-open="DSS023F1" class="link link-reveal link-table xref-fig">1</a>A, which can be streamlined as follows: </p>                        <a id="10646327" scrollto-destination="10646327"></a>
<div id="DSS023F1" data-id="DSS023F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss023/2/m_dss02301.jpeg?Expires=2147483647&Signature=nZlhIyeDpAAeabZ7i6Gkm6SJq5Eq9dvQbZvyz8Unmcjikl7m~qdUqee0Yy0VtkAk8V1rRUB5gWOhbwNH95LYXFwR4Fo4IxijAthDB0d8WIvrovgsp4ASycLL8DeUseF6V3lRTdBqtSbHij~Wum2bhhHTILO9hnvHrtNB0tkRdl95JI28GgOR-nV9bKmQ3M2wH~zRzbsZ-wYeLYA-tPPkhpoXFQJXKN2ey-FcpeWj-KL-ginpVUFxytO2yQkuKnrOhnnXjLk14C67QYtE~rLCQvnh~18QttgelsjCHOOePx1G77YjwRMb65pc93oCmtsou9apC5fDMF3XU~LsQUu7Xw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Heteroduplex production and detection by TAGE and PAGE. (A) Flowchart showing heteroduplex production by PCR and heteroduplex detection by PAGE and TAGE. (B and C) Biallelic heteroduplex model assay. Plasmids containing a WT allele (WT1) and a mutant allele with an 18-bp deletion (D18) were mixed with indicated dilution factor for PCR. (B) TAGE detection. D18 with a dilution factor of 16 can be detected. (C) PAGE detection. D18 with a dilution factor of 256 can be detected. Size markers in base pairs are indicated to the left. Arrowheads denote a double band. Asterisks depict heteroduplexes. Hm, homoduplex; Ht, heteroduplex; PAGE, polyacrylamide gel electrophoresis; T7, T7 endonuclease I; TAGE, T7-digestion and agarose gel electrophoresis. This figure appears in colour in the online version of DNA Research." path-from-xml="dss02301.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10646327/dss02301.jpeg" path-from-xml="dss02301.jpeg" target="_blank">View large</a><a section="10646327" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss023/2/dss02301.jpeg?Expires=2147483647&Signature=SX4SkrYD4wvYxgtcOK5OEndeqKuFVD62ig4oFFhTnWoRRUzvBG8WEbqRf1YR4rjN-IIMQVCru5awhSt3NgIM4rjtGF1CHOVzPQkd5cdaaReRpNTIOIGHmLJf6JOkSJGNWardeLRkFP5Hga3xGpN56Eu2n-xEIaFhzSF9JH0FhOQwuCyYdJj11omkDJnfaEkrHeTfSk7Bzc~Tog4jCjv93M47BJwR-ORH2xSUdRCtynpxeOVGa8UgKnN8CuaRisoAciIMYfbtkPOi9WrX0GMGALb~zJluwT9L9zx6kIHYIzcSSZW5Q8mzrVJ1-UWXtfVrgWL-PbFWnu6dg-atcdSMwQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10646327&ar=629170&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02301.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Heteroduplex production and detection by TAGE and PAGE. (<strong>A</strong>) Flowchart showing heteroduplex production by PCR and heteroduplex detection by PAGE and TAGE. (<strong>B</strong> and <strong>C</strong>) Biallelic heteroduplex model assay. Plasmids containing a WT allele (WT1) and a mutant allele with an 18-bp deletion (D18) were mixed with indicated dilution factor for PCR. (<strong>B</strong>) TAGE detection. D18 with a dilution factor of 16 can be detected. (<strong>C</strong>) PAGE detection. D18 with a dilution factor of 256 can be detected. Size markers in base pairs are indicated to the left. Arrowheads denote a double band. Asterisks depict heteroduplexes. Hm, homoduplex; Ht, heteroduplex; PAGE, polyacrylamide gel electrophoresis; T7, T7 endonuclease I; TAGE, T7-digestion and agarose gel electrophoresis. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><div content-id="DSS023F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss023/2/m_dss02301.jpeg?Expires=2147483647&Signature=nZlhIyeDpAAeabZ7i6Gkm6SJq5Eq9dvQbZvyz8Unmcjikl7m~qdUqee0Yy0VtkAk8V1rRUB5gWOhbwNH95LYXFwR4Fo4IxijAthDB0d8WIvrovgsp4ASycLL8DeUseF6V3lRTdBqtSbHij~Wum2bhhHTILO9hnvHrtNB0tkRdl95JI28GgOR-nV9bKmQ3M2wH~zRzbsZ-wYeLYA-tPPkhpoXFQJXKN2ey-FcpeWj-KL-ginpVUFxytO2yQkuKnrOhnnXjLk14C67QYtE~rLCQvnh~18QttgelsjCHOOePx1G77YjwRMb65pc93oCmtsou9apC5fDMF3XU~LsQUu7Xw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Heteroduplex production and detection by TAGE and PAGE. (A) Flowchart showing heteroduplex production by PCR and heteroduplex detection by PAGE and TAGE. (B and C) Biallelic heteroduplex model assay. Plasmids containing a WT allele (WT1) and a mutant allele with an 18-bp deletion (D18) were mixed with indicated dilution factor for PCR. (B) TAGE detection. D18 with a dilution factor of 16 can be detected. (C) PAGE detection. D18 with a dilution factor of 256 can be detected. Size markers in base pairs are indicated to the left. Arrowheads denote a double band. Asterisks depict heteroduplexes. Hm, homoduplex; Ht, heteroduplex; PAGE, polyacrylamide gel electrophoresis; T7, T7 endonuclease I; TAGE, T7-digestion and agarose gel electrophoresis. This figure appears in colour in the online version of DNA Research." path-from-xml="dss02301.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10646327/dss02301.jpeg" path-from-xml="dss02301.jpeg" target="_blank">View large</a><a section="10646327" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss023/2/dss02301.jpeg?Expires=2147483647&Signature=SX4SkrYD4wvYxgtcOK5OEndeqKuFVD62ig4oFFhTnWoRRUzvBG8WEbqRf1YR4rjN-IIMQVCru5awhSt3NgIM4rjtGF1CHOVzPQkd5cdaaReRpNTIOIGHmLJf6JOkSJGNWardeLRkFP5Hga3xGpN56Eu2n-xEIaFhzSF9JH0FhOQwuCyYdJj11omkDJnfaEkrHeTfSk7Bzc~Tog4jCjv93M47BJwR-ORH2xSUdRCtynpxeOVGa8UgKnN8CuaRisoAciIMYfbtkPOi9WrX0GMGALb~zJluwT9L9zx6kIHYIzcSSZW5Q8mzrVJ1-UWXtfVrgWL-PbFWnu6dg-atcdSMwQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10646327&ar=629170&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02301.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Heteroduplex production and detection by TAGE and PAGE. (<strong>A</strong>) Flowchart showing heteroduplex production by PCR and heteroduplex detection by PAGE and TAGE. (<strong>B</strong> and <strong>C</strong>) Biallelic heteroduplex model assay. Plasmids containing a WT allele (WT1) and a mutant allele with an 18-bp deletion (D18) were mixed with indicated dilution factor for PCR. (<strong>B</strong>) TAGE detection. D18 with a dilution factor of 16 can be detected. (<strong>C</strong>) PAGE detection. D18 with a dilution factor of 256 can be detected. Size markers in base pairs are indicated to the left. Arrowheads denote a double band. Asterisks depict heteroduplexes. Hm, homoduplex; Ht, heteroduplex; PAGE, polyacrylamide gel electrophoresis; T7, T7 endonuclease I; TAGE, T7-digestion and agarose gel electrophoresis. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><ol class="order"><li><p>DNA extraction: extract and purify genomic DNA from gene-disrupted individuals.</p></li><li><p>Primers design: design a pair of primers flanking the ZFN/TALEN cleavage site to yield PCR products of ∼300 bp.</p></li><li><p>Genomic PCR: perform PCR with the DNA samples, one WT sample as negative control and one mixture of mutant and WT alleles as positive control if available.</p></li><li><p>Heteroduplex formation: <ul class="alpha-lower"><li><p>for heterozygous founder individuals containing disrupted alleles, heteroduplexes are formed during PCR. No additional step is required;</p></li><li><p>for progenies that might be homozygous, each PCR product should be mixed with WT PCR product with a ratio of ∼1:1, heated to 94°C for 3 min and slowly returned to room temperature to form heteroduplexes.</p></li></ul></p></li><li><p>PAGE detection: <ul class="alpha-lower"><li><p>prepare 8% polyacrylamide gels with 1× TBE buffer;</p></li><li><p>4–6 μl of PCR product is loaded with 6× loading dye to each well of the gel. Electrophoresis is carried out at 12 V/cm for ≥2 h;</p></li><li><p>the gels are stained with 0.05 µg/ml ethidium bromide for 20 min and documented on a bio-imaging system.</p></li></ul></p></li><li><p>Allelic alteration identification: bands of the sample are compared with those of the negative control. The presence of different bands from the control indicates allelic alterations. If positive control shows no heteroduplex band, experiment should be redone.</p></li><li><p>Repeat Steps 3–6 to reduce or eliminate false-positive samples.</p></li><li><p>Gel recovery, cloning and sequencing: <ul class="alpha-lower"><li><p>bands of heteroduplexes are recovered by smashing and soaking in 10–20 μl of TE in 37°C for overnight;<sup><a reveal-id="DSS023C32" data-open="DSS023C32" class="link link-ref link-reveal xref-bibr">32</a></sup></p></li><li><p>1 μl of the recovered DNA are used as templates for one round of successive PCR of 30 cycles;</p></li><li><p>the products are TA cloned into pGEM-T Easy vector. At least 10 colonies are picked to do mini-preps to collect plasmid DNA using the standard alkaline–SDS procedure;</p></li><li><p>sequencing is run on the insert of the plasmid DNA and the result is compared with the WT allele to validate allelic variations.</p></li></ul></p></li></ol>                            <h2 scrollto-destination=10646329 id="10646329" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=10646330 id="10646330" class="section-title" data-legacy-id=s3a>3.1 Rationale and experimental design</h3>
<p>Following the procedure above, after repeated PCR cycles of denaturing and annealing, genomic DNA containing gene-targeted alleles will generate three types of PCR products, namely, wild-type (WT) homoduplex (Hm), mutant (MT) Hm and WT–MT heteroduplex (Ht). These PCR products do not show a visible difference on AGE. In TAGE, Ht DNA fragments are cut by T7 at the mismatch site and become distinguishable from the uncut Hm molecules, which can be discriminated as distinct bands on AGE.</p><p>Alternatively, on native PAGE, such PCR products can be directly separated due to the differences in configuration and length. Since Ht shift slower than Hm on PAGE, Ht bands can be found clearly on the top of Hm bands, as an unambiguous indicator of successful GD. Moreover, after PAGE, all PCR products remain intact. Accordingly, Ht DNA fragments are recoverable from the gel for cloning and sequencing, or can be further re-amplified by consecutive rounds of PCR before cloning and sequencing. Therefore, PAGE features high resolution of separation and the integrity of DNA fragments, which should permit separation and quantification of each PCR product, and more importantly, selective enrichment of Ht DNA for analysis. Its quantification ability leads to an estimation of GD proportion in a sample, providing information for deciding the sample size for screening by cloning and sequencing. Its enrichment ability reduces the WT alleles by eliminating Hm fractions, facilitating the identification of GD-containing samples via reducing the workload of cloning and sequencing in each sample.</p>                            <h3 scrollto-destination=10646333 id="10646333" class="section-title" data-legacy-id=s3b>3.2 Biallelic model</h3>
<p>The procedures and the usefulness of TAGE and PAGE were first established and evaluated for Ht detection in a biallelic model system. To this end, two plasmids, WT1 and D18, which contain an insert of 321 and 303 bp as the WT allele and 18-bp deletion allele (thus D18) of the medaka <em>nanog</em> gene respectively, were linearized and mixed to form a series of dilution (D18 diluted by WT1 at a factor of 2–256). The serial dilution mixtures were used as the templates for PCR and PCR products were subjected to TAGE and PAGE detection.</p><p>On TAGE, Hm WT1 and D18 fragments remained intact and appeared as bands that were slightly different in size (Fig. 1B). When mixed at a 1:1 ratio, WT1 and D18 without T7 cleavage produced a double band and, upon T7 cleavage, formed two distinct bands with a similar intensity. Serial dilution revealed that D18 remained detectable at a dilution factor of 16, producing a sensitivity level of ∼6.3% for TAGE in Ht detection (Fig. <a reveal-id="DSS023F1" data-open="DSS023F1" class="link link-reveal link-table xref-fig">1</a>B). In consistence with the principle of TAGE, T7 digestion leads to reduced band intensities. These observations reveal the feasibility and sensitivity of the established TAGE procedure for Ht detection. On the other hand, in the PAGE detection procedure, D18 remained detectable even at a dilution factor of 256 (Fig. <a reveal-id="DSS023F1" data-open="DSS023F1" class="link link-reveal link-table xref-fig">1</a>C). This produces a sensitivity level of ∼0.4% for PAGE in Ht detection, which is 16-fold more efficient than TAGE.</p><p>A closer inspection revealed several advantageous features of PAGE procedure for detecting allelic modifications (Fig. <a reveal-id="DSS023F1" data-open="DSS023F1" class="link link-reveal link-table xref-fig">1</a>C). WT1 and D18 Hm fragments formed distinct bands of sufficient difference in size. Moreover, Ht fragments migrated at a much lower rate than Hm fragments and formed bands with a clear distance from Hm DNAs. As illustrated in Fig. <a reveal-id="DSS023F1" data-open="DSS023F1" class="link link-reveal link-table xref-fig">1</a>A, two alleles will form two different configurations of Ht fragments depending on the involvement of top or bottom strands of WT or MT alleles.<sup><a reveal-id="DSS023C33" data-open="DSS023C33" class="link link-ref link-reveal xref-bibr">33</a></sup> In our biallelic model, WT1 and D18 formed two configurations, which were resolved as two distinct bands on PAGE (Fig. <a reveal-id="DSS023F1" data-open="DSS023F1" class="link link-reveal link-table xref-fig">1</a>C). Consequently, PCR products from a biallelic DNA template give rise to four distinct bands, compared with two visible bands in TAGE (compare Fig. <a reveal-id="DSS023F1" data-open="DSS023F1" class="link link-reveal link-table xref-fig">1</a>B and C). In addition, Ht fragments on TAGE formed a faint band that was embedded in a smear at a dilution factor of 16 (Fig. <a reveal-id="DSS023F1" data-open="DSS023F1" class="link link-reveal link-table xref-fig">1</a>B), compared with clear bands with little background even at a dilution factor of 256 (Fig. <a reveal-id="DSS023F1" data-open="DSS023F1" class="link link-reveal link-table xref-fig">1</a>C). Most importantly, upon increasingly serial dilution, the Hm fragment of D18 as the rare allele became invisible but contributed to Ht formation, thus increasing the detection sensitivity.</p><p>To mimic the real situation of detecting allelic alterations in a targeted genome, the serially diluted mixtures of linearized WT1 and D18 (cDNA original) were added to medaka genomic DNA and subjected to analyses by PCR and PAGE. As illustrated in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>, addition of <em>af</em> genomic DNA did not interfere with the formation of Ht fragments and their detection by PAGE, because the band pattern and detectable dilution factor remained unchanged with genomic DNA (compare Fig. <a reveal-id="DSS023F1" data-open="DSS023F1" class="link link-reveal link-table xref-fig">1</a>C with <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). Taken together, in a biallelic model system, TAGE is able to detect a subtle allelic alteration at a limited resolution and sensitivity of 6.3%, whereas PAGE offers high resolution and sensitivity of 0.4% in the absence and presence of genomic DNA.</p><p>It needs to be noted that, allelic differences, either natural polymorphisms or experimentally induced alterations, will generate Ht PCR products and Ht bands on PAGE. To understand the impact of sequence polymorphisms on our PAGE procedure, we tested the PCR products of <em>gsdf</em> locus from two medaka inbred lines along with our gene targeted fish 4 (see below) in three lengths: 572, 372 and 305 bp. Primers used are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>. The result indicates that PCR products of <em>af</em> sample exhibit more natural polymorphism Ht bands (asterisks) than <em>HdrR</em> in size of 572 and 372 bp (Supplementary Fig. S2C). Following the PAGE recovery procedure provided above, we sequenced the asterisked Ht and discover a 10 bp deletion polymorphism in <em>af</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2B</a>). By shortening the PCR product size to bypass the polymorphism site (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2B</a>), <em>af</em> shows only faint background bands. Meanwhile, fish 4 PCR products of both sizes exhibit additional bands, which are generated from GT. Therefore, a WT sample serving as a negative control is required in every batch of our PAGE detection.</p>                            <h3 scrollto-destination=10646339 id="10646339" class="section-title" data-legacy-id=s3c>3.3 Multi-allelic model system</h3>
<p>In the practice of ZFN-mediated GT experiments, additions and deletions may occur independently at different stages of development and in different cells, leading to the production of various different MT alleles. We furthered our experiments to establish and utilize a multi-allelic model system for the detection by PAGE. To this end, a set of five different alleles were produced within exon 1 of the medaka <em>gsdf</em> gene via ZFN-mediated GT in medaka embryos (see below). They were WT2, A18, D7, D6 and D1 (Fig. <a reveal-id="DSS023F2" data-open="DSS023F2" class="link link-reveal link-table xref-fig">2</a>A). WT2 of 320 bp in length is the WT allele of <em>gsdf</em> (WT1 described above is the WT allele of the medaka <em>nanog</em>). These allelic sequences were cloned in pGEM-T Easy, and linearized plasmids were mixed at various ratios as described in Fig. <a reveal-id="DSS023F2" data-open="DSS023F2" class="link link-reveal link-table xref-fig">2</a>B. As expected, PCR produced a single band on PAGE when DNA containing a single plasmid was used as template (lanes 1–5, Fig. <a reveal-id="DSS023F2" data-open="DSS023F2" class="link link-reveal link-table xref-fig">2</a>B), while allelic variations, as tiny as 1-bp difference, were clearly distinguishable on PAGE as Hm and Ht forms (lanes 6, 7, 9, 11, Fig. <a reveal-id="DSS023F2" data-open="DSS023F2" class="link link-reveal link-table xref-fig">2</a>B). Notably, PAGE is capable of simultaneous detection of multiple alleles as distinct bands (lanes 8, 10, 12, 13, Fig. <a reveal-id="DSS023F2" data-open="DSS023F2" class="link link-reveal link-table xref-fig">2</a>B). To confirm that the Ht bands are not from artificial non-specific PCR products, PCR products manifesting single band in lanes 1–5 of Fig. <a reveal-id="DSS023F2" data-open="DSS023F2" class="link link-reveal link-table xref-fig">2</a>B were mixed at the same ratio as indicated in Fig. <a reveal-id="DSS023F2" data-open="DSS023F2" class="link link-reveal link-table xref-fig">2</a>B and then were subjected to a heteroduplex formation procedure. PAGE profiles (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>) of these mixtures show exactly the same patterns as in Fig. <a reveal-id="DSS023F2" data-open="DSS023F2" class="link link-reveal link-table xref-fig">2</a>B. </p>                        <a id="10646341" scrollto-destination="10646341"></a>
<div id="DSS023F2" data-id="DSS023F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss023/2/m_dss02302.jpeg?Expires=2147483647&Signature=kiG5FAJT5vkIKhcjqnpbJ1iYB-q9mhZHfP~UsuPGhHgUE4-giDVR75YAljN2UjwxPGfjNE7pA63-51Gm53NWf82nS02~wEHH02IMq07LxfL3TmL31FCONLWaTl1oYbhQVdjzITfp5pnYCaMi8RmwPXYX1HP~QsnyXge2RE1uC~ZmNiTki-oD3Fqld-XBtmffOUYL13FEGmm5gTwONTbydQhTfwR7nfijcavCesbzzUjD4E29kGtzi2egE8zJBo4l-9cH5cawSqZQTPjFttO7IvEYE6XtKgy6j1JCKqPTEdFBwnYMdvXHfV7sOXousoDEnstV1r7-zhRDy52x54NX5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="PAGE detection of multiple alleles. (A) Partial sequences of five different alleles of the medaka gsdf gene. Additions are shown as a loop. Deletions are depicted by gaps. Bold letters indicate the cleavage site of ZFN. (B) PAGE profiles of different mixtures of the alleles. Mix ratios of the alleles are indicated above lanes. Dashed lines delineate the same heteroduplexes cross lanes. Asterisks highlight heteroduplexes. (C) Correlation between the number of alleles and total number of different PCR products which may show different bands on PAGE. Colour background depicts the positive correlation between the concentrations of each DNA strands and the corresponding band intensities. Capital letters, upper strands; small capital case letters, lower strands; underlining, homoduplexes; Hm, homoduplex; Ht, heteroduplex. This figure appears in colour in the online version of DNA Research." path-from-xml="dss02302.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10646341/dss02302.jpeg" path-from-xml="dss02302.jpeg" target="_blank">View large</a><a section="10646341" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss023/2/dss02302.jpeg?Expires=2147483647&Signature=U1ueI2MdNjBPaLDfgPVN8xTHhuSLCX7aL57un62P3xnZcTcTra-OL7nkR3QH0hhb~ZCBdkxRIYWiJlvOe8wqwcoIWFIf2h40q4kj7pN4KsNsN8Xge93TMW~2kN79tpuUPA6veXetnu5afB1jGK8KRHWX0d05KO~lNLQVTtBrc09TGeFw4f19C5xMgLll595B0xzsWeu1PPLfktcylPBk54BDFH67theOIYx58mYXHPOD8C6dw7ihWYJliIKZL3isWYJ6Ze0npCWmJ7IUIvwg4Bwo6WTDqfJpgA~hOeH9Vv8fTdxwTwAUX1piYugPyTCXvp-YPw55NjkW4p7mix5xYw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10646341&ar=629170&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02302.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>PAGE detection of multiple alleles. (<strong>A</strong>) Partial sequences of five different alleles of the medaka <em>gsdf</em> gene. Additions are shown as a loop. Deletions are depicted by gaps. Bold letters indicate the cleavage site of ZFN. (<strong>B</strong>) PAGE profiles of different mixtures of the alleles. Mix ratios of the alleles are indicated above lanes. Dashed lines delineate the same heteroduplexes cross lanes. Asterisks highlight heteroduplexes. (<strong>C</strong>) Correlation between the number of alleles and total number of different PCR products which may show different bands on PAGE. Colour background depicts the positive correlation between the concentrations of each DNA strands and the corresponding band intensities. Capital letters, upper strands; small capital case letters, lower strands; underlining, homoduplexes; Hm, homoduplex; Ht, heteroduplex. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><div content-id="DSS023F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss023/2/m_dss02302.jpeg?Expires=2147483647&Signature=kiG5FAJT5vkIKhcjqnpbJ1iYB-q9mhZHfP~UsuPGhHgUE4-giDVR75YAljN2UjwxPGfjNE7pA63-51Gm53NWf82nS02~wEHH02IMq07LxfL3TmL31FCONLWaTl1oYbhQVdjzITfp5pnYCaMi8RmwPXYX1HP~QsnyXge2RE1uC~ZmNiTki-oD3Fqld-XBtmffOUYL13FEGmm5gTwONTbydQhTfwR7nfijcavCesbzzUjD4E29kGtzi2egE8zJBo4l-9cH5cawSqZQTPjFttO7IvEYE6XtKgy6j1JCKqPTEdFBwnYMdvXHfV7sOXousoDEnstV1r7-zhRDy52x54NX5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="PAGE detection of multiple alleles. (A) Partial sequences of five different alleles of the medaka gsdf gene. Additions are shown as a loop. Deletions are depicted by gaps. Bold letters indicate the cleavage site of ZFN. (B) PAGE profiles of different mixtures of the alleles. Mix ratios of the alleles are indicated above lanes. Dashed lines delineate the same heteroduplexes cross lanes. Asterisks highlight heteroduplexes. (C) Correlation between the number of alleles and total number of different PCR products which may show different bands on PAGE. Colour background depicts the positive correlation between the concentrations of each DNA strands and the corresponding band intensities. Capital letters, upper strands; small capital case letters, lower strands; underlining, homoduplexes; Hm, homoduplex; Ht, heteroduplex. This figure appears in colour in the online version of DNA Research." path-from-xml="dss02302.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10646341/dss02302.jpeg" path-from-xml="dss02302.jpeg" target="_blank">View large</a><a section="10646341" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss023/2/dss02302.jpeg?Expires=2147483647&Signature=U1ueI2MdNjBPaLDfgPVN8xTHhuSLCX7aL57un62P3xnZcTcTra-OL7nkR3QH0hhb~ZCBdkxRIYWiJlvOe8wqwcoIWFIf2h40q4kj7pN4KsNsN8Xge93TMW~2kN79tpuUPA6veXetnu5afB1jGK8KRHWX0d05KO~lNLQVTtBrc09TGeFw4f19C5xMgLll595B0xzsWeu1PPLfktcylPBk54BDFH67theOIYx58mYXHPOD8C6dw7ihWYJliIKZL3isWYJ6Ze0npCWmJ7IUIvwg4Bwo6WTDqfJpgA~hOeH9Vv8fTdxwTwAUX1piYugPyTCXvp-YPw55NjkW4p7mix5xYw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10646341&ar=629170&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02302.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>PAGE detection of multiple alleles. (<strong>A</strong>) Partial sequences of five different alleles of the medaka <em>gsdf</em> gene. Additions are shown as a loop. Deletions are depicted by gaps. Bold letters indicate the cleavage site of ZFN. (<strong>B</strong>) PAGE profiles of different mixtures of the alleles. Mix ratios of the alleles are indicated above lanes. Dashed lines delineate the same heteroduplexes cross lanes. Asterisks highlight heteroduplexes. (<strong>C</strong>) Correlation between the number of alleles and total number of different PCR products which may show different bands on PAGE. Colour background depicts the positive correlation between the concentrations of each DNA strands and the corresponding band intensities. Capital letters, upper strands; small capital case letters, lower strands; underlining, homoduplexes; Hm, homoduplex; Ht, heteroduplex. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><div class="block-child-p">A closer inspection established a correlation between the total number of bands and the total number of alleles (Fig. <a reveal-id="DSS023F2" data-open="DSS023F2" class="link link-reveal link-table xref-fig">2</a>B and C): a single allele produces one PCR product (lanes 1–5), two alleles result in four products (lanes 6, 7, 9 and 11), and three alleles give rise to nine products which form six to eight distinctive bands on PAGE depending on the relative intensity and inter-band distance (lanes 8 and 10). This correlation is formulated as follows: <a id="DSS023M1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSS023M1"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dss023eq1.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss023/2/m_dss023eq1.gif?Expires=2147483647&amp;Signature=j-u~NPVH0xyXzG8x0nphlnJc~MTb4gsiUxmvAqS0ZA7KRC9eQ~Bdr3xELf-dFHvPfOgjjBiZIGz8Ihj-Ojyw5Iv-A3WBYKaLltCpDnlqJWFs7HuckHdzFii6CBYqBWjr7V5XvueI5~CWHE7wwriNlHweGo9dSRLsSVPIG0F-ZVZXngfEOBLFqGKPSEIPygknAkyDkNb-CnlkbimL4Umf0Q9MhYW5i4OiMdzzBz03ZRAt58pPyse3yOHnc5t1PaVN9QFwGGpsvuxC6QSC2ewwjvOHq-EUat39pa7y7alK26xSHZB3UKPdnOYJ-GL5kQZVHTjDJh6-KMP18UqUKUrtzg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dss023eq1.gif" /></a></div></div><div class="label">(1)</div></div><a id="DSS023M2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSS023M2"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dss023eq2.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss023/2/m_dss023eq2.gif?Expires=2147483647&amp;Signature=iKdCq9bTc4dKoxOAB9JNqTZxqwXFqjg0BydEjmTk~2OrG0KpXTGlMQRHv~V3gIc1w96BIkAEG7tPu9W-dHG6X~ZRh2i0AIzyoBBOGRtWjn8QxEIlK8zA6ZMa1wzyYITDPmBAZqOWKjLog~9lOq79W4ds18V3iJa3PkkCpvGlKRW4~kux3oQqUMOdY5hQdyWYe4a3a-j4FNeHVTxkzdMuTV3QQ9p6TKBu9JH3l-vKINhISJceW0xLFdIzJfM76I~hkGDKRBCiImROOneBm7dHDXN~B6Iw-Eyc6UEwv97DUUaMetboOzaj-mVoiDyldsNmnkfGbwuf1ChQHT4OkQj5dA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dss023eq2.gif" /></a></div></div><div class="label">(2)</div></div> where <em>N</em> is the number of different alleles and <em>B</em> is the number of Ht bands. Since values are integers in our case, for shorting the calculation, Equation (2) can be simplified with a ceiling function which leads to the smallest following integer: <a id="DSS023M3" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSS023M3"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dss023eq3.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss023/2/m_dss023eq3.gif?Expires=2147483647&amp;Signature=fbcCDSq8WqlWhfMw931Eeo~hWlVN68N9j2cG5M-JHkK4Z2AeJ3TRKLFCiXCepBRv2qsv22i5Av0EIPJTnFJoFWWNNrcLaFWTt30y8fi6OL-zgM17Zt7Y6pIQo3N78i8CbGVafX3yYlyLYvPA6T~THxmYPQCqCICWDBbtYGieoUyzjyZ98Ml5ABE7jvQBZjsOJbYeIVArvQ4srFJ5VThGJYRAms4oh6uy9Egn~gLpj8DuC87pCdnc0aWHTGjrXvhKv4BSey~jf7XC01HxuoUTz1KtX0ZvOBNelfUSU6kgXRfDRJ99ICvwIauj-ChAcATUq2Rubq7iIh0rLj4Kh42TMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dss023eq3.gif" /></a></div></div><div class="label">(3)</div></div> Therefore, the total number of alleles in a sample can be easily estimated by Ht band number without solving an equation. This trait of PAGE allows us to intuitively observe the main composition of the sample.</div><p>In case of multiple alleles, the number of Hm PCR products is the same as the number of alleles (Fig. <a reveal-id="DSS023F2" data-open="DSS023F2" class="link link-reveal link-table xref-fig">2</a>C, underlined), and the remainder is Ht PCR products (Fig. <a reveal-id="DSS023F2" data-open="DSS023F2" class="link link-reveal link-table xref-fig">2</a>C, not underlined). Abundances of these Ht products show a positive correlation with the corresponding allelic abundance as depicted by the colour gradient. Therefore, not all Ht bands are visible, and the number of Ht bands on PAGE will indicate the minimal number of allelic alterations in a sample.</p>                            <h3 scrollto-destination=10646344 id="10646344" class="section-title" data-legacy-id=s3d>3.4 Enrichment of rare alleles</h3>
<p>A final proof for targeted alterations in a ZFN-treated sample comes from sequence analysis. ZFN-mediated GT has been reported to occur in 1–3% cells of zebrafish embryos,<sup><a reveal-id="DSS023C34" data-open="DSS023C34" class="link link-ref link-reveal xref-bibr">34</a>,<a reveal-id="DSS023C35" data-open="DSS023C35" class="link link-ref link-reveal xref-bibr">35</a></sup> where only one gene copy is targeted in most cases. Consequently, the targeted allele is usually present at ∼1% in total genomic DNA from a positive sample. Accordingly, the validation of GT is a time-consuming task in a ZFN-mediated GT experiment, because it requires preparing and sequencing several hundreds of recombinant colonies.<sup><a reveal-id="DSS023C34" data-open="DSS023C34" class="link link-ref link-reveal xref-bibr">34</a>,<a reveal-id="DSS023C35" data-open="DSS023C35" class="link link-ref link-reveal xref-bibr">35</a></sup> The high resolution and band distinctness of intact PCR products on PAGE provoked us to develop a procedure of enriching for rare alleles, which is gel recovery and successive PCR (grsPCR). In grsPCR, Ht bands of first-round PCR are gel recovered, and DNA is released by diffusion from gel slices for use as template for successive rounds of PCR. To test this procedure, a DNA mixture of four alleles at defined ratio (column 1, Fig. <a reveal-id="DSS023F3" data-open="DSS023F3" class="link link-reveal link-table xref-fig">3</a>C) was used for first round of PCR; Ht fractions were gel recovered collectively, and DNA released was used for second and third rounds of PCR (Fig. <a reveal-id="DSS023F3" data-open="DSS023F3" class="link link-reveal link-table xref-fig">3</a>A), and PCR products were quantified by densitometry (Fig. <a reveal-id="DSS023F3" data-open="DSS023F3" class="link link-reveal link-table xref-fig">3</a>B) and cloned for validation by sequencing (Fig. <a reveal-id="DSS023F3" data-open="DSS023F3" class="link link-reveal link-table xref-fig">3</a>C). Upon two to three rounds of PCR, an increase in Ht fractions was indeed apparent on both PAGE and TAGE when compared with the Hm fractions (Fig. <a reveal-id="DSS023F3" data-open="DSS023F3" class="link link-reveal link-table xref-fig">3</a>A). To quantify a change in intensity during grsPCR, various bands were defined and measured by densitometry (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4A and B</a>). According to the band patterns of this four-allelic model system as shown in Fig. <a reveal-id="DSS023F2" data-open="DSS023F2" class="link link-reveal link-table xref-fig">2</a>B, major bands of the four alleles are identifiable, which permits intensity quantifications of the four alleles by combining two bands of each allele (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4C</a>). In this way, the intensity of dominant WT allele was found to be reduced considerably from 62.0% in the first PCR over 36.0–26.3%, while intensities of rare alleles increased after one and two rounds of grsPCR (Fig. <a reveal-id="DSS023F3" data-open="DSS023F3" class="link link-reveal link-table xref-fig">3</a>B and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4C</a>). Cloning and sequencing led to a similar enrichment factor (Fig. <a reveal-id="DSS023F3" data-open="DSS023F3" class="link link-reveal link-table xref-fig">3</a>C). After one single round of grsPCR, for example, the WT allele decreased from 79 to 55%, and rare alleles increased relatively. This verifies quantitative enrichment for rare alleles. Taken together, PAGE profile enables the quantification of each component, while grsPCR allows for the enrichment of multiple rare alleles. </p>                        <a id="10646346" scrollto-destination="10646346"></a>
<div id="DSS023F3" data-id="DSS023F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss023/2/m_dss02303.jpeg?Expires=2147483647&Signature=ACvzK2VAzwtivOPw4CDtsQU35gabcu2yWPtygG~B0xBgXA2OsilKmkP~SCQggpQfdTmcYdMRH7btbi5BnKXkGa8R-uPwuO-u74IfNYWgXFa5I~af~mXFl6cu8kijpQAxqoriNcy8yZM34wW8ooT7K0aFAbyIowmVkxOvYsfmKb9XihoWbKWnuc1ZrAiltKHHxaJZCKpukhNjloKT-VfajJSEDf5opSJk3lleahaUeXTBj0hGRv6BLEHvf7JWwO2fObEG342Au3mJAS9A0URuXxojdBDrupc2q83Fl56o1rzAd2GaBViM8NglL4~39r7Tco8oprGcAF3NBeeHlPpOkQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="PAGE detection, enrichment and quantification of multiple alleles. (A) PAGE and TAGE profiles of multiple alleles after successive rounds of PCR following gel recovery (dash frame). DNA mixture containing four alleles shown in lane 12 of Fig. 2B was used as template with mix ratios indicated in (C). (B) Relative intensity of different bands of each round. Round ‘0’ represent the initial percentages of each allele. (C) The number of clones and percentages of each allele from PCR products of each round. This figure appears in colour in the online version of DNA Research." path-from-xml="dss02303.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10646346/dss02303.jpeg" path-from-xml="dss02303.jpeg" target="_blank">View large</a><a section="10646346" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss023/2/dss02303.jpeg?Expires=2147483647&Signature=cCVnEmB~d4jszUWVG50BD~0wl72HbFU85eyetjrm0GUd-uoLY2Q7qv6ohf6GeB4aQIRJlPrrgnuJNsm3yYfQOPw7cRlOEv2Gl3Xk5OQ1123Vu3Lp~sHfHvbTVdTnJNxBXxlid3G6IM3snpznLi2Kni1JBAkUS9uzP8T4~ZsHjv2f-ipxInec~hCa-QqaSztvdAKT1HREktckJWCDZVmGFYSepyhbK08aBG5jYL4MDsBfublnhzKbxtY8O9JvpicAfSG70zG0shqIRoLCb0cQhHix5yIWQ9nu0EoMHVykA13nk7qI7kZTL3Uvir0f9FO1sPdYJjVwSpYtsXgZK4h8jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10646346&ar=629170&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02303.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>PAGE detection, enrichment and quantification of multiple alleles. (<strong>A</strong>) PAGE and TAGE profiles of multiple alleles after successive rounds of PCR following gel recovery (dash frame). DNA mixture containing four alleles shown in lane 12 of Fig. <a reveal-id="DSS023F2" data-open="DSS023F2" class="link link-reveal link-table xref-fig">2</a>B was used as template with mix ratios indicated in (<strong>C</strong>). (<strong>B</strong>) Relative intensity of different bands of each round. Round ‘0’ represent the initial percentages of each allele. (<strong>C</strong>) The number of clones and percentages of each allele from PCR products of each round. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><div content-id="DSS023F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss023/2/m_dss02303.jpeg?Expires=2147483647&Signature=ACvzK2VAzwtivOPw4CDtsQU35gabcu2yWPtygG~B0xBgXA2OsilKmkP~SCQggpQfdTmcYdMRH7btbi5BnKXkGa8R-uPwuO-u74IfNYWgXFa5I~af~mXFl6cu8kijpQAxqoriNcy8yZM34wW8ooT7K0aFAbyIowmVkxOvYsfmKb9XihoWbKWnuc1ZrAiltKHHxaJZCKpukhNjloKT-VfajJSEDf5opSJk3lleahaUeXTBj0hGRv6BLEHvf7JWwO2fObEG342Au3mJAS9A0URuXxojdBDrupc2q83Fl56o1rzAd2GaBViM8NglL4~39r7Tco8oprGcAF3NBeeHlPpOkQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="PAGE detection, enrichment and quantification of multiple alleles. (A) PAGE and TAGE profiles of multiple alleles after successive rounds of PCR following gel recovery (dash frame). DNA mixture containing four alleles shown in lane 12 of Fig. 2B was used as template with mix ratios indicated in (C). (B) Relative intensity of different bands of each round. Round ‘0’ represent the initial percentages of each allele. (C) The number of clones and percentages of each allele from PCR products of each round. This figure appears in colour in the online version of DNA Research." path-from-xml="dss02303.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10646346/dss02303.jpeg" path-from-xml="dss02303.jpeg" target="_blank">View large</a><a section="10646346" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss023/2/dss02303.jpeg?Expires=2147483647&Signature=cCVnEmB~d4jszUWVG50BD~0wl72HbFU85eyetjrm0GUd-uoLY2Q7qv6ohf6GeB4aQIRJlPrrgnuJNsm3yYfQOPw7cRlOEv2Gl3Xk5OQ1123Vu3Lp~sHfHvbTVdTnJNxBXxlid3G6IM3snpznLi2Kni1JBAkUS9uzP8T4~ZsHjv2f-ipxInec~hCa-QqaSztvdAKT1HREktckJWCDZVmGFYSepyhbK08aBG5jYL4MDsBfublnhzKbxtY8O9JvpicAfSG70zG0shqIRoLCb0cQhHix5yIWQ9nu0EoMHVykA13nk7qI7kZTL3Uvir0f9FO1sPdYJjVwSpYtsXgZK4h8jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10646346&ar=629170&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02303.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>PAGE detection, enrichment and quantification of multiple alleles. (<strong>A</strong>) PAGE and TAGE profiles of multiple alleles after successive rounds of PCR following gel recovery (dash frame). DNA mixture containing four alleles shown in lane 12 of Fig. <a reveal-id="DSS023F2" data-open="DSS023F2" class="link link-reveal link-table xref-fig">2</a>B was used as template with mix ratios indicated in (<strong>C</strong>). (<strong>B</strong>) Relative intensity of different bands of each round. Round ‘0’ represent the initial percentages of each allele. (<strong>C</strong>) The number of clones and percentages of each allele from PCR products of each round. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div>                            <h3 scrollto-destination=10646347 id="10646347" class="section-title" data-legacy-id=s3e>3.5 Experimental ZFN-mediated GT system</h3>
<p>The experiments described so far dealt with model systems by using linearized plasmids mixed at defined ratios as templates for PCR and analyses via TAGE and PAGE. We wanted to demonstrate the usefulness of PAGE in a real experiment of direct GT in embryos of a vertebrate. We chose the ZFN approach and medaka as the model organism. Medaka is a laboratory fish and a lower vertebrate model for stem cell biology<sup><a reveal-id="DSS023C27" data-open="DSS023C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DSS023C36" data-open="DSS023C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DSS023C37" data-open="DSS023C37" class="link link-ref link-reveal xref-bibr">37</a></sup> and germ cell biology.<sup><a reveal-id="DSS023C31" data-open="DSS023C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSS023C38" data-open="DSS023C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Its genome has been sequenced (<a class="link link-uri" href="http://www.ensembl.org/index.html">http://www.ensembl.org/index.html</a>). There are many inbred lines/strains (<a class="link link-uri" href="http://biol1.bio.nagoya-u.ac.jp:8000/">http://biol1.bio.nagoya-u.ac.jp:8000/</a>) and experimental tools including embryo microinjection.<sup><a reveal-id="DSS023C31" data-open="DSS023C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSS023C39" data-open="DSS023C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DSS023C40" data-open="DSS023C40" class="link link-ref link-reveal xref-bibr">40</a></sup></p><p>We are interested in the development of approaches for direct GT by using ZFNs and TALENs in medaka. By microinjection of RNAs for a pair ZFNs into embryos of <em>af</em> medaka, we obtained several targeted alleles at the medaka <em>gsdf</em>, a gene whose RNA expression is spatially and temporally correlated with early testicular differentiation.<sup><a reveal-id="DSS023C41" data-open="DSS023C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Four of the alleles were used in the multi-allelic model assay described above (Figs <a reveal-id="DSS023F2" data-open="DSS023F2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSS023F3" data-open="DSS023F3" class="link link-reveal link-table xref-fig">3</a>). More than hundred of fish derived from ZFN-injected embryos were grown into adulthood. Genomic DNA was extracted from fin clips of a subset of samples and subjected to TAGE and PAGE assays. We detected successful GT in 2 out of 48 fish examined, because the two fish (4 and 6) exhibited clear bands of Ht PCR products on PAGE (Fig. <a reveal-id="DSS023F4" data-open="DSS023F4" class="link link-reveal link-table xref-fig">4</a>A), and a smear or faint band indicative of Ht products on TAGE (Fig. <a reveal-id="DSS023F4" data-open="DSS023F4" class="link link-reveal link-table xref-fig">4</a>B). The band patterns of two fish samples on PAGE and TAGE were similar to that observed in model systems (Figs <a reveal-id="DSS023F1" data-open="DSS023F1" class="link link-reveal link-table xref-fig">1</a> and <a reveal-id="DSS023F2" data-open="DSS023F2" class="link link-reveal link-table xref-fig">2</a>). Sequencing 72 recombinant colonies of the Ht PCR products from the third-round PCR (a total of two rounds of grsPCR) revealed three different alleles. One is the WT allele that was present in 45 clones (63%), and the remainder is targeted alleles D7 (22%) and A3D7 (15%; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>). Interestingly, allele A3D7 has both a 3-bp addition and a 7-bp deletion, and more intriguingly, also four mismatches of 1–2 bp each at three separate positions. It follows that ZFNs can simultaneously introduce addition, deletion and mutation into one and same targeted allele. Hence, ZFN-mediated endogenous GT is possible in medaka embryos, and PAGE is effective in detecting and cloning ZFN-targeted rare alleles in fish. </p>                        <a id="10646350" scrollto-destination="10646350"></a>
<div id="DSS023F4" data-id="DSS023F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss023/2/m_dss02304.jpeg?Expires=2147483647&Signature=ZakuTWiXgUZGqX8-sN-~KiNWgT5p~WCPWMRaJ2hbn2ll9~LQvS68JEKjvX4SQk6bpliMPyyKGfnrWh2noAj9qPJkUkeR25Co2nlfYC00U8BYhQbh-pq7Yrldp8yS6Wdip9hW0UrVo2MOTMyICDBbznlWjHf0aBG~Ymtl7s5An3hEhq-4cUPvEoVpJDBMqpg2OoVgcRoLLuQ4NLcOITF5v7XiMM38OR6uTdaG9OBZx4GvAwAaMTa-n6DB1DPlZqXKVTWrs67mbZWvyrGFZ6EJJtOQaLkVUAJeH2jLPENR4gCkf9y7rRqE4Iw3TgspXkSXTBvM4kRvSiLGpP4gnD9~GA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of ZFN-mediated allelic alterations in medaka. Medaka embryos were microinjected with RNAs for a pair of ZFNs that targeted the first exon of gsdf locus, and were allowed to develop into adult animals. Genomic DNA was extracted from fin clips of eight randomly sampled adults and was subjected to PCR and PAGE detection of targeted allelic alterations. (A) PAGE profiles of PCR products, unambiguously showing the presence of heteroduplex DNA (asterisks) in two out of eight fish. (B) TAGE profile of PCR products, showing a faint band of heteroduplex DNA (asterisks) within smears in the same two fish." path-from-xml="dss02304.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10646350/dss02304.jpeg" path-from-xml="dss02304.jpeg" target="_blank">View large</a><a section="10646350" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss023/2/dss02304.jpeg?Expires=2147483647&Signature=JQX~FR2OAhU~SJ69cLyy-yKx4xfgyacExV74mPU08vylEt7I6wWKTf2bYrCsbtmQe3xje7oXvmfi55QCTdF7MET5QlTBnMlMQd4hjYJGJmP3fQav0Nbb6zkxbLrZKasmU3wfJYZwUbIvx5LNVjKFfMssVQ7hy~6Fsk~RreepyhoqTNopuE6V45dUrN9Rmh4mZbxfBgCkFxc2k3B4TD41CB6LpNtrg3Mv9YNcbDSpmlns0GIsJutVXibXDEWUk~1aTxUDHQsVJMljM1fmB~TsCHQoAuHerJE-MaAhnVmK0qzTsJMbAi2GY8UemhghULCT3FzQjssXxwULfRLFh441EA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10646350&ar=629170&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02304.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of ZFN-mediated allelic alterations in medaka. Medaka embryos were microinjected with RNAs for a pair of ZFNs that targeted the first exon of <em>gsdf</em> locus, and were allowed to develop into adult animals. Genomic DNA was extracted from fin clips of eight randomly sampled adults and was subjected to PCR and PAGE detection of targeted allelic alterations. (<strong>A</strong>) PAGE profiles of PCR products, unambiguously showing the presence of heteroduplex DNA (asterisks) in two out of eight fish. (<strong>B</strong>) TAGE profile of PCR products, showing a faint band of heteroduplex DNA (asterisks) within smears in the same two fish.</p></div></div><div content-id="DSS023F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss023/2/m_dss02304.jpeg?Expires=2147483647&Signature=ZakuTWiXgUZGqX8-sN-~KiNWgT5p~WCPWMRaJ2hbn2ll9~LQvS68JEKjvX4SQk6bpliMPyyKGfnrWh2noAj9qPJkUkeR25Co2nlfYC00U8BYhQbh-pq7Yrldp8yS6Wdip9hW0UrVo2MOTMyICDBbznlWjHf0aBG~Ymtl7s5An3hEhq-4cUPvEoVpJDBMqpg2OoVgcRoLLuQ4NLcOITF5v7XiMM38OR6uTdaG9OBZx4GvAwAaMTa-n6DB1DPlZqXKVTWrs67mbZWvyrGFZ6EJJtOQaLkVUAJeH2jLPENR4gCkf9y7rRqE4Iw3TgspXkSXTBvM4kRvSiLGpP4gnD9~GA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of ZFN-mediated allelic alterations in medaka. Medaka embryos were microinjected with RNAs for a pair of ZFNs that targeted the first exon of gsdf locus, and were allowed to develop into adult animals. Genomic DNA was extracted from fin clips of eight randomly sampled adults and was subjected to PCR and PAGE detection of targeted allelic alterations. (A) PAGE profiles of PCR products, unambiguously showing the presence of heteroduplex DNA (asterisks) in two out of eight fish. (B) TAGE profile of PCR products, showing a faint band of heteroduplex DNA (asterisks) within smears in the same two fish." path-from-xml="dss02304.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10646350/dss02304.jpeg" path-from-xml="dss02304.jpeg" target="_blank">View large</a><a section="10646350" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss023/2/dss02304.jpeg?Expires=2147483647&Signature=JQX~FR2OAhU~SJ69cLyy-yKx4xfgyacExV74mPU08vylEt7I6wWKTf2bYrCsbtmQe3xje7oXvmfi55QCTdF7MET5QlTBnMlMQd4hjYJGJmP3fQav0Nbb6zkxbLrZKasmU3wfJYZwUbIvx5LNVjKFfMssVQ7hy~6Fsk~RreepyhoqTNopuE6V45dUrN9Rmh4mZbxfBgCkFxc2k3B4TD41CB6LpNtrg3Mv9YNcbDSpmlns0GIsJutVXibXDEWUk~1aTxUDHQsVJMljM1fmB~TsCHQoAuHerJE-MaAhnVmK0qzTsJMbAi2GY8UemhghULCT3FzQjssXxwULfRLFh441EA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10646350&ar=629170&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02304.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of ZFN-mediated allelic alterations in medaka. Medaka embryos were microinjected with RNAs for a pair of ZFNs that targeted the first exon of <em>gsdf</em> locus, and were allowed to develop into adult animals. Genomic DNA was extracted from fin clips of eight randomly sampled adults and was subjected to PCR and PAGE detection of targeted allelic alterations. (<strong>A</strong>) PAGE profiles of PCR products, unambiguously showing the presence of heteroduplex DNA (asterisks) in two out of eight fish. (<strong>B</strong>) TAGE profile of PCR products, showing a faint band of heteroduplex DNA (asterisks) within smears in the same two fish.</p></div></div>                            <h2 scrollto-destination=10646351 id="10646351" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>The introduction of engineered ZFNs<sup><a reveal-id="DSS023C6" data-open="DSS023C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSS023C42" data-open="DSS023C42" class="link link-ref link-reveal xref-bibr">42</a></sup> and TALENs<sup><a reveal-id="DSS023C8" data-open="DSS023C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSS023C9" data-open="DSS023C9" class="link link-ref link-reveal xref-bibr">9</a></sup> has made it possible to achieve GT directly in developing animal embryos and selection-independent GT in mouse ES cells.<sup><a reveal-id="DSS023C12" data-open="DSS023C12" class="link link-ref link-reveal xref-bibr">12</a></sup> The ability to efficiently identify subtle allelic alterations is essential in these ZFN- and TALEN-mediated GT experiments. In this study, we have developed TAGE and PAGE in model systems and a real GT experiment as versatile procedures for detecting allelic variations in ordinary laboratories without specialized equipment and devices. Under our conditions, TAGE identifies the digests of mismatch-containing Ht fragment as a clear band when the rare allele is present ≥6.3% in a biallelic model system, but as a faint band within a smear in DNA containing ZFN-altered alleles. These results demonstrate the ability and limit of TAGE in allelic detection, as has well be documented.<sup><a reveal-id="DSS023C17" data-open="DSS023C17" class="link link-ref link-reveal xref-bibr">17</a></sup></p><p>We showed that PAGE is of choice because of its five major advantages over TAGE. First, PAGE is straightforward and does not require any additional step of enzymatic treatment of PCR products, facilitating the cloning of wanted PCR products by gel recovery. Secondly, PAGE is capable of detecting rare alleles as low as 0.4%, sensitivity of which is 16 times higher than TAGE. Thirdly, PAGE provides an extremely high resolution, as desirable allelic alterations are present in heteroduplexes that form distinct bands with large distances from the bulk WT homoduplex, which enables the simultaneous identification and quantification of multiple different alterations. Fourthly, PAGE allows for discrimination between Ht from natural polymorphisms and those introduced by GD. Finally, on the basis of high resolution and band distinctness of intact PCR products on PAGE, we have developed a gel recovery and successive PCR procedure, which, for the first time to our knowledge, enables enrichment for rare alleles. The percentage of the rare alleles could reach 37% in our experiment (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>); hence, only dozens of recombinant colony sequencing are needed to detect the desirable rare alleles, considerably reducing the work load compared with previous studies when hundreds of colonies were sequenced.<sup><a reveal-id="DSS023C34" data-open="DSS023C34" class="link link-ref link-reveal xref-bibr">34</a>,<a reveal-id="DSS023C35" data-open="DSS023C35" class="link link-ref link-reveal xref-bibr">35</a></sup></p><p>PAGE has been used for heteroduplex analysis for years with well-established model systems.<sup><a reveal-id="DSS023C28" data-open="DSS023C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSS023C29" data-open="DSS023C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSS023C33" data-open="DSS023C33" class="link link-ref link-reveal xref-bibr">33</a>,<a reveal-id="DSS023C43" data-open="DSS023C43" class="link link-ref link-reveal xref-bibr">43</a></sup> Based on these previous reports and our experience, for PAGE detection, we suggest that the design of PCR primers amplifying fragments around 300 bp, which can save the time of electrophoresis and avoid polymorphisms while containing enough sequence information. Natural polymorphisms will exhibit Ht bands, thus it is necessary to add one sample of the WT allele in every batch of our PAGE detection as negative control. Only those showing different patterns or additional bands compared with the WT sample can be deemed as positive ones. This represents one of the merits of the PAGE method because in the T7 detection method, polymorphisms cannot be distinguished from introduced mutation since they show similar shortened bands. Also the PAGE method provides a tool to survey the site of interest for natural polymorphisms, revealing the suitability of each animal population for such GT experiments.</p><p>As for the detection for progenies of GD individuals, which might contain homologous disrupted sequence, a heteroduplex formation procedure after mixing with WT PCR product is necessary to avoid omission. In addition, our protocol also suggests repeating the detection for positive samples to rule out false-positive ones.</p><p>Since PAGE is only a quick screening method, we might overlook certain one-base-deletion or point mutation in our sample. For point mutation, previous research using ‘hydrolink mutation detection enhancement gels’ cannot detect all types of the mutation via heteroduplex analysis.<sup><a reveal-id="DSS023C33" data-open="DSS023C33" class="link link-ref link-reveal xref-bibr">33</a></sup> However, almost all published cases of deletion or addition occur in ZFN/TALEN-mediated GT show disruption with more than 1 bp.<sup><a reveal-id="DSS023C6  DSS023C8" data-open="DSS023C6  DSS023C8" class="link link-ref link-reveal xref-bibr">6–8</a>,<a reveal-id="DSS023C34" data-open="DSS023C34" class="link link-ref link-reveal xref-bibr">34</a>,<a reveal-id="DSS023C35" data-open="DSS023C35" class="link link-ref link-reveal xref-bibr">35</a>,<a reveal-id="DSS023C42" data-open="DSS023C42" class="link link-ref link-reveal xref-bibr">42</a></sup> These large lesions can be detected by our PAGE method clearly.</p><p>Our first trial of ZFN-mediated GT in medaka has led to the production and identification of several different alleles from a limited number of embryos and growing animals, revealing the functionality of ZFNs used and the efficiency of this ZFN-mediated GT approach in this organism. It needs to be noted that ZFNs are capable of simultaneous introduction of addition, deletion and point mutation in a single allele, pointing the proficiency of ZFNs in producing a wide variety of targeted allelic alterations in medaka.</p><p>Although developed for analysing experimentally induced minor sequence differences, our procedures, PAGE in particular, will find a wide variety of applications in detection, quantification and enrichment of polymorphisms and alterations in basic and applied research as well as medicine.</p>                            <h2 scrollto-destination=10646359 id="10646359" class="section-title" data-legacy-id=s5>Authors’ contributions</h2>
<p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p>                            <h2 scrollto-destination=10646361 id="10646361" class="section-title" data-legacy-id=s6>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=10646363 id="10646363" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This work was supported by the National Research Foundation of Singapore under its Competitive Research Program (NRF-CRP7-2010-03), the Grants-in-Aid from Japan Society for the Promotion of Science (50113428) and the Biomedical Research Council of Singapore (R154-000-427-305). We acknowledge NUS scholarship to J. B. Chen, X. Zhang and T. S. Wang.</p>                            <h2 scrollto-destination=10646365 id="10646365" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Jiaorong Deng for fish breeding, Foong Choy Mei and Wong Kway Yip Veronica for laboratory management.</p>                            <h2 scrollto-destination=10646367 id="10646367" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSS023C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Capecchi</div> <div class="given-names">M.R.</div></div>. </div><div class="article-title">Gene targeting in mice: functional analysis of the mammalian genome for the twenty-first century</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">507</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20targeting%20in%20mice%3A%20functional%20analysis%20of%20the%20mammalian%20genome%20for%20the%20twenty-first%20century&amp;author=M.R.%20Capecchi&amp;publication_year=2005&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=6&amp;pages=507-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1619" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1619" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1619"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15931173" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">R.H.</div></div>,  <div class="name"><div class="surname">Sengupta</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Impaired DNA damage response, genome instability, and tumorigenesis in SIRT1 mutant mice</div>, <div class="source">Cancer Cell</div> , <div class="year">2008</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">312</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Impaired%20DNA%20damage%20response%2C%20genome%20instability%2C%20and%20tumorigenesis%20in%20SIRT1%20mutant%20mice&amp;author=R.H.%20Wang&amp;author=K.%20Sengupta&amp;author=C.%20Li&amp;publication_year=2008&amp;journal=Cancer%20Cell&amp;volume=14&amp;pages=312-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ccr.2008.09.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ccr.2008.09.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ccr.2008.09.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18835033" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Collins</div> <div class="given-names">F.S.</div></div>,  <div class="name"><div class="surname">Rossant</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wurst</div> <div class="given-names">W.</div></div>. </div><div class="article-title">A mouse for all reasons</div>, <div class="source">Cell</div> , <div class="year">2007</div>, vol. <div class="volume">128</div> (pg. <div class="fpage">9</div>-<div class="lpage">13</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20mouse%20for%20all%20reasons&amp;author=F.S.%20Collins&amp;author=J.%20Rossant&amp;author=W.%20Wurst&amp;publication_year=2007&amp;journal=Cell&amp;volume=128&amp;pages=9-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2006.12.018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2006.12.018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2006.12.018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17218247" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tong</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">N.L.</div></div>,  <div class="name"><div class="surname">Yan</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ying</div> <div class="given-names">Q.-L.</div></div>. </div><div class="article-title">Production of p53 gene knockout rats by homologous recombination in embryonic stem cells</div>, <div class="source">Nature</div> , <div class="year">2010</div>, vol. <div class="volume">467</div> (pg. <div class="fpage">211</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Production%20of%20p53%20gene%20knockout%20rats%20by%20homologous%20recombination%20in%20embryonic%20stem%20cells&amp;author=C.%20Tong&amp;author=P.%20Li&amp;author=N.L.%20Wu&amp;author=Y.%20Yan&amp;author=Q.-L.%20Ying&amp;publication_year=2010&amp;journal=Nature&amp;volume=467&amp;pages=211-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature09368" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature09368" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature09368"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20703227" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Clark</div> <div class="given-names">K.J.</div></div>,  <div class="name"><div class="surname">Voytas</div> <div class="given-names">D.F.</div></div>,  <div class="name"><div class="surname">Ekker</div> <div class="given-names">S.C.</div></div>. </div><div class="article-title">A TALE of two nucleases: gene targeting for the masses?</div>, <div class="source">Zebrafish</div> , <div class="year">2011</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">147</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20TALE%20of%20two%20nucleases%3A%20gene%20targeting%20for%20the%20masses%3F&amp;author=K.J.%20Clark&amp;author=D.F.%20Voytas&amp;author=S.C.%20Ekker&amp;publication_year=2011&amp;journal=Zebrafish&amp;volume=8&amp;pages=147-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/zeb.2011.9993" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fzeb.2011.9993" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2Fzeb.2011.9993"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21929364" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Doyon</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">McCammon</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">J.C.</div></div>, et al. </div><div class="article-title">Heritable targeted gene disruption in zebrafish using designed zinc-finger nucleases</div>, <div class="source">Nat. Biotechol.</div> , <div class="year">2008</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">702</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Heritable%20targeted%20gene%20disruption%20in%20zebrafish%20using%20designed%20zinc-finger%20nucleases&amp;author=Y.%20Doyon&amp;author=J.M.%20McCammon&amp;author=J.C.%20Miller&amp;publication_year=2008&amp;journal=Nat.%20Biotechol.&amp;volume=26&amp;pages=702-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1409" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1409" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1409"> </span></div> </div></div></div></div></div><div content-id="DSS023C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ansai</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ochiai</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kanie</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Targeted disruption of exogenous EGFP gene in medaka using zinc-finger nucleases</div>, <div class="source">Dev. Growth Differ.</div> , <div class="year">2012</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">546</div>-<div class="lpage">56</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20disruption%20of%20exogenous%20EGFP%20gene%20in%20medaka%20using%20zinc-finger%20nucleases&amp;author=S.%20Ansai&amp;author=H.%20Ochiai&amp;author=Y.%20Kanie&amp;publication_year=2012&amp;journal=Dev.%20Growth%20Differ.&amp;volume=54&amp;pages=546-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/dgd.2012.54.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fdgd.2012.54.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fdgd.2012.54.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22642582" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Xiao</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Heritable gene targeting in zebrafish using customized TALENs</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2011</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">699</div>-<div class="lpage">700</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Heritable%20gene%20targeting%20in%20zebrafish%20using%20customized%20TALENs&amp;author=P.%20Huang&amp;author=A.%20Xiao&amp;author=M.%20Zhou&amp;author=Z.%20Zhu&amp;author=S.%20Lin&amp;author=B.%20Zhang&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol.&amp;volume=29&amp;pages=699-700" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1939" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1939" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1939"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21822242" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sander</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Cade</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Khayter</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Targeted gene disruption in somatic zebrafish cells using engineered TALENs</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2011</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">697</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20gene%20disruption%20in%20somatic%20zebrafish%20cells%20using%20engineered%20TALENs&amp;author=J.D.%20Sander&amp;author=L.%20Cade&amp;author=C.%20Khayter&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol.&amp;volume=29&amp;pages=697-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1934" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1934" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1934"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21822241" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Johnson</div> <div class="given-names">R.D.</div></div>,  <div class="name"><div class="surname">Jasin</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Double-strand-break-induced homologous recombination in mammalian cells</div>, <div class="source">Biochem. Soc. Trans.</div> , <div class="year">2001</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">196</div>-<div class="lpage">201</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Double-strand-break-induced%20homologous%20recombination%20in%20mammalian%20cells&amp;author=R.D.%20Johnson&amp;author=M.%20Jasin&amp;publication_year=2001&amp;journal=Biochem.%20Soc.%20Trans.&amp;volume=29&amp;pages=196-201" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/bst0290196" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2Fbst0290196" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2Fbst0290196"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11356153" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ramirez</div> <div class="given-names">C.L.</div></div>,  <div class="name"><div class="surname">Certo</div> <div class="given-names">M.T.</div></div>,  <div class="name"><div class="surname">Mussolino</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Engineered zinc finger nickases induce homology-directed repair with reduced mutagenic effects</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2012</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">5560</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Engineered%20zinc%20finger%20nickases%20induce%20homology-directed%20repair%20with%20reduced%20mutagenic%20effects&amp;author=C.L.%20Ramirez&amp;author=M.T.%20Certo&amp;author=C.%20Mussolino&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=5560-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks179" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks179" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks179"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22373919" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Osiak</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Radecke</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Guhl</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Selection-independent generation of gene knockout mouse embryonic stem cells using zinc-finger nucleases</div>, <div class="source">PLoS One</div> , <div class="year">2012</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e28911</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selection-independent%20generation%20of%20gene%20knockout%20mouse%20embryonic%20stem%20cells%20using%20zinc-finger%20nucleases&amp;author=A.%20Osiak&amp;author=F.%20Radecke&amp;author=E.%20Guhl&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e28911" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0028911" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0028911" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0028911"> </span></div> </div></div></div></div></div><div content-id="DSS023C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kleinstiver</div> <div class="given-names">B.P.</div></div>,  <div class="name"><div class="surname">Wolfs</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Kolaczyk</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Roberts</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Hu</div> <div class="given-names">S.X.</div></div>,  <div class="name"><div class="surname">Edgell</div> <div class="given-names">D.R.</div></div>. </div><div class="article-title">Monomeric site-specific nucleases for genome editing</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2012</div>, vol. <div class="volume">109</div> (pg. <div class="fpage">8061</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Monomeric%20site-specific%20nucleases%20for%20genome%20editing&amp;author=B.P.%20Kleinstiver&amp;author=J.M.%20Wolfs&amp;author=T.%20Kolaczyk&amp;author=A.K.%20Roberts&amp;author=S.X.%20Hu&amp;author=D.R.%20Edgell&amp;publication_year=2012&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=109&amp;pages=8061-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1117984109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1117984109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1117984109"> </span></div> </div></div></div></div></div><div content-id="DSS023C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takasu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kobayashi</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Beumer</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Targeted mutagenesis in the silkworm <em>Bombyx mori</em> using zinc finger nuclease mRNA injection</div>, <div class="source">Insect Biochem. Mol. Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">759</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20mutagenesis%20in%20the%20silkworm%20Bombyx%20mori%20using%20zinc%20finger%20nuclease%20mRNA%20injection&amp;author=Y.%20Takasu&amp;author=I.%20Kobayashi&amp;author=K.%20Beumer&amp;publication_year=2010&amp;journal=Insect%20Biochem.%20Mol.%20Biol.&amp;volume=40&amp;pages=759-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ibmb.2010.07.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ibmb.2010.07.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ibmb.2010.07.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20692340" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">H.J.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">H.J.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Cho</div> <div class="given-names">S.W.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">J.S.</div></div>. </div><div class="article-title">Targeted genome editing in human cells with zinc finger nucleases constructed via modular assembly</div>, <div class="source">Genome Res.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">1279</div>-<div class="lpage">88</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20genome%20editing%20in%20human%20cells%20with%20zinc%20finger%20nucleases%20constructed%20via%20modular%20assembly&amp;author=H.J.%20Kim&amp;author=H.J.%20Lee&amp;author=H.%20Kim&amp;author=S.W.%20Cho&amp;author=J.S.%20Kim&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=1279-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.089417.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.089417.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.089417.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19470664" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Maeder</div> <div class="given-names">M.L.</div></div>,  <div class="name"><div class="surname">Thibodeau-Beganny</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Osiak</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Rapid ‘open-source’ engineering of customized zinc-finger nucleases for highly efficient gene modification</div>, <div class="source">Mol. Cell</div> , <div class="year">2008</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">294</div>-<div class="lpage">301</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20%E2%80%98open-source%E2%80%99%20engineering%20of%20customized%20zinc-finger%20nucleases%20for%20highly%20efficient%20gene%20modification&amp;author=M.L.%20Maeder&amp;author=S.%20Thibodeau-Beganny&amp;author=A.%20Osiak&amp;publication_year=2008&amp;journal=Mol.%20Cell&amp;volume=31&amp;pages=294-301" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2008.06.016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2008.06.016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2008.06.016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18657511" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Babon</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">McKenzie</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Cotton</div> <div class="given-names">R.</div></div>. </div><div class="article-title">The use of resolvases T4 endonuclease VII and T7 endonuclease I in mutation detection</div>, <div class="source">Mol. Biotechnol.</div> , <div class="year">2003</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">73</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20use%20of%20resolvases%20T4%20endonuclease%20VII%20and%20T7%20endonuclease%20I%20in%20mutation%20detection&amp;author=J.%20Babon&amp;author=M.%20McKenzie&amp;author=R.%20Cotton&amp;publication_year=2003&amp;journal=Mol.%20Biotechnol.&amp;volume=23&amp;pages=73-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1385/MB:23:1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1385%2FMB:23:1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1385%2FMB:23:1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12611271" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Young</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Dong</div> <div class="given-names">Q.</div></div>. </div><div class="article-title">Two-step total gene synthesis method</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> pg. <div class="fpage">e59</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two-step%20total%20gene%20synthesis%20method&amp;author=L.%20Young&amp;author=Q.%20Dong&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=e59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gnh058" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgnh058" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgnh058"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15087491" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kozlowski</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Krzyzosiak</div> <div class="given-names">W.J.</div></div>. </div><div class="article-title">Combined SSCP/duplex analysis by capillary electrophoresis for more efficient mutation detection</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2001</div>, vol. <div class="volume">29</div> pg. <div class="fpage">e71</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Combined%20SSCP%2Fduplex%20analysis%20by%20capillary%20electrophoresis%20for%20more%20efficient%20mutation%20detection&amp;author=P.%20Kozlowski&amp;author=W.J.%20Krzyzosiak&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res.&amp;volume=29&amp;pages=e71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/29.14.e71" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.14.e71" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F29.14.e71"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11452040" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lagally</div> <div class="given-names">E.T.</div></div>,  <div class="name"><div class="surname">Medintz</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Mathies</div> <div class="given-names">R.A.</div></div>. </div><div class="article-title">Single-molecule DNA amplification and analysis in an integrated microfluidic device</div>, <div class="source">Anal. Chem.</div> , <div class="year">2001</div>, vol. <div class="volume">73</div> (pg. <div class="fpage">565</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single-molecule%20DNA%20amplification%20and%20analysis%20in%20an%20integrated%20microfluidic%20device&amp;author=E.T.%20Lagally&amp;author=I.%20Medintz&amp;author=R.A.%20Mathies&amp;publication_year=2001&amp;journal=Anal.%20Chem.&amp;volume=73&amp;pages=565-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/ac001026b" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fac001026b" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fac001026b"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11217764" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wittwer</div> <div class="given-names">C.T.</div></div>,  <div class="name"><div class="surname">Reed</div> <div class="given-names">G.H.</div></div>,  <div class="name"><div class="surname">Gundry</div> <div class="given-names">C.N.</div></div>,  <div class="name"><div class="surname">Vandersteen</div> <div class="given-names">J.G.</div></div>,  <div class="name"><div class="surname">Pryor</div> <div class="given-names">R.J.</div></div>. </div><div class="article-title">High-resolution genotyping by amplicon melting analysis using LCGreen</div>, <div class="source">Clin. Chem.</div> , <div class="year">2003</div>, vol. <div class="volume">49</div> (pg. <div class="fpage">853</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-resolution%20genotyping%20by%20amplicon%20melting%20analysis%20using%20LCGreen&amp;author=C.T.%20Wittwer&amp;author=G.H.%20Reed&amp;author=C.N.%20Gundry&amp;author=J.G.%20Vandersteen&amp;author=R.J.%20Pryor&amp;publication_year=2003&amp;journal=Clin.%20Chem.&amp;volume=49&amp;pages=853-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1373/49.6.853" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1373%2F49.6.853" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1373%2F49.6.853"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12765979" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ishikawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kamei</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Otozai</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">High-resolution melting curve analysis for rapid detection of mutations in a Medaka TILLING library</div>, <div class="source">BMC Mol. Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">70</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-resolution%20melting%20curve%20analysis%20for%20rapid%20detection%20of%20mutations%20in%20a%20Medaka%20TILLING%20library&amp;author=T.%20Ishikawa&amp;author=Y.%20Kamei&amp;author=S.%20Otozai&amp;publication_year=2010&amp;journal=BMC%20Mol.%20Biol.&amp;volume=11&amp;pages=70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2199-11-70" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2199-11-70" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2199-11-70"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20840787" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lopez</div> <div class="given-names">C.M.R.</div></div>,  <div class="name"><div class="surname">Asenjo</div> <div class="given-names">B.G.</div></div>,  <div class="name"><div class="surname">Lloyd</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Wilkinson</div> <div class="given-names">M.J.</div></div>. </div><div class="article-title">Direct detection and quantification of methylation in nucleic acid sequences using high-resolution melting analysis</div>, <div class="source">Anal. Chem.</div> , <div class="year">2010</div>, vol. <div class="volume">82</div> (pg. <div class="fpage">9100</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Direct%20detection%20and%20quantification%20of%20methylation%20in%20nucleic%20acid%20sequences%20using%20high-resolution%20melting%20analysis&amp;author=C.M.R.%20Lopez&amp;author=B.G.%20Asenjo&amp;author=A.J.%20Lloyd&amp;author=M.J.%20Wilkinson&amp;publication_year=2010&amp;journal=Anal.%20Chem.&amp;volume=82&amp;pages=9100-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/ac1024057" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fac1024057" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fac1024057"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20945868" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Walton</div> <div class="given-names">K.E.</div></div>,  <div class="name"><div class="surname">Styer</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Gruenstein</div> <div class="given-names">E.I.</div></div>. </div><div class="article-title">Genetic polymorphism in normal human fibroblasts as analyzed by two-dimensional polyacrylamide gel electrophoresis</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1979</div>, vol. <div class="volume">254</div> (pg. <div class="fpage">7951</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20polymorphism%20in%20normal%20human%20fibroblasts%20as%20analyzed%20by%20two-dimensional%20polyacrylamide%20gel%20electrophoresis&amp;author=K.E.%20Walton&amp;author=D.%20Styer&amp;author=E.I.%20Gruenstein&amp;publication_year=1979&amp;journal=J.%20Biol.%20Chem.&amp;volume=254&amp;pages=7951-60" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/468800" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">G.M.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Romano</div> <div class="given-names">L.J.</div></div>. </div><div class="article-title">Human MutSalpha specifically binds to DNA containing aminofluorene and acetylaminofluorene adducts</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1996</div>, vol. <div class="volume">271</div> (pg. <div class="fpage">24084</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20MutSalpha%20specifically%20binds%20to%20DNA%20containing%20aminofluorene%20and%20acetylaminofluorene%20adducts&amp;author=G.M.%20Li&amp;author=H.%20Wang&amp;author=L.J.%20Romano&amp;publication_year=1996&amp;journal=J.%20Biol.%20Chem.&amp;volume=271&amp;pages=24084-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.271.39.24084" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.271.39.24084" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.271.39.24084"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8798646" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hong</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Hong</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Fish stem cell cultures</div>, <div class="source">Int. J. Biol. Sci.</div> , <div class="year">2011</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">392</div>-<div class="lpage">402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fish%20stem%20cell%20cultures&amp;author=N.%20Hong&amp;author=Z.%20Li&amp;author=Y.%20Hong&amp;publication_year=2011&amp;journal=Int.%20J.%20Biol.%20Sci.&amp;volume=7&amp;pages=392-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.7150/ijbs.7.392" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7150%2Fijbs.7.392" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.7150%2Fijbs.7.392"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21547056" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hong</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Winkler</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Schartl</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Production of medakafish chimeras from a stable embryonic stem cell line</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1998</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">3679</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Production%20of%20medakafish%20chimeras%20from%20a%20stable%20embryonic%20stem%20cell%20line&amp;author=Y.%20Hong&amp;author=C.%20Winkler&amp;author=M.%20Schartl&amp;publication_year=1998&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=3679-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.7.3679" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.7.3679" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.7.3679"> </span></div> </div></div></div></div></div><div content-id="DSS023C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ruano</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Kidd</div> <div class="given-names">K.K.</div></div>. </div><div class="article-title">Modeling of heteroduplex formation during PCR from mixtures of DNA templates</div>, <div class="source">PCR Methods Appl.</div> , <div class="year">1992</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">112</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Modeling%20of%20heteroduplex%20formation%20during%20PCR%20from%20mixtures%20of%20DNA%20templates&amp;author=G.%20Ruano&amp;author=K.K.%20Kidd&amp;publication_year=1992&amp;journal=PCR%20Methods%20Appl.&amp;volume=2&amp;pages=112-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2.2.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2.2.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2.2.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1362127" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Highsmith</div> <div class="given-names">W.E.</div>Jr.</div>,  <div class="name"><div class="surname">Jin</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Nataraj</div> <div class="given-names">A.J.</div></div>, et al. </div><div class="article-title">Use of a DNA toolbox for the characterization of mutation scanning methods. I: construction of the toolbox and evaluation of heteroduplex analysis</div>, <div class="source">Electrophoresis</div> , <div class="year">1999</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1186</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Use%20of%20a%20DNA%20toolbox%20for%20the%20characterization%20of%20mutation%20scanning%20methods.%20I%3A%20construction%20of%20the%20toolbox%20and%20evaluation%20of%20heteroduplex%20analysis&amp;author=W.E.%20Highsmith&amp;author=Q.%20Jin&amp;author=A.J.%20Nataraj&amp;publication_year=1999&amp;journal=Electrophoresis&amp;volume=20&amp;pages=1186-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1522-2683" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1522-2683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1522-2683"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10380758" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Birnboim</div> <div class="given-names">H.C.</div></div>,  <div class="name"><div class="surname">Doly</div> <div class="given-names">J.</div></div>. </div><div class="article-title">A rapid alkaline extraction procedure for screening recombinant plasmid DNA</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1979</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">1513</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20rapid%20alkaline%20extraction%20procedure%20for%20screening%20recombinant%20plasmid%20DNA&amp;author=H.C.%20Birnboim&amp;author=J.%20Doly&amp;publication_year=1979&amp;journal=Nucleic%20Acids%20Res.&amp;volume=7&amp;pages=1513-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/7.6.1513" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F7.6.1513" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F7.6.1513"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/388356" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hong</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Medaka vasa is required for migration but not survival of primordial germ cells</div>, <div class="source">Mech. Dev.</div> , <div class="year">2009</div>, vol. <div class="volume">126</div> (pg. <div class="fpage">366</div>-<div class="lpage">381</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Medaka%20vasa%20is%20required%20for%20migration%20but%20not%20survival%20of%20primordial%20germ%20cells&amp;author=M.%20Li&amp;author=N.%20Hong&amp;author=H.%20Xu&amp;publication_year=2009&amp;journal=Mech.%20Dev.&amp;volume=126&amp;pages=366-381" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mod.2009.02.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mod.2009.02.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mod.2009.02.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19249358" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sambrook</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Russell</div> <div class="given-names">D.W.</div></div>. </div><div class="article-title">Isolation of DNA fragments from polyacrylamide gels by the crush and soak method</div>, <div class="source">Cold Spring Harb Protoc.</div> , <div class="year">2006</div><div class="pub-id-doi"><a href="https://doi.org/10.1101/pdb.prot2936" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/pdb.prot2936</a></div></div></div></div></div><div content-id="DSS023C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nataraj</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Olivos-Glander</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Kusukawa</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Highsmith</div> <div class="given-names">W.E.</div>Jr.</div>. </div><div class="article-title">Single-strand conformation polymorphism and heteroduplex analysis for gel-based mutation detection</div>, <div class="source">Electrophoresis</div> , <div class="year">1999</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1177</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single-strand%20conformation%20polymorphism%20and%20heteroduplex%20analysis%20for%20gel-based%20mutation%20detection&amp;author=A.J.%20Nataraj&amp;author=I.%20Olivos-Glander&amp;author=N.%20Kusukawa&amp;author=W.E.%20Highsmith&amp;publication_year=1999&amp;journal=Electrophoresis&amp;volume=20&amp;pages=1177-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1522-2683" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1522-2683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1522-2683"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10380757" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Foley</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Maeder</div> <div class="given-names">M.L.</div></div>,  <div class="name"><div class="surname">Pearlberg</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Joung</div> <div class="given-names">J.K.</div></div>,  <div class="name"><div class="surname">Peterson</div> <div class="given-names">R.T.</div></div>,  <div class="name"><div class="surname">Yeh</div> <div class="given-names">J.R.J.</div></div>. </div><div class="article-title">Targeted mutagenesis in zebrafish using customized zinc-finger nucleases</div>, <div class="source">Nat. Protoc.</div> , <div class="year">2009</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">1855</div>-<div class="lpage">68</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20mutagenesis%20in%20zebrafish%20using%20customized%20zinc-finger%20nucleases&amp;author=J.E.%20Foley&amp;author=M.L.%20Maeder&amp;author=J.%20Pearlberg&amp;author=J.K.%20Joung&amp;author=R.T.%20Peterson&amp;author=J.R.J.%20Yeh&amp;publication_year=2009&amp;journal=Nat.%20Protoc.&amp;volume=4&amp;pages=1855-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2009.209" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2009.209" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2009.209"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20010934" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Meng</div> <div class="given-names">X.D.</div></div>,  <div class="name"><div class="surname">Noyes</div> <div class="given-names">M.B.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">L.H.J.</div></div>,  <div class="name"><div class="surname">Lawson</div> <div class="given-names">N.D.</div></div>,  <div class="name"><div class="surname">Wolfe</div> <div class="given-names">S.A.</div></div>. </div><div class="article-title">Targeted gene inactivation in zebrafish using engineered zinc-finger nucleases</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2008</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">695</div>-<div class="lpage">701</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20gene%20inactivation%20in%20zebrafish%20using%20engineered%20zinc-finger%20nucleases&amp;author=X.D.%20Meng&amp;author=M.B.%20Noyes&amp;author=L.H.J.%20Zhu&amp;author=N.D.%20Lawson&amp;author=S.A.%20Wolfe&amp;publication_year=2008&amp;journal=Nat.%20Biotechnol.&amp;volume=26&amp;pages=695-701" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1398" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1398" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1398"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18500337" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hong</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Winkler</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Schartl</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Pluripotency and differentiation of embryonic stem cell lines from the medakafish (<em>Oryzias latipes</em>)</div>, <div class="source">Mech. Dev.</div> , <div class="year">1996</div>, vol. <div class="volume">60</div> (pg. <div class="fpage">33</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pluripotency%20and%20differentiation%20of%20embryonic%20stem%20cell%20lines%20from%20the%20medakafish%20(Oryzias%20latipes)&amp;author=Y.%20Hong&amp;author=C.%20Winkler&amp;author=M.%20Schartl&amp;publication_year=1996&amp;journal=Mech.%20Dev.&amp;volume=60&amp;pages=33-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0925-4773(96)00596-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0925-4773(96)00596-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0925-4773(96)00596-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9025059" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hong</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Hong</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Generation of medaka fish haploid embryonic stem cells</div>, <div class="source">Science</div> , <div class="year">2009</div>, vol. <div class="volume">326</div> (pg. <div class="fpage">430</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Generation%20of%20medaka%20fish%20haploid%20embryonic%20stem%20cells&amp;author=M.%20Yi&amp;author=N.%20Hong&amp;author=Y.%20Hong&amp;publication_year=2009&amp;journal=Science&amp;volume=326&amp;pages=430-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1175151" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1175151" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1175151"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19833967" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hong</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Establishment of a normal medakafish spermatogonial cell line capable of sperm production <em>in vitro</em></div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">8011</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Establishment%20of%20a%20normal%20medakafish%20spermatogonial%20cell%20line%20capable%20of%20sperm%20production%20in%20vitro&amp;author=Y.%20Hong&amp;author=T.%20Liu&amp;author=H.%20Zhao&amp;publication_year=2004&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=8011-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0308668101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0308668101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0308668101"> </span></div> </div></div></div></div></div><div content-id="DSS023C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hong</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Zeng</div> <div class="given-names">Z.</div></div>, et al. </div><div class="article-title">Accessibility of host cell lineages to medaka stem cells depends on genetic background and irradiation of recipient embryos</div>, <div class="source">Cell. Mol. Life Sci.</div> , <div class="year">2010</div>, vol. <div class="volume">67</div> (pg. <div class="fpage">1189</div>-<div class="lpage">202</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accessibility%20of%20host%20cell%20lineages%20to%20medaka%20stem%20cells%20depends%20on%20genetic%20background%20and%20irradiation%20of%20recipient%20embryos&amp;author=N.%20Hong&amp;author=M.%20Li&amp;author=Z.%20Zeng&amp;publication_year=2010&amp;journal=Cell.%20Mol.%20Life%20Sci.&amp;volume=67&amp;pages=1189-202" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00018-009-0247-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00018-009-0247-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00018-009-0247-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20238480" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">T.M.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Wei</div> <div class="given-names">Q.W.</div></div>,  <div class="name"><div class="surname">Hong</div> <div class="given-names">Y.H.</div></div>. </div><div class="article-title">Sperm nuclear transfer and transgenic production in the fish Medaka</div>, <div class="source">Int. J. Biol. Sci.</div> , <div class="year">2011</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">469</div>-<div class="lpage">75</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sperm%20nuclear%20transfer%20and%20transgenic%20production%20in%20the%20fish%20Medaka&amp;author=T.M.%20Liu&amp;author=L.%20Liu&amp;author=Q.W.%20Wei&amp;author=Y.H.%20Hong&amp;publication_year=2011&amp;journal=Int.%20J.%20Biol.%20Sci.&amp;volume=7&amp;pages=469-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.7150/ijbs.7.469" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7150%2Fijbs.7.469" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.7150%2Fijbs.7.469"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21547064" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shibata</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Paul-Prasanth</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Expression of gonadal soma derived factor (GSDF) is spatially and temporally correlated with early testicular differentiation in medaka</div>, <div class="source">Gene Expr. Patterns</div> , <div class="year">2010</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">283</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20of%20gonadal%20soma%20derived%20factor%20(GSDF)%20is%20spatially%20and%20temporally%20correlated%20with%20early%20testicular%20differentiation%20in%20medaka&amp;author=Y.%20Shibata&amp;author=B.%20Paul-Prasanth&amp;author=A.%20Suzuki&amp;publication_year=2010&amp;journal=Gene%20Expr.%20Patterns&amp;volume=10&amp;pages=283-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gep.2010.06.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gep.2010.06.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gep.2010.06.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20601164" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cui</div> <div class="given-names">X.X.</div></div>,  <div class="name"><div class="surname">Ji</div> <div class="given-names">D.N.</div></div>,  <div class="name"><div class="surname">Fisher</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">Y.M.</div></div>,  <div class="name"><div class="surname">Briner</div> <div class="given-names">D.M.</div></div>,  <div class="name"><div class="surname">Weinstein</div> <div class="given-names">E.J.</div></div>. </div><div class="article-title">Targeted integration in rat and mouse embryos with zinc-finger nucleases</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2011</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">64</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20integration%20in%20rat%20and%20mouse%20embryos%20with%20zinc-finger%20nucleases&amp;author=X.X.%20Cui&amp;author=D.N.%20Ji&amp;author=D.A.%20Fisher&amp;author=Y.M.%20Wu&amp;author=D.M.%20Briner&amp;author=E.J.%20Weinstein&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol.&amp;volume=29&amp;pages=64-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1731" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1731" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1731"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21151125" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS023C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS023C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rossetti</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Corra</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Biasi</div> <div class="given-names">M.O.</div></div>,  <div class="name"><div class="surname">Turco</div> <div class="given-names">A.E.</div></div>,  <div class="name"><div class="surname">Pignatti</div> <div class="given-names">P.F.</div></div>. </div><div class="article-title">Comparison of heteroduplex and single-strand conformation analyses, followed by ethidium fluorescence visualization, for the detection of mutations in four human genes</div>, <div class="source">Mol. Cell. Probes</div> , <div class="year">1995</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">195</div>-<div class="lpage">200</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20heteroduplex%20and%20single-strand%20conformation%20analyses%2C%20followed%20by%20ethidium%20fluorescence%20visualization%2C%20for%20the%20detection%20of%20mutations%20in%20four%20human%20genes&amp;author=S.%20Rossetti&amp;author=S.%20Corra&amp;author=M.O.%20Biasi&amp;author=A.E.%20Turco&amp;author=P.F.%20Pignatti&amp;publication_year=1995&amp;journal=Mol.%20Cell.%20Probes&amp;volume=9&amp;pages=195-200" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/mcpr.1994.0028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fmcpr.1994.0028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fmcpr.1994.0028"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7477013" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Osamu Ohara</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=629170&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss023/2/dss023-dss023supp.pdf?Expires=2147483647&amp;Signature=eB26tQhMbx0jN7~3amt8ut~FJoJOmLyZaVmtZeYW-eIE5Cc9dPVybqYKi~iLKzDsOyUEZVv7P7aeBqu86PPbRIWQWaJi~QDlZZYdbEgrbjQpjZ7a3~A8zFjxCI0-~16j1LLgLnR2eJtiPJheNBl5UUyIoLTl0tziYXxj2uJ0ihjnnXrBCJhUAWjnNBudlhBo68xF465Oi8zF4MEgVpXLJstDiXvAMOWU04NfR8GBjc3Xbz5kl4unNTDBp1O6adfit8VtSqV6yiO-iCXSDpmHw~pigiCrPj8-KA0Qg3ArzYRUXXUW6QhMBvRe2SPKo-f4HN7mY0LP693CueMEmXzZ-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">357</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000310822200006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">27</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dss023"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="629170"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000310822200006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f19%2f5%2f423%2f629170">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000310822200006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (27)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F19%2F5%2F423%2F629170" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/629170'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Efficient Detection, Quantification and Enrichment of Subtle Allelic Alterations" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dss023");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f19%2f5%2f423%2f629170">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

