

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/17/3/197/353879","@type":"ScholarlyArticle","name":"Mutation@A Glance: An Integrative Web Application for Analysing Mutations from Human Genetic Diseases","datePublished":"2010-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/17/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2010-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsq010","keywords":["genetic disease","mutation","polymorphism","bioinformatics","protein structure"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/17/3/197/353879","author":[{"name":"Hijikata, Atsushi","affiliation":"1  Laboratory for Immunogenomics ,  RIKEN Research Center for Allergy and Immunology ,  1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045 ,  Japan","@type":"Person"},{"name":"Raju, Rajesh","affiliation":"2  Research Unit for Immunoinformatics ,  RIKEN Research Center for Allergy and Immunology ,  1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045 ,  Japan    3  Institute of Bioinformatics, International Technology Park ,  Bangalore 560 066 ,  India    4  Department of Biotechnology and Bioinformatics ,  Kuvempu University ,  Jnanasahyadri, Shimoga 577 451 ,  India","@type":"Person"},{"name":"Keerthikumar, Shivakumar","affiliation":"2  Research Unit for Immunoinformatics ,  RIKEN Research Center for Allergy and Immunology ,  1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045 ,  Japan    3  Institute of Bioinformatics, International Technology Park ,  Bangalore 560 066 ,  India    4  Department of Biotechnology and Bioinformatics ,  Kuvempu University ,  Jnanasahyadri, Shimoga 577 451 ,  India","@type":"Person"},{"name":"Ramabadran, Subhashri","affiliation":"2  Research Unit for Immunoinformatics ,  RIKEN Research Center for Allergy and Immunology ,  1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045 ,  Japan    3  Institute of Bioinformatics, International Technology Park ,  Bangalore 560 066 ,  India","@type":"Person"},{"name":"Balakrishnan, Lavanya","affiliation":"2  Research Unit for Immunoinformatics ,  RIKEN Research Center for Allergy and Immunology ,  1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045 ,  Japan    3  Institute of Bioinformatics, International Technology Park ,  Bangalore 560 066 ,  India","@type":"Person"},{"name":"Ramadoss, Suresh Kumar","affiliation":"2  Research Unit for Immunoinformatics ,  RIKEN Research Center for Allergy and Immunology ,  1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045 ,  Japan","@type":"Person"},{"name":"Pandey, Akhilesh","affiliation":"3  Institute of Bioinformatics, International Technology Park ,  Bangalore 560 066 ,  India    5  McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University School of Medicine ,  733 N. Broadway, BRB Room 527, Baltimore, MD 21205 ,  USA","@type":"Person"},{"name":"Mohan, Sujatha","affiliation":"2  Research Unit for Immunoinformatics ,  RIKEN Research Center for Allergy and Immunology ,  1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045 ,  Japan    3  Institute of Bioinformatics, International Technology Park ,  Bangalore 560 066 ,  India","@type":"Person"},{"name":"Ohara, Osamu","affiliation":"1  Laboratory for Immunogenomics ,  RIKEN Research Center for Allergy and Immunology ,  1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045 ,  Japan    6  Laboratory of Genome Technology, Department of Human Genome Research ,  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"}],"description":"Abstract.  Although mutation analysis serves as a key part in making a definitive diagnosis about a genetic disease, it still remains a time-consuming step to i","pageStart":"197","pageEnd":"208","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/3/1/m_cover.gif?Expires=2147483647&Signature=zrlqOYlr8puhWaI06gf8sYMMP6iQcSaE6WnxD18CI1vM-vSYmV7zw1fL1UUm1LAo6OR8YaC2ow7Gg3gFyOuRxjrsW2e9seZ-Fc4W0zpKOKeLDqycZiol4B5gPhMqbsAY9k5dM2sXst3uhEwLyf9p6Clk-Cv9XLljwpdq37Ye9OThamAMiMSAoDxynPtt~5t-2ainSI295gFWPk3t9zFxKYx4zJXORsGXGnkmtct97jCcoDB0mHB0FSIfJFRj9SkO~ccPVDFJ5bkP0CQlUseB~SQw04UmN2w0van9EOcwauxu2Sl9Oe1AFiN92gYP2rZEbuX5qeGuIN6geNJvEhApPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Mutation@A Glance: An Integrative Web Application for Analysing Mutations from Human Genetic Diseases","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/3/1/m_cover.gif?Expires=2147483647&Signature=zrlqOYlr8puhWaI06gf8sYMMP6iQcSaE6WnxD18CI1vM-vSYmV7zw1fL1UUm1LAo6OR8YaC2ow7Gg3gFyOuRxjrsW2e9seZ-Fc4W0zpKOKeLDqycZiol4B5gPhMqbsAY9k5dM2sXst3uhEwLyf9p6Clk-Cv9XLljwpdq37Ye9OThamAMiMSAoDxynPtt~5t-2ainSI295gFWPk3t9zFxKYx4zJXORsGXGnkmtct97jCcoDB0mHB0FSIfJFRj9SkO~ccPVDFJ5bkP0CQlUseB~SQw04UmN2w0van9EOcwauxu2Sl9Oe1AFiN92gYP2rZEbuX5qeGuIN6geNJvEhApPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Mutation@A Glance: An Integrative Web Application for Analysing Mutations from Human Genetic Diseases" />
    <meta property="og:description" content="Abstract.  Although mutation analysis serves as a key part in making a definitive diagnosis about a genetic disease, it still remains a time-consuming step to i" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/17/3/197/353879" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2010-04-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Hijikata, Atsushi" /><meta name="citation_author_institution" content="Laboratory for Immunogenomics, RIKEN Research Center for Allergy and Immunology, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_author" content="Raju, Rajesh" /><meta name="citation_author_institution" content="Research Unit for Immunoinformatics, RIKEN Research Center for Allergy and Immunology, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_author_institution" content="Institute of Bioinformatics, International Technology Park, Bangalore 560 066, India" /><meta name="citation_author_institution" content="Department of Biotechnology and Bioinformatics, Kuvempu University, Jnanasahyadri, Shimoga 577 451, India" /><meta name="citation_author" content="Keerthikumar, Shivakumar" /><meta name="citation_author_institution" content="Research Unit for Immunoinformatics, RIKEN Research Center for Allergy and Immunology, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_author_institution" content="Institute of Bioinformatics, International Technology Park, Bangalore 560 066, India" /><meta name="citation_author_institution" content="Department of Biotechnology and Bioinformatics, Kuvempu University, Jnanasahyadri, Shimoga 577 451, India" /><meta name="citation_author" content="Ramabadran, Subhashri" /><meta name="citation_author_institution" content="Research Unit for Immunoinformatics, RIKEN Research Center for Allergy and Immunology, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_author_institution" content="Institute of Bioinformatics, International Technology Park, Bangalore 560 066, India" /><meta name="citation_author" content="Balakrishnan, Lavanya" /><meta name="citation_author_institution" content="Research Unit for Immunoinformatics, RIKEN Research Center for Allergy and Immunology, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_author_institution" content="Institute of Bioinformatics, International Technology Park, Bangalore 560 066, India" /><meta name="citation_author" content="Ramadoss, Suresh Kumar" /><meta name="citation_author_institution" content="Research Unit for Immunoinformatics, RIKEN Research Center for Allergy and Immunology, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_author" content="Pandey, Akhilesh" /><meta name="citation_author_institution" content="Institute of Bioinformatics, International Technology Park, Bangalore 560 066, India" /><meta name="citation_author_institution" content="McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University School of Medicine, 733 N. Broadway, BRB Room 527, Baltimore, MD 21205, USA" /><meta name="citation_author" content="Mohan, Sujatha" /><meta name="citation_author_institution" content="Research Unit for Immunoinformatics, RIKEN Research Center for Allergy and Immunology, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_author_institution" content="Institute of Bioinformatics, International Technology Park, Bangalore 560 066, India" /><meta name="citation_author" content="Ohara, Osamu" /><meta name="citation_author_institution" content="Laboratory for Immunogenomics, RIKEN Research Center for Allergy and Immunology, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_author_institution" content="Laboratory of Genome Technology, Department of Human Genome Research, Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_title" content="Mutation@A Glance: An Integrative Web Application for Analysing Mutations from Human Genetic Diseases" /><meta name="citation_firstpage" content="197" /><meta name="citation_lastpage" content="208" /><meta name="citation_doi" content="10.1093/dnares/dsq010" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="17" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2010/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=McKusick's Online Mendelian Inheritance in Man (OMIM); Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=D793-6; " /><meta name="citation_reference" content="citation_title=Primary immunodeficiency diseases: an update; J. Allergy Clin. Immunol.;  citation_year=2004;  citation_volume=114;  citation_pages=677-87; " /><meta name="citation_reference" content="citation_title=Primary immunodeficiency diseases: an update from the International Union of Immunological Societies Primary Immunodeficiency Diseases Classification Committee; J. Allergy Clin. Immunol.;  citation_year=2007;  citation_volume=120;  citation_pages=776-94; " /><meta name="citation_reference" content="citation_title=Is it necessary to identify molecular defects in primary immunodeficiency disease?; J. Allergy Clin. Immunol;  citation_year=2008;  citation_volume=122;  citation_pages=1069-73; " /><meta name="citation_reference" content="citation_title=A second generation human haplotype map of over 3.1 million SNPs; Nature;  citation_year=2007;  citation_volume=449;  citation_pages=851-61; " /><meta name="citation_reference" content="citation_title=Power of deep, all-exon resequencing for discovery of human trait genes; Proc. Natl Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=3871-6; " /><meta name="citation_reference" content="citation_title=Identification of deleterious mutations within three human genomes; Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=1553-61; " /><meta name="citation_reference" content="citation_title=Prediction of deleterious human alleles; Hum. Mol. Genet.;  citation_year=2001;  citation_volume=10;  citation_pages=591-7; " /><meta name="citation_reference" content="citation_title=Classification of rare missense substitutions, using risk surfaces, with genetic- and molecular-epidemiology applications; Hum. Mutat.;  citation_year=2008;  citation_volume=29;  citation_pages=1342-54; " /><meta name="citation_reference" content="citation_title=Assessing pathogenicity: overview of results from the IARC Unclassified Genetic Variants Working Group; Hum. Mutat.;  citation_year=2008;  citation_volume=29;  citation_pages=1261-4; " /><meta name="citation_reference" content="citation_title=In silico analysis of missense substitutions using sequence-alignment based methods; Hum. Mutat.;  citation_year=2008;  citation_volume=29;  citation_pages=1327-36; " /><meta name="citation_reference" content="citation_title=Evaluation of in silico splice tools for decision-making in molecular diagnosis; Hum. Mutat.;  citation_year=2008;  citation_volume=29;  citation_pages=975-82; " /><meta name="citation_reference" content="citation_title=The Human Gene Mutation Database: 2008 update; Genome Med.;  citation_year=2009;  citation_volume=1;  citation_pages=13" /><meta name="citation_reference" content="citation_title=MutDB: update on development of tools for the biochemical analysis of genetic variation; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D815-9; " /><meta name="citation_reference" content="citation_title=The NCBI dbGaP database of genotypes and phenotypes; Nat. Genet.;  citation_year=2007;  citation_volume=39;  citation_pages=1181-6; " /><meta name="citation_reference" content="citation_title=Immunodeficiency mutation databases (IDbases); Hum. Mutat.;  citation_year=2006;  citation_volume=27;  citation_pages=1200-8; " /><meta name="citation_reference" content="citation_title=RAPID: Resource of Asian Primary Immunodeficiency Diseases; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=D863-7; " /><meta name="citation_reference" content="citation_title=dbSNP: the NCBI database of genetic variation; Nucleic Acids Res.;  citation_year=2001;  citation_volume=29;  citation_pages=308-11; " /><meta name="citation_reference" content="citation_title=Planning the human variome project: the Spain report; Hum. Mutat.;  citation_year=2009;  citation_volume=30;  citation_pages=496-510; " /><meta name="citation_reference" content="citation_title=F-SNP: computationally predicted functional SNPs for disease association studies; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D820-4; " /><meta name="citation_reference" content="citation_title=FASTSNP: an always up-to-date and extendable service for SNP function analysis and prioritization; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=W635-41; " /><meta name="citation_reference" content="citation_title=SNPs3D: candidate gene and SNP selection for association studies; BMC Bioinformatics;  citation_year=2006;  citation_volume=7;  citation_pages=166" /><meta name="citation_reference" content="citation_title=SNPnexus: a web database for functional annotation of newly discovered and public domain single nucleotide polymorphisms; Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=655-61; " /><meta name="citation_reference" content="citation_title=An integrated database-pipeline system for studying single nucleotide polymorphisms and diseases; BMC Bioinformatics;  citation_year=2008;  citation_volume=9;  citation_issue=Suppl 12;  citation_pages=S19" /><meta name="citation_reference" content="citation_title=Genatlas database, genes and development defects; C. R. Acad. Sci. III;  citation_year=1998;  citation_volume=321;  citation_pages=805-17; " /><meta name="citation_reference" content="citation_title=SIFT: predicting amino acid changes that affect protein function; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=3812-4; " /><meta name="citation_reference" content="citation_title=Human non-synonymous SNPs: server and survey; Nucleic Acids Res.;  citation_year=2002;  citation_volume=30;  citation_pages=3894-900; " /><meta name="citation_reference" content="citation_title=PANTHER: a library of protein families and subfamilies indexed by function; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=2129-41; " /><meta name="citation_reference" content="citation_title=SNAP: predict effect of non-synonymous polymorphisms on function; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=3823-35; " /><meta name="citation_reference" content="citation_title=PMUT: a web-based tool for the annotation of pathological mutations on proteins; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=3176-8; " /><meta name="citation_reference" content="citation_title=nsSNPAnalyzer: identifying disease-associated nonsynonymous single nucleotide polymorphisms; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=W480-2; " /><meta name="citation_reference" content="citation_title=Pathogenic or not? And if so, then how? Studying the effects of missense mutations using bioinformatics methods; Hum. Mutat.;  citation_year=2009;  citation_volume=30;  citation_pages=703-14; " /><meta name="citation_reference" content="citation_title=Predicting the effects of amino acid substitutions on protein function; Annu. Rev. Genomics Hum. Genet.;  citation_year=2006;  citation_volume=7;  citation_pages=61-80; " /><meta name="citation_reference" content="citation_title=UniProtKB/Swiss-Prot; Methods Mol. Biol.;  citation_year=2007;  citation_volume=406;  citation_pages=89-112; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool; J. Mol. Biol.;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice; Nucleic Acids Res.;  citation_year=1994;  citation_volume=22;  citation_pages=4673-80; " /><meta name="citation_reference" content="citation_title=The Protein Data Bank; Acta Crystallogr. D Biol. Crystallogr.;  citation_year=2002;  citation_volume=58;  citation_pages=899-907; " /><meta name="citation_reference" content="citation_title=Comparative protein structure modeling of genes and genomes; Annu. Rev. Biophys. Biomol. Struct.;  citation_year=2000;  citation_volume=29;  citation_pages=291-325; " /><meta name="citation_reference" content="citation_title=Statistical potential for assessment and prediction of protein structures; Protein Sci.;  citation_year=2006;  citation_volume=15;  citation_pages=2507-24; " /><meta name="citation_reference" content="citation_title=Environment and exposure to solvent of protein atoms. Lysozyme and insulin; J. Mol. Biol.;  citation_year=1973;  citation_volume=79;  citation_pages=351-71; " /><meta name="citation_reference" content="citation_title=The DISOPRED server for the prediction of protein disorder; Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=2138-9; " /><meta name="citation_reference" content="citation_title=Towards a structural basis of human non-synonymous single nucleotide polymorphisms; Trends Genet.;  citation_year=2000;  citation_volume=16;  citation_pages=198-200; " /><meta name="citation_reference" content="citation_title=The amino-acid mutational spectrum of human genetic disease; Genome Biol.;  citation_year=2003;  citation_volume=4;  citation_pages=R72" /><meta name="citation_reference" content="citation_title=SNPs, protein structure, and disease; Hum. Mutat.;  citation_year=2001;  citation_volume=17;  citation_pages=263-70; " /><meta name="citation_reference" content="citation_title=Characterization of disease-associated single amino acid polymorphisms in terms of sequence and structure properties; J. Mol. Biol.;  citation_year=2002;  citation_volume=315;  citation_pages=771-86; " /><meta name="citation_reference" content="citation_title=coliSNP database server mapping nsSNPs on protein structures; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D409-13; " /><meta name="citation_reference" content="citation_title=Conservation of intrinsic disorder in protein domains and families: I. A database of conserved predicted disordered regions; J. Proteome Res.;  citation_year=2006;  citation_volume=5;  citation_pages=879-87; " /><meta name="citation_reference" content="citation_title=Protein interactions in human genetic diseases; Genome Biol.;  citation_year=2008;  citation_volume=9;  citation_pages=R9" /><meta name="citation_reference" content="citation_title=STAT3 mutations in the hyper-IgE syndrome; N. Engl. J. Med.;  citation_year=2007;  citation_volume=357;  citation_pages=1608-19; " /><meta name="citation_reference" content="citation_title=Dominant-negative mutations in the DNA-binding domain of STAT3 cause hyper-IgE syndrome; Nature;  citation_year=2007;  citation_volume=448;  citation_pages=1058-62; " /><meta name="citation_reference" content="citation_title=The Pfam protein families database; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D281-8; " /><meta name="citation_reference" content="citation_title=Database resources of the National Center for Biotechnology Information; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=D5-15; " /><meta name="citation_reference" content="citation_title=Human Protein Reference Database—2009 update; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=D767-72; " /><meta name="citation_reference" content="citation_title=GeneCards: a novel functional genomics compendium with automated data mining and query reformulation support; Bioinformatics;  citation_year=1998;  citation_volume=14;  citation_pages=656-64; " /><meta name="citation_reference" content="citation_title=Construction of an open-access database that integrates cross-reference information from the transcriptome and proteome of immune cells; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=2934-41; " /><meta name="citation_reference" content="citation_title=KEGG: kyoto encyclopedia of genes and genomes; Nucleic Acids Res.;  citation_year=2000;  citation_volume=28;  citation_pages=27-30; " /><meta name="citation_reference" content="citation_title=Three-dimensional structure of the Stat3beta homodimer bound to DNA; Nature;  citation_year=1998;  citation_volume=394;  citation_pages=145-51; " /><meta name="citation_reference" content="citation_title=BLAT—the BLAST-like alignment tool; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=656-64; " /><meta name="citation_reference" content="citation_title=The first Korean genome sequence and analysis: full genome sequencing for a socio-ethnic group; Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=1622-9; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/17/3/197/1012177/dsq010.pdf" /><meta name="description" content="Abstract.  Although mutation analysis serves as a key part in making a definitive diagnosis about a genetic disease, it still remains a time-consuming step to i" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/17/3/197/353879" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/17/3/197/353879" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/17/3/197/353879");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsq010';
    </script>
<title>
	Mutation@A Glance: An Integrative Web Application for Analysing Mutations from Human Genetic Diseases | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./353879" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bf+JATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGlqaWthdGEsIEF0c3VzaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IGZvciBJbW11bm9nZW5vbWljcywgUklLRU4gUmVzZWFyY2ggQ2VudGVyIGZvciBBbGxlcmd5IGFuZCBJbW11bm9sb2d5LCAxLTctMjIgU3VlaGlyby1jaG8sIFRzdXJ1bWkta3UsIFlva29oYW1hLCBLYW5hZ2F3YSAyMzAtMDA0NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJSYWp1LCBSYWplc2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSZXNlYXJjaCBVbml0IGZvciBJbW11bm9pbmZvcm1hdGljcywgUklLRU4gUmVzZWFyY2ggQ2VudGVyIGZvciBBbGxlcmd5IGFuZCBJbW11bm9sb2d5LCAxLTctMjIgU3VlaGlyby1jaG8sIFRzdXJ1bWkta3UsIFlva29oYW1hLCBLYW5hZ2F3YSAyMzAtMDA0NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgQmlvaW5mb3JtYXRpY3MsIEludGVybmF0aW9uYWwgVGVjaG5vbG9neSBQYXJrLCBCYW5nYWxvcmUgNTYwIDA2NiwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb3RlY2hub2xvZ3kgYW5kIEJpb2luZm9ybWF0aWNzLCBLdXZlbXB1IFVuaXZlcnNpdHksIEpuYW5hc2FoeWFkcmksIFNoaW1vZ2EgNTc3IDQ1MSwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLZWVydGhpa3VtYXIsIFNoaXZha3VtYXIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSZXNlYXJjaCBVbml0IGZvciBJbW11bm9pbmZvcm1hdGljcywgUklLRU4gUmVzZWFyY2ggQ2VudGVyIGZvciBBbGxlcmd5IGFuZCBJbW11bm9sb2d5LCAxLTctMjIgU3VlaGlyby1jaG8sIFRzdXJ1bWkta3UsIFlva29oYW1hLCBLYW5hZ2F3YSAyMzAtMDA0NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgQmlvaW5mb3JtYXRpY3MsIEludGVybmF0aW9uYWwgVGVjaG5vbG9neSBQYXJrLCBCYW5nYWxvcmUgNTYwIDA2NiwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb3RlY2hub2xvZ3kgYW5kIEJpb2luZm9ybWF0aWNzLCBLdXZlbXB1IFVuaXZlcnNpdHksIEpuYW5hc2FoeWFkcmksIFNoaW1vZ2EgNTc3IDQ1MSwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJSYW1hYmFkcmFuLCBTdWJoYXNocmkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSZXNlYXJjaCBVbml0IGZvciBJbW11bm9pbmZvcm1hdGljcywgUklLRU4gUmVzZWFyY2ggQ2VudGVyIGZvciBBbGxlcmd5IGFuZCBJbW11bm9sb2d5LCAxLTctMjIgU3VlaGlyby1jaG8sIFRzdXJ1bWkta3UsIFlva29oYW1hLCBLYW5hZ2F3YSAyMzAtMDA0NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgQmlvaW5mb3JtYXRpY3MsIEludGVybmF0aW9uYWwgVGVjaG5vbG9neSBQYXJrLCBCYW5nYWxvcmUgNTYwIDA2NiwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCYWxha3Jpc2huYW4sIExhdmFueWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSZXNlYXJjaCBVbml0IGZvciBJbW11bm9pbmZvcm1hdGljcywgUklLRU4gUmVzZWFyY2ggQ2VudGVyIGZvciBBbGxlcmd5IGFuZCBJbW11bm9sb2d5LCAxLTctMjIgU3VlaGlyby1jaG8sIFRzdXJ1bWkta3UsIFlva29oYW1hLCBLYW5hZ2F3YSAyMzAtMDA0NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgQmlvaW5mb3JtYXRpY3MsIEludGVybmF0aW9uYWwgVGVjaG5vbG9neSBQYXJrLCBCYW5nYWxvcmUgNTYwIDA2NiwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJSYW1hZG9zcywgU3VyZXNoIEt1bWFyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUmVzZWFyY2ggVW5pdCBmb3IgSW1tdW5vaW5mb3JtYXRpY3MsIFJJS0VOIFJlc2VhcmNoIENlbnRlciBmb3IgQWxsZXJneSBhbmQgSW1tdW5vbG9neSwgMS03LTIyIFN1ZWhpcm8tY2hvLCBUc3VydW1pLWt1LCBZb2tvaGFtYSwgS2FuYWdhd2EgMjMwLTAwNDUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUGFuZGV5LCBBa2hpbGVzaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBCaW9pbmZvcm1hdGljcywgSW50ZXJuYXRpb25hbCBUZWNobm9sb2d5IFBhcmssIEJhbmdhbG9yZSA1NjAgMDY2LCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik1jS3VzaWNrLU5hdGhhbnMgSW5zdGl0dXRlIG9mIEdlbmV0aWMgTWVkaWNpbmUsIEpvaG5zIEhvcGtpbnMgVW5pdmVyc2l0eSBTY2hvb2wgb2YgTWVkaWNpbmUsIDczMyBOLiBCcm9hZHdheSwgQlJCIFJvb20gNTI3LCBCYWx0aW1vcmUsIE1EIDIxMjA1LCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNb2hhbiwgU3VqYXRoYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlJlc2VhcmNoIFVuaXQgZm9yIEltbXVub2luZm9ybWF0aWNzLCBSSUtFTiBSZXNlYXJjaCBDZW50ZXIgZm9yIEFsbGVyZ3kgYW5kIEltbXVub2xvZ3ksIDEtNy0yMiBTdWVoaXJvLWNobywgVHN1cnVtaS1rdSwgWW9rb2hhbWEsIEthbmFnYXdhIDIzMC0wMDQ1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBCaW9pbmZvcm1hdGljcywgSW50ZXJuYXRpb25hbCBUZWNobm9sb2d5IFBhcmssIEJhbmdhbG9yZSA1NjAgMDY2LCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9oYXJhLCBPc2FtdSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgZm9yIEltbXVub2dlbm9taWNzLCBSSUtFTiBSZXNlYXJjaCBDZW50ZXIgZm9yIEFsbGVyZ3kgYW5kIEltbXVub2xvZ3ksIDEtNy0yMiBTdWVoaXJvLWNobywgVHN1cnVtaS1rdSwgWW9rb2hhbWEsIEthbmFnYXdhIDIzMC0wMDQ1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgR2Vub21lIFRlY2hub2xvZ3ksIERlcGFydG1lbnQgb2YgSHVtYW4gR2Vub21lIFJlc2VhcmNoLCBLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLUthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJNdXRhdGlvbkBBIEdsYW5jZTogQW4gSW50ZWdyYXRpdmUgV2ViIEFwcGxpY2F0aW9uIGZvciBBbmFseXNpbmcgTXV0YXRpb25zIGZyb20gSHVtYW4gR2VuZXRpYyBEaXNlYXNlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjE5NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMjA4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNxMDEwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTAvMDYvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NY0t1c2ljaydzIE9ubGluZSBNZW5kZWxpYW4gSW5oZXJpdGFuY2UgaW4gTWFuIChPTUlNKTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9RDc5My02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmltYXJ5IGltbXVub2RlZmljaWVuY3kgZGlzZWFzZXM6IGFuIHVwZGF0ZTsgSi4gQWxsZXJneSBDbGluLiBJbW11bm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTQ7ICBjaXRhdGlvbl9wYWdlcz02NzctODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByaW1hcnkgaW1tdW5vZGVmaWNpZW5jeSBkaXNlYXNlczogYW4gdXBkYXRlIGZyb20gdGhlIEludGVybmF0aW9uYWwgVW5pb24gb2YgSW1tdW5vbG9naWNhbCBTb2NpZXRpZXMgUHJpbWFyeSBJbW11bm9kZWZpY2llbmN5IERpc2Vhc2VzIENsYXNzaWZpY2F0aW9uIENvbW1pdHRlZTsgSi4gQWxsZXJneSBDbGluLiBJbW11bm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMjA7ICBjaXRhdGlvbl9wYWdlcz03NzYtOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzIGl0IG5lY2Vzc2FyeSB0byBpZGVudGlmeSBtb2xlY3VsYXIgZGVmZWN0cyBpbiBwcmltYXJ5IGltbXVub2RlZmljaWVuY3kgZGlzZWFzZT87IEouIEFsbGVyZ3kgQ2xpbi4gSW1tdW5vbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMjI7ICBjaXRhdGlvbl9wYWdlcz0xMDY5LTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNlY29uZCBnZW5lcmF0aW9uIGh1bWFuIGhhcGxvdHlwZSBtYXAgb2Ygb3ZlciAzLjEgbWlsbGlvbiBTTlBzOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDQ5OyAgY2l0YXRpb25fcGFnZXM9ODUxLTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3dlciBvZiBkZWVwLCBhbGwtZXhvbiByZXNlcXVlbmNpbmcgZm9yIGRpc2NvdmVyeSBvZiBodW1hbiB0cmFpdCBnZW5lczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDY7ICBjaXRhdGlvbl9wYWdlcz0zODcxLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIGRlbGV0ZXJpb3VzIG11dGF0aW9ucyB3aXRoaW4gdGhyZWUgaHVtYW4gZ2Vub21lczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0xNTUzLTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW9uIG9mIGRlbGV0ZXJpb3VzIGh1bWFuIGFsbGVsZXM7IEh1bS4gTW9sLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz01OTEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2xhc3NpZmljYXRpb24gb2YgcmFyZSBtaXNzZW5zZSBzdWJzdGl0dXRpb25zLCB1c2luZyByaXNrIHN1cmZhY2VzLCB3aXRoIGdlbmV0aWMtIGFuZCBtb2xlY3VsYXItZXBpZGVtaW9sb2d5IGFwcGxpY2F0aW9uczsgSHVtLiBNdXRhdC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz0xMzQyLTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3Nlc3NpbmcgcGF0aG9nZW5pY2l0eTogb3ZlcnZpZXcgb2YgcmVzdWx0cyBmcm9tIHRoZSBJQVJDIFVuY2xhc3NpZmllZCBHZW5ldGljIFZhcmlhbnRzIFdvcmtpbmcgR3JvdXA7IEh1bS4gTXV0YXQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9MTI2MS00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbiBzaWxpY28gYW5hbHlzaXMgb2YgbWlzc2Vuc2Ugc3Vic3RpdHV0aW9ucyB1c2luZyBzZXF1ZW5jZS1hbGlnbm1lbnQgYmFzZWQgbWV0aG9kczsgSHVtLiBNdXRhdC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz0xMzI3LTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmFsdWF0aW9uIG9mIGluIHNpbGljbyBzcGxpY2UgdG9vbHMgZm9yIGRlY2lzaW9uLW1ha2luZyBpbiBtb2xlY3VsYXIgZGlhZ25vc2lzOyBIdW0uIE11dGF0LjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTk3NS04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEh1bWFuIEdlbmUgTXV0YXRpb24gRGF0YWJhc2U6IDIwMDggdXBkYXRlOyBHZW5vbWUgTWVkLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdXREQjogdXBkYXRlIG9uIGRldmVsb3BtZW50IG9mIHRvb2xzIGZvciB0aGUgYmlvY2hlbWljYWwgYW5hbHlzaXMgb2YgZ2VuZXRpYyB2YXJpYXRpb247IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPUQ4MTUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIE5DQkkgZGJHYVAgZGF0YWJhc2Ugb2YgZ2Vub3R5cGVzIGFuZCBwaGVub3R5cGVzOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPTExODEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1tdW5vZGVmaWNpZW5jeSBtdXRhdGlvbiBkYXRhYmFzZXMgKElEYmFzZXMpOyBIdW0uIE11dGF0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTEyMDAtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UkFQSUQ6IFJlc291cmNlIG9mIEFzaWFuIFByaW1hcnkgSW1tdW5vZGVmaWNpZW5jeSBEaXNlYXNlczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9RDg2My03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1kYlNOUDogdGhlIE5DQkkgZGF0YWJhc2Ugb2YgZ2VuZXRpYyB2YXJpYXRpb247IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTMwOC0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGxhbm5pbmcgdGhlIGh1bWFuIHZhcmlvbWUgcHJvamVjdDogdGhlIFNwYWluIHJlcG9ydDsgSHVtLiBNdXRhdC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz00OTYtNTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GLVNOUDogY29tcHV0YXRpb25hbGx5IHByZWRpY3RlZCBmdW5jdGlvbmFsIFNOUHMgZm9yIGRpc2Vhc2UgYXNzb2NpYXRpb24gc3R1ZGllczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9RDgyMC00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GQVNUU05QOiBhbiBhbHdheXMgdXAtdG8tZGF0ZSBhbmQgZXh0ZW5kYWJsZSBzZXJ2aWNlIGZvciBTTlAgZnVuY3Rpb24gYW5hbHlzaXMgYW5kIHByaW9yaXRpemF0aW9uOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz1XNjM1LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TTlBzM0Q6IGNhbmRpZGF0ZSBnZW5lIGFuZCBTTlAgc2VsZWN0aW9uIGZvciBhc3NvY2lhdGlvbiBzdHVkaWVzOyBCTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTE2NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNOUG5leHVzOiBhIHdlYiBkYXRhYmFzZSBmb3IgZnVuY3Rpb25hbCBhbm5vdGF0aW9uIG9mIG5ld2x5IGRpc2NvdmVyZWQgYW5kIHB1YmxpYyBkb21haW4gc2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz02NTUtNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGludGVncmF0ZWQgZGF0YWJhc2UtcGlwZWxpbmUgc3lzdGVtIGZvciBzdHVkeWluZyBzaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc21zIGFuZCBkaXNlYXNlczsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9pc3N1ZT1TdXBwbCAxMjsgIGNpdGF0aW9uX3BhZ2VzPVMxOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmF0bGFzIGRhdGFiYXNlLCBnZW5lcyBhbmQgZGV2ZWxvcG1lbnQgZGVmZWN0czsgQy4gUi4gQWNhZC4gU2NpLiBJSUk7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MzIxOyAgY2l0YXRpb25fcGFnZXM9ODA1LTE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TSUZUOiBwcmVkaWN0aW5nIGFtaW5vIGFjaWQgY2hhbmdlcyB0aGF0IGFmZmVjdCBwcm90ZWluIGZ1bmN0aW9uOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0zODEyLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUh1bWFuIG5vbi1zeW5vbnltb3VzIFNOUHM6IHNlcnZlciBhbmQgc3VydmV5OyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0zODk0LTkwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UEFOVEhFUjogYSBsaWJyYXJ5IG9mIHByb3RlaW4gZmFtaWxpZXMgYW5kIHN1YmZhbWlsaWVzIGluZGV4ZWQgYnkgZnVuY3Rpb247IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MjEyOS00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U05BUDogcHJlZGljdCBlZmZlY3Qgb2Ygbm9uLXN5bm9ueW1vdXMgcG9seW1vcnBoaXNtcyBvbiBmdW5jdGlvbjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9MzgyMy0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UE1VVDogYSB3ZWItYmFzZWQgdG9vbCBmb3IgdGhlIGFubm90YXRpb24gb2YgcGF0aG9sb2dpY2FsIG11dGF0aW9ucyBvbiBwcm90ZWluczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0zMTc2LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPW5zU05QQW5hbHl6ZXI6IGlkZW50aWZ5aW5nIGRpc2Vhc2UtYXNzb2NpYXRlZCBub25zeW5vbnltb3VzIHNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPVc0ODAtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGF0aG9nZW5pYyBvciBub3Q/IEFuZCBpZiBzbywgdGhlbiBob3c/IFN0dWR5aW5nIHRoZSBlZmZlY3RzIG9mIG1pc3NlbnNlIG11dGF0aW9ucyB1c2luZyBiaW9pbmZvcm1hdGljcyBtZXRob2RzOyBIdW0uIE11dGF0LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTcwMy0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlZGljdGluZyB0aGUgZWZmZWN0cyBvZiBhbWlubyBhY2lkIHN1YnN0aXR1dGlvbnMgb24gcHJvdGVpbiBmdW5jdGlvbjsgQW5udS4gUmV2LiBHZW5vbWljcyBIdW0uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9NjEtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVuaVByb3RLQi9Td2lzcy1Qcm90OyBNZXRob2RzIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDA2OyAgY2l0YXRpb25fcGFnZXM9ODktMTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYXNpYyBsb2NhbCBhbGlnbm1lbnQgc2VhcmNoIHRvb2w7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9MjE1OyAgY2l0YXRpb25fcGFnZXM9NDAzLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DTFVTVEFMIFc6IGltcHJvdmluZyB0aGUgc2Vuc2l0aXZpdHkgb2YgcHJvZ3Jlc3NpdmUgbXVsdGlwbGUgc2VxdWVuY2UgYWxpZ25tZW50IHRocm91Z2ggc2VxdWVuY2Ugd2VpZ2h0aW5nLCBwb3NpdGlvbi1zcGVjaWZpYyBnYXAgcGVuYWx0aWVzIGFuZCB3ZWlnaHQgbWF0cml4IGNob2ljZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9NDY3My04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFByb3RlaW4gRGF0YSBCYW5rOyBBY3RhIENyeXN0YWxsb2dyLiBEIEJpb2wuIENyeXN0YWxsb2dyLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT01ODsgIGNpdGF0aW9uX3BhZ2VzPTg5OS05MDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIHByb3RlaW4gc3RydWN0dXJlIG1vZGVsaW5nIG9mIGdlbmVzIGFuZCBnZW5vbWVzOyBBbm51LiBSZXYuIEJpb3BoeXMuIEJpb21vbC4gU3RydWN0LjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTI5MS0zMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0YXRpc3RpY2FsIHBvdGVudGlhbCBmb3IgYXNzZXNzbWVudCBhbmQgcHJlZGljdGlvbiBvZiBwcm90ZWluIHN0cnVjdHVyZXM7IFByb3RlaW4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTI1MDctMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVudmlyb25tZW50IGFuZCBleHBvc3VyZSB0byBzb2x2ZW50IG9mIHByb3RlaW4gYXRvbXMuIEx5c296eW1lIGFuZCBpbnN1bGluOyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTczOyAgY2l0YXRpb25fdm9sdW1lPTc5OyAgY2l0YXRpb25fcGFnZXM9MzUxLTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgRElTT1BSRUQgc2VydmVyIGZvciB0aGUgcHJlZGljdGlvbiBvZiBwcm90ZWluIGRpc29yZGVyOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTIxMzgtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG93YXJkcyBhIHN0cnVjdHVyYWwgYmFzaXMgb2YgaHVtYW4gbm9uLXN5bm9ueW1vdXMgc2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtczsgVHJlbmRzIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTE5OC0yMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBhbWluby1hY2lkIG11dGF0aW9uYWwgc3BlY3RydW0gb2YgaHVtYW4gZ2VuZXRpYyBkaXNlYXNlOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPVI3MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNOUHMsIHByb3RlaW4gc3RydWN0dXJlLCBhbmQgZGlzZWFzZTsgSHVtLiBNdXRhdC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0yNjMtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgZGlzZWFzZS1hc3NvY2lhdGVkIHNpbmdsZSBhbWlubyBhY2lkIHBvbHltb3JwaGlzbXMgaW4gdGVybXMgb2Ygc2VxdWVuY2UgYW5kIHN0cnVjdHVyZSBwcm9wZXJ0aWVzOyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTMxNTsgIGNpdGF0aW9uX3BhZ2VzPTc3MS04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Y29saVNOUCBkYXRhYmFzZSBzZXJ2ZXIgbWFwcGluZyBuc1NOUHMgb24gcHJvdGVpbiBzdHJ1Y3R1cmVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz1ENDA5LTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zZXJ2YXRpb24gb2YgaW50cmluc2ljIGRpc29yZGVyIGluIHByb3RlaW4gZG9tYWlucyBhbmQgZmFtaWxpZXM6IEkuIEEgZGF0YWJhc2Ugb2YgY29uc2VydmVkIHByZWRpY3RlZCBkaXNvcmRlcmVkIHJlZ2lvbnM7IEouIFByb3Rlb21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTg3OS04NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvdGVpbiBpbnRlcmFjdGlvbnMgaW4gaHVtYW4gZ2VuZXRpYyBkaXNlYXNlczsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1SOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNUQVQzIG11dGF0aW9ucyBpbiB0aGUgaHlwZXItSWdFIHN5bmRyb21lOyBOLiBFbmdsLiBKLiBNZWQuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM1NzsgIGNpdGF0aW9uX3BhZ2VzPTE2MDgtMTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURvbWluYW50LW5lZ2F0aXZlIG11dGF0aW9ucyBpbiB0aGUgRE5BLWJpbmRpbmcgZG9tYWluIG9mIFNUQVQzIGNhdXNlIGh5cGVyLUlnRSBzeW5kcm9tZTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ0ODsgIGNpdGF0aW9uX3BhZ2VzPTEwNTgtNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBQZmFtIHByb3RlaW4gZmFtaWxpZXMgZGF0YWJhc2U7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPUQyODEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGF0YWJhc2UgcmVzb3VyY2VzIG9mIHRoZSBOYXRpb25hbCBDZW50ZXIgZm9yIEJpb3RlY2hub2xvZ3kgSW5mb3JtYXRpb247IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zNzsgIGNpdGF0aW9uX3BhZ2VzPUQ1LTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IdW1hbiBQcm90ZWluIFJlZmVyZW5jZSBEYXRhYmFzZeKAlDIwMDkgdXBkYXRlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz1ENzY3LTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lQ2FyZHM6IGEgbm92ZWwgZnVuY3Rpb25hbCBnZW5vbWljcyBjb21wZW5kaXVtIHdpdGggYXV0b21hdGVkIGRhdGEgbWluaW5nIGFuZCBxdWVyeSByZWZvcm11bGF0aW9uIHN1cHBvcnQ7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9NjU2LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgYW4gb3Blbi1hY2Nlc3MgZGF0YWJhc2UgdGhhdCBpbnRlZ3JhdGVzIGNyb3NzLXJlZmVyZW5jZSBpbmZvcm1hdGlvbiBmcm9tIHRoZSB0cmFuc2NyaXB0b21lIGFuZCBwcm90ZW9tZSBvZiBpbW11bmUgY2VsbHM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MjkzNC00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9S0VHRzoga3lvdG8gZW5jeWNsb3BlZGlhIG9mIGdlbmVzIGFuZCBnZW5vbWVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0yNy0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhyZWUtZGltZW5zaW9uYWwgc3RydWN0dXJlIG9mIHRoZSBTdGF0M2JldGEgaG9tb2RpbWVyIGJvdW5kIHRvIEROQTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTM5NDsgIGNpdGF0aW9uX3BhZ2VzPTE0NS01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QkxBVOKAlHRoZSBCTEFTVC1saWtlIGFsaWdubWVudCB0b29sOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTY1Ni02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGZpcnN0IEtvcmVhbiBnZW5vbWUgc2VxdWVuY2UgYW5kIGFuYWx5c2lzOiBmdWxsIGdlbm9tZSBzZXF1ZW5jaW5nIGZvciBhIHNvY2lvLWV0aG5pYyBncm91cDsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0xNjIyLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xNy8zLzE5Ny8xMDEyMTc3L2RzcTAxMC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgQWx0aG91Z2ggbXV0YXRpb24gYW5hbHlzaXMgc2VydmVzIGFzIGEga2V5IHBhcnQgaW4gbWFraW5nIGEgZGVmaW5pdGl2ZSBkaWFnbm9zaXMgYWJvdXQgYSBnZW5ldGljIGRpc2Vhc2UsIGl0IHN0aWxsIHJlbWFpbnMgYSB0aW1lLWNvbnN1bWluZyBzdGVwIHRvIGkiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM1Mzg3OSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGS3xg5wpBcNDgi2/+7Et4l5zWcDmA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f3%2f197%2f353879" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f3%2f197%2f353879" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13687" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/17/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/3/1/m_cover.gif?Expires=2147483647&amp;Signature=zrlqOYlr8puhWaI06gf8sYMMP6iQcSaE6WnxD18CI1vM-vSYmV7zw1fL1UUm1LAo6OR8YaC2ow7Gg3gFyOuRxjrsW2e9seZ-Fc4W0zpKOKeLDqycZiol4B5gPhMqbsAY9k5dM2sXst3uhEwLyf9p6Clk-Cv9XLljwpdq37Ye9OThamAMiMSAoDxynPtt~5t-2ainSI295gFWPk3t9zFxKYx4zJXORsGXGnkmtct97jCcoDB0mHB0FSIfJFRj9SkO~ccPVDFJ5bkP0CQlUseB~SQw04UmN2w0van9EOcwauxu2Sl9Oe1AFiN92gYP2rZEbuX5qeGuIN6geNJvEhApPQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/17/3">
                        <div class="volume trailing-comma">Volume 17</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2010        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5042638">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5042638">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5042640">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5042640">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5042644">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5042644">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5042663">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5042663">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5042685">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5042685">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5042687">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5042687">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5042689">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5042689">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5042691">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5042691">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/17/3/185/354288">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/17/3/209/354504">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Mutation@A Glance: An Integrative Web Application for Analysing Mutations from Human Genetic Diseases
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Atsushi Hijikata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Laboratory for Immunogenomics</div>, <div class="insititution">RIKEN Research Center for Allergy and Immunology</div>, <div class="addr-line">1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Atsushi+Hijikata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hijikata A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hijikata A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Atsushi Hijikata</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rajesh Raju</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Research Unit for Immunoinformatics</div>, <div class="insititution">RIKEN Research Center for Allergy and Immunology</div>, <div class="addr-line">1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">3</div><div class="insititution">Institute of Bioinformatics, International Technology Park</div>, <div class="addr-line">Bangalore 560 066</div>, <div class="country">India</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Biotechnology and Bioinformatics</div>, <div class="insititution">Kuvempu University</div>, <div class="addr-line">Jnanasahyadri, Shimoga 577 451</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rajesh+Raju">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Raju R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Raju R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Rajesh Raju</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shivakumar Keerthikumar</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Research Unit for Immunoinformatics</div>, <div class="insititution">RIKEN Research Center for Allergy and Immunology</div>, <div class="addr-line">1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">3</div><div class="insititution">Institute of Bioinformatics, International Technology Park</div>, <div class="addr-line">Bangalore 560 066</div>, <div class="country">India</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Biotechnology and Bioinformatics</div>, <div class="insititution">Kuvempu University</div>, <div class="addr-line">Jnanasahyadri, Shimoga 577 451</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shivakumar+Keerthikumar">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Keerthikumar S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Keerthikumar S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shivakumar Keerthikumar</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Subhashri Ramabadran</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Research Unit for Immunoinformatics</div>, <div class="insititution">RIKEN Research Center for Allergy and Immunology</div>, <div class="addr-line">1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">3</div><div class="insititution">Institute of Bioinformatics, International Technology Park</div>, <div class="addr-line">Bangalore 560 066</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Subhashri+Ramabadran">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ramabadran S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ramabadran S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Subhashri Ramabadran</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Lavanya Balakrishnan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Research Unit for Immunoinformatics</div>, <div class="insititution">RIKEN Research Center for Allergy and Immunology</div>, <div class="addr-line">1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">3</div><div class="insititution">Institute of Bioinformatics, International Technology Park</div>, <div class="addr-line">Bangalore 560 066</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Lavanya+Balakrishnan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Balakrishnan L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Balakrishnan L&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Lavanya Balakrishnan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Suresh Kumar Ramadoss</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Research Unit for Immunoinformatics</div>, <div class="insititution">RIKEN Research Center for Allergy and Immunology</div>, <div class="addr-line">1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Suresh+Kumar+Ramadoss">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ramadoss S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ramadoss S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Suresh Kumar Ramadoss</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akhilesh Pandey</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Institute of Bioinformatics, International Technology Park</div>, <div class="addr-line">Bangalore 560 066</div>, <div class="country">India</div></div><div class="aff"><div class="label -label">5</div><div class="insititution">McKusick-Nathans Institute of Genetic Medicine, Johns Hopkins University School of Medicine</div>, <div class="addr-line">733 N. Broadway, BRB Room 527, Baltimore, MD 21205</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akhilesh+Pandey">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pandey A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Pandey A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Akhilesh Pandey</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sujatha Mohan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Research Unit for Immunoinformatics</div>, <div class="insititution">RIKEN Research Center for Allergy and Immunology</div>, <div class="addr-line">1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">3</div><div class="insititution">Institute of Bioinformatics, International Technology Park</div>, <div class="addr-line">Bangalore 560 066</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sujatha+Mohan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mohan S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mohan S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sujatha Mohan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Osamu Ohara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Laboratory for Immunogenomics</div>, <div class="insititution">RIKEN Research Center for Allergy and Immunology</div>, <div class="addr-line">1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">6</div><div class="addr-line">Laboratory of Genome Technology, Department of Human Genome Research</div>, <div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 45-503-9696. Fax. <div class="fax">+81 45-503-9694</div>. Email: <a href="mailto:oosamu@rcai.riken.jp" target="_blank">oosamu@rcai.riken.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Osamu+Ohara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohara O">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohara O&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Osamu Ohara</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 17, Issue 3, 1 June 2010, Pages 197–208, <a href='https://doi.org/10.1093/dnares/dsq010'>https://doi.org/10.1093/dnares/dsq010</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 April 2010</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">17 January 2010</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">08 March 2010</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="353879" href="/dnaresearch/article-pdf/17/3/197/1012177/dsq010.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Atsushi Hijikata, Rajesh Raju, Shivakumar Keerthikumar, Subhashri Ramabadran, Lavanya Balakrishnan, Suresh Kumar Ramadoss, Akhilesh Pandey, Sujatha Mohan, Osamu Ohara;  Mutation@A Glance: An Integrative Web Application for Analysing Mutations from Human Genetic Diseases, <em>DNA Research</em>, Volume 17, Issue 3, 1 June 2010, Pages 197–208, <a href='https://doi.org/10.1093/dnares/dsq010'>https://doi.org/10.1093/dnares/dsq010</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/353879?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/353879?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/353879?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/353879?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/353879?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/353879?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/353879?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Mutation%40A%20Glance%3A%20An%20Integrative%20Web%20Application%20for%20Analysing%20Mutations%20from%20Human%20Genetic%20Diseases&amp;publicationDate=2010-04-01&amp;volumeNum=17&amp;issueNum=3&amp;author=Hijikata%2C%20Atsushi%3B%20Raju%2C%20Rajesh&amp;startPage=197&amp;endPage=208&amp;contentId=10.1093%2Fdnares%2Fdsq010&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f3%2f197%2f353879" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5042638 id="5042638" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Although mutation analysis serves as a key part in making a definitive diagnosis about a genetic disease, it still remains a time-consuming step to interpret their biological implications through integration of various lines of archived information about genes in question. To expedite this evaluation step of disease-causing genetic variations, here we developed Mutation@A Glance (<a class="link link-uri" href="http://rapid.rcai.riken.jp/mutation/">http://rapid.rcai.riken.jp/mutation/</a>), a highly integrated web-based analysis tool for analysing human disease mutations; it implements a user-friendly graphical interface to visualize about 40 000 known disease-associated mutations and genetic polymorphisms from more than 2600 protein-coding human disease-causing genes. Mutation@A Glance locates already known genetic variation data individually on the nucleotide and the amino acid sequences and makes it possible to cross-reference them with tertiary and/or quaternary protein structures and various functional features associated with specific amino acid residues in the proteins. We showed that the disease-associated missense mutations had a stronger tendency to reside in positions relevant to the structure/function of proteins than neutral genetic variations. From a practical viewpoint, Mutation@A Glance could certainly function as a ‘one-stop’ analysis platform for newly determined DNA sequences, which enables us to readily identify and evaluate new genetic variations by integrating multiple lines of information about the disease-causing candidate genes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genetic disease&quot;">genetic disease</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="mutation">mutation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="polymorphism">polymorphism</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="bioinformatics">bioinformatics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;protein structure&quot;">protein structure</a></div>                            <h2 scrollto-destination=5042640 id="5042640" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Genetic diseases are caused by structural changes in genes and/or chromosomes. In the Online Mendelian Inheritance in Man (OMIM, <a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/sites/entrez?db=omim">http://www.ncbi.nlm.nih.gov/sites/entrez?db=omim</a>) database, more than 2200 genes are known to have mutations causing genetic diseases.<sup><a reveal-id="DSQ010C1" data-open="DSQ010C1" class="link link-ref link-reveal xref-bibr">1</a></sup> For instance, primary immunodeficiency diseases (PIDs) are caused by congenital defects in genes involved in the development and maintenance of the immune system,<sup><a reveal-id="DSQ010C2" data-open="DSQ010C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSQ010C3" data-open="DSQ010C3" class="link link-ref link-reveal xref-bibr">3</a></sup> and they can be diagnosed using mutation analysis that identifies pathogenic mutations in candidate PID genes. This process plays a critical role in improving the quality of life for PID patients.<sup><a reveal-id="DSQ010C4" data-open="DSQ010C4" class="link link-ref link-reveal xref-bibr">4</a></sup> In this regard, the recent advances in DNA sequencing technology will extremely expedite this process. Thus, the next bottleneck to be addressed is obviously how to clarify the associations between newly identified patient-specific genetic variations and disease phenotypes, even when familial disease history is absent. To eliminate the bottleneck in mutation analysis, we need a bioinformatics tool that would enable us to readily evaluate the impact of a genetic variation on the structure/function of a gene product at the molecular level. Towards this end, our first step was to develop an integrated ‘one-stop’ analysis platform where we could cross-reference multiple lines of information regarding known genetic variations, including a huge amount of non-synonymous (ns) single-nucleotide polymorphisms (nsSNPs) in healthy individuals,<sup><a reveal-id="DSQ010C5  DSQ010C7" data-open="DSQ010C5  DSQ010C7" class="link link-ref link-reveal xref-bibr">5–7</a></sup> in genes of interest.</p><p>Bioinformatics resources and methods played an indispensable role in creating this platform.<sup><a reveal-id="DSQ010C8  DSQ010C12" data-open="DSQ010C8  DSQ010C12" class="link link-ref link-reveal xref-bibr">8–12</a></sup> Although a number of databases regarding reported human disease mutations and SNPs have been already constructed,<sup><a reveal-id="DSQ010C13  DSQ010C25" data-open="DSQ010C13  DSQ010C25" class="link link-ref link-reveal xref-bibr">13–25</a></sup> these databases were launched as a static archive for genetic variation data, not necessarily an interactive tool for evaluating newly identified sequence variation data. Several computational algorithms for predicting the effects of ns substitutions on a corresponding protein have been developed using evolutionary and protein three-dimensional (3D) structure information.<sup><a reveal-id="DSQ010C26  DSQ010C31" data-open="DSQ010C26  DSQ010C31" class="link link-ref link-reveal xref-bibr">26–31</a></sup> However, despite public availability of these software/web servers, there are at least two hurdles, especially for clinical researchers to exploit them for the mutation analysis: (i) since these servers usually require information about the position of the genetic variation occurred in a submitted sequence as a query input, the users have to specify the variation position in the sequence before submitting the query; (ii) since these servers do not necessarily incorporate known disease-associated mutation data into their systems, the users have to manually compare their newly identified genetic variation data from patients with previously reported data. Thus, we thought it was important to integrate predictive bioinformatics tools, such as the one described above, with a comprehensive set of known genetic variation data, to create a ‘one-stop’ mutation analysis platform.<sup><a reveal-id="DSQ010C32" data-open="DSQ010C32" class="link link-ref link-reveal xref-bibr">32</a></sup></p><p>In this context, here we present Mutation@A Glance (<a class="link link-uri" href="http://rapid.rcai.riken.jp/mutation/">http://rapid.rcai.riken.jp/mutation/</a>), a new web-based integrated bioinformatics tool for analysing mutations from human genetic diseases. The user-friendly graphical interface of Mutation@A Glance makes it possible to allocate known disease-associated mutation data on the nucleotide and amino acid sequences of a gene of interest, and to link these mutation data to the 3D structure of the gene product along with various lines of information about the mutated amino acid residues (e.g. the extent of evolutional sequence conservation, post-translational modifications and molecular interactions). Furthermore, this tool enables users to identify and evaluate newly identified sequence variations in a query DNA sequence from a gene of interest by comparing them with known disease-associated mutation data and using the SIFT program,<sup><a reveal-id="DSQ010C26" data-open="DSQ010C26" class="link link-ref link-reveal xref-bibr">26</a></sup> which is one of the most accurate and widely used program to specifically predict the effects of ns substitutions based on evolutionary information for each residue position.<sup><a reveal-id="DSQ010C33" data-open="DSQ010C33" class="link link-ref link-reveal xref-bibr">33</a></sup> Therefore, Mutation@A Glance surely serves as a ‘one-stop’ informational platform to identify and evaluate new genetic variations by integrating multiple lines of information about the disease-causing candidate genes.</p>                            <h2 scrollto-destination=5042644 id="5042644" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5042645 id="5042645" class="section-title" data-legacy-id=s2a>2.1. Data resources for disease-associated genes and sequence variations</h3>
<p>Human disease-associated mutation data were obtained from the following three databases: OMIM (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/omim/">http://www.ncbi.nlm.nih.gov/omim/</a>),<sup><a reveal-id="DSQ010C1" data-open="DSQ010C1" class="link link-ref link-reveal xref-bibr">1</a></sup> UniProt (<a class="link link-uri" href="http://www.uniprot.org/">http://www.uniprot.org/</a>)<sup><a reveal-id="DSQ010C34" data-open="DSQ010C34" class="link link-ref link-reveal xref-bibr">34</a></sup> and RAPID (<a class="link link-uri" href="http://rapid.rcai.riken.jp/">http://rapid.rcai.riken.jp/</a>).<sup><a reveal-id="DSQ010C17" data-open="DSQ010C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Sequence variations that were associated with OMIM in the dbSNP database (Build 130, <a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/projects/SNP/">http://www.ncbi.nlm.nih.gov/projects/SNP/</a>)<sup><a reveal-id="DSQ010C18" data-open="DSQ010C18" class="link link-ref link-reveal xref-bibr">18</a></sup> were considered to be disease-associated mutations and other variations were considered non-disease associated. For the mutation data in the UniProt database, VARIANT features associated with diseases in the human entries were considered. RAPID is a molecular database that we have recently established for reported disease mutation data in genes causing PIDs.<sup><a reveal-id="DSQ010C17" data-open="DSQ010C17" class="link link-ref link-reveal xref-bibr">17</a></sup> The RAPID database is directly connected to our local server and the mutation data (as of August 2009) are retrieved using a Perl script. The human genome sequence (Build 36.3), RefSeq sequences for nucleotides and proteins of human were downloaded from the NCBI ftp site (ftp://ftp.ncbi.nlm.nih.gov/). Information regarding residue-wise functional features (Transmembrane helix, signal peptide, nucleotide binding, disulphide bond, metal binding, active site and post-translational modification site) was extracted from the human entries in the UniProt database. Information regarding the exon–intron structures of each gene was downloaded from the NCBI ftp site.</p>                            <h3 scrollto-destination=5042647 id="5042647" class="section-title" data-legacy-id=s2b>2.2. Calculation of sequence conservation in ns substitution sites</h3>
<p>Homologous protein sequences in other organisms to the human proteins encoded by disease-causing genes were identified using the BLAST program<sup><a reveal-id="DSQ010C35" data-open="DSQ010C35" class="link link-ref link-reveal xref-bibr">35</a></sup> against the RefSeq database (6 691 817 amino acid sequences) with a cut-off <em>E</em>-value of 10<sup>−4</sup>. If the sequence identity and the coverage between a sequence hit and the human were higher than 40% and 80%, respectively, the sequence was selected as a homologous sequence. When two or more sequences from an organism were found as homologous sequences, the sequence with the highest sequence identity was only considered. The homologous protein sequences from various organisms were aligned using the CLUSTAL W program.<sup><a reveal-id="DSQ010C36" data-open="DSQ010C36" class="link link-ref link-reveal xref-bibr">36</a></sup> A degree of sequence conservation at each amino acid position in the multiple sequence alignment (simply designated as ‘residue conservation’ in Fig. <a reveal-id="DSQ010F1" data-open="DSQ010F1" class="link link-reveal link-table xref-fig">1</a>) was defined as the ratio of (the number of the homologous protein sequences which carried an identical amino acid residue to that in the human sequence) to (the number of the aligned homologous protein sequences) at the specified position in the multiple sequence alignment. For example, if Ala appears in an aligned position in the human sequence and the corresponding positions in all of the other homologous sequences are also Ala, the residue conservation in this position is defined as 1.0. The frequency distribution of the residue conservations in disease-associated missense mutation or nsSNP positions for proteins analysed in this study was represented using bins of the interval of 0.2. The value in each bin was normalized by the frequency of the total number of residues in each bin.</p>                        <a id="5042649" scrollto-destination="5042649"></a>
<div id="DSQ010F1" data-id="DSQ010F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq010/2/m_dsq01001.jpeg?Expires=2147483647&Signature=mdge4uRw0BOhHZftNngO9lJuQEx~ePmya~zi2u1sswKJxZP9yOL-uJw~n5vJbghh8fmcjz9mVgqUKWmv8gSDfHfiLFy6jy4V3-Eobb98xFFP2NaDBfXy5z~rSH4V2TSflZZrVrQ4YtyGd1BgZ7EOYpB2c0Gpe~RS-HhEHr7vlFt4tmBKZvWIIfRofMtADQQ-4GBgYXCSnhJ3KR4VVQH0SxlMQWjdG-d8oJWcDQm4kDX36lYwkf4nRMr9c1gCXiwRJm8B66LsWqbTKNEtyiSmKhWXkRTufuNcdlcACLBvln3SdvcSyJuLPEzeGW~rMKMUNPq6I1NZ-rnuIV6jKoDDEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of frequency distributions of residue conservations in disease-associated missense mutations and nsSNPs. The vertical axis depicts the log-odds ratio of the frequency of ns substitution residue positions (disease-associated mutations or nsSNPs) to those of total number of residues in each residue conservation bin." path-from-xml="dsq01001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5042649/dsq01001.jpeg" path-from-xml="dsq01001.jpeg" target="_blank">View large</a><a section="5042649" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq010/2/dsq01001.jpeg?Expires=2147483647&Signature=I0NfDWLPP6E99U6nSvcwpnEd5LP50pllJZDT6NFasPwFfYY9F0o~6wNiVJ6IV2o8ly4NuDcOkX8kkwTiDUqfS8t21Owy5S4kkHPjsNNtW-29iIZNfYUMCMqUFbKlex1AHIDesv0o2k0rd3YC6Iah~2Xgntbagsznb4X8YdpWoAHgSxADgw8fY3iE5rPEEaDMh44MVNaGGh~aBFnz4G29Y1NR13XYv~6O7XPuAtEISxuImyuGn5wvryIvKiP~zOOlVuyWkaXUVy5D0ULl81x7yQKXzJsaMjBIr5M8pTJNOXf-uB6K6qDP3wsf~JGmqxoH4HcubPPixoOup33svVwQ8g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5042649&ar=353879&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of frequency distributions of residue conservations in disease-associated missense mutations and nsSNPs. The vertical axis depicts the log-odds ratio of the frequency of ns substitution residue positions (disease-associated mutations or nsSNPs) to those of total number of residues in each residue conservation bin.</p></div></div><div content-id="DSQ010F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq010/2/m_dsq01001.jpeg?Expires=2147483647&Signature=mdge4uRw0BOhHZftNngO9lJuQEx~ePmya~zi2u1sswKJxZP9yOL-uJw~n5vJbghh8fmcjz9mVgqUKWmv8gSDfHfiLFy6jy4V3-Eobb98xFFP2NaDBfXy5z~rSH4V2TSflZZrVrQ4YtyGd1BgZ7EOYpB2c0Gpe~RS-HhEHr7vlFt4tmBKZvWIIfRofMtADQQ-4GBgYXCSnhJ3KR4VVQH0SxlMQWjdG-d8oJWcDQm4kDX36lYwkf4nRMr9c1gCXiwRJm8B66LsWqbTKNEtyiSmKhWXkRTufuNcdlcACLBvln3SdvcSyJuLPEzeGW~rMKMUNPq6I1NZ-rnuIV6jKoDDEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of frequency distributions of residue conservations in disease-associated missense mutations and nsSNPs. The vertical axis depicts the log-odds ratio of the frequency of ns substitution residue positions (disease-associated mutations or nsSNPs) to those of total number of residues in each residue conservation bin." path-from-xml="dsq01001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5042649/dsq01001.jpeg" path-from-xml="dsq01001.jpeg" target="_blank">View large</a><a section="5042649" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq010/2/dsq01001.jpeg?Expires=2147483647&Signature=I0NfDWLPP6E99U6nSvcwpnEd5LP50pllJZDT6NFasPwFfYY9F0o~6wNiVJ6IV2o8ly4NuDcOkX8kkwTiDUqfS8t21Owy5S4kkHPjsNNtW-29iIZNfYUMCMqUFbKlex1AHIDesv0o2k0rd3YC6Iah~2Xgntbagsznb4X8YdpWoAHgSxADgw8fY3iE5rPEEaDMh44MVNaGGh~aBFnz4G29Y1NR13XYv~6O7XPuAtEISxuImyuGn5wvryIvKiP~zOOlVuyWkaXUVy5D0ULl81x7yQKXzJsaMjBIr5M8pTJNOXf-uB6K6qDP3wsf~JGmqxoH4HcubPPixoOup33svVwQ8g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5042649&ar=353879&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of frequency distributions of residue conservations in disease-associated missense mutations and nsSNPs. The vertical axis depicts the log-odds ratio of the frequency of ns substitution residue positions (disease-associated mutations or nsSNPs) to those of total number of residues in each residue conservation bin.</p></div></div>                            <h3 scrollto-destination=5042650 id="5042650" class="section-title" data-legacy-id=s2c>2.3. Protein 3D structure information</h3>
<p>Protein 3D structure data were downloaded from the Protein Data Bank (PDB, <a class="link link-uri" href="http://www.rcsb.org/pdb/">http://www.rcsb.org/pdb/</a>).<sup><a reveal-id="DSQ010C37" data-open="DSQ010C37" class="link link-ref link-reveal xref-bibr">37</a></sup> In cases where the 3D structure of a human protein had not yet been determined, we searched the available sequences in the PDB entries for a template structure for homology modelling using the BLAST program as described above. When the alignment of the human protein sequence and a known 3D structure showed &gt;30% identity and &gt;90% coverage, a homology model was built using the MODELLER package.<sup><a reveal-id="DSQ010C38" data-open="DSQ010C38" class="link link-ref link-reveal xref-bibr">38</a></sup> For each target, 20 model structures were generated and their reliabilities were assessed with the Discrete Optimized Protein Energy (DOPE) method.<sup><a reveal-id="DSQ010C39" data-open="DSQ010C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Eventually, the model with the best DOPE score was selected as the final model for each protein. Information about protein quaternary structures was also extracted from the PDB database. Entries from the PDB that contained information about the biological unit structure and entries with polypeptide chains showed &gt;85% identities with a human protein sequence were considered. When a distance of one atom in a residue in a given polypeptide chain was &lt;5.0 Å from that of another residue in the other polypeptide or nucleotide chain, the residue was considered to be located at a molecular interaction interface.</p>                            <h3 scrollto-destination=5042652 id="5042652" class="section-title" data-legacy-id=s2d>2.4. Solvent accessibility calculations</h3>
<p>The solvent accessibilities of the amino acid residues in a 3D modelled structure were calculated using a modification of the Shrake and Rupley method,<sup><a reveal-id="DSQ010C40" data-open="DSQ010C40" class="link link-ref link-reveal xref-bibr">40</a></sup> with a water molecule represented by a 1.4 Å radius sphere. The solvent accessibility is represented by values ranging from 0 to 1. The residue was considered as an exposed residue on the protein surface, if the solvent accessibility was &gt;0.25 and buried otherwise.</p>                            <h3 scrollto-destination=5042657 id="5042657" class="section-title" data-legacy-id=s2e>2.5. Disorder prediction</h3>
<p>We used the DISOPRED2 program<sup><a reveal-id="DSQ010C41" data-open="DSQ010C41" class="link link-ref link-reveal xref-bibr">41</a></sup> to analyse each amino acid sequence of a gene product and predict intrinsically unstructured (disordered) regions in the protein sequence. If the program predicted a region consisting of more than three amino acid residues in a sequence to be ‘disordered’, we assigned this region as an intrinsically unstructured one.</p>                            <h3 scrollto-destination=5042659 id="5042659" class="section-title" data-legacy-id=s2f>2.6. Predicting the effect of ns substitutions on proteins</h3>
<p>The effects of ns substitutions on a given protein were evaluated on a local server using the SIFT program<sup><a reveal-id="DSQ010C26" data-open="DSQ010C26" class="link link-ref link-reveal xref-bibr">26</a></sup> which predicts the effects of missense substitutions on a protein based on evolutionary information from homologous protein sequences.</p>                            <h3 scrollto-destination=5042661 id="5042661" class="section-title" data-legacy-id=s2g>2.7. System implementation</h3>
<p>At the server end, a set of common gateway interface programs was written in Perl and is running on an Apache web server. The information regarding the disease-associated genes and the sequence variations described above was integrated into a MySQL database implemented in the server. At the client end, JavaScript frameworks such as prototype.js (<a class="link link-uri" href="http://www.prototypejs.org/">http://www.prototypejs.org/</a>) and scriptaculous.js (<a class="link link-uri" href="http://script.aculo.us/">http://script.aculo.us/</a>) were used to make the user interface more interactive. Jmol, a Java applet (<a class="link link-uri" href="http://www.jmol.org/">http://www.jmol.org/</a>), was implemented for visualizing protein 3D structures in a web browser.</p>                            <h2 scrollto-destination=5042663 id="5042663" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5042664 id="5042664" class="section-title" data-legacy-id=s3a>3.1. Statistics of the sequence variation data on Mutation@A Glance</h3>
<p>From three data resources for human disease mutations, OMIM, UniProt and RAPID, we obtained 25 616 disease-associated mutations and 21 199 nsSNPs in 2656 human genes (Table <a reveal-id="DSQ010TB1" data-open="DSQ010TB1" class="link link-reveal link-table xref-fig">1</a>) and integrated into the local database. Functional classification of the proteins encoded by the disease-associated genes showed a wide variety of molecular functions such as metabolic enzymes, protein kinases, transcription factor/regulators and structural proteins (Table <a reveal-id="DSQ010TB1" data-open="DSQ010TB1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Because we have been actively analysing mutations found in patients of PIDs with paediatricians in Japan, we constructed RAPID and used it as our original data resource for genetic variations in genes responsible for PIDs.<sup><a reveal-id="DSQ010C17" data-open="DSQ010C17" class="link link-ref link-reveal xref-bibr">17</a></sup> RAPID contains manually curated mutation data from published literature, including nonsense (582 sites in 96 genes), frame-shift (851 sites in 101 genes) and insertion/deletion (85 sites in 42 genes) mutations as well as missense mutations (1564 sites in 116 genes) in the protein-coding regions of 155 PID genes (as of August 2009). For non-PID genes, we used two publicly available data sets from UniProt and OMIM. The UniProt database contains only missense mutation data (22 258 entries in 2614 genes). On the other hand, the OMIM database contains a large number of missense mutation (1899 entries in 556 genes) and a relatively small number of the other types of mutations (99 entries in 13 genes). The RAPID and the OMIM databases also contain 699 disease-associated mutation data in intronic regions of 147 genes that cause splice anomaly effects. Thus, the most frequent mutation type in our data sets was missense mutation (89% of the total entry) as reported in the previous study.<sup><a reveal-id="DSQ010C13" data-open="DSQ010C13" class="link link-ref link-reveal xref-bibr">13</a></sup></p>                        <a id="5042666" scrollto-destination="5042666"></a>
<div content-id="DSQ010TB1" class="table-modal"><div class="table-wrap"><div id="DSQ010TB1" class="table-wrap-title" data-id="DSQ010TB1"><div class="label -label">Table 1</div><div class="caption"><p>Functional classification of disease-associated gene products</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Molecular class </th><th>No. of genes </th><th>No. of mutations<sup>a</sup> </th><th>No. of nsSNPs </th></tr></thead><tbody><tr><td>Enzymes </td><td>410 </td><td>5406 (5003) </td><td>2476 </td></tr><tr><td>Protein kinases </td><td>258 </td><td>1947 (1340) </td><td>2452 </td></tr><tr><td>Transcription factor/regulator </td><td>239 </td><td>2889 (2743) </td><td>1502 </td></tr><tr><td>Structural proteins </td><td>132 </td><td>1588 (1377) </td><td>1800 </td></tr><tr><td>Cell surface receptors </td><td>123 </td><td>1271 (1165) </td><td>838 </td></tr><tr><td>Transport/cargo protein </td><td>116 </td><td>1617 (1411) </td><td>1139 </td></tr><tr><td>DNA/RNA binding proteins </td><td>97 </td><td>429 (369) </td><td>580 </td></tr><tr><td>Integral membrane protein </td><td>87 </td><td>446 (434) </td><td>698 </td></tr><tr><td>Channels </td><td>79 </td><td>958 (948) </td><td>639 </td></tr><tr><td>GTPase/GTPase regulators </td><td>71 </td><td>371 (351) </td><td>450 </td></tr><tr><td>Membrane transport protein </td><td>67 </td><td>755 (751) </td><td>523 </td></tr><tr><td>Immunity proteins </td><td>58 </td><td>496 (183) </td><td>423 </td></tr><tr><td>Extracellular matrix protein </td><td>53 </td><td>886 (884) </td><td>939 </td></tr><tr><td>Proteases </td><td>53 </td><td>345 (284) </td><td>368 </td></tr><tr><td>Cell adhesion molecules </td><td>52 </td><td>390 (363) </td><td>428 </td></tr><tr><td>Others </td><td>430 </td><td>4647 (4091) </td><td>3863 </td></tr><tr><td>Unclassified </td><td>331 </td><td>1175 (987) </td><td>2081 </td></tr><tr><td>Total </td><td>2656 </td><td>25 616 (22 684) </td><td>21 199 </td></tr></tbody></table></div><div content-id="DSQ010TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Molecular class </th><th>No. of genes </th><th>No. of mutations<sup>a</sup> </th><th>No. of nsSNPs </th></tr></thead><tbody><tr><td>Enzymes </td><td>410 </td><td>5406 (5003) </td><td>2476 </td></tr><tr><td>Protein kinases </td><td>258 </td><td>1947 (1340) </td><td>2452 </td></tr><tr><td>Transcription factor/regulator </td><td>239 </td><td>2889 (2743) </td><td>1502 </td></tr><tr><td>Structural proteins </td><td>132 </td><td>1588 (1377) </td><td>1800 </td></tr><tr><td>Cell surface receptors </td><td>123 </td><td>1271 (1165) </td><td>838 </td></tr><tr><td>Transport/cargo protein </td><td>116 </td><td>1617 (1411) </td><td>1139 </td></tr><tr><td>DNA/RNA binding proteins </td><td>97 </td><td>429 (369) </td><td>580 </td></tr><tr><td>Integral membrane protein </td><td>87 </td><td>446 (434) </td><td>698 </td></tr><tr><td>Channels </td><td>79 </td><td>958 (948) </td><td>639 </td></tr><tr><td>GTPase/GTPase regulators </td><td>71 </td><td>371 (351) </td><td>450 </td></tr><tr><td>Membrane transport protein </td><td>67 </td><td>755 (751) </td><td>523 </td></tr><tr><td>Immunity proteins </td><td>58 </td><td>496 (183) </td><td>423 </td></tr><tr><td>Extracellular matrix protein </td><td>53 </td><td>886 (884) </td><td>939 </td></tr><tr><td>Proteases </td><td>53 </td><td>345 (284) </td><td>368 </td></tr><tr><td>Cell adhesion molecules </td><td>52 </td><td>390 (363) </td><td>428 </td></tr><tr><td>Others </td><td>430 </td><td>4647 (4091) </td><td>3863 </td></tr><tr><td>Unclassified </td><td>331 </td><td>1175 (987) </td><td>2081 </td></tr><tr><td>Total </td><td>2656 </td><td>25 616 (22 684) </td><td>21 199 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The numbers in parentheses indicate the number of disease-associated missense mutations.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5042666" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ010TB1" class="table-wrap-title" data-id="DSQ010TB1"><div class="label -label">Table 1</div><div class="caption"><p>Functional classification of disease-associated gene products</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Molecular class </th><th>No. of genes </th><th>No. of mutations<sup>a</sup> </th><th>No. of nsSNPs </th></tr></thead><tbody><tr><td>Enzymes </td><td>410 </td><td>5406 (5003) </td><td>2476 </td></tr><tr><td>Protein kinases </td><td>258 </td><td>1947 (1340) </td><td>2452 </td></tr><tr><td>Transcription factor/regulator </td><td>239 </td><td>2889 (2743) </td><td>1502 </td></tr><tr><td>Structural proteins </td><td>132 </td><td>1588 (1377) </td><td>1800 </td></tr><tr><td>Cell surface receptors </td><td>123 </td><td>1271 (1165) </td><td>838 </td></tr><tr><td>Transport/cargo protein </td><td>116 </td><td>1617 (1411) </td><td>1139 </td></tr><tr><td>DNA/RNA binding proteins </td><td>97 </td><td>429 (369) </td><td>580 </td></tr><tr><td>Integral membrane protein </td><td>87 </td><td>446 (434) </td><td>698 </td></tr><tr><td>Channels </td><td>79 </td><td>958 (948) </td><td>639 </td></tr><tr><td>GTPase/GTPase regulators </td><td>71 </td><td>371 (351) </td><td>450 </td></tr><tr><td>Membrane transport protein </td><td>67 </td><td>755 (751) </td><td>523 </td></tr><tr><td>Immunity proteins </td><td>58 </td><td>496 (183) </td><td>423 </td></tr><tr><td>Extracellular matrix protein </td><td>53 </td><td>886 (884) </td><td>939 </td></tr><tr><td>Proteases </td><td>53 </td><td>345 (284) </td><td>368 </td></tr><tr><td>Cell adhesion molecules </td><td>52 </td><td>390 (363) </td><td>428 </td></tr><tr><td>Others </td><td>430 </td><td>4647 (4091) </td><td>3863 </td></tr><tr><td>Unclassified </td><td>331 </td><td>1175 (987) </td><td>2081 </td></tr><tr><td>Total </td><td>2656 </td><td>25 616 (22 684) </td><td>21 199 </td></tr></tbody></table></div><div content-id="DSQ010TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Molecular class </th><th>No. of genes </th><th>No. of mutations<sup>a</sup> </th><th>No. of nsSNPs </th></tr></thead><tbody><tr><td>Enzymes </td><td>410 </td><td>5406 (5003) </td><td>2476 </td></tr><tr><td>Protein kinases </td><td>258 </td><td>1947 (1340) </td><td>2452 </td></tr><tr><td>Transcription factor/regulator </td><td>239 </td><td>2889 (2743) </td><td>1502 </td></tr><tr><td>Structural proteins </td><td>132 </td><td>1588 (1377) </td><td>1800 </td></tr><tr><td>Cell surface receptors </td><td>123 </td><td>1271 (1165) </td><td>838 </td></tr><tr><td>Transport/cargo protein </td><td>116 </td><td>1617 (1411) </td><td>1139 </td></tr><tr><td>DNA/RNA binding proteins </td><td>97 </td><td>429 (369) </td><td>580 </td></tr><tr><td>Integral membrane protein </td><td>87 </td><td>446 (434) </td><td>698 </td></tr><tr><td>Channels </td><td>79 </td><td>958 (948) </td><td>639 </td></tr><tr><td>GTPase/GTPase regulators </td><td>71 </td><td>371 (351) </td><td>450 </td></tr><tr><td>Membrane transport protein </td><td>67 </td><td>755 (751) </td><td>523 </td></tr><tr><td>Immunity proteins </td><td>58 </td><td>496 (183) </td><td>423 </td></tr><tr><td>Extracellular matrix protein </td><td>53 </td><td>886 (884) </td><td>939 </td></tr><tr><td>Proteases </td><td>53 </td><td>345 (284) </td><td>368 </td></tr><tr><td>Cell adhesion molecules </td><td>52 </td><td>390 (363) </td><td>428 </td></tr><tr><td>Others </td><td>430 </td><td>4647 (4091) </td><td>3863 </td></tr><tr><td>Unclassified </td><td>331 </td><td>1175 (987) </td><td>2081 </td></tr><tr><td>Total </td><td>2656 </td><td>25 616 (22 684) </td><td>21 199 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The numbers in parentheses indicate the number of disease-associated missense mutations.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5042666" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5042667 id="5042667" class="section-title" data-legacy-id=s3b>3.2. Evolutionary, structural and functional features of the ns substitution positions</h3>
<p>In general, disease-associated missense mutations tend to occur at evolutionarily conserved positions, because these positions are usually essential for the structure and/or function of a protein.<sup><a reveal-id="DSQ010C26" data-open="DSQ010C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSQ010C42" data-open="DSQ010C42" class="link link-ref link-reveal xref-bibr">42</a>,<a reveal-id="DSQ010C43" data-open="DSQ010C43" class="link link-ref link-reveal xref-bibr">43</a></sup> To verify this using the up-dated data set, we compared the frequency of disease-associated missense mutation sites (19 128 unique positions in 2622 genes) in each residue conservation bin with that of nsSNP sites (20 605 positions in 2494 genes) (Fig. <a reveal-id="DSQ010F1" data-open="DSQ010F1" class="link link-reveal link-table xref-fig">1</a>). The results indicated that the previously reported tendency was still true for the 2622 genes in our data set; the disease-associated mutation sites were preferably appeared in the highest residue conservation bin, while nsSNP sites showed the opposite trend (Fig. <a reveal-id="DSQ010F1" data-open="DSQ010F1" class="link link-reveal link-table xref-fig">1</a>). Next, we cross-referenced amino acid positions of the disease-associated missense mutations and nsSNPs to the functional features and 3D structures of the protein data in Mutation@A Glance. We classified these positions in terms of their functional features in a protein (annotated in the UniProt databases; Table <a reveal-id="DSQ010TB2" data-open="DSQ010TB2" class="link link-reveal link-table xref-fig">2</a>). More disease-associated missense mutations were found in the positions annotated to have some functional features, except in the ‘signal peptides’ and ‘post-translational modification sites’, than nsSNPs. Using a homology modelling technique, we mapped 10 939 out of 19 128 disease-associated mutation sites (57.2%) to protein 3D structures (Fig. <a reveal-id="DSQ010F2" data-open="DSQ010F2" class="link link-reveal link-table xref-fig">2</a>). Of these sites, 6616 sites (60.4%) were located in regions buried in protein structures (solvent accessibility &lt;0.25). In the same way, 7106 out of 20 605 nsSNP sites (34.4%) were mapped to 3D structures, and 4258 sites (59.9%) were located on the surfaces of proteins (Fig. <a reveal-id="DSQ010F2" data-open="DSQ010F2" class="link link-reveal link-table xref-fig">2</a>A). This observation is basically consistent with the previous findings from structural analysis.<sup><a reveal-id="DSQ010C44" data-open="DSQ010C44" class="link link-ref link-reveal xref-bibr">44</a>–<a reveal-id="DSQ010C46" data-open="DSQ010C46" class="link link-ref link-reveal xref-bibr">46</a></sup> Interestingly, nsSNP sites were located in regions predicted as intrinsically disordered at a three times higher frequency than disease-associated mutation sites (Fig. <a reveal-id="DSQ010F2" data-open="DSQ010F2" class="link link-reveal link-table xref-fig">2</a>A). This might be ascribed to the observation that conservation in the intrinsically disordered regions is relatively lower than that in ordered regions.<sup><a reveal-id="DSQ010C47" data-open="DSQ010C47" class="link link-ref link-reveal xref-bibr">47</a></sup></p>                        <a id="5042670" scrollto-destination="5042670"></a>
<div content-id="DSQ010TB2" class="table-modal"><div class="table-wrap"><div id="DSQ010TB2" class="table-wrap-title" data-id="DSQ010TB2"><div class="label -label">Table 2</div><div class="caption"><p>Structural and functional loci of mutation/nsSNP sites</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Property </th><th>Disease mutations </th><th>nsSNPs </th></tr></thead><tbody><tr><td>Transmembrane helix </td><td>1283 </td><td>648 </td></tr><tr><td>Nucleotide binding </td><td>670 </td><td>102 </td></tr><tr><td>Disulfide bond </td><td>385 </td><td>39 </td></tr><tr><td>Metal binding site </td><td>226 </td><td>147 </td></tr><tr><td>Signal peptide </td><td>101 </td><td>262 </td></tr><tr><td>Post translational modification site </td><td>97 </td><td>104 </td></tr><tr><td>Binding site<sup>a</sup> </td><td>48 </td><td>12 </td></tr><tr><td>Active site<sup>a</sup> </td><td>25 </td><td>6 </td></tr></tbody></table></div><div content-id="DSQ010TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Property </th><th>Disease mutations </th><th>nsSNPs </th></tr></thead><tbody><tr><td>Transmembrane helix </td><td>1283 </td><td>648 </td></tr><tr><td>Nucleotide binding </td><td>670 </td><td>102 </td></tr><tr><td>Disulfide bond </td><td>385 </td><td>39 </td></tr><tr><td>Metal binding site </td><td>226 </td><td>147 </td></tr><tr><td>Signal peptide </td><td>101 </td><td>262 </td></tr><tr><td>Post translational modification site </td><td>97 </td><td>104 </td></tr><tr><td>Binding site<sup>a</sup> </td><td>48 </td><td>12 </td></tr><tr><td>Active site<sup>a</sup> </td><td>25 </td><td>6 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>As defined in UniProt database (described in the text).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5042670" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ010TB2" class="table-wrap-title" data-id="DSQ010TB2"><div class="label -label">Table 2</div><div class="caption"><p>Structural and functional loci of mutation/nsSNP sites</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Property </th><th>Disease mutations </th><th>nsSNPs </th></tr></thead><tbody><tr><td>Transmembrane helix </td><td>1283 </td><td>648 </td></tr><tr><td>Nucleotide binding </td><td>670 </td><td>102 </td></tr><tr><td>Disulfide bond </td><td>385 </td><td>39 </td></tr><tr><td>Metal binding site </td><td>226 </td><td>147 </td></tr><tr><td>Signal peptide </td><td>101 </td><td>262 </td></tr><tr><td>Post translational modification site </td><td>97 </td><td>104 </td></tr><tr><td>Binding site<sup>a</sup> </td><td>48 </td><td>12 </td></tr><tr><td>Active site<sup>a</sup> </td><td>25 </td><td>6 </td></tr></tbody></table></div><div content-id="DSQ010TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Property </th><th>Disease mutations </th><th>nsSNPs </th></tr></thead><tbody><tr><td>Transmembrane helix </td><td>1283 </td><td>648 </td></tr><tr><td>Nucleotide binding </td><td>670 </td><td>102 </td></tr><tr><td>Disulfide bond </td><td>385 </td><td>39 </td></tr><tr><td>Metal binding site </td><td>226 </td><td>147 </td></tr><tr><td>Signal peptide </td><td>101 </td><td>262 </td></tr><tr><td>Post translational modification site </td><td>97 </td><td>104 </td></tr><tr><td>Binding site<sup>a</sup> </td><td>48 </td><td>12 </td></tr><tr><td>Active site<sup>a</sup> </td><td>25 </td><td>6 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>As defined in UniProt database (described in the text).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5042670" target="_blank">View Large</a></div></div>                        <a id="5042671" scrollto-destination="5042671"></a>
<div id="DSQ010F2" data-id="DSQ010F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq010/2/m_dsq01002.jpeg?Expires=2147483647&Signature=iXMtqJowWZCnd8Pad7WRQr4u3wsAAiDpDqZNZhVpiui3DmGOCKOIjRA-3G3dyGUEwZ4Wrl5bX20RRrLKpO7hE4nkL4h~ks0WljgN~pfKkEqKLrz6CkoM6tQVSQLol~dg17Sf-0QV9VAM9uv0pigeXUaU6H1uNEWLl9bO1Qpqu2daitnuFyFP8hwc3oS9-HrYVUU5srkTPK~cJGd-Cmo9Up34mYXui6RQ439~R6hA6t8AV1fRccxHzWbIujWutDnfYBQnZh22u7XPIk~c6uir81HKpCK~fxQ~PXgk7odz1EPDxoxzqKMX9v4Gja9h0CCeQSK9k7HM-l50s9L-CETrZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Classification of disease-associated mutations and nsSNPs according to their location on protein 3D structure. (A) The numbers in the pie charts depict those of ns substitution positions. (B) Proportion of ns substitution positions in the disease-associated mutations or nsSNPs that were located on the interface of the experimentally determined quaternary structures." path-from-xml="dsq01002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5042671/dsq01002.jpeg" path-from-xml="dsq01002.jpeg" target="_blank">View large</a><a section="5042671" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq010/2/dsq01002.jpeg?Expires=2147483647&Signature=r-70jT~a5sPiEWmUdk7lo-o73n1RZ-MW3JKYwXF4Xj9dFeiJtT~Hvqn4VFkY6LZlxCrB-9~OwOAcQsdt3gNbSkYDCcTK1Dt3-6ODoI8d5XGrVGLbGklmnkQM6y7svdRw04jZbNTwZYK4zMQRZ1GyEGlp~E5U9rrKXObIxRQwd~5H2DzheauzQTUGR1fBpvVsvxZq6Y-0kEpj3wBmBh2AUa9ALXrbSx1MgcqXq8qWLObiBuUy6BqVEAXZAlDLchQv9ROj94lNLq1WPCWCaG0R8lPuHuT2YMwFBiGpTbrk8axrpQ8FlD03jd1f34ifM70PFFqPcJnsWVjHk30ILcrMpA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5042671&ar=353879&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Classification of disease-associated mutations and nsSNPs according to their location on protein 3D structure. (A) The numbers in the pie charts depict those of ns substitution positions. (B) Proportion of ns substitution positions in the disease-associated mutations or nsSNPs that were located on the interface of the experimentally determined quaternary structures.</p></div></div><div content-id="DSQ010F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq010/2/m_dsq01002.jpeg?Expires=2147483647&Signature=iXMtqJowWZCnd8Pad7WRQr4u3wsAAiDpDqZNZhVpiui3DmGOCKOIjRA-3G3dyGUEwZ4Wrl5bX20RRrLKpO7hE4nkL4h~ks0WljgN~pfKkEqKLrz6CkoM6tQVSQLol~dg17Sf-0QV9VAM9uv0pigeXUaU6H1uNEWLl9bO1Qpqu2daitnuFyFP8hwc3oS9-HrYVUU5srkTPK~cJGd-Cmo9Up34mYXui6RQ439~R6hA6t8AV1fRccxHzWbIujWutDnfYBQnZh22u7XPIk~c6uir81HKpCK~fxQ~PXgk7odz1EPDxoxzqKMX9v4Gja9h0CCeQSK9k7HM-l50s9L-CETrZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Classification of disease-associated mutations and nsSNPs according to their location on protein 3D structure. (A) The numbers in the pie charts depict those of ns substitution positions. (B) Proportion of ns substitution positions in the disease-associated mutations or nsSNPs that were located on the interface of the experimentally determined quaternary structures." path-from-xml="dsq01002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5042671/dsq01002.jpeg" path-from-xml="dsq01002.jpeg" target="_blank">View large</a><a section="5042671" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq010/2/dsq01002.jpeg?Expires=2147483647&Signature=r-70jT~a5sPiEWmUdk7lo-o73n1RZ-MW3JKYwXF4Xj9dFeiJtT~Hvqn4VFkY6LZlxCrB-9~OwOAcQsdt3gNbSkYDCcTK1Dt3-6ODoI8d5XGrVGLbGklmnkQM6y7svdRw04jZbNTwZYK4zMQRZ1GyEGlp~E5U9rrKXObIxRQwd~5H2DzheauzQTUGR1fBpvVsvxZq6Y-0kEpj3wBmBh2AUa9ALXrbSx1MgcqXq8qWLObiBuUy6BqVEAXZAlDLchQv9ROj94lNLq1WPCWCaG0R8lPuHuT2YMwFBiGpTbrk8axrpQ8FlD03jd1f34ifM70PFFqPcJnsWVjHk30ILcrMpA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5042671&ar=353879&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Classification of disease-associated mutations and nsSNPs according to their location on protein 3D structure. (A) The numbers in the pie charts depict those of ns substitution positions. (B) Proportion of ns substitution positions in the disease-associated mutations or nsSNPs that were located on the interface of the experimentally determined quaternary structures.</p></div></div><p>Proteins function with other molecules in molecular networks (e.g. signalling pathways) in many cases. Hence, the effects of mutations on molecular interactions must be intriguing in mutation analysis.<sup><a reveal-id="DSQ010C48" data-open="DSQ010C48" class="link link-ref link-reveal xref-bibr">48</a></sup> We thus analysed whether or not the missense mutation positions were located in the molecular interaction sites based on the quaternary protein structures available from the PDB. Consequently, 714 out of 1738 disease-associated mutation sites (41.1%) were found to locate at the interfaces of 474 distinct proteins known to be involved in protein complex structures (Fig. <a reveal-id="DSQ010F2" data-open="DSQ010F2" class="link link-reveal link-table xref-fig">2</a>B; see Section 2.3). In contrast, the same was true for only 346 out of 1128 nsSNP sites (30.7%) in 447 genes. We confirmed that the frequency of disease-associated mutation sites located at the molecular interaction interface was significantly higher than that of nsSNP sites by χ<sup>2</sup> test (<em>P</em> &lt; 0.01). These results implicated that ns substitutions at positions involved in the molecular interaction tend to be disease-related as we expected.</p>                            <h3 scrollto-destination=5042673 id="5042673" class="section-title" data-legacy-id=s3c>3.3. The user interface for visualizing sequence variations</h3>
<p>Figure <a reveal-id="DSQ010F3" data-open="DSQ010F3" class="link link-reveal link-table xref-fig">3</a> shows the front page of the Mutation@A Glance website. It has two types of query forms, for visualizing known disease mutation data (Fig. <a reveal-id="DSQ010F3" data-open="DSQ010F3" class="link link-reveal link-table xref-fig">3</a>A) and for evaluating novel genetic variations in query DNA sequences (Fig. <a reveal-id="DSQ010F3" data-open="DSQ010F3" class="link link-reveal link-table xref-fig">3</a>B). For the visualization, a user inputs a given gene symbol of interest in the form. When the user enters some characters in the form, a list of gene names containing the input character string is shown to assist the user input. In addition, a user can also search for the gene name of interest from an entire list of genes available in Mutation@A Glance, which is displayed by clicking ‘Select from List’ button (Fig. <a reveal-id="DSQ010F3" data-open="DSQ010F3" class="link link-reveal link-table xref-fig">3</a>A). Just as information for users, the mutation data set used for each gene is noted near the ‘Select from List’ button. Figure <a reveal-id="DSQ010F4" data-open="DSQ010F4" class="link link-reveal link-table xref-fig">4</a> shows sample screenshots for the <em>STAT3</em> gene, which is known to be causative to hyper-IgE syndrome (HIES).<sup><a reveal-id="DSQ010C49" data-open="DSQ010C49" class="link link-ref link-reveal xref-bibr">49</a>,<a reveal-id="DSQ010C50" data-open="DSQ010C50" class="link link-ref link-reveal xref-bibr">50</a></sup> At the DNA level, positions of the disease-associated mutations, including substitution, insertion and deletion, as well as SNPs are shown on a set of exon sequences or genomic DNA sequence with/without the open-reading frame for the gene of interest (Fig. <a reveal-id="DSQ010F4" data-open="DSQ010F4" class="link link-reveal link-table xref-fig">4</a>A). If two or more alternative transcripts exist in the RefSeq database, the genetic variation data are allocated on the reference sequence that encodes the longest amino acid sequence among the alternative transcripts whereas all the alternative transcripts are indicated in the top panel of the genomic structure. At the protein level, the disease-associated mutation and SNP sites are highlighted in the primary structure of the gene products along with available functional annotation information of the amino acid residues from the UniProt database (e.g. enzymatic active sites and post-translational modification sites, etc.) (Fig. <a reveal-id="DSQ010F4" data-open="DSQ010F4" class="link link-reveal link-table xref-fig">4</a>B). Information regarding conserved domain from Pfam (<a class="link link-uri" href="http://pfam.sanger.ac.uk/">http://pfam.sanger.ac.uk/</a>)<sup><a reveal-id="DSQ010C51" data-open="DSQ010C51" class="link link-ref link-reveal xref-bibr">51</a></sup> and predicted intrinsically disordered regions are also displayed. When 3D structure information for the protein is available, the positions of mutation and SNP data can be viewed on the monomer or complex 3D structures with the Jmol applet (Fig. <a reveal-id="DSQ010F4" data-open="DSQ010F4" class="link link-reveal link-table xref-fig">4</a>B). Detailed information about nucleotide or amino acid residues of interest is displayed in another window after clicking on a residue (Fig. <a reveal-id="DSQ010F4" data-open="DSQ010F4" class="link link-reveal link-table xref-fig">4</a>C). In particular, at the protein level, an amino acid residue becomes highlighted in the 3D structure when clicking on it (Fig. <a reveal-id="DSQ010F4" data-open="DSQ010F4" class="link link-reveal link-table xref-fig">4</a>B). The amino acid sequence of human can be compared with those of other organisms by clicking ‘Multiple Alignment’ button (Fig. <a reveal-id="DSQ010F4" data-open="DSQ010F4" class="link link-reveal link-table xref-fig">4</a>D). The representation of the 3D structure can be selected from two model types (ribbon or space-filling models) and three colouring types (by rainbow, highlighting mutation positions or residue conservation) (Fig. <a reveal-id="DSQ010F4" data-open="DSQ010F4" class="link link-reveal link-table xref-fig">4</a>E). The ‘External Links’ button provides links to NCBI Entrez Gene (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/sites/entrez?db=gene">http://www.ncbi.nlm.nih.gov/sites/entrez?db=gene</a>)<sup><a reveal-id="DSQ010C52" data-open="DSQ010C52" class="link link-ref link-reveal xref-bibr">52</a></sup> for general information regarding the gene, Human Protein Reference Database (<a class="link link-uri" href="http://www.hprd.org/">http://www.hprd.org/</a>)<sup><a reveal-id="DSQ010C53" data-open="DSQ010C53" class="link link-ref link-reveal xref-bibr">53</a></sup> for information about the gene product, GeneCards (<a class="link link-uri" href="http://www.genecards.org/">http://www.genecards.org/</a>),<sup><a reveal-id="DSQ010C54" data-open="DSQ010C54" class="link link-ref link-reveal xref-bibr">54</a></sup> the Reference Database of Immune Cells (<a class="link link-uri" href="http://refdic.rcai.riken.jp/">http://refdic.rcai.riken.jp/</a>)<sup><a reveal-id="DSQ010C55" data-open="DSQ010C55" class="link link-ref link-reveal xref-bibr">55</a></sup> for gene expression profiling data and the KEGG pathway (<a class="link link-uri" href="http://www.genome.jp/">http://www.genome.jp/</a>)<sup><a reveal-id="DSQ010C56" data-open="DSQ010C56" class="link link-ref link-reveal xref-bibr">56</a></sup> for pathways involving this gene (Fig. <a reveal-id="DSQ010F4" data-open="DSQ010F4" class="link link-reveal link-table xref-fig">4</a>F). By using this visualization facility, mapping amino acid positions of known ns substitutions on the crystal structure of the STAT3–DNA complex (PDB code: 1bg1)<sup><a reveal-id="DSQ010C57" data-open="DSQ010C57" class="link link-ref link-reveal xref-bibr">57</a></sup> revealed that the disease-associated missense mutation residue positions were spatially located at the interface of the homodimer or at the DNA binding site, whereas the nsSNP residue positions were located on a surface outside of the molecular interaction sites (Fig. <a reveal-id="DSQ010F5" data-open="DSQ010F5" class="link link-reveal link-table xref-fig">5</a>). This suggests that disease-causative missense mutations in <em>STAT3</em> directly affect the protein–protein and/or protein–DNA interaction as reported previously.<sup><a reveal-id="DSQ010C49" data-open="DSQ010C49" class="link link-ref link-reveal xref-bibr">49</a>,<a reveal-id="DSQ010C50" data-open="DSQ010C50" class="link link-ref link-reveal xref-bibr">50</a></sup> This is a good demonstration how Mutation@A Glance could help us interpret mutation effects at the molecular level.</p>                        <a id="5042675" scrollto-destination="5042675"></a>
<div id="DSQ010F3" data-id="DSQ010F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq010/2/m_dsq01003.jpeg?Expires=2147483647&Signature=Ub4S7xaNdpmcumpe69UDakv4iO5mrH1o61NJ6DCFKf-~FPFGk3E0KCVLlOwmH2nEyHItnGtna3vxBd~GqBFtgImTmxook5SOvlENHu-~8E1gSvte0E1XGgARTSgCeZ~-OnnRMxVJeJygLh-ZH2xnYyxoRvOx6K5iZ~kvui~-pjCTarM8TqOaY1kp7GTFIaZnAksyzKWs89EiSa1OR3z9pSGDCg-4EQYoP53lHoa4jN1iMQyPKH4EyMQUJrhFuMKjftwUdE2NGwYRzdR64W8qqu3d84wgmruSWp0dpTL8G2nA9tkfKFP5YOET04MTU-RzBL-wA7fMOwnRayi8ihfIsg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The front page of Mutation@A Glance. There are two types of query interface for (A) browsing known mutation data and (B) evaluating novel sequence variations in DNA sequences of interest. See the main text for details of the mutation data available in Mutation@A Glance." path-from-xml="dsq01003.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5042675/dsq01003.jpeg" path-from-xml="dsq01003.jpeg" target="_blank">View large</a><a section="5042675" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq010/2/dsq01003.jpeg?Expires=2147483647&Signature=mqN7WrmnttjW5th8-hll1j0kFyO5VR4X2RavuviiQl0oC40rA29QmG3Ltj0QySDKaWbxoIZr0DDdflzDUlxINaWjllB51xfGqrjlG6L~BAO9sRS-Lf0OZq1P68yHxJx4iAxORNJC0GK9y64WVFfimvsU3fNJ~Pqypgavuyei9s7gtV3DMbpThaH6SrCxUqVQFsGfLxOL~lucPQNziMPO9x9dbBeZ0IWshUmDEj0KzdxXRfI5a5LwF5OTZhPF0DotRXhRJ3QW4FbdJlEY6PDvvoVftvvO-73IoCruokpPQYEH9wZNBb70xNvqRJGJShphDNdau7HvzH8whqJcgSHG~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5042675&ar=353879&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01003.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The front page of Mutation@A Glance. There are two types of query interface for (A) browsing known mutation data and (B) evaluating novel sequence variations in DNA sequences of interest. See the main text for details of the mutation data available in Mutation@A Glance.</p></div></div><div content-id="DSQ010F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq010/2/m_dsq01003.jpeg?Expires=2147483647&Signature=Ub4S7xaNdpmcumpe69UDakv4iO5mrH1o61NJ6DCFKf-~FPFGk3E0KCVLlOwmH2nEyHItnGtna3vxBd~GqBFtgImTmxook5SOvlENHu-~8E1gSvte0E1XGgARTSgCeZ~-OnnRMxVJeJygLh-ZH2xnYyxoRvOx6K5iZ~kvui~-pjCTarM8TqOaY1kp7GTFIaZnAksyzKWs89EiSa1OR3z9pSGDCg-4EQYoP53lHoa4jN1iMQyPKH4EyMQUJrhFuMKjftwUdE2NGwYRzdR64W8qqu3d84wgmruSWp0dpTL8G2nA9tkfKFP5YOET04MTU-RzBL-wA7fMOwnRayi8ihfIsg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The front page of Mutation@A Glance. There are two types of query interface for (A) browsing known mutation data and (B) evaluating novel sequence variations in DNA sequences of interest. See the main text for details of the mutation data available in Mutation@A Glance." path-from-xml="dsq01003.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5042675/dsq01003.jpeg" path-from-xml="dsq01003.jpeg" target="_blank">View large</a><a section="5042675" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq010/2/dsq01003.jpeg?Expires=2147483647&Signature=mqN7WrmnttjW5th8-hll1j0kFyO5VR4X2RavuviiQl0oC40rA29QmG3Ltj0QySDKaWbxoIZr0DDdflzDUlxINaWjllB51xfGqrjlG6L~BAO9sRS-Lf0OZq1P68yHxJx4iAxORNJC0GK9y64WVFfimvsU3fNJ~Pqypgavuyei9s7gtV3DMbpThaH6SrCxUqVQFsGfLxOL~lucPQNziMPO9x9dbBeZ0IWshUmDEj0KzdxXRfI5a5LwF5OTZhPF0DotRXhRJ3QW4FbdJlEY6PDvvoVftvvO-73IoCruokpPQYEH9wZNBb70xNvqRJGJShphDNdau7HvzH8whqJcgSHG~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5042675&ar=353879&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01003.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The front page of Mutation@A Glance. There are two types of query interface for (A) browsing known mutation data and (B) evaluating novel sequence variations in DNA sequences of interest. See the main text for details of the mutation data available in Mutation@A Glance.</p></div></div>                        <a id="5042676" scrollto-destination="5042676"></a>
<div id="DSQ010F4" data-id="DSQ010F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq010/2/m_dsq01004.jpeg?Expires=2147483647&Signature=pTV5dQivSgi87mCUKuxtWjQ1TiWRwdfchOvbIKKBd2wG7TPLNIKXLsRWxv65Poxo2q7XOqtCB0dXDEY0OQmWMi-7cRDMAz4V2Ljzz3n0ftKcN8t9URVbPgn3En3x3cql012~ffXTJoIOvTT8r7vcu8B3ENqaIrLr97GBNxD1xyeLOrJ0~1wdkz-nd-O9fgAOTn8FgscdWot1qAJ0vrDizfGuSb3kbdKI4QiA7sdNHiehvGLeg23CTh~vkbPQNzyc1woFnTKMfAKBlMIcz07Z7~bLBJ3Tjg-ARRrB~-C0ixHJvdWqWUMd23ZoDfk296j1vUcYdkSdxGOZC8Ct1n5bog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Screenshots of Mutation@A Glance. An example of visualizing mutation data for STAT3 is shown at the DNA (A) and the protein levels (B). The nucleotide and amino acid positions of disease-associated mutations and SNPs are coloured magenta and green, respectively. At the protein level, various types of information for the mutated amino acid residues can be viewed. (C) The detailed information about the position of nucleotide or amino acid residues selected. (D) A multiple sequence alignment of human and the other organisms STAT3 protein sequences. (E) Detailed information about the 3D structure displayed with Jmol and the representation option menu for 3D structure information. (F) External links to other website for various types of information about the gene, e.g. gene expression and signalling pathway." path-from-xml="dsq01004.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5042676/dsq01004.jpeg" path-from-xml="dsq01004.jpeg" target="_blank">View large</a><a section="5042676" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq010/2/dsq01004.jpeg?Expires=2147483647&Signature=TdSHFzArZiPjodHmTjj5FiQGkkQ-~8OM1bJiWxxhsWAQCjz3iTu-JyewBTN95TV9JM0JxmBIauuEMTDoxZUR4OAW7siqVDYyg8NeHFGQplXe2DvEFkYiHPPKDoP2PiAc5RmeVFjTKgXEsON-WhLyh8Lxw~l9AoUjPEZj7N6UymfwUfeKZgZ~Cv3epPEVpApPFYd4hv1wFJfhCKfcCDA6c8zRa6ahm1bvHRoHJNOhIM8MePGWRzzuELTxbMXKDKb~CRCPVEA4tdQeqtLsSu5KYQFI3eeSbIxWvH5F~42ICiHnSi7e59EtRwsTgRMOc1qyXisiQR4uUgOZ6uo8IbFH3g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5042676&ar=353879&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01004.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Screenshots of Mutation@A Glance. An example of visualizing mutation data for <em>STAT3</em> is shown at the DNA (A) and the protein levels (B). The nucleotide and amino acid positions of disease-associated mutations and SNPs are coloured magenta and green, respectively. At the protein level, various types of information for the mutated amino acid residues can be viewed. (C) The detailed information about the position of nucleotide or amino acid residues selected. (D) A multiple sequence alignment of human and the other organisms STAT3 protein sequences. (E) Detailed information about the 3D structure displayed with Jmol and the representation option menu for 3D structure information. (F) External links to other website for various types of information about the gene, e.g. gene expression and signalling pathway.</p></div></div><div content-id="DSQ010F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq010/2/m_dsq01004.jpeg?Expires=2147483647&Signature=pTV5dQivSgi87mCUKuxtWjQ1TiWRwdfchOvbIKKBd2wG7TPLNIKXLsRWxv65Poxo2q7XOqtCB0dXDEY0OQmWMi-7cRDMAz4V2Ljzz3n0ftKcN8t9URVbPgn3En3x3cql012~ffXTJoIOvTT8r7vcu8B3ENqaIrLr97GBNxD1xyeLOrJ0~1wdkz-nd-O9fgAOTn8FgscdWot1qAJ0vrDizfGuSb3kbdKI4QiA7sdNHiehvGLeg23CTh~vkbPQNzyc1woFnTKMfAKBlMIcz07Z7~bLBJ3Tjg-ARRrB~-C0ixHJvdWqWUMd23ZoDfk296j1vUcYdkSdxGOZC8Ct1n5bog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Screenshots of Mutation@A Glance. An example of visualizing mutation data for STAT3 is shown at the DNA (A) and the protein levels (B). The nucleotide and amino acid positions of disease-associated mutations and SNPs are coloured magenta and green, respectively. At the protein level, various types of information for the mutated amino acid residues can be viewed. (C) The detailed information about the position of nucleotide or amino acid residues selected. (D) A multiple sequence alignment of human and the other organisms STAT3 protein sequences. (E) Detailed information about the 3D structure displayed with Jmol and the representation option menu for 3D structure information. (F) External links to other website for various types of information about the gene, e.g. gene expression and signalling pathway." path-from-xml="dsq01004.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5042676/dsq01004.jpeg" path-from-xml="dsq01004.jpeg" target="_blank">View large</a><a section="5042676" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq010/2/dsq01004.jpeg?Expires=2147483647&Signature=TdSHFzArZiPjodHmTjj5FiQGkkQ-~8OM1bJiWxxhsWAQCjz3iTu-JyewBTN95TV9JM0JxmBIauuEMTDoxZUR4OAW7siqVDYyg8NeHFGQplXe2DvEFkYiHPPKDoP2PiAc5RmeVFjTKgXEsON-WhLyh8Lxw~l9AoUjPEZj7N6UymfwUfeKZgZ~Cv3epPEVpApPFYd4hv1wFJfhCKfcCDA6c8zRa6ahm1bvHRoHJNOhIM8MePGWRzzuELTxbMXKDKb~CRCPVEA4tdQeqtLsSu5KYQFI3eeSbIxWvH5F~42ICiHnSi7e59EtRwsTgRMOc1qyXisiQR4uUgOZ6uo8IbFH3g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5042676&ar=353879&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01004.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Screenshots of Mutation@A Glance. An example of visualizing mutation data for <em>STAT3</em> is shown at the DNA (A) and the protein levels (B). The nucleotide and amino acid positions of disease-associated mutations and SNPs are coloured magenta and green, respectively. At the protein level, various types of information for the mutated amino acid residues can be viewed. (C) The detailed information about the position of nucleotide or amino acid residues selected. (D) A multiple sequence alignment of human and the other organisms STAT3 protein sequences. (E) Detailed information about the 3D structure displayed with Jmol and the representation option menu for 3D structure information. (F) External links to other website for various types of information about the gene, e.g. gene expression and signalling pathway.</p></div></div>                        <a id="5042677" scrollto-destination="5042677"></a>
<div id="DSQ010F5" data-id="DSQ010F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq010/2/m_dsq01005.jpeg?Expires=2147483647&Signature=BIRZV9IMjn6MgMq6tt796s5heo0zY435sYn1hpyLysLjpPFNhEaIxbFNauFCmGdLAFJI-xFy3cYlfJDYo7tdKMtSr20R6eYQi8B6ypcHoneSLE7OkuZ6yIyvrZvx2EUUeckVhZUvafHqaWL~EZeaAFPfNN6Iqhug~hUfZODEv7YDg6ldxE1W-mGzqrNhlbNNIDkclLKSQVENGU~oeTmyPZcWXgaE4sVJHqaLhedvHTulF2M7CgOfr89UoKfFXWPCZXfoqErlypDUiMEw36Vw5b6JpzEuB0n~oCyVLBfSXpT8HDM8DglqHS2ElrpoPcLtN5JIBaiOBf4CeNvK5LZ8sw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Spatial localization of disease-associated missense mutation sites on the STAT3 protein structure. Two STAT3 subunits are represented as a space-filling model coloured white (subunit A) and a ribbon model coloured pink (subunit B), respectively. A double-stranded DNA is represented as a ribbon model coloured light green. The disease-related missense mutations and nsSNPs of STAT3 (subunit A) are coloured magenta and green, respectively." path-from-xml="dsq01005.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5042677/dsq01005.jpeg" path-from-xml="dsq01005.jpeg" target="_blank">View large</a><a section="5042677" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq010/2/dsq01005.jpeg?Expires=2147483647&Signature=Q7DSSyjZ39w-h~82W8q140hJPTtCK0AqjvMySBTDiMMJYKj5qOyEZzzOy4fceCmD-DUxEYcK0mUUiHCXdT~oY1PhXn2SP19ERKgAKlWXqQiPOIFZkTf-aMAO1dYAzqX1rCCbbMSLo97VAU5JpZMuMNbjFWmKPUZIAAV0fELjCeOHNCNe-GsRG94nGZR6ECqt4GOjvJ5Iuf4Os9dcG8h2Kkf4FnU~pEiyav8Fz~p3G338PsrKRo~n2Oo7ZaUQ58-8fTxXkNIrIIQ0rOlvH5FXaKG9-7lHPGQ7lPjh28rtGDkP70yG7S4P1T-oPbgPGbh1oEorriRV~vEv1qPSgRvEDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5042677&ar=353879&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01005.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Spatial localization of disease-associated missense mutation sites on the STAT3 protein structure. Two STAT3 subunits are represented as a space-filling model coloured white (subunit A) and a ribbon model coloured pink (subunit B), respectively. A double-stranded DNA is represented as a ribbon model coloured light green. The disease-related missense mutations and nsSNPs of STAT3 (subunit A) are coloured magenta and green, respectively.</p></div></div><div content-id="DSQ010F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq010/2/m_dsq01005.jpeg?Expires=2147483647&Signature=BIRZV9IMjn6MgMq6tt796s5heo0zY435sYn1hpyLysLjpPFNhEaIxbFNauFCmGdLAFJI-xFy3cYlfJDYo7tdKMtSr20R6eYQi8B6ypcHoneSLE7OkuZ6yIyvrZvx2EUUeckVhZUvafHqaWL~EZeaAFPfNN6Iqhug~hUfZODEv7YDg6ldxE1W-mGzqrNhlbNNIDkclLKSQVENGU~oeTmyPZcWXgaE4sVJHqaLhedvHTulF2M7CgOfr89UoKfFXWPCZXfoqErlypDUiMEw36Vw5b6JpzEuB0n~oCyVLBfSXpT8HDM8DglqHS2ElrpoPcLtN5JIBaiOBf4CeNvK5LZ8sw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Spatial localization of disease-associated missense mutation sites on the STAT3 protein structure. Two STAT3 subunits are represented as a space-filling model coloured white (subunit A) and a ribbon model coloured pink (subunit B), respectively. A double-stranded DNA is represented as a ribbon model coloured light green. The disease-related missense mutations and nsSNPs of STAT3 (subunit A) are coloured magenta and green, respectively." path-from-xml="dsq01005.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5042677/dsq01005.jpeg" path-from-xml="dsq01005.jpeg" target="_blank">View large</a><a section="5042677" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq010/2/dsq01005.jpeg?Expires=2147483647&Signature=Q7DSSyjZ39w-h~82W8q140hJPTtCK0AqjvMySBTDiMMJYKj5qOyEZzzOy4fceCmD-DUxEYcK0mUUiHCXdT~oY1PhXn2SP19ERKgAKlWXqQiPOIFZkTf-aMAO1dYAzqX1rCCbbMSLo97VAU5JpZMuMNbjFWmKPUZIAAV0fELjCeOHNCNe-GsRG94nGZR6ECqt4GOjvJ5Iuf4Os9dcG8h2Kkf4FnU~pEiyav8Fz~p3G338PsrKRo~n2Oo7ZaUQ58-8fTxXkNIrIIQ0rOlvH5FXaKG9-7lHPGQ7lPjh28rtGDkP70yG7S4P1T-oPbgPGbh1oEorriRV~vEv1qPSgRvEDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5042677&ar=353879&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01005.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Spatial localization of disease-associated missense mutation sites on the STAT3 protein structure. Two STAT3 subunits are represented as a space-filling model coloured white (subunit A) and a ribbon model coloured pink (subunit B), respectively. A double-stranded DNA is represented as a ribbon model coloured light green. The disease-related missense mutations and nsSNPs of STAT3 (subunit A) are coloured magenta and green, respectively.</p></div></div>                            <h3 scrollto-destination=5042678 id="5042678" class="section-title" data-legacy-id=s3d>3.4. Evaluating the sequence variations in query sequences</h3>
<p>One of the issues of diagnosis of genetic diseases is how to evaluate the pathogenicity of newly identified sequence variations. To address this issue, Mutation@A Glance has an interface that allows clinical researchers to assess the impact of an observed sequence variation in a given DNA sequence for a candidate disease-causing gene as the second query form (Fig. <a reveal-id="DSQ010F3" data-open="DSQ010F3" class="link link-reveal link-table xref-fig">3</a>B). When a user submits DNA sequences of a candidate gene in question, this tool returns a list of sequence variations found in the input DNA sequences at both the DNA and the protein levels (Fig. <a reveal-id="DSQ010F6" data-open="DSQ010F6" class="link link-reveal link-table xref-fig">6</a>). To identify genetic variations that occur in input DNA sequences of a given gene, the BLAT program<sup><a reveal-id="DSQ010C58" data-open="DSQ010C58" class="link link-ref link-reveal xref-bibr">58</a></sup> is implemented to align the input DNA sequences with the reference genomic DNA sequence for the corresponding gene. Figure <a reveal-id="DSQ010F6" data-open="DSQ010F6" class="link link-reveal link-table xref-fig">6</a>A represents the alignment status of the query sequence to the reference sequence. If a sequence variation is found, multiple lines of detailed information about the variation, such as the variation types (e.g. substitution, insertion and deletion), the mutated region (e.g. exon, intron and 5′- or 3′-splice sites constituting the GT-AG rule), the amino acid changes (e.g. missense, nonsense, insertion/deletion and frame-shift), the known variation data (disease-associated mutation and SNP) and structure/function features of the position at the protein level, are displayed based on the reference human genome sequence in the public database (Fig. <a reveal-id="DSQ010F6" data-open="DSQ010F6" class="link link-reveal link-table xref-fig">6</a>B). Sequence alignments between the query and reference sequences are also displayed (Fig. <a reveal-id="DSQ010F6" data-open="DSQ010F6" class="link link-reveal link-table xref-fig">6</a>C). If a ns substitution is found in the query DNA sequence, it was evaluated by the SIFT program<sup><a reveal-id="DSQ010C26" data-open="DSQ010C26" class="link link-ref link-reveal xref-bibr">26</a></sup> (incorporated in the local system), which predicts whether amino acid substitutions in a protein will be ‘Deleterious’ or ‘Tolerated’ using evolutionary information from the homologous proteins (Fig. <a reveal-id="DSQ010F6" data-open="DSQ010F6" class="link link-reveal link-table xref-fig">6</a>B). We tested the prediction accuracy of SIFT with our data sets of disease-associated mutations and non-disease-associated nsSNPs, and found that the false-negative rate (falsely predicted as ‘Tolerated’ for disease-associated mutations) and the false-positive rate (falsely predicted as ‘Deleterious’ for nsSNPs) were 25% and 39%, respectively. These accuracy values were comparable to those evaluated in previous study.<sup><a reveal-id="DSQ010C33" data-open="DSQ010C33" class="link link-ref link-reveal xref-bibr">33</a></sup> The current version of Mutation@A Glance does not implement a method for quantitative evaluation of mutation effects on RNA splicing, mainly because we considered the evaluation method is not matured enough yet. However, because the evaluation of mutation effects on RNA splicing/stability is very intriguing, we will place a high priority on the implementation of the evaluation tool for genetic variations affecting RNA splicing/stability in the future development.</p>                        <a id="5042680" scrollto-destination="5042680"></a>
<div id="DSQ010F6" data-id="DSQ010F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq010/2/m_dsq01006.jpeg?Expires=2147483647&Signature=oCSVssfVobC73CUTJ6qdfyr6iVN~X5EulgCocZ1EG45L68otZQR110SKGewSkc4b2UWmJjYmbYQfDtW02Kaw01DOHU3sXJz4sk1OVHwO6002KZbJgSQxN7ZZRFpQYj~KhzKifurPLw~wxz5f053TbADl0JVIiGMpj-DJJHBbpRkx0Cph64~IcawTMP4uhDtma5nEcQc2njuJxSJ7hDRsOzOzoWCWpwDOJJHBzEZBFdd0uE2-gtiG7xdsiyExU4hn-dYu~tDtx-M2XXYMjyK-iSYE3uOr7X491KaA1KDh7vH0ax5ARKbC-mCMqSeMGakx2TEgse51hW6rWzIgU4UF5g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="An example of evaluating sequence variations in query STAT3 DNA sequences. (A) The mapping status of each query sequence to the reference sequence is shown. (B) If a variation is found in the query sequence, the detailed information is shown for each variation (e.g. the positions on the DNA/protein sequences, the type of variation and the description as to whether or not it is known as a disease-associated mutation or SNP). Results from the SIFT program (‘Tolerated’ or ‘Deleterious’) are also shown if the variation caused ns substitutions. (C) The query-reference sequence alignment around the altered nucleotides is depicted. (D) The variations can be visualized in the viewer, represented by different colours for known disease mutations or SNPs as ‘User's Data’." path-from-xml="dsq01006.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5042680/dsq01006.jpeg" path-from-xml="dsq01006.jpeg" target="_blank">View large</a><a section="5042680" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq010/2/dsq01006.jpeg?Expires=2147483647&Signature=sJ5R8s5b3l4mKJX6X2XYZYq2ZKg~b~SVO60vXZD8t92gF7cXHuzoue26ontrpxn53u4-sIEyErsbHluHSvtuY-Xo1mLgRA4phwsUQPxfu~H7tbvMQ2FjmGRO5YidWaI-aRNM6NPVpSmWccmOJSIqVc61IG9wGMmayO5CE6C5wTjSsgeJq3DlWp43vBUcV6xVZ4wGzMQIgHxSdWcZk5OqBO-n7uGtab3tgAkOvJ-1HXfJqbkS5avuhE43rkLerJ4R1SnStXv8UumRrVf-O6mqsjBwQSywCpMgs5LnvqBqZj-yWw-gctyEIzbO9JXjv7ehI93AADsSmnnLZG~oNCszzQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5042680&ar=353879&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01006.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>An example of evaluating sequence variations in query <em>STAT3</em> DNA sequences. (A) The mapping status of each query sequence to the reference sequence is shown. (B) If a variation is found in the query sequence, the detailed information is shown for each variation (e.g. the positions on the DNA/protein sequences, the type of variation and the description as to whether or not it is known as a disease-associated mutation or SNP). Results from the SIFT program (‘Tolerated’ or ‘Deleterious’) are also shown if the variation caused ns substitutions. (C) The query-reference sequence alignment around the altered nucleotides is depicted. (D) The variations can be visualized in the viewer, represented by different colours for known disease mutations or SNPs as ‘User's Data’.</p></div></div><div content-id="DSQ010F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq010/2/m_dsq01006.jpeg?Expires=2147483647&Signature=oCSVssfVobC73CUTJ6qdfyr6iVN~X5EulgCocZ1EG45L68otZQR110SKGewSkc4b2UWmJjYmbYQfDtW02Kaw01DOHU3sXJz4sk1OVHwO6002KZbJgSQxN7ZZRFpQYj~KhzKifurPLw~wxz5f053TbADl0JVIiGMpj-DJJHBbpRkx0Cph64~IcawTMP4uhDtma5nEcQc2njuJxSJ7hDRsOzOzoWCWpwDOJJHBzEZBFdd0uE2-gtiG7xdsiyExU4hn-dYu~tDtx-M2XXYMjyK-iSYE3uOr7X491KaA1KDh7vH0ax5ARKbC-mCMqSeMGakx2TEgse51hW6rWzIgU4UF5g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="An example of evaluating sequence variations in query STAT3 DNA sequences. (A) The mapping status of each query sequence to the reference sequence is shown. (B) If a variation is found in the query sequence, the detailed information is shown for each variation (e.g. the positions on the DNA/protein sequences, the type of variation and the description as to whether or not it is known as a disease-associated mutation or SNP). Results from the SIFT program (‘Tolerated’ or ‘Deleterious’) are also shown if the variation caused ns substitutions. (C) The query-reference sequence alignment around the altered nucleotides is depicted. (D) The variations can be visualized in the viewer, represented by different colours for known disease mutations or SNPs as ‘User's Data’." path-from-xml="dsq01006.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5042680/dsq01006.jpeg" path-from-xml="dsq01006.jpeg" target="_blank">View large</a><a section="5042680" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq010/2/dsq01006.jpeg?Expires=2147483647&Signature=sJ5R8s5b3l4mKJX6X2XYZYq2ZKg~b~SVO60vXZD8t92gF7cXHuzoue26ontrpxn53u4-sIEyErsbHluHSvtuY-Xo1mLgRA4phwsUQPxfu~H7tbvMQ2FjmGRO5YidWaI-aRNM6NPVpSmWccmOJSIqVc61IG9wGMmayO5CE6C5wTjSsgeJq3DlWp43vBUcV6xVZ4wGzMQIgHxSdWcZk5OqBO-n7uGtab3tgAkOvJ-1HXfJqbkS5avuhE43rkLerJ4R1SnStXv8UumRrVf-O6mqsjBwQSywCpMgs5LnvqBqZj-yWw-gctyEIzbO9JXjv7ehI93AADsSmnnLZG~oNCszzQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5042680&ar=353879&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01006.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>An example of evaluating sequence variations in query <em>STAT3</em> DNA sequences. (A) The mapping status of each query sequence to the reference sequence is shown. (B) If a variation is found in the query sequence, the detailed information is shown for each variation (e.g. the positions on the DNA/protein sequences, the type of variation and the description as to whether or not it is known as a disease-associated mutation or SNP). Results from the SIFT program (‘Tolerated’ or ‘Deleterious’) are also shown if the variation caused ns substitutions. (C) The query-reference sequence alignment around the altered nucleotides is depicted. (D) The variations can be visualized in the viewer, represented by different colours for known disease mutations or SNPs as ‘User's Data’.</p></div></div><p>There are several advantages of Mutation@A Glance over other existing web servers for evaluating the effects of mutations. First, users are only required to have DNA sequences from a particular gene as their input and thus do not need to pre-process their submission data; other websites for evaluating the mutation effects require a list of genetic variations as a query, not raw sequence data.<sup><a reveal-id="DSQ010C26  DSQ010C31" data-open="DSQ010C26  DSQ010C31" class="link link-ref link-reveal xref-bibr">26–31</a></sup> Secondly, Mutation@A Glance identifies and addresses multiple types of sequence variations (e.g. insertion/deletion, frame-shifts) from input query DNA sequences whereas the other web servers do not. Thirdly, newly identified genetic variations can be easily compared with known mutation and SNP data using the graphical visualization interface of Mutation@A Glance (Fig. <a reveal-id="DSQ010F6" data-open="DSQ010F6" class="link link-reveal link-table xref-fig">6</a>D).</p><p>From a viewpoint of clinical use, it is obvious that any mutation analysis platform cannot serve as a useful one without reliable mutation data sets. However, whereas large amounts of disease-associated mutation data for various genetic diseases have been reported, most of them are dispersed and stored locally. Only a few websites, e.g. OMIM and UniProt, integrate disease-associated mutation data and allow us to download their contents. However, the mutation data in such databases have a relatively low integrity in terms of updating and coverage. Thus, we have begun to comprehensively collect and manually curate the disease-associated mutation data from published literature focusing on PIDs and established a resource of PID research for clinical use, named RAPID.<sup><a reveal-id="DSQ010C17" data-open="DSQ010C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Mutation@A Glance thus uses these manually curated data sets for over 150 PID genes in the RAPID database, which is solid enough for clinical use at least for PID analysis. To make Mutation@A Glance a reliable and general mutation analysis platform for other various genetic diseases in the future, we consider that data sharing with experts in particular diseases will be highly important as in the case of PID; otherwise it would take a long time to accumulate extensive mutation data of all human disease genes to an acceptable level for clinical use. In fact, similar efforts along this direction have been being made by the research community.<sup><a reveal-id="DSQ010C19" data-open="DSQ010C19" class="link link-ref link-reveal xref-bibr">19</a></sup></p><p>As new technologies for determining genetic variation in humans have rapidly and continuously emerged (such as next generation DNA sequencing), amounts of genetic variation data of human are exponentially growing.<sup><a reveal-id="DSQ010C6" data-open="DSQ010C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSQ010C7" data-open="DSQ010C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSQ010C59" data-open="DSQ010C59" class="link link-ref link-reveal xref-bibr">59</a></sup> Therefore, we will continue to update and improve the Mutation@A Glance system, in order to cope with the larger-scale data analysis for more comprehensive identification of disease-causative candidate genes. Implementing API programs into Mutation@A Glance for query submissions and a retrieval system through command line scripts would be more convenient for this purpose.</p><p>In summary, Mutation@A Glance provides a highly integrated bioinformatics tool for mutation analysis not only for facilitating visualization of sequence variation data along with various types of information, including primary and tertiary structures of the gene products, but also for evaluating the effects of novel sequence variations in a query input DNA sequence. This tool works solely on a web browser through Internet and is open to the public. Hence, Mutation@A Glance can be used as a ‘one-stop’ integrated bioinformatics platform for analysing genotype–phenotype relationships of genetic diseases from molecular as well as clinical perspectives.</p>                            <h2 scrollto-destination=5042685 id="5042685" class="section-title" data-legacy-id=s4>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=5042687 id="5042687" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work is supported by a Grant-in-aid for Special Coordination Funds for Promoting Science and Technology, from the Ministry of Education, Culture, Sports, Science and Technology of Japan.</p>                            <h2 scrollto-destination=5042689 id="5042689" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors would like to thank Drs Shigeaki Nonoyama, Kohsuke Imai, Hirokazu Kanegane, Toshio Miyawaki, Koichi Oshima, Fumihiko Ishikawa and Reiko Kikuno-Fukaya for their critical suggestions about this work.</p>                            <h2 scrollto-destination=5042691 id="5042691" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSQ010C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Amberger</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bocchini</div> <div class="given-names">C.A.</div></div>,  <div class="name"><div class="surname">Scott</div> <div class="given-names">A.F.</div></div>,  <div class="name"><div class="surname">Hamosh</div> <div class="given-names">A.</div></div>. </div><div class="article-title">McKusick's Online Mendelian Inheritance in Man (OMIM)</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">D793</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkn665" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkn665</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=McKusick's%20Online%20Mendelian%20Inheritance%20in%20Man%20(OMIM)&amp;author=J.%20Amberger&amp;author=C.A.%20Bocchini&amp;author=A.F.%20Scott&amp;author=A.%20Hamosh&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=D793-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn665" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn665" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn665"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18842627" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Notarangelo</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Casanova</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Fischer</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Primary immunodeficiency diseases: an update</div>, <div class="source">J. Allergy Clin. Immunol.</div> , <div class="year">2004</div>, vol. <div class="volume">114</div> (pg. <div class="fpage">677</div>-<div class="lpage">87</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.jaci.2004.06.044" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.jaci.2004.06.044</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Primary%20immunodeficiency%20diseases%3A%20an%20update&amp;author=L.%20Notarangelo&amp;author=J.L.%20Casanova&amp;author=A.%20Fischer&amp;publication_year=2004&amp;journal=J.%20Allergy%20Clin.%20Immunol.&amp;volume=114&amp;pages=677-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jaci.2004.06.044" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jaci.2004.06.044" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jaci.2004.06.044"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15356576" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Geha</div> <div class="given-names">R.S.</div></div>,  <div class="name"><div class="surname">Notarangelo</div> <div class="given-names">L.D.</div></div>,  <div class="name"><div class="surname">Casanova</div> <div class="given-names">J.L.</div></div>, et al. </div><div class="article-title">Primary immunodeficiency diseases: an update from the International Union of Immunological Societies Primary Immunodeficiency Diseases Classification Committee</div>, <div class="source">J. Allergy Clin. Immunol.</div> , <div class="year">2007</div>, vol. <div class="volume">120</div> (pg. <div class="fpage">776</div>-<div class="lpage">94</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.jaci.2007.08.053" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.jaci.2007.08.053</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Primary%20immunodeficiency%20diseases%3A%20an%20update%20from%20the%20International%20Union%20of%20Immunological%20Societies%20Primary%20Immunodeficiency%20Diseases%20Classification%20Committee&amp;author=R.S.%20Geha&amp;author=L.D.%20Notarangelo&amp;author=J.L.%20Casanova&amp;publication_year=2007&amp;journal=J.%20Allergy%20Clin.%20Immunol.&amp;volume=120&amp;pages=776-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jaci.2007.08.053" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jaci.2007.08.053" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jaci.2007.08.053"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17952897" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Notarangelo</div> <div class="given-names">L.D.</div></div>,  <div class="name"><div class="surname">Sorensen</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Is it necessary to identify molecular defects in primary immunodeficiency disease?</div>, <div class="source">J. Allergy Clin. Immunol</div> , <div class="year">2008</div>, vol. <div class="volume">122</div> (pg. <div class="fpage">1069</div>-<div class="lpage">73</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.jaci.2008.08.038" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.jaci.2008.08.038</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Is%20it%20necessary%20to%20identify%20molecular%20defects%20in%20primary%20immunodeficiency%20disease%3F&amp;author=L.D.%20Notarangelo&amp;author=R.%20Sorensen&amp;publication_year=2008&amp;journal=J.%20Allergy%20Clin.%20Immunol&amp;volume=122&amp;pages=1069-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jaci.2008.08.038" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jaci.2008.08.038" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jaci.2008.08.038"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18992927" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Frazer</div> <div class="given-names">K.A.</div></div>,  <div class="name"><div class="surname">Ballinger</div> <div class="given-names">D.G.</div></div>,  <div class="name"><div class="surname">Cox</div> <div class="given-names">D.R.</div></div>, et al. </div><div class="article-title">A second generation human haplotype map of over 3.1 million SNPs</div>, <div class="source">Nature</div> , <div class="year">2007</div>, vol. <div class="volume">449</div> (pg. <div class="fpage">851</div>-<div class="lpage">61</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature06258" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature06258</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20second%20generation%20human%20haplotype%20map%20of%20over%203.1%20million%20SNPs&amp;author=K.A.%20Frazer&amp;author=D.G.%20Ballinger&amp;author=D.R.%20Cox&amp;publication_year=2007&amp;journal=Nature&amp;volume=449&amp;pages=851-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06258" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06258" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06258"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17943122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kryukov</div> <div class="given-names">G.V.</div></div>,  <div class="name"><div class="surname">Shpunt</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Stamatoyannopoulos</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Sunyaev</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">Power of deep, all-exon resequencing for discovery of human trait genes</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2009</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">3871</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.0812824106" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.0812824106</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Power%20of%20deep%2C%20all-exon%20resequencing%20for%20discovery%20of%20human%20trait%20genes&amp;author=G.V.%20Kryukov&amp;author=A.%20Shpunt&amp;author=J.A.%20Stamatoyannopoulos&amp;author=S.R.%20Sunyaev&amp;publication_year=2009&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=3871-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0812824106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0812824106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0812824106"> </span></div> </div></div></div></div></div><div content-id="DSQ010C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chun</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Fay</div> <div class="given-names">J.C.</div></div>. </div><div class="article-title">Identification of deleterious mutations within three human genomes</div>, <div class="source">Genome Res.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">1553</div>-<div class="lpage">61</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.092619.109" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.092619.109</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20deleterious%20mutations%20within%20three%20human%20genomes&amp;author=S.%20Chun&amp;author=J.C.%20Fay&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=1553-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.092619.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.092619.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.092619.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19602639" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sunyaev</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ramensky</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Koch</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Lathe</div> <div class="given-names">W.</div>III</div>,  <div class="name"><div class="surname">Kondrashov</div> <div class="given-names">A.S.</div></div>,  <div class="name"><div class="surname">Bork</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Prediction of deleterious human alleles</div>, <div class="source">Hum. Mol. Genet.</div> , <div class="year">2001</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">591</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/hmg/10.6.591" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/hmg/10.6.591</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20deleterious%20human%20alleles&amp;author=S.%20Sunyaev&amp;author=V.%20Ramensky&amp;author=I.%20Koch&amp;author=W.%20Lathe&amp;author=A.S.%20Kondrashov&amp;author=P.%20Bork&amp;publication_year=2001&amp;journal=Hum.%20Mol.%20Genet.&amp;volume=10&amp;pages=591-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/hmg/10.6.591" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fhmg%2F10.6.591" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fhmg%2F10.6.591"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11230178" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tavtigian</div> <div class="given-names">S.V.</div></div>,  <div class="name"><div class="surname">Byrnes</div> <div class="given-names">G.B.</div></div>,  <div class="name"><div class="surname">Goldgar</div> <div class="given-names">D.E.</div></div>,  <div class="name"><div class="surname">Thomas</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Classification of rare missense substitutions, using risk surfaces, with genetic- and molecular-epidemiology applications</div>, <div class="source">Hum. Mutat.</div> , <div class="year">2008</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">1342</div>-<div class="lpage">54</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/humu.20896" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/humu.20896</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Classification%20of%20rare%20missense%20substitutions%2C%20using%20risk%20surfaces%2C%20with%20genetic-%20and%20molecular-epidemiology%20applications&amp;author=S.V.%20Tavtigian&amp;author=G.B.%20Byrnes&amp;author=D.E.%20Goldgar&amp;author=A.%20Thomas&amp;publication_year=2008&amp;journal=Hum.%20Mutat.&amp;volume=29&amp;pages=1342-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/humu.v29:11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fhumu.v29:11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fhumu.v29:11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18951461" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tavtigian</div> <div class="given-names">S.V.</div></div>,  <div class="name"><div class="surname">Greenblatt</div> <div class="given-names">M.S.</div></div>,  <div class="name"><div class="surname">Goldgar</div> <div class="given-names">D.E.</div></div>,  <div class="name"><div class="surname">Boffetta</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Assessing pathogenicity: overview of results from the IARC Unclassified Genetic Variants Working Group</div>, <div class="source">Hum. Mutat.</div> , <div class="year">2008</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">1261</div>-<div class="lpage">4</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/humu.20903" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/humu.20903</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessing%20pathogenicity%3A%20overview%20of%20results%20from%20the%20IARC%20Unclassified%20Genetic%20Variants%20Working%20Group&amp;author=S.V.%20Tavtigian&amp;author=M.S.%20Greenblatt&amp;author=D.E.%20Goldgar&amp;author=P.%20Boffetta&amp;publication_year=2008&amp;journal=Hum.%20Mutat.&amp;volume=29&amp;pages=1261-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/humu.v29:11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fhumu.v29:11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fhumu.v29:11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18951436" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tavtigian</div> <div class="given-names">S.V.</div></div>,  <div class="name"><div class="surname">Greenblatt</div> <div class="given-names">M.S.</div></div>,  <div class="name"><div class="surname">Lesueur</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Byrnes</div> <div class="given-names">G.B.</div></div>. </div><div class="article-title">In silico analysis of missense substitutions using sequence-alignment based methods</div>, <div class="source">Hum. Mutat.</div> , <div class="year">2008</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">1327</div>-<div class="lpage">36</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/humu.20892" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/humu.20892</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20silico%20analysis%20of%20missense%20substitutions%20using%20sequence-alignment%20based%20methods&amp;author=S.V.%20Tavtigian&amp;author=M.S.%20Greenblatt&amp;author=F.%20Lesueur&amp;author=G.B.%20Byrnes&amp;publication_year=2008&amp;journal=Hum.%20Mutat.&amp;volume=29&amp;pages=1327-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/humu.v29:11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fhumu.v29:11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fhumu.v29:11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18951440" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Houdayer</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Dehainault</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Mattler</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Evaluation of in silico splice tools for decision-making in molecular diagnosis</div>, <div class="source">Hum. Mutat.</div> , <div class="year">2008</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">975</div>-<div class="lpage">82</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/humu.20765" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/humu.20765</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evaluation%20of%20in%20silico%20splice%20tools%20for%20decision-making%20in%20molecular%20diagnosis&amp;author=C.%20Houdayer&amp;author=C.%20Dehainault&amp;author=C.%20Mattler&amp;publication_year=2008&amp;journal=Hum.%20Mutat.&amp;volume=29&amp;pages=975-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1098-1004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1098-1004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1098-1004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18449911" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stenson</div> <div class="given-names">P.D.</div></div>,  <div class="name"><div class="surname">Mort</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ball</div> <div class="given-names">E.V.</div></div>, et al. </div><div class="article-title">The Human Gene Mutation Database: 2008 update</div>, <div class="source">Genome Med.</div> , <div class="year">2009</div>, vol. <div class="volume">1</div> pg. <div class="fpage">13</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/gm13" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/gm13</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Human%20Gene%20Mutation%20Database%3A%202008%20update&amp;author=P.D.%20Stenson&amp;author=M.%20Mort&amp;author=E.V.%20Ball&amp;publication_year=2009&amp;journal=Genome%20Med.&amp;volume=1&amp;pages=13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gm13" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgm13" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgm13"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19348700" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Singh</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Olowoyeye</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Baenziger</div> <div class="given-names">P.H.</div></div>, et al. </div><div class="article-title">MutDB: update on development of tools for the biochemical analysis of genetic variation</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D815</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkm659" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkm659</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MutDB%3A%20update%20on%20development%20of%20tools%20for%20the%20biochemical%20analysis%20of%20genetic%20variation&amp;author=A.%20Singh&amp;author=A.%20Olowoyeye&amp;author=P.H.%20Baenziger&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D815-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm659" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm659" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm659"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17827212" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mailman</div> <div class="given-names">M.D.</div></div>,  <div class="name"><div class="surname">Feolo</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Jin</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">The NCBI dbGaP database of genotypes and phenotypes</div>, <div class="source">Nat. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">1181</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/ng1007-1181" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/ng1007-1181</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20NCBI%20dbGaP%20database%20of%20genotypes%20and%20phenotypes&amp;author=M.D.%20Mailman&amp;author=M.%20Feolo&amp;author=Y.%20Jin&amp;publication_year=2007&amp;journal=Nat.%20Genet.&amp;volume=39&amp;pages=1181-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1007-1181" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1007-1181" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1007-1181"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17898773" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Piirila</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Valiaho</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Vihinen</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Immunodeficiency mutation databases (IDbases)</div>, <div class="source">Hum. Mutat.</div> , <div class="year">2006</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">1200</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/humu.20405" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/humu.20405</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Immunodeficiency%20mutation%20databases%20(IDbases)&amp;author=H.%20Piirila&amp;author=J.%20Valiaho&amp;author=M.%20Vihinen&amp;publication_year=2006&amp;journal=Hum.%20Mutat.&amp;volume=27&amp;pages=1200-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/humu.v27:12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fhumu.v27:12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fhumu.v27:12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17004234" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Keerthikumar</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Raju</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Kandasamy</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">RAPID: Resource of Asian Primary Immunodeficiency Diseases</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">D863</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkn682" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkn682</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RAPID%3A%20Resource%20of%20Asian%20Primary%20Immunodeficiency%20Diseases&amp;author=S.%20Keerthikumar&amp;author=R.%20Raju&amp;author=K.%20Kandasamy&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=D863-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn682" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn682" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn682"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18842635" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sherry</div> <div class="given-names">S.T.</div></div>,  <div class="name"><div class="surname">Ward</div> <div class="given-names">M.H.</div></div>,  <div class="name"><div class="surname">Kholodov</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">dbSNP: the NCBI database of genetic variation</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2001</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">308</div>-<div class="lpage">11</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/29.1.308" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/29.1.308</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=dbSNP%3A%20the%20NCBI%20database%20of%20genetic%20variation&amp;author=S.T.%20Sherry&amp;author=M.H.%20Ward&amp;author=M.%20Kholodov&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res.&amp;volume=29&amp;pages=308-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/29.1.308" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.1.308" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F29.1.308"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11125122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kaput</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Cotton</div> <div class="given-names">R.G.</div></div>,  <div class="name"><div class="surname">Hardman</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Planning the human variome project: the Spain report</div>, <div class="source">Hum. Mutat.</div> , <div class="year">2009</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">496</div>-<div class="lpage">510</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/humu.20972" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/humu.20972</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Planning%20the%20human%20variome%20project%3A%20the%20Spain%20report&amp;author=J.%20Kaput&amp;author=R.G.%20Cotton&amp;author=L.%20Hardman&amp;publication_year=2009&amp;journal=Hum.%20Mutat.&amp;volume=30&amp;pages=496-510" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/humu.20972" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fhumu.20972" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fhumu.20972"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19306394" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">P.H.</div></div>,  <div class="name"><div class="surname">Shatkay</div> <div class="given-names">H.</div></div>. </div><div class="article-title">F-SNP: computationally predicted functional SNPs for disease association studies</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D820</div>-<div class="lpage">4</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkm904" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkm904</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=F-SNP%3A%20computationally%20predicted%20functional%20SNPs%20for%20disease%20association%20studies&amp;author=P.H.%20Lee&amp;author=H.%20Shatkay&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D820-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm904" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm904" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm904"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17986460" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yuan</div> <div class="given-names">H.Y.</div></div>,  <div class="name"><div class="surname">Chiou</div> <div class="given-names">J.J.</div></div>,  <div class="name"><div class="surname">Tseng</div> <div class="given-names">W.H.</div></div>, et al. </div><div class="article-title">FASTSNP: an always up-to-date and extendable service for SNP function analysis and prioritization</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">W635</div>-<div class="lpage">41</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkl236" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkl236</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=FASTSNP%3A%20an%20always%20up-to-date%20and%20extendable%20service%20for%20SNP%20function%20analysis%20and%20prioritization&amp;author=H.Y.%20Yuan&amp;author=J.J.%20Chiou&amp;author=W.H.%20Tseng&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=W635-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl236" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl236" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl236"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16845089" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yue</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Melamud</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Moult</div> <div class="given-names">J.</div></div>. </div><div class="article-title">SNPs3D: candidate gene and SNP selection for association studies</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">166</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2105-7-166" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2105-7-166</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNPs3D%3A%20candidate%20gene%20and%20SNP%20selection%20for%20association%20studies&amp;author=P.%20Yue&amp;author=E.%20Melamud&amp;author=J.%20Moult&amp;publication_year=2006&amp;journal=BMC%20Bioinformatics&amp;volume=7&amp;pages=166" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-7-166" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-7-166" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-7-166"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16551372" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chelala</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Khan</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lemoine</div> <div class="given-names">N.R.</div></div>. </div><div class="article-title">SNPnexus: a web database for functional annotation of newly discovered and public domain single nucleotide polymorphisms</div>, <div class="source">Bioinformatics</div> , <div class="year">2009</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">655</div>-<div class="lpage">61</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/btn653" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/btn653</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNPnexus%3A%20a%20web%20database%20for%20functional%20annotation%20of%20newly%20discovered%20and%20public%20domain%20single%20nucleotide%20polymorphisms&amp;author=C.%20Chelala&amp;author=A.%20Khan&amp;author=N.R.%20Lemoine&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=655-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn653" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn653" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn653"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19098027" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">J.O.</div></div>,  <div class="name"><div class="surname">Hwang</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Oh</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bhak</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Sohn</div> <div class="given-names">T.K.</div></div>. </div><div class="article-title">An integrated database-pipeline system for studying single nucleotide polymorphisms and diseases</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> <div class="issue">Suppl 12</div>pg. <div class="fpage">S19</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2105-9-S12-S19" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2105-9-S12-S19</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20database-pipeline%20system%20for%20studying%20single%20nucleotide%20polymorphisms%20and%20diseases&amp;author=J.O.%20Yang&amp;author=S.%20Hwang&amp;author=J.%20Oh&amp;author=J.%20Bhak&amp;author=T.K.%20Sohn&amp;publication_year=2008&amp;journal=BMC%20Bioinformatics&amp;volume=9&amp;pages=S19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-9-S12-S19" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-9-S12-S19" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-9-S12-S19"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19091018" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Frezal</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Genatlas database, genes and development defects</div>, <div class="source">C. R. Acad. Sci. III</div> , <div class="year">1998</div>, vol. <div class="volume">321</div> (pg. <div class="fpage">805</div>-<div class="lpage">17</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S0764-4469(99)80021-3" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S0764-4469(99)80021-3</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genatlas%20database%2C%20genes%20and%20development%20defects&amp;author=J.%20Frezal&amp;publication_year=1998&amp;journal=C.%20R.%20Acad.%20Sci.%20III&amp;volume=321&amp;pages=805-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0764-4469(99)80021-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0764-4469(99)80021-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0764-4469(99)80021-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9835018" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ng</div> <div class="given-names">P.C.</div></div>,  <div class="name"><div class="surname">Henikoff</div> <div class="given-names">S.</div></div>. </div><div class="article-title">SIFT: predicting amino acid changes that affect protein function</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">3812</div>-<div class="lpage">4</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkg509" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkg509</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SIFT%3A%20predicting%20amino%20acid%20changes%20that%20affect%20protein%20function&amp;author=P.C.%20Ng&amp;author=S.%20Henikoff&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=3812-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg509" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg509" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg509"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12824425" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ramensky</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Bork</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Sunyaev</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Human non-synonymous SNPs: server and survey</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">3894</div>-<div class="lpage">900</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkf493" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkf493</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20non-synonymous%20SNPs%3A%20server%20and%20survey&amp;author=V.%20Ramensky&amp;author=P.%20Bork&amp;author=S.%20Sunyaev&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res.&amp;volume=30&amp;pages=3894-900" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkf493" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkf493" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkf493"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12202775" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thomas</div> <div class="given-names">P.D.</div></div>,  <div class="name"><div class="surname">Campbell</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Kejariwal</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">PANTHER: a library of protein families and subfamilies indexed by function</div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">2129</div>-<div class="lpage">41</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.772403" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.772403</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PANTHER%3A%20a%20library%20of%20protein%20families%20and%20subfamilies%20indexed%20by%20function&amp;author=P.D.%20Thomas&amp;author=M.J.%20Campbell&amp;author=A.%20Kejariwal&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=2129-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.772403" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.772403" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.772403"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12952881" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bromberg</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Rost</div> <div class="given-names">B.</div></div>. </div><div class="article-title">SNAP: predict effect of non-synonymous polymorphisms on function</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">3823</div>-<div class="lpage">35</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkm238" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkm238</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNAP%3A%20predict%20effect%20of%20non-synonymous%20polymorphisms%20on%20function&amp;author=Y.%20Bromberg&amp;author=B.%20Rost&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=3823-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm238" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm238" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm238"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17526529" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ferrer-Costa</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Gelpi</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Zamakola</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Parraga</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">de la Cruz</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Orozco</div> <div class="given-names">M.</div></div>. </div><div class="article-title">PMUT: a web-based tool for the annotation of pathological mutations on proteins</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">3176</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/bti486" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/bti486</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PMUT%3A%20a%20web-based%20tool%20for%20the%20annotation%20of%20pathological%20mutations%20on%20proteins&amp;author=C.%20Ferrer-Costa&amp;author=J.L.%20Gelpi&amp;author=L.%20Zamakola&amp;author=I.%20Parraga&amp;author=X.%20de%20la%20Cruz&amp;author=M.%20Orozco&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=3176-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti486" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti486" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti486"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15879453" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bao</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Cui</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">nsSNPAnalyzer: identifying disease-associated nonsynonymous single nucleotide polymorphisms</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">W480</div>-<div class="lpage">2</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gki372" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gki372</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=nsSNPAnalyzer%3A%20identifying%20disease-associated%20nonsynonymous%20single%20nucleotide%20polymorphisms&amp;author=L.%20Bao&amp;author=M.%20Zhou&amp;author=Y.%20Cui&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=W480-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki372" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki372" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki372"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15980516" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thusberg</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Vihinen</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Pathogenic or not? And if so, then how? Studying the effects of missense mutations using bioinformatics methods</div>, <div class="source">Hum. Mutat.</div> , <div class="year">2009</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">703</div>-<div class="lpage">14</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/humu.20938" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/humu.20938</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pathogenic%20or%20not%3F%20And%20if%20so%2C%20then%20how%3F%20Studying%20the%20effects%20of%20missense%20mutations%20using%20bioinformatics%20methods&amp;author=J.%20Thusberg&amp;author=M.%20Vihinen&amp;publication_year=2009&amp;journal=Hum.%20Mutat.&amp;volume=30&amp;pages=703-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/humu.v30:5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fhumu.v30:5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fhumu.v30:5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19267389" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ng</div> <div class="given-names">P.C.</div></div>,  <div class="name"><div class="surname">Henikoff</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Predicting the effects of amino acid substitutions on protein function</div>, <div class="source">Annu. Rev. Genomics Hum. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">61</div>-<div class="lpage">80</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1146/annurev.genom.7.080505.115630" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1146/annurev.genom.7.080505.115630</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20the%20effects%20of%20amino%20acid%20substitutions%20on%20protein%20function&amp;author=P.C.%20Ng&amp;author=S.%20Henikoff&amp;publication_year=2006&amp;journal=Annu.%20Rev.%20Genomics%20Hum.%20Genet.&amp;volume=7&amp;pages=61-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genom.7.080505.115630" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genom.7.080505.115630" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genom.7.080505.115630"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16824020" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Boutet</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Lieberherr</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Tognolli</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Schneider</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Bairoch</div> <div class="given-names">A.</div></div>. </div><div class="article-title">UniProtKB/Swiss-Prot</div>, <div class="source">Methods Mol. Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">406</div> (pg. <div class="fpage">89</div>-<div class="lpage">112</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/978-1-59745-535-0_4" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/978-1-59745-535-0_4</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=UniProtKB%2FSwiss-Prot&amp;author=E.%20Boutet&amp;author=D.%20Lieberherr&amp;author=M.%20Tognolli&amp;author=M.%20Schneider&amp;author=A.%20Bairoch&amp;publication_year=2007&amp;journal=Methods%20Mol.%20Biol.&amp;volume=406&amp;pages=89-112" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18287689" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">E.W.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">Basic local alignment search tool</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1990</div>, vol. <div class="volume">215</div> (pg. <div class="fpage">403</div>-<div class="lpage">10</div>)<div class="pub-id">
                [<a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a>]
            </div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.W.%20Myers&amp;author=D.J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol.&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thompson</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Higgins</div> <div class="given-names">D.G.</div></div>,  <div class="name"><div class="surname">Gibson</div> <div class="given-names">T.J.</div></div>. </div><div class="article-title">CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1994</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">4673</div>-<div class="lpage">80</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/22.22.4673" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/22.22.4673</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CLUSTAL%20W%3A%20improving%20the%20sensitivity%20of%20progressive%20multiple%20sequence%20alignment%20through%20sequence%20weighting%2C%20position-specific%20gap%20penalties%20and%20weight%20matrix%20choice&amp;author=J.D.%20Thompson&amp;author=D.G.%20Higgins&amp;author=T.J.%20Gibson&amp;publication_year=1994&amp;journal=Nucleic%20Acids%20Res.&amp;volume=22&amp;pages=4673-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F22.22.4673" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F22.22.4673"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7984417" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Berman</div> <div class="given-names">H.M.</div></div>,  <div class="name"><div class="surname">Battistuz</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Bhat</div> <div class="given-names">T.N.</div></div>, et al. </div><div class="article-title">The Protein Data Bank</div>, <div class="source">Acta Crystallogr. D Biol. Crystallogr.</div> , <div class="year">2002</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">899</div>-<div class="lpage">907</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1107/S0907444902003451" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1107/S0907444902003451</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Protein%20Data%20Bank&amp;author=H.M.%20Berman&amp;author=T.%20Battistuz&amp;author=T.N.%20Bhat&amp;publication_year=2002&amp;journal=Acta%20Crystallogr.%20D%20Biol.%20Crystallogr.&amp;volume=58&amp;pages=899-907" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1107/S0907444902003451" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1107%2FS0907444902003451" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1107%2FS0907444902003451"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12037327" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marti-Renom</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Stuart</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">Fiser</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sanchez</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Melo</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Sali</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Comparative protein structure modeling of genes and genomes</div>, <div class="source">Annu. Rev. Biophys. Biomol. Struct.</div> , <div class="year">2000</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">291</div>-<div class="lpage">325</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1146/annurev.biophys.29.1.291" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1146/annurev.biophys.29.1.291</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20protein%20structure%20modeling%20of%20genes%20and%20genomes&amp;author=M.A.%20Marti-Renom&amp;author=A.C.%20Stuart&amp;author=A.%20Fiser&amp;author=R.%20Sanchez&amp;author=F.%20Melo&amp;author=A.%20Sali&amp;publication_year=2000&amp;journal=Annu.%20Rev.%20Biophys.%20Biomol.%20Struct.&amp;volume=29&amp;pages=291-325" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.biophys.29.1.291" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.biophys.29.1.291" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.biophys.29.1.291"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10940251" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shen</div> <div class="given-names">M.Y.</div></div>,  <div class="name"><div class="surname">Sali</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Statistical potential for assessment and prediction of protein structures</div>, <div class="source">Protein Sci.</div> , <div class="year">2006</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">2507</div>-<div class="lpage">24</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1110/ps.062416606" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1110/ps.062416606</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistical%20potential%20for%20assessment%20and%20prediction%20of%20protein%20structures&amp;author=M.Y.%20Shen&amp;author=A.%20Sali&amp;publication_year=2006&amp;journal=Protein%20Sci.&amp;volume=15&amp;pages=2507-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1110/(ISSN)1469-896X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1110%2F(ISSN)1469-896X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1110%2F(ISSN)1469-896X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17075131" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shrake</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Rupley</div> <div class="given-names">J.A.</div></div>. </div><div class="article-title">Environment and exposure to solvent of protein atoms. Lysozyme and insulin</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1973</div>, vol. <div class="volume">79</div> (pg. <div class="fpage">351</div>-<div class="lpage">71</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/0022-2836(73)90011-9" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/0022-2836(73)90011-9</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Environment%20and%20exposure%20to%20solvent%20of%20protein%20atoms.%20Lysozyme%20and%20insulin&amp;author=A.%20Shrake&amp;author=J.A.%20Rupley&amp;publication_year=1973&amp;journal=J.%20Mol.%20Biol.&amp;volume=79&amp;pages=351-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(73)90011-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(73)90011-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(73)90011-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4760134" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ward</div> <div class="given-names">J.J.</div></div>,  <div class="name"><div class="surname">McGuffin</div> <div class="given-names">L.J.</div></div>,  <div class="name"><div class="surname">Bryson</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Buxton</div> <div class="given-names">B.F.</div></div>,  <div class="name"><div class="surname">Jones</div> <div class="given-names">D.T.</div></div>. </div><div class="article-title">The DISOPRED server for the prediction of protein disorder</div>, <div class="source">Bioinformatics</div> , <div class="year">2004</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">2138</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/bth195" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/bth195</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20DISOPRED%20server%20for%20the%20prediction%20of%20protein%20disorder&amp;author=J.J.%20Ward&amp;author=L.J.%20McGuffin&amp;author=K.%20Bryson&amp;author=B.F.%20Buxton&amp;author=D.T.%20Jones&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=2138-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bth195" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbth195" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbth195"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15044227" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sunyaev</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ramensky</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Bork</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Towards a structural basis of human non-synonymous single nucleotide polymorphisms</div>, <div class="source">Trends Genet.</div> , <div class="year">2000</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">198</div>-<div class="lpage">200</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S0168-9525(00)01988-0" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S0168-9525(00)01988-0</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Towards%20a%20structural%20basis%20of%20human%20non-synonymous%20single%20nucleotide%20polymorphisms&amp;author=S.%20Sunyaev&amp;author=V.%20Ramensky&amp;author=P.%20Bork&amp;publication_year=2000&amp;journal=Trends%20Genet.&amp;volume=16&amp;pages=198-200" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(00)01988-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(00)01988-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(00)01988-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10782110" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vitkup</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Sander</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Church</div> <div class="given-names">G.M.</div></div>. </div><div class="article-title">The amino-acid mutational spectrum of human genetic disease</div>, <div class="source">Genome Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">4</div> pg. <div class="fpage">R72</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/gb-2003-4-11-r72" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/gb-2003-4-11-r72</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20amino-acid%20mutational%20spectrum%20of%20human%20genetic%20disease&amp;author=D.%20Vitkup&amp;author=C.%20Sander&amp;author=G.M.%20Church&amp;publication_year=2003&amp;journal=Genome%20Biol.&amp;volume=4&amp;pages=R72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2003-4-11-r72" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2003-4-11-r72" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2003-4-11-r72"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14611658" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Moult</div> <div class="given-names">J.</div></div>. </div><div class="article-title">SNPs, protein structure, and disease</div>, <div class="source">Hum. Mutat.</div> , <div class="year">2001</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">263</div>-<div class="lpage">70</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/humu.22" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/humu.22</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNPs%2C%20protein%20structure%2C%20and%20disease&amp;author=Z.%20Wang&amp;author=J.%20Moult&amp;publication_year=2001&amp;journal=Hum.%20Mutat.&amp;volume=17&amp;pages=263-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1098-1004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1098-1004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1098-1004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11295823" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ferrer-Costa</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Orozco</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">de la Cruz</div> <div class="given-names">X.</div></div>. </div><div class="article-title">Characterization of disease-associated single amino acid polymorphisms in terms of sequence and structure properties</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2002</div>, vol. <div class="volume">315</div> (pg. <div class="fpage">771</div>-<div class="lpage">86</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1006/jmbi.2001.5255" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1006/jmbi.2001.5255</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20disease-associated%20single%20amino%20acid%20polymorphisms%20in%20terms%20of%20sequence%20and%20structure%20properties&amp;author=C.%20Ferrer-Costa&amp;author=M.%20Orozco&amp;author=X.%20de%20la%20Cruz&amp;publication_year=2002&amp;journal=J.%20Mol.%20Biol.&amp;volume=315&amp;pages=771-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.2001.5255" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.2001.5255" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.2001.5255"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11812146" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kono</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Yuasa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nishiue</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yura</div> <div class="given-names">K.</div></div>. </div><div class="article-title">coliSNP database server mapping nsSNPs on protein structures</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D409</div>-<div class="lpage">13</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkm801" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkm801</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=coliSNP%20database%20server%20mapping%20nsSNPs%20on%20protein%20structures&amp;author=H.%20Kono&amp;author=T.%20Yuasa&amp;author=S.%20Nishiue&amp;author=K.%20Yura&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D409-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm801" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm801" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm801"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17921498" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">J.W.</div></div>,  <div class="name"><div class="surname">Romero</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Uversky</div> <div class="given-names">V.N.</div></div>,  <div class="name"><div class="surname">Dunker</div> <div class="given-names">A.K.</div></div>. </div><div class="article-title">Conservation of intrinsic disorder in protein domains and families: I. A database of conserved predicted disordered regions</div>, <div class="source">J. Proteome Res.</div> , <div class="year">2006</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">879</div>-<div class="lpage">87</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1021/pr060048x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1021/pr060048x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conservation%20of%20intrinsic%20disorder%20in%20protein%20domains%20and%20families%3A%20I.%20A%20database%20of%20conserved%20predicted%20disordered%20regions&amp;author=J.W.%20Chen&amp;author=P.%20Romero&amp;author=V.N.%20Uversky&amp;author=A.K.%20Dunker&amp;publication_year=2006&amp;journal=J.%20Proteome%20Res.&amp;volume=5&amp;pages=879-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/pr060048x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fpr060048x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fpr060048x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16602695" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schuster-Bockler</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Bateman</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Protein interactions in human genetic diseases</div>, <div class="source">Genome Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">R9</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/gb-2008-9-1-r9" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/gb-2008-9-1-r9</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20interactions%20in%20human%20genetic%20diseases&amp;author=B.%20Schuster-Bockler&amp;author=A.%20Bateman&amp;publication_year=2008&amp;journal=Genome%20Biol.&amp;volume=9&amp;pages=R9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2008-9-1-r9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2008-9-1-r9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2008-9-1-r9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18199329" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Holland</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">DeLeo</div> <div class="given-names">F.R.</div></div>,  <div class="name"><div class="surname">Elloumi</div> <div class="given-names">H.Z.</div></div>, et al. </div><div class="article-title">STAT3 mutations in the hyper-IgE syndrome</div>, <div class="source">N. Engl. J. Med.</div> , <div class="year">2007</div>, vol. <div class="volume">357</div> (pg. <div class="fpage">1608</div>-<div class="lpage">19</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1056/NEJMoa073687" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1056/NEJMoa073687</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=STAT3%20mutations%20in%20the%20hyper-IgE%20syndrome&amp;author=S.M.%20Holland&amp;author=F.R.%20DeLeo&amp;author=H.Z.%20Elloumi&amp;publication_year=2007&amp;journal=N.%20Engl.%20J.%20Med.&amp;volume=357&amp;pages=1608-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1056/NEJMoa073687" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1056%2FNEJMoa073687" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1056%2FNEJMoa073687"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17881745" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Minegishi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Saito</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tsuchiya</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Dominant-negative mutations in the DNA-binding domain of STAT3 cause hyper-IgE syndrome</div>, <div class="source">Nature</div> , <div class="year">2007</div>, vol. <div class="volume">448</div> (pg. <div class="fpage">1058</div>-<div class="lpage">62</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature06096" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature06096</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dominant-negative%20mutations%20in%20the%20DNA-binding%20domain%20of%20STAT3%20cause%20hyper-IgE%20syndrome&amp;author=Y.%20Minegishi&amp;author=M.%20Saito&amp;author=S.%20Tsuchiya&amp;publication_year=2007&amp;journal=Nature&amp;volume=448&amp;pages=1058-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06096" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06096" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06096"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17676033" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Finn</div> <div class="given-names">R.D.</div></div>,  <div class="name"><div class="surname">Tate</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Mistry</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">The Pfam protein families database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D281</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkm960" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkm960</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Pfam%20protein%20families%20database&amp;author=R.D.%20Finn&amp;author=J.%20Tate&amp;author=J.%20Mistry&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D281-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm960" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm960" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm960"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18039703" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sayers</div> <div class="given-names">E.W.</div></div>,  <div class="name"><div class="surname">Barrett</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Benson</div> <div class="given-names">D.A.</div></div>, et al. </div><div class="article-title">Database resources of the National Center for Biotechnology Information</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">D5</div>-<div class="lpage">15</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkn741" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkn741</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Database%20resources%20of%20the%20National%20Center%20for%20Biotechnology%20Information&amp;author=E.W.%20Sayers&amp;author=T.%20Barrett&amp;author=D.A.%20Benson&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=D5-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn741" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn741" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn741"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18940862" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Keshava Prasad</div> <div class="given-names">T.S.</div></div>,  <div class="name"><div class="surname">Goel</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Kandasamy</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Human Protein Reference Database—2009 update</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">D767</div>-<div class="lpage">72</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkn892" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkn892</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20Protein%20Reference%20Database%E2%80%942009%20update&amp;author=T.S.%20Keshava%20Prasad&amp;author=R.%20Goel&amp;author=K.%20Kandasamy&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=D767-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn892" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn892" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn892"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18988627" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rebhan</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Chalifa-Caspi</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Prilusky</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lancet</div> <div class="given-names">D.</div></div>. </div><div class="article-title">GeneCards: a novel functional genomics compendium with automated data mining and query reformulation support</div>, <div class="source">Bioinformatics</div> , <div class="year">1998</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">656</div>-<div class="lpage">64</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/14.8.656" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/14.8.656</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GeneCards%3A%20a%20novel%20functional%20genomics%20compendium%20with%20automated%20data%20mining%20and%20query%20reformulation%20support&amp;author=M.%20Rebhan&amp;author=V.%20Chalifa-Caspi&amp;author=J.%20Prilusky&amp;author=D.%20Lancet&amp;publication_year=1998&amp;journal=Bioinformatics&amp;volume=14&amp;pages=656-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/14.8.656" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F14.8.656" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F14.8.656"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9789091" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hijikata</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kitamura</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kimura</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Construction of an open-access database that integrates cross-reference information from the transcriptome and proteome of immune cells</div>, <div class="source">Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">2934</div>-<div class="lpage">41</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/btm430" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/btm430</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20an%20open-access%20database%20that%20integrates%20cross-reference%20information%20from%20the%20transcriptome%20and%20proteome%20of%20immune%20cells&amp;author=A.%20Hijikata&amp;author=H.%20Kitamura&amp;author=Y.%20Kimura&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2934-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm430" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm430" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm430"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17893089" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kanehisa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Goto</div> <div class="given-names">S.</div></div>. </div><div class="article-title">KEGG: kyoto encyclopedia of genes and genomes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2000</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">27</div>-<div class="lpage">30</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/28.1.27" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/28.1.27</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KEGG%3A%20kyoto%20encyclopedia%20of%20genes%20and%20genomes&amp;author=M.%20Kanehisa&amp;author=S.%20Goto&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res.&amp;volume=28&amp;pages=27-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.1.27" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.1.27" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.1.27"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10592173" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Becker</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Groner</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Muller</div> <div class="given-names">C.W.</div></div>. </div><div class="article-title">Three-dimensional structure of the Stat3beta homodimer bound to DNA</div>, <div class="source">Nature</div> , <div class="year">1998</div>, vol. <div class="volume">394</div> (pg. <div class="fpage">145</div>-<div class="lpage">51</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/28101" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/28101</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Three-dimensional%20structure%20of%20the%20Stat3beta%20homodimer%20bound%20to%20DNA&amp;author=S.%20Becker&amp;author=B.%20Groner&amp;author=C.W.%20Muller&amp;publication_year=1998&amp;journal=Nature&amp;volume=394&amp;pages=145-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/28101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F28101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F28101"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9671298" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kent</div> <div class="given-names">W.J.</div></div>. </div><div class="article-title">BLAT—the BLAST-like alignment tool</div>, <div class="source">Genome Res.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">656</div>-<div class="lpage">64</div>)<div class="pub-id">
                [<a href="http://www.ncbi.nlm.nih.gov/pubmed/11932250" class="link link-pub-id" target="_blank">PubMed</a>]
            </div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BLAT%E2%80%94the%20BLAST-like%20alignment%20tool&amp;author=W.J.%20Kent&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=656-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.229202. Article published online before March 2002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.229202. Article published online before March 2002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.229202. Article published online before March 2002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11932250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ010C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ010C59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ahn</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">T.H.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">The first Korean genome sequence and analysis: full genome sequencing for a socio-ethnic group</div>, <div class="source">Genome Res.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">1622</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.092197.109" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.092197.109</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20first%20Korean%20genome%20sequence%20and%20analysis%3A%20full%20genome%20sequencing%20for%20a%20socio-ethnic%20group&amp;author=S.M.%20Ahn&amp;author=T.H.%20Kim&amp;author=S.%20Lee&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=1622-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.092197.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.092197.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.092197.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19470904" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2010. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=353879&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/3/10.1093/dnares/dsq010/2/dsq010-dsq010_supp_table.xls?Expires=2147483647&amp;Signature=eSsXZpZVWkdOSNTuIVKmhBWyhiJT1pmkoG0-X14AoQKoeKUjfJkkZXYN2DBqVMr4eHvF-CGpnTXMogXHEdUoLxkIdjJQANTYNxqD-mHiWE9AqME8CxUF1cpPYAx1cAYM2pvHs-yPJt9gGv63dyldzE1JCwXKsjVIYzsQhPxz8bKOQobg8AZ-iY0DcC7XvWOv3eU~B0DdkV8MXot7bZIaO6kY9FqpHdK22eR4dt39zkVN06F3JqE9FMwqHGt8p1mPtQeSnis~9mhTDHA658cxlcJCq58er7i~yOtowxfHh5WkTnBs8BbC9PWk4zrkyvn-9bbz56qVaOHHZ9s8KIqRNg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">185</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000279411300006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">14</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsq010"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="353879"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000279411300006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f17%2f3%2f197%2f353879">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000279411300006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (14)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F17%2F3%2F197%2F353879" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/353879'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Mutation@A Glance: An Integrative Web Application for Analysing Mutations from Human Genetic Diseases" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsq010");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f17%2f3%2f197%2f353879">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

