

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":0,"applicationTime":1796,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/16/4/213/391702","@type":"ScholarlyArticle","name":"High-density Integrated Linkage Map Based on SSR Markers in Soybean","datePublished":"2009-06-16","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/16/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2009-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsp010","keywords":["EST-derived SSR marker","integrated linkage map","microsatellite marker","polymorphism information content"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/16/4/213/391702","author":[{"name":"Hwang, Tae-Young","affiliation":"1  National Agricultural Research Center for Hokkaido Region ,  1 Hitsujigaoka, Toyohira, Sapporo, Hokkaido 062-8555 ,  Japan    2  Graduate School of Agriculture ,  Hokkaido University ,  Kita 9 Nishi 9, Kita, Sapporo, Hokkaido 060-8589 ,  Japan","@type":"Person"},{"name":"Sayama, Takashi","affiliation":"1  National Agricultural Research Center for Hokkaido Region ,  1 Hitsujigaoka, Toyohira, Sapporo, Hokkaido 062-8555 ,  Japan","@type":"Person"},{"name":"Takahashi, Masakazu","affiliation":"3  National Agricultural Research Center for Kyushu Okinawa Region ,  2421 Suya, Koshi, Kumamoto 861-1192 ,  Japan","@type":"Person"},{"name":"Takada, Yoshitake","affiliation":"4  National Agricultural Research Center for Tohoku Region ,  297 Uenodai, Kariwano, Daisen, Akita 019-2112 ,  Japan","@type":"Person"},{"name":"Nakamoto, Yumi","affiliation":"1  National Agricultural Research Center for Hokkaido Region ,  1 Hitsujigaoka, Toyohira, Sapporo, Hokkaido 062-8555 ,  Japan","@type":"Person"},{"name":"Funatsuki, Hideyuki","affiliation":"1  National Agricultural Research Center for Hokkaido Region ,  1 Hitsujigaoka, Toyohira, Sapporo, Hokkaido 062-8555 ,  Japan","@type":"Person"},{"name":"Hisano, Hiroshi","affiliation":"5  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Sasamoto, Shigemi","affiliation":"5  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Sato, Shusei","affiliation":"5  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Tabata, Satoshi","affiliation":"5  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kono, Izumi","affiliation":"6  Institute of Society for Techno-Innovation of Agriculture, Forestry, and Fisheries ,  446-1 Kamiyokoba, Tsukuba, Ibaraki 305-0854 ,  Japan","@type":"Person"},{"name":"Hoshi, Masako","affiliation":"7  Faculty of Horticulture ,  Chiba University ,  648 Matsudo, Chiba 271-8510 ,  Japan","@type":"Person"},{"name":"Hanawa, Masayoshi","affiliation":"7  Faculty of Horticulture ,  Chiba University ,  648 Matsudo, Chiba 271-8510 ,  Japan","@type":"Person"},{"name":"Yano, Chizuru","affiliation":"7  Faculty of Horticulture ,  Chiba University ,  648 Matsudo, Chiba 271-8510 ,  Japan","@type":"Person"},{"name":"Xia, Zhengjun","affiliation":"8  National Institute of Agrobiological Sciences ,  2-1-2 Kannondai, Tsukuba, Ibaraki 305-8602 ,  Japan","@type":"Person"},{"name":"Harada, Kyuya","affiliation":"8  National Institute of Agrobiological Sciences ,  2-1-2 Kannondai, Tsukuba, Ibaraki 305-8602 ,  Japan","@type":"Person"},{"name":"Kitamura, Keisuke","affiliation":"2  Graduate School of Agriculture ,  Hokkaido University ,  Kita 9 Nishi 9, Kita, Sapporo, Hokkaido 060-8589 ,  Japan","@type":"Person"},{"name":"Ishimoto, Masao","affiliation":"1  National Agricultural Research Center for Hokkaido Region ,  1 Hitsujigaoka, Toyohira, Sapporo, Hokkaido 062-8555 ,  Japan","@type":"Person"}],"description":"Abstract.  A well-saturated molecular linkage map is a prerequisite for modern plant breeding. Several genetic maps have been developed for soybean with various","pageStart":"213","pageEnd":"225","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/4/1/m_cover.gif?Expires=2147483647&Signature=KixCr2M0LlinzZEFB8nkzQbIGvWUPfCHMMnb1HEM4rLsLPtPw1WxHRM~3wSTgHca1h7ODdDw6cqpZwDcBtDB5cvMmAMOurPOoQS0BIzs30jp8r~JAQtWE3og9uxAEzWt7tYCTXVUp3R96yLrM8jbua3o2HYemMJpVAvuNM2mdcIDjO8dDR4VLEhZgMz9vD4Mgo~7X40rIJRdU89cuFZ-es1advHNbP4DSmboI0fmFNs-0MVR206dhxd0q7sk6et-~kJk~GXjkkLEy8a0H2cSmsP3xSxoZDkKVqEWVw2rjyMN144G~brfO6mG0-KjFvnH6w2O1vIyutTwcUiTgCKyRA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"High-density Integrated Linkage Map Based on SSR Markers in Soybean","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/4/1/m_cover.gif?Expires=2147483647&Signature=KixCr2M0LlinzZEFB8nkzQbIGvWUPfCHMMnb1HEM4rLsLPtPw1WxHRM~3wSTgHca1h7ODdDw6cqpZwDcBtDB5cvMmAMOurPOoQS0BIzs30jp8r~JAQtWE3og9uxAEzWt7tYCTXVUp3R96yLrM8jbua3o2HYemMJpVAvuNM2mdcIDjO8dDR4VLEhZgMz9vD4Mgo~7X40rIJRdU89cuFZ-es1advHNbP4DSmboI0fmFNs-0MVR206dhxd0q7sk6et-~kJk~GXjkkLEy8a0H2cSmsP3xSxoZDkKVqEWVw2rjyMN144G~brfO6mG0-KjFvnH6w2O1vIyutTwcUiTgCKyRA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="High-density Integrated Linkage Map Based on SSR Markers in Soybean" />
    <meta property="og:description" content="Abstract.  A well-saturated molecular linkage map is a prerequisite for modern plant breeding. Several genetic maps have been developed for soybean with various" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/16/4/213/391702" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2009-06-16" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Hwang, Tae-Young" /><meta name="citation_author_institution" content="National Agricultural Research Center for Hokkaido Region, 1 Hitsujigaoka, Toyohira, Sapporo, Hokkaido 062-8555, Japan" /><meta name="citation_author_institution" content="Graduate School of Agriculture, Hokkaido University, Kita 9 Nishi 9, Kita, Sapporo, Hokkaido 060-8589, Japan" /><meta name="citation_author" content="Sayama, Takashi" /><meta name="citation_author_institution" content="National Agricultural Research Center for Hokkaido Region, 1 Hitsujigaoka, Toyohira, Sapporo, Hokkaido 062-8555, Japan" /><meta name="citation_author" content="Takahashi, Masakazu" /><meta name="citation_author_institution" content="National Agricultural Research Center for Kyushu Okinawa Region, 2421 Suya, Koshi, Kumamoto 861-1192, Japan" /><meta name="citation_author" content="Takada, Yoshitake" /><meta name="citation_author_institution" content="National Agricultural Research Center for Tohoku Region, 297 Uenodai, Kariwano, Daisen, Akita 019-2112, Japan" /><meta name="citation_author" content="Nakamoto, Yumi" /><meta name="citation_author_institution" content="National Agricultural Research Center for Hokkaido Region, 1 Hitsujigaoka, Toyohira, Sapporo, Hokkaido 062-8555, Japan" /><meta name="citation_author" content="Funatsuki, Hideyuki" /><meta name="citation_author_institution" content="National Agricultural Research Center for Hokkaido Region, 1 Hitsujigaoka, Toyohira, Sapporo, Hokkaido 062-8555, Japan" /><meta name="citation_author" content="Hisano, Hiroshi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Sasamoto, Shigemi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Sato, Shusei" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Tabata, Satoshi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kono, Izumi" /><meta name="citation_author_institution" content="Institute of Society for Techno-Innovation of Agriculture, Forestry, and Fisheries, 446-1 Kamiyokoba, Tsukuba, Ibaraki 305-0854, Japan" /><meta name="citation_author" content="Hoshi, Masako" /><meta name="citation_author_institution" content="Faculty of Horticulture, Chiba University, 648 Matsudo, Chiba 271-8510, Japan" /><meta name="citation_author" content="Hanawa, Masayoshi" /><meta name="citation_author_institution" content="Faculty of Horticulture, Chiba University, 648 Matsudo, Chiba 271-8510, Japan" /><meta name="citation_author" content="Yano, Chizuru" /><meta name="citation_author_institution" content="Faculty of Horticulture, Chiba University, 648 Matsudo, Chiba 271-8510, Japan" /><meta name="citation_author" content="Xia, Zhengjun" /><meta name="citation_author_institution" content="National Institute of Agrobiological Sciences, 2-1-2 Kannondai, Tsukuba, Ibaraki 305-8602, Japan" /><meta name="citation_author" content="Harada, Kyuya" /><meta name="citation_author_institution" content="National Institute of Agrobiological Sciences, 2-1-2 Kannondai, Tsukuba, Ibaraki 305-8602, Japan" /><meta name="citation_author" content="Kitamura, Keisuke" /><meta name="citation_author_institution" content="Graduate School of Agriculture, Hokkaido University, Kita 9 Nishi 9, Kita, Sapporo, Hokkaido 060-8589, Japan" /><meta name="citation_author" content="Ishimoto, Masao" /><meta name="citation_author_institution" content="National Agricultural Research Center for Hokkaido Region, 1 Hitsujigaoka, Toyohira, Sapporo, Hokkaido 062-8555, Japan" /><meta name="citation_title" content="High-density Integrated Linkage Map Based on SSR Markers in Soybean" /><meta name="citation_firstpage" content="213" /><meta name="citation_lastpage" content="225" /><meta name="citation_doi" content="10.1093/dnares/dsp010" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="16" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2009/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_author=FAS; citation_publisher=United States Department of Agriculture, Washington, DC; Oilseeds: World Markets and Trade. Foreign Agricultural Service (FAS);  citation_year=2008; " /><meta name="citation_reference" content="citation_title=Qualitative genetics and cytogenetics;  citation_edition=2nd Ed.; citation_publisher=Agronomy, Madison, Wisconsin; Soybeans: Improvement, Production, and Uses;  citation_year=1987;  citation_pages=135-209; " /><meta name="citation_reference" content="citation_title=RELP mapping in soybean: association between marker loci and variation in quantitative traits; Genetics;  citation_year=1990;  citation_volume=126;  citation_pages=735-742; " /><meta name="citation_reference" content="citation_title=DNA polymorphisms amplified by arbitrary primers are useful as genetic markers; Nucleic Acids Res.;  citation_year=1990;  citation_volume=18;  citation_pages=6531-6535; " /><meta name="citation_reference" content="citation_title=Length polymorphisms of simple sequence repeat DNA in soybean; Genetics;  citation_year=1992;  citation_volume=132;  citation_pages=1131-1139; " /><meta name="citation_reference" content="citation_title=Integration of simple sequence repeat DNA markers into a soybean linkage map; Crop Sci.;  citation_year=1995;  citation_volume=35;  citation_pages=1439-1445; " /><meta name="citation_reference" content="citation_title=A soybean transcript map: gene distribution, haplotype and single-nucleotide polymorphism analysis; Genetics;  citation_year=2007;  citation_volume=176;  citation_pages=685-696; " /><meta name="citation_reference" content="citation_title=PCR-amplified microsatellites as markers in plant genetics; Plant J.;  citation_year=1993;  citation_volume=3;  citation_pages=715-723; " /><meta name="citation_reference" content="citation_title=Microsatellite and amplified sequence length polymorphisms in cultivated and wild soybean; Genome;  citation_year=1995;  citation_volume=38;  citation_pages=715-723; " /><meta name="citation_reference" content="citation_title=Automated sizing of fluorescent-labeled simple sequence repeat (SSR) markers to assay genetic variation in soybean; Theor. Appl. Genet.;  citation_year=1997;  citation_volume=95;  citation_pages=723-733; " /><meta name="citation_reference" content="citation_title=Molecular marker diversity of SCN-resistant sources in soybean; Genome;  citation_year=2006;  citation_volume=49;  citation_pages=938-949; " /><meta name="citation_reference" content="citation_title=Genetic diversity of cultivated and wild soybeans including Japanese elite cultivars as revealed by length polymorphism of SSR markers; Breed. Sci.;  citation_year=2008;  citation_volume=58;  citation_pages=315-323; " /><meta name="citation_reference" content="citation_title=Microsatellite fingerprinting and mapping of soybean; Methods Mol. Cell. Biol.;  citation_year=1994;  citation_volume=5;  citation_pages=49-61; " /><meta name="citation_reference" content="citation_title=RFLP and microsatellite mapping of a gene for soybean mosaic virus resistance; Phytopathology;  citation_year=1994;  citation_volume=84;  citation_pages=60-64; " /><meta name="citation_reference" content="citation_title=PCR-multiplexes for a genome-wide framework of simple sequence repeat marker loci in cultivated sunflower; Theor. Appl. Genet.;  citation_year=2003;  citation_volume=107;  citation_pages=6-19; " /><meta name="citation_reference" content="citation_title=An integrated genetic linkage map of the soybean genome; Crop Sci.;  citation_year=1999;  citation_volume=39;  citation_pages=1464-1490; " /><meta name="citation_reference" content="Cregan P. B An integrated genetic linkage map of the soybean July 2003 2003 http://bldg6.arsusda.gov/~pooley/soy/cregan/soymap.htm (July 2003) " /><meta name="citation_reference" content="citation_title=A new integrated genetic linkage map of the soybean; Theor. Appl. Genet.;  citation_year=2004;  citation_volume=109;  citation_pages=122-128; " /><meta name="citation_reference" content="citation_title=Nuclear DNA content of some important plant species; Plant Mol. Biol. Rep.;  citation_year=1991;  citation_volume=9;  citation_pages=208-218; " /><meta name="citation_reference" content="citation_title=An integrated high-density linkage map of soybean with RFLP, SSR, STS, and AFLP markers using a single F2 population; DNA Res.;  citation_year=2007;  citation_volume=14;  citation_pages=257-269; " /><meta name="citation_reference" content="citation_title=Characterization of the soybean genome using EST-derived microsatellite markers; DNA Res.;  citation_year=2007;  citation_volume=14;  citation_pages=271-281; " /><meta name="citation_reference" content="citation_title=An informative linkage map of soybean reveals QTLs for flowering time, leaflet morphology and regions of segregation distortion; DNA Res.;  citation_year=2001;  citation_volume=8;  citation_pages=61-72; " /><meta name="citation_reference" content="citation_title=Analysis of QTLs for reproductive development and seed quality traits in soybean using recombinant inbred lines; Breed. Sci.;  citation_year=2004;  citation_volume=54;  citation_pages=399-407; " /><meta name="citation_reference" content="citation_title=Comprehensive structural analysis of the genome of red clover (Trifolium pratense L.); DNA Res.;  citation_year=2005;  citation_volume=12;  citation_pages=301-364; " /><meta name="citation_reference" content="citation_publisher=Plant Research International, Wageningen, the Netherlands; JoinMap 4.0 Software for the Calculation of Genetic Linkage Maps in Experimental Populations;  citation_year=2006; " /><meta name="citation_reference" content=" citation_edition=3rd Ed.; citation_publisher=Whitehead Institute, Massachusetts, Cambridge; Constructing Genetic Maps with MAPMAKER/EXP 3.0, Whitehead Institute Technical Report;  citation_year=1992; " /><meta name="citation_reference" content="citation_title=The estimation of map distance from recombination values; Ann. Eugen.;  citation_year=1944;  citation_volume=12;  citation_pages=172-175; " /><meta name="citation_reference" content="citation_title=MapChart: software for the graphical presentation of linkage maps and QTLs; J. Hered.;  citation_year=2002;  citation_volume=93;  citation_pages=77-78; " /><meta name="citation_reference" content="citation_title=Evaluation of soybean RFLP marker diversity in adapted germ plasm; Theor. Appl. Genet.;  citation_year=1992;  citation_volume=85;  citation_pages=205-212; " /><meta name="citation_reference" content="citation_title=Characterization and identification of (CT)n microsatellites in soybean using sheared genomic libraries; DNA Res.;  citation_year=2000;  citation_volume=7;  citation_pages=103-110; " /><meta name="citation_reference" content="citation_title=Two simple sequence repeat markers to select for soybean cyst nematode resistance conditioned by rhg1 locus; Theor. Appl. Genet.;  citation_year=1999;  citation_volume=99;  citation_pages=811-818; " /><meta name="citation_reference" content="SoyBase Soybean chromosome number to linkage group assignments 2008 http://soybase.org/LG2Xsome.php (30 July 2008) " /><meta name="citation_reference" content="citation_title=Variability for recombination frequencies in the AP12 soybean population; Crop Sci.;  citation_year=1990;  citation_volume=30;  citation_pages=545-549; " /><meta name="citation_reference" content="citation_title=Development and annotation of perennial Triticeae ESTs and SSR markers; Genome;  citation_year=2008;  citation_volume=51;  citation_pages=779-788; " /><meta name="citation_reference" content="citation_title=Application of comparative genomics in developing molecular markers tightly linked to the virus resistance gene Rsv4 in soybean; Genome;  citation_year=2006;  citation_volume=49;  citation_pages=380-388; " /><meta name="citation_reference" content="citation_title=Estimating genome conservation between crop and model legume species; Proc. Natl Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=15289-15294; " /><meta name="citation_reference" content="The Department of Energy's Joint Genome Institute and the Center for Integrative Genomics Phytozome Glyma 1 2008 http://www.phytozome.net/soybean.php (8 December 2008) " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/16/4/213/1104533/dsp010.pdf" /><meta name="description" content="Abstract.  A well-saturated molecular linkage map is a prerequisite for modern plant breeding. Several genetic maps have been developed for soybean with various" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/16/4/213/391702" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/16/4/213/391702" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/16/4/213/391702");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsp010';
    </script>
<title>
	High-density Integrated Linkage Map Based on SSR Markers in Soybean | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./391702" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BaNoPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJId2FuZywgVGFlLVlvdW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgQWdyaWN1bHR1cmFsIFJlc2VhcmNoIENlbnRlciBmb3IgSG9ra2FpZG8gUmVnaW9uLCAxIEhpdHN1amlnYW9rYSwgVG95b2hpcmEsIFNhcHBvcm8sIEhva2thaWRvIDA2Mi04NTU1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBBZ3JpY3VsdHVyZSwgSG9ra2FpZG8gVW5pdmVyc2l0eSwgS2l0YSA5IE5pc2hpIDksIEtpdGEsIFNhcHBvcm8sIEhva2thaWRvIDA2MC04NTg5LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNheWFtYSwgVGFrYXNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEFncmljdWx0dXJhbCBSZXNlYXJjaCBDZW50ZXIgZm9yIEhva2thaWRvIFJlZ2lvbiwgMSBIaXRzdWppZ2Fva2EsIFRveW9oaXJhLCBTYXBwb3JvLCBIb2trYWlkbyAwNjItODU1NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYWthaGFzaGksIE1hc2FrYXp1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgQWdyaWN1bHR1cmFsIFJlc2VhcmNoIENlbnRlciBmb3IgS3l1c2h1IE9raW5hd2EgUmVnaW9uLCAyNDIxIFN1eWEsIEtvc2hpLCBLdW1hbW90byA4NjEtMTE5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYWthZGEsIFlvc2hpdGFrZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEFncmljdWx0dXJhbCBSZXNlYXJjaCBDZW50ZXIgZm9yIFRvaG9rdSBSZWdpb24sIDI5NyBVZW5vZGFpLCBLYXJpd2FubywgRGFpc2VuLCBBa2l0YSAwMTktMjExMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWthbW90bywgWXVtaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEFncmljdWx0dXJhbCBSZXNlYXJjaCBDZW50ZXIgZm9yIEhva2thaWRvIFJlZ2lvbiwgMSBIaXRzdWppZ2Fva2EsIFRveW9oaXJhLCBTYXBwb3JvLCBIb2trYWlkbyAwNjItODU1NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGdW5hdHN1a2ksIEhpZGV5dWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgQWdyaWN1bHR1cmFsIFJlc2VhcmNoIENlbnRlciBmb3IgSG9ra2FpZG8gUmVnaW9uLCAxIEhpdHN1amlnYW9rYSwgVG95b2hpcmEsIFNhcHBvcm8sIEhva2thaWRvIDA2Mi04NTU1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikhpc2FubywgSGlyb3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXNhbW90bywgU2hpZ2VtaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXRvLCBTaHVzZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFiYXRhLCBTYXRvc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Iktvbm8sIEl6dW1pIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIFNvY2lldHkgZm9yIFRlY2huby1Jbm5vdmF0aW9uIG9mIEFncmljdWx0dXJlLCBGb3Jlc3RyeSwgYW5kIEZpc2hlcmllcywgNDQ2LTEgS2FtaXlva29iYSwgVHN1a3ViYSwgSWJhcmFraSAzMDUtMDg1NCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIb3NoaSwgTWFzYWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRmFjdWx0eSBvZiBIb3J0aWN1bHR1cmUsIENoaWJhIFVuaXZlcnNpdHksIDY0OCBNYXRzdWRvLCBDaGliYSAyNzEtODUxMCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIYW5hd2EsIE1hc2F5b3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZhY3VsdHkgb2YgSG9ydGljdWx0dXJlLCBDaGliYSBVbml2ZXJzaXR5LCA2NDggTWF0c3VkbywgQ2hpYmEgMjcxLTg1MTAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFubywgQ2hpenVydSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZhY3VsdHkgb2YgSG9ydGljdWx0dXJlLCBDaGliYSBVbml2ZXJzaXR5LCA2NDggTWF0c3VkbywgQ2hpYmEgMjcxLTg1MTAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWGlhLCBaaGVuZ2p1biIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBBZ3JvYmlvbG9naWNhbCBTY2llbmNlcywgMi0xLTIgS2Fubm9uZGFpLCBUc3VrdWJhLCBJYmFyYWtpIDMwNS04NjAyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhhcmFkYSwgS3l1eWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgQWdyb2Jpb2xvZ2ljYWwgU2NpZW5jZXMsIDItMS0yIEthbm5vbmRhaSwgVHN1a3ViYSwgSWJhcmFraSAzMDUtODYwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLaXRhbXVyYSwgS2Vpc3VrZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBBZ3JpY3VsdHVyZSwgSG9ra2FpZG8gVW5pdmVyc2l0eSwgS2l0YSA5IE5pc2hpIDksIEtpdGEsIFNhcHBvcm8sIEhva2thaWRvIDA2MC04NTg5LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IklzaGltb3RvLCBNYXNhbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEFncmljdWx0dXJhbCBSZXNlYXJjaCBDZW50ZXIgZm9yIEhva2thaWRvIFJlZ2lvbiwgMSBIaXRzdWppZ2Fva2EsIFRveW9oaXJhLCBTYXBwb3JvLCBIb2trYWlkbyAwNjItODU1NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkhpZ2gtZGVuc2l0eSBJbnRlZ3JhdGVkIExpbmthZ2UgTWFwIEJhc2VkIG9uIFNTUiBNYXJrZXJzIGluIFNveWJlYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyMTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjIyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzcDAxMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDA5LzA4LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fYXV0aG9yPUZBUzsgY2l0YXRpb25fcHVibGlzaGVyPVVuaXRlZCBTdGF0ZXMgRGVwYXJ0bWVudCBvZiBBZ3JpY3VsdHVyZSwgV2FzaGluZ3RvbiwgREM7IE9pbHNlZWRzOiBXb3JsZCBNYXJrZXRzIGFuZCBUcmFkZS4gRm9yZWlnbiBBZ3JpY3VsdHVyYWwgU2VydmljZSAoRkFTKTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbGl0YXRpdmUgZ2VuZXRpY3MgYW5kIGN5dG9nZW5ldGljczsgIGNpdGF0aW9uX2VkaXRpb249Mm5kIEVkLjsgY2l0YXRpb25fcHVibGlzaGVyPUFncm9ub215LCBNYWRpc29uLCBXaXNjb25zaW47IFNveWJlYW5zOiBJbXByb3ZlbWVudCwgUHJvZHVjdGlvbiwgYW5kIFVzZXM7ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICBjaXRhdGlvbl9wYWdlcz0xMzUtMjA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SRUxQIG1hcHBpbmcgaW4gc295YmVhbjogYXNzb2NpYXRpb24gYmV0d2VlbiBtYXJrZXIgbG9jaSBhbmQgdmFyaWF0aW9uIGluIHF1YW50aXRhdGl2ZSB0cmFpdHM7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTEyNjsgIGNpdGF0aW9uX3BhZ2VzPTczNS03NDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSBwb2x5bW9ycGhpc21zIGFtcGxpZmllZCBieSBhcmJpdHJhcnkgcHJpbWVycyBhcmUgdXNlZnVsIGFzIGdlbmV0aWMgbWFya2VyczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9NjUzMS02NTM1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MZW5ndGggcG9seW1vcnBoaXNtcyBvZiBzaW1wbGUgc2VxdWVuY2UgcmVwZWF0IEROQSBpbiBzb3liZWFuOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzI7ICBjaXRhdGlvbl9wYWdlcz0xMTMxLTExMzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVncmF0aW9uIG9mIHNpbXBsZSBzZXF1ZW5jZSByZXBlYXQgRE5BIG1hcmtlcnMgaW50byBhIHNveWJlYW4gbGlua2FnZSBtYXA7IENyb3AgU2NpLjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPTE0MzktMTQ0NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzb3liZWFuIHRyYW5zY3JpcHQgbWFwOiBnZW5lIGRpc3RyaWJ1dGlvbiwgaGFwbG90eXBlIGFuZCBzaW5nbGUtbnVjbGVvdGlkZSBwb2x5bW9ycGhpc20gYW5hbHlzaXM7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE3NjsgIGNpdGF0aW9uX3BhZ2VzPTY4NS02OTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBDUi1hbXBsaWZpZWQgbWljcm9zYXRlbGxpdGVzIGFzIG1hcmtlcnMgaW4gcGxhbnQgZ2VuZXRpY3M7IFBsYW50IEouOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz03MTUtNzIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb3NhdGVsbGl0ZSBhbmQgYW1wbGlmaWVkIHNlcXVlbmNlIGxlbmd0aCBwb2x5bW9ycGhpc21zIGluIGN1bHRpdmF0ZWQgYW5kIHdpbGQgc295YmVhbjsgR2Vub21lOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9NzE1LTcyMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXV0b21hdGVkIHNpemluZyBvZiBmbHVvcmVzY2VudC1sYWJlbGVkIHNpbXBsZSBzZXF1ZW5jZSByZXBlYXQgKFNTUikgbWFya2VycyB0byBhc3NheSBnZW5ldGljIHZhcmlhdGlvbiBpbiBzb3liZWFuOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTk1OyAgY2l0YXRpb25fcGFnZXM9NzIzLTczMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIG1hcmtlciBkaXZlcnNpdHkgb2YgU0NOLXJlc2lzdGFudCBzb3VyY2VzIGluIHNveWJlYW47IEdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT00OTsgIGNpdGF0aW9uX3BhZ2VzPTkzOC05NDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgZGl2ZXJzaXR5IG9mIGN1bHRpdmF0ZWQgYW5kIHdpbGQgc295YmVhbnMgaW5jbHVkaW5nIEphcGFuZXNlIGVsaXRlIGN1bHRpdmFycyBhcyByZXZlYWxlZCBieSBsZW5ndGggcG9seW1vcnBoaXNtIG9mIFNTUiBtYXJrZXJzOyBCcmVlZC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT01ODsgIGNpdGF0aW9uX3BhZ2VzPTMxNS0zMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3Jvc2F0ZWxsaXRlIGZpbmdlcnByaW50aW5nIGFuZCBtYXBwaW5nIG9mIHNveWJlYW47IE1ldGhvZHMgTW9sLiBDZWxsLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9NDktNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJGTFAgYW5kIG1pY3Jvc2F0ZWxsaXRlIG1hcHBpbmcgb2YgYSBnZW5lIGZvciBzb3liZWFuIG1vc2FpYyB2aXJ1cyByZXNpc3RhbmNlOyBQaHl0b3BhdGhvbG9neTsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT04NDsgIGNpdGF0aW9uX3BhZ2VzPTYwLTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QQ1ItbXVsdGlwbGV4ZXMgZm9yIGEgZ2Vub21lLXdpZGUgZnJhbWV3b3JrIG9mIHNpbXBsZSBzZXF1ZW5jZSByZXBlYXQgbWFya2VyIGxvY2kgaW4gY3VsdGl2YXRlZCBzdW5mbG93ZXI7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTA3OyAgY2l0YXRpb25fcGFnZXM9Ni0xOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gaW50ZWdyYXRlZCBnZW5ldGljIGxpbmthZ2UgbWFwIG9mIHRoZSBzb3liZWFuIGdlbm9tZTsgQ3JvcCBTY2kuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9MTQ2NC0xNDkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJDcmVnYW4gUC4gQiBBbiBpbnRlZ3JhdGVkIGdlbmV0aWMgbGlua2FnZSBtYXAgb2YgdGhlIHNveWJlYW4gSnVseSAyMDAzIDIwMDMgaHR0cDovL2JsZGc2LmFyc3VzZGEuZ292L35wb29sZXkvc295L2NyZWdhbi9zb3ltYXAuaHRtIChKdWx5IDIwMDMpICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbmV3IGludGVncmF0ZWQgZ2VuZXRpYyBsaW5rYWdlIG1hcCBvZiB0aGUgc295YmVhbjsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDk7ICBjaXRhdGlvbl9wYWdlcz0xMjItMTI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OdWNsZWFyIEROQSBjb250ZW50IG9mIHNvbWUgaW1wb3J0YW50IHBsYW50IHNwZWNpZXM7IFBsYW50IE1vbC4gQmlvbC4gUmVwLjsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MjA4LTIxODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gaW50ZWdyYXRlZCBoaWdoLWRlbnNpdHkgbGlua2FnZSBtYXAgb2Ygc295YmVhbiB3aXRoIFJGTFAsIFNTUiwgU1RTLCBhbmQgQUZMUCBtYXJrZXJzIHVzaW5nIGEgc2luZ2xlIEYyIHBvcHVsYXRpb247IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MjU3LTI2OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbiBvZiB0aGUgc295YmVhbiBnZW5vbWUgdXNpbmcgRVNULWRlcml2ZWQgbWljcm9zYXRlbGxpdGUgbWFya2VyczsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0yNzEtMjgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbmZvcm1hdGl2ZSBsaW5rYWdlIG1hcCBvZiBzb3liZWFuIHJldmVhbHMgUVRMcyBmb3IgZmxvd2VyaW5nIHRpbWUsIGxlYWZsZXQgbW9ycGhvbG9neSBhbmQgcmVnaW9ucyBvZiBzZWdyZWdhdGlvbiBkaXN0b3J0aW9uOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9NjEtNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIFFUTHMgZm9yIHJlcHJvZHVjdGl2ZSBkZXZlbG9wbWVudCBhbmQgc2VlZCBxdWFsaXR5IHRyYWl0cyBpbiBzb3liZWFuIHVzaW5nIHJlY29tYmluYW50IGluYnJlZCBsaW5lczsgQnJlZWQuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz0zOTktNDA3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wcmVoZW5zaXZlIHN0cnVjdHVyYWwgYW5hbHlzaXMgb2YgdGhlIGdlbm9tZSBvZiByZWQgY2xvdmVyIChUcmlmb2xpdW0gcHJhdGVuc2UgTC4pOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTMwMS0zNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1QbGFudCBSZXNlYXJjaCBJbnRlcm5hdGlvbmFsLCBXYWdlbmluZ2VuLCB0aGUgTmV0aGVybGFuZHM7IEpvaW5NYXAgNC4wIFNvZnR3YXJlIGZvciB0aGUgQ2FsY3VsYXRpb24gb2YgR2VuZXRpYyBMaW5rYWdlIE1hcHMgaW4gRXhwZXJpbWVudGFsIFBvcHVsYXRpb25zOyAgY2l0YXRpb25feWVhcj0yMDA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSIgY2l0YXRpb25fZWRpdGlvbj0zcmQgRWQuOyBjaXRhdGlvbl9wdWJsaXNoZXI9V2hpdGVoZWFkIEluc3RpdHV0ZSwgTWFzc2FjaHVzZXR0cywgQ2FtYnJpZGdlOyBDb25zdHJ1Y3RpbmcgR2VuZXRpYyBNYXBzIHdpdGggTUFQTUFLRVIvRVhQIDMuMCwgV2hpdGVoZWFkIEluc3RpdHV0ZSBUZWNobmljYWwgUmVwb3J0OyAgY2l0YXRpb25feWVhcj0xOTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXN0aW1hdGlvbiBvZiBtYXAgZGlzdGFuY2UgZnJvbSByZWNvbWJpbmF0aW9uIHZhbHVlczsgQW5uLiBFdWdlbi47ICBjaXRhdGlvbl95ZWFyPTE5NDQ7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xNzItMTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBDaGFydDogc29mdHdhcmUgZm9yIHRoZSBncmFwaGljYWwgcHJlc2VudGF0aW9uIG9mIGxpbmthZ2UgbWFwcyBhbmQgUVRMczsgSi4gSGVyZWQuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTkzOyAgY2l0YXRpb25fcGFnZXM9NzctNzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2YWx1YXRpb24gb2Ygc295YmVhbiBSRkxQIG1hcmtlciBkaXZlcnNpdHkgaW4gYWRhcHRlZCBnZXJtIHBsYXNtOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTg1OyAgY2l0YXRpb25fcGFnZXM9MjA1LTIxMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbiBhbmQgaWRlbnRpZmljYXRpb24gb2YgKENUKW4gbWljcm9zYXRlbGxpdGVzIGluIHNveWJlYW4gdXNpbmcgc2hlYXJlZCBnZW5vbWljIGxpYnJhcmllczsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTEwMy0xMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVR3byBzaW1wbGUgc2VxdWVuY2UgcmVwZWF0IG1hcmtlcnMgdG8gc2VsZWN0IGZvciBzb3liZWFuIGN5c3QgbmVtYXRvZGUgcmVzaXN0YW5jZSBjb25kaXRpb25lZCBieSByaGcxIGxvY3VzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTk5OyAgY2l0YXRpb25fcGFnZXM9ODExLTgxODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iU295QmFzZSBTb3liZWFuIGNocm9tb3NvbWUgbnVtYmVyIHRvIGxpbmthZ2UgZ3JvdXAgYXNzaWdubWVudHMgMjAwOCBodHRwOi8vc295YmFzZS5vcmcvTEcyWHNvbWUucGhwICgzMCBKdWx5IDIwMDgpICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZhcmlhYmlsaXR5IGZvciByZWNvbWJpbmF0aW9uIGZyZXF1ZW5jaWVzIGluIHRoZSBBUDEyIHNveWJlYW4gcG9wdWxhdGlvbjsgQ3JvcCBTY2kuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9NTQ1LTU0OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgYW5kIGFubm90YXRpb24gb2YgcGVyZW5uaWFsIFRyaXRpY2VhZSBFU1RzIGFuZCBTU1IgbWFya2VyczsgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTUxOyAgY2l0YXRpb25fcGFnZXM9Nzc5LTc4ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXBwbGljYXRpb24gb2YgY29tcGFyYXRpdmUgZ2Vub21pY3MgaW4gZGV2ZWxvcGluZyBtb2xlY3VsYXIgbWFya2VycyB0aWdodGx5IGxpbmtlZCB0byB0aGUgdmlydXMgcmVzaXN0YW5jZSBnZW5lIFJzdjQgaW4gc295YmVhbjsgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTQ5OyAgY2l0YXRpb25fcGFnZXM9MzgwLTM4ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXN0aW1hdGluZyBnZW5vbWUgY29uc2VydmF0aW9uIGJldHdlZW4gY3JvcCBhbmQgbW9kZWwgbGVndW1lIHNwZWNpZXM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTAxOyAgY2l0YXRpb25fcGFnZXM9MTUyODktMTUyOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IlRoZSBEZXBhcnRtZW50IG9mIEVuZXJneSdzIEpvaW50IEdlbm9tZSBJbnN0aXR1dGUgYW5kIHRoZSBDZW50ZXIgZm9yIEludGVncmF0aXZlIEdlbm9taWNzIFBoeXRvem9tZSBHbHltYSAxIDIwMDggaHR0cDovL3d3dy5waHl0b3pvbWUubmV0L3NveWJlYW4ucGhwICg4IERlY2VtYmVyIDIwMDgpICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xNi80LzIxMy8xMTA0NTMzL2RzcDAxMC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgQSB3ZWxsLXNhdHVyYXRlZCBtb2xlY3VsYXIgbGlua2FnZSBtYXAgaXMgYSBwcmVyZXF1aXNpdGUgZm9yIG1vZGVybiBwbGFudCBicmVlZGluZy4gU2V2ZXJhbCBnZW5ldGljIG1hcHMgaGF2ZSBiZWVuIGRldmVsb3BlZCBmb3Igc295YmVhbiB3aXRoIHZhcmlvdXMiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM5MTcwMiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQnN1tLDf8zurSb0nep3ifWHY52SA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f4%2f213%2f391702" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f4%2f213%2f391702" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=15321" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/16/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/4/1/m_cover.gif?Expires=2147483647&amp;Signature=KixCr2M0LlinzZEFB8nkzQbIGvWUPfCHMMnb1HEM4rLsLPtPw1WxHRM~3wSTgHca1h7ODdDw6cqpZwDcBtDB5cvMmAMOurPOoQS0BIzs30jp8r~JAQtWE3og9uxAEzWt7tYCTXVUp3R96yLrM8jbua3o2HYemMJpVAvuNM2mdcIDjO8dDR4VLEhZgMz9vD4Mgo~7X40rIJRdU89cuFZ-es1advHNbP4DSmboI0fmFNs-0MVR206dhxd0q7sk6et-~kJk~GXjkkLEy8a0H2cSmsP3xSxoZDkKVqEWVw2rjyMN144G~brfO6mG0-KjFvnH6w2O1vIyutTwcUiTgCKyRA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/16/4">
                        <div class="volume trailing-comma">Volume 16</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2009        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5776785">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5776785">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5776787">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5776787">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5776792">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5776792">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5776805">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5776805">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5776834">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5776834">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5776836">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5776836">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5776838">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5776838">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5776840">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5776840">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/16/4/195/391999">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/16/4/227/392840">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">High-density Integrated Linkage Map Based on SSR Markers in Soybean
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tae-Young Hwang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">National Agricultural Research Center for Hokkaido Region</div>, <div class="addr-line">1 Hitsujigaoka, Toyohira, Sapporo, Hokkaido 062-8555</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Graduate School of Agriculture</div>, <div class="insititution">Hokkaido University</div>, <div class="addr-line">Kita 9 Nishi 9, Kita, Sapporo, Hokkaido 060-8589</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tae-Young+Hwang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hwang T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hwang T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Tae-Young Hwang</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takashi Sayama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">National Agricultural Research Center for Hokkaido Region</div>, <div class="addr-line">1 Hitsujigaoka, Toyohira, Sapporo, Hokkaido 062-8555</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takashi+Sayama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sayama T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sayama T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Takashi Sayama</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masakazu Takahashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">National Agricultural Research Center for Kyushu Okinawa Region</div>, <div class="addr-line">2421 Suya, Koshi, Kumamoto 861-1192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masakazu+Takahashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takahashi M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takahashi M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Masakazu Takahashi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshitake Takada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">National Agricultural Research Center for Tohoku Region</div>, <div class="addr-line">297 Uenodai, Kariwano, Daisen, Akita 019-2112</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshitake+Takada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takada Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takada Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yoshitake Takada</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yumi Nakamoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">National Agricultural Research Center for Hokkaido Region</div>, <div class="addr-line">1 Hitsujigaoka, Toyohira, Sapporo, Hokkaido 062-8555</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yumi+Nakamoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakamoto Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakamoto Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yumi Nakamoto</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideyuki Funatsuki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">National Agricultural Research Center for Hokkaido Region</div>, <div class="addr-line">1 Hitsujigaoka, Toyohira, Sapporo, Hokkaido 062-8555</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideyuki+Funatsuki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Funatsuki H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Funatsuki H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hideyuki Funatsuki</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroshi Hisano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroshi+Hisano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hisano H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hisano H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hiroshi Hisano</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shigemi Sasamoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shigemi+Sasamoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sasamoto S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sasamoto S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shigemi Sasamoto</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shusei Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shusei+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shusei Sato</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Tabata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Tabata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tabata S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tabata S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Satoshi Tabata</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Izumi Kono</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="insititution">Institute of Society for Techno-Innovation of Agriculture, Forestry, and Fisheries</div>, <div class="addr-line">446-1 Kamiyokoba, Tsukuba, Ibaraki 305-0854</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Izumi+Kono">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kono I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kono I&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Izumi Kono</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masako Hoshi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="addr-line">Faculty of Horticulture</div>, <div class="insititution">Chiba University</div>, <div class="addr-line">648 Matsudo, Chiba 271-8510</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masako+Hoshi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hoshi M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hoshi M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Masako Hoshi</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masayoshi Hanawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="addr-line">Faculty of Horticulture</div>, <div class="insititution">Chiba University</div>, <div class="addr-line">648 Matsudo, Chiba 271-8510</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masayoshi+Hanawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hanawa M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hanawa M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Masayoshi Hanawa</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chizuru Yano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="addr-line">Faculty of Horticulture</div>, <div class="insititution">Chiba University</div>, <div class="addr-line">648 Matsudo, Chiba 271-8510</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chizuru+Yano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yano C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yano C&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Chizuru Yano</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zhengjun Xia</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">8</div><div class="insititution">National Institute of Agrobiological Sciences</div>, <div class="addr-line">2-1-2 Kannondai, Tsukuba, Ibaraki 305-8602</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zhengjun+Xia">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xia Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Xia Z&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Zhengjun Xia</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kyuya Harada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">8</div><div class="insititution">National Institute of Agrobiological Sciences</div>, <div class="addr-line">2-1-2 Kannondai, Tsukuba, Ibaraki 305-8602</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kyuya+Harada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Harada K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Harada K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Kyuya Harada</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Keisuke Kitamura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Graduate School of Agriculture</div>, <div class="insititution">Hokkaido University</div>, <div class="addr-line">Kita 9 Nishi 9, Kita, Sapporo, Hokkaido 060-8589</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Keisuke+Kitamura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kitamura K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kitamura K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Keisuke Kitamura</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masao Ishimoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">National Agricultural Research Center for Hokkaido Region</div>, <div class="addr-line">1 Hitsujigaoka, Toyohira, Sapporo, Hokkaido 062-8555</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81-11-857-9312. Fax. <div class="fax">+81-11-859-2178</div>. E-mail: <a href="mailto:ishimoto@affrc.go.jp" target="_blank">ishimoto@affrc.go.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masao+Ishimoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ishimoto M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ishimoto M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Masao Ishimoto</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="FN1"><div class="fn"><div class="label -label"></div><div class="footnote-content">Present address: National Agricultural Research Center for Western Region, 1-3-1 Senyuu, Zentsuuji, Kagawa 765-8508, Japan</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 16, Issue 4, 1 August 2009, Pages 213225, <a href='https://doi.org/10.1093/dnares/dsp010'>https://doi.org/10.1093/dnares/dsp010</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">16 June 2009</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">02 April 2009</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">25 May 2009</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="391702" href="/dnaresearch/article-pdf/16/4/213/1104533/dsp010.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Tae-Young Hwang, Takashi Sayama, Masakazu Takahashi, Yoshitake Takada, Yumi Nakamoto, Hideyuki Funatsuki, Hiroshi Hisano, Shigemi Sasamoto, Shusei Sato, Satoshi Tabata, Izumi Kono, Masako Hoshi, Masayoshi Hanawa, Chizuru Yano, Zhengjun Xia, Kyuya Harada, Keisuke Kitamura, Masao Ishimoto;  High-density Integrated Linkage Map Based on SSR Markers in Soybean, <em>DNA Research</em>, Volume 16, Issue 4, 1 August 2009, Pages 213225, <a href='https://doi.org/10.1093/dnares/dsp010'>https://doi.org/10.1093/dnares/dsp010</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/391702?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/391702?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/391702?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/391702?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/391702?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/391702?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/391702?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=High-density%20Integrated%20Linkage%20Map%20Based%20on%20SSR%20Markers%20in%20Soybean&amp;publicationDate=2009-06-16&amp;volumeNum=16&amp;issueNum=4&amp;author=Hwang%2C%20Tae-Young%3B%20Sayama%2C%20Takashi&amp;startPage=213&amp;endPage=225&amp;contentId=10.1093%2Fdnares%2Fdsp010&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f4%2f213%2f391702" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5776785 id="5776785" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>A well-saturated molecular linkage map is a prerequisite for modern plant breeding. Several genetic maps have been developed for soybean with various types of molecular markers. Simple sequence repeats (SSRs) are single-locus markers with high allelic variation and are widely applicable to different genotypes. We have now mapped 1810 SSR or sequence-tagged site markers in one or more of three recombinant inbred populations of soybean (the US cultivar Jack  the Japanese cultivar Fukuyutaka, the Chinese cultivar Peking  the Japanese cultivar Akita, and the Japanese cultivar Misuzudaizu  the Chinese breeding line Moshidou Gong 503) and have aligned these markers with the 20 consensus linkage groups (LGs). The total length of the integrated linkage map was 2442.9 cM, and the average number of molecular markers was 90.5 (range of 70114) for the 20 LGs. We examined allelic diversity for 1238 of the SSR markers among 23 soybean cultivars or lines and a wild accession. The number of alleles per locus ranged from 2 to 7, with an average of 2.8. Our high-density linkage map should facilitate ongoing and future genomic research such as analysis of quantitative trait loci and positional cloning in addition to marker-assisted selection in soybean breeding.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;EST-derived SSR marker&quot;">EST-derived SSR marker</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;integrated linkage map&quot;">integrated linkage map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;microsatellite marker&quot;">microsatellite marker</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;polymorphism information content&quot;">polymorphism information content</a></div>                            <h2 scrollto-destination=5776787 id="5776787" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Soybean [<em>Glycine max</em> (L.) Merrill] is one of the most important grain legumes because it is a staple source of high-quality vegetable protein and oil for food products and industrial material. Over the past 20 years, the global growing area of soybean has increased by a factor of 1.7, whereas the average yield has increased by a factor of 2.3, reaching 2.5 tons/hectare in 2006.<sup><a reveal-id="DSP010C1" data-open="DSP010C1" class="link link-ref link-reveal xref-bibr">1</a></sup> However, the rate of growth in soybean yield has declined during the last 10 years, increasing by only 8% in this period. It is therefore imperative to develop new technologies and resources that will allow the supply of soybean to meet the large growth in demand anticipated in the near future.</p><p>An accurate and well-saturated genetic linkage map is fundamental to modern plant breeding because it allows both the identification of agronomic trait loci, including quantitative trait loci, and an understanding of genetic diversity and genome structure of genetic resources. Furthermore, such a linkage map is required for construction of a physical map. Since the first genetic map of soybean was constructed with phenotypic traits,<sup><a reveal-id="DSP010C2" data-open="DSP010C2" class="link link-ref link-reveal xref-bibr">2</a></sup> several linkage maps have been developed either alone or in combination with various types of molecular markers such as restriction fragment length polymorphism (RFLP), random amplification of polymorphic DNA (RAPD), amplified fragment length polymorphism (AFLP), simple sequence repeat (SSR) and single nucleotide polymorphism (SNP) markers.<sup><a reveal-id="DSP010C3  DSP010C7" data-open="DSP010C3  DSP010C7" class="link link-ref link-reveal xref-bibr">37</a></sup> Among these molecular markers, SSRs (also known as microsatellite markers) possess several favorable features: (i) they are inherited in a codominant manner at individual loci; (ii) they exhibit high levels of polymorphism and exist in multiple alleles; (iii) they are distributed evenly and randomly in the genome and (iv) they can be readily analyzed by the polymerase chain reaction (PCR) and subsequent gel electrophoresis.<sup><a reveal-id="DSP010C5" data-open="DSP010C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSP010C6" data-open="DSP010C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSP010C8" data-open="DSP010C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSP010C9" data-open="DSP010C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Multiallelic molecular markers with a high level of polymorphism are useful for the detection of allelic differences among many genetic resources.<sup><a reveal-id="DSP010C10  DSP010C12" data-open="DSP010C10  DSP010C12" class="link link-ref link-reveal xref-bibr">1012</a></sup> SSR markers are applicable to analysis of many of the segregating populations derived from the hybridization of any given genotypes. In addition, most PCR primer pairs for SSR markers yield a single amplification product for each soybean genotype,<sup><a reveal-id="DSP010C13" data-open="DSP010C13" class="link link-ref link-reveal xref-bibr">13</a></sup> allowing the use of a variety of means for the detection of SSR length polymorphisms. The amplification products are usually analyzed by electrophoresis on a polyacrylamide or agarose gel, but they can also be analyzed with a genetic sequencer in combination with a labeling protocol.<sup><a reveal-id="DSP010C6" data-open="DSP010C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSP010C10" data-open="DSP010C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSP010C14" data-open="DSP010C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Moreover, SSR markers can be genotyped more rapidly and cost efficiently by simultaneous detection of multiple loci with the use of multiplex PCR analysis.<sup><a reveal-id="DSP010C15" data-open="DSP010C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Such analysis is ideal for high-throughput and repetitive genotyping applications for which common sets of SSR marker loci are advantageous.</p><p>Given that SSR markers were found to map to individual loci with the same order in three different mapping populations, 606 such loci were integrated and aligned with RFLP, RAPD, AFLP and classical markers into a consensus set of 20 linkage groups (LGs) corresponding to the 20 pairs of soybean chromosomes.<sup><a reveal-id="DSP010C16" data-open="DSP010C16" class="link link-ref link-reveal xref-bibr">16</a></sup> The integrated linkage map was subsequently updated with 420 newly developed SSR markers and two additional mapping populations, giving a total of 1849 molecular markers including 1015 SSR, 709 RFLP, 73 RAPD, 6 AFLP and 46 classical markers.<sup><a reveal-id="DSP010C17" data-open="DSP010C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSP010C18" data-open="DSP010C18" class="link link-ref link-reveal xref-bibr">18</a></sup> The framework of 20 consensus LGs with an average of 50 SSR loci per group has provided a resource for construction of a linkage map for genetic analysis of qualitative and quantitative traits. However, there remain several genomic intervals of at least 20 cM that contain no SSR markers, and the number of SSR markers is not sufficient for application to all hybridizing combinations of germplasm, given that the genome size of soybean is 1.1 Gb.<sup><a reveal-id="DSP010C19" data-open="DSP010C19" class="link link-ref link-reveal xref-bibr">19</a></sup></p><p>Xia et al.<sup><a reveal-id="DSP010C20" data-open="DSP010C20" class="link link-ref link-reveal xref-bibr">20</a></sup> recently developed 702 new SSRs from analysis of genomic DNA or by surveying expressed sequence tag (EST)-derived SSRs in the public database, and 121 of these SSR markers were mapped in an F<sub>2</sub> population derived from a cross between the Japanese cultivar Misuzudaizu and the Chinese experimental line Moshidou Gong 503. Additionally, 6920 primer pairs were designed to amplify SSRs from 63 676 non-redundant soybean ESTs available in the public database, and 668 EST-derived SSR markers were assigned to a molecular linkage map constructed with F<sub>8</sub> recombinant inbred lines (RILs) of Misuzudaizu  Moshidou Gong 503.<sup><a reveal-id="DSP010C21" data-open="DSP010C21" class="link link-ref link-reveal xref-bibr">21</a></sup> These new SSR markers enrich and saturate the soybean linkage map and provide a framework for analysis of the entire genome. We have now analyzed polymorphism of the new SSRs between parents of three mapping populations including the Misuzudaizu  Moshidou Gong 503 RIL population, generated three molecular linkage maps in combination with the public SSR markers<sup><a reveal-id="DSP010C18" data-open="DSP010C18" class="link link-ref link-reveal xref-bibr">18</a></sup> and integrated the new SSR markers into the framework of 20 consensus LGs. In addition, we investigated allelic polymorphism for the mapped EST-derived SSR markers in comparison with the public SSR markers, and we consider the transferability of the new SSR markers in soybean germplasm.</p>                            <h2 scrollto-destination=5776792 id="5776792" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5776793 id="5776793" class="section-title" data-legacy-id=s2a>2.1. Mapping populations</h3>
<p>To develop an integrated linkage map, we used the following three mapping populations. (i) The MM population. This population originates from a cross between the Japanese cultivar Misuzudaizu and the Chinese weedy experimental line Moshidou Gong 503, consists of 165 RILs and has been described previously.<sup><a reveal-id="DSP010C22" data-open="DSP010C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSP010C23" data-open="DSP010C23" class="link link-ref link-reveal xref-bibr">23</a></sup> The RILs were available from the National BioResource Project of <em>Lotus</em> and <em>Glycine</em> (<a class="link link-uri" href="http://www.legumebase.agr.miyazaki-u.ac.jp/index.jsp">http://www.legumebase.agr.miyazaki-u.ac.jp/index.jsp</a>). We studied 94 individual plants representing the F<sub>8</sub> generation of the RILs to construct a linkage map.<sup><a reveal-id="DSP010C21" data-open="DSP010C21" class="link link-ref link-reveal xref-bibr">21</a></sup> (ii) The JF population. This population is derived from a cross between the US cultivar Jack and Fukuyutaka, a leading Japanese cultivar, and comprises 179 RILs. We randomly selected 91 individual plants representing the F<sub>7</sub> generation of the RILs and used for mapping. (iii) The PA population. This population is created from a cross between the Chinese cultivar Peking and the Japanese cultivar Akita. We randomly selected 68 individual plants representing the F<sub>9</sub> generation of 120 RILs.</p>                            <h3 scrollto-destination=5776795 id="5776795" class="section-title" data-legacy-id=s2b>2.2. DNA extraction and marker analysis</h3>
<p>Total genomic DNA was extracted from young leaves or seed flour of individual samples with the use of an Automatic DNA Isolation System PI-50a (Kurabo, Osaka, Japan) according to Plant DNA Extraction Protocol (version 2). The concentration of DNA in solution was quantified by comparison with a serial dilution of lambda phage DNA (Promega, Madison, WI, USA) and was then adjusted to 10 ng/L with sterile water. PCR amplification was performed in a reaction mixture (10 L) containing 10 ng of total genomic DNA, 0.4 M of each primer and 5 L of GoTaq Green Master Mix (Promega) and with the use of a GeneAmp PCR System 9700 (Applied Biosystems, Foster City, CA, USA). The amplification protocol comprised an initial denaturation for 2 min at 95C; 30 cycles of denaturation for 30 s at 95C, annealing for 30 s at 47C<sup><a reveal-id="DSP010C13" data-open="DSP010C13" class="link link-ref link-reveal xref-bibr">13</a></sup> or 55C,<sup><a reveal-id="DSP010C20" data-open="DSP010C20" class="link link-ref link-reveal xref-bibr">20</a></sup> and extension for 30 s at 72C; and a final extension for 5 min at 72C. The EST-derived SSR markers were amplified with the modified Touchdown PCR protocol.<sup><a reveal-id="DSP010C24" data-open="DSP010C24" class="link link-ref link-reveal xref-bibr">24</a></sup> PCR products were resolved by 10% polyacrylamide gel electrophoresis in Tris-borate-EDTA (TBE) buffer. The gel was stained with ethidium bromide, and the bands were visualized and photographed under ultraviolet light.</p>                            <h3 scrollto-destination=5776797 id="5776797" class="section-title" data-legacy-id=s2c>2.3. Screening and genotyping of DNA markers</h3>
<p>SSR markers developed by USDA-ARS (United States Department of Agriculture, Agricultural Research Service) and DuPont Corporation<sup><a reveal-id="DSP010C16" data-open="DSP010C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSP010C18" data-open="DSP010C18" class="link link-ref link-reveal xref-bibr">18</a></sup> as well as SSR and sequence-tagged site (STS) markers developed by the Chiba University<sup><a reveal-id="DSP010C20" data-open="DSP010C20" class="link link-ref link-reveal xref-bibr">20</a></sup> were screened to detect polymorphism between parents of the JF and PA populations in addition to the MM population. The polymorphic markers were mapped with each of the three mapping populations. A total of 6920 EST-derived SSR primer pairs developed by the Kazusa DNA Research Institute<sup><a reveal-id="DSP010C21" data-open="DSP010C21" class="link link-ref link-reveal xref-bibr">21</a></sup> were used to detect polymorphisms between parents of the two mapping populations (JF and PA), and the polymorphic markers were analyzed in the two populations. The genotype data of 693 EST-derived SSR markers in the MM population were used from the previous study by Hisano et al.<sup><a reveal-id="DSP010C21" data-open="DSP010C21" class="link link-ref link-reveal xref-bibr">21</a></sup></p>                            <h3 scrollto-destination=5776799 id="5776799" class="section-title" data-legacy-id=s2d>2.4. Construction of an integrated linkage map</h3>
<p>We first used JoinMap (version 4.0)<sup><a reveal-id="DSP010C25" data-open="DSP010C25" class="link link-ref link-reveal xref-bibr">25</a></sup> to group and order molecular marker loci within each of the JF, PA and MM populations. The segregated markers were grouped in LGs on the basis of an LOD (logarithm of the odds ratio for linkage) score of 7.0 and referral to previously reported LGs of the public SSR marker loci.<sup><a reveal-id="DSP010C16" data-open="DSP010C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSP010C21" data-open="DSP010C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Marker order was subsequently determined on the basis of a minimum LOD score of 1.0 and recombination threshold of 0.4 in each LG. Markers were tested for deviation from expected Mendelian segregation by the chi-squared test performed with the JoinMap software under the Locus Genotypic Frequency command. Markers were sorted on the basis of the chi-squares test with a <em>P</em>-value of &lt;0.05.</p><p>The three maps of each LG were integrated with the used of a minimum LOD score of 3.0 and recombination threshold of 0.4 under the Combine Groups for Map Integration command of the JoinMap software, and they were reorganized with reference to the 20 consensus linkage map<sup><a reveal-id="DSP010C16" data-open="DSP010C16" class="link link-ref link-reveal xref-bibr">16</a></sup> and a previously reported map of EST-derived SSR markers.<sup><a reveal-id="DSP010C21" data-open="DSP010C21" class="link link-ref link-reveal xref-bibr">21</a></sup> The large interval between the two markers Sat_235 and GMES1325 in LG C1 was calculated with the use of MAPMAKER/EXP, version 3.0.<sup><a reveal-id="DSP010C26" data-open="DSP010C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Recombination values were converted to genetic distance (cM) with the use of the Kosambi mapping function.<sup><a reveal-id="DSP010C27" data-open="DSP010C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Each map of the 20 LGs was graphically visualized with MapChart.<sup><a reveal-id="DSP010C28" data-open="DSP010C28" class="link link-ref link-reveal xref-bibr">28</a></sup></p>                            <h3 scrollto-destination=5776802 id="5776802" class="section-title" data-legacy-id=s2e>2.5. Evaluation of mapped markers and allele frequencies</h3>
<p>Polymorphism analysis was performed with the same 24 soybean cultivars and lines used in a previous study,<sup><a reveal-id="DSP010C21" data-open="DSP010C21" class="link link-ref link-reveal xref-bibr">21</a></sup> namely 15 Japanese cultivars and lines (Enrei, Fukuyutaka, Hayahikari, Himeshirazu, Ibarakimame 7 gou, Koitozairai, Misuzudaizu, Nourin 2 gou, Saikai No. 20, Suzuyutaka, Tamahomare, Toyokomachi, Toyomusume, Yukihomare and Toiku No. 237), two Japanese breeding material lines (Tokei 758 gou and To-8E), three US cultivars (Adams, Harosoy and Jack), two Chinese lines (Peking and Moshidou Gong 503), an Indonesian cultivar (Wilis) and the Japanese wild accession (Funaba 3). The number of alleles and polymorphism information content (PIC) were analyzed for 304 USDA-ARS SSR markers on a 3% agarose gel<sup><a reveal-id="DSP010C12" data-open="DSP010C12" class="link link-ref link-reveal xref-bibr">12</a></sup> using Tris-acetate-EDTA buffer and for 934 EST-derived SSR markers mapped in this study on a 10% polyacrylamide gel using TBE buffer. The PIC for each SSR marker locus (<em>i</em>) was calculated according to the formula described by Keim et al.:<sup><a reveal-id="DSP010C29" data-open="DSP010C29" class="link link-ref link-reveal xref-bibr">29</a></sup> PIC(<em>i</em>)=1  <em>P</em><sub><em>ij</em></sub>2, where <em>P</em><sub><em>ij</em></sub> is the frequency of the <em>j</em>th allele of the <em>i</em>th SSR locus and summation extends over <em>n</em> alleles.</p>                            <h2 scrollto-destination=5776805 id="5776805" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5776807 id="5776807" class="section-title" data-legacy-id=s3a>3.1. Screening of polymorphic molecular markers</h3>
<p>We performed PCR amplification of template DNA from the parents of two mapping populations, the JF and PA populations, in addition to the MM population with EST-derived SSR primer pairs.<sup><a reveal-id="DSP010C21" data-open="DSP010C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Amplification of template DNA from the parents Jack and Fukuyutaka yielded products from both parents with 6079 primer pairs and from only one parent with 31 pairs. Obvious polymorphism between the parents was observed with 360 (5.2%) of the primer pairs (Table<a reveal-id="DSP010TB1" data-open="DSP010TB1" class="link link-reveal link-table xref-fig">1</a>). Similarly, PCR products were obtained from both parents Peking and Akita with 5517 primer pairs and from only one parent with 82 pairs. Unambiguous polymorphism between the parents was observed with 406 (5.9%) of the EST-derived SSR primer pairs. The polymorphism frequencies obtained for the parents of the JF and PA populations (5.2 and 5.9%, respectively) are approximately half of that previously reported (9.8%) for the parents (Misuzudaizu and Moshidou Gong 503) of the MM population (680 polymorphisms among the 6920 used for analysis).<sup><a reveal-id="DSP010C21" data-open="DSP010C21" class="link link-ref link-reveal xref-bibr">21</a></sup> A total of 1127 (16.3%) EST-derived SSR primer pairs gave polymorphisms in at least one of each parental pair of the three mapping populations.</p>                        <a id="5776810" scrollto-destination="5776810"></a>
<div content-id="DSP010TB1" class="table-modal"><div class="table-wrap"><div id="DSP010TB1" class="table-wrap-title" data-id="DSP010TB1"><div class="label -label">Table1</div><div class="caption"><p>Numbers of SSR or STS markers analyzed, found to be polymorphic and mapped in each population of RILs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Status</th><th colspan="12">RIL population<hr /></th></tr><tr><th></th><th colspan="4">Jack  Fukuyutaka<hr /></th><th colspan="4">Peking  Akita<hr /></th><th colspan="4">Misuzudaizu  Moshidou Gong 503<hr /></th></tr><tr><th></th><th>ES<sup>a</sup></th><th>CS<sup>b</sup></th><th>US<sup>c</sup></th><th>Total</th><th>ES</th><th>CS</th><th>US</th><th>Total</th><th>ES<sup>d</sup></th><th>CS</th><th>US</th><th>Total</th></tr></thead><tbody><tr><td>Primer pairs</td><td>6920</td><td>370</td><td>1015</td><td>8305</td><td>6920</td><td>370</td><td>1015</td><td>8305</td><td>6920</td><td>370</td><td>1015</td><td>8305</td></tr><tr><td>Polymorphic</td><td>360</td><td>100</td><td>402</td><td>862</td><td>406</td><td>103</td><td>320</td><td>829</td><td>680</td><td>NA<sup>e</sup></td><td>NA<sup>e</sup></td><td>NA<sup>e</sup></td></tr><tr><td>Mapped</td><td>344</td><td>97</td><td>398</td><td>839</td><td>396</td><td>99</td><td>315</td><td>810</td><td>693</td><td>9</td><td>299</td><td>1006</td></tr></tbody></table></div><div content-id="DSP010TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Status</th><th colspan="12">RIL population<hr /></th></tr><tr><th></th><th colspan="4">Jack  Fukuyutaka<hr /></th><th colspan="4">Peking  Akita<hr /></th><th colspan="4">Misuzudaizu  Moshidou Gong 503<hr /></th></tr><tr><th></th><th>ES<sup>a</sup></th><th>CS<sup>b</sup></th><th>US<sup>c</sup></th><th>Total</th><th>ES</th><th>CS</th><th>US</th><th>Total</th><th>ES<sup>d</sup></th><th>CS</th><th>US</th><th>Total</th></tr></thead><tbody><tr><td>Primer pairs</td><td>6920</td><td>370</td><td>1015</td><td>8305</td><td>6920</td><td>370</td><td>1015</td><td>8305</td><td>6920</td><td>370</td><td>1015</td><td>8305</td></tr><tr><td>Polymorphic</td><td>360</td><td>100</td><td>402</td><td>862</td><td>406</td><td>103</td><td>320</td><td>829</td><td>680</td><td>NA<sup>e</sup></td><td>NA<sup>e</sup></td><td>NA<sup>e</sup></td></tr><tr><td>Mapped</td><td>344</td><td>97</td><td>398</td><td>839</td><td>396</td><td>99</td><td>315</td><td>810</td><td>693</td><td>9</td><td>299</td><td>1006</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>EST-derived SSR markers developed by Kazusa DNA Research Institute.<sup><a reveal-id="DSP010C21" data-open="DSP010C21" class="link link-ref link-reveal xref-bibr">21</a></sup></p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>SSR or STS markers developed by Chiba University.<sup><a reveal-id="DSP010C20" data-open="DSP010C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSP010C22" data-open="DSP010C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Public SSR markers developed by USDA-ARS and DuPont Corporation.<sup><a reveal-id="DSP010C16" data-open="DSP010C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSP010C18" data-open="DSP010C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>Genotype data were used from the previous study by Hisano et al.<sup><a reveal-id="DSP010C21" data-open="DSP010C21" class="link link-ref link-reveal xref-bibr">21</a></sup></p></div></div><div content-id=""><div class="fn"><p><sup>e</sup>Not applicable (NA); not all primer pairs analyzed.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5776810" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP010TB1" class="table-wrap-title" data-id="DSP010TB1"><div class="label -label">Table1</div><div class="caption"><p>Numbers of SSR or STS markers analyzed, found to be polymorphic and mapped in each population of RILs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Status</th><th colspan="12">RIL population<hr /></th></tr><tr><th></th><th colspan="4">Jack  Fukuyutaka<hr /></th><th colspan="4">Peking  Akita<hr /></th><th colspan="4">Misuzudaizu  Moshidou Gong 503<hr /></th></tr><tr><th></th><th>ES<sup>a</sup></th><th>CS<sup>b</sup></th><th>US<sup>c</sup></th><th>Total</th><th>ES</th><th>CS</th><th>US</th><th>Total</th><th>ES<sup>d</sup></th><th>CS</th><th>US</th><th>Total</th></tr></thead><tbody><tr><td>Primer pairs</td><td>6920</td><td>370</td><td>1015</td><td>8305</td><td>6920</td><td>370</td><td>1015</td><td>8305</td><td>6920</td><td>370</td><td>1015</td><td>8305</td></tr><tr><td>Polymorphic</td><td>360</td><td>100</td><td>402</td><td>862</td><td>406</td><td>103</td><td>320</td><td>829</td><td>680</td><td>NA<sup>e</sup></td><td>NA<sup>e</sup></td><td>NA<sup>e</sup></td></tr><tr><td>Mapped</td><td>344</td><td>97</td><td>398</td><td>839</td><td>396</td><td>99</td><td>315</td><td>810</td><td>693</td><td>9</td><td>299</td><td>1006</td></tr></tbody></table></div><div content-id="DSP010TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Status</th><th colspan="12">RIL population<hr /></th></tr><tr><th></th><th colspan="4">Jack  Fukuyutaka<hr /></th><th colspan="4">Peking  Akita<hr /></th><th colspan="4">Misuzudaizu  Moshidou Gong 503<hr /></th></tr><tr><th></th><th>ES<sup>a</sup></th><th>CS<sup>b</sup></th><th>US<sup>c</sup></th><th>Total</th><th>ES</th><th>CS</th><th>US</th><th>Total</th><th>ES<sup>d</sup></th><th>CS</th><th>US</th><th>Total</th></tr></thead><tbody><tr><td>Primer pairs</td><td>6920</td><td>370</td><td>1015</td><td>8305</td><td>6920</td><td>370</td><td>1015</td><td>8305</td><td>6920</td><td>370</td><td>1015</td><td>8305</td></tr><tr><td>Polymorphic</td><td>360</td><td>100</td><td>402</td><td>862</td><td>406</td><td>103</td><td>320</td><td>829</td><td>680</td><td>NA<sup>e</sup></td><td>NA<sup>e</sup></td><td>NA<sup>e</sup></td></tr><tr><td>Mapped</td><td>344</td><td>97</td><td>398</td><td>839</td><td>396</td><td>99</td><td>315</td><td>810</td><td>693</td><td>9</td><td>299</td><td>1006</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>EST-derived SSR markers developed by Kazusa DNA Research Institute.<sup><a reveal-id="DSP010C21" data-open="DSP010C21" class="link link-ref link-reveal xref-bibr">21</a></sup></p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>SSR or STS markers developed by Chiba University.<sup><a reveal-id="DSP010C20" data-open="DSP010C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSP010C22" data-open="DSP010C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Public SSR markers developed by USDA-ARS and DuPont Corporation.<sup><a reveal-id="DSP010C16" data-open="DSP010C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSP010C18" data-open="DSP010C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>Genotype data were used from the previous study by Hisano et al.<sup><a reveal-id="DSP010C21" data-open="DSP010C21" class="link link-ref link-reveal xref-bibr">21</a></sup></p></div></div><div content-id=""><div class="fn"><p><sup>e</sup>Not applicable (NA); not all primer pairs analyzed.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5776810" target="_blank">View Large</a></div></div><p>Of the more than 1000 SSR or STS primer pairs developed by the Chiba University,<sup><a reveal-id="DSP010C20" data-open="DSP010C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSP010C22" data-open="DSP010C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSP010C30" data-open="DSP010C30" class="link link-ref link-reveal xref-bibr">30</a></sup> we selected 370 pairs on the basis of their mapping information and banding patterns in order to detect polymorphism between each pair of the parents of the JF and PA populations. Obvious polymorphism between the parents of the JF or PA population was observed with 100 (27.0%) and 103 (27.8%) of the primer pairs, respectively (Table<a reveal-id="DSP010TB1" data-open="DSP010TB1" class="link link-reveal link-table xref-fig">1</a>). A total of 145 (39.2%) primer pairs yielded polymorphisms for at least one of the parental pairs of the JF and PA populations.</p><p>The public soybean SSR markers developed by USDA-ARS and DuPont Corporation<sup><a reveal-id="DSP010C16" data-open="DSP010C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSP010C18" data-open="DSP010C18" class="link link-ref link-reveal xref-bibr">18</a></sup> were examined for polymorphism between each pair of the parents of the three populations in order to construct a framework for the 20 consensus LGs. Clear polymorphism between the parents of the JF or PA population was observed with 402 (39.6%) and 320 (31.5%) of the 1015 SSR primer pairs, respectively (Table<a reveal-id="DSP010TB1" data-open="DSP010TB1" class="link link-reveal link-table xref-fig">1</a>). In addition, 164 primer pairs for the 377 public SSR markers analyzed previously<sup><a reveal-id="DSP010C12" data-open="DSP010C12" class="link link-ref link-reveal xref-bibr">12</a></sup> as well as the 300 SSR markers previously reported<sup><a reveal-id="DSP010C21" data-open="DSP010C21" class="link link-ref link-reveal xref-bibr">21</a></sup> yielded unambiguous polymorphism between the parents of the MM population. Consequently, obvious polymorphism between at least one of each parental pair of the three populations was observed with a total of 614 (60.5%) SSR markers.</p>                            <h3 scrollto-destination=5776813 id="5776813" class="section-title" data-legacy-id=s3b>3.2. Construction of each linkage map</h3>
<p>A total of 862 polymorphisms were scored in the JF population, with each primer pair yielding polymorphic bands at a single locus. After exclusion of 16 unlinked markers and 7 unsuccessfully positioned markers, 839 marker loci, which included 344 EST-derived SSR markers, 97 SSR or STS markers developed by Chiba University and 398 public SSR markers, were assembled into 26 LGs with a total genetic length of 2363.6 cM. Sixteen of these LGs were matched with A1, A2, B1, B2, D1a, D1b, D2, E, F, G, I, J, K, L, M and N of the 20 consensus LGs.<sup><a reveal-id="DSP010C18" data-open="DSP010C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSP010C31" data-open="DSP010C31" class="link link-ref link-reveal xref-bibr">31</a></sup> The other four consensus LGs (C1, C2, H and O) were divided into two or three fragments.</p><p>After exclusion of 12 unlinked markers and 7 unsuccessfully positioned markers, a total of 810 marker loci, including 396 EST-derived SSR markers, 99 SSR or STS markers, and 315 public SSR markers, were grouped into 32 LGs in the PA population. Twelve of these LGs corresponded to A1, A2, B2, D1a, D1b, D2, F, I, J, M, N and O of the 20 consensus LGs. The other eight consensus LGs (B1, C1, C2, E, G, H, K and L) were divided into 24 fragments. All but one of the primer pairs yielded polymorphic bands at a single locus; the primer pair for CSSR534 detected two loci. The linkage map covered a genetic distance of 1575.9 cM.</p><p>Genotype data for 693 EST-derived SSR loci in the MM population were used from the previous study by Hisano et al.<sup><a reveal-id="DSP010C21" data-open="DSP010C21" class="link link-ref link-reveal xref-bibr">21</a></sup> In addition to these markers, 308 marker loci, including 9 SSR or STS markers and 299 public SSR markers, were assembled into 22 LGs with a total genetic length of 2187.2 cM, after the elimination of one unlinked marker. Eighteen of these LGs matched 18 of the 20 consensus LGs. The other two consensus LGs (C1 and E) were each fragmented into two short pieces. After the addition of two public SSR markers, Sat_137 and Satt684, of LG A1, five EST-derived SSR markers (GMES4205a, GMES6481, GMES2036, GMES4016 and GMES2497) mapped at one end of LG A1, though they were previously assigned to LG C2.<sup><a reveal-id="DSP010C21" data-open="DSP010C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Twenty-four and one primer pairs detected double and quadruple loci, respectively. Segregation distortion was observed in the same region of consensus LG I in two mapping populations, JF (CSSR366, CSSR529, GMES1137, GMES2783, GMES6339 and Sat_189) and MM (CSSR529, GMES0289, GMES2783, GMES4080, GMES6339 and GMES6428).</p>                            <h3 scrollto-destination=5776817 id="5776817" class="section-title" data-legacy-id=s3c>3.3. Construction of an integrated linkage map</h3>
<p>Since 648 of the mapped marker loci were common among at least two of the mapping populations, JoinMap analysis of the 1074 EST-derived SSR including 387 newly mapped marker loci, 141 SSR or STS including 81 newly mapped marker loci and 595 public SSR markers resulted in their successful merger into a genetic linkage map comprised the 20 consensus LGs and spanning 2442.9 cM of Kosambi map distance (Fig.<a reveal-id="DSP010F1" data-open="DSP010F1" class="link link-reveal link-table xref-fig">1</a>, Table<a reveal-id="DSP010TB2" data-open="DSP010TB2" class="link link-reveal link-table xref-fig">2</a>). The average genetic length of the 20 LGs was 122.1 cM, although the length ranged from 175.8 (LG A2) to 77.4 (LG J) cM (Table<a reveal-id="DSP010TB2" data-open="DSP010TB2" class="link link-reveal link-table xref-fig">2</a>). The number of marker loci for the LGs ranged from 114 (LG F) to 70 (LG N), with an average of 90.5. The average genetic distance between markers varied between 0.94 (LG J) and 1.78 (LG A2) cM. A cluster of five EST-derived SSR markers (GMES4205a, GMES6481, GMES2036, GMES4016 and GMES2497) was previously assigned to LG C2,<sup><a reveal-id="DSP010C21" data-open="DSP010C21" class="link link-ref link-reveal xref-bibr">21</a></sup> but this cluster together with GMES0706, CSSR1,<sup><a reveal-id="DSP010C20" data-open="DSP010C20" class="link link-ref link-reveal xref-bibr">20</a></sup> GMES5782, GMES3776 and GMES1678 was assembled at one end of LG A1 in the present integrated linkage map as well as individual linkage maps (Fig.<a reveal-id="DSP010F1" data-open="DSP010F1" class="link link-reveal link-table xref-fig">1</a>). Most marker loci were relatively evenly distributed along the LGs, although there were some regions with a high or low marker density (Fig.<a reveal-id="DSP010F1" data-open="DSP010F1" class="link link-reveal link-table xref-fig">1</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). The LG C1 still contained a large genetic interval (35.3 cM) between Sat_235 and GMES1325 for which there was no marker locus (Table<a reveal-id="DSP010TB3" data-open="DSP010TB3" class="link link-reveal link-table xref-fig">3</a>). There were additional six large genetic intervals with a gap of &gt;10 cM (Table<a reveal-id="DSP010TB3" data-open="DSP010TB3" class="link link-reveal link-table xref-fig">3</a>).</p>                        <a id="5776819" scrollto-destination="5776819"></a>
<div id="DSP010F1" data-id="DSP010F1" class="fig fig-section"><div class="label fig-label">Figure1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/4/10.1093/dnares/dsp010/2/m_dsp01001a.jpeg?Expires=2147483647&amp;Signature=XTSyDkxb9m5Q4vh-gdTaQt-~4bm8QCFy0oHlbS7bB0g4dbfYO04CrsA0YcCsRtbkeiCqmwvnY7SDH13vCF18ZfEtS8YWn8lonF48O0k4Or~4EDSxsCRQrrDzyyi0EdDEzpr6THkBl9DfhvoCr8HSgaTcBVGAKmgdeOcxVusoOtHTLSoc49Nraej8NDM8Qfo~3TnrJPtaCaLtZ25A1Xv3I5D7SffDfRTvpVbAfY27pqEGn6J9SltvCw7PNw5i9E~-2VsZ7ldgrR7aJoG4seBKutJUWyM6M2ABF5XpcE8w5lARHASuvRjeNoC-7cRU60MG9Lba6Bxki7FuyLzcrkweMw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic linkage map of soybean constructed with SSR and STS markers. The linkage map was visualized graphically with MapChart. The LGs were constructed from 595 public SSR markers developed by USDA-ARS and DuPont Corporation (black characters), 1074 EST-derived SSR markers developed by Kazusa DNA Research Institute (red), and 141 SSR or STS markers developed by the Chiba University (blue). A total of 468 newly mapped PCR-based markers were polymorphic in one or more of the three mapping populations (bold). The EST-derived SSR markers GMES4205a, GMES6481, GMES2036, GMES4016 and GMES2497 were previously assigned to LG C2,21 but the JoinMap analysis here positioned these markers as well as GMES0706, CSSR1, GMES5782, GMES3776 and GMES1678 at one end of LG A1 (boxed region)." path-from-xml="dsp01001a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5776819/dsp01001a.jpeg" path-from-xml="dsp01001a.jpeg" target="_blank">View large</a><a section="5776819" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/4/10.1093/dnares/dsp010/2/dsp01001a.jpeg?Expires=2147483647&amp;Signature=LCrK3Bc3jmbULEJLSLQbyD3EqRb8HgSOiCVkzNBWTBQ6fqCo7EEgvoxF~WKQhYOygzZi4o3uHsj0VszCDfRQeB6cUG0A~JaEhPZXhn1tK~lJsMLJJqAKGF~CGcM6vQT~CXYWSnt2cDRZvUC~f9S62jGF5L~rPkYXk60U7xoW3IdTdRRe563ZlOdhnCiuvZDKgQJVvqPCXgrRiTU5JaLrnIiD9YJ2j~99bsogY8ke1jIJ4k2kU4D8wJtmhaVl-TRvf~~ERsQnPRJuS1QsPCUSPIjcHFLtoyDAJkAUCIwEjCwJlR1UmhZepl32jSWm1~CY-O2DyOfY88nskhYs7dODPQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5776819&amp;ar=391702&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsp01001a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/4/10.1093/dnares/dsp010/2/m_dsp01001b.jpeg?Expires=2147483647&amp;Signature=P3IqNn93jWovyF5G0oUuI6zANd~gHOPnSOqevh4E83SGASy17222QmrWIADuQEMcSKlL9LUyCBj88uzMf8PfNRZpMk02k1hxKALaem0z8Y8N5jcixN2auISKsmm-l6mzmSOcPJoTKgqvwlCzMsdLPpwgMcYfxqtVjUvB5tSJdHkpcWOElLHK0r1I1bhkRNvzaalJiEaEIleX2VOevWIVrzjE9uAHXT-S6J5ICnQ~kuc4VlGsxv1WQ0O~lmZZTWudY6PgAbk8PqW9X3ggXfXKXoDrsDtg3rs35zRFwoSHXY0Gkv3V6RpwUa144c9hS4-SPXR11cDnnKbFfTS4nLpH-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic linkage map of soybean constructed with SSR and STS markers. The linkage map was visualized graphically with MapChart. The LGs were constructed from 595 public SSR markers developed by USDA-ARS and DuPont Corporation (black characters), 1074 EST-derived SSR markers developed by Kazusa DNA Research Institute (red), and 141 SSR or STS markers developed by the Chiba University (blue). A total of 468 newly mapped PCR-based markers were polymorphic in one or more of the three mapping populations (bold). The EST-derived SSR markers GMES4205a, GMES6481, GMES2036, GMES4016 and GMES2497 were previously assigned to LG C2,21 but the JoinMap analysis here positioned these markers as well as GMES0706, CSSR1, GMES5782, GMES3776 and GMES1678 at one end of LG A1 (boxed region)." path-from-xml="dsp01001b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5776819/dsp01001b.jpeg" path-from-xml="dsp01001b.jpeg" target="_blank">View large</a><a section="5776819" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/4/10.1093/dnares/dsp010/2/dsp01001b.jpeg?Expires=2147483647&amp;Signature=AXB3APD0U7alclxj0gjrxjfuhvxygy1muR7ONtFkkr3~eyb97l~xQ3OSYYsxfxDAhDBPTilaGvGO~UBdr7McLUncI7i2HR4SbS5nSehEjbZnFwo10RKHe6ybzuglz-pTbYWdxt3lc9c386HnswqukSFcCVm7JtTttf8zPXfY7Lsxe233jJMWkql24SRA8W5BBKRC0Wor4U4V1cOESR7RB3FDQtNwR8MB1cKHZ2KcQKF-X82rhAHMt3WAGnBc9IV6ukAss2eBT8LezUIxuAczPGSvRNDFIFTL-TkClr8pQc2CD-9b~FxJaW3HkEMmzOjlGTKlOgAiDtNrcQdLqo2HwA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5776819&amp;ar=391702&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsp01001b.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/4/10.1093/dnares/dsp010/2/m_dsp01001c.jpeg?Expires=2147483647&amp;Signature=N17yotZl7WjWWKz-RQmMupo-92mog29jVha9rmbQJcghJ7vzGoT1pdmIgs2IPWf7N0spuLeId~pL6MGMg4eJa7pQ5DQ4t-yVuegeNG5rwWYa0WPrT1BlQE8iE47dz6KTRXlh4Yq0J702EbbdVPpXkrpMAiWUQkl12xF~7Mf5XYyIO2sZvlsUpDH-xhSD1uG79dDHjtFcAoSpglukfdXM0u-lG7RFiXLxn2nttcphvM-n6-1fInds1HNQK81H9BCumNJTM3vDd7-u8PQPi2zoHJKalmyjUA7qlb0LtcKRlXjn4ERBbnewVn1Wu1VWL9UD9j5ZQis0RHK1twY0VQBbBw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic linkage map of soybean constructed with SSR and STS markers. The linkage map was visualized graphically with MapChart. The LGs were constructed from 595 public SSR markers developed by USDA-ARS and DuPont Corporation (black characters), 1074 EST-derived SSR markers developed by Kazusa DNA Research Institute (red), and 141 SSR or STS markers developed by the Chiba University (blue). A total of 468 newly mapped PCR-based markers were polymorphic in one or more of the three mapping populations (bold). The EST-derived SSR markers GMES4205a, GMES6481, GMES2036, GMES4016 and GMES2497 were previously assigned to LG C2,21 but the JoinMap analysis here positioned these markers as well as GMES0706, CSSR1, GMES5782, GMES3776 and GMES1678 at one end of LG A1 (boxed region)." path-from-xml="dsp01001c.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5776819/dsp01001c.jpeg" path-from-xml="dsp01001c.jpeg" target="_blank">View large</a><a section="5776819" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/4/10.1093/dnares/dsp010/2/dsp01001c.jpeg?Expires=2147483647&amp;Signature=bHRy1TWkAlEF0IVqT5tg373~33g0n65ohyQeqvUgEOTauO6Cjmfg8fF2A-tkaQNxTXmZii7Qr1w-Tp0mzleErzCb1SdHU9MzqeeybhFc52L22h8V5eTOCBq-us5BW96fb11ng4-DAk0jU-I214eXs3x0J5iIIlHNr4YARfXlaVfixAcDPwkISSsu5IEhU9fGxUma9wsKIc5vlfxH9dopeSknXvFNzNO4S2JurFTKUBHNOju94k2PeTUtkhtriJdCwwVPbbDCf220vdc81g4bIIRBWDDw2LHBa~l6EIPWeyPd~8JBhQRSf4yudlCunmLp~uAzsxedo9HSuvIqQCRcUw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5776819&amp;ar=391702&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsp01001c.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/4/10.1093/dnares/dsp010/2/m_dsp01001d.jpeg?Expires=2147483647&amp;Signature=P7ipHq4P5F~fZTUqTy6JOOT9t0QHBjxi2fKwZxeHnlZffbPxzsIcAybLdy~cb9RM~9NfiH74R5cLjXPUPMSI-yg-bPs~MzsHeyNZ7r5zD~q2MZN4PjgEhQAchS99kqQem2Obc-a~2iuMXZCr0lqyeUhb~owCzNUSPEbPEKxfDnMlsxNgEP35nk8tXR487xalofTI1RefNvlPfdrkardRVzrp4oYW0Qvccxy~MYeQJFpq0wQ4d3S8qTaMZZsRXSwLQElPGKiN1JY4W7DiDpAa3ZsGdzUf5a~Vo2eemQJ9VPIXBow0Ihh4wSn9AyPe-FK0tx3i5o4LyPjPRSaZ3i-HmA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic linkage map of soybean constructed with SSR and STS markers. The linkage map was visualized graphically with MapChart. The LGs were constructed from 595 public SSR markers developed by USDA-ARS and DuPont Corporation (black characters), 1074 EST-derived SSR markers developed by Kazusa DNA Research Institute (red), and 141 SSR or STS markers developed by the Chiba University (blue). A total of 468 newly mapped PCR-based markers were polymorphic in one or more of the three mapping populations (bold). The EST-derived SSR markers GMES4205a, GMES6481, GMES2036, GMES4016 and GMES2497 were previously assigned to LG C2,21 but the JoinMap analysis here positioned these markers as well as GMES0706, CSSR1, GMES5782, GMES3776 and GMES1678 at one end of LG A1 (boxed region)." path-from-xml="dsp01001d.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5776819/dsp01001d.jpeg" path-from-xml="dsp01001d.jpeg" target="_blank">View large</a><a section="5776819" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/4/10.1093/dnares/dsp010/2/dsp01001d.jpeg?Expires=2147483647&amp;Signature=mNOvjSfA0wdK9JUU-7yR4XDU3agCHyAWHxme1aGUNtd-To-B~586aQJvJsSkF8fpZ6SWtBkP5r4tQbR8f6gTCs03rIFQD5A3zpwW0-ZcOjIzh1re-CuQ6KxK-NCJg~GN1TH5H9wO5rk96vderLP6IJYkr0h8e9asfxNYqwp1lEVGolWFFxFCaa7zB~nZTEqb5pHAPvsVmeixRL98v~-B-QHjV4DeldRfR4-srbKeyWLeN~hvD378L2hrw8K37V0w~WFPAsKzcBP~Zdo1f5ximt2XWYnJ8foMAE2QKIYU4Nc5~npvwUAK0McIE--M755PtJrnhK00XxUcSYl4p0~2ew__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5776819&amp;ar=391702&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsp01001d.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic linkage map of soybean constructed with SSR and STS markers. The linkage map was visualized graphically with MapChart. The LGs were constructed from 595 public SSR markers developed by USDA-ARS and DuPont Corporation (black characters), 1074 EST-derived SSR markers developed by Kazusa DNA Research Institute (red), and 141 SSR or STS markers developed by the Chiba University (blue). A total of 468 newly mapped PCR-based markers were polymorphic in one or more of the three mapping populations (bold). The EST-derived SSR markers GMES4205a, GMES6481, GMES2036, GMES4016 and GMES2497 were previously assigned to LG C2,<sup><a reveal-id="DSP010C21" data-open="DSP010C21" class="link link-ref link-reveal xref-bibr">21</a></sup> but the JoinMap analysis here positioned these markers as well as GMES0706, CSSR1, GMES5782, GMES3776 and GMES1678 at one end of LG A1 (boxed region).</p></div></div><div content-id="DSP010F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/4/10.1093/dnares/dsp010/2/m_dsp01001a.jpeg?Expires=2147483647&amp;Signature=XTSyDkxb9m5Q4vh-gdTaQt-~4bm8QCFy0oHlbS7bB0g4dbfYO04CrsA0YcCsRtbkeiCqmwvnY7SDH13vCF18ZfEtS8YWn8lonF48O0k4Or~4EDSxsCRQrrDzyyi0EdDEzpr6THkBl9DfhvoCr8HSgaTcBVGAKmgdeOcxVusoOtHTLSoc49Nraej8NDM8Qfo~3TnrJPtaCaLtZ25A1Xv3I5D7SffDfRTvpVbAfY27pqEGn6J9SltvCw7PNw5i9E~-2VsZ7ldgrR7aJoG4seBKutJUWyM6M2ABF5XpcE8w5lARHASuvRjeNoC-7cRU60MG9Lba6Bxki7FuyLzcrkweMw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic linkage map of soybean constructed with SSR and STS markers. The linkage map was visualized graphically with MapChart. The LGs were constructed from 595 public SSR markers developed by USDA-ARS and DuPont Corporation (black characters), 1074 EST-derived SSR markers developed by Kazusa DNA Research Institute (red), and 141 SSR or STS markers developed by the Chiba University (blue). A total of 468 newly mapped PCR-based markers were polymorphic in one or more of the three mapping populations (bold). The EST-derived SSR markers GMES4205a, GMES6481, GMES2036, GMES4016 and GMES2497 were previously assigned to LG C2,21 but the JoinMap analysis here positioned these markers as well as GMES0706, CSSR1, GMES5782, GMES3776 and GMES1678 at one end of LG A1 (boxed region)." path-from-xml="dsp01001a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5776819/dsp01001a.jpeg" path-from-xml="dsp01001a.jpeg" target="_blank">View large</a><a section="5776819" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/4/10.1093/dnares/dsp010/2/dsp01001a.jpeg?Expires=2147483647&amp;Signature=LCrK3Bc3jmbULEJLSLQbyD3EqRb8HgSOiCVkzNBWTBQ6fqCo7EEgvoxF~WKQhYOygzZi4o3uHsj0VszCDfRQeB6cUG0A~JaEhPZXhn1tK~lJsMLJJqAKGF~CGcM6vQT~CXYWSnt2cDRZvUC~f9S62jGF5L~rPkYXk60U7xoW3IdTdRRe563ZlOdhnCiuvZDKgQJVvqPCXgrRiTU5JaLrnIiD9YJ2j~99bsogY8ke1jIJ4k2kU4D8wJtmhaVl-TRvf~~ERsQnPRJuS1QsPCUSPIjcHFLtoyDAJkAUCIwEjCwJlR1UmhZepl32jSWm1~CY-O2DyOfY88nskhYs7dODPQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5776819&amp;ar=391702&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsp01001a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/4/10.1093/dnares/dsp010/2/m_dsp01001b.jpeg?Expires=2147483647&amp;Signature=P3IqNn93jWovyF5G0oUuI6zANd~gHOPnSOqevh4E83SGASy17222QmrWIADuQEMcSKlL9LUyCBj88uzMf8PfNRZpMk02k1hxKALaem0z8Y8N5jcixN2auISKsmm-l6mzmSOcPJoTKgqvwlCzMsdLPpwgMcYfxqtVjUvB5tSJdHkpcWOElLHK0r1I1bhkRNvzaalJiEaEIleX2VOevWIVrzjE9uAHXT-S6J5ICnQ~kuc4VlGsxv1WQ0O~lmZZTWudY6PgAbk8PqW9X3ggXfXKXoDrsDtg3rs35zRFwoSHXY0Gkv3V6RpwUa144c9hS4-SPXR11cDnnKbFfTS4nLpH-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic linkage map of soybean constructed with SSR and STS markers. The linkage map was visualized graphically with MapChart. The LGs were constructed from 595 public SSR markers developed by USDA-ARS and DuPont Corporation (black characters), 1074 EST-derived SSR markers developed by Kazusa DNA Research Institute (red), and 141 SSR or STS markers developed by the Chiba University (blue). A total of 468 newly mapped PCR-based markers were polymorphic in one or more of the three mapping populations (bold). The EST-derived SSR markers GMES4205a, GMES6481, GMES2036, GMES4016 and GMES2497 were previously assigned to LG C2,21 but the JoinMap analysis here positioned these markers as well as GMES0706, CSSR1, GMES5782, GMES3776 and GMES1678 at one end of LG A1 (boxed region)." path-from-xml="dsp01001b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5776819/dsp01001b.jpeg" path-from-xml="dsp01001b.jpeg" target="_blank">View large</a><a section="5776819" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/4/10.1093/dnares/dsp010/2/dsp01001b.jpeg?Expires=2147483647&amp;Signature=AXB3APD0U7alclxj0gjrxjfuhvxygy1muR7ONtFkkr3~eyb97l~xQ3OSYYsxfxDAhDBPTilaGvGO~UBdr7McLUncI7i2HR4SbS5nSehEjbZnFwo10RKHe6ybzuglz-pTbYWdxt3lc9c386HnswqukSFcCVm7JtTttf8zPXfY7Lsxe233jJMWkql24SRA8W5BBKRC0Wor4U4V1cOESR7RB3FDQtNwR8MB1cKHZ2KcQKF-X82rhAHMt3WAGnBc9IV6ukAss2eBT8LezUIxuAczPGSvRNDFIFTL-TkClr8pQc2CD-9b~FxJaW3HkEMmzOjlGTKlOgAiDtNrcQdLqo2HwA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5776819&amp;ar=391702&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsp01001b.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/4/10.1093/dnares/dsp010/2/m_dsp01001c.jpeg?Expires=2147483647&amp;Signature=N17yotZl7WjWWKz-RQmMupo-92mog29jVha9rmbQJcghJ7vzGoT1pdmIgs2IPWf7N0spuLeId~pL6MGMg4eJa7pQ5DQ4t-yVuegeNG5rwWYa0WPrT1BlQE8iE47dz6KTRXlh4Yq0J702EbbdVPpXkrpMAiWUQkl12xF~7Mf5XYyIO2sZvlsUpDH-xhSD1uG79dDHjtFcAoSpglukfdXM0u-lG7RFiXLxn2nttcphvM-n6-1fInds1HNQK81H9BCumNJTM3vDd7-u8PQPi2zoHJKalmyjUA7qlb0LtcKRlXjn4ERBbnewVn1Wu1VWL9UD9j5ZQis0RHK1twY0VQBbBw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic linkage map of soybean constructed with SSR and STS markers. The linkage map was visualized graphically with MapChart. The LGs were constructed from 595 public SSR markers developed by USDA-ARS and DuPont Corporation (black characters), 1074 EST-derived SSR markers developed by Kazusa DNA Research Institute (red), and 141 SSR or STS markers developed by the Chiba University (blue). A total of 468 newly mapped PCR-based markers were polymorphic in one or more of the three mapping populations (bold). The EST-derived SSR markers GMES4205a, GMES6481, GMES2036, GMES4016 and GMES2497 were previously assigned to LG C2,21 but the JoinMap analysis here positioned these markers as well as GMES0706, CSSR1, GMES5782, GMES3776 and GMES1678 at one end of LG A1 (boxed region)." path-from-xml="dsp01001c.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5776819/dsp01001c.jpeg" path-from-xml="dsp01001c.jpeg" target="_blank">View large</a><a section="5776819" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/4/10.1093/dnares/dsp010/2/dsp01001c.jpeg?Expires=2147483647&amp;Signature=bHRy1TWkAlEF0IVqT5tg373~33g0n65ohyQeqvUgEOTauO6Cjmfg8fF2A-tkaQNxTXmZii7Qr1w-Tp0mzleErzCb1SdHU9MzqeeybhFc52L22h8V5eTOCBq-us5BW96fb11ng4-DAk0jU-I214eXs3x0J5iIIlHNr4YARfXlaVfixAcDPwkISSsu5IEhU9fGxUma9wsKIc5vlfxH9dopeSknXvFNzNO4S2JurFTKUBHNOju94k2PeTUtkhtriJdCwwVPbbDCf220vdc81g4bIIRBWDDw2LHBa~l6EIPWeyPd~8JBhQRSf4yudlCunmLp~uAzsxedo9HSuvIqQCRcUw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5776819&amp;ar=391702&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsp01001c.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/4/10.1093/dnares/dsp010/2/m_dsp01001d.jpeg?Expires=2147483647&amp;Signature=P7ipHq4P5F~fZTUqTy6JOOT9t0QHBjxi2fKwZxeHnlZffbPxzsIcAybLdy~cb9RM~9NfiH74R5cLjXPUPMSI-yg-bPs~MzsHeyNZ7r5zD~q2MZN4PjgEhQAchS99kqQem2Obc-a~2iuMXZCr0lqyeUhb~owCzNUSPEbPEKxfDnMlsxNgEP35nk8tXR487xalofTI1RefNvlPfdrkardRVzrp4oYW0Qvccxy~MYeQJFpq0wQ4d3S8qTaMZZsRXSwLQElPGKiN1JY4W7DiDpAa3ZsGdzUf5a~Vo2eemQJ9VPIXBow0Ihh4wSn9AyPe-FK0tx3i5o4LyPjPRSaZ3i-HmA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic linkage map of soybean constructed with SSR and STS markers. The linkage map was visualized graphically with MapChart. The LGs were constructed from 595 public SSR markers developed by USDA-ARS and DuPont Corporation (black characters), 1074 EST-derived SSR markers developed by Kazusa DNA Research Institute (red), and 141 SSR or STS markers developed by the Chiba University (blue). A total of 468 newly mapped PCR-based markers were polymorphic in one or more of the three mapping populations (bold). The EST-derived SSR markers GMES4205a, GMES6481, GMES2036, GMES4016 and GMES2497 were previously assigned to LG C2,21 but the JoinMap analysis here positioned these markers as well as GMES0706, CSSR1, GMES5782, GMES3776 and GMES1678 at one end of LG A1 (boxed region)." path-from-xml="dsp01001d.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5776819/dsp01001d.jpeg" path-from-xml="dsp01001d.jpeg" target="_blank">View large</a><a section="5776819" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/4/10.1093/dnares/dsp010/2/dsp01001d.jpeg?Expires=2147483647&amp;Signature=mNOvjSfA0wdK9JUU-7yR4XDU3agCHyAWHxme1aGUNtd-To-B~586aQJvJsSkF8fpZ6SWtBkP5r4tQbR8f6gTCs03rIFQD5A3zpwW0-ZcOjIzh1re-CuQ6KxK-NCJg~GN1TH5H9wO5rk96vderLP6IJYkr0h8e9asfxNYqwp1lEVGolWFFxFCaa7zB~nZTEqb5pHAPvsVmeixRL98v~-B-QHjV4DeldRfR4-srbKeyWLeN~hvD378L2hrw8K37V0w~WFPAsKzcBP~Zdo1f5ximt2XWYnJ8foMAE2QKIYU4Nc5~npvwUAK0McIE--M755PtJrnhK00XxUcSYl4p0~2ew__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5776819&amp;ar=391702&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsp01001d.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic linkage map of soybean constructed with SSR and STS markers. The linkage map was visualized graphically with MapChart. The LGs were constructed from 595 public SSR markers developed by USDA-ARS and DuPont Corporation (black characters), 1074 EST-derived SSR markers developed by Kazusa DNA Research Institute (red), and 141 SSR or STS markers developed by the Chiba University (blue). A total of 468 newly mapped PCR-based markers were polymorphic in one or more of the three mapping populations (bold). The EST-derived SSR markers GMES4205a, GMES6481, GMES2036, GMES4016 and GMES2497 were previously assigned to LG C2,<sup><a reveal-id="DSP010C21" data-open="DSP010C21" class="link link-ref link-reveal xref-bibr">21</a></sup> but the JoinMap analysis here positioned these markers as well as GMES0706, CSSR1, GMES5782, GMES3776 and GMES1678 at one end of LG A1 (boxed region).</p></div></div>                        <a id="5776820" scrollto-destination="5776820"></a>
<div content-id="DSP010TB2" class="table-modal"><div class="table-wrap"><div id="DSP010TB2" class="table-wrap-title" data-id="DSP010TB2"><div class="label -label">Table2</div><div class="caption"><p>Summary of the integrated linkage map and mapped markers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LG</th><th colspan="4">Number of markers<hr /></th><th>Length (cM)</th><th>Average interval (cM)</th></tr><tr><th></th><th>ES<sup>a</sup></th><th>CS<sup>a</sup></th><th>US<sup>a</sup></th><th>Total</th><th></th><th></th></tr></thead><tbody><tr><td>A1</td><td>38</td><td>6</td><td>35</td><td>79</td><td>130.4</td><td>1.65</td></tr><tr><td>A2</td><td>61</td><td>6</td><td>32</td><td>99</td><td>175.8</td><td>1.78</td></tr><tr><td>B1</td><td>59</td><td>7</td><td>21</td><td>87</td><td>131.3</td><td>1.51</td></tr><tr><td>B2</td><td>49</td><td>11</td><td>21</td><td>81</td><td>97.8</td><td>1.21</td></tr><tr><td>C1</td><td>42</td><td>9</td><td>32</td><td>83</td><td>136.1</td><td>1.64</td></tr><tr><td>C2</td><td>56</td><td>10</td><td>34</td><td>100</td><td>146.3</td><td>1.46</td></tr><tr><td>D1a</td><td>43</td><td>6</td><td>24</td><td>73</td><td>122.0</td><td>1.67</td></tr><tr><td>D1b</td><td>70</td><td>4</td><td>32</td><td>106</td><td>144.1</td><td>1.36</td></tr><tr><td>D2</td><td>64</td><td>4</td><td>31</td><td>99</td><td>117.7</td><td>1.19</td></tr><tr><td>E</td><td>46</td><td>9</td><td>23</td><td>78</td><td>104.7</td><td>1.34</td></tr><tr><td>F</td><td>69</td><td>8</td><td>37</td><td>114</td><td>122.0</td><td>1.07</td></tr><tr><td>G</td><td>63</td><td>9</td><td>34</td><td>106</td><td>124.1</td><td>1.17</td></tr><tr><td>H</td><td>46</td><td>1</td><td>30</td><td>77</td><td>105.6</td><td>1.37</td></tr><tr><td>I</td><td>48</td><td>12</td><td>29</td><td>89</td><td>109.6</td><td>1.23</td></tr><tr><td>J</td><td>48</td><td>7</td><td>27</td><td>82</td><td>77.4</td><td>0.94</td></tr><tr><td>K</td><td>58</td><td>10</td><td>28</td><td>96</td><td>112.2</td><td>1.17</td></tr><tr><td>L</td><td>50</td><td>7</td><td>31</td><td>88</td><td>111.9</td><td>1.27</td></tr><tr><td>M</td><td>56</td><td>6</td><td>31</td><td>93</td><td>124.1</td><td>1.33</td></tr><tr><td>N</td><td>37</td><td>4</td><td>29</td><td>70</td><td>108.0</td><td>1.55</td></tr><tr><td>O</td><td>71</td><td>5</td><td>34</td><td>110</td><td>141.8</td><td>1.29</td></tr><tr><td>Total</td><td>1074</td><td>141</td><td>595</td><td>1810</td><td>2442.9</td><td></td></tr><tr><td>Average</td><td>53.7</td><td>7.1</td><td>29.8</td><td>90.5</td><td>122.1</td><td>1.36</td></tr></tbody></table></div><div content-id="DSP010TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>LG</th><th colspan="4">Number of markers<hr /></th><th>Length (cM)</th><th>Average interval (cM)</th></tr><tr><th></th><th>ES<sup>a</sup></th><th>CS<sup>a</sup></th><th>US<sup>a</sup></th><th>Total</th><th></th><th></th></tr></thead><tbody><tr><td>A1</td><td>38</td><td>6</td><td>35</td><td>79</td><td>130.4</td><td>1.65</td></tr><tr><td>A2</td><td>61</td><td>6</td><td>32</td><td>99</td><td>175.8</td><td>1.78</td></tr><tr><td>B1</td><td>59</td><td>7</td><td>21</td><td>87</td><td>131.3</td><td>1.51</td></tr><tr><td>B2</td><td>49</td><td>11</td><td>21</td><td>81</td><td>97.8</td><td>1.21</td></tr><tr><td>C1</td><td>42</td><td>9</td><td>32</td><td>83</td><td>136.1</td><td>1.64</td></tr><tr><td>C2</td><td>56</td><td>10</td><td>34</td><td>100</td><td>146.3</td><td>1.46</td></tr><tr><td>D1a</td><td>43</td><td>6</td><td>24</td><td>73</td><td>122.0</td><td>1.67</td></tr><tr><td>D1b</td><td>70</td><td>4</td><td>32</td><td>106</td><td>144.1</td><td>1.36</td></tr><tr><td>D2</td><td>64</td><td>4</td><td>31</td><td>99</td><td>117.7</td><td>1.19</td></tr><tr><td>E</td><td>46</td><td>9</td><td>23</td><td>78</td><td>104.7</td><td>1.34</td></tr><tr><td>F</td><td>69</td><td>8</td><td>37</td><td>114</td><td>122.0</td><td>1.07</td></tr><tr><td>G</td><td>63</td><td>9</td><td>34</td><td>106</td><td>124.1</td><td>1.17</td></tr><tr><td>H</td><td>46</td><td>1</td><td>30</td><td>77</td><td>105.6</td><td>1.37</td></tr><tr><td>I</td><td>48</td><td>12</td><td>29</td><td>89</td><td>109.6</td><td>1.23</td></tr><tr><td>J</td><td>48</td><td>7</td><td>27</td><td>82</td><td>77.4</td><td>0.94</td></tr><tr><td>K</td><td>58</td><td>10</td><td>28</td><td>96</td><td>112.2</td><td>1.17</td></tr><tr><td>L</td><td>50</td><td>7</td><td>31</td><td>88</td><td>111.9</td><td>1.27</td></tr><tr><td>M</td><td>56</td><td>6</td><td>31</td><td>93</td><td>124.1</td><td>1.33</td></tr><tr><td>N</td><td>37</td><td>4</td><td>29</td><td>70</td><td>108.0</td><td>1.55</td></tr><tr><td>O</td><td>71</td><td>5</td><td>34</td><td>110</td><td>141.8</td><td>1.29</td></tr><tr><td>Total</td><td>1074</td><td>141</td><td>595</td><td>1810</td><td>2442.9</td><td></td></tr><tr><td>Average</td><td>53.7</td><td>7.1</td><td>29.8</td><td>90.5</td><td>122.1</td><td>1.36</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Abbreviations are as in Table<a reveal-id="DSP010TB1" data-open="DSP010TB1" class="link link-reveal link-table xref-fig">1</a>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5776820" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP010TB2" class="table-wrap-title" data-id="DSP010TB2"><div class="label -label">Table2</div><div class="caption"><p>Summary of the integrated linkage map and mapped markers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LG</th><th colspan="4">Number of markers<hr /></th><th>Length (cM)</th><th>Average interval (cM)</th></tr><tr><th></th><th>ES<sup>a</sup></th><th>CS<sup>a</sup></th><th>US<sup>a</sup></th><th>Total</th><th></th><th></th></tr></thead><tbody><tr><td>A1</td><td>38</td><td>6</td><td>35</td><td>79</td><td>130.4</td><td>1.65</td></tr><tr><td>A2</td><td>61</td><td>6</td><td>32</td><td>99</td><td>175.8</td><td>1.78</td></tr><tr><td>B1</td><td>59</td><td>7</td><td>21</td><td>87</td><td>131.3</td><td>1.51</td></tr><tr><td>B2</td><td>49</td><td>11</td><td>21</td><td>81</td><td>97.8</td><td>1.21</td></tr><tr><td>C1</td><td>42</td><td>9</td><td>32</td><td>83</td><td>136.1</td><td>1.64</td></tr><tr><td>C2</td><td>56</td><td>10</td><td>34</td><td>100</td><td>146.3</td><td>1.46</td></tr><tr><td>D1a</td><td>43</td><td>6</td><td>24</td><td>73</td><td>122.0</td><td>1.67</td></tr><tr><td>D1b</td><td>70</td><td>4</td><td>32</td><td>106</td><td>144.1</td><td>1.36</td></tr><tr><td>D2</td><td>64</td><td>4</td><td>31</td><td>99</td><td>117.7</td><td>1.19</td></tr><tr><td>E</td><td>46</td><td>9</td><td>23</td><td>78</td><td>104.7</td><td>1.34</td></tr><tr><td>F</td><td>69</td><td>8</td><td>37</td><td>114</td><td>122.0</td><td>1.07</td></tr><tr><td>G</td><td>63</td><td>9</td><td>34</td><td>106</td><td>124.1</td><td>1.17</td></tr><tr><td>H</td><td>46</td><td>1</td><td>30</td><td>77</td><td>105.6</td><td>1.37</td></tr><tr><td>I</td><td>48</td><td>12</td><td>29</td><td>89</td><td>109.6</td><td>1.23</td></tr><tr><td>J</td><td>48</td><td>7</td><td>27</td><td>82</td><td>77.4</td><td>0.94</td></tr><tr><td>K</td><td>58</td><td>10</td><td>28</td><td>96</td><td>112.2</td><td>1.17</td></tr><tr><td>L</td><td>50</td><td>7</td><td>31</td><td>88</td><td>111.9</td><td>1.27</td></tr><tr><td>M</td><td>56</td><td>6</td><td>31</td><td>93</td><td>124.1</td><td>1.33</td></tr><tr><td>N</td><td>37</td><td>4</td><td>29</td><td>70</td><td>108.0</td><td>1.55</td></tr><tr><td>O</td><td>71</td><td>5</td><td>34</td><td>110</td><td>141.8</td><td>1.29</td></tr><tr><td>Total</td><td>1074</td><td>141</td><td>595</td><td>1810</td><td>2442.9</td><td></td></tr><tr><td>Average</td><td>53.7</td><td>7.1</td><td>29.8</td><td>90.5</td><td>122.1</td><td>1.36</td></tr></tbody></table></div><div content-id="DSP010TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>LG</th><th colspan="4">Number of markers<hr /></th><th>Length (cM)</th><th>Average interval (cM)</th></tr><tr><th></th><th>ES<sup>a</sup></th><th>CS<sup>a</sup></th><th>US<sup>a</sup></th><th>Total</th><th></th><th></th></tr></thead><tbody><tr><td>A1</td><td>38</td><td>6</td><td>35</td><td>79</td><td>130.4</td><td>1.65</td></tr><tr><td>A2</td><td>61</td><td>6</td><td>32</td><td>99</td><td>175.8</td><td>1.78</td></tr><tr><td>B1</td><td>59</td><td>7</td><td>21</td><td>87</td><td>131.3</td><td>1.51</td></tr><tr><td>B2</td><td>49</td><td>11</td><td>21</td><td>81</td><td>97.8</td><td>1.21</td></tr><tr><td>C1</td><td>42</td><td>9</td><td>32</td><td>83</td><td>136.1</td><td>1.64</td></tr><tr><td>C2</td><td>56</td><td>10</td><td>34</td><td>100</td><td>146.3</td><td>1.46</td></tr><tr><td>D1a</td><td>43</td><td>6</td><td>24</td><td>73</td><td>122.0</td><td>1.67</td></tr><tr><td>D1b</td><td>70</td><td>4</td><td>32</td><td>106</td><td>144.1</td><td>1.36</td></tr><tr><td>D2</td><td>64</td><td>4</td><td>31</td><td>99</td><td>117.7</td><td>1.19</td></tr><tr><td>E</td><td>46</td><td>9</td><td>23</td><td>78</td><td>104.7</td><td>1.34</td></tr><tr><td>F</td><td>69</td><td>8</td><td>37</td><td>114</td><td>122.0</td><td>1.07</td></tr><tr><td>G</td><td>63</td><td>9</td><td>34</td><td>106</td><td>124.1</td><td>1.17</td></tr><tr><td>H</td><td>46</td><td>1</td><td>30</td><td>77</td><td>105.6</td><td>1.37</td></tr><tr><td>I</td><td>48</td><td>12</td><td>29</td><td>89</td><td>109.6</td><td>1.23</td></tr><tr><td>J</td><td>48</td><td>7</td><td>27</td><td>82</td><td>77.4</td><td>0.94</td></tr><tr><td>K</td><td>58</td><td>10</td><td>28</td><td>96</td><td>112.2</td><td>1.17</td></tr><tr><td>L</td><td>50</td><td>7</td><td>31</td><td>88</td><td>111.9</td><td>1.27</td></tr><tr><td>M</td><td>56</td><td>6</td><td>31</td><td>93</td><td>124.1</td><td>1.33</td></tr><tr><td>N</td><td>37</td><td>4</td><td>29</td><td>70</td><td>108.0</td><td>1.55</td></tr><tr><td>O</td><td>71</td><td>5</td><td>34</td><td>110</td><td>141.8</td><td>1.29</td></tr><tr><td>Total</td><td>1074</td><td>141</td><td>595</td><td>1810</td><td>2442.9</td><td></td></tr><tr><td>Average</td><td>53.7</td><td>7.1</td><td>29.8</td><td>90.5</td><td>122.1</td><td>1.36</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Abbreviations are as in Table<a reveal-id="DSP010TB1" data-open="DSP010TB1" class="link link-reveal link-table xref-fig">1</a>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5776820" target="_blank">View Large</a></div></div>                        <a id="5776821" scrollto-destination="5776821"></a>
<div content-id="DSP010TB3" class="table-modal"><div class="table-wrap"><div id="DSP010TB3" class="table-wrap-title" data-id="DSP010TB3"><div class="label -label">Table3</div><div class="caption"><p>Marker loci at the boundaries of genomic intervals of &gt;10 cM containing no SSR or STS markers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LG</th><th>Flanking SSR loci</th><th>Interval distance (cM)</th></tr></thead><tbody><tr><td>A1</td><td>GMES6481GMES5019</td><td>10.8</td></tr><tr><td>A2</td><td>GMES3189GMES1806</td><td>12.3</td></tr><tr><td>C1</td><td>Sat_235GMES1325</td><td>35.3</td></tr><tr><td>C2</td><td>Sat_130GMES5629</td><td>12.5</td></tr><tr><td>E</td><td>GMES6270GMES4123</td><td>11.7</td></tr><tr><td>F</td><td>AW756935GMES4726</td><td>10.4</td></tr><tr><td>K</td><td>GMES2057CSSR554</td><td>12.9</td></tr></tbody></table></div><div content-id="DSP010TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>LG</th><th>Flanking SSR loci</th><th>Interval distance (cM)</th></tr></thead><tbody><tr><td>A1</td><td>GMES6481GMES5019</td><td>10.8</td></tr><tr><td>A2</td><td>GMES3189GMES1806</td><td>12.3</td></tr><tr><td>C1</td><td>Sat_235GMES1325</td><td>35.3</td></tr><tr><td>C2</td><td>Sat_130GMES5629</td><td>12.5</td></tr><tr><td>E</td><td>GMES6270GMES4123</td><td>11.7</td></tr><tr><td>F</td><td>AW756935GMES4726</td><td>10.4</td></tr><tr><td>K</td><td>GMES2057CSSR554</td><td>12.9</td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5776821" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP010TB3" class="table-wrap-title" data-id="DSP010TB3"><div class="label -label">Table3</div><div class="caption"><p>Marker loci at the boundaries of genomic intervals of &gt;10 cM containing no SSR or STS markers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LG</th><th>Flanking SSR loci</th><th>Interval distance (cM)</th></tr></thead><tbody><tr><td>A1</td><td>GMES6481GMES5019</td><td>10.8</td></tr><tr><td>A2</td><td>GMES3189GMES1806</td><td>12.3</td></tr><tr><td>C1</td><td>Sat_235GMES1325</td><td>35.3</td></tr><tr><td>C2</td><td>Sat_130GMES5629</td><td>12.5</td></tr><tr><td>E</td><td>GMES6270GMES4123</td><td>11.7</td></tr><tr><td>F</td><td>AW756935GMES4726</td><td>10.4</td></tr><tr><td>K</td><td>GMES2057CSSR554</td><td>12.9</td></tr></tbody></table></div><div content-id="DSP010TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>LG</th><th>Flanking SSR loci</th><th>Interval distance (cM)</th></tr></thead><tbody><tr><td>A1</td><td>GMES6481GMES5019</td><td>10.8</td></tr><tr><td>A2</td><td>GMES3189GMES1806</td><td>12.3</td></tr><tr><td>C1</td><td>Sat_235GMES1325</td><td>35.3</td></tr><tr><td>C2</td><td>Sat_130GMES5629</td><td>12.5</td></tr><tr><td>E</td><td>GMES6270GMES4123</td><td>11.7</td></tr><tr><td>F</td><td>AW756935GMES4726</td><td>10.4</td></tr><tr><td>K</td><td>GMES2057CSSR554</td><td>12.9</td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5776821" target="_blank">View Large</a></div></div><p>The integrated linkage map contains 132 dominant marker loci, including 83 of the 1074 EST-derived SSR markers, 27 of the 141 SSR or STS markers, and 22 of the 595 public SSR markers, in at least one of the mapping populations. However, amplified products of each of 77 of these 132 marker loci exhibited polymorphism with bands of different sizes among 23 soybean cultivars and 1 wild soybean line, indicating that most of these loci could be used as codominant markers dependent on the combination of genotypes. Among the mapped molecular markers, 32 EST-derived SSR markers and 2 SSR markers developed by the Chiba University detected double loci in one or more than two of the mapping populations. One EST-derived SSR marker, GMES4205, gave four polymorphic bands positioned in four different LGs, whereas two SSR and one STS markers developed by the Chiba University were mapped to positions different from those reported previously.<sup><a reveal-id="DSP010C20" data-open="DSP010C20" class="link link-ref link-reveal xref-bibr">20</a></sup> These multiple loci were discriminated by appending a lowercase letter, as in GMES4205a and GMES4205b (Fig.<a reveal-id="DSP010F1" data-open="DSP010F1" class="link link-reveal link-table xref-fig">1</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>).</p>                            <h3 scrollto-destination=5776823 id="5776823" class="section-title" data-legacy-id=s3d>3.4. Polymorphism of USDA-ARS SSR and EST-derived SSR markers</h3>
<p>We examined allelic variation among the 24 soybean genotypes with the mapped SSR markers on the integrated linkage map. We selected 934 EST-derived SSR markers and 304 public SSR markers on the basis of both their unambiguous banding patterns and their distribution throughout the entire genome.<sup><a reveal-id="DSP010C12" data-open="DSP010C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSP010C21" data-open="DSP010C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Though the two groups of the SSR markers were analyzed by the different electrophoresis systems on either polyacrylamide or agarose gel, the resolutions of amplified products were almost equivalent in both systems. For confirmation, the electrophoresis system for each marker was indicated in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>. The number of alleles per locus ranged from 2 to 7 with an average of 2.8 for the EST-derived SSR markers and 3.1 for the public SSR markers (Fig.<a reveal-id="DSP010F2" data-open="DSP010F2" class="link link-reveal link-table xref-fig">2</a>, Table<a reveal-id="DSP010TB4" data-open="DSP010TB4" class="link link-reveal link-table xref-fig">4</a>). The most frequent allele numbers were 2 and 3 for the EST-derived SSR and public SSR markers, respectively. The PIC value, a measure of allelic diversity at a locus, ranged from 0.08 to 0.83, with an average value of 0.40 for all markers; the average was 0.38 for the EST-derived SSR markers and 0.46 for the public SSR markers (Fig.<a reveal-id="DSP010F2" data-open="DSP010F2" class="link link-reveal link-table xref-fig">2</a>, Table<a reveal-id="DSP010TB4" data-open="DSP010TB4" class="link link-reveal link-table xref-fig">4</a>). Marker loci with PIC values between 0.40 and 0.50 were the most common among both groups. The highest and lowest average PIC values for the EST-derived SSR markers were detected in LG G (0.49) and LGs D2 and M (0.34), respectively. In contrast, the highest and lowest such values for the public SSR markers were detected in LG D2 (0.52) and LG H (0.34), respectively (Table<a reveal-id="DSP010TB4" data-open="DSP010TB4" class="link link-reveal link-table xref-fig">4</a>). Clusters of marker loci with a higher than average PIC value, such as those located between GMES1587 and GMES4303 (average PIC value of 0.58), were detected in LG G (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). In contrast, marker loci located between GMES0688 and GMES4161 in LG O showed an average PIC value of only 0.19.</p>                        <a id="5776825" scrollto-destination="5776825"></a>
<div id="DSP010F2" data-id="DSP010F2" class="fig fig-section"><div class="label fig-label">Figure2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/4/10.1093/dnares/dsp010/2/m_dsp01002.jpeg?Expires=2147483647&Signature=BW3JNVNeerdomxcBm5AhrZ2JIpdQaHiJoy-ZZhW4Xk2IlTxHtqcb1vaZuTNCO38m~gNMJQopm5XLJPu~pE3R2KS6UN1IP-h7gFXkGCHE8ynWCo2YJ3h4ma-sMqIXK1ePIqjUCQX6fBuwsCTY2urcjpnMgbG-fVDlVSLqEC8KBUDsSXk~lHuhDcS6a-76ckXxSYSsMJLSxIrzpUo2plLJdqr7OnnsEIUaVpas9Q9pATKD-f1zpFoGn9cBqF2R62AMhzd2GkyGTGoUgwt-u00w8~W~-EWg2eiMyoVEdnfJL5gvKnRRDGbt8GaO4NOlyUGME74ri0brOr7ZvrOKwCJ5FA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of allele frequencies and PIC values for markers in 23 soybean cultivars and one wild accession. (A) Public SSR markers developed by USDA-ARS and DuPont Corporation. (B) EST-derived SSR markers." path-from-xml="dsp01002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5776825/dsp01002.jpeg" path-from-xml="dsp01002.jpeg" target="_blank">View large</a><a section="5776825" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/4/10.1093/dnares/dsp010/2/dsp01002.jpeg?Expires=2147483647&Signature=IyJUMdZ83x8FyV31Hls2Z4QhaQz-i4anZH2lsdWWx-VGidF9c11ARXYL6yHvxvVY~sd8yPdOJlnkXW3c9em1SKlNIc3rCC-2EWdKC3ycLJELwDUD04795LkYyOs6UGNjKocNfeAHRk8HDQIoIHZXRD00VUx10Ek5H5919CX5iQp6q8DuPqfIZKgqj7CrCoR~yfaIeofrUCINJT-bza4OgEEs07mw7WkHNi4OTt5Hyuk1-doM~OEbOTvkJNkYIoxVfV~e5Wz9MeUUeAIRHD3ykgQY-fdRQXpjTgWDNKWvOIjUBCYAMIbVfIUiRrzVnddbhnTkThseMAnc8WiBP2TGVQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5776825&ar=391702&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of allele frequencies and PIC values for markers in 23 soybean cultivars and one wild accession. (<strong>A</strong>) Public SSR markers developed by USDA-ARS and DuPont Corporation. (<strong>B</strong>) EST-derived SSR markers.</p></div></div><div content-id="DSP010F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/4/10.1093/dnares/dsp010/2/m_dsp01002.jpeg?Expires=2147483647&Signature=BW3JNVNeerdomxcBm5AhrZ2JIpdQaHiJoy-ZZhW4Xk2IlTxHtqcb1vaZuTNCO38m~gNMJQopm5XLJPu~pE3R2KS6UN1IP-h7gFXkGCHE8ynWCo2YJ3h4ma-sMqIXK1ePIqjUCQX6fBuwsCTY2urcjpnMgbG-fVDlVSLqEC8KBUDsSXk~lHuhDcS6a-76ckXxSYSsMJLSxIrzpUo2plLJdqr7OnnsEIUaVpas9Q9pATKD-f1zpFoGn9cBqF2R62AMhzd2GkyGTGoUgwt-u00w8~W~-EWg2eiMyoVEdnfJL5gvKnRRDGbt8GaO4NOlyUGME74ri0brOr7ZvrOKwCJ5FA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of allele frequencies and PIC values for markers in 23 soybean cultivars and one wild accession. (A) Public SSR markers developed by USDA-ARS and DuPont Corporation. (B) EST-derived SSR markers." path-from-xml="dsp01002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5776825/dsp01002.jpeg" path-from-xml="dsp01002.jpeg" target="_blank">View large</a><a section="5776825" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/4/10.1093/dnares/dsp010/2/dsp01002.jpeg?Expires=2147483647&Signature=IyJUMdZ83x8FyV31Hls2Z4QhaQz-i4anZH2lsdWWx-VGidF9c11ARXYL6yHvxvVY~sd8yPdOJlnkXW3c9em1SKlNIc3rCC-2EWdKC3ycLJELwDUD04795LkYyOs6UGNjKocNfeAHRk8HDQIoIHZXRD00VUx10Ek5H5919CX5iQp6q8DuPqfIZKgqj7CrCoR~yfaIeofrUCINJT-bza4OgEEs07mw7WkHNi4OTt5Hyuk1-doM~OEbOTvkJNkYIoxVfV~e5Wz9MeUUeAIRHD3ykgQY-fdRQXpjTgWDNKWvOIjUBCYAMIbVfIUiRrzVnddbhnTkThseMAnc8WiBP2TGVQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5776825&ar=391702&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp01002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of allele frequencies and PIC values for markers in 23 soybean cultivars and one wild accession. (<strong>A</strong>) Public SSR markers developed by USDA-ARS and DuPont Corporation. (<strong>B</strong>) EST-derived SSR markers.</p></div></div>                        <a id="5776826" scrollto-destination="5776826"></a>
<div content-id="DSP010TB4" class="table-modal"><div class="table-wrap"><div id="DSP010TB4" class="table-wrap-title" data-id="DSP010TB4"><div class="label -label">Table4</div><div class="caption"><p>Average numbers of alleles and PIC values for mapped markers in each LG</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LG</th><th colspan="3">ES<sup>a</sup><hr /></th><th colspan="3">US<sup>a</sup><hr /></th><th colspan="3">Total<hr /></th></tr><tr><th></th><th>Number</th><th>Average PIC value</th><th>Average number of alleles</th><th>Number</th><th>Average PIC value</th><th>Average number of alleles</th><th>Number</th><th>Average PIC value</th><th>Average number of alleles</th></tr></thead><tbody><tr><td>A1</td><td>35</td><td>0.39</td><td>2.9</td><td>16</td><td>0.48</td><td>3.2</td><td>51</td><td>0.42</td><td>3.0</td></tr><tr><td>A2</td><td>56</td><td>0.39</td><td>2.7</td><td>15</td><td>0.47</td><td>3.1</td><td>71</td><td>0.41</td><td>2.8</td></tr><tr><td>B1</td><td>51</td><td>0.36</td><td>2.6</td><td>12</td><td>0.44</td><td>2.8</td><td>63</td><td>0.38</td><td>2.7</td></tr><tr><td>B2</td><td>40</td><td>0.43</td><td>2.8</td><td>13</td><td>0.50</td><td>3.2</td><td>53</td><td>0.45</td><td>2.9</td></tr><tr><td>C1</td><td>35</td><td>0.37</td><td>2.8</td><td>18</td><td>0.48</td><td>3.1</td><td>53</td><td>0.41</td><td>2.9</td></tr><tr><td>C2</td><td>50</td><td>0.36</td><td>2.8</td><td>20</td><td>0.48</td><td>3.4</td><td>70</td><td>0.40</td><td>2.9</td></tr><tr><td>D1a</td><td>38</td><td>0.42</td><td>2.9</td><td>13</td><td>0.45</td><td>2.8</td><td>51</td><td>0.42</td><td>2.9</td></tr><tr><td>D1b</td><td>58</td><td>0.37</td><td>2.7</td><td>13</td><td>0.47</td><td>3.3</td><td>71</td><td>0.39</td><td>2.8</td></tr><tr><td>D2</td><td>53</td><td>0.34</td><td>2.5</td><td>14</td><td>0.52</td><td>3.9</td><td>67</td><td>0.38</td><td>2.8</td></tr><tr><td>E</td><td>38</td><td>0.41</td><td>2.7</td><td>13</td><td>0.42</td><td>2.8</td><td>51</td><td>0.41</td><td>2.7</td></tr><tr><td>F</td><td>58</td><td>0.39</td><td>2.9</td><td>16</td><td>0.47</td><td>3.0</td><td>74</td><td>0.41</td><td>2.9</td></tr><tr><td>G</td><td>58</td><td>0.49</td><td>3.0</td><td>21</td><td>0.47</td><td>3.4</td><td>79</td><td>0.48</td><td>3.1</td></tr><tr><td>H</td><td>43</td><td>0.41</td><td>2.9</td><td>11</td><td>0.34</td><td>2.5</td><td>54</td><td>0.39</td><td>2.9</td></tr><tr><td>I</td><td>42</td><td>0.38</td><td>2.6</td><td>17</td><td>0.48</td><td>3.4</td><td>59</td><td>0.41</td><td>2.8</td></tr><tr><td>J</td><td>41</td><td>0.38</td><td>2.9</td><td>18</td><td>0.45</td><td>3.1</td><td>59</td><td>0.40</td><td>2.9</td></tr><tr><td>K</td><td>51</td><td>0.39</td><td>3.0</td><td>16</td><td>0.42</td><td>3.0</td><td>67</td><td>0.40</td><td>3.0</td></tr><tr><td>L</td><td>45</td><td>0.37</td><td>2.8</td><td>16</td><td>0.46</td><td>3.3</td><td>61</td><td>0.39</td><td>2.9</td></tr><tr><td>M</td><td>49</td><td>0.34</td><td>2.4</td><td>13</td><td>0.49</td><td>3.1</td><td>62</td><td>0.37</td><td>2.5</td></tr><tr><td>N</td><td>34</td><td>0.35</td><td>2.5</td><td>14</td><td>0.46</td><td>3.0</td><td>48</td><td>0.38</td><td>2.6</td></tr><tr><td>O</td><td>59</td><td>0.35</td><td>2.6</td><td>15</td><td>0.46</td><td>3.1</td><td>74</td><td>0.37</td><td>2.7</td></tr><tr><td>Total</td><td>934</td><td>0.38</td><td>2.8</td><td>304</td><td>0.46</td><td>3.1</td><td>1238</td><td>0.40</td><td>2.8</td></tr></tbody></table></div><div content-id="DSP010TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>LG</th><th colspan="3">ES<sup>a</sup><hr /></th><th colspan="3">US<sup>a</sup><hr /></th><th colspan="3">Total<hr /></th></tr><tr><th></th><th>Number</th><th>Average PIC value</th><th>Average number of alleles</th><th>Number</th><th>Average PIC value</th><th>Average number of alleles</th><th>Number</th><th>Average PIC value</th><th>Average number of alleles</th></tr></thead><tbody><tr><td>A1</td><td>35</td><td>0.39</td><td>2.9</td><td>16</td><td>0.48</td><td>3.2</td><td>51</td><td>0.42</td><td>3.0</td></tr><tr><td>A2</td><td>56</td><td>0.39</td><td>2.7</td><td>15</td><td>0.47</td><td>3.1</td><td>71</td><td>0.41</td><td>2.8</td></tr><tr><td>B1</td><td>51</td><td>0.36</td><td>2.6</td><td>12</td><td>0.44</td><td>2.8</td><td>63</td><td>0.38</td><td>2.7</td></tr><tr><td>B2</td><td>40</td><td>0.43</td><td>2.8</td><td>13</td><td>0.50</td><td>3.2</td><td>53</td><td>0.45</td><td>2.9</td></tr><tr><td>C1</td><td>35</td><td>0.37</td><td>2.8</td><td>18</td><td>0.48</td><td>3.1</td><td>53</td><td>0.41</td><td>2.9</td></tr><tr><td>C2</td><td>50</td><td>0.36</td><td>2.8</td><td>20</td><td>0.48</td><td>3.4</td><td>70</td><td>0.40</td><td>2.9</td></tr><tr><td>D1a</td><td>38</td><td>0.42</td><td>2.9</td><td>13</td><td>0.45</td><td>2.8</td><td>51</td><td>0.42</td><td>2.9</td></tr><tr><td>D1b</td><td>58</td><td>0.37</td><td>2.7</td><td>13</td><td>0.47</td><td>3.3</td><td>71</td><td>0.39</td><td>2.8</td></tr><tr><td>D2</td><td>53</td><td>0.34</td><td>2.5</td><td>14</td><td>0.52</td><td>3.9</td><td>67</td><td>0.38</td><td>2.8</td></tr><tr><td>E</td><td>38</td><td>0.41</td><td>2.7</td><td>13</td><td>0.42</td><td>2.8</td><td>51</td><td>0.41</td><td>2.7</td></tr><tr><td>F</td><td>58</td><td>0.39</td><td>2.9</td><td>16</td><td>0.47</td><td>3.0</td><td>74</td><td>0.41</td><td>2.9</td></tr><tr><td>G</td><td>58</td><td>0.49</td><td>3.0</td><td>21</td><td>0.47</td><td>3.4</td><td>79</td><td>0.48</td><td>3.1</td></tr><tr><td>H</td><td>43</td><td>0.41</td><td>2.9</td><td>11</td><td>0.34</td><td>2.5</td><td>54</td><td>0.39</td><td>2.9</td></tr><tr><td>I</td><td>42</td><td>0.38</td><td>2.6</td><td>17</td><td>0.48</td><td>3.4</td><td>59</td><td>0.41</td><td>2.8</td></tr><tr><td>J</td><td>41</td><td>0.38</td><td>2.9</td><td>18</td><td>0.45</td><td>3.1</td><td>59</td><td>0.40</td><td>2.9</td></tr><tr><td>K</td><td>51</td><td>0.39</td><td>3.0</td><td>16</td><td>0.42</td><td>3.0</td><td>67</td><td>0.40</td><td>3.0</td></tr><tr><td>L</td><td>45</td><td>0.37</td><td>2.8</td><td>16</td><td>0.46</td><td>3.3</td><td>61</td><td>0.39</td><td>2.9</td></tr><tr><td>M</td><td>49</td><td>0.34</td><td>2.4</td><td>13</td><td>0.49</td><td>3.1</td><td>62</td><td>0.37</td><td>2.5</td></tr><tr><td>N</td><td>34</td><td>0.35</td><td>2.5</td><td>14</td><td>0.46</td><td>3.0</td><td>48</td><td>0.38</td><td>2.6</td></tr><tr><td>O</td><td>59</td><td>0.35</td><td>2.6</td><td>15</td><td>0.46</td><td>3.1</td><td>74</td><td>0.37</td><td>2.7</td></tr><tr><td>Total</td><td>934</td><td>0.38</td><td>2.8</td><td>304</td><td>0.46</td><td>3.1</td><td>1238</td><td>0.40</td><td>2.8</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Abbreviations are as in Table<a reveal-id="DSP010TB1" data-open="DSP010TB1" class="link link-reveal link-table xref-fig">1</a>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5776826" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP010TB4" class="table-wrap-title" data-id="DSP010TB4"><div class="label -label">Table4</div><div class="caption"><p>Average numbers of alleles and PIC values for mapped markers in each LG</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LG</th><th colspan="3">ES<sup>a</sup><hr /></th><th colspan="3">US<sup>a</sup><hr /></th><th colspan="3">Total<hr /></th></tr><tr><th></th><th>Number</th><th>Average PIC value</th><th>Average number of alleles</th><th>Number</th><th>Average PIC value</th><th>Average number of alleles</th><th>Number</th><th>Average PIC value</th><th>Average number of alleles</th></tr></thead><tbody><tr><td>A1</td><td>35</td><td>0.39</td><td>2.9</td><td>16</td><td>0.48</td><td>3.2</td><td>51</td><td>0.42</td><td>3.0</td></tr><tr><td>A2</td><td>56</td><td>0.39</td><td>2.7</td><td>15</td><td>0.47</td><td>3.1</td><td>71</td><td>0.41</td><td>2.8</td></tr><tr><td>B1</td><td>51</td><td>0.36</td><td>2.6</td><td>12</td><td>0.44</td><td>2.8</td><td>63</td><td>0.38</td><td>2.7</td></tr><tr><td>B2</td><td>40</td><td>0.43</td><td>2.8</td><td>13</td><td>0.50</td><td>3.2</td><td>53</td><td>0.45</td><td>2.9</td></tr><tr><td>C1</td><td>35</td><td>0.37</td><td>2.8</td><td>18</td><td>0.48</td><td>3.1</td><td>53</td><td>0.41</td><td>2.9</td></tr><tr><td>C2</td><td>50</td><td>0.36</td><td>2.8</td><td>20</td><td>0.48</td><td>3.4</td><td>70</td><td>0.40</td><td>2.9</td></tr><tr><td>D1a</td><td>38</td><td>0.42</td><td>2.9</td><td>13</td><td>0.45</td><td>2.8</td><td>51</td><td>0.42</td><td>2.9</td></tr><tr><td>D1b</td><td>58</td><td>0.37</td><td>2.7</td><td>13</td><td>0.47</td><td>3.3</td><td>71</td><td>0.39</td><td>2.8</td></tr><tr><td>D2</td><td>53</td><td>0.34</td><td>2.5</td><td>14</td><td>0.52</td><td>3.9</td><td>67</td><td>0.38</td><td>2.8</td></tr><tr><td>E</td><td>38</td><td>0.41</td><td>2.7</td><td>13</td><td>0.42</td><td>2.8</td><td>51</td><td>0.41</td><td>2.7</td></tr><tr><td>F</td><td>58</td><td>0.39</td><td>2.9</td><td>16</td><td>0.47</td><td>3.0</td><td>74</td><td>0.41</td><td>2.9</td></tr><tr><td>G</td><td>58</td><td>0.49</td><td>3.0</td><td>21</td><td>0.47</td><td>3.4</td><td>79</td><td>0.48</td><td>3.1</td></tr><tr><td>H</td><td>43</td><td>0.41</td><td>2.9</td><td>11</td><td>0.34</td><td>2.5</td><td>54</td><td>0.39</td><td>2.9</td></tr><tr><td>I</td><td>42</td><td>0.38</td><td>2.6</td><td>17</td><td>0.48</td><td>3.4</td><td>59</td><td>0.41</td><td>2.8</td></tr><tr><td>J</td><td>41</td><td>0.38</td><td>2.9</td><td>18</td><td>0.45</td><td>3.1</td><td>59</td><td>0.40</td><td>2.9</td></tr><tr><td>K</td><td>51</td><td>0.39</td><td>3.0</td><td>16</td><td>0.42</td><td>3.0</td><td>67</td><td>0.40</td><td>3.0</td></tr><tr><td>L</td><td>45</td><td>0.37</td><td>2.8</td><td>16</td><td>0.46</td><td>3.3</td><td>61</td><td>0.39</td><td>2.9</td></tr><tr><td>M</td><td>49</td><td>0.34</td><td>2.4</td><td>13</td><td>0.49</td><td>3.1</td><td>62</td><td>0.37</td><td>2.5</td></tr><tr><td>N</td><td>34</td><td>0.35</td><td>2.5</td><td>14</td><td>0.46</td><td>3.0</td><td>48</td><td>0.38</td><td>2.6</td></tr><tr><td>O</td><td>59</td><td>0.35</td><td>2.6</td><td>15</td><td>0.46</td><td>3.1</td><td>74</td><td>0.37</td><td>2.7</td></tr><tr><td>Total</td><td>934</td><td>0.38</td><td>2.8</td><td>304</td><td>0.46</td><td>3.1</td><td>1238</td><td>0.40</td><td>2.8</td></tr></tbody></table></div><div content-id="DSP010TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>LG</th><th colspan="3">ES<sup>a</sup><hr /></th><th colspan="3">US<sup>a</sup><hr /></th><th colspan="3">Total<hr /></th></tr><tr><th></th><th>Number</th><th>Average PIC value</th><th>Average number of alleles</th><th>Number</th><th>Average PIC value</th><th>Average number of alleles</th><th>Number</th><th>Average PIC value</th><th>Average number of alleles</th></tr></thead><tbody><tr><td>A1</td><td>35</td><td>0.39</td><td>2.9</td><td>16</td><td>0.48</td><td>3.2</td><td>51</td><td>0.42</td><td>3.0</td></tr><tr><td>A2</td><td>56</td><td>0.39</td><td>2.7</td><td>15</td><td>0.47</td><td>3.1</td><td>71</td><td>0.41</td><td>2.8</td></tr><tr><td>B1</td><td>51</td><td>0.36</td><td>2.6</td><td>12</td><td>0.44</td><td>2.8</td><td>63</td><td>0.38</td><td>2.7</td></tr><tr><td>B2</td><td>40</td><td>0.43</td><td>2.8</td><td>13</td><td>0.50</td><td>3.2</td><td>53</td><td>0.45</td><td>2.9</td></tr><tr><td>C1</td><td>35</td><td>0.37</td><td>2.8</td><td>18</td><td>0.48</td><td>3.1</td><td>53</td><td>0.41</td><td>2.9</td></tr><tr><td>C2</td><td>50</td><td>0.36</td><td>2.8</td><td>20</td><td>0.48</td><td>3.4</td><td>70</td><td>0.40</td><td>2.9</td></tr><tr><td>D1a</td><td>38</td><td>0.42</td><td>2.9</td><td>13</td><td>0.45</td><td>2.8</td><td>51</td><td>0.42</td><td>2.9</td></tr><tr><td>D1b</td><td>58</td><td>0.37</td><td>2.7</td><td>13</td><td>0.47</td><td>3.3</td><td>71</td><td>0.39</td><td>2.8</td></tr><tr><td>D2</td><td>53</td><td>0.34</td><td>2.5</td><td>14</td><td>0.52</td><td>3.9</td><td>67</td><td>0.38</td><td>2.8</td></tr><tr><td>E</td><td>38</td><td>0.41</td><td>2.7</td><td>13</td><td>0.42</td><td>2.8</td><td>51</td><td>0.41</td><td>2.7</td></tr><tr><td>F</td><td>58</td><td>0.39</td><td>2.9</td><td>16</td><td>0.47</td><td>3.0</td><td>74</td><td>0.41</td><td>2.9</td></tr><tr><td>G</td><td>58</td><td>0.49</td><td>3.0</td><td>21</td><td>0.47</td><td>3.4</td><td>79</td><td>0.48</td><td>3.1</td></tr><tr><td>H</td><td>43</td><td>0.41</td><td>2.9</td><td>11</td><td>0.34</td><td>2.5</td><td>54</td><td>0.39</td><td>2.9</td></tr><tr><td>I</td><td>42</td><td>0.38</td><td>2.6</td><td>17</td><td>0.48</td><td>3.4</td><td>59</td><td>0.41</td><td>2.8</td></tr><tr><td>J</td><td>41</td><td>0.38</td><td>2.9</td><td>18</td><td>0.45</td><td>3.1</td><td>59</td><td>0.40</td><td>2.9</td></tr><tr><td>K</td><td>51</td><td>0.39</td><td>3.0</td><td>16</td><td>0.42</td><td>3.0</td><td>67</td><td>0.40</td><td>3.0</td></tr><tr><td>L</td><td>45</td><td>0.37</td><td>2.8</td><td>16</td><td>0.46</td><td>3.3</td><td>61</td><td>0.39</td><td>2.9</td></tr><tr><td>M</td><td>49</td><td>0.34</td><td>2.4</td><td>13</td><td>0.49</td><td>3.1</td><td>62</td><td>0.37</td><td>2.5</td></tr><tr><td>N</td><td>34</td><td>0.35</td><td>2.5</td><td>14</td><td>0.46</td><td>3.0</td><td>48</td><td>0.38</td><td>2.6</td></tr><tr><td>O</td><td>59</td><td>0.35</td><td>2.6</td><td>15</td><td>0.46</td><td>3.1</td><td>74</td><td>0.37</td><td>2.7</td></tr><tr><td>Total</td><td>934</td><td>0.38</td><td>2.8</td><td>304</td><td>0.46</td><td>3.1</td><td>1238</td><td>0.40</td><td>2.8</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Abbreviations are as in Table<a reveal-id="DSP010TB1" data-open="DSP010TB1" class="link link-reveal link-table xref-fig">1</a>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5776826" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5776827 id="5776827" class="section-title" data-legacy-id=s3e>3.5. General discussion</h3>
<p>Several SSR-based linkage maps have recently been developed independently for different mapping populations.<sup><a reveal-id="DSP010C16" data-open="DSP010C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSP010C17" data-open="DSP010C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSP010C20" data-open="DSP010C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSP010C21" data-open="DSP010C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Although these previous studies accumulated almost 2000 SSRs mapped to the 20 consensus LGs, the positional relations among SSRs have remained unclear. We have mapped a total of 1810 SSR or other kinds of PCR-based markers in one or more than three mapping populations and have generated an integrated linkage map with the use of JoinMap 4.0 software. In addition to 1343 known SSR markers, 387 EST-derived SSR markers and 81 SSR or STS markers developed by the Chiba University were newly positioned on the map, allowing determination of the relative position for each microsatellite marker. The total length of the present integrated linkage map is 2442.9 cM, which is almost identical to that of an updated integrated map (2524.0 cM)<sup><a reveal-id="DSP010C16  DSP010C18" data-open="DSP010C16  DSP010C18" class="link link-ref link-reveal xref-bibr">1618</a></sup> including SSR, RFLP, RAPD, AFLP and classical markers as well as to that of a transcript map (2550.3 cM) with SNP markers.<sup><a reveal-id="DSP010C7" data-open="DSP010C7" class="link link-ref link-reveal xref-bibr">7</a></sup> The genetic length of individual LGs in the present integrated map correlates well with that of those in the previous maps (<em>r</em> = 0.73 and 0.72, respectively). Whereas LGs A1 and E are &gt;20 cM longer in the present map than in the previous maps, LGs B2, D2 and F are &gt;20 cM shorter. The estimated length of these LGs, with the exception of LG A1, agrees relatively well with that of those in the latest version of the integrated linkage map.<sup><a reveal-id="DSP010C32" data-open="DSP010C32" class="link link-ref link-reveal xref-bibr">32</a></sup> In contrast, several new SSR marker loci were mapped outside the existing SSR loci in LG A1, extending its genetic length by &gt;20 cM. The present integrated linkage map thus appears to achieve high coverage of the entire soybean genome.</p><p>The present integrated linkage map is composed of 1810 molecular markers including 1074 EST-derived SSR markers, 141 SSR or STS markers developed by the Chiba University and 595 public SSR markers with an average marker density of 0.74/cM. Although the markers are distributed relatively evenly, randomly and densely throughout the genome, there remain seven gaps of &gt;10 cM between markers. In particular, LG C1 contains the largest gap of 35.3 cM between Sat_235 and GMES1325, an interval for which no SSR markers were reported in previous studies.<sup><a reveal-id="DSP010C17" data-open="DSP010C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSP010C21" data-open="DSP010C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSP010C31" data-open="DSP010C31" class="link link-ref link-reveal xref-bibr">31</a></sup> This gap may reflect either a lack of polymorphic markers in a highly homozygous region or the presence of hot spots of recombination that enlarge the genetic distance corresponding to a short physical distance.</p><p>Public SSR markers, which have been developed by USDA-ARS and DuPont Corporation,<sup><a reveal-id="DSP010C17" data-open="DSP010C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSP010C31" data-open="DSP010C31" class="link link-ref link-reveal xref-bibr">31</a></sup> were strictly standardized to yield a single amplification product for a given soybean genotype. All the primer pairs for these markers thus corresponded to a single locus in the present mapping populations. In contrast, EST-derived SSR markers sometimes gave multiple bands, which were mapped separately to different LGs in some cases. Thirty-two and one primer pairs for these SSRs detected double and quadruple loci, respectively. A total of 84 public SSR markers were common to the three individual linkage maps and mapped to the 20 consensus LGs described previously.<sup><a reveal-id="DSP010C17" data-open="DSP010C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Most markers were arranged in the same order and with similar genetic intervals in the JF, PA, MM and present integrated linkage maps as well as in the consensus integrated linkage map. However, reversions or altered orders of markers relative to the consensus linkage map were observed for LGs D1a, D1b, E, J and N. Each RIL population used in this study was derived from a cross between a Japanese cultivar and foreign cultivar or experimental line. All the parents were genetically classified into distant groups by cluster analysis of length polymorphism of SSR markers.<sup><a reveal-id="DSP010C12" data-open="DSP010C12" class="link link-ref link-reveal xref-bibr">12</a></sup> In contrast, with one exception, previous consensus integrated linkage maps were constructed on the basis of mapping populations derived from US domestic cultivars.<sup><a reveal-id="DSP010C16" data-open="DSP010C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSP010C18" data-open="DSP010C18" class="link link-ref link-reveal xref-bibr">18</a></sup> The discrepancy in marker order may be due to inversion, translocation, insertion or deletion of genomic regions specific to mapping populations with different genetic backgrounds.<sup><a reveal-id="DSP010C7" data-open="DSP010C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Similarly, the frequency of recombination has been found to vary substantially among populations,<sup><a reveal-id="DSP010C33" data-open="DSP010C33" class="link link-ref link-reveal xref-bibr">33</a></sup> resulting in divergence of genetic distance among mapping populations.</p><p>In the present study, we analyzed allelic variation for a large number of SSR loci including 934 EST-derived and 304 public SSRs. The overall average PIC value for the EST-derived markers (0.38) was lower than that for the public SSR markers (0.46), consistent with previous observations.<sup><a reveal-id="DSP010C21" data-open="DSP010C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSP010C30" data-open="DSP010C30" class="link link-ref link-reveal xref-bibr">30</a></sup> The PIC values and their distribution are indicative of the transferability of mapped SSR markers to different genotypes. The most frequent number of alleles per EST-derived marker was two, suggesting that several such markers might reflect insertiondeletion polymorphisms (indels), as is the case for STS markers.<sup><a reveal-id="DSP010C21" data-open="DSP010C21" class="link link-ref link-reveal xref-bibr">21</a></sup> The PIC values of EST-derived markers in individual LGs did not correlate with those of the public SSR markers (<em>r</em> = 0.23). The public SSR marker loci with the highest and lowest average PIC values were found in LG D2 (0.52) and LG H (0.34), respectively, whereas the highest and lowest average PIC values for the EST-derived SSR markers were detected in LG G (0.41) and both LG D2 and LG M (0.34), respectively. The reason for this difference between the two SSR marker groups of different origin remains unclear.</p><p>Soybean is a member of the phaseoloid legumes, which include numerous crops of economic importance such as common bean (<em>Phaseolus vulgaris</em> L.), cowpea [<em>Vigna unguiculata</em> (L.) Walp.], mungbean [<em>V. radiata</em> (L.) Wilczek] and azuki bean [<em>V. angularis</em> (Willd.) Ohwi et Ohashi]. The availability of an integrated linkage map with markers, such as EST-derived SSR markers, that are transferable to closely related species would allow comparative genetic analysis between soybean and other legume crops as a strategy for applied agriculture.<sup><a reveal-id="DSP010C21" data-open="DSP010C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSP010C34" data-open="DSP010C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Comparative genetic mapping and synteny analysis would facilitate gene discovery through the development of markers that are closely linked genetically to the target gene.<sup><a reveal-id="DSP010C35" data-open="DSP010C35" class="link link-ref link-reveal xref-bibr">35</a></sup> In addition, information on EST-derived SSR markers provides a basis for evaluation of the structural diversity and evolution of these various species.<sup><a reveal-id="DSP010C36" data-open="DSP010C36" class="link link-ref link-reveal xref-bibr">36</a></sup></p><p>A chromosome-based assembly of sequence data in soybean, Glyma1.0, is available at Phytozome.<sup><a reveal-id="DSP010C37" data-open="DSP010C37" class="link link-ref link-reveal xref-bibr">37</a></sup> The accuracy and resolution of genetic linkage maps and physical maps (whole-genome sequencing) can thus be confirmed and refined reciprocally. The availability of physical maps does not diminish the importance of genetic linkage maps, with the two map types having a synergistic influence on modern soybean genomics as well as marker-assisted selection for target traits. The large number of SSR markers positioned in the present and previous studies promote understanding of the soybean genome at the structural and functional levels.</p>                            <h2 scrollto-destination=5776834 id="5776834" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available online at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5776836 id="5776836" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by the Ministry of Agriculture, Forestry, and Fisheries of Japan [Genomics for Agricultural Innovation (DD-3260) and Research Project for Utilizing Advanced Technologies in Agriculture, Forestry, and Fisheries (18063)].</p>                            <h2 scrollto-destination=5776838 id="5776838" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Asuka Ueki, Motomi Suzuki, Takako Yamamoto and Ayako Suzuki for technical assistance.</p>                            <h2 scrollto-destination=5776840 id="5776840" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSP010C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="collab">FAS</div>,<div class="source">Oilseeds: World Markets and Trade. Foreign Agricultural Service (FAS)</div>,<div class="year">2008</div><div class="publisher-loc">Washington, DC</div><div class="publisher-name">United States Department of Agriculture</div></div></div></div></div><div content-id="DSP010C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Palmer</div> <div class="given-names">R. G.</div></div>, <div class="name"><div class="surname">Kilen</div> <div class="given-names">T. C</div></div>.</div><div class="person-group"><div class="name"><div class="surname">Wilcox</div> <div class="given-names">J. R.</div></div>.</div><div class="article-title">Qualitative genetics and cytogenetics</div>,<div class="source">Soybeans: Improvement, Production, and Uses</div>,<div class="year">1987</div><div class="edition">2nd Ed.</div><div class="publisher-loc">Madison, Wisconsin</div><div class="publisher-name">Agronomy</div>(pg.<div class="fpage">135</div>-<div class="lpage">209</div>)</div></div></div></div><div content-id="DSP010C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Keim</div> <div class="given-names">P.</div></div>, <div class="name"><div class="surname">Diers</div> <div class="given-names">B. W.</div></div>, <div class="name"><div class="surname">Olson</div> <div class="given-names">T. C.</div></div>, <div class="name"><div class="surname">Shoemaker</div> <div class="given-names">R. C.</div></div>.</div><div class="article-title">RELP mapping in soybean: association between marker loci and variation in quantitative traits</div>,<div class="source">Genetics</div>,<div class="year">1990</div>,vol.<div class="volume">126</div>(pg.<div class="fpage">735</div>-<div class="lpage">742</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RELP%20mapping%20in%20soybean%3A%20association%20between%20marker%20loci%20and%20variation%20in%20quantitative%20traits&amp;author=P.%20Keim&amp;author=B.%20W.%20Diers&amp;author=T.%20C.%20Olson&amp;author=R.%20C.%20Shoemaker&amp;publication_year=1990&amp;journal=Genetics&amp;volume=126&amp;pages=735-742" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1979039" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSP010C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Williams</div> <div class="given-names">J. G. K.</div></div>, <div class="name"><div class="surname">Kubelik</div> <div class="given-names">A. R.</div></div>, <div class="name"><div class="surname">Livak</div> <div class="given-names">K. J.</div></div>, <div class="name"><div class="surname">Rafalski</div> <div class="given-names">J. A.</div></div>, <div class="name"><div class="surname">Tingey</div> <div class="given-names">S. V</div></div>.</div><div class="article-title">DNA polymorphisms amplified by arbitrary primers are useful as genetic markers</div>,<div class="source">Nucleic Acids Res.</div>,<div class="year">1990</div>,vol.<div class="volume">18</div>(pg.<div class="fpage">6531</div>-<div class="lpage">6535</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20polymorphisms%20amplified%20by%20arbitrary%20primers%20are%20useful%20as%20genetic%20markers&amp;author=J.%20G.%20K.%20Williams&amp;author=A.%20R.%20Kubelik&amp;author=K.%20J.%20Livak&amp;author=J.%20A.%20Rafalski&amp;author=S.%20V%20Tingey&amp;publication_year=1990&amp;journal=Nucleic%20Acids%20Res.&amp;volume=18&amp;pages=6531-6535" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/18.22.6531" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F18.22.6531" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F18.22.6531"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1979162" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSP010C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Akkaya</div> <div class="given-names">M. S.</div></div>, <div class="name"><div class="surname">Bhagwat</div> <div class="given-names">A. A.</div></div>, <div class="name"><div class="surname">Cregan</div> <div class="given-names">P. B.</div></div>.</div><div class="article-title">Length polymorphisms of simple sequence repeat DNA in soybean</div>,<div class="source">Genetics</div>,<div class="year">1992</div>,vol.<div class="volume">132</div>(pg.<div class="fpage">1131</div>-<div class="lpage">1139</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Length%20polymorphisms%20of%20simple%20sequence%20repeat%20DNA%20in%20soybean&amp;author=M.%20S.%20Akkaya&amp;author=A.%20A.%20Bhagwat&amp;author=P.%20B.%20Cregan&amp;publication_year=1992&amp;journal=Genetics&amp;volume=132&amp;pages=1131-1139" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1459432" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSP010C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Akkaya</div> <div class="given-names">M. S.</div></div>, <div class="name"><div class="surname">Shoemaker</div> <div class="given-names">R. C.</div></div>, <div class="name"><div class="surname">Specht</div> <div class="given-names">J. E.</div></div>, <div class="name"><div class="surname">Bhagwat</div> <div class="given-names">A. A.</div></div>, <div class="name"><div class="surname">Cregan</div> <div class="given-names">P. B.</div></div>.</div><div class="article-title">Integration of simple sequence repeat DNA markers into a soybean linkage map</div>,<div class="source">Crop Sci.</div>,<div class="year">1995</div>,vol.<div class="volume">35</div>(pg.<div class="fpage">1439</div>-<div class="lpage">1445</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integration%20of%20simple%20sequence%20repeat%20DNA%20markers%20into%20a%20soybean%20linkage%20map&amp;author=M.%20S.%20Akkaya&amp;author=R.%20C.%20Shoemaker&amp;author=J.%20E.%20Specht&amp;author=A.%20A.%20Bhagwat&amp;author=P.%20B.%20Cregan&amp;publication_year=1995&amp;journal=Crop%20Sci.&amp;volume=35&amp;pages=1439-1445" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci1995.0011183X003500050030x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci1995.0011183X003500050030x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci1995.0011183X003500050030x"> </span></div></div></div></div></div></div><div content-id="DSP010C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Choi</div> <div class="given-names">I. Y.</div></div>, <div class="name"><div class="surname">Hyten</div> <div class="given-names">D. L.</div></div>, <div class="name"><div class="surname">Matukumalli</div> <div class="given-names">L. K.</div></div>,et al.</div><div class="article-title">A soybean transcript map: gene distribution, haplotype and single-nucleotide polymorphism analysis</div>,<div class="source">Genetics</div>,<div class="year">2007</div>,vol.<div class="volume">176</div>(pg.<div class="fpage">685</div>-<div class="lpage">696</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20soybean%20transcript%20map%3A%20gene%20distribution%2C%20haplotype%20and%20single-nucleotide%20polymorphism%20analysis&amp;author=I.%20Y.%20Choi&amp;author=D.%20L.%20Hyten&amp;author=L.%20K.%20Matukumalli&amp;publication_year=2007&amp;journal=Genetics&amp;volume=176&amp;pages=685-696" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.107.070821" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.107.070821" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.107.070821"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17339218" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSP010C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morgante</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Olivieri</div> <div class="given-names">A. M.</div></div>.</div><div class="article-title">PCR-amplified microsatellites as markers in plant genetics</div>,<div class="source">Plant J.</div>,<div class="year">1993</div>,vol.<div class="volume">3</div>(pg.<div class="fpage">715</div>-<div class="lpage">723</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PCR-amplified%20microsatellites%20as%20markers%20in%20plant%20genetics&amp;author=M.%20Morgante&amp;author=A.%20M.%20Olivieri&amp;publication_year=1993&amp;journal=Plant%20J.&amp;volume=3&amp;pages=715-723" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-313X.1993.tb00020.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.1993.tb00020.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-313X.1993.tb00020.x"> </span></div></div></div></div></div></div><div content-id="DSP010C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Maughan</div> <div class="given-names">P. J.</div></div>, <div class="name"><div class="surname">Saghi Maroof</div> <div class="given-names">M. A.</div></div>, <div class="name"><div class="surname">Buss</div> <div class="given-names">G. R</div></div>.</div><div class="article-title">Microsatellite and amplified sequence length polymorphisms in cultivated and wild soybean</div>,<div class="source">Genome</div>,<div class="year">1995</div>,vol.<div class="volume">38</div>(pg.<div class="fpage">715</div>-<div class="lpage">723</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellite%20and%20amplified%20sequence%20length%20polymorphisms%20in%20cultivated%20and%20wild%20soybean&amp;author=P.%20J.%20Maughan&amp;author=M.%20A.%20Saghi%20Maroof&amp;author=G.%20R%20Buss&amp;publication_year=1995&amp;journal=Genome&amp;volume=38&amp;pages=715-723" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g95-090" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg95-090" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg95-090"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7672605" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSP010C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Diwan</div> <div class="given-names">N.</div></div>, <div class="name"><div class="surname">Cregan</div> <div class="given-names">P. B.</div></div>.</div><div class="article-title">Automated sizing of fluorescent-labeled simple sequence repeat (SSR) markers to assay genetic variation in soybean</div>,<div class="source">Theor. Appl. Genet.</div>,<div class="year">1997</div>,vol.<div class="volume">95</div>(pg.<div class="fpage">723</div>-<div class="lpage">733</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20sizing%20of%20fluorescent-labeled%20simple%20sequence%20repeat%20(SSR)%20markers%20to%20assay%20genetic%20variation%20in%20soybean&amp;author=N.%20Diwan&amp;author=P.%20B.%20Cregan&amp;publication_year=1997&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=95&amp;pages=723-733" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220050618" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220050618" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220050618"> </span></div></div></div></div></div></div><div content-id="DSP010C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">Y.</div></div>, <div class="name"><div class="surname">Wang</div> <div class="given-names">D.</div></div>, <div class="name"><div class="surname">Arelli</div> <div class="given-names">P.</div></div>, <div class="name"><div class="surname">Ebrahimi</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Lelson</div> <div class="given-names">R. L.</div></div>.</div><div class="article-title">Molecular marker diversity of SCN-resistant sources in soybean</div>,<div class="source">Genome</div>,<div class="year">2006</div>,vol.<div class="volume">49</div>(pg.<div class="fpage">938</div>-<div class="lpage">949</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20marker%20diversity%20of%20SCN-resistant%20sources%20in%20soybean&amp;author=Y.%20Chen&amp;author=D.%20Wang&amp;author=P.%20Arelli&amp;author=M.%20Ebrahimi&amp;author=R.%20L.%20Lelson&amp;publication_year=2006&amp;journal=Genome&amp;volume=49&amp;pages=938-949" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G06-057" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG06-057" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG06-057"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17036069" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSP010C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hwang</div> <div class="given-names">T. Y.</div></div>, <div class="name"><div class="surname">Nakamoto</div> <div class="given-names">Y.</div></div>, <div class="name"><div class="surname">Kono</div> <div class="given-names">I.</div></div>,et al.</div><div class="article-title">Genetic diversity of cultivated and wild soybeans including Japanese elite cultivars as revealed by length polymorphism of SSR markers</div>,<div class="source">Breed. Sci.</div>,<div class="year">2008</div>,vol.<div class="volume">58</div>(pg.<div class="fpage">315</div>-<div class="lpage">323</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20of%20cultivated%20and%20wild%20soybeans%20including%20Japanese%20elite%20cultivars%20as%20revealed%20by%20length%20polymorphism%20of%20SSR%20markers&amp;author=T.%20Y.%20Hwang&amp;author=Y.%20Nakamoto&amp;author=I.%20Kono&amp;publication_year=2008&amp;journal=Breed.%20Sci.&amp;volume=58&amp;pages=315-323" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.58.315" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.58.315" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.58.315"> </span></div></div></div></div></div></div><div content-id="DSP010C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cregan</div> <div class="given-names">P. B.</div></div>, <div class="name"><div class="surname">Bhagwat</div> <div class="given-names">A. A.</div></div>, <div class="name"><div class="surname">Akkaya</div> <div class="given-names">M. S.</div></div>, <div class="name"><div class="surname">Rongwen</div> <div class="given-names">J</div></div>.</div><div class="article-title">Microsatellite fingerprinting and mapping of soybean</div>,<div class="source">Methods Mol. Cell. Biol.</div>,<div class="year">1994</div>,vol.<div class="volume">5</div>(pg.<div class="fpage">49</div>-<div class="lpage">61</div>)</div></div></div></div><div content-id="DSP010C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">Y. G.</div></div>, <div class="name"><div class="surname">Saghai Maroof</div> <div class="given-names">M. A.</div></div>, <div class="name"><div class="surname">Buss</div> <div class="given-names">G. R.</div></div>, <div class="name"><div class="surname">Maughan</div> <div class="given-names">P. J.</div></div>, <div class="name"><div class="surname">Tolin</div> <div class="given-names">S. A.</div></div>.</div><div class="article-title">RFLP and microsatellite mapping of a gene for soybean mosaic virus resistance</div>,<div class="source">Phytopathology</div>,<div class="year">1994</div>,vol.<div class="volume">84</div>(pg.<div class="fpage">60</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RFLP%20and%20microsatellite%20mapping%20of%20a%20gene%20for%20soybean%20mosaic%20virus%20resistance&amp;author=Y.%20G.%20Yu&amp;author=M.%20A.%20Saghai%20Maroof&amp;author=G.%20R.%20Buss&amp;author=P.%20J.%20Maughan&amp;author=S.%20A.%20Tolin&amp;publication_year=1994&amp;journal=Phytopathology&amp;volume=84&amp;pages=60-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/Phyto-84-60" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FPhyto-84-60" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FPhyto-84-60"> </span></div></div></div></div></div></div><div content-id="DSP010C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tang</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Kishore</div> <div class="given-names">V. K.</div></div>, <div class="name"><div class="surname">Knapp</div> <div class="given-names">S. J</div></div>.</div><div class="article-title">PCR-multiplexes for a genome-wide framework of simple sequence repeat marker loci in cultivated sunflower</div>,<div class="source">Theor. Appl. Genet.</div>,<div class="year">2003</div>,vol.<div class="volume">107</div>(pg.<div class="fpage">6</div>-<div class="lpage">19</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PCR-multiplexes%20for%20a%20genome-wide%20framework%20of%20simple%20sequence%20repeat%20marker%20loci%20in%20cultivated%20sunflower&amp;author=S.%20Tang&amp;author=V.%20K.%20Kishore&amp;author=S.%20J%20Knapp&amp;publication_year=2003&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=107&amp;pages=6-19" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12835928" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSP010C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cregan</div> <div class="given-names">P. B.</div></div>, <div class="name"><div class="surname">Jarvik</div> <div class="given-names">T.</div></div>, <div class="name"><div class="surname">Bush</div> <div class="given-names">A. L.</div></div>,et al.</div><div class="article-title">An integrated genetic linkage map of the soybean genome</div>,<div class="source">Crop Sci.</div>,<div class="year">1999</div>,vol.<div class="volume">39</div>(pg.<div class="fpage">1464</div>-<div class="lpage">1490</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20genetic%20linkage%20map%20of%20the%20soybean%20genome&amp;author=P.%20B.%20Cregan&amp;author=T.%20Jarvik&amp;author=A.%20L.%20Bush&amp;publication_year=1999&amp;journal=Crop%20Sci.&amp;volume=39&amp;pages=1464-1490" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci1999.3951464x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci1999.3951464x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci1999.3951464x"> </span></div></div></div></div></div></div><div content-id="DSP010C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cregan</div> <div class="given-names">P. B</div></div>.</div><div class="article-title">An integrated genetic linkage map of the soybean July 2003</div>,<div class="year">2003</div><div class="comment"><a class="link link-uri" href="http://bldg6.arsusda.gov/~pooley/soy/cregan/soymap.htm">http://bldg6.arsusda.gov/~pooley/soy/cregan/soymap.htm</a> (July 2003)</div></div></div></div></div><div content-id="DSP010C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Song</div> <div class="given-names">Q. J.</div></div>, <div class="name"><div class="surname">Marek</div> <div class="given-names">L. F.</div></div>, <div class="name"><div class="surname">Shoemaker</div> <div class="given-names">R. C.</div></div>,et al.</div><div class="article-title">A new integrated genetic linkage map of the soybean</div>,<div class="source">Theor. Appl. Genet.</div>,<div class="year">2004</div>,vol.<div class="volume">109</div>(pg.<div class="fpage">122</div>-<div class="lpage">128</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20integrated%20genetic%20linkage%20map%20of%20the%20soybean&amp;author=Q.%20J.%20Song&amp;author=L.%20F.%20Marek&amp;author=R.%20C.%20Shoemaker&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=109&amp;pages=122-128" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-004-1602-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-004-1602-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-004-1602-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14991109" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSP010C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Arumaganthan</div> <div class="given-names">K.</div></div>, <div class="name"><div class="surname">Earle</div> <div class="given-names">E</div></div>.</div><div class="article-title">Nuclear DNA content of some important plant species</div>,<div class="source">Plant Mol. Biol. Rep.</div>,<div class="year">1991</div>,vol.<div class="volume">9</div>(pg.<div class="fpage">208</div>-<div class="lpage">218</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nuclear%20DNA%20content%20of%20some%20important%20plant%20species&amp;author=K.%20Arumaganthan&amp;author=E%20Earle&amp;publication_year=1991&amp;journal=Plant%20Mol.%20Biol.%20Rep.&amp;volume=9&amp;pages=208-218" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02672069" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02672069" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02672069"> </span></div></div></div></div></div></div><div content-id="DSP010C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xia</div> <div class="given-names">Z.</div></div>, <div class="name"><div class="surname">Tsubokura</div> <div class="given-names">Y.</div></div>, <div class="name"><div class="surname">Hoshi</div> <div class="given-names">M.</div></div>,et al.</div><div class="article-title">An integrated high-density linkage map of soybean with RFLP, SSR, STS, and AFLP markers using a single F<sub>2</sub> population</div>,<div class="source">DNA Res.</div>,<div class="year">2007</div>,vol.<div class="volume">14</div>(pg.<div class="fpage">257</div>-<div class="lpage">269</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20high-density%20linkage%20map%20of%20soybean%20with%20RFLP%2C%20SSR%2C%20STS%2C%20and%20AFLP%20markers%20using%20a%20single%20F2%20population&amp;author=Z.%20Xia&amp;author=Y.%20Tsubokura&amp;author=M.%20Hoshi&amp;publication_year=2007&amp;journal=DNA%20Res.&amp;volume=14&amp;pages=257-269" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsm027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsm027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsm027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18192280" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSP010C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hisano</div> <div class="given-names">H.</div></div>, <div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Isobe</div> <div class="given-names">S.</div></div>,et al.</div><div class="article-title">Characterization of the soybean genome using EST-derived microsatellite markers</div>,<div class="source">DNA Res.</div>,<div class="year">2007</div>,vol.<div class="volume">14</div>(pg.<div class="fpage">271</div>-<div class="lpage">281</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20the%20soybean%20genome%20using%20EST-derived%20microsatellite%20markers&amp;author=H.%20Hisano&amp;author=S.%20Sato&amp;author=S.%20Isobe&amp;publication_year=2007&amp;journal=DNA%20Res.&amp;volume=14&amp;pages=271-281" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsm025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsm025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsm025"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18192281" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSP010C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamanaka</div> <div class="given-names">N.</div></div>, <div class="name"><div class="surname">Ninomiya</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Hoshi</div> <div class="given-names">M.</div></div>,et al.</div><div class="article-title">An informative linkage map of soybean reveals QTLs for flowering time, leaflet morphology and regions of segregation distortion</div>,<div class="source">DNA Res.</div>,<div class="year">2001</div>,vol.<div class="volume">8</div>(pg.<div class="fpage">61</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20informative%20linkage%20map%20of%20soybean%20reveals%20QTLs%20for%20flowering%20time%2C%20leaflet%20morphology%20and%20regions%20of%20segregation%20distortion&amp;author=N.%20Yamanaka&amp;author=S.%20Ninomiya&amp;author=M.%20Hoshi&amp;publication_year=2001&amp;journal=DNA%20Res.&amp;volume=8&amp;pages=61-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/8.2.61" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F8.2.61" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F8.2.61"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11347903" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSP010C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Watanabe</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Tajuddin</div> <div class="given-names">T.</div></div>, <div class="name"><div class="surname">Yamanaka</div> <div class="given-names">N.</div></div>, <div class="name"><div class="surname">Hayashi</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Harada</div> <div class="given-names">K</div></div>.</div><div class="article-title">Analysis of QTLs for reproductive development and seed quality traits in soybean using recombinant inbred lines</div>,<div class="source">Breed. Sci.</div>,<div class="year">2004</div>,vol.<div class="volume">54</div>(pg.<div class="fpage">399</div>-<div class="lpage">407</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20QTLs%20for%20reproductive%20development%20and%20seed%20quality%20traits%20in%20soybean%20using%20recombinant%20inbred%20lines&amp;author=S.%20Watanabe&amp;author=T.%20Tajuddin&amp;author=N.%20Yamanaka&amp;author=M.%20Hayashi&amp;author=K%20Harada&amp;publication_year=2004&amp;journal=Breed.%20Sci.&amp;volume=54&amp;pages=399-407" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.54.399" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.54.399" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.54.399"> </span></div></div></div></div></div></div><div content-id="DSP010C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Isobe</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Asamizu</div> <div class="given-names">E.</div></div>,et al.</div><div class="article-title">Comprehensive structural analysis of the genome of red clover (<em>Trifolium pratense</em> L.)</div>,<div class="source">DNA Res.</div>,<div class="year">2005</div>,vol.<div class="volume">12</div>(pg.<div class="fpage">301</div>-<div class="lpage">364</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20structural%20analysis%20of%20the%20genome%20of%20red%20clover%20(Trifolium%20pratense%20L.)&amp;author=S.%20Sato&amp;author=S.%20Isobe&amp;author=E.%20Asamizu&amp;publication_year=2005&amp;journal=DNA%20Res.&amp;volume=12&amp;pages=301-364" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsi018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsi018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsi018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16769692" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSP010C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Van Ooigen</div> <div class="given-names">J. W</div></div>.</div>,<div class="source">JoinMap 4.0 Software for the Calculation of Genetic Linkage Maps in Experimental Populations</div>,<div class="year">2006</div><div class="publisher-loc">Wageningen, the Netherlands</div><div class="publisher-name">Plant Research International</div></div></div></div></div><div content-id="DSP010C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lincoln</div> <div class="given-names">S. E.</div></div>, <div class="name"><div class="surname">Daly</div> <div class="given-names">M. J.</div></div>, <div class="name"><div class="surname">Lander</div> <div class="given-names">E. S</div></div>.</div>,<div class="source">Constructing Genetic Maps with MAPMAKER/EXP 3.0, Whitehead Institute Technical Report</div>,<div class="year">1992</div><div class="edition">3rd Ed.</div><div class="publisher-loc">Massachusetts, Cambridge</div><div class="publisher-name">Whitehead Institute</div></div></div></div></div><div content-id="DSP010C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kosambi</div> <div class="given-names">D. D.</div></div>.</div><div class="article-title">The estimation of map distance from recombination values</div>,<div class="source">Ann. Eugen.</div>,<div class="year">1944</div>,vol.<div class="volume">12</div>(pg.<div class="fpage">172</div>-<div class="lpage">175</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20estimation%20of%20map%20distance%20from%20recombination%20values&amp;author=D.%20D.%20Kosambi&amp;publication_year=1944&amp;journal=Ann.%20Eugen.&amp;volume=12&amp;pages=172-175" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-1809.1943.tb02321.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-1809.1943.tb02321.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-1809.1943.tb02321.x"> </span></div></div></div></div></div></div><div content-id="DSP010C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Voorrips</div> <div class="given-names">R. E.</div></div>.</div><div class="article-title">MapChart: software for the graphical presentation of linkage maps and QTLs</div>,<div class="source">J. Hered.</div>,<div class="year">2002</div>,vol.<div class="volume">93</div>(pg.<div class="fpage">77</div>-<div class="lpage">78</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MapChart%3A%20software%20for%20the%20graphical%20presentation%20of%20linkage%20maps%20and%20QTLs&amp;author=R.%20E.%20Voorrips&amp;publication_year=2002&amp;journal=J.%20Hered.&amp;volume=93&amp;pages=77-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/93.1.77" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2F93.1.77" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2F93.1.77"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12011185" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSP010C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Keim</div> <div class="given-names">P.</div></div>, <div class="name"><div class="surname">Beavis</div> <div class="given-names">W.</div></div>, <div class="name"><div class="surname">Schupp</div> <div class="given-names">J.</div></div>, <div class="name"><div class="surname">Freestone</div> <div class="given-names">R.</div></div>.</div><div class="article-title">Evaluation of soybean RFLP marker diversity in adapted germ plasm</div>,<div class="source">Theor. Appl. Genet.</div>,<div class="year">1992</div>,vol.<div class="volume">85</div>(pg.<div class="fpage">205</div>-<div class="lpage">212</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evaluation%20of%20soybean%20RFLP%20marker%20diversity%20in%20adapted%20germ%20plasm&amp;author=P.%20Keim&amp;author=W.%20Beavis&amp;author=J.%20Schupp&amp;author=R.%20Freestone&amp;publication_year=1992&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=85&amp;pages=205-212" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24197306" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSP010C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hossain</div> <div class="given-names">K. G.</div></div>, <div class="name"><div class="surname">Kawai</div> <div class="given-names">H.</div></div>, <div class="name"><div class="surname">Hayashi</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Hoshi</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Yamanaka</div> <div class="given-names">N.</div></div>, <div class="name"><div class="surname">Harada</div> <div class="given-names">K</div></div>.</div><div class="article-title">Characterization and identification of (CT)n microsatellites in soybean using sheared genomic libraries</div>,<div class="source">DNA Res.</div>,<div class="year">2000</div>,vol.<div class="volume">7</div>(pg.<div class="fpage">103</div>-<div class="lpage">110</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20and%20identification%20of%20(CT)n%20microsatellites%20in%20soybean%20using%20sheared%20genomic%20libraries&amp;author=K.%20G.%20Hossain&amp;author=H.%20Kawai&amp;author=M.%20Hayashi&amp;author=M.%20Hoshi&amp;author=N.%20Yamanaka&amp;author=K%20Harada&amp;publication_year=2000&amp;journal=DNA%20Res.&amp;volume=7&amp;pages=103-110" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/7.2.103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F7.2.103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F7.2.103"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10819325" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSP010C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cregan</div> <div class="given-names">P. B.</div></div>, <div class="name"><div class="surname">Mudge</div> <div class="given-names">J.</div></div>, <div class="name"><div class="surname">Fickus</div> <div class="given-names">E. W.</div></div>, <div class="name"><div class="surname">Danesh</div> <div class="given-names">D.</div></div>, <div class="name"><div class="surname">Denny</div> <div class="given-names">R.</div></div>, <div class="name"><div class="surname">Young</div> <div class="given-names">N. D.</div></div>.</div><div class="article-title">Two simple sequence repeat markers to select for soybean cyst nematode resistance conditioned by rhg1 locus</div>,<div class="source">Theor. Appl. Genet.</div>,<div class="year">1999</div>,vol.<div class="volume">99</div>(pg.<div class="fpage">811</div>-<div class="lpage">818</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20simple%20sequence%20repeat%20markers%20to%20select%20for%20soybean%20cyst%20nematode%20resistance%20conditioned%20by%20rhg1%20locus&amp;author=P.%20B.%20Cregan&amp;author=J.%20Mudge&amp;author=E.%20W.%20Fickus&amp;author=D.%20Danesh&amp;author=R.%20Denny&amp;author=N.%20D.%20Young&amp;publication_year=1999&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=99&amp;pages=811-818" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220051300" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220051300" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220051300"> </span></div></div></div></div></div></div><div content-id="DSP010C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="collab">SoyBase</div><div class="article-title">Soybean chromosome number to linkage group assignments</div>,<div class="year">2008</div><div class="comment"><a class="link link-uri" href="http://soybase.org/LG2Xsome.php">http://soybase.org/LG2Xsome.php</a> (30 July 2008)</div></div></div></div></div><div content-id="DSP010C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pfeiffer</div> <div class="given-names">T. W.</div></div>, <div class="name"><div class="surname">Vogt</div> <div class="given-names">S. D.</div></div>.</div><div class="article-title">Variability for recombination frequencies in the AP12 soybean population</div>,<div class="source">Crop Sci.</div>,<div class="year">1990</div>,vol.<div class="volume">30</div>(pg.<div class="fpage">545</div>-<div class="lpage">549</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variability%20for%20recombination%20frequencies%20in%20the%20AP12%20soybean%20population&amp;author=T.%20W.%20Pfeiffer&amp;author=S.%20D.%20Vogt&amp;publication_year=1990&amp;journal=Crop%20Sci.&amp;volume=30&amp;pages=545-549" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci1990.0011183X003000030014x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci1990.0011183X003000030014x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci1990.0011183X003000030014x"> </span></div></div></div></div></div></div><div content-id="DSP010C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bushman</div> <div class="given-names">B. S.</div></div>, <div class="name"><div class="surname">Larson</div> <div class="given-names">S. R.</div></div>, <div class="name"><div class="surname">Mott</div> <div class="given-names">I. W.</div></div>,et al.</div><div class="article-title">Development and annotation of perennial Triticeae ESTs and SSR markers</div>,<div class="source">Genome</div>,<div class="year">2008</div>,vol.<div class="volume">51</div>(pg.<div class="fpage">779</div>-<div class="lpage">788</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20annotation%20of%20perennial%20Triticeae%20ESTs%20and%20SSR%20markers&amp;author=B.%20S.%20Bushman&amp;author=S.%20R.%20Larson&amp;author=I.%20W.%20Mott&amp;publication_year=2008&amp;journal=Genome&amp;volume=51&amp;pages=779-788" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G08-062" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG08-062" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG08-062"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18923529" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSP010C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hwang</div> <div class="given-names">T. Y.</div></div>, <div class="name"><div class="surname">Moon</div> <div class="given-names">J. K.</div></div>, <div class="name"><div class="surname">Yu</div> <div class="given-names">S.</div></div>,et al.</div><div class="article-title">Application of comparative genomics in developing molecular markers tightly linked to the virus resistance gene Rsv4 in soybean</div>,<div class="source">Genome</div>,<div class="year">2006</div>,vol.<div class="volume">49</div>(pg.<div class="fpage">380</div>-<div class="lpage">388</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Application%20of%20comparative%20genomics%20in%20developing%20molecular%20markers%20tightly%20linked%20to%20the%20virus%20resistance%20gene%20Rsv4%20in%20soybean&amp;author=T.%20Y.%20Hwang&amp;author=J.%20K.%20Moon&amp;author=S.%20Yu&amp;publication_year=2006&amp;journal=Genome&amp;volume=49&amp;pages=380-388" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G05-111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG05-111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG05-111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16699558" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSP010C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Choi</div> <div class="given-names">H. K.</div></div>, <div class="name"><div class="surname">Mun</div> <div class="given-names">J. H.</div></div>, <div class="name"><div class="surname">Kim</div> <div class="given-names">D. J.</div></div>,et al.</div><div class="article-title">Estimating genome conservation between crop and model legume species</div>,<div class="source">Proc. Natl Acad. Sci. USA</div>,<div class="year">2004</div>,vol.<div class="volume">101</div>(pg.<div class="fpage">15289</div>-<div class="lpage">15294</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimating%20genome%20conservation%20between%20crop%20and%20model%20legume%20species&amp;author=H.%20K.%20Choi&amp;author=J.%20H.%20Mun&amp;author=D.%20J.%20Kim&amp;publication_year=2004&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=15289-15294" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0402251101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0402251101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0402251101"> </span></div></div></div></div></div></div><div content-id="DSP010C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP010C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="collab">The Department of Energy's Joint Genome Institute and the Center for Integrative Genomics</div><div class="article-title">Phytozome Glyma 1</div>,<div class="year">2008</div><div class="comment"><a class="link link-uri" href="http://www.phytozome.net/soybean.php">http://www.phytozome.net/soybean.php</a> (8 December 2008)</div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Masahiro yano</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement"> The Author 2009. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=391702&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/4/10.1093/dnares/dsp010/2/dsp010-dsp010supp.xls?Expires=2147483647&amp;Signature=0MkSPgaB9XbPNdyg7x7siaCCWn9J315J2DGC6gnfIqzCMb7KXehJowE7-mGgCMcvMmCU1Ktk3xxKRf1YqTyaeiyl2lLePyq7b-dkZ60T6hdbuF6Ff~JVWADqm36QSRUPULfnH1xEzMQaEWxw3V65vgba0drYClBe1p5AWYirGS2HCWexO6B4E2Z67PFK3vZbq-wDYKESjbtKWzMxL-rleHEEMPtIIs0AKluGZqn3lYurqJaKd6yGnzpE4Y7xF0cjMPLfQPyTD5AJeLdTCOy-L-jGYaOy--cdSkRU0rbzN66RuR6Rsf8w8AeTf-OMJnrOg35ANWC6l~zdHigt2B6ciw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">228</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000269600200002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">90</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsp010"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="391702"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000269600200002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f16%2f4%2f213%2f391702">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000269600200002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (90)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F16%2F4%2F213%2F391702" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/391702'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transpositionduplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="High-density Integrated Linkage Map Based on SSR Markers in Soybean" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsp010");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f16%2f4%2f213%2f391702">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

